<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/1965" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/1965.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/1965.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/1965.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/1965.atom" />
<meta name="HW.pisa" content="ggg;3/11/1965" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.113.008144" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Norris et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Variation in gene copy number can significantly affect organism fitness. When one allele is missing in a diploid, the phenotype can be compromised because of haploinsufficiency. In this work, we identified associations between Saccharomyces cerevisiae gene properties and genome-scale haploinsufficiency phenotypes from previous work. We compared the haploinsufficiency profiles against 23 gene properties and found that genes with higher level of connectivity (degree) in a protein–protein interaction network, higher genetic interaction degree, greater gene sequence conservation, and higher protein expression were significantly more likely to be haploinsufficient. Additionally, haploinsufficiency showed negative relationships with cell cycle regulation and promoter sequence conservation." />
<meta name="DC.Contributor" content="Matthew Norris" />
<meta name="DC.Contributor" content="Simon Lovell" />
<meta name="DC.Contributor" content="Daniela Delneri" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Variation in gene copy number can significantly affect organism fitness. When one allele is missing in a diploid, the phenotype can be compromised because of haploinsufficiency. In this work, we identified associations between &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; gene properties and genome-scale haploinsufficiency phenotypes from previous work. We compared the haploinsufficiency profiles against 23 gene properties and found that genes with higher level of connectivity (degree) in a protein–protein interaction network, higher genetic interaction degree, greater gene sequence conservation, and higher protein expression were significantly more likely to be haploinsufficient. Additionally, haploinsufficiency showed negative relationships with cell cycle regulation and promoter sequence conservation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/1965" />
<meta name="citation_id" content="3/11/1965" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/1965" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/1965.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/1965.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/1965.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008144" />
<meta name="citation_pmid" content="24048642" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1965" />
<meta name="citation_lastpage" content="1977" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Matthew Norris" />
<meta name="citation_author_institution" content="Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom" />
<meta name="citation_author" content="Simon Lovell" />
<meta name="citation_author_institution" content="Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom" />
<meta name="citation_author_email" content="d.delneri@manchester.ac.uk" />
<meta name="citation_author_email" content="simon.lovell@manchester.ac.uk" />
<meta name="citation_author" content="Daniela Delneri" />
<meta name="citation_author_institution" content="Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom" />
<meta name="citation_author_email" content="d.delneri@manchester.ac.uk" />
<meta name="citation_author_email" content="simon.lovell@manchester.ac.uk" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Allen;citation_author=H. M. Davey;citation_author=D. Broadhurst;citation_author=J. K. Heald;citation_author=J. J. Rowland;citation_title=High-throughput classification of yeast mutants for functional genomics using metabolic footprinting.;citation_pages=692-696;citation_volume=21;citation_year=2003;citation_issue=6;citation_pmid=12740584;citation_doi=10.1038/nbt823" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. M. Cherry;citation_author=E. L. Hong;citation_author=C. Amundsen;citation_author=R. Balakrishnan;citation_author=G. Binkley;citation_title=Saccharomyces Genome Database: the genomics resource of budding yeast.;citation_pages=700-705;citation_volume=40;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. M. F. de Godoy;citation_author=J. V. Olsen;citation_author=J. Cox;citation_author=M. L. Nielsen;citation_author=N. C. Hubner;citation_title=Comprehensive mass-spectrometry-based proteome quantification of haploid versus diploid yeast.;citation_pages=1251-1254;citation_volume=455;citation_year=2008;citation_issue=7217;citation_pmid=18820680;citation_doi=10.1038/nature07341" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Delneri;citation_author=D. C. Hoyle;citation_author=K. Gkargkas;citation_author=E. J. M. Cross;citation_author=B. Rash;citation_title=Identification and characterization of high-flux-control genes of yeast through competition analyses in continuous cultures.;citation_pages=113-117;citation_volume=40;citation_year=2008;citation_issue=1;citation_pmid=18157128;citation_doi=10.1038/ng.2007.49" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Deutschbauer;citation_author=D. F. Jaramillo;citation_author=M. Proctor;citation_author=J. Kumm;citation_author=M. E. Hillenmeyer;citation_title=Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.;citation_volume=169;citation_year=2005;citation_pmid=15716499;citation_doi=10.1534/genetics.104.036871" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. B. Do;citation_author=S. Batzoglou;citation_title=What is the expectation maximization algorithm?;citation_pages=897-899;citation_volume=26;citation_year=2008;citation_issue=8;citation_pmid=18688245;citation_doi=10.1038/nbt1406" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=R. Duden;citation_author=L. Kajikawa;citation_author=L. Wuestehube;citation_author=R. Schekman;citation_title={epsilon}-COP is a structural component of coatomer that functions to stabilize {alpha}-COP;citation_pages=985-995;citation_volume=17;citation_year=1998;citation_issue=4;citation_pmid=9463377;citation_doi=10.1093/emboj/17.4.985" />
<meta name="citation_reference" content="citation_journal_title=Sociometry;citation_author=L. C. Freeman;citation_title=A set of measures of centrality based on betweenness.;citation_pages=35-41;citation_volume=40;citation_year=1977;citation_doi=10.2307/3033543" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning: Data Mining, Inference, and Prediction;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=X. He;citation_author=J. Zhang;citation_title=Why do hubs tend to be essential in protein networks?;citation_pages=e88-e88;citation_volume=2;citation_year=2006;citation_issue=6;citation_pmid=16751849;citation_doi=10.1371/journal.pgen.0020088" />
<meta name="citation_reference" content="citation_journal_title=TIG;citation_author=L. D. Hurst;citation_title=The Ka/Ks ratio: diagnosing the form of sequence evolution. Trends in genetics;citation_pages=486-487;citation_volume=18;citation_year=2002;citation_pmid=12175810;citation_doi=10.1016/S0168-9525(02)02722-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Jeong;citation_author=S. P. Mason;citation_author=A.-L. Barabási;citation_author=Z. N. Oltvai;citation_title=Lethality and centrality in protein networks.;citation_pages=41-42;citation_volume=411;citation_year=2001;citation_issue=6833;citation_pmid=11333967;citation_doi=10.1038/35075138" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=N. Patterson;citation_author=M. Endrizzi;citation_author=B. Birren;citation_author=E. S. Lander;citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.;citation_pages=241-254;citation_volume=423;citation_year=2003;citation_issue=6937;citation_pmid=12748633;citation_doi=10.1038/nature01644" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=D.-U. Kim;citation_author=J. Hayles;citation_author=D. Kim;citation_author=V. Wood;citation_author=H.-O. Park;citation_title=Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe.;citation_pages=617-623;citation_volume=28;citation_year=2010;citation_issue=6;citation_pmid=20473289;citation_doi=10.1038/nbt.1628" />
<meta name="citation_reference" content="citation_journal_title=Statistical Methods in Medical Research;citation_journal_abbrev=Stat Methods Med Res;citation_author=M. G. Kenward;citation_author=J. Carpenter;citation_title=Multiple imputation: current perspectives;citation_pages=199-218;citation_volume=16;citation_year=2007;citation_issue=3;citation_pmid=17621468;citation_doi=10.1177/0962280206075304" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=E. N. Koch;citation_author=M. Costanzo;citation_author=J. Bellay;citation_author=R. Deshpande;citation_author=K. Chatfield-Reed;citation_title=Conserved rules govern genetic interaction degree across species.;citation_pages=R57-R57;citation_volume=13;citation_year=2012;citation_issue=7;citation_pmid=22747640;citation_doi=10.1186/gb-2012-13-7-r57" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. L. Y. Koh;citation_author=H. Ding;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=K. Toufighi;citation_title=DRYGIN: a database of quantitative genetic interaction networks in yeast.;citation_pages=502-507;citation_volume=38;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=H. B. Mann;citation_author=D. R. Whitney;citation_title=On a test of whether one of two random variables is stochastically larger than the other.;citation_pages=50-60;citation_volume=18;citation_year=1947;citation_doi=10.1214/aoms/1177730491" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society A: Mathematical, Physical and Engineering Sciences;citation_journal_abbrev=Phil Trans R Soc A;citation_author=K. Pearson;citation_title=Mathematical Contributions to the Theory of Evolution. III. Regression, Heredity, and Panmixia;citation_pages=253-318;citation_volume=187;citation_year=1896;citation_issue=0;citation_doi=10.1098/rsta.1896.0007" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. Rhind;citation_author=Z. Chen;citation_author=M. Yassour;citation_author=D. A. Thompson;citation_author=B. J. Haas;citation_title=Comparative functional genomics of the fission yeasts.;citation_volume=332;citation_year=2011;citation_pmid=21511999;citation_doi=10.1126/science.1203357" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=D. R. Scannell;citation_author=O. A. Zill;citation_author=A. Rokas;citation_author=C. Payen;citation_author=M. J. Dunham;citation_title=The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus;citation_pages=11-25;citation_volume=1;citation_year=2011;citation_issue=1;citation_pmid=22384314;citation_doi=10.1534/g3.111.000273" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=P. T. Spellman;citation_author=G. Sherlock;citation_author=M. Q. Zhang;citation_author=V. R. Iyer;citation_author=K. Anders;citation_title=Comprehensive Identification of Cell Cycle-regulated Genes of the Yeast Saccharomyces cerevisiae by Microarray Hybridization;citation_pages=3273-3297;citation_volume=9;citation_year=1998;citation_issue=12;citation_pmid=9843569;citation_doi=10.1091/mbc.9.12.3273" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Stark;citation_title=BioGRID: a general repository for interaction datasets.;citation_pages=535-539;citation_volume=34;citation_year=2006;citation_doi=10.1093/nar/gkj109" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=V. Wood;citation_author=M. A. Harris;citation_author=M. D. McDowall;citation_author=K. Rutherford;citation_author=B. W. Vaughan;citation_title=PomBase: a comprehensive online resource for fission yeast.;citation_volume=40;citation_year=2012;citation_pmid=22039153;citation_doi=10.1093/nar/gkr853" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=E. Zotenko;citation_author=J. Mestre;citation_author=D. P. O’Leary;citation_author=T. M. Przytycka;citation_title=Why do hubs in the yeast protein interaction network tend to be essential: reexamining the connection between the network topology and essentiality.;citation_pages=e1000140-e1000140;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18670624;citation_doi=10.1371/journal.pcbi.1000140" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Variation in gene copy number can significantly affect organism fitness. When one allele is missing in a diploid, the phenotype can be compromised because of haploinsufficiency. In this work, we identified associations between Saccharomyces cerevisiae gene properties and genome-scale haploinsufficiency phenotypes from previous work. We compared the haploinsufficiency profiles against 23 gene properties and found that genes with higher level of connectivity (degree) in a protein–protein interaction network, higher genetic interaction degree, greater gene sequence conservation, and higher protein expression were significantly more likely to be haploinsufficient. Additionally, haploinsufficiency showed negative relationships with cell cycle regulation and promoter sequence conservation." />
<meta name="og-title" property="og:title" content="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/1965" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Variation in gene copy number can significantly affect organism fitness. When one allele is missing in a diploid, the phenotype can be compromised because of haploinsufficiency. In this work, we identified associations between Saccharomyces cerevisiae gene properties and genome-scale haploinsufficiency phenotypes from previous work. We compared the haploinsufficiency profiles against 23 gene properties and found that genes with higher level of connectivity (degree) in a protein–protein interaction network, higher genetic interaction degree, greater gene sequence conservation, and higher protein expression were significantly more likely to be haploinsufficient. Additionally, haploinsufficiency showed negative relationships with cell cycle regulation and promoter sequence conservation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9mCmt4mESgCmYSjsJWgbb0zQCID7Fbgu44W4de6vPj8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459576","apath":"\/ggg\/3\/11\/1965.atom","pisa":"ggg;3\/11\/1965","ac":{"\/ggg\/3\/11\/1965.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/1965.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1965"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1965"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/1965.figures-only":true,"\/content\/3\/11\/1965":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kT6hiyEhx8hLRu0UBPzTloVTns7Qho+XVmt7eP5yK90Z6b0Gszk\\\\\\\/EqeC\\\\\\\/SL67mj4y5Zo9\\\\\\\/S2yrbCDWYwqa2GawjUrClv5mUXBF7Un4PXT8+UBgF1R+B+NqduRDNYGeT9ZBlmNx\\\\\\\/CJGcqK8OGpfz2NLBuXvWwfyPRy75aW4Py6trtI73rlWNkFXY9PZL3SPvNY6QBhPkAz+1iguLy8BfuaZXDRbWplCZgY3ZVIwXHf4HXAo7oydTb+Q7yaq8HTyk71aTp9z2OnnaNflUrH9Y7cojP\\\\\\\/76QPjdHsMDoJ758cO\\\\\\\/mcpO36MNjpFkeZS7VarHoIoX4HseGeLaVuDXSlTby\\\\\\\/vQTl3rH9A8xIWzq6tV7mPDJJE8wyalr+5pScjBnJzicFGKI\\\\\\\/JlcWBHg2oZnQwgpJchopxavHrPZ69guF\\\\\\\/JFm6hmLBTlJ8zIzGCD31FugNQnYcbwihvYbvYkWGzriOhdoSoTwUrFLaeD3yBdvH4AZ3XJlQxTXY9VcOwWkRLypdPs\\\\\\\/+EA1MOkUcEunEls5ZXFOO\\\\\\\/LHnwE74g+oDdwr3UZsdt3lwqFeNdQz9WCBLJigRTBTQYAEyXoB\\\\\\\/Zf4dissO+\\\\\\\/h2PmrJBWZTktjsvJcVyMpzoxu4+ZgIrHBovxf4hZZVFEcXp80rLAlF\\\\\\\/JGk6Z5cbgPxZWMr0QvpJDM86QmPjDPO9BNij+a8\\\\\\\/KH+OkdPfDzpHx4waBANhunQuZBJPZIqMJHnu1VOFsvTAL+nAMLwniOO5pdfp7EgJs24AjY3Yulh+eUU7pGhHcRG8Wj8k3EQ9ZHY8ZvoH4HrAOigkjgfFYYDAvY71QIpSAzmlSmDZQv8tMH3\\\\\\\/9mgJZVhXZoIWqLoYKqWlw3XZgrrhCAUuEqq8R5C+5rA04fAXujvhxSgkXRfILTAnHTkkHkcJDBLCc7KPpwo40va52ZNFTcv8461m9vpoA51ZYh9zKLZfmo9MxHSQqD4iqfl\\\\\\\/bj0R4ckuqDvQSa2s7aQSAUAqEYWjWBAotb3No8RL7sqeiVf92AsCf9ZZqK7TKTER9RCxMPk22gL0h7AhuyBuSs54GvNORh6i1Qpy2Yx4WDFXrMEsuvGHdVrzX8hyJ8dnMJqnYjD51fGgWNAe2g9t\\\\\\\/jkyMGwPi5s\\\\\\\/RpxWAJ8Hn5Qp\\\\\\\/K0i2A1JJmClsZaft0dUeR9dbRd1vB20+vg1kTtZsoP2FiB61qwtF776pbRi8Y8d4bktY0EeAdGBZkOr2whbb4yh3M3RikIr12hs+EbfbBr1SzDO4Q1NW7vUWkuKehgUexkQrRq0b3ROy\\\u0022,\\\u0022iv\\\u0022:\\\u00225TkXIyoGtDu7iEQxbNhHOA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002242ea8aaac0b5cf70c51b34a07eab2d792198ffe370ac9daf1043ed3ea0bf5d17\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AmlB7PyUWztCWM2FwkiVrSqaeRblC8BGu3jZDl3kS37nrd2THQ1kW2PwqbnDzYqHSlleVP+oHIhrEJ++16v+09mlc34I3+Xu2i4ZLWHr2YTEvUI4MogolQn07Nc0DysRYKyvHeAzTe0Qlf0o3mqdv4ejxMQHI0MAzqodiF45uoJcnEeh8egozXl76fJ0gCwPgx1INupmQmpQlCwxbRRGL\\\\\\\/2RdP8fTmykW1uveThbzRvWE4L\\\\\\\/1ychUJUDCxMiPnrP0wCmqq1CHXYjC5mCvtT3eQS\\\\\\\/QcV\\\\\\\/n\\\\\\\/c+v8b8TxlqaF5lDRR+Hog\\\\\\\/1tqxRPP5dOE\\\\\\\/1HFZuzqCnYQlIWyTWTilxTIq8zh+iFSpx8aOcxjDxq0w0f9b+bhLJN8jH5hw6EjNbIH71C3qJJ0SHWUQE8XVElanGV12gti5EcNxf3D2ii5DtylHY7Z5gydtSPwow1eiOOEZTcN\\\\\\\/QQBWQYD7Q\\\\\\\/uyiwPl4u4wobVxzris5Vf8buADaltrZEzJv8UhVJf9FxZ1KRCAkGEtofJhZI0bc1d0UP2Orv9PJNh\\\\\\\/OtlPYdfE71ATp9ONR7VM6mSZ4msKBL1G5LdfgQWENJxOdzTVVMtkDcKRQ45+Jgj4qBS637bJWTSa+k6FTAWv3dTikxub5JZuVxb0+4O\\\\\\\/rR\\\\\\\/Z0ZQSBCepTBDkXbnYUJzvy0a6x3vlFh9R0VEIYdXk53xsTY2iNsKk3oWKPU0nmGNAcJ2LnVrTbhzUsaEXTjaZbAtuzzc3Jbom1lqbIy964gr\\\\\\\/1FV1OP5ZR9QL\\\\\\\/mYnRJuRieL\\\\\\\/UJ8NbCKJguytxcnvQVWU0s491AGTJLDNrPeHGMoV4PE2RQ6ZSNXTRfNSwkSu9HLuo\\\\\\\/9MCfKBpWFibUg1+jPxmEVh5zVCAkNQfBdTBO3IrwW8blEZTZgBupooMc+AW9V0Qdeegs5JlIP773frL2zlSFSxDDmA8k\\\\\\\/pm+iYLMroRaUx+6Z8WfPEgGkSDPBDpCQfVvjkI6a3c+b0OH\\\\\\\/2RbDxmVi0j6NfRRzjeCJGrMrL9BWrRMDI5UbJRDmSZl6rxIrIuSMMlr7yknFe\\\\\\\/GTfwctzW03A6uxDIycIXjqrO6quJAAc9X\\\\\\\/drQQSJq+JKPTTvsYCAGhNN+K9KTrUs2ZC\\\\\\\/jcuq0Fno\\\\\\\/LRV47EIck+s+HFBm5zQBTdxPzKx77PdfRpbNfRxZoOMEoAJSv\\\\\\\/d+rt3P2U2e0069AAvKvOs97qTP3uSIqFuU8vdpO1rQOvJYK1HB0bqIDWtZkFXx72IYhAOpUCUik8vGz3jc2iwkqY1IMNUpurmoZ7yaa3biIs23ptpTCEipAcIxEB6fK0s3nNKBkMgewqyt4Chp7dbzgwXItk8UdBhON1DN3mjs7dSfnlLt+\\\\\\\/6NWwp9fz+OdYQJ1yd6+DKGPfQEwk+v+XiS8BwTS+1wyx9yNOckUL1J+Gc\\\\\\\/ZCK397jdZWCTTuAr+8yr6CWd3Zw3Vn6B9shCkKWnmVfOI2cIFIA58h6xmqINkK65KBjmAubnrou2LsfXvl1gNWJ+bAn2DMqCi2zqKhIueIoH+oO4+OmbDXEJCa5SDgrG5xrF2CnfleeShhbSeAdrhyZzopNCGOaEH4BZLENvRwDpURzxqaVf9YUcDs73pbs83Cj2ZsNDPwSf00z5hxcTzx\\\\\\\/VieDFxhl2dH0L0JevzhzCyp68Kz426IciNKYQGo8b21isZDcb4CQSeFp7bgYjbE5o7H\\\\\\\/ZDwUQb4g\\\\\\\/GvHyGo3jjiTH65yvPb4+0SPa1F7XjhX3dHQyOGNJ3LSBpJRfWqt6ALiE1J3HVd5zd+iDu\\\\\\\/ITUyrXTo5hRQdiVH6CRfABGUV5WV8s9EXYVmsKle9uN8G3WBqmtiUWkp2PLzuZUjYm67259xjAWuxY5evP6YNDSZ6Yt8AsMmb5hjIlc\\\\\\\/Ys3hR5walQllAfGpApPAe+zAwD2XC3NKqlC8I9WB5Hy4hL4SM\\\\\\\/UseqTpuQxdIVGFZ07PU3XRj9pR4Y7T8QuU1hyVDceP9KcNrP3Um0LCvFAtJlsG4xfseUVLon1iKwE9+mxCH\\\\\\\/RW8deiq+2tKUC5Ns6QGLV2WUhwe2+5rKLw0NTt2G6K4wglQT+Tyj1IjOXOJuXs4j7dfL\\\\\\\/KBneQtm0cz6vzKYUrYY2WVdWrPEemwv7gIdVnjKqHA+nHZnRqTB\\\\\\\/sG6Byh7r3OfyEHkwMfeXyEbPKhIE23kibzYDMNf5uOfvhNWi4XypQH+lwYnLIoNfulEMXJBuvqcaYj8b72bik7aEe+UHyQgXJGQaEe4w2mQUW+Ay5BSCE7GEOgtP\\\\\\\/0c7o62k5oWcXqnY\\\\\\\/UIq95yxhTr0qwWgSbbdsvfgXitwCQnc6ryBhMM\\\\\\\/JIIxBhVVPjjV1wNsBXM6v+Ok6jJDXgyadLotMHKzUeQrbDmI1Q9Y75CyTOYV7oCeD0lciihp5MgrfSOdBH5a5rhXY3tv8nbDkAIhnKyHkms\\\\\\\/EIkUBfXQHbJ1FfGr2sjDacvz\\\\\\\/SkwBebhBr6gyWFJwId4N+xfl\\\\\\\/S30wc5HOczMPgD8tXvis7YyD9JB2PJE11aom5Fxxrgc\\\\\\\/aIPNgNGpWULcR3rzW8aPFIyK0xCyJPsR8LE5hnroGIbDCCJ\\\\\\\/baIfNDeknADv7\\\\\\\/FUe8cdP+WA8nJbixRFtCx4QRQyzZgelwSXNs5aqf3rre3DT9VLjIN5dBVWs0W4L0WCloEhr503DgdW3bPIDeO7FFm97wiKncMEiwEcwnwqE0nnx54WrU3tN8IEhIOKi2ryPwzAHOCSGoKU1sRIY4fxfEhOTt60d3sqvF04TRwORNUVBMkgYRyV+Fws+qmi9QsRdqZambqv6yOYrokQE41+ywllaUpLP+0k42dkthpZWiZ3DB0\\\\\\\/x4tfv0jJv7Fj5rnTvqH2+C3AOWJVvZtfUdjAyu2RgzgZyBzV7qHLqYnpeW2kXHDjIFYLV\\\\\\\/u5HZE45VENFui\\\\\\\/btRWTqdakigf\\\\\\\/m4kzZ\\\\\\\/SaZb4zZi4G5hR\\\\\\\/ouUSQ7ycEAbDkuB2iDR9LzlFy21EIgW297Gp8tIcZi1TMPr03IHmLHBXsrToa\\\\\\\/\\\\\\\/O1ZoIwD48StynAyOd6DEZSmVIbyZppH9dh4Gbg\\\\\\\/mcUEk\\\\\\\/ao6YU33GsuAq5D2ALjFMqmAeR6SrDVAQUlbJ46h6py4DvrcWOvHWvMQGM9J+tCIZCREKbBx0ThVg50pytydIcTb80pgObfdy8bBROmRSnkhinUuZsBya0B\\\\\\\/EzbkPJMbXYtIMAGPgrfPADcJ+zAClVJdtBcqyoscIWKyBP+OXET6N0eAdd\\\\\\\/J7X0lCnFhI\\\\\\\/XOuLXShCgFtGObiEM\\\\\\\/SXKspwvbg8qWwZjxAbEsGHdH4b+yt16+jR4s5ifvGvy9hiKL44Vt+NuoGbiIR2mMfSX7QcMtR7CKxFayubHtFCtsnrwunn2qCR4\\\\\\\/eNtk\\\\\\\/snxKW9+nkTSYrGFS0heuVOx\\\\\\\/vqlamE3+yvgHbbKx6NRdkfb4fvBEah7C2czMHZ5lgkGFN1XnWEa21GQIzjk5GVmZB0IK03f1XZ5PArUQuGwouMsGpZYENrfzzZ14b7wrAhYfH385EvDc+3UbJDOVf8NqwirNRnPhD89Juub8o1MrgKbMzzVCR2pAv5p0oMl04E32yW7A0HSGXtPynNXiUftWyGc8l9fVcd3US5uR9sROVHQetsUF5daaGtwyjgiYvFr4YtQO2mVB4xOLzIUWkfVoAqUyiIwrbQbhguC74GOUG5tSqTYQH\\\\\\\/P0Gwfm\\\\\\\/yROOP2IPcfGv\\\\\\\/GRS8Sw6mML7mNEsmUndouzLC7mzujH84n05XzxoA2cV8Tv2UVNAkp1lsTVTKMB47mv0v7fPNY6lxSodveeTpqI95pxDj9pbiQlaPTI9RlZvjcW9BFc1XiXAVJ4oE\\\\\\\/gKxs0ak4rRDu0cpZ9PX0mi\\\\\\\/CXOEqAZsXd3JKHZGxQGoFkZCu6i9MqkGUai4ZRxtwMxz5pJYuw0GzCB9XD\\\\\\\/aWYGSJkJH6F5SZcd1qgoYHhIWR75FHV20sm\\\\\\\/jbaJBzZLhFLQwNawco7ZOTmhUEWMuMET2MSI4Y8cPRot13I2Uqz4\\\\\\\/FMCagzjYN4CBfUtdUKijbISUuGVbb6xUe3nxkqhsEwU2RPMoyMXP5JE3ps\\\\\\\/Tbn7MS9G4C9sgXofrvo1sh4Pv8pXVp5hQHuzWs+Mlqgit6ms+Xa8ZBaa7anSSq8ES5E6X13eKpOxBh5gxFqR0iNfdenvVihOMuyveYJNsHeh393MLMv6S+c7zSxCV2yWvltYxlfLzNNnIIOB\\\\\\\/FcLqSZGgoirwtsSQlVnyB9sTk+w5a\\\\\\\/vIzUa3SRluu6SXfzm+sz0PtnGb+SJlpGr6j9cWE4xmtdVkVv4Tk7s2NFI8RSUWr44N7H9mdJJk8z8J86Jz\\\\\\\/1PkMzuDbdDlmJHBqNC4\\\\\\\/64AEf++3jjUr0aBTIRY71qgGPvC8I3w9VTCxrkUqsK8Z7pG3lBhtGlO8QVFGzGeDYIYjSAIxVRmOipuvIVLd2w+2pVLJgRK60+ijLYIn46MmFuGHSFlKp8bQ6+9x1ZwzDyiS0Y+iRU\\\\\\\/YVyPiK4SbDSHuOc1fAtjjLWBeD66OC9gCRtrmaiAUVIVjNFV6mZMthG04V8MwgQcqT7WetPZSbs61KG7OWfT7VmYuvAsFnepI32wMs8CTY8Kdpldwwa4eAaDYT\\\\\\\/CGSV4JG6a9DSb08zqnYFGu+aniytmiFIj5kyvpoEgjxNDfDqWw9JCcP3na+paU81gbkEfguxvpWqKQZAhcxmRhLZUdwtkRnOiqijESHiGBURYjwFya\\\\\\\/\\\\\\\/TaDS66fncb1Oxwjn48uTOnbzSQYnQYrp41dRmNSkb35p\\\\\\\/q7DOdBZHTmQ3QU0d3X5cWYudIFfHmd3zA9lqw31eRRWquVR7WU3e7thRBhSj1wMSZ4oKW8LMibLkvC7Z2JLwZesgqIFPQTMf6VmiUtbpESaU0vEdOV\\\\\\\/h77VF7ffwVCivdPpfVTabSCKXaxH\\\\\\\/4c89GwvcE2xE2t7GboGN12qp3A1ZI2iU6AejUryL8W3qzq46sGy1naSdx52RPjKe01z00sU2T4koQ45e+O\\\\\\\/uFbvmFaDPGZGRYsYOB49irwA\\\\\\\/JbTgoE3OjJQqkuh+1I5bAMB9I3z7fgek+BKv2MumXpgJAgMPV4V6l9QWPOgaGv0\\\\\\\/usz4YI+rglmVPmPLdZNtcNnFZLKzmqlDbUy2zl9h+G2KgHSNbWkO7h2ZxSNz28NH2+yZF08z3iFQiZchav\\\\\\\/ZdpjzybHPI9MeMzS\\\\\\\/4j65JZvjd80yYmpNxPZI1bu0mIrQps2TNpiRoXeZJ+GVeEw1NS8fV3J06hw4mJMiBKjXsuOwvA1vW+\\\\\\\/LTW0QKHVxPA+qJXNOKWhJMMSw3aTryrB1bVrHT\\\\\\\/eHkHVp2aZRiYw25Pi0LCrExmbgDdXt7ZBKvDzFt4rFlyWnls059iTKWlfBpBZB3b+JlAuKGpChXIz6nC7l27RURO+QTarZ6H0KhqXj8jAeKkqu6xjGvGMX8bTOe+rBk\\\\\\\/v2F3bKTJMZ2uDOSTROSwBXGGKzxMgypAVjTqdYdfPSZZcJba6wZDOWc4HIIcg6XQtzDEkg7XE+SBKOjTv6RoJVeC3KpAP9t589S7mL8sRpq3ECbwNfhnAtn0uxyvzNs1namR35DQOaCx3n5yNaFMiWlp9osC75J4QPaIRruK9L1ipG12Qv2\\\\\\\/8Hyn9KkVo+bYO4+9+v8IiQeMsAHcbm1yB+e+7xMlNFVysbp73Mng4SWEgmvUfG7VkEeu0YkV4YnBwei5QFesGnJIeCShX6I\\\\\\\/jqRZNFF7rISbJOQQ8x++KouG9ThZMKO\\\\\\\/gBCY\\\\\\\/AjC76qNmHXelQ2DJoQ39Qn9jnU+ze04JprboXNK65y7f7Kcb5BXPixnq3jZ+Ozc1Vu8RGjgKCvSRv\\\\\\\/RvpzzYNcaDaQWK9IOlKHOtRRFZBvIA+dImf5\\\\\\\/LhO5lnJSucvACTCRp9wsedUS9FS4401BDPMk+WQZyWnQENMqY\\\\\\\/2T7jL2YHNyfNkbOlLXtnToqIvQWF2k9Fz9s0wgRZnfSRj+Tbe5JJs4QuWFFNO0AWI\\\\\\\/iQivGGxWtz\\\\\\\/sAiiKz8YK2wwvuYYD5jExidCzeoCMGh77jTErScoimUbNzE\\\\\\\/QrRQk562gzKZdmwUdTA17bUSJP8DEe6kSGG2SaKZQoQvD3BmGjBxh6pt9caNnQXDTwD\\\\\\\/TX6ah9gxdRt0RzweAmVr4PS6AiUKXceP8XWMAT3qJrZ3DfdL+PFO70uGKpGkDtDuYAzfL5cl7PeEYIkTJ1dujVooL\\\\\\\/0UEHBGU4do5pTjkZ7RDFF8oIz+PJaqQ\\\\\\\/3w2n\\\\\\\/bicbfm\\\\\\\/Hrde2yQNgUAG36U5Yl9uTVL+mHy\\\\\\\/qZToKarq\\\\\\\/dkXfLQlT1ttfe9i7SEszrhOQs\\\\\\\/nyIaAS\\\\\\\/9gVno2UpyQKPioL5t2FnfgtCNWhaHx+f246dva\\\\\\\/0FB8A6TUwGGArBbm6hw8NTapuLr0ovadHm6fR3lxiRerYvmxVOUDzMZ42A+XNYuVNyYtLK01Kz72eDXvF16r0qJIev2D09nrMhtdWG9i083HHAE4mBuJ5vK5tJWheao0kCakQvVP6Far9ooN9AigzWbSRg7NigBYqmW+LBdW0OMDH6RNAupV7+r3jF+f3\\\\\\\/XL5sF8q3t+CwrpFTDjuJsU+ESbGwhLn3oaEpEg594ddaYLE0VTJ1hOKGgE9TMbOf1EAdDct9vbHXcKR3qHFlobNW1XLjjvGRjMXWeUdd24GRmNOjJdgl+swuZx4rtb7vq4DOoYGFBleBd+11gd8vWCe5jPTiTvDpgkF+hbW0eOdWZkddwl72q3DudGSJY84RzZb9psK18B5YifGmgetsap39\\\\\\\/MV6+dvLeCEEqJV4gOfA5Mkoa8MWw\\\\\\\/Kd9BxxQu0WsdwdP5W08iB4DjPkTDdj\\\\\\\/KvLhxxmcu\\\\\\\/OZ5MsB79OxbTruprO1dNdiTS5ny9duPYT7Swy+5qLFHcD6qXwE5sDYh1mP\\\\\\\/v47+FIZLzf13TUxcNmeyQIvMQ\\\\\\\/VxUYzS3LzeBibS9x089J200wtN5aPuIXmrYigCT2SEH5\\\\\\\/zNqZLOKOn04HPzHaFo7nhTAROoIWsFWWsEJBR8ofBisdU73JcIbDpOhsfI4giZdSbibOToq+bV3fuw2L0UBxocKvdvpNpbv1w2rSrzR9viHgZXayFgqXJFrPuqsZl\\\\\\\/bOmGZZxc3MchGXDJQuV4gw8Pm\\\\\\\/qWnaRhA4Y02umSuhRC7siED+YCRHXbZjU7UtZUnFwB36k\\\\\\\/f3hvPK3bt5B\\\\\\\/H53BVqDugZ4BTyNs+ljMWTCbSbyJdzax\\\\\\\/LphiXfqGi1MPBOAs0oSDPN8eveLWc7Mn4aH+2mVXzoXtu1ManqXXreDBU61y9kmQvsbCYuaFuxxxR4r7nBNUr5dm7+CA9Bq\\\\\\\/U6InzpbVhglFU9dN\\\\\\\/IkXMkHwqui5WqoHClZxeDQFYZx3NGtUM47jp6CHfKlZYryy+xogukO+vlfGfeImFsZ\\\\\\\/yCiTfCdbhghRo72HZMeQcBoDtSObvPYd\\\\\\\/u79l\\\\\\\/f6m8SfMENpLii0CrLFcVz1Xula2bX2FrZHkFcVPXds0Qr70SI6dxdALxBVVecOhnANUWl8O0GKd3jPUe9SmSlazPvJZFGJCGgyG8Wsh24hV10U2a8AsGSSCz38qAFWqna\\\\\\\/VnQvBy03WQQkqPpfsau3M3MmCtkWO5nqb5QcxnOf\\\\\\\/gRjtvwUDTBzRd8HtXdWusME+q3zKfJ1WYLIiSb2NZFriLlOMf1UNnkAtFpCIhvTTholQeH9aF5xQY7w3BCLBn3ZQ27WkvkPsZuHa4MdSMhHBRyPx5fkG19UL+KgCVvcKym+1LKeEKXpU9Uep6nIK5QWekOvpXN2BAmQjrpohCfumgObdKDayykF8PgLHgkb07nzdd\\\\\\\/+R6CZUME8GyTw\\\\\\\/EOLqp3WHryAS2zuPwkq0JK0eH+\\\\\\\/Q75tP5+mx+gjQpDwi2Ir3h4EraU1NsA05mylKQW4fphYh7ySojKm2L2N\\\\\\\/n7WT++mkr5IzqQudohyJzLflpZcNgR2pYVl61oR4Gf9iSIcfV5b+qbY\\\\\\\/X2WiyvQJOznhbL2iQiKS6aElGDbq4dPKREGb0Qljk7CaZLAYj+rOzPfS7YAsHUJ9Jjqdd8ZUxna1r86dVSaJnYxW1MIhoMVPlL+HkXoy4b0+4PvwQTM2MAKVngkbJICEPpLn+VnssfMPl8m9GqivxeYivlbIJ9Y9l6CPKPGl1GLDi7wjOFDx8eeTT+W\\\\\\\/kAyuOdTN\\\\\\\/UItGe9QsUfiOAN+Sw0CaiQVBAOwm\\\\\\\/NKu2E0wRziuxbPQU8oohXDaeMd4wbYQTJywTdZYp47Z4MSXdgF3eaMTDPpf\\\\\\\/b8n7KencZ3DfZ3OG3puBVXDnUVMn93laQQmcYVjsIhxezgxggQL\\\\\\\/4MM2JIfSw96r68aen2gfoUGdWUrEsH6FWT3Q3CzZGUBWmetwPrPU8+PeXw0+J4SGdRkYhDT6OF2qBzDRqE9pA13xAEQxBV+T361Gdym4uKgd6YZkKcBvpZcYwXbOKMYH\\\\\\\/48oHxIq2UvyA\\\\\\\/tSXkFB9wvauJ55SYBX8UIR0sMyJ6CJTx5EljOoUl+lpYmeSrm9HFYAEVL1piA2XsXj3\\\\\\\/yHEbRVmA4JwU8jpNjSyrAc0xZOvcxsS4yUghab7SbnIdTkBIn\\\\\\\/bj6Lbh0nNnEn3RRDpVph33kbxwQc6I\\\\\\\/rDegFjynoJStRnpIaSY22GgngNscixCY1GIgN29vSA87THQH2FwZq7Qr28FfAuvBRcOImyXH9h43g49R+6ODcanCeKNVMBg7GKENNF55s+nU3TAwDjUEiVp2IN\\\\\\\/9LiHF7+2YxvyteQjieCAVFBG4skYVoMb3j\\\\\\\/pDBJUTnYYNUZGXzcOLpvG1eVXJz8JRiaVQV\\\\\\\/nzCl4hOvNvIa1Skt8R5NG9Nmfx3rEIbJ37X3bh4br\\\\\\\/CXUSWUTvXa+wpgT7Eml7k0WkfGFZ7O9sGGqA9\\\\\\\/U1ycPAs6MtS\\\\\\\/03AFROqdElltN+Fb3xhEXAhJLLCf1OHKnT1h26VydOOGD7R9HNJ4Bt2C4Z3ama+Dh4qVOFBaGHYz7HN3J8FtwdZeSsDRvQm5TSg+4frkGX7ORQlN2XRJC8u6PA1DGcxD7uhb7uAWwS41gLqki5LiRZ3JN4hSKXRUgcz5lxV5oIosN4C0wBm3QIm0IcHa4AUCIZAOdSMfWRGP+jc3L7FNNiJB\\\\\\\/kZXYtf8DoolHANlSmabOG5htYNz5\\\\\\\/7lmcoKVic9nDXOuzPb9EflFtSXobigwEXAmC8+YKM37KsXlUua9JWeUY87lH7MxdbOFg1BFb36ijZ28od3Q0GE+NGAR9nLfT403AdLaubZpH4g6sRPq0cQ925sppS+1lWH6FJHnj6+TVwhYhakNFwDNvyvUD63YTRqYJ9ohG+58qxFT3u70BbWsLZ9mZTNF5W0tCPbnjQ\\\\\\\/GZC35exiUjTdS6y5iORm+AxbRujdZqY4D\\\\\\\/tE5PMsI0djymrbEpkz74CprjcQej5DXVdXyuTqGRR6rGPMirM0szg7z2xb6BaMFdBqhFg30Zx2BPLYNJkIhQHKA0J+3kz7ZHFW10O+3HCxKTE+CqR\\\\\\\/OVlxUbfc27AbiDuJsKttu0gz6psITf\\\\\\\/c2B\\\\\\\/2cvJxTsO9b5YaOyDoddbxkUSN4O1X1Ua6Q7HFzxe2VdPBvBUSOEp\\\\\\\/9FadPniD2icxR\\\\\\\/S0QNlCzMfk2mIuMqgtjCk9mGx+6bOtLBVZvzPVYPwntQ0ibL06rmIsD7+EJjvlWSDuOK\\\\\\\/aOljPyiIzyCWLLMBOdbU7RDqVoX1iYgfAza6w9Ts5l3hyZZPT6bnU9LGit2+8c0IySrxFM437ZNBBae\\\\\\\/yjPEDdMAYWdFnT\\\\\\\/MLyqj2wxy5pCVzRBAnWAoDBWdUVJmNYfULeTLaQ2wBac80ax7m1Ee36QYda9njaWbmE82C1QzTY8JYBOHVhe7B9ikUasOi+zkqsRSUeJrafL4Rh04MrBqd8eM6+blrvtVGmPDwCiOF5T\\\\\\\/gFDe+XldtAj2BZe15VhpeWhEuYj85BcvDLUk3hQk7NmBKW943nEcmoZ8o3IbUeIiPxb+1J\\\\\\\/BvmYpg\\\\\\\/yGnO2tN2QusBQtXvvpeP8BEuSnX1pPqgKXN9fQdJpcHDVvjwrjrVWGjDNcQ16TqizuUuFQZicdwWvJbMA6scgp8z+D6HGZY\\\\\\\/F0nw4de37el3C32dmlJ95yaS5tVX7oTUt9co77htUf5JV0GPqTZ3nXfxartqpOyLqO+Lpq+eCghLDF6SF2Ymrl5ghQQbQ3XMyfeGqlMd3+5Z011H4QJ20bf3KGdtAMxQcaT8xOlkne0O2UnTV160TFVMMzn7ja8xLkWP35RGGFyQwFRRJVOhSaaHKGaDDr\\\\\\\/OCzSMPIqmpYygQBAKzZZH9w2Zq3O2SAJULNulyZ7m3yJS33UsNUzaAm\\\\\\\/9iNpV8of189bcdQjL0WsoUU69iuG1VEt+eijDi5e0NYe1JifHqUgIoCoi++6rPFywJyayy825Ww8ln0bEq7244BHDLVXcV\\\\\\\/E+Oy\\\\\\\/ghLMtXOxp3DK\\\\\\\/y5vTAGS2MjbD1jRIqvYuf3xfPm58QP4+GBd8CLdEMs9BdgtT1spEmXXemRdkm9bEoSAj8T0M6oSINbXpCGNQu0M6+1FCuEr1irxn1fzmS0pBqE1jRZ7jjg5NljhUYnOWaCYqV\\\\\\\/OUCVA+f4fjIzznQack5uh\\\\\\\/cI7WFHXggHpQ6IySB1GMe385lIFMlyV4qdgfLXSrp338soVF6hxrnQgrOi0L9VXa8iBvorCdwRDP6kQwLCpJBzpkQdmqXe+D7LhviJsRac2IDmz1K1xlMRAIV0yzouCQasjg6aLta0gz8Ciw4SMqZP5Lhx7dkdBkem0tZLOpaaKniHezJRDe1VbgHRjqRY9rPHOLaw2eVrP8hEB7LdYHcDLs6ZVLUBSPBHFJUWGFzqjxY\\\\\\\/ANPXUTBhVBbTc04M\\\\\\\/fpj\\\\\\\/0CYZ65boq555mf8Lgc4EceT0O9uDdDSOkUJm5yEbyJRlAokvW\\\\\\\/vFCHObT4gfIjAgr\\\\\\\/+Y1IhmtqHOigSkfs9FsP1dUZ0ZuSVJ7TQaH4EwjxXUXf3aFHqMEem8pcpU\\\\\\\/jRJkODxscC5jHNiBY2SaRuuIYqqHCH9a4gutfeCK3awuvb5aWi9FjsEXqay7+1DAw7in7OwJH4WXxJz+Zx6G01dvYn2QtP09T+zjnFAXvWkTyKGr+rJjFTsgzosRxNw2TCD\\\\\\\/oKTkWkzqKckd4tUw8BK2OlwL+O77j6b\\\\\\\/YeVGukKncTJVgPX4Ti9xz0M2XG0+KAp27QxvFFmf97ofFSevGATi0LeF8Hjzbk7ITob6Nj43Y+wRB3Zm8przYmWwU7gtaZoOSCrVrlvERKCHPwI1N7q4gSwGQMwXof4tEAynQQu6drq1wkmiOwnuCHWcl0Jzcy9tdwWd8af1eyONHtbFCe9+hEzSkJhOFCdVJqvlHzkixuXnnV+yUpDJIrU3Jg6xs+Mzqg1HvLJGHZJPcC2RU+aL5Z4E5P9uwfn+S4C34mXC1WCs3dTPXGze2oAUQ7HYp9\\\\\\\/GGjkpXMSWudqgYvR8S1CZ2uRXHEuYtXCzY6Rp+leToQHJNhnxQ7ftjjwHtoCn4rMUJStY6igK4Ctt56UJ2+5CmaR38QTyMEgzECDO+Yvfm3AGkqsrUOlrOF\\\\\\\/O1UAQXDezlmKhCUDIJ2BPIeGM8ddEO6kmuNwBkPW7+KpyCAlr5azlbmB6crWAj8dpuMrwHJbcLI\\\\\\\/7ErlcR+ZolJNmOppLzyDhFPA0V08vIx1UOMyfr64MlhlSDuR9BDtDElmKSzAs0sxYenua8zlY3baRMWk5Y\\\\\\\/Mwkds9MdEp6g+8IKT8CThwdkvG0LHX\\\\\\\/60GxjQLHBivsH7vW6u1K8rzgDmLAK\\\\\\\/+wEBxSQUBZlGIi5pOpoHeloyXiu0doTFxo2SkE07bjpBviA3nlXczWEuB5mSzSaUFaJH8Ymq0tjtHQ8Fk6NrekEhsEBTdLzO3VtF1sx89GSE\\\\\\\/EzV9WALIjcHmbq+tepJ7kWtgBHWm8+q5rhhydPtgd4\\\\\\\/Qlda6ut6OcujEM8OB63CGxpzxiR9rSk9uCkrCMeL6UIBAi34KHYzOgEhU+Nhngxk4u2ubiKts3QZOWPG0uNThU2hrGyfcHa78iRcthG\\\\\\\/LDYu12QyQ3bMYpfE0OXkDdSNFXGDd+KMsQbZF8LKIy8+pfApAj8Z0xgid5FlD5eEnxl+SbJs4RaZe\\\\\\\/+AcGiqWR7YmgOT4A+2CdzPpyvpIXFY0FaZ\\\\\\\/Pgc64hYWH3fPlF+DgfJvh\\\\\\\/ndMGkif62NFABHDE+rrhP8fHihFio1EWOa0UzBM9tAODV4cQXbFxLLQscmEl0qh5T1E3Ac9nba8GL2GQ3l3OqlOfFhkTy0ZtYbrGzpDhWv42ynaGnAH+4mHO90Lb88N899xaHwQfQ\\\\\\\/71h\\\\\\\/gwDPf5V+Rq9TLjsOHRFeuMgm1cheHOhBLUvk6DTZ8Z26W\\\\\\\/+i1DIS\\\\\\\/oA5ZQFHF7fx+fYG\\\\\\\/iNhqtV27ZrmPus6SAexgmy69YFtWlxl1bxnmoAA16cAstK+Bxv\\\\\\\/TZKDROrMhOZQUs8JnQn4e7QFWGmgUjknmVBDntHDMMa1bhtoidN1SXs8lYWZKvCe5jRhDDFI2C2XMNJplxu68rL8g+HwVWY+G460Nxckh+L1Hce9uljG+qzSFkh\\\\\\\/cNqSqy5q2l2V9vDwEiKT+\\\\\\\/hCUMyGQaduNcTkScZPb0n1DR3+wZuYFWVp9aqQP\\\\\\\/CByik1CZPY3l67hlu4cSjeU+IZ59nCmLBtjT+nihP7vaHxhlYtzgZMjEWqwPUxXOe3zLSSe3c\\\\\\\/PlP5LkHYkPv6MhTaB+w0FbKQ+Xdg8tHHzjfqSynSsg6rV71JzcNwxky5sX7+9gYteRMGizq1vR1u3fXstpONypmEz4w+LihRhnYeyZ638sf1VJvgXwFPJadH70VvF9jArWL0pkGqfy2xO5EK3WNBPC8VdaAsnl5IRJ7Bg0zt1TaRoWHPq1tgR6D3uphiugprFNRAC57zA4CUF\\\\\\\/r0pHYpNlmMYclQDVuA1Mzx7O7upfK3jUSY7Os+q8RwCDzB3woIa47LSXAO07dftlu+p+c9Sm82SM4leo7g4ASO+6D2u7Z\\\\\\\/8wDJqjTk5xHiTA0R71SLYjTfibQ68PaDGCTTxs2mpF45nWB42Bs2k7j+yn10Rkj8UK9drGA0Je3AfTd2H2EbQx4N7Q3DJccp+TU2Z+xhl2lto2oyVaczcESB3+273+EVQnN+qZ1ZPac\\\\\\\/LTHXFeLDOX0Lh5JGLLDmZ0TJVk9QVJrhqvanC\\\\\\\/8jLvwHc9VCrKTPMdIXGv1UxpRMsnRp7jhFeKHSX9Cn9eLBBi\\\\\\\/UOcKh5kBYrQE6tjhVorlobPjdhZx\\\\\\\/p2XTzfW+d2oYwA8s03JTjWu4GHVevmuSMAI4woyBb5Fn6IBiu9GxMf2aCCYM9DYY74cOGCnFSGDQPUTe0eHGUy3L160N1UuLq21KT4tRoRRLwcUaIfXdnRmLKYPIyAK6Zyi796zTtxHdNC1wOg8CZgi8fETLnWXAjyO6U3ZkFQefJ6++I9bK1H\\\\\\\/yMHTJx\\\\\\\/Or1Voj2FAZBXxNnU7tZ3tXxNBJao1CG78FIw+Q9IUOSKMnb0uv5VN2XxT4VCxNHS6R5I2eoru8YCEZlEI\\\\\\\/9VarqSkQDDLlM6sJXEJQk22+gl4as3xaQfW+biVqUbRSpFZMpbr\\\\\\\/dsfLJyFzz\\\\\\\/dKy4VNR1R9l8xNw9RvgcNPuzvtnIh2oZY9\\\\\\\/7ykU2R1oRib2b1uaAGiDVX2BoEdlUuhgf8TRrjynR\\\\\\\/IuR1VaSvT2b+s7\\\\\\\/2bLXDiPIqW94WSTVP+MxegQsgCHc6MoayL4PFAGTQ3txy3eW1doflQPJ5UyRK6czcPZ4Mt\\\\\\\/EzA1e0f5wzw8f\\\\\\\/qia4eKVeOf\\\\\\\/Q3DJwk68bRjF+ekYThicGUUGSrM2XBmd\\\\\\\/nBh3uj99Y0AJ0NAsxcllOZNzYz3QEwbsB6Qwo4EZ82fg4leaZjhc5jIq\\\\\\\/OsboZIn9DjyA1DB0HHolztO0yoGszDaqatL\\\\\\\/Lcm7n8\\\\\\\/Hp1fk+md5j0bfH7p6j8jkZ9U0UMe72BWuYOA\\\\\\\/hM3UwHcvGnC\\\\\\\/g95f3osXkDhpm9+xuEy9z2yY4NL2wS8zONUGsCEPBUFatHOZQFIRCUWbZbHOBlUPQ8NkpyI+MD6LjO7tGHh0IokFl4Jo+acTzKK7dPHR2vb1ULaoOZZj\\\\\\\/8XWLYq\\\\\\\/v+n7ooHaG7UMjXS7C9keY7bjqVQHvBzwp6ItrbC\\\\\\\/qkAJIGxBikplDFWAV653EYn\\\\\\\/uSUVAmvC7ICp7DLMSAW9iz2Xszy+xx2uHRJtkFv4ty\\\\\\\/igEQmJR5KArexkadALFepma1pu41VjDB\\\\\\\/9R5KrYedzKcm2AV0XC+ec8eK+CJS4bWkQ2a2wvCDUeDe+rqzXL2MF9KgW5kVeYLx9qFooyRjnKN62A7t0ysMl6w3B+AaT9hqiU0RPQbDuTSof+dksjw5ciI7sT4aKsFTZ4wY59ebvKND+w\\\\\\\/eOUKxAVwqcOMOK1w8JxD\\\\\\\/397uoOOI39dFuWPXeIUpmON+mFcLHgeKYYIS5p4g80B+0pQLENTAI0OWz3FGHEBTrnEaVhhIT3SNG4zFjqZ0\\\\\\\/pAwJl3i3BU5XkyZnLss3+Pn9QzRRP7BAZfGdgZDruArm0lA2uSk+CUjq3LKHcHGSwm4pTqRIKmwPvjj+HypIed43HJqF5aQhSx4MKmzm86nb5ZJSicXjELqGZQ+eB9RfBXil3wfvtB7YtFQ9\\\\\\\/E+d8ScWFBUA217fUO4ZNwv12rA\\\\\\\/cQHtXHRP4wOGyW0C25puTEG+8hT3oQThqawBaJPzVFjV8uR+2UbF\\\\\\\/cBjuxCy8+0YpyNcRNnGsHMUx\\\\\\\/xjGLM6APWBjZa680hJhI7lFiyWsymiy3uThRg1g28Bkgl\\\\\\\/dN+s1CU0Co6rVkzjXNZtyN1PqBYkB8dK7GDBLUZUKYvSwkykJ5fQKwZaQH8ExvMuWb+d1Q1Cog\\\\\\\/31BUhf3e7blYUKcoO1HPxh2sn7JJVreFgQtI6jjXrqbA8LWdaH4ZOMbN916DE23Rmpkm9HgEeqr+6rbzPha9rnuzLY11Y+RfWwH76KF5LXIbI7rbnLoVVyPKlXo5Up0je6DwC5wNoXWcfXYOOs\\\\\\\/VMVY9LmbFJ\\\\\\\/jSGuAMmOFwytpbzewNnus9R1YQU08dFyxPmh75P\\\\\\\/Ma3nxpbbVtCJA6\\\\\\\/WZnxYP++rDuC95E\\\\\\\/Brp42ZLbxDFj3b8Q6hTJElp7OC1nJheBFvqqvb\\\\\\\/xMVgIK94JH5+k6I9KVu0psCJbgI3yoB583CuJECp6FjCkmkOQCicakS1PaUjoJgxIgeRb9DlTSdqyZw\\\\\\\/OC6yH0QB9NVSjB1BQg+v+BovyU1FAOCb6ft88nCnx5CJfVliv\\\\\\\/6TUZP2H9d5INvggY49w8P9pDu5ecr9UvqbHmZKOhqSBj7wS6DSpC7mFYXRR5Gv+f40YEU3xi5mXRSnr9j7kbl9h2P7nHis2gYLCvzR0mioABXdlAbkC1qnvkcMwQDCRp3NnoUpET08kY+pvPPhOTZyK2uy\\\\\\\/prPymiqjZcoaOpPe8R2W9kOHJtO5IR8UWPETQ5PjFej2XQ2VRrmlXqZu7L\\\\\\\/iMdMNnpnZeAMige7Cz5rsIvtfhaVUWtqnkYdIQPPO9scY0qXlA1kXvQCHj\\\\\\\/x3agZedqvM2ib2VRLySwxGG2FyxosnWARmSaKFGUfyMXjy8mVKIxlbT+REr9BHIVQXMGzOo8tigFaw2GTfI0JFGyKVBYJdnoraCwn3BhTiv2jgMeCUKJlb9ExgardA1m5W7oP6HGa197P24XXE1VEfY1HDrXn4B56OiiBKQyr+SqR6mhyXR9\\\\\\\/zecsy7ZMYSq66PciKTMzgWnu1ab6+xucF9VHjI6VRuwmdiMhGLfa6AQg5YIn4xSW7CxHWerYOBSdowwCe08A\\\\\\\/vUIvcZt8ddndoz13L60J2YW7kF9zH5WWFTDyJA2RJn88Zvk4niOLEqgrsMdtkNl+adlPMAvVTXsY8JK39nBi4HmwbZv+QU\\\\\\\/rV1NJBgtmuj1iMyRDSIei08a3IOSkU5SvVeVm4QCB86SwOpXp47CdA4\\\\\\\/mr9zU9ADmxP69bhmnJqduUv456Zm0u1DKlIn569LT0utymeoWktSv2wiBjCMOaE7bsGTziStMLyx9RZXyw4pl0Q+JSPU81PFeO6ivg6LDP1z6Kdcgbt2dtiDW3IujkPpu4HXKnJf9FVUKWdupSnNtpKXJqGK0e5UQYIR7j46QOqE6XiqVElcUqsHPiEX9DvsmSGdG\\\\\\\/RXaKZ6V2z5tMWyzqYJAKcnLGltaJSDtvI+slLA402LNmn\\\\\\\/5IHRXilVx+biGeiE9I71RHTLM42m3kbhjSq8VftoKbFp1qKCcuo44eSvD7J9+87YUJ5mG9cWfEh+NoKPu4PbqHKOSv6XobE76hk9cPc33eONNz6NuJCNg0aN22np8EAZPydAUEjKS5ni0w1XmLu0Hu5lVs3M7fdaN+pO\\\\\\\/VSvaoiaxJRa2c+Ppyk1oL2ce0MBuoGdMvGQVTjO7T+XboSpXRgZKEd3rT3L99NjhurqoVllcgKcauwUpXUx+vL8LWtRQItteawVzXAGTPJK\\\\\\\/T3RK9o60qZmCMzHvfPF\\\\\\\/w+zzwy7O2qB11oKWKrt9N89JOGsVmdpZo+FwSpfm7uGRmfDlh\\\\\\\/E+Mc1dGKvyei3DdRaCmY7vr9fASZvRV49qKr\\\\\\\/xtNucU2MSPVvlW7bRIrmcLVsmHFIGdJBwiQglkeZOUjxVvAv+j8a19eNnzkv9Bd2SOi7MU7qx0IK4EvtIA70+VkCDUh1a3PpFvU\\\\\\\/rtLxOWxownVZxwGskjiuUfB\\\\\\\/OrngCJuh+eqCvCqnhZHlsrTCtv6Y\\\\\\\/QJnkQRGuK3EjHVA7jErqhX0pVpo4cdSK84ppF3J0GFDjfoOKtIAc2cpxym1crhUpd5RITuDt77f0\\\\\\\/1smsnFPlGbSOQLWI2dRPu73\\\\\\\/y9lVG6v726VdF0Wt4qmN0ywRfh50GD+0d2iaGm3FNVC9N5EwcMEUmDt\\\\\\\/sxVoNO6rZXdV8S1Z3YjVcvoNoaTRhU9lFDVia0RdxAmaMQSguCY8VjBfhaBiUAqJGsD3DUI3JzCd0oZ5D59LEMHbTNb+Gmy0qQtckbiEIMZmBwQ5X7Rggwj\\\\\\\/tZGFdxUa8LrI+ohv6q1FOd+xgAOMJlaFz7F+5XbuZoPLMQ2t8f7qB75z0L6uJQ64nDVVrw9iouVzOYRmzOmVNb1vHfSKXy9JL3tkBzyljbye04I\\\\\\\/1KU9WdML41Su2jqB82E929N5uE4yYrfm87a0egq12LU9U5R4\\\\\\\/Xa2lQSMhI1ULJe0afL+Rga7Hsdc+sApDyLM2u98Yn814mddNk892kGffTIWcauYwDggbwOvN9eILa\\\\\\\/JhRRoHQFjyRL4dg56zQT10ivLBX0s9bt0sk0MudVnill6ghspR24O39X6OLUKq2Jvp\\\\\\\/zyQjYA5Bcjx6eJWk1jDHTQzzEjw8s8cQ4\\\\\\\/L6+7VPWLeHmtA+ddv383VbzEN5IAhZH+1C+1LBlUcr99YSa8P6c2Fd2\\\\\\\/VmgtaTW0HQT+Vd16tdNgm6yWPTkkwVHQZxehnAVBu\\\\\\\/0DsYbbTmFCyg0\\\\\\\/GuwekdmyGuhZRkMRYJceqfZ94fcJ2FECax8CnBregsfn9BMdkBXVfJy8L4Yr\\\\\\\/cxroklOVayrb8vJX9DSTbHniszn+AMoxh3H0jYT9sjYo+nOgYIgfAFYqh\\\\\\\/sj52OY9N8xxeWrx8TeZ4NRsW3uO5xDWDLKtV6T9yhtL2\\\\\\\/cnQpfmEwxKepFU+ShzH\\\\\\\/AvitWg\\\\\\\/wiLeuKkPNsVEBDIUpgG+gjc6o\\\\\\\/7lDgwYtn2pPXjyjAPb1fbpoWD8mjOqdssz6IHtSkTirVRoax8djsKmyjFy6lNqz7Woa7JOxdcyF6vqkqLNyq9skfTe87dUJLjcFOfOzVo5S83xlrpP8\\\\\\\/3XcdJfZNxSH6ec2MyKFKnAEJyb8MbS0nsKfYL42NCltN6vrvSorDYJqyTHQcAWWciLfuymkLffxZiYo4yXK6iUY\\\\\\\/BEo1+TgcOOXxpOqQR9I7\\\\\\\/x\\\\\\\/aKIeLI2VQfAgjChdaant1OL1gbNmCTcxXqDn4bcUrlBNTttQf7nn86GPYweSafwM2SLjNBgXU0ndd3RQVVSOJmk1VeBlThnEz1TeA4wk5NcI6YipNISY6NXNEyeAYHy47h1r8ypXW+q+D8NmfqA9r7YPS3TyqBXmR+l40Rb+ujOLJHPdHHh6W6DK2S5IbpED\\\\\\\/EvW84D1CMQv37+DATxdNvV6ermX4\\\\\\\/Pt37AP0mOZmu8DtoBvWjecVN64\\\\\\\/GIINKjbx\\\\\\\/LCll9S1VazK96RuTOL2u9dSQQP3OjrEtuRfKg3XA18zpXMAW2ks3\\\\\\\/ZLhgSf16wVRaqJozlvMzoQPjrQQvEfWIKMjO\\\\\\\/FRtw\\\\\\\/nCzYgiZFfjMpKpMsXI4eiqd6vRrBhiDPwoKlYaVHBdz09XZg3AmF5gy3jXCfUDUhdknvm47XyYMVNPnpkiKIFKcXGCm76NeAU\\\\\\\/Sg5Ukje0eGvotmaR+9w8T0AiOEk2MkWxesEcitNDqW1yiEHdJ7GMsjXhX6tLalOrs5TqKRy4kOtfkAh8OoPjgNX0C\\\\\\\/DSUx44s5+sZ9XM3Rv43WcDGB2mmuisWdlM2ec6W\\\\\\\/I3G4rj4oBCWQlvtMfH2MRpKytsy78gIdb5lHqS4r97vbYd5TIKOTziClMQGLXv0+vx+SX0TOY1+jaQ0O6fJj5sCkV\\\\\\\/PjUnym3STK8Afst68lb6fAw2d3aG5o502QodYvzczmMHI\\\\\\\/cYA9Mw9NmKFAHuBKxPgrHpkXR5KH99Lcse7xTtNH4EevOP2z9EAmiTMcZCPa0mkFsNjeZw8NFHkfgT3UdnbrPaS9Za1t3oZj1E66hW7yDHhChqoq91lLU0176VPxQ2DZxkBwepXJvcKfpBhCBmT+i1hajWk+kQ23mmAz3w5DID5UUgBj3AsGpJHrxTGnGjSxf3WPwZnnfxK5o0cdRkwL5mlk6JaChQLrx7V0RRA\\\\\\\/AGhyDGgaPVC5PQiJcsrUNM65UQNAPFQk4946D562JbDUtyobVHazrLMcnHr\\\\\\\/lkn7DSiOW9CzDSb3lOYm797\\\\\\\/5YBrx0XngUTLnAobiKzlFi+NVSJdnRvNo\\\\\\\/2aNb9un6zfU+b\\\\\\\/UFZXI\\\\\\\/o3DXnZ5krDAEpK+d3\\\\\\\/MOIFbU9MwLtFkd7zb2\\\\\\\/es61ccaPrH9dBqhtzj+5hMZkDHrZtLMtzeTtK+hrknvbsdlpdDO4gdQls4QqX2Bf46f55ec2oSedABfrE1klBjD4dB6I7YREIernXml0LNdBvT7poGe2igCtYD3lt1hWqrR31tWYUsEzzZsRq10TFavDb\\\\\\\/PHySWWJ5c7hl3LGaH5GZacT5KGxSJG\\\\\\\/qsl0KBWZ\\\\\\\/jVOkbSAw9A\\\\\\\/LamSlTrbvmXo6DZAZvJvZ85ituojo5HwvOW+O7d\\\\\\\/lijT7YvKAODjhylBRpARiYZs+AmuSASKe1be4rihsB8eLjUbk8FbJMzWb5XAWCy3Lrw4Bg+suu1zokkao0h8ZLyGsbxIzd9LBTckfp8lAVWJx0PKw6RkBeZ+TB7SNkh7+i6RBgC+OmWXk7TyGzfKI45BkPZ\\\\\\\/L97GBKI7VMUzznbUOi1zP0oTf+IWyfXCi3ZOWhudfiAl2oMDMJXETeo53d9gcZeVDhGu+p1oRMl4oP9HKEzOYofcijxVS+G4W0Q1h3mx\\\\\\\/y+AmcFmjY1hL91bvETQQ7zsnc\\\\\\\/\\\\\\\/aC2H3HsIysxfjHBFg2aGbdcTdBivr0QJ1D3uUwica1QongFlftkVfbk76VsMCg70jxqawoxQu+YXS16gqOpH\\\\\\\/R5UBbqO9sqZ4ZCZCC92yi9y1l6gMtt9psabX1DOpLAVscp38\\\\\\\/yx4K5j1DKfgMBp59z3rYquhNembfA1Dt8VuCTka0DPOztbkvmjFizT2ru2+G2Z0V5XAaVHcKSAn9YwOhvF2hse+Ayn5w9b4pTPmcHASN9aZaNzMSLde5O0jh0VuWXbulLt6\\\\\\\/LQJ5AWbQf8ye1t2KWl+A9vmXQ2biY3XjJOWc1CcYbAJuiOh6NCdWrqDQAG51sosz38Kds05rBlf1ONAwENgebIwi9a0hpnzlLaJkFFEcB4H179ztXZh5UI\\\\\\\/l1rCuk16OxXUFjHFeArEEnjCh7d7coYRFF5Nijce6q5B9Q1prjCtv0k9sBB19TphTczaG4\\\\\\\/0SVa9h6GL0BOttOokN83YSbg2NqJ+feL2ADWHvgb6SG4hMhQlZRPXXKbxIGLZGAjBWkI4fdqNkvBJnfR+OgwDBp7KBeRb3jfUyJmSKd\\\\\\\/Mv9Wlr5kdfAAzQAQDhhJrRoAB8k+Bgyp2Z9LdUYhlZsFfPMfcPGo4g\\\\\\\/UtcSjY+ZbFeXDRQvA94Oi+cRd2g+gaHJkPuvw98XP6vowwuM5boOgotnyzi79PT61hszt6a\\\\\\\/uc70jrsHTIeOOxiMZ3DlrzshNqYbMWzGGL1RoWrXQgBqYTrVMj7x8CR+tBkNlv1DWyq9mnk1alrKsFuMixeLMbo\\\\\\\/ghNK+wwujrRSgtg1\\\\\\\/0MLyhViWloyM8U9nV4LowDPG4MTohlm+ibbDjokyDGzbw3GBV7xorHkcvD7u+1U8vOPxnbERBxOiYcmGMBHtmZ0UVMM8ipFPxgN0m0Sk+7Hca0kJVKsiGxF7n5pbSSU6t+hfaf5AFpFL54IQM1ly4VS5Yu5UcMTr4ZN+vtpRugEV0PgZNwieYmO57XV4LDWJqdmIGFxwRmqKhWtjnmGJEERJBrunxb5MMEgq92CaghMZgCFVtbMtd6sWwK7e33T0GjRP2J4g0PSbDOBa56gvkdysRlqwac+3udJnkpgezg9VDUcJuflTG7ISK0mUKusP9NxdxF\\\\\\\/Pb+lLiQnfGYcKlCmXEOC9Fv41yod3OYdQxguPefq2bmjO5te9Qv+9lxbkgEHPwhiMHIqXGWgY9gDDoYm50kgoJaL2Ej0+Nq0vGNWiwBIy1Vx\\\\\\\/9J\\\\\\\/RZbsr2CmSzvSg7k9zXGrdDryhMDyWiJzErP\\\\\\\/+ZwoqROMNftKr033REq2r\\\\\\\/BDhNJdoryE1SkwD77AlTtedahMO8ZoRXsuqKkPcogt5nxP8zbMFSzdBwH26TBNtNNpzrS0+xNARl+40CY+QC5dGgj+OS364TkSJaj4rzgpJVzxquPGa0frtPN4cZaICSpqGBLDFIKJ+RcGNJNsclX2rCk\\\\\\\/2\\\\\\\/CfVV2d67zIBBYJ5DZ9SOgI9xUO14O3b+WxxPpelZzmd+c6n3fiQoPJvX8fJ+X0lAJCnQAi432yjrLyRTR6P8Dogdh+mRiH3SGINIivgCd7e1dAf0MnWdlnOcGb2x765f2fbzSsJaQuOpbuBhaPhcJ0KY1AOyoqVlem0iFPdipHym3oUQiUkWq9wyNd7b\\\\\\\/K3hLYbqJaBaEvCMFWQqUUkspJSlnjkEO6L1Qx1w2gc1iNxaVypRHzaQsBosItOyiQFEJdOvmZ6qdGZ\\\\\\\/DrWQguHtNEzvewQee1Ow6tDPsu54zpnk7ORjZa2M+PUzDLk+1DfT0y1AefqPhVUCKMk4evgvYUGl1Sdhqy8\\\\\\\/9IF8UBsmcQSWYKnilT\\\\\\\/4WH2\\\\\\\/UCiUaIpwcfHudtE8jSrZI4ONVaFqcTRUktae2QDJ5kupSFmtTqzIHi51Tq5uZFDtdF+irRcSUN3j\\\\\\\/1XCkhDT+moGD8m4uuVBIcHNXGC+AYNvEvE\\\\\\\/qbr4ysqOAkflsxQ1a2wc7SrVonQXIOwIP\\\\\\\/c45fVlBZhQA5xTU+t7Gcr56MXkioEXGF8GkBHx2ZYbkh5QM4RClLchhYIbmpI9GT8cQm+UMghMGoPdlAjnZvM6b0HOUm0ouuZxMZsHet0h+qEcmmc5nw0eE9jsIjiui6Z0GnEEPXUnMAt6SU4\\\\\\\/eJ\\\\\\\/GGeWoy+SYrLqKYUm9Os4djGx72TsrhO1Evn9a5mIcUR9A\\\\\\\/8ZQqAt2Vi7BX2\\\\\\\/r2Py\\\\\\\/kG\\\\\\\/tG6AtaNoLQ9W+PmproFtgRWcqsJxlGLg7yMRML55Cy10ZjdAnTeDnimPxMMBcikUVStKpEyuYYv58BKW3Zxj1xVyAcktrH4Hr6lmRqMyRxnizKnv5djaxkf75r+JwN1bi2PVCs096qvd8wljXpAPWHnxVQN099LBNxfWmZcBzmG3gfVv4ndVzbfbczyqz8ypp2BD0L6LAJXEoQMQV8FlXki+O74HY2zflVHpgVGiYt0JaAPE42xe1lDhrwvbP+DfIYKN\\\\\\\/ORTGia9Z8UiuvCU\\\\\\\/jWvGoduGB0QnDEqV8U83qE6O4myEa3AmNZPtWJEcaDXSUiqesV03OIqXY5Buo3G2+Xasnx29w0P7yiWEtRqsOV2RtDtpo5\\\\\\\/Jo74kuxCpb3eAn1jY2Oa3MVgKWVeGiAQguxJOYd+EAN1wHbjJtvmvZxV5WMGKU4+vaUq404m6QkJnaAWvk1sf\\\\\\\/2utBwn+d3MT0bZnOK8ZXb3G2Rt0g3LCdfNEyg7M\\\\\\\/GxIb\\\\\\\/QCfaIAql30chFhKOM4WoM8pjPV3XQtWM+Pmo00bM8OtclmPhnUA1owhoDovNIz3iDtY+MlYgOUag+bkv7wLGisLl2J41\\\\\\\/YjhoTliL2wgObfIo2+fJic4+1wqw9I3dtlJEoGQm1ETClFq4ussbHTNlobBQzcYLT5I3EoXM94BendR1LgusUbd1Gfc2RHIH1QNhvCr43t1+9BXnNgqQdOvc9gqzpprrX4cxsde1LjxmUtwBIb1veqR5o+5iYWw6jk3T\\\\\\\/V\\\\\\\/SlngpZIFgeUrsXlwoU11lR9XoxqyZjsMMC5WnefOp9f1w1lVChjybQ5y6DB\\\\\\\/ULIt6d4usBmJIg3Zrn+mNNkASbMzKGvE0edmghy6iOv3jKFSM\\\\\\\/XWknvNHUsvyA3hkIAWol4cRXYIAC5xeuO\\\\\\\/l9qEd8gOIeyXOzUV17KqPFBxnp3OuaTFiBK80DnW3CNG8cjjcwsR57oTsgwLSxphUjT0CviEMBiaMod6sTh58DyPJ6Rt3t25BhJBIheWFnLP41JO9j4ytMhnGvlofbNcnJdru62z0EDYqqXlH7gHqeKMgjZ+xPWDPhEV+eH97bDUI1gYhpIsQ+ChXVoeCTThBRNq952t3k5EdkV27BEilO5s\\\\\\\/F2uLQtK\\\\\\\/HgIFNJctVbyL6\\\\\\\/eNNqI8AO7FAz3l1IZx3CYKRtxGalUiVhVos1e224mYtqOYIrcGdogzc2M1u73yDn7Jj1MQ5Fb8c7GkmAlWv\\\\\\\/x8pEqZFCucLilJB5EgsuDRjHhmJAYEdM29klcMjTp\\\\\\\/qLvdS1fgDcJrFyZIh0e5Lk7iq8M4KiJm34M+taCHRqBSL9ge4YvSVcvDbAOGj2tbpTh\\\\\\\/YJbkuSMKEPgZWu7AuL+pxiC4hsBLQuVzci7fzw5uMQl\\\\\\\/xujxRIUFm1v4BMG3Ydu+GSm4WoY2q9+9uJvvr+91VkrEBdkj5QDX2pqliQOIxEaBZvwPSN7GzAtpHDziL+dS57wuA5kgIERssaJeH85Sf\\\\\\\/8FLhkGZ\\\\\\\/WPSwNLha9TgtQhLAjWwNS57eYjevLM3MVjfLvJqctNf9Z9CWx1KYAPvQ1xAfXIEeLp4a0uh6NVc4S4ATmD5ZjDMYAktDYR\\\\\\\/lyf+nX\\\\\\\/DE77fK5AxBiLEO5nSx85ysAMRsDuLruqkWWkNdH2HzrcOiIP+VdLBrf5YK9NauROLW1U1XAHUI2uNSlc9Iek5imk47WOYjzI\\\\\\\/cqNgvHlJ25lO4WA1ydS+0NLhUrcBN\\\\\\\/J+Umz6BeN8LciB9it5kSlRfuhR6BlLw1\\\\\\\/SzcYbB2uputA+jln8UC12vEYOnVrDwIrv6T4wT31KQE+nhCIykaWq02Hsm9fI5UaPa2LbHBkeZ837Ye2tDe9QyDs0\\\\\\\/4n2ImcW5TprhbejuOITYSD8eKk5jfSh2DTTmSGL61wFWCvOHVKRuTt3jzlKOQ5HOyH04tzOlNbwgX6jaEH+GhzpOVrFRoNEjEsFVJPj3A4RPO1cIR1dgiplps+miJbsY58w8+GsXfsf\\\\\\\/U44dx3lvrdOPFhhuYTiHU8ef\\\\\\\/IV6CUZ871qz5Apl8\\\\\\\/sQNDIPgG7yIz9cOji7XnpgZQ1Y04kcoqSJu5rvOMBLoBfgFv82XjJBoOifreOWIlKwmChPNv5AiU3yASc5NATDqomTD1GcI\\\\\\\/l7sUdiVmg3Jj4YCL\\\\\\\/yVNpBIdPbGYLZrn2zneWIRPvRKkIPvJowUiNg2NTn9PFL9f\\\\\\\/YGC0hdaNUBq6NSzgqQ8Y6n2hDlcY771LwX4qDfyF+AMgs3JZRqNp3K8+eh2GkvbRjGf4TiHKaDA5l6a9TQC8ghKGEXCxTdjq+a0yB5WtK1VJ49HWBvELayolxL9uCve0sB1KlyHhi+\\\\\\\/FGaXHDbfRTEcQDxvTJlFNxl5iTG\\\\\\\/DgNg5pC8fK5VGTHZG1yoJbg9djxtZ8gkGp8mvb9sJwukK7L1nUp+EGQILJXcsZKeoqih9hsDqmfQ3wU6FX3ldP3QGg78U05vMPoeBIkSTgHVZ5Jl0Ux9LVs23jLhahocem2GMjlkFeaX5LHA6wv8nte14tao8X\\\\\\\/m\\\\\\\/9+Zw0fex5aKVV+qanNKKuKRTBYVxVvNPVOLmhCirNXC32agGDpJih3sOgyZWiQs1FWTgoNXPnCyFp5XsSe6wF8\\\\\\\/DV0hHCIPsYsuibCDTuDDeFlNEAiIILJYD6bo9CPQZ7dC8bLKF2UOacvIDJg9ffeCBsfD\\\\\\\/xm7QOdGrkCenXMZTbWI86J6DAvsok2iFXTsWWLiULAQp020+7GwK9pFGl3FqRlJOKgWTuOyefIhegtMeCWOBGwXGEL\\\\\\\/Pl8AJ5ORfDnBuGMr3zjqrvTEif9kOeS4UW8CNkDz0r+lZrpinmDvAhjn7Lrz+fy00GugPyEbj0MF9bTeERJVGvwJSrtDFq4ZSJ5bBH1cSo8OxeQFPi43vU\\\\\\\/8b\\\\\\\/N8Tbb4GjOCak+L2XNpwe7zamUbujqc\\\\\\\/7Dm0XJAtD9v01r5vU\\\\\\\/mkeEe0EIWdTcsO6el5dQr43KkrA7o9G9G1qslEsn6BDnbNxVe4JjBog05VREadk760xOnY8AhvuO+DLDLQcqFvBFCzpCfGaNQwT4AFsRc8DX3kYbr2JqlA7Nf6Ga5uLvvhccQvXSGYl9cnBNZfPa6ZxMhsolzCUAsiX09TURLXmKXfFjSQRE\\\\\\\/fGzWApf64HXP1LP7JUr1uLiK8\\\\\\\/zaLZoMHZOzyHDsoJ8Mg29fcnbKJpUCNXPKkvpmgr0UEzaH7ClPFIvGz1xeIbORjcfPCeRFNiWSahW3a3ZqLwZovx2BoyF6Piok8lIh4HtQGg65CZBR5AIv8HpXiF2VBxDMnQPOrbfVOHC1BvPwIeinQ8sHE131f6Qtv5XjxJHUS2xPc0oNZ7dxfWoGg0o3Bkh5UsxNhOgnmWisSfLgAokXVjGeFDkgsWqEnnSZxSGVA2f45++Agth7B8u\\\\\\\/+SHytTZ8hIxbzzQxNpPVA6YGdSqXBAEYOLhc9Tt4L22aATM2M+WquMzQqvTWZ1kCz4VSc5I4IAbpRnXFWSPqqjYs66TU08tsiTwPbih8JpDVYf6\\\\\\\/o8TnVUhbygE089Hs2CjrKgp2wg0whH3kwhuiHTg5AFO87lw\\\\\\\/mtV52l3agwaw5AOt6qil5WSk5HAqs2vl5JUDgxdtDrfvFJRLFxmFkr00d\\\\\\\/qoRwoZDBhe2+f8RQ5GuA3S8JgJZ4mSP1fSzLOAIYqJc0VDOaUrLByD4d3MdDvNKnAoISBkvJ1HKAkE2GSq5CUy+T7y9SjpmjujypZSgWKWLwZOkPtwY9zPNv0XuNkYlXRdRLXC1v2Ia+\\\\\\\/TNgOELW0ci2gCIXifpGA5VgpLK5Hwb4\\\\\\\/NCc4zUbralsa1ao\\\\\\\/dbsKS0HlInK0S2+Panm6TG09e5caIT6r5SHA3TWFs9HlZg1CnnbeX5+MG7y4nG0JeKuR+wW33irHZYgHHNz8v56uY+GF+KjPF++qXrl\\\\\\\/5Vn1QH7LjmTSyi9e5XU38QVHJy+\\\\\\\/iXmbOYiYH20LEqTfsHQ0oiNn6pIk7puN3WO7mXmTp4G7JyRu0vcDLNXkhGqJh+\\\\\\\/plh+X3mj6leOF+FyUUGR3sfr0wXZkalPD4rPitjHNNVowczq0HB+tqbFppAhTI6sjcdK\\\\\\\/oJb1WEkHYnrSQFoj9JPKjyyIPSSjb7s2NR\\\\\\\/QaZChPOweZk7CvRXsY0DbcaL3nD3Zz2UXFlXer0Ye10KJG4coZCgqbkNS666bWq3tVHiHJN9xg2tcX\\\\\\\/VHLakOVkOnKsjCaGNYioVx2ocKI+RX1uqUAJRiH+vZ9tniHqB\\\\\\\/s\\\\\\\/ASw96DRY9H06icAS3FNvBVwYt81X2WQshSjwY1Xz1fYlgfpisUbQTNiFUVWLCxYoxK7FMNtHjQwMbxyHZgbz2XiDe+l2p00ipzS+AXPO5L8LpKCRWOEXdjRk7C8Zb\\\\\\\/qQOru6bg8GrMx5XEVXKFHlg2VQnt8ThdDF4S\\\\\\\/iWXAZ5YQyqIeAzda4EIyD4ewwI3WPWsRYSS6ZSTZwJGTg2RmAh3sj9oefasrhKFnOUidTK5fkzGYXGkWrGUClAtSH9pNzWZUBcGGP5hU4qXyxhNUpxQiD8Bn9ih+aM8EoDSgzq7EgQSnbwGOGTT8nX7swmCKLGxeE5UCIUdyvl5hQwarSUvw2iaL19hby0VXSCZMRhplmLYckeovV+aIMkTSfx58EGOiW7kFNR0zbvvV4YhSJzW9dBGppmZoqxs6bNEFBrQ4wETQFfO8eFlMv9aBZMK39wbUf8su0QTci8jtY7cEAwTbzwyEOC9Y1NRutTBDuWJvd1ZvP4i8p80JJBoByF5Bp3PSM4KwrQqglA4ElrsoCGpZRKrVBz8vEqzv3bG1nbpJzEUYLcikIEJCvatxqBDJ3NUwfDbcS52Gz\\\\\\\/1Bjayqi7cVvX11mOQM9VQvv06uLSfzJL1eiinIVycKTG3BCxGZ5mN0N5sQARC8RezyyxQ7WcjuzBT1j5bdiYZG5i\\\\\\\/V6+GhqeMw0RJOkjqPj4wF7DfU0fMAdkcr9IwJgpvtv1EpgK+gYvJVIKjPxxLfIVnHBhbQiSy2AfTdEaONO5wNK7ZE5wekc9NHkm1KEDf6o6Sksoz+kvOuv9LBsqeJEyOCSVoP\\\\\\\/UHc09\\\\\\\/gmSrVzZ2HD60KjVuvQ5d83+UJMosSqJaCiIE\\\\\\\/TzJROAMrsvthoQIorqSIqFio497keZvQ4OcLUT91yD4hpYoZFFURr4qpTgDL3WLd6CDVr6xkwRt2VTAxL8\\\\\\\/G8q23ZILYro2ac4BbrRViMAKtDhdOybEW1l8iggixXLrHKFR\\\\\\\/Z\\\\\\\/LsAMOKr\\\\\\\/e+4kz3pLQBEYKTMF0h8CANpxanOWlNYgiDNDHSByyFA6EulsxJfsP0sp9gWigW49MPOJZVe5qI3cH9kNcOZVW\\\\\\\/gg6+KhMtblIHVoHfzHqcGBw7deowtyDgrRwBWQbvrcCBTcAYbApTXiXuZRnm3NNug5gvWHJkbpSqBRA4pcZo5x8K7kNFgsMgql9At5J\\\\\\\/q94E4NKHCJnp85KuD6z+9qu7Kx9oAndJioYMZtDIsz8Jtyzipx+6gWhmDRkun8uMQyU7wszJG3mDoqOnYewFurp5kcGFd2Bj\\\\\\\/aWcs\\\\\\\/TsKk5ZjLrE7GK\\\\\\\/IE3Q5+MIZpeUnXj5gSCsItUDH\\\\\\\/TaEOleuni1tStz5X94Iy6xofTN82iSA91uLWAk3d6GnLNFRG+3FKUURm8xTnvvKceQAYGSN73WEfZslH50IR3CovjgDsCsfTyG5G8HzlWZVv4mlqsmov61fYG3EGmZzAviuWLLOxH5T6fTcCWwtjebeVfzaKVx23L26P5HpKTXr7AmQ1tGyHt2TqAsjwnm7eS6b83c6GQ2bhlouEayDEEbksRROA4enznmGPrr7cGtZPt7QJDc2n1chMVPzvWeQ6bAOYKvkIvKy12Xzn2Xo2vYOi8\\\\\\\/TMBg8AnohqJablokwaIbQwFzTh2DXUKt7j4WK9DzDKm53ITz0o8WOyzdbX8QXUTVVa3Szp73wCJayR4x6hgcFy9El+vtxZa8hKbAS+lICi31qIr+mSWimxV2oFg85edJEyZFbi4\\\\\\\/P2u+6SLzXBcVpVjbNhJjn7JYhegWYaB\\\\\\\/SKXO2yX+tPP3mr8lQNhnYtEnTYVYUR3tzVxiFrWRyfSqCb+Hl4UDhSykYoKPPfMHVbSGXgXjEfCwsgjvd9gwAtPr3BsauYxBUMl4JsRvzR2NTLgNq0ldViub8tKIkLOdhRQsyqbZfdaD5wCo2A4WMI\\\\\\\/EO3zCLx5OV7+xaxOvxGaSk9ZWEQDpFUwtmkKMhuGC8s7nOWWtx6neMLoVJM9HIUaDT766VVZ6VGjzweQupVwj0OtZpPYdpeySukvYC7r9u2\\\u0022,\\\u0022iv\\\u0022:\\\u0022J3HuDfA0dQqRMX21GsCfZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022da4db31762166e2534f73bffa91d2465dffe311abbd94bd03a3fea883ae42497\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aYrFl8fHMoxRYqXPokt45\\\\\\\/iIjJcC2HQfI4mnyyWI4kY2I\\\\\\\/wriCaPepS\\\\\\\/H1v0lcqOnbFOrO5UY6n9GpRK47E7sW824pqvkTmpqIHs8unDU3wPZeuKyDKoPBTEl8QGBWxhl+sjTiQFccgw9Xyl0qfUnGds0cLqBi+l+sSqSBHhQMaHQBAbzsLBPo3wSo3aCjdzCGW2JSIbAg+xizPtm\\\\\\\/tB3cNcrebFaylZIxlhBT\\\\\\\/lPnSRafqbftKcRAAgATsLLlkgm6rAuBqZWWkEC68FgM6iMpEYbAGdbEXJQBHgdI+JcIJ7jPwfC0L1VE1rg\\\\\\\/uc9ZycdH3rW9XP3nXdAUH8CON5PXYMN4rC\\\\\\\/pRNF115quo25h7jbM0j0EzoB+pylGrbUrdel57ae1PwqTRObhmbmlORka898UZXAW0fv22H86ghG\\\\\\\/TTPN7vxCVTU8l0DwDSyxEb5KGKcXUGTRrXkib+\\\\\\\/t4MIcKIgzyDO744EmYCL3ioAGWOzxmKExjkwDxD6wM15DonG+EPB\\\\\\\/378HGKr6\\\\\\\/wJsB7g\\\\\\\/IDal30qZNHOXDZeLyOHI+y6gUNi121+wHrg4x0dWwOD2YUU3zOzYLqL\\\\\\\/sdXGqBQefp3og94ksqxDHFdm5BzAqrdXZPfvhizZXoncBNFgx+wt0Weprax5f31zfwoHMK5ll0emFjVUPWxUBxUkeH9pCrO9ED7T564aVwXN3OkGJCimgFfltjbg2mo0Fh6yPHJXkey7I79aOQugA0gHWyp1qA2LjsiqsfGYiI6GQ4opsg7KmiTuvtia5CV0\\\\\\\/WUdtedfjtKVLqs99SA6h9VuPmUkt7gudHQ2aebxmf2QG6hy3wH1WDE8mDyw7JvpSvPox\\\\\\\/9+Q0WYhgaS1VR3ynado9TCl3Wg5XH7uSe0EjlMBdyeX7RNonfnJsItU8wRrLTcM+cOD8Glx3IEp1NixH4HwIH4Ie9D7PDuuFu5ikZdsN7tHqJ5gL1\\\\\\\/X57dZb\\\\\\\/E9DfN+K0qZ26uJdiCPC4C1EHIRtF1+DegkwlXC3VeTm6Z4mq3KEmUheRCL4gJ3dAr0YkiFcPiyF4Xgn3n3fz8eKSiPjbUsifXIfdDVv5gZsR3jpuXrVCI+2Jx3yvutS7kHQtgeJzniNG0QtEUZw9X4rUycuGgw2iVbioP5gkc7Y7MTIQR6L0Jd8VZZgn4xq30WUSmyonlece9eB4tlk3dbn8K6kVUTgEyW4Q4qPXcxpM4dxH\\\\\\\/C4pULylE3UkXZtT7QSSvI8SXn9Y+SOoQ8IyfoQcSoqkVAVz6Qbzypt4+3zDOCegchjhgyG8BomEdGun3lBdw\\\\\\\/IoBZYplT5\\\\\\\/eP1DyrumEKU5KSbhEMGjsbuKzxi9P\\\\\\\/qOvHPFmZbvUPl5XXe7SUw9TQaLmSscbBOMxyp8qyY\\\\\\\/yKfknWzPEYXT0+52MBFoOnOEt8Ecspa6bqgLhmSiAfLoYg3Pwvnfyf7raJq5Szfm+Qw8wztqwZsfru7X7aV07hPItWhba3D4\\\\\\\/QeVBMq4fklrhoGXHHkwY8uXMVsCmebJMmQTEZDoY76PhbtCpyHwgVbCAuFqjp0W6pZCGyTwc10017hqRbtS0ACFcOqjSVvs6maqsDoJxcrasiGFa4LzFyJypiPs+Ndq4Mxmf7SrVWtmn28pSYjI7HbRH1BP0rg8761urPV27gY4HZNzF4FhEOry9J4p1PjI35iA4JDLxaShr7BE9wEsPoBUAloy\\\\\\\/lGAQQbvr9TGj90beKQ5YAgDNsIiU4f2MQj+pOeuG8y3NDKJKZA9KTnhHEi9IYoeODRuL\\\\\\\/pql+GG0PTYDgQjKs2DJzOrRDuoRNXWbpz1iYTRe7LQRvNgFh4UUNsrc\\\\\\\/OsEIuPnIvk2hbNvUMVCVsp6\\\\\\\/EXE2a62HbTJeaJcFEBnOmRitS+Pg73sCU\\\\\\\/UXr48R0vGsqV9pmrxjcfyJsBPk9rDUE1mNFI5ORgGV\\\\\\\/VsywOYcj8viTXzq\\\\\\\/CDdDFt3y3SbXOKJZ1q0ff3dLMWTgL7CVm09Mtg3was6SyPNANR18a6DzMFhwVkM0rTMhtywRmJwCOz30xXCwt3zAFfYBVwE+gjS8n4DZBpB0\\\u0022,\\\u0022iv\\\u0022:\\\u0022KpQ2eHq9WDIUGk4iHeJWgA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226cae2b451cb7a1103fd55de58532467d249822dbe219f5abec80d11e1047681c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459576 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1965" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Oib146oSkSND-iEon2lJ4Pn_t2HqVlrbAdgvh8hnnP0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1965" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-c9v_5k73AGiBkpluskkIRCF9i0JRTrF51boi1z7k0Aw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1955" title="A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1979" title="Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459576" id="node459576" data-pisa="ggg;3/11/1965" data-pisa-master="ggg;g3.113.008144" data-apath="/ggg/3/11/1965.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew Norris</span>, <span class="highwire-citation-author" data-delta="1">Simon Lovell</span> and <span class="highwire-citation-author" data-delta="2">Daniela Delneri</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1965-1977;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008144 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459576" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew Norris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BNorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BNorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Simon Lovell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BLovell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lovell%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BLovell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.delneri@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.delneri@manchester.ac.uk</a>
<a href="mailto:simon.lovell@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">simon.lovell@manchester.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniela Delneri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniela%2BDelneri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delneri%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniela%2BDelneri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.delneri@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.delneri@manchester.ac.uk</a>
<a href="mailto:simon.lovell@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">simon.lovell@manchester.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/1965" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459576" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459576/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1965.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459576" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459576/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1965.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459576" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459576/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1965.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459576" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459576/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/1965.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459576" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459576/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Variation in gene copy number can significantly affect organism fitness. When one allele is missing in a diploid, the phenotype can be compromised because of haploinsufficiency. In this work, we identified associations between <em>Saccharomyces cerevisiae</em> gene properties and genome-scale haploinsufficiency phenotypes from previous work. We compared the haploinsufficiency profiles against 23 gene properties and found that genes with higher level of connectivity (degree) in a protein–protein interaction network, higher genetic interaction degree, greater gene sequence conservation, and higher protein expression were significantly more likely to be haploinsufficient. Additionally, haploinsufficiency showed negative relationships with cell cycle regulation and promoter sequence conservation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/haploinsufficiency" class="hw-term hw-article-keyword hw-article-keyword-haploinsufficiency" rel="nofollow">haploinsufficiency</a></li><li class="kwd"><a href="/keyword/prediction" class="hw-term hw-article-keyword hw-article-keyword-prediction" rel="nofollow">prediction</a></li><li class="kwd"><a href="/keyword/machine-learning-0" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">machine-learning</a></li><li class="kwd"><a href="/keyword/correlation" class="hw-term hw-article-keyword hw-article-keyword-correlation" rel="nofollow">correlation</a></li><li class="kwd"><a href="/keyword/genome" class="hw-term hw-article-keyword hw-article-keyword-genome" rel="nofollow">genome</a></li></ul><p id="p-4">We exploited the association between <em>Saccharomyces cerevisiae</em> gene properties and genome-scale haploinsufficiency (HI) phenotypes using linear discriminant analysis to predict HI in existing data and to guide experimental identification of six novel haploinsufficient phenotypes, previously undetected in genome-scale screenings. Using a similar approach, we identified significant relationships between haploinsufficiency and two gene properties in <em>Schizosaccharomyces pombe</em>, relationships that hold despite the lack of conserved HI between <em>Saccharomyces cerevisiae</em> and <em>Sz</em><em>. pombe</em> ortholog pairs. These data suggest associations between haploinsufficiency and gene properties are conserved among hemiascomycetes yeasts. The relationships and predictive model presented here are a useful step toward understanding haploinsufficiency and its underlying mechanisms.</p><p id="p-5">A gene is considered haploinsufficient when a reduction of the gene copy number from 2 to 1 induces a growth defect. HI may be caused by a lower amount of protein being produced, such that it is insufficient to perform the biological function effectively. Moreover, if the protein is part of a larger network, then stoichiometry among different members of a protein complex can be altered. Alternatively, if a protein is an enzyme with a high flux control coefficient, the entire metabolic pathway efficiency can be disrupted.</p><p id="p-6">The availability of a comprehensive <em>S. cerevisiae</em> knockout library has enabled studies of HI across the genome. A previous work determined the HI profile of almost all genes in yeast cultures grown in rich and minimal medium (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>). The study showed that approximately 3% of yeast genes display HI, comprising 98 essential genes and 86 nonessential genes. Interestingly, the fitness distributions of essential and nonessential HI strains are approximately the same, and only 98 out of 1102 essential genes are HI in rich medium. These observations suggest that the relationship between gene essentiality and HI is not simple.</p><p id="p-7">More recently, HI phenotypes have been studied in chemostat, where nutrients and pH were rigorously kept constant throughout the competition. The HI profile was analyzed in cultures grown on complex natural media, such as in grape juice extract, and in different chemical-defined media limited for carbon, nitrogen, and phosphate (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Delneri <em>et al.</em> 2008</a>). Although in grape juice a small proportion of genes were found to be HI, in the nutrient-limited medium a much larger number of HI genes were detected (10–20% of the genome).</p><p id="p-8">It is, however, likely that in rich medium, some HI phenotypes were missed because of the large-scale “top down” nature of such experiments. These data are obtained by simultaneously competing 6000 different mutant strains, and therefore are likely to contain “false-negative” observations because of interactions between strains in the culture. Because all the knockout strains share the same environment during the competition, different metabolites can be excreted by the different mutants, altering the fitness of the other strains independently from the environmental limitation imposed. Studies of the yeast exometabolome have shown how complex and diverse the pattern of metabolites in the supernatant of different yeast mutants can be (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen <em>et al.</em> 2003</a>). To avoid strain interactions, HI profiles should ideally be recorded by comparing growth curves of monocultures, because these contain only the mutant strain being studied. One-to-one competition experiments with a reference strain may also be better than pooled competition experiments in this respect, although results from such experiments may still be affected by interactions between the reference strain and the mutant. Another issue with genome-scale studies is that the significance of HI identification is determined by levels of agreement between biological replicates, and genuine HI assignments can be disregarded if one biological replicate is of low quality.</p><p id="p-9">In our study, we set out to explore cases in which genuine HI phenotypes may have been missed in earlier genome-scale experiments. We searched for these cases using a computational model that, for each gene, calculates an inferred probability of the gene being HI. The model infers HI probabilities by exploiting relationships between previously observed HI and systems-level gene properties. Our modeling approach based on gene properties is similar to a study published recently in which gene properties were used to predict genetic interaction (GI) degree in <em>S. cerevisiae</em> and <em>Sz. pombe</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Koch <em>et al.</em> 2012</a>). Many gene properties correlated with GI degree in both organisms, despite the lack of GI degree conservation between <em>S. cerevisiae</em> and <em>Sz. pombe</em> ortholog pairs.</p><p id="p-10">In this work, we chose a set of candidate genes that had high model-inferred HI probability but were not found to be HI in an earlier genome-scale screen. We have opted to determine HI phenotypes using monoculture experiments, because this method avoids phenotypic effects caused by strain interactions. The gene prioritization approach allowed us to find novel phenotypes without resorting to the laborious process of measuring monoculture growth curves of strains hemizygous for all known yeast genes.</p><p id="p-11">To build our model, we compared HI profiles in six nutrient environments against 23 systems-level gene properties. For many of these gene properties, values differed significantly between HI and non-HI genes. We used six gene properties that associated strongly with HI but had low correlation between them to build a machine-learning model for predicting HI. Many gene properties such as protein interaction degree (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jeong <em>et al.</em> 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">He and Zhang 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Zotenko <em>et al.</em> 2008</a>) have been shown to positively correlate with gene essentiality. Because essential genes are over-represented among HI genes (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>), we investigated the possibility that gene essentiality might be a confounding factor between HI and gene properties used in our model. Our data show that in most cases, gene essentiality does not fully explain the relationships between HI and gene properties. This finding suggests that there is an interesting difference between the mechanisms of HI and gene essentiality.</p><p id="p-12">Our model used the established linear discriminant analysis (LDA) method (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hastie <em>et al.</em> 2009</a>). Training of the model was performed using HI profiles in rich, minimal, carbon-limited, nitrogen-limited, phosphate-limited, and grape juice environments (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Delneri <em>et al.</em> 2008</a>).</p><p id="p-13">We found that the models constructed for rich and minimal media were capable of predicting HI in these environments effectively. Of the mutants that were not identified as HI in the large-scale rich medium study, we selected 23 genes with the highest probabilities of HI according to our model. We tested the corresponding mutants for HI in rich medium by measuring and comparing growth curves of monocultures. Interestingly, 26% of our candidate genes showed a clear HI phenotype, validating our prediction method for use in refining HI phenotype profiles in <em>S. cerevisiae</em>.</p><p id="p-14">To explore whether the relationships between gene properties and HI found in <em>S. cerevisiae</em> hold for other organisms, we examined ORF conservation and protein interaction degree in <em>Sz. pombe</em> and found weak but significant relationships with HI in rich medium. We argue that our HI prediction method could be applied to other species, because two <em>S. cerevisiae</em> associations are also found in <em>Sz. pombe</em>; these are seen even though HI profiles are not conserved between <em>S. cerevisiae</em> and <em>Sz. pombe</em> orthologs. The associations uncovered here between gene properties and HI are valuable in terms of identifying novel HI phenotypes and understanding underlying HI mechanisms.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Haploinsufficiency data</h3><p id="p-15">Haploinsufficiency data for yeast peptone dextrose (YPD) and minimal media (MM) were obtained from a previous study (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>). Genes identified as significantly HI in the earlier study were considered HI in this work. Haploinsufficiency data for carbon-limited, nitrogen-limited, phosphate-limited, and grape juice environments were also accessed (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Delneri <em>et al.</em> 2008</a>). As before, genes identified as significantly HI were considered HI in this work. ORF names from the two data sources were standardized to the most up-to-date <em>Saccharomyces</em> Genome Database (SGD) gene identifier (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cherry <em>et al.</em> 2011</a>). In some cases, because of gene name changes, gene identifiers from data sources mapped to either no SGD ORF or to multiple SGD ORF identifiers; such genes were excluded from the analysis. Genes identified as dubious in SGD were also disregarded.</p></div><div id="sec-3" class="subsection"><h3>Gene property data</h3><p id="p-16">For each gene property, standardization of gene names from the appropriate data source was performed as described. Genes that did not appear in either of the two genome-scale HI studies were excluded. Gene essentiality data for YPD medium were obtained from a previous study (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Giaever <em>et al.</em> 2002</a>).</p><p id="p-17">The Perl Graph::Undirected library was used to calculate genetic interaction degree and betweenness centrality (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Freeman 1977</a>) from the stringent, intermediate, and lenient cut-off datasets available in the Drygin database (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Koh <em>et al.</em> 2009</a>). Scores for mRNA expression variation through the yeast cell cycle were obtained from a previous study (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Spellman <em>et al.</em> 1998</a>). Summed intensities describing combined haploid and diploid abundance of protein were obtained from a previous proteomics study (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">de Godoy <em>et al.</em> 2008</a>).</p><p id="p-18">Interaction network data were downloaded from the BioGRID database (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Stark 2006</a>) on 26 January 2012. Two filtered datasets were produced alongside an unfiltered dataset; these excluded reactions reported less than either twice or three times in the literature. Both the filtered and the unfiltered datasets only included interactions reported as physical, thereby excluding genetic interactions. Degree and betweenness (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Freeman 1977</a>) for physical interactions were then calculated using the Perl Graph::Undirected library.</p><p id="p-19"><em>S. paradoxus</em> strain CBS432 sequence data were retrieved from the <em>Saccharomyces</em> Genome Resequencing project website (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Liti <em>et al.</em> 2009</a>). Sequences of the <em>S. kudriavzevii</em> strain IFO 1802 and the <em>S. bayanus</em> strain MCYC623 were obtained from two studies (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Scannell <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kellis <em>et al.</em> 2003</a>). The <em>S. cerevisiae</em> sequence was the SGD reference strain, S288C (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Cherry <em>et al.</em> 2011</a>). Alignments were performed using the BioPerl library; for each gene, cDNAs from <em>S. cerevisiae</em> and a selected clade member were first translated into protein, followed by alignment using dynamic programming (with the BioPerl dpAlign function, default parameters), and then projection of protein sequence alignments back to DNA. DNA and protein percentage identity, <em>dN</em>, <em>dS</em>, and <em>dN/dS</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hurst 2002</a>) were then calculated.</p><p id="p-20">Because of the high rate of evolution for promoter regions, sequence comparisons were performed between <em>S. cerevisiae</em> and <em>S. paradoxus</em>, the most closely related <em>sensu stricto</em> (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Scannell <em>et al.</em> 2011</a>) species according to current knowledge. Promoter sequence was defined as the region spanning 500 bp upstream of the ORF start codon, but trimmed such that it did not overlap adjacent ORFs. Although consideration of transcription start sites (TSS) would be advantageous, these were not considered as gene boundaries because of unavailable TSS annotations for <em>S. paradoxus</em>. Hence, promoter regions used in this work were inclusive approximations based on the current knowledge of ORF boundaries. Sequences were aligned using dynamic programming with the BioPerl dpAlign method (default parameters) and percentage identity was calculated.</p><p id="p-21">After standardization of gene names as described, gene property data from the sources described were combined into a single table with genes as rows and properties as columns, allowing data to be easily manipulated. The resultant table, used in statistics tests and LDA model generation, is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/008144SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FileS1.xlsx">File S1</a>.</p></div><div id="sec-4" class="subsection"><h3>Statistical tests</h3><p id="p-22">To determine whether gene property value distributions were significantly different between HI and non-HI genes, the Mann-Whitney <em>U</em>-test (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mann and Whitney 1947</a>) was performed using the wilcox.test() function from the R “stats” package to produce estimated <em>p</em>-values. To provide information about the difference between the means of HI and non-HI gene properties, <em>z</em>-scores were also calculated. To assess the predictive ability of single gene properties, receiver-operating characteristic (ROC) plot areas under curves (AUCs) were calculated using the R “ROCR” package and 20 repeats of five-fold cross-validation were performed as described. Pairwise relationships between gene properties were assessed by determining Pearson product-moment coefficient (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pearson 1896</a>) using the R statistics cor.test() function.</p><p id="p-23">In many cases described in this text, the variance-stabilizing transformation log10(<em>x</em> + 0.5) was applied to gene properties with values that followed a log-normal distribution. Properties transformed in this manner include protein–protein interaction (PPI) degree and betweenness, genetic interaction (GI) degree, mRNA expression variation, and protein expression levels.</p></div><div id="sec-5" class="subsection"><h3>HI prediction in <em>S. cerevisiae</em></h3><p id="p-24">LDA was performed using the lda() function from the R package “MASS.” This method assumes data are normally distributed. Many gene properties had values following a log-normal distribution, so variance-stabilizing transformations were applied as described. Overfitting was avoided by performing five-fold cross validation, with 20 repeats to overcome variability. Five-fold cross-validation was chosen to ensure a reasonably large sample size in the left-out set.</p><p id="p-25">Several methods were attempted to deal with incomplete cases in which the properties of genes had missing values. The simplest method removed genes with missing data. Another method involved drawing the posterior probability of a gene’s haploinsufficiency from the most complete simpler model when missing values were encountered. In other words, if a gene had four properties, A, B, C, and D, and B was missing, then the posterior probability of HI was taken from the model that considered the properties A, C, and D.</p><p id="p-26">Imputation methods to estimate missing values were also examined. Expectation maximization (EM) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Do and Batzoglou 2008</a>) and multiple imputation (MI) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kenward and Carpenter 2007</a>) were performed using functions from the R “mix” package. MI and EM were performed separately for each cross-validation fold. EM and MI methods were used with selected gene properties along with observed haploinsufficiency as a Boolean categorical variable. MI was also used as an alternative method. For each gene property, missing values were set to the median of that property within either the HI or the non-HI category based on the gene’s observed haploinsufficiency. When cross-validation and missing value imputation were performed together, the left-out fold was hidden from the imputation process.</p><p id="p-27">After producing LDA models, the R predict() function was used to generate posterior probabilities, describing the likelihood of particular genes being either HI or non-HI. To assess level of agreement between predicted HI likelihoods and HI observations, ROC curves were produced using the R “ROCR” package. False-positive rate (FPR) and true-positive rate (TPR) were calculated by sweeping a posterior probability cut-off. In cases in which cross-validation was performed, 5 × 20 ROC curves were produced.</p></div><div id="sec-6" class="subsection"><h3>Analysis of HI in <em>Sz. pombe</em></h3><p id="p-28">PPI data were downloaded from the BioGRID website (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Stark 2006</a>), and predicted GI degrees were obtained from a previous study (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Koch <em>et al.</em> 2012</a>). cDNA sequence data for <em>Sz. pombe</em>, <em>Sz. octosporus</em>, and <em>Sz. cryophilus</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rhind <em>et al.</em> 2011</a>) were downloaded from the Broad Institute website (<a href="http://www.broadinstitute.org/">http://www.broadinstitute.org/</a>). Orthologous cDNAs were then aligned using the same algorithm as described for <em>S. cerevisiae</em>. DNA and protein percentage identity, <em>dN</em>, <em>dS</em>, and <em>dN</em>/<em>dS</em>, for <em>Sz. pombe</em> against <em>Sz. octosporus</em> and <em>Sz. pombe</em> against <em>Sz. cryophilus</em> were then calculated and considered as gene properties. These gene properties were compared against HI data from a previous work (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2010</a>). Genes with fitness relative to wild-type (WT) less than 0.95 and <em>p</em>-value &lt; 0.05 were considered significantly HI. Relationships between sequence conservation statistics and HI were examined through the Mann-Whitney <em>U</em>-test, calculation of <em>z</em>-scores, and drawing of ROC curves as described for <em>S. cerevisiae</em>. We used orthology data from PomBase (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Wood <em>et al.</em> 2012</a>) to determine the level of rich medium HI phenotype overlap between <em>S. cerevisiae</em> and <em>Sz. pombe</em> orthologs.</p></div><div id="sec-7" class="subsection"><h3>Strains and growth media</h3><p id="p-29">The <em>S. cerevisiae</em> BY4743 WT strain and hemizygous mutants for genes of interest were obtained from a genome-wide deletion collection (EUROSCARF <a href="http://web.uni-frankfurt.de/fb15/mikro/euroscarf/index.html">http://web.uni-frankfurt.de/fb15/mikro/euroscarf/index.html</a>).</p><p id="p-30">YPD medium was prepared using 2% (w/v) peptone, 1% (w/v) yeast extract, and 2% (w/v) glucose. YPDA was made as YPD, but with the addition of 1% (w/v) agar. Synthetic-defined F1 medium was prepared as described previously (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Delneri 2011</a>) with the following per liter: 62 mg inositol; 14 mg thiamine HCl; 4 mg pyridoxine; 4 mg calcium panthothenate; 0.3 mg biotin; 70 µg ZnSO<sub>4</sub>(H<sub>2</sub>O)<sub>7</sub>; 10 µg CuSO<sub>4</sub>(H<sub>2</sub>O)<sub>5</sub>; 10 µg H<sub>3</sub>BO<sub>3</sub>; 10 µg KI; 50 µg FeCl<sub>3</sub>(H<sub>2</sub>O)<sub>6</sub>; 3.13 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>; 2 g KH<sub>2</sub>PO<sub>4</sub>; 0.55 g MgSO<sub>4</sub>(H<sub>2</sub>O)<sub>7</sub>; 0.1 g NaCl; 90 mg CaCl<sub>2</sub>(H<sub>2</sub>O)<sub>2</sub>; and 2% w/v glucose. To produce F1 medium with carbon limitation, the F1 medium mixture was modified by changing the glucose concentration to 0.25% (w/v). F1 medium with nitrogen limitation was produced as the synthetic-defined F1 medium, but with (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> concentration changed to 0.46 g/liter. All F1 medium preparations were supplemented with the following per liter: 20 mg histidine; 100 mg leucine; 30 mg lysine; 20 mg methionine; and 20 mg uracil.</p></div><div id="sec-8" class="subsection"><h3>Experimental determination of HI in <em>S. cerevisiae</em></h3><p id="p-31">To assay for HI, strains were plated onto YPDA medium. After 48 hr, strains were inoculated into overnight cultures, comprising either 5 ml YPD for obtaining growth curves in YPD medium or 5 ml synthetic-defined F1 medium for testing in F1 media with carbon or nitrogen limitation. These cultures were left in a shaker-incubator at 30° for 20 hr. The next day, cultures were diluted to a final OD<sub>595</sub> of 0.1 in either YPD, nitrogen-limited media, or carbon-limited media as appropriate. The diluted cultures were then distributed in a 96-well plate (240 µL per well) with six replicates per strain. The outermost wells were filled only with sterile medium to test for cross-contamination of culture between wells. The positions of all the biological replicates of the mutant strains were randomized on the plate to minimize bias. Growth curves were obtained by measuring absorbance at 595 nM in 5-min intervals for 24 hr with shaking at each interval using a BMG Labtech FLUOstar OPTIMA plate reader. The average absorbance of the wells containing sterile medium was subtracted from absorbance of each well containing culture. The lag phase, growth rate, and biomass of the growth curves were then compared between mutant and WT strains.</p><p id="p-32">To analyze the data and determine growth defects, we calculated the area under the growth curve (AUGC) across each strain and replicate. AUGCs were calculated for the first 15 hr of growth in YPD, the first 25 hr of growth in nitrogen-limited medium, and the first 23 hr of growth in carbon-limited medium. These time intervals were chosen to capture the region of the growth curve before the stationary phase. Using Welch <em>t</em>-test, we calculated <em>p</em>-values for the significance of the difference between the mutant and WT AUGC sets. The set of <em>p</em>-values for all mutants <em>vs.</em> WT was then corrected using the Benjamini–Hochberg procedure (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Benjamini and Hochberg 1995</a>) to control for false discoveries.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Strategy to assess relationships between gene properties and HI in <em>S. cerevisiae</em></h3><p id="p-33">We investigated associations between 23 gene properties (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and HI detected in six different nutrient environments. After comparing HI and non-HI gene property value distributions to identify gene properties significantly associated with HI, we chose six gene properties that were suitable for building a machine-learning model. Comparisons between these properties and HI across six nutrient environments are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463518/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463518/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of the 23 gene properties that were considered for LDA model building</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of gene property values with respect to HI phenotypes in six different media. To find relationships between HI and gene properties, the gene property data were divided into HI and non-HI groups. The significance of the difference between HI and non-HI property values was then estimated through the Mann-Whitney U-test (significant differences are indicated with white panels and p-values). The X axis indicates fitness loss values relative to wild-type (WT) in six nutrient environments, whereas the Y axis describes gene property values. We have visualized the raw HI fitness values using a scatter plot, with each dot representing an individual gene. The overlaid box plots represent non-HI (blue) and HI (red) gene property distributions. The box represents the upper and lower quartiles, and the central line represents the median. Whiskers represent the lowest point within the 1.5 interquartile range (IQR) of the lower quartile and represent the highest point within 1.5 IQR of the upper quartile." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728338711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of gene property values with respect to HI phenotypes in six different media. To find relationships between HI and gene properties, the gene property data were divided into HI and non-HI groups. The significance of the difference between HI and non-HI property values was then estimated through the Mann-Whitney &lt;em&gt;U&lt;/em&gt;-test (significant differences are indicated with white panels and &lt;em&gt;p&lt;/em&gt;-values). The X axis indicates fitness loss values relative to wild-type (WT) in six nutrient environments, whereas the Y axis describes gene property values. We have visualized the raw HI fitness values using a scatter plot, with each dot representing an individual gene. The overlaid box plots represent non-HI (blue) and HI (red) gene property distributions. The box represents the upper and lower quartiles, and the central line represents the median. Whiskers represent the lowest point within the 1.5 interquartile range (IQR) of the lower quartile and represent the highest point within 1.5 IQR of the upper quartile.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1965/F1.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/1965/F1.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463627" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-35" class="first-child">Distributions of gene property values with respect to HI phenotypes in six different media. To find relationships between HI and gene properties, the gene property data were divided into HI and non-HI groups. The significance of the difference between HI and non-HI property values was then estimated through the Mann-Whitney <em>U</em>-test (significant differences are indicated with white panels and <em>p</em>-values). The X axis indicates fitness loss values relative to wild-type (WT) in six nutrient environments, whereas the Y axis describes gene property values. We have visualized the raw HI fitness values using a scatter plot, with each dot representing an individual gene. The overlaid box plots represent non-HI (blue) and HI (red) gene property distributions. The box represents the upper and lower quartiles, and the central line represents the median. Whiskers represent the lowest point within the 1.5 interquartile range (IQR) of the lower quartile and represent the highest point within 1.5 IQR of the upper quartile.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">To select particular gene properties suitable for building a model, we performed a number of analyses. For each of the 23 candidate gene properties, the difference between the HI and non-HI gene property value means was examined by classifying each mutant as either HI or non-HI in each environment. For each analysis subsequently performed, genes were removed when they had missing gene property values. The <em>z</em>-scores were calculated, which describe the number of SDs the HI and non-HI gene property means are below or above the population mean. A positive <em>z</em>-score for the HI group indicates that HI genes tend to have greater gene property values, with a negative <em>z</em>-score indicating tendency of HI genes to have lower gene property values. Estimated <em>p</em>-values for the significance of the difference between HI and non-HI gene property distributions were obtained through the Mann-Whitney <em>U</em>-test. The <em>p</em>-values and <em>z</em>-scores for rich medium are summarized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. The <em>p</em>-values and <em>z</em>-scores for all six nutrient environments examined are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FigureS1.pdf">Figure S1</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationships between HI and non-HI gene properties in rich medium. (A) The p-values testing the difference between HI and non-HI gene property value distributions. These are on a log10 scale and are as estimated by the Mann-Whitney U-test. The vertical line shows a p-value of 0.05. (B) Mean z-scores of HI (red) and non-HI (blue) gene properties. Error bars represent the SEM. (C) The receiver-operating characteristic (ROC) area under curve (AUC) distributions. These were generated using cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The vertical line in the center of the chart represents the random expectation for the ROC plot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728338711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationships between HI and non-HI gene properties in rich medium. (A) The &lt;em&gt;p&lt;/em&gt;-values testing the difference between HI and non-HI gene property value distributions. These are on a log10 scale and are as estimated by the Mann-Whitney &lt;em&gt;U&lt;/em&gt;-test. The vertical line shows a &lt;em&gt;p&lt;/em&gt;-value of 0.05. (B) Mean &lt;em&gt;z&lt;/em&gt;-scores of HI (red) and non-HI (blue) gene properties. Error bars represent the SEM. (C) The receiver-operating characteristic (ROC) area under curve (AUC) distributions. These were generated using cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The vertical line in the center of the chart represents the random expectation for the ROC plot.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1965/F2.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/1965/F2.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Relationships between HI and non-HI gene properties in rich medium. (A) The <em>p</em>-values testing the difference between HI and non-HI gene property value distributions. These are on a log10 scale and are as estimated by the Mann-Whitney <em>U</em>-test. The vertical line shows a <em>p</em>-value of 0.05. (B) Mean <em>z</em>-scores of HI (red) and non-HI (blue) gene properties. Error bars represent the SEM. (C) The receiver-operating characteristic (ROC) area under curve (AUC) distributions. These were generated using cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The vertical line in the center of the chart represents the random expectation for the ROC plot.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Our data generally show that gene properties associate best with HI scored in rich medium (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Relationships in minimal medium are moderate, and associations with HI in the limited nutrient environments including grape juice are much weaker. However, HI measured in nitrogen-limited medium interestingly associates significantly with several gene properties, and many of these relationships are not present in the other two limited media.</p><p id="p-39">The observation that gene property values associate with rich medium HI more than the other media HI could be a consequence of most gene property values being obtained from rich medium experiments, making them more relevant to that environment. The strong relationships in rich medium might also be a consequence of the yeast laboratory strain being better adapted to rich medium. Therefore, we mainly focused on rich medium phenotypes and their relationships with the gene properties.</p><p id="p-40">In rich medium, protein interaction network degree associates strongest with HI. Other gene properties that show strong relationships with HI in this medium include GI network degree, ORF sequence identity, and protein expression magnitude. We observed weaker associations with HI for promoter sequence conservation and mRNA expression through the cell cycle. Interestingly, although promoter sequence conservation showed a weak relationship with HI, we found significant relationships in five out of six environments, although this result should be interpreted with caution, because there is a strong link between promoter conservation and gene essentiality.</p><p id="p-41">To check the predictive power of the 23 gene properties, we performed ROC curve analysis on each gene property and nutrient environment combination. We generated 100 ROC curves through 20 repeats of five-fold cross-validation. For each of the 100 ROC curves, we calculated the AUC. The distributions of these AUC data show that predictive abilities of gene properties are strongest for rich medium HI. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> shows gene property prediction effectiveness in rich medium, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FigureS1.pdf">Figure S1</a> shows details of this analysis in all six environments.</p><p id="p-42">We focused on HI prediction in rich medium because the gene properties associate strongest with HI in this environment. To determine gene properties suitable for machine-learning model construction, we selected properties with an AUC higher than 0.5, because this represents the neutral expectation under a random model. Some properties could be expressed in several ways, an example being GI degree, in which calculations came from “lenient,” “intermediate,” and “stringent” cut-off datasets (see Materials and Methods for full details of gene property variants). When variants of gene properties existed, the variant with the best predictive ability was carried forward and the others were discarded.</p><p id="p-43">Using the method described, we selected eight gene properties as candidates for building a prediction model. These properties included protein expression level, mRNA cell-cycle regulation score, promoter sequence identity between <em>S. cerevisiae</em> and <em>S. paradoxus</em>, cDNA sequence identity between <em>S. cerevisiae</em> and <em>S. kudriavzevii</em>, degree and betweenness in a genetic interaction network, PPI network degree calculated using the unfiltered BioGRID interaction network, and PPI network betweenness calculated using the BioGRID network previously filtered to remove interactions reported less than twice.</p></div><div id="sec-11" class="subsection"><h3>Relationships between the gene properties</h3><p id="p-44">To build a machine-learning model, we sought sets of gene properties that each contributed unique information. To provide initial clues about the uniqueness of gene properties, we correlated each gene property pair using the Pearson method. These correlations were completed after performing variance-stabilizing transformations when appropriate (see Materials and Methods). Most gene property pairs showed weak correlation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FigureS2.pdf">Figure S2</a>), and these properties were considered later for building the machine-learning model. PPI and GI network betweenness were excluded from our model, because we observed a very strong correlation between GI degree and betweenness and a similar strong correlation between PPI degree and betweenness. In both cases, betweenness was discarded because it yielded the lower AUC (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). The correlation between degree and betweenness for both the GI and PPI networks suggests that genes and proteins with large numbers of interactions are considerably more likely to be central in their networks.</p><p id="p-45">To demonstrate the uniqueness of gene properties, we produced models with different gene properties left out. We found that models incorporating all chosen gene properties (<em>i.e.</em>, those with low correlations between them) performed best, demonstrating that each gene property contributes unique information to the model.</p></div><div id="sec-12" class="subsection"><h3>Associations between gene essentiality, HI, and gene properties</h3><p id="p-46">The rich medium study performed previously, on which most of our analysis is based (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>), showed that HI genes have a tendency to be essential. We sought to determine whether gene essentiality exhibits a confounding effect between HI and our chosen gene properties, because some properties, such as PPI degree (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Jeong <em>et al.</em> 2001</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">He and Zhang 2006</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Zotenko <em>et al.</em> 2008</a>), have been shown to correlate with gene essentiality.</p><p id="p-47">We investigated this by stratifying the data according to essentiality in rich media. Within each of the two resulting strata, corresponding to either essential or nonessential genes, we analyzed the relationships between HI and each gene property. The results of this analysis for five out of six gene properties are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Genetic interaction degree is missing from the analysis because the data we used for this gene property (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Costanzo <em>et al.</em> 2010</a>) only include interactions between homozygous-null mutants and thus do not contain any information about essential genes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationships between five gene properties and HI, stratified according to gene essentiality. The p-values and z-scores represent the differences between the distributions of HI and non-HI gene property values among nonessential and essential, essential, and nonessential gene sets. Error bars represent the SEM. Gene properties include (A) PPI network degree, (B) ORF sequence identity between S. cerevisiae and S. kudriavzevii, (C) promoter sequence identity between S. cerevisiae and S. bayanus, (D) mRNA expression variation through the cell cycle, and (E) protein expression level." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728338711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationships between five gene properties and HI, stratified according to gene essentiality. The &lt;em&gt;p&lt;/em&gt;-values and &lt;em&gt;z&lt;/em&gt;-scores represent the differences between the distributions of HI and non-HI gene property values among nonessential and essential, essential, and nonessential gene sets. Error bars represent the SEM. Gene properties include (A) PPI network degree, (B) ORF sequence identity between &lt;em&gt;S. cerevisiae&lt;/em&gt; and &lt;em&gt;S. kudriavzevii&lt;/em&gt;, (C) promoter sequence identity between &lt;em&gt;S. cerevisiae&lt;/em&gt; and &lt;em&gt;S. bayanus&lt;/em&gt;, (D) mRNA expression variation through the cell cycle, and (E) protein expression level.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1965/F3.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/1965/F3.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463603" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-48" class="first-child">Relationships between five gene properties and HI, stratified according to gene essentiality. The <em>p</em>-values and <em>z</em>-scores represent the differences between the distributions of HI and non-HI gene property values among nonessential and essential, essential, and nonessential gene sets. Error bars represent the SEM. Gene properties include (A) PPI network degree, (B) ORF sequence identity between <em>S. cerevisiae</em> and <em>S. kudriavzevii</em>, (C) promoter sequence identity between <em>S. cerevisiae</em> and <em>S. bayanus</em>, (D) mRNA expression variation through the cell cycle, and (E) protein expression level.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">Among nonessential genes, all five gene properties associate significantly with HI. For essential genes, four gene properties associate significantly with HI, with the remaining relationship involving promoter sequence identity being insignificant. Because HI associates with PPI degree, gene sequence conservation, cell-cycle expression variation, and protein abundance among both essential and nonessential genes, we can infer that although a confounding effect exists between gene essentiality and HI (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), gene essentiality does not fully explain the relationships between the gene properties and HI. Although the association between promoter sequence identity and HI among essential genes was insignificant, suggesting a strong confounding effect, we included this property in our model because it improves predictive performance.</p></div><div id="sec-13" class="subsection"><h3>Construction of LDA model for HI prediction in <em>S. cerevisiae</em></h3><p id="p-50">Using the selection methods, we ultimately chose six unique gene properties that we inferred would be effective at building our HI prediction model. These properties were protein abundance, mRNA expression variation through the cell cycle, DNA percentage identity of the promoter between <em>S. cerevisiae</em> and <em>S. paradoxus</em>, ORF DNA percentage identity between <em>S. cerevisiae</em> and <em>S. kudriavzevii</em>, GI degree from DRYGIN, and PPI degree from BioGRID calculated using the unfiltered network.</p><p id="p-51">The model with the best combination of gene properties was established by producing an LDA model for each possible gene property combination. Every model thus produced was validated through 20 repeats of five-fold cross-validation (see Materials and Methods), yielding 100 ROC curves.</p><p id="p-52">All gene properties included in our model have missing data for some genes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). To handle missing data, we tested five methods, including rolling back to a simpler model, excluding incomplete cases, EM algorithm, MI, and median imputation (see Materials and Methods). We examined model performance by calculating the AUC in the region of the ROC plot where false-positive rate is no larger than 0.1 (FPR ≤ 0.1 AUC). A high FPR ≤ 0.1 AUC minimizes the FPR and indicates a model in which genes with the highest posterior probabilities are most likely to be HI. High FPR ≤ 0.1 AUC models therefore allow genes to be ranked by their posterior probabilities such that genes at the top of the list are highly enriched for HI.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463536/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463536/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">List of the 6 gene properties used in the 6GP model showing proportion of gene property data missing across the yeast genome</span><div class="sb-div caption-clear"></div></div></div><p id="p-54">The missing value handling methods produced similar ROC curves, showing that prediction quality is largely independent of the imputation method used. We chose median imputation to produce our candidate models because it yields a high FPR ≤ 0.1 AUC and has low cross-validation AUC variation. The FPR ≤ 0.1 AUC distributions for our median imputation models incorporating all possible combinations of gene properties are presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. These distributions are shown for all imputation methods tested in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FigureS3.pdf">Figure S3</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="False-positive rate (FPR) &#x2264; 0.1 area under curve (AUC) distributions across all combinations of gene properties, using median imputation. This demonstrates that model performance tends to increase as more gene properties are added. Our candidate six gene properties (6GP) model is highlighted with an arrow. The three letter codes identify gene properties and are described in the legend. Distributions are for 100 receiver-operating characteristic (ROC) curves generated during cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The black horizontal line represents the random expectation from the ROC plot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728338711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;False-positive rate (FPR) &#x2264; 0.1 area under curve (AUC) distributions across all combinations of gene properties, using median imputation. This demonstrates that model performance tends to increase as more gene properties are added. Our candidate six gene properties (6GP) model is highlighted with an arrow. The three letter codes identify gene properties and are described in the legend. Distributions are for 100 receiver-operating characteristic (ROC) curves generated during cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The black horizontal line represents the random expectation from the ROC plot.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1965/F4.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/1965/F4.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463489" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-55" class="first-child">False-positive rate (FPR) ≤ 0.1 area under curve (AUC) distributions across all combinations of gene properties, using median imputation. This demonstrates that model performance tends to increase as more gene properties are added. Our candidate six gene properties (6GP) model is highlighted with an arrow. The three letter codes identify gene properties and are described in the legend. Distributions are for 100 receiver-operating characteristic (ROC) curves generated during cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The black horizontal line represents the random expectation from the ROC plot.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">We observed a positive correlation between number of gene properties in the model and the FPR ≤ 0.1 AUC, indicating that each gene property contributes useful information for predicting HI. The model incorporating all six gene properties (6GP) was chosen as our candidate for prediction because it has the highest value for FPR ≤ 0.1 AUC. The ROC curve for the 6GP model is shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. We used this model to produce posterior probabilities of HI for all yeast genes, which are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FileS2.xlsx">File S2</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of the six gene property (6GP) candidate model. Receiver-operating characteristic (ROC) curve of the best model (6GP), which combines the six gene properties described in the text. The dark line shows the average of 100 ROC curves, with error bars indicating 1 SD. Gray lines represent 100 ROC curves produced during cross-validation superimposed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728338711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Performance of the six gene property (6GP) candidate model. Receiver-operating characteristic (ROC) curve of the best model (6GP), which combines the six gene properties described in the text. The dark line shows the average of 100 ROC curves, with error bars indicating 1 SD. Gray lines represent 100 ROC curves produced during cross-validation superimposed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1965/F5.medium.gif" width="440" height="396"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/11/1965/F5.medium.gif" width="440" height="396"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463600" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-57" class="first-child">Performance of the six gene property (6GP) candidate model. Receiver-operating characteristic (ROC) curve of the best model (6GP), which combines the six gene properties described in the text. The dark line shows the average of 100 ROC curves, with error bars indicating 1 SD. Gray lines represent 100 ROC curves produced during cross-validation superimposed.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Prediction of HI and experimental verification of HI in <em>S. cerevisiae</em></h3><p id="p-58">We used the LDA 6GP model to rank each gene by its inferred probability of being HI. To test whether the model could infer novel and previously undetected HI phenotypes, we selected 23 candidate genes that had not been identified as HI in the previous large-scale study (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>) but had the highest probability scores according to our model. Assays for HI were performed for all candidate genes. We found fitness defects for 6 out of 23 candidates in rich medium, identified 5 out of 23 candidates as HI in F1 medium with nitrogen limitation, and found HI in 1 out of 23 candidate genes for carbon-limited medium.</p><p id="p-59">To identify significantly HI strains, we calculated an AUGC for each replicate well containing either mutant or WT culture. The <em>p</em>-value, describing the difference between mutant and WT AUGC distributions, was then calculated using the Welch <em>t</em>-test. We have reported the ratio between WT and mutant AUGC means as a quantitative measure of fitness for each candidate gene in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> together with <em>p</em>-values for these differences as calculated using the Welch <em>t</em>-test and corrected using the Benjamini-Hochberg procedure.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463607/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463607/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of phenotypes for the 23 candidate genes tested</span><div class="sb-div caption-clear"></div></div></div><p id="p-62">In addition to the 23 candidates, we selected three positive control genes that were detected as significantly HI in the earlier rich medium study. These included two genes, <em>TUB1</em> and <em>RPL25</em>, displaying severe fitness loss in heterozygosis (relative fitness losses of 0.921 and 0.818, respectively), and <em>RPN11</em>, displaying a weak but significant HI phenotype (with a relative fitness loss of 0.971). As negative controls, we selected 24 genes that had both low HI likelihood according to the model and no significant HI phenotype in the earlier experimental work. As background controls, we randomly selected 42 genes from the entire genome to produce an estimate of the rate of HI in the genome, as measured via rich medium monocultures. All controls were tested for HI using the same method used with the candidate genes.</p><p id="p-63">Only 1 out of 42 background control genes showed a significant HI phenotype, in line with the expected frequency of HI as described in previous studies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/TableS2.pdf">Table S2</a>). Growth curves for all significantly HI strains, corresponding to both candidate genes and controls, are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FigureS4.pdf">Figure S4</a></p><p id="p-64">Our candidate strains each carried mutations in genes that were previously undetected in the large-scale rich medium HI profiling study (<a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>). This result confirms that our proposed method for choosing HI candidates has a very good predictive power, because 26% of the candidate genes from the model were found to be HI compared to a total of 3% HI genes detected in the earlier rich medium experiment and 2.3% HI gene coverage among our randomly selected background controls. All of our negative control genes showed no significant growth rate difference from the WT BY4743 strain, demonstrating that our new phenotype identifications are unlikely to be attributable to experimental sample variation.</p><p id="p-65">The HI-positive controls, <em>RPL25</em> and <em>TUB1</em>, showed pronounced and significant haploinsufficiency. <em>RPN11</em>, used as a weak haploinsufficiency-positive control, appeared to grow slightly slower than the WT strain, although the difference was not significant. These data suggest that the experimental method used may miss some weakly HI genes; therefore, the extent of the verification of the HI from our model is probably conservative.</p><p id="p-66">We tested the 23 candidate genes in F1 media with nitrogen and carbon limitation, in addition to our rich medium studies, to see whether the new HI phenotypes are displayed in other environments (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The gene <em>BCY1</em> showed a HI phenotype in all three environments tested, and the genes <em>RPS6B</em>, <em>BCY1</em>, and <em>BRE5</em> showed HI phenotypes in both rich medium and the synthetically defined media with nitrogen limitation. Interestingly, in F1 medium with nitrogen limitation, we found another two candidate genes to be HI, <em>SEC28</em> and <em>HMO1</em>.</p></div><div id="sec-15" class="subsection"><h3>Analysis of HI in <em>Sz. pombe</em></h3><p id="p-67">In addition to our work performed on <em>S. cerevisiae</em>, we examined the available HI data for <em>Sz. pombe</em> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2010</a>). In particular, we looked at relationships between HI and DNA sequence conservation, PPI degree from BioGRID (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Stark 2006</a>), and predicted GI degree from a recent study (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Koch <em>et al.</em> 2012</a>). Other biological properties were not examined because of unavailability of comprehensive data in <em>Sz. pombe</em>. To determine gene sequence conservation, we compared the ORF sequence data between <em>Sz. pombe</em> and two closely related species, <em>Sz. octosporus</em> and <em>Sz. cryophilus</em>. Using genome annotations, we performed ORF alignments and calculated sequence conservation statistics before comparing these statistics against <em>Sz. pombe</em> HI data. Interestingly, we found significant positive associations with HI for DNA and protein sequence identity between <em>Sz. pombe</em> and both <em>Sz. octosporus</em> and <em>Sz. cryophilus</em>, although correlation between HI and <em>dN</em>/<em>dS</em> was insignificant (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). We also found a significant moderate positive relationship between HI and PPI degree. The association between HI and predicted GI degree was not significant. In general, these preliminary results suggest that our HI prediction method based on the biological properties of the cell could be applied to other microorganisms.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationships between HI and non-HI gene properties for Sz. pombe in rich medium. (A) The p-values testing the difference between HI and non-HI gene property value distributions. These are on a log10 scale and are as estimated by the Mann-Whitney U-test. The vertical line shows a p-value of 0.05. (B) Mean z-scores of HI (red) and non-HI (blue) gene properties. Error bars represent the SEM. (C) The receiver-operating characteristic (ROC) area under curve (AUC) distributions. These were generated using cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The vertical line in the center of the chart represents the random expectation for the ROC plot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728338711" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationships between HI and non-HI gene properties for &lt;em&gt;Sz. pombe&lt;/em&gt; in rich medium. (A) The &lt;em&gt;p&lt;/em&gt;-values testing the difference between HI and non-HI gene property value distributions. These are on a log10 scale and are as estimated by the Mann-Whitney &lt;em&gt;U&lt;/em&gt;-test. The vertical line shows a &lt;em&gt;p&lt;/em&gt;-value of 0.05. (B) Mean &lt;em&gt;z&lt;/em&gt;-scores of HI (red) and non-HI (blue) gene properties. Error bars represent the SEM. (C) The receiver-operating characteristic (ROC) area under curve (AUC) distributions. These were generated using cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The vertical line in the center of the chart represents the random expectation for the ROC plot.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1965/F6.medium.gif" width="440" height="116"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/11/1965/F6.medium.gif" width="440" height="116"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1965/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463590" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-68" class="first-child">Relationships between HI and non-HI gene properties for <em>Sz. pombe</em> in rich medium. (A) The <em>p</em>-values testing the difference between HI and non-HI gene property value distributions. These are on a log10 scale and are as estimated by the Mann-Whitney <em>U</em>-test. The vertical line shows a <em>p</em>-value of 0.05. (B) Mean <em>z</em>-scores of HI (red) and non-HI (blue) gene properties. Error bars represent the SEM. (C) The receiver-operating characteristic (ROC) area under curve (AUC) distributions. These were generated using cross-validation (see Materials and Methods). Whiskers represent the lowest point within 1.5 interquartile range (IQR) of the lower quartile and the highest point within 1.5 IQR of the upper quartile. Dots represent outliers of the aforementioned ranges. The vertical line in the center of the chart represents the random expectation for the ROC plot.</p><div class="sb-div caption-clear"></div></div></div><p id="p-69">We also examined the level of HI phenotype conservation between <em>S. cerevisiae</em> and <em>Sz. pombe</em> ortholog genes. A set of 2372 <em>Sz. pombe</em> genes was considered. This set contained 279 <em>Sz. pombe</em> and 93 <em>S. cerevisiae</em> HI genes. The set also only contained those genes with no duplicate gene orthologs in <em>S. cerevisiae</em> and was such that rich medium heterozygous-null mutant phenotypes were available from both organisms. We found that only 17 out of 2372 genes were HI in both <em>S. cerevisiae</em> and <em>Sz. pombe</em>. This result shows that orthology is not informative of HI in these two yeasts because HI does not tend to be conserved between members of orthologous gene pairs.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-70">In this work, we identified several gene properties with significant differences between HI and non-HI genes. We exploited these associations by building a machine-learning model that prioritizes candidate genes according to their inferred HI probabilities. By prioritizing candidate genes, we discovered novel HI phenotypes that were not detected previously in genome-scale screens. In addition, the relationships uncovered here between HI and gene properties provide useful clues in terms of understanding HI mechanisms.</p><p id="p-71">The relationships identified here show that HI genes tend to be highly expressed and highly conserved, with large numbers of genetic and physical interactions. We have also shown that HI genes tend to have slightly less conservation at the promoter and slightly less cell-cycle regulation. The reduced cell-cycle regulation suggests that HI genes tend to be constitutively expressed (<em>i.e.</em>, “always on”), at least in terms of the cell cycle. One would expect hemizygosity of a constitutively expressed gene to cause a greater reduction in protein expression than a more highly regulated gene, because there are probably fewer regulatory mechanisms to respond to gene copy number variation in constitutively expressed genes. The relationship between transcriptional regulation and HI could be explored more fully by examining microarray data beyond the cell cycle.</p><p id="p-72">We found that PPI degree associates most strongly with HI in rich medium. This relationship supports the idea that gene dosage is important for viability of protein complexes. During construction of our model, we tested three PPI networks, each of which was generated from physical interaction data in BioGRID. These included an unprocessed interaction network along with two stringent networks that only considered interactions reported at least two or three times. Interestingly, the unprocessed network performed best. We think the stringent networks performed less well because considerable amounts of data are excluded during the filtering process. Approximately 80% of interactions are removed when only considering interactions reported at least twice, whereas approximately 90% are removed when only considering interactions reported at least three times.</p><p id="p-73">Our study demonstrates that some HI phenotypes are missing from existing genome-scale data. This could be attributable to interactions between heterogeneous strains sharing the same environment, disagreements between biological replicates, or assay sensitivity limitations. For example, there could be a dependency of HI phenotypes on the structure of the competing population and from general growth conditions. In fact, competition environments in which multiple mutants are grown together can be rather different from medium containing only monocultures or two strains. Six thousands mutants competing in the same nutritional context are likely to excrete several different types and quantities of metabolites (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Allen <em>et al.</em> 2003</a>), which could affect the growth of the population both in the negative direction and in the positive direction. Although the extent of this cross-feeding effect is not known, obtaining genome-scale monoculture HI profile data would be a useful step toward understanding it.</p><p id="p-74">The level at which false-positives are present in large-scale genomic data is also not entirely known. <a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> (2005)</a> tested 30 HI genes in individual growth assays to prove the validity of the molecular bar-coding to reproducibly score quantitative fitness differences as small as 2%.</p><p id="p-75">HI phenotype differences could also arise from differences between experimental methods. This study and the two previous studies (<a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>; <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Delneri <em>et al.</em> 2008</a>) each used different experimental methods and metrics to identify HI genes. In the <a id="xref-ref-8-10" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> (2005)</a> study, a batch serial dilution competition approach was used. Two independently constructed pools of hemizygous strains were diluted at five generation intervals. Hybridization tags were used in combination with a microarray and a regression-based approach to quantify differences between the growth rates of strains relative to WT. Genes were considered significantly HI when both replicates had a fitness value less than 0.95 and at least one tag in both replicates had a <em>p</em>-value &lt; 0.05.</p><p id="p-76">The <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Delneri <em>et al.</em> (2008)</a> work used competition experiments in continuous cultures to identify HI genes via hybridization of the strain bar codes. The advantage of this method was that both growth rate and pH are kept constant, allowing a more sensitive identification of HI phenotypes. To determine significant HI strains, a regression-based method was used, followed by calculation of <em>p</em>-values. False discovery rates were estimated using the Benjamini-Hochberg method to produce <em>Q</em>-values. Growth rates with <em>Q</em> &lt; 0.01 were considered statistically significant.</p><p id="p-77">Both of these studies, based on strain-specific tag hybridization, are sensitive and quantitative in nature for the determination of HI. In our study, we measured growth curves of monocultures and used the difference between mutant and WT AUGC means as a quantitative measure of fitness. To determine the significance of the difference between these means, we calculated <em>p</em>-values using the Welch <em>t</em>-test, followed by correction using the Benjamini–Hochberg procedure. Despite the fact that growth curves are typically less sensitive than genome-wide competition experiments to detect fitness declines, we were successful in identifying new HI phenotypes.</p><p id="p-78">Our gene properties associate most strongly with rich medium HI, with weaker relationships between gene properties and HI in minimal, nitrogen-limited, carbon-limited, phosphate-limited, and grape juice media. For some gene properties, this might be because property data were produced largely in experiments that used rich medium. Properties potentially affected by environmental conditions include PPI and GI degree, protein expression level, and magnitude of cell-cycle regulation. For example, the PPI data mostly comprise interactions reported through affinity capture methods, which are affected by the nutrient environment. The strong associations with HI in rich medium may also be attributable to the laboratory strains being more adapted to rich medium, thus resulting in gene properties showing stronger relationships in this environment. The associations between gene properties and HI are much weaker for carbon-limited, nitrogen-limited, phosphate-limited, and grape juice environments. This may be a result of different experimental conditions in the corresponding study (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Delneri <em>et al.</em> 2008</a>); in that work, the cell culture was held at maximum growth rate, whereas the rich medium study (<a id="xref-ref-8-11" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>) and our study have not attempted to do this. In general, however, more experimental HI phenotype data, ideally using different strain backgrounds, would be necessary to determine the extent to which these conditions might affect HI phenotypes detected.</p><p id="p-79">To investigate the sensitivity of our method for detecting small fitness differences, we selected the gene <em>RPN1</em> as a “weak” positive control. This gene had a subtle but significant HI phenotype in a previous rich medium study (fitness difference for <em>RPN1</em> = 0.971 <em>vs. TUB1</em> = 0.921 and <em>RPL25</em> = 0.818 for our two “strong” positive controls). When we tested this gene for HI using our growth curve assay, the fitness loss registered was very small and therefore not significant. This undetected HI might be either a consequence of the limited sensitivity of the method compared to the genomic bar code analysis or a consequence of the lack of yeast competitors in the monoculture. An improved sensitivity for HI phenotypes might, in fact, be achieved via one-to-one competition experiments, although such experiments could be affected by cross-feeding interactions between the reference strain and the mutant. These results show that our experimental validation of the model is likely to underestimate the HI phenotypes and is a conservative evaluation of the predictive power of our method.</p><p id="p-80">Ribosomal and core cellular process genes were heavily over-represented in the earlier rich medium study (<a id="xref-ref-8-12" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>). Our six newly identified rich medium HI genes include two ribosomal genes (<em>RPS6B</em> and <em>RPL10</em>), a proteasome gene (<em>RPN10</em>), and three core cellular process genes (<em>BCY1</em>, <em>BRE5</em>, and <em>GCN5</em>), one of which is an enzyme (<em>GCN5</em>).</p><p id="p-81">To see whether our rich medium phenotypes were displayed in other environments, we performed additional experiments in different nutrient-limited conditions using our 23 candidate genes. Five candidates were found to be HI in F1 medium with nitrogen limitation, and only one HI gene was detected in F1 medium with carbon limitation. These data suggest that there is some overlap of HI phenotypes between YPD and nitrogen-limited media, whereas in the carbon-limited medium the overlap is smaller. This observation was not unexpected because the HI genes in nitrogen-limited medium associate moderately well with the various gene properties in the model, whereas carbon-limited HI does not (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1/FigureS1.pdf">Figure S1</a>, C and D). One of the genes we found to be HI in nitrogen-limited medium, <em>SEC28</em>, is not HI in rich medium. Sec28p has been previously shown to interact with and stabilize Cop1p (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duden <em>et al.</em> 1998</a>), and <em>COP1</em> was found to be HI in nitrogen-limited medium previously (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Delneri <em>et al.</em> 2008</a>). These results suggest that the genes <em>SEC28</em> and <em>COP1</em> might have a shared role in nitrogen stress response.</p><p id="p-82">Previous work (<a id="xref-ref-8-13" class="xref-bibr" href="#ref-8">Deutschbauer <em>et al.</em> 2005</a>) showed that HI genes have a higher likelihood of being essential than other genes. Because PPI degree is known to correlate with gene essentiality (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Jeong <em>et al.</em> 2001</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">He and Zhang 2006</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Zotenko <em>et al.</em> 2008</a>), we set out to investigate the level by which gene essentiality is a confounding factor between HI and our chosen gene properties. We divided genes into essential and nonessential groups and then examined the associations between gene properties and HI within those groups. For PPI degree, gene sequence conservation, cell-cycle expression variation, and protein abundance, relationships between HI and gene properties were significant among both essential and nonessential genes. This demonstrates that associations between these properties and HI cannot fully be explained by the over-representation of gene essentiality among HI genes. We therefore suggest that HI and gene essentiality are, in part, independent phenomena.</p><p id="p-83">Promoter sequence identity was an exception among the properties tested; although it associated significantly with HI among nonessential genes, the relationship between essential gene HI and this variable was insignificant. This might be attributable to either the overall weak association between HI and promoter sequence identity or a result of gene essentiality fully explaining the relationship between promoter conservation and HI. We opted to leave promoter sequence identity in the model because it improved predictive performance.</p><p id="p-84">Availability of gene property data in other organisms may allow their HI to be predicted. We show that this may be possible in <em>Sz. pombe</em> by demonstrating positive relationships between <em>Sz. pombe</em> HI and both PPI degree and ORF sequence conservation. The sequence conservation associations, although significant, were only weak. This may reflect the fact that the two <em>Schizosaccharomyces</em> clade members used to calculate conservation are more distantly related to <em>Sz. pombe</em> than the <em>Saccharomyces sensu stricto</em> clade members are to <em>S. cerevisiae</em>. This greater evolutionary distance may cause signal loss when calculating sequence conservation statistics for <em>Sz. pombe</em>. Evidence supporting this argument includes the fact that DNA sequence conservation associates less with HI than protein sequence conservation. Additionally, there is no significant relationship between <em>dN</em>/<em>dS</em> and HI in <em>Sz. pombe</em>. The association between PPI degree and HI in <em>Sz. pombe</em> is weaker than the equivalent in <em>S. cerevisiae</em>, possibly because the <em>Sz. pombe</em> interaction dataset is less complete than that of <em>S. cerevisiae</em>. In accordance with the <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> (2010)</a> study, we found low HI profile conservation between <em>S. cerevisiae</em> and <em>Sz. pombe</em> orthologs, suggesting that methods that implement prediction of HI based on gene properties could be crucial for identifying HI genes <em>a priori</em> in different species. It is worth noting that <em>Sz. pombe</em> only becomes diploid transiently, just before meiosis, and therefore the biological significance of HI for this organism may be limited to a defined time interval.</p><p id="p-85">In conclusion, we have shown significant relationships between gene properties and HI phenotypes scored in rich medium. We have used these associations to create a model to identify novel HI genes, and we experimentally identified six new hemizygous mutant strains with compromised fitness in rich medium, along with five HI phenotypes in F1 medium with nitrogen limitation and one HI phenotype in carbon-limited medium. This method could be applied to uncover HI phenotypes in other species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-86">We thank Ryan Ames, who provided assistance in calculating betweenness from protein and genetic interaction data, and Valerie Wood for providing <em>S. cerevisiae</em> and <em>Sz. pombe</em> orthology mapping data. We thank the Biotechnology and Biological Sciences Research Council for funding through grant number BB/I020489/1 (to D.D. and S.L.) and for a studentship through the Doctoral Training Centre (to M.N.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008144/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-87"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-08-07"><span class="received-label">Received </span>August 7, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-06"><span class="accepted-label">Accepted </span>September 6, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Norris <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.1" data-doi="10.1038/nbt823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadhurst</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heald</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowland</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">High-throughput classification of yeast mutants for functional genomics using metabolic footprinting.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DAllen%26rft.auinit1%253DJ.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D692%26rft.epage%253D696%26rft.atitle%253DHigh-throughput%2Bclassification%2Bof%2Byeast%2Bmutants%2Bfor%2Bfunctional%2Bgenomics%2Busing%2Bmetabolic%2Bfootprinting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt823%26rft_id%253Dinfo%253Apmid%252F12740584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12740584&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183220800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amundsen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binkley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Saccharomyces Genome Database: the genomics resource of budding yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253D700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.4" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.5" data-doi="10.1038/nature07341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Godoy</span>  <span class="cit-name-given-names">L. M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubner</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comprehensive mass-spectrometry-based proteome quantification of haploid <em>vs.</em> diploid yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">455</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253Dde%2BGodoy%26rft.auinit1%253DL.%2BM.%26rft.volume%253D455%26rft.issue%253D7217%26rft.spage%253D1251%26rft.epage%253D1254%26rft.atitle%253DComprehensive%2Bmass-spectrometry-based%2Bproteome%2Bquantification%2Bof%2Bhaploid%2Bversus%2Bdiploid%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07341%26rft_id%253Dinfo%253Apmid%252F18820680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18820680&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260462100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.1965.6"><div class="cit-metadata unstructured">Delneri, D., 2011 Competition experiments coupled with high-throughput analyses for functional genomics studies in yeast, pp. 271–282 in <em>Yeast Systems Biology</em>, <em>Methods in Molecular Biology</em> edited by J.I. Castrillo, S.G. Oliver Humana Press New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.7" data-doi="10.1038/ng.2007.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delneri</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyle</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gkargkas</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">E. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rash</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification and characterization of high-flux-control genes of yeast through competition analyses in continuous cultures.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDelneri%26rft.auinit1%253DD.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D117%26rft.atitle%253DIdentification%2Band%2Bcharacterization%2Bof%2Bhigh-flux-control%2Bgenes%2Bof%2Byeast%2Bthrough%2Bcompetition%2Banalyses%2Bin%2Bcontinuous%2Bcultures.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.49%26rft_id%253Dinfo%253Apmid%252F18157128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18157128&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252118600027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.8" data-doi="10.1534/genetics.104.036871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036871%26rft_id%253Dinfo%253Apmid%252F15716499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5MTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.9" data-doi="10.1038/nbt1406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzoglou</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">What is the expectation maximization algorithm?</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DDo%26rft.auinit1%253DC.%2BB.%26rft.volume%253D26%26rft.issue%253D8%26rft.spage%253D897%26rft.epage%253D899%26rft.atitle%253DWhat%2Bis%2Bthe%2Bexpectation%2Bmaximization%2Balgorithm%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1406%26rft_id%253Dinfo%253Apmid%252F18688245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18688245&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258325500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.10" data-doi="10.1093/emboj/17.4.985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duden</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kajikawa</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wuestehube</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schekman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">epsilon-COP is a structural component of coatomer that functions to stabilize alpha-COP.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DDuden%26rft.auinit1%253DR.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D985%26rft.epage%253D995%26rft.atitle%253D%257Bepsilon%257D-COP%2Bis%2Ba%2Bstructural%2Bcomponent%2Bof%2Bcoatomer%2Bthat%2Bfunctions%2Bto%2Bstabilize%2B%257Balpha%257D-COP%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.4.985%26rft_id%253Dinfo%253Apmid%252F9463377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIxNy80Lzk4NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.11" data-doi="10.2307/3033543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">A set of measures of centrality based on betweenness.</span> <abbr class="cit-jnl-abbrev">Sociometry</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSociometry%26rft.volume%253D40%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3033543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3033543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977CZ20900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.12" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.1965.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">The Elements of Statistical Learning: Data Mining, Inference, and Prediction</span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Springer-Verlag, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.14" data-doi="10.1371/journal.pgen.0020088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Why do hubs tend to be essential in protein networks?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHe%26rft.auinit1%253DX.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DWhy%2Bdo%2Bhubs%2Btend%2Bto%2Bbe%2Bessential%2Bin%2Bprotein%2Bnetworks%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020088%26rft_id%253Dinfo%253Apmid%252F16751849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16751849&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.15" data-doi="10.1016/S0168-9525(02)02722-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Ka/Ks ratio: diagnosing the form of sequence evolution. <em>Trends in genetics</em></span>. <abbr class="cit-jnl-abbrev">TIG</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTIG%26rft.volume%253D18%26rft.spage%253D486%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902722-1%26rft_id%253Dinfo%253Apmid%252F12175810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02722-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12175810&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.16" data-doi="10.1038/35075138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barabási</span>  <span class="cit-name-given-names">A.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oltvai</span>  <span class="cit-name-given-names">Z. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Lethality and centrality in protein networks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJeong%26rft.auinit1%253DH.%26rft.volume%253D411%26rft.issue%253D6833%26rft.spage%253D41%26rft.epage%253D42%26rft.atitle%253DLethality%2Band%2Bcentrality%2Bin%2Bprotein%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35075138%26rft_id%253Dinfo%253Apmid%252F11333967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35075138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11333967&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168432800033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.17" data-doi="10.1038/nature01644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">423</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D423%26rft.issue%253D6937%26rft.spage%253D241%26rft.epage%253D254%26rft.atitle%253DSequencing%2Band%2Bcomparison%2Bof%2Byeast%2Bspecies%2Bto%2Bidentify%2Bgenes%2Band%2Bregulatory%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01644%26rft_id%253Dinfo%253Apmid%252F12748633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12748633&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182853100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.18" data-doi="10.1038/nbt.1628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.-U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H.-O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of a genome-wide set of gene deletions in the fission yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKim%26rft.auinit1%253DD.%2BU.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D617%26rft.epage%253D623%26rft.atitle%253DAnalysis%2Bof%2Ba%2Bgenome-wide%2Bset%2Bof%2Bgene%2Bdeletions%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1628%26rft_id%253Dinfo%253Apmid%252F20473289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20473289&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278820200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.19" data-doi="10.1177/0962280206075304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kenward</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multiple imputation: current perspectives.</span> <abbr class="cit-jnl-abbrev">Stat. Methods Med. Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistical%2BMethods%2Bin%2BMedical%2BResearch%26rft.stitle%253DStat%2BMethods%2BMed%2BRes%26rft.aulast%253DKenward%26rft.auinit1%253DM.%2BG%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D199%26rft.epage%253D218%26rft.atitle%253DMultiple%2Bimputation%253A%2Bcurrent%2Bperspectives%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0962280206075304%26rft_id%253Dinfo%253Apmid%252F17621468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3BzbW0iO3M6NToicmVzaWQiO3M6ODoiMTYvMy8xOTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.20" data-doi="10.1186/gb-2012-13-7-r57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatfield-Reed</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Conserved rules govern genetic interaction degree across species.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKoch%26rft.auinit1%253DE.%2BN.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253DR57%26rft.epage%253DR57%26rft.atitle%253DConserved%2Brules%2Bgovern%2Bgenetic%2Binteraction%2Bdegree%2Bacross%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-7-r57%26rft_id%253Dinfo%253Apmid%252F22747640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-7-r57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22747640&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toufighi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DRYGIN: a database of quantitative genetic interaction networks in yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">502</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D38%26rft.spage%253D502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.22" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.23" data-doi="10.1214/aoms/1177730491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitney</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">On a test of whether one of two random variables is stochastically larger than the other.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D18%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177730491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177730491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.24" data-doi="10.1098/rsta.1896.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1896</span> <span class="cit-article-title">Mathematical contributions to the theory of evolution. III. Regression, heredity, and panmixia.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. A Contain. Pap. Math. Phys. Character</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BA%253A%2BMathematical%252C%2BPhysical%2Band%2BEngineering%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BA%26rft.aulast%253DPearson%26rft.auinit1%253DK.%26rft.volume%253D187%26rft.issue%253D0%26rft.spage%253D253%26rft.epage%253D318%26rft.atitle%253DMathematical%2BContributions%2Bto%2Bthe%2BTheory%2Bof%2BEvolution.%2BIII.%2BRegression%252C%2BHeredity%252C%2Band%2BPanmixia%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsta.1896.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJyb3lwdGEiO3M6NToicmVzaWQiO3M6OToiMTg3LzAvMjUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.25" data-doi="10.1126/science.1203357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rhind</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative functional genomics of the fission yeasts.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">930</span>–<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203357%26rft_id%253Dinfo%253Apmid%252F21511999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzIvNjAzMi85MzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.26" data-doi="10.1534/g3.111.000273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zill</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The awesome power of yeast evolutionary genetics: New genome sequences and strain resources for the <em>Saccharomyces sensu stricto</em> genus.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DScannell%26rft.auinit1%253DD.%2BR.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D25%26rft.atitle%253DThe%2BAwesome%2BPower%2Bof%2BYeast%2BEvolutionary%2BGenetics%253A%2BNew%2BGenome%2BSequences%2Band%2BStrain%2BResources%2Bfor%2Bthe%2BSaccharomyces%2Bsensu%2Bstricto%2BGenus%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000273%26rft_id%253Dinfo%253Apmid%252F22384314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjEvMS8xMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.27" data-doi="10.1091/mbc.9.12.3273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Comprehensive identification of cell cycle-regulated genes of the yeast <em>Saccharomyces cerevisiae</em> by microarray hybridization.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">3273</span>–<span class="cit-lpage">3297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DSpellman%26rft.auinit1%253DP.%2BT.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D3273%26rft.epage%253D3297%26rft.atitle%253DComprehensive%2BIdentification%2Bof%2BCell%2BCycle-regulated%2BGenes%2Bof%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%2Bby%2BMicroarray%2BHybridization%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.9.12.3273%26rft_id%253Dinfo%253Apmid%252F9843569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjkvMTIvMzI3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.28" data-doi="10.1093/nar/gkj109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">BioGRID: a general repository for interaction datasets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D34%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.29" data-doi="10.1093/nar/gkr853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDowall</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutherford</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PomBase: a comprehensive online resource for fission yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D695</span>–<span class="cit-lpage">D699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr853%26rft_id%253Dinfo%253Apmid%252F22039153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9ENjk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1965.30" data-doi="10.1371/journal.pcbi.1000140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zotenko</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mestre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przytycka</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Why do hubs in the yeast protein interaction network tend to be essential: reexamining the connection between the network topology and essentiality.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DZotenko%26rft.auinit1%253DE.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000140%26rft.epage%253De1000140%26rft.atitle%253DWhy%2Bdo%2Bhubs%2Bin%2Bthe%2Byeast%2Bprotein%2Binteraction%2Bnetwork%2Btend%2Bto%2Bbe%2Bessential%253A%2Breexamining%2Bthe%2Bconnection%2Bbetween%2Bthe%2Bnetwork%2Btopology%2Band%2Bessentiality.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000140%26rft_id%253Dinfo%253Apmid%252F18670624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18670624&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/1965.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1955" title="A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1979" title="Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1965.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KWHHcOIrl4KSb0tvgQM_RMJJHVbUB-_scE3F0WBxyhg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/1965" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/1965" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/1965" class="active" data-icon-position="" data-hide-link-title="0">Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459576" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AC9GNQOnWr0iselLxndxgFpriYetSg5pa3CWTKdGrgM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1965.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/1965" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6kCciDxmIscqRHGjRJenrHyNhwrsZzOqbFz4MStVfQE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/1965" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1965.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459576" id="node459576--2" data-pisa="ggg;3/11/1965" data-pisa-master="ggg;g3.113.008144" data-apath="/ggg/3/11/1965.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew Norris</span>, <span class="highwire-citation-author" data-delta="1">Simon Lovell</span> and <span class="highwire-citation-author" data-delta="2">Daniela Delneri</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1965-1977;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008144 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459576--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew Norris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BNorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BNorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Simon Lovell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BLovell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lovell%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BLovell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.delneri@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.delneri@manchester.ac.uk</a>
<a href="mailto:simon.lovell@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">simon.lovell@manchester.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniela Delneri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniela%2BDelneri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delneri%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniela%2BDelneri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.delneri@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.delneri@manchester.ac.uk</a>
<a href="mailto:simon.lovell@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">simon.lovell@manchester.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/1965" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/1965&amp;title=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/1965&amp;title=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/1965&amp;title=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/1965&amp;text=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/1965&amp;title=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/1965&amp;t=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/1965&amp;title=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/1965&amp;title=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459576/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459576" id="node459576--3" data-pisa="ggg;3/11/1965" data-pisa-master="ggg;g3.113.008144" data-apath="/ggg/3/11/1965.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew Norris</span>, <span class="highwire-citation-author" data-delta="1">Simon Lovell</span> and <span class="highwire-citation-author" data-delta="2">Daniela Delneri</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1965-1977;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008144 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459576--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew Norris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BNorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BNorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Simon Lovell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BLovell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lovell%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BLovell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.delneri@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.delneri@manchester.ac.uk</a>
<a href="mailto:simon.lovell@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">simon.lovell@manchester.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniela Delneri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, Manchester, Lancashire, M13 9PT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniela%2BDelneri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delneri%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniela%2BDelneri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:d.delneri@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">d.delneri@manchester.ac.uk</a>
<a href="mailto:simon.lovell@manchester.ac.uk" class="" data-icon-position="" data-hide-link-title="0">simon.lovell@manchester.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1965&amp;rft.epage=1977&amp;rft.atitle=Characterization%20and%20Prediction%20of%20Haploinsufficiency%20Using%20Systems-Level%20Gene%20Properties%20in%20Yeast&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Norris%2C+Matthew&amp;rft.au=Lovell%2C+Simon&amp;rft.au=Delneri%2C+Daniela"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459576/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459576/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459576/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459576/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459576/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459576/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459576/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459576/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459576/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459576/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459576/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459576/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/1965" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/1965.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/1965.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/11/1965.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/11/1965.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
