<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/281" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/281.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/281.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/281.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/1/281/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/281.atom" />
<meta name="HW.pisa" content="ggg;10/1/281" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.119.400620" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Mollinari et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The hexaploid sweetpotato ( Ipomoea batatas (L.) Lam., 2n = 6x = 90) is an important staple food crop worldwide and plays a vital role in alleviating famine in developing countries. Due to its high ploidy level, genetic studies in sweetpotato lag behind major diploid crops significantly. We built an ultra-dense multilocus integrated genetic map and characterized the inheritance system in a sweetpotato full-sib family using our newly developed software, MAPpoly. The resulting genetic map revealed 96.5% collinearity between I. batatas and its diploid relative I. trifida . We computed the genotypic probabilities across the whole genome for all individuals in the mapping population and inferred their complete hexaploid haplotypes. We provide evidence that most of the meiotic configurations (73.3%) were resolved in bivalents, although a small portion of multivalent signatures (15.7%), among other inconclusive configurations (11.0%), were also observed. Except for low levels of preferential pairing in linkage group 2, we observed a hexasomic inheritance mechanism in all linkage groups. We propose that the hexasomic-bivalent inheritance promotes stability to the allelic transmission in sweetpotato." />
<meta name="DC.Contributor" content="Marcelo Mollinari" />
<meta name="DC.Contributor" content="Bode A. Olukolu" />
<meta name="DC.Contributor" content="Guilherme da S. Pereira" />
<meta name="DC.Contributor" content="Awais Khan" />
<meta name="DC.Contributor" content="Dorcus Gemenet" />
<meta name="DC.Contributor" content="G. Craig Yencho" />
<meta name="DC.Contributor" content="Zhao-Bang Zeng" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The hexaploid sweetpotato (&lt;i&gt;Ipomoea batatas&lt;/i&gt; (L.) Lam., 2n = 6x = 90) is an important staple food crop worldwide and plays a vital role in alleviating famine in developing countries. Due to its high ploidy level, genetic studies in sweetpotato lag behind major diploid crops significantly. We built an ultra-dense multilocus integrated genetic map and characterized the inheritance system in a sweetpotato full-sib family using our newly developed software, MAPpoly. The resulting genetic map revealed 96.5% collinearity between &lt;i&gt;I. batatas&lt;/i&gt; and its diploid relative &lt;i&gt;I. trifida&lt;/i&gt;. We computed the genotypic probabilities across the whole genome for all individuals in the mapping population and inferred their complete hexaploid haplotypes. We provide evidence that most of the meiotic configurations (73.3%) were resolved in bivalents, although a small portion of multivalent signatures (15.7%), among other inconclusive configurations (11.0%), were also observed. Except for low levels of preferential pairing in linkage group 2, we observed a hexasomic inheritance mechanism in all linkage groups. We propose that the hexasomic-bivalent inheritance promotes stability to the allelic transmission in sweetpotato.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/281" />
<meta name="citation_id" content="10/1/281" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/281" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/281.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/281.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/281.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400620" />
<meta name="citation_pmid" content="31732504" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="281" />
<meta name="citation_lastpage" content="292" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marcelo Mollinari" />
<meta name="citation_author_institution" content="Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina," />
<meta name="citation_author_institution" content="Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina," />
<meta name="citation_author_email" content="mmollin@ncsu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7001-8498" />
<meta name="citation_author" content="Bode A. Olukolu" />
<meta name="citation_author_institution" content="Department of Entomology and Plant Pathology, University of Tennessee, Knoxville, Tennessee," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4143-8909" />
<meta name="citation_author" content="Guilherme da S. Pereira" />
<meta name="citation_author_institution" content="Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina," />
<meta name="citation_author_institution" content="Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7106-8630" />
<meta name="citation_author" content="Awais Khan" />
<meta name="citation_author_institution" content="Plant Pathology and Plant-Microbe Biology Section, Cornell University, Geneva, New York, and" />
<meta name="citation_author" content="Dorcus Gemenet" />
<meta name="citation_author_institution" content="International Potato Center, ILRI Campus, Nairobi, Kenya" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4901-1694" />
<meta name="citation_author" content="G. Craig Yencho" />
<meta name="citation_author_institution" content="Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6583-0628" />
<meta name="citation_author" content="Zhao-Bang Zeng" />
<meta name="citation_author_institution" content="Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina," />
<meta name="citation_author_institution" content="Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3115-1149" />
<meta name="citation_reference" content="citation_journal_title=Zuo Wu Xue Bao;citation_author=L. Ai-xian;citation_author=L. Qing-chang;citation_author=W. Qing-mei;citation_author=Z. Li-ming;citation_author=Z. Hong;citation_title=Establishment of Molecular Linkage Maps Using SRAP Markers in Sweet Potato.;citation_pages=1286-1295;citation_volume=36;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=K. S. Aitken;citation_author=P. A. Jackson;citation_author=C. L. McIntyre;citation_title=Construction of a genetic linkage map for Saccharum officinarum incorporating both simplex and duplex markers to increase genome coverage.;citation_pages=742-756;citation_volume=50;citation_year=2007;citation_issue=8;citation_pmid=17893734;citation_doi=10.1139/G07-056" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=P. Arana;citation_author=R. B. Nicklas;citation_title=Orientation and segregation of a micromanipulated multivalent: familiar principles, divergent outcomes.;citation_pages=399-412;citation_volume=101;citation_year=1992;citation_issue=7;citation_pmid=1618023;citation_doi=10.1007/BF00582834" />
<meta name="citation_reference" content="Austin, D. F., 1988 Exploration, Maintenance and Utilization of Sweet Potato Genetic Resources.No Title. In Report of the First Sweet Potato Planning Conference, pp. 27–59, Lima, Peru, International Potato Center." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. P. Bilton;citation_author=M. R. Schofield;citation_author=M. A. Black;citation_author=D. Chagné;citation_author=P. L. Wilcox;citation_title=Accounting for Errors in Low Coverage High-Throughput Sequencing Data When Constructing Genetic Maps Using Biparental Outcrossed Populations.;citation_volume=209;citation_year=2018;citation_pmid=29487138;citation_doi=10.1534/genetics.117.300627" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. M. Bourke;citation_author=V. W. Gitonga;citation_author=R. E. Voorrips;citation_author=R. G. Visser;citation_author=F. A. Krens;citation_title=Multi-environment QTL analysis of plant and flower morphological traits in tetraploid rose.;citation_pages=2055-2069;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3132-4" />
<meta name="citation_reference" content="citation_title=Discussions in cytogenetics;citation_year=1962" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. V. Butruille;citation_author=L. S. Boiteux;citation_title=Selection-mutation balance in polysomic tetraploids: impact of double reduction and gametophytic selection on the frequency and subchromosomal localization of deleterious mutations.;citation_volume=97;citation_year=2000;citation_pmid=10823890;citation_doi=10.1073/pnas.100101097" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. A. Cartwright;citation_author=M. Troggio;citation_author=R. Velasco;citation_author=A. Gutin;citation_title=Genetic mapping in the presence of genotyping errors.;citation_volume=176;citation_year=2007;citation_pmid=17277374;citation_doi=10.1534/genetics.106.063982" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=J. C. Cervantes-Flores;citation_author=G. C. Yencho;citation_author=A. Kriegner;citation_author=K. V. Pecota;citation_author=M. A. Faulk;citation_title=Development of a genetic linkage map and identification of homologous linkage groups in sweetpotato using multiple-dose AFLP markers.;citation_pages=511-532;citation_volume=21;citation_year=2008;citation_doi=10.1007/s11032-007-9150-6" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=J. Cheema;citation_author=J. Dicks;citation_title=Computational approaches and software tools for genetic linkage map estimation in plants.;citation_volume=10;citation_year=2009;citation_pmid=19933208;citation_doi=10.1093/bib/bbp045" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Doerge;citation_author=B. A. Craig;citation_title=Model selection for quantitative trait locus analysis in polyploids;citation_pages=7951-7956;citation_volume=97;citation_year=2000;citation_issue=14;citation_pmid=10884425;citation_doi=10.1073/pnas.97.14.7951" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=R. A. Fisher;citation_title=The theoretical consequences of polyploid inheritance for the mid style form of Lythrum salicaria.;citation_pages=31-38;citation_volume=11;citation_year=1941;citation_doi=10.1111/j.1469-1809.1941.tb02268.x" />
<meta name="citation_reference" content="citation_title=Quantitative genetics and breeding methods in autopolyploids plants;citation_year=2003" />
<meta name="citation_reference" content="Gemenet, D. C., G. da Silva Pereira, B. De Boeck, J. C. Wood, M. Mollinari, et al., 2019 Quantitative trait loci and differential gene expression analyses reveal the genetic basis for negatively associated β-carotene and starch content in hexaploid sweetpotato [Ipomoea batatas (l.) lam.]. Theor Appl Genet https://link.springer.com/article/10.1007/s00122-019-03437-7" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_author=Å. Gustafsson;citation_author=I. Gadd;citation_title=Mutations and crop improvement III. Ipomoea batatas (L.) Poir. (Convolvulaceae).;citation_pages=77-89;citation_volume=53;citation_year=1965;citation_doi=10.1111/j.1601-5223.1965.tb01981.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=C. Hackett;citation_author=J. E. Bradshaw;citation_author=R. C. Meyer;citation_author=J. W. Mcnicol;citation_author=D. Milbourne;citation_author=R. Waugh;citation_title=Linkage analysis in tetraploid species: a simulation study.;citation_pages=143-153;citation_volume=71;citation_year=1998;citation_doi=10.1017/S0016672398003188" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. D. Hollister;citation_title=Polyploidy: Adaptation to the genomic environment.;citation_pages=1034-1039;citation_volume=205;citation_year=2015;citation_pmid=25729801;citation_doi=10.1111/nph.12939" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=R. C. Jackson;citation_author=J. Casey;citation_title=Cytogenetic Analyses of Autopolyploids: Models and Methods for Triploids to Octoploids.;citation_pages=487-501;citation_volume=69;citation_year=1982;citation_doi=10.2307/2443057" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=C. Jiang;citation_author=Z.-B. Zeng;citation_title=Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines.;citation_pages=47-58;citation_volume=101;citation_year=1997;citation_issue=1;citation_pmid=9465409;citation_doi=10.1023/A:1018394410659" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=A. Kriegner;citation_author=J. C. Cervantes;citation_author=K. Burg;citation_author=R. O. Mwanga;citation_author=D. Zhang;citation_title=A genetic linkage map of sweetpotato [Ipomoea batatas(l.) lam.] based on aflp markers.;citation_pages=169-185;citation_volume=11;citation_year=2003;citation_doi=10.1023/A:1022870917230" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. S. Lander;citation_author=P. Green;citation_title=Construction of multilocus genetic linkage maps in humans.;citation_pages=2363-2367;citation_volume=84;citation_year=1987;citation_issue=8;citation_pmid=3470801;citation_doi=10.1073/pnas.84.8.2363" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. J. Leach;citation_author=L. Wang;citation_author=M. J. Kearsey;citation_author=Z. Luo;citation_title=Multilocus tetrasomic linkage analysis using hidden markov chain model.;citation_volume=107;citation_year=2010;citation_pmid=20142473;citation_doi=10.1073/pnas.0908477107" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=T. Lenormand;citation_author=J. Engelstädter;citation_author=S. E. Johnston;citation_author=E. Wijnker;citation_author=C. R. Haag;citation_title=Evolutionary mysteries in meiosis.;citation_pages=1-14;citation_volume=371;citation_year=2016;citation_pmid=26833842;citation_doi=10.1098/rstb.2015.0124" />
<meta name="citation_reference" content="citation_title=The Sweetpotato;citation_author=G. Loebenstein;citation_title=Origin, Distribution and Economic Importance;citation_year=2009;citation_doi=10.1007/978-1-4020-9475-0_2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. W. Luo;citation_author=C. A. Hackett;citation_author=J. E. Bradshaw;citation_author=J. W. McNicol;citation_author=D. Milbourne;citation_title=Construction of a Genetic Linkage Map in Tetraploid Species Using Molecular Markers;citation_pages=1369-1385;citation_volume=157;citation_year=2001;citation_issue=3;citation_pmid=11238421" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. W. Luo;citation_author=R. M. Zhang;citation_author=M. J. Kearsey;citation_title=Theoretical basis for genetic linkage analysis in autotetraploid species.;citation_volume=101;citation_year=2004;citation_pmid=15100415;citation_doi=10.1073/pnas.0304482101" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. L. Magoon;citation_author=R. Krishnan;citation_author=K. Vijaya Bai;citation_title=Cytological evidence on the origin of sweet potato.;citation_pages=360-366;citation_volume=40;citation_year=1970;citation_doi=10.1007/BF00285415" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=K. Mather;citation_title=Segregation and linkage in autotetraploids.;citation_pages=287-314;citation_volume=32;citation_year=1936;citation_doi=10.1007/BF02982683" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Mollinari;citation_author=A. A. F. Garcia;citation_title=Linkage analysis and haplotype phasing in experimental autopolyploid populations with high ploidy level using hidden markov models. G3: Genes, Genomes;citation_pages=3297-3314;citation_volume=9;citation_year=2019" />
<meta name="citation_reference" content="citation_author=M. Mollinari;citation_author=O. Serang;citation_title=Quantitative SNP Genotyping of Polyploids with MassARRAY and Other Platforms.;citation_volume=1245;citation_year=2015;citation_doi=10.1007/978-1-4939-1966-6_17" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=Y. Monden;citation_author=M. Tahara;citation_title=Genetic linkage analysis using DNA markers in sweetpotato.;citation_pages=41-51;citation_volume=67;citation_year=2017;citation_doi=10.1270/jsbbs.16142" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=P. Muñoz-Rodríguez;citation_author=T. Carruthers;citation_author=J. R. Wood;citation_author=B. R. Williams;citation_author=K. Weitemier;citation_title=Reconciling Conflicting Phylogenies in the Origin of Sweet Potato and Dispersal to Polynesia.;citation_pages=1246-1256.e12;citation_volume=28;citation_year=2018;citation_doi=10.1016/j.cub.2018.03.020" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. W. Mwathi;citation_author=M. Gupta;citation_author=C. Atri;citation_author=S. S. Banga;citation_author=J. Batley;citation_title=Segregation for fertility and meiotic stability in novel Brassica allohexaploids.;citation_pages=767-776;citation_volume=130;citation_year=2017;citation_pmid=000398737400;citation_doi=10.1007/s00122-016-2850-8" />
<meta name="citation_reference" content="Pereira, G. d. S., D. C. Gemenet, M. Mollinari, B. A. Olukolu, J. C. Wood, et al., 2019 Multiple QTL mapping in autopolyploids: a random-effect model approach with application in a hexaploid sweetpotato full-sib population. bioRxiv p. 10.1101/622951" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. F. Preedy;citation_author=C. A. Hackett;citation_title=A rapid marker ordering approach for high-density genetic linkage maps in experimental autotetraploid populations using multidimensional scaling.;citation_pages=2117-2132;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2761-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=L. Rabiner;citation_title=A tutorial on hidden markov models and selected applications in speech recognition.;citation_pages=257-286;citation_volume=77;citation_year=1989;citation_doi=10.1109/5.18626" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=J. Ramsey;citation_author=D. W. Schemske;citation_title=Neopolyploidy in Flowering Plants.;citation_volume=33;citation_year=2002;citation_doi=10.1146/annurev.ecolsys.33.010802.150437" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=M. I. Ripol;citation_author=G. A. Churchill;citation_author=J. A. G. D. Silva;citation_author=M. Sorrells;citation_title=Statistical aspects of genetic mapping in autopolyploids.;citation_pages=31-41;citation_volume=235;citation_year=1999;citation_issue=1-2;citation_pmid=10415330;citation_doi=10.1016/S0378-1119(99)00218-8" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Roullier;citation_author=A. Duputié;citation_author=P. Wennekes;citation_author=L. Benoit;citation_author=V. M. Fernández Bringas;citation_title=Disentangling the Origins of Cultivated Sweet Potato (Ipomoea batatas (L.) Lam.).;citation_pages=e62707;citation_volume=8;citation_year=2013;citation_pmid=23723970;citation_doi=10.1371/journal.pone.0062707" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Santos;citation_author=D. Alfaro;citation_author=S. J. Armstrong;citation_author=F. C. H. Franklin;citation_author=G. H. Jones;citation_title=Partial Diploidization of Meiosis in Autotetraploid Arabidopsis thaliana;citation_pages=1533-1540;citation_volume=165;citation_year=2003;citation_issue=3;citation_pmid=14668400" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Serang;citation_author=M. Mollinari;citation_author=A. A. Garcia;citation_title=Efficient exact maximum a posteriori computation for bayesian SNP genotyping in polyploids.;citation_pages=e30906-e30906;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363513;citation_doi=10.1371/journal.pone.0030906" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=K. Shirasawa;citation_author=M. Tanaka;citation_author=Y. Takahata;citation_author=D. Ma;citation_author=Q. Cao;citation_title=A high-density SNP genetic map consisting of a complete set of homologous groups in autohexaploid sweetpotato (Ipomoea batatas).;citation_pages=44207;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep44207" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Stift;citation_author=C. Berenos;citation_author=P. Kuperus;citation_author=P. H. van Tienderen;citation_title=Segregation Models for Disomic, Tetrasomic and Intermediate Inheritance in Tetraploids: A General Procedure Applied to Rorippa (Yellow Cress) Microsatellite Data.;citation_volume=179;citation_year=2008;citation_pmid=18689891;citation_doi=10.1534/genetics.107.085027" />
<meta name="citation_reference" content="citation_title=Meiotic configurations;citation_author=J. Sybenga;citation_year=1975;citation_doi=10.1007/978-3-642-80960-6" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=K. Ukoskit;citation_author=P. G. Thompson;citation_title=Autopolyploidy vs. allopolyploidy and low-density randomly amplified polymorphic DNA linkage maps of sweetpotato.;citation_pages=822-828;citation_volume=122;citation_year=1997;citation_doi=10.21273/JASHS.122.6.822" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. van Geest;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=A. Marasek-Ciolakowska;citation_author=Y. Liao;citation_title=An ultra-dense integrated linkage map for hexaploid chrysanthemum enables multi-allelic qtl analysis.;citation_pages=2527-2541;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2974-5" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=P. A. Wadl;citation_author=B. A. Olukolu;citation_author=S. E. Branham;citation_author=R. L. Jarret;citation_author=G. C. Yencho;citation_title=Genetic Diversity and Population Structure of the USDA Sweetpotato (Ipomoea batatas) Germplasm Collections Using GBSpoly.;citation_pages=1166;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.01166" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. K. Wu;citation_author=W. Burnquist;citation_author=M. E. Sorrells;citation_author=T. L. Tew;citation_author=P. H. Moore;citation_title=The detection and estimation of linkage in polyploids using single-dose restriction fragments.;citation_pages=294-300;citation_volume=83;citation_year=1992;citation_pmid=24202510;citation_doi=10.1007/BF00224274" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Wu;citation_author=K. H. Lau;citation_author=Q. Cao;citation_author=J. P. Hamilton;citation_author=H. Sun;citation_title=Genome sequences of two diploid wild relatives of cultivated sweetpotato reveal targets for genetic improvement.;citation_pages=4580;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-06983-8" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=J. Yang;citation_author=M.-H. Moeinzadeh;citation_author=H. Kuhl;citation_author=J. Helmuth;citation_author=P. Xiao;citation_title=Haplotype-resolved sweet potato genome traces back its hexaploidization history.;citation_pages=696-703;citation_volume=3;citation_year=2017;citation_doi=10.1038/s41477-017-0002-z" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=N. Zhao;citation_author=X. Yu;citation_author=Q. Jie;citation_author=H. Li;citation_author=H. Li;citation_title=A genetic linkage map based on AFLP and SSR markers and mapping of QTL for dry-matter content in sweetpotato.;citation_pages=807-820;citation_volume=32;citation_year=2013;citation_doi=10.1007/s11032-013-9908-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zheng;citation_author=R. E. Voorrips;citation_author=J. Jansen;citation_author=C. A. Hackett;citation_author=J. Ho;citation_title=Probabilistic multilocus haplotype reconstruction in outcrossing tetraploids.;citation_volume=203;citation_year=2016;citation_pmid=26920758;citation_doi=10.1534/genetics.115.185579" />
<meta name="citation_reference" content="citation_title=Poliploidy and Genome Evolution;citation_title=Meiosis in Polyploid Plants;citation_pages=33-55;citation_year=2012" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="The hexaploid sweetpotato ( Ipomoea batatas (L.) Lam., 2n = 6x = 90) is an important staple food crop worldwide and plays a vital role in alleviating famine in developing countries. Due to its high ploidy level, genetic studies in sweetpotato lag behind major diploid crops significantly. We built an ultra-dense multilocus integrated genetic map and characterized the inheritance system in a sweetpotato full-sib family using our newly developed software, MAPpoly. The resulting genetic map revealed 96.5% collinearity between I. batatas and its diploid relative I. trifida . We computed the genotypic probabilities across the whole genome for all individuals in the mapping population and inferred their complete hexaploid haplotypes. We provide evidence that most of the meiotic configurations (73.3%) were resolved in bivalents, although a small portion of multivalent signatures (15.7%), among other inconclusive configurations (11.0%), were also observed. Except for low levels of preferential pairing in linkage group 2, we observed a hexasomic inheritance mechanism in all linkage groups. We propose that the hexasomic-bivalent inheritance promotes stability to the allelic transmission in sweetpotato." />
<meta name="og-title" property="og:title" content="Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/281" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The hexaploid sweetpotato ( Ipomoea batatas (L.) Lam., 2n = 6x = 90) is an important staple food crop worldwide and plays a vital role in alleviating famine in developing countries. Due to its high ploidy level, genetic studies in sweetpotato lag behind major diploid crops significantly. We built an ultra-dense multilocus integrated genetic map and characterized the inheritance system in a sweetpotato full-sib family using our newly developed software, MAPpoly. The resulting genetic map revealed 96.5% collinearity between I. batatas and its diploid relative I. trifida . We computed the genotypic probabilities across the whole genome for all individuals in the mapping population and inferred their complete hexaploid haplotypes. We provide evidence that most of the meiotic configurations (73.3%) were resolved in bivalents, although a small portion of multivalent signatures (15.7%), among other inconclusive configurations (11.0%), were also observed. Except for low levels of preferential pairing in linkage group 2, we observed a hexasomic inheritance mechanism in all linkage groups. We propose that the hexasomic-bivalent inheritance promotes stability to the allelic transmission in sweetpotato." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2s6cdx1qjAPrhkGSXu7R347QN5nYc6n2JY-oRWChROc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495261","apath":"\/ggg\/10\/1\/281.atom","pisa":"ggg;10\/1\/281","ac":{"\/ggg\/10\/1\/281.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/281.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/281"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/281"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/281":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z+zsWtUYkBGVB2vqPCKWJURsFpnWCm04AbZuQZrNcQAcFKxdrckLnSEyXR5Mq0nr+pcla3kN+VaxVjUB8GjDB7ubc4PU3FrjUdmWv+d\\\\\\\/zMDs1Xo0ozCeLPLVUFtRNr+JNidVaVJQACltBu025sHwtEmvxCNZ\\\\\\\/7a1NaZFzvHWn8GwPTXvg5o4zcRcaLHcWa57V9CrGDpafTg4Jc3zeD2clWcHSFw7iIEaDEzJOhLCNjljEKk27jG9TvcqpZbHsY3H2ZbCaIvcnpNy\\\\\\\/76qUpQ8wGCCN9hvsu\\\\\\\/K2yBN01AYQBv7eDCcpRzeMl5j3Txly3xB9FlZ4Ffnwh8OLjlUcd4k3Zw1Rvn8HVdSXiYX+a+y9n\\\\\\\/1Qp\\\\\\\/PN4nJD4RnFLSTfgvhYxKQvLyBtu0lwjk1KT7DY5DBUx8mbHJlWUSzWEhDugRe1YOGHRwf5qLqsBS2fZXMxL+0Vj+Gof19oVrSo1NZtFnYlsWcHGkzxdhlUYqcOatywu2Dw9t2+i65loQvwPLEFysP\\\\\\\/unucCfFOFlvkUTI68tsmuNMvRYLjT3Aa2qSP\\\\\\\/3O2kluAbrVtuzZWBiR42+V5WpBhKU5rmk5uxh881XTC\\\\\\\/QFn72d4YGiOcZ2bXdWawxvxcMZoZrcNBAW0ckJrrNTX3+B5KOEAizjNRCJ0rV9tlf61zysZxMM+hwD3jkDE511IHDujVmUt2orcmPstvBRkXqg8DNx9ImHL44Y5e\\\\\\\/FVVjngn07fTF\\\\\\\/QxrbEWv25ug2GB1IWOUtP61505zJbz00fCCPa0TQmHhiKUyHcLQmPPbdvEJp5CYggVY8d1WF672QzU4Dsa+FkJrcfCNnJ8Z\\\\\\\/6iS3j1lgFesNg2538L3IdMQszdEOwIO6O5hf0S5I+O6WpYZErT0ak8JahSmtCnt3XKBPyUeX2JallKknBUZqG3ZNMS7p+f77Ajc1uZUAxuQEnqfwQvpRynEJun4cxRdJy+iICnC5GqIk+wVcGHXK7GN4xjpNegZSnI3qZnD4U95LyRmbqxJ2z6ggxhkfFfP7yK4f9m+LjLx4KkcQZoP0tenN07+jNywq8oRdtvKLpjMChy2RzshVcLd+mT3Zegy9F1643J3l8c7NJ\\\\\\\/C25JsWDDFrp\\\\\\\/hDflGW\\\\\\\/lCPwxZw4P3i2dRpcPVjxLK6tVFZr6oxZG6q7NcjS2DXTaBXRopm2Af4ohWu5WZtV75\\\\\\\/tHEVrGmNlluUIZU4yTW5+Mbe6vywA23HJR6kyd\\\\\\\/33syT40HZAd7b9\\\\\\\/CMgEHJz9lBW7dNf3JzDKmZnelMgKmqcJfFdL0M\\\u0022,\\\u0022iv\\\u0022:\\\u0022QSXjehfc9R7H5DYmSSIxHg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002227f26eadc123197628e9f4d93a4859a1a004f941016bf7fefbe9fa8a61b851a3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oho4S4CpQVhwWzz+f5L6Gedg1BvUt+i4DxGaOHIb8UnSA+ZB\\\\\\\/9km\\\\\\\/k5WRbn1zNZ0bOuvs61kClvIGyVsv\\\\\\\/bpWNnGJKDC8Oz0BR5WgAid1+JNUWJTVu3y\\\\\\\/2d0SQSvZHgisz+\\\\\\\/cXqqu8bp+tJ2dNDmtXx7EhZuoDuN6T\\\\\\\/O2zabh4TSYjMuVgRwX9\\\\\\\/tGsEGijfz4MoEsyC8PEul1BzTtj0jLbIEy+ourP0sBnUA2tldEBPK4MXJdLTL9qPd\\\\\\\/9ChjfBFWZhwarh6deA94lFoP7icXP40uW27KslSBnyU+DlDsOc7WXNZGUcUku7hmnqp2v6iD+7++j8UzYZgfxOzshpU57UM0ulCrr2SnrR4Lntx\\\\\\\/2kOyXuAkge0DYssuIbGxcAHhYHNvT5Z6aYnAv41NtHVujefPWcXQhio9X1tO9gGfEm4Oh\\\\\\\/Cb3RKHiKmQfT4d7dxWz9Zn1Lj7gQIqKm5W0EfG8YyZ5fFxcHcPAFkfY6o7tiNLHSBgtvfxUkS4MMC77yt\\\\\\\/1YDwaehpBw8Q+YlBbH9uTa36Da7bccRJYB0rfxX9kmwbwCzv3P4IRtybx5lzguwwnA3lrPFNchJmKz\\\\\\\/EWscASroiwfvRBT2pSw5dAqNCBgmxwDdBC\\\\\\\/BwXjPpjtq69Gn585IHX4RyH6kH+lbVkRmU87YTD3brpJKD4IAF0XgKuv6eR\\\\\\\/N14BMvfL4nGILoYzG0ZFy81\\\\\\\/El7Jr9VVJCum2TYYd0e\\\\\\\/juXuFspxh5h4KMXafjojqO0petn4pOGLggia189sT7BIaWjRbMkRS6Ge4tquhCoPNSkpN2pBkNfo8uTlLbTuQFmM9gI9C11ENljFZHwXL\\\\\\\/qS9R5HhgJJ8eGb331ugCqbu0kB9tthacLqu0MV4odUrIwLTnYvt+gyDL93pSf0pHweyGXQmPPFIgDAF\\\\\\\/eg\\\\\\\/KGyQXFDL2usMB0ZHKrRb8D84Xr1BPQpgGKgdKFdN8GCYiYYRHanmQJKY5QMpcY5RYk8gkRby6hHgSNDzfO0jUPi7iH7XgCWRNmt4Y5URIEcLMWz6zY2OY8ChBJK6tVoHnX5oFPgfd4nGVoJBczp7R8OY2Ie++3yYS9selQFdniL\\\\\\\/02p98WClYUBzPB33lY7TVnDbSqopBhgfSQzoNrOVYDdw67+TSCQS9axIN3Agayxf1hD14qJhaKnBuI8JlHFwMbc7G6jOKdla70xq6TYMcks1NepkBjcBWEM+G3O+ePUeuQ90IYyqL62CnF6CkMGuUoO4ikiy1tj0W8NTPKPiiJoCbWLl3jnLBN88k3qpkYgrAcXqk1I+S0+9JUYITZ1j2p64X4rf3FYJJyBz9KZKw\\\\\\\/ykwFUkmZYBfvQkNrceE3MkG+FpMHnCZjyKCRVHYlXi\\\\\\\/CKV9g3ajJlV4+AK57d01rsSVB+HOBJIbd03lOLHRCMjebs2cbn6Jh3aySGjFJqB6Z\\\\\\\/i9uTjE3F4PvBLLQL426OZ1FwcEkkCavIevMwOzT57IT9E0YyvyayTtW3weGgALML83pAsAPe\\\\\\\/HzpTMflJ75Kycu+I61Et07zXRvymmVI0YYFSEiJ7CxiTLUretiCBmWrjYDZzsRV2Yv2MCgecYA4XCWHmr4\\\\\\\/hGfAg8bXk5i8zbwTngb8ItIE2arFniM\\\\\\\/JWbCYHv4crp6ltrX4r4POgeElrja9Ee7OHQXuVSGjwgATWUHFJZd3PL8t4hCV23WK51ENkV5s4zVZ5GiUNdY6T\\\\\\\/OgiWvtibboht9mdc5hPMIs7WzXW2W7ynOIQ6N1cnaQIuXriYFyc\\\\\\\/uXb0myS7P9OM3QWRgaU0tB7RNlLIxd\\\\\\\/6FKBU82MPkQAJVgKajBODLZntoUrtaYsLx6VympGmzkCAc6eRjFwil0RmEU4CgxDH01Uult0e6Ita7hMCDF8p2Au00+g\\\\\\\/O34z7H6wpyqF54E3lzSFPHl+KfzDMR9ZMWE4lyJN5N9mtlGsB1pyuDO4s41grYDEKXKKLM78XdxRn\\\\\\\/L8RLLuXIkhKrH+aRoO6Y99P7qEQ7i4Oocexw5TIKvcm3KsjN+2H8tq8yhcQZ9jYG8T+vZwBQD5h5Frmy7N00EYi4mya5Rh9\\\\\\\/w09nEtUjdUzOuD7SRw7tgikMka91Fc5yNMnUUFn4EKgXuxmQbz\\\\\\\/OB9AwjhNnadjd0cmHv8BQ8S9pIoYhgyTX3FOoJDfIrEzvDyrZ5BRq4X798W61e+Kp\\\\\\\/Gsg\\\\\\\/y+X+bdGEX\\\\\\\/yFQzPqRavu\\\\\\\/8RTNBoyggiHf1FT6StaYraRlEYwClRYkPT3ih1K3+a3MB46O\\\\\\\/GUaSqUDV5hffdhs\\\\\\\/ZRjLkr0peJj0sRCZ34FA8G5huDJaoEZZXnS4eL3CRsOpD67\\\\\\\/2jr6Vze5D4iQMcifGUX0QKq+SCASrVAiBe8RZCOuoGt\\\\\\\/8mipkfRPG\\\\\\\/hWm7kYefjW9uBGy\\\\\\\/3\\\\\\\/ruisgGTGU\\\\\\\/96mEtBL09F56SHaUzInYSIbOpwhNehfaLj4f8PjrpCV4y+DgmyAxlyGuC+uOti2hDFpaXMukCBSbVFBhPB9L+fAECP1G8w0ZHuQ8X8Z2YdPG1SbBjYPoKfoE526aNGGlJpHvHCO4gp3RQ\\\\\\\/K04shYIUxbTmbLv\\\\\\\/7NI8FBHVdueWSt9tSLqCyVPlGlQGrvqy5PcVEUI8y+dXGev8Ha2PBOCVUw4TbNKptV\\\\\\\/nGi4xKZADYnCBSKVZE0VhaZ29upN+OIZqu2fQvNqlV0\\\\\\\/IFslr5er2biIn+b7\\\\\\\/tNzaPqTsSNsxMPVkkR9o26jxVLZ2lty9QuiFKW3JmoYXhK6OLSW5\\\\\\\/759xSI1DFwZKlpZCQBGdZguFV+tOW7xRsYQy5MdH5Jf1UXMZFbGuUwLmWqVCBo8HHnJT\\\\\\\/OdoxO7rrpQ32\\\\\\\/+qQZ6c33ZPEs5h6gfYqyVr0gjAXGqQDzaEgkpHGwKCahP0XX0BhR1r0WpkxTigtZWz+I41szTvPEOxz3uMsh+WuGdrMw6LePVQKL09C9PWlafehtCKl9RyrQamtLx5s77Hp5P6lE5iz03QJ6v7ZD6IFFDWFEcGMCkjF6xpoayww1CsZRX03eYD96jc0OdAlld68aixuy7irigv43D8sXTz6xr0\\\\\\\/mGAtGM+NtYUBHTCWRXSk2C32woYqICdLxH\\\\\\\/9CQ1gSxVOs72o4ycEsXlhu9FNROqdCYw5FyA6AGSJneGclaXPhedjAvzqvMJCzzFwcJoV1UiLpsgy6gP\\\\\\\/ovXE9CA2DKfVKUUGIGEKltv0Arcr3k9T5X++2Hg7N7f6l0XzqgfBRDm36ArkR\\\\\\\/iETv5kMakSGWDIqNylz9borUgrs5iVb1KaMUMXB2JG6lWJkex4bniIWfsPS5Rk2pi57IBvWHdormp2eKLA5jRGIdgXQ\\\\\\\/WX9DlQX5r5KW0dYPDY4RnME1LR95WsgrQ4DbT5qkIuA0XBk5DBPXdT7gAee1opNF1Jd1IZzJFc4BQmqgqVuxEHuOAXajEsZIK14r+6qnEW5cIOLN5DRPZtL3NWhV2AGc9gBhbHWeSb+kSzHWEsihGOjh9RsNoZLo5BvLOd66dpwMKrdht6QI1w0PKxMUdFGP4cO9a+pVEUonTam46z2mG0Jm+1nuYYI97cAyA2Qr72mZa5wtvrQm+PpIJllx8xdHtxksjIsDQPXQTVKNlZhbCS9giz6Ih8fViBQ6fH\\\\\\\/cEV\\\\\\\/Dcbm4p7nKLdSLhRbzfW4qUMFoFGHDIXsceIyY8OecvNJgU++OECIDdoASWhOM\\\\\\\/HqeMEcHKHzrlOI7fvPmm6uOWtsfhua+\\\\\\\/IbxKZZO28QbUJekJhmEng6Jm4EvpvH7HcHoVoAK1Ok4nH\\\\\\\/2EUpCZ8FNoTv19RTwJmNKmcjgnhGhDvwMcbYPrtWQfJ3Mw\\\\\\\/NRsKukkdmhtdKZ26g5t2dgOTMroo3RdXej4n6Y1eLa0ieklLqT64KilL4wtuwTlA\\\\\\\/eGhERH7\\\\\\\/rDyzoMRSn+a\\\\\\\/XCmTjVmHcunczQcLMR5pSRQ6dhZDReVgivazNWhhX9DfUciD5K+8nwF8ORbGPZPUtgiQe9i2gUk9\\\\\\\/8PHoJl66rfVpkAqDWc5g+r3X5CEaMOBw1ADF2DOLcEA\\\\\\\/uk\\\\\\\/JUO\\\\\\\/i3WkuAVgyTUCvFiveJzjKbqNSgC2EwhlBRNjUVDWIDlIFJKtheCYQV\\\\\\\/JEYiSClrCqaWpQ33Lg23lGVCljfl\\\\\\\/2L3L+63EEBpPm3COWRB6L21ur08V9l6EvGJ0kWSWZZpIok7yw0VCFIE7xkmoktBag8Y\\\\\\\/B5s5icgcDtSK2YDM4P3Sba+F8jfmACbLMNW5Y9T5eFFSszOYmLhZ8Eb7TU6bE9BsAJyWL8ilZx01rxyB7w9blj3GphScsnVAugKMhKGGjnOi82g\\\\\\\/xxyhdB+1WXe\\\\\\\/2\\\\\\\/08qUZjGu+Bbqam++d6a91cx8L25zfwqeX8xfAtrCZcPXjg\\\\\\\/wVFn\\\\\\\/ldwluuvD2eKPQaYNZHw\\\\\\\/okMFCUTbrfrQa7JhX0Xf\\\\\\\/tMiziFPlfJtyl48CRzZpbXvo0T338XDzbLWk91ZuovrXqykLV6StGWxovti9+9+RoFIizuXywooW4acJoLQ89E1BLiYusG\\\\\\\/VzSDfdA3kbquJC\\\\\\\/wRcPEOqpKiBH\\\\\\\/xiAETQ9Qc7DmPIwUDzK8JF9VxgHNtKEK8jvJT9mpC+FWY6pb3bZAc\\\\\\\/0SBh6utNUuzevlQyxXyMUf06sQXHU9g4KDNG+yOPTwP36n7Q\\\\\\\/E\\\\\\\/aMhpDKNj2GauVrGWFpud3+14yCi0ixwxHQbhUy6uZdNXKwunm9qjJvyciOHnp6wPkYzgWItd4tZeMyvAUoUfA3HodDpFkVwvgCcDaUqLqRt4y4LpmHn\\\\\\\/z4Dud\\\\\\\/tkGC6c1QX3UrozI\\\\\\\/3hohEFFAzVmhTAXk1qLYaUCoqHw+syk\\\\\\\/CT7lNAIRrOwMyD304M7UIMQwOeUGob35VgnzEDISCuCB8ETB7PX6sXAMOmkrOIRFgLhoUwF4ibF4+NFxSFjNpb7ZBJFOa5VivdF\\\\\\\/Y\\\\\\\/RUDHnuk0NZYl25gm+rT5DZm7\\\\\\\/H+Zinv5wHgZt\\\\\\\/oZftHHk1bykk1WRjpn3TRmIDBp5yckXJGoIWKvX\\\\\\\/UmuvnKJ\\\\\\\/XetOxEjRWNoCGd04aG7CbBFigonRbf2cj9i9B2ogxRd1aEVaINKqkdaXQSPrvZ0K4ITf2tXxk+NZNwoC5U\\\\\\\/Tn5fjSe3hGpE1wUHbEUaTrYr9vSbZrV6MvwVWgp25p1e9k3xgv+ympILIQg0\\\\\\\/3lNgw93Fhp6CYgICRJ\\\\\\\/YhWYfUtTVjOPjLmoDz37NbNSvIHe9ATGaE96tSphCEnsPG7T60F0Rt2C4Pvv69lBIcFQ26fZTmPoy10akoGQwtoz64rHRNojFQqEtYBD\\\\\\\/ib7DzrLVNRqvflBEt9lKci1T2mH00c+wp5aa49\\\\\\\/dZ\\\\\\\/eD+w0twdIL4YcctVSDbG4kBRNYUxXVAnUDI5\\\\\\\/S13NdHIQP2lhIJteCdcNGYe\\\\\\\/HQuZjOTHIu0xrOUJ1Dmvhk5kKYRHQPOjOti8cBNE6q4Q0a+jwBbxaudslx+J1Uum\\\\\\\/sHjGFewTLX\\\\\\\/ntOqmpGNJhlxnH2ePJxsAH8wThBMOB63Fp4nciHmndNjJZU\\\\\\\/JBEk\\\\\\\/xqtAmRcn30ktWTaDINqcugq8JXMbPRvTkb7LA+A6CYqFZzmG0+Q03Dzt52j9GhcP2btnYhe3F25GGhZ9OLbuvJITyqQ51wctL+Nj2IW14Gx7UlKJ\\\\\\\/yikmupNa008DJphNosMZcFlqZqFf0X1sYmrawE4GUeK6D7nwGVFoutFcQV6JnmgcFZvs9fkNX5Cv8Feb5J7lc8YwxJMplQwRGZZveh1ivdLrpbq4fVAdK6Bwy9l80+2hWKSoDDFuWht2wRZ2mwL2rftMLodbV4PIFRVtWeWGIMGa+1waitYN7Jwk\\\\\\\/vYZl6t4Pd64fQCPL9zkxyyvk7lbEcDDm45gh9tfTn20iec4U1qfdmX3u2i6OHurtWaTqmxjrheXVzijS7uv1TlMLnoYSWAu7jf++k7ehtFxzLu8ezIHRQKQqW0jIlVUPijkrJsdY2Yw6HZ2dihNQ28DAx\\\\\\\/pUc+y8jrjFrHERS+yK\\\\\\\/K+GENJ3gIGG7sxVaRU5utufYyvI0S19kk5TY+11T+bAemmKNa0rwERK220CF7KC4uggyymp1cfULmt\\\\\\\/UM0ASzwOsLL7t0VMSlJkcXMCLy9Bsy1gYfgAnr8daCVO5Hm90wGlkZ03qB2y6T7fYuEqXDrn8L80nC7NYhp6wDjadduke9KbCm9+zM2c23I1C+nHIM7m0Sh8vYsVV+3HHTadLBQ+MFpn1gHIQr9TqPfv\\\\\\\/jlMWj+IuxrWCQaC4l4JND8bt4o1j1euRwiRR\\\\\\\/VDmq98+qDNYLJFzRumBaOGYJTHRGedC2PBsXClt1gGik7cy1Pdv3F4lr2H0Tl1kOOSigxH7G26aam2rg8MSQoN74UkjI5RrNSIgyZOsLb7Wtnjml\\\\\\\/neWzw2cP5m3xpVtuh7lZnMIawnKW2ynmE8xEUE6uHhUTDEr3kObF0HHpkxEqIz4iNHbJwGS1Cr0xNNfG1MopxFsQrDX6v\\\\\\\/CBDdR6SZQhCrad3tPgC0JD+oyy0eoObWBFbcE4j0hxv55IAIlCuV\\\\\\\/CMDGBYuu5jN5rIoBJWFI\\\\\\\/1iOnjkiuAjXCy\\\\\\\/XRk8eHDu\\\\\\\/ODhc8yhVEVohZ7wHHDgc9Q\\\\\\\/5LfCfZGS5svI54u645z9CzyBv9rx1Azw7DYfLFfVwLJS4NhINqvJkcN1af\\\\\\\/kalwVB91GS9mtPlR+hM4ZanbqpUHLPI08BWtFNTS+IETBJJ24MTwRKk4l0q+0M8udl7HbtG4ZwIEO0EVV1In82HIDfKEzwPXgTuUSK9rRISDPK4GgHaZOxSKzUD9HvM64Daz+1AfRl7Fva2c0IWQ2T7GnsyQwNuCKXVh+nEYiWhbSNm2kXfss7+ZNnOydLp8x4RgNGC0621j5GKwJlG7z+qN3+0LSgyE4sTcDmVslYIYo6bOOCwHwEbJnRY8FP8DPYWWqQj\\\\\\\/9oby9GCtbFgxcji8z3y0VPZFqK8jDSEv4TJclgEc2Bq9Nm5MwEXQE5dxr9HQg+c3YgjtowDorFEbxUcCfmRCMS3DM0oHSFaasSiUeW8fbFBSpasj\\\\\\\/juApex8frFzJ77aUEcRxEafcQto2rbhtQ+ctYkGjf7pw96fbkX7WNF5qCsd\\\\\\\/n4E+uTqhlls66+tRd2SYstLkKHAOojtV2ItzOE5BGmfYKUU9Y9EBibO\\\\\\\/hA1ovooicFhQ7jEO6yqdyGE7F4BKyWoD\\\\\\\/e2OEW+n2HFk2TbZP5C4Kci\\\\\\\/aw5sxJWsiO2cKmGOFjVKbkW7WvgzCfEHtSM8+J+XjCxg9RDx9V547Q6gxjDuZtvUEG7HKY2HXUo9UFPejesDxQjLhpC1zhCsHCMp+Al190TJPX7S6PvPt4JvfmmQjM0SZFpVl7a7lhLFEuXGEyCUxJpiRAvXbUUX51+aRHrWcfVhYpG5hwdmKzG2u35a7S5Fnws2V4QQM3CnMp6Slif0E1cKjSK+ZYzOzcN92SYflw3Dk0DE+S30CPfWq1egIQ+1kJYeI933yujH6hNmKoccO4eBzxOgo9ZbyDm3A\\\\\\\/TTLHKbrtYheev2iIeNFmxUBRD9GyLscREKr2N876KnWM0Upf\\\\\\\/u23wC8n5tIrH9kYTifcMo7PdFcZAw4Saq6ddx1UgIEKkLw7kfbgOpFktTjVrJQkCROedBP7yhvlbqqGKwplhUy+\\\\\\\/9R8OJZUs1S3VbYJ087wgvjJz8VGoI4lj9n+X08XxXNeGX3By0slC1Q4hQ0AS9DfCItfBJ1rWzkZGsAYP4zRhaf96580M26WmHTkEkhMB9Va8MXG2CeT2hcD\\\\\\\/SVi9LiUHyrJ7RbTVJq1D42ERA8ODO\\\\\\\/fECmCAW+Yptk5IOTouC08AwMuRyjCcOXCKsRWFTgxRrXihWGFec4Wcfd053Qiw8XjZQiVzrGgwawrsf+Nfuu+N3cT\\\\\\\/hoC3o7r\\\\\\\/jHukBWkUqq7DoWzm5LwjIwlS6xOlzY2N0AXWMb9IYJ5NNn+uRaT1ZAhzDFlrgagPmjP20KtVS+bV1fM65WQdo3VBMgsEBR6yUQmmtdvEOFRAW3+IwmzwRscvQg3bmk2ukItzXL7a4XfEOvug+MjedskUs602oUCOfpqdKq8t1qKqxzkk9K7p9ipbcFwoZ0mi68xdh1IBukG5Jfkc6qL+debiU4XBoKBwq+RY5AwcYOOX9t+jnP4qUCJVKFytdNbE4QpQnkQ19IfS7MGnGt7jmYAlaFYi6r1Fw9LeycxX\\\\\\\/XQ9Kr8+uzS+ZUygoDgKl3qqK3aGDGGLT7TQfeW3QNk6XiHG5rZk06r7e2aVUtMx6pBXiAOZap5BVoRnnStkPPkqthg\\\\\\\/KxgxRzWK4XrbVrG0AqRkPkVTWUa4R6cPrR9ea8SQMC0vuf1R65wZPuFXUtI8cfP8edV1kCpGkDmo4reC\\\\\\\/X72nXJAw2Rmh2wDFQ0ERQowg4IJiGS1QH5j8Ho\\\\\\\/BUD3wH0RQs2+fXqgX9JypX+AghQ2H5yE7r9cJ1V3XJpdl1KpWfrZr49spmUxJEbXsYXuoxKcfC9kFZIiMoenx5ppFD0erlAbpFUgvP48zAhq0gMGj\\\\\\\/Efl7pd+LRj4V5LqGBqLEu\\\\\\\/4Pd0eVHhqn5B3d3+j9O76eWYTa1+UUdpggYk4V+7pV+uAoyv5fvTuED5lXwqkM5mmXRPRus3iDJNQoSQua\\\\\\\/Xub4YQzGoQxzQaMA\\\\\\\/N6e\\\\\\\/5g5jLcciLdpRRcR7DrIjmIBSvABmvWizsvxiAIbhgjd6gBy6EcfEE3h8ISKd9bnIkm5FqeCTBUJ\\\\\\\/G8XEgMxHK25HMqg4HfF9xBCsvAJ\\\\\\\/PLLl44CpFscmMjc5O5cxgnRFnSKOmvEYrlYtWLyXRxG5vU0DSklGawIWvl7aOv7YiYLnHH7+wcIGiFqjxVgKa8M6Rbw+Sr85eiwJJpnWUQOPXDErdQIfROmmIVT\\\\\\\/rJod3KA+7YD6K+bkZqTIPipPS\\\\\\\/9mVcYlzTEyvy+PYOgPeY2NfIvxCgKODMsPv6xTRf+DD4xalOAUgnmW7nOC\\\\\\\/bEGQxNrXpBLU5\\\\\\\/kQaAU3MfefHErHk\\\\\\\/ZGxXuVBRRsw0SFhd+\\\\\\\/hqY7+N+RXsdwoq1OdIMibHY0mqS9omEKwzOPSWluOpZmWRjY5LzHk9xnRA4xU1+0YkspO+\\\\\\\/nEsuTst1k8LF8kEh7X83QKCEnqhi25DB+eOu+YigIBhjeRoG\\\\\\\/ETM9bFhjmjNKZ\\\\\\\/OYJvEwq\\\\\\\/WizGVJqjs0macmOKYUG+2fkKVBw8hvam2Ib8qiiDrlA5kxXAOW1W2JZ5RjpsBW5HSTIDKHoVwtZN\\\\\\\/bIY++XY8Laa+z\\\\\\\/i6tjtX7lgn39k5jElcXi7TplKJJ0IXc3XklIKJG3PXM7dpKqNSW7nV4p6GoucJ8zpkwAhYbmEC0HjT6KkSM0O9e9QwsdQc5Fc3JLzyfVnvwfbMUij\\\\\\\/ACRKoMwRUp7DQ3tzzmEL+ICBrJEcpbDdU2dTyD33weWoPc59KVdd+dQCrOxhl\\\\\\\/3Y0DCO4mUIzhT+UJK\\\\\\\/kn99alO+C6Zu07efRqBfWv4IFtU+0EcViHp+iBzGcRK0mVK\\\\\\\/xEjsV+OtTnPS9yXkYjsWQi+Fe4UnLvLNyZlPVXGxtYjATZmX5tKE3TNkJfrrulMMWFEYxzrrfXMQlUAwuUpBApjH8QwsQX5o\\\\\\\/hrATG854tlu82gJUJBz6y5G0wTn4q2V\\\\\\\/AjPBCuXL6OqyA4HB0W2284Nh7KjlCWXtYIRaUqySkgoYcPasoLXGXUYw\\\\\\\/xf4D5WsGduyNoQjq1hMFWsTdQnABSulUlK9FFvklJVT7Eny7qReVBaKHx5wEt60+Caq+IMb45Cz4T0Ue9dz34ezA6uR7mh6WsqrGyz2\\\\\\\/2Tdv5799QaKGAzoLkx0B\\\\\\\/pU9iVWszsb1VEYflccIvTCeDS0DTRBOyVnhQyQZxpXlzWw44dxoqGEjmt713TTurK97I\\\\\\\/kcH4DyO0GK2N1Sbf2buEHOP+0xSpmVu0KWCqeiiMYeRv+xLN4bnO6iGZ4yxz8+o+E9IzjSILEcfu3alUMJDIrwxmxAHtm0rKZ2yHlZeoh3Z1CuBfbTl8MxCkiIX7GwuFFuyEWnOT0adjwcnJYtVpqJsr9wQdnSPsLPTBNbJCGJdCjrXxR0lI46aomPxlIMJPKVMyA7f+D59T4sjudPygTLkCm7jRmbF5KjWQkIv9av9sbVfrO+CFT9P+WYwQ6SnUyM16Xr2InUKxxHn1MTRu0xB7Q2CTSL+tN2B3xAgIcq9zOibERzmcJVQP0meGqRSMom6NIwa89TOhn+orFRTMIaqeHZYK4\\\\\\\/Dud77WmVE6BmqW7AuhRPI20d8K1+IM3\\\\\\\/CXz+e\\\\\\\/6W3+Z7eCyEvxiCGEcxbwwL+dsfSoHZRPwEPYX8GZpyy+NMcOtEadkOY8aZ9bE\\\\\\\/Fn57dEOzHHjXQZLU1WVXBFfADOHQ9FU9RbxwBB5LdRj3\\\\\\\/eX44gYhhiL8UxV5FH6wxRjzeIiUD40mHacTu+\\\\\\\/ArK1vdhy6bLLud8uxDtFejb1cvc3kqbYsr6vhbLsaNPusBnAMMgnniCftLaaEYXcEmRTRzAfgp6zQa6gUCwxJ2bOPxCnFicUL8muDlSPe9fBa+Lzq5mjTvCQMeSI6WvtxpFuoBWvQGVOVfH50AjFTrZkjSOr6klxqxPtWYuRDlOagvEMqJYOOODwG3ENYKDZFHMKk17\\\\\\\/dc6Y0\\\\\\\/DWSY2pCombNbr5yEcKHxD786xwiHxF6NjgHEcPaJrGVjnniZ5UFzcOo9Y6zPCtBhuUTC61X0zzrQvuwy2H5maivAEDn4311rLGY0VxcFijjlnVefAr45fz8v2zFB6g0BXIA9wCvwGTU58xdQKRh\\\\\\\/vDMEZsVYkMDWqHrXqgbfSWiRSuH4p1q0Dex7Uyd38Ch15fv1g4Xjt2luBpSjCgKYmrG8qX5AWqLAYFRP0wVDHQ911gNBPmaCtvVbU+nRyB8b5XzCD8c5zgRGpsjzuZAlEWXmu2sH3inxb8qUhXeOTR3NAl3Xe+HbvUbrfnXcWJQTZFq+Ff7XrsQvDn2EW7G3nqQdwdH91PXSPXKWEO+2FVqvIHx9ccSzAZg8JMO6objK8GvNlAEWVCbs2UkiAVJ8K\\\\\\\/ueZdK9jGFXuNNnsA9qnH5bhwH6tYPaVVabZEgLjXDfc5jr\\\\\\\/F1YxjJTMlFP6aVsG8tRblbzn6zJlHesMuZQkqOxzzuNZgoqng+RJBZzkEZ7MMFdO0+Q2TGIW\\\\\\\/V26+lbhgKE36eOF0h15KcvLDJpm0wf4GJO2q4EtlFxfgD498Mj\\\\\\\/sM9aihzuWjljBeZKBXz5EGycvq1gZZKsc8y4OmGAQRRpZNlC8xu7X6iaO5TxoqaLhqMC2pNJQNOhx2ZKHNcAgn+PrmRxW9iqx7ivJ+O7L2E+FTAbQ3hZ50DkAPcMKi1FEAXitHdEtARoAHR19vKNFBP2+psdoyN1PwqSuYQpxH5gAcJwxaY9yNDXvN1bF5w1vbYTGfX9UqNNxy1fdgif0m4lv5u6\\\\\\\/j32++A4+FrLDHtjqUMZ7BSPXLaAcwnby8MlRvu9LJuuRm9lfLzxfqOIaCi7ce3uLhJezAt6o+gKp0ycf1+SSi3auUz4eXSQ7Fx1uGotFVgbmCkZ2N6jC8iWx3WeCsbFR3ufI2tSSte76LZPqY7aSCnMyhPKIYDC1TDDqoWmHDUV4Mt8gtf7T\\\\\\\/Ml6YIUDbDt86+jYUQBdE\\\\\\\/4ARW\\\\\\\/RvHEt8z1DZ08Ua+ZwxsrfCw7HqXclp+CPE5GMGORq9BTMtb4ukyw6Xi746SfvuzdZa\\\\\\\/DKwxrSmeyH0jV\\\\\\\/UT1d9acPNY50g5De+ETxcavmyjGHdF1FM+LXNMNSj+nl+mi573w8X9Lunpo8+\\\\\\\/8pMNjauYUMHodXjy0J3GdUnMyAWlXkQeEQkr46O3hvqYkpt2h3A33SpvRNhUxbUVx5isO4KB8FsOuFSkKP+685TYyQaglHbDst9pwxN+H0+TF+g8GmZvB5xxLiHBDJ3wwpiVOPnKFMztIpiSU3yXWgCdr5lK55eg7UBEmCUdsLVZ87od+mkKKkiU6gSp79Ic79jkCsrLDX0MwdVzc4vING+CD7e3NxWeYyxVmPJMKoDy0wae27Y8MUsDfIwhRaUDxLCf+yMNi0L++cYw1UKDEiFpgaCLpYkdMeFoO\\\\\\\/9NIPGHnSRuAbWHy8DRuqG1sgbWzqsheS2wTkrX2jl0WtTdEYwuHViK3q4IrOHZJtiq1K4\\\\\\\/KsmclbK4YzR+REX+9ZAnTx3cPYMxQ4d4Ya5fWcNy5XZhEeVABF1nXPI9O4yYdeTqHbKKbNpcmzf46+FbMr6OPGYxEd450qgH0rC6DTcIWQP89L0Z7+LlISG0rY1nUPmJBJB+\\\\\\\/Q+KZ0PTrULX0O7RbGTkoMvNcTaDmsjDlYHS9Yi+Ul2k9q1rEE7eMcASwcHCZ\\\\\\\/8EkhkZwxaAnZGqxq+ixCD5Gybg9l4Bv8YW8x6OLAkAXsyY91\\\\\\\/EV+hvlJDRy\\\\\\\/XTJjXjsHXNJHRGwD3e3HWYp4QGNdJJX+8wRUaU5115ej5\\\\\\\/GEi4MeeKhTgswBKR+9Gjva3JE9mCqJPUIWixZhakHGjo0E6Lke9m7uGKS1djpahhzKAien\\\\\\\/wJOowIGyKIqIjRjPMNWHByLGZh6IPiizidHe+1sBYtSdsLM09THVl2FG5VCFCBwD1Z4gE3IMqhf83dmV4eoSl06jbShrr8JLV5\\\\\\\/U5z+yQZqrvZcIgyVoCXGT46MQ7ikANgSY\\\\\\\/k2CRu4ENCcmyz1uLA3Tdm7GKlJBjybO6EoVgB112lTPOw0fyP9stLV2KhoImMpELfADi3U1XlcWofy79M9NFgkmDecvUXW4sE4xG6XJh+fHgCbE9XMJ5W91c64yETXJFIRkqyUMe\\\\\\\/1GDYuq0peNeqkEINCE7Kkl6DkUWXyEsQzesCyjv0bo5tsqGZxtceUWsf8yCDRmxCk1BqBjg8bTx5gAHgB1K9n\\\\\\\/asTnXOVmUcB+DAHbevOotKKDFe54N78XmcZwFE9LSY9wXaJGKivcRKzrNRybY60pKs3V\\\\\\\/JHb7T+eA5mYUqg\\\\\\\/5gFvwM3uxHcNx2VBRnjs0jt0TSOkkKdpoYrH\\\\\\\/TXJFOiWuC83J74c73JUi0dnqOFHE24efKgFr8NmojaHpdUsDzvk4AA90NcAjS9ERnyluPLQ3JqaMdKvXFXALfupk4T5pW1OyW2ElvysCXqQ++XTBR+hqh651zMEAPk2gS+5S90fj9yMWEKuVbP0v7mr05iaUZ4kNWULChsUl257B8TZ06lPwTQaJR8RGmvF\\\\\\\/m6BF0iSIKbuI6AZzXvC7SkkN58NGAv7thT\\\\\\\/OhRJx+3KIGx6\\\\\\\/4GYdyZQ3HHdffgHmepKfLP\\\\\\\/K884OL2fzrGGhptd+tSe2M4dPUxaFbai7KXgF7JHWPOHIeNZm2boiBWLK5f4s7Fjt3iOEQvOAfR8Jr8v0ZBMovo2tbc3n0zQNJbMECN2NlsCz7XQCK7t5n6pzEy3ps8jA2TlyQBGgL4+jgCvqd1QhKLCrRyQWZEOVfTXm2WwDYj6Wa8\\\\\\\/rw9r3wA5xIyCiseXdp0ddA48e3gAjbOPvwxc5yQrmOY3Jy0LQMCDNtI3jC0i4wDIKL\\\\\\\/IXH+rIRjNZWzxnuC2sW9OpWD\\\\\\\/YZ+fqgsLc9rkbeXhLE54HKVLg+GoSpK24i7jnAqnrwFb60ooFWlOrht7puyU1JXEfo6wSJMQKgndb\\\\\\\/9sXkieJVCAR+ykXdojGaZ4XG4u6GWlCMJc6bCu4+kx0bXoqtvv6MbVwb6KOnpeEtcGOVDcqDXI9MSBNfaAG3Pg+ueqIA8V8kmvF5HSwDDBWBynMj5wwMIz+YAwd+98v1iMCklJ1p0a9ZUF9V4bNeCbAnwJiPPaNNkcks66SwFbuCLB7Q\\\\\\\/ZWvBbrKN86QcNmLlTItyFmm1Taq77QSb1pM7sb1TCqWqE0SDajS5zKoImSfPPlPr6ppoNSakZOwV3CgZxX7VbsOL++NpvV+PjTvYrSZBQJEsQdowcJnidW5jeRBVfO8o9R6LM5GWJy3wHcC68bHZ2CCvZvijXd5Rtzf1KYomcQthMM8c93tYNO24ogrirINr2xVmsZ2fhfH0ZSrqG5ZRaUvxwdJznIYkwmsDo3AHhVK6WqGqLAwiqOaRdkJUh8u5U\\\\\\\/e4roeL7ir25sDCDM15+FNij151pfLZuea4wZDQH1e0PE2AB4uGSH9Zug1jV\\\\\\\/Uq9pkgzAB9gXeaWhqGXybCfLZAH8dPSxjkH7nZH5qI7WikipW5ujSiLQ\\\\\\\/67JN\\\\\\\/LuEo8q+0TBUbR87V9JJ3nHOGRJphzjq1HVS7u61Ozg3nhuWB2Q1GSqZ3+oGP8\\\\\\\/QGwSBhpuUgCaQPxGAiRyF25vmW89a1qmY8I6ogNFV+86WwEu6aI2QQRiZEoNXA\\\\\\\/L25wwDFRn4jwo11mjUXJcMaMs8IwxwA6fpfyLYWCspvSjnorExSBfKOh5XEODR5u6Mp78jMFE3uPG8BPWrxAGCJIRvMnqpZJFfvq20yiFHGB2PGINvNs8d3338w3DG5pLnYXG7CvGrlIiKcBl3D7X9QQ7VGPIGq+Ini4qwpbWXQp0tSC2WktDGYVuQ4leaHQJu\\\\\\\/8LC0RA2rlyR43bLl01nncanpHo1le1fE5gzEYh0004lr5RyhXregKujAa+6874AItE76qkqjBpsVqtSnBXSR\\\\\\\/PPJ0jna5lN\\\\\\\/kJV064BN\\\\\\\/kujOuhtjqbilX9IWvsWMmm9bggcX5M5Rz6yH1Ge9Rid40mPjyD0zLow5SjgFak9du6nGtTSV88o\\\\\\\/DcZpwy7jhWTQAQPpmSW4j8nuhpb8eHuBOa7ujlXTPnoXSQSlXgBB7lQgWd+NFkHbLbkErxNyP38GfS5CeETF93NKbOP0WqGXob\\\\\\\/3dgjRSgrO3Iqlps1oZPNI4Ne7KjrpvTwlkMb5cvTaisrGX+CkxF3bV9wrvtKvslOIpu0fTFZkzM8zw5tBiou3KHgfpRxm+ssoWTeK0mLvh8MGM3B1K0lwFbuhnYvKDKj09SI6BCqI4\\\\\\\/iw2hiKxHe42qoxQ+eeN9i0mFCVxrIovXo0\\\\\\\/fAOyUfI4BiQQOygRw+V6MTXY0Jsb31vBy4np9ETsO1g1QdOWnx9amax+HfzgsoNJyRkYSIouOT1cwCWBtApOusPYZshxwr7kffIu6YqQVzjQVIOX1FS4iQGJM\\\\\\\/rlOk3dR9sKhKjZi7mLlLsCqU6IpBp1hJSXOR4dkvGhv8FNKmmAK6HX0D8f84xz9xOHO49XGzv6RGxe2gdbSpwsCaXac7ytBBLpCTBPYUJuHmsW8ayLHdhpSKmHcONmoRVlkvaOMvfwK2qUOLvJqVuNHSndaRnAucLOmRao7pFLvM\\\\\\\/c29og3HWVar9F0VOLH6LwZEST+N9bApcn1pzI2V5BYqOE4IQbUyl9RRlWkcDjOHxipTAnSprWIWf\\\\\\\/HY\\\\\\\/Cpc\\\\\\\/F0C8fV1FKU81oPC5Ct2jnYxNAikk\\\\\\\/aSxgVnWa0NzRiRHe\\\\\\\/i\\\\\\\/ZcE0R9FazsDk6QogNGRRYpIOelAxJhDypJh3D6XAwAyL4DvkRu7pSJlq+bpUV\\\\\\\/nVMAH+O9J5bjfihG9MlZMqxYc763UV8zA3nSGLS0tEFiO8\\\\\\\/kDUREaWGPi\\\\\\\/9qp4iDOfVYJJr9cgEczfL7Vj7eUQDU+Rs2gfb8t88C4JObch7vML+q1M\\\\\\\/QZU9Mji03Yx6mELTFF95OOnDbA264XpwInGwjSkgwZq6pt07TKr2aYtxBAcC86y6INp0Kq+\\\\\\\/gWiKyxxUl1c7JTPJ\\\\\\\/E5+l6bmEaAgv39DWqEZ7MtAF8apyCYZ3N3IcsPf3IKP0rHl5LUieXzHm3Ra5BcD28NRCRp5POhaxJxpIqQRor8GasqcNZQExEbXnOcBC9mjxjUGlbH5ZeBgrcBLRN1PWcPr1ywDd8r17po\\\\\\\/p7WgrRWuY6pcb2uN5Cbnkxakvk9Wcp3kgc8elhAauP\\\\\\\/Qc6o5imIBE9BvJABHih1mynj4ncxUS3k\\\\\\\/1lqjupCWbP3GceiaFstk3mznPiTPrlkAho6720Gy33efk9tUNjdNbSyMbBnd\\\\\\\/I6QMjMWeGQ1kK52yhxdBwxXq17auR8bieF5cW8cA7MLLlnbugJLDsNq7WoqTWZoalDgavqCPeEYeQtNT5xavQvyuIySbsT7R0+oT84qKWXUBaUzn5enRfUcgH8\\\\\\\/NLs5gCpxK0lFLmCAIe7fLWBzrKAZD9S4cA6w6Q5tm9DMMaQJGz0aKGjNnfDdWf3RYfQ9Zz4rIDoOz9Rbf9wSQPP3y36GblHJRfpFWdlGRerSkZ8P8TNnCRDnC\\\\\\\/DL8skxYYUA7M7Y2y+9VuPiRxKk1uSnsWmmFQLaNigNsdvt4EjpzsQfjg7YZePhceSUh4\\\\\\\/3GQrtR3a1\\\\\\\/xwj\\\\\\\/DvcPVGfXVtt5oeOyua5F+Ai\\\\\\\/DtnN6jrNtaIU0BWDgrR30sy0W2Nbo1eJpqu+Cbg8+i+QIMTqRuXTZZDIY9FD\\\\\\\/GuWl+CzSfvXyQH2Ba41\\\\\\\/7A57aTPVyZTOXqCCEebgBqtIwk\\\\\\\/8w0vtTIb45ytx+MPKxVyYo2bVzYFOFlA0Zht8WD2+NSn6vsxShgjo6PrJoOsEOv2n4Zkoy59q4igY8tYgf9c1hHlacBSlAx9YrXf4AmoyvluMDVqMiYN2P5lQTuCnRRf2e8RZz0+\\\\\\\/+\\\\\\\/Cnyhen4DnKxlvVCvnp1H\\\\\\\/gTGiYojx18nrcnh3VnZvprFUyr76VzRySPeoam59W5mm92aAgdR8LBgmipFfl\\\\\\\/Dc6SB5kH1nC6bvLZgDw3JR3jVDKWkH6wAJ6TV9xvUuRB6CKF6b3q2D4wp6Umyf9ZnOA6rY15KAPiq3nTpWRVtYtKhnrNxw98m46HI8uNPcoiwZ4Ix3rivQ\\\\\\\/7iNObP7bLVOTBcSFppsd2OONlD58vyEsD33FXormVMB7w7ZhtzZVxTftbsidh78\\\\\\\/7kSKtGwor+\\\\\\\/HFQh9mrTKp8NS14OkZdQNRIQxH3Iw3qjInE5pAYS8IbSHneRU3DNFSdMtHjEPuxmmh2ho+0nDcnQDaS4ub6ZjtU+Y155remP5V0nb9+HsaPEYwj5ffc69C+uS09CdiqXP5SZ5pyLnz1d\\\\\\\/ZvBwyGJFS\\\\\\\/MAqhqRq3ED0L59G5tamwT+jh4v2NOnNyDX++IHWVj6\\\\\\\/g4tGfD\\\\\\\/TnL8p+mCf8FhgEkOJc72r9YPHdib3zU7n9j9\\\\\\\/4fAjWJCfpb8QCO+MV90ADmIasvQbOHm182yu+83rBDm1cmUaPqcP0k+haIr1XmTrPbj+oQ9AHNujkjtwhkMdsqDUxRcMuripDNMWWWlV9FmiELsDa0MuEdVo8QVZYwiRG4H\\\\\\\/DpnVbuZM8sFCgq0vR+Aiwv6JxVjMS+IWX+ICc6wRUqanZ4YBNMhrY0aDvY6ObkuBjY\\\\\\\/U6VgeFvO\\\\\\\/\\\\\\\/XEfB\\\\\\\/UozcEv\\\\\\\/iOyvCaT3ldZo3YNxlFZ5Fihwa41cCg6nRITYXrC3e2x8eD1op6gM1YyO1M7f1bYep2kYOySFzfMHpL7xjrMltZlG\\\\\\\/YbLgnbrcqG1EZnGKKZBc3SokenhZpIXJogc\\\\\\\/1eRKIpr+W6JuJIYoOwBmk+OJ5AuGW4L0Ce2QMkngcM8xlVeMH+mgyDqKdXsmtWqzr2pa6aYfCaDtYqoNIyWrdigHC8f+k27HT2f4bZsX4ALYCHYpLlke0LDIwPCWRbT6UX100ylIePE8qA3u1R\\\\\\\/NaNQVYBPre7FS6+5bNWzIp6oHhcYMwZiwR1Z1decmS3KvY677IyPoXj\\\\\\\/bPJHImIuWQlRSvFp7OtxeF3ZceaA1ovDf\\\\\\\/67uQJzd\\\\\\\/en7q5o763gO9Pv57gtRjWOA1LKY4bV+Ygo8eK\\\\\\\/P\\\\\\\/PzONj7JIYYYvN8qkmGyZYOwn58q91y7BMTj1p+3MMz9nDFKJ9swIPi\\\\\\\/TC4SYMUtdbJAchGczuBVSVJQb7LPp\\\\\\\/xi9UtVNxD\\\\\\\/\\\\\\\/DB7CRRL1W69S9Z1QxhFwvDCbJrZ1asCcVDtoyDnGDZ6aWW9\\\\\\\/y6z\\\\\\\/\\\\\\\/nGSlcSrMIRB8QQaei8aTj9nTHunq2V1WBtLXxW8\\\\\\\/YYj4YOfvFy4U4\\\\\\\/w4W1KW4q8QHU+wWhq4cR3zW9VGALnVw2fXv2eWD1gJnwMZnsRq\\\\\\\/nx7BW1OWqUL+oaPd1dnalrITNzDdQBdxYbeQHaqt4YWVfhO1XLrV0mhJX3HlUOimR42ogJ4Ge77pGl\\\\\\\/XzT4Ti1ae7tLhUXma9QzsQO7\\\\\\\/Qadxy4YEmkMNntX6Y68wvLi5bwpEoG10fXHVl3clj+Vu9Pc96GZ2SdVYc5HhPJcS6HNWfBKknvK+3dIkLv74E2QReZsWAvY+DQ6\\\\\\\/KtD4ctjXPbKmQk7iwD8HZYbPUwnj\\\\\\\/OiFAwo8XUtiXAJwJxKCufB5PBtQaxv1t0FqFJ3hi3mne9LrkaP8NAEA+X66XXNwtpOd0gtZ3bsbtHuyrwyBVRBxyVZjzWWwptcIV0\\\\\\\/ArXlKRu6gJnL\\\\\\\/cBah5HvyISUBs\\\\\\\/AaUjJ5Stw2Ajv+P5PzuJdq3L\\\\\\\/ryGwgL8QyqSGO1ZpD5bF1cfSJZ\\\\\\\/X3i22Mhgg6yqYjrYwBHwqpYKEx5+53gsjmjkRxh0h2+58sM3UZERTlf6x0Zo3ckFIF\\\\\\\/+jIMU\\\\\\\/MSw8dfADN+bpwI9Sja5hjILmpPQpUyTVsOCqOKYSFKx7n\\\\\\\/PUdP6EyRn88A6RvuTA+y+bLcAATSEaGjvCB+a26Qahc6SzCEucC4\\\\\\\/Npn9chC+DTFx7IO0jLL+OTJESx2bDxvaeHN28eWTCwVXgD421qbnRkEqFsP6idw7tL65osJmh\\\\\\\/bTbn9EkOrafe5+sFGsKMAU9tPaKRi4ndF+s+dPbJSHWhz1vbgZPZREnJwepGUEC9QDmEMtKSbLLuGuJvkR\\\\\\\/5cc5Y3PjDiw0D\\\\\\\/HX4zXkgj49ChGoHlIp2f7IXIDMnLGTLZf1YAIwe4\\\\\\\/S0OjUZBctJD5W2o0B1NYv+nAfhtPccmJBtU8MzMWwNKH5kSSyTUpWDUK6qY1IQFWmrU3r4sKOl7Si8Lzi\\\\\\\/Sz1yQZcMTC+cJ6V70V2eg8KZJPDh2oG8l+yrKLsDkNmk3kN9q6LDrip91bN9WVwkL1gqc2PeHXCUOJ8dHmePEVHjcw0VISZ2shXGVz2RrlunSHhCDJbM91yc4fgBEkcK+MRr1h9ib3pdI7P8A54RI4oZKaOsIxfGhSpE8b5Xo7QhGDfqO27RLGuualzb63n3QU+xe8VA+tQmcdE6V\\\\\\\/VGMGrxuw\\\\\\\/xVntMS1iNUA2EqONp8PRaa0zPliNOD0TMnmhRdNJn+p+jp1TUPBu388PqXSU42mSeauDIXhsDnCtdPk+p9Q1k\\\\\\\/O48RHjh3q7pCpztn+xAFPlbWEcsmThildCUlS+re6WRAo6kCg9xLEOcfn7oTDO9Y6nlI+ojGPdWvunLBBMbZpT0RnKtcuWR7KRVKfSQ6BbUDo+AUWkq9b4MFwn1wQFIeMGaMEW8TbkoOgIWJof0jbe04shenk0WC+kB7Gs8Rvt0UbFMereOp3E9vuJgWmAIXFcAxAFYqJ7ydjEWLsRdXyEnWSqKI\\\\\\\/SsSrc3bUcu\\\\\\\/CD9WrgLuub9uysDxZZL5lHRtrcqpGr+s1Z0nIiibX2uixBhUBFC3LfFB\\\\\\\/94KW+M5q7zu48ejmWCYyJxZbmGzkUtucrHv4+byxtNT9G+krkCXnozrfXxSBoUWksF14sDndBvAGclzD5OdhL5pD9p9eg+oK\\\\\\\/t8R8TqH29PZO+zTNMofP295PTyDzTX6EJVuQpo\\\\\\\/oQxZ7EL4BBsHC2SnVe66L3Fy9uYo9N9\\\\\\\/itVK8Ct1YwevEnpGhytXjwcbk3sDjkdDG5fBs3AtWVDA4h2HQhjZOmSbXX9bRDljw6K4D7jcisNChtutJ6bi1QytA1ItPqRLN2L0J8VFO6oqLnetsj7VJRRbRPbjM3zIt6W2o9Sh4kv8XSO0BiDj6c1bFeSnbnVjcrvnS+S4q7Ct7v+XxaTouJMUCQcEbPQ+Y5lcKpgRbLK9qlsfKqusPfIlvn6SaJ1dClayyIxdWDn6ujPC4BTWzwRz2T3cPBxnZMAmaduHcR8FLjSL+u2npZEUmbGxZWC95auZwvhguDK176ToqLjDoBRpwNn4LLDooeuTWXvqldjzBw3MyfEA+ua01Hyopf\\\\\\\/+yxojmJ6eFUlDOepqc0l\\\\\\\/1mY3KxTQiwFUxMGrEIkhW7IWZ5SHbMKcop+hy0iJdhsDTrxZrGi4YFnKEcJQ3w0gUeeI8SIM35GN7AU1JXp4jWK2T1rQiVgxQM4gg5JBsr7awwBMBLS6qU+40dtGCxeY\\\\\\\/6eK6f6MgHOe1kh\\\\\\\/KE7\\\\\\\/kZU2j6k1jEgnoayEEikhA6sdbZanutQna6a1QIcKu0FxjqCpHtHWefFgFbyaAJ1sRsDYyeap2kZOTrKtV9fFJuoYVemUT9S20mEG94qABNXH\\\\\\\/d3aYu2M\\\\\\\/vb1450t99MlCuaY1S2hk3L75QbletMg5I0Bk6ihM9TjW6MUMT4tbAYwQNDg4LgfIxOu+ZAayGCe8c95LGUFf9KHVsq4HeQEmZ5ifTkdzqvtKwyJshfukyGcyxMJlHfp00HO3DyqPxMvS6G0w5U1CjPp2lYBMZO2taljykzM\\\\\\\/YqryiTkYiY6X90Ado4LWzao+ZWXqaXrjVoCk+9b\\\\\\\/io8+UnokDLgYC\\\\\\\/H04\\\\\\\/NBGhvO2wdMGaJL6F564X1gQvLae+6exBKLgs+ChShgk7b7iKzyFUG8ernHfv6nimVhHSktjTVJUPDhd8\\\\\\\/fhSGCdJykZEpQQnA50C6uW\\\\\\\/Sm85LHDFqx3uQglaJO7g12x0HrXM4BXR7td8lwTkkRIQlDITWHqzfNA1KFh\\\\\\\/W0qQimFd497+eSJGqRy6BMjXGoi+s0tVybaAL2cp5LO9ROp1\\\\\\\/jM09zSI\\\\\\\/LNM4ZgZMx588LnG2\\\\\\\/cfXTGbJ5Fd\\\\\\\/FaLhVe1Xd3ianDfbthswawOthr7Vnmj3BL0O7vcVmZcgzEJvqRHXK6uGaCBD6YeahyUXFTHi79iGP3n\\\\\\\/qZW3Ovwqn5OQDbSB3zk6J3flInzAb1xmVVRiA49lAi76fmhpvHgUOLJRiP\\\\\\\/fmQCaBLva7c0cQJpdH6x6fDQVzdPjKGFWQo8zNAxEK7k9D1rTG75UdUZ4Knv2qNRGYYSuOV9EvjIyJfDdJt7Juh6+9GgAMqrYzF7P1M1KMKMYwNSjhxryQ50wRaquvLcihqGoORGNYh\\\\\\\/p14flpWb2ZuWmOQt8lauMpL0YNJcqsO56dfyT6voVJYrHqhrKgdIPZM42bgYPLNXKFb7MFU4b46LJiaoUwMbnw\\\\\\\/GJVG0oQFGO7VcQIXQBnkJuJxSpLCaLNDwEpIevookh2rjiVM1I7d6USdbBK5dfUt0lzCtu3atpJlXBxOi+EgXDkNLsFU3ISqay5CU4lSyiSiPmvGEeJBimwt6\\\\\\\/14UkvKzKfMyUPP\\\\\\\/PbCDuM+PpzTvsB4Gp3\\\\\\\/g8lnK+BVrH1EaHRq4JDDvnyY6YPpIzwdFCZv620Lb7qoAAhHZGuVA8AkbkAsioPnFJOGVk6n7ZY4FsewZW5C3H9gu5Z3UM3Of83e6S\\\\\\\/VM6XMaVtnh6jL6YME0c31aThU43\\\\\\\/CE0u6t+kAkg\\\\\\\/DZzc4tDgqxO32Oxi5oCsj5EnfcvV4HH3QhFYwi\\\\\\\/6dCfOInEkTn9PtOYAxlwcdHO1kF1j3fhh8R9h9BZ+en\\\\\\\/LA2te+6ttSpXPJzrG1fNhv\\\\\\\/PBbASZqX6WVbxaJCFzWXDR6UNGFYNxwzCBHoq1eXi7GtgWeCU06KiQ7uXzpp2mCWtDU2SDqn92SU9fBSDtWVaoPc0QUCy\\\\\\\/hmFZgQC7zeHhUimXjxLi\\\\\\\/VGV9rMZMxtAy5VRhK0RFUjIYxxH6UlI4Cm4pXv56DsX7ODoUZ3qQOagSQnK6q189QcYqt3nRkQAf3J4RYKOg0Azb\\\\\\\/9u7hiWVslH\\\\\\\/L\\\\\\\/UgR2v\\\\\\\/nQYSGD64t\\\\\\\/J2gHR\\\\\\\/zA3i3ivB6oBp69ZWa2WeO0h8q4xaD+N65QOhl11iM9FbqiMstvkOXlWvQPbkTQKDRk1VwC6VBepw0Qm\\\\\\\/e1fGp72ws\\\\\\\/l6ZglAHhzwXqEYuU42Xhi\\\\\\\/+WwsVG0M4nu9n8TCKu+5fHm7wlf3GpZv3s69RYc3tLkkcqVHdSQEQgt7QpysXGZjSUEj0r6E+rU93B1EogftCssHBqGIs\\\\\\\/xXbDDUJlAURBoCDRDDcCcFM784pzBLczzhtIXKEDpFje3QnopCSepi66GPe6JxpVymkOFquJjU\\\\\\\/\\\\\\\/S0rA1LTY93bXVcteVrpSrmN9e7wKpH5a6SPDi2hFkv35h7Py1CB0eak3rYrlNbWUh7LA4D7ztw71LsxZNswBV8yz\\\\\\\/NXhuFDFmAGDVKCfPsVgtze4FKlXuJ2PqluEeCZcGsNjPv8fkxMXdBT6JlM1YkKKs\\\\\\\/+2vEQ16dlhMEYC1H1MiHkRpCMntJ7JddsdUQbfQfBpfJ4l90zqVVh2C43\\\\\\\/F7NbFolSfbGqBoMJHByV431ZHl9GYcaP4AT5QLWIN6IIksh0ijHo79POD6HcSr7hjc4CLDJF93YUqu\\\\\\\/bpxO8v\\\\\\\/AilXz4S7jdnZ6DjVZRpYDNg8SRZ92ASh3mY3BZDw5wY4WiYJ\\\\\\\/LZAq4avfhXgF2mRRKwHiUzi22XvPcY2u3G3voSqMpND4VW2OL5wL68jDRRX2mRog3sIXiWZSbXZxUXGq72IG6mMVkxeGQfd0G6iUygTcXmhR08T5cniY1tsPZK+EryGnk1etV+nJ0XRHhLWkKp0tSCGlJCCRY79p+AuJPMzAGh2jHc763ysgHsHzYeRrtb+v2APs7Be2MIJo1I5tbULDEsvL1DD+JwFdtJVHUqctmDVMMyEtOKVORXCFRwoYvD66Z2ZgC3+PaID+SztPbvmOuH+cDDkMbcgauvfCidlc81J\\\\\\\/qfpVcORZq+kn\\\\\\\/0ItW2DVuIISJ8iTU9q8+evjbDeAj1VM\\\\\\\/yghbOyaKPYU+sd\\\\\\\/IGbgAyPjxM+huKtFF8Nky\\\\\\\/rqH4dLd+qNfnbCDZr96AWWti5ZmUVgxpCNXi7Zl\\\\\\\/AIUt89J34hJvkazQOCy2eyCR2cW\\\\\\\/F4FqUY8ItVhgK+bOAuBLFf+38TnztRWrZBdaxd8\\\\\\\/2IGnjwJ4kyR8ld3h4IGEmrQhIjfq53hiXlYvM+mRs4SaDN95dqWST1ExMUuy8qM5bnOz6aajN0VoNepPU7FT34SgMD2e6+T9dRhjKLIsagaexe4N7BCR1kO1S6UEnOb7Z\\\\\\\/wM\\\\\\\/iVx3jlyM28lbtaARMZgStwOzeTeipdD1Kgo+UZmJz5xAmqqrwIRx15lNpy9QwSEMVBJfnGiVpxBZtRzDq6wgCVOrfSKgjMDKSj8dn+J6l6CNFGOXZc+R\\\\\\\/OvnJgxR8lk1Wv6dPttuEKejaBaCHeIAg3w8AyT7\\\\\\\/FF+b5o2HKETiRd5ISayoQx6pwsKGYJeqw+cS1PRXPHckUBW5\\\\\\\/STYpP74kVnXtTUqsbse\\\\\\\/oP4sNm+SkutdbsyZPMJ7v0Uoq2YFE9vEibJcpZbrPiJqEaVaFbpGLYJcSvR1rRLgaIBenmJnLF5UH4eKPswx2Hr5EbFfRHwqr+uWwfvzXVCw\\\\\\\/ur2A9rK2OuxJynTZk7o+ASoxvqTi8pcd++YyZrt3Rl3ag2k03FMPhoZ5a8qzZg5YqwqgIcwNIzCJtcN2njhaAfLdey6trfbyMGU8hY9\\\\\\\/Nm8BeP\\\\\\\/a\\\\\\\/+TOQlzn7OvkBeYKmGyHd7g8DsHo2mtixvVozc5lanUzVxhhHZQLYg\\\\\\\/kqrd2PXoEDZ00uTquFzGZINmzYVr9rfdmyq3A1q4NiFcmSpSKfe3WWlESWehqHwrCwobIw78dfg88z8GquTAVP161Z5GtjbFU0rhnYG++MsacrBEdm2Qiw826SPXP60Kq+OGs1yG2Ui34g8ZWV\\\\\\\/\\\\\\\/kG1qNsIUpadiziCftfBQOVxT5sBNdKBdnM2V1jAbQFJLjf9PpOTnZzwz6bQm79hI8wjEjjd5SvmTPfJF8sfaZcoU+njheH7yPa1Dqil1kdhCyAb\\\\\\\/P\\\\\\\/n9dMaM12HE58m3+3KktbOSituLBriPZ8+NqopTWjzZSG8wvTAm1u+i3NcXZSxbWYdjfr7hYFLy0sq\\\\\\\/Jjh+QUvNCB\\\\\\\/c\\\\\\\/7UFAmT9MgfRCebdSTJOGVteZOU\\\\\\\/CmecxUtXUIRZ+CFogEFELMzW5CHB8mP9N22W6Yn4Z+evtlboqhMSic0SCw0L6fiPbHvgPAHoemQmqmG3wVXmEM0F14KrnF88Qt+ADT9aqUo\\\\\\\/trW8G5vmqhUwXXqVAx2vgErUItipGoHu3\\\\\\\/kTDgZvDP9hLgXtTL2UV9MD8ctl4Zvrax5Btul9W9f5T64N44aRDMHQlyYBThuHfirrEEscxqJwGUsvBiG1PTBjfI8hg\\\\\\\/oYuAfelXnozHbyqaUkwJXx8ueWwLhXCV3S97TOXIZwBZKduBFsbUdfiWZC7WBiRnGYI8Vo0lT6Gs1AvvhKam4qm3nVToFIJpw\\\\\\\/coBILVnvbh83MQ8xM773Dgx3pXeTKROLEP1DXejhcW15PbHql2lpBL6xj1qNpiP2dA7EihRN4Emt9PeUGi9PaepZKZdJq4kObyt8DJOn6KtepUFgRdAWbQ\\\\\\\/ITYUGupsgYuQUFimel1N5mCXHC1oYLQq8NGJ3XW9nYTIKNQtXOzz2qVYCDYk\\\\\\\/1LGWcsYtJEH84Posku8kURHQH\\\\\\\/anB\\\\\\\/nwD9rHC1S5lCL+N+6rb2pFRIag46U\\\\\\\/gSEShNR+IPxREVFmBrdqe3EfC2pMFa8Iuph2TEbbqd4RNd65uAcV84ocK9m6PT7akDcFMRBbQHQMt5a2ZltjlxFbENQaJuNWBt4tflFkpWE6EIsCW8gt2aN1IohxccaHZI6CpnW9y4aFcMtXvEI0DkZz6Lg3vMMSYXqW\\\\\\\/Zz3S+Dq\\\\\\\/LPfMvfRytytMingGTQMYUAdKdtzw7AolbBfyvUYZnnH8gOB0VZ7WvANRSUybNmQ9qE4O6MRoxAeGCcu2jbOaplX5FaKoNbx3mQuVKzu797BhsZjB8LWpaGnaCAhcG8T7QOBVwpbpmsc69ChaejDP6fnePB2fPChMkMVYJdi7S4vOhLFc5KlvMKT0knVGLSVHaU2a2bGKTY8uGNuLc9G\\\\\\\/isvGo0aLcRMSlQztPeTRoWKff0\\\\\\\/HFLLR1sTy2QEObcJQDJz\\\\\\\/gTl05e3PyJHWmCRpvvIpROUk1kufF\\\\\\\/CPaSQVWRK7Lm3jSa0peUooh+NRJl1VRAIhgvD7u\\\\\\\/ldMyyaOVUw17SFGUag5RW6aigsZITo7QAtcgWrJRgBpu3Z85J\\\\\\\/rviwJYglD9zYcQ\\\\\\\/xBTsippEXFhouQJ5\\\\\\\/5QeJf\\\\\\\/aYEw+h3nEmdASpcbRI87jKWpNGNsAG9EZOdBKZW6h1BW\\\\\\\/VlJ\\\\\\\/J6ZtxHtR0vwdZLVbdceVLuolKsEPOLq+PbTpEUgOml\\\\\\\/PIj33m8vQ1aVrD0rN9Frp4D\\\\\\\/tqroG8UrAcM1j18GUwUYgrY8RjHygoNfFB+0b36lcg\\\\\\\/8O4viPTCI3CrzIbgqbNCBCaXjIhFHHDf2DtHcmuMW610FK4qXkJXsVIb5cFsDAsVDotp1EeeNbPhwY5h5jJP28g\\\\\\\/lUVgW4uoocbSYGOLgFIt\\\\\\\/5asf8jE4OBFsedlOYGUHnPks\\\\\\\/L\\\\\\\/mumSB9pz70y0btmpU1oiU9nP0wtIR+uZIUfovU4UIXN6cYWmqup690pd4Wn8QMx89ARIIyF\\\\\\\/ioAu6fjMNp+c0ULKMRYYWRXo5iEXWdEidW2rhRpJQZitKYh9CGJVgmdTabrNqItf5oZXUFK8ZeoZ1YVljLOu1AcAOs82SBTJfqw26jyCDMQQKbFQx1JDMZRWlpafcoMA46MMDWB5SRwpWX0AlllfdMjsFapy0F2Ufsz1FP3l5IyyPdAcaF8h34xM0j1k+2CrmxG7PZPXbpEPFGenVjpcla9wP\\\\\\\/kyAz5q8VeZdUI5man1ZjtujoOzIQ4deaCk9jTyJDTPMQgzoXILbAqBqkq6ers3r+pki+6B4aShKykg4LfWySZ4+YpYZXhuw3D5FWWSCo5ux2Yyv9N0ZF9Mp0YnuuJB02sWWIbjNZx4iAZ7hWy88S663ccoUWlfqzQV4VDkcNK1AHppuoKJ6a3cnj0M\\\\\\\/93Sm3D5Lfa\\\\\\\/ttFTqFE752P9X2Vy\\\\\\\/o4QuDzxzneCQubStDyrOTU8rWQz3zZYJ4PV4SGQTPCIjdd0EeVNxS1W0K21K0E4Y1wpVTC5FIntwWDAMfU5r\\\\\\\/Sqa5KIrcpqkfMPGnrJIrlC9Mxe7Vv\\\\\\\/JmwN0f51RJEXWW+EkVd2cTt+QCp3qmnlziH0UCW9LOhMEgqfZ9t+UvLf\\\\\\\/tmGwU\\\\\\\/nG3gDGL68C7K548SE+GbMyf70xUibCKfLcb9g0pS835vbbtIQF9\\\\\\\/ps0dJSm7vyRdEnVgLrIbcGuuyxqG9GzAVkBqseiTTBpVCYQDLrF8Lw0YFtyBMfXcns8g2w6E6KlFlsqQTjdRjilRl7H5Qg3OPWcJp1Xm0t25o8OD5YJXP9y79l5PNiLlHlpCKOrM9o8FxtplXwqcb7SE0eoGtd7A0tpjH9IVgYUIAv184qCqJM8uKziXAmuUyaVtv8rQEtHbgEWuWqDs6s0sW46J\\\\\\\/QAsOqteKydQnUiXZPbPPm7o3nZO+7MXXHOOyyfu3oPKURX8BwyU95qEtbs1IQDdpkZQzZeDw5FmhXyVGmZPdB89glfGD8USafNTvtxpBsf+cIplaUrO+3JpD95AMW9bMBg4ekkyCMkxrj4pqyXBfpOwFujiV9WKjXS\\\\\\\/dhcEhyNGds3RYCfH16eprQR2rIgrUrMj+PyhFue9rLaTgJWlHhVb5b87h6NYvhuigC7iv7l7JaNSmw82ii\\\\\\\/wLwYaQ1wtyNsa98MXHRhazXCIKJmeQojpGD0+w+8yJUbi62N7RCG0Ol6NP9qkWrxf0XncWH7fSBX+Vbj\\\\\\\/OUGchzx75iGOcf9ad8uU54Bxk1PuRboQi0lYaSf9v\\\\\\\/xPgHifqaUVpw5Bc4WiGXhlk715\\\\\\\/BI\\\\\\\/\\\\\\\/FYGCpEkj+i9sD\\\\\\\/vWzvcV5yCw6QQcpz5LeEsIRzTp+a4VjXfSbH3y5oXKASweLaN9TuWF7GlRCfuJphJnMyVUeZj1yfssoCoX28bJFe7amkdEQrXMydxSOA0uwUJTvcXp7htzkAi7JBB8o2nd5112tNycwhl33ahUfiRQ3V25AsVzr7Nz4h70jJfKAf3tpfBnDJtuENQ4Su+IFbuSNDjkTnR7uGa0TC3bUL9GcQn0Nz019Iy2PpO6j8BJdhK\\\\\\\/UC4hpuTyF+bimyhVM9y6FKE3Orji1ySMd12ivFr4d2s\\\\\\\/hwBLZosgKrA+IrDOFXJ5FwnokkwDCP1dxyG62RMl+Pm2D0TrFL9LtEKnJtL26gaIuuPPhvGGPHMx4jW1QlgDDOcHMy4HOTD2DscEJZ2lwoYIkw7KLeP4mJg5qUADmQmPqMhGsIyBCLmwVJ\\\\\\\/kuHj1uveCVCsmy5Njo6w0GIyK6\\\\\\\/rBT\\\\\\\/c6tn6I86lCaOfEgsy2\\\\\\\/N8O0ptqW0lw3hECqJtqzuik5ZUjwo4g6Ov1KwbCfBgVSLOCB2KaichEHsgn0vYx5R9CNkbhO3q5jxKpMGC4CcyHB2sdyezRGqHBT421FYT3kQ6HH6NPgkHLXm4UgZFVFmZghpkfFtym1+Er\\\\\\\/RC3UQxGvJcxDEVo1DW29kAR10MnfqYPITrRjdzoUJF2O6bHZVcRWV71JB66nuTzrOZWEJ9b9ed6jjB\\\\\\\/VJXS7IwxBto4TusIYu\\\\\\\/BppuSGM7LsgobjcuHA2obny+mQGZkfmYBCyvz62hrk6amUmJOTAEfZqQWecDx\\\\\\\/4stsdi1kthXClTW2fJlf01iNMnFu3Sl85GfXPFipWqhbIabixdExzV8UBxN\\\\\\\/fjK4+iC2KNurUfFJiSfqBN0qXBowTBwjrBU6CRVOHGWw8E2eJbFluRDY61pxlaHC0yhgZsSypYdO8ERVypEusjjcBjasUFv6TNSgjBCBgV\\\\\\\/gWZLe6sHOE9\\\\\\\/EA86IcmD3AZI2LY+Pcq3wrGIe1AZu5+2wNjQjYHFsKZIcFxrw3NH5vsq9ZDMOhooG5CxYM5s2cK\\\\\\\/uhIMfRs1OyXYH7XdPVBWEOWbvHmK2hCJAjW25x9tAyFn3aU1U1rClSWwn\\\\\\\/eeafCfRIlRv77z0wZE34KUR6\\\\\\\/wTIsQ7DFHEr0namo+HG5pWnGCR8KH3qzR8KNldg6L04X4M3nuJjS6X9R+ByxSrYAljF\\\\\\\/nNOkujwa44M0+eIiaO4oFUVF4QuBYeAZJO8zq0tFEnlBDT6vfWt4gfHxnGiBQFYUZw37kJGySOYOUc1ovzeMn1PcMMcwj7frObRnqiCxQNWIavOGNwCnN7Pg0ZLw\\\\\\\/6lZB1KgBqsH5TJZ3S2WxyXZmOlDGa8FJkA==\\\u0022,\\\u0022iv\\\u0022:\\\u00222rPtwfw2Z1HG7lTkUmWhRw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002268027439b91019cdae5ec8460ffed4fcfd08f6f07dbbb1c57f4001243bc533b6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022f99aGgkgkPP7j+KjJal46zT7MAN0DTrm36FVU\\\\\\\/\\\\\\\/gJ8ZtomIAyHR2tq80dJrO8nWrsBoMoKzaLbJIaUtL0ttTJiEGZ9CNLPwQjau9p\\\\\\\/b1LdBH\\\\\\\/fjYCZX7MAinFK4nult5Ao69WBgXVj\\\\\\\/u02uE2Tc2t7CINfO3HGrXLq1L7t8jXjKIETMUXUxrrJZLg8uRH+p8lpX3XiYnSNURM5Vy50AZ1rhgrzH+bI94ZHvlr1kb7JKpfSIe3MBUFWiUD3nUhgQcRQlJMCxdCRUZJfEzq4Aa0s9YeZ0TIgI+5lZxCtsmtS73X13NKUK0sm5vIPLOXIEfTyq1etelMGHh2LRle5NikQi7NQxEsXjmEKKj68xQjHF6zWqOj2AmCou+w6ay\\\\\\\/f2t8fLqErQIRvNoZ9vFLBRYuO7ZN8CYktDReLv\\\\\\\/L\\\\\\\/uViOkmcy\\\\\\\/g6apxititrR0UFq8Gea0UbyCQ6HK4wxyrWQTyA8Mtk8rmoeUUrxXzp5r00FOZxI0s2okJbuWF2dII9YoFqXVJldj1cQpBPeprkwIT8BoRCS8p56nAhnOZARYImoiEZAJ+G0bhWG2n4AtPgk4zuL3J2EOdpkEJDfUp96h4YRZxwN5NwnmyYY410RC+pCBrce0AuC8juIzCPIK9LBYorlPIvl+TbtEZT7q8Gw\\\\\\\/Q3DBm8k8wIG5ti81Ya2+uPHsk1sGjsJa8PTmtR995TCLjy6p5bOuaJGTrKj5glEuMR3OYO8RZX4s8DIAtdca3V1zFzmAsPFKnA\\\\\\\/w2o9xP\\\\\\\/7giSTVAh7fpR\\\\\\\/MKoDbIT8f7By5Rnb4TCgSxhHLPeDFxGiA6jQ12cAC8hPAZ4Ru9WdgnEPsUYi5w5NeKD\\\\\\\/5LPFpNemw5dcQfI3\\\\\\\/Y5ZPMeGHrGA3hRsKWqB76EiRtxvEAkM7QRx7TJQYOyn2pN3+QFvHrTV89EkrdtzSeFPtVxer07dFj6FvWybrIX8vu7I2mER\\\\\\\/gTj1MXW2b0X2lquNCl956hpF0w5fX7tNkMHqQw8e0yOk8w8UZeqF94ln6\\\\\\\/\\\\\\\/Pf2hEh5WViT0AQW3wn0RPTmENjvu7ztyWUOFfD72BFJ8u1VK4tivrweFe5xzy\\\\\\\/+a+oNeFB5JCbKoRtWzVnH81jVkTTrhPdR0ke8mMJdTiBwE\\\\\\\/GOBr5MsIVGje5I78q6Xvjyama7o59kESBckBz9Ko3x6PCwzpl+aCvTygPvGIN4HN1mL3rvTyW0nVB6HzRjGtHFirPvE4H8xuxSOJ3XdaMxWxv+EjQexOkiomj2OmYgWNxULC6C2u3lVv+ifXeDt5UtjAaxBVcJ+c2n\\\\\\\/1OdVfblFLsC0hvTCq3137YKjOLIHICAmapiOVfSXU8j4tWJJ6FdI2NwNho5I5aY9TN18WLPJsAmjzecO2tDO4Gi0qk5wy4MsgJ2iSpVyFU+8nakyfT2X5AVI70gr+XaDZd9h84+H6kq6PugrD1l0dPpz1uthXcAff+n30cYRXqp9c5wNaBgjsJdYAp7L2Qjnax9slsWU0HuWnLhYewOIGQPOydJxc9Ee39W0nnGpCAS+sf17V1eMlj04aLpP\\\\\\\/0h4JMbpz7zvgWSONrUyqBUmRTMKpvDLEbLropjFKfpwzJob5Dq9ik\\\\\\\/+4lBLdv2zSTGJ6PDrpYvftAJQBe5x0cwLvaEZBIiGKSSTzNgdmaFh\\\\\\\/kOcV3THeqcI1kLfgVCsayC7IQLsVxHNmOssx3uOLUlm6NXMOhG\\\\\\\/6xfqtqOP6C01LIvw2ZXPqrn6VrVBZYbDMxE83yMD+nQo1ezveyq7YTKOAiqiNYUP6EDIaUcC1OjFXxvWtDQIkkjGNoCrax2yJzj6iPZb+EHj5ptKEZ3Pc+lXQqrna0nnwKsIgKmntUuEthE0VyWsSO4TTTXVrwM7\\\\\\\/gDdykrg0luHnJxcrMpX6WbdvDaZzU5P\\\\\\\/15F34M5F2HvHfVyEVmEvPf2C7LhMOLB36sS3vkRyBBhQhCasAOF0bs\\\\\\\/nCpOcAQDp+WL05taW04CLsu0VJHledzGy2CRB6AxY957Ixf1hfcADIhg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022yrrm9+lhNIYcl5hxQ0m00Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225cccbfda9e1749558868772ca32ea7c9508c2bdeb65bde73b2576477c622f01c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495261 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/281" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9F_CjclbgiMc9mYPd3i2zpL2CYNY--x8WMaZmlie9NI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/281" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AWAwP7TZOggBScqM2wlFocDDBq0U9Y92MroAeHELtmk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/267" title="Rapid Degradation of Caenorhabditis elegans Proteins at Single-Cell Resolution with a Synthetic Auxin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/293" title="Whole Genome Analysis of a Single Scottish Deerhound Dog Family Provides Independent Corroboration That a SGK3 Coding Variant Leads to Hairlessness" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495261" id="node495261" data-pisa="ggg;10/1/281" data-pisa-master="ggg;g3.119.400620" data-apath="/ggg/10/1/281.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcelo Mollinari</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4143-8909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bode A. Olukolu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme da S. Pereira</span>, <span class="highwire-citation-author" data-delta="3">Awais Khan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4901-1694" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dorcus Gemenet</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6583-0628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>G. Craig Yencho</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhao-Bang Zeng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">281-292;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400620 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495261" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcelo Mollinari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcelo%2BMollinari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mollinari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcelo%2BMollinari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcelo Mollinari</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mmollin@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">mmollin@ncsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bode A. Olukolu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Entomology and Plant Pathology, University of Tennessee, Knoxville, Tennessee,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bode%2BA.%2BOlukolu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olukolu%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABode%2BA.%2BOlukolu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4143-8909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bode A. Olukolu</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guilherme da S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2Bda%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2Bda%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme da S. Pereira</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Awais Khan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Pathology and Plant-Microbe Biology Section, Cornell University, Geneva, New York, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Awais%2BKhan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAwais%2BKhan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dorcus Gemenet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>International Potato Center, ILRI Campus, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorcus%2BGemenet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gemenet%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADorcus%2BGemenet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4901-1694" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dorcus Gemenet</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">G. Craig Yencho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BCraig%2BYencho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yencho%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AG.%2BCraig%2BYencho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6583-0628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for G. Craig Yencho</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Zhao-Bang Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao-Bang%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhao-Bang%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhao-Bang Zeng</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/281" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495261" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/281.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495261" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/281.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495261" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/281.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495261" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495261/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The hexaploid sweetpotato (<em>Ipomoea batatas</em> (L.) Lam., 2n = 6x = 90) is an important staple food crop worldwide and plays a vital role in alleviating famine in developing countries. Due to its high ploidy level, genetic studies in sweetpotato lag behind major diploid crops significantly. We built an ultra-dense multilocus integrated genetic map and characterized the inheritance system in a sweetpotato full-sib family using our newly developed software, MAPpoly. The resulting genetic map revealed 96.5% collinearity between <em>I. batatas</em> and its diploid relative <em>I. trifida</em>. We computed the genotypic probabilities across the whole genome for all individuals in the mapping population and inferred their complete hexaploid haplotypes. We provide evidence that most of the meiotic configurations (73.3%) were resolved in bivalents, although a small portion of multivalent signatures (15.7%), among other inconclusive configurations (11.0%), were also observed. Except for low levels of preferential pairing in linkage group 2, we observed a hexasomic inheritance mechanism in all linkage groups. We propose that the hexasomic-bivalent inheritance promotes stability to the allelic transmission in sweetpotato.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">Polyploidy</a></li><li class="kwd"><a href="/keyword/genetic-linkage" class="hw-term hw-article-keyword hw-article-keyword-genetic-linkage" rel="nofollow">Genetic Linkage</a></li><li class="kwd"><a href="/keyword/hexasomic-inheritance" class="hw-term hw-article-keyword hw-article-keyword-hexasomic-inheritance" rel="nofollow">Hexasomic Inheritance</a></li><li class="kwd"><a href="/keyword/haplotyping" class="hw-term hw-article-keyword hw-article-keyword-haplotyping" rel="nofollow">Haplotyping</a></li><li class="kwd"><a href="/keyword/preferential-pairing" class="hw-term hw-article-keyword hw-article-keyword-preferential-pairing" rel="nofollow">Preferential Pairing</a></li><li class="kwd"><a href="/keyword/multivalent" class="hw-term hw-article-keyword hw-article-keyword-multivalent" rel="nofollow">Multivalent</a></li></ul><p id="p-3">The cultivated hexaploid sweetpotato (<em>Ipomoea batatas</em> (L.) Lam., 2n = 6x = 90) is an important staple food crop worldwide with an annual production of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-1.gif"/></noscript></span></span> million tons (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">FAO 2017</a>). It plays a vital role in alleviating famine, especially in developing countries in Africa and Southeast Asia (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Loebenstein 2009</a>). Despite its undeniable social and economic importance, genetic studies in sweetpotato significantly lag behind major diploid crops due to its complex polyploid genome. Polyploids are organisms with more than two complete sets of homologous chromosomes. They are grouped into two categories, allopolyploids or autopolyploids, when these chromosomes are originated from either different or from the same species, respectively (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Comai 2005</a>). While in diploid organisms the study of allelic transmission and genetic linkage are relatively simple, these studies are considerably complicated in polyploids due to the wide range of meiotic configurations these species undergo (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sybenga 1975</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gallais 2003</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zielinski and Scheid 2012</a>). Moreover, current linkage analysis methods for complex polyploids (<em>i.e.</em>, ploidy level <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-2.gif"/></noscript></span></span>) are mostly based on pairwise (or two-point) marker analyses (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fisher 1941</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ripol <em>et al.</em> 1999</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kriegner <em>et al.</em> 2003</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken <em>et al.</em> 2007</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cervantes-Flores <em>et al.</em> 2008</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">van Geest <em>et al.</em> 2017</a>). These methods rely on the assumption that the information in isolated pairs of markers is sufficient to detect recombination events between them accurately. In complex polyploids, however, this is rarely the case due to the limited mapping population size and the incomplete information provided by biallelic markers. Here, we present a fully informative multilocus genetic map of a full-sib hexaploid sweetpotato population derived from a cross between the cultivars ‘Beauregard’ and ‘Tanzania’ (BT population) scored with more than 30,000 informative single nucleotide polymorphisms (SNPs) using our newly developed R package called MAPpoly. We also inferred the haplotypes of all individuals in the full-sib population, which provided novel insights into the multivalent formation and preferential pairing in the sweetpotato genome.</p><p id="p-4">Our multilocus analysis considers multiple SNPs simultaneously and propagates their information through the linkage group (LG) to overcome the typical low informativeness of some two-loci combinations. This strategy is fundamentally important for complex polyploid genome analysis since pairs of biallelic markers carry very little information about the recombination process individually (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Luo <em>et al.</em> 2004</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>). Moreover, the signal-to-noise (S/N) ratio in complex polyploid SNP data sets is considerably lower as compared to that in diploids and tetraploids (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mollinari and Serang 2015</a>), thus making the genotype calling more prone to errors. The multilocus approach takes into account these errors by using the probability distribution of genotypes provided by the genotype calling software (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>). Therefore, multilocus methods are essential to make appropriate use of the information of multiple-dose markers and assess complex polyploid inheritance systems.</p><p id="p-5">Several studies attempted to elucidate the polyploidy nature in sweetpotato (allo <em>vs.</em> autopolyploid), including cytological and molecular marker analyses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gustafsson and Gadd 1965</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Magoon <em>et al.</em> 1970</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shiotani and Kawase 1987</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Austin 1988</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ukoskit and Thompson 1997</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kriegner <em>et al.</em> 2003</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cervantes-Flores <em>et al.</em> 2008</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhao <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Monden and Tahara 2017</a>), and more recently sequence-based studies (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Roullier <em>et al.</em> 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yang <em>et al.</em> 2017</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Muñoz-Rodríguez <em>et al.</em> 2018</a>). Two polyploidization scenarios were proposed: the first suggests an allopolyploid origin involving the hybridization of two sweetpotato wild diploid relatives, <em>I. trifida</em> and <em>I. triloba</em> (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Austin 1988</a>); the second, well supported by the literature, suggests an autopolyploid origin with <em>I. trifida</em> having a dual role in the polyploidization process (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Shiotani and Kawase 1987</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Roullier <em>et al.</em> 2013</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Yang <em>et al.</em> 2017</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Muñoz-Rodríguez <em>et al.</em> 2018</a>). Corroborating this scenario, the polysomic inheritance observed in several molecular marker studies rules out the strict allopolyploid sweetpotato origin (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kriegner <em>et al.</em> 2003</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cervantes-Flores <em>et al.</em> 2008</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Zhao <em>et al.</em> 2013</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Monden and Tahara 2017</a>).</p><p id="p-6">Nevertheless, none of these studies presented a comprehensive profile of chromosomal pairing for all homology groups across the whole genome nor the potential formation of multivalents at a population level. Solving these missing pieces of information is essential to unravel the precise mode of inheritance in sweetpotato, and consequently, allow an efficient application of molecular techniques in this complex polyploid breeding system. The BT population coupled with high-coverage sequence genotyping used in this study has two essential characteristics that enabled high-quality mapping: 1) high and uniform sequence read depth across the genome, which allows for high-quality genotype calling including multidose markers, and 2) sufficiently large sample size to allow the detection of recombination events in a hexaploid scenario. Additionally, we considered the uncertainty in the genotype calling by modeling the error during the map construction using a hidden Markov model (HMM) (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>). Moreover, all methods can be readily used in tetraploid and octoploid full-sib populations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-7">The mapping population consists of 315 full-sib individuals originated from a cross between the oranged-flesh cultivar ‘Beauregard’ (CIP440132 - male) and the African landrace ‘Tanzania’ (CIP440166 - female). These two cultivars were selected due to their agronomic importance and contrasting traits, such as beta-carotene and dry matter contents, drought tolerance, and resistance for viruses and nematodes (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Cervantes-Flores <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gemenet <em>et al.</em> 2019</a>), for further QTL studies.</p></div><div id="sec-3" class="subsection"><h3>Optimized genotyping-by-sequencing protocol - GBSpoly</h3><p id="p-8">Next-Generation Sequencing (NGS) library preparation protocol was optimized for polyploids and highly heterozygous genomes to produce uniform coverage across samples and loci, GBSpoly (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wadl <em>et al.</em> 2018</a>) (details in S1 Extended Material and Methods). The optimizations were based on re-engineered barcoded adapters that ensure accurate demultiplexing and base calls. The 6-9 bp variable length barcodes were designed to account for both nucleotide substitution and indel errors (based on edit/levenshtein distance), to minimize phasing errors and to maintain nucleotide diversity at every position along the reads. We also introduced buffer sequences upstream of the barcodes to ensure that the barcodes lie in high-quality base regions by avoiding the elevated error rates at the ends of the reads. The adapters were ligated to fragments generated by double digests, <em>Tse</em>I and <em>Cvi</em>AII, and then size selected to minimize PCR bias. By designing barcodes that did not reconstitute the restriction sites, ligated fragments were subjected to a secondary digest to eliminate chimeric fragments. Sequencing was performed on the Illumina HiSeq 2500. For more details, see S1 Extended Material and Methods.</p></div><div id="sec-4" class="subsection"><h3>Genotype calling</h3><p id="p-9">We used the software SuperMASSA (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Serang <em>et al.</em> 2012</a>) to perform the genotype calling of parents and offspring of the full-sib population. For quality control purposes, we eliminated SNPs with read depth <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-3.gif"/></noscript></span></span> and estimated ploidy levels different from six. We also filtered out SNPs with more than 25% of missing data and with segregation distortion (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-4.gif"/></noscript></span></span>). Additionally, we removed four individuals with less than 100 reads on average for the selected SNPs (see S1 Extended Material and Methods). We obtained the physical positions of the selected markers in two diploid reference genomes of <em>I. trifida</em> and <em>I. triloba</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wu <em>et al.</em> 2018</a>) and classified them into shared between both genomes or private to a specific genome based on the full-sib population genotype calls.</p></div><div id="sec-5" class="subsection"><h3>De novo map construction</h3><div id="sec-6" class="subsection"><h4>Grouping and SNP ordering:</h4><p id="p-10">We computed recombination fractions for all marker pairs considering all possible linkage phase configurations. For each marker pair, we selected the recombination fraction associated with the most likely linkage phase and assembled a recombination fraction matrix for all marker pairs. Using Unweighted Pair Group Method with Arithmetic Mean (UPGMA) hierarchical clustering, we generated a dendrogram representing 15 LGs corresponding to the 15 sweetpotato homology groups. To order the SNPs within each LG, we converted the recombination fractions to genetic distances using Haldane’s map function and applied the unconstrained Multidimensional Scaling (MDS) algorithm with the squared LOD Scores to construct the stress criterion (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Preedy and Hackett 2016</a>).</p></div><div id="sec-7" class="subsection"><h4>Phasing and multilocus map estimation:</h4><p id="p-11">The parental linkage phase configuration was obtained by serially adding markers to the map sequence and evaluating two-point likelihoods associated with possible configurations between the inserted markers and the ones already positioned. If the LOD Score between the two most likely configurations was less than ten for a subset of configurations, we compared the multipoint likelihoods of these phase configurations to proceed to the next marker insertion. When the last marker was inserted, we re-estimated the multipoint recombination fractions between all adjacent markers (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>). For more details, see S1 Extended Material and Methods.</p></div></div><div id="sec-8" class="subsection"><h3>Reference genome-assisted map improvement</h3><p id="p-12">Using the <em>I. trifida</em> reference, we detected collinearity blocks within each LG by visually inspecting abrupt breakages in the scatter plots continuity (Supplemental Material, S5 Fig.). For each collinearity block, we evaluated the multilocus likelihood associated with the initial MDS-based “<em>de novo</em>” order and the order provided by <em>I. trifida</em> reference. We selected the maximum likelihood order for each block, tested several orientations among them (S1 File), and chose the configuration that yielded the highest multilocus likelihood for the complete map. Next, we inserted the remaining private SNPs from <em>I. triloba</em> using the genomic position constraints imposed by SNPs shared by both genomes. We also eliminated SNPs that caused substantial map expansions (see S1 Extended Material and Methods). Finally, we re-estimated the map by considering the probability distribution of the genotype calling provided by SuperMASSA (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Serang <em>et al.</em> 2012</a>). We also computed the Genotypic Information Content (GIC) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>) for each homolog across the entire genome.</p></div><div id="sec-9" class="subsection"><h3>Probability distribution of the offspring genotypes</h3><p id="p-13">The probability distribution for all possible 400 hexaploid genotypes was calculated using the HMM framework detailed in S1 Extended Material and Methods. Briefly, if <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-5.gif"/></noscript></span></span> denote the <em>j</em><sup>th</sup> genotype, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-6.gif"/></noscript></span></span>, of an individual in a hexaploid full-sib population at locus <em>k</em>, the conditional probability distribution of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-7.gif"/></noscript></span></span> is defined as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-9.gif"/></noscript></span></span> is a sequence of observations of <em>z</em> markers, <em>λ</em> denotes the map parameters, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-10.gif"/></noscript></span></span> denotes the joint probability of the partial observation sequence to the left of marker <em>k</em> (including <em>k</em>) and genotype <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-11.gif"/></noscript></span></span>, given the map parameters <em>λ</em>; similarly, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-12.gif"/></noscript></span></span> denotes the probability of the partial observation sequence to the right of the position given the genotype <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-13.gif"/></noscript></span></span> and the map parameters <em>λ</em>. The quantities <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-14.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-15.gif"/></noscript></span></span> can be obtained using the classical <em>forward-backward</em> algorithm (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rabiner 1989</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jiang and Zeng 1997</a>). Their derivation is presented in (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>) and briefly described in S1 Extended Material and Methods.</p></div><div id="sec-10" class="subsection"><h3>Offspring haplotype reconstruction</h3><p id="p-14">The probability that an offspring individual carries a specific parental homolog at position <em>k</em> can be obtained using<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-17.gif"/></noscript></span></span> is the inherited homolog at locus <em>k</em>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-18.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-19.gif"/></noscript></span></span>, 0 otherwise. We obtained the haplotype probability profile for all 15 homology groups (one curve for each homologs, from <em>a</em> to <em>l</em>) for all individual in the bi-parental population by computing <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-20.gif"/></noscript></span></span> at every marker across the genome. For more details, see S1 Extended Material and Methods.</p></div><div id="sec-11" class="subsection"><h3>Heuristic algorithm to detect crossing-over events</h3><p>Given the probabilistic nature of the haplotype profiles, we proposed the following heuristic algorithm to detect crossing-over events:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-16">Regions with haplotype probabilities greater than 0.8 are assumed to be 1.0, otherwise 0.0, forming a binary profile;</p></li><li id="list-item-2"><p id="p-17">SNPs within a continuous segment of homolog or gaps flanked by crossing-overs smaller than 10 centimorgans (cM) are removed.</p></li><li id="list-item-3"><p id="p-18">If the remaining SNPs represent 20% or more of all SNPs in the analyzed LG, use Equation 1 to re-estimate the 400 genotypes across the whole LG and compute a new homolog probability profile using Equation 2. Otherwise, consider the probability profile inconclusive.</p></li><li id="list-item-4"><p id="p-19">The crossing-over points are assessed by checking the points of probability transition across the LG. Homologs involved in the chromosomal exchange can be trivially assessed.</p></li><li id="list-item-5"><p id="p-20">Exchange points closer than 0.5 cM are considered inconclusive since the haplotypes involved in the exchange could be erroneously assigned due to the lack of resolution in the mapping population.</p></li></ol><p>We applied this procedure to the 15 LGs of all individuals in the population. We also present an interactive version of the heuristic algorithm at <a href="https://gt4sp-genetic-map.shinyapps.io/offspring_haplotype_BT_population/">https://gt4sp-genetic-map.shinyapps.io/offspring_haplotype_BT_population/</a>.</p></div><div id="sec-12" class="subsection"><h3>Preferential pairing profiles</h3><p id="p-21">Considering that all homologs pair during a hexaploid meiosis, there are 15 possible configurations for a chromosomal segment. Let <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-21.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-22.gif"/></noscript></span></span> denote a set containing all 15 possible configurations (see S1 Extended Material and Methods). The posterior probability distribution of the pairing configurations at any position in the genome can be computed using<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <em>n</em> is the number of individuals in the population, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-24.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-25.gif"/></noscript></span></span> for hexaploids) if <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-26.gif"/></noscript></span></span> is consistent with <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-27.gif"/></noscript></span></span>, <em>i.e.</em>, if genotype <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-28.gif"/></noscript></span></span> can be originated from the pairing configuration <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-29.gif"/></noscript></span></span>, 0 otherwise (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>). We also assessed the preferential pairing for specific homolog pairs <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-30.gif"/></noscript></span></span> using<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-32.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-33.gif"/></noscript></span></span>, 0 otherwise. In both situations, to test whether the observed homolog configurations differ from their expected frequencies under random pairing, we used <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-34.gif"/></noscript></span></span> test with <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-35.gif"/></noscript></span></span> to declare significance. We also used the likelihood associated to recombination fractions of single-dose markers to assess preferential pairing, as suggested by <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wu <em>et al.</em> (1992)</a>. Further details of methods are given in the S1 Extended Material and Methods.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-22">The raw DNA sequences are available on the FTP server of the SweetPotatoBase (<a href="http://ftp://ftp.sweetpotatobase.org/ncsu/">ftp://ftp.sweetpotatobase.org/ncsu/</a>). Raw VCF files are available from figshare. File S2 contains SNP locations in <em>I. trifida</em> and <em>I. triloba</em> reference genomes, available in the Sweetpotato Genomics Resource website (<a href="http://sweetpotato.plantbiology.msu.edu">http://sweetpotato.plantbiology.msu.edu</a>). All remaining relevant data are within the manuscript and its Supporting Information files available from figshare. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10255844">https://doi.org/10.25387/g3.10255844</a>.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Genotype calling</h3><p id="p-23">Next-generation sequencing produced several millions of barcoded reads, resulting in approximately 41 million tags, which were aligned against the genomes of two sweetpotato diploid relatives, <em>I. trifida</em> and <em>I. triloba</em> (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wu <em>et al.</em> 2018</a>), resulting in 1,217,917 and 1,163,397 SNPs, respectively. We used the software SuperMASSA (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Serang <em>et al.</em> 2012</a>) to call a total of 442,184 SNPs anchored to <em>I. trifida</em> genome and 438,808 anchored to <em>I. triloba</em> genome. After filtering out low-quality and redundant SNPs (S1 Fig. A), we obtained 38,701 SNPs scored in 311 individuals. SNPs that did not meet the significance threshold (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-36.gif"/></noscript></span></span>) were uniformly distributed across both reference genomes (S2 Fig.). They also presented lower read depth compared with SNPs that passed the threshold, indicating that the distortion observed is rather due to data quality than a biological characteristic of sweetpotato. For all SNPs, we obtained the probability distributions of the dosage calls (exemplified in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). From the total SNPs, 55.5% were classified as simplex (single-dose markers present in one parent) or double-simplex (single-dose markers present in both parents) and 44.5% were classified as multiplex (S1 Fig. B).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/281/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of genotype call of SNP Tf_S1_30010438. (A) Scatter plot of the read counts for the two allelic variants A and G. The axes represent the read counts of both allelic variants. Squared and triangle dots represent parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; respectively, and regular dots represent the offspring. Dashed lines indicate seven possible dosages in a hexaploid individual. The different colors indicate the dosages assigned to the individuals by SuperMASSA. The low number of individuals observed between genotypic classes (gray dots, with genotype probability smaller than 0.8), outlines a data set with low noise, producing a clear classification. The genotypes of both parents were estimated as three doses of the allelic variant A three doses of G. The genotype calling model also considered the expected Mendelian segregation ratio, which under random chromosome pairing is 1:18:99:164:99:18:1. (B) Inferred probability distribution of genotypes for each individual in the offspring. The colored dots correspond to individuals with the same genotypic classes in panel A. Loci where the highest posterior probability was smaller than 0.8 were assigned as missing data (gray dots)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840838757" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example of genotype call of SNP Tf_S1_30010438. (A) Scatter plot of the read counts for the two allelic variants A and G. The axes represent the read counts of both allelic variants. Squared and triangle dots represent parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; respectively, and regular dots represent the offspring. Dashed lines indicate seven possible dosages in a hexaploid individual. The different colors indicate the dosages assigned to the individuals by SuperMASSA. The low number of individuals observed between genotypic classes (gray dots, with genotype probability smaller than 0.8), outlines a data set with low noise, producing a clear classification. The genotypes of both parents were estimated as three doses of the allelic variant A three doses of G. The genotype calling model also considered the expected Mendelian segregation ratio, which under random chromosome pairing is 1:18:99:164:99:18:1. (B) Inferred probability distribution of genotypes for each individual in the offspring. The colored dots correspond to individuals with the same genotypic classes in panel A. Loci where the highest posterior probability was smaller than 0.8 were assigned as missing data (gray dots).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/281/F1.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/281/F1.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/281/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/281/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495210" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Example of genotype call of SNP Tf_S1_30010438. (A) Scatter plot of the read counts for the two allelic variants A and G. The axes represent the read counts of both allelic variants. Squared and triangle dots represent parents ‘Beauregard’ and ‘Tanzania’ respectively, and regular dots represent the offspring. Dashed lines indicate seven possible dosages in a hexaploid individual. The different colors indicate the dosages assigned to the individuals by SuperMASSA. The low number of individuals observed between genotypic classes (gray dots, with genotype probability smaller than 0.8), outlines a data set with low noise, producing a clear classification. The genotypes of both parents were estimated as three doses of the allelic variant A three doses of G. The genotype calling model also considered the expected Mendelian segregation ratio, which under random chromosome pairing is 1:18:99:164:99:18:1. (B) Inferred probability distribution of genotypes for each individual in the offspring. The colored dots correspond to individuals with the same genotypic classes in panel A. Loci where the highest posterior probability was smaller than 0.8 were assigned as missing data (gray dots).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Initial “de novo” map construction</h3><p id="p-25">To build the genetic map, we implemented the R package MAPpoly (<a href="https://mmollina.github.io/MAPpoly/">https://mmollina.github.io/MAPpoly/</a>). The software comprises routines to perform all steps involved in the map construction of autopolyploid species using a combination of pairwise recombination fraction and HMM-based map estimation. First, we obtained the recombination fractions and associated likelihoods for each possible linkage phase for all SNP pairs (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-37.gif"/></noscript></span></span> million pairs). Next, we selected the recombination fractions associated with the most likely linkage phase configuration for each SNP pair and applied the UPGMA hierarchical clustering. We formed 15 distinct clusters representing <em>I. batatas</em> homology groups (S3 Fig.). For the 15 groups, 93.4% of the SNPs were co-located on the same chromosomes in both references and LGs (S1 Table). These co-located SNPs were selected to build the initial map. Since each LG had the majority of their SNPs corresponding to a distinct chromosome in both references, LGs were numbered after the diploid references.</p><p id="p-26">To order the SNPs in each LG, we used the MDS algorithm (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Preedy and Hackett 2016</a>). The reordered recombination fraction matrix is shown in S4 Fig. A. With the proposed MDS order, the parental allelic variants were phased using the procedure presented by <a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Mollinari and Garcia (2019)</a>. The algorithm is based on LOD Scores of pairwise markers as the first source of information to sequentially position the allelic variants in specific homologs. For situations where pairwise analysis had limited power, the algorithm used the likelihood of multiple markers in a Markov chain for the map construction (see Materials and Methods and S1 Extended Material and Methods).</p><p id="p-27">The initial “<em>de novo</em>” multilocus map is presented in S4 Fig. B. The length of the LGs ranges from 723.7 cM in LG 8 to 2,037.0 cM in LG 4, with a total map length of 20,201.8 cM and 32,200 SNPs (average inter-locus distance 0.63 cM), with no considerable gaps between SNPs. Although the MDS algorithm yielded adequate global marker orders for all LGs (S4 Fig. C), the resulting map is considerably large. Two main reasons for this inflation are the misplacement of closely linked SNPs and genotyping errors (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cartwright <em>et al.</em> 2007</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cheema and Dicks 2009</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bilton <em>et al.</em> 2018</a>; <a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>), which will be systematically addressed in the next sections. The alignment of the initial “<em>de novo</em>” map against the reference genomes is shown in S5 Fig. Despite several chromosomal rearrangements, we observed high levels of collinearity between both reference genomes and the estimated map. The collinearity extended in blocks with few megabase pairs (Mb), as in LGs 2 and 7, up to the whole chromosome in LGs 5, 9, 10, 11, 12, 14, and 15. In cases where the collinearity extended through the whole chromosome, we observed sites of suppressed recombination (plateaus in S5 Fig.), possibly indicating the location of centromeric regions.</p></div><div id="sec-17" class="subsection"><h3>Reference genome-assisted map improvement and modeling of genotyping errors</h3><p id="p-28">To reduce the effects of the local marker misplacement in map inflation, we used <em>I. trifida</em> reference genome to propose alternative SNP orders within collinearity blocks and evaluated the likelihood of the resulting maps, keeping the one with the higher likelihood (see Material and Methods and S1 File). We used <em>I. trifida</em> as the primary reference genome because the quality of the assembly is superior and more closely related to <em>I. batatas</em> when compared to <em>I. triloba</em> (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wu <em>et al.</em> 2018</a>). After the order improvement, 97.0% of the <em>I. trifida</em> SNPs present in the map were locally reordered assuming the <em>I. trifida</em> genomic order, (<em>i.e.</em>, the genomic order yielded higher likelihoods for the majority of the cases, see Material and Methods and S1 File). From the remaining <em>I. trifida</em> SNPs, 1.3% were kept in their initial “<em>de novo</em>” order and 1.7% were eliminated since their inclusion caused map inflation higher than 2.00 cM. We then positioned the SNPs private from <em>I. triloba</em> reference genome into the resulting map using the constraints imposed by both genomes (see Material and Methods). The reference genome-assisted reordering resulted in a map with 30,723 SNPs spanning 12,937.3 cM with an average inter-locus distance of 0.42 cM, representing a reduction of 1.sixfold when compared to the initial “<em>de novo</em>” map (S6 Fig., blue map). To address the effects of genotyping errors, we re-estimated the map using the probability distribution of the genotype callings provided by SuperMASSA (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Serang <em>et al.</em> 2012</a>) as prior information in the HMM emission function (<a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>), as implemented in MAPpoly (S6 Fig., green map). In this case, the map length was 4,764.1 cM with an average inter-locus distance of 0.16 cM, representing a map reduction of 2.sevenfold when compared to the reference genome-assisted map.</p></div><div id="sec-18" class="subsection"><h3>Probability distribution of multiallelic genotypes and final map estimation</h3><p id="p-29">For all individuals in the BT offspring, we obtained the conditional probability distribution of the 400 possible hexaploid genotypes in the whole genome given the estimated genetic map. We used the Markovian process to propagate the information throughout each LG. The genotypic probability distribution at each genome position was assessed by using the information of all markers in the LG in all individuals of the full-sib population (S2 Table and S7 Fig.). Next, we removed 13 individuals with inconsistent genotypic profiles (S8 and S9 Figs.) and, keeping the marker order, we re-estimated the final map considering 298 individuals. A comparison between the initial “<em>de novo</em>” and the final maps shows a length reduction of 7.fivefold due to the removal of spurious recombination events through the several steps of map improvement (S6 Fig.).</p><p id="p-30">The final map contains 30,684 SNPs spanning 2,708.4 cM (average inter-locus distance of 0.09 cM), with 60.7% simplex and double-simplex markers, and 39.3% multiplex (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). All homologs showed allelic variations along the LGs indicating that their inheritance pattern can be assessed in the full-sib population. However, several LG segments presented identical composition for a subset of homologs, as shown by the Genotypic Information Content (GIC) (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bourke <em>et al.</em> 2018</a>). In our results, 81.9% of all map positions in ‘Beauregard’ and 77.2% in ‘Tanzania’ had a GIC &gt; 80%, revealing that we can reliably trace back the inheritance of the most homologs from the offspring to the parents (S10 Fig.). A small number of homologs presented an identical allelic composition for certain segments, which is the case, for example, of homologs <em>i</em> and <em>j</em> for the most of LG 2 and <em>l</em> and <em>k</em> across the whole LG 11. The complete map can be interactively browsed at <a href="https://gt4sp-genetic-map.shinyapps.io/bt_map/">https://gt4sp-genetic-map.shinyapps.io/bt_map/</a>. For a selected segment, the browser provides the name of markers, dosages in the parents and the linkage phase configuration of the allelic variants. S2 File shows more map information, including the linkage phase configuration in both parents. S3 and S4 Tables summarize the results of collinearity blocks containing the identical SNP sequences between <em>I. batatas</em> genetic map and <em>I. trifida</em> and <em>I. triloba</em> genomes, respectively. Thirty-nine blocks were aligned to 326.5 Mb of <em>I. trifida</em> genome, covering 96.5% of the <em>I. batatas</em> map (2,614.8 cM), with an average density of one SNP/14.2 kb; 107 blocks were aligned to 258.8 Mb of <em>I. triloba</em> genome, covering 83.1% of the map (2,251.8 cM), with an average density of one SNP/13.4 kb. The averaged genetic to physical map ratios for these regions were of 124.8 kb per cM for <em>I. trifida</em> and 114.9 kb per cM for <em>I. triloba</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495234/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495234/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of sweetpotato genetic map</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/281/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sweetpotato genetic map. For each of the 15 LGs, we present the I. batatas genetic map with its SNPs anchored in both diploid reference genomes. Blue lines connecting the map and reference genomes indicate SNPs shared between I. trifida and I. triloba reference genomes and red lines indicate private SNPs. Above each map, we present a graphical representation of the parental linkage phase configuration of the homology groups for parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019;. Black and gray rectangles indicate two allelic variants in each marker in all 12 parental homologs (6 &#xD7; in &#x2018;Beauregard&#x2019; and 6 &#xD7; in &#x2018;Tanzania&#x2019;). The Genotypic Information Content (GIC), is presented below each homology group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840838757" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sweetpotato genetic map. For each of the 15 LGs, we present the &lt;em&gt;I. batatas&lt;/em&gt; genetic map with its SNPs anchored in both diploid reference genomes. Blue lines connecting the map and reference genomes indicate SNPs shared between &lt;em&gt;I. trifida&lt;/em&gt; and &lt;em&gt;I. triloba&lt;/em&gt; reference genomes and red lines indicate private SNPs. Above each map, we present a graphical representation of the parental linkage phase configuration of the homology groups for parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019;. Black and gray rectangles indicate two allelic variants in each marker in all 12 parental homologs (6 &#xD7; in &#x2018;Beauregard&#x2019; and 6 &#xD7; in &#x2018;Tanzania&#x2019;). The Genotypic Information Content (GIC), is presented below each homology group.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/281/F2.medium.gif" width="295" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/281/F2.medium.gif" width="295" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/281/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/281/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495250" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Sweetpotato genetic map. For each of the 15 LGs, we present the <em>I. batatas</em> genetic map with its SNPs anchored in both diploid reference genomes. Blue lines connecting the map and reference genomes indicate SNPs shared between <em>I. trifida</em> and <em>I. triloba</em> reference genomes and red lines indicate private SNPs. Above each map, we present a graphical representation of the parental linkage phase configuration of the homology groups for parents ‘Beauregard’ and ‘Tanzania’. Black and gray rectangles indicate two allelic variants in each marker in all 12 parental homologs (6 × in ‘Beauregard’ and 6 × in ‘Tanzania’). The Genotypic Information Content (GIC), is presented below each homology group.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Haplotype reconstruction and multivalent formation</h3><p id="p-35">To obtain the haplotype composition of all individuals in the full-sib population, we assessed the conditional probability distribution of the genotypes and appropriately combined them to build 12 profiles (one for each homolog) indicating the probability of inheritance of a particular homolog across the whole chromosomes for all individuals in the BT population (see Materials and Methods). The results can be accessed at <a href="https://gt4sp-genetic-map.shinyapps.io/offspring_haplotype_BT_population/">https://gt4sp-genetic-map.shinyapps.io/offspring_haplotype_BT_population/</a>. By evaluating the recombination points and the homologs involved in the chromosomal exchange, we proposed a heuristic algorithm to obtain chains of homologs linked by recombination events. These chains represent the inference of the meiotic process. The number of parental homologs present in a single homolog of a particular offspring individual indicates the minimum valency of the meiotic configuration involved in its gamete formation (see example in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/281/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of haplotype reconstruction and distribution of meiotic configurations for individual BT05.320, linkage group 1. A) and B) Probability profiles for 12 homologs indicating the segments inherited from parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019;, respectively. The red line indicates the approximated centromeric region obtained using the I. trifida reference genome. The arrows indicate recombination points; C) Recombination signature table indicating the homolog pairs involved in each crossing-over and their position in the map; D) Possible meiotic configuration that originated gametes for individual BT05.320 in &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; and resulting gamete. Each chromosome is represented by one chromatid; E) Representation of the meiotic results as a graph: nodes represent the homologs and the edges represent recombination events between them." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840838757" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Example of haplotype reconstruction and distribution of meiotic configurations for individual BT05.320, linkage group 1. A) and B) Probability profiles for 12 homologs indicating the segments inherited from parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019;, respectively. The red line indicates the approximated centromeric region obtained using the &lt;em&gt;I. trifida&lt;/em&gt; reference genome. The arrows indicate recombination points; C) Recombination signature table indicating the homolog pairs involved in each crossing-over and their position in the map; D) Possible meiotic configuration that originated gametes for individual BT05.320 in &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; and resulting gamete. Each chromosome is represented by one chromatid; E) Representation of the meiotic results as a graph: nodes represent the homologs and the edges represent recombination events between them.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/281/F3.medium.gif" width="440" height="414"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/281/F3.medium.gif" width="440" height="414"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/281/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/281/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495178" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Example of haplotype reconstruction and distribution of meiotic configurations for individual BT05.320, linkage group 1. A) and B) Probability profiles for 12 homologs indicating the segments inherited from parents ‘Beauregard’ and ‘Tanzania’, respectively. The red line indicates the approximated centromeric region obtained using the <em>I. trifida</em> reference genome. The arrows indicate recombination points; C) Recombination signature table indicating the homolog pairs involved in each crossing-over and their position in the map; D) Possible meiotic configuration that originated gametes for individual BT05.320 in ‘Beauregard’ and ‘Tanzania’ and resulting gamete. Each chromosome is represented by one chromatid; E) Representation of the meiotic results as a graph: nodes represent the homologs and the edges represent recombination events between them.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Thus, recombination chains with two homologs indicate the formation of at least a bivalent, three homologous, at least a trivalent, and so forth. For each LG, we calculated the percentage of the maximum number of homologs involved in the same recombination chain (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Most of the configurations involve recombination of two homologs (73.8% in ‘Beauregard’ and 72.8% in ‘Tanzania’), indicating that there was no evidence of a multivalent formation in the majority of gametes formed. We also observed 12.8% of gametes in ‘Beauregard’ and 15.2% in ‘Tanzania’ with haplotype configurations involving three or four parental homologs in a recombination chain (indicating trivalent or quadrivalent formation), and less than 2% of the meiotic configurations with five or six homologs (indicating pentavalent or hexavalent formation; details per LG in S5 Table). We also detected a significant positive linear correlation (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-38.gif"/></noscript></span></span>) between the number of individuals with meiotic configurations originated from multivalent formations and the length of LGs (S11 Fig.).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/281/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of maximum number of homologs connected in the same recombination chain during metaphase I in &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; for all 15 LGs. LG 11 for &#x2018;Tanzania&#x2019; was mostly inconclusive and is not shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840838757" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percentage of maximum number of homologs connected in the same recombination chain during metaphase I in &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; for all 15 LGs. LG 11 for &#x2018;Tanzania&#x2019; was mostly inconclusive and is not shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/281/F4.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/281/F4.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/281/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/281/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495443" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Percentage of maximum number of homologs connected in the same recombination chain during metaphase I in ‘Beauregard’ and ‘Tanzania’ for all 15 LGs. LG 11 for ‘Tanzania’ was mostly inconclusive and is not shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Preferential pairing</h3><p id="p-39">In a hexaploid organism, there are 15 possible pairing configurations for a chromosome segment during the prophase I of meiosis. To assess the level of preferential pairing among homologs, we calculated the probability profile for each of the 15 possible meiotic pairing configurations (S12 Fig.) and 15 possible homolog pairs (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) across all LGs for both parents. We did not observe significant preferential pairing across the whole sweetpotato genome, except LG 2 which showed a low but significant preferential pairing between homologs <em>i</em> and <em>j</em> in parent ’Tanzania’ (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/embed/mml-math-39.gif"/></noscript></span></span>, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). To further ascertain homolog preferential pairing, we evaluated the simplex marker information, which confirmed our preferential pairing findings using the multilocus framework (S13 Fig.).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/281/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probability profiles for 15 homolog pairs in parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; across 15 LGs. The dashed lines in the probability profiles indicate the pairing probability expected under random pairing (). The lower panels indicate  of a  independence test for all possible homolog pairs. Dashed lines indicate . Homologs i and j presented a low, but significant preferential pairing in LG 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840838757" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Probability profiles for 15 homolog pairs in parents &#x2018;Beauregard&#x2019; and &#x2018;Tanzania&#x2019; across 15 LGs. The dashed lines in the probability profiles indicate the pairing probability expected under random pairing (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-40&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The lower panels indicate &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-41&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; of a &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; independence test for all possible homolog pairs. Dashed lines indicate &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Homologs &lt;em&gt;i&lt;/em&gt; and &lt;em&gt;j&lt;/em&gt; presented a low, but significant preferential pairing in LG 2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/281/F5.medium.gif" width="327" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/1/281/F5.medium.gif" width="327" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/281/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/281/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495247" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Probability profiles for 15 homolog pairs in parents ‘Beauregard’ and ‘Tanzania’ across 15 LGs. The dashed lines in the probability profiles indicate the pairing probability expected under random pairing (<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-40.gif"/></noscript></span></span>). The lower panels indicate <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-41.gif"/></noscript></span></span> of a <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-42.gif"/></noscript></span></span> independence test for all possible homolog pairs. Dashed lines indicate <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/281/F5/embed/mml-math-43.gif"/></noscript></span></span>. Homologs <em>i</em> and <em>j</em> presented a low, but significant preferential pairing in LG 2.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-41">We have built the first multilocus integrated genetic map of a hexaploid species, sweetpotato, using our newly developed software MAPpoly. In the map, 90 homologs were densely represented in the 15 homology groups of cultivars ‘Beauregard’ and ‘Tanzania’ exhibiting high collinearity to two closely related diploid sweetpotato genomes, <em>I. trifida</em> and <em>I. triloba</em>. The high collinearity found by using our ultra-dense map corroborates with the high levels of alignment (&gt; 90%) between the diploid genomes and the parent ‘Tanzania’ reported by <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Wu <em>et al.</em> (2018)</a>, suggesting that the diploid genome assemblies could be used as robust references for the hexaploid sweetpotato. We also have constructed the hexaploid haplotypes of all individuals in the offspring, estimating the level of preferential pairing and multivalent formation during the meiotic process at a population level. We used two high-quality reference genomes to improve the quality of our map. However, it is important to notice that in the absence of a reference genome, it is possible to obtain good estimates of the inheritance patterns in the studied population just by using the initial MDS “<em>de novo</em>” order and the probability distribution of the genotype calls in the map construction.</p><p id="p-42">Haplotype inference is the ultimate attainment in linkage analysis since it contains the complete information about genome transmission across generations. The challenge of performing such inference both in parents and offspring, would require new approaches to model the multiallelic transmission in a very complex meiotic scenario. Here we accomplished this by propagating the incomplete information of dosage-based SNPs throughout the LG using a Markov chain. As a result of the efficient combination of multiple SNPs, several LGs displayed fully informative parental haplotypes in most of their length (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and S10 Fig.). Nevertheless, LG11 had two homologs (<em>k</em> and <em>l</em>) carrying the same allelic variations across its entire length, which leads us to speculate that these two homologs were formed by nondisjunction of sister chromatids in meiosis II in one of Tanzania’s parent resulting in an unreduced gamete transmitted to the next generation (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burnham 1962</a>). Even though in some cases where not all homologs could be distinguished, we estimated their probability distribution, which can be readily used in further genetic studies, such as quantitative trait loci mapping performed for the BT population (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pereira <em>et al.</em> 2019</a>). Moreover, our multipoint method mitigates the effect of a possible limited sample size presented in some studies (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hackett <em>et al.</em> 1998</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Ripol <em>et al.</em> 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Doerge and Craig 2000</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Luo <em>et al.</em> 2001</a>) by using the propagation of the information of multiple markers. In doing so, the estimates of the recombination fractions are obtained considering the structure of the whole homology group, rather pairwise marker combinations. We also investigated how the assembled parental homologs were transmitted to their offspring by assessing the probability distribution of the multiallelic genotypes across the whole genome for all individuals in the mapping population. Based on the inferred probability distributions, we presented a comprehensive probabilistic reconstruction of the haplotypes of all individuals in a full-sib hexaploid population. We found that 15% of the offspring showed the evidence of multivalent formation, <em>i.e.</em>, offspring homologs containing more than two parental homologs. This leads to intra-homolog variation, which could not be due to exclusive bivalent pairing.</p><p id="p-43">Multivalent configurations often cause faulty chromosomal segregation leading to aneuploidy (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arana and Nicklas 1992</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hollister 2015</a>). Such a phenomenon could cause unbalanced gametes, and consequently the production of pollen and seeds with low viability, posing a significant hindrance to a stable genomic transmission throughout generations in polyploids (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mwathi <em>et al.</em> 2017</a>). Multivalents are usually observed in high numbers in recently formed polyploids, as in the case of the synthetic autopolyploid <em>Arabidopsis thaliana</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Santos <em>et al.</em> 2003</a>). Most of the established autopolyploids, however, show considerably fewer multivalents. In a survey involving 93 autopolyploid species, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ramsey and Schemske (2002)</a> showed that the average frequency of bivalents was 63.7% whereas the average frequency of quadrivalents was 26.8%, which are significantly different from the theoretically expected (1 × two bivalents (II + II) to 2 × one quadrivalent VI) (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Sybenga 1975</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jackson and Casey 1982</a>). For hexaploids, the theoretical proportion of bivalent to multivalent configurations is 1 × three bivalents (II + II + II) to 6 × one quadrivalent plus one bivalent (IV + II) to 8 × one hexavalent (VI) (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Jackson and Casey 1982</a>). However, in our work, the number of multivalent signatures observed was notably low, whereas the number of bivalents was relatively high (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). These results corroborate the previous cytological study by Magoon and co-authors (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Magoon <em>et al.</em> 1970</a>), who found similar levels of multivalent configurations in sweetpotato pachytene cells. Nevertheless, our results provide population-level evidence to the prevalence of bivalent configurations in sweetpotato meiosis.</p><p id="p-44">In a scenario of scarce multivalent formation, the double reduction (DR) phenomenon becomes a rare event. The DR of a given locus is a consequence of a series of cytological events: multivalent formation, crossing-over between the locus and centromere, and migration of the the duplicated segment carrying the locus to the same pole of the cell at anaphases I and II (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mather 1936</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Butruille and Boiteux 2000</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stift <em>et al.</em> 2008</a>). Thus, multivalent formation is a necessary but not sufficient condition for the occurrence of DR. Consequently, the low frequency of multivalent formation observed in this work indicates that the occurrence of DR is a rare phenomenon in the BT population. Although we did not take into account DR events during the construction of the genetic map, it would have little impact on our results since the algorithm used here was found to be robust under low levels of multivalent formation (<a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">Mollinari and Garcia 2019</a>). Nevertheless, although a rare even, DR could generate transgressive genotypes that can be inherited through the next generations.</p><p id="p-45">All sweetpotato genetic maps publish to date (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ukoskit and Thompson 1997</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Kriegner <em>et al.</em> 2003</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Cervantes-Flores <em>et al.</em> 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ai-xian <em>et al.</em> 2010</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Zhao <em>et al.</em> 2013</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Monden and Tahara 2017</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Shirasawa <em>et al.</em> 2017</a>) have acknowledged the hexasomic segregation in sweetpotato. However, none of them systematically characterized this phenomenon using the information of multiple markers assembled in complete hexaploid homology groups. Here we used the multilocus map to assess this information generating preferential pairing profiles (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). We showed that sweetpotato inheritance is vastly autopolyploid-like and random chromosome pairing enables recombination between all homologous across generations. This results are in agreement with studies based on nuclear and chloroplast phylogenies (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Roullier <em>et al.</em> 2013</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Muñoz-Rodríguez <em>et al.</em> 2018</a>) which demonstrated the autopolyploid origin of sweetpotato.</p><p id="p-46">A variety of intrachromosomal rearrangements were observed between <em>I. batatas</em> map and <em>I. trifida</em> and <em>I. triloba</em> genomes. Rearrangements mapped to both diploid references, such as the chromosome inversion at the beginning of LG 6 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), represent structural changes exclusive to <em>I. batatas</em>. While the occurrence of such rearrangements could cause instability to meiotic process at some point of the evolutionary history of a polyploid species (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lenormand <em>et al.</em> 2016</a>), given the high level of bivalent signatures and the stable hexasomic segregation observed in our analysis, we concluded that these structural changes became fixed and did not cause major disturbances to the meiotic process in sweetpotato.</p><p id="p-47">More than a linear order of genetic markers positioned in LGs, a genetic map is a statement about the inheritance pattern involved in the transmission of the genome from parents and their offspring. A full characterization of this process can be achieved if the mapping method allows the estimation of haplotypes in both generations. In diploid organisms, a hidden Markov model was proposed by Lander and Green for linkage analysis of multiple markers (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lander and Green 1987</a>). Later on, several studies paved the way for a linkage map construction and haplotype inference in autotetraploid species (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hackett and Broadfoot 2003</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Leach <em>et al.</em> 2010</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zheng <em>et al.</em> 2016</a>). However, for complex polyploids, the map construction was restricted mostly to two-point marker analysis. We present the first integrated multilocus genetic map with fully phased haplotypes for both parents and offspring in a complex polyploid and, accompanied with it, the fully developed statistical methods and computational tool MAPpoly. This opens the door for detailed genetic analysis in complex polyploid species in general.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">This research was supported by Bill &amp; Melinda Gates Foundation [grant number OPP1052983]. Research at the International Potato Center (CIP) was undertaken as part of the consultative group on international agricultural research (CGIAR)-Research Program on Roots, Tubers and Bananas (RTB) which is supported by CGIAR Fund Donors (<a href="http://www.cgiar.org/about-us/our-funders/">http://www.cgiar.org/about-us/our-funders/</a>). The authors thank the anonymous reviewers for their insightful comments and valuable suggestions. We acknowledge the CIP sweetpotato breeding team in Peru for running field experiments.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-49">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10255844">https://doi.org/10.25387/g3.10255844</a>.</p></li><li class="fn" id="fn-5"><p id="p-50"><em>Communicating editor: R.W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-08-12"><span class="received-label">Received </span>August 12, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-12"><span class="accepted-label">Accepted </span>November 12, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Mollinari <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ai-xian</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Qing-chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Qing-mei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Li-ming</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Hong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Establishment of Molecular Linkage Maps Using SRAP Markers in Sweet Potato.</span> <abbr class="cit-jnl-abbrev">Zuo Wu Xue Bao</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1286</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZuo%2BWu%2BXue%2BBao%26rft.volume%253D36%26rft.spage%253D1286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.2" data-doi="10.1139/G07-056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Jackson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">McIntyre</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Construction of a genetic linkage map for Saccharum officinarum incorporating both simplex and duplex markers to increase genome coverage.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">742</span>–<span class="cit-lpage">756</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/G07-056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DAitken%26rft.auinit1%253DK.%2BS.%26rft.volume%253D50%26rft.issue%253D8%26rft.spage%253D742%26rft.epage%253D756%26rft.atitle%253DConstruction%2Bof%2Ba%2Bgenetic%2Blinkage%2Bmap%2Bfor%2BSaccharum%2Bofficinarum%2Bincorporating%2Bboth%2Bsimplex%2Band%2Bduplex%2Bmarkers%2Bto%2Bincrease%2Bgenome%2Bcoverage.%26rft_id%253Dinfo%253Adoi%252F10.1139%252FG07-056%26rft_id%253Dinfo%253Apmid%252F17893734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/G07-056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17893734&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.3" data-doi="10.1007/BF00582834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arana</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Nicklas</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Orientation and segregation of a micromanipulated multivalent: Familiar principles, divergent outcomes.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00582834</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DArana%26rft.auinit1%253DP.%26rft.volume%253D101%26rft.issue%253D7%26rft.spage%253D399%26rft.epage%253D412%26rft.atitle%253DOrientation%2Band%2Bsegregation%2Bof%2Ba%2Bmicromanipulated%2Bmultivalent%253A%2Bfamiliar%2Bprinciples%252C%2Bdivergent%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00582834%26rft_id%253Dinfo%253Apmid%252F1618023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00582834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1618023&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-confproc" id="cit-10.1.281.4"><div class="cit-metadata unstructured">Austin, D. F., 1988 Exploration, Maintenance and Utilization of Sweet Potato Genetic Resources.No Title. In <em>Report of the First Sweet Potato Planning Conference</em>, pp. 27–59, Lima, Peru, International Potato Center.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.5" data-doi="10.1534/genetics.117.300627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bilton</span>,  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Schofield</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Black</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Chagné</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Wilcox</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accounting for Errors in Low Coverage High-Throughput Sequencing Data When Constructing Genetic Maps Using Biparental Outcrossed Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">76</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300627%26rft_id%253Dinfo%253Apmid%252F29487138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA5LzEvNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.6" data-doi="10.1007/s00122-018-3132-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. W.</span> <span class="cit-name-surname">Gitonga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Visser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Krens</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multi-environment QTL analysis of plant and flower morphological traits in tetraploid rose.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">2055</span>–<span class="cit-lpage">2069</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3132-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D2055%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3132-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3132-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.281.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burnham</span>,  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-source">Discussions in cytogenetics</span>, <span class="cit-publ-name">Burgess Publishing</span>, <span class="cit-publ-loc">Mineapolis</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.8" data-doi="10.1073/pnas.100101097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butruille</span>,  <span class="cit-name-given-names">D. V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Boiteux</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Selection-mutation balance in polysomic tetraploids: impact of double reduction and gametophytic selection on the frequency and subchromosomal localization of deleterious mutations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">6608</span>–<span class="cit-lpage">6613</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.100101097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.100101097%26rft_id%253Dinfo%253Apmid%252F10823890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTIvNjYwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.9" data-doi="10.1534/genetics.106.063982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cartwright</span>,  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Troggio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Velasco</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gutin</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic mapping in the presence of genotyping errors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2521</span>–<span class="cit-lpage">2527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.063982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.063982%26rft_id%253Dinfo%253Apmid%252F17277374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzI1MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.10" data-doi="10.1007/s11032-007-9150-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cervantes-Flores</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Yencho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kriegner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Pecota</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Faulk</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Development of a genetic linkage map and identification of homologous linkage groups in sweetpotato using multiple-dose AFLP markers.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">532</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-007-9150-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D21%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-007-9150-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-007-9150-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.11" data-doi="10.1093/bib/bbp045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheema</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dicks</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Computational approaches and software tools for genetic linkage map estimation in plants.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">608</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbp045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbp045%26rft_id%253Dinfo%253Apmid%252F19933208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbp045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19933208&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.12" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.13" data-doi="10.1073/pnas.97.14.7951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>,  <span class="cit-name-given-names">R. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Craig</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Model selection for quantitative trait locus analysis in polyploids.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7951</span>–<span class="cit-lpage">7956</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.97.14.7951</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D97%26rft.issue%253D14%26rft.spage%253D7951%26rft.epage%253D7956%26rft.atitle%253DModel%2Bselection%2Bfor%2Bquantitative%2Btrait%2Blocus%2Banalysis%2Bin%2Bpolyploids%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.14.7951%26rft_id%253Dinfo%253Apmid%252F10884425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTQvNzk1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-10.1.281.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">FAO</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <a href="http://www.fao.org/faostat/en">http://www.fao.org/faostat/en</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.15" data-doi="10.1111/j.1469-1809.1941.tb02268.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1941</span> <span class="cit-article-title">The theoretical consequences of polyploid inheritance for the mid style form of Lythrum salicaria.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1941.tb02268.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D11%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1941.tb02268.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1941.tb02268.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.281.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Quantitative genetics and breeding methods in autopolyploids plants</span>, <span class="cit-publ-name">INRA</span>, <span class="cit-publ-loc">Paris</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.17"><div class="cit-metadata unstructured">Gemenet, D. C., G. da Silva Pereira, B. De Boeck, J. C. Wood, M. Mollinari, <em>et al.</em>, 2019 Quantitative trait loci and differential gene expression analyses reveal the genetic basis for negatively associated <em>β</em>-carotene and starch content in hexaploid sweetpotato [<em>Ipomoea batatas</em> (l.) lam.]. Theor Appl Genet <a href="https://link.springer.com/article/10.1007/s00122-019-03437-7">https://link.springer.com/article/10.1007/s00122-019-03437-7</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.18" data-doi="10.1111/j.1601-5223.1965.tb01981.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>,  <span class="cit-name-given-names">Å.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Gadd</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Mutations and crop improvement III. <em>Ipomoea batatas</em> (L.) Poir. (Convolvulaceae).</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-5223.1965.tb01981.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.volume%253D53%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1965.tb01981.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.1965.tb01981.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.19" data-doi="10.1017/S0016672398003188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Bradshaw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. C.</span> <span class="cit-name-surname">Meyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Mcnicol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Milbourne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Waugh</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Linkage analysis in tetraploid species: a simulation study.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672398003188</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D71%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672398003188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672398003188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.20" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Broadfoot</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.21" data-doi="10.1111/nph.12939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>,  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Polyploidy: Adaptation to the genomic environment.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D205%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12939%26rft_id%253Dinfo%253Apmid%252F25729801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25729801&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.22" data-doi="10.2307/2443057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">R. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Casey</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Cytogenetic Analyses of Autopolyploids: Models and Methods for Triploids to Octoploids.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/j.1537-2197.1982.tb13284.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D69%26rft.spage%253D487%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2443057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2443057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982NP89700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.23" data-doi="10.1023/A:1018394410659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.-B.</span> <span class="cit-name-surname">Zeng</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1018394410659</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DJiang%26rft.auinit1%253DC.%26rft.volume%253D101%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D58%26rft.atitle%253DMapping%2Bquantitative%2Btrait%2Bloci%2Bwith%2Bdominant%2Band%2Bmissing%2Bmarkers%2Bin%2Bvarious%2Bcrosses%2Bfrom%2Btwo%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1018394410659%26rft_id%253Dinfo%253Apmid%252F9465409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1018394410659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9465409&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071700000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.24" data-doi="10.1023/A:1022870917230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kriegner</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Cervantes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Burg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. O.</span> <span class="cit-name-surname">Mwanga</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A genetic linkage map of sweetpotato [<em>Ipomoea batatas</em>(l.) lam.] based on aflp markers.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1022870917230</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D11%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1022870917230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1022870917230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.25" data-doi="10.1073/pnas.84.8.2363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">E. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Green</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Construction of multilocus genetic linkage maps in humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">2363</span>–<span class="cit-lpage">2367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.84.8.2363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D84%26rft.issue%253D8%26rft.spage%253D2363%26rft.epage%253D2367%26rft.atitle%253DConstruction%2Bof%2Bmultilocus%2Bgenetic%2Blinkage%2Bmaps%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.8.2363%26rft_id%253Dinfo%253Apmid%252F3470801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NC84LzIzNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.26" data-doi="10.1073/pnas.0908477107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leach</span>,  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Kearsey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Luo</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multilocus tetrasomic linkage analysis using hidden markov chain model.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">4270</span>–<span class="cit-lpage">4274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0908477107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0908477107%26rft_id%253Dinfo%253Apmid%252F20142473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzkvNDI3MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.27" data-doi="10.1098/rstb.2015.0124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Engelstädter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Johnston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Wijnker</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Haag</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolutionary mysteries in meiosis.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">371</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2016.0001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft.volume%253D371%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2015.0124%26rft_id%253Dinfo%253Apmid%252F26833842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2015.0124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26833842&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.281.28" data-doi="10.1007/978-1-4020-9475-0_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loebenstein</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Origin, Distribution and Economic Importance</span>, pp. <span class="cit-fpage">9</span>–<span class="cit-lpage">12</span> in <span class="cit-source">The Sweetpotato</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Loebenstein</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Thottappilly</span></span>. <span class="cit-publ-name">Springer Netherlands</span>, <span class="cit-publ-loc">Dordrecht</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4020-9475-0_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BSweetpotato%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4020-9475-0_2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4020-9475-0_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Hackett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Bradshaw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">McNicol</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Milbourne</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Construction of a genetic linkage map in tetraploid species using molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1369</span>–<span class="cit-lpage">1385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLuo%26rft.auinit1%253DZ.%2BW.%26rft.volume%253D157%26rft.issue%253D3%26rft.spage%253D1369%26rft.epage%253D1385%26rft.atitle%253DConstruction%2Bof%2Ba%2BGenetic%2BLinkage%2BMap%2Bin%2BTetraploid%2BSpecies%2BUsing%2BMolecular%2BMarkers%26rft_id%253Dinfo%253Apmid%252F11238421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny8zLzEzNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.30" data-doi="10.1073/pnas.0304482101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Kearsey</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Theoretical basis for genetic linkage analysis in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">7040</span>–<span class="cit-lpage">7045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0304482101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0304482101%26rft_id%253Dinfo%253Apmid%252F15100415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE4LzcwNDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.31" data-doi="10.1007/BF00285415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magoon</span>,  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Krishnan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Vijaya Bai</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Cytological evidence on the origin of sweet potato.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00285415</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D40%26rft.spage%253D360%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00285415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00285415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.32" data-doi="10.1007/BF02982683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">Segregation and linkage in autotetraploids.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02982683</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D32%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02982683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02982683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200612400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Linkage analysis and haplotype phasing in experimental autopolyploid populations with high ploidy level using hidden markov models. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">3297</span>–<span class="cit-lpage">3314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D9%26rft.spage%253D3297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.281.34" data-doi="10.1007/978-1-4939-1966-6_17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Serang</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative SNP Genotyping of Polyploids with MassARRAY and Other Platforms.</span> Methods Mol Biol. <span class="cit-vol">1245</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-1966-6_17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-1966-6_17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-1966-6_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.35" data-doi="10.1270/jsbbs.16142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monden</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tahara</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic linkage analysis using DNA markers in sweetpotato.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1270/jsbbs.16142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D67%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.16142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.16142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.36" data-doi="10.1016/j.cub.2018.03.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Rodríguez</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Carruthers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Wood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Weitemier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Reconciling Conflicting Phylogenies in the Origin of Sweet Potato and Dispersal to Polynesia.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1246</span>–<span class="cit-lpage">1256.e12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2018.03.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D28%26rft.spage%253D1246%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2018.03.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2018.03.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.37" data-doi="10.1007/s00122-016-2850-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mwathi</span>,  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gupta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Atri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Banga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Batley</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Segregation for fertility and meiotic stability in novel Brassica allohexaploids.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2850-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D767%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2850-8%26rft_id%253Dinfo%253Apmid%252F000398737400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2850-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000398737400&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.38" data-doi="10.1101/622951"><div class="cit-metadata"><cite>Pereira, G. d. S., D. C. Gemenet, M. Mollinari, B. A. Olukolu, J. C. Wood, <em>et al.</em>, 2019 Multiple QTL mapping in autopolyploids: a random-effect model approach with application in a hexaploid sweetpotato full-sib population. bioRxiv p.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/622951</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1101%252F622951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI2MjI5NTF2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.39" data-doi="10.1007/s00122-016-2761-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preedy</span>,  <span class="cit-name-given-names">K. F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Hackett</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A rapid marker ordering approach for high-density genetic linkage maps in experimental autotetraploid populations using multidimensional scaling.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2117</span>–<span class="cit-lpage">2132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2761-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2117%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2761-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2761-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.40" data-doi="10.1109/5.18626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabiner</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A tutorial on hidden markov models and selected applications in speech recognition.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/5.18626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D77%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.18626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/5.18626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.41" data-doi="10.1146/annurev.ecolsys.33.010802.150437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsey</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Schemske</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Neopolyploidy in Flowering Plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.33.010802.150437</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.33.010802.150437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.33.010802.150437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000180007000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.42" data-doi="10.1016/S0378-1119(99)00218-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ripol</span>,  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A. G. D.</span> <span class="cit-name-surname">Silva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Statistical aspects of genetic mapping in autopolyploids.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">235</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0378-1119(99)00218-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DRipol%26rft.auinit1%253DM.%2BI.%26rft.volume%253D235%26rft.issue%253D1-2%26rft.spage%253D31%26rft.epage%253D41%26rft.atitle%253DStatistical%2Baspects%2Bof%2Bgenetic%2Bmapping%2Bin%2Bautopolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900218-8%26rft_id%253Dinfo%253Apmid%252F10415330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00218-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10415330&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081851700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.43" data-doi="10.1371/journal.pone.0062707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roullier</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Duputié</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wennekes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Benoit</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Fernández Bringas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Disentangling the Origins of Cultivated Sweet Potato (<em>Ipomoea batatas</em> (L.) Lam.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e62707</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0062707</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62707%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062707%26rft_id%253Dinfo%253Apmid%252F23723970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23723970&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Alfaro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Armstrong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C. H.</span> <span class="cit-name-surname">Franklin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Jones</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Partial Diploidization of Meiosis in Autotetraploid.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSantos%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D1533%26rft.epage%253D1540%26rft.atitle%253DPartial%2BDiploidization%2Bof%2BMeiosis%2Bin%2BAutotetraploid%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F14668400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS8zLzE1MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.45" data-doi="10.1371/journal.pone.0030906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serang</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mollinari</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient exact maximum a posteriori computation for bayesian snp genotyping in polyploids.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e30906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0030906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSerang%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30906%26rft.epage%253De30906%26rft.atitle%253DEfficient%2Bexact%2Bmaximum%2Ba%2Bposteriori%2Bcomputation%2Bfor%2Bbayesian%2BSNP%2Bgenotyping%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030906%26rft_id%253Dinfo%253Apmid%252F22363513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363513&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.46"><div class="cit-metadata"><cite>Shiotani, I. and T. Kawase, 1987 Synthetic hexaploids derived from wild species related to sweet potato. Japan. J. Breed. 37: 367–376.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1270/jsbbs1951.37.367</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.47" data-doi="10.1038/srep44207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tanaka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Takahata</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Cao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A high-density SNP genetic map consisting of a complete set of homologous groups in autohexaploid sweetpotato (<em>Ipomoea batatas</em>).</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">44207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep44207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D44207%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep44207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep44207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.48" data-doi="10.1534/genetics.107.085027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stift</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Berenos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kuperus</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">van Tienderen</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Segregation Models for Disomic, Tetrasomic and Intermediate Inheritance in Tetraploids: A General Procedure Applied to Rorippa (Yellow Cress) Microsatellite Data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">2113</span>–<span class="cit-lpage">2123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.085027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085027%26rft_id%253Dinfo%253Apmid%252F18689891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS80LzIxMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.281.49" data-doi="10.1007/978-3-642-80960-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sybenga</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-source">Meiotic configurations</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-80960-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMeiotic%2Bconfigurations%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-80960-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-80960-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.50" data-doi="10.21273/JASHS.122.6.822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ukoskit</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. G.</span> <span class="cit-name-surname">Thompson</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Autopolyploidy <em>vs.</em> allopolyploidy and low-density randomly amplified polymorphic DNA linkage maps of sweetpotato.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">822</span>–<span class="cit-lpage">828</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.21273/JASHS.122.6.822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D122%26rft.spage%253D822%26rft_id%253Dinfo%253Adoi%252F10.21273%252FJASHS.122.6.822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21273/JASHS.122.6.822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.51" data-doi="10.1007/s00122-017-2974-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Geest</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Bourke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Marasek-Ciolakowska</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An ultra-dense integrated linkage map for hexaploid chrysanthemum enables multi-allelic qtl analysis.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2527</span>–<span class="cit-lpage">2541</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2974-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2527%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2974-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2974-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.52" data-doi="10.3389/fpls.2018.01166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wadl</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Olukolu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Branham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Jarret</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Yencho</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic Diversity and Population Structure of the USDA Sweetpotato (<em>Ipomoea batatas</em>) Germplasm Collections Using GBSpoly.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.01166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D1166%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.01166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.01166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.53" data-doi="10.1007/BF00224274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Burnquist</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Tew</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Moore</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The detection and estimation of linkage in polyploids using single-dose restriction fragments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">294</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00224274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00224274%26rft_id%253Dinfo%253Apmid%252F24202510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00224274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24202510&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HB49800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.54" data-doi="10.1038/s41467-018-06983-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Lau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Cao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Hamilton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sun</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome sequences of two diploid wild relatives of cultivated sweetpotato reveal targets for genetic improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">4580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-06983-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D4580%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-06983-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-06983-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.55" data-doi="10.1038/s41477-017-0002-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-H.</span> <span class="cit-name-surname">Moeinzadeh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kuhl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Helmuth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Xiao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Haplotype-resolved sweet potato genome traces back its hexaploidization history.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41477-017-0002-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D3%26rft.spage%253D696%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41477-017-0002-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41477-017-0002-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.56" data-doi="10.1007/s11032-013-9908-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Jie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genetic linkage map based on AFLP and SSR markers and mapping of QTL for dry-matter content in sweetpotato.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">820</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-013-9908-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D32%26rft.spage%253D807%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-013-9908-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-013-9908-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.281.57" data-doi="10.1534/genetics.115.185579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorrips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Hackett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ho</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Probabilistic multilocus haplotype reconstruction in outcrossing tetraploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.185579</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185579%26rft_id%253Dinfo%253Apmid%252F26920758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvMTE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.281.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zielinski</span>,  <span class="cit-name-given-names">M.-L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. M.</span> <span class="cit-name-surname">Scheid</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiosis in Polyploid Plants</span>, pp. <span class="cit-fpage">33</span>–<span class="cit-lpage">55</span> in <span class="cit-source">Poliploidy and Genome Evolution</span>, <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/281.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/267" title="Rapid Degradation of Caenorhabditis elegans Proteins at Single-Cell Resolution with a Synthetic Auxin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/293" title="Whole Genome Analysis of a Single Scottish Deerhound Dog Family Provides Independent Corroboration That a SGK3 Coding Variant Leads to Hairlessness" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/281" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xrnGgL2tLN-daKU8HxaJR4Lw-Uq7QlrFDxy5kCzbxMU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/281" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/281" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/281" class="active" data-icon-position="" data-hide-link-title="0">Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495261" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-KnqrKG_vOcscqI6Xng73rNXelQrlypup-dQPaVfus4A" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/281.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/281" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EbtHmJLAxovn0cLDgDzsn5SiZs8sq6Xo8u5dnT9VfVU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/281" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/281.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495261" id="node495261--2" data-pisa="ggg;10/1/281" data-pisa-master="ggg;g3.119.400620" data-apath="/ggg/10/1/281.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcelo Mollinari</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4143-8909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bode A. Olukolu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme da S. Pereira</span>, <span class="highwire-citation-author" data-delta="3">Awais Khan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4901-1694" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dorcus Gemenet</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6583-0628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>G. Craig Yencho</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhao-Bang Zeng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">281-292;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400620 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495261--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcelo Mollinari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcelo%2BMollinari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mollinari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcelo%2BMollinari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcelo Mollinari</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mmollin@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">mmollin@ncsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bode A. Olukolu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Entomology and Plant Pathology, University of Tennessee, Knoxville, Tennessee,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bode%2BA.%2BOlukolu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olukolu%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABode%2BA.%2BOlukolu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4143-8909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bode A. Olukolu</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guilherme da S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2Bda%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2Bda%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme da S. Pereira</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Awais Khan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Pathology and Plant-Microbe Biology Section, Cornell University, Geneva, New York, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Awais%2BKhan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAwais%2BKhan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dorcus Gemenet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>International Potato Center, ILRI Campus, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorcus%2BGemenet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gemenet%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADorcus%2BGemenet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4901-1694" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dorcus Gemenet</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">G. Craig Yencho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BCraig%2BYencho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yencho%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AG.%2BCraig%2BYencho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6583-0628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for G. Craig Yencho</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Zhao-Bang Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao-Bang%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhao-Bang%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhao-Bang Zeng</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/281" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/281&amp;title=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/281&amp;title=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/281&amp;title=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/281&amp;text=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/281&amp;title=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/281&amp;t=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/281&amp;title=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/281&amp;title=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495261/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495261" id="node495261--3" data-pisa="ggg;10/1/281" data-pisa-master="ggg;g3.119.400620" data-apath="/ggg/10/1/281.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Unraveling the Hexaploid Sweetpotato Inheritance Using Ultra-Dense Multilocus Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcelo Mollinari</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4143-8909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bode A. Olukolu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme da S. Pereira</span>, <span class="highwire-citation-author" data-delta="3">Awais Khan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4901-1694" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dorcus Gemenet</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6583-0628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>G. Craig Yencho</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhao-Bang Zeng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">281-292;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400620 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495261--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcelo Mollinari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcelo%2BMollinari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mollinari%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcelo%2BMollinari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7001-8498" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcelo Mollinari</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mmollin@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">mmollin@ncsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bode A. Olukolu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Entomology and Plant Pathology, University of Tennessee, Knoxville, Tennessee,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bode%2BA.%2BOlukolu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olukolu%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABode%2BA.%2BOlukolu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4143-8909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bode A. Olukolu</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guilherme da S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2Bda%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2Bda%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme da S. Pereira</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Awais Khan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Pathology and Plant-Microbe Biology Section, Cornell University, Geneva, New York, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Awais%2BKhan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAwais%2BKhan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dorcus Gemenet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>International Potato Center, ILRI Campus, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorcus%2BGemenet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gemenet%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADorcus%2BGemenet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4901-1694" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dorcus Gemenet</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">G. Craig Yencho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BCraig%2BYencho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yencho%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AG.%2BCraig%2BYencho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6583-0628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for G. Craig Yencho</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Zhao-Bang Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center, North Carolina State University, Raleigh, North Carolina,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticultural Science, North Carolina State University, Raleigh, North Carolina,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao-Bang%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhao-Bang%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhao-Bang Zeng</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=281&amp;rft.epage=292&amp;rft.atitle=Unraveling%20the%20Hexaploid%20Sweetpotato%20Inheritance%20Using%20Ultra-Dense%20Multilocus%20Mapping&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mollinari%2C+Marcelo&amp;rft.au=Olukolu%2C+Bode+A.&amp;rft.au=Pereira%2C+Guilherme+da+S.&amp;rft.au=Khan%2C+Awais&amp;rft.au=Gemenet%2C+Dorcus&amp;rft.au=Yencho%2C+G.+Craig&amp;rft.au=Zeng%2C+Zhao-Bang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495261/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495261/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495261/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495261/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495261/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495261/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495261/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495261/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495261/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495261/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495261/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495261/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/281" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/281.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/281.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/281.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
