<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/225" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/225.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/225.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/225.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/225.atom" />
<meta name="HW.pisa" content="ggg;10/1/225" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Single, Shared Triploidy in Three Species of Parasitic Nematodes" />
<meta name="DC.Identifier" content="10.1534/g3.119.400650" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Schoonmaker et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The root-knot nematodes of the genus Meloidogyne are important and damaging parasites capable of infecting most flowering plants. Within this genus, several species of the Meloidogyne incognita group show evidence of paleopolyploidy in their genomes. We used our software tool POInT, the Polyploidy Orthology Inference Tool, to phylogenetically model the gene losses that followed that polyploidy. These models, and simulations based on them, show that three of these species ( M. incognita , M. arenaria and M. javanica ) descend from a single common hybridization event that yielded triplicated genomes with three distinguishable subgenomes. While one of the three subgenomes shows elevated gene loss rates relative to the other two, this subgenome does not show elevated sequence divergence. In all three species, ancestral loci where two of the three gene copies have been lost are less likely to have orthologs in Caenorhabditis elegans that are lethal when knocked down than are ancestral loci with surviving duplicate copies." />
<meta name="DC.Contributor" content="Ashley Schoonmaker" />
<meta name="DC.Contributor" content="Yue Hao" />
<meta name="DC.Contributor" content="David McK. Bird" />
<meta name="DC.Contributor" content="Gavin C. Conant" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Single, Shared Triploidy in Three Species of Parasitic Nematodes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The root-knot nematodes of the genus &lt;i&gt;Meloidogyne&lt;/i&gt; are important and damaging parasites capable of infecting most flowering plants. Within this genus, several species of the &lt;i&gt;Meloidogyne incognita&lt;/i&gt; group show evidence of paleopolyploidy in their genomes. We used our software tool POInT, the Polyploidy Orthology Inference Tool, to phylogenetically model the gene losses that followed that polyploidy. These models, and simulations based on them, show that three of these species (&lt;i&gt;M. incognita&lt;/i&gt;, &lt;i&gt;M. arenaria&lt;/i&gt; and &lt;i&gt;M. javanica&lt;/i&gt;) descend from a single common hybridization event that yielded triplicated genomes with three distinguishable subgenomes. While one of the three subgenomes shows elevated gene loss rates relative to the other two, this subgenome does not show elevated sequence divergence. In all three species, ancestral loci where two of the three gene copies have been lost are less likely to have orthologs in &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; that are lethal when knocked down than are ancestral loci with surviving duplicate copies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/225" />
<meta name="citation_id" content="10/1/225" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/225" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/225.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/225.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/225.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400650" />
<meta name="citation_pmid" content="31694855" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="225" />
<meta name="citation_lastpage" content="233" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ashley Schoonmaker" />
<meta name="citation_author_institution" content="Bioinformatics Research Center," />
<meta name="citation_author" content="Yue Hao" />
<meta name="citation_author_institution" content="Bioinformatics Research Center," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5065-8906" />
<meta name="citation_author" content="David McK. Bird" />
<meta name="citation_author_institution" content="Bioinformatics Research Center," />
<meta name="citation_author_institution" content="Department of Plant Pathology," />
<meta name="citation_author" content="Gavin C. Conant" />
<meta name="citation_author_institution" content="Bioinformatics Research Center," />
<meta name="citation_author_institution" content="Department of Biological Sciences, and" />
<meta name="citation_author_institution" content="Program in Genetics, North Carolina State University, Raleigh, NC, 27695" />
<meta name="citation_author_email" content="gconant@ncsu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8677-4933" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. H. Zhang;citation_author=Z. Zhang;citation_title=Gapped Blast and Psi-Blast: A new-generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=S. Álvarez-Ortega;citation_author=J. A. Brito;citation_author=S. A. Subbotin;citation_title=Multigene phylogeny of root-knot nematodes and molecular characterization of Meloidogyne nataliei Golden, Rose &amp; Bird, 1981 (Nematoda: Tylenchida).;citation_pages=11788;citation_volume=9;citation_year=2019;citation_doi=10.1038/s41598-019-48195-0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Bekaert;citation_author=G. C. Conant;citation_title=Copy number alterations among mammalian enyzmes cluster in the metabolic network.;citation_volume=28;citation_year=2011;citation_pmid=21051442;citation_doi=10.1093/molbev/msq296" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Birchler;citation_author=R. A. Veitia;citation_title=Gene balance hypothesis: connecting issues of dosage sensitivity across biological disciplines.;citation_volume=109;citation_year=2012;citation_pmid=22908297;citation_doi=10.1073/pnas.1207726109" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Phytopathol.;citation_author=A. F. Bird;citation_title=Plant response to root-knot nematode.;citation_pages=69-85;citation_volume=12;citation_year=1974;citation_doi=10.1146/annurev.py.12.090174.000441" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. Blanc-Mathieu;citation_author=L. Perfus-Barbeoch;citation_author=J.-M. Aury;citation_author=M. Da Rocha;citation_author=J. Gouzy;citation_title=Hybridization and polyploidy enable genomic plasticity without sex in the most devastating plant-parasitic nematodes.;citation_pages=e1006777;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006777" />
<meta name="citation_reference" content="citation_journal_title=J. Nematol.;citation_author=J. Brito;citation_author=S. Subbotin;citation_author=H. Han;citation_author=J. Stanley;citation_author=D. Dickson;citation_title=Molecular Characterization of Meloidogyne christiei Golden and Kaplan, 1986 (Nematoda, Meloidogynidae) Topotype Population Infecting Turkey Oak (Quercus laevies) in Florida.;citation_pages=169;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=Genome Sequence of the Nematode C. elegans: A Platform for Investigating Biology;citation_pages=2012-2018;citation_volume=282;citation_year=1998;citation_issue=5396;citation_pmid=9851916;citation_doi=10.1126/science.282.5396.2012" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=G. C. Conant;citation_title=Neutral evolution on mammalian protein surfaces.;citation_pages=377-381;citation_volume=25;citation_year=2009;citation_issue=9;citation_pmid=19716195;citation_doi=10.1016/j.tig.2009.07.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=G. C. Conant;citation_title=Comparative genomics as a time machine: How relative gene dosage and metabolic requirements shaped the time-dependent resolution of yeast polyploidy.;citation_volume=31;citation_year=2014;citation_pmid=25158798;citation_doi=10.1093/molbev/msu250" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=G. C. Conant;citation_author=A. Wagner;citation_title=GenomeHistory: a software tool and its application to fully sequenced genomes;citation_pages=3378-3386;citation_volume=30;citation_year=2002;citation_issue=15;citation_pmid=12140322;citation_doi=10.1093/nar/gkf449" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. C. Conant;citation_author=A. Wagner;citation_title=Asymmetric Sequence Divergence of Duplicate Genes;citation_pages=2052-2058;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952876;citation_doi=10.1101/gr.1252603" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=G. C. Conant;citation_author=K. H. Wolfe;citation_title=Functional partitioning of yeast co-expression networks after genome duplication.;citation_pages=e109-e109;citation_volume=4;citation_year=2006;citation_issue=4;citation_pmid=16555924;citation_doi=10.1371/journal.pbio.0040109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. C. Conant;citation_author=K. H. Wolfe;citation_title=Probabilistic cross-species inference of orthologous genomic regions created by whole-genome duplication in yeast.;citation_volume=179;citation_year=2008;citation_pmid=18562662;citation_doi=10.1534/genetics.107.074450" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. DeLuna;citation_author=K. Vetsigian;citation_author=N. Shoresh;citation_author=M. Hegreness;citation_author=M. Colon-Gonzalez;citation_title=Exposing the fitness contribution of duplicated genes.;citation_pages=676-681;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18408719;citation_doi=10.1038/ng.123" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=P. P. Edger;citation_author=J. C. Pires;citation_title=Gene and genome duplications: the impact of dosage-sensitivity on the fate of nuclear genes.;citation_pages=699-717;citation_volume=17;citation_year=2009;citation_issue=5;citation_pmid=19802709;citation_doi=10.1007/s10577-009-9055-9" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. Emery;citation_author=M. M. S. Willis;citation_author=Y. Hao;citation_author=K. Barry;citation_author=K. Oakgrove;citation_title=Preferential retention of genes from one parental genome after polyploidy illustrates the nature and scope of the genomic conflicts induced by hybridization.;citation_pages=e1007267;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007267" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. M. Evangelisti;citation_author=G. C. Conant;citation_title=Nonrandom survival of gene conversions among yeast ribosomal proteins duplicated through genome doubling.;citation_volume=2;citation_year=2010;citation_pmid=20966100;citation_doi=10.1093/gbe/evq067" />
<meta name="citation_reference" content="citation_journal_title=Molecular and biochemical parasitology;citation_journal_abbrev=Mol Biochem Parasitol;citation_author=C. D. Giorgi;citation_author=P. Veronico;citation_author=F. D. Luca;citation_author=A. Natilla;citation_author=C. Lanave;citation_title=Structural and evolutionary analysis of the ribosomal genes of the parasitic nematode Meloidogyne artiellia suggests its ancient origin.;citation_pages=91-94;citation_volume=124;citation_year=2002;citation_issue=1-2;citation_pmid=12387855;citation_doi=10.1016/S0166-6851(02)00161-5" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=L. Hakes;citation_author=J. W. Pinney;citation_author=S. C. Lovell;citation_author=S. G. Oliver;citation_author=D. L. Robertson;citation_title=All duplicates are not equal: the difference between small-scale and genome duplication.;citation_pages=R209-R209;citation_volume=8;citation_year=2007;citation_issue=10;citation_pmid=17916239;citation_doi=10.1186/gb-2007-8-10-r209" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y. Hao;citation_author=J. D. Washburn;citation_author=J. Rosenthal;citation_author=B. Nielsen;citation_author=E. Lyons;citation_title=Patterns of population variation in two paleopolyploid eudicot lineages suggest that dosage-based selection on homeologs is long-lived.;citation_pages=999-1011;citation_volume=10;citation_year=2018;citation_doi=10.1093/gbe/evy061" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biochem. Parasitol.;citation_author=K. L. Howe;citation_author=B. J. Bolt;citation_author=M. Shafie;citation_author=P. Kersey;citation_author=M. Berriman;citation_title=WormBase ParaSite - a comprehensive resource for helminth genomics.;citation_pages=2-10;citation_volume=215;citation_year=2017;citation_pmid=27899279;citation_doi=10.1016/j.molbiopara.2016.11.005" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kirkpatrick;citation_author=C. D. J. Gelatt;citation_author=M. P. Vecchi;citation_title=Optimization by Simulated Annealing;citation_pages=671-680;citation_volume=220;citation_year=1983;citation_issue=4598;citation_pmid=17813860;citation_doi=10.1126/science.220.4598.671" />
<meta name="citation_reference" content="citation_journal_abbrev=J Nematol;citation_author=S. Koenning;citation_author=C. Overstreet;citation_author=J. Noling;citation_author=P. Donald;citation_author=J. Becker;citation_title=Survey of crop losses in response to phytoparasitic nematodes in the United States for 1994.;citation_pages=587-618;citation_volume=31;citation_year=1999;citation_issue=4S;citation_pmid=19270925" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. O. Korbel;citation_author=T. Tirosh-Wagner;citation_author=A. E. Urban;citation_author=X. N. Chen;citation_author=M. Kasowski;citation_title=The genetic architecture of Down syndrome phenotypes revealed by high-resolution analysis of human segmental trisomies.;citation_volume=106;citation_year=2009;citation_pmid=19597142;citation_doi=10.1073/pnas.0813248106" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=D. H. Lunt;citation_author=S. Kumar;citation_author=G. Koutsovoulos;citation_author=M. L. Blaxter;citation_title=The complex hybrid origins of the root knot nematodes revealed through comparative genomics.;citation_pages=e356;citation_volume=2;citation_year=2014;citation_pmid=24860695;citation_doi=10.7717/peerj.356" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=H. B. Mann;citation_author=D. R. Whitney;citation_title=On a test of whether one of two random variables is stochastically larger than the other.;citation_pages=50-60;citation_volume=18;citation_year=1947;citation_doi=10.1214/aoms/1177730491" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. L. McGrath;citation_author=J.-F. Gout;citation_author=P. Johri;citation_author=T. G. Doak;citation_author=M. Lynch;citation_title=Differential retention and divergent resolution of duplicate genes following whole-genome duplication.;citation_volume=24;citation_year=2014;citation_pmid=25085612;citation_doi=10.1101/gr.173740.114" />
<meta name="citation_reference" content="citation_journal_abbrev=J Nematol;citation_author=H. Melakeberhan;citation_author=H. Ferris;citation_title=Impact of Meloidogyne incognita on Physiological Efficiency of Vitis vinifera.;citation_pages=74-80;citation_volume=21;citation_year=1989;citation_issue=1;citation_pmid=19287578" />
<meta name="citation_reference" content="citation_title=Root-knot Nematodes;citation_author=M. Moens;citation_author=R. N. Perry;citation_author=J. L. Starr;citation_title=Meloidogyne species – a diverse group of novel and important plant parasites;citation_year=2009;citation_doi=10.1079/9781845934927.0001" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=M. Neiman;citation_author=T. Sharbel;citation_author=T. Schwander;citation_title=Genetic causes of transitions from sexual reproduction to asexuality in plants and animals.;citation_pages=1346-1359;citation_volume=27;citation_year=2014;citation_pmid=24666600;citation_doi=10.1111/jeb.12357" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. Notredame;citation_author=D. G. Higgins;citation_author=J. Heringa;citation_title=T-Coffee: A novel method for fast and accurate multiple sequence alignment.;citation_pages=205-217;citation_volume=302;citation_year=2000;citation_issue=1;citation_pmid=10964570;citation_doi=10.1006/jmbi.2000.4042" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. H. Opperman;citation_author=D. M. Bird;citation_author=V. M. Williamson;citation_author=D. S. Rokhsar;citation_author=M. Burke;citation_title=Sequence and genetic map of Meloidogyne hapla: A compact nematode genome for plant parasitism.;citation_volume=105;citation_year=2008;citation_pmid=18809916;citation_doi=10.1073/pnas.0805946105" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Pavelka;citation_author=G. Rancati;citation_author=J. Zhu;citation_author=W. D. Bradford;citation_author=A. Saraf;citation_title=Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.;citation_pages=321-325;citation_volume=468;citation_year=2010;citation_issue=7321;citation_pmid=20962780;citation_doi=10.1038/nature09529" />
<meta name="citation_reference" content="citation_title=Numerical Recipes in C;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. R. Scannell;citation_author=A. C. Frank;citation_author=G. C. Conant;citation_author=K. P. Byrne;citation_author=M. Woolfit;citation_title=Independent sorting-out of thousands of duplicated gene pairs in two yeast species descended from a whole-genome duplication.;citation_volume=104;citation_year=2007;citation_pmid=17494770;citation_doi=10.1073/pnas.0608218104" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Scannell;citation_author=K. H. Wolfe;citation_title=A burst of protein sequence evolution and a prolonged period of asymmetric evolution follow gene duplication in yeast.;citation_volume=18;citation_year=2008;citation_pmid=18025270;citation_doi=10.1101/gr.6341207" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=K. Scienski;citation_author=J. C. Fay;citation_author=G. C. Conant;citation_title=Patterns of Gene Conversion in Duplicated Yeast Histones Suggest Strong Selection on a Coadapted Macromolecular Complex.;citation_volume=7;citation_year=2015;citation_pmid=26560339;citation_doi=10.1093/gbe/evv216" />
<meta name="citation_reference" content="citation_title=Biometry;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=L. Stein;citation_author=P. Sternberg;citation_author=R. Durbin;citation_author=J. Thierry-Mieg;citation_author=J. Spieth;citation_title=WormBase: network access to the genome and biology of Caenorhabditis elegans;citation_pages=82-86;citation_volume=29;citation_year=2001;citation_issue=1;citation_pmid=11125056;citation_doi=10.1093/nar/29.1.82" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Szitenberg;citation_author=L. Salazar-Jaramillo;citation_author=V. C. Blok;citation_author=D. R. Laetsch;citation_author=S. Joseph;citation_title=Comparative Genomics of Apomictic Root-Knot Nematodes: Hybridization, Ploidy, and Dynamic Genome Change.;citation_pages=2844-2861;citation_volume=9;citation_year=2017;citation_doi=10.1093/gbe/evx201" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Tang;citation_author=M. R. Woodhouse;citation_author=F. Cheng;citation_author=J. C. Schnable;citation_author=B. S. Pedersen;citation_title=Altered patterns of fractionation and exon deletions in Brassica rapa support a two-step model of paleohexaploidy.;citation_volume=190;citation_year=2012;citation_pmid=22308264;citation_doi=10.1534/genetics.111.137349" />
<meta name="citation_reference" content="citation_journal_title=Nematology;citation_author=M. Tigano;citation_author=R. Carneiro;citation_author=A. Jeyaprakash;citation_author=D. Dickson;citation_author=B. Adams;citation_title=Phylogeny of Meloidogyne spp. based on 18S rDNA and the intergenic region of mitochondrial DNA sequences.;citation_pages=851-862;citation_volume=7;citation_year=2005;citation_doi=10.1163/156854105776186325" />
<meta name="citation_reference" content="citation_title=An Advanced Treatise on Meloidogyne;citation_title=Cytogenetics, cytotaxonomy and phylogeny of root-knot nematodes;citation_pages=113-126;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=D. L. Trudgill;citation_author=V. C. Blok;citation_title=APOMICTIC, POLYPHAGOUS ROOT-KNOT NEMATODES: Exceptionally Successful and Damaging Biotrophic Root Pathogens;citation_pages=53-77;citation_volume=39;citation_year=2001;citation_issue=1;citation_pmid=11701859;citation_doi=10.1146/annurev.phyto.39.1.53" />
<meta name="citation_reference" content="citation_journal_title=Nematology;citation_author=W. Wesemael;citation_author=N. Viaene;citation_author=M. Moens;citation_title=Root-knot nematodes (Meloidogyne spp.) in Europe.;citation_pages=3-16;citation_volume=13;citation_year=2011;citation_doi=10.1163/138855410X526831" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis;citation_year=2016" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Single, Shared Triploidy in Three Species of Parasitic Nematodes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="The root-knot nematodes of the genus Meloidogyne are important and damaging parasites capable of infecting most flowering plants. Within this genus, several species of the Meloidogyne incognita group show evidence of paleopolyploidy in their genomes. We used our software tool POInT, the Polyploidy Orthology Inference Tool, to phylogenetically model the gene losses that followed that polyploidy. These models, and simulations based on them, show that three of these species ( M. incognita , M. arenaria and M. javanica ) descend from a single common hybridization event that yielded triplicated genomes with three distinguishable subgenomes. While one of the three subgenomes shows elevated gene loss rates relative to the other two, this subgenome does not show elevated sequence divergence. In all three species, ancestral loci where two of the three gene copies have been lost are less likely to have orthologs in Caenorhabditis elegans that are lethal when knocked down than are ancestral loci with surviving duplicate copies." />
<meta name="og-title" property="og:title" content="A Single, Shared Triploidy in Three Species of Parasitic Nematodes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/225" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The root-knot nematodes of the genus Meloidogyne are important and damaging parasites capable of infecting most flowering plants. Within this genus, several species of the Meloidogyne incognita group show evidence of paleopolyploidy in their genomes. We used our software tool POInT, the Polyploidy Orthology Inference Tool, to phylogenetically model the gene losses that followed that polyploidy. These models, and simulations based on them, show that three of these species ( M. incognita , M. arenaria and M. javanica ) descend from a single common hybridization event that yielded triplicated genomes with three distinguishable subgenomes. While one of the three subgenomes shows elevated gene loss rates relative to the other two, this subgenome does not show elevated sequence divergence. In all three species, ancestral loci where two of the three gene copies have been lost are less likely to have orthologs in Caenorhabditis elegans that are lethal when knocked down than are ancestral loci with surviving duplicate copies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>A Single, Shared Triploidy in Three Species of Parasitic Nematodes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Ds6WT1vRBIiVaT-E9VLF1GlL-m0xL6UUrQ8aWHYjXEQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495268","apath":"\/ggg\/10\/1\/225.atom","pisa":"ggg;10\/1\/225","ac":{"\/ggg\/10\/1\/225.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/225.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/225"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/225"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/225.figures-only":true,"\/content\/10\/1\/225":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022olESrM3N2eeagxSTn5hIH5wJPc3Zf561jLphMUzLb8Hdlnkd6XiYmh4zDf+YL\\\\\\\/JWB1bFKXbUwMfkmZzlff9XZrP+PHwiFflIu26oq+Fz0lHkHZSJNo79XJlk8jXbG2jXwJzMrcCluKPqkZMdfg1vkob+fBTnMVPoeezKBpMIm459kCfYKDDqQ5mKmyRwBJLFL\\\\\\\/m+B95etQ4PF7Hhj5JrjZKUaPpWWTVHdh9HyvrUogoFdxZ0QFBFFiZtqBONk+3QV\\\\\\\/1ldEw0nqMq3XhV7VsFQe3zGaWCPuWcknsz\\\\\\\/w7xBjnPPCcbNl6q69TVkVrB2+GM9Ma2Gsfn799JQU0Fn+vur9ybceN9CtdxLDJxvDN2tHNRhY4BGUhQdNClbS\\\\\\\/Jn2aKeD\\\\\\\/SYoEZw2+6f9AbLTvB6rAuTT\\\\\\\/faUWOj5IigQuLntp+e3mq6EFtkQELPU38RmIG2vAq\\\\\\\/llYogjI53iNxCvFWbVf6pn1iPzjyxFrqp2cvjhjGnBnfDdQ5Uw2LJDlmCdlrJ9QNp1oijD249hvK+IqV04Yi2QuDjlVnhaVkLtxKi1Y2UmhO8d75QoX8yoVIZcBqDyZH8Qo8ZUeCNKJWjkVWaCrsYDw3abwnGHCDNKDILLUY8i0A2fJVnDmv\\\\\\\/kfN5rqC\\\\\\\/ZfcBvhOl77xDvS4e4XULv3U4QDq3rfsGqggkj7Ykva0BEJokaz5yy01mAdvwPvpS7FP6XboJqEQyTekQRayV5aLcdFd7f+M+4MeO5jOGK6XziwzGJzKN5YbPnh672D2MEDZKK2fCRvUwGZ2\\\\\\\/ASXGkH1cw\\\\\\\/iCtnlEUAN3fH6\\\\\\\/PVOlB4ikTUf\\\\\\\/evSWB54xqUcjmsunRNz0D0QfteoTplpxEle3pIoZE\\\\\\\/Zz4wlFFkv1xiomGQ5toVuMVn5TdBxrDPOw+g\\\\\\\/4isDigBx\\\\\\\/1wFaRfdSxr5owVsN7G09R53V\\\\\\\/rlmqsSiufLXjSSaKWfwYKfYMBPlow0IhdbXpUeePRcW5sGLP2BcmCALQXUeLtLAJCsdmdATcEEUOgWAVB0UAEjqnjMkVur6hKlKXR9oSLpeNbVwDioQunqm9beNoyVRHXcbE91fE3E1omYuty44zIjfhw4alTK1aOgSq\\\\\\\/Xwhs1bZRxHWXRMiTwR8d+k9Ael8anLMuvVDj\\\\\\\/yTJHFYkcdsMfTjaaGwcyGSze5rtLRysVPHUv7rYL6KwwdkVkKv3MRU4srvHoebvNiUglByER883m8K7Xj6g\\\\\\\/ToIl\\\\\\\/1FrDQHEa196vIRVg\\\\\\\/mnQAaSCOBsM1xcv8S23YcSYvqcogolphDGwTt\\\u0022,\\\u0022iv\\\u0022:\\\u0022PEzmkt6G5P9Rfog\\\\\\\/YtpShA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002250de19a2259b5e9e4250cf2086b039b30f53313bea0acb0b4e6a649e6aa453e3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hm1MmEWhygOAOy8vG2N1WmJtxJt\\\\\\\/l2AwGO+ytcxyE1XhO\\\\\\\/1m6V2aqhdBezq\\\\\\\/13B85w2wGsuGE9Hngp+A2FvQuTDh4GxUUN8eBTdVsIUwwRw9jergEKXhZePNfcoU7PIDZ5\\\\\\\/5yR\\\\\\\/29B01JQ74jYo8u8u4az3FeDr4Qsel2Y+YnHaIbBCyl9IxrkQzEE5GRfB+EMrQtMXtRPueKa5B+55JLsIowenYJG98Quq4KtXwa+BNuYAt8w6HuECRFkqOuIUyz3CIZLKrErQGfbXdRnx+AjCxzkQE0HwedFedJMuhqRQFXEiuTND01kGMJ1DwFyX9X\\\\\\\/cHMIXKdxtLqyia+Gz\\\\\\\/wws9vs\\\\\\\/LuWLKHzcKbJg0\\\\\\\/HZXn2AwhFrpA8uA9nHWU6ydNv7o9VobI5iWSFRx\\\\\\\/9DT2ZCR4jucMi08vn07n2opO+K6cJHJSl87lrVH2ZBaq0iZ03ajvBHJtrQIFZeVzwfxFxYjLpiiIhL1cvl80u0laPC5xF1BbYq4zAKW7Soy6l3Ruc1JrR4xn8BBAT8mMKbii1MJ67Lo9A+V3rtyvZ2PFO6wra3vSnBhPrP0hNBeUokKJLYtTvee+yD6dkvtlwzFC2nkanItjhLqAyLPEs4RX04mHA9wrWoOqc+gosYV\\\\\\\/\\\\\\\/gNx2FAZS4J+2eYGyCtXsF3rRYYdF+G2en4btzZLB4tf\\\\\\\/eUu4aInUaOSqClkTWQvjGC4VJP66DZhBXbqnhJgLEx3mUQG\\\\\\\/B\\\\\\\/RHP2gbwqiemUROPqa0nzfrGfe3efxItkn8Yps08EFTpN8joj9WsecjMwGmJAJitWzLWuNGLdhXazeLNTBNsQFIWU+zHYyU3cparZWSEEuDfn92wafLe6hvxQAx3KstqmDfdJCeB2YB5KvkrZF\\\\\\\/GM8mx2m5FSF2GBSvIZ+VdgF2QPtJmfeOWb1KyxAlkz5nosmlPlDH9jpVcCWmq8AvD8QqBFjt9f+a0HXHX4wYDoEqpBtLwumEOERocbfFaHIe8Vdxr0HtI4Wda9pqMEz9bZ+K54tFCBePfJ5Ij5v4p0u\\\\\\\/oZwyLqUR+1FAabLMDPy8aa7D6YHO2Kk7dMj6E76ot+vB5GsoNrVPkK0ZDUvJVXm4nkukcI5Bit3ohwGNaLsGRLU0StfElvNqkPYRygzz\\\\\\\/xz8ts68WjPSpeU7SV4PscwthG6BzfbmnXynoLqfIHKu1UHH7+LCTEU7GDVkvrukdpuAR8HClwuasW2opKKM3fJCTRx5QCDlkTjdU9U4K6qN+C88YvDiUkaZ7h+6egbjZufPgvFD1osyxz1VWs48jCJxFQr9r80r17LVbQz0fS5s4vIAzMk39StD+5oWHOWWOPb8xXjK\\\\\\\/8sQ8WTJLwqO5OdkuN0DrtShgcRux5V9Js7FsUh5yfrfQehcVmms1ERR+yKK46VpcfYFxsj5+jSTbueUE2+eRlKTQd8HfeEhF6JXuKaVckNYSZpgMXGWQ8fEfP5p4qxI68h3gJIBRgQ6Iw7FYLgie8kYfRGFCWFd6v3wJIynI34pqdExwYsgsBfaY2q9DDg9xDRT9xnvXMv7LIRclMFz2WQpTqc97jRTuJYN5S5pSDQ+hjFkoKvE56ymvFDNBIm+QZA4YLizpkRkSygqJTdh+rmq7UNO97ppZ0tzaU6qLk5TqdR7fNufNZnYtgYU2IQQPfdxUIICgnGOWX7\\\\\\\/5auc1yjf1\\\\\\\/dPCoSYBYR9q3VcK\\\\\\\/hMn5863OHH44mlyT0g06rwWoqT6+AA9yH2VEw3JJzGLPR6vN629RUtXfb+fvEDpMprr50jDL2vNMbGFKDPlbWr8F9uo2QmRZ6dXSG1gzxPa6vTEtIIpEUH+I0DJ3vwKcnaBjf6csvROrD6FFRDrFV+0tS35un9zOd+z7iMgNxhoOdaOmsycJTnwdMiIWE6Av5RZdY\\\\\\\/GqBt26UCVFoEj1ZRQVJTvZ91YclOJzvvvIG2fpESHRthA8XRW5dNfR4NP1j9QYVmv81R3zipNal4kVItwfIugm9ODdf51wz4W46cxeKCLdslRRaVQBQlo5iA3W+Dt5E28vU+bKbkRLmfJ0WDlt5X+fqrZPL\\\\\\\/V7+U4zQbSJI99UHKpAzvW3ix5Sim9\\\\\\\/mGoB9Dcqyu3GkH2J0sy\\\\\\\/DTkuxfYF089pZct+iiK8k8Mfc6tCR1UOGAb+NVg3PVPBnatbul7qEhNHcxhwQyPOoz7DDf5TCLMqCTjjsVasJf54Ql+ZKQvDSOS9iUhhnfaHk39TAOno\\\\\\\/aPOU7sueVyIe+y2MJOSuBXWgHhxjUXqVL\\\\\\\/1SR74dyJLWlf9LgLGWV95tjGOToFpC28mCf\\\\\\\/ELKi95A+9UXJVuJP4zEhmVDpiyZAnvh+gZG0guyogb68kCYonj5yuCxUQGIFJpNk\\\\\\\/QszO0gIcEcULZ0yzsA8lAJU2sPH7cOy+EvxB6kwiy0orEWtn+ejV\\\\\\\/okZ5T2y7gj72dbk4qZ7\\\\\\\/EwfWVGu7tk+U9iA0EtSAd8A8bxKgB3\\\\\\\/ffcxuSrKqCxt9fnqUb7hM6D+6yDg5ld68c\\\\\\\/AtxS5hh2oAYKtaNx\\\\\\\/kB4Zdrg9fIkAraKOklOsKdQKKZGjIh304E0spcjKSfLKwJfqtFjivejalH1rVXmsUgvEqBHn19ME3v1VssD7iXFnzRRvLmQk6H5aE1cJyN8Xh3ssETklVzU6naLQ6KP+YxncIDO6dVOFc1W93f+yJ6Bn+82oW8mWxW+fGUPdphzNd+7GFpKUNYp\\\\\\\/LF+0y\\\\\\\/YOd55K3TtescPNWG5qAEZd2Ao34vO0o6W0696F4RoEh0aEpnOOJ8m9MZpJdsM7W4F4D6Nc+IPZ4Yi5UURIQIgoRTzOaX0gcR2z13uFzYYFvQAiZdi5ctsRRJ1B4OQewj\\\\\\\/1Y0FnNFHFBANNH+ZMd499HLmR1ScWtu2RYklycHLprsLbTWI9xSJU7bD5kc7usepUuLC5VamsKu0Gb+lL7yifn6Qj4Aw1q2OO7R7hak6aW8ZaWz\\\\\\\/fvL3dyeZyr42Lwaok08MsnQW0d7e9q16AFdMCrf\\\\\\\/AZIAxcafePRktbcjgNjbdcLCzzUauGvfUkYjBbdwf5\\\\\\\/71SaQejrnqfr02rbCf5TZiSRzIBB7h2N1HkShAMzR4TTkLxbJBQw9ePemhpXgtfqfd96dMfejQJmWgUk0RR\\\\\\\/aavo3oNdXpS6By5gm5aupkTqsikQdRai2lDaWfy7k6tlE4SkugdRO785tkSLx0auThFp8Wnhqa2BI0iu+VmdWsAHRSPreful7BonKOsMfSEXarr3JbwUWTstlYPklTcDMmj0tKAKTHatFh2PJ9VqDDcqA3VXBteopntE9Bb7vxdkQ02XQuCF5qGqAYgZP\\\\\\\/5kQhQo3pyscGyKiaeVD0PovGwdSTxbfmQUbvFWzbNXVfWMtWQaTPMD0H\\\\\\\/zqicWrU4H382LocFdwog9tjsKWuTnqtahwGzu3aCm9Z5620+\\\\\\\/tHNmgF9MNeEpzobBAAifKB\\\\\\\/nNaElhYH4Nyw\\\\\\\/uA5UV3\\\\\\\/z8r1DM9O26g1J98QGIhUwHO6Agup6roUfHb1XVoaTnscCZ4DAhCaWoorFJyuGJblfBN1c6khIvoRJqSNrwg8PadCLBa6lrsJK9x0Cv9X\\\\\\\/DqSomQTyop7WmQvLPEE2OCtTWyU3X8Geuj3Gsn8N53ZO7c+wjEcMeaEhRnAvkwU06xqpahH9p+NG7dfUJ\\\\\\\/7ucnKrHtRS8LC2z3dthJ9pry5XX2Q6fGXliKi0xErLtwkjljoQIBjWHcBavjhlKXGEQMcf2MWL0uzhSs+oQ2\\\\\\\/x3QzYWlzpY+LkNSzGNMC6ldMHHwZdM9gHOs71cVny3fkzE0352sD3lN8WnEk6BF4Xe4vqKCVv0UDLWIhgT9b6cytPWYVyLjU7OX8eIJ+2ryu8axNeMzAYnh7AKp5AkduopLIOHBa4kAJ2YJ\\\\\\\/T3BD0dl2cn6xSX0QJnXsjGlpEQxccwFmlPxPdDvkurHuZhnOJxNSyrU7qBW2c\\\\\\\/r0eEtlnKlI4Jpuyt4YQBwokwnDJmM5s8\\\\\\\/01TqQftbdproZrSFsrFhueS5E1hNjwxCrXYxY+b\\\\\\\/hf6BCBzZQGFc4SDYxYllg2kMbRYoPqw9KLqcl11VIN2gJYL4rUd1WrgGX2lh6APWXdjcJ9QxDYywJ2sjdcmcGN70X0ssTtxeMLuSskSFj6sktuNiBFnZOfzoMBnuxLQ0kZgc5qI2DLVPY4qkLoxdPgZVq95Fge8kREq8swURlaXzsn7hZmOFgo4qZs8GE2\\\\\\\/9OaFVEH92rUQrTeefxutcLzKhHTr4okHeL\\\\\\\/wpzxMrXh+L+oTpXgRxrrty6ftm4uBzNHTJB5EJILPUaKuZQ6tDNcBkvuBlc9jG+4Umse+OskjiBSV3cp2gGpM42iG2rPw0dhcCuZ4xtcHsvwWzVU1gb2YxqOytbYw818vS0LkbSldwNkxucJve5kesJ\\\\\\\/ukaDYlZVlacWSpeHwHIeYKdRTIBR8bpsw6L8ags9hF4J9eLd8lIHamxxOEToEWLYwn0P7Yj74BOYFyX735MF2PjbPuOUytTPRsnuYpyR\\\\\\\/QG6yhwPOeAL+ZDC3deDMDtEId90vTCRknTZi0KcU4VVVELSnZIxQXYEAB+LUWvRd90exJvuqzgHbHiMtRVGCHlRLWfn8f28tFvypUOFafeTunEtLVIuwfnceN5SehY3u5sm2btnchr5XSrZml3saUR7XvfI3K3\\\\\\\/RuU8m09mvcCyyYAPkWGFRWmtA\\\\\\\/fA+DAVq0Tc30RaUgZenAyT3lrbdK8cRwYJNAP5KFYS4pF7T1ppdubQicnK7W1WPFgYmr2BcVa9GQ5Ap1KXdlN3Jwik+nR2ALqVAh8ZqW8loiav\\\\\\\/ZptZM4a5jk8j9P\\\\\\\/YugDGsWF0WDRQD51\\\\\\\/IZYiL8zJt1wVoTaYwtqqRzwh4tFaCvl\\\\\\\/Z4NFpAyKqxawlcQfpghkRR8zvVfXWYa3Z5vXQInfAcBlfYZV43+9rDIW\\\\\\\/ZlGuui9GLU+tIE1174pX3YX2j9e6TlTdIixLHDlQm+mmxSDIo+fmWnVJhFMUiUbb5GnPJ9np+ScUEqFqdY5WoD7Omgj956Goj4uRiJZkVjppi0+8dNHPjKcazffE90R3gqeihp1znyJxCHYM5HixcjCNwXB7q\\\\\\\/x86t61X9OC+HGQLE5v9SSJgQS2EWU93iAopAdyeAkWxIJAfRlo8brYNQ6JW\\\\\\\/rvpDv14DydRxhxj1ij8Iqkkf+9Vdvek7na5PRmjqEDeKpev0qBo99Pmj5AXphKsiz1MvhPOCKpNv7SpFfg8mz3sXjW9TCU5arMhwTZ4AlyL+VxbYKr0kXNMoHBTA4Cct\\\\\\\/tHNQ\\\\\\\/o6wKqVCSjqeS+D7lmdLsFVHnEakUghYbc3cr9hoKyyk5PbAFo41lnrN\\\\\\\/ie\\\\\\\/GXK\\\\\\\/bVvz1spm1xgsJJ1Z9vo2mlzPtyycpksxomVz90xzUFsiu1ahGE1GiLX3ySLMBEIZKDerYfubtUVtRrb6WcTAKHeVA72AuKbayCEXADEn6MRkF7+m8k9wHBsFaPhjR55LL34kHWD9inGm0eGjW0Rpzh6fGD9qeU4ZFF6Xtvgrl6SkVf8yKyHYh0XVpFjDshWPcsJyLVuExBFtpfsU0gJzrszPiuJl0LfjGYdwMBWIW73VWmqhs\\\\\\\/V7dP08WQptpMGjWrol5zJ7IBJ\\\\\\\/7jhpokqVpB1P\\\\\\\/DT6kju1rKZsPY+iTl1Gn9WhEpNUNqkGULVn\\\\\\\/zaifMVCRpRwnHwnLXyCcLQnLV2ufgn4Yyguy+IqwAreV2gJUyQ43VL\\\\\\\/HIdIb8O\\\\\\\/rNrl8xxQcWFZMrCOK+jHhfAZi0u+nejolGxMQTL1KrhHzev3O2bvLwR9yAW8JuCaer2EZKk0ZxU1q9gdr5TuHs+kGpyUj86kFF0QOZci1J7qwwY5rb53CE0xTR44VEBcSUSOOqO0E49i3hmLzMMqkvjMKU1qk2XDqpeVMN8v7EvZJRr5ijvwZSKMxeoBuLvZtAggOIybJ3H4oZ0KCPHJlkNX43z8CuOv3ON6anjQf3WNhFXHFGboa77CI+vVoIC48G1yH\\\\\\\/M9NdDsMjftWqL8AH1N2nvJDKh7CrAflsVm9yOyihlN33ii4fq13maBWGBe6o8SVJUKi+6aXkc0yMkfzRiBeMOf5914Ngf2XvxnELO6u66ZHhlNQKX+jAWq82d5iHawzxYd9jF6Bm+i\\\\\\\/P4u8koc2njTG7CzANgh8BGHpMjmvbi8TukIOgqQ0iywuu2cIUcGwORUbvL0DS6XXWQyy1iKu5B7YCHit1zw\\\\\\\/TZf7CdCwAjcIcvwb6UZpmaK+xjqgnq9dGoZwfqabxoU5rp4wKuCQB7gNFhIhr54+X9WxvJ77Q4+GlTInM5Gz\\\\\\\/IekX6W2Xw\\\\\\\/sv1jE8pNgyaJNJ18TKArAaCoQosnjUMQ9nniKwrdJcE8vqXt\\\\\\\/lzCcAwhjW+2rWRqMU+YkPSf89m4Z0tP9TX16UoSGvQIyFV6cFWRM35GN1gTKQxkuL8t0ve5T8JV92f3toNcj\\\\\\\/Co+hcBBY7yAeQ2cb+8t1J67Te6uN+GoQiP9D\\\\\\\/K1uJ5x5nhaG+KmSxXYn8jxUAOjIL+7cl5LCuBnmpPuCkuDcujEV8Oks4R+rIYg5aBZdDEiDglc6mswmOAmQTlS3AYlMxSeYkB7BWfawkfnKDHRpqL0lRd1cTA0ZaYL0VsmwfolMWhV4xi4BA1qlaUr94VvBNW51N0Peup+K95brVtQyDFXFYcXDTlk8QVD6odPJVIiPNLmzH91OZw+mKEVqWVz2U4n9I\\\\\\\/ZR39IkwTr\\\\\\\/W+UTWdEguEwoOKBwXxMswj5rPKe0qxFAN5q+x8\\\\\\\/oubr6rVKyHMHLjjCz00Is3o5anWF1StOiBl14yB0Lpgfq80SA\\\\\\\/\\\\\\\/S+nfnoRAIrQKZg4\\\\\\\/2nH\\\\\\\/7ZCIR7Vd2VG5KS\\\\\\\/ZJPi\\\\\\\/ZpJ+cl7Tqru5ZKkN9CrCHZ6AY2LnX0tW1ZC2ySZCB\\\\\\\/+xvbeMUvkeyTQl580UdKe3Bl4K\\\\\\\/4pQF7AoBUEHXszy66gsbuotuiJ0YpyLC75yKujW7Bi2IBzK97BojuRd8Gaz6eVztDOGs\\\\\\\/H2rwdJMJsacqITnak3FLoKDnLqXh03w5kB3ulFgwBcL5s87PW\\\\\\\/BDkGL\\\\\\\/ZO2c\\\\\\\/CMdhFS4FI64nwlRPLo7kSc7G7V8sESOnUP8AVNOmoVnnRQyMwgSIaI52BAgQskj7hGA867QB5tkyEKl3YbNyBlKW2stJW2J85MnnvHJaS5Zs3ng4AK0tgJERKak0YtvAofRAPNQQFvw4+d5m2QSKjqnSzCnqZbhFl\\\\\\\/egpyD1zeq4VoU2FXeH2+URC61h\\\\\\\/dXMc\\\\\\\/fU9OXNXDf4LV45Rs\\\\\\\/nld51NWaQEO4rcMUxqCWFd4vkyWg5gyu4CFiGECv6fQPwkcxupQ+W\\\\\\\/BQ4xDf7UQIS6ksYWN2bhJEwIDeQxMmFR+X8Y35ccMdQgxtyF+xj8IYoE8MKZT94aVHRUU+Z8NPPpA\\\\\\\/uPuDrDLH8DzmOhaa\\\\\\\/IKeTaJaKXoot0MnRniqHUtYZVtTPu\\\\\\\/yxYd0U1qccGXMiRazDj0UBFIq4ZFOYXWCoJoupO\\\\\\\/+7WiErIZJ4IR9XDbF2BLnYT8HrJsJymZ50bFXzs6sSMMjLcux5r8JUIS5Ma2lJ\\\\\\\/v0XS3xX4hIn4IE4Syw2enRPP3mdcsXK2RZehdj\\\\\\\/Oa1fLA+Uq4GZPPYAFGPJZpMdsZgOoNtVNGzBhob1egTvqDr7mhwqSMH+gkdbjmHY2EBKbDFjquDjCLtcy2eI9bE36thPN5BVMOPgSXt9MUQ3DoNj65hnirUbGmnb1DOt0mm6ru1TLt8EFw0rMikbfckvm8uRHbDtX0JOn0h4TAPoPZo9DM3r6oQQMF3SA0LHDcULnHl1CcL+2loEN9h0A16EG7mq6hwXiJucUhqU2OW8afORM9\\\\\\\/zjNfq4x0aflImCcZjM7HP\\\\\\\/iInpZaLBPHDxR7VA8qtgVTLUMulRJxa\\\\\\\/+E30BijyVxzApf1JygVl8Xh0BEiNvMJQOSkKUhztxA3J\\\\\\\/JjsdXnBzPS3s6AFGMNYkzBOU7\\\\\\\/zVnBatyeiuzoglZlezzERUr2jwIuGEMwDlJh9Dn7lLgrovlo+P46W7LDqYz1s8iJIxGNu\\\\\\\/VtWPa4Adc5gbHdfJ+IUBFfvBwrEeSmlVtEakSB81+T1URavyULeZcUFh259qILsALkphxnI5tafolp6orSLgZ5NKXfJ4toW4o9oqTtHQb4d4x9wkjG35pEVSJLXwuU937Lm0CiPUSm8ChIfqomaL73tg6Co2EkzLs7InDSsCL6UwhEeepfnCmW7xkvTy83P5Y7p6hvYtPNM1FSAhhDIMDHeKJ9EfS7dFx5aeyaQEpdDubbw1Bx2fyiKgpI59YnN0FFFgYP29b1Yi93d86pcR\\\\\\\/On+680AsxNDXDD\\\\\\\/vXRXN5URzaPs7wjiLb0+e2aECocU\\\\\\\/bU1Mdk+5R1f80UADS7qKwNRrsHvKkb8Ep6X0FGio5sUiwpGdlgXL5PE3UuDOfYq\\\\\\\/m9DozRwCxGEv9z90tdIXctLcO\\\\\\\/4wtvqQLTUduIlDhL9vRCLPO+ml4UTi1Lfznw\\\\\\\/gM7ZYV09RmpdqMz1eG0d6amo6J\\\\\\\/xkSR1ttJ212eHJh\\\\\\\/tkDc2EU2lqfB9AEdjovtBVOAuA\\\\\\\/MyiCC1lLDP\\\\\\\/sufK6iPKIo+WhaqE3Edi8vyxpcEAti6gpI\\\\\\\/yPED0ZfYAntt7PF5V9HERfWFhxBrXhdPqJaSQXBlZIWaqnSrwA6p7dIbGqy1y3WBQ7NHAvBEWrvhHS3izmUq0\\\\\\\/oLjfn0q+K79ANBDd4PZg3Do2XzVTCIu9g0JNBaoXeYVVQl5Alh0AedL3WHsegW0PruN7BeAecWLZ8y+007uT++Ewl951IpanDuRaMpPamd1A1o6LaQUX1NC4bkX1qGwIgu6JCb8jiW+M45WathYPqBOu4cKTIf5B+9Yfk051cX+KnOz9pFPja\\\\\\\/q0Q9gweuhR7CRaYsb96gNevu47g33go7B2vz6\\\\\\\/T1zxUsNR67Fwmf6VBqe\\\\\\\/K02OM\\\\\\\/CcT67IoBz3HweI6fxFnuiMmcnG44kVAjjHb+PTyKw3zylxnUJ4GWbzIIADepDN6gOlIaJeGfqccgsRria5jaT8iINFHA3QrDiZfxyj0VRxZfANteTpe6Uy18DSq1Wuu+A4LAr8y0T6MrgdDX\\\\\\\/QXdobQRM2OycyPPh7gsczrO9rqfqrU+xcKVphTp9XUGnD\\\\\\\/Pgocr0YoramaB4r\\\\\\\/EpehTemjIN2qSyQvGNvmtP5tN1sYAhxDRKIC59deMHr70xCCTnyrUJ6fmHP167Pj5flZiI1+3xIIykz36JZPgOFNfZi\\\\\\\/utZ3hu3ecuKhtrMPgO8\\\\\\\/ZSuYLxdLte1yKo8xL2WcWzYGwENY01LgrtB2hg56xTbBPm0q23vOPdn9zznkXFf1xpq8aoomUmKJxIBGdpzvnl995tR7M7hnUndxdaP00zr3khOXIkc7oyxLagSAlnxTFsNaIZ2+Bl3FRJ9uYtz\\\\\\\/Q9whJjFJYxf7dJPaVsd2EfJJ\\\\\\\/NvfTtFJQA5nqloliGrzKBcuru1KIjCUBPcy3HDsG2Z9dWI4tAfcgDu71yQdB101RAk2uGYx8kApYbyN1EbAL4DqzkSBMVkl+OI8kDlmvXfv33XZjbCC1zik1Sf6rmQbE5zhq4EcY7UtNi6YNaUw\\\\\\\/MLlfD0VCbu+Pg5B4h+Up8lONVapHOr+3OsueygLppndl+POICQm06B2sk0Q3euh55xXGM52cyQ9+5jTISW9bknFZpjj\\\\\\\/x1eGVBTPl7DYQWtqrNL5\\\\\\\/F+0dZW187ZUBvPwSGlG1gPqoR4mTR0+ErPRXQR+mBdg9LBkCsaYzCjYkKJiXQtAPVGD5+BE7XDr\\\\\\\/QGJeznjR5+Qn8ypj7MKSy7rt9zHIpSeNzQi7Gq7iKCFNHKI7OQJwWR39znRDDNLYXW4gk78AKaUoTD9tiVzfDzsXYdV9Ykb9zIar7zyIZDql37Z7BV35JgZfkJlIV7PfEgkSWLocHz7gc56NZExWg4viYFZ9MpqJkfmg0AMNzt+Lwtfk0Y6q22tU7m4o7ykW1GN499SOErqCjdWGgxzfpCrqyBoy0EvOgWEWcVMo4xhcQrDkLMmbAwDbfvblDkVO7Vk3gQeQxUftRmzFVkWcuOoj29hoDVAu+3xZkDA+A9+sDdgF50ICsLNDj+xdYvyOCifW6l5fPlsO0tp8j0rBMltFfe6DeqVZ8FgtKkSpuFsi5BCSsEfusc5jEWc2j9ivdYWiMzJSZt0OPqHWGtMC\\\\\\\/eTZaSvHlhLdsb7B9BaHQR7Rfq5cd4RJwUtbHzIoutPOgm6M4i9Ghr8m3r3Msnhn6Uu\\\\\\\/xDPXbuFj4h3lhYj0Oc1zjgI6bquu88adZio9Jpwev8AZA3IPl0u6gITN5iaFFeuJSAvn34rE02wsQzqFFX9iqL6AAYApZlvtm4\\\\\\\/\\\\\\\/hmdM3Fxtr+kEesPpH1ZGPRTdROfx3ldjb51yTlcJvvyKiNGKMcPJwhlLb\\\\\\\/1qs\\\\\\\/vllHLfib\\\\\\\/83dQrJAMPsP1ml7CB1guYnZuNmVr2AXqxmDfJwvuD\\\\\\\/DQdIFeuT95IPMaf1uKZGMiO4m\\\\\\\/rEUBfC2lG0oaeEY+hKa8LnT9iRelfohxrCc6gmRUtyBlmuBXWeN\\\\\\\/YGEO5SEc2UtZqHtFNRDv72dJAAHsj6Dz9DeH8LzvA0oLWFgHPfF8iXW4Dj+AfBTO9\\\\\\\/CV0megFM38Onu25ZYTiJxjX2cmoEz28mAgoBaSrTBfLQWpexDWVNA4Jed\\\\\\\/CTShJcKr5eRzMEUhISKPYvgmhhJ0g7+7egPWMToU7NzmXqkA8keEUYXuZqmJYdNK6FrKb7D1lg6FXTaSUiIlt0uv7FAsQYlxyN6PxjCMc9Cly+Qzqfl7bMMqbSzdqKZOZrmVwtEY0VsvlyNaAX8A\\\\\\\/q4cdFiPuL9W0ZSQHAy4F+XA6ih7TYOWQs5ywV7yGC8cj1Z\\\\\\\/iluPgV6JfTA5wmkXX0cf5nWmNXcmqYMMSo83ywxnNjnzxZ+7luHFix9Xm08r6TFtna43n0N8D2raHLPv1TpBu612uoWkefaBgigAf8EIQBzMEUqOHJhUnnDWH59S4pcdF38ShHjjzkz5WSFnR0R2I8QY56SHPBdS4+La9HUXWXMYnrvHtWgBvm\\\\\\\/gawjuAPy8vVFfQs7xcIzTZk6hn+scgnrePrHvu1tse5KshUcRNNQnlH0u3\\\\\\\/Jw\\\\\\\/+4sBtRQEiEb8a0vmSYRjL1N2pa6jeOtrEzzajkAJs6TPSkfXj6rVMDRpHpI7dVvbU69k\\\\\\\/4b5c5h4WnBPuJYpAVT2Zy981DHa2GJ16bFdsMDeeCGz0e0wn3U49PFipxX1MuW7j7Q6Uennq\\\\\\\/Rl6J0OgMbqprT+x+ZFFRGsLvBfD\\\\\\\/NqpRQh\\\\\\\/KcmsimRHicfNJTLWC+j2xWyQN+9HnpjWRrzKbx8Pw99dL0r2WkfZq\\\\\\\/kMiU5CgwPVb3xLAW1nXBxsMsycI5Xo8iXp2a8umghFFNDYMTx4h1R9m4f+FfDvFRb91Q295jkwhnUbxbiEbVFUtuC5bxdXsJDzA0xOWOPOj4FeaHxW9wYPHFC2iZnbS0fagydRUSkkv0Hqutc2ed2xkMuZ0eFm\\\\\\\/Ivm\\\\\\\/2aAgnGkcKB3VFK9eKVBk2lBI2WKqI0f436jb6kRnoSKoDfWsgpxlgSAVscsEE\\\\\\\/+oBefCCriVD\\\\\\\/nP8PMlSTgHUVihdzYZieEpd2kJXhzh06Z2NuLa3gvOSootEj4xBlfhDpBq98jgyi6Ct2QIUHBdNEODUx6rA+KeZz7iG\\\\\\\/2pRWQp8Y8WKTiAU\\\\\\\/Ij\\\\\\\/Aqp\\\\\\\/3BnphfyPN5ReBHPkAXAYpobxiEp5svyEqBNCeZ1wq3BhM37tsN6ddBgVDvzIMapg2hNxpzJA+HhajcOZk2O4a\\\\\\\/9Y0zNS8pGuMtaXDlHE9XTdi2SZpSSGg\\\\\\\/RSyiZiHekxp8c+DDtvE+cvDnGquIWy6rkRl8XeiI\\\\\\\/fF+NZP+KuE9VOCHK\\\\\\\/Ifygt7QnPFoBObqfGep3CfKUkroGwDF4b1GGKTcpMQZYxOjfUrD2V5q+CG0jU2Qj+vSljF13+bLOkYZH33bPv4w+tQJzW3UhCW6Sw0iqeY7bm7uuQjjhIEYwNCQs7TlOUddElSXF788EI11\\\\\\\/SLeriX\\\\\\\/mbillCi9LgbVzpEZEcsHuOPAOrf9g1VvJmNUB0QRMvKYo00opuoTm1vo537at+7G6och3WKKaW9ZOw2HgElmwwPnyPckYUfit\\\\\\\/053ZJQBHb3Ag5pkrU+MdlKcRglMQOWh3bjf9VZA0S\\\\\\\/sSOFmXCcRLnT5FSdGB09kFOvhf\\\\\\\/7Pin18HQTNnX0c1qNkyh7QpVhB0m8Lx4cKQLhK8egtQYq45o7\\\\\\\/HTbXIx7r34Q1AtDTwnyKNi2XAaE476WkedDCurF79YdklOjovztw9TdtEUEnLnrh9JviglOwH\\\\\\\/XGmK6h\\\\\\\/DTG8sVZNCQQMw1l+\\\\\\\/3v\\\\\\\/YAT9IZdiiv69uzpygp8azmnR39DeXpXB9vyxUIzIwXvAXenywb2FQGMEBqrUIfWOPv4GqwZiu4beY7AFBzov5WeFNwr\\\\\\\/Yguj9YMqzf2XcCXE1DYe4VYWOA2HlPIQvfn5az9qsRDO8XbToGn+aN0uJvJ94NI3dbOYV\\\\\\\/oqQtCmF8+mkYxLs74VTvCnTuv2NEAuE\\\\\\\/UDlZQrt7LIh+jqYuBvv5evWMVvezl0nY2jbN5UqOGaA6CaMhlWIyFDxPSo8LyyrtcXAbvUvdg1xQH8184r6pGQnW+oj8yiCI89T3kUDYsiphtZIa7Ho3SaUzfyzlWwzak8r6umXP1UhxHUozUTwKtxZf5i8R4M0xpoposjkDGONRd5Vq6FhEhJkbbOVlHLm\\\\\\\/Gy+LF7Vk5Qtu07na92ZKKDqDPAUN2ESI0odYh7KdOJ\\\\\\\/vXJ74\\\\\\\/C5qWxNYBX+6Ci5BV1P4f2x3rUEJLVI9EgQ3IRBjyG7z8HsxAtY8bduayz9AqWZc3Yp7+Ddi\\\\\\\/p4lyai9UMGYBlF5Lk17aQqnCvQmE6dy\\\\\\\/Jbdi5uCyxgGccAxsFbWZTVgSHODbaKXNwnuzIOdDwHd3zj3tPWUf5QaGHy89Z+sNAV5pfnotnpG7nVBfxzOZkIqT\\\\\\\/YjzNefRvT3tCDBvV0RYiSp84Pd5GoIksiApgsQyni7hhDHNE4TffNpASElqeGHP\\\\\\\/o\\\\\\\/hb3WTuHisHeSi9CXm6RMS\\\\\\\/Td0kfsQE3yt3yz\\\\\\\/VdiifVngFVS4KkWND3tG9vIKs2YF\\\\\\\/EWHjUzCaOGu+1\\\\\\\/W+6P0S2EBBhjnHh1IzeGOev9+7NaOhcnUYE12mmSvjJYZpG8jdEQalLxH5aXO+n4Ju5cTlJzYAHI2inesbm7BngjlAOL\\\\\\\/tvuNGoWKRCCC0PSjmHz4\\\\\\\/C8D5GTQMF9joRN5UMg+aqS0LSRvbKeomCCowKU4AuXraKr2osZyMC8c2arCC8MoHoYKK0\\\\\\\/\\\\\\\/FCtg4cNxq67\\\\\\\/PdiYGYxjAY3H8q9Qzug349Jnr8SREchkG4LAQqB4uje5eavlk1Wqx6OzBCcxu\\\\\\\/CYxEEh\\\\\\\/JMJrpOO\\\\\\\/utRX8ywaxU0Thx7UfcyYjrXlb+gJRcbza9pu\\\\\\\/TgZLQCR\\\\\\\/4jFIymdswp8JikvpVD0PI92qmnVmnFzQspoFiWsNt2PyBpnpiyO8BvQJJAvAuu\\\\\\\/LfhjE+HcIRvUtQDRfa0bOeb4QnrIxsv1eNH5LNsnssiTUIZcuom1IVU7IKbUiVeCCUnq7n+lIHUGixpRdhasBAFfhgXpUzUx4wsjFnrh5GeFaXwQV0tY9UOanyLXdhGaa5+JLUYYUd6+guJDFbra75b01Q4hHh0CQFShNzecVuU9G7xB9rYgw30Tr\\\\\\\/qdeigHfLhajdjBxpcMh5HOQQexAdsDKSagDnBIiRqaTzcua6jPGzE+OtldyaKh2ZojZ33oIQjNBCh\\\\\\\/SXmKVNjoEploy5\\\\\\\/pkLQXKJXdigAxgc\\\\\\\/TSNgvtSCNH0Z+3\\\\\\\/bI60ksmwsWt2omx\\\\\\\/oDYVWke5+spwVBRyjE3tPLR0nnjdtna+Hac72rWDSDm8Ma1m686E84vNrHZXfIGnu1z2d7wb28Nl9wl7NhMyHmFmoMQd4tjQkYCm\\\\\\\/FREtrukbcNzXF58zOyMqu46hRXlbl5sq9Tor0drYebz0xFh7KP5xtkmwYDkmyi0xXfvx+XBLNC+MvimEqOwTMRXruyBE6vjyn6JLPk+a5a\\\\\\\/Os7YSlyKTf9ylyK7ee0lX8VqdDwGTxfGdN8\\\\\\\/E4ptOCVr11YuDtvsKadKYimRZt1gWZzOTaJ8eQTmq9mKdbnfA8Kq2Iz+UEJVI7ynMYyh7wSFf7Mwqi+nMp2GYwem6abBPMvJAXBCPj3f12LN41ss2+pyZMwbMDXFm0om97dfAEpjmtC6f8oAu1beUoWCOXNTC0OpW1SGyqZbemWzlWu9iAmwPQ0aSQtflkCw\\\\\\\/RFxfA3Yuk+VW5shK4BBaepH3T+daV4rtFJdciIzT8nSArpj2582ChyHyPvBKPM55QLrZe8cSOlKa6cDGe\\\\\\\/JRXGLm14qNZPYIXiO1h1VPmDwnwYsb\\\\\\\/kKj32dpeNelHSYU864AqFljhawDdr2\\\\\\\/CGEBdgly2vZQbOwyVhEgfdykOfKzkekIYE7lgY8kk00MkNuLgiokmdq6vX3Iwtk2VHaSTCjefRNwclWU\\\\\\\/rLgiYks7x4HL9bYmYomUq\\\\\\\/Ik2x3zDOUDjgJR6cQpg88UyqbM6D7t8edtWl9bIsSYWQ+LoEMwRTO0xsU9dmPLnctEvgK3S\\\\\\\/kmAYGCmhddB6nMy045LM9jiJL9v\\\\\\\/Owgb5ypIqZW5YbITD5rIl2pZsaKn\\\\\\\/bmMvVMAQhFhFwKjN8o3xQ5uhgkRp9qno+nkLwA\\\\\\\/m1tkUQNaJ2dg3HPnYavIVWf2mMikGHeLc343x2a5b2XzKJF2+UCYMi0\\\\\\\/df9szEnCVVHFGMagnJMuhxPGMsTbBPBf3\\\\\\\/enrUZaLDK1I47wlrjkfJu\\\\\\\/4BZoGGxHwrHmwRUyWllrIJcOgPubaPMe91bdGjbi\\\\\\\/+Ic1\\\\\\\/EoumVc6D9xeGoJiQtnzkkIq8NWwvJskpAH9z9\\\\\\\/2+FIXTlb+I4xcIhmqlZhyQ+bM\\\\\\\/HGIwHI\\\\\\\/xnYCu83BSjF90RSrv2hIk0\\\\\\\/9mMctyhSpeZFJoviOGgXt8T8QUrd6P9X+kJnUMxx1hwTzOjro2\\\\\\\/tWFKB1IgUW357RVInmkY2R76ttxz4c6LWS5x9FSCXJ5BbPadhkv0GQ\\\\\\\/vDh5Fr8JJYS+XpgVx91ENuyjfo+z0Ky+IrGCwrg9mMIF0lRYcKUP0B+ZotBCCs3tEUwoZPYkj7h9EyxstbO\\\\\\\/L00NFFdbUvNhTexhjcDHoux\\\\\\\/Cy63UeJTO\\\\\\\/M9m6g9O0qN8OqNcJ9U+gpA5lDZRIitPrafa1Ntc7PdTlSkA8j9FcRW4f\\\\\\\/f820Nk3h9Mf\\\\\\\/hC9BTBm2hffnEFGosV1nNJNZAiLk9TcjrCUdxzyFW8SW78WtKscEZbxu1AZpH9mrhTrTYtIgt3pTOuX0mqiJEUHUNpmiRsDeSA\\\\\\\/GbqOPNCWcDqdbQo8xQI2TwpqkbiiF1sz5\\\\\\\/KJPGl41g9nhBR2vcf3qpnUg4GLxof3y3i5Qy3K9zFyDRoBlmQrXA9DligMxCM6rEC7y7IRqwJkQudrdUn37t\\\\\\\/XGBhGXu\\\\\\\/+MiKGvMKyyztBUquSfz94nbWgcvjb0kubACjhJxcKmp1OaXsOtt4Po3tiVbORPvx1LkF68yFUfZcnnlIHyzxrwOi4aZYBnc200vnbhpc6r\\\\\\\/zqv0900c6G1f4rw502DV3gD+0Pb6x9yTjO2Y2vYWbMkSyJ77T8qIlz8gzfgWzq8l+0JMx++e\\\\\\\/g+YnkvcsJUfBgrYiZu9KeT9yWKejYyrDAWPrEiQZrXFKdKcqlIlgW\\\\\\\/+L8XqgmoU8Db75\\\\\\\/1SrzCB6zXvfpGynMlOkMqiA0Lk2zK4NTEmJiz3UThgRp2zni7Ta1ULDa1GIqALt3XApxPEHTOKcID+Skf71y5SX+5vbnlo6ECuQoGj1dw4i2+fLP3AX4t0QyodOmrxvErCSjVPyx73KQpL8InToNfrNDM+xclqnpRzgcNnP7e7aHLPUPAareVHUXP4lu8jhgsBF5ZPQhKk2qgpVpZbC8hdT\\\\\\\/L+2tWldc6OQBjgqL6ovYhoojT66dU+THA9zSrjm9Wr9tCTwjNoD3qc0+Pg7c2hqujTjCGE6GCMHC1OVlE5kUU3XUcmo2Gpn0IGFpLi5pC6kB8UOcU9ysoJQb7x337gntlHCcKYrBAGHj9LPT2ytCNgip3bf3n3t2dnbaDYCiaoGadz1DpxslKnzg7TAKf9urtvFwGMfpIOQYDxeQU1j4FuvyYoMae8JfBrOKehmNBjUk9xEo4BCfYR5tvVWfzW4kEDHmpqvCsteSmSKf\\\\\\\/0dee\\\\\\\/mVVURRSUtgdPOkaYujMtFuxd3nf0Ab0KMGWMiIKr1cXSi0zseV1IRNJ4UUCj1emXJcHComZNeWB4qfxKr4kb\\\\\\\/RW0GAmcmVadWnrD005YnKnAoIscxbXan78\\\\\\\/2H3HT4eoFsq+u1WKE6HzHb\\\\\\\/cpRNa2sg7JL2q9VrnAwFo2Ar9Fz\\\\\\\/mP8vin4Ai0ibzXL9gntUfZw0Id+su6xSu8LUZoRjXs9etsDOWjVShXbCIq0BAgJLT+IaV\\\\\\\/NRhYAqzMiNFx+fPFa8+Ye7rQ4AyD8ibSfecySSB0BBKu\\\\\\\/HwdTc3uVHvKJh1X4tJ3KJLUurGk3TRnrZVWo1HMnuzGQxxrgh1iJfq9rLT4Q1dJBE2a2LL5ntu+Wg\\\\\\\/lxt\\\\\\\/tS4fhP7F1h9GFAG4hHn4J3MSQJ9xowhhXGDcSBI42phVXY+llcV6Oim7RnB2383l0fFj\\\\\\\/jHbbHWg1deY+m9nNgBNaBOwYp5R4DDNepAWtVOq78sfhtpN9JZIUKsOYQR8t3tiWZ1ss10pxxw3BbRGfElclOiTldWXszQymBiO7tyae6Jvoq9aJ2kRqmqIpC6EsIv6FBSWeuKwT3B47VurGSKwGU5VwbDJmaJyWGjZg3ETPNDaCDtOi3PdS+GomaxSi3kDyiwUx\\\\\\\/AlDmwhuxPbOdkH253+Yu5HI+JPx8QT6NXkm7PS1DaZwm+HfU9U\\\\\\\/PT0yyKZdXBpDF+61L9IOJDITb0rISMDxueEpD786qf7Xog3lOxlUxjxjh0D12QrIZRcrPV+xFi0fu47hgnKAg6GoYuF7oAeLEGzEMyVJPgzYWuxYyOUxlSJ3\\\\\\\/1ltiXf8ELxPwHQNv7it2+oQk9\\\\\\\/PHAb2ceiaojJvejYmZa4K1zGUBXlBYsi9oCzDhZq1xNW7WslBpkJuhAtsXYJBgabU7REdV+oEeWBgeCrUvSk3CaXGY2KlNhG8mLNSVdukbHyg\\\\\\\/l9SBYOOo3Mb5sgQphBmkediD2VtOTseVAD8hlc\\\\\\\/2PxQl2dNPM7cA6ogG7brfSY4AbVzF7McVBGT+LQ1i\\\\\\\/N4tU9TOHBKJQmb8FRWg2CPTmo7nnSmLmBjtabaCE6Mj0uvH76aL5PLSIjq65oIrvCfGW8ACrbHAz0+YEv8dlyusCrZwigWAk6EKZkwi9jrZxUAYMpp9dwOHloc0AFRXi3SLoXSCYCkFytmd3J4ikwWSA\\\\\\\/7Cmx8H7aawSsLxjt0LS7WTVDmARVk67qP2k93wvL4y2vRsQx7bIPuKCn0X92MXn19eQuZAnLG8\\\\\\\/Q057bRKFo1kI9Sk9LSaEtRIN1d+mQCamuXSEgNjH2U\\\\\\\/QKBA8PlcSWgwPDVJbCYPLayJLvydp8VRXq9jkastU4\\\\\\\/xaZeXWCuu8XduO0XCOzaNaXpjPyB\\\\\\\/Vqzhe3y41aRlka+DKaB85QISBkFkG0SpmL\\\\\\\/qGlt0Xta4nO9P8T+Begb0fo44PmvZGeRXEIykLp0QtIzP3OTmV2Uf7CFwCk51+fbLAtLfnWFs0VNYjSGxXrMCzFNf99dq5ilD9\\\\\\\/yl5rLu5aqRSwrPjDcHC3P8k3mbjGmTJMYK+yOy0z29WB1SP6x5Pqb2ymrwJ7gD69w6zbzjGltw\\\\\\\/8M3e5AZrfQwf0CCZX2fXMds1XuKKhmS0NSQqkcFbU9\\\\\\\/hB0JXBDWYypTGM0eKu5CfX52lw8tm7ARlLDA1HoVo2fFFGbswGJp4yL6REAv0dMUuRvzRSvTdYs9YAJRse6fvUmjY+nfSRi\\\\\\\/3B1lNFW7J9NIiPwWVixj9h0IP+JILsW\\\\\\\/qSXXTvU6LwnHCXofrPlRC+E7AryuEkIn5yr+1ofo+1aciLgdSj0nmNgwlX2MDwKb9YOesHAShO3m5SLyMRV1uyr21HbHCYb7Sc3Cq1z8QDq9JOJCVTMsyJIE4pISz5PHzFzMrowpcQsyabw7g+jwiqd28kHqtzXMCmYGwNPbELmX9d60knUpY3aQbrQrECN0ASQ9dKPSEMKL+WE3fb3ms\\\\\\\/KJeJrc3tNkMxkcO5C97myaY3vQ4Kix7HWaRv51zItnZviBoPkSIHJcd7dlDMR5xvsgyjmQE965c6TRl9nxaJa6qjxBEMRR\\\\\\\/25rznNCiU5FltDYLN37IPmW1VC4sTOK+RnIPLJeXTlaIKaApIUbUyTHc+3SjSLJvVNXyukXCEEBwHNKY+i0Mu9klmH7Gbt\\\\\\\/9feKZGMmE9gPl91AdiXUtjzxQajlStk3VpAFZm0Fc6oIjUWQPIhHf9PyaEP7gQMLRVE0GGqmJXr5teDajyNGAu+9KF2uq1lAyDVwU+92jAzMmfwpECu\\\\\\\/5d7z307sa3wSPkdTTPghryfAwL2AorDBReSM0fEMsWh880E488llDexGlpv7Qk9JiXVUNO+Ndy7Y+NWL0bMuGKaKETBBqefc+GmUW+OSoeGuS\\\\\\\/9FJiiql9\\\\\\\/0PoTWqalDlE6ARG6Q0RwIHwIeL1Wn+FEn3o0xn12FjJw+C7u6o4ZTy+SDzs5l45JGRnAgq9DzTTERluIJflIrCx4v+3pbkVcFTrjjlA7LsciBviGNdwpHcSQv\\\\\\\/f5kHQnF\\\\\\\/FVx+XqYfp+KHuKWHdig\\\\\\\/hPUwn5c4QnvcOk\\\\\\\/apkFYluv0fuKQxtMjiUcLi0zdupcen6NlG\\\\\\\/GuFsFiJ7snbMbiRC9TbpTa3TCF1+iTpylL3qEoBRl\\\\\\\/in3tfhPO7nHnI4AWu3XocxPNaCWSboUBFs9bOKCS6eCmwsIdg\\\\\\\/HbsQObSiMX3VcMQY3C2tBXiPtZcFICcj+37hXwKQCRA\\\\\\\/7LrTX242pVIpsNUBcopdtwaGWP+pBso4guf5neRd6D+ptV\\\\\\\/BkbUR4nOkWxJNMKITawfC+0Mxy3NvfUbLysJvF5O438TaGhskLsvazwneEgsijNwBdZJ9X73CGQ5YawK2DafDe70XErPRwGYY8lPZ0vwsHga5KelKcBjTP6UihsZlOKgkLJLfLrBXLFVUCiZkNuOyKtB1fDdxfLttv3HJMazHgikhDpMilQs+6lTs0lF1hQJt9KlMLKpDJdkqQw69J1ghw1PD6xWOsFlreiKIpe58S6b\\\\\\\/PVBVtGuYZTYcpiyN6KZqcoBgHPA47vD++E6EbCC2LYO7C+iQmh4qrMfRkxCGtSUj+sKbaj6Y9S5xNxbsSwH87gvf\\\\\\\/yOto9Sc2\\\\\\\/JEG7EPntAY\\\\\\\/l8xbGGcDGT0UkWWKXG3q659QOTR5kmVfSCBufbcVnBsqrKXA8Y9t2Ilp9TkAB4b3gAxqmwB\\\\\\\/pyaFwChc0gcIPkTLsazarbtdseUXswnn5vjlRliNrBGyCz+1iG6iCnuUBNGzQ30ciBVSUjSoSRuQ\\\\\\\/iMHLHkt9XNg0RqngXJGPWn1WtaWVLeonkktqQXxHTWWmdaxz7B8FeKwHuRAIAiuZW\\\\\\\/QaqZ7\\\\\\\/mWG1it\\\\\\\/27OatST8S0JOZV6YxiYY\\\\\\\/mm97hZtROwriyywnJYcy\\\\\\\/k84JNFWsJOJEqeVIhGZ5CyCT\\\\\\\/HMsKEed+53SjhHNCokDbUofL0yXD7uLidCgOpT4w6W1iPvwZMDFyyjj5hDWcmqMXGNm223R+V3Nc+G4h9Rr29HThNn3zWgEUlz75+i5o+qPkr+FqvoAWAmD+6DgqJOOYAo8Q+bSVd2iOhcecIa61sJjzq9OTjqNt4Yx5VJIDIdh3B0iykTDeETufqQ02qM5jTfjVTFjdmLGx3r9yPoiv8yGuLiyxY7LRkwMZfgwYhE2g+6rQdJJ6qAhXLtzb18muVAzC449YkIl2GVoE9RRv\\\\\\\/NmklvDeIX5bXoUZyKynx4MYiguqI0WjxKfFK0oFjkuMuH7Qtmb0oRVRVB4\\\\\\\/ygioxTrkbxk1YvEMQwFMx8qW24HhZ3Pal+rRpniv3Llrb5Y5ESav68gwzPYsYdjPTHpzRTQGXbC1z3mXDhXYRkQwQMceuxYvfPG8Vax8AOPddsAz\\\\\\\/lQaEGeUnPZ34AgOGAxiYF27BFBsyaO86bLndhuoCwXXQ7jN4oiTqA0dV\\\\\\\/qTbQnhDLVNW5WVRZgNKfWporwsmJ3vEBofLwVSZfVXL1v2\\\\\\\/rNFUOqKZVXnK6Kei4UvKRM2A8iHEI0v0E3kLJx1qZoKcT8mEbevzDjMZm\\\\\\\/hU6GGy0203XOYcrN63eRT8dHmUpHrr9vkKKZedXj\\\\\\\/lmQRTQ+alwBFP7bFvd2jR9VxBB+5hGe3nBhF0df+nLfv1z7eQ7sK243YxnWbMETSwD\\\\\\\/MVhhM8vlgll27M4nLWDfEB5pQ2Sr5263iiDAcTXIdDtzK1Q8\\\\\\\/pX6SdIzc9iTcC7FnkUwIHqtjmrRFe1oK2xK+zDO66MDiZ0eZbYn8\\\\\\\/5+tYVWywIeMa+8a2zm7kwGUzV\\\\\\\/HYP4NB3RAosYLSbJvmihQqeFqlszVIQYoqeeU+Y6fmdPHvYF+W16622s4qIqykZb6QUtsu675JZbAG47vVZnCUtnn2037Gl0smag2CizVohFgWWARn48b5B1eHjBc4cjGlVLH1bZ1+dMZL4w8WtJ6NNIRoNQkOd6Q5lqr4Cl8NcAemuGPH1AENRlZSge2ZYLaUW9H6aEW72Q5AxvQZ\\\\\\\/tn6Hb5+CxIpTXBXir8lgi4VxzAOSOy2BG+52YKK6xS3g6GvEyTAg8aaM\\\\\\\/zx0hQiafzR3BFinHNLSUix\\\\\\\/y2oB7yn00cB04prcq8dqHgaBJBdMChD3k37xv8lv4HOCZI4DPynCJmxzl7hpGuYLhGR+WemTZE8jQ6HhPoJ19A9290Z12RtNz4JCTsViVN7Sgq4ySR5XIs7Ir5fdJKIy4J0vGpzrNXbH0k2\\\\\\\/ey15wqbr8bu\\\\\\\/v0cBaiYTHuJK6PJHGJBE+SSJHWCp\\\\\\\/Bop5sASuH3WlJkdwpIi1Va+iTJtvX3evyzeLoG3+ha6aX+bWL2tt\\\\\\\/eg9J\\\\\\\/7XKuB7mXCEvVr0Lwi0TTRr6o07ruEh9YOP3zfaA9iMznYEVUJUSl64TLVOe4qQRUVdlCKEuupAuhgNlvqLAa\\\\\\\/N8WJ+46xFVNmHs6XMxPDgS0VGjTipdllSBkWrDGUy5DEX6ejQLfReyK1jJ801Hc2uIGk7gcXa724OAGKLSFHbXdAeULS3gK+x0CuVuNxQaXm52XJmvczZ4BmMepkvqASFBB9DFKbvwhtm4IVzNyv6hDcuHMp5t5OjoTmcsttKFUKt2F+qAg\\\\\\\/9JIy3BfTSJA61KmArEdXbLnTY9\\\\\\\/TTyPxDfcH2Hkl2S0NbaULEEwLSgyv8WfEIcTx5DYdn\\\\\\\/HYpcRclVsgowl\\\\\\\/2bljL0uaPQJoQLeJGZkNPEFHXPnT2NIcCg5Pc2Jc9aL6YGVKXuOMjomYs5VJl36xuZpd\\\\\\\/DDSGix1TIjDd2Q4ESUzpWdUTWd9BKbUWNHvmHlcB8RxKSw23F3NN415A5YkNzfz5GUYDQ3VI1eBCAzidb36w9rHppfAZDURtXDDYPAq8UpH5Rl\\\\\\\/imTVeVRNx1u9a2M9YHFP0hMwapwXPmRZyvqMToVXGdZGAne1Yd9+Zxx\\\\\\\/2NRuKoCZ8QMtpMTXlJUeZCGBNajLBe0NTmCm2Mil6L14VUO8DN15q82HvCanUebDmPLR1xVcygNjp\\\\\\\/AY8bU2IQJUYqA\\\\\\\/ICGRO129odOC+80K8bCgDE+55otoTNC1Jf3\\\\\\\/PkkzMM222dGhHjpY41guf7bWhZ\\\\\\\/dPL\\\\\\\/HQ4KjaaRoqcDtU4bp\\\\\\\/eqwNBhqnw68kXco\\\\\\\/UDFLuVWuCidvQ0\\\\\\\/W0FuqvDgK2XIiiR57KcKc740MJj+fIDZOFOjVpPdWT2TRGIx5fCOLFE04go9P4pr0yRjQQdoaF8SS0wpLOqRNley9oxKpPFbWusIzakwbcqZRY3P\\\\\\\/350BmB4vojiEqueQYJ1GHcMY9GnQWIA2SF4kbheDjGbW6h3bnV92+cgSwxgIp4UrJNg0cXf5vGhDeOOimzLXZgtgwxOPJpakTxx7xqenczWKsBLpI0llRwxPT1HMx3fdlH0OKC71rUzYsEmOnRYMKAjv37c8rid424ajjYASILX9\\\\\\\/O+jGqGKq\\\\\\\/7slkssxzxLyQEUmx6TU6dfwry25qNfRzWvHvHecj9LEkoSUF8PSGIc\\\\\\\/6szgkZ3YvLublbxqlBTbWpGOUJqOYc\\\\\\\/FLeUavckoRxwLjWJlZ2YQTSbLselzidHtFeAvfbftDr2gWyytstuakh4bdEg7hSI02s0a6hK7LUB0Idksc2ZivtWino7toaDjW\\\\\\\/2WB0XSEPPPenfKMU0y8PgPpGVjuz6v\\\\\\\/dbLW14hPGO1dzt26cl57dqkA3tCjljKe5GvQwrnegX\\\\\\\/hmQ92uINMOYQNf1DQiAflqEmOPuVnd8JbUiqVbniGthIj61zNCmL5zdweKNM9qne+sntnlmQA\\\\\\\/VpJRrAY7+s8Qd3bZY8k0u1B2QjzM8Ls3DYXMicKNWoK9C+DeGd5RvHxHR7IJS\\\\\\\/oveclW8btcUHqQIBofH+RG7T7h63MOH76BgVi57sX8oqRSoumcod2F9FlHgrawaQnPcwDIXHs\\\\\\\/FEM\\\\\\\/TRW\\\\\\\/b33u+fryX8YfB9DyQmFKDACKHGl8BKFBwIcz2P4RmHHJHO3BrbJCV4tOeyiAh+r\\\\\\\/8wELNu0LrYhOowrLPAM0gaVs5UDIe4WuTAy1OD1JoEUpjGaulmnxIUdrt+V9SzfhbKLVI8FeMyaiZx81MIpLR9vpDFVnAKLNNqW87o6yWlEwt5Mw597fCS6SWJaoahkoCXfyCI0KGgjDDGypnuKygPzNY3Dmdcy5NFFAcZ6Fo3x\\\\\\\/ecsV0y+EFhPoMzYVADk0TQRYZtKAXhpaw0b9mCVPctN9wZBjIFj\\\\\\\/s15fnhJdiGOoHF3zaQtXKz3LPOjShtBX\\\\\\\/3XV5B+mc93yzrSdQGlfamUUHGlaXqy7dvbaPsHnFhYbf6etcmrPWpcus2gVXCdFlA9u9v5pVUTZgkfi5uxFF8\\\\\\\/6XoYU7ciUjiA0fpL9JyCWADZ7iXF2Mbqkf8TbiboA7v3NTaofAeIYjUd7bLF4D3sC00Ubokh8PBmBV5XlpNVa4mwMnSq6akL\\\\\\\/Ykbemx7NnpvLsp0S4+T\\\\\\\/zKqZOGCDxB\\\\\\\/fCo81+RUpFQJIr\\\\\\\/y2qOTw9\\\\\\\/HKKLfFwccrjW4Fxq\\\\\\\/xe\\\\\\\/WZ8FltWzF4knPBC53uSwtNDxX02Ph3vvlB\\\\\\\/\\\\\\\/8VOUPNIRapSdm116xCeH6ciyTF7prdi8UBuEaBM3GBo7YK71g1EPkgyP3LNJXhtiDz+ge7aUpyOwjAY75lLnGTnHPnj9ypY0PUSliKpDF79x7svgk8Q0lHcajuw3i4edsmAyJ6DZCnV7ilmpWsXhXgJaxJJ1wMt6hZv5lTCSgBMMmIml2TZ0xYVHBGQOlUXj1Axf+vVfJQJgR3QFA8p20zYD2Wp4pijXGZbC882P1dszHp7NSAU\\\\\\\/Dtab7GQaOZU2kzB2ZDIk0rbBV4F1vvAl2EIpcqGZUXpAiMRyOKDUwfSAeu4S9UNVuVxJq+XXUDuXpYBr1HDfHAI4cZ+0839DadH5Ex5WoSsVnQtXQcFDw3VlzvyoQ0bzp5fTiwrjNS0uBZDXbMO5WG4iNdbzgBulRstWhqq3OAxs1lJQgMNzPaQ8KX24m3dpqgF2yANB1+ab5VVfHdYZDwKxNVBcClgaopC7XcAqIrI17OuC2KlV\\\\\\\/05YzBE1UfwIhWPdlBdeoJRzUmFiGSgGBHVVxXT9qev\\\\\\\/+5vyBRKVZbD6pLo0xCGmYSplpaStXhfeNUOvL0nkUHx+ttMrUVCXMRc7um0Rats33PORGplPGtI5rvLTExL7vQLgQIAQRwzF+VTX7BDZRjN0bVsTw1+Ub4Ho7oMjComMi3pGPWBMAr3E92oIfcCCRPYIvHwus4s3o\\\\\\\/IeXbtgkrc\\\\\\\/GS\\\\\\\/NVU+ZQWH40KCxVeI9jV0Sg3IcqNeCE08UA8wik3tkDNchW+hfLEMRwQypvTJApWjAWRlJBYsxfOwO0PUkjYwgyaAy2hDMeUdJNOPs0iMxE3uR2q+4dzWtaGBAiPo\\\\\\\/\\\\\\\/CgQWuV94q5VWqdSZY1f2RFrJv6i3EHXMm\\\\\\\/5OLiPP2wWd9KIuuOkFc1JxSp0JZt5qqtMstkYWjnGXB0bgcKwXZNC\\\\\\\/AeyyM92\\\\\\\/KaOh9wnX38D83+fD9HDtkz6u4gdo2hMorlA9++CvcB7rC3k+qyhcGwF\\\\\\\/8LqSbcre3Pi0jIjpeGkJ2mo14qWkPe381hbUf8cSRnKSuUypLwT9pwg6qZDyXs8wevpizK1vmomOFvD54BWOa23ewX81hmoYhhu0l0JZ9eEldO9qskDcbT52C7ehldqOyzsaIT8CWKbm16PVzmM4\\\\\\\/sRm+\\\\\\\/ezlQhxlQ\\\\\\\/fLuhW3MKZUfMAJCTF9AdFNr74sk\\\\\\\/vkygJkN8yaWhwXYt8A798KZb2E4yLHPd6mYRXp+\\\\\\\/ZL8Rhl8OkWyjBGHA8puBcUvkGt5lLctC23Yw+8ySjxz83yTc\\\\\\\/6XhhzPMSjWST1WKOF+tUAhVwYUGmRBKeoCSM6k\\\\\\\/Kc3ouTUkF4VfbRzYQSAjxOCcarwSbZGP2C1eKwKzIUtcOjq0naGN2qAYaG91dl\\\\\\\/FJ0TlsBO5aFeJCw5FWmHMPYrcwr5Z7jcxCeySE6N510ant\\\\\\\/khER5WkCqArQQcytKojDKML8VOYySb9sMPRZwEhgv18OCED5uL0PLpYhQe30r5MtnAqmP4UbmpDObHhxlT8wfeRfRVE8ZLYjdumheGZBYnXb7kVx\\\\\\\/7dXxjPM0T4L0O6jeuGEp5kiRsdLo5\\\\\\\/uLqdKzZ3HvnJCtdif7nGewg7aNl\\\\\\\/521P8Wc+2NZoMd06VSVVbF8b8YQD1WbnKL\\\\\\\/lGSY2to+15UFu2zvDAQZK6MkRCDeJ\\\\\\\/hIMm1d5++Htuu1mVowUZ6wZnstqaLWsSBPnZE59iufIZVJSc4\\\\\\\/B6htVXFaIoO+6t1RDVCqHpWN6PP4FJD4pxfTyNMB8amAbHNjTk06NGE4Nk\\\\\\\/\\\\\\\/SneX1WpuwVhZxkLeYLAnQ+PrmUsqi2xUp3qYEOLvxuhWypsTjYph+ZWP7DxmFbmJElL\\\\\\\/kbh9Kwcs40DVjhTZjj5VSsDBZuNcbsAKUmufybw46Yanfs96AzWBmS9B3Z7D8qk8gPH\\\\\\\/4sDHm8iNfQ7eci+4reelYvaBOKziWl4ia2LwTcn9hiCSbB2chD\\\\\\\/owkctfjbzB7If\\\\\\\/LVqyda4wcpEF5kUmfX8CrS74QhF+W1WlCIOBN6GgIIXxUe3FGcgTFHvDIDE0JkdRniEsV2aXcuDXyfTYsBIXqb5WQtHhFNzjJRyZuXhysdGsDt5e5zQbn\\\\\\\/fQ\\\\\\\/c5LyEbsCjXWZNdR5BXdREEmBEMIIMLMewiQYHKdrKka7DxnvUOZAByTYhelQswGximMXt+ePc5+teJ6NE5dIIf0UllVV4QCqueKZG0W6DmoU2xHJjbSmnEjlu8pYOMuRax3LX5Cg1Vi+exjIgcMJTR+o6C1LHXpjaLMnKcW4Wvac23cfNFgHlT2SE2XZDvw+99LhBJmyRMl4ImhYMJWEvjQy4EpMMkTKY7Q1y4h3FwV0nCUoVp7bbOrLIJEh69dsl5BjRRcaxrKnywkTBbo4cprwXrP2Bym1u91bUHvJsIw8GgLA17Ld0Z3c0th98BqLbrRn2jxfT5W9W\\\\\\\/igGB2uEt2jEcTkdvhrhhc8fHFPLjjd6kAzo4XUKPP2CZeCUhA9mkyxJvjEtFr6JcCd\\\\\\\/AcdatZ8726Jy3JUzOMLmlwqp2Elq1upU+nmjlKKV8jCKno3X5LxzJE9yblRo\\\\\\\/iymg1bNWBgUWc1OeYXWIP2y6+R46hWPAVaEuyG31vePUS9lR6VMKsW2izauZf3PG6vhvoG6r6M5UgDDticjguVRPMe4sl5NbnEBUsMHyjYGJD2a2RSndHZvkdaqkaJSK\\\\\\\/FhsWFYrf7oRSJ9oEqMju\\\\\\\/OP3ILP88xx5Bz8C3Qnfj5HbhZhIU2Z3SWCcqqwJGpPpnqLMxgzX1d5qwLQADT1GsrMIKxds+T2GWT8MJUOFt+iv4lj0Btq\\\\\\\/E7aBjfeOymQK8AuMTmSMa25t4AjiaatV3F6WB1XHptPtivc+\\\\\\\/HQD9gldP3fFYoH\\\\\\\/Zuy9mlnFNhkGoq0CWD6qdRJdRzXxJFeClSH6A1bF6urfNmLk\\\\\\\/xXbrDPun65de1ptzBuAJmRkGPUXnHmu68PZDZGRb8Iy\\\\\\\/i3K3svKd4H1vQk5f9sv5kmlYZrJeur937XoehXOgrGTndXhV7SEPYcHoKSUtYEG9QUwZJNZCLDHJQxJQxkuPgOoL1\\\\\\\/lHGTzZddAMLORWTuot4amXMl43YbLhWFltEQZ3bJKI+aCNDnz8QG4QxQhr9ACNFs7VnBtYOFyxBTbliql\\\\\\\/fUYFNl1C0xTdNnvA609XM3hE1lCPSkqopQl2wb1TG4mYeTLM6BgsJFzIX7aZBX3I2KUaQnMmb3UTGBoSk\\\\\\\/Uzp3rUUFTSlLgLdlKvQrPLZ6gELod\\\\\\\/W1SqhNO\\\\\\\/LFqEjgrJU6bCXz96NUFgsordFAYkVx6de1LOUqqIwQdNDm\\\\\\\/Wli\\\\\\\/aR38bz+7BUqw3YvNcgtjxOt1duIwggFTQDj7mNPULEZPCjeABfRztVJSi23M3F2QCXKlEt7tY1x1EYMaJAR3vP1ZiUul3NGYJwk6FNyFPKRr8qicjo4oogdmesGTpueDVzlm0Fipv6fGLywmfx2mc\\\\\\\/35YKpDebZgphiTucW3fVHeKy7ASnvzs0pRBlxxBi0a4CSoWW7j2Ubt7LxjROUqaP3KgLBvYADE870wfx4brxbZJ0a710EoHvWyeiZMYDGdyR80u7OjKtn6P5VwqCRcowSIC7TGYwoZnrr1\\\\\\\/M+i59NekRr\\\\\\\/1vP4yotvfd38EEBKKmVitXB\\\\\\\/rd7E0xR6IcVJ7phQQVntqyVcBBexPbxxeZc0gMbTpzyQuVNelVr7OVe2sUqHB0X\\\\\\\/IldP\\\\\\\/91T6ycJQU0Nf2DCJ2NFK2HXEyc2ylsUZI3A0NS1qZhn79vcDYhX5Px8HI+9SAGbVImsnRR2Tts7I1+347XfLC+jIG\\\\\\\/DX5XWJ+dJcSJsI8Hyz\\\\\\\/rC4HryhgIhFIMM9hF9M\\\\\\\/OtIys\\\\\\\/yjhIbzuCvI66iWWQt6wCeKiXlMXK2bTlb3bn8HLrA5vC5ondQRYra6virYZaFhUmWvhk1fpvaKtQ7zK\\\\\\\/aCBuR4AamLQazVm6q4MMA936X8k\\\\\\\/1VjVbAx2ZkQUuR2iVIWm0n5+lixDb6mRHl8bDUjFdeOPmT+E+J9EeFmCGMDqMP8b+8u1+7j1nYCXhAQ2Gxb9PjqI0Uea5pIU4i76DjQ8E7StskDG0g0IlPs0\\\\\\\/h5mJJlrETZqS3veC\\\\\\\/hQxZvwiuMJct6k7OTmzDEFzynpdpAs4dD\\\\\\\/zB3pfPsGCSbyFT42nmNiunSzEnLBFraCBAWyHGrGXgPAKzyaAx9xewqMidid5LG7QlVBcf6De3SkIN8+0Z6fq3nBDAFSYLjdTXp07\\\\\\\/HJw3V8HSaMPGeNHjObR2jgbc59KeZk\\\\\\\/QI2Pv4nUjjlDdCrNN6sdL2BDnmn+JH9XS\\\\\\\/Y8mnmutyGN0pq6tYwnjHyXAsUwBmoT4nl9uASP7LZ\\\\\\\/pN4vWfa05wXozQog\\\\\\\/4O2Brl6cu2VsDNtR0Uk7zD2VyDdelvaaNS0ebvV1CW8ZCHhXInyfVs6F5JBL3x0E\\\\\\\/pkTqYIylqRPD4TMiqksveiCW97LP0f5Aqc6oJ7Z416zWcgex7vIoOLyAPZfCnsHtEk3sHoU+XBwbIu491m0zLRvwHUetIzh4xOpOonjeVSg5v7AX4cvX2OoP8SnOKRn58bqo3J62tKxsc+zsj\\\\\\\/1uxLsRIO5lA7KtvOWVkHFqq0G80PXtRTF0oiozvVTF0nrs4ZwhZF\\\\\\\/E8uqkFkxns7ivUf1nqGmRuSpfnJs8VMpwvWPoZs+auvSHcDRrtWxClWtCglMkG4rVT032vpxGoJNiD+IpADWz+xoL8I1zW3pLXKksQlZj4TS5VKehzKbpXY4a6+kbb\\\\\\\/KewHALoLl7TOsJQUw7nM7uImNUqdobgE1FeOL91y706cgLz9+xOZXn9ZYKeMVmbAehoZZQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BOE1ePpDVc\\\\\\\/Aftobnmg0GA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c1a2487a9d2c18e659cdd81b35f9b4ee7e1566694394bf7d68996543562f77e2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TOuW9UPUtMVHnLVukVrOnbmYgweszRcRtTpOWVhpPxwByUni9M2H\\\\\\\/G\\\\\\\/vPMcMDGqaMiFjhIfdA0hekUOarX4QJckd9UeGdw4PYNk+aB2vjIZYyo\\\\\\\/L1MF7IYE79HSN0xCrD1srgNzbi1feVxAGHAcOTNXS1Uv57futLRMnwulwaiY\\\\\\\/nKz2suKHffcNBjlKGKQzyFjQnUraOyiIFApykRFWEJVyWib7jk5RmSKDIrKF3DSlsP0G6C60ZMT1sn1Cx0gq41uvfygfu4EuQMfPklqBrrb40GwlNZPQRBUj2\\\\\\\/sPn6YecMKNr+UOlXKp\\\\\\\/7va0TAx+7ekIoi7LoWwUoZ1vx5bYG5UuFUcEz+M0pkyj8EVn5cBnPSM48b1omINOsJ36vL1+BsFEsrl9\\\\\\\/ljoWsjgaNJgP7cC7KHvdcNvmXpfezrmz4pwEHfvkfuMJE4BSIs3sbgK\\\\\\\/0IAEr4qgVdNIX7+dmvqsstorVuMGBxJqLdNYzLArR0\\\\\\\/jtGl+qLBFH+rNi3MKPCk9RGG6wElkWI3ALHlCCymdo5UBkKyQVw43U3HB0kNxn+y8kx64pFZ8Sasw9tchMjc3o2tDpYU0qltxT+gXYeWdfeXmPGqBdLnpe0tELnjMTx5nYJDTm0o7+e+5CAylOQOB\\\\\\\/nx2JESx7+NUU\\\\\\\/w9PRb+W5UTyh4t5lcni1zNn0jaUt5guLLz7O\\\\\\\/jeTjvBXj4vyFVSDzXcGVDH7BghQ2Bj29Usqzy6LbKs2Qh1YrqdwYwD05xoxwReKmEvtZoiDQ9VOMEvHBT633+\\\\\\\/KbBp99cpsB0fBhJkY6sE7uNfBJnzs443z2RX8bU\\\\\\\/68p09OxyLeHAjRXn80jv1hhevDUT74p6n5aN\\\\\\\/jcjtTWj+jv3n3lolmnDItrTFWGwPnqlTDnr7pEj+zjZb1SELEIRzDxpMSPcOSA\\\\\\\/kKwZdvoomIU9lEiFPHlKYZXLiPbJZ5c2iQbjtfJ0w2fxTvjs5nseru7t29JdZ+Wc4IZ2ReV6uuV+Byl\\\\\\\/Zem5tFx\\\\\\\/\\\\\\\/aaueKgAp6+SDfLhAywRsLMe6qbcYJRZafLBcsYYYffBNw\\\\\\\/sBiyxbsd4dNXAe2i+EXHBeO3yJ7FXNW3P8aN3R7Kk2PDxjmKJ3QkAAsgRk3WfX4cOnOvrBALMREBGKFLxGDAWbCzKH2dT0\\\\\\\/f8My\\\\\\\/Ef8k7cC6Wlweb3rna7npZzxipVQ0njvu9RdCtlKEQsurxl5RiftIK3Z4XBHMM6Y6sRvcXbAFSsrjhjBMOceNOilPdgXy3WDNxb5AB0E9NI+cNfBA9OI4BWCErsz+wCwrURU8y9B2ZEG8a4iU\\\\\\\/2LCRaoDT4V+l8I7TQoAw7oe\\\\\\\/tl58SQBtBYi5vH7+6HoApV2RwqXHXefyGf7FyeAMlYglLUPaK7If6\\\\\\\/j0g5V1C2B9h+ppSdeZMuEaI0amrfy8XDQqs380EcbWOEE\\\\\\\/H0utvQHJyCSBgaXa61Hdc\\\\\\\/EuVRI1hAFzX4pRuLi5Wf+nbiRbWIlTf5A+\\\\\\\/xvF8m6POigB+q\\\\\\\/DHovnEAHtwjohIbVRm70bKByxgkTIx6pZMFaXQjtmetF\\\\\\\/r6kyDoeko7qTPTatJQh1ab4iM4rUgr8ktnDOayE7FM1yeuujbkTBmI5+fWSbpeDNHCSiZkwFCmFN5nmsDfptLFnWzfQtCpHONe1ek+eq1rgk\\\\\\\/Wk+4WOJbvQA2v+gdVM0zmFyYoK\\\\\\\/AckJU4o\\\\\\\/oF3io9wcGq+T0EYhcUqwX7WTjrh+eprLDi2CXCMLKo+P+MDlmRqdYkiD0\\\\\\\/pVJH0xGc+zJFE0fOsDS0C9oH5MZ3AC2XtW3QRR1PmnDk\\\\\\\/mPm\\\\\\\/ErUcAMbzmYfV5ji56f0CjeQiIFerJnFLiPWyA00qDxI6xDospX5PSotkgfjf3knXpXhmglLmwMJQFLfu27xsDEaI8B6sMnQdpvXKCtGUoXJYt4VHqpX9F\\\\\\\/mbroJ5J6ILK\\\\\\\/EkZXjIQYHI0gHMJngApvmP8W3+dx3e86mgOPk2Q8\\\\\\\/gsK\\\\\\\/loKqdZP7sGK7WbIuWpNzubY4xJrj9RSeGB0Qa8Cbw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nrqPIzf5+iWNZJtMTUEsKA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224623a3aa9e7d22a63899ba80d005d3901a1369f2a8d40c6a0545a219b7e3ca9a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495268 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/225" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-reNOlHARpQKKPUZI_dKLp310mASyEECImlIF2MJRaQo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/225" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3a8QqhHNWZFN8DjkM2gMLs636WXyXOIhNPb9LCeiQdc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/211" title="Inference of Historical Population-Size Changes with Allele-Frequency Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/235" title="Transcription Profiles of Age-at-Maturity-Associated Genes Suggest Cell Fate Commitment Regulation as a Key Factor in the Atlantic Salmon Maturation Process" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495268" id="node495268" data-pisa="ggg;10/1/225" data-pisa-master="ggg;g3.119.400650" data-apath="/ggg/10/1/225.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Single, Shared Triploidy in Three Species of Parasitic Nematodes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ashley Schoonmaker</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5065-8906" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yue Hao</span>, <span class="highwire-citation-author" data-delta="2">David McK. Bird</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8677-4933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gavin C. Conant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">225-233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400650 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495268" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ashley Schoonmaker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashley%2BSchoonmaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoonmaker%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshley%2BSchoonmaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yue Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYue%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5065-8906" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yue Hao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David McK. Bird </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BMcK.%2BBird%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bird%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BMcK.%2BBird%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gavin C. Conant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Program in Genetics, North Carolina State University, Raleigh, NC, 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gavin%2BC.%2BConant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conant%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGavin%2BC.%2BConant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8677-4933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gavin C. Conant</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gconant@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">gconant@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/225" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495268" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/225.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495268" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/225.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495268" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/225.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495268" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The root-knot nematodes of the genus <em>Meloidogyne</em> are important and damaging parasites capable of infecting most flowering plants. Within this genus, several species of the <em>Meloidogyne incognita</em> group show evidence of paleopolyploidy in their genomes. We used our software tool POInT, the Polyploidy Orthology Inference Tool, to phylogenetically model the gene losses that followed that polyploidy. These models, and simulations based on them, show that three of these species (<em>M. incognita</em>, <em>M. arenaria</em> and <em>M. javanica</em>) descend from a single common hybridization event that yielded triplicated genomes with three distinguishable subgenomes. While one of the three subgenomes shows elevated gene loss rates relative to the other two, this subgenome does not show elevated sequence divergence. In all three species, ancestral loci where two of the three gene copies have been lost are less likely to have orthologs in <em>Caenorhabditis elegans</em> that are lethal when knocked down than are ancestral loci with surviving duplicate copies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">polyploidy</a></li><li class="kwd"><a href="/keyword/evolutionary-model" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-model" rel="nofollow">evolutionary model</a></li><li class="kwd"><a href="/keyword/root-knot-nematode" class="hw-term hw-article-keyword hw-article-keyword-root-knot-nematode" rel="nofollow">root-knot nematode</a></li></ul><p id="p-3">Root-knot nematodes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) are a group of destructive parasites that attack plant roots and infest a wide variety of crops and other angiosperm lineages across the globe (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Trudgill and Blok 2001</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Moens <em>et al.</em> 2009</a>). Their lifecycle involves the invasion of plant roots, which generally results in the formation of the gall from which the name “root-knot” is derived (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bird 1974</a>). The invasion is followed by the nematodes inducing the plant to form special multinucleated giant cells that nourish the next generation of parasites (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Trudgill and Blok 2001</a>). In addition to the direct effects on root performance that may result from infestation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wesemael <em>et al.</em> 2011</a>), the nematodes impose indirect costs on the plant by redirecting a portion of its photosynthetic output to feed the growing nematodes, a cost that can be on the order of 15% of the total energy budget in grape vines (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Melakeberhan and Ferris 1989</a>). As a result, the world economic impact of <em>Meloidogyne</em> infection is significant, on the order of tens of billions of U.S. dollars annually (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Koenning <em>et al.</em> 1999</a>). Among the most diverse and damaging of these organisms are the tropical nematodes of the <em>Meloidogyne incognita</em> group (MIG), which includes <em>Meloidogyne incognita</em>, <em>Meloidogyne arenaria</em> and <em>Meloidogyne javanica</em> (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Trudgill and Blok 2001</a>). These nematodes’ adaptability is illustrated by their rapid invasions of numerous crop species in the few thousand years since the origins of agriculture (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lunt <em>et al.</em> 2014</a>). As suggested by the cartoon phylogeny in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, the relationships of these three MIG species are still contested, with some published studies proposing that <em>M. arenaria</em> and <em>M. javanica</em> are sister to each other (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tigano <em>et al.</em> 2005</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Szitenberg <em>et al.</em> 2017</a>) and others arguing for <em>M. incognita</em> and <em>M. arenaria</em> as sisters (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brito <em>et al.</em> 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Álvarez-Ortega <em>et al.</em> 2019</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/225/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A single, shared triplication inferred across three Meloidogyne genomes. A) A schematic phylogeny of the root-knot nematodes showing the placement of the hybridization event shared by the MIG group. The tree topology was adapted from published phylogenetic inferences (Tigano et al. 2005; Brito et al. 2015; Szitenberg et al. 2017; &#xC1;lvarez-Ortega et al. 2019). B) Of the three possible rooted topologies for the three MIG taxa, we show the one that gives the maximum likelihood of observing the set of gene losses found in these genomes (Methods). Branch lengths are reported as the product of the &#x3B1; parameter in Figure 2 and time (&#x3B1;t), while above each branch is POInT&#x2019;s estimate of the number of gene copies from the triplicated loci lost along that branch. In the table at right are the net number of triplicated, duplicated and single-copy loci in each of three genomes. C) Distribution (y-axis) of simulated internal branch lengths (x-axis) when the underlying genomes share no common ancestry (see Methods). The actual values from the three genomes correspond to the branch lengths in B and are indicated with arrows for reference. Note that in each simulation, we inferred the maximum likelihood topology for that simulation and use the internal branch from that topology, which may not be identical to the topology in B. D) As for C, but for the root branch." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-733759135" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A single, shared triplication inferred across three &lt;em&gt;Meloidogyne&lt;/em&gt; genomes. A) A schematic phylogeny of the root-knot nematodes showing the placement of the hybridization event shared by the MIG group. The tree topology was adapted from published phylogenetic inferences (Tigano &lt;em&gt;et al.&lt;/em&gt; 2005; Brito &lt;em&gt;et al.&lt;/em&gt; 2015; Szitenberg &lt;em&gt;et al.&lt;/em&gt; 2017; &#xC1;lvarez-Ortega &lt;em&gt;et al.&lt;/em&gt; 2019). B) Of the three possible rooted topologies for the three MIG taxa, we show the one that gives the maximum likelihood of observing the set of gene losses found in these genomes (Methods). Branch lengths are reported as the product of the &#x3B1; parameter in Figure 2 and time (&#x3B1;t), while above each branch is POInT&#x2019;s estimate of the number of gene copies from the triplicated loci lost along that branch. In the table at right are the net number of triplicated, duplicated and single-copy loci in each of three genomes. C) Distribution (&lt;em&gt;y&lt;/em&gt;-axis) of simulated internal branch lengths (&lt;em&gt;x&lt;/em&gt;-axis) when the underlying genomes share no common ancestry (see Methods). The actual values from the three genomes correspond to the branch lengths in B and are indicated with arrows for reference. Note that in each simulation, we inferred the maximum likelihood topology for that simulation and use the internal branch from that topology, which may not be identical to the topology in B. D) As for C, but for the root branch.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/225/F1.medium.gif" width="311" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/225/F1.medium.gif" width="311" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/225/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/225/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495293" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-4" class="first-child">A single, shared triplication inferred across three <em>Meloidogyne</em> genomes. A) A schematic phylogeny of the root-knot nematodes showing the placement of the hybridization event shared by the MIG group. The tree topology was adapted from published phylogenetic inferences (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Tigano <em>et al.</em> 2005</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brito <em>et al.</em> 2015</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Szitenberg <em>et al.</em> 2017</a>; Álvarez-Ortega <em>et al.</em> 2019). B) Of the three possible rooted topologies for the three MIG taxa, we show the one that gives the maximum likelihood of observing the set of gene losses found in these genomes (Methods). Branch lengths are reported as the product of the α parameter in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and time (αt), while above each branch is POInT’s estimate of the number of gene copies from the triplicated loci lost along that branch. In the table at right are the net number of triplicated, duplicated and single-copy loci in each of three genomes. C) Distribution (<em>y</em>-axis) of simulated internal branch lengths (<em>x</em>-axis) when the underlying genomes share no common ancestry (see Methods). The actual values from the three genomes correspond to the branch lengths in B and are indicated with arrows for reference. Note that in each simulation, we inferred the maximum likelihood topology for that simulation and use the internal branch from that topology, which may not be identical to the topology in B. D) As for C, but for the root branch.</p><div class="sb-div caption-clear"></div></div></div><p id="p-5">The genomes of the MIG species contain diverged duplicated gene copies (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Szitenberg <em>et al.</em> 2017</a>) not present in the diploid genome of the related northern root-knot nematode, <em>Meloidogyne hapla</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Opperman <em>et al.</em> 2008</a>). Although these MIG species diverged from <em>M. hapla</em> about 43 MYA, the radiation within the group is likely much more recent, on the order of 5 MYA (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Giorgi <em>et al.</em> 2002</a>). It now appears that the origins of these MIG species and their radiation was complex: <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lunt <em>et al.</em> (2014)</a> have proposed that <em>M. incognita</em>, <em>M. arenaria</em> and <em>M. javanica</em> all descend from hypotriplicated hybridization. This hybridization event involved the addition of another copy of the genome to an existing diploid genome, with the subsequent loss of some of the triplicated loci, hence <em>hypo</em>triplication (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blanc-Mathieu <em>et al.</em> 2017</a>). It was originally suggested that one of the hybrid parents was closely related to the extant <em>M. floridensis</em> (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Lunt <em>et al.</em> 2014</a>), but analyses of new MIG genomes has led to the suggestion that <em>M. floridensis</em> is instead a species within the hypotriplicated group (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Szitenberg <em>et al.</em> 2017</a>). The MIG species’ triploid state has important implications for their biology. In particular it is probably the reason that all are believed to reproduce exclusively by asexual parthenogenesis (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Triantaphyllou 1985</a>): indeed polyploidy, hybridization and asexuality appear to co-occur reasonably often, although the causality of this association is imperfectly understood (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Neiman <em>et al.</em> 2014</a>).</p><p id="p-6">Thus, these MIG genomes can be thought of as originating from a paleopolyploidy event somewhat akin to the paleohexaploidy found in the genus Brassica. In this group of plants, two ancient genomes merged in the first round of hybridization, and a third genome later contributed to form the hexaploid (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tang <em>et al.</em> 2012</a>). However, in the nematodes, the process involved one haploid and one diploid gamete, not the merging of two diploid genomes (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Lunt <em>et al.</em> 2014</a>). It may also be the case that, of the three subgenomes present in these taxa’s genomes, one is more diverged relative to the other two: we will refer to this posited third subgenome as the “diverged subgenome.”</p><p id="p-7">In light of the uncertainties surrounding the evolution of these species, we analyzed three <em>Meloidogyne</em> genomes (those of <em>M. incognita</em>, <em>M. arenaria</em> and <em>M. javanica</em>) with POInT, the Polyploidy Orthology Inference Tool (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Conant and Wolfe 2008</a>). This tool models the resolution of a polyploidy event along a phylogenetic tree, using gene loss events and shared gene order information (synteny) to infer the combination of phylogenetic relationships, duplicated or triplicated gene loss rates and orthology relationships among the surviving genes that best explains the extant genome structures. As such, POInT is uniquely able to use the patterns of gene loss after the nematode hybridization to provide statistically rigorous answers to four separate questions. First, is there clear evidence for the presence of three distinct subgenomes in these taxa’s genomes? Second, are the three species descended from a single, shared hybridization? Third, are the three inferred subgenomes distinct in their post-hybridization evolution, and can we identify a diverged subgenome? And fourth and finally, does an ancestral gene’s propensity to survive in multiple copies after the hybridization depend on its functional role?</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Identifying the relics of an ancestral polyploidy from conserved synteny blocks</h3><p id="p-8">We previously created a pipeline for inferring blocks of conserved synteny from a set of polyploid genomes and an outgroup lacking the polyploidy (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Emery <em>et al.</em> 2018</a>). To understand the polyploid history of the MIG, we applied this pipeline to three nematode genomes, those of <em>Meloidogyne arenaria</em>, <em>Meloidogyne incognita</em> and <em>Meloidogyne javanica</em> (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Blanc-Mathieu <em>et al.</em> 2017</a>). The genome of <em>Meloidogyne hapla</em>, released through Wormbase, was used as the outgroup (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stein <em>et al.</em> 2001</a>). This pipeline involves three steps. The first is a homology search of each polyploid genome against the <em>M. hapla</em> genome, which was performed with GenomeHistory (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Conant and Wagner 2002</a>). Homologous genes between each of the three pairs of genomes were retained if they shared 70% amino acid identity, had a BLAST E-value (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) of 10<sup>−9</sup> or smaller, and had a nonsynonymous divergence (K<sub>a</sub>) less than 0.4.</p><p id="p-9">We used these homologs as inputs to our second step: the search for conserved regions of synteny from the polyploidy. This step places homologous genes in each polyploid genome into blocks of <em>double</em> or <em>triple</em> conserved synteny (DCS or TCS, respectively): because one of the questions of this analysis was whether this polyploidy was a genome duplication or triplication, we made separate inferences assuming each possibility. This analysis step is based on the concept of a “pillar:” one gene in the <em>M. hapla</em> genome and its 1 to 3 homologs in one of the polyploid genomes (<em>e.g.</em>, surviving or lost duplicate copies from the polyploidy). Using simulated annealing (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kirkpatrick <em>et al.</em> 1983</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Conant and Wolfe 2006</a>), we sought a combination of homolog assignments to pillars and a relative order of the pillars themselves that maximized the number of cases where the genes in neighboring pillars were also neighbors in their respective genomes (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Emery <em>et al.</em> 2018</a>). Finally, in the third step, we merged these inferences across the three polyploid genomes to give a set of pillars with at least one surviving gene in each pillar from each polyploid genome. We then again used simulated annealing to search for a globally optimal pillar order in terms of the number of neighboring pillars that are in synteny in their respective genomes (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Emery <em>et al.</em> 2018</a>). Under the assumption of a whole genome duplication (WGD), the result was a set of 5,628 pillars with at least one surviving gene from each polyploid genome: the corresponding figure under the assumption of a whole genome triplication (WGT) was 5,316 pillars.</p></div><div id="sec-3" class="subsection"><h3>Modeling evolution after WGD or WGT</h3><p id="p-10">Using POInT (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Conant and Wolfe 2008</a>), we fit models of post-WGD and post-WGT evolution to these data (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). POInT’s computation works in two parts. First, at each pillar, the likelihood of all <em>2<sup>n</sup></em> (for a WGD) or <em>6<sup>n</sup></em> (for a WGT) possible orthology relationships is computed, given the observed gene presence and absence data and one of the evolutionary models shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. In the second step, POInT conditions these <em>2<sup>n</sup>/6<sup>n</sup></em> likelihoods on the corresponding likelihoods for all loci to the left and to the right of the current locus in the ancestral gene order, giving the likelihood of the full set of pillars. The parameters of the model and the tree branch lengths are optimized using standard numerical techniques (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Press <em>et al.</em> 1992</a>). The resulting optimized likelihood can then be compared to that of other proposed phylogenetic topologies, which, because the models used in POInT are not time reversible (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), are rooted (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). As far as we are aware, there is no other fully integrated statistical modeling tool for studying polyploid genomes that would enable the types of analyses we describe below.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/225/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Models of duplicate gene loss after genome duplications or triplications support a biased pattern of gene loss after a genome triplication in members of the MIG group. A) Structure of our null model of evolution after a genome duplication (WGD) and model state definitions. This model was used for the simulation of WGD events for the test of a difference between a WGD and WGT for the three genomes. Duplicated loci from the polyploidy (D) can remain duplicated or lose the copy from subgenome 2 (S1) or subgenome 1 (S2), losses that occur at instantaneous rate &#x3B1;. B) Nested models of genome triplication (WGT). All loci (pillars) start as triplicated genes (T), which may then transition to one of three duplicated states after the first gene loss (D1,2, D1,3 or D2,3). Some pillars may experience a further loss from one of the duplicated states, resulting in a single-copy gene at that pillar (S1, S2, or S3), a process that occurs at a rate &#x3B2; relative to the rate of gene losses at triplicated loci. To better understand post-triploidy evolution, we added parameter f1 to the model, allowing the rate of survival of the S1 copy into the duplicated state to differ from the survival rates for genes from subgenomes 2 and 3 (states S2 and S3, models WGT Null vs. WGT 1Frac). We next allowed for differential loss rates from the duplicated states to S1 verses S2 and S3 (parameters &#x3B2;d and &#x3B2;f, model WGT 1Frac vs. WGT 1FracS). Finally, we allowed for differential loss rates to state S2 and to state S3 (parameter l3, models WGT 1FracS vs. WGT 1FracS2). In all three cases, the fit of the model to the observed data, given by the natural log of the likelihood of the genomic data given the tree and model (lnL), improves (see Results)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-733759135" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Models of duplicate gene loss after genome duplications or triplications support a biased pattern of gene loss after a genome triplication in members of the MIG group. A) Structure of our null model of evolution after a genome duplication (WGD) and model state definitions. This model was used for the simulation of WGD events for the test of a difference between a WGD and WGT for the three genomes. Duplicated loci from the polyploidy (&lt;em&gt;D&lt;/em&gt;) can remain duplicated or lose the copy from subgenome 2 (&lt;em&gt;S&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt;) or subgenome 1 (&lt;em&gt;S&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;), losses that occur at instantaneous rate &#x3B1;. B) Nested models of genome triplication (WGT). All loci (pillars) start as triplicated genes (&lt;em&gt;T&lt;/em&gt;), which may then transition to one of three duplicated states after the first gene loss (&lt;em&gt;D&lt;sub&gt;1,2&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;,&lt;/sub&gt; &lt;em&gt;D&lt;sub&gt;1,3&lt;/sub&gt;&lt;/em&gt; or &lt;em&gt;D&lt;sub&gt;2,3&lt;/sub&gt;&lt;/em&gt;). Some pillars may experience a further loss from one of the duplicated states, resulting in a single-copy gene at that pillar (&lt;em&gt;S&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt;, &lt;em&gt;S&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;, or &lt;em&gt;S&lt;sub&gt;3&lt;/sub&gt;&lt;/em&gt;), a process that occurs at a rate &#x3B2; relative to the rate of gene losses at triplicated loci. To better understand post-triploidy evolution, we added parameter &lt;em&gt;f&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; to the model, allowing the rate of survival of the &lt;em&gt;S&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; copy into the duplicated state to differ from the survival rates for genes from subgenomes 2 and 3 (states &lt;em&gt;S&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt; and &lt;em&gt;S&lt;sub&gt;3&lt;/sub&gt;&lt;/em&gt;, models &lt;em&gt;WGT Null&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;WGT 1Frac&lt;/em&gt;). We next allowed for differential loss rates from the duplicated states to &lt;em&gt;S&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; verses &lt;em&gt;S&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt; and &lt;em&gt;S&lt;sub&gt;3&lt;/sub&gt;&lt;/em&gt; (parameters &#x3B2;&lt;sub&gt;d&lt;/sub&gt; and &#x3B2;&lt;sub&gt;f&lt;/sub&gt;, model &lt;em&gt;WGT 1Frac&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;WGT 1FracS&lt;/em&gt;). Finally, we allowed for differential loss rates to state &lt;em&gt;S&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt; and to state &lt;em&gt;S&lt;sub&gt;3&lt;/sub&gt;&lt;/em&gt; (parameter &lt;em&gt;l&lt;sub&gt;3&lt;/sub&gt;&lt;/em&gt;, models &lt;em&gt;WGT 1FracS&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;WGT 1FracS&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;). In all three cases, the fit of the model to the observed data, given by the natural log of the likelihood of the genomic data given the tree and model (lnL), improves (see &lt;em&gt;Results&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/225/F2.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/225/F2.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/225/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/225/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495301" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">Models of duplicate gene loss after genome duplications or triplications support a biased pattern of gene loss after a genome triplication in members of the MIG group. A) Structure of our null model of evolution after a genome duplication (WGD) and model state definitions. This model was used for the simulation of WGD events for the test of a difference between a WGD and WGT for the three genomes. Duplicated loci from the polyploidy (<em>D</em>) can remain duplicated or lose the copy from subgenome 2 (<em>S<sub>1</sub></em>) or subgenome 1 (<em>S<sub>2</sub></em>), losses that occur at instantaneous rate α. B) Nested models of genome triplication (WGT). All loci (pillars) start as triplicated genes (<em>T</em>), which may then transition to one of three duplicated states after the first gene loss (<em>D<sub>1,2</sub></em><sub>,</sub> <em>D<sub>1,3</sub></em> or <em>D<sub>2,3</sub></em>). Some pillars may experience a further loss from one of the duplicated states, resulting in a single-copy gene at that pillar (<em>S<sub>1</sub></em>, <em>S<sub>2</sub></em>, or <em>S<sub>3</sub></em>), a process that occurs at a rate β relative to the rate of gene losses at triplicated loci. To better understand post-triploidy evolution, we added parameter <em>f<sub>1</sub></em> to the model, allowing the rate of survival of the <em>S<sub>1</sub></em> copy into the duplicated state to differ from the survival rates for genes from subgenomes 2 and 3 (states <em>S<sub>2</sub></em> and <em>S<sub>3</sub></em>, models <em>WGT Null</em> <em>vs.</em> <em>WGT 1Frac</em>). We next allowed for differential loss rates from the duplicated states to <em>S<sub>1</sub></em> verses <em>S<sub>2</sub></em> and <em>S<sub>3</sub></em> (parameters β<sub>d</sub> and β<sub>f</sub>, model <em>WGT 1Frac</em> <em>vs.</em> <em>WGT 1FracS</em>). Finally, we allowed for differential loss rates to state <em>S<sub>2</sub></em> and to state <em>S<sub>3</sub></em> (parameter <em>l<sub>3</sub></em>, models <em>WGT 1FracS</em> <em>vs.</em> <em>WGT 1FracS<sub>2</sub></em>). In all three cases, the fit of the model to the observed data, given by the natural log of the likelihood of the genomic data given the tree and model (lnL), improves (see <em>Results</em>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Comparing the fit of a WGD and a WGT to the MIG genomes</h3><p id="p-12">To test for the presence of a WGT in the MIG, we adopted a simulation approach that sought to assess if the degree of TCS that we inferred in the MIG genomes might be artifactual. To do so, we explored the patterns of TCS that would be observed when the underlying event was known to be a genome <em>duplication</em>. We started with our inferred set of optimal DCS blocks from the three MIG genomes. Using those blocks, we simulated 100 WGD events, using the phylogeny of <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>, the WGD null model (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>) and the maximum likelihood estimates of this model’s parameters. The result of each simulation was a set of single-copy and duplicated loci that follow the original DCS blocks but show different patterns of duplicate loss and retention within them. We then merged the new simulated duplicate genes with all of the remaining homologous genes from each of the three MIG genomes that were not placed into DCS blocks. Thus, each simulation step created three new simulated MIG genomes that mixed simulated single-copy and duplicate genes with these remaining unmapped genes. We then applied our pipeline for TCS block inference to each of these simulated genomes. As a test statistic, we compared the proportion of the total loci from each simulation that were triplicated across all three simulated genomes to this same value from the actual TCS blocks (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <em>Results</em>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/225/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The proportion of loci with three surviving copies of an ancestral gene in all three MIG genomes is greater than expected under a model of genome duplication. On the x-axis is the proportion of all loci in the 100 simulated datasets where all three genomes have surviving genes from each of the three subgenomes: on the y-axis is proportion of simulations with that value of x. The arrow at the right gives this same proportion for the actual MIG genomes: the real genomes possess more shared triplicated genes than can be explained by a genome duplication and chance associations of other homologous genes (see Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-733759135" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The proportion of loci with three surviving copies of an ancestral gene in all three MIG genomes is greater than expected under a model of genome duplication. On the &lt;em&gt;x&lt;/em&gt;-axis is the proportion of all loci in the 100 simulated datasets where all three genomes have surviving genes from each of the three subgenomes: on the &lt;em&gt;y&lt;/em&gt;-axis is proportion of simulations with that value of &lt;em&gt;x&lt;/em&gt;. The arrow at the right gives this same proportion for the actual MIG genomes: the real genomes possess more shared triplicated genes than can be explained by a genome duplication and chance associations of other homologous genes (see &lt;em&gt;Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/225/F3.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/225/F3.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/225/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/225/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495402" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-13" class="first-child">The proportion of loci with three surviving copies of an ancestral gene in all three MIG genomes is greater than expected under a model of genome duplication. On the <em>x</em>-axis is the proportion of all loci in the 100 simulated datasets where all three genomes have surviving genes from each of the three subgenomes: on the <em>y</em>-axis is proportion of simulations with that value of <em>x</em>. The arrow at the right gives this same proportion for the actual MIG genomes: the real genomes possess more shared triplicated genes than can be explained by a genome duplication and chance associations of other homologous genes (see <em>Methods</em>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Simulation of three independent triploidies</h3><p id="p-14">To model the possibility that these three MIG genomes are the products of independent triploidies, we simulated post-polyploidy evolution along a star topology. To do so, we fit the observed pillar data from the three actual MIG genomes to a star topology using POInT (<em>e.g.</em>, no shared ancestry between the three genomes). As we have described previously (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Conant and Wolfe 2008</a>), we then used POInT to simulate 100 sets of three genomes under this star tree and estimated model parameters. For each of these 100 simulated datasets, we again used POInT to infer the maximum likelihood topology (of the three possible) and extracted the lengths of the root and shared branches. We compared the distribution of the 100 estimates to the length of these two branches seen in the actual dataset, where we had again inferred the maximum likelihood topology (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>).</p></div><div id="sec-6" class="subsection"><h3>Sequence divergence between parental subgenomes</h3><p id="p-15">Using the <em>WGT 1FracS<sub>2</sub></em> model (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>), we identified surviving triplicated genes from each of the three species where POInT was able to infer the member of the triplet deriving from the derived subgenome with ≥95% confidence. This requirement for high-confidence assignments of genes to the derived subgenome is essential for our analysis but does limit the number of potential triplets for analysis in each MIG genome (see below). For each of the three species, the protein sequences that these triplets of genes code for were aligned with T-Coffee (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Notredame <em>et al.</em> 2000</a>) and codon-preserving nucleotide alignments inferred. We computed branch-wise estimates of K<sub>s</sub> and K<sub>a</sub> for each triplet with our previously described tool (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Conant and Wagner 2003</a>). After removing two outlier triplets with abnormal K<sub>s</sub> values (K<sub>s</sub> &gt;&gt; 20), 58 <em>M. arenaria</em> triplets, 86 <em>M. incognita</em> triplets and 34 <em>M. javanica</em> triplets were used for further analysis.</p><p id="p-16">Distributions of K<sub>s</sub> and K<sub>a</sub> values for genes from different parental subgenomes were visualized using the ggplot2 package (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wickham 2016</a>) in R version 3.4.1. Mann–Whitney <em>U</em>-tests (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mann and Whitney 1947</a>) were performed to detect differences between the median of K<sub>s</sub> for genes from the most fractionated subgenome and that of genes from the two less fractionated subgenomes.</p></div><div id="sec-7" class="subsection"><h3>Identifying M. hapla orthologs of essential and non-essential genes in C. elegans:</h3><p id="p-17">Using our published pipeline for using synteny to identify orthologous genes between pairs of genomes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Conant 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bekaert and Conant 2011</a>), we inferred a set of genes from <em>M. hapla</em> that are orthologous to genes from the model nematode <em>Caenorhabditis elegans</em> (C. Elegans Sequencing Consortium 1998). Briefly, we defined an <em>M. hapla</em> gene to be homologous to a gene from <em>C. elegans</em> if the two shared a BLASTP (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) E-value of 10<sup>−7</sup> or less and were 30% or more identical at the protein level over 80% of the length of the shorter of the two translated coding regions. From this set of homologs, we defined a pair of genes to be orthologs if either <em>a</em>) they were each others’ only homolog in the other genome and had a nonsynonymous divergence (K<sub>a</sub>) less than 0.75 or <em>b</em>) they were syntenic neighbors of another pair of orthologs from the two genomes that met criteria <em>a</em> or <em>b</em> (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Conant 2009</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bekaert and Conant 2011</a>). There were 3,162 <em>M. hapla</em> genes with identified orthologs in <em>C. elegans:</em> of these, 1,988 were among the 5,316 <em>M. halpa</em> genes that anchored the pillars of the WGT analysis (see above). We also obtained a list of RNA interference phenotypes for <em>C. elegans</em> genes from Wormbase (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Howe <em>et al.</em> 2017</a>) and extracted from it 3,340 “essential” genes annotated with phenotypes of “embryonic lethal,” “larval lethal,” “adult lethal,” and “lethal.”</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-18">Supporting data, including Figure S1, are available through figshare <a href="https://doi.org/10.25387/g3.10248929">https://doi.org/10.25387/g3.10248929</a>; the POInT software package is available from GitHub (<a href="http://www.github.com/gconant0/POInT">www.github.com/gconant0/POInT</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>A single, shared, triploidy in the three nematodes</h3><p id="p-19">Using POInT, we simulated independent polyploidies for three MIG genomes and estimated the apparent degree of shared ancestry that POInT would infer in the absence of any true shared post-polyploidy ancestry, as we have done previously (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Conant and Wolfe 2008</a>). Because we estimated the phylogeny of these three taxa in our analysis of the actual data, we also inferred the optimal topology from each of the simulations and extracted our branch lengths from this optimal tree. As shown in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a> and <a id="xref-fig-1-7" class="xref-fig" href="#F1">1D</a>, the degree of apparent common ancestry seen in polyploidies that are known from the simulation parameters to be independent is much smaller (in terms of phylogenetic branch length) than was the case for our actual dataset (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figures 1C</a> and <a id="xref-fig-1-9" class="xref-fig" href="#F1">1D</a>, <em>P &lt;</em> 0.01 in both cases). These results lead us to conclude that these three MIG genomes descend from a single, common hybridization event. Curiously, the maximum likelihood topology inferred with POInT for these three organisms differs from both of the published ones (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Tigano <em>et al.</em> 2005</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Brito <em>et al.</em> 2015</a>; <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Szitenberg <em>et al.</em> 2017</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Álvarez-Ortega <em>et al.</em> 2019</a>): we speculate that it may be difficult to infer phylogenetic relationships among these species with standard gene tree approaches due to the paralogy introduced by the triploidy.</p></div><div id="sec-11" class="subsection"><h3>The three MIG genomes each contain three subgenomes</h3><p id="p-20">Formally distinguishing a genome duplication (WGD) from a genome triplication (WGT) is difficult, because a WGD is a special case of a WGT. While the MIG genomes do not exactly fit the common view of polyploids, we can think of the computational problem as that of showing that a WGT better describes the paralogs in these (pseudo)-haploid genomes than does a WGD. We employed a simulation approach that compared the set of TCS blocks that were inferred from a set of simulated genomes known to have undergone only a WGD event to the actual set of TCS blocks from the MIG genomes (<em>Methods)</em>. We compared the number of triplicated loci in the simulated TCS blocks to the number seen from the actual MIG genomes. Because we simulated a WGD, not a WGT, any triplicated loci seen in the simulations are due to stochastic effects. The real MIG TCS blocks show many more triplicated loci than the simulated ones, rejecting the null hypothesis that the MIG genomes descend from a WGD event (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>; <em>P</em> &lt; 0.01).</p></div><div id="sec-12" class="subsection"><h3>Testing nested models of triplicate loss after WGT</h3><p id="p-21">Given the proposed events that generated these three species, we explored a series of nested models describing the evolution of the triplicated loci in the MIG genomes. The simplest model, <em>WGT Null</em>, was used for the simulations testing for the presence of the shared WGT above. This model makes the assumption that the rate of loss from all three parental subgenomes is equal (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>). We then applied a new <em>WGT 1Frac</em> model that designates one of the parental subgenomes (corresponding to state <em>S<sub>1</sub></em> in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>) to be less likely to generate surviving members in duplicate states (<em>D<sub>1,2</sub></em>, <em>D<sub>1,3</sub></em> and <em>D<sub>2,3</sub></em>) following the loss of a gene from the triplicated state <em>T</em>. The logic of this model is that the hybridization event may have included loci present in the original two subgenomes of the diploid (<em>S<sub>2</sub></em> and <em>S<sub>3</sub></em>), but not in <em>S<sub>1</sub></em>, making states <em>D<sub>1,2</sub></em> and <em>D<sub>1,3</sub></em> disfavored over <em>D<sub>2,3</sub></em>. We then added sequential models that allowed losses from the duplicated states to occur at a different rate to state <em>S<sub>1</sub></em> relative to <em>S<sub>2</sub></em> and <em>S<sub>3</sub></em> (model <em>WGT 1FracS</em>) and then also to differ between states <em>S<sub>2</sub></em> and <em>S<sub>3</sub></em> (model <em>WGT 1FracS<sub>2</sub></em>). If we draw a progression of more complex models, from <em>WGT Null</em> through <em>WGT 1FracS<sub>2</sub></em>, we find that each prior model is a special case of the new model (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2B</a>). As a result, we can use likelihood ratio tests (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sokal and Rohlf 1995</a>) to assess if the more complex models offer a better fit to our data. Each more complex model in fact offers such an improvement over all its parent models (<em>P &lt;</em> 10<sup>−10</sup> in all cases, <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2B</a>). <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> gives a breakdown of POInT’s predictions of the number of genes from each subgenome in each of the three MIG genomes.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495209/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495209/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Distribution of MIG genes across parental subgenomes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>The more fractionated genome does not display elevated sequence divergence relative to the other two subgenomes</h3><p id="p-25"><a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Szitenberg <em>et al</em>., (2017)</a> have presented evidence for a mixture of gene copies in the MIG genomes, generally consisting of two very similar copies of each gene, with the addition of a third more diverged copy, and argue for the presence of the hypotriploidy on this basis. Given that our model suggests the presence of a more fractionated subgenome and two less fractioned subgenomes, it was natural to ask if this more fractionated subgenome also showed evidence for greater sequence divergence. However, such was not the case: we analyzed surviving triplicated loci, computing their synonymous and nonsynonymous divergences. We find that, on average, the more fractionated subgenome shows lower branch-specific divergences in K<sub>s</sub> than do the other two subgenomes (<em>P =</em> 0.049, <em>P =</em> 0.018 and <em>P</em> = 0.003 for <em>M. arenaria</em>, <em>M. incognita</em> and <em>M. javanica</em>, respectively, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) with little significant difference in K<sub>a</sub> between the more fractionated subgenome and the other two for two species (<em>P &gt;</em> 0.05; <em>M. arenaria</em> and <em>M. javanica</em>; Figure S1) and a larger mean K<sub>a</sub> for the more fractionated subgenome for <em>M. incognita</em> (<em>S<sub>1</sub></em>; <em>P</em> = 0.017).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/225/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The more fractionated subgenome does not show elevated substitution rates relative to the other two subgenomes. We show the distributions of Ks for genes from the more fractionated subgenome (S1 in Figure 2) and the combination of two less fractionated subgenomes. Violin and box plots only show Ks values that are smaller than 1, which constitutes more than 90% of the data. Black dots show the mean of all Ks values in each group. P-values are from one-sided Mann&#x2013;Whitney U-tests of differences between the median of Ks values in subgenome 1 and the median of Ks values in subgenome 2&amp;3. M. arenaria P = 0.04924, M. incognita P = 0.01757, M. javanica P = 0.00257." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-733759135" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The more fractionated subgenome does not show elevated substitution rates relative to the other two subgenomes. We show the distributions of K&lt;sub&gt;s&lt;/sub&gt; for genes from the more fractionated subgenome (&lt;em&gt;S&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; in Figure 2) and the combination of two less fractionated subgenomes. Violin and box plots only show K&lt;sub&gt;s&lt;/sub&gt; values that are smaller than 1, which constitutes more than 90% of the data. Black dots show the mean of all K&lt;sub&gt;s&lt;/sub&gt; values in each group. &lt;em&gt;P&lt;/em&gt;-values are from one-sided Mann&#x2013;Whitney &lt;em&gt;U&lt;/em&gt;-tests of differences between the median of K&lt;sub&gt;s&lt;/sub&gt; values in subgenome 1 and the median of K&lt;sub&gt;s&lt;/sub&gt; values in subgenome 2&amp;3. &lt;em&gt;M. arenaria P&lt;/em&gt; = 0.04924, &lt;em&gt;M. incognita P&lt;/em&gt; = 0.01757, &lt;em&gt;M. javanica P&lt;/em&gt; = 0.00257.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/225/F4.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/225/F4.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/225/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/225/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495364" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">The more fractionated subgenome does not show elevated substitution rates relative to the other two subgenomes. We show the distributions of K<sub>s</sub> for genes from the more fractionated subgenome (<em>S<sub>1</sub></em> in <a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2</a>) and the combination of two less fractionated subgenomes. Violin and box plots only show K<sub>s</sub> values that are smaller than 1, which constitutes more than 90% of the data. Black dots show the mean of all K<sub>s</sub> values in each group. <em>P</em>-values are from one-sided Mann–Whitney <em>U</em>-tests of differences between the median of K<sub>s</sub> values in subgenome 1 and the median of K<sub>s</sub> values in subgenome 2&amp;3. <em>M. arenaria P</em> = 0.04924, <em>M. incognita P</em> = 0.01757, <em>M. javanica P</em> = 0.00257.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>MIG genes returned to single-copy are less likely to have essential orthologs in C. elegans</h3><p id="p-27">Using our previously described orthology inference tool (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Conant 2009</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bekaert and Conant 2011</a>) and data from Wormbase (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Howe <em>et al.</em> 2017</a>), we compared the proportion of genes in the three MIG genomes that had only a single gene in that pillar (<em>e.g.</em>, the other two copies had been lost) that had essential orthologs in <em>C. elegans</em> to the same proportion among genes with two or three surviving copies (<em>Methods</em>). In all three MIG genomes, the single-copy genes were less likely to have an essential ortholog (<em>P</em> ≤ 0.005, chi-square test, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). In <em>M. incognita</em>, the surviving triplicated genes were also <em>more</em> likely to have essential orthologs (<em>P</em> = 0.022), but the enrichment of essential orthologs among the triplicated genes in the other two MIG genomes was non-significant (<em>P &gt;</em> 0.05).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495187/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495187/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Single-copy genes in the MIG genomes are less likely to have orthologs with knockdown phenotypes in <em>C. elegans</em></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-33">We confirm two key aspects of the origins of the MIG: namely that they are hybrids with (the remnants of) three subgenomes in each individual and that at least the three species studied descend from a single, common hybridization event. While neither of these findings are likely to be controversial, the modeling framework inherent in POInT allows for testing them in a straightforward and yet rigorous way. Hence, our results tend to further support the hybridization model proposed by <a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">Szitenberg <em>et al</em>., (2017)</a>.</p><p id="p-34">Notably, the three subgenomes found are not interchangeable. As with our previous analyses (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Emery <em>et al.</em> 2018</a>), we find significant evidence that the three subgenomes differ in their patterns of gene losses: in particular, if we look at the cases were only a single gene of the original three triplicates has been lost, it is much more common for that loss to have been from the fractionated subgenome. This pattern is what would be expected from the proposed hypotriploid model (<a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">Szitenberg <em>et al.</em> 2017</a>), where the one subgenome either did not possess the gene in question (and so no triplicate was even formed) or would be more likely to be lost due to conflicts with the other two subgenomes. We also observe some evidence for a difference in survival rate between the other two subgenomes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <em>S<sub>2</sub></em> and <em>S<sub>3</sub></em> in <a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2</a>). However, given the somewhat fragmented genome assemblies used, we are reluctant to over-interpret these results, as they may reflect modeling artifacts resulting from the imperfect synteny relationships.</p><p id="p-35">One surprising outcome of these analyses is that while we identified a fractionated subgenome with strong statistical confidence, that subgenome did not show greater sequence divergence from the other two subgenomes (as compared to their own divergences; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). One could read this finding as contradicting data presented by <a id="xref-ref-41-9" class="xref-bibr" href="#ref-41">Szitenberg <em>et al</em>., (2017)</a>, who suggested the presence of a diverged subgenome based on phylogenetic and sequence identity measures. However, several alternative explanations are possible. Both these and other polyploid genomes show evidence of gene conversion (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Evangelisti and Conant 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mcgrath <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Scienski <em>et al.</em> 2015</a>; <a id="xref-ref-41-10" class="xref-bibr" href="#ref-41">Szitenberg <em>et al.</em> 2017</a>), and it is possible that the diverged gene copies previously detected have moved via gene conversion out of their original syntenic context, making our inferences unable to detect the divergence. A perhaps more likely explanation would be that the more fractionated subgenome is simply not the diverged one: instead there is one distant subgenome that retains many of its genes and two more closely related ones, one of which has lost many genes while the other has not. If the proposal of Lunt <em>et al.</em>, that the two similar gene copies are actually also the products of hybridization (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Lunt <em>et al.</em> 2014</a>), is correct, one could imagine a hybridization of two close relatives, followed by losses in one, followed by the more distant hybridization. This sequence would be consistent with our data and similar to the pattern seen in hexaploid Brassicas (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tang <em>et al.</em> 2012</a>). Unfortunately, the fractionation patterns seen are not sufficiently strong for POInT to be able to distinguish loci from subgenomes 2 and 3 with high statistical confidence (data not shown), making it difficult to further analyze any differences in patterns of sequence evolution in these two subgenomes.</p><p id="p-36">These hybrid nematode genomes show patterns of genome evolution that differ significantly from both diploid and other polyploid genomes. The extensive gene loss seen in all three subgenomes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) is unlike the evolution of diploid genomes. On the other hand, unlike angiosperm polyploids, these three MIG genomes show only a relatively short period of shared evolution by gene loss prior to the speciation event separating <em>M. arenaria</em> from the other two species, with only about 1400 gene losses along the root branch (from &gt;5300 <em>triplicated</em> loci). In contrast, in angiosperm WGDs, &gt;50% of the duplicated loci had returned to single-copy before the first observed speciation (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Emery <em>et al.</em> 2018</a>). On the other hand, while the yeast WGD showed few gene losses before the first speciation event (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Scannell <em>et al.</em> 2007</a>), it was characterized by very even gene loss patterns (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Conant 2014</a>; <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Emery <em>et al.</em> 2018</a>), unlike the very definite bias in losses to one subgenome seen here.</p><p id="p-37">The fact that genes returned to single copy in the MIG genomes tend not to have essential <em>C. elegans</em> orthologs is in keeping with studies of other polyploid organisms. In yeast and plants, genes that are preserved in multiple copies post-polyploidy tend to be those that were more likely to be essential or show higher selective constraint prior to the polyploidy (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hakes <em>et al.</em> 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">DeLuna <em>et al.</em> 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Scannell and Wolfe 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hao <em>et al.</em> 2018</a>). That a similar (and arguably stronger) pattern is seen here suggests two points: first that conceptually thinking of the MIG nematodes as polyploids can be fruitful, and second that the patterns of which genes did and did not experience gene loss after the MIG hybridization is non-random.</p><p id="p-38">Further study of these nematodes will prove fruitful for a number of reasons. As mentioned, they represent a serious threat to agricultural productivity, and the relationship between their unusual genetics and their propensity to attack crops is still uncertain. There is also a significant body of literature exploring the role played by constraints on relative gene <em>dosage</em> in genome evolution (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Edger and Pires 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Birchler and Veitia 2012</a>), with genes that form part of the same functional complex tending to share the same copy number state after polyploidy (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Conant 2014</a>). The fact that the patterns of essentiality seen in the MIG genomes is similar to that from polyploid genomes suggests that similar rules may have governed the loss of genes after this hybridization. However, only an extension of the genome annotations of these organisms can fully validate this hypothesis, because existing data are incomplete. For instance, we investigated whether the previously published Pfam domains found in the <em>M. hapla</em> genome (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Opperman <em>et al.</em> 2008</a>) were predictive of a gene’s propensity to remain in triple-copy after the MIG hybridization, but found no significant effects (data not shown). This null result is likely due to the relatively sparse nature of the functional mapping: no Pfam domain was mapped to more than 94 <em>M. hapla</em> genes. With a more refined functional understanding of these genomes, these organisms could be powerful animal models for exploring questions such as the differences between full ploidy changes and aneuploidy, including in human genetic disease (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Korbel <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Birchler and Veitia 2012</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">This work was supported by National Science Foundation grants NSF-IOS-1339156 (YH and GCC) and NSF-CCF-1421765 (GCC).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-40">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10248929">https://doi.org/10.25387/g3.10248929</a>.</p></li><li class="fn" id="fn-10"><p id="p-41"><em>Communicating editor: M. Rockman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-08-20"><span class="received-label">Received </span>August 20, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-04"><span class="accepted-label">Accepted </span>November 4, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Schoonmaker <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Madden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Schaffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped Blast and Psi-Blast: A new-generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.17.3389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.2" data-doi="10.1038/s41598-019-48195-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Álvarez-Ortega</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Brito</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Subbotin</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Multigene phylogeny of root-knot nematodes and molecular characterization of Meloidogyne nataliei Golden, Rose &amp; Bird, 1981 (Nematoda: Tylenchida).</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">11788</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-019-48195-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D9%26rft.spage%253D11788%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-019-48195-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-019-48195-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.3" data-doi="10.1093/molbev/msq296"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Conant</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Copy number alterations among mammalian enyzmes cluster in the metabolic network.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msq296</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq296%26rft_id%253Dinfo%253Apmid%252F21051442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msq296&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21051442&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287112300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.4" data-doi="10.1073/pnas.1207726109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>,  <span class="cit-name-given-names">J. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Veitia</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene balance hypothesis: connecting issues of dosage sensitivity across biological disciplines.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14746</span>–<span class="cit-lpage">14753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1207726109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1207726109%26rft_id%253Dinfo%253Apmid%252F22908297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM3LzE0NzQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.5" data-doi="10.1146/annurev.py.12.090174.000441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>,  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Plant response to root-knot nematode.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.py.12.090174.000441</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPhytopathol.%26rft.volume%253D12%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.py.12.090174.000441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.py.12.090174.000441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974T927000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.6" data-doi="10.1371/journal.pgen.1006777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc-Mathieu</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Perfus-Barbeoch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Aury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Da Rocha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gouzy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Hybridization and polyploidy enable genomic plasticity without sex in the most devastating plant-parasitic nematodes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e1006777</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006777</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006777%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brito</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Subbotin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Han</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Stanley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dickson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Molecular Characterization of Meloidogyne christiei Golden and Kaplan, 1986 (Nematoda, Meloidogynidae) Topotype Population Infecting Turkey Oak (Quercus laevies) in Florida.</span> <abbr class="cit-jnl-abbrev">J. Nematol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNematol.%26rft.volume%253D47%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.225.8" data-doi="10.1126/science.282.5396.2012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">C. elegans Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genome sequence of the nematode C. elegans: A platform for investigating biology.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">2012</span>–<span class="cit-lpage">2018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.282.5396.2012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BC.%2Belegans%2BSequencing%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D282%26rft.issue%253D5396%26rft.spage%253D2012%26rft.epage%253D2018%26rft.atitle%253DGenome%2BSequence%2Bof%2Bthe%2BNematode%2BC.%2Belegans%253A%2BA%2BPlatform%2Bfor%2BInvestigating%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5396.2012%26rft_id%253Dinfo%253Apmid%252F9851916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5Ni8yMDEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.9" data-doi="10.1016/j.tig.2009.07.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>,  <span class="cit-name-given-names">G. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Neutral evolution on mammalian protein surfaces.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2009.07.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DConant%26rft.auinit1%253DG.%2BC.%26rft.volume%253D25%26rft.issue%253D9%26rft.spage%253D377%26rft.epage%253D381%26rft.atitle%253DNeutral%2Bevolution%2Bon%2Bmammalian%2Bprotein%2Bsurfaces.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.07.004%26rft_id%253Dinfo%253Apmid%252F19716195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.07.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19716195&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270185900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.10" data-doi="10.1093/molbev/msu250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>,  <span class="cit-name-given-names">G. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative genomics as a time machine: How relative gene dosage and metabolic requirements shaped the time-dependent resolution of yeast polyploidy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3184</span>–<span class="cit-lpage">3193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu250%26rft_id%253Dinfo%253Apmid%252F25158798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25158798&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.11" data-doi="10.1093/nar/gkf449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>,  <span class="cit-name-given-names">G. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wagner</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">GenomeHistory: A software tool and its application to fully sequenced genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3378</span>–<span class="cit-lpage">3386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkf449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DConant%26rft.auinit1%253DG.%2BC.%26rft.volume%253D30%26rft.issue%253D15%26rft.spage%253D3378%26rft.epage%253D3386%26rft.atitle%253DGenomeHistory%253A%2Ba%2Bsoftware%2Btool%2Band%2Bits%2Bapplication%2Bto%2Bfully%2Bsequenced%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf449%26rft_id%253Dinfo%253Apmid%252F12140322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkf449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140322&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177325100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.12" data-doi="10.1101/gr.1252603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>,  <span class="cit-name-given-names">G. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wagner</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Asymmetric sequence divergence of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2052</span>–<span class="cit-lpage">2058</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.1252603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DConant%26rft.auinit1%253DG.%2BC.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2052%26rft.epage%253D2058%26rft.atitle%253DAsymmetric%2BSequence%2BDivergence%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1252603%26rft_id%253Dinfo%253Apmid%252F12952876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjA1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.13" data-doi="10.1371/journal.pbio.0040109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>,  <span class="cit-name-given-names">G. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Wolfe</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Functional partitioning of yeast co-expression networks after genome duplication.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0040109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DConant%26rft.auinit1%253DG.%2BC.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De109%26rft.epage%253De109%26rft.atitle%253DFunctional%2Bpartitioning%2Bof%2Byeast%2Bco-expression%2Bnetworks%2Bafter%2Bgenome%2Bduplication.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040109%26rft_id%253Dinfo%253Apmid%252F16555924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16555924&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.14" data-doi="10.1534/genetics.107.074450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>,  <span class="cit-name-given-names">G. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Wolfe</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Probabilistic cross-species inference of orthologous genomic regions created by whole-genome duplication in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1692</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074450</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074450%26rft_id%253Dinfo%253Apmid%252F18562662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE2ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzIyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.15" data-doi="10.1038/ng.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeLuna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Vetsigian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Shoresh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hegreness</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Colon-Gonzalez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Exposing the fitness contribution of duplicated genes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDeluna%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D676%26rft.epage%253D681%26rft.atitle%253DExposing%2Bthe%2Bfitness%2Bcontribution%2Bof%2Bduplicated%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.123%26rft_id%253Dinfo%253Apmid%252F18408719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18408719&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.16" data-doi="10.1007/s10577-009-9055-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>,  <span class="cit-name-given-names">P. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Pires</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene and genome duplications: the impact of dosage-sensitivity on the fate of nuclear genes.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-009-9055-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DEdger%26rft.auinit1%253DP.%2BP.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D699%26rft.epage%253D717%26rft.atitle%253DGene%2Band%2Bgenome%2Bduplications%253A%2Bthe%2Bimpact%2Bof%2Bdosage-sensitivity%2Bon%2Bthe%2Bfate%2Bof%2Bnuclear%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-009-9055-9%26rft_id%253Dinfo%253Apmid%252F19802709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-009-9055-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19802709&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270448800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.17" data-doi="10.1371/journal.pgen.1007267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emery</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M. S.</span> <span class="cit-name-surname">Willis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Barry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Oakgrove</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Preferential retention of genes from one parental genome after polyploidy illustrates the nature and scope of the genomic conflicts induced by hybridization.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1007267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007267</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007267%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.18" data-doi="10.1093/gbe/evq067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evangelisti</span>,  <span class="cit-name-given-names">A. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Conant</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nonrandom survival of gene conversions among yeast ribosomal proteins duplicated through genome doubling.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evq067</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq067%26rft_id%253Dinfo%253Apmid%252F20966100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evq067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20966100&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.19" data-doi="10.1016/S0166-6851(02)00161-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giorgi</span>,  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Veronico</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. D.</span> <span class="cit-name-surname">Luca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Natilla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lanave</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Structural and evolutionary analysis of the ribosomal genes of the parasitic nematode Meloidogyne artiellia suggests its ancient origin.</span> <abbr class="cit-jnl-abbrev">Mol. Biochem. Parasitol.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0166-6851(02)00161-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DDe%2BGiorgi%26rft.auinit1%253DC.%26rft.volume%253D124%26rft.issue%253D1-2%26rft.spage%253D91%26rft.epage%253D94%26rft.atitle%253DStructural%2Band%2Bevolutionary%2Banalysis%2Bof%2Bthe%2Bribosomal%2Bgenes%2Bof%2Bthe%2Bparasitic%2Bnematode%2BMeloidogyne%2Bartiellia%2Bsuggests%2Bits%2Bancient%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0166-6851%252802%252900161-5%26rft_id%253Dinfo%253Apmid%252F12387855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0166-6851(02)00161-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12387855&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.20" data-doi="10.1186/gb-2007-8-10-r209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hakes</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Pinney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Lovell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Oliver</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Robertson</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">All duplicates are not equal: the difference between small-scale and genome duplication.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2007-8-10-r209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHakes%26rft.auinit1%253DL.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253DR209%26rft.epage%253DR209%26rft.atitle%253DAll%2Bduplicates%2Bare%2Bnot%2Bequal%253A%2Bthe%2Bdifference%2Bbetween%2Bsmall-scale%2Band%2Bgenome%2Bduplication.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-10-r209%26rft_id%253Dinfo%253Apmid%252F17916239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-10-r209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17916239&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.21" data-doi="10.1093/gbe/evy061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Washburn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rosenthal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lyons</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Patterns of population variation in two paleopolyploid eudicot lineages suggest that dosage-based selection on homeologs is long-lived.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">999</span>–<span class="cit-lpage">1011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy061</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D999%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevy061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evy061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.22" data-doi="10.1016/j.molbiopara.2016.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>,  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Bolt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shafie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kersey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Berriman</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">WormBase ParaSite - a comprehensive resource for helminth genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Biochem. Parasitol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molbiopara.2016.11.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiochem.%2BParasitol.%26rft.volume%253D215%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molbiopara.2016.11.005%26rft_id%253Dinfo%253Apmid%252F27899279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molbiopara.2016.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899279&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.23" data-doi="10.1126/science.220.4598.671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D. J.</span> <span class="cit-name-surname">Gelatt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Vecchi</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Optimization by simulated annealing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">220</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">680</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.220.4598.671</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKirkpatrick%26rft.auinit1%253DS.%26rft.volume%253D220%26rft.issue%253D4598%26rft.spage%253D671%26rft.epage%253D680%26rft.atitle%253DOptimization%2Bby%2BSimulated%2BAnnealing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.220.4598.671%26rft_id%253Dinfo%253Apmid%252F17813860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyMjAvNDU5OC82NzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzIyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koenning</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Overstreet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Noling</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donald</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Becker</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Survey of crop losses in response to phytoparasitic nematodes in the United States for 1994.</span> <abbr class="cit-jnl-abbrev">J. Nematol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BNematol%26rft.aulast%253DKoenning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D31%26rft.issue%253D4S%26rft.spage%253D587%26rft.epage%253D618%26rft.atitle%253DSurvey%2Bof%2Bcrop%2Blosses%2Bin%2Bresponse%2Bto%2Bphytoparasitic%2Bnematodes%2Bin%2Bthe%2BUnited%2BStates%2Bfor%2B1994.%26rft_id%253Dinfo%253Apmid%252F19270925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19270925&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085322500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.25" data-doi="10.1073/pnas.0813248106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korbel</span>,  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Tirosh-Wagner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Urban</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. N.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kasowski</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of Down syndrome phenotypes revealed by high-resolution analysis of human segmental trisomies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">12031</span>–<span class="cit-lpage">12036</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0813248106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0813248106%26rft_id%253Dinfo%253Apmid%252F19597142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzI5LzEyMDMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.26" data-doi="10.7717/peerj.356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunt</span>,  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Koutsovoulos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Blaxter</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The complex hybrid origins of the root knot nematodes revealed through comparative genomics.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.356</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253De356%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.356%26rft_id%253Dinfo%253Apmid%252F24860695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24860695&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.27" data-doi="10.1214/aoms/1177730491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>,  <span class="cit-name-given-names">H. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Whitney</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">On a test of whether one of two random variables is stochastically larger than the other.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aoms/1177730491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D18%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177730491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177730491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.28" data-doi="10.1101/gr.173740.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>,  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-F.</span> <span class="cit-name-surname">Gout</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Johri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Doak</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Differential retention and divergent resolution of duplicate genes following whole-genome duplication.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1665</span>–<span class="cit-lpage">1675</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.173740.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.173740.114%26rft_id%253Dinfo%253Apmid%252F25085612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMC8xNjY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melakeberhan</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ferris</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Impact of Meloidogyne incognita on physiological efficiency of Vitis vinifera.</span> <abbr class="cit-jnl-abbrev">J. Nematol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BNematol%26rft.aulast%253DMelakeberhan%26rft.auinit1%253DH.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D74%26rft.epage%253D80%26rft.atitle%253DImpact%2Bof%2BMeloidogyne%2Bincognita%2Bon%2BPhysiological%2BEfficiency%2Bof%2BVitis%2Bvinifera.%26rft_id%253Dinfo%253Apmid%252F19287578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19287578&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.225.30" data-doi="10.1079/9781845934927.0001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moens</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. N.</span> <span class="cit-name-surname">Perry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Starr</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Meloidogyne species – a diverse group of novel and important plant parasites</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span> in <span class="cit-source">Root-knot Nematodes</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Perry</span>,  <span class="cit-name-given-names">R. N.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Moens</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Starr</span></span>. <span class="cit-publ-name">CAB International</span>, <span class="cit-publ-loc">Wallingford</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1079/9781845934927.0001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRoot-knot%2BNematodes%26rft_id%253Dinfo%253Adoi%252F10.1079%252F9781845934927.0001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/9781845934927.0001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.31" data-doi="10.1111/jeb.12357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neiman</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sharbel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Schwander</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic causes of transitions from sexual reproduction to asexuality in plants and animals.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1346</span>–<span class="cit-lpage">1359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12357</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D27%26rft.spage%253D1346%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12357%26rft_id%253Dinfo%253Apmid%252F24666600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24666600&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.32" data-doi="10.1006/jmbi.2000.4042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Notredame</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Higgins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Heringa</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">T-Coffee: A novel method for fast and accurate multiple sequence alignment.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">217</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jmbi.2000.4042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNotredame%26rft.auinit1%253DC.%26rft.volume%253D302%26rft.issue%253D1%26rft.spage%253D205%26rft.epage%253D217%26rft.atitle%253DT-Coffee%253A%2BA%2Bnovel%2Bmethod%2Bfor%2Bfast%2Band%2Baccurate%2Bmultiple%2Bsequence%2Balignment.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4042%26rft_id%253Dinfo%253Apmid%252F10964570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10964570&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089352000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.33" data-doi="10.1073/pnas.0805946105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Opperman</span>,  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Bird</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Williamson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Rokhsar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Burke</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequence and genetic map of Meloidogyne hapla: A compact nematode genome for plant parasitism.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">14802</span>–<span class="cit-lpage">14807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0805946105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0805946105%26rft_id%253Dinfo%253Apmid%252F18809916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzM5LzE0ODAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.34" data-doi="10.1038/nature09529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Rancati</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. D.</span> <span class="cit-name-surname">Bradford</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Saraf</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">325</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09529</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPavelka%26rft.auinit1%253DN.%26rft.volume%253D468%26rft.issue%253D7321%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DAneuploidy%2Bconfers%2Bquantitative%2Bproteome%2Bchanges%2Band%2Bphenotypic%2Bvariation%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09529%26rft_id%253Dinfo%253Apmid%252F20962780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20962780&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284051000054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.225.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Press</span>,  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Teukolsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Vetterling</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Flannery</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Numerical Recipes in C</span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.36" data-doi="10.1073/pnas.0608218104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>,  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Frank</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Conant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Byrne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Woolfit</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Independent sorting-out of thousands of duplicated gene pairs in two yeast species descended from a whole-genome duplication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8397</span>–<span class="cit-lpage">8402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0608218104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608218104%26rft_id%253Dinfo%253Apmid%252F17494770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzIwLzgzOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzIyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.37" data-doi="10.1101/gr.6341207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Wolfe</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A burst of protein sequence evolution and a prolonged period of asymmetric evolution follow gene duplication in yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.6341207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6341207%26rft_id%253Dinfo%253Apmid%252F18025270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.38" data-doi="10.1093/gbe/evv216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scienski</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Fay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Conant</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Patterns of Gene Conversion in Duplicated Yeast Histones Suggest Strong Selection on a Coadapted Macromolecular Complex.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3249</span>–<span class="cit-lpage">3258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv216%26rft_id%253Dinfo%253Apmid%252F26560339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26560339&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.225.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sokal</span>,  <span class="cit-name-given-names">R. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Rohlf</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source">Biometry</span>, <span class="cit-edition">Ed. 3rd</span>. <span class="cit-publ-name">W. H. Freeman and Company</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.40" data-doi="10.1093/nar/29.1.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sternberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Thierry-Mieg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Spieth</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">WormBase: Network access to the genome and biology of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/29.1.82</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStein%26rft.auinit1%253DL.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D82%26rft.epage%253D86%26rft.atitle%253DWormBase%253A%2Bnetwork%2Baccess%2Bto%2Bthe%2Bgenome%2Band%2Bbiology%2Bof%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.1.82%26rft_id%253Dinfo%253Apmid%252F11125056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.1.82&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11125056&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166360300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.41" data-doi="10.1093/gbe/evx201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szitenberg</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Salazar-Jaramillo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. C.</span> <span class="cit-name-surname">Blok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Laetsch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Joseph</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative Genomics of Apomictic Root-Knot Nematodes: Hybridization, Ploidy, and Dynamic Genome Change.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2844</span>–<span class="cit-lpage">2861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evx201</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D2844%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.42" data-doi="10.1534/genetics.111.137349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Woodhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Schnable</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Pedersen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Altered patterns of fractionation and exon deletions in Brassica rapa support a two-step model of paleohexaploidy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1563</span>–<span class="cit-lpage">1574</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.137349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137349%26rft_id%253Dinfo%253Apmid%252F22308264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzIyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.43" data-doi="10.1163/156854105776186325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tigano</span>,  <span class="cit-name-given-names">M.</span>, </span></li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Carneiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jeyaprakash</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dickson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Adams</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Phylogeny of Meloidogyne spp. based on 18S rDNA and the intergenic region of mitochondrial DNA sequences.</span> <abbr class="cit-jnl-abbrev">Nematology</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">862</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1163/156854105776186325</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNematology%26rft.volume%253D7%26rft.spage%253D851%26rft_id%253Dinfo%253Adoi%252F10.1163%252F156854105776186325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1163/156854105776186325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.225.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Triantaphyllou</span>,  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Cytogenetics, cytotaxonomy and phylogeny of root-knot nematodes</span>, pp. <span class="cit-fpage">113</span>–<span class="cit-lpage">126</span> in <span class="cit-source">An Advanced Treatise on Meloidogyne</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Sasser</span>,  <span class="cit-name-given-names">J. N.</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Carter</span></span>. <span class="cit-publ-name">North Carolina State University Graphics</span>, <span class="cit-publ-loc">Raleigh, N.C.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.45" data-doi="10.1146/annurev.phyto.39.1.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trudgill</span>,  <span class="cit-name-given-names">D. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. C.</span> <span class="cit-name-surname">Blok</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Apomictic,polyphagous root-knot nematodes Exceptionally successful and damaging biotrophic root pathogens.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">77</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.phyto.39.1.53</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DTrudgill%26rft.auinit1%253DD.%2BL.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D77%26rft.atitle%253DAPOMICTIC%252C%2BPOLYPHAGOUS%2BROOT-KNOT%2BNEMATODES%253A%2BExceptionally%2BSuccessful%2Band%2BDamaging%2BBiotrophic%2BRoot%2BPathogens%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.39.1.53%26rft_id%253Dinfo%253Apmid%252F11701859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.39.1.53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11701859&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F225.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170927100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.225.46" data-doi="10.1163/138855410X526831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wesemael</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Viaene</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Moens</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Root-knot nematodes (Meloidogyne spp.) in Europe.</span> <abbr class="cit-jnl-abbrev">Nematology</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1163/138855410X526831</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNematology%26rft.volume%253D13%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1163%252F138855410X526831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1163/138855410X526831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.225.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/225.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/211" title="Inference of Historical Population-Size Changes with Allele-Frequency Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/235" title="Transcription Profiles of Age-at-Maturity-Associated Genes Suggest Cell Fate Commitment Regulation as a Key Factor in the Atlantic Salmon Maturation Process" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/225.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DucW-ytbecedY0D8JldWq50I2do-nbW48H5zvbfme_k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/225" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/225" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/225" class="active" data-icon-position="" data-hide-link-title="0">A Single, Shared Triploidy in Three Species of Parasitic Nematodes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495268" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Q8X-9mRZKkh87vuhWP4iJRPqRsKI3QbBBxWdVcT59lA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/225.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/225" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZsF3AVJlTVaT3AqCbn5gDcw8UVkzLrplzF6QSbY1p70" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/225" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/225.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495268" id="node495268--2" data-pisa="ggg;10/1/225" data-pisa-master="ggg;g3.119.400650" data-apath="/ggg/10/1/225.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Single, Shared Triploidy in Three Species of Parasitic Nematodes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ashley Schoonmaker</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5065-8906" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yue Hao</span>, <span class="highwire-citation-author" data-delta="2">David McK. Bird</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8677-4933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gavin C. Conant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">225-233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400650 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495268--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ashley Schoonmaker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashley%2BSchoonmaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoonmaker%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshley%2BSchoonmaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yue Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYue%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5065-8906" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yue Hao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David McK. Bird </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BMcK.%2BBird%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bird%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BMcK.%2BBird%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gavin C. Conant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Program in Genetics, North Carolina State University, Raleigh, NC, 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gavin%2BC.%2BConant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conant%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGavin%2BC.%2BConant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8677-4933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gavin C. Conant</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gconant@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">gconant@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/225" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/225&amp;title=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/225&amp;title=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/225&amp;title=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/225&amp;text=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/225&amp;title=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/225&amp;t=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/225&amp;title=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/225&amp;title=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495268/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495268" id="node495268--3" data-pisa="ggg;10/1/225" data-pisa-master="ggg;g3.119.400650" data-apath="/ggg/10/1/225.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Single, Shared Triploidy in Three Species of Parasitic Nematodes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ashley Schoonmaker</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5065-8906" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yue Hao</span>, <span class="highwire-citation-author" data-delta="2">David McK. Bird</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8677-4933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gavin C. Conant</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">225-233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400650 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495268--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ashley Schoonmaker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashley%2BSchoonmaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoonmaker%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshley%2BSchoonmaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yue Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYue%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5065-8906" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yue Hao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David McK. Bird </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BMcK.%2BBird%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bird%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BMcK.%2BBird%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gavin C. Conant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Bioinformatics Research Center,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Program in Genetics, North Carolina State University, Raleigh, NC, 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gavin%2BC.%2BConant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conant%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGavin%2BC.%2BConant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8677-4933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gavin C. Conant</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gconant@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">gconant@ncsu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=225&amp;rft.epage=233&amp;rft.atitle=A%20Single%2C%20Shared%20Triploidy%20in%20Three%20Species%20of%20Parasitic%20Nematodes&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Schoonmaker%2C+Ashley&amp;rft.au=Hao%2C+Yue&amp;rft.au=Bird%2C+David+McK.&amp;rft.au=Conant%2C+Gavin+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495268/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495268/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495268/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495268/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495268/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495268/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495268/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495268/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495268/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495268/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495268/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495268/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/225" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/225.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/225.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/225.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
