<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/333" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/333.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/333.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/333.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/333.atom" />
<meta name="HW.pisa" content="ggg;10/1/333" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in Caenorhabditis Endoderm Specification" />
<meta name="DC.Identifier" content="10.1534/g3.119.400724" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Maduro. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gene regulatory networks and their evolution are important in the study of animal development. In the nematode, Caenorhabditis elegans , the endoderm (gut) is generated from a single embryonic precursor, E. Gut is specified by the maternal factor [SKN-1][1], which activates the MED → [END-1][2],3 → [ELT-2][3],7 cascade of GATA transcription factors. In this work, genome sequences from over two dozen species within the Caenorhabditis genus are used to identify MED and [END-1][2],3 orthologs. Predictions are validated by comparison of gene structure, protein conservation, and putative cis -regulatory sites. All three factors occur together, but only within the Elegans supergroup, suggesting they originated at its base. The MED factors are the most diverse and exhibit an unexpectedly extensive gene amplification. In contrast, the highly conserved [END-1][2] orthologs are unique in nearly all species and share extended regions of conservation. The [END-1][2],3 proteins share a region upstream of their zinc finger and an unusual amino-terminal poly-serine domain exhibiting high codon bias. Compared with [END-1][2], the [END-3][4] proteins are otherwise less conserved as a group and are typically found as paralogous duplicates. Hence, all three factors are under different evolutionary constraints. Promoter comparisons identify motifs that suggest the [SKN-1][1], MED, and END factors function in a similar gut specification network across the Elegans supergroup that has been conserved for tens of millions of years. A model is proposed to account for the rapid origin of this essential kernel in the gut specification network, by the upstream intercalation of duplicate genes into a simpler ancestral network.

 [1]: https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724
 [2]: https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724
 [3]: https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724
 [4]: https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724" />
<meta name="DC.Contributor" content="Morris F. Maduro" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in Caenorhabditis Endoderm Specification" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gene regulatory networks and their evolution are important in the study of animal development. In the nematode, &lt;i&gt;Caenorhabditis elegans&lt;/i&gt;, the endoderm (gut) is generated from a single embryonic precursor, E. Gut is specified by the maternal factor SKN-1, which activates the MED → END-1,3 → ELT-2,7 cascade of GATA transcription factors. In this work, genome sequences from over two dozen species within the &lt;i&gt;Caenorhabditis&lt;/i&gt; genus are used to identify MED and END-1,3 orthologs. Predictions are validated by comparison of gene structure, protein conservation, and putative &lt;i&gt;cis&lt;/i&gt;-regulatory sites. All three factors occur together, but only within the Elegans supergroup, suggesting they originated at its base. The MED factors are the most diverse and exhibit an unexpectedly extensive gene amplification. In contrast, the highly conserved END-1 orthologs are unique in nearly all species and share extended regions of conservation. The END-1,3 proteins share a region upstream of their zinc finger and an unusual amino-terminal poly-serine domain exhibiting high codon bias. Compared with END-1, the END-3 proteins are otherwise less conserved as a group and are typically found as paralogous duplicates. Hence, all three factors are under different evolutionary constraints. Promoter comparisons identify motifs that suggest the SKN-1, MED, and END factors function in a similar gut specification network across the Elegans supergroup that has been conserved for tens of millions of years. A model is proposed to account for the rapid origin of this essential kernel in the gut specification network, by the upstream intercalation of duplicate genes into a simpler ancestral network.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/333" />
<meta name="citation_id" content="10/1/333" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/333" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/333.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/333.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/333.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400724" />
<meta name="citation_pmid" content="31740453" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="333" />
<meta name="citation_lastpage" content="356" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Morris F. Maduro" />
<meta name="citation_author_institution" content="Molecular, Cell and Systems Biology Department, University of California, Riverside, CA 92521" />
<meta name="citation_author_email" content="mmaduro@ucr.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6257-7612" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. A. Allen;citation_author=L. W. Hillier;citation_author=R. H. Waterston;citation_author=T. Blumenthal;citation_title=A global analysis of C. elegans trans-splicing.;citation_pages=255-264;citation_volume=21;citation_year=2011;citation_issue=2;citation_pmid=21177958;citation_doi=10.1101/gr.113811.110" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=J. H. An;citation_author=T. K. Blackwell;citation_title=SKN-1 links C. elegans mesendodermal specification to a conserved oxidative stress response.;citation_volume=17;citation_year=2003;citation_pmid=12869585;citation_doi=10.1101/gad.1107803" />
<meta name="citation_reference" content="citation_journal_title=Proceedings / . International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology;citation_journal_abbrev=Proc Int Conf Intell Syst Mol Biol;citation_author=T. L. Bailey;citation_author=C. Elkan;citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers.;citation_pages=28-36;citation_volume=2;citation_year=1994;citation_pmid=7584402" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Barriere;citation_author=S. P. Yang;citation_author=E. Pekarek;citation_author=C. G. Thomas;citation_author=E. S. Haag;citation_title=Detecting heterozygosity in shotgun genome assemblies: Lessons from obligately outcrossing nematodes.;citation_volume=19;citation_year=2009;citation_pmid=19204328;citation_doi=10.1101/gr.081851.108" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=L. R. Baugh;citation_author=A. A. Hill;citation_author=D. K. Slonim;citation_author=E. L. Brown;citation_author=C. P. Hunter;citation_title=Composition and dynamics of the Caenorhabditis elegans early embryonic transcriptome;citation_pages=889-900;citation_volume=130;citation_year=2003;citation_issue=5;citation_pmid=12538516;citation_doi=10.1242/dev.00302" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. D. Bergemann;citation_author=Z. W. Ma;citation_author=E. M. Johnson;citation_title=Sequence of cDNA comprising the human pur gene and sequence-specific single-stranded-DNA-binding properties of the encoded protein.;citation_pages=5673-5682;citation_volume=12;citation_year=1992;citation_issue=12;citation_pmid=1448097;citation_doi=10.1128/MCB.12.12.5673" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. Bhambhani;citation_author=A. J. Ravindranath;citation_author=R. A. Mentink;citation_author=M. V. Chang;citation_author=M. C. Betist;citation_title=Distinct DNA binding sites contribute to the TCF transcriptional switch in C. elegans and Drosophila.;citation_pages=e1004133;citation_volume=10;citation_year=2014;citation_pmid=24516405;citation_doi=10.1371/journal.pgen.1004133" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. K. Blackwell;citation_author=B. Bowerman;citation_author=J. R. Priess;citation_author=H. Weintraub;citation_title=Formation of a monomeric DNA binding domain by Skn-1 bZIP and homeodomain elements;citation_pages=621-628;citation_volume=266;citation_year=1994;citation_issue=5185;citation_pmid=7939715;citation_doi=10.1126/science.7939715" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. E. Boeck;citation_author=T. Boyle;citation_author=Z. Bao;citation_author=J. Murray;citation_author=B. Mericle;citation_title=Specific roles for the GATA transcription factors end-1 and end-3 during C. elegans E-lineage development.;citation_pages=345-355;citation_volume=358;citation_year=2011;citation_issue=2;citation_pmid=21854766;citation_doi=10.1016/j.ydbio.2011.08.002" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. N. Booth;citation_author=B. B. Tuch;citation_author=A. D. Johnson;citation_title=Intercalation of a new tier of transcription regulation into an ancient circuit.;citation_pages=959-963;citation_volume=468;citation_year=2010;citation_issue=7326;citation_pmid=21164485;citation_doi=10.1038/nature09560" />
<meta name="citation_reference" content="citation_journal_title=Sitzungsberichte der Gesellschaft für Morphologie und Physiologie in München;citation_author=T. Boveri;citation_title=Ueber die Entstehung des Gegensatzes zwischen den Geschlechtszellen und den somatischen Zellen bei Ascaris megalocephala, nebst Bemerkungen zur Entwicklungsgeschichte der Nematoden.;citation_pages=114-125;citation_volume=8;citation_year=1892" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. Bowerman;citation_author=B. A. Eaton;citation_author=J. R. Priess;citation_title=skn-1, a maternally expressed gene required to specify the fate of ventral blastomeres in the early C. elegans embryo.;citation_pages=1061-1075;citation_volume=68;citation_year=1992;citation_issue=6;citation_pmid=1547503;citation_doi=10.1016/0092-8674(92)90078-Q" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=G. Broitman-Maduro;citation_author=K. T.-H. Lin;citation_author=W. Hung;citation_author=M. Maduro;citation_title=Specification of the C. elegans MS blastomere by the T-box factor TBX-35.;citation_volume=133;citation_year=2006;citation_pmid=16831832;citation_doi=10.1242/dev.02475" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=G. Broitman-Maduro;citation_author=M. F. Maduro;citation_author=J. H. Rothman;citation_title=The noncanonical binding site of the MED-1 GATA factor defines differentially regulated target genes in the C. elegans mesendoderm.;citation_pages=427-433;citation_volume=8;citation_year=2005;citation_issue=3;citation_pmid=15737937;citation_doi=10.1016/j.devcel.2005.01.014" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=G. Broitman-Maduro;citation_author=M. Owraghi;citation_author=W. Hung;citation_author=S. Kuntz;citation_author=P. W. Sternberg;citation_title=The NK-2 class homeodomain factor CEH-51 and the T-box factor TBX-35 have overlapping function in C. elegans mesoderm development.;citation_volume=136;citation_year=2009;citation_pmid=19605496;citation_doi=10.1242/dev.038307" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. S. Carroll;citation_author=D. E. Gilbert;citation_author=X. Liu;citation_author=J. W. Cheung;citation_author=J. E. Michnowicz;citation_title=SKN-1 domain folding and basic region monomer stabilization upon DNA binding;citation_pages=2227-2238;citation_volume=11;citation_year=1997;citation_issue=17;citation_pmid=9303538;citation_doi=10.1101/gad.11.17.2227" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=H. Choi;citation_author=G. Broitman-Maduro;citation_author=M. F. Maduro;citation_title=Partially compromised specification causes stochastic effects on gut development in C. elegans.;citation_pages=49-60;citation_volume=427;citation_year=2017;citation_doi=10.1016/j.ydbio.2017.05.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. Coghlan;citation_author=K. H. Wolfe;citation_title=Fourfold Faster Rate of Genome Rearrangement in Nematodes Than in Drosophila;citation_pages=857-867;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12045140;citation_doi=10.1101/gr.172702" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=C. Coroian;citation_author=G. Broitman-Maduro;citation_author=M. F. Maduro;citation_title=Med-type GATA factors and the evolution of mesendoderm specification in nematodes.;citation_pages=444-455;citation_volume=289;citation_year=2006;citation_issue=2;citation_pmid=16325171;citation_doi=10.1016/j.ydbio.2005.10.024" />
<meta name="citation_reference" content="citation_journal_title=Molecular and biochemical parasitology;citation_journal_abbrev=Mol Biochem Parasitol;citation_author=A. Couthier;citation_author=J. Smith;citation_author=P. McGarr;citation_author=B. Craig;citation_author=J. S. Gilleard;citation_title=Ectopic expression of a Haemonchus contortus GATA transcription factor in Caenorhabditis elegans reveals conserved function in spite of extensive sequence divergence.;citation_pages=241-253;citation_volume=133;citation_year=2004;citation_issue=2;citation_pmid=14698436;citation_doi=10.1016/j.molbiopara.2003.10.012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. D. Cutter;citation_title=Divergence times in Caenorhabditis and Drosophila inferred from direct estimates of the neutral mutation rate.;citation_volume=25;citation_year=2008;citation_pmid=18234705;citation_doi=10.1093/molbev/msn024" />
<meta name="citation_reference" content="citation_journal_title=Evol Lett;citation_author=A. D. Cutter;citation_author=R. H. Garrett;citation_author=S. Mark;citation_author=W. Wang;citation_author=L. Sun;citation_title=Molecular evolution across developmental time reveals rapid divergence in early embryogenesis.;citation_volume=3;citation_year=2019;citation_doi=10.1002/evl3.122" />
<meta name="citation_reference" content="citation_journal_title=Mol Cell;citation_author=A.M. Darnell;citation_author=A.R. Subramaniam;citation_author=E.K. O’Shea;citation_title=Translational Control through Differential Ribosome Pausing during Amino Acid Limitation in Mammalian Cells.;citation_pages=229-243 e211;citation_volume=71;citation_year=2018;citation_doi=10.1016/j.molcel.2018.06.041" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. H. Davidson;citation_title=Emerging properties of animal gene regulatory networks.;citation_pages=911-920;citation_volume=468;citation_year=2010;citation_issue=7326;citation_pmid=21164479;citation_doi=10.1038/nature09645" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=E. H. Davidson;citation_author=J. P. Rast;citation_author=P. Oliveri;citation_author=A. Ransick;citation_author=C. Calestani;citation_title=A provisional regulatory gene network for specification of endomesoderm in the sea urchin embryo.;citation_pages=162-190;citation_volume=246;citation_year=2002;citation_issue=1;citation_pmid=12027441;citation_doi=10.1006/dbio.2002.0635" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Dieterich;citation_author=S. W. Clifton;citation_author=L. N. Schuster;citation_author=A. Chinwalla;citation_author=K. Delehaunty;citation_title=The Pristionchus pacificus genome provides a unique perspective on nematode lifestyle and parasitism.;citation_pages=1193-1198;citation_volume=40;citation_year=2008;citation_issue=10;citation_pmid=18806794;citation_doi=10.1038/ng.227" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=A. Dineen;citation_author=E. Osborne Nishimura;citation_author=B. Goszczynski;citation_author=J. H. Rothman;citation_author=J. D. McGhee;citation_title=Quantitating transcription factor redundancy: The relative roles of the ELT-2 and ELT-7 GATA factors in the C. elegans endoderm.;citation_pages=150-161;citation_volume=435;citation_year=2018;citation_doi=10.1016/j.ydbio.2017.12.023" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. Driever;citation_author=C. Nusslein-Volhard;citation_title=The bicoid protein is a positive regulator of hunchback transcription in the early Drosophila embryo.;citation_pages=138-143;citation_volume=337;citation_year=1989;citation_issue=6203;citation_pmid=2911348;citation_doi=10.1038/337138a0" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=L. Du;citation_author=S. Tracy;citation_author=S. A. Rifkin;citation_title=Mutagenesis of GATA motifs controlling the endoderm regulator elt-2 reveals distinct dominant and secondary cis-regulatory elements.;citation_pages=160-170;citation_volume=412;citation_year=2016;citation_doi=10.1016/j.ydbio.2016.02.013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=F1000Prime Rep.;citation_author=R. E. Ellis;citation_author=S. Y. Lin;citation_title=The evolutionary origins and consequences of self-fertility in nematodes.;citation_pages=62;citation_volume=6;citation_year=2014;citation_doi=10.12703/P6-62" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Etheve;citation_author=J. Martin;citation_author=R. Lavery;citation_title=Dynamics and recognition within a protein-DNA complex: a molecular dynamics study of the SKN-1/DNA interaction.;citation_volume=44;citation_year=2016;citation_pmid=26721385;citation_doi=10.1093/nar/gkv1511" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. A. Félix;citation_title=Cryptic quantitative evolution of the vulva intercellular signaling network in Caenorhabditis.;citation_pages=103-114;citation_volume=17;citation_year=2007;citation_issue=2;citation_pmid=17240335;citation_doi=10.1016/j.cub.2006.12.024" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. A. Félix;citation_author=C. Braendle;citation_author=A. D. Cutter;citation_title=A streamlined system for species diagnosis in Caenorhabditis (Nematoda: Rhabditidae) with name designations for 15 distinct biological species.;citation_pages=e94723;citation_volume=9;citation_year=2014;citation_pmid=24727800;citation_doi=10.1371/journal.pone.0094723" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Force;citation_author=M. Lynch;citation_author=F. B. Pickett;citation_author=A. Amores;citation_author=Y. L. Yan;citation_title=Preservation of Duplicate Genes by Complementary, Degenerative Mutations;citation_pages=1531-1545;citation_volume=151;citation_year=1999;citation_issue=4;citation_pmid=10101175;citation_doi=10.1007/s10709-008-9311-5" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=C. Frøkjær-Jensen;citation_author=N. Jain;citation_author=L. Hansen;citation_author=M. W. Davis;citation_author=Y. Li;citation_title=An Abundant Class of Non-coding DNA Can Prevent Stochastic Gene Silencing in the C. elegans Germline.;citation_pages=343-357;citation_volume=166;citation_year=2016;citation_pmid=27374334;citation_doi=10.1016/j.cell.2016.05.072" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Fukushige;citation_author=B. Goszczynski;citation_author=H. Tian;citation_author=J. D. McGhee;citation_title=The Evolutionary Duplication and Probable Demise of an Endodermal GATA Factor in Caenorhabditis elegans;citation_pages=575-588;citation_volume=165;citation_year=2003;citation_issue=2;citation_pmid=14573471" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=T. Fukushige;citation_author=M. G. Hawkins;citation_author=J. D. McGhee;citation_title=The GATA-factor elt-2 is essential for formation of the Caenorhabditis elegans intestine.;citation_pages=286-302;citation_volume=198;citation_year=1998;citation_issue=2;citation_pmid=9659934;citation_doi=10.1016/s0012-1606(98)80006-7" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. Fukushige;citation_author=M. J. Hendzel;citation_author=D. P. Bazett-Jones;citation_author=J. D. McGhee;citation_title=Direct visualization of the elt-2 gut-specific GATA factor binding to a target promoter inside the living Caenorhabditis elegans embryo;citation_pages=11883-11888;citation_volume=96;citation_year=1999;citation_issue=21;citation_pmid=10518545;citation_doi=10.1073/pnas.96.21.11883" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=J. Gaudet;citation_author=S. Muttumu;citation_author=M. Horner;citation_author=S. E. Mango;citation_title=Whole-genome analysis of temporal gene expression during foregut development.;citation_pages=e352-e352;citation_volume=2;citation_year=2004;citation_issue=11;citation_pmid=15492775;citation_doi=10.1371/journal.pbio.0020352" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=W. Q. Gillis;citation_author=B. A. Bowerman;citation_author=S. Q. Schneider;citation_title=The evolution of protostome GATA factors: molecular phylogenetics, synteny, and intron/exon structure reveal orthologous relationships.;citation_pages=112-112;citation_volume=8;citation_year=2008;citation_pmid=18412965;citation_doi=10.1186/1471-2148-8-112" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=W. Q. Gillis;citation_author=J. St John;citation_author=B. Bowerman;citation_author=S. Q. Schneider;citation_title=Whole genome duplications and expansion of the vertebrate GATA transcription factor gene family.;citation_pages=207-207;citation_volume=9;citation_year=2009;citation_pmid=19695090;citation_doi=10.1186/1471-2148-9-207" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Grishkevich;citation_author=T. Hashimshony;citation_author=I. Yanai;citation_title=Core promoter T-blocks correlate with gene expression levels in C. elegans.;citation_volume=21;citation_year=2011;citation_pmid=21367940;citation_doi=10.1101/gr.113381.110" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=L. G. Guilgur;citation_author=P. Prudencio;citation_author=D. Sobral;citation_author=D. Liszekova;citation_author=A. Rosa;citation_title=Requirement for highly efficient pre-mRNA splicing during Drosophila early embryonic development.;citation_volume=3;citation_year=2014;citation_pmid=24755291;citation_doi=10.7554/eLife.02181" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. S. Haag;citation_author=D. H. A. Fitch;citation_author=M. Delattre;citation_title=From “the Worm” to “the Worms” and Back Again: The Evolutionary Developmental Biology of Nematodes.;citation_volume=210;citation_year=2018;citation_pmid=30287515;citation_doi=10.1534/genetics.118.300243" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=E. S. Haag;citation_author=C. G. Thomas;citation_title=Fundamentals of Comparative Genome Analysis in Caenorhabditis Nematodes.;citation_pages=11-21;citation_volume=1327;citation_year=2015;citation_doi=10.1007/978-1-4939-2842-2_2" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Hao;citation_author=Z. Hu;citation_author=D. Sieburth;citation_author=J. M. Kaplan;citation_title=RIC-7 promotes neuropeptide secretion.;citation_pages=e1002464-e1002464;citation_volume=8;citation_year=2012;citation_issue=1;citation_pmid=22275875;citation_doi=10.1371/journal.pgen.1002464" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=W. Houthoofd;citation_author=K. Jacobsen;citation_author=C. Mertens;citation_author=S. Vangestel;citation_author=A. Coomans;citation_title=Embryonic cell lineage of the marine nematode Pellioditis marina.;citation_pages=57-69;citation_volume=258;citation_year=2003;citation_issue=1;citation_pmid=12781682;citation_doi=10.1016/S0012-1606(03)00101-5" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. P. Hunter;citation_author=C. Kenyon;citation_title=Spatial and temporal controls target pal-1 blastomere-specification activity to a single blastomere lineage in C. elegans embryos.;citation_pages=217-226;citation_volume=87;citation_year=1996;citation_issue=2;citation_pmid=8861906;citation_doi=10.1016/S0092-8674(00)81340-9" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=H. Kaneko;citation_author=E. Kobayashi;citation_author=M. Yamamoto;citation_author=R. Shimizu;citation_title=N- and C-terminal transactivation domains of GATA1 protein coordinate hematopoietic program.;citation_volume=287;citation_year=2012;citation_pmid=22556427;citation_doi=10.1074/jbc.M112.370437" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=W. J. Kent;citation_author=A. M. Zahler;citation_title=Conservation, Regulation, Synteny, and Introns in a Large-scale C. briggsae-C. elegans Genomic Alignment;citation_pages=1115-1125;citation_volume=10;citation_year=2000;citation_issue=8;citation_pmid=10958630;citation_doi=10.1101/gr.10.8.1115" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=K. C. Kiontke;citation_author=M. A. Felix;citation_author=M. Ailion;citation_author=M. V. Rockman;citation_author=C. Braendle;citation_title=A phylogeny and molecular barcodes for Caenorhabditis, with numerous new species from rotting fruits.;citation_pages=339-339;citation_volume=11;citation_year=2011;citation_pmid=22103856;citation_doi=10.1186/1471-2148-11-339" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Konrad;citation_author=S. Flibotte;citation_author=J. Taylor;citation_author=R. H. Waterston;citation_author=D. G. Moerman;citation_title=Mutational and transcriptional landscape of spontaneous gene duplications and deletions in Caenorhabditis elegans.;citation_volume=115;citation_year=2018;citation_pmid=29941601;citation_doi=10.1073/pnas.1801930115" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=Z. Koren;citation_author=E. N. Trifonov;citation_title=Role of everlasting triplet expansions in protein evolution.;citation_pages=232-239;citation_volume=72;citation_year=2011;citation_issue=2;citation_pmid=21161200;citation_doi=10.1007/s00239-010-9425-0" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=I. Korf;citation_author=P. Flicek;citation_author=D. Duan;citation_author=M. R. Brent;citation_title=Integrating genomic homology into gene structure prediction;citation_pages=S140-148;citation_volume=17;citation_year=2001;citation_issue=90001;citation_pmid=11473003;citation_doi=10.1093/bioinformatics/17.suppl_1.S140" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Kozlov;citation_author=D. Darriba;citation_author=T. Flouri;citation_author=B. Morel;citation_author=A. Stamatakis;citation_title=RAxML-NG: A fast, scalable, and user-friendly tool for maximum likelihood phylogenetic inference.;citation_volume=35;citation_year=2019;citation_doi=10.1093/bioinformatics/btz305" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=M. Li;citation_author=C. Knyaz;citation_author=K. Tamura;citation_title=MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms.;citation_pages=1547-1549;citation_volume=35;citation_year=2018;citation_pmid=29722887;citation_doi=10.1093/molbev/msy096" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. A. Lambert;citation_author=A. W. H. Yang;citation_author=A. Sasse;citation_author=G. Cowley;citation_author=M. Albu;citation_title=Similarity regression predicts evolution of transcription factor sequence specificity.;citation_pages=981-989;citation_volume=51;citation_year=2019;citation_doi=10.1038/s41588-019-0411-1" />
<meta name="citation_reference" content="citation_journal_title=The International journal of developmental biology;citation_journal_abbrev=Int J Dev Biol;citation_author=M. Laugsch;citation_author=E. Schierenberg;citation_title=Differences in maternal supply and early development of closely related nematode species.;citation_pages=655-662;citation_volume=48;citation_year=2004;citation_issue=7;citation_pmid=15470638;citation_doi=10.1387/ijdb.031758ml" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=M. Levin;citation_author=T. Hashimshony;citation_author=F. Wagner;citation_author=I. Yanai;citation_title=Developmental milestones punctuate gene expression in the Caenorhabditis embryo.;citation_pages=1101-1108;citation_volume=22;citation_year=2012;citation_issue=5;citation_pmid=22560298;citation_doi=10.1016/j.devcel.2012.04.004" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=K. T. Lin;citation_author=G. Broitman-Maduro;citation_author=W. W. Hung;citation_author=S. Cervantes;citation_author=M. F. Maduro;citation_title=Knockdown of SKN-1 and the Wnt effector TCF/POP-1 reveals differences in endomesoderm specification in C. briggsae as compared with C. elegans.;citation_pages=296-306;citation_volume=325;citation_year=2009;citation_issue=1;citation_pmid=18977344;citation_doi=10.1016/j.ydbio.2008.10.001" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Lin;citation_author=S. Thompson;citation_author=J. R. Priess;citation_title=pop-1 encodes an HMG box protein required for the specification of a mesoderm precursor in early C. elegans embryos.;citation_pages=599-609;citation_volume=83;citation_year=1995;citation_issue=4;citation_pmid=7585963;citation_doi=10.1016/0092-8674(95)90100-0" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. J. Lipinski;citation_author=J. C. Farslow;citation_author=K. A. Fitzpatrick;citation_author=M. Lynch;citation_author=V. Katju;citation_title=High spontaneous rate of gene duplication in Caenorhabditis elegans.;citation_pages=306-310;citation_volume=21;citation_year=2011;citation_issue=4;citation_pmid=21295484;citation_doi=10.1016/j.cub.2011.01.026" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=Q. Liu;citation_author=P. Onal;citation_author=R. R. Datta;citation_author=J. M. Rogers;citation_author=U. Schmidt-Ott;citation_title=Ancient mechanisms for the evolution of the bicoid homeodomain’s function in fly development.;citation_pages=e34594;citation_volume=7;citation_year=2018;citation_doi=10.7554/eLife.34594" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. C. Lo;citation_author=S. Ha;citation_author=I. Pelczer;citation_author=S. Pal;citation_author=S. Walker;citation_title=The solution structure of the DNA-binding domain of Skn-1;citation_pages=8455-8460;citation_volume=95;citation_year=1998;citation_issue=15;citation_pmid=9671699;citation_doi=10.1073/pnas.95.15.8455" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Lowry;citation_author=J. Yochem;citation_author=C. H. Chuang;citation_author=K. Sugioka;citation_author=A. A. Connolly;citation_title=High-Throughput Cloning of Temperature-Sensitive Caenorhabditis elegans Mutants with Adult Syncytial Germline Membrane Architecture Defects.;citation_volume=5;citation_year=2015;citation_pmid=26311651;citation_doi=10.1534/g3.115.021451" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. A. Lowry;citation_author=W. R. Atchley;citation_title=Molecular evolution of the GATA family of transcription factors: conservation within the DNA-binding domain.;citation_pages=103-115;citation_volume=50;citation_year=2000;citation_issue=2;citation_pmid=10684344;citation_doi=10.1007/s002399910012" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. A. Lowry;citation_author=R. Gamsjaeger;citation_author=S. Y. Thong;citation_author=W. Hung;citation_author=A. H. Kwan;citation_title=Structural analysis of MED-1 reveals unexpected diversity in the mechanism of DNA recognition by GATA-type zinc finger domains.;citation_volume=284;citation_year=2009;citation_pmid=19095651;citation_doi=10.1074/jbc.M808712200" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. A. Lynch;citation_author=A. E. Brent;citation_author=D. S. Leaf;citation_author=M. A. Pultz;citation_author=C. Desplan;citation_title=Localized maternal orthodenticle patterns anterior and posterior in the long germ wasp Nasonia.;citation_pages=728-732;citation_volume=439;citation_year=2006;citation_issue=7077;citation_pmid=16467838;citation_doi=10.1038/nature04445" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. Maduro;citation_author=R. J. Hill;citation_author=P. J. Heid;citation_author=E. D. Newman-Smith;citation_author=J. Zhu;citation_title=Genetic redundancy in endoderm specification within the genus Caenorhabditis.;citation_pages=509-522;citation_volume=284;citation_year=2005;citation_issue=2;citation_pmid=15979606;citation_doi=10.1016/j.ydbio.2005.05.016" />
<meta name="citation_reference" content="citation_journal_title=Semin. Cell Dev. Biol.;citation_author=M. F. Maduro;citation_title=Gut Development in C. elegans.;citation_pages=3-11;citation_volume=66;citation_year=2017;citation_doi=10.1016/j.semcdb.2017.01.001" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=M. F. Maduro;citation_author=G. Broitman-Maduro;citation_author=H. Choi;citation_author=F. Carranza;citation_author=A. Chia-Yi Wu;citation_title=MED GATA factors promote robust development of the C. elegans endoderm.;citation_pages=66-79;citation_volume=404;citation_year=2015;citation_pmid=25959238;citation_doi=10.1016/j.ydbio.2015.04.025" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. F. Maduro;citation_author=G. Broitman-Maduro;citation_author=I. Mengarelli;citation_author=J. H. Rothman;citation_title=Maternal deployment of the embryonic SKN-1--&gt;MED-1,2 cell specification pathway in C. elegans.;citation_pages=590-601;citation_volume=301;citation_year=2007;citation_issue=2;citation_pmid=16979152;citation_doi=10.1016/j.ydbio.2006.08.029" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. F. Maduro;citation_author=J. J. Kasmir;citation_author=J. Zhu;citation_author=J. H. Rothman;citation_title=The Wnt effector POP-1 and the PAL-1/Caudal homeoprotein collaborate with SKN-1 to activate C. elegans endoderm development.;citation_pages=510-523;citation_volume=285;citation_year=2005;citation_issue=2;citation_pmid=16084508;citation_doi=10.1016/j.ydbio.2005.06.022" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. F. Maduro;citation_author=R. Lin;citation_author=J. H. Rothman;citation_title=Dynamics of a developmental switch: recursive intracellular and intranuclear redistribution of Caenorhabditis elegans POP-1 parallels Wnt-inhibited transcriptional repression.;citation_pages=128-142;citation_volume=248;citation_year=2002;citation_issue=1;citation_pmid=12142026;citation_doi=10.1006/dbio.2002.0721" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. F. Maduro;citation_author=M. D. Meneghini;citation_author=B. Bowerman;citation_author=G. Broitman-Maduro;citation_author=J. H. Rothman;citation_title=Restriction of mesendoderm to a single blastomere by the combined action of SKN-1 and a GSK-3beta homolog is mediated by MED-1 and -2 in C. elegans.;citation_pages=475-485;citation_volume=7;citation_year=2001;citation_issue=3;citation_pmid=11463373;citation_doi=10.1016/S1097-2765(01)00195-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Mathelier;citation_author=X. Zhao;citation_author=A. W. Zhang;citation_author=F. Parcy;citation_author=R. Worsley-Hunt;citation_title=JASPAR 2014: an extensively expanded and updated open-access database of transcription factor binding profiles.;citation_volume=42;citation_year=2014;citation_pmid=24194598;citation_doi=10.1093/nar/gkt997" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. D. McGhee;citation_author=T. Fukushige;citation_author=M. W. Krause;citation_author=S. E. Minnema;citation_author=B. Goszczynski;citation_title=ELT-2 is the predominant transcription factor controlling differentiation and function of the C. elegans intestine, from embryo to adult.;citation_pages=551-565;citation_volume=327;citation_year=2009;citation_issue=2;citation_pmid=19111532;citation_doi=10.1016/j.ydbio.2008.11.034" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=A. P. McGregor;citation_title=How to get ahead: the origin, evolution and function of bicoid.;citation_pages=904-913;citation_volume=27;citation_year=2005;citation_issue=9;citation_pmid=16108065;citation_doi=10.1002/bies.20285" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. McVey;citation_author=S. E. Lee;citation_title=MMEJ repair of double-strand breaks (director's cut): deleted sequences and alternative endings.;citation_pages=529-538;citation_volume=24;citation_year=2008;citation_issue=11;citation_pmid=18809224;citation_doi=10.1016/j.tig.2008.08.007" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=N. Memar;citation_author=S. Schiemann;citation_author=C. Hennig;citation_author=D. Findeis;citation_author=B. Conradt;citation_title=Twenty million years of evolution: The embryogenesis of four Caenorhabditis species are indistinguishable despite extensive genome divergence.;citation_pages=182-199;citation_volume=447;citation_year=2019;citation_doi=10.1016/j.ydbio.2018.12.022" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=I. P. Moskowitz;citation_author=J. H. Rothman;citation_title=lin-12 and glp-1 are required zygotically for early embryonic cellular interactions and are regulated by maternal GLP-1 signaling in Caenorhabditis elegans;citation_pages=4105-4117;citation_volume=122;citation_year=1996;citation_issue=12;citation_pmid=9012530" />
<meta name="citation_reference" content="citation_journal_title=Development, growth &amp; differentiation;citation_journal_abbrev=Dev Growth Differ;citation_author=R. Murakami;citation_author=T. Okumura;citation_author=H. Uchiyama;citation_title=GATA factors as key regulatory molecules in the development of Drosophila endoderm.;citation_pages=581-589;citation_volume=47;citation_year=2005;citation_issue=9;citation_pmid=16316403;citation_doi=10.1111/j.1440-169X.2005.00836.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Nomoto;citation_author=Y. Kubota;citation_author=Y. Ohnishi;citation_author=K. Kasahara;citation_author=A. Tomita;citation_title=Gene Cascade Finder: A tool for identification of gene cascades and its application in Caenorhabditis elegans.;citation_pages=e0215187;citation_volume=14;citation_year=2019;citation_doi=10.1371/journal.pone.0215187" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. G. Omichinski;citation_author=G. M. Clore;citation_author=O. Schaad;citation_author=G. Felsenfeld;citation_author=C. Trainor;citation_title=NMR structure of a specific DNA complex of Zn-containing DNA binding domain of GATA-1;citation_pages=438-446;citation_volume=261;citation_year=1993;citation_issue=5120;citation_pmid=8332909;citation_doi=10.1126/science.8332909" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=N. Omranian;citation_author=Z. Nikoloski;citation_title=Computational Approaches to Study Gene Regulatory Networks.;citation_pages=283-295;citation_volume=1629;citation_year=2017;citation_doi=10.1007/978-1-4939-7125-1_18" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. Owraghi;citation_author=G. Broitman-Maduro;citation_author=T. Luu;citation_author=H. Roberson;citation_author=M. F. Maduro;citation_title=Roles of the Wnt effector POP-1/TCF in the C. elegans endomesoderm specification gene network.;citation_pages=209-221;citation_volume=340;citation_year=2010;citation_issue=2;citation_pmid=19818340;citation_doi=10.1016/j.ydbio.2009.09.042" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=B. D. Page;citation_author=W. Zhang;citation_author=K. Steward;citation_author=T. Blumenthal;citation_author=J. R. Priess;citation_title=ELT-1, a GATA-like transcription factor, is required for epidermal cell fates in Caenorhabditis elegans embryos.;citation_pages=1651-1661;citation_volume=11;citation_year=1997;citation_issue=13;citation_pmid=9224715;citation_doi=10.1101/gad.11.13.1651" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Pal;citation_author=M. C. Lo;citation_author=D. Schmidt;citation_author=I. Pelczer;citation_author=S. Thurber;citation_title=Skn-1: Evidence for a bipartite recognition helix in DNA binding;citation_pages=5556-5561;citation_volume=94;citation_year=1997;citation_issue=11;citation_pmid=9159111;citation_doi=10.1073/pnas.94.11.5556" />
<meta name="citation_reference" content="citation_journal_title=Curr. Top. Dev. Biol.;citation_author=I. S. Peter;citation_author=E. H. Davidson;citation_title=Implications of Developmental Gene Regulatory Networks Inside and Outside Developmental Biology.;citation_pages=237-251;citation_volume=117;citation_year=2016;citation_pmid=26969981;citation_doi=10.1016/bs.ctdb.2015.12.014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Raj;citation_author=S. A. Rifkin;citation_author=E. Andersen;citation_author=A. van Oudenaarden;citation_title=Variability in gene expression underlies incomplete penetrance.;citation_pages=913-918;citation_volume=463;citation_year=2010;citation_issue=7283;citation_pmid=20164922;citation_doi=10.1038/nature08781" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=S. M. Robertson;citation_author=M. C. Lo;citation_author=R. Odom;citation_author=X. D. Yang;citation_author=J. Medina;citation_title=Functional analyses of vertebrate TCF proteins in C. elegans embryos.;citation_pages=115-123;citation_volume=355;citation_year=2011;citation_issue=1;citation_pmid=21539828;citation_doi=10.1016/j.ydbio.2011.04.012" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. E. Rocheleau;citation_author=W. D. Downs;citation_author=R. Lin;citation_author=C. Wittmann;citation_author=Y. Bei;citation_title=Wnt signaling and an APC-related gene specify endoderm in early C. elegans embryos.;citation_pages=707-716;citation_volume=90;citation_year=1997;citation_issue=4;citation_pmid=9288750;citation_doi=10.1016/S0092-8674(00)80531-0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. W. Roy;citation_author=W. Gilbert;citation_title=The pattern of intron loss.;citation_volume=102;citation_year=2005;citation_pmid=15642949;citation_doi=10.1073/pnas.0408274102" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. W. Roy;citation_author=D. Penny;citation_title=Smoke without fire: most reported cases of intron gain in nematodes instead reflect intron losses.;citation_volume=23;citation_year=2006;citation_pmid=16943250;citation_doi=10.1093/molbev/msl098" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=D. Rudel;citation_author=J. Kimble;citation_title=Evolution of discrete Notch-like receptors from a distant gene duplication in Caenorhabditis.;citation_pages=319-333;citation_volume=4;citation_year=2002;citation_issue=5;citation_pmid=12356262;citation_doi=10.1046/j.1525-142X.2002.02027.x" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=K. Sawicka;citation_author=M. Bushell;citation_author=K. A. Spriggs;citation_author=A. E. Willis;citation_title=Polypyrimidine-tract-binding protein: a multifunctional RNA-binding protein.;citation_volume=36;citation_year=2008;citation_pmid=18631133;citation_doi=10.1042/BST0360641" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Sawyer;citation_author=S. Glass;citation_author=T. Li;citation_author=G. Shemer;citation_author=N. D. White;citation_title=Overcoming redundancy: an RNAi enhancer screen for morphogenesis genes in Caenorhabditis elegans.;citation_volume=188;citation_year=2011;citation_pmid=21527776;citation_doi=10.1534/genetics.111.129486" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=E. Schierenberg;citation_title=Three sons of fortune: early embryogenesis, evolution and ecology of nematodes.;citation_pages=841-847;citation_volume=23;citation_year=2001;citation_issue=9;citation_pmid=11536296;citation_doi=10.1002/bies.1119" />
<meta name="citation_reference" content="citation_author=E. Schierenberg;citation_title=Embryological variation during nematode development.;citation_year=2006;citation_doi=10.1895/wormbook.1.55.1" />
<meta name="citation_reference" content="citation_journal_title=Dev. Genes Evol.;citation_author=P. H. Schiffer;citation_author=N. A. Nsah;citation_author=H. Grotehusmann;citation_author=M. Kroiher;citation_author=C. Loer;citation_title=Developmental variations among Panagrolaimid nematodes indicate developmental system drift within a small taxonomic unit.;citation_pages=183-188;citation_volume=224;citation_year=2014;citation_doi=10.1007/s00427-014-0471-2" />
<meta name="citation_reference" content="citation_journal_abbrev=EvoDevo;citation_author=J. Schulze;citation_author=E. Schierenberg;citation_title=Evolution of embryonic development in nematodes.;citation_pages=18-18;citation_volume=2;citation_year=2011;citation_issue=1;citation_pmid=21929824;citation_doi=10.1186/2041-9139-2-18" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=P. Shetty;citation_author=M. C. Lo;citation_author=S. M. Robertson;citation_author=R. Lin;citation_title=C. elegans TCF protein, POP-1, converts from repressor to activator as a result of Wnt-induced lowering of nuclear levels.;citation_pages=584-592;citation_volume=285;citation_year=2005;citation_issue=2;citation_pmid=16112103;citation_doi=10.1016/j.ydbio.2005.07.008" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Siepel;citation_author=G. Bejerano;citation_author=J. S. Pedersen;citation_author=A. S. Hinrichs;citation_author=M. Hou;citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.;citation_volume=15;citation_year=2005;citation_pmid=16024819;citation_doi=10.1101/gr.3715005" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=E. M. Sommermann;citation_author=K. R. Strohmaier;citation_author=M. F. Maduro;citation_author=J. H. Rothman;citation_title=Endoderm development in Caenorhabditis elegans: the synergistic action of ELT-2 and -7 mediates the specificationâdifferentiation transition.;citation_pages=154-166;citation_volume=347;citation_year=2010;citation_issue=1;citation_pmid=20807527;citation_doi=10.1016/j.ydbio.2010.08.020" />
<meta name="citation_reference" content="citation_author=J. Spieth;citation_author=D. Lawson;citation_author=P. Davis;citation_author=G. Williams;citation_author=K. Howe;citation_title=Overview of gene structure in C. elegans.;citation_year=2014;citation_doi=10.1895/wormbook.1.65.2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Stauber;citation_author=H. Jackle;citation_author=U. Schmidt-Ott;citation_title=The anterior determinant bicoid of Drosophila is a derived Hox class 3 gene;citation_pages=3786-3789;citation_volume=96;citation_year=1999;citation_issue=7;citation_pmid=10097115;citation_doi=10.1073/pnas.96.7.3786" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=L. D. Stein;citation_author=Z. Bao;citation_author=D. Blasiar;citation_author=T. Blumenthal;citation_author=M. R. Brent;citation_title=The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics.;citation_pages=E45-E45;citation_volume=1;citation_year=2003;citation_issue=2;citation_pmid=14624247;citation_doi=10.1371/journal.pbio.0000045" />
<meta name="citation_reference" content="citation_journal_title=Evol Lett;citation_author=L. Stevens;citation_author=M. A. Felix;citation_author=T. Beltran;citation_author=C. Braendle;citation_author=C. Caurcel;citation_title=Comparative genomics of 10 new Caenorhabditis species.;citation_pages=217-236;citation_volume=3;citation_year=2019;citation_doi=10.1002/evl3.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Sullivan-Brown;citation_author=P. Tandon;citation_author=K. E. Bird;citation_author=D. J. Dickinson;citation_author=S. C. Tintori;citation_title=Identifying Regulators of Morphogenesis Common to Vertebrate Neural Tube Closure and Caenorhabditis elegans Gastrulation.;citation_volume=202;citation_year=2016;citation_pmid=26434722;citation_doi=10.1534/genetics.115.183137" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. E. Sulston;citation_author=E. Schierenberg;citation_author=J. G. White;citation_author=J. N. Thomson;citation_title=The embryonic cell lineage of the nematode Caenorhabditis elegans.;citation_pages=64-119;citation_volume=100;citation_year=1983;citation_issue=1;citation_pmid=6684600;citation_doi=10.1016/0012-1606(83)90201-4" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Taheri-Ghahfarokhi;citation_author=B. J. M. Taylor;citation_author=R. Nitsch;citation_author=A. Lundin;citation_author=A. L. Cavallo;citation_title=Decoding non-random mutational signatures at Cas9 targeted sites.;citation_volume=46;citation_year=2018;citation_pmid=30032200;citation_doi=10.1093/nar/gky653" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. J. Thorpe;citation_author=A. Schlesinger;citation_author=J. C. Carter;citation_author=B. Bowerman;citation_title=Wnt signaling polarizes an early C. elegans blastomere to distinguish endoderm from mesoderm.;citation_pages=695-705;citation_volume=90;citation_year=1997;citation_issue=4;citation_pmid=9288749;citation_doi=10.1016/S0092-8674(00)80530-9" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=Y. N. Torres Cleuren;citation_author=C. K. Ewe;citation_author=K. C. Chipman;citation_author=E. R. Mears;citation_author=C. G. Wood;citation_title=Extensive intraspecies cryptic variation in an ancient embryonic gene regulatory network.;citation_pages=e48220;citation_volume=8;citation_year=2019;citation_doi=10.7554/eLife.48220" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=M. Tremblay;citation_author=O. Sanchez-Ferras;citation_author=M. Bouchard;citation_title=GATA transcription factors in development and disease;citation_pages=dev164384-dev164384;citation_volume=145;citation_year=2018;citation_issue=20;citation_pmid=30348673;citation_doi=10.1242/dev.164384" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=J. R. True;citation_author=E. S. Haag;citation_title=Developmental system drift and flexibility in evolutionary trajectories.;citation_pages=109-119;citation_volume=3;citation_year=2001;citation_issue=2;citation_pmid=11341673;citation_doi=10.1046/j.1525-142x.2001.003002109.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. van Schendel;citation_author=M. Tijsterman;citation_title=Microhomology-mediated intron loss during metazoan evolution.;citation_volume=5;citation_year=2013;citation_pmid=23737326;citation_doi=10.1093/gbe/evt088" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. F. Walther;citation_author=E. Atlas;citation_author=A. Carrigan;citation_author=Y. Rouleau;citation_author=A. Edgecombe;citation_title=A serine/threonine-rich motif is one of three nuclear localization signals that determine unidirectional transport of the mineralocorticoid receptor to the nucleus.;citation_volume=280;citation_year=2005;citation_pmid=15737989;citation_doi=10.1074/jbc.M501548200" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=M. T. Weirauch;citation_author=A. Yang;citation_author=M. Albu;citation_author=A. G. Cote;citation_author=A. Montenegro-Montero;citation_title=Determination and inference of eukaryotic transcription factor sequence specificity.;citation_pages=1431-1443;citation_volume=158;citation_year=2014;citation_pmid=25215497;citation_doi=10.1016/j.cell.2014.08.009" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Wiegmann;citation_author=M. D. Trautwein;citation_author=I. S. Winkler;citation_author=N. B. Barr;citation_author=J. W. Kim;citation_title=Episodic radiations in the fly tree of life.;citation_volume=108;citation_year=2011;citation_pmid=21402926;citation_doi=10.1073/pnas.1012675108;citation_georef=2013024764" />
<meta name="citation_reference" content="Wiesenfahrt, T., J.Y. Berg, E.O. Nishimura, A.G. Robinson, B. Goszczynski et al., 2015 The Function and Regulation of the GATA Factor ELT-2 in the C. elegans Endoderm. Development." />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=E. S. Witze;citation_author=E. D. Field;citation_author=D. F. Hunt;citation_author=J. H. Rothman;citation_title=C. elegans pur alpha, an activator of end-1, synergizes with the Wnt pathway to specify endoderm.;citation_pages=12-23;citation_volume=327;citation_year=2009;citation_issue=1;citation_pmid=19084000;citation_doi=10.1016/j.ydbio.2008.11.015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. C. Woodruff;citation_author=O. Eke;citation_author=S. E. Baird;citation_author=M. A. Felix;citation_author=E. S. Haag;citation_title=Insights into species divergence and the evolution of hermaphroditism from fertile interspecies hybrids of Caenorhabditis nematodes.;citation_volume=186;citation_year=2010;citation_pmid=20823339;citation_doi=10.1534/genetics.110.120550" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Z. Yang;citation_author=L. Gu;citation_author=P. H. Romeo;citation_author=D. Bories;citation_author=H. Motohashi;citation_title=Human GATA-3 trans-activation, DNA-binding, and nuclear localization activities are organized into distinct structural domains.;citation_pages=2201-2212;citation_volume=14;citation_year=1994;citation_issue=3;citation_pmid=8114750;citation_doi=10.1128/MCB.14.3.2201" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Yoshimura;citation_author=K. Ichikawa;citation_author=M. J. Shoura;citation_author=K. L. Artiles;citation_author=I. Gabdank;citation_title=Recompleting the Caenorhabditis elegans genome.;citation_volume=29;citation_year=2019;citation_pmid=31123080;citation_doi=10.1101/gr.244830.118" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. Zhao;citation_author=N. Ihuegbu;citation_author=M. Lee;citation_author=L. Schriefer;citation_author=T. Wang;citation_title=Conserved Motifs and Prediction of Regulatory Modules in Caenorhabditis elegans.;citation_pages=469-481;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.111.001081" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=Z. Zhao;citation_author=T. J. Boyle;citation_author=Z. Bao;citation_author=J. I. Murray;citation_author=B. Mericle;citation_title=Comparative analysis of embryonic cell lineage between Caenorhabditis briggsae and Caenorhabditis elegans.;citation_pages=93-99;citation_volume=314;citation_year=2008;citation_issue=1;citation_pmid=18164284;citation_doi=10.1016/j.ydbio.2007.11.015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhao;citation_author=S. Flibotte;citation_author=J. I. Murray;citation_author=D. Blick;citation_author=T. J. Boyle;citation_title=New tools for investigating the comparative biology of Caenorhabditis briggsae and C. elegans.;citation_volume=184;citation_year=2010;citation_pmid=20008572;citation_doi=10.1534/genetics.109.110270" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. Zhu;citation_author=T. Fukushige;citation_author=J. D. McGhee;citation_author=J. H. Rothman;citation_title=Reprogramming of early embryonic blastomeres into endodermal progenitors by a Caenorhabditis elegans GATA factor;citation_pages=3809-3814;citation_volume=12;citation_year=1998;citation_issue=24;citation_pmid=9869634;citation_doi=10.1101/gad.12.24.3809" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. Zhu;citation_author=R. J. Hill;citation_author=P. J. Heid;citation_author=M. Fukuyama;citation_author=A. Sugimoto;citation_title=end-1 encodes an apparent GATA factor that specifies the endoderm precursor in Caenorhabditis elegans embryos;citation_pages=2883-2896;citation_volume=11;citation_year=1997;citation_issue=21;citation_pmid=9353257;citation_doi=10.1101/gad.11.21.2883" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in Caenorhabditis Endoderm Specification" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="Gene regulatory networks and their evolution are important in the study of animal development. In the nematode, Caenorhabditis elegans , the endoderm (gut) is generated from a single embryonic precursor, E. Gut is specified by the maternal factor [SKN-1][1], which activates the MED → [END-1][2],3 → [ELT-2][3],7 cascade of GATA transcription factors. In this work, genome sequences from over two dozen species within the Caenorhabditis genus are used to identify MED and [END-1][2],3 orthologs. Predictions are validated by comparison of gene structure, protein conservation, and putative cis -regulatory sites. All three factors occur together, but only within the Elegans supergroup, suggesting they originated at its base. The MED factors are the most diverse and exhibit an unexpectedly extensive gene amplification. In contrast, the highly conserved [END-1][2] orthologs are unique in nearly all species and share extended regions of conservation. The [END-1][2],3 proteins share a region upstream of their zinc finger and an unusual amino-terminal poly-serine domain exhibiting high codon bias. Compared with [END-1][2], the [END-3][4] proteins are otherwise less conserved as a group and are typically found as paralogous duplicates. Hence, all three factors are under different evolutionary constraints. Promoter comparisons identify motifs that suggest the [SKN-1][1], MED, and END factors function in a similar gut specification network across the Elegans supergroup that has been conserved for tens of millions of years. A model is proposed to account for the rapid origin of this essential kernel in the gut specification network, by the upstream intercalation of duplicate genes into a simpler ancestral network.

 [1]: https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724
 [2]: https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724
 [3]: https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724
 [4]: https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724" />
<meta name="og-title" property="og:title" content="Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in Caenorhabditis Endoderm Specification" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/333" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gene regulatory networks and their evolution are important in the study of animal development. In the nematode, Caenorhabditis elegans , the endoderm (gut) is generated from a single embryonic precursor, E. Gut is specified by the maternal factor [SKN-1][1], which activates the MED → [END-1][2],3 → [ELT-2][3],7 cascade of GATA transcription factors. In this work, genome sequences from over two dozen species within the Caenorhabditis genus are used to identify MED and [END-1][2],3 orthologs. Predictions are validated by comparison of gene structure, protein conservation, and putative cis -regulatory sites. All three factors occur together, but only within the Elegans supergroup, suggesting they originated at its base. The MED factors are the most diverse and exhibit an unexpectedly extensive gene amplification. In contrast, the highly conserved [END-1][2] orthologs are unique in nearly all species and share extended regions of conservation. The [END-1][2],3 proteins share a region upstream of their zinc finger and an unusual amino-terminal poly-serine domain exhibiting high codon bias. Compared with [END-1][2], the [END-3][4] proteins are otherwise less conserved as a group and are typically found as paralogous duplicates. Hence, all three factors are under different evolutionary constraints. Promoter comparisons identify motifs that suggest the [SKN-1][1], MED, and END factors function in a similar gut specification network across the Elegans supergroup that has been conserved for tens of millions of years. A model is proposed to account for the rapid origin of this essential kernel in the gut specification network, by the upstream intercalation of duplicate genes into a simpler ancestral network.

 [1]: https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724
 [2]: https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724
 [3]: https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724
 [4]: https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in Caenorhabditis Endoderm Specification | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DmuUeUIUmsr1QqDil6aVaX5YPc-_ubqkuy7LRjuoVJo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495168","apath":"\/ggg\/10\/1\/333.atom","pisa":"ggg;10\/1\/333","ac":{"\/ggg\/10\/1\/333.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/333.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/333"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/333"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/333":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WICiFOPjWGZnedCSqaQtzOdr63H2v+QjZwfMfYS4ZXVxTXoDEfpk2QMx4+G5GeZg5HHlHP3IFlWmqpxeqkG3EUbMQPDO4pInQTnxKxU3ij6QmynYCSjohHVHOBauRjxJ\\\\\\\/mWZ1HSiYwmZxfjCNqMB3cWelFCj8s1Ny3O6hVUX1nIipDEwqx4FvQ6sB+tUlw2XtJBAuRT+9xJmqWHd\\\\\\\/zqihxQ4njQ0J1sdzTsQ6VFpsty9eGT9C9COLprTes+vidv3SamzPQ2ef7sXqOa4luizvhm+i25p+e5NVDTHZ4R3cmBpvw3dACU7LwUewP7Cgx2R8uvFvaDbcfZnqk6GbJoaWZr3Z6X7R571mSfI8xYsTSF\\\\\\\/fLUmx016rJwwslc8QSQnZbBBO4R3tn6vdbOVINTB8kGRxRN9paBBIKmAhp24b5yji\\\\\\\/iBFeP5QZSYRBfpZiFZ3SbQND40cLvLG+d6e4EYW7ynSP+630xwhpY8Oo2ZUdJa7N+9QjFjLgEdT5tfPAu4nNHS9WzNAUydepxKcUPFdbmIZ2hkI13RT2Z5gz4ujfJqdjYp0Ygn3Yl3KUtFkX5pzMadfNBp5RSnBKPvUhqblK6KK5PX5wMd2JIMgqErhhW7S6JezhpKJ7gNUuVQ2x+os5psR8Hx0xdhwu0g+PwRrveM5Hxum+ir1sWV++ZZJIeyeBkmPYb3Xt\\\\\\\/2VH+avEHFqX2c7Us8zbplvPAM4S9ihTFNhGwaQjg9RorCXXayRnwhPLVW5xBdRbyF\\\\\\\/SO1I+2M\\\\\\\/q+zXB1hnqDaBX+AWpy\\\\\\\/WBfg78kJmCdkw4yb\\\\\\\/xqBAntX\\\\\\\/CzmDEJc5qKt1OUt0EUGJT53joiG1uz+rQtf2LV4KDI4dyvbirQ4XpChUQc2Ao9EftiCblMcoKIrjgw1Bk0B8TsjCAGE8B\\\\\\\/SWpM32XgBjZRAFKp9CwWEiXhDQZ3jGINVQUBs0UG2mDIvnqwGrSYEuhJXpEDAZsKcRZ3\\\\\\\/hzXTn9d10N5nEN1fN4kAReHU+rOfSgy\\\\\\\/QkgyJHwLzg4ahwu+xicxa5Y0pn6dmv2hLGrWuteSqunoXIovlnzamk1W2wfuzTPgBxfU8rDrN3vhBkv745YEPfCiVEsLyTCUSqBmhvhjThOp7nHPT2ywVhuAnqhh6vF2\\\\\\\/LQU6Uwl8jvL6FI+OpgzFxJ+FZmvqROFwD5Lu4YSLWpZ5t6QC0AFKNreyxduUdefqd+n8vuDbxWpdbFL5DrU9oVmw1qqJn0PI+zSUMJHHpDA2vzRApWzt7HUQFcgOgeXhHZ5PBFY0FEU1FWM\\\u0022,\\\u0022iv\\\u0022:\\\u0022WJFNrKngSOr2qIi9PnyRrA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f0d8e03cfb64336d7ada6d88970c003464359ac76078e36727c522ae34506d3d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022usn50u6ehiB491eomAMEGNVQ2tSzUIoQsq8zHTYopsFCwPTcWMK57vJ1wfrWfWHzJlgFDwolXbRSMpbddjrgEMRG8kkZiu1lHz24NYUgx5WNkC0oVYArEldKASQeei2m3rcy3+uQmhZrltouSriNzHgBq+rYfylFOPKvJB3ucRmTsbhay2QB6+7+AqDgfhlp6ebMD56UxMVgYJUj79LXWIimJ2aIgr8dGlsy+UpC+cToOclJCvzCA3jALtTDlXWVoC3\\\\\\\/WmovxTAbuaUJSBYtOvaXgA8Av+AjiNBH+OCvQQv8wUGye5kU0uvzoONAgz3tWhL92jbROSB7MUtrdMU13zn1dMxgiyNsY9WXImNrZXmq7pIZP1eGkEhAxO8WQ9MXFcrwQNn\\\\\\\/52tMguws3x\\\\\\\/n4XNdurkysd75dR+VdxkfMPrph3ldvslICYgRAMaTOCTUMRK+b7+tCfrif6ny7mOhfqhPlosl4Pn+efn2PzPbxcUK9MkmyW4p64LT\\\\\\\/IKFRbT5rJNn3hus3oV\\\\\\\/EVeMJSr6eLMJzM9sv5TDG05pNzoadAAQibbsdWGIyCHirMK9\\\\\\\/2Ad46sO9n8HeBpUvJfaVH7y85vBBdEkLOuEqufsBde964iZh5VJhkDyKOEOVkD+mC+7gbYY0e1eLWRoAz+SZmA9GceESELcTzBux3KYA8omO8Rd53COOP0XEtuc0nGUmXJtKFGsWj+wST5Rc9RXRA3+BWBaD7LpBMgp+vEzCVwlcxO6kg6jKRq1haoRC8g7X0ncYGUn4kgYPU1SoGozUTkAdYngyCgJ+qPvEoU4gToiGCR+02gVvf4AiFwmY0\\\\\\\/zypFuC1kXb81RuYKRk0ScXLNdi0LKI6jAOP8biZ+QPKL3BlEo8Do+d3TiXARuNWU2CB+OB+\\\\\\\/7WnW1R4SU31hhvjFhAmeh\\\\\\\/BJzhwm0Fizm9crDiP5YVqjJy5G+4v+hIR9wZ3a4p85jhdAf5UsvQ7033ppUjay+oA+wQyENatYCyHUjp3zlgh+ZGu3GJOupUNVCh3Sfspg4d4tNJvJIYsQddKYi604Y1IZcK4dOaMEVZQpjroUF6ZF00+wlD8b+5GzMDp2GVVPsNycIssscVB0tOu7QuBcr1IrpLY0QQJDzqsZP3aYDOB6JgvYWGjUbuWywQJBCRvPAIC544nfeJu2wBSEohvvBkZfcnYAxViBieroo1ISeQPXRHNic\\\\\\\/ek2p3Bcscw1ubF7f+vUZaktuH4qExFP4NLDUWW1Bflbr7jxQ7pglSh40g24UIOzK9FTZUQMuwNOhh\\\\\\\/3le6PgNvTCfoWziE0uZbnRD\\\\\\\/ig4KlzF3woqlOGsoIKvp6vB5hOwrnOj+e\\\\\\\/kECJUvfdbxE7SaQRrWsfsNTeEfGdkrXfnB4Yt6SgmW8dZVyZmjlD03SWnFZJpfCLN3TKplWWNgR4BOeBxqjGQ\\\\\\\/GDYN2IaTB69cK\\\\\\\/q22\\\\\\\/qnGt8OJ09tm8EIkE2wjyqcZx5ogBHy17kEIuCGM3LJgAVHUo9YbWyE9sxc5e\\\\\\\/Cw06kcJb0sNsw3r10rQwlrUkfOzuKq5NhjrYcVjRpbws9xEGRDx0RE6PsGqxa2\\\\\\\/izK+xzYWEjNbCFAjtqqW\\\\\\\/pFxoQBh+avQCrr9BmR8c7sG6Neg9N6EMVBNELivgHqfdIB6DARlDZB0ccZyBbfO9M83qA5g75BBwGZ4An2468aQsBifgCFbimkT0imI8aRicpOVfWhKAw11J\\\\\\\/Yz1pn6CQtKUR0T+FazPOYIdroptXXNEG+98p+PKbfZI2TjnplLAXXkCkpD\\\\\\\/vAw9AIMInW1CHHzfvBYBmPKZNxf8uqByAAV3YcXeevq79urBXhOXgDlGjbOYadfPR3fMFtxvyGVFW1UC1ExVQIW+dghtxizV1vLFRBg3+Gg8sEaAyH3\\\\\\\/S1CUW+lOOqmgt3Sr4YfPHYlwr30pa\\\\\\\/MGOoo6RCftbHkK\\\\\\\/nFA2xGhBmU1V4aOVbKxQ0oARL1W+fK9cMfl+W65ADGLOMoae14j217ykQb7zQmEKWsZcgDLnTcKmZU0GDv4S+K\\\\\\\/X\\\\\\\/npo7e+U2IWBVz3y2YH11fUvJDSz2L9do8Qq2jYYeEVaGWfdcwbVpHsYiPdpKukXLkJsOySUPGmQyqAmx4R7pLjC302QfFpBXi4WJlrjqtoUt4iiOMlW0MAKwPu09YMN6Ewfo03\\\\\\\/HpUkOVTEt\\\\\\\/SxthYZdrP92pVWFq2ESucdEanCUZqC8RXfBbFthn19nWf3UDTfi3CgKaRUcaKXxCZLPJ5j\\\\\\\/puT\\\\\\\/8hFMCVM8WRq+eFoV9DfAFMBTlfUa3oq\\\\\\\/g+NfMCwnrBhixY1IDChIV4HmbsgZs73Abx\\\\\\\/bQ3psTW1iqU4LN\\\\\\\/FVPc2hQXfCoodEuPIc4mKRhTpi2May3P\\\\\\\/kdQdMsTP9EK\\\\\\\/Rt+vJbC6nFcZSYQJLF8XEa9jtdoaZHo16bwxDFVvW0j8BbQ3wgkOmnUsa02ozW3Wer76E9nlm8PdHpqNvqkwXSYKRnpLLzP5EeLOPP3mWrs3LDSQJRAc4uQO8ZA3eNND5S6OOjDtJux6XZ9q0WZPYpZvKB8wMF\\\\\\\/jxil11myK4D5lRrrZk8og4\\\\\\\/48+J3Z5I5C2C7iVQ29sPO8pMK1zs2RIGGSQJBYYvuxDAibpp2OE4k5FMxLafDdnBRQeUdpcTuX\\\\\\\/xapsG7Dr3xsRtFGAZ+9Y8KQeFQnz2Bt+1ZkgauW3r2eGPMOhODSMyZpsjD2\\\\\\\/cd8OUUlOOaFtaVfXYhsrR5+zVZohFBTeMEb5YonfoqXKUiAoMl7q2kK6N+uhnzls04hN\\\\\\\/Mnru4qTq8Pa793Wsm+XRyZcqXWOZYNcACJKHrWmWRg42K0UEJGPkK7jnmiKdsdwxzCvFOAWQ\\\\\\\/Ra1AxWvcBMDiWMNaR7iu3iPyVo3LGYNXbGly0D8aBZyUZXs5620+kruu8JOGgX9gLF1kTWuDtYR+7pAgzOco61QX8D4WEfaHwkN5uIFFhwOPsmTUlMMzI5LeElr7x14kp41Ynrp4rkwJpimJnGOyUlii0mhfUsPPO\\\\\\\/+QeU9kT5GSO0uDYXUtkWXHYl5qod4lc1AUaA6pk7djqvIWNivJN1M\\\\\\\/9aRhHK4lSUv0AQ5l59BbmcULJO0JAi8LOQ\\\\\\\/lofEKX4fewGZKT6M72TWtvyD4ar8vuCypQUJbSImuA4qQl4eNfIvv1D96+XW0RIUZzK1HUg3iAHr5v2ZSUlqeGlqyrEJxZ15zwLVvfhSeCby6pR60H5JfAANcGcmpISlADp2sut6C7rb5TNvQ6q88FSoQs+XlFcgOepb4AmqeSYGmPvWTEl97zUMcF\\\\\\\/D7y16uwFaIeKW9aOh62JHLYFQpdBMxD4fRMuBs11LU8YWmmJyJ\\\\\\\/4FCx2n\\\\\\\/NYDklWo2H7duZOT8nfVLhKjC0DqZI3oMNPhcR1qg0QAOSM7E2lKFRV4Lpd+9oXCNSr2crron78duaYjL540NOUkbQ5+C4M\\\\\\\/JYEzJkMUYAECMBZP9110+mlokbqoZ4pwGnRjteArGRPkzHWEQY4thcMMPWWBusqV\\\\\\\/nPyIq0XbUEQ47j5H\\\\\\\/7hsmueYBLkO7PRTfOqD8DVkSAJAT\\\\\\\/Tkid1LWkd+90JWGjhOjh3EFjGJyAujnvoxgnxTqiKdedP1JogNQUVGFhRr75LTKu6juZm4D+kg2F8sSCFUfj3ht0UC11TTH+60grdKvNPknPxCHjRkhzo2Y1K9cGHibBWvHrUSn5CFo4Nxl\\\\\\\/POBFMe7XLfTTIKU2A6+Brfn5JspfUL\\\\\\\/v18pCuee7ChXUPC4mnirZPugdWkARHCUpnMT+3XGM6WM3rTRoEDOkGaETPPTR5IzwQBzaep5\\\\\\\/5wHpKxHYZJ0AH7dP4Pt++LdKZG5Dr48sd7luN1ckyvRYgcK8vCectllZ2ychDY2nxYBQ7gmRgcdvXP9xds0KdvqfXs1QGyQFw68Mzi0qieuLdA\\\\\\\/9ZwjWbIqp\\\\\\\/KDOYqfanVp9PItZb0w5mjd2KHRD4MviZ6IQbE6YtAA\\\\\\\/+W4hpCHvdyHyv0vFQmGMm3wSlKNH7bzGBVavTJ09S\\\\\\\/ktMxroDlq\\\\\\\/fi7xhHpmH2xujFiBqi96wD3wLMh6uEOLPZmGTB1ORSdpl65UkS\\\\\\\/5pGu+79xGjC\\\\\\\/zCstx4SHBSugNNvLrALJR\\\\\\\/ePFWeqG+96RHcLvJRqK3XV\\\\\\\/ZJHDslBjwmVy2ea98dsp7DD3zpYVFoIvoX1QpY+RDaQxFyRiLVXg8AyXHL70kcNWgUgSEbs494mSeyRPgxu5WEzivOWpG7klKYf6bkDTiTC9yVjv4JSV52dX38rFRvQHNlptdr1H4Uqs3TFap48zDVDqs3QfsMrpkMOovhy6kLag8mw5ncBtZkV70aWTFsDqM4dkAJWwT8HQ3elZKdebFG3aQhXJb06JmoMYV99sUJwpnI3HYoxm+7wqggtikl4uo+2Mp2D0RSWNoHd4ZZdo2pc40nrBYs7coTpZtNUlW7kNlCD5PC6Ak+WehllBY3gwVKtxEczL7m9izjVhQvTbrgYOfOPzKxePxTmaQDehOBc1jKr65Tc7TgKzouhVrsc4RILdMEmhwWyXR6wKFqkSvaDDsNySSUNR6RrL\\\\\\\/xqVS\\\\\\\/li2b2zpXsj\\\\\\\/bYRi7gx7oDSqkzmZaKfB7bgEvgyjwVrr6ZARzRYT9VRn81VaDf4G1eZMoQeUoIEr1kBmgCoy6G9ORQHgxZc2VSxC8+IBE+JEiG8PWf6vX3clpuCsbH55S8XUZ8isBObOtfEWLkGURGJAOOGBQ4yZPFlG4+aLBBrS1dKZmVTcBqDEtJo8V4mmH8bSVLDBnKEL0jlorRtYLxSMsOdvnQlip+ZYy0LQbSBTsOPYFGutx\\\\\\\/2Fk1XHzPXzdlOl5WvaN9OGU8Ym333QzRDxXMkVk0V+S25bh+uRddhVtW46k\\\\\\\/c5ha6j\\\\\\\/56OxX+v4AAqJF\\\\\\\/L4D8tuKFQP40NwO+PIRgfNC3sl\\\\\\\/iEHaXB0AdZeR7sz9ejxq9hhANpEYQxtQVd7YX7pwen\\\\\\\/fXBCrDvwbVegoXfj13O5OpFxtp4I9yiGYHzBnEoEjovMiBi06q5lB15ABwSwCjTvA4otp6hwT+dmx6wbMkFCCEWPPGKHCBM2uMrIkEtMR0qmJRtjV5kHvb+M+HrJmcw6VW0Ob7RKpZqy\\\\\\\/JnKnnTdbfLhObdGMd3sylPAk8kRRTcXHp7Pg7FtvI+RSl8E\\\\\\\/Zk3lpUm\\\\\\\/FgY5GBono8dBIcZYRdkTlFDqmEBElxASN1J1Jop933Bk1qdL\\\\\\\/n8Dclo9im+v\\\\\\\/JzKo+4bPnblPEGWRsa7spSQUsfohSRWQ4aPLfOAGL8PnK4c4mSzE\\\\\\\/Dv068fWgRHcprxI8aNHBxqVViqC92nNhIsdjqggkLd6ut3HI8Cm1BOw9E41xJB2fZA3LvQ3xEePwvAS74ad\\\\\\\/2SoVNiJrOlCbk2z+iLShkATZEx084iMiH\\\\\\\/xAkeL9aImzW+drBiK2Awt63ziHdf\\\\\\\/styqpEyIkeE86X1KqCyk2yiO7o83tnrkR5NLB0zZ7IbGiazIAm+3nE0va4vdkeLP8MXgqOrrYe6hnsljN6gYTvslnLPERoErYqG0YJi\\\\\\\/nIGaHmd9wQYtw2VAzT7\\\\\\\/WSoGNKvmvrEOiK5LoxKmdKIsm9Xqqaugy6eoL9QhLf6mHrBKYMMEifgDHFIC8XHsAjLCedWRp726eH+tzfqylBPSnKbmdba1m6OM\\\\\\\/LgCtw2UsiujYi7FL+VQWKpqws4Hn2I+6SAS66tXV+Lz75A+bUjf+2b4Y3F88KiODF3Ukm8elih2hW\\\\\\\/5E\\\\\\\/DZniYNSuYY88Ym0veyCtaQByEFSQjVdkE54yrINLYGhBbPLf1egXlWdx\\\\\\\/s5Dx2\\\\\\\/bs6yHf8WXSuTzl9VFC\\\\\\\/PVnFqKuR2PJ6Y9RpwAj5gdDfKs+\\\\\\\/NwEeAzNvINpU26P33KGDKtwwYWT8f6QaIgMbX0zoV3w+WcqC9Ue6vK3RGi11twpIcnb7cblw3lBC1APZ4pPWdS\\\\\\\/fAVyGZ9upIxetH+Z9XP7TmZm4UcnUd9Sl5QJOGKyVLcmLNqvJdTpxK2WxoWrPEfWgP8rXPAoxPcJLpAsUG7B9cH4jTLaR\\\\\\\/d4iFU0NMTlB1BN56yDnU1dPTE8kGCNlvHbGPa5\\\\\\\/VdF5Movotf7HLGUXSqvZMgb44v1CWHpNc6ZaeM8\\\\\\\/RKaOTGeCA4UxlGCX7fucBuz1YSskT1vD48okr7Yr867oeCI\\\\\\\/6RS+q5Y9s7xzs9mPC+FpgVtEMJCU+VvRDbtVbrjXR5tfDct5EvKH83JG+j7R8HJa+H2LgJU13uze+6DHj\\\\\\\/9TckogFHnmn53KmLd0jdRIRuvk9qaBGWx0jkX3tCyG35305ZxfDhwejxkSpvsYdlpw3QVo2zCc3Iou4l0TpxLK7fB4cQkZagdEzilCSIluz3DRzv\\\\\\\/XBdcHiyf4V9hbphk+pu2gZw3Mk+iGiKI+6gjzWxzjOMO7EWvuoHYWbkw4Myh9GSgqGIHNkJSzDAO6wPGfNSGCNrB2d4CJWtGtri7IWhPWJQEpwz\\\\\\\/6stQDKnnSgtg9fx1Ykgx18udPi+3fMck4a3B88jAvrlOlSXvtJ5WGxRRUw+FMHD6zqDjQ8Gg6XXZRc8++7w3cw7HK1fnr+P6kiUxr3\\\\\\\/7tcoQ8MZyDSSJ0nKmpOKnGUH6fEzyXL3d5PjZPkjSvRq6DLh7FxYMBTwgQq25vK4Nc3nGuJS+V8XSyfXiCby4I9+H4wFO1Z2IJs1M5SIURYcUbxaQ8aarYMb0DAPbUkh2\\\\\\\/Z6kfGlf\\\\\\\/8qPWlPg7PbLXFdDUZpdiH8wdv6ez\\\\\\\/YGeL4oepmUfiEKY3KRWwGiT9KVwtrR2l1p4JuRMdiBrprdzdItsDbBeioseIqnbsvPpMSNTGOjhvCWwkv6eX9khrL5IJGv1L5DaySr\\\\\\\/n7If7Ad2eir2yuJrnuA1Ka8pIsFAffikQdHsWEPQ4u\\\\\\\/r1Jz3avElAyI55AZ2RHdnxEiFS4AyGI9nLguMsI9FIk12u9WKitezJ2HX\\\\\\\/v8g7zl5aMxnT28mimHSLP9KihtPedi\\\\\\\/wekTeaItfAbje9IihED99TF7CYxuJEgtpJSNaO0kqW9c6oTpgSThWGq3gguwJ8jnZ7yOBRwN8haa81I\\\\\\\/vb4LAwaId\\\\\\\/ZSpngLkOMMx7j7IXwR4ZPklrR9zbOrvWXoNYYGr22m9JwveYxjxD75cDMpoUXWYQkfP9OOVJb8Kwcjf\\\\\\\/28b3sWrTMk95quxH+VW\\\\\\\/+t+W6Dj0IBQDge\\\\\\\/m61f9DO4ohTCCk1PQn0npAX\\\\\\\/Wfz+q83w1LyrHH4REOdGzst59dtDQxoZU41s66JDst1IsnF4xsL8ILVjNwKo4TMCDmmFDUP7cU+GGhTI80CICJbBTTRj33N4vfX8pV7cV\\\\\\\/tUadM3vwtGVQR371GBaO4nw3v11CYPD1+7kiYM6EH7OB1Ifn+jC4lMTjuLyeH1dqQ4futrXlnN9oZyCGmB5Xevl+UXEjzXf2zF8xAtBdf9F73tomWcZq3XSyohr28mxPAREtwb3W0NA\\\\\\\/TJU1aHhZmV8lbu3zGWbecUamNQwdbyNHphJy9zfVuJyDd2K2LIDpv9brlOWdT3HUmYXGEzqp96I9S\\\\\\\/OwEEzdVC7VWbGbrwAuMuuwhjo0KH2E2jB0YxUSsilF+V\\\\\\\/P7w9fNQBrJ7hNM7yZX0ctmqpLFKNKGinr09yh5CqmYFXBFaw7jWWeX28KQeKZkaebTMsKYBGe8PzRe9pEbrt7FmHUkhQScERiFrOVAzVPNbDqAj5lLfkI8\\\\\\\/\\\\\\\/\\\\\\\/wurjwuorWXWMZ6czY+tcCrTgHsuQ+nXB6IiLlf\\\\\\\/MxgMBcYwVMPyZu8+0bcdgoNaqWAOqVTpamORFibiwz9lJVvog1PTyEPYydd\\\\\\\/28qp+cyHHqQRo2SxZzpkg+F1fy4iGUgE\\\\\\\/GSRzWJ2wnfOItKb2c8U4+e2RuIIY2TV4AxE5zZu0I\\\\\\\/aRIqD0kExxIrZANsU8FbatbbKbRExohpUxKvFLKgoCGqTxlJOMpCq7HCiwkZ3tWp\\\\\\\/vhqKdimdIxkC+KNgBprSi9CkC38Dq3Z8iLXd3Uqy4Y0WnxvGqHDmyGoD6lr+05aUbISdk4h3j1+qwpRTji7U7rAdfFskGyaocm0gqVW2zSkuGXuZY7bzas+SwqoPpiUoitbInVaE5U4\\\\\\\/E8tDUyEDFXVSRUaYRfJnTz\\\\\\\/6YWWWpQ+2oG0l6bdi0XRGotrEKKtgoBFIVy4eOXPIOtyBMlyhYpY6yJPAVxuV1sJfCmwEYu3zUwjRj0yGjJQnS9mocAJCmjPMo3AhWa4GFj0rfjme606RMuI2mrcv5OMY9ksoBJ1bD0edRXj77j1syTAzR3s1+h\\\\\\\/Uo8Tt8WX\\\\\\\/S8LgDRGzzirDMoqos4UqLfrRQF1pIXlzgWET6fxIC0kw7tIhgAYdg\\\\\\\/zgGKXCSzEAToskHfBG3SUksVmLEzLIf3vEnm8Rx8HrrK+Br5+aWuJnSxAEK5VWmChEVbl\\\\\\\/ICiXI8\\\\\\\/+rfr64atsHP\\\\\\\/94midoQm9vwXs2zKM0Vp2GNbuMwqJzxiFGAHCOoVbLlelsY4wzGVA4dGlxb8CskYUrzXviNXNlc8wDApkykV5s6lxQRmWnJ0U9Ofo7NXXgaWfnabdCeAutjptxJloRbdqf5ZT\\\\\\\/6gM54FehB\\\\\\\/c134Iez+5MQAr\\\\\\\/5zoYFA+T72UUPQ0bLeFjlfVH9Dsqm0fSTx9VQobzV1MykdT1+CO+\\\\\\\/hOxh\\\\\\\/IkufgKCPB5b2j4g\\\\\\\/DX7SzZh1Gb\\\\\\\/VP\\\\\\\/7Fbp044OU2+4PCxQBnjAOnO\\\\\\\/2Jlnvf8RNo+m8VL7Mnudd9ADVDlY0DeCYSKIofHIlinDf7kV63LaYuggZBuvZIgxotblIqZuW84VwCCyATXX1+4N2k+tJTKlR5903wijGI4LpqXXJLROuQqIrXQw12ssTPGAV2pFz+0TzNid1nW2iYPRSEXKaFGQHE5aGoOImonxhP\\\\\\\/rR7HwQT83eZbESsoGRh7b0oSb78YhkATQIZlul0m8985C4U+IXFKB8+oWxMNkMt9BnxG1AwJCLIaw5cjx67sM4bvj7e3lsmgnK6oH9OrrGXdkkZytT2Dm7BDKXtvDT7B0pByLMzHDHL8+5TKUKQL\\\\\\\/KjSE7Emf6D2b6AZ4xZypU1GwQnir5lwjKST\\\\\\\/pIYNJXv+NrL9cn9FsyDyWWgPmFj+LBHKk33MW1hMN6FiLJEugmQCx1NI\\\\\\\/02oeVnDa\\\\\\\/MtKVAYiapwRWst2Knja61gh+iVVnWq8sKLcyz4r0DygnpjPF3SPlrW+izMtndL\\\\\\\/D90vPySBBG4AUDnpDJLUBrrZOt2fosq1pyyoHLfzyW0+n84YZ5tQBZQwRj8ydOW78wqPcmPi6FhXvD1Sims4+e1z2h1ApHvjZuCm8FSVftq1+7MfKu+46EkcPA90gh2E7h66VROeRf8R6RgQd0VFOE0+9RY0Htgetn\\\\\\\/LXhY3pS+F8mrlG+1gRZ1Qm37KGydGq0B8\\\\\\\/QJwD573zYtAkhr8hIAp8z8XfBu+F8AaJstaDimkQItX4CWP5i5WybPMddISfiXL7aWznaM5bKrcnGeFiNJVNvxvAe3GHR4nqODe6vKStLOw0iXkjJ7pOPtMVbiAXctF5key8WgBrSNUQT5jmR56hyBMb1aSg8U2TA925ZWhYK5CPk8a\\\\\\\/d+iMLWiW7e92cz6w4iClwoQORJyD\\\\\\\/W3BbMeSwKcd8sI7djNmfakOmsIcayx1LmP4Ii7Wfv0GwHZxMnCkrk+MpUllOieyesseK8LOOC\\\\\\\/VQKDN6mZkWO8QH5f21s4jMiW9R3LIA7zTBjbCBJ9dULSHVr5dWPChZeJuJrg3CXJ2\\\\\\\/35v89mJXljzW8+iYzcCaga+uUTerQvKMbxfucm2TkvctKDooxDLTtq5Ak4\\\\\\\/ZrO5OfxF+Ung4TIwQdYDBDbUKjDrPsBNyeC8fX6L20ZjHgaE3bFNGASPk0z1E4FdrPzBeDUvBou1EPgYbxEjDmdM9gPzE3vrTrpsq0eqAb7Gdhe4XxU3EcwgOKP21iLvMk93A7dCG7sNTENyRehGQWSDVgYWTD0n42EKitQ1G8deJLRX+AoPkfgomPOPIdPrl1eG0CRSQA5PQOBKUqjvfTgDNoZeVJIVQWqlSFoKBG7+4E1GqN+RRkuaBxIBgEqZz6hA+I1WlVJzjFkSF3QDIFFGJTx7vw5jP08chyL6auH\\\\\\\/efVM8L5kOy1dyn1UUJ1bwjcFZ3vLrpzjcIYePzMoTlGdIZS+VcAieOicAj+WIRJNWKpFoaBCCR5jK8P2SxysUmgevlaAzxBi5pl\\\\\\\/cwMPGk+cGNDc8M4gD4sI6U0qD82xUb24IejoY3MFKppCZNPU21DdF7xtZdL6TZTcIkPJH8dUxBqRkejsYhpxaXD9Z78\\\\\\\/J5EXPECTU5DpaHYupxz1WTnDsueOueLCaqTdyAGBUqDvA19NiMHnM+8rx7PB0KJZm+bf+u5lRZVRyFOsXWNyl8+p2k7tKHuW+okJfSlg1nTeji\\\\\\\/MDGmEqvZRQNzbYISAspGfDdYsNFNsgUbsJ9\\\\\\\/0fzxwna7ZiDkfr+Pb\\\\\\\/wmC6B3VvFQEcVXxCzczMKWthJu9xoxdBqnq+JfNPNl45nl+r2fRdRIWPvmfLcvbJm29\\\\\\\/dMef+ZmW5X5TTzFTq5AjFRsVPARCib4LvHW0VC0h82pfqy2wYNDsMqX0EHC0+3Pqt4I\\\\\\\/wOP80mDXMjysXV0NRL6lFC65J9JLK21jPUUF6dtVH2cdZrsxej9CzkA1DwpySFC\\\\\\\/RltaO2tCCD7a4fvH8suAJPQL5d6XU2biAPP7Iz00Fa4cuPLfixM4Va4whyAP95dApf6CP+hJ+tScPdEEPcyTA2WoBxMi2Buwli0ptVAz0cm4evk38GEO8uYXz42qlYdy4UWZQxMBFOpYq3efgxHnd3OCJhAcrwkF5JVhYmnHhcljYXkI5J1lZC0hHnIceyxUKuLqYRdRWw42wC7XQWhpJN9zPEfMMAqkB9KJ0LrfGgnk2ST\\\\\\\/O7uBQ87UBTiDl5E4Tt5aQHjsMAU4prvGHhzh9TzbvGmuSbRvvEKsAEuzDPN7Md\\\\\\\/GzdZa7\\\\\\\/Zbf2Uky4CiZaKc0LtANae21+DbTftN8d21SJFCOSzKs4XOE1soD4zY0bt6h5c+FO7\\\\\\\/8GA\\\\\\\/LgrLr1S3IiKq0ok0srEY9\\\\\\\/psjn2GVoe1pCRiuoyzyp1P6ccnLPeLGIHjqwmPnu9gq9VppuwhLgKT9fHNXdhOhbN74MekvOoBqHvgco4tBE6mFMiui4gril\\\\\\\/nXwjsQ+yRVCTgLIvzzm2B\\\\\\\/cjTlPsKjhSpBELtAw8Ozwvct61T2qaL8DH+vZKoy+tVZeqfea6YGrk9CdVEAjpKdCvbDCDk8c4u6kcJPWWzv0uTD4MW0VZrRM+zRu\\\\\\\/xPQFSy4veQ4LcqnWqg+PkIRAewCqQ2K9CbqjyxxHekNJL7elqBv14ADlwmsGUDlwnubmEpc+0l26vhwZU\\\\\\\/kjJv+WyrtDti2y3UtpGVLFDRrKquqERU7xPmdfk2bfgBPZbZkKP+cqugOLBBDOUP3XZIsoFYSM1LtVodust0GPVNSSFL5t\\\\\\\/MHVn\\\\\\\/7GA\\\\\\\/HIJAFNihYSTTi7fAt+N+IV1MwVhGMdLhktwbHtaNa+DwvdXEgj2vzs6Gd48t2GItD7Qz82UYt4q1+7+W+\\\\\\\/LZgjuQY9a0DhOywMGSL8lLNEh6CTfIo2lvbz5PpHtTqYBligMDEa0Qyzpw13HpMuwtNcJVASWV5EpsLBpMoY3BD59Sfr1ISZb58VKQseIeI8JWrGCwBOnmBRNTCkQVi\\\\\\\/1F0aybfHJ+HklGAm0Mnu1CMDsUFQNBtSpAJuBoUBIAFj3CHBl2QpjT0zpVi7uvZFsgIvLPlHkzXotkrqhNp\\\\\\\/hlx39FASq6UwcV1+E7AQ\\\\\\\/LtLmGNLzLaqp5hDlaSRqG\\\\\\\/71+v5rgDSK4OdLyj\\\\\\\/Ter3SWn9hfq\\\\\\\/Hdbw3NF44ZyHDfRa\\\\\\\/TlfXEHLdtxeDVktOEUoykcT\\\\\\\/\\\\\\\/IvbNLv8WJn+jD\\\\\\\/KlKrmQhATXiJgLGlX4oF9cTp5yMnwLyxDkpCSNd2zFpBkyPtUOZMKCnvnMDj6ELGdBAdcqYcoJftI4mO4+UnggUDvIP2VbugLvGmX2PnL9Il0np+tcXI2ZNPvzQvg1Dg6uQ7JEaYIVfqjJSf1yPfaM0JBQjKDSHJUWVl+p4Tn0jvgEhLdukg+4RPGV0SlgbZYH924haZnwBsKwoPqAs9wXFT4XzsWYGN4CwR8H5lKpKnSm56M5nlJPtaQXtBTDvwkzULXdrNeC53jfA1AmBZHvd65x1WQe4DG4nve3mEaWc4Mz80AiaC8j\\\\\\\/\\\\\\\/cgfc3I3IRFNP3P4rpy6oOiHOFEvpfc4UlXC6Z7tE8lU5PlDGkhJ+7fRd+SB1RU4b2YZnUXhNFZO2316UuckEYnkfDQ+m6u88YbuaS2DIhucnXf+S9XeOww+0pXQ\\\\\\\/MnTXRzMgoMfb2W6dKnP88HyE7xZvjWo2t+Q1vNqORUbckRY4IFsmXPoyEaJQ\\\\\\\/fwiq1GV\\\\\\\/v2frb5vpmSjQdXI2Nf5dIZuViKeDJC\\\\\\\/YZnQZdjEwY6TAdgaw87K50drLVIAFeQFQXYxDnUeOK60BMVAOFEYFyRUtan+HzC0kGFpvj\\\\\\\/DaG6Ts+X8x8aaHRk4CLwZePLMl1k4nJTIrBOubPp524FHb6f\\\\\\\/E9270pVnIlxL2BfRGk0GJc7mqnqMEFS0mLmyHj649jfdvjIpmNYXpOcH3YzfMl8XgwTaUSDqLm6m6n9d8dz3mptup6nIhERclteI6bXeAmmwUUuFtXyNM4oCpUb4sGv1LiupTsE3g1l85M+y969V23QumlHx+7Woyt37Pl2QYlyOrg8QpZt87mlNGMBspxlz4rDziejtYHH1A2aMQXzAgF4kvkZcM+18xf\\\\\\\/cJcGhx4IUes2I06Q6nCD6ccMZWk6gQ2BystLIzxyY63utmB9kNR5izuGo3kqX4fhrvwyA2WPiF4TlPymJ20YpIVuQjZ9mxmvyn9crSYQPNDTlvInXfwHp1pam1TwlqoGXtusSHBS+4rk4ySxtTit7kTwqldflOHPo5fk97cGwkyJUo4pl9JrspHv6DS9ilmVLMEvb1d\\\\\\\/np6KvVBZn7\\\\\\\/8bjSlQnjNKkcXtuGE2mPzbz7Dmfu+zidTeY963oRb5DsHE0jM0+80eduqkIz\\\\\\\/SO3989uwV2n6xou+AJcth7XE8K8NhTr+GUfbQWH1EKOYJmDOhuxgXXKjdm3TGn8i2Lh9nwyHEH8zFmh5idk14+OlZsYJPNjVz0564FWrFFyPk+zgVF96hC6e1xiDIoZiJNBVz\\\\\\\/m7dOOI8HPC2fkXXq9PL\\\\\\\/BAMzfsS9DpB\\\\\\\/XmVAu6PTqOl2BbgdAwqI1kJtmV6rukwH7hMBfxy4O2FNDmHH9wU64+123\\\\\\\/YKQ7sB3C6fPsiQRK4itDXtszid1AnYyDCb018CP4Kydgpqe93atkQ\\\\\\\/+J4mObgHQZCzmSTCoSk2cl8PVoichoK5G+nxfnpzeq3wnJNNZmja5M+hMC8NbAq9C0wDtim71WlL5C76QVx9le\\\\\\\/ZG2QjfAA7i8FZkpHbtu0bB1a3XdlGg9IQGugG1J3Y9kYZoZUZNa9LOqIJewLYksvt8GwYI8A1e5u+In4lhR9S7XnYbLsqLJazHPZyzHsPpX5t20vEwxww2Dz0tc1jMI7vPvzFQyLWeAi4qjW8jOZjWU2fjsrS7Inog38ZJxXYzZMHw1idcGP0FbHZOcVQ9Ap4OauXlQbgTJ1Rim7ga8x2xumSU\\\\\\\/pSSGlTqIOLEolYAoEK75YS6K4hB4x\\\\\\\/5xvYgb+tTAKnLFZkh7d\\\\\\\/ocrutTiXj7bgS9mT8WNzsabBEqmosVlPY4Z\\\\\\\/2pXqC6ZlMOoG4HhOmOrIb67PDwGcoLf1eyAxpUQA2K8o4F2hf7T+Hv9sQxWCTGfL+tBvw6mHEIrMajY6xuXs1yPposINqiXe5mNmA1gPkvQl9tgFXbdVmv1TBG2q2p3cCw0bkPoMQ8HbRYuQri3owM4xDpbVkBA\\\\\\\/OKUwfnES0QfWjdicDAemOI44NH9Hkx80LFWAB2Csy7h5bdAeDOWWrDD9NPsWGibznSUwk8wPLfBRU6FADS5v58U1lkjL4tHn0HjDESXN5WKLFbnQjQ77oqj5TufbT9HlsC2WCcSQCWEfg+RxNdXm+qBmyxeWU2Eb2LVsLYXet5Wzgymz9bGoUXhbJCvxIgH2d2AH1qiOL+5mTovWmIDkkN9LcUAqzlVlV+jla4aRM\\\\\\\/6Aqj7YDyGI+NDVgsNft5ho1wvKYGSdCoy7iyo0p4koxTMi3MKcxcUT+eJyXkFbJRJkJerEc3Tp8iVWPmn67KiXxkP4KbOsv1EnQT\\\\\\\/YqUBw7q5pKz+8i5Aiy0e2MklAs+sBowtJTNAOe2deeqogw2TNSkO4Gc\\\\\\\/2uveUagyefB\\\\\\\/i0jEK2ZzitgP19YIQ7hNn2F3hwQlhvxD2ilBTLc8ji6JUxOShnkMsZpij2Ogvnc8kFwGvMeeeXovW8nkBZCBTdWryRU+FoQCc2POE+Ecuc\\\\\\\/Lq2AaVhhfgQlBVXzrhQjenBkbRRAQqr5V1s0jrMnplGkIxJBSLfNUnY2fyxfgtsQTvWaRDM\\\\\\\/g8J8kq0TbLRoBn89ujNKoiiBkI+anARE5YIMdAr4l4bnHPRnGHmW8glI7ryEaS9XqA8rfWQx23uv3bNIxtNZNMEPtcFFLNsve6IoEXNvRldQGt9jUbwVMBwUXk5r+y0WJDRa47kRbtFQaTMVXnlnlbDq6ntNNTQsrdEAxaXqPU8OUQKHHu4UILcm\\\\\\\/57H97r+pdO+fCAUuGn54VpxFbgafOcWuyylLa3bJ8LP7jfXJzP2SmHm0WU1b3HMSUXxARtukijPrqgWSC256sR6bIx+Mn3gOCJhIDSVkudE4pw+klpdPq0wuy4OGv+wku5oixt62baovig5DXRtXEOMLGYnsKqrr3wZ20KurBl43nwrXqt\\\\\\\/b6Dc7pHTsmTg4LFSpYT\\\\\\\/NWFzhx4tNFDprYZr1+Awa+m1n5SD6mLknJzquqaFRnfs9rZpMAkstPZP8e6yZa3Got8OTSjZHB4dUhGgxZiK7Ax9+FToxYcc+DmplaOX5OqB1okt4UqAAA4vbSsSzl03DWkTtLmi6khM5s7rNg7Tpzdw1R1fSdTHJwzhu5k1ljfY4xREU4WsKjmt41UUbRW7lGaQ58oEC3s3qA4A818FnCaPg+ubtWfumCsMiSU98Lv0r4Nd4WRLHvyKxEr+9dEHgQGAzg1BG3CXSbozWVuNkX14VqS+0QTypN0AAMxIlFASW4Z7lRtjHso6kvrT5U+7circddcQ2kZFzbqTpJrsyr9cexiBWqA2UAM0xlwLxLh7+1t1+5OWY\\\\\\\/HVaAbucjHwl1Z9mIPSAvLAJp4+c048CIBMJQukasccn64uaECtgjkcWW++Eod9voPAU\\\\\\\/b1udQ6YJ+qqMHlPXjwPUvKA6Zc7S6V15L3qVHTHec0qIkntOO6X7Qw+yU4XCcD1O34AaSlkoBDBg2bp9+xvkvPrOeIiOZcuQdUtOgPgVmY5jF26s+\\\\\\\/pqawSffkCGI973ySGR9o04e38e4SmgLfWXaP3QQrJP6WZA0ZctNN2jaRCjvIdJaRF65L612BziqGw1wfmsdCt\\\\\\\/f9ZOp8EMnnK7iNAi6pcPtTN8n4kyS43Ai8slMQO9p7lT4QbGhr02hk56X3ANyVB8XyU9DOYXnxbC9s+4FnjJzWoW39L2JsKZKBwVDaOANJCgiikZBHxT24RHecHB\\\\\\\/DEU1ywXWNpJ7g7mX0B5aaicos71blDbmI9pJhZmigWWHlYSQXuJcFfztrVF4BHJpVGSUFOkt7PSisnPT9DKHp3xR5xbDdslW\\\\\\\/I+RWwnp\\\\\\\/rF5O4cuL5IKu4iDDqUVGQy+aJinRBTaBcUiwCcneOnYlMFQOpNF9G3DGHtWZmy2mc83C6bpu+6ZPWY+CJiEW918gn4GGtddOi372norRMZLOzirLrycmHIIGK4xRbiny0MuhdRqc3Ta\\\\\\\/Z765AI635StHon4faVb9+QQ6Z2LyYpcqrFA95XBChpSYvbguVsp\\\\\\\/PdxAbaHS9GzkrzSE\\\\\\\/XZZZ9TY\\\\\\\/utm\\\\\\\/DmENsBP8MMfpYZmcD1BrfrKgMrx977fzFRQPLykl8PYN1eoKHGUxyRP3B9tGtuvwHPwLK9f30IpFZs9xJ8OQbJEy7350OUg74UvRdQOMcOGnVqExEPS7VjZt+TfU03OiludRf4ahYezTKwhyMUQvJR6Ialxm2tj3VC+sPBOskEEJDmt2p2cst\\\\\\\/XgRU2n8HegpR1XcwxWNksIdK41ygnIfPzI32XeoYi85cz3RE1t0s7TH5U87Ka\\\\\\\/jUqDh4ZQXXH7iHUkmDV91eEM4PurPjyBHkhuYlkbz\\\\\\\/zncPLjsqieCjRPzBantwVDNpSFAjttle4yznY14y5qrwyidFJaXGNNoVWvda9s2U1Rc1AvblmJZwGUcvMfICCGYwD9b+Joij3a2zkCSCSzqYp3Qt+E6Xbj2o07q8GSofUVQF3rQl92hVG1ljUeQHEemdQDV\\\\\\\/W+\\\\\\\/vVsgFMC49ScstBRkB8B0WCUFVvNZdgHS3NGnK2e3KQLc\\\\\\\/uFPyduKG2Twv8lH1XWY2RcQ9zaCKvT5zWwlTTW8FBuBt23R3rcTDLU+vMyE9NecGimXbUMiewbTKy\\\\\\\/8rN6zmCKlWrghcxzponAixxAgIw+KrE4U90xCYHacJbq6OMbagVE0igyLMfSEOHUy963bUrzfDK1nZYo4lGvgOWECQd7kX9EKMUiJD+mzAOKR2e6nbmpXUo3WOVHjNk1xGrn4e49uAvFYcuFcUJLVR1PU9hgJO2T71trzzb\\\\\\\/Yjr+MQtlq2WSsQmcmCKt9nUp5ugJJBzLGj6jeVUDTq0hnkws+SMnaXKgft4N\\\\\\\/o4U+6nnD77wfH\\\\\\\/A439Li8eHsPZr4iCp5ALOLXL1WG9ZPjTMdkdjtDkfI1Uy4OUxg4+pxuYP5WZtwmtRkyYN4tr7vleGxW9dyDi1thbsy874Zb67ZG159rar5s8TxtOGB9RRqvQgS9Ll3jIlEXTIL\\\\\\\/fWSynxNGtmdNKRxDremwMtwqYnU7dGtjNgoT6Tzx9preN90pFyWxggA6nhVpTGzYis9KU6mJWfd1YtMshcdzkHVQ+fQvSVhWSAEKlH28\\\\\\\/GJz\\\\\\\/ZL\\\\\\\/w4z1ozrIpiMjDTbxGFizCpy3OCiorSWwGwZcoaOBPIldGrXC1MKCCXGvmugZ3y5HKQjkpzT6sKwq6M7vrrnsiR4d41J3ObqLB7RZI9\\\\\\\/noPcrPqOyEgkh6c5YVHbNOLuoIFbduOdlmoDJHsJR+7wv3vAbIhr0Yle5n4Md1Gk45YxDbVOyjxkIS335\\\\\\\/rUY9JSan2HLGymYP2BiXjnPwza5JXoM6FYlOLS+FNyiomoE3SBzdPq0HgE7IAoIjcKGi9RgtdEhLD+ml4lxeJ\\\\\\\/tOrjYteD3N4YncpMHxqBy3FU4qcMJi+\\\\\\\/ngfy3TS8quE99LicodaMlDThhpxdCZC+UowbSZkhdoUpO8VhZFO8QB5k8q3JAA+YKWp9GBXqiOXLZlAz+jNJp8dsGydwlMDT7tR7HVwQvzlDXZyZyKha5dB4JMds0LzdDhw7C3J2TekMfkfwyRccDJZ969jZdCPzfFcx+1js0IqZnIcEQCPKGPm2CRFZM0dfXuS09YZjZec6GYPblCyXO\\\\\\\/90c3kvl+HE\\\\\\\/7B1V\\\\\\\/Kc7Ja0K5bBcqqYkc6+9Ur1GnIjAqigVPms7dvNHce73IAnjXeUWT0T0qUoUgEKdRQLTRgybWQXQcXINmqHXTnx6T3ikkIoQPcVWqOkqp9aLFPngZjSTM9VnJLk8w4w7l1p60aP0lEGhfJLX\\\\\\\/63+zJQT1Ans3O4xbzgogsjlcU6osN\\\\\\\/g3ixuc3oa0QvXzqSToyjw1+Jfr+fVC6yVYJn2nZeP0IHljiDB7bnJ0b3UjehgLtw5sN6q\\\\\\\/QJmV48v6Un1HN4jrK\\\\\\\/gBs1r5tpM0b5xgE81aYDKATpsi2Oa+d08luW9VPTWRq6YMbxbHgbxBxyCdH0KAYpF57KfQLFVZExO0vcU0cKNQ1+\\\\\\\/mw9LGP1jAR9LMfFZ77eFkxyjFVbImE859hhrVJMvnsZRhdbXbnezdzLwAqjgj0YupseqlecOnMpNVT4OLldWxg48jE8NGq\\\\\\\/v3xCUgkZDvhAa9R990SsAd554Gu+UzYs+CYj7UKw7O1I14ZH5Ah2T1zCUhUqupVm+i0C4A+O6pCgKkPbatT8uMno5Pc4cu2OQhmr8cth2YnzdYaDZrh7NICHvLGiMw1xdajUhYHKcXkWUndJtghVbqyMiPVhKG5H7qJjNrP7MRWV\\\\\\\/Yuo07zINlU+ndVjGPquuLMxfy56P4Xann47NZxXgSimwMql3hmc1g6DxJ6ysukbVZPIn66pl+bHdc6HC+5\\\\\\\/jB5p4FKTLljA8u1oG6JvWKJwNbjuManu6\\\\\\\/YmlKFNhM7gebJGRU4NFEefsX1z+J6C3ufcQBbirmTC3JmT1X5Q7modyYSN7bDfAE2CGAY+TrOwN1BbsizvE5MN0OhyZ6jSAZ2tj6dSNgWp081GQ8AwemZhETtVL9PGuE9rMvdwk5UP2B3cp2Ujsh3bccM9cYmUtrqcl9X60GQG9V5uZrzLgc6n4qz3Ua\\\\\\\/GNFpl1eLP6axh3P9ebyndXXArHb\\\\\\\/Yr65mney7WJYuuxlvXBJvUQIYRCJpHjPzwgPCHTWOwx7bJ07Mbhqsa3dr+rQbdYcTU7+GnZV+D57PuXX9ooJ65\\\\\\\/WVnJXNYdfQN7VKBC7LbcE+rwPs7Lo+wa49jA2yWioXoYGnIE5v0oF0MKUYZNXVR7i18yZXQdhWRC5WMcZ\\\\\\\/dr2FzJMHFEQGIABVKaFT1Y75Dw0ccDNXInT0tusqR9749ortNpOyjMf7+ckesV1uS6MhLLwGE191qeyjAJItKj\\\\\\\/cvo8pjWISBwmrvjUezmS273ryEovsB20RWH8IymCH+VcTUN3F\\\\\\\/rg4eSTSrKZaulx33gpGp6xQuwiPuP8Oi8khDgSKHFnOdFH+ZQS7YRVOOGCDLy0o0GzUUft+tqZ5Bb9SB9bDEdUmP58mdaGLo3RRKdXIMw8469foi8Nr0vB1hhsPXnTp1myFpYUT9DIQZkhvmcIM436pJtyZdTZ8Fnn629Q39wgxEh8GLH8HROsFSN6v6KMOPcntIerKGgOa5z7stGiDbSdvGlsUwyp2VC4f5gpL0XoXLNbNlHeXVOaSq9sJMSQgvtupqENwMaqEBTcOTP87OqLMKn\\\\\\\/z5pjkq+CW3qGAdiua\\\\\\\/S+jDXO2FLOFtzKSg31UkotFKlEiypG7cuTcROUe1H+c1tYpefXb+4utdsenSWpYGA\\\\\\\/Qkr\\\\\\\/mqHOQeN4uQQcTAbZBU4FLSx4UIYL5Bwkb4Zt0mJDdYwM3P98LotzT3CsnreQFjcnVmITFBh+F1HmdCyxq4j49m+0KtLSnGdU8QFBfmgYoTRVYNiAeiCrXz6ObhOSex3rARyjKAMJXpEiTUhaC4cHl3m\\\\\\\/qZXjbkoWwgxhNekR\\\\\\\/ixqp5xeLniCTa0dNN6idJ1lEPqmQacfwndiRDNDBtHaR8Pnyji0Nugv8Lg14Mh3n3scj6Dc7rFfMjVyb+SZXrudY50laG7ct2yqyHjztwZIOBdTfzhljX+CGIJlqdC84nSiCxZ5r6wclzW43P7Fl4w1+V1zDKvAP1iumJGZGzJmUdETjxLMvu5HOl7jnHO613tI5Y5pbOFdB2QEpQbN9oknBH7yeJ7luVbSEdm9kkJFp6DMRHJrdGMrSCMg10YKprM13aWtccBG4eIEUcsYVPxUO7nW7YmKU2HGCyghS4Dey2m3wvdeOzpnOGyQ4ZqeelrrxfbXpW04XtiPwW90KiXsnICw1zRAGqQvXrI522yVz9ltNhT31U2zD+39evY0umrIuQtzFrChMcILgvd\\\\\\\/Jg4IcDPQohF+Fx3+RCFyn5UA8MnJFHdHE7Czufk54C1JHlorh0vgamI2qCJOyfcSjME+0YDd5o3T7iFBHRL0mjYDh8pgvZp5tHHjxbavqLbnkgTpYFd13qK\\\\\\\/vQo6bp0N\\\\\\\/43UoGhCIUwE9eWI+zkmhTL\\\\\\\/petmg0AYn6R9st\\\\\\\/l1QHMwu14b\\\\\\\/copFc0zgh7NgW+O2PaQMhygt4M248qCrXWLYDYlLyL0GopT\\\\\\\/AUQIz69jW\\\\\\\/4CE\\\\\\\/wutA8atCOaCOQdtUypscUfx12eNAPDISLWN2vbFIx\\\\\\\/JhthfSlloKt0YKrevh8tS2M6045uYvwETgKqYlOZnoaIWknGxCdGl6KkIMIHVb3RyA+gcgedZkqJn8KRrNoWUz\\\\\\\/ggc518XBkmhjsNSQ8Ov\\\\\\\/BUz\\\\\\\/ndYoG9mVHt9LYvSF9WwZPWj+yRaYxpHdk6PrR6rC2ghNVbqqxXutRhgOqoOHant9aZ\\\\\\\/Ldrwo8JoLI1HxmU+x9dbfMNLEFARbjN9HyZaomM6m7XN6g6zeliugpHAz58UtS2015acs0RosWaiPIvv3kZwVnp14tHH52U2FG06157N6QLmv9HsiRYaM452KzElJOIJo9JiBuBnWnbPWeRbPgOFDZbIK66WZHtIz+pFcWHmoh64HcgxvP70ibgZY2e5GIKusgQQZH5CE28GPN0ixOQDgCvJExlor\\\\\\\/QopjXAbuFjzhYcDYSbUXGqPGB5FKt9\\\\\\\/49H8hbXBziGofrqeUU5X+gAurw3XnICkO+OLvHgrHtnyliaYVSq7hzkuHQATIigV2pXx3ohrAjvBIsPY4n7m8VNsdMm2VxM9dQpkkCkizEpKnJf01pwpgWAI1wlyAAA8FM2d\\\\\\\/WbWZ+gHdBkBqUcMkQDdckJ1yXob0C\\\\\\\/hqN+\\\\\\\/OKeXAyg8IibFcr50eF3BjOTRTS9fZARveRG4a8jipfgZPcXY+A8jhGMEcYQILc6a5ry\\\\\\\/hwNgLVFrCspD8duLcAbtvebpond56TdcrEIq0HecTW\\\\\\\/+Aa8NqaLf1u5DwxfAhDrB8fP+6BkzSQ7nJ5XQysRvIKgayRbwXq4eJy1etKzT370vMHBQpptIYFYb1nVrAqec\\\\\\\/BCwFhD4TsT74LhNAMZnq7sdYXXpnbTufD2YkbZGixBLC\\\\\\\/4\\\\\\\/0W06WyTOukLS9895QR6NnyII7I+uoJkrEUDfo49HtEedg59JSJP7wC5RXTR2bX4CuA7kYyJxLKjN7KjLr2zxEWeQUyXvaMbD9fUnmf6BGClfmxiefEwgikeO4CQIo\\\\\\\/pdNKHu4vbK0h4fDWZ6aHaMGtns\\\\\\\/3KDmU9HLpOsEvlUxrF8n49gmeVMJ71UusIAeLAeVwEw9mkzOfGR+lMUkOgPGt+S9AnBMARmh1ygeYEHvPmk5ilp0abePAAZoj72bEvZl5hByVOc5S6PMIq1eZY1Sxhq5wzETT5jIdKrv75K6ELiLHaSGhUK6tabgCa3921oszskYsvYmTY1Ng2PirnejjIM+BF11bQseJN1HVEog6aYJL8GoS+eF6QOuZFKQ3CokZEkLCZUwT2kL1RgTc\\\\\\\/j\\\\\\\/yft1o23hwBH\\\\\\\/puiwUhFxzyv5SmqbuFRKWGs1RcMTY8+lCflJghHgzx7tqlk5zPGfhRMP5eyOSOmAUc8HDJ1p\\\\\\\/oCCKZMtnP3J9bRtKWoVlxr26tcyIyBP91j\\\\\\\/OaZ9cd8t2WmSd34FX\\\\\\\/XR2ycHn6s6eivg1Oumg4uRBveK9hzkkc3ZJEiw61o+694grXmmd4ifbDrU4vF0aSr4Dcu6prH2HWeLMUpSn\\\\\\\/jA+PcMTV\\\\\\\/BtSo\\\\\\\/VtGCTPXa6+Ra\\\\\\\/hRumQvgZ92PWMn4Jm61D5vfE1l9nyHSELRVBpF+wLYl4jeLA4aIteOnN3A6oqbYfhHoKnozh8vVGJjUqqQMObhJc5OgZfBN4D5RjKOCXGTQ8j\\\\\\\/LhWluV3sQQs+H0Hbbd91R3liZTib8nN6Wo8Us943rKWU0ehFCZZgenGFrc8wMtK0DkGkvyjDM1XBsw8UaHNg1t2WN9azdWtK8CVCPNeL4yn5EaY0AGA4Y2IkFuILnNhePERTcOpuTj41wGQuj90uV\\\\\\\/A1\\\\\\\/lSd1ABporAbxWdMx8Qngpk0fMbQecLYbYx4HxhPnJ\\\\\\\/XmqKigY2w5cAg9\\\\\\\/CpmdagBgvz3bV2bsPQ8MmVRs3rjSTeC\\\\\\\/5C5AQZ2HiQYXYIZhpO5cFvj2k5RQIQfMBzuL3DrzvGDXijKGemRFjx22IUyakoI0ds1\\\\\\\/6vDYciN5KOgJ\\\\\\\/vFJlanifi1NzB4WbcW4Njmq4r\\\\\\\/Pjw5eClKSEufelLV3PxxLG\\\\\\\/FcNlfzWkZV6wKIvqQQttRTjpf0FBRI1nvHO2jfp0ASDeMIiqO4a3BB6SFC4uFCD0xDW4XKNcAE3hmRsX9Zx2vJwP774dIcH+zta+yLmn+7P9WR2qGpZMHtVTDQ2iA0GXdnIyPuqOUSmhEeHstc9QO4tXTN0trSGC+b6307KlSJPhqIOuym5wFr8QpFEJ4CkD3BQWmDRxprycYMXt\\\\\\\/ldmuOuKkFAkHWXmV6N1XjhlVuH9IMJUZ0RuyuyO4c5e8w3O+FUyErCFq0dXUPPF3Fed2qQYmC2hTOSJm40eBvvUqu3j6tBUeQ9dgSt6V2oiRCxCzysthCOADRJy5dvOo4IxnV3amJENCXb\\\\\\\/NYdJ3Mipybvthz37IAd0wRg\\\\\\\/ED96MgkQe6yXwzDuGqhfvDB3r02qCzhka6b2TR66XMSbvfAd2T47p9bj1B6g4o71Qb4OUkS3hIqcTLRJJ6StLGy3LLzE7LTA0LyHAm6B5RMMv\\\\\\\/EbY20iwwZuc5gnOUHdzOQ3QeWCrwD8o0SQsqlOfZjBAbbxKh+DposmFzz8vKaMTdj7uiMrslRYBnODWLLlNGj8a7OOHJa2laqKQirQdRwD+Lv\\\\\\\/uFZT7\\\\\\\/bsSzzBDtO+NmUTe9iJF0lKopmPGpZl+8ym+fojsymDnAutSXgh2CXNEw3agXAXk+sZdbEJN0ZaMTmYkhU4Y1jngzA0CP\\\\\\\/641EkJUJAKIGEwMjWrPaTChKyI4zeAY0lxYeeqzbwou1w3bogoZCbZyWdchBV+Xt\\\\\\\/KVeuf8byAys45KIuQSNuVOBf5YNK\\\\\\\/PxD3I0t7PEFIMzLZysTaynU7CKEMTANJjBmJAPPaJna4VxxXjUqXEu+7kE41QfsI1xs+1WnjaQ+C3h2YPoAKr+jcO2JDsi5ZVpOYuo14zHBKnU85NJASIYXVFL95M58shabPxHVFRCOpY46Om4SbFcsF8L41lhnjrRge9Iv15tUWTeE5V1WvMCcxxxal0FLCVPGs82dDyXlXvyFgixQvLkGW6Ywtivyl7ztt3uD6Ce6s8n19qUhYXhed6pGqiAg4HyhwUl9NTJuXBCD\\\\\\\/0e3Ki3ML5rXbSxBxc8eDt8mYzeHCvbebjQBx32zHAE+Ft6Amn4YsnZQXFlIpwgVX\\\\\\\/RCZcWcaU+jO\\\\\\\/jEUMDNYcMPT7FExebeOw+ebBa+AGFHhpLMxZDEHgMWYNm49rqzOJe97lMDDoBzieN04Bf5ZGnw4JmQypcSKCQbPqfxQv8sDyIX5u8TE23AG0mJHIeB\\\\\\\/68tybzSp6A3Axsf\\\\\\\/pADoxHwiHKCqtjmKxi9lmQZGGM1RLtGCReSiFrrO\\\\\\\/diwvVsY+ePRB6hr7XqbduFLZd0DuovzSlxLlcaqjsE4vNbe0R4+JkUPh7hZ+rNPBK9cggX4iXCrmUm6KFQG5IQNFWyahyAtczObh9VsxU0nd2Zpelj+ZoksxNHrmm43EPjfOhT9du0xtlag\\\\\\\/f4nwnBHCBWM5wbqOSnO7pKDGWMtPp\\\\\\\/bCmHnDi\\\\\\\/RkmjDKl2JQ7X66WBqCskotCqVIyhwjfVE+S9K4DBUwVUPN6txsqKAuZd\\\\\\\/8eUue\\\\\\\/DbQ6MuNrouabd5w+bHJnEzmUdb3UKaOEySFN0Rai9dEbhJcEdSh43Bi40o8Gcug0A1HtgYNB+evGiZFUosHzABtV8iXWYtUwauQm7ooyrJabalDopCBn5crJ7K+cA1lRwpSmFZfMfZcCGfjSr2XjfZk\\\\\\\/tzVSBo3b\\\\\\\/ljySCLyF2go6jpwEjgQoEQL2MAiPJQuMvpAz6iJ8Tm1s0B0yxlBIaqIDpu5Nf4HaWs46Y5thA8o\\\\\\\/7shVv6WuuS7mbXvu2oVFf5fxvmUyLJHclAFGNrjg5cdLZlMXwHveXMWYaF8vMxU8ojuOpn6iHa\\\\\\\/GPXgDbqMqTkBw1Ot1cwOed\\\\\\\/VgDgiSENIlNVt8zjhrA8AU22gntuePzRTYDRw\\\\\\\/74bcxknK\\\\\\\/QCt1exVxXTqkNirnaxupycshEwgOZU6X7FzoVjo4Bd0GCQ4YdfON86I6xQsQ8sTMFXmoqheSvqeBmR6GlY+nGSA1pVKPexydY6IcQBFYAByFJuv57686s8Ds+hJJtzNRHy5vOYvhzN9bTEMsYYrBT8Cj7CPRSZ4CZk4JiwwYnbFFc57QCRQEIr97LjY2R8duSfhNvxhuSvZ13oKADAH4X9Yq8zqXofQDwk5AhNJsQ9JgMvq4smcZ\\\\\\\/daZpCgX6HACZbgvKf0uL+vm+bRmIez6kDZ7shsvMcvhCYhUDaI5gWlRdQyAG98k+f8U0PIGx0ChVGUI6DmmIIL\\\\\\\/uCTFNPRZ\\\\\\\/VYnB\\\\\\\/7BIsloUkoaNelEujEdhCT9vay4g0rNco1E8NOOXZL+WMzrmX8mXThwzQuigP\\\\\\\/AaQWsAdztVpIXU7p+Ca+o+zWGGH8APQGZjbTnD8mFWrlu8W\\\\\\\/ks8+7JJlTOIqZzmLBmAF+kYgOLXNlqxfBwMTxbIbkjeXOQNk40pGxKX\\\\\\\/01zCJI\\\\\\\/k\\\\\\\/CXbnbUhKlY1JU+UqCEy77nOcNpKMbhaBPVaEhXFIqE1tYVa3L\\\\\\\/O6vE7tD1BTYjMbSoBErDgTAOMnkIkTAk5FQLhvaFFquAdrmsqyTyXBvf+P40yClQ3kpz4VCmO8KrCjXBx8CIxNp6Qq\\\\\\\/b1YB+0C3BvOuzQaeELxK30xof5yinhok4FShZOyFzMIG9eHapMKPE7VNvlPeVFVv0NaEfmlNWhTzpM0px9WvdVtnfncxKDxNDnoDVrJ0uBr0OZ\\\\\\\/ocpvNlxPtKCjBOz8le6C83TeAokEiLeeOZB\\\\\\\/BwPYI5d5RKPNzC6hjb7XvpN8qfMsDF6vsXba8wg9I1pIHVbFc5Wjrwi49pOE24basngit7Miz7kbst9DC9tNrfakhaI7YS9rw1HfYliXPL+SRUq8Ho8XKFBnIFYCp4vW8UUS83+ZU4oxzbfuCqQnV4LOucc2r2sIIwCjX4Jn7FdgnZo38cNSQLv70Zy0dFy6kiOfrHIpDi+ojzEGI7zlNMgoe8N4z5uyULHvCwuWKAGB6dW9bsA01Rd1paOlCBEmQVLnIynh\\\\\\\/AHoZ4T+nvSUEpldX+JX5JC2RWounyfTfgqE3JmopGSkxCs0KZ6VShSG1RQWUHaydSnI+s6w4ildCAeeZkRrdGeXS3kgRUUG4cnUh430R\\\\\\\/ZzqesOJjUl1LWwE6vIvGXsLaR1AU2GZYzYhiQCyVYC0LBVa7Injkmq7f69s+X4w7t4NB6v7y54J2NNLWvKPvGyCqNZ4qMunsE5JnoadNv\\\\\\\/lE\\\\\\\/q\\\\\\\/MPYtrmjm9qWOkf\\\\\\\/gcu762Nak6OwwepCsmR+\\\\\\\/E9CGr1pJpfLnCxHc4L5HHEndODstTjNajeWlhdgC4SWAaNVLu7+\\\\\\\/BK3GNzr2N0AplSSPUtnWp\\\\\\\/dt5Fuji70bOXzgE5KsSSTnZOxtA2zdYmwLg1Mosk8MIzX7w1\\\\\\\/OESVeFE5fl65IU2cmhvKrOUozhibKzFt5jQLCQ34Es9e08C\\\\\\\/ItMvRxB4ANx\\\\\\\/puQsxeaGP2d3D5+dULvQ5jThuRMpCIS6GQn4rnH3Oc6fMrEtNEAnVX+8Q4uDRBJoegJq54tPHYBNvgAm18mPtjSLo5DSWlP9yXtADwyeHSPFNVYavSJHwXtAAxPjCGMnSD1IwTgOolc32\\\\\\\/n8yYYT+Ch7koFULVrrxH4jw\\\\\\\/HF2maUF7IHQDwxbbM90Vhh\\\\\\\/wvl1k6LoywojSQQueGjGLr2JS1jkG6CxeDtL7rN03EARCY3\\\\\\\/G6k3SBM5CakCMFe2i9L7wxLbSJdjcSzoz4OgUEJs5opaNUsByX3dPnux7rg72Ya\\\\\\\/x4zSm0aszGyrU0veLVkX2sTESGQDTZ6rJw+48BRUBB\\\\\\\/7716Z7Is3JFyFTtFhbXk7ILl6eW4DclWTe4ztBj0b5EdOcjwkddyvRoPxCAORPR7Gw89aSOQKYHQpj9S4kplb+O94seYhpwGnXCqSAdLdzEG+eutqFjK1ddJ4dqBWTpjH7M6v1udKz0iKE31ZyPTP8RMZKrmUwj\\\\\\\/rQ2T6FYyqelFnYI1+afLge9Lq3WbDfDq\\\\\\\/\\\\\\\/AtVp5WB9yzDuyZ04oIP5cAigiu3C9Dk1oOYvmFejeKo4dYBgoXYx2hqxB5ues9eDV0GEwQS1kV3MEzmBJq+j865x46XkZyekQE9iIFPQHm1fCQsUjFH2L3+nOjuy+J5CUgrYkYspEqlQ5QJOSIvRwYT8QZnhrU\\\\\\\/fgmaOZu8CD7qqokQZpJkOQZxUxJJw4fwq8+YV26DGQTpvO\\\\\\\/0i0nHe4OgkUP3BJ95lg1Mz0vJcBTtzS2FXuWS1WQz+G5a+4378A0\\\\\\\/MiAAtgQPlxdgVvgWqFPt7uAHTudt7wQdqtqGx3zfdPBM+8eswFU\\\\\\\/zNYtokjbdv6xSYKS8hMdpXmeDwuwAyiccfUM\\\\\\\/mNYcF4k+YriCMaVGurRNE5\\\\\\\/0gn3TdKKs79V9ZyyWIVgDOqqwfNexv98u8yTWKxOVQ8rJWel\\\\\\\/MFf0ugErG9S1uxFjoqoOCHajFX\\\\\\\/8PM5z6e77OBuw95Ek3aoa9609ye78vjOQnGOuJjpYuWMQaN2p5SvTuDBBR6UZYUEKIH56110mvr+BJkBcIBKYG4+wvOcJcme8EV+JCp9RL\\\\\\\/AWmEJWpMsfgjfrOmS9OsAocHgunQM3QKZqyydhCiwblTTNJqQCvH6tD8ampWcRteok6WXNwqKDwA7KArwby7mf5taPKo2fJcPtFcV0F\\\\\\\/q2onT7P2zOUenc3yzRSFHWME1EQyVpk+HQqNDYkKK\\\\\\\/8GjGXEopDt9In0likI1THTJNEonYBy8PWZqJc4HBJGgEtBbjTKtAYeIDE8xCU1Qgu2fGNMt+4vn2xb6A4FHqn06mHh4apiQVlx2SDR+GA4nuxk8oQvFu+QtvOhipRbz183kzgpmqXvYW95d\\\\\\\/u4HbOM36VDk9fSzPRMtKYTFHSg7H5NrbZ5ZBnZLFrhCM7NJ2Yy5+6bJST\\\\\\\/bNFpuAFm8HGy2jSYLtgaIktbrttDZTZkYQNEBaWAvVEMRJoUttQKwb\\\\\\\/fGSDmQgETzhx9+iiz4L6fGmx34uiZtq6d2hBxFlDQfuhOSD8KyA1SvGPYqtQhTKp60jSMfFmpmRT2D6zv+OqfMUMNaLnKYW7aqJqgH7PGXzKCDiPW7EDUh1kOrDSllg4slmpzKuAxbbtV8KgTRD2e9+24kgtD+wQAHpG2lajK5DpnAgE7QnSL3Mnu9LzYonlP49bOsRfecEBhmlnuJV+VQBlxhHDtIiWzQl5ULxjqZIwwqWxVIkWShpeiz7QP\\\\\\\/wkHzWJuZGFeOrJHRC8FguwU\\\\\\\/y9KygvvcIndgUESwBPJYJ1ThUkXyUGQJcAl2ZU4p335ekZ\\\\\\\/Nxy282Vh\\\\\\\/io91y+MxlCOm5qRmCwrgCjnlBZiusinADw8njiIHTfx5nyDZI2HvKWPetyDWwuB5DYtelc6D9xlSdNfLGUlw9p7HPOIOC3Lw6nrQRROtgW5mjVoN+XCNSeQ7NomBZSdM\\\\\\\/VPYVJZbo1vHJoBGJEZv0\\\\\\\/\\\\\\\/mxUutJreLuUG9AcBaq5EZ+UvjAatUH4htxgK1dXwsoCKRBYeO6C0EPZMmMjfU\\\\\\\/4IynUrzr+kZOBcX6QtdsnZ4GkmdAsVjcPvEr5amo\\\\\\\/v\\\\\\\/GJgKeLHGlfJCNGfepoKS5Eluw5Cok12\\\\\\\/r79nBlN0i6iZdMR2XawE4hadSlFdbsJ9Xm1yeXmh4HaDhk5lIGHeGPrEz0c2iN5pD5c4y\\\\\\\/vACPB0LaufDpOf8BYCUppm2UUtO\\\\\\\/uMnx5zdt1hEYI7GUdVi132kL2Au89Tn7\\\\\\\/+9Oy4dQp+L\\\\\\\/hilJzPEdk7LVwQJmiLG23abmLhh7\\\\\\\/Tvlc5w+IN3\\\\\\\/gdTfFXV5lfMZjba9ZseGIxCfvVGrgRwy0MmKRcgLSQeu6LX9spMnws\\\\\\\/4FKl\\\\\\\/MtBf9dI08qssOky1lfoFow3Yg3Mng3Aq4s+Yle5y7WP0JlioZAh8zJ3r20I7dGvsmIZU0LrLPtA7Fy7w4Fd13Us\\\\\\\/4h2VdbrXipiGTTm8YXM1TTm68AEvcw4kCx0jbBLfuY9Yy7J23zZ1MGJLeES9kWiay0evmJyP2tpA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022AAwC9xpEJFipRgmn7aa4OQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227be2f71243f82ae2eceacc544355940923f2372714fe6ec2616e4ef1eab21b7a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XSjw42HvP7\\\\\\\/VU4HD8cKcVcAcW39DkvGDWKZ8KM4MAuOyCSyFuUKBnOuJ+b1TfrzZ+W6RsQSIsxC0YT\\\\\\\/81KExmRXCJy4BGbFnjWUJmXRRSWOBbcXuM2YXTuQtYzboRN+QR74jIDMUs3unrKa\\\\\\\/1Pv1ti9xSgeIN2i40jXOkU5aPg\\\\\\\/W3tyDDRBb73eDMtkntbs7eUSQ2pN3ILEmb6\\\\\\\/GcC0IhiNzmOqsU2sn09tnYc6PmnysfYucUBQJLzurr3ijcLUXNJkn2JtbDgDyGEUiMijhGr5n1TA6DtzztMVbiVsaBafZ5zmor8wRVqrBfSnqCIVab\\\\\\\/z152nXgs7g04lraj+AGwysT1NRjWSOm12\\\\\\\/+EpDYTG+NS8VET7LZmw2jvOqZ4EQ\\\\\\\/JxZZtIL7vl6UemS2d0kiRLwUwHvo2IhoVeyuVLQv9lbY85+u6l9WcEYE17k3k33vNYirDoD+KJt\\\\\\\/4Forqa8LFlUBcKLYrKHEJ67hlScipZ9floI\\\\\\\/hqVhvO\\\\\\\/Mck5JCBFmZV7k3NbzE2wgq9BoVJCj\\\\\\\/qii04VZMCfykByGB6OXHVq88T73rg5LnBb4Aq4kkTbeJhcJN9vUBGQFTNDUc5o4seGpVFFIgaO+e6ylk1qUCdrxUW6bmTIV65\\\\\\\/YzwilqfX3hmmS18ikvsIT7KOnV\\\\\\\/PjTCDdkeSFHbNrp5U1LT5V0MPCsjw+iPlfocDvhQAW\\\\\\\/DsPcfOPv8u2V+x9DEaAgyeh4a4bKqnmvWrBbSICuqy7vAF6nOEVxhwZJgURppQlbUY74SJ5AQCE8lR7EBWDe1wD\\\\\\\/PSmhRiZ5Hg79V5AJDu\\\\\\\/HLVYABuAYM3JelWDiFQCF0vYgQnYZy4+4OBoDkmIzJYa0RJjj1FWr6dMgRVJkXXoZOUwk2OHdMte8briAEgLp8+01nbyggWl\\\\\\\/dJlYlJa41\\\\\\\/Sa6jfZZSn99pfklIqgqdXksaC91ivqu5Mw1P2J5b+Slpx9fpydJhEt5nxAwGi9YRu1Fe+uwBqir9EcnMzmgSb5LLpMcvc7gVsll5jGAB0tV8KillTgDlrM5YYREvCge2jRTeM7NZimzFFGY7BHSajGEXhBWLldZ0wdngRpH+oaQn6ME6hx0P\\\\\\\/HZYTSXlq0HB1Awi2DUOSVLqNbsyhX6pOxy7VeiduSRbFlZ2Am5N+h+BWA9p7lQXmAZjG\\\\\\\/s+gd5HgHizww0bUQEn0WhZo5r6kLyEeourir65w795ppJA9WWFUToEiSK6TPkkGIb08kKnODG3wRnj359KoyI0JxAHP0gg4PKgAe25cTvcM16c02hAslUQqSmdaB\\\\\\\/bKekBisKeIpuuhQVhjKx6eACEMes2xoqCvaWJ+QWPP1dj8KWnsd9jYYbfjMOnew0B7KJrOVoBvQcwUz3rSapakLnxo24aJgFwHEmL4lhLbcGK5Yx4mQ+07uInS9tUWgupJjCj7NqT2ND+KY4r5cwbzrcB5VzqGnm3rZCRS2dGWv17ytyo38xF5D7Wh5UexTCBdfyJN1+DXSUeHQHDuuznNR+DS26NFacCaVLj53GOfqFswqKOryTc4+5eq5EAu\\\\\\\/c5vyB7Aw2st+PgIgn6fhiMBfjq4KBiOLl9olwDHPcb0LHzNGzVshIFaLGD598yh82WbqN\\\\\\\/R312FEX6wPOUBAwhyKWVfGwTwjfo8\\\\\\\/LUpGC5RXLWqxi9rzCHe\\\\\\\/T3kPuHSSyabqVM2Szl7eUDDy4ua73OlxKItBPYj4QQXfHdgFvQc2QXvmoSjXp3R8hYzqIfyvmJyXSKA3u0qnfC9F7JD3RkloTdiz+N2iThsmxC8pTOnrz5kC1I2KP4UxJLnxs6LC2CbqjgSH06u84RQJwAOL7C5IAc7kysN6YTkcVB+i3RvvvPOfTQ0rgM0Zgb3hl7QPGHBbuwpmL4BZnLVsjxI69fR8ZinrMTFzUE7iWxZjPnDK7HFkQSL4m4FNbtj+zbD+jMFcdWSfua3asHNxGSyjWX36cXG3GkUaTdnmXxt4ictX5jEXXamOyZZWCxFNR9\\\\\\\/+73Rsqbxz9IfQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FT6wKTdUup0iJeIXUGVQLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022294b4731e2ecf8a6852a90c1eddb059339a84b0ef3286fc48fe9ad876660766f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495168 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/333" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WEXEYmUGcYUR1XNAZVe0Rzv8XxFoB4Bzg5M3lZ04uOs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/333" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oFuWV3BsURTruAUNwUNUacfrZEth8fDO3GYbdSwbJ5g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/321" title="A Novel cis Element Achieves the Same Solution as an Ancestral cis Element During Thiamine Starvation in Candida glabrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/357" title="Nonsynonymous Mutations in Linker-2 of the Pdr5 Multidrug Transporter Identify a New RNA Stability Element" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495168" id="node495168" data-pisa="ggg;10/1/333" data-pisa-master="ggg;g3.119.400724" data-apath="/ggg/10/1/333.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in <em>Caenorhabditis</em> Endoderm Specification</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6257-7612" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Morris F. Maduro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">333-356;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400724 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495168" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Morris F. Maduro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular, Cell and Systems Biology Department, University of California, Riverside, CA 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morris%2BF.%2BMaduro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maduro%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMorris%2BF.%2BMaduro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6257-7612" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Morris F. Maduro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mmaduro@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">mmaduro@ucr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/333" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495168" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495168/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/333.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495168" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495168/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/333.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495168" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495168/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/333.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495168" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495168/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Gene regulatory networks and their evolution are important in the study of animal development. In the nematode, <em>Caenorhabditis elegans</em>, the endoderm (gut) is generated from a single embryonic precursor, E. Gut is specified by the maternal factor <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>, which activates the MED → <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 → <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a>,7 cascade of GATA transcription factors. In this work, genome sequences from over two dozen species within the <em>Caenorhabditis</em> genus are used to identify MED and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 orthologs. Predictions are validated by comparison of gene structure, protein conservation, and putative <em>cis</em>-regulatory sites. All three factors occur together, but only within the Elegans supergroup, suggesting they originated at its base. The MED factors are the most diverse and exhibit an unexpectedly extensive gene amplification. In contrast, the highly conserved <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs are unique in nearly all species and share extended regions of conservation. The <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 proteins share a region upstream of their zinc finger and an unusual amino-terminal poly-serine domain exhibiting high codon bias. Compared with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> proteins are otherwise less conserved as a group and are typically found as paralogous duplicates. Hence, all three factors are under different evolutionary constraints. Promoter comparisons identify motifs that suggest the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>, MED, and END factors function in a similar gut specification network across the Elegans supergroup that has been conserved for tens of millions of years. A model is proposed to account for the rapid origin of this essential kernel in the gut specification network, by the upstream intercalation of duplicate genes into a simpler ancestral network.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gata-factors" class="hw-term hw-article-keyword hw-article-keyword-gata-factors" rel="nofollow">GATA factors</a></li><li class="kwd"><a href="/keyword/cell-fate-specification" class="hw-term hw-article-keyword hw-article-keyword-cell-fate-specification" rel="nofollow">cell fate specification</a></li><li class="kwd"><a href="/keyword/gene-regulatory-network" class="hw-term hw-article-keyword hw-article-keyword-gene-regulatory-network" rel="nofollow">gene regulatory network</a></li><li class="kwd"><a href="/keyword/developmental-system-drift" class="hw-term hw-article-keyword hw-article-keyword-developmental-system-drift" rel="nofollow">developmental system drift</a></li><li class="kwd"><a href="/keyword/caenorhabditis" class="hw-term hw-article-keyword hw-article-keyword-caenorhabditis" rel="nofollow">Caenorhabditis</a></li></ul><p id="p-3">Central to the development of a metazoan is the activation of tissue-specific gene regulatory networks (GRNs) that drive subdivision of progenitors and emergence of features of terminal differentiation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Davidson 2010</a>). On evolutionary time scales, changes in such networks drive appearance of novel features, but these changes can also occur without changes in morphology or development (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Peter and Davidson 2016</a>). Such differences in GRNs that nonetheless drive homologous developmental processes exemplify Developmental System Drift (DSD) (<a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">True and Haag 2001</a>). In the nematode genus <em>Caenorhabditis</em>, which includes the well-studied species <em>C. elegans</em>, examples of DSD include the gene networks that produce the derived character of hermaphroditism, which evolved at least three independent times in the genus, and vulval development (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Haag <em>et al.</em> 2018</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Félix 2007</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ellis and Lin 2014</a>).</p><p id="p-4">A relatively understudied area in <em>Caenorhabditis</em> is the evolutionary dynamics of GRNs that drive embryonic development. One reason may be that the close relatives to <em>C. elegans</em> exhibit indistinguishable embryogenesis, differing perhaps by the timing of some developmental milestones (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Memar <em>et al.</em> 2019</a>; <a id="xref-ref-128-1" class="xref-bibr" href="#ref-128">Zhao <em>et al.</em> 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Levin <em>et al.</em> 2012</a>). Another reason for the paucity of evo-devo studies in embryogenesis is that the dissection of a GRN requires cause-and-effect associations to be probed through experimental perturbations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Davidson <em>et al.</em> 2002</a>). The powerful tools of forward and reverse genetics in <em>C. elegans</em> have only recently become available in related species, most notably <em>C. briggsae</em>, which like <em>C. elegans</em> is hermaphroditic and supports RNA-mediated interference (<a id="xref-ref-129-1" class="xref-bibr" href="#ref-129">Zhao <em>et al.</em> 2010</a>). A third, and more important limitation, is that very few embryonic GRNs are known at high resolution in <em>C. elegans</em> that could serve as a comparison.</p><p id="p-5">The gene regulatory network that specifies the <em>C. elegans</em> endoderm is an example of a set of interacting transcription factors that has been studied in great detail (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Maduro 2017</a>). In the early embryo, the founder cells E and MS are born (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The E cell generates the entire endoderm (intestine), while its sister cell MS generates many mesodermal cell types, including the part of the pharynx, and many body muscle cells (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Sulston <em>et al.</em> 1983</a>). Many components of the GRN underlying MS and E development are known with high precision, and in most of cases, regulatory inputs have been confirmed to be direct and <em>cis</em>-regulatory sites have even been identified in upstream regions (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Maduro <em>et al.</em> 2001</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Broitman-Maduro <em>et al.</em> 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>; <a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">Wiesenfahrt <em>et al.</em> 2015</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Du <em>et al.</em> 2016</a>). This network is therefore a highly suitable system in which to examine questions of GRN evolution and developmental system drift.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Embryonic origin of the E blastomere and simplified diagram of the gene regulatory network for endomesoderm specification in C. elegans. (A) The E cell and its sister cell MS are found ventrally in the 8-cell embryo (approximately 50 &#x3BC;m long). MS generates mesodermal cells including body muscles and the posterior portion of the pharynx, shown in red on the diagram of the larva (approximately 200 &#x3BC;m long). E generates the 20 cells of the intestine, whose nuclei are shown in green on the larva. (B) Specification of MS and E fates begins with the same SKN-1 and MED-1,2 factors, but then bifurcates into an MS pathway that includes the T-box factor TBX-35 and the homeobox factor CEH-51, while endoderm specification involves activation of END-3 and END-1. These upstream transient factors ultimately activate ELT-2 (and its paralogue ELT-7) which maintain intestinal fate. Additional input into E specification occurs by input from TCF/POP-1 and Caudal/PAL-1. All of MED-1,2, END-1,3 and ELT-2,7 are GATA type transcription factors. Arrows indicate transcriptional activation of the gene encoding a downstream factor." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Embryonic origin of the E blastomere and simplified diagram of the gene regulatory network for endomesoderm specification in &lt;em&gt;C. elegans&lt;/em&gt;. (A) The E cell and its sister cell MS are found ventrally in the 8-cell embryo (approximately 50 &#x3BC;m long). MS generates mesodermal cells including body muscles and the posterior portion of the pharynx, shown in red on the diagram of the larva (approximately 200 &#x3BC;m long). E generates the 20 cells of the intestine, whose nuclei are shown in green on the larva. (B) Specification of MS and E fates begins with the same SKN-1 and MED-1,2 factors, but then bifurcates into an MS pathway that includes the T-box factor TBX-35 and the homeobox factor CEH-51, while endoderm specification involves activation of END-3 and END-1. These upstream transient factors ultimately activate ELT-2 (and its paralogue ELT-7) which maintain intestinal fate. Additional input into E specification occurs by input from TCF/POP-1 and Caudal/PAL-1. All of MED-1,2, END-1,3 and ELT-2,7 are GATA type transcription factors. Arrows indicate transcriptional activation of the gene encoding a downstream factor.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F1.medium.gif" width="440" height="152"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/333/F1.medium.gif" width="440" height="152"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495175" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">Embryonic origin of the E blastomere and simplified diagram of the gene regulatory network for endomesoderm specification in <em>C. elegans</em>. (A) The E cell and its sister cell MS are found ventrally in the 8-cell embryo (approximately 50 μm long). MS generates mesodermal cells including body muscles and the posterior portion of the pharynx, shown in red on the diagram of the larva (approximately 200 μm long). E generates the 20 cells of the intestine, whose nuclei are shown in green on the larva. (B) Specification of MS and E fates begins with the same <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>,2 factors, but then bifurcates into an MS pathway that includes the T-box factor <a href="https://identifiers.org/bioentitylink/WB:WBGene00006554?doi=10.1534/g3.119.400724">TBX-35</a> and the homeobox factor CEH-51, while endoderm specification involves activation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>. These upstream transient factors ultimately activate <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> (and its paralogue <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724">ELT-7</a>) which maintain intestinal fate. Additional input into E specification occurs by input from TCF/<a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> and Caudal/<a href="https://identifiers.org/bioentitylink/WB:WBGene00003912?doi=10.1534/g3.119.400724">PAL-1</a>. All of <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>,2, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a>,7 are GATA type transcription factors. Arrows indicate transcriptional activation of the gene encoding a downstream factor.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">The endomesoderm specification network works as follows. A simplified diagram is shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>. Specification of both MS and E begins with accumulation of maternal <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> protein. <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> is an unusual transcription factor that binds DNA as a monomer through a Skn domain consisting of a homeodomain-like amino half recognizing an A/T-rich sequence, and a bZIP-like carboxyl basic domain recognizing a TCAT sequence (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Pal <em>et al.</em> 1997</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Carroll <em>et al.</em> 1997</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blackwell <em>et al.</em> 1994</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Lo <em>et al.</em> 1998</a>). <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> directly activates expression of <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00003181?doi=10.1534/g3.119.400724"><em>med-2</em></a>, which encode nearly identical divergent GATA-type transcription factors that recognize an atypical AGTATAC core site (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Lowry <em>et al.</em> 2009</a>). <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>,2 are important for specification of both MS and E, as loss of activity of these genes results in a penetrant failure to specify MS, and an incompletely penetrant failure to specify E (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bowerman <em>et al.</em> 1992</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Maduro <em>et al.</em> 2001</a>). In MS, the MEDs specify mesodermal fate in part through activation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00006554?doi=10.1534/g3.119.400724"><em>tbx-35</em></a> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Broitman-Maduro <em>et al.</em> 2006</a>). In E, <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>,2 contribute to activation of the paralogous <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes. These encode similar GATA factors that are expressed in the early E lineage, with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> being activated slightly earlier than <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Maduro <em>et al.</em> 2002</a>; <a id="xref-ref-131-1" class="xref-bibr" href="#ref-131">Zhu <em>et al.</em> 1997</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baugh <em>et al.</em> 2003</a>). In turn, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> proteins activate <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>, a GATA factor that sets and maintains, through positive autoregulation, the fate of intestinal cells and is the central regulator for all intestinal genes (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">McGhee <em>et al.</em> 2009</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Fukushige <em>et al.</em> 1998</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Fukushige <em>et al.</em> 1999</a>). The <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> gene encodes a similar GATA factor that shares function and expression with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>, but which itself is not essential for normal development (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Sommermann <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Dineen <em>et al.</em> 2018</a>). All of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724">ELT-7</a> have similar DNA-binding properties and interact with canonical GATA binding sites of the type HGATAR (<a id="xref-ref-122-2" class="xref-bibr" href="#ref-122">Wiesenfahrt <em>et al.</em> 2015</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Du <em>et al.</em> 2016</a>). Many additional studies have revealed unexpected nuance and complexity to the myriad of factors in this network, confirming that the sum of upstream inputs into <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> activation is not merely additive. Upstream factors have distinguishable roles in establishment of robust cell divisions, gut morphogenesis and activation of genes important for metabolic function of the intestine (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Dineen <em>et al.</em> 2018</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Boeck <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2017</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Sawyer <em>et al.</em> 2011</a>).</p><p id="p-8">Integrated with the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> → <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>,2 → <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 feed-forward regulatory chain is the Wnt/β-catenin asymmetry pathway, which acts in the asymmetric MS <em>vs.</em> E fate decision through the nuclear effector TCF/<a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Lin <em>et al.</em> 1995</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Maduro <em>et al.</em> 2002</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Owraghi <em>et al.</em> 2010</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Rocheleau <em>et al.</em> 1997</a>; <a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Shetty <em>et al.</em> 2005</a>; <a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Thorpe <em>et al.</em> 1997</a>). In MS, <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> represses gut fate by preventing activation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>, while in E, <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> is an activator that contributes to activation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> through its association with a divergent β-catenin, <a href="https://identifiers.org/bioentitylink/WB:WBGene00006379?doi=10.1534/g3.119.400724">SYS-1</a> (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Maduro <em>et al.</em> 2005b</a>; <a id="xref-ref-103-2" class="xref-bibr" href="#ref-103">Shetty <em>et al.</em> 2005</a>). The <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> contribution to gut specification is not the major regulatory input, however, because loss of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724"><em>pop-1</em></a> still results in endoderm specification from E (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Lin <em>et al.</em> 1995</a>). The contribution of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> is detectable when depletion of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724"><em>pop-1</em></a> is combined with loss of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724"><em>skn-1</em></a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a><em>,2</em> (together) or <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>, which produces loss of gut specification in a majority of embryos (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Maduro <em>et al.</em> 2005b</a>; <a id="xref-ref-103-3" class="xref-bibr" href="#ref-103">Shetty <em>et al.</em> 2005</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Maduro <em>et al.</em> 2007</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>; <a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Owraghi <em>et al.</em> 2010</a>). An additional minor input into gut specification in <em>C. elegans</em> is through maternally provided <a href="https://identifiers.org/bioentitylink/WB:WBGene00003912?doi=10.1534/g3.119.400724">PAL-1</a> protein, a Caudal-like factor whose primary role is specification of a different blastomere called C (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Hunter and Kenyon 1996</a>; <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Maduro <em>et al.</em> 2005b</a>).</p><p id="p-9">A small number of studies have investigated the evolutionary dynamics of gut specification in species closely related to <em>C. elegans</em>. In <em>C. briggsae</em>, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs (the latter of which is found as two nearby paralogues, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a><em>.1</em> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a><em>.2</em>) are expressed in the early E lineage, and simultaneous knockdown of <em>C. briggsae </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a><em>.1</em> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a><em>.2</em> by RNAi results in a failure to specify gut (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Lin <em>et al.</em> 2009</a>; <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). In <em>C. briggsae</em> and <em>C. remanei</em>, most orthologs of the <em>med</em> genes, when introduced individually as high-copy transgenes, can fully complement the embryonic lethality of <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a><em>,2(-)</em> embryos (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>). Together these studies suggest that the <em>med</em> and <em>end</em> factors play similar roles in all three species, as might be expected. Somewhat unexpectedly, however, knockdown of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724"><em>skn-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724"><em>pop-1</em></a> orthologs in <em>C. briggsae</em> was found to produce different phenotypes from <em>C. elegans</em>, suggesting that the way that <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> interact with their downstream target genes is subject to evolutionary changes even among very closely related species, <em>i.e.</em>, the hallmark of developmental system drift (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Lin <em>et al.</em> 2009</a>; <a id="xref-ref-129-2" class="xref-bibr" href="#ref-129">Zhao <em>et al.</em> 2010</a>). From these few studies, then, a model emerges of a core endoderm specification pathway, where some regulatory inputs into the pathway are subject to more rapid evolutionary change than others.</p><p id="p-10">An important way that properties of a GRN can be studied on an evolutionary scale is to examine features of orthologous genes in related species (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Peter and Davidson 2016</a>). However, given the essential requirement for the gut specification network in <em>C. elegans</em>, a paradox became apparent when genome sequences outside of the genus were completed: No <em>med</em> or <em>end</em> orthologs could be identified in the related nematode <em>Pristionchus pacificus</em>, while putative orthologs of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724"><em>skn-1</em></a> can be found in <em>Pristionchus</em> and in even more divergent species (data not shown) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dieterich <em>et al.</em> 2008</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Schiffer <em>et al.</em> 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Couthier <em>et al.</em> 2004</a>). In recent years, however, the number of known species within the <em>Caenorhabditis</em> genus has grown considerably, opening possibilities for studying evolution of development through sequence comparisons (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Kiontke <em>et al.</em> 2011</a>). In the past two years, new sequence assemblies have become available for over two dozen <em>Caenorhabditis</em> genomes both within and outside of the so-called “Elegans supergroup” of species that are most closely related to <em>C. elegans</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Félix <em>et al.</em> 2014</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> 2019</a>). Collectively, this powerful set of sequences captures tens of millions of years of genome evolution (<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Stein <em>et al.</em> 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cutter 2008</a>).</p><p id="p-11">In this work, I have used a primarily <em>in silico</em> approach to identify orthologs of the <em>med</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes among the <em>Caenorhabditis</em> genome sequence assemblies (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Haag and Thomas 2015</a>). Patterns of conservation of gene structure, protein structure and putative <em>cis</em>-regulatory sites are revealed in the <em>med</em> and <em>end</em> genes that confirm known information from <em>C. elegans</em> and reveal new insights into the MED and END proteins and the evolutionary dynamics of the network. The results complement studies that identify genome-wide conserved putative <em>cis</em>-regulatory motifs among close relatives of <em>C. elegans</em> (<a id="xref-ref-127-1" class="xref-bibr" href="#ref-127">Zhao <em>et al.</em> 2012</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Siepel <em>et al.</em> 2005</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Grishkevich <em>et al.</em> 2011</a>). A surprising finding is that the endoderm network likely originated at the base of the Elegans supergroup, in a manner that can be hypothesized to have resulted from the rapid serial intercalation of successive duplications of an ancestral GATA factor, likely <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>. Other unexpected findings are that the MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> proteins are evolving at different rates, and that <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> contains previously unrecognized, highly conserved domains that distinguish it from <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>. The resulting suite of MED/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors from 20 species forms a starting point for future studies on GRN evolution in <em>Caenorhabditis</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identification of putative med and end orthologs</h3><p id="p-12">Sequence scaffolds and predicted proteins were downloaded from the <em>Caenorhabditis</em> Genomes Project (CGP) website (<a href="http://download.caenorhabditis.org">http://download.caenorhabditis.org</a>) in late 2017. Searches were performed using the NCBI Windows 64-bit BLAST 2.7.1+ executable (<a href="ftp://ftp.ncbi.nlm.nih.gov/blast/executables/LATEST/">ftp://ftp.ncbi.nlm.nih.gov/blast/executables/LATEST/</a>) on a 64-bit Core i7 PC running Microsoft Windows 10, complemented by searching on both the CGP site and WormBase (<a href="http://wormbase.org">http://wormbase.org</a>). FASTA files containing sequence scaffolds, and others containing protein predictions, were searched by TBLASTN and BLASTP respectively using the protein sequences of <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>. The updated <em>C. elegans</em> VC2010 sequence was also searched to confirm the <em>med</em> and <em>end</em> genes (<a id="xref-ref-126-1" class="xref-bibr" href="#ref-126">Yoshimura <em>et al.</em> 2019</a>).</p><p id="p-13">Putative orthologous genes were identified using recommended best practices (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Haag and Thomas 2015</a>). Genes were first predicted by matching high-scoring segment pairs from TBLASTN results with genomic sequence, predicting the gene structure by identifying consensus intron splice donor and acceptor sequences, and comparing with the predicted genes from the assembly projects (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Spieth <em>et al.</em> 2014</a>; <a id="xref-ref-110-2" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> 2019</a>). Identification of gene structure started with the coding region for the DNA-binding domains and progressed both upstream and downstream. As analysis progressed, conserved features of the <em>med</em> and <em>end</em> genes and their gene products, within and among closely related species, became apparent, and these were used to refine the gene predictions. Searching of representative orthologs from each species back to the <em>C. elegans</em> genome confirmed that the predictions were the best matches. In some cases, the gene predictions from the assembly projects included short (&lt;50 bp) predicted introns that could also be read through as coding. For these, a case-by-case judgment was made as to whether to include such introns in favor of maximizing amino-acid level homology. Some of the predictions within less-conserved regions could be incorrect, but these would not be expected to dramatically affect the analysis presented here. Similar judgments were made when multiple in-frame start codons were possible at the 5′ end of a gene, or when open reading frames could be extended in the 3′ direction by splicing around a stop codon. While no molecular validation of predicted genes was made, the manual curation of gene predictions favoring maximal similarity of gene and protein structures provides a surrogate validation by conservation across related species. This is the approach taken computationally for gene predictions by algorithms such as TWINSCAN (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Korf <em>et al.</em> 2001</a>).</p><p id="p-14">It is highly likely that the gene set described here includes artifactual duplicates, particularly among the MEDs. The quality and coverage of the genome assemblies, as well as the maintenance of heterozygosity in sequenced strains, are known to produce artifactual paralogues that are really alleles of one locus (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Haag and Thomas 2015</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barriere <em>et al.</em> 2009</a>). Some of these may still have been included as orthologs because they corresponded to a predicted gene from the sequence assembly. For example, the two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes in <em>C. brenneri</em> are nearly identical with one found on a small sequence scaffold, suggesting that there is only one <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> ortholog in this species. The inclusion of such nearly identical duplicates is not expected to affect inter-species comparisons, for which a representative single gene/protein was chosen. Gene models categorized as pseudogenes were more straightforward to find because they were truncated, had in-frame stop codons or frame shifts in the DNA-binding domain, or were missing essential amino acids such as one of the four cysteines in the C4 zinc finger. These may be expressed genes but were deemed unlikely to result in a functional protein.</p><p id="p-15">Comparison of my protein predictions to those of the various sequence projects validated the approach used to identify <em>med</em> and <em>end</em> orthologs. Of the genes identified and deemed not to be pseudogenes, 54% (94/174) were identical to a predicted coding sequence (CDS) from the assemblies, 32% (56/174) partially overlapped an existing CDS, and 14% (24/174) did not correspond to a predicted CDS. Differences from assembly project predictions often resulted from missing carboxyl and/or amino ends because of large introns, or extensions of open reading frames that maximized ORF length only. Completely missed predictions tended to be of the small intronless <em>med</em> genes that are often missed by gene-finding algorithms. Data from cDNA sequences were generally not found to be useful, likely because the transient expression of the <em>med</em> and <em>end</em> factors in the earliest stages of embryogenesis means that <em>med</em> and <em>end</em> RNAs are generally absent from mixed-stage cDNA preparations.</p><p id="p-16">Predicted genes/proteins have been provisionally named <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a>.n/<a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>.n, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>.n/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>.n, and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>.n/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>.n (where n = 1, 2, 3, etc.). Lower numbers correspond roughly to the rank order of identified high-scoring segment pairs from the TBLASTN search, which favors both stronger similarity with the <em>C. elegans</em> search sequence and scaffolds that contain multiple hits. Where a single ortholog was found in a species, it was named as <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a>/<a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> or <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>. For analyses where a single representative of a set of paralogues was used, it was the first numbered one, except for pseudogenes or one of the apparent two-fingered MEDs, in which case the next paralogue was used.</p></div><div id="sec-3" class="subsection"><h3>Identification of conserved regulatory motifs</h3><p id="p-17">A representative set of promoters, one per Elegans supergroup species per factor, was compiled to identify putative <em>cis</em>-regulatory motifs. This was done to reduce artifacts arising from overrepresentation of sets of very similar promoters resulting from intraspecific paralogs, which tended to have very similar promoters (data not shown). To identify sites starting with known binding sites, a JavaScript program was written to count occurrence of sites and compute <em>p</em> values assuming a Poisson distribution, following the approach used in a prior work (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>). To identify motifs <em>ab initio</em> by their conservation, MEME (<a href="http://meme-suite.org/tools/meme">http://meme-suite.org/tools/meme</a>) was used with expected site distribution with any number of repetitions (anr), the number of motifs to be identified as 10, and a maximum motif width of 12. Alternative parameters generally retrieved the same highly represented sites, except that motifs with higher E-values (and hence less conserved) could be different. Searches of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> promoters as separate groups produced qualitatively similar results as those that used both together, except that MED-like sites became rare enough among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes that they were not reported as significant by MEME. I did not consider sites whose E-values were greater than 1e-02 as these occurred among a small number of <em>med</em> and/or <em>end</em> genes. Some of these may represent less-conserved regulatory motifs, although they were not recognized as belonging to known factors from <em>C. elegans</em>. The site locations and promoter sequences are in Supplemental File S1.</p></div><div id="sec-4" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-18">Alignments and simple Maximum-Likelihood trees were performed using MUSCLE as implemented in MEGA-X (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Kumar <em>et al.</em> 2018</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Edgar 2004</a>). The tree for the DNA-binding domains was produced using RAxML as implemented in the RAxML-NG web service (<a href="https://raxml-ng.vital-it.ch">https://raxml-ng.vital-it.ch</a>) with default parameters, except that the BLOSUM62 substitution matrix was used and bootstrapping was activated (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Kozlov <em>et al.</em> 2019</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Stamatakis 2014</a>). I note that construction of trees using the proteins described here results in disagreements with the more robust trees of <a id="xref-ref-110-3" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> (2019)</a>, with only closely related species retaining the same relationship, such as the interfertile species <em>C. briggsae</em> and <em>C. nigoni</em> (<a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">Woodruff <em>et al.</em> 2010</a>). This is what would be expected from rapidly evolving genes. Consistent with this, calculations of synonymous and non-synonymous substitutions rates did not produce interpretable information because of the high rates of molecular evolution in <em>Caenorhabditis</em> in general (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Cutter 2008</a>). Moreover, the fastest rates of evolution in <em>Caenorhabditis</em> occur in early zygotic regulators with transient expression, which accurately describes the MED and END factors (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cutter <em>et al.</em> 2019</a>). Because fast-evolving proteins are being compared among 20 species (as opposed to only two or three), the major conclusions regarding conserved amino acids and stringency of selection are nonetheless self-evident from the alignments and topology of phylogenetic trees.</p></div><div id="sec-5" class="subsection"><h3>Additional software</h3><p id="p-19">Gene modeling, sequence alignments and other analyses were performed with Vector NTI 6 and the MEGA-X software package (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Kumar <em>et al.</em> 2018</a>). Generation of tables and drawing of to-scale diagrams in SVG format were aided by custom programs written by the author in JavaScript and Python. These scripts are available by request. Protein alignments were annotated using BoxShade (<a href="https://embnet.vital-it.ch/software/BOX_form.html">https://embnet.vital-it.ch/software/BOX_form.html</a>) to generate EPS-formatted files. Data were compiled in Microsoft Excel and figures were assembled in Adobe Illustrator.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-20">Sequences identified in this work are available as Supplemental files. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9820622">https://doi.org/10.25387/g3.9820622</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Med, end-3 and end-1 are found together in the elegans supergroup</h3><p id="p-21">I searched sequence scaffolds from 27 species of the <em>Caenorhabditis</em> Genomes Project (<a href="http://caenorhabditis.org">http://caenorhabditis.org</a>) with TBLASTN using the protein sequences of <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>. <em>C. elegans</em>, <em>C. briggsae</em> and <em>C. remanei</em> were included as their sequences have been updated since earlier reports on <em>med</em> and <em>end</em> genes from these (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>; <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>; <a id="xref-ref-126-2" class="xref-bibr" href="#ref-126">Yoshimura <em>et al.</em> 2019</a>). As shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, at least one ortholog of each of the three genes was found in 20 species comprising the Elegans supergroup, a clade that includes the Japonica and Elegans groups (<a id="xref-ref-110-4" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> 2019</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Kiontke <em>et al.</em> 2011</a>). Consistent with the absence of even more distant MED or END orthologs, the number of putative GATA factors in the genomes of species outside the Elegans supergroup was smaller, typically 5 or fewer, and putative orthologs were better matched to other <em>C. elegans</em> GATA factors like <a href="https://identifiers.org/bioentitylink/WB:WBGene00001251?doi=10.1534/g3.119.400724">ELT-3</a> (data not shown). Across the 20 species searched in the Elegans supergroup, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> orthologs were unique in each genome except for <em>C. brenneri</em> (which may have two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes), while multiple paralogs within a species was the norm for the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs with an average of 2.0 copies per genome, and the <em>med</em> orthologs, found an average of 5.6 copies. The high average copy number of the <em>med</em> orthologs is driven by the 20 or more genes found in <em>C. doughertyi</em> and <em>C. brenneri</em>. Excluding these two species, the average number of <em>med</em> genes is 3.7 copies per genome. Of 208 genes identified for all three factors, 34 were deemed to be the result of unresolved heterozygosity or were likely pseudogenes (counted together under “pseudo” in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>); these were eliminated from further study. It is still likely that some falsely identified <em>med</em> paralogues persist in the predicted gene set; hence, occurrence of nearly identical paralogues should be interpreted with caution (see Materials and Methods). In any event, the identification of false duplicates would not change the results of inter-species comparisons, for which a single representative gene was chosen for each factor. I note that because many comparisons were done with a single representative ortholog for each factor per species, it is possible that some species-specific evolutionary novelty will be missed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Orthologs of the MED, END-3 and END-1 factors among species whose sequences were searched. Species are shown after the most recent phylogeny (Stevens et al. 2019) with the Japonica group in light blue and the Elegans group in pink. The species C. parvicauda, C. castelli, C. quiockensis, and C. virilis, which contain no orthologs of the MED and END factors, have been omitted for simplicity. Table cells are colored by the number of orthologs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Orthologs of the MED, END-3 and END-1 factors among species whose sequences were searched. Species are shown after the most recent phylogeny (Stevens &lt;em&gt;et al.&lt;/em&gt; 2019) with the Japonica group in light blue and the Elegans group in pink. The species &lt;em&gt;C. parvicauda&lt;/em&gt;, &lt;em&gt;C. castelli&lt;/em&gt;, &lt;em&gt;C. quiockensis&lt;/em&gt;, and &lt;em&gt;C. virilis&lt;/em&gt;, which contain no orthologs of the MED and END factors, have been omitted for simplicity. Table cells are colored by the number of orthologs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F2.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/333/F2.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495304" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Orthologs of the MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors among species whose sequences were searched. Species are shown after the most recent phylogeny (<a id="xref-ref-110-5" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> 2019</a>) with the Japonica group in light blue and the Elegans group in pink. The species <em>C. parvicauda</em>, <em>C. castelli</em>, <em>C. quiockensis</em>, and <em>C. virilis</em>, which contain no orthologs of the MED and END factors, have been omitted for simplicity. Table cells are colored by the number of orthologs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Conserved linkage of end-1 and end-3 orthologs</h3><p id="p-23">In <em>C. elegans</em> and <em>C. briggsae</em> the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes are within ∼30 kbp of each other (<a id="xref-ref-70-5" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). Microsynteny of this type has been observed in other genes of these two species (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Kent and Zahler 2000</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Coghlan and Wolfe 2002</a>). To see if microsynteny of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> is common, I examined whether <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs in other species may be linked. As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, in 12/18 of the remaining Elegans supergroup species, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> are found on the same scaffold with an average separation of ∼37 kbp and a range of 20-63 kbp. In <em>C. brenneri</em>, which has two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and five <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs, one scaffold carries both an <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and an <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>, however the distance between them is ∼530 kbp. In the remaining five species, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes are found on different scaffolds. Because it is possible for sequence scaffolds to break between two linked genes, there may be additional synteny among these. For example, in <em>C. sinica</em> the scaffold containing the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> ortholog is 32 kbp in size with the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> gene located 3 kbp from one end, raising the possibility that although its <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> ortholog is on a different scaffold, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> may be nearby in the genome. Closely related species have similar patterns of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> synteny, for example between <em>C. afra</em> and <em>C. sulstoni</em>, and between <em>C. zanzibari</em> and <em>C. tribulationis</em> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). Although synteny is conserved, the relative orientation of linked <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> paralogues varies, with examples of all four possible linked arrangements. In <em>C. elegans</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> are encoded on the same strand with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> upstream of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>. In <em>C. sulstoni</em>, two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> paralogs are upstream of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> with all three genes on the same strand. In <em>C. zanzibari</em> and <em>C. tribulationis</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> is on one strand in between two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> paralogs on the other strand, hence in one <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a><em>/3</em> pair the genes point toward each other, and in the other they are divergently transcribed. These differing arrangements are consistent with the high rate of intrachromosomal rearrangements previously noted for <em>Caenorhabditis</em> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Coghlan and Wolfe 2002</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synteny and relative orientation among med and end genes found on sequence scaffolds. Except where noted by a number, inter-gene distances are shown relative to the scale bar at the top of each panel. (A) Patterns of microsynteny among end-1 (dark blue) and end-3 (light blue) orthologs among the Elegans supergroup species. (B) Patterns of microsynteny among med orthologs for a subset of species in the Elegans supergroup." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Synteny and relative orientation among &lt;em&gt;med&lt;/em&gt; and &lt;em&gt;end&lt;/em&gt; genes found on sequence scaffolds. Except where noted by a number, inter-gene distances are shown relative to the scale bar at the top of each panel. (A) Patterns of microsynteny among &lt;em&gt;end-1&lt;/em&gt; (dark blue) and &lt;em&gt;end-3&lt;/em&gt; (light blue) orthologs among the Elegans supergroup species. (B) Patterns of microsynteny among &lt;em&gt;med&lt;/em&gt; orthologs for a subset of species in the Elegans supergroup.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F3.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/333/F3.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495373" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Synteny and relative orientation among <em>med</em> and <em>end</em> genes found on sequence scaffolds. Except where noted by a number, inter-gene distances are shown relative to the scale bar at the top of each panel. (A) Patterns of microsynteny among <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> (dark blue) and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> (light blue) orthologs among the Elegans supergroup species. (B) Patterns of microsynteny among <em>med</em> orthologs for a subset of species in the Elegans supergroup.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Prevalance of linked med and linked end-3 duplications</h3><p id="p-25">In <em>C. briggsae</em>, two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> paralogues are found in an inverted orientation within several kbp, and in <em>C. remanei</em>, two clusters of closely linked <em>med</em> paralogues are found (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>; <a id="xref-ref-70-6" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). Similar linked duplications of these genes are found in other species. Among the <em>end</em> genes shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, 7/10 species with at least two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes show two of them within 10 kbp. Among the 18 species with at least two <em>med</em> genes, linked pairs can be found in nine of them, in which at least two <em>med</em> genes occur within 5 kbp of each other. Examples of linked <em>med</em> duplications are shown for four of the Elegans supergroup species in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>. In the most extreme case, 9/25 <em>C. brenneri med</em> orthologs are clustered across a 23-kbp region, with an additional tandem pair located ∼22 kbp away. Linked duplications are therefore a common occurrence, particularly for the <em>med</em> genes.</p></div><div id="sec-11" class="subsection"><h3>Absence of a conserved intron in the Elegans group</h3><p id="p-26">I next examined the evolutionary changes in <em>med</em> and <em>end</em> gene structures across the Elegans supergroup. For simplicity, a single representative <em>med</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> gene was used for each species because intraspecific paralogs generally showed identical splicing patterns. The gene structures are shown in scale diagrams in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, depicting intron/exon structures arranged by the phylogeny of <a id="xref-ref-110-6" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> (2019)</a>. Intron positions are also indicated on diagrams of the predicted proteins in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>. Of particular significance, prior work found that the <em>med</em> genes of <em>C. elegans</em>, <em>C. briggsae</em>, and <em>C. remanei</em> have no introns, unlike all other GATA factors in these species including the <em>end</em> genes (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Gillis <em>et al.</em> 2008</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Maduro <em>et al.</em> 2001</a>). As shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, while all representative <em>med</em> genes are found to be intronless across the Elegans group, the <em>med</em>s from the Japonica group share a common intron (indicated by an asterisk) within the C4 zinc finger coding region that is found in the same position in all <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes. In addition to this conserved intron, within the Japonica group, the <em>C. japonica</em> and <em>C. panamensis med</em> genes each have one more upstream intron at non-homologous positions.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="med and end gene structures and conserved promoter motifs. (A) Gene structures. 600bp of promoter are shown as a line, and the coding DNA sequence (CDS) predictions are shown relative to the scale bar at the top. Boxes are exons, and spaces joined by a &#x2019;V&#x2019; are introns. Bent arrows indicate the location of the predicted start codon. An asterisk denotes the intron conserved among all end genes and Japonica group med genes. (B) Motifs identified by MEME for the med and end-1,3 genes. The motifs are symbolized by a colored circle on the promoters in (A). Some of the motifs are shown in their reverse complement from the MEME output files in Supplemental Files S13 and S14." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;med&lt;/em&gt; and &lt;em&gt;end&lt;/em&gt; gene structures and conserved promoter motifs. (A) Gene structures. 600bp of promoter are shown as a line, and the coding DNA sequence (CDS) predictions are shown relative to the scale bar at the top. Boxes are exons, and spaces joined by a &#x2019;V&#x2019; are introns. Bent arrows indicate the location of the predicted start codon. An asterisk denotes the intron conserved among all &lt;em&gt;end&lt;/em&gt; genes and Japonica group &lt;em&gt;med&lt;/em&gt; genes. (B) Motifs identified by MEME for the &lt;em&gt;med&lt;/em&gt; and &lt;em&gt;end-1&lt;/em&gt;&lt;em&gt;,3&lt;/em&gt; genes. The motifs are symbolized by a colored circle on the promoters in (A). Some of the motifs are shown in their reverse complement from the MEME output files in Supplemental Files S13 and S14.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F4.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/333/F4.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495226" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child"><em>med</em> and <em>end</em> gene structures and conserved promoter motifs. (A) Gene structures. 600bp of promoter are shown as a line, and the coding DNA sequence (CDS) predictions are shown relative to the scale bar at the top. Boxes are exons, and spaces joined by a ’V’ are introns. Bent arrows indicate the location of the predicted start codon. An asterisk denotes the intron conserved among all <em>end</em> genes and Japonica group <em>med</em> genes. (B) Motifs identified by MEME for the <em>med</em> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a><em>,3</em> genes. The motifs are symbolized by a colored circle on the promoters in (A). Some of the motifs are shown in their reverse complement from the MEME output files in Supplemental Files S13 and S14.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Differences in introns among end-3 and end-1 genes</h3><p id="p-28">The conserved intron that interrupts the zinc finger is the only one shared between the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). As a group, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs show the highest variability in the number of introns, with <em>C. tropicalis</em> having only the one conserved intron, <em>C. becei</em> having four introns total, and the remaining species having two or three. The <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> orthologs are far less diverse, sharing the same four exons with three introns, except for <em>C. brenneri</em> which is missing the second intron. In terms of size, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> introns tend to be smaller overall, with introns larger than 100 bp most apparent within the Elegans group <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes. Hence, the positions of introns in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> orthologs appear to be under a greater constraint than those of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes.</p></div><div id="sec-13" class="subsection"><h3>Identification of conserved promoter motifs</h3><p id="p-29">The occurrence of <em>med</em> and <em>end</em> genes in 20 related species affords the opportunity to identify conserved <em>cis</em>-regulatory sites and infer conservation of the structure of the gut specification network. The expectation is that conserved regulatory inputs found in <em>C. elegans</em> should be reflected in the occurrence of similar <em>cis</em>-regulatory sites mediating the same promoter-DNA interactions in the other species. I first searched for known binding sites for <em>C. elegans</em> factors among the Elegans supergroup <em>med</em> and <em>end</em> orthologs using methods previously used in <em>C. elegans</em> (<a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>). A size of 600bp upstream of the ATG was chosen for these and subsequent analyses, as the known regulatory interactions with the <em>C. elegans med</em> and <em>end</em> genes generally occur within a few hundred base pairs of the ATG (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>; <a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Maduro <em>et al.</em> 2001</a>; <a id="xref-ref-103-4" class="xref-bibr" href="#ref-103">Shetty <em>et al.</em> 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bhambhani <em>et al.</em> 2014</a>). Among the <em>med</em> upstream regions, I found widespread conservation of only <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>-like sites, and among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs, only MED sites (Supplemental Material, Tables S1, S2 and S3). While these results support conservation of activation of <em>med</em> orthologs by a <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>-like factor, and activation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs by MED-like factors, a complementary (and superior) approach is to search for over-represented motifs <em>ab initio</em>. I therefore searched 600bp upstream of representative <em>med</em> and <em>end</em> genes from all 20 species using the MEME discovery algorithm (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bailey and Elkan 1994</a>). The results are summarized in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>, with the sites indicated by color-coded circles on the promoters in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>. The locations of the sites diagrammed in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a> are listed in Supplemental File S1.</p></div><div id="sec-14" class="subsection"><h3>SKN-1 binding sites in the med and end genes</h3><p id="p-30">Among the <em>med</em> orthologs, a motif resembling two overlapping <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites was identified in 19/20 species. The core of this motif, RTCATCAT, is found in two clusters in the <em>C. elegans med</em> genes and DNA fragments containing these sites are capable of binding recombinant <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> DNA-binding domain <em>in vitro</em> (<a id="xref-ref-76-5" class="xref-bibr" href="#ref-76">Maduro <em>et al.</em> 2001</a>). The same core is found in <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> binding sites in <a href="https://identifiers.org/bioentitylink/WB:WBGene00001527?doi=10.1534/g3.119.400724"><em>gcs-1</em></a>, a known <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> target gene in the fully developed intestine (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">An and Blackwell 2003</a>). As in <em>C. elegans</em>, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <em>med</em> genes are found within 300 bp of the predicted start site in most of the other species, which is apparent from the diagram in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>. In <em>C. panamensis</em>, which contains only a single putative <em>med</em> gene, an RTCATCAT site was not identified by MEME although six ’core’ RTCAT sites were found by direct searching (<em>P</em> ≤ 0.05, Poission distribution). The low E-value of 1.1e-102 and presence of an average of 3.5 sites per species strongly suggest that activation of <em>med</em> orthologous genes likely occurs by <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> in most Elegans supergroup species.</p><p id="p-31">Among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes, a TC<span class="underline">ATT</span>Y<span class="underline">TCATC</span> site was identified by MEME in 12/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes and 14/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes (E-value 2.9e-11). Most of this site (underlined) overlaps with 8/9 bases of the WWWRTCATC site for <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Etheve <em>et al.</em> 2016</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Mathelier <em>et al.</em> 2014</a>). Unlike the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <em>med</em> genes, which occur an average of 3.5 times per gene, these putative <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <em>end</em> genes, when present, occur only 1.5 times per <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> gene and 1.6 times per <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> gene. I hypothesize that this site represents a degenerate (low-affinity) <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> binding site. Prior evidence in <em>C. elegans</em> had suggested that <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> contributes directly to <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a><em>,3</em> activation independently of the MEDs, though the precise sites have not been reported (<a id="xref-ref-72-5" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>).</p></div><div id="sec-15" class="subsection"><h3>Sp1 binding sites</h3><p id="p-32">A motif resembling the binding site for Sp1 is found in the promoters of <em>med</em> (17/20 species, E-value of 2.0e-33), <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> (20/20 species), and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes (15/20 species), with an E-value of 4.8e-55 for the two <em>end</em> genes. This same motif has been found among many <em>C. elegans</em> promoters, suggesting that regulation by Sp1 is not restricted to gut specification (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Grishkevich <em>et al.</em> 2011</a>). Reduction of function of <a href="https://identifiers.org/bioentitylink/WB:WBGene00012735?doi=10.1534/g3.119.400724"><em>sptf-3</em></a>, a gene encoding an Sp1-like factor, causes a decrease in specification of E and a reduction in expression of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> reporters (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Sullivan-Brown <em>et al.</em> 2016</a>). From the widespread conservation of the Sp1 binding sites, it is likely that Sp1 contributes to E specification across many species in the Elegans supergroup through direct binding of the <em>med</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologous genes.</p></div><div id="sec-16" class="subsection"><h3>MED binding sites in the end-1 and end-3 genes</h3><p id="p-33">Prior work identified the binding sites for the MED factors in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes, defining a core sequence of AGTATAC that is distinct from the HGATAR site of canonical GATA factors (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Broitman-Maduro <em>et al.</em> 2006</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Lowry <em>et al.</em> 2009</a>). As anticipated by the results from searching for this site directly, MEME identified a highly conserved MED site motif in 9/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes and 20/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes (E-value 7.8e-53 across both <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>). Across the nine species with MED sites identified in <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>, there are an average of 1.2 sites per gene, while for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>, there are 2.6 sites on average. The location and spacing of the sites are consistent with results from <em>C. elegans</em>, with sites occurring within 200 bp of the predicted translation start site and showing a spacing (when multiple sites are present) of ∼50 bp (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>).</p></div><div id="sec-17" class="subsection"><h3>Polypyrimidine motif</h3><p id="p-34">MEME identified a pyrimidine-rich motif in 15/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes and 9/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes (E-value 2.5e-05). This motif, consisting primarily of C and T, is most apparent among the Japonica group <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes. The complement of the pyrimidine-rich motif is purine-rich, hence these motifs are called PPY/PPU (polypyrimidine/polypurine) tracts (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Sawicka <em>et al.</em> 2008</a>). This motif shows a strand bias by gene: 30/34 sites among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes have the polypyrimidines on the top strand, while the sites are evenly distributed on either strand (9/16 on the top strand) in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes. Polypyrimidine tracts are generally associated with messenger RNAs where they would be present as one strand, and interact with polypyrimidine-tract binding proteins (PTBs) (<a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Sawicka <em>et al.</em> 2008</a>). The human Pur-alpha protein (PURA) can bind to purine-rich motifs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bergemann <em>et al.</em> 1992</a>). A Pur-alpha-like protein in <em>C. elegans</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00004046?doi=10.1534/g3.119.400724">PLP-1</a>, was previously identified as having a regulatory input into <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> activation through a purine-rich site (<a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">Witze <em>et al.</em> 2009</a>). However, the PPY/PPU motif identified by MEME was not found in either of the <em>C. elegans end</em> genes.</p></div><div id="sec-18" class="subsection"><h3>Additional overrepresented motifs</h3><p id="p-35">Three additional sites were found by MEME among the <em>med</em> genes. A motif containing a TCTKCAC core is found in 9/20 species <em>med</em> genes with an average of 1.6 sites per gene (E-value 4.2e-08). The motif sequence does not immediately suggest a putative regulatory factor, although it tends to be found among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites, suggesting it is related to <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> binding. A motif containing TTTNNAAA was found at a higher E-value of 2.3e-04 in 10/20 <em>med</em> genes with an occurrence of 3.3 sites per gene, with one species <em>C. zanzibari</em>, containing 16 of them. This site resembles previously identified periodic AT clusters (PATCs) suggesting it may be a more general motif (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Frøkjær-Jensen <em>et al.</em> 2016</a>). A motif resembling a TATA-box was found in 13/20 species’ <em>med</em> genes with an even higher E-value of 1.3e-02 (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Grishkevich <em>et al.</em> 2011</a>). This may be a <em>bona fide</em> basal promoter site, as it is found within tens of base pairs from the translation start in these 13 genes. Finally, among the <em>end</em> genes, an “SL1 motif” was found in 12/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes and 11/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes (E-value 8.5e-04) (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Grishkevich <em>et al.</em> 2011</a>). The SL1 sequence is typically found at the 5′ end of genes whose transcripts become <em>trans</em>-spliced to the SL1 spliced leader sequence (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen <em>et al.</em> 2011</a>). The motif was not found in the <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a><em>/3</em> genes, consistent with prior work that neither of these genes in <em>C. elegans</em> is known to be <em>trans</em>-spliced (<a id="xref-ref-131-2" class="xref-bibr" href="#ref-131">Zhu <em>et al.</em> 1997</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Allen <em>et al.</em> 2011</a>). Its relevance as a motif is uncertain, as in most of the <em>end</em> promoters that contain it, the site is more than 300bp upstream of the predicted start site.</p></div><div id="sec-19" class="subsection"><h3>Phylogenetic analysis confirms that med, end-3 and end-1 form distinct clades</h3><p id="p-36">The gene structure and promoter motifs suggest that the <em>med</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes form distinct families among the 20 species of the Elegans supergroup. To confirm that this is reflected at the protein level, I aligned the DNA-binding domains (DBDs) among representative MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors (one per species) and used this to construct a phylogenetic tree <em>ab initio</em> with the RAxML-NG method (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Kozlov <em>et al.</em> 2019</a>; <a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Stamatakis 2014</a>). As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> form three broad clades, with the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors showing the highest similarity as a group, followed by the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> factors, and finally the more diverse MED factors. A high diversity of the MED factors was previously observed among the <em>med</em> genes from <em>C. elegans</em>, <em>C. briggsae</em> and <em>C. remanei</em> (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>). The grouping of the factors increases confidence that the correct orthologs have been assigned and shows that different rates of protein evolution have occurred among the three factors.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of representative MED, END-3 and END-1 DNA-binding domains. The DNA-binding domains of C. elegans ELT-2 and chicken GATA1 are shown as outgroups. Each of the three factors forms a distinct clade, with the END-1 factors showing the highest similarity, followed by END-3, then the MEDs as the most diverse group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic tree of representative MED, END-3 and END-1 DNA-binding domains. The DNA-binding domains of &lt;em&gt;C. elegans&lt;/em&gt; ELT-2 and chicken GATA1 are shown as outgroups. Each of the three factors forms a distinct clade, with the END-1 factors showing the highest similarity, followed by END-3, then the MEDs as the most diverse group.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F5.medium.gif" width="310" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/1/333/F5.medium.gif" width="310" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495199" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Phylogenetic tree of representative MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> DNA-binding domains. The DNA-binding domains of <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and chicken GATA1 are shown as outgroups. Each of the three factors forms a distinct clade, with the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors showing the highest similarity, followed by <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>, then the MEDs as the most diverse group.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Gene amplification within and among species</h3><p id="p-38">While <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> is represented by a unique ortholog among all species (except <em>C. brenneri</em> which may have two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes), <em>med</em> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs are often found as two or more duplicate genes within a species. The two <em>C. briggsae</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> paralogues are highly similar, suggesting recent duplication, and the multiple <em>med</em> genes among <em>C. elegans</em>, <em>C. briggsae</em> and <em>C. remanei</em> are also much more alike within each species (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>; <a id="xref-ref-70-7" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). To test how general this phenomenon is, I aligned and constructed trees for all MED DBDs, and separately, the END DBDs. In the tree of MED factors shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, most <em>med</em> duplications have occurred post-speciation from a small number of founding genes. The 20 MED factors in <em>C. doughertyi</em> cluster in a way that suggests there may have been only one or two ancestral <em>med</em> genes that underwent multiple rounds of amplification. In the case of <em>C. brenneri</em>, the MEDs form two clusters of 22 and 3 genes each, suggesting there were only a few ancestral factors. A similar division occurs among the <em>C. tropicalis</em> MEDs, which suggests two ancestral <em>med</em> genes. There are three groups in which paralogous MED factors are clustered within species pairs: <em>C. briggsae</em> with <em>C. nigoni</em>, <em>C. becei</em> with <em>C. nouraguensis</em>, and <em>C. latens</em> with <em>C. remanei</em>. Within each cluster, the pattern suggests that both species inherited two or three <em>med</em> paralogues from a common ancestor, which then each underwent further amplification post-speciation. Among the remaining 9 species that have 2-5 <em>med</em> genes each, the paralogous MEDs clustered together as a single group, suggesting a single ancestral gene. This unusually widespread pattern of duplications both pre- and post-speciation, not seen in the <em>end</em> genes, shows that the <em>med</em> genes are under different evolutionary constraints.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of all MED factors, showing high prevalence of duplications across the Elegans supergroup. In most cases, paralogous duplicates likely arose post-speciation, although there are examples that suggest that some species each inherited two or three genes from a common ancestor that later underwent further duplications. The tree was generated by RAxML using the MED DNA-binding domains (Kozlov et al. 2019; Stamatakis 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic tree of all MED factors, showing high prevalence of duplications across the Elegans supergroup. In most cases, paralogous duplicates likely arose post-speciation, although there are examples that suggest that some species each inherited two or three genes from a common ancestor that later underwent further duplications. The tree was generated by RAxML using the MED DNA-binding domains (Kozlov &lt;em&gt;et al.&lt;/em&gt; 2019; Stamatakis 2014).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F6.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/1/333/F6.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495292" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-39" class="first-child">Phylogenetic tree of all MED factors, showing high prevalence of duplications across the Elegans supergroup. In most cases, paralogous duplicates likely arose post-speciation, although there are examples that suggest that some species each inherited two or three genes from a common ancestor that later underwent further duplications. The tree was generated by RAxML using the MED DNA-binding domains (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Kozlov <em>et al.</em> 2019</a>; <a id="xref-ref-107-3" class="xref-bibr" href="#ref-107">Stamatakis 2014</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">I note here that six genes were found that encode MED-like factors with two C4 zinc fingers, indicated on the tree in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>. In each case, the two fingers are highly similar, so only one of the two fingers was used to generate the tree. Four of the “two-fingered” genes are present as two paralogous pairs in <em>C. nigoni</em>, one is found in <em>C. briggsae</em>, and another is found in <em>C. brenneri</em> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). <em>C. nigoni</em> and <em>C. briggsae</em> are very closely related, suggesting they inherited the same two-fingered <em>med</em> gene from a common ancestor (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Kiontke <em>et al.</em> 2011</a>). The positions of the six two-fingered MED factors in the phylogeny are hence consistent with two-finger MED-type GATA factors having arisen twice, likely by an interstitial duplication, because the two fingers in each share a nearly identical amino acid sequence. The observation of two-fingered GATA factors is noteworthy because among vertebrates, GATA factors generally have two zinc fingers, and even within <em>C. elegans</em>, there is a two-fingered GATA factor, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001249?doi=10.1534/g3.119.400724">ELT-1</a> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Gillis <em>et al.</em> 2009</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Lowry and Atchley 2000</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Page <em>et al.</em> 1997</a>).</p><p id="p-41">A tree of the DBDs of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> orthologs is shown in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. As mentioned earlier, all <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs are unique in each species except for the two possible <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> paralogues in <em>C. brenneri</em>. Among the END-3s, intraspecific amplification is implied for all species with two or more END-3s, except for a cluster containing <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> paralogues from <em>C. sinica</em>, <em>C. tribulationis</em>, and <em>C. zanzibari</em>. This portion of the tree is most consistent with two paralogous <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes having been present in the common ancestor of all three species. Hence, duplications do occur among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> paralogues, but at a far lower frequency than with the <em>med</em> genes.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of all END-3 and END-1 factors, showing tendency for END-1 factors to be unique, and END-3 factors to have undergone some duplications. The tree was generated by RAxML using the END-3 and END-1 DNA-binding domains (Kozlov et al. 2019; Stamatakis 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic tree of all END-3 and END-1 factors, showing tendency for END-1 factors to be unique, and END-3 factors to have undergone some duplications. The tree was generated by RAxML using the END-3 and END-1 DNA-binding domains (Kozlov &lt;em&gt;et al.&lt;/em&gt; 2019; Stamatakis 2014).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F7.medium.gif" width="389" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/1/333/F7.medium.gif" width="389" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495386" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-42" class="first-child">Phylogenetic tree of all <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors, showing tendency for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors to be unique, and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> factors to have undergone some duplications. The tree was generated by RAxML using the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> DNA-binding domains (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Kozlov <em>et al.</em> 2019</a>; <a id="xref-ref-107-4" class="xref-bibr" href="#ref-107">Stamatakis 2014</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Conserved domains of MED, END-3 and END-1</h3><p id="p-43">Prior alignments of the ENDs from <em>C. elegans</em> and <em>C. briggsae</em> revealed three conserved domains: An amino-terminal polyserine (Poly-S) region, a short region immediately upstream of the zinc finger, called the Endodermal GATA Domain (EGD), and the GATA-type zinc finger and basic domains (<a id="xref-ref-70-8" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). Among the MEDs, only the latter two domains are conserved (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>). Taking advantage of the 20 Elegans supergroup species, I aligned representative MED and END proteins to both generalize these earlier findings and to identify other conserved domains that might have been missed. The alignments revealed both expected and previously unknown conserved regions, shown diagrammatically in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>. On this figure, the corresponding positions of introns are also indicated to reveal patterns of conservation of the gene structure in relation to these conserved regions.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Conserved MED and END protein domains. The top part of the figure shows the MED, END-3 and END-1 protein structures with conserved domains in colored regions. Triangles represent the positions of introns in the coding regions as shown in the gene models in Fig. 4A. The bottom of the figure shows the names of the domains, which are shown at the amino acid level in Figs. 9 and 10. The MED orthologs have a variable region high in serine and threonine (Poly-S/T), while END-1 and END-3 share an amino-terminal polyserine domain (Poly-S) of variable length and an Endodermal GATA Domain (EGD). The END-1 orthologs share three additional regions not found in END-3. The species are arranged after the phylogeny in (Stevens et al. 2019)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Conserved MED and END protein domains. The top part of the figure shows the MED, END-3 and END-1 protein structures with conserved domains in colored regions. Triangles represent the positions of introns in the coding regions as shown in the gene models in Fig. 4A. The bottom of the figure shows the names of the domains, which are shown at the amino acid level in Figs. 9 and 10. The MED orthologs have a variable region high in serine and threonine (Poly-S/T), while END-1 and END-3 share an amino-terminal polyserine domain (Poly-S) of variable length and an Endodermal GATA Domain (EGD). The END-1 orthologs share three additional regions not found in END-3. The species are arranged after the phylogeny in (Stevens &lt;em&gt;et al.&lt;/em&gt; 2019).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F8.medium.gif" width="382" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/10/1/333/F8.medium.gif" width="382" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495360" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-44" class="first-child">Conserved MED and END protein domains. The top part of the figure shows the MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> protein structures with conserved domains in colored regions. Triangles represent the positions of introns in the coding regions as shown in the gene models in <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4A</a>. The bottom of the figure shows the names of the domains, which are shown at the amino acid level in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figs. 9</a> and <a id="xref-fig-10-1" class="xref-fig" href="#F10">10</a>. The MED orthologs have a variable region high in serine and threonine (Poly-S/T), while <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> share an amino-terminal polyserine domain (Poly-S) of variable length and an Endodermal GATA Domain (EGD). The <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs share three additional regions not found in <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>. The species are arranged after the phylogeny in (<a id="xref-ref-110-7" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> 2019</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>MED, END-3 and END-1 DNA-binding domains</h3><p id="p-45">An alignment of representative DBDs for the MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors, one per species, is shown in <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a> (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Edgar 2004</a>). Consistent with their recognizing an atypical binding site, the MED DBDs share features that distinguish them from the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> DBDs (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9A</a>). Among the Elegans group MED factors, the C4 zinc finger has 18 amino acids between the two pairs of cysteines, with a structure of CXXC-X<sub>18</sub>-CXXC, while the Japonica group members are diverged from this structure and have 16-17 amino acids, <em>i.e.</em>, CXXC-X<sub>16-17</sub>-CXXC. A consensus sequence with 11 invariant amino acids is shown below the alignment in <a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9A</a>. While the group of MED factor DBDs appears to be diverse, the identification of a conserved MED-like motif among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> promoters suggests that the MED factors have nonetheless coevolved to continue recognizing a similar binding site in each species. The solution structure of a <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a> DBD::binding site complex revealed that recognition of the MED binding site is mediated by 9 amino acids, indicated at the bottom of <a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure 9A</a> (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Lowry <em>et al.</em> 2009</a>). In comparing these with the corresponding amino acids in the other MED DBDs, there is evidence of conservation as shown by asterisks. Two of the 9 amino acids, a tyrosine (Y) and arginine (R) just after the zinc finger, are invariant. Five of the remaining amino acids are found in most of the MED DBDs. The remaining two are the isoleucine (I) and the first arginine in the zinc finger. The arginine is somewhat conserved, as in most MEDs it is an arginine or a lysine (K), both of which are basic. The isoleucine (I) is not conserved, and is replaced by a cysteine (C) in most other MEDs. This amino acid may not be critical for recognition of a MED binding site, however, as prior work showed that transgenes containing individual <em>med</em> genes from <em>C. briggsae</em> and <em>C. remanei</em> can fully complement the embryonic lethal phenotype of <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a>; <a href="https://identifiers.org/bioentitylink/WB:WBGene00003181?doi=10.1534/g3.119.400724"><em>med-2</em></a> double mutants; in the MED factors from both of these species, the corresponding amino acid is a cysteine. Overall, despite the higher divergence among the MEDs as a group, there appears to be selection for the 8/9 amino acids known to be involved in site recognition in <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>. Added to the apparent conservation of MED-like binding sites in the respective <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs in every species, the data suggest maintenance of the DNA-binding specificity of the MEDs.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA-binding domains (DBDs) and additional carboxyl amino acids aligned using MUSCLE (Edgar 2004). The zinc fingers and basic domains are shown for representative sequences of (A) MED, (B) END-3, (C) END-1, and (D) a representative subset of all three factors. Consensus sequences are shown below each alignment. The phylogeny of Stevens et al. (2019) is shown to the left of the species names for reference. Under the consensus sequences, the amino acids that mediate site recognition by the C. elegans MED-1 DBD for (A) and cGATA1 for (B), (C) and (D) are shown (Omichinski et al. 1993; Lowry et al. 2009). Asterisks show corresponding amino acids that are invariant (black) or are generally conserved (gray)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DNA-binding domains (DBDs) and additional carboxyl amino acids aligned using MUSCLE (Edgar 2004). The zinc fingers and basic domains are shown for representative sequences of (A) MED, (B) END-3, (C) END-1, and (D) a representative subset of all three factors. Consensus sequences are shown below each alignment. The phylogeny of Stevens &lt;em&gt;et al.&lt;/em&gt; (2019) is shown to the left of the species names for reference. Under the consensus sequences, the amino acids that mediate site recognition by the &lt;em&gt;C. elegans&lt;/em&gt; MED-1 DBD for (A) and cGATA1 for (B), (C) and (D) are shown (Omichinski &lt;em&gt;et al.&lt;/em&gt; 1993; Lowry &lt;em&gt;et al.&lt;/em&gt; 2009). Asterisks show corresponding amino acids that are invariant (black) or are generally conserved (gray).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F9.medium.gif" width="257" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/10/1/333/F9.medium.gif" width="257" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495446" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-46" class="first-child">DNA-binding domains (DBDs) and additional carboxyl amino acids aligned using MUSCLE (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Edgar 2004</a>). The zinc fingers and basic domains are shown for representative sequences of (A) MED, (B) <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>, (C) <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>, and (D) a representative subset of all three factors. Consensus sequences are shown below each alignment. The phylogeny of <a id="xref-ref-110-8" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> (2019)</a> is shown to the left of the species names for reference. Under the consensus sequences, the amino acids that mediate site recognition by the <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a> DBD for (A) and cGATA1 for (B), (C) and (D) are shown (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Omichinski <em>et al.</em> 1993</a>; <a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Lowry <em>et al.</em> 2009</a>). Asterisks show corresponding amino acids that are invariant (black) or are generally conserved (gray).</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">In contrast with the divergent MEDs, the DBDs of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs are more alike and share greater similarity to those of canonical GATA factors. The ENDs, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and cGATA1 have an invariant CXXC-X<sub>17</sub>-CXXC zinc finger structure with 17 amino acids between the 2<sup>nd</sup> and 3<sup>rd</sup> cysteines. Consensus sequences for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>, shown below the alignments in <a id="xref-fig-9-6" class="xref-fig" href="#F9">Figures 9B</a> and <a id="xref-fig-9-7" class="xref-fig" href="#F9">9C</a>, contain 23 invariant amino acids for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>, and 31 for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>, <em>i.e.</em>, 2x and 3x more than the 11 invariant amino acids among the MED DBDs. A solution structure for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> or <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> has not been reported, but as a surrogate I have shown, beneath both alignments, the 18 amino acids in the cGATA1 zinc finger known to mediate base contacts (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Omichinski <em>et al.</em> 1993</a>). <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> is conserved at 7/18 of these positions with 4 amino acids being invariant, while <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> has 10/18 positions conserved, of which 8 are invariant. Hence the END-1s are structurally more like cGATA1 than are the END-3s. Moreover, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs are also invariant at more positions, indicating that they are under the most evolutionary constraint.</p><p id="p-48">An amino acid in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> DBD is worth further comment. The proline between the 3<sup>rd</sup> and 4<sup>th</sup> cysteines of the zinc finger, in sequence CNPC, was substituted by a leucine in the EMS-induced <em>C. elegans</em> mutant <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a><em>(</em><a href="https://identifiers.org/bioentitylink/WB:WBVar00275536?doi=10.1534/g3.119.400724"><em>zu247</em></a><em>)</em> (<a id="xref-ref-70-9" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). This mutant has a phenotype indistinguishable from the null mutant <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a><em>(</em><a href="https://identifiers.org/bioentitylink/WB:WBVar00092662?doi=10.1534/g3.119.400724"><em>ok1448</em></a><em>)</em> which lacks most of the DBD (<a id="xref-ref-87-3" class="xref-bibr" href="#ref-87">Owraghi <em>et al.</em> 2010</a>). While this position is also a proline in 12/20 species, among the other END-3s it is serine (S) or alanine (A). Serine has a short polar side chain, while alanine is short and hydrophobic, however leucine is also hydrophobic but longer, suggesting that the longer side chain at this position compromises the structure of the zinc finger. This position is variable among the MED and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs, where it is a proline (P), alanine (A), serine (S), or glycine (G), indicating this position is under relaxed selection.</p><p id="p-49">Another difference between the END-3s and END-1s is the amino end of the C4 zinc finger between the 1<sup>st</sup> and 2<sup>nd</sup> cysteines. GATA factors in general, including the MEDs, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and cGATA1, have two amino acids in the pattern CXXC. Most of the END-3s are CSNC, while the END-1s have either CSNPNC (12 species), CSNPSC (6 species), CSNQNC (<em>C. afra</em>) or CNPNC (<em>C. becei</em>). It is not known what effect the extra one or two amino acids have on the structure of the zinc finger, however this variation in structure is found only in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs.</p><p id="p-50">Finally, as a set, the DBDs from the MEDs and ENDs of a subset of the Elegans supergroup species are shown with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and cGATA1 in <a id="xref-fig-9-8" class="xref-fig" href="#F9">Figure 9D</a>, showing that all three factors share conserved amino acids with each other and with canonical GATA factors. Overall, 7/18 of the amino acids known to mediate DNA recognition in cGATA1 are broadly conserved (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Omichinski <em>et al.</em> 1993</a>).</p></div><div id="sec-23" class="subsection"><h3>Serine-rich domains in MEDs and ENDs</h3><p id="p-51">The MED and END factors share an upstream region of variable size enriched in serine, with or without threonine. Both are polar amino acids. These are shown diagrammatically in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>, as the amino-most conserved domain among the MEDs and ENDs, and in amino acid sequence alignment in <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10A</a>. Among the MEDs, the Poly-S/T region is variable in size, consists of both serines and threonines, and is the only other conserved feature upstream of the DNA-binding domain. Because of the size variability, the alignment in <a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10A</a> represents only part of an overlapping region among MEDs of all 20 species. Among the ENDs, a similar Poly-S domain, consisting almost exclusively of homopolymeric clusters of serines, is found at the amino terminus starting at the 3<sup>rd</sup> or 4<sup>th</sup> amino acid (<a id="xref-fig-10-4" class="xref-fig" href="#F10">Figure 10A</a>). In one exception, the Poly-S domain is all but gone in <em>C. japonica</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>. As noted earlier, the Poly-S region had been previously recognized in the <em>C. elegans</em> and <em>C. briggsae end</em> genes (<a id="xref-ref-70-10" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Other conserved domains of unknown significance among the MED and END proteins. (A) A portion of the alignment of Poly-S/T domains (MED factors) and the Poly-S domains (END-3 and END-1). Serines are highlighted in blue and threonines in green. (B) Extended Endodermal GATA Domains (EGDs) immediately upstream of the zinc fingers of END-3 and END-1. A consensus sequence is shown beneath each alignment, with amino acids similar between END-3 and END-1 shown with an asterisk (*). (C) Highly conserved regions among the END-1 factors showing highly conserved amino acids and a consensus sequence beneath the alignment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Other conserved domains of unknown significance among the MED and END proteins. (A) A portion of the alignment of Poly-S/T domains (MED factors) and the Poly-S domains (END-3 and END-1). Serines are highlighted in blue and threonines in green. (B) Extended Endodermal GATA Domains (EGDs) immediately upstream of the zinc fingers of END-3 and END-1. A consensus sequence is shown beneath each alignment, with amino acids similar between END-3 and END-1 shown with an asterisk (*). (C) Highly conserved regions among the END-1 factors showing highly conserved amino acids and a consensus sequence beneath the alignment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F10.medium.gif" width="393" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/10/1/333/F10.medium.gif" width="393" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495415" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-52" class="first-child">Other conserved domains of unknown significance among the MED and END proteins. (A) A portion of the alignment of Poly-S/T domains (MED factors) and the Poly-S domains (<a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>). Serines are highlighted in blue and threonines in green. (B) Extended Endodermal GATA Domains (EGDs) immediately upstream of the zinc fingers of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>. A consensus sequence is shown beneath each alignment, with amino acids similar between <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> shown with an asterisk (*). (C) Highly conserved regions among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors showing highly conserved amino acids and a consensus sequence beneath the alignment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">An unexpected feature of the Poly-S region in the <em>end</em> genes bears further description. Although serine is coded by six codons – TCT, TCC, TCA, TCG, AGT and AGC – the serines among the Poly-S regions in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> orthologs are coded almost exclusively (99%, 554/557) by TCN codons (N = any base). Moreover, two of the four TCN codons, TCT and TCC, are used 50% and 22% of the time. Among <em>C. elegans</em> genes, TCN represents 75% of serine codons, and among these, TCT and TCC occur only 28% and 18% of the time, respectively (<a href="https://www.genscript.com/tools/codon-frequency-table">https://www.genscript.com/tools/codon-frequency-table</a>). This preferential use of TCT and TCC codons for serine in the Poly-S regions, among the TCN codons, is statistically significant (<em>P</em> &lt; 10<sup>−40</sup>, χ<sup>2</sup>-test). The possible implications of this codon bias are discussed later.</p></div><div id="sec-24" class="subsection"><h3>Conservation of the end family gata domain (EGD)</h3><p id="p-54">Previous work identified the END family GATA Domain, or EGD, immediately upstream of the <em>C. elegans</em> and <em>C. briggsae</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> DBDs (<a id="xref-ref-70-11" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). This domain does not occur among the other <em>C. elegans</em> GATA factors, suggesting it is uniquely important for function of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>. Among the 20 species in the Elegans supergroup, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> orthologs across 20 species do contain a conserved region immediately upstream of the zinc finger. This is shown diagrammatically in <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a>, and by sequence alignment in <a id="xref-fig-10-5" class="xref-fig" href="#F10">Figure 10B</a>. Whereas the original report had the domain consisting of 9 amino acids, an extended domain is apparent that consists of approximately 25 amino acids. Seven of these (shown by an asterisk in the figure) are highly conserved between the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors, but there are additional conserved amino acids within each group of factors. Moreover, the domain is more conserved among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> orthologs. While the EGDs tend to be enriched in basic amino acids, suggesting they may be involved in general DNA binding, their significance remains unknown.</p></div><div id="sec-25" class="subsection"><h3>END-1 specific domains</h3><p id="p-55">Among the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> orthologs, the region between the Poly-S and the EGD regions is variable in size and does not exhibit sequences with extensive conservation (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8</a>). In contrast, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs display three additional domains that are highly conserved across all 20 species (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Figures 8</a> and <a id="xref-fig-10-6" class="xref-fig" href="#F10">10C</a>). A consensus sequence shows high conservation with many invariant regions. These domains are apparently novel, as a BLAST search using this region of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> did not identify related proteins other than predicted orthologs of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> within <em>Caenorhabditis</em>. With the identification of these extended sequence similarities, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs across the 20 species are highly conserved throughout their lengths, while the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and MED orthologs are conserved only in parts.</p></div></div><div class="section discussion" id="sec-26"><h2 class="">Discussion</h2><p id="p-56">In this work I have identified and compared the gene and protein structures of the MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> GATA transcription factors among 20 <em>Caenorhabditis</em> species of the Elegans supergroup. Predictions were made by manual curation, guided by known features of the network from <em>C. elegans</em> and informed by comparison of gene and protein structures together. The results confirm coevolution of <em>cis</em>-regulatory sites, gene structures and protein sequence over tens of millions of years of evolution. Many of the conserved features, including the DNA-binding domains, and binding sites for <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>, MED, and an Sp1-like factor, are consistent with known properties of the <em>med</em> and <em>end</em> genes in <em>C. elegans</em> (<a id="xref-ref-76-6" class="xref-bibr" href="#ref-76">Maduro <em>et al.</em> 2001</a>; <a id="xref-ref-72-6" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>; <a id="xref-ref-111-2" class="xref-bibr" href="#ref-111">Sullivan-Brown <em>et al.</em> 2016</a>; <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>). Prior work has also shown that orthologous <em>med</em>s and/or <em>end</em>s from a few of these species can function as transgenes in <em>C. elegans</em> (<a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>; <a id="xref-ref-70-12" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). Hence, I hypothesize that the <em>med</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes function in a core endoderm specification network across the Elegans supergroup that originated in a common ancestor.</p><div id="sec-27" class="subsection"><h3>High rates of med gene duplication</h3><p id="p-57">The <em>med</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes showed distinct patterns of gene duplication among species. Occurrence of duplicate <em>med</em> genes is disproportionately high, with an average of 5.6 <em>med</em> genes per species (or 3.7 if <em>C. doughertyi</em> and <em>C. brenneri</em> are not counted), compared with 2.0 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes and a single <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> per species, except for <em>C. brenneri</em> which may have two <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). In most cases, sequence similarity was consistent with most <em>med</em> duplicates having arisen post-speciation, with exceptions resulting from likely inheritance of two or three <em>med</em> genes from a recent common ancestor (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>).</p><p id="p-58">The disproportionate amplification of the <em>med</em>s compared with the <em>end</em>s suggests that there is ongoing selective pressure for increased numbers of <em>med</em> genes. The high amplification of the <em>med</em>s is unusual, as redundancy of GATA factors in tissue specification is typically not more than twofold in other systems (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Gillis <em>et al.</em> 2009</a>; <a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Tremblay <em>et al.</em> 2018</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Murakami <em>et al.</em> 2005</a>). Across the Elegans supergroup, the occurrence of MED binding sites in the <em>end</em> genes (particularly <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>) argues for positive selection for the presence of these sites, and hence the MED factors that bind them. Loss of MED binding sites in the <em>C. elegans end</em> genes results in aberrant intestinal lineage development, metabolic defects, and reduced viability (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2017</a>; <a id="xref-ref-72-7" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>). Hence, duplications of <em>med</em> genes might select for increased <em>med</em> expression to make gut specification more robust. <em>C. elegans</em> has a high rate of segmental duplications compared with other species, with a higher gene dose generally leading to increased mRNA production (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Konrad <em>et al.</em> 2018</a>). Alternatively, it may be that MED factors in some species have become collectively reduced in their ability to be activated or to activate target genes, in a way that maintains multiple copies due to complementary degenerative mutations (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Force <em>et al.</em> 1999</a>). Protein degeneracy would be consistent with the lower degree of sequence conservation among the MED DNA-binding domains in <em>C. brenneri</em>, which has experienced an extreme amplification of <em>med</em> genes (<a id="xref-fig-9-9" class="xref-fig" href="#F9">Figure 9</a>). However, this does not explain amplification of <em>med</em> genes in <em>C. doughertyi</em>, whose MED DNA-binding domains are more similar as a group, unless they are all collectively degenerate in some way. In <em>C. elegans</em>, which has two nearly identical <em>med</em> genes, either <em>med</em> gene is dispensable, although when <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a> is deleted, <a href="https://identifiers.org/bioentitylink/WB:WBGene00003181?doi=10.1534/g3.119.400724"><em>med-2</em></a> becomes haploinsufficient in 35% of embryos due to a failure to specify the MS blastomere (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Maduro <em>et al.</em> 2007</a>). Hence, maintenance of copies of <em>med</em> genes may be occurring by selection for robust specification of MS rather than E (<a id="xref-ref-76-7" class="xref-bibr" href="#ref-76">Maduro <em>et al.</em> 2001</a>). This still does not explain the extreme amplification, although it could explain why a driving force for duplications is not apparent from the structure of the <em>end</em> genes.</p><p id="p-59">Rather than increase expression through gene duplication, it seems equally possible for a small number of mutations to increase expression or activity of any one <em>med</em> gene. Hence, some other constraint may select against a small number of <em>med</em> genes in some species. For example, a reduction in <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> activity could limit the expression of individual <em>med</em> genes and select for <em>med</em> gene amplification as a compensatory mechanism. It is also likely that at least some duplicated <em>med</em> genes have acquired new essential functions. Consistent with this, not all <em>med</em> orthologs from <em>C. remanei</em> are able to rescue <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a>; <a href="https://identifiers.org/bioentitylink/WB:WBGene00003181?doi=10.1534/g3.119.400724"><em>med-2</em></a> double mutants, even as multicopy transgenes, which would be expected to overcome expression limitations (<a id="xref-ref-19-9" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>). Future work to quantify the contributions of individual <em>med</em> genes in other Elegans supergroup species, or to test expression of these when introduced into <em>C. elegans</em> as single-copy transgenes, may shed some light on what mechanisms may be driving increased <em>med</em> copy number.</p></div><div id="sec-28" class="subsection"><h3>Linkage of end orthologs</h3><p id="p-60">In most species, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> was found within ∼35 kbp of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>). One possibility for maintenance of this synteny is that the two genes may be coregulated. Three lines of evidence argue against this possibility, at least for <em>C. elegans</em>. First, there is at least one unrelated gene between the <em>end</em>s, the neural gene <a href="https://identifiers.org/bioentitylink/WB:WBGene00010259?doi=10.1534/g3.119.400724"><em>ric-7</em></a> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Hao <em>et al.</em> 2012</a>). Second, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a><em>,3</em> genes are not precisely co-expressed as accumulation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> mRNA precedes that of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Baugh <em>et al.</em> 2003</a>; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Maduro <em>et al.</em> 2007</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Raj <em>et al.</em> 2010</a>). Third, unlinked single-copy transgenes of wild-type <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> are able to completely replace function of the endogenous genes when introduced into an <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a><em>,3(-)</em> strain, suggesting that linkage is not a prerequisite for their expression (<a id="xref-ref-72-8" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>). It may be, therefore, that synteny of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> merely reflects their origin as a tandem duplication of an ancestral <em>end</em> gene.</p><p id="p-61">A pair of partially redundant developmental factors in <em>C. elegans</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00003001?doi=10.1534/g3.119.400724">LIN-12</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001609?doi=10.1534/g3.119.400724">GLP-1</a>, which encode highly similar Notch orthologs, are a good comparison for the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>/3 pair (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Rudel and Kimble 2002</a>). These paralogous genes are similar in structure and have overlapping function in <em>C. elegans</em> development (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Moskowitz and Rothman 1996</a>). The two genes are approximately 30 kbp apart in the <em>C. elegans</em> genome with apparently unrelated intervening genes (<a href="http://wormbase.org">http://wormbase.org</a>). The <a href="https://identifiers.org/bioentitylink/WB:WBGene00003001?doi=10.1534/g3.119.400724"><em>lin-12</em></a>/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001609?doi=10.1534/g3.119.400724"><em>glp-1</em></a> pair is conserved in closely related species, and likely arose from the duplication of a progenitor gene at the base of the Elegans supergroup (<a id="xref-ref-110-9" class="xref-bibr" href="#ref-110">Stevens <em>et al.</em> 2019</a>; <a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Rudel and Kimble 2002</a>). A search of the Elegans supergroup genomes finds examples where <a href="https://identifiers.org/bioentitylink/WB:WBGene00003001?doi=10.1534/g3.119.400724"><em>lin-12</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001609?doi=10.1534/g3.119.400724"><em>glp-1</em></a> orthologs are found within tens of kbp on the same sequence scaffolds, suggesting microsynteny is conserved in at least some species (data not shown). The conservation of microsynteny for <a href="https://identifiers.org/bioentitylink/WB:WBGene00003001?doi=10.1534/g3.119.400724"><em>lin-12</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001609?doi=10.1534/g3.119.400724"><em>glp-1</em></a>, like that of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>, then, likely results from the origin of the genes as a linked duplication, followed by the tendency for genomic segments tens of kbp in size to stay intact within the genus (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Coghlan and Wolfe 2002</a>).</p></div><div id="sec-29" class="subsection"><h3>Identification of known and previously unrecognized cis-regulatory sites</h3><p id="p-62">The MEME search recovered binding sites for regulators previously known to activate the <em>med</em> and <em>end</em> genes in <em>C. elegans</em> (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4B</a>). In the case of the <em>med</em> orthologs, these were binding sites for <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>, while for the <em>end</em> genes, these were binding sites for both <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>. The conservation of these sites supports the hypothesis that these genes have maintained the same regulatory hierarchy as in <em>C. elegans</em>, with <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> activating the <em>med</em> genes, and both <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and the MED proteins activating the <em>end</em> genes. The MED sites in the Elegans supergroup <em>end</em> genes are found in all <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologs but only 9/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> orthologs. <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> has four MED sites and these are collectively essential for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> activation, although even a single MED site in a single-copy <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> transgene is sufficient for activation (<a id="xref-ref-72-9" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>). In contrast, <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> has only two MED sites, and these are less important for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> expression due to the stronger parallel input by TCF/<a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00003912?doi=10.1534/g3.119.400724">PAL-1</a> into <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> as compared with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> (<a id="xref-ref-72-10" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>; <a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Maduro <em>et al.</em> 2005b</a>). Hence, the lower number of MED sites in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes may reflect stronger input from other factors. The likely sites for <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> in <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> were not previously known because they do not contain the same pattern of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> site core sequences as present in the <em>med</em> promoters. An intriguing hypothesis is that the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <em>end</em> genes may be of lower affinity than those in the <em>med</em> genes. Because expression of the <em>end</em> genes is delayed by at least one cell cycle compared with <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a><em>,2</em>, lower-affinity <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites could potentially allow for delayed activation, preventing expression of the <em>end</em>s before EMS has divided into MS and E. A similar affinity difference has been hypothesized for early- and late-acting binding sites of the pharynx regulator <a href="https://identifiers.org/bioentitylink/WB:WBGene00004013?doi=10.1534/g3.119.400724">PHA-4</a> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Gaudet <em>et al.</em> 2004</a>). As the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <em>end</em> genes were not found in all species, it is possible that the input from <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> directly into gut specification through the <em>end</em>s is lost or further weakened in some species. This might make the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> → MED → <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 pathway more strictly linear, similar to the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> → MED → <a href="https://identifiers.org/bioentitylink/WB:WBGene00006554?doi=10.1534/g3.119.400724">TBX-35</a> pathway that specifies MS in <em>C. elegans</em> (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Broitman-Maduro <em>et al.</em> 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Broitman-Maduro <em>et al.</em> 2009</a>). In MS, loss of the MED factors results in the absence of MS-derived fates, consistent with an inability of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> to specify MS without the MED factors. Finally, an additional suspected regulatory input was from an Sp1-like factor, likely to be <a href="https://identifiers.org/bioentitylink/WB:WBGene00012735?doi=10.1534/g3.119.400724">SPTF-3</a> (<a id="xref-ref-111-3" class="xref-bibr" href="#ref-111">Sullivan-Brown <em>et al.</em> 2016</a>). Most of the <em>med</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> orthologs have a consensus Sp1 binding site (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4B</a>). Together, the recovery of these sites from an <em>ab initio</em> search of their putative promoters lends strong support to the hypothesis of conservation of this gene network across the Elegans supergroup.</p><p id="p-63">MEME-identified sites of lower significance, and not as broadly conserved, are either unknown or reflect putative core promoter elements. These include one with core sequence TCTKCAC, a polypyrimidine motif, putative PolyA/T cluster, a TATA-binding protein (TBP) site, and an SL1 motif. The latter two were previously found in many promoters in five Elegans supergroup species (<a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Grishkevich <em>et al.</em> 2011</a>). The putative PolyA/T cluster is associated with germline expression (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Frøkjær-Jensen <em>et al.</em> 2016</a>). The other two motifs are of unknown significance. The TCTKCAC motif is found in the <em>C. elegans med</em> genes, hence it is possible to test its significance directly. The site was found three times, and close to the previously identified <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites, suggesting the site may play an accessory role to <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> activation.</p><p id="p-64">It is particularly conspicuous that sites for minor regulatory inputs known in <em>C. elegans</em> were not found to be widely conserved, either by a direct search or through MEME. This includes sites for TCF/<a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> and the Caudal ortholog <a href="https://identifiers.org/bioentitylink/WB:WBGene00003912?doi=10.1534/g3.119.400724">PAL-1</a>, both of which are genetically known to contribute to <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> expression, and for which binding sites are known or suspected based on prior work (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bhambhani <em>et al.</em> 2014</a>; <a id="xref-ref-74-5" class="xref-bibr" href="#ref-74">Maduro <em>et al.</em> 2005b</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Robertson <em>et al.</em> 2011</a>; <a id="xref-ref-103-5" class="xref-bibr" href="#ref-103">Shetty <em>et al.</em> 2005</a>). In <em>C. elegans</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> is also a suspected contributor to activation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> based on reduction of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> mRNA in an <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> mutant background (<a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Maduro <em>et al.</em> 2007</a>). The failure to recover sites for these regulators suggests that these inputs are poorly conserved or lie outside of the regions that were searched, or else the binding sites have changed among the various species. Given how easily <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and MED sites were found, it could also be that different species have evolved different sets of supportive regulatory inputs. The apparent qualitative differences in regulatory input of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a> in <em>C. briggsae</em>, revealed through cryptically different reduction-of-function phenotypes between <em>C. briggsae</em> and <em>C. elegans</em>, suggests that reinforcing regulatory inputs may evolve rapidly (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Lin <em>et al.</em> 2009</a>). Even within <em>C. elegans</em>, widespread cryptic variation in input from <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and the Wnt pathway (which acts through <a href="https://identifiers.org/bioentitylink/WB:WBGene00004077?doi=10.1534/g3.119.400724">POP-1</a>) was observed among <em>C. elegans</em> wild isolates (<a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Torres Cleuren <em>et al.</em> 2019</a>). An emerging model seems to be that the core <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> → MED → <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 regulatory cascade is conserved, while additional regulatory inputs that reinforce this cascade evolve rapidly and would thus be expected to be species-specific. Putative <em>cis</em>-regulatory sites that mediate these supporting inputs might therefore occur in only a subset of species in the Elegans supergroup and would be missed in the analysis done here.</p></div><div id="sec-30" class="subsection"><h3>End-3 and end-1: The same but different</h3><p id="p-65">In <em>C. elegans</em>, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> clearly have overlapping function. Complete loss of both genes has a fully penetrant failure to specify endoderm, while null alleles either for gene alone have either no effect (<a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>) or a weak effect (<a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>) on gut specification (<a id="xref-ref-70-13" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>; <a id="xref-ref-87-4" class="xref-bibr" href="#ref-87">Owraghi <em>et al.</em> 2010</a>). A similar result was obtained using RNAi in <em>C. briggsae</em> (<a id="xref-ref-70-14" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>). As well, overexpression of either <em>end</em> gene in <em>C. elegans</em> is sufficient to induce endoderm differentiation in non-endodermal lineages (<a id="xref-ref-70-15" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>; <a id="xref-ref-130-1" class="xref-bibr" href="#ref-130">Zhu <em>et al.</em> 1998</a>). Within their DNA-binding domains, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs are clearly more similar to each other than they are to the MEDs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figures 5</a>, <a id="xref-fig-9-10" class="xref-fig" href="#F9">9</a>).</p><p id="p-66">Despite these similarities, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> differ in ways that suggest they have at least some unique functions. First, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> DBDs are more highly conserved as a group, while those of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> are under slightly more relaxed selection. This is apparent in the way that the DBDs appear in a phylogenetic tree (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>) and in the degree of invariant amino acids in an alignment (<a id="xref-fig-9-11" class="xref-fig" href="#F9">Figures 9B</a>, <a id="xref-fig-9-12" class="xref-fig" href="#F9">9C</a>). Within their DBDs, the END-1s have twice as many similar amino acids in common with vertebrate cGATA1 than the END-3s have in common with cGATA1, notably in amino acid positions known to mediate sequence recognition (<a id="xref-fig-9-13" class="xref-fig" href="#F9">Figures 9B</a>, <a id="xref-fig-9-14" class="xref-fig" href="#F9">9C</a>).</p><p id="p-67">Additional evidence is consistent with both shared and divergent activity of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> in <em>C. elegans</em>. Recent work inferred the binding sites for <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> as RSHGATAASR and RKWGATAAGR, respectively, which are very similar though not identical (<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">Weirauch <em>et al.</em> 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Lambert <em>et al.</em> 2019</a>). Other work has shown that recombinant DNA-binding domains of <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> can bind canonical GATA sites in the promoter of <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>, although <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> has a higher affinity for such sites (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Du <em>et al.</em> 2016</a>; <a id="xref-ref-122-3" class="xref-bibr" href="#ref-122">Wiesenfahrt <em>et al.</em> 2015</a>). From this work, Endoderm GATA Domains (EGDs) immediately upstream of the DBDs show conserved amino acids between END-3s and END-1s but many more that are unique to either EGD (<a id="xref-fig-10-7" class="xref-fig" href="#F10">Figure 10B</a>). Although the function of the EGDs remains unknown, their conservation and proximity to the DBDs suggest an accessory role in protein-DNA interaction that is unique to the ENDs among the <em>Caenorhabditis</em> GATA factors.</p></div><div id="sec-31" class="subsection"><h3>The Poly-S region of END-3 and END-1: protein domain or polypyrimidine tract?</h3><p id="p-68"><a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> share an amino-terminal segment, far from the DNA-binding domain, that is enriched for homopolymers of serine (<a id="xref-fig-10-8" class="xref-fig" href="#F10">Figure 10A</a>). Such a domain is not found in the other <em>C. elegans</em> GATA factors, nor is enrichment for serine found in vertebrate GATA factors (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kaneko <em>et al.</em> 2012</a>; <a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">Yang <em>et al.</em> 1994</a>). This suggests that the Poly-S domain plays some other function besides DNA binding and transactivation. The selection for TCT and TCC codons suggests that the Poly-S regions have been maintained for a reason other than a selection for what they contribute to the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> proteins. Beyond transcriptional activation of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes, post-transcriptional regulatory mechanisms could potentially fine-tune <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 protein levels. At the level of mRNA, the preference for these codons, as opposed to UCG and UCA, results in maintenance of a polypyrimidine tract in the mRNA. Support for a possible role of such a tract in the endoderm GRN is that in some species (<em>e.g.</em>, <em>C. latens</em> and <em>C. remanei</em>), the <em>med</em> orthologs also have an apparent enrichment of T and C bases in the first part of their coding regions. In other systems, polypyrimidine tract binding proteins (PTBs) have various roles in RNA metabolism, including regulation of splicing and mRNA stability, though in these cases the tracts occur outside of coding regions (<a id="xref-ref-97-3" class="xref-bibr" href="#ref-97">Sawicka <em>et al.</em> 2008</a>). There is a <em>C. elegans</em> PTB gene, <a href="https://identifiers.org/bioentitylink/WB:WBGene00004207?doi=10.1534/g3.119.400724"><em>ptb-1</em></a>, but its function has not been described (<a href="http://wormbase.org">http://wormbase.org</a>). At the level of translation, repeats of the same UCY serine codon could cause starvation for limiting amounts of a particular seryl-tRNA<sup>Ser</sup>, leading to ribosome pausing (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Darnell <em>et al.</em> 2018</a>). However, it is not clear why there would be selection to delay translation of <em>end</em> mRNA, particularly as given the rapid early cell divisions of the <em>C. elegans</em> embryo, it makes more sense to express the gene products as rapidly as possible. A more benign reason for the maintenance of the serine codon repeats is that they might be an artifact of a trinucleotide repeat expansion process (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Koren and Trifonov 2011</a>). Indeed, in that study, amino acid repeats in vertebrate proteins were most likely to be found in the first exon, <em>i.e.</em>, at the amino end, consistent with their location in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes. Hence, the role of the Poly-S domain, if any, remains open for speculation until structure-function studies are performed.</p></div><div id="sec-32" class="subsection"><h3>END-1 orthologs are conserved throughout their lengths</h3><p id="p-69">An additional unexpected finding emerged from the alignment of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs that distinguishes them among the MED/END proteins. Between the Poly-S and EGD domains, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> orthologs as a group are diverse in size and sequence, whereas the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs are more similar in size and show several regions of high conservation (<a id="xref-fig-10-9" class="xref-fig" href="#F10">Figure 10C</a>). These <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>-specific domains can be grouped into three regions containing blocks of invariant amino acids. The most striking of these is the center domain which contains an invariant sequence of FGQYF across all species END-1s. None of these highly conserved domains is found in other proteins, apart from predicted <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs. The high conservation is further supported by the conservation of introns in the <em>end</em> genes. The <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes have four introns with only one of these absent in <em>C. brenneri</em> (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4A</a>). In contrast, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes were more likely to experience intron gains and losses over the same evolutionary time period, with most of these occurring in the variable region between the amino-terminal Poly-S and EGD domains (<a id="xref-fig-8-7" class="xref-fig" href="#F8">Figure 8</a>). A cursory examination of the amino acids in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>-specific domains suggests that these are on the outside of the protein, perhaps mediating protein-DNA or protein-protein interactions that do not occur with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> (data not shown).</p><p id="p-70">Taken together, these data show that across the Elegans supergroup, the END-1s are highly conserved proteins with greater similarity to vertebrate GATA factors than the more diverse END-3s proteins. This predicts that <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> has unique features in transcriptional activation, and that the target genes activated by each of these factors are likely to include both common and distinct targets.</p></div><div id="sec-33" class="subsection"><h3>Med othrologs: A divergent and diverse subclass of GATA factors</h3><p id="p-71">The MED orthologs among the 20 species were found to be divergent from the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a>/<a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors, and to comprise a more diverse group of proteins, even within the DNA-binding domain (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figures 5</a>, <a id="xref-fig-9-15" class="xref-fig" href="#F9">9</a>). The divergence of the DBD from that of the ENDs, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and cGATA1 is expected, because the <em>C. elegans</em> MEDs were recognized to be divergent GATA factors that recognize a different binding site with an AGTATAC core (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Lowry <em>et al.</em> 2015</a>). Despite the high divergence of the MED factors as a group, indicating relaxed selection, there appears to be maintenance of their binding site sequence over evolutionary time. This is supported by the conservation, across all 20 species, of most of the amino acids that were found to mediate protein-DNA recognition in <em>C. elegans</em> <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a> (<a id="xref-fig-9-16" class="xref-fig" href="#F9">Figure 9A</a>), and more importantly, by the MEME identification of AGTATAC binding sites among all <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> orthologous genes and 9/20 <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4</a>). Furthermore, transgenes of most of the <em>C. briggsae</em> and <em>C. remanei med</em>s were individually able to complement <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724"><em>med-1</em></a><em>,2</em> double mutants in both gut and mesoderm specification despite limited conservation, albeit in high copy number transgenes (<a id="xref-ref-19-10" class="xref-bibr" href="#ref-19">Coroian <em>et al.</em> 2006</a>). Selection is likely not acting solely on the MEDs for <em>end</em> gene activation, as there are other direct MED targets in <em>C. elegans</em> whose orthologs in the Elegans supergroup were not investigated here, including in the early MS lineage (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Broitman-Maduro <em>et al.</em> 2006</a>; <a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Broitman-Maduro <em>et al.</em> 2005</a>). The lower conservation suggests that the MED DBDs may simply be more accommodating of amino acid substitutions than are the DBDs of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> or <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>.</p><p id="p-72">Outside of the DNA-binding domain, the MEDs as a group lack the type of conserved regions seen in the ENDs. The only other feature found is a variable enrichment for serine and threonine of unknown significance. This region does not resemble the homopolymeric serine regions at the amino end of the ENDs (<a id="xref-fig-10-10" class="xref-fig" href="#F10">Figure 10A</a>). Rather, it is a higher prevalence for S/T that lacks a recognizable context. A serine-threonine rich motif was found to be important for nuclear localization of the mineralocorticoid receptor in vertebrates, suggesting that this region of the MED orthologs may play a similar role (<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">Walther <em>et al.</em> 2005</a>). Until structure-function analyses are done, the significance of the serine/threonine enrichment will remain unknown.</p></div><div id="sec-34" class="subsection"><h3>The MED/END cascade is a derived charachter</h3><p id="p-73">The existence of a gut precursor is a conserved lineage feature found in more distantly related nematode species (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Schierenberg 2006</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Houthoofd <em>et al.</em> 2003</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Schulze and Schierenberg 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boveri 1892</a>). It must therefore be that species outside the Elegans supergroup specify the gut precursor without MED/END factors. The most upstream factor <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>, and the downstream gut identity factor <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a>, are also more widely conserved than just the Elegans supergroup (<a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Schiffer <em>et al.</em> 2014</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Couthier <em>et al.</em> 2004</a>). If <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> still specifies MS and E outside of the Elegans supergroup, the simplest hypothesis is that specification of gut occurs by direct activation of an <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>-like gene by <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>. An attempt to demonstrate bypass of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> genes was successful using an <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> transgene under regulatory control of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> promoter in a <em>C. elegans</em> strain lacking <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> (<a id="xref-ref-122-4" class="xref-bibr" href="#ref-122">Wiesenfahrt <em>et al.</em> 2015</a>). However, this transgene worked best in a high copy-number array, and not in single-copy. Furthermore, expression of this transgene is likely to be at least partially dependent upon regulatory input by <a href="https://identifiers.org/bioentitylink/WB:WBGene00003180?doi=10.1534/g3.119.400724">MED-1</a>,2, based on studies with an <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> promoter lacking MED binding sites (<a id="xref-ref-72-11" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>). As an alternative to direct <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> → <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> regulation, there could be one or more non-GATA regulators between them, analogous to the MED/END cascade. Regardless of how gut specification occurs outside of the Elegans supergroup, some set of evolutionary events must have set in motion a breakdown of the ancestral specification mechanism, favoring the evolution and fixation of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>/MED/END cascade as the dominant mode of E specification.</p></div><div id="sec-35" class="subsection"><h3>Evolutionary Origin Of the SKN-1 → MED → end-1,3 cascade</h3><p id="p-74">The co-occurrence of the MED and END factors suggests that these genes evolved within a short time at the base of the Elegans supergroup (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11A</a>). A preliminary search for orthologs of <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724">ELT-7</a> also found evidence that this factor likely originated at the same time, as 18/20 of the Elegans supergroup species have a clear <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> ortholog while species outside do not (data not shown). At the start of this work there was an expectation that there might have been one or more “transitional” species with only part of the network upstream of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a>, for example with only the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> factors, or only one <em>end</em>-like factor. Since no such species were found apart from the two species that may lack <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> orthologs, it may be that for the <em>med</em> and <em>end</em> factors, a transitional species has not yet been sequenced, or is extinct, or that the orthologs are highly diverged. The reduced number of recognizable GATA factors in species outside of the Elegans supergroup argues against this possibility, however.</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/333/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Origin of the MED, END-3 and END-1 factors. (A) Origin of all three factors at the base of the Elegans supergroup, followed by loss of a conserved intron in an ancestral med gene at the base of the Elegans group. (B) Hypothetical microhomology-mediated end joining (MMEJ) event that could delete the conserved zinc finger intron at the base of the Elegans group, using a 6-bp identity in-frame microhomology in an extant C. japonica med gene. At top, the microhomology is shown for the top strand. In the bottom part, complementary strands are shown pairing across the microhomology, which if resolved could result in an in-frame deletion of the intron, after (van Schendel and Tijsterman 2013). This would also require maintenance of the AAC codon for asparagine immediately to the right of the homology. (C) Speculative model for generation of the SKN-1/MED/END regulatory cascade through intercalation by serial duplications of an ancestral autoregulating elt-2 gene. A bent arrow indicates the transcription start site, with the regulatory activity of the protein product of the gene shown as a colored line from the bent arrow. The promoter is to the left of the bent arrow. The positions in the promoters are only meant to qualitatively convey positive regulation and not indicate number or position of binding sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1835404595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Origin of the MED, END-3 and END-1 factors. (A) Origin of all three factors at the base of the Elegans supergroup, followed by loss of a conserved intron in an ancestral &lt;em&gt;med&lt;/em&gt; gene at the base of the Elegans group. (B) Hypothetical microhomology-mediated end joining (MMEJ) event that could delete the conserved zinc finger intron at the base of the Elegans group, using a 6-bp identity in-frame microhomology in an extant &lt;em&gt;C. japonica med&lt;/em&gt; gene. At top, the microhomology is shown for the top strand. In the bottom part, complementary strands are shown pairing across the microhomology, which if resolved could result in an in-frame deletion of the intron, after (van Schendel and Tijsterman 2013). This would also require maintenance of the AAC codon for asparagine immediately to the right of the homology. (C) Speculative model for generation of the SKN-1/MED/END regulatory cascade through intercalation by serial duplications of an ancestral autoregulating &lt;em&gt;elt-2&lt;/em&gt; gene. A bent arrow indicates the transcription start site, with the regulatory activity of the protein product of the gene shown as a colored line from the bent arrow. The promoter is to the left of the bent arrow. The positions in the promoters are only meant to qualitatively convey positive regulation and not indicate number or position of binding sites.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/333/F11.medium.gif" width="353" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/10/1/333/F11.medium.gif" width="353" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/333/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/333/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495200" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-75" class="first-child">Origin of the MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> factors. (A) Origin of all three factors at the base of the Elegans supergroup, followed by loss of a conserved intron in an ancestral <em>med</em> gene at the base of the Elegans group. (B) Hypothetical microhomology-mediated end joining (MMEJ) event that could delete the conserved zinc finger intron at the base of the Elegans group, using a 6-bp identity in-frame microhomology in an extant <em>C. japonica med</em> gene. At top, the microhomology is shown for the top strand. In the bottom part, complementary strands are shown pairing across the microhomology, which if resolved could result in an in-frame deletion of the intron, after (<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">van Schendel and Tijsterman 2013</a>). This would also require maintenance of the AAC codon for asparagine immediately to the right of the homology. (C) Speculative model for generation of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>/MED/END regulatory cascade through intercalation by serial duplications of an ancestral autoregulating <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> gene. A bent arrow indicates the transcription start site, with the regulatory activity of the protein product of the gene shown as a colored line from the bent arrow. The promoter is to the left of the bent arrow. The positions in the promoters are only meant to qualitatively convey positive regulation and not indicate number or position of binding sites.</p><div class="sb-div caption-clear"></div></div></div><p id="p-76">The data strongly suggest that the <em>med</em> and <em>end</em> genes might have been derived from the same ancestral gene. This hypothesis is supported by the existence of an intron in the zinc finger domain of all <em>med</em> and <em>end</em> genes, except for the Elegans group <em>med</em> genes where loss of this intron occurred. This intron is also found in <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> in <em>C. elegans</em> and at least some of the other species in the Elegans supergroup (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Fukushige <em>et al.</em> 1998</a>; <a id="xref-ref-105-2" class="xref-bibr" href="#ref-105">Sommermann <em>et al.</em> 2010</a>)(data not shown). Intron loss is common throughout the genus, and occurs more frequently than intron gain (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Roy and Penny 2006</a>). One mechanism by which this particular intron could have been lost in an ancestral <em>med</em> gene of the Elegans group is through germline gene conversion from a reverse-transcribed (spliced) mRNA (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Roy and Gilbert 2005</a>). An alternative mechanism could be through microhomology-mediated end joining, or MMEJ, of a double-stranded break in the gene (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">McVey and Lee 2008</a>; <a id="xref-ref-118-2" class="xref-bibr" href="#ref-118">van Schendel and Tijsterman 2013</a>). Indeed, in one of the <em>C. japonica med</em> genes, a short stretch of six base pairs upstream of this intron recurs close to the 3′ splice site of the intron itself, such that a repair of a double-stranded chromosome break by MMEJ would result in an in-frame removal of the intron (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure 11B</a>). This would also require that the asparagine codon (AAC) is somehow maintained, which may be possible given the observed types of MMEJ repair of double-stranded breaks induced by Cas9 cleavage, <em>e.g.</em>, (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Taheri-Ghahfarokhi <em>et al.</em> 2018</a>). Regardless of the mechanism, loss of this intron likely occurred only once in the last common ancestor to the Elegans group. I note in passing that the converse property, lack of intron gain in the Elegans group <em>med</em> genes, may be accounted for by selection for rapid gene expression through avoidance of mRNA splicing; most early zygotic <em>Drosophila</em> genes are intronless, for example (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Guilgur <em>et al.</em> 2014</a>). However, a small number of the <em>med</em> gene predictions in the Elegans supergroup do have introns (Supplemental File S1).</p><p id="p-77">The structural conservation among the 20 Elegans supergroup MEDs and ENDs lead me to propose a model by which the MED/END cascade arose through a process of duplication and intercalation, from <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> upwards, as shown in <a id="xref-fig-11-3" class="xref-fig" href="#F11">Figure 11C</a>. This model combines gene duplications, which shape <em>Caenorhabditis</em> genomes, and the mechanism of intercalation of factors into an ancestral regulatory network (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Booth <em>et al.</em> 2010</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Lipinski <em>et al.</em> 2011</a>). I include duplication of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> to produce <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> based on preliminary data suggesting that this gene also originated at the same time as the MEDs and ENDs. Indeed, a common origin of all these upstream factors is further supported by their similar size of 174-242 amino acids, while <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> is approximately twice as large. One interpretation of this size difference could be that <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a>, as the central regulator of intestinal fate, has additional structural features unique to this role (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">McGhee <em>et al.</em> 2009</a>). In contrast, the upstream <em>med</em> and <em>end</em> factors are transiently expressed and seem to serve to robustly activate <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>, while <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> plays an accessory role with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> (<a id="xref-ref-70-16" class="xref-bibr" href="#ref-70">Maduro <em>et al.</em> 2005a</a>; <a id="xref-ref-72-12" class="xref-bibr" href="#ref-72">Maduro <em>et al.</em> 2015</a>; <a id="xref-ref-105-3" class="xref-bibr" href="#ref-105">Sommermann <em>et al.</em> 2010</a>; <a id="xref-ref-122-5" class="xref-bibr" href="#ref-122">Wiesenfahrt <em>et al.</em> 2015</a>; <a id="xref-ref-131-3" class="xref-bibr" href="#ref-131">Zhu <em>et al.</em> 1997</a>). Indeed, function of the <em>end</em>s and <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> can be replaced by early activation of just <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> alone, as mentioned earlier (<a id="xref-ref-122-6" class="xref-bibr" href="#ref-122">Wiesenfahrt <em>et al.</em> 2015</a>).</p><p id="p-78">Patterns of structural similarity among the factors upstream of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> lead to hypotheses about their origin. The similarity of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs and their tendency to be &lt;50 kbp apart in a species suggests that they originated from a common progenitor together, or that one was a duplicate of the other. Considering the stronger resemblance of the DNA-binding domain of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> with that of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and vertebrate cGATA1, a reasonable hypothesis is that <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> originated first, as a duplicate of an ancestral <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> gene that was both activated by <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> and maintained its own expression through positive autoregulation. In parallel, <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a> would be duplicated from <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> to become its paralogue. Positive autoregulation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724">ELT-7</a> is known and for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724">ELT-2</a> has even been visualized <em>in vivo</em> (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Fukushige <em>et al.</em> 1999</a>; <a id="xref-ref-105-4" class="xref-bibr" href="#ref-105">Sommermann <em>et al.</em> 2010</a>). Duplication of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> has likely occurred to generate the extant paralogous (and likely inactive) <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00001252?doi=10.1534/g3.119.400724"><em>elt-4</em></a> gene, and more significantly, <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00015981?doi=10.1534/g3.119.400724"><em>elt-7</em></a>, a paralogue of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> that shares overlapping function, expression and autoregulation with <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> (<a id="xref-ref-105-5" class="xref-bibr" href="#ref-105">Sommermann <em>et al.</em> 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Fukushige <em>et al.</em> 2003</a>). Although not necessary at this step, if the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> promoter became degenerate, the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> prototype would be stable because it would be necessary to relay input from <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> into <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a><em>,7</em>. A paralogous <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> prototype gene might then have originated as a simple linked duplication of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>. Lending support for <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> as a progenitor for the <em>end</em> genes is the presence of the conserved intron in the zinc finger coding region found in all <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a><em>/3</em> orthologs and in <em>C. elegans </em><a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>/<em>7</em>. The two <em>end</em> genes could be stabilized by the complete loss of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a> promoter, degeneracy of <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> promoter, and coevolution of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> with binding sites in the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> promoter. In this state, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> also acts to amplify input from <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> into <a href="https://identifiers.org/bioentitylink/WB:WBGene00001250?doi=10.1534/g3.119.400724"><em>elt-2</em></a>.</p><p id="p-79">A challenge is to account for the origin of a <em>med</em>-like progenitor, given the evidence that they form a structurally divergent set of regulators. In this work it was found that while the Elegans group species have intronless <em>med</em> genes, obscuring their origin, the putative Japonica group <em>med</em>s share a common intron in the zinc finger coding region that is in the same location as the aforementioned intron in all extant <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> genes. This leads to the hypothesis that a prototype <em>med</em> gene arose as a duplicate of one of these genes. The slightly higher structural similarity of the MED DBD with that of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>) suggests the prototype may have arisen from <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>, but it could also have been <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>. Co-evolution of the MED DNA-binding domain with cognate sites in <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> would reduce autoregulation of the <em>end</em> genes and fix the MED factor within the network, though <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> could retain the ability to contribute to <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a> activation. Degeneration of the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> sites in <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> would strengthen the requirement for the MED factors as they would become necessary to relay <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> input to <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a>. Further refinement of the network would strengthen regulatory input of the <em>med</em>s by <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a>, activation of <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724"><em>end-3</em></a> by the MEDs, and other regulatory inputs into <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724"><em>end-1</em></a>. Further selection on the <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> coding region might have been enforced by protein-protein interactions with other factors that contribute to gut specification.</p><p id="p-80">Although this model is highly speculative, there is supporting evidence for a similar model in evolution of the <em>Bicoid</em> (<em>Bcd</em>) gene in an ancestor to cyclorrhaphan flies, a group that includes <em>Drosophila</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Driever and Nusslein-Volhard 1989</a>; <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Stauber <em>et al.</em> 1999</a>). <em>Bcd</em> specifies anterior fates in early cyclorrhaphan embryos, while outside of this group <em>bcd</em> is not found, and other factors play an analogous role (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Lynch <em>et al.</em> 2006</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">McGregor 2005</a>). <em>Bcd</em> arose as a duplicate of the Hox gene <em>Zen</em>, and likely acquired derived DNA-binding characteristics primarily through two missense mutations in the DNA-binding domain (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Liu <em>et al.</em> 2018</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">McGregor 2005</a>). From studies in the flour beetle <em>Tribolium</em>, which lacks <em>bcd</em>, it is hypothesized that <em>Bcd</em> took over functions of some of its downstream gap gene targets, which it then became an activator of (<a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">McGregor 2005</a>). <em>Bcd</em> is proposed to have originated ∼140 Mya at the base of the Cyclorrhapha, a longer time period than the estimated tens of millions of years since the common ancestor to the Elegans supergroup (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">Wiegmann <em>et al.</em> 2011</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Coghlan and Wolfe 2002</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Cutter 2008</a>). Recruitment of <em>Bcd</em> into A/P specification in <em>Drosophila</em> likely required more steps than the MED/END cascade, because in my proposed model for <em>C. elegans</em> endoderm specification, the cascade originated through duplication and modification of a factors already in an ancestral version of the network. Hence, it is plausible that emergence of the MED/END network could have occurred at the base of the Elegans supergroup on a shorter evolutionary time scale. Furthermore, in analogy to <em>Bcd</em>, the initial evolution of the MED DBD that resulted in a change in its binding site to a non-GATA target site might have been driven by a small number (or even just one) change(s) in a key amino acid. With the sequences of <em>med</em> genes from 20 species, such structure-function correlations can now be examined.</p><p id="p-81">Studies on the evolution of <em>Bcd</em> suggest a possible explanation as to why a more layered gene cascade might have evolved for embryonic gut specification within the Elegans supergroup. The emergence of <em>Bcd</em> may have conferred a more rapid specification of segment identity, allowing developmental time to become faster without sacrificing robustness (<a id="xref-ref-79-4" class="xref-bibr" href="#ref-79">McGregor 2005</a>). By extension to the Elegans supergroup, it is possible that the <a href="https://identifiers.org/bioentitylink/WB:WBGene00004804?doi=10.1534/g3.119.400724">SKN-1</a> → MED → <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a>,3 gene regulatory cascade coincided with an increase in developmental speed in <em>Caenorhabditis</em>, perhaps as part of the transition to very early and rapid cell fate specification (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Schierenberg 2001</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Laugsch and Schierenberg 2004</a>). Elucidation of gut specification mechanisms in <em>Caenorhabditis</em> species outside of the Elegans supergroup, compared with their developmental speed, could provide evidence for this hypothesis, or alternatively identify non-GATA factors that play the same role as the MED/END cascade.</p><p id="p-82">In the meanwhile, the identification of MED, <a href="https://identifiers.org/bioentitylink/WB:WBGene00001311?doi=10.1534/g3.119.400724">END-3</a> and <a href="https://identifiers.org/bioentitylink/WB:WBGene00001310?doi=10.1534/g3.119.400724">END-1</a> orthologs in 20 species sets the stage for studies to test hypotheses about evolution of gene regulatory networks, structure-function correlations in the evolution of novel DNA-binding domains, and features of developmental system drift. As the study of gene regulatory networks becomes more computational, the set of MED and END orthologs identified here will provide a basis for future studies integrating gene network architecture with transcriptomics data, for example (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Omranian and Nikoloski 2017</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Nomoto <em>et al.</em> 2019</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-83">I am indebted to Mark Blaxter, Lewis Stephens and colleagues at the <em>Caenorhabditis</em> Genomes Project in Edinburgh for prepublication access to the genome sequences of <em>Caenorhabditis</em> species and for advice during this work. I also thank Eric Haag (University of Maryland, College Park) for helpful advice in interpretation of search results. I am similarly grateful to the anonymous reviewers for their very helpful insights and suggestions. Earlier versions of this work were completed under my NSF grant IOS#1258054. I also thank Christian Turner, a former UCR undergraduate supported by NIH Award T34GM062756 from the National Institute of General Medical Sciences (MARCU-STAR) program to UC Riverside, for having performed preliminary searches of available <em>Caenorhabditis</em> sequences in 2014.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-84">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9820622">https://doi.org/10.25387/g3.9820622</a>.</p></li><li class="fn" id="fn-2"><p id="p-85"><em>Communicating editor: M.-A. Félix</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-12"><span class="received-label">Received </span>September 12, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-15"><span class="accepted-label">Accepted </span>November 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Maduro</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.1" data-doi="10.1101/gr.113811.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. W.</span> <span class="cit-name-surname">Hillier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Waterston</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Blumenthal</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A global analysis of C. elegans trans-splicing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.113811.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DAllen%26rft.auinit1%253DM.%2BA.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D255%26rft.epage%253D264%26rft.atitle%253DA%2Bglobal%2Banalysis%2Bof%2BC.%2Belegans%2Btrans-splicing.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113811.110%26rft_id%253Dinfo%253Apmid%252F21177958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.113811.110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21177958&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286804100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.2" data-doi="10.1101/gad.1107803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">J. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. K.</span> <span class="cit-name-surname">Blackwell</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">SKN-1 links C. elegans mesendodermal specification to a conserved oxidative stress response.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1882</span>–<span class="cit-lpage">1893</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1107803</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1107803%26rft_id%253Dinfo%253Apmid%252F12869585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE3LzE1LzE4ODIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">T. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Elkan</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Fitting a mixture model by expectation maximization to discover motifs in biopolymers.</span> <abbr class="cit-jnl-abbrev">Proc. Int. Conf. Intell. Syst. Mol. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2B%252F%2B...%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%2B%253B%2BISMB.%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%26rft.stitle%253DProc%2BInt%2BConf%2BIntell%2BSyst%2BMol%2BBiol%26rft.aulast%253DBailey%26rft.auinit1%253DT.%2BL.%26rft.volume%253D2%26rft.spage%253D28%26rft.epage%253D36%26rft.atitle%253DFitting%2Ba%2Bmixture%2Bmodel%2Bby%2Bexpectation%2Bmaximization%2Bto%2Bdiscover%2Bmotifs%2Bin%2Bbiopolymers.%26rft_id%253Dinfo%253Apmid%252F7584402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7584402&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.4" data-doi="10.1101/gr.081851.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barriere</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Pekarek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Haag</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detecting heterozygosity in shotgun genome assemblies: Lessons from obligately outcrossing nematodes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.081851.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081851.108%26rft_id%253Dinfo%253Apmid%252F19204328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNDcwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.5" data-doi="10.1242/dev.00302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baugh</span>,  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Slonim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Brown</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Hunter</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Composition and dynamics of the Caenorhabditis elegans early embryonic transcriptome.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">900</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.00302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DBaugh%26rft.auinit1%253DL.%2BR.%26rft.volume%253D130%26rft.issue%253D5%26rft.spage%253D889%26rft.epage%253D900%26rft.atitle%253DComposition%2Band%2Bdynamics%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bearly%2Bembryonic%2Btranscriptome%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.00302%26rft_id%253Dinfo%253Apmid%252F12538516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMzAvNS84ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.6" data-doi="10.1128/MCB.12.12.5673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergemann</span>,  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. W.</span> <span class="cit-name-surname">Ma</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Johnson</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Sequence of cDNA comprising the human pur gene and sequence-specific single-stranded-DNA-binding properties of the encoded protein.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">5673</span>–<span class="cit-lpage">5682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.12.12.5673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBergemann%26rft.auinit1%253DA%2BD%26rft.volume%253D12%26rft.issue%253D12%26rft.spage%253D5673%26rft.epage%253D5682%26rft.atitle%253DSequence%2Bof%2BcDNA%2Bcomprising%2Bthe%2Bhuman%2Bpur%2Bgene%2Band%2Bsequence-specific%2Bsingle-stranded-DNA-binding%2Bproperties%2Bof%2Bthe%2Bencoded%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.12.12.5673%26rft_id%253Dinfo%253Apmid%252F1448097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMi81NjczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.7" data-doi="10.1371/journal.pgen.1004133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhambhani</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Ravindranath</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Mentink</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Betist</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Distinct DNA binding sites contribute to the TCF transcriptional switch in C. elegans and Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004133%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004133%26rft_id%253Dinfo%253Apmid%252F24516405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24516405&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.8" data-doi="10.1126/science.7939715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blackwell</span>,  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bowerman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Priess</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Weintraub</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Formation of a monomeric DNA binding domain by Skn-1 bZIP and homeodomain elements.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">628</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.7939715</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBlackwell%26rft.auinit1%253DT.%26rft.volume%253D266%26rft.issue%253D5185%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DFormation%2Bof%2Ba%2Bmonomeric%2BDNA%2Bbinding%2Bdomain%2Bby%2BSkn-1%2BbZIP%2Band%2Bhomeodomain%2Belements%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7939715%26rft_id%253Dinfo%253Apmid%252F7939715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjYvNTE4NS82MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.9" data-doi="10.1016/j.ydbio.2011.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeck</span>,  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Boyle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Bao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Murray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Mericle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Specific roles for the GATA transcription factors end-1 and end-3 during C. elegans E-lineage development.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">358</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2011.08.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DBoeck%26rft.auinit1%253DM.%2BE.%26rft.volume%253D358%26rft.issue%253D2%26rft.spage%253D345%26rft.epage%253D355%26rft.atitle%253DSpecific%2Broles%2Bfor%2Bthe%2BGATA%2Btranscription%2Bfactors%2Bend-1%2Band%2Bend-3%2Bduring%2BC.%2Belegans%2BE-lineage%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2011.08.002%26rft_id%253Dinfo%253Apmid%252F21854766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2011.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21854766&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.10" data-doi="10.1038/nature09560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Booth</span>,  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. B.</span> <span class="cit-name-surname">Tuch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Johnson</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Intercalation of a new tier of transcription regulation into an ancient circuit.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09560</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBooth%26rft.auinit1%253DL.%2BN.%26rft.volume%253D468%26rft.issue%253D7326%26rft.spage%253D959%26rft.epage%253D963%26rft.atitle%253DIntercalation%2Bof%2Ba%2Bnew%2Btier%2Bof%2Btranscription%2Bregulation%2Binto%2Ban%2Bancient%2Bcircuit.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09560%26rft_id%253Dinfo%253Apmid%252F21164485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21164485&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285344600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boveri</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1892</span> <span class="cit-article-title">Ueber die Entstehung des Gegensatzes zwischen den Geschlechtszellen und den somatischen Zellen bei Ascaris megalocephala, nebst Bemerkungen zur Entwicklungsgeschichte der Nematoden.</span> <abbr class="cit-jnl-abbrev">Sitzungsberichte der Gesellschaft für Morphologie und Physiologie in München</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSitzungsberichte%2Bder%2BGesellschaft%2Bf%25FCr%2BMorphologie%2Bund%2BPhysiologie%2Bin%2BM%25FCnchen%26rft.volume%253D8%26rft.spage%253D114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.12" data-doi="10.1016/0092-8674(92)90078-Q"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowerman</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Eaton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Priess</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">skn-1, a maternally expressed gene required to specify the fate of ventral blastomeres in the early C. elegans embryo.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(92)90078-Q</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBowerman%26rft.auinit1%253DB.%26rft.volume%253D68%26rft.issue%253D6%26rft.spage%253D1061%26rft.epage%253D1075%26rft.atitle%253Dskn-1%252C%2Ba%2Bmaternally%2Bexpressed%2Bgene%2Brequired%2Bto%2Bspecify%2Bthe%2Bfate%2Bof%2Bventral%2Bblastomeres%2Bin%2Bthe%2Bearly%2BC.%2Belegans%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990078-Q%26rft_id%253Dinfo%253Apmid%252F1547503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90078-Q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1547503&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HK67400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.13" data-doi="10.1242/dev.02475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broitman-Maduro</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. T.-H.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Hung</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Maduro</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Specification of the C. elegans MS blastomere by the T-box factor TBX-35.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">3097</span>–<span class="cit-lpage">3106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.02475</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.02475%26rft_id%253Dinfo%253Apmid%252F16831832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMzLzE2LzMwOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.14" data-doi="10.1016/j.devcel.2005.01.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broitman-Maduro</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Maduro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The noncanonical binding site of the MED-1 GATA factor defines differentially regulated target genes in the C. elegans mesendoderm.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2005.01.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DBroitman-Maduro%26rft.auinit1%253DG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D427%26rft.epage%253D433%26rft.atitle%253DThe%2Bnoncanonical%2Bbinding%2Bsite%2Bof%2Bthe%2BMED-1%2BGATA%2Bfactor%2Bdefines%2Bdifferentially%2Bregulated%2Btarget%2Bgenes%2Bin%2Bthe%2BC.%2Belegans%2Bmesendoderm.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2005.01.014%26rft_id%253Dinfo%253Apmid%252F15737937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2005.01.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15737937&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.15" data-doi="10.1242/dev.038307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broitman-Maduro</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Owraghi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Hung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kuntz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Sternberg</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The NK-2 class homeodomain factor CEH-51 and the T-box factor TBX-35 have overlapping function in C. elegans mesoderm development.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">2735</span>–<span class="cit-lpage">2746</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.038307</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.038307%26rft_id%253Dinfo%253Apmid%252F19605496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM2LzE2LzI3MzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.16" data-doi="10.1101/gad.11.17.2227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>,  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Gilbert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Cheung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Michnowicz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">SKN-1 domain folding and basic region monomer stabilization upon DNA binding.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2227</span>–<span class="cit-lpage">2238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.11.17.2227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DCarroll%26rft.auinit1%253DA.%2BS.%26rft.volume%253D11%26rft.issue%253D17%26rft.spage%253D2227%26rft.epage%253D2238%26rft.atitle%253DSKN-1%2Bdomain%2Bfolding%2Band%2Bbasic%2Bregion%2Bmonomer%2Bstabilization%2Bupon%2BDNA%25A0binding%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.17.2227%26rft_id%253Dinfo%253Apmid%252F9303538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzE3LzIyMjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.17" data-doi="10.1016/j.ydbio.2017.05.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Broitman-Maduro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Maduro</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Partially compromised specification causes stochastic effects on gut development in C. elegans.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">427</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2017.05.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D427%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2017.05.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2017.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.18" data-doi="10.1101/gr.172702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coghlan</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Wolfe</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Fourfold faster rate of genome rearrangement in nematodes than in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">867</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.172702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCoghlan%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D857%26rft.epage%253D867%26rft.atitle%253DFourfold%2BFaster%2BRate%2Bof%2BGenome%2BRearrangement%2Bin%2BNematodes%2BThan%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.172702%26rft_id%253Dinfo%253Apmid%252F12045140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvODU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.19" data-doi="10.1016/j.ydbio.2005.10.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coroian</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Broitman-Maduro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Maduro</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Med-type GATA factors and the evolution of mesendoderm specification in nematodes.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2005.10.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DCoroian%26rft.auinit1%253DC.%26rft.volume%253D289%26rft.issue%253D2%26rft.spage%253D444%26rft.epage%253D455%26rft.atitle%253DMed-type%2BGATA%2Bfactors%2Band%2Bthe%2Bevolution%2Bof%2Bmesendoderm%2Bspecification%2Bin%2Bnematodes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2005.10.024%26rft_id%253Dinfo%253Apmid%252F16325171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2005.10.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16325171&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.20" data-doi="10.1016/j.molbiopara.2003.10.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Couthier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">McGarr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Craig</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Gilleard</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Ectopic expression of a Haemonchus contortus GATA transcription factor in Caenorhabditis elegans reveals conserved function in spite of extensive sequence divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Biochem. Parasitol.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molbiopara.2003.10.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DCouthier%26rft.auinit1%253DA.%26rft.volume%253D133%26rft.issue%253D2%26rft.spage%253D241%26rft.epage%253D253%26rft.atitle%253DEctopic%2Bexpression%2Bof%2Ba%2BHaemonchus%2Bcontortus%2BGATA%2Btranscription%2Bfactor%2Bin%2BCaenorhabditis%2Belegans%2Breveals%2Bconserved%2Bfunction%2Bin%2Bspite%2Bof%2Bextensive%2Bsequence%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molbiopara.2003.10.012%26rft_id%253Dinfo%253Apmid%252F14698436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molbiopara.2003.10.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14698436&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.21" data-doi="10.1093/molbev/msn024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>,  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Divergence times in Caenorhabditis and Drosophila inferred from direct estimates of the neutral mutation rate.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">778</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msn024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn024%26rft_id%253Dinfo%253Apmid%252F18234705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msn024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18234705&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254004900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.22" data-doi="10.1002/evl3.122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>,  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Garrett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sun</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Molecular evolution across developmental time reveals rapid divergence in early embryogenesis.</span> <abbr class="cit-jnl-abbrev">Evol Lett</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">373</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/evl3.122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevl3.122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evl3.122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.23" data-doi="10.1016/j.molcel.2018.06.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darnell</span>,  <span class="cit-name-given-names">A.M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.R.</span> <span class="cit-name-surname">Subramaniam</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.K.</span> <span class="cit-name-surname">O’Shea</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Translational Control through Differential Ribosome Pausing during Amino Acid Limitation in Mammalian Cells.</span> <abbr class="cit-jnl-abbrev">Mol Cell</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">243 e211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2018.06.041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%26rft.volume%253D71%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2018.06.041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2018.06.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.24" data-doi="10.1038/nature09645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Emerging properties of animal gene regulatory networks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">911</span>–<span class="cit-lpage">920</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09645</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDavidson%26rft.auinit1%253DE.%2BH.%26rft.volume%253D468%26rft.issue%253D7326%26rft.spage%253D911%26rft.epage%253D920%26rft.atitle%253DEmerging%2Bproperties%2Bof%2Banimal%2Bgene%2Bregulatory%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09645%26rft_id%253Dinfo%253Apmid%252F21164479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21164479&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285344600037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.25" data-doi="10.1006/dbio.2002.0635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Rast</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Oliveri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ransick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Calestani</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A provisional regulatory gene network for specification of endomesoderm in the sea urchin embryo.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">246</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.2002.0635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DDavidson%26rft.auinit1%253DE.%2BH.%26rft.volume%253D246%26rft.issue%253D1%26rft.spage%253D162%26rft.epage%253D190%26rft.atitle%253DA%2Bprovisional%2Bregulatory%2Bgene%2Bnetwork%2Bfor%2Bspecification%2Bof%2Bendomesoderm%2Bin%2Bthe%2Bsea%2Burchin%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.2002.0635%26rft_id%253Dinfo%253Apmid%252F12027441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.2002.0635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027441&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176065900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.26" data-doi="10.1038/ng.227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dieterich</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. W.</span> <span class="cit-name-surname">Clifton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. N.</span> <span class="cit-name-surname">Schuster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chinwalla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Delehaunty</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Pristionchus pacificus genome provides a unique perspective on nematode lifestyle and parasitism.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDieterich%26rft.auinit1%253DC.%26rft.volume%253D40%26rft.issue%253D10%26rft.spage%253D1193%26rft.epage%253D1198%26rft.atitle%253DThe%2BPristionchus%2Bpacificus%2Bgenome%2Bprovides%2Ba%2Bunique%2Bperspective%2Bon%2Bnematode%2Blifestyle%2Band%2Bparasitism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.227%26rft_id%253Dinfo%253Apmid%252F18806794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18806794&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259651000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.27" data-doi="10.1016/j.ydbio.2017.12.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dineen</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Osborne Nishimura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Goszczynski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">McGhee</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Quantitating transcription factor redundancy: The relative roles of the ELT-2 and ELT-7 GATA factors in the C. elegans endoderm.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">435</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2017.12.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D435%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2017.12.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2017.12.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.28" data-doi="10.1038/337138a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Driever</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Nusslein-Volhard</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The bicoid protein is a positive regulator of hunchback transcription in the early Drosophila embryo.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/337138a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDriever%26rft.auinit1%253DW.%26rft.volume%253D337%26rft.issue%253D6203%26rft.spage%253D138%26rft.epage%253D143%26rft.atitle%253DThe%2Bbicoid%2Bprotein%2Bis%2Ba%2Bpositive%2Bregulator%2Bof%2Bhunchback%2Btranscription%2Bin%2Bthe%2Bearly%2BDrosophila%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F337138a0%26rft_id%253Dinfo%253Apmid%252F2911348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/337138a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2911348&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.29" data-doi="10.1016/j.ydbio.2016.02.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tracy</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Rifkin</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mutagenesis of GATA motifs controlling the endoderm regulator elt-2 reveals distinct dominant and secondary cis-regulatory elements.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">412</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2016.02.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D412%26rft.spage%253D160%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2016.02.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2016.02.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.30" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.31" data-doi="10.12703/P6-62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">R. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Y.</span> <span class="cit-name-surname">Lin</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The evolutionary origins and consequences of self-fertility in nematodes.</span> <abbr class="cit-jnl-abbrev">F1000Prime Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12703/P6-62</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000Prime%2BRep.%26rft.volume%253D6%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.12703%252FP6-62%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12703/P6-62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.32" data-doi="10.1093/nar/gkv1511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etheve</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Martin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lavery</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Dynamics and recognition within a protein-DNA complex: a molecular dynamics study of the SKN-1/DNA interaction.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1440</span>–<span class="cit-lpage">1448</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1511</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1511%26rft_id%253Dinfo%253Apmid%252F26721385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26721385&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.33" data-doi="10.1016/j.cub.2006.12.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Félix</span>,  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cryptic quantitative evolution of the vulva intercellular signaling network in Caenorhabditis.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2006.12.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DFelix%26rft.auinit1%253DM.%2BA.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D103%26rft.epage%253D114%26rft.atitle%253DCryptic%2Bquantitative%2Bevolution%2Bof%2Bthe%2Bvulva%2Bintercellular%2Bsignaling%2Bnetwork%2Bin%2BCaenorhabditis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.12.024%26rft_id%253Dinfo%253Apmid%252F17240335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.12.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17240335&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243811200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.34" data-doi="10.1371/journal.pone.0094723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Félix</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Braendle</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Cutter</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A streamlined system for species diagnosis in Caenorhabditis (Nematoda: Rhabditidae) with name designations for 15 distinct biological species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e94723</span>. Erratum: 10: e0118327.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0094723</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De94723%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0094723%26rft_id%253Dinfo%253Apmid%252F24727800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0094723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24727800&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.35" data-doi="10.1007/s10709-008-9311-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Force</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. B.</span> <span class="cit-name-surname">Pickett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amores</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. L.</span> <span class="cit-name-surname">Yan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Preservation of duplicate genes by complementary, degenerative mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1531</span>–<span class="cit-lpage">1545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DForce%26rft.auinit1%253DA.%26rft.volume%253D151%26rft.issue%253D4%26rft.spage%253D1531%26rft.epage%253D1545%26rft.atitle%253DPreservation%2Bof%2BDuplicate%2BGenes%2Bby%2BComplementary%252C%2BDegenerative%2BMutations%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9311-5%26rft_id%253Dinfo%253Apmid%252F10101175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1MS80LzE1MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.36" data-doi="10.1016/j.cell.2016.05.072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frøkjær-Jensen</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Jain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Hansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An Abundant Class of Non-coding DNA Can Prevent Stochastic Gene Silencing in the C. elegans Germline.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">357</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.05.072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.072%26rft_id%253Dinfo%253Apmid%252F27374334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27374334&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fukushige</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Goszczynski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tian</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">McGhee</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Evolutionary Duplication and Probable Demise of an Endodermal GATA Factor in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFukushige%26rft.auinit1%253DT.%26rft.volume%253D165%26rft.issue%253D2%26rft.spage%253D575%26rft.epage%253D588%26rft.atitle%253DThe%2BEvolutionary%2BDuplication%2Band%2BProbable%2BDemise%2Bof%2Ban%2BEndodermal%2BGATA%2BFactor%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F14573471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzIvNTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.38" data-doi="10.1016/s0012-1606(98)80006-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fukushige</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Hawkins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">McGhee</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The GATA-factor elt-2 is essential for formation of the Caenorhabditis elegans intestine.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DFukushige%26rft.auinit1%253DT.%26rft.volume%253D198%26rft.issue%253D2%26rft.spage%253D286%26rft.epage%253D302%26rft.atitle%253DThe%2BGATA-factor%2Belt-2%2Bis%2Bessential%2Bfor%2Bformation%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bintestine.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0012-1606%252898%252980006-7%26rft_id%253Dinfo%253Apmid%252F9659934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0012-1606(98)80006-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9659934&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074576400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.39" data-doi="10.1073/pnas.96.21.11883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fukushige</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Hendzel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Bazett-Jones</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">McGhee</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Direct visualization of the elt-2 gut-specific GATA factor binding to a target promoter inside the living Caenorhabditis elegans embryo.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">11883</span>–<span class="cit-lpage">11888</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.96.21.11883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFukushige%26rft.auinit1%253DT.%26rft.volume%253D96%26rft.issue%253D21%26rft.spage%253D11883%26rft.epage%253D11888%26rft.atitle%253DDirect%2Bvisualization%2Bof%2Bthe%2Belt-2%2Bgut-specific%2BGATA%2Bfactor%2Bbinding%2Bto%2Ba%2Btarget%2Bpromoter%2Binside%2Bthe%2Bliving%2BCaenorhabditis%2Belegans%2Bembryo%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.21.11883%26rft_id%253Dinfo%253Apmid%252F10518545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjEvMTE4ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.40" data-doi="10.1371/journal.pbio.0020352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaudet</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Muttumu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Horner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Mango</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Whole-genome analysis of temporal gene expression during foregut development.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0020352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGaudet%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253De352%26rft.epage%253De352%26rft.atitle%253DWhole-genome%2Banalysis%2Bof%2Btemporal%2Bgene%2Bexpression%2Bduring%2Bforegut%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020352%26rft_id%253Dinfo%253Apmid%252F15492775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15492775&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.41" data-doi="10.1186/1471-2148-8-112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>,  <span class="cit-name-given-names">W. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Bowerman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Q.</span> <span class="cit-name-surname">Schneider</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The evolution of protostome GATA factors: molecular phylogenetics, synteny, and intron/exon structure reveal orthologous relationships.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-8-112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DGillis%26rft.auinit1%253DW.%2BQ.%26rft.volume%253D8%26rft.spage%253D112%26rft.epage%253D112%26rft.atitle%253DThe%2Bevolution%2Bof%2Bprotostome%2BGATA%2Bfactors%253A%2Bmolecular%2Bphylogenetics%252C%2Bsynteny%252C%2Band%2Bintron%252Fexon%2Bstructure%2Breveal%2Borthologous%2Brelationships.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-112%26rft_id%253Dinfo%253Apmid%252F18412965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18412965&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.42" data-doi="10.1186/1471-2148-9-207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>,  <span class="cit-name-given-names">W. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">St John</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bowerman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Q.</span> <span class="cit-name-surname">Schneider</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole genome duplications and expansion of the vertebrate GATA transcription factor gene family.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-9-207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DGillis%26rft.auinit1%253DW.%2BQ.%26rft.volume%253D9%26rft.spage%253D207%26rft.epage%253D207%26rft.atitle%253DWhole%2Bgenome%2Bduplications%2Band%2Bexpansion%2Bof%2Bthe%2Bvertebrate%2BGATA%2Btranscription%2Bfactor%2Bgene%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-9-207%26rft_id%253Dinfo%253Apmid%252F19695090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-9-207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19695090&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.43" data-doi="10.1101/gr.113381.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grishkevich</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hashimshony</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Yanai</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Core promoter T-blocks correlate with gene expression levels in C. elegans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.113381.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113381.110%26rft_id%253Dinfo%253Apmid%252F21367940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvNzA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.44" data-doi="10.7554/eLife.02181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guilgur</span>,  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Prudencio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sobral</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Liszekova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rosa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Requirement for highly efficient pre-mRNA splicing during Drosophila early embryonic development.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e02181</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.02181</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.02181%26rft_id%253Dinfo%253Apmid%252F24755291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.02181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24755291&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.45" data-doi="10.1534/genetics.118.300243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haag</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H. A.</span> <span class="cit-name-surname">Fitch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Delattre</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">From “the Worm” to “the Worms” and Back Again: The Evolutionary Developmental Biology of Nematodes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300243</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300243%26rft_id%253Dinfo%253Apmid%252F30287515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzIvMzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.46" data-doi="10.1007/978-1-4939-2842-2_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haag</span>,  <span class="cit-name-given-names">E. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Thomas</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fundamentals of Comparative Genome Analysis in Caenorhabditis Nematodes.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1327</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-2842-2_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1327%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2842-2_2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2842-2_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.47" data-doi="10.1371/journal.pgen.1002464"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sieburth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Kaplan</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">RIC-7 promotes neuropeptide secretion.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002464</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHao%26rft.auinit1%253DY.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1002464%26rft.epage%253De1002464%26rft.atitle%253DRIC-7%2Bpromotes%2Bneuropeptide%2Bsecretion.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002464%26rft_id%253Dinfo%253Apmid%252F22275875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22275875&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.48" data-doi="10.1016/S0012-1606(03)00101-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houthoofd</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Jacobsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Mertens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vangestel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Coomans</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Embryonic cell lineage of the marine nematode Pellioditis marina.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">258</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">69</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0012-1606(03)00101-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DHouthoofd%26rft.auinit1%253DW.%26rft.volume%253D258%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D69%26rft.atitle%253DEmbryonic%2Bcell%2Blineage%2Bof%2Bthe%2Bmarine%2Bnematode%2BPellioditis%2Bmarina.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0012-1606%252803%252900101-5%26rft_id%253Dinfo%253Apmid%252F12781682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0012-1606(03)00101-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12781682&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183380300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.49" data-doi="10.1016/S0092-8674(00)81340-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>,  <span class="cit-name-given-names">C. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kenyon</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Spatial and temporal controls target pal-1 blastomere-specification activity to a single blastomere lineage in C. elegans embryos.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)81340-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHunter%26rft.auinit1%253DC.%2BP.%26rft.volume%253D87%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D226%26rft.atitle%253DSpatial%2Band%2Btemporal%2Bcontrols%2Btarget%2Bpal-1%2Bblastomere-specification%2Bactivity%2Bto%2Ba%2Bsingle%2Bblastomere%2Blineage%2Bin%2BC.%2Belegans%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981340-9%26rft_id%253Dinfo%253Apmid%252F8861906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81340-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8861906&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VN40300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.50" data-doi="10.1074/jbc.M112.370437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Kobayashi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yamamoto</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Shimizu</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">N- and C-terminal transactivation domains of GATA1 protein coordinate hematopoietic program.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">21439</span>–<span class="cit-lpage">21449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M112.370437</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.370437%26rft_id%253Dinfo%253Apmid%252F22556427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvMjUvMjE0MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.51" data-doi="10.1101/gr.10.8.1115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Zahler</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Conservation, regulation, synteny, and introns in a large-scale C. briggsae-C. elegans genomic alignment.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.8.1115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DKent%26rft.auinit1%253DW.%2BJ.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D1115%26rft.epage%253D1125%26rft.atitle%253DConservation%252C%2BRegulation%252C%2BSynteny%252C%2Band%2BIntrons%2Bin%2Ba%2BLarge-scale%2BC.%2Bbriggsae-C.%2Belegans%2BGenomic%2BAlignment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.8.1115%26rft_id%253Dinfo%253Apmid%252F10958630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEwLzgvMTExNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.52" data-doi="10.1186/1471-2148-11-339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiontke</span>,  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Felix</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ailion</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Braendle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A phylogeny and molecular barcodes for Caenorhabditis, with numerous new species from rotting fruits.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-11-339</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DKiontke%26rft.auinit1%253DK.%2BC.%26rft.volume%253D11%26rft.spage%253D339%26rft.epage%253D339%26rft.atitle%253DA%2Bphylogeny%2Band%2Bmolecular%2Bbarcodes%2Bfor%2BCaenorhabditis%252C%2Bwith%2Bnumerous%2Bnew%2Bspecies%2Bfrom%2Brotting%2Bfruits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-339%26rft_id%253Dinfo%253Apmid%252F22103856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22103856&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.53" data-doi="10.1073/pnas.1801930115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konrad</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Flibotte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Taylor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Waterston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Moerman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mutational and transcriptional landscape of spontaneous gene duplications and deletions in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">7386</span>–<span class="cit-lpage">7391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1801930115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1801930115%26rft_id%253Dinfo%253Apmid%252F29941601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzI4LzczODYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.54" data-doi="10.1007/s00239-010-9425-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">Z.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. N.</span> <span class="cit-name-surname">Trifonov</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Role of everlasting triplet expansions in protein evolution.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-010-9425-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKoren%26rft.auinit1%253DZ.%26rft.volume%253D72%26rft.issue%253D2%26rft.spage%253D232%26rft.epage%253D239%26rft.atitle%253DRole%2Bof%2Beverlasting%2Btriplet%2Bexpansions%2Bin%2Bprotein%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9425-0%26rft_id%253Dinfo%253Apmid%252F21161200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-010-9425-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21161200&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.55" data-doi="10.1093/bioinformatics/17.suppl_1.S140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Flicek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Duan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Brent</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Integrating genomic homology into gene structure prediction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">S140</span>–<span class="cit-lpage">S148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.suppl_1.S140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D17%26rft.issue%253D90001%26rft.spage%253DS140%26rft.epage%253D148%26rft.atitle%253DIntegrating%2Bgenomic%2Bhomology%2Binto%2Bgene%2Bstructure%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.suppl_1.S140%26rft_id%253Dinfo%253Apmid%252F11473003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.suppl_1.S140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11473003&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.56" data-doi="10.1093/bioinformatics/btz305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozlov</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Darriba</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Flouri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Morel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stamatakis</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">RAxML-NG: A fast, scalable, and user-friendly tool for maximum likelihood phylogenetic inference.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">4453</span>–<span class="cit-lpage">4455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btz305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtz305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btz305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.57" data-doi="10.1093/molbev/msy096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stecher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Knyaz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tamura</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1547</span>–<span class="cit-lpage">1549</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29722887&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.58" data-doi="10.1038/s41588-019-0411-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W. H.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sasse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Cowley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Albu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Similarity regression predicts evolution of transcription factor sequence specificity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">981</span>–<span class="cit-lpage">989</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-019-0411-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D51%26rft.spage%253D981%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-019-0411-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-019-0411-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.59" data-doi="10.1387/ijdb.031758ml"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laugsch</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Schierenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Differences in maternal supply and early development of closely related nematode species.</span> <abbr class="cit-jnl-abbrev">Int. J. Dev. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1387/ijdb.031758ml</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BInternational%2Bjournal%2Bof%2Bdevelopmental%2Bbiology%26rft.stitle%253DInt%2BJ%2BDev%2BBiol%26rft.aulast%253DLaugsch%26rft.auinit1%253DM.%26rft.volume%253D48%26rft.issue%253D7%26rft.spage%253D655%26rft.epage%253D662%26rft.atitle%253DDifferences%2Bin%2Bmaternal%2Bsupply%2Band%2Bearly%2Bdevelopment%2Bof%2Bclosely%2Brelated%2Bnematode%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1387%252Fijdb.031758ml%26rft_id%253Dinfo%253Apmid%252F15470638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1387/ijdb.031758ml&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15470638&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225032600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.60" data-doi="10.1016/j.devcel.2012.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hashimshony</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wagner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Yanai</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Developmental milestones punctuate gene expression in the Caenorhabditis embryo.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2012.04.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DLevin%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D1101%26rft.epage%253D1108%26rft.atitle%253DDevelopmental%2Bmilestones%2Bpunctuate%2Bgene%2Bexpression%2Bin%2Bthe%2BCaenorhabditis%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2012.04.004%26rft_id%253Dinfo%253Apmid%252F22560298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2012.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22560298&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304291700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.61" data-doi="10.1016/j.ydbio.2008.10.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Broitman-Maduro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. W.</span> <span class="cit-name-surname">Hung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cervantes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Maduro</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Knockdown of SKN-1 and the Wnt effector TCF/POP-1 reveals differences in endomesoderm specification in C. briggsae as compared with C. elegans.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">296</span>–<span class="cit-lpage">306</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2008.10.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DLin%26rft.auinit1%253DK.%2BT.%26rft.volume%253D325%26rft.issue%253D1%26rft.spage%253D296%26rft.epage%253D306%26rft.atitle%253DKnockdown%2Bof%2BSKN-1%2Band%2Bthe%2BWnt%2Beffector%2BTCF%252FPOP-1%2Breveals%2Bdifferences%2Bin%2Bendomesoderm%2Bspecification%2Bin%2BC.%2Bbriggsae%2Bas%2Bcompared%2Bwith%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2008.10.001%26rft_id%253Dinfo%253Apmid%252F18977344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2008.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18977344&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262459600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.62" data-doi="10.1016/0092-8674(95)90100-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Thompson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Priess</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">pop-1 encodes an HMG box protein required for the specification of a mesoderm precursor in early C. elegans embryos.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">609</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(95)90100-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLin%26rft.auinit1%253DR.%26rft.volume%253D83%26rft.issue%253D4%26rft.spage%253D599%26rft.epage%253D609%26rft.atitle%253Dpop-1%2Bencodes%2Ban%2BHMG%2Bbox%2Bprotein%2Brequired%2Bfor%2Bthe%2Bspecification%2Bof%2Ba%2Bmesoderm%2Bprecursor%2Bin%2Bearly%2BC.%2Belegans%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990100-0%26rft_id%253Dinfo%253Apmid%252F7585963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90100-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7585963&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TF24800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.63" data-doi="10.1016/j.cub.2011.01.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipinski</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Farslow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Fitzpatrick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Katju</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High spontaneous rate of gene duplication in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2011.01.026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DLipinski%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D306%26rft.epage%253D310%26rft.atitle%253DHigh%2Bspontaneous%2Brate%2Bof%2Bgene%2Bduplication%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.01.026%26rft_id%253Dinfo%253Apmid%252F21295484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.01.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21295484&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.64" data-doi="10.7554/eLife.34594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Onal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Datta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Rogers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Schmidt-Ott</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Ancient mechanisms for the evolution of the bicoid homeodomain’s function in fly development.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e34594</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.34594</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D7%26rft.spage%253De34594%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.34594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.34594&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.65" data-doi="10.1073/pnas.95.15.8455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>,  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Pelczer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Walker</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The solution structure of the DNA-binding domain of Skn-1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">8455</span>–<span class="cit-lpage">8460</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.95.15.8455</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLo%26rft.auinit1%253DM.-C.%26rft.volume%253D95%26rft.issue%253D15%26rft.spage%253D8455%26rft.epage%253D8460%26rft.atitle%253DThe%2Bsolution%2Bstructure%2Bof%2Bthe%2BDNA-binding%2Bdomain%2Bof%2BSkn-1%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.15.8455%26rft_id%253Dinfo%253Apmid%252F9671699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTUvODQ1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.66" data-doi="10.1534/g3.115.021451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yochem</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Chuang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sugioka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Connolly</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-Throughput Cloning of Temperature-Sensitive Caenorhabditis elegans Mutants with Adult Syncytial Germline Membrane Architecture Defects.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2241</span>–<span class="cit-lpage">2255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021451</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021451%26rft_id%253Dinfo%253Apmid%252F26311651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTEvMjI0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.67" data-doi="10.1007/s002399910012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>,  <span class="cit-name-given-names">J. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. R.</span> <span class="cit-name-surname">Atchley</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Molecular evolution of the GATA family of transcription factors: conservation within the DNA-binding domain.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s002399910012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DLowry%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D103%26rft.epage%253D115%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bthe%2BGATA%2Bfamily%2Bof%2Btranscription%2Bfactors%253A%2Bconservation%2Bwithin%2Bthe%2BDNA-binding%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002399910012%26rft_id%253Dinfo%253Apmid%252F10684344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002399910012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10684344&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085036100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.68" data-doi="10.1074/jbc.M808712200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gamsjaeger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Y.</span> <span class="cit-name-surname">Thong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Hung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Kwan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Structural analysis of MED-1 reveals unexpected diversity in the mechanism of DNA recognition by GATA-type zinc finger domains.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">5827</span>–<span class="cit-lpage">5835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M808712200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M808712200%26rft_id%253Dinfo%253Apmid%252F19095651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODQvOS81ODI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.69" data-doi="10.1038/nature04445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Brent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Leaf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Pultz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Desplan</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Localized maternal orthodenticle patterns anterior and posterior in the long germ wasp Nasonia.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">439</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">732</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature04445</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLynch%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D439%26rft.issue%253D7077%26rft.spage%253D728%26rft.epage%253D732%26rft.atitle%253DLocalized%2Bmaternal%2Borthodenticle%2Bpatterns%2Banterior%2Band%2Bposterior%2Bin%2Bthe%2Blong%2Bgerm%2Bwasp%2BNasonia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04445%26rft_id%253Dinfo%253Apmid%252F16467838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16467838&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235193100045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.70" data-doi="10.1016/j.ydbio.2005.05.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maduro</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Heid</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Newman-Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">Genetic redundancy in endoderm specification within the genus Caenorhabditis.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2005.05.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DMaduro%26rft.auinit1%253DM.%2BF.%26rft.volume%253D284%26rft.issue%253D2%26rft.spage%253D509%26rft.epage%253D522%26rft.atitle%253DGenetic%2Bredundancy%2Bin%2Bendoderm%2Bspecification%2Bwithin%2Bthe%2Bgenus%2BCaenorhabditis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2005.05.016%26rft_id%253Dinfo%253Apmid%252F15979606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2005.05.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15979606&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.71" data-doi="10.1016/j.semcdb.2017.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maduro</span>,  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Gut Development in C. elegans.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.semcdb.2017.01.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin.%2BCell%2BDev.%2BBiol.%26rft.volume%253D66%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2017.01.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2017.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.72" data-doi="10.1016/j.ydbio.2015.04.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maduro</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Broitman-Maduro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Choi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Carranza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chia-Yi Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">MED GATA factors promote robust development of the C. elegans endoderm.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">404</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2015.04.025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D404%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2015.04.025%26rft_id%253Dinfo%253Apmid%252F25959238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2015.04.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25959238&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.73" data-doi="10.1016/j.ydbio.2006.08.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maduro</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Broitman-Maduro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Mengarelli</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Maternal deployment of the embryonic SKN-1→MED-1,2 cell specification pathway in C. elegans.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">590</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2006.08.029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DMaduro%26rft.auinit1%253DM.%2BF.%26rft.volume%253D301%26rft.issue%253D2%26rft.spage%253D590%26rft.epage%253D601%26rft.atitle%253DMaternal%2Bdeployment%2Bof%2Bthe%2Bembryonic%2BSKN-1--%253EMED-1%252C2%2Bcell%2Bspecification%2Bpathway%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2006.08.029%26rft_id%253Dinfo%253Apmid%252F16979152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2006.08.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16979152&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.74" data-doi="10.1016/j.ydbio.2005.06.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maduro</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Kasmir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">The Wnt effector POP-1 and the PAL-1/Caudal homeoprotein collaborate with SKN-1 to activate C. elegans endoderm development.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2005.06.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DMaduro%26rft.auinit1%253DM.%2BF.%26rft.volume%253D285%26rft.issue%253D2%26rft.spage%253D510%26rft.epage%253D523%26rft.atitle%253DThe%2BWnt%2Beffector%2BPOP-1%2Band%2Bthe%2BPAL-1%252FCaudal%2Bhomeoprotein%2Bcollaborate%2Bwith%2BSKN-1%2Bto%2Bactivate%2BC.%2Belegans%2Bendoderm%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2005.06.022%26rft_id%253Dinfo%253Apmid%252F16084508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2005.06.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16084508&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232574700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.75" data-doi="10.1006/dbio.2002.0721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maduro</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dynamics of a developmental switch: recursive intracellular and intranuclear redistribution of Caenorhabditis elegans POP-1 parallels Wnt-inhibited transcriptional repression.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">248</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.2002.0721</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DMaduro%26rft.auinit1%253DM.%2BF.%26rft.volume%253D248%26rft.issue%253D1%26rft.spage%253D128%26rft.epage%253D142%26rft.atitle%253DDynamics%2Bof%2Ba%2Bdevelopmental%2Bswitch%253A%2Brecursive%2Bintracellular%2Band%2Bintranuclear%2Bredistribution%2Bof%2BCaenorhabditis%2Belegans%2BPOP-1%2Bparallels%2BWnt-inhibited%2Btranscriptional%2Brepression.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.2002.0721%26rft_id%253Dinfo%253Apmid%252F12142026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.2002.0721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12142026&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177259200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.76" data-doi="10.1016/S1097-2765(01)00195-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maduro</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Meneghini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bowerman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Broitman-Maduro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Restriction of mesendoderm to a single blastomere by the combined action of SKN-1 and a GSK-3beta homolog is mediated by MED-1 and -2 in C. elegans.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1097-2765(01)00195-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMaduro%26rft.auinit1%253DM.%2BF.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D475%26rft.epage%253D485%26rft.atitle%253DRestriction%2Bof%2Bmesendoderm%2Bto%2Ba%2Bsingle%2Bblastomere%2Bby%2Bthe%2Bcombined%2Baction%2Bof%2BSKN-1%2Band%2Ba%2BGSK-3beta%2Bhomolog%2Bis%2Bmediated%2Bby%2BMED-1%2Band%2B-2%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900195-2%26rft_id%253Dinfo%253Apmid%252F11463373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00195-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11463373&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167966600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.77" data-doi="10.1093/nar/gkt997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathelier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Parcy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Worsley-Hunt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">JASPAR 2014: an extensively expanded and updated open-access database of transcription factor binding profiles.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D142</span>–<span class="cit-lpage">D147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt997</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt997%26rft_id%253Dinfo%253Apmid%252F24194598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24194598&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.78" data-doi="10.1016/j.ydbio.2008.11.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGhee</span>,  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fukushige</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Krause</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Minnema</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Goszczynski</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">ELT-2 is the predominant transcription factor controlling differentiation and function of the C. elegans intestine, from embryo to adult.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">565</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2008.11.034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DMcGhee%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D327%26rft.issue%253D2%26rft.spage%253D551%26rft.epage%253D565%26rft.atitle%253DELT-2%2Bis%2Bthe%2Bpredominant%2Btranscription%2Bfactor%2Bcontrolling%2Bdifferentiation%2Band%2Bfunction%2Bof%2Bthe%2BC.%2Belegans%2Bintestine%252C%2Bfrom%2Bembryo%2Bto%2Badult.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2008.11.034%26rft_id%253Dinfo%253Apmid%252F19111532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2008.11.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19111532&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.79" data-doi="10.1002/bies.20285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGregor</span>,  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">How to get ahead: the origin, evolution and function of bicoid.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">913</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.20285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DMcGregor%26rft.auinit1%253DA.%2BP.%26rft.volume%253D27%26rft.issue%253D9%26rft.spage%253D904%26rft.epage%253D913%26rft.atitle%253DHow%2Bto%2Bget%2Bahead%253A%2Bthe%2Borigin%252C%2Bevolution%2Band%2Bfunction%2Bof%2Bbicoid.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20285%26rft_id%253Dinfo%253Apmid%252F16108065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16108065&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231603900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.80" data-doi="10.1016/j.tig.2008.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Lee</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MMEJ repair of double-strand breaks (director’s cut): deleted sequences and alternative endings.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">538</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2008.08.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMcVey%26rft.auinit1%253DM.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D529%26rft.epage%253D538%26rft.atitle%253DMMEJ%2Brepair%2Bof%2Bdouble-strand%2Bbreaks%2B%2528director%2527s%2Bcut%2529%253A%2Bdeleted%2Bsequences%2Band%2Balternative%2Bendings.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.08.007%26rft_id%253Dinfo%253Apmid%252F18809224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18809224&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261533700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.81" data-doi="10.1016/j.ydbio.2018.12.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Memar</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schiemann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hennig</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Findeis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Conradt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Twenty million years of evolution: The embryogenesis of four Caenorhabditis species are indistinguishable despite extensive genome divergence.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2018.12.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D447%26rft.spage%253D182%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2018.12.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2018.12.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moskowitz</span>,  <span class="cit-name-given-names">I. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">lin-12 and glp-1 are required zygotically for early embryonic cellular interactions and are regulated by maternal GLP-1 signaling in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">4105</span>–<span class="cit-lpage">4117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DMoskowitz%26rft.auinit1%253DI.%26rft.volume%253D122%26rft.issue%253D12%26rft.spage%253D4105%26rft.epage%253D4117%26rft.atitle%253Dlin-12%2Band%2Bglp-1%2Bare%2Brequired%2Bzygotically%2Bfor%2Bearly%2Bembryonic%2Bcellular%2Binteractions%2Band%2Bare%2Bregulated%2Bby%2Bmaternal%2BGLP-1%2Bsignaling%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F9012530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTIyLzEyLzQxMDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.83" data-doi="10.1111/j.1440-169X.2005.00836.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Okumura</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Uchiyama</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GATA factors as key regulatory molecules in the development of Drosophila endoderm.</span> <abbr class="cit-jnl-abbrev">Dev. Growth Differ.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1440-169X.2005.00836.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%252C%2Bgrowth%2B%2526%2Bdifferentiation%26rft.stitle%253DDev%2BGrowth%2BDiffer%26rft.aulast%253DMurakami%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D9%26rft.spage%253D581%26rft.epage%253D589%26rft.atitle%253DGATA%2Bfactors%2Bas%2Bkey%2Bregulatory%2Bmolecules%2Bin%2Bthe%2Bdevelopment%2Bof%2BDrosophila%2Bendoderm.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1440-169X.2005.00836.x%26rft_id%253Dinfo%253Apmid%252F16316403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1440-169X.2005.00836.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16316403&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233529500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.84" data-doi="10.1371/journal.pone.0215187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nomoto</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kubota</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ohnishi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kasahara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tomita</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Gene Cascade Finder: A tool for identification of gene cascades and its application in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e0215187</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0215187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D14%26rft.spage%253De0215187%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0215187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0215187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.85" data-doi="10.1126/science.8332909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Omichinski</span>,  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Clore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Schaad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Felsenfeld</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Trainor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">NMR structure of a specific DNA complex of Zn-containing DNA binding domain of GATA-1.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">261</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">446</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.8332909</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOmichinski%26rft.auinit1%253DJ.%26rft.volume%253D261%26rft.issue%253D5120%26rft.spage%253D438%26rft.epage%253D446%26rft.atitle%253DNMR%2Bstructure%2Bof%2Ba%2Bspecific%2BDNA%2Bcomplex%2Bof%2BZn-containing%2BDNA%2Bbinding%2Bdomain%2Bof%2BGATA-1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8332909%26rft_id%253Dinfo%253Apmid%252F8332909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjEvNTEyMC80MzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.86" data-doi="10.1007/978-1-4939-7125-1_18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Omranian</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Nikoloski</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Computational Approaches to Study Gene Regulatory Networks.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1629</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-7125-1_18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1629%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-7125-1_18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-7125-1_18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.87" data-doi="10.1016/j.ydbio.2009.09.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Owraghi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Broitman-Maduro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Luu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Roberson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Maduro</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Roles of the Wnt effector POP-1/TCF in the C. elegans endomesoderm specification gene network.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">340</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2009.09.042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DOwraghi%26rft.auinit1%253DM.%26rft.volume%253D340%26rft.issue%253D2%26rft.spage%253D209%26rft.epage%253D221%26rft.atitle%253DRoles%2Bof%2Bthe%2BWnt%2Beffector%2BPOP-1%252FTCF%2Bin%2Bthe%2BC.%2Belegans%2Bendomesoderm%2Bspecification%2Bgene%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2009.09.042%26rft_id%253Dinfo%253Apmid%252F19818340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2009.09.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19818340&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.88" data-doi="10.1101/gad.11.13.1651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>,  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Steward</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Blumenthal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Priess</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">ELT-1, a GATA-like transcription factor, is required for epidermal cell fates in Caenorhabditis elegans embryos.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1661</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.11.13.1651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPage%26rft.auinit1%253DB%2BD%26rft.volume%253D11%26rft.issue%253D13%26rft.spage%253D1651%26rft.epage%253D1661%26rft.atitle%253DELT-1%252C%2Ba%2BGATA-like%2Btranscription%2Bfactor%252C%2Bis%2Brequired%2Bfor%2Bepidermal%2Bcell%2Bfates%2Bin%2BCaenorhabditis%2Belegans%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.13.1651%26rft_id%253Dinfo%253Apmid%252F9224715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzEzLzE2NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.89" data-doi="10.1073/pnas.94.11.5556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Lo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Schmidt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Pelczer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Thurber</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Skn-1: evidence for a bipartite recognition helix in DNA binding.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">5556</span>–<span class="cit-lpage">5561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.94.11.5556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPal%26rft.auinit1%253DS.%26rft.volume%253D94%26rft.issue%253D11%26rft.spage%253D5556%26rft.epage%253D5561%26rft.atitle%253DSkn-1%253A%2BEvidence%2Bfor%2Ba%2Bbipartite%2Brecognition%2Bhelix%2Bin%2BDNA%2Bbinding%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.11.5556%26rft_id%253Dinfo%253Apmid%252F9159111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTEvNTU1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.90" data-doi="10.1016/bs.ctdb.2015.12.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>,  <span class="cit-name-given-names">I. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Davidson</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Implications of Developmental Gene Regulatory Networks Inside and Outside Developmental Biology.</span> <abbr class="cit-jnl-abbrev">Curr. Top. Dev. Biol.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/bs.ctdb.2015.12.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BTop.%2BDev.%2BBiol.%26rft.volume%253D117%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.ctdb.2015.12.014%26rft_id%253Dinfo%253Apmid%252F26969981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.ctdb.2015.12.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26969981&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.91" data-doi="10.1038/nature08781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Rifkin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Andersen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">van Oudenaarden</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variability in gene expression underlies incomplete penetrance.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">918</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08781</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRaj%26rft.auinit1%253DA.%26rft.volume%253D463%26rft.issue%253D7283%26rft.spage%253D913%26rft.epage%253D918%26rft.atitle%253DVariability%2Bin%2Bgene%2Bexpression%2Bunderlies%2Bincomplete%2Bpenetrance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08781%26rft_id%253Dinfo%253Apmid%252F20164922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20164922&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274582700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.92" data-doi="10.1016/j.ydbio.2011.04.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Lo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Odom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. D.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Medina</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Functional analyses of vertebrate TCF proteins in C. elegans embryos.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">355</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2011.04.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DRobertson%26rft.auinit1%253DS.%2BM.%26rft.volume%253D355%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D123%26rft.atitle%253DFunctional%2Banalyses%2Bof%2Bvertebrate%2BTCF%2Bproteins%2Bin%2BC.%2Belegans%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2011.04.012%26rft_id%253Dinfo%253Apmid%252F21539828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2011.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21539828&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.93" data-doi="10.1016/S0092-8674(00)80531-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocheleau</span>,  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. D.</span> <span class="cit-name-surname">Downs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wittmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Bei</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Wnt signaling and an APC-related gene specify endoderm in early C. elegans embryos.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80531-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRocheleau%26rft.auinit1%253DC.%2BE.%26rft.volume%253D90%26rft.issue%253D4%26rft.spage%253D707%26rft.epage%253D716%26rft.atitle%253DWnt%2Bsignaling%2Band%2Ban%2BAPC-related%2Bgene%2Bspecify%2Bendoderm%2Bin%2Bearly%2BC.%2Belegans%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980531-0%26rft_id%253Dinfo%253Apmid%252F9288750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80531-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9288750&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XT06600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.94" data-doi="10.1073/pnas.0408274102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>,  <span class="cit-name-given-names">S. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gilbert</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The pattern of intron loss.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0408274102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0408274102%26rft_id%253Dinfo%253Apmid%252F15642949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDIvMy83MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.95" data-doi="10.1093/molbev/msl098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>,  <span class="cit-name-given-names">S. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Penny</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Smoke without fire: most reported cases of intron gain in nematodes instead reflect intron losses.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2259</span>–<span class="cit-lpage">2262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msl098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl098%26rft_id%253Dinfo%253Apmid%252F16943250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16943250&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241955400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.96" data-doi="10.1046/j.1525-142X.2002.02027.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudel</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kimble</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution of discrete Notch-like receptors from a distant gene duplication in Caenorhabditis.</span> <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1525-142X.2002.02027.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DRudel%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D319%26rft.epage%253D333%26rft.atitle%253DEvolution%2Bof%2Bdiscrete%2BNotch-like%2Breceptors%2Bfrom%2Ba%2Bdistant%2Bgene%2Bduplication%2Bin%2BCaenorhabditis.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1525-142X.2002.02027.x%26rft_id%253Dinfo%253Apmid%252F12356262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1525-142X.2002.02027.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12356262&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177775100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.97" data-doi="10.1042/BST0360641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawicka</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bushell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Spriggs</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Willis</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Polypyrimidine-tract-binding protein: a multifunctional RNA-binding protein.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1042/BST0360641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0360641%26rft_id%253Dinfo%253Apmid%252F18631133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo4OiIzNi80LzY0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.98" data-doi="10.1534/genetics.111.129486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Glass</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Shemer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">White</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Overcoming redundancy: an RNAi enhancer screen for morphogenesis genes in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">564</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.129486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.129486%26rft_id%253Dinfo%253Apmid%252F21527776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNTQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.99" data-doi="10.1002/bies.1119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierenberg</span>,  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Three sons of fortune: early embryogenesis, evolution and ecology of nematodes.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.1119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DSchierenberg%26rft.auinit1%253DE.%26rft.volume%253D23%26rft.issue%253D9%26rft.spage%253D841%26rft.epage%253D847%26rft.atitle%253DThree%2Bsons%2Bof%2Bfortune%253A%2Bearly%2Bembryogenesis%252C%2Bevolution%2Band%2Becology%2Bof%2Bnematodes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.1119%26rft_id%253Dinfo%253Apmid%252F11536296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.1119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11536296&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170697000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.100" data-doi="10.1895/wormbook.1.55.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierenberg</span>,  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Embryological variation during nematode development.</span> (January 02, 2006), WormBook, ed. The C. elegans Research Community, WormBook<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1895/wormbook.1.55.1</span>, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1895%252Fwormbook.1.55.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1895/wormbook.1.55.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.101" data-doi="10.1007/s00427-014-0471-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiffer</span>,  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Nsah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Grotehusmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kroiher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Loer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Developmental variations among Panagrolaimid nematodes indicate developmental system drift within a small taxonomic unit.</span> <abbr class="cit-jnl-abbrev">Dev. Genes Evol.</abbr> <span class="cit-vol">224</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00427-014-0471-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BGenes%2BEvol.%26rft.volume%253D224%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00427-014-0471-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00427-014-0471-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.102" data-doi="10.1186/2041-9139-2-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Schierenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of embryonic development in nematodes.</span> <abbr class="cit-jnl-abbrev">Evodevo</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/2041-9139-2-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEvoDevo%26rft.aulast%253DSchulze%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DEvolution%2Bof%2Bembryonic%2Bdevelopment%2Bin%2Bnematodes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2041-9139-2-18%26rft_id%253Dinfo%253Apmid%252F21929824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2041-9139-2-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21929824&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.103" data-doi="10.1016/j.ydbio.2005.07.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shetty</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Lo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Robertson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lin</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">C. elegans TCF protein, POP-1, converts from repressor to activator as a result of Wnt-induced lowering of nuclear levels.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2005.07.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DShetty%26rft.auinit1%253DP.%26rft.volume%253D285%26rft.issue%253D2%26rft.spage%253D584%26rft.epage%253D592%26rft.atitle%253DC.%2Belegans%2BTCF%2Bprotein%252C%2BPOP-1%252C%2Bconverts%2Bfrom%2Brepressor%2Bto%2Bactivator%2Bas%2Ba%2Bresult%2Bof%2BWnt-induced%2Blowering%2Bof%2Bnuclear%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2005.07.008%26rft_id%253Dinfo%253Apmid%252F16112103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2005.07.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16112103&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232574700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.104" data-doi="10.1101/gr.3715005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bejerano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Pedersen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Hinrichs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1050</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3715005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3715005%26rft_id%253Dinfo%253Apmid%252F16024819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTAzNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.105" data-doi="10.1016/j.ydbio.2010.08.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sommermann</span>,  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Strohmaier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Maduro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Endoderm development in Caenorhabditis elegans: the synergistic action of ELT-2 and -7 mediates the specification→differentiation transition.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">347</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2010.08.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSommermann%26rft.auinit1%253DE.%2BM.%26rft.volume%253D347%26rft.issue%253D1%26rft.spage%253D154%26rft.epage%253D166%26rft.atitle%253DEndoderm%2Bdevelopment%2Bin%2BCaenorhabditis%2Belegans%253A%2Bthe%2Bsynergistic%2Baction%2Bof%2BELT-2%2Band%2B-7%2Bmediates%2Bthe%2Bspecification%25E2%2586%2592differentiation%2Btransition.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2010.08.020%26rft_id%253Dinfo%253Apmid%252F20807527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2010.08.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20807527&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.106" data-doi="10.1895/wormbook.1.65.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spieth</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lawson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Williams</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Howe</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Overview of gene structure in C. elegans.</span> (October 29, 2014), WormBook, ed. The C. elegans Research Community, WormBook<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1895/wormbook.1.65.2</span>, <a href="http://www.wormbook.org">http://www.wormbook.org</a>. </cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1895%252Fwormbook.1.65.2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1895/wormbook.1.65.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.107" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.108" data-doi="10.1073/pnas.96.7.3786"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stauber</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jackle</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Schmidt-Ott</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The anterior determinant bicoid of Drosophila is a derived Hox class 3 gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">3786</span>–<span class="cit-lpage">3789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.96.7.3786</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DStauber%26rft.auinit1%253DM.%26rft.volume%253D96%26rft.issue%253D7%26rft.spage%253D3786%26rft.epage%253D3789%26rft.atitle%253DThe%2Banterior%2Bdeterminant%2Bbicoid%2Bof%2BDrosophila%2Bis%2Ba%2Bderived%2BHox%2Bclass%2B3%2Bgene%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.7.3786%26rft_id%253Dinfo%253Apmid%252F10097115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni83LzM3ODYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.109" data-doi="10.1371/journal.pbio.0000045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>,  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Bao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Blasiar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Blumenthal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Brent</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Genome Sequence of Caenorhabditis briggsae: A Platform for Comparative Genomics.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">E45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0000045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DStein%26rft.auinit1%253DL.%2BD.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253DE45%26rft.epage%253DE45%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2BCaenorhabditis%2Bbriggsae%253A%2Ba%2Bplatform%2Bfor%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0000045%26rft_id%253Dinfo%253Apmid%252F14624247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0000045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14624247&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.110" data-doi="10.1002/evl3.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Felix</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Beltran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Braendle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Caurcel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Comparative genomics of 10 new Caenorhabditis species.</span> <abbr class="cit-jnl-abbrev">Evol Lett</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/evl3.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BLett%26rft.volume%253D3%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevl3.110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evl3.110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.111" data-doi="10.1534/genetics.115.183137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sullivan-Brown</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tandon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Bird</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Dickinson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Tintori</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identifying Regulators of Morphogenesis Common to Vertebrate Neural Tube Closure and Caenorhabditis elegans Gastrulation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.183137</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183137%26rft_id%253Dinfo%253Apmid%252F26434722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzEvMTIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.112" data-doi="10.1016/0012-1606(83)90201-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sulston</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Schierenberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">White</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Thomson</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The embryonic cell lineage of the nematode Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0012-1606(83)90201-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSulston%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D100%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D119%26rft.atitle%253DThe%2Bembryonic%2Bcell%2Blineage%2Bof%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252883%252990201-4%26rft_id%253Dinfo%253Apmid%252F6684600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(83)90201-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6684600&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RN28500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.113" data-doi="10.1093/nar/gky653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taheri-Ghahfarokhi</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J. M.</span> <span class="cit-name-surname">Taylor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nitsch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lundin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Cavallo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Decoding non-random mutational signatures at Cas9 targeted sites.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">8417</span>–<span class="cit-lpage">8434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky653</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky653%26rft_id%253Dinfo%253Apmid%252F30032200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30032200&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.114" data-doi="10.1016/S0092-8674(00)80530-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorpe</span>,  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schlesinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Carter</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bowerman</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Wnt signaling polarizes an early C. elegans blastomere to distinguish endoderm from mesoderm.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">705</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80530-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DThorpe%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D90%26rft.issue%253D4%26rft.spage%253D695%26rft.epage%253D705%26rft.atitle%253DWnt%2Bsignaling%2Bpolarizes%2Ban%2Bearly%2BC.%2Belegans%2Bblastomere%2Bto%2Bdistinguish%2Bendoderm%2Bfrom%2Bmesoderm.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980530-9%26rft_id%253Dinfo%253Apmid%252F9288749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80530-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9288749&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XT06600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.115" data-doi="10.7554/eLife.48220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres Cleuren</span>,  <span class="cit-name-given-names">Y. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Ewe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. C.</span> <span class="cit-name-surname">Chipman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Mears</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Wood</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Extensive intraspecies cryptic variation in an ancient embryonic gene regulatory network.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e48220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.48220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D8%26rft.spage%253De48220%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.48220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.48220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.116" data-doi="10.1242/dev.164384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tremblay</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Sanchez-Ferras</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bouchard</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">GATA transcription factors in development and disease.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">dev164384</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.164384</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DTremblay%26rft.auinit1%253DM.%26rft.volume%253D145%26rft.issue%253D20%26rft.spage%253Ddev164384%26rft.epage%253Ddev164384%26rft.atitle%253DGATA%2Btranscription%2Bfactors%2Bin%2Bdevelopment%2Band%2Bdisease%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.164384%26rft_id%253Dinfo%253Apmid%252F30348673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxNjoiMTQ1LzIwL2RldjE2NDM4NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference  in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.117" data-doi="10.1046/j.1525-142x.2001.003002109.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">True</span>,  <span class="cit-name-given-names">J. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Haag</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Developmental system drift and flexibility in evolutionary trajectories.</span> <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1525-142x.2001.003002109.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DTrue%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D109%26rft.epage%253D119%26rft.atitle%253DDevelopmental%2Bsystem%2Bdrift%2Band%2Bflexibility%2Bin%2Bevolutionary%2Btrajectories.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1525-142x.2001.003002109.x%26rft_id%253Dinfo%253Apmid%252F11341673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1525-142x.2001.003002109.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11341673&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168050300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference  in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.118" data-doi="10.1093/gbe/evt088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Schendel</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tijsterman</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Microhomology-mediated intron loss during metazoan evolution.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1212</span>–<span class="cit-lpage">1219</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evt088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt088%26rft_id%253Dinfo%253Apmid%252F23737326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23737326&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference  in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.119" data-doi="10.1074/jbc.M501548200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walther</span>,  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Atlas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Carrigan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Rouleau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Edgecombe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A serine/threonine-rich motif is one of three nuclear localization signals that determine unidirectional transport of the mineralocorticoid receptor to the nucleus.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">17549</span>–<span class="cit-lpage">17561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M501548200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M501548200%26rft_id%253Dinfo%253Apmid%252F15737989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMTcvMTc1NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference  in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.120" data-doi="10.1016/j.cell.2014.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>,  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Albu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Cote</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montenegro-Montero</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Determination and inference of eukaryotic transcription factor sequence specificity.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1431</span>–<span class="cit-lpage">1443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.08.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D158%26rft.spage%253D1431%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.08.009%26rft_id%253Dinfo%253Apmid%252F25215497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25215497&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343094200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference  in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.121" data-doi="10.1073/pnas.1012675108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiegmann</span>,  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Trautwein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. S.</span> <span class="cit-name-surname">Winkler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. B.</span> <span class="cit-name-surname">Barr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Kim</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Episodic radiations in the fly tree of life.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">5690</span>–<span class="cit-lpage">5695</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1012675108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012675108%26rft_id%253Dinfo%253Apmid%252F21402926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE0LzU2OTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference  in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.122"><div class="cit-metadata unstructured">Wiesenfahrt, T., J.Y. Berg, E.O. Nishimura, A.G. Robinson, B. Goszczynski <em>et al.</em>, 2015 The Function and Regulation of the GATA Factor ELT-2 in the C. elegans Endoderm. <em>Development</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference  in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.123" data-doi="10.1016/j.ydbio.2008.11.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Witze</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Field</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Hunt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">C. elegans pur alpha, an activator of end-1, synergizes with the Wnt pathway to specify endoderm.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2008.11.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DWitze%26rft.auinit1%253DE.%2BS.%26rft.volume%253D327%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D23%26rft.atitle%253DC.%2Belegans%2Bpur%2Balpha%252C%2Ban%2Bactivator%2Bof%2Bend-1%252C%2Bsynergizes%2Bwith%2Bthe%2BWnt%2Bpathway%2Bto%2Bspecify%2Bendoderm.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2008.11.015%26rft_id%253Dinfo%253Apmid%252F19084000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2008.11.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19084000&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference  in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.124" data-doi="10.1534/genetics.110.120550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>,  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Eke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Baird</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Felix</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Haag</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Insights into species divergence and the evolution of hermaphroditism from fertile interspecies hybrids of Caenorhabditis nematodes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.120550</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120550%26rft_id%253Dinfo%253Apmid%252F20823339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzMvOTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference  in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.125" data-doi="10.1128/MCB.14.3.2201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Romeo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bories</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Motohashi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Human GATA-3 trans-activation, DNA-binding, and nuclear localization activities are organized into distinct structural domains.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2201</span>–<span class="cit-lpage">2212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.14.3.2201</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DYang%26rft.auinit1%253DZ%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D2201%26rft.epage%253D2212%26rft.atitle%253DHuman%2BGATA-3%2Btrans-activation%252C%2BDNA-binding%252C%2Band%2Bnuclear%2Blocalization%2Bactivities%2Bare%2Borganized%2Binto%2Bdistinct%2Bstructural%2Bdomains.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.3.2201%26rft_id%253Dinfo%253Apmid%252F8114750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE0LzMvMjIwMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-126-1" title="View reference  in text" id="ref-126">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.126" data-doi="10.1101/gr.244830.118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshimura</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ichikawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Shoura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Artiles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Gabdank</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Recompleting the Caenorhabditis elegans genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1022</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.244830.118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.244830.118%26rft_id%253Dinfo%253Apmid%252F31123080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI5LzYvMTAwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-127-1" title="View reference  in text" id="ref-127">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.127" data-doi="10.1534/g3.111.001081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ihuegbu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Schriefer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Conserved Motifs and Prediction of Regulatory Modules in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">481</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.001081</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.001081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-128-1" title="View reference  in text" id="ref-128">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.128" data-doi="10.1016/j.ydbio.2007.11.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Boyle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Bao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. I.</span> <span class="cit-name-surname">Murray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Mericle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative analysis of embryonic cell lineage between Caenorhabditis briggsae and Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">99</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2007.11.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DZhao%26rft.auinit1%253DZ.%26rft.volume%253D314%26rft.issue%253D1%26rft.spage%253D93%26rft.epage%253D99%26rft.atitle%253DComparative%2Banalysis%2Bof%2Bembryonic%2Bcell%2Blineage%2Bbetween%2BCaenorhabditis%2Bbriggsae%2Band%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2007.11.015%26rft_id%253Dinfo%253Apmid%252F18164284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2007.11.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18164284&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253031100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-129-1" title="View reference  in text" id="ref-129">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.129" data-doi="10.1534/genetics.109.110270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Flibotte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. I.</span> <span class="cit-name-surname">Murray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Blick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Boyle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New tools for investigating the comparative biology of Caenorhabditis briggsae and C. elegans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.110270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.110270%26rft_id%253Dinfo%253Apmid%252F20008572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzMvODUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-130-1" title="View reference  in text" id="ref-130">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.130" data-doi="10.1101/gad.12.24.3809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fukushige</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">McGhee</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Rothman</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Reprogramming of early embryonic blastomeres into endodermal progenitors by a Caenorhabditis elegans GATA factor.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">3809</span>–<span class="cit-lpage">3814</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.12.24.3809</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZhu%26rft.auinit1%253DJ.%26rft.volume%253D12%26rft.issue%253D24%26rft.spage%253D3809%26rft.epage%253D3814%26rft.atitle%253DReprogramming%2Bof%2Bearly%2Bembryonic%2Bblastomeres%2Binto%2Bendodermal%2Bprogenitors%2Bby%2Ba%2BCaenorhabditis%2Belegans%2BGATA%2Bfactor%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.12.24.3809%26rft_id%253Dinfo%253Apmid%252F9869634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEyLzI0LzM4MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-131-1" title="View reference  in text" id="ref-131">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.333.131" data-doi="10.1101/gad.11.21.2883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Heid</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fukuyama</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sugimoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">end-1 encodes an apparent GATA factor that specifies the endoderm precursor in Caenorhabditis elegans embryos.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2883</span>–<span class="cit-lpage">2896</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.11.21.2883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZhu%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D21%26rft.spage%253D2883%26rft.epage%253D2896%26rft.atitle%253Dend-1%2Bencodes%2Ban%2Bapparent%2BGATA%2Bfactor%2Bthat%2Bspecifies%2Bthe%2Bendoderm%2Bprecursor%2Bin%2BCaenorhabditis%2Belegans%25A0embryos%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.21.2883%26rft_id%253Dinfo%253Apmid%252F9353257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzIxLzI4ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/333.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/321" title="A Novel cis Element Achieves the Same Solution as an Ancestral cis Element During Thiamine Starvation in Candida glabrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/357" title="Nonsynonymous Mutations in Linker-2 of the Pdr5 Multidrug Transporter Identify a New RNA Stability Element" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/333" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vLU6ddqZyiuHRoxAK-jzjvYKJeavpdNtXm32ZWn1-bc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/333" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/333" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/333" class="active" data-icon-position="" data-hide-link-title="0">Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in Caenorhabditis Endoderm Specification</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495168" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wEeWu6jpFSsJUFS8D0zDXMNSEjfj7ZBlpYK7HJeb9Qc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/333.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/333" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wh64kkPymk5Madz0xFO1WzTZ9j5GLEULLKWQGzRMkuQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/333" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/333.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495168" id="node495168--2" data-pisa="ggg;10/1/333" data-pisa-master="ggg;g3.119.400724" data-apath="/ggg/10/1/333.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in <em>Caenorhabditis</em> Endoderm Specification</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6257-7612" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Morris F. Maduro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">333-356;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400724 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495168--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Morris F. Maduro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular, Cell and Systems Biology Department, University of California, Riverside, CA 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morris%2BF.%2BMaduro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maduro%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMorris%2BF.%2BMaduro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6257-7612" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Morris F. Maduro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mmaduro@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">mmaduro@ucr.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/333" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/333&amp;title=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/333&amp;title=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/333&amp;title=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/333&amp;text=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/333&amp;title=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/333&amp;t=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/333&amp;title=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/333&amp;title=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495168/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495168" id="node495168--3" data-pisa="ggg;10/1/333" data-pisa-master="ggg;g3.119.400724" data-apath="/ggg/10/1/333.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolutionary Dynamics of the SKN-1 → MED → END-1,3 Regulatory Gene Cascade in <em>Caenorhabditis</em> Endoderm Specification</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6257-7612" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Morris F. Maduro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">333-356;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400724 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495168--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Morris F. Maduro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular, Cell and Systems Biology Department, University of California, Riverside, CA 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morris%2BF.%2BMaduro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maduro%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMorris%2BF.%2BMaduro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6257-7612" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Morris F. Maduro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mmaduro@ucr.edu" class="" data-icon-position="" data-hide-link-title="0">mmaduro@ucr.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=333&amp;rft.epage=356&amp;rft.atitle=Evolutionary%20Dynamics%20of%20the%20SKN-1%20%E2%86%92%20MED%20%E2%86%92%20END-1%2C3%20Regulatory%20Gene%20Cascade%20in%20Caenorhabditis%20Endoderm%20Specification&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Maduro%2C+Morris+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495168/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495168/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495168/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495168/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495168/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495168/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495168/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495168/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495168/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495168/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495168/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495168/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/333" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/333.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/333.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/333.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
