<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/379" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/379.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/379.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/379.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/379.atom" />
<meta name="HW.pisa" content="ggg;10/1/379" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.119.400827" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2020 Yang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Guangxi indigenous chicken breeds play a very important role in promoting the high-quality development of the broiler industry in China. However, studies on genomic information of Guangxi indigenous chicken to date remain poorly explored. To decipher the population genetic structure and differentially selected regions (DSRs) in Guangxi indigenous chickens, we dug into numerous SNPs from seven Guangxi native chickens (GX) by employing the restriction site associated with DNA sequencing (RAD-seq) technology. Another three breeds, Cobb, White Leghorn, and Chahua (CH) chicken, were used as a control. After quality control, a total of 185,117 autosomal SNPs were kept for further analysis. The results showed a significant difference in population structure, and the control breeds were distinctly separate from the Guangxi native breeds, which was also strongly supported by the phylogenetic tree. Distribution of FST indicated that there were three SNPs with big genetic differentiation (FST value all reach to 0. 9427) in GX vs. CH group, which were located on chr1-96,859,720,chr4-86,139,601, and chr12-8,128,322, respectively. Besides, we identified 717 DSRs associated with 882 genes in GX vs. Cobb group, 769 DSRs with 476 genes in GX vs. Leghorn group, and 556 DSRs with 779 genes in GX vs. CH group. GO enrichment showed that there were two significant terms, namely GPI-linked ephrin receptor activity and BMP receptor binding, which were enriched in GX vs. Leghorn group. In conclusion, this study suggests that Guangxi native chickens have a great differentiation with Cobb and Leghorn. Our findings would be beneficial to fully evaluate the genomic information on Guangxi native chicken and facilitate the application of these resources in chicken breeding." />
<meta name="DC.Contributor" content="Zhuliang Yang" />
<meta name="DC.Contributor" content="Jixian Deng" />
<meta name="DC.Contributor" content="Dongfeng Li" />
<meta name="DC.Contributor" content="Tiantian Sun" />
<meta name="DC.Contributor" content="Li Xia" />
<meta name="DC.Contributor" content="Wenwen Xu" />
<meta name="DC.Contributor" content="Linghu Zeng" />
<meta name="DC.Contributor" content="Hesheng Jiang" />
<meta name="DC.Contributor" content="Xiurong Yang" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Guangxi indigenous chicken breeds play a very important role in promoting the high-quality development of the broiler industry in China. However, studies on genomic information of Guangxi indigenous chicken to date remain poorly explored. To decipher the population genetic structure and differentially selected regions (DSRs) in Guangxi indigenous chickens, we dug into numerous SNPs from seven Guangxi native chickens (GX) by employing the restriction site associated with DNA sequencing (RAD-seq) technology. Another three breeds, Cobb, White Leghorn, and Chahua (CH) chicken, were used as a control. After quality control, a total of 185,117 autosomal SNPs were kept for further analysis. The results showed a significant difference in population structure, and the control breeds were distinctly separate from the Guangxi native breeds, which was also strongly supported by the phylogenetic tree. Distribution of F&lt;sub&gt;ST&lt;/sub&gt; indicated that there were three SNPs with big genetic differentiation (F&lt;sub&gt;ST&lt;/sub&gt; value all reach to 0. 9427) in GX &lt;i&gt;vs.&lt;/i&gt; CH group, which were located on chr1-96,859,720,chr4-86,139,601, and chr12-8,128,322, respectively. Besides, we identified 717 DSRs associated with 882 genes in GX &lt;i&gt;vs.&lt;/i&gt; Cobb group, 769 DSRs with 476 genes in GX &lt;i&gt;vs.&lt;/i&gt; Leghorn group, and 556 DSRs with 779 genes in GX &lt;i&gt;vs.&lt;/i&gt; CH group. GO enrichment showed that there were two significant terms, namely GPI-linked ephrin receptor activity and BMP receptor binding, which were enriched in GX &lt;i&gt;vs.&lt;/i&gt; Leghorn group. In conclusion, this study suggests that Guangxi native chickens have a great differentiation with Cobb and Leghorn. Our findings would be beneficial to fully evaluate the genomic information on Guangxi native chicken and facilitate the application of these resources in chicken breeding.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/379" />
<meta name="citation_id" content="10/1/379" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/379" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/379.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/379.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/379.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400827" />
<meta name="citation_pmid" content="31744899" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="379" />
<meta name="citation_lastpage" content="386" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhuliang Yang" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Guangxi University, Nanning, 530004, China," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5418-0751" />
<meta name="citation_author" content="Jixian Deng" />
<meta name="citation_author_institution" content="Guangxi Institute of Animal Science, Nanning, 530001, China, and" />
<meta name="citation_author" content="Dongfeng Li" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Nanjing Agricultural University, Nanjing, 210095, China" />
<meta name="citation_author" content="Tiantian Sun" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Guangxi University, Nanning, 530004, China," />
<meta name="citation_author" content="Li Xia" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Guangxi University, Nanning, 530004, China," />
<meta name="citation_author" content="Wenwen Xu" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Guangxi University, Nanning, 530004, China," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3829-7922" />
<meta name="citation_author" content="Linghu Zeng" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Guangxi University, Nanning, 530004, China," />
<meta name="citation_author" content="Hesheng Jiang" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Guangxi University, Nanning, 530004, China," />
<meta name="citation_author" content="Xiurong Yang" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Guangxi University, Nanning, 530004, China," />
<meta name="citation_author_email" content="yangxiurong09@163.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6252-7455" />
<meta name="citation_reference" content="citation_journal_title=Neural Dev.;citation_author=M. R. Allen-Sharpley;citation_author=K. S. Cramer;citation_title=Coordinated Eph-ephrin signaling guides migration and axon targeting in the avian auditory system.;citation_pages=29;citation_volume=7;citation_year=2012;citation_doi=10.1186/1749-8104-7-29" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=M. Cariou;citation_author=L. Duret;citation_author=S. Charlat;citation_title=How and how much does RAD-seq bias genetic diversity estimates?;citation_pages=240;citation_volume=16;citation_year=2016;citation_doi=10.1186/s12862-016-0791-0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=S. Bassham;citation_author=T. Wilson;citation_author=M. Currey;citation_author=C. OâBrien;citation_title=The population structure and recent colonization history of Oregon threespine stickleback determined using restriction-site associated DNA-sequencing.;citation_pages=2864-2883;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12330" />
<meta name="citation_reference" content="citation_journal_title=Epigenetics;citation_author=L. DâAmato;citation_author=C. DellâAversana;citation_author=M. Conte;citation_author=A. Ciotta;citation_author=L. Scisciola;citation_title=ARHGEF3 controls HDACi-induced differentiation via RhoA-dependent pathways in acute myeloid leukemias.;citation_pages=6-18;citation_volume=10;citation_year=2015;citation_doi=10.4161/15592294.2014.988035" />
<meta name="citation_reference" content="citation_title=Plant Genotyping: Methods and Protocols;citation_title=Genotyping Analysis Using an RFLP Assay;citation_pages=91-99;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=D. A. R. Eaton;citation_author=R. H. Ree;citation_title=Inferring Phylogeny and Introgression using RADseq Data: An Example from Flowering Plants (Pedicularis: Orobanchaceae).;citation_volume=62;citation_year=2013;citation_pmid=23652346;citation_doi=10.1093/sysbio/syt032" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=P. D. Etter;citation_author=S. Bassham;citation_author=P. A. Hohenlohe;citation_author=E. A. Johnson;citation_author=W. A. Cresko;citation_title=SNP discovery and genotyping for evolutionary genetics using RAD sequencing.;citation_pages=157-178;citation_volume=772;citation_year=2011;citation_pmid=22065437;citation_doi=10.1007/978-1-61779-228-1_9" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761;citation_doi=PubMed PMID: 88" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_title=Unified nomenclature for Eph family receptors and their ligands, the ephrins. Eph Nomenclature Committee.;citation_pages=403-404;citation_volume=90;citation_year=1997;citation_issue=3;citation_pmid=9267020;citation_doi=10.1016/S0092-8674(00)80500-0" />
<meta name="citation_reference" content="citation_journal_title=BMC Res. Notes;citation_author=S. Gonen;citation_author=S. C. Bishop;citation_author=R. D. Houston;citation_title=Exploring the utility of cross-laboratory RAD-sequencing datasets for phylogenetic analysis.;citation_pages=299;citation_volume=8;citation_year=2015;citation_pmid=26152111;citation_doi=10.1186/s13104-015-1261-2" />
<meta name="citation_reference" content="citation_journal_title=Analytical Chemistry (Washington, DC);citation_journal_abbrev=Analytical Chemistry (Washington, DC);citation_author=T. J. Griffin;citation_author=L. M. Smith;citation_title=Genetic identification by mass spectrometric analysis of single-nucleotide polymorphisms: ternary encoding of genotypes.;citation_pages=3298-3302;citation_volume=72;citation_year=2000;citation_issue=14;citation_pmid=10939403;citation_doi=10.1021/ac991390e" />
<meta name="citation_reference" content="citation_journal_title=Exp. Cell Res.;citation_author=I. S. Gul;citation_author=P. Hulpiau;citation_author=Y. Saeys;citation_author=F. van Roy;citation_title=Evolution and diversity of cadherins and catenins.;citation_pages=3-9;citation_volume=358;citation_year=2017;citation_doi=10.1016/j.yexcr.2017.03.001" />
<meta name="citation_reference" content="citation_journal_title=J. Cell. Biochem.;citation_author=N. N. He;citation_author=Y. L. Wang;citation_author=C. Zhang;citation_author=M. Wang;citation_author=Y. J. Wang;citation_title=Wnt signaling pathway regulates differentiation of chicken embryonic stem cells into spermatogonial stem cells via Wnt5a.;citation_pages=1689-1701;citation_volume=119;citation_year=2018;citation_doi=10.1002/jcb.26329" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. E. Holsinger;citation_author=B. S. Weir;citation_title=Genetics in geographically structured populations: defining, estimating and interpreting F(ST).;citation_pages=639-650;citation_volume=10;citation_year=2009;citation_issue=9;citation_pmid=19687804;citation_doi=10.1038/nrg2611" />
<meta name="citation_reference" content="citation_journal_title=Matrix Biol.;citation_author=S. Hosseini-Farahabadi;citation_author=P. Geetha-Loganathan;citation_author=K. Fu;citation_author=S. Nimmagadda;citation_author=H. J. Yang;citation_title=Dual functions for WNT5A during cartilage development and in disease.;citation_pages=252-264;citation_volume=32;citation_year=2013;citation_pmid=23474397;citation_doi=10.1016/j.matbio.2013.02.005" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Hou;citation_author=M. D. Nowak;citation_author=V. Mirre;citation_author=C. S. Bjora;citation_author=C. Brochmann;citation_title=Thousands of RAD-seq Loci Fully Resolve the Phylogeny of the Highly Disjunct Arctic-Alpine Genus Diapensia (Diapensiaceae).;citation_pages=e0140175;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0140175" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=J. D. Jontes;citation_title=The Cadherin Superfamily in Neural Circuit Assembly.;citation_volume=10;citation_year=2018;citation_pmid=28778868;citation_doi=10.1101/cshperspect.a029306" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=T. Katagiri;citation_author=T. Watabe;citation_title=Bone Morphogenetic Proteins;citation_pages=a021899-a021899;citation_volume=8;citation_year=2016;citation_issue=6;citation_pmid=27252362;citation_doi=10.1101/cshperspect.a021899" />
<meta name="citation_reference" content="citation_journal_title=Reproduction;citation_journal_abbrev=Reproduction;citation_author=P. G. Knight;citation_author=C. Glister;citation_title=TGF-{beta} superfamily members and ovarian follicle development.;citation_pages=191-206;citation_volume=132;citation_year=2006;citation_issue=2;citation_pmid=16885529;citation_doi=10.1530/rep.1.01074" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Z. Li;citation_author=S. Wei;citation_author=H. Li;citation_author=K. Wu;citation_author=Z. Cai;citation_title=Genome-wide genetic structure and differentially selected regions among Landrace, Erhualian, and Meishan pigs using specific-locus amplified fragment sequencing.;citation_pages=10063;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-09969-6" />
<meta name="citation_reference" content="citation_journal_title=Development, growth &amp; differentiation;citation_journal_abbrev=Dev Growth Differ;citation_author=D. Listyorini;citation_author=S. Yasugi;citation_title=Expression and function of Wnt5a in the development of the glandular stomach in the chicken embryo.;citation_pages=243-252;citation_volume=48;citation_year=2006;citation_issue=4;citation_pmid=16681649;citation_doi=10.1111/j.1440-169X.2006.00861.x" />
<meta name="citation_reference" content="citation_journal_title=Oncotarget;citation_author=T. H. Liu;citation_author=F. Zheng;citation_author=M. Y. Cai;citation_author=L. Guo;citation_author=H. X. Lin;citation_title=The putative tumor activator ARHGEF3 promotes nasopharyngeal carcinoma cell pathogenesis by inhibiting cellular apoptosis.;citation_pages=25836-25848;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=Y. Liu;citation_author=S. Y. Du;citation_author=M. Ding;citation_author=X. Dou;citation_author=F. F. Zhang;citation_title=The BMP4-Smad signaling pathway regulates hyperandrogenism development in a female mouse model.;citation_volume=292;citation_year=2017;citation_pmid=28572510;citation_doi=10.1074/jbc.M117.781369" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=A. Mastretta-Yanes;citation_author=N. Arrigo;citation_author=N. Alvarez;citation_author=T. H. Jorgensen;citation_author=D. Pinero;citation_title=Restriction site-associated DNA sequencing, genotyping error estimation and de novo assembly optimization for population genetic inference.;citation_pages=28-41;citation_volume=15;citation_year=2015;citation_pmid=24916682;citation_doi=10.1111/1755-0998.12291" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=N. C. Rochette;citation_author=J. M. Catchen;citation_title=Deriving genotypes from RAD-seq short-read data using Stacks.;citation_pages=2640-2659;citation_volume=12;citation_year=2017;citation_doi=10.1038/nprot.2017.123" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=N. A. Rosenberg;citation_title=DISTRUCT: a program for the graphical display of population structure.;citation_pages=137-138;citation_volume=4;citation_year=2004;citation_doi=10.1046/j.1471-8286.2003.00566.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1093/oxfordjournals.molbev.a040454" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Shendure;citation_author=H. L. Ji;citation_title=Next-generation DNA sequencing.;citation_pages=1135-1145;citation_volume=26;citation_year=2008;citation_issue=10;citation_pmid=18846087;citation_doi=10.1038/nbt1486" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=E. Shiuan;citation_author=J. Chen;citation_title=Eph Receptor Tyrosine Kinases in Tumor Immunity.;citation_volume=76;citation_year=2016;citation_pmid=27811149;citation_doi=10.1158/0008-5472.CAN-16-1521" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=T. Tahira;citation_author=Y. Kukita;citation_author=K. Higasa;citation_author=Y. Okazaki;citation_author=A. Yoshinaga;citation_title=Estimation of SNP allele frequencies by SSCP analysis of pooled DNA.;citation_pages=193-207;citation_volume=578;citation_year=2009;citation_pmid=19768595;citation_doi=10.1007/978-1-60327-411-1_12" />
<meta name="citation_reference" content="citation_journal_title=British poultry science;citation_journal_abbrev=Br Poult Sci;citation_author=D. Vanmontfort;citation_author=L. R. Berghman;citation_author=L. Rombauts;citation_author=G. Verhoeven;citation_author=E. Decuypere;citation_title=Developmental changes in immunoreactive inhibin and FSH in plasma of chickens from hatch to sexual maturity.;citation_pages=779-790;citation_volume=36;citation_year=1995;citation_issue=5;citation_pmid=8746979;citation_doi=10.1080/00071669508417822" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Wang;citation_author=M. Y. Li;citation_author=H. Hakonarson;citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.;citation_volume=38;citation_year=2010;citation_pmid=20601685;citation_doi=10.1093/nar/gkq603" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=X. Q. Wang;citation_author=L. Zhao;citation_author=D. A. Eaton;citation_author=D. Z. Li;citation_author=Z. H. Guo;citation_title=Identification of SNP markers for inferring phylogeny in temperate bamboos (Poaceae: Bambusoideae) using RAD sequencing.;citation_pages=938-945;citation_volume=13;citation_year=2013;citation_pmid=23848836;citation_doi=10.1111/1755-0998.12136" />
<meta name="citation_reference" content="citation_journal_title=Bone Res.;citation_author=M. Wu;citation_author=G. Chen;citation_author=Y.-P. Li;citation_title=TGF-Î² and BMP signaling in osteoblast, skeletal development, and bone formation, homeostasis and disease.;citation_pages=16009;citation_volume=4;citation_year=2016;citation_pmid=27563484;citation_doi=10.1038/boneres.2016.9" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Zou;citation_author=A. M. Teixeira;citation_author=M. Kostadima;citation_author=W. J. Astle;citation_author=A. Radhakrishnan;citation_title=SNP in human ARHGEF3 promoter is associated with DNase hypersensitivity, transcript level and platelet function, and Arhgef3 KO mice have increased mean platelet volume.;citation_pages=e0178095;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0178095" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="Guangxi indigenous chicken breeds play a very important role in promoting the high-quality development of the broiler industry in China. However, studies on genomic information of Guangxi indigenous chicken to date remain poorly explored. To decipher the population genetic structure and differentially selected regions (DSRs) in Guangxi indigenous chickens, we dug into numerous SNPs from seven Guangxi native chickens (GX) by employing the restriction site associated with DNA sequencing (RAD-seq) technology. Another three breeds, Cobb, White Leghorn, and Chahua (CH) chicken, were used as a control. After quality control, a total of 185,117 autosomal SNPs were kept for further analysis. The results showed a significant difference in population structure, and the control breeds were distinctly separate from the Guangxi native breeds, which was also strongly supported by the phylogenetic tree. Distribution of FST indicated that there were three SNPs with big genetic differentiation (FST value all reach to 0. 9427) in GX vs. CH group, which were located on chr1-96,859,720,chr4-86,139,601, and chr12-8,128,322, respectively. Besides, we identified 717 DSRs associated with 882 genes in GX vs. Cobb group, 769 DSRs with 476 genes in GX vs. Leghorn group, and 556 DSRs with 779 genes in GX vs. CH group. GO enrichment showed that there were two significant terms, namely GPI-linked ephrin receptor activity and BMP receptor binding, which were enriched in GX vs. Leghorn group. In conclusion, this study suggests that Guangxi native chickens have a great differentiation with Cobb and Leghorn. Our findings would be beneficial to fully evaluate the genomic information on Guangxi native chicken and facilitate the application of these resources in chicken breeding." />
<meta name="og-title" property="og:title" content="Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/379" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Guangxi indigenous chicken breeds play a very important role in promoting the high-quality development of the broiler industry in China. However, studies on genomic information of Guangxi indigenous chicken to date remain poorly explored. To decipher the population genetic structure and differentially selected regions (DSRs) in Guangxi indigenous chickens, we dug into numerous SNPs from seven Guangxi native chickens (GX) by employing the restriction site associated with DNA sequencing (RAD-seq) technology. Another three breeds, Cobb, White Leghorn, and Chahua (CH) chicken, were used as a control. After quality control, a total of 185,117 autosomal SNPs were kept for further analysis. The results showed a significant difference in population structure, and the control breeds were distinctly separate from the Guangxi native breeds, which was also strongly supported by the phylogenetic tree. Distribution of FST indicated that there were three SNPs with big genetic differentiation (FST value all reach to 0. 9427) in GX vs. CH group, which were located on chr1-96,859,720,chr4-86,139,601, and chr12-8,128,322, respectively. Besides, we identified 717 DSRs associated with 882 genes in GX vs. Cobb group, 769 DSRs with 476 genes in GX vs. Leghorn group, and 556 DSRs with 779 genes in GX vs. CH group. GO enrichment showed that there were two significant terms, namely GPI-linked ephrin receptor activity and BMP receptor binding, which were enriched in GX vs. Leghorn group. In conclusion, this study suggests that Guangxi native chickens have a great differentiation with Cobb and Leghorn. Our findings would be beneficial to fully evaluate the genomic information on Guangxi native chicken and facilitate the application of these resources in chicken breeding." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"V9vV3id3e7uSm-SEx4LR2jCul8belWWrE8A5aZO4-aM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495280","apath":"\/ggg\/10\/1\/379.atom","pisa":"ggg;10\/1\/379","ac":{"\/ggg\/10\/1\/379.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/379.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/379"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/379"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/379":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U5yrkTKM\\\\\\\/glp3DTCzQdov\\\\\\\/2E+dweRp5Th5LXsSNQl8+xtvp2wSUOMjmsVso0Yb0Xoz6Mduc\\\\\\\/ktI0ih1f0tNvKiP\\\\\\\/bU3lWKGipcPHrm38+i8vONdK6ZBGbZ\\\\\\\/aBMQmkBV4A2xfsIQpnKKhwYUgxhz9f3HP0icTwVO73XMMO1fytRE1UA5+0dMh2bnOTO3EDYw+mSV8NNn0j9rp9TJ1ig68g0skVnScwy0M\\\\\\\/ZUFrm21KTs78sBPe9\\\\\\\/TvP423RQJxDfSwny09nRTjK3UyPSKCPw8Vxb41oqW8yg5dbHgyQOUT49o7acxWxEPP\\\\\\\/Kk3fctSRcvgYYTRUvFSppe\\\\\\\/AcY\\\\\\\/bHaJOwq1NAMkLFd1U4jakYeOAys+G7dLOhNjgnxRbaR2i\\\\\\\/O1gMMl+CHsHxTQCnye8ATeT\\\\\\\/mSiaxdMRMul\\\\\\\/V+wSCT5ZLVYeKQMcHntziPMfB27N6aE8cz0Im7\\\\\\\/RzxS437nihtWgRCAsHn9mpnXaGqruXTdsRaLmhtjXNU228kDLRP8RGDNZ3\\\\\\\/\\\\\\\/2R+CM3KeIBwNE9gwCrlLnBuXE3JLMPtudJuiVEGc22zzhJs0lFvt\\\\\\\/Vxu9KuIwzwzpHMzAvVAqn9fnjDNFVMoH\\\\\\\/aiCOL1K3NHUeEp69LdemRl6sSAMjq4aQJAUuEE\\\\\\\/6O3KoNwm2adGV1O+nmNdLxr7kSExvfvxQ0DR1TJ3DzZiKiQZq5R2ja92Ijb38fHBe6wYxwV2iamqvgCsFh4QI02w2Pl4\\\\\\\/ddbxgxwvyy8wEtemEPTr2hOkq6m8ZYH7JgiGfPxdUyLrHf+8YTWwlSvywDkUI5M2h8hYYwt4ScBVoekMBfG+kdpIOHHrLW\\\\\\\/rQl7q1c2Y77IkzLdmYtXhJB7OYgJTmsKqCc+Yth1LPilzBPCeLgBBmWmZcb+JHxDA2tKWaJOmrCDBOrxcy04BZJyBD\\\\\\\/Z9FbmNzzrFd9FJhOZ8KJ0vk+v9hADdNcg0ldnvOngDUW\\\\\\\/QK8sD03BSnA4IFaO9CH7r5hwvLErPPazeoChsJtkCkHfZAUX7dsaxnUN+QtnE2ejcFp6PP7YUQa7HzkFzNAp1iI0NHHxEr7bzQ2pdKxeTkHWOmWd44HRMkCrEoLgqLiv5nrCcnwT567PMTtCRkfIsVAjckpvaOQN00AgiFdC\\\\\\\/JyjLtIe++91+Dh8s0ic5oZiK1S6xYJ7C65OWl8dgAPOKGHnqDcOYT717ENlQLerwn\\\\\\\/pPPh8lGc+dpCnrNDKLkBIIpe78rkv6jDaXHAphMDTWY3UZC8JGiv2c6qwrHEqwocgzEz4utFn\\\\\\\/yPmk\\\u0022,\\\u0022iv\\\u0022:\\\u00222VMqUWLivuiaWiWtuK0lmA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d8f722932245daa96fcbc26d2d6fe3245365ade7f371d22aac35d997015ad95\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dXMFWmPZSzqtLVm6aNGCJfrVeF9aci1thK6zIBUIOvSZielp6JjxP7vlRWW7WTn3zMD62IxAk9Ep0ff3OdJhmN\\\\\\\/Pgmuzkbc4LFMouIUc6emX8NbYiHShlb6PzNZ9898NrW\\\\\\\/1828Et7RSBJYi1dbdb3oulpYv093lbGRE9PfJYGd99RAZ9dJhEKM20uee6mcoQO4LqW2NFgcb9aBxr6XsKjziP2PIhuJyaj4O4FybwQB8RiAxqU3+Syo0O1QWrgmaKINVX9eLP20Ifs\\\\\\\/eJTeHMTZxwphzUPjVVDFdzQjJElw1qTho6J8Kz3MSQAWRTZWU8FwYlW87pQv38iD5sfsvVV7Q7+l1H+rbWUjBcAJ4AzhhKRSYBuwPqCKUjysIaemXv3Ij5vLZTB0rGt9LR7OLJALtNlj\\\\\\\/kMBPLkiHFzY5LvahvcLKb6MUYZTxA1ycf1fIFfSQ90jHf+D3zVosTRiBX7sQA4LIOPpF8nKPnY2Tubcf+0gidX8E56DUTu\\\\\\\/s85vL1C6LewDsoevKDQDG2JOWGEL6sbvKOQXY1R1d6VtLXywJdBNdBuTpsgriul3Bu7qswBUNCtPI84vyXfdYLEOf\\\\\\\/IYouq2XIntL+Iz6mp3iMR2w32Q+LF3M6zQaNKy0bT3x8aURNzsrANXtVAkvmtvnYmbQ1WXIlEHuvKEBQ1yPWlJA+AjlJK1UaqAb+AubBXQhxIqq1OLpnxjMUuKb83Pfy6pWqLc1mBYvyCKpXeOSvVI7Ts5Fv+tnOyRJ83RcEDNtVpv9p2okEckqg+K80aPjt+BI08zpHDHlvoFPcYjFRivLovAt9hZXrJOXVq+xMmngJR+eCQtVMSrHe0iViNNuTN8NntA8mB5P4Vz9QsKi7uCYG2hkSTLV96DFW8zhdoZx+LykyZjIMADhZSNOcoKImTN1IMHQxTIvAsp66Cwetvejifk\\\\\\\/ajRPNKYI+GaWcFpK7hUMkILsM55CwM8hnyhN1iXZ3Z+oYXM4kNkfd4wzmg3tiZKBVvG9wvcSwwsVjx\\\\\\\/7tjbtjmhDQ6RPZsHW3lq0+Xat0vpiIiKULqPNeQDDZ4llqqnmiqfE3WlEHP8yTKYj+QuCFILSOVFlF6pzJ10s3rDlUOWquqRgOQAsh9llglxKKfdJ404121gUbwFE22OgiBqbTNOYV5SsxuFkWeKeOPVZAE+\\\\\\\/KpEFSZ23SA0oBM8TY9NKmlssLngKruNNgBWTU0AQFahfr+su2h4hasErQ7+VPPEw4yNZSbtSCIFSrdduv2M5sbLWtXRGxJiQzuTlFhwU4odw4TbBJke\\\\\\\/jm6dwEAfPH0fca\\\\\\\/3Oq6QMrS\\\\\\\/zAnZvH0fBI4xWQRBS5uJc6XN74xLk3EvhcRc4LRaQeVOBoaoT51MDPnNZIfFqz3nwHpKWnGjt9qEzmJjX+ccRxW\\\\\\\/HNyFkRFzBlVPM8pKrCztcC7f0hgBbv1sLixY6Y2Xp8lMZGxlFirLQRYqOqtR2oyKXHhkmAUShAvhbIb3rvIUHGJHJlGsRDICfXwKBGbASeUfTKSq8FK\\\\\\\/PCcEGrCL1EgrdGitWn8MbWMOns1wimt8BCtSHEEdebA0\\\\\\\/5YmBO\\\\\\\/Ke9R2Ua57MC58cPKaYdEzHd3+rsbZ7peM3MqnDAOD02OPunkamPCoMkd4zwEvz1t9DLZdvmtht9ZF\\\\\\\/XYYUgb51yhWXlaB5Jle+r03jverFL6802NgYn\\\\\\\/27ver0Jpw1lfkxG6eSyOJD6BOsOKey3XTcuQpreQWD\\\\\\\/+QjbBpVzup47iwDrUYtY2u3Gtg933DhXUJJTMpE0GK7UjtRbOVxAz30wYxH2YlcSbEvW3j3SxQQBAqDSdzT7GZ6XhMmsEvBrfpryD0MqqjA1kMKyU2vFKRCzerREvuQ3XuqyBllFDTGdjyAoHE2xvtaxmEuaFr44WzIPww4MJZtAc9uPhNrkX9ibOqbrfQrcO+CChlHnuVI3KrMFb2bKo7UB51B6dKGmguY2SCyt\\\\\\\/oFeMIbkvssDI6csxJ4TbF\\\\\\\/vC6aLnxsWotYiVi0xrktfv2Gaj2b6xM4KpwOCGGklXUa6l9bWJLBGI4KIQhzcAhRGfzoBSbXGEXPYkxlyAM522FDn1oa1gvSxWtUptdBDsdkHsWoIS7HrP8rU26ObsqGe3m47hfI9\\\\\\\/O7qulX\\\\\\\/hEEEVjcQtmPvRdsWuP5iuDIPr8IdWJ4LzTpl189HksYJiEUp\\\\\\\/Wo8dmSijZpf61xCg6LvRPlZ\\\\\\\/nDtpRFWBE0eYztVY9k1v6uq+NapV1k6DlmzNw3z60\\\\\\\/9YWb7q4uvdCRMZqmjwyJP4h20UGLmlDX2fBpDwOOlbuITVOgO77M1CaMjv2ZIGXOUMwFdsJaxf6Q6AhGx\\\\\\\/sbQTgpfxT04Q4eoXsBD3PFMnqsNgCi5KieLZfYkaeku4DI72wK6jF6CnvZcWrQkA3p8k61CiaYZT8FFv5tS3f0gfDqqCaCKKipL17bpmpCIFHfFyZlmLHfuQ9WgVi1VASiYkL8kykgnUnGYfjc6j\\\\\\\/ZmkOv06cRoTAB2mKdxqYH8Ij+FZ7U7kcwKn1X7K5j9WEACzMWKqBBRLojdy\\\\\\\/ANwo5TikBQcnylmpz4c8tik\\\\\\\/plEzvgJ8Ho+9AtdYeG4u0YoLoiWVymB97CpicuSLeQA6Roc2ykZuIaNwkr144fEr3XVGiQ37lXYw0OoPd1T2jfoculItmDq00JUan4vQqbzuYJMwaNsAcRwsHTy5I8aRK5GqsEo88d5tN1EoMPL584wetnvwEkNIfXDoXZz2fpn8NxKwB42x78TKWNwDSDDyQrP3LQTunH9VMo\\\\\\\/mMCIc7twT8ikdqHfVBQgomBgp\\\\\\\/MSjbsPlI3dtwyiVVz0ICzwMDqIUAvFwQnXStaMpmsI2Tg5zBvoWMpddsixuloy+MKnkdh3qfQiTz8lLjiE0ya7m5REQNjFjIutMOV7BCQjtoEnG9Pqhz8CHOmJFx7AbS8MY4N4swHEBaP\\\\\\\/QkTLbf92wvSlIbP1SkUfXVXgIdf9RAMDExMe3ENNOEIJoMWizn9+Tds94qGkcYugDygk11V\\\\\\\/Q1kE+TdEbY8xk2aV97ONVu\\\\\\\/6hWQiFikt2Kcb6HGHSJssRaloHiDKt2zViOuAueXV27NiQ+6j5f0Eu1ti8wS88HvAm2izVsjdwoRUf5xi1UTWQqGeiY6Nfim+Y1WRI5rYJtiV8tsy0v+IKBXJHs0yU+WmjxDESUIDbzvDTp4loG53ZKKNf2LSj2x7vqj3TMKVOEgTGm4hqBdB\\\\\\\/Fq91m22Vx32Owo43Im2VMopv8sFOE5GtbMubqo+T7HlvxuQfLr8deZNFP+v5YwifdsVtmdaMz425XlnJQBtnGLlPYJhH8FQgSDcFzv4Q2d7QjzqkUQYLlQQ+T5aTek7+R6oybJsXsHOUSbUkiQrW4VLHl5W\\\\\\\/eKcQaGAWE6DViX4vMRAg9NP1h+RkoPWdz6BclGJIgyqNwVPOcPnVCqp2TMCCw\\\\\\\/OpdacCAiUqjevTGliiJz8fpJGcp5e87ySdr7S4jsbzWxr3zcPV+rxk\\\\\\\/IiNZ+kzIMxQQPs4Fh9ibftVsoOOS+xg18uO17cGzcm4PywPFBSwP1DuzYUIqQzE+idiwTJmCCQNKEaHk2yqejP5BURANBfkVmSsy9UzHbQvN5xlQ3Wd705DMeQN1NvI8uXAXGO7Of9rZXNfH1sKNkqGithN+M6lNNUjwy+0HhG40T3eTetMspvW22LfKSv6PLTRmqv6mrYCuEQzzBqoA5mvgVC9A+7F03+CuBYzTDnJuTSkpCCOsKoFH7EDu4gzmNJPZOKAcjcLIiojY0wj630grAREVvDWrnHJlrnjzara\\\\\\\/KqADE0snVdbyXU7j\\\\\\\/pDNiZwEQVXco8L933QBcuW5InQcd4NsY+D+st8pqux9Ufx6Ku3Yufl\\\\\\\/G\\\\\\\/ix6VoA7ooL1Sh9Sk9og5qCzp4gkfLbo63uXm0t74ORAKyYbY0Do2aovuJhJ8nlgGcYJBDshEU+g6bVas0gVpqCW9C0qlt+63X4eSV8IUaDbCyUfdEyNTXeQEdT1zTn\\\\\\\/T1gx2mTK7GoeYD\\\\\\\/pYk6M1zk8BBubfu9HxUnlyReqfcMWKNvHyTifbbXN7FTZxcj+U+K9G6\\\\\\\/bQ3YTkSoRNq+divV0fg20zmEC2KT\\\\\\\/k+Bp8mKYu45+yJFCFg0VowmZkqJq\\\\\\\/J6sfwrrZjsdh4teuS0QweZOPdhRs7u0H7T+NxhYKVSjvLznTk76BWIOrJgCCvzDtnTfNoaukUe\\\\\\\/XIv1yJhx6RAHulByKDBurCbDP4NdO3ys4Q0b6hauS2iJ9UHix0e+45Xm9Bp5GjEod9jW3lnLa1rXo2PaLGZs0z5TyDoRynwTe6HkpkSGHgCVlMDXrzxhiupI9VPDu6UaS7RIZQGDD\\\\\\\/Ip0212YD7CHDuk70\\\\\\\/jcP+dAoQ+fb+0pf4xjBJ4F7lHJ4vlMLgpgvHUAdzR1Iz4ec1i\\\\\\\/izCxeIGa8X3gabO\\\\\\\/n8tC+P1O9\\\\\\\/wImGL8fPQYtMFe+4CsVoSFBoiaSONGv5hVaFFtsgPhOtKWXcq2MJkTxmKR9S\\\\\\\/6d6as7z7p8vU1VHKhmQVEUlZhgF1r4XtDm\\\\\\\/R+d52UeLrCbOji02H+M8xA9e4\\\\\\\/60jsLuVK4RUsxF9Ch4+A1i7MNt\\\\\\\/xXu5JWW9cra9PzaFKJW56963\\\\\\\/lqNRnekj++F4p\\\\\\\/hI\\\\\\\/fOtNVGkqv1xXe9Dg4Uht90IXdYZx\\\\\\\/D1ykgwVRjNOVu1F2uXmbE961VeAB30YVQWYVR0IiX0uT4y9e49eBK35xv2ZUmQSknp62nbS7eTjap5J3ixajMvUTDrwdbxEdbvpUi6geAQ9O92nSLKURd\\\\\\\/TuJP3ohS3SImKe5tTemxqpPCr6aMQ9lIqHOO7fjFzd4MJazG64OXE0L0m2uzIwovB\\\\\\\/hstTC+JJbFndny4mXQ2qk4DqwYAg8cFT0EkFjfSY1oM0DEx4kvb5xLngpvCHc37jJOJVPaqoVUtJXwm0CLJkbjP9UAw30TV+D0if\\\\\\\/\\\\\\\/YtQvKKFQpOXY+\\\\\\\/Poc2xuDpq\\\\\\\/TlH0TMK8\\\\\\\/i3M4Xuzv5abJ2RhukJj89AdL6iZOC1pb7Q2HwlH+m9AIcNdv7Bkg3Ao5dYnMcx0qV712tn9aHrs3m7cR6XFCIytJJmKYZWEI3JnT7+O6XXYRc4ftyK5k7jnPfBmDAoiINO\\\\\\\/mPYAfWrar9e0euJzq6hMTlIBvXsmeqjVXuQ+P+a13VWDEpwe0Pd5FKHXbfy\\\\\\\/jAwg0N2WSfDQMQjWopa3RqHfHmYlsKssde732Ky0sUHGVOz3cgAoKEAhC2dmdmP03fSEDRmUYSFfcZSlCq7hEl39u\\\\\\\/W6euVK7mceF7QaHKo8ox4nL4ciqKoeJSuwkFDR+2BKBVpsG8+X8vJm5mRRP+Snv\\\\\\\/B4akLlqnbU4BAG6WijyxJcUegqi9PD\\\\\\\/WlwRykcv0nzJmAgg2tGNmNF\\\\\\\/QLT\\\\\\\/1OoHnFTOoJXOgDVUPp0DlACHoY0im6h1NbAlVbN7\\\\\\\/8lsu1V1eTG947Zju2D7daHvj0oMrM6bLxcnwxyGlWJjBuZGkQh7x+mCuoVNsNvxsrgRviLLrhSCoNEKZqjvKnqBEuN\\\\\\\/EjmO7kHFLWL5RJ68h2bAb9I1Pw6f0H0QdkwRJSx7sRD7L5yC5dJMih8uSUsJk5HS3lGE+3\\\\\\\/q81Zbf5xxdZNtbITfCIl1ic3Neoi9ChHsAcJzLiV9Sw4J8ocTtPZPtRmVMKkAMDKTmoKAuTiZMqmgN2OL35VCGHEaFRz0C+\\\\\\\/fcrl+4x4h9na\\\\\\\/MGFupPep2ESIpoo1N1i4DETDi\\\\\\\/AgH\\\\\\\/m+e5wkL5f\\\\\\\/XrRxA1wMd6Dedse++stVKumd4HIsUqu35LqVxl7m18+kmbfKWuGeCTbD\\\\\\\/q0JFlVgrcR\\\\\\\/rZjb8loQpgJZynMWPfyrLDS6JZ+lRLki5smms\\\\\\\/JStav2nM6EdhAhrnmN5tg5iGu1onXcENg0i6Gx7oYaXa+vrqFJsEw3JBd87l\\\\\\\/a3EsvohDYQOw5Y3mr339qdKXGF\\\\\\\/RHjW6djxon2MuubauNoUzPC6VqQRbnhcvw58Hj7CbX1E5Ds3uJaLvV8bVM1elNoy1\\\\\\\/m1TjtHrgFqNwRAewwhA7313UpbBaStHu2JcGBVEUYLtt3TUO1JjkQyvbtK8WBR0fnQ57+7xzIQD9pev38FlHEcfqlCshEoUOrPwKkHIw7pW040pfF4QojdORwnwIwdKZzcUiyHn8DqOF5wHGqZPQEuMhazQFiCNfuA9Xv0+0iYOYnT\\\\\\\/xpWllMrp7N+SuD6dTSq2IOu1dxHTB1xEhuGoy9svRMCbF9eu+LDP+qDKyrgT8ZTLlAxG\\\\\\\/cQacjv5SzesMf0lGiCjlzrXrfOynAbyXz9cgXUTvyMRSZMtCM1Z7HVC9uN63BOMfsSGlWQv1Xe2VL8LSbHqsc6HTqNi7Y7WvY8ktC94SrbzHEhXEaAqRV8lVBYlNWEt9BM5Wk5841Zdwi9uGOIpKLGPAOMNzJs3XOdPK\\\\\\\/6x66II9D4PQN\\\\\\\/756GxGYk7tNMmJIIlp377u9ANI1MAXxh37cyGzBUsgiACVjoVeTPBoRjYOHVhBAvzHZB1iMKCw+CTXMhtVP3mLcMVpBl2lCzD5hTPSTa1Luh29wnlmX75o+DNVm5MiEfht95rtT605Tj1kISeAyrrProhxxN44TiK1A1Va6MTn+uc8hFhqb6j2UnecoXeUqsC3yq5jbkvfdgkOIzWK4WqAyQlow6xUjzIXSo59\\\\\\\/9TF6sGF4GVaKUSuRgrrzNM\\\\\\\/hBlnnDtcXNCZWawu0SYZEvI4LqBchrre5VETwPXA4Q7i9b0ANNjieVRRrNEOpAXj9+IXIPnZ9H4oWpBBg7s7jVxhXWx6UdJ4+XuJUXOD2qg4\\\\\\\/AyiGcSiC\\\\\\\/FTRKuQWwQSWElGtoE8HF8Fb+05i0LUhOEc6FJr4E9LinsoyWH1dL55Krnx5OW0pVIu5jrOPzlhRwClskS11vCwUKHbiuofzvK1wcuhBgsQaUEDhz5zkEUd\\\\\\\/ftg7SmD\\\\\\\/aMPHhCPIA3ywGT5Rt0WMlToOfM7dE3oVS6VhbNN+jYt9M0KxC5kUYqFwPAKJ7ZE0GUABd8JKBJ1PO+Ht67OthLs\\\\\\\/KLqPhIXbA9eCExV9Z6y+o8wY8btizBT8qVWFB+8P4OGK+k\\\\\\\/UW\\\\\\\/j0ybEIZaI3BKfhDE3LEWjIboO6mdXFt8TnSSTGnQ8x0OeK7QQLpvJwVNyYLBEtvXePn2i0e36r3QNY+5HOIQzAxt\\\\\\\/OZjKV9oZWBWnak9gTk7C7bhggsTeuAfM04t1DmB9NC5wPyuHTxSzd5zlyoxD00JBu8HvxtvZl3qVIAaTxomE5rzikyxiLp9HWrHx29py7kJvBzmI0qgZLI8O2qwnmO+w7mxEUn4AxrBZO1UjDVFIkob1ELTw8N62EE6s7WzQPud9DwNzPiyPhGRJDwb5rVpEOR19gHNL6xoXiPIhAzUs4aSFVjkVPEP+Cc0ogFNFIfKL47CkAIRjOPQ5W3d7lB1ijzuy\\\\\\\/fzyQ5GsCYGipIY6bB5NxgiU+HHdZpWYJeu7svfhqvIO7CnXAdVcuaVCihMF9+RnIYD3RAfA+qd5yNbys0Er2Fwt9DHgylHuEitI3j7Qy3lFwZhM3BqfIYpDRkL9TXoxq5Axnfuv6ymyhN1XxGSFgP601IOg8zx0w7MqGG5MsrnvocNqMoIwCFkwPEiS54uk34o1g4vETit9VsYGw0keS5pmuyok7YSuL7nAbql9Bp1YujD2iTLouvHuxLK+6XVc4GDHRYMy9spVyzuDyuAHkGd4KpvhhQ7xnYO1K5ZDkU9uGI+HrzAD1yLE8Ri8ApoMBugEwyKjEGyuJwJsYkCqWtdFZCoD5PAwjz8u6AjdVNWzZQ50Azq0VyK08s6ukQgoN65ppXhiPGnTlRkXy0i3xGcvmY8tya3oFGwPBASLyOKmxYZ3i0AWSP0M0V3o27l3wJlZ7IPIFPWGJUlqrlyk\\\\\\\/2ROhPHq4bo0a0YTgS5ZO4fxKEXgS5UEdYjESJzEW\\\\\\\/0bVJBWxeoZ8k6OpzcjvuqnUpAPXtwzGkrIKgd81nM7IEXCMROe+sWLAvn5onw0XNQMk7pEUOL0o6tgCUVHl5lRZ+QNU+1Nim9DHiza5WfKe\\\\\\\/vZzpCJBxb6HBRpYxvs8hvteN8MBeDdbzOrP7EP2tUY79pQneX3K6UcD\\\\\\\/RXZepSg3II0q2t4F0l7nOaTyXiF1zSav3pfIraJowZpLKi8aZ38j0lye6JffR25NMP\\\\\\\/vXd2CkxEWuoDR2Skoblh9HpER8g3wPe47+YrRTZoL9S7f9NvGW3RxKTvqcOOX+Yq2DuKXJO\\\\\\\/DioO+Tb3qOVwKA1fHPMq\\\\\\\/9DixXcRINrvLIKAjBKFCDIpIy6eSXSwKuAOTYt\\\\\\\/kjMSG+FlgQU+DkwK3s2sVfAZpllx1zNmP9bGjs+i5ovtyRGYnnOMgyXFtEf5ZfyEeHxdkrZcIjZbakRixlw07rCK\\\\\\\/0v7yXvqR5gwhGtGSWC+wt8JYRgHLUg5j\\\\\\\/Fn7\\\\\\\/MCXlSjueiih3A72LPTUcypd8xInLzOq2+2L3NZqRDQyr\\\\\\\/36xyMovOjedwS\\\\\\\/zF3N6PVnygx\\\\\\\/970lG9xXswWGzE+CKCEkQg378i3KUJU69ygyFbvfZ06aASj34+hTF9eRGdck6Sxtusik5ibwKSsXrrfPJtlhq8PLvxJs4ZPbYofcuR1TWghlDsfdCISFTvUNRi8OwmzPxEC+nyHaGdJvlH8ayi4Lt\\\\\\\/WHpk4y\\\\\\\/32P1qlm9shO3DCTbajZRYBvfgLwd4ghAt6+9nI+V3nwCAJ3ERgCqxHZKpitDZalIpXhVmMehGQNnoT8zhdFAluiBe3qP\\\\\\\/DKxtb6CzZHXmc8qGbgQt8yWel8EKnXEWWxTnkzWj1XDtc4vFrAaLD+4pC6AZeYMGZN5OlwTPN0yLiukvusbLHVMehRFMoBzjp1tpamz9hHXkx1Kj\\\\\\\/sYfUfznIKuON9ynGk4K2RCxHlhV5kYSBADTqGn44Kn8SK0PWi6j9TftjQF4Ibsl\\\\\\\/yBEJHdJwURX\\\\\\\/2JhmZtKql\\\\\\\/srkIAUJ60SoqfyQW02WbeWNRkPJDU9tCqeWyD8XFTyzDg8GHVWH5aR4H00DlJ3945bkFbtAkSx8NLxjzvTUo8FOZ7dSmv5oqvGn6jsiVVDkrFK4EOiaIbzdR7zeHD42kVfSq5WkfwbM42HoQ8wFSPAPlx64nwBpA\\\\\\\/jJOwbLxHwBYT3caDtLk4Wyj\\\\\\\/WIMLmT00YFHe5oaiFQaQyvBrgdxd71kC739Kk9dyjAj8PH0\\\\\\\/0MqgofM5kFOO+KiJIhUtyj0W0fWC1vK\\\\\\\/lzbUFrYAGkPsS1LlYL8wwiX1SHInulB2CVmdEffQjfsOLAeYDKo+g16AcYeAKPJ+G8jcRjR+lSuwd4byFpMD5Z5qBHeDrb41FxPAp0plM8fqiWXpQdUIvy\\\\\\\/JmzKPukfB2OhqfDXjSnt534xuhGUt2jf4FcVqMT9F3qmOsBQNB+zuqI3fsU9t9TwDXLI+oNdXu6djTglR153X9gZZZxLEREccM3uXyJuAqQM4rfMo8MSTAiQufRwSCgM0xBq+C5bIk6iankQDgCUX8oHH6uzP0UHMqICsFglB3JEkEOgPuhVySdeNRFMMIdj35deZzhJldl3WONgOC5UEGBFiAzNuxm\\\\\\\/6UJWOtpblkbi6BDwL5TmJpgXY\\\\\\\/1PGmH+2Q3AIJ48csn96mW5K6NiT3LJsjIetaLxCpFX7zAQF5ppe8VOYSuJDlt4FnrP2zKhE7xjzozBDhUYfEpxlcAWCgUb\\\\\\\/8M9lK8k184XIe9Avjtwyken9LqfvVC6gVqthGmB\\\\\\\/VjFF87n8tQsQqHBptJ7V2XsurkfnbqLcWiTw1wwuPnA+VxjSWd0gu2xweRIFfixmn2MdWiwOvTjLbOYJ42R7DxjIdJPU\\\\\\\/NKKL9d5mQYhCAwW7cQ7qYeQnS\\\\\\\/SYtBFKWurB681VPE1zTfXg+Y15Qfe+ao3ge3mYa8oqS16mgBrg74nA\\\\\\\/ccHehgaQlCCJYHPJN3uaHkLG9vhiQJxD0jeARNjTLjVjySTqupBIotxeKLCMn1QLmC\\\\\\\/ITTS0DhnmDwTi\\\\\\\/R0tyxpNBR9cVLa+wt7c00j7s3UY8vpC+kHny9V3z734xYdfFWTAZ6w+H2TbTIbMhnyPITBZZi2TQk0UpAUqE+OASVQfYINNaIPl36ZTLhZOiTqFwM8cOU5oQU\\\\\\\/nWuhMDha2JVc7Pm473H9wsdJDZVQKajtyODqZxxPK3yyo8NQMkv0uaVyRKp2k0J4ZytskYhRhc8RlxWUdiSDsKgEDpcbpOtbhtNydYgSVKBcMeRNOXGDrAlzNT1GLKnsoSYXwV6ZRm1Ekwuf4MUlbLHHpQRPreTg6Dp0sNcfsK7yqN\\\\\\\/sr\\\\\\\/2SHvknj9\\\\\\\/Y9bnthXvK7b4OIABJ+g28qxqZUylJbNJCJ9dJe+swqWOgMvxZmxEQH2bpAuIEkTauiPQE0vbcEXBPmJYmrORnDDxMUSYvssNxx8cx6zWcorulvFV1t3zS5lmice3qekAUOwEY9gw0iVbbeT4Bfld6xBeLiMbAlfJvPuQE4ntrm0V5aDA6Id\\\\\\\/YGpfZ5T1XnelBViuaqAf6YKdc1HXiFvOYFzTwcHxKf554\\\\\\\/+zfUmOpgQ0B\\\\\\\/scaKYbThJnrfwcUGtPad5pcxjLmWH3ueTfJ7KDHMSPp5bl0taVABPtTXfwUZGA2RM7BrPgHqmKfjGlKjJBzmbFzwXo1EqkFeqFDgN4IPYpctTWNp\\\\\\\/OFWe\\\\\\\/miWKSSNKEuaX\\\\\\\/\\\\\\\/95F6JJYVsX12o8EP7nW+VkFheeO9btGm8nCn\\\\\\\/tHqFR5Oq0SRdmMBeRpoz23kVzB0BF4GDMxEiByffFr\\\\\\\/HXCP9j3gvSlYD+8S0LDCIH0D3+49LhS3DRtTzKM9+S9LMWS7+YFDY\\\\\\\/uu96Ll8NWIQ8k4fGL9jbPnkz9SmP7EQlzAk9orVWPh+9T5laS47iqYU69sa3zqJ8PPRU9n3GKxgPLwSVOYCHOf0FwNm6xvxO+vbjdZQFIbpWLFoRzJx7\\\\\\\/E5v3ao6kLxnUlA+581tYNhrc1Le9RjUS3TnY2j2+CQKybCFkXSaZ\\\\\\\/DW\\\\\\\/EH1tXurFc4hhmSY+7E0VhpHDv7OvCcR9xC4+\\\\\\\/xXUREL9u7m\\\\\\\/QKCvHU8pHz8WlXYQCByJSUq0oOiazq211YbvyInghioMGbiMkz2G6lb3EVnWcWUdXxr294ePkNQPdiqHTOkVwc1fZgeULVL3DeiK9c+a2PN6zWT3c9k3lhpBN3l1sTlBFMeyh2GcR2cMPgQHRusEyjuEs3uH3Vn8X6jSQ+\\\\\\\/UhHBOme2AG5hbeeq3JUUVFNznFFlOsCDrM3QymTNZbiFxnDl6lqlOXM+NUXM1qqdqMQ0F3vv9FXEHv3XTajJ74oPMgpGAkiDkLbjsaYenJvIabtHbr\\\\\\\/1EmFaJvu7XUySyewtAGIXpEtMwppkNvdV0dmnFbyyRUzhNDSZhUOz9jAyFruhmXYDChCl\\\\\\\/FXT2z28pktmlMyF+lPDa5hItm1lQDxy9PMpGSZkRL3+HEnikKaS6bUKPevkI81sYXKarhYo+INHpzzSs66oGj3xdplt+IGeoYZO4TsuMquOUu8K12xvC1REaNNtm1fwjHYjTw4z+RdeIrOJ97rjnvZKV2rJUM1+DEI6D5R3hPFd7cIvI0AQcZShQysDoCSySNaakCPx2QZ4gmm4eXNmJO6T\\\\\\\/Vkvn7t\\\\\\\/fg2cjSAAtNnDSrVn2qsNd52mXDteebWnkkswbejfzoP4DuQ2+BqOrjFXo4fXIkQ9VSzcusxkkK9YKulrKT+5XTr53k7YO+Mjs8pWSsWVTyVaAFaMcdEskKUut0GgLlXDKpN8I4DTKs7UxlnyhbCQpvh37ETugiI1FI0Ax\\\\\\\/yYsfnvt2YkCVeDpYDhUV1nfNCmLldn8xvpe1rwdSiURlmnqw9G3i255WzcWBGoShCYCBKL4GpyleGHQHqhScg227dArxw4uOxuVyPtFGrUISBO6s8eFmRH9dsCzVQiSaOeKr63od9ZgQu6j9l6s2G7cznJ6eknCoDUE1SPzJMzu46IR9d\\\\\\\/ej9p1nBIAG9h94QQFqqi2OSiiRrzLt4BWAepjCfIoIDH90pSlCI0QQqIVrl7zAOMyF9\\\\\\\/Z4QUWXPfGoYN9J2T6xeAbVBhmVfw\\\\\\\/pzvAz6i9\\\\\\\/csZG5qLmiDSyQ32sBNjT+q+cCC+LvM4lU3\\\\\\\/Ssfw0+lN317HbZf6IsMvp3SUJlYZubt0RqPJVPPU\\\\\\\/Mm6S9\\\\\\\/Z+2p2DSWE8iE37+2F0MSr7w911KkoTRHnKT62\\\\\\\/Tl5AWAutrWSpocCUTFRmoe1XQf0vnb5qlHFMCoRg3OjjTaH4hkjupn3LJg8xU8h5sMxUuo+n5LQw6s7rCkbglxp4\\\\\\\/VrgdRfnE0cHvbe3rg8E2\\\\\\\/MYRYI\\\\\\\/+suDAaEKVBZlaC9IHhX9rtmhdionprBSQNZekKNLHRJkty\\\\\\\/HTPCqH5QyIn6OD8uB5ioQ6LzcMnCk8503jpPgtdoM9cholIdWZgyoEKlUrfhB6zvvC6VXzLnLQ8sjmPce7dMj1YEkpM\\\\\\\/fTCEFf1wzLV5g2B\\\\\\\/VinXFNsN1UPCO65x3tSv2NDOXGY0bPKIlZEDZRQ\\\\\\\/e+lw\\\\\\\/KOq2LevGIA8WczjaXT5rJ3WXhgZPl5m1c+3vHrYU8dlXYA8iJidYe+MRfVH9He28qz01NblyD8s9vVnN6Q38U3hGBTt\\\\\\\/yF1nzhNdm\\\\\\\/2KCuYG\\\\\\\/4UvsAVvFgqtv2HtqZisl+leCSq7biX+u7ZBuJT672HUQQXTNbwDgNNKYiwSoEfNotkONXLtjHFuou+gntZXRYjRNMDRViZECRMjP6uZfkqUyzzaoZCtOOuhp\\\\\\\/fGyDlR7hXHGbYgCNiAS0b6wXn2HjUofAYgNvmGeOxU+ahDCVv6Zjl6AiRXGQi5zki00G2OisOlunh\\\\\\\/6zmKEPKA05l\\\\\\\/JYpuj6\\\\\\\/ABQPWeiCODP5jwb+Bdr2sdCKSXiogwxJtN4wL\\\\\\\/fWs5pXz+7vUyzGVFzDZSIXzgEt6NLHN5S5TEoYjrS3c1R0rG0c4NEWft8R3EdclUTFHPg26tRMP\\\\\\\/8Q+7VhiIQJlI1q7eNtQaCXZylILMLUSaZJm9uTteSGmR07DTo5jH3mCR2CaR87TQMP7wQQ77a1O2w4wNExe9m9n+tkN8A3PTa+TwoS3RuWaZtA6GB8Npgyec3SK1NXhGBKB9uwWGyaNh+cfmCmKNViIvllxhBQzzq6FbiuxcnCphwQlhKIzX\\\\\\\/RJcUE3Gc4klkSjUSQgUTu1ZsaAFoeny2bdRWKJgy76kdSOAt28rTscYo5XCjRLDam\\\\\\\/f8wEVdZgHbUiF3cdGdRZKyfcrgITi6APu4Eu6HNqF\\\\\\\/wA3BaMp55UAX62JdXRBqIrvi2bgn1HFvpNuMpexI\\\\\\\/lAmyjdWEsAReTRWlCdYuGpWqoD2IacqkeZ5y7WBCwY01CJ9G9xQX0N0cqDJKSLtrszQlTSNB3qN45R4wg+Ywdi6gisK7h16p0cgA7U1OWn4u57iarg9bKYiWXPyT3Il8bGEiyIY\\\\\\\/1mQRDIRyR5oQAOJjmadB9qN0I7ev+QA\\\\\\\/\\\\\\\/k5ImTprpUCynHBmFKlxBuXZRNXBRMD+4BgDlW4yjkT3F8iyqSXC\\\\\\\/8GKn2JItnbDTpWGW\\\\\\\/goUstrJheLnY5\\\\\\\/2+3smheC44NvAR0S04zhTTJSGH1Am+Jx0duYD\\\\\\\/fYsLAdMCap0vBRzKEg7n2Ukk2U9SwbhKpyuHPGWRqyiBgHJtUVaWfn8MmGm2LZUDSlSxmm4wsRIH5UKmTOPBw7FYtPaQMvmNWIbFtghzxmsZAeCZ0guDgXswSWYJo9pSS9xYakx4al3g5z1JAw8ZZyOSbyJzU1yx783g2JNBexPSBGVN50BNW0cpN9NggMqvNGwPQTmo12Io1wQgva6LtyZnM\\\\\\\/+XY3B4xadMil\\\\\\\/9rSQfxti8jhJ437lkiItGNzpfEPEpp37hINEKxOvQV6DgTnOXVP\\\\\\\/+IAlh3AEtqxCSmI5cMdd5jkbpQkywQaVhoUlKG2ei6iyXJPBLKbqXl0sdNBR278EOMMTyfqmuMWw7qAQLPUoTcvECB1b9DH4QAmC3KksnLXiyvnXjjSequdNs3qcqkgmgIvyNCV3BGMpDGcrpiuS3S+i1uo7SHVAmi1C0R1QwOT4OgK1c3glrJ28db56\\\\\\\/w8pWZxNsASL9jG63Lm8uksZBxCTWCs8JbXnD1+3eJMOASK9nkiwgO1CEUeEgsz0UhL0c15hzoG6WQizbl1kIuIjcFbngOjNB+DTIdR7FttCaw7+A\\\\\\\/M30EUE7tXnwgmDYNZS5rYR1Yl+UXgNC3v1H3VLFZSDN0fDEDTsxi2aWI3TGhu1ynZAaD9kMl79t0H6QYLRJ7hyQlMy+uCy+LnFmNlFhkL3AuXd7lYJZcltZcKQZSZ3U4jFGY9pJM3hWL36bCe5n93jHR1WvPn4QZefMjfk\\\\\\\/P009leabpc7bHEc2AxVI0QodcPGpec5wK\\\\\\\/ah\\\\\\\/hGN2EaqTFX2CHpWeQY111G1lduuunVoePqsYWd7cwmAoDYn+OkP6x6XPkaHK26bOvgsDJjYUblzxpSEnuc\\\\\\\/80iSJKj+inDyzLK0w+kSrABSwtR7aZhNZHluUYuIVnJHoCUx\\\\\\\/EsrNqXEBtexlrqxJrQvl6sWMTjWYMtOwfCJcCt2bH54mgD4j1H2T8VweyFiSQglA00+7xio6DShemwUemNtu+6hhN2v\\\\\\\/io55rdD8Qz5MeHj4r2CujXO0KgODTQ3eRRmgbr476fVQo8ptZ6F0uS+YLcKU7+fvWGmHGQAq+4cEpv5oz7Tgb5k1UYm9YF0lGEqa+yqeO6pZrI3mVhb5wuytR0jgKmRw7YLxTNatxTsnfJHwCj+XnZNzcDLm\\\\\\\/E6IZl0yq32zbo7CJCgPGFD\\\\\\\/TSh4koeWGolDjGM8mE8M483lwWoQKFQ5\\\\\\\/TFzD4IxHTGkP2m1jz+EeYjfzSG08Dsq5BUS0O2smGn8NDAyphe6fKUv+G1jKUbPtZmfY0hrWLErc+Es3+Pr7MNploMMdeWszSaMDIk\\\\\\\/HNv1+tXy3hQ\\\\\\\/SY1fIWnDo6S9+AOUFFnbS8fOfjfZKOaGQgcmjUm+JFn+bdh8+0lVrrtKox9MtNmqGkIXi\\\\\\\/Xg9Gt31PgNxWmppyEmDhjS5p74KPdEG6rWo5ZXQlmquYLz26qQG2DNiGO9Wf\\\\\\\/mLHpxB0zeXZe5+\\\\\\\/Jchoj6DZex54dG1aKsa+4tJT+PvSctDkyh5zoFKwrMalhlQOY5r3Mkh96BrprLMQPKeEuw8y7J43YS3+Z5tSCcYgwK5LbE8XeGQBEs9fxsyrUJr91rolVs4r485GXmkMUHBy3GTGSo+ZZbGE+4l4QuZj+bPRRZZyif\\\\\\\/0KfuThGw\\\\\\\/ZeP7YGWJmXRaLJ1huENMj7E4vYAMsuqoub9EHHr4JVlDEk+HjiipSyfix02bwsrHSs6uW8OgGFutnkH2MFTmsWvGOsBo6l6jBcAZfmZdqeQFDNHSoCHGvp40QqlVCNMpBJhbKCVEff3HYQvOso++7cn4o6T0xW5fPaa9DGruigICxoEoHFf5TmFdJzi9ekPMUjSx3qG959Bi+CqMcJdeBZy2HYIIvs18Ck29yPetsMgoCYSucqc8J\\\\\\\/OH93KoqQclicwjcf\\\\\\\/uLiLxaNq5rkz5saLkoAjV1mdT63Q3BVCIcE0sDLwslRWkC+TgiHN6P2BYda5qus+GbD3vLXarDIDfyTF2cz8x4jp3KFMKKCqbbyyuSL3yQdMmL8a08jLNx5Ei9DXIdeM9lgsDwkJjSaSRQAJLRTnIzKgX\\\\\\\/x2GqQZrZ2vblrol1e9x8kgm0e\\\\\\\/GCXuyC+Zvvu58Lq8kr7xqlC\\\\\\\/aEUGXXFV65ZMVsWlVf+yqOpOIO17QeXFWoG1SdYZmDOF1BGLqhob1VISOn4FHCoJBgJN+umRY3Fbr3\\\\\\\/vQuQXNXGcMaY4rv2JK1WOeJ1DbocSqMXAdhfwb1g+AHEKRCNyes55xkanxe\\\\\\\/et\\\\\\\/yyBlwabTdoLXUktpqSYIlOf56RTjnuVX0ajAHMMOGaW7886kq26segoYVP0qMwks\\\\\\\/ThnfjK1L6KgjGTJME6eZ8S1Gu85NwMS\\\\\\\/n\\\\\\\/6NiIJH02SmVwtgnZS7ThpRDsDPYM0H9DDGcR5f9isEqJ2UNyYKgjzTzCdSpwUv34HURMGsVzAuZ4UXAMdtWP83Mb+Ajmxbwy1XZG8vb8Awycrxyd3uRkZdzbzfLXaOcwRlzjU5Gsyw0QZmVUjh0DuJo2yji5lGuZPLsLBGQq+qVLg2MK4MRo9xuqNBVW+ak4oYNJJZ+Zp62RJh5ucAWPhniavoYwxQYQ+yxsnFavyLdxXxhIyFJ2CqxBuRuORrmL8fktlu3ZanMfIIWs30FnhuGRdEIxTtlNFVY4VwcJcCTTK5ICVRvPOLUgzOtwSb8a0byNizmxAI+4qcYhnTKnLwgMhlMmycsL+lb9PsHmnfaaViZC9B1B+3Tamgxe9xsOIrzlJKhONiX1b3Ifq6kFedtufOXgMO\\\\\\\/kEAKf6EA3WXTRW7G4BNMMgyFZEp29NW8TN\\\\\\\/OA+0Wi+2Rh87CG3jgnV5YIGAgztcIE\\\\\\\/8J99lqKLgB+ZlzDfnx5vix5NuwdlWzzc7V\\\\\\\/sncBcH6v6vgRZmkmz9Lbabsng08EUHevIkvtK3atl75VeQqJpDO9iIlyOMge57NJ0DRgSc9mwPHiK2W8TJJZh6fkNlwUBBni2IoFCdJAqy1MAIOP5M\\\\\\\/MChwQgUsn1HZ4uXb9Irrqde0RNVzat6psoduljHXbP7\\\\\\\/ELgOd9xTkgGMdjBbsi4caeCfgs5MjwSh+sYU\\\\\\\/WzEAUb4fq\\\\\\\/v91CQlMKeN2p9CVKkAeSw\\\\\\\/bQG92FRbPRu4C31JBTm0rrXkXhppbIkgj\\\\\\\/p2HwfzGrobG3XuI\\\\\\\/sCgjI1zvgU3lBhivu6D8OoTwZovEVpTSSKB7eBM8w9jQeGyHio4Ty2iYTo4ltEmb3JQzAiDyv0VTYe324XeKrMVAJMIXb3YlgHqkW1aZ5b0AcH+fDGklsHB2ZSzXjGq5zKp1UrnA1pZuvkQ00xmotMmSferU1aXEDuXPjjooF7PiREQDzaWPGkaKLx22yXeq35hM6gJEIh3p73D3DnX51G5VDWMWCbZ0\\\\\\\/593CBB3pttFH3N\\\\\\\/gJDkmejoOIdRuSOlUnrfyz4MuFXWX63RdehzcCa8VppJW8tYa8jVVJEvVR3Z\\\\\\\/ume4r5F9i4A+FGJdgTj+ssauzs3MHaWdVHBCHzUDbts1ycB7OAXMf5izddu6LZLUVFUwKal3uhY\\\\\\\/DoYaUCkOPxoPvR8Hh9S5c9OkXtJSTI2LAL7BuiRoDBajlC0ff7NpLX1KYQy0laGg9cVSwpinue32f0tPYQr4usfsUNbGHQwnMZ7q7sAJwOmSTjmn7lBWEW4\\\\\\\/1egS1aIuAOwBH4OlFf1D9un2aq765Zijt4mOc3X70vfLrZ7JBpU4iTmBymYGUxCoMV4FzonopKkDl3NIflela6aH3sNXIYe3WrKpDYtEVwJ49FDKJnf+KYDiIc3vMhNTCluyLzhox1iezIQ6zkezHWgQMIoX65e1+SGGEmvKWVssxxb3VpyQiQ0AEyGYnVrpN5wumnPv5Vq1qvTDxKDTmy7UBgSevyZOkphX3OjvTmlge4fQHgc\\\\\\\/o6audRA3ifjz0W\\\\\\\/M0XjHNXstSP6WOd87PJtt\\\\\\\/8KKoZQ\\\\\\\/QOOf8xnuAJWwcqVMGB1VEqRki6NxW\\\\\\\/jORVfkFaNbUvcMdsvZOAGLJDkrvOZW1XPOMuP7zU17XLNf4borKl6Z212G865w5wJ\\\\\\\/iMVoB3q2jZwBFHGX9W25n4iKe7j\\\\\\\/9U5rMMh7N\\\\\\\/gm2h81Ue46qzof\\\\\\\/lzD82+RQ4zedPaH9LHqGKCPAhQsDL+Df6doCthMFWwIehT3u+xrYQknEGruvNfZaOqmfxyBRR95GdmhaJjQTU2G5s+8c4vVA7hGbFQZtYzBkjk2BbjmJ2q6VfV0Y3fVoC5vNi1+OxW8+Z7w6bS4CAabzLaaBBOl1pnFffPwrcR5DDZqEWL40t\\\\\\\/gN4UzfZdmdcen+lrMz5gZk6aCe4HYCPnWMFgXqL0Xa2Kl3FQfThNjMh2x9Djb\\\\\\\/K811lC5uMrWm7pzcoShU4RHESlVVrK4SCs7a0E1jRDketheHVFQsDz1bw3ozjR9dtMZfRXb3nmXu4QOeZbRBG+F3aW1+5SJ76hJkhwXfmRYNqh00RIYfwffE+AdVlPXpOH+wnv567P8\\\\\\\/\\\\\\\/jzJQPoJvmgLoPeKRNPiNhLjWvWcHeSxv6zwUxLLPeI9kzPpruqGz9pYiOAQUFezyJZG70u5qIiRLktLCdUXLQXKRIMHe3iWpXz8h5o\\\\\\\/XymjW8Y4hFqmAk5APCg6YAwXOTQgpxu1vCputkFeFnfYln\\\\\\\/bEnw7g5twNzrTiQsbcJZRdrqavjOCM0hafLU\\\\\\\/YZjZ6Rr2brb2HGvlZTGUCua7UDxrRVkKwAcnhf90SbH5it9Fbs6xs7UAvyeu+L10L3rLamc9Zf60JVNCUQgYRfwcGi+tku+FbhDEV5Xz+tig+8wyyFj0s2U7NIuWYkbLqyA0KxhZ+hXjb7IZBK2D+YnDxqdYSNaMmtXV7PBhD7KWo0e8dmTNOwtE4pLsXC0DxgASmAz\\\\\\\/lHwu+6HoD+usRuOv09DmxAs1IZ1EbtaC899LEigAT1Um5nxey4vQxKR6agprkIHVogfAj6dsJtkEAfQD1pSTp2Day2cok30KKQ7H0XZxPfyzx6ksjIatwBN6ZMD57WrwsT+Vx+Ubh71PmG3XrhrEceUYTQWtUVmaxoZomUIeQAgvG9smWJbOowOWqHyEhWYuCAP8VtZvcxqw7iQ6ZplOuVRmHjwTU9OBK5OZlJ5Py13AWC2xK0\\\\\\\/e+MOtAN6zRCS0EfYbbz1CfzrZfAJFv7Iuy5pNVksSPGurk6jC7xrPAgNksVhTNlT7hQ14Lo0MGmz8ZKRbNGkZZq04yjXCgxktQjsBG7nles+l4n+c86L3Rf4P\\\\\\\/w5+FWcLNjXeiFY0LzzPzrplAsA76AfC3F50iRtCeciwu57wMIxtmvw5+SLn4ms5IVTcL\\\\\\\/fJ5O2TApKJFgJf8+kxphTLay94zcdwO+LjdtcWFUEjbkc5dyXzfoGEhGlSQFH+xCwAnA18fydVM1XP\\\\\\\/Wgg7UBko17uCEsJEOn7XuWSA0kYiL5Robo9PDTnaI25oAMhhYCWd\\\\\\\/GXMFMu2wlroUGOqcuefR0S43oaUMaXunAIJnZaDahKrNGft+KoLLCTOVRTwSYvnH1gSuQsUVtkizNoc2gOnopEthkZwBkFQUY4MEWgyze8Z2+N0bFHEYL3WzcOTVbX75qKAsASMyBp9b\\\\\\\/fY+HE+HeM\\\\\\\/Bm9lxVLtZVb8Qlz9iT8EHdBD3qJ3dvctPlJfMfYXwyn5k\\\\\\\/si\\\\\\\/INI\\\\\\\/z\\\\\\\/bAH1xmnxefLBxJ7pYAvEVdboaozoV8Q24jygY22uZd93IgJtgDNfCTb5ZfGeLa38L+P+Zw3Ny2bcpqT6Ue95cUL76VBrj4vdWfdXMhHS03wXZrML\\\\\\\/w6aYogEp3s3QT2SmOaSBRyxw9uoGpQpnoMbVKMiEcU5rr4a\\\\\\\/FWID1wHBSiIFtnXmTvec5k3Rk2Yu2WpGcHn0TSTl\\\\\\\/9wmerTjmiwGs0zt9PbKqSpQGqLdi0xFpJhwBbqBqtodA7Q4JxRRl7ZRMxmUag7ALJctRGyfna7dkylfeQbQmr5rz9VGGFNYD\\\\\\\/bRC8Il+7e91d1Bjg\\\\\\\/cmuY6doLXMSKyD6mgWj5Ms7UnLpWy\\\\\\\/y88ZrQNAdu+qsnoAYqOq1ax3hfdxt4ZVIQiTyD6KaIpz\\\\\\\/0Z7LW5Yt1SMKEzSXwD5xbmkJ4QmnEhlBYJZwAMDLa1SJBJrhgMMyHcRXHIS8MfGxcTUNImOekj\\\\\\\/tzJ8\\\\\\\/wlDHBavXsvW1R81V7WVGKDGOFlW4j+4uumOf+UMm+5oe0t7cDY954dNUGwdppHYwAbQUrXqOkmuC\\\\\\\/wmcj7ysK97PhINSk9vZ240bv0BizBpuTkRmhAlV6\\\\\\\/tRA8J+7PYyUxOR3VTSz0q2t+SQKkZc265b83nTjsEjNEyyJw+d8CvzYcZ2YNaGSzgbUwjM4QGB0LKryfzG\\\\\\\/2Gw9T6T3u3aU88NCT90dgF\\\\\\\/DZrdjbmimPcCJUHg\\\\\\\/HUSHZ1N3OBbReGgO5NgfS\\\\\\\/zIQVjNsQkz9e56AgZN+0yYseWZmsn1WBvW\\\\\\\/0u3+wW9TD0QlHtBUWLzWjWFpBFulYmQ1NzsL5m28wCP6+FevKMsYsCVJueibXYDaWkiDFiVUuW2mYsr4lkIVIe5Z3SWdfiQQfRBW8ZAeXa4W0xFBfizFb4GKvWj+UhmZWAPSolGU7xb7x9d1CiLS7weF4ofzHKCDt6PKiYxRTgda4h7pVRmt5TiOxxr7tWhqn1RiiiefrT6ls75ioX+TpyASQ9Q3FoXztzAz8cfQobTUrq8WlcwTgYcvTPvdplw+LIL0BjPTyhU4Gev+XRm6ficNb4\\\\\\\/iG0\\\\\\\/KtP7OYjTmD8NgipeFPyWI4adkA96vpVVaFPSCmBkqTQ5CkJOyv7Ntt1z7rQe5JxtPN9UoDUKiZ1A4Lxmbkadi1E4llSBnhtf3FHKgoDuEPDP2Ky+XINFl5VgmKe1k5VJ7uceMN7I9THEnSz1TjgpZ6pron0\\\\\\\/LPSAzlEtLdwadYZH2MUUsVMHEegvxT+N0YJxQ4d5y9rujf7fsm9GMPFOGETjjEZDAO4aobO4p+\\\\\\\/Ytq+mQfdXd+iLBvxq5D+LZSlVSDdvgNBb1NmXmf2QevRtfm9Bz5+8cxp6VoILh6XaCH2fk7Kdl8s7wBcs3I\\\\\\\/PlF\\\\\\\/AbRsMBSOAOuVNJ6ftrN1oxiC8Tg48wEmOV3qOH5VRg1Xtm37B7ZHin7zJh3kbgiqTzaDKG6ERdTlo85WxKlN1QFIb7c\\\\\\\/tB3\\\\\\\/skJg5Y166IHMh8GoRrlaIzRJjtHEjiNOD0un31PKtTA2fNLHvF+MvbrOU389i1xIRfw34Y9Tjz5hxOSiiwYhaDOYBWntvuZEFDwa+1OJtc\\\\\\\/SkwSXQZ3SB2anCtAWT4sTqcUkDg1W9x2VRHaQ9WhAYbpv6kxRjRfW3RZK8HkxHFfKoZzk\\\\\\\/O95R6EYubI8wjCI0Aoj7rOEIji+3vOS1VGj5N\\\\\\\/BbWVps1+jaEqQgrJ35r\\\\\\\/6XjbSHYy5BOXkZii2NuU3+wXFeF\\\\\\\/yKmY314ZWxMGwNyhvPFxTQaRU2Gvz+xjwIDcZKhzt2OvvPYodvXY9mxWqYZzCiUSU+YLLOfrOSKug\\\\\\\/G42b0c1FgMl793veWQ0xNfEPjQx2QEpGNx1ZFYM+dDhFa5ycr1ah8jx7SO822jcO73nIsIGnXs1BAN7P6WNAeITiBn0HgZpvE0dqk\\\\\\\/Kp3n9sIoV8NVDRC6ffdV6WCrbmuJMWSFCE95eW3g1Hj4o3OcxXNG6zg9dOee+sODrzpOclgrCXUdyFDSb+TCN9gqNQCW2IolKxpLL3OwJWNYxEkaqTZCoPpPh+356PAELNQlhy9FutwmKT\\\\\\\/hhoz5Rh3\\\\\\\/TQAh+Z4jdi64ZUg+gldD5XKmNYEjxbNYgMrXJ7tPrDhuC2JrPxioPQdHoNP6O9X8PfvYZYzXjC33aXdnWx9swJ06Kr++rvGl+pWTNXGtAmKU8qf9\\\\\\\/xy5Sl9m+cFPZjFmhMb4WtNXOu7dFd1koZTCOUNk0Z4SHt6CEBP6b3YwbrjNIPQeIVdDCWsDkZ2HV02XuAmsV7FMJuMgS6ztZQmLcz9Hbfs0s9lJyTc4SuzCrRwa8X2yDcRgtWDnZcfVNFV37uKBp9W+eIxPVY5aX33wj32L7Xaa28LTMH2chFH+eLkR3nIuw4zeipdu4Km1JAEEKzO4U7NMN+4qy5RjjxZbR6Kct2MMkOVTORoUMgIDc8HUPmvsfT7TSAeiJ06nmmK6\\\\\\\/rFDLMJXrInLDE58x7uy6n9Y7YZLFS+dvkJIKj9V0hIonM0MXisgrZx2rN3BTXS8vGiyiNqgx3n2E9t+Gdc2lEwnmqAnZXwojTld+R7Ib1Xt5ED72gf+oOvQII0vrSmjcwXY29v2jktxiQQjckXpgXSBZJKU5\\\\\\\/NQmyQxJwRBeXnB67L7VcwBIitN06PqVpKZqoOnbyzQm4i0ENSaFBfIajKymUhFy+\\\\\\\/\\\\\\\/cEEFkf41nqWOTDErsgXc7Pf699ICGs08YDgd\\\\\\\/49+foQeYV+KHBfa3wjBMZOHwuSrTVc3YMO4er3nNLzoOsmT6+9AHoK3UJnTgZQEm+HkPrJsj\\\\\\\/sbBkJs1odI4v6JfzDFzkgBCsE+JVFskjoZA42UMjRwjLaA+P+kGHfM9uQ73dWXXpF3Z6nMr4JkteL78TfnC1jpq6nzP6jnCgJbL6BIcmOplHX0tcs6vghqnloPo7ZIi2GQxuhxXR5rNt71gr4lRIgvE3UX3tD2rO9bRANQRkvyJi4plf93EjXe22uZNyuhTcuJB5x2u5cfy4dRMaMVkIVtUx9y\\\\\\\/+VnAmzuyh1GirbYnhBUUV5q2\\\\\\\/yFETF3GujTWJsybj0RneH6CrRHnG1M57\\\\\\\/67TgSj+fbCexD34ugQCrPtS4XLzfE3BNumCtvPaam8Ex9WHQvEEH4EOMZCEQrYKDgTo0kIEVC9aQQZHd1j+Lz7245fPDUaniAjbWxnhzK20asd86L01N9GaJwQu+Xb4Jx2IDBKGstgoO71Ebl\\\\\\\/Ha44x1\\\\\\\/IJexgD1EGTdPJ6nTGFJW+6WmoDVFKb5JvuujIyM4mAp0EXbtuAZj9YOse+TRSD0uHx41RXZfQCK3Cfuwk2gy2cFdc19Zxt3\\\\\\\/f3H\\\\\\\/qAFdlcDXgUx4Nw5ASiibs59tpdajoaD0y\\\\\\\/y0gmfM7255431A5yAxX6LH5spI6+V\\\\\\\/Iu1tjVJOFMcpGs\\\\\\\/yS8bR3grEpDQehN2hf7y24FkyaahVnkRVKmNNfTZQNEFmu8tR0RDtB1TD7+eVgjz8EYQ0+jNkKh0khK48Np\\\\\\\/pV7vPZv4XuYbn\\\\\\\/lVEAAduXO5BDx5z322tlPLtKlrT8OJdcT79QT5xU1xtNaf8xhnXF5jo7FQ4qi1i0c+nRQfYyfUeSyuuAbXHXXUK5I5mZ8cZEvcwvJZzN6sb3aSHlvIb8JZXK4zc8iLhDbWKfL7Q8TzLA78eUU8orLi3ULkzMAa8CVPy1olK2LiZzLsvmYFgBmPf2G\\\\\\\/BBprTJm537u\\\\\\\/M52y5nGMSHDaj8X7Ycd8E9NQk7\\\\\\\/0\\\\\\\/\\\\\\\/r0qBT6nWBlSCcLvRhSQHfml9leRiWNVhuKrwzRsrfL72x04U9mJJdny9ErDEVmbDynfKm7dqEfu0GgAh9KqPPPvZ8g4CVXeU8Sij2UsKlvOCazRTl9nDg0NsmQc6o4m7GSeUhmkLWFlccWDt0Hq9Am6cg68YtgWNf\\\\\\\/9bELzWpKDv4aXA3eZdPd7ILrbwPBtblfi\\\\\\\/GQZfH+bivpRscOAkoazAv\\\\\\\/bwd8XmFB5cOYj8hx0ZD+D7ZvgMibfe9XeWkEJ8QkJDMMAdStzliGDAfZypsGZpv+05BskwgM44ng2iF2lcBaUERDUEuWNmK\\\\\\\/2cvLLrHUBB5PjV6csRaeakGQmc4lxB\\\\\\\/mYQLup0Ezw5vjHg6c5pRyDJbFCEXMZcJwpG3gTW2\\\\\\\/2kJH4fh44I6ChY3L2NUyd5K6FPjSupG4Qhe2YfdzAidbGqqg9IlAnHNPxTYnT05xsfkF\\\\\\\/lZeN3\\\\\\\/qNruMPDr41aqEKHtGwzRiwVjOab5W9TqMRiQKgNWLWPBcS93nH5uxulQdWFFDgjsZQjVlPYpfkWWYdpeCPHrRLZv1foOwAh4rQ9ufY4LTKnkwv31tiSvFV\\\\\\\/m82z0itpohvGcVr3lt8Wo0SzrEd4Zhejpn5fzxhPPwRDzUBC3kYMsgVV3G5cNa7oQVmpJYRY+0iKnSLvEA6MIR+tysP5sTJudZJayGUV407pp4cwHFZK\\\\\\\/ZU3ZZobKj3h1kZfdOUSmMGfAxbrhhChyRnpguvvPFN\\\\\\\/KAGESoEq4pOOEGM\\\\\\\/n\\\\\\\/HCbXDOIxlZ\\\\\\\/lIRnaeg3obTpDX7\\\\\\\/UxRTBCR7aoec7+T+97z3jyb42P3fereRkx+kY0\\\\\\\/gaRn3YghxrmjF85rq7E\\\\\\\/ECx3A39d4wDJSYWL3pSBGWh56AkJCa22P4cbGtTin+9tdcJp6WTF+xs\\\\\\\/HQDbIz5U82r0QnBNaug2rwiJwQUZ5pslW65Ui\\\\\\\/BIgimUT4+LTqTZbvhhPO\\\\\\\/QIoKZrqAn8IU7xb0+Uv+iXp9dCsCNv5mB1qkN5r29vvnd\\\\\\\/3jkPoZlfcWasfUEMKL404yvTbFbxTNvqLKYhTEj5VoTLzWXtU7AvX6ZQaWpBkTbFx+KlsCn0MwZHDCL3h\\\\\\\/VKwj290eV20YweWkuN6I+dN3uJjCF5Wybe81sEr6plFodQeyj9JGEPiNxNlYhKf3BUC4IHXY9TuyU66+YeqVsGFzmEWCDdSei+DjrD5mr2MQpmSpULbYq4fgV8SzA81+V7s2zL6RIjA+5KF66KIXbMkGm8pN5QLkp2HTG4iizHBn8NUS5ynlpZ\\\\\\\/fvzCpuKXnrYw+rnxhPy3Z1UywtkrubjXgd562O7DSPOEizhBkPCItUBlynnkugZLwdTqxnx\\\\\\\/4McfyQnrgZlmjnemkqF\\\\\\\/C4AveV73jhCo\\\\\\\/Mv19Bz6AoxTMZk2VWVWvWulCMdbRimdpMHwNP\\\\\\\/u0pE6SqFVKjamFQJeHyvoQhCIWKXcdM9peN12XOAUxdvm\\\\\\\/29sroIj+xnkKe7pBixPKBFFjIT4mnaA691Vu6A+FkxxaII3ShhNpIAwFURJQA3LhXjbhl6Tq1\\\\\\\/OKbJ\\\\\\\/tA+y8\\\\\\\/nJGuqRwRaO79iBMCbml6Sbj1bW35lyjI1JTnrr7enp1Zi7zrhCpMk8eFT6P6FOmDCm+\\\\\\\/T5WIzfrocntgf\\\\\\\/WBQaRiLphh7ZOOdMlVCGU9iOmEx+LwL1exLdoePCIu2LsXtlGrgkpHLQ8xaqtm9VEyEhzvZnHxjZJataGrCvkBRSnCU\\\\\\\/f4gtWTOT9+TqNvSBksjZte\\\\\\\/oO7MWsUMMUCUhlyRgamguYON3Gel\\\\\\\/IC3l73DbzYg2\\\\\\\/VyjUBFL62C6xeALRNQJGg71EdeJ1pJQV+QIp6PXwYdhqw7IruZke0\\\\\\\/utME3hvw4iG0ZZ5dP9sstJ7HG4atyKx+FBRWmVjwuI9SVCKNQmAPVnDunGJwpHXnBfPo0J+104T2L9H45aJuH1rSO9Vm1BGbVviUKzw\\\\\\\/aRmjGMExWp1iHFO9KqPwmkUrug95RaxKE7W8ipifrdgj5w+GvAfKkAh+JdH\\\\\\\/laMxJ\\\\\\\/KVAxBAkcFvg5MTi75BNn69LNcuvUiTKotV2aFuEUPld4Ozzqh+oejt23KnT4vuD7zT2ttCSsyiEB1sGgcHXRRv3uUTY53NhQjk9VYYQDxVR+QpTRrpRJfxmu3QkHCz6Cn4\\\\\\\/XuzCYBKrX51wNfdwvCD4ios38CaJp\\\\\\\/C48EaqnXnow1pw9CDhLfjaISO89JGsqBjOCuqZY\\\\\\\/8nYWwXt26m53\\\\\\\/IzRTYWOvhVAK9CIkd98ZE0W5ayA\\\\\\\/uNNQMcl+yRaBJOkK2q5FEfgcDIaGp7AY2rf+tu340KRxB\\\\\\\/JNRj7Ex9wad6jMFVKVQHR8VEtGVtgdxAFtL\\\\\\\/Ghg+PKFMx\\\\\\\/WlxwER\\\\\\\/8TpUnSJK9f\\\\\\\/XAqy7HHu2Fyw0hnYKwjN71hTaX70JdVF9wwV6fw\\\\\\\/F+rCb8zFEggMy\\\\\\\/aHJNvDWwfyi1zGGf+Xn1BkQza2wfm0pZVvItloaaY2NLgRW\\\\\\\/tUxf3OW\\\\\\\/ZQwBSuL2pAG\\\\\\\/FyGzM2OCwvq\\\\\\\/NBkMaEm9mHGoFGw35Z1NeShWoEIdeKGmQk8vHAFkWhtpqR1BwUfXP1LI+UzPqb3mbKIs758YsQVyZNFhOLTrzNj8vQEeyns0sMX9E2IkG13\\\\\\\/zDnI8fOqc72jl\\\\\\\/irrUg9RyhyHVcSD0CyfkG8N2gI2\\\\\\\/U4UrhP5vSDOlAO55TwcQPTSF9dJopmMgNx9Y\\\\\\\/VBmwF+\\\\\\\/Nea\\\\\\\/mc5bCBYNkMcZ40uWRJqLWMziZo+GYxxH1PJxWK+5\\\\\\\/79\\\\\\\/6F26Ywo4coPaIZKnHhn2zO04ByuHs8U9eo42TKCQpBsMmJWRNEsqRusv+qov2arKZKzvqA7uWjJegYtxR\\\\\\\/JP+OyLAtnRgp815xZuBhKApIp1CgzFqbLXPkof4wmJpniP2RcyLOqX1XOeKxuI94yVG0pa40En0cE8FABw3rsHuM0XVIyA4bTr93K2DWDO+ITST2AGMYb93lNNIXHYjG8hyzbndbHA5BuYH2K30GkZR9tWE35dzAhEnRdh4BBcZ16rSsJoEKwwC8Qf50lfV3i2wP+wz1ajWOl6Ib\\\\\\\/T1Q2yEhN8wadCTdD5L6+QT4ADcq\\\\\\\/Mu1fiAQEpdkJezU9gcaUOkHu4gOIQFdCtbRtGnc1ZeHKR92val7Qcl0Bf6rX5ZFIW2Wn2szpd72OKoL7EC26tXdgc\\\\\\\/hVwEkVnkzaQOnp4hJyK2rFVKVk96CE5CR0EgdBmnSIfPIZbSCb4X4TkdxIhAfcQg362bln1D64nbxALUnobSmoozhVTBLCVslnTz8TebAlq80ohsvwpBuPUCpCYF4reY3Gi2zjw\\\\\\\/YQvyOpm6wbNfsrDnW9zyriK7jhoCfHw1A8s1QtMWjCYzvTWWtEtXoYaA3ifKTKZFfiTUs9YuFpqNJUzMQhwcQmQvIORWhZLdfXpHDTSbgtihXtWW1efSevNhmXNMwjP\\\\\\\/hpWDBUhpP5btLl8ZuowdOueYeC3gEyVVEyrUzFCtUZSUm2YjwxW\\\\\\\/Y9CUNoZSwqIWWmSoJCFhOSLbVa\\\\\\\/pLsr5TVnE+7o\\\\\\\/g+jmBQW6D0iB92ZYNl2spRpPpevDWb\\\\\\\/mYHpNVDhRqwVN4V+P\\\\\\\/NbaW3wSdgbpzvVTCIzrC3VE1De5Wl\\\\\\\/JOHaZ0UecVxpKUEYs\\\\\\\/FCIKFHEeyIoPeM0GbePkAoBEmKy8LLJx5SneXy98qihh+7RmFvRaipXw8witodTGi7wplUawZDeC8g9w4ifNBf5Eoluixquk2gwSoDubB4RV1rwKDFeur6eXl8DhlBE8L8MOtVzKw60HOEOdjgpcNKZOnuiQVvfCm7spxz2\\\\\\\/+dZ5aQvnuiLgW1DkuxKbiKDXD5OAnUeeTajkNAEzg8046JAoYdCCUcjs5+8zBU6u8qJ9ReMqPBz1Ec6haBc6Ozh93HqJQdSagdH0wBHw\\\\\\\/YNO6NnkXRa\\\\\\\/B7np\\\\\\\/w+IoXSHm1Pxx3ml9acQrhMEzLcVd2I71H\\\\\\\/TZZCxxxF9QBBjtPzVHwEf8K0UNpG56\\\\\\\/jQEyHnNbaYpgTH4VsxFhnCK8G35U4Yweb3RcSuv2FLEe+ULTzeShPGT5mHnYAXagC0lszeWHS+TdNYX0mZisZd+5rEW6m4IGZiwFpLKWCQkCtxsbRIm4FLGkWb1xoiwukH4g2A81XwrcvoIgspHd6pt2zSePCVQuqqYLqpvwnq+hRQGUrGChsszvQQjZkgUlWePJ2bFnVHUWr6rIwtu+WmbWAhhh1Nab75ueaqlvkLqpGSFwfvHNtGx1m8QxJCcve\\\\\\\/ZUZxiZhMIURpVnsSTydape3uLFvgJ0AvY\\\\\\\/RlX85UtqR47itSFcz94RcjSKPF\\\\\\\/WiR5xTH7WpL58MjGS9vPd980AuTFdvzqmbFKN7KaV68LUsKUumgEq1yfpOtFhN++4T6\\\\\\\/9FzsM4BQJOuEHTiWTTMYHcjodN\\\\\\\/kuHLiUUbE67nPBjFl1Mn6uqgHitLpM3Kw7TldNmxhI7FZE6uaI9CDOAqV9\\\\\\\/HwMOxkJVeE2fe\\\\\\\/rsouzLb9w9XBuplhziFw5FgiLl3RT64WdE9qk5ZbZjpQUVmqXj1nOe6k2NO322pxtBmUNhGvZuVRgn9j4sieJCrwoL6GNcm1yk7kL\\\\\\\/9hXrD0Ng4R5cga9FJZOHvIqDGCheJbJSQWdpM77jDFr14QWewlTt1Pkz8pXrO4OyHIR3RR2QLjIP11R87iWWtw51HZx7zD\\\\\\\/Pra9c+EoL2fcr99bEL7SmpDOr\\\\\\\/Br9Nfvt3rGlmOZkKvgOMO3aB78EHvEBVwdhz4tBUZ7Yhu06h08EPE1KnPT88suvVxo1xAtt8aKGew2jSRm\\\\\\\/0HU35ZLtXC7jGVsbk5EWJB6TTx1HP3kVqWUg08kOdran1maY0V3kj9WxTzp2g4wkX6LVJvCN1zgfWCKonWe4HsDYarhejV4f04+eQT8t8lO2PvzY1h48Ly+PraRmywA0IU9gPma0\\\\\\\/s6Qe+eYzjH4P9IMzrXUDumxulPJVDnewroG9NHGMtBdOEq1GClAY1\\\\\\\/WNYoJ8tcqiReHiFDCL\\\\\\\/tzyGb0lofntzEsNrdTH5N9sfOmUgtBhvU\\\\\\\/YGMnmroEfJVgOKTxSg5xE6ZaoVGZw==\\\u0022,\\\u0022iv\\\u0022:\\\u00223ouIdq1P7agKEy8gluzMdw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f715acb56a014e02f6b99895bfa9633900a02e0a8d2582cb3ce01d8656ce4a9c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222TsVi639M9HdkJezrYH\\\\\\\/rz7+iV3xrFpLixfcFs1SsFrH8V8322x3XzrYBiAcp8WB+YeMEsyPhBfqEAtMeOK5gOYXDfIaPZDL7Sc4gtnl4AKF8JL7rOhGnicehdX6i3CGLLHWKjq+djULJbALLwc7g+YdUHMRXBcSEqvwpMjnggl4MfM7Tc4nUccoYCSKg1VtoflB\\\\\\\/pq5sSpgJ3hJ5wnsUcKNiuAs74HWgGN0IpvJ\\\\\\\/Y6ZUbpDDsQSUl47ufLl9YqaGqAZoDKE6zD4pMyhI8TpDmfiIqTTJDqt7nyVlDRxNtWuI8IcAcQCuxthQDdxsA\\\\\\\/qUoiQPnsPgPOJ\\\\\\\/gxSiQ0O4Tzkf87CnsAGx0AJLgmxUhQm6FZ9iRbHPTJ5VIgCZAC1906gNZD0YX8D+u2s7pHoNq3cCMAGyRqAcFQP4Kxhd3mb\\\\\\\/5O7iYg\\\\\\\/uSgVbTWy78dLaccvLAx+KtGdZiSet6\\\\\\\/Fnret0zzjNNDpA0WDxi2J+U8v74Y\\\\\\\/t8qC+lnHSwZSnEPSxBDi\\\\\\\/lq8e4WbkB2Udxn0xjKO+migH7aPLGW0vtmnny69U+n4AcZyRInZO88FcBGLFkVfP6fNq7bK\\\\\\\/mFo4OX3yuQcf0FIZSrMa1jcHvAMn023OGTXUgb9w30SQjVyeKooleSYC2YpSPmrG6\\\\\\\/6nG\\\\\\\/dNq9E6mTFSm45uGSdnvIyTqN54IWQeJCPfFTH\\\\\\\/NpW2GUcFuRY7gVo8VtzveRicH+FV3VCcr42Ve3UiAxkL4Mn2ithV4jI1KFgx5FO6PAGFDvAGfxvVA7O+HYoFpM40EF0BojgC43ZpOK7SpLh7Agq31m3PpR7U8j5wPC7lNSLi+yRPq8EPL8NG1VHK5CX6VjlhjCxGXpSo4Facqb9\\\\\\\/9vgxuDMCvIJgCLQM6TFAoBZlFYMyQXAX5jLk+vjTMg6KigqQQtjhgKKKkA\\\\\\\/t+EEFMDONWqFnTNOw6ZkgCNqqR9CZiB6i3N4iDGoYwm1\\\\\\\/E5sl\\\\\\\/Br0mMUyNawG3\\\\\\\/WGIBW8QBKkunhWuUZhva0m+oHrfyAC+O0T1vYZsssCMZfZLH+Bydpt8zf\\\\\\\/shfrANtZZyv9ZBkA479yYrUj5dWz69\\\\\\\/pa6O6CxXsd9UFpKIfQNiGt7iqQgmrdIE2nxM6jt\\\\\\\/Se\\\\\\\/RKQKQQ7mSJUbVfVum4vdI8XagaM5LdE7xma\\\\\\\/PV+\\\\\\\/llSjr7Q1rop3TWKOf2fkiu\\\\\\\/C+WlywXNluAUxHiQYufkWVLU2ueETnffPO9cSNLCkECykCaUoKiWQ3aKLqiD67F55+ER8n5igCWDpOMl83lMNpRzIvZShTUTmi+9\\\\\\\/wlzshnb5TwcxuH2YNV5iX8HK71iEaUj0mPQklK4rZtsOHY7qUy4+kACftsRzmXHruJPse9WNbcBRnn850ULWfYxIIBJ+ofPmf7a45xKftl85nSpvPsbIp4LUW9J0ZOAoIGmItPwLpeXB5t+8kMSD58IBOZJeYqKTp2fi8NmxlV+C5sKQFL6nFBkLDHcUyJZ0s40N8R3Fs5osd+8Vm9NKlQcaKtsKpKJhTqNvL\\\\\\\/KkL9syYEXgoxXHLz15JIlM0mWnlT98tgJ4dcqrmBTtBnsFXS3+ae5XTPG7qPy652jW9J+5RXecGgnzGrkwPIhLqRVJcfNZ2HZjzpc3G4oFW2FmFQy\\\\\\\/zizc5LuJ\\\\\\\/eCkqT5qISnabt55+26aQBLJeUN+oH+1WJuxqWFHjZ1edfU1\\\\\\\/sj08nmGA9UDbCffAhavLUDDAG0aDe7RUPYenwzo8BqoZRhYacGWtAaAJhIsMf5Pvj6BGZjGU\\\\\\\/DCNI2qNoFs0cxKI+PKn\\\\\\\/EQvw\\\\\\\/owssnC3+pvo1sIJE+PDvmZtAhqXvQy54vrZpzSXy+ptuIMArzv74wLyL\\\\\\\/TmM9\\\\\\\/ORE99V0k+DvxVpzqiYN7tC4tedMh4hOkjSkgt9Si6T0wPpjmmq7pUGh6AMc4nDEZ4jQT7oJvVKR+wJ0ZtuOwgL8yo7BON5liBR5aSCR0P9lFWD1KLdyFpZ6tDhhhYxdImqIXVbFkDlbyU\\\\\\\/ZXUz83C\\\\\\\/PjVMhd\\\\\\\/g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XNN+7+0yfzvqBSqO4xDiyQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002242b5e4a462540647267b87ac6d480852e3c035d5204f16ba75470c478000fcf1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495280 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/379" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-80Gq47X0M2qM_nSIZBJLIYiVPb-wjtelbn0GfEc1z6o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/379" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o_pTYCR2THfnE_3CNsFeBm9hqiT-3BBXiJ_l9qSJkFE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/371" title="A Genetic Screen To Identify Genes Influencing the Secondary Redox Couple NADPH/NADP+ in the Yeast Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/387" title="Akirin Is Required for Muscle Function and Acts Through the TGF-Î² Sma/Mab Signaling Pathway in Caenorhabditis elegans Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495280" id="node495280" data-pisa="ggg;10/1/379" data-pisa-master="ggg;g3.119.400827" data-apath="/ggg/10/1/379.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5418-0751" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhuliang Yang</span>, <span class="highwire-citation-author" data-delta="1">Jixian Deng</span>, <span class="highwire-citation-author" data-delta="2">Dongfeng Li</span>, <span class="highwire-citation-author" data-delta="3">Tiantian Sun</span>, <span class="highwire-citation-author" data-delta="4">Li Xia</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3829-7922" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wenwen Xu</span>, <span class="highwire-citation-author" data-delta="6">Linghu Zeng</span>, <span class="highwire-citation-author" data-delta="7">Hesheng Jiang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6252-7455" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiurong Yang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">379-386;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400827 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495280" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhuliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhuliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhuliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5418-0751" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhuliang Yang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jixian Deng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Guangxi Institute of Animal Science, Nanning, 530001, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jixian%2BDeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJixian%2BDeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dongfeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>College of Animal Science and Technology, Nanjing Agricultural University, Nanjing, 210095, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongfeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongfeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tiantian Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiantian%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiantian%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Li Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALi%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wenwen Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenwen%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenwen%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3829-7922" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wenwen Xu</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Linghu Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linghu%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinghu%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hesheng Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hesheng%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHesheng%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Xiurong Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiurong%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiurong%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6252-7455" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiurong Yang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yangxiurong09@163.com" class="" data-icon-position="" data-hide-link-title="0">yangxiurong09@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/379" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495280" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/379.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495280" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/379.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495280" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/379.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495280" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Guangxi indigenous chicken breeds play a very important role in promoting the high-quality development of the broiler industry in China. However, studies on genomic information of Guangxi indigenous chicken to date remain poorly explored. To decipher the population genetic structure and differentially selected regions (DSRs) in Guangxi indigenous chickens, we dug into numerous SNPs from seven Guangxi native chickens (GX) by employing the restriction site associated with DNA sequencing (RAD-seq) technology. Another three breeds, Cobb, White Leghorn, and Chahua (CH) chicken, were used as a control. After quality control, a total of 185,117 autosomal SNPs were kept for further analysis. The results showed a significant difference in population structure, and the control breeds were distinctly separate from the Guangxi native breeds, which was also strongly supported by the phylogenetic tree. Distribution of F<sub>ST</sub> indicated that there were three SNPs with big genetic differentiation (F<sub>ST</sub> value all reach to 0. 9427) in GX <em>vs.</em> CH group, which were located on chr1-96,859,720,chr4-86,139,601, and chr12-8,128,322, respectively. Besides, we identified 717 DSRs associated with 882 genes in GX <em>vs.</em> Cobb group, 769 DSRs with 476 genes in GX <em>vs.</em> Leghorn group, and 556 DSRs with 779 genes in GX <em>vs.</em> CH group. GO enrichment showed that there were two significant terms, namely GPI-linked ephrin receptor activity and BMP receptor binding, which were enriched in GX <em>vs.</em> Leghorn group. In conclusion, this study suggests that Guangxi native chickens have a great differentiation with Cobb and Leghorn. Our findings would be beneficial to fully evaluate the genomic information on Guangxi native chicken and facilitate the application of these resources in chicken breeding.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dsrs" class="hw-term hw-article-keyword hw-article-keyword-dsrs" rel="nofollow">DSRs</a></li><li class="kwd"><a href="/keyword/guangxi-native-breeds" class="hw-term hw-article-keyword hw-article-keyword-guangxi-native-breeds" rel="nofollow">Guangxi Native Breeds</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">Population Structure</a></li><li class="kwd"><a href="/keyword/rad-seq" class="hw-term hw-article-keyword hw-article-keyword-rad-seq" rel="nofollow">RAD-seq</a></li></ul><p id="p-4">Chicken is a classical model organism in livestock, most studies in the agricultural field are focused on diseases, nutrition, and breeding. Since commercial chickens depend heavily on their ancestors, genetic breeding plays a highly important role in chicken production. Chinese indigenous breeds have great advantages in chicken breeding, due to their extensive genetic diversity. Nevertheless, the breeding progress needs to be greatly improved in many areas of China, as many excellent breeds have not been well utilized yet, especially in Guangxi Zhuang Autonomous Region, a part of southern China. Guangxi Zhuang Autonomous Region has numerous indigenous chicken breeds, which are crucial for the development of China quality poultry industry. However, only sporadic studies were reported on genomic information about Guangxi native breeds.</p><p id="p-5">As known, single nucleotide polymorphisms (SNPs) have been widely used in studies of genetic variation and molecular markers, and they are easily assayed by several different methods, such as mass spectrometry(<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Griffin and Smith 2000</a>), single-stand conformation polymorphism (SSCP)(<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tahira <em>et al.</em> 2009</a>), restriction fragment length polymorphism(<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dai and Long 2015</a>), DNA sequencing and so on. Notably, most of these ways are usually time-consuming or expensive, especially for those experiments with a large sample size. In order to cost-effectively get sufficient SNPs, the routine approach is using a restriction site associated with DNA sequencing (RAD-seq) to analyze the genetic differences in Guangxi native breeds. RAD-seq is one of the reduced representation genomes sequencing strategy based on Next-generation sequencing (NGS). This approach can obtain thousands of SNPs inexpensively compared to the whole genome sequencing, although the number of SNPs is less than the latter. Currently, it has become an important method to generate genome-wide molecular data for population genetic studies (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hou <em>et al.</em> 2015</a>). RAD markers were applied using microarrays initially and subsequently adapted to NGS (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shendure and Ji 2008</a>). Now, RAD-seq has been widely used in the research of genetic variation. Miller <em>et al.</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2007</a>) used the RAD-seq approach to efficiently map the stickleback major lateral plate locus to linkage group (LG) IV. They indicated that the RAD techniques can be utilized for genetic analysis in the model and non-model organisms. Since then, many empirical studies suggested that RAD data could be applicable for the estimation of genetic relationships (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Eaton and Ree 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gonen <em>et al.</em> 2015</a>) and evolutionary history (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Catchen <em>et al.</em> 2013</a>).</p><p id="p-6">As a typical broiler, Cobb has the advantage of low feed conversion ratio (FCR), high growth rate, and less costly nutrition. White Leghorn is a great layer of white eggs, it has been much used to highly productive egg-laying hybrids. In addition, Chahua chicken is an indigenous breed in Yunnan Province, which has a close genetic relationship with <em>Gallus gallus</em>. Therefore, in this study, we performed the RAD-seq to analyze the population structure and differentially selected regions among seven Guangxi native breeds and three other chicken breeds, which are Cobb, White Leghorn, Chahua chicken, respectively, to investigate the genetic information of Guangxi native breeds.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Samples collection and DNA extraction</h3><p id="p-7">In the current study, ten chicken breeds were selected. Among these breeds, seven of them were selected as the major objective of this study. All these seven breeds were from Guangxi Zhuang Autonomous Region, which are Three-Yellow chicken (SH), Nandan-Yao chicken (NDY), Lingshan-Xiang chicken (LSX), Xiayan chicken (XY), Longsheng-Feng chicken (LSF), Donglan-Wu chicken (DLW) and Lingyun-Wu chicken (LYW), respectively. The information is presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Besides, one breed named Chahua chicken (CH, Yunnan Province, China) as well as two commercial chicken breeds (Cobb and White Leghorn) were chosen as reference populations. Each breed with many individuals was sampled, and blood was collected from wing sinus for DNA extraction. Genomic DNA was extracted using the phenol/chloroform method. Chickens were handled in accordance with the principles and procedures outlined by the Guangxi Universityâs Animal Care and Use Committee.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495207/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495207/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sample collection data</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Library for RAD-seq and sequence analysis</h3><p id="p-8">We randomly picked out 10 samples from each breed for library construction. After all DNA samples were digested with the restriction enzyme <em>Eco</em>RI, the fragments with P1 and P2 adapters were enriched by PCR amplification. Subsequently, the sequencing procedure was operated in Tianjin Novogene Bioinformatics Technology Co., Ltd.</p><p id="p-9">After sequencing by Illumina HiSeqXten PE150, the raw data were subjected to quality control (QC) procedure to remove reads under cut-offs. Subsequently, eligible reads were aligned to the reference genome (Gallus_gallus-5.0/galGal5) using BWA software(<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li and Durbin 2009</a>) with default parameters, and duplicate removal was performed using SAMtools software(<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2009</a>). The information about SNP detection was stored in the VCF files. To include SNPs in further analysis, we only kept autosomal SNPs genotyped in at least 70% of the samples of each breed.</p></div><div id="sec-4" class="subsection"><h3>Population structure</h3><p id="p-10">The analysis of the population structure was performed on 10 breeds. The SNPs analyzed in STRUCTURE (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pritchard <em>et al.</em> 2000</a>) were randomly sampled from each breed, all these SNPs were filtered by minor allele frequency (MAF) â¥ 0.2. A total of 1,000 loci were selected with 10 times for STRUCTURE analysis. The K value we set in this study was ranged from 1â¼11, and for all analyses, 10,000 burn-in steps and 10,000 replicates were used. The optimal K was determined on webpage STRUCTURE Harvester (<a href="http://taylor0.biology.ucla.edu/struct_harvest/">http://taylor0.biology.ucla.edu/struct_harvest/</a>) by Evanno method (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Evanno <em>et al.</em> 2005</a>), and we processed these data for final results by using CLUMPP (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jakobsson and Rosenberg 2007</a>) and DISTRUCT software (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rosenberg 2004</a>). For phylogenetic analysis, we sampled at least 15,000 SNPs from each breed to avoid the effect caused by unique loci. Finally, the evolutionary history was inferred using the Neighbor-Joining method (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Saitou and Nei 1987</a>) conducted in MEAG7 software(<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kumar <em>et al.</em> 2016</a>) with 10,000 replicates.</p></div><div id="sec-5" class="subsection"><h3>Calculated for F<sub>ST</sub></h3><p id="p-11">For F<sub>ST</sub> calculation, the seven Guangxi native breeds were set as one indigenous population (GX) to compare with three other breeds, which were Cobb, Leghorn, and Chahua chicken (CH), respectively. The F<sub>ST</sub> value of each SNP was calculated using the R packages (âhierfstastâ), and the distribution of F<sub>ST</sub> of three compared groups was plotted by R packages (âCMplotâ). Subsequently, we chose the SNPs of which values of F<sub>ST</sub> were distributed within the top 1% and the top 5% as the top significant SNPs. These SNPs were selected as elements in the discovery of DSRs.</p></div><div id="sec-6" class="subsection"><h3>Differentially Selected Regions</h3><p id="p-12">To locate the potentially functional genes that may directly influence the population differentiation, Differentially Selected Regions (DSRs) based on F<sub>ST</sub>-SNPs were used to search for the most likely candidate genes in three compared groups. Two algorithms were used to define a valued DSR: 1) there should be a significant SNP (top 1% in F<sub>ST</sub> values) as a center, the boundary of region would be determined by the adjacent SNPs, until there were no more than 2 sequential SNPs (top 5%); 2) there were more than 5 sequential SNPs (top 5%) without high significant SNPs (top 1%) also can be considered as a DSR (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2017</a>).</p></div><div id="sec-7" class="subsection"><h3>GO enrichment</h3><p id="p-13">The SNPs derived from DSRs were annotated by ANNOVAR (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2010</a>). The gene functional enrichment analysis was performed using Panther bioinformatics resources (<a href="http://www.pantherdb.org">www.pantherdb.org</a>). We selected Fisherâs Exact as test type and Bonferroni correction for multiple comparisons. Finally, significant terms with the enrichment value of more than 1 and the P-value is less than 0.05 were reserved (Supplemental Material, Table S2-S4).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-14">Raw sequence data were deposited in the NCBI Sequence Read Archive under project accession number PRJNA589666. Table S1 contains information about DSRs in three compared groups. Table S2-S4 contain information about the genes derived from DSRs and the GO enrichment in three compared groups. Figure S1-S2 are supplemental materials for STRUCTURE analysis. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10310750">https://doi.org/10.25387/g3.10310750</a>, <a href="https://doi.org/10.25387/g3.10278128">https://doi.org/10.25387/g3.10278128</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Sequencing data quality</h3><p id="p-15">A total of 997.57 million reads were obtained from all individuals with an average depth of 8.41Ã. After quality control, several individuals are filtered because of the low level of sequencing data. As shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, the range of Q30 and GC content were 81.48â95.31% and 39.45â55.01%, respectively. The average number of SNPs per breed were 565,326, 560,692, 517,315, 581,273, 591,750, 634,112, 651,443, 528,045, 624,448 and 625,699, respectively.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495404/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495404/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The summary statistics of each breed</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">With a series of filtered control, a total of 185,117 SNPs (autosomal) were left, all of these SNPs would be used in F<sub>ST</sub> analysis and differentially selected region (DSR) analysis.</p></div><div id="sec-11" class="subsection"><h3>Population structure</h3><p id="p-17">To infer the potential population structure, we performed the STRUCTURE analysis and phylogenetic analysis on all 10 breeds. For STRUCTURE analysis, we selected the <em>STRUCTURE</em> software (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Falush <em>et al.</em> 2003</a>) to calculate genetic structure, which is applicable to most studies of genetic variation. Normally, the loci analyzed in <em>STRUCTURE</em> should not be in tight linkage (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Pritchard <em>et al.</em> 2000</a>), but multiple SNPs derived from a single RAD site are assumed to be in perfect linkage (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Catchen <em>et al.</em> 2013</a>). To avoid this situation, we analyzed 1,000 randomly chosen loci for STRUCTURE analysis. By using the deltaK approach proposed by Evanno <em>et al.</em>(<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Evanno <em>et al.</em> 2005</a>), we found that the optimal K was K = 9 (Figure S1-S2). As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, although Nandan-Yao chicken can not exhibit as an independent cluster while K = 9, Guangxi native chicken breeds distinctly separate from others. Moreover, it is distinct that each breed can represent as a cluster while K = 10. For phylogenetic analysis, we performed the Neighbor-Joining method (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Saitou and Nei 1987</a>) on 10 breeds, the expected relationships in the current experiment are given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/379/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population STRUCTURE: The analysis of population structure in 10 breeds. Population structure in 10 breeds with K = 8, K = 9, K = 10. The clusters are arranged in order, which respectively are Cobb, Leghorn, Chahua chicken (CH), Three-Yellow chicken (SH), Nandan-Yao chicken (NDY), Lingshan-Xiang chicken (LSX), Xiayan chicken (XY), Longsheng-Feng chicken (LSF), Donglan-Wu chicken (DLW) and Lingyun-Wu chicken (LYW)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-731820633" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population STRUCTURE: The analysis of population structure in 10 breeds. Population structure in 10 breeds with K = 8, K = 9, K = 10. The clusters are arranged in order, which respectively are Cobb, Leghorn, Chahua chicken (CH), Three-Yellow chicken (SH), Nandan-Yao chicken (NDY), Lingshan-Xiang chicken (LSX), Xiayan chicken (XY), Longsheng-Feng chicken (LSF), Donglan-Wu chicken (DLW) and Lingyun-Wu chicken (LYW).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/379/F1.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/379/F1.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/379/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/379/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495215" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Population STRUCTURE: The analysis of population structure in 10 breeds. Population structure in 10 breeds with K = 8, K = 9, K = 10. The clusters are arranged in order, which respectively are Cobb, Leghorn, Chahua chicken (CH), Three-Yellow chicken (SH), Nandan-Yao chicken (NDY), Lingshan-Xiang chicken (LSX), Xiayan chicken (XY), Longsheng-Feng chicken (LSF), Donglan-Wu chicken (DLW) and Lingyun-Wu chicken (LYW).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/379/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of 10 breeds: The evolutionary relationships were estimated using the Neighbor-Joining method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-731820633" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic tree of 10 breeds: The evolutionary relationships were estimated using the Neighbor-Joining method.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/379/F2.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/379/F2.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/379/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/379/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495181" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Phylogenetic tree of 10 breeds: The evolutionary relationships were estimated using the Neighbor-Joining method.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Calculation of F<sub>ST</sub></h3><p id="p-20">For better understanding the genetic variation of local chickens in Guangxi Zhuang Autonomous Region, we integrated seven Guangxi chickens as one indigenous population to compare with the control breeds, which were Cobb, Leghorn, and Chahua chicken (CH), respectively. F<sub>ST</sub> is commonly used to define the variance of allele frequencies between populations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Holsinger and Weir 2009</a>), therefore, we calculated all F<sub>ST</sub> values for every SNP locus. The global distribution of F<sub>ST</sub> in three compared groups are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. For all F<sub>ST</sub> values, we observed three SNPs located on chromosome 1, chromosome 4 and chromosome 12, derived from the GX-CH group with the highest F<sub>ST</sub> values (0.9427).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/379/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of FST in three compared groups. Each dot represents an SNP in the dataset. The plots indicate FST values (y-axis) against their corresponding position on each chromosome (x-axis). A) Distribution of FST between Cobb and GX on 1-33 autosomes. B) Distribution of FST between Leghorn and GX on 1-33 autosomes. C) Distribution of FST between CH and GX on autosomes 1-33." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-731820633" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of F&lt;sub&gt;ST&lt;/sub&gt; in three compared groups. Each dot represents an SNP in the dataset. The plots indicate F&lt;sub&gt;ST&lt;/sub&gt; values (y-axis) against their corresponding position on each chromosome (x-axis). A) Distribution of F&lt;sub&gt;ST&lt;/sub&gt; between Cobb and GX on 1-33 autosomes. B) Distribution of F&lt;sub&gt;ST&lt;/sub&gt; between Leghorn and GX on 1-33 autosomes. C) Distribution of F&lt;sub&gt;ST&lt;/sub&gt; between CH and GX on autosomes 1-33.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/379/F3.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/379/F3.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/379/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/379/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495244" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Distribution of F<sub>ST</sub> in three compared groups. Each dot represents an SNP in the dataset. The plots indicate F<sub>ST</sub> values (y-axis) against their corresponding position on each chromosome (x-axis). A) Distribution of F<sub>ST</sub> between Cobb and GX on 1-33 autosomes. B) Distribution of F<sub>ST</sub> between Leghorn and GX on 1-33 autosomes. C) Distribution of F<sub>ST</sub> between CH and GX on autosomes 1-33.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Identification of DSRs and analysis of gene enrichment</h3><p id="p-22">To find the difference among Guangxi native breeds and the control breeds in the genome, we defined regions with shared differentially selection signals across breeds as differentially selected regions (DSRs) (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2017</a>). The results also demonstrated as three groups, due to the algorithm for finding DSRs was based on F<sub>ST</sub> values. A total of 717 DSRs were validated from the GX-Cobb group with 882 genes annotated, simultaneously, 769 DSRs from the GX-Leghorn group with 476 genes and 556 DSRs from the GX-CH group with 779 genes (Table S1). A Venn diagram displayed the unique genes in each group, as shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, we recognized 469, 216 and 408 special genes respectively.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/379/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagram: The number of unique and common genes among the three groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-731820633" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagram: The number of unique and common genes among the three groups.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/379/F4.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/379/F4.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/379/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/379/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495236" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-23" class="first-child">Venn diagram: The number of unique and common genes among the three groups.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">All associated genes with DSRs were used to GO analysis in each group. A large number of terms were simulated (Table S2-S4), therefore we just exhibited the most enriched 10 terms within every ontology in each group (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>, and <a id="xref-fig-7-1" class="xref-fig" href="#F7">7</a>). Notably, as <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> showed, there were two significant terms with fold enrichment over 20 in the GX-Leghorn group, namely BMP receptor binding and GPI-linked ephrin receptor activity. Moreover, some interesting GO terms were observed in three groups. Many terms associated with female gonad development were found in the GX-Cobb group, such as âdevelopment of primary sexual characteristicsâ, âfemale sex differentiationâ and âregulation of gonadotropin secretionâ, we also found these same terms in the GX-CH group. Meanwhile, several terms contained âembryo development ending in birth or egg hatchingâ, âdevelopmental process involved in reproductionâ, âregulation of hormone levelsâ and so on, were detected both in the GX-Leghorn group and the GX-CH group, all these terms were related to reproduction. The detail information was listed in Table S2-S4.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/379/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The most enriched GO terms in the GX-Cobb group: the results only showed the top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-731820633" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The most enriched GO terms in the GX-Cobb group: the results only showed the top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/379/F5.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/1/379/F5.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/379/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/379/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495191" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">The most enriched GO terms in the GX-Cobb group: the results only showed the top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/379/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The most enriched GO terms in the GX-Leghorn group: the results only showed the top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-731820633" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The most enriched GO terms in the GX-Leghorn group: the results only showed the top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/379/F6.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/1/379/F6.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/379/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/379/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495176" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-26" class="first-child">The most enriched GO terms in the GX-Leghorn group: the results only showed the top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/379/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The most enriched GO terms in the GX-CH group: the results only showed top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-731820633" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The most enriched GO terms in the GX-CH group: the results only showed top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/379/F7.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/1/379/F7.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/379/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/379/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495425" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-27" class="first-child">The most enriched GO terms in the GX-CH group: the results only showed top 10 terms in each ontology, which are biological process, molecular function, cellular component, respectively.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-28">Cobb is a well-known and most commonly raised modern high-yielding broilers with low feed conversion but a high growth rate, and White Leghorn is usually regarded as a high-productive layer. They are two classic commercial chicken breeds. In addition, there are abundant germplasm resources with varieties of chicken breeds in Guangxi Zhuang Autonomous Region. To investigate the genomic information and genetic variation in Guangxi native breeds, in the current study, we developed genetic analysis among Guangxi native breeds and three other breeds using reduced represent SNPs which were originated from RAD-seq. As the results showed above, we described the population structure by STRUCTURE and phylogenetic tree analyses, plotted the distribution of F<sub>ST</sub> by calculating F<sub>ST</sub> values and identified the DSRs between Guangxi native breeds and the control breeds. In addition, the genes from DSRs were all performed on GO enrichment for functional analysis. According to these results, we discussed separately as below:</p><p id="p-29">From STRUCTURE analysis, all breeds were separate and each breed displayed as a cluster while K = 10, suggesting obvious genetic differentiation in Guangxi native breeds. But as we concluded from the Evanno method (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Evanno <em>et al.</em> 2005</a>), the optimal genetic structure exhibited when K was 9, it suggested that the Nandan-Yao chicken was more similar to Longsheng-Feng breed. We inferred that there still existed many linked RAD-sites, which led us to underestimate the genetic diversity of Guangxi native breeds (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Cariou <em>et al.</em> 2016</a>). Furthermore, as shown in the phylogenetic tree, Cobb and Leghorn were clustered in one branch and had further genetic distance with Guangxi native breeds. All of these results could suggest that Guangxi native breeds have various genetic diversity on the genome.</p><p id="p-30">Notably, all of the F<sub>ST</sub> values of SNPs are under the 0.9 in three compared groups, excepted three sites in which the value of F<sub>ST</sub> is the same (0.9427) in the GX-CH group. These three loci were located in chr1-96,859,720, chr4-86,139,601 and chr12-8,128,322, respectively. But only two sites could be found in significant DSRs (chr1-96,859,720 and chr12-8,128,322), and these two sites are closed to three genes (<em>HSPA13</em>, <em>WNT5A</em>, <em>ARHGEF3</em>), which were found in all three DSR datasets concurrently. <em>WNT5A</em> (<em>Wnt</em> family member 5A) has been reported in many studies, implying that it exerts pivotal effects in the differentiation of chicken embryonic stem cells (ESCs) into spermatogonial stem cells (SSCs)(<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">He <em>et al.</em> 2018</a>) and the differentiation of the glandular stomach(<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Listyorini and Yasugi 2006</a>) in the chicken embryo. Meanwhile, it has dual functions during cartilage development and in disease (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hosseini-Farahabadi <em>et al.</em> 2013</a>). By contrast, there are few reports on <em>ARHGEF3</em> and <em>HSPA13</em> in chickens, most of the articles about the <em>ARHGEF3</em> are related to disease in human, such as acute myeloid leukemias (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">DâAmato <em>et al.</em> 2015</a>), nasopharyngeal carcinoma cell pathogenesis (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 2016</a>) and platelet function (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zou <em>et al.</em> 2017</a>), as for <em>HSPA13</em>, few studies in NCBI database. These three genes might reveal that Guangxi native breeds have promising values in disease research.</p><p id="p-31">In addition, we acquired some information about functional genes from GO enrichment. 1) Many terms (GO:0016342, GO:0019987, GO:0016339, etc.) involved with cadherin superfamily genes (<em>CDH2</em>, <em>CDH6</em> and so on.) were observed from GX-Cobb group. Cadherin superfamily is made up of a large and multitudinous collection of molecules, related to various fundamental cellular processes such as cell recognition and cell-cell adhesion (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gul <em>et al.</em> 2017</a>), furthermore, members in superfamily heavily impact the neural development (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jontes 2018</a>). Meanwhile, we got two genes (<em>INHBA</em>, <em>INHBB</em>) that are relevant to the generation of activin and inhibin, the presence of them can regulate FSH. According to the trend between inhibin and FSH, the developmental period of chicken might be estimated (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Vanmontfort <em>et al.</em> 1995</a>). Considering the difference of growth rate among the Guangxi native breeds and Cobb, we thought that these two types of genes might have an important role in muscle growth and development. 2) The <em>BMP4</em> (Bone morphogenetic protein-4) gene was strikingly observed in the GX-Leghorn group. This gene is a member of bone morphogenetic protein family(<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Katagiri and Watabe 2016</a>), with great effects in embryonic development, organ formation(<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wu <em>et al.</em> 2016</a>) and regulation of sex hormone levels(<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Liu <em>et al.</em> 2017</a>), as well as transition of primordial-to-primary follicle(<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Knight and Glister 2006</a>). All of these functions confirm the importance of <em>BMP4</em> for egg production, suggesting that we can make use of this gene to improve the egg-laying performance of Guangxi native breeds. 3) We found many <em>Eph</em> family genes both in Leghorn and Chahua comparison groups. The family of <em>Eph</em> receptor tyrosine kinases and their ephrin ligands have important functions in axonal guidance, bone remodeling, immune system and cancer(<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Shiuan and Chen 2016</a>), because of their binding relationship for either the glycosylphosphatidylinositol-linked ephrin-A ligands or the transmembrane-bound ephrin-B ligands, <em>Ephs</em> can be divided into two subclasses, <em>EphAs</em> and <em>EphBs</em> (Eph Nomenclature Committee 1997). In this study, <em>EphAs</em> appeared in two groups, but <em>EphBs</em> only enriched in the GX-CH group. This might speculate that Chahua or Guangxi native breeds have more robust neuro-development, because the <em>EphB</em> signaling has a distinct effect on axon guidance and morphogenesis (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen-Sharpley and Cramer 2012</a>). From the above, the analysis of genetic difference showed some surprising results, which may be since we treated all Guangxi native breeds as one population to compare with others, thus many traits those we more expected and their associated genes were not prominent. But combining with the results of population structure, this also partly reflected the rich genetic diversity of Guangxi native breeds.</p><p id="p-32">With decreasing costs in NGS, various sequencing strategies can be used to study population genomics and genetic variation according to experimental requirements. Therefore, a valid even unique analytical method for each NGS project is necessary and vital(<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Holsinger and Weir 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Etter <em>et al.</em> 2011</a>). Even though many researchers had focused on exploring software and quantifying the error rates for improving the reliability and accuracy of RAD data (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Etter <em>et al.</em> 2011</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mastretta-Yanes <em>et al.</em> 2015</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rochette and Catchen 2017</a>), it was still deficient compared with the whole-genome resequencing. Based on this condition, we may have more consideration of details of data processing, for deeply excavating the genomic information in Guangxi native breeds based on RAD-seq.</p><p id="p-33">The experiment has analyzed the genetic variation of Guangxi native chicken breeds by comparing Guangxi native chickens with three other breeds, and the results suggested that indigenous chickens have abundant genetic information that deserves us to further explore. In conclusion, this study provided a rich data resource and established a theoretical basis for further exploring the genetic mechanism of Guangxi native chickens, as well as accelerating modern chicken breeding.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">This work was supported in part by the National Natural Science Foundation of China (31660635) and the Science and Technology Major Project of Guangxi (GK AA17204027).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-35">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10310750">https://doi.org/10.25387/g3.10310750</a>, <a href="https://doi.org/10.25387/g3.10278128">https://doi.org/10.25387/g3.10278128</a>.</p></li><li class="fn" id="fn-3"><p id="p-36"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-10-10"><span class="received-label">Received </span>October 10, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-18"><span class="accepted-label">Accepted </span>November 18, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2020 Yang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.1" data-doi="10.1186/1749-8104-7-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen-Sharpley</span>,  <span class="cit-name-given-names">M. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Cramer</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Coordinated Eph-ephrin signaling guides migration and axon targeting in the avian auditory system.</span> <abbr class="cit-jnl-abbrev">Neural Dev.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1749-8104-7-29</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2BDev.%26rft.volume%253D7%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1749-8104-7-29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1749-8104-7-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.2" data-doi="10.1186/s12862-016-0791-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cariou</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Duret</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Charlat</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">How and how much does RAD-seq bias genetic diversity estimates?</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12862-016-0791-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D16%26rft.spage%253D240%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-016-0791-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-016-0791-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.3" data-doi="10.1111/mec.12330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bassham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Currey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">OâBrien</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">The population structure and recent colonization history of Oregon threespine stickleback determined using restriction-site associated DNA-sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2864</span>â<span class="cit-lpage">2883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D2864%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.4" data-doi="10.4161/15592294.2014.988035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DâAmato</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">DellâAversana</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Conte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ciotta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Scisciola</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">ARHGEF3 controls HDACi-induced differentiation via RhoA-dependent pathways in acute myeloid leukemias.</span> <abbr class="cit-jnl-abbrev">Epigenetics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">6</span>â<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/15592294.2014.988035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpigenetics%26rft.volume%253D10%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.4161%252F15592294.2014.988035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/15592294.2014.988035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-book" id="cit-10.1.379.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genotyping Analysis Using an RFLP Assay</span>, pp. <span class="cit-fpage">91</span>â<span class="cit-lpage">99</span> in <span class="cit-source">Plant Genotyping: Methods and Protocols</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Batley</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.6" data-doi="10.1093/sysbio/syt032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>,  <span class="cit-name-given-names">D. A. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Ree</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Inferring Phylogeny and Introgression using RADseq Data: An Example from Flowering Plants (Pedicularis: Orobanchaceae).</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">689</span>â<span class="cit-lpage">706</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/syt032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyt032%26rft_id%253Dinfo%253Apmid%252F23652346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syt032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23652346&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.7" data-doi="10.1007/978-1-61779-228-1_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>,  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bassham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Johnson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Cresko</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">SNP discovery and genotyping for evolutionary genetics using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">772</span>: <span class="cit-fpage">157</span>â<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-61779-228-1_9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DEtter%26rft.auinit1%253DP.%2BD.%26rft.volume%253D772%26rft.spage%253D157%26rft.epage%253D178%26rft.atitle%253DSNP%2Bdiscovery%2Band%2Bgenotyping%2Bfor%2Bevolutionary%2Bgenetics%2Busing%2BRAD%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-228-1_9%26rft_id%253Dinfo%253Apmid%252F22065437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-228-1_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22065437&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321876000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.8" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Regnaut</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Goudet</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>â<span class="cit-lpage">2620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02553.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.9" data-doi="PubMed PMID: 88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Inference of population structure using multilocus genotype data: Linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>â<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Adoi%252FPubMed%2BPMID%253A%2B88%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.379.10" data-doi="10.1016/S0092-8674(00)80500-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Eph Nomenclature Committee</span></li></ol><cite> <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Unified nomenclature for Eph family receptors and their ligands, the ephrins.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80500-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D404%26rft.atitle%253DUnified%2Bnomenclature%2Bfor%2BEph%2Bfamily%2Breceptors%2Band%2Btheir%2Bligands%252C%2Bthe%2Bephrins.%2BEph%2BNomenclature%2BCommittee.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980500-0%26rft_id%253Dinfo%253Apmid%252F9267020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80500-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9267020&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XQ06300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.11" data-doi="10.1186/s13104-015-1261-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Bishop</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Houston</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Exploring the utility of cross-laboratory RAD-sequencing datasets for phylogenetic analysis.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13104-015-1261-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BRes.%2BNotes%26rft.volume%253D8%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13104-015-1261-2%26rft_id%253Dinfo%253Apmid%252F26152111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13104-015-1261-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26152111&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.12" data-doi="10.1021/ac991390e"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffin</span>,  <span class="cit-name-given-names">T. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genetic identification by mass spectrometric analysis of single-nucleotide polymorphisms: Ternary encoding of genotypes.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">3298</span>â<span class="cit-lpage">3302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/ac991390e</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DGriffin%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D72%26rft.issue%253D14%26rft.spage%253D3298%26rft.epage%253D3302%26rft.atitle%253DGenetic%2Bidentification%2Bby%2Bmass%2Bspectrometric%2Banalysis%2Bof%2Bsingle-nucleotide%2Bpolymorphisms%253A%2Bternary%2Bencoding%2Bof%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac991390e%26rft_id%253Dinfo%253Apmid%252F10939403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac991390e&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10939403&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.13" data-doi="10.1016/j.yexcr.2017.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gul</span>,  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hulpiau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Saeys</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">van Roy</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Evolution and diversity of cadherins and catenins.</span> <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">358</span>: <span class="cit-fpage">3</span>â<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.yexcr.2017.03.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BCell%2BRes.%26rft.volume%253D358%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yexcr.2017.03.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.yexcr.2017.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.14" data-doi="10.1002/jcb.26329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. J.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Wnt signaling pathway regulates differentiation of chicken embryonic stem cells into spermatogonial stem cells via Wnt5a.</span> <abbr class="cit-jnl-abbrev">J. Cell. Biochem.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">1689</span>â<span class="cit-lpage">1701</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jcb.26329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell.%2BBiochem.%26rft.volume%253D119%26rft.spage%253D1689%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcb.26329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcb.26329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.15" data-doi="10.1038/nrg2611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holsinger</span>,  <span class="cit-name-given-names">K. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Weir</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetics in geographically structured populations: defining, estimating and interpreting F(ST).</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">639</span>â<span class="cit-lpage">650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHolsinger%26rft.auinit1%253DK.%2BE.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D639%26rft.epage%253D650%26rft.atitle%253DGenetics%2Bin%2Bgeographically%2Bstructured%2Bpopulations%253A%2Bdefining%252C%2Bestimating%2Band%2Binterpreting%2BF%2528ST%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2611%26rft_id%253Dinfo%253Apmid%252F19687804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19687804&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269063200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.16" data-doi="10.1016/j.matbio.2013.02.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosseini-Farahabadi</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Geetha-Loganathan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Fu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nimmagadda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Yang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Dual functions for WNT5A during cartilage development and in disease.</span> <abbr class="cit-jnl-abbrev">Matrix Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">252</span>â<span class="cit-lpage">264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.matbio.2013.02.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMatrix%2BBiol.%26rft.volume%253D32%26rft.spage%253D252%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.matbio.2013.02.005%26rft_id%253Dinfo%253Apmid%252F23474397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.matbio.2013.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23474397&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.17" data-doi="10.1371/journal.pone.0140175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Nowak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Mirre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Bjora</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Brochmann</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Thousands of RAD-seq Loci Fully Resolve the Phylogeny of the Highly Disjunct Arctic-Alpine Genus Diapensia (Diapensiaceae).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0140175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0140175</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0140175%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0140175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0140175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.18" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Rosenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>â<span class="cit-lpage">1806</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.19" data-doi="10.1101/cshperspect.a029306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jontes</span>,  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">The Cadherin Superfamily in Neural Circuit Assembly.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">10</span>: a029306<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a029306</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a029306%26rft_id%253Dinfo%253Apmid%252F28778868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjEyOiIxMC83L2EwMjkzMDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.20" data-doi="10.1101/cshperspect.a021899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katagiri</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Watabe</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Bone Morphogenetic Proteins.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">8</span>: a031989<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a021899</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DKatagiri%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253Da021899%26rft.epage%253Da021899%26rft.atitle%253DBone%2BMorphogenetic%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a021899%26rft_id%253Dinfo%253Apmid%252F27252362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI4LzYvYTAyMTg5OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.21" data-doi="10.1530/rep.1.01074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>,  <span class="cit-name-given-names">P. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Glister</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">TGF-beta superfamily members and ovarian follicle development.</span> <abbr class="cit-jnl-abbrev">Reproduction</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">191</span>â<span class="cit-lpage">206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1530/rep.1.01074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproduction%26rft.stitle%253DReproduction%26rft.aulast%253DKnight%26rft.auinit1%253DP.%2BG%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D191%26rft.epage%253D206%26rft.atitle%253DTGF-%257Bbeta%257D%2Bsuperfamily%2Bmembers%2Band%2Bovarian%2Bfollicle%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1530%252Frep.1.01074%26rft_id%253Dinfo%253Apmid%252F16885529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicmVwcm9kIjtzOjU6InJlc2lkIjtzOjk6IjEzMi8yLzE5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.22" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stecher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tamura</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>â<span class="cit-lpage">1874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.23" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>â<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.24" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>â<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.25" data-doi="10.1038/s41598-017-09969-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Cai</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Genome-wide genetic structure and differentially selected regions among Landrace, Erhualian, and Meishan pigs using specific-locus amplified fragment sequencing.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">10063</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-09969-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D10063%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-09969-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-09969-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.26" data-doi="10.1111/j.1440-169X.2006.00861.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Listyorini</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yasugi</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Expression and function of Wnt5a in the development of the glandular stomach in the chicken embryo.</span> <abbr class="cit-jnl-abbrev">Dev. Growth Differ.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">243</span>â<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1440-169X.2006.00861.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%252C%2Bgrowth%2B%2526%2Bdifferentiation%26rft.stitle%253DDev%2BGrowth%2BDiffer%26rft.aulast%253DListyorini%26rft.auinit1%253DD.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D243%26rft.epage%253D252%26rft.atitle%253DExpression%2Band%2Bfunction%2Bof%2BWnt5a%2Bin%2Bthe%2Bdevelopment%2Bof%2Bthe%2Bglandular%2Bstomach%2Bin%2Bthe%2Bchicken%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1440-169X.2006.00861.x%26rft_id%253Dinfo%253Apmid%252F16681649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1440-169X.2006.00861.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16681649&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. Y.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. X.</span> <span class="cit-name-surname">Lin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">The putative tumor activator ARHGEF3 promotes nasopharyngeal carcinoma cell pathogenesis by inhibiting cellular apoptosis.</span> <abbr class="cit-jnl-abbrev">Oncotarget</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">25836</span>â<span class="cit-lpage">25848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncotarget%26rft.volume%253D7%26rft.spage%253D25836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.28" data-doi="10.1074/jbc.M117.781369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Y.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Dou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. F.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">The BMP4-Smad signaling pathway regulates hyperandrogenism development in a female mouse model.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">292</span>: <span class="cit-fpage">11740</span>â<span class="cit-lpage">11750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M117.781369</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M117.781369%26rft_id%253Dinfo%253Apmid%252F28572510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTIvMjgvMTE3NDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.29" data-doi="10.1111/1755-0998.12291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mastretta-Yanes</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Arrigo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Alvarez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Jorgensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Pinero</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Restriction site-associated DNA sequencing, genotyping error estimation and de novo assembly optimization for population genetic inference.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">28</span>â<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12291</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12291%26rft_id%253Dinfo%253Apmid%252F24916682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24916682&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.30" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Dunham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amores</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Cresko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Johnson</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">240</span>â<span class="cit-lpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.5681207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.31" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donnelly</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>â<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.32" data-doi="10.1038/nprot.2017.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rochette</span>,  <span class="cit-name-given-names">N. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Catchen</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Deriving genotypes from RAD-seq short-read data using Stacks.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2640</span>â<span class="cit-lpage">2659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2017.123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D12%26rft.spage%253D2640%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2017.123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2017.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.33" data-doi="10.1046/j.1471-8286.2003.00566.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>,  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">DISTRUCT: a program for the graphical display of population structure.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">137</span>â<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1471-8286.2003.00566.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D4%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2003.00566.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2003.00566.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000189159500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.34" data-doi="10.1093/oxfordjournals.molbev.a040454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nei</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>â<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a040454%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a040454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3447015&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J406700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.35" data-doi="10.1038/nbt1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. L.</span> <span class="cit-name-surname">Ji</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Next-generation DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1135</span>â<span class="cit-lpage">1145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt1486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DShendure%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D1135%26rft.epage%253D1145%26rft.atitle%253DNext-generation%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1486%26rft_id%253Dinfo%253Apmid%252F18846087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846087&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259926000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.36" data-doi="10.1158/0008-5472.CAN-16-1521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiuan</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Eph Receptor Tyrosine Kinases in Tumor Immunity.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">6452</span>â<span class="cit-lpage">6457</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/0008-5472.CAN-16-1521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-16-1521%26rft_id%253Dinfo%253Apmid%252F27811149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8yMi82NDUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8zNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.37" data-doi="10.1007/978-1-60327-411-1_12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tahira</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kukita</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Higasa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Okazaki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Yoshinaga</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Estimation of SNP allele frequencies by SSCP analysis of pooled DNA.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">578</span>: <span class="cit-fpage">193</span>â<span class="cit-lpage">207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-60327-411-1_12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DTahira%26rft.auinit1%253DT.%26rft.volume%253D578%26rft.spage%253D193%26rft.epage%253D207%26rft.atitle%253DEstimation%2Bof%2BSNP%2Ballele%2Bfrequencies%2Bby%2BSSCP%2Banalysis%2Bof%2Bpooled%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-411-1_12%26rft_id%253Dinfo%253Apmid%252F19768595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-411-1_12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19768595&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.38" data-doi="10.1080/00071669508417822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vanmontfort</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Berghman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rombauts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Verhoeven</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Decuypere</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Developmental changes in immunoreactive inhibin and FSH in plasma of chickens from hatch to sexual maturity.</span> <abbr class="cit-jnl-abbrev">Br. Poult. Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">779</span>â<span class="cit-lpage">790</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00071669508417822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBritish%2Bpoultry%2Bscience%26rft.stitle%253DBr%2BPoult%2BSci%26rft.aulast%253DVanmontfort%26rft.auinit1%253DD.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D779%26rft.epage%253D790%26rft.atitle%253DDevelopmental%2Bchanges%2Bin%2Bimmunoreactive%2Binhibin%2Band%2BFSH%2Bin%2Bplasma%2Bof%2Bchickens%2Bfrom%2Bhatch%2Bto%2Bsexual%2Bmaturity.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00071669508417822%26rft_id%253Dinfo%253Apmid%252F8746979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00071669508417822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8746979&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.39" data-doi="10.1093/nar/gkq603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. Y.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hakonarson</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-elocation-id">e164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq603%26rft_id%253Dinfo%253Apmid%252F20601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20601685&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.40" data-doi="10.1111/1755-0998.12136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Eaton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. Z.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. H.</span> <span class="cit-name-surname">Guo</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Identification of SNP markers for inferring phylogeny in temperate bamboos (Poaceae: Bambusoideae) using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">938</span>â<span class="cit-lpage">945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D13%26rft.spage%253D938%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12136%26rft_id%253Dinfo%253Apmid%252F23848836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23848836&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.41" data-doi="10.1038/boneres.2016.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Chen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-P.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">TGF-Î² and BMP signaling in osteoblast, skeletal development, and bone formation, homeostasis and disease.</span> <abbr class="cit-jnl-abbrev">Bone Res.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">16009</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/boneres.2016.9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBone%2BRes.%26rft.volume%253D4%26rft.spage%253D16009%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fboneres.2016.9%26rft_id%253Dinfo%253Apmid%252F27563484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/boneres.2016.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27563484&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-10.1.379.42" data-doi="10.1371/journal.pone.0178095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Teixeira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kostadima</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Astle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Radhakrishnan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">SNP in human ARHGEF3 promoter is associated with DNase hypersensitivity, transcript level and platelet function, and Arhgef3 KO mice have increased mean platelet volume.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e0178095</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0178095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0178095%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0178095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0178095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/379.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/371" title="A Genetic Screen To Identify Genes Influencing the Secondary Redox Couple NADPH/NADP+ in the Yeast Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/387" title="Akirin Is Required for Muscle Function and Acts Through the TGF-Î² Sma/Mab Signaling Pathway in Caenorhabditis elegans Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/379" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YXY8XXmZ4Tgt0nQtAQsFoRXmOTLcXFdxJja6oGLoQ3s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/379" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/379" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/379" class="active" data-icon-position="" data-hide-link-title="0">Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495280" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-GJ-_5x9drK-GqrTDh5gaN1cZFuKzLMfb89uFsZv2ysA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/379.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/379" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VX60hNCu_pNkHkce9Y9ib1-80sSLi31j___xPiSujsw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/379" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/379.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495280" id="node495280--2" data-pisa="ggg;10/1/379" data-pisa-master="ggg;g3.119.400827" data-apath="/ggg/10/1/379.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5418-0751" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhuliang Yang</span>, <span class="highwire-citation-author" data-delta="1">Jixian Deng</span>, <span class="highwire-citation-author" data-delta="2">Dongfeng Li</span>, <span class="highwire-citation-author" data-delta="3">Tiantian Sun</span>, <span class="highwire-citation-author" data-delta="4">Li Xia</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3829-7922" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wenwen Xu</span>, <span class="highwire-citation-author" data-delta="6">Linghu Zeng</span>, <span class="highwire-citation-author" data-delta="7">Hesheng Jiang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6252-7455" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiurong Yang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">379-386;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400827 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495280--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhuliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhuliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhuliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5418-0751" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhuliang Yang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jixian Deng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Guangxi Institute of Animal Science, Nanning, 530001, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jixian%2BDeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJixian%2BDeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dongfeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>College of Animal Science and Technology, Nanjing Agricultural University, Nanjing, 210095, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongfeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongfeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tiantian Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiantian%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiantian%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Li Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALi%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wenwen Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenwen%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenwen%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3829-7922" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wenwen Xu</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Linghu Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linghu%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinghu%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hesheng Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hesheng%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHesheng%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Xiurong Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiurong%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiurong%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6252-7455" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiurong Yang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yangxiurong09@163.com" class="" data-icon-position="" data-hide-link-title="0">yangxiurong09@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/379" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/379&amp;title=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/379&amp;title=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/379&amp;title=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/379&amp;text=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/379&amp;title=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/379&amp;t=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/379&amp;title=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/379&amp;title=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495280/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495280" id="node495280--3" data-pisa="ggg;10/1/379" data-pisa-master="ggg;g3.119.400827" data-apath="/ggg/10/1/379.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Analysis of Population Structure and Differentially Selected Regions in Guangxi Native Breeds by Restriction Site Associated with DNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5418-0751" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhuliang Yang</span>, <span class="highwire-citation-author" data-delta="1">Jixian Deng</span>, <span class="highwire-citation-author" data-delta="2">Dongfeng Li</span>, <span class="highwire-citation-author" data-delta="3">Tiantian Sun</span>, <span class="highwire-citation-author" data-delta="4">Li Xia</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3829-7922" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wenwen Xu</span>, <span class="highwire-citation-author" data-delta="6">Linghu Zeng</span>, <span class="highwire-citation-author" data-delta="7">Hesheng Jiang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-6252-7455" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiurong Yang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">379-386;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400827 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495280--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhuliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhuliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhuliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5418-0751" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhuliang Yang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jixian Deng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Guangxi Institute of Animal Science, Nanning, 530001, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jixian%2BDeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJixian%2BDeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dongfeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>College of Animal Science and Technology, Nanjing Agricultural University, Nanjing, 210095, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongfeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongfeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tiantian Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiantian%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiantian%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Li Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALi%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wenwen Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenwen%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenwen%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3829-7922" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wenwen Xu</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Linghu Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linghu%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinghu%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hesheng Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hesheng%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHesheng%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Xiurong Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Animal Science and Technology, Guangxi University, Nanning, 530004, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiurong%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiurong%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6252-7455" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiurong Yang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yangxiurong09@163.com" class="" data-icon-position="" data-hide-link-title="0">yangxiurong09@163.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=379&amp;rft.epage=386&amp;rft.atitle=Analysis%20of%20Population%20Structure%20and%20Differentially%20Selected%20Regions%20in%20Guangxi%20Native%20Breeds%20by%20Restriction%20Site%20Associated%20with%20DNA%20Sequencing&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yang%2C+Zhuliang&amp;rft.au=Deng%2C+Jixian&amp;rft.au=Li%2C+Dongfeng&amp;rft.au=Sun%2C+Tiantian&amp;rft.au=Xia%2C+Li&amp;rft.au=Xu%2C+Wenwen&amp;rft.au=Zeng%2C+Linghu&amp;rft.au=Jiang%2C+Hesheng&amp;rft.au=Yang%2C+Xiurong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495280/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495280/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495280/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495280/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495280/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495280/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495280/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495280/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495280/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495280/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495280/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495280/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/379" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/379.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/379.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/379.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
