<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/211" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/211.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/211.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/211.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/1/211/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/211.atom" />
<meta name="HW.pisa" content="ggg;10/1/211" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inference of Historical Population-Size Changes with Allele-Frequency Data" />
<meta name="DC.Identifier" content="10.1534/g3.119.400854" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Lynch et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="With up to millions of nearly neutral polymorphisms now being routinely sampled in population-genomic surveys, it is possible to estimate the site-frequency spectrum of such sites with high precision. Each frequency class reflects a mixture of potentially unique demographic histories, which can be revealed using theory for the probability distributions of the starting and ending points of branch segments over all possible coalescence trees. Such distributions are completely independent of past population history, which only influences the segment lengths, providing the basis for estimating average population sizes separating tree-wide coalescence events. The history of population-size change experienced by a sample of polymorphisms can then be dissected in a model-flexible fashion, and extension of this theory allows estimation of the mean and full distribution of long-term effective population sizes and ages of alleles of specific frequencies. Here, we outline the basic theory underlying the conceptual approach, develop and test an efficient statistical procedure for parameter estimation, and apply this to multiple population-genomic datasets for the microcrustacean Daphnia pulex ." />
<meta name="DC.Contributor" content="Michael Lynch" />
<meta name="DC.Contributor" content="Bernhard Haubold" />
<meta name="DC.Contributor" content="Peter Pfaffelhuber" />
<meta name="DC.Contributor" content="Takahiro Maruki" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Inference of Historical Population-Size Changes with Allele-Frequency Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;With up to millions of nearly neutral polymorphisms now being routinely sampled in population-genomic surveys, it is possible to estimate the site-frequency spectrum of such sites with high precision. Each frequency class reflects a mixture of potentially unique demographic histories, which can be revealed using theory for the probability distributions of the starting and ending points of branch segments over all possible coalescence trees. Such distributions are completely independent of past population history, which only influences the segment lengths, providing the basis for estimating average population sizes separating tree-wide coalescence events. The history of population-size change experienced by a sample of polymorphisms can then be dissected in a model-flexible fashion, and extension of this theory allows estimation of the mean and full distribution of long-term effective population sizes and ages of alleles of specific frequencies. Here, we outline the basic theory underlying the conceptual approach, develop and test an efficient statistical procedure for parameter estimation, and apply this to multiple population-genomic datasets for the microcrustacean &lt;i&gt;Daphnia pulex&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/211" />
<meta name="citation_id" content="10/1/211" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/211" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/211.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/211.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/211.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400854" />
<meta name="citation_pmid" content="31699776" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="211" />
<meta name="citation_lastpage" content="223" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Michael Lynch" />
<meta name="citation_author_institution" content="Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287," />
<meta name="citation_author_email" content="mlynch11@asu.edu" />
<meta name="citation_author" content="Bernhard Haubold" />
<meta name="citation_author_institution" content="Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön 24306, Germany, and" />
<meta name="citation_author" content="Peter Pfaffelhuber" />
<meta name="citation_author_institution" content="Faculty of Mathematics and Physics, University of Freiburg, Freiburg 79104, Germany" />
<meta name="citation_author" content="Takahiro Maruki" />
<meta name="citation_author_institution" content="Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287," />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Bhaskar;citation_author=Y. X. Wang;citation_author=Y. S. Song;citation_title=Efficient inference of population size histories and locus-specific mutation rates from large-sample genomic variation data.;citation_volume=25;citation_year=2015;citation_pmid=25564017;citation_doi=10.1101/gr.178756.114" />
<meta name="citation_reference" content="citation_title=Mathematical population genetics;citation_author=W. J. Ewens;citation_year=2004;citation_doi=10.1007/978-0-387-21822-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. K. Chen;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_title=Fast and flexible simulation of DNA sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19029539;citation_doi=10.1101/gr.083634.108" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Probab.;citation_author=I. Dahmer;citation_author=G. Kersting;citation_title=The internal branch lengths of the Kingman coalescent.;citation_pages=1325-1348;citation_volume=25;citation_year=2015;citation_doi=10.1214/14-AAP1024" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=Y.-X. Fu;citation_title=Statistical properties of segregating sites.;citation_pages=172-197;citation_volume=48;citation_year=1995;citation_issue=2;citation_pmid=7482370;citation_doi=10.1006/tpbi.1995.1025" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Gattepaille;citation_author=T. Günther;citation_author=M. Jakobsson;citation_title=Inferring past effective population size from distributions of coalescent times.;citation_volume=204;citation_year=2016;citation_pmid=27638421;citation_doi=10.1534/genetics.115.185058" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. C. Griffiths;citation_title=The frequency spectrum of a mutation, and its age, in a general diffusion model.;citation_pages=241-251;citation_volume=64;citation_year=2003;citation_issue=2;citation_pmid=12948685;citation_doi=10.1016/S0040-5809(03)00075-3" />
<meta name="citation_reference" content="citation_journal_title=Commun. Stat. Stoch. Models;citation_author=R. C. Griffiths;citation_author=S. Tavaré;citation_title=The age of a mutation in a general coalescent tree.;citation_pages=273-295;citation_volume=14;citation_year=1998;citation_doi=10.1080/15326349808807471" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. N. Gutenkunst;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.;citation_pages=e1000695-e1000695;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851460;citation_doi=10.1371/journal.pgen.1000695" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=H. C. McPartlan;citation_author=M. E. Goddard;citation_title=Novel multilocus measure of linkage disequilibrium to estimate past effective population size.;citation_volume=13;citation_year=2003;citation_pmid=12654718;citation_doi=10.1101/gr.387103" />
<meta name="citation_reference" content="citation_title=Gene Genealogies, Variation and Evolution: a Primer in Coalescent Theory;citation_year=2005" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. D. Keightley;citation_author=B. C. Jackson;citation_title=Inferring the probability of the derived vs. the ancestral allelic state at a polymorphic site.;citation_volume=209;citation_year=2018;citation_pmid=29769282;citation_doi=10.1534/genetics.118.301120" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. Keith;citation_author=A. E. Tucker;citation_author=C. E. Jackson;citation_author=W. Sung;citation_author=J. I. Lucas Lledó;citation_title=High mutational rates of large-scale duplication and deletion in Daphnia pulex.;citation_volume=26;citation_year=2016;citation_pmid=26518480;citation_doi=10.1101/gr.191338.115" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. Kelleher;citation_author=A. M. Etheridge;citation_author=G. McVean;citation_title=Efficient coalescent simulation and genealogical analysis for large sample sizes.;citation_pages=e1004842;citation_volume=12;citation_year=2016;citation_pmid=27145223;citation_doi=10.1371/journal.pcbi.1004842" />
<meta name="citation_reference" content="citation_title=The Neutral Theory of Molecular Evolution;citation_author=M. Kimura;citation_year=1983;citation_doi=10.1017/CBO9780511623486" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Probab.;citation_author=J. F. C. Kingman;citation_title=On the genealogy of large populations.;citation_pages=27-43;citation_volume=19;citation_year=1982;citation_pmid=11338422;citation_doi=10.2307/3213548" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lapierre;citation_author=A. Lambert;citation_author=G. Achaz;citation_title=Accuracy of demographic inferences from the site frequency spectrum: the case of the Yoruba population.;citation_volume=206;citation_year=2017;citation_pmid=28341655;citation_doi=10.1534/genetics.116.192708" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Li;citation_author=R. Durbin;citation_title=Inference of human population history from individual whole-genome sequences.;citation_pages=493-496;citation_volume=475;citation_year=2011;citation_issue=7357;citation_pmid=21753753;citation_doi=10.1038/nature10231" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=X. Liu;citation_author=Y.-X. Fu;citation_title=Exploring population size changes using SNP frequency spectra.;citation_pages=555-559;citation_volume=47;citation_year=2015;citation_pmid=25848749;citation_doi=10.1038/ng.3254" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lynch;citation_author=M. Ackerman;citation_author=K. Spitze;citation_author=Z. Ye;citation_author=T. Maruki;citation_title=Population genomics of Daphnia pulex.;citation_volume=206;citation_year=2017;citation_pmid=27932545;citation_doi=10.1534/genetics.116.190611" />
<meta name="citation_reference" content="Maruki, T., and M. Lynch, 2019 Genomic analyses of population structure reveal metabolism as a primary driver of local adaptation in Daphnia pulex. biorxiv/2019/807123." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. W. Messer;citation_title=Measuring the rates of spontaneous mutation from deep and large-scale polymorphism data.;citation_volume=182;citation_year=2009;citation_pmid=19528323;citation_doi=10.1534/genetics.109.105692" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=K. Strimmer;citation_author=O. G. Pybus;citation_title=Exploring the Demographic History of DNA Sequences Using the Generalized Skyline Plot;citation_pages=2298-2305;citation_volume=18;citation_year=2001;citation_issue=12;citation_pmid=11719579;citation_doi=10.1093/oxfordjournals.molbev.a003776" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Tenesa;citation_author=P. Navarro;citation_author=B. J. Hayes;citation_author=D. L. Duffy;citation_author=G. M. Clarke;citation_title=Recent human effective population size estimated from linkage disequilibrium.;citation_volume=17;citation_year=2007;citation_pmid=17351134;citation_doi=10.1101/gr.6023607" />
<meta name="citation_reference" content="citation_title=Coalescent Theory;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=D. B. Weissman;citation_author=O. Hallatschek;citation_title=Minimal-assumption inference from population-genomic data.;citation_pages=e24836;citation_volume=6;citation_year=2017;citation_doi=10.7554/eLife.24836" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inference of Historical Population-Size Changes with Allele-Frequency Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="With up to millions of nearly neutral polymorphisms now being routinely sampled in population-genomic surveys, it is possible to estimate the site-frequency spectrum of such sites with high precision. Each frequency class reflects a mixture of potentially unique demographic histories, which can be revealed using theory for the probability distributions of the starting and ending points of branch segments over all possible coalescence trees. Such distributions are completely independent of past population history, which only influences the segment lengths, providing the basis for estimating average population sizes separating tree-wide coalescence events. The history of population-size change experienced by a sample of polymorphisms can then be dissected in a model-flexible fashion, and extension of this theory allows estimation of the mean and full distribution of long-term effective population sizes and ages of alleles of specific frequencies. Here, we outline the basic theory underlying the conceptual approach, develop and test an efficient statistical procedure for parameter estimation, and apply this to multiple population-genomic datasets for the microcrustacean Daphnia pulex ." />
<meta name="og-title" property="og:title" content="Inference of Historical Population-Size Changes with Allele-Frequency Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/211" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="With up to millions of nearly neutral polymorphisms now being routinely sampled in population-genomic surveys, it is possible to estimate the site-frequency spectrum of such sites with high precision. Each frequency class reflects a mixture of potentially unique demographic histories, which can be revealed using theory for the probability distributions of the starting and ending points of branch segments over all possible coalescence trees. Such distributions are completely independent of past population history, which only influences the segment lengths, providing the basis for estimating average population sizes separating tree-wide coalescence events. The history of population-size change experienced by a sample of polymorphisms can then be dissected in a model-flexible fashion, and extension of this theory allows estimation of the mean and full distribution of long-term effective population sizes and ages of alleles of specific frequencies. Here, we outline the basic theory underlying the conceptual approach, develop and test an efficient statistical procedure for parameter estimation, and apply this to multiple population-genomic datasets for the microcrustacean Daphnia pulex ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Inference of Historical Population-Size Changes with Allele-Frequency Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VD9NiykzD7cbj8A6bhq-e2h-stCRILGU9TQBOPPz7gA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495284","apath":"\/ggg\/10\/1\/211.atom","pisa":"ggg;10\/1\/211","ac":{"\/ggg\/10\/1\/211.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/211.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/211"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/211"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/211.abstract":true,"\/content\/10\/1\/211":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WTS1mcKeFr2DPk\\\\\\\/upC0xHqfQp13t5y75qEHEqdtuQOq53H27FhrlaZ9SmWMB6HA1rhg\\\\\\\/\\\\\\\/1Vcc1x5T2ry\\\\\\\/oqj79kNPTXJDr8uvWjwmQbMtXzeyxYGETTyAYhMlqswmb8sHoyt9Iwv6amRdKGja0B3Rrwuu+N+koA6XVpVwq0HFVEBpD\\\\\\\/LX9Iz1JVSo1lw5r\\\\\\\/8nDSY5qsi9Zp7rg5el0cX0zcwdg1PBFGVwfeS5G+XmLBd387BU25xxkyf2u1nKXkhpEivRTrXlZ2xdeVFEiHDXUZS7ClyX3E0RcuxCorkL4UoEPq+U8vjHtrSGN\\\\\\\/Chq\\\\\\\/zyoX0jwyOIk44oE8L0F1sY1ecqIZbKISqDuC0P7OTap3\\\\\\\/QIIdoD5XwYjBvvqZ4B9Wjo1gT\\\\\\\/TOidq5hzzCwqfv0N3A3xO95dkX7OPOHfjCRV8doi3zF1U+sFjjV9X2VKljRjBr7qqZtDYOcWRBbX\\\\\\\/Ka8T1P210Bo5YN99iN2afFZI9WmtPzUZJf\\\\\\\/m2dFF\\\\\\\/ctEweJ7v2noupkl7IIe2udK0RRsA7FCJgq0gtN\\\\\\\/NF8CnlSFkowQT\\\\\\\/GPJyxFPwI8Tf57nsxnBGELlP7cOJMmn1XI\\\\\\\/X5zgQE9ZUUPi39VgXE0Ovv6aX1NlYcdZ3JiTwEmuw8R5Zw2Bka\\\\\\\/mVblggi2ng9D5fG56BmtxGj0+yIPnPfg+UD2\\\\\\\/8EzOq49MHwdnWhA1G9lRTxazTlmiwrW2kVAvAbT+WVcK++bV0lsEbOa7KoHKG4unGfQLks6ehTKNfjdIqVo4FoSZI59UlX\\\\\\\/smqqDxFnFwxOVEa1kfpyv6qsgcxyCzvrPb7pqEf242Ogod0CT1vN+trMq3YLcmIk7Ain0LT8uWllfm\\\\\\\/Tos\\\\\\\/PBuTrj5vI6\\\\\\\/\\\\\\\/rJKSfW1OK2ylc43qPJbCyi\\\\\\\/xCySf0fdX6NaLXK1cPHNYUnoB5kPTSN3oTOe9nSptMl4iqEfedWlkFOdT1qSpBtaGzXVmMupOHfqE1IUKH3K9q1MZTv5gL1NyDQBCttrGZk1MlwJCaFtcdeYNFWU4HvBG0jiYyPIEv+y5XDNByrXd3coEe9DJB5wrT1sWMvPELAHV6NclgIDUPse7ZoU1wUD3tZliGSiKYCIbbJA0wUQfd+CFSIcUio++u+zaJCXcjd61SQJX9rJMKT+TqmDzePgVUMqfsXg\\\\\\\/RWtP1Y0dJBJWzCAW9o1B\\\\\\\/iNb+DJnFmw28mQK5gB8HgWkUkEM0f1vpisNCDAjsE7QPWAJeYFvs+ijhjJUvJui0eD6CoA2iGJWkNFrqRLmHt35OTTEjsprjC\\\u0022,\\\u0022iv\\\u0022:\\\u0022kgPEZOhgePm\\\\\\\/I4eKCX\\\\\\\/w0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002285af34b6fb36decc703117c941290df0656f89e4cf1f6357ab5ef44749e77cf5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bwh8ndhRYkrhdmz105ZQv4ievIKXMj9Q6GTl5CToDEkvrMlgOw5w0Uzdvipji0QMMwcTzanzOvFQRVNLE\\\\\\\/lVz+wooeLV25ontBZuGJNdncoyUpMOpQ7ILY5GT6quET2kAo1LSzyuDD9V6CVASJWueS4r8fSMB7XOQ467BY\\\\\\\/48S10lxk9Qox7Fqks3EpD+DCVS2SixDSJo2DTmVXkROE0Db\\\\\\\/0gdB83\\\\\\\/CTt79ll5WTg+XMGwzl7TSgF4jtilRFSZuXi8mAlrz++o+tANu8E0BPuy2zO9MA9hGAXehp9jhZGOfdgcnhKj1cCd9Xv2AWGldp07OlRk98LZcMv4gcQP5Tlwdfm\\\\\\\/794t40pvy7p17TNKk6khZjxI6EoX2OvCLJ46IOAb9i5PtNBcRZz\\\\\\\/xPj1kk7fNTn2bSPv9kNMbByqdRL28k+lm+D\\\\\\\/Dd02CrtPieF2RsGVwNLO9qX8qSHuwgTO83z+IFIKOE8NGbOqeI8dIOnSzOoDKTSr\\\\\\\/numsgacBEpHILUEqtcE3w7osK828rDORXNGyX8KVxSYi3aGxBcAmqvso8daMFfqJX2yLCltGBFpmomcBI\\\\\\\/+EFIA3GYJLVzudDCL\\\\\\\/F4Xy44QArJ+2xlq+yI7gt6IhdoSpBWJzuapdKYybmS0S9WYhEnvNMI4ci1HZKz1HXCJAa966ycnJO4wVykkvv+W1e5eEvobLCKLhj2QxPSeJrt7pOosPTuJyRWHOMzHf1pLB9gVZSgbttFDZErWIZkwb1it+oktSE\\\\\\\/a7GPeP6MvTff3nVQPEf+90qegiJu\\\\\\\/ZKIrI631ZlLVRtgp8bKGZ0yJDHKXEFREo8JhM0umc4tr+3d03PnK2uD\\\\\\\/39LC6Y2Ju76lCKH88l+z8y3i8DV4Q9Q\\\\\\\/vPEsylRk4\\\\\\\/J4rvjNiS9\\\\\\\/D4K4qkC65vgJXUF5\\\\\\\/jcfBmECrbsOwelq3JeeDAK31HtMX+QLJMKIes7SpZUcTSMI1QuEYYN7d5MYQhcQpIJIEThUxHA6c5Gd\\\\\\\/CmtV\\\\\\\/znSAaSdRD5PAUAYany1rEOH79NhDXQmuxnxdtS89i3f\\\\\\\/\\\\\\\/RmO1Y5YjkBAiUYg6l9IceHKnsNw\\\\\\\/Nt\\\\\\\/\\\\\\\/jPNKTsd8eH7J1og0tiWUOaalpwQnUWjm4jOTt7xJLQpfupXDD5TkwalRUukG9DMu1boD96vta2YIXlcG2S8qs4srOfCYismBZeZYk6RMlQO1r7Up3PRPGbIYZmF\\\\\\\/DC4VrvtP3J3GBPmfxvqirL0R24A0VTDn7456mTAzUX5Yw1yRP5PX1eAvDlUykFTdWjEay7VtLBgV7EeeZBhEvFk9E2VY7Wz+y5fJUySfcDO2NxfXMQ34ZDAkoTCLOgMe\\\\\\\/QvVjZDP6ZuVM1qdaQTdmk6uKGq2uI4NQrfnvH0PTKUzZ6ztKtbsDzG88lMiAh0+na7\\\\\\\/i+oTsoOABSubglyZJ0eDjyBsdJMcNDtUUgheKFIc0Sr3s3cXnrifARVQuwDLCSorz+ogkHLDAnwBfNIoG3vIXYOjXTxjRnr59lb074Qq7etvsSLDIlDkIG+dRxnCvO7WthE3UiNuZYU2DY25mNdyeY6h2AOalMlqziphzDKaZc+f0yGilyv\\\\\\\/vdCp4pW3GjvE2VexzdQSqH2sLGCg5r537Xq3dLvJ7EyRQ2BbDG+qIdlb6u\\\\\\\/D+i7PsoFKplP2Rd\\\\\\\/kqNbqYXjdKgVN9Hs\\\\\\\/6lFg1ttCVAfN11nPXXK8KRcq7DQJmcIgx1ppS\\\\\\\/\\\\\\\/40sF1kjrcNYyulOIXkV2yrHWw1Vv2dg5BpZbKF19HoKrJma18fZASLeRhNfX6sjoZNEtHFfB\\\\\\\/it+yCuzzl6E2exxi9UO7B54s+sP7dUpqIsrLzSozOed5PIWM559O18x+VIY24gXq3dx1q9IB+54nrT4\\\\\\\/8KVpgt11xpYb2yvUWU1EEvibis\\\\\\\/tMnOqtwBkoOAFkdpoixm9ARHHc36jnwoA4OjWdWNjhawkRw4RI3e9pD8KBVMj2098diP0royZgRKfZu3CqQ\\\\\\\/e0CL5qAKqTUtLL1p2lFjPAP9BBaMX8AFk\\\\\\\/QLG+Wm62IgFwCe+gDZIjpfwqUyTy6uk\\\\\\\/F42anj4\\\\\\\/lT33GSolFURmrLjVK\\\\\\\/W9haUY4GvoTGOQwuWWWkwpKtEFZFwnTbGXOIM9flnA3Vpfnduq913RuU+FOAXtuH+ySPod9+4eE4b5aFfwSjJ9YUJk8LqQQ8s6aaYqNhk9aSBqrsQcHr4yGeCFL8QCUmO8zCVs\\\\\\\/HVzNzgjoAeXDzlC8u5moaJV9pgYzvfkpcfOqTnPI7baGKAiwoBjEb\\\\\\\/cg2f47IC6sljZ\\\\\\\/61erhWeBMbytJLKpC0VDvcmbIYbB2HDZ2elt94wnxKIiURhnfwE1mI9ddiITQNBBQUlNBun1wljZyTlKEnHdgRnS+d8asoySvehHWC4aJDxiqVdrFphsVBymzfc2vBf0kWf3vDAdcKaKPpV2yeDjAsXMSwUE8GE\\\\\\\/hUqqGd1rfjEawL4nZIrL3r8HW4OGte0gi6Evr9fvMKLKyxsV7tpUgP1Umw6BZB7oz+191YqpiSUp3PHSaP00v1z70JJT1C3GL0Xgq6gd4HRXTFQkA4eVZWaGBmMvt1KQCwxKfMJObiszUXkuaCZNTk6EJjBRHxdO2\\\\\\\/LkhmCeih92py466U6uen9k6nI9ghZpnX7THIud+s98QR\\\\\\\/dNqyNySxLQ8RoqoWqPlBCp2dP7AL946pNauE3fOMNiy+OMOYH3uUv4hTNLL4c7lVeNoL6zgzQYlu3\\\\\\\/PQ4cgSK4JiiYIt03+zB5YpL5EZKM7IQKaXb7kXT8E\\\\\\\/W5ntUbvXvrnCLdG83LwiiVcsQvpkLJ\\\\\\\/UjBK+BCyhPhM6okF6lha+F0GGOsujKDy9IE+Z8kx1XupGGvnEKEWkQX7oc\\\\\\\/swa0lHGWM326JdaQBZGXr5mkV7Tz+0Lqcj3HVLWLpKO0Gc2iDvSMeTxBGpQ0YqAOe3g5yv9gRJx7ND6WiFJ1wWmPRf6t2AM5bJlXwDeYghoFsUjyEU1sq+kU3Umg7LNh4mtgR7I2uanN99y85+2KX1HwAB29mf3IpZ+HtKnFl2PRYsiE+kbCMVETRu9z\\\\\\\/oWroZDwrH8xBOxJly34gQ4iQeROXZ\\\\\\\/GfY82AkEVp8pO9uLhv7fT5i\\\\\\\/UfSuDcPYIL3iOr8YrFQrkueEELXUZC0bfV4G7PujfS++GmL5D7dN3ktuW5UMfHN+g3baePiqAb9Tg7JcyUzTV6e4mCx\\\\\\\/\\\\\\\/P69WG6kfxrItsR6Jg1\\\\\\\/TEzQcA+CqG01d41tKTkz+htM7XzVegrw\\\\\\\/KjfNrhFX934wypszHmdAD7VJRrN2yABSyJhvgFu+JFsCFTj0Nvq8Z5jkbyYB1ar\\\\\\\/q8IL2L6W2i2WW9yzyrbzx6SO750KNqW4LK+LiBEX+STtqjEc6JsQuyrUt3B7oSOS3UzfI6dYdnlw2wtr3UKzYQlcHDBAyhEdNYJTU1WCUhcK6Eg2BVw5oSx91+FgRH2K6drfhKULgTonOHePxsexldSRRhep1u3EEdIt3SvxNuKNxjvbR5wa2VKeLV0XnF1xo6rMNGXZFzez3g7VgxAKkdbbEO9Kysd+oTQlQlkpHMGM+CwrCEaKr3A3Cm5meegVMhhRoGdbQfMMNDrp5PfzVDADXya0BSO65jt2t\\\\\\\/0KJKcm9tW8RvkHGZ7pXVljwywrDTsjsnjxx1p\\\\\\\/rayINYSlti8zAZUIjJ38lkklEMPmngFxZiIKFSDcvJfRZ8Xez6KkHyUCoKlIiH3hPMkHj7bT194VJCL\\\\\\\/qXEfei7K5gDfKuCpXBtj9sVmUffuzk0\\\\\\\/UPH46CncoasE1PQ4ZAxLVnfFZI8PdRQhiJu4GoAN4JHItfGhSQLieSvujO8u7i\\\\\\\/evGJD1m+\\\\\\\/XlV5+6FLVIWsbAojzcu2AO3S8Xkxi+OkVQ5oEE3bRG6DnuZIFc8e+q09eR\\\\\\\/en3ocjCgB5VYQ+\\\\\\\/csPnRURRiaxiSDH39Yd7ElSQ+2ySfLGJaDO+eDn\\\\\\\/h0ey7F1BBd+4tKqXJ3IlLRNvAuFlxQSTYvi5GuZ5TWxJWskYU6Qe6bSbGYjk6mvdtaGPSnWwPExe4SYQH6Tt9DjwPMkcGiUHWJGzkCzsf8jJ5rgPiF+\\\\\\\/Go9BqzZojjMwSrZcV5dRr7xNcjq1zH47j6JvVuwmx5eC84KDnE5bgWctIHUm123dygRtjnSMAQ50Hpnk0QsyZBkCXsPh5loWwtsfyV22P3Is4teSDNlX2oNzTPwEV8AVTw+uOBoXCN2MzmKl8HdCTmZr21fpcmx3Vz2D4cPC2jtYQzJdRCM0O4tA2p79lY7LyaaFVi9AYRphoU+Q0mRNYmTqwzExLs2Js2+kBbr\\\\\\\/KzJqIignWMwXwUppVS3UrTYpF7gXWsn2VNu+2I\\\\\\\/A7ktIVFaczoc3V2wYA9yh1lcQvJOP7x2QD+L+e09VocrICPg6UM5yHZduBHU+q0EDePSQqQD2DQm97JNfPqvmXx3mDKIpqsylh53RlXPUd7rHMtuW+0vxfkjB75DL4kIxr3PzCMeqVhHsaDtBRtfYzpyNXjJ+FbZJP\\\\\\\/VVyr4CaJt3w3o6vX91ibfI19zliPCzL9smwAIT9qShgx+aLC0cY+OcTuItpFxHUb9NDXnvBGx0A4wDwu20GSLR8s2mOJTj5uDbcNW9ulEZBS59pdHU3m9hrrqYrISU10bYsKPZyNZPkAadVyFDInXQAUco+00O0tDvXSGUnUiBurdIsoNi7kQyGMeXEOikKFz9jNneWuBsfSw0CxlDagFgh3qUbN1RVIdFuhLjkYwnoIKVjs8Fm0uYXQ25dNeyo+zZf5X3FQeVKtd\\\\\\\/sP7dn30AKHJqFcx8h36tbJhwwtkRkv3xJkBrKmu8tXoreJAmaCf3OjCJYN5tb5B3aWkSVe5uATkh8oXQtvHddIxPbZP30jpib8nOrPDOWq5YBxRYeJN8FKWMSWFV7U+kfLl5OCMBfRuJ2zJkUjxrmATUWSDLjTVmGTi\\\\\\\/ZbxYOqLdb0AsKKSzxwSxWyXBxqCPZ6HwMkByBeikW3VktrrkjZoT4MFN30XoDIcobs3LU+BITSqt+Bjk2gDUMVn\\\\\\\/wCEug9JLujhzhJayQ2Aw1Xti75zyi84qFOFJC0UTz5Dsuxjr7oETXB+8j5DBgdlds9u3BjHN0mTzDlPzlGkjutER5\\\\\\\/rqHEyQ256mo5l4I9Z0B8u959HJFp6Te3LE47LyZKgOiYkkS64BcVFSfRuThEx01PQ\\\\\\\/PZ4iAUM3twmPahN4EDfT+IvFZDYufMWWrrrSevgUBJugbe4IEuWyLNIBqhz3uda2OrA3KQzxbgPwYp\\\\\\\/MnvIxBcAugZJRhSjAzOodvGy0dzc8fcVL01vUPEFbytiE5llVQr7CgdsNoqBUSQacZRrsJibwYVeBZJBTAlwJgFYQDYesKrPZI0WD1lWvDwkOKnDWC3ncycwRt9HN1eyJ4ad9Yyi8rfmI77A5Rk5oCwkjRsQejbg52CQ436P\\\\\\\/ONzjfgrmHr3z\\\\\\\/i\\\\\\\/xGrs7RZoxuy2qjHH0sBFk6dAyt6DTMrfLnqlPupz1cVAATCAW4TLjofMalvjgG+VxljSkJH6v+ydeAFq7FVHo5oNukt+xBosmLPUVKVKLFPoG0+4VyYYvH1XBTB1M84dJ8JJx303ccluQByu+D4poiDFDxnOqPSkfC1pPHLRWWBA0tTBgOBLSvyLIN\\\\\\\/d80jB6VK3tZ6olgaYWIQqHZVGpnWc1odWQ74w9FiTJdby4R388UEl+K4tZX8Fs6gVXvfAZrmJxdqJQKI5SWiXRarmF\\\\\\\/TCsZoo1wxMtZVW5h0y6lVNS0njm2sH9lPUxEnxwVwudDqwP0e8dtfeZx5BnjuXshMFWvtvdhsFtxp6W1Wg0Kcyw4zd9zTPpOeteDuTNrUscv1GbMGt78SgVZj7SLUflFGIwaOoMlMHgeG7oD+ZD43Qa455XNBLEvC3VS7q\\\\\\\/PLLUInwoynHHxspgm19FrxI1eBPv7kDWCMrIREleQz8vuqzcxetRieIf8aNenVk7lwuzEmxvIaIv4IITODnjKIsj5izQ8KFmmfuaZdF4lbzjAlRQtcGpOg6kWDQOh3+Tu6ZnLIlH6TZrkAjgkPFcH7EdMjTWnROupiqVK+PKJr62QRGkxSfBUgjAtRJNXiem41ysOrXaa13uLz4slEzMA1CcSDWSlZhh4pQzIIrk8LYMRr7\\\\\\\/t41R9p3yXZz\\\\\\\/FwGZtXSlmRdb+dii3hhIBPmQf0h9BLxF7KfOcJxzUB4BqwmC5AqM4m+ySHn8DnptlzgLNH1bw7Y+Xo1B82r\\\\\\\/JXn\\\\\\\/kKLW53hRpbpqZ8c\\\\\\\/B2hQ\\\\\\\/Q1KbrwMFWe7livsoWjMJ4UnW8bVy32PYwVi2Uj7JALP4XC118wT161xP954q4kX0uy13sDpVN1c3rTGNdVv6+37Vzk8dCgCFz+UxpH4k1CGr5BDf7esFCy81\\\\\\\/MLZaC2Dowk28fIh80at8xotFDSSuJ\\\\\\\/xjtEoB1a2agv97zz016vp558dIR9cONm8gnsEZPdU03d\\\\\\\/C3W3NnO\\\\\\\/Me5wounEqPvt3DLTUBkhOaVH5cXJwim6Yo2LPTHNW92iHMHJkmPKDcvxSn4vPLzIzu+qL0K4k0D\\\\\\\/YiAB\\\\\\\/Y+JhS+d+qUCHjf83XcERlCDC6QaEy73ii6okfaK44arxXT8zcVVhLknmaWU3x63dyO9MmQE04w\\\\\\\/AaUYK0jDDp7kik0WEypFi2djpVvoqocY43sWxeOFP6HnjWkDNEmBMGPLC\\\\\\\/8PHe3m4TxQmIoyXpleGdUf0HcsU+qRwZTvvwXTHJs521D2UPqugcd2+HWlNNatW2XHTAMvDmt\\\\\\\/1TdpJ4FsStgyiW1s\\\\\\\/pMz2\\\\\\\/+8CSaBLsgMM9RurBOouHt97vC8yHnc3OVSqFOODAvLB4Bxoh99iMaEY6Wq43jJc8e1JEnap2RftEsqvB6fzEvaGz1ZOiNCAJEVJgZaeX5z4JZXMCEFYSO2\\\\\\\/wsFouNtGw\\\\\\\/4Om407C2jA6kIUnNiHGWZJzoa7AzEtpZ1kTACdkT4k9P0A+rCpz94rbXEX4\\\\\\\/n\\\\\\\/2YLzMsiPmsPv+iLn6uAECHEVQcc6MRILJw5flYvob9M6efj9k+V4WsQyH7IiXGAzuioOmhciQhR1V29VtY8JItSj9c8PcnB0RwctQEW+nEyLyKmx68BtOxVXCtNVmJv1+wK9GYGkrT5lda76CfnDWYg9P4s2syGMDVrLYQlM\\\\\\\/wnHW9hCaNkBKA8coqm2\\\\\\\/ql3i6Ywc4s99\\\\\\\/EPGdT4eXc+tLgwwB+UYpXrS4oszTObOAUuqA2TMvnBvhpTrM6FdIAy34ciR7jyIaPkkZBnu\\\\\\\/SDQMsSWbBEHj2tEnl+RkIVmcfyhU+WhIsFNvnmWunIrvZQ5sqLLy1bQa3TXpAvPHbn7fcgrBp3YYkZaxYaoHmtHqjFX5VEfDeA+SGXUZEi87gyE0nQoyMvzP4gTl+Po49NypDS0NGFBb4lrhdeEYGVSR1LxMoT8DHUn+wqjxt0n2lhJUFWeJs7HAzDnaj\\\\\\\/XffGZaoc6imFFPEyUzf0hhDlKAVS1O\\\\\\\/i328sjUR1VJr9hFijejuq4HDofq6X9Jl\\\\\\\/dbYUo0H5PxIlvj0WRzyuUuPs+01VFjEjpY76dywx\\\\\\\/t34NDg8d4QjzA3p\\\\\\\/AMbLA8\\\\\\\/1rcaSmo4NDRRec8qZtziGaaYv638yelLja9yAsZgNQizNwbJDsNdKrunbhSVnOFNcgIXL2tqGu6lsdqeyBq3cmmXbp\\\\\\\/\\\\\\\/4WdhwkTXuQsJAQHZp6sUTIn2WKGXZj1ylAXe6xq7PtiRfqi2SI0UK2xfxpOQXLngAX+LrKEHvtfvs1BtR9u9KXKQJRJhLWwHg3u7+\\\\\\\/YnUhBlYuu\\\\\\\/ayqZuu8\\\\\\\/Dm5UMU8\\\\\\\/cus1DrX7+ePjP6bMXYf93YJjP+TUMUrtlYMti8nTji8fECSjGS4GhRtpV7cA8D7lC0ytTjWyZPf4npt7YCvCKFgwNdcL1fBgHNGdtYnS\\\\\\\/QX3InH9PDAD5ASo6xeXJAt4iUF1kXIac6fteZjiMCn81x5B0eHpxwTJRyBh+hsi8kJhkS2Ra9xCDQlOWimj2GcnmQp\\\\\\\/C4OkC5yyEiQ4Y7HVBXhDQxmBMa4fBfAAs4YwPOM6WAz1KwDZ56+z00cD0qgmxhzMs0Xm0YkqD0CUOuDBMHKKWG2iep0qcgEGazGQSU8JPdIbpC+NaGTuYrt9a662h9MVCwXjllh29HPdI9dtpx5NOz5NFXNxYxZhrtYxdC\\\\\\\/wF4Sbwtca8sZQJaZPIZGduY3a\\\\\\\/eikjWXZTrwCK4dI\\\\\\\/qbP1penDs+K3MMmR6O8dF0HzI5wQmsGRvfODmiMmO3JytizH2bcLEvVYWnnLI3Z4EVHi7ZXBEbBEH6UM1Fxn9dIBj9jf4E5MWoZVce9ivSCgciMa0wocWSSQvLyXhtRrE4E\\\\\\\/IYtcVSBcYkuf6grod3ZfeSrRr1dfPgk00W4tYaATRalnRYBYbrrfXPo\\\\\\\/Nxxb\\\\\\\/HPOb0xsFWLFJ87YXACeMfYfAgB5A5QKatRq3NrFcPCLKniH\\\\\\\/8O3g5DDJF3f6AJhiFA0WO9HZXgRRjQaq6R4\\\\\\\/fewCa\\\\\\\/fnt1tPRqDYOwplWeTHZ2ttpRsx+CGULY7nJWT5nKlxLIZ3SuXAFViNSNbNHSxU86NysIX\\\\\\\/s3NUsQ9VrBuw8qXitT8iXnr9Qk6VghUPDeplFphqAfPfa5GUdSB3yv1yNSVAxxZU+eylRCAg88r5dcsUj\\\\\\\/8+cH2wtwFx1SXVPC1qH7FjsuiB3i1T3yCTVWfAv+kyiTxsXhbXhExl+empxANaNzz7kaVqLrTK+6Phc3muEukLhBHyPs133Qu78joVzPaNV8KHOOUmh4LOHTbGRev\\\\\\\/blzyvzYAbc0GwIcIFLIFvo8nVWV\\\\\\\/hQgfb+nH4TXP12lUuNQEtOWEnW2EP4z1auyC168gDP88DmZJnL2TdDlfVNeLQ35eSS7y32ZOa\\\\\\\/hf9DD\\\\\\\/7revD\\\\\\\/+BEZBocTpXu6dlPE4ynMnlNJ0XTcoiDKWqZ+t5Pywv9EQlez3yE3qLrO39D1mUqCSl3OScEwIo2g5Qa268V0kMQtr4BsosRpv3XhBn4aGevzzHx+h9LKVGfTe5XHttFSqyR63jaq\\\\\\\/fD1ll8i9SPuR3gg4vWGLZDxqJFm2Ud8P2IPJ6Zj8zYnErI\\\\\\\/HHdwW47TjkJVvYktginUK1PLjPFj8ncyFf1u9gsyCM8d7C2bUyRq6VRVGf\\\\\\\/jzFGLDRXae\\\\\\\/Q8klB6\\\\\\\/acTWtQX\\\\\\\/\\\\\\\/KFtGgnl6ebauRlCwSf2jKymNYXyfjpwW2aZ2RgJ+9W6gUyfUSTZ\\\\\\\/y2XOnPu2kidwSiNM6vmrk3gVMc3gCJCHHnAsn0CTQmI7LIRsQFJh0YPGzWsyRMfbvzd8SOq5zQd06Hz9HrHHnW9ltNLBRVPZXGXQqVew+DWH1UFNP+olScGU48tfzHKRxOxrdFaWyzg\\\\\\\/5kkBhwc38YbgVzYA6fMSCL+pgX8c+a8p4xWHXFD7cR1FQDQti72jm8BvwATJSQkbCvutihi6cUVpO0egct7uvBJ3EAHAb79c+wDUom+jOYbOi31knkMgQb0Kwfwe1W9dZrlCqhL1nmzLMXWgeA+hAXTWJAG62tWvJZhK6M6FNXr\\\\\\\/aUhvmkXt3o1SENnu7OXecxiy+YCCWMEcTmFdimoNvSyeX6whsY9JkXftKrMbDmwCWURxpIP40G53tj8LVEkLMzqqcBbrNGFkduF+0OsGSZQsyhVQPOEBzgAQEeMMTtY8F7RlXtCvLXih9NJXSXEQnTcoPuV5HsMniTofUaWZz3psWp4HFL2V27U80FDOm37CSxvpBpPXbx6JXQ\\\\\\\/yXorEAnRkx2GoOEqDJ1bzQVeirDWGYK+LEZc2yK4pwfcQ4oeF9Y2syCalO7bLprmyX6v5HoLkSlDCQQqYxTIgLQWxABpV8KZ3J+b5\\\\\\\/xeU\\\\\\\/Q74pGeirPXBqCofYBWLmb\\\\\\\/De+jewsr420k2kzSJftHcREo9PUZSfcQFJXOymdQJ+lMdmfuSVJd78Lht+vvdlKIgmFGBfyWkezT\\\\\\\/MjryCZN0fmVZHLnJovw5qr55qOj5gnkAYajOS\\\\\\\/Wt\\\\\\\/ZXXx6VhtZA6jcs7h\\\\\\\/M5xIiDgj+t9nkTm0IuzPTlJBPb3XgZ5iZf6mWQnRZiBK6j7lN9vRFSHq2ROa+0M+zuLRilWMOgkDl5jjSaiwmvngIvp2BQFrmCCrHdELwtgLCUSd80kXtDeGqVTnZ9EwDZY7sFVXVpFE0ddYcyIkNo9F1mjtPHvrBhw0eqT3kU2H+3XKEXqD\\\\\\\/hWMlgcovSn7hBNY\\\\\\\/h9VOOO7rf4EL1VpV24J1JHGzhJ19ywrNxYreoIvCqC7yTuUoGj4TtJ1LV4\\\\\\\/XxP6dTrOUaAMi6H4Or+M6P3YcskTAUHjDXET7F\\\\\\\/A7NyGNGIlxdtxVB9BqgyUQDllfETld3Do4NgR52NCiHGwUCDYxN+sK+8alXhuZvIgQ3+CjMFFnHkP1fZ4U7PNFWSTwjAc0g0+zoI6cJuVLZcioO4NtCYToyoxrGXrFUwUubYQcKghuD3UOIm6s05G0nMinWXnynaHZ3mUuL0VOzDO8TYRdiAWTsBdyv4RSz\\\\\\\/KnoDGpg6iWTRJHNx2rLXkcCl7v4dhVRIXEOl7IWYzsXt\\\\\\\/mZM38NddpOqOe8sMIhP\\\\\\\/rk+Paw96vu+Ze3fudoLGNUs\\\\\\\/ykULZtEKIszE5LNqgZAl32gkq72eKTbvlzkDi2xe9rRqHsKprEr8ti5fotuTnrf7cfO64dEz+9UD6XAzgYCPWCDH2a44zOB0UseDuXT2do6fd4dIQoAynIO8rkVo9QWQH3g9mNXSV+RkrL9zJdx1\\\\\\\/JKhn3zKLjYkrffcPFH7XuTBUdwGASquGn+jg6ZUrgcwBYx10N9rAvusev5jlCQtH8V5pCxCJYS1F0efWLDSjA9xYDiKZyP0xjM+82PKZBMRIjz5LW2+cnHXj3JhaIwJ200izEuGRtwFuXMxTcQOS1eo9pBamZBtH\\\\\\\/ZhneuzHI8kITJRWhDsbqJd9rxy93PLHPDQVwFNFUztgT8znWerIjYqH1enrVIiCHw+T5rQCNmNVUeuANG7ZExfXF7nh\\\\\\\/YemKgoQgJuDloTRgXbCBV1h\\\\\\\/sm8Aw+debx0YGe1VVXCTYJCIVLnThX4U45iYFER0WLd1BH0XXmkRvRCdY0HvvH6A22Jv3hPssRz6IxJ1hwmFNrrI1p1Hv+6\\\\\\\/4928zbK2WgbMwrI84gSSsCV1uRY5ciU53zo+tB51pFtboTadmfKckV0p+MKeufTorQH10++Pe2BcbcVGuLHjv0+25dt2gTIOpRK83\\\\\\\/9WvelcAgw46F8tRXA1DorA0iHNWqcxVrItT5M\\\\\\\/LZaTguef7QlB7WKn5yNLUqhbjvKfc9E9na6Cz9FrcaEr2CoR9IT0OWvZHA+TxsPWOHese8iudVNxWga5XyzcSLMxJL1A5ueCFGwN2ZFmHaz+QUq+jeuQNVMIR7YY1XFaJduiJ5\\\\\\\/cwPbgJpgkd6drg0FohtbbMtp0fYed1eSyx0F9TKmHOkvn9IPGj6HMQpe3Km1hqLgG3AJUmt479jTLhNWvMg7zM3K8Ht3EfVS6G2T0sdDkVfo5Ft0XFj8FRDfGGXAULSqBLhu3BGePaK8lpGwFjuWKRLwOt1+93xPp9BuDfQgrfVG93bHuoaMzAyNZP7IZpn7QmzS0xOb75ZplSj6Td8K60zyg9XBF2pHXaokNOCF\\\\\\\/ZnAEiSG3V6pYHoUGF3anKJhQe5+piK8S2gKIsBY35CFoNpejtB2civv6bbDjM\\\\\\\/VH38HTbL75SmwIc1XbwkEjRTNuSOwrwpY4mWjYNokzRQT2G1rP2grBCWL5BYVhooP2P6YUore+EglpYOXCsTYIVrgxkCWN7UUdUHWJ\\\\\\\/kcpV7Yfb38vg1RxkncVTxKrNU2EwCvLyEa\\\\\\\/BwpIyByK26SLWpZFfuVUuAcO8VxdVbLp+ct\\\\\\\/jkcFm22Ni6\\\\\\\/uKNJKju9PdACNE3yMfJQwZ6QPOsZpr+6ApcBWc1zlMEo1PJ8GbbjG3x8dGH3O+99n5XW+N+2tttQLyF37Z20+GQvNEg\\\\\\\/95uqLhZ2tEwMgk7KCK3p8FeIkMepJnJhVAZSUJIH97Hd\\\\\\\/qjjX68hhKL8Q\\\\\\\/1b1H85TYbcMjw2bwCQ\\\\\\\/uZbz1tYX74TQzkTwcbX+39nlHGIV4ETwBM5MhFD5+wdWBSY3Akgbt\\\\\\\/6QmaEHRKqRkKKASBBX5SER2D9If2\\\\\\\/3CM8w81ckcg9Fak2S1sfFsqGyaMQPf2jodF4wfCxnucm+6ws9\\\\\\\/FLb9ua4SYbJQ67KkaXnLArRvQRsPE6ZyyDFpOrvf6CLIP3YaG7gBWTNJ7b4y\\\\\\\/hb3XuxMdNA9Bo6AyaLBRik6wIIXloHQLwefWlzZb9\\\\\\\/2YeZberIiuMhYaLXIWvAC4TJJcBAZwQnIMHk6CM+YoSG6SQIz8pknHzH3bvkQ3bbU48b9M0PbBOuTeMIBZj4nL2UBZY4GtQH3dMjeI0sD6kZ5oqDz\\\\\\\/HgO2mLTzl4fZC4+vTxMjHLAS2Lg9G2TYj6LE7dlxJmYmCafMc77s01dNNea0JmUQrYLVyNHkH3\\\\\\\/G9ZJCIdkeOpsN1o\\\\\\\/+ObdhU5L9+LgCz7BGeuhItQbgqTsLlS2OnBlHLTTe4ECS0QRWobNly083bvZ1SwOCvUkK+q051mRen7I7ZkAXPa\\\\\\\/MDgZ+8RoDHchviucNbnfq1L\\\\\\\/\\\\\\\/RlLplXxSkS1Uwz9o\\\\\\\/PTasKgKsfS0dtqtJ9ThDzVjfJ2uDj9xoL4wXU4ZwuJi8lEgVk1OCFk1jty8jzj9SukpTXniWcFRof9jgZauCKnvlhlhMYIiqg0VuwuMHxv6DRJLbhtrs4fpDQr4rhzyHcL8mH2JNs7ubAP8P\\\\\\\/kB1vzeIKIicwQ2VEKQoDHxvUmL0maiSu1C1vqwJCeHegT\\\\\\\/hZwtr1i7TrjquzkAAK5+7DZu4R5xtF8q7DdlGPN7g5zbtHTLpbM012vQkhNc1KeLTu\\\\\\\/4KCq+f5o9LXh8PVGNsl7U7Bd05Z9+Fy6U1YjWWhaItwBQ3zYh\\\\\\\/waX4fUtcvRFMSSiD11VYketk86dZN\\\\\\\/swKvg2yy6Aep2RGWqtmjPFhttFFfbbcWRAnPJdDJyly+JM44ejAN\\\\\\\/H2kED5wO0bj2s2oczKBrU3c6vCiBxbaw2h\\\\\\\/TU4XmcZtY0qoGtzgD4XFjSaE75X1YTjkAhQFNDRDBwpPLTWIofqstNm1KUidFwFyuFN+MfItCp4gny4n7oZbzf2Z2fU5znHFBblon0PiitJW8VCLNuaV8zn8x84Qvq19S3hkZdP2XgymS8dDKZL2cQiS3pKCGWlJNFs4jPYxIT9cUW1uiQNEZjULypvA5qlLwJi0VXVybUleyNNLMWFMXnYnqz+hx8S0qcn1q0Ex76YM319IjU15x+1O421zyG3BlrPa70jVtY9Rd\\\\\\\/PDqMsXF0zC2XGyBNVQVKtpZOqJthalXTbGXFLk2470mxAtczfNMEFArT3RC\\\\\\\/XE6qGYZq0RjS6+AM8cjzZM9q63o0TMf69FI3LghWfhL4PevS+Bc4WBqxVEmKJ+u77lc8dBMcUAewTRL2Vvi1a+pt9mwIgYRgvpTVfUitpExlvVVqcUiOJO3YZhhKBadj8ptg4KaAov7RGSL3DdDOZavXSRS3Ck4rKr1wdGEAxbRP6auHIPhhVE0ZLnCAVohZxayXoeu6XesFDCTc1C+BTpiaXV5hzqmL+PKobNp4ZWSqDNX492smSPVTCuHSiMQ+0x2bs5IDigcqbKU\\\\\\\/ZlRwxxuBuhKHgDfz+6GlaJO3mk1dhmQFsK5NF7ADiNqq0HMXmq8Q+Zzf9\\\\\\\/C9y43dZDUxnnVmqVdEW+sg3Xmrfres5ungGv+2Yr+SrDOo9XYur8rCA5q1wo\\\\\\\/\\\\\\\/8OZLjDGZtS6\\\\\\\/G5dgUhYtc6q12kO3UPuMO4MfGfbcH8OlkX\\\\\\\/J1TneQVoYRHFb6upUhDIEhV6IUPgpI\\\\\\\/UbVE2Mbhx7TjlSg3PVeKDeR8WLe2B1Sp64TYvjgdykjOX8v\\\\\\\/Vi8Sw+JFYL4pVhS3Neil+ZFXGcAPloZAxyU9RmWMElvL8ZBLVd3AkBAO9eT7XBoqz9LVteyvCsolyX2345E573gN2JWj7Y6J+hKYqOCJa+m53LckGRgbdT4cqAgBp1TnPMIeGYqiA0s8PQk1P0fAUPAOjiIC0hSm1WS4LWE\\\\\\\/IJBxPnyXhS\\\\\\\/9aNMglSxx6scNddJndcJU34AGMHdGmHGrAnZ\\\\\\\/o5b5njZuZ3XLeB9wLwLLmlL0LphZJXDAyuYx6NaAoVOdc3Htmk6n6HNAfbcduUrc8B6\\\\\\\/2o8znhkkzZkr\\\\\\\/073bXUY6tz\\\\\\\/zVi\\\\\\\/9xe3+kTFYOoa4L2zKgjpPnZBIPuBUIl7U228ubZKY9bvwUyd27+lPJ5tzI9Mhp9OfcZQ8gtSnrkn6SYKzEVac+F4JmbOWZfXKvCjXBZbHhWLtmUO7riVteqqwcluOyJEcmAUtsLWFo5PcYqyu1GgWzYF7ClXxqZE8wbGPOYMQS3IUI\\\\\\\/mv0ktLWTsJ8n7jHu6gx1qdfkzyNKs31qVV9aHp4T1UxNwQI6fuwJFaoMuPV7gd3DHcd65OzvSqoX7VvIZMUgXvdWGTv2IirU1mZoBIkCNnZKdnuaRCM\\\\\\\/5t2LEiVT7sn7BLQNVOlnt8bAVLugTSIVB6XEp0tsXO\\\\\\\/bqP6oUeGZexUKNhe7suaabz5FqeAhSws8rC4X6hAOqUdO4DPWyyJl\\\\\\\/+nGNpWliL7X0dJtaKpC40\\\\\\\/vCwJxpdM12BNoLb1RPKFWOJDfhI\\\\\\\/QsKuPTY77lBvY7a6hP3m+RpEmrMqOb3pnkbONSBFqvUU310iFcUj0qYZWtEeADu8JsYEWUvYaKbIuvy7tcCzKigQp5EIsRK9HSXYqJ08IoBlbHqNonABMRW0P9mG0HPkn9XYVcwa\\\\\\\/YGOJ5vI6ZScT+CZO2+zdL0cVH730rzozlPOM21MENa7Dufowy4iQomWU2SJPbmkAy+RR2A1S0Dxv04QR0SWhXLzD+A7N7LC89ysoWMq4gNTBxETjPLDEr++kkTb2tHl7VFmy2RZ0+tmjqOkkUTlASZ6mi+ulcGlt9LDWNyStJKzoVsdACUmsgQ00tKO7HAPYFAmpmOmxmM1488Ae1mxjtY5mj8w1uoGtVzQP5jYpKFv0QdDX5MVrD79JWKIY3eCweDnP6EhcImwJn\\\\\\\/i8yrm0epk11CSZDx3Q55iBPTZNR+gp0nc\\\\\\\/kV39KZgaWQXpvOGIM8ev9vwU4\\\\\\\/4bunD3thuN18LjPgdN1Jsboo3DRJAPGoFVoCY6u9KhmTqSZR5s+iYa7rZfECKHasPpOhmjS6s\\\\\\\/8yTh9t7T9g\\\\\\\/7pJEBgeRdAyZiBR+YBltEGfMtRZ4jq6ZQnh8jpGfVdXiz4XrOrvLmcx\\\\\\\/BnGq0\\\\\\\/1JmEB0z9Be\\\\\\\/iVFBB4gpMp6tyDutarEiymfA18q8AXCIV0pkeBIzJdoN8NHbSUG1qqtm4RDk2pZQ+RqaNy7e0O2EBiTPu61wjWoPFu+I3ymdE+YLXjwU85OXFu6wmtKdiuTRovC0kMnI350T6kVp77jMRo1QW5\\\\\\\/NDv+xKtou9pzKH11Gjl8nij1394x7GYwLL70oJCEFPpNBqAZ0Mfr2t0CvuwMsxh7Gmb0G+oI7DwRa0ksXdq0\\\\\\\/pzEEoIScIC1ik6osnG+sb+5KhJoWd3MJzKSNxY3CE2vWSMIepOPij0xsP4ORoo+etJs\\\\\\\/QvUISeK7qXe7VQyPc39vpkrXKyIgebvDhc2walyMzEr4oxN8EcG18tHgERqDdUVeZvjucTpTp9YpyJWqa0Z3UfUXmFfX8Y0yw\\\\\\\/lhhBiQeXurokruQLhHHIzXJF17pmEsdlHcnwYUHBCHNNsIBxCYq+TQUgk3aIc++cQ6GuOYqaU2N8r+MW5usVrGwPKbhZbYQAyXC6oyhrhGN0T5ivuWWU6V6n5gaG5GItnMKh2IX25QgGe\\\\\\\/rY9kqWsSSkflDe4Y06h27OvwWGm1FcP57fz+vnHPPP9YmfjTeCCiodOiw7Oi+YX\\\\\\\/ypkMrMNBlV+tYLjvPOplnEBDH8CevZ9+T8mvwDCMXrU9BmDiFH1JUfWIiuA5H8RGa3lJPbGo3p9f2HocLYg+LBw4xe5dEjVbTy3EsbYo7vrEL9i9oCdr6a1wCzYIyMiDOD3C7ueuF4ADwCJRWK+g7gtDv03hj2LMHc7SIwUJOzaBXH8EZBtg7cGVU7B6QrNWYyHBoMKKq58tGNcihwfLMv24kFg1tlZ2V2la7V0CptQBH+ll2T1k5NrRzd\\\\\\\/dDLQmi4UzheGTr36vJ0KMXuindN5lruSjdRps\\\\\\\/YsFa7mwKj2NHGvnf7UKqGTfa1YMBqUykCQnoAqptLnyGOHpsk17o1uMhYHR9sw23l74sH0rrpbUQSbe2YsQloVsAQZy6ACFWBMHnsQ1ZNXtrqhKScoUbehLdbIeI+udouZgtGfbAs71yZn7vwGIOYteOgTQlydmGTbp443PC9ie9K++6nd6Qponn2shJ1Q\\\\\\\/H181dlQlPFbU28+9U0WPJOdUMtQB8L9CYv9CQs80401ZA4qKkdhCZ71wsI2Fpv0MyJY6jGQaIdMLWh24qc8QH8S7JTn0G3zHniO1v35KYu2a0sTpBhuuQ+4N8kF7kulPq6x09CGGJcS97xfEt4Wp1Zke0DmQaGy1t8zg6O4r\\\\\\\/UAv9Dek36Mmpu0HT7QwXWIpt3FGufp15MUXZe6CEbXzCDKrWngpzdv9RG6NYEeaCzgSIdoPJwWnzs1z6zYeuNfBBOvJXYH44Y8Rfke1QaBtFtCS65s364mXWvjST4T3QJcOchFseJjImWk1Nxi45m3H+\\\\\\\/p128iIhUQ27rYJH0nePC3BtlCnUKIFmEj3Sr6Iwyjq1frFpDj\\\\\\\/Zf\\\\\\\/1\\\\\\\/1tWnCeraS6IXuvErr3B2ZxAoM2JveuL9KGHQfMxAYuAOliTOoFvjkOwYVvNrrAjgCtfsqNNMGhMCIlvso6ajwDiBe696HZqGvoIvUzz71MO\\\\\\\/1SdaeeXs+WC7SNwuko9IgPUKcaSemaHykERte\\\\\\\/WYVriPQzJiWg7Bj5Wb1ezUAYdmkg1NeHgClNevkSjVc50XzR9izWEkOD3HdclY2h7bJ\\\\\\\/yke9pDs5x55+HLDiyAD7lDFN1OBqfOTa6ySlVzKwGKMhgcdtX8D4Xf\\\\\\\/kdpiJh8VDAByV+wPtEU5xVcPJVvLl7Mjujld0n9+eNUf\\\\\\\/bx3COVKSno2skkODMy20NwpyGfnN4LumlQdxJcszD8vfnJHmAcpFHuTIPHsc+GjhG3z0VMi7rJNfx\\\\\\\/lrD29Tt0HBfCUlUpLnuaODMPc7xGzujPHf8O2YyLUuYvkJT91HmJ4MA1B\\\\\\\/zkUzzScDGLLR3jkxisfvfV7KYYDAZynIwrMBGicVL0vDVsrzlIE90gsKNpVHdfDcmWGLgyTaP4QbQ\\\\\\\/i+uiLeCJF2PMSL0rHAiucynquL8cacrzzxS\\\\\\\/1ghOtPVOkb1Il\\\\\\\/Z4amq2AluzjBMkhVzSNKLENYSD5inHR2q\\\\\\\/u2Dr2U2GRaPHQ0g8TFhvMFSwIneCLCvjhCci+qax6\\\\\\\/JqA8daCHUUDjWKgruJ\\\\\\\/zYSid6D8RJkQyGhWKYau38PQdB8DRy9rmZYRejqCCWYwhd\\\\\\\/WjVaIHKwc0OgnysIrcNf1RAyQrv1fYWU\\\\\\\/lJM57mSUJYRaoAip9R\\\\\\\/2EDvTviRGzhj2L4FJpY7HAEUloznAADhVkv5jLnVelbFGz1cxH1S6\\\\\\\/WgETNJ9GPPi1a23qoZgmpcgHr2GoGkkctmyLSvqkUPN9fO8HybtQrlSHtEMx4F3n3qp4MKUUCu8axSi1E2TZFninxeqEHj2DaDyWVpFQar5MBGkKH+ZVXEPTwWCHxx05ohzKx\\\\\\\/1v+7SGS5VQURvAU9BA\\\\\\\/HWXNBz76TJBMjZomoL9ocNaHZDNLpVeS4GtYwzEELBdlGGh\\\\\\\/6acjmXYtJ88NE43+a4yHowkftR\\\\\\\/UKqS3eM++NydoN7q57XNE16pnP4e62FILeDRySDeXHR9gurueHDOYj3J44SUtAM1ERit8HIpjPwmPqtg6gATz9IIcMDXFwUV591ipqioZfzLFWPAuzK4vg6CCIaodb6I3q74FL3BszX+EC+dYPqjrrvBgwfnz0RgAZbqlD2pGbZEojTHuZQG8\\\\\\\/Mm+bYEBA+yWtQ7bSze7WrfRbqjEVeFwd3kM4EesEmIm\\\\\\\/Pnpn4zbZ2GXWglfD3eoDhmpd+ig0UPbAe7zNVb4WHs16tXwZvi2cSnNxw8cN2tDg6LEkGPhNv4vULcHC4p+t6a8\\\\\\\/\\\\\\\/NUxr\\\\\\\/5f3+4r0qU89EWo0dWlz6gFVTgPXY5q9Tdpa8Oeqe7SvdN5VwNw54mPetxjst8ofhr94MvyXWhVVEeDyuFoZb7ecKUWlk2sSDGdwQgddgRsOmYaLg6sfpoFlzV9dQXeDJlBvSwcWdIq85RpyoIxTW1lMnAOWJewXNLxshXNR5iY6ZkTdZ18pNC1fj4UDQk0Z7vziHY6WyCKkrExkWnLHQiagL68ubBx86mosLfxJyxxsLUU+V5bkLCiKlMj1g07ns\\\\\\\/ubg48S19qCUQhFEiaHRjweM1TAd9oxjckjiyR3RvQq8oXTo0FSDyR4T0w0R+uBzneFaIIe2vRFrUCYY3ULa2nQf5FfR9273x48Mi49crzbJlnWTk1AZzpbndGy4YLfExYEuZbfYAuPtR2sfnLP9\\\\\\\/kMIpSugjKep3rpoI0H1BISbN7jmkfWSwnMiq1V08GNGrMut49VEyVlkQNjqWhO47T6kNNlZOoyEhamBEs4bQ827gi\\\\\\\/Ob+TuA63opZyUhfEwqonylAhIk1Ct\\\\\\\/+kYp2gZBI\\\\\\\/RgE+RjLooZvHphly0ARx4jkHkwf1fc6i0JIMsN8khtIBDNLlftaxiR7wCXmnoA9uVhBvYShsngMzCdDa\\\\\\\/lqDrcySj4XjlIusiLhcGJJdM8rdOY+hjXvMv3UCC\\\\\\\/D+QkcOWI\\\\\\\/ZWfhAK9PngHouDEFsjqORvWuwtpGGjahu65WABdAYjrEWTnZXTSUgV2DExYFQayxVRhMR4v\\\\\\\/kCnTud3mrLNCLaOYhsfOx8dMR4gnWx42fU7WcQ68SdszNfMi+6sV4JaLinAOBX4aZQi4l\\\\\\\/TolsnJVLHcUJChzPOYhV1plkfGUhf4u+BhXYvj55xHwys2GpJ8OPElgzkeXRyHJi0uCZgZTI4xE\\\\\\\/Fis9j1QPA2w2z5boEU7SnxkpO8k3R+mAsvya9I3DzwVb5a\\\\\\\/W0VLLf3PCcT0adeE0vXt+FZU0neb6FYD0teviIZxj0CNcouXkV6vNNDLGKlSMk1psiftJF+z4QsKMe92XGbRmY0uTlu9f5l3ahXS9jDMsLFpmHLqioL95Cz0IRt8MCY1asoRpsmD7wji3t0o2DCM73Yxx\\\\\\\/XLrI4R0g0hWnV\\\\\\\/Qstn22qxtGdI6k+GRvKR2MmJOFerDDfm0dVvfqcLBmHXhvdolO8so6kGoDwiFFSxqxbpWSgQElvSOWsMz8bJhshwvRLcMm4wXOsbOpqHdLJiY0DKeXdMTQFXaS1PC9GJGf8BOXeuuiWi48ysAqKs2i8GlddaPRw265y823STYtHWABFqTypggJejVAP\\\\\\\/s\\\\\\\/JVKFZbPfTEbOwpD4KYKQRxuflb7qi3iB6UsePY+xdlTLgA86jCkgBivKzHmIDPJ1kgDQds3m\\\\\\\/SkcLJDpUpKDFLFPsrZ6F0jbJbv1a2O\\\\\\\/j08SgpS48ki9SccqoA\\\\\\\/2rePr4fGBGLZISdpaS6U7KcPNilB+fEvM8eobGwZ1JYIvy\\\\\\\/dcw6QVznGc+fmmqdcnr7mIE3zKK5GBMpvtFHJiAI022lFIDylNvSqY4NaPPEVDc4Ru98ggB9xER1ICDKxmR6\\\\\\\/ePksw5uz+SSK5Iy3TXJ3ydovddPilBoCFdaT6DozMnrgRQViPWBnbYj8eRSdGOtsLf7HAivM0GuCSgKyXbtx+TsXQSJYa32qCda\\\\\\\/E37KWc+oS\\\\\\\/KutkWJnB11EsaysYM9hoaRdx3qeUhI\\\\\\\/1rYitThgkPALBEo0e9AoecAJFjfb0bfLujm4E+mWbCf9dJddSN+jRBHZsYv4\\\\\\\/pSLmZ6V8\\\\\\\/G\\\\\\\/2Rsnv83IAv3iomlRPN1KKpqvZwge7gQIJkExrWshJ3hi+l91uEzVfSRZBE2U0Kw6GVjyWlB\\\\\\\/jFmJXf9mHKAm+iVfFU41fqi0Sgz+KiNyW0lbiEAu1AalID8I3gSy8IJW8E2M66Ju4jXdUGHAFa8bX++GjAUpygjnje7w0JFJjRMpej3cQYxSAf54xqz4klJJFN0u0ImlscEAoX8UlUFuymWW6rg85XI2aYAPrtu+MIvacktMYfGk7CpVg8iwIMUW1bmZ6+AH3cnWmBx2rcFkWe\\\\\\\/9Go7VMIpktbrYwIcW+Pyc\\\\\\\/YAp70cyIgqeexfYhjQDYNWueMubH9eithDgRtJp\\\\\\\/3L1eNmHlI5RoILpHae0csER6PwRB7QQXVQlFz4lYDgZjEKQ9ey5gf1M5WbyhNyIXl+85aqsNaEC+cm4pB+0fzNa4lXi8gMIH4xY3fSx6sU78PCqM97nkv\\\\\\\/GSIsnEZrqIasDyB\\\\\\\/pJH3pdNKSJi\\\\\\\/59dsaJMmTftKUmJK3LvVBmOVPwGz8Y6fY3Mvb6i8Fpyz2J1y6mSl+cjYZJHHauqrEw8CeUZCd51l0kVtyZDEkRv32OUgsFr7y1UmWrzPV\\\\\\\/4IHxKvzwciXPhXZI3KPLz5088l96yZBHoSct+Yf+EmVFI1lKAW4utcV24N4hhczWLts50zC0OPNN46KXPILd+sEl5YkqXW5DCjwFH8i\\\\\\\/3egOAHJZ2WsA4Aq2GkbtTI7LYjEKuJDqihFRyD+bf0nB3lq9ICtfAx6kwAheH5PUeruP6KmddVEYR9\\\\\\\/6zUtK6ivdLXshKHpCAhvE+meJgiGuGRQOOIKTzAOmGrTttqR4yrB\\\\\\\/FJBeBl8klKuOylaCgUznHqHPdb1KbN2+NATP6QSvmD1XdILB4Ar2uDS+KKX6\\\\\\\/48ELw2X1mgyuyS8rx15hhyAqJt3UHvzeob6HrrZSNpIy1kYDnUujmHUwUAEqiAGugVmfWBs\\\\\\\/K+yZ4umILidVNAvuO\\\\\\\/ob4mzhKSlxmOc\\\\\\\/yOpbTlrjIwiKTUUQH4E+n4RcuYD0JG6n+VEo0l9XwP2PeF6udQyhnBkikPffNjNUWmbBNa9is4EodwJsms1qP3TY4p+Z4qyOSdRSGK06CEFNV27dgQH3MgCP5Ch4\\\\\\\/\\\\\\\/uenXWBBOT1KjkQI10AQOpvXM8mlukAxYHHo\\\\\\\/7GtWRl\\\\\\\/jPHYWO2jduUJH+foqAFMh7yT0n4SBzOnuEOS6eZCOGFKqI7BBA3DbY4KuXaRYYAYhemybdyBk7jzbBfU8UGJvVP0dwHBeS\\\\\\\/lMGWo8fpP1NQGtrplUNFGuUon0yP2ldP2+T4BqqmsO\\\\\\\/h0xcHHgoyFqwICTbmbKr9HEWxcBO2Epg3hSdbJogqrW+5LiBy8IC1AGVisj3Xn0BAaL6b\\\\\\\/khcFgArd4dlzt350ADlKfrvyuEuSG2GBnkdhKovT24XxPu3\\\\\\\/lxGamCahRvW8Q1CCnS5wtlR4l6Mja1tIT\\\\\\\/js1Xa+MuNIozxMdXibO0dUelSqR36Mt6cOTWg2Il\\\\\\\/jASu3nrbGojAz+ZRmm0pCHj17x0CiMo0bZYlUfosXXa1NM5HmWRu\\\\\\\/cm+tq2twtJTnwqMPSVpzM4dMbOZ+GzwBS4adg\\\\\\\/9aKDPsfefzUr8Y1Ok8dmPg4kUkIDUFdaeow90NMWu+xHLE1FMmsmp53RXEW\\\\\\\/7bGVpfLuhghUaF53VtSKnort52zd1z1FNeTFjhj5x6v45RhfU5EswUV61yv5YiuWzZm4oQ5HnNSCV3QAJhPorSK9TReEE+bNpXk2hHKl16p8V+hTCFz1WfKxKBQ4ZHlhMzTjNTuAmscGNsGHJEgmkFbai2MfALtnWo3ULdRpSJiofaPYzkVUbvYFAyZ61snRJWqwj6isYEVzB0KTpyHsZOG\\\\\\\/8YYxe756PAWjdrNXoBLXs1WeQc72bND4V40pS4L9r7J15n\\\\\\\/SW5i3LC\\\\\\\/\\\\\\\/IB5Jv7xgjzjQ42LAvxIAn+Uh8b1nTLz+xOd7JcJ07dU\\\\\\\/OeKOClM4IWjOvtJpfMEyChpuURHgHa4oBPcJee6IjTy6aDw57XNmv\\\\\\\/dYyrvXcvN0LDfWu8t9IKPqxP75nkba\\\\\\\/4CZDvAiylh+GCC5PMMW9KrmYW9xU8aMRhHkMTYp7jfBtHtPDr71tDYRAj8k8faONaQzIx+K\\\\\\\/uNSiVmE\\\\\\\/161MH+aW4pIib9eJE7L4JDG8G5z6z0InFlMr42Ieyxkwyt3KCbPE\\\\\\\/Hl9jcrYAnBF\\\\\\\/DZ\\\\\\\/VRqDtqogwrYRva4RzPP2fLeILLn3gO1uUAlBKdnWe5I\\\\\\\/JZU9zbnE7EfiGXSria2YJQQOuYjMTA6Zz6lupsOKFqnHlqpmkKbiHQCEU5GYLqUb3LqtCMGpP4KV1eOtyyHBF+4YM77CBO9KLWoEeDnPw7oVYqGDywaLH0n6631PqaJxTrV9A\\\\\\\/\\\\\\\/x59+9Z859m80FUbNc+M6z6BT2nIgNfWrCp9lrDCiWxfybr6+jPR78A7AFfZQ5I7fwCeaP9vel2XWgp762JSeK20tcS3UVoo6yUSWlbd2wTzA6idQNi2F9KWvCsCi6aCX57k3WZ4x\\\\\\\/aWcqqpmlM+RBh4amDYUgleFS1UJgDg9C4JsxfWC2IpMPsw66D3m7XSFvt9wa24Lv72s5xoMaguTrAeM0vVgHA\\\\\\\/bWr37RUszGRSIvR9xifcnKOHA6TJMIQ0cco2vsLh1yIwhzhHQmM6rpzHa3X7Ue7gPiq5NBxOniqloAPO9DH1ycTojmzIVFjdyngi0JBE+Hgq3zDi7k3tVv6HoqVSSE7RnQZM2+02cietZejFkBUtRvpgIiMDQpJ1Ldy9pg\\\\\\\/Qi2pJKlhgSFHLuo3AUS1bJ3FjxRnJtp6g5qntTcCjPrjpQcwtbIvvheQttG5c2nSmapMNr3uDFd47Nz8PI6zTykbvRbo7r06xK8guo5C0fAnh9VUnUloMxAq8SGZtW5gbZcIHTj19n1as\\\\\\\/7oBdkkcZxOkUYWEClorgpQ04Zy6XSoWHmVCAL8ABIb\\\\\\\/OQ+X8liy8hOh9d2q0\\\\\\\/slkk\\\\\\\/phXhSf3jq0AduJRw9btI1QElRorKcLVpohGwYzDLpk0x0ZmIWPe4LHyrrpvSOWGM8Xyf4c25VodrQiBjbHVqs7cWPUCWn8aGDVo7gdpi+NFf+a9p54Y2wcgqEGvVEKW95DxUy1JJMxUy4ZmMQEBDN1HLGI4gUyGC2GGhlXg8jzHHWAPre9Zny9oEBYP3BvGMpISvY4HYhLdTeRApOOswauDAoIEu2lP8LK599VwqKPiA2YQanO\\\\\\\/6X9LQ76ef\\\\\\\/9ztSBf0vXra+CKuA8kcwTymXtsgmF\\\\\\\/lWkBUOxZuJEpPdfaTDPUrwup5ImBDx8Gpl7yEhctUghvJABDQ8DOiK3v96b3WthBzI0HFkD32\\\\\\\/pUJS2v9tX5l33ofChro\\\\\\\/w9vecExew4or5AEgC2jveScRT+iozXV5DiG\\\\\\\/S4ejc2iWdAQXAt+fSIRIa9P2+oRr0K4rMFHJ6B1a0i3cIiAXFnj7BxF0H5xX1MxMQy4dPgfNSaRKExgwDa8I8grfVqZ6DCQxX4wZQWHlFCjuL+zvki+lVsxcJpCTGf5ZlMHl44gNbXbUi8LDyE1iaGg5fzeM5xFntn3zkVY7t9Hw6NBvNd6MGfIbQE0mLMUegTSIOiMLUk42NS8zkzwKlsae+fUS6pZWjPvVdZ0G4nFgevWKUB46KOdMISUMgYYrLuz98R8XyjCeusUhTnr8nZO9\\\\\\\/2BjynO9ICUSCMGax81pTZKomVTSyog08F2HZhRdNA3JZliXaIKPXKSL3tqMPc+IEoTOr3E78dy6NHS6CnJie55fleVHTeOc9qIteoqTdxGU\\\\\\\/jKzvLvlvZRiPRH3uco3xsflHRD0KNRnbTZ1z1hV3UxoFw\\\\\\\/grEX7OWnyvwM0VXufCQQsFLYvpthqnGz8SX+Mc8CcY+7vagYIqEl9dxXQEkPewfrTNCaXH5PyV\\\\\\\/mTuSDlIKMCq4h1Oyx2MmpRnpgNXEfqnt\\\\\\\/zDaXG7jmJgByCVWvidWFA\\\\\\\/l7\\\\\\\/1rWre9vIIviwsXiZ85zEq7iXTCsRaGNHdMfUFqI0l0dIrMkN3dOMAJTEDTfl7RcQDygxsf2EAm5V3k+h2KsPdFn5suCclToqcg83dYJGVrXB1tI2UVaKRXKBY4vXIw6ffkfQ1cl46L9uoVuFFl1uUuOguI3KXCwWBiBKQG5apgaVUnQz9f0\\\\\\\/L03b9xgoacT3FchuSEtiiYCiMLgjoIzYX85xF2KGd2VcQMxGqbDwnH++YPfUv42OmHrqBnPKCEpSdqAQw1nuRSQW7icsklp1PpMha3DR8pPN9GN0\\\\\\\/1vQ45bteL+40\\\\\\\/U6+GqTgb34iZXuivJCOZBNvjKTiwuqcAOfU0OXxQp1nxyLWNfSO\\\\\\\/ukC897VuRdrqPCoynY74Fh2KpKH6bfbbsJy\\\\\\\/KWQpbxbFCOTMdQLOdTarU8mRYtCPKRNu0ctXU8Inj3LytLPDIvnrRQGgIjZcXXTs0A7gqyeW+HzlcThLkFhwctjjQ9+Z+gXteT\\\\\\\/ueXk+RGowEOSb99O9ZDB6Qgvl1EJNmCP1eFmulsMrHpeH5lr2HD3TEWYhCqKr2JWEsYM8ygkwatSYWWc7k\\\\\\\/XgRb37v3ZskkrvG7+G9z2NLfrdXG6pMDr7xCLMzs\\\\\\\/2Z6MS3NEnIom6V1FLGDIJDO1en\\\\\\\/kE5XiM9qknK3DkEieLeqIuQcwt7n6tAi48zH6Ix8Gs9cSdx8YMK7iquG3ju+b5nNSqRYFQCO49cxc5Kw0ajXydb3CL\\\\\\\/8JSsZaYeI\\\\\\\/37MAqoGbivdkpVxsP4ZFr8gRGzGLHA4kbYPX70aQfiyGuPE9FZtIsmI8Yiu7utfbiMLV++0JUD\\\\\\\/p4fylj8rJ1BqFXtVMvAaJwo8YNQalQnMGqrCPJOap8v2Eg46a14KlhpE5CJ2iKWR8RF97ywbv+wb5HwKYSXUhQPEIszRO1IWAdiZqkSILv6MhZXs8RP5iplqP972Te9ooNZe+Xzuv5GNb6RlNKCn0CmZGoIkBnCBYH\\\\\\\/JygMFfE9tVHwCfN1w92pjof6JF22edYnyPhtLYQm++XWkGuB8IKSOFON2wzfoBZImYo6yj60AKw94hshkWN0ieuKVrBJA673ih50jxN3yU6lAvEGc66eoKeGt4RcbHelH97Jz8qF5hcZIUQndKPBSClc1SZ1ANHLP+6XrqltDw6wQctueT3pdDKm8qK0GI2VUJ97HlYcQWL1BuhgPgT0H6txQmMeRfKd4FHXo0KE96sCSPf8I228oSRZUxNFIWAVnZm8mE7kWrTLSkrLOSBl0+4EC9SDUYAdF4sMvv80FMCIp0CG38qmP92TIvy7Eqqk4hqdni5VqPnVl53gSxGToyQKBcbJHRJEkAWvZoU1HNDMI00N9BulOLTpe5cIc0hZWzyDiBdNl9qT1O4VrjFsYCNRVeRJr1HceVa3HS+aJ1lmVlS0rUnYAVT4isly2tlmpmB\\\\\\\/4sDmHwnsfSaydDVsRC9zrwsVuVLlB7U4yeo2JoZIg0yFsihGRQpqHct2QTFhVrRcBSuk\\\\\\\/3F1zel+ydaBVFcoVzoL4PMlhIg7yiJO+Gxt5MG1muiewKTst3+2gtlqcGR3V1hdmF7pPlmKKMSBxJF1\\\\\\\/RDh54y0dCEYtQL6HkADsuC46R3OPtoOu8TlHP3HiSfWU3eAO6YofObtpdBMkqanR\\\\\\\/ZGjCubTt022uzTnoj\\\\\\\/a5svAwZw1lO96qKNhOY3Wpqpor7UBlTG+o\\\\\\\/8ua58p8iQtAqMH1hoVSIMDzCx8zW20DOybq5kGKR40qFkDb3SXF1\\\\\\\/J8AvNCmX33sQKh6O5jdUYwPlw1abD0m1SYjcdk0SD+N8abJpB9G3FAUNN1ZIfHb+C7vOsdG8llUXSRWlHsv87B18ANAV6oljD6jpPLkf5R2sVTSyNxl6wzv6whc1HQW0VrOg30VopMeUap8SV9IYilqm9eMV93wX9R9fT0\\\\\\\/+PDK404MCBz0PXJ2FvPZam3pZhO4W3pXnAFxWcNBAcSzpWQvC2\\\\\\\/663Kf4Qu+cxsEu1QT0TGqxVdgaNSyqxYaV4lLzPdNzmmihNJNLWrJU41bhzohjkszp6nbx4ufHtyg2B0tA05Bt+55\\\\\\\/3hnkJBhLBnBwjJ3+mkSumjptu5VwgdoorYfA0AK5cTzaedqPNUFS1q6CwX1HvhH8Ofenvmxy3ZvfqM1CeZHPOPVfuURp5Z1yN7BbcRcBjoVj8dieAPmlQabUwQah6oAL2BWm8S0BWzHW9\\\\\\\/mgYELlltr7UnK1B2fotqH\\\\\\\/6aWj9LYVVegyOAI9qwLmoTGd+YuCMPgqjalq4EjwvaeL0t3F1bnqYrnLHsInjfa9semR\\\\\\\/KYE6yWUWEsbLAEYMDeXZk\\\\\\\/ICnOeYskhAJW\\\\\\\/xdTNQeMCywQnkWg95p30\\\\\\\/V69qyy3sCja7KH3YqGKmiI27tnJEg6T\\\\\\\/qR6xVWzN0wQYMQghArAY9x2oTYQsPgQIpIRfjMvBMfpDp9HZckWC9PKDLUmmnH3ASkkOBvOSkBZEUtBYtC5eA8eAb+hDwsl1UqLqkz49pB1PyQxz68e1Eu\\\\\\\/qnKS6zs653BVv4tR+KHt\\\\\\\/7YhU7V1Jbaj\\\\\\\/kRupQNks99x15y\\\\\\\/3TCm5y\\\\\\\/tctAJIk2uEv7Od0dQLhArfMYcMWhfVVDPI+ILRYhSE6K6zTbCABd1C9BhqDbh\\\\\\\/hdiNoIzRegVDOIplH1PBnTr1Sg6PCjXtODBdm4fj98EU088Ghu4pVQsjBoxBpt+mu7S9rKzgnZM9fc3piYTsfjf9QVXBB2C7JUGZrjnM9YfAE01Cbo6p8CuPoKxP8tO15G0Gsk7JUXZp9ItCSHOPJAXYyg8MMQyClnguiFUd20BTNZ2r7t3FxG2Sb34d06yCeRUwcVrHIje2xNXUsRZCWnrmpMLW5M3sWUFaLbsXAvEihneNH4OOKIa3N0KdWu5SAXPHLxPCbl77Drtun1ssHl80a9MNLe8nsluag8ackIkzaFuV4iWocY\\\\\\\/s4RdMMbnGkIB\\\\\\\/1AOWQ30LwAEv8dSjTnMOcszv+lEliAGOSWKEsdZu9OoO+y7NK8MgjEtL+6z14Hp7xjt3L\\\\\\\/MGE9RuHoXYKf+akbtuWRc+rXSMxs9FzK0a4Edrb3JQShqxCMoqV+aFrltzg5ufbDUrSOTSd++xlkJMBKlsaGxmIExmdFcDDlMVNsooA\\\\\\\/fCMqqNVe1Dm7tZfove1+tAFZrpSiIOSpd2gsdxtUhjgWsy+gZzDIlhcpQ3UopbhVtzG2Jwb3B\\\\\\\/prL2AJRUOZkVzH1p2CW0boSX3Hb6j0u+ZRZlHeZQJRknvFYsfXQT3IbNEuwLpBO2E3k\\\\\\\/+z4TStxKSUhQALRDxIXeO9lXJ28\\\\\\\/cc\\\\\\\/W0rAqeP3gTeVfO8PhuSIuJDFXdVILiXOGUBGYpl2YPghzqthBU9E4JJeGmB48swvGJ1K8BstYY0n7l1iEiVR8lVBhYaBkh0tYxarnWgwWoQVliltAnbCuNy30kKA4GIvEK+awq2LmlAZxZVu4cngSQ0bZ1sAz1ZSwjEPtJiiMTWxE1k7fLsgW+DD8SS9jA9eVmg0X7PSjGymqcRPcsiLMsMMGJfYXDAuhlszWs6puciHrq8zNwkDQe5sLcU6yWH09IWgjCLBT4GF+ZRqPwf3RvtRRVD0KOr6PK7Kp4N1l5X\\\\\\\/eK++xxe8ip5KBWTdKZPNDL2p1yYseXf9FDR10dU0WOcr\\\\\\\/77r451Tt8nR9NPq0L75zB75cM5XO4f\\\\\\\/8C43GYfGGQdOf7wF690mgPSrhaJBMYAasZFvDr9iYaiS+LcfyqrGqafA0fa5l\\\\\\\/MEmEBu3xMqyzNnmddW2taKBtU1XZuC90p\\\\\\\/BRyxH4S3s0PJF207y5cGLwn\\\\\\\/PnJGIc6zOh1yOg2p0kEn18pxkXV\\\\\\\/FZiDwGtEYZSK36HsCMJz4ChqeJfWiwYhl3qiagQrlUaA4ilvhF6uoY5NZh6JhjY+BA3SQUGUS+Qk6sh8uCTCiteDGB3Mi8ENSedhBE1TLhFEb9tjZV3Ch6a59XgHzlcAHqRCOmAvUhjK7G8Jmpf4GcPfSeA+Hvcbkg3aoAvi456BYimOA9urW7hfAyb8dRwc3\\\\\\\/EPMfzxNYMJnNK6kSsqGKFJf4DqMfdziJQ8bIFZhm0YgCB38qPfI41frvnOiiKwtB5QYDW5Rv7sV9em1juWzTM4Veoz3I1XfxwLAWRm+F35H2IMFmDEklgZWiGgRhRAKMySyTV6cUz3pp5YlldyWdQGDQ7x5uO1zevuXSn6D63lSK3JSVdOUBfeAA0\\\u0022,\\\u0022iv\\\u0022:\\\u00227t3WcAUPyZFxI9hesOJCQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e2296af8c7c0c1e4ad619651e33a8bafadb453a2d1ffb0fe4c0008e3809c5e7d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222EjfD0Cj+\\\\\\\/oF7Xn9y8y0dwzNQS7nsiALQDDFv0SgZP5WCqPQRczSXYXJ6NgMzFI8+Z19I1hgSvYGR8\\\\\\\/SVLAcXL6yow\\\\\\\/K6TeP2EmNqXLZWc6ub0CVMTsjWx3OyYYMP10ThnMxwk486PcGix38Vv+PUHyEAEc\\\\\\\/cZyVzgl3OVu698Eon6KA0GSv68mfgK\\\\\\\/XJeDFetZeU13tUaHPGee2TEUWYARuUIS+0QbIjsAVNdn5r6ULQr0UC1qi7UO3kDGXja6Nzhj4HNj\\\\\\\/VvczT+MWhhL18QAXc8AI5nztD0\\\\\\\/KBEQtlR6KRkpdrIH1tqBDX9+XK8bPOnasNgSPDAjFA2+bQ9BZsr9qpUfUHxO9Vh6nKwEi177lyQpfSHikn1Z\\\\\\\/LIUCVkbUlQ4mCADoHNd3yvmDrw21anMVAwHT+gN0IK698vTHX5h1pTJeI0yLcsBGqRpS+JnxYS\\\\\\\/L1g8uqAGR2fdEpZR\\\\\\\/4HzN89Ol3h2VqVacAJwdxFhWQtSXuoztg0T+lq6rWGFCuE3d6To7Nqpp1f00JmkbG0+yVYGtBZqSI\\\\\\\/cS4RkTLQqZVzDs+U+K3RwtkS7xdXemyv\\\\\\\/skngrOR9xP7FB7n64SWECB3f3ICnLwp+lq9y6Qfo5TMWuTBCpybjOZfDUT+ILMBVyUd8jxaa\\\\\\\/a407EL9DovGeUcBG01n8Oh5QNDZrvx0gEVfKOaHZUlyP6wg3KWIc3ONd0v8oVcUaTshBXuuDGT4Y3LCBTU8tNy2Gqebia23BvfkZnVje5P3BIECW+yZVfVsODirFnCh5bDRUCTsnNSNm9smMfhsuEi4u7nwrPxeRCibwM7Djc1cVkVwUGkkBVKAG3NvA4Bx2V+kBcKi7GO3Y1P7Ix\\\\\\\/Gj5Tf0p0KHbmkVRjGXNkpQYf2S+iH8V5INgRkwZxBp+W8dGmCgoQUQhkk5TfGqVoAntTky\\\\\\\/9O8lP3JfjrNzWobSBkVkWcBE6FGI90KRAMrXQV7qaVzT1ag+G7JKU7g+3yoOEJEMzSzTJT3fKRlAgd+j4qNNxf5\\\\\\\/hNHZCR3sGkSFSrzB06\\\\\\\/FAlupGuXYhDjeUKYHPVV9am+w0dSXR2KtCl5XCqQLWxWLeTyJiqLERc+PCZnrqpTO5o48vcNyGwQLjXBqBuwr6P4mOCL5tdItCQWu4fcxAWZE85eL1G9d8xJCy71nlTk82zQO+arMvjA49BQPoPAE5aWDm86S\\\\\\\/gu6yUN2WTsHdpcULdhH0EekqLMthYUuN\\\\\\\/SVuAlW7yjoZ2hMPOoTHRaMNDNmTHxs2XmgXOhxbsUDeCv948jlR6Y0bdeZ2+QaDE5PMAlM6XZVCDF4Vl\\\\\\\/IfJJmeFc\\\\\\\/NWqxYZfh8n9Gmf94jfKwDeCCwX06Y+z9uQci5HvoWLevKzKF9eT36aMDzxDdVOOWaUQ67CNhXqRLdlVrlFxGBNVUP53Lff7klPu3RgxLhaRK8DbBBHZ6YGp6YJOarAdoQakqpOgtD08nzKjAv\\\\\\\/ATOlT\\\\\\\/1jtyuQvNet1hFkZAQdVDOAzDIj4wohEju8Vhlai8xkJqkevwJngr5x\\\\\\\/at7Kr6grKQHBw4wrqP8S\\\\\\\/iMSqSd\\\\\\\/evd0sttzXQ1YbZkqdBadQnPYuqUcUBvPxAjxyJcCSD+UlFFyHiw5wlAgJ5AOgkyKlSnKmKPs7712AP0izzlXq7QqfagydptUN7C8KtTe4twm3ZE2UAts2Hu2nVnIKwKTp7ySKKFPkQEf8uk3CxAia371x335jZLKXlKUj5RVd3Fy4P3T6AV+U3B5wvf86doYfNShsWSciWuglUcVC4M6aQ\\\\\\\/\\\\\\\/9QqsQanDiCdPasJYEaMnIvKFmP5KBi5QgN7Gif4D2RF6HP0+JTQT7nJLZbFXt6Rc++OWr6RmollRXQzplrx9+XrJYARkEC8ddQlDLDgTbXWtFSzuZ7VWuXNpKep4nAGgdikAIh2tALb2nQrx2U4abz5WrBpwjcsa0aq2RQ3ZsGyv5XkqVyjZkK6NS0RtsdcnF9RNmKUkxdfMsVhlNEEHP7EzOAkBA1TxVrfvGFbs25eEaFBTF8nunsV3\\\\\\\/2q7d4g=\\\u0022,\\\u0022iv\\\u0022:\\\u00224Z8E9B22\\\\\\\/tZAm\\\\\\\/+kR6cAwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222ceaaaf8e87aeabeb4ac205bfffd2e5f0927a82cdea13c533d50354b343ae058\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495284 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/211" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rbi88KLLnVT1125cm7L0jWT6uJ6pQoesypU6BcA7SWk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/211" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-36yU7IvhdxvvMwA3rU15hbODsKHQijqhs8pYKubnh3o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/199" title="The NALCN Channel Regulator UNC-80 Functions in a Subset of Interneurons To Regulate Caenorhabditis elegans Reversal Behavior" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/225" title="A Single, Shared Triploidy in Three Species of Parasitic Nematodes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495284" id="node495284" data-pisa="ggg;10/1/211" data-pisa-master="ggg;g3.119.400854" data-apath="/ggg/10/1/211.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inference of Historical Population-Size Changes with Allele-Frequency Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael Lynch</span>, <span class="highwire-citation-author" data-delta="1">Bernhard Haubold</span>, <span class="highwire-citation-author" data-delta="2">Peter Pfaffelhuber</span> and <span class="highwire-citation-author" data-delta="3">Takahiro Maruki</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">211-223;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400854 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495284" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mlynch11@asu.edu" class="" data-icon-position="" data-hide-link-title="0">mlynch11@asu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bernhard Haubold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön 24306, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BHaubold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haubold%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernhard%2BHaubold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Pfaffelhuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Mathematics and Physics, University of Freiburg, Freiburg 79104, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BPfaffelhuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfaffelhuber%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BPfaffelhuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Takahiro Maruki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiro%2BMaruki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maruki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATakahiro%2BMaruki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/211" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495284" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495284/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/211.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495284" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495284/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/211.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495284" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495284/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/211.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495284" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495284/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">With up to millions of nearly neutral polymorphisms now being routinely sampled in population-genomic surveys, it is possible to estimate the site-frequency spectrum of such sites with high precision. Each frequency class reflects a mixture of potentially unique demographic histories, which can be revealed using theory for the probability distributions of the starting and ending points of branch segments over all possible coalescence trees. Such distributions are completely independent of past population history, which only influences the segment lengths, providing the basis for estimating average population sizes separating tree-wide coalescence events. The history of population-size change experienced by a sample of polymorphisms can then be dissected in a model-flexible fashion, and extension of this theory allows estimation of the mean and full distribution of long-term effective population sizes and ages of alleles of specific frequencies. Here, we outline the basic theory underlying the conceptual approach, develop and test an efficient statistical procedure for parameter estimation, and apply this to multiple population-genomic datasets for the microcrustacean <em>Daphnia pulex</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/coalescent" class="hw-term hw-article-keyword hw-article-keyword-coalescent" rel="nofollow">coalescent</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/daphnia" class="hw-term hw-article-keyword hw-article-keyword-daphnia" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Daphnia</em></a></li><li class="kwd"><a href="/keyword/demographic-history" class="hw-term hw-article-keyword hw-article-keyword-demographic-history" rel="nofollow">demographic history</a></li><li class="kwd"><a href="/keyword/effective-population-size" class="hw-term hw-article-keyword hw-article-keyword-effective-population-size" rel="nofollow">effective population size</a></li><li class="kwd"><a href="/keyword/site-frequency-spectrum-0" class="hw-term hw-article-keyword hw-article-keyword-site-frequency-spectrum" rel="nofollow">site-frequency spectrum</a></li></ul><p id="p-3">Because polymorphisms with different allele frequencies arise at different average times in the past, information on the amount of variation associated with different allele-frequency classes in a population sample can provide insight into the history of population-size change. This is especially true for neutral variants, whose temporal dynamics depend only on stochastic sampling effects. This simple idea has led to the development of several technical and computationally demanding approaches for estimating historical changes in the sizes of populations, either from patterns of segregating variation at the single-nucleotide level or from information on linkage disequilibrium between nucleotide sites (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Strimmer and Pybus 2001</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hayes <em>et al.</em> 2003</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Tenesa <em>et al.</em> 2007</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li and Durbin 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bhaskar <em>et al.</em> 2015</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Liu and Fu 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gattepaille <em>et al.</em> 2016</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Weissman and Hallatschek 2017</a>). All of these methods make numerous assumptions, some of which can be difficult to validate (<em>e.g.</em>, the negligible influence of nonneutral sites), are almost certainly violated (<em>e.g.</em>, linearity of the relationship between the recombination rate and physical distance betwen sites), and/or require information that is not available for most species (<em>e.g.</em>, the identification of derived <em>vs.</em> ancestral alleles). Moreover, it remains to be seen whether simpler, more intuitive approaches might yield results that perform to a comparable (or even greater) degree of accuracy.</p><p id="p-4">The approach taken here is conceptually straight-forward, the main biological assumptions being that the sites underlying the analysis have evolved in a neutral fashion for a considerable number of generations (roughly speaking, for at least four times the current effective population size, which is the expected coalescence time to common ancestry under current conditions), and that there be no substantial population structure. All aspects of the analysis are based on samples of the site-frequency spectrum (SFS) for such sites. Letting <em>n</em> be the number of sampled haploid genomes (typically twice the number of individuals in a sample from a diploid population), the number of polymorphic genomic sites with <em>r</em> copies of the derived allele is denoted <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-1.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-2.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-3.gif"/></noscript></span></span> The number of monomorphic sites is <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-4.gif"/></noscript></span></span>, and the SFS is defined as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-5.gif"/></noscript></span></span> where <em>G</em> is the total number of monomorphic and polymorphic sites evaluated across the genome. In the following, a mutation in class <em>r</em> will be referred to as <em>r</em>th order, with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-6.gif"/></noscript></span></span> denoting singletons, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-7.gif"/></noscript></span></span> doubletons, etc.</p><p id="p-5">The methods that follow, which adhere to the theory utilized in the stairway method of <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a>, are based on three principles. First, the frequencies of sites residing within different classes are functions of the historical pattern of effective population size (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-8.gif"/></noscript></span></span>) – all other things being equal, increases in <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-9.gif"/></noscript></span></span> elevate the probability of an allele residing in a particular polymorphic state, but the relative frequency also depends on the sequence of <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-10.gif"/></noscript></span></span> experienced by all other allelic classes. Second, the SFS for neutral sites scales with the mutation rate per site per generation (<em>u</em>) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kimura 1983</a>), so if quantitative information on <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-11.gif"/></noscript></span></span> is desired, an estimate of <em>u</em> is required. Third, the frequency of a mutation provides information on its age – under the process of neutral drift, the time for a new mutation to reach a frequency class is a monotonic function of the frequency, although there is considerable noise around the expectation.</p><p id="p-6">The goal here is to use these principles to determine the long-term series of effective population sizes most compatible with the SFS, and given the measures of interval-specific <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-12.gif"/></noscript></span></span> to estimate the temporal history of past population-size changes experienced by segregating polymorphisms. We present analytical solutions for a broad set of genealogical features of a sample that are independent of the demographic history, and use this theory to develop estimators for the average age of single-nucleotide polymorphisms within each frequency class and the average <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-13.gif"/></noscript></span></span> experienced during their history. These results are worked out for the case of the unfolded SFS, and extended to the folded SFS, which summarizes the incidence of the minor-allele frequency classes, as investigators only rarely know which allele segregating at a locus is derived. A computationally efficient method for estimating such parameters is presented, validated with comparisons to computer-simulated data, and applied to large population-genomic data sets of <em>Daphnia pulex</em>.</p><div class="section other1" id="sec-1"><h2 class="">Theory</h2><p id="p-7">The <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kingman (1982)</a> coalescent provides the theoretical basis for all that follows. Under this view, members of a genealogy of <em>n</em> samples (assumed to be <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-14.gif"/></noscript></span></span>) randomly coalesce each generation until the entire genealogy has congealed to one common ancestor at the base of the tree after the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-15.gif"/></noscript></span></span>th coalescence event. Although the number of possible tree topologies is enormous with large sample sizes, many of the summary features of the coalescent are known (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hein <em>et al.</em> 2005</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wakeley 2009</a>).</p><p id="p-8">Here, we are concerned with the average features of alleles within different frequency classes (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-16.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-17.gif"/></noscript></span></span>) in the sample, which requires an understanding of the nature of the branch segments on which mutations of the different classes can reside. These probabilistic features can be summarized with a knowledge of <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-18.gif"/></noscript></span></span>, the probability that an allele (SNP, or single nucleotide polymorphism) with frequency <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-19.gif"/></noscript></span></span> resides on an uninterrupted branch starting at level <em>k</em> and ending at level <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-20.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-21.gif"/></noscript></span></span> denotes the branch tips and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-22.gif"/></noscript></span></span> denotes the base of the tree (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For any class of mutations, the underlying branch segments can start as early as level <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-23.gif"/></noscript></span></span> (singleton branches always start at level <em>n</em>) and can end as deeply as level 1. This means that internal branches starting at level <em>k</em> can span up to <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-24.gif"/></noscript></span></span> possible coalescence events in the tree. Each coalescence event can potentially be associated with a unique effective population size.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/211/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A genealogy comprising a sample size of  with the k values denoting the ordered coalescence events ( representing the root of the tree). The external branches, which can contain only singleton mutations, are given as solid black lines. There are three internal branches of order  (upon which doubleton mutations must reside, denoted with large dashes); the red, green, and purple doubleton branches have starting and ending levels of (7,5), (6,5), and (4,2), respectively. The tree also contains single branches of orders  (tripletons, medium dashes), 4 (quadrupletons, dot/dashes), and 5 (quintupletons, small dashes). The consecutive coalescence times are denoted by , with  denoting the effective population size from level k to ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121834100" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;A genealogy comprising a sample size of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-25&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with the &lt;em&gt;k&lt;/em&gt; values denoting the ordered coalescence events (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-26&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; representing the root of the tree). The external branches, which can contain only singleton mutations, are given as solid black lines. There are three internal branches of order &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (upon which doubleton mutations must reside, denoted with large dashes); the red, green, and purple doubleton branches have starting and ending levels of (7,5), (6,5), and (4,2), respectively. The tree also contains single branches of orders &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (tripletons, medium dashes), 4 (quadrupletons, dot/dashes), and 5 (quintupletons, small dashes). The consecutive coalescence times are denoted by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-29&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, with &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-30&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; denoting the effective population size from level &lt;em&gt;k&lt;/em&gt; to &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/211/F1.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/211/F1.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/211/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/211/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495362" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">A genealogy comprising a sample size of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-25.gif"/></noscript></span></span> with the <em>k</em> values denoting the ordered coalescence events (<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-26.gif"/></noscript></span></span> representing the root of the tree). The external branches, which can contain only singleton mutations, are given as solid black lines. There are three internal branches of order <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-27.gif"/></noscript></span></span> (upon which doubleton mutations must reside, denoted with large dashes); the red, green, and purple doubleton branches have starting and ending levels of (7,5), (6,5), and (4,2), respectively. The tree also contains single branches of orders <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-28.gif"/></noscript></span></span> (tripletons, medium dashes), 4 (quadrupletons, dot/dashes), and 5 (quintupletons, small dashes). The consecutive coalescence times are denoted by <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-29.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-30.gif"/></noscript></span></span> denoting the effective population size from level <em>k</em> to <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/F1/embed/mml-math-31.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">A key point is that the <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-32.gif"/></noscript></span></span> coefficients are completely independent of the underlying demographic history, as the coordinates are simply denoted by the enumerated coalescence events, and are functions of only the sample size (<em>n</em>) and the allele class (<em>r</em>). Only the branch lengths are functions of the population size. As described below, the full set of coefficients (derived in the Appendix) provide the basis for analytical expressions for various useful statistical features of SNPs. Here, we adhere to the infinite-sites model, so that each new mutation arising on a genealogy is assumed to appear at a unique site, with the mutation rate to a novel SNP being equal to <em>u</em> per site per generation.</p><div id="sec-2" class="subsection"><h3>An interval-specific view of <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-33.gif"/></noscript></span></span></h3><p id="p-11">We start with the assumption that <em>n</em> sequences have been sampled randomly (from <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-34.gif"/></noscript></span></span> diploid individuals, or <em>n</em> haploids) at each of <em>L</em> nucleotide sites known to behave in an effectively neutral manner. Under neutrality, for a population with constant effective size, the expected number of sites occupying frequency class <em>r</em> in an unfolded site-frequency spectrum is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-36.gif"/></noscript></span></span> (for diploids, assumed here, or <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-37.gif"/></noscript></span></span> for haploids), with <em>u</em> being the mutation rate per nucleotide site per generation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Watterson 1975</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fu 1995</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ewens 2004</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Messer 2009</a>). Here, we take a more refined approach by explicitly evaluating the way in which each SNP class reflects the historical series of coalescence events within a sample, averaging over all possible coalescent topologies for a sample of size <em>n</em>. Looking into the past, a tree composed of <em>n</em> sequences (branch tips) experiences <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-38.gif"/></noscript></span></span> coalescence events to the base, and to each level <em>k</em> we assign an effective population size <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-39.gif"/></noscript></span></span>, such that <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-40.gif"/></noscript></span></span> denotes the average effective size between the branch tips and the first coalescence event, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-41.gif"/></noscript></span></span> represents the average in the interval between the first and second coalescence events, and so on. At level <em>k</em>, the expected time (in generations) to the next coalescence event into the past is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>generations. For example, starting with <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-43.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-44.gif"/></noscript></span></span> denotes the expected number of generations until the first coalescence event in the sample; and the second coalescence event, which depends on <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-45.gif"/></noscript></span></span> is obtained by setting <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-46.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-47.gif"/></noscript></span></span></p><p id="p-12">Given knowledge of the expected internal features of the coalescent, for each SNP frequency class, the expected value of <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-48.gif"/></noscript></span></span> can be expressed as a function of the full set of relevant <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-49.gif"/></noscript></span></span> which determine the lengths of branches upon which mutations arise. This also requires expressions for the expected number of branches of order <em>r</em> at each relevant level of the coalescent, averaged over all possible random genealogies in a sample of size <em>n</em>. These are derived in the Appendix.</p><p id="p-13">As an example, consider the simplest case of the singleton class. All singleton mutations must be present on external branches, which always start at level <em>n</em> but may end at any level in the genealogy from <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-50.gif"/></noscript></span></span> (the first coalescent) to 1 (extending to the base of the tree). The expected number of singletons in the sample is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-51.gif"/></noscript></span><span class="disp-formula-label">(3a)</span></span>where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-52.gif"/></noscript></span></span> is the probability of a singleton branch (starting at level <em>n</em>) not having coalesced by level <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-53.gif"/></noscript></span></span>. This expression is equal to the sum of the product of the expected number of singleton branches surviving at each level and the length of the subsequent coalescence interval, all multiplied by the probability of a mutation arising per site per generation. Using the expression for <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-54.gif"/></noscript></span></span>, Equation (A3) in the Appendix, the preceding expression simplifies to<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-55.gif"/></noscript></span><span class="disp-formula-label">(3b)</span></span>where <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-56.gif"/></noscript></span></span> is the arithmetic average of the interval-specific <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-57.gif"/></noscript></span></span> from the top (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-58.gif"/></noscript></span></span>) to the bottom (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-59.gif"/></noscript></span></span>) levels of the tree. This result applies regardless of the mode of population-size change, showing that an estimate of the arithmetic average <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-60.gif"/></noscript></span></span> across all coalescence events is provided by the incidence of singletons, <em>i.e.</em>, as <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-61.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-62.gif"/></noscript></span></span> denotes an estimate of the number of singletons.</p><p id="p-14">Things are more complicated for the higher-order site-frequency classes because internal branches no longer initiate at the same levels. However, by extension from Equation (3b), one can infer that the probability of a mutation arising on a single branch starting at level <em>k</em>, allowing for variable ending points, is <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-63.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-64.gif"/></noscript></span></span> is the arithmetic average of the interval-specific <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-65.gif"/></noscript></span></span> starting at level <em>k</em> and descending down to the base of the tree. From Equation (A7), the expected number of order-<em>r</em> branches starting at level <em>k</em> is <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-66.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-67.gif"/></noscript></span></span> is a coefficient defining the expected number of segments of order <em>r</em> present at level <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-68.gif"/></noscript></span></span> given by Equation (A5). Summing these contributions over all levels,<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(4a)</span></span>which can also be written as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-70.gif"/></noscript></span><span class="disp-formula-label">(4b)</span></span>also obtained by <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a>. These expressions show that the expected frequencies of all mutation classes are defined by differentially weighted averages of the interval-specific <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-71.gif"/></noscript></span></span>. When <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-72.gif"/></noscript></span></span> Equation (4b) yields (3b), and with constant <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-73.gif"/></noscript></span></span>, it reduces to <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-74.gif"/></noscript></span></span> in accordance with Equation (1); considerable simplification is also possible if many adjacent <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-75.gif"/></noscript></span></span> have the same values (see Supplemental Material).</p><p id="p-15">Before proceeding, recall that there are two forms of a site-frequency spectrum. The unfolded distribution, described above, requires information on the ancestral allelic states of each SNP site, ideally inferred from at least two suitably distant outgroup species (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Keightley and Jackson 2018</a>). Such a distribution is a summary of all sites having derived-allele frequencies <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-76.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-77.gif"/></noscript></span></span> If ancestral allelic states are unknown, as is often the case, one must work with the folded site-frequency spectrum, which summarizes the minor-allele frequencies in classes <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-78.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-79.gif"/></noscript></span></span>. The folded site-frequency spectrum, with <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-80.gif"/></noscript></span></span> is defined as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-81.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>with <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-82.gif"/></noscript></span></span> if <em>n</em> is even.</p></div><div id="sec-3" class="subsection"><h3>Average age of a SNP</h3><p id="p-16">Whereas the previous results are concerned with the demographic history of the population, an alternative viewpoint considers the average ages and demographic features of SNPs of various frequencies. Once the interval-specific estimates of <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-83.gif"/></noscript></span></span> are available, the statistical machinery developed in the Appendix can be used to infer both order-specific measures. There, we show that for an unfolded SFS the average age (in generations) of an <em>r</em>th-order SNP in terms of the historical effective population sizes is<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(6a)</span></span>The expected second moment is expressed as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-85.gif"/></noscript></span><span class="disp-formula-label">(6b)</span></span>so the variance of ages of SNPs can be obtained as <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-86.gif"/></noscript></span></span>, after substituting in the estimates for the <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-87.gif"/></noscript></span></span></p><p id="p-17">Although the preceding expressions apply to an unfolded SFS, where the designated alleles are known to be derived (by use of appropriate outgroup species for identifying ancestral allelic states), studies without such a luxury must rely on a folded SFS. In this case, each frequency class will be a mixture of derived and ancestral alleles with different average ages. For low-frequency alleles in large samples, almost the entire set of sampled SNPs will consist of derived alleles, and the preceding expressions can still be used to obtain reasonably precise estimates. This follows from Equation (1), which shows that the expected frequency of SNPs of order <em>i</em> is inversely proportional to <em>i</em>. Thus, for <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-88.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-89.gif"/></noscript></span></span>, the fractional contribution of the former to the folded distribution is of order <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-90.gif"/></noscript></span></span> provided the <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-91.gif"/></noscript></span></span> associated with the two classes are not greatly different (and larger than this if <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-92.gif"/></noscript></span></span> is larger for the younger alleles). For <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-93.gif"/></noscript></span></span> almost all of the SNPs within folded class <em>r</em> will be derived alleles.</p><p id="p-18">A more precise approach is to explicitly treat the frequencies of the folded distribution as mixtures of classes of derived alleles of order <em>r</em> and ancestral alleles of order <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-94.gif"/></noscript></span></span>, with respective relative probabilities <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-95.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-96.gif"/></noscript></span></span>. The expected age of a SNP of order <em>r</em> in a folded SFS can then be written as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-97.gif"/></noscript></span><span class="disp-formula-label">(7a)</span></span>where <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-98.gif"/></noscript></span></span> The components of <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-99.gif"/></noscript></span></span> can be estimated by substitution of the estimates for the <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-100.gif"/></noscript></span></span> into Equation (4b), and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-102.gif"/></noscript></span></span>, and their expected squared values, are estimated by use of Equations (6a,b). The variance of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-103.gif"/></noscript></span></span> is then</p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-104.gif"/></noscript></span><span class="disp-formula-label">(7b)</span></span></div><div id="sec-4" class="subsection"><h3>Average <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-105.gif"/></noscript></span></span> of a SNP</h3><p id="p-19">For a population experiencing temporal changes in size, alleles of different order will generally experience different long-term effective population sizes from birth to the present. Letting the population size at time <em>s</em> in the past be <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-106.gif"/></noscript></span></span>, the expected average population size experienced by an allele of frequency <em>r</em> is<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-107.gif"/></noscript></span></span>If one has information on the ancestral states of alleles, and hence an unfolded site-frequency spectrum, the mean <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-108.gif"/></noscript></span></span> experienced by an allele of order <em>r</em> can be obtained from the theoretical results on the mean time spent in different intervals. Weighting of the interval-specific durations by their associated <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-109.gif"/></noscript></span></span> values leads to</p><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-110.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span><p id="p-20">see Appendix. With a folded site-frequency spectrum, the weighting approach used for the age of an allele in the preceding section can be applied using the definitions of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-112.gif"/></noscript></span></span>, as well as <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-113.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-114.gif"/></noscript></span></span> as defined in Equation (8). Nonetheless, with large sample sizes, the proposed approach is still expected to yield fairly accurate information on the average <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-115.gif"/></noscript></span></span> of rare alleles. This again follows from Equation (1), which shows that the expected frequency of SNPs of order <em>i</em> is inversely proportional to <em>i</em>.</p></div></div><div class="section other2" id="sec-5"><h2 class="">Estimation procedure</h2><p id="p-21">The results summarized in Equation (4b) amount to a series of <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-116.gif"/></noscript></span></span> equations, each a function of the mutation rate, <em>u</em>, and one or more of the interval-specific effective population sizes, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-117.gif"/></noscript></span></span>. Thus, in principle, working backward, one could apply the elements of the observed SFS to Equation (4b) to recursively estimate the full set of <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-118.gif"/></noscript></span></span> necessary to account for the data, <em>i.e.</em>, solving a set of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-119.gif"/></noscript></span></span> equations for <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-120.gif"/></noscript></span></span> unknowns. However, with large numbers of unknowns and imperfectly estimated <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-121.gif"/></noscript></span></span>, such an approach leads to aberrant results, including negative population-size estimates. Moreover, in the case of a folded site-frequency spectrum, the number of possible unknown population sizes exceeds the number of observed frequency classes.</p><p id="p-22">It then becomes necessary to pool adjacent population sizes so as to reduce the number of parameters to be estimated. Consistent with <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a>, we have adopted a stepwise procedure, implemented in a likelihood framework. Consider a sampled site-frequency spectrum given by <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-122.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-123.gif"/></noscript></span></span> is the number of singletons, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-124.gif"/></noscript></span></span> the number of doubletons, etc. With <em>L</em> sampled sites, the number of monomorphic sites is <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-125.gif"/></noscript></span></span>. For any set of interval-specific population sizes, Equations (4b,5) give the expected frequencies of SNPs in the full set of classes. Using a composite-likelihood approach, <em>i.e.</em>, assuming that the elements of the sampled SFS are all essentially independent and Poisson distributed with parameters equal to the frequency expectations times <em>L</em>, the likelihood function is given in Supplemental Material.</p><p id="p-23">We have implemented the above procedure in the program epos (Estimating POpulation Sizes), which runs under the UNIX command line. The C sources of epos and tutorial-style documentation are available from github at <a href="https://github.com/EvolBioInf/epos">https://github.com/EvolBioInf/epos</a>. The starting point of the estimation procedure assumes a constant population size throughout the entire history of the sample. The maximum-likelihood estimator of <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-126.gif"/></noscript></span></span> is then equivalent to <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Watterson’s (1975)</a> estimator. The next most complicated model involves a single coalescent breakpoint <em>k</em> flanked by two different <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-127.gif"/></noscript></span></span>, such that <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-128.gif"/></noscript></span></span> is a constant <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-129.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-130.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-131.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-132.gif"/></noscript></span></span>. The formulae for the expected SFS then reduce to a three-parameter model, whose solution requires a search for the combination of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-133.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-134.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-135.gif"/></noscript></span></span> that maximizes the composite likelihood of the observed SFS, which can be found by Newton-Raphson iteration. This procedure is then iterated in a stepwise fashion, with each iteration increasing the number of breakpoints by one, until the difference in adjacent likelihoods no longer improves beyond a critical value. To this end, we employ the Akaike Information Criterion (AIC), moving on to the next iteration provided the log-likelihood has increased by at least 2.0 in the preceding iteration. The end result is a stepwise plot of interval-specific <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-136.gif"/></noscript></span></span> estimates, with the breakpoints converted to time (in generations) using the interval-specific expected coalescent times given by Equation (2) and a mutation rate provided by the user.</p><p id="p-24">Several additional features are built into epos. First, it is possible to analyze folded as well as unfolded SFSs. Second, the auxiliary program bootSfs (<a href="http://github.com/EvolBioInf/bootSfs">github.com/EvolBioInf/bootSfs</a>) implements the bootstrap to estimate the sampling variance of the estimated demographic history. Third, it is possible to exclude classes from the SFS, if for example the singleton class is deemed unreliable owing to sequencing errors. Fourth, the user can specify the AIC stopping criterion. Fifth, all possible combinations of breakpoint locations can be re-evaluated at each iteration, as opposed to sequentially adding fixed breakpoints; to accomplish this, there are two versions of the function nextConfig in epos: a fast, greedy version, which adds one new level at a time and a slow, exhaustive version, which goes through all possible combinations of levels. This flexibility is provided because the the number of possible sets of breakpoints increases exponentially as the stepwise estimation procedure advances.</p><p id="p-25">Based on the performance of the Stairway Plot algorithm of <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a>, as applied to a single human (Yoruba) population sample, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lapierre <em>et al.</em> (2017)</a> have raised concerns about the use of model-flexible approaches to estimating historical demography, as opposed to using model-constrained approaches that pre-specify the form of population growth and breakpoints in demographic features. In this particular application, these authors showed that the Stairway Plot algorithm predicts a complex demographic history with multiple recent bottlenecks, with a poor least-squares fit to the observed SFS (with a weighted mean-squared distance of <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-137.gif"/></noscript></span></span>). In contrast, simpler pre-specified models (<em>e.g.</em>, linear, exponential, and sudden) predicted consistent increases in population size to the present (all with <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-138.gif"/></noscript></span></span> in the range of <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-139.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-140.gif"/></noscript></span></span>). Application of epos to the same data set predicts an increase in population size from the deep past to the present, but with a short intervening population bottleneck <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-141.gif"/></noscript></span></span> years ago (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), and has a fivefold reduction of <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-142.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-143.gif"/></noscript></span></span>. The current <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-144.gif"/></noscript></span></span> predicted by epos is comparable to that obtained by other methods. Thus, contrary to the conclusions of <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Lapierre <em>et al.</em> (2017)</a>, these results suggest that constrained models are not inherently superior to flexible models, but simply that the quality of the results obtained in the latter context can be suboptimal if the algorithmic approach of <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a> is applied.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/211/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results from the application of epos to the Yoruba SNP data set applied in Lapierre et al. (2017). The Yoruba data set was bootstrapped 10,000 times (average given by solid line) and the 5 and 95% quantiles (lower and upper dashed lines, respectively) computed from these replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121834100" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results from the application of epos to the Yoruba SNP data set applied in Lapierre &lt;em&gt;et al.&lt;/em&gt; (2017). The Yoruba data set was bootstrapped 10,000 times (average given by solid line) and the 5 and 95% quantiles (lower and upper dashed lines, respectively) computed from these replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/211/F2.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/211/F2.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/211/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/211/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495409" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Results from the application of epos to the Yoruba SNP data set applied in <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Lapierre <em>et al.</em> (2017)</a>. The Yoruba data set was bootstrapped 10,000 times (average given by solid line) and the 5 and 95% quantiles (lower and upper dashed lines, respectively) computed from these replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We have further evaluated the utility of epos by fitting <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-145.gif"/></noscript></span></span> histories to various demographic scenarios by generating sample SFSs using the coalescent software of <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kelleher <em>et al.</em> (2016)</a> and <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> (2009)</a> in the analyses in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3A-E</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">3F</a>, respectively. Comparison of our results to those obtained by the algorithms of <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a> shows that epos performs as well and in some cases better than the Stairway method (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). For each evaluated scenario, ten SFSs were generated, and 2,000 bootstrap replications were used to find the mean and percentiles of the effective-population size estimates, except under the last scenario (F), where 200 bootstrap replications were used. Epos is at least 1,000 times faster than <a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Liu and Fu’s (2015)</a> Stairway procedure. For example, epos and Stairway Plot v2 took 0:00:14 and 6:11:51, respectively, to analyze one site-frequency spectrum under the scenario in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3a</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/211/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the estimation quality of epos (blue) to results from the Stairway Plot (SP, red) method of Liu and Fu (2015). In each panel, the means of 10 medians and 5 and 95% quantiles are shown (dotted and dashed lines, based on averages of 10 independently derived SFSs). The black lines denote the true underlying demography used to simulate the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121834100" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the estimation quality of epos (blue) to results from the Stairway Plot (SP, red) method of Liu and Fu (2015). In each panel, the means of 10 medians and 5 and 95% quantiles are shown (dotted and dashed lines, based on averages of 10 independently derived SFSs). The black lines denote the true underlying demography used to simulate the data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/211/F3.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/211/F3.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/211/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/211/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495348" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Comparison of the estimation quality of epos (blue) to results from the Stairway Plot (SP, red) method of <a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a>. In each panel, the means of 10 medians and 5 and 95% quantiles are shown (dotted and dashed lines, based on averages of 10 independently derived SFSs). The black lines denote the true underlying demography used to simulate the data.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section other3" id="sec-6"><h2 class="">Application to Daphnia population-genomic data</h2><p id="p-29">We applied epos to the SFSs from eight <em>Daphnia pulex</em> populations, an emerging model system in population genomics. A practical issue in any population-genomic study with moderate sequence coverage per site is that not all sites are scored in identical numbers of individuals. In this particular study, for each population 8 to 14 SFSs were available for sample sizes of 40,000 to 2,400,000 nucleotides (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), with separate analyses performed for fourfold redundant silent sites in protein-coding genes and internal intron sites known to behave in a nearly neutral fashion (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lynch <em>et al.</em> 2017</a>). Although the individuals used within these within-population analyses were largely overlapping, the sites employed were fully nonoverlapping. The numbers of individuals associated with each SFS range from 62 to 93.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495377/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495377/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The site-frequency spectra analyzed in this study. Details on data acquisition can be found in Maruki <em>et al.</em> (2019) For each population, 8 to 14 SFSs were used, with a range of numbers of sampled individuals and nucleotides as noted in the text</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">This type of partitioning is required because the SFS theory involves discrete distributions, <em>i.e.</em>, frequencies from different sample sizes should not be amalgamated into a single pooled SFS. However, such replication in analysis also provides some guard against sampling variance issues. For each of the samples, 10,000 bootstraps of the SFS were performed to generate a median demographic-history estimate, assuming a mutation rate of <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-146.gif"/></noscript></span></span> per site per generation (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Keith <em>et al.</em> 2016</a>). The final demographic-history estimates for each population are then given as the means of separate median estimates (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), and a further summary mean over all populations is given in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/211/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated demographic history of eight Daphnia pulex populations, measured as the means of the medians (thick black lines) of the replicated 10,000 bootstrap estimates derived for the number of samples noted in Table 1. Deviations of single standard errors of the means are given as red and blue dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121834100" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimated demographic history of eight &lt;em&gt;Daphnia pulex&lt;/em&gt; populations, measured as the means of the medians (thick black lines) of the replicated 10,000 bootstrap estimates derived for the number of samples noted in Table 1. Deviations of single standard errors of the means are given as red and blue dashed lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/211/F4.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/211/F4.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/211/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/211/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495229" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Estimated demographic history of eight <em>Daphnia pulex</em> populations, measured as the means of the medians (thick black lines) of the replicated 10,000 bootstrap estimates derived for the number of samples noted in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Deviations of single standard errors of the means are given as red and blue dashed lines.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/211/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average pattern of demographic history over the entire D. pulex metapopulation, plotted on the linear and logarithmic time scale, with deviations from the mean given as single standard errors among the profiles for each of the eight populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121834100" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Average pattern of demographic history over the entire &lt;em&gt;D. pulex&lt;/em&gt; metapopulation, plotted on the linear and logarithmic time scale, with deviations from the mean given as single standard errors among the profiles for each of the eight populations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/211/F5.medium.gif" width="224" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/1/211/F5.medium.gif" width="224" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/211/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/211/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Average pattern of demographic history over the entire <em>D. pulex</em> metapopulation, plotted on the linear and logarithmic time scale, with deviations from the mean given as single standard errors among the profiles for each of the eight populations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Although there are significant differences among populations, these analyses suggest a fairly consistent demographic history among all populations (focusing on an order-of-magnitude time scale). From <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-147.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-148.gif"/></noscript></span></span> generations in the past, <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-149.gif"/></noscript></span></span> was almost always in the range of 0.5 to <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-150.gif"/></noscript></span></span>, with little evidence of dramatic changes. All populations exhibit evidence of a twofold or so decline in <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-151.gif"/></noscript></span></span> in the very recent past, followed by an interval of population-size expansion around 20,000 generations ago (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Assuming five to ten generations per year, these recent demographic shifts would represent post-Pleistocene changes, with the point of initiation of population-size expansion being 2,000 to 4,000 years ago (roughly corresponding to the Neopluvial, a period of wetter and cooler climate in North America). The ending points in the demographic profiles (<span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-152.gif"/></noscript></span></span> million years ago) fall in the mid-Pleistocene. Influences from European settlement, deforestation, and agriculture would date no further back than 5,000 generations, and are not discernible.</p><p id="p-34">Finally, the relationships between the mean <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-153.gif"/></noscript></span></span> of SNPs and their average age is given for each population in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The left panel provides an example of the variation among sample-size classes for one particular population (with each point representing a particular SFS class for a particular number of individuals scored). The right panel summarizes the average results for each population as simple first- or second-order polynomial least-squares regressions. The main point again is that these <em>Daphnia</em> populations do not exhibit major demographic shifts across allele-frequency classes, with the population average <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-154.gif"/></noscript></span></span> associated with SNPs of all ages almost always falling in the range of <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-155.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-156.gif"/></noscript></span></span></p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/211/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Left) The average effective population sizes and ages associated with each site-frequency class of alleles for the KAP population of D. pulex, with open symbols for internal intron sites (as defined in Lynch et al. 2017) and closed symbols for fourfold redundant sites. Sample sizes ranging from 72 to 78 diploid individuals have associated numbers of scored sites given in the inset for internal intron sites and fourfold redundant sites, respectively. Right) Summary of the results for each population, simply given as best second-order polynomial fits to the full sets of data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121834100" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Left) The average effective population sizes and ages associated with each site-frequency class of alleles for the KAP population of &lt;em&gt;D. pulex&lt;/em&gt;, with open symbols for internal intron sites (as defined in Lynch &lt;em&gt;et al.&lt;/em&gt; 2017) and closed symbols for fourfold redundant sites. Sample sizes ranging from 72 to 78 diploid individuals have associated numbers of scored sites given in the inset for internal intron sites and fourfold redundant sites, respectively. Right) Summary of the results for each population, simply given as best second-order polynomial fits to the full sets of data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/211/F6.medium.gif" width="440" height="211"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/1/211/F6.medium.gif" width="440" height="211"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/211/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/211/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495391" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">Left) The average effective population sizes and ages associated with each site-frequency class of alleles for the KAP population of <em>D. pulex</em>, with open symbols for internal intron sites (as defined in <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lynch <em>et al.</em> 2017</a>) and closed symbols for fourfold redundant sites. Sample sizes ranging from 72 to 78 diploid individuals have associated numbers of scored sites given in the inset for internal intron sites and fourfold redundant sites, respectively. Right) Summary of the results for each population, simply given as best second-order polynomial fits to the full sets of data.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-36">The details on data acquisition, curation, and deposition appear in <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Maruki and Lynch (2019)</a>; the FASTQ files of the raw-sequence data are publicly available via the NCBI Sequence Read Archive (accession number SRP155055). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10265867">https://doi.org/10.25387/g3.10265867</a>.</p></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-37">The methods developed herein provide a model-independent means for estimating the past demographic history of a sample, using information on the frequency distribution of nucleotide sites assumed to behave in a neutral to nearly neutral manner. The approach taken assumes that changes in population size occur only at specific points in a genealogy, <em>i.e.</em>, at the times of average occurrence of coalesence events. This, of course, will never happen precisely in any natural population. However, as the times of coalescent events vary widely among genealogies, such granularity can be expected to average out. Moreover, the approach taken does provide an increasingly fine dissection of the overall time scale under evaluation with increasing sample size (<em>n</em>). So as shown by <a id="xref-ref-20-10" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a> and herein, the method has the potential to closely approximate the more continuous patterns of population-size changes that likely occur in nature.</p><p id="p-38">In the proposed method of estimation, Epos simply starts with an assumption of constant <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-157.gif"/></noscript></span></span>, and then progressively searches for points of change in <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-158.gif"/></noscript></span></span> that, when invoked, yield significant improvements in the likelihood of the observed SFS data in a stepwise manner. Application of the bootstrap yields a further smoothing of the output estimates as well as confidence intervals on the overall pattern. Further smoothing is obtained by partitioning the SFS data into classes differing in sample sizes (or from different classes of sites, such as fourfold redundant codon sites <em>vs.</em> internal intron sites, both of which behave in a nearly neutral fashion). SFS sample-size variation will generally be the rule in low-coverage population-genomic sequencing data, where some individuals will have inadequate sequence at random sites.</p><p id="p-39">The theoretical basis of the methods described herein is the same as that adopted by <a id="xref-ref-20-11" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a>, although we have derived a number of extensions. In addition, the estimation procedures embodied in Epos deliver advances over the pioneering work of <a id="xref-ref-20-12" class="xref-bibr" href="#ref-20">Liu and Fu (2015)</a> in a number of ways. First, by using a Newton method for maximizing the likelihood rather than a (slow) genetic algorithm for optimization, the overall algorithmic approach is considerably more efficient, improving computational speed by over an order of magnitude without sacrificing accuracy in estimation (and in some cases apparently improving it). Second, Epos is capable of an exhaustive search for the best-fit demographic scenario, up to a number of steps specified by the user. Under this exhaustive search model, in adding a new breakpoint to the demography, each step in the iterative fitting re-evaluates the positions of all preceding breakpoints and their flanking <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-159.gif"/></noscript></span></span> estimates. Third, epos returns estimates on the average ages and <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-160.gif"/></noscript></span></span> (and sampling errors) of alleles within different frequency classes.</p><p id="p-40">One potential concern with our method is its reliance on a composite-likelihood approach that ignores the nonindependence of linked SNPs. There are two reasons to believe that this is a minor issue with respect to the final analyses. First, most organisms have ten or more chromosomes, so only a minor fraction of pairs of loci are even on the same chromosome, and even a smaller fraction are within the <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-161.gif"/></noscript></span></span> bp where linkage disequilibrium is likely to be significant. Second, our simulation studies on algorithm performance, which generated data based on a recombining chromosome and then applied the composite likelihood, did indeed yield results consistent with simulated demographies. Although desirable, full-likelihood methods allowing for linked loci would be enormously computationally demanding, but more importantly would require detailed information on chromosomal map structures, which are available for few species.</p><p id="p-41">Like all polymorphism-based methods, our approach is expected to become increasingly unreliable at very distant times in the past, owing to the increasing granularity of the coalescent process, and the fact that few polymorphisms are expected to survive for more than <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-162.gif"/></noscript></span></span> generations. In addition, the ability to estimate very recent population-size changes is a function of the sample size, as there can be no power to estimate a span of time during which there is essentially zero chance of a <em>de novo</em> mutation appearing in a sample.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">This work was supported by NIH grants R01-GM101672 and R35-GM122566-01 and NSF grant DEB-1257806 to ML. We thank Yun-Xin Fu for helpful discussion.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX</h2><p id="p-42">Results at various intermediate steps had to be derived before arriving at the main results of the text, some scattered in the prior literature (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Janson and Kersting 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dahmer and Kersting 2015</a>). Here we summarize things in one place to make the overall approach more transparent.</p><div class="section" id="sec-9"><h3>Probability distribution for external branches.</h3><p id="p-43">By definition, singleton mutations exist only on external branches, which always start at level <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-163.gif"/></noscript></span></span> in the coalescent and can end at levels <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-164.gif"/></noscript></span></span> to 1, where <em>n</em> is the sample size. These locations denote the consecutive coalescence events across the tree (with <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-165.gif"/></noscript></span></span> denoting the base of the genealogy). We wish to determine the probability distribution of branch lengths in units of coalescence events across the tree, with <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-166.gif"/></noscript></span></span> denoting the probability that an external branch (starting at level <em>n</em>) ends at level <em>k</em>. This can be accomplished by letting <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-167.gif"/></noscript></span></span> denote the probability that a singleton branch descends to at least level <em>k</em> without having been absorbed by a coalescence event. By definition, <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-168.gif"/></noscript></span></span>, and<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-169.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span><span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-170.gif"/></noscript></span></span> is the fraction of singleton branches extending down the tree until at least level <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-171.gif"/></noscript></span></span> whereas <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-172.gif"/></noscript></span></span> denotes the fraction of all singleton branches terminating at level <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-173.gif"/></noscript></span></span></p><p id="p-44">To obtain the probability of a particular external branch surviving until the first coalescence event across the tree, note that two draws must be made without replacement from the <em>n</em> initial branch tips. The probability that neither draw involves the focal branch tip is<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-174.gif"/></noscript></span><span class="disp-formula-label">(A2a)</span></span>The probability of the focal branch tip continuing to survive the next coalescence event (<em>i.e.</em>, not being drawn) is obtained by noting that there are now <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-175.gif"/></noscript></span></span> possible draws,<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-176.gif"/></noscript></span><span class="disp-formula-label">(A2b)</span></span>which generalizes to,<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-177.gif"/></noscript></span><span class="disp-formula-label">(A3)</span></span>Substituting into Equation (A1) yields the general expression for a singleton branch terminating at the <em>x</em>th coalescence event in the tree,</p><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-178.gif"/></noscript></span><span class="disp-formula-label">(A4)</span></span></div><div class="section" id="sec-10"><h3>Probability distribution for internal branches</h3><p id="p-45">The situation is more complicated with internal branches, which have variable starting and ending points, and also vary in number among alternative tree topologies. However, progress is made possible with a result from <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Dahmer and Kersting (2015)</a>, which states that for a sample of size <em>n</em> the expected number of segments of order <em>r</em> present at level <em>k</em> (just below the coalescence at this point) is<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-179.gif"/></noscript></span><span class="disp-formula-label">(A5)</span></span>for <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-180.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-181.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-182.gif"/></noscript></span></span>. This result can also be obtained by extrapolating Equation (14) in <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Fu (1995)</a>. Again note that <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-183.gif"/></noscript></span></span> denotes the branch-tip level, <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-184.gif"/></noscript></span></span> denotes the first coalescence event in the sample, and <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-185.gif"/></noscript></span></span> denotes the base of the genealogy (the final coalescent), so that starting points (nearer the branch tips) have higher integer values than ending points.</p><p id="p-46">From this expression, it is possible to deconvolute the expected number of internal branches initiating at level <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-186.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-187.gif"/></noscript></span></span>, by noting the birth-death process involving segments of order <em>r</em> as one descends down the tree,<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-188.gif"/></noscript></span><span class="disp-formula-label">(A6a)</span></span>where<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-189.gif"/></noscript></span><span class="disp-formula-label">(A6b)</span></span>is the probability that a particular group at level <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-190.gif"/></noscript></span></span> is not a participant in the next coalescence event. Upon rearrangement, and substitution from above,<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-191.gif"/></noscript></span><span class="disp-formula-label">(A7)</span></span>The highest level for a nonzero value of this birth rate is <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-192.gif"/></noscript></span></span>, and thereafter <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-193.gif"/></noscript></span></span>, except for the case of <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-194.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-195.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-196.gif"/></noscript></span></span> as the first coalescence event always produces a doubleton.</p><p id="p-47">From the statistical properties of coalescence events arising subsequent to the origin of a segment noted above, the expected number of internal branches of order <em>r</em> initiating at level <em>k</em> and ending at level <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-197.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-198.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-199.gif"/></noscript></span><span class="disp-formula-label">(A8)</span></span>which follows directly from Equation (A4).</p><p id="p-48">The complete probability distribution for segment spans of order <em>r</em> is then<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-200.gif"/></noscript></span><span class="disp-formula-label">(A9a)</span></span>where the denominator is the expected total number of branches of order <em>r</em> in a genealogy<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-201.gif"/></noscript></span><span class="disp-formula-label">(A9b)</span></span>None of the features in this entire section on the genealogical structure of a sample depend on demographic history, although the lengths of the individual branches do.</p></div><div class="section" id="sec-11"><h3>Average age of a SNP</h3><p id="p-49"><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Griffiths (2003)</a> obtained a general expression for the average age of a derived allele of arbitrary frequency under the assumption of constant population size, given in generations,<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-202.gif"/></noscript></span><span class="disp-formula-label">(A10)</span></span>However, here we are concerned with the more complex issue of estimating the average age of SNPs when the population size is not constant. The central challenge is that mutations of various orders can appear on branches that start and end at various levels in the tree, each of which may be associated with a particular <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-203.gif"/></noscript></span></span></p><p id="p-50">Here, we take advantage of a derivation of Griffiths and Taveré (1998), their Equation (5.4), which requires a definition of <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-204.gif"/></noscript></span></span>, the probability that a random line, at the time there are <em>k</em> total lines in the coalescent, is subtended by <em>r</em> leaves in the tree. This is equivalent to <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-205.gif"/></noscript></span></span> with simplification of Equation (A5) leading to<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-206.gif"/></noscript></span><span class="disp-formula-label">(A11)</span></span>Letting <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-207.gif"/></noscript></span></span> be the time the coalescent has <em>k</em> lines (defined by Equation (2) in the main text), and <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-208.gif"/></noscript></span></span>, then as <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Griffiths and Tavare (1998)</a> argue above their Equation (5.1), the expected age of an allele arising on a branch when the coalescent has <em>k</em> lines, is <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-209.gif"/></noscript></span></span>, where U is a uniformly distributed on <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-210.gif"/></noscript></span></span> independent of all other random variables.</p><p id="p-51">To obtain the moments of the ages, we take advantage of a derivation of Griffiths and Taveré (1998), their Equation (5.4),<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-211.gif"/></noscript></span><span class="disp-formula-label">(A12)</span></span>The denominator, which is independent of <em>j</em>, reduces to<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-212.gif"/></noscript></span><span class="disp-formula-label">(A13)</span></span>For the numerator, we just provide the results necessary for the first two moments, which are required for estimates of the mean and variance of the average age. For <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-213.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-214.gif"/></noscript></span></span>which leads to<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-215.gif"/></noscript></span><span class="disp-formula-label">(A14)</span></span>For <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-216.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-217.gif"/></noscript></span></span>and therefore<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-218.gif"/></noscript></span></span>which leads to<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-219.gif"/></noscript></span><span class="disp-formula-label">(A15)</span></span>Substitution of Equations (A13-15) into Equation (A12) leads to the expressions for the mean and variance in the main text, Equations (6a,b).</p></div><div class="section" id="sec-12"><h3>Average <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-220.gif"/></noscript></span></span> of a SNP</h3><p id="p-52">The results from the previous section summarize the average amounts of times that alleles spend at the various population sizes. For SNPs of any order <em>r</em>, the average <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-221.gif"/></noscript></span></span> experienced can be determined by weighting the relative time spent in each interval by the interval-specific population sizes. From Griffiths and Taveré (1998), their Equation (5.1), we know that (6a) is in fact equivalent to<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-222.gif"/></noscript></span></span>where <em>U</em> is uniformly distributed on <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-223.gif"/></noscript></span></span>; see also the explanation below (A11). This formula can be interpreted such that the <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-224.gif"/></noscript></span></span>-term is proportional to the probability that the SNP of size <em>r</em> occurs within level <em>k</em>. The expectation in the numerator then gives the time how long in the past this level happened. For estimating the average <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-225.gif"/></noscript></span></span> of a SNPs in size <em>r</em>, we need to weigh this expectation by the experienced population sizes, leading us to<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-226.gif"/></noscript></span></span>For the denominator, note that<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-227.gif"/></noscript></span></span>So, we compute<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-228.gif"/></noscript></span></span>The numerator is actually almost the same except for a different weight of the <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-229.gif"/></noscript></span></span>’s, <em>i.e.</em><span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/211/embed/mml-math-230.gif"/></noscript></span></span>Dividing the last two displays then gives Equation (8).</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-54">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10265867">https://doi.org/10.25387/g3.10265867</a>.</p></li><li class="fn" id="fn-2"><p id="p-55"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-20"><span class="received-label">Received </span>May 20, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-04"><span class="accepted-label">Accepted </span>November 4, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Lynch <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.1" data-doi="10.1101/gr.178756.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhaskar</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. X.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient inference of population size histories and locus-specific mutation rates from large-sample genomic variation data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.178756.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.178756.114%26rft_id%253Dinfo%253Apmid%252F25564017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzIvMjY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.211.2" data-doi="10.1007/978-0-387-21822-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewens</span>,  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Mathematical population genetics</span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin, Germany</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-21822-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bpopulation%2Bgenetics%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-21822-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-21822-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.3" data-doi="10.1101/gr.083634.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marjoram</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Wall</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and flexible simulation of DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.083634.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083634.108%26rft_id%253Dinfo%253Apmid%252F19029539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.4" data-doi="10.1214/14-AAP1024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dahmer</span>,  <span class="cit-name-given-names">I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kersting</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The internal branch lengths of the Kingman coalescent.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Probab.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1325</span>–<span class="cit-lpage">1348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/14-AAP1024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BProbab.%26rft.volume%253D25%26rft.spage%253D1325%26rft_id%253Dinfo%253Adoi%252F10.1214%252F14-AAP1024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/14-AAP1024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.5" data-doi="10.1006/tpbi.1995.1025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">Y.-X.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Statistical properties of segregating sites.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/tpbi.1995.1025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D172%26rft.epage%253D197%26rft.atitle%253DStatistical%2Bproperties%2Bof%2Bsegregating%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1995.1025%26rft_id%253Dinfo%253Apmid%252F7482370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1995.1025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7482370&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RZ41300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.6" data-doi="10.1534/genetics.115.185058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gattepaille</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Günther</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jakobsson</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inferring past effective population size from distributions of coalescent times.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">1191</span>–<span class="cit-lpage">1206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.185058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185058%26rft_id%253Dinfo%253Apmid%252F27638421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC8zLzExOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzIxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.7" data-doi="10.1016/S0040-5809(03)00075-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The frequency spectrum of a mutation, and its age, in a general diffusion model.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0040-5809(03)00075-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DGriffiths%26rft.auinit1%253DR.%2BC.%26rft.volume%253D64%26rft.issue%253D2%26rft.spage%253D241%26rft.epage%253D251%26rft.atitle%253DThe%2Bfrequency%2Bspectrum%2Bof%2Ba%2Bmutation%252C%2Band%2Bits%2Bage%252C%2Bin%2Ba%2Bgeneral%2Bdiffusion%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0040-5809%252803%252900075-3%26rft_id%253Dinfo%253Apmid%252F12948685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0040-5809(03)00075-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12948685&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185265500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.8" data-doi="10.1080/15326349808807471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>,  <span class="cit-name-given-names">R. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tavaré</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The age of a mutation in a general coalescent tree.</span> <abbr class="cit-jnl-abbrev">Commun. Stat. Stoch. Models</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15326349808807471</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BStat.%2BStoch.%2BModels%26rft.volume%253D14%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15326349808807471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15326349808807471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.9" data-doi="10.1371/journal.pgen.1000695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>,  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Hernandez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Williamson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Bustamante</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000695</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000695%26rft.epage%253De1000695%26rft.atitle%253DInferring%2Bthe%2Bjoint%2Bdemographic%2Bhistory%2Bof%2Bmultiple%2Bpopulations%2Bfrom%2Bmultidimensional%2BSNP%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000695%26rft_id%253Dinfo%253Apmid%252F19851460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851460&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.10" data-doi="10.1101/gr.387103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. C.</span> <span class="cit-name-surname">McPartlan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Novel multilocus measure of linkage disequilibrium to estimate past effective population size.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.387103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.387103%26rft_id%253Dinfo%253Apmid%252F12654718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzQvNjM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.211.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hein</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Schierup</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wiuf</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Gene Genealogies</em>, <em>Variation and Evolution: a Primer in Coalescent Theory</em></span>, <span class="cit-publ-name">Oxford Univ. Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-10.1.211.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janson</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kersting</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> The external lengths in Kingman’s coalescent. arXiv: 1004.5011.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.13" data-doi="10.1534/genetics.118.301120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>,  <span class="cit-name-given-names">P. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Jackson</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Inferring the probability of the derived <em>vs.</em> the ancestral allelic state at a polymorphic site.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301120%26rft_id%253Dinfo%253Apmid%252F29769282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA5LzMvODk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.14" data-doi="10.1101/gr.191338.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keith</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Tucker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Jackson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Sung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. I.</span> <span class="cit-name-surname">Lucas Lledó</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High mutational rates of large-scale duplication and deletion in <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">69</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.191338.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.191338.115%26rft_id%253Dinfo%253Apmid%252F26518480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjI2LzEvNjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzIxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.15" data-doi="10.1371/journal.pcbi.1004842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Etheridge</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McVean</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient coalescent simulation and genealogical analysis for large sample sizes.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1004842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004842</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D12%26rft.spage%253De1004842%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004842%26rft_id%253Dinfo%253Apmid%252F27145223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27145223&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.211.16" data-doi="10.1017/CBO9780511623486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source">The Neutral Theory of Molecular Evolution</span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/CBO9780511623486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNeutral%2BTheory%2Bof%2BMolecular%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1017%252FCBO9780511623486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/CBO9780511623486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.17" data-doi="10.2307/3213548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingman</span>,  <span class="cit-name-given-names">J. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">On the genealogy of large populations.</span> <abbr class="cit-jnl-abbrev">J. Appl. Probab.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/3213548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BProbab.%26rft.volume%253D19%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3213548%26rft_id%253Dinfo%253Apmid%252F11338422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3213548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11338422&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.18" data-doi="10.1534/genetics.116.192708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lapierre</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lambert</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Achaz</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accuracy of demographic inferences from the site frequency spectrum: the case of the Yoruba population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.192708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.192708%26rft_id%253Dinfo%253Apmid%252F28341655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzEvNDM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.19" data-doi="10.1038/nature10231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inference of human population history from individual whole-genome sequences.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10231</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D475%26rft.issue%253D7357%26rft.spage%253D493%26rft.epage%253D496%26rft.atitle%253DInference%2Bof%2Bhuman%2Bpopulation%2Bhistory%2Bfrom%2Bindividual%2Bwhole-genome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10231%26rft_id%253Dinfo%253Apmid%252F21753753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21753753&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293167900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.20" data-doi="10.1038/ng.3254"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-X.</span> <span class="cit-name-surname">Fu</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exploring population size changes using SNP frequency spectra.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">559</span>. Erratum: 1099.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3254</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D555%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3254%26rft_id%253Dinfo%253Apmid%252F25848749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25848749&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.21" data-doi="10.1534/genetics.116.190611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ackerman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Spitze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ye</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Maruki</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Population genomics of <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.190611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.190611%26rft_id%253Dinfo%253Apmid%252F27932545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzEvMzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-web" id="cit-10.1.211.22"><div class="cit-metadata unstructured">Maruki, T., and M. Lynch, 2019 Genomic analyses of population structure reveal metabolism as a primary driver of local adaptation in <em>Daphnia pulex</em>. biorxiv/2019/807123.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.23" data-doi="10.1534/genetics.109.105692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>,  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Measuring the rates of spontaneous mutation from deep and large-scale polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.105692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105692%26rft_id%253Dinfo%253Apmid%252F19528323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEyMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzIxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.24" data-doi="10.1093/oxfordjournals.molbev.a003776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. G.</span> <span class="cit-name-surname">Pybus</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Exploring the demographic history of DNA sequences using the generalized skyline plot.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2298</span>–<span class="cit-lpage">2305</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a003776</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DStrimmer%26rft.auinit1%253DK.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D2298%26rft.epage%253D2305%26rft.atitle%253DExploring%2Bthe%2BDemographic%2BHistory%2Bof%2BDNA%2BSequences%2BUsing%2Bthe%2BGeneralized%2BSkyline%2BPlot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003776%26rft_id%253Dinfo%253Apmid%252F11719579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11719579&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172657900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.25" data-doi="10.1101/gr.6023607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tenesa</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Navarro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Duffy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Clarke</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recent human effective population size estimated from linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">526</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.6023607</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6023607%26rft_id%253Dinfo%253Apmid%252F17351134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNTIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yMTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.211.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Coalescent Theory</span>, <span class="cit-publ-name">Roberts and Company Publishers</span>, <span class="cit-publ-loc">Greenwood Village, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.27" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>,  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(75)90020-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F211.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.211.28" data-doi="10.7554/eLife.24836"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>,  <span class="cit-name-given-names">D. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Hallatschek</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Minimal-assumption inference from population-genomic data.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>: e24836.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.24836</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253De24836%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.24836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.24836&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/211.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/199" title="The NALCN Channel Regulator UNC-80 Functions in a Subset of Interneurons To Regulate Caenorhabditis elegans Reversal Behavior" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/225" title="A Single, Shared Triploidy in Three Species of Parasitic Nematodes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/211.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2GnpmXzytiq5z7m1vdH5DMw7pSwMIQl9JS32iSOCiys" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/211" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/211" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/211" class="active" data-icon-position="" data-hide-link-title="0">Inference of Historical Population-Size Changes with Allele-Frequency Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495284" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-pRHiNPixjrX31g_eJHj-wFSwYv_y3JDCjRw_sCvnvkU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/211.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/211" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-uHkYqJPuRBwOewYnR_anwAEMNSC0i8dMTcAvvm_cZIA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/211" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/211.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495284" id="node495284--2" data-pisa="ggg;10/1/211" data-pisa-master="ggg;g3.119.400854" data-apath="/ggg/10/1/211.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inference of Historical Population-Size Changes with Allele-Frequency Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael Lynch</span>, <span class="highwire-citation-author" data-delta="1">Bernhard Haubold</span>, <span class="highwire-citation-author" data-delta="2">Peter Pfaffelhuber</span> and <span class="highwire-citation-author" data-delta="3">Takahiro Maruki</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">211-223;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400854 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495284--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mlynch11@asu.edu" class="" data-icon-position="" data-hide-link-title="0">mlynch11@asu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bernhard Haubold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön 24306, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BHaubold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haubold%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernhard%2BHaubold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Pfaffelhuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Mathematics and Physics, University of Freiburg, Freiburg 79104, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BPfaffelhuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfaffelhuber%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BPfaffelhuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Takahiro Maruki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiro%2BMaruki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maruki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATakahiro%2BMaruki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/211" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/211&amp;title=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/211&amp;title=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/211&amp;title=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/211&amp;text=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/211&amp;title=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/211&amp;t=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/211&amp;title=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/211&amp;title=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495284/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495284" id="node495284--3" data-pisa="ggg;10/1/211" data-pisa-master="ggg;g3.119.400854" data-apath="/ggg/10/1/211.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inference of Historical Population-Size Changes with Allele-Frequency Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael Lynch</span>, <span class="highwire-citation-author" data-delta="1">Bernhard Haubold</span>, <span class="highwire-citation-author" data-delta="2">Peter Pfaffelhuber</span> and <span class="highwire-citation-author" data-delta="3">Takahiro Maruki</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">211-223;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400854 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495284--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mlynch11@asu.edu" class="" data-icon-position="" data-hide-link-title="0">mlynch11@asu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bernhard Haubold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön 24306, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BHaubold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haubold%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernhard%2BHaubold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Pfaffelhuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Mathematics and Physics, University of Freiburg, Freiburg 79104, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BPfaffelhuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfaffelhuber%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BPfaffelhuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Takahiro Maruki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodesign Center for Mechanisms of Evolution, Arizona State University, Tempe, AZ 85287,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiro%2BMaruki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maruki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATakahiro%2BMaruki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=211&amp;rft.epage=223&amp;rft.atitle=Inference%20of%20Historical%20Population-Size%20Changes%20with%20Allele-Frequency%20Data&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lynch%2C+Michael&amp;rft.au=Haubold%2C+Bernhard&amp;rft.au=Pfaffelhuber%2C+Peter&amp;rft.au=Maruki%2C+Takahiro"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495284/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495284/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495284/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495284/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495284/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495284/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495284/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495284/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495284/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495284/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495284/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495284/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/211" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Theory</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Estimation procedure</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Application to Daphnia population-genomic data</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/211.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/211.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/211.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
