<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/299" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/299.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/299.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/299.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/299.atom" />
<meta name="HW.pisa" content="ggg;10/1/299" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (Populus tremula)" />
<meta name="DC.Identifier" content="10.1534/g3.119.400504" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Apuli et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The rate of meiotic recombination is one of the central factors determining genome-wide levels of linkage disequilibrium which has important consequences for the efficiency of natural selection and for the dissection of quantitative traits. Here we present a new, high-resolution linkage map for Populus tremula that we use to anchor approximately two thirds of the P. tremula draft genome assembly on to the expected 19 chromosomes, providing us with the first chromosome-scale assembly for P. tremula ([Table 2][1]). We then use this resource to estimate variation in recombination rates across the P. tremula genome and compare these results to recombination rates based on linkage disequilibrium in a large number of unrelated individuals. We also assess how variation in recombination rates is associated with a number of genomic features, such as gene density, repeat density and methylation levels. We find that recombination rates obtained from the two methods largely agree, although the LD-based method identifies a number of genomic regions with very high recombination rates that the map-based method fails to detect. Linkage map and LD-based estimates of recombination rates are positively correlated and show similar correlations with other genomic features, showing that both methods can accurately infer recombination rate variation across the genome. Recombination rates are positively correlated with gene density and negatively correlated with repeat density and methylation levels, suggesting that recombination is largely directed toward gene regions in P. tremula .

 [1]: #T2" />
<meta name="DC.Contributor" content="Rami-Petteri Apuli" />
<meta name="DC.Contributor" content="Carolina Bernhardsson" />
<meta name="DC.Contributor" content="Bastian Schiffthaler" />
<meta name="DC.Contributor" content="Kathryn M. Robinson" />
<meta name="DC.Contributor" content="Stefan Jansson" />
<meta name="DC.Contributor" content="Nathaniel R. Street" />
<meta name="DC.Contributor" content="Pär K. Ingvarsson" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (Populus tremula)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The rate of meiotic recombination is one of the central factors determining genome-wide levels of linkage disequilibrium which has important consequences for the efficiency of natural selection and for the dissection of quantitative traits. Here we present a new, high-resolution linkage map for &lt;i&gt;Populus tremula&lt;/i&gt; that we use to anchor approximately two thirds of the &lt;i&gt;P. tremula&lt;/i&gt; draft genome assembly on to the expected 19 chromosomes, providing us with the first chromosome-scale assembly for &lt;i&gt;P. tremula&lt;/i&gt; (Table 2). We then use this resource to estimate variation in recombination rates across the &lt;i&gt;P. tremula&lt;/i&gt; genome and compare these results to recombination rates based on linkage disequilibrium in a large number of unrelated individuals. We also assess how variation in recombination rates is associated with a number of genomic features, such as gene density, repeat density and methylation levels. We find that recombination rates obtained from the two methods largely agree, although the LD-based method identifies a number of genomic regions with very high recombination rates that the map-based method fails to detect. Linkage map and LD-based estimates of recombination rates are positively correlated and show similar correlations with other genomic features, showing that both methods can accurately infer recombination rate variation across the genome. Recombination rates are positively correlated with gene density and negatively correlated with repeat density and methylation levels, suggesting that recombination is largely directed toward gene regions in &lt;i&gt;P. tremula&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/299" />
<meta name="citation_id" content="10/1/299" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/299" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/299.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/299.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/299.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400504" />
<meta name="citation_pmid" content="31744900" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="299" />
<meta name="citation_lastpage" content="309" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Rami-Petteri Apuli" />
<meta name="citation_author_institution" content="Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden," />
<meta name="citation_author" content="Carolina Bernhardsson" />
<meta name="citation_author_institution" content="Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden," />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Ecology and Environmental Science, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3258-275X" />
<meta name="citation_author" content="Bastian Schiffthaler" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9771-467X" />
<meta name="citation_author" content="Kathryn M. Robinson" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden" />
<meta name="citation_author" content="Stefan Jansson" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7906-6891" />
<meta name="citation_author" content="Nathaniel R. Street" />
<meta name="citation_author_institution" content="Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6031-005X" />
<meta name="citation_author" content="Pär K. Ingvarsson" />
<meta name="citation_author_institution" content="Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden," />
<meta name="citation_author_email" content="par.ingvarsson@slu.se" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9225-7521" />
<meta name="citation_reference" content="Van der Auwera, G. A., M. O. Carneiro, C. Hartl, R. Poplin, G. Del Angel et al., 2013 From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. Curr. Prot. Bioinf. 43: 11.10.1–33. 10.1002/0471250953.bi1110s43" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. H. Barton;citation_title=Linkage and the limits to natural selection.;citation_pages=821-841;citation_volume=140;citation_year=1995;citation_issue=2;citation_pmid=7498757" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Begun;citation_author=C. F. Aquadro;citation_title=Levels of naturally occurring DNA polymorphism correlate with recombination rates in D. melanogaster.;citation_pages=519-520;citation_volume=356;citation_year=1992;citation_issue=6369;citation_pmid=1560824;citation_doi=10.1038/356519a0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. R. Booker;citation_author=R. W. Ness;citation_author=P. D. Keightley;citation_title=The recombination landscape in wild house mice inferred using population genomic data.;citation_volume=207;citation_year=2017;citation_pmid=28751421;citation_doi=10.1534/genetics.117.300063" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. H. Chan;citation_author=P. A. Jenkins;citation_author=Y. S. Song;citation_title=Genome-wide fine-scale recombination rate variation in Drosophila melanogaster.;citation_pages=e1003090;citation_volume=8;citation_year=2012;citation_pmid=23284288;citation_doi=10.1371/journal.pgen.1003090" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=K. Choi;citation_author=I. R. Henderson;citation_title=Meiotic recombination hotspots - a comparative view.;citation_pages=52-61;citation_volume=83;citation_year=2015;citation_pmid=25925869;citation_doi=10.1111/tpj.12870" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=K. Choi;citation_author=X. Zhao;citation_author=K. A. Kelly;citation_author=O. Venn;citation_author=J. D. Higgins;citation_title=Arabidopsis meiotic crossover hot spots overlap with H2A.Z nucleosomes at gene promoters.;citation_pages=1327-1336;citation_volume=45;citation_year=2013;citation_pmid=24056716;citation_doi=10.1038/ng.2766" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. E. DePristo;citation_author=R. Banks;citation_author=K. V. Poplin;citation_author=J. R. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_title=Biology of Populus and its implications for management and conservation;citation_title=The genecology of Populus;citation_pages=33-55;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. Fracheboud;citation_author=V. Luquez;citation_author=L. Björkén;citation_author=A. Sjödin;citation_author=H. Tuominen;citation_title=The control of autumn senescence in European aspen.;citation_volume=149;citation_year=2009;citation_pmid=19201914;citation_doi=10.1104/pp.108.133249" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=S. M. Fullerton;citation_author=A. B. Carvalho;citation_author=A. G. Clark;citation_title=Local Rates of Recombination Are Positively Correlated with GC Content in the Human Genome;citation_pages=1139-1142;citation_volume=18;citation_year=2001;citation_issue=6;citation_pmid=11371603;citation_doi=10.1093/oxfordjournals.molbev.a003886" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Giraut;citation_author=M. Falque;citation_author=J. Drouaud;citation_author=L. Pereira;citation_author=O. C. Martin;citation_title=Genome-wide crossover distribution in Arabidopsis thaliana meiosis reveals sex-specific patterns along chromosomes.;citation_pages=e1002354-e1002354;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072983;citation_doi=10.1371/journal.pgen.1002354" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. He;citation_author=H. K. Dooner;citation_title=Haplotype structure strongly affects recombination in a maize genetic interval polymorphic for Helitron and retrotransposon insertions.;citation_volume=106;citation_year=2009;citation_pmid=19416860;citation_doi=10.1073/pnas.0902972106" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=Q. Haenel;citation_author=T. G. Laurentino;citation_author=M. Roesti;citation_author=D. Berner;citation_title=Meta-analysis of chromosome-scale crossover rate variation in eukaryotes and its significance to evolutionary genomics.;citation_pages=2477-2497;citation_volume=27;citation_year=2018;citation_doi=10.1111/mec.14699" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=I. R. Henderson;citation_title=Control of meiotic recombination frequency in plant genomes.;citation_pages=556-561;citation_volume=15;citation_year=2012;citation_issue=5;citation_pmid=23017241;citation_doi=10.1016/j.pbi.2012.09.002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. K. Ingvarsson;citation_title=Natural selection on synonymous and nonsynonymous mutations shapes patterns of polymorphism in Populus tremula.;citation_volume=27;citation_year=2010;citation_pmid=19837657;citation_doi=10.1093/molbev/msp255" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=S. Jansson;citation_author=C. J. Douglas;citation_title=Populus: a model system for plant biology.;citation_pages=435-458;citation_volume=58;citation_year=2007;citation_pmid=17280524;citation_doi=10.1146/annurev.arplant.58.032806.103956" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Kim;citation_author=V. Plagnol;citation_author=T. T. Hu;citation_author=C. Toomajian;citation_author=R. M. Clark;citation_title=Recombination and linkage disequilibrium in Arabidopsis thaliana.;citation_pages=1151-1155;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17676040;citation_doi=10.1038/ng2115" />
<meta name="citation_reference" content="citation_journal_title=J. Nat. Conserv.;citation_author=J. Kouki;citation_author=K. Arnold;citation_author=P. Martikainen;citation_title=Long-term persistence of aspen – a key host for many threatened species – is endangered in old-growth conservation areas in Finland.;citation_pages=41-52;citation_volume=12;citation_year=2004;citation_doi=10.1016/j.jnc.2003.08.002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. Krueger;citation_author=S. R. Andrews;citation_title=Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications.;citation_volume=27;citation_year=2011;citation_pmid=21493656;citation_doi=10.1093/bioinformatics/btr167" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Kulathinal;citation_author=S. M. Bennett;citation_author=C. L. Fitzpatrick;citation_author=M. A. F. Noor;citation_title=Fine-scale mapping of recombination rate in Drosophila refines its correlation to diversity and divergence.;citation_volume=105;citation_year=2008;citation_pmid=18621713;citation_doi=10.1073/pnas.0801848105" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. Lenormand;citation_author=J. Dutheil;citation_title=Recombination difference between sexes: a role for haploid selection.;citation_pages=e63-e63;citation_volume=3;citation_year=2005;citation_issue=3;citation_pmid=15736976;citation_doi=10.1371/journal.pbio.0030063" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with burrows-wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y.-C. Lin;citation_author=J. Wang;citation_author=N. Delhomme;citation_author=B. Schiffthaler;citation_author=G. Sundström;citation_title=Functional and evolutionary genomic inferences in Populus through genome and population sequencing of American and European aspen.;citation_volume=115;citation_year=2018;citation_pmid=30373829;citation_doi=10.1073/pnas.1801437115" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=V. Luquez;citation_author=D. Hall;citation_author=B. R. Albrectsen;citation_author=J. Karlsson;citation_author=P. K. Ingvarsson;citation_title=Natural phenological variation in aspen (Populus tremula): the SwAsp collection.;citation_pages=279-292;citation_volume=4;citation_year=2008;citation_doi=10.1007/s11295-007-0108-y" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=M. Mandel;citation_author=J. Marmur;citation_title=[109] Use of ultraviolet absorbance-temperature profile for determining the quanine plus cytosine content of DNA.;citation_pages=195-206;citation_volume=12;citation_year=1968;citation_doi=10.1016/0076-6879(67)12133-2" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. Ma;citation_author=J. Wang;citation_author=G. Zhou;citation_author=Z. Yue;citation_author=Q. Hu;citation_title=Genomic insights into salt adaptation in a desert poplar.;citation_pages=2797;citation_volume=4;citation_year=2013;citation_pmid=24256998;citation_doi=10.1038/ncomms3797" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A. P. Marand;citation_author=H. Zhao;citation_author=W. Zhang;citation_author=Z. Zeng;citation_author=C. Fang;citation_title=Historical meiotic crossover hotspots fueled patterns of evolutionary divergence in rice.;citation_volume=31;citation_year=2019;citation_pmid=30705136;citation_doi=10.1105/tpc.18.00750" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_title=The fine-scale structure of recombination rate variation in the human genome. Science;citation_pages=581-584;citation_volume=304;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=M. W. Nachman;citation_title=Variation in recombination rate across the genome: evidence and implications.;citation_pages=657-663;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433578;citation_doi=10.1016/S0959-437X(02)00358-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Nordborg;citation_author=D. Weigel;citation_title=Next-generation genetics in plants.;citation_pages=720-723;citation_volume=456;citation_year=2008;citation_issue=7223;citation_pmid=19079047;citation_doi=10.1038/nature07629" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=I. Paolucci;citation_author=M. Gaudet;citation_author=V. Jorge;citation_author=I. Beritognolo;citation_author=S. Terzoli;citation_title=Genetic linkage maps of Populus alba L. and comparative mapping analysis of sex determination across Populus species.;citation_pages=863-875;citation_volume=6;citation_year=2010;citation_doi=10.1007/s11295-010-0297-7" />
<meta name="citation_reference" content="Quinlan, A. R. 2014 BEDTools: The Swiss‐army tool for genome feature analysis. Curr. Prot. Bioinf. 47: 11.12.1–11.12.34." />
<meta name="citation_reference" content="R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for statistical Computing, Vienna, Austria. https://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Rezvoy;citation_author=D. Charif;citation_author=L. Guéquen;citation_author=G. A. B. Marais;citation_title=MareyMap: an R-based tool with graphical inter-face for estimation recombination rates.;citation_volume=23;citation_year=2007;citation_pmid=17586550;citation_doi=10.1093/bioinformatics/btm315" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Rodgers-Melnick;citation_author=P. J. Bradbury;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=C. B. Acharya;citation_title=Recombination in diverse maize is stable, predictable, and associated with genetic load.;citation_volume=112;citation_year=2015;citation_pmid=25775595;citation_doi=10.1073/pnas.1413864112" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=H. Saze;citation_author=T. Kakutani;citation_title=Differentiation of epigenetic modifications between transposons and genes.;citation_pages=81-87;citation_volume=14;citation_year=2011;citation_issue=1;citation_pmid=20869294;citation_doi=10.1016/j.pbi.2010.08.017" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. Schiffthaler;citation_author=C. Bernhardsson;citation_author=P. K. Ingvarsson;citation_author=N. R. Street;citation_title=BatchMap: A parallel implementation of the OneMap R package for fast computation of F1 linkage maps in outcrossing species.;citation_pages=e0189256;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0189256" />
<meta name="citation_reference" content="citation_journal_title=Genom Evol. Biol.;citation_author=A. F. Shanfelter;citation_author=S. L. Archambeault;citation_author=M. A. White;citation_title=Divergent Fine-Scale Recombination Landscapes between a Freshwater and Marine Population of Threespine Stickleback Fish.;citation_pages=1573-1585;citation_volume=11;citation_year=2019;citation_doi=10.1093/gbe/evz090" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=O. B. Silva-Junior;citation_author=D. Grattapaglia;citation_title=Genome-wide patterns of recombination, linkage disequilibrium and nucleotide diversity from pooled resequencing and single nucleotide polymorphism genotyping unlock the evolutionary history of Eucalyptus grandis.;citation_pages=830-845;citation_volume=208;citation_year=2015;citation_pmid=26079595;citation_doi=10.1111/nph.13505" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=W. Si;citation_author=Y. Yuan;citation_author=J. Huang;citation_author=X. Zhang;citation_author=Y. Zhang;citation_title=Widely distributed hot and cold spots in meiotic recombination as shown by the sequencing of rice F2 plants.;citation_pages=1491-1502;citation_volume=206;citation_year=2015;citation_pmid=25664766;citation_doi=10.1111/nph.13319" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=G. T. Slavov;citation_author=S. P. Difazio;citation_author=M. J. Schackwitz;citation_author=W. Muchero;citation_author=E. Rodgers-Melnick;citation_title=Genome resequencing reveals multiscale geographic structure and extensive linkage disequilibrium in the forest tree Populus trichocarpa.;citation_pages=713-725;citation_volume=196;citation_year=2012;citation_issue=3;citation_pmid=22861491;citation_doi=10.1111/j.1469-8137.2012.04258.x" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond., B;citation_author=J. Stapley;citation_author=P. G. D. Feulner;citation_author=S. E. Johnston;citation_author=A. W. Santure;citation_author=C. M. Smadja;citation_title=Variation in recombination frequency and distribution across eukaryotes: patterns and processes.;citation_volume=372;citation_year=2017;citation_pmid=29109219;citation_doi=10.1098/rstb.2016.0455" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H. Tang;citation_author=X. Zhang;citation_author=C. Miao;citation_author=J. Zhang;citation_author=R. Ming;citation_title=ALLMAPS: robust scaffold ordering based on multiple maps.;citation_pages=3;citation_volume=16;citation_year=2015;citation_pmid=25583564;citation_doi=10.1186/s13059-014-0573-1" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot.;citation_author=G. Taylor;citation_title=Populus: Arabidopsis for forestry. Do we need a model tree?;citation_volume=90;citation_year=2002;citation_pmid=12451023;citation_doi=10.1093/aob/mcf255" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=G. P. Tiley;citation_author=J. G. Burleigh;citation_title=The relationship of recombination rate, genome structure, and patterns of molecular evolution across angiosperms.;citation_pages=194;citation_volume=15;citation_year=2015;citation_pmid=26377000;citation_doi=10.1186/s12862-015-0473-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Tong;citation_author=H. Li;citation_author=Y. Wang;citation_author=X. Li;citation_author=J. Ou;citation_title=Construction of high-density linkage maps of Populus deltoides × P. simonii using restriction-site associated DNA sequencing.;citation_pages=e0150692;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0150692" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. Tuskan;citation_author=S. DiFazio;citation_author=S. Jansson;citation_author=J. Bohlmann;citation_author=I. Grigoriev;citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp;amp; Gray).;citation_pages=1596-1604;citation_volume=313;citation_year=2006;citation_issue=5793;citation_pmid=16973872;citation_doi=10.1126/science.1128691" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. Wang;citation_author=J. Ding;citation_author=B. Tan;citation_author=K. M. Robinson;citation_author=I. H. Michelson;citation_title=A major locus controls local adaptation and adaptive life history variation in a perennial plant.;citation_pages=72;citation_volume=19;citation_year=2018;citation_doi=10.1186/s13059-018-1444-y" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Wang;citation_author=S. Du;citation_author=S. Dayanandan;citation_author=D. Wang;citation_author=Y. Zeng;citation_author=J. Zhang;citation_title=Phylogeny reconstruction and hybrid analysis of Populus (Salicaceae) based on nucleotide sequences of multiple single-copy nuclear genes and plastid fragments.;citation_pages=e103645;citation_volume=9;citation_year=2014;citation_pmid=25116432;citation_doi=10.1371/journal.pone.0103645" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wang;citation_author=N. R. Street;citation_author=D. G. Scofield;citation_author=P. K. Ingvarsson;citation_title=Natural selection and recombination rate variation shape nucleotide polymorphism across the genomes of three related Populus species.;citation_volume=202;citation_year=2016;citation_pmid=26721855;citation_doi=10.1534/genetics.115.183152" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_title=EVOLUTION IN MENDELIAN POPULATIONS;citation_pages=97-159;citation_volume=16;citation_year=1931;citation_issue=2;citation_pmid=17246615;citation_doi=10.1007/BF02459575" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Zhang;citation_author=Z. Zhang;citation_author=K. Yang;citation_author=B. Li;citation_title=Genetic mapping in (Populus tomentosa x Populus bolleana) and P. tomentosa Carr. using AFLP markers.;citation_pages=657-662;citation_volume=108;citation_year=2004;citation_issue=4;citation_pmid=14564399;citation_doi=10.1007/s00122-003-1478-7" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=A. V. Zhigunov;citation_author=P. S. Ulianich;citation_author=M. V. Lebedeva;citation_author=P. I. Chang;citation_author=S. V. Nuzhdin;citation_title=Development of F1 hybrid population and the high-density linkage map for European aspen (Populus tremula L.) using RADseq technology.;citation_pages=180;citation_volume=17;citation_year=2017;citation_doi=10.1186/s12870-017-1127-y" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (Populus tremula)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="The rate of meiotic recombination is one of the central factors determining genome-wide levels of linkage disequilibrium which has important consequences for the efficiency of natural selection and for the dissection of quantitative traits. Here we present a new, high-resolution linkage map for Populus tremula that we use to anchor approximately two thirds of the P. tremula draft genome assembly on to the expected 19 chromosomes, providing us with the first chromosome-scale assembly for P. tremula ([Table 2][1]). We then use this resource to estimate variation in recombination rates across the P. tremula genome and compare these results to recombination rates based on linkage disequilibrium in a large number of unrelated individuals. We also assess how variation in recombination rates is associated with a number of genomic features, such as gene density, repeat density and methylation levels. We find that recombination rates obtained from the two methods largely agree, although the LD-based method identifies a number of genomic regions with very high recombination rates that the map-based method fails to detect. Linkage map and LD-based estimates of recombination rates are positively correlated and show similar correlations with other genomic features, showing that both methods can accurately infer recombination rate variation across the genome. Recombination rates are positively correlated with gene density and negatively correlated with repeat density and methylation levels, suggesting that recombination is largely directed toward gene regions in P. tremula .

 [1]: #T2" />
<meta name="og-title" property="og:title" content="Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (Populus tremula)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/299" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The rate of meiotic recombination is one of the central factors determining genome-wide levels of linkage disequilibrium which has important consequences for the efficiency of natural selection and for the dissection of quantitative traits. Here we present a new, high-resolution linkage map for Populus tremula that we use to anchor approximately two thirds of the P. tremula draft genome assembly on to the expected 19 chromosomes, providing us with the first chromosome-scale assembly for P. tremula ([Table 2][1]). We then use this resource to estimate variation in recombination rates across the P. tremula genome and compare these results to recombination rates based on linkage disequilibrium in a large number of unrelated individuals. We also assess how variation in recombination rates is associated with a number of genomic features, such as gene density, repeat density and methylation levels. We find that recombination rates obtained from the two methods largely agree, although the LD-based method identifies a number of genomic regions with very high recombination rates that the map-based method fails to detect. Linkage map and LD-based estimates of recombination rates are positively correlated and show similar correlations with other genomic features, showing that both methods can accurately infer recombination rate variation across the genome. Recombination rates are positively correlated with gene density and negatively correlated with repeat density and methylation levels, suggesting that recombination is largely directed toward gene regions in P. tremula .

 [1]: #T2" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (Populus tremula) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"j3hcEz1ZiznyyLE_aoJQpr60qLH4iCZyDP-dbumKwFk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495171","apath":"\/ggg\/10\/1\/299.atom","pisa":"ggg;10\/1\/299","ac":{"\/ggg\/10\/1\/299.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/299.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/299"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/299"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/299":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KY9ikqkMMtUv13JPTxeb0VJsQc8kPFjdEDRBz9qdJNQlgrWO8z1+QkSV5aTozemP22v6VY3fWDUEJ5BSVNwalfn3LQ56EzWw8ACK+upW4TXArjxPQ\\\\\\\/CtK1DGknKJfqkNaci0o8rA29Wt5bBv8YjLwn7Mu2CgxajJ2R06Er+syvBZidE4eoq6D40gyI2+EJRT0ez6xgndvz2lWuW4dnU2E\\\\\\\/VugBhxmlIbbj\\\\\\\/lsfXHNSsU2UqbSANS\\\\\\\/rXB2Cl13yeCgKUZVzVxa77tF4Fd79SVdJM7HoNetBzO9mQdCYlUC1gB4gdI4QuP1f2+CybdvpYcJLmyfq1mihGcyb\\\\\\\/FuZ4Uo87D1lqqk\\\\\\\/1YeytkQlbGy46rKqEE2CW1BNS1oZ4R12a9l4Z4+V0SJSbsgIxYa8YDBhZspw4aykweyfP9rZkNflhDng+QJEjptp6ygnBtMkQcFB8UtpNeeqWRYcRlsbvWaSpq5lNObWAqZc35+ve2wtMjxFSK5VWEIJd0g\\\\\\\/C7MRzAui3Wns9+Gt31vhrQHGwfwDm3KJFlNQqwrr9WC7mqYPHEMUD3Jc+o649yoB\\\\\\\/pYW0UTGhPEf7toVjySkXGx4hyO\\\\\\\/gjdUmOCWel18M6fXfaJ\\\\\\\/JwnWx8H28a90vovlWLrKxLclpNExH9n7EFeSWO1jUaiQNOP7Kve7DsqfQxgJau4JpCM1ejgQlgqu6CftVEu9EAR5wJt5Z9N5+\\\\\\\/fO6YcEGO2TRObNJZuFlvp+xEHD+0uWls99Bz4xzNyRFtSgXDADg906S6bAglVPex6MUAg98kOYu67UMKMJ\\\\\\\/2udPN6tt3QrvgNOCY0h6fSOXepV+ER1rTxk9Hs02lbJbm906Kx5QuIlzEMsWAAAFlzH9ilzQZ2r8lM\\\\\\\/DQfrmyeyVTHSegWDlU7ma2dkLH4VR6BDjqyTlOFgx0k2MIL3RJ2jrqu3iAZGD6aF7paemG02LQsxn3lMBoIHdlQdVdjLbPabFF5FaHFHz4hkBhLh5YjXM\\\\\\\/LBA\\\\\\\/da50bKqrHGIRSlmXPNw7145e6YxxyTbnlXXyjkzs+UAznbMxfUSMOzdVzEOuohVFrP+togk54z5XeY06el3cd9R2PQaMo57CsqYYMIwVRCOaz0KK\\\\\\\/ZCQK\\\\\\\/XJ+mszdCLuoh+31WKHcg3QzHK1xPqjFCUL8tkop1xlb7\\\\\\\/LWP57qtimaxSm1z2x6ZnGDYKjV5AGlgzDiEKvAIs1wN0shUMthYeGAINhVfLmXQymLBYBNhgmBCR49mkhFqJXIs1nLPPFleF2yzXN8SIj05FyyO5ca2gS\\\u0022,\\\u0022iv\\\u0022:\\\u00227JxqDf6jBVzJcQEFyZHJMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002269f4e174aad3cf76609175e564d68a71b874842ce70a3e7792631384a95ff7a7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c7Jpc1772DJ3M4\\\\\\\/a2LQ7Z6Ob4TZyvCqv35zXMeZrnyejU+3r7EmFqP97o6D4fE03gxrEDWUjWsZrU1xQAzOBklIhEoHQxktry56ScMiObBv1TfjFQoupFjNimKTFZaYwEA4ITjSSa\\\\\\\/xfemo4soiMivWTiuYW8c9tJ4SF7qXM4Nv5FYLuMU3gzh4YfD6bhtkFVc8jr2Bxw8tRAq1H0L6t270yoQLX3TW0WesrsEIm\\\\\\\/5mEe7MIy2mqyOgUUbcpjlKSKltK5Bx9UJtEXEIZ87opI3kZ2+dvQzTT+yMrhuqEQNi60DUqLl2snpa4ruVs66KqcYE492zDix8GVBc+9\\\\\\\/39QpaSQTrH4H8d7PX7kyHvORcCamxBeS2\\\\\\\/NxhCxgreSZ4NhNuMsLJIBaD73xaDNztlzeYHJolwS6v\\\\\\\/F1KxPmK26LvQoAvK5CSrNK6B\\\\\\\/53LYvJfQLxvY4RhlmWGZxpAyx8TpIXDGWvbS22yQPCfGVuyRRwNv8fyVaEFfvfNTxMY+MGJPefDYDdCyjMFWJGczuBxaDDeZ4U\\\\\\\/dpsGX9Tu0yK9kFt3h4C3wYMqZyp6UW2LAKyeVU467HDeH0DdNFNv6UG5SJlrcOsaPhpOAXGfIq22O3W+nrx9NTzLsuLqlJPtQUFtgv+zxpVpbauhdsw+qwMGUpRQ+nlSopSF6IjfT6R7q8qOdabIVNmeA\\\\\\\/9fFpwCT6k8ho7vKW\\\\\\\/fnVfzMeB+BrZz5Sn3VnhHqeIoPMNDyWBGAe9xsZs09G7nNQLN2seJqEEa3zWNte6hjH5Jo0LGtj9FmaF\\\\\\\/LpW0bgWxE93798XcFERnAqP12WXMAQF6VUuUCZOzbHg9JkIuZyYKN42T81os2MBYXXaIODrx0u3p0vsfERWOuTLbG9FRLz44Fkx4l4aG4pFY99M014qGyg6FmULO01Wf9dCv53WEKRugPzWxUsbF3KcMt8QNB9RF2uuQSXVNRzPdh9SIFDHmngPuXbtL0Nd99OFPwI8E0Awvs4C15BzdNTxj5hum+n9Iis1J1x3ibk+xwmv3EPmXDD6BDIEzhQTerGI7\\\\\\\/lqeYAJ+zT8JwQWgKBoCAf4GHHOtxQlxBYcDYuKZ\\\\\\\/7tF4MZgGWvVhPfCKvFPp0JIyDYAqHTsYqhtUaRq0hvQDm10JUW5iUnwmpsezGqVytskwQowTSoaxgz3n2wY9mDur8rPbv2ZiYeE0oRqlhSuldNjiamPC41\\\\\\\/JKTArQWcUkqgq44IzCQ+6KIylgL1qLRysfqZcl08wGdBbDshW2FThpFH8Lg8IYIuCohEsM6RtRiwoV0v0CMcjfVK75tH7W\\\\\\\/Q\\\\\\\/j1FWlLzh\\\\\\\/qylPSl56G9GNB5\\\\\\\/vV2u0LFDWN1S4e18obVnKgFBDP703pnlYvqNM73OvtCF8t6p73onVrMHGnNUvCpmtwu+kgEK8RO9JakY9CVT1ZtFKMvxk4ktxT3QVwfRUQCFW9NG1eaIYVjyBkGKM5MZlvqQ1nXNWlII+HDB930LrIst8v9zfKTaPX9uKOHKaYHYQc7rjKQkG733xAF1hdLtYAsHrEEImM7v4FBud9xcOyDxPnxivP5LSNdna4ShmA1s6NSaSTiC72E+BWuLO97nmJQ3rcWNvZ4Br9HRRtLs3qROuFfTLxmh22qu4Qcxt+eF661f5a9BoDFGxwewTEuJF4RlVd1G0k5nJHdcTBhROOZSmoNsABzC4UeGoK\\\\\\\/wA0508BKJwC08JP7I1ba2WN4c1xqCM4SXs30zw8XIlerkDMvLfRP7bvTnSA\\\\\\\/iYyDFOIBjt7YdUwxwI\\\\\\\/L05Wd4uZXaLKeJgu\\\\\\\/li7S1qrGgR+ginkkKyD3EU879dv+W06IIEr6MhE3MKvxK9e3hctmWlzf4wOsCIw2wlkGV69JkJYu0lgEAJQ84sTCM8pH4l2W7tFLImbaUvc0PdjnK5KmEXglgUGtubKety8OKRfGeGlIJtG5LzWPeFci\\\\\\\/ZUA2A0YrIr7pyUSOHcEWX1ocmlAaqtf4IlYo+D8nAvgHsExZuZeV0btRP3Z62j\\\\\\\/8z3x5ahTozXWqMo5hLolEFgNjIXbR8bkvpFZmrdr8ABRLST+9lPuSYGxD+bBUIB+SDChVdk\\\\\\\/swHJM1oXrdqUNoW44sLsXR7sY0UvikUIGAAdQ5NzjMx4fkMJaCz3Ia30kj5\\\\\\\/gI8eujPc4hdRGa4AFHKl4smbRDadcK\\\\\\\/xvA5Y00DViGeb9KNWMEvvE4uHNQlhykQ9BzcmYM\\\\\\\/jsl0WpPyKxRBttmApbtcAcKpPDJ\\\\\\\/BhzL\\\\\\\/numrHzUuG\\\\\\\/6YiNOvs9\\\\\\\/gn\\\\\\\/pgLSjTdNdbJihVHhkf7h9cAu4CbmfjRUKw+4JXZLZUVyIYc9ciKChXWaHgyMKuiUNPQ0d5qdQQLFQUHLlFrH+1tjdwDRHSbHHFHtZTxmTqo2vfxAU\\\\\\\/XU74ZKghnex+UEn5PMsADpSwzIkUKzi5vqqfC9WGeeT04n\\\\\\\/Lc5vLUN4srCm\\\\\\\/0ZhIOjRxn9uRsIM43r37b0WsfyRMkaIf7QtNJt5kAK91DuqaBvFbP3jp9\\\\\\\/ugKyMT7EK4wNIXnbJMeBZJfRz8wLUnwXBckDlz6\\\\\\\/CKKV3k\\\\\\\/tiGlycA0hd\\\\\\\/OxurGWSImQ3QV8QXxYK06B\\\\\\\/sVYavtTGAhb5Kq7p1GLkuPrhTS9QWdHPfRPfFH19R2zAU\\\\\\\/1uUf4Gsy0xkrGXW+uuTtjm1n0zQS6NBIO3kU5Mlp\\\\\\\/69ajDOVGxkFnjNxjbTwAxSHbriG+aGWoSSZI\\\\\\\/edGUw2X\\\\\\\/xb6HPa8q\\\\\\\/Bn17IlwBP3c6LIng8gPDgYr0fK71QPxMXnhgDWXSKpu0HMxbVn8s\\\\\\\/tRA79c+AUxQb8AXpzRobC1s6clvu9s+FTCULOfTP+HnQgkqTUahCIwfAPTwe0zQXvSb1rWmlS0jky9oGXVnRn99aONMo7ndMEhluh0h8\\\\\\\/P6k5D\\\\\\\/9124WcYqUDbe+KMiN3+8QLefoinaWmPEzRODsuwwxCw5TAcQFrW4M90LJCiCdj1ebyqeYc6NuIn3pP4xl1D3d1T2OLwT5v9kxqIA2SIEjpoh8QgJUX9ah8KZHsYBnkeJKh3sgsKrBj+eKD16TKKfHCgB8Ffv\\\\\\\/WEjsGJM2FtZZTJ9MVnK87zCo0F0AElgQaU1nQSJ\\\\\\\/5Bf+as4PIKSIygTv1wYZag\\\\\\\/YGWaiXpmGwbUW7fi4TJWaAKaKI3CTO+cBC\\\\\\\/XUaxP8wNL8FT4iQ9cF0uy1kjcdICGB5Mai1gylTPCBuE4qfzknR4k93TU+HEKETg41meGLihduMET7yYwS4maV4gw0uQKhaZLq9m6aPB3B+bUtn1RwLX+ZexpyoD3ENnw1TsAFbD\\\\\\\/ASQLvuDliQ8B0BAJuoQGqrlA9LjU6IyhT+qALvJExjG8fPomtSQCLtzDTe2AmzqxEFXYMyZ85gG9indxtK5gpio\\\\\\\/B1WvA5H+DW1erY24TxbmTwEcdV3Nsx4kBd+FeCgD0o2GIDjcOYaXTCOpsQQ4udl\\\\\\\/bd2\\\\\\\/zzXqdDPT8edDcnWthBrPNM7+o6Zkm4DGlwZ5M8LXc\\\\\\\/r2y82DaLbzsRqVAZrgCtCnwjNNFE84Ea5xIYEP3mHS7CO\\\\\\\/p3iYkhD00ZWDHtbDZSnuwBb6xON7jcYF+WuQQIcMHNW8xv0nRTxpSxDjjDz9W7HBlQ4Yvw4hWbyCgtG3flHucu0CVXnUVhvSaRdaY\\\\\\\/620tjlnWWSMVXjBD21QCHB46eywuhG8KWgbiNvOxTpUNMrL3VxTMURhAaEQJcc3lKfm9DSaB9rHoMdSfaEPJ6kEdOAhHcBT4biYg86vyKgfvDcnZNj+sv53ie1t71\\\\\\\/i43XdBQ3OtywOrABiH\\\\\\\/BeZkCTrudoUXt9X6qZPDI9ZEuH3aRmWFVFjPuuVnwYfzJtiG5mee7vBi9zdyAv7JkHYJYR6TdscT0m+rrxmB3t2889YEAdxlfEDRE\\\\\\\/n5MFxSBRod3h6bnEY1SkBVZe10dSsjpKN7IzPupFKqQsqn0SvX01PPPrbvVBJynv1XIGOjGe4cuSfBx2kcS85Hjs\\\\\\\/oLszHcEQlQCs2c6x9Fj\\\\\\\/5xdhTFKiMtch2Vqjz14H6F8jVVlvZPs53WO7KrL91+SVZ+a4PC1RLHcHuz01GqhROSvHG8sj9w69iytQRo0yApCB+t9aYMizfQwXNvsuEfEMPRfhkfost4JNxbEIIfTOpp9lHPJ8LJjaU5L3iJU577SdVlBJ6p5\\\\\\\/7xtRgNySPhpr1+heJIXTM3qYacM1ZKlRjR1oDdEq6or5Ze7dtIq74YaTMnuqHXjygNslPrrFwBV7BeFHwPDJAt0QyNWvNTKPnN2zM66WRoJVvAjjPhdwLkJuzQBh1gu5E7MpTla4tevAl6qa2NgmUv0xJ7YxokFzOlqElx8l9lc9Askw5m9szeJzSJhBVlZeo7RwaYWzpNu9bNsr+iw\\\\\\\/Xxhhp8GTEGGW6EKCbbO17gN+kvsE62\\\\\\\/9UAc0IzOI\\\\\\\/xAZTOHwhIyNXbgfVDzuTXYvAkNqF45WupFE2ADoR11A1lgHdxq+7gjTQUgH1QRBNd7LMO37EqY\\\\\\\/lzYiZTlfGZL6Sdg3STQA9yr3knPjwH+YVjvoSwiRwC+kv6pvLQ9tfNy\\\\\\\/tqh2Yl8D0LzxwnF\\\\\\\/ywQwe\\\\\\\/\\\\\\\/Xsl3nXW4FE8omKFzY5Jb9mrbDQ4Mt7CFsvLP39pOvdTOeUp7aTa13akHjaZvMNehSekdZ\\\\\\\/zDxX7btU9XhgcVHsPb8t9QGmFZdgWd6Uul3Y1TLHg1vI6m+spSDK29v87nfPLKnU9sOM9Ct+vCBdBVwXK1J5FjCj4NzGzxUcRNOHyi5mxBNMHvpj8Cv8gCwWCXOiIB3SlG97ezHOt4NS+aEBwM60ptmwPhUVaLMGlwOxhEffZLL9nRdVe3v2l0PiPDoc6ifhRjj0VwFrIhilr7gJsLNM4+HjEB7CFfi6Sa87Dzp4V6JOGlLvAkTST7HKud9tpDIj3MC3VVmy2K5SQMFp+wljC\\\\\\\/Cj94xrwLSw8YauNjTYWBAKlIxIOQrVIXM4i1WUAj0O34IJEzLekAloSIIuxDm1OlBsdGN9RwGdevjVOUY1my04EuUMCmGJwfv6\\\\\\\/6W8gSwqJhcB4kuZum\\\\\\\/qPe3I\\\\\\\/V8Fw8NRq5H2NvQQhFS68U+YDH7BDmpRrG+gehqSTSz77S+wbP2+4Su4Lv9pH88T4eEvvVgqX2lB+ttFhROceehxokCrh6dGW4C1RM3tKI\\\\\\\/r0B5tcQkg7V9CNx99XWTqdxn4WE4HUU7R7ZWlcErghTjML7D\\\\\\\/LGyBsNjJ+Qii4X4gv+UfS4WUxfDPdpJJ9lTRWAAqjEcqs+aPn2URH\\\\\\\/TwuJ\\\\\\\/drHEcarbgDs1kmB5JLGNNSysU0mRy\\\\\\\/r7UBztjUqY9b+0oIQ5s0GeNkS+203+zoGW\\\\\\\/hL4gBD2xQY\\\\\\\/vzpA8VhgVK+8mA4f+QeI2klDHFrkSJwow5cQTx+p4goUHYq2WGBUS3NUS2XtYufpyckhaBStLiEb5jbuUstQbG8jt0aSg\\\\\\\/AgSRz4jejqUShO6HNGeZ00M2Rv9x9qJe\\\\\\\/Cd8BySFUqE\\\\\\\/p+WHi8cM+LdV5IKEsDpnEN8BnllYZJ2qEHr\\\\\\\/XOgWCZ1+2z+tF2yDoC7KMTnwC\\\\\\\/wIcAWmR4pLUgWchsquDvN8n8I9n+4xonhMSbey2kWwuLLUl+lXRsj1Ng7FKUeTR3i7ijF4w88IkdRhiisUKcOV3svD9L4kz7nrdbrxl1V6VvuTRTIDl89rAU8Xj6Jhrla4CQFr8qQpOp6ZALItTAjZPzfo\\\\\\\/YKWZQI8P3dBRn3HCb2L5f79wAEpw0BJzE8Hm1eM9MVO0C9DN5zucI5TzkoQJs2OW1kj7ooTsmssm4ZMFJZUFwW1bUfNuKBS2K8pIe\\\\\\\/s6uHIe05MAI+7+shoje7Q3bwDkRwUMPt\\\\\\\/lR0od\\\\\\\/jD9jist4VN\\\\\\\/vkGafLkC\\\\\\\/Ko2WaMG2CQmwzOc+lRAaNc5hOkaA6h\\\\\\\/lRRLhlA0WI2aogKMCjrgKmLZHMvUI3ROZIN3IQl8lQP7VxYQapV2N6IyhgVC+Z1OHYajBt39LVWP1G6yXglrD\\\\\\\/mvDXoa9poGKzSdNYQswVOEgreLloAtXUa8pNMNRntBZcRewEf7phUL+5cJcCpewIlbnjGgWb658akMonwCsoz21wuJrTlrqL4cC77SNrpN57W51ek\\\\\\\/QI51Ign0SNUX6iOa0NqrBFGEtxnwgsAZsqBYlRp9i4pb90lTtdoUE8ptCbyvMEcqMS+4fVUFV4X\\\\\\\/gyTfglY6YFz8XSlIQJn9OFwmhFbSKSdSDjJcN7zP8XZJd6CLe15vFJ5gV862zanGlktVS9Cf\\\\\\\/OIq3R5H08Cd0zz6X+CwpOCxgtpDuBqPBCiXJ0TJLsb5XIBus7DXpAeoHQnJbv2Hy\\\\\\\/1dvgwMSJycuTu01lGUnRhGksKBZ9xQEG2tbFpUkfg4dkualBmR8RbmVer7ZnKAeEVCfQYeebV1ZX\\\\\\\/sToNPzfUDnWO\\\\\\\/id7lU+aUkilYxaNkqHOPALeHuPCZu4c+K9V1U7yuzYZlaep02pJb94es6vqMVEQ6r66V\\\\\\\/mjLxFMp7Wj0fFpc7KCt8J2KwiMqJvRov7mySpvuWbj5B3AZs0nia\\\\\\\/Q7mrRow2ySYn7Iv6+\\\\\\\/QZ6PJiBdJ5S\\\\\\\/KkNHqLEoRGMNY5FAA+Jw\\\\\\\/w5KLwFBBDrl4PV1frlVpNYHGkwSoS37qkWunnj2MfbbfF\\\\\\\/ySbolP+P41XOcm0Mw55mS79al7UkmbzDXelT4ZKs14jzeU1TMQOeo8OeOFKeFkFagXdw7bkvs\\\\\\\/CjGiy+1NgHnbJ1aau8EVd6OkBiVt9OP+AnCC6ErlR90N2hsTg\\\\\\\/BjixMEulmu2vaLcPGszcmucJ8b9SjejfWt6k\\\\\\\/rkhJ4DyyTnTDMYMYg3MfAq9w5jQsqGiQMy3872slXF9KuouEUnLzqqMwfJTccNcoojfKhnwkK1WMeR3CCQwJ91nMqVho0BlZ5w\\\\\\\/yha0rogaSPHcQo6wZhYCqkecppFoGHbxLoC4MNxU19T+47IrVZV1qJCwgKcsgfSaFe0JuIVGwccaKKZRFokX7bbbb3Afkjg3sICrYv6lxSLaW4k\\\\\\\/\\\\\\\/tdRNYlVuCHJDswRfBzIXYEjfqyY+TxXrkfk1mqq10GDKCaPpldstdnAv9nQuhK8XD5XKvNlcSsUWg7kL85OTWtgAspLYIHK54g01SU1eNfFN66U8\\\\\\\/gmjDP83tv+PHF1qFuItAE7LWv6SPceFoHTT0DEG8LX481szs5hDDidsM+KAVQmBbOcbqx3AcaPYJn2P8C6ktYsNIle9ufc+r3guIPsmTb2JwC7F+R4z3vwowxPQpcJKcCUDbnhpapw\\\\\\\/v4OTKIAJbUNOzdUrhN315Av14Pq+j4qD1gHUaeLe5FD42r6pZ67sAGJ8ksl5zNbZBDR2YZCWe4SMWQS78eE6N+Q29Z1NiIsNSzRxExi+\\\\\\\/2dbK37iRpsugadApeEGsDe9HyQNEr+fosmUjRI+su+1cnuyuSxrJvPK\\\\\\\/qalyf8OvSKnDfMankWmUnCcv1Pf9JGiCpHRPP4rVtIVtgBRh9oCaaw1elMApmAZegzug3gLs1kb\\\\\\\/MCqLfC31MzHq3wCPD5tUfu9nJ0amDepfLvrHK3hdvKbxvBqi2XdYKoafapiGfLfu8rlEIgxyhcXUFsYLYff40VZEVdTnH7dSUK7ybFoPeH4qdGdwgpWnvpEzWfJZmPtKUVBsGfssxxfa6Jer6sWnUcU8uAWfoobAeAGyaOD3Kn\\\\\\\/nlVQjUk3IEd5S\\\\\\\/Iv2NaoIkEFYRCwtyHMN8lQyc2wxqB35yum6DoRkOuCP33pLmdsRuEz7XXtfMkO0ZmeREO5dCpDDac5shaPnrLVnmCtIHGRQhKJHSKvcXeBpVVsycjhhXd6DxlciMTVBJABfpibzmd+kUz3Lk28yMl52XqVaqro7GDJXrT6CpINRwA6uEyJCaMRReaewFCTW1msWSoqBvysKKWfBGsly+61dIfYrX+rRfUeNIWdN4ctN3udjSqbZs5mz1zxoaESSICwuJIF\\\\\\\/Q8MRSTID5BaSk+BorwBY4TmBRSgI2KIsw8LFqqCw2BXlJc\\\\\\\/wS9iskwHwpwiTPMV0e7MFqbW6qojgDA6mkZKyoZs8LJ3LhKOLli39cUsqRIjLSGMnDxEzYySgrd7jAxdJ9NC+2SQ2b00\\\\\\\/cn2\\\\\\\/ubmSbxzWa0C\\\\\\\/JeFtPLAKydXbL9NINOX4Ds+JXUW1ZP4lZwv6FNEr6DI7l2qi1k5hBIzLj\\\\\\\/YOvV3hqndtcwvaK2J911vN17nGjY\\\\\\\/aqmlPZVtL4uVtGzuHyK0Slf6mrN2VLqDeOyBtVGv\\\\\\\/4M6vNZXw4e3LQFB+i0PYwQAbsV2b0T5rSbDTpaxA4nZk3IdaLTu1uidGuClfj1NndOq1P+k6XC60U6d4hOqSyCSVlRKa6st4jlm+1ptn5ceRwOS\\\\\\\/Gdb20og7LNDUGjviI+O+A\\\\\\\/tAg5ZI7HZH0YaRfJ6p2wxkzIFZ58L1sD9mGd7OFkaZ+Ej03QyFK1BmNn8CS7hQh0pNzwread6rPCqx7O0sK97PzE4AAcvYE56sCd790mnqcnRuJ8Y3PiISPeuH1lBYcKYmQY9tP41UqVY+eZvPoHci32pcIOFXQxJUIEQp3WKR1h3LUSUfY0VkXTYUYhoDGHpzuNqIqmrvqyskDwrg9fb3D\\\\\\\/mFEoF\\\\\\\/\\\\\\\/FMQ\\\\\\\/jlBnoFQ3aRWK3BzO4CureAg29ZL1\\\\\\\/vnTWnzUoWClUJZEYIQtjiXe\\\\\\\/L\\\\\\\/Fwy4niWFWmM8wExoG5A7MCB8z0MbN6oDT+GCs+3IHSU1m0nmvroKXXg41\\\\\\\/zvG\\\\\\\/VyY2WIYMVLTHovEGtdCaScgLf5exnPdPBhWJO2HYLeeuNt9o2vXtXyyxd8dGVdOd\\\\\\\/xslkxeauXqf7wYyJNLk619MNh\\\\\\\/Kwr9JaIfhukuTvMTc3\\\\\\\/8aDpZeSg66QkKvE9LSfN+qrFjziH78t9ZprpDWrln42xeFzIxd8iwELFuDvuuinH+IkmgIoxgkaDhqyhTK6xU0+RICo2878lK\\\\\\\/B1KJWQVfYpKWh8iqCaaOz4p2LKe7GcRp6YTw4QJAafaT6ZkLD8nj6m3sg4N1Ti5x5ho2Q6WvsCR5bLGPo+FqwD4Px8sFvle\\\\\\\/RZPP+hZZZG+oZBRhtWHhV2Ls\\\\\\\/zliAf\\\\\\\/sipAIvMjtIx7mI1vU9l3AelbgkDNUh1X90+nJlWQVA6ezvCSUMTAJ3Nl0bPvPrDUilHXf2QTAajK6Rx1C+jOc7cy0ilhe7Td0P\\\\\\\/GKZNhFcIZNT5YxL\\\\\\\/ng8wn\\\\\\\/pXEpwG201gvH1um6jr7yKMzuF2BFLRaX22BSoQkj+LteEORIYy7FZ7A0OqzhpzG\\\\\\\/sUuGdV1LnLBm2BwvEbeitvaaIQLgTqZJrzhNraQOEIwi4jzwtyjjaVV8fdAJURafYZnhLFISF8zWS8IvfDtxZJZlylyIxf91qyM0JWTWMapshPsTG3pMpa0HQLXoovVQ77WSCJnXr9IW2W7TMPp8NJX6MguXPmvzPpwd4MNUN6vaxQrhjol\\\\\\\/bAFbSNEzPcsN3znKPubnbmjKcSVB52FDtAxtJLrBlfwNEgnbKcewJEvDJxDYWRdphxpm9hT2JxVaquaTsL33\\\\\\\/C9A8E1v5\\\\\\\/yslOvagnxKpjMx+\\\\\\\/R5YQf6lQ7MtlF88z5B8JySLTE0X0P68MCztkJxGReX3w6XKrrSerE1tCJ2eRlj+GK8knIP000uep2ULFDEVsgqeISF8K1aOftYfecsktYlGHx89PurfLVtpcAMmHi9q4Qaju26cK1FTZOPjrrRbQPGgcPcYCII1Nkhb0tmweXpIrr5nE3bktM8Rnrb9FU0L5MNtEcuEXWxTVTZVJvOanq\\\\\\\/wL16sKVlDVYIOWMUZsWf+PadeCYMuSZuMo6humxItF6FIf6fWpw+xMExJ6Li9zuevo76coBvaT7pzECDU65ovU9rK13M1xgOftrxpNH3HNR1CrqcgSpBecmHS7uDlm6pV1pFmvJJt9BHqzAwHkk\\\\\\\/xxuUtVstcnvksFNxwRW4pgScx4soqugG9UW+lQGPYtyr2sJAL0mwxxs0+hBPz4HIqKdOBc3dJ9uh645agXoO\\\\\\\/5Xa7icFxhfV9bGez3ZF48ox6pdwuGuIqz9IInNaoobpchsCGonRBJzZVi\\\\\\\/jUhGstZXtYxfqpRGHZfEfywmmVgRTDOqHIM08O6iQl9XG33ZsYcNL332lG3+N3CW1vjBQLpMqA7faKqTY1EVY7ZY5fYlRkk4HKAXNS9PZNhs6FqRlT8k+br5GX9zoLggVvtrIJ+pVcRbBBdiK4XJ7DdSreRkWv3lS1grGXZX7UEp8a\\\\\\\/32geLtHZ1g9nhOIPh9ZPadMgzRN15ELODqCKaQ+KYO8KswgjxcA8f93DVB9G48htBr1lzmyL\\\\\\\/amdOWqCycujzV8Rv1niLQk72y51ikHvCpCiy6dxPfLWiOTs2FHzG0NFVN\\\\\\\/91UcPvtp0QbK63+lekTXVGERvT13LJ0Uj4pxFYebnMh\\\\\\\/fHa9W5m+YneUK+VHjr+PM0x5nZx6+2xqm5VSHOEfxMaF5jasAWNuN++p+mGPStOhb1l7OhVB6WuUV97XEgmLMSNs3v6mSYb+FlYMsxQbKSoB6ovJKScvkMSaSINfefaah8fZW+PIaB2gF+SCCHjQbTPE\\\\\\\/32Szy0Sv1+0AZiDvDo2FFHJhmbXTqfH3IF6909Fd+l+eAWDfG6SoyhWB01lOvcTRPuQexcJ7h+Nlb\\\\\\\/xIrtx\\\\\\\/C8mzXUMJxJIXnn6p1fhySSOoR\\\\\\\/AZSv0xJdhJuzOepQE9op83+ZicYb41nMLBIKvgx+yOB7cjKxZVOtu8Ab91ZRbRuOH0XLNQdap0Wbl8lr1ye88zZEZY28rFDHxjPc8154kkI2K370N8+ui+PhCl5iiIIomR6jvagiU6Vr22fxFm7RNtBluoKjlYctXp1ZVPX83KTtp2+TE5KwhDaPPC8181cYhnHQN0OEczf6Iv2j3EQQnXBjCLTTnpl0vDe7kQ0Gj89HhOJ2emJmrDmYLfLdxjOkwHVd6Boz0aoe9zI5LKqkmW\\\\\\\/yPj\\\\\\\/AZqzF1f\\\\\\\/2YTd5HOfP+5M2EbtrQrKmGH8cIGFZCXvaLEXrhQVjEiZO+ZRNo+4X6AbcZ\\\\\\\/f\\\\\\\/Q7BLzBSn6hfA+94UEjm56oWNk0hyIvlIAJSwXiP5ZuYkG629G9WX3anuK4DAUPSv3KA78ykLgSa67jlTpknTQAvPAUo2Y1cDwkIsPC5EhTY8ulZpw+0Q21CZOS45MnomoxElSR2kq\\\\\\\/LyoYUwR7qR\\\\\\\/nf+Mt\\\\\\\/vvIEwEFIza\\\\\\\/rff2ZSDNgXlLEMY7b0eJ+G76nNYlg1p3A5ocI1BQE3dwmH88LmFx3eEaUZhRwEgDZmrjrl9VrBi3n58XwAJOj6JPqPWJoqBf\\\\\\\/sNXdH9uUrQAC801vZ+MOPJF7NgLd7y+FP8C41GaWjTZJQ2p8cWMg9iPzz2JD0KgbpFPRSZ6lUUsDr8qowC3JiEg7U1Etq4OUFYXii8EzmX\\\\\\\/tRXSazjoPFZJjlvtdAj3ZeFrsem5t6G7y5mKUy0BCC9jFJGTYorM+NMSXLA9XtkWmTZH1vo7VzG8\\\\\\\/+unEJfH0cR11fgRPwqItleDMGLQBvQS09GTsQx1wdiaUtQ3YDYK4ljFNvzfg+vallCa2qvoeTL760U6Y0o6WXwDqxHHDE2JBFSE53yVmHO7NdD5m35xvvUcxospW7lTgWFH19Ug61fQKNCp89NeR3e43GZMaDjODVmvPW3kLoIGXLWR4D11roGSzhMEdMv0AQF0JvznDUoN96jhlsaGx0SQNo1I0o2KWcUMp7inE6ptqeEe2kmYcTnnjR+O3MxpR6NfPymuPG90QISYmy5MXCmFGjzziI+beRDGMBBc8duJjxTlPgJmeibL6329X4Bdk36Iv5djM3LJ90pyF\\\\\\\/OaPNwHO6RRPYXRniJ9ZFWDsG7r6M3KVCyRNK0nxM2UljO6Rawijn0Y0Wyxx3RHKaTd\\\\\\\/+xZDI97G7c3uztQnMCsMxYAW+bZaYIO\\\\\\\/YHv3Ik5ujZvfVf5S3EKJwGGADXHSBwPwLA2tCPPfKGsSQHZSHa7fflTLmO1EdSh1TMB\\\\\\\/Kx2XC\\\\\\\/mdKntCLsDGufJ56C0AMOPns05XflAYDpj1YG9rsAo6YdjIWVWzsv+ssYwbI4RBf4uuZB1oIt0UYwLvlpxGj4TZRF6IEXT+x0\\\\\\\/09FcCi5i3qO0vFqg3cdusLcjPj3\\\\\\\/x2igBUPw7roC3Sxl5d66pRtZM\\\\\\\/YdyA0UCBSeQCEh2A4S7kqVDE30kAjGHLE7ch6YMvuyQevhbQmU7Xlj\\\\\\\/IFTKdo9a4EfZVm2DyOJ8kZVkYCzfoKkupwTZuma6WMRc9j1TYsMPPMRNshGsIuRTBJCAAtt3Q9UnVLiupHUnPakna+4VtEuNdQCZ6AoSD6zJKr\\\\\\\/Vh9Id5nFWOnPM+IXmD03r11TJis\\\\\\\/Gu7NbvCJQ23p0pTvZC7+mYNWsGdb3Y7NeUHH64grmCfvinawUqJUMBzuvuXN3glvGjmbpEJiEI4ERQEnFHZAbVgAvMaTAGnDrvlU0EdJVVxVlta9PrqFNuBa8ALtQ0ihV\\\\\\\/H8PEwVHUyM1eiKqShKDPxAQVDT\\\\\\\/Q7RNEQDRav4SjZ070LWbaCfOULqIAHFMeNSDT\\\\\\\/vuohzbPogTuHus19\\\\\\\/aZzZN3sovQNfw5gOmGIDPoKrdaTcz6utPPJ2aACIfjK3XPVAhsTPjDfp9Zvu50SvEiDQ7ZB9hCxm4bWbHz+dGVNFJt3RC2\\\\\\\/ZdeBMg974YtGrcAlI\\\\\\\/kZ2tsv381uxeJaFazSY01FNiRPN7thgiOz91RyFBEi6Ol+pTCeCVOGax\\\\\\\/zrQV2mym8CqfXfJQzRiHxgm+ID0YCG+np45hqiJjpeFNVFgQjeZ8DkVlB2pEN536WcNwNRQOPslk+mJLI+q1H90vuZe78jH\\\\\\\/sCsO42aKYQJ43xJu6RxyYpQFAs5mnyslwSBGj2vSyQRV+ApOt7xdL91PWqO6Om2PGp0f64kiz1bVUnuTgora3q7M69wYGj9KlfxeNji+uSyyEVnQ2nwok0KEeIGPpt\\\\\\\/47sxjlTje3\\\\\\\/CfBn3OKZMlav8bZmjup6LZuvfHCbZXD6gBzCBJu3dOkWjsuJkbbDRkU+tWBWBVXZS+D1K9FA3SEbYLluf3dBpXcNY3Gshw4po8ASictpB50yoFGXUZzPP5dw0w\\\\\\\/bWXyqRpjmZqmnppk5t5PVO8dnO3gCYmxBMAXGutG4m8ZTKdxpqIFEku5\\\\\\\/C8FskPa2yPVkHs9Q9LPy0t4OVl3DWT66OP7ChkUgRdf4NMqQdhHRXfZof3yh+2z8jzTwB\\\\\\\/WTcaFewnJcoN1WdfY3sJKgDQUG9+39kBs6vb5uXl+NDbxb05T+6xQkt2yBDAj4IZ7i+VtM1EGjzJh7ZwcuIHmlNmrTsEnZmrZFQu\\\\\\\/FMfjqp5sYLzoIeOrSXSRy4onha2t8twdaOekKheYxI\\\\\\\/MyWqWL1jKTquOL1F1fildHREqUL0iE9YRO1RRs4cR9O4SVsfmocrYbOHnBJdl4WDnwT4BKcrhoMsdoldpmJufrxFUngUsTGTGvssoY7RMD\\\\\\\/JjZX9guVgLXtctrB+YkfX6qCsADytHOWaAPoKT7DvhHr7pfIsxvcMs31jgjbt+F1pXTcp5gsRVH5wd4IXdt4BgSDSwrW8iCA04BcdXL3UWLKK0yDgYzVsMmOW3zebHZ5e93AS2bv5zr2s2\\\\\\\/yUBcZZuComjXLu1G6hGtzRK86+4R2qhqlwDIBNxIsjZEP0Fnji2zNCNSTeQI+Ys5YaU\\\\\\\/vAVz0wwO2C5Gm+sbqhCwNQdIA64PbKJ3gFrGkvw4QaH5i0rel+fWwrvJmSByPz6bpSWIVLU7UPPJbD+FwSdH5vTgJ+gub5ZzqtY+8dyxbAD0I7+rQCwXeBAcyoB56eYMnkLO0NIWEl7uLrislpe1Ez8yHOGXoUkAg3oGcNPkINWl2NHyGSa+tJvr0mkyp\\\\\\\/IzOIHQ3yuAGFIq8FcR2IYJcbPMwyJenAZHySeRKxo8RZPL+CVy1Ge2RwXm0kkCGYJ4N\\\\\\\/3oPDz7m\\\\\\\/qBomrDulpF7KVMti9U\\\\\\\/54oKaK32iYgfkzJ+HLhligTuyQxaNQQo1OU7iFk7LzuEGTGYWz+vTyHeg4jtsc6OVEz9Nw9ZQ3ClUasSJVsZVh3gWVkSnGraIj7L4GGNJsJpOu2BBg+iWuO6PqtasfkxgxeUzSoNUpyB4QxxfmbpfKxuDQmfvVXOojdp54iwQheRq9Vwk2gc1AJ9cuk7rT4ZqJKjZQS0\\\\\\\/gWJGxb\\\\\\\/Tvt20VDqmfe9qFV4SVkIcEnNnzklKodibrbqPQBf5\\\\\\\/uozmS6Yt4m6l4JwpLwwUsAkQo7hTyyPvijxPjyQJKNYxieMIrOvJtZUjSSp0Ra5z9ubfz7xefPZQ9ppsTpBvlOIq5L8MvP1FBe2C4vUmg4lFoxneyFhuElt5RDb8HFnT\\\\\\\/GDklnxbArEoFLG+tMpV5Qbqus65jdWmTdSR2+oBaYY6kNp4chwkkcz59DnTgo2\\\\\\\/0687vghyFzYKDDD9i7YBaOi4qFh7wBx2KEec74\\\\\\\/nvcdohtuZZvlAQeS3tzVGiO0WbQmPtoyO6js1cS4RCUAOSD2MFE9xL5zBCi8JG897hjMHVePlawiMJk0G+PQKRmYHH7RjbIHxeZjpAWYY7YyT9cOi\\\\\\\/PyrbM3nwUpHkP8XUPjzHe2cPOMifkg630+wO2KnkE9VQ4HYzzeTP7ubBPK0lxv0jRvBzz66CFNJBdVW2cZctso2GEz66PJH\\\\\\\/XvYo2rZ5ClUrIW0cfX2rPf9B6pPfHMeVCLj1gZWqHVpHDTNtet\\\\\\\/KV+3eR6RlhMEP0Wx\\\\\\\/jJ+sVftcpeLbZJ0GjqjSQ3SPCfNdzfYX8ADs5LHvkW\\\\\\\/W0ltrYXI1vKncpRd5ZrWJqzOi6N9FbkKp2bTPGH6ta3c8TBHt5cmIm09X2W48CTJIBkq8pHOc7IPnddM0V22V2BTLoeA\\\\\\\/dPAqI74JJmUSCVAiTZ69pnuea8VRFS8+7KL5N+UNAT0iUkXKYjLIe1n23xgaJImqZ3OdEgsmVkT81+uW42A1EwEVam1fWNZmV4uS7iERYbV8kuxIxmDirUsF9TXEHTRlbiQfIAeYc1lC1ut2Z1Hs8XOzkkLOKAwJ5R782OnZYdLGRWyV30MVAkZZI1mf6yllKX13+1OvlmVBeFYjp5RAn2kz6RMDaldQfeJvF\\\\\\\/eGR\\\\\\\/bnFx1z66wgwwgBF+MwtcLF14\\\\\\\/R9oa3qzVOx95wvtTElTnDhuwo9Etel\\\\\\\/0u01\\\\\\\/cVDmC9RUWoO8SlLGAfDBcGfjx2dWnqsBj1u2k1Trcyj3eRwnDYbpnwWTAbZ\\\\\\\/WNIcACRs0KVYcoHARWXbxl\\\\\\\/obnKZxHhuCultr\\\\\\\/1fL2eOYNTX5i4LxSA8cpz4G88dsjMh0e7fTH0odooLlaToWcAUz7q\\\\\\\/76AAFzuhXPx5vcIsLzNKsZe\\\\\\\/xUQa27r9VkPB65hyxOvI2myj3PjFeHqLQosXVfGQE1vDHYPwUBMZZFTFOgWwuJXEwYDwt+YJfLWr5bro1aWRs4dsC0idAUfZbZlb9LLEqErYCDJIFzHy78OsWJIrXSq0yK5gB8lwujfh08gYqni\\\\\\\/\\\\\\\/SUnk5LtS941m33R3NLCFycoCmQIH4vzacvjeS08aw6EK1lU0DWzDc\\\\\\\/kCDVkoHLj0zRY9CzO6rO8PEjqqNBhB2vsvv\\\\\\\/ZS9ZJR1jHV\\\\\\\/rytZj8D5F+\\\\\\\/cgANRh4s7vIbLBViksz9AKUA7vwpgksiJF\\\\\\\/QpuUEyfCE2xGg+37dD6pBiUqlLt7UZQ2G2dSSXEuq9DQ0r5UIZp57Al12DL9eAKVRMGqgD2PPEK+maq2Ae4KGfHt2XE2rzaLH7h5atSA4jmUkD3yPzCnU+qn3H02u+Y4O+apG1d\\\\\\\/X9zWNYOiiwDh9ZXSnhIdLRLs\\\\\\\/c\\\\\\\/BprMQkOnIfQ0XwnoDjtP2A+\\\\\\\/tq16mpEgFyor0ECZf8uf3GznK0mFMuuKHX\\\\\\\/3f7FE9eoFKrS\\\\\\\/xOXqvoUO19ZH4TqM0p\\\\\\\/eT3ihqZp18OlqW1d0Rx8vPGNRygPRkIId0uAZLTAU5vqkVIpKjse54UVHLpCiab9KvdJBWmdbpNlnMeRPCH56F57u7ral6CqtBBzgZfsQ7ZCp5NZOgD6Y1J9ZrrG\\\\\\\/7T8k7sJL6GCFMIM2UHV2MxjIkDXTScOBN0emkFv7wZnFirYxlsXf4\\\\\\\/gjcl7O959RSMx3rEUglEb7bK9gWFHqrhTUTNgw6n2E\\\\\\\/OGQhf1ztz4JYG25\\\\\\\/UAIcVB+r84wx8MOEJYtPBSkz+7mz34poNy43HqXTMtw2SoZES2c0wO\\\\\\\/r+TUZlBmsp6PaDCxhyufwfg9WAuACjWlzoYwyqv8nLOhgU7qSql3lnuoQdDCWDjKSU0lFYTFQyUVmpd5S30lwm95bs\\\\\\\/Bo+5ru15\\\\\\\/D9hnVBMYpuMQndSAz+S90SX+WTscnrasGtgeHVFpd046INeE3JDf4KXGmDSto5e0gxuyPSUrpIcNgtzBX9BWBFejHPDCw6TVQIDMuHlpMVu4p5uCfcbT55CBWBAhM0cnPGKkSO3EaLZkBAphiugt2s\\\\\\\/1o1qwd5hVSg2Z\\\\\\\/kA1A2rltyX7EjDsG2X2WHSEtTpst56++i4e504Y3ZmFhamrzkB3tEgAAkd4hEsMUHvT0NeRvy1meur\\\\\\\/sLLp2d+XGjkA4pXxtbdagRF0+0+QwWI\\\\\\\/WwNb5FfeRnlgCE\\\\\\\/BKHeBZm5rYQN+Qi2GjtXWq7KF5k2aBDKej+axEX6zAg5Y1G2vfj3jGfkIlNYtHgFUnH7zJcDSHWtg9KoONvbsRtJBbsPA\\\\\\\/lWBn3cNTxm5oHWo0Uu1Aa2\\\\\\\/zMJClW\\\\\\\/1g1wEFsKpq5aZ0yTbNHvg99EIBBUy2wh5AkHgxmkXQhzVVWOQwBaXaeO8bpF99Vx\\\\\\\/ECVOEBCsGDyMfdm1ZWTKDfRQRHwEckv\\\\\\\/QBw5gWY8uDaEhPpywg3jfA\\\\\\\/VhaEuuRF4xfN5n794JqTe0rqwjXgfZDUg0FXS\\\\\\\/RZ\\\\\\\/gn41GInIdD\\\\\\\/SPjcDeqc5uMbNa678ltgfHT9A+OdkoVzwcmUP09hUoeqNz4Sr7uAblo5Ul3gbQkSYiKQET6ZY\\\\\\\/JI8tinL18OsR7DP1MiqTkFfhHNYQQYvipyeR8sHomF3x3Uw\\\\\\\/uTyHd1N3FSyLH2+aXYbIUhBPjJ62RHre6iunJ9IGY+HyjbPuKnMiGyQZGTMNrExxIokA4edtdfGAAFqofaunaSx1m5HYQhesDLXvfVjiMYTcm4Vvbk1Zh2vpVQNyN6nozdGypb\\\\\\\/Fl9St95eXvqCintVkSe\\\\\\\/S+D4kATrW4MbaYYJi6Gxbgt9NDdCZnJSFb5BDZQ3GlMMK7peCxLA4FTOKSv7TStbm61vROo09t+4gx4be+9RDQ8QBHolYJP952Xc7WJs6iISgD3pbERum31ent8l83zNvfUj77\\\\\\\/RapjTHwXynNUA\\\\\\\/ZinkWZu+tZFHSD3wRxEUB0aMGYP7pZ8PBq0YP4bSrkNGOQQQdIqj0Frp8t795sjd1WM3tUIpoJGxltSGyW0hS6eBcMTc94\\\\\\\/xx5sY+V2aHvn\\\\\\\/KiIwmBIblMFmTzHEiwav88E0GKRfBFHeu54FpvBYrySaRnVWycEk7Hl1DNECwDNW\\\\\\\/y8Uqi4oiGJ9wNF7h4B\\\\\\\/MNqh6yDahS6JSfhJRRn5\\\\\\\/\\\\\\\/HqvakQkw2o409MqtVt8YT4q17la+IUIAMxu5W829VO6CKyZif0DE38mNXYmrQITPJovydyAL25ffFlOqcmJmr2ilyx17iZ8t8216tz3DCaQFfU3DR7nDmQRj1xlSDllVXEx9xv2yHPIjEwilFvLWrTd+umW7ThFH8\\\\\\\/rPi4\\\\\\\/wZjqmjJFyMtak3kL\\\\\\\/9vgTd9hJsWQZi2zTzgK3OE\\\\\\\/8a6c6mse7EOsZoZM9falf1WvnMzpL599ch3CIRI5CVCwfXRMlb2bq0W6ZUsUUBqjtIX5hQLcQjpYquUcI0fflvWyxyPP3qQc6\\\\\\\/CivIpnT8VbbSFnDAaQuir2ArKxhHIT5EVeSDXCvlJOz36rou4FnLtAqHmNoLwaeVDxztaJnZQmH\\\\\\\/vT\\\\\\\/\\\\\\\/5WZacPoUvZR60doCzvK8ixq2YmzfdR9yk2RVEZwwsDty4zycddLU93MDYbnZ0Rj0asJTyuETyyf6OpapEmeA9dZePzBTSJpAJ8kwP1RIqdPMRTQA5DOuZcgCJuFZkOPVILC5d+gNXhwRP5WwoJPX5uZBrDrUiuzt1t2ZkgjlXYWzs+miMcAIZXk1RhvP7hNJ65RlNzNOdCST3\\\\\\\/opOa9U+OazDBhcZRFcXahgK+6SlaUwC9aM0sVp2UBS7JuJoyxptSKzHui6PwsS8yo7wAudBxvhlsV7wEvBSPMcTbd4URo+3tSU+gIXBB0q69P5rLfBjWD\\\\\\\/Y3GPpfmjQ+egr5bGbuHLvexmChefDBpWbm+5NUeS494nUqdD+8thB5Fe4VjiAVKsSUTEv3eHM+TTrfTYy2\\\\\\\/5Lx4sY93S7h55udFqlR8ubY1TwphO5B\\\\\\\/RCd0xo+r3QY7+YpYlnrJfhdGWM0+GsNjLmA2enyp3tA9JGABJTC8JDia5tPYme1EgRBNWYyXE2tGlZA7jo2HHJxxH2SzPoOTJFeUYEVDg2bd3ay2s2OKr0Vc+FLZGbrUReIK5SbgMbv6xulQsKsNTU6TDYxElzpv7UXO8dafVDhEgTpfnn4ji9nobPaNAPEg\\\\\\\/mr6BnzGsAEAI\\\\\\\/My5DtzawWK7glmQcNN9cLHbr140VmefeL5qePzs\\\\\\\/oWMA00crNXdqVsyjs3aViAGLPpWI4Q2ysjJAMyPr6obE6CoFWkC5nyFaXxUB9xY+aBJoK5JAh+dcy7\\\\\\\/WaA7CQgdVuyjwMFnl7qBCTc9eVqxzDWwx9mN9EUBdKfBuLM\\\\\\\/2R5cZ05fd9uvSwMu9mlY1DavP3Fz+EjUT5OUMvbto4Ajaa54nxpSNVxxaxhg8qPqNG92eOOBAZr85+zvHMrRsNPcz0DIQEVD40FNS8GEtAnsEqmARWooZSH7VZlbXcvILBOJESjeYGLghqI36w1hcgNnNhi4T04ludwMcUzt8taqrx5LYoO6XPPPfwayN8O85IqjKO3qiiLhxwIqY5tx52fWLusi5DokHfVUW5Auaj3H1GiAl8bRDTD4UD8G9KkuDDQd8wZ4vhH6mVLhLpgqfFnkm5eAzvShrq21jMSPpEoATSwLXkRl57EU49xeiMWGqGSL7Q2SWI1Mt1nQxlBrLsBkGCkynwTQ2n5LKrWYZvPzNcAno+Uyl98BzGWc3Iow\\\\\\\/+Om4KHOIyb2J23Ou2zXSabh+MAfoPY4nNkZ2DuJ5NIxRsbA53gFEvKiLyUv9RtPZ7+Vo8De1Uu9B667DFcjTo48TYmmy5niT7hE4wrT8HUffp0JDivmQuLOML7cxgYHBe8EWKxNbOnBN4c1iUJa3XAID\\\\\\\/CkwiazvzQs6e+zyDDk4X3k0FUO7fDu+JyzC1rwUXPNe5RWhel0VWgkhLne8Q9zADjW8iKFj82JV058fucZnkeSR4aE1B+Jbzl3UMK2lU0ATbHcfyBdkPCayo5LS5Qx6IblFBAcKFtUVoC08LlhRTUAxn7FLLW91jJJz+OZ7ExSGi+TYOn0gRtmMparCfUVdyGB846t1ZoFdj\\\\\\\/MsfNtpw4cR5Aahp801L5DKIUUwRKUm3Sb0FsolgflE8InE\\\\\\\/Za5EDYyfmqiB+Zz8\\\\\\\/qb50Qw+CBn7P2wNwaxKMzYv52Yh2mEafRpwEJErsUrIGptVD9puGObd6FnYs+xZLBxsEpFHe8CfunIYII2kHZtFD\\\\\\\/6V+mUhQGILG1YOPG3Jefy8fABivV2hBONVjgCDBFi\\\\\\\/LqW04pc2eY8\\\\\\\/lDNY3aMgprWu6+QnT7vW7d53zUYo4iwbgTxVzQ\\\\\\\/VypP7A7xxQrPWOyhj4+eAQ6QL8k990GUjEJs5mHrz1d5dmOHziLogyTZa9SxAHzGVdv0o8MAsX+FSnelsr2xArHJC1a4kNjlNYQq7mUfM7Mst0dCaMLmNX3CYbziZPd1\\\\\\\/kIfstCBfhc2MsFEHv\\\\\\\/+QqTkKCDK0aTGOstO6Xtye0P27\\\\\\\/ttw6dxQxPqcAzuREYD1UwVcoLphO6JfnQtm0+THCjhcuDfqknkJWeNWETDAHE8IPxzuPMdORIVBWW3B4IeZcT7S5FGvi4gEERkXM83ULnarO3yS5bgpFvtq7OglWuC6Dc8Cigzem+6Tua9qB2MWSF9qS1hQJ1+ESfjIW6o7prlfBEMbq71OEjQBuz+0+7rJI5403trOAl7b7PTOev6jzVto0VkfSekXoNPDXy2PbOnTaVwBVcEhYm8MNmoXDa3BzfvaPyWhVaF01JfYArFEoRhsx5twy6QZqpE326fJZVAh6lgqSezqM1Z0YfIrkESFWzqsHcznZY7oTJ7U9ocMBB\\\\\\\/XLM36n2t0JRv9pDcHPJ1OR\\\\\\\/48ufZhE1WkbFU4FB7RyGA0pyYVZD7y4qTir2xfrIBbONdWlqiEO2nW+c8Puf\\\\\\\/V\\\\\\\/4ZvUCuHTJUyq0rGPWOGnDkT\\\\\\\/OwlDzA1P\\\\\\\/41i0j7Sc9tSrCxW\\\\\\\/CHQ9AEBh+QAlqSn1\\\\\\\/PZa+IXm\\\\\\\/Q+V48lCe+ofsRYiffF4KnS7uXKeQ58YMqQySVm2Yz4RfxrRVaSGFdv2OWm5rOrGxAc4Ocdo6L05ADm18O03seefeANgRJqwneYWUUlHjYh82hVkMxW4vLe1QugUg\\\\\\\/+EpQsZsS9yeLkl21WRDQeoq6b1g7+GWofROq7y\\\\\\\/RqAhKV9383Ijzhpj+9guCZ8uEt0XuhAHAA3baBp0LsZlO7NZLu4X\\\\\\\/cwmJvnrLR847tP35xVU8GIaKqiEPWe4j5eaiOYLAfLqcsPcOGQ96mo940N4iBBpzCeAG58kOUR3rIe6c+0gSb\\\\\\\/7HAdxE\\\\\\\/dR7Dq6zzfPiXwPjg76Kx84SHGf2HTGnct\\\\\\\/HuRaAuCOWxkto8vNyrTEp81jYM405vOKOomkUKr7S9pamRaE0ehOtSwKOKuut09NH89e9eMZQt0kf\\\\\\\/k+VP+Rq6sVTh1uHyBc6YdP7M1+uL7kESIeK+2JKWSMGnuW94um6Qh88nlWgYFALmIUUx76KtsYMQIV4DnqcH+096VBzGai1neGdOmBu0GOeUbLvZKq8gPU9L3u9lYvgLXWZ0Fass2QuKLWBX2U4HjOnGNrGRpPBA1\\\\\\\/m1RM5CyIlbrXb7wA+Xqh28rq1AyNutaU95n3ZbiIzHYI1HSWoqkp16vFfdhYcuQiHlUaeeZWm5H6OEQZxhSx2e9nqOlcb\\\\\\\/czQrSdh7OiQo3VdN1CT1tmDjvLepfjmFS+RELC+RUBC1ZHgjYToP7xDMENA\\\\\\\/bYitKw8y9+QuBc2e28p6nHs4ViGZia+dzu8Mf5LqGEKUU3I16cGnz18PIiU\\\\\\\/qkNb+Ho3qd9TRLuz0mjHvF0cGTvn3tSpi6tDgc4K6QHW1JdIP7kFKi0gtN5+9VBIXXxQksPh2\\\\\\\/7C6+Uyt+gurxpXsSHNFrWuJO1y+d6aQDplu7NloX5wGtLFBHPtw5Adteaj3j1pKbeRHWyqcMVreO5M7+YkWyuxG+\\\\\\\/\\\\\\\/tRLsvKgc5trTD1PTdLRx\\\\\\\/+IZkdiGrqLgqFhEt7XB6uEeS1Fc7u2cvnBFltaGpLlGXj+dObW7+gMxiROs4H3LBVi3IiQvtg94qzLOLtRvOe9gFiPFFSQ4Nrgo0n0ifis6RNAlrwP9JMSoeo2ZeFjiJ8MqJ57pYQV\\\\\\\/BidSSgTX1h8+WJSOeGgzaAf3TNUI2oFEJ1fPlzVchTYtjPbnA6dqNmWDqI\\\\\\\/jJmTQGn09U64bs0RbyS7DijW0ZxGmdk8AIwicx+q9RzsHxVQ6LeGJgr4EsoRz3pxSlPUOcdHG6EiV+8Y5S+CZ\\\\\\\/eXvJ46qtcP+RrZNtwvGz7Rf4rwo2pu+fQQRBgyCiy\\\\\\\/W6juwqSkZJ7x6BjFhFPofpirUOfgyOmzKGb2ho7NaUEs8O7oWcjWPEKx4kpJ06ix3Uf3510Jy8FgaVKJcU33jWx4EpMy0KAnEdiLbkT\\\\\\\/yPONfgbmv0lxZeAb56HM9QKBqFm2tVCR0Ezn9qK9BTXxf4LbYyyCmz9X\\\\\\\/AloFGD3FD18pEvvQSeYTEK3Djux4POiHZSKEiBXlI97TqffEal6JVmddRgXsEJhHWlp9cNjF3YtwmZHyQLapbBgy09tzIqIdCnW+Y3clVFYHbzh7lwSXazQmCqVj9WfmacBRmUxkINSwDDr3tpMBmf+tamN3hwvLjb7StfcG3t86Iin4RI+MXaAGe88oPv9rbX\\\\\\\/MRVQP1czb7CFuaCkfZIihhL7SlJQiXnDel6h7L2F6ZuXEgJZjNQkEeIz5sa5HnBy4MmnZqMpcoqSqV\\\\\\\/M\\\\\\\/QMIXqrBfB9bWkc87uj5\\\\\\\/jSD69vmo6DnIzd0ozpniItQT0SiA5KQxIElgBknmSu3Uk9XezNhoWhm4o5dIRdC3ENdnGRSfkXYm1WEJ7FZOY4YJfvjXGk0UItnh840gtkrybavkOLWvHMjvUr50rZQHVRZy4CWkHpHQ7\\\\\\\/qFms8dcxEHb\\\\\\\/ScUlbKBsFBKmUEbqyCKz13WQ1tfUacJ+qZkP7AUUYhcjWbZ4uu5ntL5fxS4htEk3kjBvg9xI8Ez36wH\\\\\\\/Tym\\\\\\\/YFyXA4fIWUQ4OIeVwqqmfbUB6v\\\\\\\/uuQMGFmLXGZs7BSUmrB9v4LSsIurOroUXQVVmJWVgqQLpx27icywcxICe8Q2a+Eh6+No7kCYyTzXT8+nIer\\\\\\\/Gp3RJY74uHtR9\\\\\\\/bzXgXv98F82qKv5MjrFSiL5LRiZfRBwV8ZIuQUXuCb01CVxPgx7gvudjSA57qr+lFJdYM2IjE7C2CP\\\\\\\/KqdFXrRJCkB97q5B5v5XU933cQ3P65xpreDQ4ihiu8XImiS4RqTcB2a9F6HB8VO32+CS7MK980B1xxrx\\\\\\\/keUfjGW2DfuVza9oJpxEtRC2rW9pi77WzTkkdVwezT0SgvQeuncRgUX\\\\\\\/h0mGXsNI3ilrIZoOmdSSfO4TJt9JS9yfB6N1byvwLg+jlN\\\\\\\/uc4ZaqvUjATAfCWXElWuzTeWJlF5NNl0NLjzYIs1yFZ6RQVgMP7+rBufu\\\\\\\/cDswl84yZVERX\\\\\\\/S0Fl3rM1GOw0f7iflxVX1\\\\\\\/DSUovScORdzah7cYFTpB9PtOrowTUzCPENtjpMW22brR7imevAW\\\\\\\/9HmlNmpBC\\\\\\\/Hyv9qnDwTWTXq3Kfpfn0Z0LnHOx3xElA3Tk54vzXWCcPCcsGydjXpV+UXcqXjLDEpZdd67SwF3Zua9B3H316GBK2zlmp0WfeCnKBzRcxYkEDk1wiEzsN0m6TVf+SsBkc5M9y3WuwkU+Os+FCOA7SB+DhNUqYxZx8pOzpKM0XEirGn9BGrn8sqDGvBazSs7oUsr\\\\\\\/zJhx8wtPJSy4ABYNm6Ov0xLVpHgVuUq3XoME+5wLg\\\\\\\/q5gF8bLtSV5l9cQWkCmqh+fPQESFNyYnuSIFJ9ghh6swXdUTCrYl3ziaZG8ZArVxhs2WbP87sEX6eO4BGdy6ivTlIuERgMH7KvZA9gh9Za0eJxk+UAiIfa7mj\\\\\\\/i\\\\\\\/jmS9MOphiCKRPsQYqVNrw9jNwLDjwZVNaVPTHvPQSoJA2kPQMQhstZocz2f1rH53t+5lXmNj8TRX1U4kYhzQKw4UBvYQE9LIX4L6Cw\\\\\\\/oQJAssi27sVSACpqHOVF7OzpzUgf821aIGDwaxNLACPMqMqNR8AtjRVYNIpTh5Uv2HwSwFXi1Y7KeaPuYY14pwmbZ3b1RoU5pjSp1SNwtMIKHj7ubqfvAlgC1W9tAq9wS2yISsODCS88gdnWlliFczi7Zi1F6Kfb538OjWQFgQgsZ1PNEHApC+HOagRE0xiBjXNLWR2KdzFnopateCHvlrnqBTgqGpdunCfzpsReEVWeZWBBCplUuyCOhLIHz4OQAk2e9UzayRHA4bngdXS7zq4VzJEtJzAp7YYMS5cCot9dKq8uDgfidQEGCpI4hvKbvfCesHXGDIa9L\\\\\\\/nv1X\\\\\\\/C10wP8pdtKvYFizmdp40sc7hb6c4U1pMuZ1waMVYfHBoG78hVEPna9x9lh48wgVvIWvdyUA5HZNAb9OvREWuVkyaPzprpl6\\\\\\\/1Q7R\\\\\\\/u6nrJFI2Kh1KdbqPHkBeugdEeA7MP9daB0qY4Sq9+N9dwUIICSHrTTm1m5dazPRToB6TOX2szwDZQC0oy9euUmSDQ5PAmPESXzPXvtSEnoCT\\\\\\\/cOXdY\\\\\\\/s6EuPrZaGPyi5lUvYl68CJAdZ6ObgcvkLcAKfZzEtV\\\\\\\/UVGie2UsLUXln1CzoQy6qQDbXkntQN2Ca3GzTt0E5WFX0aOJ1BVH4\\\\\\\/LVL6JeS+i671pu6dJMWy8ZPXgm9A0QibdNFva8\\\\\\\/VJXMlPHF0laHVMscrvYBpHch+w49IpadxwRCWxY72NTiCyNBjLp9JO0w\\\\\\\/TiiyhTxiV6+ERrV5IdbsO5EhPN7lDWYTAlTLhFqNSGUQwMCwM8v81jeWCDiyXaDMEQl8r7U5yA34nAwo5vewEL8CsMYU7pY+wPXNrniatln8hMzjGML1mJTCXh88fsAVcxw+0imiCHO8Y4+vWE4qN57EZsma25G8+qJK8YeUQxJFVeoPtfx8kMzj9vHWogy6nWOL3v9fphQIu6JRsvPI9qy0Qlfs0Tmb5LlUz+Q8vncQG57rGU\\\\\\\/jT2n3735Invb7UKGRy7m06F5A6vt9yuEU5w3snLayQan6i2U0L2nNjcyEWW9O4mlmWrB4XaY8sFZlqNjZLcjH4uz\\\\\\\/Q6VYrxYsNMMYvth\\\\\\\/MKQVRcOVyHWJZ+No8taNvSuBWVdH2fjregHMNx259yV32b5GzD5qnobsruO7YusIsB8zcIVS1Zl2VJdsyd2\\\\\\\/fORhGHBHUIribQhtGfJGhVAnPJSFBUelk3Hg\\\\\\\/7e5ctor+DaZLWDyMcN0\\\\\\\/jUYdVjXS50MPlrWI6dRO63LazEwJaDQMrjVGSWMSAUvEdcICD+3IsHcEQ2882QZHN3s4jJ2Vz1M\\\\\\\/o0G+FYq9L0LpwLAnTzCXq94U980D\\\\\\\/juD+3n3FfKUKFvTkVZ9ypZwVW3EdveigLOjic4Ipo8aCj7mODAcXLqghJ6kLNKteFYQlawHZR48jpQ1sYmyPnopuNFHwq9Ht21whNBhbvB4\\\\\\\/bA\\\\\\\/dC3EuvzXcljpI7q+sY3kqtHdg8PczSdpEq+cPbnfQ71eDPgGGRQ+w31Bt+Qfr755J9kOzbqSW+\\\\\\\/V+FqTe30M7gZmlKRhljWGojvYOGO6KnQToQP+3S32S56M5lBntwYnob0NRpZ5Dxy78Uxl5l0yh98\\\\\\\/zTRfU\\\\\\\/amA3OA+YhKj5UZSngAwjK4jG2+Zd8cvEnFi18hwL2MQjJTUFQkW1fXlpZEDVg6rsXSQFnnmbs1ZMCjc+IuDYCQUcBQY9f3ZEdBrRhPO0ZMiynjW8CBtbsd9YnQDzqA8qqD8LjlS4S7ab\\\\\\\/AvqEfVRJwpF7Zy09ZDsOX+cUzRWa+zA+Byz6mKzoZvW6GmRM+ClWCzolJKIBm4ZuBqSPiPZ0XyCPtVGY73a3oF3oSQawYPlCwP\\\\\\\/jAs0uGUWbrBLLtvxx5LZARoSy\\\\\\\/9Q+SCWoyL\\\\\\\/0bQAqN2xulajMpLzLKG8rNZLOuJjAq+mKrkq1MB8kKcd+anwZsz6jIGFd92ZC9huZ82spEgRNF1MuXkHqwnVRO3FngQimRFP5ulSCIrT6snkEF9G5QqwwDqKJV0BgxAqxUzbaWeM2Q\\\\\\\/kvxbY2C1DcaLrEbITcAgKxaVXBTWWzVmF3nzytIl5lN4KCtgu9fGJlyWx4HlmrDSbHFMh8hNUmGlqbARvUSeapji85+Xdmmu+tIo6clLXmbnySveereJWNgl0dZh9e+x6RyMthmJyo58PnFsmni83qnJJtkQrOQkFwf+1kWQ08TdEbouudaqbDBVdGpMP5eHqth69XaekSskgZs26br6SbJGc5B3IOpT13Q0s7k\\\\\\\/TUlEGYVvB\\\\\\\/yZ0QbaFNZz\\\\\\\/Up15eI5Zp4Yahevsnwj62d6KN4qiWEoULjTPAQQzVbLDrDHS+vJxgdSuzhIPX2WS9qgaDd60xw794Jg3EbbXFc\\\\\\\/c3oxvFK1Fy\\\\\\\/b2ZVeweAeKT77wtEbH5LUE4rNcrkPVm1ZQXsgurcG8ywGT8jB2HH9KgfG\\\\\\\/A0OoeqxxfbqPeQwTMVmybGB90HYIjTWlN0tbQUjcdZ9Z0rAzVTwTaxVIjpRJrAR2waBeZgV7rjRWwHrBRptqwvNeMgD1tFY5dQtvD1QEkZRFN+cpkqIgcnS5bS89udBpaTxtgK+Q\\\\\\\/t9oPijf8Nm76LnO6j+6vCluU24g3vSIr88QzdKtPd664hMa89vXbgvcS0YQSH4hTCwtRur4nxVfs6t4bbNzd4bFnkm2MtCm5ly5VGdHLHHndQWIFS00mt9RRp01qwuBZcvpplznIBIx1h0RS5\\\\\\\/vRhrm5Z34r4kRS3woUxioh1M17gc6I0h3aB5GDB6lEYmmBoCFsY9U+YlNnZOYEIgsWCBCLjzRObRVOOzc\\\\\\\/iZUzGpRQpAF0coDryYR80Y72bQ2FCFVUMeuLHOO8oEddAOBZr5NSRdemmOQrQbRc1phw6s7GajS7OTVZyzK2m02p4bKLCpjGXGCeXyBGkTxQHNqHnLa\\\\\\\/6HW37NbZDA\\\\\\\/9cVpcb3iPI0hrPLutqMV8HGtumfIea2ym5sg62UR+LiDsLRjPoem+wuu4rrmVn+XAPNufszDb16kc1JAMRQil0Sa3b+aFok\\\\\\\/r2\\\\\\\/0NzYkhi5tyXbU2mGCw8cYjhayzM46F6og4th349v2KoGbGEQuki80nxvMK2+Ev\\\\\\\/4LdYgQy9VpkTXYcAw7XevG92v1kVIu11IR0DowcHOfSsHHncaOcS6iJKCiVZ7R0Vt3qrW702z98EpukozGVKR8Y4zg0T6LR0zz+yERGYa1ngPSoyEQS4VCUB56CHyFFIStHSLqIUDFC9oN+EqeeYOBXbjwq66OvMho2RUnSHz\\\\\\\/T+Dio1UsqwTM0NrMC7KwMQYKSGqIPxaVKhhc8aWnNKNX4hZnjVqzPvUFWf0Ep4SxtEREAUz9sErFP5Rq39YNaZhXZCUraqWF7DUrPiDif09ShQzQLOyTVyjvEVAM5TfUd4+oJulOlx5aK7eSAWEk1vAsG4jFbXuIU4sFVjzU4sEY6o2VuNLp6lr1Js2F9AkaFnGTwXm\\\\\\\/WArNJCor3PHirEyB8nvHGuhYQQ8ND7ubmVs4xfKyYnbCJ1bAZJ\\\\\\\/d5TsBqLbEiyGZkQ8zCmZ08D+1XSBprd+4dxNOc9V121k9z9fAfN\\\\\\\/cVeI\\\\\\\/9Gb62kQ0r+MmFv\\\\\\\/yxFDnYPhhwSDeTcbGuMPzp4Ccx3Rh3XpkpXgwC7WnbL46XYRCz6z076dYb\\\\\\\/IqxEOvkDJ3fZ+oXxjJ3tyoAbQ0jojT\\\\\\\/0FyuzflbIw6cy0dJGJzhDudtmyLI8N4T7denDGuIkZXLxpwJg9MKSiASwQ2p9rFwnQpDRaTZWKExiPb3FD9tDNMBEHG9FL\\\\\\\/2aB8PqjJn\\\\\\\/BixzsKKK9kfHKNWw0T6Uv9TqWzWhiByFDCOHF9INrpwecAoYyTWBgLI3VP9oBYAoL9dQIHxJfUFkbgtWQGKDUdaH9RwFJTXC0NlTC2zML+w6jYvhPwsib+4ts4Ou15WNHWJlJAKQ4\\\\\\\/egR3huhXxo7CaMuu0VUQTxi4pTJF9BL5Hw\\\\\\\/SE8F0usWENk4KZgQKl9D1Qx6PYudSrw9lLm2ZsXsFe4mY+OF6JdwF7DNJhykDUf4Po7VFLJTRU\\\\\\\/o5LXtHPWdT+dzPZAbt3Oa92s0bpVHaymz3LGwDIfRjpAnyGYqsSaBjmG3heYcscZkBIxaj\\\\\\\/njYe9TZAQwcjs+8rZZvTRbWwAjF9UFFpAdQ+85myb79Hf03uJmNGtIL8oEWCzD+\\\\\\\/usBjU1TJnwcXcsF7Kx8+L7YwjkzloFFjEonfck62o081lEBZWlu9GCbH2nGevWIX011GHEdSjSSrTqvbR5onrmXoKX0I7Y0e3nilBRgonIVj+cGohWlTnfPaZR6jvhasRNWA2gwZBx85W98aOwXXCwBZsGj6BZTJN7kIRo\\\\\\\/pW3BcfY8zb\\\\\\\/hPPof3+xoAz46uakgPmFpaRtp5mtisnEcO6WelowcQT5dQHJjhqUFj90CdOerRQf4tWlac9PTwucEdUPSeHoZKApmRoL7A62hwsgfZdXah4CAOl2G0IQEYTxNIN58EVTQEA\\\\\\\/3RHtMFpyKS+iydIGij+93mL1xhThU0Usciy4HQfBhwMpXEvgDVkddF2xnJXUq+2gaDy4hnAmwncejILY+rcr7DegRiv5tYJgkiqzWMJd3hBQV\\\\\\\/upDIJYsTyy32YSQ0nW5QSAWNsTv3DtmHA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wsGZhuOH6qiHZqi2sxugZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228dde41c8d917932542dbc424db73e326100c553f1175694d1397ab4fcb4271d7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ssAMnVG\\\\\\\/T4UeNlRexuf4JuoNs1P+wdZu\\\\\\\/UNqWeVH0mFzcu\\\\\\\/WtOG8Rhn5088xLcFTt3qwSyNlAlj6jVpNSLKOSjNvZuS1YYiqyF5PxeMurms7Lnat+6MaZc3W7iGQebC+COalv7sSB27yVTEPBfTQ85PkZuNa8JunOHj3YTkquIzFoOvudxHfaPvXNVrs7zjWDeCqa8dsfrkW83xVG+ajCWdGzOuCHjjLq7wY+y7Svs79BtPRw04jCjzuKXSrtxJHV4OSf3dNXV7\\\\\\\/8QFrdL+8ios85HFJz5GwvhHO2WHsB1jIt3sSD5p74jCfskunkf3XUBVDWFA1qX2kKwvRiETFaye7vetfzGa3eX2H3qO\\\\\\\/48WR06zsMzBxjoBJhd7B7Xzg3JkXcqu1EXM\\\\\\\/rIMSLJ5ERatSwNkMbiNqLacKoTMEed5IiyTa+v9AJYy64mkoRfg7F9DdPqMB1pXM2CvQzeNE\\\\\\\/OwNaul+sj7n8PkCe26tRBVCCr1J3YXnBfrZ1iTwUdbE7wnbR+5Hup2PouzWX+abyMJcR\\\\\\\/xEipysAOO\\\\\\\/aUFzvc9nmIyW7e299BJFVHPDrzcGgkVPKyjd0HVsaHarWv7e3vIPNljDBIkjJKiDvFiNtvzey1QIC1F0pZ4WRnPH0aBR33HkOaxIqVRIADcNzHRWoRqXemmMPLRZtOMd867nL\\\\\\\/DrVIfIG9+UX5WFU0hj1PSeHDL4WTxLZYyGQrfJxWoef7uIAWZXyUlM03QaThkz7y6G6oWCEAlrKdza+gan7Mi1JEBc\\\\\\\/oUTfJr4RtpwiPAe90FNpI30wkByh28X9WUtlT338gF2rlfz\\\\\\\/vm8F8t4dv4GDVxec\\\\\\\/eMrmAWJKlaYaTTZh0EtdNUwp+bPddOJ4EhA8lwvGMTeTevPMnIB0+W+XjTNjCK3YiNvJRl4iCPjdR7d8ebYwBZxRo6Wf37u9Om6ombhg\\\\\\\/WcYNYp4KIFA90J3xOEH0\\\\\\\/xR5TrjoA1C+xdEwf6cPtlTOckZ7j+5snSvViF1C8D2k1s4eCTfN7BLsrmpNCY15Sfaems9zu6KpjwyJcF4CYoMYK\\\\\\\/s2n5QyRjYE1TfQczxW0+SeHpmkncUi4rhLUNl6pqvVVpzQ8Vrrqfz0O6IgyufrIic5grJKWSg2j+h0kSuGEhn3xC9UcY424+EkFmmRLiS2bnOhv7QnVJFWmLWbiM7jsYCOiD2VxsLbWBO1P0JFTc\\\\\\\/eRE\\\\\\\/bUyPqBju3qYMRp9XCUUMFMddOYQlJgo0qKN1Y\\\\\\\/S8uXZbg6VWXMciwqFzKtuh9SENaP4AjnFRsfXyYpaL3dzbD9587rCIs1WxQspDXQRY6AqJxAxfxUKXAt9EZKJLFVfxBydSNnMtKHPgE\\\\\\\/C78oS+Oou+f5WxbJTaVcmU4Qnn4YFeP8DtnD7EM08TsXLz1dTu7zFFFhmfRRjabIpO+0OCFor0HWaDiobbk1BTo9nY2B0HwFXtYeZtFudVGidyB\\\\\\\/WNGQyAs998wbNC0cpd\\\\\\\/7TELL69H21qmuTF8bxvqsMD24kXFUHcm9x8ozdrwrou4ggOyLaLecQ8RPOgT3oBh4zg8m7l0ymEkjjbcMwxgFarYVWL5m65ID+98d0+rXdesbBj1LnrkQsOR8cM5u3YqoFR5oMAsWzpm6FsOuMiIluKPYdF1sGluXOZELVtiZCA2lU9tbUxkrOMhtGs6lklHabzDLjCVxnyl88NeHuQNg6HkhXYJkFDFPk7V\\\\\\\/GkY8Vwhd\\\\\\\/GUuyRjXL+3uzeQHkBXeBy2oDCA4Q0lmfjwLrsm1zQpDbH+93y3fzXLNtrtYD+VFsh+P\\\\\\\/P3k3WBY9uQAM59VAw9cCgfrYSmAARR22RNjaZ1VaKj6h9j4r6x8pGS0D\\\\\\\/TxoHLHbXiQhzVV9ZUefUU\\\\\\\/BEMtCP3gocKOVK03g2zlXIBYML91AdI7mGYJHJPvyYrKguJjSMIXl8r2oTBJRNDxbVeB9obqKfazaOFaPsLkN56f9D2weGDSWLSmSeGhfPo5W7SYfkoyL0XLL2WFCg9J4y4nstAU3l473A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XElPi+AqBcKU48MO6qvgVg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d16e5167dce309d6ab5f9c2bca4389b69a24d9a62cb10ed4c16b5dd41a88a3c2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495171 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/299" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-c_zueB0ubQI4znDFfzq-klXNIYKXt5zwGwJ4KbBwHG4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/299" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WbV_IFtz_wV1vQgQzJwb_sMpB1nAg7hzyfFU3D5lXAA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/293" title="Whole Genome Analysis of a Single Scottish Deerhound Dog Family Provides Independent Corroboration That a SGK3 Coding Variant Leads to Hairlessness" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/311" title="Identification of kit-ligand a as the Gene Responsible for the Medaka Pigment Cell Mutant few melanophore" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495171" id="node495171" data-pisa="ggg;10/1/299" data-pisa-master="ggg;g3.119.400504" data-apath="/ggg/10/1/299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (<em>Populus tremula</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rami-Petteri Apuli</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Carolina Bernhardsson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bastian Schiffthaler</span>, <span class="highwire-citation-author" data-delta="3">Kathryn M. Robinson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7906-6891" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stefan Jansson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathaniel R. Street</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pär K. Ingvarsson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">299-309;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400504 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495171" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rami-Petteri Apuli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rami-Petteri%2BApuli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Apuli%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARami-Petteri%2BApuli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carolina Bernhardsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Ecology and Environmental Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBernhardsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernhardsson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolina%2BBernhardsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Carolina Bernhardsson</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bastian Schiffthaler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bastian%2BSchiffthaler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schiffthaler%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABastian%2BSchiffthaler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bastian Schiffthaler</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kathryn M. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BM.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BM.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stefan Jansson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BJansson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jansson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BJansson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7906-6891" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan Jansson</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nathaniel R. Street </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathaniel%2BR.%2BStreet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Street%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathaniel%2BR.%2BStreet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathaniel R. Street</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pär K. Ingvarsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P%C3%A4r%2BK.%2BIngvarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingvarsson%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP%25C3%25A4r%2BK.%2BIngvarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pär K. Ingvarsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/299" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495171" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/299.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495171" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/299.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495171" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/299.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495171" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495171/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The rate of meiotic recombination is one of the central factors determining genome-wide levels of linkage disequilibrium which has important consequences for the efficiency of natural selection and for the dissection of quantitative traits. Here we present a new, high-resolution linkage map for <em>Populus tremula</em> that we use to anchor approximately two thirds of the <em>P. tremula</em> draft genome assembly on to the expected 19 chromosomes, providing us with the first chromosome-scale assembly for <em>P. tremula</em> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We then use this resource to estimate variation in recombination rates across the <em>P. tremula</em> genome and compare these results to recombination rates based on linkage disequilibrium in a large number of unrelated individuals. We also assess how variation in recombination rates is associated with a number of genomic features, such as gene density, repeat density and methylation levels. We find that recombination rates obtained from the two methods largely agree, although the LD-based method identifies a number of genomic regions with very high recombination rates that the map-based method fails to detect. Linkage map and LD-based estimates of recombination rates are positively correlated and show similar correlations with other genomic features, showing that both methods can accurately infer recombination rate variation across the genome. Recombination rates are positively correlated with gene density and negatively correlated with repeat density and methylation levels, suggesting that recombination is largely directed toward gene regions in <em>P. tremula</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/linked-selection" class="hw-term hw-article-keyword hw-article-keyword-linked-selection" rel="nofollow">linked selection</a></li><li class="kwd"><a href="/keyword/methylation" class="hw-term hw-article-keyword hw-article-keyword-methylation" rel="nofollow">methylation</a></li><li class="kwd"><a href="/keyword/nucleotide-diversity" class="hw-term hw-article-keyword hw-article-keyword-nucleotide-diversity" rel="nofollow">nucleotide diversity</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li></ul><p id="p-3">Meiotic recombination (hereafter recombination) is an important evolutionary force that directly alters levels of linkage disequilibrium (<em>e.g.</em>, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wright 1931</a>). Recombination therefore has important consequences for how effective natural selection is at removing deleterious mutations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Felsenstein 1974</a>), increasing the frequency of beneficial mutations (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barton 1995</a>) and for determining the resolution of association mapping for the dissection of quantitative traits (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nordborg and Weigel 2008</a>). Recombination rates are known to vary between species, among individuals within species and even among different regions in a genome (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nachman 2002</a>).</p><p id="p-4">Local recombination rates have been shown to be positively correlated with neutral genetic diversity across a wide range of organisms (reviewed in <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Nachman 2002</a>). One possible explanation for such an association is that cross-over events and/or associated processes, such as gene conversion and double-strand break repair, have direct mutagenic effects and thus act to increase nucleotide polymorphism (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kulathinal <em>et al.</em> 2008</a>). An alternate explanation is that natural selection has indirect effects on sites linked to a site under selection and therefore also acts to reduce diversity on these sites (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Begun and Aquadro 1992</a>). Since recombination breaks down linkage disequilibrium, areas of high recombination are characterized by a rapid decay of linkage disequilibrium and linked selection will hence impact fewer sites in the vicinity of a selected site in these regions (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Begun and Aquadro 1992</a>). Conversely, in areas of low recombination rates, linkage disequilibrium will be extensive and indirect selection will impact a larger genomic region. Variation in recombination rates across the genome will generate an association between recombination and sequence diversity. Local variation in recombination rates is therefore an important factor for understanding how natural or artificial selection shapes sequence diversity across the genome of an organism. Recombination rates are also known to be associated with a number of different genomic features, such as gene density, repeat density, and cytosine methylation, although the magnitude and direction of these associations are still under debate. Recombination rates have been shown to be both positively and negatively correlated with gene density (positively: <em>e.g.</em>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>, negatively: <em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giraut <em>et al.</em> 2011</a>), GC-content (positively: <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2007</a>, negatively: <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Giraut <em>et al.</em> 2011</a>), repeat density and methylation levels (positively: <em>e.g.</em>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rodgers-Melnick <em>et al.</em> 2015</a>, negatively: <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Giraut <em>et al.</em> 2011</a>). Characterizing associations between recombination rates and various genomic features at the genus or species level is thus important to avoid making incorrect assumptions about the strength and/or direction of these associations.</p><p id="p-5">Traditionally, recombination rates have been estimated from the relationship of marker positions in linkage maps (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stapley <em>et al.</em> 2017</a>) and more recently recombination rates have also been linked to physical regions of a genome through whole genome sequencing (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Nachman 2002</a>). However, producing linkage maps is time consuming and may even be infeasible in some species as it in many cases requires controlled crossing of known parents and the establishment of a large segregating progeny population (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Stapley <em>et al.</em> 2017</a>). Therefore, methods have been developed that infer recombination rates from linkage disequilibrium (LD) between segregating polymorphisms in individuals sampled from natural populations (<em>e.g.</em>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McVean <em>et al.</em> 2004</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chan <em>et al.</em> 2012</a>). Due to the relative ease of obtaining sequence information with modern sequencing methods even from wild populations, these LD-based methods for estimating recombination rates have been widely employed (<em>e.g.</em>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">McVean <em>et al.</em> 2004</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kulathinal <em>et al.</em> 2008</a> <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Silva-Junior and Grattapaglia 2015</a>, <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Booker <em>et al.</em> 2017</a>). Detailed knowledge of local variation in recombination rates can be used to infer the action of linked selection by establishing a correlation between the levels of nucleotide diversity and recombination rates across the genome of an organism (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">McVean <em>et al.</em> 2004</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chan <em>et al.</em> 2012</a>). Using polymorphism data to infer recombination rates and then using these inferred recombination rates to explain variation in genetic diversity could be problematic, but simulations and studies performed using well-established animal model species such as <em>Mus musculus</em> (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Booker <em>et al.</em> 2017</a>) and in a number of <em>Drosophila</em> species (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Kulathinal <em>et al.</em> 2008</a>, <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chan <em>et al.</em> 2012</a>) suggest that indirect methods for estimating recombination rates are not strongly affected by natural selection. However, comparisons of LD-based and genetic linkage map-based methods for estimating recombination rates are not readily available in plant species. Genome structure, and in particular local rates of recombination, show large scale differences between plants and animals (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haenel <em>et al.</em> 2018</a>), and it would therefore also be valuable to assess how well indirect methods for inferring recombination rates perform in plants.</p><p id="p-6">The genus <em>Populus</em> has emerged as an important model system for forest genetics due to its rapid growth rate, ability to generate natural clones and a manageable genome size of ca. 480 Mbp distributed across a haploid set of 19 (2<em>n</em> = 38) chromosomes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Taylor 2002</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>). Furthermore, both large and small scale synteny is highly conserved across species in the genus, enabling the transfer of genetic resources between species within the genus (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jansson and Douglas 2007</a>). Further interest in <em>Populus</em> has been spurred by their economical (<em>e.g.</em>, <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Taylor 2002</a>) and ecological importance (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kouki <em>et al.</em> 2004</a>) and over the past two decades a growing number of the ca. 40 species in the genus have been fully sequenced, including <em>Populus trichocarpa</em> (Black cottonwood) (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tuskan <em>et al.</em> 2006</a>), <em>P. euphratica</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ma <em>et al.</em> 2013</a>) and <em>P. tremula</em> (European aspen) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>). Linkage maps have also been produced for many of the species in the genus (<em>e.g.</em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Paolucci <em>et al.</em> 2010</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tong <em>et al.</em> 2016</a>), including <em>Populus tremula</em> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zhigunov <em>et al.</em> 2017</a>) but these maps have generally been relatively coarse, utilizing a few hundred up to a few thousand markers and typically employing mapping populations consisting of fewer than 300 progenies. Consequently, many of these maps have failed to resolve the expected 19 linkage groups typical for the genus and there is thus a need for developing a high-resolution, fine-scale linkage maps for the whole genus. <em>P. tremula</em> is a species of special interest within the genus as it has the largest distribution of any tree species in Eurasia, spanning from Spain and Scotland in the west to pacific China and Russia in the east, Iceland and northern Scandinavia in the north to northern Africa and southern China in the south (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Luquez <em>et al.</em> 2008</a>). Such extensive geographic distribution means <em>P. tremula</em> has adapted to a great variety of different environments, making it a promising species for studying the effects of spatially varying selection and adaptation (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Farmer 1996</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Luquez <em>et al.</em> 2008</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2018</a>). Here we present a newly developed, fine-scale genetic map for <em>Populus tremula</em> and use this map to anchor scaffolds from the current draft genome assembly of <em>P. tremula</em> (Potra v1.1, <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>) to chromosomes. We then use this new resource to estimate local variation in recombination rates and use this to assess the correlation with recombination rates inferred from data on linkage disequilibrium in sample of unrelated individuals. We assess how different genomic features, such as gene density, repeat content and methylation levels are associated with estimates of local recombination rates. These results provide a valuable resource for enhancing our understanding of genome evolution and the recombination landscape in <em>Populus</em> and will also further facilitate the identification of loci controlling quantitative traits of ecological and economic value.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-7">In 2013, a controlled F<sub>1</sub> cross was performed between two unrelated <em>P. tremula</em> individuals (UmAsp349.2 x UmAsp229.1) from the Umeå Aspen (UmAsp) collection that consists of c. 300 individuals collected in the vicinity of Umeå in northern Sweden (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fracheboud <em>et al.</em> 2009</a>). This cross yielded 764 full sib progenies that were planted and monitored in a common garden at the Forestry Research Institute of Sweden’s research station in Sävar, 20 km north-east of Umeå (63.9N 20.5E). In addition, we utilized SNP data for 94 individuals of <em>P. tremula</em> belonging to the SwAsp collection that consists of 116 individuals sampled from 12 sites across Sweden (6-10 individuals per site, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Luquez <em>et al.</em> 2008</a>) displaying no population structure (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2018</a>). The SNP data has previously been described in <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> (2018)</a> and consists of 4,425,109 SNPs with a minor allele frequency exceeding 5%.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and sequence capture</h3><p id="p-8">In 2015 leaf samples were collected from all progenies of the F<sub>1</sub> cross. DNA was extracted using the Qiagen Plant Mini kit according to manufacturer guidelines and sent to Rapid Genomics (<a href="http://www.rapid-genomics.com">http://www.rapid-genomics.com</a>) for genotyping using sequence capture probes. The probe set contain 45,923 probes of 120 bases each that were designed to target unique genic regions in the v1.1 <em>P. tremula</em> genome assembly (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>), as well as an additional 70 probes that were designed to specifically target the putative sex determination region on chromosome 19 of the <em>P. trichocarpa</em> genome assembly v3.0 (<a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a>). Parents and all offspring were subjected to sequence capture and subsequently sequenced on an Illumina HighSeq 2000 platform using paired-end (2x100bp) sequencing to an average depth of 15x per sample. All sequence capture data were delivered from Rapid Genomics in the spring of 2016. In addition, the two parents of the F<sub>1</sub> cross were whole-genome re-sequenced to an average depth of 15x on an Illumina HiSeq 2500 platform with paired-end sequencing (2x150 bp) at the National Genomics Infrastructure at the Science for Life Laboratory in Stockholm, Sweden.</p><p id="p-9">All raw sequencing reads were mapped against the complete <em>P. tremula</em> v.1.1 reference genome using BWA-MEM v.0.7.12 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>) using default parameters. Following read mapping, PCR duplicates were marked using Picard (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>) and local realignment around indels was performed using GATK RealignerTargetCreator and IndelRealigner (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">DePristo <em>et al.</em> 2011</a>). Genotyping was performed using GATK HaplotypeCaller (version 3.4-46, (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Van der Auwera <em>et al.</em> 2013</a>) with a diploid ploidy setting and gVCF output format. CombineGVCFs was then run on batches of ∼200 gVCFs to hierarchically merge samples into a single gVCF and a final SNP call was performed using GenotypeGVCFs jointly on the combined gVCF file, using default read mapping filters.</p></div><div id="sec-4" class="subsection"><h3>Construction of a high-density linkage map</h3><p id="p-10">Linkage maps were built separately for the two parents using a pseudo-testcross strategy (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grattapaglia &amp; Sederoff 1994</a>) by employing bi-allelic SNPs that were segregating according to Mendelian expectations. Pairwise estimates of recombination frequency were calculated between all markers and marker pairs showing no evidence for recombination were collected into bins and one representative marker was used for map construction. Markers were grouped into linkage groups (LGs) using a LOD threshold of 12 and ordered using the Kosambi mapping function as implemented in the BatchMap software (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schiffthaler <em>et al.</em> 2017</a>). AllMaps (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Tang <em>et al.</em> 2015</a>) was used to create physical chromosomes from the <em>P. tremula</em> genome assembly v.1.1 (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>) and the male and female genetic maps, with equal weight given to the two maps. For more details about the construction of the genetic and physical maps, please refer to Supplementary Methods.</p></div><div id="sec-5" class="subsection"><h3>Genetic map and linkage disequilibrium-based recombination rates</h3><p id="p-11">The genetic and physical maps for all 19 <em>P. tremula</em> chromosomes were read into MareyMap (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rezvoy <em>et al.</em> 2007</a>, <a href="https://cran.r-project.org/web/packages/MareyMap">https://cran.r-project.org/web/packages/MareyMap</a>) and the ‘sliding window’ method in MareyMap was used to estimate recombination rate in windows of 1 Mbp with a step size of 250 kbp. We only used windows with at least 8 SNPs in order to avoid regions with large gaps being assigned recombination values.</p><p id="p-12">To generate a LD-based recombination map we used LDhelmet v.1.10 (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Chan <em>et al.</em> 2012</a>) with a random subset of 25 diploid individuals (<em>i.e.</em>, 50 haplotypes) from the SwAsp data set (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2018</a>). We used the default values from the LDhelmet manual as <em>Populus tremula</em> has similar levels of nucleotide polymorphism and extent of linkage disequilibrium as <em>Drosophila melanogaster</em>, on which the default settings in LDhelmet are based upon (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>). As LDhelmet outputs estimates of recombination in units of ρ/bp whereas the genetic map is in units of cM, we converted the LDhelmet results to cM distances following the method outlined in <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Booker <em>et al.</em> (2017)</a>. We did this to be able to make comparisons between the recombination rates estimated using the two methods. The conversion assumes that the physical size of a chromosome is constant for the two methods so that the cumulative genetic distance in either cM or ρ should be the same but on different scales. The cumulative ρ was calculated by multiplying the ρ/bp estimates with the distance in bp between the adjacent SNP’s and then summed across chromosomes. Knowing the cumulative ρ and corresponding cM-values, it is possible to derive a ‘scaling factor’ to calculate cM values from the corresponding ρ values. The resulting cM values were read into MareyMap and recombination rates were estimated as described earlier for the genetic map-based recombination map. More details on the estimation of recombination rates from the genetic maps can be found in Supplementary Methods.</p></div><div id="sec-6" class="subsection"><h3>Correlation of recombination rate estimates, genetic correlates of recombination rate and model of recombination rate</h3><p id="p-13">We compared recombination rates inferred from the consensus genetic map or from the sequence data by calculating Spearman’s rank correlations (hereafter correlations) across 1 Mbp windows. We also assessed correlations between the two recombination rates and a number of genomic features, including gene density, repeat density, GC-content, substitution density, neutral diversity (π) and methylation.</p><p id="p-14">Gene and repeat density were estimated using bedtools (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Quinlan 2014</a>) based on the annotation for the v1.1 <em>P. tremula</em> genome (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>, <a href="ftp://plantgenie.org/Data/PopGenIE/Populus_tremula/v1.1/gff3/">ftp://plantgenie.org/Data/PopGenIE/Populus_tremula/v1.1/gff3/</a>). GC-content was calculated from the genome FASTA using an in-house developed awk script (modified from: <a href="https://www.biostars.org/p/70167/#70172">https://www.biostars.org/p/70167/#70172</a>). The original script was modified to have window functionality across a FASTA sequence and to take into consideration sequence gaps. Windows with more than 80% gaps (N) were discarded to avoid biased results.</p><p id="p-15">Substitutions relative to <em>P. trichocarpa</em> were estimated from a vcf-file with SNP-calls for a single <em>Populus trichocarpa</em> individual mapped against the <em>P. tremula</em> reference genome v1.1. Comparisons of putative substitutions were then made against a list of SNP positions from the 94 SwAsp <em>P. tremula</em> data set. Substitutions relative to <em>P. tremuloides</em> were inferred in a similar way based on SNP calls from five <em>P. tremuloides</em> individuals mapped against the <em>P. tremula</em> reference genome v1.2 (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>). Files containing old, new and the total number of substitutions were used to calculate substitution densities in windows across the genome. Neutral genetic diversity (π) was estimated using vcftools and only fourfold degenerate sites or intergenic sites located at least 2 kbp from an annotated gene. For nucleotide diversity calculations, data for all 94 SwAsp samples were used.</p><p id="p-16">Methylation levels were estimated using bisulfite sequencing data from six SwAsp individuals that were bisulfite sequenced using two biological replicates per individual. Samples were sequenced using paired-end (2x150) sequencing on an Illumina HiSeq X at the National Genomics Infrastructure facility at Science for Life Laboratory in Uppsala, Sweden. Methylation levels were estimated using the Bismark tools (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Krueger and Andrews 2011</a>, <a href="https://www.bioinformatics.babraham.ac.uk/projects/bismark/">https://www.bioinformatics.babraham.ac.uk/projects/bismark/</a>). Following trimming and quality control, sequence reads were mapped against polymorphism-substituted versions of the <em>P. tremula</em> v1.1 assembly (<a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>) for each individual sample. Following read mapping BAM files were deduplicated to remove optical duplicates. Methylation levels were then extracted separately for the different methylation contexts (GpG, CHG and CHH) and average values were calculated across 1 Mbp window using a step size of 250 kbp across the <em>P. tremula</em> genome. Further details on the handling of the methylation sequencing data can be found in Supplementary Methods. Correlations between the two recombination maps and between the recombination maps and the various genomic features were calculated in R (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">R Core Team 2018</a>). We also assessed the independent effects of different genomic features on the two recombination rate estimates using multiple regression.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-17">The raw sequencing reads used for the construction of the genetic map and raw bisulfate sequencing reads are available from ENA under study number PRJEB34662 (<a href="https://www.ebi.ac.uk/ena/data/view/PRJEB34662">https://www.ebi.ac.uk/ena/data/view/PRJEB34662</a>). BatchMap input files for the female and male genetic maps, the two component maps and the consensus map files as well as all processed files of the bisulfate sequencing data are available from <a href="http://zenodo.org">zenodo.org</a> under DOI number 10.5281/zenodo.3257544 (<a href="https://doi.org/10.5281/zenodo.3257544">https://doi.org/10.5281/zenodo.3257544</a>). Additional scripts and files used for the analyses are available at <a href="https://github.com/parkingvarsson/recombination_rate_variation">https://github.com/parkingvarsson/recombination_rate_variation</a>. All SNP data used for population genomic analyses is already publicly available through ENA under accession number PRJNA297202 (<a href="https://www.ebi.ac.uk/ena/data/view/PRJNA297202">https://www.ebi.ac.uk/ena/data/view/PRJNA297202</a>). A filtered VCF files with final SNP data are available to download from <a href="http://zenodo.org">zenodo.org</a> under DOI number 10.5281/zenodo.3546316 (<a href="https://doi.org/10.5281/zenodo.3546316">https://doi.org/10.5281/zenodo.3546316</a>). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.8481641">https://doi.org/10.25387/g3.8481641</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>P. tremula linkage maps</h3><p id="p-18">14,598 unique markers in the female map and 13,997 unique markers in the male map were distributed across 3,861 and 3,710 scaffolds, respectively. Markers in both parental framework maps grouped into 19 linkage groups (LGs), corresponding to the haploid number of chromosomes in <em>Populus</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Among the mapped scaffolds, 19 scaffolds contained markers that mapped to different positions within the same LG, but that were more than 20 cM apart (Supplemental Material, Figure S3) and 340 scaffolds contained markers that mapped to two or more LGs (Figures S2 and S4). These scaffolds were split according to criteria described in Supplementary methods. Additionally, there were 49 scaffolds split within gene models (Figure S5). Two ambiguous markers in scaffold Potra001073 were removed. After splitting 14,596 (12,900 binned) and 13,996 (12,382 binned) markers from 4,184 and 4,011 scaffolds remained. These markers spanned 4072.72 cM and 4053.68 cM for the female and male framework maps, respectively. The parental maps were used to produce a consensus genetic map consisting of 19,519 markers derived from 4,761 scaffolds spanning 4059.00 cM (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Linkage groups (LG) were assigned to the corresponding <em>P. trichocarpa</em> homologs through synteny assessment (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, Figure S1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495300/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495300/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of female male and consensus linkage maps for each chromosome</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Physical assembly Potra v1.2</h3><p id="p-19">The parental framework maps were used to produce a physical map, Potra v1.2, of the <em>P. tremula</em> chromosomes that we used to estimate recombination maps (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Figure S6). The scaffolds anchored from the parental framework maps spanned 210.7 Mbp and 205.1 Mbp for the female and male respectively. This corresponds to 54.6% and 53.1% of the 385.8 Mbp covered by the v1.1 assembly (Table S1). Of the 4,761 scaffolds with markers, 96.6% could be anchored in the assembly providing a total physical assembly consisting of 223.4 Mbp. This corresponds to 57.9% of the v1.1 <em>P. tremula</em> assembly (<a id="xref-ref-26-9" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). 75.7% of the physical map was both anchored and oriented, while the remaining 24.3% was only anchored. 199,967 of the v1.1 assembly scaffolds were not covered by the framework maps and thus could not be anchored to the physical map. There was a clear distinction between the scaffolds we could and could not anchor to the Potra v1.2 assembly. The median length for the 4214 scaffolds anchored in the map was 37 kbp and these scaffolds contain 26,808 predicted gene models. Conversely, the median length on unanchored scaffolds was only 0.3 kbp and they collectively contain only 8,501 predicted gene models (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). After the initial assembly, gap estimation added 43 Mbp of gap sequences across the genome, increasing the estimated total size of the v1.2 assembly to 265 Mbp. This is approximately 55% of the 479 Mbp genome size estimated for <em>P. tremula</em> (<a id="xref-ref-26-10" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/299/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic maps and resulting physical map created by Allmaps for Chr1 and Chr5. The left panel shows the marker distribution (in cM) for the genetic maps and the anchored genomic region (in Mbp) for the physical map, while the right panel is showing the Marey maps, i.e., the correspondence between the physical (x-axis) and recombination-based (y-axis) position of markers. The female map is depicted in green and the male map is depicted in orange." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693661649" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic maps and resulting physical map created by Allmaps for Chr1 and Chr5. The left panel shows the marker distribution (in cM) for the genetic maps and the anchored genomic region (in Mbp) for the physical map, while the right panel is showing the Marey maps, &lt;em&gt;i.e.&lt;/em&gt;, the correspondence between the physical (x-axis) and recombination-based (y-axis) position of markers. The female map is depicted in green and the male map is depicted in orange.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/299/F1.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/299/F1.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/299/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/299/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495214" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Genetic maps and resulting physical map created by Allmaps for Chr1 and Chr5. The left panel shows the marker distribution (in cM) for the genetic maps and the anchored genomic region (in Mbp) for the physical map, while the right panel is showing the Marey maps, <em>i.e.</em>, the correspondence between the physical (x-axis) and recombination-based (y-axis) position of markers. The female map is depicted in green and the male map is depicted in orange.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495378/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495378/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of physical assembly Potra v1.2</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Recombination estimates</h3><p id="p-21">Recombination estimates were produced based on the consensus linkage map (LMB) and from LD data (LDB) derived from 25 randomly selected individuals from the SwAsp collection. The LMB recombination rate estimates varied between 1.605 cM/Mbp on chromosome 4 to 26.911 cM/Mbp on chromosome 11, while the LDB estimates varied between 1.969 cM/Mbp on chromosome 5 to 231.801 cM/Mbp on chromosome 1. The median estimated recombination rate in the LMB map was 16.0 cM/Mbp with a mean of 15.6 cM/Mbp, whereas the median recombination rate for the LDB map was 14.0 cM/Mbp with a mean of 16.1 cM/Mbp (Table S2, Figure S7). The majority of all recombination rate estimates (97%) for both maps fell in the range of 2 - 27 cM/Mb (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Figure S8). There were 20 windows where the LDB estimates are 1.5-15-fold higher compared to the corresponding rates from the LMB estimates and 2-14-fold higher than the mean recombination rate estimate from the LDB map. 13 of these windows had recombination rates exceeding 27 cM/Mbp, while seven were within 2-27 cM/Mbp. Conversely, there were also 23 windows where the LDB estimates were 2-4 times lower than the corresponding LMB estimates (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, Figures S8 and S9).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/299/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombination rates and genomic features calculated in 1 Mbp windows across chromosome 1 and 5 with a step size of 250kbp. A) Recombination rate estimated from the linkage map (cM/Mbp) B) Recombination rate estimated from sequence LD data (cM/Mbp) C) Nucleotide diversity (1/bp) D) Divergence (sites/Mbp) E) Gene density (percentage coding/Mbp) F) Repeat density (percentage repeats/Mbp) G) CpG methylation (percentage/Mbp)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693661649" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Recombination rates and genomic features calculated in 1 Mbp windows across chromosome 1 and 5 with a step size of 250kbp. A) Recombination rate estimated from the linkage map (cM/Mbp) B) Recombination rate estimated from sequence LD data (cM/Mbp) C) Nucleotide diversity (1/bp) D) Divergence (sites/Mbp) E) Gene density (percentage coding/Mbp) F) Repeat density (percentage repeats/Mbp) G) CpG methylation (percentage/Mbp).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/299/F2.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/299/F2.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/299/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/299/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495342" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Recombination rates and genomic features calculated in 1 Mbp windows across chromosome 1 and 5 with a step size of 250kbp. A) Recombination rate estimated from the linkage map (cM/Mbp) B) Recombination rate estimated from sequence LD data (cM/Mbp) C) Nucleotide diversity (1/bp) D) Divergence (sites/Mbp) E) Gene density (percentage coding/Mbp) F) Repeat density (percentage repeats/Mbp) G) CpG methylation (percentage/Mbp).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Correlation of recombination rate estimates and genomic features</h3><p id="p-23">The Spearman’s rank correlation between recombination rate estimates for the LMB and LDB maps was 0.478 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This was the strongest positive correlation of all of the correlations calculated for both maps and strongest correlation overall for the LDB map. Correlations between the LMB and LDB recombination maps and neutral diversity were the second strongest positive correlations for both maps, 0.447 and 0.442 respectively. This correlation was the second strongest overall for LDB map. Correlation with neutral diversity was also the only variable where there was no notable decrease in the correlation coefficient from the LMB to the LDB recombination maps (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/299/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between recombination rates and genomic features. All correlations are significant (P &lt; 0.01) with the exception of those marked with * (P &gt; 0.01)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693661649" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlations between recombination rates and genomic features. All correlations are significant (&lt;em&gt;P&lt;/em&gt; &lt; 0.01) with the exception of those marked with * (&lt;em&gt;P&lt;/em&gt; &gt; 0.01).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/299/F3.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/299/F3.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/299/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/299/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495218" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Correlations between recombination rates and genomic features. All correlations are significant (<em>P</em> &lt; 0.01) with the exception of those marked with * (<em>P</em> &gt; 0.01).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">For the LMB map we observed strong negative correlations with CHG methylation (-0.515), CHH methylation (-0.511) and CpG methylation (-0.505). For the LDB map the corresponding correlations were -0.379 (CHH), -0.371 (CHG) and -0.353 (CpG). Methylation levels were also strongly correlated with each other (0.918-0.984) and with repeat density (0.800-0.840). Repeat density was also moderately negatively correlated with recombination rate estimates from both the LMB (-0.408) and LDB maps (-0.291). Both recombination maps showed only weak correlations (-0.1&lt;<em>ρ</em>&lt;0.1) with either old or new neutral substitution densities (-0.06 - -0.1). Neutral substitution densities and neutral diversity showed only weak negative correlations (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Overall, the LMB estimates displayed consistently stronger correlations with the different genomic features compared to the LDB estimates. This is in line with 5% of the total variation being explained by a multiple regression model for the LDB recombination estimates compared to 35% variation explained for the LMB estimates (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495383/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495383/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Multiple regression of recombination rate and various genomic features</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>P. tremula fine-scale genetic maps and physical assembly Potra v1.2</h3><p id="p-26">The genetic maps presented here are the most marker-dense maps produced for <em>P. tremula</em> to date. Our female map is only 20 cM larger than the male map, despite having 518 more informative markers (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) and most chromosomes have size differences of less than 10 cM between sexes (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, Figure S6). However, in cases where we observed differences between the maps for the two sexes exceeding 10 cM, the male map is shorter in all but one case. These results, together with the overall shorter linkage map for the male, could suggest overall lower recombination rates in males, in line with what has been observed in many other highly outcrossing plant species (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lenormand and Dutheil 2005</a>). The high marker density in our framework genetic maps allowed us to anchor 57.9% of the <em>P. tremula</em> v1.1 genome assembly on to the expected 19 chromosomes, providing us with the first chromosome-scale assembly for <em>P. tremula</em> (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-27">The map length in the section <em>Populus</em>, to which <em>P. tremula</em> belongs (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> 2014</a>), has previously been estimated to be 1,600-3,500 cM (<em>e.g.</em>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhang <em>et al.</em> 2004</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Paolucci <em>et al.</em> 2010</a>, <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Zhigunov <em>et al.</em> 2017</a>). The most relevant comparison for our purposes is the recently produced linkage maps in <em>P. tremula</em> by <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Zhigunov <em>et al.</em> (2017)</a>. Their map contains 2000 informative markers with an average marker distance of 1.5 cM that were observed in 122 progenies, resulting in a total map length of 3000-3100 cM. Our framework maps are much denser with ca. 12000-13000 informative markers (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>) and with an average distance of ∼0.3 cM between markers. In addition, our map is based on a mapping population consisting of 764 progenies and we are hence able to achieve a far greater resolution in our maps. However, larger data sets, both with respect to the number of markers and the number of progenies used, increase the risk of genotyping errors. Genotyping errors will ultimately lead to an inflation of map sizes as errors can be interpreted as recombination events during map creation and this could help explain why our maps are roughly 1000 cM longer than those reported by <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Zhigunov <em>et al.</em> (2017)</a>, given that we use 5-7 -fold more markers and a mapping population that is six times larger.</p><p id="p-28">On the other hand, our framework genetic maps are similar in size to the ca. 4200 cM and 3800 cM maps presented by <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Tong <em>et al.</em> (2016)</a> for the more distantly related species <em>Populus deltoides</em> and <em>Populus simonii</em>, respectively. The large size of these maps led <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Tong <em>et al.</em> (2016)</a> to suggest that their maps were suffering from inflation due to the difficulty of properly ordering a large number of markers within a linkage group. While we likely also suffer from such size inflation, these issues appear to be less severe in our <em>P. tremula</em> parental maps, which contain between 8-14 times the number of markers used in the <em>P. deltoides</em> (1601) and <em>P. simonii</em> (940) maps and yet yield linkage maps of similar size. One explanation for this is our considerably larger mapping population compared to the <em>P. deltoides</em> and <em>P. simonii</em> maps (299 progenies). A greater number of segregating progenies helps mitigate the problems of ordering a larger number of markers by increasing resolution of recombination detection.</p></div><div id="sec-15" class="subsection"><h3>Recombination rate estimates</h3><p id="p-29">Recombination rates estimated from both the consensus linkage map and from polymorphism data showed substantial variation across all chromosomes on Mbp scales (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). For the consensus genetic map-based estimates and LD-based estimates, the majority of our observations fell in the range 0-27 cM/Mbp (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, Figure S8) which is similar to what has been observed in other plants such as <em>Arabidopsis thaliana</em> (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Giraut <em>et al.</em> 2011</a>), <em>Populus trichocarpa</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Slavov <em>et al.</em> 2012</a>) and <em>Eucalyptus grandis</em> (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Silva-Junior and Grattapaglia 2015</a>), where recombination rates across chromosomes mostly fall within 0-25 cM/Mbp.</p><p id="p-30">We observed a small number of genomic windows where the LDB recombination rates were either 1.5-15-fold higher or lower than the corresponding estimates based on the consensus genetic map (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, Figure S8). While we do not know for certain what causes these large differences in recombination rates, a possible explanation could be that such windows harbor recombination hotspots or coldspots that the comparatively coarse linkage map fails to detect. Recombination hotspots, with local recombination rates 10 to 100-fold higher than the genome-wide average, have been observed in a number of species, including <em>Drosophila melanogaster</em> (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Chan <em>et al.</em> 2012</a>), <em>Arabidopsis thaliana</em> (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2007</a>), <em>Zea mays</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">He and Dooner 2009</a>), <em>Oryza sativa</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Si <em>et al.</em> 2015</a>) and <em>Eucalyptus grandis</em> (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Silva-Junior and Grattapaglia 2015</a>). Similarly, coldspots have been identified in <em>Zea mays</em> (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">He and Dooner 2009</a>) and <em>Oryza sativa</em> (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Si <em>et al.</em> 2015</a>) among others. Hotspots or coldspots for recombination are, however, often quite restricted in size (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Choi and Henderson 2015</a>), spanning only a few kbp, and the relatively coarse recombination maps produced here are consequently not suitable for accurate detection of such regions. It is also worth noting that in species with lower levels of diversity, the resolution of LD based estimates would be lower and the power to detect hot- and coldspots for LDB and LMB estimates may not differ much for these species.</p><p id="p-31">The average recombination rate in <em>P. tremula</em> is 2-27 times higher than those found in a number of, mostly domesticated, plant and animal species (reviewed in <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Henderson (2012)</a> and <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tiley and Burleigh (2015)</a>), suggesting that <em>P. tremula</em> exhibits recombination rates that are among the highest recorded in the animal and plant kingdoms. Of the species covered in these reviews, <em>P. trichocarpa</em> (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Slavov <em>et al.</em> 2012</a>) makes for the most interesting comparison since it is one of the few undomesticated species listed, is closely related to <em>P. tremula</em> and has previously been compared with <em>P. tremula</em> (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>). Despite the close relationship, the average recombination rate in <em>P. trichocarpa</em> (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Slavov <em>et al.</em> 2012</a>) is less than a third of what we estimated for <em>P. tremula</em>. Similar observations were previously made by <a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> (2016)</a> who found that population-based recombination rates in <em>P. trichocarpa</em> were on average only a quarter of the corresponding values in <em>P. tremula</em>. <a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> (2016)</a> argued that the differences they observed in recombination rates between <em>P. tremula</em> and <em>P. trichocarpa</em> could at least partly stem from differences in the effective population size (N<sub>e</sub>) of the two species (<a id="xref-ref-54-7" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>). In light of this, it would be interesting to perform further comparisons of recombination rates between <em>P. tremula</em> and other <em>Populus</em> species that have wide distribution ranges and large N<sub>e</sub>, such as <em>P. deltoides</em> (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Tong <em>et al.</em> 2016</a>) or <em>P. tremuloides</em> (<a id="xref-ref-54-8" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>).</p></div><div id="sec-16" class="subsection"><h3>Correlations between recombination rate and genomic features</h3><p id="p-32">Recombination rate estimates from the consensus linkage map and from polymorphism data showed a moderately strong positive correlation (&gt;0.4) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). A similar correlation between linkage map and LD-based estimates of recombination was also observed in <em>Mus musculus</em> by <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Booker <em>et al.</em> (2017)</a>, suggesting that LDB recombination rate estimates are reliable substitutes for genetic map-based recombination rate estimates. A recent study in <em>Gasterosteus aculeatus</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shanfelter <em>et al.</em> 2019</a>), however, found even stronger correlations between the two estimates (∼0.8).</p><p id="p-33">We observed a strong positive correlation between recombination rate and gene density (0.45 and 0.41 respectively) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). This is in line with earlier observations in plants (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Tiley and Burleigh 2015</a>, <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Stapley <em>et al.</em> 2017</a>) and implies that recombination may be linked to gene-dense regions through a higher recruitment of the recombination machinery to euchromatic genome regions. Preferential recruitment of recombination to euchromatic genome regions has also been put forward as an explanation for why recombination rates across plants generally show stronger correlations with gene density compared to genome size (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Henderson 2012</a>, <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Tiley and Burleigh 2015</a>). Studies in plants like <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em> have shown that while crossover events are enriched in genic regions, they mostly occur in promoters a few hundred bps upstream of the transcription start site or downstream of the transcription termination site (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Choi <em>et al.</em> 2013</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Marand <em>et al.</em> 2019</a>).</p><p id="p-34">We observed negative correlations between local recombination rates and both repeat density and methylation (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>), in line with earlier results that highlighted the role of chromatin features in establishing crossover locations in plants (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Choi <em>et al.</em> 2013</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Marand <em>et al.</em> 2019</a>). For instance, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Choi <em>et al.</em> (2013)</a> showed that methylation is lower at observed sites of crossovers and <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Rodgers-Melnick <em>et al.</em> (2015)</a> showed that cross-over density in <em>Zea mays</em> is negatively correlated with repeats and CpG methylation. All methylation contexts were highly correlated in our data and also strongly correlated with repeat density (≥0.8, <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>), in line with the observation that most repetitive elements in plant genomes are strongly methylated (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Saze and Kakutani 2011</a>).</p><p id="p-35">Compared to earlier results from <em>P. tremula</em>, we observed a weaker correlation between recombination rates and gene density (<a id="xref-ref-54-9" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>). One possible reason for this is likely to be the reference genome used. <a id="xref-ref-54-10" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> (2016)</a> based their analyses on the <em>P. trichocarpa</em> reference genome whereas our analyses were based on a <em>de novo</em> assembly for <em>P. tremula</em>. The <em>P. trichocarpa</em> assembly, while more contiguous than our current <em>P. tremula</em> assembly, is less ideal for these types of analyses since divergence between the two species leads to substantially reduced rates of read mapping primarily in intergenic regions (<a id="xref-ref-26-11" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2018</a>). Our current assembly, while only representing 55% of the expected genome size <em>of P. tremula</em>, likely offers a more unbiased set of genomic regions where we are able to call genetic variants. In contrast, the data derived from using the <em>P. trichocarpa</em> reference genome likely suffers from under-representations of repeat-rich regions and other intergenic regions (<a id="xref-ref-54-11" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>).</p><p id="p-36">GC-content was positively correlated with both our recombination rate estimates, similar to what has been observed in humans (<em>Homo sapiens</em>) (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fullerton <em>et al.</em> 2001</a>) and <em>Arabidopsis thaliana</em> (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2007</a>) among others. However, when GC-content was included in a multiple regression model with other genomic features, the direct effect of GC content was actually negative for the LMB recombination rate (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). GC-content is strongly correlated with gene density (0.50) in <em>P. tremula</em>, and gene density is in turn also strongly positively correlated with recombination (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>). The strand separation needed in the strand invasion of meiotic recombination is harder to achieve in areas with high GC-content due to higher annealing energy and can explain why GC-content has a direct negative effect on recombination rates when effects of gene density are accounted for (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, <em>e.g.</em>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mandel and Marmur 1968</a>).</p></div><div id="sec-17" class="subsection"><h3>Effects of linked selection on patterns of nucleotide diversity in P. tremula</h3><p id="p-37">Both of our recombination rate estimates were strongly correlated with nucleotide diversity at putatively neutral sites (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>). A positive correlation between local recombination rate and nucleotide polymorphism is usually interpreted as a signature of ubiquitous natural selection acting either through positive (hitchhiking) or negative (background) selection (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Begun and Aquadro 1992</a>). Alternatively, such a correlation could also arise if recombination itself is mutagenic (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Begun and Aquadro 1992</a>). However, if recombination is mutagenic one also expects to see a correlation between recombination and sequence divergence at neutral sites (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Begun and Aquadro 1992</a>). Our data show little evidence supporting the idea that recombination has a direct mutagenic effect as we observed only a weak and negative correlation between local recombination rates and substitutions at putatively neutral sites (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>). In light of this, and in line with earlier results, we observed that linked selection has pervasive effects on neutral diversity across the <em>P. tremula</em> genome (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ingvarsson 2010</a>, <a id="xref-ref-54-12" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>).</p></div><div id="sec-18" class="subsection"><h3>Conclusions</h3><p id="p-38">Our high-density <em>Populus tremula</em> genetic maps and the new chromosome-scale genome assembly we present here provide a valuable resource not only for <em>P. tremula</em>, but also for comparative genomics studies within the entire genus <em>Populus</em>. Estimates of recombination rates derived from two different approaches were in broad agreement and showed similar correlations various genomic features. However, our results suggest that LD-based estimates of recombination might be especially useful for identifying fine scale recombination variation and for identifying features such as recombination hot- or cold-spots as it is based on information derived from population-scale variation. We have also verified and extended the observation that linked selection is an important force shaping genome-wide variation in <em>P. tremula</em> by showing that the positive correlation between local recombination rates and nucleotide diversity and neutral sites is robust even when factoring in the effects of other genomic features. Although a positive correlation between recombination and diversity is a hallmark signature of linked selection, the pattern can be established by either positive or negative selection. We have earlier documented evidence for both a reduction in levels of standing variation due to recurrent hitchhiking (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Ingvarsson 2010</a>) and a reduction in the efficacy of purifying selection at eliminating weakly deleterious variants in regions of low recombination (<a id="xref-ref-54-13" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2016</a>). More work is thus needed to assess the relative importance of positive and negative selection in shaping genome-wide variation in <em>P. tremula</em> and having access to the resources developed here will facilitate these studies.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">NRS, PKI and SJ planned and designed the research. RPA, CB, BS, KR, NRS and PKI performed experiments, conducted fieldwork and analyzed data. RPA, CB and PKI wrote the manuscript.</p><p id="p-40">The research has been funded through research grants from the Swedish Research Council (PI), the Knut and Alice Wallenberg Foundation (PI, BS) and ‘Trees and Crops for the Future’ (NRS, PKI and SJ). The authors acknowledge support from the National Genomics Infrastructure in Stockholm funded by Science for Life Laboratory, the Knut and Alice Wallenberg Foundation and the Swedish Research Council, and SNIC/Uppsala Multidisciplinary Center for Advanced Computational Science for assistance with massively parallel sequencing and access to the UPPMAX computational infrastructure through projects b2011141 and SNIC 2017/1-499.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-41">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.8481641">https://doi.org/10.25387/g3.8481641</a>.</p></li><li class="fn" id="fn-2"><p id="p-42"><em>Communicating editor: M. Hufford</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-03"><span class="received-label">Received </span>July 3, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-13"><span class="accepted-label">Accepted </span>November 13, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Apuli <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.1" data-doi="10.1002/0471250953.bi1110s43"><div class="cit-metadata"><cite>Van der Auwera, G. A., M. O. Carneiro, C. Hartl, R. Poplin, G. Del Angel <em>et al.</em>, 2013 From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. Curr. Prot. Bioinf. 43: 11.10.1–33.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471250953.bi1110s43</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi1110s43%26rft_id%253Dinfo%253Apmid%252F25431634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi1110s43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25431634&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Linkage and the limits to natural selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D140%26rft.issue%253D2%26rft.spage%253D821%26rft.epage%253D841%26rft.atitle%253DLinkage%2Band%2Bthe%2Blimits%2Bto%2Bnatural%2Bselection.%26rft_id%253Dinfo%253Apmid%252F7498757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzIvODIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.3" data-doi="10.1038/356519a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>,  <span class="cit-name-given-names">D. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Aquadro</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Levels of naturally occurring DNA polymorphism correlate with recombination rates in <em>D. melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">356</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/356519a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D356%26rft.issue%253D6369%26rft.spage%253D519%26rft.epage%253D520%26rft.atitle%253DLevels%2Bof%2Bnaturally%2Boccurring%2BDNA%2Bpolymorphism%2Bcorrelate%2Bwith%2Brecombination%2Brates%2Bin%2BD.%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F356519a0%26rft_id%253Dinfo%253Apmid%252F1560824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/356519a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1560824&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HN23100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.4" data-doi="10.1534/genetics.117.300063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Booker</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Ness</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Keightley</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The recombination landscape in wild house mice inferred using population genomic data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300063%26rft_id%253Dinfo%253Apmid%252F28751421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA3LzEvMjk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.5" data-doi="10.1371/journal.pgen.1003090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Jenkins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide fine-scale recombination rate variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1003090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003090%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003090%26rft_id%253Dinfo%253Apmid%252F23284288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284288&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.6" data-doi="10.1111/tpj.12870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. R.</span> <span class="cit-name-surname">Henderson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meiotic recombination hotspots - a comparative view.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12870</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D83%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12870%26rft_id%253Dinfo%253Apmid%252F25925869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12870&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25925869&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.7" data-doi="10.1038/ng.2766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Kelly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Venn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Higgins</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Arabidopsis</em> meiotic crossover hot spots overlap with H2A.Z nucleosomes at gene promoters.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1327</span>–<span class="cit-lpage">1336</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2766</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D1327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2766%26rft_id%253Dinfo%253Apmid%252F24056716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24056716&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.8" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>,  <span class="cit-name-given-names">M. A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Poplin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Garimella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Maguire</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.299.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>,  <span class="cit-name-given-names">R. E.</span>,   <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1996</span> <span class="cit-article-title">The genecology of Populus</span>. pp. <span class="cit-fpage">33</span>–<span class="cit-lpage">55</span> in: <span class="cit-source">Biology of Populus and its implications for management and conservation</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Stettler</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bradshaw</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Heilman</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hinckley</span></span>, <span class="cit-publ-name">NRC Research Press</span>, <span class="cit-publ-loc">Ottawa, Canada</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.11" data-doi="10.1104/pp.108.133249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fracheboud</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Luquez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Björkén</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sjödin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tuominen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The control of autumn senescence in European aspen.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1982</span>–<span class="cit-lpage">1991</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.108.133249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.133249%26rft_id%253Dinfo%253Apmid%252F19201914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQ5LzQvMTk4MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.12" data-doi="10.1093/oxfordjournals.molbev.a003886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fullerton</span>,  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Carvalho</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Clark</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Local rates of recombination are positively correlated with GC content in the human genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1139</span>–<span class="cit-lpage">1142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a003886</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DFullerton%26rft.auinit1%253DS.%2BM.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D1139%26rft.epage%253D1142%26rft.atitle%253DLocal%2BRates%2Bof%2BRecombination%2BAre%2BPositively%2BCorrelated%2Bwith%2BGC%2BContent%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003886%26rft_id%253Dinfo%253Apmid%252F11371603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11371603&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169180200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.13" data-doi="10.1371/journal.pgen.1002354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraut</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Falque</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drouaud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pereira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. C.</span> <span class="cit-name-surname">Martin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide crossover distribution in <em>Arabidopsis thaliana</em> meiosis reveals sex-specific patterns along chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGiraut%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002354%26rft.epage%253De1002354%26rft.atitle%253DGenome-wide%2Bcrossover%2Bdistribution%2Bin%2BArabidopsis%2Bthaliana%2Bmeiosis%2Breveals%2Bsex-specific%2Bpatterns%2Balong%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002354%26rft_id%253Dinfo%253Apmid%252F22072983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072983&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Sederoff</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.15" data-doi="10.1073/pnas.0902972106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Dooner</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Haplotype structure strongly affects recombination in a maize genetic interval polymorphic for <em>Helitron</em> and retrotransposon insertions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">8410</span>–<span class="cit-lpage">8416</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0902972106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0902972106%26rft_id%253Dinfo%253Apmid%252F19416860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIxLzg0MTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.16" data-doi="10.1111/mec.14699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haenel</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Laurentino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Roesti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Berner</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Meta-analysis of chromosome-scale crossover rate variation in eukaryotes and its significance to evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2477</span>–<span class="cit-lpage">2497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D27%26rft.spage%253D2477%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.17" data-doi="10.1016/j.pbi.2012.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">I. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Control of meiotic recombination frequency in plant genomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2012.09.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DHenderson%26rft.auinit1%253DI.%2BR.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D556%26rft.epage%253D561%26rft.atitle%253DControl%2Bof%2Bmeiotic%2Brecombination%2Bfrequency%2Bin%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.09.002%26rft_id%253Dinfo%253Apmid%252F23017241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23017241&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.18" data-doi="10.1093/molbev/msp255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>,  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Natural selection on synonymous and nonsynonymous mutations shapes patterns of polymorphism in <em>Populus tremula</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">650</span>–<span class="cit-lpage">660</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msp255</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp255%26rft_id%253Dinfo%253Apmid%252F19837657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19837657&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274786900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.19" data-doi="10.1146/annurev.arplant.58.032806.103956"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Douglas</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Populus: a model system for plant biology.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">458</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.arplant.58.032806.103956</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DJansson%26rft.auinit1%253DS.%26rft.volume%253D58%26rft.spage%253D435%26rft.epage%253D458%26rft.atitle%253DPopulus%253A%2Ba%2Bmodel%2Bsystem%2Bfor%2Bplant%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.58.032806.103956%26rft_id%253Dinfo%253Apmid%252F17280524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.58.032806.103956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17280524&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247703600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.20" data-doi="10.1038/ng2115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Plagnol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. T.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Toomajian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Clark</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination and linkage disequilibrium in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng2115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DRecombination%2Band%2Blinkage%2Bdisequilibrium%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2115%26rft_id%253Dinfo%253Apmid%252F17676040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676040&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.21" data-doi="10.1016/j.jnc.2003.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kouki</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Arnold</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Martikainen</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Long-term persistence of aspen – a key host for many threatened species – is endangered in old-growth conservation areas in Finland.</span> <abbr class="cit-jnl-abbrev">J. Nat. Conserv.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jnc.2003.08.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNat.%2BConserv.%26rft.volume%253D12%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jnc.2003.08.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jnc.2003.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.22" data-doi="10.1093/bioinformatics/btr167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krueger</span>,  <span class="cit-name-given-names">F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Andrews</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1571</span>–<span class="cit-lpage">1572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr167</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr167%26rft_id%253Dinfo%253Apmid%252F21493656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21493656&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291062400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.23" data-doi="10.1073/pnas.0801848105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Bennett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Fitzpatrick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. F.</span> <span class="cit-name-surname">Noor</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Fine-scale mapping of recombination rate in <em>Drosophila</em> refines its correlation to diversity and divergence.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">10051</span>–<span class="cit-lpage">10056</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0801848105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0801848105%26rft_id%253Dinfo%253Apmid%252F18621713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzI5LzEwMDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.24" data-doi="10.1371/journal.pbio.0030063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dutheil</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination difference between sexes: a role for haploid selection.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0030063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLenormand%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253De63%26rft.epage%253De63%26rft.atitle%253DRecombination%2Bdifference%2Bbetween%2Bsexes%253A%2Ba%2Brole%2Bfor%2Bhaploid%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030063%26rft_id%253Dinfo%253Apmid%252F15736976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15736976&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.25" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with burrows-wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.26" data-doi="10.1073/pnas.1801437115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Delhomme</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schiffthaler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sundström</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Functional and evolutionary genomic inferences in <em>Populus</em> through genome and population sequencing of American and European aspen.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">E10970</span>–<span class="cit-lpage">E10978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1801437115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1801437115%26rft_id%253Dinfo%253Apmid%252F30373829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMzoiMTE1LzQ2L0UxMDk3MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.27" data-doi="10.1007/s11295-007-0108-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luquez</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Hall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Albrectsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Karlsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. K.</span> <span class="cit-name-surname">Ingvarsson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural phenological variation in aspen (<em>Populus tremula</em>): the SwAsp collection.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-007-0108-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D4%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-007-0108-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-007-0108-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.28" data-doi="10.1016/0076-6879(67)12133-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mandel</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Marmur</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">[109] Use of ultraviolet absorbance-temperature profile for determining the quanine plus cytosine content of DNA.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0076-6879(67)12133-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D12%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252867%252912133-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(67)12133-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.29" data-doi="10.1038/ncomms3797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Hu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic insights into salt adaptation in a desert poplar.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms3797</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2797%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3797%26rft_id%253Dinfo%253Apmid%252F24256998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24256998&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.30" data-doi="10.1105/tpc.18.00750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marand</span>,  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zeng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Historical meiotic crossover hotspots fueled patterns of evolutionary divergence in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.18.00750</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.18.00750%26rft_id%253Dinfo%253Apmid%252F30705136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjMxLzMvNjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.31" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivachenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cibulskis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Myers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hunt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Deloukas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Bentley</span></span>, <span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2004</span> <span class="cit-article-title">The fine-scale structure of recombination rate variation in the human genome. Science</span> <span class="cit-vol">304</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">584</span>. </cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.33" data-doi="10.1016/S0959-437X(02)00358-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>,  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Variation in recombination rate across the genome: evidence and implications.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0959-437X(02)00358-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D657%26rft.epage%253D663%26rft.atitle%253DVariation%2Bin%2Brecombination%2Brate%2Bacross%2Bthe%2Bgenome%253A%2Bevidence%2Band%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900358-1%26rft_id%253Dinfo%253Apmid%252F12433578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00358-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433578&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.34" data-doi="10.1038/nature07629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Weigel</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation genetics in plants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07629</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D456%26rft.issue%253D7223%26rft.spage%253D720%26rft.epage%253D723%26rft.atitle%253DNext-generation%2Bgenetics%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07629%26rft_id%253Dinfo%253Apmid%252F19079047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079047&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261559900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.35" data-doi="10.1007/s11295-010-0297-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paolucci</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gaudet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Jorge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Beritognolo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Terzoli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic linkage maps of <em>Populus alba</em> L. and comparative mapping analysis of sex determination across <em>Populus</em> species.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">875</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-010-0297-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D6%26rft.spage%253D863%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0297-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0297-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000284421100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.36"><div class="cit-metadata unstructured">Quinlan, A. R. 2014 BEDTools: The Swiss‐army tool for genome feature analysis. Curr. Prot. Bioinf. 47: 11.12.1–11.12.34.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-10.1.299.37"><div class="cit-metadata unstructured">R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for statistical Computing, Vienna, Austria. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.38" data-doi="10.1093/bioinformatics/btm315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rezvoy</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charif</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Guéquen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A. B.</span> <span class="cit-name-surname">Marais</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MareyMap: an R-based tool with graphical inter-face for estimation recombination rates.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2188</span>–<span class="cit-lpage">2189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm315%26rft_id%253Dinfo%253Apmid%252F17586550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586550&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249818300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.39" data-doi="10.1073/pnas.1413864112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodgers-Melnick</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bradbury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Elshire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Acharya</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination in diverse maize is stable, predictable, and associated with genetic load.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">3823</span>–<span class="cit-lpage">3828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1413864112%26rft_id%253Dinfo%253Apmid%252F25775595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzEyLzM4MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.40" data-doi="10.1016/j.pbi.2010.08.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saze</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kakutani</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Differentiation of epigenetic modifications between transposons and genes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2010.08.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DSaze%26rft.auinit1%253DH.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D87%26rft.atitle%253DDifferentiation%2Bof%2Bepigenetic%2Bmodifications%2Bbetween%2Btransposons%2Band%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2010.08.017%26rft_id%253Dinfo%253Apmid%252F20869294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2010.08.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20869294&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.41" data-doi="10.1371/journal.pone.0189256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiffthaler</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bernhardsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. K.</span> <span class="cit-name-surname">Ingvarsson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Street</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">BatchMap: A parallel implementation of the OneMap R package for fast computation of F1 linkage maps in outcrossing species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e0189256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0189256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0189256%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0189256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0189256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.42" data-doi="10.1093/gbe/evz090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shanfelter</span>,  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Archambeault</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">White</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Divergent Fine-Scale Recombination Landscapes between a Freshwater and Marine Population of Threespine Stickleback Fish.</span> <abbr class="cit-jnl-abbrev">Genom Evol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1573</span>–<span class="cit-lpage">1585</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evz090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenom%2BEvol.%2BBiol.%26rft.volume%253D11%26rft.spage%253D1573%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevz090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evz090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.43" data-doi="10.1111/nph.13505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva-Junior</span>,  <span class="cit-name-given-names">O. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Grattapaglia</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide patterns of recombination, linkage disequilibrium and nucleotide diversity from pooled resequencing and single nucleotide polymorphism genotyping unlock the evolutionary history of <em>Eucalyptus grandis</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">830</span>–<span class="cit-lpage">845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13505</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D830%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13505%26rft_id%253Dinfo%253Apmid%252F26079595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26079595&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.44" data-doi="10.1111/nph.13319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Si</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yuan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Widely distributed hot and cold spots in meiotic recombination as shown by the sequencing of rice F2 plants.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D206%26rft.spage%253D1491%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13319%26rft_id%253Dinfo%253Apmid%252F25664766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25664766&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.45" data-doi="10.1111/j.1469-8137.2012.04258.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slavov</span>,  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Difazio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Schackwitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Muchero</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rodgers-Melnick</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome resequencing reveals multiscale geographic structure and extensive linkage disequilibrium in the forest tree <em>Populus trichocarpa</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2012.04258.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSlavov%26rft.auinit1%253DG.%2BT.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D713%26rft.epage%253D725%26rft.atitle%253DGenome%2Bresequencing%2Breveals%2Bmultiscale%2Bgeographic%2Bstructure%2Band%2Bextensive%2Blinkage%2Bdisequilibrium%2Bin%2Bthe%2Bforest%2Btree%2BPopulus%2Btrichocarpa.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2012.04258.x%26rft_id%253Dinfo%253Apmid%252F22861491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2012.04258.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22861491&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309757200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.46" data-doi="10.1098/rstb.2016.0455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. G. D.</span> <span class="cit-name-surname">Feulner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Johnston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Santure</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Smadja</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in recombination frequency and distribution across eukaryotes: patterns and processes.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond., B</abbr> <span class="cit-vol">372</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2016.0455</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%252C%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0455%26rft_id%253Dinfo%253Apmid%252F29109219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2016.0455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29109219&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.299.47" data-doi="10.1186/s13059-014-0573-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Miao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ming</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ALLMAPS: robust scaffold ordering based on multiple maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0573-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0573-1%26rft_id%253Dinfo%253Apmid%252F25583564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0573-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583564&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.48" data-doi="10.1093/aob/mcf255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>Populus</em>: Arabidopsis for forestry. Do we need a model tree?</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">689</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcf255</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcf255%26rft_id%253Dinfo%253Apmid%252F12451023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcf255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12451023&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.49" data-doi="10.1186/s12862-015-0473-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tiley</span>,  <span class="cit-name-given-names">G. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Burleigh</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The relationship of recombination rate, genome structure, and patterns of molecular evolution across angiosperms.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12862-015-0473-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D15%26rft.spage%253D194%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-015-0473-3%26rft_id%253Dinfo%253Apmid%252F26377000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-015-0473-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26377000&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.50" data-doi="10.1371/journal.pone.0150692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of high-density linkage maps of <em>Populus deltoides</em> × <em>P. simonii</em> using restriction-site associated DNA sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0150692</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0150692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0150692%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0150692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0150692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.51" data-doi="10.1126/science.1128691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>,  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">DiFazio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jansson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bohlmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Grigoriev</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1128691</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTuskan%26rft.auinit1%253DG.%2BA.%26rft.volume%253D313%26rft.issue%253D5793%26rft.spage%253D1596%26rft.epage%253D1604%26rft.atitle%253DThe%2Bgenome%2Bof%2Bblack%2Bcottonwood%252C%2BPopulus%2Btrichocarpa%2B%2528Torr.%2B%2526amp%253B%2BGray%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128691%26rft_id%253Dinfo%253Apmid%252F16973872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTMvNTc5My8xNTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.52" data-doi="10.1186/s13059-018-1444-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Robinson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Michelson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A major locus controls local adaptation and adaptive life history variation in a perennial plant.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-018-1444-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D19%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-018-1444-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-018-1444-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.53" data-doi="10.1371/journal.pone.0103645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dayanandan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zeng</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phylogeny reconstruction and hybrid analysis of <em>Populus</em> (Salicaceae) based on nucleotide sequences of multiple single-copy nuclear genes and plastid fragments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e103645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0103645</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De103645%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103645%26rft_id%253Dinfo%253Apmid%252F25116432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25116432&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.54" data-doi="10.1534/genetics.115.183152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Street</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Scofield</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. K.</span> <span class="cit-name-surname">Ingvarsson</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Natural selection and recombination rate variation shape nucleotide polymorphism across the genomes of three related <em>Populus</em> species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.183152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183152%26rft_id%253Dinfo%253Apmid%252F26721855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi8zLzExODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.55" data-doi="10.1007/BF02459575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Evolution in mendelian populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D159%26rft.atitle%253DEVOLUTION%2BIN%2BMENDELIAN%2BPOPULATIONS%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02459575%26rft_id%253Dinfo%253Apmid%252F17246615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIxNi8yLzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8yOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.56" data-doi="10.1007/s00122-003-1478-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic mapping in (<em>Populus tomentosa</em> x <em>Populus bolleana</em>) and <em>P. tomentosa</em> Carr. using AFLP markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-003-1478-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DD.%26rft.volume%253D108%26rft.issue%253D4%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DGenetic%2Bmapping%2Bin%2B%2528Populus%2Btomentosa%2Bx%2BPopulus%2Bbolleana%2529%2Band%2BP.%2Btomentosa%2BCarr.%2Busing%2BAFLP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1478-7%26rft_id%253Dinfo%253Apmid%252F14564399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1478-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14564399&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.299.57" data-doi="10.1186/s12870-017-1127-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhigunov</span>,  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Ulianich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Lebedeva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. I.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. V.</span> <span class="cit-name-surname">Nuzhdin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Development of F1 hybrid population and the high-density linkage map for European aspen (<em>Populus tremula</em> L.) using RADseq technology.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-017-1127-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D17%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-017-1127-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-017-1127-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/299.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/293" title="Whole Genome Analysis of a Single Scottish Deerhound Dog Family Provides Independent Corroboration That a SGK3 Coding Variant Leads to Hairlessness" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/311" title="Identification of kit-ligand a as the Gene Responsible for the Medaka Pigment Cell Mutant few melanophore" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/299" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OS8w2e0BAFHcV4qDykbAvn1foiPHAZt2bAf7Z9CHx4Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/299" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/299" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/299" class="active" data-icon-position="" data-hide-link-title="0">Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (Populus tremula)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495171" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9PRIjCpZZ7ZJoLnk1jO9IBGtxsbpNWMV9g9NqhN-noY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/299.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/299" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-eXHPE90ZeskXyUxvbOr8_CR2lGVNbDIG56zxb1KIsq4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/299" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/299.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495171" id="node495171--2" data-pisa="ggg;10/1/299" data-pisa-master="ggg;g3.119.400504" data-apath="/ggg/10/1/299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (<em>Populus tremula</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rami-Petteri Apuli</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Carolina Bernhardsson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bastian Schiffthaler</span>, <span class="highwire-citation-author" data-delta="3">Kathryn M. Robinson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7906-6891" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stefan Jansson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathaniel R. Street</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pär K. Ingvarsson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">299-309;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400504 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495171--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rami-Petteri Apuli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rami-Petteri%2BApuli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Apuli%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARami-Petteri%2BApuli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carolina Bernhardsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Ecology and Environmental Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBernhardsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernhardsson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolina%2BBernhardsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Carolina Bernhardsson</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bastian Schiffthaler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bastian%2BSchiffthaler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schiffthaler%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABastian%2BSchiffthaler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bastian Schiffthaler</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kathryn M. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BM.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BM.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stefan Jansson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BJansson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jansson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BJansson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7906-6891" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan Jansson</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nathaniel R. Street </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathaniel%2BR.%2BStreet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Street%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathaniel%2BR.%2BStreet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathaniel R. Street</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pär K. Ingvarsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P%C3%A4r%2BK.%2BIngvarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingvarsson%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP%25C3%25A4r%2BK.%2BIngvarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pär K. Ingvarsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/299" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/299&amp;title=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/299&amp;title=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/299&amp;title=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/299&amp;text=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/299&amp;title=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/299&amp;t=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/299&amp;title=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/299&amp;title=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495171/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495171" id="node495171--3" data-pisa="ggg;10/1/299" data-pisa-master="ggg;g3.119.400504" data-apath="/ggg/10/1/299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring the Genomic Landscape of Recombination Rate Variation in European Aspen (<em>Populus tremula</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rami-Petteri Apuli</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Carolina Bernhardsson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bastian Schiffthaler</span>, <span class="highwire-citation-author" data-delta="3">Kathryn M. Robinson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7906-6891" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stefan Jansson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathaniel R. Street</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pär K. Ingvarsson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">299-309;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400504 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495171--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rami-Petteri Apuli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rami-Petteri%2BApuli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Apuli%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARami-Petteri%2BApuli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carolina Bernhardsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Umeå Plant Science Centre, Department of Ecology and Environmental Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBernhardsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernhardsson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolina%2BBernhardsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3258-275X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Carolina Bernhardsson</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bastian Schiffthaler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bastian%2BSchiffthaler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schiffthaler%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABastian%2BSchiffthaler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9771-467X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bastian Schiffthaler</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kathryn M. Robinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BM.%2BRobinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robinson%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BM.%2BRobinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stefan Jansson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BJansson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jansson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BJansson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7906-6891" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan Jansson</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nathaniel R. Street </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Umeå Plant Science Centre, Department of Plant Physiology, Umeå University, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathaniel%2BR.%2BStreet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Street%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathaniel%2BR.%2BStreet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6031-005X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathaniel R. Street</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pär K. Ingvarsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Linnean Centre for Plant Biology, Department of Plant Biology, Uppsala BioCenter, Swedish University of Agricultural Science, Uppsala, Sweden,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P%C3%A4r%2BK.%2BIngvarsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingvarsson%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP%25C3%25A4r%2BK.%2BIngvarsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9225-7521" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pär K. Ingvarsson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:par.ingvarsson@slu.se" class="" data-icon-position="" data-hide-link-title="0">par.ingvarsson@slu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=299&amp;rft.epage=309&amp;rft.atitle=Inferring%20the%20Genomic%20Landscape%20of%20Recombination%20Rate%20Variation%20in%20European%20Aspen%20%28Populus%20tremula%29&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Apuli%2C+Rami-Petteri&amp;rft.au=Bernhardsson%2C+Carolina&amp;rft.au=Schiffthaler%2C+Bastian&amp;rft.au=Robinson%2C+Kathryn+M.&amp;rft.au=Jansson%2C+Stefan&amp;rft.au=Street%2C+Nathaniel+R.&amp;rft.au=Ingvarsson%2C+P%C3%A4r+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495171/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495171/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495171/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495171/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495171/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495171/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495171/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495171/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495171/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495171/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495171/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495171/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/299" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/299.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/299.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/299.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
