<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/177" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/177.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/177.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/177.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/1/177/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/177.atom" />
<meta name="HW.pisa" content="ggg;10/1/177" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Improving Imputation Quality in BEAGLE for Crop and Livestock Data" />
<meta name="DC.Identifier" content="10.1534/g3.119.400798" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Pook et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Imputation is one of the key steps in the preprocessing and quality control protocol of any genetic study. Most imputation algorithms were originally developed for the use in human genetics and thus are optimized for a high level of genetic diversity. Different versions of BEAGLE were evaluated on genetic datasets of doubled haploids of two European maize landraces, a commercial breeding line and a diversity panel in chicken, respectively, with different levels of genetic diversity and structure which can be taken into account in BEAGLE by parameter tuning. Especially for phasing BEAGLE 5.0 outperformed the newest version (5.1) which in turn also lead to improved imputation. Earlier versions were far more dependent on the adaption of parameters in all our tests. For all versions, the parameter ne (effective population size) had a major effect on the error rate for imputation of ungenotyped markers, reducing error rates by up to 98.5%. Further improvement was obtained by tuning of the parameters affecting the structure of the haplotype cluster that is used to initialize the underlying Hidden Markov Model of BEAGLE. The number of markers with extremely high error rates for the maize datasets were more than halved by the use of a flint reference genome (F7, PE0075 etc.) instead of the commonly used B73. On average, error rates for imputation of ungenotyped markers were reduced by 8.5% by excluding genetically distant individuals from the reference panel for the chicken diversity panel. To optimize imputation accuracy one has to find a balance between representing as much of the genetic diversity as possible while avoiding the introduction of noise by including genetically distant individuals." />
<meta name="DC.Contributor" content="Torsten Pook" />
<meta name="DC.Contributor" content="Manfred Mayer" />
<meta name="DC.Contributor" content="Johannes Geibel" />
<meta name="DC.Contributor" content="Steffen Weigend" />
<meta name="DC.Contributor" content="David Cavero" />
<meta name="DC.Contributor" content="Chris C. Schoen" />
<meta name="DC.Contributor" content="Henner Simianer" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Improving Imputation Quality in BEAGLE for Crop and Livestock Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Imputation is one of the key steps in the preprocessing and quality control protocol of any genetic study. Most imputation algorithms were originally developed for the use in human genetics and thus are optimized for a high level of genetic diversity. Different versions of BEAGLE were evaluated on genetic datasets of doubled haploids of two European maize landraces, a commercial breeding line and a diversity panel in chicken, respectively, with different levels of genetic diversity and structure which can be taken into account in BEAGLE by parameter tuning. Especially for phasing BEAGLE 5.0 outperformed the newest version (5.1) which in turn also lead to improved imputation. Earlier versions were far more dependent on the adaption of parameters in all our tests. For all versions, the parameter ne (effective population size) had a major effect on the error rate for imputation of ungenotyped markers, reducing error rates by up to 98.5%. Further improvement was obtained by tuning of the parameters affecting the structure of the haplotype cluster that is used to initialize the underlying Hidden Markov Model of BEAGLE. The number of markers with extremely high error rates for the maize datasets were more than halved by the use of a flint reference genome (F7, PE0075 etc.) instead of the commonly used B73. On average, error rates for imputation of ungenotyped markers were reduced by 8.5% by excluding genetically distant individuals from the reference panel for the chicken diversity panel. To optimize imputation accuracy one has to find a balance between representing as much of the genetic diversity as possible while avoiding the introduction of noise by including genetically distant individuals.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/177" />
<meta name="citation_id" content="10/1/177" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/177" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/177.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/177.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/177.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400798" />
<meta name="citation_pmid" content="31676508" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="177" />
<meta name="citation_lastpage" content="188" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Torsten Pook" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Animal Breeding and Genetics Group," />
<meta name="citation_author_institution" content="Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany," />
<meta name="citation_author_email" content="torsten.pook@uni-goettingen.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7874-8500" />
<meta name="citation_author" content="Manfred Mayer" />
<meta name="citation_author_institution" content="Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2687-8316" />
<meta name="citation_author" content="Johannes Geibel" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Animal Breeding and Genetics Group," />
<meta name="citation_author_institution" content="Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7172-3263" />
<meta name="citation_author" content="Steffen Weigend" />
<meta name="citation_author_institution" content="Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany," />
<meta name="citation_author_institution" content="Friedrich-Loeffler-Institut, Institute of Farm Animal Genetics, 31353 Neustadt-Mariensee, Germany, and" />
<meta name="citation_author" content="David Cavero" />
<meta name="citation_author_institution" content="H&amp;N International, 27472 Cuxhaven, Germany" />
<meta name="citation_author" content="Chris C. Schoen" />
<meta name="citation_author_institution" content="Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany," />
<meta name="citation_author" content="Henner Simianer" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Animal Breeding and Genetics Group," />
<meta name="citation_author_institution" content="Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany," />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Albrechtsen;citation_author=F. C. Nielsen;citation_author=R. Nielsen;citation_title=Ascertainment biases in snp chips affect measures of population divergence.;citation_volume=27;citation_year=2010;citation_pmid=20558595;citation_doi=10.1093/molbev/msq148" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=L. E. Baum;citation_author=T. Petrie;citation_title=Statistical inference for probabilistic functions of finite state markov chains.;citation_pages=1554-1563;citation_volume=37;citation_year=1966;citation_doi=10.1214/aoms/1177699147" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. W. Bellott;citation_author=H. Skaletsky;citation_author=T. Pyntikova;citation_author=E. R. Mardis;citation_author=T. Graves;citation_title=Convergent evolution of chicken Z and human X chromosomes by expansion and gene acquisition.;citation_pages=612-616;citation_volume=466;citation_year=2010;citation_issue=7306;citation_pmid=20622855;citation_doi=10.1038/nature09172" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=Tassel: Software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Efficient multilocus association testing for whole genome association studies using localized haplotype clustering.;citation_pages=365-375;citation_volume=31;citation_year=2007;citation_issue=5;citation_pmid=17326099;citation_doi=10.1002/gepi.20216" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype imputation with millions of reference samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=Y. Zhou;citation_author=S. R. Browning;citation_title=A one-penny imputed genome from next-generation reference panels.;citation_pages=338-348;citation_volume=103;citation_year=2018;citation_pmid=30100085;citation_doi=10.1016/j.ajhg.2018.07.015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. Groenen;citation_author=P. Wahlberg;citation_author=M. Foglio;citation_author=H. H. Cheng;citation_author=H.-J. Megens;citation_title=A high-density snp-based linkage map of the chicken genome reveals sequence features correlated with recombination rate.;citation_volume=19;citation_year=2009;citation_pmid=19088305;citation_doi=10.1101/gr.086538.108" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=R. Babu;citation_author=G. de los Campos;citation_title=Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs.;citation_pages=654-663;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.07.0358" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. M. Hickey;citation_author=B. P. Kinghorn;citation_author=B. Tier;citation_author=J. F. Wilson;citation_author=N. Dunstan;citation_title=A combined long-range phasing and long haplotype imputation method to impute phase for SNP genotypes.;citation_pages=12-12;citation_volume=43;citation_year=2011;citation_pmid=21388557;citation_doi=10.1186/1297-9686-43-12" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. C. Hölker;citation_author=M. Mayer;citation_author=T. Presterl;citation_author=T. Bolduan;citation_author=E. Bauer;citation_title=European maize landraces made accessible for plant breeding and genome-based studies.;citation_pages=3333-3345;citation_volume=132;citation_year=2019;citation_doi=10.1007/s00122-019-03428-8" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.;citation_pages=695-716;citation_volume=432;citation_year=2004;citation_issue=7018;citation_pmid=15592404;citation_doi=10.1038/nature03154" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Jiao;citation_author=P. Peluso;citation_author=J. Shi;citation_author=T. Liang;citation_author=M. C. Stitzer;citation_title=Improved maize reference genome with single-molecule technologies.;citation_pages=524-527;citation_volume=546;citation_year=2017;citation_pmid=28605751;citation_doi=10.1038/nature22971" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Kranis;citation_author=A. A. Gheyas;citation_author=C. Boschiero;citation_author=F. Turner;citation_author=L. Yu;citation_title=Development of a high density 600k snp genotyping array for chicken.;citation_pages=59;citation_volume=14;citation_year=2013;citation_pmid=23356797;citation_doi=10.1186/1471-2164-14-59" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Li;citation_author=M. Stephens;citation_title=Modeling Linkage Disequilibrium and Identifying Recombination Hotspots Using Single-Nucleotide Polymorphism Data;citation_pages=2213-2233;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704198" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Lin;citation_author=D. J. Cutler;citation_author=M. E. Zwick;citation_author=A. Chakravarti;citation_title=Haplotype inference in random population samples.;citation_pages=1129-1137;citation_volume=71;citation_year=2002;citation_issue=5;citation_pmid=12386835;citation_doi=10.1086/344347" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=P.-R. Loh;citation_author=P. F. Palamara;citation_author=A. L. Price;citation_title=Fast and accurate long-range phasing in a uk biobank cohort.;citation_pages=811-816;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3571" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. K. Malomane;citation_author=H. Simianer;citation_author=A. Weigend;citation_author=C. Reimer;citation_author=A. O. Schmitt;citation_title=The synbreed chicken diversity panel: A global resource to assess chicken diversity at high genomic resolution.;citation_pages=345;citation_volume=20;citation_year=2019;citation_doi=10.1186/s12864-019-5727-9" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_author=S. Myers;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A new multipoint method for genome-wide association studies by imputation of genotypes.;citation_pages=906-913;citation_volume=39;citation_year=2007;citation_issue=7;citation_pmid=17572673;citation_doi=10.1038/ng2088" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Money;citation_author=K. Gardner;citation_author=Z. Migicovsky;citation_author=H. Schwaninger;citation_author=G.-Y. Zhong;citation_title=Linkimpute: fast and accurate genotype imputation for nonmodel organisms. G3: Genes, Genomes;citation_pages=2383-2390;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Theory Probab. Appl.;citation_author=E. A. Nadaraya;citation_title=On estimating regression.;citation_pages=141-142;citation_volume=9;citation_year=1964;citation_doi=10.1137/1109020" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. Nei;citation_title=Genetic distance between populations.;citation_pages=283-292;citation_volume=106;citation_year=1972;citation_doi=10.1086/282771" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=L. R. Rabiner;citation_title=A tutorial on hidden markov models and selected applications in speech recognition.;citation_pages=257-286;citation_volume=77;citation_year=1989;citation_doi=10.1109/5.18626" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Sargolzaei;citation_author=J. P. Chesnais;citation_author=F. S. Schenkel;citation_title=A new approach for efficient genotype imputation using information from relatives.;citation_pages=478;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-478" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. S. Schnable;citation_author=D. Ware;citation_author=R. S. Fulton;citation_author=J. C. Stein;citation_author=F. Wei;citation_title=The B73 Maize Genome: Complexity, Diversity, and Dynamics;citation_pages=1112-1115;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965430;citation_doi=10.1126/science.1178534" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. J. Scott;citation_author=K. L. Mohlke;citation_author=L. L. Bonnycastle;citation_author=C. J. Willer;citation_author=Y. Li;citation_title=A genome-wide association study of type 2 diabetes in finns detects multiple susceptibility variants.;citation_volume=316;citation_year=2007;citation_pmid=17463248;citation_doi=10.1126/science.1142382" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=K. Swarts;citation_author=H. Li;citation_author=J. A. Romero Navarro;citation_author=D. An;citation_author=M. C. Romay;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.;citation_pages=1-12;citation_volume=7;citation_year=2014;citation_doi=10.3835/plantgenome2014.05.0023" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Unterseer;citation_author=E. Bauer;citation_author=G. Haberer;citation_author=M. Seidel;citation_author=C. Knaak;citation_title=A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k snp genotyping array.;citation_pages=823;citation_volume=15;citation_year=2014;citation_pmid=25266061;citation_doi=10.1186/1471-2164-15-823" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Unterseer;citation_author=S. D. Pophaly;citation_author=R. Peis;citation_author=P. Westermeier;citation_author=M. Mayer;citation_title=A comprehensive study of the genomic differentiation between temperate dent and flint maize.;citation_pages=137;citation_volume=17;citation_year=2016;citation_doi=10.1186/s13059-016-1009-x" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=S. Unterseer;citation_author=M. A. Seidel;citation_author=E. Bauer;citation_author=G. Haberer;citation_author=F. Hochholdinger;citation_title=European flint reference sequences complement the maize pan-genome.;citation_year=2017;citation_doi=doi: 10.1101/103747" />
<meta name="citation_reference" content="citation_journal_title=Zuchtungskunde;citation_author=S. Weigend;citation_author=U. Janßen-Tapken;citation_author=M. Erbe;citation_author=U. Ober;citation_author=A. Weigend;citation_title=Biodiversität beim huhn–potenziale für die praxis.;citation_pages=25-41;citation_volume=86;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=G. Yan;citation_author=R. Qiao;citation_author=F. Zhang;citation_author=W. Xin;citation_author=S. Xiao;citation_title=Imputation-based whole-genome sequence association study rediscovered the missing qtl for lumbar number in sutai pigs.;citation_pages=615;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-00729-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Zhang;citation_author=X. Zhan;citation_author=N. A. Rosenberg;citation_author=S. Zöllner;citation_title=Genotype imputation reference panel selection using maximal phylogenetic diversity.;citation_volume=195;citation_year=2013;citation_pmid=23934887;citation_doi=10.1534/genetics.113.154591" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Improving Imputation Quality in BEAGLE for Crop and Livestock Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="Imputation is one of the key steps in the preprocessing and quality control protocol of any genetic study. Most imputation algorithms were originally developed for the use in human genetics and thus are optimized for a high level of genetic diversity. Different versions of BEAGLE were evaluated on genetic datasets of doubled haploids of two European maize landraces, a commercial breeding line and a diversity panel in chicken, respectively, with different levels of genetic diversity and structure which can be taken into account in BEAGLE by parameter tuning. Especially for phasing BEAGLE 5.0 outperformed the newest version (5.1) which in turn also lead to improved imputation. Earlier versions were far more dependent on the adaption of parameters in all our tests. For all versions, the parameter ne (effective population size) had a major effect on the error rate for imputation of ungenotyped markers, reducing error rates by up to 98.5%. Further improvement was obtained by tuning of the parameters affecting the structure of the haplotype cluster that is used to initialize the underlying Hidden Markov Model of BEAGLE. The number of markers with extremely high error rates for the maize datasets were more than halved by the use of a flint reference genome (F7, PE0075 etc.) instead of the commonly used B73. On average, error rates for imputation of ungenotyped markers were reduced by 8.5% by excluding genetically distant individuals from the reference panel for the chicken diversity panel. To optimize imputation accuracy one has to find a balance between representing as much of the genetic diversity as possible while avoiding the introduction of noise by including genetically distant individuals." />
<meta name="og-title" property="og:title" content="Improving Imputation Quality in BEAGLE for Crop and Livestock Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/177" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Imputation is one of the key steps in the preprocessing and quality control protocol of any genetic study. Most imputation algorithms were originally developed for the use in human genetics and thus are optimized for a high level of genetic diversity. Different versions of BEAGLE were evaluated on genetic datasets of doubled haploids of two European maize landraces, a commercial breeding line and a diversity panel in chicken, respectively, with different levels of genetic diversity and structure which can be taken into account in BEAGLE by parameter tuning. Especially for phasing BEAGLE 5.0 outperformed the newest version (5.1) which in turn also lead to improved imputation. Earlier versions were far more dependent on the adaption of parameters in all our tests. For all versions, the parameter ne (effective population size) had a major effect on the error rate for imputation of ungenotyped markers, reducing error rates by up to 98.5%. Further improvement was obtained by tuning of the parameters affecting the structure of the haplotype cluster that is used to initialize the underlying Hidden Markov Model of BEAGLE. The number of markers with extremely high error rates for the maize datasets were more than halved by the use of a flint reference genome (F7, PE0075 etc.) instead of the commonly used B73. On average, error rates for imputation of ungenotyped markers were reduced by 8.5% by excluding genetically distant individuals from the reference panel for the chicken diversity panel. To optimize imputation accuracy one has to find a balance between representing as much of the genetic diversity as possible while avoiding the introduction of noise by including genetically distant individuals." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Improving Imputation Quality in BEAGLE for Crop and Livestock Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9Y29ZaEYa-PB0dKP27saIYLHnl6LjleZsnyTKEuj6TE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495275","apath":"\/ggg\/10\/1\/177.atom","pisa":"ggg;10\/1\/177","ac":{"\/ggg\/10\/1\/177.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/177.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/177"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/177"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/177":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J12Nibxyu6VhPqE29EKgGyvZb1wTYzhxUjJF3s+zF8RqxNzeuuvTZvl6H\\\\\\\/wupCrQTgMGNNFlq\\\\\\\/Rlv7z1ku2Qb6EYeY2B35XrHpUKdugyUZm647LyAV2ewVIEqbbuxxmMzgb\\\\\\\/eXwfrX59jf7ArqgZF9nRJfrVDwAIXSr3bonrixBM3xs77johRvoUKeZKyShqehPnDM4nCMiDOOxAuFxAPFY3OV1llJIPQRFJq3ON6B65tswI92fTKWrd2VV6L0keSJYu2lAP7BLIpLaUqo7fi3hYYck0skRYra4DD8Y9QPCWJlJgxBLQ4do7KdO250qVSTkGP4nsAGXO5nvUO6SBZF+6reQk6Wy58TeNic9HVU\\\\\\\/fSInOy0ItPjZgUIRrD7QJqiqcl2VgsTaXtOJAFuAQcQgxkjEDZy3ovXvYkfeypoxRQ3qi\\\\\\\/oDQ0Qw6nFo0fW+4torMYI+vLCfvza5mm4mWYAaftvq+A7qqIRf0+EiacggJ6sr9Vk4Z7Z4bDUqxFlyrxlEgZ7dpNI8fTS9irfZWP08PjB7RmXLYW6vjDWsa1XoBS6qx4dgjd0TBMaC48roNg+OSf7rLIz5WM1I9ClHTvjlF3n4Sw1ELS5KUOs8pLXoCpcgDcpKFLodCku4xSzlk\\\\\\\/iCfVzL3ozgVl\\\\\\\/bW+4c944nuVIb+yBhcccIY\\\\\\\/+\\\\\\\/X4+H2hxODyBM5TEoHePZaxtPKrZNy9bshvoOt30AsERsdbLD4BKwID9XFZN3nIdQMnTZGBdupVPp3BPP9y\\\\\\\/5sChSP0Fg3mk9aiPEPLwHlmMdR4VotXY1hCnwHxGaux\\\\\\\/rpQNKP3hjzZu08PkGOEEOP\\\\\\\/tHtjpLxCv1Unv+CyPz3bfUGqZzwCipMxPaRCWPHLslwrp1xas5zmEV3kWNmeJkr0GHOSmasE\\\\\\\/pqkZ4Xhx8KWsYtXpD2\\\\\\\/8TJBFuhneVaCpjHtNnrcSxwSnDgt1BgdeXhXKCcFfE+N\\\\\\\/MR4LCPrT8Lm07i0zm9mQue8O+wF0ZFKRrpqRxZAwHK4e4uo98Yuy6e0vAACAP52QsbBlWmANAc4oVSiV+wnjgzs9Za+lIBr36989V3rsYjhGPTZCXFl86C6GjfLRZmFeL6N2T1c+4Ykk4Kd0go6DEl3A0\\\\\\\/6YK1BMsoZrBs7MKJTgZTuf+6WOCZ2x6kUZcN8Md\\\\\\\/8Cs2fcsMq4nHubzMolHAsbmLIyy83Wk+3quMFXLtQ0wNLJAHi33xY828yNrl3xTniy1L1Djkruvz+jjL15eNZeX9h2eSCcbkhAs8\\\\\\\/\\\\\\\/g4ETHh4PjupCW164+9\\\u0022,\\\u0022iv\\\u0022:\\\u0022nmQzrMNLRIW18TSWWBLRSQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225780aa2e969b0f6a11ed5abd960e37b0a72a83600f91a3c643c517ae993adaf6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QhDKNGDX\\\\\\\/zyWAXCySvPmYAaJQqZBGzbK\\\\\\\/wI9MTRledvF3oxx\\\\\\\/Ocv64zHQ+OrR9c5TvoL8z6Lcv2zyEJP+HzNji\\\\\\\/hZMHQdIuAL1DuuRGPr\\\\\\\/LSRaaZMLBLbD\\\\\\\/bRk5eeSORtfdE1c7YXCXQ3\\\\\\\/d6S\\\\\\\/Y6PfoaKj00XnMWgAdQEmWBd8PgYP7dkccjnGihh0fKs+NtZEzXGQLuZdlgZYF+j\\\\\\\/tP1mFtHmexS9g\\\\\\\/vFajRHd8FW\\\\\\\/kl\\\\\\\/y5uQFvsH+2K0kRzgS8LxI69g0PV0yokPemG0H+HMycoNlJXo2rzfymUArwWxobOIavyybpDnJpRxVeJd4LVeQmoM80j\\\\\\\/OBGout38XFfRoTeG3JgFqaazFgmYUPjfcymHJjz6OoM8KUEnDz9aqSZhlV44a0JNn9cY8SswBFMTuWO2FLtOQJtNfeGPaYYTnG+\\\\\\\/VsEO2WFBI1RLSwLPPSt4i7JqT2FasafsnlrifgiVLQmheFi6Xr03mwEH8NAZidYnirWy+qWe4Gy\\\\\\\/pcK5egHRHz3KJuJ6UKSBNqsg\\\\\\\/LO5NXGNBRRckuOMeIbvxVnHDIuPRxB9fhfxsucu5CCvvBbjaYb8PMOgmH++1xraIkWnxFnOl6S\\\\\\\/vSI5MVXbdLwYTLMQybFhQLJQv01M4fA5KVPlUeLBVwF7BiYVlwH6\\\\\\\/MzZZL9ijUybPwe3AO4Q0uVQ3gx5P3ugHG5bGbMC\\\\\\\/K2n3lIEDLKPWFgzSTDJdsQDQU\\\\\\\/zpIPy77EEDCHcWRsO\\\\\\\/38O0CrQZu6nMoIwB1DfO\\\\\\\/NW3syiLfEHHJY7YnWUJFxv1GtpAqKpR2SC8BGrBhl\\\\\\\/0ywMv\\\\\\\/7R428O+VgRB6ZKepH9mARLy1qBfOlK8r9vURkl3QQPxYa6fQ8jqWR5ICofYlptYVdQqD0HVXmAB6F54Me3CQqsrdhFjHPTQPGJh1sBe+3sz1N7dzB1t\\\\\\\/T7TJna+v7\\\\\\\/xcf+Ma7cibtFdo1WhSdjMYUe7o0k2Rg4qdnHHVkxYZdLsIfhNdzT7y4u4ee1N0SJ9hhrOEouAJoFD6cAxs+LZoMKjI0\\\\\\\/jTbogyKt95MCbWijtM+F4NP5f2Xa7D6SQ2wnzn\\\\\\\/bAUj4cDMZnSwDYeyswRoXAL7VyMx92VcHvxS\\\\\\\/15+iBq4L0IIokdqWWS+\\\\\\\/MFGi+x3ngmldziDcqx3gs8wVIam921HT316Jujl1e1MHIR+YF\\\\\\\/KQIYoztPKp1PUvapDYcXM2\\\\\\\/14zcj63JWLh5CZ4gNbWlMJSvJW5A\\\\\\\/L8PoSDKNGE7u3s8yYWRyiIIK0mR1vct6Q5IYHfTfxvOJMYj17yTSDdfW0Br2K\\\\\\\/mbnVS2lBVfHwcIxUYRrQ65fEFmR1pYqS1GoQXuR6KdJh2eVC1sa4M8q6sbsIVw0Sw1GrknzUJ\\\\\\\/x+lVlx+zVOLORnCzd+Dy4latcZk+O6uP83NFS9JhyT+4GnwyjWeDKH1\\\\\\\/EeWByE9EsMzqBxKQOW6VHqdC2ML4VB7W+h8phgiMCzzlYgnvTbmqUV0l1\\\\\\\/kqG1Mqj2vgx7lyjsEhFGrocHAp1\\\\\\\/dbxjhyUxHS1lLuVci5t8V4Fxsu1qMhWan5Zf+YDMRuPKg1VMc99VkODXRg\\\\\\\/mN455JIiwfxCTXHOjw\\\\\\\/BRmeKJljKA\\\\\\\/47X+hFqmlP9rX0Q79RfDxBQxqr0hCUqjqZDhkvS1Y8M5A9+3Ad6YRhXVIVRS5mqEKYej8YRMcoIooLQ+kH0UvlPzGiYnWmdaedmUaer8W0nxf3dzV9j6AP8TSVW1fspl8QCVPmJe3BhV8z5EZBeGaRqTgROGEW4\\\\\\\/drQo15nyYcwhxFQiF8d\\\\\\\/Ht6d+cPx7z+sufgyUpBpcH\\\\\\\/CZdTTk8ShOFfseOOGh2enYj\\\\\\\/nPr+kU69pi6a\\\\\\\/4Thp8EAPFJI9KWhOP9YHyyMyvlL0JvtP7YJz4HLYNBWQQAhmFnmIWYy\\\\\\\/z9gsOc7WryZPENiEii4ILjHQyWt7hELLhkL6k0gUUTUDdNcxN6YYyrpySvtP7IELPGBlGMnPxPpdmJpXDHYb2sVWc94xzPHTQeZLD1UE0Zc5ZimzIXdS2slxU6offLnZNqUgdJesy\\\\\\\/DiSJMDWQhzrmR32vTur9k3AgUpvLkzpf3GrYdPpRGfWr\\\\\\\/pbj1KhipGw7SShPgqYqOjNEgK3nBqOJdxWi7ZOPXYP4ldfGbAnq0Br9sCNNnUcyUcVfa\\\\\\\/MykqryhjHC+BeAN4jrD8SvE2tIWcVNV0adx\\\\\\\/V6EvCbPvxG\\\\\\\/\\\\\\\/kLxzLC3JpbiWwmommumuixJSQe31pgYyr6A5THAumvbeuD5R3uNDfqlQ3pO88LLPR8nzz\\\\\\\/pn4hvK6yE05lRACab4qcqoWmda2s0XTcMzfuVxmS3rB3cifsFY7LA3L5Y10u361GOTr6O\\\\\\\/idyi64kcdZSZeRfNK97B65vIdyZa1Vz5EPRcU0XOxxCBX0h2maCJyNPmLU6BG9cAQiosJAWUxWSOwIuMIfsfAdpDYWSlSxfT87eyzJKQ+tK2Kcu+5VbY7R7aEsDtitdGL2g81vNhpfrMWXNZ2WKNH5\\\\\\\/AyWLdEMvCNCBNm6g5y6M2cc0GwUoSImwLGZslmX5f3aM0Hre1NOLVSmymY\\\\\\\/dEVg1nBSimNT9jHx58D4GxTG18w9tOGun2AbEM3+WZCVPtoVfXUZI12udXp4cyLW1nTTy8E9QeR2Vk4DQTEUhtIgBl3tngANPNfjFyk4DIJc55Okrul5QM0QTHB54g55IAKldWrLd0CwFWlG+\\\\\\\/9rBiqwYAXpIuHAjpE2kLMfkXUfh1zLd5H3RdYSBeNrgvRpXq55AuAq1IXy119bHEEGK1skRu3XjhTAEnHayxjnxKYxZTvLcf5Rs5Ehi47lLeoS0bjFtVZDKMDlsah45b5ke8hehy4BR2jnfmdc\\\\\\\/lfCAO\\\\\\\/I5b2TTNDx7VUshIkuj+W+bQSbcYB15\\\\\\\/LtlXVERZNEnIMBfatPQAVy3FMV7LpYgHF\\\\\\\/Yxb52EmVcpVX9VfmweLEbUJ1TJT0M1uqwB\\\\\\\/TDdJJObrAgY7s6bLO3dJ6Du\\\\\\\/gWd83VkPWDuz0Zg6YcyrwxLjOGUMuoHjVvJ9By1w17x6PVaMkOI5K9e2aCTHAMEmVNGq\\\\\\\/bPBL\\\\\\\/Uyr0lmOe8tJ79ek1D8jEtqPZWNPeMws7tgVq1sTfGmNUvjW\\\\\\\/ECD8L6YjlST7UB8O912WqsYEM+vcMccbp3xrGsZJp\\\\\\\/1PKj5x\\\\\\\/El8moxWf3kKOf0p9qhcKoYhfGqm+f9sorMuskrr\\\\\\\/eKH7y2C+Srd8pu4b2HkjTS7w17MKxe4v7jH7kM9Ygt9VxtSxdY7qxp00jYPovuFN\\\\\\\/SqEiFoqN0Y9NkslslyNSVtfk2flLHfr0Z95fJz5oT5C++3duiwAFqSIFws9QMAvQex8FDExY4MWDI+a3QAC7ZTUA3BauIV1LIV9VKkWY4sLQ49Ke7IcOH4vvAfNyXh+lUP8J6lfGKxhap8uHb7nAFa9lZRLnta\\\\\\\/C\\\\\\\/pvc2\\\\\\\/zgiFR\\\\\\\/enWXCV\\\\\\\/ukpAVkjry4CZ2ztNvW9l5ezB5fhboFreEiSA3ZYFTKjCQv7wXFCAT38ub\\\\\\\/QjdEmXvvH7aw80nWKoCtaIkFVdI55JEboeSWpV51PI4hMzAkI9E8aWfvpr6jKYryf0KiTjt46Jzrh+20L424naILDcwsJVQE6vKNiVd4D51A9Exxs\\\\\\\/uRqrQrBQ4OLkSNKctAtX5OwgrNH2t5Xlatjx4zjey7Gu9bRHkc08jifk0pd6TNUllXvACFNgR15r1dTPGszRi7TdHKwK99wdMAt\\\\\\\/LAdNqbAr2ZBY3heMYTdaaCm6RwduVPMYr+TaTJdfancS1FOdbdX9tQfo1U0WgQEi0+NcVXvffMpsdRO2Um1NiDD1G2Kqd+Rr8CL7vrjh47C94dUw8ykK77ESdS36qDjDi6LEyjvTJ2rV9qBHSrG23PMPepwkWcAiVFsLmh4mAQd4aA10tM2P3goorQiWVNDMMGsKrtqJbGBPzj7Gx922DI8F6s\\\\\\\/4Ww3tGnOixFUQgSlXlbfDySgVWld8B\\\\\\\/oz663zmPeDmXakGuvRFAZEov+VCBu9qaAMs5WuySLvRFRg0MtjrONWdeOuqO98PllglwDIPPtq1izCzmzdoVdHqG45k68x9Wq2FsYnAQl+3awYyVbAG8NBNt7mcgGRpIA+\\\\\\\/9+dMc4jcD0hQlXApHNzZEpfsUkZoUnthfAfC7VlYWOiJT1+WgCI6jA4cdzLFSNWefiNP6pTYKXAVeoqw0PrK6WY1k0qFjv5qdMlVmNg5JLkC2OSD4BHABf1\\\\\\\/ZhiVQ8fKPMNfX4rKh3SSybdkPE1ydGqpv0WhISfO5n\\\\\\\/zY5hYNEGKEI+oApr6+0UZ5uil\\\\\\\/TFCgqiOny5d11QeWkcbzZKzVlsM83MMq0X7IaLqIolvQECvlWN3m8qRzsB\\\\\\\/eGjJ+mQzAt4IYSPohwWjTUxKa+B6aSJIMRG0Fczcjkenom96zXJwztezjRTeVviYLdLWAJ4F+OK6f0DjjRDBRHXlkicBa\\\\\\\/ziE6J9uMa0Jdmp46lpFfYFgSnIduuwIidK2CGKyGZT6UNNchGTyPptuYbXwb6odR5UerRUWQQBTzLoMJg16QjslX2Urlhtj\\\\\\\/Ek1dQ4TB34rHb9+Nru7KfWM9dvqHDhNsy0nU9C+1EmlMXzNtTg03WNM3T0G6PGPp\\\\\\\/1dVouWd8ZuqccIvvJ77sYGFUJEWGoHBLEWwdcF5oGgUiW8e23qFsPQQZCZ0a4PbYRNJedusgoVCQ0b1Ti3CgO68OLrqO8xKQW02ggPpoYAhNBvwNAdme\\\\\\\/9JhCIUfF2LlQOAcmxt6i9daLA3MfpKUIERbUN1sHoOAbuyeCyyu1LY+dL7BZ4VtZTVlhIRDRY0gXJm5qYhsVTY8LqGPHp2mLtVgOwrrEUTT+uIVblDwa0D4UWWZ335IZXbHLYzoc2nd3s7DT4WwlCsLczLfCze0UKB53ZL2qzIuWKzxbPKKazFWFgqbDXzt8F2gUTI0us5KKcoBWvRPAdUlfuLwsUUE5fmteSEkAorlTjPYCY\\\\\\\/ELeoJRjHbdLj8q0MAF2ptJ5un1z6gwPJfarU6WUBrbBHYjAdLKK+GI5hNdKa2kWnKXhUu5hu9OnxBi7Lk70v8tPfNucjX4CpNUM81N5XfAd0kui0YSH+WeX5Cj\\\\\\\/cMKSJekHHxSO\\\\\\\/9xryqYUhX6FwMi5R4gICC0x75NNDw8P0b5A8J6V9uwWegOuSP5YSzKeLHZW1qN\\\\\\\/0Zdue1gglHteJ2Onw72qK9pYTIDH7LMz6r3MeAMogzOc3tqxFyLUNbj2dEyReoXmZIkU\\\\\\\/UcXCU5rzbJbUej0UjPOej8PlrEzk1abWGYIuxVtf8BNK8jVYwM\\\\\\\/ylatErLxmkyKbzEd7VCTdLFM\\\\\\\/7QMXWrTOafVIzcVIzq6jkc3PVEqgiGFoij5VcPnuRmiZk81+z\\\\\\\/HxwpiBzKXNq97mEGLJ08i1hhRFJJVJMbBXXpypgStta2SIB3j8ZGJmkRiyBxbroGQV7+NElNxzFSoK2V67jCbiX7ATrzoykbLs8aENbH9BThdNzcc6Ru\\\\\\\/bhOxn82wh3N6dvLUVAIVX+UTxU1\\\\\\\/1bJHO484\\\\\\\/AUMgHE3Q+dbXcXw2FAofOG2eOBHk8IGbvHoBvHF45eD9pRYCwX6xKP4HBGU9qsiZ8fIM3WuYv6MeJ0QhieDNkYgH0jDEiIaCS8fqpdxz5ORBjt\\\\\\\/OV4vCvEJmmUqNLItpyJO3cAjHlWYqdINEb\\\\\\\/PwKzzGSLnDAUNGKlK+r84qAU8Or7I1eyxkjBf1S+3Oi\\\\\\\/qq10AE8pDHXBd8XkLtXrBEIVwdeXL1S8\\\\\\\/27tPlrZwgHi5rwxM9Y84KdkqLV3qZn+lDuVRwcYoI0tVSobdzrCGpYVxGkNTXYq4Ua8dGBlEr2WWp+ompZ2l44GDNUyyEa01zyxkDS6ItnFCFADwxmPWHR6\\\\\\\/9rgpCwzvSpGe8BYwgAzwi5eW+zkTYZaknu4n2v2Ov5eOxrz1I6DUpaPdfcI4iV7bQFttgYCDTgQ50azUXhdC5N3b5SCGvyNMJRsrCFm4P32Fw7zim+NozZCbIHWfe8ZXdxTy4LMU21DmJiMXHvyO+\\\\\\\/qoCTg8r1YW+mnIwfR23WB7nCsEmJvxHYF9LzYu2Dd85RgyhO9yK8D0iqIkGv6VsSAbNe6oRQdPfFYcxvD57eqiU48sx6On3wyu8Etyh+h5iaKkjNB3w8zYhhhh+R6\\\\\\\/UdtaW0Wu48Q3\\\\\\\/8VfeF5Jk3swdeToacydmBnPkrGuC80noXUz8OEURrHLlPWgHvpup\\\\\\\/C+ugDXANunuXBi3uLgGnNI2meAEiQrxQtyFT1RO4anovSq9yqTMQl8\\\\\\\/PSA\\\\\\\/010aC3n\\\\\\\/Gs8WZVglljQNlwWjIbn8qnz50VCfu9bveF80SMHqYfVXaxUZGDVHfyICf4jhKILJBmFdRYafFrv0CK0IPS5mPWzCK6ou2lsu7y+hgdeiTvPayFMYy6SWODcR4DZEoKeZNYpAnysBviPl5Z+lwIAhaO6gkUVOCkQhYwub+13GYC6urGWo6qd2wrIn8YDHsBKUwbUJ8lYgldd9UT4zlTWT\\\\\\\/fRAtKFj5W11Dwujvtq8KXhgqlcO+qH4DWqV5OsXL7veIbo2ncVw63BtWxHdY15DbkG9SVKkkHOsDD9TMO7YmaXzRpTVucOitNtV1hK29PiUxs\\\\\\\/Ao5TMC+biiJ82BCr5dO2sAnoF3DE7wX\\\\\\\/pNjQ5rhuRhbnz74jsbehOibQ+it5GmgJHONpNfkzDK664gg3Xc5pu5hewzVuwxLdpmLA8vfRfEkIWJ9XkTdNRAWOya5vL0CzBrYhxKbfMoW1kC7bzK0L+\\\\\\\/B+9W2clUOj47xnXaOIaaG+Zvi5yWrEB7VZE58WpV\\\\\\\/HblrxOWLsiO1f\\\\\\\/ozgHVtCwgjNLkG9nsnvhVGJbY\\\\\\\/5QLQnbgUGea0tD7fGoYC6f4rXrUZGXBqvTidh+OWjWBVn4l3h53OxbmjEim9wkhdf5jYhcLU0Ebu2xmOPrda6e5UYolvwGruPMA0m23NTm5aDlM5e+K2Bak556j7U8zj+pu962XOjXUq2rOIGoUOmJQZUNd9hUkKk3X4hPF4Ssr0GRv4T0l9MyrA+ddFiC3PpE\\\\\\\/gtAKCx6ui9hxg6hUV\\\\\\\/QiM+0LP7nKB+\\\\\\\/hSAOov66Jte3QuHxYpGmD9K3weNryqofV4c94m9I10NWYGc3yean83p5axDzx8vDbi68gGH3RdnGOJO7RbxSk5tMMi\\\\\\\/jBQB9uSd3ENnRNZZHcA91jDLM\\\\\\\/K9I41mpfm0hhE4K0oqIFLM3hC9N65QLKxxAGum04I8tPvGi55ZtjWWcgq2\\\\\\\/sOsWPI5qd8krprOmdTAPnNvWxmHo4kV3Fsr+wThgiJaESXHb5zCalOL\\\\\\\/dKxi07a7uJzU\\\\\\\/UNZ+RdLl9Jlv7\\\\\\\/o+dGC6l3PQsDsWZNK7WRIaDP+vdinBss3MjfCceI76W3O43zCtWauZqOgkSNrGfgj+qKK0wNNXSchv5sT6VztF\\\\\\\/t6wu2QXi0H1rQdW1mO5Iyx4z8OHKyiGWIDER7vQvZCfMeePDSGUifLn8JASo34j2oGEAgqBM9tZd8n4GD8qq6fgyk8mNfqkBGH3W7uDmnfANU\\\\\\\/HNJu7EgHRfX8VfLQnzL1qvB8q08duqIsVtz2axIaehXC4jl2\\\\\\\/g2l88cDmhXSHHiRlxtanB+G4VgcSZ1g\\\\\\\/6NtmJZ71Q+zEi2URTTeIS1i2KGcy+adKPPbtpA6aUvT9jJP61vnfav3YD0j880kiGEd1mOP8mi9LncMQ7apok9tak6YS7p8UBsqAgsVBQoX\\\\\\\/FyJ2q1Kyn0ylH1tMjcMGhkH8VoPqk+FE0Z6pQb9mBunvqlSytFxqJTwywHJ4XgDE\\\\\\\/oDs6Ad2x5dUo0BjSBUZ19+kdAQ5h6CpMAz63dlVHuJvD+leQXrKWykzF3uD4qFm+fYKBoHXgtbe0yQHXy3vO40rGUQ6HoJ2cTZD3WPh3UB9xjV1uaSkDDd8JlJyjVWfHDiludM044198TZ2k4xGkeG4Rl7GR3IITKzGkzwlBWdEhIHpwNrJpHutqv9978sF7ug3YMKT8B1nipI9NAM9JDFZQTS1zBvlQRSwPdCYylJ9KTRMcq3ounFCuGPZII+1joWl8gtR75cUgKC+cryQFL0AK5OO2Nrvv79C5PkXx3MRWw4kvu0mAp3L7Lm8eQR0lKLy3nCC9XXnsOy8B9lwrvjJRrtgmZZlzcEBocyfdJ2\\\\\\\/mZAdqe4yvZ31hcGgZ3Y8LITx4xwLW\\\\\\\/vTIWf2TKcqemQpOXA0OBpp+h3L2x7YT7eeRSwcd9SmWt0IsvZDDE227cfgSMHH3XfdnjLEycQFmo0gnJORtwIzjNru7hqE4Sad8QqsoCWTXjCHZlpbeds87baT5fu+1snCw6xXSpcuyNg+akNQ8iH7KBVpbhV2AxTbHP7SphGiVRYngTt4ZxXGVgqt30Kzjdn1YBIHdeX3HxPSpHpa6Tz4ALy\\\\\\\/tArXhl7iuL9zBb6wnvMKbCPfPFeewZeSPDVRfA40kfQg9Plprpfhdl70a28dGGK9XivIhNWHIlyMQ0wfSDCI5Uoth3iEZw1YkyXrzCR8NxQiMvhE+eAEgw+SmNyT5MhrQCdUB+3Zmm6qu\\\\\\\/q9BgXSuw4\\\\\\\/VdRd24G7UTzUruQeh01VGUxxOHvMfLe6vUQeNHTb6SimsEPthjYeSSZXKkHr2q4frzE9Dz7Aw3zLiy\\\\\\\/o2Y2xDU8WnIiskFHTKvmFEYnzGnE0Vx1A16OymRbuPHxVaPOjlWYieM8C04EmETfD5Lvf1F9xjK0RagNFAzXgl+OIuDESTmmlWk3fC0WlfCLK\\\\\\\/c7Or69v+7Ujv8\\\\\\\/6ugVntnMUrdE5KQNlyZBIHvJU3nOPVoyQQeu7Q231KQDDpbkhjghGyinQRKQiKTt1CQZ1ddKmi1VcYdKo6FlXX\\\\\\\/UFLyC0jzBPxsmwEfG52JChvzHjbXB9oQaJyMABi8Wok4\\\\\\\/c2YMFObJqA\\\\\\\/uaxNk4rAVzd\\\\\\\/OAA+pPXZEpuyxRX1LspdGBJ3KRQgN36HtJHGlOuPCEpgAaC0z\\\\\\\/ztMm82A92w5OSramVFoO3bxxcWoEnFm4wVlAs2NVpd7OkHoaD2aYzVHgeIrIMeyCWgZJZvyslg3R3I4UvRoFde1j7u5MrV7STgYi0cNZbWeIRiqDKWQXevapmFVyCrUJy997VPS0izjNPKK2QH8Gpl3VkCDlW9ALH3OOh8CFVvhdxtZqOEomx46Bjgmy8dn5eSFDsVh9g9vefMX3nI8pg0z5UwhZnwNiDWxMq6gmTVZlPVBbwXWJLQ0utHyQtMrtOfRJTBIK4L9rCjHwXCT9RNK1x1nhZ3oXCeqZ2ZKXm51FwtS3Ob9Ah96jaNafCXRWTgR\\\\\\\/O\\\\\\\/FQd46XlxM3c8VjCjZCS7hEqUZX8TCG\\\\\\\/cphKqfoYH83fr58kBHQ47WTKYpSHqZ82Fxy7TEzmIXw\\\\\\\/0iYvUmVusFNrRtsjTYYsZZOvUkGna71lV0Of2hLsNmqW0iimmh+DRoGOBoOdNSHDajGIox8Ni\\\\\\\/RbSeqDkXDs\\\\\\\/WFgizHCk7qaj0EZxyRRs+N9Lnd5cHfmDylW4w\\\\\\\/XJg9SYEN6\\\\\\\/FuS+bBVknTtaayv6oEWj0s3Z01kBX+ah6iNU0l6yVNq9kLXnmlD2pLQYfcuh\\\\\\\/mcT9Ms7FUCYt33i1MQGGZvn+EqpcOmF2BjBFE1m38KvZ4IVUx2KOa2rvZXtrkCbtGvwWfcARBgjjJjb6ZSqtzdnXcQy60LiMZJ+znXtPtWqYeDSKqMimi55rq+csvFLtIrqbmPhtlhKkucpFB8IZXqrwRQyiIBKc9KDZ9+1gnKL60Q3Hm7NCeaanGcNtaM3lLZP2vbwqI1vMqmkTGKv9Lt7atFf4ew63zzg1CTlYrXJkdoWgteXc5hbhrZSSPDrXW0mLofc2\\\\\\\/8tobHruY34sr3xJ1lArLO1Y\\\\\\\/c9T5acMIYZrFDQpmjIttq+p1mIVQTYPkyjbl1mKBwpX5qRwPzNTBLlmujdZ5GtblJXWHazBC9riNdxXQbEUdoiejb0J8rweJZbs49cz+ElCp294sILaEZiFyq8iNpf1WbtNByWB2hP2nvPW2QM+a8HdmCEe\\\\\\\/daVnrJ7NvuOLlrADVBdrcQ0wqrwrwcm7wUCiq0i7FT4tVGT2EhIjLcQnpH+M3ezQN35Icom+6P2Y3FI06Vk0ZQK9gkrgvxRx+bhv+3vh5gEZfbJFvTxcbcomFoKmeWZV1MqFvm1F9ljc+tl0znYI5U5AGGVPYI7oToGOmAZ+VW1kO0hxOLiGdXvgboEpGRBIaCNchXXslemxjiEr4eT5k6tbtMeyybgB06aIAupFQgniYy9qhruoCymMZ+A335rQ87Z9RJDyJa44HJxvv6rfF\\\\\\\/q8GRY36AnJgBKShEn0cniaq+YMwq9KtluBrXl8axe76drSwK4TQjbvU3q93fRC4r0\\\\\\\/+yqn6bBav1oMQUH4WHht8iXgZZtbkcm9k175v1mbwuOydtIXugyz9wM\\\\\\\/QAX2WZsD3BOvhoXJJIdej3w2ie8hmhcvX4stUqHE3hd8RAHasXnK68mu1J6UgY48clGX9Ffj8hc1Lh50hewlv23CYTBjS\\\\\\\/btUzboBuDln62hiaO84PKNphAJ5G+o035RInPxmU\\\\\\\/t\\\\\\\/bgV1j08eENgW3O6XsJbg3Kdovg2k+blXUgIL5jOKByQKxDXSfPEkMoM1EoVkOjUSv1ANAkmk7M5UedbBSCUJgnF6WpAvSP2hvofEI+\\\\\\\/6ULEb1QvUYFsoAgWXoWyERqJasOTYVjHi4Pm64KH3uXRWoHKX7xCRTpOB+9ELGd6B3varx56L4YoRym0HUJTld1GbD7iEgYQTRyT9L1FPqCV3p0o2Q8UrGtBg\\\\\\\/c2ees5B6R9t1KV+0cOOUIr10ClROKnIgcTBwzYl+SRVA0haWZhAg5HNGL86Tw0BFnSvWzh1SAacDEkRlT9a3+1p3SnaAyFQKqk4YBkXu8\\\\\\\/ht4Gq7TOHlDdjlR3iMhPa+USMRR\\\\\\\/hiFIpIDoiCPuVcIvvOq9yJksxDVfo3lbs2twnQ4M49NbDOGaNusbgth5Z9KFV5G9IuRL8v+OFJufRxDcCx+6cJbL2LSPrxp0LpVRk2rb8iM4rE19X4\\\\\\\/yitRdfKNM8YVOeeOAUivoTfBG5DpVqM67qWygISTcZAJeTCA0x9PAZ7aG9OCcbSkcpvxh+QTXoCfINQ\\\\\\\/G902VDSjm5vf+9cpk6+f8lR9QR4m\\\\\\\/POL1Eo3P5y\\\\\\\/eWuKe8yIipPiYl17NzAUPFlA0uRazNHv8V0t4jqomtnwpFOYs73\\\\\\\/kHTsFEA1COfvulN4POwyDsUTeUg4W3Af+cEWoUA8F72+ydlFf65oyE\\\\\\\/F8INUvFgyjBaLk\\\\\\\/g8JMMHi72pgxHa60032cLNqBCz88bSw5xDJCkEBUuXu0mPu5X3W5aGk2SCXGYA3fGMmxen7f8VbBjpld903yRpIKoX1X166W+KvDXRZkg\\\\\\\/7UaMt+qcd\\\\\\\/UhFqeyFwnk1HHC1si3edhoCRCqmncgvkV2w8Dw0jMq4sVo0O6NI08EMVrJDHtKnZoaTOCnAYa3FS8IQTaH5N6kEd3BQ\\\\\\\/7b3lwh3+u9Kh3DsAHcqVZ155XTdign\\\\\\\/CVvo1V6Y0R2OXt4ORGH4iDgswvBMiBNPjNr6twZjRy0N05vhWGhvXpeqpuyHCsDbN15WuONjiWVEI6YFz43asFMA2Mk6XSDVWMGP1ttWq5Wo4yz6+Ug2GxicEw2oBlnNT95tl29xqaX8jcX9mQhi\\\\\\\/qZWn3UxAUkxA5p0gKTWlxsV1j49aMo0LADEigGkcNYPeUS76Sm0mRR2PM1lzuUWvwPJ4hWI6gESXsYsyUfeiTn8fN95jtXGY3rEB682PcOy7ik1RgfwTLt7+XXJVNrgxTeDgGBEFn1TCNx8elL387YjMMyRBIc6wfp7h6OcYsBOcW8j1KPaDnPX6bAF3snRx7yfVSI87qHpzD8l+bxBeWeVeFCHb9PImSRHyRMBpuicp2cVv3BAsJxb2e9azTCwQFcu11tfgN4DF9QhTRJubGsJUfcgM+w3BOGCz9eRQ98wjxtd198sQoCUfQSP3Zz++20BvcVztAkWh4iLGzhJkqpv+AkXoED4ppbWht3a24edZ4JfyvAXQ4opAaybb6o8qya3BpXttak7wihoeGZiYfSa500PYoIdsGSev2mdp5H7zjyvORCJ4r4zyazLkyCZkxl0O0fGBS3\\\\\\\/DAusydVeFIORHM5uw4ooSqOg\\\\\\\/8FA24BaUaP+vehRjEnRuRp3q\\\\\\\/spqR4\\\\\\\/\\\\\\\/ZhY6z57vgciMMc6VS2mabSgvtNHHqw0Yw\\\\\\\/PAlBrKC4Y\\\\\\\/QaIGKB3gR85G\\\\\\\/10GjwNBz2yedL7jbamfVmy\\\\\\\/CTlcEWMyp4Tnvh4hWiVfs0Kfn17Yl7vH+jBhjxLuV1O3DCdqBt6MFgUHkZjfH97Fa49W+BriQrocKyyYjQ+vfcBTg8DfGxsf41t2TRhUx3RxOGXz7j3bj4dv6cvNIRMY6jK8z+sp4+1ly7GuV0mJX+tT65qfPz8wiYpu8iJ0x49dbVUVoIMSaXZa+Zwr1lAEO7Jy2wp4YLbz8vGOQNkjAyJwmvFVHHpIB++0rRb4CygeUcVtBq95UAriOIm+sG\\\\\\\/60SOkr9fBGNAvvaaYDE7TrB+dF9E93GxuP6xPq6Z0umuB8G1TdmtnBSMYK3Pl2+65muY3vnIuiPVQNdAXnQgJ3JDH1mz8VMKmZ+xvxN5O\\\\\\\/SU4Xj55PEcN3ckJgGnF4dtIJlJhSnuyNTt+CzqkCA4I7lAjDXwrDF7xihW1LgYXhPTsMWPTZrgayEzSoS4bal0YudKCES4DeLx19i4y+tRv8WRSx2aOIlebPdV04Zcsb9eyikuHHDy3DtzmymFsEYc7Se4p9Od\\\\\\\/uzZcVwFhmEiIwBDzsgzr+\\\\\\\/S300tE9vhvENHifqm0sYr5F2QndemdOH6+EdweXFJjOBDmWU6J4fxI2RNjAoeIdLl6TX4fPzL0upaJkyIkGoOwXl4Le\\\\\\\/6c4c+58hg9LzyjHXQ6XPaBY8l3AKmGfgh7Y2ngclYcUhvi7vuA43uCzgNg8JMKF5gj\\\\\\\/e9Lid2nUxwHOIx7kOp0zd8GY9OPcMpRrumStwEoplNlxHgofKQP3Q9HWM4BneiedX8V2uJBnXv31XpCEy74p5JAuMrrB1d+WBtkeWW2EixfH6Mon7ENR\\\\\\\/5lhB6e+nv8dD\\\\\\\/GZZ1clgZEbdIp1OlHM4Sm\\\\\\\/3oeUES+sjj2zFDIHBBrPK7TBZUc\\\\\\\/N7xdhc5nvWamDSGZdA5DVRyBzlaRGus5\\\\\\\/bVHxb6H\\\\\\\/MdM2sQvWJIdGcOwaFdah183e4K33JDg3R30s82jZ\\\\\\\/CJNCvmQVA+n2qPKBUEAK\\\\\\\/tRwYOPICE0KwQxKCZIEKAyWjFY5GCSj+gBDGdKG782UJVi6CwpoMELA\\\\\\\/RrPzjZF521RogiWrWe3fo6PJF+K8WdCgJUg4UcwcUqArWQ08j0jbuthE6SF9ytvhLU5YDl8Cm21M9s50bd1bPUjO39qoY0EsW\\\\\\\/\\\\\\\/vlmuw20KyIDU1hVkYiQVU208LRYeI5SU1\\\\\\\/2oKw3Dw3ecHv7MN0u80nVhfic0ShEMsHPL0ivtr\\\\\\\/h29j1KzwErzLhH7YTZJ5g7dRx1BrdUzFSnwm\\\\\\\/kCzuU+PzX6OKAddGvVveMZUZEwQtVrxiDGRjHYSRZzbWCTWQ02rV7fehptTlHqrJa6fAtdqvKVPiz6poYZmzkrJFT0vvAsE2n+LhliSMFKH18sG6Ht83lBWo8nV1ui38lYVB89GjgZ0nAXDe8bn3dPPJnHrNySPwGM2Fg52FzgHdHIKtF607tmANi1+8JL8ooX7PleRSkiYeNCGY41vaufH624D+YVx\\\\\\\/LE3fDzkv20YL+GkmHrZuGgEmfgsnJVmRNbvDz92j1JEf\\\\\\\/gIhkTwwCURnzckL1LmupnQ2ir89G2dPcwg4YFH\\\\\\\/c2mdYQ\\\\\\\/Q02Ywu6vSks+P4fc0ZbKZ+zp7NKeWAJCST44\\\\\\\/UvexsrVJWDEHDM2ELoWvX6CU1KTTKNGzOPT5GWgRDUgyKAB91oQZrDSx4r6NWr7mhtq7T\\\\\\\/Xlb9TboOY6kcQvhCExhOVN\\\\\\\/7TltUIfH5eteSeQMq98BH2wBb14vKqBfqdKGzcqd09fc6wGeOKI50OHG+Y3qtHuahWhGmAPxinPAEtSmxy7ma\\\\\\\/UhEmxIelF5jD8\\\\\\\/MPQCB4rNCm2GtJYTDOqB8KvmivY3stbHAoPLhuiMXrdEFB4IgvP6zqlels5Zqb1Ep\\\\\\\/lvSMld+RhmWtccXT6UBRyeWhIqyPYh1Bui12pfggBtgO9mdBoMQWslHne4i5jsxg4ZZik4i\\\\\\\/mSToKU4MbkoJU26a\\\\\\\/Tg4rUmyEcSz13xynQTLEPZrRBjn+cr2oZDFwEgE2oSatDPCKXbfP5IDQwzXSpuOmtEWOT6KMTS\\\\\\\/g9w1Im18QzCiYvLfpTZ2o51F\\\\\\\/X5x1bx7LK+ey2sJsEyPN\\\\\\\/kAtSjTR04eu0URPCrQpyrepusQANVWyVTV4fb1jCNzRgSXMDwNMIBSzALIt5RpivJFypKZopvaU\\\\\\\/F42g7179wZaXyqa7CPm5c\\\\\\\/KWIVBzh+x5GdLY3glf3m7bb5TcJXcEg0fWxPGqtJd136u7bsWc5s2knBKafVDt0P7HaCpIBJl7J1bSFDpGyHDrzEWORIrPgWGiR1VpoLBm0fcfDD26FJitfEyKm2+r1Rp9aTYHUMdc5nUunqe42nsIDC4XFdv1tz18qPbtbCtBh+xc5MFi1MqHo3OfrIp2PCoO8aTUsVwkHuwvJFQfVWqOFsE31xNAHV75RxrK2O6\\\\\\\/LtOhKil6M5ZcgpfvUA5u8Cs6uLySQFV256Dx6wwjqfzl9\\\\\\\/JEqHzqj2a20cNLJy3H8ZM157\\\\\\\/Ga8aSbiN9GSfPEPvoQnclb5FqPErPowYDsgN1u17ripLhi2o4UL4DfwjwFcbzSnxlAX2SWIco2UHQe7q9t35E5giZ9iRn\\\\\\\/FJUOLlkmjrmhO8aScCOjafQ8qC1rGhnprKWq5u3K0gUv1mnRBAdfKAhqOZkbXvepggA7PFEyzWuL+vT4PPyCwlt0VPvptbTZ9LoMsWu2Rq9eYleauERwX1n53nVO8\\\\\\\/9\\\\\\\/jQjpqMVCtBS\\\\\\\/z6l1xaVCThlU1AFqIPigUvcdw2kXFyLdNRtP09pSebAlIWo57VMOowh\\\\\\\/JN6rHS9ymIabImbIgh6JZbbZf8JwQ1isL\\\\\\\/Fsc838Q4xRmxJQmflNa7qsvIrdyZ\\\\\\\/u9Ar2+Xhb7TDMOAGFNoi0lvV6mDcv5Wj4jw3ZLlz6ds5xgx9QTnBx0eSoZnNI\\\\\\\/QbUMpLm3oCF7vblylkjwAecAyilCIkjoAsvSrtA3FrmTt8MDf6LyKFssOrneMsS5s\\\\\\\/981fXBinZTPtmjuvikX6XR9vWz0FHFpYaKqdaB0Uy9\\\\\\\/E3aBW9Q4lurTvakK8ejGl9XvB13iQzKSU3ywIfbuEtTHybACpgXMba8GTYPERyHMctOva7obgKbSShLvg86e6yCLt9bdmYlHnoTwvmvkFrv38qbQWv56lMd7sY9fYgAB7HqwR86gODiEmAjv0MUY1BQVx9oRjK2KhYZZ+fa64LlV+iN+K8GQs2H+Qok5CCJwv\\\\\\\/Bt\\\\\\\/QMYIDTB4m8v9J7erP\\\\\\\/\\\\\\\/I83\\\\\\\/C8lvmEf0rgjhFhtdnKUlTgujmFnGwdeuHH8j2h3toZl0jWz997JkMnjpEnt+zzzjJkZvd8qTdsBNUEHc87iSx6Ek5i1TY6oF\\\\\\\/ncsk\\\\\\\/ztcGgOhzgNfalVgGi4p8iqT5U1Dl6LXtFbhxTGFqvPTbMiTVrMkOEqOWykkIgvOjRbcEKmTkbmVKK40Kd8ogBVfgZO\\\\\\\/3RklUUrHWU9PpVvLuFzp3vfgK329DTjgSqMGVjtLk\\\\\\\/9mufTU1vljuVvSRa5+jEBYhQyJUhHK3UKbZrG5r8g2cByMZJ5FoZ1xFwnDYvfUVpWHtzt7bf1Hbuy+bYfxfLb2AehNLomOwa64pw\\\\\\\/VG5A9nAhiBHOo3e6rOdVsgzxVjuOld8xhZi42WAhERC88WTpBxsp+6XwdFchfmMJBRncAqlu7YhTdcV3HlMSYGWjz34nSQZ22rlfT0wLy9DH0SO1L7iW\\\\\\\/wa6T5EtF6bLTcsNVhU1oDBoBPiKffNQQ5F1\\\\\\\/9sjY2x3aOLfi3rRgVB3DwwIqztDj\\\\\\\/SMT5yzaObcKNsVXG9bpIhsUuKCqDKZyPsejUn7\\\\\\\/Jyvz9hXi1eM+qJ7PZie3fIrdvHc7xt+MLXDuwaRT41myDCpasxrah2CIKMndG8jRpX8hFEzo\\\\\\\/iLfpbv0e8tt1S7U3ItbwIpJQbQ4JSLggib6jOczepYsDBXdI7ByVbX5\\\\\\\/rYaEhidkRgUCxcDedLF6NFUZEDkYpLze9EMzNEoHVYvXZqIq50qTKm2lGq63QCbEC784VrdXarDkVBSxUoEAZgunS4cQhR3xBnd5gflz1+IbJwwsvInRy6i5kV8Uiqmz+c\\\\\\\/wtmQD0HEvQqPTTEgQwZrg8dJmeILruZlm2v+P7UGNNYO1AqgwgUVU+DqR5zFqcdF1m2XKlnJhAhuGDOgSl1p1AvF3nIrTF5TGBLRvtMhI0CI\\\\\\\/ylZKkGm9HOvxAjVbZQeeOpnGgq5N41iKcfQ2mbMJ0bq0uksMymsaTaMi9HKgbTAUgCKHfJ+FUF8utHCfgJa8DfOxi8ofyE8TtOYKbRnjs4XB33q4lDkpqqiGM8xXWothLX5+MRk2feZJJI02kVbjqz0UNXr2lljDNONJuzw9BoxOkNIxJS9QG7czSV2Eb24qBQAAzxYkZJc71F3dmqYYt3UcA\\\\\\\/xAEkszF5qeBGGkGJac8JUs0USdwX321xzsGH6MDaNTUCkQAZPUiwrKqW1cQIif72jS46+CVTTXMvWVHMiHfBLLn9R9scLy80Z8F+hLPNOYj2zigkmiWR4CB0\\\\\\\/gBERb47ELftpXOEchlgvJYaMjNTPgjZiKvDN+d1Q8XPRm53AFBB+0dS0igS2J\\\\\\\/it7Y9V9OeSPwOABPJQ7rZulqg11BoM1W4iyCI7boYjdnuUTDDBaOHHViP61rczz+z\\\\\\\/x\\\\\\\/We7Dn0oeBmKJUi7503PgKtDBn5YJQA496JMLi+x1cGnLdJz4dqbFxsO1Nf7Hh\\\\\\\/vP\\\\\\\/9M4lOAVN+iG1P6Eh3RNWwmQULu7GOXZqT23FRvLuUbbVNDRickrF4L6lkIi4xl2lfWtlDBvkVFEldAH1r77RXLuFFhwJs8sXy9jx9b6YeWBu1G0ZuCKcD0mnWCjG00kDHhTsTPCRWU8l56A0ZsYwjTKJIdbyhQaSlQPCc4nNkxfHVFm0KVi6\\\\\\\/llUOS9U8qUamaLyyuqJHLRC0XfZpM6bydNS7qwEIS9InL9xLI\\\\\\\/keA5sMkLs79txYPaKIPoi\\\\\\\/qfxro8B9dp66EwcLs9SiOrkXwoJeeBLFQQqwTj1IRWkZL7\\\\\\\/o2gA8r92i4OYppDhsUgGTqF\\\\\\\/AV1A31DwB7G1UwD\\\\\\\/b3x\\\\\\\/63mb\\\\\\\/JA1NrNBeIhDFVELKfJMtZ\\\\\\\/KDc1Jk86CRdXuXDie6s\\\\\\\/WYD1oVHosvAaAcxWv8+YHeGL9G\\\\\\\/d3GBY8jqIDqkyzAAnQJxivPhjETMDymxaM7AjtdIPH7Z39lUgqVoqKMhtQ2YsOFNKmQg1P0oXWfgoTlAh\\\\\\\/KN3BJu3JBO0L2CaanXKKL1BdkzMnvlxP5Kg4FlG0MZoCh58P4wqhXDiAEp9v5Ab7rrDPRqATEq7FPnECoaNVZCcXWRM+YIxzFAPogeEyVw8H4c9ZwJyqn22rPaovNaM37X+lmMB2GvkOL6vNTXsqpN\\\\\\\/47w6nWtnxoe617YyP6tC4Q5M34Y5ZeQ12Vhpyer3jknQZmnzQc95u8goqSXt8KIU\\\\\\\/iobIUURYg\\\\\\\/I\\\\\\\/YpqEkUFyNyBeaZCUy3B+Nak0S6rm9e4se8mZIQSOq+x63kROvNU1RCPOVtYleqY1uPnbqlgJoQ6\\\\\\\/eEfqCIpgxhF3e4rk01Jz+RmaiX8Rd+n6Xfk+Pd9gWktsSnbnUNbXykJLROxNSY7kxogCRiwVQsxfUYJs7Jf1BMYoEysfsPsxJ0jmJx6NxlKzetKNmb04vVZy0BiHHhNd4pXDvogvzRpPYY7gql8HVh7+OO3mYMCMGks6c\\\\\\\/Gukz86GMJdAkcXH6OsTVMxU1+iOjtYqJrcoHB\\\\\\\/oDaTvUtCaEAroiaLW4N3tT\\\\\\\/hqD\\\\\\\/QywzMO9HOtCFHQRM79TlqfFGzBOZVFyYkRW4+H8rasqb9vcJL9IqUrmrPEHTFOg7ordmZVC3dwYwLBZBr+txqQhBy4bRlmEa\\\\\\\/gURyv9AwJp+bxUzSKFA7OuMwtfh1YFFUdha8rdT4L6bFuA6TPej7i8k6yfwzmO9p9gKw0MncjUoMFgTly4th8TJZ90Aqi0q\\\\\\\/00Fwa\\\\\\\/TA52cPxTf+bq4+QPvQdKVAtH9Kzu2\\\\\\\/uNrIGMgSCpCWE\\\\\\\/r26KE3CLJPVgTQqg2jYRwx1ztonxtRkBWzj\\\\\\\/6IiXKE1A0rRH7BhASDmCzGPk1eY76104ovQR2c+ppxNy7g230AH4qyRIBuhvst1lzuP1Morq+oygYPSbAYiy4grnH85yYf4TQjc9\\\\\\\/GwGBDEMTY1rEp0SQqU3roWdWC0jW\\\\\\\/j2HSs2nNFtj6YbJYN9loAsOIIioOlkR6GSi5z+rJk+SKddNOWN2HBmMDXNVdg2jyAX\\\\\\\/eoCKiAEWMm5XFCDeydl2Y8f3Bnyz1T3ucL0gVVhiGYAluIVqNDkR8hCNxGiadNlXhAuOk1yTfysYAsInYUwYPbiSBIhEiTG9udeqdA3SlFzan438frGs6C2FbanvC5h3tua7uE+a1Wri77LjZoFNoahbdbBb5ikXqiOmWfXSyRds5wNNdPvqL6gVi8bv5OAhVPQ5wbJnXSn\\\\\\\/TW4bCz1SPBTYIGhutFeEHtsPLkXNNNk6APun15r6gaBKo1CwEhcbpS1s38MVpMTaPNMdwJM6\\\\\\\/wMg\\\\\\\/6o2hN3Wr2eJr6m\\\\\\\/m62Yp\\\\\\\/nrHjJLosch7B4hG3wl5qf1ksZDS483UX7T91vBNggLZJo6L8lV064lrxz8RZat\\\\\\\/kWlh5\\\\\\\/Ua0v0ZBK3TzKwkAVghyXSrK4D\\\\\\\/kkbjScDX9appBP4LqfGHGsPB9b5wt+ePGEHAYIaiHPb5Wyhdd0XNlXmTXWnhfRerzqVOLMFIWrWHf0VOyVdKogP8TsDa0dpvF2ccxjmiUK8O6a0wefCspuO2Cr+EIjoDDnyxjhqr1Uc9noNAroa1zsT6FIPO+ygVZOb6o15HsfMNvJ1ZrLZ8uTw6FycbZPr9ZnqUzKSVZkzLjsGv5c6sz75gS\\\\\\\/xWOZf1bjYSBLqjcP8YbHEJwVf0lVS55E91L7SFfwSXx0GXeuSYM\\\\\\\/wPfIH\\\\\\\/+gJRIW+G6LxYHLtXky2kPhNllzURgpgM1tUthbXqKvtRVFpA1rFSj1DJ0O6ww7TQeYbQGbvTUIQ+5cnEn5RccP+ZYUSnkJ\\\\\\\/XeS3Rcs5da1E73Dwo3wuWZr4vavQXgVL7c9xhuDJiCBA90mnoIEWGTSTkogEyp+qP0CpKBt7I1BZus0J3hoOl+8r1iAvGvYqHbLCxM7FEpfncqvnQbkO4MJoBiFVJglY5jlaL9KrSF9Or\\\\\\\/2MNmNWO4iudShPVa7f3FwwPOpiHLac5MCHCFvqAE60cPcZIx8sRZ6DXTUXw1NQtzri91chqU+2g1TLkSC1AvUmRYR6FD1Bs6UtQhYHjHwFq\\\\\\\/xkaOlIpl6n88L4QZrU6WWk3vaPRzeR\\\\\\\/cdbFEipoKwZAPHI3enKFOQm2ZsYfKOpMWhVUNTCNYPb2bBa8w9nqi\\\\\\\/192I5I1RwMn1Lka0UlC8RiBjyxG7Hd2Y365JC9Iv4CM17i0hcA74\\\\\\\/npC8i6BpRkO4UJxxZkHo7aXqcr53tliop8LmIzV+pEojcEG0qPsfeMdtQoA9hLCjIsr8HEDOkM6Ci1Qewgik82MnDuXEX\\\\\\\/m8TffAGm0Rc7Kq48rRfrxYmHSC9N0xBJuQccri6\\\\\\\/Z\\\\\\\/aWjBkf7xc+l6ZcME0kj75oU4Zfz1PwczoLFszW6gxCzT5IPcsUOfS\\\\\\\/Ta60mQKU0BS9M8qLgLuU3BF1BwKe2jc6t5oFrjhQMvaNOQfVeYPhnheSZLuxSlxts\\\\\\\/TchCUUuY88Vq8TZf7kHutJf1A7EC2TbN4PxfrTYRDX6rm7z0q1H0ROZsxMxGB9b4lHfDCpMBq1KmYCAWCYArjVAX0N09OdAsGl7ipzFwc98hJOFrDVmyBHfccUgpDPTLpFweaO9lI2WVyN23vRMPuFAQ41CEOpNU6Z8AUL7v06qKlK0Bqzi925oJz1JMcLk5pTDi1L6hlI2UusS+Kt9pT7E5yWAAY0PQaJ5obeEAMmDQveosfzSPPHyFBT5mW3mOLTiG1XzCjfhFNAV45edKugxArUmlW5Vb5\\\\\\\/O8wMK6pFR2m25f3b5gA8jY7YhvHSEIjCSx0pT4E8iA3eJkfPzeWld8RH0+kEuQkj+BJfMNtVznWJ0aQ+O0UI5jkyRCl9DVQ9Z1RskuTCr\\\\\\\/1UaT2jPCMhGNPvCh+5GJ4xPbFxQkvJIjlSmbnvmi\\\\\\\/CEsoCN1iwqzupisRVNlQa+q7ubjQe2mvkZn2IsHAx58KvDpOJoKUdZXVKH3Xm3KfrbZLseF7XUiIp2uKawNtLnNjUFuMK+aQRdWInNKHBjgR2CLScGCs8y1dP65lXmJWPqkuHRfcaHbjslC6jJ\\\\\\\/Ry9za+WRHC1eHaNCU5xV7c+M20RiCA1MDVJ5QletDjw\\\\\\\/clYCupBPgPaLgo96tRHE3zeuQUNAWZ0PEa4Qy1MaCTLX7LBzuP2plG\\\\\\\/evH1OPrBSInaPxa4HNrBPpHQhACjMf\\\\\\\/MSJENX6DPP7HLVyp49g8xISaTwRitNlzhUYufw2JWCFDDtLiYXWH3DHI0UvjSCa7RIcoDs6rzZkFvrx\\\\\\\/T+fKOrBP7tq3UIcUKNid0pfeo\\\\\\\/gp1S1qtsIZ7aZ7rp191Z1YqXtirl6U42gfa6hWPFVH6fR0QC1VwRKpA7SbR3CibdyWMU5larLtoZqhTXgDlViW9XZ\\\\\\\/BGPN4h9bHHG9UD\\\\\\\/yB0nzeFrxix6F4oKXOLfUAYllOmXiyR5Z6RDPWZFVfnlLDrv4WtPX3Qi6VaIgPV2FqO1p9Ki0Lblowf8ngxG2Pr9TvS1ZJ8lvj+P2+RI+U14tyPxufT1mWBAqkDlGMM23uDglnwqUOTaytujzWMPb3Q2M7OJSuq9LNzhDjB+q1iPeOPTu9yGWAecgR0lTIprI0BSSCPneIFNWYfQ98xzcs6jRO0mnrgEzhz8PAT9JpPsd2XinY5m7QKe8hApX2udmtNkXEDolyzXEB8nEPieZvAbruy6MUI6WYaUQtdGLSJmolov4MbDLcbIHpfBwMda9Lvx2haYv6s\\\\\\\/1Ce0huDHOU0b8q+v8qzel8f+wAqCk1H6l7MyZac\\\\\\\/aml\\\\\\\/ypE\\\\\\\/KKbd\\\\\\\/uLylrXTNxcJ0IYJFlvWIZDSzXq25f7qKRLPGPgF1U\\\\\\\/WWTafbHKkGGM6v\\\\\\\/y\\\\\\\/\\\\\\\/3ovhrc892XdHvJuDF+6i9VR7icZCfGil9Y32iScDCAYx7+MKL43gricGMQWrXMGNN+NBI6HJNU2RXJqai3+MWBY4i1eXBL0PBtybFVamvytJCzcO5nBeSsxe9YLwRE9kHfY0jkGTkMtqqHuQCXjySEicuS9Xa6fkxlLSQ\\\\\\\/WprRMeqTk1eVfn+WT6dPYN8xxDAuAeG6SyoFL4jTRaOALe8P5TXJKxdc9W1dJHyLUB0GsYABEj0Ht1Dr2lBJ8Z6kaikkxDp32Mcz1aIgwKrgOsvNYPlUVUrrloSm6GwwhGo+vjmtuKxETh2Iu+JV1NSNCQpqU4ERQKB6JeeQRyucbyKnucp56YWCfPw1PxtuMXke3fElNIfBNmz9CW5E+iexwl0AG6Un8ESfUYC6cHwpNfBR6C5BulYssCXHy+GuRrfkK9gMr6dZmpNi2cyawfpH9VMYqt155O0Qhy3JjFYv34nfdJIJKxq87B80HHVfCno0S8Sf6jCc\\\\\\\/wmpykcXRrB\\\\\\\/PuDJlrqz9ugin2k4A2yUsiRp7TOpSWRAi4DF6O8fbHbvDtnitACp+mla+vPyRD3Xsu4ZlojrVr5X2X8YhrBi1E5L+SDcOBXfR3qEv5nFd7JQSdhnWOV2uuypdSJdVgOM53ydus28fwaeFXH4lsUqtEyTJspYAfvUDGFDRpLIXQDyVvnknSzmWljzKbnfzQ+HhgqMaBuuEnrea6rcHiD3PNvP9MOOuf+gESze0CqfbBp8lzqGtk0gxp10bwHHS+0e\\\\\\\/V\\\\\\\/B0xxXQ1cWw1eOuXGj4S4FpwBXUPR0Amqy2Lntd1jQJI4i6QA8FLmm3gr5giyaJRBVQt6d2gjonEMFhx0o7APdOjq+lM1eF3p9TnMLD2q8yVIqem\\\\\\\/OIOFAsZdLihBEV8z2BPdEa9qoSY2d4qZBeJsN5mVQPG6ND1dseuWPZxwrTwQ78qF9GrS\\\\\\\/634\\\\\\\/nMWcYveHUCved9CQY3HbVq6Cesbw4WEfZQ31GFQNnQ1liC9D0yy91ES+ks+zPgc5kfWGO514xbd3uFNHIRegoBygSzJ98vShXBCTcmkeH+WpqlCadLjfQfVFjdaGXP29PUpI93PmtkRe9O30OqUON0O5N8I4xpMRcQh2B5JFMs53J4LIVVnJ09w\\\\\\\/mGVFBZ0ORkN4eGD\\\\\\\/nnAVlN0TkF9vNMfSiI88pGQfHLb+NtYKFlDajqxC6dJnd9BHEN+HM6A3NHYt8aOmX0u9zy+OmKbVftpwFHitqU93G2OInosl3qzdLPX+ee5pGKLs2CaghirdWc+GqVpqAMqakkv6lKwVjAntf5ngsnsPQzsW3ksclESIGPSPC+PeRzyt22MvHTujlF\\\\\\\/s9aQ18ln76pmbrulQjsyJV2Z0p3sKpUrzOSc13vKH0dXRLFls\\\\\\\/tOGiqWIP6t\\\\\\\/YkIwHQ7qaAmq4hNYxQWo4yzyAnhKqge\\\\\\\/r\\\\\\\/ZA7wsbszz9oKskr7rEHgTpQNhoE+Kv21xnXJWR1bi5emgI7YInQ+Tx0rpI7U02QNzBm9CcwQ4xFzZLlJEP9jiQ3m0UKFOW5Kf51z+cr4d7OB3\\\\\\\/+fQ5yvtbMxol4Fh63ByYXmu83pdd8ENbPKKllpKXW3ARsrTcV\\\\\\\/0bwvbnaYMd3q1hphcSqXw46t5Wcc2q5aPfkTaqFU+NFOaotTKur20RZg\\\\\\\/7WqmZ99w18IRZWXfeF2WWENNA+pTG6UHh+WICqCIxB\\\\\\\/pJBgF3o7VBmhr5HbtVNquLoGuop+oo3vKTBHWZmNnojUz9AafhQT2s2DpMO4IcyTgxrsJvIuRY6nvzVSb2tBkfCnEDtG6vw1cLbBYE7xT6TYDNuEKDc3Z\\\\\\\/vcf1efO805OdbUG+\\\\\\\/ZN2p2wP5C9CiHYsmjNsvbga6SerjJx2Q3on1sKumGS\\\\\\\/MVM5YOv3hO8zoHil10ePkTV6jm0Bjsi0J9T+mVZeHNYrVoM\\\\\\\/9M8dI1\\\\\\\/rSQ\\\\\\\/ZQSPV8xIRHCPRCjaNIJYGQ6v+1XoEbgelJ9mH3LpMb2QQsOuiOMJtvAiSmW65hYXMIsyG0aOaHr\\\\\\\/y38IROKdVVktN8dedtT1Vmv+2CGh5jC6cVHCo4RfUqqoTMBgmmekA3IAzqRHcqecZSaANf3p+9ct1GMF+MLJ09WZAOG2DZUwNR2YVqOAP2NE00vNO0ZGWqFmHdZI+daKDf6kDo7dBrgesMZdwh\\\\\\\/gnFkXtSV9tgKfxKy3Rin8nIpRN5lEQD0fzGoj3TKxoUZB0Aeo41+LRdffx\\\\\\\/dU6eC6DoAI4922zpywj6ULox9jecbGfQtH5sy1B4wvGAEpVADzijzaD61hQ2kUuHSqGKJ+fALYsyMHZh1mKE40Xyr+QW91k5mjIoa8qi+vTjy+kos5L\\\\\\\/wwdC53YBGNgeSH8stg+aTLPLT9Kp5nohugnptrYt22sSQLjcgxUfSN9F9BglxUHsHtp12lCIvW5yBSUSGh8bT1gjyPahqwXpmnjOagzHFssiORlmEM2xDqXYM0MrulYiE4T37ySaLo3OEwtccnkYDPe+FbPcqRHBju\\\\\\\/zWXrhe6M5jv4DrIWoWLouEMNZuqhYszL7g9AdR2iVL4Y5wGE9q21Krd9AMsX5xiDF8ex5JKuDfn1RpfB9XbEG45XwnQkpgy7XQO7cfbvZtqClX8FQHLmhfVZsHvcQhZX\\\\\\\/rqoMLdJqTvX4yOfr0F1apUGbAreCfYorkEz2DYC+XJZsI7nD5bVyvVkpsQzyX5OpwDhn+PNitooQBlJr8eWCHlv1PAZx4vY9aMy7AUzmaV8g6eJEkQYM9gHk2Vn9rb6cRZKg0BrDCHNstpQ7hRb9EYl3counllGDgy+D58of8sSE\\\\\\\/V\\\\\\\/Eq6xZ\\\\\\\/yjm40Jv3W4c7LI9gUEAeEZDchJM5SRKWETM9+63pvZS7qkcxZXd9aEObgjYmM9gaOBMtr6F9th9LDHshoYx1xYEu9zu86DbLYtCr2am5Y8EIyNCUOou60aOsz\\\\\\\/9rEMRp+4xj\\\\\\\/4MNvSRGRCp6Zdqi7ZIpBlZOt2gD5Kvr+NOTax1BjkBnZntCgXfNHVHUk7WV4GGRr1i2EQ2Cwfu6iPA\\\\\\\/TKh7u2MuapnY4Mzn5Js5ZGSV10Rhezne0LwDRbc\\\\\\\/LJHT68ZsrzZiPbkGsoIJ+88EJjxF0OWa7eom3LsaZXUvkY1LnkHGSG4pl26actCVVNtoXwvvHv1Rf9llv\\\\\\\/ecUBdO3Xgxd30qrNdcvLK\\\\\\\/NMqwJomjkzjtQY0SFlrWU7gsqpMlay4F6o4B6R37dCdN1Mrw33yHMYvmwUU31BxiQ+IHjHRTaosT3BoCm87RJVSd8QokdxpogrXBQPLdaTATaorzuNp2Rz7GRnNGUHNE5\\\\\\\/9zhe7sV36+pvpChtIAdfrGb\\\\\\\/txjHAB5llgY6D2Z+7KmeQJRDvSPgeCyQJ2u1HJISW9SggsGuFGRItJZ7qyutlJqJUiCXgbAlYKXyQIl6ZqFMBVDhyDFgOGBf07UZvQ9UYMLgpRgH4s2y\\\\\\\/G4sb2ajqJ\\\\\\\/aPaOgIHxVZ6znh9oEMLz7\\\\\\\/CVWM\\\\\\\/AwzuFXqNa98Fxre7F0Rli4Tzru3Bl6JLWbjhduYRSTwaAuuQnwRs4ARR8HI2w0XvVh3WdmflzYzse0VgnM+Rkzk+gl3Sy250FxIbgvT2KKxtRFX3aQ3NA6D0He4TYi22wAXCGH5GM0V54YdhXFKFnbEEDTqE48pzDXytK3hKXLm43+1Lbi\\\\\\\/VN8VqWQL1tCFSHHSoAFs0V8fWN33qurjW7B8hr4mdCi840NUh\\\\\\\/2f5FyciZly\\\\\\\/OGAw9Tw\\\\\\\/q1rXrxdyeYGrH9Onrl+q8kdmVJHx3mEDq2eBrgFTVvNzNSd5ic6B1Y5+QP8bTstRfGRUhdU6qcQC4LKA7up4A4oh748Kmo0Vgv7Nrx0ctXpwNRh5uWnRkiuSf1pRLVADr+mFNz9aPGEdAYupCUOHKzyzERrpjwKJaep51chImZZ3D1rcUaO09At1nKKu7Itbttzhx1F5WJA5n2L4YCqAuC8aXLWfzbJDoXNkx5ITnJ7BC4uABpO+ifFY+P+CyhkUKvpYDD5PZEPtadIBcmModyf0eaMMIiLkA0ZuwTcCi+5\\\\\\\/6y6msWDvm+S47zu8HJCjYd11r7jF26YZE8sa8QRV5eJRLdJC17he1l7BllajFALWk4pLq9TKkclNimOzIQeVwtZYOt\\\\\\\/mN2924wIXb0ouF5uSrLk3PUNiCjrgd37iM6jMXbqz3+UynBTBGRXCHpNi3cEGQPOpshhEttHx3\\\\\\\/onrMRxQi\\\\\\\/8ivxILXFb8TffLVO8sL4T3uw9fYxUqBIhnX8hymVIuy1xUzS\\\\\\\/BJRlASUIIraPy5W+KXe5fkwBkmgC9tQGZ+smxK82pbhW1cW4r9D4R2eY9G\\\\\\\/yTeUqIHo5\\\\\\\/CB2Q0osx0pDX92DimYR36QPNE6bF+S2ECeGZYpsRcVY\\\\\\\/ZTE5LBKSKGzVNeO9AQdnfN8vY0cMsh6PVcuFGHwz6D\\\\\\\/9VSk0vsnxE5mOfSz6cnZOu7Z3Y108a2M+L2br52a+tzS+Jc2hhb\\\\\\\/W8eTdXfb201CRnQ9o\\\\\\\/O1WCggzexvQFE3N1K1ssJG7MeWjEJyQ5p0RVtVr5iV0T0ns1covZEkhSrOtlJYQMHjHvr9dQco9tm1m\\\\\\\/NM71WnjjezLzFQbjTw8UINMs8ibKRE82dB0xotKE2y9AHAeZ\\\\\\\/WMtGXLPGLxVAUTCn9Mukc0+1Zx8LabJs7QWuZyeRuRrLl8wCjn7Y98vFBh27RJoyrScKN7\\\\\\\/K4o88flfOmrnSygKE0ikdp7hAqYtyMUub6yhq+l38DXtWwG4G7ESoyv6jyNIoGxTm47VHEU\\\\\\\/YBSLxsJBHIio4higbMjJU1XsI\\\\\\\/AwAZXnsAoWGjfYhPfcdsWkXDOfd5MowGRe4myk6+0C7VH+wKY6UUVz02YS1ic6l8heQEk5YuFUv0qK1GGUEbwcuYJru+LcUas8VWOnG29Y0EIdU5p0htiFSCwFz4x7JPDG3WIFo65RDCP+aoVMgJB\\\\\\\/99gtZDVI+e2mEY0voBOo4veNonXsQtFT7h580A6icV8wB9xyzS6X7eNC4+Atw3ZmVFTZlFUqXd3GsdyCA5sKu5\\\\\\\/rUTdG\\\\\\\/XvDMrM4eWUyfM3ZpDnsDjbIRuFi7TsVrsEPnKfJLI0DKy\\\\\\\/m4JvARK5ZhE1U3AijRI0mrzMIeF4GxIA1grRjN7\\\\\\\/zaCnzRLmw\\\\\\\/35M70okfUNifVlLiGCKd0Z1w++JVeLgczrxOudnqKIufzhxJZYUoW3KCEDzCiiYvkgybwX+jQtpGTXMpByiSzaQ807lgnmOIemRtjdPiziYu1o1DDt3uCdw35faE0\\\\\\\/HF8UDCoVPfC0HijMFkOgFAZkUVCNHWm48YOGGfu1KNCaOcTueT2pOYdguNApeYrUijmUX0fyofTIv956pEgTaYEJ4CCBOWsY\\\\\\\/UfKIvvvv+z8HcsSrHI0K0JtOBEAew9KyhxVoNHz48a9lP6mAFRKXw+hHX\\\\\\\/KRR3N7NzjcwsiX6N2GuEKLrW50GIlIEKk81EvEgxc4ODAEUsY5q43dWDq4CaZBAk3ZlI9hW8yIbiTbb9SIMEymJ+\\\\\\\/iGpqMZDkEYErTCvWbPV5kwHcWa4LGQhvfeP3azNRiDoDnkliDkbNNCKZmOCJfqQc4pc91HbZyz+wJCmQgudLzMMRj2XBwR1Y4giNAhOmSkbrXdTbEP1bJ\\\\\\\/rZgTu2bCunnRaMHlLeq\\\\\\\/q7Ap+37N6BBemtwOQThasRBzIGZSviAuOluyqz48AwyMRi32E1zvcBPQdARCI8Ow9TsXL2MQuM\\\\\\\/3nhUxsrISGmuh6iatpfomXXJY2QiPeIWne31xSLiNd4AWNYVwjEAKBZJuvpBoGOIFawUyl3dZbnpsh45AREMEOqlNXQ\\\\\\\/Ap1oK4uWO4QIoOadZK48D6q76zQBq9ELTZ7Zo1RtMl6\\\\\\\/ADPj2aR9r\\\\\\\/\\\\\\\/Z1PZzVoiWc6\\\\\\\/V2amEdGg37OZQ6Jablszdp6dlAKnflK591n4tTvwjxGM94STHKHFu7agvI97ts5i4TNL28nG1GYd3kLIE4W58kfUyg+gorCq4JIpPBUXjPdsGUfF9hRAG\\\\\\\/vdW7khkrR7GSX62+v3ZQDexTrfYgmbUwzAYvmWm9LTPzRlJUULu5Eb5cWeXUv3BWI6nuI4KApxodJq7Se\\\\\\\/Yk8wYkIU0r0SeGamzz4j6BJHlr1DKHR1jpFiV2K2BRGgab\\\\\\\/0FagohDU9sN5Lp7J2bLU2GF8osrA3OZRvDpLLcB8RNkIB9bwKc5zaoqAM6hWHT4eHSDsyVNXtAPKta4h57TrVk\\\\\\\/0ViauIkC71nw9zeu5\\\\\\\/bXnEiQGwxhA7foGzFlJ2yJl7OSOHzSU6VqLsmPSGksidct5Myw65Ku0wjHdVfJRpZZ30C8mlPaWwBtqwKFSZcJcpC0zPTyZ6LHv35q4OcmaxCQog1qdc5n0R6j7ulrako\\\\\\\/Ur6uPSYruCxop1X0vL4cbuZ1uhCEj4cNmG1fLHtuyfVHKzn1pMy7g3fuQGFYuHbZ1MCOhbhvQ+QoitXK6GoBVo70ylFVMSZeIfZmiKEpXrym4uQtXX0Xr1XGJoH5BTTuqYLfFD3fMaZn9ApYLBFy1SA4rOLCMvCd6ZpYZEhQO56ecd07UmE38YVvPtLSHo8QTEvdRHrsf34v8sYxsofcIoSIrGVFoPW7txamhzm+PayebwVVi3OSehuPdUeH3mHHtZNKeX3Z0U8qolWo+VOngMmkdSgH5EbmgLNc4yiWBQANT2w9a6fFzk6U04y9M6O6Jo2G\\\\\\\/lmMYokHdqE\\\\\\\/kbEupONqbvFBvNZKi9BYlV3fOwbaAMHCpt6ol56etWpajQu72Eo5x1eQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qoCMDvAVrQ3Ro2K5Yk7O2g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f8abf23c6425ebe1b80db62e3705623f5bd317fc5567dec90f60268adcffe464\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tMkuk+UhS1bndyze6DY8waeHX7XpZCIGtQf52UuZ4FnNW1\\\\\\\/z8ddAdOmkbIOIwMz0qGgfgjjPhUlyGQw5\\\\\\\/AcLfOLR9rwszsTTjU20kEzIAghNy+0tDOSm+MaaXwX4mbWeI75CPB1lsH9gH73brzvjoqDMPeup6Z7y1XDX1L2LV1yFqA8BUagswYvru61Pe66c1kMP1iihAn6owSSesN0C+Wh6vSWoPdiPr47zYeByQTOUne9S0ZPNlGpaUMRbt37JbKI7GcnV1RsWrs2bzekTvhmO0hlN4RYL9PCx9xNzkA9Ud3f3H17yxxAqHyeaX0C\\\\\\\/a1Uql25bf6VhsOboU2JqY4KYS7SbH\\\\\\\/AmnWyy2BRljyuWmmMqA3GRezeE+T246cRrvtSzl5mGPJjYqNZ0ugm9+mjB1qZoA231qxEgK+BTHiPmBNMOjp6q0dZ+RL0Iz7ThxWxxzMWAL7UTcMz1ffVyhUfGTmvH+bs09fFuBa7WtZm0xpPwBq7cjRVYjqWQGtJXL2Sl7oVXipe4EXCnX4CDtpVvi3\\\\\\\/ZDAAx4+V3nH9i3FGOD5iSbQKE7sLh7+PftMwT2E4cGnWKTRC\\\\\\\/3P4Y\\\\\\\/s3lSJF9Ufbn2M5O\\\\\\\/eWB9npHBUgSIYUkPkxDdVemPZAehiOCkh9QYNsuQd+25Jb2jkFD7c6ibkXTDSxovVR3qDbyUqXXn6REWRkg4tbyDu2a\\\\\\\/9RVOQxPjS7Nz45wXPJ0poaOjD0jWJmNqjeC82zcXOQa0s6ZjXpMQf7uO6zKLPd1YKMprLVb8CIwxiB4TLXG6iixs4W\\\\\\\/dfKiFRNDSt0kbhBcXAH7W+5kCIJgWmyBOXJ1HpOgyLX09h8werzDzk7rxS9qsTLnQFcoGh2+zcvhqme6iqEEI\\\\\\\/feAinFk50GneZgjE1zNbjEPKjTwHP13tzeV7Gvdl0BHghYC+3fmnKSjih2IcRu86KVsFcpKeP5CRY5ADozxRPQfv8KvBAVJVMpKfOeVmeNN0Kl4u76APGXcF3dsUuHFPoKPNyn03bXtwpAWx7o7Ihy4qKhpiWGEN4NhgQ8KRAJKb+KewBQpmGL447\\\\\\\/hUURGU77bqKMirrjJP5KlF9CH4ua57WzdnMoMYoA25OcIEsp35cUIUuljGtAPR84f1JXr42RiUMCf1sGbtdGLe+fpDg7WWMntiX\\\\\\\/tOTGz6It2q94fhONvOw4V5\\\\\\\/GftrDR9EnpeWOV+of5YFkU11+sONBtncAZluUcbL184rDYrdo9iuwnJ3aqBKxbbVKEMaCuE26df3izUSm+bKmO2R\\\\\\\/IrB6TBAUHKUve1Wommva5pVB4bmksDAlWgcuvT5HgV40xWG8FXxrwUKNfLQYTNYd2iD1cA8aPYR76FlRDbgz4dhpuqPy6CdRfpH4xu0BJE\\\\\\\/lb7ZmNve4bJsF97GceIlbVznamfXC2bXJqi8qR57JeMHT+wvPgXlzYCDKvPIZQRY01IKeukeUByHOUS7KGJZRhQaWPIoMDcjNuxDXaygBwDX2n4Mr6d2kqD8HiKtmNxQIkvBj53+6sPc2mNeQaZnlPwdMXyt\\\\\\\/\\\\\\\/T5y\\\\\\\/1QPENtvFvcgqQUa29zeXZZZ7EeHDjWfcrJAl9Wgnkpo1lN4bkzh9+0IY9VgdUQ83IYTqnC2s4k4zHO34c7At+qxJv5VhnXMAOIXHqiU8RUAS6bj0eU1FfFVp\\\\\\\/w3n4NeC9ipdp2Pd7lZOpUJc6dAy2\\\\\\\/eD7wt\\\\\\\/Ebuu2Nbxicd9gTlr8vVL0Y2kd4Lf96RF9aZ3GqRtr28+C6P0Ja5a4Utpvf8k3yB6d3A3ikv\\\\\\\/52UxbAVB8VcvNxu7puUjsSsdvGO4hTdCddoKhXvYs39HmbzvI5Qv9ZbXIR3c45JldPpEjslKDu6XTcef9zj8CVW26puO3Zqle0cmxLgBYT6Bq4Z9uBdirRkYtYPIGTmnUU0IxwBJMuw5HF189vyEzEUdZyBvj33m\\\\\\\/MbCJ2s5iXCsA7Nn2mthEEWYBErjoZaenO\\\\\\\/pht5B9ldtkEEWMD\\\\\\\/6yW32BtL4X2cUIJrMqDixQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BkNoYQ5MaM1qyaZSJT74TQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223ed1e7628cfd9081aedf5bdf175e96a99175f477e41a5f3b35c86ed5b3e1357c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495275 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/177" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PO9KFlKROE_XCsYPxaUFUPVgMRiEMe2324-jar6RjSQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/177" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9Lv_MGMNp6NVsf8MjAYzpZl7IrDhfGl6X-mMKrq9zBE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/165" title="Maternal Priming of Offspring Immune System in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/189" title="Escherichia coli Metabolite Profiling Leads to the Development of an RNA Interference Strain for Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495275" id="node495275" data-pisa="ggg;10/1/177" data-pisa-master="ggg;g3.119.400798" data-apath="/ggg/10/1/177.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improving Imputation Quality in BEAGLE for Crop and Livestock Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7874-8500" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Torsten Pook</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2687-8316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manfred Mayer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7172-3263" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Johannes Geibel</span>, <span class="highwire-citation-author" data-delta="3">Steffen Weigend</span>, <span class="highwire-citation-author" data-delta="4">David Cavero</span>, <span class="highwire-citation-author" data-delta="5">Chris C. Schoen</span> and <span class="highwire-citation-author" data-delta="6">Henner Simianer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">177-188;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400798 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495275" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Torsten Pook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Torsten%2BPook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pook%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATorsten%2BPook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7874-8500" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Torsten Pook</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:torsten.pook@uni-goettingen.de" class="" data-icon-position="" data-hide-link-title="0">torsten.pook@uni-goettingen.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Manfred Mayer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BMayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mayer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AManfred%2BMayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2687-8316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Manfred Mayer</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Johannes Geibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BGeibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geibel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohannes%2BGeibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7172-3263" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Johannes Geibel</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steffen Weigend </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Friedrich-Loeffler-Institut, Institute of Farm Animal Genetics, 31353 Neustadt-Mariensee, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steffen%2BWeigend%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weigend%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteffen%2BWeigend%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Cavero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>H&N International, 27472 Cuxhaven, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BCavero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavero%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BCavero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chris C. Schoen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BC.%2BSchoen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoen%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BC.%2BSchoen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henner Simianer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henner%2BSimianer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simianer%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenner%2BSimianer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/177" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495275" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/177.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495275" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/177.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495275" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/177.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495275" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495275/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Imputation is one of the key steps in the preprocessing and quality control protocol of any genetic study. Most imputation algorithms were originally developed for the use in human genetics and thus are optimized for a high level of genetic diversity. Different versions of BEAGLE were evaluated on genetic datasets of doubled haploids of two European maize landraces, a commercial breeding line and a diversity panel in chicken, respectively, with different levels of genetic diversity and structure which can be taken into account in BEAGLE by parameter tuning. Especially for phasing BEAGLE 5.0 outperformed the newest version (5.1) which in turn also lead to improved imputation. Earlier versions were far more dependent on the adaption of parameters in all our tests. For all versions, the parameter ne (effective population size) had a major effect on the error rate for imputation of ungenotyped markers, reducing error rates by up to 98.5%. Further improvement was obtained by tuning of the parameters affecting the structure of the haplotype cluster that is used to initialize the underlying Hidden Markov Model of BEAGLE. The number of markers with extremely high error rates for the maize datasets were more than halved by the use of a flint reference genome (F7, PE0075 etc.) instead of the commonly used B73. On average, error rates for imputation of ungenotyped markers were reduced by 8.5% by excluding genetically distant individuals from the reference panel for the chicken diversity panel. To optimize imputation accuracy one has to find a balance between representing as much of the genetic diversity as possible while avoiding the introduction of noise by including genetically distant individuals.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/beagle" class="hw-term hw-article-keyword hw-article-keyword-beagle" rel="nofollow">BEAGLE</a></li><li class="kwd"><a href="/keyword/reference-panel" class="hw-term hw-article-keyword hw-article-keyword-reference-panel" rel="nofollow">reference panel</a></li><li class="kwd"><a href="/keyword/reference-genome" class="hw-term hw-article-keyword hw-article-keyword-reference-genome" rel="nofollow">reference genome</a></li></ul><p id="p-3">Imputation is one of the key steps in preprocessing genetic data generated by SNP-chips or DNA sequencing, as follow-up applications like genomic prediction (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meuwissen <em>et al.</em> 2001</a>) often do not allow for missing values. In some applications the use of a higher marker density can lead to better results even though individuals were not genotyped for most markers (<em>e.g.</em>, in genome-wide association studies previously not identified regions can be detected (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2017</a>)).</p><p id="p-4">The imputation of genotype data were first introduced by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li and Stephens (2003)</a>. The basic idea of the algorithm is the fitting of a Hidden Markov Model (HMM, (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baum and Petrie 1966</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rabiner 1989</a>)) to the sequence of alleles of a haplotype. Over the years, a wide variety of tools with similar basic frameworks, but improvements to the computational efficiency for larger datasets (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Howie <em>et al.</em> 2009</a>), reference panels (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Browning <em>et al.</em> 2018</a>) or modifications for improved modeling have been developed. Among others, improvements to the modeling include the use of coalescent trees (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Marchini <em>et al.</em> 2007</a>), haplotype clusters (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Scheet and Stephens 2006</a>) and pre-phasing steps (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Scott <em>et al.</em> 2007</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Howie <em>et al.</em> 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Loh <em>et al.</em> 2016</a>).</p><p id="p-5">To account for the specific structure of livestock and crop datasets, special tools for both cases have been developed. As fully homozygous lines are commonly present in crops, the software TASSEL (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradbury <em>et al.</em> 2007</a>) was developed to work well on this data structure (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Swarts <em>et al.</em> 2014</a>). Since pedigrees in animal breeding can be much denser than in human populations (both w.r.t. depth and family size), tools like FImpute (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sargolzaei <em>et al.</em> 2014</a>) and AlphaImpute (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hickey <em>et al.</em> 2011</a>) have been developed to fully utilize this information.</p><p id="p-6">In the imputation process all those methods use the fact that physically close markers are likely inherited together, resulting in non-random associations of alleles. These methods thereby rely on the knowledge of the physical position or at least the order of markers for modeling linkage and thus the resulting linkage disequilibrium (LD). In contrast, the software LinkImpute (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Money <em>et al.</em> 2015</a>) accounts for LD between pairs of markers and not their physical positions. This can be particularly relevant for species in which no reference sequence is available or whose genomes are known for a high amount of translocations and inversions.</p><p id="p-7">In contrast to other methods using a HMM, the Markov chain in BEAGLE is not initialized by the genotypes or haplotypes themselves, but instead the genetic dataset is used to initialize a haplotype cluster (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Browning and Browning 2007</a>), which subsequently initializes the HMM. In essence, imputation is then performed by identifying the most likely path through the haplotype cluster based on the non-missing genotypes. As BEAGLE was originally developed for application in human genetics, default settings are chosen to work well for imputation in outbred human populations. Nevertheless, the user still has considerable flexibility to tune the algorithm to the specific genetic structure of the respective dataset. As imputation is usually just a step in the preprocessing and quality control protocol, authors tend to use the default settings of a recent version of some imputation software.</p><p id="p-8">To increase the operational marker density via imputation an additional dataset (reference panel) that is genotyped under a higher density can be used. With increasing computational power and more efficient methods available the common advice here is to use as many individuals as possible to get a good representation of the population (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhang <em>et al.</em> 2013</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Browning <em>et al.</em> 2018</a>).</p><p id="p-9">In this paper, we compare different BEAGLE versions (4.0 / 4.1 / 5.0 / 5.1) and perform bench-marking tests in regard to imputation quality on virtually all parameters in BEAGLE for a variety of livestock and crop datasets, as it is one of the most frequently used tools in both animal and plant breeding and a new version of the tool has been recently published (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Browning <em>et al.</em> 2018</a>). We further evaluate which individuals to include in a reference panel when aiming at increasing the marker density of a dataset.</p><p id="p-10">Since imputation algorithms like BEAGLE rely on the assumed physical order of markers, the used reference genome influences the imputation quality. Recently, a variety of new maize reference genomes have been made public (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Unterseer <em>et al.</em> 2017</a>). We here compare the imputation performance of the commonly used B73v4 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schnable <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jiao <em>et al.</em> 2017</a>) and new reference genomes from flint lines in maize that should be genetically closer to our material. To this day, all reference genomes derived in chicken were generated based on an inbred Red Jungle Fowl (<em>Gallus gallus gallus</em>; (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">International Chicken Genome Sequencing Consortium 2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bellott <em>et al.</em> 2010</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotype data used</h3><p id="p-11">In the following, we will consider genotypic data of 910 doubled haploid (DH) lines of two European maize (<em>Zea mays</em>) landraces (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-1.gif"/></noscript></span></span> Kemater Landmais Gelb (KE) and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-2.gif"/></noscript></span></span> Petkuser Ferdinand Rot (PE), (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hölker <em>et al.</em> 2019</a>)) genotyped using the 600k Affymetrix Axiom Maize Array (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Unterseer <em>et al.</em> 2014</a>). Markers were filtered for being assigned to the highest quality class (Poly High Resolution (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pirani <em>et al.</em> 2013</a>)), having a callrate of at least 90%, and for having at most 5% heterozygous calls, as no heterozygous calls are expected for DH lines. The remaining heterozygous calls were set to NA and subsequently imputed using BEAGLE 4.0 with <em>nsamples</em> = 50, resulting in a dataset of 501,124 markers with known haplotype phases.</p><p id="p-12">We further considered two chicken (<em>Gallus gallus</em>) datasets genotyped with the 580k SNP Affymetrix Axiom Genome-Wide Chicken Genotyping Array (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kranis <em>et al.</em> 2013</a>). First, a chicken diversity panel containing 1,810 chicken of 82 breeds including Asian types, European types, wild types, commercial broilers and layers (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Weigend <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Malomane <em>et al.</em> 2019</a>). Second, a dataset containing 888 chicken of a commercial breeding program from Lohmann Tierzucht GmbH. For quality control SNPs / animals with less than 99% / 95% callrate were removed. We will here focus on chromosome 1, 7 and 20 with 56,773 / 65,177, 12,585 / 13,533 and 5,539 / 5,940 SNPs representing cases for large, medium and small size chromosomes. Remaining missing genotypes for both chicken panels were imputed using BEAGLE 4.1 default.</p><p id="p-13">For tests regarding imputation of ungenotyped markers in maize we used the overlapping markers (45,655 SNPs) of the Illumina MaizeSNP50 BeadChip chip (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ganal <em>et al.</em> 2011</a>) as a smaller SNP array. As there is no similar public smaller array with a majority of overlapping markers for the chicken panels, we simply used a subset of every tenth marker. All tests regarding imputation quality were performed on imputed datasets. This might favor the respective method used for the imputation. As the missingness in the maize data (1.20%), diversity panel (0.27%) and commercial chicken breeding line (0.32%) were low in the raw data, this effect should only be minor and is neglected here.</p><p id="p-14">To assess the genetic diversity of the three datasets, we derived the LD decay (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) resulting in the highest rates of association for the European maize landraces, followed by the commercial chicken dataset and the chicken diversity panel. The overall genetic diversity in all used datasets should be far smaller than in an outbred human population, which is the data structure BEAGLE was originally developed for. It should be noted that this comparison does not account for possible differences in ascertainment bias (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrechtsen <em>et al.</em> 2010</a>) between the arrays or the genetic diversity of species and their genomes. Since BEAGLE (and other HMM based imputation methods) are relying on local associations between markers this should still be a good indication for potential imputation performance.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LD decay based on physical length (A) and marker distance (B) for chromosome 1 for all considered datasets. Outliers in (A) are corrected for by using a Nadaraya-Watson-estimator (Nadaraya 1964), using a Gaussian kernel and a bandwidth of 50 kb. (B) is using averaged values for each SNP distance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;LD decay based on physical length (A) and marker distance (B) for chromosome 1 for all considered datasets. Outliers in (A) are corrected for by using a Nadaraya-Watson-estimator (Nadaraya 1964), using a Gaussian kernel and a bandwidth of 50 kb. (B) is using averaged values for each SNP distance.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F1.medium.gif" width="440" height="196"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/177/F1.medium.gif" width="440" height="196"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495459" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">LD decay based on physical length (A) and marker distance (B) for chromosome 1 for all considered datasets. Outliers in (A) are corrected for by using a Nadaraya-Watson-estimator (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nadaraya 1964</a>), using a Gaussian kernel and a bandwidth of 50 kb. (B) is using averaged values for each SNP distance.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Evaluation Pipeline</h3><p>The imputation process itself can be split up into three internally linked steps which can be of different importance based on the data at hand and, in the following, will be analyzed separately:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-17">Inference: All partly or fully missing individual genotypes in the actual dataset are completed, but no additional markers are added.</p></li><li id="list-item-2"><p id="p-18">Imputation of ungenotyped markers (UM imputation): Additional markers are added to the genetic data based on information provided by a second dataset (reference panel) with higher marker density.</p></li><li id="list-item-3"><p id="p-19">Phasing: The two haplotypes of diploid individuals, <em>i.e.</em>, their gametic phases, are estimated from genotype data.</p></li></ol><p>To assess the quality of inference and UM imputation we used the following testing pipeline and repeated the procedure 100 times for each test. We start from a completed dataset in which missing genotypes have been imputed, and consider this as the ”true” genotype dataset:</p><ol class="list-ord " id="list-2"><li id="list-item-4"><p id="p-20">Randomly generate missing values (NAs) in the “true” genotype dataset.</p><ul class="list-simple " id="list-3"><li id="list-item-5"><p id="p-21">In case of inference set randomly chosen alleles of all genotypes to NA (in our case: 1% of all alleles with no partly missing genotypes).</p></li><li id="list-item-6"><p id="p-22">In case of UM imputation additionally set all entries in a particular marker to NA (maize: according to existing low density array (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Ganal <em>et al.</em> 2011</a>); chicken: 90% of all markers).</p></li></ul></li><li id="list-item-7"><p id="p-23">Perform the imputation procedure under a given parameter setting, software and potential use of a reference panel.</p></li><li id="list-item-8"><p id="p-24">Evaluation of performance by comparison to the ”true” dataset. For more on this we refer to the following subsections.</p></li></ol></div><div id="sec-4" class="subsection"><h3>Evaluation of inference and UM imputation quality</h3><p id="p-25">To evaluate the quality of inference and UM imputation we count the total number of entries in the genotype matrix that are different to the “true” dataset (allelic error rate). In this procedure, markers with a low minor allele frequency have a lower impact on the overall quality than in the commonly used practice of calculating the correlation between imputed and ”true” dataset (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hickey <em>et al.</em> 2012</a>). To account for this, we will provide error rates depending on the allele frequency as well. A disadvantage of using a correlation is that it does not account for fixed markers as correlation is not defined for those markers, leading to them being excluded from the analysis. As rare variants tend to be more difficult to impute and those variants tend to be fixed at a higher rate, this leads to lower average correlations for methods imputing a rare allele (instead of just imputing the same variant everywhere). Therefore, a fair comparison should only consider those markers that are not fixed over all settings and different software. Especially for UM imputation this would lead to a much smaller set of markers to be considered.</p></div><div id="sec-5" class="subsection"><h3>Evaluation of phasing quality</h3><p id="p-26">To evaluate phasing quality we use the switch error rate as defined in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lin <em>et al.</em> (2002)</a>, which evaluates the number of switches between neighboring heterozygous sites to recover the true haplotype phase compared to the total number of heterozygous markers. Since the true haplotype phase is usually not known the assessment of phasing quality is usually not as straight forward. As we are working with doubled haploid lines in the maize dataset, the true gametic phase is known and a “true” dataset for testing was generated by randomly combining two doubled haploid lines to a Pseudo <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-3.gif"/></noscript></span></span>. The rest of the pipeline can be performed in the same way as for the inference testing. For this analysis, we considered datasets with no missing genotypes to remove any potential noise caused by inference errors.</p></div><div id="sec-6" class="subsection"><h3>Choice of reference panel in UM imputation</h3><p id="p-27">A common first question when planning to generate genetic data are how many individuals need to be genotyped with high marker density to obtain sufficient imputation quality for individuals genotyped with lower marker density. To evaluate this, we performed imputation on datasets containing 50 individuals as the ”true” dataset in our pipeline and generated reference panels containing 25, 50, 100, 150, 200, 250, 300, and 350 individuals, respectively.</p><p>Furthermore, we investigate how to chose the individuals to include in a reference panel. This is especially relevant when potential candidates for the reference panel vary in their relationship to the dataset itself. For this, we split the chicken diversity panel into ten subpopulations by iteratively minimizing the total sum of squared genetic distances between breeds within the subpopulations. Distances between the breeds were calculated as Nei standard genetic distances (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nei 1972</a>). In a first step, the custom made algorithm randomly assigned the breeds to ten equal sized subpopulations. The contribution of each breed to the sum of squared distances was calculated and the algorithm started iteratively exchanging the most noisy breeds to other subpopulations. If there was a reduction of the total sum of squared distances within the subpopulations, the exchange was accepted and the contributions were calculated again. The process was repeated until no exchange could improve the fit. To overcome results depending on specific starting positions, the process was repeated for 60 random starting points. Nei standard genetic distances for evaluation of UM imputation quality of BEAGLE were calculated based on the subpopulation assignment of individuals and UM imputation was performed using the following reference panels:</p><ol class="list-alphaupper " id="list-4"><li id="list-item-9"><p id="p-29">All other individuals of the same subpopulation</p></li><li id="list-item-10"><p id="p-30">All individuals of one other subpopulation</p></li><li id="list-item-11"><p id="p-31">All individuals of all other subpopulations</p></li><li id="list-item-12"><p id="p-32">All individuals of subpopulations with below-average Nei standard genetic distance to the dataset</p></li><li id="list-item-13"><p id="p-33">All individuals of those subpopulations with reduced error rates when testing A + B compared to A as the reference panel</p></li></ol><p>Additionally combinations of panels A + B, A + C, A + D and A + E were tested. Tests were repeated 20 times for each subpopulation with datasets containing 50 randomly sampled individuals. For each dataset, all different reference panels were tested. The interested reader is referred to Supplementary Table S8 for a detailed list of the used subpopulation assignments and Supplementary Figure S1 for the resulting neighbor-joining-tree.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-34">Genetic data for chromosome 1 for all three panels used are available at <a href="https://github.com/tpook92/HaploBlocker">https://github.com/tpook92/HaploBlocker</a>. Table S1 and S2 contains error rates of UM imputation for the commercial breeding line and the diversity panel in chicken. Table S3 provides phasing error rates for the set of Pseudo <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-4.gif"/></noscript></span></span> with no missing data. Table S4 contains inference error rates for the PE DH-lines using different reference genomes. Table S5 and S6 contain lists of “critical” markers for KE and PE. Table S7 gives error rates of UM imputation using different reference panels for the subpopulations. Table S8 contains the subpopulation assignments for all chicken from the diversity panel. Table S9 contains the minimal error rates and used parameter settings for all performed tests.</p><p id="p-35">Figure S1 provides the neighbor-joining-tree for the ten subpopulations of the chicken diversity panel. Figure S2 displays the relation between local LD and error rate for chromosome 9 in maize. Figure S3 displays the change in the number of errors in each marker by using low values of <em>buildwindow</em>. Figure S4 and S5 display the relation between DR2 and the number of errors per marker. Figure S6 - S24 display the relation between input parameters and error rates for inference in the maize data. Figure S25 - S45 display the relation between input parameters and error rates for inference and phasing for the set of Pseudo <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-5.gif"/></noscript></span></span>. Figure S46 - S74 display the relation between input parameters and error rates for UM imputation for the maize data, the commercial chicken line and the chicken diversity panel.</p><p id="p-36">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9977087">https://doi.org/10.25387/g3.9977087</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-37">In the following, obtained error rates of the imputation under a variety of tuning option in BEAGLE are discussed. Here, we consider virtually all available parameters in BEAGLE, the size of the reference panel, and the underlying genetic map. The effect on the error rate of different tuning options are somewhat independent from each other as they commonly affect different parts of the imputation algorithm. Therefore, we will first consider each tuning option individually and later discuss suggested imputation pipelines for the different use cases.</p><p id="p-38">Unless otherwise mentioned, we will report for maize the error rates in the landrace KE averaged over all chromosomes. Results for PE were similar with, on average, slightly increased error rates.</p><div id="sec-9" class="subsection"><h3>Inference quality</h3><p id="p-39">On default, BEAGLE 5.0 (error rate: 0.0142%) and BEAGLE 5.1 (0.0127%) both clearly outperform BEAGLE 4.1 (0.255%) and BEAGLE 4.0 (0.201%) for the maize data. For all four versions the error rates are significantly higher for alleles with low frequency (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In regard to the location of inference errors one can observe a high volatility with a tendency to have increased error rates in telomeric regions (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Additionally, error rates in regions of high LD tend to be lower (Supplementary Figure S2).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele specific error rate depending on the allele frequency under different BEAGLE settings for the maize data. Only those dataset entries with the respective allele in the &#x201D;true&#x201D; dataset are considered when deriving the allele specific error rate. Y-axis is log-scaled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Allele specific error rate depending on the allele frequency under different BEAGLE settings for the maize data. Only those dataset entries with the respective allele in the &#x201D;true&#x201D; dataset are considered when deriving the allele specific error rate. Y-axis is log-scaled.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F2.medium.gif" width="440" height="195"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/177/F2.medium.gif" width="440" height="195"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495441" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Allele specific error rate depending on the allele frequency under different BEAGLE settings for the maize data. Only those dataset entries with the respective allele in the ”true” dataset are considered when deriving the allele specific error rate. Y-axis is log-scaled.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference error rate based on the location of the genome. Outliers are corrected for by using a Nadaraya-Watson-estimator (Nadaraya 1964), using a Gaussian kernel and a bandwidth of 3,000 markers for the maize data. Y-axis is log-scaled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Inference error rate based on the location of the genome. Outliers are corrected for by using a Nadaraya-Watson-estimator (Nadaraya 1964), using a Gaussian kernel and a bandwidth of 3,000 markers for the maize data. Y-axis is log-scaled.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F3.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/177/F3.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495430" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Inference error rate based on the location of the genome. Outliers are corrected for by using a Nadaraya-Watson-estimator (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Nadaraya 1964</a>), using a Gaussian kernel and a bandwidth of 3,000 markers for the maize data. Y-axis is log-scaled.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">For all four versions the biggest improvement was obtained by tuning parameters that are affecting the structure of the haplotype cluster. The optimal parameter values (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) for <em>buildwindow</em> (4.0), <em>singlescale</em> (4.0), <em>modelscale</em> (4.1) lead to less similar haplotypes being clustered jointly. <em>Phase-segment</em> (5.0), <em>phase-states</em> (5.0 / 5.1) affect the minimum length and number of different haplotypes in the haplotype cluster. Overall, all these settings lead to longer and/or less related haplotypes to be considered jointly. The gains by fitting those parameters are much higher in BEAGLE 4.0 and 4.1 but overall error rates are still higher than in BEAGLE 5.0 and 5.1 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) with BEAGLE 5.1 performing best. Improvements in overall inference quality can be observed for all allele frequency classes and regions in the genome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2</a> &amp; <a id="xref-fig-3-2" class="xref-fig" href="#F3">3</a>). It should be noted that in contrast to later tests in UM imputation the use of low (and probably more realistic) values for <em>ne</em> (effective population size) can lead to substantially increased error rates (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The interested reader is referred to Supplementary Figures S6 - S24 for the effect on the inference error rate for different parameters. For the maize data the inference error rates were basically unaffected by the number of iterations performed in any of the imputation steps in BEAGLE (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Since the haplotype phase in DH-lines is known and the main purpose of further iterations in BEAGLE is to improve that haplotype phase, this should not be that surprising. After parameter tuning error rates are still lowest in BEAGLE 5.1 with 0.0122% but differences are considerably reduced (BEAGLE 4.0: 0.0307%, BEAGLE 4.1: 0.0436%, BEAGLE 5.0: 0.0132%, Supplementary Table S9). Tuning of both <em>singlescale</em> and <em>buildwindow</em> in BEAGLE 4.0 jointly did not further improve performance with <em>buildwindow</em> overall performing better for inference. Even though error rates for extremely low values for <em>buildwindow</em> are lowest, this change is not recommended as some markers do show massively increased error rates (Supplementary Figure S3).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495328/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495328/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Inference error for the KE DH-lines by changing a single imputing parameter</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of the parameter ne on the inference error rates for the maize data in BEAGLE 5.0 and 5.1. Default settings are indicated by the vertical line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of the parameter &lt;em&gt;ne&lt;/em&gt; on the inference error rates for the maize data in BEAGLE 5.0 and 5.1. Default settings are indicated by the vertical line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F4.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/177/F4.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495427" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Effect of the parameter <em>ne</em> on the inference error rates for the maize data in BEAGLE 5.0 and 5.1. Default settings are indicated by the vertical line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The inference error rates for the chicken diversity panel are much higher for all versions (∼1%) and the relative improvement obtained by adapting parameter settings is lower. As the chicken diversity panel contains more variation and is structurally more similar to outbred human data than the European landraces in maize, this should not be that surprising. With the exception of the parameter <em>err</em> the change from the default was always in the same direction as for the maize data. As <em>err</em> is controlling the allele mismatch probability of known alleles when identifying the most likely path through the haplotype cluster (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Browning <em>et al.</em> 2018</a>) this can be seen as an indicator for a higher overall data quality for the maize data. Lowest obtained error rates are 1.01% for BEAGLE 4, 0.80% for BEAGLE 4.1, 0.81% for BEAGLE 5.0 and 0.82% for BEAGLE 5.1 (Supplementary Table S9).</p><p id="p-45">Inference error rates for the datasets from the commercial chicken breeding program are between 0.20% and 0.23% for basically all tested settings, leading us to conclude that for inference on this dataset there is not much potential to decrease error rates. A potential reason for this is that other error sources like SNP calling errors may be higher than inference error rates.</p><p id="p-46">When working with the Pseudo <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-6.gif"/></noscript></span></span> in maize instead, ideal parameter settings are very similar with the key difference of additional gains by increasing the number of iterations performed (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). As the algorithm starts with randomly phased genotypes and improves the phase in each iteration, this should again not be surprising. However, excessive <em>burnin</em> iterations prior to the actual algorithm only worsened results. The interested reader is referred to Supplementary Figure S25 - S45 for parameter influences on both inference and phasing quality for the Pseudo <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-7.gif"/></noscript></span></span>. Inference accuracies after parameter tuning are again similar with BEAGLE 5.0 performing best (BEAGLE 4.0: 0.0193%, BEAGLE 4.1: 0.0168%, BEAGLE 5.0: 0.0109%, BEAGLE 5.1: 0.0148%). Note that error rates given in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> are just for chromosome 10, as not all tests were performed in sufficient sample size for all chromosomes but effect of parameters results should be very similar for all chromosomes. For all three datasets containing heterozygous individuals BEAGLE 5.0 outperformed BEAGLE 5.1, with differences being highest for the set of Pseudo <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-8.gif"/></noscript></span></span>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495312/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495312/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Inference and phasing error for the 250 Pseudo <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/T2/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/T2/embed/mml-math-9.gif"/></noscript></span></span> lines based on the KE DH-lines for chromosome 10. * BEAGLE crashed for this dataset when using <em>phase-segment</em> &gt; 10, <em>phase-states</em> &lt; 100 or <em>phase-states</em> &gt; 10,000</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Phasing quality</h3><p id="p-47">The number of phasing errors for the set of Pseudo <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-10.gif"/></noscript></span></span> in maize is extremely low with just one phasing error per 2,540 heterozygous markers in BEAGLE 5.1, which should be sufficient for most applications, and the obtainable improvements by parameter tuning were relatively low (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Error rates in BEAGLE 5.0 were about 10% lower (2,716). Biggest improvements in both BEAGLE 5.0 and 5.1 were obtained by adaptation of <em>ne</em>. For <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-11.gif"/></noscript></span></span> the ideal parametrization in BEAGLE 5.0 for phasing is much higher than for inference (Figure S25). Especially for BEAGLE 4.0 and 4.1 parameters influencing the structure of the haplotype library had substantial impact on the error rates. In contrast to inference and UM imputation the ideal parametrization for <em>buildwindow</em> (4.0) and <em>phase-states</em> (5.0 / 5.1) are higher than the default settings (Table S3). This in turn leads to only highly related haplotypes to be considered jointly.</p><p id="p-48">To further isolate the structure of phasing errors the same tests were performed on a set of Pseudo <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1/177/embed/mml-math-12.gif"/></noscript></span></span> without missing alleles. The interested reader is referred to the Supplementary Table S3 for detailed results on this. As phasing is not affected by potential inference errors in this case, error rates are even lower (BEAGLE 5.1 default: one error per 5′756 heterozygous markers, BEAGLE 5.0: 6,141) but the direction of improvement for all parameters stays the same. It should be noted that the maize dataset considered in this study contains highly related individuals and a substantial ascertainment bias toward markers with medium allele frequency (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albrechtsen <em>et al.</em> 2010</a>) which both should improve phasing accuracy. For datasets containing less related individuals and sequence data, phasing accuracies can be substantially worse.</p></div><div id="sec-11" class="subsection"><h3>UM Imputation quality</h3><p id="p-49">The algorithm used for UM imputation in BEAGLE 5.0 and 5.1 is the same, thereby differences are only caused because of slightly different techniques for phasing (B. Browning, personal communication). As no phasing is required for the DH-lines error rates never differed by more than 0.001% and are here reported jointly. When performing UM imputation, error rates were much higher than in the inference case. For all considered datasets tuning of <em>ne</em> was absolutely essential (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), because individuals in the considered livestock and crop datasets are far more related than in an outbred human population with an effective population size of 1,000,000 that is assumed in BEAGLE as default. In the imputation algorithm a low value for <em>ne</em> is leading to a reduced probability to switch to a random node in the haplotype cluster and should therefore be beneficial for highly related individuals (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Browning and Browning 2016</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Browning <em>et al.</em> 2018</a>). BEAGLE 4.0 does not provide a parameter for the effective population size and is just assuming equidistant markers and fixed switch rates.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495309/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495309/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">UM imputation error for the KE DH-lines by changing a single imputing parameter with <em>ne</em> = 1,000 for BEAGLE 5.0 / 5.1 and <em>ne</em> = 300 for BEAGLE 4.1</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of the parameter ne on the UM imputation error rate for the maize data, the commercial chicken line and the chicken diversity panel in BEAGLE 5.0. Default settings are indicated by the vertical line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of the parameter &lt;em&gt;ne&lt;/em&gt; on the UM imputation error rate for the maize data, the commercial chicken line and the chicken diversity panel in BEAGLE 5.0. Default settings are indicated by the vertical line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F5.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/1/177/F5.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495248" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">Effect of the parameter <em>ne</em> on the UM imputation error rate for the maize data, the commercial chicken line and the chicken diversity panel in BEAGLE 5.0. Default settings are indicated by the vertical line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">All other parameter settings were tested with adapted <em>ne</em>, as relative effects were virtually zero otherwise. Appropriate parameter settings for the other parameters were similar to the inference case (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) but the overall deviations from the default for <em>buildwindow</em>, <em>singlescale</em> and <em>modelscale</em> were slightly lower. As the number of informative markers in a window with a set number of markers is lower than in the inference case this also makes sense from a modeling perspective. In BEAGLE 5.0 and 5.1 there are additional parameters to control the structure of the haplotype cluster that are only available for UM imputation (<em>imp-segment</em>, <em>imp-states</em>, <em>cluster</em>). Similar to inference, the optimized parameter settings lead to longer and less related haplotypes to be considered jointly. Furthermore, a method to detect identity-by-state (IBS) segments (<em>imp-step</em>, <em>imp-nsteps</em>) has been added in BEAGLE 5.1 but defaults are already adequately chosen for the maize data. After parameter adaptation error rates in BEAGLE 5.0 and 5.1 were lowest (0.0856% / 0.0857%), followed by BEAGLE 4.1 (0.0887%) and BEAGLE 4.0 (0.139%) (Supplementary Table S9).</p><p id="p-52">For both chicken datasets similar results were obtained with BEAGLE 5.0 slightly outperforming BEAGLE 5.1 in for these sets. The interested reader is referred to Supplementary Table S1 and S2 for detailed results for UM imputation for the chicken panels. Overall, the relative gains by adaptation of <em>ne</em> for both the commercial breeding line (0.774–0.280% in BEAGLE 5.0) and the diversity chicken panel (3.313–2.484% in BEAGLE 5.0) were lower than for the maize data. The optimal parametrization for the effective population for the diversity panel was highest (<em>ne</em> = 3,000). With this, the smaller gains by tuning the effective population size nicely support our expectation of the effective population sizes of the underlying populations. It should still be noted that especially BEAGLE 5.0 and 5.1 were very robust to changes in the effective population size (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> and S46) and overall error rates differ by only 0.013% for an effective population size between <em>ne</em> = 1 and <em>ne</em> = 10,000 for the maize dataset, indicating that the use of any reasonable value should work here. As the default of 1,000,000 is not realistic for most livestock and crop datasets, adaptation is necessary and critical when performing UM imputation. For BEAGLE 4.1 there were usually no statistically significant differences between reasonable <em>ne</em> values and overall variance in error rates between runs was slightly higher.</p><p id="p-53">As one would expect a larger reference panel leads to smaller error rates for UM imputation (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Overall, the effect of a larger reference panel in BEAGLE 5.0 was higher than for BEAGLE 4.1. It should still be noted that even for a reference panel with 20 individuals error rates after parameter tuning were below 1% for the maize data and overall error rates only reduce slightly after reaching a size of 150. With higher amounts of overall genetic diversity, the required size of the reference panel should be increasing (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Zhang <em>et al.</em> 2013</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Error rates for UM imputation depending on the size of the reference panel in the maize data. Y-axis is log-scaled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Error rates for UM imputation depending on the size of the reference panel in the maize data. Y-axis is log-scaled.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F6.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/1/177/F6.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-54" class="first-child">Error rates for UM imputation depending on the size of the reference panel in the maize data. Y-axis is log-scaled.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Comparison of reference genomes</h3><p id="p-55">The most commonly used reference genome in maize genetics is the dent line B73 (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Schnable <em>et al.</em> 2009</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Jiao <em>et al.</em> 2017</a>). The European maize landraces tested here are considered as flint germplasm with potential major differences in the physical map (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Unterseer <em>et al.</em> 2016</a>). After reducing error rates of inference by choosing appropriate parameter settings, markers with high error rates tend to be clustered (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Markers and regions with high inference error rate can be considered as candidates for misalignment in the genetic map. We compared our results obtained with B73v4 (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Jiao <em>et al.</em> 2017</a>) to those obtained with reference genomes of the flint lines F7, EP1, DK105 and PE0075 (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Unterseer <em>et al.</em> 2017</a>). Since the array itself was constructed using B73 as a reference (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Unterseer <em>et al.</em> 2014</a>) more markers can be mapped to the B73 reference than to the other reference genomes. For those markers mapped to both B73 and the respective flint reference genomes average error rates for inference are reduced by 3–5% (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). This improvement is mainly caused by a much reduced number of markers with extremely high error rates. On average, the overall number of markers with error rates above 10% (here referred to as: “critical” markers) is reduced by 57%. For a detailed list of the ”critical” markers for all reference genomes mapped on the 600k array (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Unterseer <em>et al.</em> 2014</a>), we refer to Supplementary Table S5 and S6. No notable difference in inference quality for PE when using PE0075 as the reference genome compared to other flint references (Supplementary Table S4) was found.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Error rate per marker for the first 100,000 SNPs according to physical position (starting with chromosome 1) using BEAGLE 5.0 default with B73v4 (Jiao et al. 2017) as a reference genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Error rate per marker for the first 100,000 SNPs according to physical position (starting with chromosome 1) using BEAGLE 5.0 default with B73v4 (Jiao &lt;em&gt;et al.&lt;/em&gt; 2017) as a reference genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F7.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/1/177/F7.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495327" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-56" class="first-child">Error rate per marker for the first 100,000 SNPs according to physical position (starting with chromosome 1) using BEAGLE 5.0 default with B73v4 (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Jiao <em>et al.</em> 2017</a>) as a reference genome.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495410/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495410/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Inference error rates using different reference genomes compared to B73 for KE DH-lines. Only markers mapped on both the flint reference genome &amp; B73v4 (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Jiao <em>et al.</em> 2017</a>) are considered for “critical” markers (error rate &gt; 10%)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Use of a genetic map</h3><p id="p-57">Up to BEAGLE 4.0 all markers are assumed to be equidistant, whereas in BEAGLE 4.1, 5.0 and 5.1 the genetic distance between markers can be provided. On default, the position in base pairs is converted by a ration of 100,000,000 base pairs per Morgan. This might be realistic for human genetics but for chicken a ratio of 41,203,130 / 33,955,860 / 26,631,160 base pairs per Morgan for chromosomes 1 / 7 / 20 is more realistic (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Groenen <em>et al.</em> 2009</a>). However, the use of those genetic maps without any further parameter adaptation leads to massively increased error rates. Error rates for UM imputation increased to 3.23% for the commercial line and 15.8% for the diversity panel compared to the 0.774% and 3.313% without a provided genetic map in BEAGLE 5.0. A potential reason for this is that other parameters like <em>ne</em> and <em>imp-segment</em> are implicitly affected by the higher distance between markers, leading to smaller segments being considered jointly in the haplotype cluster. After additional fitting of <em>ne</em> error rates reduced to values (0.276% / 2.50%) which were very similar to those obtained without use of a genetic map (0.280% / 2.48%; Supplementary Table S1 and S2).</p></div><div id="sec-14" class="subsection"><h3>Quality control using Dosage R-Squared</h3><p id="p-58">When performing UM imputation BEAGLE is providing the measurement Dosage R-Squared (DR2; (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2009</a>)) as an estimate for the uncertainty for the imputation quality in each respective marker. When using BEAGLE 5.0 with adapted <em>ne</em>, only some markers have low DR2 values and the observed error rates in those markers are highly increased (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8.A</a>). Markers with DR2 values below 0.8 on average had 140 times as many imputing errors for UM imputation. Note that no scaling for the allele frequency was performed here and no apparent correlation between DR2 values and minor allele frequencies could be observed. In case of no adaption of the effective population size, the number of markers with low DR2 values is massively increased. Even though error rates are still a higher for markers with low DR2, the relative differences are much lower (18 times as many errors for markers with DR2 &lt; 0.8). Even more problematic for filtering is that in contrast to the 44 problematic markers after parameter adaptation, a total of 31,635 of the 62,986 markers in the panel have DR2 values below 0.8 (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8.B</a>). Results for the commercial chicken line (Supplementary Figure S4) and the diversity panel (Figure Supplementary Figure S5) are similar even though differences in DR2 are not as distinct for adapted parameter settings.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DR2 values in relation to the obtained number of error per marker after fitting of ne (A) and on default (B) in BEAGLE 5.0 for the maize data. 50 / 350 DH-lines were used for study / reference sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DR2 values in relation to the obtained number of error per marker after fitting of &lt;em&gt;ne&lt;/em&gt; (A) and on default (B) in BEAGLE 5.0 for the maize data. 50 / 350 DH-lines were used for study / reference sample.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F8.medium.gif" width="440" height="183"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/10/1/177/F8.medium.gif" width="440" height="183"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495179" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-59" class="first-child">DR2 values in relation to the obtained number of error per marker after fitting of <em>ne</em> (A) and on default (B) in BEAGLE 5.0 for the maize data. 50 / 350 DH-lines were used for study / reference sample.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Choice of the reference panel</h3><p id="p-60">In case the reference population has a lot of stratification, the design of a good reference panel for UM imputation is more difficult, as genetically distant individuals may introduce more noise than relevant information to the model. When comparing results for all considered reference datasets for UM imputation of a single subpopulation it becomes apparent that UM imputation without other individuals from the same subpopulation leads to extremely high error rates (&gt;15%) and thus should in practice only be performed with extreme caution. In contrast, the decision to include other subpopulations in the reference panel is not as clear. When including single other subpopulations in the reference panel we observe significant effects on the overall error rate of UM imputation. Absolute differences of UM imputation error rates are between -0.307% and +0.604% with overall error rates between 1% and 4%. For a detailed list containing all changes in error rates when including a single other subpopulation in the reference panel, we refer to Supplementary Table S7. It should be noted that subpopulations with lower genetic distance to the dataset tend to reduce the error rate and a less related subpopulation leads to an increased error rate (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). For all ten subpopulations the slope of the error rate in regard to distance to the subgroup is statistically significantly positive with the main difference between the subpopulations being the intercept. The most extreme case for this is subpopulation 6 (turquoise △ in <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>; including all wild types). For this group the inclusion of any other subpopulation in the reference panel decreases the imputation quality and is ignored for all averages and statistics in this subsection. Even though SNP-based genetic distances to other subgroups are relatively low, the time to the last common ancestor of any other subpopulation is most likely relatively high. Overall imputation quality when using a reference panel containing all subpopulations is worse than when using a reference panel with only those subpopulation with below average genetic distance (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Nei 1972</a>) to the dataset (2.25% <em>vs.</em> 2.18% - <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of the inclusion of a single subpopulation in the reference panel based on their genetic distance to the dataset for the chicken diversity panel. Colors according to the subpopulation used as the real dataset in Supplementary Figure S1. For a detailed list of subpopulation assignment we refer to Supplementary Table S8. Subpopulation 6 (including wild types - turquoise &#x25B3;) is ignored in the regression." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of the inclusion of a single subpopulation in the reference panel based on their genetic distance to the dataset for the chicken diversity panel. Colors according to the subpopulation used as the real dataset in Supplementary Figure S1. For a detailed list of subpopulation assignment we refer to Supplementary Table S8. Subpopulation 6 (including wild types - turquoise &#x25B3;) is ignored in the regression.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F9.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/10/1/177/F9.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495294" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-61" class="first-child">Effect of the inclusion of a single subpopulation in the reference panel based on their genetic distance to the dataset for the chicken diversity panel. Colors according to the subpopulation used as the real dataset in Supplementary Figure S1. For a detailed list of subpopulation assignment we refer to Supplementary Table S8. Subpopulation 6 (including wild types - turquoise △) is ignored in the regression.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/177/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of error rates of UM imputation for different reference panels for the different subpopulations in the chicken diversity panel. Y-axis is log-scaled. For a detailed list on which individual is assigned to which subpopulation we refer to Supplementary Table S8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2111287805" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of error rates of UM imputation for different reference panels for the different subpopulations in the chicken diversity panel. Y-axis is log-scaled. For a detailed list on which individual is assigned to which subpopulation we refer to Supplementary Table S8.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/177/F10.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/10/1/177/F10.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/177/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/177/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495206" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-62" class="first-child">Comparison of error rates of UM imputation for different reference panels for the different subpopulations in the chicken diversity panel. Y-axis is log-scaled. For a detailed list on which individual is assigned to which subpopulation we refer to Supplementary Table S8.</p><div class="sb-div caption-clear"></div></div></div><p id="p-63">Even though results are statistically significant (two-sample <em>t</em>-test: p-value: 0.0117), differences are only minor and of limited practical relevance for most applications. In our analysis a reference panel containing only the individuals of the same subpopulation on average lead to an UM imputation error of 2.26% with no statistically significant difference to reference panels containing all subpopulations. When performing in-depth analysis for which regions of the dataset UM imputation quality is improved, we observed that especially those individuals with rare variants and overall higher error rates benefited from including more samples in the reference. On the contrary, already well imputed individuals usually had similar or slightly increased error rates. When using a reference panel containing all those subpopulations that individually lead to reduced error rates, average error rates are reduced to 2.06%. It should be noted that a selection based on error rates in UM imputation is usually not possible in practice. Nevertheless, this result demonstrates that there is potential in the use of more sophisticated approaches than just selecting all subpopulation with below average Nei distance (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Nei 1972</a>) as the reference panel. For a detailed list containing error rates for all four different structures of reference panels, we refer to Supplementary Table S7.</p></div></div><div class="section discussion-conclusions" id="sec-16"><h2 class="">Discussion and Conclusions</h2><div id="sec-17" class="subsection"><h3>Significance of improvement</h3><p id="p-64">When comparing error rates under different parameter settings one has to keep in mind the relevance of that optimization. A difference in error rates of 1% in a dataset containing 1% missing genotypes will only result in an improved overall data quality of 0.01% and thus might be negligible compared to other error sources like calling errors (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Unterseer <em>et al.</em> 2014</a>). If those improvements would mainly occur in the markers of interest (<em>e.g.</em>, markers with low minor allele frequency) or the overall share of missing positions is high (as in UM imputation), this improvement could still be significant for later steps of the analysis.</p><p id="p-65">It should be noted that positions set to NA in this study are chosen at random whereas in a real dataset there might be causal reasons like deletions, leading to some markers with much higher missing rates. When performing imputation on the actual NAs, we observed a higher variance in the imputed allele under different random seeds. As all considered methods always input one of the two allelic variants, this is ignored here but it should be noted that actual error rates are probably a bit higher than reported in this study.</p></div><div id="sec-18" class="subsection"><h3>Genetic map and DR2</h3><p id="p-66">The used reference genome only mildly affected overall error rates in maize. As the number of markers with extremely high error rates is reduced, we still recommend the use of a reference genome of a more related individual. This of course requires its existence and similar overall quality. The overall gains should not be high enough to justify the generation of a new reference genome just for imputation. Instead one could consider either removing critical markers from the set or use imputation methods like LinkImpute (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Money <em>et al.</em> 2015</a>) that do not rely on a genetic map.</p><p id="p-67">We highly recommend the use of DR2 to check validity of results obtained in BEAGLE 5.0 and 5.1. First, observation of a high number of low values of DR2 can be seen as an indicator of overall poor imputation quality. Second, one should consider removing markers with low values for DR2 as error rates of UM imputation are typically massively increased. Here, one has to find a balance between removing potentially informative high quality markers and working with low quality markers that could potentially lead to false positive results in later steps of an analysis. In any case, markers that tend to have large effects (<em>e.g.</em>, in a genome-wide association study) should be checked for their DR2 value.</p></div><div id="sec-19" class="subsection"><h3>Reference panel</h3><p id="p-68">Without any knowledge of the genetic structure or excessive testing of genetic relatedness, we recommend to use all available individuals genotyped under high marker density for the reference panel, as the BEAGLE algorithm seems to be quite good at filtering out irrelevant information. However, in case most of the genetic diversity of the study sample can be represented in a subset of the individuals in a reference panel (<em>e.g.</em>, a reference panel containing all founder individuals), significant improvements to UM imputation performance can be made by excluding genetically distant individuals. Representing a high share of the genetic diversity of a dataset however is far more important as error rates increase massively if no genomic data of highly related individuals is available in the reference.</p></div><div id="sec-20" class="subsection"><h3>Parameter adaptation</h3><p id="p-69">Overall, we can conclude that the quality for inference, UM imputation and phasing in BEAGLE 5.0 and 5.1 was better or at least as good as in BEAGLE 4.0 and 4.1 and less tuning of parameters is necessary to obtain good performance for livestock and crop datasets. However, even in BEAGLE 5.0 and 5.1 the adaptation of the parameter <em>ne</em> is absolutely necessary when working with genetic datasets with less diversity than a human outbred population. Especially when no parameter tuning in BEAGLE 4.0 and 4.1 was done, one should consider re-running previous preprocessing and quality control protocols. However, a switch from BEAGLE 5.0 to 5.1 is not necessary, nor even recommended as error rates for phasing (and thereby inference and UM imputation) were lower in BEAGLE 5.0. It should be noted that all datasets in this study contain less genetic diversity than an outbred human population and for datasets with higher genetic diversity like those of UK Biobank (<a href="http://www.ukbiobank.ac.uk/">http://www.ukbiobank.ac.uk/</a>) BEAGLE 5.1 is supposed to have around 25% lower error rates (B. Browning, personal communications).</p><p id="p-70">Especially for UM imputation and in case of heterozygous individuals an increase of the number of iterations improved results slightly. As long as computing time is no issue we suggest to increase the number of iterations. As the gains by a higher number of iterations is relatively low one can also consider reducing the number of iterations to 4 (or in case of DHs to 2) for large datasets which will dramatically reduce computing time.</p><p id="p-71">Other than in the case of <em>ne</em> for UM imputation, improvements in BEAGLE 5.0 and 5.1 by parameter tuning are relatively small, leading us to conclude that the use of default settings should be enough for most applications. Especially for datasets with relatively low genetic diversity one should consider increasing the parameters <em>phase-segments</em>, <em>imp-segments</em> and <em>window</em> while reducing <em>imp-states</em> and <em>ne</em>. For substantial changes of the imputing parameters and for maximizing the imputing accuracy we strongly suggest to apply a testing pipeline similar to the one suggested in the methods section. As potential gains should not be much higher than 5–10% one has to decide based on the application if this additional effort is worth it. Obtainable improvements in BEAGLE 4.0 and 4.1 are high but we do not recommend to use these versions anymore. Additional benefits of the use of BEAGLE 5.0 and 5.1 are massively reduced computing times and memory requirements. Two potential exceptions to this are if high quality pedigree is available, as only BEAGLE 4.0 is able to incorporate pedigree data in its imputation algorithm and in case only genotype likelihoods are available as input as BEAGLE 5.0 and 5.1 only allow for genotypes as input.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">The authors thank the German Federal Ministry of Education and Research (BMBF) for the funding of our project (MAZE – “Accessing the genomic and functional diversity of maize to improve quantitative traits”; Funding ID: 031B0195). The ”Synbreed - Synergistic Plant and Animal Breeding” project was funded by the German Federal Ministry of Education and Research (FKZ 0315528E). We also thank Brian Browning for providing quick and thoughtful replies to all our questions regarding insights into BEAGLE and providing us with personalized software updates for BEAGLE 5.1.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-73">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9977087">https://doi.org/10.25387/g3.9977087</a>.</p></li><li class="fn" id="fn-2"><p id="p-74"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-12"><span class="received-label">Received </span>March 12, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-31"><span class="accepted-label">Accepted </span>October 31, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Pook <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.1" data-doi="10.1093/molbev/msq148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C.</span> <span class="cit-name-surname">Nielsen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ascertainment biases in snp chips affect measures of population divergence.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2534</span>–<span class="cit-lpage">2547</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msq148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq148%26rft_id%253Dinfo%253Apmid%252F20558595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msq148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20558595&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283118000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.2" data-doi="10.1214/aoms/1177699147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baum</span>,  <span class="cit-name-given-names">L. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Petrie</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Statistical inference for probabilistic functions of finite state markov chains.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1554</span>–<span class="cit-lpage">1563</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aoms/1177699147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D37%26rft.spage%253D1554%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177699147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177699147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.3" data-doi="10.1038/nature09172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellott</span>,  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Skaletsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Pyntikova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Mardis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Graves</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Convergent evolution of chicken z and human x chromosomes by expansion and gene acquisition.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">466</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBellott%26rft.auinit1%253DD.%2BW.%26rft.volume%253D466%26rft.issue%253D7306%26rft.spage%253D612%26rft.epage%253D616%26rft.atitle%253DConvergent%2Bevolution%2Bof%2Bchicken%2BZ%2Band%2Bhuman%2BX%2Bchromosomes%2Bby%2Bexpansion%2Band%2Bgene%2Bacquisition.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09172%26rft_id%253Dinfo%253Apmid%252F20622855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20622855&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280412100051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.4" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>,  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Kroon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ramdoss</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Tassel: Software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.5" data-doi="10.1002/gepi.20216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Efficient multilocus association testing for whole genome association studies using localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">375</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D31%26rft.issue%253D5%26rft.spage%253D365%26rft.epage%253D375%26rft.atitle%253DEfficient%2Bmultilocus%2Bassociation%2Btesting%2Bfor%2Bwhole%2Bgenome%2Bassociation%2Bstudies%2Busing%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20216%26rft_id%253Dinfo%253Apmid%252F17326099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17326099&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247603700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.6" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2009.01.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.7" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype imputation with millions of reference samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.8" data-doi="10.1016/j.ajhg.2018.07.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhou</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A one-penny imputed genome from next-generation reference panels.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2018.07.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D103%26rft.spage%253D338%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2018.07.015%26rft_id%253Dinfo%253Apmid%252F30100085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2018.07.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30100085&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.9" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>,  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Durstewitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Polley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bérard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (zea mays l.) snp genotyping array: development and germplasm genotyping, and genetic mapping to compare with the b73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e28334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0028334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.10" data-doi="10.1101/gr.086538.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wahlberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Foglio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. H.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-J.</span> <span class="cit-name-surname">Megens</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A high-density snp-based linkage map of the chicken genome reveals sequence features correlated with recombination rate.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">519</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.086538.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086538.108%26rft_id%253Dinfo%253Apmid%252F19088305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNTEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.11" data-doi="10.2135/cropsci2011.07.0358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Babu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">654</span>–<span class="cit-lpage">663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.07.0358</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D654%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.07.0358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.07.0358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.12" data-doi="10.1186/1297-9686-43-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Kinghorn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Dunstan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A combined long-range phasing and long haplotype imputation method to impute phase for snp genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHickey%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D43%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DA%2Bcombined%2Blong-range%2Bphasing%2Band%2Blong%2Bhaplotype%2Bimputation%2Bmethod%2Bto%2Bimpute%2Bphase%2Bfor%2BSNP%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-12%26rft_id%253Dinfo%253Apmid%252F21388557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21388557&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.13" data-doi="10.1007/s00122-019-03428-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hölker</span>,  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mayer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Presterl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bolduan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bauer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">European maize landraces made accessible for plant breeding and genome-based studies.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">3333</span>–<span class="cit-lpage">3345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-019-03428-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D132%26rft.spage%253D3333%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-019-03428-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-019-03428-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.14" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fuchsberger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Marchini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Abecasis</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.15" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>,  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donnelly</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Marchini</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000529</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.16" data-doi="10.1038/nature03154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Chicken Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature03154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHillier%26rft.auinit1%253DL.%2BW.%26rft.volume%253D432%26rft.issue%253D7018%26rft.spage%253D695%26rft.epage%253D716%26rft.atitle%253DSequence%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bchicken%2Bgenome%2Bprovide%2Bunique%2Bperspectives%2Bon%2Bvertebrate%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03154%26rft_id%253Dinfo%253Apmid%252F15592404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592404&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225597200038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.17" data-doi="10.1038/nature22971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Peluso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Stitzer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improved maize reference genome with single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature22971</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22971%26rft_id%253Dinfo%253Apmid%252F28605751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22971&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28605751&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.18" data-doi="10.1186/1471-2164-14-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kranis</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Gheyas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Boschiero</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Turner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development of a high density 600k snp genotyping array for chicken.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-59%26rft_id%253Dinfo%253Apmid%252F23356797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23356797&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2213</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLi%26rft.auinit1%253DN.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2213%26rft.epage%253D2233%26rft.atitle%253DModeling%2BLinkage%2BDisequilibrium%2Band%2BIdentifying%2BRecombination%2BHotspots%2BUsing%2BSingle-Nucleotide%2BPolymorphism%2BData%26rft_id%253Dinfo%253Apmid%252F14704198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIyMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzE3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.20" data-doi="10.1086/344347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Cutler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Zwick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chakravarti</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Haplotype inference in random population samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1129</span>–<span class="cit-lpage">1137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/344347</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLin%26rft.auinit1%253DS.%26rft.volume%253D71%26rft.issue%253D5%26rft.spage%253D1129%26rft.epage%253D1137%26rft.atitle%253DHaplotype%2Binference%2Bin%2Brandom%2Bpopulation%2Bsamples.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F344347%26rft_id%253Dinfo%253Apmid%252F12386835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/344347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12386835&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178884300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.21" data-doi="10.1038/ng.3571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>,  <span class="cit-name-given-names">P.-R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Palamara</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Price</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Fast and accurate long-range phasing in a uk biobank cohort.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">811</span>–<span class="cit-lpage">816</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3571</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D811%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.22" data-doi="10.1186/s12864-019-5727-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malomane</span>,  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Simianer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Weigend</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Reimer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. O.</span> <span class="cit-name-surname">Schmitt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The synbreed chicken diversity panel: A global resource to assess chicken diversity at high genomic resolution.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-019-5727-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D20%26rft.spage%253D345%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-019-5727-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-019-5727-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.23" data-doi="10.1038/ng2088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Howie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Myers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McVean</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donnelly</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new multipoint method for genome-wide association studies by imputation of genotypes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">906</span>–<span class="cit-lpage">913</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng2088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D906%26rft.epage%253D913%26rft.atitle%253DA%2Bnew%2Bmultipoint%2Bmethod%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bby%2Bimputation%2Bof%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2088%26rft_id%253Dinfo%253Apmid%252F17572673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572673&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247619800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.24" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzE3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Money</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gardner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Migicovsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Schwaninger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.-Y.</span> <span class="cit-name-surname">Zhong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Linkimpute: fast and accurate genotype imputation for nonmodel organisms. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2383</span>–<span class="cit-lpage">2390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D5%26rft.spage%253D2383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.26" data-doi="10.1137/1109020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadaraya</span>,  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">On estimating regression.</span> <abbr class="cit-jnl-abbrev">Theory Probab. Appl.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1137/1109020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheory%2BProbab.%2BAppl.%26rft.volume%253D9%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1137%252F1109020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1137/1109020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.27" data-doi="10.1086/282771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic distance between populations.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D106%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M475000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pirani</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bellon</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Webster</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Best practices for genotyping analysis of plant and animal genomes with affymetrix axiom arrays:p0997.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.29" data-doi="10.1109/5.18626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabiner</span>,  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A tutorial on hidden markov models and selected applications in speech recognition.</span> <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/5.18626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D77%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.18626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/5.18626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.30" data-doi="10.1186/1471-2164-15-478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Chesnais</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Schenkel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new approach for efficient genotype imputation using information from relatives.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-478</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D478%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.31" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: Applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/502802</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.32" data-doi="10.1126/science.1178534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>,  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ware</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Fulton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Stein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wei</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The b73 maize genome: complexity, diversity, and dynamics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1178534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchnable%26rft.auinit1%253DP.%2BS.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1112%26rft.epage%253D1115%26rft.atitle%253DThe%2BB73%2BMaize%2BGenome%253A%2BComplexity%252C%2BDiversity%252C%2Band%2BDynamics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178534%26rft_id%253Dinfo%253Apmid%252F19965430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.33" data-doi="10.1126/science.1142382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>,  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Mohlke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Bonnycastle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Willer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A genome-wide association study of type 2 diabetes in finns detects multiple susceptibility variants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1142382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142382%26rft_id%253Dinfo%253Apmid%252F17463248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgyOS8xMzQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.34" data-doi="10.3835/plantgenome2014.05.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Romero Navarro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">An</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Romay</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.35" data-doi="10.1186/1471-2164-15-823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unterseer</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Haberer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Seidel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Knaak</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k snp genotyping array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D823%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-823%26rft_id%253Dinfo%253Apmid%252F25266061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25266061&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.36" data-doi="10.1186/s13059-016-1009-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unterseer</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Pophaly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Peis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Westermeier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mayer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A comprehensive study of the genomic differentiation between temperate dent and flint maize.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-1009-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1009-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1009-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.37" data-doi="doi: 10.1101/103747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unterseer</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Seidel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Haberer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Hochholdinger</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">European flint reference sequences complement the maize pan-genome.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/103747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252Fdoi%253A%2B10.1101%252F103747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi: 10.1101/103747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigend</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Janßen-Tapken</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Erbe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Ober</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Weigend</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Biodiversität beim huhn–potenziale für die praxis.</span> <abbr class="cit-jnl-abbrev">Zuchtungskunde</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZuchtungskunde%26rft.volume%253D86%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.39" data-doi="10.1038/s41598-017-00729-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Qiao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Xin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Xiao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Imputation-based whole-genome sequence association study rediscovered the missing qtl for lumbar number in sutai pigs.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-00729-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D615%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-00729-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-00729-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.177.40" data-doi="10.1534/genetics.113.154591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Rosenberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zöllner</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotype imputation reference panel selection using maximal phylogenetic diversity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.154591</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.154591%26rft_id%253Dinfo%253Apmid%252F23934887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvMzE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/177.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/165" title="Maternal Priming of Offspring Immune System in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/189" title="Escherichia coli Metabolite Profiling Leads to the Development of an RNA Interference Strain for Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/177" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gArMEnfebYc7KcNe7Kx6Pk29wDFER3jpn1RpqEuo4so" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/177" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/177" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/177" class="active" data-icon-position="" data-hide-link-title="0">Improving Imputation Quality in BEAGLE for Crop and Livestock Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495275" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_ZhbL6AfuHpec_C5gvfJUsYL9Mxi4QAK_hBInwxhbPk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/177.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/177" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0Rt1rGzxzSyqRXfMEJd1A-gIID113xiHK6oivwCYEP8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/177" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/177.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495275" id="node495275--2" data-pisa="ggg;10/1/177" data-pisa-master="ggg;g3.119.400798" data-apath="/ggg/10/1/177.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improving Imputation Quality in BEAGLE for Crop and Livestock Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7874-8500" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Torsten Pook</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2687-8316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manfred Mayer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7172-3263" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Johannes Geibel</span>, <span class="highwire-citation-author" data-delta="3">Steffen Weigend</span>, <span class="highwire-citation-author" data-delta="4">David Cavero</span>, <span class="highwire-citation-author" data-delta="5">Chris C. Schoen</span> and <span class="highwire-citation-author" data-delta="6">Henner Simianer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">177-188;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400798 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495275--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Torsten Pook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Torsten%2BPook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pook%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATorsten%2BPook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7874-8500" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Torsten Pook</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:torsten.pook@uni-goettingen.de" class="" data-icon-position="" data-hide-link-title="0">torsten.pook@uni-goettingen.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Manfred Mayer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BMayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mayer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AManfred%2BMayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2687-8316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Manfred Mayer</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Johannes Geibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BGeibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geibel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohannes%2BGeibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7172-3263" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Johannes Geibel</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steffen Weigend </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Friedrich-Loeffler-Institut, Institute of Farm Animal Genetics, 31353 Neustadt-Mariensee, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steffen%2BWeigend%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weigend%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteffen%2BWeigend%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Cavero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>H&N International, 27472 Cuxhaven, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BCavero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavero%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BCavero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chris C. Schoen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BC.%2BSchoen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoen%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BC.%2BSchoen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henner Simianer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henner%2BSimianer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simianer%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenner%2BSimianer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/177" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/177&amp;title=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/177&amp;title=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/177&amp;title=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/177&amp;text=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/177&amp;title=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/177&amp;t=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/177&amp;title=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/177&amp;title=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495275/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495275" id="node495275--3" data-pisa="ggg;10/1/177" data-pisa-master="ggg;g3.119.400798" data-apath="/ggg/10/1/177.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improving Imputation Quality in BEAGLE for Crop and Livestock Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7874-8500" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Torsten Pook</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2687-8316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manfred Mayer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7172-3263" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Johannes Geibel</span>, <span class="highwire-citation-author" data-delta="3">Steffen Weigend</span>, <span class="highwire-citation-author" data-delta="4">David Cavero</span>, <span class="highwire-citation-author" data-delta="5">Chris C. Schoen</span> and <span class="highwire-citation-author" data-delta="6">Henner Simianer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">177-188;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400798 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495275--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Torsten Pook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Torsten%2BPook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pook%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATorsten%2BPook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7874-8500" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Torsten Pook</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:torsten.pook@uni-goettingen.de" class="" data-icon-position="" data-hide-link-title="0">torsten.pook@uni-goettingen.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Manfred Mayer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BMayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mayer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AManfred%2BMayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2687-8316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Manfred Mayer</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Johannes Geibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BGeibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geibel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohannes%2BGeibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7172-3263" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Johannes Geibel</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steffen Weigend </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Friedrich-Loeffler-Institut, Institute of Farm Animal Genetics, 31353 Neustadt-Mariensee, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steffen%2BWeigend%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weigend%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteffen%2BWeigend%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Cavero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>H&N International, 27472 Cuxhaven, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BCavero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavero%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BCavero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chris C. Schoen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Technical University of Munich, Plant Breeding, TUM School of Life Sciences Weihenstephan, 85354 Freising, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BC.%2BSchoen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoen%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BC.%2BSchoen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henner Simianer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Animal Breeding and Genetics Group,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrated Breeding Research, University of Goettingen, 37075 Goettingen, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henner%2BSimianer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simianer%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenner%2BSimianer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=177&amp;rft.epage=188&amp;rft.atitle=Improving%20Imputation%20Quality%20in%20BEAGLE%20for%20Crop%20and%20Livestock%20Data&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Pook%2C+Torsten&amp;rft.au=Mayer%2C+Manfred&amp;rft.au=Geibel%2C+Johannes&amp;rft.au=Weigend%2C+Steffen&amp;rft.au=Cavero%2C+David&amp;rft.au=Schoen%2C+Chris+C.&amp;rft.au=Simianer%2C+Henner"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495275/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495275/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495275/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495275/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495275/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495275/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495275/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495275/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495275/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495275/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495275/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495275/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/177" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion and Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/177.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/177.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/177.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
