<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/109" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/109.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/109.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/109.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/109.atom" />
<meta name="HW.pisa" content="ggg;10/1/109" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Influence of Genetic Interactions on Polygenic Prediction" />
<meta name="DC.Identifier" content="10.1534/g3.119.400812" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Dai et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Prediction of phenotypes from genotypes is an important objective to fulfill the promises of genomics, precision medicine and agriculture. Although it’s now possible to account for the majority of genetic variation through model fitting, prediction of phenotypes remains a challenge, especially across populations that have diverged in the past. In this study, we designed simulation experiments to specifically investigate the role of genetic interactions in failure of polygenic prediction. We found that non-additive genetic interactions can significantly reduce the accuracy of polygenic prediction. Our study demonstrated the importance of considering genetic interactions in genetic prediction." />
<meta name="DC.Contributor" content="Zhijun Dai" />
<meta name="DC.Contributor" content="Nanye Long" />
<meta name="DC.Contributor" content="Wen Huang" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Influence of Genetic Interactions on Polygenic Prediction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Prediction of phenotypes from genotypes is an important objective to fulfill the promises of genomics, precision medicine and agriculture. Although it’s now possible to account for the majority of genetic variation through model fitting, prediction of phenotypes remains a challenge, especially across populations that have diverged in the past. In this study, we designed simulation experiments to specifically investigate the role of genetic interactions in failure of polygenic prediction. We found that non-additive genetic interactions can significantly reduce the accuracy of polygenic prediction. Our study demonstrated the importance of considering genetic interactions in genetic prediction.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/109" />
<meta name="citation_id" content="10/1/109" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/109" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/109.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/109.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/109.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400812" />
<meta name="citation_pmid" content="31649046" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="109" />
<meta name="citation_lastpage" content="115" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhijun Dai" />
<meta name="citation_author_institution" content="Department of Animal Science, and" />
<meta name="citation_author" content="Nanye Long" />
<meta name="citation_author_institution" content="Institute for Cyber-Enabled Research, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author" content="Wen Huang" />
<meta name="citation_author_institution" content="Department of Animal Science, and" />
<meta name="citation_author_email" content="huangw53@msu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6788-8364" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=E. A. Boyle;citation_author=Y. I. Li;citation_author=J. K. Pritchard;citation_title=An Expanded View of Complex Traits: From Polygenic to Omnigenic.;citation_pages=1177-1186;citation_volume=169;citation_year=2017;citation_pmid=28622505;citation_doi=10.1016/j.cell.2017.05.038" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. K. Chen;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_title=Fast and flexible simulation of DNA sequence data.;citation_volume=19;citation_year=2008;citation_pmid=19029539;citation_doi=10.1101/gr.083634.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=L. M. Evans;citation_author=R. Tahmasbi;citation_author=S. I. Vrieze;citation_author=G. R. Abecasis;citation_author=S. Das;citation_title=Comparison of methods that use whole genome data to estimate the heritability and genetic architecture of complex traits.;citation_pages=737-745;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0108-x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. García-Ruiz;citation_author=J. B. Cole;citation_author=P. M. VanRaden;citation_author=G. R. Wiggans;citation_author=F. J. Ruiz-López;citation_title=Changes in genetic selection differentials and generation intervals in US Holstein dairy cattle as a result of genomic selection.;citation_volume=113;citation_year=2016;citation_pmid=27354521;citation_doi=10.1073/pnas.1519061113" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. B. Gerstein;citation_author=A. Kundaje;citation_author=M. Hariharan;citation_author=S. G. Landt;citation_author=K.-K. Yan;citation_title=Architecture of the human regulatory network derived from ENCODE data.;citation_pages=91-100;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955619;citation_doi=10.1038/nature11245" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive Genetic Variability and the Bayesian Alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Gudbjartsson;citation_author=G. B. Walters;citation_author=G. Thorleifsson;citation_author=H. Stefansson;citation_author=B. V. Halldorsson;citation_title=Many sequence variants affecting diversity of adult human height.;citation_pages=609-615;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18391951;citation_doi=10.1038/ng.122" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=PLOS Genet.;citation_author=W. Huang;citation_author=T. F. C. Mackay;citation_title=The Genetic Architecture of Quantitative Traits Cannot Be Inferred from Variance Component Analysis.;citation_pages=e1006421;citation_volume=12;citation_year=2016;citation_pmid=27812106;citation_doi=10.1371/journal.pgen.1006421" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. M. Purcell;citation_author=N. R. Wray;citation_author=J. L. Stone;citation_author=P. M. Visscher;citation_title=Common polygenic variation contributes to risk of schizophrenia and bipolar disorder.;citation_pages=748-752;citation_volume=460;citation_year=2009;citation_issue=7256;citation_pmid=19571811;citation_doi=10.1038/nature08185" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=J. C. Reif;citation_title=Modeling Epistasis in Genomic Selection.;citation_volume=201;citation_year=2015;citation_pmid=26219298;citation_doi=10.1534/genetics.115.177907" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. V. Khera;citation_author=M. Chaffin;citation_author=K. G. Aragam;citation_author=M. E. Haas;citation_author=C. Roselli;citation_title=Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations.;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0183-z" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Lango Allen;citation_author=K. Estrada;citation_author=G. Lettre;citation_author=S. I. Berndt;citation_author=M. N. Weedon;citation_title=Hundreds of variants clustered in genomic loci and biological pathways affect human height.;citation_pages=832-838;citation_volume=467;citation_year=2010;citation_issue=7317;citation_pmid=20881960;citation_doi=10.1038/nature09410" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Lello;citation_author=S. G. Avery;citation_author=L. Tellier;citation_author=A. I. Vazquez;citation_author=G. de los Campos;citation_title=Accurate Genomic Prediction of Human Height.;citation_volume=210;citation_year=2018;citation_pmid=30150289;citation_doi=10.1534/genetics.118.301267" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. Lettre;citation_author=A. U. Jackson;citation_author=C. Gieger;citation_author=F. R. Schumacher;citation_author=S. I. Berndt;citation_title=Identification of ten loci associated with height highlights new biological pathways in human growth.;citation_pages=584-591;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18391950;citation_doi=10.1038/ng.125" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=X. Liu;citation_author=Y. I. Li;citation_author=J. K. Pritchard;citation_title=Trans Effects on Gene Expression Can Drive Omnigenic Inheritance.;citation_pages=1022-1034.e6;citation_volume=177;citation_year=2019;citation_doi=10.1016/j.cell.2019.04.014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. C. Mackay;citation_title=Epistasis and quantitative traits: using model organisms to study gene-gene interactions.;citation_pages=22-33;citation_volume=15;citation_year=2014;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_journal_title=Genome Med.;citation_author=T. F. Mackay;citation_author=J. H. Moore;citation_title=Why epistasis is important for tackling complex human disease genetics.;citation_pages=124;citation_volume=6;citation_year=2014;citation_pmid=25031624;citation_doi=10.1186/gm561" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Makowsky;citation_author=N. M. Pajewski;citation_author=Y. C. Klimentidis;citation_author=A. I. Vazquez;citation_author=C. W. Duarte;citation_title=Beyond missing heritability: prediction of complex traits.;citation_pages=e1002051-e1002051;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21552331;citation_doi=10.1371/journal.pgen.1002051" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. R. Martin;citation_author=M. Kanai;citation_author=Y. Kamatani;citation_author=Y. Okada;citation_author=B. M. Neale;citation_title=Clinical use of current polygenic risk scores may exacerbate health disparities.;citation_pages=584-591;citation_volume=51;citation_year=2019;citation_pmid=30926966;citation_doi=10.1038/s41588-019-0379-x" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=J. W. R. Martini;citation_author=N. Gao;citation_author=D. F. Cardoso;citation_author=V. Wimmer;citation_author=M. Erbe;citation_title=Genomic prediction with epistasis models: on the marker-coding-dependent performance of the extended GBLUP and properties of the categorical epistasis model (CE).;citation_pages=3;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12859-016-1439-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=M. Momen;citation_author=A. A. Mehrgardi;citation_author=A. Sheikhi;citation_author=A. Kranis;citation_author=L. Tusell;citation_title=Predictive ability of genome-assisted statistical models under various forms of gene action.;citation_pages=12309;citation_volume=8;citation_year=2018;citation_doi=10.1038/s41598-018-30089-2" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=F. Morgante;citation_author=W. Huang;citation_author=C. Maltecca;citation_author=T. F. C. Mackay;citation_title=Effect of genetic architecture on the prediction accuracy of quantitative traits in samples of unrelated individuals.;citation_pages=500-514;citation_volume=120;citation_year=2018;citation_doi=10.1038/s41437-017-0043-0" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=H. Mostafavi;citation_author=A. Harpak;citation_author=D. Conley;citation_author=J. K. Pritchard;citation_author=M. Przeworski;citation_title=Variable prediction accuracy of polygenic scores within an ancestry group.;citation_year=2019;citation_doi=10.1101/629949" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=U. Ober;citation_author=W. Huang;citation_author=M. Magwire;citation_author=M. Schlather;citation_author=H. Simianer;citation_title=Accounting for genetic architecture improves sequence based genomic prediction for a Drosophila fitness trait.;citation_pages=e0126880;citation_volume=10;citation_year=2015;citation_pmid=25950439;citation_doi=10.1371/journal.pone.0126880" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Twin research :  the official journal of the International Society for Twin Studies;citation_journal_abbrev=Twin Res;citation_author=K. Silventoinen;citation_author=S. Sammalisto;citation_author=M. Perola;citation_author=D. I. Boomsma;citation_author=B. K. Cornes;citation_title=Heritability of adult body height: a comparative study of twin cohorts in eight countries.;citation_pages=399-408;citation_volume=6;citation_year=2003;citation_issue=5;citation_pmid=14624724;citation_doi=10.1375/136905203770326402" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=L. Varona;citation_author=A. Legarra;citation_author=M. A. Toro;citation_author=Z. G. Vitezica;citation_title=Non-additive Effects in Genomic Selection.;citation_pages=78;citation_volume=9;citation_year=2018;citation_doi=10.3389/fgene.2018.00078" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=P. M. Visscher;citation_author=N. R. Wray;citation_author=Q. Zhang;citation_author=P. Sklar;citation_author=M. I. McCarthy;citation_title=10 Years of GWAS Discovery: Biology, Function, and Translation.;citation_pages=5-22;citation_volume=101;citation_year=2017;citation_pmid=28686856;citation_doi=10.1016/j.ajhg.2017.06.005" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=P. Wainschtein;citation_author=D. P. Jain;citation_author=L. Yengo;citation_author=Z. Zheng;citation_title=Recovery of trait heritability from whole genome sequence data.;citation_year=2019;citation_doi=10.1101/588020" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. N. Weedon;citation_author=H. Lango;citation_author=C. M. Lindgren;citation_author=C. Wallace;citation_author=D. M. Evans;citation_title=Genome-wide association analysis identifies 20 loci that influence adult height.;citation_pages=575-583;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18391952;citation_doi=10.1038/ng.121" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=GCTA: a tool for genome-wide complex trait analysis.;citation_pages=76-82;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21167468;citation_doi=10.1016/j.ajhg.2010.11.011" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Influence of Genetic Interactions on Polygenic Prediction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="Prediction of phenotypes from genotypes is an important objective to fulfill the promises of genomics, precision medicine and agriculture. Although it’s now possible to account for the majority of genetic variation through model fitting, prediction of phenotypes remains a challenge, especially across populations that have diverged in the past. In this study, we designed simulation experiments to specifically investigate the role of genetic interactions in failure of polygenic prediction. We found that non-additive genetic interactions can significantly reduce the accuracy of polygenic prediction. Our study demonstrated the importance of considering genetic interactions in genetic prediction." />
<meta name="og-title" property="og:title" content="Influence of Genetic Interactions on Polygenic Prediction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/109" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Prediction of phenotypes from genotypes is an important objective to fulfill the promises of genomics, precision medicine and agriculture. Although it’s now possible to account for the majority of genetic variation through model fitting, prediction of phenotypes remains a challenge, especially across populations that have diverged in the past. In this study, we designed simulation experiments to specifically investigate the role of genetic interactions in failure of polygenic prediction. We found that non-additive genetic interactions can significantly reduce the accuracy of polygenic prediction. Our study demonstrated the importance of considering genetic interactions in genetic prediction." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Influence of Genetic Interactions on Polygenic Prediction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UPJujgpkgkuL-K1ycYHkQqlx7z2K0UYoD1Ukhbf7M4U","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495165","apath":"\/ggg\/10\/1\/109.atom","pisa":"ggg;10\/1\/109","ac":{"\/ggg\/10\/1\/109.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/109.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/109"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/109"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/109":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/kXY06xJM0xW2qnD\\\\\\\/CEO3PNan\\\\\\\/BU5MZWm8hh9v8oSy1d2+Z8R9nDHATdbrfu1H9oYTTEYprc0ZzSre4JLZOYiCBqaUZ4dRGwsn5A8dCQGJN8TUuod1wpVbOCtjP9G86+MK654eshJqNFvslxCgQFV+LUNWMKkPy4mehAXH1yGlTmljpzHtflyDiKd5bBuVp+8rpEiAA7hZLZR1O2UgbMXfNbGOuAq9UiBziC0DBvfkQ0M8p70eeR0ZRiHXbamtkVIFKYHwdA4+crl+8h6136wSvyFWYgoH4agbPU6S4STJw5AUdc7BDWrpWH8gkZDdhOY8sdrQG9pINRgWJLab2FkBHu+IhdOR4Flj3PxZFPCEBrLh7Q\\\\\\\/WOJYulTwYZOlLCgzb5o\\\\\\\/Z86BK316E94d+w8Bva\\\\\\\/WuHeMdWQHhbI+aD8ejAteZZtGJnWVEab18stFL6YySxoUGnKmVnyaSjlHKHNGVOrWcbdWIJ+yc1wAg5ijFAJubYuRn4sKADnc+Q5jkk6BWwko0qU6h03bkIkN3qTau1pWIli4TClxeQ53WltDlMbfT9ZS+yhYvyi6OXIav8NmPLfzNqJuvrdQbp3doILe5AtvhE8K4+XeRd7W5bCCFOLPKIh3MVRKAKLZlQFNZqTj+VNRq7\\\\\\\/Jk6PmtBH9YaLVWx8OYYia+TBPtmw\\\\\\\/lmfJOD9b+G9awgkT782K+s9yXo\\\\\\\/ksA5imubhptMwSryKgfJ2DcS4D4aB5xHaSFaY3topNGYB\\\\\\\/rHvUoWOmCRFqvnQkhU9m6Ey0oxvZhnZkkgFPUoJFBt32sY+3vOzgxrkczJydjc28HmNUQB57No\\\\\\\/MA5jtZjH4PdnHJoaCA3GiCD0gtXJFCwV9Ax6uokYooBN6471vV6l0I1Pia6XnjA45zFLDBHACc8cRZAAI7pJTRnBzKMHHMjlKJ9gPG98LitlNrti77BEOd6g0wG+GM3lNLQFBHPKb2mTg\\\\\\\/pSSpiSygKYHDz0yZ1RM8nByc18U2hC8o6LiXowGnI92AIrtmn7tRAXEHtuMR4Vdbx7L\\\\\\\/kaLbu3ERO9wus+FhsEPw4z1boQRlop\\\\\\\/57cwnzromTTc7uLM\\\\\\\/s0N8xv+Q8JuVjYatWy9QoRy0ZSB3Bu2PzwdV3PBkmA1uSGpWHXFZ+84\\\\\\\/Tj9ymGJ4p+0UqrsHwF0o0czvemauI9PsNoX5fyZ18Cw9yvyPMRNiS27Hler\\\\\\\/LXIkSxTHm61TPEYh\\\\\\\/LDZOGsw6MiD27PXQ5y6O76lhU3lUEiJsTLlhABGxgklZZzs2QqeNZWD0pm4ppC1R\\\u0022,\\\u0022iv\\\u0022:\\\u0022N6MkhNQLI0BKz9Kiln\\\\\\\/rXQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002226b2806f8a7c48a2807ed89b80c6695c7c3f6073d79459cddce40b71ccbaf6e0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022m6D2Gf3UFbMifGcT+VRjwQ9s2HNLzXp92A157dPuQOB6yqXcsgzhgU388n1cm5vdPeupFY5PJdzRLf+l8922AftPHIKkzQzenxJ1HVV69e2Y8zRL9BiWhH+njiRRuzO\\\\\\\/QCXrealsSzmpRVKYhaaoPAr9v8s0WsWXKHmpQqm\\\\\\\/UaZsMr9nK\\\\\\\/daQRzDD202hPnHc31xdd2jspWXoVTkiYXT\\\\\\\/mdOV5FOT8FfW1jlZfObN1rQ3vAauKlMVfkyOw5E2S+3XM8DrXlLYiUwmtPMW0e+V9ord+Yuu4Ly9XH8gVT7V+X6PS9wYzC7pszrjZ12zOylbc+CNXj\\\\\\\/9J3pPi0UuVBuOLcsq9LxZxlhZc4B3Y4gDCKMEaJuUI3HEaxN3QoJdA7VHY9m9hQrPmkLrpFneapN7oobf\\\\\\\/A7nR73ayhmkwvDDRc+PkZXPzyotEgKA\\\\\\\/cA8Q6LAArUYtCiStJ8qds0ezoMxR\\\\\\\/HeS0x+KewrdIGktN7i9kTfAVIqEgL9NNe7C9sirQW4Al2S0JG9q3goypLWmo6HtsjLZCtd8AJo2twz8orJXDVa4\\\\\\\/+MQRzlKw3Y\\\\\\\/95RkcGbz9w4E023h8bo+LSDSPyFZT5t6UIAG2oarK20fLOOHE1KuyLfv1JkMkUt9nMr1NREf1Qzahm+lBWmCCHTjd5eABdih4YiTvOZ\\\\\\\/wgzi5awDcYzFhcGmQG4i4X35fPEEE6nhQmp0dw\\\\\\\/Tfsi9MrtPRd3u3S1EZwI3i7U7Opx0zlWWjQeT6BNbC4pPq+Jt1o\\\\\\\/DK+eZ5s12tF5SX0zhzFe7kz+jMpsDDMDZrdEG060we5Yd3PDlzlCwdlXAMz7PaQSafl2IBVfg\\\\\\\/HBuy6t\\\\\\\/R8xXaMMW4asvKOM62LwvUyBYurxyh48hq1dDisZD9xL5pL0TQ05mOs+dDjv728BFQoNcPx+SeEQQzc51Sbqptagd3kqDiFrprWd\\\\\\\/XmXKmdQiEcE1u571KOAVpjHT6WZANjeQzP\\\\\\\/JuMttuRFLic8X8CoFGrBkgAvC+JnNMUPJW352VUDHvGAYj0sKs7yt18s3GrFQSQedZWFvijHNOw7G7LFfkS+f+NJ5D90XRE0zGxpSZfFdnRb\\\\\\\/+JyYr\\\\\\\/XS6HRzE28RS802owALYW+1OvMXcC8smKBWmQjmKvVp1ecMpxSd9FtgL0UF5UB2ZGx+c5skTE6HS6G5vJqzGUrJa0u8ruczrnILbmb9vV8hEF8om0bhwWk8RRY\\\\\\\/j4JZuO3bZtgtXpXnTmM1tcSW7iXfDdM7+WdWLHrMCQlTzjot3TFavqlK2hlbTlbxCN\\\\\\\/hGvFptUb\\\\\\\/owSw57jBIM9ImCo5ET31rb3RwutWq4f5rJbvj4fa3KgqXlQbojHUAmgUjgK3xzORdN+hFkDwImrvHQMUYOGPKDtXyXBdHEkpVdeLFSPZ43bk3D32jqlhhcdy8b+Hx\\\\\\\/Eo9fRCkEnRhlVTjRhLjepo\\\\\\\/Qyr6iUzc\\\\\\\/+yghbHOMlCBiZhabJNuQMfxuPJJl0gpnYJ3vYJsKAimhU9NBMxaQCx4wJyjclXlyxeKe4EaJXvIOsacFIVVuNeCOUV7VYhXCrW+xFsNSQoXyjq3ugAxXu0Aji+jpC3YybDXzmDeB6LB6ilypDhroZVNLGeMVEL7odukzedrAsVI+Sf7rvAmlzXBKMQKtG4wlJ79M98eqXQkgUeaDlviMtwUrp4nP1H8KrTIXu1st0vNLapW9M3f6Hkuu9uxVwnBMCJ\\\\\\\/mVegqfJKMKjTmphKu4Nbv4+iRXqyD2WKpW3v1CxFHvg1FlBtIqqATR6iCS8v57WBY8EQBB652XjUZtut1gjctqQBHsWG0lrlip3nU1mixkWRwlEEGi4C2vkZH3BOhUrrmgOgkSD8p3ZYpkcDdAbmKLBuOm86zd\\\\\\\/JeObqthEgVxB1GVeAh8bRlYdNVWm3zCu5wacKeAPvVcPYZg62XDLRvr3bp9FrWCePnGd77Vr7SP\\\\\\\/dYClXbU5h3hd9OlLPiBRmoXLGIv2SziWn\\\\\\\/iJU9nmWf\\\\\\\/fVjhA\\\\\\\/jaXVXpACWb\\\\\\\/R1zND9zlpf4wl8BIZZGLLsEFtRR9Nb6VsEIwma0NqiwUkZgE3qb\\\\\\\/hcs7ViH89umhsO\\\\\\\/z7HB52u\\\\\\\/zxESXAEbRj+vW6A8iU9l5d163uOGmpm8OVn5lFru6ng6G\\\\\\\/fcDwSTvzahodah+yWoEhpfb0TjIBcIYvB8pJCa2RoBq4HhQT4y5JJTrsWVWxyuRA1iACusPrkjmndBQEmAfkt8cUk5DSkl\\\\\\\/HPppYLtTkrX0ZCEo8l8B4+\\\\\\\/PuABauXbo50Z0PtO5SjamPgKBUuI1o+RwPnl2XCgKZHL0eDPPID1yBnOJCD7hswAbimHr31nT3zTLRMHXIsKH0VG1mvO3dbuZXg3S8evVvK5RsDNJE5ZTe4vpQAP10oHQKQj+EEnW7SGQn+vsKbCblCBR5MlzvCH+QQMzXdm3idi23uCQbsoOmeWjNzRRIH52Pim7P9QSwUBjKeG8mkH26KwClhGbo1zwexfrETdeBWJZMpSHZQ+tiY3FzZcL9hQaIUfpcY0+oIuwoSRgs3ARC+geBh8AhaFf5JegDs2Z5EaWpJFFEBOmZxp2Qd+v+eZSOzwAwLQh7A688PWV\\\\\\\/F\\\\\\\/D\\\\\\\/TXSQgpEBQHDzKHQ1UKZBW5ctOtXaB8yTEhak5jgu53tOssFMaUB8z0MCTYe9Vlg8BPCeu8Cf6Hf7mxesxgjIS6DjssYwHKOpdkbDL\\\\\\\/EmfX2kqIRRZMWn+e4RMgr6yTLRzq5VivGu9DhTOqBl75ePHEk+vEXt2tsVKZ1xZb71TpGfP7\\\\\\\/QliJT3QDHMYxGzukfgTSLLVqUaYtOJ1EmUo9CeFFz50kCNgS5E7dn+xn4fDFYPb2LkPxDPKrex3uV1v\\\\\\\/bZYADmWj\\\\\\\/4aYNokXSxlSQ33N5nFokBkDeqsBFqZC2PQL8\\\\\\\/C9VTqxjQPtwcEwI2ftGwIEHO+Viq+\\\\\\\/bsDkQooDWjSyM+ETtKx5TXNefB0Vs5uEWPfqBMdr\\\\\\\/rmOOOoPiFW+5oQf156ZVaLtf9dtZ0adsH15x4oLJX+U8Zxe7jT3t8ZJDBKVF9v59gf7i449u7lJaIw3emgrMO3hystH87hvOMeU5hRvpTZ8vegar9mB2rHKE2pXSTJ\\\\\\\/zQBY3CiB+7QDmBgHuNIlGf+6ywxeJ6p2684Dd0AA5GscmsDmpG3CjVt\\\\\\\/AjXgpgoShipGZgYAKZ7myn\\\\\\\/P2wmD4JZ4urXEeDYxg9boUkcVyCDs+3sXL+3\\\\\\\/+xcCp4tryKZ3Wv95wfHItxbsMG5+\\\\\\\/\\\\\\\/FtzVV\\\\\\\/VzsRT5c1H1BbNhKgPbwLIPsVf4LXlIgmz2Mncl063eTtVipufpXUiTtqRYtged2qjmDv+rWsd0s\\\\\\\/gIGUDgvsyL8XSUaZoWIajtrZ40hx3ccRB1KuaVGLEcfelrTguSdry9zm1AcgNjhGmCstXSJyOv+8N+TZm5bzjr4eiU0xWFMVNUPPwfFFlA8giboQ+VMy6zp6pukdaqr\\\\\\\/bTHdYve8Ncv8Wx783Ke6CXuhqB6tD7ciM22t6e2NzWqV3poolHt1BiSb\\\\\\\/bXwV3SAmaV8lLmBIy8AHio\\\\\\\/N36QkCfcXhhIwbdO4yaQdgAsevROdF9wD8ppm1hTBTSe68HK3A3pAMRs+wWHN+jGbeweb\\\\\\\/WZg\\\\\\\/rdwSPv973Ddpx4eQxsVJzbNjm+9PrOLHuKTPKtxH2BJ4XsCVDC0i4Fm8+ctWM3ENaU\\\\\\\/8zohdf2q4w\\\\\\\/XZrr8UEsHNQ82a29n5ncU5yrX1P+PWYTyvlar+b9AxNvwI8YDMHvtsr5T8i29Qog6lnMxDx5Ln+pGYdpy0MLrwXMnzRaNXw+nfjWUKiU9XzhI9WH5bzd6kPQxjXJD5ApNeH1yP2E1HWREfOYKqsE7NCQ4vvZKzUkNoiULs1N+cVwXbZAlIa5uNVA0hmpW+mFpRyUK\\\\\\\/1rQQbgIHISyYwpdHiMAkABdWWLq5AdhtKE4W9nnv9rDWH8v9amjrO4XjOw1n3xhFv2sWthZj3H4UAtLYZnhD3s5sHkUruNo9guOzOY0EzdQbKwexl8Z2vC6sc06AtDhKhrVxvGjb5AWU3xNB5WSXrOlUKCW4fI4+0UUxU3pruf3t6FML4OMTenfQt3X3bM6ZdrqAfwUKMxy7GZ8CcjtFPFEd+ck6MjYQM2mFzRewzhM633Du4tNPBQ\\\\\\\/hhLs+yxdUVR8Xysv7TL96LcqR4O6VR7HsTqli29P1hEoWrNdWqzjJxDHtrBSfkWRJZt5iDipGA2\\\\\\\/vacgroC0oxrRgwQlkUgo9PA2uh6E1XU7tuy\\\\\\\/ydTg7D3GiXUAs69UDWNTf5+A4IJEwTxuGzAhfghiLYVGX6MGS4ULCt4B3RR\\\\\\\/NhRMHrLZsKyDmD1VvfdHSfKmrHFpx2h8LQFMrVtv309mzyI1SA8CkcKKM1a0ZndyKh2YrTIGsXYQPCpuAidqB6waOscXde68OpG11gzDVYbBUW0Ytz6\\\\\\\/hBQ7sLPU0vBR\\\\\\\/RF5hAj6nsrmaa5nTs+4xlnSfVzYp5U04L196F9WUA5OUSvXrvtSrQeUkz+t7IpbVA\\\\\\\/BoRdTW6438Gf9N4ukbALFfB7d\\\\\\\/ZKfpDTaSos7l\\\\\\\/CLD52l58S0gb0rwQUJO8d9eauva5yKJRujBftnEZ+4ROapewqNVojd3nKvx0MWmFoglD7VyrC+xEL6BSRWYrzOAAVZnMAqtayy1M18LG7AG1c5E\\\\\\\/HvsBn6iro121imhkzb3PXHZSInM4pDboiEFx98zn49xJF9EeZT3DzhkadExBPfStMorpRX+u1DKcJ3W8vR8jFo6oBm4Jrb3xEv81fGIVMxT8mdbGyDI7EPm\\\\\\\/6T3KMCnXmj03FCKDT\\\\\\\/FNAbzOZPhh0Du\\\\\\\/zUl\\\\\\\/rmdJNxw7f9Qw+IIDTAt4A24jm2AnxPZqrP2qLDhff71ZxmOyw2cugNZfA0fnlHvZivQWDNuwSkWlZaGmJD0wNhOFyHcl\\\\\\\/jfnCXDtXHriczCAVn3yeO661bXzTbuexhgXwNt5PlPmtezeyeCTI97p2hnb+RMC41Zus9Q8Y1RxkplO0Rk71qZ9s8p3gYgnjja\\\\\\\/SkzVEumQaKPe1EfBaSUnCTEptLSY1RcN2opAQVcMoG4NZXND+UGkrSc2dbn1RooRvXmWjTsD8LwUyhWVY8CSrhjYaOCp5H77pOK6RIdYMhJFez9Wm6cFEcH5dSSDTosDfKgQKN6qGHzwE+MdS57nZQL00g3tzKxAaGVg35gG6FdqVpbjdnwsuommBXByYBKAV40O1jOq8HoGnIBjFS7KBQWrUb6n5zJKEv+uwbbXzFHlfB1f3kW\\\\\\\/JB6rMftDFLAo\\\\\\\/PmHeQFIOaa6g0wSYrEFz1BQdqkU5kR2k0ijsyaV8gRh5s7xDG0XSafrBSFBamZONkRCPoM20JJe1nWBHT9LyR\\\\\\\/bmIT+izfINF64dDn+Rv8OEzrGNn+Fq9BwTcx9gD1lYY7n50IokquIfA0MAkL9ELNob15fFODq5\\\\\\\/wTNC58Y1Q3z9b4ThJ\\\\\\\/BsGjSrYw9Mwv6Wevpa2YYBgqA7ikDfnhuz\\\\\\\/bOAGsMj80XKCjyLfCz6PBCJKepnN3wo9wkFhi8ZlGyeaiLqsTVY4hSONDoeHdfTS\\\\\\\/Br7t5385qSC5oOA\\\\\\\/6Lt8qY7cr5qWMdGaVwXL2DPc2pYEy2N84uumYIi3wqTvupI8ziu6tKixdRY6D\\\\\\\/UBpZmmiLo08OWdZe1r53vluK7DG3g51cj9CsVzv81keBxK712TwZ+PXeS9S7ef4EDpu22z6Sq1ov9i8dsBiY9xyepXHo+73OwVnIlm3iDYYuG\\\\\\\/uoJbe236+XN1V3atm4CuZ0BhYCUBdj8+g87UfgkaLGYK+qktOS9aOvfBujsp7bHwOxe8nWKTS+h9cabi6TIhH8fbm+EICclXbj8gGazjiO9hjLLA71qCt9f77Bpq0GNeYLYYVAZzCXHU89i+etdvFMO4edifabm2b6ed9OTmRGVV34Mdt2vGvrQYs3aJZD8Rey\\\\\\\/Ki2mjbpzuXNEzCOoekwAsUfLkHzurClC2GootGGoZ\\\\\\\/lSD3IBvvEBzw864Mve\\\\\\\/GZlZtpM2OZQ4Tgas\\\\\\\/t24B8KDmO53BEyftn7xPLp\\\\\\\/4iZL0cu+gqVEgnoAagmjC7PTRv4ggs06vSOUMaygEWT7bzF0owfBTSHZ5Rd1XPcgVx9uqgQo0A53tqsoPe9nxDEkiozgANbRzsoVfqXYOy34jFGKapczd81Qr4iZJNqqfPVEmle\\\\\\\/dneQf4xkxQ0uf9Eeg+Nh4z9F4O8oibMV+9Rk2IN2MF9fGA0aFJ+M4RoGlIDp4dvziEnuO0voQBRQdv1DFSpHxmTMnCqevzlOWIsFGZrES5Qm5H8hVsK7zj9fhik9z3PGrySp94qdaakGZjtov9+5rz5HbntQw7Ovr5BTVkmz9T0VU3n6tuj\\\\\\\/1yugLuH3MnCnyEgNUw+CvP4j05vNnt0o7DGN7BjXlU0VS2rQ0bGRwI1a177fmtYwytvRq4iCKfe951cr+ARsj0HndMufWGG9abEnKCyaLkwNBQD+iO2Ic65UoVjcdHZu3cTKzHl8n5aZr7e9XVEE8O\\\\\\\/KSHPM\\\\\\\/YeM16HkXKsEHmbu\\\\\\\/tzSWuzSAuRPe4NqZxPKBOz7L1LK7o8nkpoLA1Fcoa9WpLmlRFxV29qcY53EAt71uxq730y9N1ByZdq58FIYOR0bzaJugryhBm3WhJc5t7lU2zNvK4EO\\\\\\\/Jtco+Hdm3WadCWJvO9GW0scvGhtDRtmdt+MJJWwNqBN0BWPLR42Ice9ETNc9AghPciaOb\\\\\\\/\\\\\\\/X40z2jGH4ENjGMxC6KxqZwHMLIGS\\\\\\\/Q4PIEes\\\\\\\/\\\\\\\/YnYeQnEYKlfeFElAkyIcKY9VBnmDq8UF2vG9OT5CReZKjqqwTuZM3DiW4NFc3SqpfCta1L8kbm8urd929sHIHEDmxRm921c88mLjAmIILsN7dm\\\\\\\/2KPOBPeFJTnkI5Sfe9CG5thsyBSGrNisrSLZS8gps3P3lBd60wfyiPn+E5fpws3z7tvGzRNi+I2mmMc+6HmhrmOsKRMbdhxBYiTqAQLOZhSFAwMBQqCkIgNWg2iLaK5HkyRZ9xLsJq5uGX+IQCWM\\\\\\\/h0pNjRG8CC95+XzoqnKqTNgvfgNxsczfiQNA\\\\\\\/HSlMd2oNLeAc9QjuEl0r1fVDw1qxMQnu9oUj+YhRP\\\\\\\/WoDCCOAK4nJRSYhbJPq5ZOgvvaZQ+Ij17Jb5HJ+70wvsbJxKiklDtogAqKpCDtdHOnAQSWAdLvMA\\\\\\\/VZ\\\\\\\/HR873DcmHI8h0oc0BEKsqxfrWzoorlBtRVvRkrZk6otR4k\\\\\\\/LF4UD\\\\\\\/w9p7d0Jc9bVawespOsVcGJWGvjlxotk8hJYlWnGPkNqOSMwzEwp3+XfnZ4cLNTRuMRfApxu8EY9SA4UUoGeKJI792qzri6kNtPrW0xN9WbvVbgffm7vFDt96D81DAwX593fcyFfzplyzyBfTrO\\\\\\\/3\\\\\\\/UgsHkZdOhRL9ge9AReUMDn1wxUyZ+CoQiRtDYKpx40WFPmf2YDCijUlQmC\\\\\\\/xcwK4DG7GxaqUtpPz0dHTf0c7Csr5UNMYZZjCrTWZ09+c3pztZYoprWUz1bM+Jh7QIjucqf8XddC+684YMdFNbanrH2kwmlcfWhTNBWyqjUXccUKXDUMmQ77hgAW2KK+sblxwUqXBVC+qwI8NAvNn2Jc3U0oKmxfETzsHI6CiOusYH+ikdTuh4DN6G797t3Slqkzf6xdDv+t40rU94sizMUrWKQEGXpwZhCMBz3iTtaFw3PRV1AspAkPXrAVjPPs9M9LUH97a7tnDbLiSBBdHZzik+sf0FLbumPohITd110VVHkehURF2WWXNqUFbGMt9zVjQ2wu5oVUP9fvbsYUf0dYtjotKekiLzCjt\\\\\\\/gEnhmm6JwgpMNefxAvRxtfi1NOjwj8X9YcMedQW9FYg3uOXcYUce\\\\\\\/pV9L1\\\\\\\/d5bUHEn8NeXUluJqhj+x1H\\\\\\\/8P7w7qS\\\\\\\/nTWi0Ab16\\\\\\\/95PMXeFQFNI4cQ5\\\\\\\/5A7uv1bLemmUctR+ljCV3MrBPFk5u8EK0g1m2zdcrHd7Byto2dPweW2VPOlEgebrNoRzsrX9fir5n4qGAR6NBxcfBhPylrqECt6IaawwOpOV99KCteDutFzun134puSVTCAqrvnpPQ1ifCCGjwl7tMht8wIUdY+2MWElr83pki7cOLNq1DWWnhjSAlG8\\\\\\\/ZcP8KDSOEHYfiekpRiblzoj1\\\\\\\/6qMAihrHXVcIOTppqus5ic7H0DPG7Bn7TJ7gZ\\\\\\\/4Av2x0kecksbcIyEQmN5dapABQrgtmyxFU1pZDKGZbX+SYl3ABvaiS7\\\\\\\/z113k1Np60+Sit7kq8U4A9jrMcVUJdqymScLC5ypVrvslHXBNF6GeDhd6ccRoJWcJlsTwylaH2NYkL5UZ0g4NjKvzIj6B3ydqWJF427ER0ggMQzpjHZm12GUm6S51kvJFcoYCaHVeOkiTII9ZStbW1fds2Sf6TW\\\\\\\/Igedl4WzmJwPmXe34SzVMIJD4EktADDqD6GkEr\\\\\\\/PgE9T6Q8aalojFPFxzl21WYPo85Ni+FPnrUK7ucUUIXAHx1bRlyc8zvUDaENTijk+p4a+uSxicsbnlEJpX+SqdDzOPrRrkqupVBscOVjfdj4pOs1I1svXbm6DJ1O7kfQtq1lqtjvT4A+8w9Wk7hAZq8RSqiMpBqTUkoNwcpthw7n2d6+Ej31S\\\\\\\/NXRzsGwohFd4EYL4j3Ur7O99j+Gy2R4HdQdLcwDtV4boycCzKBSO7ESmT2gQ331ZP520oMfPRjK\\\\\\\/6RSxm\\\\\\\/amn0iMvFXSZLCzU4V3BaExdPko5mDWnrsBHld8oY4EFIqQTVy7ElUfJue\\\\\\\/\\\\\\\/TQRPIhDoKTHvRsX2hRdxbl6TTz0Udm0EzXuhb6gzMEM5F234LOkIPoN+KXMiDMeCvmm9X1d43WqW3JcRVbXmL+ojpINIeVsUiEkhXSP1jDAKKiKuCC7eXGuMp0FZRlRnEKgPYP5Bqab+fonH5JBFZsrQS2gOdcD5Z14isdlF7irdu30zuSVqqR7qgiEzmFWMws2zaT0+e5\\\\\\\/RtlEN89fV2XzJavquLe0E7+9xUL0rWTMUZpT9BW4VC+Lgbt3oIP8pdjzVO12qlPyzQwl8ScEllwosgKv4WwOf2Xnx559h6MxPg08RGg5ZR+IMDBTygC6McP1FndDr5c8Bq7DbQmJNp3DgjIa0sySes9mI49ramJalSlR4xrNi351LaAFX7\\\\\\\/o5NBsFJSgzkfsHTJr6Y7bFQ6viP8uWManPcgfGuS42VwcBqr2zIaveB1aaPgnGcthXEb97PjhwX7C3Y1t0bPLA6PMWpg6aPx88+LrnHcxWzkn0DcWFMCgTFmCIf9TgLd17N7Fnr+78+DDkUcpIWJZQsp2zOfbrHTeS3T1lQYwJanJeTwHUq3XjTJV9blp5x0EfENe\\\\\\\/b8AQaI7X2NdaDPor18\\\\\\\/KV6kIsiDjBKJQkVLpVw+\\\\\\\/gWtXUXSbYGz8B5IcgyhW3BtQKpb1nSIn0Vg74Tvnlm1t5VFZWQ0DRbkAQCvOa0SvX1PZZ2wHSwpM2JSvK\\\\\\\/gkANgfX2fF1n563pSZeEQTuruLB5cUM0NU7UnlM7hbCTYATt1gIGPjbZhp8rW5wDCsIVNl3vlYzA\\\\\\\/l8e3fTF+xT3L18mmr48UaDQ5a+WtfmXdysw8yV\\\\\\\/ks7adQX88rb+umjYi62\\\\\\\/tFekXx6chW5mNx2ulgWw4661OkwNaqzQb61F2h10yodl2jbdu1eeWsSEBcBZFMNolYbt9yljeiBT7ptNArzvUaE9RD4C4tWSDK8P\\\\\\\/a\\\\\\\/g3udHBzRjuoqnfg74SQrVKSkS0YvW50jWasanhDcqz8CcLcbkiNsa2YqodDtix7KLE8ZJX\\\\\\\/ffla6w4OPLV7cYDpnRXCGF9GezoHTVeFvc4xUaLayiIiUaydKwCl8lFvfgSc82zy4ovb+f8AdoN\\\\\\\/APtEROWrinLl1eDa4OOszRP\\\\\\\/A2igGpmg+SxVpLfZWgRiKmZdcwPQpKB8F3mwkfXjLQeh6Jfdcubpv9lAtd0vdo50C6JQbBelOgJ4YodxMFUewtcllSkQXx7Uclu8zhyqyA2X3Rir8AZe6uFEG3ENbtBOirURyqAXKGdLVnnOLmzQG7VhjjqjWxyqRNgFIWYhyLRJHZsgywcNwnyMP4DY4NwUrEUmzqobyruD23dJxcwym9aw57EggzbWg2aGqM02jg3INHGWLWx8mC+qJzoF86GAOxZ2LDGZiv4JW8RUOHvKqwbvxatUdea5QqPssabTa7QgQsvhuJCQ3sR2iJz+TpIuVrMqO+n7CqdVbJxNp78R4D0ucJvm4lgLxSaL7jl7YVqglnavAN01GfyPniy\\\\\\\/eomljoYhtQsxhhyykPwjMOdt5HYw3haCpInxKguF4Hfya5T4hg+5q4Upq+P96DDF\\\\\\\/QVuOzWIiZnjXd63S8549V0z6c2RsdzJDwLLoOQ8ilJYCuA\\\\\\\/PAsG7bXdPwICL8K8gJXMi9w0scO81KgIpPFwGMVXepS9ynn3kAFw3k0wKNflarzjMoih9V+oY\\\\\\\/C0hvZ0+sXb5vGqF2r1jwtnKP4YGcoqY+8cpcNtGbuMp\\\\\\\/AlHuYvyQ6XZZEwHFkVslBNHt4PN5E\\\\\\\/OxZcobAeW5ZRo1jYXRYVETYsIIZibeR6XUG+Ia8Gyak5IhT31TViMphDs5WNbicaz1Z08I28LV8FnaHpnawsQGfvje6MUQqrB4vO0bSTmGdvt2w+IBo8DCcSswzJwhQC2C0Nf3sNZwCztPa9lOVnmchB6KOa5Nx3STAe4FW2tVEzpVLpAb+tI59wbQGGv3gXfbDVi281VdoqIsaNtNzdC5ltfKeUAtMWfwiaNS9SmvNsY1bYzY87UMKxwBQvG9th6z2nE\\\\\\\/izjh+HzjhJjc2Xh7iXNwe1XtIibGUIxcfFDj4e1QtTz6AGLi4PUVMAMwtvHdMtGt+A\\\\\\\/JQ2h78L5Nj\\\\\\\/fc3hmluoxZZ8dDXzP56DDDvNT2+FTNCD6SxPzrftiQI+eaOqKhpO4W4hJey8cQRGJ4nqxkkEUl2e96o9+GsMkCEK+Ia+nF+oqj1QNZLl\\\\\\\/OuREQppaZrsMuoiuYCjS5w87hFUsshiUsbuyELweQUoGUYH1mgb2iaLDFmVA7z2sDGaTPlKKmaAcI7iq6P7B74shM8eOdVzPD2bk7MQcrQefm2\\\\\\\/gQsZlWZ8JACya0CyL6u92QmsYGqG2F93IVfhMe6YQ56X58qITc9YZPYT+ZJIUG9p3Dy6kp1qLGq2h6vlQOx1vuJ1teJOyNFPiRZtEe+a3JNON7f7TJQirzBQ9e4p+h7jfNmWgNv\\\\\\\/5OWY+sF6oDjeGMex6SPuDF0UlMWQPmZNYfTXOhMEG5qdeXnvI7FDDwSX+C2XABikt1zVh6flvesqpdAqRbGY2UGkMc0WqSd7DYYzXbmh+t6HllkrICsjnmRaKomN\\\\\\\/BHYdXW\\\\\\\/X8XSm6SvTORW7GxbRD1FRd\\\\\\\/oBOV7BZZ\\\\\\\/wyVYJaInf6pDSnQcBRvWyJCFfKBohbV6Yclf\\\\\\\/S1GF\\\\\\\/rj8CTBu5jVrwx2z1ZbiCa0hBomKEh8mnBqiY9RBqiGgR2Fi1XJ\\\\\\\/V2xMDX+v9041SogLTD7i31RatWpqVp9cRsPnrdRPRpvpSg8qpBH42HKpoATakczIvmSxzjLRc0Zbj\\\\\\\/wiOmmlIqutCqlhn5yV69iYUKCTmotAjNEGLiSlGjk3hELS8PP1ax3+N1ONM3Xj1fkXjXt8znD8ybGWEk9scCOpLCgKF2TPjRtb5+1XLwFW9z4Xfdquv5tongYLuhzWHTIBaxBBQP+du9in4FInAM18zuAdz1sVx0naupFdQwkha+ePaVaIIxl75EILS+zZi\\\\\\\/USzhNQ50pjd7Je\\\\\\\/y0526CbWmWsSpzTs\\\\\\\/CSjFB2gJl1B+hC8TNbjiuzUAnY8Q7\\\\\\\/efxN4d\\\\\\\/OdZCJv\\\\\\\/hAtIgxUac3nlStiXlths44JAbB1sLrbdQwHN8p47JgI8mxS7VMSI5IA\\\\\\\/IdyOvScH5FtOm7Smg5GZRNEV5IskjeGHiHAdTpfgl2EO\\\\\\\/Qb5u5KK6z7Agv1L7XfFdRdWmDwIlyIwc\\\\\\\/NInXiZps6lAQfaqaHuEA4lIWwVNqKD8Io0p3rvHm+4mYEPEQJZfA4+hOJ+LF+Yfxcm5Sv28BYEuAMadoZTkTyE4XdmB7+nd+st+wX4GhXyVCvwjN+8O+hzw7jEKBM5sUzXRiDmwrjxUejzbeGkiyXbnuaN+wQaGk39xPEvDoX59MDomL0wRuSkCBMN0Dx+V1xnxhhp+beDUkheF2mP6Ijj4KlYNnjeyXzzbIotDKrOcHm\\\\\\\/wYREy9j6LmhA5l58YZ5M07YMDjskRTZcsRPoHepnSbAvBGFgTpQxrzXfABrxvQaP0+F2qWHysT7ddee7K4dE+LjQr0v44QUFMaV9rf+8twZrNigzhnyWhzOHiExZgaSLJxJqBd1s6DtrbvF7Dk5USXBrdqyqx1UpzkHA79GfjlCWtaF9I6Uvuk+oLjhew3rvY8cdrWnZ8QkpLaV5nANoH0A8y1QzHoNZSlOfpSX7yo\\\\\\\/DK8voORJ1pLSTL7A5+\\\\\\\/mAqcFdQK98oyjijklXDamJ7mMLFKFG1N\\\\\\\/NcQrlhUiCYsGu8HCKSKHZj+l311exb0shKnHf8tKxLLv+u9NAwuyF7elUKz73\\\\\\\/0tXZeHFRaA0WzpOVKT5QT0zJGGRtbvGV27p4VKNl7k3LU2bN0ODhd91BljZoSfjmJc+LEmeXCP3t+DKPo3IxQP6hWi6tJnV37lDD2vYBgBDKHV8dNVuiSzcWVChuzhq9ogM1IvLMI4x1Cbrc59X0BA3AfJ\\\\\\\/1m+6+eFDWOwb2y0oLewWZMu5NWKye0bQUQxkJ4kmfluFbYlUSJGwLC7cAazDLrbh3CIwj9Mw18WWbTsX4v9QbhjR81QogwmXLS8pMWqJNshnfFERF8qViRkYEXLvtYjG74wqHGBqih3VEXbSyNFDwGGl514x45GodGS+MruDvcN\\\\\\\/roiwbaiHoJrx\\\\\\\/0IU11nN3SQfP1AiicdFktGzcIPS8ODX8xqerKfnpz019aZEqwrc1BL0CiaYxRrkBA99O6UyqJEI78EeZgCG\\\\\\\/lhslVojkEoPekf8\\\\\\\/+OGQyOuPGgzINCvy+ptPVRnxi0C74C7RVqNE2GlGX8gVZ0rOq8UzKedS\\\\\\\/IE9MFYXSJbgfFUyBQX2Zd1ntFFhy5XdqZSnze2VYAS2lxHQK4N3P4xBoV84hVv2gTMe2IsXeTXJBtIv7G7IvQld6c3kQiS7Hxt6Upb8By1SQM8Dx6x9tPgN8U9FHd548kmmtKSSR0G8Ut5R2H+yTql6cGiNSEVGTe60lXZkS6uJruszTbn6G\\\\\\\/+jQnS3abnujuKD6othGwDTnODUoo0fUseV7jiHUBSoiRNgPwnTWzJjeyhEBWFnTiqNcsoDuj2mByTNu30ikP5s5GozPo5LdfsD665QVaCNehpitDgUTBLj9Z\\\\\\\/oKNvmJtaDSUk+lN+79inQZbZSUl49KJVUqhvHlvsLIQj8LmIaVQnPf\\\\\\\/zYnCyIX\\\\\\\/leI18cNs9XBUF82C02NbFVzlMl0gkCO6g47XAhIu76Cjl+R7a3qDGX39Qr+pxH+wTvWFN+UtGFmtELLmSXFwhN78aRh+AXCdyw+ySQj5WjBUFgHyx0Jq2ynxY482DkHSR42cNI3p+PGiY\\\\\\\/hWz5Pk4OYdxt1kVllQsdOZZgJUirqyNYTMCkqbmrlChlPZTuOTWqUdRICNRg\\\\\\\/9\\\\\\\/pCBAZgDcN4lPInBW637rebcnO3qwej\\\\\\\/ThahndoRyOKr8dWULeI480XShKRYjhHwrW8Ta5W12dmjPLbTBfP0xm8OoMsUUm4Kid\\\\\\\/bLQJ1v\\\\\\\/SRJwYgVE5Rw3r6mkn+bnLOj3TI5JBAEahcq03Hk7G5BFX2mNhAl+pzgfDRqIPoYKUvP58ist6NmfoUG045GqYBIQ6CNbYLUZQYXHav6i8vO1zm765xJOPfDQB791wIOhDwGqUGFQKRv+2t70Z3Of3lgs2tVMsA2RicXWYS8KS4jIpJ2nkXMIwmEld6mhY9T0zPvYudppba8UEwbFNQopfIxfSydAycGGfn3k1KbuQLza4PM5hcTVxaifDO2qiLC4K2J1kwOFIIMPJtSr0ZxxmvG4kEpqpoQtHNltx0cqSxTdEyc06LZYK7KosBuE+601yjbbmUUdqhUiZs9wnuCKnQlZzQBxV1IvZo0ZJdB60o8epvsS9VEfnHGGNpPEAifJ\\\\\\\/rBvTiSPTbpp5gT\\\\\\\/pr9jXN1b8rt5VbXsphu7ynuztvxmyUkv227GQDu5B51thEC0VQBnWrO+7ppbSHmuD1MvC7LzDN5Q1H4Y\\\\\\\/NMcFirx535QEyTqzfIUsjBw0uNr3+2erBHOxU\\\\\\\/31dqm9r7RbnWofRCQhS84E6Bb4yoZML1wFY3xVN6kQ+N\\\\\\\/PK1hoFSu45eaDLAr1uPypuujkrEw7sTbM+XAFuuUZjxu6IWqiPCPy8j+ZNfhVrb1hsVymCoz0c5NihlZuGv4MsHAwpRPHrdRBlY2PJal6oOrRefwfkqCdAvK1RY9Ea4ePAf8NAEFTtSJbgbbZPVP8a3cCKtbJdazY\\\\\\\/4B3tlcZH0L5zrBUpRlMtehDlCQx0FcKqs4CJvuTzyjS3uMpjatv2fDYqWxCnVbqlBlmCf9b8TgwhenkRj39Za6QGaD5zYCrGggggOA1LAZOx2k6NR+UtH92yIYAQUwqSnc4MsifIKSTpi5Fk3vLgbFoGahxR\\\\\\\/n9X+L7C2qwRyQhxPteyH04rWwFxeY4xHkF+Ed5oRM3SjUvNEYFhrWjUtscvq9uDvqJkUiQPdPgQU6018hvUk6m18DtfDeZ9QmhuUSBEqW\\\\\\\/Gzy97teMm3OX59hcSRcP0N7nTALkkZcDZyl0+79+qke5kkZMfYbM626h0SbzhJKLP6AH\\\\\\\/NaKFGkyNXl+2dKYOMZAqZ6hxJ1Z\\\\\\\/7gnIEn9WF8Hy8W3BDlyj142W+v1xZFTUBS2dyC0dxuCXYKPDPIbVTSxs7e9zkVokUfkiKVcqlQ3\\\\\\\/hOrLZjp\\\\\\\/9JUdPZyAb9NIN9c9zGqCcm6e5\\\\\\\/FA1C7KRJ8ao\\\\\\\/vQr2PK56BVdvFlIBqs72ZACpwndibc6f1+XxZ0nUSr+2IkrLVfs6Jf0gdaf6GlZ1PTbrVtqOjf+NEOOYyQQwnJU+\\\\\\\/3mMqPFH0QnXd8nkZG7ELosZ3aQC1Ujz41OK7WPFa\\\\\\\/E+9oaExSmY9pYlzg+OdxExK9txuuyiQ6faok8bdhfjB9sLpoQSAetL5Eu7Cfnohp\\\\\\\/Gi4q+q9NGXdMmanTop\\\\\\\/RKD78JDbtRDMw\\\\\\\/OYIAbVIfjC\\\\\\\/2\\\\\\\/5NJrWITol2AybTJJ92xPJzUFPPBDO1ABDiFGJLQ2qa12X5Wybfpq7\\\\\\\/d\\\\\\\/XPiRClb3KmT\\\\\\\/1gQp3nXSWA6yN\\\\\\\/i59kCjWd78xCHyWbZZN3AvRJyhjBGEIZIj\\\\\\\/lqYxGQzC2jUJk+fqcpu+WqmlYmD5CQ+zYqA\\\\\\\/XK5+pesqid8vNyxn+sq7j0trPEJss+ACJri6TqZuwu1mqQyx1f6sUfp30Gis0DP432ZA9a5jTi+0JHU9UPL9KL1mGP1EMsIyESzypHcEp3rpLK6mDtwhCe95f7iSCTqUaaJ99B0uwIg9BawcLSqjlwaOXSQdkgF7zqhw1J2RW\\\\\\\/05ntUyE+z1PGZ3ZaztWKqOLMISmdg+jz1d8QSjt83H2Ucjzg35DdSSgk5sOpugaDSJgB9QK19qaSL44nOSGZpr3JB5WEZlhcBWQztMhadlnt4d+1wQtIZr+OL2VGjKfHGzCPNXaIsPl\\\\\\\/pVm4ehHZ25J3mt0t+FRK1l55ZHHzO8QMRJRp6aG0Z8v0At0cdK2OGnltXSFgYIy5bUa6f5owa0ih9hZup1Ry+H\\\\\\\/7KeCc2pwNKfRm22MOjlUT3NbLcv9gTjBYE9IIH9PJQ2AQyJkDlblSG\\\\\\\/hb6p5V4b9EUdv4+Nnvc4W2luGFPQokfBkGRf1PC6gpzYY5YtBMxbaeQe0zTZbkKw+eH0dyWM5JeSMcgt6puLwYAzEiLDegNaXwZEMZ2nxNEiY849r\\\\\\\/wL7LtZgJrfiWMxvwW+xTyOYWBkqg6xLDgnBf4FT3f1zeavbJ1ks6N4T2zpcrlF7endQKm6+BDJTGPUP1EAxzKRAI+9GiCWjvcIdM17B0UK1bGU+EUrKlc50x\\\\\\\/H\\\\\\\/R8+evf8BUoqja\\\\\\\/KabwFXP4EFrDMkMnf4FcO1RFLTbv5\\\\\\\/SbDH3gCSOJ1byf30+2czqZlivwAY+oU6YyVPT2HpsRumUaNyN+HBPaIkAmg0HYsJrdL1bhe\\\\\\\/64XctD7t8KpiEUI06\\\\\\\/i+DLANTG\\\\\\\/Yec5+Umnkh7jDJ5OguAkUOhnVzqzXW7dmNBwmttMqYn6avN\\\\\\\/aPHr54YuYyxROHo5Q6J2IlgMiixKTSXA6a2bh3wWRyJvgVDuWfEa358HhxkGyCw243q7wMUsajBCpe7hvCiMvV2Tv60T+NSlSrcAlRbtEQRk1PNgqGAEvntF5SpLNnOfVc4BBu5REArNqjUk8bsF0hMRDbZoTYwznnRmy\\\\\\\/4Hzvqr1msIKjZVIy75AXhENuc6fvfWcQCQzwVEU2SHgrz4D0muc\\\\\\\/IP64QHxdWdMn4fICoG7uUJNx97gZLSbQ7eo3I418x5\\\\\\\/ICnZMvmxVcvdWXZhKI7yd4cZjxKRxYTElCfKGGLOvnnZDybzbc0PDDTQQRkUjEKMiVpOYUZ9yZgw6FUgEHG93a5WU3bCaDIan8uZ+l9xh2xVNZOTsbBgcmvnoKOUpsqvwLSuNl\\\\\\\/l3msjsBK6UVr00v\\\\\\\/4hgnbF2aY5kmfa0WBa4MexIYk6VHM71M6ZMXqQxs0gbJt+HZTl\\\\\\\/h+jSVx1GwbRLpniW+XjuRVmY15fBDsurvbPymd9K2eIbgArELH5+SuPQV2ufLIK1fPREwALVJne5ltxjw5NopMwzzkrUC6kvUwrutKW3KITen+qcdgTSz0cp56WnTr9mTQ0YvcKFAptPFdyfOs56ATI355AmqXvwi0c7Ci04B5mMHYwNWKbmyr9mOJQHd\\\\\\\/shcfNzqV96fUH2KO96Z5XjzPG70dt2oz70NR7ceqVei3Nc7t5+QxdY+nZYGvb4D7WUKP0ZmYnesLZMwUTCWgvOl5l0Ry8rd2zJDvm\\\\\\\/h24AS\\\\\\\/K+1gGbu+3lWzocsMZofHTqCHrnC\\\\\\\/iBo1IuxzYLCjzrceYV43YNRYJbLfCB+pUt+BvNtzkU61cPfICOHejSAI7qnWiSElcc2HZe35ClWzFSoOF5WMj5QMYSt3DcEc8k9by+kvqvWVJPXfroEJYGYniY2hofdyhY9FBmrYrpEmrIRItY7KGpsjQtDQlTSHxxXoYR4xzD9jLdkctJJXYFAyHpJmwEurtrxXU0PpuC351fD8fj2bWXcGaSmS18tkc0q6Ud4Y0E3\\\\\\\/jnKH0d8xl4Z0fAnIJgWdbH\\\\\\\/PZXQRnVFAWsafwtynky818k5xpAzxkLo2Ej0yhUuEsjsGsNVihiIdNNTaIc1CuwCyCrqlT+moB7BaK+WuVBCHaxwuRsGQBXkhSTxGMhtv1r8neRzlXOgYrbTr8Qz2o+GsJ9+YlT\\\\\\\/s7jqHf4E3xsxH4ESBItIZWyeobqT2gSJh11NRB0yv\\\\\\\/QK164jseK9vlqzJkleI1PGsf91TMQ76R+YHc6VTn7UWp9uPvwrsgH8j9ZbcOZY+8eYlWEVR\\\\\\\/jHbr9oGwQzIfdvcRLLvEC4Z+6ZbIgIHJji1QRUPMrVwHFe4aYN4Zhp1JH6C2crTFzvxT6a\\\\\\\/LuyqnNgPvjFlFEwA\\\\\\\/Qi0j94PDu6ATq3h6Cb2019CwcdLH76OLRyPO5AhjydCx4qP0\\\\\\\/X6ONvuhxgmiLjNp+LG70E7OdH3Q0sqSYkgNYArmuxgL5UBZY7YAG1DnTHT6qlA5PHj9Bt5q7jIdKWw5pf9dUf5U3uy8aVzB1QvBDX4lYcHKDrEy\\\\\\\/LGGHKwSuD6XedcsWjS4El5kjbUHVRHBG\\\\\\\/lzLkk1KYeDMMS1PlWPqluorFKlE8c8fTEqAj7Bd3clouoTAPSUovLOlNl\\\\\\\/1jBf2Z\\\\\\\/3VWBsaGqd4WDOnYZXQNAr0aMmFV85arq7Apxlf9k\\\\\\\/ZN4IY2dmo9vujRtGSrS2329UrT+lf8j8cNiiYABNSgDYSo5NbRi6CHExGlF6IqNylxCsw\\\\\\\/7Ym+5dsz7lz5CXeOl25Dfb3cnbakTK\\\\\\\/6GMdhk6FCsd8NlZdDK7\\\\\\\/j+OaeTk8XA3EQZnXXaI39ymv01uY1NenMTUMI6pAvFOGJX+\\\\\\\/DrlWYIIyWhVVFvtXXjSRs49UlIJm4\\\\\\\/yeuzV+tbBQ9c00gwdNUoTzrghZlcHdz9BlkqDHvb16GMpRP\\\\\\\/dY8XpvaGxyxTmzIcwrZob8xkgL1UVNGA5AttTxfpgytPc6XnOUqxcVEDeqPsTgRJxFNglExgblI\\\\\\\/WdgpjIF0dgy21oIHuurxphiyA4vcTOmi3TYsIjPpbgfMNDsdQCkT2VZJjnfVk\\\\\\\/BWgpdtuGGh5CRs1J06jRUP\\\\\\\/Yr5COb3CuuMG6\\\\\\\/6F5FREaE9mK3W6co8y2g8Kfs66LokedSDiu36eUIdNlMEAF5y0bNcfjoYaoQR4CRl3KXX77LyA3Nl3cXhblqGmMJdAF32bDXFDjMlaWjRtmh2X4Ewd0xvnQFPGg2xMb89ULQkbmgvWXyJ61CIRbr2kuDYF\\\\\\\/bG5FZd3JMrw72zaXnf1piZYfs8AH9BfDLIK8gXQRUaDtMuj4t8UMsp\\\\\\\/mWZrBc1r79geALAqOaCiBLQXU6SVEEa1QIfxlxmqm\\\\\\\/NCgZKQG0WuaAR+pW88jzSHJKlPmGyvmALXZ+KFRlfi3Yq9Mo+WYpHLsgIPPKJ6LZHm1ywiq2dUbeWxtb\\\\\\\/LoNeRY71p89HXtaDNek2cBxzqlfVHZBzlWFoCOu9RaqIDkzduTFG0H8Qkmx+a4CeFQrShu1nKQtXrY4uqPaRiQh2gTt4vrnRtlXbusySZooTY4hYf1qT19w8A\\\\\\\/WOUnqGPv0d8I\\\\\\\/fIawV+GF2lv35kuiWMyrDAsF5xgpi2m1CNBr\\\\\\\/T5j8FEkS9ha65MYoaahkL8V9xQdUghkDGYnNB3vZbGtcdy1jKMwDhlb2SXOOq849p+EuRoL12HrtUIIpaaZMWHL2Fpzq8ExKUUjPNaDyd91v+X8ri9TNc3thW0CVx8HtmSCrogGGw3bt0esPHx9lIj40PFj5SHjuxvmOpcfMwpfHJZTQdVpgeuUugJMWES0QtTFTsLYjrsVHGKkMbfM2lkrydmytQEiM2A84KNM6BnsnVBknpj0+KwN3un2ST4vWQx6z44mEKh3pjUQUa9UbvdvdQgQxq66WvYxeYvqpvjZHPiPUr8veN+Aea9PYwm+reCm57+n3lZZHSo+7ExPB+5SX+9MhMdN5rmTyS\\\\\\\/QdLMQL8L+BxTX4sVwzZ\\\\\\\/AgWwL3TT\\\\\\\/6056PzzfqSaFNYTLVr0R90oIEr7esRCOtwiV\\\\\\\/HrKr+cxrcAazNoVSKoW62prs7B3L39JPYjPOGZWOjrXs7LXFflzL1TMU128rMEKWES4PyFWsvjctqCqxSze8Alq15hgYy6h+fwCGy1tsqjb1+k0McfFAaWdfug+nyuw81rSj+TJDZYKFt378jFSvRBa20oWJd4FhAEturQc6L7IJY6a\\\\\\\/OqE9KTv8BWO+RgaGKX8ErgdjP8+YJzSzy36Sok8bTI+d8bxjZDy9jISmOjQTqHwDbu3Xc7SM9i2PLLQTHF6d6uU\\\\\\\/wTD8a7m+eo2RzafEmRghKqGniF+GkBIfr7ZvOZBvhVOBcw1C\\\\\\\/AB6oikdbE7jugizOEEDZSyZ8gfQfzKJ2H\\\\\\\/zAYs+tv9t7AbxUcpTN1Ski7hO0mBlHixR9E8GbOOQabnLPElbptFaEh\\\\\\\/HQ1+X6jdyD7UX+rdGGhYsdEyPxZe26TSiFytmJRE79Ktik\\\\\\\/wMymZ\\\\\\\/V6Zl42Wn6coi\\\\\\\/TDBHVajJgFMphejRKY0yKnzfQhrkiYooPwmV841API9+VxQZWdB3MutquRBDdTYey7hZmFZdrxLTC\\\\\\\/f4JdgoEWaMVlL4kwL2bfwatyj1mzt6C2iZSzUVjUNiFuAhH5AgZV3qPH1y7KQlWG46AKPyPXDSTgsczARlRBIRZU1cVoK1wom4YmbE8kJhAd4naagQzqGFPpl\\\\\\\/yZxVSDczz4ADZkBS75xxPEVToYOOIz2bMgRh7awCDT5XPqHqEq8hYgfRPN56e0yNAs5reVWz6PZE\\\\\\\/Nok1uRPwVlVAzBjgf1pyDUMpaiAwtVs+CefqGpoeKxFqo18p7YFf588kgvn90XBYIXsqUSBqMBBvJnO2\\\\\\\/BLRxlvpK2ur1wqz\\\\\\\/FA\\\\\\\/1RPVYh+wZFFnaNAXTUbL\\\\\\\/anTcQz7jEbcqCD\\\\\\\/lv5MUt84sn+iP0zsAlgYkC8r\\\\\\\/D\\\\\\\/8zqrDvcA1WDdr6lONH70dmXW+eCIJKe2ZL0yweqFnKJUGn\\\\\\\/KxPkuNlkswGWWZg2cq5W6mVWDwmSH3xCjrctKHbCSqz\\\\\\\/fOk246YbCdeyLGx\\\\\\\/JvJhGCrLLJNW\\\\\\\/doI0guHM6HJ7AoRdjsQHzGRPH9leFp98HH3nHuvswEcSLSXGOTMV4OzK4K1rjdxt3DWaE2zI1dm5+BHCRRWJjD26YRv33PlcFWtUhbV0jnSbYmON6Ko1XwWCetMrkwK7c8Hejwpd0r6ThSvVw4G655YGtJK+v0OkODOUOCWv4AespS2xdkqZFNDE\\\\\\\/raKwKiMoc701ByzKnB4nnih1JQaCcNoIqZMii\\\\\\\/JA0u+H4wTjSkuuXwcu9O2kMrpabe+OBslVK8SEanjFsBfzoh725GCeUOOc50YMaJR0Ut4BQLlMlIrH7HqeXMAlr\\\\\\\/+QNuWImo2nhA7qmgf+N3J\\\\\\\/W7YQpca0fu7kAGerjy1bWpK2531pT1Bfw\\\\\\\/rPL7djTGwFTO9a\\\\\\\/5FyhvkFcL5+zYI\\\\\\\/hJJsf+juQckzsWwc9d5xu0RSFLRIBC\\\\\\\/5avi25DGtQbDAzFVzoQcgRje9LXcuvnvA3dJG9kWrkG1DCTtyT9+ho7A5PEAtU17KNWRTctpMzWu\\\\\\\/qKQxk2\\\\\\\/MKOI2aZ3psdTXk3RDJ+sub1R8BgedYXGb4Mrnc94bCEvJXkg0Hb2Ybzhp1bwZsMBF49keADJHtsZjW9JDJX\\\\\\\/hY65B+Sg2Qin7izZBCCXkX\\\\\\\/oT8LvJ9lDIXY+p7UuiG2+hL3qfBBoMLXzor2BGy25SHLHuktICBM5bpLlWN0rQ1xSUfU3+LW4UnPwNM5gGZbySdDJTlZwG2rXq7mMiWYq3nuOIftbWbP8JAtMmys6gxblTObOh+TLFXBHQccs84gk1iBaReyE4xAcj8sEp0A+bY6Q1Ott0YAOac4YRLTti6R5g2STdENzqEH+TGb+C6qpsjNt1kMB+kwQCiOvIlN0iPTYZwe\\\\\\\/RhaGJ\\\\\\\/eJuIKREI20fGM7LZqmIdR3\\\\\\\/ggLn8RJPykeN\\\\\\\/nupSqABc3U+IV3DY7IgcD\\\\\\\/Yu241mMdzWTtUTZUENSc9ROukYJ8Bub4AceWq3a35uTM61BAraz2Dc2Pj9VGqDIVqnIUW1rbUte25ioJt87gxo\\\\\\\/xhI1P1BR4lEHCpeufSIEI53C1AiC3aEkvZvWsc\\\\\\\/czCAVZ8vdz5is8pZpAVq31XR19VKhkdKFyhasIuXs0m8sGjwQMpdipuKBtr8NymQF\\\\\\\/tLiAODEvDwJ3oQpWxURn9i29fHL0knss4THDBbytbU64DNG24zZ0PTuMprNOmDyLKMVdvLqfRO2K5QTWktN9kDyjJ\\\\\\\/MJ1v2rqMbX0UyLXQQcQS9CXc77cTcJnk\\\\\\\/BWU2VpFsoAOGgSWZScdV5LUlbcjfxrysoWTspzpdM20swSWmuYAElV7UdiFM\\\\\\\/fUF2S+e0jjnMAndwSnlaFP8Ir3bHqb+E3WcQObGKEcHZTmSABhIypzNVqXTmw8MyJN8Mij831Dx+fTFdKWEmeTuwN5LEAdka8o452tIOaQVPAqxEuRgnwQ8PvrUeyaasbr+5qwkzWYDJ301F7BDeWouUVzk9+h1C8AyriOogRJWLSD5KLfQ3XfAY0tp7H+aUj5+MxtyXddy1jJG7IAASj6PT8zlVbPQPRDBibDLnsmKXXWZDXY8fXTjjFoE1CkHVxIAtKSC4pMDoWYDLY8MEq5b24rk\\\\\\\/FGOPK2Ao99TCKrepgutYLEFQKTias7HwSxTGHFEGqenlelt5rZO\\\\\\\/iE4zpJ4kTV7HdBVCliHN24PoGoSVbUnVK31FdP8fX\\\\\\\/Vc3O8\\\\\\\/tv2AlQublUkidwpA5+1cRG9kfdmnS1foJT26VQfdV\\\\\\\/tedr3MbhGvHStI3Z8T3MN3IFTB\\\\\\\/CHeUoCjyOa5cFTSsx4HktmQtZLlu\\\\\\\/x5BfPNlFeg4J1LG3mATU\\\\\\\/eKF0xLWCPWqz0sfJgTxVeMkWJGNjTIkc44s8KqwlYzhT1B4IUtK9BCAj52EAE6EzbsoX5lsxoo1V2GcAZDaxSDJcIG9dR8R\\\\\\\/IvuGDBPZyTgeWLinZdJhvtfRzkJEWhvG9w8m8s\\\\\\\/tk+XN4s5sS9RJ9veI5YbkVCgEA2uVvKHPDFCEv5u3YssAQIIm2uQpCI4DJoK\\\\\\\/eF6aLZAXiXNtZiJFWP46qbdqs2gk7ndGTnkyvHxoqmjivHFNF2P4NL+DflIMq1Pwnmeh1X\\\\\\\/HuqwUiHv\\\\\\\/h\\\\\\\/FIeJQGITANKQyWpIc5CQEXvjvU01NvYmk3QIGEdhvvFzNnJCC3MwI5hw\\\\\\\/IKgsbLhmfb+GWPuAKelTb\\\\\\\/iv1KICEWQn3k0dmacUuU\\\\\\\/rFmJI8abOWgAGVnpFN8n\\\\\\\/vuXlsXaIfY4Qzf9ioEfkhNrHAjTVJn\\\\\\\/umN6eUpv0rDU30a4py2w2ouuBCm8RX5YhqiQM5zQ7kVKqZMGHkMvMncyz3OqmWImgeWZJdaH73O5EILSiEGXu16hu2+dq5xjpZ4okjrBmIHbRfDpuyjtDR9iu7DgDlEURBUbKoRx+1wIqanzlR464J4NxJt4KN3TVMGeBAFGnTYi7wp+qDPJtyY7I8XiEMV\\\\\\\/iLH+Rt9YFus6Tjzd7ZSXgpWllOm+q6BLZ0uCsseVT3md9lOvGGmE\\\\\\\/hYbm+cGSGpVncvPz0LZJwhRTqKMEE1bfP00B\\\\\\\/2JRh7lzDkloPZAnPK2p\\\\\\\/YLw+qrQ8fT9AH7UGUsBvtbolNDdVWYVoJ92YmV768KkPzvbq4C1M10PalFUP7Fh50YnRM\\\\\\\/d1R6KY7VUlAZHL1CR9h4K7WigcBHTCMyfCqIrqT2Mdom1iEJ98r1QJgQpDw252KHGX9kHXvsh+1tilLSpSvdjJagubkM7UCMlw7uDgv6W\\\\\\\/Kw5LJQeBGu8bREncjkbg52ZRxYFE1TvLEoFXTFGHUpt1zh4XMXykGNqZI3wPE4bfF4bgiAMDuYknOMRZPMHD0c1nlKF0KJXOAlCt8Rej+3jwqXtl1gfjxbRcO6OWxe4vU9r5zHfZeS+TlR7C2oUdtL3gVd2+2gJM6oJkp9IiDmnpqXogIP0wdKHyNGk19mmccISH3UgOM2o\\\\\\\/3V96mz3xPUjyxsTTmQA5hD+qPd0eAunGqELz7oDhnkKMZvTe5Z\\\\\\\/v5sI+8IX+fAd4GHdGGWdGrZ5jWJioHK8k\\\\\\\/lzGF2BE1E20R24X7zIe4Ol1dogkMF9nqHGOcPBT3RMhwrhrlsWY3VeEUvbMf9J8N5\\\\\\\/Q80k8TdYD1WOR5aGsdZg6de+AyVUt4RWD6FLpLYkGHKyUW0NMHB3ea\\\\\\\/p2I0RkbcTJWJGut5WMoyZOHB\\\\\\\/alKLgFA33VOqqt1Kw5vkD7Hi0dcboMP5eGHWyX5yq9Fc1pwP2Xbd9wWyK34XUu+wu++M0E8prGIdiWLEVoqIp+b2aMEnMNJlXb39q7PPVVfRJ4uu3NQw0vuKc5uwduPGep+a8vcyccM6+jY3sMV\\\\\\\/p4omDJbrcYgdguLGLhToTxDSX13yE74WGM6kDEZJ7azPH17xeXXagJpcTevQkXukxn+K7ERWNUhWTFmlTkszRT52VI2nP97OujMGgESPofqqU4IBMyrm89N1BOqTBprESxmN\\\\\\\/Z8P4A\\\\\\\/uwKTYAWgQOlmXNKE5KG92vI4f10XzXlDNaKZzVv4kg2oNLXvWyR2tPLyBObYK\\\\\\\/gzrRnL9w7TPN71EDyba6hoSXa486saBhLVv3\\\\\\\/8t9cfBIkqGYt5r1kCYauxsCCTwcDgPMnh10DrleZDgD\\\\\\\/bBdlF8daWtchDtdqr0BDqUPNHDf1\\\\\\\/7NH4LluF4F+in0wJ9+9qMk\\\\\\\/ZduSjrBSmdOzoq7R1jQ4kJK3VSvGXRRU73Vzp7wWKC9I5zpfR8HN+R0teTEkSoZVATLE5Jvxh7H24UxJ7uWgSK+cwtuMl6P9K9d2Qfdkt2HneCXgVcVQ+EklwNTZD974UCoKtF\\\\\\\/fO0gAIwTCX0mP8AaV1JFB+cOTR9vrHtNXEJTAkahijGIAD78FyjS+rp0Wqo69y\\\\\\\/uJUj7xGlAc2PvVH330jYg8l5\\\\\\\/zsHy+xIlyPe7FMgi\\\\\\\/t7zamkuLJNORHdYcNDe5DGCU2ekNnZJNriPTV+fZcJt6KLDF7l8kNAdHNut5XmgME2nok1yzgVyj2Q3lRUv2exXGl6PlnDX6RTOseo+wKoClwK3ampsxGpd41UhBM1FhSPWurUSoAbjC4legMNhaADSViMcYt\\\\\\\/em6Ml1dKo51U2BI+dFMmw7UFYZXI9UOYWcZe3yT6DPEVPDbhaCg+ZZNNOFKPwdGrXURcyE4cmqYIy+dTVYuZmLWRwMzxnSbAj8hyBIdiRBo4a5QCyUH568DWpcWSYGTRRlTlBJNXEVC7IWipqvrdrTeu5+J8i+gpQN6Td2dcvkSehjQNbHxhbQWoFaI8Zs\\\\\\\/GwoYp10yEihiG4n+CXa7gTFfoVn79oigfSkR7KuVyIc8oCEtPCuWB5d7kJ558PiYK5QpekAHniSYDLfnTKsB6Z62gGeN7X7YDdnE9V3RekeCwlU2ABR75zDjE0dGgb8w2xi1G9uHZ3iF7FOjayGhhL16LRd8XLK1T3FNeDxQOeeSY5QdMSEa\\\\\\\/J5KSNcWf2uOpAoUnf4f4eg1Pvq7ApMdYhG\\\\\\\/5sfpEuud6zlrGxVHg42u9Cv48x4ofc\\\\\\\/1tyOFK0gTe6bxKPunQ8Ykm7CBPGtPWU8T3gwm7vfWRiYYFkzEf1EouCErg+ifBtVBUGg5YRHXDy+uGH3NgPGCJTW3F1C2TnRJrNl8iFGyYflC6ebjBlp4IuofpPvd\\\\\\\/xNprjjAXyvYCz63hqBlRz\\\\\\\/tIyBcqs5QV8vTPtciqAPV3EbbMynytrqtXWQEDYMnw+MGV2UwX7swj9Wg72EHlWAtmYr4dJcY\\\\\\\/5th1z3eKgnYxeesiamr7pmqpcjFguWwZqQ9jp9zIGXv4oeCigP1aVROwou2KGeLYF2iA0yzS0+mFnudoPwWF6XJdztfWxXFuG+9AhmL0HW325zRL2Xiqt34dR4VW9GLo9MGg65fWejuMzqyj8Pks2edNQsis+mbsgnC9y27cBQI8dVgQvbBuhB6KRIMK6S8rm5gmOW0qNVnLkEZ7zM3FYkNGG+W2tbj\\\\\\\/aNeeltqScXwFklNQ6+ofVPD21J3U5DYy7v96Paw5J0khmDVNbv7CA3EEKRuansQSbwf5k\\\\\\\/51Rapux\\\\\\\/49UQIDz8U8G5Ae9B1vvOtZ7Hs3Mprytr\\\\\\\/HeE8bMzzlNL4Ye6Iz0MoExllWZDnEOZWeUOOr5BNkIB8JrMjDWGJKLod+Ys0JMH5gz79Q4rZ2l3qxnpAHrm0CdNAHRFz4cZiEZcuroN6VPMBl6lDOeVZ5laadiH2c1RoMMJMAbqKw9il13dqPX9Mp9ks15ftz8QNBMiHMbZ2S3Z7JMT9HipIsdam0syosbCG\\\\\\\/4qcg5rtpRaBaJb6fX4U0umMoP7zWHiBl9JqlojYQyvtb307Bi6moyXfA86wP2qsRfP7YyazjL+G+Q9dg5\\\\\\\/z5O8Oi3iwoIZ0PsqjexHUMNHMlQMhL\\\\\\\/IvUS0Gj2KEldgP2rzqTddBmPeSbqjrlzBgsrk0XLXWiQZDfNVqK9O5SE3zKCQAjuVQXCIFB6YzQtaWvEeF0lzRIajv7qgEbIo0MSY0UA1GxzettRV2JxHLSa4fp6Fub1zLG0nWsiqCXvrX9VH8RvTpmxaqccXDdEbBfi\\\\\\\/heceA1w7ir70HqQU8XQVgz3mK3QFfWfQathpXDveKgOpDvo91zGVwQqS5bCG8mxgVvtefZ9IJtUawWVs06vrQoi8MKS6dKtpdPBRHYDgJMprS\\\\\\\/RLVxb11x8DZVyq0jYHrYSHCq1IMRQws3aCf5EcpCJhB9QMQaQO\\\\\\\/6ZZnPWxmkWTiXrQpV5lTI+IFAA0xkw3iD41jOFwnlpnReMEwy7DhjirPM4KPuDuR\\\\\\\/Z59TMKffo9CL9aysbVrJAcyKpo7s8GxasusEXQpZ0czMvDsj0wfqklbpgtuUqmFKd6OVrR6hZiGIm6ztq+VPDIkuQf1JWfeCaDu9X4HN9eAaB251hBpGCGZEMKqerOIicxlZcSn1G\\\\\\\/\\\\\\\/tvvPlK9WoUS6pCRi2C3WNbYQC97isjdIvZDbsEHS\\\\\\\/8QpTUqHoUjJiINCxiqPnEWpgEV9jwgxCuT2qosJNupHd8DCWZzpT013sbI2K4AjgsYuurmsLxqkupsSbIRVqQDikXdgplPTAvc1eZ2UWbNxxT+8UnM96HyWgLPuozH\\\\\\\/Ri8VSmZ+ZV6\\\\\\\/Fe5z+IZTWp0MG3Dca15uyfQ2h03ACnevElOpgOnV5Bi8Ji1eH6SAgy0W7Yc5CDOKsCdvAUn6zVi8fxLNssx5S2gdf5WDKHHGFBPVcX2icIwObr1IyWPvyMPKdBAkR+KEmm\\\\\\\/ypkbyTbGjvemPcLN8uZvE2BUWExkwCHcewA7nVOpgp7MS5Q9Faw200goFnCLHQhbvcH+WIHh07sF+OIHKd\\\\\\\/ymqV+OCsqyw+HLJnbG0I39FXdprPX6gAb1QSPT7IAsutZo0uhmHD7ENEbDFH77vTRe7koGrQuviqB0nFZiCqsRzUACnYLlkNFz0YDVhsL63i9QF0cJWDG7r+v\\\\\\\/0JaozNs0HNa7A4Trxvq06tca9w15zaX3CwBvXcz7zqGIT+nsHbkYM0slS17hc8BzzeWX6a9xQHkeD1ex7pmqR1ZtLaa2q7dqtWMpqTQgWi+O7dgxVprj+C46HpOhkigCRT45cRPY7qzAaLXTHiHlXm00aOudw6\\\\\\\/7sRT5qbMvMZvieUvK\\\\\\\/WCEH\\\\\\\/QkU+\\\\\\\/\\\\\\\/6QbrittluBmUoeX+keDjcdeIo3Ob9isowaUKpFHYh7jCzQPmN\\\\\\\/Ict3Zow+HT6FfTKcVYoMbIsHpol9sI1yzU5wtaIc5ZbXyFRi1\\\\\\\/Z8DBBhHRf6qLKU5UMBxUt5skR80eynkzSbmtCVd\\\\\\\/yVk6eHdHpIEZIJrRXgwaczzgOtoSd3bX5IJmipQNbttIemvj+OkHKpKrYX42omxohZbeJRANUHx7iZeRUvKhgQa0R6KOyk4Jje079a77YUfnJv5Zjq8wbobfP1sJjqAGH7+knFi2eNtN8NXo\\\\\\\/oXfJ7wv9LuL7V5sODs6XaJzY3MVnYziCSAvRPuKiPlcWNQV4kdfmYoA5Gg50PW8MdSuZl5Ckmlru9pKEI3Z8\\\\\\\/6X0G0DAKAArZZ9HIS17gBGbq34JmTWqkUe4bc4ENNsQ+\\\\\\\/r\\\\\\\/s\\\\\\\/oKiK1vjWaqhFbMFX9bT2WRSNo98NBq2K4LZctH2EnWVfNo\\\\\\\/sdbbAPnxcG+6KBx53ZKIDQcoU2BSWgExy1rnibm9eMzIIkIvgq7hzOgAcj+29UmR7EGjRdXePlL6MJLvvH\\\\\\\/afkdxZkzlfjdKZiZot3RSOcbJEmxYNkWCVCM+soa56ItaIpuOEUpaJ62apXKIJqfoTy3pmAOAoc5EhF80taQaPM2bcF7k9qt\\\\\\\/q5UXh8YxJLIBarXZi1fFzpSoiqEwVpP8UvWqDCp\\\\\\\/HPDbxDtPk+7U1Qh1xN92BR8DaDZPfUGNODIUxYjQ7z87Hj\\\\\\\/7m7AVVJDBxiEe8jEbA8KCNxkGUEWV4TrMiKMiabMf7L2THKKVldwGVO4tSJpwKbYDNVMeDvu268o14Tqs8Vg6guW1gEWAU1uyxbptsCbs9iipDoi2kP4nW2U4ky2zUy1GDUlCIOezoT1Sl4ilkUhvWdEDzWCUXl4acwAjR+6e1bQDkC0\\\\\\\/dW+k9IfF+uAupke05BbLexwTa6vuAs+KtqwH5iPptxe5QDtCDsoXCQ+VyK2gCoNWZg2BDqm1y\\\\\\\/\\\\\\\/vswihD4pcSJHbpLVUEvc660l88K8ESGd+Ji8XhiMCHArYToMdPD4sP1eBunIEOvLUNHsqCJosQtooTBdVzrsLNiHb81e55qDuF8vNBPVu4PxQWcnO6qVotMnPezXQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022o3JODkfMdw5kH1DMln6tsg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b3a17647f4f2c964103db577f0c61258e1ee7c86837fbc74d539fdd685e6afaf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/t0ao9GKS\\\\\\\/RcdRhTDOlayMF8hczOvFaJLYRmYAdRKuM9fcjDcQdlpq3wHkHaUiC0+zFUY8mRmZTuwQ8AISSqQuniibhO\\\\\\\/qy+Rh+Iv8WmUwCZ8S1ZppzD6fCinJZCwQ\\\\\\\/lGtnSMvt9poTB\\\\\\\/71B9oSnxS6lFEiyi9wgR0VhTAnL42o\\\\\\\/WeRBUHkurTRPR6Edwvub5jxVNgaRGd9F32nJjAAsTLNDgAk\\\\\\\/EEglAzHGmRgz9peBq7loTwTcmgaYaS6ODTiGiWc0vd1A11JVOGbgiC0SG4xnJRDZ54wimZ0bR7moubhEYgvTe+f3bxAzDz7Jn3MNjySpgDCt44l5H6Gj165LpV3s\\\\\\\/fxN9vhPZ82mf848P7cYcAqtUny8CTgfDE5kHGQIe7oQLxNdtrhNrpqpX0APyk54iRkScLw1zp1ISQpiozds3zxi9PSJCLrWtO0vOUsQd0KTNtvsGWr938OayV87Me6HB9QJ8LMtvzsTl5i+uER7WT8YHrlMQEUg\\\\\\\/Tbco7IzJ7r\\\\\\\/jJQC4T4KJ51g9u+Zb6mc7ClhMOhqXhom0yeOyElpYTpHhxRAXmeTLKTPtoXLZkWUEOAQNQ7Gbimq0a5aWOhPnOEOR4KLzrdcj\\\\\\\/PyLwlJDNUBDBkO7y+o4XNpMOanjQvumDTLZ8H7ORS13fPGrgfbXKvq7TysMHbIlGfE4RKounfp8PKZfutVFqy3BcQAC2\\\\\\\/Vxs6ODvvwzDHKIs80pWTVVDkEnF1Tn\\\\\\\/6XE1cJuLiCAZOtklbtWi9hGTT+U23Z+Xkwc6r0wqxJcLiTfwDdi3zqveRLaZPK7P44GiW\\\\\\\/N8gku+O\\\\\\\/vvGQ\\\\\\\/9ycJX3bZsU8zAnqutcgaEbl7bHA85wgd49gjUpXl685x3+yKaNVf02Gv+HFecs7DhYNsNaYEw5iBwcvZi0blFqk2QFZgb8h3J2B+NIidD5NM25AuRb4qxMoGFu0a36TZaNQMbE28cAfybihn0XVtAg06l9Gf8L+CMs\\\\\\\/068U67TtKuB6DA6tffue5+ZqC1PAXTtZL3g0x+TTQk7BrCg1gajujC\\\\\\\/fam8TgXJsoaIWpva8F97Rz7qn5Ip05cDQxvKfC4K7FosyAJBz2whSo6vGDqiksVY3l6L5\\\\\\\/LFyjzSMovPicxO7icK7SH4LHUIueJ2MyGVYC17aWqzxLNNzoL+QpKbYvMLYbUIwnPn1dzLNTsPSYr8251eL9rNEqzzZVwcuomtEqlmerBOtTvq5leuSHN4m\\\\\\\/pqV+AQoNZcNIMH1OBXQ2zoQVEFuBzrkEHJ5mUdhXMpOg76g95tMu1lufwH44uCj4BNjXer1WIm3EqLw+g8UH64k7xSMVyAwwdYm4Iw\\\\\\\/8t+BHb6V3M\\\\\\\/vTQ82z\\\\\\\/U82PCkjdFUjrtYR7KBwwruscF10xWmBuRVaC4scbkRDigVG8diEerZ8LhG69K6d1p5lYskD9CPVCO9RiV4spupnI5P5akrxJga2\\\\\\\/ED9OLAGoYIVHdfRohZVGVpyWsqEFlxS8nfewEi1HW7nKiPPnGS7EMcsxb\\\\\\\/cLu8w0cSeeQAvC15mFatqxm4X7ZSNWDH26EPZtiL4vtshgnLyyRwqR2o6I7DBNOm0MKA3oNIO3TCnc0AyMZWpncyCYSCb78CgvF\\\\\\\/QmKW5\\\\\\\/5VybYYWwxIXMQGU7ThAEBndIAc\\\\\\\/zlqQ5h6TleapfaLWTyfkTOzgwLqMdXVcC47I+ZfLnaFOn4MkvUkYFzBvnQ4K2wuohSHZPW\\\\\\\/\\\\\\\/mu0fBBr8U8EjbQaQj+HXbMmN57ijLqWpGR8+lh5NzpXkAQebgdLm9GZqRYfHK5Bj0PhfDH3z+\\\\\\\/GpgLP7B5uFxQqHBRdIf4XC5Fu5LKqdNQ4zfcD+HUklKp+qgdmLw+inHB7ama0wFFwzTpkC8+fWopT3Z+xdqAH8nkjTBNCP8NqH2XiqmQAxBpd2M5QIMbmugkDTJOzZ0FohWE9xKj6IWEcCtOPzuLRvxXKpAn7wsMkka6ULskMrKfSII1hBKuphtJA6fIJXm8Y5AxDFCxaZ5tWesLpCZSj0olKaQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ISRZns0tYhRGqpN4Z5AttA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002271b4d5fda9e33ff2d6691774af569521de2a033be3a968107c4688429a4b8936\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495165 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/109" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-icQqjGDWrPKn35RKKxjRvGHxh4jyAD4RMkw4dPNvkac" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/109" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U_p_2gb-ni7YtRWeci-dhVyfkrpsOWAhSLqxyzFhsh4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/89" title="Which Recurrent Selection Scheme To Improve Mixtures of Crop Species? Theoretical Expectations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/117" title="A Functional Analysis of the Drosophila Gene hindsight: Evidence for Positive Regulation of EGFR Signaling" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495165" id="node495165" data-pisa="ggg;10/1/109" data-pisa-master="ggg;g3.119.400812" data-apath="/ggg/10/1/109.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Influence of Genetic Interactions on Polygenic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhijun Dai</span>, <span class="highwire-citation-author" data-delta="1">Nanye Long</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6788-8364" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wen Huang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">109-115;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400812 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495165" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhijun Dai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhijun%2BDai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhijun%2BDai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nanye Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Cyber-Enabled Research, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nanye%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANanye%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wen Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wen%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWen%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6788-8364" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wen Huang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:huangw53@msu.edu" class="" data-icon-position="" data-hide-link-title="0">huangw53@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/109" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495165" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/109.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495165" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/109.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495165" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/109.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495165" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495165/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Prediction of phenotypes from genotypes is an important objective to fulfill the promises of genomics, precision medicine and agriculture. Although it’s now possible to account for the majority of genetic variation through model fitting, prediction of phenotypes remains a challenge, especially across populations that have diverged in the past. In this study, we designed simulation experiments to specifically investigate the role of genetic interactions in failure of polygenic prediction. We found that non-additive genetic interactions can significantly reduce the accuracy of polygenic prediction. Our study demonstrated the importance of considering genetic interactions in genetic prediction.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genetic-interactions" class="hw-term hw-article-keyword hw-article-keyword-genetic-interactions" rel="nofollow">genetic interactions</a></li><li class="kwd"><a href="/keyword/polygenic-prediction" class="hw-term hw-article-keyword hw-article-keyword-polygenic-prediction" rel="nofollow">polygenic prediction</a></li><li class="kwd"><a href="/keyword/cross-population-prediction" class="hw-term hw-article-keyword hw-article-keyword-cross-population-prediction" rel="nofollow">cross-population prediction</a></li></ul><p id="p-4">Significant progress has been made in our understanding of the genetic architecture of complex quantitative traits in recent years, due largely to large-scale genome-wide association studies (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Visscher <em>et al.</em> 2017</a>). For example, human adult height is a classical quantitative trait with a narrow sense heritability (<em>h<sup>2</sup></em>) of approximately 0.8 based on twin studies (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Silventoinen <em>et al.</em> 2003</a>). However, early GWAS studies identified common variants explaining only a total of 2–4% phenotypic variance (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gudbjartsson <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lettre <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Weedon <em>et al.</em> 2008</a>) with sample sizes in the order of 20,000. In 2010, a landmark study increased this proportion to about 45% by fitting ∼300,000 SNP markers regardless of their significance in the model for ∼4,000 individuals with the covariance among individuals determined by genome-wide SNP similarity (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yang <em>et al.</em> 2010</a>). Applying the same idea, the most recent study using whole genome sequences of ∼20,000 individuals in the TOPMed project almost entirely captured all heritability (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wainschtein <em>et al.</em> 2019</a>). These studies suggested that complex traits are highly polygenic, with many loci of individually small effects.</p><p id="p-5">However, our ability to predict complex quantitative traits from genotype data remains limited. A perfect genetic model with precise effects and model specification should be able to predict unobserved phenotypes with an accuracy (measured by <em>r<sup>2</sup></em>) equal to the heritability. However, this is rarely the case. For example, a large GWAS on human adult height with almost 200,000 individuals identified over 180 loci, which could only predict phenotypes with an accuracy of ∼10% (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lango Allen <em>et al.</em> 2010</a>). This prediction accuracy was measured based on “leave-one-out” out-of-sample prediction (International Schizophrenia Consortium <em>et al.</em> 2009), <em>i.e.</em>, the effects of the genetic loci were estimated in one subset of the sample and polygenic scores (genetic effects summed over all significant loci) was computed to predict phenotypes in another subset. The partition between the subsets conveniently followed sample origin from different European countries (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Lango Allen <em>et al.</em> 2010</a>). In animal and plant breeding, genomic prediction is widely used, where effects of genetic markers across the whole genome, regardless of their statistical significance, are summed to compute genetic prediction (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">VanRaden 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">de los Campos <em>et al.</em> 2013</a>).</p><p id="p-6">Recently, there has been renewed interest in the application of polygenic score (International Schizophrenia Consortium <em>et al.</em> 2009) with the advent of large public data sets such as the UK Biobank (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Khera <em>et al.</em> 2018</a>). In particular, many studies have observed poor prediction by polygenic scores across different ancestry groups (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Martin <em>et al.</em> 2019</a>) or even within an ancestry group but with variable characteristics (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mostafavi <em>et al.</em> 2019</a>). In fact, earlier studies with smaller sample sizes observed similar patterns, but were interpreted as missing heritability (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Lango Allen <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Makowsky <em>et al.</em> 2011</a>). In animal breeding, similar observations have also been made. Although genomic prediction works exceedingly well within a breed of cattle, cross-breed prediction generally fails (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hayes <em>et al.</em> 2009</a>). The explanation is obvious, genetic effects can be context dependent and heterogeneous between groups. Variable linkage disequilibrium (LD) patterns, environments, and other factors can all contribute to the variable genetic effects, manifesting as variable accuracy of polygenic prediction.</p><p id="p-7">Genetic interactions are pervasive, and an important type of context dependent effects (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mackay 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mackay and Moore 2014</a>). The presence of genetic interactions does not have a strong effect on the proportion of phenotypic variance attributable to the additive effects of all markers (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Huang and Mackay 2016</a>), therefore the magnitude of additive variance explained by all markers offers no indication of the genetic architecture. However, genetic interactions may influence genomic prediction accuracy. Models explicitly taking into account the complexity can improve prediction (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ober <em>et al.</em> 2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jiang and Reif 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Martini <em>et al.</em> 2017</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Morgante <em>et al.</em> 2018</a>). Moreover, non-parametric models that do not rely on the additivity of the model can outperform parametric additive models when the genetic architecture is non-additive (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Momen <em>et al.</em> 2018</a>). These results clearly suggest that the simplification of genetic architecture to the additive infinitesimal model when the true model is not, although convenient and no comparable alternatives exist, can be risky. In this study, we specifically investigate the influence of genetic interactions on polygenic prediction of phenotypes, with an emphasis on prediction across diverged populations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Population simulation</h3><p id="p-8">We used the coalescent simulator MaCS (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chen <em>et al.</em> 2008</a>) to simulate genome sequences of 75,000 individuals, with 25,000 in each of the populations, according to the demographic history in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. We simulated 1,000 independently inherited chromosomes of 100,000 base pairs in size and set mutation rate as 1.25 × 10<sup>−8</sup> per bp and recombination as 1.25 × 10<sup>−8</sup> per bp (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Effective population size was set to 20,000. The MaCS command for one chromosome was “macs 150000 100000 -s “$random_seed” -i 1 -h 1000 -t 0.001 -r 0.001 -I 3 50000 50000 50000 0 -ej 0.0125 3 2 -ej 0.025 2 1”. This simulation was performed once but the partitions between samples were repeated 20 time, which were summarized as box plots in figures. We defined three sets of variants. 1) causal variants: one variant was sampled from each of the 1,000 chromosomes to constitute the causal variants. 2) tag variants: all variants excluding the causal variants. 3) all variants: all variants including the causal variants.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/109/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation of genome sequences, population structure, and genetic architecture. (A) Three populations (P1, P2, P3) were simulated with an effective population size of 20,000 each. P1 and P2 diverged 1,000 generations before present and P1 and P3 diverged 2,000 generations ago. (B) 1,000 independently inherited chromosomes were simulated, each containing one QTL. Three sets of variants were considered, including &#x201C;causal&#x201D;, &#x201C;tag&#x201D;, and &#x201C;all&#x201D; as illustrated. (C) Six different genetic architecture were simulated, each illustrated by one of the panels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-617998863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation of genome sequences, population structure, and genetic architecture. (A) Three populations (P1, P2, P3) were simulated with an effective population size of 20,000 each. P1 and P2 diverged 1,000 generations before present and P1 and P3 diverged 2,000 generations ago. (B) 1,000 independently inherited chromosomes were simulated, each containing one QTL. Three sets of variants were considered, including &#x201C;causal&#x201D;, &#x201C;tag&#x201D;, and &#x201C;all&#x201D; as illustrated. (C) Six different genetic architecture were simulated, each illustrated by one of the panels.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/109/F1.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/109/F1.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/109/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/109/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495204" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Simulation of genome sequences, population structure, and genetic architecture. (A) Three populations (P1, P2, P3) were simulated with an effective population size of 20,000 each. P1 and P2 diverged 1,000 generations before present and P1 and P3 diverged 2,000 generations ago. (B) 1,000 independently inherited chromosomes were simulated, each containing one QTL. Three sets of variants were considered, including “causal”, “tag”, and “all” as illustrated. (C) Six different genetic architecture were simulated, each illustrated by one of the panels.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Simulation of quantitative phenotypes</h3><p id="p-10">We simulated quantitative phenotypes according to the genetic architecture depicted in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>. For each of the three possible genotypes for a biallelic locus with alleles A and a, we used the additive coding aa = -1, Aa = 0, and AA = 1 and the dominance coding aa = 0, Aa = 1, AA = 0 to code genotypes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The simulation of phenotypes consisted of two steps. In the first step, the corresponding genotype coding for an individual or product of genotype codings (in the case of between-loci interactions) were multiplied by a genetic effect randomly drawn from the standard normal distribution and summed over all loci or all pairs of loci to obtain the genetic values. In the second step, an environmental effect was added by drawing from a normal distribution with a computed variance such that the broad sense heritability <em>H<sup>2</sup></em> = 0.8. These steps are summarized in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> with illustrative examples for nine possible genotypes across two loci (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). It’s straightforward to extend this to all loci. We performed this simulation in each of the 20 random partitions of populations and independently sampled causal variants and genetic effects.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495346/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495346/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of genotype-phenotype relationships in simulations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Fitting GREML</h3><p id="p-11">We fitted the GREML model using GCTA (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2011</a>) with 20,000 individuals from each of the P1, P2, and P3 populations and P1 + P2 and P1 + P3. The GREML partitioned phenotypic variance into a genomic (<em>σ<sup>2</sup><sub>g</sub></em>) and an environmental component (<em>σ<sup>2</sup><sub>e</sub></em>) by fitting a mixed model using REML with covariance matrix determined by a relationship matrix calculated based on standardized genotypes (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Yang <em>et al.</em> 2010</a>). Genomic heritability was computed as <em>h<sup>2</sup><sub>g</sub></em> = <em>σ<sup>2</sup><sub>g</sub></em>/(<em>σ<sup>2</sup><sub>g</sub></em> + <em>σ<sup>2</sup><sub>e</sub></em>).</p></div><div id="sec-5" class="subsection"><h3>Polygenic score prediction</h3><p id="p-12">The BLUP estimates of SNP effects were obtained using GCTA and provided to PLINK2 (<a href="https://www.cog-genomics.org/plink/2.0/credits">https://www.cog-genomics.org/plink/2.0/credits</a>) to compute a polygenic score in 5,000 new individuals either from the same population as the fitted model or from a different population. Prediction accuracy of polygenic score was computed as the <em>r<sup>2</sup></em> of correlating predicted polygenic scores and the simulated true phenotypes. In the case of prediction using causal variants with the correct dominance by dominance model (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), we constructed pseudo-variants using the relevant genotype coding (for D x D, double heterozygotes were coded as one genotype class and all others the other) and ran GREML and polygenic score prediction the same way as an additive model.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-13">All procedures to simulate the data are described in the manuscript and codes can be found at <a href="https://github.com/qgg-lab/epistasis-prediction">https://github.com/qgg-lab/epistasis-prediction</a>. We provide the simulated genotype data for all 75,000 individuals (25,000 per population) in PLINK binary format (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Purcell <em>et al.</em> 2007</a>) on figshare (<a href="https://figshare.com/projects/Influence_of_genetic_interactions_on_polygenic_prediction/70427">https://figshare.com/projects/Influence_of_genetic_interactions_on_polygenic_prediction/70427</a>). There are a large number of random partitions for the replicates and the associated phenotypes, these are not directly provided but they are easy to recapitulate with the description of methods and the computer codes. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10031807">https://doi.org/10.25387/g3.10031807</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Experimental design</h3><p id="p-14">Because it’s not yet possible to unambiguously know the true genetic architecture of a quantitative trait, all experiments in this study were performed using simulated data instead of real data. This allows us to specifically ask simple questions while eliminating influence from other factors. We simulated a sample of 75,000 diploid individuals from three ancestry groups, where population P1 and P2 diverged 1,000 generations ago and their ancestors diverged from population P3 an additional 1,000 generations ago (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). This specification is qualitatively similar to the global human population history where the ancestral population that went out of Africa were further split into multiple populations.</p><p id="p-15">We considered three possible variant sets (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>); 1) causal: all and only causal variants; 2) tag: all variants except causal variants; and 3) all: all variants including causal variants. These represent three simplified scenarios 1) a best case scenario where causal variants have been identified, 2) a realistic scenario where causal variants are tagged by genotyped variants, and 3) an achievable scenario in the near future with whole genome sequences. We did not consider variants that were rare (MAF &lt; 0.01) in all three populations as they led to gross overestimation of genomic heritability approaching one, similar to findings in a simulation study using real genotypes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Evans <em>et al.</em> 2018</a>). The three variant sets were used to compute genomic heritability and perform polygenic prediction. There were a total of approximately 680,000 variants in the ‘all’ variants case. When performing polygenic prediction, we did not select variants based on association tests. This choice was based on the consideration that selection of markers introduced another variable in the experiment to complicate the design and interpretation. Instead, we draw from the distinction between causal and all variants to represent the extreme scenarios where a perfect selection or no selection was performed.</p><p id="p-16">We simulated a quantitative trait controlled by 1,000 independently inherited QTL (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>) of broad sense heritability <em>H<sup>2</sup></em> = 0.8 but different types of genetic architecture. When the genetic architecture is strictly additive, the narrow sense heritability <em>h<sup>2</sup></em> = <em>H<sup>2</sup></em> = 0.8, whereas in other cases <em>h<sup>2</sup></em> &lt; 0.8. Six simple models of genetic architecture were simulated, including additive, dominance, overdominance, and pairwise additive by additive (A x A), additive by dominance (A x D), and dominance by dominance (D x D) (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). No higher order interaction was simulated and effects across loci or across pairs were additive.</p></div><div id="sec-9" class="subsection"><h3>Genomic heritability misses little heritability</h3><p id="p-17">We first recapitulated a result that has been consistently shown (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Huang and Mackay 2016</a>). We fitted a linear mixed model in each of the three populations or combined samples using GREML implemented in the GCTA (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2011</a>) with 20,000 individuals. We found that <em>h<sub>g</sub><sup>2</sup></em> were uniformly high when the genetic architecture was additive, dominance, or additive by additive, accounting for nearly all heritability (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Figure S1 online). Whether or not the variant sets included casual variants appeared to have little effects on <em>h<sub>g</sub><sup>2</sup></em>; variant sets excluding causal variants performed as well as causal variants only and there was a slight tendency of upward bias (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Similar results were obtained regardless of whether the samples were from a homogeneous population or a mixture of samples from two diverged populations (Figure S1). When the genetic architecture was entirely overdominance, additive by dominance, or dominance by dominance, <em>h<sub>g</sub><sup>2</sup></em> was lower, but still consistently explained &gt; 50% of the heritability (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, Figure S1). Taken together, these results suggest that as long as a large number of genome-wide markers were fitted, little heritability was missed, regardless of the genetic architecture. In other words, the magnitude of genomic heritability offers no discrimination of the underlying genetic architecture (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Huang and Mackay 2016</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/109/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic heritability in the simulated populations. Box plot (median indicated on top) showing the genomic heritability (hg2) estimated using GREML under different genetic architecture, where Add. = additive, Dom. = dominance, Overdom. = overdominance, A x A = additive by additive, A x D = additive by dominance, D x D = dominance by dominance, and random is a non-genetic model where the phenotypic variation was entirely due to random environmental variation. The population in which the genomic heritability was estimated was indicated in the top right corner. Genomic heritabilities in all other populations were given in Figure S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-617998863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic heritability in the simulated populations. Box plot (median indicated on top) showing the genomic heritability (&lt;em&gt;h&lt;sub&gt;g&lt;/sub&gt;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) estimated using GREML under different genetic architecture, where Add. = additive, Dom. = dominance, Overdom. = overdominance, A x A = additive by additive, A x D = additive by dominance, D x D = dominance by dominance, and random is a non-genetic model where the phenotypic variation was entirely due to random environmental variation. The population in which the genomic heritability was estimated was indicated in the top right corner. Genomic heritabilities in all other populations were given in Figure S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/109/F2.medium.gif" width="440" height="134"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/109/F2.medium.gif" width="440" height="134"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/109/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/109/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495336" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Genomic heritability in the simulated populations. Box plot (median indicated on top) showing the genomic heritability (<em>h<sub>g</sub><sup>2</sup></em>) estimated using GREML under different genetic architecture, where Add. = additive, Dom. = dominance, Overdom. = overdominance, A x A = additive by additive, A x D = additive by dominance, D x D = dominance by dominance, and random is a non-genetic model where the phenotypic variation was entirely due to random environmental variation. The population in which the genomic heritability was estimated was indicated in the top right corner. Genomic heritabilities in all other populations were given in Figure S1.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Accuracy of polygenic prediction with an additive genetic architecture</h3><p id="p-19">We then asked a simple question. If genome-wide variants are able to capture the majority of heritability, are they able to predict phenotypes accurately? This question directly addresses the distinction between the two definitions of missing heritability. If there is no missing heritability based on mixed model fitting, is there missing heritability in polygenic prediction? Many illuminating results could be obtained by comparing different scenarios of simulations (Figure S2).</p><p id="p-20">We first considered the simplest and best scenario, in which the genetic architecture was fully additive, and all and only causal variants were known. In this case, the statistical model took the form of the true model and only model parameters needed to be estimated. We trained the model in one population (n = 20,000, training data) and computed polygenic scores of new individuals (n = 5,000, test data) either in the same population or a different population (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). To test the performance of cross-population prediction, we considered three possible relationships between the training and test populations, representing a gradient of divergence between training and test data (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/109/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polygenic prediction under additive genetic architecture. (A) Polygenic prediction was performed according to the diagram, where the model was trained in population P1 and tested in populations P1, P2, and P3 at increasing divergence. (B) Prediction accuracy was plotted according to the training &#x2013; test population relationships. For comparison, genomic heritability was also plotted along side. Only the additive genetic architecture was considered in this plot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-617998863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Polygenic prediction under additive genetic architecture. (A) Polygenic prediction was performed according to the diagram, where the model was trained in population P1 and tested in populations P1, P2, and P3 at increasing divergence. (B) Prediction accuracy was plotted according to the training &#x2013; test population relationships. For comparison, genomic heritability was also plotted along side. Only the additive genetic architecture was considered in this plot.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/109/F3.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/109/F3.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/109/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/109/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495423" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Polygenic prediction under additive genetic architecture. (A) Polygenic prediction was performed according to the diagram, where the model was trained in population P1 and tested in populations P1, P2, and P3 at increasing divergence. (B) Prediction accuracy was plotted according to the training – test population relationships. For comparison, genomic heritability was also plotted along side. Only the additive genetic architecture was considered in this plot.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">As expected, the accuracy of polygenic prediction was very high in this best case scenario, approaching the true heritability (‘causal’ in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). There was a small decline in accuracy when cross-population prediction was performed and the degree of population divergence negatively affected prediction accuracy. However, when non-causal variants were included to make predictions, accuracy plummeted from ∼0.8 to ∼0.4 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>) even when training and test samples were from the same population. This was likely due to the inclusion of independent predictors whose number vastly exceeded that of the causal variants. As populations become more divergent, prediction accuracy further dropped, the rate of which was much more pronounced when tag or all variants were used. These results (in the cases of tag or all variant sets) largely agreed with the large body of empirical work that accuracy of polygenic prediction was substantially lower than genomic heritability and cross-population prediction was poor (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Lango Allen <em>et al.</em> 2010</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Makowsky <em>et al.</em> 2011</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Martin <em>et al.</em> 2019</a>).</p><p id="p-23">One important lesson could be learned in this simple experiment. The facts that simply adding non-causal variants to the model drastically reduced prediction accuracy, and that the rate of decay in the accuracy of cross-population prediction was much greater in the presence of non-causal variants indicated that the agreement between model and true genetic architecture mattered. This is in sharp contrast to genomic heritability estimation, where including more variants generally improves model fit (compare ref (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Yang <em>et al.</em> 2010</a>) with ref (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wainschtein <em>et al.</em> 2019</a>)).</p></div><div id="sec-11" class="subsection"><h3>Accuracy of polygenic prediction in the presence of genetic interactions</h3><p id="p-24">We then tested the influence of genetic interactions on the accuracy of polygenic prediction, which fits an additive model. In a favorable condition when all causal variants were known (but not their effects or interactions) and prediction was performed within the same homogenous population, polygenic prediction accuracy was highly dependent on the genetic architecture (P1 -&gt; P1 in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The accuracy ranged from 0.78, nearly the theoretical maximum in the case of an additive genetic architecture to less than 0.20 in the case of a dominance by dominance genetic architecture (P1 -&gt; P1 in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). In general, prediction accuracy was higher for genetic architecture with higher <em>h<sub>g</sub><sup>2</sup></em>, such as additive, dominance, and additive by additive. In contrast, under overdominance, additive by dominance, and dominance by dominance genetic architecture, polygenic prediction performed substantially worse (P1 -&gt; P1 in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). When all variants were used, including non-causal ones, the prediction accuracies decreased dramatically, from 0.78 to 0.37 in the most favorable within-population additive case (additive case in P1 -&gt; P1 in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a> and <a id="xref-fig-4-5" class="xref-fig" href="#F4">4B</a>). Furthermore, the dependency on genetic architecture appeared to be stronger when non-causal variants were included (P1 -&gt; P1 in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/109/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polygenic prediction with different genetic architecture. (A) Polygenic prediction was performed using causal variants only for six different genetic architecture. The median prediction accuracy (r2) across 20 replicates in each scenario was listed below the graph, as well as genomic heritability (hg2). Each point on the graph represents a normalized median r2, dividing each prediction accuracy by its counterpart in the within population (P1 -&gt; P1) prediction. (B) Polygenic prediction with all variants. Data are presented the same way as in (A). Data in these graphs were summarized from Figure S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-617998863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polygenic prediction with different genetic architecture. (A) Polygenic prediction was performed using causal variants only for six different genetic architecture. The median prediction accuracy (&lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) across 20 replicates in each scenario was listed below the graph, as well as genomic heritability (&lt;em&gt;h&lt;sub&gt;g&lt;/sub&gt;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;). Each point on the graph represents a normalized median &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;, dividing each prediction accuracy by its counterpart in the within population (P1 -&gt; P1) prediction. (B) Polygenic prediction with all variants. Data are presented the same way as in (A). Data in these graphs were summarized from Figure S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/109/F4.medium.gif" width="440" height="233"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/109/F4.medium.gif" width="440" height="233"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/109/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/109/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495183" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Polygenic prediction with different genetic architecture. (A) Polygenic prediction was performed using causal variants only for six different genetic architecture. The median prediction accuracy (<em>r<sup>2</sup></em>) across 20 replicates in each scenario was listed below the graph, as well as genomic heritability (<em>h<sub>g</sub><sup>2</sup></em>). Each point on the graph represents a normalized median <em>r<sup>2</sup></em>, dividing each prediction accuracy by its counterpart in the within population (P1 -&gt; P1) prediction. (B) Polygenic prediction with all variants. Data are presented the same way as in (A). Data in these graphs were summarized from Figure S2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We then asked how genetic interactions influence the rate of decay in prediction accuracies when the training and test populations diverge. We set the accuracy of within-population prediction as the baseline and compared cross-population prediction accuracies to this baseline. When all variants were used for polygenic prediction, the accuracy of cross-population prediction dropped to about 40–60% of the accuracy of within-population prediction, depending on genetic architecture (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>). Additive, additive by additive, and dominance genetic architecture, those with the highest <em>h<sub>g</sub><sup>2</sup></em> and <em>r<sup>2</sup></em>, retained the most prediction accuracy while overdominance, additive by dominance, and dominance by dominance lost the most (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4B</a>). The more diverged the populations were, the more predictive ability of polygenic scores was lost (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-27">There are many reasons why polygenic prediction failed when test population diverged from training population. In our simple simulation setting, genetic effects were the same across populations and were not sensitive to any non-genetic factors. The difference in the linkage disequilibrium structure between populations may in part explain the drop when all variants were used (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4B</a>). Importantly, simulations allowed us to directly use causal variants for prediction, thus eliminating the influence of LD (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4A</a>). Remarkably, while the accuracy of cross-population prediction was lower for all genetic architecture, the rate of decay was much greater when the genetic architecture was over-dominance, additive by dominance, or dominance by dominance (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4A</a>, compare slopes of the different lines). These results clearly suggest that genetic interactions can not only cause cross-population polygenic prediction to fail, but also in a more severe manner compared to an additive genetic architecture.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-28">We demonstrate in this study through simulations that genetic interactions can influence the accuracy of polygenic prediction. In particular, cross-population polygenic prediction performed worse than intra-population prediction in all cases. For traits controlled by genetic interactions, the cross-population decay in prediction accuracy was far greater (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4</a>). The results make intuitive sense. For a statistical model to predict new data accurately, two conditions must be met. First, the model specification must be correct or at least sufficiently accurate to capture variation in the data. Second, parameters in the model must be precise. When genetic interactions are present, the additive polygenic model clearly is not accurate.</p><p id="p-29">Previous studies have mostly focused on improving parameter estimation, through increasing sample size and methodological improvement. For example, increasing sample size substantially increased accuracy of polygenic prediction of height within individuals of European ancestry (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lello <em>et al.</em> 2018</a>). Inclusion of samples of different backgrounds in the training data also helped (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Martin <em>et al.</em> 2019</a>) (Figure S2).</p><p id="p-30">However, the complexity of the genetic architecture of a quantitative trait makes it nearly impossible to specify a model prior to modeling. As a consequence, the polygenic infinitesimal model or variants of it (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gianola <em>et al.</em> 2009</a>) has been used as the default model. The infinitesimal model has been instrumental and allowed for many theoretical insights as well as applications to be developed. In particular, prediction of breeding values in animal and plant breeding relying on the infinitesimal model has been very successful (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">García-Ruiz <em>et al.</em> 2016</a>). However, its limitations are also apparent. Cross-population and cross-breed polygenic prediction was low in accuracy (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Lango Allen <em>et al.</em> 2010</a>; <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Martin <em>et al.</em> 2019</a>). Although many factors may contribute to this limitation, our simulation results clearly indicated that genetic interactions unaccounted for was a major contributor. Indeed, if the correct genetic model could be specified, cross-population prediction can achieve very high accuracy (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). There have been attempts to explicitly model non-additive genetic effects in the context of polygenic prediction; some moderate improvement was observed (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Martini <em>et al.</em> 2017</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Varona <em>et al.</em> 2018</a>). However, these studies modeled non-additive effects using genome-wide markers, which added a large number of independent predictors as noise to the model and may negatively impact the performance.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/109/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Agreement between model and true genetic architecture improves polygenic prediction. Two genetic architectures were considered, the additive and D x D. The prediction was performed with either an additive model (Add. model) as implemented in GREML or a D x D model (indicated in text below the plot) in which the correct genetic model was presumed to be known and fitted. Only causal variants were used in these analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-617998863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Agreement between model and true genetic architecture improves polygenic prediction. Two genetic architectures were considered, the additive and D x D. The prediction was performed with either an additive model (Add. model) as implemented in GREML or a D x D model (indicated in text below the plot) in which the correct genetic model was presumed to be known and fitted. Only causal variants were used in these analyses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/109/F5.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/1/109/F5.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/109/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/109/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495232" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Agreement between model and true genetic architecture improves polygenic prediction. Two genetic architectures were considered, the additive and D x D. The prediction was performed with either an additive model (Add. model) as implemented in GREML or a D x D model (indicated in text below the plot) in which the correct genetic model was presumed to be known and fitted. Only causal variants were used in these analyses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We did not analyze existing public data sets with real genotypes and phenotypes, some of which contained subjects from multiple ancestries. Previous work with real data has consistently shown that cross-population polygenic prediction generally fails (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Martin <em>et al.</em> 2019</a>), which agreed with results obtained by simulations in this study. However, it is difficult to disentangle the different factors that may contribute to effect heterogeneity and the failure of prediction in real data sets. Using simulations, we can focus on specific questions and our results clearly indicated a contribution of genetic interactions to the failure of cross-population polygenic prediction. While the additive infinitesimal model is the most sensible model when no other information is available, our study suggests that the development in the field should be expanded to include efforts to more explicitly model genetic interactions. Although it is challenging, recent advances in modeling (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Boyle <em>et al.</em> 2017</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Liu <em>et al.</em> 2019</a>) and genomic assays informing regulatory networks (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gerstein <em>et al.</em> 2012</a>) may finally offer new ways to develop biologically sensible models.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">This research is supported by Michigan State University, MSU AgBioResearch, and National Institute of Food and Agriculture.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-34">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10031807">https://doi.org/10.25387/g3.10031807</a>.</p></li><li class="fn" id="fn-3"><p id="p-35"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-11"><span class="received-label">Received </span>July 11, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-23"><span class="accepted-label">Accepted </span>October 23, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Dai <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.1" data-doi="10.1016/j.cell.2017.05.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. I.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An Expanded View of Complex Traits: From Polygenic to Omnigenic.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2017.05.038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D169%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.05.038%26rft_id%253Dinfo%253Apmid%252F28622505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.05.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28622505&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.2" data-doi="10.1101/gr.083634.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marjoram</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Wall</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Fast and flexible simulation of DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.083634.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083634.108%26rft_id%253Dinfo%253Apmid%252F19029539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.3" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.4" data-doi="10.1038/s41588-018-0108-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tahmasbi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Vrieze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Abecasis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Das</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Comparison of methods that use whole genome data to estimate the heritability and genetic architecture of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0108-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D737%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0108-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0108-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.5" data-doi="10.1073/pnas.1519061113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">García-Ruiz</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Cole</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">VanRaden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Wiggans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Ruiz-López</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Changes in genetic selection differentials and generation intervals in US Holstein dairy cattle as a result of genomic selection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E3995</span>–<span class="cit-lpage">E4004</span>. Erratum: xxx.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1519061113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1519061113%26rft_id%253Dinfo%253Apmid%252F27354521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzI4L0UzOTk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.6" data-doi="10.1038/nature11245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>,  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kundaje</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hariharan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Landt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-K.</span> <span class="cit-name-surname">Yan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Architecture of the human regulatory network derived from ENCODE data.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGerstein%26rft.auinit1%253DM.%2BB.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D91%26rft.epage%253D100%26rft.atitle%253DArchitecture%2Bof%2Bthe%2Bhuman%2Bregulatory%2Bnetwork%2Bderived%2Bfrom%2BENCODE%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11245%26rft_id%253Dinfo%253Apmid%252F22955619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955619&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.7" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manfredi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive Genetic Variability and the Bayesian Alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.8" data-doi="10.1038/ng.122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>,  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. B.</span> <span class="cit-name-surname">Walters</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Thorleifsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Stefansson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. V.</span> <span class="cit-name-surname">Halldorsson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Many sequence variants affecting diversity of adult human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGudbjartsson%26rft.auinit1%253DD.%2BF.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D609%26rft.epage%253D615%26rft.atitle%253DMany%2Bsequence%2Bvariants%2Baffecting%2Bdiversity%2Bof%2Badult%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.122%26rft_id%253Dinfo%253Apmid%252F18391951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391951&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.9" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Chamberlain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Verbyla</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-51</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.10" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.11" data-doi="10.1371/journal.pgen.1006421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Genetic Architecture of Quantitative Traits Cannot Be Inferred from Variance Component Analysis.</span> <abbr class="cit-jnl-abbrev">PLOS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006421%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006421%26rft_id%253Dinfo%253Apmid%252F27812106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27812106&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.109.12" data-doi="10.1038/nature08185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Schizophrenia Consortium</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Purcell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Wray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Stone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Common polygenic variation contributes to risk of schizophrenia and bipolar disorder.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">460</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%2BM.%26rft.volume%253D460%26rft.issue%253D7256%26rft.spage%253D748%26rft.epage%253D752%26rft.atitle%253DCommon%2Bpolygenic%2Bvariation%2Bcontributes%2Bto%2Brisk%2Bof%2Bschizophrenia%2Band%2Bbipolar%2Bdisorder.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08185%26rft_id%253Dinfo%253Apmid%252F19571811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19571811&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268670300041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.13" data-doi="10.1534/genetics.115.177907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Reif</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling Epistasis in Genomic Selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177907%26rft_id%253Dinfo%253Apmid%252F26219298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.14" data-doi="10.1038/s41588-018-0183-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khera</span>,  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chaffin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. G.</span> <span class="cit-name-surname">Aragam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Haas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Roselli</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0183-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0183-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0183-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.15" data-doi="10.1038/nature09410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lango Allen</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Estrada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Lettre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Berndt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. N.</span> <span class="cit-name-surname">Weedon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hundreds of variants clustered in genomic loci and biological pathways affect human height.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09410</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLango%2BAllen%26rft.auinit1%253DH.%26rft.volume%253D467%26rft.issue%253D7317%26rft.spage%253D832%26rft.epage%253D838%26rft.atitle%253DHundreds%2Bof%2Bvariants%2Bclustered%2Bin%2Bgenomic%2Bloci%2Band%2Bbiological%2Bpathways%2Baffect%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09410%26rft_id%253Dinfo%253Apmid%252F20881960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20881960&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282898700065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.16" data-doi="10.1534/genetics.118.301267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lello</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Avery</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Tellier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Vazquez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accurate Genomic Prediction of Human Height.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301267</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301267%26rft_id%253Dinfo%253Apmid%252F30150289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzIvNDc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMS8xMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.17" data-doi="10.1038/ng.125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. U.</span> <span class="cit-name-surname">Jackson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gieger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. R.</span> <span class="cit-name-surname">Schumacher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Berndt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of ten loci associated with height highlights new biological pathways in human growth.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.125</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLettre%26rft.auinit1%253DG.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D584%26rft.epage%253D591%26rft.atitle%253DIdentification%2Bof%2Bten%2Bloci%2Bassociated%2Bwith%2Bheight%2Bhighlights%2Bnew%2Bbiological%2Bpathways%2Bin%2Bhuman%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.125%26rft_id%253Dinfo%253Apmid%252F18391950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391950&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.18" data-doi="10.1016/j.cell.2019.04.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. I.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Trans Effects on Gene Expression Can Drive Omnigenic Inheritance.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1022</span>–<span class="cit-lpage">1034.e6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2019.04.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1022%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.04.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.04.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.19" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epistasis and quantitative traits: using model organisms to study gene-gene interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.20" data-doi="10.1186/gm561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Moore</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Why epistasis is important for tackling complex human disease genetics.</span> <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gm561</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D6%26rft.spage%253D124%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm561%26rft_id%253Dinfo%253Apmid%252F25031624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25031624&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.21" data-doi="10.1371/journal.pgen.1002051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makowsky</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Pajewski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. C.</span> <span class="cit-name-surname">Klimentidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Vazquez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Duarte</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Beyond Missing Heritability: Prediction of Complex Traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002051</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMakowsky%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002051%26rft.epage%253De1002051%26rft.atitle%253DBeyond%2Bmissing%2Bheritability%253A%2Bprediction%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002051%26rft_id%253Dinfo%253Apmid%252F21552331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552331&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.22" data-doi="10.1038/s41588-019-0379-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kanai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kamatani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Okada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Neale</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Clinical use of current polygenic risk scores may exacerbate health disparities.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-019-0379-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D51%26rft.spage%253D584%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-019-0379-x%26rft_id%253Dinfo%253Apmid%252F30926966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-019-0379-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30926966&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.23" data-doi="10.1186/s12859-016-1439-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>,  <span class="cit-name-given-names">J. W. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Cardoso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Wimmer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Erbe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction with epistasis models: on the marker-coding-dependent performance of the extended GBLUP and properties of the categorical epistasis model (CE).</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1439-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1439-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1439-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.24" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8xLzEwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.25" data-doi="10.1038/s41598-018-30089-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Momen</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Mehrgardi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sheikhi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kranis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Tusell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Predictive ability of genome-assisted statistical models under various forms of gene action.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">12309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-30089-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D8%26rft.spage%253D12309%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-30089-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-30089-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.26" data-doi="10.1038/s41437-017-0043-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maltecca</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Effect of genetic architecture on the prediction accuracy of quantitative traits in samples of unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">500</span>–<span class="cit-lpage">514</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-017-0043-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D120%26rft.spage%253D500%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-017-0043-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-017-0043-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.27" data-doi="10.1101/629949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mostafavi</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Harpak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Conley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Przeworski</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Variable prediction accuracy of polygenic scores within an ancestry group.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/629949</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F629949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI2Mjk5NDl2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.28" data-doi="10.1371/journal.pone.0126880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Magwire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schlather</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Simianer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accounting for genetic architecture improves sequence based genomic prediction for a Drosophila fitness trait.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0126880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0126880%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0126880%26rft_id%253Dinfo%253Apmid%252F25950439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0126880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950439&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.29" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. R.</span> <span class="cit-name-surname">Ferreira</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.30" data-doi="10.1375/136905203770326402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silventoinen</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sammalisto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Perola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. I.</span> <span class="cit-name-surname">Boomsma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. K.</span> <span class="cit-name-surname">Cornes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Heritability of Adult Body Height: A Comparative Study of Twin Cohorts in Eight Countries.</span> <abbr class="cit-jnl-abbrev">Twin Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">408</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1375/136905203770326402</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTwin%2Bresearch%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BTwin%2BStudies%26rft.stitle%253DTwin%2BRes%26rft.aulast%253DSilventoinen%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D399%26rft.epage%253D408%26rft.atitle%253DHeritability%2Bof%2Badult%2Bbody%2Bheight%253A%2Ba%2Bcomparative%2Bstudy%2Bof%2Btwin%2Bcohorts%2Bin%2Beight%2Bcountries.%26rft_id%253Dinfo%253Adoi%252F10.1375%252F136905203770326402%26rft_id%253Dinfo%253Apmid%252F14624724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1375/136905203770326402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14624724&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187573800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.31" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient Methods to Compute Genomic Predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.32" data-doi="10.3389/fgene.2018.00078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Toro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. G.</span> <span class="cit-name-surname">Vitezica</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Non-additive Effects in Genomic Selection.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2018.00078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D9%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2018.00078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2018.00078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.33" data-doi="10.1016/j.ajhg.2017.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Wray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sklar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. I.</span> <span class="cit-name-surname">McCarthy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">10 Years of GWAS Discovery: Biology, Function, and Translation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2017.06.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D101%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2017.06.005%26rft_id%253Dinfo%253Apmid%252F28686856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2017.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28686856&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.34" data-doi="10.1101/588020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wainschtein</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Jain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yengo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth cit-collab">TOPMed Anthropometry Working Group</span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Recovery of trait heritability from whole genome sequence data.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/588020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F588020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1ODgwMjB2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzEvMTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.35" data-doi="10.1038/ng.121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>,  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lango</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Lindgren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wallace</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Evans</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide association analysis identifies 20 loci that influence adult height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">583</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWeedon%26rft.auinit1%253DM.%2BN.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D575%26rft.epage%253D583%26rft.atitle%253DGenome-wide%2Bassociation%2Banalysis%2Bidentifies%2B20%2Bloci%2Bthat%2Binfluence%2Badult%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.121%26rft_id%253Dinfo%253Apmid%252F18391952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391952&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.36" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Benyamin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">McEvoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gordon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Henders</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.109.37" data-doi="10.1016/j.ajhg.2010.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GCTA: A tool for genome-wide complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2010.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGCTA%253A%2Ba%2Btool%2Bfor%2Bgenome-wide%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.011%26rft_id%253Dinfo%253Apmid%252F21167468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21167468&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F109.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/109.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/89" title="Which Recurrent Selection Scheme To Improve Mixtures of Crop Species? Theoretical Expectations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/117" title="A Functional Analysis of the Drosophila Gene hindsight: Evidence for Positive Regulation of EGFR Signaling" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/109" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2irvZouQzd7S5rio6T2W_T0jfJa_69xzuK2xKW3dKUQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/109" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/109" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/109" class="active" data-icon-position="" data-hide-link-title="0">Influence of Genetic Interactions on Polygenic Prediction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495165" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-R4drRObwiHDLrJGniE75sqs7lq6pY-7LlD-o6iPfsuk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/109.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/109" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-lsF2vP31vnI04oZW_mEzk2w4z1If60voYVGGuF9_ZfU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/109" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/109.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495165" id="node495165--2" data-pisa="ggg;10/1/109" data-pisa-master="ggg;g3.119.400812" data-apath="/ggg/10/1/109.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Influence of Genetic Interactions on Polygenic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhijun Dai</span>, <span class="highwire-citation-author" data-delta="1">Nanye Long</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6788-8364" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wen Huang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">109-115;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400812 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495165--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhijun Dai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhijun%2BDai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhijun%2BDai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nanye Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Cyber-Enabled Research, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nanye%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANanye%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wen Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wen%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWen%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6788-8364" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wen Huang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:huangw53@msu.edu" class="" data-icon-position="" data-hide-link-title="0">huangw53@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/109" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/109&amp;title=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/109&amp;title=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/109&amp;title=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/109&amp;text=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/109&amp;title=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/109&amp;t=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/109&amp;title=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/109&amp;title=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495165/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495165" id="node495165--3" data-pisa="ggg;10/1/109" data-pisa-master="ggg;g3.119.400812" data-apath="/ggg/10/1/109.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Influence of Genetic Interactions on Polygenic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhijun Dai</span>, <span class="highwire-citation-author" data-delta="1">Nanye Long</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6788-8364" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wen Huang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">109-115;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400812 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495165--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhijun Dai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhijun%2BDai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhijun%2BDai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nanye Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Cyber-Enabled Research, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nanye%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANanye%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wen Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wen%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWen%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6788-8364" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wen Huang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:huangw53@msu.edu" class="" data-icon-position="" data-hide-link-title="0">huangw53@msu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=109&amp;rft.epage=115&amp;rft.atitle=Influence%20of%20Genetic%20Interactions%20on%20Polygenic%20Prediction&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dai%2C+Zhijun&amp;rft.au=Long%2C+Nanye&amp;rft.au=Huang%2C+Wen"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495165/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495165/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495165/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495165/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495165/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495165/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495165/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495165/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495165/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495165/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495165/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495165/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/109" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/109.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/109.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/109.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
