<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/1/23" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/1/23.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/1/23.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/1/23.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/1/23.atom" />
<meta name="HW.pisa" content="ggg;10/1/23" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Sequencing and Functional Annotation of the Whole Genome of Shiraia bambusicola" />
<meta name="DC.Identifier" content="10.1534/g3.119.400694" />
<meta name="DC.Date" content="2020-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Ren et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Shiraia bambusicola is a rare medicinal fungus found in China that causes bamboo plants to decay and die with severe infection. Hypocrellin, its main active ingredient, is widely used in several fields, such as medicine, agriculture, and food industry. In this study, to clarify the genomic components, taxonomic status, pathogenic genes, secondary metabolite synthesis pathways, and regulatory mechanisms of S. bambusicola , whole-genome sequencing, assembly, and functional annotation were performed using high-throughput sequencing and bioinformatics approaches. It was observed that S. bambusicola has 33 Mb genome size, 48.89% GC content, 333 scaffolds, 2590 contigs, 10,703 genes, 82 tRNAs, and 21 rRNAs. The total length of the repeat sequence is 2,151,640 bp. The annotation of 5945 proteins was obtained from InterProScan hits based on the Gene Ontology database. Phylogenetic analysis showed that S. bambusicola belongs to Shiraiaceae, a new family of Pleosporales. It was speculated that there are more than two species or genus in Shiraiaceae. According to the annotation, 777 secreted proteins were associated with virulence or detoxification, including 777 predicted by the PHI database, 776 by the CAZY and Fungal Cytochrome P 450 database, and 441 by the Proteases database. The 252 genes associated with the secondary metabolism of S. bambusicola were screened and enriched into 28 pathways, among which the terpenoids, staurosporine, aflatoxin, and folate synthesis pathways have not been reported in S. bambusicola . The T1PKS was the main gene cluster among the 28 secondary metabolite synthesis gene clusters in S. bambusicola . The analysis of the T3PKS gene cluster related to the synthesis of hypocrellin showed that there was some similarity between S. bambusicola and 10 other species of fungi; however, the similarity was very low wherein the highest similarity was 17%. The genomic information of S. bambusicola obtained in this study was valuable to understand its genetic function and pathogenicity. The genomic information revealed that several enzyme genes and secreted proteins might be related to their host interactions and pathogenicity. The annotation and analysis of its secondary metabolite synthesis genes and gene clusters will be an important reference for future studies on the biosynthesis and regulation mechanism of the secondary metabolites, contributing to the discovery of new metabolites and accelerating drug development and application." />
<meta name="DC.Contributor" content="Xiyi Ren" />
<meta name="DC.Contributor" content="Yongxiang Liu" />
<meta name="DC.Contributor" content="Yumei Tan" />
<meta name="DC.Contributor" content="Yonghui Huang" />
<meta name="DC.Contributor" content="Zuoyi Liu" />
<meta name="DC.Contributor" content="Xuanli Jiang" />
<meta name="article:published_time" content="2020-01-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Sequencing and Functional Annotation of the Whole Genome of Shiraia bambusicola" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Shiraia bambusicola&lt;/i&gt; is a rare medicinal fungus found in China that causes bamboo plants to decay and die with severe infection. Hypocrellin, its main active ingredient, is widely used in several fields, such as medicine, agriculture, and food industry. In this study, to clarify the genomic components, taxonomic status, pathogenic genes, secondary metabolite synthesis pathways, and regulatory mechanisms of &lt;i&gt;S. bambusicola&lt;/i&gt;, whole-genome sequencing, assembly, and functional annotation were performed using high-throughput sequencing and bioinformatics approaches. It was observed that &lt;i&gt;S. bambusicola&lt;/i&gt; has 33 Mb genome size, 48.89% GC content, 333 scaffolds, 2590 contigs, 10,703 genes, 82 tRNAs, and 21 rRNAs. The total length of the repeat sequence is 2,151,640 bp. The annotation of 5945 proteins was obtained from InterProScan hits based on the Gene Ontology database. Phylogenetic analysis showed that &lt;i&gt;S. bambusicola&lt;/i&gt; belongs to Shiraiaceae, a new family of Pleosporales. It was speculated that there are more than two species or genus in Shiraiaceae. According to the annotation, 777 secreted proteins were associated with virulence or detoxification, including 777 predicted by the PHI database, 776 by the CAZY and Fungal Cytochrome&lt;i&gt;P&lt;/i&gt;450 database, and 441 by the Proteases database. The 252 genes associated with the secondary metabolism of &lt;i&gt;S. bambusicola&lt;/i&gt; were screened and enriched into 28 pathways, among which the terpenoids, staurosporine, aflatoxin, and folate synthesis pathways have not been reported in &lt;i&gt;S. bambusicola&lt;/i&gt;. The &lt;i&gt;T1PKS&lt;/i&gt; was the main gene cluster among the 28 secondary metabolite synthesis gene clusters in &lt;i&gt;S. bambusicola&lt;/i&gt;. The analysis of the T3PKS gene cluster related to the synthesis of hypocrellin showed that there was some similarity between &lt;i&gt;S. bambusicola&lt;/i&gt; and 10 other species of fungi; however, the similarity was very low wherein the highest similarity was 17%. The genomic information of &lt;i&gt;S. bambusicola&lt;/i&gt; obtained in this study was valuable to understand its genetic function and pathogenicity. The genomic information revealed that several enzyme genes and secreted proteins might be related to their host interactions and pathogenicity. The annotation and analysis of its secondary metabolite synthesis genes and gene clusters will be an important reference for future studies on the biosynthesis and regulation mechanism of the secondary metabolites, contributing to the discovery of new metabolites and accelerating drug development and application.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="ggg;10/1/23" />
<meta name="citation_id" content="10/1/23" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/1/23" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/1/23.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/1/23.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/1/23.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400694" />
<meta name="citation_pmid" content="31712259" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="23" />
<meta name="citation_lastpage" content="35" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xiyi Ren" />
<meta name="citation_author_institution" content="College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China," />
<meta name="citation_author_institution" content="Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China," />
<meta name="citation_author_institution" content="Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and" />
<meta name="citation_author" content="Yongxiang Liu" />
<meta name="citation_author_institution" content="Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China," />
<meta name="citation_author_institution" content="Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and" />
<meta name="citation_author" content="Yumei Tan" />
<meta name="citation_author_institution" content="Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China," />
<meta name="citation_author_institution" content="Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and" />
<meta name="citation_author" content="Yonghui Huang" />
<meta name="citation_author_institution" content="Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China," />
<meta name="citation_author_institution" content="Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and" />
<meta name="citation_author" content="Zuoyi Liu" />
<meta name="citation_author_institution" content="Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and" />
<meta name="citation_author_institution" content="Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China" />
<meta name="citation_author_email" content="jxl3237@163.com" />
<meta name="citation_author_email" content="gzliuzuoyi@163.com" />
<meta name="citation_author" content="Xuanli Jiang" />
<meta name="citation_author_institution" content="College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China," />
<meta name="citation_author_email" content="jxl3237@163.com" />
<meta name="citation_author_email" content="gzliuzuoyi@163.com" />
<meta name="citation_reference" content="citation_journal_title=Bulletin of the National Science Museum;citation_author=N. Amano;citation_title=Studies on the japanese loculoascomycetes. ii. taxonomic postion of the genus shiraia.;citation_pages=55-60;citation_volume=6;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. Benson;citation_title=Tandem repeats finder: a program to analyze dna sequences.;citation_volume=27;citation_year=1999;citation_pmid=9862982;citation_doi=10.1093/nar/27.2.573" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=E. Birney;citation_author=M. Clamp;citation_author=R. Durbin;citation_title=GeneWise and Genomewise;citation_pages=988-995;citation_volume=14;citation_year=2004;citation_issue=5;citation_pmid=15123596;citation_doi=10.1101/gr.1865504" />
<meta name="citation_reference" content="citation_journal_title=Current protocols in bioinformatics;citation_author=N. Chen;citation_title=Using Repeat Masker to identify repetitive elements in genomic sequences.;citation_pages=4-10;citation_volume=5;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Journal of basic microbiology;citation_journal_abbrev=J Basic Microbiol;citation_author=T. F. Cheng;citation_author=X. M. Jia;citation_author=X. H. Ma;citation_author=H. P. Lin;citation_author=Y. H. Zhao;citation_title=Phylogenetic study on Shiraia bambusicola by rDNA sequence analyses.;citation_pages=339-350;citation_volume=44;citation_year=2004;citation_issue=5;citation_pmid=15378525;citation_doi=10.1002/jobm.200410434" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Chong;citation_author=R. Nielsen;citation_author=Y. Wu;citation_title=REPdenovo: inferring de novo repeat motifs from short sequence reads.;citation_pages=e0150719;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0150719" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=M. Talon;citation_title=Blast2go: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Non-coding RNA.;citation_author=A. Expósito Villén;citation_author=E. Aránega;citation_author=D. Franco;citation_title=Functional role of non-coding RNAs during epithelial-to-mesenchymal transition.;citation_pages=14;citation_volume=4;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=N. Funa;citation_author=T. Awakawa;citation_author=S. Horinouchi;citation_title=Pentaketide resorcylic acid synthesis by type iii polyketide synthase from neurospora crassa.;citation_volume=282;citation_year=2007;citation_pmid=17374612;citation_doi=10.1074/jbc.M701239200" />
<meta name="citation_reference" content="citation_journal_title=Acta Biochimica et Biophysica Sinica.;citation_author=L. Han;citation_author=H. Li;citation_author=M. Cai;citation_author=S. He;citation_author=X. Shao;citation_title=Ricin-b-lectin enhances microsporidia nosema bombycis infection in bmn cells from silkworm bombyx mori.;citation_volume=48;citation_year=2016;citation_pmid=27649890;citation_doi=10.1093/abbs/gmw093" />
<meta name="citation_reference" content="citation_journal_title=Bot. Jahrb. Syst. Pflanzengesch. Pflanzengeogr.;citation_author=P. C. Hennings;citation_title=Fungi japonica.;citation_pages=259-280;citation_volume=28;citation_year=1900" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=X. M. Jia;citation_author=X. H. Xu;citation_author=B. C. Zhuang;citation_author=H. P. Lin;citation_title=The progress of biological research of medicinal fungus shiraia bambusicola.;citation_pages=147-150;citation_volume=33;citation_year=2006;citation_doi=10.1360/yc- 006&amp;ndash;1627" />
<meta name="citation_reference" content="citation_author=N. Jiang;citation_pages=275;citation_volume=1057;citation_year=2013;citation_pmid=23918436;citation_doi=10.1007/978-1-62703-568-2_20" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=Blat-the blast-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="Khaldi, N., F. T. Seifuddin, G. Turner, D. Haft, W. C. Nierman et al., 2010 Smurf: genomic mapping of fungal secondary metabolite clusters. Fungal Genetics &amp; Biology Fg &amp; B, 47: 736–741. 10.1016/j.fgb.2010.06.003" />
<meta name="citation_reference" content="citation_title=Dictionary of the Fungi;citation_author=P. M. Kirk;citation_author=P. F. Cannon;citation_author=J. C. David;citation_author=J. A. Stalpers;citation_year=2001;citation_doi=10.1590/S0036&amp;ndash;46651996000400018" />
<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution;citation_author=P. Kück;citation_author=K. Meusemann;citation_title=Fasconcat: convenient handling of data matrices.;citation_pages=1115-1118;citation_volume=56;citation_year=2010;citation_pmid=20416383;citation_doi=10.1016/j.ympev.2010.04.024" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Lechner;citation_author=S. Findeiss;citation_author=L. Steiner;citation_author=M. Marz;citation_author=P. F. Stadler;citation_title=Proteinortho: detection of (co-)orthologs in large-scale analysis.;citation_pages=124-124;citation_volume=12;citation_year=2011;citation_pmid=21526987;citation_doi=10.1186/1471-2105-12-124" />
<meta name="citation_reference" content="citation_journal_title=Sci. China Life Sci.;citation_author=Q. LiangHu;citation_title=Non-coding rna annotation:deciphering the second genetic code.;citation_pages=865-866;citation_volume=56;citation_year=2013;citation_doi=10.1007/s11427-013-4544-7" />
<meta name="citation_reference" content="Li, H., and R. Durbin, 2009 Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25: 1754–1760. 10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Q. Li;citation_author=W. Fan;citation_author=G. Tian;citation_author=H. M. Zhu;citation_author=L. He;citation_title=The sequence and de novo assembly of the giant panda genome.;citation_pages=311-317;citation_volume=463;citation_year=2010;citation_issue=7279;citation_pmid=20010809;citation_doi=10.1038/nature08696" />
<meta name="citation_reference" content="citation_journal_title=Forest. Sci. and Tech.;citation_author=H. P. Lin;citation_author=S. Chen;citation_author=. Mi;citation_author=C. L. Chen;citation_title=Shiraia bambusicola, a medicinal fungi needed to be developed. Journ. of Zhej;citation_pages=77-71;citation_volume=22;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Guiz. Agri. Sci.;citation_author=Y. X. Liu;citation_author=Z. Y. Liu;citation_author=Y. Quan;citation_title=Basic Nutrition for Stroma Development of Shiraia bambusicola and Hypocrellin A(HA) Yield by Mycelium Fermentation[J].;citation_pages=98-102;citation_volume=39;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Phytotaxa;citation_author=Y. X. Liu;citation_author=K. D. Hyde;citation_author=H. A. Ariyawansa;citation_author=W. J. Li;citation_author=D. Q. Zhou;citation_title=Shiraiaceae, new family of Pleosporales (Dothideomycetes, Ascomycota).;citation_pages=51-60;citation_volume=103;citation_year=2013;citation_doi=10.11646/phytotaxa.103.1.4" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. Marcet-Houben;citation_author=A. R. Ballester;citation_author=D. L. F. Beatriz;citation_author=E. Harries;citation_author=J. F. Marcos;citation_title=Genome sequence of the necrotrophic fungus Penicillium digitatum, the main postharvest pathogen of citrus.;citation_pages=646-646;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23171342;citation_doi=10.1186/1471-2164-13-646" />
<meta name="citation_reference" content="Medema, M. H., B. Kai, C. Peter, D. J. Victor, Z. Piotr et al., 2011 Antismash: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences. Nucleic Acids Research (suppl_2), 39(WebServer issue): W339–346. 10.1093/nar/gkr466" />
<meta name="citation_reference" content="citation_journal_title=BMC neuroscience [electronic resource];citation_journal_abbrev=BMC Neurosci;citation_author=T. R. Mercer;citation_author=I. A. Qureshi;citation_author=S. Gokhan;citation_title=Long noncoding RNAs in neuronal-glial fate specification and oligodendrocyte lineage maturation.;citation_pages=14-14;citation_volume=11;citation_year=2010;citation_pmid=20137068;citation_doi=10.1186/1471-2202-11-14" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=V. Moktali;citation_author=J. Park;citation_author=N. D. Fedorova-Abrams;citation_author=B. Park;citation_author=J. Choi;citation_title=Systematic and searchable classification of cytochrome P450 proteins encoded by fungal and oomycete genomes.;citation_pages=525-525;citation_volume=13;citation_year=2012;citation_pmid=23033934;citation_doi=10.1186/1471-2164-13-525" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. D. Rawlings;citation_author=D. P. Tolle;citation_author=A. J. Barrett;citation_title=Merops: the peptidase database.;citation_volume=36;citation_year=2008;citation_pmid=17991683;citation_doi=10.1093/nar/gkm954" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. A. Salamov;citation_author=V. V. Solovyev;citation_title=Ab initio Gene Finding in Drosophila Genomic DNA;citation_pages=516-522;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779491;citation_doi=10.1101/gr.10.4.516" />
<meta name="citation_reference" content="citation_journal_title=wydawca nieznany;citation_title=De Diagnostica et nomenclatura mycologica, Admonita quaedam[M].;citation_year=1902" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=H. A. Schmidt;citation_author=K. Strimmer;citation_author=M. Vingron;citation_author=A. von Haeseler;citation_title=TREE-PUZZLE: maximum likelihood phylogenetic analysis using  quartets and parallel computing;citation_pages=502-504;citation_volume=18;citation_year=2002;citation_issue=3;citation_pmid=11934758;citation_doi=10.1093/bioinformatics/18.3.502" />
<meta name="citation_reference" content="citation_journal_title=Food Science;citation_author=Y. Shao;citation_author=A. H. Chen;citation_author=J. P. He;citation_title=Stability and anti-tumor activity of hypocrellin from liquid-cultured mycelia of shiraia bambusicola strain zh-5-1.;citation_pages=96-100;citation_volume=32;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Zhongguo Zhongyao Zazhi;citation_author=Y. X. Shen;citation_author=X. G. Rong;citation_author=Z. H. Gao;citation_title=studies on the chemical constituents of shiraia bambusicola. Zhongguo Zhong yao za zhi = Zhongguo zhongyao zazhi =;citation_pages=674-676;citation_volume=27;citation_year=2002;citation_doi=10.3321/j.issn:1001&amp;ndash;5302.2002.09.010" />
<meta name="citation_reference" content="citation_journal_title=Food Res. &amp; Devel.;citation_author=W. Y. Shi;citation_author=P. Li;citation_author=T. C. Zhang;citation_title=Study on fermentation mediumfor new pigment hypocrellin.;citation_pages=182-185;citation_volume=43;citation_year=2016;citation_issue=62" />
<meta name="citation_reference" content="Stanke, M., and S. Waack, 2003 Gene prediction with a hidden markov model and a new intron submodel. Bioinformatics, 19(Suppl 2), ii215-ii225. 10.1093/bioinformatics/btg1080" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=R. J. Taft;citation_author=C. Simons;citation_author=S. Nahkuri;citation_title=Nuclear-localized tiny RNAs are associated with transcription initiation and splice sites in metazoans.;citation_pages=1030-1034;citation_volume=17;citation_year=2010;citation_issue=8;citation_pmid=20622877;citation_doi=10.1038/nsmb.1841" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=G. Talavera;citation_author=J. Castresana;citation_title=Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments.;citation_volume=56;citation_year=2007;citation_pmid=17654362;citation_doi=10.1080/10635150701472164" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=S. Washietl;citation_title=Sequence and structure analysis of noncoding RNAs.;citation_pages=285-306;citation_volume=609;citation_year=2010;citation_pmid=20221926;citation_doi=10.1007/978-1-60327-241-4_17" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Winnenburg;citation_author=M. Urban;citation_author=A. Beacham;citation_author=T. K. Baldwin;citation_author=H. Sabrina;citation_title=Phi-base update: additions to the pathogen-host interaction database.;citation_volume=36;citation_year=2008;citation_pmid=17942425;citation_doi=10.1093/nar/gkm858" />
<meta name="citation_reference" content="Yin, Y., X. Mao, J. Yang, X. Chen, F. Mao, et al., 2012 dbCAN: a web resource for automated carbohydrate-active enzyme annotation. Nucleic Acids Res,40(Web Server issue): W445–51. 10.1093/nar/gks479" />
<meta name="citation_reference" content="citation_journal_title=Chinese Journal of Family Planning &amp; Gynecotokology;citation_author=J. F. Zhao;citation_author=L. Xi;citation_author=L. F. Yang;citation_author=Z. H. Huang;citation_author=M. H. Tian;citation_title=Clinical observation of curative effect on Focusedultrasound combined with hypocrellinointment to treat leukoplakia vulvae in 164 cases.;citation_pages=61-63;citation_volume=4;citation_year=2012;citation_doi=10.3969/j.issn.1674&amp;ndash;4020.2012.01.017" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Molecular Sciences;citation_author=N. Zhao;citation_author=Z. ;citation_author=L. Xi;citation_author=Q. Shan-Shan;citation_author=L. Zhi-Mei;citation_author=C. Shuang-Lin;citation_title=De novo transcriptome assembly in shiraia bambusicola to investigate putative genes involved in the biosynthesis of hypocrellin a.;citation_pages=311;citation_volume=17;citation_year=2016;citation_doi=10.3390/56ijms17030311" />
<meta name="citation_reference" content="citation_journal_title=Acta Phytopathologica Sin.;citation_author=Q. C. Zhu;citation_title=Inhibition of hypocrellin a against 18 pathogenic fungi.;citation_pages=107-109;citation_volume=44;citation_year=2014;citation_doi=10.3969/j.issn.0412&amp;ndash;0914.2014.01.013" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Sequencing and Functional Annotation of the Whole Genome of Shiraia bambusicola" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
<meta name="twitter:description" content="Shiraia bambusicola is a rare medicinal fungus found in China that causes bamboo plants to decay and die with severe infection. Hypocrellin, its main active ingredient, is widely used in several fields, such as medicine, agriculture, and food industry. In this study, to clarify the genomic components, taxonomic status, pathogenic genes, secondary metabolite synthesis pathways, and regulatory mechanisms of S. bambusicola , whole-genome sequencing, assembly, and functional annotation were performed using high-throughput sequencing and bioinformatics approaches. It was observed that S. bambusicola has 33 Mb genome size, 48.89% GC content, 333 scaffolds, 2590 contigs, 10,703 genes, 82 tRNAs, and 21 rRNAs. The total length of the repeat sequence is 2,151,640 bp. The annotation of 5945 proteins was obtained from InterProScan hits based on the Gene Ontology database. Phylogenetic analysis showed that S. bambusicola belongs to Shiraiaceae, a new family of Pleosporales. It was speculated that there are more than two species or genus in Shiraiaceae. According to the annotation, 777 secreted proteins were associated with virulence or detoxification, including 777 predicted by the PHI database, 776 by the CAZY and Fungal Cytochrome P 450 database, and 441 by the Proteases database. The 252 genes associated with the secondary metabolism of S. bambusicola were screened and enriched into 28 pathways, among which the terpenoids, staurosporine, aflatoxin, and folate synthesis pathways have not been reported in S. bambusicola . The T1PKS was the main gene cluster among the 28 secondary metabolite synthesis gene clusters in S. bambusicola . The analysis of the T3PKS gene cluster related to the synthesis of hypocrellin showed that there was some similarity between S. bambusicola and 10 other species of fungi; however, the similarity was very low wherein the highest similarity was 17%. The genomic information of S. bambusicola obtained in this study was valuable to understand its genetic function and pathogenicity. The genomic information revealed that several enzyme genes and secreted proteins might be related to their host interactions and pathogenicity. The annotation and analysis of its secondary metabolite synthesis genes and gene clusters will be an important reference for future studies on the biosynthesis and regulation mechanism of the secondary metabolites, contributing to the discovery of new metabolites and accelerating drug development and application." />
<meta name="og-title" property="og:title" content="Sequencing and Functional Annotation of the Whole Genome of Shiraia bambusicola" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/1/23" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Shiraia bambusicola is a rare medicinal fungus found in China that causes bamboo plants to decay and die with severe infection. Hypocrellin, its main active ingredient, is widely used in several fields, such as medicine, agriculture, and food industry. In this study, to clarify the genomic components, taxonomic status, pathogenic genes, secondary metabolite synthesis pathways, and regulatory mechanisms of S. bambusicola , whole-genome sequencing, assembly, and functional annotation were performed using high-throughput sequencing and bioinformatics approaches. It was observed that S. bambusicola has 33 Mb genome size, 48.89% GC content, 333 scaffolds, 2590 contigs, 10,703 genes, 82 tRNAs, and 21 rRNAs. The total length of the repeat sequence is 2,151,640 bp. The annotation of 5945 proteins was obtained from InterProScan hits based on the Gene Ontology database. Phylogenetic analysis showed that S. bambusicola belongs to Shiraiaceae, a new family of Pleosporales. It was speculated that there are more than two species or genus in Shiraiaceae. According to the annotation, 777 secreted proteins were associated with virulence or detoxification, including 777 predicted by the PHI database, 776 by the CAZY and Fungal Cytochrome P 450 database, and 441 by the Proteases database. The 252 genes associated with the secondary metabolism of S. bambusicola were screened and enriched into 28 pathways, among which the terpenoids, staurosporine, aflatoxin, and folate synthesis pathways have not been reported in S. bambusicola . The T1PKS was the main gene cluster among the 28 secondary metabolite synthesis gene clusters in S. bambusicola . The analysis of the T3PKS gene cluster related to the synthesis of hypocrellin showed that there was some similarity between S. bambusicola and 10 other species of fungi; however, the similarity was very low wherein the highest similarity was 17%. The genomic information of S. bambusicola obtained in this study was valuable to understand its genetic function and pathogenicity. The genomic information revealed that several enzyme genes and secreted proteins might be related to their host interactions and pathogenicity. The annotation and analysis of its secondary metabolite synthesis genes and gene clusters will be an important reference for future studies on the biosynthesis and regulation mechanism of the secondary metabolites, contributing to the discovery of new metabolites and accelerating drug development and application." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/1.cover-source.jpg" />
    <title>Sequencing and Functional Annotation of the Whole Genome of Shiraia bambusicola | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TSUor36WZ3_zTjePV26iA_-GtReh1M96XdYQZygLt7o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495264","apath":"\/ggg\/10\/1\/23.atom","pisa":"ggg;10\/1\/23","ac":{"\/ggg\/10\/1\/23.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/1\/23.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/1\/23"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/1\/23"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/1\/23":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z\\\\\\\/tJV1GmpsFZFOyCoV87RBbubGyl3y81I2lYMhYAw7OOcM24ofXabYJTNhPqhtEH52oIhycqsw7D\\\\\\\/O1TgDIBiSetIgpxMWBowdHnGG7srU8mIZqptobZ0mJ7ACV0Odhf1feq5+0dOghOxpulhCi\\\\\\\/gLxVwGZ8KSnxGaUx2vRdltQDopdTVijltfOkXibPM+DgyRTPdiY9oE0WqG0hLaVtXNkz2s+UlyIf6TxSFSWPXlQ2mPohd0Ls\\\\\\\/QcJYlqEES+6xtdtCKGxwQDYMLKnt+5g0PPhN98050rqaCSFbp+oTb8SvY19ITESr6PLI2roVQwmq7KDgoTwdZzgPNm1c4FkQ8rHtGSRcYYTZ4tRJpzN1VcPphxkrMvdtrR\\\\\\\/Cv1fb2s+gcnyrhqjVXivizp19NcwkT\\\\\\\/r5\\\\\\\/GzhuatEqBVgJnTDjCbezpFI0VWLMPdI1Ste+g80QKk0zYJ4GM+5U1lvHHIVOcgOwd3BDSetto7cYLk9RvU7fQKnoZEloLAdffABPx1gtf5o1lhdPXBGquRas6QAyGxvGPQsBPN5eCzNaOECeOC1Yh58M93AVHauN6bBGzjcuSRAKHtyL+4ygGhEvd2NQWX5JXos7EJPTsZtrC8UVYmj6qxxBq6m4KWI4Fu82pXN5L8Z0eCzAABnalW2j5CU5Afxp5vixdloadhLpZX5todUNGUvxK8s6INAhBjbAyRtf1DRbksCLkp5MntGbYKb7ficiuKYsc+xMBv1DqJ8TVYu8jXusWE5HHv6r0jai7jQpY2\\\\\\\/mh3CfhB7rf3H6VgQpmM7I7YDJRz6jKEg8uOoILZsikPinv2q38B8JCU3MpMR5hk9l8SPldzeNLRGQbM5pIN4UOmckh2pZIldfTeXLWN7vbRyFchLgD0WxTsILoh2lUQH+NsBOQrW3AshI3HbmlxF2JdSQQddOkOASH2AnjYyAKvxVnTc3jJ3c\\\\\\\/o8XkFk1g32c0LJdTCn9UfNCaMw1u53rVHOAh7Nc\\\\\\\/QQq\\\\\\\/UPyxKoeH+biY\\\\\\\/nRRjWC2Dx8xg0HU8VcWEVJvCdU13n\\\\\\\/qaGZ3ULYJGzXsu5HssK3mM1wblTE\\\\\\\/JO0vT26lHOOanoPsDE9YXAB\\\\\\\/NcRnGX\\\\\\\/cRBcIud7RuAxGncFr\\\\\\\/FHAp\\\\\\\/6yId24FHVNHf5JRJBYdE+6JJXqQttkd5i5N3N32gdr+dAAYN314Ixm6nxx6\\\\\\\/Pqr2ZycC9cPszNEC0xFRwPLpIBfRr1Ny3Z8V4Le\\\\\\\/UFBVKo6d6diP4PGhBFiFNSh6gIwUSkwIEsTwCKH\\\\\\\/oGpD0sjgw+JMvq8\\\u0022,\\\u0022iv\\\u0022:\\\u0022FniKS\\\\\\\/6pT+phxk1CPiLztw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022040f47ee324eefbae53832fb7415c8f0f178ea4ec47c35beba38908b4facf043\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/L3n12ybN75PpLNwmYJuPxGWdQl7LcGUcNEWScgETLPByLZtUKlKpPTSf11OObbz0iJHBOpJEwAgd8VRREJG327hkH7FADEYRYb0b5DvDOX+t9Y9XxcoWos8mdKj3QAzytbbr\\\\\\\/YEHTeoK+64PWqy3B2z22B1PV7eXcRe2IasrNXVvJJbabRpLcvTHr9qh3bRX1\\\\\\\/PfSuKWW6xffuhcBnAA4ZiyWoVaeNpFLuWoI34Ae8ByFG38EfOFeAny0WpbpBNcDliFnvD1aMgZPBvGBC2eaCPwaXfBJL6A693vnKi0BUUr9uZ+rp3SD9PfP+nBWyuJNtiiJ3blsvmFvVddS67l8mbs9ugAhkzPScFIpMtxv2tSBaZJVkAxd2IKpUABTRFi2nOwLBUGoLXfHfyvBO5F\\\\\\\/jobkTz+6PzcYhlP3nxmD6DYOzvjJV8t\\\\\\\/BcnEq55t\\\\\\\/RVwD0ZVhWOOkNtG6jgTxUTJPFq2GttMx3aTTFGdhujoTe7czJrKoU+igideqDagLc1L5b48zFoMYv3fPAJ8eUs0Fd7O3UakZ8Z7zZ+lfeAMLrttnb\\\\\\\/x5xZyqdfdF9qEkltQrqryD3lSxiL2H+c1A8qW6zo2vNh0JWwcwZxkE6HfcnqUlGJbQBtfXEFcCRgwrjV\\\\\\\/US1TQU33wgbUBizNEnEKzQc2fXdGZpLMpYzjaADmK4ugO9M4fSAt7VtYVjZELHkneHJxpFx4BvLuINlmWV7BIQRWN+WU8rKmFOPjI2443DkCxZapQbgu0CoiEIUCY2wxc\\\\\\\/yM3WvPu+1gE2ziYCfdpteJUZfxwqJeemPpglaTwiLg\\\\\\\/tkwXOtZik26BcSNa9migz1XI6ipvb2LgH26ivHiXVXdPLlg7viYkZLzpTqHaYP3mcK8hSm4CyBU7hskde5LNfIdS\\\\\\\/I8rYqw7yfkiIyF6ubPMZ86GysB9JmTG1RM7v8sNRVPuloqochicYJIJuknRdGiV0xmVpOFAb2KH7ykrlKLysyn8fP43G35Qbj5JkSJ5KS09Paj2AfR21p8zYeDWHIGMnUMNiV4SsWL7N82b2Ddks3yFOVGUnkgl5+bxo\\\\\\\/Qoc0jQmoEF4Us7LK5W4yrT7ljGdcckUfEKR5wLgEu3909UIN0b8d9zBswGQhC7wrYe1k73wV4eiBkvPnWkSfwTKbilTfTs38\\\\\\\/nR5xzax4G3OyTlq+TcPP6Am\\\\\\\/X9AeYrjGwN\\\\\\\/b0NWgOABbxc9TxlPzf72aEDltL0RGFZT5vSyS3808FXjYYnW8Stnb9RcTVGTB3aI5tF7ahYrD+GrIE7eQuExV0AwHB9cRaDlDpfaR3yf57HNXobnnzGUXyk2L7qH2QI6Tb3eXyOOKxFspGNpfX5ciB+HVICeF\\\\\\\/YpbnuvlMn59VINknaevCnayMqSWCYfN4WYFPjLAJozZ1lWV3fwFEJ35LeebfS8ZvpjrP8hYABdrUBhyeU39LkMqprr78laZXUmVxU0sN8MpSWqU76aLj+Tzt63Vs459CfsZWH8g1w2EtJyElP3MqhXJbY8SLKp5708\\\\\\\/OjxZte6p3zbNqcENyayLlWiRN5nVpSCfA3HgSLQH1G3cGhgVlRUu8rkdIFoGsrCPGQwJU2rRkIaEB7uWVc\\\\\\\/7TvulzPG+g7K+4rSTikK+kpNiub+LhkIibkKMjjhXkuNZVZ30UAHoLTEzenmjlFJXk8fkwvPEVJGq6ZlHVAlMlv7XLRyINZHU3s5iTXfZNImmkyZRTXax3vPwM+oAY1alpkobjuluE7T2ieOoqcBDiqtirX\\\\\\\/Y7Pz0rjSVEvQOmfJJTam9jwAOANKgtuQ9zmNBLGhO1OjSSOUsoE2YRFNl5HkIKIXqKUIzd0FqBs0ogfmKdKo3JYOLdaysI+JOS5RyLjCjLperrsOPiZfC\\\\\\\/tL5JIisIwtQlVPxmc0ik\\\\\\\/79Nz0E3LCdgwuPOxF1Usf9tBytJ92t812cffjtkZbz5SRhF9Cc5kQ4kUehTIHB+asHJa\\\\\\\/BN9t\\\\\\\/RckRzVVcOI0f+styfbaw9cJ6x\\\\\\\/8p9Xv7uV9MpERliO01OSgIX1n5R+kURvs80Rl\\\\\\\/gPY8qm633EAVOz8gLmkK3S2UYtR+DYRpgcQ4u31LGUrIj5EjMS4U925TO4bSLv\\\\\\\/s\\\\\\\/\\\\\\\/nK3M30qBM+xoA0yo1xhyFw2y\\\\\\\/D3jWScqV286LZsFS5beR6eehb9i0GRsveWlLltcPUhIygIOzH31et\\\\\\\/QRmW5Jj2xBwkcrLuOBsBNQamC+3lihPeoY5u9a82btyzaPqCylZqUoCkYYV+1Wwatunp8Fk9idkfV\\\\\\\/XZz2Ub75LS9\\\\\\\/MQUEqFXGspokyrl2Gg03jVLP90xMv\\\\\\\/7RQhu2tYiXg5ZNzkIQjVnuhBiCaFivvf5vdXJRo5A3PgMYNMh9jjUoUHQKSAymLBOCmZ1ZHV6k4zEGkuXZ7pNqHsl2RFzlyzfdu36LUw1GZbvWoF2q3jGBw7O3evfn1nbdYiTxuNDe6tg5GGczT2RujdPZ3LY9+bGmwaRAI2vdiyyXaH+dMGVZxKxBvUZU4btipeiLQVNLsexJNWUPfckL0CILMqddIZ6ZQR2P0a8Q+ritrL\\\\\\\/YFq65yGuBQDZsAdsCeVyBVrgY3vPodwxMkOi3FGBOnA7tPLGnWm64ydz\\\\\\\/rq2r0UAlV8lbUtPDyGWh0nrW2QucWdsQ\\\\\\\/rxpmnPswkBKZrRnAXogdo1p3cB0lBLdpbDN+2gVn3JeaFflWEmlfKpejFhGdCAQs5rVE2tpKpAShV3tSRTEpU1eOMBa23CGRgYRHWUCjeuhJZz97BQuq22ufNFkN5f6xbeX0kukJQ2YmAl0Q1EjVDyi3XbSh0uOMZbUSTh1GRwsVAcEGDGPGqwVOYBWxk6JOdJQjz54LioBlT48DxPhGQfbclJirHFrNGIOaIskBrq5luTV5juzywIMB\\\\\\\/tnjvQO6RA69DIcA2+YndiJYXSYHiNxJ9AJSTDWUDk61uuYfTNrM8dfVyGnOSWxaJGSWKDidBhQuXzZ5oX\\\\\\\/ZnRydeIAejqV0WY3\\\\\\\/H2GHOIT+9f3saIYJRkKfIMdpt8dk60g4MtH9ZBQVKlYYPdVlSc1X3K1i+EJzzOQas\\\\\\\/sfd8SHoIYhPdzhkjR8SjHuCZDOba\\\\\\\/dkkagRrS80Ed7aL2HaFmaX\\\\\\\/Cdsh7QOnEzv\\\\\\\/ugMT5GmWTQCvUW5GFj5vevwg4x32De+WsdW3hARzAfg8O6j3YmHizZbJVLCABY\\\\\\\/\\\\\\\/WG5KscQqFf3+xs6s+N7nF9eNILNe2H+JcVopv3jjPP303gUkYEXScMOAaI60n47e5KU1\\\\\\\/aymJltGNXsnmZ\\\\\\\/fZaiMLmRTErDWNmcncgQJFZucVjj3CrvKuANLVFUFcbG\\\\\\\/MgFUWHPl5n8XFzvZdZQ1CQXApcX93Ivm5prh08pVIFo5D\\\\\\\/Pkl9ZrwV7+rMX9K4StatiXSn2yEribKq6oEA1SO\\\\\\\/CpOWRcS6SWxsjkx+cZTgKsvVfVG44mH9JBjvvFQbflecgQXa0xfv\\\\\\\/wSPcZJ90SAZr97z6FMiktFQUQH4qKEbE5jIrFoACf3I0r2NqjEhvfDQWTXNHnS9AFdHUJR2CsFRy+WOJlYsxATeRW0LOHfHhjdP\\\\\\\/ONq42ttKadTqq08OjfsxhV4jRtJQ9FheHnXS0nT+VrsPrcdTsunarNaj6v7wIWqIge2kxAQunM10X\\\\\\\/rJ8qHHh8hn43zzuN\\\\\\\/W7Wp0GEIHj2qu5HP0zIoY2Al49lsqIO+bZsggEwG2fypLZE7t04RBQuh0SCYZbmUN2XEVfDFkdP4YHHrPmrxgdu\\\\\\\/JDoNmPGnEIwwpDp6omHu0INXOYdFq9W0XlqE3sIIKHkNEXMDbE5sCi34ZsgnOLnEF41kHgBczascFXa80vUPFv5qIS7bis4P9NTiMQ1zPUvpdegRejXp6KLN8m8dAar\\\\\\\/r6AluRJ\\\\\\\/mtjmG\\\\\\\/cek+pc3zc9+9\\\\\\\/MEd4lilW2IBjsm4dUTCpDG1Npx\\\\\\\/fWHsD4TBJc7wGJHAcEuen0HFoGy\\\\\\\/6r3dsCvDukvUvKaQdKkaTYRgg1JnHHv7i+5PQOZjfNEeFMYw4aaPWvwawxaaoLF1o8sPlqiWqUa\\\\\\\/cr2O1TdsCqoI8v2+BYtwbGmSevHP2ZAexOCB3zn0YqBGf1frJQntavPNTUPbG8BeztAdMTC9Bc+U+PetkzRGEoTOn0SefzhNiwpWUCz9JJ4NDgzXfD6xnUEHhcmdv6fPfeOF56ZlCq3jslCxXuSiAWGMoS06Vpp8zc2l5UyX2855TK+KW50Xdo8uvEt2wxk3\\\\\\\/OZOZ++e3HB2NkT+AkytLFP2nGLRHDjozDy0iJH2udN8I\\\\\\\/oKFvCoXAruYReY8NaPstxco8AgqescKexgagKVm4EvNFPijJb5Tiq7QZ4vsSjEUlBCSZaEdYrfTkhgQBk5VARhmuU0Ft68EY52YTVMlUmyz8nYxEl1isp\\\\\\\/13HMtH9\\\\\\\/wmv7QxIvZEoHd6tFS1zUg1Zi6PgNYePh\\\\\\\/ZWB92L4oEbHYd8jzyGeu1WZMW51jXXRb4gZDL07NJleGntZzA72GCNGyh0GIb7gFCqQYQfslSexkUwVLM6Edau3aM\\\\\\\/O0M48BvEGIegjTtZj6FVPII+rxH4DA6BRtmz\\\\\\\/1QWkM1a7GNuSsEKp1l2w8iKDQau0uDKZzVBA30f8u67HXlemzWE1bMH\\\\\\\/GIurIGHuNWnPEGfc9gixcKb0Y7HJALq3Alk65Yjp6+ozaWPsOPws8iut+TkvrFftUE0CNpMGFDQDQI4NYXmjZL+mlJ4HdtGM81hMsZsnjNQ3jgUP2spHDz\\\\\\\/KFEOFQriDpw0AXWVVWlA7BUV3x1uTV7\\\\\\\/cSgg9bJ\\\\\\\/ALifTUELW2OSwNk+ocB9ErIDISSCEfCUrvtr6tImaeXQWMHgzBqyEXWPOypLnSkM5mgAurg5\\\\\\\/\\\\\\\/JczY1x+9n6MPtYZz2TlKK1poar0sGB+yiTkYcVh9En6+Sqwa\\\\\\\/aNROVj9p1RFAeKi92pDmlO5V8lI9JyGqyIn8SRbkL+e9IUF02QYGO1\\\\\\\/9R\\\\\\\/x6xwYlhwKSuRvl3NC64ge7OKTSKRNcpnEp2i5HGkGsxgMzXqOi6UkqKshveu6zwC887zXFrUdnPeGmumW+NGFbEMqHYL+FUJZLbHne5LQgohRU\\\\\\\/lSRXiNIPEGlKcHvDxTa4C77haV8\\\\\\\/tX7HmUcUnCCZaHWRvv64qdrbuEKyCk9SZ9MHKdes692AGNrU3tH9fG6tcHNXg+iKSVmW\\\\\\\/xbRGhXskqlYU2OkcPlFM4qTstHOeS0+YHGQHqUgJhvDOa\\\\\\\/H1DbnXAuzFhPG6OHrPDD\\\\\\\/f3Q8RdvC5ZGTPyZ4\\\\\\\/jpCX7AQrCkU5la5H7FImr4U204uAV2IFnmMwnV9weasPKSv1I4sgxHYntlxrLkvWXYem8F0WqbWCJbL6b\\\\\\\/hZNVLOlUrytfzwA8nk+HHr0mL6VxIawsa7z3RVuryn3WatFIA\\\\\\\/faP53XqTh1Fo8i9PHOojo2b+0c0zVV+pKA\\\\\\\/CnJbRnZozLOjjsrf5apC7jPymfgqCJE\\\\\\\/tpH5HfDPEpdJm+QzMD2zElnXTRfdtxQncGCRMBcgKnzZRC2oSLb7LTz16sl8IQgMSwOf\\\\\\\/QGfPRbXgHCdWOntY5s+Uf8zN3oTWW0W0E1oKWj581YBvxqXo85q1yTwDkajCoFfjE0mH3flsxW4aEIj2q9zsdSPaCetN2kEandUYr7o0481MX3ZPdFKV2UBH60Udocnxo1wqy5d7+jp+4gUTJBdKRa0j9s\\\\\\\/9QkOHGFaGZv4oXGhY7Y2mJ4TU4xYXrLbfUg6bybDBwVu5BgP6lHO5TT84S1fykUbNrjPKwc1aSFQDu2wbYB8vdYomFOtC6rxTEWEFlzzvqE5RtFeWx2ObRSm\\\\\\\/xxnB+8XtSsRUXb394G34ew5bxQWxDgxR4IN29Nq8QhiwSYp1sbF8BV0Og2MIIsxHrUa2nKxP1WBX2g005PrzL+ZqVqz4Bdmt3iu0RtimyO++uCz8l4vD9wVL8\\\\\\\/P9rcozcLV9HqqGSHxs\\\\\\\/3hSrIasg3UUa3Kqz1ha55BCT09TeSPP9a1EYcdLtynfa55ugdsyOQByRFuweGsdWOFoL6MR0D0ozDrzIetj4gkot37szWYWyfpDqcyb3ZcXModaszIN2AaXLaBNbmkeHZIyF8m6ohqD9uEvFgcUfDHv5OE6BPaoRUSNaQJ0qIguSNDnLDRxtaUiSRWnNALfDrT1x+5KrCWbE8g4c8vUaJyHX4D9Fez+uRx9+g35LWCYkn+FY3kEPgB3Adz+gK5kU32V0A+9HbRuoRdsbYOzUYq9aK6FMtXhPSROKyQ3CBqt7VwwStAh\\\\\\\/z9rRbZXXdG0\\\\\\\/XxzgFUvuOFCmwIT\\\\\\\/QVG2stLebv6ojPh3\\\\\\\/OJ6I95t9Uhx4caoi5HPm7c9JT8roZUxe5kboYTxP5GQjnjDyVZ02eCYr4kon+t8MknX5KimTTNCnXAdsnS9+K9prbcfmpsh3gje2x0TlHYpgy1LRKiwfDZcj7UoE\\\\\\\/DTgWMiMGcDvHnUQhs4EtfFoYQc0jvnwcqMMBSB0EqWUbpVXP+yvZO3jRyijH0wHC8DA\\\\\\\/3RYH9nywCJLRBkEWiDdTQ7bWGtOyYBIS8SpIw6RtFgYH2ccVbGTckmdQUHAis4xwb+rx8lJCzl8okzTRVbuiYiza1Z2eWrxSyeJmMqaLwf9JmK7j1UCoKYYJhV8wgj8g2bGMuVo4xBkRntev8nCeCcVe6qUYQ6RmFV4K6IoFyKNd\\\\\\\/juwq1pwX\\\\\\\/P7+5ju51CKoOTn\\\\\\\/04dKZm3rXjdjAyIZy1LvcJTcDY6waxFpvfROxUJmh854f\\\\\\\/\\\\\\\/jJWEzbk+Wv4D1fN5mXZtl5+nBM7C8xSIz+wTeyPlp4PY1URRIaRuiY+oMynsiTAU\\\\\\\/8VL9vacNTUxy3B5P6dSaEECZprpfb9krrgXL8s2we5FrmKHexg7MIsZspHH85b1t97pk4Q\\\\\\\/asuSrzdCNSltMrsaQksaAm5jCe35h\\\\\\\/vLFdl2OkqcKfwn1iUGCBcreKETxE9rma6U+Fkzt88F0LVLQAYSGCdxVRJRFhEiah6gJ8IT0RGs9\\\\\\\/kDLLC5idqwDPjVatQEsoS+JG8t0Ed+3joXaLaP7d20Y2MksaE9psLyiMjEOUPzeaq0ne+EFGbzrUysBuSDv1K5n3eIZRYMver5VwqosJBzxlMNU+e\\\\\\\/Hzlb6RuQIdOHys6a0CGluAvvqco8yAgWdJwYV3qMckZhk9upBH8lIa5eZedtb4Kge0D80xd1MO1J2De9ufhbtmO+pwfmqG07IzVF0A\\\\\\\/hZAGGY9AmOd8zayhcbOLU+opkUBF8PoQ1lwNWk08s3L\\\\\\\/ZJ+zkHVMS9sNWQVZaNPiaPI6bHZE6m9LKgSK6pT+hHf00J2AGeopc\\\\\\\/4n11aqhv402Y5ttL5Wo+dMb2WKHnHYnJ+wDFAWuK2wjmohz6QRxlHvk+R6FoQCY+IZwqwvlW\\\\\\\/L8FaVkOeRB8gKNN65QsHeZJqbLt2NHm6PZqt+DRSd\\\\\\\/HfM3cs9g7h1ZQbXOuHR1+dcPWrNs6E3x4r154SKk6J0os1eUOk3nSw4VIc0t6xEMDdGcCymwqZI8L9K0uDFKiZCjV4uiStUZDdFFRn3u4qFLAM7r1NNxkWmqDH1zpjeoGTiMRyaJoFhsVSDQc+0ZYaRn3Pyz3eu\\\\\\\/LfgyPr\\\\\\\/L\\\\\\\/s7Ad9zBduHeBxj+Dz7XWSYh03zmp3k6XNUZpn2BWFEjHCXfImctjU7fS049V9YAKYSfldFVs2zBt+8htNFbSq8OjHatuVv6gdGChp\\\\\\\/eGiCpZgbSOpJH0cBljI7jDv70a90eWwfyh9ewe\\\\\\\/4ow66wkJmHifZVdP4bKyFppAhgAKhnaHtJ6AYfjvdboWQE6D7qJGb1d7Ye6pt6umFIcOWaaNL7nBFLr52V1mJukOZgOp+yXRXA9pFO+5dl0YbWho4mddoOHrIwCSNIKMa+FxxM+8mUxwaTfa3N\\\\\\\/Q73oBoLN+cptx7CT\\\\\\\/y6hSinZkAAnRjDtjgkfOaPLn6PEWZHnNLw2uE5bvXn4QHqAAvC4GMmA\\\\\\\/Se3pCF2QHR+OKdG0MCfG9\\\\\\\/cLWmp4vcH3FWPgkdm7S3aO5D88vB0gZfzlI8BdTrl5PD\\\\\\\/S6eruIanRwU7G6XJRjnK0NXJi5FUw\\\\\\\/3Mvl0enoPi0N06kxSxmcZ2llpEmv\\\\\\\/HVyW02snO8zelo9Ohp7qmpolnjy7yf9sr0zbM+MDof8368EdFvuTOXtQEvaaiU6BK\\\\\\\/3aqq5TOmuVZ7RmkPGU6X31efW8AweDtOpihjiCRhzoua4D9xlHJS7VkhAetFWMGHZEpNhQy\\\\\\\/UdlOmxVyltfFEp+j+4QdcCmdauU1MMHINCUPnyk2iA79SNbfYMGWb5b++ITW9PIKmCtUFVJwxvY842yPwFrpbLDHitI2zt0QAuZKRmRvcu\\\\\\\/cC0JEcM2Q5IFGOwvllkxS8RcbpSnYATqMO9rAdYmUXvvzFGr5Lj0l3KAnhwuMNraCd8wDyCMQk34CTMrOapZYHFgrMdrZt4FJbXsQcPb4HnvyqKGjfP+GFUmCHVqWtXV1vqVhh8AzlPKV\\\\\\\/kkYT\\\\\\\/mFB6YBg9SaFYxkPTRM3dsUpf+F2keHZ2fCEbcoUvjRFiMooUYz0T8t1HOOCEPp3BlLcUAkOCBCRLSRBcmaLXhIbHsGisxG13Z7XpIhFXJPLAxoc\\\\\\\/w54quTVU7L9hqR58mkJYPPSJea0yI52rfCBOvaAWnn7kwV9aB\\\\\\\/s3poSpn2VIaGXWCO\\\\\\\/EW+KUC\\\\\\\/2sQ6Z5BhfTMFQcqxYGfYyXVQ0LguTJ\\\\\\\/0ws3tko+kd0CiE6f5fyJM0khgzjuMPv\\\\\\\/+LVl200AcZn+5An8OLQ7Zi\\\\\\\/q7xuWgkzJ+0pkTKT1u22jEJGIGbVeTW+sVvLbpNxbRjZmdPO9lwhdV1CUlJp6edqr1VPbQ\\\\\\\/dvE0zcuaTzSGzbwu7nvjw350M+QMZnN0dfqCzIoNGe3GyD0lA\\\\\\\/YuFLA4BCrOHWL56TJO\\\\\\\/IGkoVj0bg56Pwd\\\\\\\/KW9n02erSK3xsjTqKTu7vvEhJ1TuUPKP45wnsKObOOYpdK00M6uHnH9Abcv0C2oqevGjoXxTlP90RjX6STReF\\\\\\\/50eCRGjomT2Ih\\\\\\\/TqRkpwAXLcYtBG6BngIXBNTziQZYCJtAYOptvUiD+DJq0eD\\\\\\\/1kBTfAuCxZzSZAata0NEPTrzt9pv+vZi4I2uPc3qIzlDZkeI0j3BKO2aWB0LCwHKJrV\\\\\\\/Of2sRXpPJWFd7Dsfs2chCVt5ztazLnCB2jgIkCR7BnXr29JiXxByli5\\\\\\\/RhuVymkUTbsf0qqa7SSQLB7W1fUFy5td7A2lFn1iY94UUPbEQa6ubAQANzLsEe7kwqAoseACjQfTKvHwNrBOQd3MxqEGr1FoFtqflUjaYnfo7QbgibkdxWFj470juGeqbLp\\\\\\\/SEG0sumCBJLPDjg9T6OLWZbNTMC2zENG6GbZq5M9R1tgc9ug0DhumswtO0MnDjJPzpJwn5Js00TtrjYS4QLMRA8XkajZPAkV\\\\\\\/JlyUN9hpf6L6XtZWHTweL5HZb4y3RZfuobVoF0P3HRVHy3m9BuBWY2Y54hta5dTERLkr3gTJyOJCggrsvvavjQOPnWn\\\\\\\/wY82roshMxpXEDTMxrEGXn+zejcPgdh56C\\\\\\\/U7ewCIWdSgpAP8ZcO7k+kxf+drnNiZiFKX8M\\\\\\\/jUJwizmHhLnTlDp+6B5zOwA4KEANw22IX86oAT\\\\\\\/ljH8SwFauSUAUYUONIV1wLawJhNE8AceV+RRi7RkC3yl8CUvv2gpzDB42R2a9HtwnnJuFv6zMHAyxP+X3WELDkZl29Jb2BjC81zOxNTNfHFkKDIN7g7tmv0gwuuM8HHNDmuSKbmtYYBQbJG8quFE\\\\\\\/4i4Sv58FT84dl4PSAxblkIfjrh\\\\\\\/I0pbIb5maJfOgBWH8EZ3swJvcoUBKMy5eVWMBcDeFYzLpOGMB2kgo9w8HBB9Kx3N1xvQCPr\\\\\\\/p9kThYCx\\\\\\\/haOl+Ep4EEryDIg5aEuTKHw\\\\\\\/rjRvkZBn9DyCfdkAV8sAM4EpgGBNjiP3erIbSzIvWogR5QbjoEp0jS0sj+4NuVJjjc\\\\\\\/Vr5hnugoSf9x2AYO8ZPgeQB3jtI4WaXCMkJuFCYnSQm99weFl9ds3WdtaBPfHK0o9LIAZ4erLVWvwejjVlsB8oLgcx57yqVG3DFaZizTNl8KXqhCW8MmxWw\\\\\\\/rQAW2XyTbewWlvPOAuCx9boeeq4SfdJaa+YqTqEucYGbJWVz0CSXWlw369HzPOxbzVMILJHzs3cbX0yM5yUQsaOtDWeU30FontQ0wp08CuV\\\\\\\/BTPXLLxniVbLTTkR4\\\\\\\/VnLDRkqr06FHSHzZYFi6JLrmXBmgpiiNG7UcbHBpLZa091iJZ1gLSlEaVUpckr3PTR+j\\\\\\\/DSAIdcbTIRA4\\\\\\\/vo+988Dorhxa3DLJ\\\\\\\/Q\\\\\\\/iAEsnJsPvnvcHsB+ZHAkgZJV8x\\\\\\\/BJtO81MIhn6RVB2vtdKrTPsENYNCvSG4Ay3NjDTKCFKZfE+0CRQyFDWRqpbKO+WhP7SYAe1RXwZ7X17vSu0or2CYv\\\\\\\/XI6kS+pwqXvEw\\\\\\\/Robf4AcwGNXVoma90gvwPuVnuKZIUDlRdvVbrqYJ7ZHgQjz93m7oVHRV11esMvZUgG11higRwSwv+1Yd9hdSOOIkg9cQIik+4ZS1W2hCRhZuo+BzoKA9JZFJYacho1\\\\\\\/+lUTq5spEsDSVLZLHIqzGMXzpXrcS3rL7K55LJYeSTh+1diMtjFbd7gZc4EjMP4MYu9Es4V\\\\\\\/nfjwysOedvUsrSudBtlVte1nOL11RrCFSh6Z8DwFbg2HulCSPN8mPCoX7EXpOLuVmBN2qb6fHsbhCsi6GSoKjczOIvGzxgXXBjOyHu\\\\\\\/hmV4AgTMTWTSTveeZ+JaTte3lWoffm9sWvWIa9PDZA8zB2aePrSiRseRmaCmybe1bqaMXz9\\\\\\\/T0P69igcaqv8ovxwogMHWfb+YOVZ98M++DLTTGTFLHU5E\\\\\\\/sLIwHoJU8nkaLYqFElf9HpUPApW0go5m+jCIVCBKk+\\\\\\\/RS9\\\\\\\/S7M5PGh3AsSswUKh65DXVdmyor28zR5ghWHmmKLY16pl3rI2UjbGhpsfgF0RPz+JAgR23S\\\\\\\/x2Y8zLu0tLDFBOxOdbOskCOERSRdfeWQPDv41qeHsyCElyU46cxRpaw7CyCJkKusX1lLVLLHRd2AEPrIXh89xkZb\\\\\\\/ZA6tL8tvOIhmaLLVXvmZq9fsDFYq0J+RSUrT490y5OkADeCCw3kqH8QICzCbQw2x5KwNV26gR5M4xIpGxUoJc0G4HpFBLOYSNxpEN96x5Sfntn9d77kVhltEsEh8r6ILxI92TP+eP0343dmg01Y2SjgOx8D1isgbYEi4K0R4XTeu+KvaXbM+AuMhTfIBeKrcGA3AYsZR+nujFnM++O89P3uk9k84e96fiESdjDy1P2heZT3upXdxB02WWCKd4N7oocClF69j8Tf\\\\\\\/eWKNH5hdGyxwu2irTcqJ\\\\\\\/Ywer+xvzAS4gcKD8Lr371TwMKf4PoJugSB3JC1r7IsqxOqtLbpk2+KToLC9CReYj0eeiXHE3HHbQHZh3Jqg+Q1aI7rQfJJiMjivG\\\\\\\/z4+FidqRZjKnhT3s7cobVyZjqoNO+QHw8GDUafdupmzL\\\\\\\/OZUs7ngtJGyZVnKZOIXkMTQuhYeYdWCp8oV0aPTbVlnxDtiNL0keXE4J1Lgz97HROt6qEmJRgbU5HPvrzDEml7RYaxZE7uqjLxWn4Z+TgIhy1gbefIsLfqm\\\\\\\/GBu4DmTeWC2o33pjntxP9Lb4s9Bf\\\\\\\/Fry9HN5B32GawYK3TmZyrXEZPN915Oite\\\\\\\/otFGeswFhq6MZvFlQZg+wU231QPNDE5uQpp5Wo7LjMNwodvtDaGjbNYAA9QnYVtnyZMU3JUS6kRwfRH09WGPPLJTDTuXRDIQ24B2DcJmBvp15DZN1elGFNTyibi225+qJWzSX3qQQBPsSEvefyIsDxPYv2lzob6lYayXKv+Qad\\\\\\\/SHxlFoRZ9THm2FbVp\\\\\\\/i03hXgO\\\\\\\/ER+v2isZD7Ac\\\\\\\/Cc4QwzgRCxZzCqfbRyGNsg1BDXNGgNT8KRYJx8WLIf9AVAbFVX8NXSriWpSQfjXtTnUzHaw28+Ft3G99il1jXSqP5UiMsqsqPxbYIHiZFOhSD0GmTaHQXSz\\\\\\\/5ijf0Pn8EYU7BUsgDjyTYFZc0thECZJKlbgxWvYhXnfnr0bTLzsjJ6mqsWLC4vvvCEuyZAzY3UlRf088yfiNiZtvFP2RGcxDzx9L1C+VtOKyD\\\\\\\/w7fAsj+tKSNdAWrE4AAoPmpZVXunpJjF9KYIFP42DmgWnAi8l\\\\\\\/vu+yLdRNgXDFaBqZbWZhrZwWeZ12fLqj8NyxHuERrHpy4X3nm+h4tt6SxXGscPGrfHBhKWsVP\\\\\\\/2Z+ZP7gIgYug5RBkOBaxkrW10mF1zhTeP5P6\\\\\\\/RA8owcsYD2WIF9XcHS07wSn6qafKYSk1kPA5WpPt7g\\\\\\\/3n7R5pgnhDJp5+2WwgLZmVnoLxXJvKgrj2vTHxzcD2xayyhHZOsux3E6h3bP8rjmzo4I6+xd9qFJitbHkf6JqqsZkoo5zhWt4ir7mpac3rCZQJOWdwzY1XD0Xg7grRQEG9q81Bm8BmOgYVo5gLKhnr0YQnYhZkZIu1HJ11yIVMm1TrLC0ZrK\\\\\\\/pFLV3HjCM54Dc5n7n1W6P\\\\\\\/E9TnfWf1H7igik7bZOZmwAJuTqbNOrFpcnKfZaJZLwFLssIFWC2QlGE6co1xuQM8v2cUNMr22kK6oR0QRq3Dghds60H\\\\\\\/cfrk3MZf4QZ0Z+aLK3tkNZ9G0JNg120yVrB5fNcvD2ua2jz32hVJmEk8A\\\\\\\/ML7hATL7lTEV\\\\\\\/G481nWcJMpfUaFDRz4Be5nHCq9aueVXBevfElIP2Dt5L7J03nIMfbqc7f+tR+mxoiZM6ohL33hINKATe\\\\\\\/MdCCnd59jZZHbWwMuD+17\\\\\\\/4p+sJCMwlr2XcrJ7Tdx4EhF4yCbEv1kwYDOdtCg8CVvmutigI0IWBF9uxEnq7kED6lcP+VkwTv\\\\\\\/MaSdcdsWoscYtqN+EU6+6sBvqzjR+oKf6t5X4xC72ht\\\\\\\/zMmOaNU5PHkUA9mXammyETOhlrDwaoEwmdWx6UTDFaER+7vyEUCiVm0Rw1bjxivF\\\\\\\/+jYHtY1cElC\\\\\\\/t2toPgBxBahOiwBkJAsG0Db5W9fXAZgUcyq0VOVDYQgZUYewZ2VKCCLi8\\\\\\\/zddQ977gr+BIgQqCfzZwpfxLhREZfTsaleM9KWq42K9LxsTB7mex+NvMqTr6afGw006EGmyVfVlgDL+G4MEXQ2NQTphxctw0cPD8a6aFCiNLavxgGMBYk3kye1RwBMHw2OAlz9JPLH35gvf9i07g5YBg\\\\\\\/2hA\\\\\\\/ZJrUfBGJXheVXqup3uptM\\\\\\\/WSaBaFyYjSgxR8GMyYm\\\\\\\/550ic8PZhZp7+qcI20IuBFcG5cqINafXM\\\\\\\/Oypm7mdDNbPAdcYeMQ82Agm6xzEL1w\\\\\\\/oooKEdD0uu7E61DKmU9AxT\\\\\\\/VIFnj+sowyuNd41ttw3XJUEqOsF47+y8fZZTfljX6w3vbThatLfeUOCzjjNOTDFViG4BmvWsgm8GXvpV8beKTQYfDfYfSPE2qKl9\\\\\\\/CUDYOLF08zFTtz4jBS6IVWIM17FJ4vYSVc2hFzsoByyVcA\\\\\\\/BH718TNVEdF1Pc32+bUYkxVhqDctsDBeEGX3ydpqCYtDalo7VDNTQwEz8+\\\\\\\/7tODsd+rXUuPCaNHIGD56cHYR1+QYbr1RRCTV2cu\\\\\\\/UkUVAPja4C3sdyQmp43U+lJEfd4DE5z1EvO5u2IJ3xFC9N1IZAtq6LGK2goR0JzakFkpq12kwkNjGMRmVMme3ZGLLISwybpXYEJrhLn0\\\\\\\/yWWSgVfEy1bmcVuNxxO93ONWzy61QjtA+JSQ7tcRcqP6zJZOgBjbAlNGF30QEjpoTmKsb0FjhjCIHscKTt7Eky6jxPUCBafrMXPtvqL414taNkVc2AAWgQWXPYMfBJGO2JzWyQoFy4sR2n2wBr\\\\\\\/NLcszL6l3oOxftbZ+qPc4wjJbIZ+wYCG9XqrYWzNTl9C99oQvM5l53f29q0mcaKbf2TVKQ5ogEuplpsHeTtG3aLOBrh9eRLs7FGaNGpUcySO\\\\\\\/uEGzbK0Bzd69odwl6xqmw4REvqR9dljFK5hxAiXcRRhP3TgfTc3dzy5d09ycymEup6dvJG5M+F48a3LPy5a21DeFj\\\\\\\/g7MeG\\\\\\\/0VBM9L2wIqIs1shDbIxDLz4YIgkrRNfhNF6UHslnkgPZXZS3GmLdDWGgv777jOpxHHBzcPoi6Y2KU4bvHrjM\\\\\\\/M02M6kQjih\\\\\\\/U7RAH8Nonav20tGezg3eaW+XiCAswBCiDehXy2rQ7x7bsScsOEKnZVjvxdrrjMq87tNqQeWX1U8xrl7GEDvMNRj9z0xGVB\\\\\\\/urTbNew\\\\\\\/tjhI\\\\\\\/npuhBeCCt++BulN3e1zqem1yCbBcUnAE23RTwMkQ0UQparOtsiogVUb3iZY7Iw3hRbxNpKMi9pfe8bPRxZioyH74SDV7rvHmLvgEOkhFvIvtOyEt+xbTAj+MlHmX54qGSH89h+s+zPimpQEgTtnTUNCz0fkCIG3uF1f\\\\\\\/gl6LZY8b6rQ1FoWaow\\\\\\\/mUzTGkWsh5\\\\\\\/RKR4bCldVdDgCSnCMb6Ie1TkfR0f\\\\\\\/U2DathbUWNErKsDFkOVrblRDmu4Ap8UGvS\\\\\\\/oSyvburK29G2Y8xgTrWtdRCWx01H\\\\\\\/d9WJuIAbZ3CZ2JPpVdhRNvuuoCm8Y4p3HjUwjQlEcqXZaIiCpcd2Z586hbyCEuxETuV1i+iR3lYzSQBGoDmA0wTEkvKQetkO08yLK9a4WzYlStiK4vFJRjNsajwnYd+JrEL8eKFxm33Qv\\\\\\\/kuwIwEK89uwTIffy6Iub3\\\\\\\/ynJCDECoqRgUq8JZ7KpcenSFMZ2+uamAJFtBsVJvwRXa6atHKlkS+Hul+FTE8BmsZgZFide0hOoBqr4qusCAVYAbpiG0EQcgn8lLU3WpJL2LdM8ASNQOSKfN2797\\\\\\\/7VZQ8rBpQGdcizwCXxgSnLuKOFb0OqRyjvEYxq4P4\\\\\\\/v5cyJpVCzYaWV+W6VTw93RgQtW\\\\\\\/X+mRpTKVqLl7\\\\\\\/sa\\\\\\\/jBkcIkFvtLcZbPCw7FKBFNfmHvHDewgpmL+GPksqPcKcIs\\\\\\\/j\\\\\\\/kHTCthq7pgVgEpP5pspBQpq6azTJ7fAx67F3wOsGqIyaLajlWqa4AgyZfiu5K+kSJRThlqw4xeLrvysrUWrE10WEpWRLEc+5WhO4yiu4IrWAancXGviNlMYlwoUVfIUCgn6fI9Mtg0GNEmpkkKuLR31ooUyNWnwiChTK0wTHEOg6jc20kLTBzCKAlmflyGNg16jTk88ezdR00+sl\\\\\\\/SIUqrHJZr7AgKwjJNqDfRgx4puBUNFebHwQDpLzmBDMAprERS\\\\\\\/mH\\\\\\\/exEjiMg9OkCRLjSY95SG3Vj3KkLWIDtC8qL2ysEWKz1tF8\\\\\\\/boqqifpnGzELWXC5Evkslxy7nH8rbBXBsDzuLOHyI3kajsZefQ8k9n1UuOHNemgWihYAEYjv+ZnGcO25IyWHb9KVA6wHzM+YEobVwBOk692dVGVCsrtJXo6tq6m1wK+9vyFxbZa9W9levhjeHxs9xOmPEFBG9v0tyXUE84dRw6Jk8rnku5Boeck3BL52Qwu0Cv\\\\\\\/N9w4QMNigIbkFDXYtZEpUxGpV9Us2KalGeoY5KoURYyECF1JXCAhhLoLCmcNZPGeNbGKGN15+MHSPB5k0ldJiszfpOhNOItINtmKEs0uAIY65mSG5A4JJnjEQdFAkpMcSdAjhZfqlRn\\\\\\\/ulKgprtETdlkAqPmHSZh\\\\\\\/PAFtqC+UEhDZpMdLtmRDk+zGghjFtZOnivE8PpiEZXa6n6ltFx\\\\\\\/zY0cQrJcZ+pL5\\\\\\\/0mEdpTvOK\\\\\\\/x\\\\\\\/fApHrFIKWkIrRL36yN+FGz4IVcxqMF6+z24gMrN\\\\\\\/t1MYVZhExLSRdXMlq9BhD47tgmTgZIPEDtYcIJl6z3UYwvcwPVAM82ORm6RzNd+PzA0mtvkRH+u+dtP492g0PQ0JFNVtwiNYVqwM8F+JMDU\\\\\\\/6idR8AMw2c\\\\\\\/GozJagb\\\\\\\/n8W7Y\\\\\\\/\\\\\\\/6LESqtVkGFXXbZZaoHleI4y0bQMnCs8xtoNYj4hfKxLq6QlnKKf6SLwpIzMzAswAjEDBqJGWrBqb6vhgnA0CU+pEJiy+VW8bny+TlSwjE8SfOW8Rte1AgdYUpHuN2JvURTPu0X6bNIef04hIU3RXUrzPtpmQFSpydwjkhH+QiZE78x6pAPNE8EM7itb5h2oIzj6NYcG7TkyD15YHGE+hH2+vT\\\\\\\/vJkX\\\\\\\/kv+iRLZ0o+jLXRaxl9Xp5VYOiML3IFeivEmMlJueARcC4DKnUXiGgzthHIgQUmNZHqHoEOndMBtV4rr5Xl30\\\\\\\/YJxvAv0ser03uHayqEFhZcvuADz6A80nPrx8gfIpy7Y7L257JLwEyuWUlRDG+qzZJZvqW3lvH7uDrn59Tzd\\\\\\\/2g3FspOA7Mbz+2oDwjI2IhbwsSFAd8J64QMqWmPnT7oFmkMdCxICwl6CDbK9+gPRscYcJYvQ0r\\\\\\\/yp2ABeWhM9oq\\\\\\\/63ApsZhSWjLS+Ur2W9wpagFR+NGNs8BCmZpUtjBfomjALij8bI6UJyINbdvn1SZeza7I7v2aHJmKsJi+95inZOekCFMVpRYlOSfzjWv+8BxYq6BtndPY\\\\\\\/Dg5nYwyBXj22sEVs3VdraDEmSzM5Ba\\\\\\\/AaXTiQwEIMY2++Zjf+7DvZg7tDpLI6qUhziTI3CFQFEfCYluW6+hl0QfLDEVWL4DcOP1XJn\\\\\\\/CPRQ\\\\\\\/FGrd72t0sfUZwMFQ\\\\\\\/f8K2Xkn8408HdAJ3KQP2V3EOrVvxA9AfvoePqSkilXT\\\\\\\/+cs9OBba6us3Tv\\\\\\\/ROiM+PD2eYGIoNkzsBIr7lXXxmAxD3WEJrqj+shtZe2UxitH7sN9hMe2oLzO58PfFgTg9i\\\\\\\/C5wNFEomOAnxLAXVVoglmXgwLh\\\\\\\/BdChTjppmq3+HED+9mS+AWnkAVkqupfdwg3RZu854Q9dEhLk6aHXphYMkDBxm+Zn+xsYvFNdEzJPzvidGc87V5WgHv\\\\\\\/HOZDnQ4S\\\\\\\/QDniv1MDOOsgIwEJtyZPPa3HcC7p6GOQ+eVhXcKRJiT0GmrMQLcvJdjCqczRQqlqeLde3JAwrmB43izoyApd2pIBZCRrIH1VkMO0KrrpdtUYCdcMmPIbd2hBsns5BP3kIAwp6OPeKBRureEYT+4p6al1vVl7MKHetFCXLXJeOO3Qv6zUaaMMwCjPh1VFVPKKQ3ifZam8yQnXkfBQ09TvfW1T4JcYquW1Bdw\\\\\\\/R4Ti+ToNf8LWDoMul2JMV0TgFCiICrgUCtpye10yUUaIuSIbzlmaFQR3I2ivz8o3Zx+vctDSpnJwIB6DzKZ4W\\\\\\\/tgQfk7rDrzcH8\\\\\\\/XGlEip7Ekia86p1GK77FKVzkhcfQrtPDibiCzWGreWuPYb+CNsOv8dpNbNHqovFnyE89t0VK\\\\\\\/LeSBSrliw7HOoaVfOFcdHc\\\\\\\/sO8+M3pgGZ\\\\\\\/mJOnFjCmFVOHSCJFPZG7Kwrn9J+wQwztmo9tc5E15\\\\\\\/ruO6HO2h8nFCuJIfyBsNqvwT9jUIrdurRtP7H7c0Us79+1mRUvgql71jADYsrmdmyM9VZf+oF+GvamL6f+iuGh1\\\\\\\/6V4N24DmzvpmBCpV46nTrESZEir4qkBGdwcJdkEw9c5dZFT70BFYwYbggl0A3N4J0pKU9oAHne5aXmF89azvwD2uHKrX5k6isgbUg9XSDFzcGxKBSTIEv116VflNeblEgO125XCoYnW+s1cup55St7tBbsixIXaHnNouXd0KoGbXjwUo7sjO0UzMMQ7egsHkEkYF0FDm8P\\\\\\\/WvZ\\\\\\\/oUZ9OTboRpk5qzD4VtvBCfsoOf572ym1ea\\\\\\\/fbt2Rqc+ky1bnOrmgKtOjtZWvtzb9k4hKlJrumNfMopl4SEkbWUPMLQyMN+a9h2e6v2M6myqrKTd97Be4mJ6QRbvy09oCtPvrC1IwP6bpyRSedvvQrLCB25UHzctteN+0Pmg0BopKkeJI\\\\\\\/ULdN1jQSuiFSAv4bnnOvodHbmyOZ6+cjVlmedkbka4IB4nG26CmEK9stq1uNzcX1Vmsj+VjYB2oALSsT1o989lCgy+Hkqhlq9w7VmCFLyRbHBg1qeNgbfavQaUZKmiazzIxIxO6t\\\\\\\/x26dnFGM5EoODQFsc6uMWAtS5ji5YzDCDv8QwAVAlaz4P079tY+MLrHITYktgmpUbsVYpw7\\\\\\\/NUUDDvVxGFF3oaeIsMglfIfMN8qxH2AK\\\\\\\/cPlG8dNI1bfSSrxgPWwMpeS9B1nlTuD+IdDaTeh9Fy2KnBZf5UWFCkKa2HkmoK5Y60VxbcBRhmQ3hU4sppx76AxViJtlocgtssQ22rUnT+w83zoo5NAZQcfnBhPzxktIVdvc+3TTVJnkeI2uTU5UfUp+Y2tDZPORWNzmwD7mnhjlq5TrlBJZYtQ2lxUlBiQNqxg0NZZBLbDyLZMZVlcll78grLzjHJfWOq0JyURGQ21FbiUwzfXr0G+i7zE1GiXcxUy6LPG4MZpsr2d3i2mXO7Aecu6We1djKHO0WhS49\\\\\\\/mBRzaS1km4yEsnSrfaPQ2DCjnE2XI21JJeXGUH4dhajRsQlfD1hERBg8mWfbe7KDHgP9bYFE17VBYt\\\\\\\/8RRghpUHJdKGqsqpsKS0MADoZAqRg8nlMIOd6cUSSqT4l0yCwZj9S6b49VqaUrK1CWyE5pV752rL7PGYeQO1\\\\\\\/J78FDa737DEPCgbHf6GTaZ6Ak5ga+WqXaqALiq3woxztxnacB9SEFIdTe4BO9hxaPEdCz8EfT2rG9j2RHKQwuxHXgCwzhsWZD0IvPEIvFQ2oK+NEeRw\\\\\\\/cv5RGfFG\\\\\\\/8e4X44TURpn14zWNWCcrsVKtkHDBejv3fIuTgxW8sNK1vMpTpdfujDgt6PLURspYgkdRi3KPOCEeW2FGXH+Mm\\\\\\\/auXYelQH3MuDPPlr54MCqkkhOGLVkFN\\\\\\\/\\\\\\\/N0U2PUSEHwjYSIN4zTAyE5dKQ7VUpoOr5CkPpnmO3fPeLZbF1Dc7DBoh44RcVXdtexvfaDcUkwVv77Od3oiRLqxF58cU6zMhqr4BSMkepVBPFNbdmHNmh9VjM0udO4l9gNA+zn9\\\\\\\/3ANZyNlC+IM7IWsjCTa21GcJEa7XVne95fur+dRAUXr9d846HNJZikgfI3JADCPYRX5HWhJV0jG41xWdDgKXuPgsrtzntB8M5mSSEkPlxvgplzWxOU8ZOURoOB4AZmFQqjHR\\\\\\\/EFE8R6hBqwYZ4RVzkW+NwAGoYMotZzcsVCx4NgNpPKUewTuF0EdibgV1seSNhMzx03Jk2pvV4yvXmiKFo13ZV7EXvBbx4cnS+Q90WFpb\\\\\\\/q\\\\\\\/qp7IiATYxUH8qopVwI6ADUmyn6xNe4nSs2VMV2Kr5olJ1UiUszlj7Hj+yVEhWqVTnBpN0\\\\\\\/6iHpcM4B6ivYb9zm\\\\\\\/BV1fz0oTS5NEql1d3AmoAHZO+oslilq9IALLMbEx58fTkQ7\\\\\\\/G2f8XoWE\\\\\\\/9RvyjLACYCeYlaHRL\\\\\\\/Axu+DP6Kdr1jDlQ91BxkrtDSVNFskEaFJ1SdO8+5yHjazCYR3++zcLCKMX2iE4XIb3YaAF6xm2BSiXtIF9l5veMMGG4jB7UlZ4oK58OmfRdzmrAPnKYxll1dbyCypLoXdQTrjIvlOuLjc269SZFOyurATCpWHLgNX5gL0XpPlmvQT9xAlutvRSwPqdkGwXxkEuAyrAQ8A5HpR4fWB8q6LCs11qq4sVLiQkZ1qG5Gvnf62v\\\\\\\/mIxr6HOSZFpvl8H8yGIME89h+mFFdAEv33YujDOY3QLZvruHNvtMCqWqlNAMCIvF4Zu77mNblvFEp1jFltWoZU+iErngHbypg4AlFR6jsQsJdxZYtj37YgZF2+JFRquXOhB6h3wjikkzjgmuKgpEoCPAuG6k4SGxySEimBzz1YlpVaQGK+Q2NK5c9SSEsvuLw7RQPTDZU4Ow4XR8hDkO6+Usvpc67+4QiQGzw55GIpCNlUdjgrVzZUgzCpkvdYXhVy\\\\\\\/8+lgIkJ0B\\\\\\\/8MG5utgSfKJMVvpw2Kub\\\\\\\/C9S9NVBZsYf6LruRMbbFscUMTgVVPz17QvVsfh6ZMDKV9d\\\\\\\/gI0nxcdTUl52nkvMWX4EOS8JzFQEjucqC4J3hEIEtxkFwfKgMXnRSnBqUuTMbVKdi0LVkMigPMpX9vgDXefQmUYSfPkbczo61nIkyMgl468pOjtNd+cXLeaKAWpkRe1cyGE0y27XIKU5tF\\\\\\\/mtpaRM2qrbzg75Y3VXioiJJclpeNCxcbzsmQEBT5OI1MypGIa9gf5Lf1KQCliBB2CK5\\\\\\\/JyHg4qCCHrGujUQEpS24WJ7sCWM3hkSMk\\\\\\\/jmTNE0rYxa0x7x341vGfExnNA9Fj2gWzP7drrVJOc220+OyMQR0abR2u79FVaRoo7z1zh14vGtxXVwCak7GVuoQz5tyyYrLo6EH2RnpB\\\\\\\/OzpTE7NGAeOvs+zxGyHQ\\\\\\\/CG46YlAYQON0KWcRw6nm+ykzN8BKrJ50aMAbJQLFUm2rrUiCjN3lR6eCRRx9WFSZ5mXpzti4w9p33IaK5lZTDkCH1vFP2lV7FPncazdDAvvR381HdJMmNvSUcM8u4hXWen+OYk3lw7bP1pChewuZb3S48CBCquOH0wV51NQAkoCK33ohsYswYXRBZNSMXy9Jhlbuegv3q85l9H58IOKQdQmjHqm2lTYo\\\\\\\/GffpH5daa3FNO9XrySjlGOA1kl384JO3bGQStLBwxn57GCM01+cYYZjqyUFncC+IGq2s3Qfh4KszJOfuuJ3rsKmboAk4uLNDpEoHLDirwYSqgqceOD4EXfypOA9FR5CNBSB7KWZs8tDCpw5g2\\\\\\\/6hqFE5B\\\\\\\/HfbLl+I+GYkCRnmftNCEw4OQHEiwU3RtFiE0slrBsZlaFeSdYP5aKW45M0pz1YpXyIGcy6QcLRyWOgpBRzo6A+B0X8xAH4OdEWe\\\\\\\/\\\\\\\/np16E5MgdnrpemML09ilIbuJRyJBDnYZkSXNbJ15ffJMgca1TWt0R\\\\\\\/EYUtuf0vOGZeu63hKqY3naU\\\\\\\/taVtMX0etSdEPpwiuNUAL9OVMk\\\\\\\/dD85QErJaFvgDElYQ8QkBrs1R1OlfaeVPA5CASQ55jFpGDpREz2p4aKoOH86+iVzWXnyOtwVPV6Q2n2m7Oi2L6gDgDacWgnYleEjWlPzD2eKhJt4FFWEEBppMXN4LCfWsSEQ4LTxXyvENkRwz1c8C+Wb3CKxRQXBX0L3KV1W+9BFDCX7+GjNhcmKFzJ8NukLD7jF7V7Yp7mciWEcyOOuiw+ZDrC+cTwV60xMxrqH8\\\\\\\/jwcjTPImdOIImC35dq5W\\\\\\\/K46Qhr\\\\\\\/xB+4QuPSargHhoxgtHIZy5Y9drlBvt3xX9yKQ8Jl8kaTYHiIVizKuqeXME3N7bVTGgdm0LfIXl9AaqEZL+nNwWnSWfZnhMOwsfGFffWYYivo6ZGej0icDN7DRkElDQgggWULybSKRTi6M2l4YrnZtkJOe4\\\\\\\/eF5rWoOoPp9HxB87L36Nf3bfib119utYZd6KF5gW2S85G7lUAaoBA0DeAHV8EJsvMxo1g4LcShFJLe58JXF\\\\\\\/nE7BSTrh64psInEPz\\\\\\\/IfM4Fd9OnDBv0K3OkSdKE6bP5nfM6dzqCAExk4LfUbX7HT3TfxykZFm5xAIQhuobGY2vVujBvKlnNtcNTHO9aeJlI1yjbd2rXZju48o12TYV\\\\\\\/sAEOPi9USZiVeF7yfkmliKfmGorxJoHGoiIGTvcM1ZpUF52dR++hV5TxUfGHBuEZ84Bgjy2Z9DHypAsPH1SlTLqXVp61srzJQCKHXuVe+wnHpGP9a8wMdUgIMR5keVZW4sYVbcvRAxHkVDoCv21UL6lKsJBxCHoQefphARBPcD7wqwMeRcI5pSb5Gmx7VP+nZgi5SHRv3onLqwecGLBaGqKEQyu+4e9OmDICAa\\\\\\\/HXeNOch1pGsBbKSzz7wNa2E3WFdlfy9Yo\\\\\\\/ByoTTadFA3Y2Vivag0RYl3iYqPUtS6aDguMUkoI0fYaQDD1RYImynf5OPHQ0A\\\\\\\/2925vcGikcEu5P1Fee6X6wAfOQOI0sl6Tn65\\\\\\\/2k8PozBtrmppTjl\\\\\\\/f0qiPk8nSWpEiR0DIYGsNmq3psll8gc1RwY9REm+o6U+gGSzzsQczS\\\\\\\/I83YqACImY417YXsoN+eyt1NWhMR2Rlb5VxZpqn6NhdgoMKcUxOFiRpdSE2JpvvLo+qM9HocKyFJiyUBBvK\\\\\\\/ihbgFCDrbYj93\\\\\\\/oOnMgYv\\\\\\\/2UiOE1lCO0JgDn5l5Jxnkc0xw2nC7EFG+qK6vOlZBZ5qrcyvKxdJJe9qNdTOprS2aqlWEz36UMVqeeZCquVKdHEHs\\\\\\\/v3t8pncOA+bgV+cus8\\\\\\\/\\\\\\\/XQFP7MBwP0UfZKZRDjkI9xY1A7alvolB9ShfvEoZB0qv2Z086K8yZWtwx\\\\\\\/6wgmuQe+826VAHBl6G1XUwU4jadpLelCyxOmxkmYS0RP8cckWW8pHJKpjeWSaJ9rkKusvGuHehZ+TAWfUOBzMfQ9fNi++43oVVbWUSbKpz1JZac606Nf6oQkSEciVVba+9+OKWUvvME2RJN4oi11Px74ADEJWGwKtZQYDHbBFwLXnb20p0Oc3CT2U5\\\\\\\/LnbRNcUmockRgtYgo+e8xFb9jnqaIooihjlk3a8W5PtgEglR7wwv8FfrFsulimNhv15TO+wP4L7eBxiXNRmmeHgCpsIjNRATPsl0a8PiO4Rr3uttgZ1bn\\\\\\\/sayeujJlBMOdV1IaepFwZH8fFcGt0ziI5xNtObt6Fr3ogahYbsNvCi0kM\\\\\\\/K8injqPnYhvAUL3YcIFS2zkyxoyP80Hro4VdDZH8zQq+lNHvdABFdFg+f87PIQ6z0ZIlzrIQ26PrS5cXdUVlhXQHDQDv73RZW0kzR1tOV0HVbQaN+x\\\\\\\/iIdkELrZDgoBrF+J3v09prp0JSL65+d2FqF3o+AFezMygGz53+xZTLwvBKQd5stZ105PJ8373MzH\\\\\\\/5tZSctv3D93DfOzntXGKOWyF7guvqO4Tp3RPOfLQj41YS1dO3Bd+8JkI3KtqNLquA8IASlpsJLktguSdHVtZb0ZsziDZdLiHcXYe\\\\\\\/lxboth5DRYc+M7vfsNcZnlLqCOriJuM4eN3bhxLATEXCiqtoLixefL5p9ZHhYX58DLCtLsWmp1Y5w1L2ysUYn9V2LrTo4F45iSWRm8NKh0njSj\\\\\\\/nZZYBnEkympVcBSMdXWRCO3odwhINDAug1mv5vnoLzC8og6cJnD0PJxmj+76kPlX8q3Tlqzjm90Mitc9oJ8MTGiQ4qGBzwMA3Gbi1pVb1tQ+oSyK5WJ1nfnlQmSYIRTiFB+bR6XL2mhFZ0TO7GrE9zOeRT5UZvZGId3vmcQuknPHfEGBnNOZaDu1\\\\\\\/Fnz1hNa8fV1tSuik+RpmCgFC+8woMkuJesuweNmkEFhHrcj7hwckK+J6K293OMOksuFXMuAFBZJR3Eji7y366w7Gx8ICTlx+ju1RmlsJGAFlJMHBKdcK63\\\\\\\/Vhb+U7T1VsPW5FO3KoTP9ADu3l+tdTAZgU03YXHIi2Y4qiB9MB4qzR61ebNPwc1Nr6ztvsUt+R9HNHO5qLjUkASaIcMh1F3KQgy845LakyMCRBvCHZU1a99M3jcSHDbYaYfjtP2SA+FTT+ZfmwEbKBCPM+XqJuS2YbL\\\\\\\/n7+WXYxygEP3bSj75rtAkI41vg8KL9gydcAbwRZsREZZsWa\\\\\\\/WGYg36GL5e\\\\\\\/SeIAKvqcLofdvjk3iwcTAKVxeHkHSnKvc0Pmb\\\\\\\/rT63iG1q5ahlLwh1047zj92HjJrAciipUueJO542e4bRa5jqqvQvItPpD+8M9hS8b57Wc1TJJitH7ATVb5WLsG8manrQ56heGf3d3ZRneRQTlS+VIqV2DPGPtr+88dporEGqNEAOwa8Gs5bhEa55BLIqMTeEMEsCu4aTufXyeWJVIiWAomKBe8Dc\\\\\\\/nNqz09Dn1OlK74QMyMW5CwOkcYJ6GOhpnoANp4B8lB4PwnL4gK2Ft4cwHIq6guOQAPB93rj3hQYK00rYN6kOsvvIzLWK40Grc70xjYytj8Pusy7nrQlr\\\\\\\/WJDZCOwQtahA9AF3olNUhkwNFEgGMZnI8HoOJqnB4H9HX1J8R5gQsa5E3MDF1vMU\\\\\\\/0\\\\\\\/8r31aXCWUP9gxpVvzyFZL62nJ6quQybGKcAX\\\\\\\/A24Pok8L1W\\\\\\\/JicGvnur5RoeLOG0Fbq095uBxSGz3W828kmAbkJ6lzjxVSf2R\\\\\\\/c2Gi2zo+z+jlLB+gujDxLG7bHHzuOCv1A32Pzyam+E6EaLQ7EVenRykj1cyw7I9h+MnNb6QQV7k\\\\\\\/gC2Q\\\\\\\/VvMMKgUDn\\\\\\\/CJbTgPdtiMoYYwHOQux+yEKsQwTQB\\\\\\\/ybLeMA68yxnygtWl6Wamj4HFsbz3DBJMXc37vk7eYddOmUEyeegQVLHiSbu+zbo77Sy3DSbxPIPwlTC\\\\\\\/t2ECKvpXAheChkR2yw3gOStusiSUYman5lS9d4yCmTFeSd9ggGY+4\\\\\\\/\\\\\\\/PjCpEeDmkI9hUgU2vK1GCkoKG7dUIrCuUo8lyviNKszvCiQdvqHRiXRlonnIa295H5sgHbjSoSa0VNTyFuDlhzgxHC+yXaQsmZSEdlIxd6q0cVsL8KBoi1e4eVZOfWLPS16Rxl6yOIn4uHUgIeO0N+GOUMKoVYEKxDiFOwVuvGSDK0N9wdniSB9EmbfcD\\\\\\\/ZuXLmFxmLDeihkPCZ7xIyZKnzWIfH3+yqBCjQ2kPuhcriy\\\\\\\/qV2ZDc7r4HVb213I6LBM6NdOc4qr0Khbv19K1R9R8ymk7JjWbadkKo7vkcyFJ3Vbxxy2i\\\\\\\/0FB5I8q1Hzwez9saXSWr\\\\\\\/SxWTa1GXGPC9PUmvd0PLi6J\\\\\\\/xA8mPZjcjrtYJYzCw2EM+Jgm+mXGgsdKFSn5h7gYbPYCfUvsTRwsLtSHOqZrVJ5CzkSIc46pTkFsH\\\\\\\/SF\\\\\\\/IJhErLlBfVqb2ZM6\\\\\\\/RZdQ3rQCVubUBPNe46XddM7mPuKKQUPIv9NfiOWvCDPJGDUB7LWj9dvd2+AwsaaRMOy2An2ZLdmXp1wIEG4iHS6+dpPkNQjlG27W9DMLJpdCK03ldspZpbVVIT4uUFGQ1Y+lJ+poUvjp5+hkUS9LF3TKneyw4Kor\\\\\\\/GSDDywjuhKbo2PoD7nSb4kRlCpJcYdJERXoQyX8AzIPEVsKvFr+nCpO6ZdH9sMEz9fw\\\\\\\/IY3fkMvfYF7BVB51jNaHOtnT\\\\\\\/dEfzZPvk0urWIUqB2fgKBZXAiHBOkIY2pfrno93XkC4BRHUJ\\\\\\\/PFXwYXCO10QwX5K6oOUaZElaH2GXSYQl25SZcWDqPPobStCGzH8f3wExx7vf36hIOolELu8BjqMke\\\\\\\/EP7aA7fwzpwmFDLUwjN\\\\\\\/bfzElLvM\\\\\\\/npR3zjG0omWb5QjOMWcOjOfsZP+POy04dwzybUWw7R3+rWnFseurqH7nXEs6PDE3RYvQKO0NKqH+3LCRT5el0SyI1mSo\\\\\\\/Br+aOwto897Frp8TBUibMd7L1zPDePixQKQ+tNjR9G4mZjip5ApBaJey5\\\\\\\/jSls5W0q5dQQJ4R7Ymq7TCl7cxJsQxDu4LyVuCSOiyzSW0qWt7zRNZojNCn6FHeQZPD71UfxIQ1qCRISD+PMUf452ERznQxyJvre14OD8t96MZzYKssrjNtHUOmcxC3Sa8+AYu6ZHWOlHuyXUlp\\\\\\\/Qv8Z\\\\\\\/cap1p7JCqempjXlx3xn2xgL18lcJBHYHZUH9puthPfuPVxztvab5DNKifeHkLVG3yT9ztEYwncXtHCV7dkHNlE1JxbrZIGDN0qoNsX1YlB1nNMik7C1vP0KJd688IQL9mtDkhsKxH2WpK8\\\\\\\/jzEVyI02ECiHRySp2flAoIMqJBkAmg5UXC7RPmLpl6Bfwlu5\\\\\\\/2Rxifj10voGat732uE4q5J+51td5Cpfn2RzskD8DFB+hvPA0iJ4lgVRmY7GaKD5o3LyjLmkVMMysjTzOe7l4uH5JrHU9HsqNnHdSLWKrSjZenZCt+d1sitJJkYMw2lokjyHvpUl5DuNFMiuKpB+B8a7QmPq4hxQ89MIqfAGwPDYaTDF1L1T+Zq9F5+04YA8v7JtEs0jyFJWOWSrxd1XnALkAqLGLW698gt+4gV2A6p2jQCnSZLTuxM38FlRIuqKcZlk4pkarBNwQaXeM6Z1H1S\\\\\\\/cKZXR32I5RJvTgPxw2nrosyIcqXCDBpvysqWoc\\\\\\\/zyc6svuQlXWskLmz1p2YVoGmJtFm680fA+YwcuIHYMs5bOYJ1WHkXBqm7s1ZxNyVMbzeiQWBPfBwYmSuizPp10WYvwlQjrWJper+yqfpCUi8cVZq9DPYkQSr5nXIRU28uyB6ohNG4xU5Xm7JwkF7d1COWSWd6rVU6rsEG9\\\\\\\/UMFrsz1kjKrfSKszNnnXS0B8pjkPhTvszNwfa0Fi76gkhDbfTqsXQ4GOb7aaMKVuacOIKivUD\\\\\\\/T7\\\\\\\/HMVUjoBP61ZOAVznGY3vCoXZHhqsFRLuVwbbb2WyVnQOG\\\\\\\/AMaGfvHu+b7Y0wxa+AQIhyXH7u2GZKTX4aw6huKA7q2R9QmBq8syUmtoB\\\\\\\/1fc7ghQJps0HGtMrVrHRQC3N42F0BiMDDNeAHSt66pOtNvoLL\\\\\\\/ISuMVRH8Q+w+TiRl9YG5Al+wNPhEDVNtX+nJ+oQcH0jjzh6E8kLuDHBbkiM3xRsUMhPBZ9AciSdOLP6NzaS1RdjkKA7Ldng4gbPLtt1U2BeMiNhgDEICmPkgJvKKyf6nUVWVucYi4pNsSIopqJHfPXsVqQvt\\\\\\\/\\\\\\\/+t4otg3SWZogQ90A6eNkmdkfh\\\\\\\/\\\\\\\/mf278GM76gyyIs9idHH30U7pukhG6eXAxTjhUgeFHo\\\\\\\/roLgxztUcROCES3j6sLKa51PKqoIUYzdrrRFmRk\\\\\\\/X6A1YQFO77hla1NLDRapr+AJ6GZdRf38iO6mdaVN6v33fxLew9kd8mn2QZAwiwy1J4yekr\\\\\\\/M2+vlIZ1\\\\\\\/Pc8FTIetYeiPQMKg8Rn7RzDUCVyqfTecSXYg938DNF\\\\\\\/RmugCxAgrpFQye5amtYiZK6pB3gcn3jWA1Ge8C4obQcoxMRlCMysoEa8P0Zv8I1yiYuuOVmYQMXhKPG+zSbrmVwZnItuH7eK2luIYbQor6s0EqdDur7jqM\\\\\\\/m6BHDrP+x9l+y2YK8v1izLdEDiT+rn7kWqqSkn05xtR60bf0KRVqlYHKHG9CZlCfmzbuEq8O4EJDtD816CbDJfFv2FYJMACcStQfx4UuuA1Kb+zuykVE9pQl6QVkFnXZty\\\\\\\/6Hlub6h8yLVWJ1iYsVL3\\\\\\\/prR+lXTaPdNBgxe92CejzLRwjsARBf6MRGEXGHOaZ7KO14t6nh\\\\\\\/6dzFsJWt8AsG9ktrPhhgD4SImAXGpadEfMaUQX3t5PBqu2lNl\\\\\\\/CjFkp8r7c25mFIBJ6nt5jEJB+NIfTpBggWd+b1VsIeQJdt7omlVAhBeLcMgxXAtLgKC+pS1FE8xJeZgtMM17tontLuNNv4rNmpSJUMWaWSX2eIYKkZLG\\\\\\\/i0m3V647wGjyoOCkMkK2gBsUkh95lMUFSMR0NtcVIg+XDh7G5EucrW2sb54DM\\\\\\\/NQ\\\\\\\/art3mzS\\\\\\\/vwXQoBK8IOszGDOB3PSnCrAZmiYi3iWdHISmpQyVehPRDnKAO16II2J7hlawV0ndMuPtzFy1+xPQkb0VS3DjytxJ9oKdsUSL0Mp5IMSSAtitp86uJD+of2iyDRhNsaslHDTS\\\\\\\/DwfVX0L\\\\\\\/3\\\\\\\/igvOUr+vrvDuTx\\\\\\\/9ZqSm\\\\\\\/CUHACdKGu47S1gUpxZmrShR3RqQ6L1g1CDyqT7NFgiTvUv1p+x2jlfrrQZ47Dsq0NWRIO5pAeinrhm3uVT4KDPZAWAJW8pPebtEHPfWign8vr5S7NaxCyVWfRFcKZg1BhkvCRbGrhSCKyPYKlUmz+hLvmwaloxvCAqav4oi9Tr1zo2wY3WV2mZADnWkbrjWnoRnctByYNEAWBw67Foqc7udOmIVVZTHTjMxVGGaviUJVUxNsJjKm+tR+yU89uyg+CMBHuEGH4rzF9g+2jKOXPOp5KOwmASXk0e5g15Y78IEU7YBoorIfa1PR\\\\\\\/M6TJPXgky9OmMBEawWZWVhVcOvJe\\\\\\\/SwCvlhd4x\\\\\\\/MS4IuRgax3FPso6MTqRqBf3AMswkbhf3NcrSEaCvluArFn8n\\\\\\\/RV9JuJ98x6KgsliM9f8QWIp+whdLNsVmlKSiQr3\\\\\\\/N6iIkXtSJ0XAQAzwRG0YMZc1GMvHYvuqRWUEa9chHK3XP7t0iduSviKxggr+uLrH2Z6qvQwYCSJOX14TqfyA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mSYw\\\\\\\/5gomf3SYeq\\\\\\\/4TLiPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221b54c51573e4a951a4348df532429959a0242c45f77685a892ae768cafa20a97\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228Wg5YAY+4LcT91TZxw0ZbB4XDg7XFAVKeaDHmPOwVtBewWlWEaNhJPYc2CloIiY67J9R\\\\\\\/A7gJW4j2fsUA2fzDBuzsh3lr1wzYecdP1MSQPMBInDu7OuDVM1I2vxRPqtaZQ2TgtfZdVcN3nrSzH2FOtuRPLHdyZUe0ETWxcF8cFpbDPDUKakYBqiJ\\\\\\\/QPWmcR2XA40aS3upvx8YzbsOTTnNnnc0mO\\\\\\\/XHONtTrjTvwIbI9PhrN0ulKRZdk5W5Wggun2GFyfMeFnSAbmiDhiQIUxwZOKkCx6rJrXyNZkmvB0mclmxhNhRJE0BDDjIcTH9dfjjWBHKqPk\\\\\\\/GAdv0CizX5urT3mieY7XUklmptOeyFAVirCFp+JF2E3\\\\\\\/6nUyi5xdcxhJvOn8Y5wtR8QMURni43o+gw3rNaaZYuzU5ZV\\\\\\\/teaIpxetJRZiR41QpDfyGBymJrhUllAVBYdHkJPReNnVJdEtmxEcEjmTAoKZEtWKBi2IIG3A8eKZMacTuhQIvhGi1b5mktUsjQ3bJq+0nKIW+Vv6Ly2YL2A2ra60z8SgKuiRip8ll690\\\\\\\/lu3joVk3rHGCJBvT6H0TTxTzLKXz6r\\\\\\\/56DiGgNjcg1SmRF3ozjKPxtBEf6lrZShWLsfOkJ4mXhU+IgpBKSOC3uoiLHUukpbiJBgs3n6HxWneXkce9AqBKU7L4EcjLrv3SQDR6IX3sgTyOb6Y0VN24LpasO\\\\\\\/t59DK04Osel\\\\\\\/Ik9x0nSUN7Pe5P49D1B920eodBDw7jGZxtvL8UmYECgQXLizkv8M2eRZcPIass1hY6gULNaDqIHCT\\\\\\\/afaui5HdswVLF0SY5Vojg1hamwsBwpwS75r2LGUyr5AoL1HHNhe3THgfrmoBLn+FUhxeqT39PRhtdausmwWK3rUFpgDBthKCNgqhqP7hw4eMzSvpWdev9TimvLyOKQEcJSjn9coiqbf3Td1hjIHP1WHeHn47oCFPw3oaAb\\\\\\\/MhtAlTK9HEgdQZ8KF8xLnJug1nTtZw6wK6hnrwownCKuwNpLUdnh\\\\\\\/lDWjksMFCPoode8fG2RnoXdt1sbk3eLJPjqvK4Qi\\\\\\\/LjGkTqYCFcLAcyhYHJHMsmq1z3mQWShQI7OHL2Axgtm142hN2JMUzH3PPsZdBT0Djs7Ui6qv9bBBd9mAjzmLqZhVX6xhtYxp07+e6KmLuD7WkQuZwEitSEQiUBc7pcMQDtLXMgZZ5E2c8huAqvCoO\\\\\\\/p8hsOOmDX5ZEDvD2zpg2wdvCgP3q5f4Gawmbhl+XGS1NXR75P5kf7Gv5iGApZr45BcNWuvd0oiFMePvSgAxdA0xODXor8jKgyUpCtwH3G\\\\\\\/23ToG4DLgRNDRuq481SjOOtwBW5hi7yD4dBnd0o\\\\\\\/Bt6tFj8bw\\\\\\\/zs\\\\\\\/w2w9Bh9BGyacU1zFQbEvAFtuUZwG1f0DCvCF+7y6+e6IEGdvi8UTEGGVxakwQy8n2cZfLx36KjiNdB\\\\\\\/NuQBAXTmGuP6WqdHhM+AWJMx+Y1lnZwxnLWnUatCj\\\\\\\/ahijZR5Gd64Cd5JIJf1aPElL286iIg5ZEvI74J8YLM1adK79NUaeL+vAYKeGhlkaPM\\\\\\\/qyw\\\\\\\/SkUG8TcEVWaFeyQbJp2NTZquTmSMTfFy8xlC9ZI25\\\\\\\/SLw5IOQlOrahNLJu5sR5LKQ43Nd1gTnLibqfb1qYp1wXP+kVedcsI0Frc+5yZuGLSm7ZBLCEaw9srvbYVsitqRG5kUyygBGllOyTwZVByUMCLA7\\\\\\\/vNzfJBh0kYbwW7kGkqi3e6Accb6Qdolte+OHpiwv16cbu2iJyBHHjcin\\\\\\\/rWR4M09khQwIiwsUcVkypdeI4G18o1ro0i0PVFkhRObzrTJJFf3jTD0vhkZsfIf6RQAHVATadDQTogxEPSbpyS4dyVOS\\\\\\\/CadgTAVn9Hi1ktwYJOEJOGTdV2EdwaHlanK+zLmaSWMWmt9VRR5gfGlnr362E9cTH1yDJYEPPVuEpv6uLOSX\\\\\\\/w\\\\\\\/t\\\\\\\/HEc8KWDLzQprUhaOo4H89SZAIyLXJNTCgJyz8DZZVjKw==\\\u0022,\\\u0022iv\\\u0022:\\\u00220QdkBZJImoXO4xeOGAqV0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223bb4e4a712485b2f9db9440ddc30c878540fa9a0eeb31d9cf7503c8b5e69381e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495264 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/23" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iYu42IyGf_LFxGum76fUkI5Nrsi2-leV10o4Cxr-CTA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/23" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o2ikHecOv4iG4XCv-GY28ZYWFOgCaiE0nWunDEzWmXE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/13" title="Comparative Analysis of Whole-Genome and Methylome Profiles of a Smooth and a Rough Mycobacterium abscessus Clinical Strain" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/37" title="Genome Sequencing of Musa acuminata Dwarf Cavendish Reveals a Duplication of a Large Segment of Chromosome 2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495264" id="node495264" data-pisa="ggg;10/1/23" data-pisa-master="ggg;g3.119.400694" data-apath="/ggg/10/1/23.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequencing and Functional Annotation of the Whole Genome of <em>Shiraia bambusicola</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xiyi Ren</span>, <span class="highwire-citation-author" data-delta="1">Yongxiang Liu</span>, <span class="highwire-citation-author" data-delta="2">Yumei Tan</span>, <span class="highwire-citation-author" data-delta="3">Yonghui Huang</span>, <span class="highwire-citation-author" data-delta="4">Zuoyi Liu</span> and <span class="highwire-citation-author" data-delta="5">Xuanli Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">23-35;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400694 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495264" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiyi Ren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiyi%2BRen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ren%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiyi%2BRen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yongxiang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yongxiang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYongxiang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yumei Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumei%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYumei%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yonghui Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonghui%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYonghui%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Zuoyi Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zuoyi%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZuoyi%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jxl3237@163.com" class="" data-icon-position="" data-hide-link-title="0">jxl3237@163.com</a>
<a href="mailto:gzliuzuoyi@163.com" class="" data-icon-position="" data-hide-link-title="0">gzliuzuoyi@163.com</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xuanli Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuanli%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuanli%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jxl3237@163.com" class="" data-icon-position="" data-hide-link-title="0">jxl3237@163.com</a>
<a href="mailto:gzliuzuoyi@163.com" class="" data-icon-position="" data-hide-link-title="0">gzliuzuoyi@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/1/23" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495264" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495264/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/23.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495264" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495264/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/1/23.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495264" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495264/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/1/23.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495264" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495264/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Shiraia bambusicola</em> is a rare medicinal fungus found in China that causes bamboo plants to decay and die with severe infection. Hypocrellin, its main active ingredient, is widely used in several fields, such as medicine, agriculture, and food industry. In this study, to clarify the genomic components, taxonomic status, pathogenic genes, secondary metabolite synthesis pathways, and regulatory mechanisms of <em>S. bambusicola</em>, whole-genome sequencing, assembly, and functional annotation were performed using high-throughput sequencing and bioinformatics approaches. It was observed that <em>S. bambusicola</em> has 33 Mb genome size, 48.89% GC content, 333 scaffolds, 2590 contigs, 10,703 genes, 82 tRNAs, and 21 rRNAs. The total length of the repeat sequence is 2,151,640 bp. The annotation of 5945 proteins was obtained from InterProScan hits based on the Gene Ontology database. Phylogenetic analysis showed that <em>S. bambusicola</em> belongs to Shiraiaceae, a new family of Pleosporales. It was speculated that there are more than two species or genus in Shiraiaceae. According to the annotation, 777 secreted proteins were associated with virulence or detoxification, including 777 predicted by the PHI database, 776 by the CAZY and Fungal Cytochrome<em>P</em>450 database, and 441 by the Proteases database. The 252 genes associated with the secondary metabolism of <em>S. bambusicola</em> were screened and enriched into 28 pathways, among which the terpenoids, staurosporine, aflatoxin, and folate synthesis pathways have not been reported in <em>S. bambusicola</em>. The <em>T1PKS</em> was the main gene cluster among the 28 secondary metabolite synthesis gene clusters in <em>S. bambusicola</em>. The analysis of the T3PKS gene cluster related to the synthesis of hypocrellin showed that there was some similarity between <em>S. bambusicola</em> and 10 other species of fungi; however, the similarity was very low wherein the highest similarity was 17%. The genomic information of <em>S. bambusicola</em> obtained in this study was valuable to understand its genetic function and pathogenicity. The genomic information revealed that several enzyme genes and secreted proteins might be related to their host interactions and pathogenicity. The annotation and analysis of its secondary metabolite synthesis genes and gene clusters will be an important reference for future studies on the biosynthesis and regulation mechanism of the secondary metabolites, contributing to the discovery of new metabolites and accelerating drug development and application.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genomic-sequencing" rel="nofollow">Genomic sequencing</a></li><li class="kwd"><a href="/keyword/functional-annotation" class="hw-term hw-article-keyword hw-article-keyword-functional-annotation" rel="nofollow">Functional annotation</a></li><li class="kwd"><a href="/keyword/phylogenetic-analysis" class="hw-term hw-article-keyword hw-article-keyword-phylogenetic-analysis" rel="nofollow">Phylogenetic analysis</a></li><li class="kwd"><a href="/keyword/pathogenic-gene" class="hw-term hw-article-keyword hw-article-keyword-pathogenic-gene" rel="nofollow">Pathogenic gene</a></li><li class="kwd"><a href="/keyword/secondary-metabolism" class="hw-term hw-article-keyword hw-article-keyword-secondary-metabolism" rel="nofollow">Secondary metabolism</a></li></ul><div class="section other1" id="sec-1"><h2 class="">Background</h2><p id="p-4"><em>Shiraia bambusicola</em> is a type of fungus that is parasitic on a few bamboo tissues. It infects a few species of bamboo and is only distributed in Japan and southern China. Importantly, it is used in Chinese medicine for treating convulsions in children, rheumatoid arthritis, sciatica, lumbar muscle strain, bruises, cold stomach pain, and acute hepatitis (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jia <em>et al.</em> 2006</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li and Durbin 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2011</a>). More than 10 types of chemical substances have been isolated from <em>S. bambusicola</em>, including mannitol, stearic acid, hypocrellin A–D (HA, HB, HC and HD), hypomycin A, ergosterol, ergosterol peroxide, 1,8-2-hydroxyanthracene, 11,11′-dideoxyverticillin, <em>S. bambusicola</em> polysaccharides, and digestive enzymes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wang <em>et al.</em>1990</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lin <em>et al.</em> 2002</a>; Fang <em>et al.</em> 2006; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shen <em>et al.</em> 2002</a>). Among these, hypocrellins are the most essential active substances; they are types perylenequinone compounds, which are used as natural coloring agents and novel photosensitizers in the food and medicine industries. Particularly, they have a very good effect on photosensitizing tumor cells, resisting viruses, treating retinopathy and diabetes, and inhibiting HIV-I virus proliferation. They have remarkable clinical effects against vulvar skin lesions in women, postburn scars, psoriasis, tinea capitis, tumors, and rheumatoid arthritis (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shao <em>et al.</em> 2011</a>;<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> 2012</a>). As antibacterial agents, hypocrellins have an obvious inhibitory effect on gram-positive bacteria, such as <em>Staphylococcus aureus</em>, and agricultural research has demonstrated that they have a certain inhibitory effect on fungi, such as rice sheath blight and some molds (Lu <em>et al.</em> 2012; Zhu <em>et al.</em> 2014). In addition, they have good potential as edible natural pigments because of their bright colors, good dyeing force, excellent fat solubility, and health-care functions (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shi <em>et al.</em> 2016</a>). Moreover, the liver-protecting function of <em>S. bambusicola</em> polysaccharide has attracted much attention. However, several years of research has demonstrated that <em>S. bambusicola</em> does not satisfy the needs of clinical medication because of its low yield, difficulty in preservation and proliferation, strict hosts, and a great deal of plucking.</p><p id="p-5">The taxonomic status of <em>S. bambusicola</em> has been controversial since 1900 when Hennings classified <em>Shiraia</em> into the Nectriaceae family of Pyrenecetes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hennings 1900</a>). In 1902, Saccardo attributed it to the Hypocreace family of Hypocreales because of its large fleshy fruit body (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Saccardo 1902</a>). In 1980, Amano indicated that the ascospores of <em>Shiraia</em> have a double-walled structure and thus, attributed it to the Polosporaceae family of Pleosporales and Loculoascomycetes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amano 1980</a>). However, because the hex-ascospore structure of <em>Shiraia</em> was inconsistent with the characteristics of the eight ascospores of the Pleosporaceae, Kirk <em>et al.</em> attributed it to the Dothideales of Loculoascomycetes in 2001 (Kirk 2001); but the status of the genus and family of <em>Shiraia</em> remained undetermined. It was found that there was only one species of the <em>Shiraia</em> genus, namely <em>Shiraia</em>. In 2004, Cheng <em>et al.</em> constructed a <em>Shiraia</em> phylogenetic tree based on 18S rDNA and ITS sequences, assigned it to Pleosporales, albeit with a different classification from that demonstrated in Amano’s study, and recommended <em>Shiraia</em> to be placed as an independent genus in the Phaeosphaeriaceae family (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2004</a>). In 2013, our research group officially established a new family—Shiraiaceae of the Pleosporales—based on the LSU-rDNA, EF, and RPB gene sequence data of <em>S. bambusicola</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2013</a>). To date, the family comprises only one genus and species.</p><p id="p-6">For a long time, many questions related to the obligate parasite relationship between <em>S. bambusicola</em> and bamboo, the hypocrellin biosynthetic pathway, regulation mechanism, and gene information have remained unresolved. These questions have restricted the development and utilization of <em>S. bambusicola</em>. In this study, we sequenced and assembled the entire genome of <em>S. bambusicola</em> GZAAS2.1243 using the Illumina HiSeq 2000 platform (BGI-Shenzhen Co., Ltd.). Based on the genome analysis, we expected to identify the genes related to their pathogenicity, secondary 2metabolite biosynthesis, and growth and development on the host.</p></div><div class="section methods" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Strains, growth conditions, and genomic DNA</h3><p id="p-7"><em>S. bambusicola</em> GZAAS2.1243 was isolated from the Jing County, Anhui Province, China. The collected samples were kept at the Key Laboratory of Agricultural Biotechnology of Guizhou Province, Guiyang city, China. The fungus was cultured in PDA (Potato Dextrose Agar) medium (200g/L potatoextract, 20-g/Lglucose,18-g/L agar) plates in the dark at 25° for 7 days. Mycelia were collected in a mortar, liquid nitrogen was added, and the samples were crushed using a pestle. Genomic DNA from the fungal mycelia was extracted using the CTAB method. The DNA pellet was dissolved in sterile water and adjusted to a concentration of 552 μg/mL.</p></div><div id="sec-4" class="subsection"><h3>Genomic DNA sequencing and assembly</h3><div id="sec-5" class="subsection"><h4>Library Construction and Genome sequencing:</h4><p id="p-8">One paired-end library (insert sizes 270 bp) and one mate-pair library (5Kb) were constructed. The construction flow of library was as follows: 1 μg genomic DNA was randomly fragmented by Covaris. The fragmented DNAs were tested using Gel-Electrophotometer and purified using the AxyPrep Mag PCR clean up Kit. The fragmented DNAs were combined with End Repair Mix and incubated at 20° for 30 min. Subsequently, the end-repaired DNAs were purified using AxyPrep Mag PCR clean up Kit. The repaired DNAs were combined with A-Tailing Mix and incubated at 37° for 30 min. Illumina adaptors were ligated to the Adenylate 3′Ends DNA and incubated at 16° for 16 h. The adapter-ligated DNAs were purified with the AxyPrep Mag PCR clean up Kit and their fragments were then sorted and selected based on the insert size. Several rounds of PCR amplification with PCR Primer Cocktail and PCR Master Mix were performed to enrich the adapter-ligated DNA fragments. The PCR products were purified again using the AxyPrep Mag PCR clean up Kit. The library was qualified by the Agilent Technologies 2100 bioanalyzer and ABI StepOnePlus Real-Time PCR System. Paired-end sequencing was performed for the qualified libraries using Illumina System.</p><p id="p-9">We used a whole-genome shotgun strategy and next-generation sequencing technologies on the Illumina HiSeq 2000 platform (BGI-Shenzhen Co., Ltd., Shenzhen city, Guangdongprovince, China) to sequence the <em>S. bambusicola GZAAS2.1243</em> genome.</p></div><div id="sec-6" class="subsection"><h4>Estimation of genome size using k-mer:</h4><p id="p-10">Based on the length of each k-mer as Kbp, a raw sequence read with L bp comprises (L – K + 1) k-mers. The frequency of each k-mer can be calculated from the genome sequence reads. k-mer frequencies along the sequence depth gradient follow a Poisson distribution in a given dataset, except for a higher representation of low frequencies owing to sequencing errors because these affect the number of k-mers that may be orphaned among all splitting k-mers. The genome size—G,—was defined as G = K_num/K_depth, where the K_num is the total number of k-mers and K_depth is the most occurring frequency.</p></div><div id="sec-7" class="subsection"><h4>Genome assembly:</h4><p id="p-11">The <em>S. bambusicola</em> genome was assembled <em>de novo</em> using SOAPdenovo 1.05 (<a href="http://soap.genomics.org.cn">http://soap.genomics.org.cn</a>) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2010</a>), which employs the de Bruijn graph algorithm to both simplify the assembly and reduce computational complexity. Low-quality reads were filtered out, and potential sequencing errors were removed or corrected by the k-mer frequency methodology. We filtered out the following type of reads: (1) reads having an “N” of &gt;10% of its length, (2) reads from short insert size libraries having &gt;65% bases with quality of ≤7 and the reads from large insert size libraries that contained &gt;80% bases with quality of ≤7, (3) reads with &gt;10 bp from the adapter sequence (allowing no &gt;2 bp mismatches), (4) small insert size paired-end reads that overlapped ≥10 bp between the two ends, (5) read 1 and read 2 of two paired-end reads that were completely identical (and thus considered to be the products of PCR duplication), and (6) reads having a k-mer frequency of &lt;4 after correction (to minimize the influence of sequencing errors).</p><p id="p-12">After these quality control and filtering steps, the data were retained for assembly. SOAPdenovo first constructs the de Bruijn graph by splitting the reads from short insert size libraries (270 bp) into 41-mers and then merging the 41-mers; contigs, which exhibit unambiguous connections in de Bruijn graphs, are then collected. All reads were aligned onto the contigs for scaffold building using the paired-end information. This paired-end information was subsequently used to link contigs into scaffolds, step by step, from short insert sizes to long insert sizes. Some intra-scaffold gaps were filled by local assembly using the reads in a read pair, where one end was uniquely aligned to a contig, whereas the other end was located within a gap. To access assembly quality, high-quality reads that satisfied our filtering criteria were aligned onto the assembly using BWA with default parameter (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li and Gao <em>et al.</em> 2009</a>). To test for completeness of the assembly, the sequencing depth of each base was calculated from the alignment, the proportion of a given depth was calculated, plotted, and compared with the theoretical Poisson distribution with a mean corresponding to the peak. If genomic regions collapse because of assembly quality, these regions would possess a higher than expected sequencing depth, <em>i.e.</em>, if two copies were merged into a single copy, the depth of the assembled region is expected to be twofold higher than the expected value.</p></div></div><div id="sec-8" class="subsection"><h3>Genome annotation</h3><div id="sec-9" class="subsection"><h4>Repeat annotation:</h4><p id="p-13">The repetitive sequences were predicted using the Repbase databases (<a href="https://www.girinst.org/repbase/">https://www.girinst.org/repbase/</a>), ProteinMask databases (<a href="http://www.Repeat-masker.org/cgi-bin/">http://www.Repeat-masker.org/cgi-bin/</a> RepeatProteinMaskRequest), TRF databases(<a href="http://tandem.bu.edu/trf/trf.html">http://tandem.bu.edu/trf/trf.html</a>), <em>de novo</em> (download at <a href="https://github.com/Reedwarbler/REPdenovo">https://github.com/Reedwarbler/REPdenovo</a>) (Jiang <em>et al.</em> 2013; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chong <em>et al.</em> 2016</a>). Tandem repeats were located across the genome using the software Tandem Repeats Finder (TRF) (<a href="http://tandem.bu.edu/trf/trf.html">http://tandem.bu.edu/trf/trf.html</a>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Benson 1999</a>). Transposable elements (TEs) were predicted in the genome by homology to RepBase sequences using RepeatProteinMask (<a href="http://repeatmasker.org/cgibin/">http://repeatmasker.org/cgibin/</a> Repeat Protein Mask Request) and RepeatMasker (<a href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>)with default parameters (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen 2004</a>).</p></div><div id="sec-10" class="subsection"><h4>Non-coding RNA annotation:</h4><p id="p-14">The tRNA region and tRNA secondary structure were predicted using a tRNA scan-SE online tool (<a href="http://lowelab.ucsc.edu/tRNAscan-SE/">http://lowelab.ucsc.edu/tRNAscan-SE/</a>) with default parameters. The rRNA were predicted using <em>de novo</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Washietl 2010</a>). The lncRNA and snRNA were obtained using the infernal software (<a href="http://eddylab.org/infernal/">http://eddylab.org/infernal/</a>) with default parameters (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">LiangHu 2013</a>).</p></div><div id="sec-11" class="subsection"><h4>Gene annotation:</h4><p id="p-15">To predict genes in the <em>S. bambusicola</em> genome, we used both homology-based methods and <em>de novo</em> methods. For the homology-based prediction, fungi proteins were downloaded from Ensembl (release 44) and mapped onto the genome using TblastN (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kent 2002</a>). Subsequently, homologous genome sequences were aligned against the matching proteins using Genewise (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Birney <em>et al.</em> 2004</a>) to define gene models. For <em>de novo</em> prediction, Augustus (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Stanke <em>et al.</em> 2003</a>) and Genscan (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Salamov <em>et al.</em> 2000</a>) were employed to predict coding genes. Finally, the homology-based and <em>de novo</em>-derived gene sets were merged to form a comprehensive nonredundant reference gene set using GLEAN (<a href="http://sourceforge.net/projects/glean-gene/">http://sourceforge.net/projects/glean-gene/</a>) by removing all the genes with sequences of &lt;50 amino acid as well as those that only had <em>de novo</em> support. Genes were also annotated using Blast2GO (<a href="http://www.blast2go.com/b2ghome/">http://www.blast2go.com/b2ghome/</a>) based on the terms “biological function,” “Cellular Component,” and “molecular process” in Gene Ontology (GO). To identify the genes involved in pathogenicity and virulence in the <em>S. bambusicola</em> genome, BLASTP with a cut-off E value set at 1e − 10 was adopted to search against the Pathogen Host Interactions database (PHI) (version 3.6) (<a href="http://www.phi-base.org/">http://www.phi-base.org/</a>), which comprises experimentally validated pathogenicity, virulence, and effector genes of fungal, oomycete, and bacterial pathogens (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Winnenburg <em>et al.</em> 2008</a>). The genome-encoding cytochrome P450s were annotated using BLASTP to search the fungal Cytochrome P450 database (version1.1) (<a href="http://drnelson.utmem.edu/CytochromeP450.html">http://drnelson.utmem.edu/CytochromeP450.html</a>) with a cut-off E value set at 1e − 10 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Moktali <em>et al.</em> 2012</a>). Proteomes were classified into proteolytic enzyme families by performing a batch Blast search against the MEROPS protease database (release 9.13) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rawlings <em>et al.</em> 2008</a>), and carbohydrate-active enzymes were classified using a HMMER (v3.1b1, with default parameters) scan against the profiles compiled with dbCAN release 4.0 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yin <em>et al.</em> 2012</a>) based on the CAZY database (version 2012-03-13) (<a href="http://www.cazy.org/">http://www.cazy.org/</a>).</p></div></div><div id="sec-12" class="subsection"><h3>Personalized bioinformatics analysis</h3><p id="p-16">Phylogenetic analysis: In this study, the information of the entire genome of <em>S. bambusicola</em> GZAAS2.1243 was phylogenetically analyzed for <em>S. bambusicola</em>, some species of Pleosporales, doubtful species of <em>S. bambusicola</em>, and some species of Ascomycotina (Supplemental Material, Table S1). Orthologous protein prediction was performed using Proteinortho (v5.11, with default parameters, except that identity =75) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lechner <em>et al.</em> 2011</a>). Among the predicted orthologous gene clusters, highly conserved single-copy gene clusters were chosen and aligned using the MUSCLE (v3.8.31_i86 linux64, with default parameters) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Edgar 2004</a>) algorithm. To eliminate divergence and ambiguously aligned blocks from the alignment, Gblocks (v0.91b) was employed under the default parameter setting (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Talavera <em>et al.</em> 2007</a>). The trimmed alignments of orthologous sequences were concatenated using a Perl script FASconCAT (v1.02, with default parameters) (Kück <em>et al.</em> 2010), and a maximum likelihood phylogenetic tree was constructed using the Dayhoff model in TREE-PUZZLE v5.3.rc16 with 1000 bootstrap replicates (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schmidt <em>et al.</em> 2002</a>). The tree was visualized using Figtree v1.42 (<a href="http://tree.bio.ed.ac.uk/software/figtree">http://tree.bio.ed. ac.uk/ software/figtree</a>).</p><p id="p-17">Analysis of candidate genes associated with hypocrellins synthesis: (1) The gene set from 10 closely related species(Table S2) was applied for gene family analysis using the TreeFam software. Thereafter, the unique genes of <em>S. bambusicola</em> species were screened out. (2) To identify the common genes, gene prediction was conducted by gene sequence alignment among the above unique genes and genes of another species (<em>Shiraia</em> sp. slf14) that also produces hypocrellin. (3) Perylenequinone-producing species (a total of five species, one of which is transcriptome data) and perylenequinone-nonproducing species were used for gene family analysis (Table S3). Genes that did not appear in any perylenequinone- nonproducing species but could be detected in at least four perylenequinone-producing species were identified as target genes. The consensus genes were screened by comparing the target genes and the genes obtained from step (2).</p><p id="p-18">On the homepage of the antiSMASH tool (<a href="https://antismash-db.Secondarymetabolites.org/#/start">https://antismash-db.Secondarymetabolites.org/#/start</a>), the genomic sequence was entered for analysis of the secondary metabolite gene cluster with default parameters.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-19">The whole-genome sequence data reported in this paper have been deposited in the Genome Warehouse in BIG Data Center, Beijing Institute of Genomics (BIG), Chinese Academy of Sciences, under accession number GWHAAZB00000000 and publicly accessible at <a href="http://bigd.big.ac.cn/gwh">http://bigd.big.ac.cn/gwh</a>. Table S1 illustrates the phylogenomic analysis results of the species. Table S2 present the details of the 10 other species, which are closely related to <em>S. bambusicola</em> for gene family analysis. Table S3 reflects the perylenequinone-producing and perylenequinone-nonproducing species. Table S4 presents the summary of some main features of <em>S. bambusicola</em> and the 13 sequenced Pleosporales fungi genomes. Table S5 provides information on the secreted proteins. Table S6 presents the genes predicted in pathogen–host interactions. Table S7 presents the genes predicted in cytochrome P450. Table S8 presents the genes predicted in carbohydrate-active enzymes. Table S9 presents the genes predicted in protease. Table S10 presents the genes shared between protease and CAZY. Table S11 presents the genes shared among protease, CAZY, and PHI. Table S12 presents the genes shared between protease and P450. Table S13 presents the genes shared among protease, cytochrome P450, and CAZY. Table S14 presents the genes shared among protease, cytochrome P450, and PHI. Table S15 presents the genes shared among protease, cytochrome P450, CAZY, and PHI. Table S16 presents the genes shared between cytochrome P450 and CAZY. Table S17 presents the genes shared between CAZY and PHI. Table S18 presents the genes shared between cytochrome P450 and PHI. Table S19 presents the genes shared among cytochrome P450, CAZY, and PHI. Table S20 presents the genes shared between protease and PHI. Table S21 presents the shared genes related to the synthesis of hypocrellins between S. bambusicola and other species that produce hypocrellins. Table S22 presents the shared genes related to the synthesis of perylenequinones between the perylenequinone-producing and perylenequinone- nonproducing species. Table S23 presents the genes related to hypocrellins biosynthesis. Table S24 presents the genes cluster which related to hypocrellins biosynthesis. Supplemental material available at figshare: <a href="https://figshare.com/s/ffefab4fb2879489cb02">https://figshare.com/s/ffefab4fb2879489cb02</a>.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results and analysis</h2><div id="sec-15" class="subsection"><h3>Genome sequencing and assembly</h3><p id="p-20">Genome sequencing of <em>S. bambusicola</em> was performed using the Illumina HiSeq 2000 platform, and the 3334 Mb raw sequence data were sequenced and filtered to obtain clean data. The <em>S. bambusicola</em> genome was assembled using the SOAPdenovo 1.05 software. The genome was assembled into 2590 contigs and 333 scaffolds with a combined length of 33,146,786 bp. The longest and shortest lengths of scaffolds were 1,295,965 and 1000 bp, respectively. The N50 and N90 values were 546,034 and 132,645 bp, respectively. The GC content was 48.89%. There were 262 fragments &gt;2 kb (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The genome size of <em>S. bambusicola</em> was similar to that of most Pleosporales fungi, but the GC content was the least similar (Table S4).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495289/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495289/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The statistical results of the genome assembly of <em>Shiraia bambusicola</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Genomic component</h3><div id="sec-17" class="subsection"><h4>Gene prediction and annotation:</h4><p id="p-21">The composition of the <em>S. bambusicola</em> genome was estimated by gene prediction, repeat sequence prediction, and noncoding RNA prediction, and <em>S. bambusicola</em> genome was found to comprise 10,703 genes, 30,545 exons, and 19,842 introns. The total and average lengths of the gene were 18,123,210 and 1693.28 bp; exons, 15,921,153 and 521.24 bp; intron, 2,202,057 and 110.98 bp; and CDS region, 15,901,268 and 1485.68 bp, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495433/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495433/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The statistical results of genomic components of <em>Shiraia bambusicola</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-22">The coding regions accounted for 87.85% of the entire genome, and each gene contained about 2.85 exons and 1.85 introns. The total length of the repeats was 2,151,640 bp, which was close to 6.5% of the entire genome size.</p></div></div><div id="sec-18" class="subsection"><h3>Repeat sequences</h3><p id="p-23">Genetic information-rich repetitive sequences were a part of the gene regulatory network, which could regulate the initiation of DNA replication and promote or terminate transcription with a variety of <em>cis</em>-regulatory elements and signal molecules(<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yamamoto 2018</a>). The repeat sequences included DNA transposons, tandem repeats(TR), and transposable elements, which were further divided into long terminal repeat transposons (LTR) and non-long terminal repeat transposons(Non-LTR), including long and short interspersed elements (LINE and SINE, respectively). The repetitive sequences were predicted using the methods of Repeat Masker, Repeat Protein Mask, Tandem Repeats Finder and <em>De novo</em>, and their total length was 2,151,640 bp, accounting for 6.4998% of the entire genome. Among them, the repetitive sequences were 551,548, 388,081, 506,412, and 1,566,152 bp based on the Repbase databases, ProteinMask databases, TRF databases, and <em>De novo</em> predictions, respectively, accounting for 1.6661%, 1.1723%, 1.5298%, and 4.73111% of the entire genome, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The prediction of different types of transposed elements revealed that the <em>S. bambusicola</em> genome comprised the following: DNA transposons of 887,263 bp, accounting for 2.6803% of the entire genome; LINE of 325,445 bp, accounting for 1.1428% of the entire genome; SINE of 2,417 bp; long terminal repeats of 316, 920 bp; others of 66 bp; and unknown of 3929 bp (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495381/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495381/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Repeat sequence statistics</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495424/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495424/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Transposon classification information statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Noncoding RNAs (ncRNAs)</h3><p id="p-26">ncRNAs are types of functional RNA molecules found in bacteria, archaea, and eukaryotic organisms; they can perform biological functions in the form of RNA molecules, regulate gene expression at the gene transcription level, RNA maturation, and protein translation, and participate in all important physiological processes, such as development, differentiation, and metabolism (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mercer 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Taft <em>et al.</em> 2010</a>; Expósitovillén 2018).</p><p id="p-27">Analysis performed using the tRNAscan and Infernal software revealed that the <em>S. bambusicola</em> genome comprises four types of ncRNAs—21 rRNAs, 82 tRNAs, 2 lncRNAs, and 32 snRNAs. The total and average lengths of the tRNAs were 7689 and 93.76 bp, respectively, accounting for 0.0232% of the entire genome. The total and average lengths of the rRNAs were 8,840 and 420.95 bp, respectively, accounting for 0.0267% of the entire genome. The total and average lengths of the lncRNA were 485 and 242.5 bp, respectively, accounting for 0.0015% of the entire genome. The total and average lengths of the snRNAs were 2871 and 89.71 bp, respectively, accounting for 0.0087% of the entire genome (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495316/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495316/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Noncoding RNA statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>GO-based distribution of the genes</h3><p id="p-29">Among the total predicted genes in <em>S. bambusicola</em>, 5945 proteins (55.55%) had InterProScan hits; their genes were investigated for distribution into functional categories based on GO. According to the analysis using GO domains and Blast2GO (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Conesa <em>et al.</em> 2005</a>), Biological Process,” “Cellular Component,” and “Molecular Function” were the three generic terms of level data that are predicted (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The major “Molecular Function” groups were established based on the following GO terms: “metabolic process” (59.61%), “binding” (58.92%), “catalytic activity” (55.34%), “establishment of localization” (6.09%), and “nucleic acid binding transcription factor activity” (3.68%). Among the cellular components, approximately 20.5% of the genes had involvements of the cell, followed by the membrane (14.99%), organelle (12.7%), virion (0.03%), synapse (0.02%). The “Biological Process” groups were established based on the following GO terms: “metabolic process” (54.7%), “cellular process” (44.02%), “localization” (12.3%), “biological regulation” (8.46%), and “response to stimulus” (4.46%).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/23/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GO-based functional annotation of genes present in the [Shiraia bambusicola P. Hennings] genome. The first one indicates Biological Process domains, the second indicates cellular component domains, and the third indicates the Molecular function domains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-980101705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GO-based functional annotation of genes present in the [Shiraia bambusicola P. Hennings] genome. The first one indicates Biological Process domains, the second indicates cellular component domains, and the third indicates the Molecular function domains.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/23/F1.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/1/23/F1.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/23/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/23/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495412" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">GO-based functional annotation of genes present in the [Shiraia bambusicola P. Hennings] genome. The first one indicates Biological Process domains, the second indicates cellular component domains, and the third indicates the Molecular function domains.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Phylogenetic relationships</h3><p id="p-31">To further confirm the taxonomic status of <em>S. bambusicola</em>, the information of the entire genome was phylogenetically analyzed for <em>S. bambusicola</em>, some species of Pleosporales, doubtful species of <em>S. bambusicola</em>, and some species of Ascomycotina in this study. The results showed that <em>S. bambusicola</em> belongs to Ascomycota, Leculoascomycetes, and Pleosporales. The three strains of <em>S. bambusicola</em> collected in this study were clustered and closely related to the <em>Stagonospora</em> sp. SRC1lsM3a of the Massarinaceae and <em>Parastagonospora nodorum</em> of the Phaeosphaeriaceae of Pleosporales, but <em>S. bambusicola</em> GZAAS2.1243 and the other two strains were not clustered. It was speculated that there are different genera or species in the Shiraiaceae; this speculation were different from the standpoints of previous researchers — only one species or one genus of the Shiraiaceae (Krik 2001; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2013</a>).</p></div><div id="sec-22" class="subsection"><h3>Predicted secreted proteins involved in virulence or detoxification</h3><p id="p-32"><em>Shiraia bambusicola</em> is a type of significant pathogenic fungus that can cause decay and death of bamboo plants. It was observed that the enzymes secreted by <em>S. bambusicola</em> played crucial roles in pathogenicity and virulence (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Winnenburg <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lombard 2014</a>). The secreted enzymes included PHI-related proteins, cytochrome P450 enzymes, carbohydrate-active enzymes, proteases, etc. With the <em>in silico</em> pipeline method, 777 of the 10,703 (7.26%) predicted proteins were potentially secreted proteins (Table S5), close to the average 8% found in most filamentous fungi (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Han <em>et al.</em> 2016</a>). By performing a whole proteome BLASTP against the pathogen–host interaction database, 1855 predicted proteins encoded by the <em>S. bambusicola</em> genome were determined to share homology with the genes implicated in pathogenicity and virulence, and 777 putative PHI-related proteins were potentially secreted proteins. Cytochrome P450 enzymes not only participated in the production of the metabolites important for the organism’s internal requirements but also played critical roles in adaptation to diverse environments by modifying harmful environmental chemicals. Using BLASTP to the fungal cytochrome P450 database, it was found that 900 of the 10,703 (8.4%) were cytochrome P450 protein (Table S7), and 776 putative cytochrome P450 proteins were potentially secreted proteins, most of which were involved in pathogen–host interactions. Using BLASTP to the Carbohydrate-Active Enzymes (CAZY) database, it was found that 1563 of the 10,703 (14.6%) proteins were predicted (Table S8) and 776 were potentially secreted proteins. A total of 493 proteases were secreted in the <em>S. bambusicola</em> genome (Table S9), and 441 were potentially secreted proteins.</p><p id="p-33">The analysis of the four databases revealed the following: the PHI, cytochrome P450, proteases, and CAZY databases shared 440 secreted proteins; the cytochrome P450, CAZY, and PHI databases shared 776; the proteases, CAZY, and cytochrome P450 databases shared 440 (Table S10-S20 &amp; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/23/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phylogenetic relationship among the Shiraia bambusicola genomes, 33 other fungi with sequenced genomes, and one other S. bambusicola transcriptome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-980101705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The phylogenetic relationship among the &lt;em&gt;Shiraia bambusicola&lt;/em&gt; genomes, 33 other fungi with sequenced genomes, and one other &lt;em&gt;S. bambusicola&lt;/em&gt; transcriptome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/23/F2.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/1/23/F2.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/23/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/23/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495429" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">The phylogenetic relationship among the <em>Shiraia bambusicola</em> genomes, 33 other fungi with sequenced genomes, and one other <em>S. bambusicola</em> transcriptome.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/23/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn graph showing the intersections among the proteases (blue), CYP450 enzymes (pink), CAZymes (green), and PHI proteins (light red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-980101705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn graph showing the intersections among the proteases (blue), CYP450 enzymes (pink), CAZymes (green), and PHI proteins (light red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/23/F3.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/1/23/F3.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/23/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/23/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495450" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Venn graph showing the intersections among the proteases (blue), CYP450 enzymes (pink), CAZymes (green), and PHI proteins (light red).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Potential pathogenesis-related genes</h3><p id="p-36">To the best of our knowledge, the genes/factors responsible for pathogenicity have not been determined in the <em>S. bambusicola</em> genome to date because of the lack of genomic resources. <em>Shiraia bambusicola</em> infects a bamboo plant, especially the leaf sheath of the branch for &gt;2 years. To obtain the putative genes associated with pathogenicity, the proteomes of <em>S. bambusicola</em> were analyzed using the pathogen–host interaction gene database, cytochrome P450 enzymes, carbohydrate-active enzymes, secretory proteins, transporters, and transcription factors.</p></div><div id="sec-24" class="subsection"><h3>Putative pathogen–host interaction genes</h3><p id="p-37">The PHI database contained the pathogenic, virulence, and effector genes validated by experiments on fungal, oomycete, and bacterial pathogens of fungi, insects, plants, and animals (Winnenburg 2008). To find the potential pathogenesis-related genes, the entire genome was annotated with a pathogen<strong>–</strong>host interaction database (gene database version 3.6 at E &lt; 1 × 10<sup>−20</sup>). All the 10,703 protein sequences of <em>S. bambusicola</em> were aligned to PHI fungal genes using BLASTP (E value of 10<sup>−10</sup>). All 1855 (17.31% of the total genes of <em>S.bambusicola</em>) putative PHI genes were identified, and they spanned across 47 fungal species (Table S6).The highest number of homologs was found in <em>Magnaporthe grisea</em> (311 genes), followed by <em>Botrytis cinerea</em> (275 genes), <em>Candida albicans</em> (262 genes), <em>Cryptococcus neoformans</em> (124 genes), <em>Ustilago maydis</em> (113 genes), <em>Fusarium oxysporum</em> (91 genes), <em>Stagonospora nodorum</em> (69 genes), <em>Colletotrichum lagenarium</em> (68 genes), <em>Cercospora nicotianae</em> (62 genes),<em>Aspergillus fumigatus</em> (51 genes), and other fungal species (427 genes). These genes perform different functions in the process of pathogen–host interaction. Most of them are involved in reduced virulence (1114 genes), unaffected pathogenicity (360 genes), loss of pathogenicity (313 genes), effector (plant avirulence determinant; 26 genes), increased virulence (hypervirulence; 17 genes), resistance to chemical (13 genes), lethality (4 genes), wild-type mutualism (2 genes), sensitivity to chemicals (2 genes), chemistry target-phenotype unknown (1 gene), and enhanced antagonism (1 gene) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/23/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The gene distribution of pathogen-host interaction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-980101705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The gene distribution of pathogen-host interaction.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/23/F4.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/1/23/F4.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/23/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/23/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495374" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">The gene distribution of pathogen-host interaction.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>Carbohydrate-active enzymes</h3><p id="p-39">Carbohydrate-active enzymes are important for the pathogenicity and toxicity of fungus and are responsible for breaking down the cell wall components of the host to establish a successful infection process. They help the pathogens acquire nutrients by degrading the storage compounds in the host (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lombard 2013</a>). The analysis of the CAZY database revealed that there are 1546 genes encoding carbohydrate-active enzymes, which fall into 153 CAZY protein families (Table S8). Among these genes, 206 proteins have transmembrane domains and 35 have signal peptides. Based on the analysis of catalytic activity using the CAZY database, it was observed that there are 677 genes that encode glycoside hydrolases (GH), which are distributed in 68 families and involved in the hydrolysis of the glycosidic bond between or within carbohydrate molecules. There are 381 glycosyltransferase (GT) genes, which were distributed across 39 families, and they use dolichol-P-glucose, polyprenol-P -mannose, and CMP-β-<em>N</em>-acetylneuraminate as the sugar donor. There are332 genes encoding carbohydrate-binding module (CBM) proteins, which belong to 25 families, and their main function is to bind cellulose, granular starch, and other substrates. The carbohydrate esterases (CE) are encoded by 179 genes, and they belonged to 14 protein families. There are 14 polysaccharide lyase (PL) enzyme genes assigned to three protein families (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495326/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495326/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Results of classification and annotation of carbohydrate enzymes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Cytochrome P450 enzymes</h3><p id="p-41">Cytochrome<em>P</em>450(CYP) is a monooxygenase superfamily that performs several functions in fungi. These enzymes play important roles in the secretion and transport of fungal toxins, and regulate fungal pathogenicity and life cycle. A total of 900 cytochrome <em>P</em>450 genes were predicted, which are distributed across 52 gene families and 58 fungal species(Table S7).</p></div><div id="sec-27" class="subsection"><h3>Secondary metabolites</h3><p id="p-42">The filamentous fungi produce various mycotoxins or other bioactive compounds that have long been exploited by the pharmaceutical industry (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Khaldi <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Medema <em>et al.</em> 2011</a>). In China, <em>S. bambusicola</em> is used as a rare and traditional medicine. There are various medicinal active ingredients (secondary metabolites) in <em>S. bambusicola</em>; among these, hypocrellins are the most important. Hypocrellins are photosensitive compounds and perylenequinoid–type pigments with good application prospects and developing values in phototherapy, photodynamic pesticide, food additive, cosmetic pigment, and photoelectric conversion material. To determine the genes associated with the production of hypocrellins in <em>S. bambusicola</em> GZAAS2.1243, gene family analysis was performed using a gene set of 10 closely related species based on TreeFam software. Approximately 1353 unique genes were identified and applied to predict hypocrellin–producing genes by alignment to the genes from <em>Shiraia</em> sp. Slf14 (a hypocrellin production species). Eventually, 719 hypocrellin–producing genes (Table S21) were obtained. Both species that produce and do not produce perylenequinones were included in the gene family analysis. Initially, 395 perylenequinone–producing genes (Table S22) were screened. Based on the preliminary screening results and criteria, 251 shared genes (Table S23) were screened between the 395 perylenequinone–producing genes and the 719 hypocrellin–producing genes. All the 251 shared genes were enriched into 88 pathways using the pathway enrichment analysis (Table S23). There are &gt; five genes for 28 pathways, and the three main enriching pathways were: and the three main enriching. The partial results of the pathway enrichment are shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. The synthetic pathway genes of terpenoids, staurosporine, aflatoxin, and folate have not been reported in <em>S. bambusicola</em> in the enriched pathway.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/23/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The partial results of the secondary metabolites pathway enrichment. The size of the black dot indicates the number of genes; the larger the dot, the more the genes are enriched. The blue shade indicates the significance of enrichment: the lighter the color, the closer it is to 1, the more significant is the enrichment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-980101705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The partial results of the secondary metabolites pathway enrichment. The size of the black dot indicates the number of genes; the larger the dot, the more the genes are enriched. The blue shade indicates the significance of enrichment: the lighter the color, the closer it is to 1, the more significant is the enrichment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/23/F5.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/1/23/F5.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/23/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/23/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495240" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">The partial results of the secondary metabolites pathway enrichment. The size of the black dot indicates the number of genes; the larger the dot, the more the genes are enriched. The blue shade indicates the significance of enrichment: the lighter the color, the closer it is to 1, the more significant is the enrichment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The secondary metabolite gene clusters were analyzed using the antiSMASH tool. A total of 28 gene clusters of secondary metabolites were found (Table S24). Among them were 14 T1PKS, 1 T3PKS, 4 NRPS, 4 terpenes, 3 T1PKS-nrps, and 2 undefined gene clusters (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). One T3PKS gene cluster, which is the key synthase in the hypocrellin synthetic process, was obtained in <em>S. bambusicola</em> (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2016</a>). T3PKS, representing chalcone synthase in plants, is a PKS similar to phenyl styrene ketone synthase discovered by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Funa <em>et al.</em> 2007</a>. This synthase directly catalyzes the condensation of pancosyl-CoA without ACP and is mainly responsible for the biosynthesis of monocyclic or bicyclic aromatic polyketides (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Peng <em>et al.</em> 2015</a>). A total of 17 genes and 12 CDS were predicted in the T3PKS gene clusters using the antiSMASH tool. The T3PKS gene clusters were located on scaffold 1 (locus: 553639–596757). and successively encoded 12 proteins (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495251/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495251/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">The secondary metabolite synthesis gene clusters in <em>Shiraia bambusicola</em></span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/1/23/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="T3PKS gene cluster analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-980101705" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;T3PKS gene cluster analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/1/23/F6.medium.gif" width="440" height="421"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/1/23/F6.medium.gif" width="440" height="421"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/1/23/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/1/23/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495290" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">T3PKS gene cluster analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Gene cluster analysis revealed that the T3PKS gene clusters of 10 species were similar to that of <em>S. bambusicola</em>; however, the similarities were poor. The T3PKS gene cluster revealed that 11% of the gene is similar to <em>S. bambusicola</em> and other 7 species, and 17% of the genes show a similarity between <em>S. bambusicola</em> and other 3 species. Among them, the condensing enzymes and GlcD- and MTHFR-coding regions of <em>S. bambusicola T3PKS</em> were similar to those of <em>P. nodorum</em>, <em>Altererythrobacter</em> sp., the <em>E. marinus</em> strain HWDM-33, <em>Erythrobacter</em> sp. s21-N3, <em>Erythrobacter</em> sp. KA37, and the <em>E. gangjinensis</em> strain K7-2. The condensing enzymes and arabinosidase-coding regions of the <em>S. bambusicola T3PKS</em> were similar to those of <em>A. parasiticus</em> SU-1 363. The proteasome, zinc finger protein, and DNase I-like protein-coding regions were similar between <em>S. bambusicola</em> and the <em>A. rambellii</em> strain SRRC1468 and <em>A. ochraceoroseus</em> strain SRRC1432. The DUF572-like protein, ANAPC3, TPR-like proteins, and GlcD- and MTHFR-coding regions of <em>S. bambusicola T3PKS</em> and <em>B. fuckeliana</em> BcDW1 unplaced genome were similar (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6 B</a>).</p></div></div><div class="section discussion" id="sec-28"><h2 class="">Discussion</h2><p id="p-47"><em>Shiraia bambusicola</em> is a cherished and endangered medicinal fungus and also a pathogen causing bamboo red dumpling disease. The entire genome of <em>S. bambusicola</em> GZAAS2.1243 was sequenced, assembled, and analyzed in this study. The <em>S. bambusicola</em> genome was estimated to be of 33 Mb, which is similar to the genome size of most of the sequenced strains of Pleosporales. The composition of this genome, including gene numbers, exons, introns, repeats, ncRNAs, and transposons, was revealed through interpretation of the assembly data. Data related to its genetic constituent were systematically analyzed. The classification status was confirmed based on the genomic information and was found to be identical to the findings of our previous study. <em>Shiraia bambusicola</em> GZAAS2.1243, <em>Shiraia</em> sp. slf14, and <em>S. bambusicola</em> RNA Seq were clustered to the same branch and were close to other strains of Pleosporales. It was speculated that there is more than one genus or one species in Shiraiaceae. This conclusion was different from the standpoints of previous researchers — only one species or one genus of the Shiraiaceae (Krik 2001; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2013</a>). Among the total 1855 pathogen–host interaction genes in <em>S. bambusicola</em>, 777 proteins were potentially secreted proteins. The reduced virulence-related protein was the most in number, followed by unaffected pathogenicity and loss of pathogenicity. Among the 1563 carbohydrate enzymes, the most common was GH enzymes, followed by GT and CBM enzymes. The number of cytochrome <em>P</em>450 enzymes and proteases was 900 and 493, respectively.</p><p id="p-48">It was gradually realized that <em>S. bambusicola</em> has a high application development value with deep research. The fruiting body could be immersed in wine and used as medicine; moreover, various medicinal components could be extracted from its fruiting body or culture. In this study, 252 <em>S. bambusicola</em>-specific genes were obtained by screening the secondary metabolism-related genes and enriched into 88 pathways. Among them, to the best of our knowledge, the terpenoids, staurosporine, aflatoxin, and folate synthesis pathways remain unreported to date. The terpenoids have many physiological activities, such as expectorant, relieving cough, dispelling wind, diaphoresis, deworming, and acting as an analgesic. Staurosporine is a typical ATP-competitive kinase inhibitor and a potential anticancer drug. Folate plays an essential role in the utilization of sugar and amino acids by <em>S. bambusicola</em> and is required for cell growth and reproduction. This study provides supporting evidence for the analgesic, wind-dispelling, and anticancer effects of terpenoids, which have previously been reported. It was speculated that terpenoids, staurosporine, and folate are synthesized in <em>S. bambusicola</em>.</p><p id="p-49">The researchers have always been concerned about the medicinal value of <em>S. bambusicola</em>. Previously, to the best of our knowledge, there were no reports describing whether <em>S. bambusicola</em> is poisonous. In this study, it was found that there are six genes enriched for the aflatoxin synthesis pathway. These genes are involved in the regulation of aflatoxin synthesis; however, whether aflatoxin is synthesized in <em>S. bambusicola</em> needs to be confirmed by further experiments. A total of 28 secondary metabolite synthesis gene clusters were predicted using the antiSMASH tool. It is noteworthy that the T3PKS gene clusters might comprise the key enzymes of hypocrellin synthesis (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2016</a>) in <em>S. bambusicola</em> GZAAS2.1243. This gene cluster comprises 12 coding regions, and its core coding region is a condensing enzyme superfamily. The T3PKS gene cluster is similar to that of 10 other species, but the similarities are low. There is a high similarity of the <em>T3PKS</em> gene cluster between <em>S. bambusicola</em> and the <em>A. rambellii</em> strain SRRC1468, the <em>A. ochraceoroseus</em> strain SRRC1432, and <em>B. fuckeliana</em> BcDW1; however, this similarity is only 17%.</p><p id="p-50">Genomic data of the <em>S. bambusicola</em> GZAAS2.1243 strain were systematically and comprehensively analyzed. The taxonomic status was verified based on the genomic information for the first time. To determine the pathogenic and secondary metabolite synthesis mechanisms, the related genome information was analyzed. We believe that this study will be a reference for exploring the <em>S. bambusicola</em>–host interaction genes. Further, this study establishes the foundation for future studies on the infection mechanism between <em>S. bambusicola</em> and the host. This study is expected to provide a basis to find the biosynthesis and the regulation genes’ information on secondary metabolism. We believe that this study will help accelerate the interpretation of the genetic basis of <em>S. bambusicola</em>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">The authors are grateful to the staff of the Guizhou Key Laboratory of Agricultural Biotechnology for contributing to this research and also would like to acknowledge the referees who took the time to review our manuscript and provide comments, which have considerably enhanced our manuscript. This work was financially supported by the National Natural Science of China (grant no. 31460199) and the Scientific and Technological Cooperation Projects of Guizhou province (grant no. LH [2015] 7064). Xiyi Ren performed the research work and drafted the manuscript; Yongxiang Liu initiated the research work; Zuoyi Liu and Xuanli Jiang conceived and designed the study and manuscript. Yumei Tan and Yonghui Huang participate in performing the experiments, genome sequencing preparations, and bioinformatics analyses. All authors have read, edited and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Equal contributors.</p></li><li class="fn" id="fn-6"><p id="p-52">Supplemental material available at figshare: <a href="https://figshare.com/s/ffefab4fb2879489cb02">https://figshare.com/s/ffefab4fb2879489cb02</a>.</p></li><li class="fn" id="fn-7"><p id="p-53"><em>Communicating editor: J. Ma</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-06"><span class="received-label">Received </span>September 6, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-11-06"><span class="accepted-label">Accepted </span>November 6, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Ren <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amano</span>,  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Studies on the japanese loculoascomycetes. ii. taxonomic postion of the genus shiraia.</span> <abbr class="cit-jnl-abbrev">Bulletin of the National Science Museum</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">60</span>. <a href="http://europepmc.org/abstract/AGR/IND81056261">http://europepmc.org/abstract/AGR/IND81056261</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BNational%2BScience%2BMuseum%26rft.volume%253D6%26rft.spage%253D55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.2" data-doi="10.1093/nar/27.2.573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Tandem repeats finder: a program to analyze dna sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/27.2.573</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.2.573%26rft_id%253Dinfo%253Apmid%252F9862982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.2.573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9862982&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078181500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.3" data-doi="10.1101/gr.1865504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Clamp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">GeneWise and Genomewise.</span> <abbr class="cit-jnl-abbrev">Genome Res</abbr>, <span class="cit-vol">14</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">995</span>. <a href="https://doi.org/10.1101/gr.1865504">https://doi.org/10.1101/gr.1865504</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBirney%26rft.auinit1%253DE.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D988%26rft.epage%253D995%26rft.atitle%253DGeneWise%2Band%2BGenomewise%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1865504%26rft_id%253Dinfo%253Apmid%252F15123596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzUvOTg4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMTAvMS8yMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Using Repeat Masker to identify repetitive elements in genomic sequences.</span> <abbr class="cit-jnl-abbrev">Current protocols in bioinformatics</abbr>, <span class="cit-vol">5</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">10</span>. <a href="https://doi.org/10.1002/0471250953.bi0410s05">https://doi.org/10.1002/0471250953.bi0410s05</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bprotocols%2Bin%2Bbioinformatics%26rft.volume%253D5%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.5" data-doi="10.1002/jobm.200410434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. M.</span> <span class="cit-name-surname">Jia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. H.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. P.</span> <span class="cit-name-surname">Lin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. H.</span> <span class="cit-name-surname">Zhao</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phylogenetic study on shiraia bambusicola by rdna sequence analyses.</span> <abbr class="cit-jnl-abbrev">J. Basic Microbiol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jobm.200410434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbasic%2Bmicrobiology%26rft.stitle%253DJ%2BBasic%2BMicrobiol%26rft.aulast%253DCheng%26rft.auinit1%253DT.%2BF.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DPhylogenetic%2Bstudy%2Bon%2BShiraia%2Bbambusicola%2Bby%2BrDNA%2Bsequence%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjobm.200410434%26rft_id%253Dinfo%253Apmid%252F15378525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jobm.200410434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15378525&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224492300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.6" data-doi="10.1371/journal.pone.0150719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wu</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">REPdenovo: inferring de novo repeat motifs from short sequence reads.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0150719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0150719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0150719%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0150719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0150719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.7" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gotz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Garcia-Gomez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Terol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Talon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2go: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti610</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16081474&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231694600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.8" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Muscle: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.23.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Expósito Villén</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Aránega</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Franco</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Functional role of non-coding RNAs during epithelial-to-mesenchymal transition.</span> <abbr class="cit-jnl-abbrev">Non-coding RNA.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">14</span>. <a href="https://doi.org/10.3390/ncrna4020014">https://doi.org/10.3390/ncrna4020014</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNon-coding%2BRNA.%26rft.volume%253D4%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.10" data-doi="10.1074/jbc.M701239200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Funa</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Awakawa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Horinouchi</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Pentaketide resorcylic acid synthesis by type iii polyketide synthase from neurospora crassa.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">14476</span>–<span class="cit-lpage">14481</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M701239200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M701239200%26rft_id%253Dinfo%253Apmid%252F17374612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvMTkvMTQ0NzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xMC8xLzIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.11" data-doi="10.1093/abbs/gmw093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">He</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Shao</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Ricin-b-lectin enhances microsporidia nosema bombycis infection in bmn cells from silkworm bombyx mori.</span> <abbr class="cit-jnl-abbrev">Acta Biochimica et Biophysica Sinica.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1050</span>–<span class="cit-lpage">1057</span>. <a href="https://doi.org/10.1093/abbs/gmw09">https://doi.org/10.1093/abbs/gmw09</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BBiochimica%2Bet%2BBiophysica%2BSinica.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fabbs%252Fgmw093%26rft_id%253Dinfo%253Apmid%252F27649890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/abbs/gmw093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27649890&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hennings</span>,  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1900</span> <span class="cit-article-title">Fungi japonica.</span> <abbr class="cit-jnl-abbrev">Bot. Jahrb. Syst. Pflanzengesch. Pflanzengeogr.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">280</span>. <a href="https://ci.nii.ac.jp/naid/10011582776/">https://ci.nii.ac.jp/naid/10011582776/</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BJahrb.%2BSyst.%2BPflanzengesch.%2BPflanzengeogr.%26rft.volume%253D28%26rft.spage%253D259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.13" data-doi="10.1360/yc- 006&amp;ndash;1627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. H.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Zhuang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. P.</span> <span class="cit-name-surname">Lin</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The progress of biological research of medicinal fungus shiraia bambusicola.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1360/yc-006–1627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.volume%253D33%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.1360%252Fyc-%2B006%2526ndash%253B1627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1360/yc- 006&amp;ndash;1627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.23.14" data-doi="10.1007/978-1-62703-568-2_20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">N.</span></span></li></ol><cite> <span class="cit-pub-date">2013</span> Overview of repeat annotation and de novo repeat identification.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-62703-568-2_20</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D1057%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-568-2_20%26rft_id%253Dinfo%253Apmid%252F23918436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-568-2_20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918436&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.15" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Blat-the blast-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMTAvMS8yMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.16" data-doi="10.1016/j.fgb.2010.06.003"><div class="cit-metadata"><cite>Khaldi, N., F. T. Seifuddin, G. Turner, D. Haft, W. C. Nierman <em>et al</em>., 2010 Smurf: genomic mapping of fungal secondary metabolite clusters. Fungal Genetics &amp; Biology Fg &amp; B, 47: 736–741.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2010.06.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DKhaldi%26rft.auinit1%253DN.%26rft.volume%253D47%26rft.issue%253D9%26rft.spage%253D736%26rft.epage%253D741%26rft.atitle%253DSMURF%253A%2BGenomic%2Bmapping%2Bof%2Bfungal%2Bsecondary%2Bmetabolite%2Bclusters.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2010.06.003%26rft_id%253Dinfo%253Apmid%252F20554054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2010.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20554054&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280933400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-10.1.23.17" data-doi="10.1590/S0036&amp;ndash;46651996000400018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirk</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Cannon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">David</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Stalpers</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source">Dictionary of the Fungi</span>, <span class="cit-edition">Ed. 9th</span>. <span class="cit-publ-name">CAB international</span>, <span class="cit-publ-name">Wallingford, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S0036–46651996000400018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDictionary%2Bof%2Bthe%2BFungi%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS0036%2526ndash%253B46651996000400018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S0036&amp;ndash;46651996000400018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.23.18" data-doi="10.1016/j.ympev.2010.04.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Meusemann</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fasconcat: convenient handling of data matrices.</span> <abbr class="cit-jnl-abbrev">Molecular Phylogenetics and Evolution</abbr>, <span class="cit-vol">56</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1118</span>. <a href="https://doi.org/10.1016/j.ympev.2010.04.024">https://doi.org/10.1016/j.ympev.2010.04.024</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D56%26rft.spage%253D1115%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2010.04.024%26rft_id%253Dinfo%253Apmid%252F20416383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2010.04.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20416383&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.19" data-doi="10.1186/1471-2105-12-124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lechner</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Findeiss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Steiner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Stadler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Proteinortho: detection of (co-)orthologs in large-scale analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-124</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLechner%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.spage%253D124%26rft.epage%253D124%26rft.atitle%253DProteinortho%253A%2Bdetection%2Bof%2B%2528co-%2529orthologs%2Bin%2Blarge-scale%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-124%26rft_id%253Dinfo%253Apmid%252F21526987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21526987&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.20" data-doi="10.1007/s11427-013-4544-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LiangHu</span>,  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Non-coding rna annotation:deciphering the second genetic code.</span> <abbr class="cit-jnl-abbrev">Sci. China Life Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">866</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11427-013-4544-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BChina%2BLife%2BSci.%26rft.volume%253D56%26rft.spage%253D865%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11427-013-4544-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11427-013-4544-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.21" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><cite>Li, H., and R. Durbin, 2009 Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25: 1754–1760.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.22" data-doi="10.1038/nature08696"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">R. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Fan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">He</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The sequence and de novo assembly of the giant panda genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">317</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08696</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DR.%26rft.volume%253D463%26rft.issue%253D7279%26rft.spage%253D311%26rft.epage%253D317%26rft.atitle%253DThe%2Bsequence%2Band%2Bde%2Bnovo%2Bassembly%2Bof%2Bthe%2Bgiant%2Bpanda%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08696%26rft_id%253Dinfo%253Apmid%252F20010809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08696&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20010809&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273748100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.23"><div class="cit-metadata"><cite>Li, X., J. Gao, Y. Yue, and C. Hou, 2009 Studies on systematics, biology and bioactive substance of Shiraia bambusicola. Forest Research, Beijing, 22: 279–284.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3321/j</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span><span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Chen</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Shiraia bambusicola, a medicinal fungi needed to be developed. Journ. of Zhej</span>. <abbr class="cit-jnl-abbrev">Forest. Sci. and Tech.</abbr>, <span class="cit-vol">22</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">71</span>. <a href="https://doi.org/10.1006/jfls.2001.0409">https://doi.org/10.1006/jfls.2001.0409</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest.%2BSci.%2Band%2BTech.%26rft.volume%253D22%26rft.spage%253D77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Quan</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Basic Nutrition for Stroma Development of Shiraia bambusicola and Hypocrellin A(HA) Yield by Mycelium Fermentation[J].</span> <abbr class="cit-jnl-abbrev">Guiz. Agri. Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">102</span>. <a href="https://doi.org/10.3969/j.issn.1001-3601.2011.08.029">https://doi.org/10.3969/j.issn.1001-3601.2011.08.029</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGuiz.%2BAgri.%2BSci.%26rft.volume%253D39%26rft.spage%253D98%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.26" data-doi="10.11646/phytotaxa.103.1.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. D.</span> <span class="cit-name-surname">Hyde</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Ariyawansa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. Q.</span> <span class="cit-name-surname">Zhou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Shiraiaceae, new family of Pleosporales (Dothideomycetes, Ascomycota).</span> <abbr class="cit-jnl-abbrev">Phytotaxa</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.11646/phytotaxa.103.1.4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytotaxa%26rft.volume%253D103%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.11646%252Fphytotaxa.103.1.4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.11646/phytotaxa.103.1.4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lombard</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fontecave</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Touati</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nivière</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Reaction of the desulfoferodoxin from desulfoarculus baarsii with superoxid anion. evidence for a superoxide reductase activity. Journal of Biological Chemistry 275. <a href="https://doi.org/10.1074/jbc.275.1.115">https://doi.org/10.1074/jbc.275.1.115</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lombard</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Hemalatha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Elodie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Coutinho</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span><span class="cit-name-surname">Bernard</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> The carbohydrate-active enzymes database (cazy) in 2013. Nucleic Acids Research(D1), D1. <a href="https://doi.org/10.1093/nar/gkt1178">https://doi.org/10.1093/nar/gkt1178</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.23.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>2012 Preparation of a kind of photosensitive germicidal dishwashing detergent. Journ. of Anhui Agr. Sci. 40: 1942–1943.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.13989/j.cnki</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.23.30" data-doi="10.1186/1471-2164-13-646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Ballester</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L. F.</span> <span class="cit-name-surname">Beatriz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Harries</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Marcos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequence of the necrotrophic fungus penicillium digitatum, the main postharvest pathogen of citrus.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMarcet-Houben%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D646%26rft.epage%253D646%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bnecrotrophic%2Bfungus%2BPenicillium%2Bdigitatum%252C%2Bthe%2Bmain%2Bpostharvest%2Bpathogen%2Bof%2Bcitrus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-646%26rft_id%253Dinfo%253Apmid%252F23171342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23171342&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.31" data-doi="10.1093/nar/gkr466"><div class="cit-metadata"><cite>Medema, M. H., B. Kai, C. Peter, D. J. Victor, Z. Piotr <em>et al., </em>2011 Antismash: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences. Nucleic Acids Research (suppl_2), 39(WebServer issue): W339–346.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr466</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr466%26rft_id%253Dinfo%253Apmid%252F21672958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21672958&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292325300055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.32" data-doi="10.1186/1471-2202-11-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. A.</span> <span class="cit-name-surname">Qureshi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gokhan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Long noncoding rnas in neuronal-glial fate specification and oligodendrocyte lineage maturation.</span> <abbr class="cit-jnl-abbrev">BMC Neuro.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">14</span>. <a href="https://doi.org/10.1186/1471-2202-11-14">https://doi.org/10.1186/1471-2202-11-14</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bneuroscience%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BNeurosci%26rft.aulast%253DMercer%26rft.auinit1%253DT.%2BR.%26rft.volume%253D11%26rft.spage%253D14%26rft.epage%253D14%26rft.atitle%253DLong%2Bnoncoding%2BRNAs%2Bin%2Bneuronal-glial%2Bfate%2Bspecification%2Band%2Boligodendrocyte%2Blineage%2Bmaturation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2202-11-14%26rft_id%253Dinfo%253Apmid%252F20137068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2202-11-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20137068&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.33" data-doi="10.1186/1471-2164-13-525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moktali</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Fedorova-Abrams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Choi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Systematic and searchable classification of cytochrome p450 proteins encoded by fungal and oomycete genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">525</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-525</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMoktali%26rft.auinit1%253DV.%26rft.volume%253D13%26rft.spage%253D525%26rft.epage%253D525%26rft.atitle%253DSystematic%2Band%2Bsearchable%2Bclassification%2Bof%2Bcytochrome%2BP450%2Bproteins%2Bencoded%2Bby%2Bfungal%2Band%2Boomycete%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-525%26rft_id%253Dinfo%253Apmid%252F23033934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23033934&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. L. </span><span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. L.</span> <span class="cit-name-surname">Wang</span></span> and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. </span><span class="cit-name-surname">Zhu</span></span></li></ol><cite>, 2015 The prokaryotic expression, purification and bioinformatics of type III polyketide synthase from Shiraia sp. Slf14, which is an endophytic fungus of Huperzia serrata. J. Jiangxi Norm. Univ. 39: 430–434. <a href="https://doi.org/10.16357/j.cnki.issn1000-5862.2015.04.19">https://doi.org/10.16357/j.cnki.issn1000-5862.2015.04.19</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.35" data-doi="10.1093/nar/gkm954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rawlings</span>,  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Tolle</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Barrett</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Merops: the peptidase database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D320</span>–<span class="cit-lpage">D325</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm954%26rft_id%253Dinfo%253Apmid%252F17991683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17991683&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252545400059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.23.36" data-doi="10.1101/gr.10.4.516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>,  <span class="cit-name-given-names">A. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. V.</span> <span class="cit-name-surname">Solovyev</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Ab initio gene finding in drosophila genomic dna.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.4.516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSalamov%26rft.auinit1%253DA.%2BA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D516%26rft.epage%253D522%26rft.atitle%253DAb%2Binitio%2BGene%2BFinding%2Bin%2BDrosophila%2BGenomic%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.516%26rft_id%253Dinfo%253Apmid%252F10779491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMTAvMS8yMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saccardo</span>,  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1902</span> <span class="cit-article-title">De Diagnostica et nomenclatura mycologica, Admonita quaedam[M].</span> <abbr class="cit-jnl-abbrev">wydawca nieznany</abbr>. <a href="https://doi.org/10.1016/S0007-1536(63)80110-2">https://doi.org/10.1016/S0007-1536(63)80110-2</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.38" data-doi="10.1093/bioinformatics/18.3.502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Strimmer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vingron</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">von Haeseler</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">502</span>–<span class="cit-lpage">504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/18.3.502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DSchmidt%26rft.auinit1%253DH.%2BA.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D502%26rft.epage%253D504%26rft.atitle%253DTREE-PUZZLE%253A%2Bmaximum%2Blikelihood%2Bphylogenetic%2Banalysis%2Busing%2B%2Bquartets%2Band%2Bparallel%2Bcomputing%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.3.502%26rft_id%253Dinfo%253Apmid%252F11934758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.3.502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11934758&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174708500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">He</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stability and anti-tumor activity of hypocrellin from liquid-cultured mycelia of shiraia bambusicola strain zh-5-1.</span> <abbr class="cit-jnl-abbrev">Food Science</abbr>, <span class="cit-vol">32</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">100</span>. <a href="https://doi.org/10.1080/10255842.2010.522188">https://doi.org/10.1080/10255842.2010.522188</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BScience%26rft.volume%253D32%26rft.spage%253D96%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.40" data-doi="10.3321/j.issn:1001&amp;ndash;5302.2002.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. G.</span> <span class="cit-name-surname">Rong</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. H.</span> <span class="cit-name-surname">Gao</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">studies on the chemical constituents of shiraia bambusicola. Zhongguo Zhong yao za zhi = Zhongguo zhongyao zazhi =</span>. <abbr class="cit-jnl-abbrev">Zhongguo Zhongyao Zazhi</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">674</span>–<span class="cit-lpage">676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3321/j.issn:1001–5302.2002.09.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZhongguo%2BZhongyao%2BZazhi%26rft.volume%253D27%26rft.spage%253D674%26rft_id%253Dinfo%253Adoi%252F10.3321%252Fj.issn%253A1001%2526ndash%253B5302.2002.09.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3321/j.issn:1001&amp;ndash;5302.2002.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Study on fermentation mediumfor new pigment hypocrellin.</span> <abbr class="cit-jnl-abbrev">Food Res. &amp; Devel.</abbr> <span class="cit-vol">43</span>:<span class="cit-issue">62</span> <span class="cit-fpage">182</span>–<span class="cit-lpage">185</span>. <a href="https://doi.org/10.3969/j.issn.1005%E2%80%936521">https://doi.org/10.3969/j.issn.1005–6521</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BRes.%2B%2526%2BDevel.%26rft.volume%253D43%26rft.spage%253D182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.42" data-doi="10.1093/bioinformatics/btg1080"><div class="cit-metadata"><cite>Stanke, M., and S. Waack, 2003 Gene prediction with a hidden markov model and a new intron submodel. Bioinformatics, 19(Suppl 2), ii215-ii225.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg1080</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D90002%26rft.spage%253Dii215%26rft.epage%253D225%26rft.atitle%253DGene%2Bprediction%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%2Band%2Ba%2Bnew%2Bintron%2B%2Bsubmodel%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg1080%26rft_id%253Dinfo%253Apmid%252F14534192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg1080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14534192&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000207434300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.43" data-doi="10.1038/nsmb.1841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taft</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Simons</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nahkuri</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nuclear-localized tiny rnas are associated with transcription initiation and splice sites in metazoans.</span> <abbr class="cit-jnl-abbrev">Nature Structural &amp; Mol. Bio.</abbr>, <span class="cit-vol">17</span>: <span class="cit-fpage">1030</span>–<span class="cit-lpage">1034</span>. <a href="https://doi.org/10.1038/nsmb.1841">https://doi.org/10.1038/nsmb.1841</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DTaft%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D1030%26rft.epage%253D1034%26rft.atitle%253DNuclear-localized%2Btiny%2BRNAs%2Bare%2Bassociated%2Bwith%2Btranscription%2Binitiation%2Band%2Bsplice%2Bsites%2Bin%2Bmetazoans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1841%26rft_id%253Dinfo%253Apmid%252F20622877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20622877&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280670300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.44" data-doi="10.1080/10635150701472164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talavera</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Castresana</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">577</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10635150701472164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701472164%26rft_id%253Dinfo%253Apmid%252F17654362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150701472164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17654362&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248359900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.45"><div class="cit-metadata"><cite>Wang J. X., L. M. Zhang and L. Q. Zhu, 1990 Study on chemical components of Shiraia bambusicola. Chinese herbs 21: 4–5.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7501/j.issn.0253–2670.1990.7.093</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.46" data-doi="10.1007/978-1-60327-241-4_17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Washietl</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequence and structure analysis of noncoding RNAs.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">609</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">306</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-60327-241-4_17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DWashietl%26rft.auinit1%253DS.%26rft.volume%253D609%26rft.spage%253D285%26rft.epage%253D306%26rft.atitle%253DSequence%2Band%2Bstructure%2Banalysis%2Bof%2Bnoncoding%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-241-4_17%26rft_id%253Dinfo%253Apmid%252F20221926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-241-4_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20221926&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.47" data-doi="10.1093/nar/gkm858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winnenburg</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Urban</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Beacham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. K.</span> <span class="cit-name-surname">Baldwin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sabrina</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phi-base update: additions to the pathogen-host interaction database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D572</span>–<span class="cit-lpage">D576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm858</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm858%26rft_id%253Dinfo%253Apmid%252F17942425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17942425&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252545400103&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.48"><div class="cit-metadata"><cite>Yamamoto, M., 2018 Repetitive Sequences. The Allium Genomes.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-95825-5_7</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.49" data-doi="10.1093/nar/gks479"><div class="cit-metadata"><cite>Yin, Y., X. Mao, J. Yang, X. Chen, F. Mao, <em>et al</em>., 2012 dbCAN: a web resource for automated carbohydrate-active enzyme annotation. Nucleic Acids Res,40(Web Server issue): W445–51.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks479</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks479%26rft_id%253Dinfo%253Apmid%252F22645317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22645317&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F1%2F23.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306670900073&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.50" data-doi="10.3969/j.issn.1674&amp;ndash;4020.2012.01.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. H.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Tian</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Clinical observation of curative effect on Focusedultrasound combined with hypocrellinointment to treat leukoplakia vulvae in 164 cases.</span> <abbr class="cit-jnl-abbrev">Chinese Journal of Family Planning &amp; Gynecotokology</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3969/j.issn.1674–4020.2012.01.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChinese%2BJournal%2Bof%2BFamily%2BPlanning%2B%2526%2BGynecotokology%26rft.volume%253D4%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.3969%252Fj.issn.1674%2526ndash%253B4020.2012.01.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3969/j.issn.1674&amp;ndash;4020.2012.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.1.23.51" data-doi="10.3390/56ijms17030311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Shan-Shan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhi-Mei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Shuang-Lin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">De novo transcriptome assembly in shiraia bambusicola to investigate putative genes involved in the biosynthesis of hypocrellin a.</span> <abbr class="cit-jnl-abbrev">International Journal of Molecular Sciences</abbr>, <span class="cit-vol">17</span>: <span class="cit-fpage">311</span>. <a href="https://doi.org/10.3390/56ijms17030311">https://doi.org/10.3390/56ijms17030311</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BMolecular%2BSciences%26rft.volume%253D17%26rft.spage%253D311%26rft_id%253Dinfo%253Adoi%252F10.3390%252F56ijms17030311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/56ijms17030311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.1.23.52" data-doi="10.3969/j.issn.0412&amp;ndash;0914.2014.01.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Q. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Inhibition of hypocrellin a against 18 pathogenic fungi.</span> <abbr class="cit-jnl-abbrev">Acta Phytopathologica Sin.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3969/j.issn.0412–0914.2014.01.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BPhytopathologica%2BSin.%26rft.volume%253D44%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.3969%252Fj.issn.0412%2526ndash%253B0914.2014.01.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3969/j.issn.0412&amp;ndash;0914.2014.01.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/1/23.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/1/13" title="Comparative Analysis of Whole-Genome and Methylome Profiles of a Smooth and a Rough Mycobacterium abscessus Clinical Strain" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/1/37" title="Genome Sequencing of Musa acuminata Dwarf Cavendish Reveals a Duplication of a Large Segment of Chromosome 2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/1"><span>Volume 10 Issue 1, January 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/1.cover-source.jpg?itok=JNp-Wa6o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/1/23" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e7KCOqg09vYoGQxGu99h6OK7ktq0RpvEMDfU7fJUXcs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/1/23" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/1/23" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/1/23" class="active" data-icon-position="" data-hide-link-title="0">Sequencing and Functional Annotation of the Whole Genome of Shiraia bambusicola</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495264" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-DkI9WZ-tMRhr1e12g_SRTQ9K6wqQjGPzuhkNnBkvNS0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/23.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/1/23" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3BmBLrTK65D8h5Dcp-YsVO6ipAlZ4tHsi9WwLRSPGAQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/1/23" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/1/23.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495264" id="node495264--2" data-pisa="ggg;10/1/23" data-pisa-master="ggg;g3.119.400694" data-apath="/ggg/10/1/23.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequencing and Functional Annotation of the Whole Genome of <em>Shiraia bambusicola</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xiyi Ren</span>, <span class="highwire-citation-author" data-delta="1">Yongxiang Liu</span>, <span class="highwire-citation-author" data-delta="2">Yumei Tan</span>, <span class="highwire-citation-author" data-delta="3">Yonghui Huang</span>, <span class="highwire-citation-author" data-delta="4">Zuoyi Liu</span> and <span class="highwire-citation-author" data-delta="5">Xuanli Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">23-35;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400694 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495264--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiyi Ren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiyi%2BRen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ren%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiyi%2BRen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yongxiang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yongxiang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYongxiang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yumei Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumei%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYumei%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yonghui Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonghui%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYonghui%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Zuoyi Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zuoyi%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZuoyi%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jxl3237@163.com" class="" data-icon-position="" data-hide-link-title="0">jxl3237@163.com</a>
<a href="mailto:gzliuzuoyi@163.com" class="" data-icon-position="" data-hide-link-title="0">gzliuzuoyi@163.com</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xuanli Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuanli%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuanli%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jxl3237@163.com" class="" data-icon-position="" data-hide-link-title="0">jxl3237@163.com</a>
<a href="mailto:gzliuzuoyi@163.com" class="" data-icon-position="" data-hide-link-title="0">gzliuzuoyi@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/1/23" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/1/23&amp;title=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/1/23&amp;title=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/1/23&amp;title=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/1/23&amp;text=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/1/23&amp;title=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/1/23&amp;t=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/1/23&amp;title=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/1/23&amp;title=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495264/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495264" id="node495264--3" data-pisa="ggg;10/1/23" data-pisa-master="ggg;g3.119.400694" data-apath="/ggg/10/1/23.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequencing and Functional Annotation of the Whole Genome of <em>Shiraia bambusicola</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xiyi Ren</span>, <span class="highwire-citation-author" data-delta="1">Yongxiang Liu</span>, <span class="highwire-citation-author" data-delta="2">Yumei Tan</span>, <span class="highwire-citation-author" data-delta="3">Yonghui Huang</span>, <span class="highwire-citation-author" data-delta="4">Zuoyi Liu</span> and <span class="highwire-citation-author" data-delta="5">Xuanli Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">23-35;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400694 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495264--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiyi Ren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiyi%2BRen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ren%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiyi%2BRen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yongxiang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yongxiang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYongxiang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yumei Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumei%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYumei%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yonghui Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of biotechnology, Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonghui%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYonghui%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Zuoyi Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Guizhou Key Laboratory of Agricultural Biotechnology, Guiyang, Guizhou, 550006, China, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Guizhou Academy of Agricultural Sciences, Guiyang, Guizhou, 550006, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zuoyi%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZuoyi%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jxl3237@163.com" class="" data-icon-position="" data-hide-link-title="0">jxl3237@163.com</a>
<a href="mailto:gzliuzuoyi@163.com" class="" data-icon-position="" data-hide-link-title="0">gzliuzuoyi@163.com</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xuanli Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Agriculture, Guizhou University, Guiyang, Guizhou, 550025, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuanli%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuanli%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jxl3237@163.com" class="" data-icon-position="" data-hide-link-title="0">jxl3237@163.com</a>
<a href="mailto:gzliuzuoyi@163.com" class="" data-icon-position="" data-hide-link-title="0">gzliuzuoyi@163.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=23&amp;rft.epage=35&amp;rft.atitle=Sequencing%20and%20Functional%20Annotation%20of%20the%20Whole%20Genome%20of%20Shiraia%20bambusicola&amp;rft.volume=10&amp;rft.issue=1&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ren%2C+Xiyi&amp;rft.au=Liu%2C+Yongxiang&amp;rft.au=Tan%2C+Yumei&amp;rft.au=Huang%2C+Yonghui&amp;rft.au=Liu%2C+Zuoyi&amp;rft.au=Jiang%2C+Xuanli"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495264/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495264/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495264/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495264/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495264/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495264/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495264/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495264/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495264/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495264/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495264/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495264/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/1/23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Background</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and analysis</a></li><li class="odd"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/1/23.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/1/23.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/1/23.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
