<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/8/873" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/8/873.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/8/873.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/8/873.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/8/873.atom" />
<meta name="HW.pisa" content="ggg;2/8/873" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster" />
<meta name="DC.Identifier" content="10.1534/g3.112.003194" />
<meta name="DC.Date" content="2012-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Hong et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Faithful annotation of tissue-specific transcript isoforms is important not only to understand how genes are organized and regulated but also to identify potential novel, unannotated exons of genes, which may be additional targets of mutation in disease states or while performing mutagenic screens. We have developed a microarray enrichment methodology followed by long-read, next-generation sequencing for identification of unannotated transcript isoforms expressed in two Drosophila tissues, the ovary and the testis. Even with limited sequencing, these studies have identified a large number of novel transcription units, including 5′ exons and extensions, 3′ exons and extensions, internal exons and exon extensions, gene fusions, and both germline-specific splicing events and promoters. Additionally, comparing our capture dataset with tiling array and traditional RNA-seq analysis, we demonstrate that our enrichment strategy is able to capture low-abundance transcripts that cannot readily be identified by the other strategies. Finally, we show that our methodology can help identify transcriptional signatures of minority cell types within the ovary that would otherwise be difficult to reveal without the CoNECT enrichment strategy. These studies introduce an efficient methodology for cataloging tissue-specific transcriptomes in which specific classes of genes or transcripts can be targeted for capture and sequence, thus reducing the significant sequencing depth normally required for accurate annotation." />
<meta name="DC.Contributor" content="X. Hong" />
<meta name="DC.Contributor" content="H. Doddapaneni" />
<meta name="DC.Contributor" content="J. M. Comeron" />
<meta name="DC.Contributor" content="M. J. Rodesch" />
<meta name="DC.Contributor" content="H. A. Halvensleben" />
<meta name="DC.Contributor" content="C. Y. Nien" />
<meta name="DC.Contributor" content="F. Bolei" />
<meta name="DC.Contributor" content="R. Metpally" />
<meta name="DC.Contributor" content="T. A. Richmond" />
<meta name="DC.Contributor" content="T. J. Albert" />
<meta name="DC.Contributor" content="J. R. Manak" />
<meta name="article:published_time" content="2012-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Faithful annotation of tissue-specific transcript isoforms is important not only to understand how genes are organized and regulated but also to identify potential novel, unannotated exons of genes, which may be additional targets of mutation in disease states or while performing mutagenic screens. We have developed a microarray enrichment methodology followed by long-read, next-generation sequencing for identification of unannotated transcript isoforms expressed in two &lt;i&gt;Drosophila&lt;/i&gt; tissues, the ovary and the testis. Even with limited sequencing, these studies have identified a large number of novel transcription units, including 5′ exons and extensions, 3′ exons and extensions, internal exons and exon extensions, gene fusions, and both germline-specific splicing events and promoters. Additionally, comparing our capture dataset with tiling array and traditional RNA-seq analysis, we demonstrate that our enrichment strategy is able to capture low-abundance transcripts that cannot readily be identified by the other strategies. Finally, we show that our methodology can help identify transcriptional signatures of minority cell types within the ovary that would otherwise be difficult to reveal without the CoNECT enrichment strategy. These studies introduce an efficient methodology for cataloging tissue-specific transcriptomes in which specific classes of genes or transcripts can be targeted for capture and sequence, thus reducing the significant sequencing depth normally required for accurate annotation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/08/01" />
<meta name="citation_mjid" content="ggg;2/8/873" />
<meta name="citation_id" content="2/8/873" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/8/873" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/8/873.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/8/873.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/8/873.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003194" />
<meta name="citation_pmid" content="22908036" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="873" />
<meta name="citation_lastpage" content="882" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="X. Hong" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author" content="H. Doddapaneni" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author_institution" content="Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author" content="J. M. Comeron" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author" content="M. J. Rodesch" />
<meta name="citation_author_institution" content="Roche NimbleGen, Madison, Wisconsin 53719" />
<meta name="citation_author" content="H. A. Halvensleben" />
<meta name="citation_author_institution" content="Roche NimbleGen, Madison, Wisconsin 53719" />
<meta name="citation_author" content="C. Y. Nien" />
<meta name="citation_author_institution" content="Department of Biology, Center for Developmental Genetics, New York University, New York, New York 10003, and" />
<meta name="citation_author" content="F. Bolei" />
<meta name="citation_author_institution" content="Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author" content="R. Metpally" />
<meta name="citation_author_institution" content="Translational Genomics Research Institute, Phoenix, Arizona 85004" />
<meta name="citation_author" content="T. A. Richmond" />
<meta name="citation_author_institution" content="Roche NimbleGen, Madison, Wisconsin 53719" />
<meta name="citation_author" content="T. J. Albert" />
<meta name="citation_author_institution" content="Roche NimbleGen, Madison, Wisconsin 53719" />
<meta name="citation_author" content="J. R. Manak" />
<meta name="citation_author_institution" content="Department of Biology, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author_institution" content="Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Iowa, Iowa City, Iowa 52242" />
<meta name="citation_author_email" content="john-manak@uiowa.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. N. Bainbridge;citation_author=M. Wang;citation_author=D. L. Burgess;citation_author=C. Kovar;citation_author=M. J. Rodesch;citation_title=Whole exome capture in solution with 3 Gbp of data.;citation_pages=R62-R62;citation_volume=11;citation_year=2010;citation_issue=6;citation_pmid=20565776;citation_doi=10.1186/gb-2010-11-6-r62" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. S. Baker;citation_title=Sex in flies: the splice of life.;citation_pages=521-524;citation_volume=340;citation_year=1989;citation_issue=6234;citation_pmid=2505080;citation_doi=10.1038/340521a0" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. J. Bamshad;citation_author=S. B. Ng;citation_author=A. W. Bigham;citation_author=H. K. Tabor;citation_author=M. J. Emond;citation_title=Exome sequencing as a tool for Mendelian disease gene discovery.;citation_pages=745-755;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=21946919;citation_doi=10.1038/nrg3031" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=M. Buescher;citation_author=F. S. Hing;citation_author=W. Chia;citation_title=Formation of neuroblasts in the embryonic central nervous system of Drosophila melanogaster is controlled by SoxNeuro;citation_pages=4193-4203;citation_volume=129;citation_year=2002;citation_pmid=12183372" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_title=Finishing a whole-genome shotgun: release 3 of the Drosophila melanogaster euchromatic genome sequence;citation_author=S. E. Celniker;citation_author=D. A. Wheeler;citation_author=B. Kronmiller;citation_author=J. W. Carlson;citation_author=A. Halpern;citation_volume=3;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Cherbas;citation_author=A. Willingham;citation_author=D. Zhang;citation_author=L. Yang;citation_author=Y. Zou;citation_title=The transcriptional diversity of 25 Drosophila cell lines;citation_volume=21;citation_year=2011;citation_pmid=21177962;citation_doi=10.1101/gr.112961.110" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. R. Chintapalli;citation_author=J. Wang;citation_author=J. A. Dow;citation_title=Using FlyAtlas to identify better Drosophila melanogaster models of human disease.;citation_pages=715-720;citation_volume=39;citation_year=2007;citation_issue=6;citation_pmid=17534367;citation_doi=10.1038/ng2049" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biomed Biotechnol;citation_author=V. Costa;citation_author=C. Angelini;citation_author=I. De Feis;citation_author=A. Ciccodicola;citation_title=Uncovering the complexity of transcriptomes with RNA-Seq.;citation_pages=853916-853916;citation_volume=2010;citation_year=2010;citation_pmid=20625424" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=H. Fairfield;citation_author=G. J. Gilbert;citation_author=M. Barter;citation_author=R. R. Corrigan;citation_author=M. Curtain;citation_title=Mutation discovery in mice by whole exome sequencing.;citation_pages=R86-R86;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21917142;citation_doi=10.1186/gb-2011-12-9-r86" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=Y. Fu;citation_author=N. M. Springer;citation_author=D. J. Gerhardt;citation_author=K. Ying;citation_author=C. T. Yeh;citation_title=Repeat subtraction-mediated sequence capture from a complex genome.;citation_pages=898-909;citation_volume=62;citation_year=2010;citation_issue=5;citation_pmid=20230488;citation_doi=10.1111/j.1365-313X.2010.04196.x" />
<meta name="citation_reference" content="citation_journal_title=Cell research;citation_journal_abbrev=Cell Res;citation_author=Q. Gan;citation_author=I. Chepelev;citation_author=G. Wei;citation_author=L. Tarayrah;citation_author=K. Cui;citation_title=Dynamic regulation of alternative splicing and chromatin structure in Drosophila gonads revealed by RNA-seq.;citation_pages=763-783;citation_volume=20;citation_year=2010;citation_issue=7;citation_pmid=20440302;citation_doi=10.1038/cr.2010.64" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. R. Graveley;citation_author=A. N. Brooks;citation_author=J. W. Carlson;citation_author=M. O. Duff;citation_author=J. M. Landolin;citation_title=The developmental transcriptome of Drosophila melanogaster.;citation_pages=473-479;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21179090;citation_doi=10.1038/nature09715" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. M. Hilton;citation_author=M. A. Lewis;citation_author=M. Grati;citation_author=N. Ingham;citation_author=S. Pearson;citation_title=Exome sequencing identifies a missense mutation in Isl1 associated with low penetrance otitis media in dearisch mice.;citation_pages=R90-R90;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21936904;citation_doi=10.1186/gb-2011-12-9-r90" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=W. Huang da;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=T. Kai;citation_author=D. Williams;citation_author=A. C. Spradling;citation_title=The expression profile of purified Drosophila germline stem cells.;citation_pages=486-502;citation_volume=283;citation_year=2005;citation_issue=2;citation_pmid=15927177;citation_doi=10.1016/j.ydbio.2005.04.018" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. C. Kopczynski;citation_author=G. W. Davis;citation_author=C. S. Goodman;citation_title=A Neural Tetraspanin, Encoded by late bloomer, That Facilitates Synapse Formation;citation_pages=1867-1870;citation_volume=271;citation_year=1996;citation_issue=5257;citation_pmid=8596956;citation_doi=10.1126/science.271.5257.1867" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=C. A. Lavoie;citation_author=B. Ohlstein;citation_author=D. M. McKearin;citation_title=Localization and function of Bam protein require the benign gonial cell neoplasm gene product.;citation_pages=405-413;citation_volume=212;citation_year=1999;citation_issue=2;citation_pmid=10433830;citation_doi=10.1006/dbio.1999.9346" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Li;citation_author=N. T. Minor;citation_author=J. K. Park;citation_author=D. M. McKearin;citation_author=J. Z. Maines;citation_title=Bam and Bgcn antagonize Nanos-dependent germ-line stem cell maintenance;citation_volume=106;citation_year=2009;citation_pmid=19470484;citation_doi=10.1073/pnas.0901452106" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=J. H. Malone;citation_author=B. Oliver;citation_title=Microarrays, deep sequencing and the true measure of the transcriptome.;citation_pages=34-34;citation_volume=9;citation_year=2011;citation_pmid=21627854;citation_doi=10.1186/1741-7007-9-34" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. R. Manak;citation_author=S. Dike;citation_author=V. Sementchenko;citation_author=P. Kapranov;citation_author=F. Biemar;citation_title=Biological function of unannotated transcription during the early development of Drosophila melanogaster.;citation_pages=1151-1158;citation_volume=38;citation_year=2006;citation_issue=10;citation_pmid=16951679;citation_doi=10.1038/ng1875" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=T. R. Mercer;citation_author=D. J. Gerhardt;citation_author=M. E. Dinger;citation_author=J. Crawford;citation_author=C. Trapnell;citation_title=Targeted RNA sequencing reveals the deep complexity of the human transcriptome.;citation_pages=99-104;citation_volume=30;citation_year=2012;citation_issue=1;citation_pmid=22081020;citation_doi=10.1038/nbt.2024" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. L. Metzker;citation_title=Sequencing technologies - the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=C. A. Middleton;citation_author=U. Nongthomba;citation_author=K. Parry;citation_author=S. T. Sweeney;citation_author=J. C. Sparrow;citation_title=Neuromuscular organization and aminergic modulation of contractions in the Drosophila ovary.;citation_pages=17-17;citation_volume=4;citation_year=2006;citation_pmid=16768790;citation_doi=10.1186/1741-7007-4-17" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Y. Nien;citation_author=H. L. Liang;citation_author=S. Butcher;citation_author=Y. Sun;citation_author=S. Fu;citation_title=Temporal coordination of gene networks by Zelda in the early Drosophila embryo.;citation_pages=e1002339-e1002339;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22028675;citation_doi=10.1371/journal.pgen.1002339" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Ohlstein;citation_author=C. A. Lavoie;citation_author=O. Vef;citation_author=E. Gateff;citation_author=D. M. McKearin;citation_title=The Drosophila cystoblast differentiation factor, benign gonial cell neoplasm, is related to DExH-box proteins and interacts genetically with bag-of-marbles;citation_pages=1809-1819;citation_volume=155;citation_year=2000;citation_pmid=10924476" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. L. Page;citation_author=R. S. Khetani;citation_author=C. M. Lake;citation_author=R. J. Nielsen;citation_author=J. K. Jeffress;citation_title=Corona is required for higher-order assembly of transverse filaments into full-length synaptonemal complex in Drosophila oocytes.;citation_pages=e1000194-e1000194;citation_volume=4;citation_year=2008;citation_issue=9;citation_pmid=18802461;citation_doi=10.1371/journal.pgen.1000194" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. M. Pierson;citation_author=D. Adams;citation_author=F. Bonn;citation_author=P. Martinelli;citation_author=P. F. Cherukuri;citation_title=Whole-exome sequencing identifies homozygous AFG3L2 mutations in a spastic ataxia-neuropathy syndrome linked to mitochondrial m-AAA proteases.;citation_pages=e1002325-e1002325;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22022284;citation_doi=10.1371/journal.pgen.1002325" />
<meta name="citation_reference" content="citation_title=The Development of Drosophila melanogaster.;citation_author=A. C. Spradling;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. Takeo;citation_author=C. M. Lake;citation_author=E. Morais-de-Sa;citation_author=C. E. Sunkel;citation_author=R. S. Hawley;citation_title=Synaptonemal complex-dependent centromeric clustering and the initiation of synapsis in Drosophila oocytes.;citation_pages=1845-1851;citation_volume=21;citation_year=2011;citation_issue=21;citation_pmid=22036182;citation_doi=10.1016/j.cub.2011.09.044" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=H. I. Wan;citation_author=A. DiAntonio;citation_author=R. D. Fetter;citation_author=K. Bergstrom;citation_author=R. Strauss;citation_title=Highwire regulates synaptic growth in Drosophila;citation_pages=313-329;citation_volume=26;citation_year=2000;citation_pmid=10839352;citation_doi=10.1016/S0896-6273(00)81166-6" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=K. H. Wan;citation_author=C. Yu;citation_author=R. A. George;citation_author=J. W. Carlson;citation_author=R. A. Hoskins;citation_title=High-throughput plasmid cDNA library screening.;citation_pages=624-632;citation_volume=1;citation_year=2006;citation_issue=2;citation_pmid=17406289;citation_doi=10.1038/nprot.2006.90" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=57-63;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19015660;citation_doi=10.1038/nrg2484" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=B. T. Wilhelm;citation_author=J. R. Landry;citation_title=RNA-Seq-quantitative measurement of expression through massively parallel RNA-sequencing.;citation_pages=249-257;citation_volume=48;citation_year=2009;citation_issue=3;citation_pmid=19336255;citation_doi=10.1016/j.ymeth.2009.03.016" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=L. Yue;citation_author=A. C. Spradling;citation_title=hu-li tai shao, a gene required for ring canal formation during Drosophila oogenesis, encodes a homolog of adducin.;citation_pages=2443-2454;citation_volume=6;citation_year=1992;citation_issue=12b;citation_pmid=1340461;citation_doi=10.1101/gad.6.12b.2443" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="twitter:description" content="Faithful annotation of tissue-specific transcript isoforms is important not only to understand how genes are organized and regulated but also to identify potential novel, unannotated exons of genes, which may be additional targets of mutation in disease states or while performing mutagenic screens. We have developed a microarray enrichment methodology followed by long-read, next-generation sequencing for identification of unannotated transcript isoforms expressed in two Drosophila tissues, the ovary and the testis. Even with limited sequencing, these studies have identified a large number of novel transcription units, including 5′ exons and extensions, 3′ exons and extensions, internal exons and exon extensions, gene fusions, and both germline-specific splicing events and promoters. Additionally, comparing our capture dataset with tiling array and traditional RNA-seq analysis, we demonstrate that our enrichment strategy is able to capture low-abundance transcripts that cannot readily be identified by the other strategies. Finally, we show that our methodology can help identify transcriptional signatures of minority cell types within the ovary that would otherwise be difficult to reveal without the CoNECT enrichment strategy. These studies introduce an efficient methodology for cataloging tissue-specific transcriptomes in which specific classes of genes or transcripts can be targeted for capture and sequence, thus reducing the significant sequencing depth normally required for accurate annotation." />
<meta name="og-title" property="og:title" content="Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/8/873" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Faithful annotation of tissue-specific transcript isoforms is important not only to understand how genes are organized and regulated but also to identify potential novel, unannotated exons of genes, which may be additional targets of mutation in disease states or while performing mutagenic screens. We have developed a microarray enrichment methodology followed by long-read, next-generation sequencing for identification of unannotated transcript isoforms expressed in two Drosophila tissues, the ovary and the testis. Even with limited sequencing, these studies have identified a large number of novel transcription units, including 5′ exons and extensions, 3′ exons and extensions, internal exons and exon extensions, gene fusions, and both germline-specific splicing events and promoters. Additionally, comparing our capture dataset with tiling array and traditional RNA-seq analysis, we demonstrate that our enrichment strategy is able to capture low-abundance transcripts that cannot readily be identified by the other strategies. Finally, we show that our methodology can help identify transcriptional signatures of minority cell types within the ovary that would otherwise be difficult to reveal without the CoNECT enrichment strategy. These studies introduce an efficient methodology for cataloging tissue-specific transcriptomes in which specific classes of genes or transcripts can be targeted for capture and sequence, thus reducing the significant sequencing depth normally required for accurate annotation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
    <title>Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yEV64ts1TIWcpVSDnx3ktURnWusPWG3w3bzSFcFlxuY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459311","apath":"\/ggg\/2\/8\/873.atom","pisa":"ggg;2\/8\/873","ac":{"\/ggg\/2\/8\/873.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/8\/873.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/8\/873"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/8\/873"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/8\/873":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MeCixlYK0D8ieksIz1JW369oNSTqGL91bGBh6qL2epbt3rQvLvZVnvTbwY5gfreZbSRDoXAYCVPdB51vStnPOFYDGKHQRkDVvp9MUFkLwPe1RithzrBdNfrH1bQHj1xy1h22h5IoUOc1lD+j222d3Ts54XgZkMWAhQmXhPG3Vhr41JYixIYr3dbmWOaF1aKCqzUhVMgSDhG5WgdAOpnPC9od2EObNn7L8IOgKETkSnTcNQnu5gYwZWg\\\\\\\/VROBUphOVt6Iy4rCsuXSROYDGHVBPn10NRaTK18h0P94ctOeKMj5zgHQMVXuovyFHFYbt1lYEXtEAwt2SZx2w4o7rpt4USyD13Cote+GGaXZsl2eG7IwTnuOc1TuIQJhwvX2J0dTxtJ5PxSi+gjGA\\\\\\\/HmKb23Kl1oKYbaX3p4iQvoSeWAOH3ujrnM+hjCunf9i\\\\\\\/Lrpgem\\\\\\\/JLdlC1gj65SqHrOuU6VqdKY8zrdPuKbCLnn28KEI7VE3AOUA4l8\\\\\\\/SIVfxl\\\\\\\/UH6Kdc4yCi4FwGGkMaDGhBEWDGqT3DFpGN3TdUr3J5HYsa2QZkn10NYdUEF8l0KA6MvjBvLkxhrzIiY0IOGEobLsHb+xDIwrs2wd1PfGJbwTELE5T58eKQ9pgHQt9ULFCuGdn30aBFZeoQCAPl8PhEInlh9l5PztCAnUr7DYGxAtQVteUoh8zChQeVvtJJQX9d4tTHbABa2FBMc2ZO6wffttCJaWEOAtvjKHDS5o1NYmxrHl0eR7By46q0uoKcDOX9vzsZuMQvT36eWWebWlSMp+u4SIaW+xa\\\\\\\/iY85sdF0rA0lPLptpohMLXt7d9cL8a4FXE8QWCnagtl95ae7G+nMSyZqOIunPRmQjNsmYkz7EmfgOKPMQK2IqdL+OxEqgy8YkDQl2bGR9m88ySDIIJsW9ylNNfUuViRJfBh5ae2cM2hpKFcHl9Q38zGa6klz\\\\\\\/oco2HIfsyx86xcBHx6XouI4aMKsxgY9vlCH45epRb7oD3QIk65gk\\\\\\\/BApUQVox331NblNQzFW+Wg1XiZ8e5oHPRsEazX3oGIBF6B73ZpNb\\\\\\\/1gkMcX3c+DoZf+uhAL+ddjsgMV3iq+rgFoVUhDegZ5RpwjZ6MM2h\\\\\\\/XzRA\\\\\\\/1hjRq\\\\\\\/GxlgivpgXFmb2d\\\\\\\/nHSsKz5bKQzJ4rVh1u0ng0bPAqJ6BZf1+toVm0I30st5\\\\\\\/2S15enUougFM\\\\\\\/iPPEI6hNKf29yKcH3dccL74XA+na8yZL28oAoZVJ6UTMFxp23JEYcM\\\\\\\/20wjoPQDVOr2S3efZt82JhRPpDr\\\u0022,\\\u0022iv\\\u0022:\\\u0022jv2cxIkqI3mr49CDepvvow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229b5f1416dcf06e35acce9cbcf46b09f12554dea9f0160cc1e77c40147a4e21b8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zOGBRq1stlrmvp+AC5At9dDH056ccD6I5hwwjmb5JanPpG4uJEfzrrj5xBRTjSkrl61hPrUHLm0H0z4q9Moy0NyPI9RiJisU2Z+Oa79nj1lIK9ZydUsFhAUKvw5t45hogMx3ixQD3iOhAPX80\\\\\\\/n\\\\\\\/eV2Jjhuse2BmxBCzXPJrSwClGMLH7kjCpqRmZ0hnDSIZ+YCxkC8\\\\\\\/Gj9cWJmJ0H6yTgDcNzxAN2OZF\\\\\\\/xpXfl\\\\\\\/8weDligGliNoR2FcaNRBmcKNRCAHeXZKiHqj6MJe+JK0DripkL7Ij5Fq9hjRONGI9eXMqgo+NlfRJd7lcJ3\\\\\\\/KQUyvt0\\\\\\\/HVDUOGtpXL\\\\\\\/lKl4bPcD07Bd+pov6y8y6aEQ6ghBgmKY705hiVyOseUF16pf6kntmyV7IWy3gpBG5N13GsbGdlW3vGWX7pu84aovRUCSke2cEDNOo0puMw5SHoOuBFSD3JbhM4yDtQSCQEjbg2nDbYY613BpRDPc60FO\\\\\\\/BLQdw+GWhKjP+lADNmSOQkVDOzIXsmb2c1UAQmMfYrFkDEMniGqnCl8W\\\\\\\/WNkTHY2KSpwFLjRil87HH6aZb9tQpcd4AXpRID5LKDHoBzKXLtzrqe0xIu3w0IVRg+SgdduA17KNfTS4HK4mCYouoEU0Lam5keFeNTiA\\\\\\\/qquNRzl\\\\\\\/gqIxXNscFxgin0wSWjWnYw8iuoFe8NtFDm5l2ESOarbDeaRHUcVPE2bpmyp1pp2naCGxR4HtKAcO8NiZdkYAr9V6zAC7D1ZoziIBKpIY8xGM2R6bAN7\\\\\\\/1FL8V+ZMGPXr\\\\\\\/v8cn2QyKLFc1ugKgpbDrU8UMrlkKjwFAem6uYRPuT0DddvDT09nG\\\\\\\/N1RN+ArljnTdYj0Gr382zDoIufkoLbbc9KbYMg2tHJhHxgQGxdmoRyuVIc\\\\\\\/ccKzbszupm4QqSPDgwAWk37H1VT2rbF9VNZjQzuS7LLc8V4adBBr7MLX+8cSuGRYjHH68p\\\\\\\/34U95+9y6QXIvRnlfTEvzvMz20I+RpD\\\\\\\/H0pvOMJrij8Xk+CC4tAbPX4BjQx640tExsFdoGptiZkX9WyAir8SfCgmJ0UghkG1\\\\\\\/bbE7VwHDgixNrU7QAMQdakC0A25saFWyGxrPWBfU1nW7PxmyjC9HEQ0tIqE1N\\\\\\\/a3HPSgIl84vUazTqBiesSMuQ9095NFdf2Gd2G4bn\\\\\\\/6YWFYY+qkWyyPJQGglQkoxKv\\\\\\\/FgpUjnCueOtLhFlP2HapTpRoC+eqYhzAlYavMPtOoCO9EoRBKWe+9JhIXFedgCleJb7uzcR2qcpzelKTWqIrKMGq7plk2sj\\\\\\\/yQjEjCB+NVIb3hQF+hVZhav2ljXyk6vGiK3XRIZU9dtPX3Btegb+FiYqX20CUg0WTZDOw9li8tsDNr5LC0jkMyMq8bQ6mpic99yUqH+CkKIfeb9uXgh1OpXM\\\\\\\/BJ+N0OXSY\\\\\\\/92hZQAf8Wme0keT1jpDzxXnazik+CgGmesCdcCcZyB+XnSV4qYhV2omVjJkOboWA0QXez4hwhvOmsLOTD4ea3FpEK\\\\\\\/LZ5X7xGqToBoU3Y+I13qQONeHsIT+6jn9VVptNqvG9pYFQ7ye72FwgFmFkxCa6H6c\\\\\\\/NpioZ2dRs9XH1O\\\\\\\/XBKRkNmGHdmxJCzqmxwkCraKiZZSeIGZJAKeZ9CGe9kT7gh+vlIhKLptdRX07ahLFvH2iXVx1oI1MCKMvRr5zDHsF3CpOMB80pcQNB25h1E9z+wIEM0GmSn5JKl783KzkC3wH86qAQoNyW4uulJKqIsePSuTtb6vy1CW4\\\\\\\/FoehohHe13skKe1DCrVObjazkRmltVPLqs+0ra8fKhJ1OjTF6DS2+7+7iaJOHWDz4BMlFBz0htVcvQwe2bEI2j8zROnXZduT5xNihUk8LnuDGZmdcQdhq2bJqdFeyCa8nHJrCT\\\\\\\/Y5SsCH9wbczyq+mOaW2jy4f+TcFzUidZ1REsMu9bP5d0X0l1sZwHCUnzmF5py1bonxaB0+b4VascplFlpNnHDy\\\\\\\/YhOnPHs4yEPjj5DFjD9VRN3psj4cJNZW1Z3IkucTQ5vdXTy+FzasjLK4iMp8RrEkS1jF2v2DWKH922GNtYuWS1WE+Ki2qO5iQVP1ogkjJskgMXfckgzf7Ef\\\\\\\/ogSGuIfd4AlED6ajBy4DBLuuONpkff472\\\\\\\/UVh0h8GdPG3BkUf7DCI+tFkaZxyEMTKfrZkUjbjg45zQK1g6\\\\\\\/lYnPH6c8YqRwGU+maFeZBCAqpKH85N0Fz4vdHa43qrmiEfsCg+ghL6fjtUq+wksCeD2BQuLTZYNhuu9miueGWcps7k4RDM9G1o29r4umP2fa9LjPwujDDo4V0G0hTm1AWh4hTI3Xcwy6GH3SF9dXa2iVSDj0iHC3W5JVmH1sNK8G35M3xcA07M0uJTgw8dYQZzTAJQUrFaKuYcFngfP1uNez2hf1aYmyxojp+GBhAAHEB2t8oYXxr\\\\\\\/K5NO+OIdvnDEMaBZE9cj8RcApzMRZUSiQ7KmXr03CxNmx1sCwfTN7\\\\\\\/cBhqDHuOVnTjhSSS5VbCl8NzvZOBXJ0ndD8n1RFqRJgOS37HE0EtxgFaP6q3F8jbVuNwNraDHXhkRZ3X6msvt9nSEIcjdafCXgX6LnuLR\\\\\\\/oNLtTzNiGqnX2ng4wdul0ghj0tqO68qTNZOeP6ygEwfrtql+0FFZOLphQY\\\\\\\/RUAb8A1cd+7w7mL4i5eWmKOSxpbVEJqEb80CQX4f6SMcbNjPFXlseSo9V+nzeghoqLp82zSty0d+18SfSFE7OfRE9mh4v7azy93aA+9och\\\\\\\/11NpP5MzM5lxzPJK66GrH8JxV88u8z1\\\\\\\/0hZTsppbBXzD+OjkxPiAGHyfnVI0Tk7jTS8ti+93yD9FSf+jrCI3gTQxmZTdqKsfoifBk2ZZV8QtAyV85z+OMm\\\\\\\/MPHeUDarZ\\\\\\\/Ky0QMJbEwFwya67sy8J7g8EqzvguJrA6ej6Pxgvy0HyAAq0dBTbhlSjryJ7bdyDgPJEYYZTaQl+k7FuBoxSBMkAET4EoywlvLf1jrmqQUQkho2SCWgxmXJEWpRv2YRwi+ny21h\\\\\\\/J1+BCJber+jQwtf2eUfz5Cw1a8XTW3IXPOoimxpSsQVJIv0GvJnazEOZG+tbWmAl66mOPz3c7+0aBLUgFn928+lebtUATpsQTyo9dAc4lLL6KFwVXirMNPUmLtyVXQ1BqK4ff8EUWE6BGnf5iCtiuf4PUgbFUm8dPyaGCshuw6dfTs3Z50azzpdDHNpyNmysECUqfIWD6+1l9k8yxesqUoS\\\\\\\/Xt47a7yxSveUW0DC2yCHOq92IrJTEUBrdx2wkefGM6sDeA9gjr\\\\\\\/K1uc1rW2dNakbyv6u2WXYNUWfwnRc+L4yrcbFu+t9eqhopzu6UvZk8hFL08Xty43ndosQYiuRn0sk19B2ArohQQsKDhCefTy4kHVR0+4GG38fP5pPRE4pEO0mu9U2GO1uRtj1Ut7tpT8tEzIaRc3cHmGoefW+Kppslz8MAwGvFnU20RNjJBqhwXw5GlGgSynggESGYpCyyakicR6vLXP2TKYZTO\\\\\\\/IZFln5yHL7M0c\\\\\\\/42SWvxb60UuqHZ7R4fS\\\\\\\/I726jzpiExFMaBA2sG6qBJttF6AD7\\\\\\\/VgLlq4VZXZfPc1awRl2U0MkxeiBcbk5hbBBJVgj4jjJLmHQ6mY2WvueDOuKCt3FgdSBY7xm7tg3fYltlx7HC6M5SWoj9+YW2dRysj0JAiWeSCpgB8r79Obe3++7VVQoB8ft2hevEr+UoIL+KTmNLQsI5dPUPfn820cXiidIg+hH862olVgv0N+qhUAPy9U2tkQxlVNqYttIyH0m0u7dj9geV3DTWrVQOg2Lfzecrcu6\\\\\\\/BSDz5cf0UxuIoLRkGcVvVSdzyVayMZ5eE+p9eouayCr+LFz5kCH5+aIWfkqAn2oR+bUiGnRSDiWsQtVAiyFjXMgwQnbzzvhO\\\\\\\/twXzo3+sd+jiLx7y\\\\\\\/qS6yDkOf847P7ZwTYJT9D9qVCJ16MiAWdwu8ObwmI7YIZwnNSrVtHQGTWKe8oaP57dFwsp8zbispLpUSE7Yw\\\\\\\/s0oGGts0O35lP6TsDgzkORgZDFccot5bwWzTPfTTcUOAHHuAfMpBMxko0TXjOaW2du9Io+xf5n98P4YQyspDi5qK4GNlJAndNl8V6l5yCCFFvzA3s5aSKedQafDV8M30BgGwWyY1OWOAMuJ1DbFdLZf+mjt713r52wmlcIefIdvDWB9gB6wVaZvA2ihgelQ2k4OaBoLleg54RKHgzswBFihjkfX0DOl6Qo5xB\\\\\\\/YNkKzzwfis76WJqxHm1L2PgYSsYlA6cKtS28DF5mwMNHTlDVzUUt+RGsrDsKdhfDqSjTh5EnhI3jx5o0syvylA+olGytPXvKOTnvwxZA8\\\\\\\/NNU1ug+zAbM+7UeA+hlgPFmu\\\\\\\/XElsFEnNkfijiv\\\\\\\/9voALWOPq\\\\\\\/JHhYRDaBD5hp4dMAc9M3nhQsuBVi7Novx1L0ZeG5Tb6XEsVJMkeKgbTBK4gIk4y1SvVhv\\\\\\\/NOONsBOtPb8Q5T6NyoGcU8O7Q+JpJNqd86n8\\\\\\\/zXZjoPSjqWh7IzYVoUgs9YvnXsV04vANCxphzsb86D1k4nNxwpaW1ZckmDv7b90pJHtqKtmTLlkfIqCVQcbaE\\\\\\\/5HtEoxJmDAlSbDyxUVkYoOfBljkQz2VZEE+AdCqjPh5QfIzWV2XM2G8v7qDVcyCqGFtVsjcP06I4eGPSxRSTH1rrBzQp\\\\\\\/OZWslqUorvJ8GaooUYfWAfGpYuQHZYlSJZ9x8L1R86bY0pd0O9lzBzx4eAO\\\\\\\/5kd0eTlwUfVW7AMYjv46XjLWF1cxmCuyg\\\\\\\/RlqC3ag5fBiBB6VFmjfoxoHEamjEeNaCXUx7pjTL1QKeYawFx7BBAn7g7wPJZjJNyVwvfltDjT0BHntqFY\\\\\\\/VUKXVII\\\\\\\/nVtx9lX4Nex4IfcDz0djBEwDdu+F6fjhl0r7xqduCRoeNVQx69NQK6VwOfwPP3irrVT7vWatFflSqd9b4JVWtAkUGUIowuN+nQNKDLcxC82DIiWxSSA2dGLFaV0JSFo4NFsnHfHXKkcAmd\\\\\\\/qkIRgmEBtRWWvfuf4nzIEje+E3iC9C8CQA6b\\\\\\\/KFSJrP34c56662RpgWZ24K1kdEAYakrTVOQckp2oBCVhHhNb28BlersP6EnJqbuRS10dsqSdZiVDJlR2F2spo2L1HmZ5hDr\\\\\\\/ShEnBZEOGqcbFEQ9cQaWMfSylh\\\\\\\/28Jpmd8ZdzK4J7LtY4vfAypi5SR7J2Df6it0mGQdCrTQqv6P3YGKbVfOhxE1ERCCV7dIUmbzSOq9b0Y7HhkncjXfmF2BxgNAelkeI58M7nk69DK\\\\\\\/36j6WEM2oLEi6AgNiQr5u++ikAK85rSzM3GMqrRdqxJWXRLEdBTP5P6OSn3\\\\\\\/dH+1euE+6QsiumoS+NO9wbZpzgraJ8ZQxX97yB0NJ9fPQXd5E\\\\\\\/iYqxb7QjC4LMajqpJjSAjJ6jtsl30z312q0sqb8OMYQxgMuXa2aJ+sKRuDIh\\\\\\\/OGbuX92QIVRWBJ4PR68TuEkE\\\\\\\/RTDYJxVmjQkuzJYTyQkDTUCUbEW1GZRAVOZSDMbx7bgRmu\\\\\\\/R8vnJ5fslj6Up+Z2pRkIL56ZxRL44qWVcT3UlDqNzi\\\\\\\/Lxp8zpQV5arcQTcRK4ZvUESNOuZgfd\\\\\\\/uu2wquMh4V60uMjSx377xU8wYzWgBOlwLf3tjyC2EDgMIzvIeaxPFSRlpmq591FUajMS2\\\\\\\/tGqraDdqDehFbZX8keLX8HPMz69tQnWuE\\\\\\\/dr9JZY7siCDoDi6NdMBW9PLDX2wdXsMTJnDive9Q3FBA2Aqe5A1DQYuN+kubB61jIMlXnEZR6lzXFEM5zUTBRAivq\\\\\\\/u4c8oRFtIVfKxobo260uqwYPc7oBab+oWNj5tBDvXZ8u6o2meWZafZ+wTC\\\\\\\/Y7sEyQgaX\\\\\\\/WsmZbAQ6\\\\\\\/279KIf74vN9VOSH0zpnkGS6ICmauRE0yWUEtXV59oIYo4Gy9VPhFxx96XbHWgoc9WOMgxTJig6eNB5Bqr5T7NGKNW5rwCsgXRrJhjhScPnAuOwEjbtiznahe05E\\\\\\\/QwjGqR+mOXLtyyTsdHVe0z+42nQiRqflzxkDMHPY6pj3xAURGGRbvela3\\\\\\\/nMivwuwOuji19S6ZogkLZnYnfxKcnrOKR40hcpM7grJhhKostyBCptzf2bZnAhOp1wcwiCCkQW3suyzFOBba4dGPp7BKQpTHJ+OCZ1XcLym6\\\\\\\/L7dNsa5Q6ive4zM4fbLp5WwtjSSbMW\\\\\\\/qt8LBsFmil27BY6TthiNQ0h5+HQIZTgYE5kfYqb3ANBUActWT\\\\\\\/BEi3NmhWDYNQduHJ\\\\\\\/QkC01+L0MrQofVTja1GnuxGhdkQBuEhH72hCuEQP7vjaLwjS6BUqZdb69n7WjYiWhWLkj56xZDsAwKtLvIU7Z8+3zyv3yDTP4vdWB1whdGKaN5V\\\\\\\/hAWgBij3xFDR1ov4VNeM9JfUAH0ydcF3O0tRpfhMBFnmSm1dn\\\\\\\/anoce0XRy9rAkgNuq2CVaCnbjWWWN2o+3hQ6XvaEkyr43DM2p5ZIRGoGEpbKXushV9Fi9Ql1FDsEvEMeMp69\\\\\\\/8MQHKzn9rSHFC6bIURB66jP+QKepShQPg71B+z+zNdyiyZkYIq0VOTAZksTPbth4LHFC2LTyiDcTUtqDT5zcn2UgUqJKAYINZjnMvOBch\\\\\\\/3WaxKxuipieLFzjQx8ykWLrRbWvExCn7bvVoHSQVtlzqPQJGgPuDZrp9L02nRHAlbf8xuBNtUqYcG+yQlONqEMHhZ79i4EcgxymJz\\\\\\\/\\\\\\\/4Pn5x4Exdpc1WqI4wWsfBOT6FvHE5hLwS\\\\\\\/OPzUdwYGcP\\\\\\\/nXITAsXvyz2MR3ZrvFeYFkYRCNJL7nmVng4Lp5gkjC1KowC1oN0X9nARvzlPyJNUPknZHQx9X5ERCi9GJfwNR0bQv2tAORuhn1gFLQRT2zctfMH37IXQqpraAIZUwzyUB48NdHDl3aMwqDvzQJJXQW5OFzKUji47dZQLItXI+tkJo6KY7F1+CsjJaJzr31r1NiosYRDpCNgC6iCqW8OFNdKqr\\\\\\\/OgPzapChh6XL5b2u\\\\\\\/vyK6E9pXD9U84hKMpvw26bBbhoG25G\\\\\\\/IZesbZyYWBve3m1lnysG6ELC8p7aHL\\\\\\\/y0dCr4\\\\\\\/YcMvDSxAwlttPGDlIVTTUr4wSXEay6OYSxK59nPyC8ojmPFoKjH9DDEqN7WzkZ4BjcuFRFRzDM2x3yjC3vJVeseGiGDEQE2V1e0TpivHafhd+L+lM9n8rtfFuIqGY\\\\\\\/j+FQ8OdVX9ogEAasj8Wr67KZz3Tm+Vs4h4MkdAnFhUj8lbtgdvqPF27ze80+kgaWmaLSNIa\\\\\\\/KrNdilhDWsLtdpKuKNOQl+LEeytNd32sC9C9WJuMml6odcc7iZT4kLw022gPD5l7LM3GsUCuifUEVZTD8n0eN0MR4vyoAn9VuGJk40UttU42edYLQgpV6uEuTdvuDEiL8AkmDeHt+yCDC\\\\\\\/nB8GLQRhDEQCJqOlOPwOsAeHhU12HN3qNrafmTip\\\\\\\/QeV1udD7S0LoFUDOe8JiUu+23qtQQZccQCjhBBqpxhCzKYu6hnnj\\\\\\\/ECe76B+pYnHk8ZKWsci3SmD10lS09OFquwN9lycIuv+qJvtvT4bF3FDSTWLPxuNKXCEDdw\\\\\\\/webzut7BSzAyGwi5Z3fyFwIiN1LxMZFbs1dP3LmLUUqSi1Y26BhoUwRyCTuntKGJj1BrQ+xtG+kvg20u91klY9HfBdEfdd3c\\\\\\\/kR8\\\\\\\/Z4bIUNbfwTMaOxNc8OYJSAy5pm4HJyI1i5YOHGzzZGs2jvNZ8wZxU2p\\\\\\\/uVrhEPDV66JeldAkEQD4jkxQoCEoXasKlFBIXjm7+Uli079IUsBlAUvEpuHQmn43HFNpu0IqE5HtZM7hVrVmlDbRCCpKSqtscknM57TySXVisryPEjPmtxwvbD+rQgKh84ZKJQPCShsoODWiOiBHmQTzMlKbI4hcQxGFmDbC9i4uqgbgE8FgCQn8IoSKhKFXDDxcQFyaJBfawaVu1alJySFLOiUSU8QQwP6ewEJ1ssRh65uGiNg0Xfrw\\\\\\\/so7+QxORgiGhdqSxmMnm6Ja5pLJhJ\\\\\\\/z0KsCGG8R6+LvebJDfk+WTKZXKLNXWuChXHe\\\\\\\/bvfLoeTdfeKaCSJSTlQXiBhpuErMYFEtX+vm52LlD4yFkpipKFVGOTtXe0R9UZ+\\\\\\\/SvrQX5NAZeBGLZm14tM3WZuccdyENvJfvrL6hdruZdecsWONveO2ho5YkFat5t9rTOJ4j5w0UE5iGt\\\\\\\/3OP5+G7GWvDCtoMisRiPZFat0At5efzEFQ3g98NxxIRS5Ql+eVOM+regB0P2bw65KQ969tfBfj4IoYYIBUgN8vXchqT2ClY1\\\\\\\/Y1QSJbpvYwe3Wl9+BiH2iDBVuDTPJYlV2ORKreNUK9DsNwzadpm4RGlvRbRAfzaTxVmnnIn\\\\\\\/gcLrjKSgNyYoBrhCXyr1SE9kns1bA7tE\\\\\\\/U66bpl\\\\\\\/R2jGVDJyanFzwVWs8LEHaHTkmiDZTmS61j\\\\\\\/k4ovcovmPFhg8RsLM04xn\\\\\\\/ARXeP4q9+5a8W5\\\\\\\/Vsdwlc1zR45JjjsXLP2Tt2tc\\\\\\\/KYOMyb5JUe8\\\\\\\/iC+iblQ+gDA1boa6RAcYVFAb7aeLbvwIj34GoNXjd5g9W8urdVGVqrJZXdL5z32OTo2tCJp8jiUWSlgMi75\\\\\\\/xs\\\\\\\/6DgfQbJ8oltiPZuiayyBJlASLoN1tHk5geDH\\\\\\\/rnqZEcDADUbMxWXjYT\\\\\\\/iDL4ZkpJxJQPANF21VWaQ9ajZzs439gBKvJuFVGRgtZRT7svZ6bEBa1zb3UFQU4ZhEqdLrRKCV6kaluw8FMnK4sI5vdUeYQajf6btToybzwLu3pxYCEP3QOgjU\\\\\\\/NLNWvAt14h6HR\\\\\\\/SziLUZVekwTH5nqLvlM7tTqIQDDpXtrepQx1PqEe16+scmyvdhushp2xpPGMo1MgWHjHz7cUSjiSkNtGXeLYa12I1lyqb95L+H9mD\\\\\\\/SvSL0LyDy8hb+2Ht82cPdkznR2UmD9XyfHhejPHEQm0jMCshgRAcgqFk\\\\\\\/qHEnTrWeNqA5zIoGXKgTwBDnSIGJqzsBdSa0rPdLl\\\\\\\/4fvMLrcOvw9U61QSqb2E+iy7f9vneftWNZhglNy4nw7Ch1FSR2BU4QTm+eXZvpmJo4EWKYqdBJd5ka1tapa\\\\\\\/r7Yt1esWFfLuDS4fAb+T68WxE22ZYYXSidzLxpP8O2oWLOBq9b9lBpX4ekPfeTlIGZ8oUUPw9oMwV3B7UWQ\\\\\\\/oNUUJJEdbSAEmpFm1Z6miGWnIIi6QbD3n5JbLslu1kL50Zw2tLtVeXLehJeUZMsHDiGNozRlLWBGMSqXHeuQ2wwWwbipxuScK74JzXefWiVfjXw2QMCBwsD06L3QHqlh4E4MZZ512vtNgXtl8bhDOAZaQCPgwfQtFQ5mRHWOxPLDyvgiw687usKchjO8JWcNlh8LEnCHkmff\\\\\\\/ff6X1Do2zvWZXuQzdVsEegwPCHIeVVsHkXgee4vxZyQmaUTuoxX1XtkvauM4bJXxdaqfqLlBh8WdtB13VNaQTzNcGH26DCsXFtoLMBI8oARaMy73zBxIzPRX6L7UrxtV3TyEnuP2qYltrNqJmgbKUQGejH\\\\\\\/qPNIMGwHrW\\\\\\\/rQY8ado8bzUYXk1omdvjNT+ypacFQgr0RLFlzOYHyoUshQPaT\\\\\\\/keKyEA\\\\\\\/spsk55y92ht6D0KUpPdUgKcQRpA8HYPtXhck7UCm7Pa2+NJ9B7Q59ZuM6817YCtcBgrGB4eTD4EAaPOfvGKZx7wAwi+5+fcmNAHlPP0zMxchI8B1iXbjrl4XzXyY8SBxColWwwE79U7N\\\\\\\/vvD1vYfbmVrN\\\\\\\/21YXC\\\\\\\/\\\\\\\/5m3hrLf3kJV+of71IGGkpbQ7exQhs+bNAnJSL0Ap0j49vByBaVZuh8QXbGqVS+GBGmyZKjDpN4\\\\\\\/9m1VxdruDh4rJg8OcqFSAHIg+jf55DRfQK6KOpt8+9qTKTzeTed3rJL8IcmCCmZlkj7YImR5febHILoHfSKyJdo\\\\\\\/K8FGQTbGTWWnyf9tbPfVXgxY+ljuyt38muZ4+lTlzgX067tWhcv3VDOAQVE47615xvFUHan8g56j6DFvnj8ZozHSSNnBowFBJRk8CxXVDmvA0kxK\\\\\\\/4GcGOvD+49SI5DiQhwhFdyZ867rcrbbO7kGLdBnAgTshaUnlfj3oNeYqtzI13GsFufjCgSQ5boVq5zKTQ7bWg4idfHAEj59XLa7U6OPWle4NEqB7DApk69CPFplhioH2W6ws9HyP5vQp9Cnidm1PHetQixR\\\\\\\/Zn5x8mQw\\\\\\\/HQOyKg7hbVU3VVbSaLV7\\\\\\\/41HZlTmMjeQ5uAGlPgK0fo0PJCoSC+GbDu76OTMZggrOiwBgvyRczECTV2zDZa4YFl3b98VOeIBSiUfZbNmfYokTo7w56e6OoMjTFrjoNHZkqzol\\\\\\\/kCwpGWVn9pZLqx3kIMiFGIeRPmmzzIQ65R2VjXoqqvvi3FcLB4HbWFtJ1yM94fnuIKg\\\\\\\/0IAIY8qMd9viHshYjVuRY7qCXcVfQwRHpEXSX2BvWylnT7+94rYEcdM+oasprEjwpH0DTj7a6EBLpyiimwl\\\\\\\/lJmmp9VoljNbulNUvp4CyynGcxKAJixDGo0yeN5r+rt6\\\\\\\/FLnoj5JfxGpL0TKkJdDlxInXslpBL5I+IHYDBWooheTHylevK5VkRFbcVLuIewnLfMRXuXzqQa0lKov\\\\\\\/Wt0LVncBdaihFR1UZ05NEgTr1SHS+flyo1T7IOupbj\\\\\\\/OXt64icPbJXZRfB\\\\\\\/fk5C8Z4VWuffUkfbUx9qsBVekh11bYMDu9WeBHc01TbLMv494PLYhQWKg2OlF3p2BBsKB40rb3zOhTyq8IK0TWaI9G8DhiMOzX08SQGaXZ0CLXNiWVeuguoLBGW7ltd+J7jc7DQ\\\\\\\/1yYeKdAOpnniZu+QxvvKZNasNsU8bHMAlfozfGkHH4qkperW8fyG3sz15Ex1uoORVJfYA5u3fhErHv07FGZXsCH7IL0J3iVV\\\\\\\/omHimOliR7FmWEZj2+dcuRVJ68IQxW6hTc14m9urzrvoIP\\\\\\\/fdZp5wHFFn8Ib8b5dU+EDXzFPqEc73pbHXP0mOeJWSqUe9336WQntvBNB3yvLx52Uzr9lhko5DcI+0wOtlEL0AMAKD9KoU12douatCDUBrls\\\\\\\/zc0J0+UMunPnclu0Ue363niH1wnyKNXLljrYxGT7serOgGGayzOnFeE9i\\\\\\\/mH64zZstw7e9GleXwrGgC8sPM77KUUBPeN6oEycM3uYbmdp8a+BOgqAhCPWGzKlIM4L06iCcH43FaKl8hHngSahWLNAesN0fcN+udwWmAHyANaZHW5sJq4i\\\\\\\/dX8+NNrx3i75\\\\\\\/c60oJRSfZuRe3Yjs6mLBFIjnA2PMV1Bb5GYAnFZDsQEtpnEegwY1+YHV7FxlBIquCn3AXITDmeR4RLxnfMSzcsfVm9C0dPgjw9vBshI3HMnAdCct4oCM9r3NHAh5V0n2626nZGrnTVrUNNRfII7jFDD3QAc7bJ2C6y8DG0ZpQs2eeVvM\\\\\\\/CLugLTh8VHL2rOveGfmbAOVBVFX3vK3TkDD3ivWGwp0zPXLnLig4TQMVVOw09GsfigbiJ0UEAB3WVWuzCC\\\\\\\/dKRQFZUyXj8JK3G301rnvgIPccNLkjux4OCpOUh1x+vKHJyYzhPHTax\\\\\\\/76idLFxa3XTyk5Lkq3Rm1RybPGIN5VC+TS99nA12wGiDboOxMvhJqgp5wNew1bP9vesxZscN7IJfOBCij2qWeZm7VDOP\\\\\\\/QJW7Gn+epuiMO6+d2KeTrYwbb36GQT4d2vQY1WGBsuBkg2+77X1expkbr8h5s7cG9z16g3KfARTjlqNq6\\\\\\\/XeJ6xPzbhJ11ldXmfzE9DpHPFt1Rw9weRmNMAKARrxGMv3HmmvGPwpoKuLq+t1dHKO72CKIs9weuGq+UPOHkXKAz\\\\\\\/dl381k12fQWmixVJC6BTKNIP9wr1r0Vlfx6C4STawDHYgbLfc7KuD4tYrMKOzjPi\\\\\\\/iv17thOqe\\\\\\\/Oq5G6Tmlk1CGpybvJMFOSjW7AjCgPWQAxYoY3o7a3CRvC+540yC9D8hZXix8K+LBbr4OiAHqiKJogSJYBot0KqDHO\\\\\\\/zPJsh+LevUSxgSpVZ99a4WuPjTwLCPJaC6L2UZlJe5uakpbaepYw9XXu3d57pfOKHvLxA3uWrGLm0pkVrX\\\\\\\/l96HqjeD8mnzwUA1CMkp\\\\\\\/3jb9Eyd90HxjlZoHbpp\\\\\\\/ujinny9Hdwn\\\\\\\/VEpax5OkZ\\\\\\\/d4ymJHGpDsaiMVG66HPxBCSbJUINTtQ9KA50Ci80owshH3szl2S6fDGU1pMf23wlPcozNyLpxZbeYbUAHfzfXk6TGhCHvcu0Vf5tFTeYG4qbjpTd7aEL6jkSMt\\\\\\\/kY1D8C5CDvjBqNgTAjKSAVlMMkmzQTAunR2qbLc8iXv0NcO1ClIbw4JcyNP\\\\\\\/+tuYibtm9RAsnEA67vwypHUDi9eMDA\\\\\\\/oRpQNt7HhY08LiCjtcO9MWl5LxTMzYg\\\\\\\/BDSMH9fvXyOI3bP6lbt5wiYbQZAfGxmhzppTBPxS6VYeDUbmjhRZnnanhP7Rs+9l2Xv7BjZMZ7cVJfG6R30YU6Jg5ctx0g7hQoILN7xC5DghabEdNtR6\\\\\\\/U+0KXaYOkwh4Jl6BvY6VrwLtMqSkcvygFqhapFGBkfAL4ygTL7dUWXA\\\\\\\/n5ntYO4D+vVL7sy4SdfLUTeWuMdjGhIug\\\\\\\/xr8TuKqObCcmnkzdUr9sJP8LHgVCL0HSPBzIt338JYd96BkMnfU6FdF4hUnj1mzHsXwrhOgyriWbHmHiJ8bAXLycebCfNL24uvsh8hRHyM+qmfvZ9VYClShUWcI8lkRPCEsdvYhXUwEJgoN2sqZoInabPTzzFsOC\\\\\\\/Wux97xOdfk4lqvKUPlccdJEoinmSW4Hj3VGgPmeG2IRDmggBiCnOYveKKx8ElhvMlpA9GI+2viJaauBfSq\\\\\\\/b81uracrfFltZJvpFZYt4ygzdbgQVZSZXuDpOeCD8JNW3E8dw6HMyyMlxSG8XluV\\\\\\\/JSf0zjzgIVRFeShyg7uqRY3nWkD7sfFsLXUK2KXZLEU9Kp+w9uANtdxQRIcZKej0x1h1s4GV6Doxc3WXuGy8ApXGRl+vMN+fOs30227XN0J3NkkLueaBrZkHNICNMJdwXCGi9+SwvZF2TDfmZitZAd8dA8q1eyDrDDMypYUZ\\\\\\\/BsE++8wW4HLxHsTO35enXLMlYhqQ38F2FgDNxZfT1zNZ3YU+ZMOE2EAmDx5Yhj6wgt3jHkWCa7J9Ue3jvl3ZUY7eIfjeU9R+z0uYAWg1iWa1F6j+psOjULaxk1q3mSa6RtEyfd8BXPfKf1lnla5ijForZ+JglobenZ2W0uGEdRl3eWAnnCl6O0XK32RbQYxRJuEV5GOw2BAGkS+eGBlxwzz2ePtxiI853RIW5ARbB6O1SQ2OmSqpdcFPu3u5aobSUxQEdEL7TLF5xouqqlrvFUe65f4ETUyf7eGoe5rmpxncO8EdLWri+jIoCxNpoVvBNJPfN9Q\\\\\\\/pp0iRN8ZvA7RB6jGpCq1ybpd5m0ZC3cqM3vuKwrYApD5CCNzH4QLkxv+U1CNBbEpthlMpwpyTwuNfQdPtJ4XQL6EGXO\\\\\\\/GV3logAlDeqrA8KUC\\\\\\\/EVcodqQ0fTu9CelgOJkHPQhuCBzpJsb7HF\\\\\\\/G6NHsxlpkqSavdcvK4nhBdqN3YySALRPOj0Zaud0PN90b4IDg10mGMth3vAP+DeXRxBubXhTku+ipafluyDJ98XHOfjHD9oARSX4jHNb7lxspT4KoVaoqJ+nxcn+DEaxQPpoyuxbSrmisOcuMgeQfz2g856d9CZN0ocrLghqZxMj9zH7CsCPLuCFpd7e7MH3ZIibOv5eY+dTWphDgX8\\\\\\\/MqVAfKSKLTsoOAh6OAzgAH0rayUZMqvtKFSxF\\\\\\\/qbqdAQYIkZLguiCRAGCJ6Y8UGeO9X7w2O2+ISM2ro1EGF66XKeYfsxpQ1MlpC8VDzZr713Kn9uNpGzLfeHoxgk4Uhvwps8EBHAjoyonHfwnXmUz18ZJhk45BVlRaZPa9lBzG63aBELyfjIKTew5D\\\\\\\/oQygnpUnELvTwPORaUc8ymL7IKw7Mb4mFQqDzDDRmKqAAtv9bWMq5rWrM\\\\\\\/RjCRXo0LPtlUyZytQMjZlxWWqsNcskeTEPe+1mPabnPDYpzSAuKK\\\\\\\/YI4jyfZxdQKCziw6v\\\\\\\/lBzZg2WHNToA2slvbU6BpTjebp75ZgPmjDh7rYkB4qwx0NQUHenAsUE4OHtvLN2xCwWKxsPj701Lhjdy4cmX9tg6VzFGCNWJbqM3US9l2aQw3yf1122Fcumf3uOTi+r9Sp+RlG12hyHFUGkQl2er0Sx8Dwrx3mQEDAkU\\\\\\\/gKc5WVOsf+LG75DDvxNlfxkciRbvK4HbzzWo5aPB85NHdj\\\\\\\/4YVqyXXJIBORmVN\\\\\\\/McTrlnaeXj77vagYW6LNKBwQevP8p1mRDWoQrQEHmB+n3W0ScE3xVrrMOghw+xXou7QQgzRH3DQ2UzMF68gp8L71lkop+AKY0PYrxmyHVUUN8IfNxfItBIgJoVvVIm13IRLqeBZpiaNQwePr0OdjxZo4eXtqRgEyhMtJ0yw2XmKJojCKbkD4uCL2m9akwiUjJUAk8UpEYpKg9q+vDtL9g1R0drfPXguUQQa9jGtaXzfKvvOWwghM7lILKh0TtKOwZ4NdfejdPotLOUdv0P7HUB0u+CenGtvI93S7DXbdahsuDfzn3g3whPdM8zPuXQGRsluxL\\\\\\\/rPb3zG7TBWCmUDmpGv2GBDzKtq0bhxw\\\\\\\/ihgVEaknDwZAgjDq2ZiUMmLN+btzdpp9fZR059UyACZG9lmthNM0Lj3I3DhNZmUa+VQqfwbBw\\\\\\\/yuu9Tc4Vr+VxCUt743OdYGD1OcCOOa3OkiO0zSNifdQTCYIhsbCPo\\\\\\\/KqnxdajctkFLeIopcW+hte\\\\\\\/gmm8HwBp8jjI2vfsldbSu+tCo5wl+PYEZvMnttcW3XTQxP3DrA8sSAJv1I\\\\\\\/VB95AhfXeiqE32Ox1Bb3B5vLH3IgL0vjEBP6pqdHqFMuuVd6bMOolgZ9GKTDRm99u+wgVD1FtVoFs8mewsDhv4KefUzU6Dm7gtDgFUC3OdQUjiEJz3r2+h8\\\\\\\/i2u0102IrLNqQBpg1Z6bAyyH1BMjNL50tbmb5jk1Boyo\\\\\\\/6b6PXhjPG1Zd86h2hBlT7ynYDFz+9s6w9sD9rB+8kreiNYswgeCkWzZjMd0LdsudYpmYTJ\\\\\\\/K77tbkLP8r+gOn\\\\\\\/z0jcL1k3lVMFCD8DRWqOMbaDc587wG6a6dCzoW3fGCEVnYcC2AtZZexYQWH0wjyyFMF8oWngBEEdljcUomQ8CbcyVSPBZzDEfmy1lcexKOJdA+58aVOP\\\\\\\/1\\\\\\\/2yaNnekOBnaL2a+W7IFKWXTxHgInVihUM7ju4CBmTrhGTitNw5mn5FoQCCgYBFaC+d2xvMy4SQjok\\\\\\\/LBr9Wd7GEDCLTxNkvE0Yo52xIM5yaIuKTEo7jQl5gL9DhRJldlwYjrwlFTnyEGQ1kokPymduU7ILGdNEqSRjNQLI3tqGtZbppV0PQe8Oy6Z5BCCdgyfGhtZ0xxWE2DIRh9dKxvU\\\\\\\/bntEEFVse1aJzfK\\\\\\\/9A4+gXVmjxcqyhIvz2eCV8t17pIxYle9k+aamlnEnf2voDPuur7sG7N466LqekDBZhjS6A\\\\\\\/BrnDvQp37spj2d3LDhQomQjafSCEuJClGrty1l01LNtHcgeKW0VfggJbVMwONWF\\\\\\\/R26vhCiTKgWLdtaSLKMsAoxfzPvzJrQnQCVlNjTcGWvmo\\\\\\\/DbD8+hEyeoWUJp51e3cjQVIFKt1NApZXqX84ZtOcDXAr6EwsCfoxsY2BWnZeIiJw8\\\\\\\/1XfQDKHGchxDIE4lIpy54L3b0z7STBt8hsdrV0ZmaQF6itMHer\\\\\\\/w+dF+DhH+tvpnRYDYKInwkkO\\\\\\\/ZIi81KtkOTjyYi6BFYjyYW7dDFt9cbYC3Sm7g6M5OB\\\\\\\/dNmAINYk3YgTZH90s0m3plrZzinRM9Xkj6jhFQ+ysz\\\\\\\/itcHvM9G2C6yds9aMwbx2GTqGGNzDEeEmZsuLtHNgIfxcSjmtJK6TcXOGxMBuDXrjGxm922fID0v69z\\\\\\\/vLH4b0Xdj2TSS1D84rCoZnNYmwkMlldKZUnLmejvR8R9KHxixeuVGL+7YI4W5AEPTfMnDbLmThsx5uFRsdNqjBVnSBwfxknWKp1z46ZPFyj99ve31bvnL3gEjM42UAI+N6c91xx0c2GynXSHIVQ2H7aRWlhUlBDJvf5sCNc0uY6PF1+5X2mDzjPW7vC46hWdHbdDk7q7AbsCDqO1\\\\\\\/P17RwH3IGlzg4XRpCX3l39XDxOVsuW0nna492uhF\\\\\\\/e+2vjOTQb8226iJ0kCX2PwOXcXgdQzEqYANdbGQ\\\\\\\/zzZFZ8vRJ\\\\\\\/YpmU8nvQtbARqb+l\\\\\\\/9LeqvHghfvqdjGU8Ie2AMDTOesT2Q2vY2JULWISzLaKAjUkYITyZHWI5R6HeicgANvzuAlAe87e90lvkv3zTieH\\\\\\\/tPYfJ1ZejLqldv27B7\\\\\\\/4omR3ogH5zuMRHlM\\\\\\\/svVmV1cZ2qwhGe\\\\\\\/j01zrCib53kPFC\\\\\\\/31eh+tGy7Tu2AIDsz56FII5wNesY8jdXAiP2yt4bzeEkk5LpMgc3pa+Y6ZvO+gJVw9cYSaaJ8SQEq8hN0wCUCOdT7laTHaw39nJESESptmxq0xn09RmA9WLEo2O8ycF8gapZIEmbRAKR33CLT\\\\\\\/9AgpGVCvXEIjUiXjDV81+armx3GZkAAZKtIW\\\\\\\/uWdQTFFauJBL34hHU+RVvSwIECQ8wAuRdN7aXyjGMuboqCBAca+i2+cpxJX7hyPS+MHPODnNSMawvcwIk\\\\\\\/N40h+xKxZcXpAfK8IDmwRdsS2G71BWSZlDEpBvgIL6g9yzvTkDsQy\\\\\\\/vGfdb4suOVxmltwLwCwhJRgnR\\\\\\\/emDSSyPFzZ+yDnZRm7+U4eWaKE4ewGGU6bBx\\\\\\\/1JFqe4flagAvvkwQpvcIzqcVLHXSg7mF8kfwavFh2Df6U6c0t7r3C1l2lhxSuA9f\\\\\\\/2OxO81qFQfqhSDsj9alIM1it\\\\\\\/v3DI3r\\\\\\\/e26VK+MtvztO5JRtnyXk4BowWvKIfXaCLEabcuyiH0tsoQO65SlHClYBPHDu7oLIo4Vs4o4fNzW40gyiAvbEziAyr7yJ1RNZSI20zEOtBNrBkOsWXlnVR0iBpdS2xIzdRYUyhZMOCj6naERwOiKDdGwdOi\\\\\\\/jMX9F139BjIA12eFN7vVN+HgL8PmVCYOp4859ZFziezkpKAUBUV7YsIQdr3fImmyjhE\\\\\\\/Ez3H8PH+OuWHep8Xhdxq55lfL7FRBdA2RjeaA0L+gdpBr6gl+Jn6btmawhQqTP1OHFagobDrWAy2Ew7EbewRbFlHiIjQptxLEXF\\\\\\\/swWBogEFh1ASfn8OnBbAn0zuQ6yZJlCvSxj5cNrb5QlOEtxQCGly1y8Ao\\\\\\\/IXDEhaN9XNcYn8a9qH4Qi9R2Mvr\\\\\\\/JAt4T4xjLCSF0bH6HeuooaueEhmTBrjkblyAuVqWCzdtUtv\\\\\\\/VbKtjwO9ztrs6OhoqKWVVAgNmLaNymkLCn7xhRh7x5eRRkSNQm+uFhXpfyG6piW3NJV1vVg1jhxL54TaxYAkB+zBOWjokcFwaM1r40q\\\\\\\/HJK5dMjTp83LbR85KtvDWB+jEWTOpw\\\\\\\/8VhOpZcItrn0dRJx5wOqd5A0mEgvKWznSTauqd4qjjjwdvpezo0TCSZbZkzxnvVEAgU37lnuecDpr1lx\\\\\\\/3WYUQi\\\\\\\/KoQFLEgvUeHNYNThR4T0MT6chDHhXWVO1cadeD7zOQ1msltrIjXf5B5rinm5h9gEy8lQVJO5HIr\\\\\\\/xmIM+maL6ofIKu9314rZmjmk4KKxh4pkcgD76G2v+sEns\\\\\\\/6WPijpBa9TbymVkK7pbsTY0fMAVNO1tJPAXa0d7i8WkthAdWd+4lslS1HtXKgsc+gw1FmfoJyYHmFRTERhgLEES8bTqfCoiI\\\\\\\/0vTK1Xal\\\\\\\/gJPYHDuGUlc2iJPthLRQ4d08h4Hbm2vNWgkGsLAlh1ktYWx3BTWET8Z4rVkiFwZPmSQGfzZEVhwU3yFcttGlxbNszKXqby+HUstA2Z1FsgnUla0lAEqlEootBJc20Mvlvu3DtE+AOkP16FkO1jatRD8jYImKWV4QyhrmjTaVgIshb7H8Gxu1hDkF6sq5wCc5vdeCfr1Fil57wOWb9giNY+CbPHbmAao6I\\\\\\\/LLGnWh4PGXYrPP\\\\\\\/anU11M\\\\\\\/o6wAuOcWhPChzXcLwBekxepOnVxu+\\\\\\\/qSb\\\\\\\/o2QzQIFe0Z1zJEt1CIfMm7ow41qPA5kAT3z1S4Cde2yVowt7crpVJCSw9D9Sv0tnuGsSyQ2VlVHuRt0mnuBVxlE\\\\\\\/sSSrJsqFs3JQpAZZrjL4412jzoUMnz54jQbexWbwrYbOjAkkASH8VPTP7b3g6K8xN1GX4LP2nMmbFbPmCJqER2GeLhdHe4URoVmT2mXTwBVq4bHu2qIuezxHN7QgxAB5KtRTALMMZx+srLmdrVvHIqvsJEY2rrypQwP\\\\\\\/tZdpRXnyttvnwJusy4i8Ky1kP3v6Xzzl07LP2MCCwu12F5XJj6eRC0h6y4R2vqcxzp0Brx33EcEv\\\\\\\/A9uoX5HLMQh0uU0fQudXrhWyGBE7Z+5hAw6kGvcahE+FCE8u6ebkFmZexsuYnB1McN\\\\\\\/MB69QpxN7Tf+h6F7XQomJaP1d9d+wvkgA0c+zf0sR4VHl95UMzajvCYsBQRLAvXuDoDuZD80EAsSz074dxFZIoKo9G0PGdfLdJngaS0m7diZSI2\\\\\\\/yj7QryuyM5FY6iek0D+VnigtHL6d5jO8JHc7dKub3vV4ZjmJvchBmM2DpxxMltUa9YK8noseywowmCJazVVb5QDoF6MZM5aXqcXqJUtBjrlbHIetqjyYBYhVWcwmUnzntkEP0O4zPGwmyvbtimch6K9\\\\\\\/kP0XGwu7P37vSTXjCUoXZkLHOhs78dIyejuzUwmH7tY7FUV3N1IzOrk2aKzs77p\\\\\\\/2g1qYA2ln\\\\\\\/E0PWiiBYkvSm\\\\\\\/8kMAcO1wv1OfPc\\\\\\\/V3sQe74Oifi5bPtSw33bv0dwlyDuM0bw4aUkb39eehVrWwLj+JTdYNABEPFajJrxUNFdi4h7fSANFQpeCnWJ2irapuzHin9G6RrFbhygfQV2hOxsLTk8ugGX0Edjp4APefQFemi0LfgoDrs4iOzL8S2nE30u3Ab6utrA7qVoDffvTN6EIG5yBO7lrj8ty1XLygU0yjum1VGl7dYHmEvPrvF567\\\\\\\/eVExT0s8GPrWts1StVZUNz6ISTcPrEbJXIVogdwH20zTmbQl98gK9l322Hstp9lXgt1CJzLCSqZLQlV++ZQv2na+mJ2VSE4T+2\\\\\\\/L2ayouT5Vgr2PPWqSIp3\\\\\\\/rCFXGh1vbVe4IQx6MqugZASeqAnPnM1t+CKKWda3NBiSxz98J5EZM0AHb8PE8S5YbtX\\\\\\\/AXmyK+TGDCDY5nvk8cWLTVnsHqB5eAOsCnBedX\\\\\\\/TAOPtDoYoN7diRAxnbVzwiA\\\\\\\/QybADsjBMJ2ix6dKuZPIMFWny5dU8gGbOUh2OSbw1pykjFch1ukhwYdjBiE\\\\\\\/R6WYmRd7Z3pgO5eGDtU3j6tNyqX8NTKf5VgwGHDJIFHuGsYZSbqpPqILIeNF9fTqi3WaSfAkmySRdrN2JnYNknUEIGSPY9i0NIdfBvAmW3GcjrplQxbF073FT6N6fnGWRODO1uSMio5Ro+YaOE0yAtUrwoynHSwAPFgeRqx4JbVzL2cXw3qgXCiSkqVyJ2UE9B3Il2ZkLGEbebl8cz8atBAUdyzeqjO+q7jGJfkLvA4KlMldrR58tckdn7WVVWvKoRkGQkMYPa0WYMfEPSMRQ9Zrzx90wvnoWl4L8kW4IcdjuG+D9xBTHGzq7sZrl35p\\\\\\\/gUrCV6ZhEG+247jmdXRXmS5arPtKoJcuShadeHhIl4Ar+9zjff8Tr9AsJA6rlf\\\\\\\/lVxllXzcfj1+AFc4Np8uqHV1J6xffpWsu585yPk4HS1N\\\\\\\/xugprBjjRjpfNQqTZpeKZ1YFnZotCx1i7zCYx1OcHtaMrfWVSAM7H3um8eYSL4IPa09Ulidyh8RBNX9fERBm+TJEpdI3iJIHiCN7YwSkTv8ITJIkovswOQdIJK3jRsazmYf4Q6kQaDG2ieY8bIsP9U+GNSWDkzxsjZiq5EW2vgaIaBlNyC5FE1CSzdPOv2lEC2D56BK86loV9Vqu0uzD67RAi6urC05bIlaU3tN40JimqFOThySbEfx4Xy\\\\\\\/asv6U8zv+vTgasfSY3bKwioQAyFUXJuNoQxOjfKsJIoCi7VfLbkObikN26QML7D2tyiMwYfX0i2ajS0n+PLx1NBKtl24mDpJks7mQ5wA1lF7pBwnY+3tJX5YEWh7rDEUbdngACTicZt19DJoVjJLDi+sG109DHFnYSq7x2CqBSPU7kZAL6eMsIqywNN0Y1wRjgU3PTEkD4D66f0E5IAPQBf7BFlbLvccLQEw7UEyTsstvTvQSWcmffpYvffloaWxCo8lEuDhtQ\\\\\\\/weRL6bMM895lGgnwKxUxUAoI55OywaSxSrGLKQjI2EAtUMgy66PVxaMGkEZd+RwRexSH9sERFZCpUsbik8qlmnr9ShQ9qPJyXWkCyS9A3FD\\\\\\\/yyID\\\\\\\/VZ89Zq+oEI7Xf5Z9LzyDwigDNArqGUQOVZoZgg809VkAXrdDHIC3aAZzmTOhpuUF3GuCCT8m3ZeNdapiRMtDYFg1olMXSlzbZZ3tfXIeO9PkEC+ms3FWDeRNn5fWF3zBe2JZm1Gi79Z+oGxWA41F3PSp4WO4\\\\\\\/EOGZUHYrTipZXFPKkFlqlwYgdEG9jYjluc1j+dAU5lZQ0jvbZRJxQmfn9TI37yjKAnpdUJw+3geWuTd6\\\\\\\/rHg09SwquqGzQ6naRK5EvOgrrHREQz1GbyvSyL37YjHfd4cjU+GZ4BuOWIK6wztpEhPLrWQOzJl2it12pkNEc2Pb\\\\\\\/PNMjWao0QpLVr0TypVRTuxDyq1\\\\\\\/UcgcQNBLayFQay3usv40K4lV2FOyhm0dZR3o6WTca+hQ5Zm\\\\\\\/7nPRPTDPbsEHRFdXCbL3spzByNihNCbE5YMEuDMmoa0n7zDG7aXHjadgS76\\\\\\\/svKBxBYqjt720jeIZgBZ\\\\\\\/x7+SV4p493MsN4Jh+6yOnUfv6zDJZ8FhQTxGZxUW+BXb3h+cDHpYQOdgjt4F0yGQriZkcIhqhReyriTcikgokU4cuTADIOlDMpQg31N4BrlUlvkc0cugDBhOZFoiX5yffHh45Fluua\\\\\\\/dCEMzuiBVQZpJ56DMQk9yN5\\\\\\\/D269bPSkVa6vsboHbmd71QpCcPCcYfN0Tzoxh1EOFifrg3uRhKAuInY2kcL0BqS7wxRNM05uLa8Wv6asl0nsZiB2xcjV+hlUnM\\\\\\\/YAMlDClDi5immUwj2dh2enYAxAAXpIYFW1c+e6jr2hgaR+ZKcMr+Bl4fd7VRkRuLFtKr5eWnA3NXCBEVqjjS1EvImjeM3uxAjrGlCsO9+Mazdl3WJ8C7rr+Bsp2blMkURaDE0qRUVmAuIITKnbreDy2KIIQ0wj8H4hjWL0l53b7ek+jLp9WFpOmuFO6UvUdljAaK5VNu0HTLPsHQ01b+7l89x2QqbDDqQMNHc6sBXeBDpqszv0aSW4cdRrqHRSCj\\\\\\\/2D0REwRsnw7ir2\\\\\\\/CcCsv23uBKpLuN6f0WLi5yIWPLNNHzJx+FpP1OjOsPZUfioBKc5wnruALS9mb+rJ2QsAZaz8X7aonObvYp9\\\\\\\/2ogJM9MgNSCQeNDhlhCJfuhi3AyWhSEyeNtqqa2K406d3Rc+YG6U\\\\\\\/LxTOFneIKwqOAPPM2vjsHNiFVdMD6a0DuVnsx+R7TV8UnwghMrhD8V+CgeniSRWjaOwHDyWzqApEePQ4jKHAwfO8+kmNhfNhy11M7a6UO\\\\\\\/DcN+ta+0zS9xGFHIJgIkSZieIETN5aVKCxMXVHY6rhdDstuvrhV+XzD4LM\\\\\\\/2WU+tP4KbIXkHfAyk0Tw8KIaExbyDQ2JG4j6\\\\\\\/JdkT3\\\\\\\/1MSUL2r\\\\\\\/pC9qGVEBqn7CXeLQfEboWQ1jKkmgKLvMfTh\\\\\\\/xNADCJZeafArWXfEl8VDwkHCn7yjySqIT8vgl67v6JZXKewK0wNjwiqC0fYcXJqsHU8VQpP+NIDeNB\\\\\\\/UiJMcSE6Dvc86PiablHobQKixiPeC36nl8S5YIxZy+bF6J3fxQsL3F1ou2w7YcX+aZekaIaxgJWE0iR5tem9lyHAo3qNBuE4E7AyexdE1RsksFaB++nmaeXXylkdmNtdEuTfK\\\\\\\/PHWE5oeqADvFMTzqXlEObzJkNyAammMI1LprlFESTm6tgArcC2oXshRZyykjiHOmFQlJvL\\\\\\\/PajaNwa0uXS\\\\\\\/eoWoWzdPcuj9KTtpGfKqJA6PBjhJJ8JlJ7N3pgCqdGxbVsSAGBe3mYcOzyCzrcjshywtlZOVvZwyt+NfMO+6cFrlX5DX0tn3wZsYOZbu38zXYMKb0visXwHty\\\\\\\/UcHlpH9WICMrFfgetBvm0+dmFEJ1bPbY4MtrOEpANRTXPy8DkXv\\\\\\\/22F8+Hz8mmBfNcP4Yd\\\\\\\/ArG45Ie14R5VSIjU5If1\\\\\\\/ZQ3Htk6D6Z4D56DUrRwMNiNFuwtsHTOojHM7zqjNZwOVJH5CH7h8eg7\\\\\\\/laeHZOS9Udq4RGNWrL+pcf6LIlB+qctyf9TFXl9uaatU2w6NiByfD0XRL5DhYjGqtTpyr9x8LotEU9mwKDYK\\\\\\\/dIwLBt5QPfugL714d9kELKRsu5kboTIZLOEkKhF34l7j7qUu6Q\\\\\\\/hQ83MxxYFuCVwnDDWKusCxGpkM8pw6QI6iS7J4E6Vi8Ye5DzvzEVnsJvAyb\\\\\\\/pN\\\\\\\/C\\\\\\\/2y4+qXg+PRlUnypo3tFvQH8A6O+HldYooOwUCAN\\\\\\\/uAm9V2fUcTXEOq9KzNvll7QHqcEGlC0fLJB+N8GGzoXO2As8jOy8eH9v4fXCxI7kRYf\\\\\\\/oV5Sv08+5FydsY6WzIlXiLk7gAB47dVK5mkivaf5LX4QoZn7TwJk6ps4D8IOO7Cz7SH53ZN2RiZIB2hcySyW5U9B1rvBCD3eLXFpQeD9eIYXrWu82+4+0\\\\\\\/PZeoZTiGYVne7whiukjv6IfROaXpbOfpKod\\\\\\\/Hf\\\\\\\/XItRbqMPUhQDe4HryV1woiAs6aFYC1+IdQccqDxNmvGCkuV+kGMrSP742VDgiuwkPgBtEzl5lThQQ6h6yVQOg9QkthbsOBGk8v+aP9WTYzxExBDl0lIQ2RZKCAYferdGoky86\\\\\\\/b3tWojw8I\\\\\\\/PyrSONfBSkoVQzA6MSmDUMrvzy3C6LVnBKhFxRImPHokRdRibNBizaQ5FxsVCL3hpHE3cmeRT7Co8p9jNkeDEyV70kj8t+\\\\\\\/2Iea3vkoOZAiNWDtfmi2NTTMwXSKQw+2gkrcQZiUf5aw9PLiF9uE40BanHUXKWaN1SmdvN10HM5j2G3IsrzhiHT0QsAgdWg5Fjh9xcdJKnwoxuTVvQOMtcePuwf3EBNxNDR9OAG2kfMjqIY9xvjGzTPVdvW377zDaxlcZgBsf5pt0BoqLrabowMwCn3vHmGWaerhuqXjM1hrd4bSXBUnd2i7AL+8EipVqW3JCrnGfy+S0CGe9wFjIvkY3rjYL9iyD0U3s6joJWyZInr7Z8JJADEZo65+GVxQ5JgveWfPrcJDxccgnHadLRmOMBYvTOVUH2qfwVPtYUjVVhaEzVm8j6J3lmCazLGk8wY3qNsDf84ISjChx4SSiseLPdDeNK7iko6ce4rxL8uKOQXqqfIT9yzZeDNrsxQ8knDOCRZenPibVEfX\\\\\\\/Z1Zsg4ZEO2LgcFqwoYVnoOatZht0KPItc6W9o6RRIYdFNsdsK00zYOyjEKKhZvsGM\\\\\\\/nVU2osXl3wFUfExXbJfu44p0Ow\\\\\\\/HNosT4uE8br2E650n13LJtmNdkAAK3+pUrDBf4FcyPRjDR6rZm6ayyQkW2i9Dr48YuUcW1zOKarFipJOlgH5QFSrdiv4kcMSJbP9K5UptwaKbIz5DESH0bzktt3pB2GRJMJpth9RghSJaMoJGQnTSsvgzayo1hpupQ7z5w\\\\\\\/+gehvGUATdN7Q8mQED+N9Oo1A2utg2aGVCOATgZQhfCYURqxygQx9x+SuwW+TP\\\\\\\/hoXhXJktKPNpEBfMuGX75aKeQTZYCzXLcLlcDYI7WXAvCLThz7wIzuhUGL7oYUy3Vibb1uuwFd9fZbgfi7\\\\\\\/CO4unfsDDiVcao6c2rvkAppCR\\\\\\\/JZsJ14y30en1TMHDAWHy9CHWaxK\\\\\\\/2\\\\\\\/aYqEk+3GIpkBTLwWlFU+gE2UdKYHoZRSzwLmEeqz1uxwe+OIh6FLC53eIV5lBUp26ZI+oVM1vXafSKBBtLz2mYXILAlVrt1e35nScr8O+9fqJVhPhdsiPg1RvCseg\\\\\\\/Yy0sDNTN+NyynsG69lwVPYDCiy9fcaIeQWTd0IunPotn7ZTm4HQKBXD1KHzW3jFbreJMUuaXPCQcQs3yR+WgdC+pxnftnkri4Hc4nDSBkbvDFz994KFSM8M0t6n350lDwNT0s4BGpqVaHGoB0E2R13HuJ0bxZzwEKVNbial\\\\\\\/Jaa\\\\\\\/l2xDTUWYNpRKoD\\\\\\\/paMwj5ZZ5WRGFBGb3X7wKKbp7TIO9keFICE4HTmM4T7gqVLAZJiTJWiVso0flQkw+lxrN2YKP7TaZWTM9pVYCIfFJ1rfv\\\\\\\/NWKcPx7Z1Kl1NSiSWjVH5sUwt\\\\\\\/\\\\\\\/KqwS+6aHiMSbXQC3p5oomVdMpwlK6H1\\\\\\\/gfeH+APfjAghvx0Vxwxw\\\\\\\/Z+Fz3RnVVcKVy7TrN6JfLykmZAJnLYP44OybcR4OBiGqp57NgaKIcvMMjCbHabJqiZeuJ01aA0rhG3LfshPQrRR\\\\\\\/31y3c70SBofvuUg7fhB8DhpYgvNqr6mGimuzqmqZZDSLLP6N1i1UsIYjPNUVSgBCowhYj+c7FnBVM1qrXdL7pxdg2MZabau4g3rSHDYp4Q\\\\\\\/NIwV5l+xfRy7NAnM6eAU1NPzsrXXvoY6di4X1KdIso8yD89VjEsLnT91ouAMCymM+Gla4uOTdOY7rJbtL4iLDjfPl7vF+9HXJd\\\\\\\/bau5xwvFdhj5xfJhgNT6ZPaf9GoOKSK0ODb5WkTCvALcIeg0aj4OHKd6mrWuNnyoKXM2qGKvSvdxF0QBy50EVei9d\\\\\\\/PawNlsTuvaxMV9c\\\\\\\/6ztDQb6Vv3Qayw1D5VB0LhNVvCKxFvB655yqcS8UlR4ECuoQs+qGpD9QvZD9XiJCj1ipc9158VOkph6Wqy7StLOLRVmUjeD\\\\\\\/0JH89wzY90fmeQwrv3em7wiAepAVotL+lTU2TNVSnQhnX8Utb6EAQTbCVvqiNlLb3W0IBySfhDjKKxEHVXQ90WT5ZN3RTcb13J2r\\\\\\\/lHyVk1Ezl6HdDbWszi\\\\\\\/aYOjcOOseVFMvl8yRQcYpySSlKR8KplPEpElnjwvyio9goE3CEsj8L2RldXiKbN9KDRfthR9d7dTWPtFoQIYQ\\\\\\\/vT9Ti\\\\\\\/MEKlAF+5tO9BZxSpUjeI5iYaP445Tcodfq53JU2j2aJAgR2ND8fgdRtYR5X8cU90LdtIXbO2BgVTpPseQLCYs6zGO\\\\\\\/jN1wfW\\\\\\\/FO4g+6a0p3Q2+sW7DhXEqBn4D0rUlkEqOYzOkL+P0gBt5\\\\\\\/qswkdTD11Gs3vysMq76x\\\\\\\/kAxMkg0+ROIDNJi6+wOybRMx4xWS5mxYbXDkPqJcLLt8SOQdnSoBzDFo20QDvhCyBmCdTB68GPrGEI1Zgoz8gJJ5M5JNTasBDxKdXkkRlahwWGX6hm5Wm+kpzLC3PTzbKz4GiTpak6l2FZPP3he1sdaEdPqm3FDz1CzdzQX5M9EJtwnBHgzaqGtD3W2dGYyS28mt0dxV\\\\\\\/v4eVH6ee36eJYNFzShJyPdns85dvK55N2p0PZ1bm33Wjw5FgVA+OYZuyvsOriZe6uPrCJCffJhfC\\\\\\\/2c38GkJfX9CoVfZzaWyGrGM6JWr5pLSdwRutXH5s2\\\\\\\/NwB64B7jBqUqDMnLS0Vry6CuEpwuB9q6VuadYmVzvXKJgrVXN9e6KTZgr79wrPQKP8Mvg9FCcdfebNb61ySeQrFyayG8W2prF8mTLsX7eltGenNeRGBQyNuy5Wa1UKBPOfmoERcL8wPR39DlrLF8n6Fp4E5IO+jEuHpCxjpROQEBMN1K\\\\\\\/F3AR4yzVoX6t6V6DBChJ5ZHLNxVahJLlGlvW5tN9QXor5080r3pc4GaLRMZIxq+0d\\\\\\\/exAV1aqV\\\\\\\/RbPTV7H1oRBfFxyTEGg8ezpcmn1cLWt+fAo6OIWesj4U2uCH+vbWiZ+vk2grimFQuNY+UMx10TwSgJUWxOi3fHhwGb+4eJwMcPNscAtbPRlqI+SCj+Es0D+D3KhgaYLzLijx7VCXjcvo80lP\\\\\\\/qMXuHbbwd3kunxpK62s5gU\\\\\\\/HPXUbWa2kvlV9G2uQgCOkXUS8Sig1kyYJ89tG1BdZBs1YqrW7suRmcVrwXBqSmLLyoR9cgGTZOVJcQXKOJcvZhvrzOzu4Gd9f9DMo651cME9Gtx5rvLuETprBcA+9SeDrhIXHuzqov5jR+vMEte9YV9a5C7WWS6VplnSa4Aom6iaLJwEIV+KPT5U2cQOxUdHMwu1aWqJODsSnWtm4aG5XKQA37lRFWJIDSdnQvIAwv+25jx3pX99gTHNE+WPFvSbogGn7lokEDIwMaHUeIHiVYp\\\\\\\/6HBzIkksoK3LPG8PNzHVN7fHrQ9HUcv36aHDGOFzYI3+ATUB1halssCYCh9Zvf4x6H61X6FzkXxdNVKWXk+ORZ4wOw54Vo3lhly6t1Dj3hY9y40c3Ony3EiOP6aOSo1kk3eurlffJ8xjG\\\\\\\/aN8deeFX+kZjXVa7U3I2fmdhXKToN6wP6ca4FoEgOCDihiir7NzYkgmrbotM6eUXqka4eheGy\\\\\\\/yjvrymVEr+XEA95vMFa2113k5TwtGdv7OqTcQI13ZIiMGhdm1Fs1eanhfu3pLOqMM5XZFk1Wb5yhG8VQcFLBnmg+lJNH7\\\\\\\/eXnQAhz3FDJkgMcp\\\\\\\/d3tgZvR8Ztlks3ZOHZyYFZwTgW1eJNOKmkv18bLbgnHxt4lUqwVsg4JGhvNUgFrrWZkFmoETgUbQDxlapPgFAnPZbDRPlkkf\\\\\\\/1Fo1Lc4mSMtDiAjkUgY3b6tpOLK8LbnkD+Dsa6KGBeU+A0P9TA5IpJPnmmdVW77rYQO7aYCITSRNaweYe7vEkjuPH\\\\\\\/nB\\\\\\\/1dT5s6rydEFwCcIq33uM0KmOZffW\\\\\\\/95Vv+zPXdGVdijOB9E18eGDdNMrD8kAdsTqkkYTCZQ1DL2ekLVJHG804qDFREqGubHnAaH1i8hJvDOdu7P9VSs\\\\\\\/tadiEE2PZXnR\\\\\\\/As5t2ETxJcEPSrpabAPacraaeN4m\\\\\\\/aeSDqEqTsm4gkiuafRYIS4SXXjthXwTTsDh2d3i4pxLv4aE2e7Dp4avNTDVMHO6gpvMGmSigO87QlHYK3OzLl9Proh4GOnFyry7PK4Dv0cU5Ll1XefieeZTb211a\\\\\\\/KrQ48eRmNj87VJ7F4VhDPehlIYQ9uLvEOm6ExVGHucicGbbFSZZBV9upYOnp6V5yHSUfFw\\\\\\\/tdjQRFp8YqeRrJzvep48OvFnZytDXGgQFp71NGatR2RJ1CqaUudm6ts1k6Qu8z2WAzc8BTF6tN3ewb4B9ZGlddzbujqFzGnP+lJsV4iwwhvsIF47jTt8PL4tA+mrEXdnTRrCZCSa2bXc2lPsYWIYmC6fyESSYk9AC9JwOcz6Pt4KY3+9LD442nD2SEgODBPCsk5uWKNbEHpUGoR8MJ74LK\\\\\\\/pPjOME5YHIMtgNNmsvvbjo4eMueT8St5OlHE2DAVS\\\\\\\/+PF\\\\\\\/WnCE6x6myFR55rVA4G3uEHMvKYjFvSVzZuDxLXMA5ciAcc2Li9c+muwNPo3wTZM2e8EY0o7f1mhT9vtFrbHjLpSAO9FdF0ZMxF+Xll+etXspNgcM31OVaSKFUJnTBUZxHGB1ogBtrAuqSJl\\\\\\\/CUr11GhJofjUszX4WFDVb8JS7gR7v2OZ+9tEcbZzXIMvU7ufXPQ9wCvLaFC0FH0j3h2RevTuhpheqWkZzPCEi1+jZuTRXpHMTCnt8G\\\\\\\/P\\\\\\\/LvB5iwqOTKUuHco6alTsuFyYYKt3nu9mD95DhXFDhuTG6hiha3Hlbni\\\\\\\/+X9f5Ltu43QtEchgxgC4MLk+5trGWaLQrxV6AVh7L+lLg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qsauIm5mPLgbTLUZZGJ5xg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a2ae581cf08f47b212b63b6c1f6d45088ba7a1b4560bd5f286941adb2347221e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220DU23Vz\\\\\\\/7+c7Vd2KbYsE7OKqZzqsjulkZMaBSFKW9ECazznLj+YDn+mho+ch55ZG9zveu4ObhN7pIQKqMM\\\\\\\/YQP+\\\\\\\/knZstcUM\\\\\\\/y614M7d56P5E6BWAtS2KcOXVFgW4SFGYWvvd17mv59R\\\\\\\/3Wvr70GFIRMezD\\\\\\\/7LCj\\\\\\\/toeSMHZEbeDGPeaW1ihMFaPBlZjO\\\\\\\/Luf\\\\\\\/Ei1Rm2leL0glrx3oe8M30JSBQUEVowfiBQNo\\\\\\\/ApOzehmXhdnNEhZetZ4KdSSSMt8he2s5uq8M35iQZLIvIz5XBN5mh0SSocm5ZHEhvV6HD2FKkd4Aw0kgAh03yXcz1FkNxftSahpVHcDqIIBXQcluZbw+lGKRNJ3CfOZm+dCOOfZoewAgYrwzGkTftZdaWvCOXYG7s8OhCDdAR05NXUzOzqHwxW3TUIjzRlWpZiSP6t9L1I\\\\\\\/QMCcCKi63hIXkTe+jTxqGmE2QXiaMld2Pw34LCXmKcd03EokOQmhP0PPQNMF3NAXKI+JFawmBFqCELIjJcosCtf3V\\\\\\\/ELGznhJ4H7jBwmltt7brLj7fivII5nDs\\\\\\\/XVPAmy43PU8Pvs73jP+YjYIyiMLaztLys9EqzoUtYJrcrYiLakIERqGFqHX3y+54oPNGt943ROgIvqZLop5nEMDpq50b8F0ZBBY9sg0gB3IuZnWxdQxn+3OL9henT66631PMUYyjRk\\\\\\\/mstrgSdRVmXRiCOCzmkWd0J3GRW5CqzXD472oCVVw6X8XHjEYgOaMNVpvA2CV7IAF9g\\\\\\\/hA69vOSg7heyJAo6XIySicjmF46OsjCeQmeRoB8bByFnIuE+Wo5yGbWglrWiNbtsrSXtPvMnMyfRMB1q+BULv0QETyRGWwpfK4zGm+AWMa0ug5ki8yk6DiluQiw2KNZj8RTN5GPsvBEmMjbmi22chFtYsRkJGKKfKPQTe\\\\\\\/qWgtwqG7nudErRr3UGQXkaNx7M45W3NozqTRWigTI5YekGVKoklmgp4e2LlHKASYi99SrO4pb\\\\\\\/fqkBJVSC9k8Eg2NZe5LSo8x+J4GqtOBEz7tOQEHtMw+3CoL1DaJmVb8EnZFF\\\\\\\/ixZyMdx46wAS9ZVGsl6EdkKxxi66uZ2jmPUl3pBEfn6j6FaPNpn\\\\\\\/\\\\\\\/TPUHH1lJaWlsh8c9rnS5MDM1S7WW4pAEuKSZTIWid3TKtlobyjRXHZsZFy1rBRW\\\\\\\/hePxLBe9tTqfaMED47X5n4VElwhHUBjyX0S1xhC4OVLZNV8kdMLkd7gKSmKWa7szlvSMlwS8rNi6yKi7g5QlKg\\\\\\\/97P4PaPSHKubgIM3w7XK0tl8ggqEXu4ent962ks2RCyRM64pTE5zPR\\\\\\\/AmGSXO8yakXrb5vbFPi0KvPIokONc7JNQQTp\\\\\\\/ffaJQzX8TTUqxgUKsNWhfQQIoCM\\\\\\\/mvRUafEqSX5Iv9ndnc++B3FcK9\\\\\\\/SkfPaull4XZyiyfdxrbkq84TNp2xuWIUkmJJB6FoR2RTSFRfPYnfdyiERFwYCoC09ZWtSKsXHhMGDsu9WpZFmwPMWHcaiWOONkggb0BcFHkL0lIR8Gfap8dm52rs+xSjmIlK7PJs87BwUlA51hHVVtXIO3Ol4p2BKM4takIV2h67lfCtUQUbq7AOLgdQNxd3q+GhXV9bzeQsAQGA8sc3rG\\\\\\\/PnwSbeQVV2eDfYxpS0eBmNDPx31yHnCEELM1qOlJoC5skUiltrQ6RLGV\\\\\\\/OFn3O0qpqgw+ZfYV3aKBnwGpeP0M1q9wnqH0Z9jm92tNezyreQLq5mBvi79rxpn2RUOWQYIsGpNU608ADtb59JL0ONi6N6BoXyoaIXp+it+ubJEQqmwb9Z9GRdgOGYfHTj7hIV+mlAjmOdQDByCLXnBHYUtXAuQ3btjJ6F9BrSVrN6++DDGH+mtXT8+m1Jauex54Ua3o0WQQ7\\\\\\\/CHFI1Uhl09HsMqAENT4AGBEKDCuFEj7fL+YvGOVfCFk43SWOqiPTo0i\\\\\\\/D9bNiqLRy+TXPTXUZnuUV\\\\\\\/w45C7MxQT\\\\\\\/i\\\\\\\/JQJCd5VTooP6lxHSOP1NuQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022fillwk3JPxsqxTzlDfInGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225d7b563581e4217d2e24251b621e6e858c3641e80d3dd006a46d58140e55d4b2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459311 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/873" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bllvT6PqQVH2SAtEify0GqXR9VyFi-Yl9KUfizM8zsc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/873" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wbmycMbxxi05BnnjbtFloGChbJavFNzJcwwN5xujIis" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/865" title="Proper Cyclin B3 Dosage Is Important for Precision of Metaphase-to-Anaphase Onset Timing in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/883" title="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459311" id="node459311" data-pisa="ggg;2/8/873" data-pisa-master="ggg;2/8/873" data-apath="/ggg/2/8/873.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">X. Hong</span>, <span class="highwire-citation-author" data-delta="1">H. Doddapaneni</span>, <span class="highwire-citation-author" data-delta="2">J. M. Comeron</span>, <span class="highwire-citation-author" data-delta="3">M. J. Rodesch</span>, <span class="highwire-citation-author" data-delta="4">H. A. Halvensleben</span>, <span class="highwire-citation-author" data-delta="5">C. Y. Nien</span>, <span class="highwire-citation-author" data-delta="6">F. Bolei</span>, <span class="highwire-citation-author" data-delta="7">R. Metpally</span>, <span class="highwire-citation-author" data-delta="8">T. A. Richmond</span>, <span class="highwire-citation-author" data-delta="9">T. J. Albert</span> and <span class="highwire-citation-author" data-delta="10">J. R. Manak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">873-882;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459311" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">X. Hong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=X.%2BHong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hong%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AX.%2BHong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">H. Doddapaneni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H.%2BDoddapaneni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doddapaneni%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH.%2BDoddapaneni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">J. M. Comeron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BM.%2BComeron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comeron%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BM.%2BComeron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. J. Rodesch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BJ.%2BRodesch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodesch%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BJ.%2BRodesch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">H. A. Halvensleben </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H.%2BA.%2BHalvensleben%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halvensleben%20HA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH.%2BA.%2BHalvensleben%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">C. Y. Nien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, Center for Developmental Genetics, New York University, New York, New York 10003, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BY.%2BNien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nien%20CY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BY.%2BNien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">F. Bolei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=F.%2BBolei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolei%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AF.%2BBolei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">R. Metpally </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Translational Genomics Research Institute, Phoenix, Arizona 85004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BMetpally%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metpally%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BMetpally%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">T. A. Richmond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T.%2BA.%2BRichmond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richmond%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT.%2BA.%2BRichmond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">T. J. Albert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T.%2BJ.%2BAlbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albert%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT.%2BJ.%2BAlbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">J. R. Manak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pediatrics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BR.%2BManak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manak%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BR.%2BManak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john-manak@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">john-manak@uiowa.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/8/873" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459311" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459311/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/873.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459311" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459311/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/873.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459311" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459311/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/873.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459311" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459311/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/8/873.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459311" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459311/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Faithful annotation of tissue-specific transcript isoforms is important not only to understand how genes are organized and regulated but also to identify potential novel, unannotated exons of genes, which may be additional targets of mutation in disease states or while performing mutagenic screens. We have developed a microarray enrichment methodology followed by long-read, next-generation sequencing for identification of unannotated transcript isoforms expressed in two <em>Drosophila</em> tissues, the ovary and the testis. Even with limited sequencing, these studies have identified a large number of novel transcription units, including 5′ exons and extensions, 3′ exons and extensions, internal exons and exon extensions, gene fusions, and both germline-specific splicing events and promoters. Additionally, comparing our capture dataset with tiling array and traditional RNA-seq analysis, we demonstrate that our enrichment strategy is able to capture low-abundance transcripts that cannot readily be identified by the other strategies. Finally, we show that our methodology can help identify transcriptional signatures of minority cell types within the ovary that would otherwise be difficult to reveal without the CoNECT enrichment strategy. These studies introduce an efficient methodology for cataloging tissue-specific transcriptomes in which specific classes of genes or transcripts can be targeted for capture and sequence, thus reducing the significant sequencing depth normally required for accurate annotation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li><li class="kwd"><a href="/keyword/array-capture" class="hw-term hw-article-keyword hw-article-keyword-array-capture" rel="nofollow">array capture</a></li><li class="kwd"><a href="/keyword/enrichment" class="hw-term hw-article-keyword hw-article-keyword-enrichment" rel="nofollow">enrichment</a></li><li class="kwd"><a href="/keyword/transcript-isoforms" class="hw-term hw-article-keyword hw-article-keyword-transcript-isoforms" rel="nofollow">transcript isoforms</a></li></ul><p id="p-6">Over the last few years, it has become evident that a significant percentage of genes in higher eukaryotic genomes encode a multitude of tissue-specific transcript isoforms expressed in temporospatially distinct patterns (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Graveley <em>et al.</em> 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Manak <em>et al.</em> 2006</a>). To comprehensively annotate a genome and understand the regulation of such complex gene structures, it is important to identify all transcript isoforms. Recently, several studies have attempted to profile the transcriptome of specific tissues or developmental stages in a variety of higher eukaryotes using several genomic methodologies, including tiled genomic microarrays, RNA-seq, and Sanger sequencing of cDNA libraries (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cherbas <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Costa <em>et al.</em> 2010</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> 2010</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Graveley <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Malone and Oliver 2011</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Manak <em>et al.</em> 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wan <em>et al.</em> 2006</a>). RNA-seq is the most high-throughput and efficient methodology of the three approaches, and it ultimately relies upon cDNA sequencing using next-generation sequencing platforms, such as Illumina, SOLiD, or 454 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Malone and Oliver 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2009</a>). The first two sequencing platforms generate large numbers of short-read sequences, whereas the 454 platform generates fewer but significantly longer read sequences (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Metzker 2010</a>). However, the standard RNA-seq procedures that utilize all of these platforms have numerous shortcomings. First, to prepare cDNA libraries for next-generation sequencing, steps are needed to remove rRNA (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wilhelm and Landry 2009</a>), as without such removal these rRNAs would contribute to a large percentage of the sequencing reads, thus reducing the ability to more deeply sequence the mRNA pool. Additionally, other undesirable RNAs, which might include highly expressed housekeeping genes, can make up a significant percentage of reads from an mRNA sample, once again hindering the ability to probe more deeply into the transcriptome. Second, we and others have found that for many standard RNA-seq experiments, even when performed using published “standards” of read number and depth coverage, such experiments often fail to fully “cover” low-expressed or even moderately expressed genes, leaving “holes” in the gene structures (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Graveley <em>et al.</em> 2011</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Malone and Oliver 2011</a>). Thus, to interrogate low-level transcripts, a larger number of sequencing reads need to be generated, which can make the methodology cost prohibitive. Third, for transcript isoforms expressed in a small subset of cells in the tissue or organism of interest, it may be difficult to characterize the transcripts specific to those cells (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Graveley <em>et al.</em> 2011</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Malone and Oliver 2011</a>). For example, if only a few progenitor heart cells begin expressing heart-specific genes in the context of a much larger group of cells, such transcripts might go undetected; this would result in a failure to identify the true initiation of heart-specific gene transcription. Fourth, for transcriptionally complex genes encoding many different cell-specific isoforms, it can be difficult to identify all the relevant transcript isoforms.</p><p id="p-7">To overcome such limitations, we developed an array-based sequence capture strategy that we call CoNECT (<span class="underline">c</span>apture <span class="underline">o</span>f <span class="underline">n</span>ovel <span class="underline">e</span>xpressed <span class="underline">c</span>ell type–specific <span class="underline">t</span>ransfrags) in which we enriched for tissue-specific cDNA fragments [transcribed fragments, or transfrags, (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Manak <em>et al.</em> 2006</a>)] using exome microarrays followed by 454 sequencing. Given the rich diversity of transcript isoforms hypothesized to be expressed in the germline, we chose to focus on <em>Drosophila melanogaster</em> testes and ovaries (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kai <em>et al.</em> 2005</a>). We used a 454 sequencing platform specifically because it generates long read lengths; indeed, in our experience, novel 5′ ends (which make up a substantial percentage of transcript diversity) can extend several hundred bases from an annotated exon, sometimes containing multiple exons. We thus wanted the capability of generating the longest length of sequence possible to faithfully annotate these exons and their appropriate splice junctions. Exome microarrays and related methodologies have recently been used to enrich exonic regions of genomic DNA for identification of disease-causing mutations in humans (Bamshad <em>et al.</em> 2011; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pierson <em>et al.</em> 2011</a>) and mice (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fairfield <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hilton <em>et al.</em> 2011</a>). We wanted to explore whether this technology could be adapted to cDNA enrichment and sequencing to identify novel transcript isoforms that had previously gone undetected in these tissues. Recently, Mercer and colleagues (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mercer <em>et al.</em> 2012</a>) used a similar strategy to capture unannotated transcripts in specific regions of the genome. However, their study focused on tiling array capture and characterization mostly of novel transcripts, whereas the study described here focuses on characterizing the novel isoforms in the protein-coding transcriptome across the entire genome through utilization of exome arrays. Validation of such a method would allow exome enrichment platforms already in the marketplace to be adapted for this purpose, as well as pave the way for the use of more specific exon-based arrays that seek to interrogate a limited number of genes. We report here on the proof of principle for this methodology and demonstrate that novel transcript isoforms can be effectively identified while minimizing cost. The study also demonstrates that CoNECT can identify genes expressed in small subsets of cells within a tissue.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly strain, tissue dissection, RNA extraction, and cDNA synthesis</h3><p id="p-8">The fly strain we used in this study is <em>Drosophila melanogaster</em> [<em><a href="http://flybase.org/reports/FBgn0004034.html">yellow</a></em> (<em><a href="http://flybase.org/reports/FBgn0004034.html">y</a></em>); <em><a href="http://flybase.org/reports/FBgn0000337.html">cinnabar</a></em> (<em><a href="http://flybase.org/reports/FBgn0000337.html">cn</a></em>); <em><a href="http://flybase.org/reports/FBgn0000241.html">brown</a></em> (<em><a href="http://flybase.org/reports/FBgn0000241.html">bw</a></em>); <em><a href="http://flybase.org/reports/FBgn0003466.html">speck</a></em> (<em><a href="http://flybase.org/reports/FBgn0003466.html">sp</a></em>)] used in the BDGP sequencing project (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Celniker <em>et al.</em> 2002</a>). Fly embryos were collected using small grape plates (0–4 hr, 25°) at day 0 and raised at 25°. Eclosing virgin females and males were collected into different bottles containing standard media from day 9 to day 10. Ovaries and testes were dissected in 1× PBS solution under a dissecting microscope from day 11 to day 12 (1–2 days after eclosion). Dissected ovaries and testes were quickly frozen in Trizol reagent at −80°. Total RNA was isolated from ovaries and testes following the Trizol RNA isolation protocol (Invitrogen) and purified on RNeasy columns (Qiagen). mRNAs were isolated using the Poly(A) purist kit (Ambion) and then transcribed into double-stranded cDNA using the Superscript Double-Stranded cDNA synthesis kit following the manufacturer's instructions (Invitrogen). Six µg of ovary and 5.7 µg of testis cDNA were sent to Nimblegen for capture.</p></div><div id="sec-3" class="subsection"><h3>CoNECT array design</h3><p id="p-9">The collection of all known and predicted exons in <em>D</em>. <em>melanogaster</em> r5.19 was downloaded from FlyBase (<a href="ftp://ftp.flybase.net/genomes/Drosophila_melanogaster/dmel_r5.19_FB2009_06/fasta/">ftp://ftp.flybase.net/genomes/Drosophila_melanogaster/dmel_r5.19_FB2009_06/fasta/</a> dmel-all-exon-r5.19.fasta and /dmel-all-predicted-r5.19.fasta). Target coordinates from these sets were consolidated to combine overlapping target regions, resulting in 83,616 target regions totaling 34,963,568 bp. Unique probes targeting these coordinates were selected against <em>D. melanogaster</em> r5.7, utilizing a rebalancing algorithm to improve uniformity of coverage across targets as previously described (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bainbridge <em>et al.</em> 2010</a>), resulting in 2,160,993 probes targeting 81,872 regions covering 34,418,101 bp.</p></div><div id="sec-4" class="subsection"><h3>CoNECT procedure</h3><p id="p-10"><em>Drosophila</em> ovary and testis cDNA samples were checked for quality and degradation using an Agilent 2100 Bioanalyzer with a DNA 7500 chip. Five hundred nanograms of each cDNA sample was used for library preparation following 454 LifeSciences <em>Rapid Library Preparation Method Manual</em> (<a href="http://dna.uga.edu/docs/GS-FLX-Titanium_RapidLibrary%20%28RL%29%20Preparation_Method_Manual%20%28Roche%29.pdf">http://dna.uga.edu/docs/GS-FLX-Titanium_RapidLibrary%20%28RL%29%20Preparation_Method_Manual%20%28Roche%29.pdf</a>). After preparation via the 454 protocol, samples were eluted in PCR-grade water and subjected to precapture ligation-mediated PCR (LM-PCR) using the Roche FastStart High Fidelity PCR system (cat no. 04738292001). Each reaction contained 10 µl FastStart buffer, 2 µl DMSO, 2 µl PCR nucleotide mix, 10 µl each of oligos LM-PCR 454 Ti-A: 5′–CCATCTCATCCCTGCGTGTC–3′ and LM-PCR 454 Ti-B: 5′–CCTATCCCCTGTGTGCCTTG–3′ at 40 µM concentration, 15 µl PCR-grade water, 50 µl cDNA sample, and 1 µl FastStart High-Fidelity Enzyme Blend. Reactions were subjected to 95° for 10 min, followed by 30 sec at 95°, 30 sec at 64°, 3 min at 72°, with repetition of steps 2–4 for 11 cycles, followed by 7 min at 72° and then held at 4°. Amplicons were then purified using a Qiagen Qiaquick column with elution in 50 µl of water. Sample hybridization was performed on the Roche NimbleGen custom 2.1M <em>Dropsophila</em> exome array described above. Hybridization cocktail containing 2 µg of precapture library with 300 µg of Cot-1 DNA was dried on high heat (60°). Once dried, 1 µl of each hybridization-enhancing oligo was added (R-A Hyb Enhancing: 5′–CCATCTCATCCCTGCGTGTCTCCGACGACT–3′ and R-B Hyb Enhancing: 5′–CCTATCCCCTGTGTGCCTTGCCTCCCACGACT–3′ at 1000 µM concentration), in addition to 11.2 µl of PCR-grade water, 18.5 µl of Nimblegen 2× hybridization buffer, 7.3 µl of NimbleGen Hybridization Component A. Sample tubes were mixed and heated at 95° for 10 min. Hybridization cocktail was loaded onto the array using an HX1 mixer and NimbleGen Hybridization Station according to manufacturer's instructions and left to mix at 42° for 68 hr. Slide washing and sample elution were performed as previously described (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fu <em>et al.</em> 2010</a>). Eluted samples were again subjected to LM-PCR as above with samples divided into four reactions to minimize potential amplification bias. Postcapture LM-PCR consisted of 20 cycles, after which samples were purified using Qiagen Qiaquick columns. Captured samples were then sequenced using the Roche 454 Genome Sequencer FLX system.</p></div><div id="sec-5" class="subsection"><h3>Identification of novel transcript isoforms</h3><p id="p-11">Newbler (v2.3) was used to assemble the 454 reads into isotigs or singletons. To identify new isoforms, we used the RepeatMasker program version 3.2.9 and the NCBI/RMBLAST that is part of the program as search engine. The RepeatMasker program ordinarily screens DNA sequences for interspersed repeats and low complexity DNA sequences against a provided database of known repeat sequences; in this case, instead of using a repeat sequence database, fasta sequences of known <em>D</em>. <em>melanogaster</em> genes or the genome were used as a database for screening. This leads to the masking of the complementary regions of the known genes in the sequenced dataset, and the regions left unmasked (either 5′, internal, or 3′ to the masked sequence) were treated as potential novel sequences for further evaluation. Specifically, two rounds of screening using the RepeatMasker program were carried out to identify novel exons and extensions using the default program settings. In the first round, isotigs and singletons were screened against the <em>D</em>. <em>melanogaster</em> all-genes database (dmel-all-gene-r5.33.fasta) to determine sequences that are fully masked, suggesting that they are fully known transcripts, as well as partially masked sequences that may include a novel extension/insertion. A sequence was considered as a potential new isoform if the region left unmasked was at one end of the sequencing read and was greater than 10 bp long, or it was inside the masked sequence. In the second round, these partially masked sequences were screened against the <em>D</em>. <em>melanogaster</em> genome (dmel-all-chromosome-r5.33.fasta) to identify the genomic location of these sequences. Depending on the matching positions of the sequences and their unmasked region, seven different categories of novel exons or extensions can be determined, including 5′ novel exons, 5′ exon extensions, 3′ novel exons, 3′ exon extensions, novel internal exons, internal exon extensions, and gene fusions. Raw sequence data have been submitted to the sequence read archive (SRA), accession number SRA050707.2. Ovary and testis isotigs over 200 bp have been submitted to the Transcriptome Shotgun Assembly Sequence Database in  GenBank The bioproject number is PRJNA89451, and the accession numbers are <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JV208106&amp;atom=%2Fggg%2F2%2F8%2F873.atom">JV208106</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JV230865.&amp;atom=%2Fggg%2F2%2F8%2F873.atom">JV230865.</a> Singleton sequences, as well as isotigs less than or equal to 200 bp, are included as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/003194SI.pdf">supporting information</a> for the ovary (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FileS2.zip">File S2</a>) and testis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FileS3.zip">File S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FileS4.zip">File S4</a>) datasets.</p></div><div id="sec-6" class="subsection"><h3>Tiling array gene expression analysis</h3><p id="p-12">The custom-designed <em>Drosophila melanogaster</em> tiled genomic microarray has been previously described (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nien <em>et al.</em> 2011</a>). This array set, which comprises two HD2 (2.1 million feature) microarrays, utilizes 50-mer oligonucleotide probes with up to 100 close matches per sequence tolerated and a median probe spacing of 33 bp. After hybridization, processing, and scanning of three technical replicate array sets, intensity readings of probes were corrected according to their GC content using a set of random probes on the arrays, and then normalized by quantile normalization (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Nien <em>et al.</em> 2011</a>). After normalization, a median filter was applied. The expression level of each exon was calculated by taking the median of all the probes covering the exon, and the expression level of each RNA isoform was calculated by averaging the expression levels of all the exons of the isoform without weighting. The background threshold was set as 2% FDR (calculated by “fdrtool” package). Genes were considered as expressed (present) if more than 70% of the probe signals were higher than the threshold; otherwise, they were considered as not expressed (absent) (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Nien <em>et al.</em> 2011</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS10.xlsx">Table S10</a>). Genes that were present in all three technical replicates were considered as expressed and further used to compare with genes identified by CoNECT.</p><p id="p-13">To make the GFF files visualized in the SignalMap genome browser, tiling array data were quantile normalized and median filtering was applied on probes from the three replicates employing a sliding window of three probes [assigning the median value to the center probe from the three replicates (3×3)]. Background subtraction was performed by subtracting the top 2% of random probe (negative control) intensity values from individual <em>D. melanogaster</em> probe intensity values. GFF files were generated after the probes with negative intensity values (after background subtraction) were rounded to zero (see GFF files in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FileS5.zip">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FileS6.zip">File S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FileS7.zip">File S7</a>).</p></div><div id="sec-7" class="subsection"><h3>Selective capture array enrichment analysis</h3><p id="p-14">To obtain CoNECT expression data directly comparable to tiling array gene expression and traditional RNA-seq (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> 2010</a>) datasets, we determined the FPKM values based on 454 sequencing reads generated by CoNECT. We first mapped 454 reads to the whole list of <em>D. melanogaster</em> transcripts (dmel-all-transcript-r5.45.fasta) using BWA-SW (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li and Durbin 2010</a>) and then obtained FPKM values per transcript based on the best match per read and using samtools idxstats (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2009a</a>). Finally, we obtained FPKM values per gene by collapsing into one all alternative transcripts when present.</p><p id="p-15">To determine whether CoNECT effectively enriches for targeted transcripts at the expense of transcripts that were not represented in the capture array design, we investigated the number of reads corresponding to the duplicated gene <em>ada1</em> (<em>ada1-1</em>, FBgn0051865; <em>ada1-2</em>, FBgn0051866), which showed strong expression on the tiling array but was not interrogated by any probes on the capture array, and the 10 most closely expressed genes determined via the tiling array analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS12.xlsx">Table S12</a>). The number of 454 reads that mapped to <em>ada1</em> and the 10 most closely expressed genes are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS12.xlsx">Table S12</a> (along with their FPKM values). Importantly, any reads with sequences identical to two locations in the genome were not removed and one (the best match) was chosen; thus, no reads mapping to <em>ada1</em> would have been discarded from both duplicated copies. To test for a significant reduction of <em>ada1</em> reads, we took into account the differences in each gene transcript size among the 11 genes.</p></div><div id="sec-8" class="subsection"><h3>David functional annotation clustering</h3><p id="p-16">Nine-hundred seventy-four capture-specific genes were uploaded onto David Bioinformatics Resources 6.7, and 135 clusters were generated using the default option setting and stringency (medium). We removed clusters and terms under each annotation cluster that had <em>P</em>-values larger than 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS13.xlsx">Table S13</a> lists 42 enriched clusters with significant <em>P</em>-values. The enrichment score of each annotation cluster is the geometric mean (in -log scale) of the member's <em>P</em>-value and is used to rank its biological significance (Huang da <em>et al.</em> 2009). Also, 32 neuro-related GO terms covering 63 genes were identified by looking through all 135 clusters, regardless of the <em>P</em>-value scores. Among these 32 terms, 4 terms under cluster 10, including neurotransmitter receptor activity, neurotransmitter binding, neuropeptide binding, and neuropeptide receptor activity, had significant <em>P</em>-values, which are indicated in the parentheses after each term (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS14.xlsx">Table S14</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Capture of expressed transcripts</h3><p id="p-17">A 2.1 million feature Roche NimbleGen microarray targeting 81,872 annotated exons from <em>Drosophila</em> release r5.7 was designed using synthesis cycles that produced oligonucleotides of up to 106 bases, with an average size of 76 bases. Total RNA was isolated from approximately 16,000 testes and 6000 ovaries and mRNA was purified, followed by double-stranded cDNA synthesis (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). After nebulization of the cDNA followed by library preparation and LM-PCR, the cDNA fragments were hybridized to the array. We used nebulization parameters that produced cDNA fragments of, on average, 700 bp as assessed by bioanalyzer traces (data not shown). Given that most of the novel 5′ exons identified in our previous study were less than 300 bp in total length (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Manak <em>et al.</em> 2006</a>), we reasoned that capture of 700 bp fragments by known exonic probes would allow for capture of entire 5′ unannotated exons. After elution from the array, the postcapture cDNA was then sequenced using the Roche 454 Genome Sequencer FLX system. The average read lengths obtained were 356 bp for the ovary sample and 374 bp for the testes sample, although we were able to obtain 53,791 and 48,218 reads over 500 bp for each sample, respectively (with the longest reads of 703 and 686 bp).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/873/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pipeline underlying the experimental design. Ovary sample (indicated by red arrows) was processed for both tiling array (bottom left) and sequence capture array (bottom right). Testis sample (indicated by green arrows) was used only for sequence capture analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1435957390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pipeline underlying the experimental design. Ovary sample (indicated by red arrows) was processed for both tiling array (bottom left) and sequence capture array (bottom right). Testis sample (indicated by green arrows) was used only for sequence capture analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/873/F1.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/873/F1.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/873/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/873/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-18" class="first-child">Pipeline underlying the experimental design. Ovary sample (indicated by red arrows) was processed for both tiling array (bottom left) and sequence capture array (bottom right). Testis sample (indicated by green arrows) was used only for sequence capture analysis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Identification of novel exons and splicing events</h3><p id="p-19">Over 540,000 and 700,000 reads were generated for the ovaries and testes cDNA samples, respectively. Isotigs and singletons were generated using Newbler (v2.3) and then mapped back to the genome (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This resulted in the creation of 9553 and 13,275 isotigs (sequences built from multiple reads that are analogous to individual transcripts) as well as 16,554 and 20,037 singletons (single read sequences) for the ovaries and testes, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The RepeatMasker was used to determine whether the expressed transcripts contained novel, unannotated transcript information. Through repeat masking, novel transcribed units appended to known exons represented on the array were identified and subsequently hand-curated for validation (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). To call a transcribed unit as a novel exon, we required perfect splice consensus sites in the genomic sequence flanking the exon.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461740/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461740/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">CoNECT Summary Statistics</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/873/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of novel transcript isoforms. RepeatMasker was used to screen singletons and isotigs against D. melanogaster all-gene database (dmel-all-gene-r5.33.fasta.gz). Partially masked sequences were then repeat masked against dmel-all-chromosome-r5.33. Total genes captured are a combination of these two categories. Novel transcript isoforms were confirmed by hand curation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1435957390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of novel transcript isoforms. RepeatMasker was used to screen singletons and isotigs against &lt;em&gt;D. melanogaster&lt;/em&gt; all-gene database (dmel-all-gene-r5.33.fasta.gz). Partially masked sequences were then repeat masked against dmel-all-chromosome-r5.33. Total genes captured are a combination of these two categories. Novel transcript isoforms were confirmed by hand curation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/873/F2.medium.gif" width="440" height="371"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/873/F2.medium.gif" width="440" height="371"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/873/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/873/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461659" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-20" class="first-child">Identification of novel transcript isoforms. RepeatMasker was used to screen singletons and isotigs against <em>D. melanogaster</em> all-gene database (dmel-all-gene-r5.33.fasta.gz). Partially masked sequences were then repeat masked against dmel-all-chromosome-r5.33. Total genes captured are a combination of these two categories. Novel transcript isoforms were confirmed by hand curation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Seven categories of novel exons or extensions (continuations of the sequence of an annotated exon) were identified, which included 5′ novel exons, 5′ exon extensions, 3′ novel exons, 3′ exon extensions, novel internal exons, internal exon extensions, and gene fusions. Overall, 298 transcripts with novel 5′ exons were identified, thus signifying novel transcription start sites (TSS), some of which are potentially germline specific, with 22 sequences represented by more than one novel exon (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS1.xlsx">Table S1</a>). The average size of novel 5′ exons was 129 bp and 135 bp for ovaries and testes, respectively, strongly suggesting that our nebulization strategy (which generated cDNA fragments of, on average, 700 bp) would allow capture of most novel first exons. For example, if the associated second (array-interrogated) exon was 100 bp and this exon was bound by a 700 bp cDNA fragment, then up to 600 bp of “extra” sequence could theoretically be “captured.” Sequencing from the novel exon end with the 454 technology would then generate enough sequence to cover the novel first exon and extend into the annotated exon. For example, testes isotig04218 contains a novel exon of 368 bp as well as an extension of the first annotated exon; thus, 484 bp of novel sequence was captured via the exome array. In summary, sequencing with a long-read platform allowed us to faithfully determine precise exon-intron-exon structures over several hundred contiguous nucleotides.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/873/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of novel transcript isoforms identified by sequence capture. (A) Testis singleton GN8820S02HWWQT matches to gene CG33218 and contains two 5&#x2032; novel exons, which are about 35 kb away from the first annotated 5&#x2032; exon. (B) Testis isotig04144 matches to gene CG13527 and contains a 3&#x2032; novel exon. (C) Testis isotig02449 matches to gene dgt1 and contains a novel internal exon. (D) Testis isotig00435 matches to gene trol and is missing several internal exons. (E) Testis singleton GN8820S02GZB4E fuses together exons of genes Obp8a and CG15369. (F) Ovary isotig00056 matches to gene CG14073 and contains a novel 5&#x2032; exon. P-element Dmel\P{GSV6}GS14564 is inserted in the middle of this novel exon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1435957390" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Examples of novel transcript isoforms identified by sequence capture. (A) Testis singleton GN8820S02HWWQT matches to gene CG33218 and contains two 5&#x2032; novel exons, which are about 35 kb away from the first annotated 5&#x2032; exon. (B) Testis isotig04144 matches to gene CG13527 and contains a 3&#x2032; novel exon. (C) Testis isotig02449 matches to gene &lt;em&gt;dgt1&lt;/em&gt; and contains a novel internal exon. (D) Testis isotig00435 matches to gene &lt;em&gt;trol&lt;/em&gt; and is missing several internal exons. (E) Testis singleton GN8820S02GZB4E fuses together exons of genes &lt;em&gt;Obp8a&lt;/em&gt; and CG15369. (F) Ovary isotig00056 matches to gene CG14073 and contains a novel 5&#x2032; exon. P-element Dmel\P{GSV6}GS14564 is inserted in the middle of this novel exon.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/873/F3.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/873/F3.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/873/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/873/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461684" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-22" class="first-child">Examples of novel transcript isoforms identified by sequence capture. (A) Testis singleton GN8820S02HWWQT matches to gene CG33218 and contains two 5′ novel exons, which are about 35 kb away from the first annotated 5′ exon. (B) Testis isotig04144 matches to gene CG13527 and contains a 3′ novel exon. (C) Testis isotig02449 matches to gene <em>dgt1</em> and contains a novel internal exon. (D) Testis isotig00435 matches to gene <em>trol</em> and is missing several internal exons. (E) Testis singleton GN8820S02GZB4E fuses together exons of genes <em>Obp8a</em> and CG15369. (F) Ovary isotig00056 matches to gene CG14073 and contains a novel 5′ exon. P-element Dmel\P{GSV6}GS14564 is inserted in the middle of this novel exon.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Similarly, we identified 2084 5′ gene extensions that extend the annotated gene in the 5′ direction without interruption by an intron (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS2.xlsx">Table S2</a>); 58 3′ novel exons (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS3.xlsx">Table S3</a>); 891 3′ gene extensions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS4.xlsx">Table S4</a>); 279 novel internal exons (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and D</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS5.xlsx">Table S5</a>), and 1010 internal gene extensions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS6.xlsx">Table S6</a>). We also identified 10 gene fusions, each of which connects genes that were previously annotated separately (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS7.xlsx">Table S7</a>). Finally, we mapped 27 P-elements (the lines of which are available at the Bloomington, Kyoto, or Harvard Medical School stock centers) to novel 5′ exons or distal 5′ extensions, thus mapping previously unmatched mutations to their respective genes (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3F</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS8.xlsx">Table S8</a>).</p></div><div id="sec-12" class="subsection"><h3>Capture of annotated germline-specific transcript isoforms</h3><p id="p-24">Alternative splicing contributes to sexual differentiation in flies (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baker 1989</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> 2010</a>), and several studies have identified germline-specific isoforms (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> 2010</a>; <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Graveley <em>et al.</em> 2011</a>). Thus, we wanted to verify that our methodology was able to identify known germline-specific isoforms. One such example is represented by the <em><a href="http://flybase.org/reports/FBgn0032341.html">Reps</a></em> gene, which contains one highly conserved exon included in males but skipped in females, consistent with previously characterized isoforms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS1.pdf">Figure S1</a>) (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Graveley <em>et al.</em> 2011</a>). Along the same lines, several genes were identified that expressed novel isoforms in both ovaries and testes, either represented by the same novel isoform or two different gender-specific isoforms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS9.xlsx">Table S9</a>). For example, in the category of 5′ novel exons, six genes shared the same novel exon in both ovaries and testes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS2.pdf">Figure S2</a>A), suggesting that transcription of these isoforms is controlled by common and potentially germline-specific promoters. Two genes expressed different novel 5′ exons in ovaries and testes, suggesting the use of gender-specific transcription start sites. In other cases, we observed germ cell–specific isoforms of the same gene, each gender-specific isoform utilizing a different assortment of either novel or annotated internal exons (for example, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS2.pdf">Figure S2</a>B).</p></div><div id="sec-13" class="subsection"><h3>Comparison with tiling array analysis</h3><p id="p-25">We compared the ovary capture array data with data generated from tiled genomic microarrays to determine whether our capture methodology could identify a similar expressed gene set. A tiling genomic microarray set designed by us (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Nien <em>et al.</em> 2011</a>) (50-mer probes tiled approximately every 33 bp) was hybridized with labeled cDNA generated from the polyA-purified mRNA (the same material used for the array capture protocol) and processed to generate gene expression calls using an algorithm previously developed by us (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Nien <em>et al.</em> 2011</a>) (see <em>Materials and Methods</em>). Using this strategy, we have recently shown that tiling arrays are more sensitive in identifying expressed genes than standard gene expression arrays (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Nien <em>et al.</em> 2011</a>). In addition to algorithmically calculating more statistically robust gene expression values due to increased probe numbers and devaluation of poorly performing probes, we can visualize tiling array data in a genome browser for confirmation of expression. This is particularly useful for assessing transcription of genes in which the interrogated transcript isoform differs from the annotated isoforms either due to partial misannotation of the gene or expression of a novel isoform that omits annotated exons. We previously found that calling a gene as expressed if 70% of the probes were above background was the most reliable measure for determining whether a gene was on or off (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Manak <em>et al.</em> 2006</a>) (see <em>Materials and Methods</em> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS10.xlsx">Table S10</a>), and we have used these parameters for this dataset. Collectively, 7922 genes were called as expressed for the capture array dataset, and 7981 genes were called as expressed for the tiling array dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS11.xlsx">Table S11</a>). Among the genes called for the capture array dataset, 87% were also called for the tiling array dataset. Conversely, among the genes called for the tiling array dataset, 87.7% were also called for the capture array dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS11.xlsx">Table S11</a>). Therefore, these results show a strong concordance between the datasets and indicate that the sequence capture methodology is capable of capturing a large percentage of the same genes as a tiling array, thereby demonstrating that the capture array probes are effective in binding their targets. It should be noted that deeper sequencing of our captured sample would likely have resulted in more genes being identified by CoNECT; we point out that the depth of sequencing necessary to identify <em>all</em> relevant expressed transcripts in a tissue is currently unclear.</p><p id="p-26">We next wanted to determine whether the capture array dataset showed evidence of “normalization” on a genome-wide scale such that lower expressed transcripts were more effectively called at the expense of transcripts that may have saturated the binding of exonic capture probes (see <em>Discussion</em>). In particular, we compared FPKM values from our CoNECT study to expression values generated from the tiling array data. As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS3.pdf">Figure S3</a>A, there is a clear and substantial compression in the range of transcript abundance in our CoNECT data relative to the tiling array data (which spans a far greater overall range). Lower expressed genes are represented by proportionally more reads than higher expressed genes, thereby allowing for more effective interrogation and/or discovery of lower expressed genes while avoiding repeated resequencing of highly expressed genes.</p></div><div id="sec-14" class="subsection"><h3>Enrichment of capture-targeted transcripts</h3><p id="p-27">We wanted to determine whether our strategy could effectively enrich for targeted transcripts at the expense of transcripts that were not represented in the capture array design. We thus identified a duplicated gene called <em><a href="http://flybase.org/reports/FBgn0051865.html">ada1</a></em> (<em>ada1-1</em>, FBgn0051865; <em>ada1-2</em>, FBgn0051866) that showed strong expression on the tiling array but was not interrogated by any probes on the capture array. As the genes are 99% identical at the mRNA sequence level and both are included on the tiling array, the tiling array–based gene expression levels for each can be added together to generate an overall expression level for this gene duplication (as hybridization of labeled cDNA to one gene or the other would be stochastic). To demonstrate that reads from this gene were preferentially excluded from targeted genes expressed at similar levels, we looked at the number of reads generated by 454 sequencing for both <em>ada1-1</em>/<em>ada1-2</em> and the 10 most closely expressed genes determined via the tiling array analysis, as well as their FPKM values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS12.xlsx">Table S12</a>; see <em>Materials and Methods</em>). In total, 822 reads corresponded to these 10 genes (with an average FPKM value of 47.84/gene), whereas no reads corresponded to <em>ada1</em> (with an FPKM value of 0). The absence of reads for <em>ada1</em> is strongly significant based on expectations of random distribution among all 11 transcripts (binomial test; <em>P</em> &lt; 1 × 10<sup>−9</sup>) once expectations are normalized by the relative length of all the 11 transcripts. Similar results were obtained when we used the median number of reads for the 10 transcripts interrogated by the capture array (<em>P</em> &lt; 1 × 10<sup>−9</sup>), which takes into account possible introduced variance in the capturing efficacy. In conclusion, CoNECT can efficiently enrich for specific transcripts at the expense of undesired ones, thus providing more sequencing power focused specifically on transcripts of interest.</p></div><div id="sec-15" class="subsection"><h3>Capture-specific transcriptome</h3><p id="p-28">We next compared the ovary capture array data with the tiled microarray expression data to determine whether we could identify mRNA transcripts that were undetectable via microarrays due to either (1) low-level expression in the ovary or (2) expression in a small minority of the cells present in ovary tissue (which would thus be strongly underrepresented in the overall transcript pool), but which were detectable via sequence capture due to selective enrichment of those isoforms. Nine hundred seventy-four genes were called as expressed via array capture but not called as expressed using the tiling array and 70% positive probe threshold (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS4.pdf">Figure S4</a> for two examples). For genes represented on the tiling array that did not have a single probe passing our background threshold after cDNA hybridization and processing (the most severe restriction we could impose), we were still able to identify 18 separate gene transcript sequences that were present in the array capture data and represented by at least two exons (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS11.xlsx">Table S11</a>). Although we identified many more sequenced regions overlapping exons in this dataset, we only considered multi-exon transcripts to be evidence of real transcription, as single exon reads could in theory be produced by genomic DNA contamination. Additionally, it should be noted that after analysis of hybridized arrays, often a subset of the tiling array probes interrogating a gene pass the threshold for being called as expressed, even if the gene is not actually expressed; this is due to the recognized limitation of array technology in which probes can experience spurious cross-hybridization for a subset of the interrogating probes. Thus, the 70% threshold for tiling array gene data are a very important parameter. Nonetheless, these data demonstrate that our capture methodology can identify transcripts below the sensitivity of the tiling array.</p><p id="p-29">One thousand thirty-three genes were called as expressed on the tiling array but were not represented by any sequencing reads from the array capture. These data suggest that the sequencing was not deep enough to capture all the relevant expressed transcripts in the ovary. Indeed, out of 25.9 µg of postcapture amplified cDNA, only 500 ng was used for sequencing (approximately 1/50 of the material), covering a total of nine lanes of the 454 sequencer (9/16 PTP). It should be emphasized that deeper sequencing of our captured sample would not only reveal transcripts called by the tiling array yet missed by the sequencing depth we achieved with this study but also would likely identify additional lower-abundance transcripts not called by the tiling array, in addition to filling out any “holes” in the transcripts that were identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-30">We next wanted to determine whether any of the transcripts enriched on the capture array but not represented in the tiling array gene expression list were potentially involved in germ cell development. Indeed, <em>hu-li tai shao</em> (<em><a href="http://flybase.org/reports/FBgn0263391.html">hts</a></em>, FlyBase ID: FBgn0263391), a gene encoding an integral membrane protein required for ring canal formation during oogenesis, was identified exclusively by the capture array. Female flies harboring mutations in this gene produce egg chambers that contain less than 15 nurse cells and lack an oocyte (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yue and Spradling 1992</a>). Another capture array–specific transcript is represented by the <em><a href="http://flybase.org/reports/FBgn0004581.html">benign gonial cell neoplasm</a></em> gene (<em><a href="http://flybase.org/reports/FBgn0004581.html">bgcn</a></em>, FlyBase ID: FBgn0004581), which has been shown to be involved in germline stem cell (GSC) maintenance (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2009b</a>). Mutations in <em><a href="http://flybase.org/reports/FBgn0004581.html">bgcn</a></em> lead to an overproliferation of gonial cells at the expense of gametocyte differentiation (<a href="http://flybase.org/reports/FBgn0004581.html">http://flybase.org/reports/FBgn0004581.html</a>). These mutations produce a distinctive “tumorous” or hyperplastic stem cell phenotype caused by symmetric GSC divisions that produce more GSCs (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lavoie <em>et al.</em> 1999</a>). <em><a href="http://flybase.org/reports/FBgn0004581.html">bgcn</a></em> transcripts are abundant in males but very rare in females (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ohlstein <em>et al.</em> 2000</a>) (transcripts are detected in 5–8 cells at the tip of the germarium); however, we were still able to detect this gene in ovaries. Finally, both <em><a href="http://flybase.org/reports/FBgn0028525.html">C(2)M</a></em> and <em><a href="http://flybase.org/reports/FBgn0038612.html">corona</a></em> (FlyBase IDs FBgn0028525 and FBgn0038612, respectively), which encode synaptonemal complex components necessary for meiotic recombination, were also capture array specific. In wild-type ovaries, <em><a href="http://flybase.org/reports/FBgn0028525.html">C(2)M</a></em> and <em>corona</em> are only expressed within a subset of nuclei in regions 2A and 2B of the germarium and within the oocyte nucleus in region 3 and early egg chambers within the vitellarium (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Page <em>et al.</em> 2008</a>), which are less than 20 cells in each ovariole (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Takeo <em>et al.</em> 2011</a>). Collectively, these results underscore the power of CoNECT to enrich for transcripts expressed in a minority of cells within a tissue, even when the overall abundance of the transcripts is low and undetectable via gene expression microarray.</p><p id="p-31">The recent study by <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> (2010)</a> allows us to directly compare traditional RNA-seq methodologies with CoNECT, because they performed an analysis on poly-A RNA isolated from ovaries as well, using the Illumina/Solexa genome analyzer to perform the RNA-seq. Approximately 30 million reads were generated compared with the approximately 550,000 reads generated in our study. However, even though we generated 55 times fewer reads, 421/974 capture array–specific genes (as determined for our tiling array dataset comparison) were not identified as expressed by the traditional RNA-seq (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS11.xlsx">Table S11</a>).</p><p id="p-32">We also determined which genes called as expressed by CoNECT were also called as expressed by <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> (2010)</a> (6700/7922, or 84.57%), as well as which genes called as expressed by <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> (2010)</a> were also called as expressed by CoNECT (6700/8434, or 79.44%). These data demonstrate that there is a strong concordance between the datasets and that the sequence capture methodology is capable of capturing a large percentage of the same genes as traditional RNA-seq.</p><p id="p-33">Finally, we compared the genes called as expressed by both CoNECT and <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> (2010)</a>, and we identified over 1000 genes (1222 in total) that were exclusively called by CoNECT compared with the standard RNA-seq protocol (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS11.xlsx">Table S11</a>). Although we recognize that there may have been some variability in both biological factors (strain differences, age, temperature, food, etc.) and technical factors (ovary dissection, RNA purification, and cDNA synthesis) between the two studies, the data nonetheless demonstrate that CoNECT is able to identify additional transcribed genes that were not identified using a more traditional RNA-seq approach, with much fewer overall sequencing reads.</p></div><div id="sec-16" class="subsection"><h3>Identification of a putative transcriptional signature</h3><p id="p-34">To determine which functional gene classes were present or statistically enriched in the capture-specific gene list, we performed functional annotation clustering gene ontology (GO) analysis utilizing the David Bioinformatics Resources (Huang da <em>et al.</em> 2009). Several gene classes (42 clusters) were identified that had highly significant enrichment <em>P</em>-value scores (less than 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS13.xlsx">Table S13</a>), as well as a significant number of classes with higher <em>P</em>-values but nonetheless informative regarding pathway analysis. For example, only a small number of neurons (two pairs of nerves) innervate the ovary, specifically the peritoneal sheath and lateral oviduct (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Middleton <em>et al.</em> 2006</a>), and yet CoNECT was able to identify a number of transcripts that carry out important roles in neurons (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS14.xlsx">Table S14</a>). Some fell into statistically significant neuronal classes (12 genes), whereas others fell into nonsignificant classes (51 genes). Additionally, several genes were identified on the capture-specific list that were expressed in neurons but did not fall into any GO classes (13 genes, termed “additional neuronal genes”; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS14.xlsx">Table S14</a>). Although we are uncertain how many of these genes are specific to this cell type, at least one of them (<em><a href="http://flybase.org/reports/FBgn0260964.html">Vmat</a></em>, the <em>Drosophila</em> vesicular monoamine transporter) is expressed in all of the ovary-innervating neurons and in none of the other cell types in the ovary (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Middleton <em>et al.</em> 2006</a>; Christopher Elliott, personal communication), providing definitive proof that a transcript from this specific cell-type could be identified by CoNECT. Other identified genes include those that play primary, if not exclusive, roles in neuronal cells (<em>e.g.</em>, <em><a href="http://flybase.org/reports/FBgn0029123.html">SoxNeuro</a></em>, <em><a href="http://flybase.org/reports/FBgn0016032.html">late bloomer</a></em>, and <em><a href="http://flybase.org/reports/FBgn0030600.html">highwire</a></em>). For example, antibody staining of the synaptic growth-controlling protein <em><a href="http://flybase.org/reports/FBgn0030600.html">highwire</a></em> in embryos and larvae showed exclusive expression in neurons, resolving into expression in the presynaptic terminals, including those at neuromuscular junctions (NMJ) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wan <em>et al.</em> 2000</a>). We note that the neurons that innervate the ovary form NMJs with ovary musculature. Moreover, <em><a href="http://flybase.org/reports/FBgn0016032.html">late bloomer</a></em> (which encodes a neural tetraspanin involved in synapse formation) is expressed almost exclusively in motor neurons during embryogenesis, with the remaining expression observed in sensory neurons (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kopczynski <em>et al.</em> 1996</a>). Finally, <em><a href="http://flybase.org/reports/FBgn0029123.html">SoxNeuro</a></em>, a gene encoding an HMG-box transcription factor, is not only expressed in neurons but also appears to be required for maintenance of neural tissue (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Buescher <em>et al.</em> 2002</a>).</p><p id="p-35">We reasoned that (out of all the other non-neuronal ovary cell types) the most likely cell type to express these “neuronal signature” genes would be the oocyte, given that maternal RNAs are deposited there by nurse cells to support early embryonic development, including the early stages of neurogenesis. We thus accessed expression data from 0–2 hr embryos (which would consist primarily of maternal message) from our previous study (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Manak <em>et al.</em> 2006</a>). Significantly, only 2 out of the 76 genes in the “neuronal signature” were called as expressed in these embryos (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS14.xlsx">Table S14</a>).</p><p id="p-36">To address the possibility that some of the genes called by CoNECT might be associated with ovary tissue that was contaminated with other tissues during dissection (an unlikely, but possible, scenario given the relative ease with which ovaries can be dissected from adult females), we looked at whether genes exclusively expressed in a neighboring tissue were called by CoNECT. We chose to look at five genes previously shown to be hindgut-specific [<a href="http://flybase.org/reports/FBgn0034553.html">CG9993</a>, FBgn0034553; <a href="http://flybase.org/reports/FBgn0033592.html">CG13215</a>, FBgn0033592; <a href="http://flybase.org/reports/FBgn0083945.html">CG13129</a>, FBgn0083945; <a href="http://flybase.org/reports/FBgn0033207.html">CG12826</a>, FBgn0033207; <a href="http://flybase.org/reports/FBgn0039196.html">CG17781</a>, FBgn0039196; (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chintapalli <em>et al.</em> 2007</a>)]. None of these genes was represented by reads generated from CoNECT, suggesting that the ovary preparations were pure. Nevertheless, we cannot entirely rule out the possibility that some reads in our dataset came from small amounts of contaminating tissues. Collectively, these data suggest that CoNECT is able to identify a transcriptional signature specific to the neurons that innervate the ovary, although it is possible (and perhaps probable) that some genes within the “neuronal signature” are expressed in other cell types in the ovary.</p><p id="p-37">We directly compared the CoNECT dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS16.xlsx">Table S16</a>) with the RNA-seq dataset by <a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> (2010)</a> genome-wide and generated a scatter plot of FPKM values produced by the two methodologies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS3.pdf">Figure S3</a>B). Similar to what was seen for the tiling array dataset comparison, we observed a “normalization” effect, in which a quadratic fit was better than a linear one (<em>P</em> = 0.002). Thus, lower expressed genes are represented by proportionally more reads than higher expressed genes for CoNECT compared with traditional RNA-seq data, with a statistically significant tendency for CoNECT to avoid highly expressed genes.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-38">Using an array-based sequence capture and enrichment strategy followed by long-read sequencing, we have identified large numbers of novel exons, splice sites, and transcription start sites in both testes and ovaries of the fruit fly. We also provide a list of available fly lines containing P-element mutations that target novel first exons we have identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS8.xlsx">Table S8</a>), which should be useful to fly researchers studying these genes. This study provides proof of principle for more directed future studies that seek to enrich more specific types of transcripts. For example, the capture and sequencing of transcriptionally complex splice isoforms of a gene, each expressed in small subsets of cells, would be an obvious extension of our study. <em>Drosophila</em> <em><a href="http://flybase.org/reports/FBgn0033159.html">Dscam</a></em> can potentially encode over 38,000 isoforms, and these isoforms are expressed in predominantly one tissue type (namely, the nervous system). Without ultra-deep sequencing of nervous system tissue, the true diversity of transcript isoforms could not be elucidated. By using a sequence capture array that only targets <em><a href="http://flybase.org/reports/FBgn0033159.html">Dscam</a></em>, a more comprehensive cataloging of the expressed isoforms can be undertaken. Interestingly, we identified two partial transcripts in testes that aligned to <em><a href="http://flybase.org/reports/FBgn0033159.html">Dscam</a></em> and represent novel isoforms that had not been annotated before, indicating that these are potential testes-specific <em><a href="http://flybase.org/reports/FBgn0033159.html">Dscam</a></em> transcripts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/TableS15.xlsx">Table S15</a>) that may be associated with the neurons that innervate the musculature of the testes.</p><p id="p-39">We chose the 454 platform for our high-throughput sequencing over other available platforms for several reasons. First, the 454 system has been shown to work well with longer input sequence fragments. Because the cDNA nebulization step produced products with an average length of 700 bp, we were confident that the platform would generate high-quality sequences. Second, and most importantly, we wanted to use a system capable of long reads so that we could easily and unambiguously annotate any novel exon structures, including their connectivities with annotated (or other unannotated) exons. We recognize that the Illumina paired-end technology has been substantially developed over the last couple years, and this type of platform could be used for further studies in conjunction with array capture.</p><p id="p-40">This study demonstrates that the CoNECT methodology can identify genes exclusively expressed in a small subset of the overall cell population of a given tissue. For example, genes expressed in two pairs of nerves in the <em>Drosophila</em> ovaries (<em>e.g.</em>, <em><a href="http://flybase.org/reports/FBgn0260964.html">Vmat</a></em>) were identified with CoNECT, even though the neurons make up less than 0.01% of the total cell number in the ovary [over 100,000 nonneuronal cells, including stem, follicle, and nurse cells, are present in a developing ovary (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Spradling 1993</a>)]. Along the same lines, both <em><a href="http://flybase.org/reports/FBgn0028525.html">C(2)M</a></em> and <em><a href="http://flybase.org/reports/FBgn0038612.html">corona</a></em> (two other genes identified by CoNECT) encode important meiosis-specific synaptonemal complex proteins and are only expressed in about 20 cells per ovariole. It is important to note that these genes and other classes of genes represented by lower overall sequencing read numbers are not necessarily genes expressed at low levels; their transcripts, even if expressed highly in a few cells, may simply be underrepresented in the overall transcript pool due to the largely nonneuronal cell population. Therefore, many of the genes identified by CoNECT that are expressed in smaller cellular populations of a tissue may in fact be important primary regulators of cellular function. By enrichment strategies followed by gene ontology analysis as outlined above, specific gene classes can be identified, even without prior knowledge of which cell types are present in the tissue of interest. Importantly, without enrichment strategies such as CoNECT, transcriptomics of minority cell types would be difficult. This methodology, alongside traditional RNA-seq (<a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Gan <em>et al.</em> 2010</a>; <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Graveley <em>et al.</em> 2011</a>) and locus-directed capture strategies, such as those detailed in <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Mercer <em>et al.</em> (2012)</a>, provides a means to more faithfully annotate biologically important transcripts expressed in all layers of a tissue of interest across the entire genome. Several exome capture platforms are already commercially available for a variety of organisms, and these can immediately be employed to probe the desired transcriptome of interest.</p><p id="p-41">It should be emphasized that we chose a whole-genome exome array for these initial studies to provide the proof of principle that our methodology works and to maximize our chances of identifying the largest number of novel exons and splicing events of the <em>Drosophila</em> germline, a tissue that has been noted to have a large diversity of transcript isoforms. Nonetheless, future studies will target specific gene transcripts, thus allowing for greater sequencing depth of the relevant transcripts while eliminating any unwanted transcripts from the sequencing step. Indeed, we have shown that the exome array can specifically enrich for genes interrogated on the array at the expense of genes not included in the array design (<em>e.g.</em>, <em><a href="http://flybase.org/reports/FBgn0051866.html">ada1</a></em>).</p><p id="p-42">Why have we been able to identify so many additional genes expressed in the ovary? Several possibilities exist. First, we have shown that CoNECT can specifically enrich for lower-expressed transcripts at the expense of higher-expressed transcripts, which is most obvious for the CoNECT/tiling array expression comparison (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS3.pdf">Figure S3</a>A) but is also seen with the CoNECT/traditional RNA-seq comparison (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1/FigureS3.pdf">Figure S3</a>B). This is likely in part due to the effect of some highly expressed transcript cDNAs saturating the available probes on the capture array such that no additional transcripts can be captured, thereby “enriching” for lower-expressed transcripts to a greater extent than higher-expressed transcripts. Second, the nature of the hybridization strategy of the capture protocol may facilitate enrichment of lower-level transcripts even in relation to transcripts that do not saturate their target exonic probes. For example, most transcripts of a gene expressed at moderate-to-high levels will efficiently hybridize to their capture probes, but the kinetics will change such that the hybridization becomes less efficient as time goes on because fewer target probes are available for hybridization. However, this is not the case for lower-expressed transcripts in which there is always an abundance of capture probes available for hybridization. Thus, over time, hybridization of lower-level transcripts would be favored over transcripts from more highly expressed genes. Third, given that highly repeated rRNA genes were omitted from the capture array design, we were able to more comprehensively interrogate the transcripts of interest, which by definition, would allow for better interrogation of low-level transcripts. Whatever the case, CoNECT was able to identify a great many additional genes as being expressed compared with either the traditional RNA-seq dataset (with 55-fold fewer reads) or with the tiling array gene expression dataset.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank Giovanni Bosco for helpful comments on the manuscript. H.A.H., M.J.R., T.A.R., and T.J.A. are employed by Roche NimbleGen Inc., and Roche NimbleGen capture arrays and reagents were used in the study.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003194/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Raw sequence data from this article have been deposited with the NCBI Sequence Read Archive under accession no. SRA050707.2.</p></li><li class="fn" id="fn-3"><p id="p-3">Ovary and testis isotigs over 200 bp have been deposited with the GenBank Transcriptome Shotgun Assembly Sequence Database as bioproject no. PRJNA89451 (accession nos. JV208106–JV230865).</p></li><li class="fn" id="fn-4"><p id="p-44"><em>Communicating editor: T. R. Hughes</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-04-30"><span class="received-label">Received </span>April 30, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-05-28"><span class="accepted-label">Accepted </span>May 28, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Hong <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.1" data-doi="10.1186/gb-2010-11-6-r62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bainbridge</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodesch</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole exome capture in solution with 3 Gbp of data</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBainbridge%26rft.auinit1%253DM.%2BN.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253DR62%26rft.epage%253DR62%26rft.atitle%253DWhole%2Bexome%2Bcapture%2Bin%2Bsolution%2Bwith%2B3%2BGbp%2Bof%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-6-r62%26rft_id%253Dinfo%253Apmid%252F20565776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-6-r62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20565776&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.2" data-doi="10.1038/340521a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Sex in flies: the splice of life</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">340</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBaker%26rft.auinit1%253DB.%2BS.%26rft.volume%253D340%26rft.issue%253D6234%26rft.spage%253D521%26rft.epage%253D524%26rft.atitle%253DSex%2Bin%2Bflies%253A%2Bthe%2Bsplice%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F340521a0%26rft_id%253Dinfo%253Apmid%252F2505080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/340521a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2505080&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AL22700047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.8.873.3" data-doi="10.1038/nrg3031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bamshad</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabor</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emond</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exome sequencing as a tool for Mendelian disease gene discovery</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBamshad%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D745%26rft.epage%253D755%26rft.atitle%253DExome%2Bsequencing%2Bas%2Ba%2Btool%2Bfor%2BMendelian%2Bdisease%2Bgene%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3031%26rft_id%253Dinfo%253Apmid%252F21946919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21946919&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hing</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Formation of neuroblasts in the embryonic central nervous system of Drosophila melanogaster is controlled by SoxNeuro</span>. <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">4193</span>–<span class="cit-lpage">4203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.volume%253D129%26rft.spage%253D4193%26rft_id%253Dinfo%253Apmid%252F12183372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12183372&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178528000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronmiller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Finishing a whole-genome shotgun: release 3 of the Drosophila melanogaster euchromatic genome sequence</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-comment">RESEARCH0079</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.6" data-doi="10.1101/gr.112961.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherbas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willingham</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The transcriptional diversity of 25 Drosophila cell lines</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.112961.110%26rft_id%253Dinfo%253Apmid%252F21177962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMzAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.7" data-doi="10.1038/ng2049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chintapalli</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dow</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using FlyAtlas to identify better Drosophila melanogaster models of human disease</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChintapalli%26rft.auinit1%253DV.%2BR.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D715%26rft.epage%253D720%26rft.atitle%253DUsing%2BFlyAtlas%2Bto%2Bidentify%2Bbetter%2BDrosophila%2Bmelanogaster%2Bmodels%2Bof%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2049%26rft_id%253Dinfo%253Apmid%252F17534367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17534367&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246859100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angelini</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Feis</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciccodicola</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Uncovering the complexity of transcriptomes with RNA-Seq</span>. <abbr class="cit-jnl-abbrev">J. Biomed. Biotechnol.</abbr> <span class="cit-vol">2010</span>: <span class="cit-fpage">853916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiomed%2BBiotechnol%26rft.aulast%253DCosta%26rft.auinit1%253DV.%26rft.volume%253D2010%26rft.spage%253D853916%26rft.epage%253D853916%26rft.atitle%253DUncovering%2Bthe%2Bcomplexity%2Bof%2Btranscriptomes%2Bwith%2BRNA-Seq.%26rft_id%253Dinfo%253Apmid%252F20625424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20625424&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.9" data-doi="10.1186/gb-2011-12-9-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fairfield</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corrigan</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutation discovery in mice by whole exome sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DFairfield%26rft.auinit1%253DH.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DMutation%2Bdiscovery%2Bin%2Bmice%2Bby%2Bwhole%2Bexome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-9-r86%26rft_id%253Dinfo%253Apmid%252F21917142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-9-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21917142&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.10" data-doi="10.1111/j.1365-313X.2010.04196.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerhardt</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Repeat subtraction-mediated sequence capture from a complex genome</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">898</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFu%26rft.auinit1%253DY.%26rft.volume%253D62%26rft.issue%253D5%26rft.spage%253D898%26rft.epage%253D909%26rft.atitle%253DRepeat%2Bsubtraction-mediated%2Bsequence%2Bcapture%2Bfrom%2Ba%2Bcomplex%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04196.x%26rft_id%253Dinfo%253Apmid%252F20230488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04196.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20230488&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278080300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.11" data-doi="10.1038/cr.2010.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chepelev</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarayrah</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamic regulation of alternative splicing and chromatin structure in Drosophila gonads revealed by RNA-seq</span>. <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DGan%26rft.auinit1%253DQ.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D763%26rft.epage%253D783%26rft.atitle%253DDynamic%2Bregulation%2Bof%2Balternative%2Bsplicing%2Band%2Bchromatin%2Bstructure%2Bin%2BDrosophila%2Bgonads%2Brevealed%2Bby%2BRNA-seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2010.64%26rft_id%253Dinfo%253Apmid%252F20440302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2010.64&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20440302&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280079500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.12" data-doi="10.1038/nature09715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The developmental transcriptome of Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGraveley%26rft.auinit1%253DB.%2BR.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D473%26rft.epage%253D479%26rft.atitle%253DThe%2Bdevelopmental%2Btranscriptome%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09715%26rft_id%253Dinfo%253Apmid%252F21179090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179090&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.13" data-doi="10.1186/gb-2011-12-9-r90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grati</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingham</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exome sequencing identifies a missense mutation in Isl1 associated with low penetrance otitis media in dearisch mice</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHilton%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253DR90%26rft.epage%253DR90%26rft.atitle%253DExome%2Bsequencing%2Bidentifies%2Ba%2Bmissense%2Bmutation%2Bin%2BIsl1%2Bassociated%2Bwith%2Blow%2Bpenetrance%2Botitis%2Bmedia%2Bin%2Bdearisch%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-9-r90%26rft_id%253Dinfo%253Apmid%252F21936904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-9-r90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21936904&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.8.873.14" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang da</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.15" data-doi="10.1016/j.ydbio.2005.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spradling</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The expression profile of purified Drosophila germline stem cells</span>. <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DKai%26rft.auinit1%253DT.%26rft.volume%253D283%26rft.issue%253D2%26rft.spage%253D486%26rft.epage%253D502%26rft.atitle%253DThe%2Bexpression%2Bprofile%2Bof%2Bpurified%2BDrosophila%2Bgermline%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2005.04.018%26rft_id%253Dinfo%253Apmid%252F15927177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2005.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15927177&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230683800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.16" data-doi="10.1126/science.271.5257.1867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kopczynski</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A neural tetraspanin, encoded by late bloomer, that facilitates synapse formation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">1867</span>–<span class="cit-lpage">1870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKopczynski%26rft.auinit1%253DC.%2BC.%26rft.volume%253D271%26rft.issue%253D5257%26rft.spage%253D1867%26rft.epage%253D1870%26rft.atitle%253DA%2BNeural%2BTetraspanin%252C%2BEncoded%2Bby%2Blate%2Bbloomer%252C%2BThat%2BFacilitates%2BSynapse%2BFormation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.271.5257.1867%26rft_id%253Dinfo%253Apmid%252F8596956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzEvNTI1Ny8xODY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.17" data-doi="10.1006/dbio.1999.9346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavoie</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohlstein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKearin</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Localization and function of Bam protein require the benign gonial cell neoplasm gene product</span>. <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DLavoie%26rft.auinit1%253DC.%2BA.%26rft.volume%253D212%26rft.issue%253D2%26rft.spage%253D405%26rft.epage%253D413%26rft.atitle%253DLocalization%2Band%2Bfunction%2Bof%2BBam%2Bprotein%2Brequire%2Bthe%2Bbenign%2Bgonial%2Bcell%2Bneoplasm%2Bgene%2Bproduct.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.1999.9346%26rft_id%253Dinfo%253Apmid%252F10433830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.1999.9346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10433830&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082123900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.18" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.19" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.20" data-doi="10.1073/pnas.0901452106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minor</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKearin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maines</span>  <span class="cit-name-given-names">J. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Bam and Bgcn antagonize Nanos-dependent germ-line stem cell maintenance</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9304</span>–<span class="cit-lpage">9309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0901452106%26rft_id%253Dinfo%253Apmid%252F19470484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIzLzkzMDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.21" data-doi="10.1186/1741-7007-9-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Microarrays, deep sequencing and the true measure of the transcriptome</span>. <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DMalone%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D9%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DMicroarrays%252C%2Bdeep%2Bsequencing%2Band%2Bthe%2Btrue%2Bmeasure%2Bof%2Bthe%2Btranscriptome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-9-34%26rft_id%253Dinfo%253Apmid%252F21627854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-9-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21627854&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.22" data-doi="10.1038/ng1875"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manak</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dike</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sementchenko</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapranov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biemar</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Biological function of unannotated transcription during the early development of Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DManak%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D38%26rft.issue%253D10%26rft.spage%253D1151%26rft.epage%253D1158%26rft.atitle%253DBiological%2Bfunction%2Bof%2Bunannotated%2Btranscription%2Bduring%2Bthe%2Bearly%2Bdevelopment%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1875%26rft_id%253Dinfo%253Apmid%252F16951679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1875&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16951679&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241251100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.23" data-doi="10.1038/nbt.2024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerhardt</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinger</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Targeted RNA sequencing reveals the deep complexity of the human transcriptome</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMercer%26rft.auinit1%253DT.%2BR.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D99%26rft.epage%253D104%26rft.atitle%253DTargeted%2BRNA%2Bsequencing%2Breveals%2Bthe%2Bdeep%2Bcomplexity%2Bof%2Bthe%2Bhuman%2Btranscriptome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2024%26rft_id%253Dinfo%253Apmid%252F22081020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22081020&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.24" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing technologies - the next generation</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMetzker%26rft.auinit1%253DM.%2BL.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D46%26rft.atitle%253DSequencing%2Btechnologies%2B-%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.25" data-doi="10.1186/1741-7007-4-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Middleton</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nongthomba</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sparrow</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Neuromuscular organization and aminergic modulation of contractions in the Drosophila ovary</span>. <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DMiddleton%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.spage%253D17%26rft.epage%253D17%26rft.atitle%253DNeuromuscular%2Borganization%2Band%2Baminergic%2Bmodulation%2Bof%2Bcontractions%2Bin%2Bthe%2BDrosophila%2Bovary.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-4-17%26rft_id%253Dinfo%253Apmid%252F16768790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-4-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16768790&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.26" data-doi="10.1371/journal.pgen.1002339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nien</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butcher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Temporal coordination of gene networks by Zelda in the early Drosophila embryo</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNien%26rft.auinit1%253DC.%2BY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002339%26rft.epage%253De1002339%26rft.atitle%253DTemporal%2Bcoordination%2Bof%2Bgene%2Bnetworks%2Bby%2BZelda%2Bin%2Bthe%2Bearly%2BDrosophila%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002339%26rft_id%253Dinfo%253Apmid%252F22028675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22028675&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohlstein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavoie</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vef</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gateff</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKearin</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Drosophila cystoblast differentiation factor, benign gonial cell neoplasm, is related to DExH-box proteins and interacts genetically with bag-of-marbles</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1809</span>–<span class="cit-lpage">1819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D1809%26rft_id%253Dinfo%253Apmid%252F10924476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10924476&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.28" data-doi="10.1371/journal.pgen.1000194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khetani</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lake</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffress</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Corona is required for higher-order assembly of transverse filaments into full-length synaptonemal complex in Drosophila oocytes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPage%26rft.auinit1%253DS.%2BL.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De1000194%26rft.epage%253De1000194%26rft.atitle%253DCorona%2Bis%2Brequired%2Bfor%2Bhigher-order%2Bassembly%2Bof%2Btransverse%2Bfilaments%2Binto%2Bfull-length%2Bsynaptonemal%2Bcomplex%2Bin%2BDrosophila%2Boocytes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000194%26rft_id%253Dinfo%253Apmid%252F18802461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18802461&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.29" data-doi="10.1371/journal.pgen.1002325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pierson</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinelli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherukuri</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-exome sequencing identifies homozygous AFG3L2 mutations in a spastic ataxia-neuropathy syndrome linked to mitochondrial m-AAA proteases</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPierson%26rft.auinit1%253DT.%2BM.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002325%26rft.epage%253De1002325%26rft.atitle%253DWhole-exome%2Bsequencing%2Bidentifies%2Bhomozygous%2BAFG3L2%2Bmutations%2Bin%2Ba%2Bspastic%2Bataxia-neuropathy%2Bsyndrome%2Blinked%2Bto%2Bmitochondrial%2Bm-AAA%2Bproteases.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002325%26rft_id%253Dinfo%253Apmid%252F22022284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022284&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-2.8.873.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spradling</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-source">The Development of Drosophila melanogaster.</span> <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.31" data-doi="10.1016/j.cub.2011.09.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lake</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morais-de-Sa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunkel</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synaptonemal complex-dependent centromeric clustering and the initiation of synapsis in Drosophila oocytes</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1845</span>–<span class="cit-lpage">1851</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DTakeo%26rft.auinit1%253DS.%26rft.volume%253D21%26rft.issue%253D21%26rft.spage%253D1845%26rft.epage%253D1851%26rft.atitle%253DSynaptonemal%2Bcomplex-dependent%2Bcentromeric%2Bclustering%2Band%2Bthe%2Binitiation%2Bof%2Bsynapsis%2Bin%2BDrosophila%2Boocytes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.09.044%26rft_id%253Dinfo%253Apmid%252F22036182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.09.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22036182&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.32" data-doi="10.1016/S0896-6273(00)81166-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiAntonio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fetter</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strauss</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Highwire regulates synaptic growth in Drosophila</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D26%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0896-6273%252800%252981166-6%26rft_id%253Dinfo%253Apmid%252F10839352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0896-6273(00)81166-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10839352&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087296500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.33" data-doi="10.1038/nprot.2006.90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoskins</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-throughput plasmid cDNA library screening</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">624</span>–<span class="cit-lpage">632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DWan%26rft.auinit1%253DK.%2BH.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D624%26rft.epage%253D632%26rft.atitle%253DHigh-throughput%2Bplasmid%2BcDNA%2Blibrary%2Bscreening.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2006.90%26rft_id%253Dinfo%253Apmid%252F17406289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2006.90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17406289&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251002500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.34" data-doi="10.1038/nrg2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.35" data-doi="10.1016/j.ymeth.2009.03.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq-quantitative measurement of expression through massively parallel RNA-sequencing</span>. <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DWilhelm%26rft.auinit1%253DB.%2BT.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D257%26rft.atitle%253DRNA-Seq-quantitative%2Bmeasurement%2Bof%2Bexpression%2Bthrough%2Bmassively%2Bparallel%2BRNA-sequencing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2009.03.016%26rft_id%253Dinfo%253Apmid%252F19336255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2009.03.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19336255&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F873.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268359300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.873.36" data-doi="10.1101/gad.6.12b.2443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spradling</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">hu-li tai shao, a gene required for ring canal formation during Drosophila oogenesis, encodes a homolog of adducin</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2443</span>–<span class="cit-lpage">2454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DYue%26rft.auinit1%253DL%26rft.volume%253D6%26rft.issue%253D12b%26rft.spage%253D2443%26rft.epage%253D2454%26rft.atitle%253Dhu-li%2Btai%2Bshao%252C%2Ba%2Bgene%2Brequired%2Bfor%2Bring%2Bcanal%2Bformation%2Bduring%2BDrosophila%2Boogenesis%252C%2Bencodes%2Ba%2Bhomolog%2Bof%2Badducin.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.6.12b.2443%26rft_id%253Dinfo%253Apmid%252F1340461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjYvMTJiLzI0NDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/8/873.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/865" title="Proper Cyclin B3 Dosage Is Important for Precision of Metaphase-to-Anaphase Onset Timing in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/883" title="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/8"><span>Volume 2 Issue 8, August 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/8.cover-source.jpg?itok=etPFDPKt" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/873" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BYnS0Rt16tHsMpWKy6NvYgwImth6i_p9GrU0ZK6KwLM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/8/873" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/8/873" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/8/873" class="active" data-icon-position="" data-hide-link-title="0">Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459311" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-e9uoezxep7IecUzK1FAsrBZoGFL-2UaCE3xi9eRToPI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/873.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/8/873" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2OEFE4Kw6nyOG_KdMejb-sACJyCusIYL_BoUDz9lhoo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/8/873" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/873.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459311" id="node459311--2" data-pisa="ggg;2/8/873" data-pisa-master="ggg;2/8/873" data-apath="/ggg/2/8/873.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">X. Hong</span>, <span class="highwire-citation-author" data-delta="1">H. Doddapaneni</span>, <span class="highwire-citation-author" data-delta="2">J. M. Comeron</span>, <span class="highwire-citation-author" data-delta="3">M. J. Rodesch</span>, <span class="highwire-citation-author" data-delta="4">H. A. Halvensleben</span>, <span class="highwire-citation-author" data-delta="5">C. Y. Nien</span>, <span class="highwire-citation-author" data-delta="6">F. Bolei</span>, <span class="highwire-citation-author" data-delta="7">R. Metpally</span>, <span class="highwire-citation-author" data-delta="8">T. A. Richmond</span>, <span class="highwire-citation-author" data-delta="9">T. J. Albert</span> and <span class="highwire-citation-author" data-delta="10">J. R. Manak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">873-882;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459311--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">X. Hong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=X.%2BHong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hong%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AX.%2BHong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">H. Doddapaneni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H.%2BDoddapaneni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doddapaneni%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH.%2BDoddapaneni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">J. M. Comeron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BM.%2BComeron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comeron%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BM.%2BComeron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. J. Rodesch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BJ.%2BRodesch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodesch%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BJ.%2BRodesch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">H. A. Halvensleben </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H.%2BA.%2BHalvensleben%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halvensleben%20HA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH.%2BA.%2BHalvensleben%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">C. Y. Nien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, Center for Developmental Genetics, New York University, New York, New York 10003, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BY.%2BNien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nien%20CY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BY.%2BNien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">F. Bolei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=F.%2BBolei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolei%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AF.%2BBolei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">R. Metpally </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Translational Genomics Research Institute, Phoenix, Arizona 85004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BMetpally%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metpally%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BMetpally%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">T. A. Richmond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T.%2BA.%2BRichmond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richmond%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT.%2BA.%2BRichmond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">T. J. Albert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T.%2BJ.%2BAlbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albert%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT.%2BJ.%2BAlbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">J. R. Manak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pediatrics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BR.%2BManak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manak%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BR.%2BManak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john-manak@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">john-manak@uiowa.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/8/873" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/8/873&amp;title=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/8/873&amp;title=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/8/873&amp;title=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/8/873&amp;text=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/8/873&amp;title=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/8/873&amp;t=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/8/873&amp;title=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/8/873&amp;title=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459311/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459311" id="node459311--3" data-pisa="ggg;2/8/873" data-pisa-master="ggg;2/8/873" data-apath="/ggg/2/8/873.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">X. Hong</span>, <span class="highwire-citation-author" data-delta="1">H. Doddapaneni</span>, <span class="highwire-citation-author" data-delta="2">J. M. Comeron</span>, <span class="highwire-citation-author" data-delta="3">M. J. Rodesch</span>, <span class="highwire-citation-author" data-delta="4">H. A. Halvensleben</span>, <span class="highwire-citation-author" data-delta="5">C. Y. Nien</span>, <span class="highwire-citation-author" data-delta="6">F. Bolei</span>, <span class="highwire-citation-author" data-delta="7">R. Metpally</span>, <span class="highwire-citation-author" data-delta="8">T. A. Richmond</span>, <span class="highwire-citation-author" data-delta="9">T. J. Albert</span> and <span class="highwire-citation-author" data-delta="10">J. R. Manak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">873-882;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003194 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459311--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">X. Hong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=X.%2BHong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hong%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AX.%2BHong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">H. Doddapaneni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H.%2BDoddapaneni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doddapaneni%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH.%2BDoddapaneni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">J. M. Comeron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BM.%2BComeron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comeron%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BM.%2BComeron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. J. Rodesch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BJ.%2BRodesch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodesch%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BJ.%2BRodesch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">H. A. Halvensleben </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H.%2BA.%2BHalvensleben%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halvensleben%20HA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH.%2BA.%2BHalvensleben%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">C. Y. Nien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, Center for Developmental Genetics, New York University, New York, New York 10003, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BY.%2BNien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nien%20CY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BY.%2BNien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">F. Bolei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=F.%2BBolei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolei%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AF.%2BBolei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">R. Metpally </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Translational Genomics Research Institute, Phoenix, Arizona 85004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BMetpally%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metpally%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BMetpally%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">T. A. Richmond </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T.%2BA.%2BRichmond%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richmond%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT.%2BA.%2BRichmond%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">T. J. Albert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Roche NimbleGen, Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=T.%2BJ.%2BAlbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albert%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AT.%2BJ.%2BAlbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">J. R. Manak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Carver Center for Genomics, University of Iowa, Iowa City, Iowa 52242</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pediatrics, University of Iowa, Iowa City, Iowa 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BR.%2BManak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manak%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BR.%2BManak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john-manak@uiowa.edu" class="" data-icon-position="" data-hide-link-title="0">john-manak@uiowa.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=873&amp;rft.epage=882&amp;rft.atitle=Microarray-Based%20Capture%20of%20Novel%20Expressed%20Cell%20Type%E2%80%93Specific%20Transfrags%20%28CoNECT%29%20to%20Annotate%20Tissue-Specific%20Transcription%20in%20Drosophila%20melanogaster&amp;rft.volume=2&amp;rft.issue=8&amp;rft.date=2012-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hong%2C+X.&amp;rft.au=Doddapaneni%2C+H.&amp;rft.au=Comeron%2C+J.+M.&amp;rft.au=Rodesch%2C+M.+J.&amp;rft.au=Halvensleben%2C+H.+A.&amp;rft.au=Nien%2C+C.+Y.&amp;rft.au=Bolei%2C+F.&amp;rft.au=Metpally%2C+R.&amp;rft.au=Richmond%2C+T.+A.&amp;rft.au=Albert%2C+T.+J.&amp;rft.au=Manak%2C+J.+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459311/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459311/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459311/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459311/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459311/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459311/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459311/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459311/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459311/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459311/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459311/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459311/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/8/873" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/8/873.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/8/873.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/8/873.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/8/873.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
