<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/8/883" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/8/883.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/8/883.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/8/883.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/8/883/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/8/883.atom" />
<meta name="HW.pisa" content="ggg;2/8/883" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" />
<meta name="DC.Identifier" content="10.1534/g3.112.002527" />
<meta name="DC.Date" content="2012-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Haubold, Pfaffelhuber. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Comparative sequencing contributes critically to the functional annotation of genomes. One prerequisite for successful analysis of the increasingly abundant comparative sequencing data is the availability of efficient computational tools. We present here a strategy for comparing unaligned genomes based on a coalescent approach combined with advanced algorithms for indexing sequences. These algorithms are particularly efficient when analyzing large genomes, as their run time ideally grows only linearly with sequence length. Using this approach, we have derived and implemented a maximum-likelihood estimator of the average number of mismatches per site between two closely related sequences, π . By allowing for fluctuating coalescent times, we are able to improve a previously published alignment-free estimator of π . We show through simulation that our new estimator is fast and accurate even with moderate recombination ( ρ ≤ π ). To demonstrate its applicability to real data, we compare the unaligned genomes of Drosophila persimilis and D. pseudoobscura . In agreement with previous studies, our sliding window analysis locates the global divergence minimum between these two genomes to the pericentromeric region of chromosome 3." />
<meta name="DC.Contributor" content="Bernhard Haubold" />
<meta name="DC.Contributor" content="Peter Pfaffelhuber" />
<meta name="article:published_time" content="2012-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Comparative sequencing contributes critically to the functional annotation of genomes. One prerequisite for successful analysis of the increasingly abundant comparative sequencing data is the availability of efficient computational tools. We present here a strategy for comparing unaligned genomes based on a coalescent approach combined with advanced algorithms for indexing sequences. These algorithms are particularly efficient when analyzing large genomes, as their run time ideally grows only linearly with sequence length. Using this approach, we have derived and implemented a maximum-likelihood estimator of the average number of mismatches per site between two closely related sequences, &lt;i&gt;π&lt;/i&gt;. By allowing for fluctuating coalescent times, we are able to improve a previously published alignment-free estimator of &lt;i&gt;π&lt;/i&gt;. We show through simulation that our new estimator is fast and accurate even with moderate recombination (&lt;i&gt;ρ&lt;/i&gt; ≤ &lt;i&gt;π&lt;/i&gt;). To demonstrate its applicability to real data, we compare the unaligned genomes of &lt;i&gt;Drosophila persimilis&lt;/i&gt; and &lt;i&gt;D. pseudoobscura&lt;/i&gt;. In agreement with previous studies, our sliding window analysis locates the global divergence minimum between these two genomes to the pericentromeric region of chromosome 3.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/08/01" />
<meta name="citation_mjid" content="ggg;2/8/883" />
<meta name="citation_id" content="2/8/883" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/8/883" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/8/883.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/8/883.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/8/883.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002527" />
<meta name="citation_pmid" content="22908037" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="883" />
<meta name="citation_lastpage" content="889" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bernhard Haubold" />
<meta name="citation_author_institution" content="Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany" />
<meta name="citation_author_email" content="haubold@evolbio.mpg.de" />
<meta name="citation_author" content="Peter Pfaffelhuber" />
<meta name="citation_author_institution" content="Mathematical Stochastics, Mathematical Institute, Albert-Ludwigs University, 79085 Freiburg, Germany" />
<meta name="citation_reference" content="citation_author=M. Abouelhoda;citation_author=S. Kurtz;citation_author=E. Ohlebusch;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=R. Arratia;citation_author=L. Gordon;citation_author=M. Waterman;citation_title=An extreme value theory for sequence matching;citation_pages=971-993;citation_volume=14;citation_year=1986;citation_doi=10.1214/aos/1176350045" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_title=Population genomics: whole-genome analysis of polymorphism and divergence in Drosophila simulans;citation_author=D. Begun;citation_author=A. Holloway;citation_author=K. Stevens;citation_author=L. Hillier;citation_author=Y. Poh;citation_volume=5;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. K. Chen;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_title=Fast and flexible simulation of DNA sequence data;citation_volume=19;citation_year=2009;citation_pmid=19029539;citation_doi=10.1101/gr.083634.108" />
<meta name="citation_reference" content="citation_journal_title=J. Zoological Syst. Evol. Res.;citation_author=J.-L. Da Lage;citation_author=G. J. Kergoat;citation_author=F. Maczkowiak;citation_author=J.-F. Silvain;citation_author=M.-L. Cariou;citation_author=D. Lachaise;citation_title=A phylogeny of Drosophilidae using the Amyrel gene: questioning the Drosophila melanogaster species group boundaries;citation_pages=46-63;citation_volume=45;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Domazet-Lošo;citation_author=B. Haubold;citation_title=Efficient estimation of pairwise distances between genomes;citation_volume=25;citation_year=2009;citation_pmid=19825795;citation_doi=10.1093/bioinformatics/btp590" />
<meta name="citation_reference" content="citation_journal_title=Mobile Genet. Elements;citation_author=M. Domazet-Lošo;citation_author=B. Haubold;citation_title=Alignment-free detection of horizontal gene transfer between closely related bacterial genomes;citation_pages=230-235;citation_volume=1;citation_year=2011;citation_doi=10.4161/mge.1.3.18065" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Domazet-Lošo;citation_author=B. Haubold;citation_title=Alignment-free detection of local similarity among viral and bacterial genomes;citation_volume=27;citation_year=2011;citation_pmid=21471011;citation_doi=10.1093/bioinformatics/btr176" />
<meta name="citation_reference" content="citation_title=Nature;citation_journal_abbrev=Nature;citation_author=Drosophila 12 Genomes Consortium;citation_title=Evolution of genes and genomes on the Drosophila phylogeny.;citation_pages=203-218;citation_volume=450;citation_year=2007;citation_issue=7167;citation_pmid=17994087;citation_doi=10.1038/nature06341" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Dworkin;citation_author=C. D. Jones;citation_title=Gentic changes accompanying the evolution of host specialization in Drosophila sechellia;citation_pages=721-736;citation_volume=181;citation_year=2009;citation_pmid=19033155;citation_doi=10.1534/genetics.108.093419" />
<meta name="citation_reference" content="citation_author=J. Felsenstein;citation_year=2005" />
<meta name="citation_reference" content="citation_title=Algorithms on Strings, Trees, and Sequences: Computer Science and Computational Biology.;citation_author=D. Gusfield;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=B. Haubold;citation_author=P. Pfaffelhuber;citation_author=M. Domazet-Lošo;citation_author=T. Wiehe;citation_title=Estimating mutation distances from unaligned genomes.;citation_pages=1487-1500;citation_volume=16;citation_year=2009;citation_issue=10;citation_pmid=19803738;citation_doi=10.1089/cmb.2009.0106" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=B. Haubold;citation_author=N. Pierstorff;citation_author=F. Möller;citation_author=T. Wiehe;citation_title=Genome comparison without alignment using shortest unique substrings.;citation_pages=123-123;citation_volume=6;citation_year=2005;citation_pmid=15910684;citation_doi=10.1186/1471-2105-6-123" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Haubold;citation_author=F. A. Reed;citation_author=P. Pfaffelhuber;citation_title=Alignment-free estimation of nucleotide diversity;citation_volume=27;citation_year=2011;citation_pmid=21156730;citation_doi=10.1093/bioinformatics/btq689" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. R. Hudson;citation_title=Properties of a neutral allele model with intragenic recombination.;citation_pages=183-201;citation_volume=23;citation_year=1983;citation_issue=2;citation_pmid=6612631;citation_doi=10.1016/0040-5809(83)90013-8" />
<meta name="citation_reference" content="citation_journal_title=Oxford Surveys in Evolutionary Biology;citation_author=R. R. Hudson;citation_title=OXFORD SURVEYS IN EVOLUTIONARY BIOLOGY;citation_pages=1-44;citation_volume=7;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. R. Hudson;citation_title=Generating samples under a Wright-Fisher neutral model of  genetic variation;citation_pages=337-338;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11847089;citation_doi=10.1093/bioinformatics/18.2.337" />
<meta name="citation_reference" content="citation_journal_title=Stochastic Process. Appl.;citation_author=J. F. C. Kingman;citation_title=The coalescent;citation_pages=235-248;citation_volume=13;citation_year=1982;citation_doi=10.1016/0304-4149(82)90011-4" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_volume=10;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Larkin;citation_author=G. Blackshields;citation_author=N. Brown;citation_author=R. Chenna;citation_author=P. McGettigan;citation_title=Clustal W and Clustal X version 2.0;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler Transform;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Li;citation_author=C. Yu;citation_author=Y. Li;citation_author=T.-W. Lam;citation_author=S.-M. Yiu;citation_title=SOAP2: an improved ultrafast tool for short read alignment;citation_volume=25;citation_year=2009;citation_pmid=19497933;citation_doi=10.1093/bioinformatics/btp336" />
<meta name="citation_reference" content="citation_author=G. Manzini;citation_author=P. Ferragina;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_title=Fast &amp;quot;coalescent&amp;quot; simulation.;citation_pages=16-16;citation_volume=7;citation_year=2006;citation_pmid=16539698;citation_doi=10.1186/1471-2156-7-16" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. R. Soc. B;citation_author=G. A. T. McVean;citation_author=N. J. Cardin;citation_title=Approximating the coalescent with recombination;citation_pages=1387-1393;citation_volume=360;citation_year=2005;citation_pmid=16048782;citation_doi=10.1098/rstb.2005.1673" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. F. Noor;citation_author=D. A. Garfield;citation_author=S. W. Schaffer;citation_author=C. A. Machado;citation_title=Divergence Between the Drosophila pseudoobscura and D. persimilis Genome Sequences in Relation to Chromosomal Inversions;citation_pages=1417-1428;citation_volume=177;citation_year=2007;citation_issue=3;citation_pmid=18039875;citation_doi=10.1534/genetics.107.070672" />
<meta name="citation_reference" content="citation_journal_title=ACM Comput. Surv.;citation_author=S. J. Puglisi;citation_author=W. F. Smyth;citation_author=A. H. Turpin;citation_title=A taxonomy of suffix array construction algorithms;citation_pages=4;citation_volume=39;citation_year=2007;citation_doi=10.1145/1242471.1242472" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=G. Reinert;citation_author=D. Chew;citation_author=F. Sun;citation_author=M. Waterman;citation_title=Alignment-free sequence comparison (I): statistics and power.;citation_pages=1615-1634;citation_volume=16;citation_year=2009;citation_issue=12;citation_pmid=20001252;citation_doi=10.1089/cmb.2009.0198" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. Vinga;citation_author=J. Almeida;citation_title=Alignment-free sequence comparison--a review;citation_pages=513-523;citation_volume=19;citation_year=2003;citation_issue=4;citation_pmid=12611807;citation_doi=10.1093/bioinformatics/btg005" />
<meta name="citation_reference" content="citation_author=J. Wakeley;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=L. Wan;citation_author=G. Reinert;citation_author=F. Sun;citation_author=M. Waterman;citation_title=Alignment-free sequence comparison (II): theoretical power of comparison statistics.;citation_pages=1467-1490;citation_volume=17;citation_year=2010;citation_issue=11;citation_pmid=20973742;citation_doi=10.1089/cmb.2010.0056" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="twitter:description" content="Comparative sequencing contributes critically to the functional annotation of genomes. One prerequisite for successful analysis of the increasingly abundant comparative sequencing data is the availability of efficient computational tools. We present here a strategy for comparing unaligned genomes based on a coalescent approach combined with advanced algorithms for indexing sequences. These algorithms are particularly efficient when analyzing large genomes, as their run time ideally grows only linearly with sequence length. Using this approach, we have derived and implemented a maximum-likelihood estimator of the average number of mismatches per site between two closely related sequences, π . By allowing for fluctuating coalescent times, we are able to improve a previously published alignment-free estimator of π . We show through simulation that our new estimator is fast and accurate even with moderate recombination ( ρ ≤ π ). To demonstrate its applicability to real data, we compare the unaligned genomes of Drosophila persimilis and D. pseudoobscura . In agreement with previous studies, our sliding window analysis locates the global divergence minimum between these two genomes to the pericentromeric region of chromosome 3." />
<meta name="og-title" property="og:title" content="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/8/883" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Comparative sequencing contributes critically to the functional annotation of genomes. One prerequisite for successful analysis of the increasingly abundant comparative sequencing data is the availability of efficient computational tools. We present here a strategy for comparing unaligned genomes based on a coalescent approach combined with advanced algorithms for indexing sequences. These algorithms are particularly efficient when analyzing large genomes, as their run time ideally grows only linearly with sequence length. Using this approach, we have derived and implemented a maximum-likelihood estimator of the average number of mismatches per site between two closely related sequences, π . By allowing for fluctuating coalescent times, we are able to improve a previously published alignment-free estimator of π . We show through simulation that our new estimator is fast and accurate even with moderate recombination ( ρ ≤ π ). To demonstrate its applicability to real data, we compare the unaligned genomes of Drosophila persimilis and D. pseudoobscura . In agreement with previous studies, our sliding window analysis locates the global divergence minimum between these two genomes to the pericentromeric region of chromosome 3." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
    <title>Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zN9-G5Qsi7vb0iRmeEXHOzB7D2HFhYS0_LldXRqqENA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459306","apath":"\/ggg\/2\/8\/883.atom","pisa":"ggg;2\/8\/883","ac":{"\/ggg\/2\/8\/883.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/8\/883.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/8\/883"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/8\/883"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/8\/883":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/4GVSFNq4gzt3BoG6Te08fBXI+FWQb3YQ8AK24ZKWc9vI4ykRZL8AH8shRWY7oytMZ+wXSiTyufYkT\\\\\\\/IXrkjO7lNIac1IQJeBS841aJJhQs5udF9jiXT7lfhmLL0SLv34Y8eggX1z\\\\\\\/Dw0b7rlvx6SvFd4fOFYZLrMsZQD15LQOjEKHiAfEpkFgaPHMFLwgmdhOBS0z4U7lr0kLeoOlAz3zBuVRzoBooP7kwc8WicO0ogCiNzAB6dnpBQYPwvnLWGnKmLK39FLVxudUOXWZWpJfGDM\\\\\\\/xRzaQI51VJ+zCLnDKgPQXHdTs+XmXBtUKz4F\\\\\\\/2+zFpGef\\\\\\\/cDBBMqL46nyki3dvrcGnrbR520q6vtsFAcoXKiXBJOyDAnR4mCokV3lnUn18nJcAYJLFxHX\\\\\\\/Q+rkA34cjmbO+n3TY075ys3K6uBSfj3rkfRjziMEhzuiRK86DS2ayi07etuuWqxSg+hI3Iwx1QWXHi5Mb3CNmiUeQmXR88VEyKjmWp7K\\\\\\\/p20kNW8zN7xMjQziiiC+ZjjK2vsGK3JFluId\\\\\\\/CvtgpFMEg96X84W20Td2JEepnbKHHA0QPRwK4SXu4OLgZ00hAhoLtZQSZSOOmN9lcmdrrmLRLq4lGFj\\\\\\\/v3dlp8AQnCnEeSv68PCIoNcfh55D9ZxaH81oDWu4gHUX8PaMrlmaapzn+Mz\\\\\\\/SC0MLEH6n2FYCrasNgPQiWEp3IDWHRqXqg5yvyRIVX6+tESfJVHBdKimLa69Ia2hyKk3BQRky9rhs1Jms9NncxwVPY6SyY6NcqmtvcC0uSYhvV6PDa5FlC+TQCkgRbU3WyQZsX5nVyHVpClsn6t5nNwy+lyGsxvueB4KJF3u0Tw4vFH5UqSZ2s7tixQO2baadjxtxEN2Pcude6Bzxw0I75\\\\\\\/y2fm6BDHZdNXaNalwevWezpBYONMwfRbFqJMB1HOjLwSOolz3dovB1ryCcYH8f8DNYUNR6OWedF5\\\\\\\/c8dl+W0cN0szfUxm49MASBCG3wGdWM25mOVUYHnoK9iGNhPWmlN7Jj8Osq7YqadrfQkXPk8XDBLCbBegs787dbJp\\\\\\\/8wpZvow3dQeEqcQf6BJVUQ3i0bO3\\\\\\\/+VEYbC4+RuVlhxhnbqNwaNYo\\\\\\\/JtKPAQBy5V8nlnOZEEuotS5uiWOPUjgJPb\\\\\\\/ka297aPWPCyAb2M7xzk9KN5T4IZ+N102SXjW8rM\\\\\\\/yotx9RZqs7jL4yjc2GN9geq4xMw5+CjHoMG2ua9mytxFineV8gRqczy6icqftP33fll\\\\\\\/Bd7Y\\\\\\\/OIaIH\\\\\\\/aAGJyday3\\\u0022,\\\u0022iv\\\u0022:\\\u0022jijzDuyWQDP2lGhCGEzr2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228cf32192a7c7670aa728a92a80998081a619fc27e0ba22344b71dadd967b6afb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/ZLGuOB+pIhI0iduI2HfaRMUjffw+PM2D3bNvdAnV1RqVHTWckEcEOWWUJ8bPtiBeNmNScsbmRJPLmGj54l8MzNxpSRaBKRkF7FRbQyCj2EJ2zCXPazvzbquESp\\\\\\\/sYgBNKyC2DY0hoFlq6Ax7yjcJNNSebnKiFokDIGy7SeBE3\\\\\\\/XHjTqm8I7PFsgTnCuNHlctEcXRfWldSySZ8zPZjfvm48MztqVm+jA\\\\\\\/k21\\\\\\\/2ItKV+sWxAuwaSCHTE17IpZ3ooVRuadvy1DSS\\\\\\\/uGlXA4KnJ+LfdAYjKAy126fTnQKWEHNBgc13HimpY9UWAYjSvCq6mxS5v0ovhXpUuvdD6CtWy5m0a0KLZcIoLBNgF4DvZsoE\\\\\\\/w\\\\\\\/0fink8iDmvp9wcx1qWAVDevIGGS\\\\\\\/m8PYammSvhQZZwq8l64G9JY8pm2zxxaaWhxN6OZ6LokKU+LzJU449P32P73fZA4i+m1qtfOTBywVjE6oyAjbKsOw1NkH5skT7dStmZMEAGzr1CCVBOwEOSoSB3eVgvGq1KYDEKcLCQl4mYFXe\\\\\\\/qljAQKuYEE0nglzGGyxmVAcoigSjjSgCniEHF3CpKBEk19bxQeJgPSGs2sfzfoEfc+zbLLRPRy+91ZWEyjrpEGOou+yrIDN\\\\\\\/bqd3kDd\\\\\\\/CcseDJF59S7B7jr4v6zzxecYQ\\\\\\\/em62x4n9U2OWrTb+Z\\\\\\\/DtTrDl6CNUlTpwIa33TpcHKJT\\\\\\\/MMUogmsMi1dt6EjAWUDi023zvHPVKpA8XGqNQx5NFzWbJf\\\\\\\/Xp3Uj19fLAtCSBJsca\\\\\\\/HWj9CDd1lTRR9rgIAvaPHcPNyHhuH7E2POq23FTFxrRyWq+fcvJ4OtDffDJRs5zSK8JvfoulGq9KuZFOSoAhjR03QfGbYW67M98gLVT3R6BsmsACV3zaPZ4iwjXuNmq0zR4U\\\\\\\/+ttafk6KQ9iRn6+lUVVBFeO0X+a9nONVe7kzRuCivOS6ir0Q8mD\\\\\\\/yd8NNKVKShqzPejiAXZcJU9MjAgRR2HYFdF8Kf1XF9VmYNvJm1U5kl3T340QH+B706HTxLUaYjTskpw+Z2C8eD2wSRgkbiBrZoVcq\\\\\\\/rK4HlKB8at7DPh3LKOCH17DjKtrpmyjOCtrFz0WuKOIpNvFupfyQlAiw6C33LvgHzp3R4OTWwtO9sLCzp0MsIlnMSWrDyoT40Y7knTsK78Dv466JyHDE4HshcItUIJxcLc9uHLXnFB4\\\\\\\/tkdrMIQhzI6M48GsJ\\\\\\\/UJfKd+SgmJQDgm7pbAoF0wYinlQIQQTM5Fv\\\\\\\/CTvLwqL3j7z4OJRVdH5ud6Ip7JQdT66uXn38v\\\\\\\/YZFDWuZwxGf5IIL2M87ORuTRX1CADW3WOOQFsdLFp8iSVW5MTNAscAsjd1IkKwEwLNcm\\\\\\\/QA1PZdh2Q\\\\\\\/9wS153B\\\\\\\/L7+zSLj\\\\\\\/E3MGJZHgWnn\\\\\\\/SALprRbs\\\\\\\/33EBYHXjRSKSAZTm2v3CBleu9z9KbmduWOoXj3kuTV26H+jg3T+U00LE1rTJE5bFdItOjSTR3jixxZCe2hwSPGJCUw7Ujcowhrie703d+u5ET\\\\\\\/1K3Djf2bZvy8\\\\\\\/AFvuopWL5JiBWBQuxeii45OiOL4LyCJL6wfKo2\\\\\\\/h2TEH9rp48OPc3Xvyz8bZRcGjRMsJeHXF0mFtfwcVsNx80yxsjB05F2GMDj5qMmMnT4njcbVOg7rK38qXVUqJhqo4MLW2boFyfP1uLvzBD6LRcMe9vs6gB2c2MvU0OM4\\\\\\\/bUGOqk4C9X2n5zfQYKMU+gIQVylmG23zVJiRBv7DqLt4PN6PXvuS2ucf3xt0MWkug0me35+Vv1aUAuaGy9FFE54b87o3rgWcVIEQ1cj0+XZR1an8kI2dyg+1oZzIY\\\\\\\/iJpHcTtLC0bWRm7hn\\\\\\\/5pN\\\\\\\/yIFT+v9ov6ILD9DHEjzxoWLoJcDaqBqTq1vdOO5PwzOVreENxQQ14SiZZlgYa3GRsBmJJ2S3stfBwT4y6o+kIk\\\\\\\/INUEkZsbfWiVsujOUa\\\\\\\/gu1yjFRPgafmluYK4tT+F6TGk25hopSQAnc7P2cfNl9uu\\\\\\\/qwFSXQgmVTqyTsXu\\\\\\\/NsMtQ1db\\\\\\\/nxGKJCKH4SmT9A\\\\\\\/LA+TyNf1564WsNQxQfMUiLi9oEaio4CpLHhAUwz6eqctQXpvGrPAiDTJOMbHgY5RqAEGryP9h1MJzSop8CQ9OTWKXbVNyRdLPuH97ckh4RahQxNdn8MudsP3lTmgrE5g8Yy4d2r0qQxsVKamcmSrkJd1IRZGN4gGANTC6W21L\\\\\\\/Q8AqlpdWeUeFKkl2XH9pHQrDdxY8+X9ApU71rhzOljqvFOc9TeZ72D4uW7hOjAual76lebkzLqHrDWdci5FGla9aH6iD\\\\\\\/IEr+gvHJrr9Qjg60xBXqWu+1UW2DkORMeobuQ2oGUWr27\\\\\\\/WonDWEqSuOS8yDsi2W7NUFcQbkLMO1buLrHk7x65WJredVoZDKUv0ANVlx6v51dEOeKSchviPOnuyVzB\\\\\\\/SHtaNgPcT+XT+hCCCPohpLUXElNKfXXsMUOsl08dF+GQep8w4pAaR3CIjuC2NBfSA8wfcC+xLpWwe5rewJLVZwHRN3zvaVS6P0I99zOc\\\\\\\/Y9Nww1UQe0f2HgMHESKIuu4VvsQiNZmwS6jxj9iuJLSIChnJLVCWkV2XEZxp683KWkHANexexHCT6j2zOYx8sl9oKX71oUNk7jq41o2xxWzw1t09fbheGHqQN+eZ5koiNirInTxawsfLuOhsWN0PER2p57uWnAJ\\\\\\\/2fpHFRg4zKbQ0mwa5zIkvnNE6csJ6rT4nzLmxdsQjODEGrZWlBNtWJSYYQCi7J3JqmGU\\\\\\\/H6z2EKWN85EBJrFMAAZth7R9lVF+zunx88uxkaTWGYGwLS212ftF+2Mi1OOkc8YXv03FbRQszSqOoQ3UHU0s3mnHb9Zal\\\\\\\/2l+RYT9tDmOjo4F77eNPLBaMLd\\\\\\\/M00LGaTvHCS4v+DDzFVtRj5GfpFgOf0QvhebqrWYbdffLPwuEh9dOfaAPBCxmyivWjA1RPQs8NSMLAlV37gi1USHJ0pwS2MQRRsgIyYrc3ZSmLygLbBoxhbtTNkaO4IzlEWAt+s03gC3TbGlmw1+z1TYtLI2FA0KvCD81Ax2klr+lYm5rPGOKKNNe\\\\\\\/0mHTgLM9BKoPnAiggbLr+4gvuiJTev\\\\\\\/63ZhkEPmG6p7+ih+D18LnvEHJb8p7sC9r1AEuef2\\\\\\\/Ex0u5S71fNg+wZDdSQmWH63DzJoZ8\\\\\\\/Xgl0K0vJ87iEDPpBTx34SBG8QIBJ+EYAb\\\\\\\/7890sSyIWSr2I5Tk36AsW2MW3uMO8pzhqLvu2DltQRieTsDsvbSjxRTvFrrXk+2j8gY7ksRomRg+amiDdgVr0woftMKtRG\\\\\\\/bDE\\\\\\\/snAe08WmjSJFC3ejRXgPNJq3PotvO0GjpuYKNTL05jnfQaIksHrsaL7VRlDPuiOORojyjxOSW+hVOupjQQkaOYffGP2J89tCk9+WwQ23pjx2UTgKblSFtOC2Yow7GPK02sLHZHPBgCTxZkqCF2OygPamiINwZF4lSO+Jz4RcFEeOnBb6ncW9SMmU+3vMr87EfhdaDkMhf4Tw07hxg\\\\\\\/4qBw4wWqhkJnC86dCg8ZTI8pvcWhFe2STpFjowAkNsUzx5NKw3Muj4HeUrrmMR1PZZ9BGmuG44BQtANizAbAOom8kiLR4Qng77MbI4vYsAuTfaFXr2Rcpy0rORAyG\\\\\\\/wZF6K5ROSzOnKnuYBNsf\\\\\\\/9npPHYTK++k4ak7t06B5+78MQKIf0E\\\\\\\/rS+jo7FVYPIACmp\\\\\\\/EY3TM+2aYuCaEEvI0oKzf+IpqdMFqCzWqZ0yP1g8JdXVXLZ4LgdI4B4gQvflduptHSzAbHu506A70Lmn8vDPD3n9buULxidejY3xq7lHAOZyED16MGUtz1nyBZEmkKHWWxl8iWsjGOWb8EB4sxkZOWcAodnqmX2sxa+RA5d70kcGPhJamaTTFW\\\\\\\/FGseZHjbMTvP0tI53rXbt9nmRb6qUjR6zAorz9EfIHcB1AQ9oV8kPqpmy\\\\\\\/5fENQSqCPgRUgDAU8J5dj5yNm7CU9qqWvHNRp0ph+zOiMsY8n59I6QAc4s9ZZKwIwxTne21WSVor1dN0nZct5uVarvKXgf67Xw0NT6C6XFJePG8bPB90aNjQBy4xdBKFpIOxpiY7Nf1D5\\\\\\\/IbApxnFEbyllw4AqzWxr54gvN8hrpIJHKfK+Z8SRFq63steJO8iYP1aMBxBeEj3vxG+2Oaa3Gc47OM2hkcabyeP3kWGtYvPSg\\\\\\\/d0UqAClYc\\\\\\\/3FR+2UiLppHZBEVr4AehOtq9rHm+io\\\\\\\/xibUVbzmSg2gMl6VcPJg+EKv15r1bpaZEKYbFvQxvh3BcVyKxZvbFrkt0mlrmVMUvuak11niTK+MzKlMJSIWIn6KMEEFF6rF6Tdsm26tB52BOqBuxVKw4XyaJh1edwyrXUMU+imRjP7NkMtJWhrHoIn4rh\\\\\\\/sLk6bSRoIas99e77jrZg6vq9pFpzEBYWi\\\\\\\/tR+G7GZLBqPaVwBG8Qi43oYLkh\\\\\\\/KhxTOp\\\\\\\/7xp7dn1s3E+gQnWlzoLOBevkP3fMT5Ac0c7HkCs9RNe9lVM8H6jCeGflphapQynJbhTu4Te07sbst7pUS0q0vLvxiw2siVecbeCidRIvIBOgUBaaskffGyhmHFBoveal65Px6\\\\\\\/TgBkf4eCc2l8qQq9VHn1WZIJ5omRU0dy0mYd3K+dDRxdKMqms9iJXoMmZu8l26BZbCMC9qHp5i4+a9gfx5j18i9aKN37W4CQf7vtsIrmCBkvMS4LjQpN1i1qoBZuOw+d\\\\\\\/IxqWABPpZlPEr1uwGkyzzI0zVfS\\\\\\\/r3pBTGdysYR4LN+RcScx2QjYWJYXDBL2VjiH7ptTCx8Efb0TJeSUrqCF24ck5\\\\\\\/3Qy+BnKxeTaZ2xck1amEVlkZoXbttJYjOUg0LvzhmS5HFuGjyhLNNLlKdME510PLt+2xqpEw\\\\\\\/fp7MYPN0+rDGkZ9OsIyJ6KDNs8IxmssuA4ilC3xJB64lcDvpb6lYo1nvXm+7vmutculszE2nYICG2XPUr6WXDZQmNAt2nEu73PmvGPMr6a6dNFvBiYdylZOT\\\\\\\/pOzVC3MU9GjBMNK6pq4rzYOkDWCq9DyjcUMFWzZrNE+KKDblC09QXOjQ8KKlZ19vjE63dI\\\\\\\/rm0dhA1gmXSaLTnToiB48lolAVVLXHFB4cyolLNTeFI9twhTQPq6wxreC3F8B3sJKsBi9iOolT84hpWanQn\\\\\\\/INNkd0UXDg7ZrrQuY+AvhmdqbzewpByJMuFOPAYKejzVlHpff89LrBfhEivASa3XTn1A8H279LGdffxr6OQjQYMzts1r1l\\\\\\\/Vy5sVKGBsM2sMaj62wWlA4+KwlrWjrkAEyRNLNcFSXYtnppd6Bcfc47098XhAFJxHdfMl0N6mlSbu+MdTJ\\\\\\\/PNJDsRm8xFqVh1YIMl3chIF1+KvMPUvc\\\\\\\/Ool+hXNHYmta6UgLvr\\\\\\\/fxll0FTA5zuGtNwoyPMEhQcYqIScpizJVvZiAJ8+3tymGO3pgM298LzCAkJPF8B835oj+vtVXNj3UCfSO0TdaRbHiYa7CTbG5SI9ILxMo0MEQ6\\\\\\\/3VAHBpWuupykdZPp8TxZ2Qkj8Fbo1BLswoOOPalktSaiG7waWDAJ6zAnrv1ULwaG1Tr+faPEzmzDzNZyYkygrFnFLwdLPGxkYzvft8ac\\\\\\\/AIplp1O1dpUxk\\\\\\\/ZAgsYITmA91DUo4K2BkRkvrwWZU4Vb3DN0hwzfC4sInja6Fa3BzbZF5B8c5c3D5qqmof9imbEUrFn79AAgSgcxvew5IWAzhge0uaNqlKWyuWpRR9u1DMUtBOfp+qltCUEFMWx1\\\\\\\/sauHp6LT\\\\\\\/jQhVHsKHA\\\\\\\/HkWwV98syQiZ0nRiDpMhbU8OSl71ZjA0Djn6+QzLFES8WjFPrs7Fejnkf4SmTwkbLJqZmtL0trBW+U1aFK+IAvModh\\\\\\\/m+i0kD7VYQRSmEsNuqCj52KClHFwj4jz7aUZc0e+fCHDYd6aB39r\\\\\\\/4E2gKSRe2JkQQmOjGyr2n9eAz1AiMdslp6EZyV2W\\\\\\\/9RiVctd7+7aK4lJplcFY7jhXiuz4NHIkEl0+NAwjomVO1wMKSG16heABllYUf\\\\\\\/M2VeNsW2jC1SXLNnw8gFmuethSCCSTah7IHEWMcvwIAIDhksvflZxD2NHfDC29UDmPmzlvHCOfjeiFmnB\\\\\\\/uYuAKMXTR5xemvgnRXkzbkQHRFqlKRTrBc3po0tUmgTubg\\\\\\\/Oi039qdiOBcDn8Q\\\\\\\/phoY8a6iv9pcTtI\\\\\\\/hr805tlfTCF2su0U7WAtJy2POMB7T+9+It\\\\\\\/PQ+4oVzCpUXkeZrzyZ3BB\\\\\\\/fuFUvLleEdP6DGqVHPhHBPl6Zh1Mkf9XlYrHXDFUkkOgrKgS+92+QIPP2X7A\\\\\\\/NL+o9KEMudIY+iV7cs9y9xKedY+yWYZ1bfTkaDHJaeUaThHejIsSWQohFTFmPODJC1SxOZvD\\\\\\\/22NghOSDpVjpJKi9+yDCalZw\\\\\\\/nxlkEMKx7ksv1SVODs1sN3p1cvglIhxfdGQeT1Ai27YqpUxsv\\\\\\\/eOk5a50XMop+8h9Ex+KYQot72sQIfbA9RAGAYgbSqzj0ch1\\\\\\\/veCLqAQYFS39QEmA5Ch3ceNVnsvzq7SVC2CiWjN36lY697EqO0E8tyz83Iwf2oIKvsNxUNr0KuLFYjK+vFOdSaEB34fCopq+Q5qXxeHzrW18UW56cHjDxZ8v\\\\\\\/tJ21Im+Rf7XTeZBCRkdapOqPNQZuI5Q12\\\\\\\/ZvC8fTc0rThs7lv2BYErQhMii\\\\\\\/EDr974sTvzchgl2RswWZH6EqiCwB3gPRN4P96ikvSuQnQKLXR6eXssQ69Nqu2KtI50c1g2HbuXWIx7yFpski6cVhjIGurolYnP9AJkbo+1s9+5Sfz9\\\\\\\/s6Z2ghrxdu\\\\\\\/J3zxS51vhQS0uSoKHa5cWvH8ADt6jLfR\\\\\\\/k1nixu8I21t4gXPr5I7wGyOtFtE3U5XREUL8TFPNfO1u8H7rcyYpN5sPVWvDzZpYW8MCaPy4XzIu\\\\\\\/zx52kBUVu7VYfsAxtzccmsYWAHwFrQN6jXEuqYIattznVxUqdZ1WduO6ue7d+A0N4canrMjj33M6Q0MYM0xwecWEwJPjcxVCFuxOZ1vWqiewrnrrMxQdJ8AvAzSvH7lN+6Zvq6dztq0Ug1IXVOZO975+Wb2tiMk9OZN\\\\\\\/vnjQJXNVfIilLY0fMhBKH52DGVvYHLDRNITcGz5\\\\\\\/JkSFA+d8iBc0LHOAo0AjwO75XZcSMEFgRm+6PQkz6UYg7FZ0347FGoxZ8z0CD+Nnvm0967eydlKsuRk+xaHhFYZmbkKD66Qs15d+C7NW7sLuhn2+GM0KOm6Ip2UYOufHcwEJb0dsPxoVrlmu4dtBAgMYXoY\\\\\\\/8Uel6grj2m08LMYIL92D8OgMgpDLLHrQcxPC7SGlDO58pmMSuVfES+jPB31WpLlFB6ptxJTGnKvCmBWsP3lX6sh3mkChna5NFHL0+Er1jRDVZQ4IJoEFBagsQT9mZ+j49ZtwbIIx6dRP4Qe5WUS\\\\\\\/GLeeCrAON3Y+aOdv3VaY4JdVZrFBDF2AHhOUichpRlNE7Wzx97XsOKzBFSd87tGDtmyuYN1l4iegaGCgQCvCfA5\\\\\\\/5a8vh8BDwqnPjStTss9fGVbYbsYXiNelAQnM5WrDik0CyOumQE3pFdGSEEvCujhwUdQ6GmcvM\\\\\\\/2zbVJhTLIXGmvPFJ0G0tfwNnKINAf4s\\\\\\\/KmGU7J7tfvgHc2EfRT3sMTZtdmaIgoefza\\\\\\\/76DrrBrlsO9SzMGJPPweGZPuMmstHDefuJEP3VkfLqc\\\\\\\/kLcPOKHrWWOenbOuSPEIx9nCM2Qzy+ocJsVC4vde1u5rckr3QyfSxRpYa0FA9RJj08JSTCc2uATPQ41AtrHxXeBZEQM2LvauPFjRLwnux3JPXSPW\\\\\\\/3Vlt2yqbuD3w6cMcrDUa+abbrd1O\\\\\\\/33Dr3qWCE5H4T+dr\\\\\\\/PS2IBMUCD93CNtvb4HOIu2bBA+8gR4EAIeRLihrdq71sG8SxqtnPsvO87H9WIgazE\\\\\\\/vN5RGi3I8iWQy1X7KR0TZiLAJdBqFiuSg0VviWIFcDPGmCGVUQGcwid27ohGhXQ\\\\\\\/3C+o5AWXdEkvjLXsHGyKCVfezdEefwFQ8SHeet6alrO\\\\\\\/c5sBNGXLoaSl7b6CYcf4xgMQzqgA9NQE2DSrV3RiqeTAVk7nSTrQDib5iT\\\\\\\/z6WV72zd2iOJoePNsSH5j\\\\\\\/yolbfEvDNmVk7b+SLl36j9vz46a9L6Z298grCmwRfqpJ7dGy\\\\\\\/8t\\\\\\\/qaKAjDIp5qsTKSkFY20fD94INryjiTDskngnA9tCepPL2ZjXDwcJSkRFbhLeEELDhG4pVGpv3YQKglyLUE8m3m74KbEmd+2ePKqk4OOQIowdJ+KJGTYV9R+g3RtloO\\\\\\\/wJLoyGYOcGS12FidJG2dwPuX8lze\\\\\\\/MCaOJcoqq+02NEOqQ3XF5dYc1QThm2e\\\\\\\/emCUgQH0GC4ozwGeQJsMzy0KHXzqcucfeWuQsQhDFaiBfAsL5F9Rq45b3iizIY7G2TMNA+Q6NEByuEwkHxiwOyFctTxqBn8qEVD\\\\\\\/1yT\\\\\\\/KuVEqbxk4zCic9QysdaYXRsmvJ2afO5ZUETWSNnTwB4sRRGIuR6TBC8y4rMUIBHOOz9x79etMgDH\\\\\\\/hZKlTr0Exwlg5NbR7DgKIYZ1f8hDsCSjBVopq0WN2Q5JPBGPtPN\\\\\\\/hqM2YETHEADPvASwEqG34opMz\\\\\\\/Ub7CIM+Ib7azn+stuhzOgJHTFsDXxt4QvPSxUMBvcj9\\\\\\\/dSIuFZit0qtDNIIJbq2YWmILHNF8E2uyCvYeR0stDWbZzu10JwFdawbJS97+Ufnra9N0lla9QwXUp4Jw2cwk6HMOHE8xSxnWSAq1JWIGyTh5GslVxOmwIZzoJg\\\\\\\/f3oTOu1y0sEAHhA74v\\\\\\\/dQ3FkbpN74MlZ971wWjc\\\\\\\/Db8ScFKfjy25EQ\\\\\\\/ZW6mP1ZUAlaysE\\\\\\\/vroc6rryO2rufnTeIbIdY5vwqnCbJVtQbQEOc0\\\\\\\/+K4ni7AxK7BcyRXAnovcOGYd+yUkPi7u2fgJdGgnDpncgIstpSEFfi0Hq\\\\\\\/9vk0t8DVuilFQn2eUrmttXC4z+maxphLEc96qcIxqekwnzjvxaJUKPEQBwHciAdqWEiS8zWTHu4RL0ZVuLMh3qKrWM+6O0dxOEUIozhoZytOoRyC7g+os70YSLwrmRj75sKnJK\\\\\\\/CP0OyzKvroDfFP9QUeGtKIb3kWcb5LfpTWSlsr2zwKijZkZf8joHxy2oPDMbEd84cnO0aQtJq8To2mgZUNoZ0m520bgqjNEamxNpIh7ttQ+79sYjqpttsmhiB56eNX3hObshKq7ErNSUhm3SNnsC00UqwRhUqrSOqWtixCC13UWn7j1oF8t43W1jlzJNXp89OFVD26wncFaFPK2iBUNlXk5nhKU1L28KBN763i6Tiw0ZSHrP+FAkNqk9GDEz+TGCkkVVbzSyjG0MBcAU6WOJ6GNoARVVOMqGDpORMTLBoExFUzht4ZN2OeYrdwvekeV4qBngMNs9q8MQ6a4V0WIZFwcZbTBYcnNw6LSOfSHYr0\\\\\\\/DeDcFSIIHe6NWvCGg6vv2V1ZLeX57zZul4dvebH\\\\\\\/HyY0xsJS2la8rTrRkXOKPwrw+c+DbgU1p3qQD+x0cpAqyKjBE1Pawz+VYb\\\\\\\/cPL2UL9uxs9ORYu\\\\\\\/wTNIKsBeyaAUPDM8khV0D26+WGp7D2h7\\\\\\\/M+PbWfPqZAVvP9aNk3PrubCKjuPRlPDPccSzujhnjP4cOzRZtTGsFmXmTgeD2GR4Sr9dq6E8cnCUL3aaLex1Oz\\\\\\\/6h\\\\\\\/1NwsZdt35N5dXZRsl+xxvMmXI8nRh5cxsUbLRtPDGxmrWPMX4J\\\\\\\/tPyAddeMLY6S3J\\\\\\\/kuP88hhVAsfOQ7M1sm7yemy9vdrxCVcT0w+boAGMhnZiZan+zrS9jBQPRH4lSobTktRE1TsrONjvM7viLVCjS8MuAsKD6eDjG97KFFnxfzyUUw+d4SiLCYO\\\\\\\/Vgz0TTipc30Ag59A6NQ0Zp9EwA7xUeSXVj+gjpLwZs1r4nrCigRXWwq5L9\\\\\\\/fmIaCy3yb37Lz+4jFtaS97n4yq2gBBDE0dBYOXjoubfuEAFGWW1Zz9THwWipMFF88T7xWLEOV60A0cs83yyv0RWEgdso4m5kSM0A2OWNL04UxnoAnBqTW9G11gWfbZiAlEgYNlqvN6sd\\\\\\\/WAwvRIKJcBVlWewsbkU5TMYRpx+unnUjk2wMPFbadnKDptaXegIB\\\\\\\/nGaS7wa8zJZ2X1s0ccWIDixvVF3\\\\\\\/+s\\\\\\\/DVADIEbx05XUn6hwSSamg+iAdC6Cdk1fP2dVmTEapKEh2FRoQ6iCrC6Q7N3MHMhGXH3ssgQ1LoAUWIYxm+JekmmdTtF4tcbD8pvVjTLlJZXOaiBlvn\\\\\\\/Wl+sDxfW9J638h71X2Q\\\\\\\/\\\\\\\/hJ7f49XGsAYrSPLkbdhWZl3YranUP0i4jUbTLvuR6l539fONxSvPxy\\\\\\\/fhicMTOFNK5UxL1bQGxiA5KKBSrrL9AKrZDPVJmZzsEx0ig4hDaLolununS1BdocZm+2T1VYHIijHy0Hv9\\\\\\\/U1lfGha\\\\\\\/2iH65Fwy9C8dUzFoUXS88A0vEi1S7hqzT1Q2A61SGNJScOPLxloPvMpKdqrk5TjQBhLzMEMwOkG47aIuNawo6ZkP48VFD5ZFjloqB6TSycu1ukbOokTsjJxtTzHfqN1B194z3s3\\\\\\\/8BGNeCL2fZfSaigW5vXmtRkRIE\\\\\\\/XSRdh0qW6ZQ6HxPZzk1r4peMfHVqW6cwHKlx3VaxzFQLl1Y1QCp\\\\\\\/QrvSDVewta31WWAoowiU52YK8RXft6Yg\\\\\\\/WKzG4RrOrqOalbPi4REnuPtPI0Nbf\\\\\\\/1SV8Tnzg7PtXAzZTbdpzsP2O02ULu46yeyU\\\\\\\/1rnGK9tCuuYyhMMK6\\\\\\\/gX6QRzBkFsgHmcSUCiG9aS8i23ndAMhOl9P2BrWm2k6TxS78+C0uZX\\\\\\\/+pjCmb4hyyoCGHKQmL+IGVqbA39yyV9WY9M\\\\\\\/q8vWUa4ndXc6gwxBFYkXy\\\\\\\/gqYAyduGxGSXa9Uc55p7A5MNkiPf8J6MMofNVwzDhhdc25jLwQcjNtdKdv\\\\\\\/sSFzAk5n2QRambkkIHhE+hrzoATFuQyDPkpoqJe9VDWHM0snP+v3wQHVrMyjzf7HWI0FXQSFHrTzBoNIW6d4NYEo0+ZRxgkDmQz9h9ZCtpBLsxGboNAHGpXJlHeLzvQVj6wPbfNjsOkVXeg2SyCu8L7e\\\\\\\/CR7HLWkOEyXkupSWo9dI4FscbFP9D28tHgP4SR\\\\\\\/aViZrws49a67ok9DIDpKHwIrKHQbm9pZ60Q7SGbO196TqlHeN2g6Xhcu3LiqTSjdSJoICMiyvtS8M90iRkv22PLTSHeVSPSBNGzcDqWlAfapAUDzGUD0UxAyP0rE08SxRdPaBSaur\\\\\\\/qMyT26zK+aQfAwjNqWBd2ZUGjDQW\\\\\\\/3+hgzIsQPnNS0dSxtXoirFSJzuaaHJ+VANoqRmQx9r+dP0h00ScoOLmd4st7CHbH3pgnZGIwSuqpf7UkiHdCNYUzUxsmXoaMZfKQfsLuXrLde\\\\\\\/fUoRnYD5fe+Xc6OxEc6cFnkaaZ5ztN59Zye\\\\\\\/un0351WM3oxzrBnVSv5YWXIC6azDWqmvL9pSEhPX1wqfhIeukP8IdzWC4VQ1DB0RsFAOW6MJ8kzJdIP7wjitt6bUf9J5q3Gw5olVILw2DdgGI98NX4y26fXoX1bOuGHCwoc0fPkUfQ76uXQ6NXW23pJeFb6ezGaRU6loPM3ipgHyxFuJPxu8Tg87bqDdxw6musTPLwGjCSzItkLbQqyPsNjDcdeDfrS7bguku0heUtQQj9HvuXeS\\\\\\\/hreC\\\\\\\/6cZDGON6PzhVbln96+8P6FipJ30UkLtewD0GR+slbC0tnnKaZmRn0UkabGWxbAXab5sHOb\\\\\\\/GKAUUbfu0xM\\\\\\\/r42JsYbQv2ttyM6zX\\\\\\\/UXLVwUWNZhq\\\\\\\/B4gUnQnArhsNpbxtRlBwfl89sFmN7LYnnYpwl0Cpsctd0n4ln28lXwe2UhLlfdAEbGA1EP6SETUbxxgJPvYVdFI154ptpTg9QRiRsuXx+J9hiy1MhHCRQvxYRJ7bT9AEYZ0u33vhGEmfvZWQfgRkqBbS\\\\\\\/yODtdFnYqbIvHsLNRKmDui5SNnYYJGXKLL6sr5tLF\\\\\\\/yPbLKNA9pZ4Z0DHrU3Y0SL\\\\\\\/7btywOoEHlqYdKmyGhGi4\\\\\\\/TwLf3lFEqVyST3ujMSSlbCYySShHo+ejmVeq00LKOYQWyF5fntdBXRjqiIUB2Paj19VNtWQ6iQlqoxAkzDkt8b4ZMj+oY0nYbWMZlO1dwxz1h0mrUU599CQ8\\\\\\\/CC0oNrFZbDJ1FqOwEfP5TujPOvZhMexiUYgD7VWCLBjTHoQ9CF+qGx9DeTDNKsh9eJYoJoa4QCHpOQI5NPiXumiPa23AyXD+5mVqzVJOwVZIh5+ooRh4McmeoAfxdpnE2CCbqeqQfuVr2Z6mkruFi3kbHgxovZP4zlilpEtk8trCtId62M2AR3wpdzUdN4\\\\\\\/9\\\\\\\/x9nG7gMRqC4m4+Xl2EGFnwzQF0R+L3a2ON49HBDCYWuwaQd36Umfrj0T7sXzvZbaOJ16nTuZsWmCrtiKBtZhIL3odJknaPH7I3BiqZqLrqV2Mk1O6wPVgAZ9\\\\\\\/cMv\\\\\\\/0cY0VbozWSb\\\\\\\/pT9ED3KJrCzFLUC4lBfZf9o4mFRqkZ6SbtLZWYO0fyYhIYTAThcl\\\\\\\/L4B2eOnQ5V0Lq822aoYs0ytcvdm\\\\\\\/ICDg0jzKlAybJEN+EMD4PZ9Js2YCynRuy02FtpocQ5x8YTt2eUISd+82cw6vi2Fma\\\\\\\/xW6\\\\\\\/0cLaVc8jGOPgpO9fEiUqnzhiTFcOtTNOh3CEmSDEjVJS08uz1sfZJw2uCWa2kUIP69CCxTGhvwVKI+uisdoXiYSoSVkqRFWGCKEM3pgmQADpwzlrZgjELrNI8Y5Zr\\\\\\\/UNGxHIc5f0qxfQc+FoxpDD2kXWk0LBYI72WMrXucIMVOBmSM1nunhDRStNi40thsXOwUrjrD3IeqzzaKt9gnFTuehp\\\\\\\/RuSa6lWPcl0g7kM29c7tTTc129RZqCEJbgd6h8zDrJP+xHxEulCymTVfPI1hvMjkfVaXnuL7r6+ILTjYpFr\\\\\\\/goI\\\\\\\/DUkielvYRb2w803Gnhp5OOh7qhDSiC437THXxRloXEvPvemQdhwHvBtYzcIT8pnokN3NeIH0tIVsxTD+3VwuSst9vcq6ltgSU8HC+WjiRraIhDZBpq6EL3nlFRTxpl4VTTk48Yt6XhaV6cBQcvof79T+mn7QIopR9YaBshgkzQY3jh4lpa7\\\\\\\/bxj7DKT6gIyFckC+V77adgVh3aJQZjndExLRkBlOYuVgnOU+2FdnA7\\\\\\\/4fYMC5haRi3nh9tDRQOYrF3RypxUnPw5tcDlbq5IU89T+tK+IxFWWZy8taFf81ApiAjOUqLzVi4kN9NCLsKXh2dAfSf8CYA\\\\\\\/H72ulybUhywTvwXWj8yXuD4CQ7RYKVjfHY8SiM6iczwduV0adTYCsoX2TO2O5UFuauq+zaFZ6WsYDLVNU+0ZnUnY\\\\\\\/DsTltVMl8Mvh2aHWYNg9FNivSD5tuuy7nSw\\\\\\\/RT1gQxoHqAXYPUXXP62zqq9UfFPWTGBryt5OprSvuF+meNLZrwZfC1wjr0bGaXaMruj00zm5Mq6iv+2SBTpFfZUvweHnTSJY2v+qld4nXtJSkFYsncN0fG09s6\\\\\\\/4Qko\\\\\\\/m6w6\\\\\\\/ggNmsqZPrqUXFoTdhKALpY5ldg0ryq0MlEu8BSMSap4tQfEGrvKcUXja+27MwQx1VFOE7mBffq1WKcP+AhnSrQDP8CQTPy9BLH\\\\\\\/2Yvy6gF2WDtSsv\\\\\\\/lh9yZk1L8mZJ1kjOBuTYPmTRLNXcgVFG62ugzssEZK7BYwwYpVi8xhTdLN3gMTl6sQTqhfRq2QzULH\\\\\\\/j52yqTqKFJGD7BUGVi3Z72Py09N2hAK09NQqhS2rvc4ylwdElhMVQiuHw2jbg\\\\\\\/FLs4+z0WBmXK3Udq5IfJ9vd55m5RD0IdJh4uV0TvX8gONzUvpCzLbR6fD9odi66\\\\\\\/S1px25WUKNnGHTaKLj+kGnD5muRiW2u6DRT83ZeaSTgpFUxSlfc\\\\\\\/0FfYth4jrOpaSZI79LaR5Giq3j+dDHEIh+zV7Ae8WhUTIbuq9s8SNoJd7HucdbipehaaV2fp0PXMXnoWyxbuTZG8r6xErsFEp8QQAijHfHQgulO5GDuDkgJbJQLGRn6SjmkCJ\\\\\\\/ha3YF2uR5I0FMb+\\\\\\\/\\\\\\\/QO8v5a3rBb6+VtFIM4xzOjTLHwSOaUTZRyF\\\\\\\/DD3dnQJ7NNV2PPjwuTngkgIJ1YLniNeuKtuVxPibB7QjHxajyS1+vGYebxEkduhcn2pHJKZoTyFai4S+HN4NXfA6zV2DaGzGauyUUUUwyrAW9pD+uaSlCFFX9MKioVnUI7t3vjkLSIOMOhf44zFmMgJEve0NL6zA2ARz1aKYT\\\\\\\/4\\\\\\\/J8cKonwpG6xOb0jzQQmLgtRnGeOOBhxqRpsZ3G+uYg79BilrZ7RnAkwTaPDJB+N+ZHhNMj\\\\\\\/lSbGm8YXHVxRDmdzUTmLTZsRWXJd8Zy62eej9i9YtW2cb7\\\\\\\/SyPLDmYJmCtBvjyxfG6i4P294C2wru8DqqSeQGo8MdovtI2GLIEh00oXC+gcbjWTJ6okqwioecsnTKfQBXbgB4W7+Y\\\\\\\/Nuoc16kFHmG1HCcHbkeeUk\\\\\\\/YTuL9CJkohy1A6Jb1QVlxc3B4VQKaqxl4rwWYGuZXOWBR6Uwt4Jdri5h2af+hGfUGGmaiR\\\\\\\/A\\\\\\\/tiwRrZ2cEIZxvCbRleXzdAWxOMfAKtzxSK\\\\\\\/xHLLnYuEcXmU9XH3yfwyR12\\\\\\\/34JFpe7uteZ17a3\\\\\\\/u6gwapU4bGiC8OeThFtAz25BSminS\\\\\\\/xKawmEjkQYztqoy4A+bwe3LMCAg05\\\\\\\/xQFNUglpDduNMmIMqT7fubKxM7x2plBptRR4V1DwIO\\\\\\\/ujf4VyMHUB63XWdK\\\\\\\/q9RGT8TypDJOZLxInNc0Czebfuu8qHMXtlS6LZGVcCl4R9+rFHHqbu2OEaL59IDTcOlz4JQdDghi4bhLNQV9bqbfrFfb3BrveYsgU3UILgN424O27KJnF0uRIK8xqm625+b\\\\\\\/dnRIE+aq4kWUnf55udc\\\\\\\/16oIZzQ3jlYsJC7DDM2YBRpICETUa5Wy6mRGo\\\\\\\/dgqDG+vaS\\\\\\\/9\\\\\\\/Rr15cLOxT9tl4NMTvalIdiGG2IoumuIk2t++mdGimqeUX0r94LaYaJwiJAzvAHEccT8y0EQ70hUP5xVCx+Mp7k+dbw7zgiI1WPLWhCF8Emgj0Vc6xk7jFY24kk\\\\\\\/OAcviTMfj9sPd4uNqNn6rT6HUh4RNdpC2i6t+KDdynolYAXqqQ\\\\\\\/hqXPEFZgV4vESXpehssfMldWAviZVkm7Xl2wbroG79SasnfXNWmRL+K\\\\\\\/5F4zi9WNkHQqYhpYS7ATwCClQRzu3gtjx0vxRn8mfWG3JApdTZmTdsW7gSme4l1w65ZUKIDZ8ue\\\\\\\/snnEmOHtp3zHHFsLzD8QojsjVUyTmB42uWjEPmFV2Gpw7xSi2TxVjPbSaUph7REHhpe8h9TJa3yHBC5LvHAyp\\\\\\\/EvnlKr2VBNjcilDgI2L7OzHGvOclWe869aLeKhn+KkFcSp\\\\\\\/dip3aUky2PNwqDUOyvXsZrDigFNRgdsi00U6\\\\\\\/t7FkKgMNqE\\\\\\\/+Ss5YBa5Q\\\\\\\/rnR15q8is9n\\\\\\\/w5OQeUmcpBolSiVY6Lc0ZDYXdVbI0pwyqNY\\\\\\\/aR4T3mdBrPINP3GXM2aAeroWpxfycM0YG4obaYgEjTBOfziYcZGy94Xo57b09nw8+AOfVVIQFwvF9XfuIa4Gc1+tTIYNRrsWqRmC7Blxs\\\\\\\/DZoDLUYA2siBmU9np7Je536ft+WPw\\\\\\\/MTAUZN0vy6TEqhbUFkI+TeD1hFvCXiaVysrH\\\\\\\/bZc7JZ26zw1r9NzVW6NuYwP6M7P1MX0mEMSlVzq3QEOzaY0b+XR2Cu6Roqm+xp2nHkAAAaNcQL1DJ+pVdFQUZDJR3J3qqAIkcVCJHkNBEMWjB\\\\\\\/9xeiASKi5MigFdTYTLzo4d\\\\\\\/bt+2HWfZ2nAgJXCTl1b85gEMYnqOlieOkbegY3bVkduePx9zCwB4VWifgUtb50MeYGg2QmORGPWWQAeKYL+rK\\\\\\\/K4taRsg9fBUb5OXdO98jgxLhQCXFOpNIQ\\\\\\\/BDhcHyHBGIKmB6FJwGfpsl13dqE5FjyYzdRnlefC3wTGVq9KADxbTw3H9bq25iWw7RA1CrT8BqpwLG2Akm1cmT7lBPAnQrWV6FHwUX9Uyz46VKdvLBDz6K3zXU0Ixfd8meSCeKKUluEE+2GYkjzlbUbm7ui6KX0o\\\\\\\/+TBfCbkeDLLitSF0a+WcaefFn8eFkmlp7xzKsAtbDl6wuo9soBPDQJJh6rp6eLlq7zhea\\\\\\\/shmSvP1DTuBvGYbWeQBJu4tMDpzv7\\\\\\\/WXtPdvHXHwK9NreBoZ6QX4GHKUvu59E6Uqt5wRwh90fcL3KdywgWEJeVjX4IRy21DZIkjtDLjI2cYZWdZRP+xHodxlx4\\\\\\\/zDLHGKULcQJsAp53c5Xo\\\\\\\/7FrNSIUnP2hxqLMFbFTU7yXOD56wXLWXfw\\\\\\\/Su0Gsrc1eO5TQH5D33toMfkb5DAHrsA0bApMpJhhLFyo48zRKI+fSwVZppt+Hw5LmwOoLICMUWfIFIH43Jzrk\\\\\\\/D0S5KAe36dUUFwdf5s\\\\\\\/\\\\\\\/8L9NTa04cDDEV30zVGvxhSAWiurGGxynwRHX56dKmdTaD9Eve1UjL1tT2Uat1K1pqwlShogUy1P0Yk0OdSmN+DRJ+jMd5UbWS8w7BOgIa+9iV8vFtexG\\\\\\\/\\\\\\\/KGh\\\\\\\/o\\\\\\\/SVzDJrvNd\\\\\\\/VQlRTV9LEODfRg47Iz7Sl+Vpg+R2OW\\\\\\\/3nPKLrIsBvLah6rJX5TITs4pBVnLwpGg0gZ\\\\\\\/+QiK\\\\\\\/trjXoFNOslS5O\\\\\\\/+tVT0jyUOd7nUND2BQUw3Rk7LDDaL+pcLAEHgjOl\\\\\\\/Q4aamDB4l9YJR5rpHpMzGwm4o2SlQWg90l6UA64GqnRCvUOxH6aIv5Uws2e1BGLR0LNXy6bSVYf2z2Vsf8GLXBKq39FtvrifG5pH6gI2fZwKyi\\\\\\\/RySt94g4WeUWCaKYrfptceDWP4TcWSBMGBw+z4uz\\\\\\\/0z7RKwftP3+bunPTEBVpDSuShnRewmJ+z1Pw4XIbnx2MooeEqKncVSoBNSMXIIKVVf2YqU1WP7Eyd5GmbNjo0JtUWYihzXYoS7xrk+H+wtuU90GXgTHeIAsiOzkUd4Ldgd+kjeWD+n4KP16luOV+GKdF8hmrFtSJkpqF7N7aq+7IOb1pTpENQkU7o6oHvCQDWU1Pl0YC6rbzjCx1HQEPf04A403zzEobm4tXg1OUMeOtUJPE0bVMVWLlHsetIcGR1dW+5tGsbvLxvpXpoDYE6oKQiyDINw+2k+wpPnO7RCN9wyOIg1Y0kJtquaCvs3cyuESTN2PxhzL9ktdnB\\\\\\\/MmEHtdK7JYbEbYo8+k7aZ4Qgz7q5Le9UphNQUakK+5UangbydOuj4AA1jFdpReKjlywrNmfiuuDg96lQJ+\\\\\\\/L1lDiFp03VJr42HyTaec+yltKcP9x95OZBY0i0fcZDmapY7aRoIwTMex9yYvXuutKrq0E2vDJdp+xeNBYgmYrO3qTj9CyPxJ\\\\\\\/Kc\\\\\\\/BxunpX0pA18E\\\\\\\/rsMrteikV9InZBycYaBkVRwM6ORD1TO2ZoG1fA4hu6zSlHMF9wzdYVLwSLAA+n7dA2FsUGjLwD61Lb51jeL7C1vWjLjjDm7\\\\\\\/d78eO1VCS3K\\\\\\\/GxNZhx4Nw57gWRO2UpIAZ8Vo9xlsdrz7YfD0PQ3ixJCfP3\\\\\\\/rXaStd3fxbr8dEteJPytOyQXDgTbszPjmkfKxSYDtzpHc8KqWyvJE\\\\\\\/\\\\\\\/AiKeES1430oEIE\\\\\\\/9qlmPT41zGF\\\\\\\/6MnIm+q+oRwS044kYEwIW1RHcI+Fz9jcgGfS\\\\\\\/P6j2JckpfgTm88p31W8OSgoYVuzSoTUEN9p0AT4ioHczTg2N7GittC5wU0pKTtv4aKX\\\\\\\/e6fiAQFdp9ss3NCNboL9x8Kw\\\\\\\/srqXuGmBN3H3VmQ7\\\\\\\/+nIvwfKEg370wx1fCY7n8zrrbhDJ97qPEmA4kQ7bY1ENMQKEiMDo6PFyCV6yV3sKVYLRbwnLDr05xspWVO8iaE581ofNOWWUq0NZTUZPFDeqN697L0yu0n05Qs9i+CHWx9lUEZXzSKHyJySQ\\\\\\\/u6d4ygO3\\\\\\\/Fl6HF\\\\\\\/AQgWQ4a+3r2KNenAv\\\\\\\/HjtwX3mHpItnevibHiCbqXNe4+qhE6pKaQiz8rVYvslRQqs7aob4BU\\\\\\\/08B43ontN0KXHnZEZoEfMTbc3W8rJO5ybGB6ANtP9VjO0VOanIoY81bGeg9tHxcp3qoZ2pvJzhqkWYB3TOQInf\\\\\\\/JpcVm0HoIeTLrCQtr2V6LYWFcjbUppkkkRpWI+ZxOnGQIJQ5TkAEvbVnmW7FZUEMhUeQM1rLO0G6AwGUzfLIx9lPanErEY6UAzbRKhY3m83mEvemP4iu0WDQ+lP8DnC+KEzywk4ZP6VFAr9Nlro7rhVQEaw2IydJz8ATqhf0jxpRCaBpeWE9liph8V7vrQSa5Lq20d9kFnEz4GKgKbuQkAibJeUvAQzTzZxCS7fx37KM3iI7XWXcHZV+BgnABbbv5qPrq7R9M306E97iTjBFbDeOi96AtmlkpPkRSmCphsCzrg6i9n3NNppjgvx6mY1WOqlpOliVLkchY8XAPrfITNT292620RhH9U2kmomQaHkRGsD26ylgpyBNiiF\\\\\\\/PDXCfeAVsqnQUG2EokckSUlGY4T0VVOT9nSTFfBQ7lFT07onzDhxnwyjXkLihaiABF0VA\\\\\\\/fXtZn+H8lMcDRxoMsNPrEB3tRj4VqlEQKXvRp9zZgHpaVr+wh1im8Xkd8Mr9hvepuFgjHawd1qeFjeEECVaxcLAqiPqeZzn0bghKDlJiy72Vb1yxvyEcbGCB+iewM2R6DO+ByngDqWtKKPV4dpF0cTHW7EotpjJJA8KBNWtveI6gRtZF3tS6w0RtIDL\\\\\\\/h5aJuxY384Psiv3DiGL9hId8UZmO\\\\\\\/pWeV6sNyJyIohXGPrzVuMDyFPG2QQQq8oTDChc7zFkJSu8MNx5s3OJ26\\\\\\\/5Oc5LVk0\\\\\\\/hDLRjLlWE28WeNVzectE8xjveee9z2a1OLXFNcW3BOXvN2\\\\\\\/hyBdWpiovKF3Qmlh8FMaEsm9ORH32ag3HFkK48NL5FNC\\\\\\\/x7D6j7VDW9e1PI0\\\\\\\/74Jr0yW7kDNkiKIT4bP+QwhzmAAUoMvLjrvJFqJU6qudE\\\\\\\/p14GHx4+rRbtXa3UNEICNmSeJSqW46XssBnHrSKAWY21L600ouZOV2FhyC3e12V5bnImGNysV+OpIJiXAYHd16Mo4F5WeAC42Q2LyCBPTdenj7n4HRMLCduz2aeQ3+fT703bNGRTxqixxbt2Q+6O72nK+JDKhEfWFKcki\\\\\\\/d0axmlyQFOoNQ6t8exKuyHZnJkrZ0Q8\\\\\\\/ipGrEj7dUDRvjt6g0OfxhiJBIy0QKXdNLcFqqRpkd\\\\\\\/pjTEo06ladvRx9zV5vSwZabxMtkkbMNEBmtKzCSamho3f6tpCKGbYxhB\\\\\\\/JIasNMCE7s5cB5IZeFFaduVUGrjoLMRs6\\\\\\\/PpGNkKVp9imkfoHDm1632LwpqVbebNK9b7ARv1WH0alYQYOSxZ13BNjEkgC\\\\\\\/rTQSWzDeqo0KaDsa6pNc+y2X4g7b5I0RJhkE8JnQ7Ap+lXxsDQSF0veRQDDbeQKipIUijxdGIoX9sHtofQgkwHDw0lRPmHV1p\\\\\\\/2nml3nV5aaQ4111JkufEFYLWAidXanKCgGTZGAmbqhIHRIdc11ORUOQagvApz4NeDSnYC\\\\\\\/3t+ZFcOusT8d5TJ9xiSvfIzOyGh7dZVPgKp0kyV7BPPTrekQFXo2OrsttgqRCnzw7wN5dR4CYXvUkY4WuB+mNFaEXhVz7zZqqDoVNZIu7k7TypL+NFZ4GgrpGwm3mkfhtaw5ErBUOzNarn1k6sy751n+0nKbgQNseNjuwDEfQvrm6keNjHG81OjM+zG3B7B2VUrR9KqlXG3rxA6K0TD2kQaWT\\\\\\\/cjmjzrNUgjOQKH9ZzayuTwf9jii9IpzlgpXd5ixtxZ3Tajr4rhue2eidFEp30UCdYT3MbrjcZEA\\\\\\\/Y0QRWF7tXtAKuxDiUbEYdgFGcCWuJ3z9huVWoWZbhahGdSKiDvQ\\\\\\\/vTqXKJ6lvndwoBQIvJD\\\\\\\/\\\\\\\/DziXpYb+uH2FCgdTlHSRqxlqLfA6eFCS05GFRERcIjIriL5HNIUZC8Vh7Ig00sSvK7vxecMZTM\\\\\\\/b\\\\\\\/5NOU3JimpwPEet5befkOsggIndxB54GNboMVa7T\\\\\\\/LHzEu2bWCSEemedYAM9qVhefLjHo2unO6ccPZKGyMFuHlsntcIXmmwN9+imRR6ZItyDrHwFanXhOJj4gSszKO0zU\\\\\\\/HTfpYfxq9NUqxspPm97NQKTHlWGkYowZfHL2N\\\\\\\/Bq6NPc0cygdVrdjlhVQnYz4Mu3wb0C7tl+UVuDtRoaCVBh4V6sDweWimgERTaHI3qYSTsFzHavBVx\\\\\\\/c1827lrgb7RnQ025ya5uQ8qoDwjY1MtnmGNyp2wIEa+k9w7t0RlAPppizP0MIfOahmJRJOb3NoSA8vIeyPbsDrLAUQpnbvFgG7515dJ2qlNfTK26O1hXgochW6f6sJpXiUaulqj63l1LL\\\\\\\/1jiNIDZZOnT4Dr1Nefaj5E0OOric4N3OMlepxlHotokt1KhI831VpryXei9BjolFdk9lxR7MBv7+i\\\\\\\/oBRPIQFFaLm8pboACn09en\\\\\\\/sPkItDTzOghPyRZ\\\\\\\/Ba5ZkIwF591rgIMd6lquH3ycmv3qEjtcxmz8vVgR3+uCZutVibeLzELJ+\\\\\\\/dmR4MIsinQaj9zR\\\\\\\/Mv7cEoQRbVHbShWgvwu49vaF+FafqGn6LmI8SoT3z3cCeOdwqj2Nyf1fnuD4snfD7c8IW+5oa0TTW4AFEzIMz4\\\\\\\/tIsHTi7ybMn4YBBA5NUKILYrT7QdGENbEIzma\\\\\\\/tz2tFmizs0VqGRoU6i2CPXL4Lpy2n\\\\\\\/rV0rdZ8ooMMEn00hmyjSJTjqrFyd1xKSlHWwyeTTbLIgoIuWaUzeMYPAdVxCO7idjmPfS+G9yoGngFCMudjcV+bhvvxHTw3AnlVQEUmKNS5mIClTAr\\\\\\\/xamUoUf5sDekJne0KCoMgvn8TZKWCyTElI1l7fNyAHCxi5JstKDMySkO5fKb+KEvHFEtmEwouQvNwAmsM6aFhe3+tzhANR4BGt0VQ\\\\\\\/np1ZcLpIDqsjJILBwya+kh0Jiu94M0baMt4K8ymDZBECxUnfCGeA9ckCvnRwOm5JlSad2K7Y1q\\\\\\\/EZiEfvKYMahbDf3mDGi4hg7upkAjkV6dliS8+i5Ii15pfG1+HZzH8YJszTXu1hAzQPSnKkjRWTxoqc94p4n6g8B\\\\\\\/ot2+IGJrS0DRFXbzAHRggw90a7zrxaYyufIRgfzfpfXbEUvGWXZ3rSx0tRxXGke5y9YA6k6bNxPck3dDpc+\\\\\\\/qXubLIWoYFezvCzhMDcSylcaDRkvsgCY5rK7cpe9zHJ24MPBRpBkBD9ebdWVOxEzJnWZz0b8L57gHei2uJyatxh8mtAqGP84ca+foJB\\\\\\\/081OwbBnjKlWSSKs4w3ZBjS4k42cITVIlX4Jv5xFUsxl7fkyKwus7aRA+EMU0ZKQCF00+vCeCwEjlwgA68MvIbB6cNaPQAVXZIPG\\\\\\\/X+2u9QwWaV5w7OVmPlgOaMWhsc+Lu1BW45KPe0EMd+lCnZ9w+Phec\\\\\\\/RNfRJOHRQbSMZ0q9plG5T+uPVcLsNTGuRLLyR7ROsqYBe73JoO4FsQf5O5KydERa\\\\\\\/GEQlNA\\\\\\\/TkfnsZT5g31UcYQvDLmqpyWeaOxmSzsZGki4z2+Yu51XQOeY27XgueUS+zWEwaMS+JL7yw7iGLb3BKGoY6oZfu\\\\\\\/3sDKCwoiaxvvcDlIk0F8MnuJ1XYZ6I4JBHwgmoRcbPfnZG6JBzR5RoH8x2KZYCpYR+aGVisOwbM+uWZjnwbqRzb\\\\\\\/ZJc90lL0G4qvDXXNV2LY+2s9tJ1cpxqo6LpOfOCoBKkXbz6nLhkmzhUs6jtzcT3wemp8jaflgSQxrOI++1QlDZOnbWhrX2FpbbuaZ4QG2lzszvHHXDdChsx2mHmMZKNSqXKGgxV50e6oiXKSdjlaMakz\\\\\\\/9s2WCm3j4ekNaKCoqEfZSqsDk+B0iFTbtNeZK1CbRwkrMgYrCpQe405om341Hg3IrADL\\\\\\\/RMzzAvCJNOXKpvUcUbkQ+zLgtdiEhotzbkO\\\\\\\/pYtt5PjTK\\\\\\\/NP6UydDmQZy0W+L8tGr0lTymc76Z0lnwBEzc6ZP6Iiucl9Cd+gVV9Ovh44c3vqy9UnpZQ7vWgizBlLkqEwX6KbN3VuQIqTx7q7uZPI4u4IZS+7CTiF6am2aZFTZYHdxYa1Takso3FFh4G8EEtpitUJv2qX7KHRPWTNc+H9jYF2MWiWZGXE2ZJmZAX90UThFzB1njn8XU55SdCUVRMrXHwUAhNdB4IjPgY0iJftvRwD3kTGW08V6DkYv8r7uidR2twooL+Qp+LAmkKRzLE8wPoOVWaCHizqnp5s1tKT66qG+fhk63wdwBjBGUPxdLfkxGgkp0Ac8FzbwlT0UerTCtV+ri4Nu0Cti4eSEjhtE\\\\\\\/1SsQVMHKKGYHweZPT6w1s2tNVHGb2UuSPW7Tl+pBk4E0s5vgUIwyzM\\\\\\\/\\\\\\\/uUwht\\\\\\\/jGGuNhzGVRtJ1vvCLw82n5XB9HOU035M8ARy0fQwSLRDTECiITjcqDZ+NFCjJ4YFC0hsaSgBB9AJk65nxvZURzbA85R8RL7O9\\\\\\\/\\\\\\\/8U9BE0D2n106jcPot0qFjNvKAJCa6v1\\\\\\\/scYVIoaXJyZb52NmLkTNQPnFy3PgEfZWHZhcIUHDe5bophPZQDZJJ3T7+b\\\\\\\/o2pFYEx5zH5sc7T5wWD2u2+6EgPGJ6mMB\\\\\\\/Lux4fOyFVsHOgwXY+WS0XGQj2SYoDMTFoYUdmhoXhaZNN4Hcwq92zNPzDyFnhfc1Klm6KDqxU26hcxo1s9ytkK6\\\\\\\/e3n5qXctN9X2SBp9DZSCYj4uVuMLXnba+wnbCBVnABcZVsni6lcRHw\\\\\\\/hIwoaAq9eThLJvJEqX6AJmFf63uQtvhfLyjxXrxAU55EEgLStSqCXR81UHCAPr5doLUFin8nZ3Sq53T9y1FMAkiNHqQqPRlBU78\\\\\\\/usb\\\\\\\/SxUNuobv2KyP9aFspWsKIZ9l8d\\\\\\\/Ow\\\\\\\/bKyP2MSUFqR2U4evyOjErkxMA69ydGDyDwuNf4r8YyBGxMvABJTGBaHOwgBZNM0MP9QSvtyLlZOZbkuyMMUuvfTec6FLN\\\\\\\/cu+\\\\\\\/u1IXW0CiuplXDUHWtyNjFs94Z8dBOvxjpjuLu+h2GpJ4J4EPx\\\\\\\/4ZzmaRu2Vq5iMvODOIiFeATGDd4iJURlieL9nOtXrEvxkUe99fvMvy93Tb3I+AyUdlFpq7wemTSZ5QXZfmeTgPkuLz1NykWQo7yt6eqtwlCVUH9q3Tv+FlxVrJCcfTWZ5wl2q0BPTZo\\\\\\\/\\\\\\\/Lu08xK1HilH\\\\\\\/GAyJnsjmgSA8C3HPPgBn+PMajh+VLFfWen7rnJsHGZuhNvYL6vAU6+z+1giqTFuD8zral19lNuGRF1pkXbkHYtA1aGi35jchbefiXS1rAkvuwUK58xFx6mMhkmaIXrwrF755U4dLtxJAODi5DbbmJxb4GiPSBD5eC7ubLY+TGfOtjJxbKH1\\\\\\\/j86TShGzFLye5xgJ6OaX8ApL0qCpYTIU9DWo441AD1XeKa6W7fMMS9UpYAHsRWgyICZ9sdfNomI4iQEl95GCqiJzJlmTn6FmWJHZttUGc27A3M8BvAUFFaDEkinFp6u0WmdByuqedQ6XgzFk5rEU4XZdah1DuvsC7892VH80i5q3Yyf5+FjGe7NIego31oGdQnlIvH97\\\\\\\/sSNurd2\\\\\\\/Uie74fGpHi5+wvvybXlYBpu4wD3ySwl5kqahLhv3\\\\\\\/jJ75aZEJSyj5j4X8ek7SkifG9Y+5m9D9LOqNM+TX3n86BJNI4Dz1Od69c0W16EMmcEyqnOIk7biQqZW4Mpg5xHTdUGiVwkDlAw83qlw\\\\\\\/Yl3C4yAix\\\\\\\/7CJTerDo51xQa9N0duF05NM2JGiXQe2OPhJzQvlf3J8+o2wxim8wrP7Q8s5\\\\\\\/+P8MuDJDh6yxuEhqagcFJaTlGgAriut\\\\\\\/IACzBqbCAjKatQBqhGycFiJna5o5LmS\\\\\\\/6vYyG5cdKQBU0OSn8fYm\\\\\\\/yfYvQrPuvHlbw+hhvHOxPDxEIh4aiwPmhvC1XFUStFXe3vW9pivXohoxpREZa\\\\\\\/MMIxi1NUR0cgD1doslvKaXpAarzVSLg6fdyMINNTjygtGEUVhvYEqplK11w\\\\\\\/sb35\\\\\\\/x5P\\\\\\\/pZ\\\\\\\/6k5\\\\\\\/yHZJul2zV3zUVVFwDdhtBqDoIWpMMQ+5tNI0Vt5H0pgrXMc+OjXFzsOMmrwSSe2W+ZJ6FfmyXnNPSd\\\\\\\/y7mKMgQudbJDEt5qqwQQ\\\\\\\/m8F4UmPwZdQ+h\\\\\\\/siR9qXRFdGIAboucZgJXdUHaXPVQHFMaFe8Dj\\\\\\\/MvOlrr+AFD4qTrr0PeZHLgP0sPnFYTuxcNrvOr9zHhfCbynoeCjGPsxOGxOMPCzGkGYX5dGj\\\\\\\/hb7Ci+XZaRzq+8+e99MMy2MNaraVarYdjTjvu3pHrrA5UyVSsLu1dZ1rH2fcg29ekr8HJoIikCyMMGFD3RNk54o7wQhFHUewC9pS67EWxGAL4wDYwqd1dhFsYYQFqyKffUVKGM7ZQ0+713xADnZLPc5iBQ3QrB7a4YorIUwttqgokeSA5DuPTEl6o\\\\\\\/2MuHuIvbIA\\\\\\\/mGmOVN3Z4DMrvGSMDMe1m2HQ\\\\\\\/hg32y9XOWtXHcTejo0SeNo33ON76EcJIbyXuWZvpea\\\\\\\/yFnJnIOMuApqPMzdHXR\\\\\\\/G5hZFt4Voil58jYroKmUri7rS+T1y1uy5aPbvrkBV1eceYk14jO3vudkktZXsO015M7n+s6ZmCv3RxhB0S5ktAA56XQpXFgJQfYSC\\\\\\\/IAR1csUZy2iYL2QtnDaBYX\\\\\\\/UbQYTcgYLysHsH\\\\\\\/k4AQ208p4K\\\\\\\/3lF7QZpnmjl06VnP4NR2w5a8RJEp03kTHDxRyWzBvsy0IqQM\\\\\\\/O8r55a5oVOZqddqadqpGQm7yXnyv4nO05nQ6\\\\\\\/FaJ7smzV4XeZtKZ\\\\\\\/WLDz7eYd+akpRaU4uvl0KkR5Kmqc9nAg5ktNyxcPydLYD1kXG3OyHPEZVbQP93h6b57j2RulNQ8+d5mUZaFb0tymR\\\\\\\/gq6GC2utpX0jE4zc1OETQItxVuACvT4KjKv0EJmg+DYdOo25CU9v7wuwQqmDgXbFbTCrE0cIm8yJ\\\\\\\/RyMpRl3rRp2z3MbiOWcSuavDsn+kpd9i6MGFLO+NlxxxY0eh36I0eiH6DG3cyVpTIvq00pYE6J7tUunQvUIMxCFyQavXh+\\\\\\\/RP+rxIAmb5x81KuW1NLHSE3hJva3jwAfRj0+\\\\\\\/Z+V808Z1Pm41HuJ\\\\\\\/Svzo8nBQ7IKlYTXjrqtjW\\\\\\\/cyaczW1VaiNSrSCF1ZEImbn7Hg4efUHlNDbPYQi0QaUCtPkt6DFsIKMdjJGo3qCRHIHptC0J5846e4TLOwOf9HKJK0TH0vHwlIFUsGdnaqoyL3uDpU7G6YQNHY4aibMuaUG3YaoKPWtFfvZGUoUSYDJXljARNC+wa8TvIUyoCP2SuwTJDUlfSOFDIg5Cpk6DZZBW7CbX5EcvoBmYBkZbGJzUQIfP\\\\\\\/BS69Z6h9MvNFMUZdVMIgiqYxLT9agfKPV8ljdhEf5OYQ9+6Bs+CAmFEvJNr+Qzrd3Kos32DvMRcxqOIvjhjBkE80twbmKVLBD3XfuqC+84t06Y9kOt9Fs8Bt6d58VXFvELVcdcPGUQjdqHbroiSeBB\\\\\\\/z2q6OUngM+RBMjb4EY+w6AxqLXExqhZsz7uJi\\\\\\\/\\\\\\\/dJol03V2uLDVxzeJX+ZfajKZSOnb7xt9iKBzX+9Z0KQkfQ98kze4eiZaYgSq1IPPfqF1HNcPCMAhyN+np6qxJIRPnMqwhy5+BprzGBo0fmFKPljSKslVjkraZAyhl5w\\\\\\\/XnBz\\\\\\\/5+6mRRDOs0GmblWKPMi5qOEKKpG5rcm9GMdljXwZom444p0UC8q0ECF9KNTfSW5Xq0C2aRMR6aBRJdcMAnNukcBpq4NOLupr\\\\\\\/dvXq1ZmFe9\\\\\\\/XzCv7vSPC4CoS3gdNfZKcy29t10NiLvWA9K0WcH0uc+XZqyzRoEhkpitt4U8CRgD76H8RWMF0tO\\\\\\\/1S2KjbixQyTiI\\\\\\\/ZHPrKgAFu5icCFrlXTzKYZE0BqBNwFD0qafQ7QwTXkV37Nmqwew2Tf8MJwJTMpkejjPihLWmO8Hi84JoGcq9d0mkdJTnwm8SZ\\\\\\\/Drv2WM9F1pLaDRmLtkV\\\\\\\/LlT407FW3YglWHEGuthXY0hP0CuPhqy4gWV7LlSlyfTHK6xCoPIZ4R+4QAVt49LeN3FNd5bFWLACnYbEWdlrcu3OmLk\\\\\\\/q\\\\\\\/wkLyy+q79dL8d173ePQQooMydHEq+aTUUqOJTyZC8AYLReguB44m\\\\\\\/CLIEX3mAQTlvOjetVhhe5SwuTSuHvyjadgdB8UjlFi\\\\\\\/w9x6tM4x1Tef3dEYm36wYpuh+TvSSzBmYXKthK5aDsHkih5roaAcPlkKu4z2QuRjeSGeNeu6Ce026+8Fbx6l354sGrvvkyADxTdnNvaXdozPe2w1EsXFgpwK9iCVhl6mkzbHtW\\\\\\\/LZqEtwC\\\\\\\/DpddpeK059KtSM0mbh\\\\\\\/uxnaHtZEliiFnRLTt\\\\\\\/8+ihi7iwZRy\\\\\\\/4z1JvGfuA\\\\\\\/dl5spb9XOk77SJROJRccxWNjxIjWaXOxPU0V3X3SNmXrflV9kcscZGjt2c6d+ENzfjjvA2oCg4875EBmB9Dch7QtKT4ehvkNWIHvH3BIEtY+dfE2pLRGOi403VX0XLCgj3O+hUn6V3Jnt+pAzP\\\\\\\/9Fcekvv1JwMMHYVqjABa0Egv8ONfpg52Jhoc\\\\\\\/YPTDd6jIFNbR8Hj6e6rtPBywzO89Ehiyfutw1lxhCuAFlh3nchEd3W0yVpmsimMA43RUxeaqUIIzuQaf1BoCidUbHW\\\\\\\/p9yTwsExqdsdyUIvaf9i8z8OTJlPUmEdl6wZnHWowxS03chLAMG8L0zYt2ftKV3PH4wo8Wf3fbICjM5bPx2FuABcu9WBG55OJB2xpjY0GpVim1smpYU75RnbWnwS9iibYCIW+MkNYVF\\\\\\\/PXeUD46D00NAvJN\\\\\\\/5XToStvG2Wv8d5Ojy0uEfCZsPrMI39dcXoaJtIc\\\\\\\/jH1ju\\\\\\\/fZ24jTTu+R5bGBI8RS5sq4lXLNAtM5+DHmwTCtx4S0nI3w5boHk+JStSOBaKDpf7RXEB9BZ3on2cvLh8lnzpOw+TrHz80SXQzCiDgES8+PjoIoa2kp46mMiUDmntF40Dq8vr2YJhoE7c9u9j9L+fzfZi9wswFPeF1r1QSeUKGeEyGk+M\\\\\\\/m5Mz8ZaDjCq0RTH+VJrmCmBtWUcMim3dN2G9SV3Its5A5I6tYdkKp4U32Z9Wmx5Giw995GpMmMJgr2B6FTx8nWsHv+VI0qW+DPWXCDENj18BOlYRXRaFzCpM8fahCrPxQwfOi4GTclc7zwh2IqudpVLx1BjvZbpoP7pqMgfZ6p7JwfzA90xHzVddzoKcz6GebBVJsgjire6iJM5QwjrXRzOlaP2G7V8aZ\\\\\\\/E4vMsIXwNWB7K64HbY3K3i3nvwvrAGI8hnpzaf8rMLedceV3lLhzAcar1qCzs6ROnMW7wtYIQzP4m1Pk1X1vxj84k5r7a+YSJL6d3z1efQ3VjdNwYZm7eFD9xHaVcD2gmBqm0X6v8F5ONV6OVnPQbHg42U\\\\\\\/8k\\\\\\\/Mcqo8LUF98KVvtF9s2pLj0PN0o0XWSDmuMjjQcCpCtQGZV5fvKKELf8QYCoDWQIaNblG+uL7fp4LcH3cUW1KxflOpLKE3oPIsiXBj+BDa0+Rm+nYlmMhs3qdbx5+sBrV1bt6CdQoVYDBVtHlAqcF7iMGwv6Y3n6Snk5w58EV3f1g89eXCbXM\\\\\\\/AkLNDXN74PdqiBW917ZShTmV80lxOMmCaaqnoXkRVbYURN4nLdAmcvmHOEBG9\\\\\\\/ZTWo9MHfQqi11KvYL282oT+aoU0lEsfc7OyJh5z1akNb6oaFeSou6aN5N\\\\\\\/5bN\\\\\\\/LSli\\\\\\\/KupwyxIMRUzgS+1jaPQQZB806dXLdQaHq+GMu1yWrVolWSR1a\\\\\\\/CvN+0M7n\\\\\\\/MsS6MEP5JU+1Pqg2Cwts4vdmVFzfGYSNSg85w6CgPMoprtxsJqYrdAxIvBO4f\\\\\\\/voW4sCjDjrgk8TJVRe3maNfIw\\\\\\\/owvjuUrP4GVA3nUf7JVF0mIjOCMGCHZ3Tt\\\u0022,\\\u0022iv\\\u0022:\\\u00228RPmFIKCFljYT6DdmIpnHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bba1b0fb86c96a81b1908a87e430565e0b4e08451111bc959abea6329134bd05\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ui5aYGiR0tYp6ADOB5LwKqTZr7UzcTxBa4d6SItMFPKyq8A1DLMNTx3JINqcfudPyfNRwzQSYSAxm\\\\\\\/U\\\\\\\/n6Yyb+tMlfob4RSAUiYcHuuYzCC0yV85jEp5+3PwMPceQiaWblPQxTYlr1N4xvCjQhJ\\\\\\\/j\\\\\\\/TYayX+PdKxZSBqYJwh515FByWW3FC14BqXkzxNVXwQI+9NDNQceKiehONtzeEDscug\\\\\\\/eRSoTYJQ8YJAYoGacxv3mTrxEdnIJgJ5SxNrRTOPbMlxvn8GPhI9r9qVQyDA6j\\\\\\\/wsnVx0yIFwFYV5Fvb4LGx0hbcgHxyLef5zA0BtuFzD\\\\\\\/NiMFG1UNC5r\\\\\\\/Y8e8hdeQH9mCjiD\\\\\\\/f1l53ORK5VC+rUx5Hpbm6JApG+2SWAXLwoVkfiB7dxwBbk40podW8c2isNDhURF2n517aDflmeJTAhdxsfr7oL30mXZRUhOJ53k5fZRX6JWNnK+M+j3cE0zaqYwsRaRpzp+ScL61CVyXYwoaummtEVJsDWs55TOeNybW0ObOqBY8WWcblytNqvlNFI3EAxQT4rwWrneIoovxzzMkXpEVDwcjQ1ccIbuwICQvbTkFsw8BT66+RQzuL+PioKWRofIVaKB0XEpp\\\\\\\/v7bwYfUcMvESAeiygAC4RC6+FUQbBno9n13EsxhVz1qMNziqa1sY8fzmkKKq9bBt3w+jxxdpZcADeYPKHbHpqBhjAFMHCa8o\\\\\\\/xYQGIlrgwwkXuul5j2Y2WqZwldzPpuDvWCC0AVx8X7sBM7l9UDIsHMQumX3cSmbt6ZtSy7YIKXQz0MkALEFYWrHhSFrKaKi\\\\\\\/4JjMnO1XIr+sh77WvIoqB4s02VW5wT586UEk2CXuS\\\\\\\/JOMkE5E8p6yw9rJUxfjVZRUxUvJlQQAKThVxDU8Yec9Wmv5ZFsgs5i8+4lxylGV6vB5Bi0QQwg1Ue1QPiKvZFmh7BbVmx5RZvjE4tGQ+8nkqSy8GfXhATnHhHFOgQxccf2+P51d1ZHPzQ5g27dLmLRNF8dLqac3y6HQDRUQkwxA9P\\\\\\\/2X4sASfV3zkk5t3tlv0NjGra4Kgi4QjH7ETKcL3aK5+vMecoFPtffknMvnu+9ZzeDv1PuhJQA2B4vP0grMvSuY3kNPKcRyWZdKu9iBnLsGfOJjCWoULxBSa1bN1Vd2gzzY0iqpyPuRysI8LkrCw5MlATymTg69owq\\\\\\\/Zak94OqHCNL0rhx77wGT2u1f8dqyJTkuja1Ryt2ella0FSeAQciLm0XRbbe3\\\\\\\/LHG2uyjri6z\\\\\\\/Va4Zx+onTT3ztu1ZSQQDm6TicZWkgtnPEIB48qck0e2pXhOJyXIuLzsOY04Mp5vIyzDubPFNe4IoHMxrsnxbA+vcaU0TAQ+\\\\\\\/5gucspcPQqq8h\\\\\\\/KOc3b6MCttx1gKM5XVRQCjNpUTJ8WL3ixrGC23BwhsmR1K0oZIK+wENe56jS1xAw6RVN8LU75Q\\\\\\\/ncpf4u4KfCrjLbwolCdTWuNLymrUWQ9jRkgsB4Jjy6RUnE0uw4ja1lga9\\\\\\\/hOs0Sb8GtIKoCs+jBdWArImcDPslNRg6fQRxZXufwGaIDWaJjCP2gSrUi1xoYvqvtRffRf4x2LHn0yjUk2o640FwY01m0oPFfu1B14tq2wC0TX4KSqEazTWfs0+JBNZQ7zBYuyD6dHSx0pqe2eN7V8FS8bJcWEDIJw8cYKtoe5FGKIAETeFhJPzoDGRF2MAyZlCODpdEeYFebCYeyrus0zud1lqAGX\\\\\\\/PAkdopWqNxfZq9dTGau8\\\\\\\/4JM2cxn\\\\\\\/x07UH52\\\\\\\/V4OlX5vy3e2eh7dP\\\\\\\/WLhATwx+3+oVZf3cpYAVhcNH0ao1sgRCq7ndCMUdFXGI1ap6qgwjgqQtV5jsDNZxvNHF76I03zducaViErfpPV3O9eVFHT2F1sIoCaIyvPy9BqNcQw7PN1U\\\\\\\/+kf\\\\\\\/V5uJ1Ub+v3NgnQlbH3RfwsEAqVoRMPBsuz0yC6C87RZ1gZMYGUCCxBbpJo57H+6pJFXUb7XY65B7oWCoObxIHDcznGjApH3YkZ3fvC8ytVT7oo7YVtTobUeiJxw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022wnxBV3\\\\\\\/0Xr9ztEdSy5UC4w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253638ac41b9c7bf88fb19a535aa06f43a65752464ac3b4fe7563604997d61b4c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459306 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/883" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-__y26TpOq5ZhZWRj8fl0fiyPp1YcjL-z2CgvdgedVHM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/883" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-c4mwpD5j168tJG_WFQEAG3ahuEoWOZGwjg44nnfFbSo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/873" title="Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/891" title="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459306" id="node459306" data-pisa="ggg;2/8/883" data-pisa-master="ggg;2/8/883" data-apath="/ggg/2/8/883.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bernhard Haubold</span> and <span class="highwire-citation-author" data-delta="1">Peter Pfaffelhuber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">883-889;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002527 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459306" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bernhard Haubold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BHaubold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haubold%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernhard%2BHaubold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haubold@evolbio.mpg.de" class="" data-icon-position="" data-hide-link-title="0">haubold@evolbio.mpg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Pfaffelhuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mathematical Stochastics, Mathematical Institute, Albert-Ludwigs University, 79085 Freiburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BPfaffelhuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfaffelhuber%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BPfaffelhuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/8/883" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459306" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/883.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459306" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/883.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459306" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/8/883.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459306" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459306/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Comparative sequencing contributes critically to the functional annotation of genomes. One prerequisite for successful analysis of the increasingly abundant comparative sequencing data is the availability of efficient computational tools. We present here a strategy for comparing unaligned genomes based on a coalescent approach combined with advanced algorithms for indexing sequences. These algorithms are particularly efficient when analyzing large genomes, as their run time ideally grows only linearly with sequence length. Using this approach, we have derived and implemented a maximum-likelihood estimator of the average number of mismatches per site between two closely related sequences, <em>π</em>. By allowing for fluctuating coalescent times, we are able to improve a previously published alignment-free estimator of <em>π</em>. We show through simulation that our new estimator is fast and accurate even with moderate recombination (<em>ρ</em> ≤ <em>π</em>). To demonstrate its applicability to real data, we compare the unaligned genomes of <em>Drosophila persimilis</em> and <em>D. pseudoobscura</em>. In agreement with previous studies, our sliding window analysis locates the global divergence minimum between these two genomes to the pericentromeric region of chromosome 3.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/alignment-free" class="hw-term hw-article-keyword hw-article-keyword-alignment-free" rel="nofollow">alignment-free</a></li><li class="kwd"><a href="/keyword/maximum-likelihood" class="hw-term hw-article-keyword hw-article-keyword-maximum-likelihood" rel="nofollow">maximum-likelihood</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila</em></a></li><li class="kwd"><a href="/keyword/match-length-distribution" class="hw-term hw-article-keyword hw-article-keyword-match-length-distribution" rel="nofollow">match length distribution</a></li></ul><p id="p-3">A central goal of modern biology is to explain in molecular detail the relationship between genotypes and phenotypes. The success of this research agenda depends on intimate knowledge of phenotypic and genotypic diversity collected from a wide variety of organisms. Historically, knowledge about genotypic diversity has been much scarcer than about phenotypic variation. This is now changing rapidly with several projects under way to sequence the complete genomes of 1000 individuals belonging to the same species.</p><p id="p-4">Quantifying the genetic diversity from such sequence data is conceptually simple: after assembly, align the sequences and calculate one or more of several well-known estimators of genetic diversity (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wakeley 2009</a>, ch. 4). However, calculating alignments between genomes can be cumbersome for two reasons. First, the sequencing phase of genome projects typically results in hundreds to thousands of contigs rather than chromosome-length assemblies. Second, genome rearrangements disrupt the synteny implicit in many alignment procedures.</p><p id="p-5">One way to avoid assembling and aligning sets of long sequences is to restrict the analysis to mapping the sequencing reads onto an existing reference genome. Still, the sheer superabundance of sequencing data has motivated the development of new computational approaches even for dealing with the comparatively simple task of mapping short reads. Some of the most efficient solutions to the mapping problem currently available are implemented in programs like bwa (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li and Durbin 2009</a>), bowtie (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Langmead <em>et al.</em> 2009</a>), and soap (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 2009</a>), which are based on recent advances in algorithms for indexing long sequences (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Puglisi <em>et al.</em> 2007</a>). Such algorithms are optimal in the strong sense that computation of the underlying indexes can be achieved in time that grows only linearly with the size of the input data.</p><p id="p-6">To take advantage of these new algorithms in population genetics, we have been working on methods for quantifying genetic diversity based on string indexing. The central idea here is that of a shortest unique substring or <em>shustring</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Haubold <em>et al.</em> 2005</a>). When considering a query sequence, <em>Q</em>, and a subject sequence, <em>S</em>, a shustring starting at position <em>i</em> in <em>Q</em> is the shortest substring <em>Q</em>[<em>i</em>..<em>i</em> + <em>x</em> – 1] that does not appear in <em>S</em>, and we say that such a shustring has length <em>x</em>. The average length of shustrings decreases with diversity, <em>i.e.</em>, if the shustrings are long, <em>S</em> and <em>Q</em> are closely related, and if the shustrings are short, <em>S</em> and <em>Q</em> are more diverged.</p><p id="p-7">This notion can be made precise by deriving an estimator of the substitution rate based on the distribution of shustring lengths (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Haubold <em>et al.</em> 2009</a>). <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Domazet-Lošo and Haubold (2009)</a> implemented this estimator in a program for quickly clustering genomes from organisms as diverse as HIV with 9 kb genomes and <em>Drosophila</em> with 170 Mb genomes. They also generalized the computation of global evolutionary distances to the detection of local homology between a query and a set of subject sequences (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Domazet-Lošo and Haubold 2011b</a>). This is particularly useful for the detection of horizontal gene transfer among bacteria (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Domazet-Lošo and Haubold 2011a</a>).</p><p id="p-8">Recently <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Haubold <em>et al.</em> (2011)</a> have begun to develop shustring-based estimators for population genetics. A first simple, but powerful, result of this work is that the expected number of pairwise mismatches, <em>π</em>, is approximately equal to the inverse of the average shustring length. <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Haubold <em>et al.</em> (2011)</a> designated this estimator <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-1.gif"/></noscript></span></span>and used it to locate the divergence minimum between <em>Drosophila simulans</em> and <em>D. sechellia</em> to a region centered on the gene pickpocket (<em><a href="http://flybase.org/reports/FBgn0020258.html">ppk</a></em>). This gene may be involved in the characteristic preference of <em>D. sechellia</em> larvae for the fruit of <em>Morinda citrifolia</em>, which is toxic to other <em>Drosophila</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dworkin and Jones 2009</a>). It took less than 23 min on a single AMD Opteron 2.3 GHz processor to calculate the local divergence along the complete genomes of <em>D. simulans</em> and <em>D. sechellia</em> (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Haubold <em>et al.</em> 2011</a>). Moreover, the genome of <em>D. sechellia</em> consisted of 14,730 contigs, which would normally complicate sequence analysis. However, the computation of shustring lengths does not require synteny and can therefore be applied to unordered contigs.</p><p id="p-9"><span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-2.gif"/></noscript></span></span> is easy to compute and is accurate in the absence of recombination. However, <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Haubold <em>et al.</em> (2011)</a> already pointed out that <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-3.gif"/></noscript></span></span> is downward biased if <em>Q</em> and <em>S</em> have undergone recombination. Intuitively, this observation can be understood from the well-known fact that SNPs tend to cluster along the genome in the presence of recombination. We therefore report here the replacement of <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-4.gif"/></noscript></span></span> by a maximum-likelihood estimator, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-5.gif"/></noscript></span></span>, based on the full distribution of shustring lengths (subscript d for distribution). In contrast to <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-6.gif"/></noscript></span></span> (subscript m for mean), which rests on the assumption of a constant coalescence time across the two sequences compared, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-7.gif"/></noscript></span></span> allows local fluctuations in coalescence times. This makes <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-8.gif"/></noscript></span></span> much more robust against recombination than <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-9.gif"/></noscript></span></span> but still simple enough to allow efficient repeated computation in sliding window analyses.</p><p id="p-10">In the following sections, we derive <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-10.gif"/></noscript></span></span> and test our implementation of it, pid, through simulation. We then apply pid to two pairs of complete <em>Drosophila</em> genomes. The first is an aligned pair taken from the Drosophila Population Genomics Project to allow comparison between <em>π</em> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-11.gif"/></noscript></span></span>. The second pair consists of the unaligned genomes of the closely related species <em>D. pseudoobscura</em> and <em>D. persimilis</em>, in which <em>D. persimilis</em> consists of 12,838 contigs. We focus our analysis on regions of low divergence. These are singled out in many studies as candidate regions affected by important evolutionary events, including introgression and selective sweeps.</p><div class="section other1" id="sec-1"><h2 class="">Approach and Data</h2><div id="sec-2" class="subsection"><h3>Shortest unique substrings</h3><p id="p-11">Let <em>Q</em> and <em>S</em> be two DNA sequences called <em>query</em> and <em>subject</em> of lengths 2ℓ<em><sub>Q</sub></em> and 2ℓ<em><sub>S</sub></em>, respectively. In our analysis, we use both the forward and reverse strands; hence, the factors 2 in the lengths of the sequences. A shustring of sequence <em>Q</em> starting at position 1 ≤ <em>i</em> ≤ 2ℓ<em><sub>Q</sub></em> is the shortest substring that differs from substrings starting at any position 1 ≤ <em>i</em>′ ≤ 2ℓ<em><sub>s</sub></em> in <em>S</em>. We denote the lengths of shustrings starting at positions <em>i</em>, <em>i</em>′ in sequences <em>Q</em>, <em>S</em> by <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-12.gif"/></noscript></span></span>. Put more formally, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-13.gif"/></noscript></span></span> if positions <em>i</em>, …, <em>i</em> + <em>x</em> – 2 in <em>Q</em> and <em>i</em>′, …, <em>i</em>′ + <em>x</em> – 2 in <em>S</em> are identical but the nucleotide <em>i</em> + <em>x</em> – 1 in <em>Q</em> differs from nucleotide <em>i</em>′ + <em>x</em> – 1 in <em>S</em>. Then, the shustring starting at position <em>i</em> in <em>Q</em> is given by <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-14.gif"/></noscript></span></span></p><p id="p-12">This definition is only useful if <em>i</em> is not too close to 2ℓ<em><sub>S</sub></em> or <em>i</em>′ is not too close to 2ℓ<em><sub>S</sub></em> because the shustring starting at <em>i</em> can at most be of length 2ℓ<em><sub>S</sub></em> – <em>i</em>. Therefore, we use <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-15.gif"/></noscript></span></span> and<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-16.gif"/></noscript></span></span>Our approach presented below works as long as we can neglect edge effects, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-17.gif"/></noscript></span></span> for most <em>i</em>. In practice, we simplify the analysis by concatenating all query contigs into one sequence and all subject sequences into another sequence. This means that shustrings can span contig borders but are cut off beyond a border after a—usually short—run of random matches. A large number of contigs will therefore lead to an excess of short shustrings and a corresponding overestimation of <em>π</em>.</p></div><div id="sec-3" class="subsection"><h3>Determining the shustring length distribution from data</h3><p id="p-13">The input for the computation of shustring lengths consists of two sequences, one query, and one subject. These have been obtained by concatenating a potentially large number of contigs. For the purposes of this exposition, let the query <em>Q</em> = CCGTT and the subject <em>S</em> = TCGT. A <em>suffix</em> is a string that starts anywhere in <em>Q</em> or <em>S</em> and ends at the end. For example, TT is a suffix of <em>Q</em>. The first step in our analysis is to index all suffixes contained in <em>Q</em> and <em>S</em>. The resulting data structure is called a <em>suffix tree</em> and is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gusfield 1997</a>). The defining feature of this tree is that the concatenated labels on the path from the root, ℛ, to a leaf labeled <em>Q</em>[<em>i</em>] spell out the suffix starting at position <em>i</em> in <em>Q</em>. For example, the path label of leaf ℒ is CCGTT$, which is the suffix starting at <em>Q</em>[1]. Notice the character $ that terminates <em>S</em> and <em>Q</em>. This ensures that no suffix of <em>Q</em> can simultaneously be a prefix of a different suffix of <em>Q</em>, a technicality that guarantees that every suffix is represented by a leaf in the tree. To look up the shustring starting at <em>Q</em>[1], we walk from ℒ toward ℛ until we find a node that contains a subject leaf in the subtree rooted on it. In our example, we find this node, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/inline-graphic-1.gif"/></noscript></span></span>, in one step. The path label of <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/inline-graphic-2.gif"/></noscript></span></span>, C, becomes the desired shustring when we extend it by one nucleotide to obtain CC. Our approach is centered on the distribution of the lengths of such shustrings starting at every position in <em>Q</em>. These can be looked up in a single traversal of the relevant suffix tree.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two sequences, Q and S (top), and the corresponding suffix tree (bottom). &#x211B;, the root; , an internal node; &#x2112;, a leaf." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Two sequences, &lt;em&gt;Q&lt;/em&gt; and &lt;em&gt;S&lt;/em&gt; (top), and the corresponding suffix tree (bottom). &#x211B;, the root; , an internal node; &#x2112;, a leaf.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F1.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F1.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461789" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-14" class="first-child">Two sequences, <em>Q</em> and <em>S</em> (top), and the corresponding suffix tree (bottom). ℛ, the root; <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F1/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F1/embed/inline-graphic-3.gif"/></noscript></span></span>, an internal node; ℒ, a leaf.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The speed of our method relies on the fact that suffix trees can be constructed in time that is linear in the number of nucleotides analyzed (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gusfield 1997</a>). In practice, an explicit suffix tree uses too much memory for genomics applications. Instead, an abstraction of a suffix tree based on a suffix array is commonly used. This is an alphabetically ordered list of all suffices in a sequence. By traversing this simple data structure, the corresponding suffix tree can also be traversed (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abouelhoda <em>et al.</em> 2002</a>).</p></div><div id="sec-4" class="subsection"><h3>Derivation of <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-18.gif"/></noscript></span></span></h3><p id="p-16">We wish to know the distribution of shustring lengths as a function of the average number of differences per site. For the derivation of the shustring length distribution, we use the well-kown fact from coalescent theory that the time to the most recent common ancestor of two lineages is approximately exponential with expectation <em>N<sub>e</sub></em>, where <em>N<sub>e</sub></em> is the effective (haploid) population size (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hudson 1990</a>).</p><p id="p-17">First, we compute the distribution of <em>X<sub>i</sub></em><sub>,</sub><em><sub>i</sub></em>, where we assume that position <em>i</em> in <em>Q</em> and <em>i</em> in <em>S</em> are homologous. The fact that our data are unaligned does not invalidate this assumption for the purpose of deriving <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-19.gif"/></noscript></span></span>. We further assume that no recombination event falls between <em>i</em> and <em>i</em> + <em>X<sub>i</sub></em><sub>,</sub><em><sub>i</sub></em> until <em>Q</em> and <em>S</em> coalesce in this genomic region. This is equivalent to assuming that mutation is more frequent than recombination. Moreover, we take <em>π</em>/<em>N<sub>e</sub></em> as a proxy for the mutation probability per generation per site. Then we obtain for an exponentially distributed random variable <em>T</em> with expectation <em>N<sub>e</sub></em><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Second, we compute the distribution of <em>X<sub>i</sub></em><sub>,</sub><em><sub>i</sub></em><sub>′</sub> for <em>i</em> ≠ <em>i</em>′. We assume that the two subsequences starting at <em>i</em> in <em>Q</em> and at <em>i</em>′ in <em>S</em> are random words with <em>GC</em>-content 2<em>p</em> and <em>AT</em>-content 1 – 2<em>p</em>. We know from equation 1 in <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Haubold <em>et al.</em> (2005)</a> and equation 4 in <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Haubold <em>et al.</em> (2009)</a> that<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>which for equiprobable nucleotides <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-22.gif"/></noscript></span></span> simplifies to<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>In this case, the distribution of max<em><sub>i</sub></em><sub>≠</sub><em><sub>i</sub></em><sub>′</sub><em>X<sub>i</sub></em><sub>,</sub><em><sub>i</sub></em><sub>′</sub> is concentrated around <em>x</em> ∼ log <sub>4</sub>(2ℓ<em><sub>S</sub></em>). By combining <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equations 1</a> and <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">2</a>, we obtain<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>and<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>As explained in the previous section, we can observe<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-26.gif"/></noscript></span></span>where <em>f</em>(<em>x</em>) is the absolute number of shustrings of length <em>x</em> for a pair of sequences and <em>ξ</em> is the length of the longest shustring. Now we assume that <em>f</em>(<em>x</em>) is the realization of a Poisson-distributed random variable with parameter <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-27.gif"/></noscript></span></span> and that <em>f</em>(1), <em>f</em>(2), … are independent. We can then readily compute the log-likelihood<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-28.gif"/></noscript></span></span>for some <em>C</em>, which does not depend on <em>π</em>. Hence, the maximum-likelihood estimator for <em>π</em>, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-29.gif"/></noscript></span></span>, is given by maximizing<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-30.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>One often needs to compute <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-31.gif"/></noscript></span></span> repeatedly during a sliding window analysis, where an interval <em>Q</em>[<em>i</em>..<em>j</em>] is fixed with, say, <em>j</em> – <em>i</em> = 10<sup>5</sup>, <em>i.e.</em>, extends over 100 kb. Then, using <em>Q</em>[<em>i</em>, …, <em>j</em>] as the query and <em>S</em> as the subject, the above computations work as well, as we can still assume that every position in <em>Q</em>[<em>i</em>, …, <em>j</em>] has a homolog in <em>S</em>. Here, we observe <em>f</em>(1), <em>f</em>(2), … for the specific window <em>Q</em>[<em>i</em>, …, <em>j</em>], and maximize the likelihood as given in <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation 6</a>.</p><p id="p-18">A problem inherent in our method is that query windows without a full homolog in the subject sequence contain an excess of short random shustrings and are hence assigned too large a value of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-32.gif"/></noscript></span></span>. We mitigated this problem by applying the criterion that if in a window of length <em>l</em><sub>w</sub> the number of shustring peaks is greater than <em>l</em><sub>w</sub> × max(<em>π</em>), the window is deemed “missing data.” A shustring peak occurs at position <em>i</em> if the shustring length at position <em>i</em> – 1 is less than or equal to the shustring length at position <em>i</em>. This means that the shustring tracked at position <em>i</em> – 1 refers to a different SNP from the shustring tracked at position <em>i</em>. Using the simulations shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> to guide us, we set max(<em>π</em>) = 0.06, as the algorithm worked for <em>π</em> = 0.04 but not any more for <em>π</em> = 0.08.</p></div><div id="sec-5" class="subsection"><h3>Implementation</h3><p id="p-19">We have implemented the calculation of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-33.gif"/></noscript></span></span> in the program pid. The underlying suffix array computation is based on a software library by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Manzini and Ferragina (2002)</a>. pid is available under the GNU General Public License from <a href="http://guanine.evolbio.mpg.de/pid/">http://guanine.evolbio.mpg.de/pid/</a></p><p id="p-20">This website hosts the C sources of the program and detailed user documentation.</p></div><div id="sec-6" class="subsection"><h3>Data</h3><p id="p-21">The genome sequences of <em>D. melanogaster</em> strains RAL-365_1 and RAL-391_2 were downloaded from the Drosophila Population Genomics Project website (<a href="http://www.dpgp.org">www.dpgp.org</a>). For the alignment-free analysis, padding Ns were removed.</p><p id="p-22">Whole-genome sequences of 21 <em>Drosophila</em> species were downloaded from the following three websites:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-23">The genomes of <em>D. grimshawi</em>, <em>D. mojavensis</em>, <em>D. virilis</em>, <em>D. willistoni</em>, <em>D. persimilis</em>, <em>D. pseudoobscura</em>, <em>D. erecta</em>, <em>D. yakuba</em>, <em>D. melanogaster</em>, <em>D. sechellia</em>, <em>D. simulans</em>, and <em>D. ananassae</em> from the website of the 12 Drosophila species genome sequencing project (Drosophila 12 Genomes Consortium 2007) (rana.lbl.gov/drosophila/caf1/all_caf1.tar.gz).</p></li><li id="list-item-2"><p id="p-24">The genomes of <em>D. bipectinata</em>, <em>D. kikkawai</em>, <em>D. elegans</em>, <em>D. ficusphila</em>, <em>D. rhopaloa</em>, <em>D. biarmipes</em>, and <em>D. takahashii</em>, which were sequenced at the Baylor College of Medicine (<a href="http://www.hgsc.bcm.tmc.edu/collaborations/insects/dros_modencode/GAsm/">http://www.hgsc.bcm.tmc.edu/collaborations/insects/dros_modencode/GAsm/</a>).</p></li><li id="list-item-3"><p id="p-25">The genome of <em>D. santomea</em> sequenced by the Andolfatto lab (<a href="http://genomics.princeton.edu/AndolfattoLab/Dsantomea_genome.html">http://genomics.princeton.edu/AndolfattoLab/Dsantomea_genome.html</a>).</p></li></ol><p id="p-26">Again, Ns were removed from the sequences of up to 23,004 contigs.</p></div><div id="sec-7" class="subsection"><h3>Simulations</h3><p id="p-27">Pairs of query/subject sequences were generated using our program generateQuerySbjct, which is also available from the pid home page. generateQuerySbjct calls the coalescent simulation program ms (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hudson 2002</a>) or macs (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2009</a>), and converts the output using our program ms2dna (available from <a href="http://guanine.evolbio.mpg.de/bioBox">http://guanine.evolbio.mpg.de/bioBox</a>).</p></div><div id="sec-8" class="subsection"><h3>Phylogeny reconstruction</h3><p id="p-28">We applied the program kr (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Domazet-Lošo and Haubold 2009</a>) to estimate all pairwise distances between the 21 complete <em>Drosophila</em> genomes currently available. The resulting distance matrix was clustered using the neighbor-joining algorithm as implemented in neighbor, which is part of PHYLIP (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Felsenstein 2005</a>). The tree was midpoint-rooted using retree and drawn with drawgram, both also part of PHYLIP.</p><p id="p-29">To compare this tree with the corresponding alignment-based phylogeny, we followed a study of <em>Drosophila</em> evolution centered on the <em><a href="http://flybase.org/reports/FBgn0020506.html">Amyrel</a></em> gene (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Da Lage <em>et al.</em> 2007</a>): we aligned the <em><a href="http://flybase.org/reports/FBgn0020506.html">Amyrel</a></em> sequences from each organism and computed the neighbor-joining tree using clustalw (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Larkin <em>et al.</em> 2007</a>). Rooting and drawing the tree was done as just described for the alignment-free cluster diagram.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><p id="p-30">We started our investigation of the properties of <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-34.gif"/></noscript></span></span> by comparing the theoretical distribution of shustring lengths with that obtained through simulation. For the simulation, we generated 1000 pairs of DNA sequences of length 100 kb conditioned on <em>π</em> = 0.01 mismatches per position and a rate of recombination of <em>ρ</em> = 0.01. From this, we averaged the distribution of the shustring lengths. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows that this simulated distribution is closely approximated by the theoretical distribution. Notice also that the distribution of shustring lengths is strongly heavy tailed in the formal sense that <em>E</em>[<em>X<sub>i</sub></em><sup>*</sup> – <em>x</em>|<em>X<sub>i</sub></em><sup>*</sup> &gt; <em>x</em>] = ∞; in particular, the shustring length distribution has no finite expectation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between the theoretical distribution of shustring lengths and the simulated distribution. &#x3C1; = 0.01, &#x3C0; = 0.01, and sequences were 100 kb long; simulations were averaged over 1000 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison between the theoretical distribution of shustring lengths and the simulated distribution. &lt;em&gt;&#x3C1;&lt;/em&gt; = 0.01, &lt;em&gt;&#x3C0;&lt;/em&gt; = 0.01, and sequences were 100 kb long; simulations were averaged over 1000 iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F2.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F2.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461702" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-31" class="first-child">Comparison between the theoretical distribution of shustring lengths and the simulated distribution. <em>ρ</em> = 0.01, <em>π</em> = 0.01, and sequences were 100 kb long; simulations were averaged over 1000 iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">To determine whether the theoretical shustring length distribution could be used to estimate <em>π</em>, we again simulated pairs of 100 kb DNA sequences at values of <em>ρ</em> ranging from 0 to 0.082, while keeping <em>π</em> = 0.01 constant. As <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Haubold <em>et al.</em> (2011)</a> had reported before, the previous estimator, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-35.gif"/></noscript></span></span>, worked well for <em>ρ</em> = 0, but was strongly downward biased for <em>ρ</em> &gt; 0 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In contrast, our new estimator, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-36.gif"/></noscript></span></span>, gives good results for <em>ρ</em> ≤ <em>π</em>. For larger values of <em>ρ</em>, it is biased upward (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between our previous alignment-free estimator of genetic diversity, , and our new estimator, , as a function of &#x3C1;. Pairs of 100 kb sequences were simulated with &#x3C0; = 0.01 and data points are mean &#xB1; SD determined from 10,000 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Comparison between our previous alignment-free estimator of genetic diversity, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and our new estimator, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-38&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, as a function of &lt;em&gt;&#x3C1;&lt;/em&gt;. Pairs of 100 kb sequences were simulated with &lt;em&gt;&#x3C0;&lt;/em&gt; = 0.01 and data points are mean &#xB1; SD determined from 10,000 iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F3.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F3.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461768" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-33" class="first-child">Comparison between our previous alignment-free estimator of genetic diversity, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F3/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F3/embed/mml-math-37.gif"/></noscript></span></span>, and our new estimator, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F3/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F3/embed/mml-math-38.gif"/></noscript></span></span>, as a function of <em>ρ</em>. Pairs of 100 kb sequences were simulated with <em>π</em> = 0.01 and data points are mean ± SD determined from 10,000 iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Instead of varying <em>ρ</em> and keeping <em>π</em> constant, we also varied <em>π</em> while keeping <em>ρ</em> constant at 0.01. <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> shows that for <em>π</em> ≤ 0.02 the estimates are very close to the true values. For more divergent sequences, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-39.gif"/></noscript></span></span> becomes downward biased and then breaks down, as shown for <em>π</em> = 0.08.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The new estimator of the number of pairwise mismatches, , as a function of the number of pairwise mismatches, &#x3C0;. 104 pairs of 100 kb sequences were simulated to compute mean &#xB1; SD. Here, we set &#x3C1; = 0.01 for all values of &#x3C0;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The new estimator of the number of pairwise mismatches, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-40&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, as a function of the number of pairwise mismatches, &lt;em&gt;&#x3C0;&lt;/em&gt;. 10&lt;sup&gt;4&lt;/sup&gt; pairs of 100 kb sequences were simulated to compute mean &#xB1; SD. Here, we set &lt;em&gt;&#x3C1;&lt;/em&gt; = 0.01 for all values of &lt;em&gt;&#x3C0;&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F4.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F4.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461811" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-35" class="first-child">The new estimator of the number of pairwise mismatches, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F4/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F4/embed/mml-math-40.gif"/></noscript></span></span>, as a function of the number of pairwise mismatches, <em>π</em>. 10<sup>4</sup> pairs of 100 kb sequences were simulated to compute mean ± SD. Here, we set <em>ρ</em> = 0.01 for all values of <em>π</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Up to now, we have estimated global values of <em>π</em>. However, it is often more interesting to study the local variation in <em>π</em> through a sliding window analysis. To investigate the suitability of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-41.gif"/></noscript></span></span> for this, we simulated a 1 Mb sequence pair with <em>π</em> = <em>ρ</em> = 0.01. In <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, 100 kb sliding windows of <em>π</em> are compared with <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-42.gif"/></noscript></span></span>. Although <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-43.gif"/></noscript></span></span> tends to exaggerate the fluctuations of the <em>π</em> curve, it tracks it much more faithfully than <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-44.gif"/></noscript></span></span> and appears to be unbiased. This visual impression is corroborated by the averages of <em>π</em> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-45.gif"/></noscript></span></span>, which are very similar (avg(<em>π</em>) = 1.003 × 10<sup>−2</sup>; <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-46.gif"/></noscript></span></span>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The values of the diversity measures &#x3C0;, , and  along a pair of simulated sequences 1 Mb long." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The values of the diversity measures &lt;em&gt;&#x3C0;&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; along a pair of simulated sequences 1 Mb long.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F5.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F5.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461709" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-37" class="first-child">The values of the diversity measures <em>π</em>, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F5/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F5/embed/mml-math-47.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F5/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F5/embed/mml-math-48.gif"/></noscript></span></span> along a pair of simulated sequences 1 Mb long.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Sliding window analyses are only feasible if the statistic of interest can be computed efficiently. Our program pid took 9.1 sec on a single Intel Xeon 3 GHz CPU to analyze the 90 windows of 100 kb summarized in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>. This is, of course, much slower than the computation of <em>π</em> given an alignment. But without an alignment, it is quick enough to analyze realistic data sets.</p><p id="p-39">To apply <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-49.gif"/></noscript></span></span> to real data sets, we compared two strains of <em>Drosophila melanogaster</em>, RAL-356 and RAL-391, whose genomes have been published as part of the Drosophila Population Genomics Project (<a href="http://www.dpgp.org">www.dpgp.org</a>). These sequences are distributed as alignments, which obviates an alignment-free approach. However, the existence of an alignment allows us to compare <em>π</em> with <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-50.gif"/></noscript></span></span>, and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows a sliding window analysis for both quantities. With the exception of the centromeric region, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-51.gif"/></noscript></span></span> appears to track <em>π</em> well. In particular, the well-known drop in diversity in the peritelomeric and pericentromeric regions is readily discernible. In the positions closest to the centromere, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-52.gif"/></noscript></span></span> is consistently larger than <em>π</em>. One reason for this might be a lack of homologous sequence in strain RAL-391. This illustrates that low genetic diversity is diagnosed more reliably by our method than is high genetic diversity, which may result from missing data. To see this connection between missing data and overestimation of genetic diversity, imagine a region in the query sequence without homolog in the subject sequence. In that region, the shustrings would reflect short random matches, which mimics the short shustrings found in homologous regions with lots of mutations.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between the diversity measures &#x3C0; and  along chromosome 2L of the two Drosophila melanogaster strains RAL-365 and RAL-391. Window length: 105 bp, windows advanced by 104 bp. In the  analysis RAL-365 served as query." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Comparison between the diversity measures &lt;em&gt;&#x3C0;&lt;/em&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-53&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; along chromosome 2L of the two &lt;em&gt;Drosophila melanogaster&lt;/em&gt; strains RAL-365 and RAL-391. Window length: 10&lt;sup&gt;5&lt;/sup&gt; bp, windows advanced by 10&lt;sup&gt;4&lt;/sup&gt; bp. In the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; analysis RAL-365 served as query.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F6.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F6.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461792" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-40" class="first-child">Comparison between the diversity measures <em>π</em> and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F6/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F6/embed/mml-math-53.gif"/></noscript></span></span> along chromosome 2L of the two <em>Drosophila melanogaster</em> strains RAL-365 and RAL-391. Window length: 10<sup>5</sup> bp, windows advanced by 10<sup>4</sup> bp. In the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F6/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F6/embed/mml-math-54.gif"/></noscript></span></span> analysis RAL-365 served as query.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">To limit the upward bias that can thus be introduced through missing data, we imposed a threshold on the number of distinct shustrings that can be reported for a given window as described under Approach and Data. If this threshold is exceeded, no <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-55.gif"/></noscript></span></span> value is returned for that window. Without this heuristic, <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-56.gif"/></noscript></span></span> would jump to 0.023 in the pericentromeric region (not shown), instead of the value of 0.013 reported by pid.</p><p id="p-42">In <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> we compare the distribution of <em>π</em> with <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-57.gif"/></noscript></span></span> values across the entire genome. In spite of the problem with missing data just discussed, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-58.gif"/></noscript></span></span> tends to be slightly smaller than <em>π</em>, which is reflected in the means of the two distributions, where <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-59.gif"/></noscript></span></span> is less than mean(<em>π</em>) = 0.0076.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The frequency distributions of the diversity measures &#x3C0; and  in 100 kb sliding windows along the full genomes of two strains of Drosophila melanogaster, RAL-365 and RAL-391." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The frequency distributions of the diversity measures &lt;em&gt;&#x3C0;&lt;/em&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-60&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in 100 kb sliding windows along the full genomes of two strains of &lt;em&gt;Drosophila melanogaster&lt;/em&gt;, RAL-365 and RAL-391.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F7.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F7.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461814" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-43" class="first-child">The frequency distributions of the diversity measures <em>π</em> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F7/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F7/embed/mml-math-60.gif"/></noscript></span></span> in 100 kb sliding windows along the full genomes of two strains of <em>Drosophila melanogaster</em>, RAL-365 and RAL-391.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">To explore a pair of unaligned genomes, we turned our attention to the 21 complete genomes of <em>Drosophila</em> species currently available. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a> shows a phylogeny of these species computed from their full genomes. For comparison, <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a> shows a corresponding alignment-based phylogeny computed just from the 2 kb sequences of the <em><a href="http://flybase.org/reports/FBgn0020506.html">Amyrel</a></em> gene (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Da Lage <em>et al.</em> 2007</a>). The two trees are reassuringly similar, especially for closely related clades.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining trees of the pairwise number of substitutions between 21 Drosophila species. (A) Substitution rates estimated without alignment from the full genome sequences. (B) substitution rates estimated from a multiple sequence alignment of the Amyrel gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Neighbor-joining trees of the pairwise number of substitutions between 21 &lt;em&gt;Drosophila&lt;/em&gt; species. (A) Substitution rates estimated without alignment from the full genome sequences. (B) substitution rates estimated from a multiple sequence alignment of the &lt;em&gt;Amyrel&lt;/em&gt; gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F8.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F8.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461776" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8 </span> <p id="p-45" class="first-child">Neighbor-joining trees of the pairwise number of substitutions between 21 <em>Drosophila</em> species. (A) Substitution rates estimated without alignment from the full genome sequences. (B) substitution rates estimated from a multiple sequence alignment of the <em>Amyrel</em> gene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">We were looking for a pair of closely related genomes that had been assembled <em>de novo</em>. There were three candidate pairs: <em>D. sechellia</em>/<em>D. simulans</em>, <em>D. yakuba</em>/<em>D. santomea</em>, and <em>D. pseudoobscura</em>/<em>D. persimilis</em>. However, <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Haubold <em>et al.</em> (2011)</a> had already investigated <em>D. sechellia</em>/<em>D. simulans</em>, and the genome of <em>D. santomea</em> appears to have been assembled on the scaffold of <em>D. yakuba</em>, yielding a pair of effectively aligned genomes. We therefore decided to compare the genomes of <em>D. pseudoobscura</em>/<em>D. persimilis</em>.</p><p id="p-47">The genome of <em>D. persimilis</em> consists of 175.6 Mb distributed over 12,837 contigs, whereas that of <em>D. pseudoobscura</em> was largely made up of 15 contigs associated with chromosomes and a further 4025 unmapped contigs comprising 146.1 Mb in total. The sliding window comparison between <em>D. pseudoobscura</em> as query and <em>D. persimilis</em> as subject took 33 min on a a single AMD Opteron CPU running at 2.3 GHz. Chromosome 3 contains the genome-wide minimum in genetic diversity in its pericentromeric region. <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> shows the location of this minimum among the fluctuating <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-61.gif"/></noscript></span></span> values along the length of chromosome 3. As previously observed by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Noor <em>et al.</em> (2007)</a>, we find that the divergence is reduced not only in the pericentromeric region but also in the peritelomeric region.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/883/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sliding 100 kb windows along chromosome 3 of Drosophila pseudoobscura compared with D. persimilis. The arrow indicates the global minimum in genetic diversity between these two genomes, and the horizontal line the chromosome-wide average of ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1083663003" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Sliding 100 kb windows along chromosome 3 of &lt;em&gt;Drosophila pseudoobscura&lt;/em&gt; compared with &lt;em&gt;D. persimilis&lt;/em&gt;. The arrow indicates the global minimum in genetic diversity between these two genomes, and the horizontal line the chromosome-wide average of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-62&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/883/F9.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/883/F9.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/883/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/883/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461637" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9 </span> <p id="p-48" class="first-child">Sliding 100 kb windows along chromosome 3 of <em>Drosophila pseudoobscura</em> compared with <em>D. persimilis</em>. The arrow indicates the global minimum in genetic diversity between these two genomes, and the horizontal line the chromosome-wide average of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F9/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/F9/embed/mml-math-62.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-49">Computation is the bridge between theory and experiment. The development of suitable computational methods has, therefore, been an integral part of population genetics for a long time. For example, Kingman's coalescent is, on the one hand, a mathematical concept (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kingman 1982</a>), but when implemented as a computer program, it becomes an efficient tool for analyzing experimental data (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hudson 1983</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hudson 2002</a>). More recent work on the ancestral recombination graph (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mcvean and Cardin 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Marjoram and Wall 2006</a>) has lead to the very fast simulation program macs (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2009</a>), to name but two examples of computational advances in population genetics.</p><p id="p-50">Our development of an alignment-free diversity estimator, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-63.gif"/></noscript></span></span>, continues this tradition of applying mathematical or algorithmic discoveries to population genetics. Like most research on alignment-free algorithms, our work is motivated by efficiency considerations (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Vinga and Almeida 2003</a>). In situations of data super-abundance, a quick <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-64.gif"/></noscript></span></span> scan could be used to guide subsequent, more detailed alignment-based investigations.</p><p id="p-51">There is a strong tradition of basing alignment-free sequence analysis on word counts as they can be computed easily (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Reinert <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wan <em>et al.</em> 2010</a>). In contrast, our method utilizes the computationally more involved distribution of shustring lengths along a genome. This distribution is similar to the match length distribution first investigated in DNA sequences by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arratia <em>et al.</em> (1986)</a>. These authors looked at the length distribution of random matches within a single sequence. In contrast, we compute match lengths between homologous pairs of sequences. The motivation for using this statistic is that it lends itself naturally to explicit evolutionary modeling, as it effectively deals with distances between SNPs.</p><p id="p-52">The central feature of our model is the standard coalescent assumption that the time to the most recent common ancestor of two homologous sequence segments is exponentially distributed. Moreover, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-65.gif"/></noscript></span></span> works not only for fully assembled genomes, but also for sets of contigs. The only two conditions imposed on the data are that (i) recombination is not much more frequent than mutation, and (ii) that edge effects can be neglected, in other words, that most shustrings end before the contig they appear in. This means that <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-66.gif"/></noscript></span></span> will be less precise if the data consists of many contigs rather than contiguous sequence, everything else being equal.</p><p id="p-53">Algorithmically, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-67.gif"/></noscript></span></span> is based on advances in string indexing, which allow fast lookup of shustring (shortest unique substring) lengths between genomes. To this preexisting technology we have added the derivation of the distribution of shustring lengths to arrive at a maximum-likelihood estimator of <em>π</em>, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-68.gif"/></noscript></span></span>. Recombination leads to fluctuating times to the most recent common ancestor along sequences, which is observable as clustered polymorphisms and an increase in the average shustring length. This effect of recombination on the average shustring length impaired the previous estimator of <em>π</em>, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-69.gif"/></noscript></span></span>, which was based on the assumption of constant coalescent times across the sequences compared (<a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Haubold <em>et al.</em> 2011</a>). By allowing the coalescent times to fluctuate and computing the new estimator <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-70.gif"/></noscript></span></span> from the whole distribution of shustring lengths (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), rather than just from their average, we have much improved the precision of our previous estimator (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">4</a>), while keeping its implementation fast.</p><p id="p-54">The advantage of the new approach is especially apparent in sliding window analyses, and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> demonstrates the accuracy of sliding <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-71.gif"/></noscript></span></span> when applied to simulated data. However, the analysis of the two strains of <em>Drosophila melanogaster</em> from North Carolina revealed a small downward bias of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-72.gif"/></noscript></span></span>. This might be due to recent gene duplications. These would lead to long shustrings and hence to an underestimation of <em>π</em>. Moreover, we assume independence between nucleotides, which is known not to apply in, for example, protein coding sequences or CpG islands. Higher order dependencies between nucleotides would also lead to longer shustrings than expected under our model and thereby to an underestimation of <em>π</em>. Finally, selection leads to longer haplotypes and concomitantly longer shustrings, which would also lower <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/883/embed/mml-math-73.gif"/></noscript></span></span>.</p><p id="p-55">The comparison of <em>D. pseudoobscura</em> with <em>D. persimilis</em> revealed a decrease in genetic diversity in the pericentromeric and the peritelomeric regions. <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a> clearly shows this valley in genetic diversity among the peritelomeric first 2 Mb of chromosome 3, which also contains the global diversity minimum. Such a reduction in genetic diversity at the ends of chromosome arms (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>) is typical for intra-species comparisons among genomes of <em>D. melanogaster</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Begun <em>et al.</em> 2007</a>). <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Noor <em>et al.</em> (2007)</a> first observed that this is also present in the inter-species comparison between <em>D. pseudoobscura</em> and <em>D. persimilis</em>. They explained this as a remnant of the recent divergence of the species, leaving the well-known correlation between local diversity and recombination in <em>Drosophila</em> intact.</p><p id="p-56">We plan to extend this work in two directions: First, we wish to develop an alignment-free test for recombination based on the fact that the mean shustring length is highly sensitive to recombination (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Such a test might be useful for detecting recombination in bacterial genomes undergoing occasional horizontal gene transfer. Second, we plan to estimate diversity from samples of more than two sequences. Here, we would apply more specific properties of the coalescent to obtain an estimator of the population mutation rate <em>θ</em>, which could then be compared to Watterson's classical estimator (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Watterson 1975</a>).</p><p id="p-57">We have shown that pid can be used to quickly compare genomes consisting of unmapped contigs. Unmapped contigs are difficult to align under the best of circumstances, but they increasingly form the end-result of genome sequencing efforts. Analysis of such data with pid could be an early step followed by more detailed investigations using alignment-based methods. Therefore, our alignment-free method is best viewed as complementary to alignment-based approaches whenever a rough and ready prescreening of population genomics data is desired. However, in spite of the simplifying assumptions we have made, our method is accurate enough to reveal the diversity landscape along metazoan chromosomes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">B.H. is supported by funds from the Max Planck Society. P.P. is supported by the Deutsche Forschungsgemeinschaft through grant Pf672/3-1.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-03-09"><span class="received-label">Received </span>March 9, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-05-29"><span class="accepted-label">Accepted </span>May 29, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Haubold, Pfaffelhuber</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.883.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abouelhoda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohlebusch</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-comment">The enhanced suffix array and its applications to genome analysis, pp. 449–463 in <em>Proceedings of the Second Workshop on Algorithms in Bioinformatics</em>, edited by R. Guigó and D. Gusfield. Springer-Verlag, London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.2" data-doi="10.1214/aos/1176350045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arratia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">An extreme value theory for sequence matching</span>. <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">971</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D14%26rft.spage%253D971%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176350045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176350045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Population genomics: whole-genome analysis of polymorphism and divergence in <em>Drosophila simulans</em></span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-comment">e310</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.4" data-doi="10.1101/gr.083634.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and flexible simulation of DNA sequence data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083634.108%26rft_id%253Dinfo%253Apmid%252F19029539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Da Lage</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kergoat</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maczkowiak</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silvain</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cariou</span>  <span class="cit-name-given-names">M.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lachaise</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A phylogeny of Drosophilidae using the <em>Amyrel</em> gene: questioning the <em>Drosophila melanogaster</em> species group boundaries</span>. <abbr class="cit-jnl-abbrev">J. Zoological Syst. Evol. Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BZoological%2BSyst.%2BEvol.%2BRes.%26rft.volume%253D45%26rft.spage%253D46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.6" data-doi="10.1093/bioinformatics/btp590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Domazet-Lošo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Efficient estimation of pairwise distances between genomes</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3221</span>–<span class="cit-lpage">3227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp590%26rft_id%253Dinfo%253Apmid%252F19825795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjQvMzIyMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.7" data-doi="10.4161/mge.1.3.18065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Domazet-Lošo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Alignment-free detection of horizontal gene transfer between closely related bacterial genomes</span>. <abbr class="cit-jnl-abbrev">Mobile Genet. Elements</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMobile%2BGenet.%2BElements%26rft.volume%253D1%26rft.spage%253D230%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fmge.1.3.18065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/mge.1.3.18065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.8" data-doi="10.1093/bioinformatics/btr176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Domazet-Lošo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Alignment-free detection of local similarity among viral and bacterial genomes</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1466</span>–<span class="cit-lpage">1472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr176%26rft_id%253Dinfo%253Apmid%252F21471011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTEvMTQ2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-2.8.883.9" data-doi="10.1038/nature06341"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Drosophila 12 Genomes Consortium</span>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of genes and genomes on the <em>Drosophila</em> phylogeny</span>. <span class="cit-source">Nature</span> <span class="cit-vol">450</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D450%26rft.issue%253D7167%26rft.spage%253D203%26rft.epage%253D218%26rft.atitle%253DEvolution%2Bof%2Bgenes%2Band%2Bgenomes%2Bon%2Bthe%2BDrosophila%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06341%26rft_id%253Dinfo%253Apmid%252F17994087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994087&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250746200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.10" data-doi="10.1534/genetics.108.093419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dworkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gentic changes accompanying the evolution of host specialization in <em>Drosophila sechellia</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D181%26rft.spage%253D721%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.093419%26rft_id%253Dinfo%253Apmid%252F19033155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.093419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19033155&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270213400030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.883.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-comment">PHYLIP (phylogeny interference package), Version 3.6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.883.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusfield</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source">Algorithms on Strings, Trees, and Sequences: Computer Science and Computational Biology.</span> <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.13" data-doi="10.1089/cmb.2009.0106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaffelhuber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domazet-Lošo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiehe</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating mutation distances from unaligned genomes</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1487</span>–<span class="cit-lpage">1500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DHaubold%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D1487%26rft.epage%253D1500%26rft.atitle%253DEstimating%2Bmutation%2Bdistances%2Bfrom%2Bunaligned%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2009.0106%26rft_id%253Dinfo%253Apmid%252F19803738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2009.0106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19803738&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270687800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.14" data-doi="10.1186/1471-2105-6-123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierstorff</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möller</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiehe</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome comparison without alignment using shortest unique substrings</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHaubold%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.spage%253D123%26rft.epage%253D123%26rft.atitle%253DGenome%2Bcomparison%2Bwithout%2Balignment%2Busing%2Bshortest%2Bunique%2Bsubstrings.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-123%26rft_id%253Dinfo%253Apmid%252F15910684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15910684&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.15" data-doi="10.1093/bioinformatics/btq689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaffelhuber</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Alignment-free estimation of nucleotide diversity</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq689%26rft_id%253Dinfo%253Apmid%252F21156730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy80LzQ0OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.16" data-doi="10.1016/0040-5809(83)90013-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Properties of a neutral allele model with intragenic recombination</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D183%26rft.epage%253D201%26rft.atitle%253DProperties%2Bof%2Ba%2Bneutral%2Ballele%2Bmodel%2Bwith%2Bintragenic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252883%252990013-8%26rft_id%253Dinfo%253Apmid%252F6612631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(83)90013-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6612631&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QU75700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Gene genealogies and the coalescent process</span>. <abbr class="cit-jnl-abbrev">Oxford Surveys in Evolutionary Biology</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOxford%2BSurveys%2Bin%2BEvolutionary%2BBiology%26rft.volume%253D7%26rft.spage%253D1%26rft.atitle%253DOXFORD%2BSURVEYS%2BIN%2BEVOLUTIONARY%2BBIOLOGY%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.18" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Generating samples under a Wright-Fisher neutral model of genetic variation</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOC8yLzMzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.19" data-doi="10.1016/0304-4149(82)90011-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingman</span>  <span class="cit-name-given-names">J. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The coalescent</span>. <abbr class="cit-jnl-abbrev">Stochastic Process. Appl.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStochastic%2BProcess.%2BAppl.%26rft.volume%253D13%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0304-4149%252882%252990011-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0304-4149(82)90011-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-comment">R25</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.21" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjEvMjk0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.22" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler Transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.23" data-doi="10.1093/bioinformatics/btp336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">T.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yiu</span>  <span class="cit-name-given-names">S.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SOAP2: an improved ultrafast tool for short read alignment</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1966</span>–<span class="cit-lpage">1967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp336%26rft_id%253Dinfo%253Apmid%252F19497933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.883.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manzini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferragina</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-comment">Engineering a lightweight suffix array construction algorithm, pp. 698–710 in <em>ESA ’02: Proceedings of the 10th Annual European Symposium on Algorithms</em>. Springer-Verlag, London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.25" data-doi="10.1186/1471-2156-7-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Fast “coalescent” simulation</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DMarjoram%26rft.auinit1%253DP.%26rft.volume%253D7%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DFast%2B%2526quot%253Bcoalescent%2526quot%253B%2Bsimulation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-7-16%26rft_id%253Dinfo%253Apmid%252F16539698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-7-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16539698&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.26" data-doi="10.1098/rstb.2005.1673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardin</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Approximating the coalescent with recombination</span>. <abbr class="cit-jnl-abbrev">Phil. Trans. R. Soc. B</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">1387</span>–<span class="cit-lpage">1393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BR.%2BSoc.%2BB%26rft.volume%253D360%26rft.spage%253D1387%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2005.1673%26rft_id%253Dinfo%253Apmid%252F16048782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2005.1673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16048782&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.27" data-doi="10.1534/genetics.107.070672"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfield</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Divergence between the <em>Drosophila pseudoobscura</em> and <em>D. persimilis</em> genome sequences in relation to chromosomal inversions</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1417</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNoor%26rft.auinit1%253DM.%2BA.%2BF.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D1417%26rft.epage%253D1428%26rft.atitle%253DDivergence%2BBetween%2Bthe%2BDrosophila%2Bpseudoobscura%2Band%2BD.%2Bpersimilis%2BGenome%2BSequences%2Bin%2BRelation%2Bto%2BChromosomal%2BInversions%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070672%26rft_id%253Dinfo%253Apmid%252F18039875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE0MTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.28" data-doi="10.1145/1242471.1242472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puglisi</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turpin</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A taxonomy of suffix array construction algorithms</span>. <abbr class="cit-jnl-abbrev">ACM Comput. Surv.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACM%2BComput.%2BSurv.%26rft.volume%253D39%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1145%252F1242471.1242472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/1242471.1242472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.29" data-doi="10.1089/cmb.2009.0198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chew</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Alignment-free sequence comparison (I): statistics and power</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1615</span>–<span class="cit-lpage">1634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DReinert%26rft.auinit1%253DG.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D1615%26rft.epage%253D1634%26rft.atitle%253DAlignment-free%2Bsequence%2Bcomparison%2B%2528I%2529%253A%2Bstatistics%2Band%2Bpower.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2009.0198%26rft_id%253Dinfo%253Apmid%252F20001252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2009.0198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20001252&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273709400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.30" data-doi="10.1093/bioinformatics/btg005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Alignment-free sequence comparison—a review</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DVinga%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D513%26rft.epage%253D523%26rft.atitle%253DAlignment-free%2Bsequence%2Bcomparison--a%2Breview%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg005%26rft_id%253Dinfo%253Apmid%252F12611807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS80LzUxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.883.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-comment"><em>Coalescent Theory: An Introduction</em>. Roberts &amp; Company, Greenwood Village, Colorado</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.32" data-doi="10.1089/cmb.2010.0056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Alignment-free sequence comparison (II): theoretical power of comparison statistics</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1467</span>–<span class="cit-lpage">1490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DWan%26rft.auinit1%253DL.%26rft.volume%253D17%26rft.issue%253D11%26rft.spage%253D1467%26rft.epage%253D1490%26rft.atitle%253DAlignment-free%2Bsequence%2Bcomparison%2B%2528II%2529%253A%2Btheoretical%2Bpower%2Bof%2Bcomparison%2Bstatistics.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2010.0056%26rft_id%253Dinfo%253Apmid%252F20973742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2010.0056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20973742&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.883.33" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/8/883.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/873" title="Microarray-Based Capture of Novel Expressed Cell Type–Specific Transfrags (CoNECT) to Annotate Tissue-Specific Transcription in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/891" title="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/8"><span>Volume 2 Issue 8, August 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/8.cover-source.jpg?itok=etPFDPKt" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/883" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oIVJkl2BZirEIIUTWhud1lbBQtYOAQTGN4NlFgeNa-4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/8/883" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/8/883" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/8/883" class="active" data-icon-position="" data-hide-link-title="0">Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459306" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TKIeQ9D55KI_xbA10opf_pfhfn9BNiSNzgo-sBQSgeg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/883.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/8/883" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-U-JkjRCTHdS634VvrwP8OZ3YcEIuoLRAw_D-aTr8PN8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/8/883" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/883.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459306" id="node459306--2" data-pisa="ggg;2/8/883" data-pisa-master="ggg;2/8/883" data-apath="/ggg/2/8/883.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bernhard Haubold</span> and <span class="highwire-citation-author" data-delta="1">Peter Pfaffelhuber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">883-889;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002527 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459306--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bernhard Haubold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BHaubold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haubold%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernhard%2BHaubold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haubold@evolbio.mpg.de" class="" data-icon-position="" data-hide-link-title="0">haubold@evolbio.mpg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Pfaffelhuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mathematical Stochastics, Mathematical Institute, Albert-Ludwigs University, 79085 Freiburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BPfaffelhuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfaffelhuber%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BPfaffelhuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/8/883" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/8/883&amp;title=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/8/883&amp;title=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/8/883&amp;title=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/8/883&amp;text=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/8/883&amp;title=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/8/883&amp;t=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/8/883&amp;title=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/8/883&amp;title=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459306/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459306" id="node459306--3" data-pisa="ggg;2/8/883" data-pisa-master="ggg;2/8/883" data-apath="/ggg/2/8/883.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bernhard Haubold</span> and <span class="highwire-citation-author" data-delta="1">Peter Pfaffelhuber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">883-889;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002527 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459306--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bernhard Haubold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, 24306 Plön, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BHaubold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haubold%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernhard%2BHaubold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haubold@evolbio.mpg.de" class="" data-icon-position="" data-hide-link-title="0">haubold@evolbio.mpg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Pfaffelhuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Mathematical Stochastics, Mathematical Institute, Albert-Ludwigs University, 79085 Freiburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BPfaffelhuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pfaffelhuber%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BPfaffelhuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=883&amp;rft.epage=889&amp;rft.atitle=Alignment-Free%20Population%20Genomics%3A%20An%20Efficient%20Estimator%20of%20Sequence%20Diversity&amp;rft.volume=2&amp;rft.issue=8&amp;rft.date=2012-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Haubold%2C+Bernhard&amp;rft.au=Pfaffelhuber%2C+Peter"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459306/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459306/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459306/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459306/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459306/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459306/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459306/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459306/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459306/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459306/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459306/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459306/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/8/883" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Approach and Data</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/8/883.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/8/883.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/2/8/883.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
