<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/8/921" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/8/921.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/8/921.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/8/921.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/8/921.atom" />
<meta name="HW.pisa" content="ggg;2/8/921" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton" />
<meta name="DC.Identifier" content="10.1534/g3.112.003392" />
<meta name="DC.Date" content="2012-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2012 Salmon et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Targeted sequence capture is a promising technology in many areas in biology. These methods enable efficient and relatively inexpensive sequencing of hundreds to thousands of genes or genomic regions from many more individuals than is practical using whole-genome sequencing approaches. Here, we demonstrate the feasibility of target enrichment using sequence capture in polyploid cotton. To capture and sequence both members of each gene pair (homeologs) of wild and domesticated Gossypium hirsutum , we created custom hybridization probes to target 1000 genes (500 pairs of homeologs) using information from the cotton transcriptome. Two widely divergent samples of G. hirsutum were hybridized to four custom NimbleGen capture arrays containing probes for targeted genes. We show that the two coresident homeologs in the allopolyploid nucleus were efficiently captured with high coverage. The capture efficiency was similar between the two accessions and independent of whether the samples were multiplexed. A significant amount of flanking, nontargeted sequence (untranslated regions and introns) was also captured and sequenced along with the targeted exons. Intraindividual heterozygosity is low in both wild and cultivated Upland cotton, as expected from the high level of inbreeding in natural G. hirsutum and bottlenecks accompanying domestication. In addition, levels of heterozygosity appeared asymmetrical with respect to genome (AT or DT) in cultivated cotton. The approach used here is general, scalable, and may be adapted for many different research inquiries involving polyploid plant genomes." />
<meta name="DC.Contributor" content="Armel Salmon" />
<meta name="DC.Contributor" content="Joshua A. Udall" />
<meta name="DC.Contributor" content="Jeffrey A. Jeddeloh" />
<meta name="DC.Contributor" content="Jonathan Wendel" />
<meta name="article:published_time" content="2012-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Targeted sequence capture is a promising technology in many areas in biology. These methods enable efficient and relatively inexpensive sequencing of hundreds to thousands of genes or genomic regions from many more individuals than is practical using whole-genome sequencing approaches. Here, we demonstrate the feasibility of target enrichment using sequence capture in polyploid cotton. To capture and sequence both members of each gene pair (homeologs) of wild and domesticated &lt;i&gt;Gossypium hirsutum&lt;/i&gt;, we created custom hybridization probes to target 1000 genes (500 pairs of homeologs) using information from the cotton transcriptome. Two widely divergent samples of &lt;i&gt;G. hirsutum&lt;/i&gt; were hybridized to four custom NimbleGen capture arrays containing probes for targeted genes. We show that the two coresident homeologs in the allopolyploid nucleus were efficiently captured with high coverage. The capture efficiency was similar between the two accessions and independent of whether the samples were multiplexed. A significant amount of flanking, nontargeted sequence (untranslated regions and introns) was also captured and sequenced along with the targeted exons. Intraindividual heterozygosity is low in both wild and cultivated Upland cotton, as expected from the high level of inbreeding in natural &lt;i&gt;G. hirsutum&lt;/i&gt; and bottlenecks accompanying domestication. In addition, levels of heterozygosity appeared asymmetrical with respect to genome (A&lt;sub&gt;T&lt;/sub&gt; or D&lt;sub&gt;T&lt;/sub&gt;) in cultivated cotton. The approach used here is general, scalable, and may be adapted for many different research inquiries involving polyploid plant genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/08/01" />
<meta name="citation_mjid" content="ggg;2/8/921" />
<meta name="citation_id" content="2/8/921" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/8/921" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/8/921.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/8/921.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/8/921.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003392" />
<meta name="citation_pmid" content="22908041" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="921" />
<meta name="citation_lastpage" content="930" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Armel Salmon" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="Joshua A. Udall" />
<meta name="citation_author_institution" content="Department of Plant and Wildlife Sciences, Brigham Young University, Provo, Utah 84602, and" />
<meta name="citation_author" content="Jeffrey A. Jeddeloh" />
<meta name="citation_author_institution" content="Roche NimbleGen Inc., Madison, Wisconsin 53719" />
<meta name="citation_author" content="Jonathan Wendel" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author_email" content="jfw@iastate.edu" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. J. Albert;citation_author=M. N. Molla;citation_author=D. M. Muzny;citation_author=L. Nazareth;citation_author=D. Wheeler;citation_title=Direct selection of human genomic loci by microarray hybridization.;citation_pages=903-905;citation_volume=4;citation_year=2007;citation_issue=11;citation_pmid=17934467;citation_doi=10.1038/nmeth1111" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=W. L. Applequist;citation_author=R. Cronn;citation_author=J. F. Wendel;citation_title=Comparative development of fiber in wild and cultivated cotton.;citation_pages=3-17;citation_volume=3;citation_year=2001;citation_issue=1;citation_pmid=11256432;citation_doi=10.1046/j.1525-142x.2001.00079.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. N. Bainbridge;citation_author=M. Wang;citation_author=D. L. Burgess;citation_author=C. Kovar;citation_author=M. J. Rodesch;citation_title=Whole exome capture in solution with 3 Gbp of data.;citation_pages=R62-R62;citation_volume=11;citation_year=2010;citation_issue=6;citation_pmid=20565776;citation_doi=10.1186/gb-2010-11-6-r62" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Besemer;citation_author=M. Borodovsky;citation_title=GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses;citation_pages=W451-W454;citation_volume=33;citation_year=2005;citation_issue=suppl_2;citation_pmid=15980510;citation_doi=10.1093/nar/gki487" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=C. L. Brubaker;citation_author=J. F. Wendel;citation_title=Reevaluating the origin of domesticated cotton (Gossypium hirsutum; Malvaceae) using nuclear restriction fragment length polymorphisms (RFLPs);citation_pages=1309-1326;citation_volume=81;citation_year=1994;citation_doi=10.2307/2445407" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Sci.;citation_author=K. M. Butterworth;citation_author=D. C. Adams;citation_author=H. T. Horner;citation_author=J. F. Wendel;citation_title=Initiation and early development of fiber in wild and cultivated cotton;citation_pages=561-574;citation_volume=170;citation_year=2009;citation_doi=10.1086/597817" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=B. Chaudhary;citation_author=R. Hovav;citation_author=R. Rapp;citation_author=N. Verma;citation_author=J. A. Udall;citation_title=Global analysis of gene expression in cotton fibers from wild and domesticated Gossypium barbadense.;citation_pages=567-582;citation_volume=10;citation_year=2008;citation_issue=5;citation_pmid=18803775;citation_doi=10.1111/j.1525-142X.2008.00272.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. Cosart;citation_author=A. Beja-Pereira;citation_author=S. Chen;citation_author=S. B. Ng;citation_author=J. Shendure;citation_title=Exome-wide DNA capture and next generation sequencing in domestic and wild species.;citation_pages=347-347;citation_volume=12;citation_year=2011;citation_pmid=21729323;citation_doi=10.1186/1471-2164-12-347" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=R. Cronn;citation_author=B. J. Knaus;citation_author=A. Liston;citation_author=P. J. Maughan;citation_author=M. Parks;citation_title=Targeted enrichment strategies for next-generation plant biology;citation_volume=99;citation_year=2012;citation_pmid=22312117;citation_doi=10.3732/ajb.1100356" />
<meta name="citation_reference" content="citation_journal_title=Brief Funct Genomics;citation_author=J. W. Davey;citation_author=M. L. Blaxter;citation_title=RADSeq: next-generation population genetics;citation_volume=9;citation_year=2010;citation_pmid=21266344;citation_doi=10.1093/bfgp/elq031" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. F. Doebley;citation_author=B. S. Gaut;citation_author=B. D. Smith;citation_title=The molecular genetics of crop domestication.;citation_pages=1309-1321;citation_volume=127;citation_year=2006;citation_issue=7;citation_pmid=17190597;citation_doi=10.1016/j.cell.2006.12.006" />
<meta name="citation_reference" content="citation_author=J. E. Endrizzi;citation_author=E. L. Turcotte;citation_author=R. J. Kohel;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics (in press);citation_title=Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton;citation_author=L. E. Flagel;citation_author=J. F. Wendel;citation_author=J. A. Udall;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=Y. Fu;citation_author=N. M. Springer;citation_author=D. J. Gerhardt;citation_author=K. Ying;citation_author=C.-T. Yeh;citation_title=Repeat subtraction-mediated sequence capture from a complex genome.;citation_pages=898-909;citation_volume=62;citation_year=2010;citation_issue=5;citation_pmid=20230488;citation_doi=10.1111/j.1365-313X.2010.04196.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. D. George;citation_author=G. McVicker;citation_author=R. Diederich;citation_author=S. B. Ng;citation_author=A. P. MacKenzie;citation_title=Trans genomic capture and sequencing of primate exomes reveals new targets of positive selection;citation_volume=21;citation_year=2011;citation_pmid=21795384;citation_doi=10.1101/gr.121327.111" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=P. Griffin;citation_author=C. Robin;citation_author=A. Hoffmann;citation_title=A next-generation sequencing method for overcoming the multiple gene copy problem in polyploid phylogenetics, applied to Poa grasses.;citation_pages=19-19;citation_volume=9;citation_year=2011;citation_pmid=21429199;citation_doi=10.1186/1741-7007-9-19" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=B. L. Gross;citation_author=K. M. Olsen;citation_title=Genetic perspectives on crop domestication.;citation_pages=529-537;citation_volume=15;citation_year=2010;citation_issue=9;citation_pmid=20541451;citation_doi=10.1016/j.tplants.2010.05.008" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=C. E. Grover;citation_author=A. Salmon;citation_author=J. F. Wendel;citation_title=Targeted sequence capture as a powerful tool for evolutionary analysis;citation_volume=99;citation_year=2012;citation_pmid=22268225;citation_doi=10.3732/ajb.1100323" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=W. J. Haun;citation_author=D. L. Hyten;citation_author=W. W. Xu;citation_author=D. J. Gerhardt;citation_author=T. J. Albert;citation_title=The composition and origins of genomic variation among individuals of the soybean reference cultivar Williams 82;citation_volume=155;citation_year=2011;citation_pmid=21115807;citation_doi=10.1104/pp.110.166736" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Hodges;citation_author=Z. Xuan;citation_author=V. Balija;citation_author=M. Kramer;citation_author=M. N. Molla;citation_title=Genome-wide in situ exon capture for selective resequencing.;citation_pages=1522-1527;citation_volume=39;citation_year=2007;citation_issue=12;citation_pmid=17982454;citation_doi=10.1038/ng.2007.42" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=E. Hodges;citation_author=M. Rooks;citation_author=Z. Xuan;citation_author=A. Bhattacharjee;citation_author=D. Benjamin Gordon;citation_title=Hybrid selection of discrete genomic intervals on custom-designed microarrays for massively parallel sequencing.;citation_pages=960-974;citation_volume=4;citation_year=2009;citation_issue=6;citation_pmid=19478811;citation_doi=10.1038/nprot.2009.68" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Hovav;citation_author=J. A. Udall;citation_author=B. Chaudhary;citation_author=E. Hovav;citation_author=L. Flagel;citation_title=The evolution of spinnable cotton fiber entailed prolonged development and a novel metabolism.;citation_pages=e25-e25;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18248099;citation_doi=10.1371/journal.pgen.0040025" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=L. Mamanova;citation_author=A. J. Coffey;citation_author=C. E. Scott;citation_author=I. Kozarewa;citation_author=E. H. Turner;citation_title=Target-enrichment strategies for next-generation sequencing.;citation_pages=111-118;citation_volume=7;citation_year=2010;citation_issue=2;citation_pmid=20111037;citation_doi=10.1038/nmeth.1419" />
<meta name="citation_reference" content="citation_journal_title=Brief Funct Genomics;citation_author=F. Mertes;citation_author=A. ElSharawy;citation_author=S. Sauer;citation_author=J. M. L. M. van Helvoort;citation_author=P. J. van der Zaag;citation_title=Targeted enrichment of genomic DNA regions for next-generation sequencing;citation_volume=10;citation_year=2011;citation_pmid=22121152;citation_doi=10.1093/bfgp/elr033" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. B. Ng;citation_author=E. H. Turner;citation_author=P. D. Robertson;citation_author=S. D. Flygare;citation_author=A. W. Bigham;citation_title=Targeted capture and massively parallel sequencing of 12 human exomes.;citation_pages=272-276;citation_volume=461;citation_year=2009;citation_issue=7261;citation_pmid=19684571;citation_doi=10.1038/nature08250" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=G. J. Porreca;citation_author=K. Zhang;citation_author=J. B. Li;citation_author=B. Xie;citation_author=D. Austin;citation_title=Multiplex amplification of large sets of human exons.;citation_pages=931-936;citation_volume=4;citation_year=2007;citation_issue=11;citation_pmid=17934468;citation_doi=10.1038/nmeth1110" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=R. A. Rapp;citation_author=C. H. Haigler;citation_author=L. Flagel;citation_author=R. H. Hovav;citation_author=J. A. Udall;citation_title=Gene expression in developing fibres of Upland cotton (Gossypium hirsutum L.) was massively altered by domestication.;citation_pages=139-139;citation_volume=8;citation_year=2010;citation_pmid=21078138;citation_doi=10.1186/1741-7007-8-139" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. Saintenac;citation_author=D. Jiang;citation_author=E. D. Akhunov;citation_title=Targeted analysis of nucleotide and copy number variation by exon capture in allotetraploid wheat genome.;citation_pages=R88-R88;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21917144;citation_doi=10.1186/gb-2011-12-9-r88" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. S. Senchina;citation_author=I. Alvarez;citation_author=R. C. Cronn;citation_author=B. Liu;citation_author=J. K. Rong;citation_title=Rate variation among nuclear genes and the age of polyploidy in Gossypium;citation_volume=20;citation_year=2003;citation_pmid=12679546;citation_doi=10.1093/molbev/msg065" />
<meta name="citation_reference" content="citation_title=Nature;citation_title=A map of human genome variation from population-scale sequencing;citation_pages=1061-1073;citation_volume=467;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. A. Udall;citation_author=J. M. Swanson;citation_author=K. Haller;citation_author=R. A. Rapp;citation_author=M. E. Sparks;citation_title=A global assembly of cotton ESTs;citation_volume=16;citation_year=2006;citation_pmid=16478941;citation_doi=10.1101/gr.4602906" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. I. Wright;citation_author=B. S. Gaut;citation_title=Molecular population genetics and the search for adaptive evolution in plants;citation_volume=22;citation_year=2005;citation_pmid=15525701" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="twitter:description" content="Targeted sequence capture is a promising technology in many areas in biology. These methods enable efficient and relatively inexpensive sequencing of hundreds to thousands of genes or genomic regions from many more individuals than is practical using whole-genome sequencing approaches. Here, we demonstrate the feasibility of target enrichment using sequence capture in polyploid cotton. To capture and sequence both members of each gene pair (homeologs) of wild and domesticated Gossypium hirsutum , we created custom hybridization probes to target 1000 genes (500 pairs of homeologs) using information from the cotton transcriptome. Two widely divergent samples of G. hirsutum were hybridized to four custom NimbleGen capture arrays containing probes for targeted genes. We show that the two coresident homeologs in the allopolyploid nucleus were efficiently captured with high coverage. The capture efficiency was similar between the two accessions and independent of whether the samples were multiplexed. A significant amount of flanking, nontargeted sequence (untranslated regions and introns) was also captured and sequenced along with the targeted exons. Intraindividual heterozygosity is low in both wild and cultivated Upland cotton, as expected from the high level of inbreeding in natural G. hirsutum and bottlenecks accompanying domestication. In addition, levels of heterozygosity appeared asymmetrical with respect to genome (AT or DT) in cultivated cotton. The approach used here is general, scalable, and may be adapted for many different research inquiries involving polyploid plant genomes." />
<meta name="og-title" property="og:title" content="Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/8/921" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Targeted sequence capture is a promising technology in many areas in biology. These methods enable efficient and relatively inexpensive sequencing of hundreds to thousands of genes or genomic regions from many more individuals than is practical using whole-genome sequencing approaches. Here, we demonstrate the feasibility of target enrichment using sequence capture in polyploid cotton. To capture and sequence both members of each gene pair (homeologs) of wild and domesticated Gossypium hirsutum , we created custom hybridization probes to target 1000 genes (500 pairs of homeologs) using information from the cotton transcriptome. Two widely divergent samples of G. hirsutum were hybridized to four custom NimbleGen capture arrays containing probes for targeted genes. We show that the two coresident homeologs in the allopolyploid nucleus were efficiently captured with high coverage. The capture efficiency was similar between the two accessions and independent of whether the samples were multiplexed. A significant amount of flanking, nontargeted sequence (untranslated regions and introns) was also captured and sequenced along with the targeted exons. Intraindividual heterozygosity is low in both wild and cultivated Upland cotton, as expected from the high level of inbreeding in natural G. hirsutum and bottlenecks accompanying domestication. In addition, levels of heterozygosity appeared asymmetrical with respect to genome (AT or DT) in cultivated cotton. The approach used here is general, scalable, and may be adapted for many different research inquiries involving polyploid plant genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
    <title>Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"s1VjMlO5V38fCCE9IoT2Kwq4GjHKRqHwPVDwh1WuQ2Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459312","apath":"\/ggg\/2\/8\/921.atom","pisa":"ggg;2\/8\/921","ac":{"\/ggg\/2\/8\/921.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/8\/921.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/8\/921"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/8\/921"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/8\/921":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bDew5jvZWjYsRMA6Pyq3fm\\\\\\\/8GVdqCvUMbzdT48eaFqrNX8bFOZ5uDtYSr1w2bVcJP\\\\\\\/+yz8M5DH6rusidP09M5OWf9U5r+vXW3wnUF5hwsZ6C0H5F9ZdfOOANeMB7yo7ryJz8\\\\\\\/D85SLLOyQOwKl1oc5d\\\\\\\/89YfmERkkz8uMTeBjgKSafhdu7S8LC766pG0RJZRmmdJVZ3U1Cgm6xl6Ufv9ZGvL32demzJBqwFIboJXCUHHBoKQcZ60+F9MI6y\\\\\\\/EdrMRAa5luVMoECTxsCbfZbe+GRmn5Odjr62jusNl8wvqq84lF\\\\\\\/unQ\\\\\\\/AOcDaq5v7wzngBknXme0R+5CnPSLLi3nyn788apslGmgb5KCwPefnupqlv6ExVh3LU+DwfC6sg+HThu54SpsWHqInj\\\\\\\/Z3Ej1s70Wkidu+C0AKGDlarAgJLapupIbrK0kdWCwiCe7STOpf\\\\\\\/Snve0qqlH8txgXoVkJIUGTWyXmfelebdSbdOC5u4pY1RzbgV2ZR2yMmIN+YVDF\\\\\\\/w0yRcTwOAsgFImWn5ANW5EzTUY2kSWvX4owggroFlIE6AgYiNgu11nREMMe141cMsEDNeRCsjXL2Eh4JBLfd6u0L3mKOx08bHbvsuvld+nMhuqxh3v9jF4PxYP0zOP5BK2LyUnJQpwvqy1NySddwgcR5M2rJOhjgiwR2zkf7SE2+FkMyawPqbm1rAENgk64Ry6UEKpC7Xw7w4eKYMK6xbP8fRktx5VS57lyzbVDBg9RsSGtqtwQqPJU85YfOVFaRMLICIp\\\\\\\/H3TRBwWIXEXFLa8CBHlXpTOtIv7ojIDLikT0ZbYta3LEjg25yBK2C0CPOHBh0sNtiQvoFSD22dE8t5vTdO9hZEwnpbYFQAtnU7eVnt5fUR1l8cBOyHFWPiD7WgZfHD6W8AVhHdzxdHxQRtGOOiUOMyjsVdrDrP0G9kdIApfokil\\\\\\\/SUG4VY+LJpuFKxXVfaHRfzm00FK04xcU\\\\\\\/caJD3f8n9pg6ASyb54KXdv3vf6erhyWS1rcZ8cZjM3U2E9CwIybaUP9+pnxQ0FOzSS4W9zOhd7j+f83wk6ezh9kg1BDW9tamzBJf9TFWbS3yqVaTVHmMQTLWmaNPQRsMDoo24bA1Lmhbd1ZfSCz5N+bysbHIj1yEGD7Uib1pd06OETbwrFqQu5ek3qQDx4elNJll\\\\\\\/LTNEL\\\\\\\/3SUl1Yy4OBN71j7RZPdQgqGza8\\\\\\\/6zrGXSaIVzhyU\\\\\\\/EtKQzYNEmTGuwNaOi9dAjXWfALAF90hgKmX4btMA0ZGXNyqkxyTWdZZO\\\u0022,\\\u0022iv\\\u0022:\\\u00223Vdhm\\\\\\\/6VTugHgnn0xRl6qA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002223c6b05767f69acb0dfd2fe2570a17a92eac6bdc2cc4adaa13589620b118a152\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XhEuq3JXgOiP9+z+Ytty\\\\\\\/Cy1LA99XI8nN1GNR0Z5NNLILvfBGB8sNWEw+fv8qFzU9TYVMyVXRbPyvlLSDli5kxwWhQE+K7nRMed6PVy3iw7jyfPsB5wmrlJ3vTy4BgiCLBbGQZc1n0naq+B3kMIPvj36WEq9\\\\\\\/kzBQwan8tec2uqsru+4A\\\\\\\/cvVOwsfldUaRH8eVrLwH7KomxA\\\\\\\/fsa\\\\\\\/sAwK1JLABaG\\\\\\\/RXVuYbyfT1NkxuRPQMkp+0YkkkN2Yy0gi\\\\\\\/sMzW0lq6p9jn5zt57GQ6s1WmLFewgxWUUu+ImjSy5uenrMNV4iPuP+WdYDMlZy2ZSgjBYid0OqoZCk2v+K7iDCtn7uFT+dwCy1kYEtjunG7HHcx0dB7OAI6iCJNhmvh1Al78VlA5SoQRUgItFaq1WynnX2PEijkH\\\\\\\/+GilKclofUO95PGX201+bei3v+ZPyUTVB427v+RWTgFbGaIe78ahONCLhlD6tZW5MTWu8x4dp209h11UL2YdOjRKHYU3cuNhITLJfeAmyJO3tgQK7KcAEgRtSlxQe0rkJdq9pP\\\\\\\/369LWKOxTI8RgxxVpjOCEHCP6FpfaaDd2D2GJM4EQjQ1I2Qfcy\\\\\\\/AEgCDvAUn0XibXo2GSgYnvER4Wrjpc4f+tMH\\\\\\\/RftOyfhZEO\\\\\\\/JWylRbYMQkbDk7ap74D3RzK\\\\\\\/T7Vg+JXE0XBUeF9xCJX6J2KsdFtaf\\\\\\\/Vv1CD0NzUIMGs85HUctZoibauBfygNY7pIY38HkxQDht\\\\\\\/h20CrsDTa2jiqj6gugDfT0ds3luuXH5H\\\\\\\/MlCXCsksoEX6FP0ANINkfUTGB224\\\\\\\/MukSjkRC\\\\\\\/GGoZBW7eZQE6MOTfD\\\\\\\/l\\\\\\\/P0d3Ue5PAV1\\\\\\\/608CdBOVmpuuef3XNMW5kFQB5+aEnkrsh9NqqM\\\\\\\/Xa7U3wdx9VZbKPCVZSYnPvefX5y+ioiVCkhVBTvxsW9zGqoS0ClB55R9DAQHqL0lGN4CW3PKfXyLIJX2RArR1LAcB1fIJG5WqZeRBDI4BjDlYs8VwPJUW2S5dO7oHIY9MrUL057lA\\\\\\\/+0TY38yX2KmawR0jxA63RUy9e56\\\\\\\/fAslIkCYKJZbopgtNEP7Qx1VKJznVGsQXPXfgAOPSevWlQ4zY8Mp2WoEmNHSVu7tbgHSR980q4OihEzeNNq0LZQWq3bx19W0nM85\\\\\\\/QFNDpTETGX9YUIU+HCJrIk9aR\\\\\\\/bxMbAIFvnOp2fgm3+BjyMy+P+VZ7e\\\\\\\/bRs4ouN295MUsXp6VJLqBaKf1alRD85OWcUqK\\\\\\\/rcE4vwO1kfqo1IL41kBsF\\\\\\\/2T1ZrTYiBCNvxb1\\\\\\\/yTe9icQdYNdnn6q4X668tvG0pDZxQuVGKfFSA0T4OpmiF4wGn6y7oGhaG4T2C07i5koWdPdYeZKG2pTQpbz\\\\\\\/NoYiA7COmThP44CejCYzmCtr342naqzjhq0WPfGV1snZ0V2HUYPdhEDLcY87oKZfRDApebvawHfeO2+fJbHvz4hl4w3reR1GpWGgenAqCuaJCFIhE5i52DnQ4ZtR2R0NXhFt1TkHmaX6IHuuKTEBvjSCyk4D2QVPHIqjcUvYQTkgetJowZrcjASPzjQJJpytvIMJ8Cuf8G03RHsFMjMaeCHa6Wt3jp6B3xs5viphCbAnXcWl8iBffvhji27lu4q7oxX6gFtb4ue4C45Y3748Mcccy6wueaB\\\\\\\/uNYFdLNLfq1CC+rZJeUd9fBUwOzzqe5OSQl9wjdrKit9DOxG\\\\\\\/QGdGblEdHJWDDhGe4ZxpIvj48zgaQ8q017hdcWEIXSqJnxj0ETDl7gOy8VuakxPOV3+1fQEaEFkVYuXD1k12rde2L\\\\\\\/CsKDmdNa+zqJql7mu0tjnlQiwXAaLDhowY15Cot9G5alF0FpSZLI8XF+Bz+DvU6JMcxSN2nl3iRCj8Ou52ci8GkuaXpANzoxCGwh1oPOOZYII1XmHtyYAwk7hAfe+vAaBwuRXfKWfd33Nj88RdZoDZBiNeD3xF5of9L9bKehvBbWRiyorUY5+hYHrE6imGPLAfwwwTqP2jj2I5G\\\\\\\/tsvj1sVVPtG4Ha\\\\\\\/IiZjMJh7FuSYuwyO5VcViVG+tCRQJ\\\\\\\/u3GeJNZHsmWJfWO\\\\\\\/KOlH5ii\\\\\\\/sWCsYrJheKHhygyqIKiWTjZO1wEtI3CqZsa3EKkYgLlRE4pF4Nsgd6xIKW4blhWXL91NXzuK2pMKrBzmqVjfGzZ3nopuu7Tcsqp\\\\\\\/I69Ss3oJIF3M0XBQrWTaIWLeVhRm9c1lhlfF5uz2U3PEA0S4eUM7BcTjCSWfMnLywTDkoD3n7hBCFCKV8MRiYoo0c7o8+AUZEbwQxq\\\\\\\/w25Y1uU3GiOV5ZazFM1\\\\\\\/A81kuHkMVmP717aU6qomuA5uSy07GnMjIyA9tm7j3jcMqRSC2pRIA18a9oBmhbJPlTonUz4rqrxTQTnLyov3zDeiK4R0WhgRTJK3zMMSQsfbAr9ctIKmSsjt2rCiSHeTyCoodyx9A3p0PeA\\\\\\\/NJTwG6u6aPc4dxxbV2SiwHCMuwj514ML7G2fN1Y9KQfQ\\\\\\\/3tEq\\\\\\\/On7Enoc6cnTEHG8\\\\\\\/A7i6Nk6xwarNxtzPHoquALsjPHDEqPy1dzbi7\\\\\\\/Wfnp2GIsDg8X7zXmT1BJyP+jw0SBd9Nrqou4uXstkvBT8C7uW1V\\\\\\\/+kUralRygo+QYxfx9MpWholRa\\\\\\\/O4oFCJU3O\\\\\\\/DVs6iCL\\\\\\\/zCmnzhB2efk9vtalE0bUdEXACgX3CO1aShcOvl7ccRcRvnHsNo8e6\\\\\\\/JX8BP26qmf6c0cdWSWi\\\\\\\/fO10h3JGW8pzeL6T9WyTH3ehzG72kyBbULH7jqN7YEJr7JmqPzGtDw0TEzZtie+bxBBpzq4YnGdOmNHCTo7oUBr3N9oFTZmnbNkDsrN6lwApaz02oNEn13TG23Y1IaMhm02i4EvCvFWPWTf0KpfWfZd2ouZVC0zRTCuoP8\\\\\\\/0kDsCs7meB0TIL6Ash+G6qlku94AUM3GuHemjRGFOPK31bkDi4XfoKmuZg5vz5Jt6lAON3cHiKTD4k5YBHsiGfr8JGubPKERgfVIi7\\\\\\\/\\\\\\\/GzvfsLIPEAbEdA4fXnHvFoYQBkFIem+CL5pg3GPyBV\\\\\\\/zEkLRRy\\\\\\\/1FoPXP\\\\\\\/eZRS8UESqeuxakKGGsVmQn\\\\\\\/LWJdpYdd4zrqPvp2xm8tUPrCChKsNYuEQgASc39T5CYMRiswjbbUGiRvsJkhpjhaVfDgH6eS7V+zba6kPt4gIfURRAOEcYOEuX1LGKMvA0y8oNpCfrmkw+8M6x68EWY2BK8bMbruOqhWxmnTG+vaxgjDQ6U+iQhgrTk3LMQtmcJ+IWzAnyOnQJPuD3VJM89UyxLBRg9HpBbSgObw+vixnXtjjshmYaYQKyulXPw1sSrvCAntQ64xZDy88Jpo6AZIxIWy\\\\\\\/G2kRvAQ6orvtwsyB4FiDAkzCa2kO6h6AaB13Od6bZUrgCsm8xM3RdgcTDia+j1XAZNphme2pLSHP1ClaHDTdeDNxoju61NAF2PhOhx5jfCHJns9XeMJg3ZZY69+XxMNVfHEG4qu+pWiO5OsSyxwPC4fI0AboZ1A2dPWVM+VnMUK7iPPuxOqBRZ0tARZlF4dma0nlH\\\\\\\/DQgYoYnuUx0tzwoHEE68pnWNN3BxN\\\\\\\/JU0VW95n4PApyY7fBuDJJsE3kVksW9vEL2bZkG6myFI1EusU8vmnr25i\\\\\\\/Mo85rjNGdeguSCnPSNdlprt0PUSTwBiTMj7An0Gx95+IGMRtOwGp8DbOge7mEJ6u9vGiliqynYEQvpUP0Eivq4vU85ajLSH1FLX1hOL1tBCRsrh2wA\\\\\\\/INN6NZHXa9Ke320COwG\\\\\\\/kQnI1Zneltfk0RNYOuCS7QphyNZ+KxNNnA8JJsH13bJ93D5P+gZIi7wio08niWcs+6QYFFskuAIQoHqNsu9D7+6pOQfCi95In2AegHGlwjFOMHu9vVYN5HkYY4qytsTPg5TLcW\\\\\\\/KTEs5PpUYRIWJYIZOPUAXZ3HZ3ALYzGirYLCbJ9dAnPH+X5DLRnCT0woYgpIB6PC1X+jIXNxdBU5EydQwtt6jrEt18Z4NA2SKPC6vt2D9eaTHBB4hHE2KQWfngsMmotBDb91GuYUJqwDggPOqqGRsQRJGt5wn8X3DOVZdMfad4lY\\\\\\\/9NUsLLooiKSPFo3bR7sCB\\\\\\\/OHRCLhHQHmgzFwZP8Z94N5PbpIvfQlArzMKa0fYUpqd5SxVGSETavsRP+bAbnErtBwfPpc\\\\\\\/hecfhA1n9qggwgSRlkxN4pKvvH0a47MfqKyZEXTU2AuQrE52Ys01joKt+y2g6ACfLupx+A5TS2y4VqJBRsPEv33KBQ38SOgKSLiA6+i2zC2zBkTY9eQEW2lCyAavpjkMchTKrSOJqBV+jawQeRFHfhu5Sp+p3jjYL55mRSVNA5rTfzX5srpulf+5b2Ve\\\\\\\/oXFxs68svOEdrTO7Jepm\\\\\\\/ckyQX4Ycg6eHAU2m9q2rkdAw1JkVvNYC7G6Mw7PV9+m1AOeWPCeBOtngNNvYmt8YNti75qJ3F2ru6c2J61nS+Fqd42O5D9+ZQrHdaCFOv9M0YMMHbPR3a4fFdA8Pwn1tyabYSvb86IpnSxkmR9TXr12s0259q\\\\\\\/F6Uw+jLPys\\\\\\\/iN\\\\\\\/cgQfvRkYdG+F9qq1G9pMm5UKc8V7cInL6V4+mLSBfHXuriXjhOzq33Da7FeebABMdvQo2VUxrKq89GVD6FVnkNfU\\\\\\\/X3yYWX+QBM4wgsbK1bEAcHvjPzgsRTvYFv5+coYw5jAq0IAgDTimpBwDN6Lhxeqm2DE7NYvqFRJN5tE5vQTl77X0nTDyRG4LsxHV+V0\\\\\\\/\\\\\\\/EczfjXA6BZuyYlZLusRiUPnlYotRM3+eEBwPEu0CX2fLEoWh5xSdcQDlbX5pg79czMcYLmRy6RHLmZrfhrzl0ybeVHUpYIZJpINWU912a5i0mEEg3VL7KjvLW2RVZ\\\\\\\/bvPKZbDl0XaEXB9SrBTX5nBybDGr2H\\\\\\\/PDJ9PDZghDAvMnw00J+1CJ6wKWTEFVE2lEATfLnsyHmCz+g\\\\\\\/K9n5V+tr\\\\\\\/WzqjjjNLWW+B6m7zPptAWGKHeMhweEe92yQEs4A7RuteDFp5\\\\\\\/+HNlbpHGLUUTPBuJ6oDPoxPMqSQhVE8MyQdy6UvYu2vu+WdepkV0am3x5h0WgGPPwlXueoTSaEPuBtSl3Hi2FqIaHoEw9kr436LaIj+lhJkqOJU64TNXt0r07Qbs2l1E+riiJVsgmBvRLDwVEDPADXDc2Crb1uYRIHVkxOtOYFX8po6M2Q5EPtkOJ\\\\\\\/edywcre7oHxvD2ag740G9KCuIH4ce9kji3iBUaeuoF+vfT31SKjatfyT0OPtRh\\\\\\\/1mC+CkV+IkQs7rJPZg\\\\\\\/cskaXAUuryw2Bb282+y8Rd8t8y8Rc6zKBqQVBFBtrM9MfZynfI29thpSprc3lqusB18FUKwFrwwr1mRXeUHWGc3dpqDa8rfKqRHY1erDMgYXgC2igeOL1VGwHQLON+mQLYm2c\\\\\\\/leQTK5Y7\\\\\\\/uKxD4jkLhdqm\\\\\\\/rRRiXYG2lUJd\\\\\\\/WYnmrYksGkL2t9QGE\\\\\\\/2HHQKmJovAHzOq1dY7gUgDZnyc2v821AumcwVhAR00cHtBZPnuuG2YLyeLSZLyp\\\\\\\/bKH5jIN2GNOz9l9Ef96f806uW5\\\\\\\/XStw6Bkskn\\\\\\\/mP+x7x2g0u527d4OVX8XsLU5mVBMikh3NQOBpBEiMahdIAu4ZJQxdhEF9F2Yckm2mdY43Fcp\\\\\\\/fQz7UBCCXFVWc\\\\\\\/ixP66VvRSTswYYllL0LGRtXGZ1wxBwPk7p1\\\\\\\/ysQf3hE7wKuSW3k43TAowtcqjDlwmsq2ROQGxGcBmTGTIvej7X3vKFcNeGsCol0YYpukLBYnsfjHW\\\\\\\/Yvb90ECqGKDjF+5dk3FYKXj2iDrCrROVcSIHmFP8hVnFE6i7z27UDBzOHoTKyJ6\\\\\\\/JcuBeA\\\\\\\/2+mnltCzu5TCivsZl0\\\\\\\/t\\\\\\\/MZFAy3IiSmHEsJMWVC3Om6BsdiovAGQc9K1T+lOD1VFYNTS0iN8ijVrcRSUF2nDwJdtsyyZclZxBijxveDM9RyYW8AcJt2OceAK7t9JEzEAsrxo6ymqBKsjU09huGKwb6c2uyE8UuDKOHgEK\\\\\\\/kwXZh9YbuDJnFPZqFRqDg+WbxcbSa1UjkxvoP3jYerT8oAxqW3NHVVlIhSTxtmDoYZfvUNIS6RLFmK34WTptm1vtCEB\\\\\\\/Aomki4\\\\\\\/Ys7YbppZtVD6PmXmgj45vXQi5c8fc+O4kAL+znucxG3N1d5eRjABiMzKgbQ3DZIF+FkwaKaZgh3X8+7IDhZearNAQqwfClvrZ9b5ocGTN0z5yIL+uOF4uUBtKnZAawKI2L0Cp0OdAWZ9xTodJRq811MDLydikLX1mKevIV4HDjOprmDr13Bl7BNDVrNZuKHKVZW5pMvXQEQBUBY0WDXtXR4lKqjmqjN0JRIyOggrC7zMy4KU5y4PLyX5L\\\\\\\/5gtbtlmTrw2R64k5rP5avoDBowA7DRMw\\\\\\\/hHFcjtYLujWFNUgEQoNY8xOXtkEN4hNS463se3rlCvbyZxThVcnupP9NZJ\\\\\\\/DHC6OcvWcDuHyDDJH6neeOu2SbOSr3PAruzol77Ec\\\\\\\/u+pEtPYMD0QUTgfF6ZlZ8YFeGcatfClz\\\\\\\/Y6+O0PKbwzbveoJ403xyXZ5NCyiKi8iuAYIEONSvbpRZtnu+5h28SbvdfN1zf6OnJ2iG23mac9B3wVyA\\\\\\\/XxvpmLw32IQTTQhcWpwVHukvVaZu08641vnksU5ei51yKeueBHkiku54EzchpQKC\\\\\\\/XdrcetzagzWLFUZB5uS\\\\\\\/GcUgd8bZJT2Jaexc+krBM2mhW3Mgs9AcrpEVthyxGYxJDuf5GGr+7ttKfYkyAGW8QMVcziJj4BOQ45RyTtL549zgvNZ+EyI75+IFXa5Ik1sukeMIJsdYqCymuBXYJF515RD3z33SdHa0TNs+L2WtDHUdubWquFs\\\\\\\/W1vgnhz\\\\\\\/q6wEfbJD5+cg0Wm1lYUz7u6628s7ELr9hJtg+I2F2AOeZD7E4Q69AE3EeBJQM3ZewPJakEMNdGNL1m+iyZA+CSYsohJWTxpGLIpjKcI1otmw8GqqFqvQHbx3RizfWKHoff6Tlvi+FlPyQROQ5cMqjDCjccHJV59nFwpTxsdqbj7MTqH3k9M8EVuB+H1\\\\\\\/jOqm9zUvMIjcE1MoFpXj72qzubu\\\\\\\/yBTm2xoQ6xZdGT5tfb7ZxzhmWFbcgVGMdRS7yuPNH71TfquLtBDFVhONxXlcRObgSosoOr0u\\\\\\\/12XpEIazTdoyowB4QXYy6l3GpR+iXXSxDi3QUgzvGbuUGFgVxQOxRJTzPO7vUm\\\\\\\/YfKaiRFN\\\\\\\/E+7v9e+Jn4726AsXAL9wAUeMJNBi+JwyWsOhUvk72dAX0FxyVh9Sxjii8SMcmLZUb6P7PJtZRe0aiga7h0d8NswGBY+Rei41qbAg86FIGm5nX40MnyxgDQ9FlwFICM\\\\\\\/MSncrax9dG8hf0BxyAnQIP27gISSndJxF68Rel2dnHRXGCWXeICMZyOdLx0RhhVsQC+h19YOWD+oajRTucuduJgXCO3D0mmSdhxNi+D\\\\\\\/QRl+INxmSzwBs7fUfVV2pCzyV0pcKwA9bfIoAeTeaq6juh5ehdM0oohYrXifTmMZCPza0ySLqk03nXrbQvozVfv2MWBCSW4WMCcEVvn\\\\\\\/tjtAjgjHqdvlXtJlnyc0kN+a3njaWqHqaYT0eo+\\\\\\\/QurV\\\\\\\/\\\\\\\/npDCnEemcPGJd9sZRG7fCD78RU7cCyF52hzgvwa6OroZ4aAcp5XhIsS+ZdTVy\\\\\\\/KS7Hu\\\\\\\/Mms5uPEpDczNJRJkAZkgp6gWd+YzGvaFMQwdAvyHdJrSw5q45ZJsHCK4EYpiM\\\\\\\/OEFHBb6azVUj8h1tSSbR64ut8kkipq45CUf176PJM8OZUX3b3jyzmMB\\\\\\\/xiw0NazOGa0jBTy4Io3htWZNGvA+MYU2\\\\\\\/KH3NZRKGEcvHkZtkK4foUitdZCSqHv1tGnPVExPziEq246bjIIOf73jsiK7gVc7TQgFttJHE1n4SG49B7GAyBtHeC0AJVfy5f6yp4J9FUoORt7FcMvJTIDLUAbIZZxp8ul4sCPBKKx6vsIKNZV96y0XPANk6tNwyCmEPsPymDHSweCk4I5Rilw2+kuaCqWNqOE8jVwKkaFT2p6S00Qz1AtGHURsqwGfVJEEjm\\\\\\\/UI7fQMbQLLxxlHEE0aV9yqff4fJ6iPwBkatfXSU+FxrPCigWy7eLO4Pba4asQNZiO7VdyDXWCKIoE9xKXRkNDV24wiXvLd+v7QluF7m0EHh1CJNdJmrz41Ur\\\\\\\/3uzNa+KSKz+O8HOTkakzbgkC1f2B6Tg6gidVb+UiPfLGF4CKEzV2uvn7aKQY\\\\\\\/NtzJxSHJXobbr5HF5L\\\\\\\/8JqNcSXjWMZd0793xqwVjsFfpUfzTHUKunJ5hF1HVi\\\\\\\/vX\\\\\\\/kxSD+iq68tUkV2BOL2TtJCeQm0tdtAMtKZ00t9Bi1j7psqtEPtA4AXc+vd8i6vYKLPxlneeMpwRlp0uRxNB2AavOitOLgdTrN5u92iAfllJVj\\\\\\\/xv3UP59T3YvoJkzMKFK2q48145+FKJI6uiZKhKy0IejXh2S3KhRuakN1YYOXWUXDTr9sSWq4wPvviKNQRFvT1w\\\\\\\/CrdzOPfpQgZfu32Jum9Z+rXvRtfdwwBlF+vAOx+4xSRHmbGkWcvxoKUJQKsaafAe+oUoalMkbg9iKaNvGHIdqoxUIbUmnCbIFLh4fT0e4ScKx+1nwI4bgKp8P3c8FhrYnhnn4MTydXPu3zM0pi1h\\\\\\\/aEg4Q3jDZainym3wAbZGWwGkgsQVHuyW1c8aWHbC0ZDorA0Q5fBgiPQ1RJK0SCMCD57r3\\\\\\\/0f2BZ+o+LY6GPeSvRZDFCe2CIKEzcCUN5tDomjwEy5tfNnD0JRozOPAb\\\\\\\/RCzNi3FKE\\\\\\\/+zcK6FBi1Z9vnVV9CNFpViTYV+o1Ui7cPb1\\\\\\\/qv8uOM0M9n\\\\\\\/739QqlbYX2xoca3Qbqw1ZU+YTko9Y7O99EIA3LUZJtGqUyApgfDShgmLj8oWYi2wJSuoiqO3nwnhCxDRqhcWqz1x7OCuWeIs6E9lfu+fjh7zodEzCtpi\\\\\\\/IbZXoTCusOAv0qmF+R7XQ24OZuZaUIN+yQBtwfZ10+k3QMQbB4JhCXtLUw7nzM\\\\\\\/5zfvRv4ww8fzQPDMfo1qAlyVEOeyDP1prSJTlXdywFDFdLIoX860HyYwt\\\\\\\/J\\\\\\\/1KH3ZM2B1pSjtrSK3m1gCojuMZsuVc\\\\\\\/47dOUGYb\\\\\\\/B0gKGuz9slwPj4U5m\\\\\\\/QWw78WnakDqGIkZqJ0zbil8tVRqKFBi\\\\\\\/d1M+Zar5+\\\\\\\/Y0R7uPtT+XfvDiHNVJ10XLRGBKiHebubda7CPLCrZTrUFbaMiIpjHeYH\\\\\\\/QJ85JHzM0z\\\\\\\/71QbFb0UbjXA1bkD+C+J5ce8TkkEybUhrfoXPoearRiYS56vRDPaLs875olaMKCIQHXxfBDGduzUenYPuN7xM39pmfFoMpsFt8JUgw06dtpeRyUzZZfwO4wh7HSchi7d4jTm\\\\\\\/FtpOEV6i6NJpISHv4CNdkonqvN4TOAt8K3pqP3mj\\\\\\\/w056pOH\\\\\\\/tUJWmwG4pwH+Mjugnt6RSv3ue2VQ9lENwCvJuFL\\\\\\\/Ete23\\\\\\\/Qqo0L4Jcl0eVtuusJW7C+sVEivwzqNEoKGmBAMoABapoIvBxAJNRdvY8IvCohrnPUriDyhXrwVRhHz5knQKyLbCMHHKyhi\\\\\\\/R\\\\\\\/xWAYlO1rZx6La30v22lIQpIgyFP4OZ5OT1WGgZpGAeuLZ0W9l9mTxnPgF8q2IfOL1hhbH7mUSFC04PTFAh6MSytYdUeN4rrudtB9DIOImGuWtiEIbkg+QzcqnwKrQwmx8EauimLoHkjxkW5YQxpT4u8B+\\\\\\\/ysRIwLDk11QgMoMGsCEpL7Awz4SHre06c5NiXXc4V+sJ21xpFCwcWgosq8jOtwLCqfI9mXO+Rd2CICiCnh3mE9TzJTLaorINK3KCknxm2kNBZdXBeoTtUINJ8CTp49vSgRDZGKmg1jd3AwgdUfFvx7PWiLr\\\\\\\/sQn8R+\\\\\\\/kzfTyeWDnsnuZcJbTWslYKOOWmhV\\\\\\\/gFFQNzB0QJJH+jUQ5fu+YR+1YQE+QrAonmjnlFJkrg0uHg8UN+MBNKxfl5qOBvS80akyqNHn0QSND8FC6BDToLhWM0s7bqMd7DZtDWmie9H5VtOEnMXQ0n6itdhAfWfSbT\\\\\\\/a9FRBY\\\\\\\/PxGoqB4s2CAxdHmjaGcq+gpuQczD4q2iZxFnLPXLcu1jhQAEvPavKQM5YaocAllMdvJ8tg0nyD++iY9\\\\\\\/lkUbknKMzC9on0TfHYk9kOLUxpOgup+axe28uqjjnAtCHfHnGczrdYc87ohZkyp7vHl8cj5vA46ZkacFG\\\\\\\/uBv+9kA41bCxiZPCxtdKSZIGbOtX64mbu454esv\\\\\\\/lk6jGq5JoKwdPL+Vt9xyFs8\\\\\\\/EXqnGv+s4pjOm8kfy30I96DepDrp2M25B13uCcXie6pOpWSJFzjq\\\\\\\/hce\\\\\\\/jY6w4hN1s7qA\\\\\\\/PVhYVnoLBSVjjmiRXBrwkQyJVXwzsXqVdeKMroTdn3fS+xfuFGbbpWfcxaSfXNRCphbFTlviY10cIeKk6QmdvcvwmLXT+a1Vdxnnb4vcUiEtZlKXJecQT98ecqL+cmLQceaVS236MMIUGy7x66HnDOsovtxdtVtQQTsLXZsnVkbmvKonsVBDwl2kqK1u2oAqLNoDu1FmN\\\\\\\/tAlGROYxqUXipwQ2zQ7BMfd1sJ6hCW\\\\\\\/RzxjHYGmfuKoS5MWRv3kVg1Dj0XgbEvUolzGk8NiSMjM3kcJ9Vecv9NdJ8J\\\\\\\/gvV0kTXt9vvT1EI9H\\\\\\\/MYEcdqW\\\\\\\/WmHEnRbwkjvejEUyNEWy76QEIw0oFZXTjed+\\\\\\\/Jab6Xam0zrRwoc3cCsJmfgSJRPBU2cgYtxIcDw78DnmiCXhx19J3QRO1BvjJWzLTNZKXjT3qSk6jAvtZuteO72XZlxAKfjYx6OjxAc6p5q4GOVfWWoZjVbHHVOyYvk1t2nQvj04u8SmxK1PDMKmYwCivntI8HhOoLsMVtJEHNp1xMbnwp6xJVB\\\\\\\/Rudsbsrfo28+LBIMqehpigVb6bivrt0QXoyIZaQVDsrkaneocP7EwW2bbLABwJrnlXDTmALjB\\\\\\\/1l0cb\\\\\\\/onS+c\\\\\\\/2DJp1M3hWylwEBkTuPyp5RyYGqvX\\\\\\\/9DnCEbXCNtbhWpYfUZMw21PBfmHKYo+4TzSp8cAxuiU7eKWz8rtfAW05dgjhqFobSd5TFSdTaoW2855\\\\\\\/TPlH59D0KL2FlA+kHpvk\\\\\\\/FZr4sPbnBeiNfjQEWTLVXusXY52su8Kull4bPNcxvbDrbPaqWqmpwDDHm7eVfz7c1GLUYHF8c\\\\\\\/N9Vz3olrP2QWS\\\\\\\/6F5eZdFT3iE6c8KAt6ATXrf9YN8eCEAtWHAWKLi6S+K7xqS5iNaTiRJN6i7YoFU1FXTYBcmYYarHrvkQ6ZC0Ntihv6nw1SPc4M5SU976GcIVrvS8OtZjrsvZp4UoDGUgihxRjnGaYZBAGhfOrld6670v766toF63Ud8+E+7+BEpDTj8DFGtvi0Zej1uLYbSgqtlfWC9FAk9hfZv7248fhgYJDKI63q2Jxzy5emo310mDduTK4eQgPIVPrN7aR1lupz6vvU5e8O1jKHXUIMiI5wuZW+bFzSokdTUq\\\\\\\/TDIIKVfNkDMaV8JJSLRvC52MLwpAxsTr80PjwFZ6Qw44bSAKBFyf8HGF8qMTJsVY++QwXD+h+Sp2onO8ykXM2z6COTE76UJK4jW2qskChCOfW6NlFLRWTLZSmx7Nxd\\\\\\\/JSWEx8VIa8SKH0LP69MO2UiAUkDVQlxt3zDSs48l08LhI+oL81BtVBXk\\\\\\\/b4V5tO+ZOOhn1wFeRcBFbOQBgZ6ZyXm8QrhN00Fb+zNAy+Hj26kc8HBQv0Vi\\\\\\\/9B1imPJMpF8CEensDPLqwb49mOgbChb3QZ+dsjVeKimIvV2zgag4gQySAkcPD1Zbc27zH6FhElfLocdan4jFkzyEOf23YdmlQ7Ch0keeXLZb1KMO1B6eSF7BvlB394OFjkuopAX457DmGylhElSrUcZlMhuWt58Hk28ouO1zUVQexs3kRvzmIsQuzUyNR2yc3Ke6LaqyDm0MauFj3ldr7GjBfbfsO2v9tsDIsQNpHboQRg2cZMD3QCLagagskqhA8ZBSgE2C0GufbpW31WkVCwGIPKPRAFXgb3vFRq1T3y0B\\\\\\\/hZlqXmnXaTqRfHWVmpwDPOZNTWGDa3KWkv3MxUgUjHacK8nvOKrqDu+J0Q4H9WbdU74jxLNIBIZHasU5laLYr\\\\\\\/ADB9blhUG3HKwtNE5Fflxs4Hf3CDswFb75jSxYSCxceiXshbamBaaXN8XxknaX7gfPSlJ7u60FRYieQVx20z3zBiJwSDPqjoPzEAxvICLXxLKkxRRDV6Aevx\\\\\\\/OuDOsHJpawDEPqLBEIUnz\\\\\\\/RqVxcQM97NR52U450W1GNW3JHaCQ7lMVrBfZx0XoT2eE757fMe31SOp2C0eq6xfWYu\\\\\\\/M7zcbTzMpiQoBugIyHrjMeTEcQiUE65oqa0MOxU3znnSu\\\\\\\/CUKGeA8DtvGlqG3h8tEWZVyqBZZJH7HE3VrqABRh4Z73rAdRBtFuv9Wd9j62lNW7ChK+TK9FyOdaP\\\\\\\/+Jhf8PlUSj2levRJYvpiinaZy+UBmGJIJksPxYFVLoJ\\\\\\\/kawSABoeY\\\\\\\/7DZUdsk3htLhA0BgDkH22T9YiyAvAbQB9bOQn9iFpOWJ9hcZtRYwWDlfuv\\\\\\\/ZqBJGT3cnXBq84OUV6+5Pajt90CgU4cBUmkt4EDbHkDAQe4CImwSJcVGaksa3l8SZ4aw\\\\\\\/ojFlhNU3xdMJGPus485x9yFeF1IKimOzSCGELXGBY0Nel6djFZETJmTIQP0hZbYmZZ+HSES8L6B5aNPRse\\\\\\\/TOaPiwkiNcA3Fra6bSpHSX21rsboQDHzKlfggyCDQXM4nCZz3FknbWa+eVb0Vqh6I4yI19U99J\\\\\\\/cbT\\\\\\\/1mO\\\\\\\/fVYXpR6Mx5cXgy\\\\\\\/afMoENh3s6Ao4szJTzhn4KRy8qdIYM7Mjm0nES0gONCU2OLOE3U3WcYVzrqk8B00a8uy60QrPy6be\\\\\\\/QsOBAVtZp6yHpqZr5Cdxc6K7YYYvACh0q+8GceWns99pzDV4tBaO51BnQFlqjOczh9YWLDKi8aXOtgVFUaI8ngSnXwJ6cvDBrnAZ4oKy1PBdPW4B6YYlGcSNHylcuM7CpQ3cLfwhlY4GdP0RXzW38DLrOOHXlETueYc\\\\\\\/ltcDa6mQrMGaKwltcApa4+ICrcOF2fUAnyIlSHdLJ5e0DbaGYFd55f2fU9Q2Ue9c8Z0bREs+lu0zQNNAduLy+mdBL9DL9NexmevkwWiA4fOFJYEVcZjNUNjS9i706XvyIiAs7F8SVdiiC80voKdWSAA5cif3URWIOUlzN+OeH7AnrSzXg3FBe1CRM5FvaisNTSQoGjWMMCB\\\\\\\/hvtk5OPZri7cssz50TO4Hf2jNPVgm1WOqSD4C3EDnwPkcUaOB9Wwxnbseh1VYiSdgt+DrSKBQTzMTQ8vGxZzr0hik6iqK8DSWnW9AQXFDicAyECqMWb1QhtiNQNW47dO4sXGiA7Brdz3KcUQDzqqb0QOqZeDpnukfrJa3X29Af9vBW\\\\\\\/\\\\\\\/V12QpHPz9OisUGpSU26rrUDUCJKd8p\\\\\\\/xp7oHvKpu4RFcOjSR3ZBx5WJBNx8FK19idwThIzWXOP3RL8bMry2xaDQaBbCyUGFsMZFeCf3QErsg5TYx3b6+UfoLjHlwfoTxZABV8KBqbxUTHGxqdih9Gb6qhNsBUPLcMuiPpqYILKLacaF3bQKz\\\\\\\/YRgR5akfAIevPVR9anx26F6JQSRYIp6WEi0KxXY6Y6SZ+hYIoOGPS0D+aLz4ZuTn06ylqAdU7dmMdXynEWPvq9RMnGWGtvk4DKFJ9edDdkmzyGDm+7vaJ7P\\\\\\\/iVlnh3aayftSm7FZAFPgx79SUj8a8Y+wUz\\\\\\\/JM9EcnlEYPJUrQ9diVViQ9VasxjE5N84FRqIFQ9JoCxMdByovmFGyPthgreCZWoyyn+HQxhBtyDhN\\\\\\\/f+DPXahM40XBt6ge3VhiVOa2ogmAdlTSrOeWU9qibyGEYZyJPyzpYetKWddjAjz0hgLOOJIglmFeqIjqkQ2PdL1B53MHDRDbX2LH9IWIRzCMne\\\\\\\/+GIK7hhhZ8Qvi6e\\\\\\\/FyEpwcReLgf7raeAs\\\\\\\/1ICT39bpguksNO0NpBGqpynRBXuqIlGMrtb6Jq5+vzawT4pRpEUcLF+zeCqdAvvHQVydRBtlY2+CyMQy9u40TXlg5KqDfezDKp5SJ1S4YhBGAuPBEwMHM4JV0ophb3Kbu1vKu2+60DZYgywxeHtVdvr19LSCXNONEgAhZTQRo2Qb5NjO8LWcnC\\\\\\\/qYJF6fxuC+nLE740t\\\\\\\/gJ9\\\\\\\/DC5RoetTAGDSQxLoNaJIzxYLx4EAS951I9mThT5uQVpGEiKbEEcGZpJJ8lP0rNI6ishFQ2CqdoB9enN209Suqn8+rxZCPsYo7oh\\\\\\\/fs7cgkAwEAmZmjWWuj3neTirPnEHqTZO1vJTBWbfsY15l0yDSL1nYI6mwNCI4SVsMjKDtbg\\\\\\\/vgUwK8JBIGBF+SNeKiXWRCfBEfTxNbUUZJY8uGPIWK\\\\\\\/r3SAUS3J5niOhD6+uBFrb7cu7ygKTD6LjtM+FZrIOSbrzEzjYGLH1uXPALalJYNvnzt+c50l0mQjYqkUYjuVObSRIQbJUXSvXAcY7ewtDbBh2An+zCN8OQKhCK0jner9cN8MWgbQAAnFzgi9LesUrN86KIPTS++PAu1e1wBnKoNkLKSGTU5jjWxA4t3nUmuRB3\\\\\\\/7ep\\\\\\\/8Qtd6iv5NcPETsVcnP13PtLU8YpFdrbVH9uhcG91ryjjwtzcFFyQsag\\\\\\\/H0cmvtthY92CbDeA2R9Kau0g8n3UJ6ZnnDcpWXJgcp7CMPkOL4tsez6v88jS0yB6A2hT5aW\\\\\\\/Hlxn1ndjCJsDLF3P7kDGKhZ9xBzhqOT35xyCfAvkGC1\\\\\\\/1l3Ui0bJyR5TsarIVwH6+PSE1x4ohkAkMy5y1jOsydjT2j+Kc2nGzWTbEuxSeKy9B6IHJ8rrj3rLOFpLaN7ZHkFEdN4PGfNdEotf1SriGANAfRcgd9hO9qEMSxL24b6nBgql65mJJ7Yd5uXKRZtLH4z9q2K8r15r2kwjVNdvoP6TqZf7FwfhFjEFTGa9UziPZW1lhC\\\\\\\/pAt5kOE13uODZnXuGgGIqtrvZD\\\\\\\/qKRL9cSLKZK8TqHOKzOYZuic9iyg\\\\\\\/dhKjq6O\\\\\\\/cjNm6\\\\\\\/euWP4LTtvTlT+IqoKmcB42NAEBD5x5y9lPZWFi03ViAVBSHQ0j2oG9UitGCbOXbUjf82alJwHSshycsffi6nZDWaFhXX8PSfa8MRDcadCyFa76bcAqtk2n4bTEpj0kFeB9oCoQD4Y+ilpKzBj6a6g62bux9+lRsH+bYVr9DHemeBiuouRd+goAly+y8G5zi+4RGgjdiby7LBZjQ3N6IuLWdNBRo0llcekjiOK5p9xYcRhxYhHN5aMJ0Dp0TRpThse4L6xOjBU5q9cr8CU5TBmmGip2T71MSbrppwy0Ud+XZeRVcAt\\\\\\\/ZnusAzQ9m1UQBZg+uXXbmFLR\\\\\\\/GFF73+mvf0dvmUW5PDZDUMNC194idoJ3cfypNzhsMLTbPfkN\\\\\\\/6NpbLL3pLcR\\\\\\\/sCaboVubAhFr\\\\\\\/Owl7ep8sQbXqe913P+EptGUR7nWfIToRSP1ufNT5wpLanuLIelkUvScqR9lX16whXrnRbqJYama8+1WinmJ+mW7hPg\\\\\\\/bcNhZYG6\\\\\\\/GO7jN0biFizWdpU32d\\\\\\\/\\\\\\\/HUbTb9MbE6OFCcQ5IJierTplaoIt8V\\\\\\\/qEACjYXvX7PakBxL03BjAxjfPSnFwLc\\\\\\\/W8dvtVMBtXAnfNeQAZWphlHTl26n5rxJPEvI\\\\\\\/f7kGmWgu87QyIRefebliM8VDPkMgiW+NSd750ml+T96FWO03H30LnMUbYwAhgQmIwMWCppiQFiiMkmvteV6yH5GO\\\\\\\/yUqULasQwyPB3bSNrd2OburB0mxYZ9ZNV4IqYXVVlpaWEiocPrOZXmbDUL72ILxalPZpuNFInxTe60laFa\\\\\\\/W\\\\\\\/J4w\\\\\\\/3flYkQrhRFyc9MCjWKu6SpCmKYRLr6kBgo0ERlg5A9jICTuAx6F4w7KnKuv1mcwSN3ijgchPf6Bpa98TdxcdMsA4v5U1DY4yFdHSIXXClcd49q6L8m1AgIPq8lNiHZpwIKFhGNGMD2xzYkBTFQHg1cj+yz60+4bzO0QASqhYJ0pP8Aobxnqn5TdkbJvS4B3uIMC26iiM2QY0tU7nVJkDfuZNz1lTnNupJ5SB1wCHU1R6sYEfutlICb1kjQrC65ewhqw6pe8oFh\\\\\\\/phK8qWken2lWLp7UlT1kw+nG4BAPWyvEqe2ji5zpCjnkPsxrCroFDtw+sdEeguytxi6sUoQ6p1qJH+Qgxo7V1me8nQeH7mdV07V1zv22hG1LY6qoo7KpWswdSXGw6OcSbwUw\\\\\\\/siC7yKeOKnUyVMzJmp6IOgD1nZsnN6Uwp\\\\\\\/XKKVOwNZcPNUsuh7dbGqpEWHbJC1OCh0iRsoCxR8tm76eykQb+r0i6xzBZV9y2xXpdpwrIcl0yNkz1QPnWOFi3AJD5y\\\\\\\/SoyfaX896+VgjcWhhLfXmvWu9s0oe8T7MkaODp01japduTM++K5zhfI0xGLa41SSGM+h4QZC6NFwlAaGJtaPOBFmaNQzHP9irkBBmVz4XzEAfE1IytRPCHttvkkyKbBxq8BOFIlg8bZaFM573+pSo0WzFRGzOlCDTsYcQ9sUUNJNjcYJa3obvTB6Z8xmvPOKTU86EgI7G0wUghxCUdi+LqQXCjHHd8KLvsnhkISICRHCKgReZ25fwLiSyamN3jyUWRUbGcxt74nbKkGmPrkgU\\\\\\\/UI\\\\\\\/cn7wco08uJPnuaWIx3ttcIR5AfSba8oBtIEquS6AyAngsD9buj28iSTYincpR3OVBKmd\\\\\\\/P2HmREHROThP+UQaGiskPrcmRXAB1sya1opSzVKw4KYX26WT6l4w3UR7aHhijBQbz0vt3z19uD7CnrFGob9exT8TPnJ8rj7LHviB8ljOnvvCnuOAP67uCYMS0mMO4iIF6EWH6jttwe0u4sE1cf5DdOqKinAl\\\\\\\/BUG28cVUanGZ3yqM3jyc0U+buB9OqH9PMKMhtv\\\\\\\/hJ2shIP6BmKXwR3waOfeB3H\\\\\\\/AZrKArMd44zoyJaRAmSJHdPttildUwV8vWl3kioXZ1Y9jY7JlnqOg0F\\\\\\\/BZ3DJLUe4kfunY+x3P6YFPV2WKyss+0Rg1NWJbwwRtqlwqzfwvKZuaeU2aqZNwSAX7++Nl9BsDP2Y3UspD7vqaWVCE2LclQwp7eN1E60YczZejfqZvJRDO+yiYB\\\\\\\/2GJYZ7E1UFLINoxRWKW8zHUx8pWFD5js08jaVcj4aKY7yS5Rr2YX9AMIb07IATqG\\\\\\\/zy4Ppicb6PXqFmcEqXLtHtT4XY8SDFrnZ3s8jIjAdyZGznCtEADNDVthvlVuZTjx3ynBgfZyC9eaY68\\\\\\\/PKXcfr7pnx5L2TKASza80unOHJhS9Cz4lJdtnZXS\\\\\\\/L2hHrdHJwMTiZuQnjPIDkSiGL0govGFSarWk4P8EsuNI5DqdCye4PkRXkRklNCDmDhXfH0o2J8TwTYLV5NiMIvWeuwl3yFeh34V2oWx82QjNues7SehgpuUIjRD0qLf9OwcqkxjRXUEb+pO9tI2uUVLHudvQqHHlhot2S4EoGsBM\\\\\\\/4ynOpRKZerpCzk2Mvezer0yZGGHAiEq6LsXdqgdMtASUL6yCPmuSrrWkO8zesybRmtVKDFIFq5NuzQbm5GkCuXulSdXRFJUoKS+Tf8\\\\\\\/BO6FGtDrOIDoJHjeFizh8Qh6rOhu0ji3mLU0L5G5XWv\\\\\\\/ruWGDNcm50r5jjba9sOm3b6Wju1XiFA0zmAxQCf8e2a1zCWOde8S7CZcXy\\\\\\\/UpgfLk+SNaGrkwme+\\\\\\\/Dkgh6BOnTcQ7KJfRueRGX\\\\\\\/WYCw1PmGwlk6Tj\\\\\\\/wgmT03pFAvi+eeNYahqRnjcUOUVutjCALDJXG8\\\\\\\/PRs2E\\\\\\\/1qzZBfvlBkhrQ3xJOguRVeXi1EHlc68oR+g1x05Xr6Jj6Mk3iS4jzAkxkdjIRPwDe683IvkI2sabNTnHqppvo2RqmmN\\\\\\\/\\\\\\\/O5WPUSgDKOpS4bQywmod1coE0hJJTWxxpgkerNKtT8k4DHURpotb9oIqcKl7LAOJtycXMO1G8c0uyOP\\\\\\\/nbERnH0fMqDW2FnwNOmrTD4\\\\\\\/ttcBbU\\\\\\\/tU1pYDjGfug3XBXw3uwnRI6XzCJQdHEKJ6WEnbvcqm\\\\\\\/a+LZwHdY+HyEpvXZol8FJhM1hflYr0s6dwHWnrzo8PbZQjyKHKt0UhAf+JPUw1n0xaCZgyE2FFwaoYMTdYEjRQC2YRfyFmSeKlBPlUAVKFgtbCa\\\\\\\/jqRa5JqCCUVQbAY2EI1GsoEXll2aprQ5FoOcyXkpkofIT44yjVbQ1ijcZNYGdqn+SjoEkTCzgeUaJVYPqIijVkMKG\\\\\\\/PKkcIBn3j1g9QPklcFPlpXqtfg5SrQl3nFjCA8ZwgZ9TyDH0q\\\\\\\/t3GgO5HkIPygd\\\\\\\/S\\\\\\\/4Ql3LQqxJt+RxUjtWfg5wyTu1S+uGwnCh6GVig+6xfZ\\\\\\\/Smbv3CXRMia2gRa2zoRbu8FQs5U8frBRi7GlM3t\\\\\\\/4xSYaUHxEz8TLUCUGuQ3QYAeheudrXsunpqhx6wZEcSGqkEoDvXTg9wXcyjHC\\\\\\\/nU3GBtCVZ701Hlm8Vcyfyc\\\\\\\/yNcjw9cm\\\\\\\/PNieCxfwPZWbl1bvhvHg9qTTp3sUWn7FvQWiL87oaxksbHiXB\\\\\\\/w0ACB67Ier3lMPKByJsTbCE1LvsOH8EBulCPd+9zfH2lyj6RgCBp2OYSEX+yUN9pq9jOYw1AULmwwoPm\\\\\\\/X4\\\\\\\/5GB1CJe7ypaYsjYwCteTHYg4A35oIbn\\\\\\\/Owm6XmsXaa2e\\\\\\\/GwH+hCajHxvN\\\\\\\/SyNdjW14rnbqnci7T2XWHzZi1Ai9+QuTOVPqGtgYd3R39w+nlkgBHyNi\\\\\\\/7kTogBFXOnz67BAxJUOVsnLrBxQIJ1mJcdPcog9ziGb8qo0rc1gSegRI\\\\\\\/zh0f43o\\\\\\\/yAEl4ePw\\\\\\\/QIOzkqiP41k6QlWpS3LBWTZzDRNnil6pHlnvYq+bOeK9iKOjJdxqZr1SYfdA0H6hpaRmZzzx+1Cs\\\\\\\/oEv8pe+qxTjswhSsH2VAk\\\\\\\/4sU2MQOt9svirth+mIDAvq4FeTA4AQuHLb9g4KJ3DN0C1JWHLyXYz40jgvVsFib30OXHLf9wj+Tlxs5sbEAvyGmg8YBhydEFsXOmTU8FyxK5bfeuc1lcJLlqjLRZfQskv4PzcRHzFpkA8ZfqJIik0WSuBw1SzxfPWtABe+s0hX6qm2iiXiMf8uEq\\\\\\\/DdGznflaAQcLdT9XF+kXtTp4wbiDHicD8R9co3AOpoBO2893aVmgMRRWzsokiQS\\\\\\\/2uNUT0tEBnYu+ep\\\\\\\/eEZCxaaj3\\\\\\\/gcxgI6QD5F33q2klPZcGDgf2cKpzqrterfvE0mbOgXu\\\\\\\/uScHvQUaVrHmJ8rtMeYlU1RZc5u53IGXk6KExeVlMT3ILuQuONBq3VHwWjHR5dOJjZekmi7bvi4Q8SDlXR4aHCoSsIWGHxjm\\\\\\\/3668eNsS2r6Q51XiF6Rkp\\\\\\\/zyNJXxZqE0RFvpyrF8U416ZSdExNozCJkULDXDqoUeq\\\\\\\/9jXNpxuB459ozAWwIng9zEu\\\\\\\/NR+2GFBhKo+9zVPSokuf+lOFsaJnuz3EQgXbGXo4dNVrCH72t2W4pvp00oOEyUT7fCQPIfXSBVnYUIysUqrvzbcVbx6pwK5qMQ0T9bLK0NjJ\\\\\\\/\\\\\\\/5LH6VyjP4AozvtUtwOLgCPIYYqQV\\\\\\\/3OWB63nb4L9MliI1Sz3D30VcsEt2nbeFN+z+AFfrx5JaSD2AEwy8GoQKQDpIpMinSJaZ7ur1NOP0PwKxVQtCvvsmM1kBsNUivcRKgT0a6GSxn30cofV9nCc9Za8DhG54z1egrD9HZ+onJLzeZnW+Qf2CDuENeb4\\\\\\\/8c8fIMtmY9T7Av3rGoi5QYiUlsFiCLPo2uQCHPEmeT7aCNPyQBqEq75gzbtIqBybuHIp04zeXuo8Nobu\\\\\\\/CZY6Rtyp7wr09H\\\\\\\/vxKXSZJzktrFyribeU0VS4YueuoG72+gEwx\\\\\\\/0wBURcwcKACWJTo337p5JwOMObXKFqGatI2SO\\\\\\\/5IcNe6LnXI3uhMne2rxtOPn2nwXx+dJLkEl5mfiJekeAAw5+0N9\\\\\\\/Fc60Aj+DvIquVUP14GOc1iXIY3cYr9O6DAPn7kV1zkXcBkwKTZe6qE8ubZroImkJtcu+h2S43P9A0uV9i9UvQ4QR\\\\\\\/TcAxNl+RhP4bIcAlxXDVCiKiWZUE9OFCN7YseaYinhXUA9f5OMZcyunmKUv9TJ1k3pMw0UlBOfUeZ7Sf91\\\\\\\/1+zKuFAPjr9\\\\\\\/6vWCiHp7TjwXJBs0PJEYFT6PkqqSHcQSOSgw3\\\\\\\/MJnN95ehJ08PKPTPA6NGQBAUGDV63VvHL3JrOugheS17OjiaeDu59wJpghcLJx+MaIY\\\\\\\/8kMQgJ7+djoUiP5KWKlc67wGjJ\\\\\\\/uYM\\\\\\\/sQ1JsQXx+Tg78qnrX3qmLSEXPiyPL2yuL4iNN22AJZlMLWChbwVqDceZ4YGzZcwHIJ7BdqlUzHmrWCu\\\\\\\/gI72IvtJuRn0djWzlhZTbkIJox41yFas1MzMVBKFF4rcAVMwTJPunHk4n+gNRq2hXmSanjML8cv6hTxSh9TIGYoB03HlIThyWoLH5243HQjjZ1aqky0WH3IH2RzCiTsbQqS6NNC8p3waaaDI+ahZ0i\\\\\\\/9a55xqE5lZdhbkBEW\\\\\\\/vY+TlvvNev+ROCSL9xaJl6W32p4+Irb4cd1S+B6HK2thu6\\\\\\\/DIT8jrVjT6BCk\\\\\\\/1eHICAisBo0ZivWL1MZASL7VgIsFibONYt6C8DqkMCR6UMbDd2Ip0l6leozQMwHVieeWHvE4ITJaehANc3VNW5ox0nsyRwoMUwDfVQmMdxx7VtQXNX9YR7KobuUX0xQAu9\\\\\\\/ky5icU4JDGKZG+v0cPq2Kse7snPsvfd+qvuTaiEP9iDZbtYyxWmpU+YW7+aT5z+J2bZ92Q8q0v\\\\\\\/HHeyHfsncj66qh2m77Y2s8gyK+jKwuxParXNThMGZ1n35jsSAYDOPUjPPP1NzuQzigT0smNx3oGfJltQTmlwoqUksVRglOF3PVCHErHF9mCsMwnrrw3NqDu75PYlltom8pAOgWp3uVQYQfnPsfQzHJjOFw0JADoFWryV2zHv4QUnm1W3Usg80QflfQV2T0Ar4LxN8K6BegId7IItPTQC7SC+AfYPAQWDhYaLuxFEFakqx2LUEJnZno6as0VFHmYHKd\\\\\\\/G9gkRPgzrfPumozuCqnivkCWJhRY1qq2I4GScY8if0EbOpADhjCAQ34XQHTAizLczTnNfTeSOykmIgsWRUo2GaxPGPVdHIeKnz\\\\\\\/e1Cp1fu\\\\\\\/FHYrA0E0wnWpnUouz2ycK\\\\\\\/QJ3nf9oK050ZJjV5iE\\\\\\\/F357bnHj22gSy5kzC0RHwI6k2cHDiocewSfUPtQJEagTQBSvGRC9Jk3cvdBcxBtJ8ipTTZeWQdAoJpRLMldFbssZg6A3w6C2uNxvOKlMAo+Ua9\\\\\\\/tJAi7oKlI2Lbc8BkjIpuVcABWfZCRPPkK3Zn34A8n+vFfeidPZ3vH4GjrnoQLXKA85D5OKRyJVsjiXUROxUWgI3G4XrcL82ytemT6aHj3oDEyZvVpYgFpcFrRAk5QgJE9MYDREZKQ0h7sIEKa3nWNGLPebNgSgodjHookPFJaNnuGPqOzVDXXTWJGSsE2wViAe49UD5\\\\\\\/N7Fg8ZOE2SN3csD9rIYzU2ea5hyPzcs\\\\\\\/AGXRGL5jdL5OQC4r3ahAY+yd2M82GgXpGuljUGXUugD9cgjGiFsHZkRKc1bQdGLycGtzmdJt7AnvW9FaK0yP+Wxms3wwdP3jSFwQk+TwLbyH50N79seMXgOGlUorKdHdggK27XHJruZnk1skp4CQ58ZBTiIDx\\\\\\\/VZNmST3b5oc74L3T7gpGx4v3ULFbL9fGBO4oLQNYrQUd3igMcfXv15QS29zoU\\\\\\\/E1xjruj3nLP793li0dy6Yuz5iI6YZpEw7RdQg63LhcTWCZxUEYURYUZFC37FQmfYaFXwS\\\\\\\/baua1FuW4qEYSYIhLQg8MRt0\\\\\\\/SZfZX2U0BTCDj6qmX8YNgiaMX81yo+HUNpILWYdsHyarQAIhm2WE\\\\\\\/PnL0Amkwg1AYfcURaFJrQTZmviaZ4W6tDdb2UepanYSA2j\\\\\\\/mJ6NGT4UG21b58u27F63o8WnfU4iv7XYjdQmwzKTRbrRGuCCsiIXnJ7QdKNRj+Q3nWjGzhP1Ru+aigjojgkjSQXakB\\\\\\\/O2QTeDGAm3tYUdrvWgLPY45EQDECCjexTuWvD918RWpCnEGqKJiSGFp1e+BgUqcr8Hm3yHme4RCSFdMBlR6W\\\\\\\/dpU2qKql2qUxi8ZNxwuAux9dy78ZXlaokaNuTnkKuWXBn5rgxAqHDA8ORVFDoHdxRbkmHaGqQp\\\\\\\/U5me2NqLM0lYAN2aw5+c\\\\\\\/lyUkizdMpWh7WHujSdTVZf\\\\\\\/o6XCWE5oF\\\\\\\/Mo1nfsdu5MtAUIBFOZC+lZbPOZL1jQ\\\\\\\/foX0jcHw\\\\\\\/s5TE5DhVt+i\\\\\\\/Yt\\\\\\\/uSfPeNCqx0GpxkqaS25i2oPPVRqraAfN2CdoClUEP81MUuhaLbnIsu3U9KrJhEEnx8Vh+\\\\\\\/vue6o0lCn2ruWv13jEuC2mUeomG+bdTTph9BnGNn3qYB+gPEdDnWLTT3pxVSed96riTsrG6XUhfTnSuGCcuvMgikTjbiGEDOIRAxHMhJIuyuOQvm6qLa9vbvgHWaM\\\\\\\/tUVnx\\\\\\\/BehzDjfk5maOuR9B0FB9suE2hdCGrqJGHUs+CODHZBldDi12f3Aq68gicB4xt9qXxWJvbAZbE7hSFwbeSGWUua\\\\\\\/kZ67O5QFTU848y+sxeCrt7FSULwIkNsuaXZy51tMwY\\\\\\\/An4zIPEMaVKtwgPwK3rHGMwsWsOVXLg9sGuBEHjWrRNcmH31YR93VJdMAlamjpaDZ2a1TWsSp3R66EmbuPtCtL7R\\\\\\\/owX+U2KXZj0RqrI\\\\\\\/mOPk64f3wpmUzyL9RIvM0BpUPueRJhflT1hzZPpVLfIxXL\\\\\\\/XijQIcqQNQJDO6vIy4Z0iNThEtq1p1RsUKPCC2b4qf\\\\\\\/K7DHnueXi5l2JxvxuF0h+g7BxwJg+KubNIO8iKKqOIndjF5VZVLZ6PNMZz+frVIgE2AXRG73DOllg2VpngnMHVly8ZeK+\\\\\\\/ysQ3aVwjTXNvU+95jbX8wIb7foJocl9vOCsm+ZufcE8Y9Dokifzr6KVWJ5kdBAj4gtYGCvc1ePSpReB7duz7MFUd7KV3ML37\\\\\\\/E7E51JjNsckosQH7e2HQSNgu6OVlm8mX+4zKmAO+\\\\\\\/PL7khpoKFJ+lvQ2SyZWNrYHPNQJp2\\\\\\\/VL9Ji87rc\\\\\\\/9D0AkGFe630ibmS7H4f59DRAz46E04rpkQH0OvnAKW2MDm6KvQUYFfo\\\\\\\/ueWXCj7kTBUlY0UI6NU6srM4yemzrbiT8we3gaZ7QLCx0NY9VPrykVk3thvc3dDijKREu4nHJnQThX76aInc5wgDzewK0vFHSpUbT1wP4P7\\\\\\\/vhj\\\\\\\/PKYWDDu2BDBvjKVjPE2hs4mLufwmFSjiM4pRzk4gwZJAEsZhvGCGOsOAW+7hfPwzTvf170ybYHkymmBJCxU102XQ3HNA4lxch48a\\\\\\\/0k03vJWv8h\\\\\\\/QCg6VW0qxAtlKUrFippflKi7bZc4QFc3TQiZrBdiLNVWFyIhC5ChynAOGoKvEOQgLE8QJJ38SrJzU474eTLlH7T34jdshArDXMZOesq3xQpvzh1GdqU2vnJYxle8SeW\\\\\\\/hMfrFi6MjUJb4Kgar+lJOm4FUeRnxcf9G3CZIi9OntIl0jIH8g6f0mXmrEfYWiYdLkD69fpogJlL6Z1+cc2WU9Qv6A38sTZGxwzWZ\\\\\\\/BWzw4kjT\\\\\\\/6+W9jnb12QFhaRNYxFBK1\\\\\\\/2hK1yunkaY+ZHkzNzQ7ZLDG1tfheKOiZey4y42DqCSGbejv+WDCFcrsSjJvQiY6WhvyUROZcGfLBjdlZSNWFpCt0msLeIBimw1Y1iZpt7yxmyhQ1\\\\\\\/msaysF7UjmCC5GBTAPqP9ybA+PbCe7c8awTzZgPaM3OEtgcUCJn7+ZZ+ARcteIZJ+MM2hoPdUr5i1pS3RK9diYCKEUMS8X5m8J4vmHnVzO7\\\\\\\/id\\\\\\\/JeGal4f2RdYvofZ7o8JJy\\\\\\\/NXfdujsbInI85t0YnRqANrHNwkfMm6IqBE59jCRny8STpa0++iP4\\\\\\\/C3OmtIM4tNGdmq6pEMHo676CcsfSrxb4njNDh7biaLTjM6gBHAS2IaM8VkL5SMG2EBDviPlDz2876+G8uvwW\\\\\\\/tQHiF0BYQTVVSE\\\\\\\/+5wrHLEQEQrg+CKp8UGvPWAIG5mvGE0TGcKIJMp0SF5rOO0vHcjAy8yvXCSxVQf41Zwe7XD56yfwtmYPblXj3HBXn9GRFH9QCKjhd7OpC2COGCr9DUg\\\\\\\/DAzwQmSiE8jQuS9Dy891rGmzBXYkyLRkWIeldq3NMekpjrdAtSok5RM0E3x2iBSHc3HynoQTYU+i7zW5DUE+5+Pne6j\\\\\\\/roNNjHnSCk5fzgXiwX2ucFLqOCIgyOvLFjDRnY0xFbjyuulepqdZDDDr4icirtiQA4FJ4ISL3N1xF6Aud4UB9qNt9Xf+RTmNmgw1zMaWJMpX1+CWLTb34jZriYuTeIlW2WZePlIstHmcAip8YucJgfeSBdPvzrtekxYVaEGoZ10fG+P+SIp\\\\\\\/OIzejq06BXtHO3XK+CwDhGYLN1QJAb\\\\\\\/UjQu96yn0g+LLuRi21044dzGigFNE1BmN0PTyZhAiMXCv01Gh6y6+RzMJRXsI+MC5sQ8+zAOMX9f3Ld30PzygZ2ya6KU6YMVq6tkA7IHiyYZRhXz2V\\\\\\\/DNQCnVuhbuvWTXjnL7BU5+BezodyCaybKU8HpTMEuTWS7zU4MTsI7mITv6lBUAOL\\\\\\\/SOLGZ0Aq8LD4ns4OoD7XeMBVlea\\\\\\\/7sZUAvSvFfKDQNTLdGWCDVhBb8V4H9RSOj6aYsZIga+FRRXVF1ncimhvhX2MfLOJW\\\\\\\/XKBHNNkcuDxj1m0zd9m2c4ngTFMypt2rxYFzMH3s21lkTmFZ2CddUjcRQ\\\\\\\/w9l4XHidD1t+YG\\\\\\\/9ifX0pDYiU1vdvPhtaa91UauUIlUKcKwzqnE3fYgkFkJD1Bx9r0OHTYG\\\\\\\/zu8B4a\\\\\\\/TnV4y6mH4ev1TFnEpgZg6F8FJJ3zAb6P3Q2FBoeCNADHhq0deS1hlkKAtT8nLMl+1k5ULR4T8G9+qZyO\\\\\\\/Gh0W2Bwun0KeuT80kz2h9ojEUif0Kbr\\\\\\\/FbDILbp5nSlsw5tYqgVznJ7mcIm3BJWOl37L9QIPMzgDfwR8ta0niNd10+8RTMZeYaX90MkioneTvdvYpo+lebuweSDXSsgtH4QPFtYnYjnwnvpPOUw0zlXgPmlLXcIOtV3f9bxezWkavnDAv0iyC+kxBR+mwSFM0GJD076HXzCr8cvFXsPlZwrYp3ElVpmzTDpUg7GvgOG47qFBbGcrGNZYSZjOXNmUAX5AE4L+JbA6XebMnVic+DeqIQlKonSspleAMX16EK5NKFXSKN1lzu4q8OPc6yyaq\\\\\\\/fVJ6R0YWdHY5aK4hlqrpFZi8zwCh4NIFQtGU68ISmWiNyN\\\\\\\/+svtByFeJ8sWq6ZTJ+Oqy2VTbOUsSqPKBJOmyCqm\\\\\\\/zFYuJttJo2wlrau+gRJhlb9v573XXgM7jz7k1XwnD\\\\\\\/AWKCAZNI5lJZQMh9JJ\\\\\\\/CoTSZmFNImtjZYsX2hzc3yAvlq2Di7A+52XskphfXf\\\\\\\/j8cs45GdbjX3WEbE7CEPyk\\\\\\\/ua5DhOQc1QuGsJl4LukbpgBfGQZASKF\\\\\\\/twecghLkQMFUdIhWWzZRu4uU5pwwBUXzFBkccdhC4YQcwRDz6MONQm+JvWEN5CdfjYykq8dt4kWwjEYQoQk7bo5ILBIxun2hCvp3AUrG\\\\\\\/IpuwRoemYXP3pApaoCaoXAoodvaLgUtMiI6z\\\\\\\/nbH2ceGx27nvPJP1J8l5vabMsV+rzfOOvtia7EBNBiDdrFjW4W\\\\\\\/FcBX\\\\\\\/0QFZ7iGiH5jqxrACZFgrZbDlW0tzi\\\\\\\/rprb1Xy3iE\\\\\\\/u3Y4P2ZCVlhN1V2epHRe0o19glJyCHG26dGPjOup4SrHTnMkzWHDkSHSXbQACwsCZpenxlnsaZbl4j\\\\\\\/njRsvolyjSjmjXG8rBleIXRt0kUqYWlmEHCP+w1nmEszQfKGXV1U3vxV8V3Rs8vuyJlOnx5BFuZsLlkm9+wlS98b0Q+mHxHMA8UjqY+qwP4Gi0LmIkqq3OrEO0kIOk80yA0DruYeaWBZOBHoKeEhfRQf\\\\\\\/PZfS2W2ctqabewbEYfMk0H2X7Ytt32xx0esw71GCIA\\\\\\\/armK0tkjdYFKLVrLFOpad3kDzLvvXfWW6fwmeDtNJOt5s4sDKWrR3dNuOf0y7oGCPXqXlZi0smmi+pWfa8rDju7xlzS78DFAP1NKdlTfhpP4MKqICh9epPxQ1bqH4OXJe4V9Wv7jeRO83orc2e8z9mppu+TtMBa0IZKz8EpMdMsfGytVygo5Vz\\\\\\\/kgHfMVufcWrxv2OkXxr\\\\\\\/QiWgYB4Z3ilfSmuCfArovkC1gzTFV4G1MO+rQmWRyX7MW53AI0xfwrvhlsbych1I31Dr\\\\\\\/TiU2URNzBO+KfjRnFOpufw8nnLZlCCP4fBH8nt1l+OKKhlRKK34\\\\\\\/xYfpD3gostrAOHP1JDhzyXAZy4E7g0skb6wHcUXOVuweNKtmrzy\\\\\\\/V8yAPx9T4MFfWneLyEdkvyUpBnLjXtHw1BWQKbFiCNTckEOQNdMmO\\\\\\\/luAHiaFy0sTfkOmvasEUJC3DZHUuvsV\\\\\\\/jDiBaUUGK0K+3eZ7XJk9bipqdIjZn4BV9tmu0yS2KEfGTJv0nJFNQqoSEjvJDaBoVtM9p42kRlcRJlYzlACq\\\\\\\/zycAbYwcI1+qn3l\\\\\\\/AODkaI\\\\\\\/SqUcrDZCrANGM0RqUxoeHw1QOTe6FHebzAjwftpg3RtCOTMLbh2yceuEBkFFzBb3ag3Zk7yxfiTWF8dM3SY0bYBukFNMvpM1semBsOGkHs1yaZ42OrGJsXyIr5N0EMdSZ\\\\\\\/xu1UQq0X8xL\\\\\\\/E8GHwnaY6XP51hoNB6yJuGolP3c4uFJEKXE\\\\\\\/9vL6D\\\\\\\/ckAm9oDl4h\\\\\\\/QJn6mO87GY0RDQtS\\\\\\\/IyuKTiLi01xYpvSpMt9WGkffCLc2gVLwl8gfySsv8rcxdecSmDpwlWInzZrLUL\\\\\\\/pNa5G7SADnodzIVNQOEoeybirr0G0bh36dBiWS2Zsbtysu3bhYJMe9PY8q2odZGDQysSxFSmZ4ABsKRl+7Z6FJB3TnjdGqn7JJoiqBmTueoe3icMBhY3FHOi3bn6ymeb94qt6eOQDiVWQ6fVZtNhG\\\\\\\/RduiGqQ6cu1L8mo61ulbz2EpKUYDy86UaOqq0+SogdkKJ2chNA9AgkDPuZqBh3Y11eua47iMHuzGzSznm+X\\\\\\\/l+MydgBxggi0UFA8rnhvYRRJEVPTh8YkDp1INsWRdA4DaAvBgu2GxBzqByAx60qOxcLfmHN8WhsrzT3tWVLlC1UBv6ZByRPhcviJ46jzhE+5H6dTN57mZrZmiu4cT6dSHH4Y+EPJtkG+ulzJsI9mDLy8MwNZT\\\\\\\/TolAQAgDVo117QXQbcUvGpV7l99+6LDeOB\\\\\\\/Z+ichSOJHPeH86zGAidCA8uh6rmdh8ptCdV8h06Lfgazw\\\\\\\/VNi6dFn4d1I31xMWMxZx3OiAcXnm9AOFPTC0ko+55LgR5Re1DIhJwWZRWMH42EtAGxJHTrGIK6aEDR6XZPYfhyceHB46dEYIFsaSM5tlhcibnojJUkBjgN2D6WiaLr\\\\\\\/64s2TQwmdDrEWYXb38RSFvOGwfvJr++F3pvbuEd\\\\\\\/1WJc8OWkovLw8jyS+H\\\\\\\/NwIGddeD8QuDpGH8Rm9a+aRbL3P\\\\\\\/mfz9q2dnlX0OKQFgCt6kCs+OSdz1I63PCQSnoNbS277vS2458QzPMKedkDNfikkK6QfKBmCiR+Wk0hNkVUWdyQ6ReqHRA4PHeOu+JCp5vI84k36IlXXwX1vZkf0ELcQznP7STaLGxnHr7Y2dkn3FytnCeND57TrbIxZsQZ2aM+7Ggu9+Uizhi\\\\\\\/e\\\\\\\/nAFclqXaTy56CzXTmVpVnZfxA0u32sdIgRCm+sz\\\\\\\/1iUemo6ss9wuLB1ki02DZU49Rezt8fNBjzTq3g6TdUzglxaDf29EAgDVSHiQjyrfKDven3TvZ58eOaxezkH0sffe8t5gXb0cC2gyQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00223gWWbsbqM8aJksraz13Y5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022738ed89005ba11e7ed431707f042997db1ffb37ca83058e8db8502d6e719c44b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Zden1pvKfWeNRzaJBvsuJsaoAERyj\\\\\\\/rouCtjKDeNd85zYcLqKwkxpUg4tDZ1YeKOTAVqM8S\\\\\\\/Knwx3mmrZHoZ6gyv9V\\\\\\\/6rAkeEJxB6mKJIR7zylUtCUpXKbCJxSpgL\\\\\\\/pbJ+Bj3aktHlR\\\\\\\/ivmsRgeRdzuWnCj23mcGlIrFldnrDsdN0KkOuiEvmWjWsA4ciq39SAG5Ev00wwP\\\\\\\/FmPEI21YLB9cCi1UYTUGs87LwYGrQ0mSHT1fJLIttIBz1oEik7SV1fN57P93NwjQ2R9bODF6fLNtJY\\\\\\\/NsXdw89z6HyZGidV\\\\\\\/NUUEwdfy0TkJJDQXNeLhvEsLIPiHjrFt26hvVVG8NiutFSpyo8RI6kY\\\\\\\/0nLTnZihLMkq8\\\\\\\/nlR11lzz\\\\\\\/nWpSZJS0IE83pP6++FraA217XGYnluzLzo2aDMk2+FeY8eAgsETNzotiyeXod\\\\\\\/9lYW4WqtX7wCnuJDS6vGze\\\\\\\/ZrMa000UmjGOkLN3R\\\\\\\/I\\\\\\\/XyNvnsKVhGb6PRxns3qGku2tIGfyy3koc99OO5qS+fb5UwwmRPAqZAB\\\\\\\/LkSVD+ElkzrvazfWkW3vNpu0aHQ\\\\\\\/MCEekWmdFLtVrrQuqTz\\\\\\\/Z6wuf9bSH5QgBR2Sq1ylvytZ+yoyuVKtnFIFJEJxU8d3fLyQGTmqvYG6PIOF4uS0iBaBhWcaN2fX5stV\\\\\\\/sVd9ICckCNYAcjM9ctIkarJhCLZllylmpVVBDnClRU+giO1ikHScEy85sv6THFNqD+kV7eAX5JzueZg\\\\\\\/Y2AkDlGZay5GavRR7STs9xQWdfo+1F4mQ6rVpd1mMadK8L5lDV+xFQPtn1WZzIV3wQqsCrYgU3i96J9RulWUP1GUr\\\\\\\/\\\\\\\/9scC8AJxU2Q2F3RwDRKIKCeiEw+Fc49hB02FRAbDgefCiVmIjbzswefX5kcTfKuLkmdUQdLkjutUC4xZmzJ1nM7XnhyRzqGDbQ3rbMBv\\\\\\\/y1vxIJYn7efeU5E6DqdhrJCuSmcNKUMBByzoWMebQNNfs7bauK8JdRN5vJHEFF1FFv9YsNGaUE2jk+E\\\\\\\/eckZC1aw+fI2bX9SIpDXiYcD7z3c62AfZL32B88hZA59\\\\\\\/eGJDarDyeFIbiBRlDje+2PhfToyXmuo5NTHv0ZMMDBv1ghSCTLqdIim2qPMsO4zH+fCgKIQN2oKENwuv4MA+JHzjaSJcxdvAVZk+eu714scKoCjUmI0+IBM6QxblEh2XgyPbjqlciPj285t7cK76EO3+0EQOYtp0wgOn5f1doyFuN1sKrvyXaRnbG8NeLeyN5qksw\\\\\\\/LojD56zvWFHhntzDvzOnp0uTsHCtNww0SkwA2+0LlrEhY3BAS2+1jkDIwKiyguGO6pJ5MWw1p+X7MM0xWMit6ajcSr3nUpGlHVL1QfGvAx\\\\\\\/2gfIroS6rKi5LXU05Cumuq8HjmQhmm1FwSrdF2HiuGzsYxZrupts\\\\\\\/XvbLaN5sta61QVkWt8IP7Nv4T+jVooGpxs2ipRcd18EIrgsbZIDWL4gS6lqBSewsAUW2VwQz2ObiDoGvINeTv3hdRgN1XTeSa76Khh2khLS08QVr4RuKFefhW21H8VzXa2LtPwPltz\\\\\\\/CNoctrZ2SwmbVmFrXBd5gy54+atW4Kvs9nMT+pwjcyD6df1I8vpUj7t12tqt8\\\\\\\/3hhV2\\\\\\\/8sm8CAMniSPK8OOD9ab\\\\\\\/HfyvH0N2e3JUBHhx\\\\\\\/BhgQuiGgT0tOpIzQKKyHyuWprO0cA+n\\\\\\\/a9rnXeuyelAw4Gqq55B8tUxSX69LpmKAFZ8zzToN3zayXOmk0gRtyaU4PyoBwyuJ2v9aIMvraeUHUv21lRj+kzwXzDLOMwI176DSdjWKwB2nx5M2VFWDhJo33l99eii4Hr5M5KDNnHz+\\\\\\\/Bp1m5yppP7RLLPJM2NEjp4d1zqX0UdRAz2g9GUu+TZM9gNciGKf9pYWCgWqSBHsFgEy+8nNMxhZGsMvz2Veb+WFMMdeCgBDYwK2c0hlxndRK9\\\\\\\/kufowS5eS3pYSQTPVS6x3PLwmF6AFtksJFXG6EFa7jgFLL6U\\\\\\\/BQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00225PC9pJmbyDCZo1u6Kl7hMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227397cae3caba748be727cabc561207779f4abdc4ad0b4bcd1f86254dc977a9f8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459312 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/921" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GDXaTR5bYr7SPEMpBFVLs7E5YIt3oGLFXmDExghWp7Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--yPGKt8QPvcQn7yr8kOnRDmBAmRfMdCxalt_Cd_61Xs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/913" title="Genetic Interactions Between Arabidopsis DET1 and UVH6 During Development and Abiotic Stress Response" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/931" title="Satellite DNA-Like Elements Associated With Genes Within Euchromatin of the Beetle Tribolium castaneum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459312" id="node459312" data-pisa="ggg;2/8/921" data-pisa-master="ggg;2/8/921" data-apath="/ggg/2/8/921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Armel Salmon</span>, <span class="highwire-citation-author" data-delta="1">Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="2">Jeffrey A. Jeddeloh</span> and <span class="highwire-citation-author" data-delta="3">Jonathan Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">921-930;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459312" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Armel Salmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Armel%2BSalmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salmon%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArmel%2BSalmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Plant and Wildlife Sciences, Brigham Young University, Provo, Utah 84602, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeffrey A. Jeddeloh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Roche NimbleGen Inc., Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BA.%2BJeddeloh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jeddeloh%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BA.%2BJeddeloh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfw@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">jfw@iastate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/8/921" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459312" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/921.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459312" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/921.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459312" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/921.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459312" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/8/921.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459312" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459312/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Targeted sequence capture is a promising technology in many areas in biology. These methods enable efficient and relatively inexpensive sequencing of hundreds to thousands of genes or genomic regions from many more individuals than is practical using whole-genome sequencing approaches. Here, we demonstrate the feasibility of target enrichment using sequence capture in polyploid cotton. To capture and sequence both members of each gene pair (homeologs) of wild and domesticated <em>Gossypium hirsutum</em>, we created custom hybridization probes to target 1000 genes (500 pairs of homeologs) using information from the cotton transcriptome. Two widely divergent samples of <em>G. hirsutum</em> were hybridized to four custom NimbleGen capture arrays containing probes for targeted genes. We show that the two coresident homeologs in the allopolyploid nucleus were efficiently captured with high coverage. The capture efficiency was similar between the two accessions and independent of whether the samples were multiplexed. A significant amount of flanking, nontargeted sequence (untranslated regions and introns) was also captured and sequenced along with the targeted exons. Intraindividual heterozygosity is low in both wild and cultivated Upland cotton, as expected from the high level of inbreeding in natural <em>G. hirsutum</em> and bottlenecks accompanying domestication. In addition, levels of heterozygosity appeared asymmetrical with respect to genome (A<sub>T</sub> or D<sub>T</sub>) in cultivated cotton. The approach used here is general, scalable, and may be adapted for many different research inquiries involving polyploid plant genomes.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/gossypium" class="hw-term hw-article-keyword hw-article-keyword-gossypium" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Gossypium</em></a></li><li class="kwd"><a href="/keyword/allopolyploidy" class="hw-term hw-article-keyword hw-article-keyword-allopolyploidy" rel="nofollow">allopolyploidy</a></li><li class="kwd"><a href="/keyword/homoeologs" class="hw-term hw-article-keyword hw-article-keyword-homoeologs" rel="nofollow">homoeologs</a></li><li class="kwd"><a href="/keyword/sequence-capture" class="hw-term hw-article-keyword hw-article-keyword-sequence-capture" rel="nofollow">sequence capture</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next-generation sequencing</a></li></ul><p id="p-5">Recent and dramatic advances in sequencing technologies have been revolutionizing many areas in biology. For some types of evolutionary and population-level analyses, whole-genome sequencing is either unnecessary or unfeasible, in which case recently developed targeted sequence enrichment methods have become quite attractive. These methods enable efficient and relatively inexpensive sequencing of hundreds to thousands of genes or genomic regions from many more individuals than is reasonable using whole-genome sequencing approaches. Current target enrichment strategies include DNA capture using microarrays [<em>e.g.</em>, NimbleGen (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert <em>et al.</em> 2007</a>) or Agilent (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Porreca <em>et al.</em> 2007</a>)], DNA capture on beads (<em>e.g.</em>, Agilent SureSelect, NimbleGen SeqCap EZ, Mycroarray Inc), and amplicon-based methods (<em>e.g.</em>, Fluidigm AccessArray and Molecular Inversion Probes). The advantages and disadvantages of these methodologies have recently been reviewed (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cronn <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grover <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mamanova <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mertes <em>et al.</em> 2011</a>), although applications in plants remain relatively few in number (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fu <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Haun <em>et al.</em> 2011</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Saintenac <em>et al.</em> 2011</a>). Because plant genomes are rife with duplications large and small, it is important to develop targeted sequencing approaches that facilitate the assignment of multiple homologous sequences into the appropriate categories of homeology, orthology, and paralogy. In addition, it would be useful to design experiments that enable the capture of flanking sequence and introns based on more limited information such as that obtained from expressed sequence tag (EST) libraries or assemblies, especially for lineages in which genomic resources (<em>i.e.</em>, reference sequences) are lacking. In this brief report, we demonstrate these two dimensions of targeted sequence capture for the allopolyploid <em>Gossypium hirsutum</em>, which is the most economically important of the four domesticated cotton species.</p><p id="p-6">Allopolyploid cottons comprise a clade of five species containing two coresident genomes, designated A<sub>T</sub> and D<sub>T</sub>, where the âTâ subscript indicates the diploid progenitor-of-origin, <em>i.e.</em>, A or D. Diploid A-genome (<em>Gossypium arboreum</em>) and D-genome (<em>Gossypium raimondii</em>) species are closely related to their counterparts in the allopolyploid nucleus, as demonstrated by both DNA sequencing and comparative cytogenetic techniques (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Endrizzi <em>et al.</em> 1985</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brubaker and Wendel 1994</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Senchina <em>et al.</em> 2003</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Flagel <em>et al.</em> 2012</a>). Because the average genic exons differ by less than 1% in sequence in A<sub>T</sub> and D<sub>T</sub> homeologous genes, we hypothesized that it would be possible to simultaneously target the sequence of both genomes by sequence capture. With successful simultaneous captures, we could assess: (1) whether sequence capture probes designed using EST contigs from an unsequenced genome would sufficiently enrich target loci; (2) evaluate whether unique identifiers could be multiplexed to âbarcodeâ 454 sequencing reads; (3) determine whether multiple samples of a polyploid could be effectively pooled before hybridization thereby allowing a single sequencing reaction; and (4) assess whether capture and sequencing efficiencies differed for two diverse accessions (the wild <em>G. hirsutum</em> accession TX2094, and the cultivar Maxxa Acala) of the same polyploid species.</p><p id="p-7">Here, we demonstrate the feasibility of target enrichment using sequence capture in polyploid cotton. We created custom hybridization probes to target 500 pairs (homeologs) of selected genes based on information from the cotton transcriptome. Collectively, this set of loci represents approximately 550 kb of haploid transcript space. Two widely divergent samples of <em>G. hirsutum</em> were hybridized to four capture arrays containing capture probes. We show that the two coresident homeologs in the allopolyploid nucleus were equally captured with high coverage, that capture efficiency was similar between the two accessions and independent of whether the samples were mixed or jointly hybridized, and that a significant amount of flanking, nontargeted sequence (untranslated regions [UTRs] and introns) also was captured and sequenced along with the targeted exons. These results offer a general approach that could be adapted for all complex plant genomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genomic MID-tagged DNA libraries</h3><p id="p-8">Two DNA samples were each extracted from young leaves of two accessions (TX2094 and Maxxa Acala, afterward simply referred to as Maxxa). The Roche GS FLX Titanium DNA Library Preparation kit was used to prepare four whole-genome shotgun fragment libraries for 454 sequencing. Samples were nebulized, and the fragments were end-repaired. Multiplex Identifier 1 (MID1) and MID2 were separately ligated (along with sequencing adapters) to the fragments of one library of Maxxa and one library of TX2094, respectively. The remaining two libraries (Maxxa and TX2094) did not have MID-encoded adapters attached to the fragments. The ligated libraries were then size-selected using the gel-cut technique (fragment size between 500 and 650 bp), and the libraries were then amplified (11 cycles) using the Ti-A and Ti-B primers (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) to obtain sufficient DNA quantity for hybridization (100 ng). The quality and the concentration of the prepared libraries were checked on a Bioanalyzer (Agilent Bioanalyser DNA 7500 Pico Chip) and a Nanodrop.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461794/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461794/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Roche NimbleGen MID-adaptors, hybridization-enhancing oligomer sequences, and qPCR primers used for the postcapture enrichment estimation of the DNA libraries</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>NimbleGen capture array design</h3><p id="p-11">Five hundred pairs of genes were targeted for enrichment based on previous studies of gene expression and from published literature of cotton. The results from gene expression experiments of developing cotton fiber and salt-stress were cross-tabulated so that both highly expressed and differentially expressed genes could be selected (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chaudhary <em>et al.</em> 2008</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hovav <em>et al.</em> 2008</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rapp <em>et al.</em> 2010</a>; J. A. Udall, personal communication). Additional genes were chosen based on studies of cotton fiber development and of trichome development in <em>Arabidopsis</em>. We also included members of several transcription factor gene families (MYB, TCP, and WRKY, based on Blast annotations). The list of targeted genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1/TableS1.xls">Table S1</a>) is available at <a href="http://cottonevolution.info">http://cottonevolution.info</a>. The length of the targets designed from EST assemblies ranged from 335 to 3377 bp with a mean length of 1085 bp (SD 479.8) and a total length of approximately 550 kb. Hybridization probe-sets for the selected targets were designed from consensus sequences calculated from A-genome, D-genome, and AD-genome ESTs sequenced from various cDNA libraries described in <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Udall <em>et al.</em> (2006)</a> and by Roche NimbleGen using proprietary software algorithms with generalized parameters for probe sequence, hybridization temperature, and length (considering that the cotton genome sequence was not available). The probes designed for each of the 500 pairs of targets overlapped 465,034 bp, or 85.6% of the targeted sequences, and were synthesized on HD2 (HX12) arrays, each slide consisted of 12 subidentical subarrays each with 135,000 probes designed against the 550-Kbp transcript space. The individual subarrays were not used <em>per se</em> in this experiment; rather, they provided a perspective on enrichment efficiency with this platform in mind for future experiments. While this paper was being written, a draft genome sequence of <em>G. raimondii</em> (D<sub>5</sub>) was released (DOE Joint Genome Institute: Cotton D V1.0, available at <a href="http://www.phytozome.net/cotton.php">http://www.phytozome.net/cotton.php</a>). We inferred the copy number of the targeted genes by searching for the sequences used for probe design within the genome sequence (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1/TableS2.xls">Table S2</a>). A total of 411 of the 500 targeted genes appeared to be single-copy.</p></div><div id="sec-4" class="subsection"><h3>Microarray hybridization and elution of captured libraries</h3><p id="p-12">A total of four HX12 slides (NimbleGen) were used, each consisting of 12 subarrays containing 135,000 probes designed from the 500 targeted genes. The Maxxa and TX2094 libraries (without MIDs) were separately hybridized to one slide each (<em>i.e.</em>, each of the 12 wells received the same library; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). A second pair of arrays was subjected to precapture library multiplexing and different mixture ratios of the Maxxa and TX2094 libraries to estimate the relative enrichment and capture efficiency of multiplexed MID-tagged libraries. One slide was loaded with a 50:50 ratio of the two MID-labeled libraries (of the subarrays, six wells loaded with one library and six wells loaded with the other) and the other with a 75:25 mix (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/921/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overall workflow for targeted sequencing of 500 genes in cotton." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2031335574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overall workflow for targeted sequencing of 500 genes in cotton.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/921/F1.medium.gif" width="440" height="384"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/921/F1.medium.gif" width="440" height="384"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/921/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/921/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461713" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-13" class="first-child">Overall workflow for targeted sequencing of 500 genes in cotton.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">The samples were prepared as following for microarray hybridization: 100 Î¼g of DNA library, 20 ÂµL of Plant Capture Enhancer solution (Roche NimbleGen), and 20 nmoles of Hybridization Enhancing A and B oligonucleotides were dried at 60Â° in a SpeedVac. Samples were resuspended in 4.8 ÂµL of water and heated at 70Â° to fully solubilize the libraries. To each sample, 8ÂµL of 2X SC Hybridization Buffer (Roche NimbleGen) and 3.2 ÂµL of SC Hybridization Component A (Roche NimbleGen) were added. DNAs were denatured at 95Â° for 10 min and placed at 42Â° until ready for hybridization. The slides were assembled according to NimbleGen instructions. Six microliters of each sample was loaded for each subarray and the hybridization performed on a NimbleGen Hybridization System 4 (mix mode B) at 42Â° for 72 hr. After hybridization, the slides were washed and eluted as previously described (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fu <em>et al.</em> 2010</a>). The eluted captured libraries were then titer-amplified using the Ti-A and Ti-B primers (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Titer-amplification sampled a single, preliminary 36 ÂµL of polymerase chain reaction (PCR) for each elute for 8, 10, 12, 14, and 16 cycles. The library amplification product was visualized on a Lonza gel, and the number of cycles for each library was individually determined. Once the number of cycles was identified, 16 PCRs (50 ÂµL) were completed using high-fidelity <em>Taq</em> DNA polymerase (Clontech), and the products were cleaned using a QiaQuick column (QIAGEN).</p></div><div id="sec-5" class="subsection"><h3>Enrichment estimation by quantitative PCR and 454 sequencing</h3><p id="p-15">To evaluate the enrichment of the captured libraries before sequencing, a quantitative PCR (qPCR) assay was performed for four targeted and one nontargeted genes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Three replicates were used for each gene, for each accession, using both the precapture and postcapture libraries. The qPCRs contained 5 ng of pre- or postcapture genomic libraries, 1X of SYBR Green Master I (Roche), 0.5 ÂµM of each primer in a final volume of 10 ÂµL. The qPCR assays were amplified on the LightCycler 480 II (Roche) for 40 cycles followed by a melting curve analysis to quantify amount of target enrichment by sequence capture. PCR efficiencies (E) were measured for each locus, and the fold-enrichment was calculated using the difference between the mean crossing point (CP) values reported between the precapture and the postcapture matrices (â<sub>CP</sub>), with: fold enrichment = E<sup>âCP</sup>. The quality and the concentration of the enriched libraries were checked on a Bioanalyser DNA7500 chip (Agilent) and on a Nanodrop, and the libraries were diluted for emulsion PCR as recommended by 454 Life Science and sequenced using a single 4-region Titanium PicoTilterPlate (PTP; 1 region per capture array) on a GS-FLX Titanium 454 Sequencing System (DNA Sequence Center, Brigham Young University, Provo, UT).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461770/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461770/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Postcapture enrichment estimations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Sequence assembly</h3><p id="p-18">A <em>de novo</em> assembly of the captured reads was performed to discover the intronic and untranslated regions of the captured genomic regions hybridized to exonic probes (GS <em>De novo</em> Assembler Software v.2.5.3; Roche). The resulting <em>de novo</em> assembled contigs were assigned to the target exonic sequences using BLASTN (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>). We assumed that the best blast hit (at a minimum of 95% sequence similarity) would be the correct match between the <em>de novo</em> assembled contig and the reference gene. Spidey (<a href="http://www.ncbi.nlm.nih.gov/IEB/Research/Ostell/Spidey/index.html">http://www.ncbi.nlm.nih.gov/IEB/Research/Ostell/Spidey/index.html</a>) was used to align genomic (<em>de novo</em> contigs) to exonic sequence (reference exonic sequence). The alignments between the best match <em>de novo</em> contigs and the reference transcripts led to the detection of large gaps corresponding to the genomic untranslated regions (<em>i.e.</em>, UTRs and introns) and flanking exonic regions. Intron/exon boundaries were verified using the ghmme3 program of genemark [<em>Arabidopsis</em> splice junctions were used as a reference (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Besemer and Borodovsky 2005</a>)].</p><p id="p-19">The full set of 454 reads (both MID-labeled and nonlabeled reads from both accessions) were mapped to two reference sequences of the 500 targeted pairs of homeologs: (1) the exonic EST templates used for capture-probe design; and (2) a genomic reference rebuilt from the <em>de novo</em> assembled contigs. Different minimum overlap identity rates (90%, 95%, and 97%) were tested, which also facilitated measuring on- and off-target rates for each library and each experiment. The results presented here were obtained with a 97% mapping stringency to avoid the comparison of putative close paralogs.</p></div><div id="sec-7" class="subsection"><h3>Single-nucleotide polymorphism detection</h3><p id="p-20">Alignments to the reference sequences were scanned (using the Ace.py program from biopython, <a href="http://biopython.org/">http://biopython.org/</a> and custom python scripts) for two types of single-nucleotide polymorphism (SNPs; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>): (1) genome-specific SNPs that differentiated the A<sub>T</sub>- and D<sub>T</sub>-genomes at homeologous positions (hereafter, homoeoSNPs); and (2) Allelic-SNPs that represented single-base differences within the homeologous copies detected (hereafter alleleSNPs). HomoeoSNP positions were determined by comparing the bases of simultaneously mapped EST orthologs from the A<sub>2</sub> and D<sub>5</sub> diploid species to the sequence references (available at <a href="http://cottonevolution.info">http://cottonevolution.info</a>). We assigned the tetraploid captured reads to either the A<sub>T</sub>- (A-genome within the tetraploid) or D<sub>T</sub>-genome by individually comparing their bases at homoeoSNP positions to the diploid bases at that position. The minority homoeoSNP was required to be present in a minimum of three reads for each tetraploid genome and in 100% of the reads of the orthologous diploid relatives. Two types of alleleSNPs were detected (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). First, alleleSNPs that distinguish between Maxxa and TX2094 were identified in each of the two coresident genomes. Second, SNPs were detected within Maxxa or TX2094 (<em>i.e.</em>, variable within Maxxa A<sub>T</sub>-genome, variable within the Maxxa D<sub>T</sub>-genome, variable within the TX2094 A<sub>T</sub>-genome, and variable within the TX2904 D<sub>T</sub>-genome). These SNPs within A<sub>T</sub>-genomes or D<sub>T</sub>-genomes indicate where the accessions were heterozygous. In each case, the minor SNP allele was required to have &gt;5x coverage and be present in 30% of the reads. Sequencing errors detected as rare variants or within homopolymeric regions (repeats of more than 4 times the same nucleotide presenting putative false-positive SNPs) were removed from our analyses.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Fold enrichment of targeted loci</h3><p id="p-21">Successful targeted capture and sequencing results in a library of sequences that is enriched for the loci of interest. The extent to which the target region(s) are enriched is a direct measure of the efficacy of the sequence capture procedure. Typically this measurement is reflected as the proportion of total reads from the target region divided by the fraction of the genome the targeted region comprises. Because this calculation requires sequencing the samples, often an assay such as quantitative PCR is used as a first-pass surrogate measure of experimental success. In our case, the fold-enrichment estimation of the capture experiment assessed by qPCR for four targeted genes (Cotton16_00329_02, GhMYB109, GhCAPRICE, and Cotton16_00219_02) varied from 24- to 317-fold, depending on the gene tested and the capture assay (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The qPCR curves depicted (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) illustrate the effect of the capture on the enrichment of the targeted genes and the expected depletion of a nontargeted gene (<em>ndhF</em>) that is estimated up to 100Ã (enrichment factor of 0.01; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Complicating our analysis was the fact that the source of our capture template was exonic, meaning that the intronic space was unaccounted for our fold enrichment estimates. Because all four positive controls (enrichment loci), and the negative control (depletion locus) exhibited the proper enrichment/depletion relationships, we sequenced the libraries using 454 sequencing.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/921/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Amplification curves pre- (dashed lines) and postcaptures (solid lines) from the captures of the two accessions tested (Maxxa, in gray; TX2094 in black) for a targeted gene, GhMYB109 (A) and a nontargeted gene, ndhF (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2031335574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Amplification curves pre- (dashed lines) and postcaptures (solid lines) from the captures of the two accessions tested (Maxxa, in gray; TX2094 in black) for a targeted gene, &lt;em&gt;GhMYB109&lt;/em&gt; (A) and a nontargeted gene, &lt;em&gt;ndhF&lt;/em&gt; (B).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/921/F2.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/921/F2.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/921/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/921/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461643" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-22" class="first-child">Amplification curves pre- (dashed lines) and postcaptures (solid lines) from the captures of the two accessions tested (Maxxa, in gray; TX2094 in black) for a targeted gene, <em>GhMYB109</em> (A) and a nontargeted gene, <em>ndhF</em> (B).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Enrichment of 454 sequenced captured libraries and MID tag recovery</h3><p id="p-23">The 454-sequencing run of the captured libraries resulted in 1,017,064 reads (available at <a href="http://cottonevolution.info">http://cottonevolution.info</a><span class="underline">)</span>. MID-tagged libraries from the two multiplexing assays (in two different 1/4 PTPs) were recovered from the total sequenced reads at approximately the expected proportion: for the 50:50 MID-tagged library multiplexing assay, the MID-tagged reads represent 42.2% and 57.8% of the sequenced reads for the MID1- and MID2-tagged libraries respectively, whereas for the 75:25 MID-tagged library multiplexing assay, the MID-tagged reads represent 75.7% and 24.3% of the sequenced reads for the MID1 and MID2-tagged libraries, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Considering pipetting and sampling error, these small deviations from the expected ratios were considered acceptable.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461682/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461682/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of reads captured and mapped at a minimum overlap identity of 90% on the targeted probes</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">Direct mapping of the captured libraries resulted in 403,654, or 39.7%, of the total reads mapping to the targeted sequences, the proportion ranging from 16.4% to 60.8% depending on the captured library. Little capture efficiency difference was observed for the two accessions tested (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Greater than 98% of the targeted genes exhibited a differential capture which differed by less than Â±20% between Maxxa and TX2094 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), illustrating the efficiency of simultaneous capture of different accessions (with or without MID-tags). We examined the distribution of mean coverage of the captured reads mapping to the 500 targeted gene pairs: more than 400 gene pairs (407 and 404 for Maxxa and TX2094, respectively) are represented by a mean read depth of more than 20X, which was our goal for identifying homeologous copies and alleles in allopolyploid cottons (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). One targeted gene pair (Cotton16_41254_01, coding a putative tRNA) was highly overrepresented, with 152,398 reads. Inadvertent inclusion of this highly repeated sequence (its copy number in the unpublished, draft D-genome is 136) in the capture design significantly lowered the resulting sequence coverage for the other targeted genes. Because of its relatively high genomic copy-number, this sequence was removed from subsequent analyses of single-copy genes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/921/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Average distribution of the differential capture efficiencies between Maxxa (untagged and MID1 libraries) and TX2094 (untagged and MID2 libraries) normalized per quarter capture array. The number of genes and proportion of genes are on the y-axes, respectively. (B) Distribution of the mean coverage for Maxxa (in light gray) and TX-2094 (in dark gray) of the captured reads mapped to the 500 targeted gene pairs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2031335574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Average distribution of the differential capture efficiencies between Maxxa (untagged and MID1 libraries) and TX2094 (untagged and MID2 libraries) normalized per quarter capture array. The number of genes and proportion of genes are on the y-axes, respectively. (B) Distribution of the mean coverage for Maxxa (in light gray) and TX-2094 (in dark gray) of the captured reads mapped to the 500 targeted gene pairs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/921/F3.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/921/F3.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/921/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/921/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461809" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-25" class="first-child">(A) Average distribution of the differential capture efficiencies between Maxxa (untagged and MID1 libraries) and TX2094 (untagged and MID2 libraries) normalized per quarter capture array. The number of genes and proportion of genes are on the y-axes, respectively. (B) Distribution of the mean coverage for Maxxa (in light gray) and TX-2094 (in dark gray) of the captured reads mapped to the 500 targeted gene pairs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Recovery of adjacent and intervening regions not targeted</h3><p id="p-26">When the <em>de novo</em> read assemblies were used, at least one nontargeted genomic region (flanking or intronic) was recovered for 463 of the targeted gene pairs. These flanking and intronic sequences facilitated reconstruction of genomic sequence from the <em>de novo</em> assembled captured reads (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). When this genomic reconstruction was used as a reference for read mapping, it increased the number of mapped bases compared with the âexonic referenceâ mapping by the inclusion of previously trimmed bases (not additional sequencing reads). All possible classes of nontargeted, contiguous sequence, <em>i.e.</em>, 5â²-UTR, intronic, flanking exonic, and 3â²-UTR, were recovered for 131 gene pairs, whereas for the other 332 targets at least one of these regions was recovered (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). A total of 67,435 bp of 5â²-UTR, 21,083 bp of flanking exon, 78,861 bp of intronic, and 52,722 bp of 3â²-UTR was assembled, allowing the reconstruction of a genomic reference sequence of 791,241 bp for the targeted genes captured from the EST-derived probes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). In all cases, the gene model inferred was validated by comparison to <em>de novo</em> gene model prediction using the genemark software (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1/TableS3.xls">Table S3</a>). The distribution of the mean sequence coverage for both translated and untranslated regions was similar between Maxxa and TX2094 after mapping the captured reads to this reconstructed genomic reference sequence (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). The untranslated region coverage is &gt;20Ã for 212 and 203 targeted genes for Maxxa and TX2094, respectively.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/921/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic reference sequence reconstruction and its effects on mapping results for the Cotton16_31308_01 EST contig. (A&#x2212;C) left: read depth per position for Maxxa (light gray) and TX2094 (dark gray); right: illustration of the mapped reads. (A) Reads mapped to the target sequences. (B) Reads mapped to different exonic parts of a gene when visualized against the reconstructed genomic reference. (C) Flanking genomic sequence detected using de novo assemblies mapped to the probe sequences and reads mapped to the genic space could be used to reconstruct the genomic sequence. (D) 1, number of mapped reads based on the targeted sequences; 2, number of mapped reads based on the recovered genomic reference; 3, number of mapped bases (divided by 100) based on the targeted sequences; and 4, number of mapped bases (divided by 100) based on the recovered genomic reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2031335574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic reference sequence reconstruction and its effects on mapping results for the Cotton16_31308_01 EST contig. (A&#x2212;C) left: read depth per position for Maxxa (light gray) and TX2094 (dark gray); right: illustration of the mapped reads. (A) Reads mapped to the target sequences. (B) Reads mapped to different exonic parts of a gene when visualized against the reconstructed genomic reference. (C) Flanking genomic sequence detected using &lt;em&gt;de novo&lt;/em&gt; assemblies mapped to the probe sequences and reads mapped to the genic space could be used to reconstruct the genomic sequence. (D) 1, number of mapped reads based on the targeted sequences; 2, number of mapped reads based on the recovered genomic reference; 3, number of mapped bases (divided by 100) based on the targeted sequences; and 4, number of mapped bases (divided by 100) based on the recovered genomic reference.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/921/F4.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/921/F4.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/921/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/921/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461661" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-27" class="first-child">Genomic reference sequence reconstruction and its effects on mapping results for the Cotton16_31308_01 EST contig. (AâC) left: read depth per position for Maxxa (light gray) and TX2094 (dark gray); right: illustration of the mapped reads. (A) Reads mapped to the target sequences. (B) Reads mapped to different exonic parts of a gene when visualized against the reconstructed genomic reference. (C) Flanking genomic sequence detected using <em>de novo</em> assemblies mapped to the probe sequences and reads mapped to the genic space could be used to reconstruct the genomic sequence. (D) 1, number of mapped reads based on the targeted sequences; 2, number of mapped reads based on the recovered genomic reference; 3, number of mapped bases (divided by 100) based on the targeted sequences; and 4, number of mapped bases (divided by 100) based on the recovered genomic reference.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/921/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Length of sequenced regions. Relative lengths of 5&#x2032; UTR, 3&#x2032; UTR, and intronic are shown in base pairs. The target (genomic) is the sum length of the reconstructed genomic sequence including the captured sequence flanking the direct targets. (B) Distribution of the number of genes with recovered genomic regions. *Recovered untranslated genomic regions in unclosed introns or for genes displaying multiple gene models (C) Distribution of the mean read depth for Maxxa and TX-2094 of the captured reads mapped to the 500 targeted gene pairs. Bars, from darkest to lightest: mean read depth for Maxxa exons, Maxxa introns, TX2094 exons, and TX2094 introns." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2031335574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Length of sequenced regions. Relative lengths of 5&#x2032; UTR, 3&#x2032; UTR, and intronic are shown in base pairs. The target (genomic) is the sum length of the reconstructed genomic sequence including the captured sequence flanking the direct targets. (B) Distribution of the number of genes with recovered genomic regions. &lt;sup&gt;*&lt;/sup&gt;Recovered untranslated genomic regions in unclosed introns or for genes displaying multiple gene models (C) Distribution of the mean read depth for Maxxa and TX-2094 of the captured reads mapped to the 500 targeted gene pairs. Bars, from darkest to lightest: mean read depth for Maxxa exons, Maxxa introns, TX2094 exons, and TX2094 introns.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/921/F5.medium.gif" width="266" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/921/F5.medium.gif" width="266" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/921/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/921/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461726" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5â</span> <p id="p-28" class="first-child">(A) Length of sequenced regions. Relative lengths of 5â² UTR, 3â² UTR, and intronic are shown in base pairs. The target (genomic) is the sum length of the reconstructed genomic sequence including the captured sequence flanking the direct targets. (B) Distribution of the number of genes with recovered genomic regions. <sup>*</sup>Recovered untranslated genomic regions in unclosed introns or for genes displaying multiple gene models (C) Distribution of the mean read depth for Maxxa and TX-2094 of the captured reads mapped to the 500 targeted gene pairs. Bars, from darkest to lightest: mean read depth for Maxxa exons, Maxxa introns, TX2094 exons, and TX2094 introns.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>SNP discovery and homeolog and allele assignment for single-copy genes</h3><p id="p-29">Many SNPs were detected in the alignments of mapped captured reads to the 411 translated regions and to the 411 reconstructed genomic reference sequences. A total of 1795 and 2166 were diagnostic of a homoeoSNP for the EST-based and the genomic reference enriched in flanking-exon sequences, respectively. These homoeoSNPs were used to assign reads from the allopolyploid <em>G. hirsutum</em> to genome-of-origin. This strategy enabled the detection of 347 A<sub>T</sub> and D<sub>T</sub> homeologs in Maxxa and 342 A<sub>T</sub> and D<sub>T</sub> homeologs for TX2094 (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). This result allowed the calculation of the polymorphism rate between the A<sub>T</sub>- and D<sub>T</sub>-genomes within the two tested accessions (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a>). Thus, the sequence capture strategy was highly effective in recovering homeologs with equal and high efficiency from both genomes (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>) and from both accessions.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461699/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461699/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Number of targeted gene pairs for which both the A<sub>T</sub> and D<sub>T</sub> homeologs were detected (and within subgenomes, number of heterozygous loci), based on a <em>de novo</em> genomic reference sequence (Genomic) and the EST-based exonic reference (Exonic) sequence (97% of minimum overlap identity mapping)</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/921/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Relative capture efficiencies of both homeologs (A-genome in gray; D-genome in black) in Maxxa (top) and TX2094 (bottom) for 342 single-copy genes from alignments to the &#x2018;genomic&#x2019; reference (97% minimum overlap identity mapping); (B) Distribution of nucleotide polymorphism between Tx2094 and Maxxa (AT-genome in gray; DT-genome in black) for captured homeologs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2031335574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Relative capture efficiencies of both homeologs (A-genome in gray; D-genome in black) in Maxxa (top) and TX2094 (bottom) for 342 single-copy genes from alignments to the &#x2018;genomic&#x2019; reference (97% minimum overlap identity mapping); (B) Distribution of nucleotide polymorphism between Tx2094 and Maxxa (A&lt;sub&gt;T&lt;/sub&gt;-genome in gray; D&lt;sub&gt;T&lt;/sub&gt;-genome in black) for captured homeologs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/921/F6.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/921/F6.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/921/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/921/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461671" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6â</span> <p id="p-31" class="first-child">(A) Relative capture efficiencies of both homeologs (A-genome in gray; D-genome in black) in Maxxa (top) and TX2094 (bottom) for 342 single-copy genes from alignments to the âgenomicâ reference (97% minimum overlap identity mapping); (B) Distribution of nucleotide polymorphism between Tx2094 and Maxxa (A<sub>T</sub>-genome in gray; D<sub>T</sub>-genome in black) for captured homeologs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Sequence capture was successful in detecting the two types of alleleSNPs. First, allele-SNPs between Maxxa-TX2094 (where each accession was homozygous) were detected (26 in TX2094 and 5 in Maxxa; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1/TableS4.xls">Table S4</a>). Second, alleleSNPs were detected within accessions where multiple bases were present within the sequences representing a single homeolog (62 in TX2094 and 21 in Maxxa; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1/TableS4.xls">Table S4</a>). This second type of alleleSNP indicated base positions within Maxxa and TX2094 that were heterozygous. As expected, a greater level of heterozygosity was detected in TX2094 than in Maxxa (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>; Fisher's exact tests: A<sub>T</sub>-genomes, <em>P</em> &lt; 0.001). Interestingly, D<sub>T</sub>-genome loci were more heterozygous than their A<sub>T</sub>-genome homeologs in the cultivar Maxxa (Fisher's exact tests: <em>P</em> &lt; 0.001). The number of reads that mapped to the reconstructed references (exonic/genomic) and the numbers of reads assigned to a given copy, and the mean coverage are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1/TableS4.xls">Table S4</a>.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-33">One of the challenges for use of the information embedded in complex plant genomes is the ability to detect and identify orthologous duplicated sequences, or homeologs. This is particularly problematic in polyploids of relatively recent origin, in that homeologous copies bear high sequence identity. Historically, methods for homeolog isolation and diagnosis have relied upon conventional approaches that entail PCR, cloning, and Sanger sequencing of selected clones. The labor-intensive nature of these methods places practical limitations on the number of homeologous pairs that can reasonably be included in many applications, often resulting in experiments with scales ranging from a few genes to tens of genes. Since the advent of next-generation sequencing methods, several new and methodologically powerful target enrichment strategies have been developed, such as RAD sequencing (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Davey and Blaxter 2010</a>), pooled amplicon sequencing (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Griffin <em>et al.</em> 2011</a>), and sequence capture (reviewed in <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cronn <em>et al.</em> 2012</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Grover <em>et al.</em> 2012</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Mamanova <em>et al.</em> 2010</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Mertes <em>et al.</em> 2011</a>). These new methods provide a powerful means for catapulting many types of studies from the population genetics into population genomics arena. The increasing application of these approaches will likely transform diverse fields, including evolutionary and ecological studies, to which genomic or even epigenomics approaches were not, until recently, feasible (reviewed in <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cronn <em>et al.</em> 2012</a> and <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Grover <em>et al.</em> 2012</a>).</p><p id="p-34">Although exome resequencing in primates has been extensively developed (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bainbridge <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cosart <em>et al.</em> 2011</a>;<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">George <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hodges <em>et al.</em> 2007</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">2009</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ng <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">T1GP Consortium 2011</a>), only a few studies have been reported to date in plants. In maize, sequence capture approaches by hybridization allowed the capture of 2.3 Mb and 43 genes between two maize lines (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Fu <em>et al.</em> 2010</a>). More recently, target enrichment has been used in soybean for exome resequencing of two accessions for SNPs discovery (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Haun <em>et al.</em> 2011</a>), and in allotetraploid wheat for exome resequencing of 3,497 genes (representing 3.5 Mb) for two accessions (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Saintenac <em>et al.</em> 2011</a>).</p><p id="p-35">Here we report the ability to simultaneously capture hundreds of loci for multiplexed samples. This hybridization-based approach, which reduces genomic sample complexity, allowed the enrichment in targeted loci by more than 300-fold. This target enrichment results in an increased read-depth of the targeted loci, a critical consideration for validating the existence of <em>bona fide</em> SNPs, for confident detection of appropriate assignment of alleles for heterozygous loci, and for diagnosis of the genomic origin of homeologous genes. We tested the use of barcoded (MID-tagged) 454 libraries in different mixture proportions (50:50 and 75:25) and obtained sequenced reads from these libraries at the expected proportions, demonstrating the feasibility of using multiplexed sequence capture libraries for larger-scale experiments. We show that both homeologs from the two accessions studies were captured with equivalent efficiency, even though the hybridization probes were designed from EST assemblies from parental and various <em>G. hirsutum</em> cultigens (including the Acala Maxxa accession) EST libraries (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Udall <em>et al.</em> 2006</a>) and from a single consensus sequence of the targeted homeologous loci.</p><p id="p-36">Although the purpose of the present study was largely methodological, we draw attention to one surprising and perhaps notable biological conclusion. Specifically, we note that heterozygosity levels are greater in the wild accession (TX2094) than in the cultigen (Maxxa), as expected given the many generations of inbreeding (generalized and actual) during cotton cultivar development (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). In addition, though, the drop in heterozygosity in Maxxa is strikingly biased with respect to genome, such that for the A<sub>T</sub> genome only a single heterozygous locus was detected. The difference in heterozygosity between the two genomes of Maxxa is dramatic, especially given the observation that A<sub>T</sub> and D<sub>T</sub> heterozygosity levels are equivalent in TX2094. Further studies are needed to explain this intriguing suggestion of differential selection accompanying domestication in the two genomes of modern Upland cotton.</p><p id="p-37">The foregoing results have broad applicability, requiring only the information present in EST sequences from the species (or clade) of interest, although as yet there is no information that addresses the rate of drop-off of target locus recovery with increasing taxonomic or genetic divergence. One key factor in target design is that probes should avoid repeated sequences, so as not to waste sequencing space on redundant and often extraneous sequences. In many cases, even for nonmodel species, comparative genomics approaches using the phylogenetically closest available genome(s) should allow copy number estimations of each target. As shown here, when steps are taken to focus on low-copy sequences, the recovery rate for targeted genes can be very high.</p><p id="p-38">One advantage of the methodology adopted here is that even though the probes were designed against expressed genes (and hence exons), captured regions can be bioinformatically extended into 5â² and 3â² flanking regions as well as introns. These noncoding regions typically are more rapidly evolving than exonic nucleotide sites and thus are of particular interest for polymorphism detection and diversity analyses. One can imagine a several-step sequence capture process to resequence coding and noncoding loci for species for which reference genomes are not available, with a first capture using targeting probes designed from ESTs, subsequent sequencing of captured libraries allowing the detection of untranslated sequences, and a second step using redesigned probes that both exclude the capture of undesirable repetitive sequences but which also include the previously recovered non-coding sequences. This second capture design would then be useful for larger-scale applications using an expanded set of samples. The net consequence of such an approach would be execution of massively parallel chromosome walking via sequence capture.</p><p id="p-39">Because of these many advantages, targeted sequence capture approaches provide a powerful new tool for plant breeding, analyses of the scale and scope of genomic diversity, and a host of evolutionary and ecological genomic studies. By making diversity assessment accessible for genes throughout the genome, it becomes more feasible, for example, to detect the targets of natural and artificial selection. This technology, therefore, should find widespread application in analyses of crop domestication (reviewed in <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doebley <em>et al.</em> 2006</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gross and Olsen 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wright and Gaut 2005</a>). With respect to cotton, in particular, domestication has been historically replicated, in parallel, for two diploid, A-genome species in the Old World (<em>G. herbaceum</em> and <em>G. arboreum</em>) and for two tetraploid cottons (AADD genomes) in the New World (<em>G. hirsutum</em> and <em>G. barbadense</em>). Cotton epidermal seed trichomes (commonly known as âfibersâ) in domesticated cottons and their wild relatives differ by various morphological and structural characters, including density, length, cell growth synchronicity, shapes, and developmental shifts during the âfiberâ elongation phase (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Applequist <em>et al.</em> 2001</a>). In polyploid cotton, domestication has been associated with selection for long, strong fibers, synchrony in fiber development, higher fiber cover, and more fiber initials on the day of anthesis (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Butterworth <em>et al.</em> 2009</a>). These transformations are concomitant with a massive repatterning of gene expression (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Rapp <em>et al.</em> 2010</a>) when comparing âwildâ (<em>e.g.</em>, TX2094) to improved (<em>e.g.</em>, Acala Maxxa) accessions. In the present study, we chose a subset of genes implicated as either affected by or involved in the domestication process. By studying a larger set of accessions spanning the wild to advanced continuum, it should be possible to use population genomic strategies to detect the genomic regions, and possibly the genes, that were unknowingly targeted by humans during the domestication process.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We thank Corrinne Grover and Kara Grupp for advice and technical assistance and people from 454 LifeSciences (T. Jarvie, T. Arnold, and J. Affourtit) and Roche NimbleGen (T. Richmond and D. J. Gerhardt). We also acknowledge the use of the Center for Plant Genomics and the DNA Facility of Iowa State University, the efforts of Ed Wilcox at DNA Sequence Center at Brigham Young University, and the Genouest bioinformatics platform at the University of Rennes 1. This work was supported in part by the NSF Plant Genome Program.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003392/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-41"><em>Communicating editor: J. A. Scott</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-04-02"><span class="received-label">Received </span>April 2, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-06-15"><span class="accepted-label">Accepted </span>June 15, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2012 Salmon <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.1" data-doi="10.1038/nmeth1111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molla</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muzny</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nazareth</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Direct selection of human genomic loci by microarray hybridization</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">903</span>â<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DAlbert%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D903%26rft.epage%253D905%26rft.atitle%253DDirect%2Bselection%2Bof%2Bhuman%2Bgenomic%2Bloci%2Bby%2Bmicroarray%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1111%26rft_id%253Dinfo%253Apmid%252F17934467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17934467&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.2" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.3" data-doi="10.1046/j.1525-142x.2001.00079.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Applequist</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Comparative development of fiber in wild and cultivated cotton</span>. <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">3</span>â<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DApplequist%26rft.auinit1%253DW.%2BL.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D17%26rft.atitle%253DComparative%2Bdevelopment%2Bof%2Bfiber%2Bin%2Bwild%2Band%2Bcultivated%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1525-142x.2001.00079.x%26rft_id%253Dinfo%253Apmid%252F11256432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1525-142x.2001.00079.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11256432&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166799200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.4" data-doi="10.1186/gb-2010-11-6-r62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bainbridge</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodesch</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Whole exome capture in solution with 3 Gbp of data</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBainbridge%26rft.auinit1%253DM.%2BN.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253DR62%26rft.epage%253DR62%26rft.atitle%253DWhole%2Bexome%2Bcapture%2Bin%2Bsolution%2Bwith%2B3%2BGbp%2Bof%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-6-r62%26rft_id%253Dinfo%253Apmid%252F20565776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-6-r62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20565776&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.5" data-doi="10.1093/nar/gki487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besemer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">W451</span>â<span class="cit-lpage">W454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBesemer%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW451%26rft.epage%253DW454%26rft.atitle%253DGeneMark%253A%2Bweb%2Bsoftware%2Bfor%2Bgene%2Bfinding%2Bin%2Bprokaryotes%252C%2Beukaryotes%2Band%2Bviruses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki487%26rft_id%253Dinfo%253Apmid%252F15980510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8yL1c0NTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.6" data-doi="10.2307/2445407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Reevaluating the origin of domesticated cotton (<em>Gossypium hirsutum</em>; Malvaceae) using nuclear restriction fragment length polymorphisms (RFLPs)</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1309</span>â<span class="cit-lpage">1326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D81%26rft.spage%253D1309%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2445407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2445407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PM43400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.7" data-doi="10.1086/597817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butterworth</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horner</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Initiation and early development of fiber in wild and cultivated cotton</span>. <abbr class="cit-jnl-abbrev">Int. J. Plant Sci.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">561</span>â<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BSci.%26rft.volume%253D170%26rft.spage%253D561%26rft_id%253Dinfo%253Adoi%252F10.1086%252F597817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/597817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.8" data-doi="10.1111/j.1525-142X.2008.00272.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hovav</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Global analysis of gene expression in cotton fibers from wild and domesticated <em>Gossypium barbadense</em></span>. <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">567</span>â<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DChaudhary%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D567%26rft.epage%253D582%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bgene%2Bexpression%2Bin%2Bcotton%2Bfibers%2Bfrom%2Bwild%2Band%2Bdomesticated%2BGossypium%2Bbarbadense.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1525-142X.2008.00272.x%26rft_id%253Dinfo%253Apmid%252F18803775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1525-142X.2008.00272.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18803775&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.9" data-doi="10.1186/1471-2164-12-347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cosart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beja-Pereira</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Exome-wide DNA capture and next generation sequencing in domestic and wild species</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCosart%26rft.auinit1%253DT.%26rft.volume%253D12%26rft.spage%253D347%26rft.epage%253D347%26rft.atitle%253DExome-wide%2BDNA%2Bcapture%2Band%2Bnext%2Bgeneration%2Bsequencing%2Bin%2Bdomestic%2Band%2Bwild%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-347%26rft_id%253Dinfo%253Apmid%252F21729323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729323&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.10" data-doi="10.3732/ajb.1100356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaus</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liston</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Targeted enrichment strategies for next-generation plant biology</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">291</span>â<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1100356%26rft_id%253Dinfo%253Apmid%252F22312117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.11" data-doi="10.1093/bfgp/elq031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">RADSeq: next-generation population genetics</span>. <abbr class="cit-jnl-abbrev">Brief Funct Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">416</span>â<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BFunct%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq031%26rft_id%253Dinfo%253Apmid%252F21266344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo5OiI5LzUtNi80MTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.12" data-doi="10.1016/j.cell.2006.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">The molecular genetics of crop domestication</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1309</span>â<span class="cit-lpage">1321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDoebley%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D127%26rft.issue%253D7%26rft.spage%253D1309%26rft.epage%253D1321%26rft.atitle%253DThe%2Bmolecular%2Bgenetics%2Bof%2Bcrop%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.12.006%26rft_id%253Dinfo%253Apmid%252F17190597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17190597&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243690000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-other" id="cit-2.8.921.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turcotte</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>. <span class="cit-comment">Genetics, cytology, and evolution of Gossypium, pp. 271â375 in <em>Advances in Genetics</em>, edited by J. G. Scandalios. Academic Press, San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.8.921.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton</span>. <abbr class="cit-jnl-abbrev">BMC Genomics (in press)</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.15" data-doi="10.1111/j.1365-313X.2010.04196.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerhardt</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Repeat subtraction-mediated sequence capture from a complex genome</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">898</span>â<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFu%26rft.auinit1%253DY.%26rft.volume%253D62%26rft.issue%253D5%26rft.spage%253D898%26rft.epage%253D909%26rft.atitle%253DRepeat%2Bsubtraction-mediated%2Bsequence%2Bcapture%2Bfrom%2Ba%2Bcomplex%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04196.x%26rft_id%253Dinfo%253Apmid%252F20230488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04196.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20230488&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278080300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.16" data-doi="10.1101/gr.121327.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVicker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diederich</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Trans genomic capture and sequencing of primate exomes reveals new targets of positive selection</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1686</span>â<span class="cit-lpage">1694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.121327.111%26rft_id%253Dinfo%253Apmid%252F21795384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMC8xNjg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.17" data-doi="10.1186/1741-7007-9-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A next-generation sequencing method for overcoming the multiple gene copy problem in polyploid phylogenetics, applied to Poa grasses</span>. <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DGriffin%26rft.auinit1%253DP.%2BC.%26rft.volume%253D9%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DA%2Bnext-generation%2Bsequencing%2Bmethod%2Bfor%2Bovercoming%2Bthe%2Bmultiple%2Bgene%2Bcopy%2Bproblem%2Bin%2Bpolyploid%2Bphylogenetics%252C%2Bapplied%2Bto%2BPoa%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-9-19%26rft_id%253Dinfo%253Apmid%252F21429199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-9-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429199&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.18" data-doi="10.1016/j.tplants.2010.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genetic perspectives on crop domestication</span>. <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">529</span>â<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DGross%26rft.auinit1%253DB.%2BL.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253D529%26rft.epage%253D537%26rft.atitle%253DGenetic%2Bperspectives%2Bon%2Bcrop%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.05.008%26rft_id%253Dinfo%253Apmid%252F20541451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20541451&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282403000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.19" data-doi="10.3732/ajb.1100323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Targeted sequence capture as a powerful tool for evolutionary analysis</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">312</span>â<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1100323%26rft_id%253Dinfo%253Apmid%252F22268225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMzEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.20" data-doi="10.1104/pp.110.166736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haun</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerhardt</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">The composition and origins of genomic variation among individuals of the soybean reference cultivar Williams 82</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">645</span>â<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.166736%26rft_id%253Dinfo%253Apmid%252F21115807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTUvMi82NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.21" data-doi="10.1038/ng.2007.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balija</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molla</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genome-wide in situ exon capture for selective resequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1522</span>â<span class="cit-lpage">1527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHodges%26rft.auinit1%253DE.%26rft.volume%253D39%26rft.issue%253D12%26rft.spage%253D1522%26rft.epage%253D1527%26rft.atitle%253DGenome-wide%2Bin%2Bsitu%2Bexon%2Bcapture%2Bfor%2Bselective%2Bresequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.42%26rft_id%253Dinfo%253Apmid%252F17982454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17982454&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251267400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.22" data-doi="10.1038/nprot.2009.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharjee</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamin Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Hybrid selection of discrete genomic intervals on custom-designed microarrays for massively parallel sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">960</span>â<span class="cit-lpage">974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHodges%26rft.auinit1%253DE.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D960%26rft.epage%253D974%26rft.atitle%253DHybrid%2Bselection%2Bof%2Bdiscrete%2Bgenomic%2Bintervals%2Bon%2Bcustom-designed%2Bmicroarrays%2Bfor%2Bmassively%2Bparallel%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.68%26rft_id%253Dinfo%253Apmid%252F19478811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19478811&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266546200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.23" data-doi="10.1371/journal.pgen.0040025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hovav</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hovav</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The evolution of spinnable cotton fiber entailed prolonged development and a novel metabolism</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHovav%26rft.auinit1%253DR.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De25%26rft.epage%253De25%26rft.atitle%253DThe%2Bevolution%2Bof%2Bspinnable%2Bcotton%2Bfiber%2Bentailed%2Bprolonged%2Bdevelopment%2Band%2Ba%2Bnovel%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040025%26rft_id%253Dinfo%253Apmid%252F18248099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18248099&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.24" data-doi="10.1038/nmeth.1419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mamanova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozarewa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Target-enrichment strategies for next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">111</span>â<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMamanova%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D118%26rft.atitle%253DTarget-enrichment%2Bstrategies%2Bfor%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1419%26rft_id%253Dinfo%253Apmid%252F20111037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20111037&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274086200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.25" data-doi="10.1093/bfgp/elr033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mertes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">ElSharawy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Helvoort</span>  <span class="cit-name-given-names">J. M. L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Zaag</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Targeted enrichment of genomic DNA regions for next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">Brief Funct Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">374</span>â<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BFunct%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felr033%26rft_id%253Dinfo%253Apmid%252F22121152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo4OiIxMC82LzM3NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC85MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.26" data-doi="10.1038/nature08250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flygare</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Targeted capture and massively parallel sequencing of 12 human exomes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">272</span>â<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNg%26rft.auinit1%253DS.%2BB.%26rft.volume%253D461%26rft.issue%253D7261%26rft.spage%253D272%26rft.epage%253D276%26rft.atitle%253DTargeted%2Bcapture%2Band%2Bmassively%2Bparallel%2Bsequencing%2Bof%2B12%2Bhuman%2Bexomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08250%26rft_id%253Dinfo%253Apmid%252F19684571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19684571&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269654600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.27" data-doi="10.1038/nmeth1110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porreca</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Multiplex amplification of large sets of human exons</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">931</span>â<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPorreca%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D931%26rft.epage%253D936%26rft.atitle%253DMultiplex%2Bamplification%2Bof%2Blarge%2Bsets%2Bof%2Bhuman%2Bexons.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1110%26rft_id%253Dinfo%253Apmid%252F17934468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17934468&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.28" data-doi="10.1186/1741-7007-8-139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haigler</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hovav</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Gene expression in developing fibres of Upland cotton (<em>Gossypium hirsutum</em> L.) was massively altered by domestication</span>. <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DRapp%26rft.auinit1%253DR.%2BA.%26rft.volume%253D8%26rft.spage%253D139%26rft.epage%253D139%26rft.atitle%253DGene%2Bexpression%2Bin%2Bdeveloping%2Bfibres%2Bof%2BUpland%2Bcotton%2B%2528Gossypium%2Bhirsutum%2BL.%2529%2Bwas%2Bmassively%2Baltered%2Bby%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-8-139%26rft_id%253Dinfo%253Apmid%252F21078138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-8-139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21078138&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.29" data-doi="10.1186/gb-2011-12-9-r88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saintenac</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Targeted analysis of nucleotide and copy number variation by exon capture in allotetraploid wheat genome</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSaintenac%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253DR88%26rft.epage%253DR88%26rft.atitle%253DTargeted%2Banalysis%2Bof%2Bnucleotide%2Band%2Bcopy%2Bnumber%2Bvariation%2Bby%2Bexon%2Bcapture%2Bin%2Ballotetraploid%2Bwheat%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-9-r88%26rft_id%253Dinfo%253Apmid%252F21917144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-9-r88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21917144&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.30" data-doi="10.1093/molbev/msg065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senchina</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Rate variation among nuclear genes and the age of polyploidy in <em>Gossypium</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">633</span>â<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg065%26rft_id%253Dinfo%253Apmid%252F12679546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzQvNjMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-other" id="cit-2.8.921.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">T1GP Consortium</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â <span class="cit-article-title">A map of human genome variation from population-scale sequencing</span>. <span class="cit-source">Nature</span> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>â<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D467%26rft.spage%253D1061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.32" data-doi="10.1101/gr.4602906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A global assembly of cotton ESTs</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">441</span>â<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4602906%26rft_id%253Dinfo%253Apmid%252F16478941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNDQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.8.921.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Molecular population genetics and the search for adaptive evolution in plants</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">506</span>â<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Apmid%252F15525701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/8/921.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/913" title="Genetic Interactions Between Arabidopsis DET1 and UVH6 During Development and Abiotic Stress Response" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/931" title="Satellite DNA-Like Elements Associated With Genes Within Euchromatin of the Beetle Tribolium castaneum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/8"><span>Volume 2 Issue 8, August 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/8.cover-source.jpg?itok=etPFDPKt" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CSfpAUZT8OK8DHSZnKtZDm0MV0YDSS38fDco0gY6tco" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/8/921" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/8/921" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/8/921" class="active" data-icon-position="" data-hide-link-title="0">Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459312" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-IqeOdaYhmmdZizS49KpSOOvY7VFff1VFqc5G6CU2uIM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/921.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/8/921" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nofCB90ecyfA9olrFHOBwaBaXPybbus359WRXmZBBhU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/8/921" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/921.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459312" id="node459312--2" data-pisa="ggg;2/8/921" data-pisa-master="ggg;2/8/921" data-apath="/ggg/2/8/921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Armel Salmon</span>, <span class="highwire-citation-author" data-delta="1">Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="2">Jeffrey A. Jeddeloh</span> and <span class="highwire-citation-author" data-delta="3">Jonathan Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">921-930;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459312--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Armel Salmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Armel%2BSalmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salmon%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArmel%2BSalmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Plant and Wildlife Sciences, Brigham Young University, Provo, Utah 84602, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeffrey A. Jeddeloh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Roche NimbleGen Inc., Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BA.%2BJeddeloh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jeddeloh%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BA.%2BJeddeloh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfw@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">jfw@iastate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/8/921" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/8/921&amp;title=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/8/921&amp;title=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/8/921&amp;title=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/8/921&amp;text=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/8/921&amp;title=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/8/921&amp;t=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/8/921&amp;title=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/8/921&amp;title=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459312/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459312" id="node459312--3" data-pisa="ggg;2/8/921" data-pisa-master="ggg;2/8/921" data-apath="/ggg/2/8/921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Armel Salmon</span>, <span class="highwire-citation-author" data-delta="1">Joshua A. Udall</span>, <span class="highwire-citation-author" data-delta="2">Jeffrey A. Jeddeloh</span> and <span class="highwire-citation-author" data-delta="3">Jonathan Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">921-930;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459312--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Armel Salmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Armel%2BSalmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salmon%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArmel%2BSalmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joshua A. Udall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Plant and Wildlife Sciences, Brigham Young University, Provo, Utah 84602, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BUdall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Udall%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BA.%2BUdall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeffrey A. Jeddeloh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Roche NimbleGen Inc., Madison, Wisconsin 53719</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BA.%2BJeddeloh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jeddeloh%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BA.%2BJeddeloh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan Wendel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfw@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">jfw@iastate.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=921&amp;rft.epage=930&amp;rft.atitle=Targeted%20Capture%20of%20Homoeologous%20Coding%20and%20Noncoding%20Sequence%20in%20Polyploid%20Cotton&amp;rft.volume=2&amp;rft.issue=8&amp;rft.date=2012-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Salmon%2C+Armel&amp;rft.au=Udall%2C+Joshua+A.&amp;rft.au=Jeddeloh%2C+Jeffrey+A.&amp;rft.au=Wendel%2C+Jonathan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459312/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459312/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459312/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459312/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459312/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459312/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459312/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459312/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459312/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459312/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459312/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459312/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/8/921" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/8/921.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/8/921.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/8/921.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/8/921.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
