<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/19" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/19.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/19.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/1/19.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/19.atom" />
<meta name="HW.pisa" content="ggg;7/1/19" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in ITS2" />
<meta name="DC.Identifier" content="10.1534/g3.116.036145" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Batovska et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Internal Transcribed Spacer 2 ( ITS2 ) is a popular DNA barcoding marker; however, in some animal species it is hypervariable and therefore difficult to sequence with traditional methods. With next-generation sequencing (NGS) it is possible to sequence all gene variants despite the presence of single nucleotide polymorphisms (SNPs), insertions/deletions (indels), homopolymeric regions, and microsatellites. Our aim was to compare the performance of Sanger sequencing and NGS amplicon sequencing in characterizing ITS2 in 26 mosquito species represented by 88 samples. The suitability of ITS2 as a DNA barcoding marker for mosquitoes, and its allelic diversity in individuals and species, was also assessed. Compared to Sanger sequencing, NGS was able to characterize the ITS2 region to a greater extent, with resolution within and between individuals and species that was previously not possible. A total of 382 unique sequences (alleles) were generated from the 88 mosquito specimens, demonstrating the diversity present that has been overlooked by traditional sequencing methods. Multiple indels and microsatellites were present in the ITS2 alleles, which were often specific to species or genera, causing variation in sequence length. As a barcoding marker, ITS2 was able to separate all of the species, apart from members of the Culex pipiens complex, providing the same resolution as the commonly used Cytochrome Oxidase I ( COI ). The ability to cost-effectively sequence hypervariable markers makes NGS an invaluable tool with many applications in the DNA barcoding field, and provides insights into the limitations of previous studies and techniques." />
<meta name="DC.Contributor" content="Jana Batovska" />
<meta name="DC.Contributor" content="Noel O. I. Cogan" />
<meta name="DC.Contributor" content="Stacey E. Lynch" />
<meta name="DC.Contributor" content="Mark J. Blacket" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in ITS2" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Internal Transcribed Spacer 2 (&lt;i&gt;ITS2&lt;/i&gt;) is a popular DNA barcoding marker; however, in some animal species it is hypervariable and therefore difficult to sequence with traditional methods. With next-generation sequencing (NGS) it is possible to sequence all gene variants despite the presence of single nucleotide polymorphisms (SNPs), insertions/deletions (indels), homopolymeric regions, and microsatellites. Our aim was to compare the performance of Sanger sequencing and NGS amplicon sequencing in characterizing &lt;i&gt;ITS2&lt;/i&gt; in 26 mosquito species represented by 88 samples. The suitability of &lt;i&gt;ITS2&lt;/i&gt; as a DNA barcoding marker for mosquitoes, and its allelic diversity in individuals and species, was also assessed. Compared to Sanger sequencing, NGS was able to characterize the &lt;i&gt;ITS2&lt;/i&gt; region to a greater extent, with resolution within and between individuals and species that was previously not possible. A total of 382 unique sequences (alleles) were generated from the 88 mosquito specimens, demonstrating the diversity present that has been overlooked by traditional sequencing methods. Multiple indels and microsatellites were present in the &lt;i&gt;ITS2&lt;/i&gt; alleles, which were often specific to species or genera, causing variation in sequence length. As a barcoding marker, &lt;i&gt;ITS2&lt;/i&gt; was able to separate all of the species, apart from members of the &lt;i&gt;Culex pipiens&lt;/i&gt; complex, providing the same resolution as the commonly used Cytochrome Oxidase I (&lt;i&gt;COI&lt;/i&gt;). The ability to cost-effectively sequence hypervariable markers makes NGS an invaluable tool with many applications in the DNA barcoding field, and provides insights into the limitations of previous studies and techniques.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/19" />
<meta name="citation_id" content="7/1/19" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/19" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/19.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/19.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/19.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036145" />
<meta name="citation_pmid" content="27799340" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="19" />
<meta name="citation_lastpage" content="29" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jana Batovska" />
<meta name="citation_author_institution" content="BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia" />
<meta name="citation_author_email" content="jana.batovska@ecodev.vic.gov.au" />
<meta name="citation_author" content="Noel O. I. Cogan" />
<meta name="citation_author_institution" content="BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia" />
<meta name="citation_author" content="Stacey E. Lynch" />
<meta name="citation_author_institution" content="BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia" />
<meta name="citation_author" content="Mark J. Blacket" />
<meta name="citation_author_institution" content="BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7864-5712" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Adey;citation_author=H. G. Morrison;citation_author=X. Asan;citation_author=J. O. Xun;citation_author=. Kitzman;citation_title=Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition.;citation_pages=1-17;citation_volume=11;citation_year=2010;citation_doi=10.1186/1465-6906-11-S1-I1" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=I. Álvarez;citation_author=J. F. Wendel;citation_title=Ribosomal ITS sequences and plant phylogenetic inference.;citation_pages=417-434;citation_volume=29;citation_year=2003;citation_issue=3;citation_pmid=14615184;citation_doi=10.1016/S1055-7903(03)00208-2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Arif;citation_author=C. Daniels;citation_author=T. Bayer;citation_author=E. Banguera-Hinestroza;citation_author=A. Barbrook;citation_title=Assessing Symbiodinium diversity in scleractinian corals via next-generation sequencing-based genotyping of the ITS2 rDNA region.;citation_pages=4418-4433;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12869" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=K. F. Armstrong;citation_author=S. L. Ball;citation_title=DNA barcodes for biosecurity: invasive species identification;citation_pages=1813-1823;citation_volume=360;citation_year=2005;citation_issue=1462;citation_pmid=16214740;citation_doi=10.1098/rstb.2005.1713" />
<meta name="citation_reference" content="citation_journal_title=Journal of the American Mosquito Control Association;citation_journal_abbrev=J Am Mosq Control Assoc;citation_author=S. Aspen;citation_author=M. B. Crabtree;citation_author=H. M. Savage;citation_title=Polymerase chain reaction assay identifies Culex nigripalpus: part of an assay for molecular identification of the common Culex (Culex) mosquitoes of the eastern United States.;citation_pages=115-120;citation_volume=19;citation_year=2003;citation_issue=2;citation_pmid=12825660" />
<meta name="citation_reference" content="citation_journal_title=Electronic Journal of Biology;citation_author=A. K. Banerjee;citation_author=N. Arora;citation_author=U. S. Murty;citation_title=How far is ITS2 reliable as a phylogenetic marker for the mosquito genera.;citation_pages=61-68;citation_volume=3;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=J. Batovska;citation_author=M. J. Blacket;citation_author=K. Brown;citation_author=S. E. Lynch;citation_title=Molecular identification of mosquitoes (Diptera: Culicidae) in southeastern Australia.;citation_pages=3001-3011;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Fungal Ecol.;citation_author=A. L. Bazzicalupo;citation_author=M. Bálint;citation_author=I. Schmitt;citation_title=Comparison of ITS1 and ITS2 rDNA in 454 sequencing of hyperdiverse fungal communities.;citation_pages=102-109;citation_volume=6;citation_year=2013;citation_doi=10.1016/j.funeco.2012.09.003" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=C. Bertrand;citation_author=D. H. Janzen;citation_author=W. Hallwachs;citation_author=J. M. Burns;citation_author=J. F. Gibson;citation_title=Mitochondrial and nuclear phylogenetic analysis with Sanger and next-generation sequencing shows that, in Área de Conservación Guanacaste, northwestern Costa Rica, the skipper butterfly named Urbanus belli (family Hesperiidae) comprises three morphologically cryptic species.;citation_pages=153;citation_volume=14;citation_year=2014;citation_pmid=25005355;citation_doi=10.1186/1471-2148-14-153" />
<meta name="citation_reference" content="citation_journal_title=Parasit. Vectors;citation_author=B. P. Bourke;citation_author=T. P. Oliveira;citation_author=L. Suesdek;citation_author=E. S. Bergo;citation_author=M. A. M. Sallum;citation_title=A multi-locus approach to barcoding in the Anopheles strodei subgroup (Diptera: Culicidae).;citation_pages=111;citation_volume=6;citation_year=2013;citation_pmid=23597081;citation_doi=10.1186/1756-3305-6-111" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. E. Bower;citation_author=R. D. Cooper;citation_author=N. W. Beebe;citation_title=Internal repetition and intraindividual variation in the rDNA ITS1 of the anopheles punctulatus group (Diptera: Culicidae): multiple units and rates of turnover.;citation_pages=66-79;citation_volume=68;citation_year=2009;citation_issue=1;citation_pmid=19123014;citation_doi=10.1007/s00239-008-9188-z" />
<meta name="citation_reference" content="citation_journal_title=Medical and veterinary entomology;citation_journal_abbrev=Med Vet Entomol;citation_author=A. Cywinska;citation_author=F. F. Hunter;citation_author=P. D. N. Hebert;citation_title=Identifying Canadian mosquito species through DNA barcodes.;citation_pages=413-424;citation_volume=20;citation_year=2006;citation_issue=4;citation_pmid=17199753;citation_doi=10.1111/j.1365-2915.2006.00653.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=D. H. Foley;citation_author=R. C. Wilkerson;citation_author=R. D. Cooper;citation_author=M. E. Volovsek;citation_author=J. H. Bryan;citation_title=A molecular phylogeny of Anopheles annulipes (Diptera: Culicidae) sensu lato: the most species-rich anopheline complex.;citation_pages=283-297;citation_volume=43;citation_year=2007;citation_issue=1;citation_pmid=17126567;citation_doi=10.1016/j.ympev.2006.10.008" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Hajibabaei;citation_author=S. Shokralla;citation_author=X. Zhou;citation_author=G. A. C. Singer;citation_author=D. J. Baird;citation_title=Environmental barcoding: a next-generation sequencing approach for biomonitoring applications using river benthos.;citation_pages=e17497-e17497;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21533287;citation_doi=10.1371/journal.pone.0017497" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=J. M. Hancock;citation_author=A. P. Vogler;citation_title=How slippage-derived sequences are incorporated into rRNA variable-region secondary structure: implications for phylogeny reconstruction.;citation_pages=366-374;citation_volume=14;citation_year=2000;citation_issue=3;citation_pmid=10712842;citation_doi=10.1006/mpev.1999.0709" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. Huang;citation_author=A. Madan;citation_title=CAP3: A DNA Sequence Assembly Program;citation_pages=868-877;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508846;citation_doi=10.1101/gr.9.9.868" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Mem. Inst. Oswaldo Cruz;citation_author=M. Laurito;citation_author=T. M. P. de Oliveira;citation_author=W. R. Almirón;citation_author=M. A. M. Sallum;citation_title=COI barcode vs. morphological identification of Culex (Culex) (Diptera: Culicidae) species: a case study using samples from Argentina and Brazil.;citation_pages=110-122;citation_volume=108;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=C.-P. Lin;citation_author=B. N. Danforth;citation_title=How do insect nuclear and mitochondrial gene substitution patterns differ? Insights from Bayesian analyses of combined datasets.;citation_pages=686-702;citation_volume=30;citation_year=2004;citation_issue=3;citation_pmid=15012948;citation_doi=10.1016/S1055-7903(03)00241-0" />
<meta name="citation_reference" content="citation_journal_title=Eur. Mosq. Bull.;citation_author=Y.-M. Linton;citation_author=A. Samanidou-Voyadjoglou;citation_author=L. Smith;citation_author=R. E. Harbach;citation_title=New occurrence records for Anopheles maculipennis and An. messeae in northern Greece based on DNA sequence data.;citation_pages=31-36;citation_volume=11;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. P. Little;citation_title=DNA barcode sequence identification incorporating taxonomic hierarchy and within taxon variability.;citation_pages=e20552-e20552;citation_volume=6;citation_year=2011;citation_issue=8;citation_pmid=21857897;citation_doi=10.1371/journal.pone.0020552" />
<meta name="citation_reference" content="citation_journal_title=Memorias do Instituto Oswaldo Cruz;citation_journal_abbrev=Memorias do Instituto Oswaldo Cruz;citation_author=M. T. Marrelli;citation_author=M. A. M. Sallum;citation_author=O. Marinotti;citation_title=The second internal transcribed spacer of nuclear ribosomal DNA as a tool for Latin American anopheline taxonomy - a critical review.;citation_pages=817-832;citation_volume=101;citation_year=2006;citation_issue=8;citation_pmid=17293975;citation_doi=10.1590/S0074-02762006000800002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=J. E. McCormack;citation_author=S. M. Hird;citation_author=A. J. Zellmer;citation_author=B. C. Carstens;citation_author=R. T. Brumfield;citation_title=Applications of next-generation sequencing to phylogeography and phylogenetics.;citation_pages=526-538;citation_volume=66;citation_year=2013;citation_pmid=22197804;citation_doi=10.1016/j.ympev.2011.12.007" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=C. Paredes-Esquivel;citation_author=M. J. Donnelly;citation_author=R. E. Harbach;citation_author=H. Townson;citation_title=A molecular phylogeny of mosquitoes in the Anopheles barbirostris subgroup reveals cryptic species: implications for identification of disease vectors.;citation_pages=141-151;citation_volume=50;citation_year=2009;citation_pmid=19000771;citation_doi=10.1016/j.ympev.2008.10.011" />
<meta name="citation_reference" content="citation_journal_title=ScientificWorldJournal;citation_author=E. Pfeiler;citation_author=C. A. Flores-López;citation_author=J. G. Mada-Vélez;citation_author=J. Escalante-Verdugo;citation_author=T. A. Markow;citation_title=Genetic diversity and population genetics of mosquitoes (Diptera: Culicidae: Culex spp.) from the Sonoran desert of North America.;citation_pages=e724609;citation_volume=2013;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Biogeogr.;citation_author=L. Puslednik;citation_author=R. C. Russell;citation_author=J. W. O. Ballard;citation_title=Phylogeography of the medically important mosquito Aedes (Ochlerotatus) vigilax (Diptera: Culicidae) in Australasia.;citation_pages=1333-1346;citation_volume=39;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Zool. J. Linn. Soc.;citation_author=J. F. Reinert;citation_author=R. E. Harbach;citation_author=I. J. Kitching;citation_title=Phylogeny and classification of tribe Aedini (Diptera: Culicidae).;citation_pages=700-794;citation_volume=157;citation_year=2009;citation_doi=10.1111/j.1096-3642.2009.00570.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Ecol Lett;citation_author=K. A. Selkoe;citation_author=R. J. Toonen;citation_title=Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers.;citation_pages=615-629;citation_volume=9;citation_year=2006;citation_issue=5;citation_pmid=16643306;citation_doi=10.1111/j.1461-0248.2006.00889.x" />
<meta name="citation_reference" content="citation_journal_title=Insect molecular biology;citation_journal_abbrev=Insect Mol Biol;citation_author=C. Severini;citation_author=F. Silvestrini;citation_author=P. Mancini;citation_author=G. La Rosa;citation_author=M. Marinucci;citation_title=Sequence and secondary structure of the rDNA second internal transcribed spacer in the sibling species Culex pipiens L. and Cx. quinquefasciatus Say (Diptera: Culicidae).;citation_pages=181-186;citation_volume=5;citation_year=1996;citation_issue=3;citation_pmid=8799736;citation_doi=10.1111/j.1365-2583.1996.tb00052.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. Shokralla;citation_author=J. F. Gibson;citation_author=H. Nikbakht;citation_author=D. H. Janzen;citation_author=W. Hallwachs;citation_title=Next-generation DNA barcoding: using next-generation sequencing to enhance and accelerate DNA barcode capture from single specimens.;citation_pages=892-901;citation_volume=14;citation_year=2014;citation_pmid=24641208" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Song;citation_author=L. Shi;citation_author=D. Li;citation_author=Y. Sun;citation_author=Y. Niu;citation_title=Extensive pyrosequencing reveals frequent intra-genomic variations of internal transcribed spacer regions of nuclear ribosomal DNA.;citation_pages=e43971-e43971;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22952830" />
<meta name="citation_reference" content="citation_journal_title=Parasit. Vectors;citation_author=J.-S. Sum;citation_author=W.-C. Lee;citation_author=A. Amir;citation_author=K. A. Braima;citation_author=J. Jeffery;citation_title=Phylogenetic study of six species of Anopheles mosquitoes in Peninsular Malaysia based on inter-transcribed spacer region 2 (ITS2) of ribosomal DNA.;citation_pages=309;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=J. Dudley;citation_author=M. Nei;citation_author=S. Kumar;citation_title=MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0.;citation_volume=24;citation_year=2007;citation_pmid=17488738;citation_doi=10.1093/molbev/msm092" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=H. R. Taylor;citation_author=W. E. Harris;citation_title=An emergent science on the brink of irrelevance: a review of the past 8 years of DNA barcoding.;citation_pages=377-388;citation_volume=12;citation_year=2012;citation_pmid=22356472;citation_doi=10.1111/j.1755-0998.2012.03119.x" />
<meta name="citation_reference" content="citation_journal_title=J. Med. Entomol.;citation_author=T. Toma;citation_author=I. Miyagi;citation_author=M. B. Crabtree;citation_author=B. R. Miller;citation_title=Identification of Culex vishnui subgroup (Diptera: Culicidae) mosquitoes from the Ryukyu Archipelago, Japan: development of a species-diagnostic polymerase chain reaction assay based on sequence variation in ribosomal DNA spacers.;citation_volume=37;citation_year=2000;citation_pmid=10916296;citation_doi=10.1603/0022-2585-37.4.554" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=I. Cutcutache;citation_author=T. Koressaar;citation_author=J. Ye;citation_author=B. C. Faircloth;citation_title=Primer3–new capabilities and interfaces.;citation_volume=40;citation_year=2012;citation_pmid=22730293;citation_doi=10.1093/nar/gks596" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=P. Vargas;citation_author=H. A. McAllister;citation_author=C. Morton;citation_author=S. L. Jury;citation_author=M. J. Wilkinson;citation_title=Polyploid speciation in Hedera (Araliaceae): phylogenetic and biogeographic insights based on chromosome counts and ITS sequences.;citation_pages=165-179;citation_volume=219;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=V. Versteirt;citation_author=Z. T. Nagy;citation_author=P. Roelants;citation_author=L. Denis;citation_author=F. C. Breman;citation_title=Identification of Belgian mosquito species (Diptera: Culicidae) by DNA barcoding.;citation_pages=449-457;citation_volume=15;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Infection, genetics and evolution :  journal of molecular epidemiology and evolutionary genetics in infectious diseases;citation_journal_abbrev=Infect Genet Evol;citation_author=C. Walton;citation_author=P. Somboon;citation_author=S. M. O’Loughlin;citation_author=S. Zhang;citation_author=R. E. Harbach;citation_title=Genetic diversity and molecular identification of mosquito species in the Anopheles maculatus group using the ITS2 region of rDNA.;citation_pages=93-102;citation_volume=7;citation_year=2007;citation_issue=1;citation_pmid=16782411" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Wang;citation_author=C. Li;citation_author=X. Guo;citation_author=D. Xing;citation_author=Y. Dong;citation_title=Identifying the main mosquito species in china based on DNA barcoding.;citation_pages=e47051-e47051;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23071708;citation_doi=10.1371/journal.pone.0047051" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=D. M. Wesson;citation_author=C. H. Porter;citation_author=F. H. Collins;citation_title=Sequence and secondary structure comparisons of ITS rDNA in mosquitoes (Diptera: Culicidae).;citation_pages=253-269;citation_volume=1;citation_year=1992;citation_issue=4;citation_pmid=1364170;citation_doi=10.1016/1055-7903(92)90001-W" />
<meta name="citation_reference" content="citation_journal_title=J. Med. Entomol.;citation_author=R. C. Wilkerson;citation_author=J. F. Reinert;citation_author=C. Li;citation_title=Ribosomal DNA ITS2 sequences differentiate six species in the Anopheles crucians complex (Diptera: Culicidae).;citation_volume=41;citation_year=2004;citation_pmid=15185940;citation_doi=10.1603/0022-2585-41.3.392" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Bot.;citation_author=V. Wissemann;citation_title=Molecular evidence for allopolyploid origin of the Rosa canina complex (Rosaceae, Rosoideae).;citation_pages=176-178;citation_volume=76;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Zhang;citation_author=K. Kobert;citation_author=T. Flouri;citation_author=A. Stamatakis;citation_title=PEAR: a fast and accurate Illumina Paired-End reAd mergeR.;citation_pages=614-620;citation_volume=30;citation_year=2013;citation_pmid=24142950" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in ITS2" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="Internal Transcribed Spacer 2 ( ITS2 ) is a popular DNA barcoding marker; however, in some animal species it is hypervariable and therefore difficult to sequence with traditional methods. With next-generation sequencing (NGS) it is possible to sequence all gene variants despite the presence of single nucleotide polymorphisms (SNPs), insertions/deletions (indels), homopolymeric regions, and microsatellites. Our aim was to compare the performance of Sanger sequencing and NGS amplicon sequencing in characterizing ITS2 in 26 mosquito species represented by 88 samples. The suitability of ITS2 as a DNA barcoding marker for mosquitoes, and its allelic diversity in individuals and species, was also assessed. Compared to Sanger sequencing, NGS was able to characterize the ITS2 region to a greater extent, with resolution within and between individuals and species that was previously not possible. A total of 382 unique sequences (alleles) were generated from the 88 mosquito specimens, demonstrating the diversity present that has been overlooked by traditional sequencing methods. Multiple indels and microsatellites were present in the ITS2 alleles, which were often specific to species or genera, causing variation in sequence length. As a barcoding marker, ITS2 was able to separate all of the species, apart from members of the Culex pipiens complex, providing the same resolution as the commonly used Cytochrome Oxidase I ( COI ). The ability to cost-effectively sequence hypervariable markers makes NGS an invaluable tool with many applications in the DNA barcoding field, and provides insights into the limitations of previous studies and techniques." />
<meta name="og-title" property="og:title" content="Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in ITS2" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/19" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Internal Transcribed Spacer 2 ( ITS2 ) is a popular DNA barcoding marker; however, in some animal species it is hypervariable and therefore difficult to sequence with traditional methods. With next-generation sequencing (NGS) it is possible to sequence all gene variants despite the presence of single nucleotide polymorphisms (SNPs), insertions/deletions (indels), homopolymeric regions, and microsatellites. Our aim was to compare the performance of Sanger sequencing and NGS amplicon sequencing in characterizing ITS2 in 26 mosquito species represented by 88 samples. The suitability of ITS2 as a DNA barcoding marker for mosquitoes, and its allelic diversity in individuals and species, was also assessed. Compared to Sanger sequencing, NGS was able to characterize the ITS2 region to a greater extent, with resolution within and between individuals and species that was previously not possible. A total of 382 unique sequences (alleles) were generated from the 88 mosquito specimens, demonstrating the diversity present that has been overlooked by traditional sequencing methods. Multiple indels and microsatellites were present in the ITS2 alleles, which were often specific to species or genera, causing variation in sequence length. As a barcoding marker, ITS2 was able to separate all of the species, apart from members of the Culex pipiens complex, providing the same resolution as the commonly used Cytochrome Oxidase I ( COI ). The ability to cost-effectively sequence hypervariable markers makes NGS an invaluable tool with many applications in the DNA barcoding field, and provides insights into the limitations of previous studies and techniques." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
    <title>Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in ITS2 | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6JWjiByheC7HybiQC_65D6NpBLwnVnJutPRfSrZLH8E","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460533","apath":"\/ggg\/7\/1\/19.atom","pisa":"ggg;7\/1\/19","ac":{"\/ggg\/7\/1\/19.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/19.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/19"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/19"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/19":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pAUqgfMC1DGp\\\\\\\/xpK7Fc9iDT2UXcYaEfDLLgmOczbbqXbEsE8VIAgNzM9xifHaaUReYRZ7YIcIpXzpYT5El1WWiHGWNBaekW6pGV2xH6WB8RZf3Ei9BUxbTZO76zCD+011TNLTgiTzuj\\\\\\\/5EWFklaEFbiG9VPTJGQDIsJLNzd2aUkd1KmjVVa6K48LfztW\\\\\\\/EHg55+fI2GdzQSBKueETVvF0Fp1YYzbOu+L5JqmD6zFxuQ0dB8D\\\\\\\/D61i+d1utNvU5I6FgxF1yvpQgPQAMqVaUAe8ny8sCCAKzfFK0zo3ug7ZYyhV4WbeTItF+7vv0BZzJncyNxQst5IDpaU9HeHg+T+SxLbm4jm4P6CdOry4uGt7E3\\\\\\\/5+6i9KOin9SZLTdIp2GN2xRNoAbYttFaya+fw4MB+WwfIWg2+i0tO4zfK7HAyChjTdMAfbcJ43T5YJtkWCgLRyX8V5izjllElG7ACMiGas+aL7FwwxdGex6FYrxzrdjhf54W1Y6vdPX\\\\\\\/U2Y+jOkgO3lkzE+jsXJnd16dZBA20vqTlogpfuzps47F97iPJiRX3f\\\\\\\/o9iq3qdEUUAHGE0V0kJ6lxZailYmIFpQ6r4uZ6TV2KY38OnK2zrzIc1iPQvNGIldfEI2\\\\\\\/crBQMVdqJsHX9Nc3HBYeZ7\\\\\\\/GKZKr5vZk6BOjEWWALmvYgj5tT8wqoFcvWRpPxGba06eO\\\\\\\/R24b6T0UWdy\\\\\\\/PLgFJ7QN0kCAJJ2CmVUZqUJiHQgTd\\\\\\\/uu7aRjeSs309MnTANOkXLiWUUyEUL9YLYBVsCpEcwQOTvrZQEn2erk+gkG1DODiwLlagw3t\\\\\\\/BXVRY0ij9P1fr00IWdHS+250PUfTKXd2XAF5hCGXjcfmUTai0qJs5Y886Wc2PsfWKErYipxQLusiLMNSM7Sn\\\\\\\/xmBIlDxGJMK\\\\\\\/bvztJdDnJZRw6qWYCFFddbkcWYU+bgPOeDiEQP68zyQIHsz27sGp9WucXyG73QCeKndUjpIlndRurtM\\\\\\\/qeB46e\\\\\\\/HGc7fQNIDayev5wwiDfRu7RIkq5\\\\\\\/GXSR2ev6xvSTqpX41742IljsWALjxvlx7x4C0KFSv8Jz\\\\\\\/jyz\\\\\\\/6ej3jd88Puy0gaqtS4G+t65pP8FE7oKJgGpznWFWGH+QrzvQ74lOniqyRWqCFgc8d084b1MWr8H6KromZ\\\\\\\/MgrmK4fHzEAZPIKEuS3lgEJPRC\\\\\\\/xifTOWTnPBP23dRfvsJEqJu+djbqrO2PqVB4ZPK81hBopm5\\\\\\\/7MhV7H5PdY\\\\\\\/amn2gxHUJda7ZydPTMfsvefX5OaXVrAhF1Kh4Lzt\\\u0022,\\\u0022iv\\\u0022:\\\u00223wsdtvKKRVzrU+ZZ4L4htg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e91f98cea401bdd640963273e4d078a764b6869c0e44fb2700efa3356dce2f26\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NtHUwxVnOBg0p+x4rPWB7P+9svNYZfjA4Y3yQZEpYs9C1YBFggYXEgO4NZqdcF78YQb50BOQLNxUPv3oHDS3Usxp9d3EGfzvSf9nKIy5jDq\\\\\\\/z7IU2baP8FGXoxxBpl6AUIc9rtMMWwWj22PXfxNt2PZNc0VW1QpCiFjqm5yN0Wj+Ntz8kyPcOp9X65PF7LL3GrA0LpPpRDIigf7zoLa6KegjnuuOl2ujmNsCibkpcgnhO72hedauA3zmY\\\\\\\/0PshamSabNYoOnrzxYkhkwUJkf6ytsqKIVZgVYHSRf7+7S7eXHIZxP3oI66f6qxx2cVhzn7XInnVegDUNWWqYts\\\\\\\/8BhLN39QDcgddTyVh6\\\\\\\/Sx1EHtRjSHtSUVcvmmnyBSnygcSGwp3nOp4HtSsf4T5SLAxN6NczN6tJp1i5NmUSaqzKi5xn+8qzx1h+3oJbABF5OaiYD5y2AplxWa6C\\\\\\\/QflwmiYMCOiuDyECUA4mlLzvw4795vqzPBkx3bBM7CJBYGjQ5OQ9SPnFs376Q+wuGW\\\\\\\/xwVZZeSAcVaoNPPB\\\\\\\/4macfVocubtoRAPwdneZ04amZkwOreYTwMaiOHeyXqRIXihfOoRjRQF4znvfiyPOIMSnqx1Ycmtr6DqV\\\\\\\/KwLYlNccuHhBl3ZDDlW3dR0g4NqhiDMFH2yu4tT68Xv3ABQFm+O73HSeYHI0viTaong061ok\\\\\\\/90cWgmVfpOLu7w60AHDNY8O9YDqfUXMGgcNFdefIE0kG4CkNSqGQ\\\\\\\/AZgmapHV0ixgJXuFYgipmBPieAJLjjk+\\\\\\\/845guEXqPuOj2a4FUaSBNdcdBbYU9jqPvVNcnF7xY69vZxtT9Gcnmma+8e9v+nB8ZpTaWo8W\\\\\\\/Xab\\\\\\\/Gl8b0FNdMAf6YqDSRV59omG+IwIMEJYj1TxLXzTB8grehhHLzeO+vpiCb3LGsOD9UsTdbCLUZuwv24Mb4bnm+\\\\\\\/tkaWFrz5RX+4f00OEaPd6t4jUoAcqrHDiMZy0Sk63dFuhga2OpqMXGE+1VJZdPCIkg8PyOhGQ+PAbInWYRQ7IkNJdiRcs+jbQfjU1S1W5L5ryv9KkzsxH7eEYUQeSb1ImZDSdyTNDVqJGwTFIfm2pPVQmZmnMQbkd8SuPRl6+WoaC2d+VM41kImzdL5eP7cYjaNQo5DYrAlLvIFnrzTWWqDMD6\\\\\\\/69JaR1ckcE60qwHZHbDqSfMLo\\\\\\\/L1DlmpxFo+XYOwqM\\\\\\\/u4J+\\\\\\\/iV1ajetbW5CVVAm5K0Ar0YTB5HhBMddCejX+hEjEmvicoNbWkdJ+lmptZbYZeJS+inPe+CtMrHUucR0Pi04IcR4R0j0G\\\\\\\/l3SwhE9UwcviLA2XVbu5QMXbN8EPgJ9KWNrFupF5IVYo3Jb2TqkmjkwP\\\\\\\/P+yIg6JfyMWmMzGcw8RJ3ciIptZKpwT2ct7L8TvTAcNcT38XV2Hy2ItUcWR1jDJrilu+ejmWDL3\\\\\\\/vFpqZttRG4vI67apIuMtOkDc7wkRWSlLKAndkXVSI6\\\\\\\/rZSweouBgWr0MhZrGfgXcA+z1HBSa+sCesFtYyHypF9FbAhJ3DHhM8uwLFbkD+2s2lxRjJgLFcB+30LmOi4Li4F45Wbom0TcWRcySDMQq2PjryNKK5aH\\\\\\\/vZloTTnLq0MsrcNCmBGlrKlX\\\\\\\/\\\\\\\/qaRJ0\\\\\\\/lNskjFU3ay63fNm0HaQBecfSOOIMrjgq3izr6xbt6ZusTpJXVGUjN3ODBq\\\\\\\/oV+vSt9Hz6EI5VRiuvJr6iklnWG6h2\\\\\\\/BlmNEZfEWpTyDwG3WBpAZDP+0FfcAYswOPnY281nxqA\\\\\\\/oC6AT1KkO2bsNrXI\\\\\\\/dVLN6kS1XUeOOiqvCGjnDXeU0vLQF7MaxIkzgNov1vdgp\\\\\\\/H0jXFkyk\\\\\\\/I843QO\\\\\\\/pUUTvJr\\\\\\\/0PIqMb1iCi5G9g5r660nFRMAOn8nyZRMxvzVurb+hqUjcamoaxSS0UbSHwLyCEir2wizvsZ+TtJm\\\\\\\/EaqIs7VloByV1VQd7x1FEewhqBEpBizcMQH0TPeAnBknjNN1kLCKMEFnmYpj7ngkMhQC\\\\\\\/rI46Z1UkCMTmdJOeI01d86gc5UTzK0xRrQF9ss5Tn9ZPtd\\\\\\\/oRFZd4RlMbpIMuNaMijKCzMt3LHwesOIhu4dlwxuGu1pmpPEZ1zHt2AApsWJFZGn\\\\\\\/XRhcnQai5vAat+PI1nvTLqGbEFbje3\\\\\\\/WsEzb9GFvwfxtOyj8fs+SeMO7vpjcxRdRO7C+2mortz49aWSQ+rMyZTTcHoJmgCD3ve+SP1ajPEyc8kuthaTiSVeSGO8Vhi0JJ49qTVLJGLW2aNWW+Rnma+EA6df0X2YgKZEt0A6nYG3HoQQQNW\\\\\\\/o3bfoJpfT13zqUdALRAeEuwbEsqNu4MF0YiBO0XWheDqMPc\\\\\\\/ey\\\\\\\/YprriaJfFK6kydXYCCoYq8Awco7UAmHJHDL0G8piphEai9szRWsnv\\\\\\\/5od8LiH0otwDFtk2QFOCik5lSxeVoCUF7ZwNcLs\\\\\\\/5\\\\\\\/nqgK8MNn0z4BtdA2qw4D9enVXMQD4uPQ69n6eT0pey4IN615ltRG22kf3sspCTDIOh8FnqzM4WK2slYRRL1B5gv2gMBpII1S7tHeCTpI78gPVAZ6NwpoF2747c9FgnELbzNwpfe+em1nDmd9CgMKtiNC6mXYxpQtxbkJJh4\\\\\\\/kpniXWYNvCHerbiH\\\\\\\/npLrqmWC\\\\\\\/DP8LK1eVTGq2iaBx8J5zHBwVa51mUKXViwVHemw1dCRfJyAg1GOwXS52kEfU+jvpHtM7v72\\\\\\\/pSEUgkiptg6YXydjvY4cE+8eCNomnpCAHzvhJ8e4Qy2E3w2YhVX0pU2jNSf7a+CmouGO3mHQtajJWP5ktpx7kV3fTn017UVc3vWM3YR467+0zMfqE\\\\\\\/ZhQGvRCjedftf\\\\\\\/kqA4np1gKEc\\\\\\\/vYB3uv5s28p2fTc4VTSymbVK6HYzqnuqefl2o7GRuc09+eTqW31j9xZM4OM7nKnhzVVvTUkchhtcgCJHs\\\\\\\/\\\\\\\/bpALzgI6a9UmXvdf1Zbp74i4lJ3q2dqFwr0ucxtKM5lNZGXYslcIeXs0bxa46nLYWB1IhQtjy+GQzXih+ckeVkgmsg4mpEbsYvoDE5jXKB\\\\\\\/hc4ehs2o52KafwvTRXx0kUmv21YF8G3Xai+0MooDNsPsu7hSGNUWYuTGD9LF0x1h+qmW9m6uYpaqXEo7Yac7ENBk928vVnLsSEvQMeMimlmwAd+kUHLBUxEEqSiBo\\\\\\\/6tE8TqsfYY4I4vujfFZD2WRcOjtTt3bxQJ8lMQtrl\\\\\\\/E2QAJ5krzWBwguckyTvVXimI+FoKocf8QoHN1zmTb8pQYgrnnodHNTj5diWPNfUnIcgoIXM0P9q\\\\\\\/+QzVFcYUO99rieWxjcMdQqOQ1LNNoa261Fl51Tbfz7A7Qsx5FeByQ9fIyDruHgpe88BJCEEs3vaT5o474\\\\\\\/nrgoX4mbez\\\\\\\/m6riBt1KGD195nGPgO0rdm45rWcGLU0YEDqp1+LXU4yls3fl9WrWewwTjvdNLWv8m1VPr11WYNrmO5ADxiqYUA54msNrNAUfr\\\\\\\/gYuBUfVfRg3EFe5e+kWyogkqIbLJCoAMeobf5zEaubap+D+CBidaTzGhU+Lkq6gB2+86R70mL73cZtPMEpKIDI71bb\\\\\\\/hvyl9nQrm\\\\\\\/q2uvrqf8VeWqsjpANr+5\\\\\\\/ZL+q5fySxMszWsSnxS4U4b7htqqe8t5Tkh8aievgFE9iB95g7\\\\\\\/rABiD5wzOVhMp6MyVcn8r12tqGIE3RMotjODziiD+yKrShZT6AgejUQBhrtXya8+9PTlobwDex3FfwGrI4ukba3ZGJuzBm155Jsu9Jp1GVKGqnQhktVhy8Zek3Bm9P5MTXBjWY8c+ts8PUiGh4Pi8qgMveZgBx6bKVicBt2Di1mscZn+wPu2+3byoiZRxY+rck6YF8Ri7LtfbWk1BPk3unX0ZnfhNjVX06m4tiNBaMIn7xjhYi5zfmrIUFfT16ALMN0DqWYAfGeD2fbWjgC1bYIvIFMP7RYmCMLXMREfxELFkVYgQAmbk7qA84Y4TLM6z44\\\\\\\/PhUWiRAkuGlzYg6xsBMLUmYuznOPwdbUpvnSRp8FJlt0LtnmuMqK5JUFfoX\\\\\\\/z4QHmwhEXXnISfVSqxWLaSZcAEUg3ddQWyz7Sjngs\\\\\\\/isfT69P9LUyAyPLG7jjepGlWb5UpovmwhjlQDz+7WG1TP4WoLN\\\\\\\/RO4aNlpBHCUg5THEWYWPdwuzTxeI234gu\\\\\\\/9JJOo34wydNBY812yyZsaqStf8TUZVdN88GfViVIoCsV7SHvS8g8hnfdRAmyHv7iyIqL78wUtq9tG0o0agcCGRorzTMKQPOs7b\\\\\\\/YkKPEAiIdOXywH\\\\\\\/0mSyfjmNCfoVrHJoBZ3\\\\\\\/sKt78rGYTY2FeElY4jp3kTG+3IBjLeFf+jHGFQEmsmwV94lyD4jVlqaBheLTGHcJ9YtxPpKYPdd+TdUU6waMu9Bka6ICb6xQ6lv0NwmSCRbIdSy+IXDB0jF0rtDf0joC\\\\\\\/icuSMZqwQTgf2fxzlHhVWbkQ1bIq9vO8Hkm+h3xhFpm30qLTkyeC91KRoj3OQo9DNZma13T1SEyCmPd95jbqwnRXYnPb330taroQ4w3IDctedtFE0Ms1LGRY7DvEscRuEYpW7KMRXECEk+C6o5NoBOwZ7y8kLeiwg3QQz0jGixoFjPxOhy9hU17Eds+xjvTToudWHSc7Ab92GKUOwCHwNaqQg0TOoykUjwlfNSwj0qFXllvTiva2NEK43Lm\\\\\\\/WPIQTPaCTQkR0UcPMjq5H64XWM3nqnwnosfTY43X2LHuoasl\\\\\\\/aCpBPnpgE\\\\\\\/ok\\\\\\\/QwoASST+EPCL8NktuVv\\\\\\\/lU\\\\\\\/E76xxHAaIaH8Eremh3K77lwuyqj6buzFhBSJj35Tm1pSpOzyvbs9\\\\\\\/9u+xr7f6V+6IKJ1WlKb3smr2Ls3CCzI+T8GtqTyiwuMBj99xJLvWXk3oCHDTR6P0ONsjN9j0L3PfKz5Fjp8RdabzkyTLRYhwuogBiMq7XpFxDcCF0MFAZ9IXxldEjcQ7zBAW3C27uHK45QRlmFzp4RZwX4nMQQ6PjGhw32kmfVznJzp5giWtRrLRS34\\\\\\\/HetXcr3p8H8A7nKZY7PE9LKpzIUkELMaqCmxQbvkFvP9RFuxPw+\\\\\\\/WGKo2knWmqdurBjH9dOEiyNVksQmRRHiY4Ln9G2ton\\\\\\\/Q\\\\\\\/1sEzQBJb5W9efhno9HP68E0UVZrDyj39dQJ88EhE6Mg\\\\\\\/DDbLBUbwF90A3bdCIneeAfyjNd+HqhWKjeTRxsY\\\\\\\/XX9a79\\\\\\\/SzKvPNMpNb2WaPx\\\\\\\/Cc4n0NGsDmVrEEyxOPYHh3KkVZBaMSWOZeyMe+nZ9RTHu5FvQBKizNDu7104FGmLTl0TIsm7i25Ct9fbMzWayPIaedfu+AQP0sEq3O+1VrMxhDcPkPrPQZsENiBQns+W2EGkjPq1kSFS4tm7Qbq7z\\\\\\\/jbPHklGfUgfdnIDdDTc2O6r2EC4cCmRtNutdjc1IL3Z\\\\\\\/6SzADeYwq2mTj+\\\\\\\/ZVZb7AQDu5tPDcSpH8+0ie9skyLGBk0wztwxNA2QvrgNSUrWMNmDC182JMFQRAZoVkI2gXOIcu6ByA+Ej8NM52fkvFqqFxKxbfAQQzOHXNu+bxhAE\\\\\\\/c8WgYQtEeI5v9FlSckbfkYZsdKDJ0POj7lsXixTHTgw+RnxeUvphxvnYQeEf8h1aiTrc1McmUj\\\\\\\/\\\\\\\/MYZNkahmX+sWWf3R+NUMfciMWTgwnLFqTzrDoFgqyAwA2H744bZyDcTVlfgxaKT6EK+OYJNAAOlZzhbC83ddxNkLinro1Yjqh9ulVxNmDLS1XHHQ4FBdcr28JQxkQh4KFKELqLMbjizCtgGCm44PQq4Ofmf1AKIGDDGpEXwNMhlG5CQxMiJ2iY\\\\\\\/9++0CyNTU4gk05G2trHX9kSUAW37KQ9chhTqAIWZ\\\\\\\/FdN3JB0kItD62zvQXy1HYYzcHKsHm3EVD4JvL\\\\\\\/wsjRApKEDfVHAY0vEtyH0IH2JiprwtLeSWFCd3Qnqj0790Cm6NbV8ogbkOhTHrgDbR3NTgdQqCstdl0X82TbDT\\\\\\\/kJldk1QwgLOOJ8BNKF80B\\\\\\\/OXt\\\\\\\/8PEGupu0zAEObOezYUhS0S0uYvAtlf4BtHej73qFuYJ\\\\\\\/2zDQt0xJP+a3iXxRSCFZJw8nPEhrOpWqNtCTgdcsbXaXO4FZYy0ydI6eoJikqF9V14aMsiO6cPzzg7ccvQ\\\\\\\/0fFyNR1n3pusU6RVhOtJEoPkoZaVWeuMuG38OAjHOp2mzNYwDSAurIeOxLhNWNREikqTTDWO3HVOOhX6qWIhTxJKmXdXhy0iA0etLSiOJ83qd+m3yORxU\\\\\\\/NzIFgBSQjiz51JSSl9mPS1P+nyHIjuh\\\\\\\/m16mWWVXCozH1SDgSGTRrV91w2yca\\\\\\\/TZsTYw61WzIJb10q6Jf+CHSf8dSyOKczz+ScgyHdyFmNUGLkjtCwTl2qD32RDDiDA8uhh07JaVZwmaXB4lBVAXLW1HuR8QoMmxnLl0RmsUO9MyZRd3yi27bwpl3s3fdra1goKQ54Wf2lbLBrsbM80ypnGXowtjdBn0PyUqC4H1AQEfjik3jLKbQ2Ci36ZhDkgXlXz9tOtjbDrjnxnaemts\\\\\\\/gvN8pkxkzpaTAtW1+PYe3\\\\\\\/b5C6DTKP6RbnGOZObpeMBB6owR+e2wKbVi+MumGV+wVneIS3ZZBlNT+lB9QLNnaFPRWsH1zP3qZw0VgElyV4sdiNeyQNxXpN5LKbwvRkSfwl3SCYLQGXRX\\\\\\\/ro1cb6QHnkuql\\\\\\\/wTiwrl8hV6a0DI3P5eSCMRBTKRgs00tj0TxxMpygdsGzN5dNMacNReBCLqorf\\\\\\\/ywCtkDoJ9qXSYQq+E4JleDLm8yk8wxC\\\\\\\/63rmb+eXlQnhDrFzFCvOxjk5SnEYUXqU7EjG8YJiqhs\\\\\\\/hqPsNkfW\\\\\\\/2ZrI7a2Hbiw4zVmKtK7yN1rO7bo0YcF+NUDm52GLez4mttWBC+ovlQ2q8x6X0pw6iFTT\\\\\\\/\\\\\\\/nA3fl1cV11RS0aQkZ6mcNX8RMSxqaTUNa\\\\\\\/SZ4Rd9TxFmM5nfZdSY482Qp4oUQrQKKv69IanxSmrjds9Nc5wwoBm4hGLNKlWhIKw4jn+r4mwsEVKZefOeiDZNBNWG9kpMJg5hoDGEfixxpfJ6V5Eq+4+EyycKoqs+SsX60wivNEZqh8pxLJs20Sl1xh9D6OGeTvDKxKMQ2LQpWUSkDdBOrFHPdKCcdpYF7m3vk+tSsmkippsOQepZc6gcdb8yp7I+eQxyqhvTMKIZxkgLwycJWk92ZaYesb4uYIcPfa5MdGZwRv0FFFp8BfbCqUVtDTbms0d3yUFFL2n+D103IXEe+udk7sm8RdFY6cXEBOF2PWwoR\\\\\\\/7DbGlcMP5S6PBjwEtT8qTQyWRdYwxYLWy4xj0OaJ4XPJMgWNgT9tkToOGcaLWmylofFC3AbIxxDeh0m4AJFbb3bAo3JQKHt18Pc2L3HDw0pTCcCRM59Bj7Alz5eSGRi2VQnYNF2ap6h\\\\\\\/Uv83MDEsj2+OKJbwAgy2DUozmudAgpM2TnjZzMMiCuv4gTtMTGwCsRhrxH9LrBVo6T2j\\\\\\\/UqmBtBYiX6Ti4EUK09IcDlTXG3ePYzRZ\\\\\\\/18num8cFs31YOJVeQxz5xB26uKdOjhOHeGUkFzHmMXQx3AEmioFUASsjSSnKa46hvmdS3yCHPCcZHdsCuOtFpkIj2FDSy6wqn1gBA5c9ic4u6CaRArYF5CxwOiwfnhUUHf+VznXUD10qSzP6pG2iEC8Re0TrcWXbgrqK+NI3G8YyGXNZA8XERY5aRoVw8m9DhDXLp9VA9dENfLNTpyWBmC1d1X+Lc\\\\\\\/LEVVwo1gM04yuaddFHpeXF6aAu9wmktLNG4x4xceeMQb74iVOCj+REuG9X7NWrGjWQHcS8uO5PksFfYCP\\\\\\\/OVmpPD56CmEXtLq\\\\\\\/z2t3Y1INv48dRG5Q94Wl7uxO1xOoqWLe3yMVTc0o77lBiNeApI1ZJnVNyYIfzw9i3k+CUHBoo39R18WuuUZRj3IBDfunk\\\\\\\/nIfQWWtgeYc8CkmsYjPElHd+IWxRxLa1Ac7r4tftsmEIOPZcm4ll8Y7m8QaThtv8bZlQhg0o8KWklgiZ8ht9Z\\\\\\\/CgSA8ee0OjA3Mn5NnbOPXrBWnuOiuAXywvr1QYeWIMSpEknfeushVoCDvD0iHbvaR2BE2N5+SmeD61k2XvxsrHXCYltHoo+egniCJdLNbOA\\\\\\\/6Sa2NKPqyaEPvEZBsw3SGT0DB5M7cLcp\\\\\\\/q6MS9WxSmeXOSshhuplAU3m+B1rPSNKoGZXUzoH0I4oZZSVFqKjFG4gBVgPc0n9PlSEOrlA95zc3IerSbd2sNPlQowZOlCfL6ZBZdqUg0UcpDzntf9VKU+RecrZHiXRkRnqTTh\\\\\\\/8OqS8aqSIaMRBeh0Ig1RqDP8MxCfLqX83mwFP+cyvdg5HBi\\\\\\\/j2hj6WmHv0suVRHe3JEJmBtOy2OxikoyEZKQgZwf1Osb7nhv7mET663m3j9dtLoz9Pzthc5opiy64fbHEU3H0rZehNoJk2wUurnvoJdEsEbBENRzfP2XYihsfsLB27xLPjZnko07QaLycvhjqeUQzeHRZ5iU\\\\\\\/16vteL0uT9Fv+RZJgi3DZ5Z6sQTR+RdJs9jcxVpjCiunI9XD+1gaDg6KPGE\\\\\\\/IMiqg+jTVkmAOg+J+OPRtsRq\\\\\\\/TPhFEhufpLG2OGPQa54ZzL+SRqIe\\\\\\\/GaIPH+JXC75LA2sFoNFdi\\\\\\\/kWuvCLKBwSMm4xWRF5UmqFKrwUxx7BdKhan9zqDy71Ex6lvgoOBZhyHtZl6Uv5ODg4qsCjRAeplKC2N3Q4BPOfb2D8gnO\\\\\\\/+286YXOqH6mEvjK53Jnkeyt5iGR+ZXxKFVTeYfQsmV4wDnwcx56sKaXIhbSYtlB+JxT1G0uYEkTYFUvqpAmCwAF3YlK+cy31PtdnFAWzgzFxhxFsMuq5F49zzosuwY\\\\\\\/\\\\\\\/Y1QJjn84lybtgqKKdAdkGurYjnXTbGauwhp8D7OKO3cXmHAdUvI4GOnXAXRT7tlUZ4TT9b+p4hkm1xL3DvjJUr3KkrQmBq5PXDGNIRjVdeVmi5YBGoeXOd0oQB27H8b9HwtVqUTw1cqE2ycCwbdOgBod63o\\\\\\\/OO8XdiKdK8nUqg3StRH1AtlkZ3zsrWgQlgvN3E\\\\\\\/Ru2hvCK5qhPl6lCeuN3orH9H9YZWpNBuWuFb6mRcu0LKKtszxeyfsskdCBgSOHGLvTniyA1nN6ZbEHH6CBHkUEoRblyHZtckVSeclTwZmanOmMv1tay1kHOA4im8JlCiTGJn9dX62LpHCKQLdSQQURbp\\\\\\\/MXgy2MjF7WvU5hO1p\\\\\\\/+Bso6DxclD3GZR3RUhxhXAKatX4fRnMygKhOE8iSAERf7Y8cK4SjFH65HmTnkDPI3DXlbvta+fHV2DiqVWhB1YLuNsJBB30OiFrE4BF3\\\\\\\/1fx+9otu5ag8kS3Cpq\\\\\\\/0Awk6p4mEAT3inxSjT3r7BRalSmzH4iFBFcok9JRz3JPe9RbYvUSxe9jMO5VYAbM\\\\\\\/x7tq3DLnWj6aDveoyQuKXLyVxk6E6OWbaCsUSTFsM84mMMg1m1fUBc0B8Il93Ye3OukatfBlsGcr9eIzPH65jpXMcFeImavxC6CrerkQwfW4sMsdFmTr6vbaw5vGUSlKAF\\\\\\\/UTqeKP4kYJbsXDvIliAY\\\\\\\/lpA7ZyWuvcUcCTX1anPcY5A1sZpiofclxE2RxF+iXAiUYR9kUhc14bL1xX0tyW19oYGzLU3Z34xrtuT+pZVwgwBV0I14iNIFFqTI2hDGDK+6XGz1mh1qmeAYt08AnsvbovcUwDi7Mfg45pedDDUPIX8LTKqjV\\\\\\\/wVdhXpFnuwy3lMwc5DlHKIHILUCDYni0pP87SD0x8uF7BmVOV9Oau51VZEHZuu5NeLlBeSF6hNbp7IpNXLmThKhvsFwnHbR3rQzXPD7qaHWGrkRD+\\\\\\\/Re5iHo8YvfTheP2hD51CkfV1zONdcnhk+GSdcuhwcOt6DznPc\\\\\\\/M1wih4p5eR73\\\\\\\/8dFowyZsVHlqeVgMa2Xbg0gZr0ljxv4kyhf6kIswa+Hf1SqPUXY6IIpPt63cJSxgm3s8k26HCYd8\\\\\\\/UuRS1tl74pnRHEz3oPR61ae2\\\\\\\/JwzsEuU6HGKWzf++cqgw+jmdNCxgNiEOoVqfwbmRejRyM85iOsvOjKfyFOZauXE\\\\\\\/yiaK0qfsJUlf2\\\\\\\/JTGOT4VGq+vNL4GcTKREo1wdFc2JzzlKVQ+Klnjn\\\\\\\/sQvn\\\\\\\/8DjuSPBomWiLnLNHuoWKw47RAdUyBOhO4J1awzkqBsKZl9Pv5Fzr1oiLItpwicTsxxQGaeLnFVAE+P3Jzv2LU5uS2Fv6R4A3YEF2C\\\\\\\/W853vRf\\\\\\\/170gm0LVeEgpuSF0SG59piIQQBMTHQQVDr4j7QpSqO+TerINimHpJkEiADPQcHeBw7ADX7IuWcWgx4+j2fEADEbO4OShEMKoBoYcKn82psh1MleK1rCs\\\\\\\/OvDI0BLMnd14hJDVFOAcbfB5lRFCzQBPSiCZBRUXfkD5ys6bj8uM60WdyhrBrXmM6sAsNb8lJwFCARs7WAM\\\\\\\/DJyvi\\\\\\\/5q42ublouQ0NNt6Qs5UHB5N+QxjPfRbnCY75Mf0d1lqMs4sl1scVxuptOTfU4tRMGXevMDMZK6W6TQYFsPf2y6gpUL8k0og+AqV2gId5Hr+WrBNGIx\\\\\\\/vUTVyhpF5cb8Zl4A8hWCz13t8qmCmWPaLKq6ttA3jjgDhpZ0Bof\\\\\\\/Hphifkf90Ykxh+WMRdIaM6klYqA63I5\\\\\\\/Y4d8VyClGQKpaY0Y8XgU8gJK\\\\\\\/mBkk5YdtRrwywXCJp0IyuRU9+Y8FTbbv6fBQq1x4qzPzbD\\\\\\\/6OyEUURZcEb3QsbHTROiwTo9vRVTV+l8H6BtywlPGNGRuzMQn94FTB+OkJri6zLhuLlsZIoFOW5KoCT1LA2M1nrflYEH4PosfrbL8rn\\\\\\\/Lm1wRYBz3iUxepsv1idGK2R3qH7yha\\\\\\\/j4\\\\\\\/ZbZ0CC9oPrpWi\\\\\\\/6QhMPE0sT6nluC31o0RTlQaDmlRiRO019iYz7wfAwSRuJ9KN\\\\\\\/3AOovROM5rrqGIJUPCXtzB5jyZhu2PuBzakNJ7zT6mP9VTuiPRqgGND5Atb3z2\\\\\\\/S7h4siDw4PoEC8CG3iIKfKtJhTbdE\\\\\\\/b9nkyQdOlImKRqPzxZ0VK4\\\\\\\/lGqppdF3th3HZVxEhN\\\\\\\/wKDq0zsWc8gz3+b+f\\\\\\\/\\\\\\\/m\\\\\\\/dtD9S5VW8+QHU4yBhXU4u3b4EwuS\\\\\\\/bVmtO3dBjvCsKA+nSTRChLDz9qEMHu8o1pTFfQuTzLyTNxDia+C645hh7+qwsGl+u\\\\\\\/CqTIyCAVcTRwvRBEKpDijhEvcVTaa8\\\\\\\/RG9qil8YLAOg+VFis42TLqZ42wvxAW3zoZlrn7429DWguQtQ\\\\\\\/qY2a6mVpb4S36dyxRm31GIYjzIM3xAToY0qf2tIx8PD6PorlVld3CaFNjsEPS2GkpUeYxZY0V6DLSwRKyqC5R\\\\\\\/LRUz1Qy8n+YTzh9\\\\\\\/\\\\\\\/N2CeO\\\\\\\/6IdPzmYEjkxJnAGo32udFEO+PohKPClQeh8tzzXfaxx3p6HXbUq5qeIAWLm6FgiFC8xgKq\\\\\\\/b\\\\\\\/hcCFQHMqmxXjYj8FsQzU2eUcWFf3ir92dlycVAERmvO+2EOLe7Ir9VudtcPxqqlBZzejL68aqVqAGgoGe6Hm0lEDEpS6LR7Av0TNvC0T+oYik1022pdLANnunRkWrVB770jt8GdGe+rdjB4ktkbl3gO87NXAts92J\\\\\\\/Q3eSufmzGU8g+sbtLkrQdzGnu5VYWuAMD5PbnCbJAmHYZNAu2+\\\\\\\/CeMwIF7UpJBjHUDAkTmL+N\\\\\\\/Ukwk6BgYun5m6dCCouJI78B23QvFeRN+PpRpvRrcUz6UtS77K9tBiliof1j8iemzAVaotelzCvJLYwggDZ91VCfPLI1T8z641qSAvJ8acs5BhvEcF7asQmL3phIIiHC0AtdCieyWD5BpZjqZ9wLrsYPtQmII\\\\\\\/7WKgGps2Sxhbku5fgS5h4J4C0P2\\\\\\\/WvFPTdobEpb6YA0f9h7iiDaL34Ho2Rk8FhQg9VbLLehsA0SQ8filcmsiGmj2\\\\\\\/YqU8ypiiCQDZZCZkwlSxY2XDa9RI3l6JRvrS0R5d7MVXTax26MuxLL63\\\\\\\/sChO4MlYhtiJY+CcC8U721k83ndNhQsJx\\\\\\\/JG+c7ZCLUrpa7+Umy8kktx4XifDp65PuyCl0aTzM\\\\\\\/F5W0+HGteeicWjxix1pnZvZZcIpvwFA\\\\\\\/PZntSzqBmAeaegGQDwh8PhwZRlnCMC3BuF0\\\\\\\/+YL6VJ0vHQR0T0TvW5PwBSkiniZ3xvyBc87E7FworNQoXt+e\\\\\\\/wVk8n7QVpdvAeA7nkQveNzDrfq2VQg3qIzKhQq7Edk8LOxX\\\\\\\/Q2aKOQ20ZM1a15inAV5CC54IBgK9+co2Jf9hVFD3\\\\\\\/UM15CkJtZcAY0VJDiuynbJOiKQa7ycCjro7mgdLzW0BG9b9ottFeg2ITtEN0gdOKp1BMDdroKOEKE3Ly29ZplS1wUU6HfRLwPlj6BCe7I3Lu4hRUM+3d1CR91ozJbOE+eunULqU1Zv6NzSov+\\\\\\\/njWwctvP7AaKpY115FKSvxzy23abjUA6umHft06IC5JdT+kccTp3Ffc6I\\\\\\\/wCbM0R3W0RPYS\\\\\\\/TG7yWE7KqVUbLfVmoo42x6DxiI41pUUCcrpcLL3kFWs2lyzrLyrCkFTKQYSCIDhaZH\\\\\\\/SEg+WlSXZZgl4dxV\\\\\\\/insZbol5xgF74x6iO3\\\\\\\/ICh9N5EMAx3Nng9Z23tcOSlMv46iENjuYQb9dHQjtV2UuC8Wd06EzEPfx2sPTve3W3A7hQPcS8MCWbnuFUWk5GDt4orkjxove1AAG810r8JpbvEIvgDoWfrvVGoXEv8ydT\\\\\\\/\\\\\\\/sTpY4ad8EXsJNPi9cmlhpPwzI8Qywrq2MpgPhwX0\\\\\\\/LgRe3qKjUVMgC7\\\\\\\/YcpYvpnA1BzGcRV6Pt0AxqrNTiesXAO\\\\\\\/bL2YAEU63HLhK3bkSdlgr\\\\\\\/BeVFXK2rYILuL8IZXIhAJPtDcVoKG3bFmOuRZNoxe1DnlIW1mnZJLq5aY+2CEOjHgIgQI8O\\\\\\\/dKk+QS3+XJzhtXNTw5wq5UDXL5fdNz44TdURfn9t+T02HxAT2S4JSE\\\\\\\/t6ZSjQMogP+t9GWSxVakQRVncVr6O4iAd72kB9aA4V2WbDwPJNavMIClb6QJiBT3o+QjfQYrM48yYJcSuYEumD4uxO1f4Y7TKjH0zsBN5lvz6B+NW8V+xswJlzDAITYhQMdah9IrEbHnEgrjhzBaCEFW56g5R84FZHTOc1sOFh73PQivzv8DQ03CXvjKtiR+DplN6I9Rc3j1E7YCv0GsbKC6OWXmJEyQS2lsdeT5YGDk0AhjWbNBNxXDZIBpAyHCvFYfjqxye70SYiC5odhPbhZAm8JygjjlbwlrV0p2863qh6qSbfOkLPHINhy6rFBiJ5JQYcvXqX97mdBtnbXuuUUhPzijeUrYyo2wY1\\\\\\\/NshGsfxSg2DeFm1U9G8\\\\\\\/IFmWcfrgiPds52w\\\\\\\/LUQ+sm4uOJM2GD0rkcJTLJkibp0WiLzrUoh99LgO9lQQzwimGgTzURFaEm5TysFWacliBuC7wsaUMbWhU5QugRvN0RToOYY7NBAAZnNvhMvDaLHIGvkDkZ9L1XoOJsFzmAsyB2aPyqFyIUMijuJP6ZPnSrKMeoZt010LDpnun2O6mDD3+lMqB6sL\\\\\\\/i2Ia0q004qpkKfNB54nO8NRfHN1\\\\\\\/zqjQh1obZ\\\\\\\/NAkAjhSDaaHJ\\\\\\\/Z4iqcsYtICXAFdtfwCzUUejUyG09P53p33GfKp0YHTYGM+2+elh+0eb4Bjr7hHyPVUJAlrSOl3WSAPlbn22jTI4rKwSvUcMQgTNt4Qrwg+sOQil1w4wz6kwM8XSeHqiyB11\\\\\\\/3PI6xvufDMwUq1bJm1mKMXyQ2o+uefWi9WF2u6Bz0VPHcCRsfTwnAnMYRvrZ7tIlOfmpNjhE4fJdkej0ocwtG2W+QRekkzZSYf13C2twItwXNDMXREgLa699gAgGqRKeKJBd9RwTn4jB52HlR6W16xc2i9RNWto0yx70ihzDbF3nNCDLhHFTwgykqyhJAeF09kvFyY8PoBt5mcgls1ZOGy9d3jtDF8YoYrNUKX+hb0BE2\\\\\\\/\\\\\\\/K0qQ\\\\\\\/7Y3aR7Ro6BgO8VGyNLmnKQg6NHywgVfJvtx1GXcSJmkx2GnTuOYNxGZ\\\\\\\/L4ZIAtbYKXU\\\\\\\/FskCnvbuwWM9eNbFPT9DEd48iGTGEXPaC+kHOAN4oqTvD2bZbuE+EBDQajdBJRSbOZLm8hnGYDnA8fBTId3yKcTi7TpIDZzEQUamRyfZoTJRTTw3taXSFS2TNS3WMRpkPLaoZlQlA5+nISL\\\\\\\/R2Db7B5XX0Pghu0kuOnAKRWjrXwmxJldC2qqGmhDslTjV+zZ1ec7L9xiwcbj3rmAAYKfKjxMzFHnbOgrXt3gf81hLYJXNWIQ28\\\\\\\/DDW1V80NwUYaMVZEHWb3B3W50xNSjH+01TmMUY6m130Y+UUmAPz7aN5UlELijSmog8Pq9knWNU+UhsvUl6uO68OX\\\\\\\/HX3cPljwQ3G+bjE2HCzwwSJU\\\\\\\/L4jHm868o+0hfnixXwrTBkOGYffQCA\\\\\\\/njnSSX0WvQai9oGuZryDs1adTN40B3o08eU2F8ZRxxrcOoPha\\\\\\\/ctiuPdJA603r3v2YE3iyKbKnZpJB7o4ck\\\\\\\/9UyGDDE\\\\\\\/nuVi03CM1YyMAXCIFmbSBWTukKNS03sr8JDA70395Asm2GeUpj+8xeBVmpl5g2GqEU\\\\\\\/05LOZjQrR4w+fmpgQdvADQMdfShauhdSOQTlUh\\\\\\\/1tr2WkH+yhR164\\\\\\\/lrGj7mqxZvH5HJuf8mdVXIEGeu+ZpHZ0Qxd8zwCHTMzDCDW9TFlNs3XADLFEUPeggsVI\\\\\\\/grlSjyHfHcXcXBHxqUt9o7+dAjm2uX5FUf3yKFwvgTFmL38+mTwuxO9nl1lmaSyWHOKfYBPbG0SzzBf55TiY39msThKKo\\\\\\\/mT44XLNl\\\\\\\/pddPNaLbQi3kFySs1WgTkYXEC\\\\\\\/Nam8t4vU9xYprt1CUL69sOHxkbj7OdrzNfnO2eoi7KSriNniTRAf4AHK4bWhCe3a6KNsXe7a0esYM\\\\\\\/ryL+1lHBgeXY5ObZQvEsgnGruK61rfaeWzdvj8bUGymQQzkbV2T\\\\\\\/+U9E8XPlsgpzA+ipanW7G3vnoSKGF6UGfG50Ya0ZhnN7OSnFTnNdl4SRJt+6K1jhAw7pEkqcCp8av5vfcVg+dyJX\\\\\\\/YA\\\\\\\/XmFQbPDTqynVOKG7UcwnjT1UTim4A\\\\\\\/R99fQkwruEOBmKHKhcrxULdiNakQVtDxNUWkh\\\\\\\/YfcJpntNxNWDhB70Ghv5DQUB0uWYJBPlgjSRfP8q8+Lk5nWoXY+vm5UGtgLbhT\\\\\\\/Pvx8s\\\\\\\/OjuY0nlaIzIv2X6QahPj9EpmrUZ\\\\\\\/3vD7YAmt3lQXOlnmUQ4KVrizfOTveDnmPS2zPqjqk+NpyvdKDOa+sNO92gbZfvbzdFvll8VmMAZvCIZZnHo8x0K4ENQREC5kXzQpeaYkwKqKTjQDox8DrrufytFAqYLgStX60QELdIowLq1nes1fXKA4Wdih9aqaFL+0CcneKzCN0K8Q6dfg1H6uwv4SZNvxdYVXdFusJGM+1CHtUvrIGeS0We3lLWoNte0q96MeHjCAK3ECziJPQpy3rAOpesxIzB+zcxwH7iktUFmve6sp7EmsplFhmCdVf4ZzsEruKf9ud27ihv1aCNfFFpd1KwFMxrw1eI6ontqhAJWMn5mav66ivIRWnQOmZZp3nMiCVXV1dNxE8mb2GhLWKeZyksV\\\\\\\/HP2KjUFBZmfiiRKxXI4xCmjcF091Qrb0XQOg88DkdRnXrby9wUbp8ziVfpys5OX4Al2Hc+0nR9Sghx9xrHvgRoiEtWfthiQtdeUdIQMygx5GMt7L4dcLkL\\\\\\\/LYF4hLccw8FkNMHYxH8LlV0D+RS2l7ufO+CpbR6\\\\\\\/62hFEU413SAZgC3S285KAnbgnpWG1G9RhcWzHSGuYsOgHzJyqWi5uXLlgKDOdyFhl2ev1IwDq+VMSpiO83gN2jwLS1OXD9Jv23IcV+BHhttgb69P7uWWlqkhfczivkWIiCagXnCWWnfXaXy+I91c8eRbTQ9JWxKbbBcw7e14vcqranNqlPST\\\\\\\/AeCw4PDliT0fzlmq4yoaW+IN4J7xPB4xrGTL0jPfba5KKt92Nqgn67mZQKXj3lZodcjPWhpiYFUDV231dFOKdZVYRAuKGN6zq\\\\\\\/Y7NFUeFZquVLHORPrz4BY5SyE7uSPFs1\\\\\\\/r9BqYHEm9wtieMjzOw6uR+zHW17KhWxnBMsI8GSgJCTU\\\\\\\/rCMpa98wZ0DMA1bO\\\\\\\/RQL9Mg6md1wyq4WxbA0c04G2TIaJ+JN9AVf31M7NRl30MxAIxAtsQZDlMmUgKEyDAuicEJBflo1EBzbrd1V04xAHiw24sWkxa6WMtW0JZYD+JS+g5AnVdD1o7SPeiRTi+1q12gAmRqc\\\\\\\/YhXnNoVE8a763NRcTZEBcQf5i\\\\\\\/Imc0VTtLeMFN13od3iJfB0pLcyOO7kj9iVmCFO2Yy8MRWZWOU7JcMMhRnR+7X6c90Cgkd+MbSFvoi5x2CZrZJ91FVY+PL+aEP5DbFJZb6ilP9XPyedK2cz9pR0bylSaf0g2Oz8i8KTIQxyXYZ0ZQ5ySs7+AA8rPVYYQ70ysYpA7G+U9TEmbk08Jx+zLLIidtThHZT1rnX4EweLEk\\\\\\\/tjgt1D48mo4xUTOfkAb9jALNPnr9jL92D7oy0PChdiKqWnQY\\\\\\\/DO0dcDD8DJwXKCAOF\\\\\\\/6niJfK\\\\\\\/pxPtrOPbK91wRJnxUrQpsfM1rLaDwdAx+3r+VgsPKWplhVKIiTCjmdPHLoXmMLwy+X4LsTPlSO+Fs8pK6WWjfsvrMdLxd\\\\\\\/cv8nAmMhjvDcgh23hkJCXpuoepWZ2YOGnnhkL15LECek6VjioL6PWt8a5NR0MjvFstEir8xCP\\\\\\\/3PI5CVf\\\\\\\/S6BpHjdnPtL99tKcyNBo1K4lq2Zv6HxeB0dZMrU5iCtLVzJxN8kEBRi9yMdn+SemWMIc+x5s78qlSMMVlpL0VDK6LIVj64UQ0dQ7cleyG0IltwIm0XkxxTBy7kBWjCOtuMPvt2C+pgEp7vU9sfzzslbcp3AWI3j+FdQbpIGx\\\\\\\/ba7Ie7eaTmZjEHi6DSLq50UZQRYXXTO0rKWP+ezEotoXgdHuW7lmTKc4pgvlmTIdkTXUS3IrEV\\\\\\\/XqoOeXOobkWPFgOCc45PaHG9ZQmQwyD1X7AFLVRGPQ\\\\\\\/TqhgRRWzRpP9QJn0zulJ7B3oVLZifOD4hUUzRD4k4tN3q5PIBQVf7ttF+M6DUdB8vSlazF6Ue+5LI0WbhXrL6cEEFfI2arN3POFyY\\\\\\\/8pi84yzV2Kbk+CltyO\\\\\\\/ZqKzDv8hhI8i0\\\\\\\/4K+K3u8JX97ixxeORktTWnnCrs\\\\\\\/0DOwWJyR51gFn6hghvI+yufWCe4DdOcPWPsWaipdNnJTDtMxcUFgDDy98Ab5Vee4ICNrWU\\\\\\\/fkqqnZXZ7bb9O4v0IQPOwZFzxrWimVpc2tRsJJdLBXvbh5yw1EFCweycV7e9Rj8CxMUkjqRXBUCUG9MvipSH\\\\\\\/xf8dxWZ+z\\\\\\\/chND9vLvWHnDRd\\\\\\\/b1SPrtxmT4VewwKOBXrLQImKs9INLhNeKPHYJXbs26yV\\\\\\\/Nzk\\\\\\\/VP6X0dFUSC\\\\\\\/DVgWDG+g3fQR+CjtKYFH\\\\\\\/Q9DUUWF0M79A8mlqDXNsZ+T4w1RUlXq\\\\\\\/8bZsB0ss4SLdylha08nFnd7+pWUshoqoZDW8B5W68DB5vqwbXsA\\\\\\\/0UzMnYT2tu6crn1vRDatVlEAq3iaAJQWme25GlMcB+N8KnrFrYbzHiyw1m8KNybJk3LXwmYP4HJcKO6EOG4aFMPq\\\\\\\/qeRCfcPvHYHNDaYOL+JzpA\\\\\\\/P40hV3jpO5L7dk+QcSRYbIEOGCrgfFhhn87gqMdzjfZTz+Ts3IUjLtUkafPBeFkD+xq1vKyxzKLADp+qN6qh+JBFfKN7jG5qUdg7X3LP5HCldcAa0Ol1pD4EVCR3QiLpMaDGJxlb5NNcWGOGTnlQKdMh60v07CkP8RNOLi6L5DfyJAAVbEcueFpQdTeb9hKa+r7iLXL6mYfFptwi0yBdPR0dk0PoaW+Ae3tDBnUxrCAlSDnAYobm6NkuzQ3yidTujcTSYdJ05AjBKvXHFR6PgdZ1iT8DjVp1iU+O29wJSdaJdWYgWCZIGGKA3J6LVB+cZiLkTBEq\\\\\\\/x2xfmyFwbvQD1pZPpAzhLBpdb6jvUL6ZwtRRhcFHKu6lkNNLeRLkmJ9UmubGIt0yYers8mlt3verlJ7nHZcb+LP\\\\\\\/ItDXS\\\\\\\/s0NNT+XQXxvlFPMZOnWMAasc0ylLzpVAL7iFdIi50IIDs6pg7nl3cw2bwHUkzKsRJmfP6QNMKekdqD0qj314qxm0vBb6N+IXvYGPecXs0pIy531pnq4\\\\\\\/ySqDPx3Q8hgjpDVbS9j1J7gSPQoaRUy8Ee7MXgRkDkDNAbZ7eriJIovznpPsKPW4s8LCph2tGeZqoL6MVmn7dYThc3RO161TjA2xi07mnadE\\\\\\\/ytFGtaO\\\\\\\/bpCkZrbYC9pAWF\\\\\\\/4I8cPQ5SlJjV5Q42OPuut+Bajulo4ICXO0rb+Qg6J4g0OCa0CfPuqp7dHDb5+3dms6nhj2BcXm+boGVQ+tQlx5LmOhnFtUY3gWfOQC+NbvXdiHYF23ocqyz2I1j4qqb30X4LSnJtSLWF5zKSTrDSPjqAA5y8RXcpcZjAwxGvOGyFR3IZ9vszmGvXvOYPJr7x3m4qnNFmrTG5n2K0tdvL0YBDr9BJHbzWm7a7LrI0YoEPRAER3aJtuIU33zaTwY+mVVFq9ShAbfGOAHZ2fa324UpBDIU\\\\\\\/BTHLaXPTowL92VPa9De2cpM8HmN+dx7XRAaJUPn64Uq+sENOjU0vmbozobLOGBb6dRVDHCU9ma88BluPyTMw8evMKIyGXpUEI6cvrg8x9WbqZy64obWGkSj6SQ0oZpEXezmZ1eZExBckBhkejxZXapPXJBYHWOO\\\\\\\/eGYHtKQa0IKDIkB6Rmhzg\\\\\\\/Y+GVHBs8xbkcolDGT7+XoLz2BU6I5ZYz6jOChww6miNGyicR\\\\\\\/O22nG2qp0q2Sipct8g2HRqY4kLuFa3VFJvszb8D4pdudaqM7QOVuU3DR8LlgOoLDu\\\\\\\/wLpPUEL9Dkz4v2tH+gLEHm+O5hnxxbG8Ff5E0x39xftZ2RFCUhJXVlN+DQYRUm2vkzK8RjXOByswoBOg6j8oGor6ZLBqaqK97VtQizHQ7V+ng+RNnBhaZBk4QFHchSfG8Akf1qQKN8ZiiQynoqGhztTfNDkGZyyD+2hMQnw98aR4JBSw6sJfAXXu\\\\\\\/hhqTYq56083yhIRGRu\\\\\\\/CA6Zlog37G0PUmT5NxpLwL1qnram8puhYnro1W1gELHcepOg0BrwC9LupZ+oireAFWLu87phHfeBRlnGbZvDZV2z1zru9vIQc2HhA9U3EbDKNEqUIvL1wZES7o0xfxe4G5oi9RSh+ujLiak6Ly3Gc+utrGMQJVrtXuScB1Gz1v0nFFewl7b1FOU9V+Hpl4xX\\\\\\\/XKctfkBVDy1ymeMrOtGq9lzLhkiOuqDajq9jz0jpccsUBCoMMyPEVyPfjYVGrbzUDkO2evJ\\\\\\\/I2lnDoXBiovXfRp0K9qdG7J+GcR9gx1XXJ4B3J9DhRVsj31JE\\\\\\\/R0Tnkn5Dms9xkKYcb0w8+uNhvnMVqsI7nWchVmF67U79OnxQDRTyBxM9AvCOONavguvbxEMgJf7Ie1GLnjba9gkME2iB7uavMiZd3ejtgeimeI22oZMMy3rWDGwtiEnCM8E9ToPCnHWnP+VLcn3yhW\\\\\\\/BORvtR0pDbERWH+ufM51z1n\\\\\\\/+biOyfZYnAYhKreyXfBVzUaNiJve0y4WFH4ijQgiTrT1qy8FtWhevSizke10hR4I5kJefx+wJiIZZ0I5+xlquJo0Nd7TewUjHUGkEIxSWRBnK+bmmI+OZNBjtMwebfzzWZMTSwxvfCpRkxASKvU7SfHAh7da+ESZcTWZloR1lFIe2GCK2ZOUPI\\\\\\\/gbR\\\\\\\/UorosU7xmYUgJpH5IszppvbhXJGyTRPpMuCF2q\\\\\\\/u2nTGFZSHycdrykddr8UjxSsj2ULO5O0\\\\\\\/dU7KTmKvBQfyZ07LeVMSbaJFSJkPbzhQIL4iD7oZiwol9+IG6A3ZfNzDdTSrdAY+kTjbEo4uYw7DzfwbRteI9IfYSaT581AOMt3lyAfLWmPu+4WyHlqig\\\\\\\/7NiBuUyZEEIMVllJ2fvG\\\\\\\/lKnIn9TdVHyyFdfCek+7d3XwIDSpx5GdQERbX7hHWcaz15Cjr1tP5uNrLKJPHj1uAjYTruIdDg91rIWmsO5rE8\\\\\\\/MiJAyBXEnvlL11SyXpIHlZYQne1SVYlGwDnQreMCbV8EkGw85n9ldYklVCb\\\\\\\/qIQjU+gRcZjnLUZPKr7KZBpUQECXD9OgYY\\\\\\\/D++TADt23BM9niJ57CPieAEJUU1mxaEjzLYQXlPY7Yp+WZHokcqXRB2GAJ9HsuOuVKkaR+CcjMnymIjWX43trmziV63Xz7891+AlVgq6TtGSrXvR1Yf6Obz33RZOS51NwbxZV6qXY\\\\\\\/XC\\\\\\\/EZtzRD7iawVtrBT97qLeFv\\\\\\\/6Yq7ttKuS0Gd1raDDXS9nFYmN4ogKuRXTtQs00F+B8qmq3x8CAzIp9GfaYuf5r3PAYbETE7zj8fkRuPHEkfVpfX1B\\\\\\\/JreGgf3QcAbevEUpjEj0YKQXSKF2QVPs5siGiLLx1lt+PJP6yn0qbdSN0KymxhLkKK4MKzIRodCTqfz14vc2oHIFX+FX01PUs\\\\\\\/G0+4Rdp8OoB+cBWeaOwkHi5M5t9Se5mRSWEQ+4YVici3Hw3cOOVn6KLLc7AcHvhGpRs0DGp5bXZLn+XmI2HslgvMQTTUoTEKEy1ywsdpdYO9A8Yk0ZNzjSYX+HojYeuySE6ctHr4qrZUPRj1ezNV26rySAbwZCSkZiJLfLEmdh7SQW0x7Hha1BoUpg7\\\\\\\/Iql7k0X+78WsSSckcoZibvHISYJrN+6nsfQ\\\\\\\/lE1G1zu0gwWg+mVsRdFvPaVjWQBIP8uot+t5kIQ\\\\\\\/\\\\\\\/87+02W8s9t95ftN34L31vsXBxjwZ93bnaPTmpyHBwbLcffERHIALTXmUpvp5Gtgt\\\\\\\/mvR8jahqWR3hQNe9zXy6Swtm+A0p4aHiqMkjYflCxRLqz0nX7Mg\\\\\\\/ApFtXOaqLBmmVP+y2wW23jbS7i16mCxvL8K0+YXUGA6xPy0tt\\\\\\\/jTIu9oTpleS1rUZYei1m8Sz+RGkof4q+zaEG6SoJnJPrD80b8QTGoeoe5Png\\\\\\\/7b8YA6r4tpuqJJ3AJjXcx0PS0FvHxbthMNIIovLVXEqUOJDRRD3EGqnG9NuAOE+G+xZov1GTmTR9yOxpGUOSOzedaTMiHu\\\\\\\/2rGMVFzKNgaEaWOIang0V2s7EaasX6IBfhjo1gJBz9uMwDxzsGMYUXJHmBZIsb4wGPsLHtNAanl1U\\\\\\\/6ZgXMgCSKbu7YJfrbYVsAMHRzOhrmhH5tEhZc7M1i\\\\\\\/abxKE10HrVcvrCWahI7sAfu02Yl3yjzxjUV92hE+\\\\\\\/R+ixjh5vwAOiEJyrZviDJlU\\\\\\\/+FZyLkGMXHEnszCK5OUdBydKS7bYOdIdPGTiOAGuoaGifxbic\\\\\\\/4XgJaqSAUaOTI11MzoLGmfusL+yREDaVrP7BmkJM\\\\\\\/aFwdrvolWP5UEYFRsi\\\\\\\/n5KU3GtHBhWrQdVjvYDZoyyR+5CxnhFE6NHMNJHkwwnDGuoG6uiF9kst2se52pDQwxJqqhtTqQhZI+ORzGC5NS1apn3lvzUY8ewYRaup8bgbtNVxTeSacDUocjopmtccbfy+TjVZVRJRxuwh5xzzOUa5QaTM+xaUY+ppsyKSfOHiCgcpUtR9eGvdMiSojzObiSAZRPU2uWDslxl\\\\\\\/5JeGBlemUsesz+Gxk6d5x4II7DwfIgt1c6X64goIun9jO9gzkbQyt3mzMdGPmnAeRmv0jKh9jJOnItwrQ\\\\\\\/xAlvFIzKn1eh2zvGpqPf7mj1HN9UcbPIo52DTTn4xfu2crKQa4QEr10aHJT+r37kgZF8GcQc+LzpUWywWeNQl3Ul5XssdLi1ymTBr+7QqwtOXnxe1CMRUnR6PUAV6\\\\\\\/jHMjPOo4rrMlSYQ\\\\\\\/d+q5kPo9M7P+10wIXXXFRlI0Jm\\\\\\\/7DW6RFWJBN1lTJPK0Db3H4KTzTUbNgbfzKhj+xRAVEsCRytTdEYYYFZ\\\\\\\/DcgbyZsx244iJZStnlZSv0zdjtZmCH39+X4ZjPdugRjt+roBWOfoj4RnDQfwjtRZqeI5jkL8FKjbk8ZATC63eulLEcuqE8Fkp2C1pA70dBw18aY6wOkIQW0HWKR3s8hafk6RUZ56cZYZudyOn6saTo0\\\\\\\/N2hbh7+NKvxstHjmAGmcB6XgG5zZ1bnLB8fWMg29VMJ7894l\\\\\\\/jon3\\\\\\\/7mWgBqTcDiuge28gqq6Pi+\\\\\\\/7z0TeeNTbYxxHH+0Us1mnXkbrMnq4EM63n6JCzUIftYMdYyzjt+8HLSctYai8SwWYufwmeYXuzyIv+iPJGVB37DMpttI6T44i3d1ry1C5rQwt72yTRG8PRIY10VG+0HJPVaUcCuoJElV+czuqqWrUGCXZaekJSW\\\\\\\/AK68a1\\\\\\\/kC5mDtWsS05vRzS4VJn0gH1C0MIji9u4UWKxlqLH7OroP\\\\\\\/gvQpfiPpWH1WIwEje8eAnEPAO\\\\\\\/ufdhoPyVCTOWS+F7We+Ja7e9eKTidp26NHBKZJY0nylV9o2MeHIfXBKKyWxzwe3ge4kp3w16E2WueUzPwkRXkJmR2CvKFivATnlUeNGmrjOhWYqUK8vFbCkamZdKqpyXNQ22PApRGn6M5Mrvoq4bevBVytPmXCo1d04KPcguBAxMQfl8JfqPIDfAJBOSCFl9X+kpGOqvT8iy4u3f6jERBe2mmljwTopL5wy+2jMGHHDYbp5AfcPXkBWi3aEBACKjf96MriUlRbpFF2yb9+08MM6Ri0mQB\\\\\\\/KWaW8wzA3YqWwto9tJMGChIYW2r\\\\\\\/yyU\\\\\\\/i3dzcXiD5p8pDOCsvPemfxOlGFok5bxDzd6wIYD3ONwsEtPTcZG\\\\\\\/ZZEJPI42LEIjGbkXfDmWr5Dwr+aFGZS+XcbK093q+du7xRj5HEHaGSMBKmZgi14qHvXD8X96YL0W+L8B9PWGKqvcRumw8D4mbA23qoP7FX9wDnqo2Mg3bXVUU6YY2g2ZN8NX\\\\\\\/a74iOcetsOQTZa2ga3RiYUUlmDmZyRYb+3GjIDDr\\\\\\\/Ok5JBKgMVgBPArn2zzp\\\\\\\/IjWfBJG8tIVnSPowr\\\\\\\/T4vz9L+0wzIxb65JMIQCSd1C91w2YHouJ4fA3g+5tH91btn0wyA4laEnRsHz\\\\\\\/VtGlVXhx3yFfyyRPtuRt\\\\\\\/tGVBgi3LJrFIQ9ScKfF6OU57PJ\\\\\\\/gUG7C9V1yHyuFnK5qtr8HgqkSso9cWVOs9uNdF1RG\\\\\\\/AUbQUQXQ3djeX\\\\\\\/mArTNYISrYGGNnep8ffdqsz+2l4F8yuBon33ZPDnMPlT7YIaSc2afmY5jF7b1b0znfuWPVBrkerxXZr3NPZCSWC2PSOvNbfhwTydNzPAkkv2cwYUS3bVxWdMwXyHa+HSTrFM\\\\\\\/NUw1+ybfRc4wt8YijN1lfHCmrQNfkE95ykEx2Uu875koTqyYVt9Tswf7F\\\\\\\/CMjuigCJDCEuy+n3eOCniAFci4YTZE\\\\\\\/eNZpodJmUBT5ITgp\\\\\\\/jUXvpmzeIEnde+VoHF22C6nmC9I1iRgLqbitEmce0tHw2xASygzadLGNHPZDy6p+Hd7jitLf55ngl\\\\\\\/JM+Mo3gCo6X791\\\\\\\/hx6zTdHAOfTuO+osSDURFuLebEI8mlVZe8z8nEi\\\\\\\/lAHZaYj8KsO1NofLRSrGFm1Y4T9v4aCFjp5nG5QNERmfFCkEgl1ExgPJ9vDvS\\\\\\\/IN2UtCjTKlrDJLoL\\\\\\\/1DmL1ImnMA0riH0+v9z93gTfw01T1\\\\\\\/8ij6Vsz9p2uwMDkiAlY34kslIiOGQk0YPGMvAyWUREqpA7jgxibEPtd5i0ReWEQLAutOiI9c\\\\\\\/4U6hk1SxZ7SfgEy4q9q8QA+O\\\\\\\/+oD9uhDCFaOnWhQIf6R4TuulylLni5Bkh4BT66ZyVEE7Sgn7cIhv\\\\\\\/TSx\\\\\\\/9CdfisLgqDgRtD4ZMJJq+p8Ul34iN+Hqy2cXGzxDrdy\\\\\\\/hXTbniJzvwXGTKBqCoz3VDei4ntsZmrOOI3ELjMtX6agrJBauTFWLzGxRKWz\\\\\\\/I\\\\\\\/sS0OtbAJTNsfOO9OtPTyh2cYl19WZYZARWjicTowEjXDFaBpMBqT+AhAgG0D6mZpdIM47Ht0Pf6\\\\\\\/EJKZ1vMQ0yk79k\\\\\\\/WJJnj7eyYMKnu2yOQxHTV0\\\\\\\/9wKPkOD\\\\\\\/yFalK1ELg4f+AoZHZSDMtQPf5ClYFhqX6sPy4YmJwzW6kDzbGdBrI010tLL0sOIq2vwaVUln+UoY0hU1fYWnIGOlNzjGfAQ\\\\\\\/TauEWKLSPp\\\\\\\/071gUZU1EikGN7vU0Hp+jywCSPL0rRWUxfZPn7kTh4onGe9izj2YXJm0ps63PzjmN+XnxVjTp04g8aFfQOVNe4WRu6WVnrGCjKLNqGTnzM1pJjZ+Zy7luE3ot3oMxawsYXTTXrlhp+2thmVahIHBpkAbRWP55l5XaQ2nGAcesuUtxql7249cZ3YmW2k80poKgpF99Mivc0K\\\\\\\/ORpFWcvSrSA3seMnWGIpVjHToH4qieg1axiV29S2VyEgp6J7XcYqbP2DwSrF4\\\\\\\/6mqj5rBwM1xoEn\\\\\\\/t\\\\\\\/rGW6Op5pVlrHRv4\\\\\\\/Tid3ck64aRl+7YL4N\\\\\\\/Uzmw3r5ybW5BBw5dBtRnqZ6h0ZLcJHXQzKEGkwakuA3SSKdBls87QX6l1enhHi5F3fZvCpsOpU5Svvtrg5iKCbqTp8bYIF2t8DB1ji6JjcJk81VTN4CnoN9cIvW\\\\\\\/QiJUGsbbvrvws\\\\\\\/FRlk30lRpsM9jtFaacUVbm977zoheEylhYotk\\\\\\\/vWWUlE3pj3hOJVQ4jStMYZIP2y\\\\\\\/PHChfK1+xy\\\\\\\/VUE2P7Z452EXlnrAW+Q1aKA9uAgu9bcGgruQJkhrUrQ7F6Bndk5eX8FIFgHh90O2ppv1JX3OJrwBduTUF9x24NDMkBEZEwPLoQfQsZkxFtTENG0YQb1WMDFTrGQUKQyNmQnmFsVFsj8WaeGhaRtNzLLjtvzx7hOGMnia1X6NDPgltdGYKLCM2sKZ6vbDnUEI9ilc4eg7MSSLifo\\\\\\\/fPIwrLbQeN5B+UzXocl21RJ8OF5kjVk0y6\\\\\\\/yHlSh5p\\\\\\\/YSpyvJjqYo06yaOjthvbtfJR3UwjjjtOHsjMGhshpcUVaoiMFAj6zpEqS1HULmgT99Jxz4WPm+MDczBDF8V92zag6FAgFQL6RvQhcapZJJ+hjM2f3V46EY\\\\\\\/Z9sho66tBzFDzchg3n4Lk9W6R4JdAbalc+7+QbRGJAGrgdBQNJGmvrZFDVenDxIOj5Q1Cq3L3qcpBaG0F5\\\\\\\/IXzvhXyazar36TBh2UE3VmhUzfEpdO7HWAu6nausC\\\\\\\/Zzn4u3TiSMathdSSlVlu42SAXPqCAyQvOPy9ZxeSCwDQOwPg3QA84xpvwho4q2ZqWEuy01kVKuMh01vSLdnYyIKbb9UYlrrBi2Pw7Q6m+UXNMhe8oFM+VBmvd7LHLqFgxUSL3+EQLTiM6XMYXU\\\\\\\/6uTFZVjbrDN426wMTQgHM0MAoykge89X318YaoBWvY5KkayM5u\\\\\\\/hneTXTccSkZ\\\\\\\/I9O1HrH6VOUfrxo+CFMSDHSpzvMWo7SdZGzhrpIsYCbrv1JRlTJskLP5UwM91IynZRr4AujkQ+EDXQ9t0O3+Mds5Dt2dGMeGTeLYJ85dPjzeXBsTZVj\\\\\\\/Xjp\\\\\\\/WUOiXz0BTfI42vNw8ERjAFLRGNtSHW4aQor2NemB1XcDXHjwOqikENKsAjX7eknCS0FyuHSt184UAC8lY2Dh2DV4G8PwuQdUtPgJ2MkkQfT0iNsMEwXnVVfNnh5YNEid9+FpbAh\\\\\\\/TCGjAQOhzUlEGnculnSHaCYy19Qis4QNfevLK9kAwT1j5KTjUTE+etcLvWLKJk7WiejsytUDshPG+4chdAHRGMw+dMv4fcJ163N9ykvhsHMDkHclx2DpHqFkSWaIDqF5E0ymqTrDeD4TLcr7NRpFXKOFl7TKgf00j2EzcFFNOWkLcgIlaWiXSR5b2QR1Fmy348BYdfQJ9xXJABd+54rAXhXE4Yf81ZJNwWC2xvKVtw4LTmNp6pMyjsKtJjGYN0fsRn9tii8Ag4i1txROMt\\\\\\\/Wlton4j0TWeRE0vX77kN9TikZdJExmZp3jYW8C3CxIANdhXfWHFzBRKUkAdiH2gZo0hRaqy0svAdI1oar1PGPlnGVptg6wOIrORUl1jp5LqVNqNNTIW3LUKP7eD8IVzXosa7XAiQJbNACrzk1xaWpTgvsVyV7qARIXd6WjgAzJqloKtTZU3kMw1DuiRo1AiuhdGe5FD9Jt9WYT3dQi7meM7dfrN35Tf3rZ2hwtjfkVa9LTS9KbF5Yq1UCsThkWlge9RDkr19H2z6trcV\\\\\\\/0gNzhtDAMBEXV2uTifIGquqWHttjx+S4r1viESzIB6kDyVt2d7Lw8F2QWptsjTLdEPcyyyOraU+1oQMnyjkOE4yqUhE53lCLITcsIdKw7oLo21Lq3yHrLRodf3zhOiq3W80JJ+V3+VL7+vxTMS1lGUo2hV5KOH7WZsdXcAZ\\\\\\\/S1c8aVtbDvbm98k4ziOHQ+zQYNO9dTF4gWL\\\\\\\/oKYKyQ8Gq2PbuU+ERSy0TRka+C3vUoBMxc2GT8wMXjiTxxAZ6nyLoYdPA0ct3o4LrPTpw6VctclaV8OhWh7xotx4HBwFU5yVqfkAMe67vyFI\\\\\\\/MPbFzH9Dcuu1W5fBbbQ5cqT9i2X1Mw+WzptSacsURD8EszVusda6d2YTwbhqlewu6Ir4mheLCk+K00xQEQOUtz0Pyv9r99O1G0JT03P8s9qWU51dRRXu6XGTcwQKiG9Fw\\\\\\\/jvJC5\\\\\\\/NUwgGCK\\\\\\\/pu61Lp1O\\\\\\\/Gy932URhp\\\\\\\/p9M\\\\\\\/9Io05vNmPvHu3Zi4tRbJLZOmtmG4YcM5ib1qLz7TkRdFzURG6ZYELcDfJXdbTt6EpKOYBMV\\\\\\\/IWP7Bicb1OGlRL9NZsrA3JVIVgybxqu3LmOhxg8m+1ix\\\\\\\/c4KhZCQbndGnd\\\\\\\/7a6Lh6roFUKpLWNiBAIk4yObuqM0CuEdtX41CStv4icMsJ7WDGBvD7sJjtis\\\\\\\/VXNLA1BG8ZtOYy1aKCenHhA6gTRQaVD01gOQe3s+htmPOcNSLcGDwaJ5DdPE4bXTj9925Agmv4oFSRsg+zEm+iUPjy2J03jJe44rM0nmpSp\\\\\\\/S5FQ1ONRj2LRWwTbxK6HCfKNP3+82o3yxVkODAteeHpxjppKHiY1wm7Vyx0F7npcDn3zYOkBFlM1tKoXqhR3DsCZSjiV3nVe69BnetzAECkYFAc3Ie8CvztoxaU02Fr2KpXL9Rwkr83Rg7aGGsOqsoE\\\\\\\/x\\\\\\\/4eaf4c3P10nfowtBk3vu5BveLUxRtYMy1r3U2ZBaZ2dka8ej\\\\\\\/+cC03gYLeTFi1mCJJ4hh3ZazB88jcbBTZTeUi\\\\\\\/n3nkmy2c\\\\\\\/UjKhJfj8kHHxTvzrvHPAopotgsjHHv3AY3orZF8pquTU3XfqMBshSJYDC7tkqBMP0yQwyhIn+REyIh7mhuBuqBfIaJPsTqQlIjLfKTeHajXe4FKXA9KEYtndRfP0hnF40NCicS7QXLYTTSN\\\\\\\/0AjOXY+NQsprxTgWpyGnKXmmAPiXTL7LvNkvD4LZd8pXiT4xQ55kktXdqMm5hzsFUK+MIIfKz7QKa0fR6a9qTKV7NLYN6\\\\\\\/TVx+UARGcOY2zVdvboddfW\\\\\\\/XxndvhNbfFpjDirsaf4H9MUfkCjr4rVWlmu8ZZd\\\\\\\/V9vzHAX+qyyeCuJvcLzPgrL8s4Q640EAUHlC+hxzrhkbH+CwAlUv9N6HLXxi8JixHPKpbUQtdE4dERLT4J5o23a5pqk4ze6lEty9XNZZnGtXYO3xEtBFgxcmMeVFaKp28KUla\\\\\\\/nCL+8+InfOVNKQKDW70170UaA5UscfSnpHG6aIxlftdUoHKI0seJYcpbo2k5HZ735PJUBlgbZBU+qALe60e73ePeaKLzWJqm6Q4WLzHYZSSICC10yBb61JizaFeO0A\\\\\\\/BDHSsU2u+WCA3qjhIOPZFJfWb4tkzUrFlF7Kf37Mc6t9acm7EYIVeSf\\\\\\\/dS1x6NZSSO51XJMRDKTqgkSmJf5\\\\\\\/JWMfdltX0+C2f8hu8oMOLElIZTR\\\\\\\/NZgScI0V+m8ch70Q+ayGDpHV406kSqQqitMfgLA2YVgTjNM7BU5A==\\\u0022,\\\u0022iv\\\u0022:\\\u00222tdF4QUtM1AUBwyIs\\\\\\\/xEQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229123f782235cbfbb7888d9ab76815ca48340efcbbe0b8a0bed80cd72ac8321db\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ih3BwM5j6Pv3Q0WO8x4wxvu3CEtyNvp1JelI5ng6Ma+Blj\\\\\\\/NoyDjEzUn9E1obTJ07WOfB46ydNjugwHWmA0dVMiekhcixOJfopLme5VgxwTSphHJekK9PuODBNJMRox165\\\\\\\/UK4Zq+jssj\\\\\\\/1p7UcDjHYby2qFprnykG5EEQ69PTrbpOgJ5dv4Nl9NqCN2gDIKjVT7DkAKNd3vj59pEdryFxO0EsKYyA88GtMYMD+eZXQ2rFAGyuLJhhrJIqKxBKw9E6h+8bGB3gfFj5rJeUjjcIvtQhokBv4\\\\\\\/jwuKJDI5+VwVdB3ZeisaR0nty9O7zpB0aByZnCZkOJxOv6hiM5NdSEW1yQKaXZ9wrPJMd3lDjZ95xNbzhVTGq9cSC3G2LKSVbpcLBR7k94rm3sknqOndmeW0Tv8nfmlrWDggUb2TXxHd\\\\\\\/H27iRevqA0DslDWSN9rar4rIty+wDEVg6eIf5yu2wV6ZokljVlf3fFVHICfM3iXpDT3xavJLtprLMDRiGha2ZbwNQDMU9\\\\\\\/lkYI1\\\\\\\/g9e6q9icNV1e\\\\\\\/CRM\\\\\\\/1S\\\\\\\/z89nf\\\\\\\/nRkT3WyGe+U+vru1kvgbZeWMlhWk3ndEGc61Mxf4dzTzKN4TaoSxiADHaoss+JO9Bq5MIcJq4GHmlsz1VJ70P\\\\\\\/ELg5iujx0TO5cya2MH85eWNEzf7kgubXAGcqL5AVIoiAvbDYkZHi2pFaKHI5Wf1cz1I+ZtQTdOqUpoe5v0gUBHq+B4RZPcRW\\\\\\\/2znOp7SEAwGFOMK5BHJTQ9Q\\\\\\\/io9tB0QiedmFIY0hsTDsETrwAoS5R5U14D62YgkiW9UJWSIzYyJum\\\\\\\/FAdk21feMg\\\\\\\/Yp5\\\\\\\/3effeEIi5blBb0omJNBu\\\\\\\/MKe1lc0KOLYpUp98r0kxmtSirW1Jt3LTWvIMOQl4OkXAEc1VRAh6Q16hAjbjO23g6NJc6CBU6tivSCdK0lpMQH6ScC7D\\\\\\\/3FKTUrpR8Hq0Ua3txSpLxfTyk\\\\\\\/+8dMH4SQ5QmYmasSdndC\\\\\\\/b1YOJeiIgWLLwXSE1aGW7fqr23gjz19sLmbPt+0ekHcLFUW6wBWAnb1MKMjWAi5\\\\\\\/\\\\\\\/tsOBy4RHEChLHj9NCUWTOqyla9EztA45t93+UCMDaWPghEm714GbC2g1jMXGnNKRby0kXH0pNQp4uTQnC3iqzwRTdZ1f5xQ\\\\\\\/HSJMKDw5zrBW46awxzVC4XFjQMj08TGndwC5zg1BqW65MikpTIkkrln2VyOQ\\\\\\\/1t18cGK\\\\\\\/PKpT8OZWy6\\\\\\\/UpSF200lzd4McfxSizjW03OBXAOFXo7YEOYCbSPtvQGAEK3ANq5HCnga1L47iozif7Kkd\\\\\\\/ucBwSF82686TbY6OHAxrBJgQieJmtXcCItu9YgeyOatcD2XGjYvM12MFLzii7dLz\\\\\\\/Y6EllRA2rO2lefBJrqQK9YtwWMdqWoaT8QUotMkElftfaplOhv7iMFxX668bjsfSf2nAhTqeb54hDkVTYq\\\\\\\/vDIYSP9LHKZIhbMRvVKbK7fylDJba\\\\\\\/A\\\\\\\/PDiPhBwNHURfqRBGZp\\\\\\\/\\\\\\\/zeCY5r8OLPQM+z\\\\\\\/xITPNZFxYIX+mh8YZzQBRqGwJVMPPdj+lkTOognsVkh13cen5PItYncqOJ0lEaAPaMoQqg0fXcIANkOs1nEGJIlSVlYUngpX5NWX4uhdA2mZwTI4oMsRBHB4Yel0Zz5t7QuNP8nOrZTT\\\\\\\/Hu3KFtoJwVAAWL+TLgn9q9L4XiAEla8j1InE2RDBUI1uZMlFQs1CkEsqCGnPueDuWjpqOsfKOrG+8QZq4Q+vlJF6rZlxhWFrpwHhbgbZHqQkFVphkJ7Aygv0QdleTwXooC9g97dPNnVRrLqKHxgXN4s75fUbBpfNyZfNTzbO432\\\\\\\/dphFqvFVzY6bR\\\\\\\/vrhrhNqiWkC5c8zHx2dDIvYgw4UlfO\\\\\\\/HYJD5Q320CLmII1d78awgwcL9Z6D1VsGcDXtDzBLF\\\\\\\/M6y0ajamqUMuVWabaxUklZp\\\\\\\/s3Ti8pLMwsaXMyjUfBNhCrAyMwc1GJv\\\\\\\/42wJn+FTZ6TXRrk9ZeopcfM4P7\\\\\\\/GBfbw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022uGphdSIX2ON9eGwaRxtZjA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eeaf8cc4a6d522cf4d966becfb8b3da9bd228889e4df16345eb3503e23aa2e16\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460533 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/19" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-njQeunLk2jx3-iAhwsIOK8MLkFYlooLGscPTFsGQds4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/19" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1ty-mDymqkpbucgsh5VpBZaZJYolcLpYqe1hzV-7OWc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/1" title="The CgHaa1-Regulon Mediates Response and Tolerance to Acetic Acid Stress in the Human Pathogen Candida glabrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/31" title="Small RNA Sequencing in Cells and Exosomes Identifies eQTLs and 14q32 as a Region of Active Export" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460533" id="node460533" data-pisa="ggg;7/1/19" data-pisa-master="ggg;g3.116.036145" data-apath="/ggg/7/1/19.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in <em>ITS2</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jana Batovska</span>, <span class="highwire-citation-author" data-delta="1">Noel O. I. Cogan</span>, <span class="highwire-citation-author" data-delta="2">Stacey E. Lynch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7864-5712" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark J. Blacket</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">19-29;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036145 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460533" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jana Batovska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BBatovska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batovska%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJana%2BBatovska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jana.batovska@ecodev.vic.gov.au" class="" data-icon-position="" data-hide-link-title="0">jana.batovska@ecodev.vic.gov.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noel O. I. Cogan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noel%2BO.%2BI.%2BCogan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cogan%20NO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoel%2BO.%2BI.%2BCogan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stacey E. Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stacey%2BE.%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStacey%2BE.%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark J. Blacket </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BJ.%2BBlacket%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blacket%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BJ.%2BBlacket%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7864-5712" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark J. Blacket</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/19" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460533" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/19.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460533" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/19.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460533" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/19.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460533" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/19.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460533" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Internal Transcribed Spacer 2 (<em>ITS2</em>) is a popular DNA barcoding marker; however, in some animal species it is hypervariable and therefore difficult to sequence with traditional methods. With next-generation sequencing (NGS) it is possible to sequence all gene variants despite the presence of single nucleotide polymorphisms (SNPs), insertions/deletions (indels), homopolymeric regions, and microsatellites. Our aim was to compare the performance of Sanger sequencing and NGS amplicon sequencing in characterizing <em>ITS2</em> in 26 mosquito species represented by 88 samples. The suitability of <em>ITS2</em> as a DNA barcoding marker for mosquitoes, and its allelic diversity in individuals and species, was also assessed. Compared to Sanger sequencing, NGS was able to characterize the <em>ITS2</em> region to a greater extent, with resolution within and between individuals and species that was previously not possible. A total of 382 unique sequences (alleles) were generated from the 88 mosquito specimens, demonstrating the diversity present that has been overlooked by traditional sequencing methods. Multiple indels and microsatellites were present in the <em>ITS2</em> alleles, which were often specific to species or genera, causing variation in sequence length. As a barcoding marker, <em>ITS2</em> was able to separate all of the species, apart from members of the <em>Culex pipiens</em> complex, providing the same resolution as the commonly used Cytochrome Oxidase I (<em>COI</em>). The ability to cost-effectively sequence hypervariable markers makes NGS an invaluable tool with many applications in the DNA barcoding field, and provides insights into the limitations of previous studies and techniques.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/amplicon-sequencing" class="hw-term hw-article-keyword hw-article-keyword-amplicon-sequencing" rel="nofollow">amplicon sequencing</a></li><li class="kwd"><a href="/keyword/ngs" class="hw-term hw-article-keyword hw-article-keyword-ngs" rel="nofollow">NGS</a></li><li class="kwd"><a href="/keyword/culicidae" class="hw-term hw-article-keyword hw-article-keyword-culicidae" rel="nofollow">Culicidae</a></li><li class="kwd"><a href="/keyword/microsatellites" class="hw-term hw-article-keyword hw-article-keyword-microsatellites" rel="nofollow">microsatellites</a></li><li class="kwd"><a href="/keyword/indels" class="hw-term hw-article-keyword hw-article-keyword-indels" rel="nofollow">indels</a></li></ul><p id="p-3">In recent years, DNA barcoding has gained popularity as a method to taxonomically identify unknown specimens. DNA barcoding has significant benefits compared to traditional morphological identification, such as differentiating similar-looking species and identifying immature or damaged specimens. The accuracy and efficiency of DNA-based species identification makes it particularly suitable to vector surveillance and biosecurity programs, where specimen identification informs public health predictions and vector control decisions (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Armstrong and Ball 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> 2016</a>).</p><p id="p-4">Mitochondrial Cytochrome Oxidase I (<em>COI</em>) is the most common barcoding gene used for animals; however, other ribosomal or nuclear genes are also often used and offer certain advantages. While mitochondrial genes are highly variable and easier to amplify due to their high copy number, nuclear genes exhibit variable rates of substitution that can provide greater resolving power (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lin and Danforth 2004</a>). The Internal Transcribed Spacer 2 (<em>ITS2</em>) region of ribosomal DNA has a high copy number and a higher evolutionary rate than most nuclear DNA; however, it can cause alignment problems due to the presence of variable copies (alleles) within individuals (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Álvarez and Wendel 2003</a>). When multiple species are being barcoded, it is beneficial to use a ribosomal or other nuclear gene in addition to a mitochondrial gene. These loci are unlinked and evolve separately, thereby giving independent estimates of genetic lineages and relationships. Numerous mosquito barcoding studies have used both mitochondrial and nuclear markers to distinguish species (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Foley <em>et al.</em> 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Paredes-Esquivel <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Puslednik <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bourke <em>et al.</em> 2013</a>). <em>ITS2</em> is often used in mosquito studies as its high evolutionary rate makes it useful for investigating closely related species, such as those in the <em>Anopheles</em> genus (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wilkerson <em>et al.</em> 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Marrelli <em>et al.</em> 2006</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Walton <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sum <em>et al.</em> 2014</a>).</p><p id="p-5">Currently, Sanger sequencing is the most commonly used method to acquire DNA barcodes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Taylor and Harris 2012</a>). While Sanger sequencing can generate reads up to 1000 bases, there are complications that limit its application in DNA barcoding. Sanger sequencing can only process a single DNA template per sample and multiple templates can cause uninterpretable results. This means that DNA regions with size-variable alleles within an individual cannot be sequenced. For instance, the size of <em>ITS2</em> varies in many mosquito species, so it must first be cloned and then sequenced in order to overcome this problem (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wesson <em>et al.</em> 1992</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Severini <em>et al.</em> 1996</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Aspen <em>et al.</em> 2003</a>). Pseudogenes can also create similar issues and have been detected in some mosquito species (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cywinska <em>et al.</em> 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2012</a>). Sanger sequencing also struggles with long homopolymeric and repetitive regions, often causing difficulties with microsatellites that may be found in barcoding markers, such as <em>ITS2</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Banerjee <em>et al.</em> 2007</a>).</p><p id="p-6">NGS technologies are able to achieve massive parallel sequencing of single DNA molecules, resulting in high-throughput data, unlike Sanger sequencing. Parallel sequencing means that multiple templates are not a challenge, and all regions and variants of the gene are sequenced, allowing the detection of pseudogenes, contaminants, or allelic variation within individual insects (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shokralla <em>et al.</em> 2014</a>). Although whole-genome sequencing was one of the initial uses of NGS technology, a variety of methods have since been developed that can generate multilocus sequence data for various applications such as genotyping and phylogenetics. One method, amplicon sequencing, involves tagging a unique barcode onto PCR-amplicons that are then pooled with other tagged amplicons and sequenced (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McCormack <em>et al.</em> 2013</a>). This multiplexing capability makes NGS a cost-effective method for DNA barcoding.</p><p id="p-7">In this study, we utilize both traditional Sanger sequencing and NGS amplicon sequencing to characterize a region of <em>ITS2</em> known to contain microsatellites and indels in mosquitoes. The success of both methods is compared in order to determine their applicability in sequencing hypervariable genes such as <em>ITS2</em>. The variability found within <em>ITS2</em> is also analyzed and its utility as a DNA barcoding marker for mosquitoes is assessed, compared with previous estimates based on the mitochondrial DNA locus <em>COI</em> (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> 2016</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Specimen collection and identification</h3><p id="p-8">Mosquitoes were collected as part of the Victorian Arbovirus Disease Control Program (VADCP). The methods used to collect, store, and morphologically identify the mosquitoes are described in <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> (2016)</a>. In total, 88 mosquito specimens were used in this study, representing 26 species and 12 genera. Specimen information, including trapping locations, can be found in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/TableS1.pdf">Table S1</a>. These specimens were previously used for a DNA barcoding project based upon <em>COI</em> (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> 2016</a>), and the same taxonomic classification has been employed here. Further details of the specimens used for these projects are recorded in the Mosquitoes of Australia – Victoria (MOAV) project on the Barcode of Life Database (BOLD, <a href="http://www.boldsystems.org">www.boldsystems.org</a>).</p></div><div id="sec-3" class="subsection"><h3>DNA isolation and ITS2 amplification</h3><p id="p-9">A single leg from each mosquito was used for DNA isolation. Magnetic bead-based nucleic acid extraction was performed using the MagMAX DNA Multi-Sample Kit (Life Technologies, Gaithersburg, MD). Sample homogenization and protocol adjustments are described in <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> (2016</a>). DNA extractions were stored at −20°.</p><p id="p-10">A region of <em>ITS2</em> (∼350–600 bp) was amplified using the novel primer pair <em>ITS2</em>-MOS-F (5′-GCTCGTGGATCGATGAAGAC-3′) and <em>ITS2</em>-MOS-R (5′-TGCTTAAATTTAGGGGGTAGTCAC-3′). These new primers are located in the conserved ribosomal DNA regions flanking <em>ITS2</em> and were designed from sequences obtained from GenBank (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wesson <em>et al.</em> 1992</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Toma <em>et al.</em> 2000</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Linton <em>et al.</em> 2001</a>) using Primer3 version 0.4.0 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Untergasser <em>et al.</em> 2012</a>). PCR reactions consisted of a total volume of 25 µl: 15.3 µl 1 × bovine serum albumin (BSA), 2.5 µl 10 × ThermoPol Reaction Buffer (New England Biolabs, Beverly, MA), 2 µl 2.5 µM dNTPs, 1.25 µl of each 10 µM/L primer, 0.2 µl 1.0 U <em>Taq</em> DNA Polymerase, and 5–15 ng template DNA. The <em>ITS2</em> PCR cycle was as follows: 94° for 2 min; 40 cycles of 94° for 30 sec, 51° for 45 sec, and 72° for 45 sec; and then finally 72° for 1 min. The PCR products were verified on a 2% agarose gel.</p></div><div id="sec-4" class="subsection"><h3>Sanger sequencing and data analysis</h3><p id="p-11">Size-verified <em>ITS2</em> PCR products were enzymatically purified and sequenced commercially in the forward direction on an ABI3730XL by Macrogen (Korea). To determine the overall quality of the sequences, the percentage of bases with a PHRED quality score (Q) above 20 was obtained from the beginning of the sequence to the final adenine peak. For quality trimming, bad quality bases were automatically removed from the 5′ and 3′ ends using a 0.01 error probability limit in Geneious version 8.1 (<a href="http://www.geneious.com">www.geneious.com</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kearse <em>et al.</em> 2012</a>). Any heterozygous bases found in the chromatograms were manually assessed and the base determined using the International Union of Pure Applied Chemistry (IUPAC) codes. Quality-trimmed Sanger sequences were aligned with ClustalW, sequence divergence was calculated (<em>p</em>-distance values), and a bootstrap neighbor-joining tree (1000 replicates) was created using MEGA version 6 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tamura <em>et al.</em> 2007</a>).</p></div><div id="sec-5" class="subsection"><h3>NGS and data analysis</h3><p id="p-12">The size-verified <em>ITS2</em> PCR products were purified using AMPure XP beads (Beckman Coulter, Brea, CA) with a 0.8 × beads ratio in all instances. Universal <em>Y</em>-shape adaptors were ligated onto the individual PCR products and excess adaptors were removed using AMPure XP beads. Unique 8 bp barcodes with Illumina P5 and P7 adapters were added onto the ligated products via PCR (× 18 cycles). An AMPure XP bead clean was performed and each sample was then quantified using a NanoDrop 1000 (Thermo Scientific, Waltman, MA). A single equimolar pool was created and then quantified using a Qubit 1.0 fluorometer (Thermo Scientific). Library size was checked with a 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA). Any remaining adaptor dimers were removed with a further AMPure XP bead clean-up. The pooled sample was sequenced on a MiSeq platform with 2 × 250 bp reads using version 3 chemistry (Illumina, San Diego, CA).</p><p id="p-13">The demultiplexed MiSeq data were trimmed to remove adapters and low quality sequence reads using a custom Perl script. Reads were removed if they had three or more consecutive ambiguous “N” bases; three or more nucleotides (nt) with ≤ Q20; a median Q score of &lt; 20; or a read length of &lt; 150 nt. Overlapping paired reads were merged using PEAR version 0.9.4 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhang <em>et al.</em> 2013</a>). Contigs (alleles) were assembled with CAP3 version date 12/21/07 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Huang and Madan 1999</a>) using a sequence identity of 95%. The allele with the largest number of reads attributed to it was considered to be the most common allele. All alleles produced were aligned using ClustalW in Geneious. Sequence divergence was calculated (<em>p</em>-distance values), and a bootstrap neighbor-joining tree (1000 replicates) was created using MEGA version 6 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Tamura <em>et al.</em> 2007</a>). Phobos version 3.3.12 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mayer 2006</a>) was used to detect microsatellites in the sequences. Microsatellites were defined as 1–6 bp with at least five repeats (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Selkoe and Toonen 2006</a>). Using ClustalW, the most common allele for each sample was aligned to the corresponding Sanger sequence in order to verify if the sequence matched. Coverage was also calculated, which was defined as the percentage of the allele covered by good quality Sanger sequence. Sanger and NGS <em>ITS2</em> neighbor-joining trees were compared to determine how many individuals were not resolved monophyletically; these included individuals that formed a complex with closely related species or did not resolve monophyletically with conspecifics. A two-by-two table was generated to compare this data, and overall percent agreement was calculated.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-14">All 382 NGS allele sequences are accessible on GenBank (accessions <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU495620&amp;atom=%2Fggg%2F7%2F1%2F19.atom">KU495620</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU495708&amp;atom=%2Fggg%2F7%2F1%2F19.atom">KU495708</a>: most common alleles; <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KX865970&amp;atom=%2Fggg%2F7%2F1%2F19.atom">KX865970</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KX866263&amp;atom=%2Fggg%2F7%2F1%2F19.atom">KX866263</a>:  other alleles). The most common alleles for each of the specimens are also available on the MOAV project on BOLD (accessions in the range of MOAV001-15–MOAV116-15), as is all associated specimen information. The raw sequencing files for all 88 samples have been deposited at the NCBI SRA database under project ID PRJNA343434.</p><p id="p-15">Individual specimen details are found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/TableS1.pdf">Table S1</a>, including trapping locations. <a href="http://www.g3journal.org/content/suppl/2016/10/31/g3.116.036145.DC1/FigureS1.pdf">Figure S1</a> compares Sanger and average NGS sequence lengths for all individual specimens. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/FigureS2.jpg">Figure S2</a> shows the location of microsatellites in all 382 NGS allele sequences. The neighbor-joining analysis of the 88 Sanger sequences can be seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/FigureS3.pdf">Figure S3</a>. </p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Sanger sequencing</h3><p id="p-16">In total, 88 Sanger sequences were generated for <em>ITS2</em>, one for each sample in the forward direction. The quality of the sequences varied among species (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). <em>Culex cylindricus</em> and <em>Aedeomyia venustipes</em> had consistently good quality sequences, with 91–93 and 92–94% of untrimmed bases &gt; Q20, respectively; whereas other species had poor quality sequences, such as <em>Dobrotworskyius rubrithorax</em> with only 59–63% of bases &gt; Q20 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Interestingly, some species had both good and bad quality sequences, including <em>Cx. pipiens</em> form <em>molestus</em> sequences ranging between 33 and 93% of bases &gt; Q20. After quality trimming, sequences were 95–523 bp in length, with the average length and SE for each species shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, and the length for each individual shown in <a href="http://www.g3journal.org/content/suppl/2016/10/31/g3.116.036145.DC1/FigureS1.pdf">Figure S1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470122/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470122/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><em>ITS2</em> sequence information for individual specimens</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/19/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ITS2 sequence information. (A) Comparison of mean Sanger sequence length post-trimming (red) and mean NGS allele length (blue) between species with standard error shown. (B) Number of alleles per individual shown as a histogram. (C) Number of alleles per species shown as a histogram. NGS, next-generation sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-829786065" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;ITS2&lt;/em&gt; sequence information. (A) Comparison of mean Sanger sequence length post-trimming (red) and mean NGS allele length (blue) between species with standard error shown. (B) Number of alleles per individual shown as a histogram. (C) Number of alleles per species shown as a histogram. NGS, next-generation sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/19/F1.medium.gif" width="357" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/1/19/F1.medium.gif" width="357" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/19/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/19/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470105" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child"><em>ITS2</em> sequence information. (A) Comparison of mean Sanger sequence length post-trimming (red) and mean NGS allele length (blue) between species with standard error shown. (B) Number of alleles per individual shown as a histogram. (C) Number of alleles per species shown as a histogram. NGS, next-generation sequencing.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>NGS</h3><p id="p-20">After bioinformatic analysis, NGS generated 382 unique sequences (alleles) for the 88 samples. When the most common allele was aligned to the corresponding trimmed Sanger sequence, there was a 100% sequence identity match in 85 out of the 88 samples. Of the three remaining samples, the Sanger sequence had a 100% sequence identity match with the second most common allele. The difference in read number between the two most common alleles for these three samples ranged from 1.5 to 27.8% of all reads used for CAP3 analysis.</p><p id="p-21">The alignments also revealed that the Sanger sequences post quality trimming covered a mean of 68.2% (range 19.1–90.7%) of the allele lengths. The mean coverage for each species can be seen in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, and the coverage for each individual sample is listed in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and shown in <a href="http://www.g3journal.org/content/suppl/2016/10/31/g3.116.036145.DC1/FigureS1.pdf">Figure S1</a>. While coverage varied among species with the aforementioned difference in Sanger sequence quality, generally all alleles were longer than the corresponding Sanger sequence. The only species with a Sanger sequence longer than the allele length was <em>Anopheles annulipes</em> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). This occurred because the targeted <em>ITS2</em> region in this species is &lt; 500 bp, which meant that the bioinformatic process could not create full-length alleles (the MiSeq read length generated was 2 × 250 bp, so the reads do not overlap the entire fragment and therefore the PEAR program cannot join them). Since the allele lengths are not an accurate representation of the full length of <em>ITS2</em> in <em>An. annulipes</em>, they were not included in the mean coverage calculation.</p><p id="p-22">It should be noted that <em>Ad. venustipes</em> had alleles longer than 500 bp (range 509–522 bp). When the raw reads for these samples were aligned to the corresponding Sanger <em>ITS2</em> sequences, reads were identified that began after the primer region. This is indicative of degradation or incomplete extension of the <em>ITS2</em> amplicons prior to library preparation, and explains how alleles longer than 500 bp could be formed. However, it appears there is a limit to this process, given that <em>An. annulipes</em> has a larger <em>ITS2</em> region (601 bp, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bower <em>et al.</em> 2009</a>) but does not assemble into a single contig.</p><p id="p-23">When examining the Sanger sequences post quality trimming, poor quality sequences were found to have resulted from a variety of allelic differences, including: SNPs, microsatellite regions, and indels. These variable regions were successfully sequenced with NGS and are present in the alleles for each individual. Examples of the effect of different types of polymorphisms in both the Sanger sequences and the alleles are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/19/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of sequences generated by Sanger and NGS technology in ITS2 regions containing different types of polymorphisms observed in this study. (A) Single nucleotide polymorphisms, (B) homopolymer region, (C) dinucleotide repeat, (D) insertion/deletion, and (E) combination. The multiple upper sequences show alleles derived from NGS data, while the lower chromatogram and single accompanying sequence were produced through Sanger sequencing. B and C are examples of microsatellites. NGS, next-generation sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-829786065" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of sequences generated by Sanger and NGS technology in &lt;em&gt;ITS2&lt;/em&gt; regions containing different types of polymorphisms observed in this study. (A) Single nucleotide polymorphisms, (B) homopolymer region, (C) dinucleotide repeat, (D) insertion/deletion, and (E) combination. The multiple upper sequences show alleles derived from NGS data, while the lower chromatogram and single accompanying sequence were produced through Sanger sequencing. B and C are examples of microsatellites. NGS, next-generation sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/19/F2.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/1/19/F2.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/19/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/19/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470210" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Comparison of sequences generated by Sanger and NGS technology in <em>ITS2</em> regions containing different types of polymorphisms observed in this study. (A) Single nucleotide polymorphisms, (B) homopolymer region, (C) dinucleotide repeat, (D) insertion/deletion, and (E) combination. The multiple upper sequences show alleles derived from NGS data, while the lower chromatogram and single accompanying sequence were produced through Sanger sequencing. B and C are examples of microsatellites. NGS, next-generation sequencing.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Variation within ITS2 alleles</h3><p id="p-25">The <em>ITS2</em> alleles were found to have many repetitive regions, with microsatellites ranging from 1 to 4 bp in length (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In total, the alleles contained 929 homopolymers, 90 dinucleotide, 12 trinucleotide, and four tetranucleotide microsatellites. Many of these microsatellites were specific to species or genera (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/FigureS2.jpg">Figure S2</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/19/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The location of microsatellites within 26 ITS2 sequences (shown from 100 bp), each representative of a species. The size of the light blue labels are proportionate to repeat length, whereas the number inside the label is representative of the repeat type. Microsatellites within all 382 alleles can be found in Figure S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-829786065" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The location of microsatellites within 26 &lt;em&gt;ITS2&lt;/em&gt; sequences (shown from 100 bp), each representative of a species. The size of the light blue labels are proportionate to repeat length, whereas the number inside the label is representative of the repeat type. Microsatellites within all 382 alleles can be found in Figure S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/19/F3.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/1/19/F3.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/19/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/19/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470118" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">The location of microsatellites within 26 <em>ITS2</em> sequences (shown from 100 bp), each representative of a species. The size of the light blue labels are proportionate to repeat length, whereas the number inside the label is representative of the repeat type. Microsatellites within all 382 alleles can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/FigureS2.jpg">Figure S2</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The presence of microsatellites and indels led to variation in <em>ITS2</em> length within and between individuals (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The mean allele length for each species is compared in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>. <em>Tripteroides atripes</em> had the shortest <em>ITS2</em> alleles (356–359 bp), while <em>Ad. venustipes</em> had the longest (509–522 bp).</p><p id="p-28">The number of <em>ITS2</em> alleles per individual also varied. Most individuals (<em>n</em> = 61, 69%) had one to three alleles, however the two members of the <em>Dobrotworskyius</em> genus had substantially more (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The number of alleles detected in <em>Db. rubrithorax</em> ranged from 24 to 35 per individual, resulting in a total of 114 alleles for the species; while the closely related <em>Db. alboannulatus</em> also had high allele numbers, with 11–17 per individual and a total of 53 for the species (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, B and C</a>).</p></div><div id="sec-11" class="subsection"><h3>ITS2 neighbor-joining analysis</h3><p id="p-29">The neighbor-joining analysis of 88 <em>ITS2</em> sequences derived from Sanger sequencing resulted in only 18 of the 26 species resolving monophyletically (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/FigureS3.pdf">Figure S3</a>). All of the <em>Culex</em> species failed to form distinct groups (except for <em>Cx. cylindricus</em>), as did <em>Ochlerotatus sagax</em>, and the <em>Dobrotworskyius</em> genus (total of 24 individuals, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). In comparison, NGS sequences provided much greater species resolution, with the neighbor-joining analysis of the 382 alleles resulting in 24 species forming distinct groups on the tree (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The only species that failed to form distinct groups were <em>Cx. pipiens</em> form <em>molestus</em> and <em>Cx. quinquefasciatus</em> (total of eight individuals, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). When the number of monophyletically resolved individuals were compared (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), Sanger and NGS had an estimate of agreement of 73%.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470086/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470086/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">A comparison of the number of individuals that resolved monophyletically using Sanger sequencing and NGS</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/19/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A summarized neighbor-joining tree, with bootstrap support values (%), based on p-distance comparisons between 382 ITS2 contig sequences from 88 mosquito samples. Tribal groups are indicated by brackets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-829786065" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A summarized neighbor-joining tree, with bootstrap support values (%), based on &lt;em&gt;p&lt;/em&gt;-distance comparisons between 382 &lt;em&gt;ITS2&lt;/em&gt; contig sequences from 88 mosquito samples. Tribal groups are indicated by brackets.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/19/F4.medium.gif" width="289" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/1/19/F4.medium.gif" width="289" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/19/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/19/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470111" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">A summarized neighbor-joining tree, with bootstrap support values (%), based on <em>p</em>-distance comparisons between 382 <em>ITS2</em> contig sequences from 88 mosquito samples. Tribal groups are indicated by brackets.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Due to the NGS data providing greater species resolution (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), it was used to calculate the percentage difference (<em>p</em>-distance) of different taxonomic categories (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The mean conspecific <em>p</em>-distance was 6.1% (range 0–24%), compared to 11.4% (range 0–23%) for congeneric divergence (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). The mean conspecific <em>p</em>-distance decreased substantially from 6.1 to 2% when alleles from the <em>Dobrotworskyius</em> genus were removed, revealing a greater differentiation of conspecific and congeneric divergences in other mosquito species (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). A DNA barcoding gap was not present (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). The mean <em>p</em>-distance between genera was 25% (range 7–73%, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). Peaks in pairwise comparison values were seen within tribes (mean = 18%), between tribes (mean = 30%), and between subfamilies (mean = 62%, <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/19/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of percentage difference (p-distances) for ITS2 in different taxonomic categories. Comparison of conspecific and congeneric differences of (A) all 88 samples, and (B) all samples excluding those from the Dobrotworskyius genus. (C) Differences between genera in 88 samples, with peaks corresponding to the differences within and between mosquito tribes, and between mosquito subfamilies. Brackets indicate the range of p-distances for each group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-829786065" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of percentage difference (&lt;em&gt;p&lt;/em&gt;-distances) for &lt;em&gt;ITS2&lt;/em&gt; in different taxonomic categories. Comparison of conspecific and congeneric differences of (A) all 88 samples, and (B) all samples excluding those from the &lt;em&gt;Dobrotworskyius&lt;/em&gt; genus. (C) Differences between genera in 88 samples, with peaks corresponding to the differences within and between mosquito tribes, and between mosquito subfamilies. Brackets indicate the range of &lt;em&gt;p&lt;/em&gt;-distances for each group.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/19/F5.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/1/19/F5.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/19/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/19/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470151" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Distribution of percentage difference (<em>p</em>-distances) for <em>ITS2</em> in different taxonomic categories. Comparison of conspecific and congeneric differences of (A) all 88 samples, and (B) all samples excluding those from the <em>Dobrotworskyius</em> genus. (C) Differences between genera in 88 samples, with peaks corresponding to the differences within and between mosquito tribes, and between mosquito subfamilies. Brackets indicate the range of <em>p</em>-distances for each group.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Sanger <em>vs.</em> NGS</h3><p id="p-34">Our results show that Sanger sequencing is not an appropriate method for characterizing <em>ITS2</em> in the majority of mosquito species tested in this study. As shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, there were a variety of polymorphisms in <em>ITS2</em> that hampered Sanger sequencing, leading to short sequences post quality trimming that are not suitable for DNA barcoding analyses (<a href="http://www.g3journal.org/content/suppl/2016/10/31/g3.116.036145.DC1/FigureS1.pdf">Figure S1</a> and <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">2</a>). When sequenced with NGS, areas with sequence polymorphisms were fully recovered, revealing the many <em>ITS2</em> alleles present in most mosquito species.</p><p id="p-35">Interestingly, some individuals had good quality Sanger sequences, yet had multiple alleles produced from the NGS data (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). This is likely to be due to an abundance of a single dominant allele. While there can be multiple <em>ITS2</em> alleles present in an individual, often only one or two are abundant. This was demonstrated by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Song <em>et al.</em> (2012)</a> when <em>ITS2</em> variants from 178 plant species were characterized and it was found that, on average, three of the most prevalent variants accounted for 91% of all <em>ITS2</em> copies per species. A similar result was obtained when <em>Symbiodinium ITS2</em> variants were sequenced, and the most common copy was on average ∼20 times more prevalent than the second most common copy (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arif <em>et al.</em> 2014</a>). Therefore, with the other alleles present in low proportions, Sanger sequencing is able to produce a clean chromatogram. The detection of the rarer alleles in our study demonstrates the sensitivity of the NGS method, as well as generating a more comprehensive sequence data set and resource for these mosquito species.</p><p id="p-36">Conversely, some individuals had disrupted Sanger sequences but only one allele identified in the NGS data (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). One possible cause is the parameters set for CAP3 when assembling the NGS reads, which only produced a contig if sequences were more than 95% similar, meaning that if there was &lt; 5% difference there would still only be one allele. Even this small amount of variation could cause issues with Sanger sequencing considering a single base pair frameshift mutation can disrupt the process. When an individual that had a poor quality Sanger sequence and a single allele was reanalyzed with a 98% cut-off set for CAP3, multiple alleles were produced, indicating that the CAP3 parameters are indeed likely to be responsible for the incongruent data. Other contributors could include the well-documented secondary structure in <em>ITS2</em>, quantification issues, and preferential sequencing of primer dimers, which can all result in poor Sanger sequences even if there is only a single <em>ITS2</em> allele (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wesson <em>et al</em>. 1992</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bertrand <em>et al</em>. 2014</a>).</p><p id="p-37">Regardless of the presence of any polymorphic allele issues, the NGS method consistently produced sequences that were closer to the full amplicon length compared to Sanger sequencing (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/content/suppl/2016/10/31/g3.116.036145.DC1/FigureS1.pdf">Figure S1</a>). The only instance in which the Sanger sequence was longer was in the <em>An. annulipes</em> species. The <em>ITS2</em> region used is larger than 500 bp in this species, and so the 250 bp paired-end reads generated in this study could not overlap to form the full <em>ITS2</em> sequence, highlighting a limitation of the NGS method. However, it should be noted that regions marginally larger than the overlapping NGS read length can sometimes be assembled, as seen with <em>Ad. venustipes</em>, which had a 509–522 bp <em>ITS2</em> region. With other markers, the problem could be solved by ensuring the amplicon being used is no larger than the overlapping NGS read length; but in <em>ITS2</em>, suitably conserved PCR priming sites are limited to the flanking ribosomal DNA regions, so reduction of the amplicon size is not feasible. When dealing with size-variable markers, another option is to sequence in only one direction; however, the actual sequence length that is useful for taxonomic assignment would need to be determined (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bertrand <em>et al.</em> 2014</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Arif <em>et al.</em> 2014</a>).</p><p id="p-38">Another advantage of NGS over Sanger sequencing is cost. If Sanger sequencing was used to generate the same level of data as NGS, it would require extensive cloning and sequencing of alleles, which would cost more than a single MiSeq run. Furthermore, Sanger sequencing requires each sample to be processed individually, whereas NGS can sequence all samples simultaneously using internal barcodes attached to Illumina adaptors. The low depth of coverage required means the number of samples can be upscaled to hundreds or thousands, which greatly reduces cost (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adey <em>et al.</em> 2010</a>).</p></div><div id="sec-14" class="subsection"><h3>ITS2 variation</h3><p id="p-39">The ability to sequence through genetic polymorphisms, such as microsatellites and indels, allowed NGS to characterize <em>ITS2</em> allelic diversity in individual mosquitoes. A total of 382 unique <em>ITS2</em> alleles were documented, with a mean of 4.3 alleles per individual (range 1–35, <a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). Concerted evolution is a process that reduces sequence variation among repeats within an individual; however, the presence of intragenomic <em>ITS2</em> variation is well-established (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bazzicalupo <em>et al.</em> 2013</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Bertrand <em>et al.</em> 2014</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Arif <em>et al.</em> 2014</a>). In our study, incomplete concerted evolution appears to be even more prevalent in particular species, with 167 of the 382 alleles attributed to the eight mosquitoes from the <em>Dobrotworskyius</em> genus (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). Furthermore, although the <em>Dobrotworskyius</em> species were recovered in this study, the large differences between alleles within the genus is indicative of incomplete lineage sorting between species (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5A</a>). Divergent copies of <em>ITS2</em> sequences may have been retained over long periods of time and could be useful in elucidating evolutionary relationships (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Vargas <em>et al.</em> 1999</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wissemann 2002</a>).</p><p id="p-40">In addition to the number of alleles detected, the length of <em>ITS2</em> also varied among species, ranging between 356–522 bp. Length was specific to species and caused difficulty in alignment, leading to large differences between some genera and particularly between subfamilies (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5C</a>). Ancient divergences lead to accumulated differences between Anophelinae and Culicinae species, however the degree of distance between these subfamilies using <em>ITS2</em> is inflated when compared to mitochondrial genes (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cywinska <em>et al.</em> 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Versteirt <em>et al.</em> 2015</a>; <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> 2016</a>). Alignment-free sequence analysis methods could possibly overcome some of the difficulty associated with comparing alleles of different sizes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Little 2011</a>).</p><p id="p-41">The alignment of alleles revealed that the differences in length were caused by the presence of numerous indels and microsatellites, which were often specific to species (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/FigureS2.jpg">Figure S2</a>). These polymorphisms are likely to be caused by DNA replication slippage, and their accumulation is furthered by the slow rate of concerted evolution in <em>ITS2</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hancock and Vogler 2000</a>). Their fast mutation rate might make them useful for population genetic studies, and species-specific microsatellites could be employed to improve genetic resolution and define boundaries for species determination (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Arif <em>et al.</em> 2014</a>).</p></div><div id="sec-15" class="subsection"><h3>ITS2 as a barcoding marker for mosquitoes</h3><p id="p-42">When using Sanger sequencing, <em>ITS2</em> is not a suitable barcoding marker for the species examined in this study, with only 69% of species resolving monophyletically (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1/FigureS3.pdf">Figure S3</a> and <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Due to the polymorphisms found in <em>ITS2</em> in these species, Sanger sequencing cannot characterize the region well, apart from when <em>An. annulipes</em> species are being sequenced (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/content/suppl/2016/10/31/g3.116.036145.DC1/FigureS1.pdf">Figure S1</a>). However, when using NGS, <em>ITS2</em> can separate species with a 96% success rate (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>), despite having overlapping conspecific and congeneric differences, and no clear barcoding gap (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5, A and B</a>). The only species that could not be separated were <em>Cx. quinquefasciatus</em> and <em>Cx. pipiens</em> form <em>molestus</em> (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). These species are part of the <em>Cx. pipiens</em> complex, which is known to be difficult to distinguish using standard barcoding methods (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Laurito <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pfeiler <em>et al.</em> 2013</a>; <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> 2016</a>). In comparison to <em>COI</em> as a marker in the same specimens/species (<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> 2016</a>), <em>ITS2</em> has similar resolution, with both markers successfully distinguishing all species with the exception of the <em>Cx. pipiens</em> complex. Mitochondrial and nuclear genes evolve independently, therefore the concordance between the two datasets observed here helps to confirm the established <em>COI</em> species and genera (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Lin and Danforth 2004</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Reinert <em>et al.</em> 2009</a>; <a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">Batovska <em>et al.</em> 2016</a>). In terms of utility as a DNA barcoding marker for mosquitoes, <em>COI</em> appears preferable to <em>ITS2</em> as it lacks the alignment issues caused by microsatellites and indels. However, <em>ITS2</em> does serve as a useful secondary marker that can now be efficiently obtained using NGS.</p></div><div id="sec-16" class="subsection"><h3>Conclusions</h3><p id="p-43">This study found that NGS is the most suitable method for characterizing the <em>ITS2</em> region in mosquitoes and this trend is likely to extend into many other insect species and genera. Multiplexing made NGS more efficient and cost-effective than Sanger sequencing, and polymorphic regions were successfully sequenced, revealing the large diversity of <em>ITS2</em> alleles present in mosquitoes. The data produced by this study captures the variation in number and size of <em>ITS2</em> alleles among species, as well as the abundant microsatellites and indels present. Future studies could use this kind of data to develop markers or investigate evolutionary histories, and lineages with unusual patterns could be examined further, such as the <em>Dobrotworskyius</em> genus. As a DNA barcoding marker, <em>ITS2</em> functions as well as <em>COI</em>; however, <em>COI</em> is a superior marker with fewer analytical limitations and complexities. The lack of depth required for NGS amplicon sequencing means that, in the future, a panel of alternative genes could be simultaneously sequenced in a variety of mosquito species in order to find a marker that could successfully separate all species and create well-supported phylogenies. The gene with best resolution may then be used for bulk DNA barcoding, where large pools of mosquitoes could be sequenced and identified, thereby significantly reducing identification time in surveillance programs (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hajibabaei <em>et al.</em> 2011</a>). The power and versatility of NGS is revolutionizing many scientific fields, including DNA barcoding, where allelic variation is no longer a limitation but rather an area for discovery.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">The authors thank the mosquito monitors for collecting mosquito samples; Karen Brown for morphologically identifying the mosquitoes; Hiroshi Shinozuka for guidance on the amplicon sequencing protocol; John Clancy from Westmead Hospital (Sydney) for providing assistance with confirming the identification of uncommon species; the Department of Economic Development, Jobs, Transport and Resources for project funding; and the Victorian Arbovirus Disease Control Program, which is funded through the Victorian Department of Health and Human Services. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-45">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036145/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-46"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-17"><span class="received-label">Received </span>July 17, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-10-10"><span class="accepted-label">Accepted </span>October 10, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Batovska <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.1" data-doi="10.1186/1465-6906-11-S1-I1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xun</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density <em>in vitro</em> transposition.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1465-6906-11-S1-I1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1465-6906-11-S1-I1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.2" data-doi="10.1016/S1055-7903(03)00208-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Álvarez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Ribosomal <em>ITS</em> sequences and plant phylogenetic inference.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DAlvarez%26rft.auinit1%253DI.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D417%26rft.epage%253D434%26rft.atitle%253DRibosomal%2BITS%2Bsequences%2Band%2Bplant%2Bphylogenetic%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1055-7903%252803%252900208-2%26rft_id%253Dinfo%253Apmid%252F14615184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1055-7903(03)00208-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14615184&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186738000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.3" data-doi="10.1111/mec.12869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arif</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daniels</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banguera-Hinestroza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbrook</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessing <em>Symbiodinium</em> diversity in scleractinian corals via next-generation sequencing-based genotyping of the <em>ITS2</em> rDNA region.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4418</span>–<span class="cit-lpage">4433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4418%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.4" data-doi="10.1098/rstb.2005.1713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">DNA barcodes for biosecurity: invasive species identification.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DArmstrong%26rft.auinit1%253DK.%2BF.%26rft.volume%253D360%26rft.issue%253D1462%26rft.spage%253D1813%26rft.epage%253D1823%26rft.atitle%253DDNA%2Bbarcodes%2Bfor%2Bbiosecurity%253A%2Binvasive%2Bspecies%2Bidentification%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2005.1713%26rft_id%253Dinfo%253Apmid%252F16214740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjAvMTQ2Mi8xODEzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aspen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crabtree</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savage</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polymerase chain reaction assay identifies <em>Culex nigripalpus</em>: part of an assay for molecular identification of the common <em>Culex</em> (<em>Culex</em>) mosquitoes of the eastern United States.</span> <abbr class="cit-jnl-abbrev">J. Am. Mosq. Control Assoc.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BMosquito%2BControl%2BAssociation%26rft.stitle%253DJ%2BAm%2BMosq%2BControl%2BAssoc%26rft.aulast%253DAspen%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D120%26rft.atitle%253DPolymerase%2Bchain%2Breaction%2Bassay%2Bidentifies%2BCulex%2Bnigripalpus%253A%2Bpart%2Bof%2Ban%2Bassay%2Bfor%2Bmolecular%2Bidentification%2Bof%2Bthe%2Bcommon%2BCulex%2B%2528Culex%2529%2Bmosquitoes%2Bof%2Bthe%2Beastern%2BUnited%2BStates.%26rft_id%253Dinfo%253Apmid%252F12825660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12825660&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183720200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arora</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murty</span>  <span class="cit-name-given-names">U. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">How far is <em>ITS2</em> reliable as a phylogenetic marker for the mosquito genera.</span> <abbr class="cit-jnl-abbrev">Electronic Journal of Biology</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectronic%2BJournal%2Bof%2BBiology%26rft.volume%253D3%26rft.spage%253D61%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Batovska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blacket</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Molecular identification of mosquitoes (Diptera: Culicidae) in southeastern Australia.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3001</span>–<span class="cit-lpage">3011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D3001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.8" data-doi="10.1016/j.funeco.2012.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bazzicalupo</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bálint</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of <em>ITS1</em> and <em>ITS2</em> rDNA in 454 sequencing of hyperdiverse fungal communities.</span> <abbr class="cit-jnl-abbrev">Fungal Ecol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BEcol.%26rft.volume%253D6%26rft.spage%253D102%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.funeco.2012.09.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.funeco.2012.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.9" data-doi="10.1186/1471-2148-14-153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janzen</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallwachs</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mitochondrial and nuclear phylogenetic analysis with Sanger and next-generation sequencing shows that, in Área de Conservación Guanacaste, northwestern Costa Rica, the skipper butterfly named <em>Urbanus belli</em> (family Hesperiidae) comprises three morphologically cryptic species.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D14%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-14-153%26rft_id%253Dinfo%253Apmid%252F25005355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-14-153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25005355&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.10" data-doi="10.1186/1756-3305-6-111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suesdek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergo</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sallum</span>  <span class="cit-name-given-names">M. A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A multi-locus approach to barcoding in the <em>Anopheles strodei</em> subgroup (Diptera: Culicidae).</span> <abbr class="cit-jnl-abbrev">Parasit. Vectors</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasit.%2BVectors%26rft.volume%253D6%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-3305-6-111%26rft_id%253Dinfo%253Apmid%252F23597081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-3305-6-111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23597081&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.11" data-doi="10.1007/s00239-008-9188-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beebe</span>  <span class="cit-name-given-names">N. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Internal repetition and intraindividual variation in the rDNA <em>ITS1</em> of the <em>Anopheles punctulatus</em> group (Diptera: Culicidae): multiple units and rates of turnover.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBower%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D79%26rft.atitle%253DInternal%2Brepetition%2Band%2Bintraindividual%2Bvariation%2Bin%2Bthe%2BrDNA%2BITS1%2Bof%2Bthe%2Banopheles%2Bpunctulatus%2Bgroup%2B%2528Diptera%253A%2BCulicidae%2529%253A%2Bmultiple%2Bunits%2Band%2Brates%2Bof%2Bturnover.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-008-9188-z%26rft_id%253Dinfo%253Apmid%252F19123014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-008-9188-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19123014&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263147100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.12" data-doi="10.1111/j.1365-2915.2006.00653.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cywinska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebert</span>  <span class="cit-name-given-names">P. D. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Identifying Canadian mosquito species through DNA barcodes.</span> <abbr class="cit-jnl-abbrev">Med. Vet. Entomol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMedical%2Band%2Bveterinary%2Bentomology%26rft.stitle%253DMed%2BVet%2BEntomol%26rft.aulast%253DCywinska%26rft.auinit1%253DA.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D413%26rft.epage%253D424%26rft.atitle%253DIdentifying%2BCanadian%2Bmosquito%2Bspecies%2Bthrough%2BDNA%2Bbarcodes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2915.2006.00653.x%26rft_id%253Dinfo%253Apmid%252F17199753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2915.2006.00653.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17199753&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243428800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.13" data-doi="10.1016/j.ympev.2006.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkerson</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volovsek</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A molecular phylogeny of <em>Anopheles annulipes</em> (Diptera: Culicidae) sensulato: the most species-rich anopheline complex.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DFoley%26rft.auinit1%253DD.%2BH.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D283%26rft.epage%253D297%26rft.atitle%253DA%2Bmolecular%2Bphylogeny%2Bof%2BAnopheles%2Bannulipes%2B%2528Diptera%253A%2BCulicidae%2529%2Bsensu%2Blato%253A%2Bthe%2Bmost%2Bspecies-rich%2Banopheline%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2006.10.008%26rft_id%253Dinfo%253Apmid%252F17126567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2006.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17126567&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245936000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.14" data-doi="10.1371/journal.pone.0017497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hajibabaei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shokralla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">G. A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Environmental barcoding: a next-generation sequencing approach for biomonitoring applications using river benthos.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e17497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHajibabaei%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De17497%26rft.epage%253De17497%26rft.atitle%253DEnvironmental%2Bbarcoding%253A%2Ba%2Bnext-generation%2Bsequencing%2Bapproach%2Bfor%2Bbiomonitoring%2Bapplications%2Busing%2Briver%2Bbenthos.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017497%26rft_id%253Dinfo%253Apmid%252F21533287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21533287&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.15" data-doi="10.1006/mpev.1999.0709"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogler</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">How slippage-derived sequences are incorporated into rRNA variable-region secondary structure: implications for phylogeny reconstruction.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">366</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DHancock%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D366%26rft.epage%253D374%26rft.atitle%253DHow%2Bslippage-derived%2Bsequences%2Bare%2Bincorporated%2Binto%2BrRNA%2Bvariable-region%2Bsecondary%2Bstructure%253A%2Bimplications%2Bfor%2Bphylogeny%2Breconstruction.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmpev.1999.0709%26rft_id%253Dinfo%253Apmid%252F10712842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/mpev.1999.0709&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10712842&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086029300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.16" data-doi="10.1101/gr.9.9.868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">CAP3: a DNA sequence assembly program.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.17" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTIvMTY0NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Oliveira</span>  <span class="cit-name-given-names">T. M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almirón</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sallum</span>  <span class="cit-name-given-names">M. A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>COI</em> barcode <em>vs.</em> morphological identification of <em>Culex</em> (<em>Culex</em>) (Diptera: Culicidae) species: a case study using samples from Argentina and Brazil.</span> <abbr class="cit-jnl-abbrev">Mem. Inst. Oswaldo Cruz</abbr> <span class="cit-vol">108</span>(<span class="cit-supplement">Suppl. 1.</span>): <span class="cit-fpage">110</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMem.%2BInst.%2BOswaldo%2BCruz%26rft.volume%253D108%26rft.spage%253D110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.19" data-doi="10.1016/S1055-7903(03)00241-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danforth</span>  <span class="cit-name-given-names">B. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">How do insect nuclear and mitochondrial gene substitution patterns differ? insights from Bayesian analyses of combined datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DLin%26rft.auinit1%253DC.%2BP.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D686%26rft.epage%253D702%26rft.atitle%253DHow%2Bdo%2Binsect%2Bnuclear%2Band%2Bmitochondrial%2Bgene%2Bsubstitution%2Bpatterns%2Bdiffer%253F%2BInsights%2Bfrom%2BBayesian%2Banalyses%2Bof%2Bcombined%2Bdatasets.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1055-7903%252803%252900241-0%26rft_id%253Dinfo%253Apmid%252F15012948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1055-7903(03)00241-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012948&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188958400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">Y.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samanidou-Voyadjoglou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbach</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">New occurrence records for <em>Anopheles maculipennis</em> and <em>An. messeae</em> in northern Greece based on DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Eur. Mosq. Bull.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BMosq.%2BBull.%26rft.volume%253D11%26rft.spage%253D31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.21" data-doi="10.1371/journal.pone.0020552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">DNA barcode sequence identification incorporating taxonomic hierarchy and within taxon variability.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e20552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLittle%26rft.auinit1%253DD.%2BP.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De20552%26rft.epage%253De20552%26rft.atitle%253DDNA%2Bbarcode%2Bsequence%2Bidentification%2Bincorporating%2Btaxonomic%2Bhierarchy%2Band%2Bwithin%2Btaxon%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0020552%26rft_id%253Dinfo%253Apmid%252F21857897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0020552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21857897&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.22" data-doi="10.1590/S0074-02762006000800002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marrelli</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sallum</span>  <span class="cit-name-given-names">M. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marinotti</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The second internal transcribed spacer of nuclear ribosomal DNA as a tool for Latin American anopheline taxonomy: a critical review.</span> <abbr class="cit-jnl-abbrev">Mem. Inst. Oswaldo Cruz</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">817</span>–<span class="cit-lpage">832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMemorias%2Bdo%2BInstituto%2BOswaldo%2BCruz%26rft.stitle%253DMemorias%2Bdo%2BInstituto%2BOswaldo%2BCruz%26rft.aulast%253DMarrelli%26rft.auinit1%253DM.%2BT.%26rft.volume%253D101%26rft.issue%253D8%26rft.spage%253D817%26rft.epage%253D832%26rft.atitle%253DThe%2Bsecond%2Binternal%2Btranscribed%2Bspacer%2Bof%2Bnuclear%2Bribosomal%2BDNA%2Bas%2Ba%2Btool%2Bfor%2BLatin%2BAmerican%2Banopheline%2Btaxonomy%2B-%2Ba%2Bcritical%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS0074-02762006000800002%26rft_id%253Dinfo%253Apmid%252F17293975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S0074-02762006000800002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17293975&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243649600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-7.1.19.23"><div class="cit-metadata unstructured">Mayer, C., 2006 Phobos 3.3.11. Available at: <a href="http://www.rub.de/spezzoo/cm/cm_phobos.htm">http://www.rub.de/spezzoo/cm/cm_phobos.htm</a>. Accessed: March 20, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.24" data-doi="10.1016/j.ympev.2011.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCormack</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hird</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zellmer</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brumfield</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Applications of next-generation sequencing to phylogeography and phylogenetics.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D66%26rft.spage%253D526%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2011.12.007%26rft_id%253Dinfo%253Apmid%252F22197804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2011.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22197804&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.25" data-doi="10.1016/j.ympev.2008.10.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paredes-Esquivel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbach</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A molecular phylogeny of mosquitoes in the <em>Anopheles barbirostris</em> subgroup reveals cryptic species: implications for identification of disease vectors.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D50%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2008.10.011%26rft_id%253Dinfo%253Apmid%252F19000771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2008.10.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19000771&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfeiler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flores-López</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mada-Vélez</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Escalante-Verdugo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markow</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic diversity and population genetics of mosquitoes (Diptera: Culicidae: <em>Culex</em> spp.) from the Sonoran desert of North America.</span> <abbr class="cit-jnl-abbrev">ScientificWorldJournal</abbr> <span class="cit-vol">2013</span>: <span class="cit-fpage">e724609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientificWorldJournal%26rft.volume%253D2013%26rft.spage%253De724609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puslednik</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J. W. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Phylogeography of the medically important mosquito <em>Aedes</em> (<em>Ochlerotatus</em>) <em>vigilax</em> (Diptera: Culicidae) in Australasia.</span> <abbr class="cit-jnl-abbrev">J. Biogeogr.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1333</span>–<span class="cit-lpage">1346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr.%26rft.volume%253D39%26rft.spage%253D1333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.28" data-doi="10.1111/j.1096-3642.2009.00570.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinert</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbach</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitching</span>  <span class="cit-name-given-names">I. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Phylogeny and classification of tribe Aedini (Diptera: Culicidae).</span> <abbr class="cit-jnl-abbrev">Zool. J. Linn. Soc.</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BJ.%2BLinn.%2BSoc.%26rft.volume%253D157%26rft.spage%253D700%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1096-3642.2009.00570.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1096-3642.2009.00570.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.29" data-doi="10.1111/j.1461-0248.2006.00889.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selkoe</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DSelkoe%26rft.auinit1%253DK.%2BA.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D615%26rft.epage%253D629%26rft.atitle%253DMicrosatellites%2Bfor%2Becologists%253A%2Ba%2Bpractical%2Bguide%2Bto%2Busing%2Band%2Bevaluating%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2006.00889.x%26rft_id%253Dinfo%253Apmid%252F16643306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2006.00889.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16643306&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236799600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.30" data-doi="10.1111/j.1365-2583.1996.tb00052.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Severini</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silvestrini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mancini</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">La Rosa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marinucci</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Sequence and secondary structure of the rDNA second internal transcribed spacer in the sibling species <em>Culex pipiens</em> L. and <em>Cx. quinquefasciatus</em> say (Diptera: Culicidae).</span> <abbr class="cit-jnl-abbrev">Insect Mol. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2Bmolecular%2Bbiology%26rft.stitle%253DInsect%2BMol%2BBiol%26rft.aulast%253DSeverini%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D181%26rft.epage%253D186%26rft.atitle%253DSequence%2Band%2Bsecondary%2Bstructure%2Bof%2Bthe%2BrDNA%2Bsecond%2Binternal%2Btranscribed%2Bspacer%2Bin%2Bthe%2Bsibling%2Bspecies%2BCulex%2Bpipiens%2BL.%2Band%2BCx.%2Bquinquefasciatus%2BSay%2B%2528Diptera%253A%2BCulicidae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2583.1996.tb00052.x%26rft_id%253Dinfo%253Apmid%252F8799736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2583.1996.tb00052.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8799736&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UT39100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shokralla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikbakht</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janzen</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallwachs</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Next-generation DNA barcoding: using next-generation sequencing to enhance and accelerate DNA barcode capture from single specimens.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">892</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D892%26rft_id%253Dinfo%253Apmid%252F24641208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24641208&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extensive pyrosequencing reveals frequent intra-genomic variations of internal transcribed spacer regions of nuclear ribosomal DNA.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e43971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSong%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De43971%26rft.epage%253De43971%26rft.atitle%253DExtensive%2Bpyrosequencing%2Breveals%2Bfrequent%2Bintra-genomic%2Bvariations%2Bof%2Binternal%2Btranscribed%2Bspacer%2Bregions%2Bof%2Bnuclear%2Bribosomal%2BDNA.%26rft_id%253Dinfo%253Apmid%252F22952830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22952830&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sum</span>  <span class="cit-name-given-names">J.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amir</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braima</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phylogenetic study of six species of <em>Anopheles</em> mosquitoes in Peninsular Malaysia based on inter-transcribed spacer region 2 (<em>ITS2</em>) of ribosomal DNA.</span> <abbr class="cit-jnl-abbrev">Parasit. Vectors</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasit.%2BVectors%26rft.volume%253D7%26rft.spage%253D309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.34" data-doi="10.1093/molbev/msm092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm092%26rft_id%253Dinfo%253Apmid%252F17488738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU5NiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.35" data-doi="10.1111/j.1755-0998.2012.03119.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">W. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An emergent science on the brink of irrelevance: a review of the past 8 years of DNA barcoding.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D12%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2012.03119.x%26rft_id%253Dinfo%253Apmid%252F22356472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2012.03119.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22356472&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.36" data-doi="10.1603/0022-2585-37.4.554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toma</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyagi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crabtree</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Identification of <em>Culex vishnui</em> subgroup (Diptera: Culicidae) mosquitoes from the Ryukyu Archipelago, Japan: development of a species-diagnostic polymerase chain reaction assay based on sequence variation in ribosomal DNA spacers.</span> <abbr class="cit-jnl-abbrev">J. Med. Entomol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">554</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1603%252F0022-2585-37.4.554%26rft_id%253Dinfo%253Apmid%252F10916296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoiam1lZGVudCI7czo1OiJyZXNpZCI7czo4OiIzNy80LzU1NCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.37" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer3–new capabilities and interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC8xNS9lMTE1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAllister</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jury</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Polyploid speciation in <em>Hedera</em> (Araliaceae): phylogenetic and biogeographic insights based on chromosome counts and <em>ITS</em> sequences.</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">219</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D219%26rft.spage%253D165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Versteirt</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">Z. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roelants</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breman</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of Belgian mosquito species (Diptera: Culicidae) by DNA barcoding.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somboon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Loughlin</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbach</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic diversity and molecular identification of mosquito species in the <em>Anopheles maculatus</em> group using the <em>ITS2</em> region of rDNA.</span> <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DWalton%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D93%26rft.epage%253D102%26rft.atitle%253DGenetic%2Bdiversity%2Band%2Bmolecular%2Bidentification%2Bof%2Bmosquito%2Bspecies%2Bin%2Bthe%2BAnopheles%2Bmaculatus%2Bgroup%2Busing%2Bthe%2BITS2%2Bregion%2Bof%2BrDNA.%26rft_id%253Dinfo%253Apmid%252F16782411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16782411&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.41" data-doi="10.1371/journal.pone.0047051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identifying the main mosquito species in China based on DNA barcoding.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWang%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De47051%26rft.epage%253De47051%26rft.atitle%253DIdentifying%2Bthe%2Bmain%2Bmosquito%2Bspecies%2Bin%2Bchina%2Bbased%2Bon%2BDNA%2Bbarcoding.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047051%26rft_id%253Dinfo%253Apmid%252F23071708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071708&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.42" data-doi="10.1016/1055-7903(92)90001-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wesson</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Sequence and secondary structure comparisons of <em>ITS</em> rDNA in mosquitoes (Diptera: Culicidae).</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DWesson%26rft.auinit1%253DD.%2BM.%26rft.volume%253D1%26rft.issue%253D4%26rft.spage%253D253%26rft.epage%253D269%26rft.atitle%253DSequence%2Band%2Bsecondary%2Bstructure%2Bcomparisons%2Bof%2BITS%2BrDNA%2Bin%2Bmosquitoes%2B%2528Diptera%253A%2BCulicidae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F1055-7903%252892%252990001-W%26rft_id%253Dinfo%253Apmid%252F1364170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/1055-7903(92)90001-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1364170&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.43" data-doi="10.1603/0022-2585-41.3.392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkerson</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinert</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Ribosomal DNA <em>ITS2</em> sequences differentiate six species in the <em>Anopheles crucians</em> complex (Diptera: Culicidae).</span> <abbr class="cit-jnl-abbrev">J. Med. Entomol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1603%252F0022-2585-41.3.392%26rft_id%253Dinfo%253Apmid%252F15185940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoiam1lZGVudCI7czo1OiJyZXNpZCI7czo4OiI0MS8zLzM5MiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS8xOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wissemann</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular evidence for allopolyploid origin of the <em>Rosa canina</em> complex (Rosaceae, Rosoideae).</span> <abbr class="cit-jnl-abbrev">J. Appl. Bot.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BBot.%26rft.volume%253D76%26rft.spage%253D176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.19.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobert</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flouri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">PEAR: a fast and accurate Illumina Paired-End reAd mergeR.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D30%26rft.spage%253D614%26rft_id%253Dinfo%253Apmid%252F24142950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24142950&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F19.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332259300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/19.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/1" title="The CgHaa1-Regulon Mediates Response and Tolerance to Acetic Acid Stress in the Human Pathogen Candida glabrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/31" title="Small RNA Sequencing in Cells and Exosomes Identifies eQTLs and 14q32 as a Region of Active Export" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/19" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KUxujWqnPUEsX_rBfoI9BHtnGmAq4P27C-tDHqjqudE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/19" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/19" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/19" class="active" data-icon-position="" data-hide-link-title="0">Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in ITS2</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460533" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5tEzzNRJa3ZZf1w2QkJWMv_nZnk_HgCXO1aBvnUMTsU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/19.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/19" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RPAgOWJqdOvR8nX6rAwRiB_5Y-st_paOLc-kFPqTkC0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/19" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/19.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460533" id="node460533--2" data-pisa="ggg;7/1/19" data-pisa-master="ggg;g3.116.036145" data-apath="/ggg/7/1/19.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in <em>ITS2</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jana Batovska</span>, <span class="highwire-citation-author" data-delta="1">Noel O. I. Cogan</span>, <span class="highwire-citation-author" data-delta="2">Stacey E. Lynch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7864-5712" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark J. Blacket</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">19-29;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036145 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460533--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jana Batovska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BBatovska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batovska%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJana%2BBatovska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jana.batovska@ecodev.vic.gov.au" class="" data-icon-position="" data-hide-link-title="0">jana.batovska@ecodev.vic.gov.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noel O. I. Cogan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noel%2BO.%2BI.%2BCogan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cogan%20NO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoel%2BO.%2BI.%2BCogan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stacey E. Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stacey%2BE.%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStacey%2BE.%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark J. Blacket </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BJ.%2BBlacket%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blacket%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BJ.%2BBlacket%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7864-5712" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark J. Blacket</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/19" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/19&amp;title=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/19&amp;title=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/19&amp;title=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/19&amp;text=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/19&amp;title=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/19&amp;t=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/19&amp;title=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/19&amp;title=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460533/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460533" id="node460533--3" data-pisa="ggg;7/1/19" data-pisa-master="ggg;g3.116.036145" data-apath="/ggg/7/1/19.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Next-Generation Sequencing for DNA Barcoding: Capturing Allelic Variation in <em>ITS2</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jana Batovska</span>, <span class="highwire-citation-author" data-delta="1">Noel O. I. Cogan</span>, <span class="highwire-citation-author" data-delta="2">Stacey E. Lynch</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7864-5712" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark J. Blacket</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">19-29;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036145 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460533--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jana Batovska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jana%2BBatovska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batovska%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJana%2BBatovska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jana.batovska@ecodev.vic.gov.au" class="" data-icon-position="" data-hide-link-title="0">jana.batovska@ecodev.vic.gov.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noel O. I. Cogan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noel%2BO.%2BI.%2BCogan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cogan%20NO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoel%2BO.%2BI.%2BCogan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stacey E. Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stacey%2BE.%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStacey%2BE.%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark J. Blacket </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>BioSciences Research, Agriculture Victoria, AgriBio Centre for AgriBioscience, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BJ.%2BBlacket%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blacket%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BJ.%2BBlacket%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7864-5712" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark J. Blacket</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=19&amp;rft.epage=29&amp;rft.atitle=Using%20Next-Generation%20Sequencing%20for%20DNA%20Barcoding%3A%20Capturing%20Allelic%20Variation%20in%20ITS2&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Batovska%2C+Jana&amp;rft.au=Cogan%2C+Noel+O.+I.&amp;rft.au=Lynch%2C+Stacey+E.&amp;rft.au=Blacket%2C+Mark+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460533/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460533/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460533/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460533/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460533/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460533/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460533/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460533/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460533/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460533/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460533/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460533/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/19.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/19.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/1/19.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/1/19.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
