<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/63" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/63.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/63.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/1/63.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/1/63/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/63.atom" />
<meta name="HW.pisa" content="ggg;7/1/63" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Phenotypic Plasticity of Duplicated Genes in Saccharomyces cerevisiae and the Origin of Adaptations" />
<meta name="DC.Identifier" content="10.1534/g3.116.035329" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Mattenberger et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gene and genome duplication are the major sources of biological innovations in plants and animals. Functional and transcriptional divergence between the copies after gene duplication has been considered the main driver of innovations . However, here we show that increased phenotypic plasticity after duplication plays a more major role than thought before in the origin of adaptations. We perform an exhaustive analysis of the transcriptional alterations of duplicated genes in the unicellular eukaryote Saccharomyces cerevisiae when challenged with five different environmental stresses. Analysis of the transcriptomes of yeast shows that gene duplication increases the transcriptional response to environmental changes, with duplicated genes exhibiting signatures of adaptive transcriptional patterns in response to stress. The mechanism of duplication matters, with whole-genome duplicates being more transcriptionally altered than small-scale duplicates. The predominant transcriptional pattern follows the classic theory of evolution by gene duplication; with one gene copy remaining unaltered under stress, while its sister copy presents large transcriptional plasticity and a prominent role in adaptation. Moreover, we find additional transcriptional profiles that are suggestive of neo- and subfunctionalization of duplicate gene copies. These patterns are strongly correlated with the functional dependencies and sequence divergence profiles of gene copies. We show that, unlike singletons, duplicates respond more specifically to stress, supporting the role of natural selection in the transcriptional plasticity of duplicates. Our results reveal the underlying transcriptional complexity of duplicated genes and its role in the origin of adaptations." />
<meta name="DC.Contributor" content="Florian Mattenberger" />
<meta name="DC.Contributor" content="Beatriz Sabater-Muñoz" />
<meta name="DC.Contributor" content="Christina Toft" />
<meta name="DC.Contributor" content="Mario A. Fares" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Phenotypic Plasticity of Duplicated Genes in Saccharomyces cerevisiae and the Origin of Adaptations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gene and genome duplication are the major sources of biological innovations in plants and animals. Functional and transcriptional divergence between the copies after gene duplication has been considered the main driver of innovations . However, here we show that increased phenotypic plasticity after duplication plays a more major role than thought before in the origin of adaptations. We perform an exhaustive analysis of the transcriptional alterations of duplicated genes in the unicellular eukaryote &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; when challenged with five different environmental stresses. Analysis of the transcriptomes of yeast shows that gene duplication increases the transcriptional response to environmental changes, with duplicated genes exhibiting signatures of adaptive transcriptional patterns in response to stress. The mechanism of duplication matters, with whole-genome duplicates being more transcriptionally altered than small-scale duplicates. The predominant transcriptional pattern follows the classic theory of evolution by gene duplication; with one gene copy remaining unaltered under stress, while its sister copy presents large transcriptional plasticity and a prominent role in adaptation. Moreover, we find additional transcriptional profiles that are suggestive of neo- and subfunctionalization of duplicate gene copies. These patterns are strongly correlated with the functional dependencies and sequence divergence profiles of gene copies. We show that, unlike singletons, duplicates respond more specifically to stress, supporting the role of natural selection in the transcriptional plasticity of duplicates. Our results reveal the underlying transcriptional complexity of duplicated genes and its role in the origin of adaptations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/63" />
<meta name="citation_id" content="7/1/63" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/63" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/63.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/63.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/63.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035329" />
<meta name="citation_pmid" content="27799339" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="63" />
<meta name="citation_lastpage" content="75" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Florian Mattenberger" />
<meta name="citation_author_institution" content="Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2727-0284" />
<meta name="citation_author" content="Beatriz Sabater-Muñoz" />
<meta name="citation_author_institution" content="Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain" />
<meta name="citation_author_institution" content="Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0301-215X" />
<meta name="citation_author" content="Christina Toft" />
<meta name="citation_author_institution" content="Department of Genetics, University of Valencia, 46100 Burjassot, Spain" />
<meta name="citation_author_institution" content="Departamento de Biotecnología, Instituto de Agroquímica y Tecnología de los Alimentos, Consejo Superior de Investigaciones Científicas, Valencia, 46980 Paterna, Spain" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1714-6703" />
<meta name="citation_author" content="Mario A. Fares" />
<meta name="citation_author_institution" content="Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain" />
<meta name="citation_author_institution" content="Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland" />
<meta name="citation_author_email" content="mfares@ibmcp.upv.es" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4345-3013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_issue=17;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Anders;citation_author=W. Huber;citation_title=Differential expression analysis for sequence count data.;citation_pages=R106-R106;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20979621;citation_doi=10.1186/gb-2010-11-10-r106" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Barkman;citation_author=J. Zhang;citation_title=Evidence for escape from adaptive conflict?;citation_pages=E1-E1;citation_volume=462;citation_year=2009;citation_issue=7274;citation_pmid=20010636;citation_doi=10.1038/nature08663" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=G. Belli;citation_author=M. M. Molina;citation_author=J. Garcia-Martinez;citation_author=J. E. Perez-Ortin;citation_author=E. Herrero;citation_title=Saccharomyces cerevisiae glutaredoxin 5-deficient cells subjected to continuous oxidizing conditions are affected in the expression of specific sets of genes.;citation_volume=279;citation_year=2004;citation_issue=13;citation_pmid=14722110;citation_doi=10.1074/jbc.M311879200" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=G. Blanc;citation_author=K. H. Wolfe;citation_title=Functional divergence of duplicated genes formed by polyploidy during Arabidopsis evolution.;citation_volume=16;citation_year=2004;citation_issue=7;citation_pmid=15208398;citation_doi=10.1105/tpc.021410" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. P. Byrne;citation_author=K. H. Wolfe;citation_title=The yeast gene order browser: combining curated homology and syntenic context reveals gene fate in polyploid species.;citation_volume=15;citation_year=2005;citation_issue=10;citation_pmid=16169922;citation_doi=10.1101/gr.3672305" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. Carretero-Paulet;citation_author=M. A. Fares;citation_title=Evolutionary dynamics and functional specialization of plant paralogs formed by whole and small-scale genome duplications.;citation_volume=29;citation_year=2012;citation_issue=11;citation_pmid=22734049;citation_doi=10.1093/molbev/mss162" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=H. C. Causton;citation_author=B. Ren;citation_author=S. S. Koh;citation_author=C. T. Harbison;citation_author=E. Kanin;citation_title=Remodeling of Yeast Genome Expression in Response to Environmental Changes;citation_pages=323-337;citation_volume=12;citation_year=2001;citation_issue=2;citation_pmid=11179418;citation_doi=10.1091/mbc.12.2.323" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=G. C. Conant;citation_author=K. H. Wolfe;citation_title=Functional partitioning of yeast co-expression networks after genome duplication.;citation_pages=e109-e109;citation_volume=4;citation_year=2006;citation_issue=4;citation_pmid=16555924;citation_doi=10.1371/journal.pbio.0040109" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. C. Conant;citation_author=K. H. Wolfe;citation_title=Turning a hobby into a job: how duplicated genes find new functions.;citation_pages=938-950;citation_volume=9;citation_year=2008;citation_issue=12;citation_pmid=19015656;citation_doi=10.1038/nrg2482" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Cormier;citation_author=R. Barbey;citation_author=L. Kuras;citation_title=Transcriptional plasticity through differential assembly of a multiprotein activation complex.;citation_volume=38;citation_year=2010;citation_issue=15;citation_pmid=20392822;citation_doi=10.1093/nar/gkq257" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_title=Elements of Information Theory.;citation_author=T. M. Cover;citation_author=J. A. Thomas;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Cui;citation_author=P. K. Wall;citation_author=J. H. Leebens-Mack;citation_author=B. G. Lindsay;citation_author=D. E. Soltis;citation_title=Widespread genome duplications throughout the history of flowering plants.;citation_volume=16;citation_year=2006;citation_issue=6;citation_pmid=16702410;citation_doi=10.1101/gr.4825606" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. J. Dean;citation_author=J. C. Davis;citation_author=R. W. Davis;citation_author=D. A. Petrov;citation_title=Pervasive and persistent redundancy among duplicated genes in yeast.;citation_pages=e1000113-e1000113;citation_volume=4;citation_year=2008;citation_issue=7;citation_pmid=18604285;citation_doi=10.1371/journal.pgen.1000113" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. DeLuna;citation_author=K. Vetsigian;citation_author=N. Shoresh;citation_author=M. Hegreness;citation_author=M. Colon-Gonzalez;citation_title=Exposing the fitness contribution of duplicated genes.;citation_pages=676-681;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18408719;citation_doi=10.1038/ng.123" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=E. T. Dermitzakis;citation_author=A. G. Clark;citation_title=Differential Selection After Duplication in Mammalian Developmental Genes;citation_pages=557-562;citation_volume=18;citation_year=2001;citation_issue=4;citation_pmid=11264407;citation_doi=10.1093/oxfordjournals.molbev.a003835" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. L. Des Marais;citation_author=M. D. Rausher;citation_title=Escape from adaptive conflict after duplication in an anthocyanin pathway gene.;citation_pages=762-765;citation_volume=454;citation_year=2008;citation_issue=7205;citation_pmid=18594508;citation_doi=10.1038/nature07092" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. A. Draghi;citation_author=T. L. Parsons;citation_author=G. P. Wagner;citation_author=J. B. Plotkin;citation_title=Mutational robustness can facilitate adaptation.;citation_pages=353-355;citation_volume=463;citation_year=2010;citation_issue=7279;citation_pmid=20090752;citation_doi=10.1038/nature08694" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. A. Drummond;citation_author=C. O. Wilke;citation_title=Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.;citation_pages=341-352;citation_volume=134;citation_year=2008;citation_issue=2;citation_pmid=18662548;citation_doi=10.1016/j.cell.2008.05.042" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. A. Drummond;citation_author=J. D. Bloom;citation_author=C. Adami;citation_author=C. O. Wilke;citation_author=F. H. Arnold;citation_title=Why highly expressed proteins evolve slowly.;citation_volume=102;citation_year=2005;citation_issue=40;citation_pmid=16176987;citation_doi=10.1073/pnas.0504070102" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. A. Drummond;citation_author=A. Raval;citation_author=C. O. Wilke;citation_title=A single determinant dominates the rate of yeast protein evolution.;citation_volume=23;citation_year=2006;citation_issue=2;citation_pmid=16237209;citation_doi=10.1093/molbev/msj038" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. A. Fares;citation_title=The origins of mutational robustness.;citation_pages=373-381;citation_volume=31;citation_year=2015;citation_issue=7;citation_pmid=26013677;citation_doi=10.1016/j.tig.2015.04.008" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. A. Fares;citation_author=O. M. Keane;citation_author=C. Toft;citation_author=L. Carretero-Paulet;citation_author=G. W. Jones;citation_title=The roles of whole-genome and small-scale duplications in the functional specialization of Saccharomyces cerevisiae genes.;citation_pages=e1003176;citation_volume=9;citation_year=2013;citation_issue=1;citation_pmid=23300483;citation_doi=10.1371/journal.pgen.1003176" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. L. Ferea;citation_author=D. Botstein;citation_author=P. O. Brown;citation_author=R. F. Rosenzweig;citation_title=Systematic changes in gene expression patterns following adaptive evolution in yeast;citation_pages=9721-9726;citation_volume=96;citation_year=1999;citation_issue=17;citation_pmid=10449761;citation_doi=10.1073/pnas.96.17.9721" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. D. Ferris;citation_author=G. S. Whitt;citation_title=Evolution of the differential regulation of duplicate genes after polyploidization.;citation_pages=267-317;citation_volume=12;citation_year=1979;citation_issue=4;citation_pmid=448746;citation_doi=10.1007/BF01732026" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Force;citation_author=M. Lynch;citation_author=F. B. Pickett;citation_author=A. Amores;citation_author=Y. L. Yan;citation_title=Preservation of Duplicate Genes by Complementary, Degenerative Mutations;citation_pages=1531-1545;citation_volume=151;citation_year=1999;citation_issue=4;citation_pmid=10101175" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. P. Francino;citation_title=An adaptive radiation model for the origin of new gene functions.;citation_pages=573-577;citation_volume=37;citation_year=2005;citation_issue=6;citation_pmid=15920518;citation_doi=10.1038/ng1579" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. J. Gibson;citation_author=J. Spring;citation_title=Genetic redundancy in vertebrates: polyploidy and persistence of genes encoding multidomain proteins.;citation_pages=46-49;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9520595;citation_doi=10.1016/S0168-9525(97)01367-X" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Goldman;citation_author=Z. Yang;citation_title=A codon-based model of nucleotide substitution for protein-coding DNA sequences;citation_pages=725-736;citation_volume=11;citation_year=1994;citation_issue=5;citation_pmid=7968486" />
<meta name="citation_reference" content="citation_journal_title=Paleobiology;citation_author=S. J. Gould;citation_author=E. S. Vrba;citation_title=Exaptation-A missing term in the science of form.;citation_pages=12;citation_volume=8;citation_year=1982;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=Z. Gu;citation_author=D. Nicolae;citation_author=H. H. Lu;citation_author=W. H. Li;citation_title=Rapid divergence in expression between duplicate genes inferred from microarray data.;citation_pages=609-613;citation_volume=18;citation_year=2002;citation_issue=12;citation_pmid=12446139;citation_doi=10.1016/S0168-9525(02)02837-8" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Ha;citation_author=W. H. Li;citation_author=Z. J. Chen;citation_title=External factors accelerate expression divergence between duplicate genes.;citation_pages=162-166;citation_volume=23;citation_year=2007;citation_issue=4;citation_pmid=17320239;citation_doi=10.1016/j.tig.2007.02.005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Ha;citation_author=E. D. Kim;citation_author=Z. J. Chen;citation_title=Duplicate genes increase expression diversity in closely related species and allopolyploids.;citation_volume=106;citation_year=2009;citation_issue=7;citation_pmid=19168631;citation_doi=10.1073/pnas.0807350106" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Halabi;citation_author=O. Rivoire;citation_author=S. Leibler;citation_author=R. Ranganathan;citation_title=Protein sectors: evolutionary units of three-dimensional structure.;citation_pages=774-786;citation_volume=138;citation_year=2009;citation_issue=4;citation_pmid=19703402;citation_doi=10.1016/j.cell.2009.07.038" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. He;citation_author=J. Zhang;citation_title=Rapid subfunctionalization accompanied by prolonged and substantial neofunctionalization in duplicate gene evolution.;citation_volume=169;citation_year=2005;citation_issue=2;citation_pmid=15654095;citation_doi=10.1534/genetics.104.037051" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. Hoegg;citation_author=H. Brinkmann;citation_author=J. S. Taylor;citation_author=A. Meyer;citation_title=Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish.;citation_pages=190-203;citation_volume=59;citation_year=2004;citation_issue=2;citation_pmid=15486693;citation_doi=10.1007/s00239-004-2613-z" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=F. G. Hoffmann;citation_author=J. C. Opazo;citation_author=J. F. Storz;citation_title=Differential loss and retention of cytoglobin, myoglobin, and globin-E during the radiation of vertebrates.;citation_volume=3;citation_year=2011;citation_pmid=21697098;citation_doi=10.1093/gbe/evr055" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. G. Hoffmann;citation_author=J. C. Opazo;citation_author=D. Hoogewijs;citation_author=T. Hankeln;citation_author=B. Ebner;citation_title=Evolution of the globin gene family in deuterostomes: lineage-specific patterns of diversification and attrition.;citation_volume=29;citation_year=2012;citation_issue=7;citation_pmid=22319164;citation_doi=10.1093/molbev/mss018" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. G. Hoffmann;citation_author=J. C. Opazo;citation_author=J. F. Storz;citation_title=Whole-genome duplications spurred the functional diversification of the globin gene superfamily in vertebrates.;citation_volume=29;citation_year=2012;citation_issue=1;citation_pmid=21965344;citation_doi=10.1093/molbev/msr207" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=E. B. Holub;citation_title=The arms race is ancient history in Arabidopsis, the wildflower.;citation_pages=516-527;citation_volume=2;citation_year=2001;citation_issue=7;citation_pmid=11433358;citation_doi=10.1038/35080508" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Huminiecki;citation_author=K. H. Wolfe;citation_title=Divergence of Spatial Gene Expression Profiles Following Species-Specific Gene Duplications in Human and Mouse;citation_pages=1870-1879;citation_volume=14;citation_year=2004;citation_issue=10a;citation_pmid=15466287;citation_doi=10.1101/gr.2705204" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. Ideker;citation_author=V. Thorsson;citation_author=J. A. Ranish;citation_author=R. Christmas;citation_author=J. Buhler;citation_title=Integrated Genomic and Proteomic Analyses of a Systematically Perturbed Metabolic Network;citation_pages=929-934;citation_volume=292;citation_year=2001;citation_issue=5518;citation_pmid=11340206;citation_doi=10.1126/science.292.5518.929" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=J. Ihmels;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=N. J. Krogan;citation_author=J. S. Weissman;citation_title=Backup without redundancy: genetic interactions reveal the cost of duplicate gene loss;citation_pages=86-86;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17389874;citation_doi=10.1038/msb4100127" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Innan;citation_author=F. Kondrashov;citation_title=The evolution of gene duplications: classifying and distinguishing between models.;citation_pages=97-108;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20051986;citation_doi=10.1038/nrg2689" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=O. M. Keane;citation_author=C. Toft;citation_author=L. Carretero-Paulet;citation_author=G. W. Jones;citation_author=M. A. Fares;citation_title=Preservation of genetic and regulatory robustness in ancient gene duplicates of Saccharomyces cerevisiae.;citation_volume=24;citation_year=2014;citation_issue=11;citation_pmid=25149527;citation_doi=10.1101/gr.176792.114" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=S. Kim;citation_author=M. J. Yoo;citation_author=V. A. Albert;citation_author=J. S. Farris;citation_author=P. S. Soltis;citation_title=Phylogeny and diversification of B-function MADS-box genes in angiosperms: evolutionary and functional implications of a 260-million-year-old duplication;citation_pages=2102-2118;citation_volume=91;citation_year=2004;citation_issue=12;citation_pmid=21652358;citation_doi=10.3732/ajb.91.12.2102;citation_georef=2015097402" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=C. R. Landry;citation_author=J. Oh;citation_author=D. L. Hartl;citation_author=D. Cavalieri;citation_title=Genome-wide scan reveals that genetic variation for transcriptional plasticity in yeast is biased towards multi-copy and dispensable genes.;citation_pages=343-351;citation_volume=366;citation_year=2006;citation_issue=2;citation_pmid=16427747;citation_doi=10.1016/j.gene.2005.10.042" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=O. Lespinet;citation_author=Y. I. Wolf;citation_author=E. V. Koonin;citation_author=L. Aravind;citation_title=The Role of Lineage-Specific Gene Family Expansion in the Evolution of Eukaryotes;citation_pages=1048-1059;citation_volume=12;citation_year=2002;citation_issue=7;citation_pmid=12097341;citation_doi=10.1101/gr.174302" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=W. H. Li;citation_author=J. Yang;citation_author=X. Gu;citation_title=Expression divergence between duplicate genes.;citation_pages=602-607;citation_volume=21;citation_year=2005;citation_issue=11;citation_pmid=16140417;citation_doi=10.1016/j.tig.2005.08.006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Linde;citation_author=S. Duggan;citation_author=M. Weber;citation_author=F. Horn;citation_author=P. Sieber;citation_title=Defining the transcriptomic landscape of Candida glabrata by RNA-Seq.;citation_volume=43;citation_year=2015;citation_issue=3;citation_pmid=25586221;citation_doi=10.1093/nar/gku1357" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Lohse;citation_author=A. M. Bolger;citation_author=A. Nagel;citation_author=A. R. Fernie;citation_author=J. E. Lunn;citation_title=RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics;citation_volume=40;citation_year=2012;citation_issue=Web Server issue;citation_pmid=22684630;citation_doi=10.1093/nar/gks540" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Journal of structural and functional genomics;citation_journal_abbrev=J Struct Funct Genomics;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The evolutionary demography of duplicate genes.;citation_pages=35-44;citation_volume=3;citation_year=2003;citation_issue=1–4;citation_pmid=12836683;citation_doi=10.1023/A:1022696612931" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Lynch;citation_author=V. Katju;citation_title=The altered evolutionary trajectories of gene duplicates.;citation_pages=544-549;citation_volume=20;citation_year=2004;citation_issue=11;citation_pmid=15475113;citation_doi=10.1016/j.tig.2004.09.001" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. Marcet-Houben;citation_author=T. Gabaldon;citation_title=Beyond the whole-genome duplication: phylogenetic evidence for an ancient interspecies hybridization in the Baker’s yeast lineage.;citation_pages=e1002220;citation_volume=13;citation_year=2015;citation_issue=8;citation_pmid=26252497;citation_doi=10.1371/journal.pbio.1002220" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Musso;citation_author=M. Costanzo;citation_author=M. Huangfu;citation_author=A. M. Smith;citation_author=J. Paw;citation_title=The extensive and condition-dependent nature of epistasis among whole-genome duplicates in yeast.;citation_volume=18;citation_year=2008;citation_issue=7;citation_pmid=18463300;citation_doi=10.1101/gr.076174.108" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=U. Ohler;citation_author=H. Niemann;citation_title=Identification and analysis of eukaryotic promoters: recent computational approaches.;citation_pages=56-60;citation_volume=17;citation_year=2001;citation_issue=2;citation_pmid=11173099;citation_doi=10.1016/S0168-9525(00)02174-0" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication.;citation_author=S. Ohno;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=S. Ohno;citation_title=Gene duplication and the uniqueness of vertebrate genomes circa 1970-1999.;citation_pages=517-522;citation_volume=10;citation_year=1999;citation_issue=5;citation_pmid=10597635;citation_doi=10.1006/scdb.1999.0332" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. P. Otto;citation_author=J. Whitton;citation_title=POLYPLOID INCIDENCE AND EVOLUTION;citation_pages=401-437;citation_volume=34;citation_year=2000;citation_issue=1;citation_pmid=11092833;citation_doi=10.1146/annurev.genet.34.1.401" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Pal;citation_author=B. Papp;citation_author=L. D. Hurst;citation_title=Highly Expressed Genes in Yeast Evolve Slowly;citation_pages=927-931;citation_volume=158;citation_year=2001;citation_issue=2;citation_pmid=11430355" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_issue=1;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. P. Rocha;citation_author=A. Danchin;citation_title=An analysis of determinants of amino acids substitution rates in bacterial proteins.;citation_volume=21;citation_year=2004;citation_issue=1;citation_pmid=14595100" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=M. X. Ruiz-Gonzalez;citation_author=M. A. Fares;citation_title=Coevolution analyses illuminate the dependencies between amino acid sites in the chaperonin system GroES-L.;citation_pages=156;citation_volume=13;citation_year=2013;citation_pmid=23875653;citation_doi=10.1186/1471-2148-13-156" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. M. Steinmetz;citation_author=C. Scharfe;citation_author=A. M. Deutschbauer;citation_author=D. Mokranjac;citation_author=Z. S. Herman;citation_title=Systematic screen for human disease genes in yeast.;citation_pages=400-404;citation_volume=31;citation_year=2002;citation_issue=4;citation_pmid=12134146;citation_doi=10.1038/ng929" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=S. Stern;citation_author=T. Dror;citation_author=E. Stolovicki;citation_author=N. Brenner;citation_author=E. Braun;citation_title=Genome-wide transcriptional plasticity underlies cellular adaptation to novel challenge;citation_pages=106-106;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17453047;citation_doi=10.1038/msb4100147" />
<meta name="citation_reference" content="citation_journal_title=IUBMB life;citation_journal_abbrev=IUBMB Life;citation_author=J. F. Storz;citation_author=J. C. Opazo;citation_author=F. G. Hoffmann;citation_title=Phylogenetic diversification of the globin gene superfamily in chordates.;citation_pages=313-322;citation_volume=63;citation_year=2011;citation_issue=5;citation_pmid=21557448;citation_doi=10.1002/iub.482" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=J. F. Storz;citation_author=J. C. Opazo;citation_author=F. G. Hoffmann;citation_title=Gene duplication, genome duplication, and the functional diversification of vertebrate globins.;citation_pages=469-478;citation_volume=66;citation_year=2013;citation_issue=2;citation_pmid=22846683;citation_doi=10.1016/j.ympev.2012.07.013" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. S. Taylor;citation_author=J. Raes;citation_title=Duplication and divergence: the evolution of new genes and old ideas.;citation_pages=615-643;citation_volume=38;citation_year=2004;citation_pmid=15568988;citation_doi=10.1146/annurev.genet.38.072902.092831" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_journal_abbrev=elife;citation_author=D. A. Thompson;citation_author=S. Roy;citation_author=M. Chan;citation_author=M. P. Styczynsky;citation_author=J. Pfiffner;citation_title=Evolutionary principles of modular gene regulation in yeasts;citation_pages=e00603-e00603;citation_volume=2;citation_year=2013;citation_issue=0;citation_pmid=23795289;citation_doi=10.7554/eLife.00603" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=B. VanderSluis;citation_author=J. Bellay;citation_author=G. Musso;citation_author=M. Costanzo;citation_author=B. Papp;citation_title=Genetic interactions reveal the evolutionary trajectories of duplicate genes;citation_pages=429-429;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=21081923;citation_doi=10.1038/msb.2010.82" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=R. A. Veitia;citation_title=Nonlinear effects in macromolecular assembly and dosage sensitivity.;citation_pages=19-25;citation_volume=220;citation_year=2003;citation_issue=1;citation_pmid=12453447;citation_doi=10.1006/jtbi.2003.3105" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=R. A. Veitia;citation_title=A sigmoidal transcriptional response: cooperativity, synergy and dosage effects.;citation_pages=149-170;citation_volume=78;citation_year=2003;citation_issue=1;citation_pmid=12620064;citation_doi=10.1017/S1464793102006036" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Wagner;citation_title=Decoupled evolution of coding region and mRNA expression patterns after gene duplication: implications for the neutralist-selectionist debate.;citation_volume=97;citation_year=2000;citation_issue=12;citation_pmid=10823904;citation_doi=10.1073/pnas.110147097" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Wagner;citation_title=Robustness against mutations in genetic networks of yeast.;citation_pages=355-361;citation_volume=24;citation_year=2000;citation_issue=4;citation_pmid=10742097;citation_doi=10.1038/74174" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=A. Wagner;citation_title=Robustness, evolvability, and neutrality.;citation_pages=1772-1778;citation_volume=579;citation_year=2005;citation_issue=8;citation_pmid=15763550;citation_doi=10.1016/j.febslet.2005.01.063" />
<meta name="citation_reference" content="citation_journal_title=Annals of the New York Academy of Sciences;citation_journal_abbrev=Ann. N. Y. Acad. Sci.;citation_author=Y. Wang;citation_author=X. Wang;citation_author=A. H. Paterson;citation_title=Genome and gene duplications and gene expression divergence: a view from plants.;citation_pages=1-14;citation_volume=1256;citation_year=2012;citation_pmid=22257007;citation_doi=10.1111/j.1749-6632.2011.06384.x" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=J. F. Wendel;citation_title=Genome evolution in polyploids.;citation_pages=225-249;citation_volume=42;citation_year=2000;citation_issue=1;citation_pmid=10688139;citation_doi=10.1023/A:1006392424384" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. O. Wilke;citation_author=D. A. Drummond;citation_title=Population genetics of translational robustness.;citation_volume=173;citation_year=2006;citation_issue=1;citation_pmid=16489231;citation_doi=10.1534/genetics.105.051300" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=K. H. Wolfe;citation_title=Origin of the yeast whole-genome duplication.;citation_pages=e1002221;citation_volume=13;citation_year=2015;citation_issue=8;citation_pmid=26252643;citation_doi=10.1371/journal.pbio.1002221" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. H. Wolfe;citation_author=D. C. Shields;citation_title=Molecular evidence for an ancient duplication of the entire yeast genome.;citation_pages=708-713;citation_volume=387;citation_year=1997;citation_issue=6634;citation_pmid=9192896;citation_doi=10.1038/42711" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_issue=8;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. H. Zhang;citation_author=D. J. Jhaveri;citation_author=V. M. Marshall;citation_author=D. C. Bauer;citation_author=J. Edson;citation_title=A comparative study of techniques for differential expression analysis on RNA-Seq data.;citation_pages=e103207;citation_volume=9;citation_year=2014;citation_issue=8;citation_pmid=25119138;citation_doi=10.1371/journal.pone.0103207" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Phenotypic Plasticity of Duplicated Genes in Saccharomyces cerevisiae and the Origin of Adaptations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="Gene and genome duplication are the major sources of biological innovations in plants and animals. Functional and transcriptional divergence between the copies after gene duplication has been considered the main driver of innovations . However, here we show that increased phenotypic plasticity after duplication plays a more major role than thought before in the origin of adaptations. We perform an exhaustive analysis of the transcriptional alterations of duplicated genes in the unicellular eukaryote Saccharomyces cerevisiae when challenged with five different environmental stresses. Analysis of the transcriptomes of yeast shows that gene duplication increases the transcriptional response to environmental changes, with duplicated genes exhibiting signatures of adaptive transcriptional patterns in response to stress. The mechanism of duplication matters, with whole-genome duplicates being more transcriptionally altered than small-scale duplicates. The predominant transcriptional pattern follows the classic theory of evolution by gene duplication; with one gene copy remaining unaltered under stress, while its sister copy presents large transcriptional plasticity and a prominent role in adaptation. Moreover, we find additional transcriptional profiles that are suggestive of neo- and subfunctionalization of duplicate gene copies. These patterns are strongly correlated with the functional dependencies and sequence divergence profiles of gene copies. We show that, unlike singletons, duplicates respond more specifically to stress, supporting the role of natural selection in the transcriptional plasticity of duplicates. Our results reveal the underlying transcriptional complexity of duplicated genes and its role in the origin of adaptations." />
<meta name="og-title" property="og:title" content="The Phenotypic Plasticity of Duplicated Genes in Saccharomyces cerevisiae and the Origin of Adaptations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/63" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gene and genome duplication are the major sources of biological innovations in plants and animals. Functional and transcriptional divergence between the copies after gene duplication has been considered the main driver of innovations . However, here we show that increased phenotypic plasticity after duplication plays a more major role than thought before in the origin of adaptations. We perform an exhaustive analysis of the transcriptional alterations of duplicated genes in the unicellular eukaryote Saccharomyces cerevisiae when challenged with five different environmental stresses. Analysis of the transcriptomes of yeast shows that gene duplication increases the transcriptional response to environmental changes, with duplicated genes exhibiting signatures of adaptive transcriptional patterns in response to stress. The mechanism of duplication matters, with whole-genome duplicates being more transcriptionally altered than small-scale duplicates. The predominant transcriptional pattern follows the classic theory of evolution by gene duplication; with one gene copy remaining unaltered under stress, while its sister copy presents large transcriptional plasticity and a prominent role in adaptation. Moreover, we find additional transcriptional profiles that are suggestive of neo- and subfunctionalization of duplicate gene copies. These patterns are strongly correlated with the functional dependencies and sequence divergence profiles of gene copies. We show that, unlike singletons, duplicates respond more specifically to stress, supporting the role of natural selection in the transcriptional plasticity of duplicates. Our results reveal the underlying transcriptional complexity of duplicated genes and its role in the origin of adaptations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
    <title>The Phenotypic Plasticity of Duplicated Genes in Saccharomyces cerevisiae and the Origin of Adaptations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EETO_Hlg1emitg_TIh7UZ-He5ZIpN9yzTPX5tQ2ZzZs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460525","apath":"\/ggg\/7\/1\/63.atom","pisa":"ggg;7\/1\/63","ac":{"\/ggg\/7\/1\/63.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/63.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/63"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/63"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/63.abstract":true,"\/content\/7\/1\/63":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228bP5TNtPq0duA7taEmdfByBMVVOz9JZXMAAJAMHRxEgI+lZ4f\\\\\\\/1CHEonTzN+MyZ57UOUhzO1Argu1U9KDGt34hbu0hROkAjkFqAMc8X3vNWzeB4bRo2AYQrEbQdFy4x6bflU\\\\\\\/opeYMjlK0raSZHQVoV\\\\\\\/8i+JHmQNZr39Hs5Wic8P9ya7pMrycQhy5r2pUea7Mo9XKCuDhEdC0R+2KymBpv8ilofYxTsg71kWhUHTOPNG1fx1TSMtPxkQnJYzpPNGDEXA3ibtEDPwraVQKR2uoP9qiaOpL+u4lNfp4DNp2ERi5nwvHPdhTA2UtCdXTM9SQ94EvVA10SufbLyVMB+rOEW3xW11uDadnE4YpFlu7QD8rKyXM17KtizR9n86tORr42dLDpLiC8gUAMPEHGP\\\\\\\/x1iv+hS+NsYCB5XmITIhdmB3AVe+dJyugKykOKEQdHHaO3WBrpkDYpFpH+tDRjD6y3vqCRsEkvDYgkJsGCRhCudHTE7Ne30QQCgVcYuobKuJOvz4Vp5H4Nj+1VZZVIqDqx2gn2zLd0GvCY5Eog\\\\\\\/4XJAN4KgHy6mMZ2\\\\\\\/J157\\\\\\\/XeHQvK9KgYwH2ddXyEjZaVsLiWhgct2YKo98e7BK0g6rRbZP2FbpQhlg1xqpHUNugTiM\\\\\\\/1nExe6si\\\\\\\/55GgGbmCPCpPXiiQik3ykKceGdBkstjUG1PbVDZz+4qxKH022E3LU5Rfmz0sxOB3rSsr5LSvIyuFrdgk4QO4gMLCXx7mpkmgejE7ub4EF1vI\\\\\\\/ehEYL0\\\\\\\/uYdjCStRv4sl371CMJFycuDBhVv4TWg2xJK\\\\\\\/+iFgE9PuGnJdy\\\\\\\/18g2vE7SSmeYmrJ709gLaYEuDn8Ml0Lj\\\\\\\/Y3j1obc484fN9gOCJLphxP9PUv2xCks67P3wjYttWkgnuNo2kAH3nrbgWkH7dAWKG1kR\\\\\\\/js8UC2Is6FlZ2E8JxC5OeE05ESksF6XvP2wuPtbqL9F+XaQE4gzJB89fNNSLsaeUrZn6F+n7\\\\\\\/OMPLhHVovgpDceCH63cFudX7r1NFIxgpUn60xcFaqYTvoY+6EY+3wRwZT0f4juw1vDbN3IBld62qK4NBByN8mxsOUSuchUT7RaoA7uNI7liGnlZgIeAcBaQaSWHVqQFHnI7eceVlKX3ubdYM46IVbZHTMnUMqxoP6iUEJ6gAoaxOTz\\\\\\\/mPQ8QOKaNLT0uH7z8RbDBg6YHymJCU+Tk3zleHGOKBQ2Z1yfo5P2BeDmPiaTjPy6dJHl\\\\\\\/cTqQGAlVBbCUBYl8POss4SNPgVPTSKCcWu2g8\\\\\\\/6IhNLFs\\\u0022,\\\u0022iv\\\u0022:\\\u0022Cl99LazxCl1XBQuzHRiXow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e6aa206a4676b71e5d84672a9881f9b281fb34398d4ba8a5dc223a123e51c661\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+9UsXRegZerch\\\\\\\/E+\\\\\\\/I7osfJbaOKS+pNgKFUlaV41gCSJ5rDQhKcWMSRH\\\\\\\/9lSf5WAaDeDpShCdasKtWGLfOJobo7Uvjk2XWq456KzeWdLJzVcHUQEvENlKwx3WkLjaTXAOAR8cHgjhOj9k2HbzHWP66O0MCBAm9lR3W3xmaAUFF79RyialgI8tlTkXMMa6wmlxFwfs2QJjh05JHJHVf8xBGGwZqaZq\\\\\\\/85YHYG\\\\\\\/t1v1R4JkeJrWDSacqlBSJs8TLmRIOcIya861kGlHW0FUGFBrHJ9lkXgwnbLb\\\\\\\/3jvGNANT+KJSRwKnQ\\\\\\\/ogjYBkjcTWiZfo3+g2QdKHP6xE32VvSLdVSv1eZFETj4BW37PTWZSg1mSJh15NdRb9zh0LnvfO00R07Sp0yDRb6neerAzTJP9Gg0pj4nyM\\\\\\\/oowGGdN5pPSQLs\\\\\\\/qb7teVd2voVgbZUX\\\\\\\/DBodZDrqZ43s2RJPsusLEPHo9ZEIPMmJZR+uSvFrD9\\\\\\\/lOiQsCu6IwA\\\\\\\/e4HUl2hgrqOjDBVty7OvC5lZrR7LyLmmtLPS6jeuCRrkjp3OpdxgE0LhoVw4HkewEuBhqwPRUPdtjgnCZbOSkyxYJPathpl1GceyKDG8UqI+mBeatKfHK118nMEsazeFtjsgRLlHN8Q4y4euUW7YpvHPo0hYUI4YCWJhC9nnq4\\\\\\\/so76qjDSIyQUL+mUTq+ht76dB03EKeJfdqoSPqPWw3TICyf4dumhs67qRoSpW5lTTCDUg0Dez8HSDAxIniwVr92NJiyrkjG78VPTZfXK11CDO1qp5KykXybpznxbKVgOxkfaEWpq25z56BqRVtAiBN5D1iI2vvAoyqBe2UvdbWLsRroyHDFEB\\\\\\\/AF2Vk9KRiRRtUl8UuxGWHifHRjXbNYVy+G3lnenb3GcFHOoPa\\\\\\\/\\\\\\\/6JEpyppZawy9LRDK190\\\\\\\/7rcN1OJhwKu2V\\\\\\\/s+rYGPIf\\\\\\\/Accttm+zNCC+byLIctjaQkm8nwzOxKAJ4J8q4SjfHLkRWw4AqhSv+tUu20JK7HHK1HrEHTiowPW1HEim3wjGt9KVl\\\\\\\/FXi4xsxu924YSO3TYOkwW8dtLfQP5kH0jC\\\\\\\/KmQRwYtl8aRrsA+WjxH\\\\\\\/TUVBUvpYswKN9vaKWlEL6LWW\\\\\\\/21lA6azZdVzBGSwA09h\\\\\\\/i51RWAZAxL9J1GenV9mbDDSPn36rGSNLCTlIE3nmflQXVGBpaF2mZlR1rsThfh\\\\\\\/O590mqPgIYDqocGuzc\\\\\\\/E14GlzuIDUhKKwhZzF1NtrRuU5cf6SVT3BVHQUoAXJSfUmPAz2sYlJ3Eyz+8MsHI2ENpNX55CiYg2j2+gQ6u7vXFCp6ChLdyaqdgjrZ4LSqD+HSJZwSdf9MKYXyV+81SZ0cgcqzH67M+gnOJEyscPlgohd+dm9Spp3HgrT1M7oSom1mOacmh481Cf4dbQam4ghZ68KX2EWj2hm3xZul2RhNluXzrhXgoBcJDSJarKdAnYTmCLkn\\\\\\\/6vcLQQnRU5+iFbRQlAGLIjIpMVA9E2JWy0o6R21+DRB25ovEpvCl2aC1\\\\\\\/90TxDCqyLz3FpUzK4+7VxDw1riPCqGSdBQjLzl+C\\\\\\\/wNMsnGLCzMmOUheQ0lutVybZBdpEVNuPCIRBZnGfV8FwiP+hM231UpZIGKnSZ\\\\\\\/riSiE6pYQA4IU3kIPbjTRQBwz5aAPc\\\\\\\/edQwYRjP5e6BO5k20ZkMqqm7EQ\\\\\\\/IL4dfj6bz7izIvTkRwdqouWt9hmZCyNcDWP8hfDEGTv+VSMRv2RVdVtx3wnQzySRIeLv\\\\\\\/39m+hh9+eqlqhd9i2S3vnIa4UFcMo3rn5u9i\\\\\\\/iUasIHjLiYWN26ym++u5+fmQNakgWT8QgXsib47gAcMlKS9Jy1MeljgGE3WGspzh\\\\\\\/QD0qUyZ4zvKSDXcxN1V42RvwadT9v+mhqu5CM+41532ZS8uM6BCKnz24SfzVB9tUmQqytzdbx4i+fRnJ2D9J6R6pB7GilPg+o9FnAQFzZBKO15\\\\\\\/ZFllrtcFyb4fZQPMPK7O3dW26w6mWpDf3ab7PRikoU3AqrytbmPcDtE1u+ymhoUJIEQD1jsrdNkwOCZBqiQL8aUivdfOmQlsbIsCORNCkbqPRtT1WS1Xw45NMesLlH0aYhYIZDHObd7LNi5gjSY3F4aovLWh0dwj3BK4d+2LXvPtycJQb+AW0cX6nOQLwi4RUzjIt6frPOYryqXob1RkiwrbsE3idPOjtQc7NVWKXSLD7e+3p+BcL3vtp3ndisajZz4\\\\\\\/c6XoJl\\\\\\\/YBvT8fwQKpCqDcmkhkwHSqSciUK9CDtmpRmB+7q4UMVXNpgpt6ti2UcfJFU1Ua80E6zbKX3BLn2qEchNOwWg7dn6kw8oqs+Xi0rr2vJx7Vtx8FG1dbBWwddOZMCCw8DDVUz\\\\\\\/ZVooBlRlZqqhOUPPfQ8XVfs4VB84EdC\\\\\\\/kJ7vUskuiD08E2\\\\\\\/HKP73m\\\\\\\/\\\\\\\/gQ4SIAKU2Ge10GfZ6P80o2zn7\\\\\\\/FL5Kh7SXcppN0OhXMSxY6\\\\\\\/+ClnRx9xTyVkRIjT9tC+WlflYAMqVxdviNWe3eijHIariYzdRnoiJkQ82idm4K3GzJO91IllKuZUuvXpRSf1DbeRtm2yHzwvKQNSmHVKzm9CYiUTQEU5oUyNQcVcXnlSvxe8XbTnog0eMeIfktzlZUY042hORaCI5yJzF464y40y5tCecI4PKW4Whdj1QIYYyVGymRNCMFIuaVM7QK+A2Y2CG6rF8aLChue7kgHLZxgVEIIqz5iQACciiuz7sVagToB5QDNkxoJZsi3gFZOFg11tCwHGyNmW83syNqC+rm+tBvO4QBMOhyfI7WmthB4QBxFABjYWx6F5Wb9obmdFPNCLgYVtvfR4+8KDkz+2YlzYRJB2Hz3Wo74j78VnChWqpnMxd4FgWrQehEAKbtZXnjNGzqXSqeIDb\\\\\\\/unaKqfg7l9alGKFjURlqK80xwPcXYpknKV+bNs82hOA8Tb67AS7i0xySsdvpy74RtJS9T\\\\\\\/5rtEOIZQWk4s2ml\\\\\\\/RE0cwbdv1ggOLYafkHcA9u7Yij5TWOVUsdp38y2vHMHh+txyD3YhCtzuQHufTZXvO8WAVTuOvyOnjc5S0fN4fFxSwbv9JTQW5INRZ+mznaHHR4+0tbMx+qDUS3czqceOifKzEh6siUv+Li\\\\\\\/MFAB4r5Di3SMbECc2mj8vGgj0xiYd8CDE0LAvJnEHLQM26hwLU5ufaeIPPgGzuTAHApBOuo3cGUwezI4fzXrodJyeaJ5hi3tEVgG0OnKQ9J39S0CVZfpTnqfuZy+VH7neZda1LVTHNogQ3LhudQpMdepgNKDGLhwNBW7LftyKXrVw9jzLh4LUW1AHzpPJLBVoZbBrt46JD4Fw3H0OXM3sg\\\\\\\/VhsksCjVRAWK+7D7FtCelhH3DY8E9dxZtZ\\\\\\\/cqpjaraf3+D6L+lzErpfD7r\\\\\\\/QGyMul6mZqLBvmyjTj\\\\\\\/ovyMzGmZgXtgjOuL\\\\\\\/lJCheAEB3YL8mdzU2IFHDWYXWbnxqdtcfv4hljs8BnNGKN1dS3ecu0zwqrbgwQBKCDVekWKNTFojw140X5PWoBBIcThGWi04nKr+JlQ0J5vo8x1vtRrc2nyO5BRvy20cHPmINKwXy+G+DY723pczJSuDYB91+wlmYv74BxGLN3Je0cg10z\\\\\\\/7KAguCyhV1gVyFH8lV7oQAkcEpSzHZD9X697Elirfm5PloVeJzCYKIzP1HCTZHu+D8wSxCZ24\\\\\\\/cRu5GESDWWuz1wjbTQ79ROYMMzBy8Fii9HiHjeY0z6DrC8PS6\\\\\\\/N8fNYg3FcinlhamCL4NgZn8+Cd2EkCnmAfV1wbrxXIxHItUCHYA4n5awz79pWyq138+5mAC4QxLig4w2\\\\\\\/NfXXqUSEtRFZlXgtzSphZM9dPagrWabSVde28AU0R8zkwGMUkTuc8I8dEUkRowAgdvsdNOSf8cvys+j8lS+JSUbB0U5sTxdUyt+M6S3Of1pSKB2yhw2NjW0L7YK0FOhRs0YiOz9guWZkJdxwhWnpFP2AoWL6fZh1g8fOWWvmelF7NlRNhCchUhSQq1\\\\\\\/jbka+hE76+J14QNOyyOZjmYQyBjfP9jgELkY6zBx3pFFLQUR\\\\\\\/vrGoHEXN+5DGzsDx8\\\\\\\/BaKBKOg9Fgp9Zg1g2biFKth0DfYBy7oQw4YHcxfuyNo58ywSxTbFC8ADRw247wNko4DvQC+Ym0Mk4qY3vXOqkQNoOlb9+orDpd0bckh7EJAecukjX8+YMqkQVvAM6fwYeXGtxeFvFRFtFsNa6qvEyV4EQMaUTlr600Gaw4Y98uzXPn\\\\\\\/EmeMyvLQO\\\\\\\/vgKtBLsYdcHfqBPLeQUbIDuhMUZHrdC9flPnB7X\\\\\\\/\\\\\\\/u5RFyZAzIvsBT4Xa9OG2e2hthXX+C4FzdYu10KcJ7Cwlit7SQQI09DFC56H9tkcHyR7YCzhKbOUnS1wBYqa5UjrlpHt\\\\\\\/VzpE3glymuL8\\\\\\\/Zw+lkC8EJACVCeRARfMnfD90JKzKJLgwZVM6VD\\\\\\\/WxOX0GClBjTAJIMGTY5wGqf7+nzwcaFdqtD+bjr1qGzxJ5yJR6HmLhbEanQzQH9lwGopdZyAiYLoMlIR6Vyxfq2Ic8QR2P4elitNQEBKdwzO5xycUuG81LKdXdF4YoQwtcFYBwObPEsOBgLwnNAvpeo9ishXCIZvgk2HSS+BnFY1TdlP4Oyc+QbN\\\\\\\/co4TcBTPyCrOAoeUjBWBDre40xxAsceE6fE\\\\\\\/43lfIvfb\\\\\\\/9qB5CsYYl28Mnm9olKg\\\\\\\/lxUfJ\\\\\\\/uMwy2keucrKhZaX\\\\\\\/ZcQaE9wgy+CEDumkbSw5fN2xT2FJVIJy0oaQFfzFlIfiXlRvk08i0+zXslJNFbWCGRcF9QK\\\\\\\/tSuE8i04BbH+gvzotmrQyk69mZZWK1F2vJWsf5gyy1BLhkMfTxUxiHTBa43eokUuQK\\\\\\\/5tOBmWowkzjRP8J72HraZWqcfLphK+l9ldf\\\\\\\/l+mr8ywhs6bgrnVZ0eMC102Nc3FVxje+kUQDsM3rxziWWSW+jXTS6mXncHcIZJtessVvELeRlICGjcn8gcaESHUgZ\\\\\\\/T3aVTzSoETpRni45Fkf2bsrUhJ\\\\\\\/Zahhnj7DUejL+zF0dOSJ\\\\\\\/d8VEj+yOhAks7P355HgzsFkqU5Yv8tUNE8zIx5EJgtT2PkI9zBvHhtSnvXDjs9lDEz9ftrMec4pSR94oIkX4EI49BZ5LyEGjY8t+aSwBsb0wA00Mexn1ywbKc\\\\\\\/rDHHOff4mRho0bHdc5dxMWbvr7ftjpJNL7KN7g1ujBbUjcoa63Sll9K3oNDtuQvkO+kB2c6WBIX+JT5jbPMfKj1cKczLKydwgRvxgTdUIr6kLe73jPwDFvWQ\\\\\\\/G1Nr7mDjT9eq7r7r\\\\\\\/bsiNOjJwFhd+G\\\\\\\/mYWH2HfzrrhuU2ZF31JHqNvre9vGWw6hktNDBWchUNewMpVG2xeLQ\\\\\\\/bkrYJuHXYNTXYQpbXSuR5qaKnzCzPa1ItzV7hS3hSIcY4uoZNM1ingsVJled7m5WpoAdyVuDMFwRmZBNnPuhoJtZD4JGQQ4r3lS1xCMffiUON0I6H7gBhs1TzoMbZVXFrf6Xo5luHw1TfFiSu\\\\\\\/SnWL6CpayNeSLM+uT3Qm1TRpdXpBln+Tlr\\\\\\\/smRQJQ\\\\\\\/8U54XVrott4CRpk9bUKMwLYQ++GyDY8m2Awm5KbXO1YI+eEBPPj2rTrGYsMVCEkUFDqOf03UTP0myRlxhYP0tNGmuAl\\\\\\\/S0kJ+JiiZQnnEwRgUa920i8g7OpdUrJu9mvmNW5MorE4G6k31RbDIbwThmDK5DKwsPuSj26Iai7TJW8v9o2V5DtTfxQWgYbEwH4Mi7+BQMVNNUN54XzOq9JHBngg+J1dAi\\\\\\\/ORS\\\\\\\/NF63qyxLPLdEGDS6K4ZbFyyEdQPGhh\\\\\\\/\\\\\\\/aBPM33T9SNo0mw5Qqh5NlVEMbN8X7aVdMBxrF1c37rGZjeyr3galI50EGs8VAqYaRIe5B2nidxdY6A9tZ3B28XeABBN\\\\\\\/yWp\\\\\\\/cgoEieSn1c0QoV0+6UQeAfFLW9VuKE004hKTJnZscBLfliWlAP7+Z3qZkSNRcjDJkB3okNI0Kiu3Xywjjk1NVaxfw+UVp\\\\\\\/4r5W2sujR3iZtpgfwuhxT0YAUrIoUVmNbXdlgAwIvAB4GVOS2Fdbg0SwaHXARSiH4UOd8QuQmi2cZgeBSoB+P94ingRoy3ZERo8hWnBefjOHCHMbnChCchCAVqSusSMikcjN8isCLYucXZOqIepKNbC7Tn3CjdDNp\\\\\\\/Hpi5QCycnEJIdOOAiJDgdn4IPXJDRVLa8frjyu\\\\\\\/e3GSH8HkFAtjH\\\\\\\/D3B0lCI54ZqDEP8ddTH+iEdd8p9NPYW6qihvtTQw58JkaBBO9bgIqDbE7EpzNmqSu0NJKD2brTTwaQIxItV6Fp5ngBBK\\\\\\\/KvBxxURqKWr\\\\\\\/ig6zyq6KJdz2rqDDRZMo4WWSRf22R+bLB9yzEvQhH19GL6yuyVIK+4TrT+UV6yyInwHWkAcWEdcfFH055F9WXqLu0GeTS\\\\\\\/btSLGpBwW43WjIm1bUykIa3f1wnqteJj1cWeXPy8QA7ygJ11EOeBil1\\\\\\\/xKIyyegyxpB3+u4ODBGZzM8oHAgbBBWcGzi0bmQLhUH5dNb0ZGtYeCZ6WlQ\\\\\\\/wgEZ9crQDWanFnkxKVoe0egUFnUtRUmWWKkeICSn8u5jO\\\\\\\/\\\\\\\/f8IeGA6Y6ecawPewQ\\\\\\\/7R1QkDPped017fxFb66+bnbaVuyss7S6yJTej\\\\\\\/dKuA\\\\\\\/vWzyfyZcX5yXHWVl4Xswr791ihXk97BWv2LuxGBHxhyJfFBS7V0wetXQpYiUL9FZimmnw9YG8xFFeTDnmoe1u3U4CsSx3DisihnjwRBEHF\\\\\\\/iF7x+j\\\\\\\/5nvH86Qw5hXDqN6cEFgXD7RIu2DJrLmLh4HTs25xOAhtY2qM6y72c4X4b90R+aKPUobU8TDrNIhjGA95fZ+luByEwZY8mZN\\\\\\\/rxjqxtI6E6q+QipuqsrjEkNkEz5cVOzKMU9xi8GQY1Ba20fdYxz0IvW1SmK0OmV3nx2xGqqhza4NjoKVo48W0QrX15pS9+G7oOvDS38kFaiHac8YFfM5Ax1zuOF3\\\\\\\/aTmkzlC\\\\\\\/LAwdFYTEMUXoAORfMKjaT0HMvqdj8i\\\\\\\/28xE1oOkIgVa0N7SpNPvJYERRvBjSDP+AgbQ3xmWt3NwPMtmsBtJ5Z3QESfOiqMrwtzcgvslzC1If0TNOLvT6Gj+2BofPb9\\\\\\\/l2xWAIcykOnp8xRW7MZfCsCIXjl9ywE+rIXHouwKYruNJcihB1LmtJ2A8apnid4tyJrzrwm2Fqa1XVxz0QCcTHlbWBcUGi3+G5dPX4Wtpt0VLSx5HA\\\\\\\/KYQ78tUjavIa+2U+wSJMRRIWDb79Ti77Km04Yl2NQwKvMzAWNcFX2+I7fwU3myhqGZ3wgFJHGrGGLSraI4uLk5FviMpn5XY4yvBFKFyfkdNsLcsluWwI\\\\\\\/dqYwy\\\\\\\/Vsdw1oaM5VN9RvWouC+byEop6dOPff+xsUEiQrJ2DxTRc7UtjnhS+o5wRZpzK5GrEuuUhGldbK7ZMcnfsSdKt20svZMzVK3WdJ+k7kj3miVzT\\\\\\\/X6XHPMfNsh66LOQQXPG\\\\\\\/HHQFlxRJjUeZXJn+uV8OL+ZISoYSvFwOVVSLtuXs3\\\\\\\/o8dgVgJT+LSBr0Ncj0nEb7R8P3YBCrEHbO\\\\\\\/+Eb5cnR8\\\\\\\/VJbJrAYm5Ba0wq2zR4l\\\\\\\/F2mqwZx1w9ZRSzOYvVmudjN\\\\\\\/ATpRfz6wC6cwZ9cKJjHMQzQfVme05j8mJjVIDCX\\\\\\\/a6W90uFcbjvwGuJtK4\\\\\\\/gauqjP8gJh2fipU1aDnn1plkQdQA7WyJp5TlyPBs1M3RDcWnD1+249ujs87i4ksZrRQPE5+M0bvB\\\\\\\/s47xheDL2iGFQItHej3DpJDtB3xDJj6ce20EHr+ulakLyKkrgbm1+NpkHJ0seUDWCwSR1RRA2rQcldeJ6LfHjrWA3nqHTsE1Uns42vh9vc0XkCM\\\\\\\/u91GhrCXSgW1lysrdQq3GTpqN+sJBYUChdPgF8hBm13YOtY9K+GBONjWMdbtMGw3TTN64kh3MUlFdAI0Xoi8hqb5YvWUVZPOfbWttbM0ZYjYGykrr+kgM9TIFzCm2vwQ08yor9xh+C61GN812va1I+exJIIKAh2Us9X4Eaul3OvvYDO6gGoADcV3zMar41bJMpFj2zU1wuS5RLOEd0tnYigw005usjwfwDRPZ7+uJ1J25fnsl9wyryufkh8wA2WHfm9uDS8flSbMb2PR+QLrnLkiORpgDa3m1cho9aZn4tT9fLBH694BnS9ZtKF4yjDhl+9WtvO\\\\\\\/iaIJJvfR2WAXzkQrni00VDTIAJE\\\\\\\/T4B8loE3x3jMlYyigQbgPeEXhxqBDwjEqu\\\\\\\/SEtoEAVBkqdtTpueaLA341Zenj9BopXY+1qqk+uKvYc4FTh47iN9hx+M2eFR5Bgzs3DlzdCqAewet1vzy4u1iS5xDsZ0HjwEm5lKIzRGlOpG\\\\\\\/hwCtfPjo7cP8JSoE9b1mh2rLIgGs3irDhX4W2DGih36VCUgmuywXhNRc1cuWVsy6uM1oOzrYNUSoHWE42aJ+lPKXo1ETplx8z1VNy5PgbFOfQCiOBHJcZ\\\\\\\/f\\\\\\\/x7PSALSLAKAuKhA4FAB8JI5EXXUn6o9IXpfJ2XSXtjQh9E3cuyEBmBMvKbXXtqF\\\\\\\/NxetyDSXtkrKtErSgNEHC0PIgtc50klEl1\\\\\\\/O2Me+0XsLduuuMwa9xGcX7F4S+1kd44hO23P4Lym+Sp5v13eKz\\\\\\\/VwWDi2KIQCrZr8dkKqM6KWkBODYd3UrdcBbqHLpOkMUGArFhg3oVhsbOEllVkiu50fgRrODIm2cdn5Q23G\\\\\\\/vWrp9ViXzt+meHF2+MJwd5uBpcoC2kg92YUUONW0XOgmNdD1odukzjs\\\\\\\/XE0pfvCYWkMTkGQuIk\\\\\\\/qwJxf4ZivWiV\\\\\\\/zkk\\\\\\\/mVrougI6Sx1lvbMoSbfexzSVgER\\\\\\\/XdtREyLHaiagzcuci6QQwzHuGCuDICpIrnP4kMRpiWjBc5WluIu8M6sVU62KsQD\\\\\\\/YI5v9QegSEJE6a7Dr3rtcYjcWf+7AAJdlA7xU2UFoSqpONUA3zSpQ16QYihvbVjHn1Dz2hcdw4XtGXAaHxAI2EZg+NL5jC2ZRbINAdIgZZ0q5pOmw794INpVBCYKjHNqFBKRb4qtiBGeECSm4oA00IZdVkaPL3N5lYzcjIadZnxSZICvJn0dXo3kVK\\\\\\\/8Mvx4UEw1+JiccjofrSMQxAIBF2isNtR7SRaIyvFoA2B4p7tqOPjSM6JJwY\\\\\\\/crOq4bPi+pH0gpV8WTA1nY9E\\\\\\\/77jlHWA+aa8sHGWWDO1JZ2KMumji+71\\\\\\\/bXeLbz6zLe7gvdTHuSNkXxBV0xkI8DeB5elOhhlcWFmu5ofz6Sefjt9jclH4cvH66nvV6AD2gCgVo3CW5PxbCCQX9bmGE7XIvnH00VWdHJqK5kAY\\\\\\\/Q+RuhnIXr3OfrXKRk+EU4yHjF3O9uUDYlORhfqoycdOtqo5y4y7DH3rpOmwjmc0B8TJXwte4njsjlmto1Tr1zwZZ1A8FZWDGXWWCTJ1CU5cQu0x5ou8DvbvJZGOjck0bTSScW0\\\\\\\/HW2uC3YYKDFJSUCELXOSuNXiRbKki4CbL2MdcvDFRe7Wq86dqouu9esIo6gzf8vK\\\\\\\/bvlw8BnK1D6zOUQqiBdxKzEhD6BX931ovMpJHMuRK2hdwzshMaG1DId0D43x+LESuc7s49SQ3b2qgDGHvvYELEoWbDR\\\\\\\/kzOjz\\\\\\\/tmSASloPM8vWQQUlWn9d3mN\\\\\\\/xh9S0Nsp\\\\\\\/XxG9tRR+A0IHdim91\\\\\\\/sKRtvVm9pr7t9YpeXDJDk284RjehJuKx6YmXDhoU5R+mHhvG1Y462esLIsQ24KlLxrMAXQrz+AzayOOz1wRusut798M9XzOwYNoQAzHe3e69OGp5jfGlUMiiKdBT\\\\\\\/8EIarrEDbS72NuYdCp8U2B6mOX8x56pteVyhohWvULlnaHUFB\\\\\\\/Q7i2gew3Gv2yuQjdqD1pQp9KSPg9qArc0PGbdPz2mrXP8Hvau9+jOtK+VwGURmm\\\\\\\/7Ku+IKSaxUpeVgC3RPY+PddgS3hoachaatL7ZEDt1NawIpv+WOgTYHc2+38wzCfYgbs4KcZsPBT5muSZ6uy\\\\\\\/WIyuzpi65dH50CwMMmxQnqwyFnqHvMpV9pqyP4aczmgKlKTAHQC6rRgSMHLwWA3azEUTMsQyp9hTUwNRQNDohlsZPDavZwFo9TCRvf7MbuAP8NCNmxIsAE9ilmBIpo4Rw8PQru1Lr1\\\\\\\/xhs8UI3An0rO9aVjKdO0DWmRgM4JDe3jDbcKtWr\\\\\\\/5BeNYnye7xEC8jNkGyIJTXPbvXzUO5irub1oYNg7feGky5O0JEUCq2schItFOblctzsaj+yqmbH9NpWjCSuO7Xhxy2PD4lXTDAPCPrhP8Zz3Dcv08ZqTwT+F7k7xqu1nzLOKIWYBdqqeLNiY7WhLz8dJ\\\\\\\/oPEKXNkvkArbVzz78HXRfSA7r2KfE0lZZZ6mrulpJQAxn9yh+4JB8c6km7eftS6IHLk0a4q4arXCwitrmJbmDES7Ze9jINh2R8C\\\\\\\/pX2yp\\\\\\\/Fwu0xQ1pa3XUQVk7MI\\\\\\\/VynxhLZChAFQbXCPsE3MCdAIIrj0O3kTMj0baPre3+HnIxpQWX4+dMJJ+2DXxAYDlIrXIpMHxIyr5RSUK4cpjmcIqkw0fJZfYIYPKdXsc5IxpfJdzOH8GNL\\\\\\\/u9eRoFT6aWU\\\\\\\/A+PYlXPn9UTNe2OGYygZrtQgAlzkUkRfiUoU8jfdQpwzkZrX3b57zGoyeSg07KSZRUMeSwWUCcfIq\\\\\\\/C5SJ\\\\\\\/HiAVxawJIPVAW25Ftia3ATa0\\\\\\\/rwyuhd4jlzgeVm7Fa\\\\\\\/U1QYRyqNUgmCYcoEP\\\\\\\/7gNI3BQSWqTv57+iKfp0UA8Sm2P5zQ9BP\\\\\\\/TTVsXnxPSmiDKdh8eYdiNLlx+dtb2TBjePXn0PfQBXIp1caFXnrd3ZjUb3MDmEsu2UNpyLCXA3xHsLDez176NqSOL0sYDL9pfcBONF5E5HDdGInVzzYQWDsnCT\\\\\\\/EcZpq+gPqBph0BPdItfp4Ww9cHs1NauSR64GulvGBRbJONyY0DH8Je2zEI9j9\\\\\\\/up1xCrlR+4L5nWXLzLtk5l9ElFgAI4latU+bXo9nzKYL4CgMrEIWud67KTdHu14Hvtm4iosPqONySkxKliEg7GSMFqfLV9XwHjYNaIsnKL6\\\\\\\/tHEsFAfWRHqzCzCQCFFFDKfOKOTj1uFY9DKvrKBQM8UQr9BAtNW0jJwMkOUseeza54hhWoaEgnwb1wYruR0sSPuCoIf7CpbVAEj5ZzKRWZMhYUM2FghpeSa3aYb6IUn01FF34iqzX5u3ugffE83p9t75rYptFkDnp3tZ3LO1fcJ1CShkfmIBh3UW6kuQ0IARwnKgFk\\\\\\\/i4NmgQpBK\\\\\\\/Qxp4rlV219bTA+sqRUkTLoyOBS21H0HIIVEGzHBucprxUgDL1Za8EjXObGYfE7oPPhcPoBcRjHVV56r6P5hYoO6nvE00BFBhDWZsylc1+sV9zNIhL4n8xM0rO3a422nruWwfDqYKZORyOCr+o\\\\\\\/JAJIWmxuKnx7290qxZTZKtro6jgVajdIFuj620lZK+3HWBFUEB0dl9+Ts8znZCW7p5Rij3SMbEs4Z+KHDy9kmPLHrT19B8jye+phT49wuDlaBsk2ktL182RZ++Iy+Y6TynT1lXqpn20RZTNPP1WvcUkXOVydIaAZW6NrW9rz5uTrD+4V99ML4E6nnxW7VU+BmrPlRGoyYdi1i3vBHbmaJimhmzP4BbrNPKsM4X2ux+BY6Y\\\\\\\/lngqgOhbR1rdE0zpFYneC6Sa4jZc7FEN50l6uTcSdilsaJudLK9ukFOFGWJh95Ks2FD13oU2TG7DsB2pqVyEFfoux9x7MK3+2fXYe2Nb+4PI\\\\\\\/BCWyZHCszxOQLjGFB7kvPq0PfwhcGdiKWkRiXyCoU1stCnJtX5QE+JvPNXO2kxLQXltOcJcThBuOgtUZTm5Lwt9F4xN1mqmFarCdUnLu\\\\\\\/FbuETKpErlHochtEqIy8XCi1BZJK3hDs\\\\\\\/z2bkumnRcKmTzFoBnGsX4QZTY6gJhAiUu0sAkKJt82mp6KFzPY5aw7OGfcfUQN1n7Vpild6VDpvCwMYkcfgaY1lQlgutJ1ppD9r\\\\\\\/fy65GUKDF0v\\\\\\\/PWu\\\\\\\/Zl5J0p90Fy9oP5tMAixVOL9mVWwz1eojv+jffNUo0YJa42XcsVlaaaSvRvUD5FXs6mLSzqF\\\\\\\/xSWzP6Bgx3FfzjDj6jVz2ByLwIFn9\\\\\\\/XbFzgkyDLo2kW2LBoluCZuj+pqeO2OYm0HZAy\\\\\\\/EIfuoBGvZ1nwijiSx+DjdE5afRnznbdOXDTsBfamj8ym3gQG4jQxmVPNuYdTgAFi8DKUkebNgGx2Y7+xzX2V9DaijSaokTf1KVI0bJND0QWoGB+9Tq9iNM4JJpuluRPs6J+AYeRo+Ybg+HIjd28ciSZnaLxI5JYkX2FoVJ0M2f7W5xu3KiXlzE+LhYfQJiGBQrXZzFqR2qY25mgH78pzB1jAqbootzF8BQpEd1y8gjYouX\\\\\\\/hZI2O7F\\\\\\\/5tYxyr5N5YvZUjHOzqvKAap3qmMHfKKfbpG62ECCB4k1CRfEFcDv5s8JgBhztpubTV6ZELH6S\\\\\\\/G9xAPYvhQKriM10lXuXsXB3fj\\\\\\\/dSUVOaflBqyhBiYLQf6J6LSDLriO0iqNqPr3Q\\\\\\\/PCrX5ymH7EOz+gxP\\\\\\\/oWbdIHZExE+p\\\\\\\/UFEOVIMiFetdkzbxKMj\\\\\\\/thMhfkTgx9lO8EJb8z5yIjIqh5MITUQzW3h+LvZ6MVho4Y7ztup++KnjICB\\\\\\\/00b1R863sxLkVCtGJVKSR39pqHZHdDyBT0MbB8NzqhuvHDdgDBRpLxogYKEsr\\\\\\\/Sovptrn7uIPfKqn\\\\\\\/oaogqnLd9lPGay7WpsvxJYXCUGZFbLJxNyOhDUs7KP9xGF2TSzxNooQbE0zwdLQjmxRoKnyYHMWTK50JreBuu2f9A2QjZPo1F\\\\\\\/J\\\\\\\/fF2YMhhL4Ch7MOIaPtWHg3VKg7jQSSqIRMkpTeooTL3MgF+v9XsUnBtt5vExFT8ixqlxb7BXOUS3dkEybuaMewgF7aQl+lqQptAf667oHWvvfdKwicuIlGHHGsP\\\\\\\/mk2DYuLAnqd27l6dn7a+zRrsdrzRjJoM70YTTDYh+e11UCzHLjOH38gOdSeme3giroLuLoKS5L03OZyDJNlSaVGA4g5GdynlivnnwM\\\\\\\/8ArfdsYFPLkiXQqfhrV3gRPe+vTzPzn65G8HNmhwi4sFNR\\\\\\\/PxuVOh+\\\\\\\/0Xf+ThVBj8JOgt3RF+uzKbD6PSR17zDWysVqs+mpHCdKunSwUAzC2DoTSdkkUSYPdODTIR3cZ8btfuXWKTrWyxDj\\\\\\\/n467IUKaPCUGnzIOsUYod9Es\\\\\\\/z9j+eGtnKw4LuZyIyz\\\\\\\/YZ+el9GRp4+a9VDRyfLBkG5z9aNNcPL0VRkchf44l0IM73hY04RkEnhDw\\\\\\\/JphmpqwGDaRWGD8kjHFeipFgh2atBuRQGS159XK0RnG+kge+GRTbq1BrETk0Vvj9gusbFQRBFmPcUIqq5CGXTgtSKJKarOhfYfT2fY57Or9gkGKtOaYG2MasuExUKydl6eG2Tz\\\\\\\/mItg\\\\\\\/hct6uiCjHOYAKuhqvlF05\\\\\\\/\\\\\\\/d\\\\\\\/cRIVl2fzIc5an6gjyWwIUj\\\\\\\/aABuoxC05pezggrA2WIu8BhzY37gsWmzJEFn7kl98O8YW3YabrVWcptmgGji8KNwC8YfTvG9wnfmKxvP+VHQ+1XOJk6p\\\\\\\/QSlFFELoVdZsck\\\\\\\/\\\\\\\/5RhkD\\\\\\\/i+6iEBw3\\\\\\\/NnJJaTXwJensi3u4DFc\\\\\\\/0sM3FyRNhprv9wcJsNkNLYRQDDpUfAxfXcSapyK4bbs+RV\\\\\\\/tm6OrMiJA\\\\\\\/WmmulFlOcR364zMDNuUBeAaJ755QiSkv1TJMC0MXVIcZdUEUz3S6KZItZUzGLMi4oWJtNET4lS2Ny+\\\\\\\/ai2mUJcMkMM1vs1IpJ+UbmA46hliX3gKBXB74UMuRT+ySAfyiMOPQ8\\\\\\\/3lxqHVWit6MAjPZ7iICRn\\\\\\\/4co0nb\\\\\\\/32LNXtliUPQozPOo0AYYiT8x0DlWOalXLB7bEtpdSHbbbUjdAiyQoUMmN7\\\\\\\/l0zqc6si3EgyZ0060ooFfVZe4OXk9DFyEGtVMjvXO1eVyTQ0PE4VE9l7VD1l79a90qM7D6DyMjh+rS9FNaHoYdTg8IP4985l4ZetlE6bA\\\\\\\/za+YpObg6CfjM7te1HpoDUi0KWauVFetKJD0zfYoA8muPZ4sV3qeR5MrRW31P6Riqk0AeUYUFx1X0aOFQzCLUVzbPgtyxdMZODTtf\\\\\\\/4xiInbye1zHudOImyvC4V3X2G3UlTrEsRZkbBeeGtHT9fuu8yhpE0zCUx3hAecdrEsA7BWJ1rO+VATphG5bFVuNqYBX2Vq8NKVkXzWSjGqWJsM+sdbCa094ZPUKU3Wt1ZP7GT8YlA3ePJGYhfg1uq7DM+X9c9fjPAdGZ9zlahypzY3qmYgzG647gGEa8\\\\\\\/1rWAfPRL9zSrIiH2YbPC+zCCvkk8R86xmfHAppCVRCToD3WsFRAGxldq+PwUmRd+NRsGZms9CIg4vFnrcPEC92\\\\\\\/2MJo471b78Ap7lB\\\\\\\/vJPMhpOHjUgj69NLPvHb1iT7grmOtyD6Zx4z8JEqpbT0KwwArqMQmrsEXnHGAqlBQj7NT\\\\\\\/YVoDAWfZP6zItkRhE2owKP1sRKAsRJ\\\\\\\/O9WfPIApzmX+k0BuroXGvbomqNUB42j8z4s36yJK9YlWfhZ8ZrTd+szi54fYls+Zm6Yza\\\\\\\/nVKbDLp6mfOS82qEYtcSofhPExHikuF4SwY3Ze1pvY6dhXH7e8fcSNxrOJ+xdNjR\\\\\\\/VO4SxIvjE+nbyi1EY1IRRIAmpSo6MLD6gRoUv8oCFzy0UR0hKk3TBIPtHsuwO+glHPdEPVVFWuziR9T6u9t0UfpCnANJouyXTT7Y1lZcJTY9f4zVews3HEtbySK4VA7E41tAA+cOTUkUwZXscvkcWvGc5\\\\\\\/EV42odpatM+J7Qo6noqT+sPXcUcjCaP6En1KuM14ANPtpxW4ZR\\\\\\\/4E\\\\\\\/rCHwEkIdbAPlX\\\\\\\/rciEvCFaFf21Nvh1QWH3ePufz8hAXc1OsNLQb+vntXHPC71XbsvJdg85D2HSkluvlWlAdqv8gF3Lhnl2IB303db2gCnMBJR43HQ2Xsd8OkbIUG41BDjPoD67WPxLoZRs555cLboIIzoTD46u3hTG5IijRzsvI5rqUkVb3akgSmgMq2LJvgMFpvW3NiOd9q2mkl+VWxa0PHGZk6sVMbnvdGetqzI2s1Dx7w\\\\\\\/D98F1tMkPKnZ80EgDCMMBfr0idM2QZ0uGV\\\\\\\/fqsCawRr8\\\\\\\/4ORpctvanD9nxwvubVy2+BSVTofDAiN0O7Am5KY5B0qiJz2RtaPzTef2NLUqVn\\\\\\\/aysb9TEK9mrOCiBr6AiJmoUe0pHXYEF0Vor4UszwCKFvXzoZrznDaFX8CSpSsXmKiabS\\\\\\\/wYRIN1Cc+qkTvGW7L2umPU1Rhb4RK\\\\\\\/3CvF1wsKRLkEBal5Gnt7q8OvTdcHmAnklsDQKQbqoSIyrxyvVIxMa8GmvuaObhQQ84kvP4wkdUAEu+UiYPSdL7fijxfy4hnNnYA+xiNca6Vy6knp4whzX101mNhpmL3qi+DiJpLhjpOq6yBmEr2hgSCPCEouIQnbvDmkd2vMD\\\\\\\/W4lq9\\\\\\\/195sWqhpUoyVj7fV1W8tJzqrqEzq1FecjnnBn\\\\\\\/agVXiBohl+fFF6ThnHGx3ptqqMZHm8sl6RbpHROrcf0K4xQRMr1qv8N2fDvIJVJFhxm9BIFz51el6SOkrwR\\\\\\\/cFJq+TdnNVd6ld4oApl7AsM\\\\\\\/8BhAm5DQIzcm2RL6etpty4bRN1DFfL9NM+4A4axbEqdsOHVAnr2Lbk5FYCssFS42f81VO33CDtXcqIuUeZA4Oht4dOWAQRBwCUvrcjc0UWqqzQeRTnN96+QCV+VCFTlsa7FqzAr5ht0wghH2yQDDCsXMYxzmKl7+xI13dgDSh4UZq6S\\\\\\\/7XuvAB9s47l2zIUCxxwuH50js9jskRfeUOUIDfwxiJUQsrRWFDHN+6AFeCz3YJy2N4H0Co12zRJBssed6SaRNA5XBWZDrcKIiZqbUWqA2DK\\\\\\\/2F51KrqzMpxSBEE\\\\\\\/U7lLY3p9v3ho6ksTnCxlBicA4clBP90FuINGuYK8qQOXnQJ2o1VdQWhltm9hu8APyIKDEC9n+rI9T\\\\\\\/Wv4QEVmCzdIfmQgFRouUAAYgW+LqhyVxHut\\\\\\\/b0lJWakRoWlMCd7o+SOVibksUhOyj2441auujthBkxKbmG+WByezqHQqomGWrWKPVoqhKBHhUwK54HbegJ8dqSyaGciKwzjsoEGVOdHh3KwiYUVfV0KIeleYNMHPc5iO6oG0xAb+w2FKSjOZ5erChahgr5p\\\\\\\/BacF9IA7bivohh5L8UF+GUCmCXxd9dzwKKQKrMg4xDGzwWrCrcV8w5HFL9S9qYORxYSrLjcVicNeCg+ocHCKGMe3NoiLdrdPnenRs3PvCUK\\\\\\\/0ynblBrAu1nhnXUSvyeUUwK7cyQ5GKHISHhyhqMT+vorUiNnSw0pqRgc+TW4V2IrE22jBVNCmShGGVIB\\\\\\\/1OxIS7mA4jEaT\\\\\\\/dJp44Ad85z0byniN3WccLt0wsfpIriXQK48NfknWVjkSBhqrWa3R8Y+8YKnF8Enn8m6nszGweDiu+vyOUW9+dEWReoLNNkcU2RkJq8W47lHyHvAITdYpMjYpAFQfrVLVsWbHVAqfKp1+s4JmDkQj4HTRucmfedIIIVix\\\\\\\/qYatftfny+PE0wfapUJXj0Lxsp2zVnvQ0OYxuhW+F+YkBi+DyUGvFEYJGv9Wxe5TiRdwsriunz61MCkEnrwqYmJDn8IBu\\\\\\\/rVKr4t7jhuDSgRUg2gyOT4XxQ\\\\\\\/asHI01BANl91aMNEC\\\\\\\/3syuTyMjwiKCWBZdVhTabE8uTQPZQHCPBGvdkB4i+25TANL1\\\\\\\/BlU2WstVqvYvSQcwje2sLnPHiNb5SlvvPTrg6Yk1TDK5rbKUZnC2ifDA4cL4SJZCq11KIPtwKxiswAtqnqp8of\\\\\\\/cIGadbMyVetZ1f57sNWPXeGCdmnkMhY9vdwd4kgdobCNYPCZNNWBeAGH1qMJKstPVeNsFmxRhGlIyDP\\\\\\\/s06untQe9+FBJ62cyi4AJCuH2255XgCUpE6N5hzD2IZ12ppzd6csmEO+UDl9p11YOPLNi1t4JxXNeYC9oEvQb0aNtcacSDlY+K5iRvfQyqCbXQUdMXoaUEv3Nwdgou1zziErTtuT3eBFtyWguHAw1kx8LlrQcsWCjdqOZU07ivgwFNEe2fo51bzeqUZxo1nstm0VoVmAtkwMB2K0evHUV8hl3apCoysCPL0\\\\\\\/M5JZDTa9t3x76Y5ZSkouTV1fLWC6SpwhcEiYqyFUFsZNFxPm7qKB+i51qk3JQPSsEb+UcpopuyV8ehSX7xvc46YBP+85hfNQXlHPkSVZ79ZBbrSzYRicRzTJ2DAFvlHjtKJTuJhxhK1FBxaO5XMJE\\\\\\\/gzegPhjhZVAb\\\\\\\/WXIQ\\\\\\\/cecj24vhpCCnRxV5JZm9\\\\\\\/MhXUxn8AZ4yQ+Rt4\\\\\\\/kRpMtCaAbw08nRrzfyXIGY0STf1qMPQ3cPPZczA1cPZ3ktHF42bbpqhc9fKlBQYDeyRsNIx0a0NQp7NIQi838F1vd+lK5EjwK2UXU3qyzfc9vvU4ZLllCv8bfu5FgX5FhsveqRE2ZgFZq65K4CgZUwWAG1ksQtl1F6g9S\\\\\\\/lIrjgTm5rYOQVGa9vRFRG9Fbg6Qm6cT03G+9zglDHIfd1\\\\\\\/6llypkZ2Bi8bxPll7OZoHS8EMzAccx9+QWnu7lrWHmqS3r2emJKZ6cNIsqYXcbLeMu\\\\\\\/\\\\\\\/PT9gEiN8gEyZZy4DxHgQMpSbOsuGa\\\\\\\/fEXYGUSUKDxEb8\\\\\\\/yeDFTb2QMrjB16WWDNgvuAiP+gRvS6Cs0Rlx3XsFqu2MUx2FneteXVJa66ng\\\\\\\/L2GOcL0JmzeLmR9P\\\\\\\/sJWK5jmTJNnLz98sMQEMVlTEI6RE4HZkrTSXlly50qZOqqsJRKL91rEqXzCV1VKa4MPFH\\\\\\\/eEBKaOfPB2m9PFtjDiUYRcUcIQiNv5trWWtnBEs5CVjLBCLV566IND73hPnkwEJJWcYnMHRIdNiJRUgwRyPUsgpqhHOPSoBQQv+9tQSClP4Uk0Cdr1hC9\\\\\\\/2wlOEBqtmr4NKdMj7CZpXcT3VLMHRtVpL9TqMFkKSb6JMcGS8MZFYL3gWFznHmhxOfNQlytUwV7rei22luVd8A3KR794Y35cpmlU1r7qutlfP8oFry8uvusk62QvsP1Gx0JQ64BJkhV1iXnwFRS4O5W0Uo6ZQlLpwCX1h3MUHD9TIz6aGp6A5+s0e8RO4KDAokQox9aCW1BwCe6xf9VTdJsh1\\\\\\\/1i5LKwPJpjAamRzc60nHAuxtSQr8MDICAn0oKnJYEX32SDBHWTHRfLFvUVAsMphcD02Rouxzv\\\\\\\/7+uUslpFGZMBROxULWZSJIaGWWCmHt3HmbPCgOyLgtqnj1Weo3FLQ1QmwvrakcrrZKUIcxOHlgusdUKDdcYmjWNI7jzlJv3OTtN4NepGaWZRM1XK4LvGDLAv9f53JuYMim8jAyP+oYlTpG2sV7o3z4ZlyAv2Pb6ccFVXLbbS7WFjuY4Y2OJVyoawpVl+vY+OTqd0\\\\\\\/Khd5B3FUfQEmPM6FWo6e+1ItVJjn5SJsUR3JYINSQyjuoe4C4\\\\\\\/ZP+22SHYc\\\\\\\/ax7FbAo05dVWk0cKxmpgDq9dmMBzkh2JhnJWCb3GVNxhjEK0m40TSAwrHaTtLQg23Jznjy\\\\\\\/iUHfzqQ8KjHKHiiGzPDBfy8+5NBDovTEIOPYMs0AlAcgy\\\\\\\/lC1j+Kjo2ROsx6mPbjc\\\\\\\/4LP5KgAfPv9APglhEgRIqgpJdFoNEliqY94VSmmkEFyeX83ahJww5P5EJNufWsDO2hfSypjOzk4IGQ0qyw\\\\\\\/TqqlvIiKvjjYBAK1SOFdQgQoHfXC+nFibuURJcJ2vEcU0uPdClB0NH4E6hKhmJLVk5c\\\\\\\/XYqKVsCLro7JxcaUScDuiXLbulCqzg\\\\\\\/0j3awy8Y1D1s6c6DANjaxWd+Dgx3q99OlvNcHxQPqhXNJL1fJOzFUaIWa+2CbsDLDNApRJ0znN+YHic2JxyA2\\\\\\\/hsQp\\\\\\\/rMhYse6daq+hikCTMxLyAwo4GbHkULW9dVNLwpq7PCI4AX7mQm1mUO4C6smwLTmY77zMvcdqCT6wxPSWrAX10N1S3szcSZ6txlYbHA4\\\\\\\/QRj4iX79Sg\\\\\\\/FEmIaMiw9L1FEJOq9x\\\\\\\/z\\\\\\\/VHvILPSDIcGy9qj4jxEini\\\\\\\/ceAzpmaWC5XVr+rCUEWNZakCfxst3iDQ8mhHqi8VBJdO5o\\\\\\\/24Ab4+97hC4XS2pLXXo8fb4POORCKxjkGKVWPlnDSz9nAn4peHm3\\\\\\\/HKXjp65zMiLnoJO50639SfMBr2f04vtYIbMpfjy6rV7+7kZtndDEIKxh9P58PvLA7oQNmKEy0530pGRwla++sKoDwuyoj0iI68KOOVSzC89x6Oa+RdkDOKiE6CKrnST5PSt5AXjG06gZ+SbeFEoexOJv7CMdQE2kem7KNQzubgYQy71lks2y3JhYem5x1m0Ew\\\\\\\/RBppwnXF5DzjHIMc3OV5f154KapVBKFwdsuF1WbQE+I2ZQ3HcgsbXmgQIcbZ0XNDNZYWVzgnE1xpJOyqhfES4JozoEyqMWq2UTnABqBo6XMKZ1qcJxJW38Y9otWiZVhpowSL+Y7OsL8QBxnYyF3OFH7wzVHwaXuJYcVVYFT0QY1z1Gyv1F\\\\\\\/5F2sXmMAc2pT1OM3qCvhMCmga48zuqMHRAaARRIJz+C7urCq50PcufsiHfe+fsu3EuRWGoxCFLCc2FB4ZXwlADvXkNFtU0ewMByt3+fndsf4yx4Rtq4zuP6G+Mbn+DdrpwDn0eZ4CVhQdoIc67242PHhfdvlBDdtEAHC8QAs38r5NQou0cl9Bf+1Xmgbz+m5puZoMBtHWfdXAEXAtQyP9r2Q7W7Lb1y45uRqvnndxZh4wn9zVz0YRuShM2CrnlIc\\\\\\\/84AJLnUBEM2VLoLCU+H755u4YHEnJd+U6a9+jjgetzCXn+B+AoATtVYOhb7KusPN7dK53uSL41KO7meSfpaDUC32MsAK6HDOkH5TekcSCjjMoXw3oIPOB1bv12V\\\\\\\/HNXhaRw4ZzLj7eMZaBkhjRpeyxwU+H4V9OnDvhcf4Oy7dxCZNCuju47PrPamSITXtKww2XlWmg0twk1ZVfOvcBv5fT6uoGu8axx2g39yXmq1WGo6WRb8duujOlIan9vk3puSkAc+en8FxTQ7bpXtD6+gtG\\\\\\\/M3\\\\\\\/F\\\\\\\/73v1Q+MCeIFwaMHZ8OS2s9KMBCA23VnW7uNXHAoZenuR8NfCzuA\\\\\\\/bNXLo09Wh0zgk7eAbvkZqBMiNdIjE+uoUrvIlaf9kpB3SNHsf6e2xWT9P1NTvegCjBXhBIRSteP\\\\\\\/hp5iZI63BVFz4g+B6P9R61eBn3P67AyWsB\\\\\\\/xFiVr6PlTYvQqEsW+obpDv9Uwt\\\\\\\/zQPzxHxvaGWFOk5NxBO4qZDw4hpBJWVsS5lwb2fhXveSKIvhgblmCYKtbhUSWJEnxR5cM1ov\\\\\\\/s5mkz+j1EJQT1VTZQlHZfM56L3YVyFzccw\\\\\\\/c0\\\\\\\/TxXWJGehhHpbm1USk2wqo9RKBhmx7VMRy5VBEgN+rA1fdSjePm4tLq+3AS7TU5Stvqsg7hSIQ0oT4kF94h9pQ+mjobAJW+NktJf9UV5Wok6\\\\\\\/QoY6XavMcnEFdfq2\\\\\\\/Xi8EqGFHMMer+RgOhLzrDEXn5RmuXaQ2JayvR9\\\\\\\/Tg53\\\\\\\/3S\\\\\\\/5E\\\\\\\/x9vGpud71Ki2uH3X4LJicjcTh+dkMY2KXwv\\\\\\\/H\\\\\\\/Z9bbbl101Bze+ri78HvWc97DKofpmRvHFJ4NWoK+7eG2WryZfe7CS3m21CWz2zCdq1JONzU8r\\\\\\\/i+zlholXA9eJwqJBI\\\\\\\/tyEEBPCrDouhfl\\\\\\\/JecgE2imImERbuy6MhMHMQKAmzXqPQNhE9K3lZHGyurpyvKHJjFEyKFJwYuOrTuTA1XpUf+NmqvOH4h2RpMl49GAFPfevTbSgizMC\\\\\\\/gsqh6djqwrwinoxqkHJGE1RrxBZ6zYRpoYYZU0R6Anu9pwZthYMoThCUJypp8X+eXZa8P9ggcwFDfGwmA3bccPF+1wpENEP6JQyMkO9rEgw4Tqj8AZJqS4\\\\\\\/J+xbcbUVuy\\\\\\\/Wd1KU3o\\\\\\\/nZJFEfaI\\\\\\\/\\\\\\\/v1LZ\\\\\\\/NyY0vLR\\\\\\\/IKh917V0ka0HuziAojEdPmozHpMdNSkBXVMGuNQuDsfu2Fz6QtLEgN78qTkGuGNU4ccElvEhQKO1pH4o3LF31\\\\\\\/+5c8jdByYv4C9KtS90Mz43YORKAkvT9+yJ0vxJxYEmK7y9ltUWItnzYDxsFgp5zFI0ltAI99wedFZsU6TfHqNSQc0RjMRu3nQgreTj9jyFnk+yDqOrpmJwUuYLfSh93EbF90GLMgrx27SxxN1ESJLfUWdr5fn\\\\\\\/c4FhA+XzseEBUYhHolyq8rxOeJtkkyC3sASqXsovBOrl5fh3KFdkhB2vM+I7tydSXlf\\\\\\\/e3BftP25ZdZl7YXc3vkTg5iH96qcT25PCG5oxN1cCKpijWOCMtjCVPI0u9HDHK07oaI9DRCKIkIfO3GnTsLpuRorzjpYWgl7D9UNJCMTF7DbEqL+YAYqDU734ukqjE\\\\\\\/+YVkDOwCtjhETfOBmmlIYrdSycS7k7NsVoHoc\\\\\\\/xEwA4E2\\\\\\\/zYEgJGuIzWnr4y77aCFGhxN7FGOeFlgOQG7o49Pi6kAqdZhLHY9xvYQsIl3M7NXZknK+zQDM3xsuY2PPtNhXF\\\\\\\/OcUthUELF6xSqtz+COPFXgEEBbOXZ\\\\\\\/WWrXoSAa8KZj733qcDphXb0Oqwz6sNuU52hL6+rx6GewhJf8\\\\\\\/7sKUv4QfyLJdz0z\\\\\\\/xrWPASUnCv0StCNkLtg5ZyWlwj3Wu0W6EKjg1GCBpsnXUVq2wovBJcsp0igWwi2UkGNTW5CLUqIfrkWlAxAyHE76pzpyW+CgMaN6X7Ke94+X\\\\\\\/zYsElWnchVJJg15EbDgyzNcSrreiLycio8HoMzsvGgd7MTsX5cXr5T\\\\\\\/fQKKzorvmH43J4nVBVGugCa11xXyRqBBQUpN\\\\\\\/ZGxHfQQE3M5Ow1puvyloxx6VmI4ZmPz95uEqDll383To3A2N4NZsWiqi0SCIhNyDSv\\\\\\\/g8HuZjqDu8\\\\\\\/tGcKRl4OUrrkbVEopPgLrh+dXMfXKj+6myWUWKkGQ7q4K4eovLBNzY2NYi9R31GJImp4XPtcICLASvkMmU+sI1Yc\\\\\\\/ZIS9rFy0HtqEYiAZ2k\\\\\\\/hDZG15TpDD9LGDetEpbN3AguyH9yhtWhBo3PImBwI+B7oLugcIKWnuOGJUvoqeZwy9mjFYwZhB8BJnMAsB1hbaMrRi2bgDsUgqiklxktY9MQoEo8Jmr9+jxgJ2UPbpf1HEqFyspGzkRwdajay+gCWZ7+HFOvhl0MaRldRk2UDnBkRzhkvED01YmS0qar25OIO\\\\\\\/j\\\\\\\/UWyQywF+pQ3ObrY+3i+N\\\\\\\/wIjvb8eyIYssJNhHSfa6vZHH2NeBnRNMC47ikiRPJzTpAWGQ5\\\\\\\/A9O8SVHN+GT7Sv5TYR1iHqXNyCD5UV9YEpZMo9DItEoeV+oOhk3kbD33FgETEkpdg505zBLRS4ulVbYzYpr7Hc7pWcy\\\\\\\/CTXLFHtLCtFMjiKCN3pMAoQY3GaglFIzzXriLCbt3UKlLckS0kklPoqtzGcQ3KmLGHZc5h6v1NZF3cpGfnrTrkV91HfTjEsmsK8IDDwjbGF3zcc+T6b3ct6ATHdugQay5znROQeoW4aUpT5lqt6sDPPe2HDdHBVakiCrvMZyuuVfvGC55w02s+ViKNdGNpeZoo5toDX5haxXkEE+g5+X6uf0zizH9LBvsoBg8AbPO0geX+q4jybdq8ycG0va+BVY6G26DyoMgdFq9VQNYCLSzVtI8Pph+Ezj2HoV\\\\\\\/Eb+DHVDHcfg8pGf8Fer9004diXAaSApC8dITcXT6nTQeEQ5PPoevY1MoBKtbk8roVeYyKtLbMrsASywaw+Y3VHePKrsXrEd5bMCv0hVhhnT\\\\\\\/7g2+yZarQx1KCSQC\\\\\\\/bTfP1StAOUwdoEkRHilzwBTOyQ4cqPFPhKAaE9FeWg7HRpBhnyBQBumpjFhlhaorpefzcSnwe+4iMGmglawn5iWA00YjMmMBHf4jtvJtcz0GUmpyt7hjjvbZogeVOxkCeGP56b63MA9wJXQY032CPVzT1lsZIK0oP0WeW3E2+DryGyroF0pMz9839FMU\\\\\\\/9PmCA5mo4dFiZm9qhYAv4ZtH6rBTHtDKyDXI+NgtItcV2\\\\\\\/HTQDZmofssuK4Rgb9+uLWD1UCpjVn49y0GsIHx1NY9lYMmjNXc+f5iOmFOqq5nNKP0qOwIbfcKwsFfdb2s61L6j0Bs7JWNKFi+bAEmuVMpsA1fuatAsavYX3m06AaKpHEqcCfYV15iG2MPRwP1Q9pFlIXnmn+HJEimu8ZJtxAURc4GRiizkUh7\\\\\\\/0ENdFjSKmM5WkKR1JIi5Dd2JNdO5NNcsX71d8K7wpXHTm1LoYDVAjiRhFDviYdoqKEc0ikrQcj9+dOlF91gt3ZW7tZQlPqUZarmuE2pKQm6Km6q7V2vnqezdjCFu65UaXI\\\\\\\/+09Gxmyumf6+F5Tobj\\\\\\\/xXJQhtA9Z1XL+he99uE9RY2ph6tRuelCA\\\\\\\/EyhvdTKqJDjWAoTCqBISPyqLKm8k9GCEkOd9nCON69C0Zdt\\\\\\\/P91fZhdHOHOuNUi1FsjLWglDuRS+svl76nVowf7KilMmJY6TZzc1dvq9YnPtur9SJzFN0NTwgw7xbjqRqDd060Db2jDTn44WJrItXa55jnNxq4ibCM9y0TZOZRT7uDjt50ASyjYByYIkAqyQe1dmvmF1I6c6+4a2+QUFlGlqbGNz2shH9jS4eLMttvmNY4tRtpyQePXWy+Ccsq8PceVCKmiy1iVva+JR0p32Jv+ZeCydt1QCpNyf\\\\\\\/dpOuPkQ3CpgN0C529M\\\\\\\/D07ki9YqyXp1\\\\\\\/uT2vEmNI+aT8VaMgadG6Qe4I+uTg+QHDb6FezHJkFA0T1TYrobqmBUZMu787jEB\\\\\\\/ZzcBL2\\\\\\\/KGZxIRFo+G5TPyQ2pk6gz8H7ebIqnt3Knt+SKyBdQMv7+hNHGGj2wGxkOols5wYUiWu7LNWrzuQYpJzuZMikoPowsQwoh79SjVFHb7RNgezhsh3hVemMmcNJeGULq5uk04nyEhXl2boh8ANCHvKeD4lAmZi27JEv2bv7IzP08Q+HMaEmQRJrP3mof\\\\\\\/ZAdRA29uG+CfvZvqOjyfJrvXSrSac1jzJgDBeIiaRs46iGBfCZjCNVcpu\\\\\\\/SGcGL99NOYbtepe49BTCT2jS6zqw086sku2pSzBzguwWfnQhH7Xogwrs7Kk0XIbPcsVHBl0XRHsR4jakEAMXEMmXHib3Btw2rhX9JSKTKr8WKsb9kSPSE7taCoU4BpqyQCb4aL5XXpqJlKqJ5Ny3g\\\\\\\/fQ2t3QhEyTAKk+YTkvqAEn0yzSi0SpxIwzHwtz2sfySRQM935QpkFXJcqqMkOmKLnzDatQRxrc5hKHTfQ\\\\\\\/X66XjjWepNPTTCbwaFd9ap7UgSmykAJIZdB6GvsvtLuLx+O0O9TIrTzPhC7s5vT8BvSOYMVzQVOEnwgdIVOT\\\\\\\/H8s5zYKJcsXE+FOl8qDf8ElVohHoE3tCjn6\\\\\\\/GKCfMdBv85DEEdjFYbptq5LJZH0tFfp9LcnFkQdIX0IaZ5E2vvO\\\\\\\/j81OSKB1vebTORq4WEWzp4PnUryEJaTZuF6xjhZXuE0MmB+80VWRlLopHjpd2ov5CoAfZ1nwvmut203Gm5ETvfWYQNQx9+dNPRqKfK7awNAxTxU2ZA5E98RHpq2AUcTf\\\\\\\/ll2m3fVWi+2MGMrIomRXTWqsXi1zW9bmwiTliRR4egv7V2VBf2+9w0cK+0v89imG0i5smXTzXgiK1f2Vef0cXhWdUXhw\\\\\\\/n+l\\\\\\\/\\\\\\\/RkQBSKUUwZj4gb+T9e7I8AopzWTpJ6w7Ed5akGPI85lQmMRb1ZyzdwP7y9zTrSQEvLP8ySzOMhslUBKDRPpzRVJkQBxoGmydHVktHIcKBDxes1bwjfNGoC2zfq37+N3SZZyMUWCbjtPw5cJ6RDW+t4\\\\\\\/P7MIbcoSrHuNYgPB0LG5xeNzTQBs6+wHurC43zl5kKNC+FpiMUFDmLg4phaQ+kUkZvO+XzNXhL3CI+zDt7qhuI7Gd7BdaIDtbexaG+Fddy4NI9YIC3P7XfDYKnr5dBA3WQ71IzQIkMLI4DmymxcWdzhgqSKSZ6JVk9vECe2MP+lTPlPBlUjLThM677kIeimQp1VoXXIhiSB8BzwSZK6iXwYPouARsTssUPFT+CK7W5JuLKgAlml705K3UuBMSV8n1bxiZdS4+7RqQbov07wplMWQVOrA3sIF2B9jsDjh65qh1sHPFHFnKr+MErHdV8w5g4gmGb9Eu7q7OapJRFBpEHXJfEr7S2pFBspz2Fx8rSd\\\\\\\/hpzD4WEWSVeSzFsCw6EQj6YLPEc7lz7YI+RDTpvYCIfNRjIRFRSkqkOQTrp3spbzUPtFxwRGb65uy\\\\\\\/M6q6mHa4v30jZKFAI3ln7e4pWU9B+4iYHVFIc2HFK2T0Xdj6V9oQA+mkhc2I8XSge7P77boCtGKZR2gwEjOT7PJoAzePmf2w8UOSrWSqxLFhB\\\\\\\/VxlzzBADYwHVwRDxtePUyKwNcHnp7o6B8vp5N+R\\\\\\\/7\\\\\\\/HVtc0MQiIlciRuptmVl0Ruk2UOY6u3XbZsKCQxmPI0EY7tsaJme8DoGx0yoQQ33gD0Q4P\\\\\\\/TLxTficiXz3OVpSd5R0kG8uEkK9B9ERrsv5GPrpaF6IsFSLgZwTscFAob8G7k6isJxgjrPzw2KXpL5HLGjKLdMpkT7WJW\\\\\\\/E+Uc8l9i3iS+4\\\\\\\/kPhHLOPx5Gi4qtneu3A76ChGhBrPNjzaDOn8rM3w8GxOEHlRfqoBoS3jz3p22BDB7iDeIXf0E0xnKvYMkOBtWbk2O2uOk7UzxuwGXGi5mpR3rhS\\\\\\\/4ikAnYHbsSJeBrortFzLlzIqUHnuWuzHPGF6rvZoMYVz1jUlHcemORS9EgbOwHr53peVrOjW\\\\\\\/d9ETQeJ8nds8juwqm9Z8CrhdJvQh4LBoAgjA0anfwOx9nKtlOZ+q\\\\\\\/hys2brxhLq085ZvZ8hB\\\\\\\/dYDE5rkLOouP+58RDUWgApO8kELAtUefN2yAVBkhbevLNv4x4QmgLIBOhF0\\\\\\\/QygGHMOtz8mcnZgkChI540pXay2\\\\\\\/4cqSpbH6XLcuYTJHNA79j7JkSWn3TN2Nve70Aos\\\\\\\/ukxLe9KiqSqDybP2D+RQCqlaqNEaqRF7Ev94asWlr\\\\\\\/kuR40ta29tNXTbyVCp+cPYeKnWEk1MM1N1qSuCOoinO\\\\\\\/KofkswFTDLyGucc282tmVrhIWbZz2aaHD7XVYTQaKsfvCCZm79P4k1N+YAw2mSj6cgQgC6njJ392v9KyFw8swsjUpChLheyd8KW3bL0jdrhN9KYCok+GHqrRuqtc6p2t28nKHWq1WFXL6iB3MPB4Pm6wg3bm\\\\\\\/eMn6c1AFYxVQ8yCBLe2U1Pf\\\\\\\/64AayvVViL7e6XnyDeMDQ+xGBSBdj3XMWcEZx7iw8gMUfymQpP9E3G+akXx2DFg6P7afX53WHkfOSkOPJxhpJshzxrE0ceY2ArTaU5qZNCzdWoYEtmQ9GwJZPpzZL+GxS4453zz1NPBRMLFH0y89dccHdCybk+DXHeOwleylqq7HDPQi\\\\\\\/G4wcZhBf5iCrXrcJ4Xz6DTCpSr0L8eredr1i0ga+Ikw5fPmr2ykI8DhuwutR\\\\\\\/m4xnwrkZc+Mkpt2kCpa1PKrF5rCxCsgZv6g9XoPKdHm+NbEBgahY+BYxXFVZDyZKUnwnTNEL19UfKgufjTutDTH+nc9TN60h+7QFSboE31P4u5yZDfSHNOXcbjfFg0\\\\\\\/okQ6P36eirT6TcxY98p6DNU8d9VLC01vrcVrZarpggLdE\\\\\\\/uXiGcwO98RJmKMFvXKEDHH7b57R8HVoeFcqmC7+6yz+vQrJ\\\\\\\/pvMtwNwLcqScqhXQUAnEqjfeiv43LFI1JfmHngLeiKMp3c62ariQiqfFK\\\\\\\/JmNWmHOP8N9STiZkAeIdmRADTeRr+cFsX2Dn+83TqAP9lXj+0yUHQQjtuBR4wVKmd1QmHwSMYlc1RgYL65rUe7B\\\\\\\/KLg3aVgY1GIvG7YHhELAocqWZFO9mzjG\\\\\\\/Ar7zXV+KTSWJsW3DDP1spgFa7mqFxAifXHTl3jyNZTumE88Ym4Svr\\\\\\\/igT\\\\\\\/hTIG91ODMHz88s3PQP0ZQJ2z5H99pf8u\\\\\\\/DhljLyYbStz\\\\\\\/dU+\\\\\\\/tSP8chxFFS5HQBPRu4LbJRLZUxkb4lS\\\\\\\/5avFhbXsxXSGVeuA85w1P6l8gVwxRyRmPKzGK32vA7c1zfamuNsRp0D9rmLVAFHOCqxO70M+ojP0X2TG+dq2ukf4DfDiRHOzaZtOgEZ09Kv4bEibGSiN5Wicrh2ibyvhh2Esiol8FmafDLzpC2tfxAdA0wuG9Cz16ySe3xnn60YsvZm3iPQ7tXgPnhjJIA9HlnNAGsIe9sDUlBI77FcwlfnM1CQIkgTyDx2S0naEhwl+\\\\\\\/5AdJcVA+APSAJWeuz0b+JylwSr4aNqdM14\\\\\\\/6OvyvWT5Fjzses2oThjPQGsuKruP1BbtIMx29Ld8ji1D9YNahRpVGYA+X+4ExHXuiWOS7bJGL5gllei1qwkUk3\\\\\\\/fUpsD8863p2d\\\\\\\/o6MJzh0z7dAB\\\\\\\/6Azu+xhZt7mleK+iYQs7XEF9FuGWd4J2j79E0nKoc4C1w1QBoIJ\\\\\\\/6bBnF9JM82EL\\\\\\\/VXl+JkQ1zbNVSKbkivHrSMRZ8VDWOXaR0dKpdAuO+0N6dkSC11gs661rVtQh1lNyPYtmC+VkWynURa0PrriHVMnPv9NTR86RUVpecpmP7J9wY3ut\\\\\\\/dF18V+dyXAPFJnHIdkfz2Jc5VM0YNMEe2ZhgY4ebtuS8tB1kGbuRt8UjAYBEaRiCCn+572tash4TwL0o1Fvy06c2XMvRuCPZOU\\\\\\\/AmLL7I8TRUmO1m4KuGqNM++IeLti3cCJ0kx\\\\\\\/fUHmnio8Hvdi7imQg4sq2kdl2zMKiM3PWDeVbYfIlQk9O4PwjYb0srGxovLD3VYN5s\\\\\\\/UvO99s2kBRaQh+22nnlbhbNxCMu4vWGMeit+QrSIEvsD7gPr02pZPOpaIXRSYlfCoF9RmoXLwnlzLfq5M2oFrFiBlKetJ94+plFKpyiU6DhYoO54FjtXA==\\\u0022,\\\u0022iv\\\u0022:\\\u00227MAFiLwgLDrv1r2qUC72jQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022edd3b813ffb5ba3d241b9af7ef89ad73b3d122d9af65af9cc6d45ed9878ef5a5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XXtu5ZwDqSX+3pkrV\\\\\\\/9s1cLwj7BNs+KkAmCV6lW+1LjZ7ONSHOypMvZlSb1qZzXEft2ohZcEnaIrzokS50tnpmBDwCWNVzgpBKjnX41E0O\\\\\\\/fdV84mFf6Ss8tJavdpAvt3d6A42fLBKFWmcPX0ipFK2++oQJljE29qPQYLCypX03FuXda9coVNtG0flGUFO\\\\\\\/bPTO4APanbZcF2+FGgsvY\\\\\\\/8LtGgw8NSASmlYzDne6h2KlThQ0oWjG0XlLeOkvSimBNGV5KMb0OrchDeLdWAGzYYAg4utXGCOMZZxglrO4U0reQsviOaZz4T0FrUaRmqV5Gq2jVrxVJ\\\\\\\/0WQFon5Mq6ChaknimlhxJ1mRzzxHQquIK2TadS\\\\\\\/5tanuMoWDe5XM74HmrKb\\\\\\\/GJfbBweqqOMYrrTy3thIATQ9vIHG326WkjqrYs5UQdvXUDRFFdZQa+gY99+8WfCHFwKh9urU\\\\\\\/X\\\\\\\/FIXLp5ZCyvFavzl83SHPX9K66JFjNA\\\\\\\/qOoOrKb9hdqNFufbm6I4IgXgP3\\\\\\\/XmxVj8vqxO8IwiERqwBjo8h1GXYPqjcyiIqzA\\\\\\\/rup\\\\\\\/zEsCPHozcEPIxlVYMX1rGPH\\\\\\\/wvG\\\\\\\/auP0IbZtpwXBXG6usLQRAiJkhGzml9pgzZU53mGXhAoYI5IGwNxZr1bTACjKCelx6ExCrY+VjJ0hXj0D7lmruvt1bYT72\\\\\\\/zuy6SR1duqXe2P8RtWx+2buOpRtlu3kY2+7O7HPgEkVKvgE8xYF0vcY7Wfan\\\\\\\/P1Ouywztv+RuhHcJ4aHO1nW37AlVjeVL81c+lATNLD4B10ymj\\\\\\\/1qRnUhye\\\\\\\/Mlztem45nUtVN\\\\\\\/wKjQUqnRMT9eYfpT61pbxoliDi36JpwyKxqiUUXZvdi9q+e0\\\\\\\/hnJURvoOtlFz02C9OYli3w16PUVUEYfaY1bfrYIiO\\\\\\\/tjuyWmYec3tSehqFDA0MMFx+8R\\\\\\\/n0eXWloEp3hCWlIlsGbiXxczY3SQfBuPHL4So\\\\\\\/7+PNwG\\\\\\\/W8mdNDCLiunoksPzTWsI2oamZW\\\\\\\/bkTZpVVFz0CuI+OrN+n0XnSFF5EKJVX2K6rQDRqyHPmv5Mylip5g6tUtbbKQ6FiP1SI70Wut8F90zSIQycxICuHdhLlEl0keFu3jELRqI3HC5qMkn2QfjdEaRZxjseGfIf1HahAbrQ9wcqSOyxEHFYNtQu7MVu8Qw4NvIaWS+m5q3wgEFOA2bLFqdfKzhUUfIN9efFixH8H1+5qROHZTfQaLkWVpzSfnSuot0vgw4DlVEiQPI7eJJfbSvii60BKxC1rFbfSAN2hVtZ4raQ\\\\\\\/j\\\\\\\/0Xqrgrf6+nJySePcfeCSLqA9UTN7lrQDec4HVtpeQNP6ELrXEDvp3GNSJY16F1StOMRMp\\\\\\\/GYJrmf6MO76VXIKFLfCfIA3K0dGOwTCk0x37DstYQGxRXlU4IrKOVT5ZeRZRN3JrXmQJFLH8VKa9OQ8A0F79PvXwVWMnJtOUwLIZsswltYSlDfjptRX+Qafv1\\\\\\\/lou4pBrX49oOGdnXRupuZE+ENKnW0Sb\\\\\\\/B4bqJFyunIiQokhk9p7xGeVxFhUyJqxG3\\\\\\\/UnQCL9cIU+XwC1IZbCSWw9asJghwTSOBTUM4duGztNTxE0pnkMrImgK2gtiJlMaj4BUWeMSKRp2Wcfof8oSsyhIVPFjYnug4b4YS4OL+du3iebXoJ8K4fDwRK8Kr5u6lIYQIGAai8n\\\\\\\/+UJTKO7azQAJYkPWIjyJWve9L2uutaGF6Pk3wr\\\\\\\/XqtOjlSCoSxjjHiCEJqRPCh9wx6+EjXecpKWDADcMDoCpzcWNgAagcmqUA2XjYOTp4mVaT0iEeStKEngg45sxiGBhUSX8Q27v1mVTBVmuQ+JLh\\\\\\\/ZeVejRwNN\\\\\\\/t+e6my4Ok9NqDXK+fbsrYcU1jl4YfeEnQAzcnsl5cNKzKrHZXP9nSQkIjrPhPK0EHgWIu9AaaLObdm0+rqmny0xauam2bAXRSgAHECs9NrotAK7tos\\\\\\\/TzU2nnXOglRDVgu4G7Uxt00wzaWnMcRRrOt349e+fSBlSQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022J+9HeR0omcVz+tnd1xsUDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022612e5b7d6fa3f9c4ced3d21ed83d4d1ff73e0b284eb255144891ab8676d703c8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460525 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/63" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SVXQXG97s_ADOSOpwaFjK2qSOdLs-khVWBVk7r5XAqA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/63" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tQl9N1CNKXUUi0Sj-PToFD0JN2kKvDjdGGiJXoCu1g4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/55" title="Assortative Mating and Linkage Disequilibrium" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/77" title="Cytogenetic and Molecular Characterization of B-Genome Introgression Lines of Brassica napus L." rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460525" id="node460525" data-pisa="ggg;7/1/63" data-pisa-master="ggg;g3.116.035329" data-apath="/ggg/7/1/63.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Phenotypic Plasticity of Duplicated Genes in <em>Saccharomyces cerevisiae</em> and the Origin of Adaptations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2727-0284" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Florian Mattenberger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0301-215X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Beatriz Sabater-Muñoz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1714-6703" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina Toft</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4345-3013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario A. Fares</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">63-75;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035329 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460525" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Florian Mattenberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florian%2BMattenberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mattenberger%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFlorian%2BMattenberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2727-0284" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Florian Mattenberger</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Beatriz Sabater-Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beatriz%2BSabater-Mu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sabater-Mu%C3%B1oz%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeatriz%2BSabater-Mu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0301-215X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Beatriz Sabater-Muñoz</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christina Toft </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, University of Valencia, 46100 Burjassot, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Biotecnología, Instituto de Agroquímica y Tecnología de los Alimentos, Consejo Superior de Investigaciones Científicas, Valencia, 46980 Paterna, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BToft%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toft%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BToft%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1714-6703" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina Toft</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mario A. Fares </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BA.%2BFares%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fares%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BA.%2BFares%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4345-3013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario A. Fares</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mfares@ibmcp.upv.es" class="" data-icon-position="" data-hide-link-title="0">mfares@ibmcp.upv.es</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/63" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460525" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/63.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460525" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/63.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460525" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/63.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460525" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/63.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460525" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460525/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Gene and genome duplication are the major sources of biological innovations in plants and animals. Functional and transcriptional divergence between the copies after gene duplication has been considered the main driver of innovations . However, here we show that increased phenotypic plasticity after duplication plays a more major role than thought before in the origin of adaptations. We perform an exhaustive analysis of the transcriptional alterations of duplicated genes in the unicellular eukaryote <em>Saccharomyces cerevisiae</em> when challenged with five different environmental stresses. Analysis of the transcriptomes of yeast shows that gene duplication increases the transcriptional response to environmental changes, with duplicated genes exhibiting signatures of adaptive transcriptional patterns in response to stress. The mechanism of duplication matters, with whole-genome duplicates being more transcriptionally altered than small-scale duplicates. The predominant transcriptional pattern follows the classic theory of evolution by gene duplication; with one gene copy remaining unaltered under stress, while its sister copy presents large transcriptional plasticity and a prominent role in adaptation. Moreover, we find additional transcriptional profiles that are suggestive of neo- and subfunctionalization of duplicate gene copies. These patterns are strongly correlated with the functional dependencies and sequence divergence profiles of gene copies. We show that, unlike singletons, duplicates respond more specifically to stress, supporting the role of natural selection in the transcriptional plasticity of duplicates. Our results reveal the underlying transcriptional complexity of duplicated genes and its role in the origin of adaptations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/evolutionary-biology" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-biology" rel="nofollow">evolutionary biology</a></li><li class="kwd"><a href="/keyword/gene-function" class="hw-term hw-article-keyword hw-article-keyword-gene-function" rel="nofollow">gene function</a></li><li class="kwd"><a href="/keyword/small-scale-duplicates" class="hw-term hw-article-keyword hw-article-keyword-small-scale-duplicates" rel="nofollow">small-scale duplicates</a></li><li class="kwd"><a href="/keyword/whole-genome-duplicates" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-duplicates" rel="nofollow">whole-genome duplicates</a></li><li class="kwd"><a href="/keyword/transcriptional-profiles" class="hw-term hw-article-keyword hw-article-keyword-transcriptional-profiles" rel="nofollow">transcriptional profiles</a></li></ul><p id="p-4">Gene duplication has been a major driving force of biological innovation in plants (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cui <em>et al.</em> 2006</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carretero-Paulet and Fares 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Holub 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lespinet <em>et al.</em> 2002</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Otto and Whitton 2000</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wendel 2000</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kim <em>et al.</em> 2004</a>) and animals (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Otto and Whitton 2000</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hoegg <em>et al.</em> 2004</a>). Arguably, understanding how gene duplication gives origin to novel functions and adaptations is a fundamental aim of evolutionary biology. The functional and transcriptional divergence between the gene copies of a duplicated gene has been proposed to facilitate the origin of novel functions (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Conant and Wolfe 2008</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lynch and Conery 2000</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ohno 1999</a>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">1970</a>). However, the tempo and mode of each divergence kind and the interplay between both remains largely unexplored.</p><p id="p-5">Ohno proposed that after the duplication of a gene, the emerging genetic redundancy leads to relaxed selection against one of the gene copies while the other copy remains under strong purifying selection (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Ohno 1970</a>, <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">1999</a>). The selectively relaxed gene copy explores novel genotypes, many of which will be deleterious and lead to the loss of the rapidly evolving gene copy (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Lynch and Conery 2003</a>). A less likely scenario is the preservation of both copies by purifying selection after a period of relaxed selection leading to novel functions in the form of sub- or neo-functionalization (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Ohno 1970</a>, <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">1999</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Lynch and Conery 2003</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Taylor and Raes 2004</a>). Particular scenarios for this general model of the functional divergence of gene copies have been proposed (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Des Marais and Rausher 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Force <em>et al.</em> 1999</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Innan and Kondrashov 2010</a>). Classic theory has also given credit to the expression divergence between gene copies as a prerequisite for the preservation of genes in duplicate and the eventual finding of new functions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ferris and Whitt 1979</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Force <em>et al.</em> 1999</a>; <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Ohno 1970</a>). Moreover, previous studies have found a genome-wide transcriptional response of <em>Saccharomyces cerevisiae</em> to a wide range of environmental perturbations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ferea <em>et al.</em> 1999</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Causton <em>et al.</em> 2001</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cormier <em>et al.</em> 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ideker <em>et al.</em> 2001</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Landry <em>et al.</em> 2006</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Stern <em>et al.</em> 2007</a>).</p><p id="p-6">The rapid evolution of gene expression after duplication (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Li <em>et al.</em> 2005</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Thompson <em>et al.</em> 2013</a>) suggests an adaptive role for the transcriptional plasticity of duplicates. However, the question remains open whether duplicates follow the general response patterns to stresses that are shown by singleton genes or, alternatively, they have allowed the origin of stress-specific adaptations that have been favored by natural selection. It also remains obscure whether the transcriptional plasticity of duplicates has driven their functional specialization. Understanding this plasticity through studies like the one conducted here provides a much wider picture of the role of gene duplication in the origin of adaptations and ecological diversification.</p><p id="p-7">Gene duplication in plants has been followed by rapid expression divergence between gene copies (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blanc and Wolfe 2004</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ha <em>et al.</em> 2007</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2009</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wang <em>et al.</em> 2012</a>). Since most duplicated genes are thought to mediate the interaction between the organism and environment, their expression changes have been suggested to be strongly linked to generating adaptations rather than responding to developmental perturbations (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Ha <em>et al.</em> 2007</a>). Most importantly, expression divergence has been seen to correlate with the sequence divergence between duplicate gene copies in plants (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Blanc and Wolfe 2004</a>) and, although less clearly (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wagner 2000a</a>), in yeast (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gu <em>et al.</em> 2002</a>). Two questions remain unexplored: (a) are duplicated genes more transcriptionally plastic than anticipated?; and (b) does transcriptional plasticity determine the functional fates of gene copies? Answering these questions would reveal the potential of gene duplicates to expedite adaptations.</p><p id="p-8">The Baker’s yeast <em>S. cerevisiae</em> duplicated its genome roughly 100 MYA (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Wolfe and Shields 1997</a>) triggered by the possible hybridization between different yeast species (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Marcet-Houben and Gabaldon 2015</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wolfe 2015</a>). Only 1120 pairs of duplicates have been retained, of which 554 belong to the whole-genome duplication event and the remaining are classified as duplications of small scale (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fares <em>et al.</em> 2013</a>). Many of the yeast-duplicated genes enable the growth of <em>S. cerevisiae</em> under stressful conditions, the genetic basis of which has enabled the exploitation of the biotechnological benefits of yeast in the multimillionaire wine industry. The genetic and biotechnological properties of this yeast offer a unique opportunity to study the role of gene duplication in innovation. In this study, we explore whether the transcriptional plasticity of duplicated genes in <em>S. cerevisiae</em> has contributed to the origin of adaptations to stress and functional specialization of duplicate gene copies. We address this question by exhaustively and extensively analyzing the expression pattern dynamics of duplicated genes in the yeast <em>S. cerevisiae</em> after subjecting it to a number of stress conditions. Here, we find that not only duplicates are more transcriptionally polymorphic as concluded before (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Ha <em>et al.</em> 2009</a>) but that they are more transcriptionally plastic than singletons under environmental stress. This transcriptional plasticity increases after gene duplication and it is strongly correlated with the functional divergence of duplicate gene copies. The study of the patterns of sequence divergence, functional interactions, and transcriptional plasticity of duplicates makes possible the identification of stress-specific as well as general transcriptional response patterns. We show that, unlike singleton genes, duplicates have given origin to stress-specific adaptations. Our data describe a complex dynamic of transcriptional evolution following the gene and genome duplications of a simple eukaryotic organism and reveal the origins of yeast adaptations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identification of duplicated genes</h3><p id="p-9">Paralog pairs of duplicated genes were identified as the resulting best reciprocal hits from all-against-all BLAST searches using BLASTP with an E-value cutoff of 1E-5 and a 50 bit score (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>). Paralogs were then divided into two groups according to the mechanism of their origin: whole-genome duplications (WGDs) and small-scale duplications (SSDs). WGDs are those extracted from the reconciled list provided by the Yeast Gene Order Browser (YGOB, <a href="http://wolfe.gen.tcd.ie//ygob">http://wolfe.gen.tcd.ie//ygob</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Byrne and Wolfe 2005</a>) (555 pairs of genes), and these were not subjected to subsequent SSD. All other paralogs were considered to belong to the category of SSDs (560 pairs of genes). The duplicates used in this study have been estimated to have their origin on the time point of the WGD that took place 100 MYA (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Wolfe and Shields 1997</a>). Also, in this study we have used the SSDs that exhibit similar distribution of synonymous substitutions as those of WGDs, so roughly belonging to the same age (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Fares <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Keane <em>et al.</em> 2014</a>).</p></div><div id="sec-3" class="subsection"><h3>Sequence alignments and analysis of divergence</h3><p id="p-10">For each protein-coding gene of <em>S. cerevisiae</em> we searched for its ortholog in the closely related species <em>S. paradoxus</em> using the program blastP. Pairwise sequence alignments were built using the program ClustalW. To calculate the distance between <em>S. cerevisiae</em> and <em>S. paradoxus</em> for each of the genes, we estimated the number of nonsynonymous nucleotide substitutions per nonsynonymous site (d<sub>N</sub>), synonymous substitutions per synonymous site (d<sub>S</sub>), and the nonsynonymous-to-synonymous rates ratio (ω = d<sub>N</sub>/d<sub>S</sub>) using the maximum-likelihood approach under the Goldman and Yang model (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Goldman and Yang 1994</a>) as implemented in the PAML package version 4.7 (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Yang 2007</a>).</p></div><div id="sec-4" class="subsection"><h3>Analysis of gene expression in S. cerevisiae</h3><p id="p-11">The transcriptomic profiling was performed in the <em>S. cerevisiae</em> Y06240 haploid <em><a href="http://www.yeastgenome.org/locus/S000005450/overview">msh2</a></em> deletion strain (BY4741; <em>Mata</em>; <a href="http://www.yeastgenome.org/locus/S000005728/overview"><em>his3</em></a><em>D1</em>; <a href="http://www.yeastgenome.org/locus/S000000523/overview"><em>leud2</em></a><em>DO</em>; <a href="http://www.yeastgenome.org/locus/S000004294/overview"><em>met15</em></a><em>DO</em>; <a href="http://www.yeastgenome.org/locus/S000000747/overview"><em>ura3</em></a><em>DO</em>; <em><a href="http://www.yeastgenome.org/locus/S000005450/overview">msh2</a></em>::<em>kanMX4</em>) (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Fares <em>et al.</em> 2013</a>), with three technical replicates for each biological stress condition [3% lactic acid (YPL), 3% ethanol (YPE), 3% glycerol (YPG), 0.25 mM H<sub>2</sub>O<sub>2</sub> (YPOx), 0.25 mM H<sub>2</sub>O<sub>2</sub> + 1.5% dextrose (YPOxD)] in comparison with the normal growth condition (Yeast extract, Peptone, Dextrose media). Total RNA extractions were performed with RNeasy kit (Qiagen) following manufacturer instructions. Ribosomal RNA was removed using the Ribo-Zero Gold rRNA removal yeast (Illumina) depletion kit. Stranded RNA libraries were constructed using TruSeq stranded mRNA (Illumina) from oligo-dT captured mRNAs from depleted samples. Libraries were run in NextSequation 500 (Illumina) at 75 nt single read using High Output 75 cycles kit v2.0 (Illumina).</p><p id="p-12">The treatment of the RNA libraries was done following a previous study in which different methods of differential expression analyses were compared (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zhang <em>et al.</em> 2014</a>). RNA libraries were sequenced at the Genomic core facility at Servicio Central de Soporte a la Investigación Experimental (SCSIE), University of Valencia, Spain. Raw reads were analyzed using FastQC report and cleaned with CutAdapt as implemented in RobiNA software package v 1.2.4 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lohse <em>et al.</em> 2012</a>). Low-quality reads were filtered and trimmed (Phred score &lt;20 and size &lt;40 nt were discarded). Since we had a reference transcriptome from S288c strain, reads were then aligned with Bowtie (up to two mismatches accepted) to the reference transcriptome (PRJNA290217) from the reference S288c strain. Statistical assessment of differential gene expression was done either with edgeR (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Robinson <em>et al.</em> 2010</a>) or with DESeq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders and Huber 2010</a>) as implemented in RobiNA. A previous study compared the different expression analysis methods, concluding that edgeR and DESeq were the best-performing methods when the objective is to analyze differential expression (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Zhang <em>et al.</em> 2014</a>). Comparison of logarithmic fold-change of our expression data between edgeR and DESeq provided very strong correlation (Spearman correlation coefficient: <em>ρ</em> = 0.995, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, Figure 1 of Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS8.pdf">File S8</a>). Significant expression changes were identified using a false discovery rate (FDR &lt;0.05). These results indicate that our quantification of expression data is robust to the method used. All newly sequenced RNA sequences are available from the Sequence Read Archive with the following accession number: SRP074821.</p></div><div id="sec-5" class="subsection"><h3>Genetic interaction data</h3><p id="p-13">We used the latest update of the genetic functional chart of <em>S. cerevisiae</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Costanzo <em>et al.</em> 2010</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS4.xls">File S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS5.xls">File S5</a> from <a href="http://drygin.ccbr.utoronto.ca/%E2%88%BCcostanzo/">http://drygin.ccbr.utoronto.ca/∼costanzo/</a>). The genetic map is based on the synthetic genetic array methodology (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tong <em>et al.</em> 2001</a>). In this methodology, synthetic lethal genetic interactions are systematically mapped to single and double mutants. In this study, two genes are considered to interact genetically if the double knockout mutant of the two genes has significantly larger or smaller effect than the multiplicative effects of simple knockouts.</p></div><div id="sec-6" class="subsection"><h3>Software</h3><p id="p-14">Calculations and statistics were performed using MS Excel and R 3.2.1, unless otherwise indicated. Data management was possible using in-house built Perl scripts.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-15">Strains are available upon request. All RNA sequences are available from the Sequence Read Archive (accession number SRP074821). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS1.xls">File S1</a> contains the significant transcriptionally altered genes in <em>S. cerevisiae</em> upon growing in ethanol stress. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS2.xls">File S2</a> contains the significant transcriptionally altered genes in <em>S. cerevisiae</em> upon growing in glycerol stress. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS3.xls">File S3</a> contains the significant transcriptionally altered genes in <em>S. cerevisiae</em> upon growing in acidic stress. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS4.xls">File S4</a> contains the significant transcriptionally altered genes in <em>S. cerevisiae</em> upon growing in oxidative stress. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS5.xls">File S5</a> contains the significant transcriptionally altered genes in <em>S. cerevisiae</em> upon growing in oxidative stress in a growth medium supplemented with dextrose. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS6.xls">File S6</a> contains the list of duplicated genes in <em>Candida glabrata</em>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS7.xlsx">File S7</a> contains the conservation indices of promoter alignments for duplicated genes with altered transcriptional profiles. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS8.pdf">File S8</a> compares the methods edgeR and DESeq for the calculation of reads mapped to each gene.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-16">To test the role of duplicated genes of <em>S. cerevisiae</em> in the origin of adaptations, we sequenced the transcriptome of a haploid <em>msh</em>2 deletion strain after growing it in normal YPD medium and under five different stress conditions: (a) ethanol, (b) glycerol, (c) lactate, (d) oxidative stress, and (e) oxidative stress in a medium supplemented with dextrose (see <em>Materials and Methods</em>). Subsequently, we compared the transcriptional modifications of <em>S. cerevisiae <a href="http://www.yeastgenome.org/locus/S000005450/overview">msh2</a></em>::<em>kanMX4</em> under each of the conditions and sought to investigate the role of duplicated genes in displaying transcriptional plasticity under stress. We used this strain because it has been allowed to evolve for hundreds of generations in YPD medium, hence is adapted to this medium, and allows the maintenance of population genetic polymorphism due to its higher mutation rate compared to the wild-type strain.</p><div id="sec-9" class="subsection"><h3>Duplicated genes exhibit significant transcriptional plasticity under stress</h3><p id="p-17">After growing biological replicates of the yeast populations under normal and each of the five different stress conditions, we extracted total RNA for RNAseq library construction and identified differentially expressed (DE) genes based on the comparison of their expression levels relative to normal conditions (see <em>Materials and Methods</em>). In total, we obtained reliable RNA sequence data for 5825 genes in the YPD medium (normal conditions) and each of the five stress conditions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS1.xls">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS2.xls">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS3.xls">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS4.xls">File S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS5.xls">File S5</a>), of which an important fraction was significantly altered under stress conditions (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470149/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470149/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Transcription alterations under stress conditions</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">In all stress conditions, there was a significant transcriptomic response affecting 2248, 2827, 2509, 101, and 2638 genes under stress induced by ethanol, glycerol, lactate, oxidative stress, and oxidative stress in a medium supplemented with dextrose, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS1.xls">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS2.xls">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS3.xls">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS4.xls">File S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS5.xls">File S5</a>). The response affected duplicates significantly more than singletons in all stress conditions, although the low number of altered genes limited the power of the test in the case of oxidative stress (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The mechanism of duplication, including WGD and SSD, also made a difference, with WGDs being more significantly enriched for altered expression genes than SSDs (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Taking all nonredundant transcriptomic responses together for all stress conditions, duplicated genes showed significantly larger increments of transcription under stress than singleton genes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Indeed, on average 837 duplicated genes out of the 2240 duplicates (37.4%) in <em>S. cerevisiae</em> exhibited significant increments of expression against 1227 out of 4580 singletons (26.8%) (Fisher’s exact test: odds ratio <em>F</em> = 1.63, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). We identified an average of 42.2%, over all the stresses of 464 duplicates out of 1100 WGDs, to be transcriptionally plastic, a proportion significantly higher than that for singletons (Fisher’s exact test: odds ratio <em>F</em> = 1.99, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). Likewise, the proportion of transcriptionally plastic SSDs (an average of 372 out of 1140 SSDs, 32.6%) was significantly higher than that for singletons (Fisher’s exact test: odds ratio <em>F</em> = 1.32, <em>P</em> = 1.1 × 10<sup>−4</sup>). WGDs also presented a significantly higher proportion of transcriptionally plastic genes than SSDs (Fisher’s exact test: odds ratio <em>F</em> = 1.51, <em>P</em> = 3.5 × 10<sup>−6</sup>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/63/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Duplicated genes exhibit higher transcriptional plasticity than singleton genes and are involved in adaptation. (A) Percentage of genes with transcriptional flexibility when S. cerevisiae is grown under each of the five stress conditions tested in this study: ethanol stress, glycerol stress, acidic stress by lactate, oxidative stress, and oxidative stress in a medium supplemented with dextrose. (B) The mean percentage of genes of the categories&#x2019; singletons (black bar), duplicates (orange bar), duplicates generated by whole genome duplication (red bar), and duplicates generated by small-scale duplication (light yellow bar) with transcriptional alterations in the five stress conditions tested in this study. (C&#x2013;F) We measured the contribution of transcriptionally altered duplicates to the fitness of S. cerevisiae under YPD and stress growth conditions using knock down gene data from Steinmetz et al. (2002). We then compared the fitness contribution of these altered duplicates (red boxes) with that of duplicates with no evidence for transcriptional plasticity under stress (blue boxes). These comparisons were performed for the sets of altered and not-altered duplicates identified under ethanol stress (C), glycerol stress (D), lactate stress (E), and oxidative stress supplemented with dextrose (F). Significant differences are indicated with *, **, and *** when the difference was significant at the levels of 0.01, 0.001, and 0.0001, respectively, using a Mann&#x2013;Whitney U-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408799095" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Duplicated genes exhibit higher transcriptional plasticity than singleton genes and are involved in adaptation. (A) Percentage of genes with transcriptional flexibility when &lt;em&gt;S. cerevisiae&lt;/em&gt; is grown under each of the five stress conditions tested in this study: ethanol stress, glycerol stress, acidic stress by lactate, oxidative stress, and oxidative stress in a medium supplemented with dextrose. (B) The mean percentage of genes of the categories&#x2019; singletons (black bar), duplicates (orange bar), duplicates generated by whole genome duplication (red bar), and duplicates generated by small-scale duplication (light yellow bar) with transcriptional alterations in the five stress conditions tested in this study. (C&#x2013;F) We measured the contribution of transcriptionally altered duplicates to the fitness of &lt;em&gt;S. cerevisiae&lt;/em&gt; under YPD and stress growth conditions using knock down gene data from Steinmetz &lt;em&gt;et al.&lt;/em&gt; (2002). We then compared the fitness contribution of these altered duplicates (red boxes) with that of duplicates with no evidence for transcriptional plasticity under stress (blue boxes). These comparisons were performed for the sets of altered and not-altered duplicates identified under ethanol stress (C), glycerol stress (D), lactate stress (E), and oxidative stress supplemented with dextrose (F). Significant differences are indicated with *, **, and *** when the difference was significant at the levels of 0.01, 0.001, and 0.0001, respectively, using a Mann&#x2013;Whitney &lt;em&gt;U&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/63/F1.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/1/63/F1.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/63/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/63/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470190" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Duplicated genes exhibit higher transcriptional plasticity than singleton genes and are involved in adaptation. (A) Percentage of genes with transcriptional flexibility when <em>S. cerevisiae</em> is grown under each of the five stress conditions tested in this study: ethanol stress, glycerol stress, acidic stress by lactate, oxidative stress, and oxidative stress in a medium supplemented with dextrose. (B) The mean percentage of genes of the categories’ singletons (black bar), duplicates (orange bar), duplicates generated by whole genome duplication (red bar), and duplicates generated by small-scale duplication (light yellow bar) with transcriptional alterations in the five stress conditions tested in this study. (C–F) We measured the contribution of transcriptionally altered duplicates to the fitness of <em>S. cerevisiae</em> under YPD and stress growth conditions using knock down gene data from <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Steinmetz <em>et al.</em> (2002</a>). We then compared the fitness contribution of these altered duplicates (red boxes) with that of duplicates with no evidence for transcriptional plasticity under stress (blue boxes). These comparisons were performed for the sets of altered and not-altered duplicates identified under ethanol stress (C), glycerol stress (D), lactate stress (E), and oxidative stress supplemented with dextrose (F). Significant differences are indicated with *, **, and *** when the difference was significant at the levels of 0.01, 0.001, and 0.0001, respectively, using a Mann–Whitney <em>U</em>-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">An alternative explanation for the adaptive value of the transcriptional plasticity of duplicates under stress is a low contribution of duplicates to fitness, with their response being the reflection of transcriptional noise caused by environmental perturbations. To test this possibility, we calculated the contribution of each duplicated gene to the fitness of <em>S. cerevisiae</em> under normal and stress conditions taking previously published fitness data (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Steinmetz <em>et al.</em> 2002</a>). To this end, we subtracted the normalized fitness values of a strain after a gene was deleted or knocked down under stress from the fitness of its ancestral strain (<em>i.e.</em>, 1). Therefore, large fitness absolute increment values indicate that the contribution of the gene to fitness is high under those conditions.</p><p id="p-25">We first compared the contribution to fitness of duplicates with and without altered transcriptomic profiles in YPD. Transcriptionally altered duplicates exhibited higher contribution to fitness than unaltered duplicates (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, C–F</a>), discarding the possibility that altered duplicates may have less contribution to fitness than not-altered duplicates consequently being less selectively constrained to change. Both transcriptionally altered and not-altered duplicates showed a significant increase in their contribution to fitness under stress (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, C–F</a>). However, this increase was sharper in transcriptionally altered duplicates than in unaltered duplicates.</p></div><div id="sec-10" class="subsection"><h3>Increased transcriptional plasticity after gene duplication</h3><p id="p-26">The higher transcriptional plasticity of duplicated genes in <em>S. cerevisiae</em> when compared to singletons may be the result of a biased preservation in duplicate of highly transcriptionally plastic genes. To test whether gene duplication increases transcriptional plasticity we examined the patterns of transcriptional plasticity of duplicates and singletons in the post-WGD yeast <em>C. glabrata</em>, a phylogenetically close species to <em>S. cerevisiae</em>. To this end, we asked the question of whether duplicates of <em>S. cerevisiae</em> had singleton orthologs in <em>C. glabrata</em> that were not more transcriptionally plastic than expected when compared to other <em>C. glabrata</em> singletons under stress and vice versa. We obtained RNA sequence data from a previous publication in which transcriptomic data were available under YPD conditions and under acidic stress (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Linde <em>et al.</em> 2015</a>), similar to our data on lactic acid stress. <em>S. cerevisiae</em> orthologs from <em>C. glabrata</em> were identified using syntheny information available in the pillars of YGOB (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Byrne and Wolfe 2005</a>). In total, we identified 4844 reliable <em>S. cerevisiae</em>: <em>C. glabrata</em> orthologs. Of these 4844 orthologs, 788 genes in <em>C. glabrata</em> were duplicated genes (394 pairs, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS6.xls">File S6</a>), of which 123 were duplicated in <em>C. glabrata</em> but not in <em>S. cerevisiae</em>. Of the 2240 duplicates of <em>S. cerevisiae</em>, we found 1019 orthologs that were singletons in <em>C. glabrata</em>. We first asked whether singletons in <em>C. glabrata</em> that are orthologs of duplicates in <em>S. cerevisiae</em> exhibit higher transcriptional plasticity than singletons in <em>C. glabrata</em> with no duplicate orthologs in <em>S. cerevisiae</em>. If this were the case, then gene duplication would have no role in transcriptional plasticity in <em>S. cerevisiae</em>. Notwithstanding that the transcriptional plasticity for a particular gene may vary among species, we found that the percentage of singletons with significant transcriptional alterations under stress in <em>C. glabrata</em> that are orthologs to <em>S. cerevisiae</em> duplicates (599 out of a total of 1019 genes, 58.7%) was not significantly higher than that of transcriptionally altered singletons in <em>C. glabrata</em> that had no duplicate orthologs in <em>S. cerevisiae</em> (1725 out of a total of 3062 singleton genes, 56.3%) (Fisher exact test: odd’s ratio <em>F</em> = 1.10, <em>P</em> = 0.17). Conversely, duplicates in <em>C</em><em>. glabrata</em> that were orthologs to singletons in <em>S. cerevisiae</em> exhibited a significantly higher percentage of transcriptionally altered genes under stress (82 out of a total of 123 genes, 66.7%) than singletons in <em>C. glabrata</em> (Fisher’s exact test: odd’s ratio <em>F</em> = 1.55, <em>P</em> = 0.02).</p></div><div id="sec-11" class="subsection"><h3>Differential patterns of transcriptional alterations within duplicated genes</h3><p id="p-27">We sought to investigate the different transcriptional profiles of pairs of duplicated genes and their contributions to the fitness of <em>S. cerevisiae</em>. We divided duplicated genes that underwent transcriptional alterations after stress into five different categories (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>): (a) duplicates in which both of the gene copies were up-regulated under stress (called herein Up pattern); (b) duplicates with both copies down-regulated under stress (Down pattern); (c) duplicates with one copy up-regulated and one copy down-regulated under stress (Discordant pattern), (d) duplicates with one copy showing not-altered transcription under stress while its sister copy shows either up-regulation or down-regulation under stress (Only-one pattern), and (e) duplicates that remained unchanged under stress (Not-altered pattern).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/63/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Duplicated genes exhibit differential patterns of transcriptional plasticity under stress. We identified five patterns of transcriptional plasticity for the duplicates of S. cerevisiae growing under stress conditions (A), including duplicates in which neither copy has been altered (category A), those with both copies up-regulated (category B), those with both copies down-regulated (category C), those with gene copies showing discordant transcriptional plasticities (category D), and those in which only one gene copy is altered while its sister copy is unaltered under stress (category E). Calculating the percentage of the duplicated genes belonging to each of the transcriptional categories (B), we found that the category with only one copy altered (yellow bar) is the one showing the highest percentage of the altered duplicates under stress. (C) We measured the conservation of the promoter regions for altered and not-altered duplicates and found that altered duplicates (black bar) exhibit lower conservation than not-altered duplicates under stress (white bar)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408799095" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Duplicated genes exhibit differential patterns of transcriptional plasticity under stress. We identified five patterns of transcriptional plasticity for the duplicates of &lt;em&gt;S. cerevisiae&lt;/em&gt; growing under stress conditions (A), including duplicates in which neither copy has been altered (category A), those with both copies up-regulated (category B), those with both copies down-regulated (category C), those with gene copies showing discordant transcriptional plasticities (category D), and those in which only one gene copy is altered while its sister copy is unaltered under stress (category E). Calculating the percentage of the duplicated genes belonging to each of the transcriptional categories (B), we found that the category with only one copy altered (yellow bar) is the one showing the highest percentage of the altered duplicates under stress. (C) We measured the conservation of the promoter regions for altered and not-altered duplicates and found that altered duplicates (black bar) exhibit lower conservation than not-altered duplicates under stress (white bar).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/63/F2.medium.gif" width="305" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/1/63/F2.medium.gif" width="305" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/63/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/63/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470158" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Duplicated genes exhibit differential patterns of transcriptional plasticity under stress. We identified five patterns of transcriptional plasticity for the duplicates of <em>S. cerevisiae</em> growing under stress conditions (A), including duplicates in which neither copy has been altered (category A), those with both copies up-regulated (category B), those with both copies down-regulated (category C), those with gene copies showing discordant transcriptional plasticities (category D), and those in which only one gene copy is altered while its sister copy is unaltered under stress (category E). Calculating the percentage of the duplicated genes belonging to each of the transcriptional categories (B), we found that the category with only one copy altered (yellow bar) is the one showing the highest percentage of the altered duplicates under stress. (C) We measured the conservation of the promoter regions for altered and not-altered duplicates and found that altered duplicates (black bar) exhibit lower conservation than not-altered duplicates under stress (white bar).</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470026/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470026/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Categories of altered expression of duplicates</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">In each of the stress conditions, the category “Only-one” comprised the largest number of duplicates with altered transcriptional profiles, with this category including 53–97% of the altered duplicates in the five stresses (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). These results support the classical view of evolution by gene duplication, according to which following gene duplication one copy undergoes rapid divergence while the other copy keeps the ancestral function. Here we show that this pattern of evolution by gene duplication is also true for the regulatory evolution of duplicated genes.</p><p id="p-30">Duplicate gene copies with higher transcriptional divergence under stress should show higher sequence divergence when compared to orthologous sequences from other phylogenetically related species if the basis for this transcriptional plasticity was encoded in the gene sequence. To test this hypothesis, we were able to obtain 537 reliable promoter alignments for duplicates in <em>S. cerevisiae</em> and at least four additional phylogenetically related yeast species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1/FileS7.xlsx">File S7</a>). The main <em>Saccharomyces</em> species we compared <em>S. cerevisiae</em> to were <em>S. bayanus</em>, <em>S. castellii</em>, <em>S. mikatae</em>, <em>S. paradoxus</em>, <em>S. kluyveri</em>, and <em>S. kudriavzevii</em>. Not all species presented annotated intergenic regions but we used all those alignments that included at least four of the species. We aligned the 600 nucleotide sequence regions upstream of duplicated genes and their orthologs, as these are likely to include most if not all the regulatory elements of the genes (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ohler and Niemann 2001</a>). We then measured the coefficient of conservation (CC) for each nucleotide site using the entropy equation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cover and Thomas 2006</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Halabi <em>et al.</em> 2009</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Ruiz-Gonzalez and Fares 2013</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/63/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/63/embed/mml-math-1.gif"/></noscript></span></span>In this equation, CC of a nucleotide (<em>a</em>) at position (<em>k</em>) in an alignment is defined as the entropy of the observed frequency of <em>a</em> at <em>k</em> (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/63/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/63/embed/mml-math-2.gif"/></noscript></span></span>) relative to the background frequency of <em>a</em> in all sequences of the alignment (<em>q<sup>(a)</sup></em>). Therefore, the more conserved the site the higher is its CC value. CC was averaged for each promoter and then these averages were used to compare altered duplicates (those belonging to the categories “Up,” “Down,” “Discordant,” and “Only-one”) with not-altered duplicates (those belonging to the category “Not-altered”). For each of the stress conditions we estimated the CC values for altered and not-altered duplicates. We then pulled all the data together from all stress conditions and compared the CC values of altered to that of not-altered duplicates. The CC values of duplicates with constant transcriptional profiles under stress (mean ± SE = 0.76 ± 0.005) were significantly larger than those of duplicates with altered transcriptional profiles (mean ± SE = 0.72 ± 0.01) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>), and the difference was significant using a parametric test (<em>t</em>-test: <em>t</em> = 3.47, d.f. = 1140.9, <em>P</em> = 5.4 × 10<sup>−4</sup>) and a nonparametric test (Mann–Whitney <em>U</em>-test: <em>P</em> = 0.003), indicating that higher transcriptional plasticity of duplicates may be due to a divergence in their promoter sequences from the ancestral preduplication state.</p></div><div id="sec-12" class="subsection"><h3>Duplicates with different transcriptional divergence patterns exhibit different functional dependencies</h3><p id="p-31">To determine whether the transcriptional plasticity of duplicated genes is accompanied by a functional divergence of gene copies, we analyzed the genetic interaction network of <em>S. cerevisiae</em> and asked how many of the duplicated genes show genetic interactions between their gene copies, hence are functionally dependent on one another (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Costanzo <em>et al.</em> 2010</a>), within each of the transcriptional categories (<em>i.e.</em>, Up, Down, Discordant, Only one, and Not-altered). To this end, we used the genetic interaction map of <em>S. cerevisiae</em> as a proxy to the functions of each of the genes (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Costanzo <em>et al.</em> 2010</a>). This map contains roughly 6.5 million genetic interactions and the functional chart for 75% of the <em>S. cerevisiae</em> genes. The number of genetic interactions for a particular gene is a proxy to the number of functions it performs, as the deletion of both of the genes identified as interacting produces significantly different fitness effects than the multiplicative effect of single gene deletions (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Costanzo <em>et al.</em> 2010</a>). We identified 762,768 significant genetic interactions (<em>i.e.</em>, epistasis, ε) in <em>S. cerevisiae</em>, of which 52% were synergistic (<em>i.e.</em>, the double mutant exhibited significantly lower fitness W<sub>12</sub> than the multiplicative effects of individual mutants: ε = W<sub>12</sub> – W<sub>1</sub>W<sub>2</sub>; ε &lt; 0) and 48% were antagonistic interactions (ε = W<sub>12</sub> – W<sub>1</sub>W<sub>2</sub>; ε &gt; 0). However, duplicated genes were largely biased regarding the sign epistasis, with the majority of the epistasis (89.5%) being synergistic (binomial test: <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). This pattern was also true for transcriptionally altered duplicates (89.74% synergistic epistasis). Dividing transcriptionally altered duplicates into the different categories provides similar results, with all such categories being equally enriched for duplicates with synergistic epistasis: up-regulated duplicates presented largely synergistic epistasis (varying between 86% in ethanol and 93% under oxidative stress supplemented with dextrose), and so did the only-one category (ranging between 87% of the interactions being synergistic under glycerol stress and 92.7% in ethanol stress). These percentages were of the same order in the “Down” and “Discordant” categories.</p><p id="p-32">In all stress conditions the category “Down” showed the highest enrichment for those duplicates with interacting gene copies (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). On average over all stress conditions, the genetically interacting duplicates enrichment followed the same pattern, with a distribution among the categories in the following decreasing manner: the category “Down,” followed by the category “Not-altered,” then the category “Only-one,” then the category “Discordant,” and finally the category “Up” (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, inset box and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/63/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The genetic dependencies between gene copies of transcriptionally plastic duplicates. We measured the number of pairs within each of the transcriptional categories with evidence of genetic interactions between duplicate gene copies using the functional landscape of S. cerevisiae (Costanzo et al. 2010). (A) The percentage of pairs with interacting gene copies was very high in the category of down-regulated duplicates (red bars), very low in the category of up-regulated duplicates (blue bars), and intermediate in the other three categories under all the stress conditions examined in this study. The mean percentage of duplicates with interacting copies across the stress conditions for each transcriptional category is presented in the inset box. The category &#x201C;Down&#x201D; presented a larger mean percentage of duplicates whose gene copies are functionally dependent upon one another than any of the other categories. Significant differences are indicated with ** and *** when the probabilities are P &lt; 0.01, P &lt; 0.001, and P &lt; 10&#x2212;4, respectively. (B) The average proportion of duplicates with interacting gene copies was compared to the proportion of genetic interactions for sets of randomly sampled pairs of singletons with altered transcription profiles under stress. Each transcriptional profile for duplicates was compared to an equivalent set of random pairs of singletons with similar transcriptional profiles. For example, up-regulated duplicates were compared to random pairs of up-regulated singletons under stress." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408799095" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The genetic dependencies between gene copies of transcriptionally plastic duplicates. We measured the number of pairs within each of the transcriptional categories with evidence of genetic interactions between duplicate gene copies using the functional landscape of &lt;em&gt;S. cerevisiae&lt;/em&gt; (Costanzo &lt;em&gt;et al.&lt;/em&gt; 2010). (A) The percentage of pairs with interacting gene copies was very high in the category of down-regulated duplicates (red bars), very low in the category of up-regulated duplicates (blue bars), and intermediate in the other three categories under all the stress conditions examined in this study. The mean percentage of duplicates with interacting copies across the stress conditions for each transcriptional category is presented in the inset box. The category &#x201C;Down&#x201D; presented a larger mean percentage of duplicates whose gene copies are functionally dependent upon one another than any of the other categories. Significant differences are indicated with ** and *** when the probabilities are &lt;em&gt;P&lt;/em&gt; &lt; 0.01, &lt;em&gt;P&lt;/em&gt; &lt; 0.001, and &lt;em&gt;P&lt;/em&gt; &lt; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;, respectively. (B) The average proportion of duplicates with interacting gene copies was compared to the proportion of genetic interactions for sets of randomly sampled pairs of singletons with altered transcription profiles under stress. Each transcriptional profile for duplicates was compared to an equivalent set of random pairs of singletons with similar transcriptional profiles. For example, up-regulated duplicates were compared to random pairs of up-regulated singletons under stress.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/63/F3.medium.gif" width="358" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/1/63/F3.medium.gif" width="358" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/63/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/63/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470115" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">The genetic dependencies between gene copies of transcriptionally plastic duplicates. We measured the number of pairs within each of the transcriptional categories with evidence of genetic interactions between duplicate gene copies using the functional landscape of <em>S. cerevisiae</em> (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Costanzo <em>et al.</em> 2010</a>). (A) The percentage of pairs with interacting gene copies was very high in the category of down-regulated duplicates (red bars), very low in the category of up-regulated duplicates (blue bars), and intermediate in the other three categories under all the stress conditions examined in this study. The mean percentage of duplicates with interacting copies across the stress conditions for each transcriptional category is presented in the inset box. The category “Down” presented a larger mean percentage of duplicates whose gene copies are functionally dependent upon one another than any of the other categories. Significant differences are indicated with ** and *** when the probabilities are <em>P</em> &lt; 0.01, <em>P</em> &lt; 0.001, and <em>P</em> &lt; 10<sup>−4</sup>, respectively. (B) The average proportion of duplicates with interacting gene copies was compared to the proportion of genetic interactions for sets of randomly sampled pairs of singletons with altered transcription profiles under stress. Each transcriptional profile for duplicates was compared to an equivalent set of random pairs of singletons with similar transcriptional profiles. For example, up-regulated duplicates were compared to random pairs of up-regulated singletons under stress.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470067/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470067/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of duplicates with genetically interacting gene copies for each transcriptional category of duplicates</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">The strong genetic interaction between the gene copies could be due to either each gene copy having a large fitness effect such that deleting both magnifies such an effect, or each gene copy having very low fitness effects due to genetic redundancy but deleting both significantly magnifies this effect (<em>i.e</em>., functional compensation of a gene deletion or both gene copies are needed to perform the function because they have subfunctionalized). The category “Up” is the one with the lowest number of gene copy interactions, therefore is likely to contain very little genetic compensation, perhaps because gene copies have diverged in their function from the ancestral preduplication gene, and as such the multiplicative effect of deleting single gene copies may be as important in their contribution to fitness as the double gene deletions. The category “Discordant” shows higher levels of genetic interactions between gene copies than the category “Up,” but lower levels than “Down.” Since all discordant duplicates exhibit synergistic epistasis, this suggests certain functional redundancy under normal conditions for transcriptionally discordant duplicates, which also applies to the categories of “One-altered” (average percentage of synergistic epistasis among all stresses: 89.9%; binomial test: <em>P</em> &lt; 3.61 × 10<sup>−7</sup>) and “Only-one” duplicates.</p><p id="p-35">To determine whether duplicate gene copies are more dependent upon each other’s functions than expected, we built sets of singleton genes for each of the duplicate sets according to their transcriptional profiles. Each of the singleton transcriptional categories was built taking random pairs of singleton genes. For example, for the “Up” category, both of the singleton genes were sampled from the set of up-regulated singleton genes under stress. We built sets of 1000 pairs and compared each of the duplicate transcriptional categories with the corresponding singleton transcriptional categories. Results show that all the categories, with the exception of the one including duplicates with both gene copies up-regulated, exhibit a significant proportion of their duplicates with interacting gene copies when compared to singletons of the same transcriptional category (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Therefore, up-regulated duplicates seem to exhibit evidence of independent evolution of their gene copies likely due to the finding of novel functions by each copy under stress.</p></div><div id="sec-13" class="subsection"><h3>Functional divergence and genetic redundancy of duplicated genes</h3><p id="p-36">The differences in the functional dependencies between gene copies found in the duplicate transcriptional categories hint at a different mode of evolution by gene duplication for these categories. We hypothesize, based on the patterns of genetic interactions, that the functional fate of duplicates in terms of neo- or subfunctionalization is dependent on the transcriptional category they belong to and the genetic redundancy between gene copies. Genetic redundancy has been shown to correlate with evolvability because it provides mutational robustness, which in turn increases the evolvability of genes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Draghi <em>et al.</em> 2010</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wagner 2000b</a>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">2005</a>).</p><p id="p-37">To test whether a given transcriptional category of duplicates is more likely to have evolved neo- or subfunctionalization, we examined two parameters linked to genetic interactions: (a) the number of shared interactions between the gene copies, (b) the number of total interactions of the gene copies. Neo-functionalized duplicates involve those in which one of the gene copies has lost all ancestral functions and acquired new functions, hence is likely to have a reduced number of genetic interactions. Conversely, subfunctionalization should affect duplicates with many functions in which each copy has become specialized in a set of ancestral functions while sharing common functions with its sister gene copy, hence is likely to be overrepresented among highly interacting duplicates. Neo-functionalization should also lead to lower levels of sharing of genetic interactions between the gene copies as one of the copies has acquired novel functions that are perhaps independent from subfunctionalization. In agreement with our hypotheses, duplicates from the down-regulated category exhibited a greater number of genetic interactions (mean ± SE: 393.39 ± 14.17) than those of the up-regulated category (mean ± SE: 313.95 ± 13.95; <em>t</em>-test: <em>t</em> = 3.99, d.f. = 551.94, <em>P</em> = 7.36 × 10<sup>−5</sup>). The index of shared interactions was calculated as: <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/63/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/63/embed/mml-math-3.gif"/></noscript></span></span>, with <em>Sh<sub>A,B</sub></em> referring to the mean number of shared interactions between gene copies A and B, <em>Sh</em> referring to the number of shared interactions, and <em>N</em> being the total number of interactions. Duplicates from the down-regulated category shared more interactions (mean ± SE: 0.15 ± 0.005) than those of the up-regulated category (mean ± SE: 0.12 ± 0.004; <em>t</em>-test: <em>t</em> = 3.29, d.f. = 209.27, <em>P</em> = 1.1 × 10<sup>−3</sup>). These results indicate that while up-regulated duplicates may have neo-functionalized, down-regulated duplicates have likely subfunctionalized.</p></div><div id="sec-14" class="subsection"><h3>Sequence divergence levels of duplicates correlate with their transcriptional profiles</h3><p id="p-38">To determine whether the transcriptional duplicate categories included specific functional divergence profiles between gene copies, we inferred the amino acid distances between duplicate gene copies for all transcriptional categories and stress conditions. Divergence between duplicate gene copies was calculated using Poisson-corrected distances. Under all four stresses, the duplicates of category “Down” presented the lowest distance between gene copies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), followed by the category “Discordant,” then the category “Only-one,” then the category “Up,” and finally the category “Not-altered” (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Taking all stresses together, we found three groups of transcriptional categories according to the divergence values between gene copies of duplicates (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). The first category is “Down”: this category exhibited the lowest divergence levels between gene copies which was significantly smaller than the following group that included “Discordant” category duplicates (median divergence values for “Down”: 0.05, median for “Discordant”: 0.12, Wilcoxon rank test: <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). The following group included duplicates belonging to the transcriptional categories of “Only-one” (median divergence between gene copies: 0.22), “Up” (median divergence between gene copies: 0.24), and “Not-altered” (median divergence of gene copies: 0.33). The category “Discordant” exhibited significantly lower divergence between gene copies than the categories “Up,” “Only-one,” and “Not-altered” (Wilcoxon rank test: <em>P</em> &lt; 2.2 × 10<sup>−16</sup>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/63/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Functional divergence analysis of duplicates with different patterns of transcriptional plasticity. (A) Poisson-corrected amino acid distance between gene copies of duplicates for each of the transcriptional plasticity profiles (&#x201C;Up,&#x201D; &#x201C;Down,&#x201D; &#x201C;Discordant,&#x201D; &#x201C;Only one,&#x201D; and &#x201C;Not-altered&#x201D;). (B) Comparison of the divergence levels between gene copies for the different transcriptional profiles. (C) Correlation analysis between the percentage of of duplicates with interacting gene copies and the divergence levels between the copies (red, blue, light yellow, yellow, and orange circles refer to the duplicate class of &#x201C;Down,&#x201D; &#x201C;Up,&#x201D; &#x201C;Discordant,&#x201D; &#x201C;One-altered,&#x201D; and &#x201C;Not-altered,&#x201D; respectively)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408799095" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Functional divergence analysis of duplicates with different patterns of transcriptional plasticity. (A) Poisson-corrected amino acid distance between gene copies of duplicates for each of the transcriptional plasticity profiles (&#x201C;Up,&#x201D; &#x201C;Down,&#x201D; &#x201C;Discordant,&#x201D; &#x201C;Only one,&#x201D; and &#x201C;Not-altered&#x201D;). (B) Comparison of the divergence levels between gene copies for the different transcriptional profiles. (C) Correlation analysis between the percentage of of duplicates with interacting gene copies and the divergence levels between the copies (red, blue, light yellow, yellow, and orange circles refer to the duplicate class of &#x201C;Down,&#x201D; &#x201C;Up,&#x201D; &#x201C;Discordant,&#x201D; &#x201C;One-altered,&#x201D; and &#x201C;Not-altered,&#x201D; respectively).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/63/F4.medium.gif" width="440" height="205"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/1/63/F4.medium.gif" width="440" height="205"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/63/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/63/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470073" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Functional divergence analysis of duplicates with different patterns of transcriptional plasticity. (A) Poisson-corrected amino acid distance between gene copies of duplicates for each of the transcriptional plasticity profiles (“Up,” “Down,” “Discordant,” “Only one,” and “Not-altered”). (B) Comparison of the divergence levels between gene copies for the different transcriptional profiles. (C) Correlation analysis between the percentage of of duplicates with interacting gene copies and the divergence levels between the copies (red, blue, light yellow, yellow, and orange circles refer to the duplicate class of “Down,” “Up,” “Discordant,” “One-altered,” and “Not-altered,” respectively).</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Importantly, the mean sequence divergence levels between the duplicate gene copies correlated negatively with the mean percentage of duplicates in which both gene copies interacted genetically (Pearson correlation: <em>ρ</em> = −0.54, <em>P</em> = 0.013; <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>), indicating that the larger the divergence between the gene copies the lower is their functional dependency. This correlation became more significant when taking only those duplicates for which at least one gene copy has showed changing transcriptional patterns under stress (Pearson correlation: <em>ρ</em> = −0.78, <em>P</em> = 3.4 × 10<sup>−4</sup>). This result is in agreement with a previous study in which the differences between pairs of WGDs in which both gene copies interacted genetically and those in which gene copies did not was analysed (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Musso <em>et al.</em> 2008</a>).</p><p id="p-41">These results strongly suggest that gene copies that become up-regulated (category “Up”) under stress have undergone accelerated evolution and divergence from their ancestral, preduplication functions perhaps allowing the adaptation to stress conditions that are often encountered by the cell in nature.</p></div><div id="sec-15" class="subsection"><h3>The origin of specific and general adaptations in S. cerevisiae</h3><p id="p-42">To determine whether the transcriptional plasticity of duplicates is the result of an adaptive process to face environmental perturbations, we sought to investigate whether this plasticity is stress specific (<em>i.e.</em>, the result of adaptive processes) or a general response to stress. We examined common transcriptionally altered duplicates for each of the transcriptional categories among stress conditions. We found that a substantial proportion of duplicates showed stress-specific transcriptional plasticity (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). This pattern was the inverse in the case of transcriptionally altered singletons, with many common such singletons responding to all four stress conditions (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Comparison of the proportion of duplicates in each of the categories for stress response (<em>i.e.</em>, stress specific, common genes response to two, three, or four stresses) revealed more significant stress-specific transcriptional alterations in duplicates than in singletons (a mean of 34.4% of duplicates with transcriptional flexibility were stress specific compared to 26% of singletons, Fisher’s exact test: odds ratio <em>F</em> = 1.47, <em>P</em> = 3.3 × 10<sup>−4</sup>), while singletons showed more common responses to all stresses than duplicates (a mean of 32.8% of singletons responded to all stress conditions compared to 23% of duplicates, Fisher’s exact test: odds ratio <em>F</em> = 1.64, <em>P</em> = 1.1 × 10<sup>−5</sup>) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). These results reveal a fundamental difference in the transcriptional plasticity of duplicates and singletons, with evidence for the role of natural selection in duplicates’ transcriptional differences as an adaptive mechanism.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/63/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcriptional plasticity of duplicates is stress-specific. We analyzed the distribution of transcriptionally altered duplicates in the different stress conditions. (A) Proportion of the transcriptionally altered duplicates from each of the duplicates classes (&#x201C;Up,&#x201D; &#x201C;Down,&#x201D; &#x201C;Discordant,&#x201D; and &#x201C;Only one&#x201D;) that are altered in one stress only, two stresses, three stresses, or in all four stresses tested in this study. (B) Proportion of singletons that are up- or down-regulated that respond to specifically one stress only, two stresses, three stresses, or all four stresses. (C) The mean percentage of altered genes across the different transcriptional classes in duplicates and singletons that are altered under one or more type of stress. (D) The mean percentage of whole-genome duplicates and small-scale duplicates that are transcriptionally altered when S. cerevisiae is faced with one or more stresses. *** indicates P &lt; 0.001 under a Fisher&#x2019;s exact test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408799095" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transcriptional plasticity of duplicates is stress-specific. We analyzed the distribution of transcriptionally altered duplicates in the different stress conditions. (A) Proportion of the transcriptionally altered duplicates from each of the duplicates classes (&#x201C;Up,&#x201D; &#x201C;Down,&#x201D; &#x201C;Discordant,&#x201D; and &#x201C;Only one&#x201D;) that are altered in one stress only, two stresses, three stresses, or in all four stresses tested in this study. (B) Proportion of singletons that are up- or down-regulated that respond to specifically one stress only, two stresses, three stresses, or all four stresses. (C) The mean percentage of altered genes across the different transcriptional classes in duplicates and singletons that are altered under one or more type of stress. (D) The mean percentage of whole-genome duplicates and small-scale duplicates that are transcriptionally altered when &lt;em&gt;S. cerevisiae&lt;/em&gt; is faced with one or more stresses. *** indicates &lt;em&gt;P&lt;/em&gt; &lt; 0.001 under a Fisher&#x2019;s exact test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/63/F5.medium.gif" width="179" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/1/63/F5.medium.gif" width="179" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/63/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/63/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470178" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">Transcriptional plasticity of duplicates is stress-specific. We analyzed the distribution of transcriptionally altered duplicates in the different stress conditions. (A) Proportion of the transcriptionally altered duplicates from each of the duplicates classes (“Up,” “Down,” “Discordant,” and “Only one”) that are altered in one stress only, two stresses, three stresses, or in all four stresses tested in this study. (B) Proportion of singletons that are up- or down-regulated that respond to specifically one stress only, two stresses, three stresses, or all four stresses. (C) The mean percentage of altered genes across the different transcriptional classes in duplicates and singletons that are altered under one or more type of stress. (D) The mean percentage of whole-genome duplicates and small-scale duplicates that are transcriptionally altered when <em>S. cerevisiae</em> is faced with one or more stresses. *** indicates <em>P</em> &lt; 0.001 under a Fisher’s exact test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Because of the fundamental difference in transcriptional plasticities between WGDs and SSDs (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>), we split the dataset for duplicates into these two groups and conducted the same comparison as above. Both the WGDs and SSDs showed very similar transcriptional flexibility patterns to the entire dataset: WGDs and SSDs had their largest transcriptional plasticity in genes that responded in a stress-specific manner (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>).</p><p id="p-45">To understand the relationship between adaptation to stress and transcriptional plasticity, we analyzed how the different transcriptional alterations in duplicates may have an important role in the adaptation to oxidative stress supplemented with dextrose (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Oxidation generates reactive oxidative molecules or species (ROS) including peroxide, superoxide, hydroxyl radicals, and single oxygen in the cell. Increasing ROS in the cell can lead to important structural damage. We found a number of important duplicates that are involved in mitochondrial respiration (<a href="http://www.yeastgenome.org/locus/S000000867/overview"><em>Icl1</em></a>/<a href="http://www.yeastgenome.org/locus/S000006210/overview"><em>Icl2</em></a>, <a href="http://www.yeastgenome.org/locus/S000004154/overview"><em>Shh4</em></a>/<em>Shh</em>1, and <a href="http://www.yeastgenome.org/locus/S000002585/overview"><em>Sdh4</em></a>/<a href="http://www.yeastgenome.org/locus/S000001631/overview"><em>Sdh1</em></a> duplicated genes, among others) as well as duplicated genes encoding transcriptional gluconeogenesis activators (<a href="http://www.yeastgenome.org/locus/S000004893/overview"><em>Cat8</em></a>/<a href="http://www.yeastgenome.org/locus/S000003625/overview"><em>Sip4</em></a> and <a href="http://www.yeastgenome.org/locus/S000006234/overview"><em>Csr2</em></a>/<a href="http://www.yeastgenome.org/locus/S000000197/overview"><em>Ecm21</em></a>) to be up-regulated under oxidative stress, perhaps to reduce the generation of ROS. Moreover, duplicated genes involved in NADH metabolism and oxidative processes in the glycolysis pathway (<a href="http://www.yeastgenome.org/locus/S000002180/overview"><em>Gpd1</em></a>/<a href="http://www.yeastgenome.org/locus/S000005420/overview"><em>Gpd2</em></a>, <a href="http://www.yeastgenome.org/locus/S000001315/overview"><em>Gpp1</em></a>/<a href="http://www.yeastgenome.org/locus/S000000864/overview"><em>Gpp2</em></a>) are down-regulated under stress (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Interestingly, as previously noticed (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Belli <em>et al.</em> 2004</a>), the gene copies (<em><a href="http://www.yeastgenome.org/locus/S000000987/overview">Pug1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000003445/overview">Rta1</a></em>) involved in heme transport and iron ion homeostasis showed discordant expression patterns, while the gene copies (<em><a href="http://www.yeastgenome.org/locus/S000005910/overview">Fit3</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000002942/overview">Fit1</a></em>) involved in ion transport showed transcriptional alterations only for <em><a href="http://www.yeastgenome.org/locus/S000005910/overview">Fit3</a></em> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/63/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of main GO processes and selected duplicate gene pairs affected by oxidative stress in the presence of dextrose. The schematic S. cerevisiae cell has been partitioned in a pie shape with the proportions being in accordance to the distribution of duplicates in categories (&#x201C;Up,&#x201D; &#x201C;Down,&#x201D; &#x201C;Discordant,&#x201D; &#x201C;One-altered,&#x201D; and &#x201C;Not-altered&#x201D;). Color codes follow the same categorization. For those categories, &#x201C;One-altered&#x201D; and &#x201C;Discordant,&#x201D; on which each gene in the pair show a differential fold-change, it has been shadowed with green if not differentially expressed, with blue if down-regulated, or with red if up-regulated. Some of the pathways affected are indicated with the corresponding genes names and ID tag, except for the most common gene names, or when not available a common name and the ID tag is provided. Curved arrows in the nucleus represent some transcription factors affected; color indicates the corresponding category. Subcellular localization of proteins has been retrieved from Saccharomyces Genome Database following Gene Ontology term analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408799095" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic representation of main GO processes and selected duplicate gene pairs affected by oxidative stress in the presence of dextrose. The schematic &lt;em&gt;S. cerevisiae&lt;/em&gt; cell has been partitioned in a pie shape with the proportions being in accordance to the distribution of duplicates in categories (&#x201C;Up,&#x201D; &#x201C;Down,&#x201D; &#x201C;Discordant,&#x201D; &#x201C;One-altered,&#x201D; and &#x201C;Not-altered&#x201D;). Color codes follow the same categorization. For those categories, &#x201C;One-altered&#x201D; and &#x201C;Discordant,&#x201D; on which each gene in the pair show a differential fold-change, it has been shadowed with green if not differentially expressed, with blue if down-regulated, or with red if up-regulated. Some of the pathways affected are indicated with the corresponding genes names and ID tag, except for the most common gene names, or when not available a common name and the ID tag is provided. Curved arrows in the nucleus represent some transcription factors affected; color indicates the corresponding category. Subcellular localization of proteins has been retrieved from Saccharomyces Genome Database following Gene Ontology term analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/63/F6.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/1/63/F6.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/63/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/63/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470145" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-46" class="first-child">Schematic representation of main GO processes and selected duplicate gene pairs affected by oxidative stress in the presence of dextrose. The schematic <em>S. cerevisiae</em> cell has been partitioned in a pie shape with the proportions being in accordance to the distribution of duplicates in categories (“Up,” “Down,” “Discordant,” “One-altered,” and “Not-altered”). Color codes follow the same categorization. For those categories, “One-altered” and “Discordant,” on which each gene in the pair show a differential fold-change, it has been shadowed with green if not differentially expressed, with blue if down-regulated, or with red if up-regulated. Some of the pathways affected are indicated with the corresponding genes names and ID tag, except for the most common gene names, or when not available a common name and the ID tag is provided. Curved arrows in the nucleus represent some transcription factors affected; color indicates the corresponding category. Subcellular localization of proteins has been retrieved from Saccharomyces Genome Database following Gene Ontology term analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">The transcriptional plasticity of the duplicates belonging to the category “Only-one” was very noticeable under oxidative stress affecting functional classes required to minimize ROS, including many duplicated genes involved in ion transport (<em><a href="http://www.yeastgenome.org/locus/S000002942/overview">Fit1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000005910/overview">Fit3</a></em>), hexose transport (<em><a href="http://www.yeastgenome.org/locus/S000002753/overview">Hxt3</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000001138/overview">Hxt5</a></em>), and heat stress response (<em><a href="http://www.yeastgenome.org/locus/S000003947/overview">Ssa2</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000000004/overview">Ssa1</a></em>). Similarly, the transcriptional category “Discordant” also showed a prominent response pattern to oxidative stress, including duplicates involved in nucleotide/nucleoside metabolism (<em><a href="http://www.yeastgenome.org/locus/S000001226/overview">Gnd1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000003488/overview">Gnd2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000555/overview">Apa1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000002938/overview">Apa2</a></em>), and in heme transport and iron ion homeostasis (<em><a href="http://www.yeastgenome.org/locus/S000000987/overview">Pug1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000003445/overview">Rta1</a></em>). Most of these duplicates have important roles in DNA replication and stress.</p><p id="p-48">Interestingly, some of these genes are involved in many stresses but their transcriptional plasticity exhibits different patterns under different stresses. For example, the duplicates (<em><a href="http://www.yeastgenome.org/locus/S000002180/overview">Gpd1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000005420/overview">Gpd2</a></em> and <a href="http://www.yeastgenome.org/locus/S000001315/overview"><em>Gpp1</em></a>/<a href="http://www.yeastgenome.org/locus/S000000864/overview"><em>Gpp2</em></a>) that drive glycerol production using dextrose through the glycolysis pathway (NADH metabolism and oxidative processes) behave transcriptionally different under the different stresses. Both of the gene copies of these duplicates are down-regulated under oxidative stress supplemented with dextrose, while only one gene copy is down-regulated when the cell is subjected to stress by lactate or glycerol, and none of the gene copies showed any differential expression level with the wild type when the cell grew under ethanol stress. Similarly, the gene copies of the duplicates (<em><a href="http://www.yeastgenome.org/locus/S000004154/overview">Shh4</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000002585/overview">Sdh4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001631/overview">Sdh1</a>/<a href="http://www.yeastgenome.org/locus/S000003581/overview">Sdh1b</a></em>) involved in oxidation of succinate and electron transfer to ubiquinone are up-regulated under oxidative stress. However, under ethanol stress, only gene copies <em><a href="http://www.yeastgenome.org/locus/S000004154/overview">Shh4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002585/overview">Sdh4</a></em> are up-regulated while their corresponding paralogs <em>Shh1</em> and <em><a href="http://www.yeastgenome.org/locus/S000001631/overview">Sdh1</a></em> show wild-type expression levels (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-49">In this study, we demonstrate that ancient duplicates of <em>S. cerevisiae</em> exhibit a large transcriptional plasticity when subjected to stress. This transcriptional plasticity may be the result of preadaptations to environmental stress. Such preadaptations may have been generated through an increase in the polymorphism of the regulatory sequence regions of duplicated genes, perhaps the same sequence changes that have led to the regulatory divergence between the gene copies of duplicates. The transcriptional divergence between gene copies has been studied in plants (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Blanc and Wolfe 2004</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Ha <em>et al.</em> 2007</a>, <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">2009</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Wang <em>et al.</em> 2012</a>) and animals (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Huminiecki and Wolfe 2004</a>). In <em>S. cerevisiae</em>, while the genome-wide transcriptional plasticity has been reported under stress (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Causton <em>et al.</em> 2001</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cormier <em>et al.</em> 2010</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Ferea <em>et al.</em> 1999</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Ideker <em>et al.</em> 2001</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Landry <em>et al.</em> 2006</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Stern <em>et al.</em> 2007</a>), the differential patterns in this plasticity between duplicates and singletons have received little attention. Findings from these studies led authors to conclude that the transcriptional plasticity of the genes in <em>S. cerevisiae</em> is the result of a general response to a wide range of stresses, sparking the possibility that this plasticity is an emerging property resulting from a universal feature of the underlying regulatory network. In this study we show that: (i) ancient duplicates of <em>S. cerevisiae</em> exhibit a large transcriptional plasticity when subjected to stress; and (ii) the transcriptional plasticity of duplicated genes differs from that of singletons, is more complex than thought before, and is likely the result of selection for an adaptive response to specific environmental challenges.</p><p id="p-50">The transcriptional changes affecting one or both of the gene copies resulting from gene duplication may be selectively advantageous in unicellular organisms because the absence of tissue-specific transcriptional subfunctionalization precludes a relief of the genetic redundancy of duplicated genes. Therefore, in unicellular eukaryotes, such as <em>S. cerevisiae</em>, the efficiency of purifying selection or positive selection must be a strong force driving the fate of duplicated genes. In agreement with this prediction, the genetic redundancy generated in <em>S. cerevisiae</em> after the WGD event that took place roughly 100 MYA was erased by purifying selection, as 92% of duplicated genes returned to single copy genes (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Wolfe and Shields 1997</a>). Despite this, the number of duplicates in <em>S. cerevisiae</em> (roughly 30% of all the genes) is higher than predicted by theory, raising the possibility that most retained genes have become functionally specialized, hence less redundant, shortly after duplication (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Force <em>et al.</em> 1999</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Lynch and Katju 2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barkman and Zhang 2009</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Des Marais and Rausher 2008</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">He and Zhang 2005</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Conant and Wolfe 2006</a>), transcriptionally divergent (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Blanc and Wolfe 2004</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Ha <em>et al.</em> 2007</a>, <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">2009</a>; <a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Francino 2005</a>), preserved due to their higher mutational robustness (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fares 2015</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Fares <em>et al.</em> 2013</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Keane <em>et al.</em> 2014</a>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Wagner 2000b</a>, <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">2005</a>), maintained owing to a selective advantage for higher gene dosage (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Conant and Wolfe 2008</a>), or kept to preserve stoichiometric balances in duplicates encoding protein complexes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gibson and Spring 1998</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Veitia 2003a</a>,<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">b</a>).</p><p id="p-51">In this study, the transcriptional plasticity identified in <em>S. cerevisiae</em> is likely the result of population polymorphism at the regulatory regions of duplicates (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>), which were selectively relaxed after gene duplication. This polymorphism has likely given rise to preadaptations to environments never before faced by the yeast and became fixed in the populations after facing such environmental perturbations. It is therefore likely that duplicates that show transcriptional plasticity, in particular those that become up-regulated under stress, are usually performing important functions in the cell and are hence maintained by purifying selection. However, under stress, such genes may encode new functions that provide the yeast with the ability to survive stress, a property encapsulated within the term exaptation (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gould and Vrba 1982</a>). The question that remains is: how important an adaptive force is the transcriptional divergence against the functional divergence of duplicates in <em>S. cerevisiae</em>?</p><p id="p-52">Functional divergence after gene and genome duplication has been the subject of intense scrutiny and a number of examples unequivocally correlate the origin of important gene families and functional specialization with the divergence between gene paralogs. Indeed, key globin proteins that specialized in different aspects of oxygen metabolism have originated through WGD events (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hoffmann <em>et al.</em> 2011</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2012a</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">b</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Storz <em>et al.</em> 2011</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">2013</a>). Functional divergence has also been observed in a number of studies and has been correlated with an asymmetric increase in the rates of sequence evolution in the duplicate gene copies (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Blanc and Wolfe 2004</a>), in good agreement with the fundamental tenet of the molecular evolution theory (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Gu <em>et al.</em> 2002</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dermitzakis and Clark 2001</a>). Expression divergence, but not transcriptional plasticity, between the copies of duplicated genes has also been demonstrated in a number of organisms.</p><p id="p-53">We propose the hypothesis supporting a link between expression and functional divergence – that is, one level of divergence necessarily drives the other level. Indeed, gene expression levels largely determine the rates of evolution of the proteins they encode (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Drummond <em>et al.</em> 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Drummond and Wilke 2008</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Pal <em>et al.</em> 2001</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rocha and Danchin 2004</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wilke and Drummond 2006</a>). The theoretical justification for this link between gene expression and its rate of evolution can be found in the misfolding–mistranslation hypothesis, according to which highly expressed genes evolve slower constrained by the need to maintain low levels of misfolded or mistranslated proteins bearing destabilizing mutations (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Drummond <em>et al.</em> 2005</a>). On the other hand, functional divergence, or acquisition of novel functions, may involve a fine-tuning of the expression of the encoding gene to perform the required function at the right rate. Whether expression divergence came before functional divergence or vice versa remains to be investigated but our data suggest a link between these two levels of divergence because different transcriptional categories exhibit different patterns of sequence evolution and divergence between gene copies (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>). We hypothesize that genetic redundancy has allowed transcriptional divergence between gene copies due to relaxed selective constraints. This has allowed divergence at the coding level driven by changes in gene expression, as gene expression is a strong determinant of sequence evolution (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Drummond <em>et al.</em> 2005</a>). Such functional divergence may have led to the acquisition of functions that enabled the adaptation to stress conditions (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>).</p><p id="p-54">In this study, we determine the plasticity that each of the gene copies has at the regulatory level, the link of this plasticity with the functional dependencies among gene copies, and the role of such a link in the response to stress. Our study reveals different modes of evolution for the different transcriptional categories. Most responsive duplicates to stress present only one copy altered, following the classic view of evolution by gene duplication. The genetic dependencies and low sequence divergence between gene copies for these duplicates also reveal the mode of evolution and innovation: these duplicates exhibit the highest proportion of cases with synergistic epistasis between gene copies, which summed to the low sequence divergence between the gene copies indicates higher genetic redundancy (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">VanderSluis <em>et al.</em> 2010</a>). This nontrivial pattern of evolution of novel functions is in agreement with previous predictions, according to which higher genetic redundancy allows the functional compensation between gene copies, the neutral exploration of genotypic space, and eventual finding of additional novel functions (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Fares 2015</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Fares <em>et al.</em> 2013</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Keane <em>et al.</em> 2014</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Wagner 2005</a>). The category of up-regulated duplicates exhibits evidence of neo-functionalization based on the rapid evolution of the gene copies when compared to their ancestor and the low functional dependency of each copy on its sister copy, suggesting the acquisition of novel functions. Duplicates with both copies being down-regulated under stress present low divergence between the gene copies and significant functional dependencies among the gene copies, suggesting the subfunctionalization of the gene copies through the partition of ancestral functions. The partition of ancestral functions in these duplicates is not complete, as the gene copies share more functions than expected. This greater sharing may reflect a selective advantage for gene dosage, particularly in the ancestral state immediately postdating genome duplication (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ihmels <em>et al.</em> 2007</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">VanderSluis <em>et al.</em> 2010</a>). Finally, the category in which gene copies exhibit discordant transcriptional alterations under stress (“Discordant”) presents a low number of genetic interactions between gene copies and low sequence divergence of the gene copies when compared to a preduplication ancestral gene, suggesting that both of the gene copies may be performing very similar functions under different conditions (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Force <em>et al.</em> 1999</a>). The category of discordant duplicates may include cases in which copies have diverged in their regulation such that one copy is active under stress and its sister copy is active under normal conditions, thereby avoiding the costly evolutionary optimization of the encoded function under two different conditions (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Conant and Wolfe 2006</a>). Remarkably, duplicates with no evidence for expression alteration under stress conditions exhibit a greater number of interactions between gene copies than expected. Because these duplicates are not affected by environmental perturbations, their enrichment for genetic interactions supports genetic buffering between the gene copies that is independent of the environment, a phenomenon important for genetic robustness (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dean <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">DeLuna <em>et al.</em> 2008</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Fares 2015</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Keane <em>et al.</em> 2014</a>; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">VanderSluis <em>et al.</em> 2010</a>).</p><p id="p-55">Our results point to the expression divergence between the gene copies of duplicated genes as a strategy to yield adaptive responses to stress conditions. This divergence is the result of differential accumulation of mutations (polymorphism) in the promoters of the duplicate gene copies, such that one copy exhibits high expression polymorphism under normal environmental conditions (<a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Keane <em>et al.</em> 2014</a>). Since yeast generally undergo stress because of the changes in the osmotic characteristics of the medium resulting from the metabolic byproducts generated, such stress conditions provide selective advantage to those strains with polymorphic expression divergence of duplicated genes that allow response to stress. A number of findings disregard expression noise as an alternative hypothesis to the adaptive value of the observed transcriptional alterations in duplicates. First, the profiles indicating adaptations to stress (<em>i.e.</em>, the transcriptional categories including duplicates with at least one gene up-regulated) include duplicated genes that under normal conditions show as much contribution to fitness as those with no response to stress (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1, C–F</a>). Second, under stress conditions, the contribution of up-regulated duplicates to fitness is significantly higher than their contribution to fitness under normal conditions. Finally, duplicates with altered expression profiles are mostly stress-specific and affect functions that are strictly related to the interaction with the environment and signal transduction.</p><p id="p-56">In conclusion, we reveal the underlying transcriptional plasticity of duplicates in <em>S. cerevisiae</em> and the potential of this plasticity to give origin to adaptations (by means of specialization of duplicates) to environmental perturbations. This study sets a new research endeavor mainly aiming at finding the as yet unexplored metabolic capabilities resulting from the evolution of duplicated genes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank members of the Fares laboratory for careful reading and discussion of the results in the manuscript. We are also grateful to colleagues at Trinity College for helpful discussions. This work was supported by a grant from the Spanish Ministerio de Economía y Competitividad (reference: BFU2015-66073-P) and a grant (reference: ACOMP/2015/026) from the local government Consellería de Educación Investigación, Cultura y Deporte, Generalitat Valenciana to M.A.F. C.T. was supported by a grant Juan de la Cierva from the Spanish Ministerio de Economía y Competitividad (reference: JCA-2012-14056).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-58">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035329/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-59"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-07"><span class="received-label">Received </span>September 7, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-10-23"><span class="accepted-label">Accepted </span>October 23, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Mattenberger <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.2" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.3" data-doi="10.1038/nature08663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barkman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evidence for escape from adaptive conflict?</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">462</span> (<span class="cit-issue">7274</span>): <span class="cit-fpage">E1</span>; discussion E2–3.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarkman%26rft.auinit1%253DT.%26rft.volume%253D462%26rft.issue%253D7274%26rft.spage%253DE1%26rft.epage%253DE1%26rft.atitle%253DEvidence%2Bfor%2Bescape%2Bfrom%2Badaptive%2Bconflict%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08663%26rft_id%253Dinfo%253Apmid%252F20010636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20010636&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272559900045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.4" data-doi="10.1074/jbc.M311879200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molina</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Ortin</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrero</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> glutaredoxin 5-deficient cells subjected to continuous oxidizing conditions are affected in the expression of specific sets of genes.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">279</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">12386</span>–<span class="cit-lpage">12395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M311879200%26rft_id%253Dinfo%253Apmid%252F14722110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzkvMTMvMTIzODYiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.5" data-doi="10.1105/tpc.021410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Functional divergence of duplicated genes formed by polyploidy during <em>Arabidopsis</em> evolution.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1679</span>–<span class="cit-lpage">1691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.021410%26rft_id%253Dinfo%253Apmid%252F15208398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzcvMTY3OSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.6" data-doi="10.1101/gr.3672305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The yeast gene order browser: combining curated homology and syntenic context reveals gene fate in polyploid species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1456</span>–<span class="cit-lpage">1461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3672305%26rft_id%253Dinfo%253Apmid%252F16169922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDU2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.7" data-doi="10.1093/molbev/mss162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carretero-Paulet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fares</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary dynamics and functional specialization of plant paralogs formed by whole and small-scale genome duplications.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3541</span>–<span class="cit-lpage">3551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss162%26rft_id%253Dinfo%253Apmid%252F22734049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMS8zNTQxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.8" data-doi="10.1091/mbc.12.2.323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Causton</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harbison</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Remodeling of yeast genome expression in response to environmental changes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">323</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DCauston%26rft.auinit1%253DH.%2BC.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D323%26rft.epage%253D337%26rft.atitle%253DRemodeling%2Bof%2BYeast%2BGenome%2BExpression%2Bin%2BResponse%2Bto%2BEnvironmental%2BChanges%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.12.2.323%26rft_id%253Dinfo%253Apmid%252F11179418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjEyLzIvMzIzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.9" data-doi="10.1371/journal.pbio.0040109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Functional partitioning of yeast co-expression networks after genome duplication.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DConant%26rft.auinit1%253DG.%2BC.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De109%26rft.epage%253De109%26rft.atitle%253DFunctional%2Bpartitioning%2Bof%2Byeast%2Bco-expression%2Bnetworks%2Bafter%2Bgenome%2Bduplication.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040109%26rft_id%253Dinfo%253Apmid%252F16555924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16555924&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.10" data-doi="10.1038/nrg2482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Turning a hobby into a job: how duplicated genes find new functions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">938</span>–<span class="cit-lpage">950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DConant%26rft.auinit1%253DG.%2BC.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D938%26rft.epage%253D950%26rft.atitle%253DTurning%2Ba%2Bhobby%2Binto%2Ba%2Bjob%253A%2Bhow%2Bduplicated%2Bgenes%2Bfind%2Bnew%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2482%26rft_id%253Dinfo%253Apmid%252F19015656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015656&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261037200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.11" data-doi="10.1093/nar/gkq257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cormier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuras</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcriptional plasticity through differential assembly of a multiprotein activation complex.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">4998</span>–<span class="cit-lpage">5014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq257%26rft_id%253Dinfo%253Apmid%252F20392822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNS80OTk4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.12" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>(<span class="cit-issue">5964</span>): <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.63.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cover</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source"><em>Elements of Information Theory</em>.</span> <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Wiley-Interscience</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.14" data-doi="10.1101/gr.4825606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leebens-Mack</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Widespread genome duplications throughout the history of flowering plants.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">738</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4825606%26rft_id%253Dinfo%253Apmid%252F16702410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzM4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.15" data-doi="10.1371/journal.pgen.1000113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pervasive and persistent redundancy among duplicated genes in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e1000113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDean%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De1000113%26rft.epage%253De1000113%26rft.atitle%253DPervasive%2Band%2Bpersistent%2Bredundancy%2Bamong%2Bduplicated%2Bgenes%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000113%26rft_id%253Dinfo%253Apmid%252F18604285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18604285&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.16" data-doi="10.1038/ng.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeLuna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vetsigian</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoresh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegreness</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colon-Gonzalez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Exposing the fitness contribution of duplicated genes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">676</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDeluna%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D676%26rft.epage%253D681%26rft.atitle%253DExposing%2Bthe%2Bfitness%2Bcontribution%2Bof%2Bduplicated%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.123%26rft_id%253Dinfo%253Apmid%252F18408719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18408719&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.17" data-doi="10.1093/oxfordjournals.molbev.a003835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dermitzakis</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Differential selection after duplication in mammalian developmental genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">557</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DDermitzakis%26rft.auinit1%253DE.%2BT.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D557%26rft.epage%253D562%26rft.atitle%253DDifferential%2BSelection%2BAfter%2BDuplication%2Bin%2BMammalian%2BDevelopmental%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003835%26rft_id%253Dinfo%253Apmid%252F11264407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE4LzQvNTU3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.18" data-doi="10.1038/nature07092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Des Marais</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rausher</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Escape from adaptive conflict after duplication in an anthocyanin pathway gene.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>(<span class="cit-issue">7205</span>): <span class="cit-fpage">762</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDes%2BMarais%26rft.auinit1%253DD.%2BL.%26rft.volume%253D454%26rft.issue%253D7205%26rft.spage%253D762%26rft.epage%253D765%26rft.atitle%253DEscape%2Bfrom%2Badaptive%2Bconflict%2Bafter%2Bduplication%2Bin%2Ban%2Banthocyanin%2Bpathway%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07092%26rft_id%253Dinfo%253Apmid%252F18594508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18594508&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258228000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.19" data-doi="10.1038/nature08694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Draghi</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutational robustness can facilitate adaptation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>(<span class="cit-issue">7279</span>): <span class="cit-fpage">353</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDraghi%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D463%26rft.issue%253D7279%26rft.spage%253D353%26rft.epage%253D355%26rft.atitle%253DMutational%2Brobustness%2Bcan%2Bfacilitate%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08694%26rft_id%253Dinfo%253Apmid%252F20090752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20090752&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273748100043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.20" data-doi="10.1016/j.cell.2008.05.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">341</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDrummond%26rft.auinit1%253DD.%2BA.%26rft.volume%253D134%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D352%26rft.atitle%253DMistranslation-induced%2Bprotein%2Bmisfolding%2Bas%2Ba%2Bdominant%2Bconstraint%2Bon%2Bcoding-sequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.05.042%26rft_id%253Dinfo%253Apmid%252F18662548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.05.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662548&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257891700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.21" data-doi="10.1073/pnas.0504070102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adami</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Why highly expressed proteins evolve slowly.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>(<span class="cit-issue">40</span>): <span class="cit-fpage">14338</span>–<span class="cit-lpage">14343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0504070102%26rft_id%253Dinfo%253Apmid%252F16176987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQwLzE0MzM4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.22" data-doi="10.1093/molbev/msj038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raval</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A single determinant dominates the rate of yeast protein evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">327</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj038%26rft_id%253Dinfo%253Apmid%252F16237209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.23" data-doi="10.1016/j.tig.2015.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fares</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The origins of mutational robustness.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">373</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D31%26rft.spage%253D373%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2015.04.008%26rft_id%253Dinfo%253Apmid%252F26013677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2015.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26013677&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.24" data-doi="10.1371/journal.pgen.1003176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fares</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toft</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carretero-Paulet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The roles of whole-genome and small-scale duplications in the functional specialization of <em>Saccharomyces cerevisiae</em> genes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1003176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003176%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003176%26rft_id%253Dinfo%253Apmid%252F23300483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300483&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.25" data-doi="10.1073/pnas.96.17.9721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferea</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Systematic changes in gene expression patterns following adaptive evolution in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">9721</span>–<span class="cit-lpage">9726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFerea%26rft.auinit1%253DT.%2BL.%26rft.volume%253D96%26rft.issue%253D17%26rft.spage%253D9721%26rft.epage%253D9726%26rft.atitle%253DSystematic%2Bchanges%2Bin%2Bgene%2Bexpression%2Bpatterns%2Bfollowing%2Badaptive%2Bevolution%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.17.9721%26rft_id%253Dinfo%253Apmid%252F10449761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTYvMTcvOTcyMSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.26" data-doi="10.1007/BF01732026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitt</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Evolution of the differential regulation of duplicate genes after polyploidization.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">267</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFerris%26rft.auinit1%253DS.%2BD.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D267%26rft.epage%253D317%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bdifferential%2Bregulation%2Bof%2Bduplicate%2Bgenes%2Bafter%2Bpolyploidization.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01732026%26rft_id%253Dinfo%253Apmid%252F448746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01732026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=448746&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GT30300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Force</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickett</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Preservation of duplicate genes by complementary, degenerative mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1531</span>–<span class="cit-lpage">1545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DForce%26rft.auinit1%253DA.%26rft.volume%253D151%26rft.issue%253D4%26rft.spage%253D1531%26rft.epage%253D1545%26rft.atitle%253DPreservation%2Bof%2BDuplicate%2BGenes%2Bby%2BComplementary%252C%2BDegenerative%2BMutations%26rft_id%253Dinfo%253Apmid%252F10101175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1MS80LzE1MzEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.28" data-doi="10.1038/ng1579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francino</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An adaptive radiation model for the origin of new gene functions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">573</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFrancino%26rft.auinit1%253DM.%2BP.%26rft.volume%253D37%26rft.issue%253D6%26rft.spage%253D573%26rft.epage%253D577%26rft.atitle%253DAn%2Badaptive%2Bradiation%2Bmodel%2Bfor%2Bthe%2Borigin%2Bof%2Bnew%2Bgene%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1579%26rft_id%253Dinfo%253Apmid%252F15920518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15920518&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229495300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.29" data-doi="10.1016/S0168-9525(97)01367-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spring</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic redundancy in vertebrates: polyploidy and persistence of genes encoding multidomain proteins.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">46</span>–<span class="cit-lpage">49, discussion 49–50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DGibson%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D46%26rft.epage%253D49%26rft.atitle%253DGenetic%2Bredundancy%2Bin%2Bvertebrates%253A%2Bpolyploidy%2Band%2Bpersistence%2Bof%2Bgenes%2Bencoding%2Bmultidomain%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252897%252901367-X%26rft_id%253Dinfo%253Apmid%252F9520595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(97)01367-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9520595&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072252300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A codon-based model of nucleotide substitution for protein-coding DNA sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">725</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DGoldman%26rft.auinit1%253DN.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D725%26rft.epage%253D736%26rft.atitle%253DA%2Bcodon-based%2Bmodel%2Bof%2Bnucleotide%2Bsubstitution%2Bfor%2Bprotein-coding%2BDNA%2Bsequences%26rft_id%253Dinfo%253Apmid%252F7968486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjExLzUvNzI1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vrba</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Exaptation-A missing term in the science of form.</span> <abbr class="cit-jnl-abbrev">Paleobiology</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleobiology%26rft.volume%253D8%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.32" data-doi="10.1016/S0168-9525(02)02837-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolae</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Rapid divergence in expression between duplicate genes inferred from microarray data.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">609</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DGu%26rft.auinit1%253DZ.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D609%26rft.epage%253D613%26rft.atitle%253DRapid%2Bdivergence%2Bin%2Bexpression%2Bbetween%2Bduplicate%2Bgenes%2Binferred%2Bfrom%2Bmicroarray%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902837-8%26rft_id%253Dinfo%253Apmid%252F12446139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02837-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12446139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179433300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.33" data-doi="10.1016/j.tig.2007.02.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">External factors accelerate expression divergence between duplicate genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">162</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHa%26rft.auinit1%253DM.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D162%26rft.epage%253D166%26rft.atitle%253DExternal%2Bfactors%2Baccelerate%2Bexpression%2Bdivergence%2Bbetween%2Bduplicate%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.02.005%26rft_id%253Dinfo%253Apmid%252F17320239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17320239&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246002700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.34" data-doi="10.1073/pnas.0807350106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Duplicate genes increase expression diversity in closely related species and allopolyploids.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">2295</span>–<span class="cit-lpage">2300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0807350106%26rft_id%253Dinfo%253Apmid%252F19168631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzcvMjI5NSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.35" data-doi="10.1016/j.cell.2009.07.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halabi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivoire</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranganathan</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Protein sectors: evolutionary units of three-dimensional structure.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">138</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">774</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHalabi%26rft.auinit1%253DN.%26rft.volume%253D138%26rft.issue%253D4%26rft.spage%253D774%26rft.epage%253D786%26rft.atitle%253DProtein%2Bsectors%253A%2Bevolutionary%2Bunits%2Bof%2Bthree-dimensional%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.07.038%26rft_id%253Dinfo%253Apmid%252F19703402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.07.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19703402&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.36" data-doi="10.1534/genetics.104.037051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid subfunctionalization accompanied by prolonged and substantial neofunctionalization in duplicate gene evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">1157</span>–<span class="cit-lpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.037051%26rft_id%253Dinfo%253Apmid%252F15654095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8yLzExNTciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.37" data-doi="10.1007/s00239-004-2613-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoegg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">190</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHoegg%26rft.auinit1%253DS.%26rft.volume%253D59%26rft.issue%253D2%26rft.spage%253D190%26rft.epage%253D203%26rft.atitle%253DPhylogenetic%2Btiming%2Bof%2Bthe%2Bfish-specific%2Bgenome%2Bduplication%2Bcorrelates%2Bwith%2Bthe%2Bdiversification%2Bof%2Bteleost%2Bfish.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-004-2613-z%26rft_id%253Dinfo%253Apmid%252F15486693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-004-2613-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15486693&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223424800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.38" data-doi="10.1093/gbe/evr055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opazo</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Differential loss and retention of cytoglobin, myoglobin, and globin-E during the radiation of vertebrates.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr055%26rft_id%253Dinfo%253Apmid%252F21697098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC81ODgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.39" data-doi="10.1093/molbev/mss018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opazo</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoogewijs</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hankeln</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Evolution of the globin gene family in deuterostomes: lineage-specific patterns of diversification and attrition.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1735</span>–<span class="cit-lpage">1745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss018%26rft_id%253Dinfo%253Apmid%252F22319164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzcvMTczNSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.40" data-doi="10.1093/molbev/msr207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opazo</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Whole-genome duplications spurred the functional diversification of the globin gene superfamily in vertebrates.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">303</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr207%26rft_id%253Dinfo%253Apmid%252F21965344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI5LzEvMzAzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.41" data-doi="10.1038/35080508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holub</span>  <span class="cit-name-given-names">E. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The arms race is ancient history in <em>Arabidopsis</em>, the wildflower.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">516</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHolub%26rft.auinit1%253DE.%2BB.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D516%26rft.epage%253D527%26rft.atitle%253DThe%2Barms%2Brace%2Bis%2Bancient%2Bhistory%2Bin%2BArabidopsis%252C%2Bthe%2Bwildflower.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35080508%26rft_id%253Dinfo%253Apmid%252F11433358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35080508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11433358&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169681600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.42" data-doi="10.1101/gr.2705204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huminiecki</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Divergence of spatial gene expression profiles following species-specific gene duplications in human and mouse.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>(<span class="cit-supplement">10A</span>): <span class="cit-fpage">1870</span>–<span class="cit-lpage">1879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuminiecki%26rft.auinit1%253DL.%26rft.volume%253D14%26rft.issue%253D10a%26rft.spage%253D1870%26rft.epage%253D1879%26rft.atitle%253DDivergence%2Bof%2BSpatial%2BGene%2BExpression%2BProfiles%2BFollowing%2BSpecies-Specific%2BGene%2BDuplications%2Bin%2BHuman%2Band%2BMouse%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2705204%26rft_id%253Dinfo%253Apmid%252F15466287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjExOiIxNC8xMGEvMTg3MCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.43" data-doi="10.1126/science.292.5518.929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ideker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorsson</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranish</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christmas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buhler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Integrated genomic and proteomic analyses of a systematically perturbed metabolic network.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">292</span>(<span class="cit-issue">5518</span>): <span class="cit-fpage">929</span>–<span class="cit-lpage">934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DIdeker%26rft.auinit1%253DT.%26rft.volume%253D292%26rft.issue%253D5518%26rft.spage%253D929%26rft.epage%253D934%26rft.atitle%253DIntegrated%2BGenomic%2Band%2BProteomic%2BAnalyses%2Bof%2Ba%2BSystematically%2BPerturbed%2BMetabolic%2BNetwork%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.292.5518.929%26rft_id%253Dinfo%253Apmid%252F11340206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTIvNTUxOC85MjkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.44" data-doi="10.1038/msb4100127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ihmels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Backup without redundancy: genetic interactions reveal the cost of duplicate gene loss.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DIhmels%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D86%26rft.epage%253D86%26rft.atitle%253DBackup%2Bwithout%2Bredundancy%253A%2Bgenetic%2Binteractions%2Breveal%2Bthe%2Bcost%2Bof%2Bduplicate%2Bgene%2Bloss%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100127%26rft_id%253Dinfo%253Apmid%252F17389874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjY6IjMvMS84NiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.45" data-doi="10.1038/nrg2689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The evolution of gene duplications: classifying and distinguishing between models.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">97</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D108%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bduplications%253A%2Bclassifying%2Band%2Bdistinguishing%2Bbetween%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2689%26rft_id%253Dinfo%253Apmid%252F20051986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051986&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273688200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.46" data-doi="10.1101/gr.176792.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toft</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carretero-Paulet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fares</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Preservation of genetic and regulatory robustness in ancient gene duplicates of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1830</span>–<span class="cit-lpage">1841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.176792.114%26rft_id%253Dinfo%253Apmid%252F25149527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMS8xODMwIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.47" data-doi="10.3732/ajb.91.12.2102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farris</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phylogeny and diversification of B-function MADS-box genes in angiosperms: evolutionary and functional implications of a 260-million-year-old duplication.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">2102</span>–<span class="cit-lpage">2118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DKim%26rft.auinit1%253DS.%26rft.volume%253D91%26rft.issue%253D12%26rft.spage%253D2102%26rft.epage%253D2118%26rft.atitle%253DPhylogeny%2Band%2Bdiversification%2Bof%2BB-function%2BMADS-box%2Bgenes%2Bin%2Bangiosperms%253A%2Bevolutionary%2Band%2Bfunctional%2Bimplications%2Bof%2Ba%2B260-million-year-old%2Bduplication%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.91.12.2102%26rft_id%253Dinfo%253Apmid%252F21652358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiI5MS8xMi8yMTAyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.48" data-doi="10.1016/j.gene.2005.10.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalieri</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide scan reveals that genetic variation for transcriptional plasticity in yeast is biased towards multi-copy and dispensable genes.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">366</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">343</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DLandry%26rft.auinit1%253DC.%2BR.%26rft.volume%253D366%26rft.issue%253D2%26rft.spage%253D343%26rft.epage%253D351%26rft.atitle%253DGenome-wide%2Bscan%2Breveals%2Bthat%2Bgenetic%2Bvariation%2Bfor%2Btranscriptional%2Bplasticity%2Bin%2Byeast%2Bis%2Bbiased%2Btowards%2Bmulti-copy%2Band%2Bdispensable%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2005.10.042%26rft_id%253Dinfo%253Apmid%252F16427747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2005.10.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16427747&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236312600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.49" data-doi="10.1101/gr.174302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lespinet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravind</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The role of lineage-specific gene family expansion in the evolution of eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1048</span>–<span class="cit-lpage">1059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLespinet%26rft.auinit1%253DO.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D1048%26rft.epage%253D1059%26rft.atitle%253DThe%2BRole%2Bof%2BLineage-Specific%2BGene%2BFamily%2BExpansion%2Bin%2Bthe%2BEvolution%2Bof%2BEukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.174302%26rft_id%253Dinfo%253Apmid%252F12097341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEyLzcvMTA0OCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.50" data-doi="10.1016/j.tig.2005.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Expression divergence between duplicate genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">602</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DW.%2BH.%26rft.volume%253D21%26rft.issue%253D11%26rft.spage%253D602%26rft.epage%253D607%26rft.atitle%253DExpression%2Bdivergence%2Bbetween%2Bduplicate%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2005.08.006%26rft_id%253Dinfo%253Apmid%252F16140417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2005.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16140417&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233021600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.51" data-doi="10.1093/nar/gku1357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duggan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sieber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Defining the transcriptomic landscape of <em>Candida glabrata</em> by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1392</span>–<span class="cit-lpage">1406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1357%26rft_id%253Dinfo%253Apmid%252F25586221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQzLzMvMTM5MiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.52" data-doi="10.1093/nar/gks540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernie</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunn</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">Web Server issue</span>): <span class="cit-fpage">W622</span>–<span class="cit-lpage">W627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks540%26rft_id%253Dinfo%253Apmid%252F22684630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9XMS9XNjIyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.53" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>(<span class="cit-issue">5494</span>): <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.54" data-doi="10.1023/A:1022696612931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The evolutionary demography of duplicate genes.</span> <abbr class="cit-jnl-abbrev">J. Struct. Funct. Genomics</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1–4</span>): <span class="cit-fpage">35</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Band%2Bfunctional%2Bgenomics%26rft.stitle%253DJ%2BStruct%2BFunct%2BGenomics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D1-4%26rft.spage%253D35%26rft.epage%253D44%26rft.atitle%253DThe%2Bevolutionary%2Bdemography%2Bof%2Bduplicate%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1022696612931%26rft_id%253Dinfo%253Apmid%252F12836683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1022696612931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12836683&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.55" data-doi="10.1016/j.tig.2004.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katju</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The altered evolutionary trajectories of gene duplicates.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">544</span>–<span class="cit-lpage">549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D20%26rft.issue%253D11%26rft.spage%253D544%26rft.epage%253D549%26rft.atitle%253DThe%2Baltered%2Bevolutionary%2Btrajectories%2Bof%2Bgene%2Bduplicates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.09.001%26rft_id%253Dinfo%253Apmid%252F15475113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15475113&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224969800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.56" data-doi="10.1371/journal.pbio.1002220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Beyond the whole-genome duplication: phylogenetic evidence for an ancient interspecies hybridization in the Baker’s yeast lineage.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e1002220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002220%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002220%26rft_id%253Dinfo%253Apmid%252F26252497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26252497&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.57" data-doi="10.1101/gr.076174.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musso</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huangfu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The extensive and condition-dependent nature of epistasis among whole-genome duplicates in yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1092</span>–<span class="cit-lpage">1099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.076174.108%26rft_id%253Dinfo%253Apmid%252F18463300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzcvMTA5MiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.58" data-doi="10.1016/S0168-9525(00)02174-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohler</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niemann</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification and analysis of eukaryotic promoters: recent computational approaches.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">56</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DOhler%26rft.auinit1%253DU.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D56%26rft.epage%253D60%26rft.atitle%253DIdentification%2Band%2Banalysis%2Bof%2Beukaryotic%2Bpromoters%253A%2Brecent%2Bcomputational%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902174-0%26rft_id%253Dinfo%253Apmid%252F11173099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02174-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11173099&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168718000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.63.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source"><em>Evolution by Gene Duplication</em>.</span> <span class="cit-publ-name">Springer Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.60" data-doi="10.1006/scdb.1999.0332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Gene duplication and the uniqueness of vertebrate genomes circa 1970–1999.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">517</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DOhno%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D517%26rft.epage%253D522%26rft.atitle%253DGene%2Bduplication%2Band%2Bthe%2Buniqueness%2Bof%2Bvertebrate%2Bgenomes%2Bcirca%2B1970-1999.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fscdb.1999.0332%26rft_id%253Dinfo%253Apmid%252F10597635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/scdb.1999.0332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10597635&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083733400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.61" data-doi="10.1146/annurev.genet.34.1.401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitton</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Polyploid incidence and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D401%26rft.epage%253D437%26rft.atitle%253DPOLYPLOID%2BINCIDENCE%2BAND%2BEVOLUTION%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.34.1.401%26rft_id%253Dinfo%253Apmid%252F11092833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.34.1.401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11092833&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166299400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Highly expressed genes in yeast evolve slowly.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">927</span>–<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPal%26rft.auinit1%253DC.%26rft.volume%253D158%26rft.issue%253D2%26rft.spage%253D927%26rft.epage%253D931%26rft.atitle%253DHighly%2BExpressed%2BGenes%2Bin%2BYeast%2BEvolve%2BSlowly%26rft_id%253Dinfo%253Apmid%252F11430355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzIvOTI3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.63" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi8xLzEzOSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An analysis of determinants of amino acids substitution rates in bacterial proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">108</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Apmid%252F14595100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIxLzEvMTA4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.65" data-doi="10.1186/1471-2148-13-156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Gonzalez</span>  <span class="cit-name-given-names">M. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fares</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Coevolution analyses illuminate the dependencies between amino acid sites in the chaperonin system GroES-L.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D156%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-156%26rft_id%253Dinfo%253Apmid%252F23875653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23875653&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.66" data-doi="10.1038/ng929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scharfe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mokranjac</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Systematic screen for human disease genes in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">400</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSteinmetz%26rft.auinit1%253DL.%2BM.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D400%26rft.epage%253D404%26rft.atitle%253DSystematic%2Bscreen%2Bfor%2Bhuman%2Bdisease%2Bgenes%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng929%26rft_id%253Dinfo%253Apmid%252F12134146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12134146&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177147100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.67" data-doi="10.1038/msb4100147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dror</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stolovicki</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide transcriptional plasticity underlies cellular adaptation to novel challenge.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DStern%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D106%26rft.atitle%253DGenome-wide%2Btranscriptional%2Bplasticity%2Bunderlies%2Bcellular%2Badaptation%2Bto%2Bnovel%2Bchallenge%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100147%26rft_id%253Dinfo%253Apmid%252F17453047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjMvMS8xMDYiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.68" data-doi="10.1002/iub.482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opazo</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">F. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phylogenetic diversification of the globin gene superfamily in chordates.</span> <abbr class="cit-jnl-abbrev">IUBMB Life</abbr> <span class="cit-vol">63</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">313</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIUBMB%2Blife%26rft.stitle%253DIUBMB%2BLife%26rft.aulast%253DStorz%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D63%26rft.issue%253D5%26rft.spage%253D313%26rft.epage%253D322%26rft.atitle%253DPhylogenetic%2Bdiversification%2Bof%2Bthe%2Bglobin%2Bgene%2Bsuperfamily%2Bin%2Bchordates.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fiub.482%26rft_id%253Dinfo%253Apmid%252F21557448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/iub.482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21557448&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291433900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.69" data-doi="10.1016/j.ympev.2012.07.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opazo</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">F. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene duplication, genome duplication, and the functional diversification of vertebrate globins.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">66</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">469</span>–<span class="cit-lpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D66%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.07.013%26rft_id%253Dinfo%253Apmid%252F22846683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22846683&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.70" data-doi="10.1146/annurev.genet.38.072902.092831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raes</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Duplication and divergence: the evolution of new genes and old ideas.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DTaylor%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D38%26rft.spage%253D615%26rft.epage%253D643%26rft.atitle%253DDuplication%2Band%2Bdivergence%253A%2Bthe%2Bevolution%2Bof%2Bnew%2Bgenes%2Band%2Bold%2Bideas.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.38.072902.092831%26rft_id%253Dinfo%253Apmid%252F15568988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.38.072902.092831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15568988&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226244600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.71" data-doi="10.7554/eLife.00603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styczynsky</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfiffner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary principles of modular gene regulation in yeasts.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D2%26rft.issue%253D0%26rft.spage%253De00603%26rft.epage%253De00603%26rft.atitle%253DEvolutionary%2Bprinciples%2Bof%2Bmodular%2Bgene%2Bregulation%2Bin%2Byeasts%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00603%26rft_id%253Dinfo%253Apmid%252F23795289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjIvMC9lMDA2MDMiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.72" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>(<span class="cit-issue">5550</span>): <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.73" data-doi="10.1038/msb.2010.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanderSluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musso</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic interactions reveal the evolutionary trajectories of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DVanderSluis%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D429%26rft.epage%253D429%26rft.atitle%253DGenetic%2Binteractions%2Breveal%2Bthe%2Bevolutionary%2Btrajectories%2Bof%2Bduplicate%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2010.82%26rft_id%253Dinfo%253Apmid%252F21081923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjYvMS80MjkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.74" data-doi="10.1006/jtbi.2003.3105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">a</span> <span class="cit-article-title">Nonlinear effects in macromolecular assembly and dosage sensitivity.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">220</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">19</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DVeitia%26rft.auinit1%253DR.%2BA.%26rft.volume%253D220%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D25%26rft.atitle%253DNonlinear%2Beffects%2Bin%2Bmacromolecular%2Bassembly%2Band%2Bdosage%2Bsensitivity.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.2003.3105%26rft_id%253Dinfo%253Apmid%252F12453447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.2003.3105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12453447&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180313000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.75" data-doi="10.1017/S1464793102006036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">b</span> <span class="cit-article-title">A sigmoidal transcriptional response: cooperativity, synergy and dosage effects.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">149</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DVeitia%26rft.auinit1%253DR.%2BA.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D170%26rft.atitle%253DA%2Bsigmoidal%2Btranscriptional%2Bresponse%253A%2Bcooperativity%252C%2Bsynergy%2Band%2Bdosage%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1464793102006036%26rft_id%253Dinfo%253Apmid%252F12620064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1464793102006036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12620064&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.76" data-doi="10.1073/pnas.110147097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">a</span> <span class="cit-article-title">Decoupled evolution of coding region and mRNA expression patterns after gene duplication: implications for the neutralist-selectionist debate.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">6579</span>–<span class="cit-lpage">6584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.110147097%26rft_id%253Dinfo%253Apmid%252F10823904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTIvNjU3OSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.77" data-doi="10.1038/74174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">b</span> <span class="cit-article-title">Robustness against mutations in genetic networks of yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">355</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWagner%26rft.auinit1%253DA.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D361%26rft.atitle%253DRobustness%2Bagainst%2Bmutations%2Bin%2Bgenetic%2Bnetworks%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F74174%26rft_id%253Dinfo%253Apmid%252F10742097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/74174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10742097&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086192800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.78" data-doi="10.1016/j.febslet.2005.01.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Robustness, evolvability, and neutrality.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">579</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1772</span>–<span class="cit-lpage">1778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DWagner%26rft.auinit1%253DA.%26rft.volume%253D579%26rft.issue%253D8%26rft.spage%253D1772%26rft.epage%253D1778%26rft.atitle%253DRobustness%252C%2Bevolvability%252C%2Band%2Bneutrality.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2005.01.063%26rft_id%253Dinfo%253Apmid%252F15763550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2005.01.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15763550&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227657800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.79" data-doi="10.1111/j.1749-6632.2011.06384.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome and gene duplications and gene expression divergence: a view from plants.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1256</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DWang%26rft.auinit1%253DY.%26rft.volume%253D1256%26rft.spage%253D1%26rft.epage%253D14%26rft.atitle%253DGenome%2Band%2Bgene%2Bduplications%2Band%2Bgene%2Bexpression%2Bdivergence%253A%2Ba%2Bview%2Bfrom%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2011.06384.x%26rft_id%253Dinfo%253Apmid%252F22257007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2011.06384.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22257007&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305688600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.80" data-doi="10.1023/A:1006392424384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome evolution in polyploids.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">225</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DWendel%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D225%26rft.epage%253D249%26rft.atitle%253DGenome%2Bevolution%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1006392424384%26rft_id%253Dinfo%253Apmid%252F10688139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1006392424384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10688139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084535800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.81" data-doi="10.1534/genetics.105.051300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population genetics of translational robustness.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">473</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.051300%26rft_id%253Dinfo%253Apmid%252F16489231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDczIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.82" data-doi="10.1371/journal.pbio.1002221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Origin of the yeast whole-genome duplication.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e1002221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002221%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002221%26rft_id%253Dinfo%253Apmid%252F26252643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26252643&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.83" data-doi="10.1038/42711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular evidence for an ancient duplication of the entire yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>(<span class="cit-issue">6634</span>): <span class="cit-fpage">708</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D387%26rft.issue%253D6634%26rft.spage%253D708%26rft.epage%253D713%26rft.atitle%253DMolecular%2Bevidence%2Bfor%2Ban%2Bancient%2Bduplication%2Bof%2Bthe%2Bentire%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42711%26rft_id%253Dinfo%253Apmid%252F9192896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9192896&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XD86900055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.84" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU4NiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.63.85" data-doi="10.1371/journal.pone.0103207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jhaveri</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A comparative study of techniques for differential expression analysis on RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e103207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De103207%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103207%26rft_id%253Dinfo%253Apmid%252F25119138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25119138&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/63.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/55" title="Assortative Mating and Linkage Disequilibrium" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/77" title="Cytogenetic and Molecular Characterization of B-Genome Introgression Lines of Brassica napus L." rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/63.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HP3AtZK39Gja6zyhJESK22s56uQAplShC-ithjlLazA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/63" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/63" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/63" class="active" data-icon-position="" data-hide-link-title="0">The Phenotypic Plasticity of Duplicated Genes in Saccharomyces cerevisiae and the Origin of Adaptations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460525" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WEmiI_5wl-aDB_tOSHOsVDvB9-lS-6oBush6bpjefHY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/63.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/63" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kAXkRHpUJxP7YGjVYI6iyRTStKUldAoeJQe8bDTwVKA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/63" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/63.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460525" id="node460525--2" data-pisa="ggg;7/1/63" data-pisa-master="ggg;g3.116.035329" data-apath="/ggg/7/1/63.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Phenotypic Plasticity of Duplicated Genes in <em>Saccharomyces cerevisiae</em> and the Origin of Adaptations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2727-0284" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Florian Mattenberger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0301-215X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Beatriz Sabater-Muñoz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1714-6703" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina Toft</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4345-3013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario A. Fares</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">63-75;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035329 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460525--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Florian Mattenberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florian%2BMattenberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mattenberger%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFlorian%2BMattenberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2727-0284" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Florian Mattenberger</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Beatriz Sabater-Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beatriz%2BSabater-Mu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sabater-Mu%C3%B1oz%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeatriz%2BSabater-Mu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0301-215X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Beatriz Sabater-Muñoz</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christina Toft </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, University of Valencia, 46100 Burjassot, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Biotecnología, Instituto de Agroquímica y Tecnología de los Alimentos, Consejo Superior de Investigaciones Científicas, Valencia, 46980 Paterna, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BToft%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toft%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BToft%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1714-6703" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina Toft</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mario A. Fares </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BA.%2BFares%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fares%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BA.%2BFares%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4345-3013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario A. Fares</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mfares@ibmcp.upv.es" class="" data-icon-position="" data-hide-link-title="0">mfares@ibmcp.upv.es</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/63" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/63&amp;title=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/63&amp;title=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/63&amp;title=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/63&amp;text=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/63&amp;title=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/63&amp;t=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/63&amp;title=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/63&amp;title=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460525/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460525" id="node460525--3" data-pisa="ggg;7/1/63" data-pisa-master="ggg;g3.116.035329" data-apath="/ggg/7/1/63.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Phenotypic Plasticity of Duplicated Genes in <em>Saccharomyces cerevisiae</em> and the Origin of Adaptations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2727-0284" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Florian Mattenberger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0301-215X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Beatriz Sabater-Muñoz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1714-6703" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina Toft</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4345-3013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario A. Fares</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">63-75;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035329 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460525--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Florian Mattenberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Florian%2BMattenberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mattenberger%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFlorian%2BMattenberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2727-0284" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Florian Mattenberger</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Beatriz Sabater-Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beatriz%2BSabater-Mu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sabater-Mu%C3%B1oz%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeatriz%2BSabater-Mu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0301-215X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Beatriz Sabater-Muñoz</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christina Toft </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, University of Valencia, 46100 Burjassot, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Biotecnología, Instituto de Agroquímica y Tecnología de los Alimentos, Consejo Superior de Investigaciones Científicas, Valencia, 46980 Paterna, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BToft%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toft%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BToft%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1714-6703" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina Toft</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mario A. Fares </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas, Universidad Politécnica de Valencia, 46022 Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BA.%2BFares%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fares%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BA.%2BFares%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4345-3013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario A. Fares</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mfares@ibmcp.upv.es" class="" data-icon-position="" data-hide-link-title="0">mfares@ibmcp.upv.es</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=63&amp;rft.epage=75&amp;rft.atitle=The%20Phenotypic%20Plasticity%20of%20Duplicated%20Genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20Origin%20of%20Adaptations&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mattenberger%2C+Florian&amp;rft.au=Sabater-Mu%C3%B1oz%2C+Beatriz&amp;rft.au=Toft%2C+Christina&amp;rft.au=Fares%2C+Mario+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460525/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460525/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460525/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460525/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460525/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460525/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460525/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460525/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460525/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460525/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460525/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460525/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/63" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/63.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/63.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/1/63.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/1/63.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
