<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/279" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/279.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/279.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/1/279.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/1/279/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/279.atom" />
<meta name="HW.pisa" content="ggg;7/1/279" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!" />
<meta name="DC.Identifier" content="10.1534/g3.116.037044" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Herricks et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cell growth is a complex phenotype widely used in systems biology to gauge the impact of genetic and environmental perturbations. Due to the magnitude of genome-wide studies, resolution is often sacrificed in favor of throughput, creating a demand for scalable, time-resolved, quantitative methods of growth assessment. We present ODELAY (One-cell Doubling Evaluation by Living Arrays of Yeast), an automated and scalable growth analysis platform. High measurement density and single-cell resolution provide a powerful tool for large-scale multiparameter growth analysis based on the modeling of microcolony expansion on solid media. Pioneered in yeast but applicable to other colony forming organisms, ODELAY extracts the three key growth parameters (lag time, doubling time, and carrying capacity) that define microcolony expansion from single cells, simultaneously permitting the assessment of population heterogeneity. The utility of ODELAY is illustrated using yeast mutants, revealing a spectrum of phenotypes arising from single and combinatorial growth parameter perturbations." />
<meta name="DC.Contributor" content="Thurston Herricks" />
<meta name="DC.Contributor" content="David J. Dilworth" />
<meta name="DC.Contributor" content="Fred D. Mast" />
<meta name="DC.Contributor" content="Song Li" />
<meta name="DC.Contributor" content="Jennifer J. Smith" />
<meta name="DC.Contributor" content="Alexander V. Ratushny" />
<meta name="DC.Contributor" content="John D. Aitchison" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cell growth is a complex phenotype widely used in systems biology to gauge the impact of genetic and environmental perturbations. Due to the magnitude of genome-wide studies, resolution is often sacrificed in favor of throughput, creating a demand for scalable, time-resolved, quantitative methods of growth assessment. We present ODELAY (One-cell Doubling Evaluation by Living Arrays of Yeast), an automated and scalable growth analysis platform. High measurement density and single-cell resolution provide a powerful tool for large-scale multiparameter growth analysis based on the modeling of microcolony expansion on solid media. Pioneered in yeast but applicable to other colony forming organisms, ODELAY extracts the three key growth parameters (lag time, doubling time, and carrying capacity) that define microcolony expansion from single cells, simultaneously permitting the assessment of population heterogeneity. The utility of ODELAY is illustrated using yeast mutants, revealing a spectrum of phenotypes arising from single and combinatorial growth parameter perturbations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/279" />
<meta name="citation_id" content="7/1/279" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/279" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/279.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/279.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/279.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.037044" />
<meta name="citation_pmid" content="27856698" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="279" />
<meta name="citation_lastpage" content="288" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Thurston Herricks" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author" content="David J. Dilworth" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author" content="Fred D. Mast" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author_institution" content="Center for Infectious Disease Research, Seattle, Washington 98109" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2177-6647" />
<meta name="citation_author" content="Song Li" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author" content="Jennifer J. Smith" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author" content="Alexander V. Ratushny" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author_institution" content="Center for Infectious Disease Research, Seattle, Washington 98109" />
<meta name="citation_author" content="John D. Aitchison" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author_institution" content="Center for Infectious Disease Research, Seattle, Washington 98109" />
<meta name="citation_author_email" content="john.aitchison@systemsbiology.org" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. D. Aitchison;citation_author=M. P. Rout;citation_title=The Yeast Nuclear Pore Complex and Transport Through It;citation_pages=855-883;citation_volume=190;citation_year=2012;citation_issue=3;citation_pmid=22419078;citation_doi=10.1534/genetics.111.127803" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=Y. Kim;citation_author=H. Ding;citation_author=J. Koh;citation_title=Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.;citation_pages=1017-1024;citation_volume=7;citation_year=2010;citation_issue=12;citation_pmid=21076421;citation_doi=10.1038/nmeth.1534" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. J. Bean;citation_author=P. A. Jaeger;citation_author=S. Bahr;citation_author=T. Ideker;citation_title=Development of ultra-high-density screening tools for microbial “omics.”;citation_pages=e85177;citation_volume=9;citation_year=2014;citation_pmid=24465499;citation_doi=10.1371/journal.pone.0085177" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. K. Breslow;citation_author=D. M. Cameron;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=J. Stewart-Ornstein;citation_title=A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.;citation_pages=711-718;citation_volume=5;citation_year=2008;citation_issue=8;citation_pmid=18622397;citation_doi=10.1038/nmeth.1234" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. K. Bryan;citation_author=A. Goranov;citation_author=A. Amon;citation_author=S. R. Manalis;citation_title=Measurement of mass, density, and volume during the cell cycle of yeast.;citation_volume=107;citation_year=2010;citation_pmid=20080562;citation_doi=10.1073/pnas.0901851107" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=N. J. Krogan;citation_author=J. S. Weissman;citation_title=A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.;citation_pages=R63-R63;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16859555;citation_doi=10.1186/gb-2006-7-7-r63" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_title=A global genetic interaction network maps a wiring diagram of cellular function.;citation_author=M. Costanzo;citation_author=B. VanderSluis;citation_author=E. N. Koch;citation_author=A. Baryshnikova;citation_author=C. Pons;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Deutschbauer;citation_author=D. F. Jaramillo;citation_author=M. Proctor;citation_author=J. Kumm;citation_author=M. E. Hillenmeyer;citation_title=Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.;citation_volume=169;citation_year=2005;citation_pmid=15716499;citation_doi=10.1534/genetics.104.036871" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Methods;citation_author=A. D. Edelstein;citation_author=M. A. Tsuchida;citation_author=N. Amodaj;citation_author=H. Pinkard;citation_author=R. D. Vale;citation_title=Advanced methods of microscope control using μManager software.;citation_pages=10;citation_volume=1;citation_year=2014;citation_pmid=25606571;citation_doi=10.14440/jbm.2014.36" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. Godin;citation_author=F. F. Delgado;citation_author=S. Son;citation_author=W. H. Grover;citation_author=A. K. Bryan;citation_title=Using buoyant mass to measure the growth of single cells.;citation_pages=387-390;citation_volume=7;citation_year=2010;citation_issue=5;citation_pmid=20383132;citation_doi=10.1038/nmeth.1452" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond.;citation_author=B. Gompertz;citation_title=On the nature of the function expressive of the law of human mortality, and on a new mode of determining the value of life contingencies.;citation_pages=513-583;citation_volume=115;citation_year=1825;citation_doi=10.1098/rstl.1825.0026" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Guarente;citation_author=R. R. Yocum;citation_author=P. Gifford;citation_title=A GAL10-CYC1 hybrid yeast promoter identifies the GAL4 regulatory region as an upstream site.;citation_pages=7410-7414;citation_volume=79;citation_year=1982;citation_issue=23;citation_pmid=6760197;citation_doi=10.1073/pnas.79.23.7410" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Food Microbiol.;citation_author=H. L. Jørgensen;citation_author=E. Schulz;citation_title=Turbidimetric measurement as a rapid method for the determination of the bacteriological quality of minced meat.;citation_pages=177-183;citation_volume=2;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=T. A. Knijnenburg;citation_author=O. Roda;citation_author=Y. Wan;citation_author=G. P. Nolan;citation_author=J. D. Aitchison;citation_title=A regression model approach to enable cell morphology correction in high-throughput flow cytometry;citation_pages=531-531;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21952134;citation_doi=10.1038/msb.2011.64" />
<meta name="citation_reference" content="citation_journal_title=Cytom. Part J. Int. Soc. Anal. Cytol.;citation_author=H. Kortmann;citation_author=L. M. Blank;citation_author=A. Schmid;citation_title=Single cell analysis reveals unexpected growth phenotype of S. cerevisiae.;citation_pages=130-139;citation_volume=75;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Lawless;citation_author=D. J. Wilkinson;citation_author=A. Young;citation_author=S. G. Addinall;citation_author=D. A. Lydall;citation_title=Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.;citation_pages=287-287;citation_volume=11;citation_year=2010;citation_pmid=20509870;citation_doi=10.1186/1471-2105-11-287" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=I. Levin-Reisman;citation_author=O. Gefen;citation_author=O. Fridman;citation_author=I. Ronin;citation_author=D. Shwa;citation_title=Automated imaging with ScanLag reveals previously undetectable bacterial growth phenotypes.;citation_pages=737-739;citation_volume=7;citation_year=2010;citation_issue=9;citation_pmid=20676109;citation_doi=10.1038/nmeth.1485" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. F. Levy;citation_author=N. Ziv;citation_author=M. L. Siegal;citation_title=Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant.;citation_pages=e1001325-e1001325;citation_volume=10;citation_year=2012;citation_issue=5;citation_pmid=22589700;citation_doi=10.1371/journal.pbio.1001325" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=N. Memarian;citation_author=M. Jessulat;citation_author=J. Alirezaie;citation_author=N. Mir-Rashed;citation_author=J. Xu;citation_title=Colony size measurement of the yeast gene deletion strains for functional genomics.;citation_pages=117-117;citation_volume=8;citation_year=2007;citation_pmid=17408490;citation_doi=10.1186/1471-2105-8-117" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=J. Monod;citation_title=The growth of bacterial cultures.;citation_pages=371-394;citation_volume=3;citation_year=1949;citation_doi=10.1146/annurev.mi.03.100149.002103" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_title=Quantifying yeast chronological life span by outgrowth of aged cells.;citation_author=C. Murakami;citation_author=M. Kaeberlein;citation_volume=27;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Pattern Recognit.;citation_author=S. Pertuz;citation_author=D. Puig;citation_author=M. A. Garcia;citation_title=Analysis of focus measure operators for shape-from-focus.;citation_pages=1415-1432;citation_volume=46;citation_year=2013;citation_doi=10.1016/j.patcog.2012.11.011" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Journal;citation_journal_abbrev=Biochem. J.;citation_author=G. L. Peskett;citation_title=Studies on the Growth of Yeast: A Nephelometric Method of Counting Yeast Suspensions;citation_pages=460-466;citation_volume=21;citation_year=1927;citation_issue=3;citation_pmid=16743859;citation_doi=10.1042/bj0210460" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Preibisch;citation_author=S. Saalfeld;citation_author=P. Tomancak;citation_title=Globally optimal stitching of tiled 3D microscopic image acquisitions.;citation_volume=25;citation_year=2009;citation_pmid=19346324;citation_doi=10.1093/bioinformatics/btp184" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. M. Roberts;citation_author=F. Winston;citation_title=SPT20/ADA5 encodes a novel protein functionally related to the TATA- binding protein and important for transcription in Saccharomyces cerevisiae;citation_pages=3206-3213;citation_volume=16;citation_year=1996;citation_issue=6;citation_pmid=8649431;citation_doi=10.1128/MCB.16.6.3206" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=N. A. Shah;citation_author=R. J. Laws;citation_author=B. Wardman;citation_author=L. P. Zhao;citation_author=J. L. Hartman;citation_title=Accurate, precise modeling of cell proliferation kinetics from time-lapse imaging and automated image analysis of agar yeast culture arrays.;citation_pages=3-3;citation_volume=1;citation_year=2007;citation_pmid=17408510;citation_doi=10.1186/1752-0509-1-3" />
<meta name="citation_reference" content="citation_journal_title=Lab on a chip;citation_journal_abbrev=Lab Chip;citation_author=J. Sun;citation_author=C. C. Stowers;citation_author=E. M. Boczko;citation_author=D. Li;citation_title=Measurement of the volume growth rate of single budding yeast with the MOSFET-based microfluidic Coulter counter.;citation_pages=2986-2993;citation_volume=10;citation_year=2010;citation_issue=21;citation_pmid=20717618;citation_doi=10.1039/c005029f" />
<meta name="citation_reference" content="citation_journal_abbrev=Comp Funct Genomics;citation_author=C. L. Tucker;citation_author=S. Fields;citation_title=Quantitative genome-wide analysis of yeast deletion strain sensitivities to oxidative and chemical stress.;citation_pages=216-224;citation_volume=5;citation_year=2004;citation_issue=3;citation_pmid=18629161;citation_doi=10.1002/cfg.391" />
<meta name="citation_reference" content="citation_journal_abbrev=Wiley Interdiscip Rev Syst Biol Med;citation_author=D. W. Van de Vosse;citation_author=Y. Wan;citation_author=R. W. Wozniak;citation_author=J. D. Aitchison;citation_title=Role of the nuclear envelope in genome organization and gene expression.;citation_pages=147-166;citation_volume=3;citation_year=2011;citation_issue=2;citation_pmid=21305702;citation_doi=10.1002/wsbm.101" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Chem Biol;citation_author=J. Warringer;citation_author=D. Anevski;citation_author=B. Liu;citation_author=A. Blomberg;citation_title=Chemogenetic fingerprinting by analysis of cellular growth dynamics.;citation_pages=3-3;citation_volume=8;citation_year=2008;citation_pmid=18721464;citation_doi=10.1186/1472-6769-8-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=K. Yoshikawa;citation_author=T. Tanaka;citation_author=C. Furusawa;citation_author=K. Nagahisa;citation_author=T. Hirasawa;citation_title=Comprehensive phenotypic analysis for identification of genes affecting growth under ethanol stress in Saccharomyces cerevisiae.;citation_volume=9;citation_year=2009;citation_pmid=19054128;citation_doi=10.1111/j.1567-1364.2008.00456.x" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=M. Zackrisson;citation_author=J. Hallin;citation_author=L.-G. Ottosson;citation_author=P. Dahl;citation_author=E. Fernandez-Parada;citation_title=Scan-o-matic: high-resolution microbial phenomics at a massive scale.;citation_volume=6;citation_year=2016;citation_pmid=27371952;citation_doi=10.1534/g3.116.032342" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=M. H. Zwietering;citation_author=I. Jongenburger;citation_author=F. M. Rombouts;citation_author=K. van ’t Riet;citation_title=Modeling of the Bacterial Growth Curve;citation_pages=1875-1881;citation_volume=56;citation_year=1990;citation_issue=6;citation_pmid=16348228" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="Cell growth is a complex phenotype widely used in systems biology to gauge the impact of genetic and environmental perturbations. Due to the magnitude of genome-wide studies, resolution is often sacrificed in favor of throughput, creating a demand for scalable, time-resolved, quantitative methods of growth assessment. We present ODELAY (One-cell Doubling Evaluation by Living Arrays of Yeast), an automated and scalable growth analysis platform. High measurement density and single-cell resolution provide a powerful tool for large-scale multiparameter growth analysis based on the modeling of microcolony expansion on solid media. Pioneered in yeast but applicable to other colony forming organisms, ODELAY extracts the three key growth parameters (lag time, doubling time, and carrying capacity) that define microcolony expansion from single cells, simultaneously permitting the assessment of population heterogeneity. The utility of ODELAY is illustrated using yeast mutants, revealing a spectrum of phenotypes arising from single and combinatorial growth parameter perturbations." />
<meta name="og-title" property="og:title" content="One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/279" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cell growth is a complex phenotype widely used in systems biology to gauge the impact of genetic and environmental perturbations. Due to the magnitude of genome-wide studies, resolution is often sacrificed in favor of throughput, creating a demand for scalable, time-resolved, quantitative methods of growth assessment. We present ODELAY (One-cell Doubling Evaluation by Living Arrays of Yeast), an automated and scalable growth analysis platform. High measurement density and single-cell resolution provide a powerful tool for large-scale multiparameter growth analysis based on the modeling of microcolony expansion on solid media. Pioneered in yeast but applicable to other colony forming organisms, ODELAY extracts the three key growth parameters (lag time, doubling time, and carrying capacity) that define microcolony expansion from single cells, simultaneously permitting the assessment of population heterogeneity. The utility of ODELAY is illustrated using yeast mutants, revealing a spectrum of phenotypes arising from single and combinatorial growth parameter perturbations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
    <title>One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY! | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Aa1F5C5GKWSKASpOFRQJmJ-oiNZQWh07DBaCJ3qgysE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460544","apath":"\/ggg\/7\/1\/279.atom","pisa":"ggg;7\/1\/279","ac":{"\/ggg\/7\/1\/279.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/279.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/279"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/279"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/279.full":true,"\/content\/7\/1\/279":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j1eGrHLmqCmAyvVFESgkPOEHzz+3Jmuf3B3T0+HIy0cdyg\\\\\\\/IvUU4O6ahDWpjCxgXdHPGQJlQOnTs9I\\\\\\\/oBKAdB9dADy0eXyJUqD9pP7zLrn70xyeZNs84h0cYhBRCswNpIAfwQRdsu1EmNyq6ofD0NhjZeGAstTPg6dVbeywIY32dPVYeqlt3UffgIHZlR2DzK+9hZelos5Z16HfdPHceh0BYwNjq7p0LtftQeQba0DhJZ60i9PRzXeJiBNihtTWZY39YIyfRn635ky64qpbMK8\\\\\\\/612GoF+vAEwmzncFNpSg705FIWbEb+TGNIFuXgnYj1ehAChZSECeh9ew\\\\\\\/PihxBqz4xpl8bWB2\\\\\\\/YXPG9bI\\\\\\\/GaXAvbEtjL70IFuM8aYOgUnfwZr7uaWIF4u8klcsuJaCyP68TuPMDvvE5HtXSN8kcCvKMXacvlIhL72YGWqdSi5dHEjR1JCTs+HlpjU\\\\\\\/fq7KfKO\\\\\\\/QuHFkF0OnESF5yAtztSJtWWI5ERp8LjApFQGU8fp0cV3nWFvkXhtQsZqScdaO6LGqtegP+ApDcwfESQ3qcndf\\\\\\\/Mtdsril1YpPCmaQzWpWgPbv2rArg3bd4AsyspQOXi1tVsCfL6fX2x72zltDBfSklp\\\\\\\/RssyeiT7gRwQfNgta+hHGpeqMHSUHMeeOP169hcSC6WVKt3HrgGPQo2dUkVz4jnm8vzLDbWl9iXWI7V16c8WbQqH5giNmGQ2gztYk4DC1NcrUuOt9r2aQPIQlortvu9Wk5IfV8OKhaRFDXfxSGRVsY+gZJl7IrbLDAMFMceuwNXrYp6wXRY1bb85HSrqy1Mk+uAkJuCpo0\\\\\\\/moMJ0jr5bcAyuTzYB6sbmQ1yXZTwSyNMyU64BGVRgAtG4XutzS8erxsxIvAsH14O52\\\\\\\/iN9UON5kZ9tXgSaqT1gNmdRco00+vQjO+tl9x5PywqEMv03uU6kk5vRdCQfvDuqUqV\\\\\\\/0W3kgCBvm4KaTiv5OnchMIbqadEV2zLo0EKNqe2rejhhAN3cA1s3zYDAJba2M8zuj9mdIbsDM004KHms7CNT0pLeS2vpK5OWVZ71xOy5Hy2i7QGkyFq7xvPw4pjpt8A6B5e3Fy41437ZepgqMpxauNx2hqdR9dBTMNvgvqLAG5cMPB5xrs3elTgR34++ZlTb6wYcgU6C4djXf53FLuc7naTyXX7WE0hoBcRVasf7vI9qhx5\\\\\\\/plKHKkHT9D\\\\\\\/CPwsE1EVqdkO9re0h9PS\\\\\\\/zuR96iIXjrmz+sQBA865iAzufgMqsYhCBumrT4Zz3I0xoA\\\u0022,\\\u0022iv\\\u0022:\\\u0022wz4XEr67RdAiAHKbAdH5Xw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002228fbd8673373b66f799f6f900c724bf1a5eabc67736fa73e5c18a9936661bb05\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CEpERW3m2ggg3rU3AN9X4fE29G8txYOFwCGQXVq6uZvZIOKARoD2hlHOfk5N1WUd9by3d0gNWVFC5NIfr6E\\\\\\\/NhOfLBHA0\\\\\\\/4BBQxWcn7j1JNtAQAPipjwKCK8YE0gc08Fhww9vaocganh9IW+Qf6s+ScmynHyi5L7KxyqKEUJUHyxrHSPXxCa\\\\\\\/StOgTalQDby+lvxkIYaKy7bURZyIzbbf8n0ws5IW1qXzGov6MquBvW32lpkNMPeu+VPRQm+p06x57gV7YW+tdtAzTT8KoCAZZYAtcQ5DfUB0HmsI6L71cqAC\\\\\\\/SoSfp9wmdmTgP39eYbpuFzG733REkok29oq6kckFL0SI5cjsfAzS9TnEW+eHLHS4wy\\\\\\\/jKuhBMERlDmkhD8YYrl8PY3zSjEC6GY0SNJt\\\\\\\/SusBSpsVC48mAnLYnQp8R8gvoAob0guTaLLiHnElop8CiFr6GMOcvtpz38qY2Ssk8I50zmcDdpEqz77QuC1d6RpGPY8Ub8L\\\\\\\/uKioATpsvVgNZd7sA\\\\\\\/xIOWbVbicInvoBsFpvrXxpm7AIaWRszVkpKBFFc5sMZE86CYXgwyc\\\\\\\/rG4QM9q3AgbadWk\\\\\\\/OHSoKlvj5zb3TerOfyOPZT2sqJ17Wx9kgOa3B8qk81KFhXQbK2cW\\\\\\\/bbmbAlVgKE0tZchtRfBtbOHNNjVXxZB8zVjLzM6jYIrmfWOH42vra1WyxTafNlCnVjQhpfg9emMb2S53RNikkPBrGv\\\\\\\/EogkV4uzjbmaz3CsbmubbwhZYpAqfS9zxeHTT5aPKQEXjnfpjWmelasPHyvyaimabZ4IJuepuV8CjbP0vFuzuCMTy1txwtnhI65vjktxSTguRkQOw5w1cXUpZSNfXY+YkxWXNYoC7zZ\\\\\\\/S2HnFl91+Ytd\\\\\\\/XPM9UT5pNFEsjwqXfSUru5YuLwuSdS+dlGgjgDiJrzlXdFwHUal\\\\\\\/FQkch30t1svEdPyaKdsSVowtLaJgnnbZRx4ee2xe9u2OLoHKOHDCDigvwpKXrFqvOt8VbYky4dB1MQ03zCl7iPZfhCSsGKL1k\\\\\\\/FxBrZsO03RQsn0PMmqemje225uIBPGbbG7zooSvNlARSH0eoO39cC9zs6Fpk5TAXqFe8Z43AVwLYP6oMmjHsPsoCpn7aJhWP6G94rICZ5En7GXJASMduke9AeQIXCHy97\\\\\\\/DrKn+C\\\\\\\/90j4d2k+9DjyRMpYFBTizj2p0rvpaqcZhomxFuD+mAYTCLS\\\\\\\/b3mOxx+CF76S7\\\\\\\/URzhK+SBuwVOI++FzYNefq0WYFDSegi3kNe\\\\\\\/qkP0N6RXC\\\\\\\/y+uiDoYVe1+7bMYcxb66WL40VEE3keABY2DE5lRjrsSFtHDz47Dp\\\\\\\/XSZyAUsQrhDi1QTOpuDT9frox+WgPK8JRz\\\\\\\/Hc1odD\\\\\\\/R6Q8F281lMQP5MyCyqxyvpC2IukGsWJUFVlz22yxjqgei0uRqWBOZgzoOB9wKL5\\\\\\\/OqTU4XzeHFEkNZmN7z60TGwGaD0fYiQM6yNoQGlwxU5ixxH7soFoMSFj1DUQyUxTjLrHQnGf4LTGGcpz5dVH5BzOx2dZR6hq0ZtQoNglAid3kpGSm3rmRW0vfwHtVgPb+hLW00DSNz1SwjSSny8coLx\\\\\\\/iqIUi9nTuyPgO2Dvx+p5lzZWMObU9ZdFRWt+zhlwr3MOwgTkUIPsPFED1m9pHPqVYf9agrHMmcqy3cLECAw45Uem\\\\\\\/GQAq7EkshermfP7H\\\\\\\/OClwmN7OgXDB2bxtmdg1h5r9MT2sNxxY5TVg\\\\\\\/S8QKX+z3ov5ymb7Y\\\\\\\/AogNmCcxe0yT18RshXOzDY7v5V9+l0GYoBOx2K\\\\\\\/qVfeBJD0z38pRwk9E4eK2ZmivjfzdH\\\\\\\/FUCH2+ELIsUgDRRQfAKdlpz\\\\\\\/tksrrd\\\\\\\/DjvF5H2qTOVC\\\\\\\/6W1y5EI0d2KN7mQ6FVP5Qm2BkexG\\\\\\\/iwD9mhOR4Iqn94AVpEDaq7kqLH6dqxi8RDvd8n3+eDgXmO6LLuHNdjaRJe0T2rUbwdZwcX6KqsQynFBG8R\\\\\\\/DAF6QdhZUfd004rVNSaFHkNokndhcQXprBJcnwYN9vUO9RE1Gw8gY8zXAr3\\\\\\\/sRJIV15p0ggMVJHjPowp9suJbIfDMZCZVnwjh1vM1kTw1qH24I9zrc70a5F7BGzZnMVhh07rqaLJvstrTQorQ50RZnWmbozuZgUVsfv\\\\\\\/WZEnKYbawWuxmSXkPJxbFnoKJhuSZK3Ycyw1YtYPj1cGqkq\\\\\\\/6B9A4GE7IG84bpz1hPALGo22H0VOP9msjUBNgvew8BbjyKr11Maa24k0jJ0gt7vYuvvPzhOJfD+DrNvRA2vEwHQS8tz2DiUooUhSpTr04VKrwbcs5\\\\\\\/2ix6tstUekiLxdv\\\\\\\/vjqpwDK1seHsejVI4i+KCabUc26nmSEX7UMjqv84XainmtA\\\\\\\/jNgJu\\\\\\\/ZmAR0cDTUXe0P0LqSSnff2dKOmivJUpXpGV5Nwpwck\\\\\\\/wxrpE9pk8J3dq8JvaktforuqXgr5xeyc4xqA2uk9ppN8xg4hRjNqm6Umt9tfgJqaSLJoXeClqa+6ZM5fOexB1A\\\\\\\/NSlW0wfCpzthVLk5TFDgYqNDnEHfktkvReGpjFLF7aacqZ7NbZukQb\\\\\\\/PWm94I8tOeL\\\\\\\/BtQ1D3IWe4DLAHD\\\\\\\/m37G5O0kAdigaArZjtBxD5AKDqsY4NNyPGhBRGe6eOrDVMdNi+XdDNfInSPwNMPkVUjWlO2EokgZEiAYkhBB2pleksidV3xuaV1a1PMBlZDiVa8aSsP6dKMEv0OgLi1HGSJSBKb3AP7BOZLs7YB8KpNUp9AytTwRHumuTJ8WUbrAhORojazUceqyLI4oEMbea+dUsTEMji7pwVEksBVTt8sichzrWQjBXZPst5Vj7+GPjVUaTzGij4G9dkBcCqtbiBV\\\\\\\/RAptg5JzFeCyiDCu2H9Ja4cbDUyOJV2eSrpTGuU2yA7EnGOOuLw7WsTmO2oEGk8PLGXMImZT72+a\\\\\\\/8wUKHudCYS5RHZl3dLQhHCPj02AWvQL4rVznsHKWTITcFdST4Whv3o+gSzXi6mUhSgy3UAjqHD4ledlnQVhQPs7KMVdNjusz0Xh396hjdM7K94op3AS83h8Wr20kClG+jWATPEnumPwyCIFs2hgqPj9qGySEtDN5BGYlK4pqRp+V0fPFJdcX4GdT2WhxyOohHRkGxqCeFFYN9Xu93jfTNyu7KxLyiMX1\\\\\\\/njH61TjN5Yie0K6dCgBreaHKFkeMWWqVteQIs6orZ\\\\\\\/IJC1iZkcawxYfcmL4aomyMS379x3WGt32VmItD4SzxlDHjVRz9Yr3YonmpvRDpD8x2tUCqoPxOvz4Fv8xWP1qtYBHWCFp+mmMC8YxLIQKRO9ra5Am09Uwir31qf3O+z2ID+UxlIyc\\\\\\\/uYZRnI+yCHZTI9nJeqeznMWX4Xsv7uJbX6RpL\\\\\\\/QywSSlJqsDnCNviJKiI1s9aM770lpDwUKxqzOzmB9Vt18EwMGQufulupYdUFOMRUU8SkORErdcT0IceYX8NTLjksqKtcSmtk2RFTB9SNpmW4S331wfmch8MuUikAB7QZNpiYuC9x1KCnyMC3zfjmIonfCj+Wbxbem69P+QyBTrvyeSjKiVgfhTfI7ECecu9\\\\\\\/wEKt3VUANOBmw301cgLVkoje+mfBgI35pttnRXUKRvoYwU+2+6s3CAqGVaPJtBoPaJ61VcM2YRvy6Z3dZDm4ff5ZbUQJGDlATrM91gnSexq\\\\\\\/c3M2xYC2nkPkFpQDkce3M9BVJlBdSsPIDPitSvM+DuDwTM0uXACocHPdPaGNljwbrqMudPhRU+9R7wfBAXYVGNtE34UYjI6ktgEYJcgO0HeuC6nCflVQwLhWcOVMR9ncjWMwUBBpM5RqQxI6L9kkrKXIOL7P\\\\\\\/AXJhgGwMr8ZWobaVvXPNlA8X+oz9vIPMrM05i0qLw3JZiQTXFqcY3CNeuiH9MJbSzVxrABL7GVw4bglDzfPNhpWTdB5BTWfmXuXSBI1YL6AUOC5WVK1BhOp342WtVCb+1jTIvpKKuLZ\\\\\\\/yJkcqfbiPFvYtRle5sm3C\\\\\\\/sDcEfVDlZ7NNE9HmZbgCq7FBcHACBDmwiRr1I30RXg82+v1Odv\\\\\\\/59xLZJP\\\\\\\/mG6uH3rjfFIkB3TO0qK2sDNmOgWROzhJXJBAXcJ2BgHNAlSjuY1gnQTtMdbLEom+STGpRY06B6UxAo8TyuikANAQZTS80AYbUV38ddk0Dqb695xo6dUuuH1swuinX40y35rh5S37VDhLWB3YugdPpIOdmVb474tBVntwnJmTI\\\\\\\/qGbMOMz\\\\\\\/wiroUjvkdvsTTMfYamGhxxCpIcopObS778uVvaHTrezAgocmVT+c8vyvG0g7vKmJpFypMBcykv5usfuOZk+RsH3Dr3DZ6OWKsfKkXozpXOi7WqfB2aJYbfw6BtHZvPXLDE9nuvrqPgz0dE4gbT8+kYHQLRV10Xx\\\\\\\/o9qFP5lIp2AN+28eZrfg3cs6DAuT5YowL\\\\\\\/T6Ofp3bsf\\\\\\\/d2FqBBjcOHvl+ewc08+YsstpRRfYUA4Wgdktie4ceskcMticupq1oNpz6+kme1F3Ah5PK5gKXNdLEQs39zmRjenO9cbxl1BPVq1hApmknxIwn0MzA3TtUZlyLFHITVAE6COnsCbi71Lia6rIchD5LZtBrrc83r6OwH3UHNWPoEbWEnionXyDu+kqejdpJchlJZVzR83By7STYk++gnR2qC\\\\\\\/M5MG2AeIcbHDPsfB0OfVbh2GHslY8ICEchf\\\\\\\/UzembcOjie3nAl5phSidAEu8Rmvi2DpCb1X1B1I1mXJ3ZfKJ1PPpbu3tDUHijkT2pI5rdaDQz25yG4CxCnWPC62\\\\\\\/gsIUy\\\\\\\/0RqPFMXutBOuZEZjbN0krcvr2U81tVxB3L9Y0\\\\\\\/l3etzKXcethC6W17NDDW5xtxN+rmGWtAgjkEVeAawG5t9BhPBstb6QfjaYc7VVh\\\\\\\/VvqCCYso+qxED1dDlQA3lfk\\\\\\\/E7HbRIMlb6tL1tvepFvH79YC7yqtfF9CN9RJ4LnpD+UuNwj4+kDqm2P18GopM2nkMw2UtNR+PoYwIepcvM8aS4mX0cF2UjQx+cyjAt5rlO8a1W6bf0Aq\\\\\\\/PGBJE1EEpuBJVkrS\\\\\\\/3xZPgpvjKldoiCuhpMrN+cPLF+illHWasJ2CZOYDOVnav\\\\\\\/YyxC6SjobTEvngbFhGhbnlf75p39rAumhXJC1uG5umQ0lRYNshVlE147vOEDukAE\\\\\\\/RaRs1n\\\\\\\/QVlKAty0nNtgq8\\\\\\\/NuXyFKxxEIu4FR4I6HlqlEiX5a+dnjHRnBfq0wOEfA+9QT2nQauoaMawHT4FqF2UgnxUR\\\\\\\/ukAGYiq8y5JXpqPRJ3l4ZdIqS56eRrPks2Y3oD0SOEzsq1NrVNf5FDFMAS4iQ5M\\\\\\\/2NO9n\\\\\\\/9JQdKf0x3ZeZpTzvNyrus\\\\\\\/fjMQvq+ln6Zp\\\\\\\/+DUJ1uROv8rHoSQ+ZDLUYxJ0SFIUb4j2G\\\\\\\/XUUsvAnGjXlk0KNs+vFT9FqYdVWy7s6zNe5zKYZAebqJkgHvjxPZ66iv1tKaUX2OMQwIjpYbAHXt4YGfBQjZMFBbxvo6iUkrDGDk5S3cCbnxGLSncL5PgauBPZE9ZZ62mFvvv2ZgYqpLT3JCd5k\\\\\\\/AUlDH7qz0v8I5GiERRTtuSSflhWQza3FEIdl+32En9VCjeTGce9v0IL3bcHLJvwWk3WDEvXlgh4j+qSvKaMIGCb5NWf79e84A0woWeFYT9ZG8M++9knww2D65p75mTcKyby9fEIMP7lhbK3nNlLWMHPB1ldaHPuTMv3CltBQoOXhbaI2w+iaPfxvHkJJt6hLpEPc2EmirTwBNl5GL0FSRzMDj0gCLzmEwJjccbmuHTWgoXSBHpkop6nZUwwB82fbrl8uuGOiE+5H+XL7KofB7LpN9AD+O9U0YKoJevUN7eNjKOwBa7Zd6IAMj3vW2Fo93IqlYB1znbpqgFcXMn6JWYC+15SFPdTyTKia9oAgeKIm49kVcSt88Hbhz+b38joDlKcA0C8RdRwnJZhluexsnYiF+Bzwqv0luaDug9HVH4A1m7cPTc2lYpJOLJl\\\\\\\/0ZaTAUFKfN2K\\\\\\\/7glxAsWoyVO8UdpOw5f4oIT\\\\\\\/9oI8k\\\\\\\/08SgTxxgnJalBoWcN\\\\\\\/RLhg0yW60w9IOTY57hxp5LjjpI6yNoFoCFR+OWjppY4FMWsddoueyXSvUpe3p9LHqrAHsPbaNXDBC0SZmlVw8MAet1\\\\\\\/OXqjpnOVLlduR\\\\\\\/kP5ZvdE5kNGgWT0tppS\\\\\\\/fVmRQh4nrAlJxx+F9TiS8meoKaS2yJd1f7HKD4af4nruZ5mDzlxxTxJQGMIQasNrm83LN1jE7CcZHaqD4HQJnx6lJDS61d+ylegNKLgv2pnog6JOaWx2JWpgP5LYKPZjpEMTMUhKtB1B3ojWDCfVEX3zPPuSevj9cbnvcB2VQfeT6wPzTmOIoP5gh0wNQ1Pt4sD7uqK72MXhhWQzMhkzLpIqL9GIz6WMfDqqAq71bU31JhQj9sUSQldG+2izMUDowUxpM\\\\\\\/cqb1GEsCme\\\\\\\/zO\\\\\\\/UEjlReW33GXDceYPCdDRyJUTYxjJPgblK3lBpI+n2jAQWTnyygTY9TH0f\\\\\\\/\\\\\\\/fQSdPDXYSzMvIfCkNA6ptT9k47B83VZS+yd0GWrUtw6A6CBBSbBFCkYNCWwmJ1efVDcH51UQ2Rvx3zCp9y3mfR4O1pJUT699ip21ga5COCR+F5QVVkx4WBEm6t9iHkN9k1B5wBCCzFmlsIQgxXxMtc0Uu5YBrfSqS7jGfjP8FI6gKvVSNS1a\\\\\\\/eSXSYsVrBQ5J8tWozalbG06B8BKulN9Lnk4xJg6i6i0A8Uf4svY5CAle7bMMRDLRVrxS05K1SSpiYsTN5gMYU7lx\\\\\\\/xnInB2+3e+VRclIKkLxqvPyMslWbS8f0b0quiqjjpEqTVy9MKgULkwHPYM822rq\\\\\\\/yo31bGNLnSlzSW85rZRnIHE1\\\\\\\/vu9k7y9jG72d7W1934Vt8gJ8zj1WZ9u6OVeEXhdVffGeIqezsZm7vTP0me5MwE3Rh\\\\\\\/XW6CpTrxvogtbzg1GgMPM1QOYFFJqUQjQe6y2z4YdEdwi3x+f7UCiAqBA4PGB0dGcgXNCTPqagYNN3YYzAGrDYSrdKj+3yAZaP19gCQcHR4AopqEgleZ\\\\\\\/Rjtq4oP7tl5azG0S6T+IAp0AkCIbm3d7ZdfqToGLWqjLU1EuQOAgQ\\\\\\\/Z6o\\\\\\\/ZqgX8ZVrTmMcu1ZBnjkBdq1tQtQhHIbXa0t9O4mPc3oooRYkSjncz6b\\\\\\\/KTjAUa3PFNVXm2jL5FeQdjqFY8ZBfz8vzEdsWLk5C7NVfJ05IAxmqxSwxzjpAZs9UzJYPXed0DacudLszRhB9yhrh4nN1xnyZ7s+KFlqxU8ZiI2SxXaBj1cikAE742M2jE3ZXDlTYmVcSHhCPbN1cKwM+AdAJcIjKUDwE90nTDmeDiQB0yvKKiV4dNTTU7rAP\\\\\\\/5dieWt8DXI7bznz4I5LT0MXVe9TpcGcNo7NZARgmPDZGv00ffuJmSSNDpb9gQXSOpRTbpDdcUKkVRnlBQwEXaoY4DRRnomfNpe6TFv43LihwgmFNqAsWNgTQJjAchdKV3h+SFth9jW1lZU9CEAn8enbJuC7rWXA0fjsQ\\\\\\\/jN6CHxexBg8OrxbDjV8h7ZSBiIDYsgv5x27rwtuWusPfmhdsEhbIPhGIdUBGejg09wKB2lgRoxWXpX7u4FB41gGq+lSqiPOuj9eX1jtl19\\\\\\\/D+J4fVVvzwr377q2IF\\\\\\\/BuRa85+C5Iz+1Gq0s6puS2WsszvOzePBu45OqVZoQVUZF4W2h\\\\\\\/WZRGCZv5WWd3Y5ZSms0gBpBtxWiYJ8gEI9BDMMyz9QBkv8esD927PA39jly\\\\\\\/Cl0nIDmkVmHhB9P4E4YhNM7ENwRs6XJm3kl1VjA0LyCSJAmCig74qtwwN53GfL+jpxUw0zo\\\\\\\/qLtXQrkN91dc3XS8kYzLU0FN3p1HRrcvUaptA0\\\\\\\/BxiGiYyndNUlh5yBo+Xqry7wl+sdoZO2E3VjcScjXDKN7AvgeZT77TM+Z83CivvizgCAmbf5zIDnKaSq8ckbzyt\\\\\\\/gmxdqXEZteEjEnM0iWc6XafHvkB8YwLrG67X5J+HxYIAs3c0H2xyRPT\\\\\\\/uYj5\\\\\\\/EmJLtuF0YoJoNJXPduMVEA0lImORgT48fdR1L9of\\\\\\\/qLX+rmAVKMXWjioudxzc3jciPGUN1GlDbDNWRoTqzDp7mazhl3doRoqWOUlmwpyNIebXh+P7meQXaDVtHCohKwcpma7xjWmZaDuPJTxNshgVi7u1K8peklun6uoynJStn3cNmjg1u\\\\\\\/VLtNjYGJ+lw4F31FBdmoV5ffvlupFzw8PhBGlV3SgfLONQHVy93ujUMAECDEW5WY7p3YSbHDOMXrvSp+thLTjHBliJzLShy22awUvF9Y9jrZXHgiiIw\\\\\\\/kncCCIjupfR9\\\\\\\/lcR\\\\\\\/FitLI3dj+tdWZlgWKYU4Ocirz6Dn+j59M49ZKnk5ASb50bkvFXZnY+GJ9srfySW6v1sclw1yWLweV7xvWeQzcba\\\\\\\/jdZ2\\\\\\\/Z2cwrQ6Mwk\\\\\\\/AGnbWDUKxHfW\\\\\\\/XDi6+VZjQJnbyZMZJI4W9ppIYAWMwE4NKqVK8dfqzVCz4LkpSm9ozrMkrIjtuGFCL4ZOGRPA26gJDb5i85VLP\\\\\\\/mKjNtklJE7GlV985T9IKBhvMBibZvNBAb6RD6CjkHdEWyMZvZlI\\\\\\\/T9hS3\\\\\\\/bq0anyQDSK3zITCl5GEPi9bNpeN0bEbXj05ImfMVhDswfqUPY9GWzW+aJEP6QqL2vi+qllNVHbT0QD3F0ETDntGeLFyzWnUto7yjISZ7N2PnCELjEQqJBCo3e\\\\\\\/htAWazCkizYqwhUXUa+8wIqL2kq5y6qopaawMpbv1mJymL0WhYhQuH18AR+eJ0HthDgKOVIycZk7JkuMBMcK513gPHzQwv4mqLAq3Y3JqV3hiHB4ZVEVBxQUlaDC4QCf\\\\\\\/nw4rKyqBlx+4d6kU7iq8AY7ctQWPn3TbXA0nEnldZMVb1KuPhV+K7\\\\\\\/PnRReMerPsJPNXxgA4pevk1Ow75aIYMy7tD+1I5wS01WDMN8sFb6GMfIXgZVnIgevrechPlhouo5QsnUpRzBoy6G+c4MAvtUVOk3+BDdqyKHh9gR596Gyeh7pDqOTP0QOD\\\\\\\/0QtMIVfe3D6lwC6SMAfiy+SUg\\\\\\\/xzkneiA+wvmk1TcuuYXGCMoGhLhMjTBGkI5K2yrzDOqu\\\\\\\/pqKsQCK+QjR3Clw1ZLxM\\\\\\\/y6xWHPHmPpgMG4HPmz7bBprYfSetYgCbJLTKllwAIi1ZByzKe8Ss\\\\\\\/7hlpRkRqCAxpojrsdJ\\\\\\\/f1Xk59BjARqBkyHW\\\\\\\/UI1yEQ3a8nmzRKhrDoH+NUhuQlI10npEUPjW8QCe3EZ9aZDC2TyiPU8w+YCLrN+bQUQ8CGF6upyejyeDP7sZP971aKIAH\\\\\\\/gg3ViD2G6831057HSazF1kUZ9zE2EO4zkALVkyaTADjq43qgXNgswjIAI3MYMDcgQsk8Qpbtq5ovTSCyHa5lWSvehtdEee54lgUo3+QAbZ79WHre\\\\\\\/3l7+RDca8EpyZrT\\\\\\\/rcH7vy\\\\\\\/s2uQnii4YduAq2s68yn811s52kpt+zXtONBmPv9qhFh1Lw03+00EjZATiXtQ1qswHyme\\\\\\\/kW4WDHRworGubfD45oDPMPIdAu+2wmo8knb+dLCBRwbEB3NN00ebH4QNAVMotnHpEiGjNHCyRAubAjm1SfVezGprLDtTyQnWOWAjlOV5kz8j511BBCEJUaM70meE7NT8PPYzrt0bA9jHKQm6qhFysE7EJ4xjTz68\\\\\\\/ZMXPxLGRbxhl+SHB71A2K2uZGfbmm6+Pwcndg3wc1gAf9suW6BB+zP0+tQ66ZM1GewHAi0llUGUf84UDip+yzX5UV3cVCApYc9W1EWQT96dpSla3gofm0uf85V7yNo7DCXJgV\\\\\\\/qUU0jLjCSGggkaUOjMJsdwFqmj8wlVW2xVwCLPTSm8wQGF1dAjrsdIVM68+GlUUcFi+w4HhcRuRIAhTMgdQ\\\\\\\/f6VnoidTKy1WLi8al6kQQZBDuwvAA27G+Hsvzf9JuBmuW0jq13YnjyARS8iGF\\\\\\\/GlrpU3VcszDad5AjBr\\\\\\\/E15JwWZVQhYs+1s0A3cZjg+msVQ6sKJb6zoOnHVZ\\\\\\\/DJ7ls5Tp1HC+pif+eptsP4nIPHMBe9ZWOqOC1jlo1DushaCfj5mQhOS3Bh7pBaGz1UXq1yDSreCF9ABvZUEH32FGmwArFBJ4WiT2dN8bnuElk8PR0qt0LjoY3RNwqWeStE9mHUKt7V\\\\\\\/H6DRfOzN7qKz8nAAoWDg1IJRNWGf6L1uL0VX6Wdebo9yghSsCt\\\\\\\/Gni9wnGP6kGEGAs8LupevcyBzPwHY\\\\\\\/vXb21D8ynFDGR2HV3UC+TuL\\\\\\\/2+mhtmNW0x\\\\\\\/2qOB2DSBVD2n54xcpBZsBIzM4OOp\\\\\\\/z6NjfYSa2HMCcD9uiIqTyOd\\\\\\\/1\\\\\\\/I7FdK0pCKQAjFJ+zSwkHMKAl6v+NyHt36VholdFLFCsMVPfQsZVOkwVIugZ6Mob0wwLXx9VyvgjaHvB30ap3Pu8sf5sddW9NyxIkaH+4dkirATHmYUGEpZDAL6c\\\\\\\/ICTWMoqsfWRvncG8lMXWjjw95I\\\\\\\/0qbiaBp\\\\\\\/5BxrPpddRmtKx+MpcvOJbzxU7m+N1CpQH+WjUd4XtUSgCaNY8E0qI3RE8GXjjhjPSfrrZNKXBqbo6LgBGEIjd3y82Lm4XWq682Lzj+kTtVQXz9riT5cm2HMVt3QCtHy1IOGE\\\\\\\/0OrbUM0uiR9FCcfzJ98FvJEpv2XIQwckJlSgWURZV3sBxRLuT77w6Tvjn+k\\\\\\\/DSt03Ao9uyfVkLztcgNfoWYZ0UgzfQ4F3lwiUOJoEXnozF6SQJDIYVn4xZtTsT3oxgO2YyLv9KV+GqbwY3CvYTItSNV\\\\\\\/H2MWSPX\\\\\\\/3h3Qt6Z4qSpPmt15i7JTMjl1GvGJWwrrIKLjVZH293YNqnG9FdnlL7rGT\\\\\\\/VaBTqAE87riZ06PDRVcWQ+OSXVc0aP+nCmTtUipU71Q6MFTS2WgGjN3aOUr7A0eUz7gos3dbR0BYIXtthDwPtqUNfG6mA8y+bndszeFzfMuVMyGTW7Nl+RyOANnvA42WdriXya2A8\\\\\\\/fm8rFnKPnJmz5LyBiVmg6dEDLHX4TUNAxqLAfulricFpKaNPj7opIQfs1BzweMhoXVc7zAwZQcx2mwm2B9AtBDEHSSUXknMfqsE72PrET5\\\\\\\/MJVqgUjZ9a\\\\\\\/HVwIgtkZ7CrmwFjrWuwVz4Up6kdoxM6R+qt4DTJgYRRq2wjhvAZpeoy2oexX4stlJHzO1+WhIPThPA5g7SLzvnAu+ITBpsmzKwAt75yU3l8VU7DrphixHvbUPp20E6f9I1kA4H9gex2ea0dHeKi2IoIAhbOFuRN76PeLEZ2Ar7H+C+4ctu4EM\\\\\\\/cBbMifVUS5SNBlds253inMdOUs1TUYeKKG004as4YSYUTqBb3ySmB1xCjcPj0n+2oYRVQWgKK4KMzzhi433JYJ6\\\\\\\/1Pfi8lBGwi3EGShSjQcakNJ5409zKvEbSHTE+pWHYYSfrKA\\\\\\\/RnFhpgh08jgbFjL3tU+d+7Q5K471QAgRURg0p8gA096ApyP1J89RxcycFzj3PDDJbhTq\\\\\\\/CeQm8Mzpbm+k+KOP+a2X1LKdb0PUjwaqgiBnpKDCPSM6sNh7UYGsJbx\\\\\\\/MyTFp6H8m\\\\\\\/g1c\\\\\\\/C03KFsmm0K1gFg7GLSlJkWzluF+BLXLZSlw2Nqxud+IhaGcuZgzn6JaqWkXTAzr0wDHq9w9lB9Q1fJ8XCnN1u4\\\\\\\/YxyWXVX4iCA3jEFBOQuW71\\\\\\\/LBMSTH1W4tyie3+BtBLw9AHSfRy4lQpAkppNeDgIsuBULU+F0j5HPj8A76ZW1Ot0bErQOlkMPujI9s3T4QgIGnkql0gCWDqDA\\\\\\\/lG9eR8vTHK0H+MRe53xb1\\\\\\\/gJ+qCVWaAPBhdwKpeLQ65ue7zvnJpw4kHXllnRnYNueNuIPruoCcSmKs8F5\\\\\\\/kwLFasW0HZfIcR3LfiHrOkT1C6HT5YWueDMmgJnyQJUHCu\\\\\\\/T7frvCoS8jzPEGsdPS5CnjX6GTtRKtPLLt5gOLGY1GySBbXlhjCB3RxhtoyAXJhFwIijbfns2OYEwVxnNYT+7\\\\\\\/x9ZX2j9y3wyGZhPRfbtowuhMwA6hGVZGYBMEKVlH7ZpxkM39pIuSJSvLC22nTE4aOxzZO2JLn3HXgzOc0rgrBvwsi6jeyxBMA5fIg+kh+xi0FSlRowz3WQYrvnbYYcMudiV5bXUK0J+u1aaMysrBYDyKVqA2vj1TwZxILzs5G3I9wx1+OV06kEPqiGkXS8LkmuDvP9+B36ATBMgwDPWv8wA93rMX3798j6ma9IZHd5tbUp\\\\\\\/OxekcqJ6E5sQhxCm2Ku9pmCyuYNDovbuezph1VGROnjjNnnTEoyc77wIOTf0zUjaXwH4Gj8Pwjer40NZ\\\\\\\/hfT1qhoNgE3r51OseqgeVcQqRUwic6t4vYMb9wpLWIoDLrzKFf4IULXnUcHcsT5PO2iXeh2MEr5kyfmI9679Jf1oeNguuPXuXzeO2KyJ0inGw7\\\\\\\/wHDcIfGEpEn5uC3oCoFgLFv3Q1t+gq\\\\\\\/jIfF61npQegYPFVgtT41iRPP6cuovKSt9bQ3210yrovBse7B+g7W+9+6+Bc2inZHeHuGFruPotsrQ9Xb2AVXFaCMAahwGenuhewkmGMov3urnF2YcttjstUuJIe6B+uEztkYA98bS9\\\\\\\/i3ZSI9be7h3p8XCs6qdj\\\\\\\/uArJGSjA1Ge5TC3QDssbpw3KjEubQEKg1SH\\\\\\\/98PwWNJDVU18VS8GaWMSHMsKy\\\\\\\/ov6\\\\\\\/RsmnZx2QEo4AR7eLNxSWeCXJfkNjxtleCGf+afkOfmyCiLxO8E+38P54ywYEek6gvITUz51LtrxZKBRlmKzYMNFcFJndGtsHTnFC2U82dis5ImU3E\\\\\\\/THTH98FwRS9N1Skc2XkQ0gql7iDd+md6GC\\\\\\\/o7epXaVdx4p6g5+lg5Fzw16ziQMgT7Tq1t7bwQ7s\\\\\\\/2gBPsqAbyKahubNXUVb8yYCp1PXYa1SfJpfFxNMnSNqRJ\\\\\\\/ZTHIVurNPdhyTpVJKDgwsZE9r7uFP4VUaxMstJvEEy1mw6ld+v7mR1Yb9L3xzPFTLcOEtwXU6hlngCn+eKywF2KHSYha6GO7Ngen2tHMG5OWaaB1VNUP9b9+z1cg+H5v8OKlfd3P5hNjLOtQdTqCvNX+ImLHeZvyvxGdvDmKnoESUv2ouScOFpVXeJ\\\\\\\/+Ti3QTY\\\\\\\/TsDVtSI1mPa2DGCDnysJpLFswxr7sV3GWlTqDl9oGPWA0kBrg5XGfnN8LZ\\\\\\\/c7UmH3dRFgqGE3ZJrDJlGmJ\\\\\\\/h8OntOpMJvPcs3SAnU81QqYEvnhgTJ8R2vUU5EuLv+FOSf2Zr8+nkilN9FgikVzRmtj1r97sSezx7\\\\\\\/iu6nsKpBO7HAA1QCu9TENRZv6fua656Tn9Xj3B9FSAhMJPnyXKhr+DG1MDVExpXJ35fjYG7L3PVbhk2z9p7oXP7gDY7GTfG3QQdqcCxEJiF6xd+43y2YLWg1V1vuO6IMIwkDi+e1eOeNRDC\\\\\\\/hCtCluISKLepAvr7JG7lrnnORfaok6tQm\\\\\\\/nCpBjPmu1q3Y2VjlqG0BpKYF1wHcdrKYEb8aISAEJKuY88eUeEkxgVu0m3wL8HTrnJvnRozY4LgonA9bTT\\\\\\\/s3Pv\\\\\\\/D0JpLiS\\\\\\\/qoPlFXiKH5yFEIP8ZLn32GmlvNDzS1U7QaGGdJSRs3fBJ9FhVF6NeaKjaaLCx1kC33ZcwvCprgOLbd5fXhYNwLveMCXH9ZTpN7x0ADCu\\\\\\\/qfDUD+1RT3Sy8c1jSysKAYBZEw1DVEyI5A0I+DuCqtP2pYfr9q70P\\\\\\\/\\\\\\\/BWZdAKg04AMZ36oCo6CSOU7yT3HRSLXp9Lddw4xQ5Edv4R3NlU2w+1aJDRgXHRyd03U\\\\\\\/LOKSr3fuO19euK+2Xi1j7hlYd4Z+khuAT5HPZKq9f9NiPyQfth5h\\\\\\\/lhasFWz60U5919V2kgn4X3OTRbN50MKxUg90jpLgZoS\\\\\\\/IYw5rymPoiDU3jbfkx3sK4O9i1mfBXjZiCNAWOze4zaVqGvnzk92MEmN2RKQ5gW87CCQwuH3jbO8CTrm5JV7ulFnxHNVlBpJ2Uk3daRjBNNGLFwRz9V4libUwqcflIz0fOoi1WqtRsJtuamAZc2uP+802HXAq8MBxHf23ZaJRI73mja3uXfqC1V92sux35gprzr7yT4KiZF96pjeTb\\\\\\\/wDozBMT4tiz8CZiBh7jyrk+phb3WXBAsRrF5wv2khStsgiZpHpkEJUTLQwRZgoPPksktQ91owTNpzan8lh+UAdtq+\\\\\\\/ONDQfYm051xTRicVK+HcbyQ+x38uSNOgSOgUoFMAdTD12lDELt6sSbnR6qz+IVx4ww2s57FJguIFAxsFH9+Y7tvVGqV0BwSa3LE9giFL8qq+mIxjVjfvA6\\\\\\\/Y8AoHEMj5tkYrA7SIlCJ4iLRydZO1sC8s\\\\\\\/086Z49xl2aHLrCvkKKinI6pffIzrMgGz0rkrzM0myH54uJdlQuRfRdwIlUpa4iLtb10Fb7zgIxtuHc6xU7SkVJQDAj+n9Hsq0iF60bhWx5dHF\\\\\\\/tpqyhv6r\\\\\\\/BxZN1CQ5x35J0XkBstfc62KETS7tT3NUCVtN0kIqzf+yId1oa+qwn9Mc2ZSW9a3MMu+1zWCKcdl+XjSOhrr6hwNqmOhpBwEpkCqMWE\\\\\\\/r4YIZxmSA32NQuINXCNE\\\\\\\/GCosGdMUCJX7Bg5P9Vf2Ja1RHV6TY6w5kUDgtPDb3baxIuTv+ntswjeYW5tPCro5oj\\\\\\\/0xEYW1c5YWTYnY30s3aBZITF39+CnCJsIIq8323TAxT+gvn9w4KVBDVN9DWYMd+K8FU07FT8yKcnXTadn7tDTrjGZL+EAxr45GcxOnbII4mksDMFtfaMVH\\\\\\\/rppOiGKzEgWLYg3VdGc2ZigBv3+jymsUnXb5nosH5lYiPuz4xr0ln1hzLSsTxxWAzQ+0lOZsGD88zUhHT\\\\\\\/EKE4wWKG\\\\\\\/abv6RP1GnFb4OjJxpPzaantObQZiRFd1VCOJCI4oNCGKex\\\\\\\/gR6BjspoSWJbKJPN0vr9k1Vp2zXlIwCSM2B56oqmNRKIZ4cOgFgtB0OHlrvjtNoxjobZgYKOUWeLwGgRrySLnU6ovRkwx\\\\\\\/\\\\\\\/V4lg62fqufFld1aUW\\\\\\\/HfM\\\\\\\/bBPl9Aa6vM8gvOoGG2du4TLk\\\\\\\/bzIvFTfX91De0gKjcZmxIdDqyIPC5pWBsK5ZVeGsuk3PIqNfae1unmVNj\\\\\\\/R4\\\\\\\/lxhzT67bIQMobkYYC3+bDYjPG5t9QDWNhUPR2VcJ+ZuAWbZ2EXsKuohfaJptdMsohOJIwmSgvC69rtOSlP2hrWSvE5eF4OAvTtJ+pkapupRMGeQbICHxJT2Yuz9s\\\\\\\/6K41bI+HVNd9V5ThTm\\\\\\\/LPqBcuMBZgBbul9EggZ2KfUfxilH\\\\\\\/BPTFdz3kNxR7Bp5S+8q0PI7vEiHB\\\\\\\/mOXQicEPIZYxWfITIUr0E7qeRlq6sZUCQAGARFIzLqFJ20ofZZZjvRkynpk+Nup2scvrnoNYArO\\\\\\\/izCD96OMk9gC0EwrH3fo374sd+AlkZR98IpDjH8th0\\\\\\\/MTyzF7TOtxav3gLehtO0ilxzRN9sVmDFPGPcoKi4KA9QB7WO\\\\\\\/+VfxjWGZswKgTXh53QBNKiEbVGgAMDATK5h0n7d+FVAWjtmhRe9LBm2opd\\\\\\\/mgTbn26C5y8QSYf9ZKbe5Jlwd45fqtomCRUJHajD5HoSMXjQba3\\\\\\\/BA0IUvYojwXgfLHTYk7khSMB+Th0OPq+N2agkzmusg2zc7AbfWmCNxCezq557KKO1yCYgog5E+aPKaPmp5wL94MLzWRxKt4+05Ck2dDvFxph76+x4XRtng6LY4Gj1IorIdSp7\\\\\\\/lfFgTslY9soY8O3iReCwyVhJiN5buc9zCux7abwCwpZ70kYF4Y6aA+884oILO7zXfy8VAoQKDY3EITlsAljOTgfCSnaRf6IQNqrwXsCd7UjEN0d\\\\\\\/uuXelyHL56Qah4JuF9rmPrGn2FYIKOjTDuvPbu\\\\\\\/6D8aPvS3Kme\\\\\\\/2Pv9FskkdaP4lFBe9YhgMPu5u791hSXdNY94u1XCjgOQPivoVF34G4o\\\\\\\/joapLoiT54fTGgcYy+mnnKKCuUECPDkgyxxWk6vxSmupx3kVyrIyYLnz8UobP0bbxC+AdgueKrCDi5yY7MaggRHKla6Ls2EGbQQu0UYzgUN0DUqzS41TqUoRK5zuXG6PNDXZ\\\\\\\/DhbFLhO5E5v2\\\\\\\/F82O6EiPGFwVGM8q2tu7W1yU71WB7W1LGBCqinnoV6I5eSE2Yuw1hiWpQln7WI0LJFrsQ3YP7Xr85lJKtoyiVRWz7Dvg37tefdIpHdm+uuFo5iG4T1JsQwsaYero9+mPgPcCVqYQC6q+BizSMHqiH\\\\\\\/agbs8uYjrRkFefXrjnUMVwWQS8RGQZQ7\\\\\\\/bo5ypSdSqH4u2lYf1OYLCvUUT1Bb4Iidagx7LyyLDF3g7b2dg0prS2zIzHZTX0zZRZ\\\\\\\/bBBP43fs+rmBU6FdR4HrUy3P3dYl3zm56B0Yua8MZFcofyD9GsgQOn71G2olczPns6TPP2T2CzuUyYOuedvA0NC8wSHGzmnAB1hu3TSm6QXvyzr9NrTb\\\\\\\/qXM82aMnkaDK0Y5ix4IAvvXJK\\\\\\\/zCRVswlD+\\\\\\\/ku9P4nPdWmL04N8vHr4T8xEGMgwSIv4xpdH3ISKfwUfqrvEF90K2gQUiaVizz2AmnBAXnv7ecfiHACGyaY+cyInoo1nLC1LTSIkJl5fGsQ5mTYFy1K6\\\\\\\/YiWgPXF3ZWwEyJ9v\\\\\\\/v5I46jqLIQ3+dSAwkR+uZWusF1M4ny1js5VqWMpxnkIxgW4P4pKjjUCqwAJjbaxJCuocAu+P9fl0RxNbatCWVj5H5vtUcHradFhYjDp4ZE7dlw+mWEhq29yAwBMe5Mdj6+9AlDi22H\\\\\\\/KALjDFo8gfQrV\\\\\\\/qr9nlaVOCRDOScEv0fvhoGiN6Z0uFRH7f95vwewBCPVBdn\\\\\\\/AqvjrB8wOYOvKbwRd\\\\\\\/Jh0OR3pPK\\\\\\\/vqJgvZU0W3duQWUzQABKcpRrj873NPkknOB\\\\\\\/Gpyzya\\\\\\\/hMXVXPbKPf8I4B298LobT2Wa4dVF1oinMf4mgN+L5r63RWom5L1\\\\\\\/ZTXkedU3ONj6QkdPm1G2ZlQuDVq3uiS3Cd3HDjxvcVIMSojQd+6Cr1n9MNkuaVLq5LYcFJHzFrgQDEb1Xclc6yJX5umz4GmCvMyhzmcvxqTW\\\\\\\/5dfHd356zSEbF0aegc6\\\\\\\/TpgnAEtO2xspk0kRbS6b1FsCMpDEiGuzjmjwykCfwZfoH74Vg0vTCLLTg97CpOb95zPgtBZvXpmXxlphMvfb3IdaA\\\\\\\/UPeiLJzHyZnjteB456maKljRG38t\\\\\\\/RR6+GRGsfeN\\\\\\\/CTfNvFYo4GaXM7WyEw1SpUoYafJyGXDDhyTAm8cbSwQuDck8my8CNb2bjygOGfVAbeTEk4CsBWNg1JrwSh4jtae+FqvWYg\\\\\\\/G0LdmvDVPJTgC3\\\\\\\/nw8I4D+VIVAhriLkW\\\\\\\/4rRk5jCqCMKqvOyoBoUOymWo4sDtTYrqZrpWNAu4l1UelDZrZvbcHd1SXWs00T62+xnt6CswwL5CYBa9ugcWZd\\\\\\\/NUG3zs7K2Nn5zqZLRdJqmKGSvgpl8k\\\\\\\/VByPwJFhh6lGtta5QKf2OSqoDSKLDkljBUFuYkwqUBPrdPboP0kZW4V4yvV8C7Kr5aOWx4wrxYOATMRb2+ctHftBdufZSMxEH3CIkI4FG5SRQZdVLU86PCULv7tBUIJOprdj33q0aOz4SPuaQD\\\\\\\/1fJB\\\\\\\/ghlAPvWKzHDzggCLuSUyeqzNVxkiYDK25a6TntmKSVbqS0GCv5XNywuBP6cFqbtYKaFmgF85O5d2xDBhJ3LVJh\\\\\\\/DJckrrBCK8QJkBxhwmIpy92fzfGThMNQlU+IuM0rfthN83nFb0XqXKMoc3Ei8N3krV26dA\\\\\\\/ba7IR2AnHSBGCo5WAR9KgEELIxCDQP+p3tfyrffy+SANYlm3bKifx0QyozwyosqnuZpcmikMCYHkCTWgEzo018dGwhfNTKosry\\\\\\\/FjC8wQgyNddJqXzGuZHRFOXfAkjHEleIew9Sjb5QuDXYJUjtWW5OYDtt7hivwm6CDsbl9Mu82moGvaztcQC8sKDvbmRSF62dx1N9k9prgvRZtqyQws2q+ZeI0CYheD\\\\\\\/XqBVUcdKxqyxAjdH\\\\\\\/gKQkFmVIDGZti7NRBFI6Rpv94HskcCMEayFaJgye787AP8SSyGOGjNck6WXJZ6uaWsHB5q7xGZwYFURaUWktuuVeyZjPQHI9xRkRJjFjGlM6BJefjG+eHrcAX7NU8uhj3IAZV9QjRvwtgxlvIW3JuZgP61GaK\\\\\\\/e2NcmDfbZZx5v11j6TNVKe9iyWbf6JFo96pUrF106y15BC5kR6MxpnnKeFQWXTEgXJzZq4BDdiGPn+09ICvnWxnlxwVoExfMW+nPHD752UowN674v3f+QgaR9uTa8bMiBpt1gzWhtXhIcgFsAyUSz6Gbav0nV8JmIPaf4ztJ6eiEAnIt6fU12AGJTUnrZoEcLHuslk8Ev3J2RRh\\\\\\\/Huulik3Djr9xcrgcfcQsC5cFVy55ERoaeRAVYOcD\\\\\\\/Cb938troQGEZkduZeP3Eb9Zez9C4HzBr9uw\\\\\\\/WH7H2UiW63gqxZmOY+Gt2fF7Jekqad9ziJmMd\\\\\\\/bqF63\\\\\\\/fqVb41SI1euexBpmb4gbC3MGbzhxJ\\\\\\\/f6gl0e0bEpL3Qt9P1yEXMU3WezZr\\\\\\\/w+nyxkYoP7fMkqfFRFuek8tV7fu2fsRjAgnoZ\\\\\\\/EqIFDABg7KQR6KEWk9xehktoslx+BmKxZM2ZpYaZjT0+82Y3Z6PMI6kpNn3qjVi3TR+kDf8EMdd5Oh7SarG8kBMPJThds9skCM5r6C3h3U1b0TMOA\\\\\\\/V14Bzmq+YniZ9gE4vSev6YEUjj\\\\\\\/sQCJfKqSIuJgN\\\\\\\/YEyXzW0rjraxmipcuWDkScqoHRoLvw2MXrjbv4BbDIw5rpIlh7suVMCustJxKqiEYhTQwi8PScVa6xYULdQ3fTW2CgjuhrHyERI\\\\\\\/ODG2HabhfgUXKXRvh1LG\\\\\\\/z0WAYHTr6VZG8pMhI7NzeoahbCna6\\\\\\\/rfQyrzvrxa1sQ+trIbeZFmR02t3wCxiHU2TvTp0VXqZpekdNl\\\\\\\/EG+3qZ+t4ZFvWgrRw6TJ22biNttjGDk+5aufrtXAAa1bFn1S+w6xWJsBITqo+lCupKpoEMeAFrRtDkrC1yA+ZQOevy\\\\\\\/9taJo7Qe5QZfj5KrjFDwUbUhv3VJ+P2zw1QL2lrrpWSqGDyZSdupCeYi4ryxg6C1smbdzrlZMCL2w7KHbwobRktW5ADzALlIyQMzMTsd785NCfw3gXuZ0kyvQwa1U+fbx0IW01yjM27TOMrZP6jCUUIUYT4dZNf15Mb+E10rJcE+GxqrJWdOit1Kr9W0FPdUqgrd19slRfBgNB5XKw22dAfk8UZ69bfIS7yZVFeIIPCRbuSxI4Z60d4Vvdk9F431ytIqe97FjaS4lcfo6tws2nIU2dpa8ezRjRuw0PJKKNTlm827BinP4pFkora86wdfUrvsSxwufmhemJ9\\\\\\\/JUkRg3+htB\\\\\\\/kf\\\\\\\/eJXjqUhx2ulEzHudiqClFloqPLWJtCK2VK7twquTy4kTthaVWH6RAQqxUaTQtLqBp1xj0JZz1PoEfrk3Sej9LQ23ghol2S76Zv5Db0RBAPSHdGkXee+xm6pUOa73SKvR2sHqhFolExPwACX8MjrWlJeQTlvFq4tpsYK5LkkPaEM+Iq+KS\\\\\\\/sRUQQ4aHInXY+R2kfPyg2xzTgzB0eoaRxjxNA85hpLotrH8imMp8aU3r3JpHXpvuKMHn0+9QF7khIA4GlUHyvtI07RZJNmzIm6QoGvVxafV5yePSynZk1wR+vL1VZZy5ZOzG0eYOzol9zc8jE4dytWK3Nef+Y8qVkehGKn+Dap7p08Jt4D\\\\\\\/aDUjGuyznpOJn4ZZy+GjXpvGczx5EWfyyhIV70FuwD1B68xdUBh+z8\\\\\\\/PDciVE6X00R+pn\\\\\\\/fuw2sClUz8Od5C\\\\\\\/J2HeTON6e56DGXJlk0WfaJF1E5WsXNF1\\\\\\\/eHoN\\\\\\\/J17NtjdmYm56RLS6QLCQAB\\\\\\\/Q6ufA\\\\\\\/WXVtJIGIwff1nnHa0+GsUrxm6utHNx0ChjbalCvgdItxrQcSHxLIfLbYnSit+gFylxN9gaTO0Dh2ezuCsxrsZRPO5DwViPftop7VZjh6osESuqvU5oyI3qMJSQ+C524fO0zMXSIzS9Wn8QJk7gAo2JKwIm1mSyUbWEDmOB8Aj1G7DQn5p5DgDuuHcb9XCn6\\\\\\\/n1WkBXd0g8yz2+O9QjT2LZ4SBae9e7dcIxg01wHLbt6oNoZKmAKHGBTTx0KsrJ0ZN3y\\\\\\\/bHztKopEHFseL9ujOeuJTCsYAEthZvZXCEeha9+uYOsbYq3d6z9vNX0I9qNQAJh15gv8pXbkWCqwziUBTWFKOP1jgDmD\\\\\\\/CfOsCQEfMfiMD+ePxdJjWXAGBthT\\\\\\\/YpecAGJRoFBRvWQ7ZYWmV5emG6qAkNYvhXcQLVn1eIRht7UDwqornuV5MKdcpNlU\\\\\\\/eM8vCEoL7NjK0BGOy8EauOYRM0RMymmONec+9r30xneYa3rqcn8JUJCqoJeZ+r3iSBEkPUSvOgurEXba7w4MTAMXBLS584wVm4urHzvWLEXstcTX0xKtJLxMQ7uu893cyM6cO6QyDuqPYLirtvyTKaetj62b212YHSIytts02lLQNzR7mNazPUSGFynR+5t2CZhXhF4kUaKzBte8xBVVRl3ThvxjXWFch4JJtX\\\\\\\/CJ1oSQmHzoWQD8BmveEScuXx9xbrM3TeWopn8xOyhLDoFtwtwSbhLMu3RAWletHcIWwNaI19usE51OZhmUtzqHpzcog6mpmpBO2rdzQxBoyUPqNDUuUUN1Sm3wJM9\\\\\\\/sW4\\\\\\\/cakgQ8ijnF6f35RD7M+lRIMW1BKy0gFHIxc90IhcNfuGcJb3jG9YYhdo8HstCIL0zl3jtQk\\\\\\\/ukaCQKUrKPkEJDJKnaAywPHgfuiLczrEX7Wwr8abSNn8FwVmol\\\\\\\/a2+kgddLL+AhhCWnM\\\\\\\/RBPw8fd8F5MOs4LFdx+AZHtTkMn\\\\\\\/5gbccHiKLcjK0kN1\\\\\\\/MGqA1vQHc5osY9AOcURPZJBy\\\\\\\/bsZZzr2w0ogDjC+LbDg9oCfnz8P3gxo4\\\\\\\/0EntkGbIqQHBHC8KtA8KLHFGiAbHQs+mGyZrfaubBW3HdjAeLBED\\\\\\\/Rq03a2hgi\\\\\\\/DEU03E1Ck8p5TfR1Oy1tja41qaKlYQ2dX2gTF3E6B6dKjbXk1r2eX7kMH6LjGTmz53HHO8gSu8DsYkBDxiaP2evEvWXwsi46CkXyyytTVuDBT9Pe\\\\\\\/kE7qOugxcEzECuxDq8VWMyQ7KDtuGOWxi4mO09N2s7heL+Q4SG1aDgRmq\\\\\\\/DXXTJbjLaItGtilQVuNQDtdP3vT3CyzroIC7DTxRe\\\\\\\/ZkuZuv4iDKGt6gUIQ+IQK9SXPwjpybBy621\\\\\\\/Y6QGcUqjFL4vJYYgwheuJ0PmgpaGEctg\\\\\\\/VexEOwmRESL5B5UXXqeFygQU42fDnZo94MNEaw1mOfCJtY8Cq+KKs8D2s6EQuswA17jRLfNbzuMH3Ac75cOe\\\\\\\/9V8bSmuwqPjVgQB9FYCLAU3M0Lh3RL0v2ZTlPVdfpcrRsIjGUzBZowhAlnASZ83nnjadEQqiwwsEsb2dZp7Fb74D2GK\\\\\\\/A1CulN\\\\\\\/msfwNULXY+Y6JG0VCA\\\\\\\/bVxaAbwrubxDObARsKhXOsQOgrx1RKAGCvfQVfQS+3olCz7hDOBZ+4wAMg4fF6GfrzndxJJYW7sjt+USCTdSWljW\\\\\\\/XIargpYWsoZaJLo6kRovxoBrXaCmzN3XKGaiagJDniYgyVUkjxdfclD\\\\\\\/yR7mBfq\\\\\\\/CStKchI+A3VTWrmC0uy6qEUonoSs5nr3KS8gtChXLILBFo9Mpmh1Fv\\\\\\\/zpE40r8ZtrzAoUnJ2JcH7HItunWvLjHgv1O+V6GxEYiomuR2Hok3m8DAWZxfZreDudydjrr5QUplTcah3NvGNmMpS2sT8MhZJTlu1mUX4809hsAkGC5yBwWem9mFjleQwdUuMy99A2T59vmDyAiIU8Ms8r\\\\\\\/1nSCayPIJUm4AcCeJ8Vt0IrcoC777wrqku4dY7Fb8SbtDEJ+vqCJEtgPfVWUfv1py52lR5zspiC\\\\\\\/4vmhvN\\\\\\\/NPVA5Q6SxrF1g2tkgO2ToigWfAt8r8zaIb5T+iegQ7roG8998W1iZ7sq1BYC9hSYVnreUMkj\\\\\\\/hg2a2oQJ8ft4HgzhhRjxxK2nyKvQzspWV1mV5Cu5AX0SdSwEoO1N5fn6uvJ9Nv9r591++yhgGvM28E8PM4rHSwCVGYvzeRQXU4Y7Ttg7T8+7cxPEpL+2tOxM9dilPOywun4widj2DFw6Wwhd9YpyufKtTep55XRrI1786PFpHIjIuWYK\\\\\\\/\\\\\\\/IPx67Xlq8+dvMb4cXx2lS82TwrIcDtNsbgjqVaMZbNUZA9mJktCWlBP6NiuAmLt8+A4sg+mrz3bToEOSErazqBCZU8SDkYQV92IIN6NxmMcdlFFNVV4wDhqruZgDahe2gDGSdDGyvV+YQLPxSXWiyqDA3yv5KRAB+JeGyDLBzoV6t458ARRxtS0ySrYvIBEPuYhfFUnlOvc22gQqUHS1jBzpuyRt\\\\\\\/HHzQeFzmBWKA6LJ\\\\\\\/hnsiXoo5bM5+2IxNtoolNAH3yJC81mPjn6fZ8AlWC6MtQlSAj327cT9G1lR3Okq4cxikMEjjLigMU02v9PNo1Il1iT7LGJhg9oreOC9hIxF2cDGbjfWrNUV+NKz0xw\\\\\\\/KIU++W\\\\\\\/rcZ7NH1j4jvojOR7Xpr5EOi9rjr4eqVti0L64MNdy5LC4Vnihk7CMXcN+o79DaE4AaR6nnzQKzEwyqrqTJkpXVpcvtdkcWxlw68aXzQDsP+md\\\\\\\/bJiOzcr2I0luFcdqiDtAZpy7AnEu9PMht8163mkMDC4+otTWBEaqo229713nJVEMahxvB1ethbOpFHh49b1NVl5XECO\\\\\\\/Ovo8iIe+1GYyoj2k8KX9yi17e1KoPJN4z\\\\\\\/Saqi81EtmP\\\\\\\/1DHlfrv8qSXwUktHDwhn\\\\\\\/fsZbb25yWl1Gg1lEZrxpp8U3bNv7xq5L2Hu\\\\\\\/zPWkpkvr\\\\\\\/XwWvk7gZ9kdy6nRT0Dyg+6WuQEfi2cA2T9njDPy+Hy9YsYXOh27VmNYkVYO8S\\\\\\\/XcF+uR3z0LrOeeTeHOiLVVMMqlSEZLOrVIKHvH9oGUdyK8WG1pCPvPGkjbqO48cL326fYGup857jDxgzeLzxWmdS00tpAYNVLV9dv97da4\\\\\\\/OtEfnXV604UnjSA9O2p7LdNM\\\\\\\/0y\\\\\\\/yokAAFfsUq1w9JbBnxUJ3U74D2Iu1RdX2cYPpURqwiQiyISB6eRbjVPz9AueHo\\\\\\\/fDW3i4TznxSGMNVzXFF730KH0sVtuuJjV70oCp57d0vnb04FamdYteJ7qwj\\\\\\\/GZ1nsmABL9Nb\\\\\\\/jbYJw9bz9sGjU15ncFb2cU+3aYfd7eQLN0pSdI7t\\\\\\\/1ZIOxPiMY3j\\\\\\\/EbTRTmr0E4Uz8too22bA+7Eycj1LEpOQ2TNShx7AERei\\\\\\\/OnxfjVSolThgMmI0\\\\\\\/hmD8W58TkzvZy+RpGbGsopzUsta9zFJvZpPkqyTjeFxOYTpypEo9aPoWRW7T8ILPTHVoJ2mFnCGaFt3c+7f+u97zES896Bgi7ZQXck6BL99H8i7X5TNMqkANtcpqHJUne4lIza49HzaLFQtgYKIQ+ocLn01aUBz1V\\\\\\\/OOc2h6k1DHGo8ak5A53rFUAPNQS84XKwhSlEG22VNUpvmlIocXicHxJ+oa1d8hfFnCD2fMbeDjZ74YBSDi5QcX5TnPkJByclQxucZ5EojDvb1CMfVOh\\\\\\\/4YoyK5oCSQpotCuPqSg213EWM6FKsFGkQ+lOyNGhZH9dKzYABr0mC31Dnt+leOK4eVF4\\\\\\\/x2nt90BhlkzKyj7eqoHIrddLxcVXR7slNnMjhmd0kM6RpiXiLYZWYhrnzpDb1kYFMe\\\\\\\/OdsnpgMYHe7\\\\\\\/4RnrI6nVHdpvvtPROdn3UeBx7Fcd\\\\\\\/o0Z+FBRSfbu5yrB+VHoX4aGzskcBMeajyYGseTjF2BFAnI5rpkLZ714ZMGh26Ui8tzNiQV1M40uCz7lQlSSxVimpW3Y1sFwFdaordvoKAbBSLcWTndh1sVKQmVQ2srOol\\\\\\\/wKdXCNCfNxECJHLd2kuBiD\\\\\\\/majOcPbjZ39fasCPJZTj9Z4tK2IwMVw3WTV1O8Z5RmXCungoHYjNioToZDp1Hspaq94LC9GdAI1mqONkPUh+zv+zyDGJ+hIkbhj7pBwNDPEPa77C2pz5ktNCI+FTcOOI81pWOFAcP2wD\\\\\\\/BbX2t\\\\\\\/aj85Yd0WNHlM7OorAmJBbxShO\\\\\\\/IRGSSk5\\\\\\\/qiNqatnVvT+YwTOgOQ3xaByoBXykZ6LPwzqt4+GYDqsRH2Q7+ZvMcj9tkcIU9HVp92Tzajo1EqYG\\\\\\\/yRzUczqIEWddDO4dgojZkJFHM6p41BGjeD2Nd4R8Mdy65fgJDast9yPvQS\\\\\\\/GckUkKH+EzkvOWsDaULyquDQ91KpR\\\\\\\/v8DD03RDlyXdpJV\\\\\\\/3divgphrGGVZNR7KO2gtFCNBO0L65m9Pe1gMRJ2tmQz+B5nGIxjXdpe0rVncfy7IY1gGdywkKyI5UbFHJmGrfCjyCUusirD3jPdJn0dpZzCdte1A55ZVGgyWVBtyNfHh9aeKYPHl7mI2jgigqfZqMVHR6\\\\\\\/fBkUoWO36iH6Z3EIJEFi63hKYuO7euRJjMFmeUUYDgBwnK4TcKTGtJIz3sprPyxOclNyOPSvTWlvVDcLG2kI6Wd8n2T6BHEnhHByo3eTc8+8Hihm8g0sDzvVL0rh90RLnPIHZxg9+iYbnxw90A3vfW+OmnM5Dsup6DFjazzA5V+OIfx7U5TVfumUnv+b9+7PWdREMh3Ey+kkIkCrz5lTvxX9O7aQ6Bt\\\\\\\/fpp\\\\\\\/GKWXEjxtpc7o2ccO9W2z3HX6xWIDuKT0nhtVQAnL3vaINxC7rE8oHvomofzJ4vRZ9G7ahswf14kzYg\\\\\\\/IbSyDvMyzUb\\\\\\\/6pNyd8EIP8GMVTaIOvZLotvC0Xl2M57g9MVt1de4kdCsnhmaTXGU\\\\\\\/P3ek8nH2vP\\\\\\\/IObvz79i54LGYo3gcVgmTerGrZxipmSo3D722\\\\\\\/pVLZnjEGaBZpcRVVcRdAsXUsl731nycQvUKiUy\\\\\\\/Aewx1+9HXDt9rBL1RCqdWqV8ooaguy\\\\\\\/ML9P1ytZGHZa\\\\\\\/RdnSPUmjdl3jXzC\\\\\\\/SqXVQVbVsOYA\\\\\\\/ds0EJcivnAegk+KAT6tJIVtMNCEPalGMh8Kf40s847Tzwat+4MbOAIB8SHG2m3JHITBFJLovpxXdtdo4q9AhMXc8CgbMZFiiyMSRtGxEYR+FwHkrXv6T2bHJ708SxP0x\\\\\\\/3IsWQ+tfhOdoG3tzVuwa74ZwBY2kk88FFaJ9wme58avrEkY1FyLppuH1yErPrZKHzrCoFOlxaxQG\\\\\\\/9asX9rdgCry35JhUSxEHC13GWjszKsIEC\\\\\\\/Okccj0P\\\\\\\/Rbga5D+9ppEQiaWWry9zg5lqV8+dT8naAMcJGbO5So5WWbn8dEznPLuIpq3Pk0Gz8+XAt3NdU6l0ozf5wthYBWcUB\\\\\\\/+unjheLK3QUATBwlNDSGFtEBwx9ul3zOyU3wDRKhXxmfBx61TXAunE794dwFFN6mjnLXgcv1KULjqgyqYrML5qfSmwQvA7CsseiTXnSBjHc+Jf9c7As+MlJDEddgfonkhB19VWGw2c5VXs7jf0dyS9qOYxVoOL5TBSV7iyEtll4WlY9kk3Jieg8JSZta9Uc3W7wrpm6OSy52UkP2ZppKgsCqq4qfXg4D1368QUUnpWRyxioiumNbz3LIhMpe+WikFSlI0of5\\\\\\\/ndKcxSUx+bp1HFE9MpGAYTRDkFGYQ87BAsXN26NJppjU+lZnZEQPbL5k\\\\\\\/VYlDbYZ2poPw3Kz45rzild4l8stpFnkN2wmL29V7MSO2Y1X5GAWAvzubAkXHdPtcZ8WlDuQXiQcnV6U6Huvs5oE2QWcpsJ9pSydPYYoiWrQj1E7yQNIF2pNOlOu7F3UoJ2ii2pLi+U8Isnb3bWOfLPP6E33mpbmMv7IqBVinwbDTFW8tM5R1m+HnExgEeEDjqwEHkDbQJmDDCi9xH1hfuXcf5MJuruOS4qCpP4SP5OiW2WN6i9qjbTI6Ilmj7E8hyqRT8dicOJCKU+fZKn8G0Fw2CWKPPkahXWKWk0zFExo6zLYzJ74BKWD5AV3\\\\\\\/eYNlyiQDlki8S+RiKjGXKhtyntfysfe+Vj\\\\\\\/wYo8L3nYYrSVTwd9CqWTY78OqvXjoltEpoaG0iG6AZdxkGt5GCQJ2PKCnUy+b99TlZiUuYSoBUG8WAFPBgDNVUoCD1NEwj7c8UO7suTbm6xe0ZbJn4SRXqRxrsNfVAQj\\\\\\\/itPgXmCLtPQbyeBwH7OMQxTY0YQn8KyyVFKvKUqPzAaNxyzEIip9eP0aX+epE0UuNmJnBkEX96v4ecVWtX2+38hjAwWLqPln77tGAR3B+v86mSZiEmxOVP3Vt\\\\\\\/Q59M3tsyppb7Hi+lwOhIMnZbvlJFcCfOAVyK3lTYRvIXKiUmH6UFC0nAm2+mvMXA+A+4xxmD8psGAxzLFHyV8kbj68lsy7EXhz8eN6YzvwQQgIPvxfIkJTrJgs7zGpCRJdS55q91cmy\\\\\\\/iMpHjOk8Hb0IOyDtb6Hh8rWZ2AVa13FzQ7vnHbjWlI9J5b+TTFshWGHv3hTGXj98IkopR4y2+G9Djxun9HFCRfkxBGe4edEn7DWhjYEd7\\\\\\\/NA2ksxVFC\\\\\\\/5WNUwpxAs1hQSvW7KaS24I3zF1gPXCCRxuq8Sk192bi8fvagJ2p5lRg9VDyU\\\\\\\/GPbr0r91vHlAScBSuCXvElVXkyPWJw+9z8AAqTcQbuMix2hXMUA+lr0Nib5Q2mSHMHRc07384MA2r0mOIWgdMIKZH+\\\\\\\/1iF7k2O5L4pSMQ95t2SMxl3EuAYG1GaViAP5bbl7m4cZfnbEG75lhPnqFFPgRTdruBsr+eSIKVjDO9tRwvCqm8+Yd9TQWgFpxAdcrJ67zpWhyNOg+dKGEJPXkKuCpE9OUBeyoIMKBMWXvbzoUCQR5qFww1o4GxDs8ih8NmZovdeQBio\\\\\\\/j7wPiT9xXcQM0fl9ZjIg6jsbgGLx6TxF5u9fQxiOooytk4\\\\\\\/p82MPGwluJdh1\\\\\\\/6hrgWI22SfO\\\\\\\/PjeF0gkjljfLimP0K7DsW\\\\\\\/tzHBNa4Cl9fTsqPS7A8\\\\\\\/yCMMoU58NmC2EEwfmfdRgU0HPkK4kQ5mxPp6v2SOK7SVH2VwLnMVFhu1rYbKw+\\\\\\\/jWtaoE0oAJHHxArXd53y6dDPeeThxH6IDXgNH7WhW\\\\\\\/35jlP6J0xyAWvpjN1+kJi0ySf\\\\\\\/XjeLZehIlZScVoVlrmzvBLvy+Frm4fjDJU+R\\\\\\\/8USdiH5QPwY87RRxzmjfsFjddS+pISm7OKRP0TblTVpLCJpBl7ZXuVw4FEEiIjgM6aNOwjs\\\\\\\/8Fb+CU6V\\\\\\\/HYiZu9F\\\\\\\/gJ8xNcKDEhvApR4qELk3nIpD+yX\\\\\\\/o0vyO2COHxPaJ9cfO5uv7iHQiEPadPAQoTQ7gT9ZIINuUg+uSb7WrIXDQ3b2NvU2Yq18hbtT4JSZ4TYfPcBDdFqkrZ++AcnPrP8oD8Ro4iAdggtuGizLBsoChesmcejynFNtwc0eC7u6gieE41dk2FDonk5iLlQ9jCTjTKZuANevQqTDmA6lwFCbvUBfbuW9b32Afrzk4y9Jwg\\\\\\\/7XNzBm61D5RQpvFGGb0v3gfOKCHzpG3YdIVFDRQU\\\\\\\/RH\\\\\\\/JoIU1E1XH5YFOMLmyVMR12e31sbD7f03Eg99W4k8ogALKbSX8hZaSGFCF80+qG8GzP1YXgeyS8ysSUaAuSBvY+hlr3EqooKcB\\\\\\\/YZwJNBdpSsOCC9kq4qGzPB6yfr7C\\\\\\\/\\\\\\\/yYlrkIPYAKM8WuuemMgPr0HTZhrHtwQwvXfyLf\\\\\\\/okW0ZfN5\\\\\\\/xuGRkZn5hu9IkAJPzCFKbX87gg9CRUn6F8tT+bZ0AW40iYnEuv9bLRGLs806dGD6Ml8q0RLtelroNOUIUiI6MeqH5QNSbTp1lgdMVNbw1gEzf63YVtzX+gvewOjyBv3h1jhbvESzTSEMSmewHsiEqE9XI0KYy6K+sF6+saP9WSjBlGjfF22Jw88iwBgXrhjlfwtYUX4\\\\\\\/a3J6PoJKJujL0V+fgmT0oozwPhcialsgI5D0t1brAHLVPBoLZGxEdvwTBVW6QitF0jR+e8ybTEFNpVgsFUgVuPzoxbONDNeAguPEU1x3HvqC\\\u0022,\\\u0022iv\\\u0022:\\\u0022b8NS\\\\\\\/mGYgzdLlYRjB+JgAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a1ac1b436130dbc9872d3de02f04d233efcbdf9de441ba02d52be3cfec751616\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nScBzNN3M3CIlvo5K8dDvXh1EojLxTzDef1+LWk2zhy2dAXpb60Q65QrLQ\\\\\\\/mxZr3ZQN9sDV1b1kBcK3R+46kxA0ocsz4T\\\\\\\/PkfrzRct0fmj0lESVkc6n0eFHyku5P+eiEAB5vFZFxN5D\\\\\\\/u7dnFBIFqQfAGkcblxMU0qVoNIC2SLYdvPnIHpKybA0Dc4BQaM0oyro7wgy4HS66aeGn+Q0fbs6fUkXGd\\\\\\\/EQyjQQ8EWIMl5PakMcKyouC+hBvdt7kNIGGIkPdHyNi\\\\\\\/bIfbKMl4Oa5NKLvwn2JvviKxl74O9T4l9868BxiNMBaUrelgsD+kbznusidqowYWJgrHbt6XJ1dzpKpmEMrKe7\\\\\\\/yJddQfnB8s0E1vGKCOYXj3\\\\\\\/aBGWdhIIabch+k1+ZVuSJjugbxWKvKCf5dCDyQjWPOC0Z166CtrVsOJH+NjCMmCxcz5wAFR5nTIwlmLgAmlNDlyg5d6dUdNnysn2+9VVVD3w3h4Sjm3BSvkovjwCYXKDxaYAQNzT+ySEmAdP7KeC2PRkq3hIgqehT3DwOdewhODmYTBiafcpsssEs1kMKBt01KQtRi\\\\\\\/pb06cnfmCyVKsKx08oHZnFBZ3zU54f\\\\\\\/lq3YJFdLe5IvQs0Q191GyjKc8Hqb8o9ZSlsYPhUe5h0WPWC+pDZgVHpdJf7MLlRS9g98Pn1n+oJlt5u1bfRz5JlNO0CfcArvh6mwUzHuX4PFMtVfVxoApc1ldyZv0TcLHdu\\\\\\\/TBeEPLjeSjAPT6bSh+vSKfXxcY0NLZJ+7DYnF\\\\\\\/FP\\\\\\\/EfBwfugp1R+yzkKsMuVyz1iLmOawR\\\\\\\/BBuR\\\\\\\/1tLRgj3fiP5bj7M7+fu9tcBHIzowx6YlC8xo\\\\\\\/fddPZugwhqlqhnJTwFEPCOw95w825sDIPRcsaQBrG08iUwb1rCnF2pliNQ86z6F0Ax3mAuUl0gP+0QG4AkWvqb5NbtmPHTcFDCCsf+0ey026YtLgPscfan93tlfmS4Ux2h82UfzSW0iMvcNHzFhBYNnt1sm4F8DENtvyTCDMCryPBhdv9rtWCOpSNeX3ztX5GOT1smxC3MNkU4vpTJWyt40Ydt+4rXtN8b9thwdqG\\\\\\\/nzouQFbv7alOBv1xrbJWl05G61tOf6j2qC18vNNFVz29VhMsyCbVM6n4TvYevnps7D\\\\\\\/pgGRTmaWAQFwpi40pko6ECQtNSCjpHzxAYBcFbbPAX0rUwYq3YP2AKhcKue5Ef4r5nihE8CLEkZoLKSDnKVUnBhMPAzqcb9KfgUKGcZ2V\\\\\\\/EEz\\\\\\\/+8Oj4JosTEiDoueMeIkA3AOW2x2WS\\\\\\\/nlHcoMCOYPuqFeJNcwNirFQ4rZ+ArOoR14A16zY0bxRpy0uSyWH4TM3vyTAwN3WFdUvKvumgKhmmD7NhEukO3U0FXkVhIpMq60XKr8hdtRCAJ9VDb5qG9uv6ldy3Io9mEM2DcWI25SlEdWk4NuVFiWVBSicc1sPrsiOSVU3HkbaQPBWjlqVDaI2DPvn4oQml4Ss5QQ68Cw\\\\\\\/fqUbE7h4AvJTGEIogPEtd\\\\\\\/pROI+Q3z8o++fsU3Jj2Y2Odb6PtXLfKnp7WKGychIAkpe1J7RJvHMxIjdZvhjPu+i\\\\\\\/4k\\\\\\\/CF1Oixnh0gU9nw0Xnr5a0hiRjiajoTJxcUmWupzNrIRCe1PO0nTjsjKmyhVJcvTb\\\\\\\/DbRm6KYJvxrD+bZw4+pYP23qZRiYBD6olT6fnJTIrjM893fpKE2yVSycVHWMbmRQRj8SA89ZeMlFSVYtpdFybNIU2eizIDVJbdaAbcJKbVjh1PiSOFgN4WeHRI90dgr\\\\\\\/4mcqBp1muEiGeSz5ucUJiNb8oTiany8H\\\\\\\/uve8AMXpT6\\\\\\\/BUlgqkNPb23Kqd+nKfMPFMMo+DfuYtd8RVQbwk0Mr3W1F+5KAjbUkVkI09T6EiOsBj2qquzA84Vc8S+qiCgipLo5lllO5LKv1YM7bK523niQNo8hGJcqtVeE2VlpAAcqt2K0PY5o4XOxzAO4p6s7zPLDNr7qLh\\\\\\\/v8jjOVhA1l21st6aj+xTQR5VSg9mE3NaQGL7gNdRaBlOI96qBsLEAoQ1eg15K3fmWT\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/7j75OZ9jG3qn7bepzcgkQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228556e78d8a46f9f826921dd6e3fa51a2a34996a50e3d97c6fba5779d41efd7c7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460544 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/279" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0QWc5YidbJOXqwnltnDXfUkWSDDwdbuSOOMVAApUb28" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/279" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-N4pDX1zk7JQeDSoAIqo1x042h9VDH6qDemyUfirZpko" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/269" title="First in Vivo Batrachochytrium dendrobatidis Transcriptomes Reveal Mechanisms of Host Exploitation, Host-Specific Gene Expression, and Expressed Genotype Shifts" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/289" title="Correlations of Genotype with Climate Parameters Suggest Caenorhabditis elegans Niche Adaptations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460544" id="node460544" data-pisa="ggg;7/1/279" data-pisa-master="ggg;g3.116.037044" data-apath="/ggg/7/1/279.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thurston Herricks</span>, <span class="highwire-citation-author" data-delta="1">David J. Dilworth</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2177-6647" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fred D. Mast</span>, <span class="highwire-citation-author" data-delta="3">Song Li</span>, <span class="highwire-citation-author" data-delta="4">Jennifer J. Smith</span>, <span class="highwire-citation-author" data-delta="5">Alexander V. Ratushny</span> and <span class="highwire-citation-author" data-delta="6">John D. Aitchison</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">279-288;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037044 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460544" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thurston Herricks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thurston%2BHerricks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herricks%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThurston%2BHerricks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David J. Dilworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BDilworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilworth%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BDilworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred D. Mast </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BD.%2BMast%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mast%20FD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFred%2BD.%2BMast%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2177-6647" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fred D. Mast</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Song Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Song%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jennifer J. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BJ.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BJ.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alexander V. Ratushny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BV.%2BRatushny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratushny%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BV.%2BRatushny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">John D. Aitchison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BD.%2BAitchison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aitchison%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BD.%2BAitchison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john.aitchison@systemsbiology.org" class="" data-icon-position="" data-hide-link-title="0">john.aitchison@systemsbiology.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/279" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460544" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460544/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/279.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460544" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460544/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/279.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460544" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460544/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/279.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460544" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460544/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/279.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460544" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460544/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Cell growth is a complex phenotype widely used in systems biology to gauge the impact of genetic and environmental perturbations. Due to the magnitude of genome-wide studies, resolution is often sacrificed in favor of throughput, creating a demand for scalable, time-resolved, quantitative methods of growth assessment. We present ODELAY (One-cell Doubling Evaluation by Living Arrays of Yeast), an automated and scalable growth analysis platform. High measurement density and single-cell resolution provide a powerful tool for large-scale multiparameter growth analysis based on the modeling of microcolony expansion on solid media. Pioneered in yeast but applicable to other colony forming organisms, ODELAY extracts the three key growth parameters (lag time, doubling time, and carrying capacity) that define microcolony expansion from single cells, simultaneously permitting the assessment of population heterogeneity. The utility of ODELAY is illustrated using yeast mutants, revealing a spectrum of phenotypes arising from single and combinatorial growth parameter perturbations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/growth-rate" class="hw-term hw-article-keyword hw-article-keyword-growth-rate" rel="nofollow">growth rate</a></li><li class="kwd"><a href="/keyword/lag-time" class="hw-term hw-article-keyword hw-article-keyword-lag-time" rel="nofollow">lag time</a></li><li class="kwd"><a href="/keyword/carrying-capacity" class="hw-term hw-article-keyword hw-article-keyword-carrying-capacity" rel="nofollow">carrying capacity</a></li><li class="kwd"><a href="/keyword/fitness-assessment" class="hw-term hw-article-keyword hw-article-keyword-fitness-assessment" rel="nofollow">fitness assessment</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-3">Growth is a well-established, sensitive metric of cellular fitness that is widely used to interrogate genetic and environmental interactions. The most basic models of microorganism population expansion over time consist of three distinct phases: lag phase, log phase, and stationary phase (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Monod 1949</a>). Each phase is defined by a specific parameter that uniquely contributes to overall fitness. Lag phase, defined by lag time, is the period after initial inoculation wherein little to no growth is observed. Following acclimation, the population enters log phase and expands exponentially at a constant, maximal rate defined by the doubling time. Finally, a rapid cessation of growth is observed as the population enters stationary phase, having reached its maximum attainable level defined by the carrying capacity. By virtue of its linear nature during exponential growth, the log plot of population number <em>vs.</em> time has classically been employed to extract the three key growth parameters. Lag time is the period up to the attainment of linearity of the log-plot, doubling time is inversely proportional to the slope of the linear region of the log-plot, and carrying capacity is the maximum population size when the slope of the log plot approaches zero.</p><p id="p-4">In light of its relatively well-understood cell biology and genetic tractability, baker’s yeast, <em>Saccharomyces cerevisiae</em>, is a model organism commonly exploited to elucidate genetic and environmental interactions on a genome-wide scale. Many methods of assessing yeast strain growth characteristics have been described. One of the oldest and most common methods to quantify yeast growth is to measure the increase of turbidity in liquid culture over time (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Peskett 1927</a>) as now routinely measured by absorbance of 600 nm light (OD600). It was also the first to be automated (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jørgensen and Schulz 1985</a>). This robust measurement has been parallelized in 96-well and 200-well formats to evaluate the genome deletion library (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Warringer <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Yoshikawa <em>et al.</em> 2009</a>). Variations in liquid culture growth assays have been extended to incorporate flow cytometry or sequencing to measure competitive growth of genetically modified mixed cultures on a genome-wide scale (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Winzeler <em>et al.</em> 1999</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tucker and Fields 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Deutschbauer <em>et al.</em> 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breslow <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kortmann <em>et al.</em> 2009</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Murakami and Kaeberlein 2009</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bryan <em>et al.</em> 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Godin <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sun <em>et al.</em> 2010</a>). Heterogeneity of yeast growth rates has been observed with time-lapse microscopy of precipitated liquid cultures (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Levy <em>et al.</em> 2012</a>). However, dynamic range limitations associated with many liquid culture methods render them unable to assess all three growth parameters within a single experimental run; thus, analyses are often restricted to only one growth parameter, most commonly doubling time. Furthermore, difficulties associated with maintaining low volume yeast cultures in suspension at high densities limit the throughput of many liquid growth analysis techniques (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Kortmann <em>et al.</em> 2009</a>).</p><p id="p-5">The shortcomings inherent to yeast liquid culture assays have made it commonplace to employ cell spotting as a proxy for strain growth. Common cell spotting assays range from colony pinning assays, in which a pin is used to deliver a patch of cells onto the surface of solid agar media to serial dilution spotting analysis, wherein single colonies are obtained (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Memarian <em>et al.</em> 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Shah <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lawless <em>et al.</em> 2010</a>). While these methods are universally accepted, there are significant caveats to their use. Foremost, most large-scale colony pinning fitness assays are traditionally assessed from a single time point (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Collins <em>et al.</em> 2006</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baryshnikova <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Costanzo <em>et al.</em> 2010</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2016</a>). The lack of temporal resolution makes it impossible to deconvolve the different stages of population growth and, therefore, apparent differences in fitness cannot be attributed to the classically defined growth parameters of doubling time, lag time, and carrying capacity.</p><p id="p-6">More recently, flatbed scanners and single-lens reflex (SLR) cameras have been used to periodically image pinned arrays of growing colonies (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Levin-Reisman <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bean <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zackrisson <em>et al.</em> 2016</a>). While these methods allow for collection of growth parameters, the recorded images have a resolution of 20–30 μm (or greater) per pixel. This pixel size thus limits spatial resolution to distinguishing features greater than about 40–60 μm in dimension. Since yeast cells are 3–5 μm in size, flatbed scanners and SLR cameras with a macrolens can only resolve colonies containing in the order of hundreds to thousands of cells, which also prevents these methods from directly observing lag time. Corrections for growth artifacts arising from competition between adjacent colonies and variations in media composition must also be included in the data analysis pipeline. In the case of the widely used synthetic genetic array (SGA), epistatic miniarray profile (E-MAP), SCANlag, and Scan-o-matic methods, effects of some confounding factors are corrected by the latest generation of analytical tools; however, given that multiple data sets involving many query strains are required to normalize for batch effects (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baryshnikova <em>et al.</em> 2010</a>), sensitivity is proportional to the scale of the study using these methods, which can limit their practical utility.</p><p id="p-7">In this work, we present a platform capable of high-density measurements of lag times prior to the attainment of doubling times during exponential growth, and carrying capacities at stationary phase through time course microscopy-based imaging of microcolonies growing on solid media. Because each microcolony is seeded from one to a few cells and hundreds of microcolonies can be analyzed for each strain, population heterogeneity of the three growth parameters can be assessed on a strain-by-strain basis. Through increased sensitivity and the potential for growth parameter profiling, the enhanced resolution afforded by this novel method of multiparameter fitness assessment can facilitate the generation and/or refinement of gene–gene and gene–environment interaction networks for yeast and other colony forming organisms.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and growth conditions</h3><p id="p-8">Unless otherwise specified, all experiments were performed at 30° temperature using rich growth media, YEPD [1% w/v yeast extract (BD), 2% w/v peptone (BD), and 2% w/v dextrose (BD)]. Galactose growth media contained 2% w/v galactose (Acros) in place of glucose and solid media contained 2% w/v agar (BD) for cell spotting assays or 1.0% w/v agarose (Invitrogen) for ODELAY analyses. <em>S. cerevisiae</em> strains used in this study are listed in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1/TableS1.pdf">Table S1</a>. All strains have been previously described (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Deutschbauer <em>et al.</em> 2005</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Memarian <em>et al.</em> 2007</a>).</p></div><div id="sec-3" class="subsection"><h3>ODELAY culture preparation</h3><p id="p-9">First, 220 μl yeast cultures were inoculated in 96-well flat bottom plates (Corning Costar) and grown overnight. Cultures were then diluted 1:11 and optical densities read using a Synergy H4 plate reader. Individual wells were then diluted to a density of 0.09 OD and the culture grown for 6 hr to ensure all strains were in exponential phase. The cultures were again measured using the plate reader and then diluted to 0.01 OD. The 96-well plate containing the prepared cultures was then sonicated for 30 sec in an ice-cold water ultrasonic bath to dissociate cell clusters.</p></div><div id="sec-4" class="subsection"><h3>ODELAY slide preparation and yeast array setup</h3><p id="p-10">It is noteworthy that the sensitivity of ODELAY makes the method sensitive to growth conditions and so reproducible slide preparation is important. Growth media was prepared as a 1:1:8 mixture of the following sterile stock solutions, respectively: 10 × YEP (10% w/v yeast extract and 20% w/v peptone), 20% w/v carbon source (glucose or galactose), and 1.33% w/v agarose in water. Typically, a 150 ml volume of 1.33% agarose stock was prepared, divided into 15 ml aliquots in 50 ml conical bottom tubes, and stored at 4° until use. Agarose aliquots with 2 ml 10 × YEP and 2 ml 20% carbon source were placed in rapidly boiling water for 20 min to completely melt the agarose gel. Water lost to evaporation was replaced by weighing the conical tube before and after boiling, yielding a final growth substrate containing 1% yeast extract, 2% peptone, 2% carbon source, and 1% agarose. The molten solution was poured into custom molds that formed 2 mm slabs of agar supported by 50 mm by 75 mm by 0.1 inch glass slides (Fisher Scientific). The apparatus was allowed to cool to room temperature and, after separation of the glass slides, the agar plates were equilibrated overnight in a humidified chamber at 4°. Careful separation of the glass slides was critical as any mechanical deformation of the agar altered the lag time and doubling time of cultures in the regions deformed. The following day, yeast cells in exponential liquid culture, diluted to an OD<sub>600</sub> of ∼0.01, were spotted onto agarose slabs using a Matrix Hydra DT fluidics robot (Thermo Scientific). Slides were air dried for ∼3–5 min and then placed inside a microscope equipped with a humidified environmental chamber maintained at 30°. This yields approximately 300 single cells per 9 mm<sup>2</sup>. Spotting at higher density leads to merging of colonies and fewer measurements overall. The stage was then leveled so that cells remained in focus across the entire 8 × 12 spotted array.</p></div><div id="sec-5" class="subsection"><h3>ODELAY image acquisition time course</h3><p id="p-11">Bright field images were captured using a Leica DMI6000 microscope (Leica) equipped with a 10 × objective. Images were recorded by a Hammamasu ORCA Flash 4.0 camera. The microscope stage movements and camera were controlled by a custom MATLAB graphical user interface using the Micromanager Core API version 1.4 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Edelstein <em>et al.</em> 2014</a>). MATLAB scripts controlled the stage to predefined positions. A custom autofocus routine found focus at the center of each spot by maximizing the image’s focus score utilizing the Laplacian variance function (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pertuz <em>et al.</em> 2013</a>). After focus was found, a 3 × 3 tiled image was recorded that covered a 9 mm<sup>2</sup> area of the agar. Since the stage was leveled before image collection, the focus did not drift appreciably across the tiled images. These steps were repeated on each of the 96 spotted strains in either 30 min or 1 hr increments for 48 hr.</p></div><div id="sec-6" class="subsection"><h3>Automated ODELAY image analysis</h3><p id="p-12">Image acquisition and panorama stitching were performed using MATLAB scripts. Briefly, images were stitched using a method based on FFT phases (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Preibisch <em>et al.</em> 2009</a>). A threshold of the stitched images was calculated by taking histograms of a subdivided image and finding the maximum intensities of 100 regions within the subdivided images. This threshold was used to binarize images and colony area was quantified using MATLAB functions. The log<sub>2</sub> of colony area was plotted <em>vs.</em> time and colony area fitted to a parameterized version of the Gompertz function (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gompertz 1825</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Zwietering <em>et al.</em> 1990</a>),<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>a<sub>o</sub></em> and <em>b</em> are parameters that represent the initial size and final saturation of the colonies; <em>v</em><sub>max</sub> maximum growth velocity, and <em>t<sub>lag</sub></em> colony lag time. Growth parameters were solved for directly. The <em>gompertzFit</em> routine calculates an initial estimate of the Gompertz function using a coarse grid optimization and then attempts to find a constrained minimum of the function at this initial estimate using the <em>fmincon</em> MATLAB function. In order to proceed to curve fitting, colonies must be matched at five or more time points through the monitored time course. In addition, colonies that do not exhibit at least a doubling in area are eliminated from curve fitting. This is achieved by only fitting data for which the maximum observed cross-sectional area of each tracked object is at least twofold greater than the object’s measured cross-sectional area at the first time point. Doubling time (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-2.gif"/></noscript></span></span>) is calculated as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-4.gif"/></noscript></span></span> is the point at which the growth rate, <em>f′(t)</em>, reaches maximum (achieved at <em>f″(t)</em> = 0). Lag time (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-5.gif"/></noscript></span></span>) is defined as the time to reach maximum growth acceleration, <em>a<sub>max</sub></em>, where <em>f″(t)</em> is greatest (achieved at the lower value of the two solutions to <em>f′″(t)</em> = 0). The carrying capacity (<em>K</em>), in pixel area, represents the cross-sectional area of the base of the modeled microcolony projected to stationary phase (<em>f(t)</em> as <em>t</em>→∞) and is calculated as follows:</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/279/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span></div><div id="sec-7" class="subsection"><h3>BioScreen doubling time determination</h3><p id="p-13">Automated optical density measurements of yeast cultures were obtained using a BioScreen C (Growth Curves USA) using the manufacturer’s suggested protocols, with the exception that culture volume was reduced to 200 μl to prevent artifacts arising from liquid splashing onto the plate lid during maximal agitation. A starting OD<sub>600</sub> of 0.05 was utilized in order to ensure that cultures were in exponential phase once they entered the empirically determined linear range of the instrument. Growth curves were fit using the <em>gompertzFitBioScreen</em> function, which is identical to the <em>gompertzFit</em> function except that it is optimized for the range of OD<sub>600</sub> values obtained from the Bioscreen C instead of observed area.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-14">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article. Datasets and files to analyze them can be found at <a href="http://aitchisonlab.com/ODELAY">http://aitchisonlab.com/ODELAY</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Development of an automated scalable solid-phase doubling time estimation platform</h3><p id="p-15">An ideal solid-phase, time-resolved, growth analysis platform would allow for high sample density and be amenable to automated data acquisition and processing. The optimized method, which we have termed ODELAY, is depicted schematically (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) and consists of four stages: spotting arrays of live yeast onto thin beds of growth substrate on a glass slide support (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>); periodic bright field image acquisition over a user-specified time course (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>); processing of raw bright field data to extract microcolony cross-sectional area data (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>); and postprocessing calculation of growth parameters for each individual microcolony within each spot (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). ODELAY is applicable to a wide range of growth substrates and incubation temperatures and is highly scalable, as it can analyze between 10<sup>5</sup> and 10<sup>6</sup> individual microcolonies per experiment.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/279/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="One-cell Doubling Evaluation by Living Arrays of Yeast (ODELAY). Solid-phase growth parameters are extracted by collecting time course image of growing colonies (A and B). Colony areas are measured from thresholded and binarized images from the time course image series (C). Colonies seeded from single yeast cells are tracked over time and the log2(Area) is used to fit a parameterized version of the Gompertz function (D, left). A control yeast strain (BY4741) was pregrown to saturation with glucose as a carbon source and then assayed on galactose-containing agar. The resulting heterogeneous colonies were clustered based on growth curve characteristics and graphed using colors to represent each cluster (D, right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-629571819" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;One-cell Doubling Evaluation by Living Arrays of Yeast (ODELAY). Solid-phase growth parameters are extracted by collecting time course image of growing colonies (A and B). Colony areas are measured from thresholded and binarized images from the time course image series (C). Colonies seeded from single yeast cells are tracked over time and the log2(Area) is used to fit a parameterized version of the Gompertz function (D, left). A control yeast strain (BY4741) was pregrown to saturation with glucose as a carbon source and then assayed on galactose-containing agar. The resulting heterogeneous colonies were clustered based on growth curve characteristics and graphed using colors to represent each cluster (D, right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/279/F1.medium.gif" width="309" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/1/279/F1.medium.gif" width="309" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/279/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/279/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470181" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">One-cell Doubling Evaluation by Living Arrays of Yeast (ODELAY). Solid-phase growth parameters are extracted by collecting time course image of growing colonies (A and B). Colony areas are measured from thresholded and binarized images from the time course image series (C). Colonies seeded from single yeast cells are tracked over time and the log2(Area) is used to fit a parameterized version of the Gompertz function (D, left). A control yeast strain (BY4741) was pregrown to saturation with glucose as a carbon source and then assayed on galactose-containing agar. The resulting heterogeneous colonies were clustered based on growth curve characteristics and graphed using colors to represent each cluster (D, right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">ODELAY consists of an automated pipeline that encompasses acquisition and processing of images, identification and measurement of microcolonies at each time point, matching of microcolonies through time, and extrapolation of growth parameters from growth curves. This current platform employs theoretical approximation of ODELAY growth curves using the Gompertz function as an unsupervised method to extract growth parameters (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gompertz 1825</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Preibisch <em>et al.</em> 2009</a>). All files required for execution of automated ODELAY analysis, as well as a demonstrative data set, are available as Supplemental Material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1/FileS2.zip">File S2</a>).</p></div><div id="sec-11" class="subsection"><h3>Determination of growth parameters by ODELAY</h3><p id="p-18">First, data are acquired, and then growth parameters of doubling time, lag time, and carrying capacity are determined by directly fitting a parameterized version of the Gompertz function (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>). For data acquisition, the first time point would ideally be acquired immediately after spotting onto agar at the desired growth temperature, but for practical purposes, the starting time is when the cells are spotted at room temperature on the solid substrate. The plate is then transferred to an environmentally controlled chamber and growing colonies are tracked until they merge with their neighbors. The time required for colonies to merge is therefore related to the initial cell density and the ultimate carrying capacity of adjacent colonies. While many colonies merge before carrying capacities are observed, ODELAY will still estimate carrying capacity as long as a sufficient number of data points are collected after maximum growth velocity is achieved. This is a feature of the Gompertz function’s symmetry about maximum growth velocity, which permits fair estimation of carrying capacity even when it is not directly measured. Note that caution should be exercised when examining phenotypes associated with increased carrying capacity because the Gompertz function may not accurately estimate all possible outcomes.</p></div><div id="sec-12" class="subsection"><h3>Comparison of ODELAY to established methods</h3><p id="p-19">We directly compared doubling times and lag times calculated by multiple ODELAY population measurements to liquid culture OD<sub>600</sub> measurements made using the BioScreen C instrument for both fast and slow growing strains taken from the MATα yeast deletion library (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Winzeler <em>et al.</em> 1999</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Population doubling times and precision of this measurement across replicates were roughly comparable between the two platforms (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Measured doubling times for 140 yeast strains correlated well between the two platforms with a Pearson coefficient of 0.76 and Spearman coefficient of 0.70 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). These correlation coefficients are similar to growth rate comparisons of growth on solid media versus liquid media reported elsewhere (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Zackrisson <em>et al.</em> 2016</a>). Growth rates measured with ODELAY correlate rather modestly with other colony pinned and liquid growth OD<sub>600</sub> assays, which is also similarly reported (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Zackrisson <em>et al.</em> 2016</a>). These differences likely reflect differences in growth conditions employed in each experiment or method. ODELAY-derived lag times showed less agreement with liquid growth OD<sub>600</sub> measurements, likely due to the liquid <em>vs.</em> solid culture medium, and the lack of sensitivity of optical density measurements at low cell concentrations (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). In addition, unlike BioScreen, ODELAY identified slow growing outliers because microcolony growth curves are derived from single cells. In contrast, liquid culture OD<sub>600</sub> curves measure an aggregate of all cells in a population, and therefore are not sensitive to the contribution of individual cells.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/279/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complex phenotypes observed by ODELAY. Comparisons of doubling times and lag times for repeated measurements (A). Red lines indicate Bioscreen C results. Median ODELAY measurements show good agreement with BioScreen C measurements in doubling time but less so in lag time (B). Population histograms of doubling time from SWR1 complex deletion and GFP-tagged strains (blue) with comparison to the parent strain BY4742 (gray) (C). Heterogeneity in doubling times is observed in strains swc3&#x394; and ARP6-GFP while arp6&#x394; and SWC3-GFP appear similar to the parent strain BY4742. GFP, green fluorescent protein; ODELAY, One-cell Doubling Evaluation by Living Arrays of Yeast." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-629571819" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Complex phenotypes observed by ODELAY. Comparisons of doubling times and lag times for repeated measurements (A). Red lines indicate Bioscreen C results. Median ODELAY measurements show good agreement with BioScreen C measurements in doubling time but less so in lag time (B). Population histograms of doubling time from SWR1 complex deletion and GFP-tagged strains (blue) with comparison to the parent strain BY4742 (gray) (C). Heterogeneity in doubling times is observed in strains &lt;em&gt;swc3&lt;/em&gt;&#x394; and &lt;em&gt;ARP6-GFP&lt;/em&gt; while &lt;em&gt;arp6&lt;/em&gt;&#x394; and &lt;em&gt;SWC3-GFP&lt;/em&gt; appear similar to the parent strain BY4742. GFP, green fluorescent protein; ODELAY, One-cell Doubling Evaluation by Living Arrays of Yeast.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/279/F2.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/1/279/F2.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/279/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/279/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470074" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Complex phenotypes observed by ODELAY. Comparisons of doubling times and lag times for repeated measurements (A). Red lines indicate Bioscreen C results. Median ODELAY measurements show good agreement with BioScreen C measurements in doubling time but less so in lag time (B). Population histograms of doubling time from SWR1 complex deletion and GFP-tagged strains (blue) with comparison to the parent strain BY4742 (gray) (C). Heterogeneity in doubling times is observed in strains <em>swc3</em>Δ and <em>ARP6-GFP</em> while <em>arp6</em>Δ and <em>SWC3-GFP</em> appear similar to the parent strain BY4742. GFP, green fluorescent protein; ODELAY, One-cell Doubling Evaluation by Living Arrays of Yeast.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Microcolony convergence is the limiting factor of ODELAY’s dynamic range, which can be controlled by altering the initial cell density obtained when spotting yeast cultures. Increased cell density decreases the time it takes for growing colonies to converge. In contrast, the dynamic range of liquid culture measurements is limited by either the nutrient capacity of the media or the linear range of the density sensor. Due to differences in strain doubling times, the dynamic range is best defined by the total number of doublings required to reach the upper limit starting from a single cell. At optimal seed density of ∼200–500 cells per spot (∼25–50 cells/mm<sup>2</sup>), the dynamic range of ODELAY is 8–12 doublings, from a single cell up to 250 or as many as 4000 cells, which compares favorably to a dynamic range of 3–5 doublings attainable by most currently available technologies.</p><p id="p-22">In traditional pinned colony assays, the size of a colony is dependent on the number of viable individuals contributing to the colony population, the number of doublings these cells have undergone, the amount of nutrients present, and the ability of the colony to transport nutrients to its reproducing members. The contribution of individuals to the overall colony size is not distinguished by traditional methods such as liquid-based or spot-based assays. In contrast, ODELAY tracks individual cells forming into colonies and can quantify population heterogeneity that other methods cannot resolve.</p></div><div id="sec-13" class="subsection"><h3>Example applications of ODELAY</h3><div id="sec-14" class="subsection"><h4>Identification of doubling time phenotypes:</h4><p id="p-23">To illustrate the ability of ODELAY to compare population heterogeneity of growth phenotypes between strains such that features of the population distributions may be evaluated, we focused on members of the SWR1 complex of chromatin modifiers (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). Chromatin modification is one way for the emergence of epigenetic differences that can manifest as heterogeneity within isogenic populations. We observed population heterogeneity in two strains, <em>swc3</em>Δ and <em>ARP6-GFP</em> (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>), but not in their respective GFP-tagged or deletion mutant. Population heterogeneity has been observed before in SWR1 deletion strains when measuring <em>POT1-GFP</em> expression during a carbon source switch from glucose to oleic acid (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Knijnenburg <em>et al.</em> 2011</a>). In that instance, deletion of other members of the SWR1 deletion complex induced bimodal expression of <em>POT1-GFP</em>. Here, the bimodality of growth phenotypes emerged from cells grown strictly on glucose media and without any stimulation from a change in carbon source. This observation demonstrates that ODELAY readily detects subpopulations of cells present in standard culture of deletion and GFP fusion strains.</p></div><div id="sec-15" class="subsection"><h4>Identification of lag time phenotype:</h4><p id="p-24">Through ODELAY analysis, outliers with highly variable, expanded, or contracted lag periods can be identified by assessing the distribution of lag times for microcolonies of a given strain (lag time variability), as well as relative lag between tested strains. To demonstrate the quantification of lag time by ODELAY, we exploited the well-studied and highly regulated response of yeast to a carbon source shift from its preferred source, glucose, to an alternative source, galactose (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Guarente <em>et al.</em> 1982</a>). This shift is characterized by a lag phase, during which the normally repressed galactose utilization machinery, including the galactose transporter, is induced. Exponentially growing yeast preconditioned in either glucose or galactose liquid medium were spotted onto solid media containing galactose and analyzed by ODELAY (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Cells preconditioned in galactose media exhibited a highly synchronized response characterized by short lag times. In contrast, more pronounced and variable lag times were observed for cells that were not primed for growth in galactose. Once glucose-grown cells acclimated to the shift to galactose and entered exponential phase, they doubled at rates similar to those observed for galactose preconditioned cells.</p><p id="p-25">As with heterogeneity of doubling times, ODELAY enables the detection of heterogeneity in lag times. To demonstrate the utility of ODELAY in assessing population heterogeneity of lag times, we compared growth parameters of a control yeast strain (BY4742) in galactose-containing medium after pregrowth in glucose media for differing amounts of time (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). We staggered seeding of cultures such that cells were pregrown in glucose media for 3, 6, 24, and 48 hr (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The resulting cultures were then spotted on galactose media and their growth phenotypes observed (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Not only did lag time correlate with the length of time that yeast was cultured in glucose but also colony-to-colony variation in lag times increased for the longer incubation times. This example demonstrates ODELAY’s ability to capture the effects of environmental perturbations on population heterogeneity, a feature which is difficult to distinguish using other solid media growth assays.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/279/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observing lag time after a carbon source switch: Growth curves and histograms depicting lag time (TLag) and doubling time (Td) for control yeast strain (BY4742) pregrown in Gal (left) or Glu (right) and then spotted on Gal media (A). Differing growth phenotypes are observed from samples of a control yeast strain taken from the same culture at multiple times after seeding a source culture (B and C). Histograms of the Td and TLag are depicted below each set of growth curves. Note the changes in the lag time distributions as the source culture is aged. This demonstrates ODELAY&#x2019;s utility and sensitivity to culture conditions of yeast. Gal, galactose; Glu, glucose; ODELAY, One-cell Doubling Evaluation by Living Arrays of Yeast." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-629571819" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Observing lag time after a carbon source switch: Growth curves and histograms depicting lag time (TLag) and doubling time (Td) for control yeast strain (BY4742) pregrown in Gal (left) or Glu (right) and then spotted on Gal media (A). Differing growth phenotypes are observed from samples of a control yeast strain taken from the same culture at multiple times after seeding a source culture (B and C). Histograms of the Td and TLag are depicted below each set of growth curves. Note the changes in the lag time distributions as the source culture is aged. This demonstrates ODELAY&#x2019;s utility and sensitivity to culture conditions of yeast. Gal, galactose; Glu, glucose; ODELAY, One-cell Doubling Evaluation by Living Arrays of Yeast.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/279/F3.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/1/279/F3.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/279/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/279/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470083" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Observing lag time after a carbon source switch: Growth curves and histograms depicting lag time (TLag) and doubling time (Td) for control yeast strain (BY4742) pregrown in Gal (left) or Glu (right) and then spotted on Gal media (A). Differing growth phenotypes are observed from samples of a control yeast strain taken from the same culture at multiple times after seeding a source culture (B and C). Histograms of the Td and TLag are depicted below each set of growth curves. Note the changes in the lag time distributions as the source culture is aged. This demonstrates ODELAY’s utility and sensitivity to culture conditions of yeast. Gal, galactose; Glu, glucose; ODELAY, One-cell Doubling Evaluation by Living Arrays of Yeast.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-16" class="subsection"><h3>Large-scale multiparameter analyses with ODELAY</h3><p id="p-27">A strength of the ODELAY platform is to extract doubling times and lag times for populations of cells growing on solid media in a high-throughput manner. To demonstrate multiparameter growth rate analysis by ODELAY, we assayed a collection of 140 strains that contained gene deletions of transcription factors, transcriptional regulators, and nuclear transport factors including nucleoporins and karyopherins. The genes selected were previously associated with regulating the response to a carbon source shift (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Winzeler <em>et al.</em> 1999</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aitchison and Rout 2012</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Knijnenburg <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Van de Vosse <em>et al.</em> 2011</a>).</p><p id="p-28">For the deletion strains, we quantified colony doubling times and lag times, and estimated carrying capacities during a carbon source switch from glucose to galactose using galactose-to-galactose transition as a control. This rich multivariate dataset underscores how ODELAY can reveal complex and heterogeneous growth phenotypes of populations of individual cells growing into colonies (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Strains with noticeably strong increases in doubling time include <em>dot1</em>Δ, <em>htl1</em>Δ, <em>eaf5</em>Δ, <em>eaf7</em>Δ, and <em>spt20</em>Δ. Of these five examples, only <em>spt20</em>Δ had been reported to have reduced growth rate on galactose media (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Roberts and Winston 1996</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/279/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of 140 deletion strains: doubling times, lag times, and estimated carrying capacities of 140 deletion strains that underwent a Glu to Gal switch vs. those that were maintained on Gal as a carbon source. Strains are grouped into categories according to annotated gene function (yeastgenome.org) including chromatin modifiers (Chrom), acetyltransferase enzymes (Acetyl), protein complexes (HDA1, INO80, ISWI NatB/C, NuA4, Rpd3S/L, SET3, SIR, SWI/SNF, SWR1, SAS, and SetC), transcriptional regulators (TRs), nucleoporins and karyopherins (Nup/Kap), and other genes associated with carbon source switching (other). Gal, galactose; Glu, glucose." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-629571819" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of 140 deletion strains: doubling times, lag times, and estimated carrying capacities of 140 deletion strains that underwent a Glu to Gal switch &lt;em&gt;vs.&lt;/em&gt; those that were maintained on Gal as a carbon source. Strains are grouped into categories according to annotated gene function (yeastgenome.org) including chromatin modifiers (Chrom), acetyltransferase enzymes (Acetyl), protein complexes (HDA1, INO80, ISWI NatB/C, NuA4, Rpd3S/L, SET3, SIR, SWI/SNF, SWR1, SAS, and SetC), transcriptional regulators (TRs), nucleoporins and karyopherins (Nup/Kap), and other genes associated with carbon source switching (other). Gal, galactose; Glu, glucose.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/279/F4.medium.gif" width="286" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/1/279/F4.medium.gif" width="286" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/279/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/279/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470170" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Comparison of 140 deletion strains: doubling times, lag times, and estimated carrying capacities of 140 deletion strains that underwent a Glu to Gal switch <em>vs.</em> those that were maintained on Gal as a carbon source. Strains are grouped into categories according to annotated gene function (yeastgenome.org) including chromatin modifiers (Chrom), acetyltransferase enzymes (Acetyl), protein complexes (HDA1, INO80, ISWI NatB/C, NuA4, Rpd3S/L, SET3, SIR, SWI/SNF, SWR1, SAS, and SetC), transcriptional regulators (TRs), nucleoporins and karyopherins (Nup/Kap), and other genes associated with carbon source switching (other). Gal, galactose; Glu, glucose.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">In general, reporting absolute values of growth parameters is rare in the literature. Here, we present a second large-scale application of ODELAY to compare doubling times of yeast mutants to the parent strain. A commonly overlooked class of mutant includes the C-terminal tagging with GFP, which is often assumed to have negligible effects on growth when compared with the more dramatic growth defects observed in deletion strains. We tested this assumption by comparing the doubling time of the previously mentioned deletion strains and the corresponding GFP fusion strains against their parent strain, BY4742. All measurements were repeated in triplicate on rich glucose media with the most frequently observed doubling time, the population mode, of each replicate compared to the parent strain using the Student’s <em>t</em>-test. ODELAY was able to resolve 12 GFP fusions with doubling times significantly decreased compared to BY4742 and 71 strains that have significantly increased doubling times (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The deletion strains group had 11 strains with significantly decreased doubling times while 72 had significantly increased doubling times (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). While the majority of the doubling time differences for the GFP strains were &lt;5 min, the presence of the GFP tag does appear to have a widespread and significant impact on growth rates on rich media.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470187/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470187/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">GFP-tagged library mutants with significant growth difference over a control strain</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470195/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470195/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Deletion library strains with significant difference in doubling time over control strain</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Comparison of ODELAY to other phenotypic analysis methods</h3><p id="p-32">ODELAY differs from colony pinning assays and liquid culture assays primarily in its ability to observe heterogeneity in lag time, doubling time, and carrying capacity of colonies forming from individual cells. A summary of features of various growth assays are described (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1/TableS2.pdf">Table S2</a>). While ODELAY is similar to previously published methods that observe colonies forming from single cells in liquid media (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Levy <em>et al.</em> 2012</a>), ODELAY is distinguished from these techniques by observing larger areas with image stitching and improved analysis of the resulting growth curves. In its current configuration, ODELAY can resolve individual cells with a pixel resolution of 0.65 μm and can evaluate 96 strains per assay. For each strain, up to 10<sup>3</sup> individual cells are observed growing into colonies with between 10<sup>5</sup> and 10<sup>6</sup> colonies observed per experiment. This feature allows clear observation of population heterogeneity, to which most other methods are insensitive, and allows new avenues for characterizing variations cellular growth phenotypes, even for well-studied organisms such as baker’s yeast.</p><p id="p-33">Periodically imaged colony pinning assays, such as ultrahigh-density omics (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bean <em>et al.</em> 2014</a>), Scan-o-matic (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Zackrisson <em>et al.</em> 2016</a>), and ScanLag (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Levin-Reisman <em>et al.</em> 2010</a>), have very high throughput in terms observing from 96 to 6144 colonies per plate, with 1536 colonies being typical (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1/TableS2.pdf">Table S2</a>). Each colony has the potential to be a unique strain, allowing complete screening of yeast libraries within a short period of time. However, arraying colonies at higher densities limits the total observation time from when a colony can be clearly resolved by the imaging methods to when the colonies merge (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bean <em>et al.</em> 2014</a>). Herein lies the tradeoff between ODELAY and pinned assays; ODELAY may observe heterogeneity at the cost of total number of strains observed. Though it should be noted that, as with colony pinning assays, ODLEAY’s strain throughput can be increased with improved tooling.</p><p id="p-34">Similar to other growth assays, there are caveats associated with ODELAY. Extraction of cell doubling time by ODELAY relies on the assumption that microcolony cross-sectional area is directly proportional to the volume of cells in a given colony and that this relationship between volume and area is unaffected by changes in growth condition and/or genetic background. There will certainly be exceptions to this assumption in yeast and other colony forming micro-organisms; however, similar to the limitations in liquid culture OD<sub>600</sub> analysis when applied to flocculent mutant strains, such exceptions may yield informative phenotypic information. Furthermore, ODELAY could be adapted to analyze the 3D volume of the growing microcolonies; however, this would trade off time for collecting images or limit the total area interrogated. Lag time measurements were also observed to have local variations, which are also commonly observed in other solid phase growth assays (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Baryshnikova <em>et al.</em> 2010</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Levin-Reisman <em>et al.</em> 2010</a>). Although all initial experiments have utilized haploid baker’s yeast, this methodology can be applied in other colony-forming organisms including medically relevant bacteria such as <em>Mycobacterium tuberculosis</em>, <em>Pseudomonas aeruginosa</em>, <em>Staphylococcus aureus</em>, and others.</p><p id="p-35">In summary, ODELAY is a quantitative tool capable of multiparameter growth analysis based on time resolved microcolony expansion on solid media. The unique features of ODELAY include its relatively large dynamic range, when compared to other available methods, which enables quantitative measurement of doubling time, lag time, and carrying capacity in a single experiment. Additionally, ODELAY has the ability to assess population heterogeneity, including viability, through the analysis of single microcolonies.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">We thank the Luxembourg Centre for Systems Biomedicine and the University of Luxembourg for support. In addition, we thank A. Blomberg and M. Zackrisson for sharing data for <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1/FigureS1.pdf">Figure S1</a>. The authors acknowledge support for this work by grants U54 RR022220, P41 GM109824, R01 GM112108 and P50 GM076547 to J.D.A. from the US National Institutes of Health. </p><p id="p-37">Author contributions: D.J.D. conceived the ODELAY method; ODELAY was developed by T.H., A.V.R., and D.J.D., with intellectual support from S.L., F.D.M., J.J.S., and J.D.A.; T.H. and S.L. performed all experiments; T.H. and D.J.D. wrote the manuscript with contributions from A.V.R., F.D.M., J.J.S., and J.D.A.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-38">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037044/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-39"><em>Communicating editor: C. Nislow</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-08-22"><span class="received-label">Received </span>August 22, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-10"><span class="accepted-label">Accepted </span>November 10, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Herricks <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.1" data-doi="10.1534/genetics.111.127803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitchison</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rout</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The yeast nuclear pore complex and transport through it.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAitchison%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D190%26rft.issue%253D3%26rft.spage%253D855%26rft.epage%253D883%26rft.atitle%253DThe%2BYeast%2BNuclear%2BPore%2BComplex%2Band%2BTransport%2BThrough%2BIt%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127803%26rft_id%253Dinfo%253Apmid%252F22419078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzMvODU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzI3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.2" data-doi="10.1038/nmeth.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBaryshnikova%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1017%26rft.epage%253D1024%26rft.atitle%253DQuantitative%2Banalysis%2Bof%2Bfitness%2Band%2Bgenetic%2Binteractions%2Bin%2Byeast%2Bon%2Ba%2Bgenome%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1534%26rft_id%253Dinfo%253Apmid%252F21076421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076421&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284686300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.3" data-doi="10.1371/journal.pone.0085177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaeger</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahr</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ideker</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of ultra-high-density screening tools for microbial “omics.”</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e85177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85177%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085177%26rft_id%253Dinfo%253Apmid%252F24465499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24465499&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.4" data-doi="10.1038/nmeth.1234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breslow</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart-Ornstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBreslow%26rft.auinit1%253DD.%2BK.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D711%26rft.epage%253D718%26rft.atitle%253DA%2Bcomprehensive%2Bstrategy%2Benabling%2Bhigh-resolution%2Bfunctional%2Banalysis%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1234%26rft_id%253Dinfo%253Apmid%252F18622397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18622397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258077400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.5" data-doi="10.1073/pnas.0901851107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goranov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manalis</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Measurement of mass, density, and volume during the cell cycle of yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">999</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0901851107%26rft_id%253Dinfo%253Apmid%252F20080562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMy85OTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.6" data-doi="10.1186/gb-2006-7-7-r63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253DR63%26rft.epage%253DR63%26rft.atitle%253DA%2Bstrategy%2Bfor%2Bextracting%2Band%2Banalyzing%2Blarge-scale%2Bquantitative%2Bepistatic%2Binteraction%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-7-r63%26rft_id%253Dinfo%253Apmid%252F16859555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-7-r63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16859555&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.7" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanderSluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pons</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A global genetic interaction network maps a wiring diagram of cellular function.</span> <abbr class="cit-jnl-abbrev">Science</abbr> DOI: 10.1126/science.aaf1420.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.9" data-doi="10.1534/genetics.104.036871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036871%26rft_id%253Dinfo%253Apmid%252F15716499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5MTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.10" data-doi="10.14440/jbm.2014.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edelstein</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchida</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amodaj</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinkard</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vale</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Advanced methods of microscope control using μManager software.</span> <abbr class="cit-jnl-abbrev">J. Biol. Methods</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BMethods%26rft.volume%253D1%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14440%252Fjbm.2014.36%26rft_id%253Dinfo%253Apmid%252F25606571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14440/jbm.2014.36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25606571&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.11" data-doi="10.1038/nmeth.1452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Godin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delgado</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Son</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grover</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Using buoyant mass to measure the growth of single cells.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGodin%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D387%26rft.epage%253D390%26rft.atitle%253DUsing%2Bbuoyant%2Bmass%2Bto%2Bmeasure%2Bthe%2Bgrowth%2Bof%2Bsingle%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1452%26rft_id%253Dinfo%253Apmid%252F20383132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20383132&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277175100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.12" data-doi="10.1098/rstl.1825.0026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gompertz</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1825</span> <span class="cit-article-title">On the nature of the function expressive of the law of human mortality, and on a new mode of determining the value of life contingencies.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%26rft.volume%253D115%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstl.1825.0026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstl.1825.0026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.13" data-doi="10.1073/pnas.79.23.7410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guarente</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yocum</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">A GAL10–CYC1 hybrid yeast promoter identifies the GAL4 regulatory region as an upstream site.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">7410</span>–<span class="cit-lpage">7414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGuarente%26rft.auinit1%253DL.%26rft.volume%253D79%26rft.issue%253D23%26rft.spage%253D7410%26rft.epage%253D7414%26rft.atitle%253DA%2BGAL10-CYC1%2Bhybrid%2Byeast%2Bpromoter%2Bidentifies%2Bthe%2BGAL4%2Bregulatory%2Bregion%2Bas%2Ban%2Bupstream%2Bsite.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.23.7410%26rft_id%253Dinfo%253Apmid%252F6760197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMjMvNzQxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jørgensen</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Turbidimetric measurement as a rapid method for the determination of the bacteriological quality of minced meat.</span> <abbr class="cit-jnl-abbrev">Int. J. Food Microbiol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BFood%2BMicrobiol.%26rft.volume%253D2%26rft.spage%253D177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.15" data-doi="10.1038/msb.2011.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knijnenburg</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roda</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitchison</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A regression model approach to enable cell morphology correction in high-throughput flow cytometry.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DKnijnenburg%26rft.auinit1%253DT.%2BA.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D531%26rft.epage%253D531%26rft.atitle%253DA%2Bregression%2Bmodel%2Bapproach%2Bto%2Benable%2Bcell%2Bmorphology%2Bcorrection%2Bin%2Bhigh-throughput%2Bflow%2Bcytometry%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.64%26rft_id%253Dinfo%253Apmid%252F21952134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kortmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blank</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single cell analysis reveals unexpected growth phenotype of S. cerevisiae.</span> <abbr class="cit-jnl-abbrev">Cytom. Part J. Int. Soc. Anal. Cytol.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytom.%2BPart%2BJ.%2BInt.%2BSoc.%2BAnal.%2BCytol.%26rft.volume%253D75%26rft.spage%253D130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.17" data-doi="10.1186/1471-2105-11-287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLawless%26rft.auinit1%253DC.%26rft.volume%253D11%26rft.spage%253D287%26rft.epage%253D287%26rft.atitle%253DColonyzer%253A%2Bautomated%2Bquantification%2Bof%2Bmicro-organism%2Bgrowth%2Bcharacteristics%2Bon%2Bsolid%2Bagar.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-287%26rft_id%253Dinfo%253Apmid%252F20509870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20509870&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.18" data-doi="10.1038/nmeth.1485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin-Reisman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gefen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fridman</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shwa</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Automated imaging with ScanLag reveals previously undetectable bacterial growth phenotypes.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLevin-Reisman%26rft.auinit1%253DI.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D737%26rft.epage%253D739%26rft.atitle%253DAutomated%2Bimaging%2Bwith%2BScanLag%2Breveals%2Bpreviously%2Bundetectable%2Bbacterial%2Bgrowth%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1485%26rft_id%253Dinfo%253Apmid%252F20676109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20676109&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281429200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.19" data-doi="10.1371/journal.pbio.1001325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLevy%26rft.auinit1%253DS.%2BF.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253De1001325%26rft.epage%253De1001325%26rft.atitle%253DBet%2Bhedging%2Bin%2Byeast%2Bby%2Bheterogeneous%252C%2Bage-correlated%2Bexpression%2Bof%2Ba%2Bstress%2Bprotectant.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001325%26rft_id%253Dinfo%253Apmid%252F22589700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589700&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.20" data-doi="10.1186/1471-2105-8-117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Memarian</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jessulat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alirezaie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mir-Rashed</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Colony size measurement of the yeast gene deletion strains for functional genomics.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMemarian%26rft.auinit1%253DN.%26rft.volume%253D8%26rft.spage%253D117%26rft.epage%253D117%26rft.atitle%253DColony%2Bsize%2Bmeasurement%2Bof%2Bthe%2Byeast%2Bgene%2Bdeletion%2Bstrains%2Bfor%2Bfunctional%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-117%26rft_id%253Dinfo%253Apmid%252F17408490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17408490&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.21" data-doi="10.1146/annurev.mi.03.100149.002103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monod</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The growth of bacterial cultures.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DMonod%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.mi.03.100149.002103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.mi.03.100149.002103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1949XS94800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeberlein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantifying yeast chronological life span by outgrowth of aged cells.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">27</span>: e1156.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.23" data-doi="10.1016/j.patcog.2012.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertuz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis of focus measure operators for shape-from-focus.</span> <abbr class="cit-jnl-abbrev">Pattern Recognit.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1415</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPattern%2BRecognit.%26rft.volume%253D46%26rft.spage%253D1415%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.patcog.2012.11.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.patcog.2012.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.24" data-doi="10.1042/bj0210460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peskett</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1927</span> <span class="cit-article-title">Studies on the growth of yeast: a nephelometric method of counting yeast suspensions.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">460</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DPeskett%26rft.auinit1%253DG.%2BL.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D460%26rft.epage%253D466%26rft.atitle%253DStudies%2Bon%2Bthe%2BGrowth%2Bof%2BYeast%253A%2BA%2BNephelometric%2BMethod%2Bof%2BCounting%2BYeast%2BSuspensions%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj0210460%26rft_id%253Dinfo%253Apmid%252F16743859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czoxMDoicHBiaW9jaGVtaiI7czo1OiJyZXNpZCI7czo4OiIyMS8zLzQ2MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.25" data-doi="10.1093/bioinformatics/btp184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preibisch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saalfeld</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomancak</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Globally optimal stitching of tiled 3D microscopic image acquisitions.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1463</span>–<span class="cit-lpage">1465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp184%26rft_id%253Dinfo%253Apmid%252F19346324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQ2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.26" data-doi="10.1128/MCB.16.6.3206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">SPT20/ADA5 encodes a novel protein functionally related to the TATA-binding protein and important for transcription in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3206</span>–<span class="cit-lpage">3213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DRoberts%26rft.auinit1%253DS.%2BM.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D3206%26rft.epage%253D3213%26rft.atitle%253DSPT20%252FADA5%2Bencodes%2Ba%2Bnovel%2Bprotein%2Bfunctionally%2Brelated%2Bto%2Bthe%2BTATA-%2Bbinding%2Bprotein%2Band%2Bimportant%2Bfor%2Btranscription%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.6.3206%26rft_id%253Dinfo%253Apmid%252F8649431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE2LzYvMzIwNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.27" data-doi="10.1186/1752-0509-1-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laws</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wardman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Accurate, precise modeling of cell proliferation kinetics from time-lapse imaging and automated image analysis of agar yeast culture arrays.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DShah%26rft.auinit1%253DN.%2BA.%26rft.volume%253D1%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DAccurate%252C%2Bprecise%2Bmodeling%2Bof%2Bcell%2Bproliferation%2Bkinetics%2Bfrom%2Btime-lapse%2Bimaging%2Band%2Bautomated%2Bimage%2Banalysis%2Bof%2Bagar%2Byeast%2Bculture%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-1-3%26rft_id%253Dinfo%253Apmid%252F17408510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-1-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17408510&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.28" data-doi="10.1039/c005029f"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stowers</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boczko</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Measurement of the volume growth rate of single budding yeast with the MOSFET-based microfluidic Coulter counter.</span> <abbr class="cit-jnl-abbrev">Lab Chip</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2986</span>–<span class="cit-lpage">2993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLab%2Bon%2Ba%2Bchip%26rft.stitle%253DLab%2BChip%26rft.aulast%253DSun%26rft.auinit1%253DJ.%26rft.volume%253D10%26rft.issue%253D21%26rft.spage%253D2986%26rft.epage%253D2993%26rft.atitle%253DMeasurement%2Bof%2Bthe%2Bvolume%2Bgrowth%2Brate%2Bof%2Bsingle%2Bbudding%2Byeast%2Bwith%2Bthe%2BMOSFET-based%2Bmicrofluidic%2BCoulter%2Bcounter.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc005029f%26rft_id%253Dinfo%253Apmid%252F20717618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/c005029f&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20717618&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.29" data-doi="10.1002/cfg.391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative genome-wide analysis of yeast deletion strain sensitivities to oxidative and chemical stress.</span> <abbr class="cit-jnl-abbrev">Comp. Funct. Genomics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComp%2BFunct%2BGenomics%26rft.aulast%253DTucker%26rft.auinit1%253DC.%2BL.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D216%26rft.epage%253D224%26rft.atitle%253DQuantitative%2Bgenome-wide%2Banalysis%2Bof%2Byeast%2Bdeletion%2Bstrain%2Bsensitivities%2Bto%2Boxidative%2Band%2Bchemical%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcfg.391%26rft_id%253Dinfo%253Apmid%252F18629161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cfg.391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18629161&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.30" data-doi="10.1002/wsbm.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Vosse</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wozniak</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitchison</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Role of the nuclear envelope in genome organization and gene expression.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. Syst. Biol. Med.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DWiley%2BInterdiscip%2BRev%2BSyst%2BBiol%2BMed%26rft.aulast%253DVan%2Bde%2BVosse%26rft.auinit1%253DD.%2BW.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D147%26rft.epage%253D166%26rft.atitle%253DRole%2Bof%2Bthe%2Bnuclear%2Benvelope%2Bin%2Bgenome%2Borganization%2Band%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwsbm.101%26rft_id%253Dinfo%253Apmid%252F21305702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wsbm.101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21305702&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.31" data-doi="10.1186/1472-6769-8-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anevski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chemogenetic fingerprinting by analysis of cellular growth dynamics.</span> <abbr class="cit-jnl-abbrev">BMC Chem. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BChem%2BBiol%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DChemogenetic%2Bfingerprinting%2Bby%2Banalysis%2Bof%2Bcellular%2Bgrowth%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6769-8-3%26rft_id%253Dinfo%253Apmid%252F18721464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1472-6769-8-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18721464&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.32" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.33" data-doi="10.1111/j.1567-1364.2008.00456.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshikawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furusawa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagahisa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirasawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive phenotypic analysis for identification of genes affecting growth under ethanol stress in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00456.x%26rft_id%253Dinfo%253Apmid%252F19054128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjY6IjkvMS8zMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.34" data-doi="10.1534/g3.116.032342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zackrisson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ottosson</span>  <span class="cit-name-given-names">L.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Parada</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Scan-o-matic: high-resolution microbial phenomics at a massive scale.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3003</span>–<span class="cit-lpage">3014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032342%26rft_id%253Dinfo%253Apmid%252F27371952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8zMDAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzI3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.279.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zwietering</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jongenburger</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rombouts</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van ’t Riet</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Modeling of the bacterial growth curve.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1875</span>–<span class="cit-lpage">1881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DZwietering%26rft.auinit1%253DM.%2BH.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D1875%26rft.epage%253D1881%26rft.atitle%253DModeling%2Bof%2Bthe%2BBacterial%2BGrowth%2BCurve%26rft_id%253Dinfo%253Apmid%252F16348228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjU2LzYvMTg3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/279.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/269" title="First in Vivo Batrachochytrium dendrobatidis Transcriptomes Reveal Mechanisms of Host Exploitation, Host-Specific Gene Expression, and Expressed Genotype Shifts" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/289" title="Correlations of Genotype with Climate Parameters Suggest Caenorhabditis elegans Niche Adaptations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/279.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cZ5FsIvTbsoXReoM6qXj70gx4XVZ9-ABE5fQXhogQp4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/279" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/279" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/279" class="active" data-icon-position="" data-hide-link-title="0">One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460544" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1zZPUouoQeGXWEUoGw-8AhCytB0vproL-RZy4QT-V7U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/279.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/279" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-V53hsXfiOp5bUBySTZNeWbnmTsRs5mBVOAjFyuYmW8E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/279" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/279.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460544" id="node460544--2" data-pisa="ggg;7/1/279" data-pisa-master="ggg;g3.116.037044" data-apath="/ggg/7/1/279.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thurston Herricks</span>, <span class="highwire-citation-author" data-delta="1">David J. Dilworth</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2177-6647" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fred D. Mast</span>, <span class="highwire-citation-author" data-delta="3">Song Li</span>, <span class="highwire-citation-author" data-delta="4">Jennifer J. Smith</span>, <span class="highwire-citation-author" data-delta="5">Alexander V. Ratushny</span> and <span class="highwire-citation-author" data-delta="6">John D. Aitchison</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">279-288;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037044 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460544--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thurston Herricks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thurston%2BHerricks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herricks%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThurston%2BHerricks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David J. Dilworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BDilworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilworth%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BDilworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred D. Mast </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BD.%2BMast%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mast%20FD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFred%2BD.%2BMast%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2177-6647" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fred D. Mast</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Song Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Song%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jennifer J. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BJ.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BJ.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alexander V. Ratushny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BV.%2BRatushny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratushny%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BV.%2BRatushny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">John D. Aitchison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BD.%2BAitchison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aitchison%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BD.%2BAitchison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john.aitchison@systemsbiology.org" class="" data-icon-position="" data-hide-link-title="0">john.aitchison@systemsbiology.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/279" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/279&amp;title=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/279&amp;title=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/279&amp;title=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/279&amp;text=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/279&amp;title=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/279&amp;t=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/279&amp;title=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/279&amp;title=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460544/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460544" id="node460544--3" data-pisa="ggg;7/1/279" data-pisa-master="ggg;g3.116.037044" data-apath="/ggg/7/1/279.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">One-Cell Doubling Evaluation by Living Arrays of Yeast, ODELAY!</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thurston Herricks</span>, <span class="highwire-citation-author" data-delta="1">David J. Dilworth</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2177-6647" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fred D. Mast</span>, <span class="highwire-citation-author" data-delta="3">Song Li</span>, <span class="highwire-citation-author" data-delta="4">Jennifer J. Smith</span>, <span class="highwire-citation-author" data-delta="5">Alexander V. Ratushny</span> and <span class="highwire-citation-author" data-delta="6">John D. Aitchison</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">279-288;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037044 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460544--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thurston Herricks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thurston%2BHerricks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herricks%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThurston%2BHerricks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David J. Dilworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BDilworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilworth%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BDilworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred D. Mast </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BD.%2BMast%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mast%20FD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFred%2BD.%2BMast%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2177-6647" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fred D. Mast</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Song Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Song%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jennifer J. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BJ.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BJ.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alexander V. Ratushny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BV.%2BRatushny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratushny%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BV.%2BRatushny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">John D. Aitchison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Infectious Disease Research, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BD.%2BAitchison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aitchison%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BD.%2BAitchison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john.aitchison@systemsbiology.org" class="" data-icon-position="" data-hide-link-title="0">john.aitchison@systemsbiology.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=279&amp;rft.epage=288&amp;rft.atitle=One-Cell%20Doubling%20Evaluation%20by%20Living%20Arrays%20of%20Yeast%2C%20ODELAY%21&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Herricks%2C+Thurston&amp;rft.au=Dilworth%2C+David+J.&amp;rft.au=Mast%2C+Fred+D.&amp;rft.au=Li%2C+Song&amp;rft.au=Smith%2C+Jennifer+J.&amp;rft.au=Ratushny%2C+Alexander+V.&amp;rft.au=Aitchison%2C+John+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460544/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460544/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460544/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460544/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460544/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460544/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460544/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460544/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460544/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460544/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460544/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460544/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/279" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/279.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/279.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/1/279.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/1/279.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
