<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/143" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/143.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/143.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/1/143.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/1/143/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/143.atom" />
<meta name="HW.pisa" content="ggg;7/1/143" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.116.034207" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Jaffe et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Systematic screens for genetic interactions are a cornerstone of both network and systems biology. However, most screens have been limited to characterizing interaction networks in a single environment. Moving beyond this static view of the cell requires a major technological advance to increase the throughput and ease of replication in these assays. Here, we introduce iSeq—a platform to build large double barcode libraries and rapidly assay genetic interactions across environments. We use iSeq in yeast to measure fitness in three conditions of nearly 400 clonal strains, representing 45 possible single or double gene deletions, including multiple replicate strains per genotype. We show that iSeq fitness and interaction scores are highly reproducible for the same clonal strain across replicate cultures. However, consistent with previous work, we find that replicates with the same putative genotype have highly variable genetic interaction scores. By whole-genome sequencing 102 of our strains, we find that segregating variation and de novo mutations, including aneuploidy, occur frequently during strain construction, and can have large effects on genetic interaction scores. Additionally, we uncover several new environment-dependent genetic interactions, suggesting that barcode-based genetic interaction assays have the potential to significantly expand our knowledge of genetic interaction networks." />
<meta name="DC.Contributor" content="Mia Jaffe" />
<meta name="DC.Contributor" content="Gavin Sherlock" />
<meta name="DC.Contributor" content="Sasha F. Levy" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Systematic screens for genetic interactions are a cornerstone of both network and systems biology. However, most screens have been limited to characterizing interaction networks in a single environment. Moving beyond this static view of the cell requires a major technological advance to increase the throughput and ease of replication in these assays. Here, we introduce iSeq—a platform to build large double barcode libraries and rapidly assay genetic interactions across environments. We use iSeq in yeast to measure fitness in three conditions of nearly 400 clonal strains, representing 45 possible single or double gene deletions, including multiple replicate strains per genotype. We show that iSeq fitness and interaction scores are highly reproducible for the same clonal strain across replicate cultures. However, consistent with previous work, we find that replicates with the same putative genotype have highly variable genetic interaction scores. By whole-genome sequencing 102 of our strains, we find that segregating variation and &lt;i&gt;de novo&lt;/i&gt; mutations, including aneuploidy, occur frequently during strain construction, and can have large effects on genetic interaction scores. Additionally, we uncover several new environment-dependent genetic interactions, suggesting that barcode-based genetic interaction assays have the potential to significantly expand our knowledge of genetic interaction networks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/143" />
<meta name="citation_id" content="7/1/143" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/143" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/143.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/143.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/143.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.034207" />
<meta name="citation_pmid" content="27821633" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="143" />
<meta name="citation_lastpage" content="153" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mia Jaffe" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford School of Medicine, California 94305" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0836-2458" />
<meta name="citation_author" content="Gavin Sherlock" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford School of Medicine, California 94305" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1692-4983" />
<meta name="citation_author" content="Sasha F. Levy" />
<meta name="citation_author_institution" content="The Laufer Center for Physical and Quantitative Biology, Stony Brook University, New York 11794" />
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Cell Biology, Stony Brook University, New York 11794" />
<meta name="citation_author_email" content="sasha.levy@stonybrook.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0923-1636" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=R. Albert;citation_title=Scale-free networks in cell biology;citation_pages=4947-4957;citation_volume=118;citation_year=2005;citation_issue=21;citation_pmid=16254242;citation_doi=10.1242/jcs.02714" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Bandyopadhyay;citation_author=M. Mehta;citation_author=D. Kuo;citation_author=M.-K. Sung;citation_author=R. Chuang;citation_title=Rewiring of Genetic Networks in Response to DNA Damage;citation_pages=1385-1389;citation_volume=330;citation_year=2010;citation_issue=6009;citation_pmid=21127252;citation_doi=10.1126/science.1195618" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=Y. Kim;citation_author=H. Ding;citation_author=J. Koh;citation_title=Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.;citation_pages=1017-1024;citation_volume=7;citation_year=2010;citation_issue=12;citation_pmid=21076421;citation_doi=10.1038/nmeth.1534" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. Bender;citation_author=J. R. Pringle;citation_title=Use of a screen for synthetic lethal and multicopy suppressee mutants to identify two new genes involved in morphogenesis in Saccharomyces cerevisiae.;citation_pages=1295-1305;citation_volume=11;citation_year=1991;citation_issue=3;citation_pmid=1996092;citation_doi=10.1128/MCB.11.3.1295" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. B. Brachmann;citation_author=A. Davies;citation_author=G. J. Cost;citation_author=E. Caputo;citation_author=J. Li;citation_title=Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.;citation_pages=115-132;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9483801;citation_doi=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. K. Breslow;citation_author=D. M. Cameron;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=J. Stewart-Ornstein;citation_title=A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.;citation_pages=711-718;citation_volume=5;citation_year=2008;citation_issue=8;citation_pmid=18622397;citation_doi=10.1038/nmeth.1234" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=N. J. Krogan;citation_author=J. S. Weissman;citation_title=A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.;citation_pages=R63-R63;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16859555;citation_doi=10.1186/gb-2006-7-7-r63" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. R. Collins;citation_author=K. M. Miller;citation_author=N. L. Maas;citation_author=A. Roguev;citation_author=J. Fillingham;citation_title=Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map.;citation_pages=806-810;citation_volume=446;citation_year=2007;citation_issue=7137;citation_pmid=17314980;citation_doi=10.1038/nature05649" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=B. VanderSluis;citation_author=E. N. Koch;citation_author=A. Baryshnikova;citation_author=C. Pons;citation_title=A global genetic interaction network maps a wiring diagram of cellular function;citation_pages=aaf1420-aaf1420;citation_volume=353;citation_year=2016;citation_issue=6306;citation_pmid=27708008;citation_doi=10.1126/science.aaf1420" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=M. M. Darby;citation_author=L. Serebreni;citation_author=X. Pan;citation_author=J. D. Boeke;citation_author=J. L. Corden;citation_title=The Saccharomyces cerevisiae Nrd1-Nab3 transcription termination pathway acts in opposition to Ras signaling and mediates response to nutrient depletion.;citation_volume=32;citation_year=2012;citation_pmid=22431520;citation_doi=10.1128/MCB.00050-12" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. P. Davierwala;citation_author=J. Haynes;citation_author=Z. Li;citation_author=R. L. Brost;citation_author=M. D. Robinson;citation_title=The synthetic genetic interaction spectrum of essential genes.;citation_pages=1147-1152;citation_volume=37;citation_year=2005;citation_issue=10;citation_pmid=16155567;citation_doi=10.1038/ng1640" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Decourty;citation_author=C. Saveanu;citation_author=K. Zemam;citation_author=F. Hantraye;citation_author=E. Frachon;citation_title=Linking functionally related genes by sensitive and quantitative characterization of genetic interaction profiles.;citation_volume=105;citation_year=2008;citation_pmid=18408161;citation_doi=10.1073/pnas.0710533105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. E. Dodgson;citation_author=S. Kim;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=D. L. Morse;citation_title=Chromosome-specific and global effects of aneuploidy in Saccharomyces cerevisiae.;citation_volume=202;citation_year=2016;citation_pmid=26837754;citation_doi=10.1534/genetics.115.185660" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. Dutkowski;citation_author=M. Kramer;citation_author=M. A. Surma;citation_author=R. Balakrishnan;citation_author=J. M. Cherry;citation_title=A gene ontology inferred from molecular networks.;citation_pages=38-45;citation_volume=31;citation_year=2013;citation_pmid=23242164;citation_doi=10.1038/nbt.2463" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=L. A. Gilbert;citation_author=M. H. Larson;citation_author=L. Morsut;citation_author=Z. Liu;citation_author=G. A. Brar;citation_title=CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes.;citation_pages=442-451;citation_volume=154;citation_year=2013;citation_pmid=23849981;citation_doi=10.1016/j.cell.2013.06.044" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gresham;citation_author=V. M. Boer;citation_author=A. Caudy;citation_author=N. Ziv;citation_author=N. J. Brandt;citation_title=System-level analysis of genes and functions affecting survival during nutrient starvation in Saccharomyces cerevisiae.;citation_volume=187;citation_year=2011;citation_pmid=20944018;citation_doi=10.1534/genetics.110.120766" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=A. Guénolé;citation_author=R. Srivas;citation_author=K. Vreeken;citation_author=Z. Z. Wang;citation_author=S. Wang;citation_title=Dissection of DNA damage responses using multiconditional genetic interaction maps.;citation_pages=346-358;citation_volume=49;citation_year=2013;citation_pmid=23273983;citation_doi=10.1016/j.molcel.2012.11.023" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Harrison;citation_author=B. Papp;citation_author=C. Pál;citation_author=S. G. Oliver;citation_author=D. Delneri;citation_title=Plasticity of genetic interactions in metabolic networks of yeast.;citation_volume=104;citation_year=2007;citation_pmid=17284612;citation_doi=10.1073/pnas.0607153104" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=P. K. Herman;citation_author=J. Rine;citation_title=Yeast spore germination: a requirement for Ras protein activity during re-entry into the cell cycle;citation_pages=6171-6181;citation_volume=16;citation_year=1997;citation_issue=20;citation_pmid=9321396;citation_doi=10.1093/emboj/16.20.6171" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Z. Huang;citation_author=S. Srinivasan;citation_author=J. Zhang;citation_author=K. Chen;citation_author=Y. Li;citation_title=Discovering thiamine transporters as targets of chloroquine using a novel functional genomics strategy.;citation_pages=e1003083-e1003083;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23209439;citation_doi=10.1371/journal.pgen.1003083" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. R. Hughes;citation_author=C. J. Roberts;citation_author=H. Dai;citation_author=A. R. Jones;citation_author=M. R. Meyer;citation_title=Widespread aneuploidy revealed by DNA microarray expression profiling.;citation_pages=333-337;citation_volume=25;citation_year=2000;citation_issue=3;citation_pmid=10888885;citation_doi=10.1038/77116" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. Jasnos;citation_author=R. Korona;citation_title=Epistatic buffering of fitness loss in yeast double deletion strains.;citation_pages=550-554;citation_volume=39;citation_year=2007;citation_issue=4;citation_pmid=17322879;citation_doi=10.1038/ng1986" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. C. Kao;citation_author=G. Sherlock;citation_title=Molecular characterization of clonal interference during adaptive evolution in asexual populations of Saccharomyces cerevisiae.;citation_pages=1499-1504;citation_volume=40;citation_year=2008;citation_issue=12;citation_pmid=19029899;citation_doi=10.1038/ng.280" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=R. Kelley;citation_author=T. Ideker;citation_title=Systematic interpretation of genetic interactions using protein networks.;citation_pages=561-566;citation_volume=23;citation_year=2005;citation_issue=5;citation_pmid=15877074;citation_doi=10.1038/nbt1096" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kryazhimskiy;citation_author=D. P. Rice;citation_author=E. R. Jerison;citation_author=M. M. Desai;citation_title=Global epistasis makes adaptation predictable despite sequence-level stochasticity;citation_pages=1519-1522;citation_volume=344;citation_year=2014;citation_issue=6191;citation_pmid=24970088;citation_doi=10.1126/science.1250939" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=P. Kumar;citation_author=S. Henikoff;citation_author=P. C. Ng;citation_title=Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.;citation_pages=1073-1081;citation_volume=4;citation_year=2009;citation_issue=7;citation_pmid=19561590;citation_doi=10.1038/nprot.2009.86" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Lehner;citation_author=C. Crombie;citation_author=J. Tischler;citation_author=A. Fortunato;citation_author=A. G. Fraser;citation_title=Systematic mapping of genetic interactions in Caenorhabditis elegans identifies common modifiers of diverse signaling pathways.;citation_pages=896-903;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16845399;citation_doi=10.1038/ng1844" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. F. Levy;citation_author=J. R. Blundell;citation_author=S. Venkataram;citation_author=D. A. Petrov;citation_author=D. S. Fisher;citation_title=Quantitative evolutionary dynamics using high-resolution lineage tracking.;citation_pages=181-186;citation_volume=519;citation_year=2015;citation_pmid=25731169;citation_doi=10.1038/nature14279" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Q.-R. Li;citation_author=A.-R. Carvunis;citation_author=H. Yu;citation_author=J.-D. J. Han;citation_author=Q. Zhong;citation_title=Revisiting the Saccharomyces cerevisiae predicted ORFeome.;citation_volume=18;citation_year=2008;citation_pmid=18502943;citation_doi=10.1101/gr.076661.108" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=G. Liu;citation_author=M. Y. J. Yong;citation_author=M. Yurieva;citation_author=K. G. Srinivasan;citation_author=J. Liu;citation_title=Gene essentiality is a quantitative property linked to cellular evolvability.;citation_pages=1388-1399;citation_volume=163;citation_year=2015;citation_pmid=26627736;citation_doi=10.1016/j.cell.2015.10.069" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mani;citation_author=R. P. St Onge;citation_author=J. L. Hartman;citation_author=G. Giaever;citation_author=F. P. Roth;citation_title=Defining genetic interaction.;citation_volume=105;citation_year=2008;citation_pmid=18305163;citation_doi=10.1073/pnas.0712255105" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_title=CRISPR/Cas9: a molecular Swiss army knife for simultaneous introduction of multiple genetic modifications in Saccharomyces cerevisiae.;citation_author=R. Mans;citation_author=H. M. van Rossum;citation_author=M. Wijsman;citation_author=A. Backx;citation_author=N. G. A. Kuijpers;citation_volume=15;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Margulies;citation_author=M. Egholm;citation_author=W. E. Altman;citation_author=S. Attiya;citation_author=J. S. Bader;citation_title=Genome sequencing in microfabricated high-density picolitre reactors.;citation_pages=376-380;citation_volume=437;citation_year=2005;citation_issue=7057;citation_pmid=16056220;citation_doi=10.1038/nature03959" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=H. Martin;citation_author=M. Shales;citation_author=P. Fernandez-Piñar;citation_author=P. Wei;citation_author=M. Molina;citation_title=Differential genetic interactions of yeast stress response MAPK pathways;citation_pages=800-800;citation_volume=11;citation_year=2015;citation_issue=4;citation_pmid=25888283;citation_doi=10.15252/msb.20145606" />
<meta name="citation_reference" content="citation_journal_title=EMBnet.journal;citation_author=M. Martin;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads.;citation_pages=10;citation_volume=17;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Moriya;citation_author=Y. Shimizu-Yoshida;citation_author=H. Kitano;citation_title=In vivo robustness analysis of cell division cycle genes in Saccharomyces cerevisiae.;citation_pages=e111-e111;citation_volume=2;citation_year=2006;citation_issue=7;citation_pmid=16839182;citation_doi=10.1371/journal.pgen.0020111" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Musso;citation_author=M. Costanzo;citation_author=M. Huangfu;citation_author=A. M. Smith;citation_author=J. Paw;citation_title=The extensive and condition-dependent nature of epistasis among whole-genome duplicates in yeast.;citation_volume=18;citation_year=2008;citation_pmid=18463300;citation_doi=10.1101/gr.076174.108" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. L. Ooi;citation_author=D. D. Shoemaker;citation_author=J. D. Boeke;citation_title=DNA helicase gene interaction network defined using synthetic lethality analyzed by microarray.;citation_pages=277-286;citation_volume=35;citation_year=2003;citation_issue=3;citation_pmid=14566339;citation_doi=10.1038/ng1258" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=X. Pan;citation_author=D. S. Yuan;citation_author=D. Xiang;citation_author=X. Wang;citation_author=S. Sookhai-Mahadeo;citation_title=A robust toolkit for functional profiling of the yeast genome.;citation_pages=487-496;citation_volume=16;citation_year=2004;citation_issue=3;citation_pmid=15525520;citation_doi=10.1016/j.molcel.2004.09.035" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Pan;citation_author=P. Ye;citation_author=D. S. Yuan;citation_author=X. Wang;citation_author=J. S. Bader;citation_title=A DNA integrity network in the yeast Saccharomyces cerevisiae.;citation_pages=1069-1081;citation_volume=124;citation_year=2006;citation_issue=5;citation_pmid=16487579;citation_doi=10.1016/j.cell.2005.12.036" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Pavelka;citation_author=G. Rancati;citation_author=J. Zhu;citation_author=W. D. Bradford;citation_author=A. Saraf;citation_title=Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.;citation_pages=321-325;citation_volume=468;citation_year=2010;citation_issue=7321;citation_pmid=20962780;citation_doi=10.1038/nature09529" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=V. Pelechano;citation_author=W. Wei;citation_author=L. M. Steinmetz;citation_title=Extensive transcriptional heterogeneity revealed by isoform profiling.;citation_pages=127-131;citation_volume=497;citation_year=2013;citation_pmid=23615609;citation_doi=10.1038/nature12121" />
<meta name="citation_reference" content="citation_title=Epistasis and the Evolutionary Process;citation_title=Beyond the average. The evolutionary importance of gene interactions and variability of epistatic effects;citation_author=P. C. Phillips;citation_author=S. P. Otto;citation_author=M. C. Whitlock;citation_pages=20-38;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Rattray;citation_author=G. Santoyo;citation_author=B. Shafer;citation_author=J. N. Strathern;citation_title=Elevated mutation rate during meisosis in Saccharomyces cerevisiae.;citation_pages=e1004910;citation_volume=11;citation_year=2015;citation_pmid=25569256;citation_doi=10.1371/journal.pgen.1004910" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. G. Robinson;citation_author=W. Chen;citation_author=J. D. Storey;citation_author=D. Gresham;citation_title=Design and analysis of Bar-seq experiments.;citation_volume=4;citation_year=2014;citation_pmid=24192834;citation_doi=10.1534/g3.113.008565" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Roguev;citation_author=S. Bandyopadhyay;citation_author=M. Zofall;citation_author=K. Zhang;citation_author=T. Fischer;citation_title=Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast.;citation_volume=322;citation_year=2008;citation_pmid=18818364;citation_doi=10.1126/science.1162609" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Schuldiner;citation_author=S. R. Collins;citation_author=N. J. Thompson;citation_author=V. Denic;citation_author=A. Bhamidipati;citation_title=Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile.;citation_pages=507-519;citation_volume=123;citation_year=2005;citation_issue=3;citation_pmid=16269340;citation_doi=10.1016/j.cell.2005.08.031" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=M. Schuldiner;citation_author=S. R. Collins;citation_author=J. S. Weissman;citation_author=N. J. Krogan;citation_title=Quantitative genetic analysis in Saccharomyces cerevisiae using epistatic miniarray profiles (E-MAPs) and its application to chromatin functions.;citation_pages=344-352;citation_volume=40;citation_year=2006;citation_issue=4;citation_pmid=17101447;citation_doi=10.1016/j.ymeth.2006.07.034" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Segrè;citation_author=A. Deluna;citation_author=G. M. Church;citation_author=R. Kishony;citation_title=Modular epistasis in yeast metabolism.;citation_pages=77-83;citation_volume=37;citation_year=2005;citation_issue=1;citation_pmid=15592468;citation_doi=10.1038/nm1161" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=A rapid and tunable method to temporally control Cas9 expression enables the identification of essential genes and the interrogation of functional gene interactions in vitro and in vivo.;citation_author=S. Senturk;citation_author=N. H. Shirole;citation_author=D. D. Nowak;citation_author=V. Corbo;citation_author=A. Vaughan;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. M. Smith;citation_author=L. E. Heisler;citation_author=J. Mellor;citation_author=F. Kaper;citation_author=M. J. Thompson;citation_title=Quantitative phenotyping via deep barcode sequencing.;citation_volume=19;citation_year=2009;citation_pmid=19622793;citation_doi=10.1101/gr.093955.109" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. D. Smith;citation_author=S. Suresh;citation_author=U. Schlecht;citation_author=M. Wu;citation_author=O. Wagih;citation_title=Quantitative CRISPR interference screens in yeast identify chemical-genetic interactions and new rules for guide RNA design.;citation_pages=45;citation_volume=17;citation_year=2016;citation_pmid=26956608;citation_doi=10.1186/s13059-016-0900-9" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. Stark;citation_author=B.-J. Breitkreutz;citation_author=T. Reguly;citation_author=L. Boucher;citation_author=A. Breitkreutz;citation_title=BioGRID: a general repository for interaction datasets;citation_pages=D535-D539;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381927;citation_doi=10.1093/nar/gkj109" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. P. St Onge;citation_author=R. Mani;citation_author=J. Oh;citation_author=M. Proctor;citation_author=E. Fung;citation_title=Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions.;citation_pages=199-206;citation_volume=39;citation_year=2007;citation_issue=2;citation_pmid=17206143;citation_doi=10.1038/ng1948" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=B. Szamecz;citation_author=G. Boross;citation_author=D. Kalapis;citation_author=K. Kovács;citation_author=G. Fekete;citation_title=The genomic landscape of compensatory evolution.;citation_pages=e1001935;citation_volume=12;citation_year=2014;citation_pmid=25157590;citation_doi=10.1371/journal.pbio.1001935" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=X. Teng;citation_author=M. Dayhoff-Brannigan;citation_author=W.-C. Cheng;citation_author=C. E. Gilbert;citation_author=C. N. Sing;citation_title=Genome-wide consequences of deleting any single gene.;citation_pages=485-494;citation_volume=52;citation_year=2013;citation_pmid=24211263;citation_doi=10.1016/j.molcel.2013.09.026" />
<meta name="citation_reference" content="citation_journal_title=Methods Microbiol.;citation_author=A. H. Y. Tong;citation_author=C. Boone;citation_title=High-throughput strain construction and systematic synthetic lethal screening in Saccharomyces cerevisiae.;citation_pages=369-707;citation_volume=36;citation_year=2007;citation_doi=10.1016/S0580-9517(06)36016-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Y. Tong;citation_author=G. Lesage;citation_author=G. D. Bader;citation_author=H. Ding;citation_author=H. Xu;citation_title=Global Mapping of the Yeast Genetic Interaction Network;citation_pages=808-813;citation_volume=303;citation_year=2004;citation_issue=5659;citation_pmid=14764870;citation_doi=10.1126/science.1091317" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Vernon;citation_author=K. Lobachev;citation_author=T. D. Petes;citation_title=High rates of “unselected” aneuploidy and chromosome rearrangements in tel1 mec1 haploid yeast strains.;citation_volume=179;citation_year=2008;citation_pmid=18458104;citation_doi=10.1534/genetics.107.086603" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=L. Wahba;citation_author=J. D. Amon;citation_author=D. Koshland;citation_author=M. Vuica-Ross;citation_title=RNase H and multiple RNA biogenesis factors cooperate to prevent RNA:DNA hybrids from generating genome instability.;citation_pages=978-988;citation_volume=44;citation_year=2011;citation_issue=6;citation_pmid=22195970;citation_doi=10.1016/j.molcel.2011.10.017" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=R. Williams;citation_author=S. G. Peisajovich;citation_author=O. J. Miller;citation_author=S. Magdassi;citation_author=D. S. Tawfik;citation_title=Amplification of complex gene libraries by emulsion PCR.;citation_pages=545-550;citation_volume=3;citation_year=2006;citation_issue=7;citation_pmid=16791213;citation_doi=10.1038/nmeth896" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Wong;citation_author=L. V. Zhang;citation_author=A. H. Y. Tong;citation_author=Z. Li;citation_author=D. S. Goldberg;citation_title=Combining biological networks to predict genetic interactions.;citation_volume=101;citation_year=2004;citation_pmid=15496468;citation_doi=10.1073/pnas.0406614101" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Xie;citation_author=M. T. Tammi;citation_title=CNV-seq, a new method to detect copy number variation using high-throughput sequencing.;citation_pages=80-80;citation_volume=10;citation_year=2009;citation_pmid=19267900;citation_doi=10.1186/1471-2105-10-80" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=P. Ye;citation_author=B. D. Peyser;citation_author=X. Pan;citation_author=J. D. Boeke;citation_author=F. A. Spencer;citation_title=Gene function prediction from congruent synthetic lethal interactions in yeast;citation_pages=2005.0026-2005.0026;citation_volume=1;citation_year=2005;citation_issue=1;citation_pmid=16729061;citation_doi=10.1038/msb4100034" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Yona;citation_author=Y. S. Manor;citation_author=R. H. Herbst;citation_author=G. H. Romano;citation_author=A. Mitchell;citation_title=Chromosomal duplication is a transient evolutionary solution to stress.;citation_volume=109;citation_year=2012;citation_pmid=23197825;citation_doi=10.1073/pnas.1211150109" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yu;citation_author=L. Tardivo;citation_author=S. Tam;citation_author=E. Weiner;citation_author=F. Gebreab;citation_title=Next-generation sequencing to generate interactome datasets.;citation_pages=478-480;citation_volume=8;citation_year=2011;citation_issue=6;citation_pmid=21516116;citation_doi=10.1038/nmeth.1597" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=O. Zuk;citation_author=E. Hechter;citation_author=S. R. Sunyaev;citation_author=E. S. Lander;citation_title=The mystery of missing heritability: genetic interactions create phantom heritability.;citation_volume=109;citation_year=2012;citation_pmid=22223662;citation_doi=10.1073/pnas.1119675109" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="Systematic screens for genetic interactions are a cornerstone of both network and systems biology. However, most screens have been limited to characterizing interaction networks in a single environment. Moving beyond this static view of the cell requires a major technological advance to increase the throughput and ease of replication in these assays. Here, we introduce iSeq—a platform to build large double barcode libraries and rapidly assay genetic interactions across environments. We use iSeq in yeast to measure fitness in three conditions of nearly 400 clonal strains, representing 45 possible single or double gene deletions, including multiple replicate strains per genotype. We show that iSeq fitness and interaction scores are highly reproducible for the same clonal strain across replicate cultures. However, consistent with previous work, we find that replicates with the same putative genotype have highly variable genetic interaction scores. By whole-genome sequencing 102 of our strains, we find that segregating variation and de novo mutations, including aneuploidy, occur frequently during strain construction, and can have large effects on genetic interaction scores. Additionally, we uncover several new environment-dependent genetic interactions, suggesting that barcode-based genetic interaction assays have the potential to significantly expand our knowledge of genetic interaction networks." />
<meta name="og-title" property="og:title" content="iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/143" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Systematic screens for genetic interactions are a cornerstone of both network and systems biology. However, most screens have been limited to characterizing interaction networks in a single environment. Moving beyond this static view of the cell requires a major technological advance to increase the throughput and ease of replication in these assays. Here, we introduce iSeq—a platform to build large double barcode libraries and rapidly assay genetic interactions across environments. We use iSeq in yeast to measure fitness in three conditions of nearly 400 clonal strains, representing 45 possible single or double gene deletions, including multiple replicate strains per genotype. We show that iSeq fitness and interaction scores are highly reproducible for the same clonal strain across replicate cultures. However, consistent with previous work, we find that replicates with the same putative genotype have highly variable genetic interaction scores. By whole-genome sequencing 102 of our strains, we find that segregating variation and de novo mutations, including aneuploidy, occur frequently during strain construction, and can have large effects on genetic interaction scores. Additionally, we uncover several new environment-dependent genetic interactions, suggesting that barcode-based genetic interaction assays have the potential to significantly expand our knowledge of genetic interaction networks." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
    <title>iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"dZnsqpRWacXYx8DoCTV2baBc5fBVX2xRS2QDqtMDjCY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460542","apath":"\/ggg\/7\/1\/143.atom","pisa":"ggg;7\/1\/143","ac":{"\/ggg\/7\/1\/143.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/143.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/143"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/143"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/143.full":true,"\/content\/7\/1\/143":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eWckW901zbKEBGBkK6A1\\\\\\\/iwhqdL69qMA7UeFgziviv9pMW0RaR1sfJb2nIDtxzDZfyTPekKqeEmu1k64sXKx3D2xiMwPr2bkRizLzrDf3XRG2YTtDlZ9m9uNBVBFefwHx14fAlsA4NyiMlQsuSSVkGU5QoeVz7Ufs9FLTTnLTTGNe568vhZ9mF+mbzuMGcWI4TiarAlweFuSR0nFkVxeg236D6pZ4PuIaj2fbJbI47teGMFK7BJC0PqbH88riswIQos9yHzRtiOvJ3zOTiXFtEL71zhyV+g5azk++9gljYZ4HphB8koCNx9HLmRqKPyJlrzD7HT+A\\\\\\\/4IfBbtZTBzROiq6eaQlmG5QT6RV3PT8QI+Oqad5isnq5udqC8qPi4vRK5jz\\\\\\\/F2vDYCNWWB1RPHh3jI3IRr12wNlXTrXkhSYZ4nG1OT7LP1a0vy2bIG07RqqS2c\\\\\\\/DYJvYwpTUCTaVpCVrVNyjpSHJj+oTcq31IIdn6+\\\\\\\/yvuK71gulddkQayZNncZ5X3hgcT0Y4rsnhcUAq\\\\\\\/kwZqYij9hO8tkVXffisdW1HwihfjZBc74LOhik+cA9a\\\\\\\/Z47cifHbdwZicW0ZKuPxG1la8huHnGav85Fj\\\\\\\/wQWXUyXGv\\\\\\\/qk0i4jhibbAxRCcWrX6iJ71z5HGBG\\\\\\\/ZDb9mYpfRYkNcEqd5GwSpWkWSd\\\\\\\/bc2lWNv4KIGPxvbP6We67ksYBDkimv2iVRgRCOfkssCEEaC6KAM0DM5Xuk+kWR9XuJPLdmmdAdzXVE4spRojvqw1Mpjn+opOuypMzPqra4dzHHyjwGGvoGTdqXH37Ccyw1dMkbze7eEr3zK7hWF87FlAYrF200HV2patwdgbrLzmnZJu3+yDnZ+0dCPxF3ttvBUzZNh0nRYZhtJHev4h9CgLDawJ9r2bJK9GBI\\\\\\\/YsQmXMncl8AHHH0hXkCWJ+T48IbWZ8D4whyyb2sDCBzZ\\\\\\\/Qt\\\\\\\/bW7dbyasDr64y+V6vr6YHNlKdEZMjhFPuEJbcOjN6rjGV6crZge0abXxAWXMWvZSj1Q+VfCZTUC3XrID5Q\\\\\\\/S4WZ6845oTFR9+tUSHcsEKRng0sxpJhWqpxMYUhtM1CjIA8d4Scb1JBrKv5ADMpM41JAnETV256gVHD22FFNAsnJenKi8BiBsvVvLrMTpzeQZrsfegLKjk+FW3aTpxRfXq+WD0lJhWVnSHjdCwzTIfbtjA+rt+7McU9Ji4Og9JA11jBzJFoX9tTFg9w580mTxKgKNrdXQoy2ETNIYi0W8i1hgYiBqZ3gLkfDrxECoBybNz\\\u0022,\\\u0022iv\\\u0022:\\\u0022JXAI4NRmzDYxv+nUcpfgxA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002282b1491d18990058051a2fb8497e77535e0cf0a3e727d9a8ae9c70d18e1adf9b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GzlLq4+Hfgz3sX973uB8VWu7SgpjywSCan3ecmrl\\\\\\\/LyKTzXMQyj05VWUJ5hfLRQTKi7sQKWE+KSMJ1r1xMPSaF+9kamk17ClFmQKKiEjq1o0IRaYJ9jytocysGPJPRNISGFkh9SStt7Ixj9iofRHL1XraI78lrGr+tSL3U3NB1fMqmmQXpEEmi\\\\\\\/hCj5By\\\\\\\/c\\\\\\\/JT6FX5q7CKbcM2zoK1oyJzPt2zqJQCamS3QOIXO48FHGNubzdBiPh2hehVIITh1ak+\\\\\\\/IMdxlM18MCz7gF8AU7+dfgENn59dwAM05ttgDz+fdwre1MSa0SJj\\\\\\\/5C3ymGDnMZIRY1ImW4eh2yBC\\\\\\\/6zRARfOpJfIqc9A61BswwRTZGcDU5yR0\\\\\\\/Nw0rQlPUxly9TXaejHO8u8WicSgXZUVfIkhyER77YkBkX\\\\\\\/E9\\\\\\\/vCS+0FFChj+H1NFc5zQtKBDjzk8LJbNPiwmKrn2y7d9y5jBhO03MgXEiB8bqspIwtkiKYfQs2q\\\\\\\/BZMUGcVBtkkolmi0PXeEaTZ9BdoqN6xVDPmWeJ01o5NJ8HvNDEEn8oZ4KejaXuQsOqi9g3r+z6pxsir1yCvmMSB4M\\\\\\\/566aQ7BGBnmqUtbPZXQgVOyz7yQMgaBaMeEqMTLstkY9rf3QLEVktnqIwh8m7IWLuavrxr2a\\\\\\\/B51PsrsFT1C3xUjLrNfo3KvZjFxPyTe5p53RqDwkUAZVrmd3TIu2e292qov7YtGgcLaZc6IJlGxF5C7CbvZ+usTi92HnkVsRWpDDK9D+XODBHwESgeRzwwtkncibhTnTbo9P6SicoFOqpCyCASI5jXK6zPkZ6eXoFjiWTzx82aQ2V53z7qd1SHNFS6LF7J9Glfw3kIEeP6sGPT+wfqVn0J9szxyCzpCJAmtGPM+ny\\\\\\\/WbyUELZecqJMLXIF2vebJ0oBrw5v7tI5aQMbAxE+S8thwytrVAvMFsE+5f6iv3wVTN3GMoKf9jjYJE4dIFvf3+ZNZ8+A9ZGZA8mjbuc8pQhfGDdvwwV++1EBc0\\\\\\\/85D2mo7qBogGE\\\\\\\/pp+RethCeEcobZOGaSFGBXve2Zkwr5tujnqY4fPvUbOG2iczAC\\\\\\\/fhn3lnd7hAOCwcTiOnn8mvAycL0J2gDwExRILVLHC9cGZmh31tOQh0QgNYpKAb9QDsJ4l1FBDZiLon9Cl6tjAlBKstFkOMwKpIG2amuKOxYgWgDvykD\\\\\\\/GLKvCIcJ3V31P\\\\\\\/g9Lffms4ft56yvBIpVAyOMS8rBDxim+a4IGW2IAx7D2OWto1mb8KhqoNSqFVYlQCV21GUeQcfNm4C4kqFXL2tzUm\\\\\\\/1hX0Rtm49M\\\\\\\/v7BBHfqPRG5MS6QDdNtwJ\\\\\\\/7lMqkMJ7D74dwSwrK5ZKJ4dVga8W7vtSRgKzOL82e4F5CdLzsVbzPaCizT+k\\\\\\\/qF6drOQQX\\\\\\\/1sn7hqytsw3y7pVxd2nPhm50KnZnhCQ8elfvt5a3lIQk94LSvSWNuNlh2mIU5YzylGDbCL7ZFbsmJHwCrX18Ut9p2zEqcSbr5GHxTuA7VViZ0mLr92MPh9bXAD2\\\\\\\/sYNvZjTE94\\\\\\\/AsYHrmBGyAl8UKgHF0buwYo4IiR2mhdeyZvb0AbkxZf4gEGkWNnBaignaY6WuW7TEFFvlgg3yN6TdIifwqpbakmSHyzOfPjq\\\\\\\/ciF2xYGX+ekvhNWVpE5DpWTaitIF4nbN8J7MNuqepGpOeI9Eo15uShzJ5\\\\\\\/oSmeGfoA0ZpviAhmscBDhdpEv4yjUuskPWwFh6RC5u\\\\\\\/FNfHT9D1Zf9gM0chKOkGSECcr67+JAdKovloRYskH5bgnd5YjBKPQSsxPk4qOG8BnLITt+tebjGn\\\\\\\/OblNS28sHd6vbEl+SMOKe1etDYZ0HDfWsakC\\\\\\\/OgHACratC2GbYer\\\\\\\/4yBmvvB78fP+mWbfZ\\\\\\\/7ccpF79M1WZA+RruV1M8zInV3QsLvDGpVRz5blUWHAC8Yruc4v7REAi5vQ3sJrCp22P41DhEtSdBTVzs\\\\\\\/YD5tdxf3CPewqIasRZ1KxusQ9Zm2McuUninCnC7Q2L73g9dLqxJIdieUDVRRd3I3QriGjhicvXaWnU4opKxLhHDLvyDNBVqWxwo8OT7x9MEzT7BuRK52R+y8NQ6k5Bj4mNt\\\\\\\/\\\\\\\/Z\\\\\\\/PHVhgFBuE2PijBUwMoiMAaSChrOhJVH90sJTSxkmgTrJG+QzjV1B0iY7x38N\\\\\\\/spCSoivNifp3hDE4R71kAinr8xQW3r4dtAjpUJG7MPYEBvo2tfm+a7XyOcI9\\\\\\\/I4EQS3Vt0aXQT59XsyK5MvF04dHRQ4yhUdHgwcbh2n\\\\\\\/dQjGxANb3DmslphqLnTdHuDbjLwM7uH0CmV1IRp6k4MLOs3+pJg525dNePai5szIxFJ339pZNcOKM6QkWpKcb1DOgz\\\\\\\/2HCPCsVIqtywx6om8tJRGCkTZdk1af\\\\\\\/6W4n+E7LDJkh4GULc4tVM8CSwKMewh7\\\\\\\/YxyWr+N85z+20wQ2Z9bhwF71bzx+T0FMn6XC8e6Tm7q190YheVgkM0onZD0MRCeI57ErgQsyS6sfNcc5Niq\\\\\\\/Eqfu6eKVSQOdKmMr6BHHOkU9QWFq7E7OAhSMDQmJ+EZt5wZZxnf7m06VRP0SmsUUeqAs5a8mNWHLCAd1Ku+UMP\\\\\\\/UmLwgjq3kxiu774nWndSccwmqhHnkC1Ov6L4k2ueDkjxi619C2uWD2tU1VrAnRh52Hmm59D2Qs1SpbERyNal8ZzsRvzsogFt890GES3s6fEewDgktSe+oyJ+XIJBOgdGFT1+nYz+q9HQgXk1x4Khe3Y6sY7osl1\\\\\\\/Q+d7d76eIrGbYziWw8pGsBiGfJU\\\\\\\/CJY3wXGmRpGvcshs7w7aKAAjLDOJlKFYvjpWzWxL0SkAY9R\\\\\\\/M09bYHQEn\\\\\\\/EMCrGE6ld9tYErCtq3zSWH2QLBDlXRto\\\\\\\/TuZk6h4WREwNzQgzPUXd+tqzO7k8BS2n9zEw9JFg9jTqphtQNk7F76W+nqY5efGjPN9w\\\\\\\/AQ3OC+68Y895cmi7KhEfMQgKN4JznSiXDZr6z4Ehio3FXHzmbTxSrJXMva21fQyeafOYcSXz1+yZlF776TwjL2YNpqVfC7lpCVZxq4xwmkJQetExX+6PL3ZjT2A2lc6E0a5gnWmwStwKruBf9Sa2O9K9Z9TGOdpcxl\\\\\\\/1iceFx+E1uKxAyOo1vJs03cm8K6itq52lo8KSCdLLYWP0ugPmW9ZMIRsksvncCwP9zo+s3ui8BE3UZ92+Pd2Gfl5hK6n5o7Yvh2OmT6cJPrVgGLZ+CIV8XZVxSAh0YIbnORy3JFnP\\\\\\\/rkC7A1uEGROqL1CCZwex2TJb0gwoFS\\\\\\\/EggWU6lKicsX5f7\\\\\\\/OVtTmGvvtt3C5nD1Glkb4zjz9dnT5cSQ3wDNcZe1FIt+IU74XzYD7YVByQpJByCW1LLOVWP06kaVgHrrraHoOn1GrVp1xz6ovgv4q11eKeL5jg0oWdba1f8mtZjDNaafgnu9tSFeti\\\\\\\/6gq4GEGUcU5q07ejBRym+le94nzOcIbvOQm0p4xe3voAAb11Np+2WAt2WXcQX9byn+unxX1oQUsyMuy5OPGhhIqBXPqNmdCb9oGXcrZqzFuoAfrq0\\\\\\\/So6mXgxkU8LA4Lpugt3Bi92xK5B4d9v5XGZ0CLrVaEOvZHYuROCUKLV3o486mNkPUkzJ2SDA4vlvpNCcbvERcDDQfXhDm+ubCxBZ+kNxDcvmImG6zFjUDjsCVxhhI3X5d089xl\\\\\\\/Lz5kPjmHyKOQXYN4rgDN6m6NmVXM2iH\\\\\\\/5kFtcDXgJinzjW\\\\\\\/0jWhkMRgxTlSFb7Nu8ZEkSGr2YskbrohHvKqXTq3W\\\\\\\/uEiZjXWZr4l4BtRIn4FBZUKfK\\\\\\\/f3PNjiHW9tuPIWtGMVZkWcIg92wpjgTDcwJt9AHTk0vAIrxd4dCSGqtMDX0ZvZNfRJLTkIgKY8h6tNOfhZeAVbwz7CzFEM7J\\\\\\\/eiBlV\\\\\\\/EFjMlLbxZqgb+n4FpMH\\\\\\\/deAffPSPObPEnv94i+NcZXZMcbbXz0hIz0M\\\\\\\/vrCp2D6aelmp8LvlqlMMj86y+WxthbVsPmK+U6hbQ1CYunQVktnFw9xXO3MDhULy+23UkYL49rv8wyDFShAm+qp5f7GkSpEUXGE\\\\\\\/cxf1aOKAoJpw8Yw9fXqKw3qoWtIHvj4dOUTSsOS4FgvQN7Q377W9dz\\\\\\\/9w0kTvl0WwKbjMc7q4sSSFYFT3rlnzY\\\\\\\/RgZDw7rzRIjccg14G4gTPRuHw44PDxmcMrWpPSC6LUxr0xdpm8SVYUgLIsJ\\\\\\\/VbWPeEyLvG+LVRmIrUjshuXCDs9UMCb1tQ6I2TZEuzTc2hZm\\\\\\\/eWOylk\\\\\\\/IxHaHXgupWzx3htKtQ\\\\\\\/ysauZsVLgC3TNuxNrurfNs8ECI6lA5a77+KOUIhOVy9aFSKzwZqjBAZuMLAqa7BT4\\\\\\\/qPs5PJchSTZNGvnvJ30m\\\\\\\/nG2Ri9RUeduEz+4rF9abbI28hF+C9QbkNyV3I5BfZEaHdoOsipHtU5\\\\\\\/szDzgfmg4Z4Nko98h0x4QBE+kPpvxQ4Ya6VtRR9W3oEajOA6UdhYT\\\\\\\/5iSx5t8kaGk3XSQhu+Ls4S24cWeQg1JGsKtUmUyU+djetprLbW7pJE\\\\\\\/3mH8oU6hJ1qC+\\\\\\\/+cL7dpbY0QlYqBgmaObUZSjnN2s9Z6g6nFApi80ny\\\\\\\/nRwHloaTrJHfr3ai+BCnCrwN1jOODMJSKQuwk2k5+9ofFp1kZtiWoOjTcYDG1WRlKegbrmJXV0sBWJGNvvQTdCP\\\\\\\/ku8M62XgVT9YfJ01t17Kblw+KZUWujglvVbVsXt2XqGk0aCNmc35uyhL\\\\\\\/6oGIWYRaAjNz8JHpe6m0JMlcSlVXiYWweaq4\\\\\\\/AQfIsqQtozNpMkX4LBsVDOOrdPsSViAWvsSZoRXXLUZGdrVE+pKtLXfAjN3InsPmf0J95sF+0d+REL8Yv4kj9kZPdicXiK8lnMYXFMPNes9vgmMVOXUOu9bFuhBxKBJjsBaLek\\\\\\\/YI1g2fjozbJVe\\\\\\\/lOfjWlLDK9LTyXbAYbsVwJyRbGatnZaAckdE8Kuh3U86\\\\\\\/VbBOqCsBm\\\\\\\/K0y1JuYLjEMKg08zx7aKNhEt8EXKtwy+8nuVR6X07x+GVACTwIozgCCxu189odq8bcGdDWtD66XF0oNIIk0xTErd14t8trtLAsyweyK37BRQnKzYtNF+UuiE7gOy2QN8MuYQX+CEoIIYbk+0D8fhJvI6xuB4pl19RNKBvxXhxAh2dI9olQ1mOVayM5npdATvFg4LlvLJf692KvSoXOYBzIonksT8pzmZb4ldy5yYQ0KEluGrNlUk4YHJEL1A+JHSwpbkOdpwP\\\\\\\/sI\\\\\\\/WZYMaiHi9rF5s9IqgcmrPn7kN7vaLg\\\\\\\/vxGKc6YGKJrvMhEhJY1tCgSAHxRPpPxqx0XTD6dDbJef6aaGbopX+jEidluLZYWQt0vI3CCSwkCWDXpZtEViA9oS0vTX7b36YOJP6VgKOtIWNd25lPsqI3QhPwBRHe6FB0B615dLWA9ZFLu8LQ0W2CT4gQEXvH87MA534VdHryrvqtfteX8zvlr3HetrBnNdtSKxCruUoH+HCs1vBvsMBJylT0NJ3V14H6tAIh4h4uv4H73oOArWDPxieDd5IMhNqv9mAELazAeZhIkltR8927D9\\\\\\\/aBoTrBn5uSOXwZvFitMfAkRSRV2LmVT\\\\\\\/12EqAQ2E4Ih\\\\\\\/zUifiw9IGQt9sW59nellCOdJbvH4TWMpixzdfOaS\\\\\\\/QM2GiLcTHWU6ez7IddXu6gPvRFrcMKlwu\\\\\\\/JpYIBeA0kQLLHzO113XhZtR7HN\\\\\\\/GKu06fvCLEzAAGh7DYjFr9i+p0VBYtbM3uNYc1pFNRiKmW8Qr45e3+MzclfuoDtdInGbC\\\\\\\/0L4w3B6FAJMYWVSPQtVupToUwnxH2Pljc3Wyet329C53aJOA7NEqsPwS5kOTcQLKyWtifrOwnTNMXe7UTOEe53MRIeAmCQtfpORxl1\\\\\\\/Z92EDB9JoFZ28mUVVg5viE\\\\\\\/r4zXtadSsKwLGE9f4ISiZIFhvpbiglRykObSFQyGXtl8jBg2O9wflqzk33enWPbQSIWftg4gQKwpexSwLbmwgNYT5\\\\\\\/kYHnWYPSsMoVyC1XKzoGq2Khzb1Sd5QLCfiHFkB+O22a1ctaZwpy3\\\\\\\/YolANoyAvNfLOeONGEa0QulAz2dFGtUQKU1HLPYqWxgvRWIhRgS5lWfMz\\\\\\\/Pl3A4J2Ng39O29905he72wN\\\\\\\/4qzHDAHSS\\\\\\\/AyDNWdXbQQD0dqmKOoBEVhRhArLaoIZxyGXbFMgVjzoQ+U3KhsfuX1GuzEcJdqyBCh2CFadVQSSb9\\\\\\\/HjD\\\\\\\/1aOBeaCKuh4csuzZPWn8y2qYxqTK84cpi4wfUPhqiQr8uoMVjIeMjWvu2HVRA6cDHnl39o9JcsQ8mYibztJcKi0WRnZU5BpvbXBdZuKSCpeD+IcGkqAPKnCZUrZEm5Kq4dKEYti\\\\\\\/QZkQrh0cFzZz9TvlkV0BqfFcXuWxJXayXM\\\\\\\/s3Bq78sHlz9CxjjfxMS965tqaAM\\\\\\\/1qdcxauUIpURoYsrjV31HI5NjomUMYW1uQTJNOng97uq1bzVidxGqdKChkK7igB99PwQfGOJEpUyswtLGLW9BWJbQnhGu45W3xvjqtG61yK1CgC4QF3kobfNpCppnIf8yTkwJy0MTH9BfHtIu6Xizcfj4jB4y5mzzY6ATXJMKSSqkDOnvNlq97vf0quuOkFwfCd\\\\\\\/RtW8aZagkfaFkrjKsXGfP1x++9lXHe7bI2F8rptlhBMv9+t\\\\\\\/j3fQbK6NAKNAo4I6E7AgUUrGhWwydHfdCNXUyM8OUUaiuh8oshvRdDUYAjV91YyI97sFGP2MdgP09wuZyEWasPHfWh835QVsM0gbpcbPAeI1xcc7aOH1ZKyIfFv\\\\\\\/NWHDfIJN3i+tZYEH006xJQd4LwypQCH7Pd90Nvk9WoWfJ8th\\\\\\\/o03cy1u1DUSeziux2aB9mJm8wFUHhpXuVq5V5SICkfdA5g34X0oY0Ofbt9j2ScwyejUp9ICxwi3LzGNB8evP0NNPYFWbsYTDdoc1+MVpl6rd0YU2ar0i\\\\\\\/YrqJXKFBpIh04eAiMpUZg7H7VuVULplmvZVcJoUebhD\\\\\\\/D1nKg9EEJd6I9dKhs4GGNnK\\\\\\\/nx6G7zylW8Og4SJmsb4Aw9jGtQ4yNXuUOKtDdw3iMHaVOktpTQGv+IPV7fU1nHidS+M6QAZi+WuMIWgM+\\\\\\\/TvphBVvWPAOZLWxwODb9PV76qbuEqismwm8pnbYLs\\\\\\\/5uFM\\\\\\\/FXlye0+JAhHo39xAJSuw48blPr2FFlbUw56Pb+t95Yvnhxp4FNTobsCM\\\\\\\/+HegcrffLIM5uh+IydnkMYF5PHuKG1HY5kwGDJJiznRZ1cKYZEQEek12upaPfJYS1CpEJhxd1UOKPJ5W++WcYELjQx\\\\\\\/MtvCR0XwR9rjUssuES02E\\\\\\\/w7wJ8LoIsnTpCbWHoIeucuZYQusjxuGmS8M8UsK\\\\\\\/ePpZjSGPjrPytrabDZLol5Vh\\\\\\\/EDrkZqOVDnopkOQkCpv0atKg4B1epxNftv0SA4Tq7NoiSGfVman4kxSu2R4N1bEuStDXxsAFINqIozTO1++mvqqyJ5AlBM+pmfIfshzl86DpOHDSXdQhMT88C5UMLc2CXao4\\\\\\\/llBmqm\\\\\\\/0quTkwZc2ey\\\\\\\/RscIzSIzyKjKHO73k2cp+9Yj4hWHwkq6WHzRMDLFDF3OMYxU8GgNZ4I9wokI9PV4y6rj9ZMtdEcj8DAFBp\\\\\\\/MxxBTiw+e2R73+v1000JMqAb8nttPL4vitulxWt9ivIGYXHILPBlnvCLUbfRmgBbAFr4C1Y6RiMxrNmJSccLfm4SOymrjGJ5E8pSvaAccOrJV6vDkvqYC\\\\\\\/BxaVx+XQ7hGqDAwzB7DHcrrQSYA0yW6\\\\\\\/jJ+VYmrI5DJ0rp5tk78vvPeGAc2VK96CtsoaA7MjSCnhlWryo1x3Y0jTVoPe3KeGgiOuBq2z6YkPucdKuFTwZ1NDyxJYtxQkM6P7X+hivyQUHqc1gb92WbOcCJwLrPqsw+7QkadllZQewrLV+BdENtFLPtcuWXgJfcQu1xLNVp0tTl5QRkC0v+mgdrlBP\\\\\\\/IAWEKF18DlD8a8lrXI5TW2ggbviqayC+N8Sjfmdl4VsTlr4LKkoXrUi9gi7i9NfJuOOV3idq+odQo9dJP\\\\\\\/3telTTL96K9rH136Dlgh+s93OaQ02iEH2t5CIw06plwoINusZi92I\\\\\\\/psdd2vurMEg0CYH54cqWacz0Hp4MkjYaw9XVnz70pLB4NSiHX0yen1v8IX6MdbGYXq2wvJKU2xryODyDWs+Lg+eyL1jdBM98CBelKD9nr3Vboxk4e7VvtIOHx1y1Kw+4E08xo8R9b2Lx0d4WOnrVRW16xfe5+oEp4SDRGHBH8bWIDet\\\\\\\/P0UrHKGx9oGcHUjPawPo\\\\\\\/gjZG5w\\\\\\\/ZzcGIjorJWELQTzJ+OtLXfrnKkvQjAd022y7gjs+rYFM3fejdfQpUx0utVKm\\\\\\\/rIe4t39VQYDFNWCNNWQYoNlD4\\\\\\\/3MIN0GH1bvPhl+4SBELiPtq7x+WNR2d0V0\\\\\\\/EfKTC3ydhCmo+S3OgMY4oFO+PbxlepZuLwBMoK4Gcn11\\\\\\\/FqB+9uN59PUGVkE\\\\\\\/xJOpXyxriVByyGdwCeGXo4DHfDbFb8wjz95rOQSQI9YvEySVLRiXD412VlY4uzfTlz8EEIxiUCVrlLsSfV8v2O1wQkc+6dEldxJdx1R1AQMhyKcKaVSy4kDdoiAAXnbj5a\\\\\\\/gXomUlFEYH9CYvZSgZSd3i8210FdOYqMsIsP+Cz4SrvuaQCtJ73lr4l54PN2jY5XPP80GUikWajzD1UYE+izCNvQJnCnJPkB8k3JVa8N8q7kojiz8bx\\\\\\\/cjLula+4ce\\\\\\\/JyUPOVN6iENXuxF\\\\\\\/tAIFJDqKxv06rMhM9Ngxg++bd\\\\\\\/5g2Tv1AeBgYd1wQDpY\\\\\\\/srIlCGL9UPvKK+\\\\\\\/dIAoZ905CpA3lqu16+ZudbYkyy4kKfz0QtG3dpCAgDYTCvTXyg2QGzsAG1AnUkux5YL2Fa2\\\\\\\/IEHU\\\\\\\/GWXVyQ8IiUrZ4rHxiPXpTv8Ogbs\\\\\\\/APrbmRnYx3Ivs2i+57eFf35d1bewgqBGPlqsjPWyFdY0RtTyJK\\\\\\\/pJ5wgxaqnxz2x4OSRIRlPUnOcqDB3QaxwiYcW5w7cAjPsCY2UfbXxXAIyMlUDJWb8JGCIQZ8PmsZHQFY4AvmdZuj98i8GSFM7ZLut49T9FbidBHbhoGra68KEBPKt3QA5814QvaNHK5cJVfggSwM0P4TuyRe7ULjkcnTRMAfBzAutL5iF5FQYrvUjJ7oh3fgrlgKWO8qy4taAfsYid7jhdVcjEV1Cyl+sQ3ww2lzjLfYtXh7\\\\\\\/1LInxhKGGD7zrrgPd7CdJIxNRk9qnf3CK7hb0W1PLYW4ls0fC3vCGsAqdR7VeilXep5Sr+7ydncBx970eD8IrThsa6\\\\\\\/+Jy3Z7g03lhnA+pVCh5t6Y3QBZEtp2VebMe+DSotANDkvMwLxz1dtCHflgp9YNRWwVw7eiRyKa6f0CoBEQhC+AKDH3RfoUaQDO9X2XekAN\\\\\\\/o3lzWxdGt+rl1arP2wRQYoeyBPMStRS7XMjD8Uhzg7IlNTl0NNOwdqcaWzpgzlLmvVdHRAZ3536QsGjqXvHW0\\\\\\\/SaYEGkX\\\\\\\/ryToYfn\\\\\\\/QCZQHCSCuzg1aN8QNEPsjTmIkNQiMFYkL0sc7Ff0Zd3RDLzhFir4Xy+x9zy3QUGSWrJ7vXkN\\\\\\\/BaNxLc0jiAt6UlQN1MbKvmSQv03IxE1H6hYo5g0cDC+OvCSSIfzNYvckR2HSlk6PP3wfilq4DMCD\\\\\\\/HvL90kFj2FLAYcO5TLybmGkS0HSZe82s62qSyBDiIWD\\\\\\\/N4wZYrh0blHx4qPIi5gw7uDkayFntr9jJ3GWOiigLMkyPwKtE7ioYxgKcmGHFfAVZLjGabm8ORzolbMo8T6PQ83a8s\\\\\\\/dmxnlEJnrf0aEt\\\\\\\/u2+GFW5eECai\\\\\\\/q6rRVLfNCvDH7+mNs+NoQ92uHK+E\\\\\\\/EH7cCeMKD17qTVmxFHV5DEqE1n\\\\\\\/6MUcrVKtihq\\\\\\\/9OcwHuoAPSg9J+iFidAYu78vimqGMsegZ1UOajyqXfI6R8xmhqHcgLoeBrEYpoBaG8AqZQ9kKmt3G7xofC\\\\\\\/0Lce9RE3NH1WCcs8BetWnXdDUDD72kl76cebh0jFcmoXNtIEOqJwUjonlOiWQTBd0tlKIECTGeL3S\\\\\\\/OTAp\\\\\\\/c1pkAOUr05GNeDIuYI17GSuDz6fnwXmQtkoZJfdZIeOQ2tXgRQ7WW5aBOmo\\\\\\\/jdkqZKDyIc5D0ZSUJ+\\\\\\\/ZSN2FzRF8+1eeGcD55dJSFIbS8DOGAFuLTLYizbjhenuSDjGzVyMmaDTfnQrZiWGyvA+hcDfHKth+CZnCfeO\\\\\\\/byAdJlpjhafRos0FzQGELjTU7UJrU8\\\\\\\/QnPrQVMVxkfqmzQtp9yDLrSMgXR3RyyS8fwt+mU5nvZPI3VAsqKn8dRurreoF907pWpqWuE0i36kz1xIn8m+WO0coKmcwUCJtNeFLGNF+0Eepl4QWPN2ARHd5bgM3jzxaBjLdVFs\\\\\\\/fcCYZmbtVrRhdVZmrRNQ5+dXCDGx1pDOJakfRpjzEViBFTbENi1Ce0wHFoQHJBZ46zE3ebwzbMUxJqy6s4FblOOgN7+XmrwgKuRYK0A9pH24\\\\\\\/4fA4QvhiFEIjn8Wq2BVl+ll+6ADZQsteuaus66UNlfV6F+QEBvCUwbISCRB8A+pswP8syKmiwlqnE4C+wRNsA8XvqfNwZq5o96wFwt+qqgq8ARndO1erHJdZuJaABX4wQ4cbTZWLAEETlgavhT79pZlLtgfvbE+mq+UEKS2atwRMdNepnlblYH9nt+e0BSZbwu1AZC3m355QRlcldY2fyCT8n2vOGneK1BZ4NCWOpgn5unsMuepyljytXfJZkoC43YuiXkqalpUzq5gBuYN63DCOsSXYruNxkXV50BMcCsDnNFvgSSiGelv5fYfXvGo41ahjoIWG8Awg0R7nSnuuzJ6UfsZyVlfGpRBwhI85TGKzdltQV9cucfEjQOgJm6teBirNRpc8zmr791Vza2KJY2hEXkxJbHj67UmffuISjzdUqHcNhHA7OQF6Pzn7Sj\\\\\\\/eNTE0TGYwhEUdfXjXyCFlVP\\\\\\\/E0puiyDiAyaW7\\\\\\\/dLGAq26ygCSqebNnnRjXB5jNL2kR5vpHcByz2WO6dExBEcRdYQEECLPLL6Aj0RIEyGUwMapR5VeiPoZFyI575d3k1FTQ6SaRN3WExi8X6zOCoT\\\\\\\/1BAnxk1Et8dRTjNDBg3O\\\\\\\/amlTRYcJzdAnYBE7JsUrUDJ\\\\\\\/g2PCXSh+cyXu6E2Eb6D0G0EBrhAwWcgAN+OpHptAHGkfU4KIaGsk5ETsQ3F0GQJVgPmkYCpXqjQADwF+qYcZg6vrjWkQFmpFsdqdLkrmKjHmQx97JCKsxCxddGArXqL7xhqAkH3jGpb\\\\\\\/yAOU4US\\\\\\\/890pJ76oknXaRc9Lh5Niu+yhHv\\\\\\\/VQzngXqF4J3iCWYzPmNyXNkohXP0DMIi+lCht9kjmUJgr6oXhW1WE8vgiqdPToQMzyM6FED3YfYgs8tgGtdmDTVS11Vbxil1RC7ktzuFG8eInBnPkqukpCML273QDDIz5nYKdHdwWmmTgDndFpQpKNU+CGoxY\\\\\\\/8E\\\\\\\/z3IaDgFpMPhoqLbwWJGd1E7yIgV5pOpF3Ps7WG+CGDflpt89Wrawa3lOJ+FGwrSrI5ArJaRuUudMTdR04MSqRzN\\\\\\\/+YVa7+B9oftnOjIBJ9lg7ylpP2zWlMX+6qy5pymQHC44CtFze2DWUYazjJnyJCf1IjO2F2dOVJG57dX\\\\\\\/Br1BBgj91Jt6E+qrpwEV+ijNOfnzDxom5WPMPmHFk\\\\\\\/m1BN3wasbbEi6sdMS80z5Hju4qDIelzwKuw9gjJ+I7a8hS8+63U3bsmxlaeIQjfTYiRFKq36eaj3ZDwdPWJVKngXJTecLLDU1VtMGDWCTJpAb\\\\\\\/aqUCSew5VLOCv451HUfSu7F2oT+zRCwvc93Q7vrCiifVudsqzp6c75eWMiJY4oLYo+aW3Vr\\\\\\\/SmsxET9l4KW8yNF1V\\\\\\\/8Lzl\\\\\\\/0+mYfm\\\\\\\/39WpAhanTd2d9dwP+mjjWMvqxOKurKhhW50Zbm2EQ9pjijuOk9vrTNABv3d+04+bADd4ZyBLz\\\\\\\/cp3q82ueHzJTmh2LvMtKf1xYKksUOQHc6Y5lKdKTKEErfJ9l7M9G7HkZcr3D0vP9yyyb+xNAHzg54gblweRFhqLUvFWoqSu4wvyM3HnA81BdBzrvAG\\\\\\\/jWPiW+Q849dxRzQbOXoOF+5ZiCxoPlBVtXaZTQf5JmIR6JIWfJiQz7lHTb5MM6QLetklX+0Dlw2M+nTjGn5gFTh\\\\\\\/477hhHnvIaRz3A2f\\\\\\\/0UZSNGeRAAn0W2PL51TYyuVXR+jLsjwVOHewCrC\\\\\\\/799+Cd3BGUUF3FuZ7HT6B8I1fBEqIxJSrdT2iE7NFUglnZl3NR+JlLe1bcXn7QN6tdRewB0Up5H+\\\\\\\/bgKgwXNjsRmdCeBQnAusDNaxtVXtcNPAthwQe02UnvDlywLObSaImKLEBwaKvC0aQFkWstoIsc91qZAquWJ6p\\\\\\\/Mnbrn\\\\\\\/rga8v\\\\\\\/A9W51KDCxiPFI3SSimJ6caap0VEJqF90Rd4Q\\\\\\\/YF5hF7eL21Sc13C5hfSCaamHzrxcF+E4yKzo65Mi42kVTWaviRek+rGD1wmc20aoXG9Gmzu0OcMYBQsdrsti3bec3NGgKrKpJUFkkLFl3K32DDWGPMbGO\\\\\\\/1YkPrVzhHRAlxZEivW2gFdXZz3t\\\\\\\/OYfsbbgIXbwMhm1InDyltAZ1TESx4CoIQmzvQCCZCBYKeXQRi4vIhXBCzJgFPHteUQ5sRqqj2b8q6FDi2X\\\\\\\/H9PPsMzfJmGUscvQQ9hOirbFinvEOABPjX7pBWGdQ0Hl5a2qy1t5Ya1IgW9eCpr34BvS4h\\\\\\\/\\\\\\\/P4k1drjxoJzuiMIQuz3QoqEvjdk5w8la8RXTJd8ZWg3Z8TNsV1azSA0kD9ZnVWRD5ZjkbGE5WOuYQHwfm3I4j78otZWGmYJHg1ynzOqTb\\\\\\\/\\\\\\\/DeItRZ7jgsgsovr\\\\\\\/lVd7kG60XSBrmmsuiSh+wYtea2UX6i4JJH8Kyc4593bjyNKRXlK5sN7Wu9YnT2gDl1qbRF5pOvTNLL6zVIiYykp1FNZn3z1qX\\\\\\\/NXF9P\\\\\\\/Yj19o36knQBOnwl5vwQuI3J\\\\\\\/DDX7Pay7frwJPYfQfUFTkP4CajaDzYr1MHg2S8dzo5+xFPGmE\\\\\\\/CX29r0wbxqS8aALsSNKROUJNGqzcZt0RN3W71uyBtMh7nwclUAVCpcna\\\\\\\/F40k6t\\\\\\\/2D539ZbTvJxzD8opemhSJvcBnMcmdT2z8B\\\\\\\/mTFLOIx5Bc0rZTDHOjQN671p+DO3Y4oecb\\\\\\\/Lh833PmItmaH7pLm8z3yqopDBs8WXFaqLBryp+XX7Umss1yLIs+tLHSDhW++lLFSvdAf49isxR0qB3KGWT1ODCm5ILdcm4MmtR1IptqZmsg7xMIKOhP6YIWek80ZNzV6gij5tWGsT7Gg5u32K5Dyat8uR5as0WK0pJbrAdxe3+NLwN+Tuc3Z4farEStZ7+QEGP0tL4jTBtdiELiJZB5CFuxwE6D8YC8yxbHs729cYU5113V3b18xmO19rw4nrr8bk4jkgLQrzGo8LBohJf5bRv2U9CqJptKVEPKkuXvHjsAV3NAI+c18TPWmL0hbmuO\\\\\\\/rMcYqhhXUXBTIca5hb43e9Uf234NrQvWmb8SAmFv\\\\\\\/ZTsooCNNJ2i+J2RIDjhHjrtCn1Mr1UXYS\\\\\\\/jC3+A2M5Qyqlr84vNJDvU9FALzi7Etolp2sC1QrzewBkC5l5mbyydwlEhYxgI2+KjGkKdWonRPdIDapx+tmzVClB7Zl8lGW4VHLHz\\\\\\\/0WYiaj0eXakMA8R1Zdri8ghHz3Hjg4YHzXYCfe3MozSK\\\\\\\/d5Nti7v2lQF7bNJOpHaR+UifgDO+xCPnHwJK\\\\\\\/rWQgYsgT04dhGe33Jbr3k+xYlgjChYITzx4Qpau06cMdQQgRGOD\\\\\\\/vgWTHz9IqOvFJPIjlixsKQhJ9gFwNo0dvbauNkNoF9xAdeqx+MJnZWGQr6lyXFqblqFRSkaDQUmtYbp0MjS2jL8PoSjTE32ozETxwf4fPdP+HSPCVK0DgHuN2C\\\\\\\/zDFOcY2XhfvggpudTculiP5+Kri5cxXLAvTGl3IWHTEo1aFq5Z2tGhAuVY2xmn9udmonGfRSTPLhYilLYVu+wQTF530FPLlyBMZ+TCBBdl8muH4h\\\\\\\/KQ+Rmsyxlf3omTpZ3X6bBR\\\\\\\/gy1q9rXRjBkTUzMKdVewMyb3PSfauue9Mf+EyHJJ5p8R+2ABAe9A3IVrv1kZtl6vD6A8i+59IpbOOwGH\\\\\\\/hFpNsRM6KuyoK\\\\\\\/dLHpVMV02n+yaBIo9wFLVDYsbIjdFJGlXC4cqFaOF2EcxF5cRwILeLaIzIqQ4mGxxc\\\\\\\/0RgKAutrtsUbbegpdOazT+GipOX6f+c+9JrjZ3MN1ohIHYdcMRidCTRLeisL4NCr26hmVm5DMM34bxNuoe7dEP70KCZGOrTiBoePhfeuXnZ2J7hMoujP5VmsUWekDufUfQ02oaaxTcQQ516iYT7XXiixxx3Egu87IvYgcuzl\\\\\\\/JiWkbEp0TVh0VGhlB+VkKaHfLqk2uRklttqmczkRE9c3p1f4U0+E4qauVJ+cf5iRFfqqxill6dWoyUXuIviL833v2uSoZQ4\\\\\\\/XbUruMheGZUyHvmdZxM139IT3uLFqcpmNy1p9B\\\\\\\/cXPSGbEDid4aE5WBJhgAnTcn9EsLWvVUCvLqrc4N8tceeWHCDIKTlntux15N7rNKm01cqv1Up\\\\\\\/tF0+SnTU2Ijy29yo8dLDwjKZqdb6i0ggiXN6+IIucuyWh8DFNYKpzsVqGZAzQUxXG54RoHHz52X+wtEc38nwvtaZnjozJM+3IvDjKUQ5BHh9pxkrs458QlOqibFEV95mY5zXUdIS\\\\\\\/OL97DGSmJ\\\\\\\/fb3TcPaYxOkhuAYJCpbjF\\\\\\\/H3Dj3qy+ynbLln09JsGxm0mTOzQ54Xju0whhYOjxO7iTunLn8vFDad6SKUJhC40LRBeRuaOPN7ekF3E9gTgGStIBUxxxU9Dmm91zKgYbJ9m9wjmP7ppnIukQ8LRoR3s3KGZxluDUHHgxEzDvv8urDYJuHzsRoqPsXvCwBwN2u4rVVrbV9\\\\\\\/hpxFsQz\\\\\\\/h4ziqQNQyYXsVI9TPXGl7kfKNME1BdziYRcNZtk4o9idkg91Jznw4LW5GqsOgNOFGNgsPkQPqwmvAy99QfeAFNDaqUICpQHG6RfKvkDckar5JSb6p5xZSGTk\\\\\\\/QxqF3Wk1Hpeyn7KDaYFlDwI\\\\\\\/RDTANHwjT\\\\\\\/QqqqZpM\\\\\\\/Ji61wbkSPHMOHniitXHflBjzj9l7JvcIUMlBT0EIOecZkXynhjUyhjuIpIxb08ASVo7KjpYxiO\\\\\\\/ZnY8s0Fh1E\\\\\\\/LohQhzs34HdIlxOeFkYQQqrkJf\\\\\\\/0zfv1dpatZ2N8RRdB1UxSOn+j2GrUcr8B+fvO6OUsFI++j1CblCdWj4K3k9YfIDxvTg7ZFdRsWGKj\\\\\\\/lvqkF8QQXFGv0YJAqkrfU60QJRyKKoaKb+vrRevoKmUXCTiiHFC1S+YONaBB1IlJl5mdQaxc6boAWp6uX9E5BcxAGqbQUJ5wTzlOQwGnIRxetnwSHG7hJwBgBHwowlhCytdc7hFQnKhc8rp0JBVlrMO24FQWsOhezJ\\\\\\\/OQzzYx5kOBYOKpL30ixDluTPD\\\\\\\/aFvrGTdUT\\\\\\\/8eUgIjoUA01QWtJSRXxLPkDsbr9p5IZ4XAyh1oMcsTbI8pKHWSpA4JZbF5tRQWJntNY0ER6634jMgqcbt+FjbnYVSycvLq7J35VwTS0NAy1qL8LOpEejWX7rBtWDmIGYtZivWHtA3YrBvrWZOyhyh5\\\\\\\/Su6302QZd0+4ClquUv\\\\\\\/3khnwJdigcQ93I8tuw2PEg9moB3\\\\\\\/8mNp3cXnBH7qof+mBP4xBzrQ1wuFf1JXKc7M1tJRyhI76wg5E2nRI0WcuagY4xh54pv78cGQXCyiDEvgqyKd5WSavOW\\\\\\\/nDd9dwFzyoHtVz9PU85fcmnWw+JOJXtBOK7LWvVIybwZtBvPA+vUth2wwZ9UlQ3Ta+sDCrRWmuGOOJPKcRfrAfnNF\\\\\\\/5gQYDqMAtAvBEsCIEF7Dui8ldAFbdsd2++ZUeENEomIUJipYJTMuwxGapyc4JSYa8sDH7RW2FFCI0Zj5YR8+oORj8JRWNcqdKTFajOclJzKUJrRMmhswgcgt6IPRBbRThH1V\\\\\\\/3+FJ529GcGEsNokNk4DYxznJqZaH+ZehkLo493XVv8YONG2SkW2i7jhMaspIBFceSbtXC4IKPl4kw2maRIAllRr6GUoCxw2y6oOj0cBl\\\\\\\/aME7QANb0sWgkNyRorjy+QlO0iIaBPWpCHhkg3LBrZOJDEITcSESrE2\\\\\\\/PpBUYaNNnCJJr6kRxei+\\\\\\\/O7bhE027PemxQeKMnBhsQJOM9pT+dQd5FydNxYzUUFdd84pRyzwFP+X3UdNtLfUfF+l8u+XIUNaEMXBOwgszwK\\\\\\\/92weOIIPP2SzicuE6OXXPeRPqpf+WQ\\\\\\\/Ra3s1NZ+Rb\\\\\\\/6ijtxdSz1Z974Vz80mwZedc4znDy0dwm\\\\\\\/0cuDlapvyilTL1eZdVQ+xVIteaKYCJQI+uSVoNs+uz5xUXNF2YaFxsmCeg9wFRm1SlT4S5K2NvU4TXHv4uMIlfsJ7M095py93Jnrmx1j463JJNiad6\\\\\\\/9cGu98azpRuHWc5nug\\\\\\\/N5LrRAR6TwNUyoL0kHJUTLN2q8wpr9JyoBlRCxmH5krNWEcNnCw\\\\\\\/dxbRFdUAPpyuDd\\\\\\\/g0N5Pj4lO7Bn9gxZl61BCx\\\\\\\/nIRF7Rv6qMYHkvIcWMZwjanzrdpgOlB6rFCn3hGPrrRgh5x8ctBudCvbYzGpceoiIynZdMcYluxJfR\\\\\\\/wNHIr4L1j8s2zqR1ussiWRcVxcQp0nFVXryA3vZBDD4LJzWGNl2TLigYgHc\\\\\\\/dsa5aTif21mKhnm5bzyMbrI6\\\\\\\/b4HFUB8SUEulYSpA19d+7UkjI9j6nrxUAPDWw7z3z3GYdb0fF0p8gC\\\\\\\/ukQAEotONIDoxkCvFDCYVXn2b4tVMHlVcQMtb1s32DCy74ccbUbwMUtngpHTolzLljULtmlkk2HN11yIqOXnAW+Uul9qE5xQ7s9h4X6zvBX8LUlxkZWZT\\\\\\\/JVm1iETGpHChpa6INYywaUJiWSs86hgvDioD36L0r58ywCdIxnJp2vvnM\\\\\\\/gKs4yAzI8MCihuUxjVZxUT0Qq5l0uCTwRdYa+ebF7K0Dp5wvNAyoI4gH86YPWiJcg6LEVJbJgJIvRnsTKBCP7aCizyqSy\\\\\\\/TuKsmmD10QvdV+110SgaQpxqbl3t6w34n0W7VQGRBCDdKZxsFiLthaCBvSd870TiqTOH3yFJ5Jajp6qAo2GtNbQz4Q67I\\\\\\\/6RDzTK1OirfLS0gDXo9gvGEfds29NmqM2lBHXjt0LX1X\\\\\\\/L3HuAvbYn09Snr1rnyBGWcpFm5lytu+STpxfvegTX7kRFaOJBqw5BBlYTz7WBWqZVHe8VxrgbNVx56O8XsattPXHsrpMNl0W1MPGQS85cZGr4MT2G6rnbeBLWVV01bsiu\\\\\\\/vFq2ozLBe+KFhsrdWaXq4R9W9rmLFGNdZGGxVynYGZhT5\\\\\\\/c36I9P2b0ef4p3LRarXs5j7V4i2vFQcvLjW4jHS1yO4Q+Rc8Ptuary2PYc+Ne4ixKYXIFqHoTelqHsm0GxZNEUQC1iLePZkpAocMOJZ28870tn3kojhiENOpM1OdEM6A5AaTH2hd+ddiNbuUe53e\\\\\\\/mVyjl2ffLbanAcOk0AvExOVFiLRgi2VYjf9dUnDVG9B3TtuNWzEKoTyMKi1rITV1ykJ9o8U5AFsEfHoLO2i8iLkm5U7Td3bh3xrAu6Lg6TQ46gXnit0MZ7o7g\\\\\\\/fvvnjxyyHHpwAbDGbn1tSL+EY8\\\\\\\/ppJjbi09YgHzovC4HNCtnupBIu4C6isR+BaMdDGzE6D3AqtEvX8GlRQoXWJpqdtPcS4omfLnYd6aywleYh3eZrfV9OtBB0q\\\\\\\/E2VDGfrb8560PvuN3sYP5dSUeNKo9lGdPUjoIZetRqlMm4G\\\\\\\/dGtQ1Hbzr4yaYZpzvNGVyeM+799UO4HAlCVsca0WT+DubCanXSttUg5aAvJTz67uU+jro51YNe66XwrF9JY0+Lnx8\\\\\\\/qgGALeNsAQOiT7VZxqNwUQDN1uGFfulNHwJMh4UmsBrOpd\\\\\\\/otphPrm7OUxDysBRWW8+t2ANgLV2Z1ZHKePEB0GLuXOwgfClJYkW29X0q6f7jmmIVJsYze8J1B8J1T0xW7DrxoFqxtjJQGsC6rlfqr+l4CP256ArKWbf7j92fBLPgx9TsDO6Apj301oSHKJoIbda6OaCpDeDlYhp+254RW2hylhV1IarY0zCq4u9oskKfZZcaIQDWqH2FhafEWhqDG4881dZMgHxL1JNzA8aeqzSf7mwoH1Zl1qFLiTVR8THmUAMnHe\\\\\\\/jIyFTkV1JlpFQnRCtKR4PTsj\\\\\\\/47U6srVN3I59PQ5T0HZL5wZwx1XSZUw77dQ8UlKlfct0Tt1JxWIe06HP635Kug4Aw++GeW01NBaq0KJaSxhhgULXEsGGRr3EuSoZIVeY2imQPR\\\\\\\/dU5sZq7mq3a1KJC\\\\\\\/JHjPNA\\\\\\\/rh7nO9Fp+Vs9J1eWumWwtcZDnT6ZSvKk0UjOmWmbvUyCNbeHHQ8Cd\\\\\\\/UT5pFk8o3WoGwl1kRCzZYoqTrlDTAC8jsBld1FbuScW+wPKsco3YJKu1xfqdzo7Fpnpc+gaEChcjvlnFxO6YPOu9N4xM\\\\\\\/xeKtgryDbhnKZ6UN03cpgYHbykbZ+7hG2VT3WpVUZGcAZ7Jo95DC8\\\\\\\/4KWjVA1XmrySeCRShWkWuM\\\\\\\/7qO2S5Fd7XwlNSu2Qb5+GcUqI5wgCnnpXDu6Wog6aijDD28fmBPzqecij3LtzVd207Ruw8drhQ5cmCFfZGrKzSsbt7djykjtPaf9wOeLCr\\\\\\\/7Vzqqiu+oAsvrjVvR2I\\\\\\\/7H\\\\\\\/UK0qL2mUodYg0sDAA8TQzTdzmmR9zdztgrXVzhTvKadIVKDX44AwZ8YwPiDiwJosdk8rlelzrrioGS9HYwRvg2efLL9tJPFRcfUghXvWo2Ug7xPKfCNs7qxPmX\\\\\\\/RPA2mGgFkAgSaraYXjoxdMoZ3TzVvEr+fZw\\\\\\\/6RkSLODhXAtpjFiP13mC8nJkBzjUabQiE\\\\\\\/AXrPUuDp+UgC536dzbie8r8dflvPqjCjX7HEgtW8uZ\\\\\\\/HZXTDbMi2W7Y7Wqm5KwhjpfSoSPE9+uFO+jKr6cYEBuOPex6hun\\\\\\\/J5A3PZcU+pZkKrI310ORZHoCkh9tkvOfW7ESmUCmBBbnyFXjeRL4KmsaGcUo6O4xRmaNJVZV1Ox5WnjF0QfcmEIKp9aeBHavnFf8sNDUKy+Xawj0+16pMRE7CZdinu9pzjTG934j18L4LGmmrF6pP4to7E4QGrpioXwLYxOXEG8Xs7KSS1yG7kMsoFooZF3LOmDcvutzu9m+6D\\\\\\\/Nz8sj\\\\\\\/ESS1MD1YIqoDd9f\\\\\\\/QXcKmGy4nvzEc5b7mGnNCUlBTSMNZiQMc8uuU4MuvbyddcjT504PawunM7YigBD8GzqY3w3MUhxM2rhTn\\\\\\\/0YHXZgCAtM7gaZpFOHIpC2CFmJqmoxnLJ40uO\\\\\\\/wrATdNYSFbTv77Pe5cd6G3P8ZshqcG90w5K7JuGHUr\\\\\\\/J8xLCjdRna\\\\\\\/4RIcft\\\\\\\/N7FglEomIpAiuGvOqE1\\\\\\\/B5Jx78bL3jiqeIygUWlQdx6ew+GGeAHsZF4nSxSCaSmaHcdG2qEpo4etuxeQ+MhF7RA4SErHJu6ccJoOChzLSjiqJcVyppP4FqZbI+lB1HjQ\\\\\\\/eZYKlObFCli9UXqtRz8UwRfz3T2XrXlxNtPGYpfCLkDDLCaeXndz84M4EDVAwRPMTJZMbF8nt\\\\\\\/G5lOf\\\\\\\/OgoVXC6ripIfSmq7KYAg3GCukNW\\\\\\\/DoIb0\\\\\\\/wy9EZvIeIt3Y8RZQsgu8H5+jvhcdEFEq2tRcAjMntEYPCc0YM7SM9JCBotMm3U1D8ePNTPW+D8CpoUG5RLr\\\\\\\/sBZL0SElirGHjxte3L3bRNVYcASyAowA4gVSMjiwcNx\\\\\\\/omtbJQ6D8rwiFau9iBNJci6fH37G7Z7dtFaUA+n2PTWYy+CD1Q68n6mrOGF8esx866Ak4WvohyktPP3BmSS2d8WacZVRfbFjAfhSnb+v2BH17xWRYFr7XXZrnTTHGr+0vXvR3gbUzkSEISuVZHUN6O8wrAMTCAxJ+srxLui40ITWeVbImcbM0cz7UI0nxjS282343GoaefD\\\\\\\/VrwAEdec3Jmv1OpMJ5JIIDFnGOygUyoIm6LG0nve45DsrUBIrT6IFi\\\\\\\/3fRroQ7amz8AGbWPC+\\\\\\\/aywGSsR5Kiw8D9vc+x+a5x4xEjuYSqQI\\\\\\\/HJ4vCe1aeFrGY3Cznc18IOW88h3ogGUC+hh3ErW21STeAjAQVKQ9k94pChxV6Elt0kStD2kldCJgfPc+MvKVOklpTjj1fxOh1KpM+u8ZCr\\\\\\\/UxxgeruDSiYI5gVwuXTaXL8ZwfsyXGizGusYvk2UYENrZ5zB02+lixpeYHfB1y+7eyKpnnFhongumhOxEPHMPRz3VTKAR4qGJFCHNHUOB\\\\\\\/sjPHpB5N9Xw22+YYhAN\\\\\\\/iDky+V9mlyBCiY+zrn9wTgZ5ly+67gDwQ3RAtrbaSCF0ZDz5a8Vn\\\\\\\/FCeJv0wIHaTM9xxXC7vtVu4C4XhccgsFReVFI6sSkxy\\\\\\\/1GyHO5f8J1T3w5pUMJaJS6jo6yvTbSwisJoS0BGItR1QekiqmX+tjS3obn0IWfNHJzq7wbzhwVSF6xi+JTjIrjNv\\\\\\\/nqgHlqwHVVQKWe+jlmUMqMLxWMy5w+RnbhDR7lW26rR4\\\\\\\/LRhcEQBGJ3q81j7y930qjPKyPcSCeD1zC+hs45YWHRmcTzdKggiNmVGvrgwnwpsm8HMneTfX1BvKVZm\\\\\\\/NjQyaauIn4xtD\\\\\\\/QkcQMKXClDuuVQZJzOvVj7nKzNaJ1F5h2vcUwb1OVnqnkpPVehQiLowpZxXbSTApqsDKrWjaQS0SA4p\\\\\\\/8Yp5qGliUqwd6C3GbtqFEi4CnkWgjpeeOyuI4o0IEbYLszPWhA6CYY3Lzlbe3tsXR6Lb+75k0CoPaEX4DNEZCS3c3ranUGtMVh0otYwGUXOrNCpS8Lupah97ai9P8E2LfVpjnXgpN+k\\\\\\\/16VWbBfCipUcgb8tS4clca7gJhQd1p3SsWWXH8wim7IbOezsvmyg16RXvUNFpM3YYLtlVr+cskq6UVcTgwEGWfjb8GPi6oBhckeu3sk8K1UGReaLiGuOBSmMbVK4ucT0gRxGnQ\\\\\\\/IN\\\\\\\/pONqffdF7j\\\\\\\/cwm2kqLwti6bjB01t++wIGPaRtqgOcFPCBHQrAmkomQwgfPWcOGg3lB8YecpGkIQWjxkQP9sW+f0Bc\\\\\\\/9tlbGSUCiHNxzpH33q7rosLOiPNywLhR0ZBcfV4JXnnGijwRQcaS1C9N2uSiivPZ0TXUp3PySMNV7d\\\\\\\/sQyBoBx3vOkFqNOe1ohf22cDpqS1iXcOquYZCaRbQ+sPpki5rNE6B6RoPpop0Uh9fn3Ms061TxNZ2ornn+RTNcpFGPkN95uzaF+\\\\\\\/3v\\\\\\\/oOHTSLckMz+APsa4AgatyQbpkGmgGm8Vdab7zUnxzStDbA7cyA34Y15dTKgXCyaWD179Cy77KpOv5M\\\\\\\/UCsGmGw9FTRFw3CeFC+1Rw25rGmIUgw4Ds8WtVFN80Mda6RqQ2EqOLZj9QEEsXp5ImoBWQbj5nr4sD0ZZUwcWplktQXg0iA5QgDfUdTtb+HvCE2kU1MccLqha5i3ieM+FtgbeeJnhaffpxgUpLrGfNUUxyz9Accdaikyy5BwHQMmSGcXS+BABrJyVFEJpaLN2LoTcWPBj8n515CYLN9yom5UhmyCNZI40ixKtM2PRZs+9hDUJCdlbr9hryN9D7S6FxMTv6gxS242eUGP22koKs2L4p2XBcEjQTUk+rLG5KLEO5Iu\\\\\\\/FeRjzRhZXY5vJ\\\\\\\/egRBgkoFnezjGtxMQ4v5gvyalgdgcLmhQIZ1teYqi\\\\\\\/uXoNOz8TEzPbObWqgJaLqau4LgSoqegC95VyQJMwElnY4IHLryMXUC+1PoukVmI6Xx9MeyrRp1P6JVR7bLSqxGhZ2LVPEnPNp2NxkMZyCGh+f4Kkz+nMShQGzSvkCjKxMfappglDu2TQ3uJ5oac2wkae0+CL\\\\\\\/Ckn02MoiHPt71espyI23Sz6l3YECXDuOlPP8GsjjI7AYvDiAhMAm9888lFOQMEMJIETjbis5l59grlblB1mO51MP9rUyfgKlfoQGLYfTeCwqQD89XJYlJn9hFE1rlKVtMNnMcvi6R9hCatz4EPrCJvrFBcqluRGdzsWe73890HKPEcNodqrpETmPkKhXvqLZ92VdR8hfXcLU673e6cSB4i9iCrFULY7J9k0G9bexfwXZ5BFeGscxao212LjYDwiVbajDEivYgRVZIzXUlDIbKNA2C4CURmqojXzYEsYPaUhgTcFVu9n4\\\\\\\/VC6z2aejJzkzdvtM2HEa9gE86S0bLWGiKXbAabN4M3qMZKAcj+u3kURgRQlt\\\\\\\/3QEzh9JQChHl2Jg181YzzkdBTc1SoifHumISHnPczt7Uzes+BCrmPLNxTFkgrjBA4VnCyV7Jg7P4crOTVgA+eQqlfddzVoV8zbMtuzuNh9R6sByEFuTE+JmceTfbYDfGfSx3DQ6j5o7aUkOnSSZLwBC5mkjESQf8eLSCXsjvt16HOoJ8serC1KlS2FJqjg2JsIbIRyxgFQy4D2skuvk53BaMm3DeAkMNnWhyuwVWEoCctL\\\\\\\/RUct2IF8UFlofPZRTXaluipF\\\\\\\/vDRjug9HAtI3LNZb5UgB8eeVzK\\\\\\\/C0OJ+89DtjOUVdn2k1e\\\\\\\/BW2Kt7joa0v2OFngFIim7eLpJrDn675pf5P379u4u2gouK\\\\\\\/8w2PIUMVgXPCVVOOzzl+q5opbTa+rUSGuH7lYzpqXgdyentvPo6vC27+K\\\\\\\/j01Z73U\\\\\\\/jnR0yJA4OS\\\\\\\/4ExcWezJTv\\\\\\\/Nz3Z64M64XCR23NfAI17qqI6hBgA\\\\\\\/ZMltUgjX+lsD8eTSMC3aw34oHKaapqmNyRnp3zidmcxMF1f9HhdXuPTZ+tuCOhbmA4G1p8Ck589W++lwJVUHeBLlNguadSQWxUgRB6naHxhtHfZfRfXOjvnGIejHOygbZr3oR42TvQD+Pxl1992rM04uf7+1f4kKkph0Ux43w\\\\\\\/NT949Xbl6BkTE82Hlt8VBwUxngPCiHNAdC2cNyHDGVMPwqP5151Hg8o+AUitQqzdyboHmI\\\\\\\/eiPsSLdbwtomVdYpxd0+7BGRIklbUVywr+nKpJMg7zrI+e1V+lyC2ahh4j7CrdO048wl2kDVNYdCaIPpnTxKQP3SxGxQ50SkqakNevmGi\\\\\\\/rz1va4EyhmWvP15u5Y16l4Su5nO52bQ2LRxnPOwb1VeYJIDFWrBJRAop5ztlWlt3E\\\\\\\/6ckEI4O7HN9+n3FBTWYRVK16NQ4TKKqKZ9xbYTcGRjLHaIEX6IjsDL+auBy\\\\\\\/4Ow7G93++4eHf40LTLQAtrX9G1T0vXtTkCiiP1EgqA8LGoI+ubvR5ptZ0+uvDk5\\\\\\\/QnOBKJ5ApAl7LQ2Ay+JyGFcz8NaoYzomxw6P1tyttHwjuLPixZOdo0y+rrO4EsTCf9HF9SY3IK8jELK7NoWgI8LBgAiVhpNmi5dm3OB5pJu3pmnUU9h6LTDMZEk\\\\\\\/7i4mTfaQMo7lWHgTQd4Ly14XCFre\\\\\\\/8X5adyQEXRFqzk+mPKDVTMUi292lTRzmtx9HmqCvu2jpflHUOuk11ozC5OCATILZ2LB9ebhPa5nfoaIVZDvPGnIFNyxxolv9FSz91mQ5HNus+JkB+LK0qqtWhtmY5EjcMPlCfsb0O4QJWi9R3Q78U\\\\\\\/Jdr4fIJI55mReev56Y7Wg5uzQjk87r93I\\\\\\\/C1MCEsdXuI4VqWyjN\\\\\\\/no1+ziAyx5tQPRNMWx4OVKGaDMDQt3rMsfnMSUT\\\\\\\/bOGX3xnbtywP4GBYpbBu0J6vP6NOZOwGCHrLHEG6YCO6FFa+NAi252TTn1NXyZY4ShD1\\\\\\\/1ytgv8ej69SyatA9kWIjClL039kOx5iaC\\\\\\\/8l\\\\\\\/Y69Z2qtEkeJryFBLJo0FywGbchqEbpyl26A\\\\\\\/QguL+Y1e\\\\\\\/5Vcmn4TDHq4Tn3\\\\\\\/X+aDEzCJgI8PDMvD+57QaeK4yada3wvWATJ\\\\\\\/36t0jWhq6kRUeFqf9MJprYVfZ3XgIgcRMncx+2gtnMBurqxWufKxiZ62JRl9ZQe4wVFLYZfRN1M5yPzyjGxflNC7rftlfcJ0Bg+PaXKPTjvbcMsO37ESSJ54+8O617jNFK3TqdjM4d6NPtsf\\\\\\\/iIVi31ZZ3U0AQ9OsrejTk6Y+OWA5R1aome2fShcc5BndwkBLskAKwwt4FgSz3EOK\\\\\\\/ZgKAM41vzDQKP7YglOZylSg3zAI+SX0T49MCbqrvrC5iOXFAF4u1nxTJWSjmFnPKtJcWmGGa4duiyQ5r1BjUoPe2XxeEyeCcW5V9XwYMIhwBVtYYCVXEGthktbDrvOx1h7ParEV8M385\\\\\\\/+BWfjXoRAcgPWzH3X4uFHA9VPyAL8AJwVReehkcX6bs2AAOhhhIHWHcVMmhmUc7pvfcunjTHGxT9tO8E3Vgtb4qpNDTmhPnYJEIJ9GGDtKAnVlbkYRvvSoQeAxCe95p21xAQBgflEy3J\\\\\\\/mTDE1kIiwW1E0tE5icJK7TRr+GpQhmhYsKKbZzoP2ck3F9sbCBgHAt0n1\\\\\\\/mrBOwLQfLmWK70IXstT64oNUeZKeg86kJR+G8Wv9KixLBUnSHfQI8mGuiblklQLlzxLTKkRIVEfDJcJeVIuHHDwDae0EWf5LYk0mghTW470S5\\\\\\\/4Ys23yJ6dGf6CEAgrhKV6jsRhm9iDaaQXFvG0abNp4ChtQxlmK1EnoYkxOir62Q4uLIw6wPPr+lF9fWMqJalF8rcB1h2T7HtaGH+VXadrG7Et\\\\\\\/AoIL675aAqUuN5wM5nf1w595+7gu42aXAydJe2ndhScON9qUaGQu9ttW54ADn5Z\\\\\\\/8WSRi0X\\\\\\\/0iV0UU6xHxVHbO27SL4iZ0EQPjmUm3aR+yuxIuupsTjgf5g3\\\\\\\/Pma962Zdqt0IYH0P6JVzfhV4FL+AmDEkMCJiKezJ9PB2YlOWC01cUsEcJl8LxtwFusbUK\\\\\\\/NHTXYnZs2tbLPyojEHgZdtgCFpGYMS0K\\\\\\\/XO5r2KNjZkJaf8krzON5DedWdEc+036DBqRfT1sni8\\\\\\\/sIJKYQeEET7ZhS3DolMVh78pX0l6iqhWW+cu9\\\\\\\/mxZpu3VD4Dn\\\\\\\/atj5X2aauq0T6VzwF4v3BDJr7qQxDKr9v5CL1iM2qS+awjS4qnSw3vEW1w8j5etStnjWFedzoC6ObNg\\\\\\\/B2sTnb49GNqy3FFXw9tLZM6u+zn263Rc3WiMg0mFBbpghBELvMsn5M+itiwS4JzQEKdTUmh6FT15kDJl5QxH\\\\\\\/qS\\\\\\\/n4FQ1wf81RmM+0W4qWHa+CbVGkVUO\\\\\\\/Ko81sgFxY6sx9yJJdYbDgZtjWMUSxSKhroSq0JMKp10PKgolei9fTHhXHrHHE7wZJr9MzxJafpyyKkghD9sduD2OL645t\\\\\\\/6PLk8OB4si3qXcDW0rjV3TBszoFsWatdxnsgD3fzwiKuTYJOYGl9FJfcXud\\\\\\\/+2oGuTCycGwuMgLFWigDhOsuL9cbBA4DeIxBljePNHBpgshI6+tnN+2I4QiuJ0kw1uDfFMFAf\\\\\\\/HuREQ8lZWODKpMkaGJ9XgVTx0ltvUB4fXglqo1JA8XAqMh\\\\\\\/LK+HHq+fGHcZ1WMcVKRm19PeoOIVcqp1BrE7vy4ib0BkwQe0hL76lWVyJw8NVlUx73POeboWi0aaANyNg9pHsQfTkD1fCpZUT4Y+mMYz1OQGUvSCPDONXwPWg5upGC7XL9N+JmCwmGlizipTDCRcaU3YBMdRotyAaAgpUWqOgjx5+3yqnbJkOVJPCFuDZA3CtJsMfUVir4p8vctHn2Jo\\\\\\\/4c9Ea55nQR3AREBBKOPj9ZN1knE8vwVUImSXVTaWHmuNLNol0iZTKtHoTemKnLUvccWKTdD399Etz2FNQUssUbwxLRo3DS5XtbthjPn9IC2EDsry20KT9rrQBbP151iBXwWNLyqEIb+H8rw8q+8lfOH0QwptAXxsS6odR3uGY6Wkvvqiqv2BBn0UwEx4STmSqS0za5wIIXLcJgDumsVtLuiA3ylY211QvTAmLRb0rBb3QlOmvjCgk4gNa\\\\\\\/S+1r2OHpCVra6ygtRkacRu7M5nskO9VmE\\\\\\\/UJjXKmSESTLYSNLGYn7RtmfwU9PVL+Qa2ICgxYFFXDTtfrZ2WOAVGd3730aaZAff4s+W5uNWTgRX\\\\\\\/xE05kuYhu3O7+E9kFM4pZhF6pslm14h6USUG8jXaiRj5UZGDF6VwkSuLoXl9m52ct49a\\\\\\\/uAXW7tiSz1rrkrhng24BnLxs+HNsrAUEGCd1TyTIaBNIzzBj5tvkNpBmHewsOYvtx7QwhtsgnfTU7CknMrelc57q72GQM7QnlpCb0zhojgskmUohNxcpJmdYbUT2fTEgln6MDnjqh2skjVA5w6lie8EetSlMoHYcU703oUNymHZ6vS5m7KDukHtZLebicEQzkfJsBcwIuLQtMVQ3f6o0bhZTF8Riqx9YrGDcAnXKLhGlMrSw1i+zqsP\\\\\\\/EbnZbi9nuzXAFKmMgSEHVaWmn1yht8vQ1ZsZXv1d3N3bd\\\\\\\/0\\\\\\\/4OkVdGiuCVJ4kYHi54KobE+z2nC+WhFy+zUgNxzZpV\\\\\\\/DunQTg1oWEe+kr+vj\\\\\\\/eU8vbN7+CSwhmf\\\\\\\/yCg82nGT\\\\\\\/qvBzBBuRg38VSN\\\\\\\/0NLOBAuuwiUeoAbna8tOgxinJcMM+Jcp1Yo1yN6f1XkdFzsQAldMHNtm\\\\\\\/Y6oPDZOAqnAff4KjkiNhZv\\\\\\\/vCzxlegbGRPw2mJI9JoP3agoEA+lP8jAQWwKIINvffb8hqBdOtLiFBRJJWmgn282KO5lqderyo5Erfc1H69klHjE4x3CpYpNzMJ+iEe6lgIy3AUaCoyX0pnQ\\\\\\\/CvrULPWNcpiyRcYjUsU85PgirvS4OFYxKf2KdxOo1Z0dq7UJI1xD4k\\\\\\\/WKQ47cEChuVVvr53aRdf3romiGn0Oc4ISASiOMNZNTmSwm52gY2FD88EXAtzqzOtHzRwJOpBBORgOKVgdebiNuVfeVeLFBoIrLk3cO8x1BZwXVdm0ru8r0a1KybwxDeoUKcqxe\\\\\\\/rXzXbI7OSLPF58jFUIkdhCM1xXL5rUB8lV\\\\\\\/IQyx09a08nqJWAyOAnajRXb3Ypf3rdFdomCc9WpKa4T3UGGFf\\\\\\\/spjY5+etNzED3xe3LSZQAQgvfbeQAZZZUV0PjACFaJ3ifcXcLMFfUXDUbNb7fYc+AsZngJHfNCLj0rvNdNpDIp7w0Tov3k5BewTR7dr+5JYeptMHV5ocq6qsonN+Vwph\\\\\\\/wxfde7+iOsLYn+CF36toKI2r8mSyBEWce3egrHFCe6LcZ+CeRJWlVSOapsMYHw1qPFNyvkJQiUNWDBcV2FkTNhJsDzeiN68t5d0o7liBsza3SdJRYHJdoPmAZnt3f2Ws6HZ\\\\\\\/Nh6qIKkQd5tzl5ZmZ1yxZX6e9znGt8bkczN1wB71Jeowkm\\\\\\\/bMewQGmrXCWubxLbpk3LeY7sP38Vvk\\\\\\\/g+hL2HtmkbF3QD7YIl0QwcUn9rg8KRZtmHL29y9r5CxqVc7JJ1jbUwwC\\\\\\\/3j\\\\\\\/GWvcxYD673IlE0gCGRWPKvuFZI4gC2eQhRstMmmQKBx5XFDbvBetHpixfq3+cazv2vJVKAV1W6mcvPDTxNCbE4BG3wYlljfIU7I1zvFxlojQvt38d83BwgfkMpwkykGVytsFIsbMauQjuC4VPfq0fHDEaPQWDECEanNKQp3bBB5SYJ0pnw6+NQbXf\\\\\\\/OXHfL8Ogozq62XdFJGe5nJTDnF4TUPOU4m6TDycysQIIm6x9PbJc88TBiXMmAZFHQpV438M8257ingjYZeE1orglUOT+9Ubx6oW5KggAENFITwHviCcMUivSyoNY3i+qre4+mV5jQrrCYKiW1rhqOpEDTFzLVr8OxWnExvhyAp8j258p1rGtKjwUQLU2n+jgESVXCYCPPU2jB4GouAawmwbhqk6HDB6erNK\\\\\\\/l6iKgHo2lGdrYkfbY5Z+Cu3OCtjcpQHB6iSRann7jmFz34UQClAo8rVrGlGUOYYiYczVOKAzz5rPGAUm5AQucAeT2lhr7VDG6SeTpmASEl+nchIo+EGmXgAWAV3eAgnKO1f\\\u0022,\\\u0022iv\\\u0022:\\\u0022s\\\\\\\/z\\\\\\\/Ed\\\\\\\/ZIgZwXOejUDNT7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002246d088b21e9ac79a534ffcae8611bf96a3ab0cabce11aa121e3d54a350f5b909\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HcfD9VBbV74my5pgyM2CxDpN7N572lYVaThBPBmjQ3X5ordNe6cDDxqzPAaENl3vFtkpnkSGXMupsKD8knS5cShW3HLHB1cPW0MST\\\\\\\/IIYn4ldOd\\\\\\\/ZKN\\\\\\\/7ODPnbycr6bu326Wo+LNRHaCwHpCDfyfF0v1jv7RgGBOd6vgHpSBqj4OriyBrfDNtLvfxpBM35Aqc7Jpsh0LvVyCZ0Zojz18\\\\\\\/PQUihbOclFAtvYmJEyZmDvJcZKfp8srck6Pb+Z+vhXrK\\\\\\\/2AHL1c2zfEXWKaZuaGK\\\\\\\/dHFXW\\\\\\\/zxtYQFJdx52F4bGNQbuSAklkz\\\\\\\/CqaT4WAmof10Oso6JoXBVvrpMxUgbG+eiYYEOUSF5szolUROjWusbL6OCW0YQWhVlJQepiE3ea9eUZF1nTgJRcjKcAxZxHX0r4XzlIK1QKh6dByN34O2f9acmpDkowT6MtcU8ToytiSgUEpndUyiCOTuI7jWTrrzsplT84KeMQqXNfA08LocFw1CdkoisI7OgkUHbvjyBHGpGxh1d5TMqomyU94c8FGmqI8vWQUsasyj7Ln4djh2wdtg0qDufvtMPiFRpgKSSmtLjyZKWyTAWoWlFIQG4Y8XdVeSS8lLhEFHtMlZPjaTemGqpQraUbG9T2PzCXU9F5xOeCON64Ba8WS5hfaspBwrQJJy2ynmmtLsL22cAq6AS9yHo4Qe4DzRcDjzjHyxh8Vu9Wptzihut2wUx7KVvxTuA7IG3S7ImaIPZcZKewz0dIm7+v8UUHl2F032RjHrB+SAFF99G\\\\\\\/tRy1ou2SYyR4wMOJ5mknxWKVaWonsYGWki2z\\\\\\\/v25M9tr9bTuoUMTjA9Gn4RCH4MX96T52dG5aeggxmbnbRtjncSz+qxlbtdTHZq9YQzdYzWO0fuc\\\\\\\/Cnk2zpx1+r7\\\\\\\/CimyeeCfcVN6QY\\\\\\\/C+8jGr0KRlq5o1yQG2x8oCN97Gm\\\\\\\/9edZmwCF\\\\\\\/S77ycN0Vk4K65vO9r\\\\\\\/sBsureOr7pYeWufZFoPh3iqni0fEUE4aRB5TU+I84A3nzUPXehYz4HUVipR3GIKddbVekOz41JasnYRD176LH8SPkPX+2bdoAwk\\\\\\\/AD\\\\\\\/3l3DZlWaAV7F+2uvkdfjQ8J1XA98aPYSvoJFIavCSxpgWJStW1x6uhJPzDhCYyk3aWr+\\\\\\\/ssqFJCvU16U0iJNwfuGvkba7Zcx9DQJA980e5s9ZkfXv4A1cgTPjcuRAiUXMjmwZSjaNRiyh4\\\\\\\/bhkMBLUzI1\\\\\\\/\\\\\\\/KatnAOgb\\\\\\\/TtKOnSYjFzRY92qrAZTu4H0FairsB4ArTZlFSUN9bEfYnJfKILQd2jg1\\\\\\\/fLJoIsUJBYdcSX\\\\\\\/Uk9GDCXz6Y+APeHUUd5f\\\\\\\/tSbOi0xZlIDdOX3YpvO8KmaVmjmKNMPnX\\\\\\\/ukOiv4bhEBKS8fvIoQ7C4gwNmqqdVRM0VhTU8txPoHVxlSL51iEtKNQ63HrQnd4Di9a04Xh6uvNC6a8XuPy8iR1g0RHfe+leHk6qiMMN3GC\\\\\\\/Nyc3aDwX8b3MJN3mzcl9+dpjAK2zJn27QKgm8+kY3zJfH36NFGxtPJbu9sLGDyS4kxlUJlv8GNGOTFKmhD5XaoZYuFLx58HVA5QlrFMYs8gddwCHFNdOdBoGDVS3wUk9ZG4tTRGOHEMYtSkvbCd3ScO13hKxv7oq0ACwXUYc02Hptk0ruIs66uaC\\\\\\\/tv2MxKoiu33HV+B1\\\\\\\/LRO6QU\\\\\\\/t2TFs9iINU2EXy85Xr56L0DrTK19n00ZCXiptSwQVKqVO16Sn0lVsVmr70JKXncJfuuKMgdTDYWPf4nFMBCKcgfdQtjYdqPJLc\\\\\\\/xVbgFzM+9UyLsU5DHTKZIll46jdqJGLMLSzygZGkAUch0784epPepAjQAjw+VbVGzvyuqQ7QEkaiOhAgUGjQkxn3T88UyFk\\\\\\\/PipMLt3J\\\\\\\/hnm2zB6e5T7mESNafYlv18KLn1o9ruIifOZX\\\\\\\/oxBXpwHxRlhxr0FNMBpBtavId8RrLEBTXfcXub83F\\\\\\\/XGEmqugDzp9mL3\\\\\\\/yuOZf58JqUU+bmpkw3aw2DdjIbwg9mmcmQkKIJm5isz4FQo=\\\u0022,\\\u0022iv\\\u0022:\\\u00221ngpIAKccJVdRo\\\\\\\/t539+ew==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e99e500689ca95dc6a784791c533c20f4ee31ba1de6cb238015085859bb85e5f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460542 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/143" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GTPj0IRXjdbaWRs4Miv9gTjmdwugq_Iuj-_0fViFlEA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/143" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8_l_cAEZNG-8MD9vNq7avatXoOCCTgjBWyeWMhO_ANs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/129" title="The Neurospora Transcription Factor ADV-1 Transduces Light Signals and Temporal Information to Control Rhythmic Expression of Genes Involved in Cell Fusion" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/155" title="The Physical Genome Mapping of Anopheles albimanus Corrected Scaffold Misassemblies and Identified Interarm Rearrangements in Genus Anopheles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460542" id="node460542" data-pisa="ggg;7/1/143" data-pisa-master="ggg;g3.116.034207" data-apath="/ggg/7/1/143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0836-2458" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mia Jaffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1692-4983" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gavin Sherlock</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0923-1636" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sasha F. Levy</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">143-153;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034207 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460542" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mia Jaffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford School of Medicine, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mia%2BJaffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaffe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMia%2BJaffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0836-2458" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mia Jaffe</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gavin Sherlock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford School of Medicine, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gavin%2BSherlock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherlock%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGavin%2BSherlock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1692-4983" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gavin Sherlock</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sasha F. Levy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Laufer Center for Physical and Quantitative Biology, Stony Brook University, New York 11794</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biochemistry & Cell Biology, Stony Brook University, New York 11794</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sasha%2BF.%2BLevy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Levy%20SF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASasha%2BF.%2BLevy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0923-1636" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sasha F. Levy</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sasha.levy@stonybrook.edu" class="" data-icon-position="" data-hide-link-title="0">sasha.levy@stonybrook.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/143" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460542" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/143.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460542" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/143.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460542" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/143.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460542" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/143.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460542" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Systematic screens for genetic interactions are a cornerstone of both network and systems biology. However, most screens have been limited to characterizing interaction networks in a single environment. Moving beyond this static view of the cell requires a major technological advance to increase the throughput and ease of replication in these assays. Here, we introduce iSeq—a platform to build large double barcode libraries and rapidly assay genetic interactions across environments. We use iSeq in yeast to measure fitness in three conditions of nearly 400 clonal strains, representing 45 possible single or double gene deletions, including multiple replicate strains per genotype. We show that iSeq fitness and interaction scores are highly reproducible for the same clonal strain across replicate cultures. However, consistent with previous work, we find that replicates with the same putative genotype have highly variable genetic interaction scores. By whole-genome sequencing 102 of our strains, we find that segregating variation and <em>de novo</em> mutations, including aneuploidy, occur frequently during strain construction, and can have large effects on genetic interaction scores. Additionally, we uncover several new environment-dependent genetic interactions, suggesting that barcode-based genetic interaction assays have the potential to significantly expand our knowledge of genetic interaction networks.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow">Saccharomyces cerevisiae</a></li><li class="kwd"><a href="/keyword/dna-barcoding" class="hw-term hw-article-keyword hw-article-keyword-dna-barcoding" rel="nofollow">DNA barcoding</a></li><li class="kwd"><a href="/keyword/genetic-interactions" class="hw-term hw-article-keyword hw-article-keyword-genetic-interactions" rel="nofollow">genetic interactions</a></li><li class="kwd"><a href="/keyword/systems-biology" class="hw-term hw-article-keyword hw-article-keyword-systems-biology" rel="nofollow">systems biology</a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">whole genome sequencing</a></li></ul><p id="p-3">Genetic interaction (GI) studies are vital for our understanding of cellular wiring, and have contributed insights into biological function, genetic robustness, and network topology, and ultimately may help explain the heritability of complex traits and define new drug targets (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tong <em>et al.</em> 2001</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ooi <em>et al.</em> 2003</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wong <em>et al.</em> 2004</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Ye <em>et al.</em> 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kelley and Ideker 2005</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davierwala <em>et al.</em> 2005</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert 2005</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Schuldiner <em>et al.</em> 2005</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pan <em>et al.</em> 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lehner <em>et al.</em> 2006</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">St Onge <em>et al.</em> 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Collins <em>et al.</em> 2007</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Musso <em>et al.</em> 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Roguev <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Costanzo <em>et al.</em> 2010</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2016</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zuk <em>et al.</em> 2012</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dutkowski <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Martin <em>et al.</em> 2015</a>). A GI between two mutations is defined as a deviation in the double mutant fitness from the multiplicative fitness of the two corresponding single mutants (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Phillips <em>et al.</em> 2000</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Segrè <em>et al.</em> 2005</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">St Onge <em>et al.</em> 2007</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Costanzo <em>et al.</em> 2010</a>). The current systematic and genome-wide GI studies performed in the eukaryotic model <em>Saccharomyces cerevisiae</em> are rooted in early synthetic lethal screens (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bender and Pringle 1991</a>). These systematic studies—<em>e.g.</em>, SGA (synthetic genetic array; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Tong <em>et al.</em> 2001</a>), dSLAM (heterozygote diploid-based synthetic lethality analyzed by microarray; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pan <em>et al.</em> 2004</a>), E-MAPs (epistatic miniarrays; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Collins <em>et al.</em> 2006</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Schuldiner <em>et al.</em> 2006</a>), and GIM (GI mapping; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Decourty <em>et al.</em> 2008</a>)—were enabled by the yeast deletion collection (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Winzeler <em>et al.</em> 1999</a>), and measure interactions between complete gene knockouts. Together, these techniques have cataloged &gt; 200,000 GIs between the nearly 6000 genes in <em>S. cerevisiae</em>, with the majority being measured by SGA under a single standard growth condition (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Stark <em>et al.</em> 2006</a>).</p><p id="p-4">Despite these advances, continued progress toward a complete and accurate genetic interactome faces several challenges. One is the relative paucity of GI data measured in nonstandard growth conditions. To date, only a handful of genetic interactions have been characterized across different environmental conditions (<em>i.e.</em>, differential interactions). Yet, these studies have yielded additional biological discoveries including new genes involved in DNA repair (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bandyopadhyay <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guénolé <em>et al.</em> 2013</a>) and cell wall integrity (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Martin <em>et al.</em> 2015</a>). Some evidence suggests that environment-dependent GIs constitute a sizable fraction of all GIs. For example, a study across &gt;1000 environments found that ∼2/3 of single mutant fitness defects are not observed on the standard yeast medium, YPD (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hillenmeyer <em>et al.</em> 2008</a>). Furthermore, systems-level flux balance analysis predicts a similar fraction of GIs would be undetectable in YPD (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Harrison <em>et al.</em> 2007</a>). Thus, environment-dependent GIs likely remain largely undiscovered. Also unknown is how GIs change across environments, a property that could be fundamental for descriptions of cellular physiology, or for assigning biological function to uncharacterized proteins.</p><p id="p-5">A second challenge is the relative difficulty of performing experimental replicates using current GI screening technologies. Because double deletion strains must be rearrayed, or replica plated, for each experimental replicate, screens typically generate only two to four measurements per genotype. Correlation values reported between replicate screens range from 0.2 to 0.8 (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Schuldiner <em>et al.</em> 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Costanzo <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dodgson <em>et al.</em> 2016</a>). Screens with higher correlations (<em>e.g.</em>, ∼0.8 in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010</a>) generally measure the same double deletion segregant pool in duplicate, whereas lower correlations are observed in screens that compare double deletion segregant pools that have been constructed independently (<em>e.g.</em>, comparisons between double deletions in which the selectable markers at the two deletion loci are swapped). It has been suggested that there is segregating genetic variation underlying these observations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jasnos and Korona 2007</a>), but data in support of this claim have been lacking. Furthermore, comparison of results between different studies can be confounded due to factors such as the low reproducibility of single deletion fitness estimates (<em>r</em><sup>2</sup> = 0.14–0.7 when comparing seven studies, <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Baryshnikova <em>et al.</em> 2010</a>), the definition of a GI itself (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mani <em>et al.</em> 2008</a>), and the fitness differences that are observed on solid media <em>vs.</em> those measured in liquid (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Musso <em>et al.</em> 2008</a>).</p><p id="p-6">The challenges above highlight two major limitations of current GI technologies. First, each strain that tests an individual GI must remain physically isolated from other strains, making it impractical to construct and store large GI libraries that contain many replicates per double mutant for retesting across multiple growth environments. Second, double deletions are typically constructed by sporulating small pools of diploids, and selecting for all cells that pass selection, resulting in an unknown number individual haploid segregants in each double deletion strain. Thus, it remains unknown how much genetic variation exists between clonal segregants with the same presumptive genotype derived from either the same or different crosses, some of which could be contributing to variability in fitness measurements and interaction scores.</p><p id="p-7">Technologies that rely on recognizing or sequencing DNA barcodes might be more suited to testing GIs at high replication and across different experimental conditions, because individual strains can be stored and tested as pools (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Pan <em>et al.</em> 2004</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Decourty <em>et al.</em> 2008</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Smith <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Robinson <em>et al.</em> 2014</a>). Indeed, we have recently shown that half a million barcodes can be tracked simultaneously by barcode sequencing (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Levy <em>et al.</em> 2015</a>). Each strain in the yeast gene deletion collection contains a pair of unique barcodes at the deletion locus (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Winzeler <em>et al.</em> 1999</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Giaever <em>et al.</em> 2002</a>), which can be used to measure relative fitness across conditions by monitoring the relative frequency of each barcode in a mixed pool by microarrays or amplicon sequencing (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Pan <em>et al.</em> 2004</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Decourty <em>et al.</em> 2008</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Smith <em>et al.</em> 2009</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2011</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Robinson <em>et al.</em> 2014</a>). In some applications of this approach, a second deletion is introduced into the barcoded deletion library either by mating with, or transformation of, a common deletion construct, with all resulting pairwise combinations assayed in a single pool (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Pan <em>et al.</em> 2004</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Decourty <em>et al.</em> 2008</a>). In theory, barcoded double deletion pools such as these could be used to examine genetic heterogeneity within a double deletion by subjecting them to severe bottlenecks such that each barcode in a post-bottleneck pool derives from a single segregant. Repeated bottlenecks would likely select different segregant representatives for each barcode, allowing one to assay the impact of genetic heterogeneity on each GI. Once a pool is constructed, it can be screened repeatedly in multiple environments to quickly and cheaply detect dynamic GIs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darby <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2012</a>). However, it is difficult to apply these methods more widely, assaying deletion combinations beyond a handful of query genes against a large array of target genes.</p><p id="p-8">One potential solution would be to barcode both the query and the target strains and use <em>double barcodes</em> to uniquely identify each double deletion combination. The major technical challenge to this approach lies in identifying which two barcodes reside within the same cell. If barcodes are located in different locations in the genome, then performing a pooled DNA preparation, or a pooled PCR of barcode libraries will destroy the association of the two barcodes that exist within the same cell. One possible solution is to use a PCR protocol that stitches the two barcode PCR products into a single amplicon before sequencing (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yu <em>et al.</em> 2011</a>). However, individual genotypes must be PCRd independently in multiwell plates prior to sequencing. Emulsion PCR (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Margulies <em>et al.</em> 2005</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Williams <em>et al.</em> 2006</a>), whereby barcode amplification of DNA from individual cells can occur in small water droplets in oil, could be combined with stitched PCR to reduce costs and improve throughput: both plastic consumables and PCR reagents would be minimized. However, it is currently unclear if emulsion PCR can be effective for large numbers of droplets within which PCR products can be stitched together for subsequent sequencing.</p><p id="p-9">Here, we introduce a novel <strong><span class="underline">i</span></strong>nteraction <strong><span class="underline">Seq</span></strong>uencing platform (iSeq), and apply it to measuring GIs. The key innovation of iSeq is a system that recombines two barcodes that exist on homologous chromosomes such that they are brought into close proximity on the <em>same</em> physical chromosome <em>in vivo</em> to form a double barcode (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). iSeq accurately assays the fitness of each uniquely marked strain in the pool by monitoring double-barcode frequencies over several growth bottleneck cycles using a quantitative double-barcode amplicon sequencing and counting protocol. In this pilot study, we demonstrate the utility of iSeq, by using it to measure the GIs between all pairwise combinations of nine deletions across three environments at high replication. For any given clonally derived double-barcode strain, we show that fitness measurements and iSeq interaction scores are highly reproducible across biological replicates, and find several new environment-dependent GIs. However, we find low reproducibility between different double-barcode strains ostensibly carrying the same double deletions, which cannot be explained by measurement error. By whole-genome sequencing 102 of our experimental strains, we find that segregating variation and <em>de novo</em> mutations that occur during strain construction can have large effects on GI scores, suggesting that validation of GIs may require multiple replicate measurements.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/143/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The iSeq platform. (A) Schematic of the iSeq barcode locus before and after Cre-mediated recombination. Two complementary barcode constructs are introduced to the same cell on homologous chromosomes via mating. Galactose-induced Cre recombination results in the two barcodes being on the same physical chromosome. Recombination events are selected for via a split URA3 marker that is only functional after recombination. (B) First set of crosses to generate F1 strains. Two versions of each of the listed systematic deletion strains (NatMX and KanMX) are each mated to two strains with unique iSeq-compatible barcode constructs. The magic marker system is used to select for haploids of a specific mating type that contain a gene deletion and an iSeq barcode. (C) Second set of crosses to generate F2 experimental strains. All pairwise combinations of barcoded deletion strains are next mated together, recombination at the barcode locus is induced, and double-barcode double-deletion haploids are selected following sporulation. (D) Histograms of experimental replication. For our pilot of nine genes, 12&#x2013;16 uniquely double barcoded strains were constructed for each of the nine possible single gene deletions (turquoise), and four to eight strains were constructed for each of the 36 possible double gene deletions (pink)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-176927663" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The iSeq platform. (A) Schematic of the iSeq barcode locus before and after Cre-mediated recombination. Two complementary barcode constructs are introduced to the same cell on homologous chromosomes via mating. Galactose-induced Cre recombination results in the two barcodes being on the same physical chromosome. Recombination events are selected for via a split &lt;em&gt;URA3&lt;/em&gt; marker that is only functional after recombination. (B) First set of crosses to generate F1 strains. Two versions of each of the listed systematic deletion strains (&lt;em&gt;NatMX&lt;/em&gt; and &lt;em&gt;KanMX&lt;/em&gt;) are each mated to two strains with unique iSeq-compatible barcode constructs. The magic marker system is used to select for haploids of a specific mating type that contain a gene deletion and an iSeq barcode. (C) Second set of crosses to generate F2 experimental strains. All pairwise combinations of barcoded deletion strains are next mated together, recombination at the barcode locus is induced, and double-barcode double-deletion haploids are selected following sporulation. (D) Histograms of experimental replication. For our pilot of nine genes, 12&#x2013;16 uniquely double barcoded strains were constructed for each of the nine possible single gene deletions (turquoise), and four to eight strains were constructed for each of the 36 possible double gene deletions (pink).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/143/F1.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/1/143/F1.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/143/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/143/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470075" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">The iSeq platform. (A) Schematic of the iSeq barcode locus before and after Cre-mediated recombination. Two complementary barcode constructs are introduced to the same cell on homologous chromosomes via mating. Galactose-induced Cre recombination results in the two barcodes being on the same physical chromosome. Recombination events are selected for via a split <em>URA3</em> marker that is only functional after recombination. (B) First set of crosses to generate F1 strains. Two versions of each of the listed systematic deletion strains (<em>NatMX</em> and <em>KanMX</em>) are each mated to two strains with unique iSeq-compatible barcode constructs. The magic marker system is used to select for haploids of a specific mating type that contain a gene deletion and an iSeq barcode. (C) Second set of crosses to generate F2 experimental strains. All pairwise combinations of barcoded deletion strains are next mated together, recombination at the barcode locus is induced, and double-barcode double-deletion haploids are selected following sporulation. (D) Histograms of experimental replication. For our pilot of nine genes, 12–16 uniquely double barcoded strains were constructed for each of the nine possible single gene deletions (turquoise), and four to eight strains were constructed for each of the 36 possible double gene deletions (pink).</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-11">Extended <em>Materials and Methods</em> and a detailed protocol for iSeq can be found in Supplemental Material, <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FileS1.pdf">File S1</a>.</p><div id="sec-2" class="subsection"><h3>Yeast barcode library construction</h3><p id="p-12">Two complementary barcode libraries, consisting of 288 clones each, were generated in a MATalpha starting strain derived from BY4742 (<em>MAT</em>α <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>Δ0 <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a></em>Δ0 <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a></em>Δ1 <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a></em>Δ0) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brachmann <em>et al.</em> 1998</a>). This starting strain also carries the magic marker construct (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tong <em>et al.</em> 2004</a>), which allows for selection of either <em>MATa</em> or <em>MAT</em>α haploids via growth on synthetic complete (SC) medium containing canavanine and lacking either histidine or leucine, respectively. The barcode construct in each strain of each library sits at the dubious open reading frame (ORF) <a href="http://www.yeastgenome.org/locus/S000000413/overview">YBR209W</a>, and consists of a DNA barcode with 20 random nucleotides, a <em>HygMX</em> selectable marker, and either the 5′ half of the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> selectable marker and lox71 in the 5′ library, or the 3′ half of the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> selectable marker and lox66 in the 3′ library.</p></div><div id="sec-3" class="subsection"><h3>Double-barcoded double-deletion yeast strain generation</h3><p id="p-13">Haploid gene deletion strains, carrying either <em>KanMX</em> or <em>NatMX</em> marked deletions, were derived from the diploid heterozygous deletion collection (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Tong <em>et al.</em> 2001</a>; <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Pan <em>et al.</em> 2004</a>) for the following genes and dubious ORFs: <em><a href="http://www.yeastgenome.org/locus/S000004075/overview">ARP6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004876/overview">SAP30</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001346/overview">SDS3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005041/overview">PHO23</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005364/overview">SIN3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005838/overview">DGK1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000629/overview">SNT1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000011/overview">DEP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005274/overview">RPD3</a></em>, <a href="http://www.yeastgenome.org/locus/S000001137/overview">YHR095W</a>, and <a href="http://www.yeastgenome.org/locus/S000001950/overview">YFR054C</a>. Each of the 11 deletion strains marked with <em>KanMX</em> was mated to two unique strains from the 5′-barcode-construct-carrying yeast library. <em>NatMX</em> marked deletion strains were each mated to two strains from the 3′-barcode-construct-carrying yeast library. Resulting diploid strains from each cross, and carrying a deletion and the barcode construct, were sporulated and plated for haploid single colonies.</p><p id="p-14">To obtain strains carrying two gene deletions and both complementary barcode constructs, all pairwise combinations of singly barcoded deletion strain were mated. In each resulting diploid, Cre-mediated recombination was induced at the barcode locus by growing on SC + 2% Galactose –Ura at 30° for 2 d. Cells were sporulated, and unsporulated diploids were digested using zymolyase as described (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Herman and Rine 1997</a>) before selecting single haploid colonies.</p></div><div id="sec-4" class="subsection"><h3>Pooled growth</h3><p id="p-15">The 393 barcoded single and double gene deletion strains were frogged from frozen glycerol stocks to 1 ml liquid YPD in 2 ml 96-well plates, and placed at 30°. After 3 d of growth, all strains were pooled, glycerol was added to a final concentration of 17%, and aliquots were stored at −80° for future inoculations. The eight barcoded wild type (WT) control strains, generated from the matings of two dubious ORF barcoded deletion strains, were grown overnight in liquid YPD, pooled, glycerol added, and aliquots were stored at −80° for future inoculations.</p><p id="p-16">The pooled fitness assay was carried out in three growth conditions: YPD, YPD 37° and YPEG (YP + 2% EtOH, 2% glycerol). The alternate conditions were chosen because in the <em>Saccharomyces</em> Genome Database, seven of nine of the single gene deletions are annotated as heat sensitive, and four of nine have decreased respiratory growth.</p><p id="p-17">For pooled growth fitness estimates, the double-barcoded WT and double-barcoded mutant pools were mixed at a 50:50 cellular ratio. For YPD, YPD 37°, and YPEG cultures, 1.5625 × 10<sup>9</sup>, 6.25 × 10<sup>8</sup>, 6.78 × 10<sup>9</sup> cells of this mixture, respectively, were used to inoculate 100 ml liquid of medium in a 500 ml flask, in triplicate. The cells were cultured shaking at 230 rpm at 30° or 37°. Every 24 hr, for a total of eight time points, 12.5 ml culture were transferred to 87.5 ml fresh medium, and placed back in the incubator. At each transfer, the remaining overnight cultures were split into two 50 ml tubes, spun down and resuspended in a 5 ml solution of 0.9 M sorbitol, 0.1 M EDTA, 0.1 M Tris-HCl, pH 7.5 for DNA extractions.</p></div><div id="sec-5" class="subsection"><h3>Barcode sequencing</h3><p id="p-18">Barcode sequencing was performed as previously described (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Levy <em>et al.</em> 2015</a>). Briefly, genomic DNA was extracted by spooling. A two-step PCR was carried out on 14.4 μg genomic DNA to amplify the barcoded region, add multiplexing tags, and add Illumina paired-end sequencing adaptors. Four initial time points were pooled and sequenced on the Illumina MiSeq. Remaining libraries were pooled and paired-end sequencing was performed over four lanes on the Illumina HiSeq 2000 (10, 11, 20, and 23 libraries per lane). Additionally, 21 libraries were resequenced on one lane on Illumina HiSeq 2000 to test for sequencing noise.</p><p id="p-19">Custom Python scripts were used to demultiplex the time points from the Illumina data, and to determine the number of reads matching each known double barcode in the pool at each time point. The observed number of counts for each strain, at each time point, in each experiment, is available in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S5.xlsx">Table S5</a>.</p></div><div id="sec-6" class="subsection"><h3>Fitness and GI estimates</h3><p id="p-20">To estimate the fitness of each strain in the pool, barcode counts at each of the first four time points were normalized for each strain by first dividing by the total number of counts at that time point to get a relative frequency. These frequencies were then normalized to the change in WT frequency, and then subsequently divided by the relative frequency at the first time point. After taking the natural logarithm of each of these normalized frequencies, a least squares linear regression was fit using the lm function in R, using a predefined intercept of 0. The fitness estimate for each strain was then defined as 1 + <em>m</em>, where <em>m</em> is the slope of the fitted line.</p><p id="p-21">To estimate quantitative genetic interaction scores, we calculated the deviation, ε, of the observed fitness of each double mutant strain (<em>f<sub>ij</sub></em>) in the pool from the expected fitness, based on the product of the observed fitness of the single mutant strains, <em>f<sub>i</sub></em> and <em>f<sub>j</sub></em>, as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/143/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/143/embed/mml-math-1.gif"/></noscript></span></span>Fitness and interaction score estimates for each experimental strain across each replicate can be found in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S2.xlsx">Table S2</a>. To call interaction scores as significantly positive or negative, a 95% confidence interval (C.I.) was calculated around the mean score from the four to eight strains with identical pairs of gene deletions. CIs for each gene pair across each condition can be found in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S4.xlsx">Table S4</a>.</p></div><div id="sec-7" class="subsection"><h3>Optical density fitness estimates</h3><p id="p-22">A total of 393 barcoded strains were streaked for single colonies on YPD. A single colony was used to inoculate a 2 ml overnight YPD culture. For three replicates of each strain, 2 μl of this overnight culture was used to inoculate 98 μl YPD in a 96-well plate. This plate was placed in the TECAN (GENios), and optical density (OD) at 595 nm (OD<sub>595</sub>) was taken every 15 min for 90 cycles, or 180 cycles for exceptionally slow growing strains.</p><p id="p-23">To estimate fitness of each strain, the region of the curve during exponential growth was found for each strain by fitting a linear regression to each window of 10 time points, across all 90 total time points (90 total windows). This windowing method was employed to adjust for the fact that not all strains started at the same OD, and to avoid choosing arbitrary threshold values within which to calculate the doubling time. The fitted line corresponding to the window with the maximum slope, and therefore maximum growth rate, was used to calculate a doubling time for each strain. Fitness estimates were calculated by dividing the doubling time of a WT strain (generated above) that was included on the plate by the doubling time of the experimental strain (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">St Onge <em>et al.</em> 2007</a>). Observed doubling times and fitness estimates for each strain are available in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S2.xlsx">Table S2</a> and <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S5.xlsx">Table S5</a>.</p></div><div id="sec-8" class="subsection"><h3>Whole-genome sequencing</h3><p id="p-24">Strains were streaked for single colonies from frozen stocks, and grown up overnight in YPD at 30°. Genomic DNA was isolated with the YeaStar Genomic DNA Kit (Zymo Research). Libraries for Illumina sequencing were constructed in 96-well format as previously described (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kryazhimskiy <em>et al.</em> 2014</a>), pooled and analyzed for quality using Bioanalyzer (Agilent Technologies) and Qubit (Life Technologies), and sequenced on one lane of Illumina HiSeq 2000. Reads were trimmed for adaptors, quality and minimum length with cutadapt 1.7.1 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Martin 2011</a>). Reads were mapped to the reference genome with BWA version 0.7.10-r789 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Durbin 2009</a>). And variants were called with GATK’s Unified Genotyper v.3.3.0 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">McKenna <em>et al.</em> 2010</a>). Significant changes in copy number were discovered using the CNV-Seq package (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Xie and Tammi 2009</a>). SIFT was used to predict the protein function tolerance of amino acid changes resulting from SNPs verified by visual inspection using samtools tview and mpileup (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kumar <em>et al.</em> 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2009</a>).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-25">Genotypes of all strains, as well as all primer and plasmid sequences are included in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FileS1.pdf">File S1</a>. All strains and plasmids are available upon request. Barcode sequencing counts observed for each strain at each time point in each experiment can be found in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S5.xlsx">Table S5</a> and were extracted from raw fastq sequencing files using custom Python scripts. Fitness and interaction score estimates for each strain in each condition can be found in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S2.xlsx">Table S2</a>. Variants called for each of 102 whole-genome sequenced strains are listed in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S3.xlsx">Table S3</a>. Whole-genome sequencing data have been made publicly available under the NCBI BioProject accession number PRJNA344503 and the SRA accession number (SRP090639).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>The iSeq platform</h3><p id="p-26">The iSeq platform consists of a novel double-barcoding technology combined with a pooled fitness assay. The double-barcoding technology uniquely identifies both parents of a mating event. While iSeq could be used to study interactions between any two genomes or genetic elements, here, we use iSeq in combination with gene deletion strains to assay interactions between pairwise combinations of deletions over three environments. Our system functions by first introducing loxP recombination sites at a common chromosomal location in both <em>MATa</em> and <em>MAT</em>α haploids. Barcodes are placed on opposite sides of the loxP sites such that mating and Cre induction causes recombination between homologous chromosomes, resulting in a barcode-loxP-barcode configuration on one chromosome (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Because these double barcodes are unlikely to dissociate during genomic DNA preparation, and are in close enough proximity to be sequenced by short-read single-end or paired-end sequencing, pools of double-barcode strains can subsequently be assayed using standard pooled barcode sequencing approaches (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Pan <em>et al.</em> 2004</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Decourty <em>et al.</em> 2008</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Smith <em>et al.</em> 2009</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2011</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Robinson <em>et al.</em> 2014</a>).</p></div><div id="sec-12" class="subsection"><h3>Experimental design: genes and controls chosen for iSeq validation</h3><p id="p-27">To validate our approach, we chose a modestly sized group of nine genes, and used iSeq to measure the genetic interactions between the 36 possible gene pair combinations. To assess iSeq across a range of values, the genotypes in this set were chosen to include a range of published quantitative interaction scores (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/TableS1.pdf">Table S1</a>). Furthermore, seven of the gene pairs have no published interaction, providing negative controls as well as the possibility of detecting novel environment-dependent genetic interactions upon growth in new conditions. By “marking” each of these gene deletions with four different iSeq barcodes as outlined in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a> and below, we were able to generate up to eight independently constructed strains for each double mutant assayed, thus providing a high level of biological replication.</p><p id="p-28">Single mutant controls, required for interaction score estimates, were generated via the same protocol as their double mutant counterparts, ensuring that all experimental strains carried iSeq double barcodes and the same set of markers. When generating single mutants, we used dubious ORF deletions as placeholders for the second gene deletion. The two dubious ORFs we chose, <a href="http://www.yeastgenome.org/locus/S000001137/overview">YHR095W</a> and <a href="http://www.yeastgenome.org/locus/S000001950/overview">YFR054C</a>, are not expressed, have no fitness defect when deleted under the conditions in which they have been tested, and have no reported genetic interactions in the BioGRID database (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Stark <em>et al.</em> 2006</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Breslow <em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pelechano <em>et al.</em> 2013</a>). Thus, strains carrying one gene deletion and one dubious ORF gene deletion should be reasonable proxies for single mutants. In total, we assayed multiple replicates of 36 double, and nine single gene deletions.</p></div><div id="sec-13" class="subsection"><h3>Construction of iSeq deletion strains</h3><p id="p-29">To generate deletion strains carrying the double-barcoding system, we first constructed two yeast iSeq barcode libraries (288 strains each, in the same <em>MAT</em>α starting strain) by replacing the dubious ORF <a href="http://www.yeastgenome.org/locus/S000000413/overview">YBR209W</a> with one of two complementary plasmid-derived constructs via homologous recombination. The <a href="http://www.yeastgenome.org/locus/S000000413/overview">YBR209W</a> site has been used successfully in our laboratory as an integration site for heterologous genetic elements (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kao and Sherlock 2008</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Levy <em>et al.</em> 2015</a>); its transcript is not expressed, and its absence does not significantly affect fitness (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Breslow <em>et al.</em> 2008</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2008</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Pelechano <em>et al.</em> 2013</a>). A detailed protocol for making these yeast barcode libraries and subsequently generated double barcode strains can be found in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FileS1.pdf">File S1</a>.</p><p id="p-30">We next chose <em>MATa</em> strains derived from the systematic deletion collection (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Winzeler <em>et al.</em> 1999</a>) that carry either a <em>NatMX</em> or a <em>KanMX</em> selectable marker at the deletion locus (F0<sub>Del</sub> haploids), and mated these to <em>MAT</em>α clones from each barcode library (F0<sub>BC</sub> haploids). Resulting diploids were sporulated, and the magic marker system (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tong <em>et al.</em> 2004</a>) was used to select <em>MATa</em> or <em>MAT</em>α haploid clones containing both the iSeq barcode and either a <em>KanMX</em> or <em>NatMX</em> marked deletion, respectively (F1 haploids, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). After selection, and for each clone, the mating type was verified, and the iSeq barcode sequence identified. In total, we barcoded each of the nine gene deletions and two dubious ORF deletions with four different single iSeq barcodes, two barcodes for each version of the deletion (<em>KanMX</em> or <em>NatMX</em>) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-31">To construct double-barcoded double-deletion strains, we mated all pairwise combinations of <em>KanMX</em> and <em>NatMX</em> strains, induced recombination at the iSeq barcode locus, sporulated, eliminated diploids by zymolyase digestion, and then selected haploid clones (F2 haploids, <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). After all matings, each double gene deletion is represented by up to eight unique iSeq double barcodes, and each single gene deletion, which brings together a gene deletion with a dubious ORF deletion, is represented by up to 16 double barcodes (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1D</a>). Finally, eight double-barcoded control strains, each intended to represent a wild-type phenotype, were generated by bringing together two dubious ORF deletions. In total, we generated 393 double-barcoded strains: 257 double gene deletions and 136 single gene deletions.</p></div><div id="sec-14" class="subsection"><h3>Pooled fitness estimates of double-barcode double-deletion strains</h3><p id="p-32">We pooled all 393 double-barcode haploid strains, and mixed this pool with a pool of the eight putative WT control strains at a ratio of 50:50. We combined pools in this way so that at least 50% of cells start with an approximately WT fitness, thereby minimizing the effects of strain–strain interactions between different mutant genotypes during pooled growth. We propagated this combined pool by serial batch culture in YPD at 30° at an effective population size of 8 × 10<sup>9</sup>, bottlenecking 1:8 at each transfer (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, every three generations). This design, which samples at multiple and relatively frequent time points, was chosen for three reasons. First, multiple measurements increase the sensitivity to detect subtle fitness differences between strains. Second, measurements every few generations enable accurate estimates of low fitness genotypes that are rapidly driven to extinction. Third, this large population size was required for our DNA extraction and barcode sequencing protocol, such that sufficient material could be extracted for barcode sequencing (see <em>Materials and Methods</em>). At each bottleneck, we extracted genomic DNA, and then sequenced the double barcodes to estimate the relative frequency of each strain in the population (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, A and B</a> and <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS1.pdf">Figure S1A</a>) (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Smith <em>et al.</em> 2009</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Levy <em>et al.</em> 2015</a>). The slope of a log-linear regression of the change in frequency relative to WT over the four time points was used as the measure of fitness for each double-barcoded strain (see all estimates in <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S2.xlsx">Table S2</a>). For each double-barcoded strain, fitness measurements were highly reproducible across biological replicates (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> and <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS1.pdf">Figure S1B</a>, Spearman’s <em>rho</em> = 0.91–0.97, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). We next investigated the possibility that our pooled fitness assay might have larger errors on lower fitness genotypes, as those genotypes could be quickly driven to low frequencies where sampling errors have a larger effect. We find no significant association between fitness and standard deviation (SD) SD of fitness in our assay (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS2.pdf">Figure S2A</a>, Spearman’s <em>rho</em> = −0.07, <em>P</em> = 0.19), with the least-fit double barcode still having a low fitness error (<em>s</em> = 0.49 ± 0.11) in YPD. We do, however, find greater errors on a small subset of low fitness strains in the two other conditions tested (see below), as, in these conditions, these strains are typically driven below our detection limit after just two time points (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS2.pdf">Figure S2, B and C</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/143/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="iSeq pooled fitness assay and reproducibility of measurements. (A) A schematic of the iSeq pooled fitness assay. Double-barcode pools are grown by serial transfer every &#x223C;3 generations. At each transfer, relative double-barcode frequencies are assayed by short-read amplicon sequencing. (B) Representative plot of relative frequencies from a pooled fitness assay. Each line is an individual double-barcode strain. Colors indicate the fitness estimate of each strain. (C, D) Scatter plot of fitnesses between two biological replicates of the iSeq assay (C) or between iSeq and a multi-well OD-based measurement (D). Spearman&#x2019;s rho is shown on each plot. (E, F) Frequency distributions of SD of the same double barcode across three growth replicates (black), or the same double deletion across four to eight double barcodes (gray) for iSeq (E) or OD-based (F) fitness measurements." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-176927663" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;iSeq pooled fitness assay and reproducibility of measurements. (A) A schematic of the iSeq pooled fitness assay. Double-barcode pools are grown by serial transfer every &#x223C;3 generations. At each transfer, relative double-barcode frequencies are assayed by short-read amplicon sequencing. (B) Representative plot of relative frequencies from a pooled fitness assay. Each line is an individual double-barcode strain. Colors indicate the fitness estimate of each strain. (C, D) Scatter plot of fitnesses between two biological replicates of the iSeq assay (C) or between iSeq and a multi-well OD-based measurement (D). Spearman&#x2019;s &lt;em&gt;rho&lt;/em&gt; is shown on each plot. (E, F) Frequency distributions of SD of the same double barcode across three growth replicates (black), or the same double deletion across four to eight double barcodes (gray) for iSeq (E) or OD-based (F) fitness measurements.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/143/F2.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/1/143/F2.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/143/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/143/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470184" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">iSeq pooled fitness assay and reproducibility of measurements. (A) A schematic of the iSeq pooled fitness assay. Double-barcode pools are grown by serial transfer every ∼3 generations. At each transfer, relative double-barcode frequencies are assayed by short-read amplicon sequencing. (B) Representative plot of relative frequencies from a pooled fitness assay. Each line is an individual double-barcode strain. Colors indicate the fitness estimate of each strain. (C, D) Scatter plot of fitnesses between two biological replicates of the iSeq assay (C) or between iSeq and a multi-well OD-based measurement (D). Spearman’s <em>rho</em> is shown on each plot. (E, F) Frequency distributions of SD of the same double barcode across three growth replicates (black), or the same double deletion across four to eight double barcodes (gray) for iSeq (E) or OD-based (F) fitness measurements.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">To validate the fitness obtained by iSeq, and to determine whether pooling strains had an effect on strain fitness, we next compared iSeq fitness measurements to those from a standard growth assay. We grew each strain in an individual well of a multi-well plate, collecting OD-based growth curves, and using the maximum exponential growth rate as a proxy for fitness (see <em>Materials and Methods</em>). Exponential growth rate might not be expected to correlate highly with fitness during sequential batch growth since potentially important growth dynamics of sequential batch growth, such as entering or leaving saturation, are not captured. Nevertheless, we find a significant positive correlation between the two methods, indicating that potential strain–strain interactions during pooled growth had little to no effect on our fitness estimates (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>, Spearman’s <em>rho</em> = 0.68, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <em>N</em> = 391 strains).</p><p id="p-35">However, despite the reproducibility of our fitness estimates for any given double barcode across replicate cultures, and its concordance with a secondary measure of fitness, we found that there was variability in fitness between strains carrying different double barcodes but the same putative gene deletions. The median SD of fitness for the same double barcode measured across independent cultures is 0.049, while the median SD of fitness of strains with different barcodes but the same deletions is 0.063 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a>). This high variability across strains was also observed in our independent OD-based measure of fitness, indicating it was not an artifact of measuring the fitness in pooled format (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2F</a>).</p></div><div id="sec-15" class="subsection"><h3>Influence of genetic background on fitness</h3><p id="p-36">Above, we observed that the fitness varied when comparing strains carrying identical gene deletions but unique double barcodes (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, E and F</a>). We hypothesized that this variation between double barcodes may be caused either by segregating genetic variation in the parental strains, and/or by <em>de novo</em> mutations that occurred during the growth, mating, or sporulation steps of strain construction. To investigate this possibility, we performed whole genome sequencing on 10 F0<sub>BC</sub>, 6 F0<sub>Del</sub>, 24 F1, and 39 F2 strains that were related by descent (see <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, B and C</a>, and <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S3.xlsx">Table S3</a>). We also sequenced our eight control F2 strains, which each carry two dubious ORF deletions, and their corresponding F0<sub>Del</sub> and F1 parental strains, in order to help determine for any mutations that did arise, whether they arose due to the strain generation protocol, or due to the presence of a gene deletion that causes a severe fitness defect.</p><p id="p-37">A subset of strains from the gene deletion collection has been shown to carry both aneuploidies and suppressor mutations (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hughes <em>et al.</em> 2000</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Teng <em>et al.</em> 2013</a>). Thus, as our sequenced F0<sub>Del</sub> strains were derived from the deletion collection, we first looked for mutations present in these strains. In each of the F0<sub>Del</sub> strains, we observed between one and four private SNPs that were not observed in any other strains except direct descendants, with similar numbers observed between the gene and control groups. Only one aneuploidy was observed, in the <em><a href="http://www.yeastgenome.org/locus/S000005041/overview">PHO23</a></em> deletion strain, on chromosome XI. Similarly, in our sequenced iSeq barcode library strains (F0<sub>BC</sub>), eight of 10 carried either one or two private SNPs, and no aneuploidy was observed.</p><p id="p-38">We next studied the mutations present in the 24 F1 strains carrying one gene deletion and one iSeq barcode (see <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1B</a>). Surprisingly, aneuploidy was extremely common, with 14 strains having an extra copy of at least one chromosome, and, of those, 12 strains carrying an extra copy of chromosome V. We also observed aneuploidy in three of the eight F1 control strains, indicating the aneuploidies were not a response to a specific gene deletion, but more likely a general result of the strain generation procedure. In addition to aneuploidy, one to seven SNPs were observed in 23 of the 24 F1 strains. Because we had sequenced both F0 parents for 10 of the F1 strains, we next determined how much of the observed variation had been inherited, and how much had occurred <em>de novo</em>. In total, 13 of the 19 unique SNPs observed in these 10 F1 strains were also observed in a parental strain, while none of the 18 aneuploidy events were present in a parental strain (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/143/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Segregating and de novo genetic variation revealed by whole-genome sequencing. (A) Total number of unique SNPs, or small indels (left), or aneuploidy events (right), observed across all F1 or F2 strains for which both parental strains had been sequenced (N&#xA0;=&#xA0;10 and N&#xA0;=&#xA0;39 strains, respectively). Bars are shaded by whether the observed variant was observed in a parental strain (light gray), or appeared to have occurred de novo during strain generation (dark gray). (B) For each of the strains sequenced (rows) in each of the double deletion groups, &#x201C;WCD&#x201D; indicates identities of duplicated chromosomes, &#x201C;SNPs&#x201D; indicates the total number of single nucleotide polymorphisms or small indels observed, and &#x201C;YPD Fit.&#x201D; indicates iSeq estimate in YPD. (C) Fitnesses for each whole-genome sequenced F2 strain. Color indicates chromosome&#xA0;V duplication events, and shape indicates gene reversion events in which sequencing reads mapped to one or two genic region(s) expected to be deleted. Error bars are the SD of estimates across three biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-176927663" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Segregating and &lt;em&gt;de novo&lt;/em&gt; genetic variation revealed by whole-genome sequencing. (A) Total number of unique SNPs, or small indels (left), or aneuploidy events (right), observed across all F1 or F2 strains for which both parental strains had been sequenced (&lt;em&gt;N&lt;/em&gt;&#xA0;=&#xA0;10 and &lt;em&gt;N&lt;/em&gt;&#xA0;=&#xA0;39 strains, respectively). Bars are shaded by whether the observed variant was observed in a parental strain (light gray), or appeared to have occurred &lt;em&gt;de novo&lt;/em&gt; during strain generation (dark gray). (B) For each of the strains sequenced (rows) in each of the double deletion groups, &#x201C;WCD&#x201D; indicates identities of duplicated chromosomes, &#x201C;SNPs&#x201D; indicates the total number of single nucleotide polymorphisms or small indels observed, and &#x201C;YPD Fit.&#x201D; indicates iSeq estimate in YPD. (C) Fitnesses for each whole-genome sequenced F2 strain. Color indicates chromosome&#xA0;V duplication events, and shape indicates gene reversion events in which sequencing reads mapped to one or two genic region(s) expected to be deleted. Error bars are the SD of estimates across three biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/143/F3.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/1/143/F3.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/143/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/143/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470106" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Segregating and <em>de novo</em> genetic variation revealed by whole-genome sequencing. (A) Total number of unique SNPs, or small indels (left), or aneuploidy events (right), observed across all F1 or F2 strains for which both parental strains had been sequenced (<em>N</em> = 10 and <em>N</em> = 39 strains, respectively). Bars are shaded by whether the observed variant was observed in a parental strain (light gray), or appeared to have occurred <em>de novo</em> during strain generation (dark gray). (B) For each of the strains sequenced (rows) in each of the double deletion groups, “WCD” indicates identities of duplicated chromosomes, “SNPs” indicates the total number of single nucleotide polymorphisms or small indels observed, and “YPD Fit.” indicates iSeq estimate in YPD. (C) Fitnesses for each whole-genome sequenced F2 strain. Color indicates chromosome V duplication events, and shape indicates gene reversion events in which sequencing reads mapped to one or two genic region(s) expected to be deleted. Error bars are the SD of estimates across three biological replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Next we analyzed the genomes of the 39 F2 strains, which were generated after the second round of mating, and were used in the pooled fitness assay (see <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1C</a>). First, as with the F1 strains, aneuploidy was common (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Of the 39 sequenced F2 strains, 21 had at least one chromosome duplicated (54%), and, of these, as with the F1 strains, chromosome V was the most likely to be duplicated (16 of 21 strains). The strains aneuploid for chromosome V generally had lower fitness than strains with the same gene deletions but no anueploidy (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). A duplication of chromosome V was also observed in one of the eight F2 control strains, indicating these aneuploidies can occur in the absence of gene deletions. In total, 25 of 30 aneuploidies observed in the 21 F2 strains appeared to be inherited, as the aneuploidies were also observed in at least one related F1 parent (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). Aneuploidies also appeared to be lost. Of the seven crosses where both F1 parents carried the same duplicated chromosome, in three cases F2 progeny did not have the aneuploidy.</p><p id="p-41">Second, by examining the coverage in the genic regions that we expected to be deleted, we observed that six of the 39 sequenced F2 strains actually carried a copy of one or both of their two intended gene deletions. In two cases, aneuploidy of chromosome I yielded a heterozygous <em><a href="http://www.yeastgenome.org/locus/S000000011/overview">DEP1</a></em> gene deletion. Two other cases (in putative <em><a href="http://www.yeastgenome.org/locus/S000004075/overview">arp6</a></em>Δ<em><a href="http://www.yeastgenome.org/locus/S000005041/overview">pho23</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000001346/overview">sds3</a></em>Δ<em><a href="http://www.yeastgenome.org/locus/S000005041/overview">pho23</a></em>Δ strains) contained reads mapping to the expected gene deletions, as well as several heterozygous SNPs, suggesting that they are diploids that somehow managed to survive digestion by zymolyase and haploid selection via the magic marker system. The two remaining cases contained reads mapping to the <em><a href="http://www.yeastgenome.org/locus/S000005041/overview">PHO23</a></em> ORF, even though it was intended to be deleted, but no evidence of either aneuploidy or diploidy. We hypothesize that a rare recombination event reinstated the <em><a href="http://www.yeastgenome.org/locus/S000005041/overview">PHO23</a></em> sequence after the second mating step to a strain carrying a wild-type <em><a href="http://www.yeastgenome.org/locus/S000005041/overview">PHO23</a></em>. These reversions did not always lead to an increase in fitness as compared to other strains in the same group, as they often coincided with other events such as aneuploidy (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). None of the F0<sub>Del</sub> strains yielded sequencing reads at their gene deletions, while two F1 strains did, due to aneuploidy (in <em><a href="http://www.yeastgenome.org/locus/S000000011/overview">DEP1</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000001346/overview">SDS3</a></em>), indicating these gene reinstatement events can occur after just one round of mating. No read coverage was ever observed in any of the 27 sequenced dubious ORF deletions (F0<sub>Del</sub>, F1, and F2), suggesting that these reversion events may be selected for because they result in increased fitness.</p><p id="p-42">Finally, there were a total of 57 unique SNPs and small indels segregating across the 39 F2 double-deletion strains sequenced (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S3.xlsx">Table S3</a>). Of these 57 SNPs, 39 we present in an F1 parent (68%), while the rest had occurred <em>de novo</em> after the second round of mating (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A</a>). Up to eight SNPs were observed in each double mutant strain, with a median of four SNPs per strain (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>). The number of SNPs per strain did not vary significantly across the five double deletion groups (<em>P</em> = 0.36, Kruskal-Wallis rank sum test). We also observed two to seven SNPs in our eight control F2 strains, illustrating that similar numbers of mutations accumulated in the absence of functional gene deletions. A majority (54%) of the SNPs and indels either fall in intergenic regions, result in synonymous changes, or result in amino acid changes predicted to be tolerated (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kumar <em>et al.</em> 2009</a>). However, 19% resulted in frameshifts, premature stop codons, or nonsynonymous changes predicted to affect protein function (see <em>Materials and Methods</em>) (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Kumar <em>et al.</em> 2009</a>). There was no significant enrichment for any GO terms for the genes hit by SNPs and indels with functional consequences; however, this might be due to the small sample size. Regardless, segregating variation likely underlies some of the differences in fitness observed for different double-barcoded strains carrying the same gene deletions.</p></div><div id="sec-16" class="subsection"><h3>Interaction score estimates using double barcodes</h3><p id="p-43">Despite the genetic variation present in our strains, we were still able to calculate an interaction score for each strain using our fitness data. An interaction score, ε, is defined as the difference between the observed double mutant fitness, and its expected value based on the product of the fitnesses of the two corresponding single mutants (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Phillips <em>et al.</em> 2000</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Segrè <em>et al.</em> 2005</a>; <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">St Onge <em>et al.</em> 2007</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Costanzo <em>et al.</em> 2010</a>). Using this definition, we find that interaction scores for each double-barcode strain are highly reproducible between pooled growth replicates (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS1.pdf">Figure S1C</a>, Spearman’s <em>rho</em> = 0.96–0.98, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <em>N</em> = 255 strains), and correlate with interaction scores derived from the maximum exponential growth rate of single and double mutants (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>, Spearman’s <em>rho</em> = 0.69, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <em>N</em> = 255 strains). However, here too we find high variance between double barcodes that represent the same putative double knockout genotype. The median SD of interaction scores across strains with identical gene deletions is 0.072, 2.5-fold higher than the variance of each double barcode across pooled growth replicates (median SD = 0.072 <em>vs.</em> SD = 0.028, <em>P</em> = 2.2 × 10<sup>−16</sup>, Wilcoxon rank-sum test, <em>N</em> = 36 gene deletion pairs and 255 strains).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/143/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identifying environment-dependent genetic interactions with iSeq. (A, B) Scatter plot of interaction scores between two biological replicates of the iSeq assay (A), or between iSeq and a multi-well OD-based measurement (B). (C) Interaction scores for individual strains carrying gene deletion pairs with a previously published positive (left) or negative (right) interaction. (D) The GI networks in each environment. For network edges, the color represents positive (red) or negative (blue) interaction scores, the width indicates relative magnitude of each score, and dashed lines are significant changes between YPD and another environment. (E) GI scores of all double-barcode replicates for three double deletions in two environments. Points and error bars in (B, C, E) are mean &#xB1; SD across three growth replicates. Red dashes in (C, E) are median values. P-values in (C, E) are Wilcoxon Mann-Whitney rank-sum test, and are 10% FDR corrected in (E)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-176927663" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identifying environment-dependent genetic interactions with iSeq. (A, B) Scatter plot of interaction scores between two biological replicates of the iSeq assay (A), or between iSeq and a multi-well OD-based measurement (B). (C) Interaction scores for individual strains carrying gene deletion pairs with a previously published positive (left) or negative (right) interaction. (D) The GI networks in each environment. For network edges, the color represents positive (red) or negative (blue) interaction scores, the width indicates relative magnitude of each score, and dashed lines are significant changes between YPD and another environment. (E) GI scores of all double-barcode replicates for three double deletions in two environments. Points and error bars in (B, C, E) are mean &#xB1; SD across three growth replicates. Red dashes in (C, E) are median values. &lt;em&gt;P&lt;/em&gt;-values in (C, E) are Wilcoxon Mann-Whitney rank-sum test, and are 10% FDR corrected in (E).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/143/F4.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/1/143/F4.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/143/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/143/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470046" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Identifying environment-dependent genetic interactions with iSeq. (A, B) Scatter plot of interaction scores between two biological replicates of the iSeq assay (A), or between iSeq and a multi-well OD-based measurement (B). (C) Interaction scores for individual strains carrying gene deletion pairs with a previously published positive (left) or negative (right) interaction. (D) The GI networks in each environment. For network edges, the color represents positive (red) or negative (blue) interaction scores, the width indicates relative magnitude of each score, and dashed lines are significant changes between YPD and another environment. (E) GI scores of all double-barcode replicates for three double deletions in two environments. Points and error bars in (B, C, E) are mean ± SD across three growth replicates. Red dashes in (C, E) are median values. <em>P</em>-values in (C, E) are Wilcoxon Mann-Whitney rank-sum test, and are 10% FDR corrected in (E).</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">We next compared the interactions we identified with those collected through literature curation (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Stark <em>et al.</em> 2006</a>). We do note, however, that these published interactions are generally derived from colony growth on plates, and some interactions can be condition-specific, such that they are only observable either during growth in liquid, or when assayed on plates (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Musso <em>et al.</em> 2008</a>). Of the 36 gene pairs we tested, 14 have a reported negative genetic interaction, 15 a positive reported interaction, and seven have no reported interaction. Our scores for interactions in strains in the positive group were significantly different from those in the negative group (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>, <em>P</em> = 1.2 × 10<sup>−4</sup>, Wilcoxon rank-sum test), suggesting that, despite the scores being generated from different experimental conditions, and the known genetic variation our strains, there are similar observable trends.</p><p id="p-46">To compare iSeq interaction scores to those previously reported from large-scale systematic screens, we calculated a mean interaction score for each double deletion (four to eight double barcodes per double gene deletion, with three replicate growth experiments each). Interaction scores derived from iSeq weakly correlate with those derived from two previous studies (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Collins <em>et al.</em> 2007</a>; <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Costanzo <em>et al.</em> 2010</a>) (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS3.pdf">Figure S3</a>, Collins: Spearman’s <em>rho</em> = 0.36, <em>P</em> = 0.063, <em>N</em> = 28 gene pairs; and Costanzo: Spearman’s <em>rho</em> <em>=</em> 0.38, <em>P</em> = 0.005, <em>N</em> = 33 gene pairs). As discussed above, complete agreement is not necessarily expected between different assays because they are performed in different growth conditions.</p></div><div id="sec-17" class="subsection"><h3>Measurement of differential interactions using iSeq</h3><p id="p-47">We next performed two additional pooled fitness assays on our set of strains—one in heat stress (YPD 37°) and one in a nonfermentable carbon source (ethanol and glycerol, YPEG). As we observed in rich medium, fitness and interaction score estimates in the two new growth conditions were highly reproducible across replicate cultures (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS1.pdf">Figure S1, B and C</a>, Spearman’s <em>rho</em> = 0.97–0.99, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, fitness median SD = 0.027, interaction score median SD = 0.024), while there was only a weak negative correlation between fitness and the SD of fitness across replicate cultures (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS2.pdf">Figure S2, B and C</a>).</p><p id="p-48">To determine whether there are changes in interaction scores across conditions, we first called significant interactions in each of the three conditions using 95% CIs (see <a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S4.xlsx">Table S4</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>). Though many changes in sign and magnitude of interaction scores were observed between YPD and the two alternate conditions, a total of three gene pairs changed interaction score in a statistically significant manner [<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4E</a>, <em>P</em> ≤ 0.005, <em>N</em> = 6–8 strains, Wilcoxon rank-sum test, 10% false discovery rate (FDR)]. Two gene deletion pairs (<em><a href="http://www.yeastgenome.org/locus/S000000011/overview">dep1</a></em>Δ<em><a href="http://www.yeastgenome.org/locus/S000005041/overview">pho23</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000004876/overview">sap30</a></em>Δ<em><a href="http://www.yeastgenome.org/locus/S000005041/overview">pho23</a></em>Δ) had no interaction in YPD but interact negatively in YPEG. One other gene deletion pair (<em><a href="http://www.yeastgenome.org/locus/S000004876/overview">sap30</a></em>Δ<em><a href="http://www.yeastgenome.org/locus/S000000629/overview">snt1</a></em>Δ) changed from no significant interaction in YPD at 30°, to a negative genetic interaction in YPD at 37°.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-49">We have developed a new double barcode interaction sequencing technology (iSeq) that can be used to quantitatively examine pairwise GIs. iSeq’s double barcoding system allowed us to use pooled serial batch growth and high-throughput sequencing to measure the fitness of hundreds of double-deletion strains simultaneously—an approach previously only possible with pools of single deletion strains, or double deletions carrying a common deletion. Our method produces extremely reproducible fitness, and GI estimates for the same double barcode across replicate pooled growth experiments. Furthermore, the pooled iSeq fitness and GI scores correlate well with measurements made during individual growth, indicating that pooled growth does not confound our results. At current sequencing costs, considering an average coverage of 100 reads per strain for each of five time points (and 50% of the pool consisting of a barcoded control strain), we estimate a sequencing cost of $0.02 per GI per replicate per environment; these costs will fall at the same rate that sequencing does.</p><p id="p-50">Importantly, we have illustrated iSeq’s ability to measure variance between individual, clonally derived strains with the same presumptive genotype by assaying several replicate strains in parallel. This ability could be applied to validate GIs measured in large screens with high replication. Here, by performing iSeq with four to eight independent constructs of the same double deletion, we found a high variance in both fitnesses and GI scores. The median correlation value for comparisons between our eight replicate strains per double gene deletion was 0.42, similar to previous reports of 0.2–0.5 (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Schuldiner <em>et al.</em> 2005</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jasnos and Korona 2007</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Dodgson <em>et al.</em> 2016</a>). However, ours is the first study, to our knowledge, to use whole genome sequencing to investigate the underlying genetic variation that might confound GI measurements and lead to relatively low reproducibility. Our observation of new aneuploidies and SNPs after the first round of mating means mutations can accumulate quickly. Furthermore, these new mutations occurred prior to the Gal-induced Cre activity, and were also observed in dubious ORF deletion carrying controls, leading us to believe they were not an artifact specific to the deletion strains we chose, or the barcoding system itself.</p><p id="p-51">However, several factors may limit generalizing our findings to previous GI studies. First, our study used the magic marker construct carrying the MFA/MFalpha promoters to select haploids. The MFA/MFalpha construct is known to be more leaky and prone to diploidization than the newer construct that uses <em><a href="http://www.yeastgenome.org/locus/S000001868/overview">STE2</a>/<a href="http://www.yeastgenome.org/locus/S000001661/overview">STE3</a></em> promoters, which is now more commonly used (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Tong and Boone 2007</a>; B. Andrews, personal communication). Further experimental work would be required to directly compare rates of aneuploidy accumulation with the <em><a href="http://www.yeastgenome.org/locus/S000001868/overview">STE2</a>/<a href="http://www.yeastgenome.org/locus/S000001661/overview">STE3</a></em> construct. It is also possible that the deletions we chose to examine have higher than average rates of mutation or chromosome segregation defects. Indeed, four of the double deletions we sequenced contain at least one gene shown to be involved in chromosome maintenance (<em><a href="http://www.yeastgenome.org/locus/S000005364/overview">SIN3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001346/overview">SDS3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005274/overview">RPD3</a></em>) (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wahba <em>et al.</em> 2011</a>). Additionally, we chose a set of deletions with generally severe fitness effects, which might be more likely to accumulate additional fitness-altering mutations. Consequently, we did observe a slightly elevated accumulation of aneuploidies and SNPs in our strains carrying gene deletions compared to those carrying dubious ORF deletions (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/Table_S3.xlsx">Table S3</a>). Finally, our strains went through one additional round of mating and selection compared with standard interaction studies, which provided more opportunity for mutations to arise and segregate across our experimental strains. These extra mating and sporulation steps are especially relevant because meiosis can exhibit elevated rates of mutation at sites of recombination (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rattray <em>et al.</em> 2015</a>).</p><p id="p-52">Regarding the specific mutations we observed in our strains, despite the fact that aneuploidy typically results in a growth defect, in some cases it can provide an advantage during stress and even help overcome the loss of a gene (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Vernon <em>et al.</em> 2008</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yona <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Liu <em>et al.</em> 2015</a>). In our experiments we find that chromosome V duplication was commonly observed in strains resulting from both the first and second rounds of mating and haploid selection. The magic marker locus we used to select for haploids of a desired mating type (<em><a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a>Δ</em>::<em><a href="http://www.yeastgenome.org/locus/S000002869/overview">MFA1</a>pr-<a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a>-<a href="http://www.yeastgenome.org/locus/S000006108/overview">MFα</a>1pr-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>), is located on chromosome V. It functions by expressing <a href="http://www.yeastgenome.org/locus/S000005728/overview">His3</a> or <a href="http://www.yeastgenome.org/locus/S000000523/overview">Leu2</a> under a MATa-dependent or MATalpha-dependent promoter, respectively. Thus, an extra copy of the magic marker locus created by duplication may produce more <a href="http://www.yeastgenome.org/locus/S000005728/overview">His3</a> or <a href="http://www.yeastgenome.org/locus/S000000523/overview">Leu2</a>, providing a benefit during selection on media lacking histidine or leucine. In our pooled growth assays, however, we found that chromosome V duplication typically correlates with a decrease in fitness, suggesting that the selective advantage only occurs during strain construction (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3C</a>). However, plating serial dilutions of our sequenced double mutants onto YPD, and the medium used for their selection, revealed no consistent growth advantage for strains carrying the duplication (<a href="http://www.g3journal.org/content/suppl/2016/11/07/g3.116.034207.DC1/FigureS4.pdf">Figure S4</a>), though such an advantage could be confounded by other segregating variation. We lacked the statistical power to determine if rarer aneuploidies or SNPs also correlate with fitness. Of particular concern is that some of these variants may be deletion-specific suppressor mutations; these have been found in the deletion collection (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Teng <em>et al.</em> 2013</a>), and have been found to establish after only a few generations of growth (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Szamecz <em>et al.</em> 2014</a>). In our sequencing, we observed five cases of an aneuploidy of a chromosome rescuing a gene deletion.</p><p id="p-53">There are several potential solutions to reduce the amount of segregating genetic variation and <em>de novo</em> mutations during genetic interaction screens. To address the common chromosome V aneuploidy we observe (in 41% of sequenced strains), one potential solution would be to include, at the magic marker locus, a gene that can be tolerated in no more than two copies in the haploid (including one copy at the endogenous locus), such as <em><a href="http://www.yeastgenome.org/locus/S000001924/overview">CDC14</a></em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Moriya <em>et al.</em> 2006</a>). Alternatively, using the <em><a href="http://www.yeastgenome.org/locus/S000001868/overview">STE2</a>/<a href="http://www.yeastgenome.org/locus/S000001661/overview">STE3</a></em>-driven magic marker, or having the construct on a plasmid rather than it being integrated in the genome, may reduce the rates of accumulation of chromosome V aneuploidy. It is clear that not all genetic variation could be controlled in this manner. A possible alternative approach to minimize the generation of confounding genetic variation would be to minimize the number of generations deletion strains undergo between the introduction of the gene deletion(s) and the fitness measurements. For example, inducible CRISPR/Cas9 systems that knockdown selected gene targets are available (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gilbert <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mans <em>et al.</em> 2015</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Senturk <em>et al.</em> 2015</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Smith <em>et al.</em> 2016</a>), and these could be used in conjunction with iSeq, by integrating gRNAs at the same time and location as barcodes in order to generate inducible double knockdowns. This strategy could also be employed to search for interactions that include essential genes.</p><p id="p-54">We envision that iSeq can be used in the future to measure interactions between large groups of gene deletions across different experimental growth conditions. However, scaling up will require modifications to our strain generation protocol, such that a double barcode no longer marks a single clonal instance of a genotype. To achieve this, one could relatively easily place iSeq barcodes in the deletion collection library using the synthetic genetic array technology (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Tong and Boone 2007</a>) in combination with robotics. Double-barcode, double-deletion strains could then be generated via another round of robot-assisted paired mating and haploid selection. Because these protocols do not use single cell bottlenecks as we did here, strains generated from this modified protocol would likely consist of many segregants, and fitnesses and GI scores would be an aggregate measurement of these segregant pools. While segregant pool measurements may be likely to be more comparable with previous studies, they would not reveal the possible confounding influence of segregating variation and <em>de novo</em> mutations as we did here. Thus, employing iSeq as we described it here, would provide a valuable secondary method to validate that interactions discovered by high-throughput methods are robust. These two complementary approaches illustrate iSeq’s flexibility. Indeed, the iSeq technology has many potential applications beyond screening for interactions between gene deletions, and could in theory be used to screen, in multiple growth conditions, for interactions between naturally occurring genetic variation or engineered genetic constructs.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank the members of the Sherlock laboratory, including Dimitra Aggeli, Lucas Herissant, Yuping Li, Barbara Dunn, and Katja Schwartz, for helpful discussions; Sandeep Venkataram for assistance with the whole-genome sequencing analysis; Jamie Blundell for input on fitness estimations; Mark Siegal for the concept of the split <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> construct; and John Pringle, Michael Bassik, and Jan Skotheim for guidance. We also thank the Stanford Center for Genomics and Personalized Medicine and the Stanford PAN Facility for their sequencing services. Funding for M.J. was provided by National Institutes of Health/National Human Genome Research Institute T32 HG000044 and a National Science Foundation Graduate Research Fellowship Program fellowship. The Stanford Center for Computational, Evolutionary and Human Genomics provided seed funding to M.J. for experimental costs. G.S. is supported by R01 GM110275. Work by S.F.L. is supported by The Louis and Beatrice Laufer Center, and National Institutes of Health grant R01 HG008354. S.F.L. has a pending patent on a related technology.</p><p id="p-56">Author contributions: The double-barcoding system was conceived and designed by S.F.L. M.J., S.F.L., and G.S. designed the experiments and wrote the manuscript. M.J. carried out the experiments and analyzed the data. S.F.L. and G.S. provided input on data analysis.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-57">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034207/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034207/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-58"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-16"><span class="received-label">Received </span>May 16, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-01"><span class="accepted-label">Accepted </span>November 1, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Jaffe <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.1" data-doi="10.1242/jcs.02714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Scale-free networks in cell biology.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">4947</span>–<span class="cit-lpage">4957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DAlbert%26rft.auinit1%253DR.%26rft.volume%253D118%26rft.issue%253D21%26rft.spage%253D4947%26rft.epage%253D4957%26rft.atitle%253DScale-free%2Bnetworks%2Bin%2Bcell%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.02714%26rft_id%253Dinfo%253Apmid%252F16254242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjExOC8yMS80OTQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.2" data-doi="10.1126/science.1195618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">M.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rewiring of genetic networks in response to DNA damage.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1385</span>–<span class="cit-lpage">1389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBandyopadhyay%26rft.auinit1%253DS.%26rft.volume%253D330%26rft.issue%253D6009%26rft.spage%253D1385%26rft.epage%253D1389%26rft.atitle%253DRewiring%2Bof%2BGenetic%2BNetworks%2Bin%2BResponse%2Bto%2BDNA%2BDamage%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1195618%26rft_id%253Dinfo%253Apmid%252F21127252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAwOS8xMzg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.3" data-doi="10.1038/nmeth.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBaryshnikova%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1017%26rft.epage%253D1024%26rft.atitle%253DQuantitative%2Banalysis%2Bof%2Bfitness%2Band%2Bgenetic%2Binteractions%2Bin%2Byeast%2Bon%2Ba%2Bgenome%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1534%26rft_id%253Dinfo%253Apmid%252F21076421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076421&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284686300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.4" data-doi="10.1128/MCB.11.3.1295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Use of a screen for synthetic lethal and multicopy suppressee mutants to identify two new genes involved in morphogenesis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBender%26rft.auinit1%253DA%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D1295%26rft.epage%253D1305%26rft.atitle%253DUse%2Bof%2Ba%2Bscreen%2Bfor%2Bsynthetic%2Blethal%2Band%2Bmulticopy%2Bsuppressee%2Bmutants%2Bto%2Bidentify%2Btwo%2Bnew%2Bgenes%2Binvolved%2Bin%2Bmorphogenesis%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.3.1295%26rft_id%253Dinfo%253Apmid%252F1996092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzMvMTI5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.5" data-doi="10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachmann</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Designer deletion strains derived from <em>Saccharomyces cerevisiae</em> S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBrachmann%26rft.auinit1%253DC.%2BB.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D132%26rft.atitle%253DDesigner%2Bdeletion%2Bstrains%2Bderived%2Bfrom%2BSaccharomyces%2Bcerevisiae%2BS288C%253A%2Ba%2Buseful%2Bset%2Bof%2Bstrains%2Band%2Bplasmids%2Bfor%2BPCR-mediated%2Bgene%2Bdisruption%2Band%2Bother%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819980130%252914%253A2%253C115%253A%253AAID-YEA204%253E3.0.CO%253B2-2%26rft_id%253Dinfo%253Apmid%252F9483801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9483801&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071817800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.6" data-doi="10.1038/nmeth.1234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breslow</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart-Ornstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBreslow%26rft.auinit1%253DD.%2BK.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D711%26rft.epage%253D718%26rft.atitle%253DA%2Bcomprehensive%2Bstrategy%2Benabling%2Bhigh-resolution%2Bfunctional%2Banalysis%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1234%26rft_id%253Dinfo%253Apmid%252F18622397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18622397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258077400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.7" data-doi="10.1186/gb-2006-7-7-r63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253DR63%26rft.epage%253DR63%26rft.atitle%253DA%2Bstrategy%2Bfor%2Bextracting%2Band%2Banalyzing%2Blarge-scale%2Bquantitative%2Bepistatic%2Binteraction%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-7-r63%26rft_id%253Dinfo%253Apmid%252F16859555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-7-r63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16859555&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.8" data-doi="10.1038/nature05649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maas</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fillingham</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">446</span>: <span class="cit-fpage">806</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D446%26rft.issue%253D7137%26rft.spage%253D806%26rft.epage%253D810%26rft.atitle%253DFunctional%2Bdissection%2Bof%2Bprotein%2Bcomplexes%2Binvolved%2Bin%2Byeast%2Bchromosome%2Bbiology%2Busing%2Ba%2Bgenetic%2Binteraction%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05649%26rft_id%253Dinfo%253Apmid%252F17314980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17314980&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245626800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.9" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.10" data-doi="10.1126/science.aaf1420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanderSluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pons</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A global genetic interaction network maps a wiring diagram of cellular function.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>: <span class="cit-fpage">aaf1420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D353%26rft.issue%253D6306%26rft.spage%253Daaf1420%26rft.epage%253Daaf1420%26rft.atitle%253DA%2Bglobal%2Bgenetic%2Binteraction%2Bnetwork%2Bmaps%2Ba%2Bwiring%2Bdiagram%2Bof%2Bcellular%2Bfunction%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf1420%26rft_id%253Dinfo%253Apmid%252F27708008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTMvNjMwNi9hYWYxNDIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.11" data-doi="10.1128/MCB.00050-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darby</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serebreni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corden</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae</em> Nrd1-Nab3 transcription termination pathway acts in opposition to Ras signaling and mediates response to nutrient depletion.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1762</span>–<span class="cit-lpage">1775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00050-12%26rft_id%253Dinfo%253Apmid%252F22431520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xMC8xNzYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.12" data-doi="10.1038/ng1640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davierwala</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brost</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The synthetic genetic interaction spectrum of essential genes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDavierwala%26rft.auinit1%253DA.%2BP.%26rft.volume%253D37%26rft.issue%253D10%26rft.spage%253D1147%26rft.epage%253D1152%26rft.atitle%253DThe%2Bsynthetic%2Bgenetic%2Binteraction%2Bspectrum%2Bof%2Bessential%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1640%26rft_id%253Dinfo%253Apmid%252F16155567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16155567&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232185600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.13" data-doi="10.1073/pnas.0710533105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Decourty</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveanu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zemam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hantraye</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frachon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linking functionally related genes by sensitive and quantitative characterization of genetic interaction profiles.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">5821</span>–<span class="cit-lpage">5826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0710533105%26rft_id%253Dinfo%253Apmid%252F18408161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzE1LzU4MjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.14" data-doi="10.1534/genetics.115.185660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chromosome-specific and global effects of aneuploidy in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1395</span>–<span class="cit-lpage">1409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185660%26rft_id%253Dinfo%253Apmid%252F26837754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi80LzEzOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.15" data-doi="10.1038/nbt.2463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutkowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surma</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A gene ontology inferred from molecular networks.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2463%26rft_id%253Dinfo%253Apmid%252F23242164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23242164&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.16" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.17" data-doi="10.1016/j.cell.2013.06.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morsut</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">442</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D442%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.06.044%26rft_id%253Dinfo%253Apmid%252F23849981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.06.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23849981&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321950700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.18" data-doi="10.1534/genetics.110.120766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">System-level analysis of genes and functions affecting survival during nutrient starvation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120766%26rft_id%253Dinfo%253Apmid%252F20944018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMjk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.19" data-doi="10.1016/j.molcel.2012.11.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guénolé</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vreeken</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dissection of DNA damage responses using multiconditional genetic interaction maps.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D49%26rft.spage%253D346%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.11.023%26rft_id%253Dinfo%253Apmid%252F23273983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.11.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23273983&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314379400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.20" data-doi="10.1073/pnas.0607153104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pál</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delneri</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Plasticity of genetic interactions in metabolic networks of yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">2307</span>–<span class="cit-lpage">2312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0607153104%26rft_id%253Dinfo%253Apmid%252F17284612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzcvMjMwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.21" data-doi="10.1093/emboj/16.20.6171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Yeast spore germination: a requirement for Ras protein activity during re-entry into the cell cycle.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">6171</span>–<span class="cit-lpage">6181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DHerman%26rft.auinit1%253DP.%2BK.%26rft.volume%253D16%26rft.issue%253D20%26rft.spage%253D6171%26rft.epage%253D6181%26rft.atitle%253DYeast%2Bspore%2Bgermination%253A%2Ba%2Brequirement%2Bfor%2BRas%2Bprotein%2Bactivity%2Bduring%2Bre-entry%2Binto%2Bthe%2Bcell%2Bcycle%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.20.6171%26rft_id%253Dinfo%253Apmid%252F9321396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTYvMjAvNjE3MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.22" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.23" data-doi="10.1371/journal.pgen.1003083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srinivasan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovering thiamine transporters as targets of chloroquine using a novel functional genomics strategy.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DZ.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003083%26rft.epage%253De1003083%26rft.atitle%253DDiscovering%2Bthiamine%2Btransporters%2Bas%2Btargets%2Bof%2Bchloroquine%2Busing%2Ba%2Bnovel%2Bfunctional%2Bgenomics%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003083%26rft_id%253Dinfo%253Apmid%252F23209439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23209439&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.24" data-doi="10.1038/77116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Widespread aneuploidy revealed by DNA microarray expression profiling.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D333%26rft.epage%253D337%26rft.atitle%253DWidespread%2Baneuploidy%2Brevealed%2Bby%2BDNA%2Bmicroarray%2Bexpression%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F77116%26rft_id%253Dinfo%253Apmid%252F10888885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/77116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10888885&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087920900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.25" data-doi="10.1038/ng1986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jasnos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korona</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Epistatic buffering of fitness loss in yeast double deletion strains.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJasnos%26rft.auinit1%253DL.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D550%26rft.epage%253D554%26rft.atitle%253DEpistatic%2Bbuffering%2Bof%2Bfitness%2Bloss%2Bin%2Byeast%2Bdouble%2Bdeletion%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1986%26rft_id%253Dinfo%253Apmid%252F17322879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17322879&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.26" data-doi="10.1038/ng.280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular characterization of clonal interference during adaptive evolution in asexual populations of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1499</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKao%26rft.auinit1%253DK.%2BC.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1499%26rft.epage%253D1504%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bclonal%2Binterference%2Bduring%2Badaptive%2Bevolution%2Bin%2Basexual%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.280%26rft_id%253Dinfo%253Apmid%252F19029899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19029899&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261215900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.27" data-doi="10.1038/nbt1096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ideker</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Systematic interpretation of genetic interactions using protein networks.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKelley%26rft.auinit1%253DR.%26rft.volume%253D23%26rft.issue%253D5%26rft.spage%253D561%26rft.epage%253D566%26rft.atitle%253DSystematic%2Binterpretation%2Bof%2Bgenetic%2Binteractions%2Busing%2Bprotein%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1096%26rft_id%253Dinfo%253Apmid%252F15877074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15877074&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228943500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.28" data-doi="10.1126/science.1250939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kryazhimskiy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jerison</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Microbial evolution. Global epistasis makes adaptation predictable despite sequence-level stochasticity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKryazhimskiy%26rft.auinit1%253DS.%26rft.volume%253D344%26rft.issue%253D6191%26rft.spage%253D1519%26rft.epage%253D1522%26rft.atitle%253DGlobal%2Bepistasis%2Bmakes%2Badaptation%2Bpredictable%2Bdespite%2Bsequence-level%2Bstochasticity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1250939%26rft_id%253Dinfo%253Apmid%252F24970088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDQvNjE5MS8xNTE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.29" data-doi="10.1038/nprot.2009.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKumar%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253D1073%26rft.epage%253D1081%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bcoding%2Bnon-synonymous%2Bvariants%2Bon%2Bprotein%2Bfunction%2Busing%2Bthe%2BSIFT%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.86%26rft_id%253Dinfo%253Apmid%252F19561590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561590&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.30" data-doi="10.1038/ng1844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crombie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tischler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Systematic mapping of genetic interactions in <em>Caenorhabditis elegans</em> identifies common modifiers of diverse signaling pathways.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">896</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLehner%26rft.auinit1%253DB.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D896%26rft.epage%253D903%26rft.atitle%253DSystematic%2Bmapping%2Bof%2Bgenetic%2Binteractions%2Bin%2BCaenorhabditis%2Belegans%2Bidentifies%2Bcommon%2Bmodifiers%2Bof%2Bdiverse%2Bsignaling%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1844%26rft_id%253Dinfo%253Apmid%252F16845399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845399&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.31" data-doi="10.1038/nature14279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blundell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkataram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative evolutionary dynamics using high-resolution lineage tracking.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">519</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D519%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14279%26rft_id%253Dinfo%253Apmid%252F25731169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25731169&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.32" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.33" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth cit-collab">1000 Genome Project Data Processing Subgroup</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.34" data-doi="10.1101/gr.076661.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvunis</span>  <span class="cit-name-given-names">A.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J.-D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Revisiting the <em>Saccharomyces cerevisiae</em> predicted ORFeome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.076661.108%26rft_id%253Dinfo%253Apmid%252F18502943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzgvMTI5NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.35" data-doi="10.1016/j.cell.2015.10.069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yong</span>  <span class="cit-name-given-names">M. Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurieva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srinivasan</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene essentiality is a quantitative property linked to cellular evolvability.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1388</span>–<span class="cit-lpage">1399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D1388%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.10.069%26rft_id%253Dinfo%253Apmid%252F26627736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.10.069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26627736&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.36" data-doi="10.1073/pnas.0712255105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">F. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Defining genetic interaction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">3461</span>–<span class="cit-lpage">3466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712255105%26rft_id%253Dinfo%253Apmid%252F18305163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzkvMzQ2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Rossum</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijsman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backx</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuijpers</span>  <span class="cit-name-given-names">N. G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPR/Cas9: a molecular Swiss army knife for simultaneous introduction of multiple genetic modifications in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: pii: fov004.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.38" data-doi="10.1038/nature03959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margulies</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egholm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attiya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome sequencing in microfabricated high-density picolitre reactors.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMargulies%26rft.auinit1%253DM.%26rft.volume%253D437%26rft.issue%253D7057%26rft.spage%253D376%26rft.epage%253D380%26rft.atitle%253DGenome%2Bsequencing%2Bin%2Bmicrofabricated%2Bhigh-density%2Bpicolitre%2Breactors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03959%26rft_id%253Dinfo%253Apmid%252F16056220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16056220&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231849100045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.39" data-doi="10.15252/msb.20145606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shales</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Piñar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molina</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Differential genetic interactions of yeast stress response MAPK pathways.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DMartin%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D800%26rft.epage%253D800%26rft.atitle%253DDifferential%2Bgenetic%2Binteractions%2Bof%2Byeast%2Bstress%2Bresponse%2BMAPK%2Bpathways%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20145606%26rft_id%253Dinfo%253Apmid%252F25888283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjExLzQvODAwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads.</span> <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr>. <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>–<span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.41" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.42" data-doi="10.1371/journal.pgen.0020111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu-Yoshida</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitano</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">In vivo robustness analysis of cell division cycle genes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMoriya%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De111%26rft.epage%253De111%26rft.atitle%253DIn%2Bvivo%2Brobustness%2Banalysis%2Bof%2Bcell%2Bdivision%2Bcycle%2Bgenes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020111%26rft_id%253Dinfo%253Apmid%252F16839182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16839182&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.43" data-doi="10.1101/gr.076174.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musso</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huangfu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The extensive and condition-dependent nature of epistasis among whole-genome duplicates in yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1092</span>–<span class="cit-lpage">1099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.076174.108%26rft_id%253Dinfo%253Apmid%252F18463300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzcvMTA5MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.44" data-doi="10.1038/ng1258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ooi</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">DNA helicase gene interaction network defined using synthetic lethality analyzed by microarray.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DOoi%26rft.auinit1%253DS.%2BL.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D277%26rft.epage%253D286%26rft.atitle%253DDNA%2Bhelicase%2Bgene%2Binteraction%2Bnetwork%2Bdefined%2Busing%2Bsynthetic%2Blethality%2Banalyzed%2Bby%2Bmicroarray.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1258%26rft_id%253Dinfo%253Apmid%252F14566339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14566339&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186273300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.45" data-doi="10.1016/j.molcel.2004.09.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sookhai-Mahadeo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A robust toolkit for functional profiling of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DPan%26rft.auinit1%253DX.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D487%26rft.epage%253D496%26rft.atitle%253DA%2Brobust%2Btoolkit%2Bfor%2Bfunctional%2Bprofiling%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.09.035%26rft_id%253Dinfo%253Apmid%252F15525520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.09.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15525520&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225004500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.46" data-doi="10.1016/j.cell.2005.12.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A DNA integrity network in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPan%26rft.auinit1%253DX.%26rft.volume%253D124%26rft.issue%253D5%26rft.spage%253D1069%26rft.epage%253D1081%26rft.atitle%253DA%2BDNA%2Bintegrity%2Bnetwork%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.12.036%26rft_id%253Dinfo%253Apmid%252F16487579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.12.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16487579&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237241200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.47" data-doi="10.1038/nature09529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saraf</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPavelka%26rft.auinit1%253DN.%26rft.volume%253D468%26rft.issue%253D7321%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DAneuploidy%2Bconfers%2Bquantitative%2Bproteome%2Bchanges%2Band%2Bphenotypic%2Bvariation%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09529%26rft_id%253Dinfo%253Apmid%252F20962780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20962780&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284051000054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.48" data-doi="10.1038/nature12121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelechano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Extensive transcriptional heterogeneity revealed by isoform profiling.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12121%26rft_id%253Dinfo%253Apmid%252F23615609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23615609&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318221500047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.143.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Beyond the average. The evolutionary importance of gene interactions and variability of epistatic effects</span>, pp. <span class="cit-fpage">20</span>–<span class="cit-lpage">38</span> in <span class="cit-source"><em>Epistasis and the Evolutionary Process</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Brodie</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.50" data-doi="10.1371/journal.pgen.1004910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rattray</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santoyo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Elevated mutation rate during meisosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004910%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004910%26rft_id%253Dinfo%253Apmid%252F25569256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25569256&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.51" data-doi="10.1534/g3.113.008565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Design and analysis of Bar-seq experiments.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008565%26rft_id%253Dinfo%253Apmid%252F24192834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjQvMS8xMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.52" data-doi="10.1126/science.1162609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zofall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162609%26rft_id%253Dinfo%253Apmid%252F18818364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTkwMC80MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.53" data-doi="10.1016/j.cell.2005.08.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denic</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhamidipati</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchuldiner%26rft.auinit1%253DM.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D507%26rft.epage%253D519%26rft.atitle%253DExploration%2Bof%2Bthe%2Bfunction%2Band%2Borganization%2Bof%2Bthe%2Byeast%2Bearly%2Bsecretory%2Bpathway%2Bthrough%2Ban%2Bepistatic%2Bminiarray%2Bprofile.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.08.031%26rft_id%253Dinfo%253Apmid%252F16269340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.08.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16269340&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233264300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.54" data-doi="10.1016/j.ymeth.2006.07.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Quantitative genetic analysis in <em>Saccharomyces cerevisiae</em> using epistatic miniarray profiles (E-MAPs) and its application to chromatin functions.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DSchuldiner%26rft.auinit1%253DM.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D344%26rft.epage%253D352%26rft.atitle%253DQuantitative%2Bgenetic%2Banalysis%2Bin%2BSaccharomyces%2Bcerevisiae%2Busing%2Bepistatic%2Bminiarray%2Bprofiles%2B%2528E-MAPs%2529%2Band%2Bits%2Bapplication%2Bto%2Bchromatin%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2006.07.034%26rft_id%253Dinfo%253Apmid%252F17101447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2006.07.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17101447&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242902600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.55" data-doi="10.1038/nm1161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segrè</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deluna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishony</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Modular epistasis in yeast metabolism.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegr%25C3%25A8%26rft.auinit1%253DD.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D83%26rft.atitle%253DModular%2Bepistasis%2Bin%2Byeast%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1161%26rft_id%253Dinfo%253Apmid%252F15592468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592468&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225997500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senturk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirole</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A rapid and tunable method to temporally control Cas9 expression enables the identification of essential genes and the interrogation of functional gene interactions in vitro and in vivo.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. Available at: <a href="http://dx.doi.org/10.1101/023366">http://dx.doi.org/10.1101/023366</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.57" data-doi="10.1101/gr.093955.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mellor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaper</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative phenotyping via deep barcode sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1836</span>–<span class="cit-lpage">1842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.093955.109%26rft_id%253Dinfo%253Apmid%252F19622793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xODM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.58" data-doi="10.1186/s13059-016-0900-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suresh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlecht</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagih</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Quantitative CRISPR interference screens in yeast identify chemical-genetic interactions and new rules for guide RNA design.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0900-9%26rft_id%253Dinfo%253Apmid%252F26956608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-0900-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26956608&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.59" data-doi="10.1093/nar/gkj109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">B.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reguly</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">BioGRID: a general repository for interaction datasets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D535</span>–<span class="cit-lpage">D539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStark%26rft.auinit1%253DC.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD535%26rft.epage%253DD539%26rft.atitle%253DBioGRID%253A%2Ba%2Bgeneral%2Brepository%2Bfor%2Binteraction%2Bdatasets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj109%26rft_id%253Dinfo%253Apmid%252F16381927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8xL0Q1MzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.60" data-doi="10.1038/ng1948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DOnge%26rft.auinit1%253DR.%2BP.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D199%26rft.epage%253D206%26rft.atitle%253DSystematic%2Bpathway%2Banalysis%2Busing%2Bhigh-resolution%2Bfitness%2Bprofiling%2Bof%2Bcombinatorial%2Bgene%2Bdeletions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1948%26rft_id%253Dinfo%253Apmid%252F17206143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17206143&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.61" data-doi="10.1371/journal.pbio.1001935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szamecz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boross</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalapis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovács</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fekete</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genomic landscape of compensatory evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1001935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D12%26rft.spage%253De1001935%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001935%26rft_id%253Dinfo%253Apmid%252F25157590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25157590&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.62" data-doi="10.1016/j.molcel.2013.09.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dayhoff-Brannigan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">W.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sing</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide consequences of deleting any single gene.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D52%26rft.spage%253D485%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.09.026%26rft_id%253Dinfo%253Apmid%252F24211263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.09.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24211263&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327418500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.63" data-doi="10.1016/S0580-9517(06)36016-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High-throughput strain construction and systematic synthetic lethal screening in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Methods Microbiol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMicrobiol.%26rft.volume%253D36%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0580-9517%252806%252936016-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0580-9517(06)36016-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.64" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.65" data-doi="10.1126/science.1091317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesage</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Global mapping of the yeast genetic interaction network.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D303%26rft.issue%253D5659%26rft.spage%253D808%26rft.epage%253D813%26rft.atitle%253DGlobal%2BMapping%2Bof%2Bthe%2BYeast%2BGenetic%2BInteraction%2BNetwork%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1091317%26rft_id%253Dinfo%253Apmid%252F14764870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1OS84MDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.66" data-doi="10.1534/genetics.107.086603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vernon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobachev</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High rates of “unselected” aneuploidy and chromosome rearrangements in tel1 mec1 haploid yeast strains.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.086603%26rft_id%253Dinfo%253Apmid%252F18458104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvMjM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.67" data-doi="10.1016/j.molcel.2011.10.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuica-Ross</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNase H and multiple RNA biogenesis factors cooperate to prevent RNA:DNA hybrids from generating genome instability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DWahba%26rft.auinit1%253DL.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D978%26rft.epage%253D988%26rft.atitle%253DRNase%2BH%2Band%2Bmultiple%2BRNA%2Bbiogenesis%2Bfactors%2Bcooperate%2Bto%2Bprevent%2BRNA%253ADNA%2Bhybrids%2Bfrom%2Bgenerating%2Bgenome%2Binstability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.10.017%26rft_id%253Dinfo%253Apmid%252F22195970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.10.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22195970&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298827200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.68" data-doi="10.1038/nmeth896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peisajovich</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magdassi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tawfik</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Amplification of complex gene libraries by emulsion PCR.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DWilliams%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253D545%26rft.epage%253D550%26rft.atitle%253DAmplification%2Bof%2Bcomplex%2Bgene%2Blibraries%2Bby%2Bemulsion%2BPCR.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth896%26rft_id%253Dinfo%253Apmid%252F16791213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16791213&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238669600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.69" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.70" data-doi="10.1073/pnas.0406614101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Combining biological networks to predict genetic interactions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15682</span>–<span class="cit-lpage">15687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0406614101%26rft_id%253Dinfo%253Apmid%252F15496468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQ0LzE1NjgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.71" data-doi="10.1186/1471-2105-10-80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tammi</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CNV-seq, a new method to detect copy number variation using high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DXie%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D80%26rft.epage%253D80%26rft.atitle%253DCNV-seq%252C%2Ba%2Bnew%2Bmethod%2Bto%2Bdetect%2Bcopy%2Bnumber%2Bvariation%2Busing%2Bhigh-throughput%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-80%26rft_id%253Dinfo%253Apmid%252F19267900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19267900&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.72" data-doi="10.1038/msb4100034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peyser</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene function prediction from congruent synthetic lethal interactions in yeast</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2005.0026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DYe%26rft.auinit1%253DP.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D2005.0026%26rft.epage%253D2005.0026%26rft.atitle%253DGene%2Bfunction%2Bprediction%2Bfrom%2Bcongruent%2Bsynthetic%2Blethal%2Binteractions%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100034%26rft_id%253Dinfo%253Apmid%252F16729061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjEzOiIxLzEvMjAwNS4wMDI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.73" data-doi="10.1073/pnas.1211150109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manor</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbst</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosomal duplication is a transient evolutionary solution to stress.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">21010</span>–<span class="cit-lpage">21015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211150109%26rft_id%253Dinfo%253Apmid%252F23197825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzUxLzIxMDEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.74" data-doi="10.1038/nmeth.1597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebreab</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next-generation sequencing to generate interactome datasets.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">478</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYu%26rft.auinit1%253DH.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D478%26rft.epage%253D480%26rft.atitle%253DNext-generation%2Bsequencing%2Bto%2Bgenerate%2Binteractome%2Bdatasets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1597%26rft_id%253Dinfo%253Apmid%252F21516116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1597&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21516116&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291031800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.143.75" data-doi="10.1073/pnas.1119675109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hechter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mystery of missing heritability: genetic interactions create phantom heritability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1119675109%26rft_id%253Dinfo%253Apmid%252F22223662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzQvMTE5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/143.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/129" title="The Neurospora Transcription Factor ADV-1 Transduces Light Signals and Temporal Information to Control Rhythmic Expression of Genes Involved in Cell Fusion" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/155" title="The Physical Genome Mapping of Anopheles albimanus Corrected Scaffold Misassemblies and Identified Interarm Rearrangements in Genus Anopheles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/143.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YTcSZG4qRg_qv2kNsxvsOdTCXDiuJnme204fuI5HIPk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/143" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/143" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/143" class="active" data-icon-position="" data-hide-link-title="0">iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460542" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-12W_0Z8cuXjO_ph0Y20l2Htg2N2X9k1qDKgZMkL0Wm4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/143.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/143" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-goWmWj-0ZbKyHu-OCtKDx7SZIzsnm6L8T49K2acWpj4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/143" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/143.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460542" id="node460542--2" data-pisa="ggg;7/1/143" data-pisa-master="ggg;g3.116.034207" data-apath="/ggg/7/1/143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0836-2458" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mia Jaffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1692-4983" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gavin Sherlock</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0923-1636" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sasha F. Levy</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">143-153;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034207 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460542--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mia Jaffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford School of Medicine, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mia%2BJaffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaffe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMia%2BJaffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0836-2458" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mia Jaffe</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gavin Sherlock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford School of Medicine, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gavin%2BSherlock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherlock%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGavin%2BSherlock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1692-4983" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gavin Sherlock</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sasha F. Levy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Laufer Center for Physical and Quantitative Biology, Stony Brook University, New York 11794</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biochemistry & Cell Biology, Stony Brook University, New York 11794</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sasha%2BF.%2BLevy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Levy%20SF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASasha%2BF.%2BLevy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0923-1636" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sasha F. Levy</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sasha.levy@stonybrook.edu" class="" data-icon-position="" data-hide-link-title="0">sasha.levy@stonybrook.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/143" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/143&amp;title=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/143&amp;title=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/143&amp;title=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/143&amp;text=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/143&amp;title=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/143&amp;t=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/143&amp;title=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/143&amp;title=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460542/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460542" id="node460542--3" data-pisa="ggg;7/1/143" data-pisa-master="ggg;g3.116.034207" data-apath="/ggg/7/1/143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">iSeq: A New Double-Barcode Method for Detecting Dynamic Genetic Interactions in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0836-2458" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mia Jaffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1692-4983" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gavin Sherlock</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0923-1636" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sasha F. Levy</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">143-153;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034207 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460542--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mia Jaffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford School of Medicine, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mia%2BJaffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaffe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMia%2BJaffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0836-2458" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mia Jaffe</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gavin Sherlock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford School of Medicine, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gavin%2BSherlock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherlock%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGavin%2BSherlock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1692-4983" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gavin Sherlock</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sasha F. Levy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Laufer Center for Physical and Quantitative Biology, Stony Brook University, New York 11794</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biochemistry & Cell Biology, Stony Brook University, New York 11794</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sasha%2BF.%2BLevy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Levy%20SF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASasha%2BF.%2BLevy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0923-1636" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sasha F. Levy</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sasha.levy@stonybrook.edu" class="" data-icon-position="" data-hide-link-title="0">sasha.levy@stonybrook.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=143&amp;rft.epage=153&amp;rft.atitle=iSeq%3A%20A%20New%20Double-Barcode%20Method%20for%20Detecting%20Dynamic%20Genetic%20Interactions%20in%20Yeast&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jaffe%2C+Mia&amp;rft.au=Sherlock%2C+Gavin&amp;rft.au=Levy%2C+Sasha+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460542/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460542/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460542/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460542/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460542/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460542/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460542/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460542/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460542/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460542/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460542/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460542/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/143" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/143.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/143.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/1/143.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/1/143.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
