<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/233" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/233.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/233.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/1/233.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/233.atom" />
<meta name="HW.pisa" content="ggg;7/1/233" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch" />
<meta name="DC.Identifier" content="10.1534/g3.116.036160" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Cromie et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Aneuploidy, a state in which the chromosome number deviates from a multiple of the haploid count, significantly impacts human health. The phenotypic consequences of aneuploidy are believed to arise from gene expression changes associated with the altered copy number of genes on the aneuploid chromosomes. To dissect the mechanisms underlying altered gene expression in aneuploids, we used RNA-seq to measure transcript abundance in colonies of the haploid Saccharomyces cerevisiae strain F45 and two aneuploid derivatives harboring disomies of chromosomes XV and XVI. F45 colonies display complex “fluffy” morphologies, while the disomic colonies are smooth, resembling laboratory strains. Our two disomes displayed similar transcriptional profiles, a phenomenon not driven by their shared smooth colony morphology nor simply by their karyotype. Surprisingly, the environmental stress response (ESR) was induced in F45, relative to the two disomes. We also identified genes whose expression reflected a nonlinear interaction between the copy number of a transcriptional regulatory gene on chromosome XVI, [DIG1][1] , and the copy number of other chromosome XVI genes. [DIG1][1] and the remaining chromosome XVI genes also demonstrated distinct contributions to the effect of the chromosome XVI disome on ESR gene expression. Expression changes in aneuploids appear to reflect a mixture of effects shared between different aneuploidies and effects unique to perturbing the copy number of particular chromosomes, including nonlinear copy number interactions between genes. The balance between these two phenomena is likely to be genotype- and environment-specific.

 [1]: http://www.yeastgenome.org/locus/S000005970/overview" />
<meta name="DC.Contributor" content="Gareth A. Cromie" />
<meta name="DC.Contributor" content="Zhihao Tan" />
<meta name="DC.Contributor" content="Michelle Hays" />
<meta name="DC.Contributor" content="Eric W. Jeffery" />
<meta name="DC.Contributor" content="Aimée M. Dudley" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Aneuploidy, a state in which the chromosome number deviates from a multiple of the haploid count, significantly impacts human health. The phenotypic consequences of aneuploidy are believed to arise from gene expression changes associated with the altered copy number of genes on the aneuploid chromosomes. To dissect the mechanisms underlying altered gene expression in aneuploids, we used RNA-seq to measure transcript abundance in colonies of the haploid &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; strain F45 and two aneuploid derivatives harboring disomies of chromosomes XV and XVI. F45 colonies display complex “fluffy” morphologies, while the disomic colonies are smooth, resembling laboratory strains. Our two disomes displayed similar transcriptional profiles, a phenomenon not driven by their shared smooth colony morphology nor simply by their karyotype. Surprisingly, the environmental stress response (ESR) was induced in F45, relative to the two disomes. We also identified genes whose expression reflected a nonlinear interaction between the copy number of a transcriptional regulatory gene on chromosome XVI, &lt;i&gt;DIG1&lt;/i&gt;, and the copy number of other chromosome XVI genes. &lt;i&gt;DIG1&lt;/i&gt; and the remaining chromosome XVI genes also demonstrated distinct contributions to the effect of the chromosome XVI disome on ESR gene expression. Expression changes in aneuploids appear to reflect a mixture of effects shared between different aneuploidies and effects unique to perturbing the copy number of particular chromosomes, including nonlinear copy number interactions between genes. The balance between these two phenomena is likely to be genotype- and environment-specific.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/233" />
<meta name="citation_id" content="7/1/233" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/233" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/233.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/233.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/233.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036160" />
<meta name="citation_pmid" content="27836908" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="233" />
<meta name="citation_lastpage" content="246" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gareth A. Cromie" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author" content="Zhihao Tan" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_institution" content="Institute of Medical Biology, Agency for Science, Technology and Research, Singapore" />
<meta name="citation_author" content="Michelle Hays" />
<meta name="citation_author_institution" content="Molecular and Cellular Biology Program, University of Washington, Seattle, Washington" />
<meta name="citation_author" content="Eric W. Jeffery" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author" content="Aimée M. Dudley" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_institution" content="Molecular and Cellular Biology Program, University of Washington, Seattle, Washington" />
<meta name="citation_author_email" content="aimee.dudley@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3644-0625" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=C. Almaguer;citation_author=D. Mantella;citation_author=E. Perez;citation_author=J. Patton-Vogt;citation_title=Inositol and Phosphate Regulate GIT1 Transcription and Glycerophosphoinositol Incorporation in Saccharomyces cerevisiae;citation_pages=729-736;citation_volume=2;citation_year=2003;citation_issue=4;citation_pmid=12912892;citation_doi=10.1128/EC.2.4.729-736.2003" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=A. R. Borneman;citation_author=J. A. Leigh-Bell;citation_author=H. Yu;citation_author=P. Bertone;citation_author=M. Gerstein;citation_title=Target hub proteins serve as master regulators of development in yeast.;citation_volume=20;citation_year=2006;citation_pmid=16449570;citation_doi=10.1101/gad.1389306" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Chou;citation_author=S. Lane;citation_author=H. Liu;citation_title=Regulation of Mating and Filamentation Genes by Two Distinct Ste12 Complexes in Saccharomyces cerevisiae;citation_pages=4794-4805;citation_volume=26;citation_year=2006;citation_issue=13;citation_pmid=16782869;citation_doi=10.1128/MCB.02053-05" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=A. Coluccio;citation_author=E. Bogengruber;citation_author=M. N. Conrad;citation_author=M. E. Dresser;citation_author=P. Briza;citation_title=Morphogenetic Pathway of Spore Wall Assembly in Saccharomyces cerevisiae;citation_pages=1464-1475;citation_volume=3;citation_year=2004;citation_issue=6;citation_pmid=15590821;citation_doi=10.1128/EC.3.6.1464-1475.2004" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. G. Cook;citation_author=L. Bardwell;citation_author=S. J. Kron;citation_author=J. Thorner;citation_title=Two novel targets of the MAP kinase Kss1 are negative regulators of invasive growth in the yeast Saccharomyces cerevisiae.;citation_pages=2831-2848;citation_volume=10;citation_year=1996;citation_issue=22;citation_pmid=8918885;citation_doi=10.1101/gad.10.22.2831" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=N. Dephoure;citation_author=S. Hwang;citation_author=C. O’Sullivan;citation_author=S. E. Dodgson;citation_author=S. P. Gygi;citation_title=Quantitative proteomic analysis reveals posttranslational responses to aneuploidy in yeast.;citation_volume=3;citation_year=2014;citation_pmid=25073701;citation_doi=10.7554/eLife.03023" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. E. Dodgson;citation_author=S. Kim;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=D. L. Morse;citation_title=Chromosome-specific and global effects of aneuploidy in Saccharomyces cerevisiae.;citation_volume=202;citation_year=2016;citation_pmid=26837754;citation_doi=10.1534/genetics.115.185660" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=A. P. Gasch;citation_author=P. T. Spellman;citation_author=C. M. Kao;citation_author=O. Carmel-Harel;citation_author=M. B. Eisen;citation_title=Genomic Expression Programs in the Response of Yeast Cells to Environmental Changes;citation_pages=4241-4257;citation_volume=11;citation_year=2000;citation_issue=12;citation_pmid=11102521;citation_doi=10.1091/mbc.11.12.4241" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=A. P. Gasch;citation_author=J. Hose;citation_author=M. A. Newton;citation_author=M. Sardi;citation_author=M. Yong;citation_title=Further support for aneuploidy tolerance in wild yeast and effects of dosage compensation on gene copy-number evolution.;citation_pages=e14409;citation_volume=5;citation_year=2016;citation_pmid=26949252" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=V. Gavrias;citation_author=A. Andrianopoulos;citation_author=C. J. Gimeno;citation_author=W. E. Timberlake;citation_title=Saccharomyces cerevisiae TEC1 is required for pseudohyphal growth.;citation_pages=1255-1263;citation_volume=19;citation_year=1996;citation_issue=6;citation_pmid=8730867;citation_doi=10.1111/j.1365-2958.1996.tb02470.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. J. Gordon;citation_author=B. Resio;citation_author=D. Pellman;citation_title=Causes and consequences of aneuploidy in cancer.;citation_pages=189-203;citation_volume=13;citation_year=2012;citation_issue=3;citation_pmid=22269907;citation_doi=10.1038/nrg3123" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. A. Granek;citation_author=P. M. Magwene;citation_title=Environmental and genetic determinants of colony morphology in yeast.;citation_pages=e1000823-e1000823;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=20107600;citation_doi=10.1371/journal.pgen.1000823" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Guo;citation_author=C. A. Styles;citation_author=Q. Feng;citation_author=G. R. Fink;citation_title=A Saccharomyces gene family involved in invasive growth, cell-cell adhesion, and mating.;citation_volume=97;citation_year=2000;citation_pmid=11027318;citation_doi=10.1073/pnas.220420397" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Halme;citation_author=S. Bumgarner;citation_author=C. Styles;citation_author=G. R. Fink;citation_title=Genetic and epigenetic regulation of the FLO gene family generates cell-surface variation in yeast.;citation_pages=405-415;citation_volume=116;citation_year=2004;citation_issue=3;citation_pmid=15016375;citation_doi=10.1016/S0092-8674(04)00118-7" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=Dosage compensation can buffer copy-number variation in wild yeast.;citation_author=J. Hose;citation_author=C. M. Yong;citation_author=M. Sardi;citation_author=Z. Wang;citation_author=M. A. Newton;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. R. Hughes;citation_author=C. J. Roberts;citation_author=H. Dai;citation_author=A. R. Jones;citation_author=M. R. Meyer;citation_title=Widespread aneuploidy revealed by DNA microarray expression profiling.;citation_pages=333-337;citation_volume=25;citation_year=2000;citation_issue=3;citation_pmid=10888885;citation_doi=10.1038/77116" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. Karunanithi;citation_author=N. Vadaie;citation_author=C. A. Chavel;citation_author=B. Birkaya;citation_author=J. Joshi;citation_title=Shedding of the mucin-like flocculin Flo11p reveals a new aspect of fungal adhesion regulation.;citation_pages=1389-1395;citation_volume=20;citation_year=2010;citation_issue=15;citation_pmid=20619652;citation_doi=10.1016/j.cub.2010.06.033" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Lardenois;citation_author=Y. Liu;citation_author=T. Walther;citation_author=F. Chalmel;citation_author=B. Evrard;citation_title=Execution of the meiotic noncoding RNA expression program and the onset of gametogenesis in yeast require the conserved exosome subunit Rrp6.;citation_volume=108;citation_year=2011;citation_pmid=21149693;citation_doi=10.1073/pnas.1016459108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Liao;citation_author=G. K. Smyth;citation_author=W. Shi;citation_title=FeatureCounts: an efficient general purpose program for assigning sequence reads to genomic features.;citation_volume=30;citation_year=2014;citation_pmid=24227677;citation_doi=10.1093/bioinformatics/btt656" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. O. Ljungdahl;citation_author=B. Daignan-Fornier;citation_title=Regulation of Amino Acid, Nucleotide, and Phosphate Metabolism in Saccharomyces cerevisiae;citation_pages=885-929;citation_volume=190;citation_year=2012;citation_issue=3;citation_pmid=22419079;citation_doi=10.1534/genetics.111.133306" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=W. S. Lo;citation_author=A. M. Dranginis;citation_title=The Cell Surface Flocculin Flo11 Is Required for Pseudohyphae Formation and Invasion by Saccharomyces cerevisiae;citation_pages=161-171;citation_volume=9;citation_year=1998;citation_issue=1;citation_pmid=9436998;citation_doi=10.1091/mbc.9.1.161" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=M. C. Lorenz;citation_author=J. Heitman;citation_title=The MEP2 ammonium permease regulates pseudohyphal differentiation in Saccharomyces cerevisiae;citation_pages=1236-1247;citation_volume=17;citation_year=1998;citation_issue=5;citation_pmid=9482721;citation_doi=10.1093/emboj/17.5.1236" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. C. Lorenz;citation_author=J. Heitman;citation_title=Regulators of Pseudohyphal Differentiation in Saccharomyces cerevisiae Identified Through Multicopy Suppressor Analysis in Ammonium Permease Mutant Strains;citation_pages=1443-1457;citation_volume=150;citation_year=1998;citation_issue=4;citation_pmid=9832522" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. J. Lyons;citation_author=A. P. Gasch;citation_author=L. A. Gaither;citation_author=D. Botstein;citation_author=P. O. Brown;citation_title=Genome-wide characterization of the Zap1p zinc-responsive regulon in yeast;citation_pages=7957-7962;citation_volume=97;citation_year=2000;citation_issue=14;citation_pmid=10884426;citation_doi=10.1073/pnas.97.14.7957" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. D. Madhani;citation_author=G. R. Fink;citation_title=Combinatorial Control Required for the Specificity of Yeast MAPK Signaling;citation_pages=1314-1317;citation_volume=275;citation_year=1997;citation_issue=5304;citation_pmid=9036858;citation_doi=10.1126/science.275.5304.1314" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=R. Mao;citation_author=C. L. Zielke;citation_author=H. R. Zielke;citation_author=J. Pevsner;citation_title=Global up-regulation of chromosome 21 gene expression in the developing Down syndrome brain.;citation_pages=457-467;citation_volume=81;citation_year=2003;citation_issue=5;citation_pmid=12706104;citation_doi=10.1016/S0888-7543(03)00035-1" />
<meta name="citation_reference" content="citation_journal_title=EMBnet J.;citation_author=M. Marcel;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads.;citation_pages=10-12;citation_volume=17;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=A. B. Oromendia;citation_author=S. E. Dodgson;citation_author=A. Amon;citation_title=Aneuploidy causes proteotoxic stress in yeast.;citation_volume=26;citation_year=2012;citation_pmid=23222101;citation_doi=10.1101/gad.207407.112" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Pavelka;citation_author=G. Rancati;citation_author=J. Zhu;citation_author=W. D. Bradford;citation_author=A. Saraf;citation_title=Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.;citation_pages=321-325;citation_volume=468;citation_year=2010;citation_issue=7321;citation_pmid=20962780;citation_doi=10.1038/nature09529" />
<meta name="citation_reference" content="citation_title=Practical Iterative de Bruijn Graph De Novo Assembler.;citation_author=Y. Peng;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=A. Pitoniak;citation_author=B. Birkaya;citation_author=H. M. Dionne;citation_author=N. Vadaie;citation_author=P. J. Cullen;citation_title=The signaling mucins Msb2 and Hkr1 differentially regulate the filamentation mitogen-activated protein kinase pathway and contribute to a multimodal response.;citation_volume=20;citation_year=2009;citation_pmid=19439450;citation_doi=10.1091/mbc.E08-07-0760" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Rancati;citation_author=N. Pavelka;citation_author=B. Fleharty;citation_author=A. Noll;citation_author=R. Trimble;citation_title=Aneuploidy underlies rapid adaptive evolution of yeast cells deprived of a conserved cytokinesis motor.;citation_pages=879-893;citation_volume=135;citation_year=2008;citation_issue=5;citation_pmid=19041751;citation_doi=10.1016/j.cell.2008.09.039" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Regenberg;citation_author=T. Grotkjaer;citation_author=O. Winther;citation_author=A. Fausboll;citation_author=M. Akesson;citation_title=Growth-rate regulated genes have profound impact on interpretation of transcriptome profiling in Saccharomyces cerevisiae.;citation_pages=R107-R107;citation_volume=7;citation_year=2006;citation_issue=11;citation_pmid=17105650;citation_doi=10.1186/gb-2006-7-11-r107" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics: A Laboratory Course Manual.;citation_author=M. D. Rose;citation_author=F. M. Winston;citation_author=P. Hieter;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=S. Rupp;citation_author=E. Summers;citation_author=H. J. Lo;citation_author=H. Madhani;citation_author=G. Fink;citation_title=MAP kinase and cAMP filamentation signaling pathways converge on the unusually large promoter of the yeast FLO11 gene;citation_pages=1257-1269;citation_volume=18;citation_year=1999;citation_issue=5;citation_pmid=10064592;citation_doi=10.1093/emboj/18.5.1257" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=A. Salehi;citation_author=J. D. Delcroix;citation_author=P. V. Belichenko;citation_author=K. Zhan;citation_author=C. Wu;citation_title=Increased App expression in a mouse model of Down's syndrome disrupts NGF transport and causes cholinergic neuron degeneration.;citation_pages=29-42;citation_volume=51;citation_year=2006;citation_issue=1;citation_pmid=16815330;citation_doi=10.1016/j.neuron.2006.05.022" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. M. Sheltzer;citation_author=A. Amon;citation_title=The aneuploidy paradox: costs and benefits of an incorrect karyotype.;citation_pages=446-453;citation_volume=27;citation_year=2011;citation_issue=11;citation_pmid=21872963;citation_doi=10.1016/j.tig.2011.07.003" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. M. Sheltzer;citation_author=E. M. Torres;citation_author=M. J. Dunham;citation_author=A. Amon;citation_title=Transcriptional consequences of aneuploidy.;citation_volume=109;citation_year=2012;citation_pmid=22802626;citation_doi=10.1073/pnas.1209227109" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. S. Swanson;citation_author=E. A. Malone;citation_author=F. Winston;citation_title=SPT5, an essential gene important for normal transcription in Saccharomyces cerevisiae, encodes an acidic nuclear protein with a carboxy-terminal repeat.;citation_pages=3009-3019;citation_volume=11;citation_year=1991;citation_issue=6;citation_pmid=1840633;citation_doi=10.1128/MCB.11.6.3009" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Tan;citation_author=M. Hays;citation_author=G. A. Cromie;citation_author=E. W. Jeffery;citation_author=A. C. Scott;citation_title=Aneuploidy underlies a multicellular phenotypic switch.;citation_volume=110;citation_year=2013;citation_pmid=23812752;citation_doi=10.1073/pnas.1301047110" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=C. Taxis;citation_author=M. Knop;citation_title=System of centromeric, episomal, and integrative vectors based on drug resistance markers for Saccharomyces cerevisiae.;citation_pages=73-78;citation_volume=40;citation_year=2006;citation_issue=1;citation_pmid=16454043;citation_doi=10.2144/000112040" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. M. Torres;citation_author=T. Sokolsky;citation_author=C. M. Tucker;citation_author=L. Y. Chan;citation_author=M. Boselli;citation_title=Effects of Aneuploidy on Cellular Physiology and Cell Division in Haploid Yeast;citation_pages=916-924;citation_volume=317;citation_year=2007;citation_issue=5840;citation_pmid=17702937;citation_doi=10.1126/science.1142210" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. M. Torres;citation_author=N. Dephoure;citation_author=A. Panneerselvam;citation_author=C. M. Tucker;citation_author=C. A. Whittaker;citation_title=Identification of aneuploidy-tolerating mutations.;citation_pages=71-83;citation_volume=143;citation_year=2010;citation_issue=1;citation_pmid=20850176;citation_doi=10.1016/j.cell.2010.08.038" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=E. M. Torres;citation_author=M. Springer;citation_author=A. Amon;citation_title=No current evidence for widespread dosage compensation in S. cerevisiae.;citation_pages=e10996;citation_volume=5;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=N. Vadaie;citation_author=H. Dionne;citation_author=D. S. Akajagbor;citation_author=S. R. Nickerson;citation_author=D. J. Krysan;citation_title=Cleavage of the signaling mucin Msb2 by the aspartyl protease Yps1 is required for MAPK activation in yeast.;citation_volume=181;citation_year=2008;citation_pmid=18591427;citation_doi=10.1083/jcb.200704079" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. L. van Dijk;citation_author=C. L. Chen;citation_author=Y. d’Aubenton-Carafa;citation_author=S. Gourvennec;citation_author=M. Kwapisz;citation_title=XUTs are a class of Xrn1-sensitive antisense regulatory non-coding RNA in yeast.;citation_pages=114-117;citation_volume=475;citation_year=2011;citation_issue=7354;citation_pmid=21697827;citation_doi=10.1038/nature10118" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=K. Voordeckers;citation_author=D. De Maeyer;citation_author=E. van der Zande;citation_author=M. D. Vinces;citation_author=W. Meert;citation_title=Identification of a complex genetic network underlying Saccharomyces cerevisiae colony morphology.;citation_pages=225-239;citation_volume=86;citation_year=2012;citation_issue=1;citation_pmid=22882838;citation_doi=10.1111/j.1365-2958.2012.08192.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Z. Xu;citation_author=W. Wei;citation_author=J. Gagneur;citation_author=F. Perocchi;citation_author=S. Clauder-Munster;citation_title=Bidirectional promoters generate pervasive transcription in yeast.;citation_pages=1033-1037;citation_volume=457;citation_year=2009;citation_issue=7232;citation_pmid=19169243;citation_doi=10.1038/nature07728" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Yona;citation_author=Y. S. Manor;citation_author=R. H. Herbst;citation_author=G. H. Romano;citation_author=A. Mitchell;citation_title=Chromosomal duplication is a transient evolutionary solution to stress.;citation_volume=109;citation_year=2012;citation_pmid=23197825;citation_doi=10.1073/pnas.1211150109" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="Aneuploidy, a state in which the chromosome number deviates from a multiple of the haploid count, significantly impacts human health. The phenotypic consequences of aneuploidy are believed to arise from gene expression changes associated with the altered copy number of genes on the aneuploid chromosomes. To dissect the mechanisms underlying altered gene expression in aneuploids, we used RNA-seq to measure transcript abundance in colonies of the haploid Saccharomyces cerevisiae strain F45 and two aneuploid derivatives harboring disomies of chromosomes XV and XVI. F45 colonies display complex “fluffy” morphologies, while the disomic colonies are smooth, resembling laboratory strains. Our two disomes displayed similar transcriptional profiles, a phenomenon not driven by their shared smooth colony morphology nor simply by their karyotype. Surprisingly, the environmental stress response (ESR) was induced in F45, relative to the two disomes. We also identified genes whose expression reflected a nonlinear interaction between the copy number of a transcriptional regulatory gene on chromosome XVI, [DIG1][1] , and the copy number of other chromosome XVI genes. [DIG1][1] and the remaining chromosome XVI genes also demonstrated distinct contributions to the effect of the chromosome XVI disome on ESR gene expression. Expression changes in aneuploids appear to reflect a mixture of effects shared between different aneuploidies and effects unique to perturbing the copy number of particular chromosomes, including nonlinear copy number interactions between genes. The balance between these two phenomena is likely to be genotype- and environment-specific.

 [1]: http://www.yeastgenome.org/locus/S000005970/overview" />
<meta name="og-title" property="og:title" content="Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/233" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Aneuploidy, a state in which the chromosome number deviates from a multiple of the haploid count, significantly impacts human health. The phenotypic consequences of aneuploidy are believed to arise from gene expression changes associated with the altered copy number of genes on the aneuploid chromosomes. To dissect the mechanisms underlying altered gene expression in aneuploids, we used RNA-seq to measure transcript abundance in colonies of the haploid Saccharomyces cerevisiae strain F45 and two aneuploid derivatives harboring disomies of chromosomes XV and XVI. F45 colonies display complex “fluffy” morphologies, while the disomic colonies are smooth, resembling laboratory strains. Our two disomes displayed similar transcriptional profiles, a phenomenon not driven by their shared smooth colony morphology nor simply by their karyotype. Surprisingly, the environmental stress response (ESR) was induced in F45, relative to the two disomes. We also identified genes whose expression reflected a nonlinear interaction between the copy number of a transcriptional regulatory gene on chromosome XVI, [DIG1][1] , and the copy number of other chromosome XVI genes. [DIG1][1] and the remaining chromosome XVI genes also demonstrated distinct contributions to the effect of the chromosome XVI disome on ESR gene expression. Expression changes in aneuploids appear to reflect a mixture of effects shared between different aneuploidies and effects unique to perturbing the copy number of particular chromosomes, including nonlinear copy number interactions between genes. The balance between these two phenomena is likely to be genotype- and environment-specific.

 [1]: http://www.yeastgenome.org/locus/S000005970/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
    <title>Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"n9fPohVkh1W-FRG4FOc_ctV_MHPgZesE_Li3wNi4aH8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460539","apath":"\/ggg\/7\/1\/233.atom","pisa":"ggg;7\/1\/233","ac":{"\/ggg\/7\/1\/233.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/233.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/233"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/233"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/233.abstract":true,"\/content\/7\/1\/233":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Se4NasnzUJz+Oni0LYs8SrV09vssuwLAT9eguyFUGZQk\\\\\\\/brklmoaggGCTZ2GioUhgMHtj7G4Ucq5d9vD4zURWk4mTUjOUkeekSLcaXsk5Msq7wQmljYE9CwixpZ9r6hEX+5JnXCNlTAWLZez0\\\\\\\/2W81iJVljLrGWO3xO1\\\\\\\/2gsEykt07jEi8BRs3UQincOj3AlQ4UA3QAxtLi8LaaAktHAWhSnIf0sY+gq7BjuMibwOokzn3FhYkuhya0Fz7gzqjzpFXHknPDv+W1p2mUJWuD683bKlPN\\\\\\\/cI2qIrMq0+a0GnLZEvQBJyWOdkiZQb3deERzXcYJu2+8WzZQMx82JANHLsKN41HEzLmNRP7N\\\\\\\/1ZWNyq5Ev5G+0S+5vdX9U273uPDb0mNw06n8scF9EZpjAAaCNdp0CIQKWul7gD8o8E2+ILeekWSPrYz8zkY5M3TT7fjVzf74ReoMGgCbbbNNxlu7omqho4R7lq+HMN0M2rjBSO7RSk0VZP2GmFITulNdwKh2BAZAv4yEQpjV5itn5Zepg6zPeztymwRTKOckIY1fD4ZqE5VCzp7JiavgFmnXJ3A554KyTHnNyDirrOYOThNz6R3isQh4OrsjIoml+EgTDoDx3aZuybHfB57Hv5bHAewUTR0Cg6cMhxHKPfSYeH+A0JDKuYC0kSyWhGhQPajPrWUI9NUrwU6\\\\\\\/tcUle7BrkESRj9aIMuGdYjMVC++AdRXMJUFmVcsYIUXcZl\\\\\\\/zuWWUitos+89tmoGAs0V+U9DIh8Yq5VjaM0W8JxGUTOejQ3byMk82u5v6SdyDGRzhYb+cmlwRO6CXwylMH0br59eJl4YI8f+Bq6OHA5Ulgnt79wCbUd0E0cmL7BBp\\\\\\\/BLvHfbp5Z1NyaeFJ3mghIPFqmxkZkE\\\\\\\/Y7N8\\\\\\\/DCuZXktRoBxhLY1QFnh1S5gNg4QsSDCynUV4L2mMl9tslxMqtMO2GdhJYdXfXKvh2LcqvkcHOrh4Ws\\\\\\\/X1fiQMcvwghJcPzbUEOmJBnKNa5nwrs9k3vksgEPBnnTn+IE3K06f32ULXNqNm4xiRhJiOzhkRhr9KLi6v1Mjowsj1zLZOoKYGYIrM0kS\\\\\\\/agy06zC5E7oXe7I7FonJWEG3FPr1RzH2U+\\\\\\\/CAREs1dJSRks\\\\\\\/NIG6vPSrhBHPZaBxhbeGaAYdW8LR65QRZ3UgOHZWLaBLYdNqljO+u73J33O5EooTg+e+pvEG4dP8EGCoNQu\\\\\\\/FnPbTj5FUBbEKZpu69A88DjavCOrIRB3b3ZWEt5jnsLZFA1LeeBvgDdsKilEN\\\u0022,\\\u0022iv\\\u0022:\\\u0022UCx+A5SyOIe0RfX0DAi4Dw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022edd58c0d012dce28b0ea3436f30dd1af05c85dd0203e30e5a2c8528ce3e196e5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FIlm+UcsBRazTKK7VHFiVsgxayitEWtQJk4+tB8joIFEBD5Ryo0poUlfREIql1TuWBsRtZTJ\\\\\\\/rONkdB3ByKvCNgY8sTNC221ouqcI\\\\\\\/WAZ4x+yf8\\\\\\\/rRUU0QL10yu4PMMULTL793NZy0clM+qUR\\\\\\\/7Q4cGQMx7FR9VugsV\\\\\\\/ObWy47tvUuiWKEnOKKuGUb5Bcyk27yNO\\\\\\\/uqOpguO2sGBmaVOGcf4QlmC9bNmZqUTNnL4UMoQecpia86yWQx2Pkr\\\\\\\/ntBtuL5IMwO4MlQPOzDnIPe+gdLmDhZVq4aDKVse7qwnjNKiK2lxB\\\\\\\/RZUl0oukpH8iQEza3sHbyIgwB1nBYX9g1JzLw702cvnZ3rUOPW2DDg8xZ49zw\\\\\\\/AGSdIvV1xWY\\\\\\\/gN+80HeGxvMmqEsnXJI2zOndu9nXmGrhldIhGkzZcyL6yoJuQCmnPEwLKH7t9TeD7LAgGAO5WHeNjW9uvvVxotNyHyHLvt2DZbp3QJ1eDE2mAaMUu7BW6veiWWwj+WQEf7haFhGLwQt5vYtOnbu9kRXLsnUFWXMP\\\\\\\/YadPyn4ZhSTV++VlKn4ypUQhR7jtfZpoTOmk9XcvF9kWH+VgIr2p+skwlNS62Z1M+rHBcTvvs8R9vMS7UelbiSqX5+konG6+BFR0KZx0PDVr2d57+96\\\\\\\/lE2rM0jaVyTGvl7P\\\\\\\/xGbGapanCiWn0ooYDuONZikQwMtcsg0yalD\\\\\\\/JYvF1EnLemb0Em4Et5B+vGDjdDEeTggY8yOa1PaePus6Yh+wYytbjteDwY6W6Gh9qHiAXv4AUC3ysc1N+S\\\\\\\/y5GgvtjpLB59G208h8QiKe0LKMJxJ7m+9ds0WMCkSiiAIHgTMAg26av4pBeBE6yTnlwW0KtRgOql231HAoYy\\\\\\\/Ylz2kYzFs\\\\\\\/asUqAIK\\\\\\\/NI6N11LgesynvQY\\\\\\\/A\\\\\\\/ecwAAz3gX4QyBNJ45u0wLcz8jDXEMDAESBeINzwsPJG0jwfKPg9fFVzr2va2wgwLJnN\\\\\\\/aH4X48kh7\\\\\\\/ebSVa9trrJfitgmjAR2YBz7jahxAudFcv4XAeb10tE57PjNcI+coGcaLNvPmWzkFHdSgB\\\\\\\/p2E\\\\\\\/PoGdUmIhoM3li57zjFIwQnLxKTqjOR2sgbtUqsCsSQa5pZRXSr9u5AtJK0bt6JIHa4KDZC6w14Td9frij\\\\\\\/mMiq7UB7IwX8CFypdN8PNJcPLI5FytDIdIz\\\\\\\/zUkJTe8MAdcpl1tTRVm00CLBvMMvcrd\\\\\\\/P9k9FWnUBDSYVZUeVxrFkCWDnqb8OEHEtwZ2J9qF16H2SROwexs8buJk5ytXrBtLFc8lWeBXg3u5ZcbfznRqOhEy1YLnPlfFFcb4iKgavtu6oX3PE9WbkDx2XkxQ0xkDE5J5vmV1Hd2AzTseWUoar7ZCzYdDYQltKvWtvRUwpS4WMI5KtFxBDXpLv0DFJfHG5F1Du67zAmuIeZTTItKSgavj63aXr0\\\\\\\/j0EYXNHo7qqSKtDU4+hONDmQ9WY1bnLiStH7Bd5BX23XnYRloCZTPDjq2XquZa\\\\\\\/t6CkvHgTzFaR3LW85Ft3Ru1bfmillf2Fc843Ul1iStA2Yup\\\\\\\/2uNIEL113aCetFxFXaWKTskT9w2WzgD+gWiPbwUbIfX9BvNo+Re36kiO\\\\\\\/qTI8C7gKa9TplbqahrRjVG+nVEq54rFgHkI\\\\\\\/7aqNe+4srzFTrb2DkNVsNngyREPJ4oGkDpPjOvbjZjbBB8bXfKr4Wey+03RNfQu7bux8s910UeX5o3rWmNlvTQaswyycg9oPYQZymEud+7YjkgDFO6I8vY6PrqbYa4Ykb8Qh6OmZQ\\\\\\\/Q1yKx0Q6WSyl+Ll+aRP58IETj\\\\\\\/3KshZ7zQFxgWSUyARA\\\\\\\/7fxBakL9LISoYGs4qh3LhraJSF7XnmAN7pCznvCfB4Z72oCrtOrwB21iHr8RbXv7Rp9INdfJMyGkfwKDWdsPMeJ+z+odUQ9VDKpFDPLWgAeAwJM8mhLiP096lM+6umKr5fPdG7Nz7gNkViNmEvNetMVg4gTJjAm9TjqQJfgqCaE59nGQ7jqTxW8foWaqxpCaD0gk87t8uz2dAnUa1e9I\\\\\\\/aYAr\\\\\\\/\\\\\\\/WkgM2nJ0z6DWBGIlDqTwZeTve+qCO\\\\\\\/6\\\\\\\/u9tT60JJGYaHno5lo4PbZ5N1haO7dSany7x7lZxhs\\\\\\\/ydGlIdLZMBnK+VkoSdZ514eyV\\\\\\\/NNpZ\\\\\\\/fP8EOOLugj918NJI3wKbZEdI2gs\\\\\\\/7knG+2joMXyzlyrXZhwBkguIzMDhIjRGspeaZKnMyet59xwUG\\\\\\\/5mXKwNtfEwqzjn4M5HKAlqh+PpTAJ9p87zdFOv1xlo1fNuVDSnocgGPqy4KX95qKLddn5RN7TH37kbxn8vsOouyTXFUtp47uJVr6LE1hOAgErQP\\\\\\\/KvkodlmB8AQ6mKhV+StyyeVxQeb\\\\\\\/7CMTnfPZKC70ABPy2XusuA+ZZeOekQVxfbvajOsW+KBIGNVvWF4mgdIqnTnY5NDEl4e6+GZR+mWlnc3B7pkKHPl3aE0lwgzGxECUdYzCYDAXF0sF6aUM67SyNeYuGqy1pYEvJNTbAhG2PmA7\\\\\\\/w2YqXBCRzC9eW5G+sYLGqi+7OQDA4SP+qYxv9wNT6Vk5jBheWCpze2oQDnTx5z2UbM1Yb0xAi1wDWaoBdsOuIbZPDNj5X+flp6SDM6CYu3P4kjvqvmzzWDKTpSsK50rzJ3UFDkS0bLlVhysl6St1LmaPUPisGEwbRcI4pkxVoKSL3UW+1SgBO7TEFWwD02LdLgDTlsO\\\\\\\/zCOydWeUbuRyArvIBb3r0Z876i\\\\\\\/ZN3V\\\\\\\/q9b2ZXOc9nT6BtDP9ZTBdtAF6QMHXGaxW7PlITG0gKFhH1nowVuKfyp4zh\\\\\\\/7qwo37m830htDCKLjV7m0gZlUZNHwrehTDxkmU6L7v\\\\\\\/DLrODkH+8aPS+JqtmUTfTJZohkXQXGM+X\\\\\\\/96lkJmnGJvsY17KLvGAN89YUDR6uvZGuih7Dng1X7ujClwMnKgxG2GVkLsrCRdU2DU+FjK2M+pLT427I7UoDWSLmRK+4cbs0qCAA+C1PXQEuvrrJGka45qbkRRhID\\\\\\\/KVP7U2FOFOaWmfiG4SiDX5BzJgBnODJ\\\\\\\/ngJe5QXGMGENXgkfv\\\\\\\/vDvmAsZOaE1+PhfPkk76\\\\\\\/VMWjdZ5vTVU74S0A7HBC0TyUyNVHoQwcVz9aFJ4WRFjgFI5m7elOnU\\\\\\\/oRmEZbK7bvsIo3qBqX9awFZSWGHvDYr60WSbb71bjn2Psf5qzebip19ypVfYfNbBFCWvhFvE4XG3dClQlIY6l5yb4iInjKND+5NYJxjfEWloRCeHeOxK8GTFULcJXIcrpzObaAjK6\\\\\\\/W2trDZVdcaQMCNxQl78CHYS2Ugg+X8\\\\\\\/8CpvhyBQ\\\\\\\/GnMhrvAt+f4hT9T3sFyAV9F0rzAV9u7nmUmKewJkgpG+R2+pGompWGYr5SAAa4p9\\\\\\\/DjBffYmox5ZUnDiJmN84O1Cn4Kxu\\\\\\\/Z7k4FIiwo9Vg\\\\\\\/MkGW2uwvfweM5sdrw9BUv5WnF9No4sAGW+rIWEwrcJEn2PfnOFUPHBor71o0bfTRgCORXane1okXx\\\\\\\/ROnKFEopEwLg6Ce9RiUL52MPOgqww1rfW4A3gar6d6oR8qt0p2XLCQWdMIue9GBp2d3VL3RJg5W24cR2NepPFcInUStIPKR6JAlX0IL1JQ5ScBJ72pZBZ97BUzgU8buaylyU8qoT8usldOhovw\\\\\\\/bF1oSL68u6\\\\\\\/thc9zfVYOJB40piTqTWnPFOzFgW7LgSNu1bye5+iDBjKg4GO3yQKOHHIz6axabkREM7dRDL3d4Kci694toRlU93uVe+bcEFTubHURpWqhqs3Jgh5jlpvNI9zLEDP6oGOSUNNERN\\\\\\\/xjRNHdwonGnoa3w8P3iWdUwAHcgPme6XtDzP35\\\\\\\/WIfboCWg1qX9GvipEjAu0kddrIJbwRb\\\\\\\/9K2\\\\\\\/sqmb6858czjU9DN0UrUBmms11FyE2GQfcXfS+t\\\\\\\/dq8b8eyxOqyLBoFWa8P9r+R7vsRtLK+HgxNS4pr9j0fTlvcKYC\\\\\\\/4t1kHAT8YpzYOn0LrBbF2Iezck7hHR9GX\\\\\\\/W0AfZO9z49S8EuUTjEGVvS6B7FedvHKorKSWBEcWLtChuDqhye1L4wocbM9bfw7obhMlJPwSVsDKRL0VIH+vrsa4NEZf8a\\\\\\\/30eknGxA2LsobohVkZsT30vIsMGGkblEhFGvApEaFxx3rV87jbWoHQlWI9ERhWZqFUsUN8Q6uafCU4\\\\\\\/JYuYi3nClHCSr\\\\\\\/WzXhEfpYDazYeoeCZfXoj8UKDUmrfvDnFyMlTFNCwfcnc\\\\\\\/\\\\\\\/l+C0qAtMqTLGvWdrEORNjBQSZHxSpyAP0FAakOiUie3RQc7ykZdWJGwqfTNHqK0RzSkJdmEsPJT2eE6faxbiRPL38AzIn1ejnATgGhFlLMyX0ujqyDG5ydvoqI63+JRreYnF6tZ2j7zxMjhObCORoj\\\\\\\/XGXC6dakFsFIpJPZcEXKmxU4roX2a9LBYRZ\\\\\\\/mDpvxHVyuF+c7f9zaCCM309kKX5ONpPGfbj3bdFkhQ1UL7ElNTX10rTWQQWgvhogUPbKnRdLjrV0PwXBcgWDHmrueNbjU2iZRyPgBeMS9r5OCL0r0DPQYMDKaqkXyG889FBV7YjW4qeLUd5HSUUWUyag482SAlzgcN5sluUThAkWKk3d6JPm8hUzMRXunYOPROBPKhE5JlB2QSAsh2QdjCq+o2zOBSI9a1jcufQkEVYBilE\\\\\\\/d5rZFp9B\\\\\\\/hbRL64MEGKhHMMSmLU8ZdRvVXx4EuS8Gni4iuG9zWoc3silrDODuzE0L8VqNg\\\\\\\/YNxWpwD98L0M1c2yEMjnsfwmfcxytmlfzuTaZjEM5x9kiL3e55P4FVa8feCrr0O+ufY1bbYoOXQVqmcbwcwg9vCTrLUHbH2f4Q2AYqGHAeHRecF9u5vJWEiIWNrIi\\\\\\\/XcTrS\\\\\\\/ol5gpXmJadpKzX\\\\\\\/DuFsQEgsWISUx3FY9hrzPq7iJQiOPBa3Q4T8\\\\\\\/dpxIj3XE2Madvdj5d9FaLsVzv8PjnGEqbHKNeF5gV9ANYQMAdEEBq\\\\\\\/Aeextt46eMyyxIcFlznyxuPuos9sxdTJPjtBIOJWin4PyuAoQJY9cALBErAUNS7\\\\\\\/7toZxAd8QnoEAdHUyzoY2GACH36vxIaauQkobeHK5DrfTAZhzj9BgtaqZgtiAzSX0uIchwupgM7Y6pU5zYhLjgilmN29i\\\\\\\/RR07JqMo17cCbLI2OP4mHL6wXMPqT68blf4jfCUVtZ+Hx6J81BY7Rhj1A7ICxH8ru6s\\\\\\\/RqJJrHjDgCf7wGGem9tpvtj7gLp9WzJWePr0jKK9XhpCrtiBqbgrKWL+fjwN\\\\\\\/GCBSGsLI3u\\\\\\\/3J5pLL7jzlxrM4gDxN4S5ZM+kdeSNijUJpUVzbj7qGt5z0BkIfZH9r8gd0oQcwyvcxPZaqNiM6GpAidYRJaTmiEpQa\\\\\\\/rMOohh01PVYmrKFU5jqGwErxc6Q3thdPLmqMr+AVJCTi61eTFaS3GPmPcn956VUliaPniEz53bfVjh+shwGgl5GbHi0QW7ejngh87NenZdBE8CE4vVV98oUDyoTy4f1mVUbQz7LZXrptdkGnRNq4DH69YiV9csGqrUT8+8BYLT+7ZpBLeXCqkaMFYJfo7aPK6REnkk7\\\\\\\/vLe5TTVddf+eHgXH2TTMpTgTXwEzF85KoUKAW\\\\\\\/uilLckCLspir8g+DPMpsn5zjADmzxCxZm3efvcNW2\\\\\\\/AqsuSzG5LWoM1o9P3bpUGOkCbLoAGPLJIJC8TjllvD0VwsxI9pt5wRQNhprNqziw2Y+GkQjQQFDnoLESnhXKyeGww2MF4opC18a3L98HoXfgzkpomqS7HATUqZFd13gxyYbW58qApfyd+FdhLFWvNUz9gU5ThqOymP1KlKIWOPI51kCFoRZgyr9F4Tsg6WOwxxCXgiqSD6LPCTfKZdc6lUXsvaw5l0NGDbBpqmkjhXqmQxHfVunmAtYI186K2S4Cn++e\\\\\\\/dW5hWCyeCSNRr3iCCqZN8IW+Dgnu5mE+oCPiG0CvYKs2Bt39HFbsnyeQj8plA9giXnnpPQXchslks8fwBlrryyd5Jy+LohEqES5aydv9QuRItg\\\\\\\/ogi\\\\\\\/6e7Wdyd6trhi2WSnT4Ie68Y5d0Shw5ABv6TZqI1IUlFPc\\\\\\\/v+s\\\\\\\/J5\\\\\\\/xzliVi9RSOExmKXziThSlN8spMklXnnQTr0kscZmwG5Rvr2Ci1OudS\\\\\\\/LAyjsdA2vje28CKq48GnyqDJYEOKdHZrscdfOXo1oVzfs1g4AzGQjYpfcjmjQkjEu61CSrPS6LnXJGRGZkOxF2nKQcMYJIh30vdBDifQW1UcFnvJSs8dc4e7yd+Wfo\\\\\\\/pG4HqKeRe1Kr4jxyXS2T1BICwQNmQZIdJlBDJPJnfS7gZGllA\\\\\\\/SYMcfO1rIDimA+DEurLU6EmgsUCocTCj1t+kp9UDG6zOkEreqbwvZIGZdE29KfQx0oz7jxnjmnmdVn\\\\\\\/39T9vNdH9IcoKAKPjLCNHQfGCgWtN1aVzR1nx59t\\\\\\\/u9lrtPUTp+0DCQx5x2KB2x8a28GBJPTA0RDwyfllIxTKGwIlcHP8S2uPnOj5\\\\\\\/TRSp6kgszLPH4\\\\\\\/kMdc62jLWYpsLF+dkBTfSN7GCEy1kbXBviBioneD0BvVY\\\\\\\/62gHLNVEU7TOxuuaUk5k73Zh\\\\\\\/LUPwaUSQgqeFa\\\\\\\/F3ERX90QhR05Hi8wRRPS5TPSadzp9GUpLjyqGkGUeaUBYuOlaoxnKWaMjhzM6mrpY545TIZJqqCJxofl4lePPM6sl3MOussXGWE5vfUuNsnY06vZWRkF2J2UXoiNaMSunv70Ob2dqLnPYmIHB09s3WSv2fWJbTbdYraHYwuQCMQPixuleoZKRfCvMR4S3shgowJNiFZarA13kfaujWie8CW4QcJTFU6lY\\\\\\\/rcsPyXiIAr13rblSReezEbrcyOUxD2aZj9+yHiYUE7mF9AsMx\\\\\\\/ROKeCn7fVRRC\\\\\\\/D5bAo6RpWbBKiwbRbjG4jMHxFEQnZM4t4x+mqacRPZGebTkWYfD9OhzSTsBFUSNDJouMehfBmv9xlreRzkxrz+lZ+Sob1kw+j1V0XSUi2prN1HOkFvlTFfDl4P48W8PZOH3CshQCplrRxxOyfzqnQ13f3Gr1LVys6vM7d7wVquxX+QMNguVGRAHI10GjFV5Q2lzDXdFtQSJbG8xgJarLDPJO6OEEsODJGHyxTAS1FlMMB7+Dr\\\\\\\/mLvCH+0CCjx6gHotuVKaa+lZGQKmpL\\\\\\\/5R6TpGVa5eM6bLkRC4RIpta5gTw1wnAgAgTSzoR9zSfrqmmPQke+m2\\\\\\\/qoqx\\\\\\\/kQxaVDKm5o4EBFiBiREIj9V56dfKSNwlhtN71NPSx7du9cdjiES+8OzuaIYfGLqFsvlrH2s+liqKFwYDZaQ6n4nRxWC9MOBfINPRLvTYEv9IRH\\\\\\\/dQsFvHMw0\\\\\\\/Jnu8Kvm2C31dY89QYMJxLvh62n7vL4I\\\\\\\/Wn\\\\\\\/Js1tbcsq\\\\\\\/lK5wX81dXIeAAT2ASzLP+kzXeppvUhJLlFIdBRDmUej1CjoW8XiSHyKoMBLGdt\\\\\\\/YMHRJiSHXHWRIXnbWksni5J\\\\\\\/XT1xaHlnk1TanWr2xwYvdINlcVwjEpp86pPswegQghZq6s+XZuWzOuPuimVeP1KdjEAp90o6Pw2Kf7G6XwEHBzOLZGpfPVRd7eOCIvl7E1+in9qi3fYiAxl9L4cVUFv+6uzGYxTT9uGGF1BbW66rynJr6KaMSbctPn8JsF6LR5ASqZoDtakKdOqvhnKzCzk4MsAI5oNN3Nw8xp0JhwLkEI5q+Dam5jpAX2H9iqI\\\\\\\/HArqmG4c10Tiri4iaHd1Ipk\\\\\\\/piXQo7Vvtk4qk12d3VaXD30OwPsHuaUx\\\\\\\/YEoIHvTisnvP1X2h6+88lRHfER\\\\\\\/alONhQ2rpuDkrCpvBRXvPLxTVUy3MZH4mp7zZmYV6vrI45x8R\\\\\\\/B97bYKvpY5phxIULpfmM66E+agRWiUx0YT8Av04Io03DmCTyStI3EbzPbB6rOGh\\\\\\\/KnOuUdXIcVzm4+pio+oS70pbdus3G0ygGX\\\\\\\/bN6RULgB6S5aBtBYqSGd9hUAeniV7qIkshtgMkWxF5W7V0ekccaGim9ZfYvsnbeRvGz71KYXk2B0U6wtKcRngPaYuQUQF0CLfvgH3K25ygaQB67txt3VyCbajnQKhoPKG59hqwRwa9W+pTVjlxrokK23dLaLCSKFsAL7xyPbo5CJbkXQGshihexoiUKc2vLJC8kqa9UnhfmiyLxfuw6TDMdLXc5zX4gaWZ0jAdEvPVVNSUiG6JbjGdRRr1\\\\\\\/z16r7GYuqt1KqAbSBbX88N28B7aNx+JvccrMTQ4UdhosNjenbjFyk7+9LLc4JFNY1+mYAs3eyNxKm9R7VHgD9Ax8Lhs32Xt1nnU9al71xfdLwAeTsKfaVcBiG7qxzbkclg7SpbUuuF8w5uP9md2wgagErKYoKxxpKNs5VlU4dG2PQGbcgprvc8Z8ciZQuYO\\\\\\\/zf8ClVqB3Uv5qgU9Oaiy74VRXRa4YxV\\\\\\\/NMhfhwNCwG5eY\\\\\\\/dzqDSJ4t6qErtwM0\\\\\\\/Vf39PRUY432E7lkGeTT7w8sJ\\\\\\\/Z3DhgipMkS2ogpBkBWdrhxTb91\\\\\\\/obd7Q5FJzZ6sGLdznGot0QkCa5wEcmpCSDeEmOmJnnqHQxI8SzBXNNOJD7tEB6YuELhqav0kKylrl87ucOhA459F8k4IVgIUTS0+w2NYUUz+uBJtfa5TH9SKEKDSGqL+KyUuszfYSx2ctw1QMIhioXokaZPyDa8oJ113745MC3SLGLXJY8vWYkwsssLH3r2\\\\\\\/24XiSTrlfXB58lx3pEANR+yYPNdyTr68mS46V1CaN2txJNtrBrkdNxMJ5\\\\\\\/uaMitlfhJ1wrEhnOZ7s1oxbYaYoawgvH7ibxzRaYwI9sVHzbk\\\\\\\/X37ucv2OBc0h58v8Cauvfe9CvJv48V5huBm6NdPD7ladSETQTuyK7WSfQNokOiUX7o0aJmmVf5AU6m0uoQ4GmxfEdOsNkYiMQ61aQc88yHHnS5bEDvY1S+R\\\\\\\/5PWduMHytzMxeEJ7TyXyJnbc0953p0OVL6wUYQ9pr1i\\\\\\\/7AoLDxgCKnCBaQjElu7Q9YYs8yo40pmVBujaBX\\\\\\\/SFbsOJtsLkDBxcxgaq9Wqi5DToa9v53ovIBL01pdrY4Jjgg5zHk+KBGW0UjZIab0zFOb9vLqfMKCfs7tNTvTE8PcegeNjV\\\\\\\/mEVGZ0cXXUFupfy+w4prz4z6aieTuUMHaoiaqKcbnI3ls\\\\\\\/DPwjBxHq3mT6oJpFzhA8\\\\\\\/Ye65+4wpxbw\\\\\\\/apnTxgMtliDkKdwuRkIwdMXU4Cgij2SXysoTsHwdr\\\\\\\/Woq5c9Yty1ijq9aeNZMGinwpi25s6g71LdqVL4x06ag8Uaa28nyD8NLcp+Gbn5pcf7OxQSITLLdvkR\\\\\\\/PR6SFNL4noVkkoSaa2KgHeANn9Z6Q6N8DRIF9+dEkx12URP0YulPnYj7RgZwuPOG5xIdag4twj\\\\\\\/khtqd5Gnr0k9O8YascwPdX2V6MB+fjclrQC6ktyGZhUeRwPQJdUxEbUFNxMjjM56qq2L9JIVfwj7cvOQ9IjrtjeH0KwACwpNli1eKQcFlzUW3I+cT6JoGZL\\\\\\\/CiJAoIL1HPYvMc3HU8NBv3JDmpq1gg7K9FfmTHJgx7vmo8CyZ1gCaEbWmYpeAVEGqCeLGhnBw1zTBPCKN2BImMDlM0qzKY4jAhtBAcdB3kpsSSvobxWIWimRx+HXUjDLQ5PbCVfhEv1qPPU9DV7b+G6XAJjRrCRdX8G6yw4ib144jjZmHfa+rg7B0cjmLUY9KAgUhTy\\\\\\\/rPJNHtmnNrxaZCB9QWWp7n+Ww\\\\\\\/PJSs9e5H8pgKXlOY\\\\\\\/HQ\\\\\\\/OawUPgjAFfBZVo7P52ypPTpwmiSQxRrv2Ko9Cw0IF47rZKhS365SN5ahsiKslMkT3Vd47r+ajNuomVUnsB53PfhqHCITgi9PXk2TqenZUg4qNI0ukYrm4JPkqXu1hYqWacgPNebyDhOnpw\\\\\\\/1Yy8SqutuhKKvpeL3EODYI8BOHIVgJUOnNJnREercwQzEVsTmGhb4ckx28RvH9FtDsCXhEQz+un0XKuF7cQl4z+n7yLe9VNe0RcmABq4rS244F3u4PVxL4LnDbzmUxFgZk+EBzZbEj0owBITAMJAEylPbD+xzLJd8QNFXFkVL561s1WsCEHU\\\\\\\/6g9\\\\\\\/5aIhvd0f2Psj5cxy+OgRH6dqqoAvlOrByFMbCMKbknC1PFdaslmNX7W4DNKOPg+fJrMqtmpsXl81e+ReufLaoXshBGHlNck1fTD67F6tqmQRmjgIB9nRlkF9FZqcw98DOiYcehSvns6yyBOe79VLT1+5tBbeCWu3a5DVoaPvCr+zCesJcmhwVAh2iK1qxB5Sy92oxG+yLw5qmjWAYeIXrVI0DdVgya98oTZszk1AT8A+9N0ZzOAfe+Y1QAtZnJfmRhDPRC9PmJoMDu7PkqXKZXsp7KrEhOmjBHkxhqQeowiPl9ss4UfMtqJ91NbClBtEe+H1Hq6epcxuEvtP7qidzUuviwnFlGN2T7ncR3jEvajU07KEPbBbEicw\\\\\\\/q6J1lY0J5HrnXhJ\\\\\\\/9WiW31lAKMyyL2+PTCdg5cdWU6Iu\\\\\\\/JNVniePGHJNSjOsmJfcpYuajMf\\\\\\\/0n3bmSzPNcGIyELJ1DKjm+d86CvJ25CwvJHX+OpwxVF0A7urPVEQPselqvuQFBaSv7CRdvH7rWzfeaeW8PyYHbQkH6rL8CoSdCfP0ljhZeEMlnrazlSG\\\\\\\/1pgPatCle+eJfg29K1o2XX1hXAAV4Y23Bfgm14sXrzvFOk31PEsQizP5TorSpvYRtbXVl+E9EUXdDoZR\\\\\\\/EVq7Sy6ESzp8u7uh6Tnnvs50idnLMxwZo3p37Eba9s2QlYol27bCMQjpmxGRpRMgUvTP2ZlxsDAXY3QGNTpOd\\\\\\\/vCkaC7RTdQZS60pW85Kc80sY5K42z4h\\\\\\\/IiSiY\\\\\\\/NeS4XA1p7RGoT+Y8KMo63avO6SnYmXZoO8T0fpg4DtyTnGMAL6hfRZmITj5RjZTaTkWsGPy+CIQd1X6020PSQjXEe0r7jZFVbadBlE3WrKryBOfLEmgotDualh2qOvvIUtxEF7rnN6AZAnUW31FVhQG6g4+WggkkDmDKvoDNP4O6svQBoO0Q6IbBu+TDjeTzTVAlzBtX98E2E1SuzzQKRl5E866cb2U\\\\\\\/YFl7OihfhxH4Jds9YcLAHx40W5NWGudrIhKZDz6\\\\\\\/ex3X0+WJnkdxDDSbjtPwR3cD8CajKLuABBLRYWVwoJPHBSCAOsFWmqRQiQeLKhLMGsH9eikb2ztMZm9mWOsmRFEqzAphPPZIa3PRM2XLHwwmX\\\\\\\/cM0lszl7ECNUdV3cgCZpoi+2lDiXmcHWoZkUri\\\\\\\/SW\\\\\\\/e9X8aJghff+irmGJmigO\\\\\\\/Rnd\\\\\\\/jJHdZI7hnDn3my67TGDdx3vu\\\\\\\/bS0JxS4p9KibSiu1\\\\\\\/vZMgE2zISbkOzl7SR8qNzqiOHE7Yx6g1xW8n2q1r+rScXCx3GOxMNHXIBkezxvuU20UH5C6jYSYrTdNQHBUpMpoe1S+ypgDp+qHvy1Phd42F55DPsqAZnv11kZuHNGQxHn0i31Y9eaCCr30CdB\\\\\\\/ppVR8AedLpZSrkKs+39XEJkTqmedLVH3NhBUTGS5a2vOlVTtVpLiAzOmbTSUKeXBL7fisSL1C\\\\\\\/EkowN52lBUwkOdlyCwh5ilfjscCWcAQw9o7jwQO66ucArTS6WhGnYzF6KOrjczEnGlqZM1Lh0686AUeAxMS\\\\\\\/UMdc2Zip01x2gBnidS35LEmPeyGxx+9D7+HeNVXMhRQfjK+JYia5DlS+MHVet2duxhivj4whWNtGDuHH1nhUPnlmYL39\\\\\\\/KpPS0YMeUoGFd4bM\\\\\\\/Lj55\\\\\\\/zL9pbowv\\\\\\\/RlUpVQ0\\\\\\\/Bu6rO81cPf3r6\\\\\\\/MA8jolhrL9e14I9ozivNb6T4L1kkOFMN1cl\\\\\\\/lSiQm8MBmyv1ZNBrR5i01vT4Nk5T8OlZ2WT2ZO3rUQ9plSbE1Si0yrRY54hfGbRjBQuXYVxgFsk6\\\\\\\/lGP5mcLJ+JRlucqKu7UYMAtdgIO570Zrrg3JVRQP5yoKLuYBxuypuk71OwGADkSn\\\\\\\/nqDs4hSGNY7h4FQ+Ah+W14MZVap0TmkfFu5mtj\\\\\\\/OWXo7jFGY5iRhaigLME0VMg\\\\\\\/7cFDrjk+9DXYnvrgVD\\\\\\\/CdTpQ545pNFmSP4DiZBPaXI2Oe6IksQlAOzqHC38FRpC3jiQ+AFLu7WjU5vnQLqeiFzWfpT+Jvh3nvvV+Kee+DncG7wO1w6cQCLwZSL82epb1X49FhnurusPS1zYFRe46Jl94L2\\\\\\\/\\\\\\\/M4tRZPNLAFQZ5eaRtsz0\\\\\\\/QhCgz3w1uzE+vZ8PaU9UNbuZD12pTRocd1jz1f6HI5qXYm6NyQREf+N9c2anMRqI2Cd7b9xbNnP90+G5OYrEXKM\\\\\\\/S2WFIuRE6QH9fG0EOHZq0UeXfhy\\\\\\\/YPz3xF2dSD60GiKcaTXsbZvHq6vJrlHbGn\\\\\\\/3OrYv2B9wj72krFlLKtiYU1lw0Iato++gWz8mRk81qbia08AIdpGwrp1cK0qJ9eegQ7\\\\\\\/Qf9uk2EpAoBJZsfkTlZmKD6nOsHVk2aOWn3N1EBUfs5Mgfm20\\\\\\\/xeNg6bxl17JQy6lZF5WQsAhF5apNqUA4cjs18IgTINLuf9Fa+WUVTZj\\\\\\\/45gX2vqX9W1h8c\\\\\\\/T6rg1MI\\\\\\\/Hbb16ES8A6tQxmQXDtF58y3IcCrkif4MqroCdFbKGgdt23nTzY07lcI53D0gF7wYPH00nN8dLb5w3NMjysnc3wqz+zNGk7Jv6ZEyniWJwbU7G8yi4cbqkDF5L+Ik1o+j\\\\\\\/SQ6gOVSpR2BFMT4r4G7mRNNlB3Ax4ssl6vOqydCcY\\\\\\\/t9Cr6WnJoQFKlad328CKzlC7esgZ2YCoVfjGxaqRYlEiyGx8A8och\\\\\\\/UTpd8QjtGfcR6uT0W284J74WWGNep8xRv2vnQWHZ51YaL+sVrHGg4+wgNa\\\\\\\/j1D6Fhb3KHdnxLXZjFs36fWig1XnZu3lE1+Foc3kQlg3CJlfH\\\\\\\/dK0Q6RZfnM42EkPsC\\\\\\\/Cl99dYXDTgZtWWlwm74ZD0aVSXHgAZgLGbPQJ996l3KtoM3+FtuaTRTOBaglqC4PHQlAxyIA5UIYu4jvhaMGuJbRjF02HWANiQk2L8Dq8VUJaM1u2Yn4ZFNUEofTLMot6vFnlHmBCd\\\\\\\/8b5990kRveGIzhbP3d9Yhq0Ydt6u747KOj+k5DNPRQePO\\\\\\\/lxBjr7reHmkimEj1rClxb8s0waVQWVdb475vVtW7xqefbiqJet3G4AQYLMAdKJjxEUCt9w2S+QsnE4StNaKanGBE2LppzM1QP04Rym0CnlKzQaTGJ5FMR\\\\\\\/L4YVOlRyL0YnIWaCw1oWKjTX68PX0UVk\\\\\\\/PPw2PepCW5lqruHIwDd3t4RUwMYJrnWlF+AeKUew6mUpsRoGfjxjFqCP5WCqfGjQdIIHzv92NmPHaB7itBOqTcp6tGIEPC0nSMLiYStNfAaBw4hoANYfOCo3pXs4EgIAUoPSfINfB9AhFGJrTGmRZbWnrLuwW0IpP\\\\\\\/N6XnzsTP+2TbJActrBG3nmC4Y4b\\\\\\\/51B92+BQctQIAiI2GyJvCnmpx7CAQAYYHmOJ\\\\\\\/CRVsV81YY69DXIeHCHRPEOgA8acg2T81FfFFrNMq\\\\\\\/IiLTeQzuBN7siRWynEt6Zw4v5S+t9TXVlxeoaINxtZUdsk5Bu0DsseJTIpRx1INyVOexds+T7WbTBAvOU6Su74xvCRhKj++HI4VPtiBiRnKFBPOll7kotxEmj4BtUHPPNWi+ZdtOSvlSlKFUiqhXFvxHjebzh82qUVLiPDZqFxQw2NaO57jRAsm\\\\\\\/jo1Qo9xhWb7y8jeCr0IK94yBq3MP0H5v2mUpOEwUMjwtORYawBJOOzO18gy5HpuMEgmbAAm8TBltm45Q5I5W4dnmq45ZXjbDBtf69NElqLjDFIxgzlYx5Vg0eT8ln1pc44+dvc9KL756MlMdSb2VqipXFQKvApkrcIpI7u567cFajgYOrbbLpQpLUz23\\\\\\\/sT5ZsmCHEeNAowm09VdFpOOSsnmYztIUZ1A1fmwlZOJurFw6nWz5jB0MFQEQxSuC6OajGmweOSH1QIqsNouRsIoBplBDhRl3bqraKmNCiuvmwlnQda3Ufd7cHvEluv8tK2v7YnmjdKYU1VhMFBPe7J42yGHwGjvl1UwQTmmzubPmd9\\\\\\\/bFcE8GH1KmVGf5mVbLjDEuHScOHd4zKdwZVzDVSKfqg9vr8oge8IS3nuxUn4W1M8EqnpuruZ2FOQtTyK9y5eW4I9SxHICduuv0KUnKo49DJQRLxD\\\\\\\/4\\\\\\\/Tl5eUV8FdM+VHIf9G\\\\\\\/6+vKxnXZLr6VmXTylcKWlG8ilaEfnn11UvfzOwAhUNjmHGr5mESzCsy5icKgPcvtu0QKj41pCJ0PLZm2cuL6\\\\\\\/V4vyqzlOsROj3McaXKVWvLPYUFbpr3gF+hUuFhvfjdgSjwSKHuF+Jf53rDq1Uw5tbQeXV8\\\\\\\/gWhv6ifxaw\\\\\\\/\\\\\\\/xnP5sDf0T0HdgKApU0hGNJnxQvgI1v5a5tCwzXheWO\\\\\\\/2qAmpDGOmrS8OOieQN0FGuY3CnNsVCYgwpcHc+EucQW7F7AfbfwovVJ37BDEcAjkYxrhKcBVody5esg\\\\\\\/vr58WSuDQA8N\\\\\\\/PM1S8eChmR02r1g342jHWSL+HooZXfcXLXKqIx8i6LNBTT+ZBdDFcosLLDNxDLVOvsfh5rMBlQFCWBSeyZ0+\\\\\\\/AITrGLT6jLcsuwuqsMyF+qkQzjZY1EqrcZzmfS15e8P44aBryULSnE+zymH\\\\\\\/Vt1uWq5T5P4DpGQsVnUPd2rvMrn5\\\\\\\/ouyi6vpfJUeQELHNjzrYQtrRJAHQCJJn7iSFBHqor1ydaMwxIh0Xbof0wsxy\\\\\\\/h4bvTTsxvq5vbgkKWcwl51d9CzNk\\\\\\\/zJXUDddniT33BFhcUDCHv+1E98ovxuTEP\\\\\\\/UTSJHftY7Ne8gUj8dA24PELry489m7Fn+MQbKx2Qevnv9nGlGDjJksJJ5Osa2EJN6+kvu4xqqbMY0YzUaSb7RjHAeicg84vXMKt\\\\\\\/AKehFxaGTNnBeF3yPc5nF\\\\\\\/fMncRajG6h9fofNKAcmI1WTxhx88ChcCZpAVajFK9B86QUeKlN4pgNTRnCG9Wa3FFacA8puprOHE74OQjy4zBbIEYEHHaELJLBjaiKOrnRo4n+NrePZRUMkAPiFoWAcfccbbnmC0ciLXFNltCwAvKWTkuFD8CgcZDajgOb5PS1auKIUbqMXcIfGSlCmC92IwCRt4Ir+5qEg3AUlzTeqwgZpzbhQbZWaWxXB9H7VksvoQMALuopcrTSnyV+IaCd8PXSPO4QIm2Y6ZAUW5+unUh+4odBtxkIBo2PKOx0dPBs+qJ7NrEZdaMLBXmSRxQ8Ar3o65SZ6yN3K7n2WlyEBbpJvgNrZLN7KuZLgNgeKoCPXCmkOr\\\\\\\/k6pqs3Wknl7IITpoX3e6KuEhxIP7Px7BFewR8GYJXwedKtxuHU8aek\\\\\\\/z5fdd8O8eXNVkVeHGH4dQNVelx0e4W6g9fWCIl7U9ewgJUScbWwhb62GwPWawYMsVx4LdZgitPUNzGEM8wZT7FmKz+IElgNu+nqai2bDCjPAMVbC11gNOEcimBjh4VHTCSqVayqrrH8lkQfZYL22qfD7u+S6PjRDwxodmCgZv371crxgBJHgksI7+gjjKxy5qsTRFCeVATtpgrs1r9bbrzmwIyXuKvWdHzMxPlIEF6puCfu+WiMZeZ\\\\\\\/AJW5N6VtPxeuukohyxuVegSe0BN2cvAvJqzgU3\\\\\\\/fvUUo4LBTl1ZteD8WcUlYrVVwkgYwhc5E4yYC9NbHGh9syOzkCiy\\\\\\\/Tcf68r4EUxptlvnzJoZatmd644PNXE4uaGtgWAIq0y6ys+G+U4OT1NqEzVseXg+l9B8Zu7xmnxLVekMAlED7QhoA+X7UAXXjcLzD56\\\\\\\/3hNspl5ymwBvwdWbR5S\\\\\\\/pHaTH6BPGr54ZXmnJSNnsqVoNcuvHcFXSJnhatnNQcynHWO5QduUj8uIARmasyVajdoZVXjw5F8BMPysxkaGwzpS8aaTTtE46PASgbNi6RGGtpwGDwj7ExptHyLIYfNg3AYTWTN344PkuuZ0mztAyIYs2xb+BBdJZ09+3fuZBArZefaPRwAZXDm8xOfOB5xWndvPlkRIflRab1S4TIuI1rGebiIrGxFZdhw8pkUYX5C5d0BhIW5laeN6Q+K\\\\\\\/s+fSuHIve\\\\\\\/Lc3t40DfuCJqhGIksG6S3uiWIE8RC\\\\\\\/H5S6dr\\\\\\\/QkhCFaGJDz\\\\\\\/INs2XkDEQZO0C81+uhTh0ZtlMVXLDySENQbftNVw90sUiJgwpPPIcDN+X4iFKKBtxozfCGtFE+9mMke89vm9NJOo3+8\\\\\\\/4y8zZq7l8wDzqm1P6C0a1Tnp4IYVpez5yNOyDpxpUhYzHYiRZZeijivzFpvpWKvVyHOnGQEfWuFDsqtrpLZT1O1fiVi1bG87UOOxmgNmsZiDagj+c4Po9xFk7Cpn1wztaoR3WPQ6vg54X\\\\\\\/WXDFkpC\\\\\\\/fcbA2kk7P3KHZGfhe\\\\\\\/EaCpnpLkGqcGNvzS\\\\\\\/fSPdrWuLTURufhA0LbV8rPLVP3TNnJnB9gkpcC40fjR54oaSWf\\\\\\\/c0JXkvnPe\\\\\\\/r6vbedG11tBMJSmJfLB5+H0Xiok+jx\\\\\\\/VcuPotDoiIqi8o7\\\\\\\/+DdWKJfyCHtdTgKVKTls4NxypI+PQGJH5X\\\\\\\/vAZNuDHzZS+goCzHbeRHxSAnKTUw4Ej52QMCQixFohGEN24T2hYu\\\\\\\/is\\\\\\\/sulMdt71JMYVY8DrFAE5qsRHyqLljKIntIuKijVs2oqb4pXdw1qTH0i684BIp9CMs6V+OrUC8nj+qojrOGC0JWemcMS7\\\\\\\/Thb9C4jLCeiAZmveS4s2UZw9yMWIhVkR01vl9YzF5gV\\\\\\\/I7JuxVKOtjZ9IUM8rPJnoRbKpu9yIftvYGAtC8ijhSGQiMBAMUxfXafnSuO94MpphH8rTb3Ee3P9XZCBUw929HF\\\\\\\/GjUIK+zhON4JwfjvKPqDUurWH6\\\\\\\/uqIP2v83kystmW5qqjGGaFsOrp2iXOfA+fOXm0LmU9Nd04prb9qpJkdYn2WUofnlR647isGhwT6IEUeMsMr1nIJYmw06WwUat2HhRL88vwvLYXK51jtYK0yL4OeFc2b8Lyqjg7r5QulZK8P\\\\\\\/AhCiQjZENHUeX4H4yiHf4eYoqLPmQp3s2cGthHCLYIoqPaPXydpm3ULwBf85yvbXBs1rfQy1r8wZmAp5zOIb44Ou5j+Ka+4NJSD9mNz2M6KrodB5i8nvvsiI\\\\\\\/77bPen4sDBW7Wt1ndTUThnBwcjBKGhIzkJeJfrL5R+\\\\\\\/j0z532p380cQkfggeH1MaULvlSJA1vsbu8kAZUthQaSJLWFKXd68sPJNoYn5ycNB8er4eVgZtV\\\\\\\/Vt+e6lm5L5H6G3Tp\\\\\\\/5dSoi8z32+oIAxeCvj5ajCAK3G0SZHkDbQYwdQlCcQnEDhGPP5W9jAhPSCdDTsUKctATzit3U4SwhDfBPrInDr98ns+\\\\\\\/nAqnBQGYdcoEzWASQjvQZEu9RYthbONziAQxW9SkCRfyMBJvG+EZg\\\\\\\/WeUtaLethiqqAo9yo5nrf2r9mOLOuI6l0ZiwfD4LaFD1GezPxapGC+HNwb8zlj4DJwec4nJiP5EjHNZOV3ZKzdq7GB3tdwLRcJZJC1IzhlDGXP\\\\\\\/wtV6\\\\\\\/h31VvDVeMNEpCJcMfgZSpPSheX985s5VyfJ26ptlKCvjJdnDcL65GwYkEL350FmlnHRD8tpSRVNXIPP+\\\\\\\/YzYAPaNzQ4qVgrkQi83uQ1iDDOIanCro1fUyI4swOTGaRBjOTERkaQSzftgtQvDX8u4nq410e+OQJi+qpaFutD3t6mhrIWWkJAaC8m7ovAC+6NC5gJ0QvSiTuTSup2Cx1STC5Xn+DT9Y8vJJt5L2Gfrt9YtiKz\\\\\\\/WV6xOo7uJcAuFU6Ciq7iglxnsmdiR5BxPKSsAb6nEttlsfdUX+p1XmMqJYJmjx8Z7H95R+kFc+OSEE3HWn3yOwBbMk\\\\\\\/IXMUtzOiPak6fbgT1IeJUSwyD7CR7q17SvqpTJoMKrvtgaB3o8JYED0xeim658ogo3x73WGzpoiiWGbG83hnyuyqmtI9QQqZCGSl7a4XgxWm6790Ijs1TVhBA\\\\\\\/u7HVhAFg60i88bo6a+02vfET6lTzoitxp7UbWMUAW+6stHejR5sJDUDKbrtiXHGOVE3ticuQMPFFh+CwR2y6zS4sJ98KpvnX8VMbcFHJJ\\\\\\\/F5ld7sZbf8rksxpjU98Xkmk6hbVIt237bhu2LHECEE1f5HqBh0MAQcAZYhvOSkMxLJ56RpAip1tmes+MWwy5EKVkw6qnFEN\\\\\\\/\\\\\\\/Gf+D+jkzkk+JllW0yhPSe9kZoEP23q7xIEgjCik6exzIve31YBYNq3+DLGymzaJrgYoMMUdrPbYZw09Xs+l52\\\\\\\/0o1oLahSzGuHwGjr3vUaWsRhPiNCeiHkwojSDtM1EW+9HBME75iNxK0XUPliHnx8LgBIl5jUz3itONEeQxow+kAxMDwcwn92zgdxTbiboYdZmvGY2VfxOXyDFYNkvpY1Grp6bw59wRr4K8wni9HWYmiXiE7gk5GCVV3VC8wwMEcgKyt2teFSzmac3O5phYmTF1+LuKiLu14c3V4nKjRIwccdOWsDXjNTd5WUOE3Tp4P6BFzXe\\\\\\\/OA1m47A+4gdeyHAxOEbUD4eHMMUWu0QPX1eildx5ZeC7FDgG5K9Dk4gi2CPOsrJvidtcMBmVg8+qsasWHfnP1V9KnbVvyEofJNILmKO92XmFz24LislL+2J351bdXZSQ4A90ErDGSBtq0Vm\\\\\\\/qXkB3iWe9euVhRjEbr76FBzGWfsrgjzekjX6pFppMvukpLkTeo8O0KdmXDKAoLVVlM9a0UevGC31GZwBaHEHZpShtDrkEWLAxaXqHmfd17mBT1uQIDB\\\\\\\/DZ6cUpUz\\\\\\\/NpFRINYCxSaThvIVnitaShMB2GComTDtOAoZtrbuBCfzzp027o4i+ylegf31ycRAN0dCbzJEBHbkeslACBJag4ah87y1N6BX9Umj0nQwJu5scLp064eEY\\\\\\\/QhrcoPwbqj\\\\\\\/hJTmnK4V8qOJIo8c6Sca0y4\\\\\\\/N1yKoFsUa3UI2fQxhZohi2VjP3fZ0sz+6fckbG+UdpmSvP2z5DIyWWzGP0oH7sg3gRsr+IxQpFZptxae7tByRoCJEUcqLsVPFFOGEp\\\\\\\/DHhHwMmRDCngDXze0ZBokHTlKS7hpwSJt33rLOBVJi1bJmxrt\\\\\\\/7bGEurNyRvo7DXrfhRASV0HYmNasbJfmaWsqML1a4v23ReWimbUi+AP0snrMtNZvjDRBj3AtfXKGTcnh\\\\\\\/glNE05769Ul1adjUhFehOl7brw+KaipiO0\\\\\\\/Jimx85IlzahjNd4D5DOhfuEYrT2fAhmT8n+3EDEk7V5bTOv8vli411ZOeBzToR8VmSLF1f+hZ0JRe5iu4tZVV1Jll3bx2RXsR6qXraAs47XNxTBxwiOEbMJH7yVzLp0eXvLBtvpiojy7XU4ppmdv3K4hwYlTMsKwbRK8c4GkFSGOxTQOw+FEVlKntKExVSXHUWlZOYC\\\\\\\/JkiBwaOO1W6jfNexWhqrZcJkebDn++JYwexmkAlKX7LYRPAbhz02NRcJZqaBxhpW\\\\\\\/kp1CsksvoOD5hqKDHVfWrV+3BHz5EouuAqIfo1PFTPWiNnow8L1\\\\\\\/ZApvhWvCbjLgUYve4X8AyZ3EN38EISrBD6b1RLJ5p9ZKpnMCbjA1t3qtWKN9FqmY24ZGJrABgGusJ8OU23uF7FsmYKp8qovZTwJIbKr9xKBGl+bHBGz++iZQ6Q\\\\\\\/7UHQrHw64yPEkUAMvd8dlHNot03LqnV00onzhNeiHEmLtwZZPZUY5fo5hJGzEwIHZoLaYOpsQaGdEqonC0cHnGybTcmDdmSq+n3FKT1+1VidQiDOgPvVEixkfWGb3p+enDbtuplNKCB9KVKvGf\\\\\\\/SHAy8Ch1YPdZBzNUzMT0QpP\\\\\\\/a5oZvrbTDKvCAdimZK\\\\\\\/V1ueeCTRP41uTNdZfdcz88SnPqtdQ+DTem2qs3KX9Xd0Gm7NLglQZhQ9qppMnS7Ko\\\\\\\/UGEqAjT1VX3ucbAzcHJQ\\\\\\\/hcWHXLwaW5Vs2PZx6bG8F4Rz4AJsW5syFZ7l9lNxH6C7UM18c\\\\\\\/Ow9pysY47uygDjpzbsNaPew8ps7G9tEWzd+VAjIBduNCjg89HkA2HNbV8XH7vaUlUPNYcKRO03GwcWimKfuBRy4uA6E5zLCGJHa4UIsgRnRpzHexOjTgTt+oeaMw0d8lleZsf0GYxBIDL5h59iimgYzMAZqAqbkNztwcIUTz2s3J6FGYxflcDYSaCxGKWpY8no00p9DZN79zYow+eEgXzH8mrhTOHlUYu36bpk1CH5ozNw0Orky3C6FS+Q25DG+jzFCDQtWHRbueb\\\\\\\/hQT\\\\\\\/LUIkKtKkMngQ22Q9IbjeLfwogDvSftEoCd181eN1mMAtpj1f8gqKbtQOOBipJwBrLAFZmmB+\\\\\\\/8TglL1o0UrzEjXk788hw6vBbmOImC4WkQCLu+4s7HmZTET1jrLOd5\\\\\\\/6zvOa24snMa\\\\\\\/WDz0mvshe8WsvZCEpEF6OrewyI\\\\\\\/UCB8GVCU7e2tQOosSVVHAFZnF8qGmRWRv3fCmwhS\\\\\\\/HrLn7mMCRPTdoCY36gEztlyecbO1mr1thQddiH2X78pc6AvGV72uv2bmEXIPg3zX1PBSDMHYBiyPwpS3WNEjFKIzFNmySZ3L36jv+VSj2rSbZN+4ZAvu\\\\\\\/a5lrSI1ITyB+plY15uJDexcbrUvPyIjch+JbVGL5uvj+IFMtG\\\\\\\/z3y+yVXKsVnUQoyIQ6ssJjWV8sBumsCR9MilpaH6cni3jv0NM8laY8K7afjH9utyQckw6XORxX+MlvC\\\\\\\/9d7Jd11Q6CNql5Lt6bpixIFS19wqowQdNBIXMaym460NXtUIYNjb9nGXR0lrBue5F3Z6PrQdqJcL5+XAZn7Luk2L86fTF0aa2p+wiOsj9A2uymnQeAFszDEWSjPeF2IaRT2USwfC9+P\\\\\\\/p5sQuSDdhALiMqGC\\\\\\\/oSgDbx07DxhBxwEE2TBh51MMgJDBKqjY9qfw9W5q2YKDAeueee5bIjNrvsiJeCmALgQXfiqldnjeKVtDL6i6mbTGXjIfwnukC4XgcAzOHbkV4MAzdOmJHNKcGoI2XBuO1iG5cjNroJzAoMQS+VVshED4eywAmLKVQN2ifz28Y9Wwx3Lz09FYp2UWBSQUj3rxjyAZIU6ssviJsSocr55S1DeWHzC76mIlJTR5H4yPflYH9NslEy73fdjFNtxXBpbPHo6dAbXi1K2EgkUlydkV\\\\\\\/8GKhcwKALIOFdfS0ix\\\\\\\/GVaxnuqE7d1nJysfx1DaRwrxKeXn7JIJWjGXx7YvhbT2O8m2Va2ewWDYEL3WKR6mkUYRvwLVzq06Yy0hqQYAgfc8dkd3TAXMW8OGqak1eT++mw+Z1A8\\\\\\\/CgF2fpJ4PAZjVBSCfuLaczoW11xj7gmwsXO14PY2\\\\\\\/rw8+oW0Cp1pfL0vuW9QONXc2UefG2QQewbojS39qvEm6x82WWaabU6qYfll9sC9aS1uETEmIUCDX3WiwFO6IrKK0xEM53F\\\\\\\/bVDpZmRS8f+rJoWs5laFmmeECzd8TloYT1MBz\\\\\\\/vZGWOSvvWyq4dC3iTN1NZmrvOU64kPCYgGljndWH3d67iwman79Qk6Lbrdcyy9kX9QeqZ1uQlFMTojKhf3As7uDMUaVL3BDrYOkpDFQqPpck5zH097P7oyo6erkz5b0EogkET1FXHoXFJUjQdjvhtf9MrDqC5KIoJT6gDYIaqFnYXDfO2AGLJwr5zJKHQ+i+F6QdBSv7f+ypKgkgdqZELr6nlLIGYEuGW74R7wvZEqoFeQ6q639pgoGigz9UHag\\\\\\\/otzNGsL\\\\\\\/P2Qcpt6tUxwyTP4n4a68SdnJwaYrBGIh2V7q48XFMCUEh3t0nluJHUTT2iCuhC5Ab1\\\\\\\/gaKq4Z3RERABbEbed2\\\\\\\/UPs\\\\\\\/PzdbyxOL2y8eiIiJa4dObapF\\\\\\\/sVH+h57eQRyXSlb31FZY7mwJ3\\\\\\\/JxiYz6lJmnT7AZo8KfrSBEDAEmG0UpfmuZmmecMiULTX+DW\\\\\\\/1pQp4TDOMAFghyfhNXK9yQYX\\\\\\\/61YbttXsudI7HskKWOEchIA070kzlfLujRhRnR8nCLN6T\\\\\\\/KJqxw4N1\\\\\\\/\\\\\\\/VobIz7RZvAVDYpwAMTlIZ0\\\\\\\/2+qgPyKqeDeI1wx3kTE8k+lijbPyvIn7qFMJ+Mru0WKYRo\\\\\\\/4vwlhX1Rja4TGKdOpjZKY7zFvMM+VR7S8Y\\\\\\\/+vSYiVOY9bPINFFHTcUIETaXafkha5l\\\\\\\/H2Cwlc9yXDNXUgDeCINYN6mXAajyqAOc5piTp0Kmi7Ljuueu1P\\\\\\\/XQQuBTSdrvWM8K38+YMofVdyk0bf5tTH4M\\\\\\\/n7Znaz2L8MtQ8YizA+qI1PhiWVibAccE+sm6rHUfp\\\\\\\/N70VkYrMi1ubU4IXOibIkGO+UAqulBih2RsvA1F7OVQi4f97O\\\\\\\/\\\\\\\/oajCaIExccwxgnfG2+K4ONYSNQdVFaYwXDx17w9UmWGZa6MMmCrilojqUHEY3z8q7ublSVanBMZWKFB2TYhwi4x9nc9bAyiX9CNAGLHBfh7PLvnt2W1Suq2CJAm0py5+vPF9lenSd9HDJv9xX4b1\\\\\\\/FTu3O984SZ+zmOb5n\\\\\\\/SrWFVpl54PYpf4oB\\\\\\\/5FBY\\\\\\\/1NSYkc+bmg1ArvcJGWWhOj7heV8WzuveaQ\\\\\\\/LhX\\\\\\\/na92LQ\\\\\\\/VctOhkqdpX0LFGjVg610XcN4HuBanSdTMj+EGNoXkchnbrvxNO0xjZSzoPVfm\\\\\\\/IVOSWA9sGdwRW7Cq2AifB4DKfpTffLbGL5uqYWBXPGst88d+LwSnhzHPFf\\\\\\\/tZvMa2VkDBmUSRaqs3pG8BIa3oXdPdOKIIxgIyVZ+GN9BysUIlLTwkzQmykFuK9LUj+hvzBE3Do886vA6mHnt4QPRrYdcS9c2Z2HVC9CTPLE88LN3ems2pbFOfBdYYG8B0ofPaXBgln5TPXcdkpJX7JcfEBk8lb6Gew6aZTtqT23toJFv3YIVZhaUTfPdldO+HEPJJJEAi62FeqQj1WF33V5auWRjHmtbWgXsjvRHmurJdlm7FU+KfNr9vZ8KsEk5VBR6F585bAlASlj\\\\\\\/VXnyJYdd\\\\\\\/x3Ny8Xv31faJQOZ8L\\\\\\\/+raKjGx3AbUzQTYF1l515N3jQpPhNL0ytnP5CN+WZ6FDF1stouWvW\\\\\\\/9AFl8MEy\\\\\\\/qQ57V40kpVtZWPXuKoN+DWtzgNuJ1\\\\\\\/jND3ELKZeVE+U\\\\\\\/3YiYPFKTWlTQQ86KtzVgkMTPb1pElHNht2x6n+ohMzJE4aMIoAb1fHZzgr57BbojSODXCYmDQErBZsH8fA+y\\\\\\\/KIuYUDHLFbGmsvFjh0h6f6gHp3fXAeShDTTubQ4MKB+xrAq4XSGfShdNvwmJltXsdBGiOIppTj1sQ0MGLXwr2k2nAaHrblnCZJILaAuAF\\\\\\\/y266+3xJo30MLSlKqFaDJnTt0F3yzQlYUOKK2\\\\\\\/8cDEFXDutw+SI5Q0atyt1ipmVvWCXvs2cyIn2lWvQFzD3x\\\\\\\/y1eXUgg9R27lEX3c4KiJBG6aJCW05fpS5Qf3vH9cWYjRN591t7QkWPdCmpsY+2biSqdmhB3QeO7HVogXeuPGASeiTHQjkkNZjEQ6P7KTK2SfD78jW9+edj2oTD12zfnILuRRGIDdb9fI9w+2XlgQEI3Iqy+1YSg18Tpk\\\\\\\/wS5BlxQ8AHceb2Gb8fHPjHQ4BQRJiiC6BRj6+pZDx7VN3WSKS6RVtfMljqA6eVyD5IUbiJLwypc0DF5C8jw8ywHlAtv5tJdFLonPu\\\\\\\/xV6GQfnSeXfMKc5JdP0ojDGDFVtELmH254iAYMfqbeoyl+d6XpCSMzKsIyABBHuVDjK1I7pMsstTUGkxe+\\\\\\\/d\\\\\\\/IlaX\\\\\\\/9q87To5UQ3qk8tOjdJ8Ga8F57ZK3CMTETtm66nNtY9w7aExse0TwHo1qqp0LHUmEh7CHRMcwFnJZcEXvHgKOsIq5ansgN7DNINSqgZIgttnwUecQUqw0MPShkncL+HUuHDrjLRLWuHmorgc0VnuPZ5W5Z0yUJgNiqEeqjq1J\\\\\\\/xIWwhubpmhvCqaUsLXttGAqHpB8yeeqiK9hkcDnn3YNC6GsMexZgTVMjPTKqNIyuqhFqhlK5YsT3zCPAQzcZquC9fRUkMPJ7KAIOXFZ2H0pH1VXYy\\\\\\\/ybpu0wYKNybTa87Miu+fay\\\\\\\/Lt\\\\\\\/+vMCl94qaTnniz\\\\\\\/y7IcQcuN9fEKQWXI5Hybsyo316pRIF65eyVGsio5vXQslO8BPvVE5cEI2n8d2YjzbGF28YZZMHAuN\\\\\\\/HWQL9iA0WkcJE51je2jnWYpvraO8i2l02JI2448G+NmMSFKnmERjGTQqCInKnQclKudJVBL\\\\\\\/w9Bwu5Yhk253LqsjPuOUQiiEZCoFmHVMlOHJRerIiRJI38GcLyLKNRR0W\\\\\\\/aRmRjpBeK\\\\\\\/M\\\\\\\/l5jN2eGPP8Y7LWC3fXdTesyicECG2fbBC2AByLNMc1nZP0yxQIAauHJD6G9CGkHD8wGof37KW32L680c4hp0PvVdYSjUhDKEdm72mnVWMz8FAhq5WCi0Os7Eidbb7ZGQSsTtaKjzQqVxKBxlOq5Lxyd3jqHq91ULEkJT91lrVhdBC4GIZMPm98zObjcz\\\\\\\/9h5BwHP\\\\\\\/iBAoQ1HFt3emkpAoYaMi9T9lqs+1hmQporiNzSDL3XH9yFxZbWydIcrYSPwQbC9CZa5l7BtiHgEmdCiGbd+vS2yA6isPY\\\\\\\/k9bzszGTnG8vv90xocZJMau3FaBdLEicJl21u7PSkSHXYsu5w+gBJHmgbCnU0uI6IDw2rqmlUMQpUseKJtMeJTauHDtFSvdRas6JIT5QyFBfDDZ7hlD+ydp\\\\\\\/HSPd3DRhpXuq4bZb9gt0KnJIv0eoyGXFqLyOqz9ARICgVbcsepHgTSV6PS3sIDnUdst+4O6nYrqC+E5eDPNIxMGMqDkwbLYW\\\\\\\/OEt8X5BaAigP\\\\\\\/kw+NY9sqPvoM6stLRQ0vSOyuaYpaLz6KYlfcLFyTSWIC29ptjYoG\\\\\\\/Ou8cov88ZcPP2trv78kMGlF7rDXeIkX9zQJnVBens\\\\\\\/vkNkvpZriGu1WktKNtMs4bp0hm8Q4jYMWTTq66rEvDvApFZlROf4GhEcbmf3wAEnvNGjKNwvi4gn7v5Q7ZJ\\\\\\\/p5mPXRRyLjn9E8AkB0DxxzxTh6w4rqDIyw8Jr0rtwSwUSDo4LVj8hmF+njm+bOzHx8xbNzx4zcdV2fW5QWl3K16golUiY5AvGdFGSt29nlpL47viZY18xHqxLi\\\\\\\/7XEufp+ZyjPPpWd\\\\\\\/xlyg8x3Y9TtWz34jXnBoe5CQ1jlTsb74mbdjzAI36imzQ3nc3ej4x2r4BFRyk1zj1Ii\\\\\\\/ajEHvbwSg9x+Cil5p3k1TQoKkCruC5JT5YiB9ZxMjk22ilGS5Nbyf74A8OnaDMsNvyCIvZsiR3va0vi7CLMxzvDyFCA6dg+2LoeVFFDrJEPStJYREqIVTDsnkAn9qRLuFayKyWTV5TR5+yltGg8OIZ8\\\\\\\/UbapUbE9e\\\\\\\/4bJpxGuqdJn2uG3ZYEUSSpXdL\\\\\\\/bdhMAyxdxuNpf8rg6qcGwOIwjsU5JkBI2fVVSXo7cj+j4VC98sPIUc+Cd1YjF\\\\\\\/jpfb6TZRCBeJijpGG9OEJjOjsxf+PRnie1o6Kf8r\\\\\\\/pTX1VB\\\\\\\/zKAk\\\\\\\/mFo4w8OlVU\\\\\\\/g2cfW7C0caCIS1H6uzleHfhgmfdmOfctagELFWOdNHjpnwggVKOPE0R\\\\\\\/VliR1InUAhzOO8Qo6rTnAcSnr4BDASouMl5E8xmIapMeZIuQS2C\\\\\\\/pAj9SzOM\\\\\\\/DHrnt8YSQfZ9\\\\\\\/q4f8CWCDNgxmAqDG\\\\\\\/UC3HtWGy6aBN6TDjQEIlQkJCXVJ2FlPHtqyADdZQpI8ptMAqI4mYA1eFZ2Nr99Xyg3Dc7QVtpVQGBLzmPDu5ognUIgVmzNcmHWacrrAm99j9X4OZ+8zzo3E\\\\\\\/yV0fzR3MaaHz9Bt3n3D4DcoODE7HP3SjzbVpzVZYEJugcHsLLL3MurRiAuFTLXyfQms3ogVII6DqY\\\\\\\/PqJf4ZMuBfLk9SXZqV9Znn336JF17TYblL6sM0hMlAKLlT\\\\\\\/f7WN+jJ1k9u6Z2mdTYLaXZkRW6cST4MOsVNF6QqflYPNLqKV3FAPDw\\\\\\\/heSHzP7eqTkCcccDC8p2wU68HpHFh5Yr17tapMFmLUJP\\\\\\\/UvWhUbJHBK4G+NGBL+iowcka5r7Kif49uZHoaZWpnVt\\\\\\\/0MrbrKyEyOYZIDQO9ZjLBa+wPQqfmK3WvSwAS8riSz\\\\\\\/nxsRbDs57DmYo4w6hx93XLI0Shfonz3VKvn1KeBzsZ74agYHIcVrQRbC0BLlzmXaa7J6YOt8er5YNZeLC0UroNSR7HgbeBRZB\\\\\\\/sh8SFBwCY7c9g4toeB1rqQfLO49QGKK9AvEITiuq0UU7y7xCi6wUbClhVacg3cB3yPW7z\\\\\\\/i0wbP\\\\\\\/Bn037TjTabLSJ30PmzpkXV8u3+NGBAGHw6FRazHTsg3TxKUy+HLckSO4plmsBVlowcsYhhPSQgHqIC\\\\\\\/00IEeeHl8fFVWiwNrGdDdERQwviAHSRxwm4St\\\\\\\/5OqGzy\\\\\\\/+xLcd0FIZvw+rRNuqnrarwUHF44AjJoOvazW2SRlFvk9X9zeAOzCP4rlRCF3JJHSNRdS9cD7vEGcCN29E65UeqiKyyioBUyVifWGywGloWsW6qAAS3Q9UkHkU4EEQiY3\\\\\\\/KpMzOyfAmC+FPYLTvSfyzjjYpVzrIthyWmsnR17FHYT9hXLk7U03a6nxEws7BGjsATOgjp\\\\\\\/2C0oTEkqZQ8z2q17xaySkF\\\\\\\/iTbRdfN7C3+MKzCs5i8qYQq5HOlJ+PuQj9A5uxCwIO02IqyYuPPC5objnJ3dRFVvgSQKMJRrbBlH2s8W1RoBJgSgeXFkxc4EvC654U\\\\\\\/Vnx9AMhjeUW0W\\\\\\\/NV8fyDKFdOmB0Yizh9d9OqkkaGfNX0rfWtL656sHs9aK9BXPqJf4ii0ZI5tcON3SMM6x6aCQ+hK+jxFBpBLLo33wl5oxnqPCKrwOUey\\\\\\\/jhKtr6+H6LE61mmJ5RrZeArCwAEtgU2qhnPWtIDmkEOW3LOV0y5fg9Jp\\\\\\\/FhWJExJk9BIu3G3HdtPXWS3CNbHNhiR0AQe+zU5dMa6VG4UkhOSFvvsNunxVqDUuRWcv1afjgd1DvUsz5AWffBMpy2n9DNsStaf\\\\\\\/JtECPfHQpDhnxHtNmqOBBlGsgGlVhvZtf26yp6Z\\\\\\\/dVcn2kFM3Cl+ZMPplRmNDTMHtHVl4cghVs0PeHI7KakesBFrzeYntVEREsUfFF2Ty3WuUz11zDHrJwpi+HoTmb6Lx3RHmXF2p0v\\\\\\\/Sfbawk8iknrf8BxQnJH+Tn4HXf0XP8zhx778YDpxrksowVZ\\\\\\\/Dz2iMKGZQuXTjAr3Gjv3zUwpKnsiqeLis\\\\\\\/YfEg8k1U2suhRomIkMib0CzYKy\\\\\\\/gcwbPv6qdn4TFu1euu3HNGCJxpsD\\\\\\\/8CK3UmzRd9z5Q5ix70CKgdZ1SrsOpBizKDSE+rmqYedDJI8yg0Q2jsckssy7ZX+PGsgVKIM2lrBYGlzH55zOfOQ6tgHTjczp30EK4fL86+G9A\\\\\\\/bBbrJg5TfnpomCZvWIAv1tDkAkGf3HLGB\\\\\\\/lHmROZVnZloe5xvFazyJowBQu+O\\\\\\\/+J5kuAlEZk2ulDS5GUbDdBTzDbgZ\\\\\\\/xvgVQRLX+f1OloA1EuVwajgEx7lPkbYke0YlNnUFdPQWIPV5Sz1Bh2LAY9MCNFQJucCWMqetpxD8xKlOL0Dmg88B60t96AgNpa\\\\\\\/IuVms2lWMPd1qClaRyTnmcH42YCvotQYZJHZbGDCjrfmy0Ld8Yqov4GCp\\\\\\\/FYJAZ6AR+bWkLuboQj1np6lUu+Q0PqWVFFhIhxaNolE\\\\\\\/Cc1pGecIOe5CfyWJxRbUMt22N9h+aYrEarFSLBDXn7i4wi1wuEO\\\\\\\/dA9HnAKJZVL4aSkXzWSEITndPKjo52ME7UKWMJ2JwUGK0xe2sOq1QROxaKD+5yggJ8QI3WGLGcnQXjB+dHbMM4uMD+hus7rttrMRtnETbTb5pNeRWdZ54Lq3KL2wKWmLwV5xPxrU4fG4OcpJq09ehpKYbLJ7Kfx4lNPasX0k3fpeapIMGLzi8SgdSj6gK5TwR3LIOyInf3ftpd2RGBHcQY8StFJenp6boF9UmLpmd\\\\\\\/QXxyQsWVJgbl7LmC3vjx1WNQ5IGMWK6n40w\\\\\\\/FJyxj9AUeEuOAXGMMSX0kcLnLULFhBbKQvITY92hy56sSdqp\\\u0022,\\\u0022iv\\\u0022:\\\u0022AjZuZcwUq\\\\\\\/O0Vudt+7twfQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221d484165ff31a78f77b24ae96b6bea50e8da13013ccd08b68b682d4876a5c772\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Po6js1JAnMesP9HmeqsWejl3kssAhZRtv6Bw\\\\\\\/OOuuMjVjF2wNBtD+cZa4kb5hnzb5wpHgNaHMUfAgXEQNJwvrJEpuSoByGzIYhEeg6yE\\\\\\\/pnlID2hDgzGLDtc\\\\\\\/SRaRVAzXpAY+0xeNzS+urcQOjxON21kWZCRpqrnCOncIFGiiJqUJbNKotqJjemm7bNDELZPR7KPgfeDe+KyvGFMMWSL4qf+ihzdLVSW539hb621VPr65dpVFPmgtUGTtYALeldY92hTKKZMmwgoXcGk+J3MOodM2fi2kkTeGg3rRtrLOUQh2HmtTsZYBauiL8L80juaiHlrMSw11e+iTaBHDVI6QImJkOR6wOg+gW4Z5zAu5K1p82NyGuO03JXu+rfgOPCZvDZrQhxLAq2FgqGn71HvdV3UtNFJtuh5+4ZST+qSO\\\\\\\/BiW1zIFLFwUFPlS+uoa20Q2WiwFl\\\\\\\/mGTXHVMbc\\\\\\\/4gYvWX8w6Yuw9Je5kClOzMrP6OS9Iy2Cny+NQXJoDi5VVWjG6a0W\\\\\\\/UT94qvEUexpBnKt7NixBue6ywv87K5CC6YNsqMc24c6bGgbTWMJlpl368dzFFdZhsXzQXJbUy7cNBVpAzXwLIg16FqMprRD9K22KnkzrQkyA4M8vggXm+QV7geqzeB57lsOUQMmPWGJwEIrdMXbiAnNX2PlmYY0nWklrefBzUcE8n8Mz40yPNHZbOL8lrtJk7XHJhc2JFJuNQNzdtd++qL+gHpDuttoBS\\\\\\\/tF1Zu7J6JKcswa7AUG+UJgGp\\\\\\\/jeNF6\\\\\\\/5Fh2Ls9Zhex+cF\\\\\\\/Rp5Je61otXtHSR4htvoR7DRq3e2izc0M7ItVH0Fu25FAijRrxzbFDOjxLWDVUv8RWHHyqDRj1Qxik2ICayhDdYj2qBSAVQqC\\\\\\\/GgZKJ1tfqMvJiMoBL740xMQ2O9K11sCb70y2UXbbBTgMnYn2f8KB3KczyITMWr3o9cI4Ryu9KDk43s7XnlCpry2lo8w+0slC8GHQvlhdRm4m3cOXnywryQWFwyas0kUhGRnN9Ou7voNsf46Iz3hR32TzYbW9TAPutLh3k41gw8p7huTkXr1DT34Ck518xmBuGX4bwUbgXuReTd4cYPQihk5l2fHUksbd6QzLZjrzlvaPIG7ugMmbmr3VmGfKrZcNgq4LboTtM2jFXU+4xG40Z8v+VSgT9UhKsW66ba8L84Kv6Vj8EMs4CdKUjaOdIA\\\\\\\/+dY4bblkR5yYQgvN8CJDf2TZIt6ddEhjgwMuAGRpGUWmOOMpiWe\\\\\\\/PRqTy9nvIDUDsfquodyErobvzR4VA7hk14wIUWHvhJhubRlpR0FdE7weK3k+Rijh1J6qi\\\\\\\/Q+pfNR96DuDbTVFEu7Iw5IQh8LQ5uDEB8kZA8o5Yxn8GjHbk8i7UGKzif41+\\\\\\\/AjCFmpQbIemBrefAxET6CwYKkynRdyYooDQfAHP2bwko5PmDtTObykbU3l3KWs0ndR+3IQoCmyn1sgF7T8qcDYfo4ieLyTMbQFvGcwx3TP2U+FklIRVvpYsuid07MeI\\\\\\\/H8iDNKaADWXaZ75I0KPUjUTeuEhq\\\\\\\/ZjmZxqa\\\\\\\/WO30wMx7VgvwD5jZiWq\\\\\\\/\\\\\\\/A07b8quckDUQkh+JtAELeobvVBAXfXHV6vo7oeb4vQ1DllGR1WtbWp0wXlBOREujg+nQ\\\\\\\/5DObaVjOhzarvXMKjdvSyZo+UpG0bsQ4ehaJGCkNUEE6Z6pWzW8a5nruX7220QoauF4S9bb47BBmlJWrAtFKn7NX7dVbXZ1hvsX3vOfgm38frQ0WhqrAJITPk++bFPbs3Lvwgj0Y4krXSszHVfRQz6R8FSclZMugx830fKXVDMXqU5HWZ9uaN6xd0pS\\\\\\\/rrrvvI9S3v62E35C5inmgiM7vuG9DX\\\\\\\/mUa2q5dTXQStK+tC76Tt9Eh6FwSXGM2lS2mbAhjw3t5H+5VXC\\\\\\\/p9hA7\\\\\\\/NNoIuR3ATUrJ48kqqq0n3mXSFkspWGq2HZ6rukAINXZYvom4C6C8imnfTmUMZbAc3vXdjgt\\\\\\\/zmb1MTwGjUG\\\\\\\/KpLL\\\\\\\/5SDcqWLkwT928ZarFnA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ndkJ2TijbqgHP9XHhW+jgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222749034bb633798d3b2b78daaa3c29045d3fb95702a3b59442cbfb5ce5174b3e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460539 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/233" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0pkAxl9TL2Ttn6kBi9WG-x-yRFFVByzQJPh48vozrTU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/233" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ee2zUXUXsMa9euiXh4UTJBX6WQdm64N7nVn-ERMhfBg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/221" title="Time-Course Analysis of Gene Expression During the Saccharomyces cerevisiae Hypoxic Response" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/247" title="Multiplex shRNA Screening of Germ Cell Development by in Vivo Transfection of Mouse Testis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460539" id="node460539" data-pisa="ggg;7/1/233" data-pisa-master="ggg;g3.116.036160" data-apath="/ggg/7/1/233.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gareth A. Cromie</span>, <span class="highwire-citation-author" data-delta="1">Zhihao Tan</span>, <span class="highwire-citation-author" data-delta="2">Michelle Hays</span>, <span class="highwire-citation-author" data-delta="3">Eric W. Jeffery</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">233-246;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036160 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460539" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhihao Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Medical Biology, Agency for Science, Technology and Research, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhihao%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhihao%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michelle Hays </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BHays%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hays%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BHays%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eric W. Jeffery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BW.%2BJeffery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jeffery%20EW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BW.%2BJeffery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/233" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460539" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/233.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460539" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/233.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460539" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/233.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460539" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/233.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460539" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460539/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Aneuploidy, a state in which the chromosome number deviates from a multiple of the haploid count, significantly impacts human health. The phenotypic consequences of aneuploidy are believed to arise from gene expression changes associated with the altered copy number of genes on the aneuploid chromosomes. To dissect the mechanisms underlying altered gene expression in aneuploids, we used RNA-seq to measure transcript abundance in colonies of the haploid <em>Saccharomyces cerevisiae</em> strain F45 and two aneuploid derivatives harboring disomies of chromosomes XV and XVI. F45 colonies display complex “fluffy” morphologies, while the disomic colonies are smooth, resembling laboratory strains. Our two disomes displayed similar transcriptional profiles, a phenomenon not driven by their shared smooth colony morphology nor simply by their karyotype. Surprisingly, the environmental stress response (ESR) was induced in F45, relative to the two disomes. We also identified genes whose expression reflected a nonlinear interaction between the copy number of a transcriptional regulatory gene on chromosome XVI, <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, and the copy number of other chromosome XVI genes. <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> and the remaining chromosome XVI genes also demonstrated distinct contributions to the effect of the chromosome XVI disome on ESR gene expression. Expression changes in aneuploids appear to reflect a mixture of effects shared between different aneuploidies and effects unique to perturbing the copy number of particular chromosomes, including nonlinear copy number interactions between genes. The balance between these two phenomena is likely to be genotype- and environment-specific.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/aneuploidy" class="hw-term hw-article-keyword hw-article-keyword-aneuploidy" rel="nofollow">aneuploidy</a></li><li class="kwd"><a href="/keyword/gene-expression" class="hw-term hw-article-keyword hw-article-keyword-gene-expression" rel="nofollow">gene expression</a></li><li class="kwd"><a href="/keyword/colony-morphology" class="hw-term hw-article-keyword hw-article-keyword-colony-morphology" rel="nofollow">colony morphology</a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-seq</a></li></ul><p id="p-3">Chromosome missegregation during meiosis or mitosis can produce cells that are aneuploid, containing a chromosome complement that is not a multiple of the haploid count. This often results in strong phenotypic effects. In humans, inheritance of a single copy of any autosome (monosomy) or three copies of most autosomes (trisomy) is an embryonic lethal event. The most common, viable, inherited aneuploidy is trisomy 21, which leads to the collection of phenotypes that comprise Down syndrome. In addition, several imbalances in sex chromosomes are viable, including Turner Syndrome (X0) and Klinefelter Syndrome (XXY).</p><p id="p-4">While meiotic chromosome missegregation events lead to inherited aneuploidy, mitotic missegregation produces aneuploid subpopulations of cells and mosaicism. This phenomenon is strongly associated with cancer. Tumors show a very high frequency of aneuploidy, but the degree to which this aneuploidy is simply a result of the disordered cellular processes in cancer <em>vs.</em> actually promoting cellular transformation is unclear (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gordon <em>et al.</em> 2012</a>).</p><p id="p-5">The mechanism linking aneuploidy to phenotype is believed to operate primarily through changes in gene expression. By altering the copy number of a subset of genes, relative to the rest of the genome, aneuploidy causes a change in the relative expression level of those genes. In the case of Down syndrome, for example, the relative expression of genes on chromosome 21 has been shown to be increased (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mao <em>et al.</em> 2003</a>), mirroring the increased copy number of those genes. This increase in gene expression can produce phenotypic effects in several ways. One mechanism is via direct downstream effects of the overexpressed genes. For example, individuals with Down syndrome show increased risk for early onset dementia, most likely because the gene encoding the amyloid precursor protein is present on chromosome 21 and therefore has elevated expression in trisomic individuals (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Salehi <em>et al.</em> 2006</a>). In addition, work in model organisms has shown that increasing the expression level of regulatory genes, such as transcription factors, can lead to a cascade of downstream effects across the rest of the genome, also producing phenotypic effects (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rancati <em>et al.</em> 2008</a>). However, for most of the phenotypes associated with Down syndrome, or other aneuploidies, neither the identity of the relevant overexpressed genes, nor the mechanism by which their overexpression produces the phenotype, are understood.</p><p id="p-6">Dissecting the mechanisms through which aneuploidy exerts phenotypic effects in humans is limited by technical and ethical constraints. However, work in model organisms, particularly yeast, has provided several insights into these questions. These studies have revealed that many of the phenotypes associated with the transition from euploidy to aneuploidy are specific to particular aneuploid states (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pavelka <em>et al.</em> 2010</a>). However, it also appears that some phenotypic effects are common to multiple aneuploid states. In general, aneuploid cells appear to suffer proliferation and growth defects, although the intensity of these effects varies across specific aneuploid states and genotypes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sheltzer <em>et al.</em> 2012</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hose <em>et al.</em> 2015</a>). Similarly, many aneuploid cells appear to suffer from protein transport defects (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dodgson <em>et al.</em> 2016</a>) and proteotoxic stress, with the latter perhaps caused by imbalances in protein stoichiometry in multi-subunit complexes (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sheltzer and Amon 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Oromendia <em>et al.</em> 2012</a>).</p><p id="p-7">In any given environment, the fitness of an aneuploid appears to reflect a combination of these general negative impacts of aneuploidy along with any phenotypes associated with the particular pattern of chromosome (and hence gene) copy number change. With sufficiently strong selective pressure, the benefits of the specific effects can sometimes outweigh the general costs of aneuploidy, giving aneuploids a selective advantage (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Sheltzer and Amon 2011</a>). If selection is then removed, a return to euploidy will be favored, while if selection is maintained, the cells appear to respond either by finding an alternative, nonaneuploid solution to the selective pressure (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yona <em>et al.</em> 2012</a>), or by maintaining aneuploidy but ameliorating the associated negative effects (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Torres <em>et al.</em> 2010</a>).</p><p id="p-8">In a previous study, we identified a budding yeast system in which transition between euploid and aneuploid states is associated with a strong phenotypic response: a dramatic change in colony morphology (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>). Our original, euploid strain produces colonies with a complex “fluffy” morphology, while certain disomic derivatives of this strain display the “smooth” colony morphology characteristic of most laboratory strains. We were able to demonstrate that, while disomy of several different chromosomes induced this phenotypic switch, disomy of several others did not. This argues that the phenotypic switch is not the result of some general effect of the aneuploid state, but rather a response to changes in the expression of dosage-sensitive genes that affect colony morphology. In support of this gene-dosage model, we found that the effect on colony morphology caused by gaining an extra copy of chromosome XVI could be replicated by increasing the copy number of a single gene on that chromosome, <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, a transcriptional regulator of colony morphology, in an otherwise haploid (euploid) genome. However, because deleting one copy of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> in a chromosome XVI disome only partially restored the fluffy phenotype of F45, we hypothesized that other dosage-sensitive genes on that chromosome also affect colony morphology (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>).</p><p id="p-9">Here, we use RNA-seq to characterize and dissect the specific factors underlying the transcriptional effects of gaining an extra copy of two different chromosomes (XV and XVI) in the F45 genetic background. Each of these disomies causes F45 to adopt a smooth colony morphology. We expected that some of the factors driving differences in gene expression between the euploid and disomic colonies would be shared between the two disomes. These include nonspecific stress responses to aneuploidy and shared environmental effects operating in the smooth (disomic) <em>vs.</em> fluffy (euploid) colonies. We expected that other factors would be specific to each disome, including direct copy number effects affecting genes on the disomic chromosomes themselves and downstream effects of altering the copy number of regulatory genes on the disomic chromosomes. We also specifically investigated how copy number changes affecting a transcriptional regulatory gene (<em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>) interact with the copy number of other genes on the same chromosome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and media</h3><p id="p-10">Unless noted, standard media and methods were used for growth and genetic manipulation of yeast (Rose 1990). The strains used in this study are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470222/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470222/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Yeast strains used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>RNA preparation and sequencing</h3><p id="p-11">After 4 d of growth on YPD (2% glucose) plates at 30°, whole colonies were harvested by scraping the surface of the agar plate. To obtain sufficient amounts of RNA, 3–5 colonies were pooled for each sample. Samples were collected and processed in the following batches, each containing euploid, wild-type F45 as a control. The first batch consisted of four replicates of: YPG725 (F45), YO902 (disome XV), YO785 (disome XVI), YO1000 (disome XVI, hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">dig1</a></em>Δ), YO1853 (control plasmid), and YO1773 (<em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> plasmid) (<a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>). The second batch consisted of three replicates of YPG725 (F45), and two replicates of YO2085 (<em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ), YO2086 (<em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ), YO2088 (<em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ), and YO2089 (<em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ). Total RNA was prepared as described previously (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Swanson <em>et al.</em> 1991</a>).</p><p id="p-12">Following extraction, total RNA from the pooled colonies was quantified by Bioanalyzer (Agilent). Total RNA for each sample (5 µg) was then processed using the Tru-Seq stranded mRNA kit (Illumina) following the manufacturer’s instructions. Individual sample libraries were pooled for sequencing. The first batch was analyzed by paired-end, 51 nucleotide read sequencing in one lane of an Illumina HiSeq2000. The second batch was analyzed by paired-end, 75 nucleotide read sequencing on an Illumina NextSeq 500.</p></div><div id="sec-4" class="subsection"><h3>Extending the reference genome and annotation</h3><p id="p-13">Before read-pair alignment, the S288c GFF (General Feature Format) file was extended to cover additional classes of noncoding RNA, specifically CUTs and SUTs (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xu <em>et al.</em> 2009</a>), MUTs (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lardenois <em>et al.</em> 2011</a>), and XUTs (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">van Dijk <em>et al.</em> 2011</a>), using data downloaded from the <em>Saccharomyces</em> Genome Database (SGD) on 4/28/2014. Additionally, the reference genome fasta and GFF files were extended to include genes present in F45, but absent in S288c. Note that the chromosomes encoding these genes are unknown. We carried out whole-genome sequencing of F45 using a MiSeq (Illumina) with a fragment size range of ∼600–800 bp and paired-end 250 bp reads. Reads were trimmed for adaptor sequences using cutadapt (v. 1.7.1) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marcel 2011</a>) and aligned to the S288c reference using the BWA (v. 0.7.5a) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li and Durbin 2009</a>) mem command. Read-pairs where neither read aligned to the reference genome then underwent <em>de novo</em> assembly using IDBA (v. 1.1.1) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Peng 2010</a>) with the parameters–mink 23–maxk 35–step 2–seed_kmer 23. Nucleotide BLAST was used to identify potential ORFs in the novel contigs, and these ORFs were appended to both the reference sequence and the annotation file (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FileS2.zip">File S2</a>). The aligned reads are available from the European Nucleotide Archive (ENA) under study PRJEB15176.</p></div><div id="sec-5" class="subsection"><h3>Read-pair alignment</h3><p id="p-14">Read-pair alignments for RNA-seq data were carried out against the S288c reference (R64-1-1), extended as described above, using Bowtie2 (version 2.1.0) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Langmead and Salzberg 2012</a>) with the parameters [-N 1 -I 50 -X 450 -p 6–reorder -x -S] and allowing one mismatch per read. For each strain, read alignments were converted to gene counts using featureCounts (version 1.4.0) in the Subread package (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Liao <em>et al.</em> 2014</a>), with the parameters [-a -o -t gene –g ID –s 2 -T 1 -p -P -d 50 -D 450]. Reads were not filtered based on mapping quality, and we are thus cautious in our interpretation of counts of genes that have paralogs with similar sequences, or which contain large low complexity regions. Read sequences and gene count tables are available from the Gene Expression Omnibus (GEO) under accession GSE85843.</p></div><div id="sec-6" class="subsection"><h3>Differential expression analysis</h3><p id="p-15">Analysis of differential gene expression was carried out using edgeR [v. 3.6.8] (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Robinson <em>et al.</em> 2010</a>) based on the tables of raw counts produced by featureCounts (<em>Materials and Methods</em>). Separate data normalization and analyses were conducted for the first and second sample batches, each of which included a wild-type control (F45 or F45 with empty vector). Library sizes were normalized using calcNormFactors, with only nuclear-encoded ORFs (genes with systematic names beginning with “Y”) used to estimate the normalization factor and excluding chromosomes XV and XVI for the first batch of samples, which included the disomic strains. This normalization factor was then applied to all genes and dispersion parameters were estimated using the estimateGLMTrendedDisp and estimateGLMTagwiseDisp commands. To identify genes differentially expressed between strains, we conducted pairwise testing using the glmFit and glmLRT commands, with a p-value cutoff of 0.01 (after Benjamini–Hochberg multiple hypothesis correction, <em>i.e.</em>, false discovery rate). Because principal coordinate analysis using moderated (plus two counts per gene) counts per gene per million reads demonstrated strong clustering on the first principal component (PC) of strains YO285 and YO286 (<em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ) <em>vs.</em> F45, and strains YO2088 and YO2089 (<em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ) <em>vs.</em> F45, RNA-seq data from each of these pairs were combined for further analysis. When plotting log<sub>2</sub> fold changes on scatterplots, calculating regression parameters, correlation coefficients, Wilcoxon rank sum tests, medians, and SD, only nuclear-encoded ORFs, present in the reference genome with median basal expression of at least one read per million reads in F45 (first batch), were included. In addition, for each genotype, the variance of the log<sub>2</sub> moderated (plus two counts per gene) counts per gene per million reads was calculated for each gene, and genes with variances ≥ 3 were removed from any analysis using data from that genotype.</p></div><div id="sec-7" class="subsection"><h3>Differential expression from disomic chromosomes</h3><p id="p-16">For chromosomes I–XIV, genes differentially expressed at the p &lt; 0.01 level with a minimum expression change of twofold between the two disomes, or between the disomes and F45, were identified as above. To identify differential expression of genes on chromosomes XV and XVI, independent of the direct effect of copy number, normalization was repeated for each disome using twice the value calculated by calcNormFactors applied to chromosomes I–XIV. These normalized values were used for comparisons involving the disomic chromosomes. The genes identified in this way as differentially expressed from chromosomes XV and XVI were added to the list from chromosomes I–XIV.</p></div><div id="sec-8" class="subsection"><h3>Functional enrichment of gene lists</h3><p id="p-17">We looked for functional enrichment of <em>Saccharomyces cerevisiae</em> gene lists using g:Profiler (<a href="http://biit.cs.ut.ee/gprofiler/">http://biit.cs.ut.ee/gprofiler/</a>). Holm–Bonferroni corrected Enrichment p–values &lt; 0.05 were accepted, with moderate hierarchical filtering.</p></div><div id="sec-9" class="subsection"><h3>Principal component analysis (PCA)</h3><p id="p-18">PCA was carried out using the princomp command in R (version 3.1.1, (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">R Core Team 2014</a>). Loadings for the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a>/<a href="http://www.yeastgenome.org/locus/S000005970/overview">dig1</a></em>Δ strain, chromosome XVI disome, <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>-overexpressing strain, and the empty plasmid control were calculated using only nuclear encoded-ORFs with median basal expression of at least one read per million reads in F45 (first batch). These loadings were then applied to the whole dataset. Significant nonzero values on the second <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> PC were identified using the generalized linear model functionality of edgeR (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Robinson <em>et al.</em> 2010</a>). Normalization for library size (calcNormFactors) was carried out using only nuclear-encoded ORFs (genes with systematic names beginning with “Y”) and excluding chromosome XVI. This normalization was then applied to all genes. Dispersion parameters were estimated using estimateGLMTrendedDisp and estimateGLMTagwiseDisp commands. Significant nonzero values on the second <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> PC were identified using the glmLRT command with a p-value cutoff of 0.01 (after Benjamini–Hochberg multiple hypothesis correction). Treating the intercept as zero, from the regression line corresponding to the first PC we expect that 1 × (hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a>/<a href="http://www.yeastgenome.org/locus/S000005970/overview">dig1</a></em>Δ strain − chromosome XVI disome expression) + 0.54 × (<em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>-overexpressing − empty plasmid control expression) will equal zero when a zero value occurs on the second PC. Therefore, significant nonzero values in the second PC were identified using glmLRT with the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a>/<a href="http://www.yeastgenome.org/locus/S000005970/overview">dig1</a></em>Δ strain, the chromosome XVI disome, the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>-overexpressing strain, and the empty plasmid control strain given the weights 1, −1, 0.54, and −0.54 to reflect the regression relationship.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-19">The datasets generated during and/or analyzed in the current study are available in the GEO repository under GSE85843 and ENA repository under PRJEB15176.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><p id="p-20">We previously isolated several disomic derivatives of the strain F45, including one harboring an extra copy of chromosome XV and another harboring an extra copy of chromosome XVI (<a id="xref-ref-44-9" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>). Because these disomies arose by mitotic missegregation, the two copies of the disomic chromosomes are homozygous. Euploid F45 produces colonies with a strong “fluffy” phenotype, characterized by an intricate pattern of ruffles and channels. In contrast, the chromosome XV and XVI disomes both produce smooth, unstructured colonies similar to commonly used laboratory strains (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). To characterize gene expression differences between these strains, we performed RNA-seq analysis on RNA isolated from fully developed colonies grown on solid medium (YPD, 2% glucose) for 4 d. Because F45 shows significant sequence divergence from the <em>S. cerevisiae</em> reference genome, we also performed whole-genome sequence analysis and <em>de novo</em> assembly of the F45 strain background. This analysis predicted several genes not present in the S288c genome, and these nonreference genes were added to our expression analysis pipeline (<em>Materials and Methods</em>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Strain colony morphologies. All images were taken with a Canon PowerShot SX101S. (B) Model showing different mechanisms by which aneuploidy can affect gene expression. Chr, chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Strain colony morphologies. All images were taken with a Canon PowerShot SX101S. (B) Model showing different mechanisms by which aneuploidy can affect gene expression. Chr, chromosome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F1.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/1/233/F1.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470051" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">(A) Strain colony morphologies. All images were taken with a Canon PowerShot SX101S. (B) Model showing different mechanisms by which aneuploidy can affect gene expression. Chr, chromosome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">In principle, the transcriptional profile of any given disome could be composed of several distinct factors, with some unique to each karyotype and others shared by multiple aneuploid states (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). First, there are <em>cis</em> effects on the expression of genes on the disomic chromosomes themselves, which should be increased ∼twofold compared to an isogenic euploid strain. Second, there are <em>trans</em> effects caused by the differential expression of regulatory genes on the disomic chromosomes, which could produce a large number of downstream effects on transcription across the whole genome. Both of these effects would be expected to be largely disome-specific. Third, either of the first two mechanisms could produce phenotypic effects that, in turn, could feed-back on the expression of additional genes. In such a case, different disomes that place cells in the same phenotypic state might be expected to share an associated transcriptional response. Fourth, general transcriptional responses to the aneuploid state, which are not specific to changing the copy number of specific chromosomes, have been described (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>) (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hose <em>et al.</em> 2015</a>). Responses of this type should be shared by many, if not all, disomes.</p><div id="sec-12" class="subsection"><h3>Gene expression from disomic chromosomes is elevated twofold</h3><p id="p-23">Consistent with other studies (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hughes <em>et al.</em> 2000</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">2016</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Rancati <em>et al.</em> 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dephoure <em>et al.</em> 2014</a>), the average expression of genes on the disomic chromosomes was increased relative to the F45 euploid (median 1.79-fold for disome XV and median 1.86-fold for disome XVI) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). To control for shared effects on expression, such as those that might be caused by the smooth colony morphology of the disomes or shared effects of the aneuploid state, we then compared gene expression on chromosomes XV and XVI between the two disomes themselves. Genes on chromosome XV had a median expression level 1.98-fold higher in the strain disomic for chromosome XV, and genes on chromosome XVI had a median expression level 1.96-fold higher in the chromosome XVI disome. The SD of the F45:disome log<sub>2</sub> ratios (0.65 for chromosome XV and 0.59 for chromosome XVI) was decreased (0.55 and 0.52, respectively) in the disome:disome comparisons. Therefore, as expected, the effect of doubling chromosome copy number, when isolated from other factors, is to double gene expression levels from the affected chromosome. However, the difference between the SD of the disome:F45 comparisons and the disome:disome comparisons suggests that factors shared between the two disomes do contribute significantly to the differential expression of genes on chromosomes XV and XVI, relative to F45, independent of the copy number of those genes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene expression from the disomic chromosomes is &#x223C;twofold higher than from the rest of the genome. Chromosome XV expression is shown in green, chromosome XVI expression in orange, and expression from the other chromosomes in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Gene expression from the disomic chromosomes is &#x223C;twofold higher than from the rest of the genome. Chromosome XV expression is shown in green, chromosome XVI expression in orange, and expression from the other chromosomes in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F2.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/1/233/F2.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470128" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Gene expression from the disomic chromosomes is ∼twofold higher than from the rest of the genome. Chromosome XV expression is shown in green, chromosome XVI expression in orange, and expression from the other chromosomes in gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Role of copy number changes of the DIG1 transcriptional repressor gene in the disome XVI transcriptional profile</h3><p id="p-25"><em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, which encodes a transcriptional repressor of the “filamentous growth” MAPK pathway (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cook <em>et al.</em> 1996</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Granek and Magwene 2010</a>; <a id="xref-ref-44-10" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>), is known to regulate complex colony morphology (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Granek and Magwene 2010</a>; <a id="xref-ref-44-11" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>) and is located on chromosome XVI. We previously demonstrated that overexpression of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, in an F45 euploid background, largely recapitulates the phenotypic effect of the full chromosome XVI disome, <em>i.e.</em>, produces smooth colonies (<a id="xref-ref-44-12" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>). Similarly, deletion of a single copy of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> in a chromosome XVI disome partially restores the fluffy phenotype of F45, producing colonies with fluffy perimeters, but smooth centers (<a id="xref-ref-44-13" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>).</p><p id="p-26">One of the mechanisms that we expected to drive disome-specific gene expression changes was through copy number changes of regulatory genes found on the disomic chromosome (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). To investigate the contribution of changes in the copy number of a chromosome XVI regulatory gene to the differential transcription profile of the full chromosome XVI disome, we used RNA-seq to analyze the expression profile of colonies grown on solid agar in a chromosome XVI disome that was hemizygous for <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a>/<a href="http://www.yeastgenome.org/locus/S000005970/overview">dig1</a></em>Δ). As expected, expression of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> itself was increased ∼twofold (1.90-fold) in the disome, relative to F45, and then reduced ∼twofold (2.28-fold) in the hemizygous deletion (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3B</a>). Relative to euploid F45, the differential expression profile of the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> chromosome XVI disome was extremely similar to that of the chromosome XVI disome, which harbored both copies of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> (<em>R</em> = 0.92, Pearson test, 2-tailed, <em>t</em> = 174.3026, df = 5777, p-value &lt; 1e−15) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). Therefore, despite the pronounced effect of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number on the colony morphology, the transcriptional differences between F45 and the chromosome XVI disome are largely independent of the effect of the copy number of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Differential gene expression in the chromosome XVI disome and the hemizygous DIG1/dig1&#x394; derivative of the chromosome XVI disome, both relative to F45. Genes on disomic chromosome XVI had expression ratios adjusted downward twofold. x = y line shown in gray. (B) Differential gene expression in the chromosome XVI disome, relative to F45 and the hemizygous DIG1 deletion, relative to the full chromosome XVI disome. Genes on disomic chromosome XVI had expression ratios adjusted downward twofold in the disome vs. F45 dataset. Red points are genes whose expression changed significantly, and &gt;twofold (up or down), between the chromosome XVI disome and the disome with the hemizygous DIG1 deletion. Gray line indicates F45 expression level. Chr, chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Differential gene expression in the chromosome XVI disome and the hemizygous &lt;em&gt;DIG1&lt;/em&gt;/&lt;em&gt;dig1&lt;/em&gt;&#x394; derivative of the chromosome XVI disome, both relative to F45. Genes on disomic chromosome XVI had expression ratios adjusted downward twofold. &lt;em&gt;x&lt;/em&gt; = &lt;em&gt;y&lt;/em&gt; line shown in gray. (B) Differential gene expression in the chromosome XVI disome, relative to F45 and the hemizygous &lt;em&gt;DIG1&lt;/em&gt; deletion, relative to the full chromosome XVI disome. Genes on disomic chromosome XVI had expression ratios adjusted downward twofold in the disome &lt;em&gt;vs.&lt;/em&gt; F45 dataset. Red points are genes whose expression changed significantly, and &gt;twofold (up or down), between the chromosome XVI disome and the disome with the hemizygous &lt;em&gt;DIG1&lt;/em&gt; deletion. Gray line indicates F45 expression level. Chr, chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F3.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/1/233/F3.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470079" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">(A) Differential gene expression in the chromosome XVI disome and the hemizygous <em>DIG1</em>/<em>dig1</em>Δ derivative of the chromosome XVI disome, both relative to F45. Genes on disomic chromosome XVI had expression ratios adjusted downward twofold. <em>x</em> = <em>y</em> line shown in gray. (B) Differential gene expression in the chromosome XVI disome, relative to F45 and the hemizygous <em>DIG1</em> deletion, relative to the full chromosome XVI disome. Genes on disomic chromosome XVI had expression ratios adjusted downward twofold in the disome <em>vs.</em> F45 dataset. Red points are genes whose expression changed significantly, and &gt;twofold (up or down), between the chromosome XVI disome and the disome with the hemizygous <em>DIG1</em> deletion. Gray line indicates F45 expression level. Chr, chromosome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Closer examination of the data, however, indicated that, among the genes most strongly induced in the full chromosome XVI disome, relative to F45, most did show a reduction in expression in the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> hemizygous strain (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, A and B</a>). The genes in this group (<em><a href="http://www.yeastgenome.org/locus/S000005514/overview">ZPS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005914/overview">YOR387C</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003227/overview">VEL1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003911/overview">DAN1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003912/overview">DAN4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000701/overview">PAU3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003614/overview">PRY3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001843/overview">YFL051C</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005094/overview">YNL150W</a></em>) mostly encode cell surface proteins, with expression of the first three responding strongly to zinc levels (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lyons <em>et al.</em> 2000</a>). However, consistent with the incomplete restoration of the fluffy phenotype in the hemizygous strain, gene expression levels were generally not fully restored to the F45 euploid level (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>, bottom right).</p><p id="p-29">In contrast to the effect of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> hemizygosity on genes induced in the chromosome XVI disome, its effect on expression of genes strongly repressed in the disome was more complex. Most genes strongly repressed in the chromosome XVI disome, relative to F45, showed a similarly low level of expression in the hemizygous strain (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3, A and B</a>). However, a small number of repressed genes were significantly (p &lt; 0.01, after multiple hypothesis correction) and substantially (&gt;twofold) more highly expressed in the hemizygous strain. This set of genes (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>) included <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em>, which encodes a flocculin that plays a critical role in complex colony morphology (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lo and Dranginis 1998</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Granek and Magwene 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Voordeckers <em>et al.</em> 2012</a>), the <em><a href="http://www.yeastgenome.org/locus/S000002930/overview">SPS2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005781/overview">OSW1</a></em> genes, which are required for the construction of spore walls (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Coluccio <em>et al.</em> 2004</a>), <em><a href="http://www.yeastgenome.org/locus/S000004032/overview">YLR042C</a></em>, which encodes an uncharacterized cell wall protein, and two dubious ORFs (<em><a href="http://www.yeastgenome.org/locus/S000002658/overview">YDR250C</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004785/overview">YMR173W-A</a></em>). Complex colony formation is known to be dependent on cell wall genes such as the flocculins (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Guo <em>et al.</em> 2000</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Voordeckers <em>et al.</em> 2012</a>), and it seems likely that among the cell wall genes identified in this group are genes important for the partial restoration of the fluffy phenotype seen in the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion. As with the set of genes that showed reduction in the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG 1</a></em> hemizygous strain, expression of these genes was changed relative to the full disome, but not fully restored to their F45 euploid levels (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>). Interestingly, expression of the <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FL010</a></em> flocculin, which was strongly repressed in the chromosome XVI disome relative to F45, was essentially unaffected by the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> hemizygous deletion, remaining just as repressed relative to F45 (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3A</a>).</p><p id="p-30">A final group of genes were both strongly repressed by the chromosome XVI disome, relative to F45, and substantially (&gt;twofold) and significantly (p &lt; 0.01, after multiple hypothesis correction) more strongly repressed in the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> strain, relative to the disome harboring two copies of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a>, bottom left). This group consisted of <em><a href="http://www.yeastgenome.org/locus/S000004592/overview">PHO84</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000500/overview">PHO89</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001178/overview">SPL2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000695/overview">GIT1</a></em>, genes involved in phosphate transport that are induced in response to low phosphate conditions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Almaguer <em>et al.</em> 2003</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ljungdahl and Daignan-Fornier 2012</a>), and two dubious ORFs (<em><a href="http://www.yeastgenome.org/locus/S000004330/overview">YLR338W</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004591/overview">YML122C</a></em>). The expression of these genes appears to respond to a nonlinear interaction between <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number and that of one or more other genes on chromosome XVI (see below).</p></div><div id="sec-14" class="subsection"><h3>Comparing the effect of changing DIG1 copy number in euploid F45 <em>vs.</em> in the chromosome XVI disome</h3><p id="p-31">Our previous study found that reducing the copy number of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, via a hemizygous deletion in the context of a chromosome XVI disome, partially converted the smooth morphology of the disome back to the fluffy phenotype of F45 (<a id="xref-ref-44-14" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>). However, moderate (∼two- to fivefold) overexpression of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG 1</a></em> in euploid F45, from a low copy number plasmid, caused a stronger change in colony morphology from fluffy to fully smooth (<a id="xref-ref-44-15" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). To compare the effect of perturbing the copy number of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> in the euploid <em>vs.</em> the chromosome XVI disome genetic backgrounds, we carried out RNA-seq and differential expression analysis on smooth colonies from the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression strain.</p><p id="p-32">Comparing the effects of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression, relative to euploid F45, to the effect of the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion, relative to the full chromosome XVI disome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), revealed a modest but significant correlation between the effects of the two perturbations (<em>R</em> = −0.42, Pearson test, two-tailed, <em>t</em> = −34.917, df = 5777, p-value &lt; 1e−15). As expected from the opposite directions of effect on <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number (overexpression <em>vs.</em> copy number reduction), this correlation was negative. However, the weakness of this correlation suggests that the effect of perturbing <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number might depend on the copy number of the genes on the rest of chromosome XVI.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Differential gene expression in the DIG1 overexpression strain, relative to F45 (with empty vector) and the hemizygous DIG1 derivative (DIG1/dig1&#x394;) of the chromosome XVI disome, relative to the full chromosome XVI disome. Gray line indicates TLS regression. (B) Rotation of the data using PCA. First principal component (x-axis) is equivalent to the TLS regression line in (A). Genes with significantly higher than expected expression in the hemizygous DIG1 strain are shown in red, genes with significantly lower than expected expression in blue. PCA, principal component analysis; TLS, total least squares." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Differential gene expression in the &lt;em&gt;DIG1&lt;/em&gt; overexpression strain, relative to F45 (with empty vector) and the hemizygous &lt;em&gt;DIG1&lt;/em&gt; derivative (&lt;em&gt;DIG1/dig1&lt;/em&gt;&#x394;) of the chromosome XVI disome, relative to the full chromosome XVI disome. Gray line indicates TLS regression. (B) Rotation of the data using PCA. First principal component (&lt;em&gt;x&lt;/em&gt;-axis) is equivalent to the TLS regression line in (A). Genes with significantly higher than expected expression in the hemizygous &lt;em&gt;DIG1&lt;/em&gt; strain are shown in red, genes with significantly lower than expected expression in blue. PCA, principal component analysis; TLS, total least squares.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F4.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/1/233/F4.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470062" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">(A) Differential gene expression in the <em>DIG1</em> overexpression strain, relative to F45 (with empty vector) and the hemizygous <em>DIG1</em> derivative (<em>DIG1/dig1</em>Δ) of the chromosome XVI disome, relative to the full chromosome XVI disome. Gray line indicates TLS regression. (B) Rotation of the data using PCA. First principal component (<em>x</em>-axis) is equivalent to the TLS regression line in (A). Genes with significantly higher than expected expression in the hemizygous <em>DIG1</em> strain are shown in red, genes with significantly lower than expected expression in blue. PCA, principal component analysis; TLS, total least squares.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34"><a href="http://www.yeastgenome.org/locus/S000005970/overview">Dig1</a> binds the <a href="http://www.yeastgenome.org/locus/S000001126/overview">Ste12</a> transcription factor and represses transcriptional activation of <a href="http://www.yeastgenome.org/locus/S000001126/overview">Ste12</a> target genes, including targets of the Ste12-Tec1 complex with roles in complex colony morphology (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Madhani and Fink 1997</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chou <em>et al.</em> 2006</a>). Examining 41 genes with conserved <a href="http://www.yeastgenome.org/locus/S000001126/overview">Ste12</a> and <a href="http://www.yeastgenome.org/locus/S000000287/overview">Tec1</a> binding motifs (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Chou <em>et al.</em> 2006</a>), we observed a significant overrepresentation (29/2197 <em>vs.</em> 12/4388; Fisher’s Exact Test, two-tailed, p-value &lt; 1.1e−06) of these genes among the set of genes, including <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004032/overview">YLR042C</a></em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>), which were both repressed in the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression strain (log<sub>2</sub>FC &lt; 0) and induced in the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000005970/overview">dig1</a></em>Δ strain (log<sub>2</sub>FC &gt;0), relative to F45 and the chromosome XVI disome harboring both copies of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, respectively.</p><p id="p-35">The effect on transcription of the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression strain appears to be stronger than the effect of the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">dig1</a></em> hemizygous deletion (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>), as total least squares (TLS) regression had a slope of −0.54. This is consistent with the larger change in <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression level in the overexpression strain (4.25-fold up, relative to F45) compared with the hemizygous deletion (2.28-fold down, relative to the full chromosome XVI disome). A simple model for <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> action would suggest that plotting the (log) effects of halving <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression against the effects of doubling <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression would give a regression slope of −1, <em>i.e.</em>, reflecting the ratio of the log changes in <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression. Given the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression changes observed in the hemizygous deletion and overexpression strains, this model predicted a slope of −0.57, and an intercept of 0, for the regression in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>, very close to the observed values of −0.54 and −0.025. Interestingly, however, there were several strong outliers from the regression trend. These included <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em>, whose expression level was changed by almost exactly the same amount after both perturbations, rather than showing a larger relative transcriptional change in the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression strain, as expected from the TLS regression slope (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>).</p></div><div id="sec-15" class="subsection"><h3>Gene expression changes reflecting a nonlinear interaction between chromosome XVI ploidy and DIG1 expression level</h3><p id="p-36">To further characterize the differences between the effects of perturbing <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number in the disome <em>vs.</em> euploid F45, we performed a principle component analysis (PCA) that was mathematically equivalent to rotating the data in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a> to make the TLS regression line into the new <em>x</em>-axis (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>) (<em>Materials and Methods</em>). The first PC (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4B</a>, <em>x</em>-axis) then captures the major axis of covariation between the effect of the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> hemizygous deletion and the effect of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression. The second PC (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4B</a>, <em>y</em>-axis) captures variation orthogonal to this first axis, <em>i.e.</em>, behavior different to that expected from the major trend. Genes whose expression responds nonlinearly to the two perturbations of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number (in the two different genetic backgrounds) should score highly on this second PC. Because the magnitudes of the two perturbations of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression level are different (4.25-fold <em>vs.</em> 2.28-fold), one explanation for the behavior of these genes could be that they are regulated solely by <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression level, but that the relationship between their expression and that of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> is nonlinear. Alternatively, the behavior of these genes could represent a nonlinear interaction between <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression level and the expression level of other regulatory genes on chromosome XVI.</p><p id="p-37">To identify genes with values significantly different to zero in the second PC, we used the generalized linear model functionality of EdgeR (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Robinson <em>et al.</em> 2010</a>) with the observed regression slope of −0.54 and treating the intercept as zero (<em>Materials and Methods</em>). These genes can then be split into two groups, those showing an expression change in the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion that is either higher or lower than expected, based on their behavior in the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression strain. In the first group, we observed only three genes whose expression change was more than twofold higher than expected in the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion, based on their behavior in the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression strain. This group consisted of <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> and the ammonium permease <em><a href="http://www.yeastgenome.org/locus/S000005086/overview">MEP2</a></em>, both known to have roles in complex colony morphology (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lo and Dranginis 1998</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lorenz and Heitman 1998a</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Voordeckers <em>et al.</em> 2012</a>), and the poorly characterized gene <em><a href="http://www.yeastgenome.org/locus/S000003468/overview">SPG1</a></em> (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4, A and B</a>). In the second group, we observed 20 genes (plus three dubious ORFs) whose expression change was more than twofold lower than expected in the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion, based on their behavior in the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression strain. This set included <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FLO10</a></em>, which was strongly repressed by overexpressing <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, but whose expression was relatively unaffected by the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion, relative to the full disome. The remaining 19 genes in this group were all repressed, to a greater or lesser degree, both by the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion (relative to the full chromosome XVI disome) and also by <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression (relative to F45), rather than showing the expected opposite directions of effect (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4, A and B</a>).</p><p id="p-38">The 19 genes repressed both by the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion and by <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression included the phosphate transport genes (<em><a href="http://www.yeastgenome.org/locus/S000004592/overview">PHO84</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000500/overview">PHO89</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001178/overview">SPL2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000695/overview">GIT1</a></em>) identified earlier (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3B</a>), along with another phosphate gene, <em><a href="http://www.yeastgenome.org/locus/S000000297/overview">PHO5</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000874/overview">VTC1</a></em>, encoding a vacuolar transport protein involved in vacuolar polyphosphate accumulation. Similarly, another subset of these genes are induced by low zinc availability (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lyons <em>et al.</em> 2000</a>), including <em><a href="http://www.yeastgenome.org/locus/S000003227/overview">VEL1</a></em>, encoding an uncharacterized cell-surface protein, its paralog <em><a href="http://www.yeastgenome.org/locus/S000005914/overview">YOR387C</a></em>, and the genes encoding the alcohol dehydrogenase <a href="http://www.yeastgenome.org/locus/S000003225/overview">Adh4</a>, the putative GPI-anchored protein <a href="http://www.yeastgenome.org/locus/S000005514/overview">Zps1</a>, and the zinc transporter <a href="http://www.yeastgenome.org/locus/S000003224/overview">Zrt1</a>. Other genes included <em><a href="http://www.yeastgenome.org/locus/S000004184/overview">YLR194C</a></em> , <em><a href="http://www.yeastgenome.org/locus/S000004406/overview">PUN1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000813/overview">TIR1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003911/overview">DAN1</a></em> encoding cell wall proteins, and <em><a href="http://www.yeastgenome.org/locus/S000003808/overview">ANB1</a></em> encoding a translation elongation factor. Finally, three uncharacterized genes, <em><a href="http://www.yeastgenome.org/locus/S000028736/overview">YBR056W-A</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004963/overview">YNL018C</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005519/overview">YOL159C</a></em>, were also repressed by both perturbations. The nonreference gene 59A_0034g, which possess a Flo11-like domain (pfam10182) and was strongly repressed in the two disomes, also behaved like a member of this group.</p><p id="p-39">The behavior of the phosphate-regulated genes (<em><a href="http://www.yeastgenome.org/locus/S000004592/overview">PHO84</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000500/overview">PHO89</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001178/overview">SPL2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000695/overview">GIT1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000297/overview">PHO5</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000874/overview">VTC1</a></em>) is particularly interesting. These genes are repressed both by <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression and also by the chromosome XVI disome with the hemizygous deletion, both relative to F45, <em>i.e.</em>, their expression is repressed by increased copy number of both <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> and one or more other genes on chromosome XVI. However, the full chromosome XVI disome, in which the copy number of both <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> and the other regulatory gene(s) is increased, shows less, rather than more, repression of these genes, relative to F45 than is seen in the disome with the hemizygous <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> deletion. Therefore, the expression of these phosphate genes appears to reflect a nonlinear interaction between <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number and that of one or more other genes on chromosome XVI.</p></div><div id="sec-16" class="subsection"><h3>Characterization of genes commonly differentially expressed in the chromosome XV and XVI disomes</h3><p id="p-40">In our model of the mechanisms by which aneuploidy can affect gene expression (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>), we specified two mechanisms that could produce effects on gene expression unique to each aneuploid and two mechanisms that could produce similar effects on transcription across multiple aneuploids. To assess the relative contribution of shared <em>vs.</em> disome-specific factors in the gene expression patterns of our disomes, we examined the degree of similarity between the expression patterns of the two disomes. After accounting for direct copy number effects, by twofold downward adjustment of expression from the disomic chromosomes, the differential expression profiles of the two disomes, relative to F45, were similar (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), with an R of 0.70 (Pearson test, two-tailed, t = 74.5175, df = 5776, p-value &lt; 1e−15). The same relationship (<em>R</em> = 0.71) was seen when only genes on chromosomes I–XIV were considered (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FigureS1.tiff">Figure S1</a>). More genes were repressed in each disome (ChrXV: 2171, ChrXVI: 1852) than induced (ChrXV: 1307, ChrXVI: 1149), at the multiple hypothesis corrected p &lt; 0.01 level. Given their similarity, it appears that the differential gene expression patterns of the two disomes are dominated by shared, rather than disome-specific, factors.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differential gene expression in the chromosome XV and chromosome XVI disomes, relative to F45. Genes on disomic chromosomes had expression ratios adjusted downward twofold. x = y line shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Differential gene expression in the chromosome XV and chromosome XVI disomes, relative to F45. Genes on disomic chromosomes had expression ratios adjusted downward twofold. &lt;em&gt;x&lt;/em&gt; = &lt;em&gt;y&lt;/em&gt; line shown in gray.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F5.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/1/233/F5.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470029" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Differential gene expression in the chromosome XV and chromosome XVI disomes, relative to F45. Genes on disomic chromosomes had expression ratios adjusted downward twofold. <em>x</em> = <em>y</em> line shown in gray.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">To begin to understand the factors contributing to the shared differential expression profiles of the two disomes, we characterized the genes commonly induced and repressed in those strains. After downward adjustment of expression from chromosomes XV and XVI in the disomes, a total of 210 genes showed a &gt;twofold decrease in expression in both strains relative to F45, statistically significant at the (multiple hypothesis corrected) p &lt; 0.01 level (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/TableS1.xlsx">Table S1</a>). This set was enriched for several GO terms including GO:0006099 “tricarboxylic acid cycle” (p-value = 1.71e−05), GO:0019953 “sexual reproduction” (p-value = 8.63e−04), GO:0006081 “cellular aldehyde metabolic process” (p-value = 7.78e−05), and GO:0030312 “external encapsulating structure” (p-value = 5.62e−07). Several genes involved in phosphate (<em><a href="http://www.yeastgenome.org/locus/S000000695/overview">GIT1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002689/overview">PHM6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005444/overview">PHM7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000839/overview">PHM8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000297/overview">PHO5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002889/overview">PHO8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004592/overview">PHO84</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000500/overview">PHO89</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000001178/overview">SPL2</a></em>) and thiamine (<em><a href="http://www.yeastgenome.org/locus/S000003376/overview">THI4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001836/overview">THI5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004227/overview">THI7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003917/overview">THI11</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005276/overview">THI12</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002403/overview">THI13</a></em>) metabolism were also significantly repressed in both disomes.</p><p id="p-43">The set of genes downregulated in both disomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/TableS1.xlsx">Table S1</a>) also contained several genes with known roles in the development of complex colony phenotypes. These included the <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FLO10</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> flocculin genes, believed to encode effectors of complex colony morphologies (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Lo and Dranginis 1998</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Guo <em>et al.</em> 2000</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Granek and Magwene 2010</a>) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Halme <em>et al.</em> 2004</a>; <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Voordeckers <em>et al.</em> 2012</a>). Similarly, the gene encoding the ammonium permease <a href="http://www.yeastgenome.org/locus/S000005086/overview">Mep2</a>, which regulates pseudohyphal growth in response to ammonium limitation (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lorenz and Heitman 1998a</a>), and the genes encoding the <a href="http://www.yeastgenome.org/locus/S000000287/overview">Tec1</a> and <a href="http://www.yeastgenome.org/locus/S000003481/overview">Mga1</a> transcription factors, known to be positive regulators of <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> gene expression and complex colony phenotypes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gavrias <em>et al.</em> 1996</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lorenz and Heitman 1998b</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rupp <em>et al.</em> 1999</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Borneman <em>et al.</em> 2006</a>), were consistently repressed in the disomes. Expression of <em><a href="http://www.yeastgenome.org/locus/S000002195/overview">BSC1</a></em>, which encodes an uncharacterized protein with a Flo11-like domain (pfam10182), was also consistently reduced. In addition, one of the unmapped, nonreference genes that we identified (59A_0034g), which encodes a protein that also possesses the Flo11-like domain (pfam10182) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FileS2.zip">File S2</a>), was also strongly repressed in the two disomes. As the chromosomes on which this and the other unmapped nonreference genes reside are unknown, no copy number adjustments were made for their expression in the disomes.</p><p id="p-44">Only 25 genes showed at least a twofold increase in expression in both disomic strains relative to F45, statistically significant at the (multiple hypothesis corrected) p &lt; 0.01 level (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/TableS2.xlsx">Table S2</a>). The only significantly enriched GO terms were GO:0005576 “extracellular region” (p-value = 1.41e−02), GO:0031225 “anchored component of membrane” (p-value = 4.18e−02), and GO:0009277 “fungal-type cell wall” (p-value = 1.23e−02). These 25 genes included <em><a href="http://www.yeastgenome.org/locus/S000005515/overview">HPF1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001431/overview">HPF1</a></em>’ (<em><a href="http://www.yeastgenome.org/locus/S000001431/overview">YIL169C</a></em>), encoding cell wall mannoproteins and one of the unmapped, nonreference genes that we identified (EC1118_1O30_0001g), predicted to encode a protein with strong sequence similarity to <a href="http://www.yeastgenome.org/locus/S000005515/overview">Hpf1</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FileS2.zip">File S2</a>). It is interesting that cell surface mannoproteins were found both among the set of genes most highly repressed in the two disomes (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FLO10</a></em>) and among the genes most strongly induced (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/locus/S000005515/overview">HPF1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001431/overview">HPF1</a></em>’)</p></div><div id="sec-17" class="subsection"><h3>Relative strength of karyotype <em>vs.</em> genotype and environment on gene expression</h3><p id="p-45">The strong similarity between the transcriptional profiles of our two disomes raises questions about how interactions between genotype, environment, and ploidy affect gene expression. One possible explanation for the similar expression profiles of our disomes is that increasing the copy number of a whole chromosome specifies a strong transcriptional response that is relatively independent of genotype and environment. If this is true, we would expect each of our disomes to have a similar transcriptional profile in other studies, and the similarity between the two disomes to be maintained in those studies. To test this hypothesis, we compared our data to the differential transcriptional profiles of chromosome XV and chromosome XVI disomes in a previous study that used a different genetic background (W303) and a different environmental condition (batch growth) (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>). Similar to our results (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), the differential expression profiles of the two disomes in that study were also similar (positively correlated with an <em>R</em> of 0.57) (<a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>). However, we observed no correlation between the transcriptional profiles of the chromosome XV disomes (<em>R</em> = 0.011) or between the transcriptional profiles of the chromosome XVI disomes (<em>R</em> = 0.00038) across the two studies. These results suggest that the expression changes that accompany different aneuploid states are highly dependent on genotype and environment, but may be consistently similar to each other within any specific combination of genotype and environment.</p></div><div id="sec-18" class="subsection"><h3>Contribution of colony morphology and cell environment to the differential expression profiles of the disomes</h3><p id="p-46">One of the mechanisms by which aneuploidy could affect gene expression patterns is indirectly through its effects on phenotype (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>). Both of the disomic strains in our study form smooth colonies, while the F45 euploid forms fluffy colonies. The similar transcriptional profiles of the disomes could therefore be a result of their shared colony morphology, perhaps reflecting a transcriptional response to the smooth <em>vs.</em> fluffy cellular environment. To test this idea, we examined gene expression in smooth colonies resulting from deletions of downstream effectors of the fluffy colony morphology. We reasoned that expression changes in these mutants, relative to F45, would give us the cleanest readout of any transcriptional response to the smooth colony “environment.”</p><p id="p-47">To test this hypothesis, we constructed deletion mutants of <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em>, encoding a cell surface flocculin important for the formation of complex colony morphologies (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Lo and Dranginis 1998</a>; <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Granek and Magwene 2010</a>; <a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Voordeckers <em>et al.</em> 2012</a>), and <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">CIS3</a></em>, encoding a mannose-containing glycoprotein and a constituent of the cell wall that we identified in a separate screen for mutations affecting colony morphology (Z. Tan and A. M. Dudley, unpublished results). Both of these deletions caused F45 colonies to lose their structured (fluffy) colony morphology, but retain their irregular boundaries (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>). As before, we performed RNA-seq on RNA isolated from these colonies after 4 d of growth on solid medium (YPD, 2% glucose).</p><p id="p-48">We first examined the effect of the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ. Despite the strong effect on colony morphology, little differential expression was seen in the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ strain, relative to F45. Only 117 genes showed a change in gene expression that was significant at the (multiple hypothesis corrected) p &lt; 0.01 level, and only 10 of these involved more than a twofold change in expression (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/TableS3.xlsx">Table S3</a>). To compare the effect of the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ to the shared expression pattern displayed by the disomes, we first calculated mean gene expression change relative to F45 for each gene across the two disomic strains. Expression from disomic chromosomes was first adjusted downward twofold to control for the direct effects of chromosome copy number. Much stronger gene expression change was seen in this common disome profile (mean absolute log<sub>2</sub>FC = 0.43) than in the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ (mean absolute log<sub>2</sub>FC = 0.22). Similarly, while the common disome and <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ differential expression profiles were statistically significantly correlated (Pearson test, two-tailed, : <em>t</em> = 26.4532, df = 5772, p-value &lt; 1e−15), this correlation was modest (<em>R</em> = 0.33) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>), leaving most of the variance in the common disomic expression profile unexplained. Taken together, these results suggest that the fluffy <em>vs.</em> smooth cell environment does not in itself induce substantial changes in gene expression and that little of the strong, shared differential expression profile of the two disomes can be explained by their shared (smooth) colony morphology.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differential gene expression in (A) cis3&#x394; and (B) flo11&#x394; colonies relative to F45, compared to the mean disome differential gene expression profile. x = y line shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Differential gene expression in (A) &lt;em&gt;cis3&lt;/em&gt;&#x394; and (B) &lt;em&gt;flo11&lt;/em&gt;&#x394; colonies relative to F45, compared to the mean disome differential gene expression profile. &lt;em&gt;x&lt;/em&gt; = &lt;em&gt;y&lt;/em&gt; line shown in gray.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F6.medium.gif" width="350" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/1/233/F6.medium.gif" width="350" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470160" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-49" class="first-child">Differential gene expression in (A) <em>cis3</em>Δ and (B) <em>flo11</em>Δ colonies relative to F45, compared to the mean disome differential gene expression profile. <em>x</em> = <em>y</em> line shown in gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Evidence for a signaling role for Flo11 in colony development</h3><p id="p-50">Given the low level of differential gene expression in the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ and the similar morphologies of the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ colonies, we were surprised to observe a much stronger differential expression pattern in the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ (mean absolute log<sub>2</sub>FC of 0.38 <em>vs.</em> 0.22). A total of 70 genes were significantly (multiple hypothesis corrected p &lt; 0.01) and strongly (&gt; twofold) induced in the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ colonies, relative to F45, while 109 genes were significantly and strongly downregulated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/TableS4.xlsx">Table S4</a>). Comparing the differential expression pattern of the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ to that of the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ revealed a modest (<em>R</em> = 0.42), but significant correlation (Pearson test, two-tailed, : <em>t</em> = 35.2996, df = 5772, p-value &lt; 1e−15) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FigureS2.tiff">Figure S2</a>). This weak correlation, and the much greater degree of differential gene expression seen in the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ strain, strongly suggest that the differential expression profile of the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ strain does not simply reflect a response to the smooth colony environment that is shared with the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ colonies.</p><p id="p-51">Similarly, comparing the differential expression pattern in the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ strain to the mean disome profile also revealed a modest (<em>R</em> = 0.36) but significant correlation (Pearson test, two-tailed, : <em>t</em> = 29.687, df = 5772, p-value &lt; 1e−15). However, most of the genes whose transcription was most strongly changed in the two disomes, relative to F45, were also strongly differentially expressed, and in the same direction, in the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ colonies (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). This included the <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FLO10</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004032/overview">YLR042C</a></em> cell wall genes and the <em><a href="http://www.yeastgenome.org/locus/S000000287/overview">TEC1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003481/overview">MGA1</a></em> regulators of colony morphology. These results demonstrate that, like the disomes, deletion of <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> affects the expression of genes involved in complex colony morphology development (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FLO10</a></em>) and the signaling pathways regulating morphology development (<em>e.g.</em>, the “filamentous growth” MAPK cascade that includes <a href="http://www.yeastgenome.org/locus/S000000287/overview">Tec1</a> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Madhani and Fink 1997</a>). As <a href="http://www.yeastgenome.org/locus/S000001458/overview">Flo11</a> is a downstream, cell surface effector of these pathways, this suggests that feedback mechanisms are operating, so that, in the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ strain, the signaling pathways regulating fluffy colony development themselves appear to be set to specify the smooth state, with downregulation of positive regulators such as <em><a href="http://www.yeastgenome.org/locus/S000000287/overview">TEC1</a></em> and of mechanistic genes such as <em><a href="http://www.yeastgenome.org/locus/S000001810/overview">FLO10</a></em>. These feedback mechanisms might be responding to some signal common to the disomes and the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">flo11</a></em>Δ strain, perhaps even to <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> expression level itself, which is strongly repressed in both disomes (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). Notably, in a subset of <a href="http://www.yeastgenome.org/locus/S000001458/overview">Flo11</a> molecules, the extracellular portion of the protein is shed from the cell surface into the surrounding medium, rather than remaining intact on the cell surface (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Karunanithi <em>et al.</em> 2010</a>). <a href="http://www.yeastgenome.org/locus/S000003246/overview">Msb2</a> and <a href="http://www.yeastgenome.org/locus/S000001791/overview">Hsk1</a>, which (like <a href="http://www.yeastgenome.org/locus/S000001458/overview">Flo11</a>) are cell surface glycoproteins, also shed their extracellular domains in this way and are known to have roles in MAPK signaling (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vadaie <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pitoniak <em>et al.</em> 2009</a>).</p><p id="p-52">Interestingly, the expression of <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">CIS3</a></em> is not significantly affected by the <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> deletion, indicating that high <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">CIS3</a></em> expression, while necessary for fluffy colony formation, can also occur in smooth strains, <em>i.e.</em>, high <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">CIS3</a></em> expression is not sufficient for fluffy colony formation. In the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">CIS3</a></em> deletion strain, <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> expression is significantly, but only weakly, reduced (to 72% of the F45 level). This suggests either that high <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> expression is also necessary, but not sufficient, for fluffy colony formation, or that even a very small reduction in <em><a href="http://www.yeastgenome.org/locus/S000001458/overview">FLO11</a></em> expression level in F45, such as that seen in the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ strain, is sufficient to cause the fluffy-smooth transition.</p></div><div id="sec-20" class="subsection"><h3>Differential expression of ESR genes in the disomes</h3><p id="p-53">In our initial model (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1B</a>), the final mechanism that we proposed to mediate the effects of aneuploidy on gene expression is through nonspecific responses to aneuploidy <em>per se</em>. Previous studies have suggested that aneuploidy in yeast triggers transcriptional effects similar to the environmental stress response (ESR) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gasch <em>et al.</em> 2000</a>) and that this response is not specific to disomy of particular chromosomes, but instead is a generalized response to the noneuploid state (<a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>). This mechanism could, potentially, explain the common differential gene expression pattern seen in our two disomes. To test this hypothesis, we examined expression of the 868 ESR genes in both of our disomic strains. After adjusting expression of genes on disomic chromosomes downward twofold, we observed a significant difference in expression between the set of genes that are downregulated in the ESR and non-ESR genes (ChrXV disome: Mann–Whitney test, two-tailed, W = 2070000, p-value &lt; 1e−15; ChrXVI disome: Mann–Whitney, two-tailed, W = 1943400, p-value &lt; 1e−15). We also observed a significant difference in expression between the set of genes that are upregulated in the ESR <em>vs.</em> non-ESR genes (ChrXV disome: Mann–Whitney test, two-tailed, W = 477460, p-value &lt; 1e−15; ChrXVI disome: Mann–Whitney test, two-tailed, W = 314290, p-value &lt; 1e−15). Surprisingly, however, the direction of effect was opposite to that expected, <em>i.e.</em>, the genes downregulated in the ESR were induced in the disomes and the genes upregulated in the ESR were repressed in the disomes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Therefore, it appears that the ESR is induced in euploid F45, relative to the two disomes, rather than vice versa.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differential expression of environmental stress response (ESR) genes in the chromosome XV and chromosome XVI disomes, relative to F45. Genes induced in ESR in red, genes repressed in blue, other genes in black. Genes on disomic chromosomes had expression ratios adjusted downward twofold. Chr, chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Differential expression of environmental stress response (ESR) genes in the chromosome XV and chromosome XVI disomes, relative to F45. Genes induced in ESR in red, genes repressed in blue, other genes in black. Genes on disomic chromosomes had expression ratios adjusted downward twofold. Chr, chromosome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F7.medium.gif" width="440" height="267"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/1/233/F7.medium.gif" width="440" height="267"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470068" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-54" class="first-child">Differential expression of environmental stress response (ESR) genes in the chromosome XV and chromosome XVI disomes, relative to F45. Genes induced in ESR in red, genes repressed in blue, other genes in black. Genes on disomic chromosomes had expression ratios adjusted downward twofold. Chr, chromosome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">Interestingly, this reversal of effect has previously been observed for a subset of the ESR genes. In the study that identified induction of the ESR response in aneuploid W303 strains grown in batch culture, a second generalized response to aneuploidy was observed when the same strains were grown in phosphate-limited chemostats (<a id="xref-ref-46-7" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>). Interestingly, there was substantial overlap between the ESR and chemostat-response gene lists, but with a reversal of direction of effect. 98/176 of the genes upregulated in the W303 aneuploid chemostat response were genes downregulated in the ESR, while no genes were upregulated in both responses. Similarly, 95/222 of the genes downregulated in the W303 chemostat response were genes upregulated in the ESR, while only two genes were downregulated in both responses. Therefore, it appears that genes can be consistently upregulated across multiple aneuploids in one environmental condition (batch growth) while being consistently downregulated across the same aneuploids in a different environmental condition (chemostats).</p><p id="p-56">In our disomes, we observed significant repression of the 222 genes downregulated among aneuploids in the chemostat study (ChrXV disome: Mann–Whitney test, two-tailed, W = 440240, p-value = 7.2e−08; ChrXVI disome: Mann–Whitney test, two-tailed, W = 292544, p-value &lt; 1e−15) and significant induction of the 176 genes upregulated in that study (ChrXV disome: Mann–Whitney test, two-tailed, W = 597023, p-value = 3.9e−10; ChrXVI disome: Mann–Whitney test, two-tailed, W = 516041, p-value = 0.018). Therefore, it appears that our samples, from aneuploid colonies, may share a transcriptional response to aneuploidy with chemostat-grown W303 strains, but not with the same strains grown in batch culture.</p></div><div id="sec-21" class="subsection"><h3>Colony morphology and the ESR</h3><p id="p-57">The similar effects on ESR gene expression seen in the two disomes could reflect the shared smooth colony morphology of the disomes, compared to the fluffy morphology of F45. To test this possibility, we examined the effect of the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">CIS3</a></em> deletion on expression of the ESR genes, as <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ colonies are also smooth and appear to represent our cleanest readout of the effect of the smooth environmental state. However, in the <em><a href="http://www.yeastgenome.org/locus/S000003694/overview">cis3</a></em>Δ mutant, little change in expression of the ESR genes was observed relative to F45 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1/FigureS3.tiff">Figure S3</a>), suggesting that colony morphology is not substantially driving the effect of the disomes on ESR gene expression.</p></div><div id="sec-22" class="subsection"><h3>Role of DIG1 in differential expression of ESR genes in the chromosome XVI disome</h3><p id="p-58">To further explore the mechanisms underlying the effect of our two disomes on ESR gene expression, we examined the importance of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number to the effect of the chromosome XVI disome. We observed that overexpression of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> from a plasmid led to induction of the ESR-repressed genes, and to a level similar to that seen in the full chromosome XVI disome. However, the ESR-induced genes, which are repressed in the disome, show little repression after <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> overexpression (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Interestingly, the reciprocal pattern was seen with the hemizygous deletion of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, where there are two copies of chromosome XVI but only one copy of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>. In this strain, relative to F45, ESR-induced genes were repressed, as in the chromosome XVI disome; however, in contrast to the full disome, ESR-repressed genes showed little induction (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). Therefore, it appears that the effect of the chromosome XVI disome on differential expression of the ESR genes can largely be split into two separate copy number mechanisms, with induction of the ESR-repressed genes caused by increasing the copy number of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em>, and repression of the ESR-induced genes caused by increasing the copy number of a gene or genes found elsewhere on chromosome XVI.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/233/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differential expression, relative to F45, of environmental stress response (ESR) genes in the chromosome XVI disome, the disome hemizygous for DIG1 (DIG1/dig1&#x394;), and in the DIG1 overexpression strain. Genes induced in ESR in red, genes repressed in blue. Genes on disomic chromosomes had expression ratios adjusted downward twofold. Median values shown by black bars. Chr, chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469995916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Differential expression, relative to F45, of environmental stress response (ESR) genes in the chromosome XVI disome, the disome hemizygous for &lt;em&gt;DIG1&lt;/em&gt; (&lt;em&gt;DIG1/dig1&lt;/em&gt;&#x394;), and in the &lt;em&gt;DIG1&lt;/em&gt; overexpression strain. Genes induced in ESR in red, genes repressed in blue. Genes on disomic chromosomes had expression ratios adjusted downward twofold. Median values shown by black bars. Chr, chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/233/F8.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/1/233/F8.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/233/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/233/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470107" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-59" class="first-child">Differential expression, relative to F45, of environmental stress response (ESR) genes in the chromosome XVI disome, the disome hemizygous for <em>DIG1</em> (<em>DIG1/dig1</em>Δ), and in the <em>DIG1</em> overexpression strain. Genes induced in ESR in red, genes repressed in blue. Genes on disomic chromosomes had expression ratios adjusted downward twofold. Median values shown by black bars. Chr, chromosome.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><p id="p-60">Aneuploidy, such as that underlying Down syndrome or found in most tumor cells, perturbs the expression of large numbers of genes simultaneously. In this study, we used a yeast model to dissect distinct components contributing to the impact of two different aneuploidies on gene expression. Our analysis was able to isolate effects on transcription that were shared between the two aneuploidies and effects unique to each. Additionally, we identified nonlinear interactions between the copy number of a transcription factor and the copy number of the other genes on the same chromosome, an interaction affecting the expression of multiple genes.</p><p id="p-61">One of the factors we expected to be unique to each disome was the direct effect of increased copy number on expression of genes from the affected chromosome. As expected, expression of genes on the disomic chromosomes was elevated relative to the rest of the genome. Previous work has indicated that, in yeast, average gene expression levels from a chromosome correlate closely with chromosome copy number, when that copy number is perturbed by aneuploidy (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hughes <em>et al.</em> 2000</a>; <a id="xref-ref-46-8" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>, <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">2016</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Rancati <em>et al.</em> 2008</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dephoure <em>et al.</em> 2014</a>). However, a recent study argued that widespread dosage compensation occurs in some yeast strains, so that certain genes, particularly those that might be harmful if overexpressed, show lower than expected increases in expression when their copy number increases through aneuploidy (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hose <em>et al.</em> 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gasch <em>et al.</em> 2016</a>). However, in that same study, the expression levels of a similar number of genes increased more than their copy number. An alternative explanation for these results is that when the copy number of a chromosome increases, expression of genes on that chromosome is increased proportionally; however, some genes may be additionally affected (up- or downregulated) by other effects of the aneuploidy (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Torres <em>et al.</em> 2016</a>), such as changes in the copy number of genes encoding transcriptional regulators (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Rancati <em>et al.</em> 2008</a>).</p><p id="p-62">In our study, we observed median increases in gene expression levels from the disomic chromosomes, relative to F45, somewhat lower than the expected twofold. However, when expression was compared between the two disomes, median expression changes closer to twofold were observed and the SD of the log ratios was reduced. This is consistent with expression of genes on the disomic chromosomes being affected by two phenomena. First, the effect of doubling gene copy number appears to increase gene expression twofold. Second, genes on chromosomes XV and XVI, just as genes elsewhere on the genome (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>), show similar changes in gene expression in both disomes and this effect is independent of chromosome copy number. When this second effect is taken into account, by comparing gene expression between the disomes, the direct effect of copy number is seen more clearly.</p><p id="p-63">We also expected to find additional transcriptional effects specific to each disome, caused by perturbing the copy number of transcriptional regulatory genes present on each of the two disomic chromosomes. To explore the role of regulatory gene copy number in gene expression changes caused by aneuploidy, we varied the copy number of the <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> transcriptional regulator in the context of an F45 background that was euploid or disomic for chromosome XVI (on which <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> is located). A significant correlation was seen between the perturbations in the two backgrounds with magnitudes that were consistent with the relative changes in <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> expression level. However, the strength of the correlation was modest, suggesting that the effect of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number changes might be highly dependent on the copy number of genes on the rest of chromosome XVI or that the effects of changes in <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number are nonlinear. We specifically identified several genes whose expression was consistent with a nonlinear interaction between <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number and the copy number of chromosome XVI. This included several genes in the phosphate regulon, suggesting interactions between <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number and the copy number of a regulatory gene or genes elsewhere on chromosome XVI specifically involved in phosphate gene expression.</p><p id="p-64">In addition to the transcriptional effects unique to each disome, we also expected to see effects in common between the disomes, and in fact the differential transcriptional profiles of our two disomes were similar. One potential source for these common transcriptional effects is the ESR, in which the expression of a large number of transcripts changes in response to various stress conditions (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gasch <em>et al.</em> 2000</a>). A previous publication identified induction of the ESR in multiple, distinct aneuploid derivatives of the laboratory yeast strain W303 grown in liquid batch culture (<a id="xref-ref-46-9" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>). Induction of this response appears to reflect the relatively slow growth rates of these strains, relative to the euploid control. Previous studies in yeast have shown that slow growth can induce the ESR (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Regenberg <em>et al.</em> 2006</a>), and analogous stress responses and impaired proliferation have also been seen in response to aneuploidy in plant and mammalian cells (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Sheltzer <em>et al.</em> 2012</a>). As such, a growth impairment leading to the induction of a stress response might be regarded as a conserved nonspecific response to aneuploidy.</p><p id="p-65">In contrast to the results seen in the W303 laboratory strain background, studies using a number of aneuploid wild strains of yeast grown in batch culture failed to identify induction of the ESR (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Hose <em>et al.</em> 2015</a>). Notably, these strains also did not appear to suffer significant growth impairment associated with their aneuploidy. Similarly, when the W303 strains were grown in phosphate-limited chemostats, eliminating the growth disadvantage of the aneuploid strains, induction of the ESR was not observed (<a id="xref-ref-46-10" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>). However, in a study assaying plate growth of S288c-derived aneuploid yeast strains (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Pavelka <em>et al.</em> 2010</a>), differential expression of ESR genes was also not consistently observed, including among aneuploid strains with strong growth defects. Taken together, and as argued by <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Pavelka <em>et al.</em> (2010)</a>, it appears that induction of the ESR is not an obligate response to aneuploidy. Instead, the induction of the ESR by aneuploidy in some studies is likely to be a consequence of specific combinations of genotype and environment.</p><p id="p-66">In our study, using colonies grown on solid medium, we did observe differential expression of the ESR genes between the euploid and aneuploid strains. Surprisingly, however, the direction of effect was reversed from that expected, with genes upregulated in the ESR having lower expression in our aneuploid strains and genes downregulated in the ESR having higher expression. This effect (the reversal of the ESR) is not easily explained by improved growth of the aneuploids, as we have previously shown that the chromosome XVI disome is at a growth disadvantage in plate growth, compared to euploid F45 (<a id="xref-ref-44-16" class="xref-bibr" href="#ref-44">Tan <em>et al.</em> 2013</a>). This implies that, in our genetic background, aneuploidy, the colony morphology state, or a combination of both might relieve a stress condition experienced by the F45 euploid strain and possibly overwhelm any opposing impact on the ESR of the growth defect associated with the disomy. Interestingly, while growth of aneuploid W303 strains in phosphate-limited chemostats did not induce the ESR, a separate transcriptional response was seen across multiple aneuploids (<a id="xref-ref-46-11" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>). Fifty percent of the genes in this response were ESR genes but, as was the case in our study, the direction of effect was reversed. Finally, our results suggest that the effect of the chromosome XVI disome on the ESR response can be split into ESR-repressed genes, which appear to respond to the effect of <em><a href="http://www.yeastgenome.org/locus/S000005970/overview">DIG1</a></em> copy number, and ESR-induced genes, which appear to respond to the copy number of a gene or genes elsewhere on chromosome XVI. This observation argues against the ESR-response in our strains as a true generalized response to the euploid/aneuploid switch.</p><p id="p-67">Finally, comparison of our results with those of a previous study demonstrated that the expression changes that accompany specific aneuploid states are highly dependent on genotype and environment. In turn, this has implications for the relationship between aneuploidy and cancer as it suggests that the effect of any given aneuploidy might be highly genotype- and tissue type-specific. Therefore, as a prognostic tool or for driving cancer treatment decisions, knowledge of ploidy may be relatively uninformative when viewed in isolation from those other factors.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">We thank Maitreya Dunham for helpful discussions. This work was funded by a grant from the National Institutes of Health (P50 GM076547) and a strategic partnership between the Institute for Systems Biology and the University of Luxembourg. Z.T. was funded by the Agency for Science, Technology and Research, Singapore.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-69">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036160/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-70"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-08-31"><span class="received-label">Received </span>August 31, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-08"><span class="accepted-label">Accepted </span>November 8, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Cromie <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.1" data-doi="10.1128/EC.2.4.729-736.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almaguer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mantella</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patton-Vogt</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inositol and phosphate regulate <em>GIT1</em> transcription and glycerophosphoinositol incorporation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DAlmaguer%26rft.auinit1%253DC.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D729%26rft.epage%253D736%26rft.atitle%253DInositol%2Band%2BPhosphate%2BRegulate%2BGIT1%2BTranscription%2Band%2BGlycerophosphoinositol%2BIncorporation%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.2.4.729-736.2003%26rft_id%253Dinfo%253Apmid%252F12912892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIyLzQvNzI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.2" data-doi="10.1101/gad.1389306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leigh-Bell</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertone</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Target hub proteins serve as master regulators of development in yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1389306%26rft_id%253Dinfo%253Apmid%252F16449570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjAvNC80MzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.3" data-doi="10.1128/MCB.02053-05"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Regulation of mating and filamentation genes by two distinct Ste12 complexes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4794</span>–<span class="cit-lpage">4805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DChou%26rft.auinit1%253DS.%26rft.volume%253D26%26rft.issue%253D13%26rft.spage%253D4794%26rft.epage%253D4805%26rft.atitle%253DRegulation%2Bof%2BMating%2Band%2BFilamentation%2BGenes%2Bby%2BTwo%2BDistinct%2BSte12%2BComplexes%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.02053-05%26rft_id%253Dinfo%253Apmid%252F16782869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMy80Nzk0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.4" data-doi="10.1128/EC.3.6.1464-1475.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coluccio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogengruber</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dresser</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briza</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Morphogenetic pathway of spore wall assembly in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1464</span>–<span class="cit-lpage">1475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DColuccio%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D1464%26rft.epage%253D1475%26rft.atitle%253DMorphogenetic%2BPathway%2Bof%2BSpore%2BWall%2BAssembly%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.3.6.1464-1475.2004%26rft_id%253Dinfo%253Apmid%252F15590821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIzLzYvMTQ2NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.5" data-doi="10.1101/gad.10.22.2831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bardwell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kron</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Two novel targets of the MAP kinase Kss1 are negative regulators of invasive growth in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2831</span>–<span class="cit-lpage">2848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DCook%26rft.auinit1%253DJ%2BG%26rft.volume%253D10%26rft.issue%253D22%26rft.spage%253D2831%26rft.epage%253D2848%26rft.atitle%253DTwo%2Bnovel%2Btargets%2Bof%2Bthe%2BMAP%2Bkinase%2BKss1%2Bare%2Bnegative%2Bregulators%2Bof%2Binvasive%2Bgrowth%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.22.2831%26rft_id%253Dinfo%253Apmid%252F8918885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEwLzIyLzI4MzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.6" data-doi="10.7554/eLife.03023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dephoure</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative proteomic analysis reveals posttranslational responses to aneuploidy in yeast.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e03023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.03023%26rft_id%253Dinfo%253Apmid%252F25073701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjMvMC9lMDMwMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.7" data-doi="10.1534/genetics.115.185660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chromosome-specific and global effects of aneuploidy in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1395</span>–<span class="cit-lpage">1409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185660%26rft_id%253Dinfo%253Apmid%252F26837754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi80LzEzOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.8" data-doi="10.1091/mbc.11.12.4241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Harel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genomic expression programs in the response of yeast cells to environmental changes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">4241</span>–<span class="cit-lpage">4257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D4241%26rft.epage%253D4257%26rft.atitle%253DGenomic%2BExpression%2BPrograms%2Bin%2Bthe%2BResponse%2Bof%2BYeast%2BCells%2Bto%2BEnvironmental%2BChanges%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.12.4241%26rft_id%253Dinfo%253Apmid%252F11102521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMi80MjQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hose</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newton</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sardi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yong</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Further support for aneuploidy tolerance in wild yeast and effects of dosage compensation on gene copy-number evolution.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e14409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De14409%26rft_id%253Dinfo%253Apmid%252F26949252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26949252&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.10" data-doi="10.1111/j.1365-2958.1996.tb02470.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gavrias</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrianopoulos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gimeno</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timberlake</span>  <span class="cit-name-given-names">W. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> <em>TEC1</em> is required for pseudohyphal growth.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DGavrias%26rft.auinit1%253DV.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D1255%26rft.epage%253D1263%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BTEC1%2Bis%2Brequired%2Bfor%2Bpseudohyphal%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.1996.tb02470.x%26rft_id%253Dinfo%253Apmid%252F8730867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.1996.tb02470.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8730867&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UE08600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.11" data-doi="10.1038/nrg3123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resio</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellman</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Causes and consequences of aneuploidy in cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGordon%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D189%26rft.epage%253D203%26rft.atitle%253DCauses%2Band%2Bconsequences%2Bof%2Baneuploidy%2Bin%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3123%26rft_id%253Dinfo%253Apmid%252F22269907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22269907&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.12" data-doi="10.1371/journal.pgen.1000823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Environmental and genetic determinants of colony morphology in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGranek%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1000823%26rft.epage%253De1000823%26rft.atitle%253DEnvironmental%2Band%2Bgenetic%2Bdeterminants%2Bof%2Bcolony%2Bmorphology%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000823%26rft_id%253Dinfo%253Apmid%252F20107600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20107600&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.13" data-doi="10.1073/pnas.220420397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A <em>Saccharomyces</em> gene family involved in invasive growth, cell-cell adhesion, and mating.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12158</span>–<span class="cit-lpage">12163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.220420397%26rft_id%253Dinfo%253Apmid%252F11027318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjIvMTIxNTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.14" data-doi="10.1016/S0092-8674(04)00118-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halme</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bumgarner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic and epigenetic regulation of the <em>FLO</em> gene family generates cell-surface variation in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHalme%26rft.auinit1%253DA.%26rft.volume%253D116%26rft.issue%253D3%26rft.spage%253D405%26rft.epage%253D415%26rft.atitle%253DGenetic%2Band%2Bepigenetic%2Bregulation%2Bof%2Bthe%2BFLO%2Bgene%2Bfamily%2Bgenerates%2Bcell-surface%2Bvariation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900118-7%26rft_id%253Dinfo%253Apmid%252F15016375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00118-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15016375&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188825800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hose</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yong</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sardi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newton</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dosage compensation can buffer copy-number variation in wild yeast.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: e05462.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.16" data-doi="10.1038/77116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Widespread aneuploidy revealed by DNA microarray expression profiling.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D333%26rft.epage%253D337%26rft.atitle%253DWidespread%2Baneuploidy%2Brevealed%2Bby%2BDNA%2Bmicroarray%2Bexpression%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F77116%26rft_id%253Dinfo%253Apmid%252F10888885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/77116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10888885&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087920900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.17" data-doi="10.1016/j.cub.2010.06.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karunanithi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vadaie</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavel</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkaya</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Shedding of the mucin-like flocculin Flo11p reveals a new aspect of fungal adhesion regulation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1389</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DKarunanithi%26rft.auinit1%253DS.%26rft.volume%253D20%26rft.issue%253D15%26rft.spage%253D1389%26rft.epage%253D1395%26rft.atitle%253DShedding%2Bof%2Bthe%2Bmucin-like%2Bflocculin%2BFlo11p%2Breveals%2Ba%2Bnew%2Baspect%2Bof%2Bfungal%2Badhesion%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.06.033%26rft_id%253Dinfo%253Apmid%252F20619652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.06.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20619652&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.18" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.19" data-doi="10.1073/pnas.1016459108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lardenois</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalmel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evrard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Execution of the meiotic noncoding RNA expression program and the onset of gametogenesis in yeast require the conserved exosome subunit Rrp6.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1058</span>–<span class="cit-lpage">1063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1016459108%26rft_id%253Dinfo%253Apmid%252F21149693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzMvMTA1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.20" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.21" data-doi="10.1093/bioinformatics/btt656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">FeatureCounts: an efficient general purpose program for assigning sequence reads to genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt656%26rft_id%253Dinfo%253Apmid%252F24227677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIzMC83LzkyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.22" data-doi="10.1534/genetics.111.133306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daignan-Fornier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Regulation of amino acid, nucleotide, and phosphate metabolism in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLjungdahl%26rft.auinit1%253DP.%2BO.%26rft.volume%253D190%26rft.issue%253D3%26rft.spage%253D885%26rft.epage%253D929%26rft.atitle%253DRegulation%2Bof%2BAmino%2BAcid%252C%2BNucleotide%252C%2Band%2BPhosphate%2BMetabolism%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.133306%26rft_id%253Dinfo%253Apmid%252F22419079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzMvODg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.23" data-doi="10.1091/mbc.9.1.161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dranginis</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The cell surface flocculin Flo11 is required for pseudohyphae formation and invasion by <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DLo%26rft.auinit1%253DW.-S.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D161%26rft.epage%253D171%26rft.atitle%253DThe%2BCell%2BSurface%2BFlocculin%2BFlo11%2BIs%2BRequired%2Bfor%2BPseudohyphae%2BFormation%2Band%2BInvasion%2Bby%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.9.1.161%26rft_id%253Dinfo%253Apmid%252F9436998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjkvMS8xNjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.24" data-doi="10.1093/emboj/17.5.1236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">a</span> <span class="cit-article-title">The MEP2 ammonium permease regulates pseudohyphal differentiation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DLorenz%26rft.auinit1%253DM.%2BC.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D1236%26rft.epage%253D1247%26rft.atitle%253DThe%2BMEP2%2Bammonium%2Bpermease%2Bregulates%2Bpseudohyphal%2Bdifferentiation%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.5.1236%26rft_id%253Dinfo%253Apmid%252F9482721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxNy81LzEyMzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">b</span> <span class="cit-article-title">Regulators of pseudohyphal differentiation in <em>Saccharomyces cerevisiae</em> identified through multicopy suppressor analysis in ammonium permease mutant strains.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1443</span>–<span class="cit-lpage">1457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLorenz%26rft.auinit1%253DM.%2BC.%26rft.volume%253D150%26rft.issue%253D4%26rft.spage%253D1443%26rft.epage%253D1457%26rft.atitle%253DRegulators%2Bof%2BPseudohyphal%2BDifferentiation%2Bin%2BSaccharomyces%2Bcerevisiae%2BIdentified%2BThrough%2BMulticopy%2BSuppressor%2BAnalysis%2Bin%2BAmmonium%2BPermease%2BMutant%2BStrains%26rft_id%253Dinfo%253Apmid%252F9832522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1MC80LzE0NDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.26" data-doi="10.1073/pnas.97.14.7957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaither</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome-wide characterization of the Zap1p zinc-responsive regulon in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7957</span>–<span class="cit-lpage">7962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLyons%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D97%26rft.issue%253D14%26rft.spage%253D7957%26rft.epage%253D7962%26rft.atitle%253DGenome-wide%2Bcharacterization%2Bof%2Bthe%2BZap1p%2Bzinc-responsive%2Bregulon%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.14.7957%26rft_id%253Dinfo%253Apmid%252F10884426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTQvNzk1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.27" data-doi="10.1126/science.275.5304.1314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madhani</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Combinatorial control required for the specificity of yeast MAPK signaling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">1314</span>–<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMadhani%26rft.auinit1%253DH.%2BD.%26rft.volume%253D275%26rft.issue%253D5304%26rft.spage%253D1314%26rft.epage%253D1317%26rft.atitle%253DCombinatorial%2BControl%2BRequired%2Bfor%2Bthe%2BSpecificity%2Bof%2BYeast%2BMAPK%2BSignaling%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.275.5304.1314%26rft_id%253Dinfo%253Apmid%252F9036858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzUvNTMwNC8xMzE0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.28" data-doi="10.1016/S0888-7543(03)00035-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zielke</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zielke</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pevsner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global up-regulation of chromosome 21 gene expression in the developing Down syndrome brain.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DMao%26rft.auinit1%253DR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D457%26rft.epage%253D467%26rft.atitle%253DGlobal%2Bup-regulation%2Bof%2Bchromosome%2B21%2Bgene%2Bexpression%2Bin%2Bthe%2Bdeveloping%2BDown%2Bsyndrome%2Bbrain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0888-7543%252803%252900035-1%26rft_id%253Dinfo%253Apmid%252F12706104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0888-7543(03)00035-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12706104&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182464800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads.</span> <abbr class="cit-jnl-abbrev">EMBnet J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet%2BJ.%26rft.volume%253D17%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.30" data-doi="10.1101/gad.207407.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oromendia</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Aneuploidy causes proteotoxic stress in yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2696</span>–<span class="cit-lpage">2708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.207407.112%26rft_id%253Dinfo%253Apmid%252F23222101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI2LzI0LzI2OTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.31" data-doi="10.1038/nature09529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saraf</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPavelka%26rft.auinit1%253DN.%26rft.volume%253D468%26rft.issue%253D7321%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DAneuploidy%2Bconfers%2Bquantitative%2Bproteome%2Bchanges%2Band%2Bphenotypic%2Bvariation%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09529%26rft_id%253Dinfo%253Apmid%252F20962780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20962780&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284051000054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.233.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source"><em>Practical Iterative de Bruijn Graph De Novo Assembler</em>.</span> <span class="cit-publ-name">RECOMB</span>, <span class="cit-publ-loc">Lisbon</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.33" data-doi="10.1091/mbc.E08-07-0760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pitoniak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkaya</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dionne</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vadaie</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullen</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The signaling mucins Msb2 and Hkr1 differentially regulate the filamentation mitogen-activated protein kinase pathway and contribute to a multimodal response.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3101</span>–<span class="cit-lpage">3114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-07-0760%26rft_id%253Dinfo%253Apmid%252F19439450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMy8zMTAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.34" data-doi="10.1016/j.cell.2008.09.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleharty</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trimble</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Aneuploidy underlies rapid adaptive evolution of yeast cells deprived of a conserved cytokinesis motor.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRancati%26rft.auinit1%253DG.%26rft.volume%253D135%26rft.issue%253D5%26rft.spage%253D879%26rft.epage%253D893%26rft.atitle%253DAneuploidy%2Bunderlies%2Brapid%2Badaptive%2Bevolution%2Bof%2Byeast%2Bcells%2Bdeprived%2Bof%2Ba%2Bconserved%2Bcytokinesis%2Bmotor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.09.039%26rft_id%253Dinfo%253Apmid%252F19041751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.09.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19041751&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261183900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.233.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite> <span class="cit-pub-date">2014</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em>.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.36" data-doi="10.1186/gb-2006-7-11-r107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Regenberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grotkjaer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winther</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fausboll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akesson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Growth-rate regulated genes have profound impact on interpretation of transcriptome profiling in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRegenberg%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253DR107%26rft.epage%253DR107%26rft.atitle%253DGrowth-rate%2Bregulated%2Bgenes%2Bhave%2Bprofound%2Bimpact%2Bon%2Binterpretation%2Bof%2Btranscriptome%2Bprofiling%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-11-r107%26rft_id%253Dinfo%253Apmid%252F17105650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-11-r107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17105650&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.37" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi8xLzEzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-7.1.233.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Methods in Yeast Genetics: A Laboratory Course Manual</em>.</span> <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.39" data-doi="10.1093/emboj/18.5.1257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rupp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madhani</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">MAP kinase and cAMP filamentation signaling pathways converge on the unusually large promoter of the yeast <em>FLO11</em> gene.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1257</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DRupp%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D1257%26rft.epage%253D1269%26rft.atitle%253DMAP%2Bkinase%2Band%2BcAMP%2Bfilamentation%2Bsignaling%2Bpathways%2Bconverge%2Bon%2Bthe%2Bunusually%2Blarge%2Bpromoter%2Bof%2Bthe%2Byeast%2BFLO11%2Bgene%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F18.5.1257%26rft_id%253Dinfo%253Apmid%252F10064592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxOC81LzEyNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.40" data-doi="10.1016/j.neuron.2006.05.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salehi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcroix</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belichenko</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Increased App expression in a mouse model of Down’s syndrome disrupts NGF transport and causes cholinergic neuron degeneration.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DSalehi%26rft.auinit1%253DA.%26rft.volume%253D51%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D42%26rft.atitle%253DIncreased%2BApp%2Bexpression%2Bin%2Ba%2Bmouse%2Bmodel%2Bof%2BDown%2527s%2Bsyndrome%2Bdisrupts%2BNGF%2Btransport%2Band%2Bcauses%2Bcholinergic%2Bneuron%2Bdegeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2006.05.022%26rft_id%253Dinfo%253Apmid%252F16815330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2006.05.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16815330&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239037700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.41" data-doi="10.1016/j.tig.2011.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheltzer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The aneuploidy paradox: costs and benefits of an incorrect karyotype.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DSheltzer%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D27%26rft.issue%253D11%26rft.spage%253D446%26rft.epage%253D453%26rft.atitle%253DThe%2Baneuploidy%2Bparadox%253A%2Bcosts%2Band%2Bbenefits%2Bof%2Ban%2Bincorrect%2Bkaryotype.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.07.003%26rft_id%253Dinfo%253Apmid%252F21872963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21872963&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296485800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.42" data-doi="10.1073/pnas.1209227109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheltzer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptional consequences of aneuploidy.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">12644</span>–<span class="cit-lpage">12649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1209227109%26rft_id%253Dinfo%253Apmid%252F22802626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMxLzEyNjQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.43" data-doi="10.1128/MCB.11.6.3009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">SPT5, an essential gene important for normal transcription in Saccharomyces cerevisiae, encodes an acidic nuclear protein with a carboxy-terminal repeat.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">3009</span>–<span class="cit-lpage">3019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSwanson%26rft.auinit1%253DM%2BS%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D3009%26rft.epage%253D3019%26rft.atitle%253DSPT5%252C%2Ban%2Bessential%2Bgene%2Bimportant%2Bfor%2Bnormal%2Btranscription%2Bin%2BSaccharomyces%2Bcerevisiae%252C%2Bencodes%2Ban%2Bacidic%2Bnuclear%2Bprotein%2Bwith%2Ba%2Bcarboxy-terminal%2Brepeat.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.6.3009%26rft_id%253Dinfo%253Apmid%252F1840633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzYvMzAwOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.44" data-doi="10.1073/pnas.1301047110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aneuploidy underlies a multicellular phenotypic switch.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">12367</span>–<span class="cit-lpage">12372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1301047110%26rft_id%253Dinfo%253Apmid%252F23812752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzMwLzEyMzY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.1.233.45" data-doi="10.2144/000112040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taxis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">System of centromeric, episomal, and integrative vectors based on drug resistance markers for <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DTaxis%26rft.auinit1%253DC.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D73%26rft.epage%253D78%26rft.atitle%253DSystem%2Bof%2Bcentromeric%252C%2Bepisomal%252C%2Band%2Bintegrative%2Bvectors%2Bbased%2Bon%2Bdrug%2Bresistance%2Bmarkers%2Bfor%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112040%26rft_id%253Dinfo%253Apmid%252F16454043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16454043&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234806900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.46" data-doi="10.1126/science.1142210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolsky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boselli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Effects of aneuploidy on cellular physiology and cell division in haploid yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTorres%26rft.auinit1%253DE.%2BM.%26rft.volume%253D317%26rft.issue%253D5840%26rft.spage%253D916%26rft.epage%253D924%26rft.atitle%253DEffects%2Bof%2BAneuploidy%2Bon%2BCellular%2BPhysiology%2Band%2BCell%2BDivision%2Bin%2BHaploid%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142210%26rft_id%253Dinfo%253Apmid%252F17702937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTg0MC85MTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.47" data-doi="10.1016/j.cell.2010.08.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dephoure</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panneerselvam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of aneuploidy-tolerating mutations.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTorres%26rft.auinit1%253DE.%2BM.%26rft.volume%253D143%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D83%26rft.atitle%253DIdentification%2Bof%2Baneuploidy-tolerating%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.08.038%26rft_id%253Dinfo%253Apmid%252F20850176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.08.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20850176&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282362000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">No current evidence for widespread dosage compensation in <em>S. cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e10996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De10996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.49" data-doi="10.1083/jcb.200704079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vadaie</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dionne</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akajagbor</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickerson</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krysan</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cleavage of the signaling mucin Msb2 by the aspartyl protease Yps1 is required for MAPK activation in yeast.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200704079%26rft_id%253Dinfo%253Apmid%252F18591427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxODEvNy8xMDczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.50" data-doi="10.1038/nature10118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Dijk</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d’Aubenton-Carafa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gourvennec</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwapisz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">XUTs are a class of Xrn1-sensitive antisense regulatory non-coding RNA in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253Dvan%2BDijk%26rft.auinit1%253DE.%2BL.%26rft.volume%253D475%26rft.issue%253D7354%26rft.spage%253D114%26rft.epage%253D117%26rft.atitle%253DXUTs%2Bare%2Ba%2Bclass%2Bof%2BXrn1-sensitive%2Bantisense%2Bregulatory%2Bnon-coding%2BRNA%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10118%26rft_id%253Dinfo%253Apmid%252F21697827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21697827&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292461300056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.51" data-doi="10.1111/j.1365-2958.2012.08192.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voordeckers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Maeyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Zande</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinces</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meert</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of a complex genetic network underlying <em>Saccharomyces cerevisiae</em> colony morphology.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DVoordeckers%26rft.auinit1%253DK.%26rft.volume%253D86%26rft.issue%253D1%26rft.spage%253D225%26rft.epage%253D239%26rft.atitle%253DIdentification%2Bof%2Ba%2Bcomplex%2Bgenetic%2Bnetwork%2Bunderlying%2BSaccharomyces%2Bcerevisiae%2Bcolony%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08192.x%26rft_id%253Dinfo%253Apmid%252F22882838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08192.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22882838&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.52" data-doi="10.1038/nature07728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagneur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perocchi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clauder-Munster</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bidirectional promoters generate pervasive transcription in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXu%26rft.auinit1%253DZ.%26rft.volume%253D457%26rft.issue%253D7232%26rft.spage%253D1033%26rft.epage%253D1037%26rft.atitle%253DBidirectional%2Bpromoters%2Bgenerate%2Bpervasive%2Btranscription%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07728%26rft_id%253Dinfo%253Apmid%252F19169243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19169243&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F233.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263425400047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.233.53" data-doi="10.1073/pnas.1211150109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manor</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbst</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosomal duplication is a transient evolutionary solution to stress.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">21010</span>–<span class="cit-lpage">21015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211150109%26rft_id%253Dinfo%253Apmid%252F23197825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzUxLzIxMDEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/233.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/221" title="Time-Course Analysis of Gene Expression During the Saccharomyces cerevisiae Hypoxic Response" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/247" title="Multiplex shRNA Screening of Germ Cell Development by in Vivo Transfection of Mouse Testis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/233.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G0n_0WqylR5JWcodzbZ6WsZjFfP46WZMS2BpuiQQMys" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/233" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/233" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/233" class="active" data-icon-position="" data-hide-link-title="0">Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460539" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Z0nWSYAZZyX3-OoLVEq8ok409BiUsMjKmk7a-AbJgO0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/233.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/233" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Q97aDkuR98ZoJHCjAB8waOpif1oWakmkXg8DQlzdRqA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/233" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/233.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460539" id="node460539--2" data-pisa="ggg;7/1/233" data-pisa-master="ggg;g3.116.036160" data-apath="/ggg/7/1/233.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gareth A. Cromie</span>, <span class="highwire-citation-author" data-delta="1">Zhihao Tan</span>, <span class="highwire-citation-author" data-delta="2">Michelle Hays</span>, <span class="highwire-citation-author" data-delta="3">Eric W. Jeffery</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">233-246;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036160 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460539--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhihao Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Medical Biology, Agency for Science, Technology and Research, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhihao%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhihao%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michelle Hays </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BHays%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hays%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BHays%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eric W. Jeffery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BW.%2BJeffery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jeffery%20EW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BW.%2BJeffery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/233" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/233&amp;title=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/233&amp;title=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/233&amp;title=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/233&amp;text=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/233&amp;title=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/233&amp;t=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/233&amp;title=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/233&amp;title=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460539/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460539" id="node460539--3" data-pisa="ggg;7/1/233" data-pisa-master="ggg;g3.116.036160" data-apath="/ggg/7/1/233.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dissecting Gene Expression Changes Accompanying a Ploidy-Based Phenotypic Switch</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gareth A. Cromie</span>, <span class="highwire-citation-author" data-delta="1">Zhihao Tan</span>, <span class="highwire-citation-author" data-delta="2">Michelle Hays</span>, <span class="highwire-citation-author" data-delta="3">Eric W. Jeffery</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">233-246;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036160 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460539--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhihao Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Medical Biology, Agency for Science, Technology and Research, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhihao%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhihao%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michelle Hays </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BHays%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hays%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BHays%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eric W. Jeffery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BW.%2BJeffery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jeffery%20EW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BW.%2BJeffery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=233&amp;rft.epage=246&amp;rft.atitle=Dissecting%20Gene%20Expression%20Changes%20Accompanying%20a%20Ploidy-Based%20Phenotypic%20Switch&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cromie%2C+Gareth+A.&amp;rft.au=Tan%2C+Zhihao&amp;rft.au=Hays%2C+Michelle&amp;rft.au=Jeffery%2C+Eric+W.&amp;rft.au=Dudley%2C+Aim%C3%A9e+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460539/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460539/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460539/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460539/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460539/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460539/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460539/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460539/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460539/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460539/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460539/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460539/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/233" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/233.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/233.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/1/233.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/1/233.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
