<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/203" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/203.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/203.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/1/203.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/203.atom" />
<meta name="HW.pisa" content="ggg;7/1/203" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation" />
<meta name="DC.Identifier" content="10.1534/g3.116.036020" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Zhao et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gene expression is controlled at transcriptional and post-transcriptional levels including decoding of messenger RNA (mRNA) into polypeptides via ribosome-mediated translation. Translational regulation has been intensively studied in the model dicot plant Arabidopsis thaliana , and in this study, we assessed the translational status [proportion of steady-state mRNA associated with ribosomes] of mRNAs by Translating Ribosome Affinity Purification followed by mRNA-sequencing (TRAP-seq) in rice ( Oryza sativa ), a model monocot plant and the most important food crop. A survey of three tissues found that most transcribed rice genes are translated whereas few transposable elements are associated with ribosomes. Genes with short and GC-rich coding regions are overrepresented in ribosome-associated mRNAs, suggesting that the GC-richness characteristic of coding sequences in grasses may be an adaptation that favors efficient translation. Transcripts with retained introns and extended 5′ untranslated regions are underrepresented on ribosomes, and rice genes belonging to different evolutionary lineages exhibited differential enrichment on the ribosomes that was associated with GC content. Genes involved in photosynthesis and stress responses are preferentially associated with ribosomes, whereas genes in epigenetic regulation pathways are the least enriched on ribosomes. Such variation is more dramatic in rice than that in Arabidopsis and is correlated with the wide variation of GC content of transcripts in rice. Taken together, variation in the translation status of individual transcripts reflects important mechanisms of gene regulation, which may have a role in evolution and diversification." />
<meta name="DC.Contributor" content="Dongyan Zhao" />
<meta name="DC.Contributor" content="John P. Hamilton" />
<meta name="DC.Contributor" content="Michael Hardigan" />
<meta name="DC.Contributor" content="Dongmei Yin" />
<meta name="DC.Contributor" content="Tao He" />
<meta name="DC.Contributor" content="Brieanne Vaillancourt" />
<meta name="DC.Contributor" content="Mauricio Reynoso" />
<meta name="DC.Contributor" content="Germain Pauluzzi" />
<meta name="DC.Contributor" content="Scott Funkhouser" />
<meta name="DC.Contributor" content="Yuehua Cui" />
<meta name="DC.Contributor" content="Julia Bailey-Serres" />
<meta name="DC.Contributor" content="Jiming Jiang" />
<meta name="DC.Contributor" content="C. Robin Buell" />
<meta name="DC.Contributor" content="Ning Jiang" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gene expression is controlled at transcriptional and post-transcriptional levels including decoding of messenger RNA (mRNA) into polypeptides via ribosome-mediated translation. Translational regulation has been intensively studied in the model dicot plant &lt;i&gt;Arabidopsis thaliana&lt;/i&gt;, and in this study, we assessed the translational status [proportion of steady-state mRNA associated with ribosomes] of mRNAs by Translating Ribosome Affinity Purification followed by mRNA-sequencing (TRAP-seq) in rice (&lt;i&gt;Oryza sativa&lt;/i&gt;), a model monocot plant and the most important food crop. A survey of three tissues found that most transcribed rice genes are translated whereas few transposable elements are associated with ribosomes. Genes with short and GC-rich coding regions are overrepresented in ribosome-associated mRNAs, suggesting that the GC-richness characteristic of coding sequences in grasses may be an adaptation that favors efficient translation. Transcripts with retained introns and extended 5′ untranslated regions are underrepresented on ribosomes, and rice genes belonging to different evolutionary lineages exhibited differential enrichment on the ribosomes that was associated with GC content. Genes involved in photosynthesis and stress responses are preferentially associated with ribosomes, whereas genes in epigenetic regulation pathways are the least enriched on ribosomes. Such variation is more dramatic in rice than that in &lt;i&gt;Arabidopsis&lt;/i&gt; and is correlated with the wide variation of GC content of transcripts in rice. Taken together, variation in the translation status of individual transcripts reflects important mechanisms of gene regulation, which may have a role in evolution and diversification.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/203" />
<meta name="citation_id" content="7/1/203" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/203" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/203.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/203.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/203.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036020" />
<meta name="citation_pmid" content="27852012" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="203" />
<meta name="citation_lastpage" content="219" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dongyan Zhao" />
<meta name="citation_author_institution" content="Department of Horticulture, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2080-8416" />
<meta name="citation_author" content="John P. Hamilton" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8682-5526" />
<meta name="citation_author" content="Michael Hardigan" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Dongmei Yin" />
<meta name="citation_author_institution" content="Department of Horticulture, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Tao He" />
<meta name="citation_author_institution" content="Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_institution" content="Department of Mathematics, San Francisco State University, California 94132" />
<meta name="citation_author" content="Brieanne Vaillancourt" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Mauricio Reynoso" />
<meta name="citation_author_institution" content="Center for Plant Cell Biology, University of California, Riverside, California 92521" />
<meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California, Riverside, California 92521" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9990-5808" />
<meta name="citation_author" content="Germain Pauluzzi" />
<meta name="citation_author_institution" content="Center for Plant Cell Biology, University of California, Riverside, California 92521" />
<meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California, Riverside, California 92521" />
<meta name="citation_author" content="Scott Funkhouser" />
<meta name="citation_author_institution" content="Genetics Program, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7798-6830" />
<meta name="citation_author" content="Yuehua Cui" />
<meta name="citation_author_institution" content="Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Julia Bailey-Serres" />
<meta name="citation_author_institution" content="Center for Plant Cell Biology, University of California, Riverside, California 92521" />
<meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California, Riverside, California 92521" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8568-7125" />
<meta name="citation_author" content="Jiming Jiang" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin–Madison, Wisconsin 53705" />
<meta name="citation_author" content="C. Robin Buell" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_email" content="buell@msu.edu" />
<meta name="citation_author_email" content="jiangn@msu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6727-4677" />
<meta name="citation_author" content="Ning Jiang" />
<meta name="citation_author_institution" content="Department of Horticulture, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_institution" content="Genetics Program, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_institution" content="Program in Ecology, Evolutionary Biology and Behavior, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_email" content="buell@msu.edu" />
<meta name="citation_author_email" content="jiangn@msu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2776-6669" />
<meta name="citation_reference" content="citation_journal_abbrev=Comp Funct Genomics;citation_author=P. R. Araujo;citation_author=K. Yoon;citation_author=D. Ko;citation_author=A. D. Smith;citation_author=M. Qiao;citation_title=Before It Gets Started: Regulating Translation at the 5' UTR.;citation_pages=475731-475731;citation_volume=2012;citation_year=2012;citation_pmid=22693426;citation_doi=10.1155/2012/475731" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Arava;citation_author=Y. Wang;citation_author=J. D. Storey;citation_author=C. L. Liu;citation_author=P. O. Brown;citation_title=Genome-wide analysis of mRNA translation profiles in Saccharomyces cerevisiae.;citation_volume=100;citation_year=2003;citation_pmid=12660367;citation_doi=10.1073/pnas.0635171100" />
<meta name="citation_reference" content="citation_journal_title=Biol. Rev. Camb. Philos. Soc.;citation_author=S. K. Behura;citation_author=D. W. Severson;citation_title=Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes.;citation_pages=49-61;citation_volume=88;citation_year=2013;citation_pmid=22889422;citation_doi=10.1111/j.1469-185X.2012.00242.x" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_pmid=16632515;citation_doi=10.1093/biostatistics/kxj037" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=T. C. Boothby;citation_author=R. S. Zipper;citation_author=C. M. van der Weele;citation_author=S. M. Wolniak;citation_title=Removal of retained introns regulates translation in the rapidly developing gametophyte of Marsilea vestita.;citation_pages=517-529;citation_volume=24;citation_year=2013;citation_pmid=23434411;citation_doi=10.1016/j.devcel.2013.01.015" />
<meta name="citation_reference" content="citation_journal_title=Arabidopsis Book;citation_author=K. S. Browning;citation_author=J. Bailey-Serres;citation_title=Mechanism of cytoplasmic mRNA translation.;citation_pages=e0176;citation_volume=13;citation_year=2015;citation_pmid=26019692;citation_doi=10.1199/tab.0176" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. RNA;citation_author=P. T. Buckley;citation_author=M. Khaladkar;citation_author=J. Kim;citation_author=J. Eberwine;citation_title=Cytoplasmic intron retention, function, splicing, and the sentinel RNA hypothesis.;citation_pages=223-230;citation_volume=5;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Campbell;citation_author=B. J. Haas;citation_author=J. P. Hamilton;citation_author=S. M. Mount;citation_author=C. R. Buell;citation_title=Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis.;citation_pages=327-327;citation_volume=7;citation_year=2006;citation_pmid=17194304;citation_doi=10.1186/1471-2164-7-327" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. A. Campbell;citation_author=W. Zhu;citation_author=N. Jiang;citation_author=H. Lin;citation_author=S. Ouyang;citation_title=Identification and characterization of lineage-specific genes within the Poaceae.;citation_volume=145;citation_year=2007;citation_pmid=17951464;citation_doi=10.1104/pp.107.104513" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Carels;citation_author=G. Bernardi;citation_title=Two Classes of Genes in Plants;citation_pages=1819-1825;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10747072;citation_doi=10.3389/fgene.2012.00055" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Chen;citation_author=W. Lee;citation_author=A. K. Hottes;citation_author=L. Shapiro;citation_author=H. H. McAdams;citation_title=Codon usage between genomes is constrained by genome-wide mutational processes.;citation_volume=101;citation_year=2004;citation_pmid=14990797;citation_doi=10.1073/pnas.0307827100" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y. Clement;citation_author=M. A. Fustier;citation_author=B. Nabholz;citation_author=S. Glemin;citation_title=The bimodal distribution of genic GC content is ancestral to monocot species.;citation_volume=7;citation_year=2015;citation_pmid=25527839;citation_doi=10.1093/gbe/evu278" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. A. Counce;citation_author=T. C. Keisling;citation_author=A. J. Mitchell;citation_title=A Uniform, Objective, and Adaptive System for Expressing Rice Development;citation_pages=436-443;citation_volume=40;citation_year=2000;citation_issue=2;citation_doi=10.2135/cropsci2000.402436x" />
<meta name="citation_reference" content="citation_journal_title=Phytochem. Bull.;citation_author=J. J. Doyle;citation_author=J. L. Doyle;citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue.;citation_pages=11-15;citation_volume=19;citation_year=1987;citation_doi=10.2307/4119796" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Duret;citation_author=D. Mouchiroud;citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis;citation_pages=4482-4487;citation_volume=96;citation_year=1999;citation_issue=8;citation_pmid=10200288;citation_doi=10.1073/pnas.96.8.4482" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=L. Duret;citation_author=N. Galtier;citation_title=Biased gene conversion and the evolution of mammalian genomic landscapes.;citation_pages=285-311;citation_volume=10;citation_year=2009;citation_pmid=19630562;citation_doi=10.1146/annurev-genom-082908-150001" />
<meta name="citation_reference" content="citation_journal_title=BMB Rep.;citation_author=Z. E.;citation_author=L. Wang;citation_author=J. Zhou;citation_title=Splicing and alternative splicing in rice and humans.;citation_pages=439-447;citation_volume=46;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. L. Fennoy;citation_author=J. Bailey-Serres;citation_title=Synonymous codon usage in Zea mays L. nuclear genes is varied by levels of C and G-ending codons;citation_pages=5294-5300;citation_volume=21;citation_year=1993;citation_issue=23;citation_pmid=8265340;citation_doi=10.1093/nar/21.23.5294" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. A. Ferguson;citation_author=D. Zhao;citation_author=N. Jiang;citation_title=Selective acquisition and retention of genomic sequences by should be Pack-Mutator-like elements based on guanine-cytosine content and the breadth of expression.;citation_volume=163;citation_year=2013;citation_pmid=24028844;citation_doi=10.1104/pp.113.223271" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=S. Glemin;citation_author=Y. Clement;citation_author=J. David;citation_author=A. Ressayre;citation_title=GC content evolution in coding regions of angiosperm genomes: a unifying hypothesis.;citation_pages=263-270;citation_volume=30;citation_year=2014;citation_pmid=24916172;citation_doi=10.1016/j.tig.2014.05.002" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=X. Guo;citation_author=J. Bao;citation_author=L. Fan;citation_title=Evidence of selectively driven codon usage in rice: implications for GC content evolution of Gramineae genes.;citation_pages=1015-1021;citation_volume=581;citation_year=2007;citation_issue=5;citation_pmid=17306258;citation_doi=10.1016/j.febslet.2007.01.088" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=R. E. Halbeisen;citation_author=T. Scherrer;citation_author=A. P. Gerber;citation_title=Affinity purification of ribosomes to access the translatome.;citation_pages=306-310;citation_volume=48;citation_year=2009;citation_issue=3;citation_pmid=19398006;citation_doi=10.1016/j.ymeth.2009.04.003" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=M. Helmy;citation_author=N. Sugiyama;citation_author=M. Tomita;citation_author=Y. Ishihama;citation_title=The Rice Proteogenomics Database OryzaPG-DB: Development, Expansion, and New Features.;citation_pages=65-65;citation_volume=3;citation_year=2012;citation_pmid=22639657" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Hirochika;citation_author=K. Sugimoto;citation_author=Y. Otsuki;citation_author=H. Tsugawa;citation_author=M. Kanda;citation_title=Retrotransposons of rice involved in mutations induced by tissue culture.;citation_pages=7783-7788;citation_volume=93;citation_year=1996;citation_issue=15;citation_pmid=8755553;citation_doi=10.1073/pnas.93.15.7783" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Hupe;citation_author=M. X. Li;citation_author=K. Gertow Gillner;citation_author=R. H. Adams;citation_author=J. M. Stenman;citation_title=Evaluation of TRAP-sequencing technology with a versatile conditional mouse model.;citation_volume=42;citation_year=2014;citation_pmid=24165879;citation_doi=10.1093/nar/gkt995" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=E. H. Hurowitz;citation_author=P. O. Brown;citation_title=Genome-wide analysis of mRNA lengths in Saccharomyces cerevisiae.;citation_pages=R2-R2;citation_volume=5;citation_year=2003;citation_issue=1;citation_pmid=14709174;citation_doi=10.1186/gb-2003-5-1-r2" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=N. T. Ingolia;citation_title=Ribosome profiling: new views of translation, from single codons to genome scale.;citation_pages=205-213;citation_volume=15;citation_year=2014;citation_pmid=24468696;citation_doi=10.1038/nrg3645" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The map-based sequence of the rice genome.;citation_pages=793-800;citation_volume=436;citation_year=2005;citation_issue=7052;citation_pmid=16100779;citation_doi=10.1038/nature03895" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=Z. Ji;citation_author=R. Song;citation_author=A. Regev;citation_author=K. Struhl;citation_title=Many lncRNAs, 5′UTRs, and pseudogenes are translated and some are likely to express functional proteins.;citation_volume=4;citation_year=2015;citation_pmid=26687005;citation_doi=10.7554/eLife.08890" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Jiang;citation_author=Z. Bao;citation_author=X. Zhang;citation_author=H. Hirochika;citation_author=S. R. Eddy;citation_title=An active DNA transposon family in rice.;citation_pages=163-167;citation_volume=421;citation_year=2003;citation_issue=6919;citation_pmid=12520302;citation_doi=10.1038/nature01214" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Jiang;citation_author=A. A. Ferguson;citation_author=R. K. Slotkin;citation_author=D. Lisch;citation_title=Pack-Mutator-like transposable elements (Pack-MULEs) induce directional modification of genes through biased insertion and DNA acquisition.;citation_volume=108;citation_year=2011;citation_pmid=21220310;citation_doi=10.1073/pnas.1010814108" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=Y. Jiao;citation_author=E. M. Meyerowitz;citation_title=Cell-type specific analysis of translating RNAs in developing flowers reveals new levels of control;citation_pages=419-419;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=20924354;citation_doi=10.1038/msb.2010.76" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Juntawong;citation_author=T. Girke;citation_author=J. Bazin;citation_author=J. Bailey-Serres;citation_title=Translational dynamics revealed by genome-wide profiling of ribosome footprints in Arabidopsis.;citation_volume=111;citation_year=2014;citation_pmid=24367078;citation_doi=10.1073/pnas.1317811111" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Kalyna;citation_author=C. G. Simpson;citation_author=N. H. Syed;citation_author=D. Lewandowska;citation_author=Y. Marquez;citation_title=Alternative splicing and nonsense-mediated decay modulate expression of important regulatory genes in Arabidopsis.;citation_volume=40;citation_year=2012;citation_pmid=22127866;citation_doi=10.1093/nar/gkr932" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. Kanaya;citation_author=Y. Yamada;citation_author=M. Kinouchi;citation_author=Y. Kudo;citation_author=T. Ikemura;citation_title=Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis.;citation_pages=290-298;citation_volume=53;citation_year=2001;citation_issue=4-5;citation_pmid=11675589;citation_doi=10.1007/s002390010219" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Kawaguchi;citation_author=J. Bailey-Serres;citation_title=mRNA sequence features that contribute to translational regulation in Arabidopsis;citation_pages=955-965;citation_volume=33;citation_year=2005;citation_issue=3;citation_pmid=15716313;citation_doi=10.1093/nar/gki240" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=Y. Kawahara;citation_author=M. de la Bastide;citation_author=J. P. Hamilton;citation_author=H. Kanamori;citation_author=W. R. McCombie;citation_title=Improvement of the Oryza sativa Nipponbare reference genome using next generation sequence and optical map data.;citation_pages=4;citation_volume=6;citation_year=2013;citation_pmid=24280374;citation_doi=10.1186/1939-8433-6-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=G. S. Khush;citation_title=Strategies for increasing the yield potential of cereals: case of rice as an example.;citation_pages=433-436;citation_volume=132;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Kikuchi;citation_author=K. Terauchi;citation_author=M. Wada;citation_author=H. Y. Hirano;citation_title=The plant MITE mPing is mobilized in anther culture.;citation_pages=167-170;citation_volume=421;citation_year=2003;citation_issue=6919;citation_pmid=12520303;citation_doi=10.1038/nature01218" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kikuchi;citation_author=K. Satoh;citation_author=T. Nagata;citation_author=N. Kawagashira;citation_author=K. Doi;citation_title=Collection, Mapping, and Annotation of Over 28,000 cDNA Clones from japonica Rice;citation_pages=376-379;citation_volume=301;citation_year=2003;citation_issue=5631;citation_pmid=12869764;citation_doi=10.1126/science.1081288" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_title=A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes.;citation_author=R. D. Knight;citation_author=S. J. Freeland;citation_author=L. F. Landweber;citation_volume=2;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Koller;citation_author=M. P. Washburn;citation_author=B. M. Lange;citation_author=N. L. Andon;citation_author=C. Deciu;citation_title=Proteomic survey of metabolic pathways in rice.;citation_volume=99;citation_year=2002;citation_pmid=12163647;citation_doi=10.1073/pnas.172183199" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=S. Komatsu;citation_title=Rice proteome database: a step toward functional analysis of the rice genome.;citation_pages=179-190;citation_volume=59;citation_year=2005;citation_issue=1;citation_pmid=16217611;citation_doi=10.1007/s11103-005-2160-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=G. Kudla;citation_author=L. Lipinski;citation_author=F. Caffin;citation_author=A. Helwak;citation_author=M. Zylicz;citation_title=High guanine and cytosine content increases mRNA levels in mammalian cells.;citation_pages=e180-e180;citation_volume=4;citation_year=2006;citation_issue=6;citation_pmid=16700628;citation_doi=10.1371/journal.pbio.0040180" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=D. H. Lackner;citation_author=T. H. Beilharz;citation_author=S. Marguerat;citation_author=J. Mata;citation_author=S. Watt;citation_title=A network of multiple regulatory layers shapes gene expression in fission yeast.;citation_pages=145-155;citation_volume=26;citation_year=2007;citation_issue=1;citation_pmid=17434133;citation_doi=10.1016/j.molcel.2007.03.002" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Lavut;citation_author=D. Raveh;citation_title=Sequestration of highly expressed mRNAs in cytoplasmic granules, P-bodies, and stress granules enhances cell viability.;citation_pages=e1002527-e1002527;citation_volume=8;citation_year=2012;citation_issue=2;citation_pmid=22383896;citation_doi=10.1371/journal.pgen.1002527" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=L. Lei;citation_author=J. Shi;citation_author=J. Chen;citation_author=M. Zhang;citation_author=S. Sun;citation_title=Ribosome profiling reveals dynamic translational landscape in maize seedlings under drought stress.;citation_pages=1206-1218;citation_volume=84;citation_year=2015;citation_pmid=26568274;citation_doi=10.1111/tpj.13073" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=P. Mukhopadhyay;citation_author=S. Basak;citation_author=T. C. Ghosh;citation_title=Nature of selective constraints on synonymous codon usage of rice differs in GC-poor and GC-rich genes.;citation_pages=71-81;citation_volume=400;citation_year=2007;citation_issue=1-2;citation_pmid=17629420;citation_doi=10.1016/j.gene.2007.05.027" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Plant.;citation_author=T. Murashige;citation_author=F. Skoog;citation_title=A revised medium for rapid growth and bioassays with tobacco tissue cultures.;citation_pages=473-497;citation_volume=15;citation_year=1962;citation_doi=10.1111/j.1399-3054.1962.tb08052.x" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=A. Mustroph;citation_author=P. Juntawong;citation_author=J. Bailey-Serres;citation_title=Isolation of plant polysomal mRNA by differential centrifugation and ribosome immunopurification methods.;citation_pages=109-126;citation_volume=553;citation_year=2009;citation_pmid=19588103;citation_doi=10.1007/978-1-60327-563-7_6" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Mustroph;citation_author=M. E. Zanetti;citation_author=C. J. Jang;citation_author=H. E. Holtan;citation_author=P. P. Repetti;citation_title=Profiling translatomes of discrete cell populations resolves altered cellular priorities during hypoxia in Arabidopsis.;citation_volume=106;citation_year=2009;citation_pmid=19843695;citation_doi=10.1073/pnas.0906131106" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. Mustroph;citation_author=S. C. Lee;citation_author=T. Oosumi;citation_author=M. E. Zanetti;citation_author=H. Yang;citation_title=Cross-kingdom comparison of transcriptomic adjustments to low-oxygen stress highlights conserved and plant-specific responses.;citation_volume=152;citation_year=2010;citation_pmid=20097791;citation_doi=10.1104/pp.109.151845" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Nakazaki;citation_author=Y. Okumoto;citation_author=A. Horibata;citation_author=S. Yamahira;citation_author=M. Teraishi;citation_title=Mobilization of a transposon in the rice genome.;citation_pages=170-172;citation_volume=421;citation_year=2003;citation_issue=6919;citation_pmid=12520304;citation_doi=10.1038/nature01219" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=R. Narsai;citation_author=A. Ivanova;citation_author=S. Ng;citation_author=J. Whelan;citation_title=Defining reference genes in Oryza sativa using organ, development, biotic and abiotic transcriptome datasets.;citation_pages=56-56;citation_volume=10;citation_year=2010;citation_pmid=20353606;citation_doi=10.1186/1471-2229-10-56" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. K. Nguyen;citation_author=E. M. Southern;citation_title=Minimising the secondary structure of DNA targets by incorporation of a modified deoxynucleoside: implications for nucleic acid analysis by hybridisation;citation_pages=3904-3909;citation_volume=28;citation_year=2000;citation_issue=20;citation_pmid=11024169;citation_doi=10.1093/nar/28.20.3904" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Q. Pan;citation_author=O. Shai;citation_author=L. J. Lee;citation_author=B. J. Frey;citation_author=B. J. Blencowe;citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing.;citation_pages=1413-1415;citation_volume=40;citation_year=2008;citation_issue=12;citation_pmid=18978789;citation_doi=10.1038/ng.259" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. H. Park;citation_author=P. J. Chung;citation_author=P. Juntawong;citation_author=J. Bailey-Serres;citation_author=Y. S. Kim;citation_title=Posttranscriptional control of photosynthetic mRNA decay under stress conditions requires 3′ and 5′ untranslated regions and correlates with differential polysome association in rice.;citation_volume=159;citation_year=2012;citation_pmid=22566494;citation_doi=10.1104/pp.112.194928" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=E. Pessia;citation_author=A. Popa;citation_author=S. Mousset;citation_author=C. Rezvoy;citation_author=L. Duret;citation_title=Evidence for widespread GC-biased gene conversion in eukaryotes.;citation_volume=4;citation_year=2012;citation_pmid=22628461;citation_doi=10.1093/gbe/evs052" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=F. Picard;citation_author=H. Milhem;citation_author=P. Loubiere;citation_author=B. Laurent;citation_author=M. Cocaign-Bousquet;citation_title=Bacterial translational regulations: high diversity between all mRNAs and major role in gene expression.;citation_pages=528-528;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23036066;citation_doi=10.1186/1471-2164-13-528" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=N. Picault;citation_author=C. Chaparro;citation_author=B. Piegu;citation_author=W. Stenger;citation_author=D. Formey;citation_title=Identification of an active LTR retrotransposon in rice.;citation_pages=754-765;citation_volume=58;citation_year=2009;citation_issue=5;citation_pmid=19187041;citation_doi=10.1111/j.1365-313X.2009.03813.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Ressayre;citation_author=S. Glemin;citation_author=P. Montalent;citation_author=L. Serre-Giardi;citation_author=C. Dillmann;citation_title=Introns structure patterns of variation in nucleotide composition in Arabidopsis thaliana and rice protein-coding genes.;citation_volume=7;citation_year=2015;citation_pmid=26450849;citation_doi=10.1093/gbe/evv189" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=M. A. Reynoso;citation_author=P. Juntawong;citation_author=M. Lancia;citation_author=F. A. Blanco;citation_author=J. Bailey-Serres;citation_title=Translating Ribosome Affinity Purification (TRAP) followed by RNA sequencing technology (TRAP-SEQ) for quantitative assessment of plant translatomes.;citation_pages=185-207;citation_volume=1284;citation_year=2015;citation_pmid=25757773;citation_doi=10.1007/978-1-4939-2444-8_9" />
<meta name="citation_reference" content="citation_journal_title=Arabidopsis Book;citation_author=B. Roy;citation_author=A. G. von Arnim;citation_title=Translational regulation of cytoplasmic mRNAs.;citation_pages=e0165;citation_volume=11;citation_year=2013;citation_pmid=23908601;citation_doi=10.1199/tab.0165" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=F. Sabot;citation_author=N. Picault;citation_author=M. El-Baidouri;citation_author=C. Llauro;citation_author=C. Chaparro;citation_title=Transpositional landscape of the rice genome revealed by paired-end mapping of high-throughput re-sequencing data.;citation_pages=241-246;citation_volume=66;citation_year=2011;citation_issue=2;citation_pmid=21219509;citation_doi=10.1111/j.1365-313X.2011.04492.x" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=C. Schweingruber;citation_author=S. C. Rufener;citation_author=D. Zund;citation_author=A. Yamashita;citation_author=O. Muhlemann;citation_title=Nonsense-mediated mRNA decay - Mechanisms of substrate mRNA recognition and degradation in mammalian cells.;citation_pages=612-623;citation_volume=1829;citation_year=2013;citation_pmid=23435113;citation_doi=10.1016/j.bbagrm.2013.02.005" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=P. Shah;citation_author=Y. Ding;citation_author=M. Niemczyk;citation_author=G. Kudla;citation_author=J. B. Plotkin;citation_title=Rate-limiting steps in yeast protein translation.;citation_pages=1589-1601;citation_volume=153;citation_year=2013;citation_pmid=23791185;citation_doi=10.1016/j.cell.2013.05.049" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms.;citation_pages=28-38;citation_volume=24;citation_year=1986;citation_issue=1-2;citation_pmid=3104616;citation_doi=10.1007/BF02099948" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. R. Starck;citation_author=J. C. Tsai;citation_author=K. Chen;citation_author=M. Shodiya;citation_author=L. Wang;citation_title=Translation from the 5' untranslated region shapes the integrated stress response;citation_pages=aad3867-aad3867;citation_volume=351;citation_year=2016;citation_issue=6272;citation_pmid=26823435;citation_doi=10.1126/science.aad3867" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. V. Tatarinova;citation_author=N. N. Alexandrov;citation_author=J. B. Bouck;citation_author=K. A. Feldmann;citation_title=GC3 biology in corn, rice, sorghum and other grasses.;citation_pages=308-308;citation_volume=11;citation_year=2010;citation_pmid=20470436;citation_doi=10.1186/1471-2164-11-308" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.;citation_pages=796-815;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130711;citation_doi=10.1038/35048692" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Z. Tian;citation_author=C. Rizzon;citation_author=J. Du;citation_author=L. Zhu;citation_author=J. L. Bennetzen;citation_title=Do genetic recombination and gene density shape the pattern of DNA elimination in rice long terminal repeat retrotransposons?;citation_volume=19;citation_year=2009;citation_pmid=19789376;citation_doi=10.1101/gr.083899.108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. B. Wang;citation_author=V. Brendel;citation_title=Genomewide comparative analysis of alternative splicing in plants.;citation_volume=103;citation_year=2006;citation_pmid=16632598;citation_doi=10.1073/pnas.0602039103" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. T. Wang;citation_author=R. Sandberg;citation_author=S. Luo;citation_author=I. Khrebtukova;citation_author=L. Zhang;citation_title=Alternative isoform regulation in human tissue transcriptomes.;citation_pages=470-476;citation_volume=456;citation_year=2008;citation_issue=7221;citation_pmid=18978772;citation_doi=10.1038/nature07509" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=H. C. Wang;citation_author=D. A. Hickey;citation_title=Rapid divergence of codon usage patterns within the rice genome.;citation_pages=S6;citation_volume=7;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=A. J. Williams;citation_author=J. Werner-Fraczek;citation_author=I. F. Chang;citation_author=J. Bailey-Serres;citation_title=Regulated phosphorylation of 40S ribosomal protein S6 in root tips of maize.;citation_volume=132;citation_year=2003;citation_pmid=12913163;citation_doi=10.1104/pp.103.022749" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. K. Wong;citation_author=J. Wang;citation_author=L. Tao;citation_author=J. Tan;citation_author=J. Zhang;citation_title=Compositional Gradients in Gramineae Genes;citation_pages=851-856;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12045139;citation_doi=10.1101/gr.189102" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=L. Xiong;citation_author=Y. Yang;citation_title=Disease resistance and abiotic stress tolerance in rice are inversely modulated by an abscisic acid-inducible mitogen-activated protein kinase.;citation_volume=15;citation_year=2003;citation_pmid=12615946;citation_doi=10.1105/tpc.008714" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=M. E. Zanetti;citation_author=I. F. Chang;citation_author=F. Gong;citation_author=D. W. Galbraith;citation_author=J. Bailey-Serres;citation_title=Immunopurification of Polyribosomal Complexes of Arabidopsis for Global Analysis of Gene Expression;citation_pages=624-635;citation_volume=138;citation_year=2005;citation_issue=2;citation_pmid=15955926;citation_doi=10.1104/pp.105.059477" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="Gene expression is controlled at transcriptional and post-transcriptional levels including decoding of messenger RNA (mRNA) into polypeptides via ribosome-mediated translation. Translational regulation has been intensively studied in the model dicot plant Arabidopsis thaliana , and in this study, we assessed the translational status [proportion of steady-state mRNA associated with ribosomes] of mRNAs by Translating Ribosome Affinity Purification followed by mRNA-sequencing (TRAP-seq) in rice ( Oryza sativa ), a model monocot plant and the most important food crop. A survey of three tissues found that most transcribed rice genes are translated whereas few transposable elements are associated with ribosomes. Genes with short and GC-rich coding regions are overrepresented in ribosome-associated mRNAs, suggesting that the GC-richness characteristic of coding sequences in grasses may be an adaptation that favors efficient translation. Transcripts with retained introns and extended 5′ untranslated regions are underrepresented on ribosomes, and rice genes belonging to different evolutionary lineages exhibited differential enrichment on the ribosomes that was associated with GC content. Genes involved in photosynthesis and stress responses are preferentially associated with ribosomes, whereas genes in epigenetic regulation pathways are the least enriched on ribosomes. Such variation is more dramatic in rice than that in Arabidopsis and is correlated with the wide variation of GC content of transcripts in rice. Taken together, variation in the translation status of individual transcripts reflects important mechanisms of gene regulation, which may have a role in evolution and diversification." />
<meta name="og-title" property="og:title" content="Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/203" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gene expression is controlled at transcriptional and post-transcriptional levels including decoding of messenger RNA (mRNA) into polypeptides via ribosome-mediated translation. Translational regulation has been intensively studied in the model dicot plant Arabidopsis thaliana , and in this study, we assessed the translational status [proportion of steady-state mRNA associated with ribosomes] of mRNAs by Translating Ribosome Affinity Purification followed by mRNA-sequencing (TRAP-seq) in rice ( Oryza sativa ), a model monocot plant and the most important food crop. A survey of three tissues found that most transcribed rice genes are translated whereas few transposable elements are associated with ribosomes. Genes with short and GC-rich coding regions are overrepresented in ribosome-associated mRNAs, suggesting that the GC-richness characteristic of coding sequences in grasses may be an adaptation that favors efficient translation. Transcripts with retained introns and extended 5′ untranslated regions are underrepresented on ribosomes, and rice genes belonging to different evolutionary lineages exhibited differential enrichment on the ribosomes that was associated with GC content. Genes involved in photosynthesis and stress responses are preferentially associated with ribosomes, whereas genes in epigenetic regulation pathways are the least enriched on ribosomes. Such variation is more dramatic in rice than that in Arabidopsis and is correlated with the wide variation of GC content of transcripts in rice. Taken together, variation in the translation status of individual transcripts reflects important mechanisms of gene regulation, which may have a role in evolution and diversification." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
    <title>Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mN1nBT5m1n6Iy7zU2uiTdOujlV99nUeUl1k4ARyYeTk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460530","apath":"\/ggg\/7\/1\/203.atom","pisa":"ggg;7\/1\/203","ac":{"\/ggg\/7\/1\/203.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/203.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/203"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/203"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/203.short":true,"\/content\/7\/1\/203":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wRYD8RsKVlxb7kNpJ\\\\\\\/dqJRtPzREWc8tmce+gkwhTt3lDISzJ0jdlSUz3NjI6E2US+9rQR8iSBP\\\\\\\/vy1lv7GcRQVXIkFEqA7Wm0+YpxqPrcGWtGXHzxyehVC4Bt1avKAmAqdv4dehIftz1zBXWNBMJAkadJMgFz8JMb0mpvYxkldqVTbJRe3fyKX43aODwAHwkghchuK7o6lcQ5NjChcPDhs4Suuvrk93WGNpIldmQeK7EmluhpAqppPQsQcgSSQcQwFKrbnPNqwGcMA8RrP9\\\\\\\/J0jEnfWPB45UytImihH7gI7cHfpoxfZZabpbASLkATQFgWTECpzJRdOW\\\\\\\/Bzy0OtVnI6sLPFseYFpYT\\\\\\\/8Iz5hQnjSxqocNIhcucGsvQ8zujRa8OShIOlN4QZ1V1\\\\\\\/7zJnTL7gJvYp5r5gLCFdkyksbMymt5I+lX9Ov0Kp7tHZSs1fPCNB0NIDs82kUE74l9itSICehrRB4c+trxMqtkUhq7L5pYiyok1MeQ\\\\\\\/H2r2oKzS8CxqFyy2uTp3SCrs\\\\\\\/vkqxwvIIEJZmHewt90Aezo92JbDPE6ykto17xQDayRPDWNxvXaRreY649Psd5K67gm0hUyKnS6Orgworb0Uf7OIOM1xr7oVBOWMA1+0IJg4sQwZbKbtQaAiNyun6gZoKsThCRDziHa6qiJfM3A6cgMq46qqeFBKT81G09B6MSuK3Ggc1WQRHA6SZF3XDZLNBMHHGIT5AFXgdSZc4D6J38I7T9cJfKwpxjGxl8gyJrpya7mSrjAga2hNSe7rPGwf36xBgzWy87wOGHnv5lunMdkkS8Bh3wK7iIbGYZxP6kucjy7sbCdRmeIwDaZ0PbagBl0X+fFRolZMcQl6zbZarpNuqPXrCoPrRpeo8tjzNaq\\\\\\\/vAKoSvV5jo\\\\\\\/wp+sKLcJTCimbsOb1C\\\\\\\/tV8eNTuS322R3xXuV818HJkdMWnVrMVlAuZkxeizrpz\\\\\\\/tQ3LLR1HO7qQHs7NsbSMtxuNC1+xymqrqJyZF18Smy50AwWExyYnGmnYA3v2KTobezDFn0ZfaqDb0WehpWTwyhVy\\\\\\\/72sYw3CXs9zSTNJLEPjdAKHI+lxzqQozJ+HM9SpMeDeWf1YwDsxao0AZ4WmXkczwjlUhAQwlRGmh34DWnPYawT\\\\\\\/xfsaNr0RFhz+bkEEewhmZD\\\\\\\/x3dJZKvpJ4Ta0dNUwGbtN2AEuoa88Ax\\\\\\\/gIfiMQHiCaXJu5jyAZ7bSg2Hc7wHlCZ6TEB\\\\\\\/5iXqJDElzPun3bI+98fPNabrO\\\\\\\/5hcL+uTUDy1eAoGbkrLLl4U\\\u0022,\\\u0022iv\\\u0022:\\\u00225M\\\\\\\/CT+kp4T0aksHPZ0qLXQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fe0bae6f88cb7660191813e7162836def5dfa83cbf780e5829900164887c8fc4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224\\\\\\\/n92g1X4DedVz1hrgxhAzM29iJRBVx9TTlB4VKfi++EFeM1w8LZLDhBIo3QQ0X+rPFVSSTNY00ePnIOKMfmzGBAPTbpSBIqzKd\\\\\\\/a70AnZGMgsRjB3vhK0tbwde3ytq5jvkUN+LoInOV9xmL43msrPilmI5p5z7vzn4JVE1\\\\\\\/BggxBXfJvPhLahYzaKDtpTMgyDWUHC3rwzBXaKwI4ERGcRIfAc79w4XCpPWWBh2ZbhSJ\\\\\\\/x79YpxM9ZDm+SlgFxXBHK8zJ3BflBJYCCMcoCEXv6KyheiHUMIcWVnFhvLvWolqOh18EhQsjTQoeG37vaGx99Kj2mxFOSzGfIME2viCBmt1JnvHg8yTLYdNtTMG8tsguz+X3HylDEINAOfCNIVHbeKF1ErtS4pmBoyubCuTTi71\\\\\\\/i+j2TsOOXBPBVNCWrhyIlkV8GP1LxVm282HwQEO2Qrwj7jx6JBU9VOPJ62cYGFID60kwf2nYPT2Jws2nsWKjkSwjAXsQVJOyS\\\\\\\/A9rBCQrQjmP0rQf4+EITJuES1dqcOEuSmKUl+kG5kY3+Cun5jAByQU58z1CTmIuGgIapjdeznoo7fi3VpxjvdjQSUt2N5Hz8AgRcbPDkqW6yRhPZCJDRvYiyvVnmFaB764E4j02M9uSTo39czpEVeWnvk6aZcBZvpwQzlNyruo0vrRpZRiwQKPg4GwX7DSUdv9JKPv2uShOUB1C39sNs3JBOAeTW8g0wYRYcHyOhfslMe+NHo4qsWw3TnNs9L383l8ELhU11SobypRKz5twKow9jJuSnHW4PxKr5BJqic4CKBF0gheKHWRmF5eJxXrPvPvMzvi8zcz\\\\\\\/0I52aNmIXDZp3YPuLZlJQT92lynC+EGLNl70NFKEKOFYxMqDVQl+gvdyXSXFT7zvASSQRHsXNQCtlz4Y7sjwlxxDs6QUb28vTvmY2F4B9gdQidLrAuo3lYdMcKbzij\\\\\\\/UVY\\\\\\\/q0Ha3sy7bZ3WAK+n4vkTsHD\\\\\\\/p2aAbmmPtaQp6MQglaixwIm1hcuSQQZi1BpeHVWQLEF4AXrJ7nEItK65GxAZQPJPPLoKrJYeU+XBNqPWq5sm+aVEOdMemHeMJaK73Tz1u4T4TF1\\\\\\\/XdZy9Q7JA4N9HDE3wWRA75LdCwvDl\\\\\\\/Hb8eFndWmid6di0epn6R7Mt+iaBMYmGtV8VU3Ox1gr1fk5dX8g1BDIO0XRX9btZ0JMFqY6fHCmi33oSvUXiB6o5LXxCx5nOg1TsGnVXvLelOpjdTDqV3xbZEaxVU0uRi7IDHUI4EfaI6ToZIsnqZ\\\\\\\/dtyHF7JDnYj0RVEroQwh9u9rWnXYQTOQtz9CAxXPjb3yq4IRx294wUIo3Q5bvM1z4aLnlERG1gF552DzdMibC1IsS8fbJOveWV+LLoyjfszdkCPeJDQT4ZG2jPYMADEPZuv+LRvTHrGY4ACgh81fD5MTDmGBknu0niLPUc0JkbkCLS1YTBtQ70FymlnEhIPmBmFl3tPPcsdjng77Q8kCelbf9eXHzH6m0u8oa65jZBH1iNJmSV6SrYlcZ7n3olkcsfYH8Q7HH4Dyy0Z2LI2pyoYoiDljoWxkApgvLSupTms\\\\\\\/hechrWN9TnCSUXXtkCpYfseVA4Jp3NFR2VYpa3lFLVZ\\\\\\\/MRu93x+qCwskje62XY5WHfskRMN9BnmCy06C85ZX0BD6IBScGS3Gki9tiGKWGqbXjQ070r7JFFj\\\\\\\/N0GMsKX4JSwv77vqrApQR\\\\\\\/haZs2QHKuY9JIBmAsRSjVYoIjPT\\\\\\\/AnspSpSa0vj4tqR99kkeOjZiwJJBy3cotXSgL\\\\\\\/bZnEzXgbZWiDNMIps2++NF4FUugOP0TTmWzAfDBTiRDWjv3WxsjPgoPFwl3SOq4kv6UX7NlQgr6+dzze\\\\\\\/P6PBU+2BThiMPnFf5E\\\\\\\/6cS6t145pluT3JrqKXqvxjWnz4htQNgtJ2I\\\\\\\/7HtwZ+h+yJgC+qTIVuYbv8BqBzm5ZxK73sLIp2WL+aKUiZw+BX1rxu9XLdRBwdPYqFeBu9SvvGyJ9q8HvxfRMyEMonIW4TaJcwMhx0ttBnH\\\\\\\/9pZk7XYQL9hPk0TwV\\\\\\\/4bq+C2ecg3dqAo9CwYnnSNowrU9P1Behz4rCRSMid5ZM2H1bvoe1sjWrwqbL6W3BvnJXWXtx2Ewr8MfFToS6KuzvYYvuJ6qVhptouVlxorZL15onkEOyWla3iNYwmIsVf90GyUEw\\\\\\\/ZXL7l2zkhuh\\\\\\\/XHQs4HalY2GwRSLc4STw3BtmAPIFfi2Opa9D4fy2S1wIqSQUQBv+vsiZSbaCcaQhWUd3vb6rqss\\\\\\\/GYKl51xpM5SeUllzSVf+bxHQr8MhtDbBfGJFMOyOc94RpBFbfaLNbVdwWLMrvcP9fLqUlQrBBsyGe+FfkzeZTVcZEIPnjCqUR8bnCmgIDyhZvLpoRaoRkr4YLqXFnMW4r3QIoZfTcqTYLgHtZHouWdT8kLXTqOOiK38QWvPr3vtL5XwVZa+0pDEBSYYrtjXgkv5pVB+XD\\\\\\\/OerhYgZ\\\\\\\/e7Y676+flfNV3XoQV8ew98JG7MSGJcbPRnD1fRu6Sxvqr6V57U3kE2QKvWuLJORtWHiglMYM9KOVGbyAHVu2rKl+jq0P05q0HGQQ0vUCdVFX+eLXpqtBaYpPelt+FMyp0+mwd7WcpWXc9ppjWeF\\\\\\\/G4DC44oLguMBh7AR718tPgoGKMTH8qCg7mdyqkpOBEqwpP\\\\\\\/+lAa8jor33CUTJwqDg\\\\\\\/zvhCUBJJDXX+8qhv8TfFK7gxj7RtnLzs763k4rZdrsL7SaIfB6yhtfJrN+M5cyjYU1UARxAGE\\\\\\\/mk8aHTkXIiKGPB5Fk67ZJexi1E0SRizQkTP8KEElrvtk8JWK7TAkQ+xo9imscfww2fG6\\\\\\\/qNh+Rs8P6g09FXPqVLEQez+BxkNwrIwHLAgytjAc3oyJjKnq0VgWEfgHofYOHMvfwuOespYLCydtsUc2D1iH2+P9jwL3sWnNc0V3RtI0t7+Tag9mWektnxzi\\\\\\\/oWi7i+WJA73NJNaXkX6kIxIcjbvHXqAiA2aLcG0rT3kwI5zXS898P7qxISD01QODJW06B9seYodzVEuV6h1J7jwsJ6rajrkugZr6k2KLtsDUj2nsyP+PQLIOSsQecFLD9JBTUx\\\\\\\/CJ7pmQrFoCdMZQAkbyl16bUWZdGVRBkKuRFUzgSdoqWSohKjk7yzyEBODzDo6kDF4l6CadZArVMrkIYAH5OmE6YZMtSnPAIgaI9CSxzCchrWw+ydiDTk5mdsHJw5HChjJ03AaTTyFk1l1W65WV4CPVmJ82N98IuvKz3DjXMRFFjjVr8rPT7ZH4SE5WrYqaEBotUZGMgAaF1UNDaTN2qhbZXae1r44Hejd92o7B2Lv4+G8IJYqJso7xigfRKJUql9LG60qB8YCUZCfnTkj+r7athMqCYXU57zmxdMWUo16jeRgcXxLzM\\\\\\\/VslC1TjxQnruk2mPHqOl4AczNixGNLOGunNTl4skroE\\\\\\\/GpjefvDz978EnEXw8pjLhAFIYcLOY2mdB0D7Rb7F7\\\\\\\/dgv4khlkIuoLT3NpOL0smXjB45BuZKZIfVD9WgRE0lKzzTr\\\\\\\/0z5J3QMBqf0eFKVsP0rS0oazb3ap63wtqB6hbcFcfK1ydrMHPhZwDdgJkEeduXdl5ywuaVaMDIloMGGVyRAlVZtAE4yn1faC1fP7bZQeOr84NJwRafv6yHqYtW1uTBKHteq7Ucd+JrZBkUiCXAHXN79R1xdQ3Suq90VuZ9ctcO67lvbm+FYAngqgKhAb4rssAn7QmueBkOj4N41PORDoH1njds\\\\\\\/+xu27xLX9vtc07uaELrXoUwZ8yH5CN0bbz2623Ds1XRe4hpaD1tTygbDNNQNJBSLz9NxkTSeb8hanS4ItBBYMwCUOkoxKMkSCDXXSDr46wf\\\\\\\/HVvqADZw7O\\\\\\\/h01NEcsr\\\\\\\/Kl\\\\\\\/LZs2rAVDcG0XwCmTn6zvfZIYSesF3HiTCMoWfFYX\\\\\\\/15g46jQGojD4uveaA\\\\\\\/KSXN94Vb3EvhUWbJmfoYFE75yUxzVoMG\\\\\\\/X9wGnlvTDX3iqL0CYHv4E5DBusgU8D+OqH3xygI109\\\\\\\/ees8K+nU4gka4O7yfCT1RInOatq3ofLoEjXcPsWnKN0KrGzieVrmPgwMddJ6kP\\\\\\\/6jNYz8bJ6LcY7ntgBwQeE\\\\\\\/9PvpXVyNdzfoG9lWLoj0drQS\\\\\\\/rtcD6g40yXGBtftablS2aTkyvE\\\\\\\/rFcAUH4lk7xMDOIqxCyO8hc+MJ4Y6bdIzYgPCg+2dQQcWg+yVY7UCKkjfXUGBtdOS8+82fLAeYJBaRBqGAiYAon7463BlAuBLIJPbDvgCm4guqgglkBKKgmTUVFbgvAyI21oBuEycZNggy70svnH9sDXyY6lanwHOQOiwlj\\\\\\\/p4G452Gx\\\\\\\/1T\\\\\\\/W7z\\\\\\\/3DZR7DJpe\\\\\\\/AKt3ta9JCLI6sgVj6ByT+Lw8lwEiGs2oR\\\\\\\/2g5QnAtqO39jXDCWA8QnF8fg4vZCZ4CNnOCVuukrfQA3qqNzsvWiE+D0\\\\\\\/8xnLv5ayrDYhpAjK0jBFw5XOBjZzErskQsKV8my8R0We\\\\\\\/\\\\\\\/oR+6hfQGaqAPJ1FnTNqkwGsys6+AjHGX9uDIGywvId1sw2ByoW+LtIavSq2U7x2\\\\\\\/lWRT93L2tL7Q17194JoUuz+0ve74srdNsROW+uh3BKAWDCxwYjcy2BDND89otNstIbh+9PlLdTeMAmTpVMhZCUGo4Ahk6JcbdMMMaYBAihr8X\\\\\\\/1+T01diq8pIX75prBRSKTwspXxzJ8ftajvMAPfysOyFa6Y2ZxjOBcaoyXKxQt9XuRoem9VGlLE2v7S0\\\\\\\/j6ou6JAyttVS1LbpCCDmq6sX7L4eqjLHHH5b087k0OVSW45VtdHsSZTSH7XM2BgO58Pn15lUSINcvZS00gMik0KiMVlUyPTWJQ81jWrxJaRmHpV7\\\\\\\/Rv4dk7qXV1VlFXj+v5iidnfig6cEu4ZLayDFOLQjmDohMQ8kugReaWNuAXNW8gbCidvVrJ9RIjzs5KAm1bPGCccuHWGUzofsZevAWbmvLC0+T2RgmKCKNDH4EG761bXgYcM89qHzVX+z2kpMIi6iTyNcLyfdLPkyE8dUZTmElEu2HW1gpVE31bso2a1fHKIJWEj2cjcPqc5pQalNkAUjAHn7cdW1n8x7QOci0ahktDNp8qIEED4hDZ2Co3KwuvxACf7VaHVKyResJIrdHjv43lYxQ2rRRPjxphKSlUSmzdMFfZ+qbUNbgUKbv\\\\\\\/PYlacgl6C7xUtIpD9rcEyZI1JHoYpThQCsjqGLSypdht9TChoUz0L65qstT5vp3SV+Bm3OlXwj3EYHsfAd6mhxaEfdrRGWQZGttdxJKKfQJvgJT2YyHTXvkptnyijglLknVa\\\\\\\/s0jUxNgjvo7ZaOn1h\\\\\\\/S0g2NAUzcBL83yrylpE+UIQpEmxBHHVlp2tYouRQjx8VPVbyv2urUhChJ4leQxWN4JS668nYlRNVeBYpQb0pd0PFxyuREL3tVyQbqtwTknXqqd+xL4WovdQlrneEdhQbewgYX6i3Hqcwm07vA4U4M0\\\\\\\/TA7JZSKQC5ECfcZbtBCu4WFwu\\\\\\\/yi6S1FXxnq3au2ZVYv8C8+ZzQr0yFRRP\\\\\\\/vbGT9OGtURS1VqHQJemJbadwIyfjall2eCCrFnDAf4HbLHQsfXL3J+IriYauVlQoYc5Iisi8MtMJOjYRlWSBLNPeNQ9T\\\\\\\/LcZAkHytQngGWOfbFl8N7Nsm1wo1eYhuEIPs22zW7s5CKPGtqrQLxkZQhxNH6rVa5TmhwoL00uMce7FvQUvvHqOBjstA+aWLPyTvdTaWdqnSXlstusq\\\\\\\/yPSsJS8d45rqZl13HphJu+WE7jFmyfQyc9qQiZdnguRrDj9z\\\\\\\/ScYYwxY+cf7uhDAyD6ugePRDTzP2Blo9whvbn720qh7FRXbtXJ3sVE+E5yNi8VgKjPFgq1XqzPZ3k6mPqjR7QmKVHmbR+pnvlJp4NBZD3sNuJRkcDLrdiNMzU\\\\\\\/rcnEvzCRpo0Rch8ne2dIsqc16megTRv4zQpcdZCZbCjDCpb55TLAI0FLfhYZy70swbN1a9gP90CdZs3UruUbfTty7tuxdP39mhXmIitfd3TrcKCV1qi\\\\\\\/UY\\\\\\\/ikC\\\\\\\/BtQ\\\\\\\/JJ+BCKvRJHfuD6GjSq\\\\\\\/fkWu\\\\\\\/8YM4FyehoHUupzdrWJHvYN0R89xTbhqdOwftrybISyjl4LbBNplj4n+N2MEZQfZ9wDYTPuSOioMaraerv6I3+UMSeodNZk0N+HW\\\\\\\/ZC4hUxuPtKegHLShnnG4A3hlyQx1vJEJUqAXjnCIwqKys\\\\\\\/algJiw0FreKcwtxJe\\\\\\\/238R1vrw99OwOgIsZeQqW3G3CLUC8dlsSqMKG+o9VNwUkr\\\\\\\/V4PZttSKS6s6qN9o5zk6\\\\\\\/oGyB5TFHU9Sj8MaMRPND4W3EmuzTRWqhVlefeDfKHw7m+Q7BCWMXdb5dWYKIoOPHPf2BlzH1wO0HiC5j+e3k9v07es4DtSRm8H6hzXKSFiw4PMJ6KwaSLWuIcrwCx4wRBn9CKv+QGf9a3P2+EqPV2S+Dqzp4a9uCtvj3OCd\\\\\\\/JwuO7zTeqgmD+QezZ5nXxreKl0ruBQKFgTIC2wtVsB5o0QoNGE\\\\\\\/dtks1byxllU+ZTCLEwjAoO00JBHFmK\\\\\\\/802kML6N+RZIZYU4gxN0BPfc+2A0CvQRBc5l\\\\\\\/E\\\\\\\/4WgT+zUEbE7ENcXE12jkxcs612h33dLJ1YDbYOlt+6+tiu5NHDcBHFegDO1nbNYe6SCNrRmhxThcLY9w3Y2UePBZWJGUcv+Zi2T8bCVj8jspltzBEnWZlYq5349O8HGLyGjrE2oWpfHC1un\\\\\\\/yszN49DvwKUhPN+XKfeN9TgQhpfs52oHSW0lRg6SEakBEjLCggjhF4NhbC5PaAHtF+K++J2AChf64IwhjxEqKuzDAkFLdHiuRh3zAeNSNBgtSAofxr9qPIUhR\\\\\\\/vgWsb1MawZ74\\\\\\\/h8tqqS8e43\\\\\\\/6cF4RBHm3fqGvgDnpmhLoA7HxP6fy7cBlW6ks11WjK80dgK4g+uPWInGlF0jkCQ8S\\\\\\\/twVz2LKhqVObG+er2KVT04Ai6WdDFJKqt0RQvuMgGAvkO3hqLBaYktiNVjOvBHvM6LkfHAM0iYclMAPtz4f8Jgl2MBfr\\\\\\\/L\\\\\\\/ircJ+2oaY3m\\\\\\\/O1T5tI4bny7sDpx5vUmDFIyTrZOyIr2xs9yZOMmAEF8jYrJoq5BxSDbkznM7UH6iwqAk9+1ABhL8XVCxUGPFLb6MNnUG0KJ0eSYDHIEvUW6KIm\\\\\\\/QH1JhqZGlkAYE1AKVs+fb5emrLons4eYYka8TvmHoz4+YEDr8Q7A7rxm7kIigpAfxNeHbb4TnHH1Vz1yh42iVearqPuZxnJLpSlS465Sn9LGyvCGgCUYkSQWqKa+hqpkApLiGq5TC3M0AWAhf6KAwUdMbLPH3yl3qNqwwQAhCPiqsQfTKIxqPvjI40rjO9lROzpQi8gg5eAQ+lWJ3pRE7uqxrwT2mlopQSM1MpJMGFc8ievsGh+YNG2Lv4jIb4gJOONTk9I2QZzH1r0+4PVGMV8e0vczLfEfjUKEZ6TbFzopbZNARQEIOgnlpotT+4Ucz64w06yl37M0Yd8NB0kvbzPvhT0imkHM8PGOqBul4vj58fz8TGyYFq5BNxXjDXhib7kSYbqzEi9ea8YOTVAXxynfdpTEWN4t6Atxq+if39KxFM8ZM6QNOS84DOAt0v3Hu5xbvZ2TRVSyMNRhtD+Zr4vhGuCDV7yumVIp7HeARWeOugoJAAW+XogC2qMFf7tMramWjSuFYKTY3CyH1nFcH7R5wQY+u+3O6zGdyY7jqnPux8YydCgjyItRYYOj3NssTTrCvJspQFnxgrpcXepC5TjOUoXZA\\\\\\\/OdGJx4CarGDGKkQSybtJTP\\\\\\\/6HJNgVoAZaH\\\\\\\/8HJ0IeR8u6gkpb+gXM9bl4VcIG7vs+vCoyvm6xt0caiL1cugnp4HazYO7XadQYygiG0ezW5sJ9eZToaO7s4kcvdAV5f4E4qMcut6\\\\\\\/VW\\\\\\\/qUPUN4NGfeMSJMGPeqPFNrKTJslOYakl\\\\\\\/eVjflEohXm\\\\\\\/oFYFq3r\\\\\\\/WctOt3oJG7Kv11xLSOzvD18s85TWvbz90TsqDLS8ouku1k\\\\\\\/hKFwQGPNm4qBLL5MTRDlux1oiFZwrjd+Einu\\\\\\\/qgwiDDlWZc5gPK9lEPShlpuNdt4Ycu4P0Dv1Dujold2TMm9qThTsmwfBdapk+\\\\\\\/c8feQxa\\\\\\\/C80Lrl1WsVKUuVSXp1LgnGYkmDNnLu7Qq7LZNnoa1Of6DKQMbsQKMkaBi9KhuggcTbZal7ytifakG1bnqKp1U4KMveIo1XVADvpPkezmq3KpuBI8pKzNcc9\\\\\\\/0xex\\\\\\\/9kLN6idDTzNYOJ\\\\\\\/ud2O6XqYZ3ZCF5OeWbC86N92cvGvUHBZ9X6hufNedTwb64wP0HRQ3bD7pryt5XY5WXdeNRLt70ObANsVY2XuSm9yE+l01iIquj9WL5wa4q7Qx1vnB+\\\\\\\/VjSWXV9ohPU+QeZcXx4ZjQJgXxRugi8MmP9LMI4Met8pEuKF1cjBD9a\\\\\\\/+mnFyZTACBp6v9a5n4oJOMwQn1X41umRB1M4Yf0casd66mdq\\\\\\\/mC\\\\\\\/OrOt\\\\\\\/3tRoBVrf57bvJYfL2io1oec8P9\\\\\\\/gJ\\\\\\\/z8zyvcDJ76Imbt13PcCvT4m0g16aRC8+KOvZJ4oCwpybdW9P91jfm2BYgdPv3CnhKpLbYNz3F48XSch6VBLdXv\\\\\\\/f+7YoaovVGBRb4Z3IEH0oW6g5rHurLXIKHdqk4XqPXwsOoXPr4SE9T5LYsSvjYn+j03KZrtYzkNqV1iZDxKJP+mQTLhwcveKTEc7GA6UbswLziAukdGhswOTXw\\\\\\\/A1F\\\\\\\/e6JyYibe7rHBarh+TJsO0rMt50Zc8vXVmt4lpYT1M+riqtIyVawqYNZPe34pwlWtGlFXGj4fszP0czS+PnNsGmcaw\\\\\\\/IrhgodHBRINRZpuUssa8T\\\\\\\/5ZFOX74HJvjogDkBLaBmbu8LE4DXCT9tbdzwLWZAaqgMsmwl1wEkBZc0ullGT1QKlkV42TVVaXkMBGcFXR+QG0zv61muxfVyUHHqsMSGQ5nFIuMCZNNKLvGS3Ta\\\\\\\/mx8ctnFPZvMeYrHD2pEV2G3C0Kmps5FfAwhITiAXD1tPuR7hr\\\\\\\/N9yVgGO1Y5E\\\\\\\/mC7EkatXuPUzrPt05Pn1T3+8BUsyDTiUJgo\\\\\\\/S2wHtwhXaCrqALdzq7gtb0UJASt5DTsx8gyYOlJU8GjUYuVBvZnDsUlqJsEy20\\\\\\\/hr2d+GW4j5UXVcYVO8FYkL6Qm4EsgqpnSsA1ZvZlvrBFvpxyIKvU9qm+694hQHJwsRze8lsaFP9Oc5vgMD6UDxwJrLZX\\\\\\\/aCJ2VWoNK4NuRcU4gQPPgz83DtvsupxrSkAjtbIjOoAvSHmDptMpIVWEjx1jlv5XWGCtX+Ao8Qoy9dxmKP+rTxDwr+xdpsMEFqM+jByIliO5UqpXgElvpK2U9qVoFvYP0OCn\\\\\\\/gRZ9gX5Y2Hs+41rMWD9VSfxm99726VegwOeMPuiuI+sfVpzdYOR\\\\\\\/2bXDONlF2IVgtsNc4BoAt6UHNawdUChDw68dUICIZYQKFvxlK045RbMyjwMRR6OTjGp1ppGokysmJV6Z7bHQzRXTf6imiFet57SbvNCZnMtfiDXOFUzhnbg4zqLYahjs6kvRi3x2iiXn8Dh4N3vsa2\\\\\\\/y\\\\\\\/\\\\\\\/EPx1naXbIPdBq+RPe6vwVZGHqIvnecPfWTDFOiUfu00u+mofms8X8uoTDlRKk5ScfAxWXT+PnEg3qmknkzR9qBc9Fj1XkFGzu7+rDe2QnWmjqmnxBd4p+3OKItQoVb9+M7z2v0pc5dyJ8LjohCpl0M9SZ+B73zgbnAYxiSZfbSZDLBYCiTmjUGRZk4wbmn\\\\\\\/4rmcBz+iqczH0IhPcj97K9mYTbOM8A30\\\\\\\/7QYIf+QPw63kFib3qICuJeeN5r37Y\\\\\\\/QpfgFY5vKpYPu4qsd\\\\\\\/1UOqvG96X9fgoXZShg36gADwgn5ScnRDJHIZBHrWARyoynl0ubpjqqwoLryn\\\\\\\/2r9qoAvYj6u1sLiey9FQKbjihYKPt6\\\\\\\/nELjqYCX6AY7SyN2L\\\\\\\/wuVDllhL8GfNwUvKEzeVy2fChEtcluzoPnPLF4SYm+hNqqD3tBIsSf1JPSG61wnBio96TYpfTGZYLO9OHLp+qSe4kLjHGtOLbM9VITfdOAzHTG7rxI3FAuz2c2xwn9w2qFnuM4MKnG5XugQ5dgA5aEB5e9lKMuDk6T6zYDws7cB1WbSZ1jFWmONyCDH0HxxeLQAJbxjDnCLXUV9S5ufJpvukcBA9mMDeykBEZjsu0UoBOT48sz45hD51JtYYfsTkODEq\\\\\\\/SnPswBgvoDZAT1mHwinEW0eMnBS67px5ZWR460nOGBTqXcwUpw+v9NW6bMXzZVHx5SEhYx+q+luBuiOPbvopGPkdYp6GdAxQW2lwDgdElUvmIYi0UPREe70tjTggQNzFYUtF3W\\\\\\\/kprIRqkkVHybgiMq\\\\\\\/rmahk+rlhFzjet5rx0JbjeIqk2YP0Vl5UPwBgmMwzO2xxB2essJwuTmugySsGPN2fHs3HzimAogjgoskK8LiyCjiXUXNEHcmYAd+r3M3zEw7uCoSsDVBqRelp7sAZgjX9kskbNvB9pFN7RgNZiv5hrCzt1Ann+2yrnPgcxr\\\\\\\/bVclfdmIcmozgJe9Thu44FWDCayKqJkoqIwu+EMaAPkhynmuBWJQ4Ca+tthKTimgFoc\\\\\\\/JZ6qszMMcq4kHzNbU9JhHoDWCXqy6xVQV\\\\\\\/sm0Zw0cRpCUD0jh2liJylX\\\\\\\/rwWpkLyy4UpGCCizG6NNy\\\\\\\/whNyNymWF7ZjXA\\\\\\\/ipFibTtmNduQi4TUiHbZn1d6RLncbxerlYFK5e6JZ8pU7zI\\\\\\\/k4yY4vd8V9FK9L0xfseqjL4+177kK+wf8TUpW9w26D3IScwaTyAovT+JA4Cosc62PfaZkDcTIzM6DhKfTxB5moGVvDnurkz0kLrnm0TNkajc7D6YEfb8d7F5viyPxxK4ZcCXy99MqaQL11ltRJAd8VdNf7sQ9rklkSgeYqKTiRaSjSX0QvR8+WTTRtqga5t+SJmMcERtb5ni1exCl+AELKRvS6lJM5KKRA2Gd\\\\\\\/JJSlASFxmbFU7uITfc5sjLu0bxsGSFHVeog8yzMNSbg9lPBsbOAqPvho1QlcVYwEvcYxGBs3NT32Gc6tNfxSiorWqAe3YEb3IntZkF4kOR79xnxU8zfdQoZSnvM1Vf2h1xHkxYCHGAetAGbOMnJZta02LcFPDKmWae69q2MdHJ5\\\\\\\/n04gPP7oXgVMD9QMoCIPpaZ2F3ok2mKEeETQ294JllNOLbrHfYkn1thU5TPrIkP5sc9gZv05D\\\\\\\/c9Id95aruzd2rx4STXP1WjU4ZNkEjYy5owHGbB859cqmn91Fl\\\\\\\/uQdFT3n5IwowrOjddnT1wcOwy4uaI+sA063cIlofIyh58JWQ5LOhh02xIqf9FC5q0iniJja1ZHTlpFKXY0MABMbSTn7eMha5pNUL+Bos2k6GsZbGXB5z8hUcvrnNLC9X5rSCKMJG8IU+n0AvyQ1Pa\\\\\\\/SppH7M8MlJIq38p34bdWJDq223AdV7sCmI7e4M7y38dvg2VkHPdiC4NwJisSOEoMUBIA03P108V3e315dpkW+k5lWoLoGGXJxWW6yagmdOIjeewyP9Txy0iepBnk8PuufwIAmRDKe6SgoRJG86oe\\\\\\\/FmXEQdjbIpba\\\\\\\/DcGeiEDKmYA8JKZLQR9vLh9UQPzhOSEPPPAqPFJUqSah7JnsLtsgloavMipa052A7VDCctI22xhLT0c\\\\\\\/GIHlj9FvX8h2Ijt\\\\\\\/tZrd+xtnLw1v0xmteS67n1lDcR\\\\\\\/07JHabGmBxHPO3iC72oRV1Swood2dmRyxcOY3+HDYdmCbQbD79vieDGtIEZ\\\\\\\/oee7mRFxNKMVVEvpEkOkVcLzxhJVKUeGJKA9GUKZPLzAY5zOnm2sFpnUVpQkGwxw\\\\\\\/6mx1LLxtHp9cFwlZfN9+v5Y6nRmlQONPdVANIxQ0GezLa1aLfrYX7J3Rf4moGMTzGJ+EENAhMWukU0ysYNBgK2y5tIpTw5eYjxGSBFMzIvXODw+DJRVUfAQR2mhCYH25ZZjgoAeKFTs1xmOydf3c\\\\\\\/b7BqwYFbqCVc5cV8wJFyGqEKq34Z45vI6qnw9RtUHA0iQScMD90q7g3i7iC8XVbALkiw2vOfLKziyaSEa26xKevyavLtS63De8gTJ9EsKJ5jzsSScobEG1Jie\\\\\\\/z7vnMyTyPqKf6c8avpbi0GHxxrg2tP8x6bumstxVz6HbqdzS1ZW47PjuVlIUov+2VVS0xmpjDj0fiokCg5LXL0UbRlNbVGRCTI86T6g8m+1ay2eRhvTcFuIafIJmYVzrK2I6R\\\\\\\/Ce3lEuelweATcO0lY\\\\\\\/fmum6mZHpuo6I57QTn+bGmiHzkdgf1RElm9o4DmimEV4kphR09LCmFRgSUclAICMqSLUefTAA0g0jternRtIGO3lCSbSimiLtYlQiR+9g\\\\\\\/yuP3gdSZzLoSKcLhAoJ4x4PPuIYUio5CT2PFwBjiQ4Bmbk5yBjVv\\\\\\\/tKADvWdWA8ofOrvp6klttljqxeCsvc9kgpe+eUQ\\\\\\\/22NCCkwkZ1D\\\\\\\/YoQiCykaO4FrG4qQF7tkaEyOWnDTHKMztF+W7aR0Fd8Xz2szWH34CKJ9clLMc3i85bOxw7fERFM3dEckmenuVdAwTb\\\\\\\/svyFkxQnOqZ\\\\\\\/VPZz5IpZYYVaZGVR9k\\\\\\\/gQ2x9wtBFajvuPNJOInfgjpnmVJHOtIEdcr5X+QtYqXQH\\\\\\\/HG6Or\\\\\\\/uAtEZ+kjslJny6o9ypRRiVn4e2n56RRi5V\\\\\\\/95K9DbvyXpA9gouDRde2vF8HEFZUzjAnoTXzqRoDTMa4zaCcogKFuPuOjhN9YBZD2puwcdkoO61wE8gWe495ncXSdP7wRDU3UbS3SxN9at5MbeHLFgWoWl\\\\\\\/bRSsUzT74Eg0XsjkymQq1GZlc2czQJgp4YTE7Dl0W5FB9zW4EhACQum7tuNLEhl725WzbczuEK0ig076\\\\\\\/xE0gvf1DFToeGC4mHS3qZCpl4vbyASF\\\\\\\/qPFu6AlMNNoyIZHz7bWioybPQ0ZD+MsR3C\\\\\\\/TaG0CHhV27C7ywuzAZTYh4bS5L2egzbl4OTdMuZANC\\\\\\\/BUBAYWY+6j1j8G1IpxQ5BKMsaszi\\\\\\\/36NGLvmzLLqYFU5EWTKhZrH2k0dJ8WW16YlE1+MflQwLypbhbSWefSc7UZ6YLs\\\\\\\/TxjYNwlwAssFpZZpEhpEFGauW71M7OzNpyU4pMClTluwwZtdIRGwi9tUlt1M0ko4l3ig8nKFqJA4\\\\\\\/gCeoM0y\\\\\\\/QFVbywgOJCDHvJUeoyY2iWU3FI8GXA+TSSAOkS72KVX1rSACT7e6wWSqpl39xKuyIuxoq4dl7cswFJQJSlq4yqzkc3XGXpEtfOGT24QIlwRcisv2l3yGes56gOUAjtpUtUVJWBUtmQW2bue5bBXqnIwTGgHO+OD1f3MSEYHkmI98DW22UytE3EVxNww\\\\\\\/2nqhgQ60r2knSUOCSqabZ0\\\\\\\/hmOqL+MRcrgMw\\\\\\\/njYqACB3p\\\\\\\/BdNPd5P5Iuk19\\\\\\\/7PpyV\\\\\\\/3\\\\\\\/Kb4i7hTFbKWhMIL5UusMJYrD8qDhosrhrIUYRkpXstKSaKHCOatyhruMjCc9\\\\\\\/Z6\\\\\\\/9Bju8bIa\\\\\\\/tyK7sJrRGztaUua0QlpSzUIDA6mr0JNR5YXAHOJqJudWJeIZdE73dkfUwjStD7+ROxboRSgL66LhkN6\\\\\\\/MM6009PY21O2I3gCxZQDO2DFUUe+GXHneyauxX980IUyf91sjKwxw5ObjdgTV5a4QynS6Tp0IFfoxR3nbJnhwjsS\\\\\\\/THELsmgNM3aL2ojGuSynjujkVpTXS9V0mrRg5Z2JlwWYSd6VFNp01lOZqdpvFEkIeTbLbXH9DGem2tMU44Fch312srfWit6Y0Pls65y2GOpp8Wq74jdKg6vlm0GCgCnTjY\\\\\\\/rWONu20NH31BGkebwE+s4E3iIdYSLnQ3GVoif6LnJV\\\\\\\/fz2\\\\\\\/R1wcWH5qIbQihYIqOi6YZ94SHMA+vr0YZPCV4QHKa+d7BxDVUy+RkextjQKz8jmqmxh\\\\\\\/vEbbGwXiW\\\\\\\/I+f1BEd3YhZigJo1ctFeexRVhxY3k5uFFgLkBMdAk4x+sQt5d0MtAhEBmDU7ynAT9cXH3bh1+WTzJb2ZfSlkM5qqCIRdtlE+TCCycIap4hKQpk6No1bhsvdHmbO7Fi5Eaiyx689a5gWjiReKt73TPYeHCLOHvxb04U702Rb508JvqCQAlpMFWeS\\\\\\\/Uk8otmGAJ9z4lOc6ynjA99ZaE5QnpHtpnpk1L5Tr\\\\\\\/eS6MiJDfKqNiSZ4KrOxHG7AkimnW+Enihf6hHpt7wRniBfVb+chqA\\\\\\\/tJ02u4Wup0+glMN50HZ\\\\\\\/Rvvxm80+4nIr5KmtI6evNqIBap77s0xatF5q6tyvFNlfwyJiASywOzH1xsrp4lscQxNp\\\\\\\/1R1O6AjlwqmEekqI6z9C7DmkApKMp0ewW+dKW\\\\\\\/bcHuwkxOAUT7L762JOeGwgPwx6OsQjGBAt1CRPwPO4zn8ZsBsQ7\\\\\\\/Bu7ILMR47AWqIfP1tyn3JiyEMVCaBzbvhUTuwx9IkcUwOOexakp\\\\\\\/yD1SyD6I\\\\\\\/Akr8Das1BywJ+aA7tGNaE\\\\\\\/5cBGJq4u54P7lCKoyk6HzVMxDzB7rVQSZAPS\\\\\\\/EsmDfUOqd1mKIFT6959uFWktx0ltSRvubQyzJNn7SeRXiPzZlKpKdjZJRsxYlXeAnEfplzlzIcW2RHhOJP5GUGfEA1L0XYi7vEThqt0jLGSwPLaw4RvhANHIfHRLVaxx7bIB8f7yDh7yfCZf3RZi+ix+kLYxlNyQ0qxp30D4Qbb3pB5XNKK6B+hHaEgnINUN6N3+erLlco27W7RnRLUVbE3GrVT+vRheM89Cqd4X1jIqGUk8HK4ujYRXHVN1AzSiFUtfQ9dknSR4X0etOu73LAawJYr7eCqPjjiWyo7PE6Rqi70WIXrBlPwn\\\\\\\/5NXVgWapBMn+ikQFGoB\\\\\\\/dtEarySvhB7crT38CGBHrIhHSr5SZZY1UqrN9aseQvMb5dPjD2es8AE83znmoBWx3I2GpF6z+xAHKW0NmbpSiPLMOWjANGoJSwLzrZ6slZFOHDvqeLPemLVvhI1IOhxZUXlOMDxqc0v5aKPOaiHWoRgxlpRgWaQC6qVsnaw7yg6VuP7Q050kCdnM057yfxn6xTVG\\\\\\\/maWWy\\\\\\\/NA7bMplU3X+FU2x\\\\\\\/JVl8+YocDs7qOYnzPgi7MfmDc+DpIhEv+nRJucBG+t+MapfRnrcOM+wEFgwOEvenLeZIbuV3LobRg2b2cJ2+QjeveWZgrtya19JgDMKv65h7936z\\\\\\\/HcgYa+pktK2nYQ9QUYAj799K5WQmkUg8oPY7m2MtK2TmmaZ338Dm\\\\\\\/S\\\\\\\/wvyNxR2mTCsxEtovoUM0e13XW0x581OFewp9BbFTV\\\\\\\/t6O0jlYhr8UrtscVM\\\\\\\/9IlAt\\\\\\\/\\\\\\\/u9D\\\\\\\/qq1\\\\\\\/zN+cHWSTmU9J3MVcmKx0fg\\\\\\\/bCKhHkVZdnluOkH7OpZL\\\\\\\/TE5tzi5ddhAsp3YNJtcTgZTJhMp6WhCFOUD6qUDLMR2c91dYkXAlB3eQf7Dfy6j+ExS7uqkFsY6n5UJ7opEI7vniCZKoP6b7yT\\\\\\\/HliWxOTc3tPGY1\\\\\\\/wo4Qz0nw6Q77xvOZFS6qP8CWTKCIQde5DxnqA8twD\\\\\\\/iFHz2blhVYXKJbhaC\\\\\\\/QTZSLWhDWL\\\\\\\/1y7391BMiBSI05L1Yul62qorppuRY7iN6oXExQToVt4guR5il9\\\\\\\/u5rECZm2Uum+vJxTJr9L4Kig0eVxxlU970SKoq\\\\\\\/N7yQ0WuXbD46LivDt+L1V8sgxO87aWWEsVPqAVL9tXTBXEkPEMtNnljBQWNMy4jzi2sI1TLXDrE+MQB9gNg91m196LT1toGVN9OBfKDNAJv16ru0i0We2W+h3qDZmLSSXUyl7JIiB20KQG3rrLzFbJyuLkRF+sziGY+rRVt+a+UiA59GAbMSaPKgC7zWQAhOm8097Yy8lkshvWPTaB086zIOQ54nC0tPOjDy2D03prnakp4wBoOTFSivOARdRnBnKTmuNt71H+MYhiRApJqQ6G+arD14oPgp3e6YMWEgsWDpEVcsYNd+ydL46tJ5vKucIBQoOW0nbHRvLglmmqN4kCuoC0G8P9rOiSV\\\\\\\/vdN5MMYRaXYEk3gvxwJMM1MTNW5PZW6bML3IBHQNv07iglCaSxIQ5IrLDBCeA8oyoNOg4O5JcvGSOdoLW7sWEPNANkoOwK00Ij8j1yuL0KAC2JJ8K5r++x6IjFkLWzazd9AYYWKoODtGzj5VqzyghgX8vKJTMmXQduWm+RQnmtb0eIqwPg0\\\\\\\/K01G86pRydUXgDGeQozWnPFPekFSruuXzrYbvdEsUgVZ0kTXSXOfgg40xN63rNYm1yPyLg5nwVubpVCKu+tCHqYVuqkzI8Od6JlEP80r5SqTRLByhnuoMO1lVJ+v+Vv2OheuxrYbJTnNCd0TVVcOIOLAMhatHXao49xl5m29TJ29oGfUEVAkwV3ZwYTBw12SIwBdS3zQVGCChpET1lqvIKlEiW27jkwR7iRXc2fD7z6up7Nq97mReXGc41FBg0MGg9juH8o04o0HDiNOf6EWbUFLVPdMmpuBFFdjX\\\\\\\/p2pe2wZzhBSUw1vdVKGb+7FlYDG5U6g5R7Pu2Ff3GPLr9S\\\\\\\/JDAJQ1mb8j+eQ+nY3ikAXc\\\\\\\/7tFdi+ePWiLVqBVF0UkRHohD1rAkUj1XpBcoXG2u54YGjyGDtkyYdnAwLXRdbhAn+aSDMAZo+NXk0ZLfzl1q+KxMZ3uiZD3GLm3WHrn4KT6bX2vmVyCLY9tLP6yJEa3fH6Kaz57XMQKtC2FIAtz6hhPoczv6pRPZcJ2021kDezeVeSfqpGjW+PvBOvhql+rvTJ+qeAQM6Ub54Oe8KJmGuNWpeZ9\\\\\\\/\\\\\\\/RjbiE6gCnUARHGTME5SK7e3H9y5IQZgunm1HKmWmhEbvOi5CviBYgM7w9fmYBQBEVokuk+hL5nDKjVTf+rMQv3v0eroTe65rpGTVRMAzg0OtPS72KkN+c7Snp7tEyNtSWsKN56OsB9allUvakC1XD5t4tkoEf1T4CUKlmzBVA0eJMZgvWraKHS9Fbk330SNeXC+DU5l4LiBBymaO6aYPZtf+J\\\\\\\/C0xalHAP2+8TZzlpzTaZulKwhFzHhYb70EGU91tEojkl34RPULrmFHkya+RIzaiuVEvYGIHrQOxxQ5Hy9cK2PJ1NcvwQqAjR4OZc2RJIaTdCyUwbjBpjUT9VaHo2ymE83loRO9nXNZLrMjRXGVXQ661OBmYSPtweP7SVnQAduQDWMh7wlO7eFphWCaYHZOlaIIcq3Z4RL1QxkqLhIrYIvyuYQztzKU+8Xo1HFb7vy36IL+tphs7EuKadDUapHsO9L8rw\\\\\\\/XbO1O6x2NCvWKz8I68cNOaCLuwfMowcHo5zkeEB6HrdfdMOjemb3qFN+FbEsHOuaxdWcHd45vS5mrm8+L2LEbL1hCgN6tRvawC0OUCamH5LS4RoF7GvT\\\\\\\/p7pnCTmPGBNfH\\\\\\\/OQ7Ryn23Q1+Psiv5RSG5rPZ3BLCFGExhO4sIOs6C46WN1wem6AzmM7uFDg1Yg+gCbAH7e935FHQ8NjSd\\\\\\\/Muv+nOSkwKe1MdbRo2hvnORBOEXmWRlEHZUbFuXidiuctd4pdhcTOeD14ELoDxtm0TzAxBekzbTYAX8i8OfYQf60M\\\\\\\/56\\\\\\\/Qov+wHCQV9+7FUsxN2NqGc1mCKH2MZlljnawnKwMBpRnc3BcnZi8AzcU5LCoAHf3fC5GOCEDshvWV1fbxm+wIf+MgAxlmpKfLd\\\\\\\/WbdaGVaq2XRSRrAy6XSVfj5S6uC9ou0K0\\\\\\\/DlerAj1LQxis2wRQbU21wz00EebUpxzXLrrQ4hYGRlksWvVAY+g4fplcDM+sl9DSdnZ\\\\\\\/fMivarwOI13YZRRJ\\\\\\\/34C9mfP4SEAafxxerc1vBpLg4TbfkgDA92IxHdOropeJBFDMASMiXFS5ntaXhoOO7xG2JlbVnXhI9VC95tN86jBT1DQcWfV2VZUKE\\\\\\\/sonjgYswGw+SAM5mrt8\\\\\\\/C9AJAU++haJQUAvoMqAkEwItrEVQwbcVpYnJrEM1kCFF3pqQwJ9xGO6f01kHeNT8J+pxPqeRD9O35R7GLl6lsFac\\\\\\\/CqLJySScsAvRrgvTnTWlH7jHUMY7pVMJZCY906lorlrgShrL1bdaBBqYx8IG6dm2mN\\\\\\\/b+KIKqb+\\\\\\\/LYmZlo5TJXs8uR3TVNKhFxKpLzimdIHYPPplRJNVO99M\\\\\\\/SvG3Pz\\\\\\\/5MW6UOZVvNcE8qHS9fzCET4WHTSrQX+0fIPC+JdWldkTn1pfXdsCikr1PuXD\\\\\\\/c38\\\\\\\/xrfP97ylK08MMjv0Qh9eaQfSFECTjMK9tEm5SDnZJsLBj+6cgrDvNsyq\\\\\\\/LKT0b\\\\\\\/TtpQGZAt8ga17hP0nwlAarqKhqiElVnibwF5SwIlmjDnOAHU6LKt\\\\\\\/Tgsb8mr8rmVDAtMceF1JMA0VBVLfe\\\\\\\/x3c6iR\\\\\\\/NecmHghb0v3b6DmVf4\\\\\\\/tySEPGHMA48gqm7qOuLMRxNAnzSt7Zm97kT4iQf8pC6CZF4Zk+949+QSgCqP\\\\\\\/c0Tp7CwVr+dQTsH\\\\\\\/3+SdrlZmec31iiyXXelq91tpI9odMF\\\\\\\/8RghKeQmeOQOnUOKjTbU5bM7LqmeMjJrnRAkRBdQjRrT+fXYBSY+lv7UhJCujmXsV8wfZHnqvM1wX+F1hQHeSa6yMsouJh\\\\\\\/iKBiM85ADWmlaxmFe5uD9Jnbzw+wCV1Rl2bD4A5isQF1gg52gGXFVZ\\\\\\\/wpPRg\\\\\\\/sv2HDkgU+txEOtSCrqUNzyVZ8yKz8a\\\\\\\/Wj3swOaaBuKAyOAG4Bm2z3fRhpfYa4N9Rf\\\\\\\/unMCoFqfhlcXrHdAnSdMfFJ1L2r7kEvWBLksdk3\\\\\\\/SYpzkrJ2GWpL4Y+nDXQxJy8Nl\\\\\\\/xBA3QHqEuU8yilrjSjgdseX9OxFWRZnxw048VmonogsmzoPdkcKcFVV0GbE1+KeiWnOC\\\\\\\/vR0g8vcMG0C68\\\\\\\/mATapuTqd8dii5BsWIEJuwj3xk4JHRdDbVp\\\\\\\/+3o+YKAeauyOvv2e\\\\\\\/Fpz5z1JepfJGFlEAvQ1NpuN+xWBBvP9OjL+hNRiw6Lh4fpewuJ+3Gc5hJCs798QPc1ImnZ6r\\\\\\\/2JW5CFJsUXtJevVFU0DKtIFlNuQw+DCjsVchQIdXqw5LD9brTNyDy4obmHhqLRJu4FDUZyBdgbncOTCwAZmjkApnaKkfkvpfb\\\\\\\/6iAD82m7laD7pI7xsQuk6DwsTLtG77uoV16WsIWUuyBUUAUPdJDixDOQJLYJivWefmv93ly8MTWpPPv1uVX1YUPIoJnBLKPIBErsEX4YUNYaQHJ9DtKYuGBh9zDdrf2jOOqSs4hM3Mojwl7IXO5NZq2JceYltXG7UY7M0nsbdkHnrEGqiGsxutXG8u9AfcBRVzW6AaQbq5xiIV9jCBDZkHusiVd3\\\\\\\/MhGYdDAIqxROdJZ5nhmgz7eM2u9snvMzlmIQU8jXXcRjCESG\\\\\\\/zqRMIAkFKgnw3aCkNlB0GYfjtu9r7OcnW7VRGWyHGzEqed8feJl6nPyVMhLbMH5wNZfJ5FJU2T4yS4Ix4TqDpLuz3j5H\\\\\\\/CRhwl9fbKLWxv1DmLtMxYrOWleIB+RaiuTqndYh\\\\\\\/no6Z1WdAExTY8vtR13Vp8DpR6jh4M4Mq2jy+gbXaT4C+BU8W4ukGQo5FrCwQ9i+jWc1PRh++hasjMv502h6UeTcgRHqKOG6fBTFHjYtOJXqm6xj86ZqxlVdZimUbzefO6iNfOEH0lID6yZ+7tXr6\\\\\\\/OIK\\\\\\\/syK9WY1ZQGL58QfxUvK9IWC87OxqJErrA0amBp0xySDR2cBl9g5ssPPmpsblmxv9PPji0bSSv5LM9Zo5VGeay7qVR3hg5VJL4psszCcPPKgk52NZvj2a3i0qHGezdB6alP9NOlpnqlAso40VT9RJzeMTPdkdxYhcjCYX7PO8UkmQgpIoonoX7X43WPBxawJlEHXczNiz6uY\\\\\\\/r\\\\\\\/6TjhxaAslAZrM02KVZSTkaCsp5rkv0A+NLBZHPxlic6GJ6Ak9\\\\\\\/5qX33QFCHdandO+ilFxtftHJIA5tnBKyx740TXba5YBIXjpKQHnVrg0+lqf3iVmGldtHnJbDvHIWKiWYIdYHUVPVIb5+5WO5K7i12BdgJKOdHvVCqpngMpi5g55MVv27brLtehw1Rr\\\\\\\/LZJNdY+8d6B79f3Moa9miv0C+E9MZw9zPsGFu\\\\\\\/YqOpYgg+RqCDgdNH1RSCpOZuPESZEYIaPpsKqxRefL+3QjmE6SimPFLH9S0GlSu20yqButFaXwj2b8AZ+bZHhUZYtSo0P6bUTAh9ap8O2GHF5xc0rwW3gngOda6\\\\\\\/IdKOyFghzFygYW\\\\\\\/nbR4gu+qoIksxdIWFS9wseayBQtalmJYr7fwwY7pXEsmCz4+QbQbHLJubLowo3PlPjTCU8bL6FSCU6jD9kGmbwULky50xAjWbEYAj5HnxKE8fvdUjYa6riFQE72u\\\\\\\/VN4vklx8kbZRH0SmE2t\\\\\\\/yF+UlZM6+iZmNKH+gZfO2fD\\\\\\\/oPDUT5a0+RT3QSjs8ixd5mv\\\\\\\/S2OOQsLQNLddjMKWZP+ypdopM1SlTa4i3xltza6h5hBtoNBRtMWEW6LYhXT8Uhanp47YIlH1kUzPfHFpfj\\\\\\\/39x+bWSmEX5dfWpKKn2Pf008Vp0HRDz25ONbh\\\\\\\/yQYxtG8EHck6PRCKeVWPzB8\\\\\\\/xkczntFfNNh1C0pQF4H8CyQex01EZY7fmovt3pxiHaHFpEsgJOhchbG4VuG3UquRhDjf+KJIwUbXKRb7dbqQtFRQX6HZLK5UNNLzg7dW+mwBSoDOO5uPKCnPg3Y349yGmWWHRSNuNMsrgyjqT234wBMtmB57mdDXV7785rw9InJY+0uUusRojj96FJz13+DokviOTHmQr+3SdICebCJOTBL6+dxq7rplaRcI8rJa6ddOugbO57dhMzFLqNYiDJPZYyt7twprW35nb\\\\\\\/WXsMmBV7gvq24MVhcw3IDhwVt4k9pLdtmLtZqTCwHbVOjZSRMcN9PZklfASmS6dRjuZ1+eHK\\\\\\\/SnmUx\\\\\\\/AC+JfE+G+WM2MLVhNskj4dGjpF3+Vam6FHng3L94qT8ToNDRWY2rSUhwufZdUpiwBYW1VfdQQvQFBa\\\\\\\/FFrQHi1VJA37VaEk5u6\\\\\\\/VlBXUbyGXclYQM95Z71YWENpZfC4lxb37WVUP2jgkcaJiiA+Er5tr\\\\\\\/6diuscfBTb1QrXsxXxeyQ6ZPB+1O9hgsufXSVU48ggGDUfkYCVz+l2OJAOHvEXb1if7s5\\\\\\\/Qua9PP2kdphPoEqdGLN5atxWd3Sv\\\\\\\/NywfZsF+SZEQ02WmNfGkzlQ27WM\\\\\\\/ltee9xFc7uNEhO3B+fqPX9nGlMBEnisCqWKZMCDm7sxG3rjZzG91SVWnNFi\\\\\\\/NFm1gDkiZGqOFn1NdGtNBPWPIQhEytYXIHxMl456zfbJ8uKWGBjjcYz6vfxB7sACzKpQg407XDKcGkkmVh97dZ6egbs5x\\\\\\\/r1vBGhQTyyhe+tTDcVTlzTYrVcwDqMG9B106PnnF4VSwdwoE1ZTOtryMqkuwYz0KcwVwGCcfoqx2rsFlFgv\\\\\\\/VAEe3sYY+pRsEb204UFgnWgOZ0QhPC\\\\\\\/wTcYJmzRsWzNUXFpCKnaqUU+1OWELu\\\\\\\/h9Rd7NSq5s9ZPVzOlVDAPbpzuXHUBVdOUAo8mw8QYVyTeqdy\\\\\\\/NoujwrYliYXf36cMK0jZWEHGAxPUGw3pmtKKNODILy0Blosswni3ES+6rvwwBsz2hxYufwNiKyiQgSZO5I9gqjwVIMDteVQeB7wyFfW\\\\\\\/L2s+tszeKrIJcZGHWQ0vg6U\\\\\\\/cp+e38Bswml5ai3RklLGOAhFCcH\\\\\\\/w7jc5igh9bfhD1ZywzrkaBpY+MeGm0Fpj9c\\\\\\\/Z2Xsso+NHvsHTbHzhPsdEQqH6m2fPU2qT8hK2nmP3gbjnNdXbs1nl5ErPw+yptV3RWsE\\\\\\\/mzp+0I01fyivmdJxMcZRgDnw5W74KWNkw82GiVwLaMKx6GJ26ZGHlraqXHmUN5O127tyaeS9\\\\\\\/doFcQSJPGndU0xLrhtxX8Za1C28EePMbMWxVdPvNWROhNAqrxlEGaEMMxiZZCDXTWSMeKxWrL7D1AtTdtNH3XSUDe4SBvxMJOCUAQ7HxgUvnYNbjvasyjMRwTkeJ5azY32cKzXk0t7mI6wcKnMS6h9YCFL0mysyUGaHHVATh96JWop7ycqTGRCE+jCr6F88EVe28P1rIpH+fv55UZI5o4K+\\\\\\\/BVS5S0+vD9NXfyA+zb18bN7m7v49y9\\\\\\\/txULrELoNTXqC+vbqOf67bRYdv6pCdBDa\\\\\\\/gHny2sspJkxhXe++weAp8KO4VoAIO+uApKnycHtICazqKUGtD+K6apXzRnYJp0HvViVpfBr1t3jIpNfb\\\\\\\/Qd44KaZEDJPd4r2zkPuwDVTzSsABFK7N25573ecossYkFGVwWaETpLj0\\\\\\\/UlmvVHcjH33UULc8rDcE+nRmlP\\\\\\\/9Uy\\\\\\\/qfQdUShKhOL4nYcaU035wO8CEMMwwfoqsOY0DE2\\\\\\\/yfdWztn1haAFYOvT3D+EMrwnkSUgIejiGk3rm8oOTHJCcbhcW\\\\\\\/GrZ7fBx+n8vZHVp6uCJfb9T0xnbjeYcmauyFWWGMps8nHAip\\\\\\\/\\\\\\\/QyzViGpY3eD99L14T1Hb+snySLffc70Bg2XcC1UdHQw5zWx50inIK2rX88icy2djhAbs+EBeDoWgxhHTK3B4RTggBhSeFrs6WAOq9yMnY+5b4E43c+yJhDDiBNnEIFmG335ymT6YM\\\\\\\/FCxgfWSbcrvqSCcA\\\\\\\/4\\\\\\\/80\\\\\\\/o5hsNBuVa1dueO2hGyFCQ8j9qpEq9PQQL1p8HB+RU7vONfVebAu47KgWyPCijcCAij4Gazc7La1nKpKZBFeUJ\\\\\\\/+UeQ7hP3\\\\\\\/e3UxPsn0cn0hlQODy9fR3y2+bzD0fv2X3uZEaoNEsDahKm6iDcARvQF8tKSJBlFGfH2ZBxPbZWpLCwHs9GOOGKeKsUm\\\\\\\/LKqRQcI07Ege876cm4+wcLFCnL6X40\\\\\\\/Kkooq2O4y68doO2PvuWlUaUgzx1o\\\\\\\/BwcPeHfpHr7up9kyw1LXa6kSDDh7yXaFvt1q32hCY1lfS635+JYjWmxXZxXQwyabJyJxzfxwmlqN8KwmnTu1cq9S02TR8jNVwf0xipdSM8ekj1Xhf991KqUtdLZUsrYK8GAGY5NCq4NFCRlG2Uw7rl58Fwct3QrbewSa87coDMxeFm2WtUmI3hiyMNq7ngbmCRJT5pOebHi+jn\\\\\\\/HeLzUdKbcDBuLqj90aOleSqxPMVebFykyWsNdRPYtGd4PGOMk9N\\\\\\\/jLC62YVZ+ka7tQGFKWmnmPMQtBITMVGE01hirJP7dpJnP6FnGEBklKtmwZs4P7eDnXw7ipAtE2lFkZTDhvheQTv4+RnFznxMOJWTtfdjWzWsu2u9MaBnPIaNXStZZsUwzrs3HsjglEaPXch1s3chQ67j0yo\\\\\\\/dcW+7n92I6\\\\\\\/Wpfg+069usWPnaOfIKBRhBmv1BCbWsEVyyevs\\\\\\\/9yyytkCgA4XuVXsBUk4t7+MImVT5+7s9Of74pDWP1pMe3o8I7oS8LPDgvJilqh2TXr4vTw3fOMLpPDFDbBOe+9nNceRwl73bZXfwjOU7+xjrMZSnMNB9W9SdKHXDP+qnELhZcyLdEy27ngcfWkmVg6JH\\\\\\\/ZKnwgV18rS0pI4TIiXyjcLmM0si84yQuWDWfQjo+NX9vl3QcVLImS2FJags+hw76kiq7IN4RYDjTrjOfZQUVrZhbO5IeC4Gilam8lNzFD3PvzZeyCaS94AapqorO680dNhO44WXM0Jihro0q+ZtyoISooox+eoLN8vpvAaBGWe+3L7Z21voFhxwfS6tZJV5jpzGntHp\\\\\\\/GKJCM2A3QvS+fQNaxT\\\\\\\/6tfnlPNe96emjRbmLiUsdgME6unPJcsyXQjCkHo7We2fe6Bw5Ue1rx\\\\\\\/bofUQTBYP81o5lV37\\\\\\\/zh+Wy5W\\\\\\\/+KKxXJShkDHR4+mLvyF5pZo0ZBHa5wefkgLE\\\\\\\/4P3G5cRWYELY07L1aw+lL5yam8kKHPLGAzSfJGs6HxzKbCT3YVt98RrWEzheosd6zG8OZSrmHVRKXFiRgT2nQBYQA2lBf2+qGah0v2GUg0NAFA8PaYX58M0kvocoJ0sJAOiaG6gHr8JmukpgGeHZCXHZUKbkDL2S5447DPQyM7icmqslpqKTji6P6cKsEqZibz2yuOdtjc4Fjrk1KG884t3vLlqw+MbRxJJ9kT5CkQgxuAzpgHgzNMbB8w7939odEtXx+ZZwx7ol5wvwwnk1\\\\\\\/9F9W63\\\\\\\/G4gopQ+Mo8e5MFU87lNWJw4GhLvKvhIL149M7NY4om3ezpitqAscAjJo1DKgtXskPiro0jySoqcOBNYsK0pra9z2K+7Td9ToCrURusTuF9o6Tbs0Kp+afKd7\\\\\\\/sP9BQrkh\\\\\\\/UlumEkB2lWKNQJ6999QFER360iE8Qg6h9U4pWhDZLJZXkMlvUFgsGEIDNwXA9zMm020isLzQ3cRA2OGatvSy9KYwZabegY4HIHwP1HXFc5N8pHbpD14ddVUQs5YXDMs6kvMfSEWC9ca07EDGD3n7mJYla23SKImnRh4dOwtsx\\\\\\\/7C93ylnj9Mn\\\\\\\/IPG3NjzDF+Bj\\\\\\\/sUUqkvGAUZENilTxxul103vAa3fxc6WQeL2H4tUwM1wYdlArGaQ1lmY0b4beHRCfLAC8GlHO3pzYoSZwxWMnDZi3Rqm4AmOnGOO7UhD7libpDK47mFC6RWCFBd9thH7\\\\\\\/ITXv5VKj51kfh+hmYMKTXYXwJ33uMEO9NKJ8NJU7pa\\\\\\\/M7YLDg63gZd7zPm+0XrytZDrRishLt7ge89dc2noQsAIVMs8bjJl9l1OSHxj5wiK+QKzUK5KF9vo5r3sTuFrquXZW8XkzPmMo1eDWfr5Rrbx2PjGfBrY5BvgmtZGUqsRczVkC+WW4l2w5l+BWc2tHKpJ9LwBS3PmXpSBNeEon2t9nqqSN+7N7pzP4Hrz8wkJG76qGJjRJPH1a0R5Su0oO0ayWFFb8vgdkaMNVfIqVSFwvEK2wtFPAllpPoBlatLBNo96l+WhL9pvq1knoehDtADOKvmGXDmWroOiWUFbJNa3NkWLPwzQDlHimMaeCNpCa5ZHdhUyz1vuGT38yl7q8baUCK78D5RVHhJ5FG+FUndDx\\\\\\\/W0ZGyv0vwqYtf9YwGUxSXSS8mz8troK1BOBD+5c0dv\\\\\\\/ESt0iK5LbsOtTC\\\\\\\/0sCHXHwOCr+3skyCXnfaTaZmTGBYLunqDGLIEkIj5kx6zD80hoxG+UuU+0qVF8oGzfTrmoOUYQHsGsfbFhB2U0fyXA5CQcPPcdNIgKMsyc49GLTmWaXno3AHFV1zH4ql2qGVynEw8lW9akdujHyWrG12NRC4nLYYem0aYEOYbsusvhRTwTur+mvGOU5N1GAVqrnTYXO1a9bEeVHY\\\\\\\/vCPAtJuUDZdqFzuMya\\\\\\\/D+KSWbzp0CVghj4dPh\\\\\\\/+xbvZG3PupSKnEUF+LUkd2OQFjZhTRLTS0IxkLPkJT1aRNGJIzkWoNiEHQkV5pB034LfLVlLdcO7W\\\\\\\/v9e86vkU7FuIa6Hsfi2bIezo1kFfdNAXdT1Lg44qx3ggYt0PRS01uAfZuzwYLPgyCD0MfTQs1asVtpcr3jMR9I\\\\\\\/QM\\\\\\\/CSYnQtfo06MOtHAtx2C7OK78UiZqwh47CQ2bUPn1uwOYjxNlRN5ma6LJbuIYbX+p7Wwkruul3ImmV7IzWiH57RLnz3v+6XzW9HWbzIAE12XIRl1DNGeGqR+b7HmY3BRD7TXgiAgUwFme\\\\\\\/LE\\\\\\\/LgaexPnW1ngNUCxlJCgmne54lko1\\\\\\\/tSIh9DWgJTvLLzCRXVEK1JgmeX1c5hfpyD1Palh5JO9mk+8ZirPLLZWoFyDmadGkaTFI0HHfOcpxRqyG5Qc64i\\\\\\\/Y6akkGp\\\\\\\/Vf3sJuEoDDwjoHVttu2CiuZPzSemj11tEQR5YL8rivJFFXmcoHEb7RKb9aCMhqRomYila\\\\\\\/C5Z8w45OtgsqHDoCksOBFGmw322c0Q4lEYerOnTwCyxcIxbgowbQ2J65iHR6xqR4ZMbya1tG8pkoH\\\\\\\/La5BCRwvuAvyKRnn1RCVkYLqYG5JcGdvZodQheUlOd7VRcTglPDTRZ+s8vLPDlwDEm2iFcig1rUsZ6p8fAZqVQllMTtWO565l4b\\\\\\\/SzS0zGWIoMdDSV+xIjE3GhIkrY7yU1QdQl58UpUXXyR0Bm4YFotUJljpm0FKLbP9qSsY7KdWd1S0y\\\\\\\/NozLWKwQFS0k8aCZFL3ehgyNEg9HYZ6GRNYopqlcfOimKnOBGpWmA\\\\\\\/m2zMUmhYR57YtXPZ\\\\\\\/uUx1MZ8\\\\\\\/UD292T2ABDT\\\\\\\/NFkFRTgo8bmR0\\\\\\\/62H3X2DsK0O9l0+VJJ14YIQwUzp8VrzRle6XP+H+0m3gKXPjLU4+4EfyLSRkK4iysT+nAdCVnmNyzQllk0xyJLvPbEWel1tRFPu\\\\\\\/xGAV03AywPwf+D9k\\\\\\\/\\\\\\\/1XsIkfkZ232+YVoZqjDD69nKMUem36I3JuiT86+E+HNP+JBe0zWtvsRENXMYnA+5YPiP\\\\\\\/HEy1ulnvFx+LYe8MVTi4v3h6VSgMcr1RdOkpLNM0j7tnk9zsUAUH1FmxtV7K+bzagNtXkJ\\\\\\\/B0H5w7Mu009aaL1YWXDt2lGTfQnGgRsFdHP6Fh5vapCpRjaLg\\\\\\\/6VHenzVjtTxcgt3D33tHI604QuJr1psps+\\\\\\\/AcPt+NA2F47C6Pq67DFu0S8NzH9IOTBMvVoAlVhtdq3kIMdwtqqwxKYT0zdkX2lWh5cy2y7HusaZ7S3GS5NF6yU7ldfCNtnjVKMRAt+jqzvp6xZ8NfJ8DsRt7D5n8H3FdCSOcPoXc71eWIEGg+BSoxa0xkq2JcKUWjDjdSFQR\\\\\\\/7Lk9SqFyjg\\\\\\\/pUfbwxyd5JS++AfeOwCIg09DxtPAZnzj7+xn6xUltKrPGaCgvTUs5Rje21pCzwLyKb0KNkWS72jbyiaca1yHqGckuwGfOFTODQSPFOem7YpPVr4uKjQX7pw4iDWRlvy1WhdgmTk4dAlbtzj8BWNOQGoOQUayrN0wWsMHv3wG+5QOEImq03tm7rRduk1sfpqs\\\\\\\/XGYFnDzVtAdAA2ynYsg+1R25aBKjaHDNkOlozWIuXXf+fv2KTOov+T+xjbeVvinp\\\\\\\/upljLOowycUevYllTHKyJizs19PzLpOLEBJWUA8I2lWCmcl+o5rXc3Hc4fxnpmRaYpXbVt5C2+3fbmbvBS6gAqiaPnPjpZKQI7RgDWCqpuLZ\\\\\\\/BFiK+9E1+fYOJjmvYJ9g6RkiRD63iRqZNWj2sIJl\\\\\\\/effCmA9hZvZV3OIT6axZ\\\\\\\/m7229fDCJfO7H6lxXdGs\\\\\\\/8h2Qtr8tF3WbssIi\\\\\\\/HjUAVil3HKV2OvwFN6B53olCSmgoyeampYH0jBHNzyPi43i6XP\\\\\\\/VKfri3r94DlpCc5o9HGDZFQeBKlEDw6Nm8Kd0Jrqn4XLRkguAyZmVi0jrCcCH\\\\\\\/Du3eANcnK4Z7OzESrl5f\\\\\\\/TjmQukih9j3Jf7FWERZH1cMEz48O8KaICYHEJWLzPIbqJtUzonkA\\\\\\\/QbWcg1UwazINvR8BZ52dTfhTOGW1jqc9NLue7fKipBCQIDCfGdcAox4RZrIE\\\\\\\/S2S4UvrHLd3LN6thcTKDNRRTxUYOhLveu75bqnXCbT1pLnHbHtyN2Wc5mCzEUKZogj5TC\\\\\\\/o\\\\\\\/O5yGrjgVvTuyRHz6gMs1m15ULRnF6Rve5HHLL0fTMCxPD7Ruu1ySDOvJgXpBYXXki46OxnSe2fAn2isYVpInIwGc4LTpHi3JIdDaFzQ7OBVVeAtHEpBgLeTpG1eckoWzqjrYcX9Gvn+UTm8ZBlomYPynrqAIcYkuLOzRAjtbEKiOvQ5axizUDlsafKThnnjxiHap\\\\\\\/+PZELD\\\\\\\/pyHNtKsrQ2+rX41t63pWXGqD\\\\\\\/yU\\\\\\\/lrxoLYpkZx\\\\\\\/t56DGk0BrHRvaPIONmFkcxGBWBz5bhGmcjBJcLAABovxIIq8Ac80K+ZjrytlkRJMBjH66mLQ38a3W6nOu6K0f+9kP0zzFFdyM9kcoAHHacXjA2w9iqun6rNcqLbLBUgkLsLbaWbsmWm0XVqElqDTLMNyvNM6j4mdTopDKAUsY1KTJaLi5C2oXrpCMaviPunQs0TyPERkILbOCcK9cVZRvIrCGOTd0Mt6A6U1TNIbKuQzlp4crio0e0\\\\\\\/HEOpCFtSWz69HnzcMZfV29czBWHDSTv0xwa\\\\\\\/vvK17eaZN7gbCwZPGnPzSJYbl\\\u0022,\\\u0022iv\\\u0022:\\\u0022l5fWQlk+mnHzMGNSSBNiUA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022058cbebd4216686265e93ded111bd26e3927b32261341ad6cab40699918c13a3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pC478TfSZo6SQdDMd31HMnAc99u09w7ZZlsTyZleA1IDHIe2a6Sf\\\\\\\/g\\\\\\\/N\\\\\\\/GDr2H0W86YAY6AFMllddU426C560NZSTdWBetXBCwsBhw8jqwu0NYr9H1hLwT29pKnEasYzr5GnNR+rNWuXW8x2TVSnxoKMQb7AoiO2PvOyi8B\\\\\\\/fmRPTIeri7lyKQkqrgRiJeyjOMNfhL1MbCXBaJARi8q32cRwb2Atm0nwXBWDfFeunnnqXXGrmn519YtooWaEPLRydu7nxc+M2iLN9wdaHUfqSuEy8sj2dq\\\\\\\/RQ3loIUv9DBpImhttWbd6Bq6gtrdJUZm0qq8vXz\\\\\\\/KZW3LGqFGVgViSVM2sx40LzTgHT8SSbfn50xgfX+dSNNE8Rq3sgc+tR2\\\\\\\/vaNiy9bG99+FVexO53jCEgeQd\\\\\\\/Rbxd3IkO9qbM6f9i7aRqvCdkIk28977d3sDW5FGPlHKV4zXL9wj8fuvd\\\\\\\/uKqAaklncdbX1NWkK8MhMh9UwVUgdU5qkHoxL9LNXwmFLUjoCbW68kKgGgAi8RHcd5AP7Ux1AGQ0wL+apetDRfiFcBoXXU3IyZsd8t2Kzwh9eIQ00qSkE8fmWW940+mpJZfPSLF5BRIy+5la4ISqb5eAYwbS3gZGsSR9uGejQtcrWp+9Wv\\\\\\\/WZmXlsH1EPe\\\\\\\/ZVbXq94D2Rk0RtWSeQCo10gVCf5up8VkfhxdTz8d36\\\\\\\/TmYi49nrJlPzI1tRkYt0+PcwmrYBDzVMMOdbk5muwrV6LoL9K6b92xNstKYFWRWGLT5HSATUrZdgRGUWrd03t\\\\\\\/RCBAtvPbHymoEgEpo\\\\\\\/bZtw\\\\\\\/eaKyFrFDQJrXaim2dDwkcBHJBgVodhlBq1g0mcxZY+wtkebvvke\\\\\\\/3HEMpxBwDImLjDtwnNyHiKIlVA19sSza0hmqC8WXsJCXX4KLq+GMtF6WQJOfo93f0vymYRpdWH2P0xOrqTt95QGScQy2ZWABreIE2WCShxMW5zQ4UM+sa52onyhz+4vReMnTeofEYyN0pvnOBKrtAIAR\\\\\\\/U1NSFxAMEVsLAYC4HDFSF7hjiyDgB1koYrjODesNWHDW+vh2FxjJuVytTsPY3o+GNi\\\\\\\/HfDYYiAKB3acN0NaCGbqfrlZ3lU+381xXrEPdFpCgS4TwGb9Wrlh8qQQoJoF4Jk1CHPfuW7Dw22cgUNHvzTgyKfe4w\\\\\\\/yJZKq\\\\\\\/bQ908FVILooDJ0U0+iYNezeXTu8qFpcnHsiE31GOndQmXpHuOKqA8RFcFZvYZf\\\\\\\/2LyJO5UoMzWuguBQlC7YjAybv71VBTaXgR+VNDXPlyNdqdVwc8BgIGwe9ztJOuVMMBprVwJlk3IlYPVnzMfnGkBDFHxNPrfWdS82+Z8\\\\\\\/p0Bi8r+9m0xvodQsPSkSWi\\\\\\\/yNXvsFro4D\\\\\\\/61KS+50dNYMyOodr5nZ8CCoZDfJIpr1zOvUaxouZOAPchNIoI1iGNEg6KOi4WjCGZRx4AOM9FKC3rLowQsSkJ12AExAaSbdCcZ+GYIIDaFGOMeiE8bx+\\\\\\\/Ax\\\\\\\/rIrhUibL7iwZhPpAxCpXnx63vivDeRN3t\\\\\\\/nSrHGEeWGcphMlu2b6CeQlERDklAhuAZPy9MuITDz1QWBnzFkUbLIphQZXcXYNeeMsaUi2IQgVlzyusEuFwsT7i4l0YQqDcCT+TUV9bHqWcP9wywaj0pgb\\\\\\\/V+qvb2uvaa\\\\\\\/JUvDVATh\\\\\\\/RJQfQFqQFDL\\\\\\\/gNVXLyxNTxtTtBbIrxQmqKpVTshQX9zeuYrJll2fzmUwjVEwBeLSS5VuFh\\\\\\\/06\\\\\\\/WvLLYlii\\\\\\\/mwXmSCoYTi7+A4AawcQsJnrPx3BaN4ot0Hfn7KZwFPzWR1YjV3UoYkgDrqLOshmEYvukjLkydYS6CTP2\\\\\\\/gqI7RbM9NziPQ\\\\\\\/gT8\\\\\\\/juUYb8vA7HS\\\\\\\/BSqCLLGG3rLTet4CroQedX86oUgAeuzwvdowQxlrDbyIm+XuRkUD2lrG1aarHfdDUbXVVBIsK3gGZkJqHH0HVrdlRetzIPkAdvMid4P2dEchjpfX1gmmNZLWEWy+NxxAWAI\\\\\\\/fVUx2bQcHO5WmExQuzITm2Yr6dTNdrgBUWvROIFl04SSkuTyl\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZTuGIMlc2jqQEatUVvymdw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f23fb5967cf2be7896c7df50457241be6cd3d0970cd1afbff4c2ebf45f7bcab6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460530 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/203" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Qfbhp-9zyN3sJSkyKqUhGdfzuSLsDYLymSPCfChQ5QY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/203" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1nWYLb72BbgXGgC-OX-3sckoytDNlJySQWgSz6zTklo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/193" title="CRISPR/Cas9-Induced Double-Strand Break Repair in Arabidopsis Nonhomologous End-Joining Mutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/221" title="Time-Course Analysis of Gene Expression During the Saccharomyces cerevisiae Hypoxic Response" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460530" id="node460530" data-pisa="ggg;7/1/203" data-pisa-master="ggg;g3.116.036020" data-apath="/ggg/7/1/203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2080-8416" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dongyan Zhao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John P. Hamilton</span>, <span class="highwire-citation-author" data-delta="2">Michael Hardigan</span>, <span class="highwire-citation-author" data-delta="3">Dongmei Yin</span>, <span class="highwire-citation-author" data-delta="4">Tao He</span>, <span class="highwire-citation-author" data-delta="5">Brieanne Vaillancourt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9990-5808" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mauricio Reynoso</span>, <span class="highwire-citation-author" data-delta="7">Germain Pauluzzi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-7798-6830" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott Funkhouser</span>, <span class="highwire-citation-author" data-delta="9">Yuehua Cui</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-8568-7125" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julia Bailey-Serres</span>, <span class="highwire-citation-author" data-delta="11">Jiming Jiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-6727-4677" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>C. Robin Buell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-2776-6669" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ning Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">203-219;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036020 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460530" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dongyan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongyan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADongyan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2080-8416" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dongyan Zhao</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John P. Hamilton</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Hardigan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BHardigan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hardigan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BHardigan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dongmei Yin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongmei%2BYin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongmei%2BYin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematics, San Francisco State University, California 94132</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brieanne Vaillancourt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brieanne%2BVaillancourt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaillancourt%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrieanne%2BVaillancourt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mauricio Reynoso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BReynoso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynoso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMauricio%2BReynoso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9990-5808" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mauricio Reynoso</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Germain Pauluzzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Germain%2BPauluzzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pauluzzi%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGermain%2BPauluzzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Scott Funkhouser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Genetics Program, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BFunkhouser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Funkhouser%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BFunkhouser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7798-6830" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott Funkhouser</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yuehua Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuehua%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuehua%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Julia Bailey-Serres </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julia%2BBailey-Serres%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bailey-Serres%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulia%2BBailey-Serres%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8568-7125" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julia Bailey-Serres</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Jiming Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Horticulture, University of Wisconsin–Madison, Wisconsin 53705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiming%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiming%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6727-4677" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for C. Robin Buell</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a>
<a href="mailto:jiangn@msu.edu" class="" data-icon-position="" data-hide-link-title="0">jiangn@msu.edu</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Ning Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Genetics Program, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Program in Ecology, Evolutionary Biology and Behavior, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ning%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANing%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2776-6669" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ning Jiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a>
<a href="mailto:jiangn@msu.edu" class="" data-icon-position="" data-hide-link-title="0">jiangn@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/203" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460530" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460530/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/203.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460530" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460530/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/203.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460530" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460530/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/203.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460530" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460530/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/203.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460530" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460530/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Gene expression is controlled at transcriptional and post-transcriptional levels including decoding of messenger RNA (mRNA) into polypeptides via ribosome-mediated translation. Translational regulation has been intensively studied in the model dicot plant <em>Arabidopsis thaliana</em>, and in this study, we assessed the translational status [proportion of steady-state mRNA associated with ribosomes] of mRNAs by Translating Ribosome Affinity Purification followed by mRNA-sequencing (TRAP-seq) in rice (<em>Oryza sativa</em>), a model monocot plant and the most important food crop. A survey of three tissues found that most transcribed rice genes are translated whereas few transposable elements are associated with ribosomes. Genes with short and GC-rich coding regions are overrepresented in ribosome-associated mRNAs, suggesting that the GC-richness characteristic of coding sequences in grasses may be an adaptation that favors efficient translation. Transcripts with retained introns and extended 5′ untranslated regions are underrepresented on ribosomes, and rice genes belonging to different evolutionary lineages exhibited differential enrichment on the ribosomes that was associated with GC content. Genes involved in photosynthesis and stress responses are preferentially associated with ribosomes, whereas genes in epigenetic regulation pathways are the least enriched on ribosomes. Such variation is more dramatic in rice than that in <em>Arabidopsis</em> and is correlated with the wide variation of GC content of transcripts in rice. Taken together, variation in the translation status of individual transcripts reflects important mechanisms of gene regulation, which may have a role in evolution and diversification.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/translating-ribosome-affinity-purification-sequencing" class="hw-term hw-article-keyword hw-article-keyword-translating-ribosome-affinity-purification-sequencing" rel="nofollow">translating ribosome affinity purification sequencing</a></li><li class="kwd"><a href="/keyword/gc-content" class="hw-term hw-article-keyword hw-article-keyword-gc-content" rel="nofollow">GC content</a></li><li class="kwd"><a href="/keyword/mrna-length" class="hw-term hw-article-keyword hw-article-keyword-mrna-length" rel="nofollow">mRNA length</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/oryza-sativa" class="hw-term hw-article-keyword hw-article-keyword-oryza-sativa" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Oryza sativa</em></a></li></ul><p id="p-3">Within a genome, a gene can encode a mRNA that encodes a protein or a noncoding RNA. Protein coding genes are first transcribed in the nucleus into a precursor mRNA (pre-mRNA) from which introns are cotranscriptionally or post-transcriptionally spliced, the 5′ end capped, and the 3′ end cleaved and polyadenylated. The processing of pre-mRNA leads to the generation of mature mRNAs that are transported to the cytoplasm and translated into proteins. Translation occurs on the ribosome complex and amino acids are provided to the nascent peptide chain via transfer RNAs (tRNAs). Not all mRNAs are translated at the same efficiency and some may be stored or degraded (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Picard <em>et al.</em> 2012</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Shah <em>et al.</em> 2013</a>). The monitoring of not only the steady-state abundance of specific transcripts but also the level at which they are associated with ribosomes can provide insight into the regulation of gene expression. With advances in sequencing technology, sequencing of cDNAs that are derived from mRNAs (mRNA-seq) has become an efficient and convenient method to quantify transcript abundance, generally interpreted as gene expression levels. While analysis of mRNA-seq results provides a comprehensive atlas of the transcriptome, post-transcriptional regulation can affect the correlation between transcript and protein abundance. To more closely estimate protein levels, the abundance of individual mRNAs associated with the ribosome, referred to as the translatome (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Halbeisen <em>et al.</em> 2009</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mustroph <em>et al.</em> 2009b</a>), can be used as a proxy of the proteome. Actively translated mRNAs are associated with multiple ribosomes in polyribosome (polysome) complexes. As translation is largely regulated during the engagement of ribosomal subunits on the mRNA (initiation phase of translation) in plants and other organisms (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Browning and Bailey-Serres 2015</a>), an mRNA associated with polysomes is most likely to be undergoing the elongation and termination phases of translation. The mapping of the nuclease-protected footprints of individual ribosomes on mRNAs largely supports this conclusion, except for mRNAs with short upstream open reading frames (uORFs) that generally dampen translation of the main protein coding region of mRNAs (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Juntawong <em>et al.</em> 2014</a>). Intact ribosomes can be immunoprecipitated using an epitope-tagged large ribosomal protein subunit from which polysome-associated mRNAs are prepared, greatly reducing contamination of mRNAs associated with messenger ribonucleoproteins of density similar to ribosome complexes (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Zanetti <em>et al.</em> 2005</a>). When this approach is combined with mRNA-seq, the translation status of genes can be determined in a genome-wide fashion, a technique called TRAP-seq (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Reynoso <em>et al.</em> 2015</a>).</p><p id="p-4">Rice is the world’s most important food crop, as it serves as a staple for more than half of the world’s population (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Khush 2013</a>). Among the cereal crops, rice has the smallest genome (389 Mb) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">International Rice Genome Sequencing Project 2005</a>), and is a model species for other cereal crops such as maize, sorghum, wheat, and barley. Rice, along with other cereals and in contrast to dicot plant species such as <em>Arabidopsis thaliana</em>, has a distinct bimodal distribution of genes with respect to GC content. For example, gene GC content in <em>Arabidopsis</em> ranges from 40–55%, with a nearly normal distribution (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wang and Hickey 2007</a>). However, for the grasses (Poaceae) including rice, the genic GC distribution is more widely dispersed, ranging from 40 to 75% (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Carels and Bernardi 2000</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wong <em>et al.</em> 2002</a>). It has been proposed that GC-biased gene conversion (gBGC) is mainly responsible for the evolution of GC content of genes in monocots (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clement <em>et al.</em> 2015</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Glemin <em>et al.</em> 2014</a>). Previous studies have shown that GC-rich genes in rice are associated with more variable expression at the transcription level (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tatarinova <em>et al.</em> 2010</a>) and GC-rich genes demonstrate stronger codon usage bias (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Guo <em>et al.</em> 2007</a>); nevertheless, the biological significance of varied GC content of rice (grass) genes is largely unclear, including its potential impact on translation.</p><p id="p-5">The availability of translatome data for rice would greatly facilitate fundamental and applied research for both protein-coding genes and transposable elements, which are DNA sequence units that can rapidly change their locations and copy numbers in the genome. Previous proteomics studies on rice provided evidence of translation for a few thousand gene transcripts (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Koller <em>et al.</em> 2002</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Komatsu 2005</a>). In addition, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Park <em>et al.</em> (2012</a>) conducted a genome-wide analysis of total and polysomal mRNAs using a rice 3′ tiling microarray. However, the focus of the study was on the influence of stresses on a subset of genes, and the translation status of the majority of individual rice genes is still unknown. In this study, we constructed an epitope-tagged rice Ribosomal Protein L18 cDNA line (<em>p35S:HF-OsRPL18</em>) to facilitate the isolation of polysomes and conduct TRAP-seq analysis. Findings made in this study provide new perspectives regarding the regulation of translation as well as guidance for future crop engineering.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Construction of the p35S:HF-OsRPL18 transgene and rice transformation</h3><p id="p-6">All genes described in this study are referred to by their MSU Rice Genome Annotation Project Release 7 gene identifier (<a href="http://rice.plantbiology.msu.edu/">http://rice.plantbiology.msu.edu/</a>) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kawahara <em>et al.</em> 2013</a>). For construction of the <em>p35S:HF-OsRPL18</em> transgene, the coding region of LOC_Os03g22180 was amplified from rice cDNA, and the PCR product was cloned into the pENTR/D-TOPO TA vector (Invitrogen, Carlsbad, CA) and then recombined into the destination vector p35S:HF-GATA, which contains a His<sub>6</sub>-FLAG tag (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Mustroph <em>et al.</em> 2010</a>). Thereafter, the <em>HF-OsRPL18</em> sequence was amplified by PCR, recloned into the TA vector, and the <em>HF-OsRPL18</em> sequence was excised using <em>Sal</em>I and <em>Pst</em>I and then placed between the same restriction sites in pCAMBIA1300S, which contains a double 35S promoter upstream of the multiple cloning site (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Xiong and Yang 2003</a>). This yielded the expression construct containing <em>p35S:HF-OsRPL18</em> used for transformation of the rice cultivar Nipponbare.</p><p id="p-7">The basal medium used for transformation was Murashige and Skoog (MS) (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Murashige and Skoog 1962</a>) supplemented with 30 g/L sucrose and Gelzan 3 g/L (PhytoTechnology Laboratories, Overland Park, KS) at pH 5.7. Callus cell growth was induced on callus induction medium (CIM, MS plus 2 mg/L 2,4-dichlorophenoxyacetic acid) using mature seeds and then cocultured for 2 d in the dark at 25° with <em>Agrobacterium tumefaciens</em> EHA 105 carrying the expression vector with <em>p35S:HF-OsRPL18</em> and 200 µM acetosyringone to enable infection. Each callus was transferred to selection medium (CIM plus 250 mg/L cefatoxime and 50 mg/L hygromycin B) to cease growth of <em>Ag. tumefaciens</em> and allow the growth of transgenic calli. Four to eight weeks after selection, each callus was transferred to selection regeneration medium (MS plus 250 mg/L cefatoxime, 50 mg/L hygromycin B, 3 mg/L 6-benzylaminopurine, and 0.5 mg/L 1-naphthaleneacetic acid) for seedling generation.</p><p id="p-8">Plants generated from hygromycin-resistant calli were transferred to soil and allowed to self-pollinate. Two-wk-old T<sub>1</sub> seedlings were tested for hygromycin resistance by placing 1 cm leaf tips on Whatman paper moistened with 50 mg/L hygromycin for 4 d. Tips from plants with successful transgene integration remained green (resistant) whereas those without the transgene turned brown (nonresistant). To determine the copy number of the T-DNA in transgenic plants, DNA was extracted from leaves of resistant T<sub>1</sub> seedlings using the CTAB method (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Doyle and Doyle 1987</a>), and digested with <em>Sac</em>I to conduct Southern DNA blot analysis. The digested DNA was resolved on a 1% (w/v) agarose gel, followed by transfer of the DNA to a nylon membrane (GE Healthcare, Pittsburgh, PA) using capillary flow. A 419 bp fragment from the hygromycin resistance gene was amplified using a pair of primers (forward 5′-CAAGGAATCGGTCAATACACTAC-3′ and reverse 5′-AAGCTCTGATAGAGTTGGTCAAG-3′) designed from the expression vector and labeled with digoxigenin, which was detected by anti-digoxigenin-AP, Fab fragments (Roche Applied Science, Indianapolis, IN). The hygromycin-resistant T<sub>1</sub> plants were advanced to the T<sub>2</sub> generation and hygromycin resistance was tested with T<sub>2</sub> seedlings. If all T<sub>2</sub> plants from a single T<sub>1</sub> plant were resistant and the relevant T<sub>1</sub> plant only harbored a single integration event, the relevant line was considered homozygous. One such homozygous line was chosen for subsequent analysis (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS1.pptx">Figure S1</a>).</p></div><div id="sec-3" class="subsection"><h3>Plant material and growth</h3><p id="p-9">To examine the assembly of HF-OsRPL18 into polysomes, T<sub>4</sub> transgenic rice seeds containing <em>p35S:HF-OsRPL18</em> were sterilized and grown on MS medium with 10 g/L sucrose. Plants were grown for 7 d at 28° with a 16 hr day/8 hr night photoperiod under 80 µmol/m<sup>2</sup>/sec photosynthetically active radiation.</p><p id="p-10">For mRNA-seq and TRAP-seq analysis, 5-wk-old calli were harvested from T<sub>4</sub> seeds grown on CIM. For shoot and panicle mRNA-seq and TRAP-seq, T<sub>4</sub> <em>p35S:HF-OsRPL18</em> plants were grown in 5 inch plastic pots filled with Profile Green’s Grade sand. The pots were placed inside a tray filled with water. Fertilizer (Peters Excel 15-5-15 Cal-Mag special) was applied twice each week as a 200 ml solution from a 10× stock to each tray with eight pots. All plants were grown in a growth chamber with a 12 hr day/12 hr night photoperiod, with light intensity 500 µmol/m<sup>2</sup>/sec; the above-ground portion of young seedlings (shoots) was harvested at 14 d from plants grown at 30/28° d/night; young panicles of 1–5 cm in length, corresponding to stage R1 (post the formation of panicle branches) according to <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Counce <em>et al.</em> (2000)</a>, were harvested from plants grown at 28/24° d/night. All tissues were harvested directly into liquid nitrogen and stored at −80° until use.</p></div><div id="sec-4" class="subsection"><h3>Sucrose density fractionation of polysomes and efficiency evaluation of ribosome immunopurification</h3><p id="p-11">To test the association of HF-OsRPL18 with translating ribosomes, polysomal complexes were isolated by differential centrifugation and processed as described previously (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mustroph <em>et al.</em> 2009a</a>). Approximately 1 g of frozen, pulverized 7-d-old shoot tissue was used to obtain a pellet enriched in ribosomes. About 3300 OD<sub>260</sub> were fractionated on a 20–60% (w/v) sucrose density gradient. Protein in each fraction was precipitated and polypeptides analyzed by standard western blot after separation by 12% (w/v) SDS-PAGE. HF-OsRPL18 was immunodetected using a mouse monoclonal antibody conjugated to horseradish peroxidase (α-FLAG-HRP; 1:1000) as previously described (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Zanetti <em>et al.</em> 2005</a>). The small ribosomal subunit protein RPS6 was immunodetected using a rabbit polyclonal antiserum against <em>Zea mays</em> RPS6 (1:5000) (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Williams <em>et al.</em> 2003</a>) as the primary antibody and a horseradish peroxidase conjugated goat anti-rabbit IgG as the secondary antibody (1:10,000; Bio-Rad, Hercules CA).</p><p id="p-12">To test the efficiency of immunopurification, ∼30 mg of pulverized shoot tissue of wild-type (Nipponbare) or <em>p35S:HF-OsRPL18</em> plants was processed for affinity purification of epitope-tagged ribosomes as described by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Mustroph <em>et al.</em> (2009a)</a> with the following modification: 5 µg of α-FLAG (Sigma-Aldrich, St. Louis MO, F1804) was bound to 50 µl Dynabeads Protein G (Life Technologies, Carlsbad CA, 1004D) instead of α-FLAG agarose beads (Sigma-Aldrich, F2426). Fractions obtained during the procedure were separated by 12% (w/v) SDS-PAGE followed by western blot detection of HF-OsRPL18 and RPS6 as described above.</p></div><div id="sec-5" class="subsection"><h3>mRNA-seq and TRAP-seq library construction, sequencing, and expression abundance estimation</h3><p id="p-13">For mRNA library construction, ∼5 g of pulverized tissue (shoot, callus, or young panicle) was homogenized in 45 ml of Polysome Extraction Buffer (PEB) and centrifuged at 4°, 16,000 × <em>g</em>, for 15 min (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Mustroph <em>et al.</em> 2009a</a>). The supernatant was filtered through Miracloth and centrifuged as described above. Five percent of the supernatant was used for isolation of total RNAs with the remainder used for affinity purification of epitope-tagged ribosomes as described previously (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Mustroph <em>et al.</em> 2009a</a>). RNAs were extracted from the PEB supernatant (for mRNA-seq) or the eluate from the immunoprecipitation (for TRAP-seq) using TRIzol reagent (Invitrogen) and the QIAGEN RNeasy kit.</p><p id="p-14">Illumina TruSeq stranded mRNA libraries (Illumina, San Diego, CA) were constructed and sequenced on an Illumina HiSeq2500 (Illumina,) in paired-end mode for 100 or 150 nucleotides (nt) at the Research Technology Support Facility of Michigan State University. The read lengths and yield of each library are summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS1.docx">Table S1</a>. Reads from libraries sequenced with read lengths greater than 100 nt were trimmed to 100 nt for consistency. The reads for each library were adapter trimmed and quality filtered using Trimmomatic (v0.32) in paired mode using the parameters: LEADING:5 TRAILING:5 SLIDINGWINDOW:4:10 MINLEN:30 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>).</p><p id="p-15">The cleaned paired reads from the mRNA-seq and TRAP-seq libraries were aligned to the rice pseudomolecules (Os-Nipponbare-Reference-IRGSP-1.0) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kawahara <em>et al.</em> 2013</a>) using TopHat (v1.4.1) (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Trapnell <em>et al.</em> 2009</a>) with a minimum intron size of 5 bp and a maximum intron size of 15 kbp. Transcript abundances for the MSU Release 7 representative gene models (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kawahara <em>et al.</em> 2013</a>) were calculated as fragments per kilobase per exon model per million mapped reads (FPKMs) from the TopHat alignments using Cufflinks (v1.3.0) (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Trapnell <em>et al.</em> 2010</a>) with the same intron parameters as used for TopHat (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS2.xlsx">Table S2</a>).</p></div><div id="sec-6" class="subsection"><h3>Translatomes, translatome enrichment index (TEI), and filtering of transposon genes</h3><p id="p-16">For each tissue, the translatome was comprised of all genes with a TRAP-seq FPKM value ≥1. The TEI was calculated as the ratio of the TRAP-seq FPKM <em>vs.</em> the mRNA-seq FPKM from the same tissue. TEIs of different gene groups (see below) were compared using the Kolmogorov–Smirnov (KS) test, and <em>P</em>-values were adjusted using the Benjamini–Hochberg method (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>). Representative gene models from MSU Release 7 (<a href="ftp://ftp.plantbiology.msu.edu/pub/data/Eukaryotic_Projects/o_sativa/annotation_dbs/pseudomolecules/version_7.0/all.dir/all.locus_brief_info.7.0">ftp://ftp.plantbiology.msu.edu/pub/data/Eukaryotic_Projects/o_sativa/annotation_dbs/pseudomolecules/version_7.0/all.dir/all.locus_brief_info.7.0</a>) were used for all analyses, with the exception of analysis of alternative splice isoforms in which all gene models were considered. In MSU Release 7, 16,937 transposon genes (including both transposons and retrotransposons) were annotated. To confirm that transposon-related genes were correctly tagged, a custom rice transposon library was used to mask the predicted cDNAs of the 16,937 transposon genes. If 50% or more of the cDNA length was masked by the library, the transcript was validated as a true transposon gene; 15,461 transposon-related genes met this criterion and were used for subsequent analyses.</p></div><div id="sec-7" class="subsection"><h3>Classification of genes by expression breadth and abundance</h3><p id="p-17">To determine the expression breadth of nontransposon genes, 23 mRNA-seq samples were downloaded from the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA) (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>). After exclusion of redundant samples (those from similar tissues), 12 distinct mRNA-seq samples were retained, in addition to the three mRNA-seq samples from this study, resulting in a total of 15 samples for the expression breadth analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS3.docx">Table S3</a>). The SRA-obtained raw reads were cleaned, aligned to the reference genome as described above, and expression abundance was estimated as described above with an FPKM value ≥1 considered as expressed. Nontransposon genes were categorized into three groups with an increasing order of expression breadth: Breadth Group 1 comprised nontransposon genes expressed in 1–7 samples; Breadth Group 2 were nontransposon genes expressed in 8–14 samples; and Breadth Group 3 consisted of nontransposon genes with expression evidence in all 15 samples and were considered constitutively expressed genes.</p><p id="p-18">The three mRNA-seq datasets generated in this study were used to classify nontransposon genes based on their expression abundance for each tissue. In each tissue, nontransposon genes were categorized into six groups with FPKM values increasing from 1 to over 1000. Each group contains comparable number of genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS4.docx">Table S4</a>).</p></div><div id="sec-8" class="subsection"><h3>Identification of conserved and lineage-specific genes</h3><p id="p-19">Rice genes were assigned to a green plant lineage based on their placement within orthologous gene clusters generated with OrthoMCL (v1) (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2003</a>). OrthoMCL was used to cluster protein sequences from 13 species: <em>Amborella trichopoda</em> (v1.0, phytozome.jgi.doe.gov), <em>Aquilegia coerulea</em> (v1.1, phytozome.jgi.doe.gov), <em>A. thaliana</em> (TAIR10, phytozome.jgi.doe.gov), <em>Brachypodium distachyon</em> (v2.1, phytozome.jgi.doe.gov), <em>Musa acuminata</em> (v1, banana-genome.cirad.fr), <em>Oryza sativa</em> (v7, rice.plantbiology.msu.edu), <em>Phoenix dactylifera</em> (v3, Qatar-weill.cornell.edu/research/datepalmGenome), <em>Phyllostachys heterocycla</em> (v1.0, bamboogdb.org), <em>Populus trichocarpa</em> (v3.0, phytozome.jgi.doe.gov), <em>Setaria italica</em> (v2.1, phytozome.jgi.doe.gov), <em>Solanum lycopersicum</em> (iTAG2.3, phytozome.jgi.doe.gov), <em>Sorghum bicolor</em> (v2.1, phytozome.jgi.doe.gov), and <em>Z. mays</em> (5b; maizegdb.org). Based on species content, orthologous gene clusters were classified as conserved in all angiosperms, monocot and dicot-restricted, monocot-restricted, Poaceae-restricted, or <em>O. sativa</em>-specific. Transposon-related genes were not assigned a lineage. To visualize the relative translation status (or TEI) of expressed genes in each lineage, kernel density plots were generated for the three tissue types (shoot, callus, and panicle) surveyed. Nucleotide content for gene and coding sequences (CDS) was calculated using BEDTools (v2.20.1) (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Quinlan and Hall 2010</a>).</p></div><div id="sec-9" class="subsection"><h3>Gene ontology (GO) analysis</h3><p id="p-20">GO assignment table of rice genes (Release 7) was downloaded from the Rice Genome Annotation Project (<a href="http://rice.plantbiology.msu.edu/">http://rice.plantbiology.msu.edu/</a>), from which, genes were grouped according to their associated biological processes. The assignment table also contains the corresponding <em>Arabidopsis</em> orthologous gene for each rice gene, which was used to synchronize the analysis between rice and <em>Arabidopsis</em> (see below). Information of gene features, including length and GC content of CDS and untranslated regions (UTRs), were obtained based on the rice genome annotation (Release 7), and only genes with both 5′ UTRs (at least 20 bp in length) and 3′ UTRs (at least 110 bp in length) were retained for the following analysis. The minimum length of UTRs (20 bp for 5′ UTRs and 110 bp for 3′ UTRs) was defined using genes with full-length cDNA evidence (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kikuchi <em>et al.</em> 2003b</a>), where 97.5% of these gene have UTRs equal or longer than the minimum length. These values were used for all analysis related to the specific roles of UTRs in translation. For each tissue, the mean values of TEIs of all genes within each category (each GO term) were calculated and only the categories with at least 40 genes were included in the comparison.</p><p id="p-21">Similar analyses were conducted in <em>Arabidopsis</em> using mRNA-seq levels and TRAP-seq levels published from <em>Arabidopsis</em> seedlings under normal conditions (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Juntawong <em>et al.</em> 2014</a>). The CDS of <em>Arabidopsis</em> genes were downloaded from TAIR10 (<a href="https://www.arabidopsis.org/">https://www.arabidopsis.org/</a>). To ensure comparability between <em>Arabidopsis</em> and rice, only genes corresponding to the genes used for GO analysis in rice shoots with TEIs were included (7966 genes for <em>Arabidopsis</em> and 11,600 for rice). Only processes with 40 or more genes were considered for comparison, as described above.</p></div><div id="sec-10" class="subsection"><h3>Estimation of recombination rate and codon usage frequency</h3><p id="p-22">The recombination rate in rice was adapted from <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Tian <em>et al.</em> (2009)</a>, where 3982 markers were anchored to the rice genome, IRGSP Build 4.0 pseudomolecules. To be consistent with the other analyses conducted in this study, these markers were aligned to the rice genome MSU Release 7, where markers having the best match with ≥ 99% identity over the entire marker sequences were used for the following analysis. The recombination rate (cM/Mb) was measured in 1 Mb bins (some bins are &gt;1 Mb if there is no marker in a region &gt;1 Mb). Pearson correlation between recombination rate and CDS GC content, as well as that between recombination rate and TEI, were conducted using SAS/9.3.</p><p id="p-23">The occurrence frequency of all 64 codons (61 for amino acids and 3 for stop codon) was determined using a perl script kindly provided by Kevin Childs and Tiffany Liu (Michigan State University). Briefly, codon usage frequency was calculated for genes with the lowest (&lt;0.2) and the highest (≥3.0) TEIs and synonymous codon usage between the two gene groups were compared; chi-square tests were conducted using SAS/9.3.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-24">RNA-sequencing reads (mRNA-seq and TRAP-seq) are available in the NCBI SRA under BioProject PRJNA298638. The mRNA-seq and TRAP-seq expression abundances are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS2.xlsx">Table S2</a> and on individual gene report pages at the Michigan State University Rice Genome Annotation Project (<a href="http://rice.plantbiology.msu.edu/">http://rice.plantbiology.msu.edu/</a>).</p></div></div><div class="section results-discussion" id="sec-12"><h2 class="">Results and Discussion</h2><div id="sec-13" class="subsection"><h3>Expression of epitope-tagged RPL18 and its incorporation into ribosomes</h3><p id="p-25">In <em>A. thaliana</em>, it has been demonstrated that HF-AtRPL18 is efficiently incorporated into the 60S subunit and that ribosomes containing HF-AtRPL18 accumulate in polysomal complexes (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Zanetti <em>et al.</em> 2005</a>). To test whether this can be replicated in rice, we constructed an expression vector that contains the orthologous rice <em>RPL18</em> gene with a His<sub>6</sub>-FLAG tag at the N-terminus of the protein (HF-OsRPL18, see <em>Materials and Methods</em> for details). Transgenic <em>p35S:HF-OsRPL18</em> rice plants used for subsequent analyses contained a single copy of the transgene and were similar to the wild-type Nipponbare plants in terms of plant height, days to flower, number of tillers, seed number per plant, and seed weight (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS1.pptx">Figure S1</a>).</p><p id="p-26">Two experiments were performed to confirm that the FLAG-epitope-tagged OsRPL18 was appropriately assembled into functional ribosomes that can be efficiently purified by TRAP. In the first experiment, the association of HF-OsRPL18 with the large ribosomal subunit and multi-ribosome complexes (polysomes) was confirmed by preparation of a shoot extract with a polysome-stabilizing buffer followed by pelleting of ribosome complexes at 170,000 × <em>g</em>, and further fractionation of the complexes by centrifugation through a sucrose density gradient. A UV absorbance profile of a representative gradient was obtained and western blot analysis was performed to evaluate the fractionation of HF-OsRPL18 and the core 40S ribosomal subunit protein RPS6. Both RPL18 and RPS6 had similar abundance in the small-to-large polysome complexes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>; fractions 6–10), confirming assembly of the epitope-tagged protein into functional ribosomes. RPS6 and HF-OsRPL18 were enriched in fractions 4 and 5, respectively, consistent with their independent assembly into the 40S and 60S subunits, which separated into distinct peaks in the sucrose density gradient.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/203/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Epitope-tagged rice RPL18 assembles into functional ribosomes that can be purified by TRAP. (A) Confirmation of RPL18 assembly into ribosomes. The p35S:HF-OsRPL18 rice line was used as the source of ribosomal complexes, which were separated by ultracentrifugation on a 20&#x2013;60% (w/v) sucrose density gradient. The absorbance at 254 nm was recorded to detect the ribosomal subunits of 40S and 60S, monosomes (80S), and polysomes. The gradient was fractionated and proteins in the 10 fractions were analyzed by SDS-PAGE separation and western blotting processed with anti-FLAG (&#x3B1;-FLAG) or anti-RPS6 (&#x3B1;-RPS6) antisera. Molecular mass markers are indicated on the left. (B) Purification of polysomes by TRAP. Equal weights of pulverized tissue from untransformed Nipponbare (control) and homozygous transgenic p35S:HF-OsRPL18 shoots were solubilized in polysome extraction buffer to obtain a clarified supernatant (total). The extract was incubated with anti-FLAG-bound Dynabeads coupled to Protein G to bind HF-RPL18. The supernatant (unbound fraction) was collected to evaluate the efficiency of the immunopurification. The magnetically captured protein&#x2013;RNA complexes (TRAP fraction) was eluted from the beads using 3X-FLAG peptide. Each fraction was analyzed by western blot with &#x3B1;-FLAG and &#x3B1;-RPS6. The expected molecular mass of HF-RPL18 is 25 kDa. SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis; TRAP, translating ribosome affinity purification." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645401416" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Epitope-tagged rice RPL18 assembles into functional ribosomes that can be purified by TRAP. (A) Confirmation of RPL18 assembly into ribosomes. The &lt;em&gt;p35S:HF-OsRPL18&lt;/em&gt; rice line was used as the source of ribosomal complexes, which were separated by ultracentrifugation on a 20&#x2013;60% (w/v) sucrose density gradient. The absorbance at 254 nm was recorded to detect the ribosomal subunits of 40S and 60S, monosomes (80S), and polysomes. The gradient was fractionated and proteins in the 10 fractions were analyzed by SDS-PAGE separation and western blotting processed with anti-FLAG (&#x3B1;-FLAG) or anti-RPS6 (&#x3B1;-RPS6) antisera. Molecular mass markers are indicated on the left. (B) Purification of polysomes by TRAP. Equal weights of pulverized tissue from untransformed Nipponbare (control) and homozygous transgenic &lt;em&gt;p35S:HF-OsRPL18&lt;/em&gt; shoots were solubilized in polysome extraction buffer to obtain a clarified supernatant (total). The extract was incubated with anti-FLAG-bound Dynabeads coupled to Protein G to bind HF-RPL18. The supernatant (unbound fraction) was collected to evaluate the efficiency of the immunopurification. The magnetically captured protein&#x2013;RNA complexes (TRAP fraction) was eluted from the beads using 3X-FLAG peptide. Each fraction was analyzed by western blot with &#x3B1;-FLAG and &#x3B1;-RPS6. The expected molecular mass of HF-RPL18 is 25 kDa. SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis; TRAP, translating ribosome affinity purification.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/203/F1.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/1/203/F1.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/203/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/203/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470045" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Epitope-tagged rice RPL18 assembles into functional ribosomes that can be purified by TRAP. (A) Confirmation of RPL18 assembly into ribosomes. The <em>p35S:HF-OsRPL18</em> rice line was used as the source of ribosomal complexes, which were separated by ultracentrifugation on a 20–60% (w/v) sucrose density gradient. The absorbance at 254 nm was recorded to detect the ribosomal subunits of 40S and 60S, monosomes (80S), and polysomes. The gradient was fractionated and proteins in the 10 fractions were analyzed by SDS-PAGE separation and western blotting processed with anti-FLAG (α-FLAG) or anti-RPS6 (α-RPS6) antisera. Molecular mass markers are indicated on the left. (B) Purification of polysomes by TRAP. Equal weights of pulverized tissue from untransformed Nipponbare (control) and homozygous transgenic <em>p35S:HF-OsRPL18</em> shoots were solubilized in polysome extraction buffer to obtain a clarified supernatant (total). The extract was incubated with anti-FLAG-bound Dynabeads coupled to Protein G to bind HF-RPL18. The supernatant (unbound fraction) was collected to evaluate the efficiency of the immunopurification. The magnetically captured protein–RNA complexes (TRAP fraction) was eluted from the beads using 3X-FLAG peptide. Each fraction was analyzed by western blot with α-FLAG and α-RPS6. The expected molecular mass of HF-RPL18 is 25 kDa. SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis; TRAP, translating ribosome affinity purification.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The second experiment was carried out to determine if ribosomes with HF-OsRPL18 could be directly and efficiently immunopurified from cell extracts prepared with polysome-stabilizing buffer. To accomplish this, the presence of HF-OsRPL18 and the 40S subunit RPS6 were monitored in the initial crude extract (total) and in the proteins obtained by immunopurification of FLAG-tagged HF-OsRPL18 and associated proteins using magnetic beads conjugated to an anti-FLAG monoclonal antibody (TRAP). As a control for the efficiency of the purification, the supernatant from the immunopurification (unbound) was also evaluated in the western blot analysis. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> shows a representative experiment that demonstrates that HF-OsRPL18 is efficiently purified along with RPS6, consistent with isolation of the 40S and 60S subunits in a complete 80S ribosome. A small amount of HF-OsRPL18 remained in the supernatant, even when an excess of antibody was used in the purification. This suggests that a small proportion of HF-OsRPL18-containing ribosomes may not have the epitope accessible to the immune-affinity matrix. RPS6 was also present in the unbound fraction. This was anticipated because 40S ribosomal subunits not coupled with HF-OsRPL18-containing 60S subunits would not be purified, as the <em>p35S:HF-OsRPL18</em> transgene was expressed in the wild-type Nipponbare background with three endogenous <em>RPL18</em> genes that are all expressed in shoots. These experiments confirm that TRAP can be used efficiently to purify functional ribosomes from <em>p35S:HF-OsRPL18</em> transgenic rice.</p></div><div id="sec-14" class="subsection"><h3>Rice transcriptome and translatome profiles</h3><p id="p-29">To elucidate the regulation of rice gene expression at the transcriptional and translational levels, we conducted mRNA-seq and TRAP-seq with three different rice tissues (shoot, callus, and panicle) using the <em>p35SHF-OsRPL18</em> line. Rice genes were categorized into transposon and nontransposon gene sets to validate the efficiency of transcription (mRNA-seq) and translation (TRAP-seq). In this study, 16,937 transposon-encoding genes are annotated in Release 7 and we performed further filtering to yield a total of 15,461 transposon-related genes (see <em>Materials and Methods</em>), of which the majority encode a putative transposase or other proteins required for transposition and are therefore putative autonomous transposons or derivatives from autonomous elements. The nontransposon gene set contains 39,049 nontransposon protein-coding genes.</p><p id="p-30">For simplicity, we defined a gene as transcribed (for mRNA-seq) and translated (for TRAP-seq) if it has an FPKM value ≥1. Moreover, we defined the ratio of FPKM values between TRAP-seq and mRNA-seq as a TEI of mRNAs associated with ribosomes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hupe <em>et al.</em> 2014</a>). As shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, over half (55%) of the nontransposon genes have evidence of translation in at least one of the three tissues sampled. If only genes with transcription evidence (<em>i.e.</em>, mRNA-seq ≥ 1 FPKM) are considered, ∼90% of the transcribed nontransposon genes are associated with ribosomes, suggesting that, once transcribed, the majority of nontransposon genes are recruited to ribosomes for translation (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Of the 22,896 nontransposon genes with an annotated function in Release 7, 16,151 (71%) are translated in at least one of the three tissues examined, suggesting that most of them are true protein-coding genes consistent with their similarity to known genes. In the rice genome, 16,153 genes are annotated as “expressed proteins” or “hypothetical proteins”: of these, 5496 (34%) are translated in at least one of the three tissues examined. This suggests that, of the unknown proteins annotated solely based on transcription evidence and/or <em>ab initio</em> gene predictions, a substantial portion could be <em>bona fide</em> protein-coding genes. For a comparison, we examined 4095 genes with proteomic evidence in OryzaPG-DB (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Helmy <em>et al.</em> 2012</a>); 3884 (94.8%) have a TRAP-seq FPKM value ≥1 in this study. Nontransposon genes with transcripts that were not detected in this study may be functional in tissues not evaluated in this study.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470136/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470136/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Gene expression evidence from transcriptomes and translatomes of three tissues of rice</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">In contrast to nontransposon genes, very few transposon genes are transcribed and even fewer are translated (2.5 and 1.4% of the total transposon genes, respectively). Less than half (48%) of the transcribed transposons are translated compared to 90% for nontransposon genes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), suggesting that transcripts encoding transposon proteins are much less likely to be translated than nontransposon transcripts. Among the transposons that are translated, the median TEI value is 0.65, which is substantially lower than that of nontransposon genes (0.93). To assess whether the lower fraction of translated transposon genes is an artifact of low sensitivity of TRAP-seq, we examined the translation status of autonomous transposons reported to be active in Nipponbare. Among the nine transposons with reported activity in Nipponbare (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hirochika <em>et al.</em> 1996</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jiang <em>et al.</em> 2003</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kikuchi <em>et al.</em> 2003a</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Nakazaki <em>et al.</em> 2003</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Picault <em>et al.</em> 2009</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sabot <em>et al.</em> 2011</a>), translational evidence was detected for six in this study (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The distinction in translatome profiles between transposons and nontransposon genes, high concordance of translatome evidence with genes supported by previous proteomic studies, and detection of the most active transposons in the translatome suggests that the TRAP-seq data generated in this study effectively separates mRNAs associated with ribosomes from pre/mRNAs in the nucleus or untranslated mRNAs in the cytosol.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470167/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470167/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Translational status of transposable elements documented to be active in Nipponbare</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">It should be noted that a high TEI could be interpreted in two ways as the abundance of mRNA on ribosomes is largely determined by three factors: initiation, elongation, and termination. The predominant regulation is the rate of initiation of translation, which is positively correlated with protein production (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Browning and Bailey-Serres 2015</a>). If the rate of elongation and termination is comparable between transcripts, then a high TEI indicates preferential translation, <em>i.e.</em>, translation is initialized on a higher fraction of available transcripts than for other individual mRNAs. Translation can be initialized by newly recruited ribosomes (<em>de novo</em> initiation) or, when translation is completed on an mRNA, the recently terminated ribosomes may reinitiate translation on the same mRNA, a process called ribosome recycling (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Browning and Bailey-Serres 2015</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Roy and von Arnim 2013</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Shah <em>et al.</em> 2013</a>). However, a high TEI could be due to other factors. One is a slower rate of translation elongation or termination, such that more transcripts are associated with ribosomes (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Browning and Bailey-Serres 2015</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Roy and von Arnim 2013</a>). Factors including the structure of the mRNA coding region, abundance of tRNAs, and codon usage contribute to the variation of elongation speed (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ingolia 2014</a>). Another factor is the presence of one or more upstream ORFs (uORFs) preceding the main protein CDS on the transcript. uORFs in the 5′ region of transcripts typically reduce the overall translation of the main CDS, in a metabolite or condition-regulated manner (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Juntawong <em>et al.</em> 2014</a>; <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Roy and von Arnim 2013</a>). Moreover, these contributions to TEI are not mutually exclusive. Although slow elongation/termination rate or uORFs may contribute to high TEI values, thereby uncoupling this statistic from protein production, a very low TEI most likely indicates limited translation.</p></div><div id="sec-15" class="subsection"><h3>Translatome enrichment associated with a short coding region and higher GC content</h3><p id="p-37">In rice, the length of gene CDS varies substantially with a median value of 1095 bp (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS5.xlsx">Table S5</a>). To test whether CDS length is related to translatome enrichment, the level of mRNAs associated with ribosomes (TRAP-seq FPKM) was plotted against total mRNA abundance (mRNA-seq FPKM) for rice genes with different length of CDS. As shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, genes with a large CDS tend to have a relatively low TRAP-seq level. When the CDS is shorter than 500 bp, the median value of TEI is about 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS3.pptx">Figure S3</a>). When the CDS length is longer than 1500 bp, the median value of TEI is about 0.5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS3.pptx">Figure S3</a>). The difference between each group is significant (<em>P</em> &lt; 0.0001, KS test). A similar trend in TEI with CDS length was observed in calli and young panicles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS2.pptx">Figure S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS3.pptx">Figure S3</a>). These results are consistent with previous observations that short genes are associated with higher translation levels in other organisms (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arava <em>et al.</em> 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jiao and Meyerowitz 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lackner <em>et al.</em> 2007</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lei <em>et al.</em> 2015</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Shah <em>et al.</em> 2013</a>). It is feasible that, when the number of initiation factors or ribosomes is limiting and the two termini of a long transcript are further apart, the first initiation event and/or the efficiency of ribosome recycling and reinitiation is reduced and therefore fewer transcripts are translated.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470143/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470143/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The effect of length and GC content of different portions of transcripts on TEI</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/203/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of expression abundance at transcription level (mRNA-seq) vs. translation level (TRAP-seq) in relation to gene length (A) and GC content (B) of genes expressed in shoots. CDS, coding sequence; FPKM, fragments per kilobase per exon model per million mapped reads; mRNA-seq, messenger RNA sequencing; TRAP-seq, translating ribosome affinity purification followed by mRNA-sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645401416" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of expression abundance at transcription level (mRNA-seq) &lt;em&gt;vs.&lt;/em&gt; translation level (TRAP-seq) in relation to gene length (A) and GC content (B) of genes expressed in shoots. CDS, coding sequence; FPKM, fragments per kilobase per exon model per million mapped reads; mRNA-seq, messenger RNA sequencing; TRAP-seq, translating ribosome affinity purification followed by mRNA-sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/203/F2.medium.gif" width="291" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/1/203/F2.medium.gif" width="291" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/203/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/203/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470081" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Plot of expression abundance at transcription level (mRNA-seq) <em>vs.</em> translation level (TRAP-seq) in relation to gene length (A) and GC content (B) of genes expressed in shoots. CDS, coding sequence; FPKM, fragments per kilobase per exon model per million mapped reads; mRNA-seq, messenger RNA sequencing; TRAP-seq, translating ribosome affinity purification followed by mRNA-sequencing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">As described in the introduction, the GC content of transcripts in grasses has a bimodal distribution (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS4.pptx">Figure S4</a>) with the high GC fraction largely absent from dicot genomes. As shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS2.pptx">Figure S2</a>, the TEI value is positively correlated with GC content in all tissues. We then divided the rice genes into four groups (GC &lt; 50%, 50–60%, 60–70%, and ≥ 70%) and examined the TEI. The average TEI of the highest GC group (≥70%) is nearly two whereas that for the lowest GC group is about 0.5 (<em>P</em> &lt; 0.0001, KS test) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS3.pptx">Figure S3</a>). The preferential association of ribosomes with GC-rich genes is consistent with the results from a recent study in maize (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Lei <em>et al.</em> 2015</a>). Since the correlation between GC content and TEI is only reported in grasses, and GC-rich genes are often associated with small size (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Guo <em>et al.</em> 2007</a>), it was reasoned whether the elevated TEI of GC-rich genes is due to their small size. To this end, we divided rice genes into six groups with increasing CDS size and calculated the correlation between CDS size and GC content in each group. The result showed that the relationship between CDS length and GC content varies among different size ranges (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS6.docx">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS7.xlsx">Table S7</a>). When the CDS length is between 1100 and 1400 bp, no significant correlation between CDS length and GC content was observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS6.docx">Table S6</a>, <em>P</em> = 0.33). Nevertheless, significant positive correlation (<em>P</em> &lt; 2.2e−<sup>16</sup>) between GC content and TEI was still observed, suggesting that CDS length and GC content played independent roles in translation.</p><p id="p-42">The high TEI associated with increased GC content suggests that the evolution of GC-rich genes in grasses may be associated with their increased translational activity. However, high GC transcripts are often associated with stronger secondary structure that may hinder the elongation process and/or cause ribosome stalling or pausing because the GC content variation is correlated with distinctions in codon usage bias (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fennoy and Bailey-Serres 1993</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kanaya <em>et al.</em> 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Knight <em>et al.</em> 2001</a>), thereby resulting in a higher TEI. In a previous study, it was shown that GC-poor and GC-rich genes are subject to differential constraints in codon usage (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mukhopadhyay <em>et al.</em> 2007</a>). The codon usage in GC-poor genes is largely selected for tRNA gene abundance; whereas that in GC-rich genes is selected both by tRNA gene abundance and by stable secondary structure (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Mukhopadhyay <em>et al.</em> 2007</a>). Accordingly, the selection for secondary structure in GC-rich genes could be due to its benefit in association with ribosomes. In mammalian cells, high GC content is associated with enhanced transcription activity (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kudla <em>et al.</em> 2006</a>) and our analysis indicates that high GC content may also favor ribosome association of relevant transcripts. Application of the ribosome footprint profiling (Ribo-seq) method, which maps individual ribosomes along transcripts, could provide further insight into the underlying mechanisms associated with the higher TEI of GC-rich mRNAs.</p><p id="p-43">As mentioned above, about one third of the “expressed proteins” or “hypothetical proteins” are likely translated. In general, the CDS of those “expressed proteins” or “hypothetical proteins” is much smaller than that of genes with known function (median 615 <em>vs.</em> 1179 bp, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS8.docx">Table S8</a>), and they are also more GC-rich (57 <em>vs.</em> 53%). The shorter CDS and higher GC content result in a higher TEI (1.14 <em>vs.</em> 0.88) compared to genes with known function. As a result, the transcripts derived from “expressed proteins” or “hypothetical proteins” are more enriched on ribosomes.</p></div><div id="sec-16" class="subsection"><h3>Translatome enrichment for genes with different expression breadth and abundance</h3><p id="p-44">In yeast, shorter genes are more efficiently translated (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arava <em>et al.</em> 2003</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Lackner <em>et al.</em> 2007</a>), which was thought to be a consequence associated with the shortened length of constitutively expressed housekeeping genes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hurowitz and Brown 2003</a>; <a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Shah <em>et al.</em> 2013</a>). As such, it minimizes the possibility that transcript length itself may have a mechanistic role in translational regulation. To test whether this is the case in rice, we examined the translatome enrichment of constitutively expressed genes and putative housekeeping genes. To this end, 12 mRNA-seq datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS3.docx">Table S3</a>) obtained from the NCBI SRA database along with the three mRNA-seq samples generated from this study (15 total mRNA-seq samples) were used to determine expression breadth. To annotate a gene as expressed, a FPKM of ≥ 1 in at least one tissue was required. With this criterion, three groups of genes were generated. One group includes genes expressed in 1–7 samples, the second group includes genes expressed in 8–14 samples, and the third group includes genes expressed in all 15 samples (constitutively expressed genes).</p><p id="p-45">As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, there is a minor yet negative correlation between expression breadth and TEI, with constitutively expressed genes having slightly lower TEI than other genes. This is consistent with the fact that the CDS of constitutively expressed genes are slightly longer and less GC-rich than other genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS5.pptx">Figure S5</a>). We further examined the TEI of 23 known reference (housekeeping) genes (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Narsai <em>et al.</em> 2010</a>). The median value of TEI for those genes ranges from 0.98 to 1.05 in three tissues, which is comparable with that of other genes (0.93) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS9.docx">Table S9</a>). The median length of the reference housekeeping genes is similar to that of other genes (1071 and 1095 bp, respectively, <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS9.docx">Table S9</a>). Taken together, the high TEI value for short genes in rice is not due to the enrichment of constitutively expressed housekeeping genes, suggesting that there is a mechanistic role of a small CDS size in translation. Moreover, there is no evidence that housekeeping genes are preferentially translated in rice. If a high TEI value does imply a high amount of protein produced per transcript, the low transcript abundance of short GC-rich genes, such as the Poaceae lineage-specific genes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Campbell <em>et al.</em> 2007</a>), maybe compensated by the higher translation efficiency, at least to some degree.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/203/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of translatome enrichment index (TEI) by transcription specificity and transcription level in shoots. (A) TEI of genes with transcripts detected across 15 mRNA-seq (messenger RNA sequencing) samples. (B) TEI of genes with transcript abundance variation as estimated by fragments per kilobase of exon per million mapped reads (FPKM). Genes with extremely high TEIs (1% of the genes in each group) were not shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645401416" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of translatome enrichment index (TEI) by transcription specificity and transcription level in shoots. (A) TEI of genes with transcripts detected across 15 mRNA-seq (messenger RNA sequencing) samples. (B) TEI of genes with transcript abundance variation as estimated by fragments per kilobase of exon per million mapped reads (FPKM). Genes with extremely high TEIs (1% of the genes in each group) were not shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/203/F3.medium.gif" width="440" height="189"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/1/203/F3.medium.gif" width="440" height="189"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/203/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/203/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470191" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">Distribution of translatome enrichment index (TEI) by transcription specificity and transcription level in shoots. (A) TEI of genes with transcripts detected across 15 mRNA-seq (messenger RNA sequencing) samples. (B) TEI of genes with transcript abundance variation as estimated by fragments per kilobase of exon per million mapped reads (FPKM). Genes with extremely high TEIs (1% of the genes in each group) were not shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">To explore the relationship between transcript level and translatome enrichment, genes were divided into six groups based on their expression abundance, with each group containing a similar number of genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS4.docx">Table S4</a>). As shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, the TEI and expression abundance (relative steady-state transcript abundance) demonstrate a quadratic relationship, <em>i.e.</em>, the TEI is higher for the genes with the highest or lowest expression abundance, whereas the genes with intermediate expression abundance have the lowest TEI; the differences between the two extreme groups and other intermediate groups are significant (<em>P</em> &lt; 0.0001, KS test). To understand the underlying mechanism of this relationship, the GC content and CDS length of each group were examined (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS5.pptx">Figure S5</a>). Interestingly, the GC content of genes shows a similar quadratic relationship, <em>i.e.</em>, the genes with the lowest and highest transcription level have higher GC content than the genes with intermediate transcription abundance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS5.pptx">Figure S5</a>). Moreover, the length of CDS has a negative quadratic relationship with expression abundance, which suggests that genes with the highest and lowest transcription levels are associated with a smaller CDS. As a result, the nonlinear relationship between TEI and expression abundance can be largely explained by the GC content and CDS length of the relevant transcripts.</p></div><div id="sec-17" class="subsection"><h3>Regulation of alternatively spliced transcripts at the translational level</h3><p id="p-48">When pre-mRNAs from genes with multiple exons are processed, a single transcript could be alternatively spliced into multiple isoforms. In humans, the majority of multi-exon genes have alternative spliced transcript isoforms (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pan <em>et al.</em> 2008</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wang <em>et al.</em> 2008</a>), which allow the generation of multiple protein isoforms from a single gene. In plants, isoforms are not as prevalent as in mammals, yet they still play important roles in protein diversity. Previous studies in rice indicate that the majority of alternative splice forms result from the retention of individual introns (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Campbell <em>et al.</em> 2006</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">E <em>et al.</em> 2013</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wang and Brendel 2006</a>). Among the 16,809 isoforms annotated in MSU Release 7, 8164 isoform pairs were SI <em>vs.</em> RI, referring to isoforms of genes with spliced introns and retained introns, respectively (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Campbell <em>et al.</em> 2006</a>). Among them, a small subset are both transcribed and translated, with calli containing the highest number of expressed SI/RI pairs (732 pairs, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS10.docx">Table S10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS11.xlsx">Table S11</a>). Whereas the median TEI of the SI isoform is comparable to other genes, the median TEI for RI isoforms is much lower than that of SI isoforms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS10.docx">Table S10</a>), suggesting that the RI may have a negative impact on translation or that the transcripts with RIs may be inaccessible to ribosomes due to nuclear localization or cytoplasmic sequestration (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lavut and Raveh 2012</a>).</p><p id="p-49">The RIs were grouped based on their location in the mRNA, including those in the 5′ UTR, CDS, and 3′ UTR. The abundance of the RI transcript isoform, as reflected by the median FPKM value through mRNA-seq, is very similar to that of their SI counterpart (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Nevertheless, when the RI is in a 5′ UTR, the TRAP-seq level of the RI isoforms is substantially lower than that of the corresponding SI isoforms (<em>P</em> &lt; 0.0003, KS test) and the trend is similar in all three tissues (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS6.pptx">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS10.docx">Table S10</a>). For RIs in the CDS and 3′ UTR, the TRAP-seq level of RI isoforms is slightly lower than that of the SI isoform, but the difference is not significant (<em>P</em> &gt; 0.3, KS test). It should be noted that the lower TRAP-seq level is unlikely due to nonsense-mediated mRNA decay (NMD) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kalyna <em>et al.</em> 2012</a>). This is because NMD involves a functional 80S ribosome and therefore occurs after the initiation of translation (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Schweingruber <em>et al.</em> 2013</a>), so a more substantial reduction of transcript abundance is expected but this is not the case here.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/203/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The impact of retained intron on transcription and translation of the relevant transcripts in shoots. (A) Expression abundance at transcription level (log2 FPKM of mRNA-seq) of isoforms with or without retained intron in different portions (5&#x2032; UTR, CDS, and 3&#x2032; UTR) of genes. (B) Expression abundance at translation level (log2 FPKM of TRAP-seq) of isoforms with a retained intron in different portions of genes. (C) Size of the retained introns. (D) GC content of the retained introns. Extreme values (top 1%) in each group were not shown. CDS, coding sequence; FPKM, fragments per kilobase per exon model per million mapped reads; mRNA-seq, messenger RNA sequencing; RI, retained intron; TRAP-seq, translating ribosome affinity purification followed by mRNA-sequencing; UTR, untranslated region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645401416" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The impact of retained intron on transcription and translation of the relevant transcripts in shoots. (A) Expression abundance at transcription level (log2 FPKM of mRNA-seq) of isoforms with or without retained intron in different portions (5&#x2032; UTR, CDS, and 3&#x2032; UTR) of genes. (B) Expression abundance at translation level (log2 FPKM of TRAP-seq) of isoforms with a retained intron in different portions of genes. (C) Size of the retained introns. (D) GC content of the retained introns. Extreme values (top 1%) in each group were not shown. CDS, coding sequence; FPKM, fragments per kilobase per exon model per million mapped reads; mRNA-seq, messenger RNA sequencing; RI, retained intron; TRAP-seq, translating ribosome affinity purification followed by mRNA-sequencing; UTR, untranslated region.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/203/F4.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/1/203/F4.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/203/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/203/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470180" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-50" class="first-child">The impact of retained intron on transcription and translation of the relevant transcripts in shoots. (A) Expression abundance at transcription level (log2 FPKM of mRNA-seq) of isoforms with or without retained intron in different portions (5′ UTR, CDS, and 3′ UTR) of genes. (B) Expression abundance at translation level (log2 FPKM of TRAP-seq) of isoforms with a retained intron in different portions of genes. (C) Size of the retained introns. (D) GC content of the retained introns. Extreme values (top 1%) in each group were not shown. CDS, coding sequence; FPKM, fragments per kilobase per exon model per million mapped reads; mRNA-seq, messenger RNA sequencing; RI, retained intron; TRAP-seq, translating ribosome affinity purification followed by mRNA-sequencing; UTR, untranslated region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">To test whether the effect of the RI in different portions of the transcripts was due to the size and/or GC content of the RI, the relevant intron sequences were characterized. The median size of RIs is &lt; 150 bp, much smaller than the median intron size of 418 bp in rice, suggesting that small introns are more likely to be retained (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The size of the RI in the 5′ UTR is slightly larger than that in the 3′ UTR (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). The RIs in the CDS are the smallest, with a median value &lt; 100 bp. As for GC content, the RIs in the CDS are associated with the highest GC content (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>), followed by that in the 5′ UTR, with the RIs in the 3′ UTR having the lowest GC content. Not surprisingly, the GC content of the RIs is lower than the corresponding genomic average GC content of the 5′ UTR, CDS, and 3′ UTR. Moreover, the difference in GC content between RI in the 5′ UTR and other 5′ UTR sequences is the most significant (40 <em>vs.</em> 57%, <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). As a result, the dramatically lower TRAP-seq level caused by RI in 5′ UTRs may be due to its slightly larger size and/or its lower GC content than average 5′ UTRs (see below).</p><p id="p-52">If RIs in the CDS leads to the generation of different protein isoforms, the evolutionary advantage for RIs in UTRs is unclear. Emerging evidence suggests that RIs followed by cytoplasmic splicing provide a newly recognized regulation of gene expression, especially for developmentally controlled genes. In the fern <em>Marsilea vestita</em>, large amounts of RI-containing transcripts are stored and later spliced in the microspore during spermatogenesis (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boothby <em>et al.</em> 2013</a>). In this way, proteins can be rapidly generated without the need for <em>de novo</em> transcription. These RI transcripts are hypothesized to be “sentinel RNAs” (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Buckley <em>et al.</em> 2014</a>). Our analysis suggests that, for some rice genes, the RI forms are transcribed similarly to their corresponding SI forms but are underrepresented on ribosomes, suggesting that “sentinel RNAs” may also apply to those rice genes. To determine what types of genes are more likely to show intron retention, the GO terms were assigned to the SI/RI isoforms and compared with that to all genes. The genes that are most enriched (χ<sup>2</sup> test, <em>P</em> &lt; 0.001) in SI/RI isoforms are those involved in the response to abiotic stimuli, followed by those in postembryonic development, growth, and the generation of precursor metabolites and energy (χ<sup>2</sup> test, <em>P</em> &lt; 0.05, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS12.xlsx">Table S12</a>). Given that the SI/RI isoforms are enriched in genes involved in the response to abiotic stimuli and some development processes, it is conceivable that the splicing and translation of the preexisting RI transcripts would allow a rapid response to abiotic stimuli as well as important developmental signals.</p></div><div id="sec-18" class="subsection"><h3>The role of UTRs in translation</h3><p id="p-53">The differential TEI between RI/SI isoform pairs allows further dissection of factors involved in translation. To explore the roles of the UTRs in translation, we extended our analysis to all nontransposon genes with annotated 5′ and 3′ UTRs. To minimize the effect of CDS, we divided the genes into different groups based on CDS length and examined the effect of UTR length within each group. In general, there is a negative correlation between the size of 5′ UTRs and TEI (coefficient &lt; 0, <em>P</em> &lt; 10<sup>−7</sup> for all gene groups except the group with CDS &gt; 1700 bp) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The effect is more dramatic among genes with CDS lengths smaller than 800 bp (coefficient &lt; 0, <em>P</em> &lt; 10<sup>−15</sup>), which includes about one-third of all translated transcripts. For genes with CDS shorter than 800 bp, if the 5′ UTR is smaller than 200 bp, the TEI is significantly higher than for genes with a 5′ UTR longer than 200 bp (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <em>P</em> &lt; 0.0001, KS test). Particularly, when the 5′ UTR is longer than 400 bp, all genes have a low TEI (&lt;1) regardless of their CDS length (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>), suggesting that an extended 5′ UTR could be detrimental to translation. The same trend is observed in calli and panicles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS7.pptx">Figure S7</a>). For the 3′ UTR, there seems to be an optimal size for TEI, when the CDS is smaller than 500 bp (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>) (<em>P</em> &lt; 10<sup>−12</sup>, quadratic regression). The same trend is observed in calli but not in panicles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS7.pptx">Figure S7</a>). Overall, the size effect of the 5′ UTR on TEI is more substantial and consistent than that of the 3′ UTR.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/203/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between translatome enrichment index (TEI) and size of untranslated regions (UTRs) in shoots. (A) Variation of TEI with length of 5&#x2032; UTR. (B) Variation of TEI with length of 3&#x2032; UTR. Error bars correspond to 2 &#xD7; SE. CDS, coding sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645401416" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation between translatome enrichment index (TEI) and size of untranslated regions (UTRs) in shoots. (A) Variation of TEI with length of 5&#x2032; UTR. (B) Variation of TEI with length of 3&#x2032; UTR. Error bars correspond to 2 &#xD7; SE. CDS, coding sequence.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/203/F5.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/1/203/F5.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/203/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/203/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470085" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-54" class="first-child">Correlation between translatome enrichment index (TEI) and size of untranslated regions (UTRs) in shoots. (A) Variation of TEI with length of 5′ UTR. (B) Variation of TEI with length of 3′ UTR. Error bars correspond to 2 × SE. CDS, coding sequence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">If the GC content of the UTR is examined in relationship to the TEI, the TEI value declines with increasing GC content in the 5′ UTR, if both the CDS and the 5′ UTR are small (CDS &lt; 500 bp and 5′ UTR &lt; 200 bp; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS8.pptx">Figure S8</a>). This is consistent with the observation that high GC content in a 5′ UTR region significantly reduces ribosome loading (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kawaguchi and Bailey-Serres 2005</a>). In contrast, TEI values increase slightly with increasing GC content of 3′ UTRs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS8.pptx">Figure S8</a>), when CDS is &lt;500 bp. However, compared with the effect of size of the UTR, the correlation between GC content of UTRs and TEI is rather weak and the difference between most data points is not significant (<em>P</em> &gt; 0.05, KS test). Based on results from this analysis, the impact of RI in a 5′ UTR on translation is most likely due to its effect of increased size of the 5′ UTR, not due to the influence on GC content <em>per se</em>. Taken together, each portion of the transcript (5′ UTR, CDS, and 3′ UTR) plays a role in translation, depending on length, GC content, and possibly other sequence features or elements within these regions. The impact of UTRs is most significant for small genes. A summary of the correlation between these features and TEI is given in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>.</p><p id="p-56">Translation in eukaryotes is initialized by the recruitment of the 40S ribosome preinitiation complex and then the 60S ribosomal subunit to the 5′ end of the transcript, followed by subunit joining and ribosome translocation from codon-to-codon as elongation proceeds (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Araujo <em>et al.</em> 2012</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Browning and Bailey-Serres 2015</a>). It is conceivable that a short 5′ UTR is favorable for translation over a long 5′ UTR because of a lower energy requirement for unwinding the 5′ UTR during the preinitiation complex scanning process. Many 5′ UTRs also harbor one or more uORFs or upstream start codons (uAUGs), which often reduce frequency of initiation of translation of the main open ORF encoding the gene product. A large fraction of uORFs are indeed translated in <em>Arabidopsis</em> (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Juntawong <em>et al.</em> 2014</a>). Based on mammalian studies, some of the peptides encoded by uORFs might be functional (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ji <em>et al.</em> 2015</a>), and translation of uORFs seems to play key roles in the integrated stress response (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Starck <em>et al.</em> 2016</a>). However, when a uORF is translated, the ribosome reinitiation for the downstream main ORF is inefficient, which causes translational suppression of the main ORF (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Roy and von Arnim 2013</a>). Since longer 5′ UTRs more likely contain uORFs or uAUGs, it might be anticipated that longer 5′ UTRs are associated with lower TEI. It is worthy of mention that RPL18 is a component of the 60S ribosomal subunit, which only assembles on the mRNA once the start codon AUG is identified by the scanning preinitiation complex composed of the 40S subunit and initiation factors. As a result, only mRNAs that are associated with at least one 80S ribosome can be obtained by TRAP and hence are detected in this study. Moreover, translation is reduced by stable secondary structures present in the 5′ UTR (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Araujo <em>et al.</em> 2012</a>). This is likely to explain the minor negative impact of GC content of the 5′ UTR on TEI because GC-rich sequences are more likely to form thermodynamically stable structures (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Kawaguchi and Bailey-Serres 2005</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Nguyen and Southern 2000</a>).</p></div><div id="sec-19" class="subsection"><h3>Translatome enrichment of genes in different plant lineages</h3><p id="p-57">To consider GC content and TEI in the context of the evolutionary origin of rice genes, nontransposon genes were compared with genes from 12 other flowering plants and assigned to nonoverlapping plant lineages based on their placement within orthologous gene clusters generated with OrthoMCL. Genes in a lineage may have arisen in a specific evolutionary time frame. Classifications included genes conserved in all angiosperms (most ancestral), genes conserved between monocots and dicots (second ancestral), monocot-specific (third ancestral), Poaceae-specific (fourth ancestral), and rice (<em>O. sativa</em>)-specific genes (most recent) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS13.xlsx">Table S13</a> for list of rice genes in each lineage). When the GC content of the CDS of gene transcripts of each lineage was profiled (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>), genes conserved in all angiosperms had the lowest GC content, and the most recent genes (rice-specific) were associated with the highest fraction of GC-rich genes, indicating that the GC-rich module has been progressively enriched in the evolutionary history of rice. Notably, the GC-rich module is largely absent for transposable elements (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), which explains why transposons have rather low TEIs compared to nontransposon genes. The fluctuation of the GC content among different gene lineages may reflect the variation in activity of different gene duplication mechanisms. For example, it is known that Pack-MULEs, which refer to <em>Mutator</em>-like transposable elements carrying gene or gene fragments, have been highly active recently in the rice genome and they specifically duplicate GC-rich gene fragments (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jiang <em>et al.</em> 2011</a>). As a result, they may contribute to the high GC content of rice-specific genes. It is worthy of mention that, due to their ability to duplicate genes, Pack-MULEs have distinct GC content compared to other transposable elements (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ferguson <em>et al.</em> 2013</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/203/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GC content and translatome enrichment index (TEI) of rice genes based on their evolutionary origin. (A) GC content of rice non-TE genes based on orthologous clustering with predicted proteomes from 12 angiosperm species. (B) TEI of nontransposon genes in rice shoots based on orthologous clustering with predicted proteomes from 12 angiosperm species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645401416" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GC content and translatome enrichment index (TEI) of rice genes based on their evolutionary origin. (A) GC content of rice non-TE genes based on orthologous clustering with predicted proteomes from 12 angiosperm species. (B) TEI of nontransposon genes in rice shoots based on orthologous clustering with predicted proteomes from 12 angiosperm species.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/203/F6.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/1/203/F6.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/203/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/203/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470058" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-58" class="first-child">GC content and translatome enrichment index (TEI) of rice genes based on their evolutionary origin. (A) GC content of rice non-TE genes based on orthologous clustering with predicted proteomes from 12 angiosperm species. (B) TEI of nontransposon genes in rice shoots based on orthologous clustering with predicted proteomes from 12 angiosperm species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">The TEIs of genes in these different lineage groups were examined (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/FigureS9.pptx">Figure S9</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS13.xlsx">Table S13</a>). In each lineage and in all three tissues, there is a broad range of TEIs ranging from near “0” to &gt; 3. Consistent with our analyses based on GC content and the bimodal nature and enrichment of high GC content genes in rice, genes that are conserved among all angiosperms and those with overall lower GC content had the lowest fraction of genes with a TEI ≥ 1. This is also reflected in TEI distribution profiles; rice genes that are conserved in all angiosperms have a more discrete TEI distribution with a clear peak TEI value of ≤1, whereas all other rice genes have a broader TEI distribution regardless of whether they are restricted to monocots and dicots, monocot-specific, Poaceae-specific, or <em>O. sativa</em>-specific. Overall, these data suggest the evolutionarily recent genes in rice are more likely to be associated with ribosomes than core genes present in all angiosperms. Whether this is due to unique functions that arose through evolution or is a consequence of higher GC content found in rice and the Poaceae remains to be determined.</p></div><div id="sec-20" class="subsection"><h3>Genes involved in distinct biological processes are differentially enriched on ribosomes</h3><p id="p-60">The dramatic difference in TEI of different genes suggests that the translation process is tightly regulated; however, the functional relevance of such regulation is unclear. To this end, we examined the translatome enrichment of genes involved in different biological processes according to GO terms. The TEI values of genes for different processes vary significantly, with mean values ranging from 0.53 to 1.78 in shoots (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS14.xlsx">Table S14</a>). Interestingly, the process that is associated with the highest TEI in shoots is translation itself, followed by DNA metabolic process and photosynthesis (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Other processes associated with high TEIs are those involved in the generation of precursor metabolites and energy, cellular homeostasis, secondary metabolic processes, and the response to stress including both abiotic and biotic stresses. Most of these processes seem to be critical for the survival of the plants. The process associated with the lowest TEI is related to “regulation of gene expression, epigenetic,” followed by those involved in carbohydrate metabolism, lipid metabolism, growth, cell cycle and differentiation, protein modification, and multicellular organismal development (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>), which might be less critical for survival. Importantly, the trend observed in shoots is also seen in calli and panicles except that in calli, photosynthesis is not among the top three processes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS14.xlsx">Table S14</a>). This is understandable since photosynthesis is not active in calli, suggesting that translation regulation may be dependent on cellular energy status. In general, the genes involved in processes with low TEIs are associated with moderate expression abundance (mRNA-seq FPKM values ranging from 17 to 46), while genes in processes with high TEIs could be associated with very high expression abundance (mRNA-seq FPKM values ranging from 28 to 449) (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470043/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470043/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Ten biological processes in which genes are associated with the highest and lowest TEI in rice shoots</span><div class="sb-div caption-clear"></div></div></div><p id="p-66">Consistent with the analyses above, a large portion of the variation in TEI of genes in different processes can be attributed to the properties of the transcripts. All features (TEI, mRNA-seq level, TRAP-seq level, CDS size, 5′ UTR length, and GC content of CDS) described in <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> are significantly different between the genes with high TEIs and low TEIs (KS test, <em>P</em> &lt; 0.0001). In general, genes with high TEIs have short CDS, a short 5′ UTR, and high GC content in coding regions (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). Conversely, genes involved in the processes with low TEIs are longer than average (both CDS and 5′ UTR) and the GC content is lower than average (<a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>). Finally, the GC content of 5′ UTRs is not significantly different between the high TEI and low TEI groups (59.0 <em>vs.</em> 59.5, KS test, <em>P</em> = 0.56) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS14.xlsx">Table S14</a>). The GC content of 3′ UTRs is only slightly different (39.2 <em>vs.</em> 39.7 KS test, <em>P</em> = 0.06). Again, this is consistent with the notion that the GC content of UTRs is associated with a minor influence on translation in rice.</p></div><div id="sec-21" class="subsection"><h3>Relationship between codon usage bias, recombination rate, and TEI</h3><p id="p-67">As discussed above, the preferential association of GC-rich transcripts with ribosomes could be due to the physical property of the transcript, such as secondary structure. Nevertheless, it could be also due to preference on codon usage, which is often linked to tRNA abundance. It has been well established that codon usage differs among and within organisms, which is referred to as codon usage bias (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Behura and Severson 2013</a>). Studies showed that highly expressed genes exhibited high codon bias in <em>Escherichia coli</em>, <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Duret and Mouchiroud 1999</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sharp and Li 1986</a>). To determine whether the codon usage bias is associated with differential ribosomal enrichment, the codon usage of genes with the extremely low (&lt;0.2) and extremely high (≥3) TEIs were compared in shoots. As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS15.xlsx">Table S15</a>, all 18 amino acids with multiple synonymous codons exhibited different codon usage frequency. Overall, genes with low TEIs showed a minor preference to codons with A/U at the third codon position whereas those with high TEIs had extremely significant bias toward G/C at the third codon position. Further dissection of codon usage revealed that base composition of the first codon position also plays a role toward the usage bias (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS16.xlsx">Table S16</a>). For all four pairs of synonymous codons differing only at the first codon position, genes with high TEIs showed significant preference to C than to A/U. The pair with the most dramatic difference is CUG <em>vs.</em> UUG for leucine, where the ratio of these two codons increased from 0.83 for low TEI genes to 4.75 for high TEI genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS16.xlsx">Table S16</a>). The impact of codon composition at the third codon position was also further investigated. In addition to the general preference for G/C for high TEI genes, the frequency varies between G and C (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS16.xlsx">Table S16</a>). For some pairs of synonymous codons, G is more preferred to C in high TEI genes (<em>e.g.</em>, the ratios between CCG and CCC for proline is 0.95 for low TEI genes and 1.77 for high TEI genes). Other pairs demonstrate the opposite trend (<em>e.g.</em>, the ratios between CUG and CUC for leucine declines from 1.16 for low TEI genes to 0.68 for high TEI genes). Different frequency is also observed for codons with A and U at the third codon position (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS16.xlsx">Table S16</a>). Among nine pairs of synonymous codons with A <em>vs.</em> U at the third codon position, five pairs showed significant usage bias between low and high TEI genes (<em>P</em> &lt; 0.05). The codon discrimination between G and C as well as between A and U at the third codon position clearly suggests that codon usage plays a role in differential association of transcripts with ribosomes.</p><p id="p-68">The significant correlation between GC content and TEI suggests that the emergence of GC-rich genes in grasses could be a result of selection for translation efficiency. However, other factors have been proposed to be responsible for the evolution of the GC content of genes. gBGC is a process that results in gene alleles with G/C at heterozygous sites being preferentially retained compared with alleles with A/T at heterozygous sites during recombination, which would lead to a high GC content at regions with high recombination rates. Recently, a large body of evidence suggested that gBGC is likely a major force in shaping chromosomal base compositions in both plants and animals (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Duret and Galtier 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pessia <em>et al.</em> 2012</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ressayre <em>et al.</em> 2015</a>). To test if the GC content of genes is associated with the genetic recombination rate in rice, we used the local recombination rate from a previous study (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Tian <em>et al.</em> 2009</a>) and compared it with CDS GC content and TEI. A negligible correlation (correlation coefficient = 0.03, <em>P</em> &lt; 10<sup>−4</sup>) was detected between the GC content of CDS and the recombination rate in rice, which was in contrast to the situation observed in <em>Gallus gallus</em>, where GC content was well correlated with recombination rate (correlation coefficient = 0.89, <em>P</em> &lt; 10<sup>−4</sup>) (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Pessia <em>et al.</em> 2012</a>). This is consistent with the results from previous studies showing a lack of significant correlation between GC content and recombination rate in rice (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Pessia <em>et al.</em> 2012</a>; <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Tian <em>et al.</em> 2009</a>). Moreover, the influence of recombination rate on TEI seems to be even more minor (correlation coefficient = 0.019801, <em>P</em> = 0.01526). Accordingly, it is unclear whether gBGC has played a major role in shaping the GC content of genes in rice. Nevertheless, it should be noted that the recombination map that is used in this study contains &lt; 4000 markers, which is insufficient to distinguish the recombination rate of individual genes. Future studies with higher resolution recombination maps will facilitate our understanding of the contribution of gBGC to GC content evolution in rice. In addition, the contribution of selection on translation efficiency and gBGC to GC content variation could be complimentary instead of mutually exclusive. It is possible that GC-rich sequences were generated through gBGC and were retained because of their advantage in translation.</p></div><div id="sec-22" class="subsection"><h3>Conservation and divergence of translational features in rice and Arabidopsis</h3><p id="p-69">To test whether the differential enrichment of transcripts on ribosomes is present in other plants, a comparative analysis was conducted using the data from <em>Arabidopsis</em> seedlings growing under standard conditions on MS media (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Juntawong <em>et al.</em> 2014</a>). To make the two datasets as comparable as possible, our analysis only included the <em>Arabidopsis</em> genes that correspond to genes used for GO term analysis in rice (see <em>Materials and Methods</em> for details). The relative order of TEI among genes in different biological processes is largely conserved between the seedlings of <em>Arabidopsis</em> and shoots of rice. For example, 9 of the top 10 processes with the highest TEI are shared in the two species, including photosynthesis, translation, generation of energy, cellular homeostasis, and responses to stresses (<a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS17.doc">Table S17</a>). Similarly, 7 of the 10 processes with the least enriched transcripts on ribosomes are shared between the two species (<a id="xref-table-wrap-4-9" class="xref-table" href="#T4">Tables 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS17.doc">Table S17</a>), suggesting that divergent plants prioritize their biological processes similarly at the translational level.</p><p id="p-70">Although the relative order of transcript enrichment on ribosomes is largely conserved between <em>Arabidopsis</em> and rice, the degree of variation is quite different. In <em>Arabidopsis</em>, the mean TEI ranges from 0.91 to 1.14 (22% variation, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS17.doc">Table S17</a>) for different processes, whereas that for rice is from 0.53 to 1.78 (<a id="xref-table-wrap-4-10" class="xref-table" href="#T4">Tables 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS14.xlsx">Table S14</a>), with over threefold variation. The variation in CDS size as well as in 5′ UTR length between high TEI and low TEI processes is rather comparable between rice and <em>Arabidopsis</em> (33 <em>vs.</em> 32%, 27 <em>vs.</em> 20%, <a id="xref-table-wrap-4-11" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS17.doc">Table S17</a>). In contrast, the variation of GC content of CDS is considerably higher in rice than that in <em>Arabidopsis</em> (7.0 <em>vs.</em> 1.7%). Genome-wide, the TEI of genes in <em>Arabidopsis</em> demonstrates a nearly normal distribution, with a maximum fraction of genes around 1.0 (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Most genes (over 80%) are associated with a TEI ranging from 0.6 to 1.4; only a small portion of the genes (∼2%) are associated with a TEI ≥ 2 (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). Furthermore, there is little variation in the GC content of CDS in different TEI bins in <em>Arabidopsis</em>. In contrast, distribution of the TEI in rice is widely dispersed. Less than 40% of the genes are associated with a TEI of 0.6–1.4, and 17% of the genes are associated with a TEI ≥ 2. Moreover, the GC content and GC3 content (GC content of the third codon position) of CDS of rice genes progressively increases as the TEI does (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). As a result, the variation of GC content of genes in rice could contribute to the more dramatic variation of TEIs observed in rice. It should be pointed out that the <em>Arabidopsis</em> and rice plants were not grown under exactly the same conditions and were not sampled at equivalent developmental stages. Nevertheless, highly similarly trends were observed among three distinct rice tissues, so it is unlikely that the dramatic difference in TEI distribution between <em>Arabidopsis</em> and rice is solely due to tissue or developmental specificity.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/203/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fraction (%) of rice and Arabidopsis genes with different TEIs and their average GC and GC3 content of CDS. CDS, coding sequence; TEI, translatome enrichment index." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645401416" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fraction (%) of rice and &lt;em&gt;Arabidopsis&lt;/em&gt; genes with different TEIs and their average GC and GC3 content of CDS. CDS, coding sequence; TEI, translatome enrichment index.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/203/F7.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/1/203/F7.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/203/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/203/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470150" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-71" class="first-child">Fraction (%) of rice and <em>Arabidopsis</em> genes with different TEIs and their average GC and GC3 content of CDS. CDS, coding sequence; TEI, translatome enrichment index.</p><div class="sb-div caption-clear"></div></div></div><p id="p-72">On average, the CDS of rice genes is about 10% more GC-rich than that in <em>Arabidopsis</em> (55.6 <em>vs.</em> 45.1%, <a id="xref-table-wrap-4-12" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS17.doc">Table S17</a>), and GC-rich genes are associated with stronger codon usage bias (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Fennoy and Bailey-Serres 1993</a>). If the GC content of individual processes is examined, the increase of the GC content of rice genes (compared with that of <em>Arabidopsis</em>) in all the processes with low TEIs is less than the average value (<a id="xref-table-wrap-4-13" class="xref-table" href="#T4">Table 4</a>). In contrast, genes in 8 out of the 10 GO categories with high TEIs are associated with a GC content increase equal to or higher than average (<a id="xref-table-wrap-4-14" class="xref-table" href="#T4">Table 4</a>). The two exceptions are genes in translation and DNA metabolism; nevertheless, genes that encode proteins that function in translation have the smallest coding region in rice, and genes in DNA metabolism have experienced the largest reduction in CDS size (from <em>Arabidopsis</em> to rice) compared to other genes (<a id="xref-table-wrap-4-15" class="xref-table" href="#T4">Table 4</a>). As a consequence, in most cases the high TEI in rice is correlated with increased GC content of CDS but could be due to the relatively small size of genes. Again, it should be noted that high TEI does not necessarily mean high protein abundance; however, if GC-rich or short mRNAs are more likely associated with ribosomes, it may increase the competition for GC-poor or long mRNAs to be translated due to limited availability of the translation machinery.</p><p id="p-73">A strong example of such a relationship is the genes involved in “regulation of gene expression, epigenetic” (GO: 0040029) process. The genes in this process are associated with the largest and comparable mean CDS size in both <em>Arabidopsis</em> and rice (2163 and 2084 bp, respectively, <a id="xref-table-wrap-4-16" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS17.doc">Table S17</a>). In <em>Arabidopsis</em>, the mean TEI of genes in this process is 0.92 whereas in rice it is 0.53 (<a id="xref-table-wrap-4-17" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS17.doc">Table S17</a>). In rice, genes in all other GO categories have a substantially higher GC content (compared to their counterparts in <em>Arabidopsis</em>), ranging from 7 to 16% (<a id="xref-table-wrap-4-18" class="xref-table" href="#T4">Table 4</a>). In contrast, the GC content difference between <em>Arabidopsis</em> and rice genes associated with epigenetic regulation of expression was only 3%. In addition, genes associated with this GO category had the lowest average GC content (47.7%) among all GO-associated rice genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS14.xlsx">Table S14</a>). The extremely low GC content of those genes is associated with the lowest mean TEI in rice and this was observed in all three rice tissues (mean TEI = 0.53, 0.60, 0.46 in shoots, calli, and panicles, respectively, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1/TableS14.xlsx">Table S14</a>). Since genes that function in epigenetic regulation are responsible for the regulation of transposable element activity, it raises the question whether the compromise in translation of these genes in rice has contributed to the high degree of amplification of transposable elements in rice compared to <em>Arabidopsis</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">International Rice Genome Sequencing Project 2005</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">The Arabidopsis Genome Initiative 2000</a>). Future analysis is required to test this hypothesis.</p></div><div id="sec-23" class="subsection"><h3>Conclusions</h3><p id="p-74">In this study, we provide a genome-wide analysis of the translation status of genes in rice. Based on our analysis, it is clear that, for individual gene transcripts, the fraction that is ribosome-associated can vary dramatically. In rice, a large portion of the variation can be attributed to the length and the GC content of the coding region, suggesting that the evolution of GC-rich genes in monocots (especially in grasses) may be associated with an advantage in translation. In addition, UTRs, especially 5′ UTRs, play a critical role in translation. Small genes are preferentially associated with ribosomes; however, if the transcript has a 5′ UTR that is longer than 400 bp, the favorable effect of the short transcript is masked by the inhibitory effect of a long 5′ UTR.</p><p id="p-75">Genes encoding proteins associated with different processes are differentially enriched on ribosomes. Genes involved in processes that are critical for plant survival, such as those related to photosynthesis, translation, and stresses responses, are more likely associated with ribosomes. Compared to corresponding translation status in <em>Arabidopsis</em>, mRNAs in rice have a more dramatic variation of TEI, which may result in differential regulation at the translational level that could be linked to phenotypic diversity. The wide variation of TEI in rice is correlated with more substantial variation of the GC content of genes, which is largely due to the emergence of GC-rich genes after the formation of angiosperms. Clearly, genes in different biological processes evolve differentially in terms of GC content, which may alter the dynamics of the relative translation efficiency of these processes. Since high TEI genes demonstrate an extraordinary bias toward GC-rich codons, it is likely that the genes in critical biological processes are more likely to retain AT to GC mutations. However, from this study, it is unclear whether AT to GC mutations are more frequently generated in some genes than in other genes, through processes such as gBGC. Future analysis with high resolution recombination maps will determine the role of gBGC in GC content evolution in rice. Despite the uncertainty of sources of GC biased mutation, we demonstrated that the low GC content of genes in epigenetic pathways in rice may be disadvantageous for their translation.</p><p id="p-76">Future experimentation is required to test whether an enhanced TEI value is associated with greater protein production. If it is, factors that impact TEI can be utilized for future crop engineering. For example, more efficient translation of a target gene can potentially be achieved by using optimal (high frequency codons found within genes with high TEI) codons or minimizing the 5′ UTR. For genes with very short coding regions, a 3′ UTR with an appropriate length is beneficial for its translation. It should be noted that, in this study, the observations are made based on only three rice tissues, with two of them (callus and young panicle) composed of a mixture of rather divergent cells. Experiments with more tissues, especially those from more specialized organs, will reveal whether translational regulation is controlled in a tissue-specific fashion. In addition, all experiments conducted in this study are under standard growth conditions, and it will be intriguing to study the impact of environmental factors on translation such as the inhibition of translation initiation under hypoxia stress in <em>Arabidopsis</em> (<a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Juntawong <em>et al.</em> 2014</a>). Furthermore, it would be interesting to test where the RI transcript isoforms are located (nucleus or cytoplasm) and whether their relative abundance is reduced under stressed conditions.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-77">We are grateful to Yinong Yang and Chuansheng Mei for providing the vector pCAMBIA1300S and Takeshi Fukao for preparing the RPL18 cDNA from rice. We thank Kevin Childs and Tiffany Liu for providing custom scripts for calculation of codon usage frequency. This study is supported by National Science Foundation grants MCB-1121650 (Y.C., J.J., C.R.B., and N.J.) and DBI-1238243 (J.B.-S.). It is also supported by the US Department of Agriculture National Institute of Food and Agriculture and AgBioResearch at Michigan State University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-12"><p id="p-78">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036020/-/DC1</a>.</p></li><li class="fn" id="fn-13"><p id="p-79"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-28"><span class="received-label">Received </span>September 28, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-03"><span class="accepted-label">Accepted </span>November 3, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Zhao <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.1" data-doi="10.1155/2012/475731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araujo</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Before it gets started: regulating translation at the 5′ UTR.</span> <abbr class="cit-jnl-abbrev">Comp. Funct. Genomics</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">475731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComp%2BFunct%2BGenomics%26rft.aulast%253DAraujo%26rft.auinit1%253DP.%2BR.%26rft.volume%253D2012%26rft.spage%253D475731%26rft.epage%253D475731%26rft.atitle%253DBefore%2BIt%2BGets%2BStarted%253A%2BRegulating%2BTranslation%2Bat%2Bthe%2B5%2527%2BUTR.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2012%252F475731%26rft_id%253Dinfo%253Apmid%252F22693426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2012/475731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22693426&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.2" data-doi="10.1073/pnas.0635171100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arava</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide analysis of mRNA translation profiles in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">3889</span>–<span class="cit-lpage">3894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0635171100%26rft_id%253Dinfo%253Apmid%252F12660367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzcvMzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.3" data-doi="10.1111/j.1469-185X.2012.00242.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behura</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severson</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D88%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.2012.00242.x%26rft_id%253Dinfo%253Apmid%252F22889422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.2012.00242.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22889422&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.4" data-doi="10.1093/biostatistics/kxj037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxj037%26rft_id%253Dinfo%253Apmid%252F16632515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxj037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16632515&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.6" data-doi="10.1016/j.devcel.2013.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boothby</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zipper</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Weele</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolniak</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Removal of retained introns regulates translation in the rapidly developing gametophyte of <em>Marsilea vestita</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D24%26rft.spage%253D517%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2013.01.015%26rft_id%253Dinfo%253Apmid%252F23434411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2013.01.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23434411&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316163000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.7" data-doi="10.1199/tab.0176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mechanism of cytoplasmic mRNA translation.</span> <abbr class="cit-jnl-abbrev">Arabidopsis Book</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArabidopsis%2BBook%26rft.volume%253D13%26rft.spage%253De0176%26rft_id%253Dinfo%253Adoi%252F10.1199%252Ftab.0176%26rft_id%253Dinfo%253Apmid%252F26019692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1199/tab.0176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26019692&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckley</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaladkar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberwine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cytoplasmic intron retention, function, splicing, and the sentinel RNA hypothesis.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. RNA</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BRNA%26rft.volume%253D5%26rft.spage%253D223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.9" data-doi="10.1186/1471-2164-7-327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCampbell%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.spage%253D327%26rft.epage%253D327%26rft.atitle%253DComprehensive%2Banalysis%2Bof%2Balternative%2Bsplicing%2Bin%2Brice%2Band%2Bcomparative%2Banalyses%2Bwith%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-327%26rft_id%253Dinfo%253Apmid%252F17194304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194304&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.10" data-doi="10.1104/pp.107.104513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification and characterization of lineage-specific genes within the Poaceae.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1311</span>–<span class="cit-lpage">1322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.104513%26rft_id%253Dinfo%253Apmid%252F17951464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQ1LzQvMTMxMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.11" data-doi="10.3389/fgene.2012.00055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carels</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Two classes of genes in plants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCarels%26rft.auinit1%253DN.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1825%26rft.atitle%253DTwo%2BClasses%2Bof%2BGenes%2Bin%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00055%26rft_id%253Dinfo%253Apmid%252F10747072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.12" data-doi="10.1073/pnas.0307827100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hottes</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAdams</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage between genomes is constrained by genome-wide mutational processes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">3480</span>–<span class="cit-lpage">3485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307827100%26rft_id%253Dinfo%253Apmid%252F14990797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEwLzM0ODAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.13" data-doi="10.1093/gbe/evu278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fustier</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nabholz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glemin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The bimodal distribution of genic GC content is ancestral to monocot species.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu278%26rft_id%253Dinfo%253Apmid%252F25527839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25527839&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.14" data-doi="10.2135/cropsci2000.402436x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Counce</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keisling</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A uniform, objective, and adaptive system for expressing rice development.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCounce%26rft.auinit1%253DP.%2BA.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D436%26rft.epage%253D443%26rft.atitle%253DA%2BUniform%252C%2BObjective%252C%2Band%2BAdaptive%2BSystem%2Bfor%2BExpressing%2BRice%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2000.402436x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2000.402436x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089410300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.15" data-doi="10.2307/4119796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue.</span> <abbr class="cit-jnl-abbrev">Phytochem. Bull.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochem.%2BBull.%26rft.volume%253D19%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.2307%252F4119796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/4119796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.16" data-doi="10.1073/pnas.96.8.4482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4482</span>–<span class="cit-lpage">4487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D96%26rft.issue%253D8%26rft.spage%253D4482%26rft.epage%253D4487%26rft.atitle%253DExpression%2Bpattern%2Band%252C%2Bsurprisingly%252C%2Bgene%2Blength%2Bshape%2Bcodon%2Busage%2Bin%2BCaenorhabditis%252C%2BDrosophila%252C%2Band%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.8.4482%26rft_id%253Dinfo%253Apmid%252F10200288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni84LzQ0ODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.17" data-doi="10.1146/annurev-genom-082908-150001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biased gene conversion and the evolution of mammalian genomic landscapes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D10%26rft.spage%253D285%26rft.epage%253D311%26rft.atitle%253DBiased%2Bgene%2Bconversion%2Band%2Bthe%2Bevolution%2Bof%2Bmammalian%2Bgenomic%2Blandscapes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082908-150001%26rft_id%253Dinfo%253Apmid%252F19630562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082908-150001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19630562&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">E.</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Splicing and alternative splicing in rice and humans.</span> <abbr class="cit-jnl-abbrev">BMB Rep.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMB%2BRep.%26rft.volume%253D46%26rft.spage%253D439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.19" data-doi="10.1093/nar/21.23.5294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fennoy</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Synonymous codon usage in <em>Zea mays</em> L. nuclear genes is varied by levels of C and G-ending codons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5294</span>–<span class="cit-lpage">5300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFennoy%26rft.auinit1%253DS.%2BL.%26rft.volume%253D21%26rft.issue%253D23%26rft.spage%253D5294%26rft.epage%253D5300%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BZea%2Bmays%2BL.%2Bnuclear%2Bgenes%2Bis%2Bvaried%2Bby%2Blevels%2Bof%2BC%2Band%2BG-ending%2Bcodons%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F21.23.5294%26rft_id%253Dinfo%253Apmid%252F8265340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/21.23.5294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8265340&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MK39700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.20" data-doi="10.1104/pp.113.223271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Selective acquisition and retention of genomic sequences by should be Pack-<em>Mutator</em>-like elements based on guanine-cytosine content and the breadth of expression.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1419</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.223271%26rft_id%253Dinfo%253Apmid%252F24028844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTYzLzMvMTQxOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.21" data-doi="10.1016/j.tig.2014.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glemin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ressayre</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">GC content evolution in coding regions of angiosperm genomes: a unifying hypothesis.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.05.002%26rft_id%253Dinfo%253Apmid%252F24916172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24916172&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338915700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.22" data-doi="10.1016/j.febslet.2007.01.088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evidence of selectively driven codon usage in rice: implications for GC content evolution of <em>Gramineae</em> genes.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">581</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DGuo%26rft.auinit1%253DX.%26rft.volume%253D581%26rft.issue%253D5%26rft.spage%253D1015%26rft.epage%253D1021%26rft.atitle%253DEvidence%2Bof%2Bselectively%2Bdriven%2Bcodon%2Busage%2Bin%2Brice%253A%2Bimplications%2Bfor%2BGC%2Bcontent%2Bevolution%2Bof%2BGramineae%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2007.01.088%26rft_id%253Dinfo%253Apmid%252F17306258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2007.01.088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17306258&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244941100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.23" data-doi="10.1016/j.ymeth.2009.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halbeisen</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherrer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Affinity purification of ribosomes to access the translatome.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DHalbeisen%26rft.auinit1%253DR.%2BE.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D306%26rft.epage%253D310%26rft.atitle%253DAffinity%2Bpurification%2Bof%2Bribosomes%2Bto%2Baccess%2Bthe%2Btranslatome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2009.04.003%26rft_id%253Dinfo%253Apmid%252F19398006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2009.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19398006&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268359300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Helmy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugiyama</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishihama</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The rice proteogenomics database oryzaPG-DB: development, expansion, and new features.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DHelmy%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.spage%253D65%26rft.epage%253D65%26rft.atitle%253DThe%2BRice%2BProteogenomics%2BDatabase%2BOryzaPG-DB%253A%2BDevelopment%252C%2BExpansion%252C%2Band%2BNew%2BFeatures.%26rft_id%253Dinfo%253Apmid%252F22639657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22639657&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.25" data-doi="10.1073/pnas.93.15.7783"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirochika</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugimoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otsuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsugawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanda</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Retrotransposons of rice involved in mutations induced by tissue culture.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">7783</span>–<span class="cit-lpage">7788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHirochika%26rft.auinit1%253DH.%26rft.volume%253D93%26rft.issue%253D15%26rft.spage%253D7783%26rft.epage%253D7788%26rft.atitle%253DRetrotransposons%2Bof%2Brice%2Binvolved%2Bin%2Bmutations%2Binduced%2Bby%2Btissue%2Bculture.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.15.7783%26rft_id%253Dinfo%253Apmid%252F8755553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTMvMTUvNzc4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.26" data-doi="10.1093/nar/gkt995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hupe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gertow Gillner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenman</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of TRAP-sequencing technology with a versatile conditional mouse model.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">e14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt995%26rft_id%253Dinfo%253Apmid%252F24165879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24165879&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.27" data-doi="10.1186/gb-2003-5-1-r2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hurowitz</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide analysis of mRNA lengths in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHurowitz%26rft.auinit1%253DE.%2BH.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253DR2%26rft.epage%253DR2%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BmRNA%2Blengths%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2003-5-1-r2%26rft_id%253Dinfo%253Apmid%252F14709174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2003-5-1-r2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14709174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.28" data-doi="10.1038/nrg3645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ribosome profiling: new views of translation, from single codons to genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3645%26rft_id%253Dinfo%253Apmid%252F24468696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24468696&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.29" data-doi="10.1038/nature03895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Rice Genome Sequencing Project</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The map-based sequence of the rice genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D436%26rft.issue%253D7052%26rft.spage%253D793%26rft.epage%253D800%26rft.atitle%253DThe%2Bmap-based%2Bsequence%2Bof%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03895%26rft_id%253Dinfo%253Apmid%252F16100779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16100779&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231116500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.30" data-doi="10.7554/eLife.08890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struhl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Many lncRNAs, 5′UTRs, and pseudogenes are translated and some are likely to express functional proteins.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e08890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.08890%26rft_id%253Dinfo%253Apmid%252F26687005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.08890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26687005&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.31" data-doi="10.1038/nature01214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirochika</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">An active DNA transposon family in rice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJiang%26rft.auinit1%253DN.%26rft.volume%253D421%26rft.issue%253D6919%26rft.spage%253D163%26rft.epage%253D167%26rft.atitle%253DAn%2Bactive%2BDNA%2Btransposon%2Bfamily%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01214%26rft_id%253Dinfo%253Apmid%252F12520302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12520302&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180267200040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.32" data-doi="10.1073/pnas.1010814108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotkin</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisch</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pack-<em>Mutator</em>-like transposable elements (Pack-MULEs) induce directional modification of genes through biased insertion and DNA acquisition.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1537</span>–<span class="cit-lpage">1542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010814108%26rft_id%253Dinfo%253Apmid%252F21220310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.33" data-doi="10.1038/msb.2010.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyerowitz</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cell-type specific analysis of translating RNAs in developing flowers reveals new levels of control.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DJiao%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D419%26rft.epage%253D419%26rft.atitle%253DCell-type%2Bspecific%2Banalysis%2Bof%2Btranslating%2BRNAs%2Bin%2Bdeveloping%2Bflowers%2Breveals%2Bnew%2Blevels%2Bof%2Bcontrol%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2010.76%26rft_id%253Dinfo%253Apmid%252F20924354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjYvMS80MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.34" data-doi="10.1073/pnas.1317811111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juntawong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Translational dynamics revealed by genome-wide profiling of ribosome footprints in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E203</span>–<span class="cit-lpage">E212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1317811111%26rft_id%253Dinfo%253Apmid%252F24367078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzEvRTIwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.35" data-doi="10.1093/nar/gkr932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalyna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syed</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewandowska</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marquez</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Alternative splicing and nonsense-mediated decay modulate expression of important regulatory genes in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">2454</span>–<span class="cit-lpage">2469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr932%26rft_id%253Dinfo%253Apmid%252F22127866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22127866&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302312400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.36" data-doi="10.1007/s002390010219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanaya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinouchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKanaya%26rft.auinit1%253DS.%26rft.volume%253D53%26rft.issue%253D4-5%26rft.spage%253D290%26rft.epage%253D298%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bgenes%2Bin%2Beukaryotes%253A%2Bcorrelation%2Bof%2Bcodon%2Busage%2Bdiversity%2Bwith%2Btranslation%2Befficiency%2Band%2Bwith%2BCG-dinucleotide%2Busage%2Bas%2Bassessed%2Bby%2Bmultivariate%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002390010219%26rft_id%253Dinfo%253Apmid%252F11675589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002390010219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11675589&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171169600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.37" data-doi="10.1093/nar/gki240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">mRNA sequence features that contribute to translational regulation in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKawaguchi%26rft.auinit1%253DR.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D955%26rft.epage%253D965%26rft.atitle%253DmRNA%2Bsequence%2Bfeatures%2Bthat%2Bcontribute%2Bto%2Btranslational%2Bregulation%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki240%26rft_id%253Dinfo%253Apmid%252F15716313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716313&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227407300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.38" data-doi="10.1186/1939-8433-6-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawahara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Bastide</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improvement of the <em>Oryza sativa</em> Nipponbare reference genome using next generation sequence and optical map data.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D6%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1939-8433-6-4%26rft_id%253Dinfo%253Apmid%252F24280374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1939-8433-6-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24280374&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khush</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Strategies for increasing the yield potential of cereals: case of rice as an example.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.40" data-doi="10.1038/nature01218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terauchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirano</span>  <span class="cit-name-given-names">H. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">a</span> <span class="cit-article-title">The plant MITE <em>mPing</em> is mobilized in anther culture.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKikuchi%26rft.auinit1%253DK.%26rft.volume%253D421%26rft.issue%253D6919%26rft.spage%253D167%26rft.epage%253D170%26rft.atitle%253DThe%2Bplant%2BMITE%2BmPing%2Bis%2Bmobilized%2Bin%2Banther%2Bculture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01218%26rft_id%253Dinfo%253Apmid%252F12520303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12520303&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180267200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.41" data-doi="10.1126/science.1081288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawagashira</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">b</span> <span class="cit-article-title">Collection, mapping, and annotation of over 28,000 cDNA clones from <em>japonica</em> rice.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKikuchi%26rft.auinit1%253DS.%26rft.volume%253D301%26rft.issue%253D5631%26rft.spage%253D376%26rft.epage%253D379%26rft.atitle%253DCollection%252C%2BMapping%252C%2Band%2BAnnotation%2Bof%2BOver%2B28%252C000%2BcDNA%2BClones%2Bfrom%2Bjaponica%2BRice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1081288%26rft_id%253Dinfo%253Apmid%252F12869764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDEvNTYzMS8zNzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeland</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landweber</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">2</span>: RESEARCH0010.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.43" data-doi="10.1073/pnas.172183199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Washburn</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andon</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deciu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Proteomic survey of metabolic pathways in rice.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">11969</span>–<span class="cit-lpage">11974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.172183199%26rft_id%253Dinfo%253Apmid%252F12163647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMTgvMTE5NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.44" data-doi="10.1007/s11103-005-2160-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komatsu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rice proteome database: a step toward functional analysis of the rice genome.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DKomatsu%26rft.auinit1%253DS.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D190%26rft.atitle%253DRice%2Bproteome%2Bdatabase%253A%2Ba%2Bstep%2Btoward%2Bfunctional%2Banalysis%2Bof%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-2160-z%26rft_id%253Dinfo%253Apmid%252F16217611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-2160-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16217611&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232498000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.45" data-doi="10.1371/journal.pbio.0040180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipinski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caffin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helwak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zylicz</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High guanine and cytosine content increases mRNA levels in mammalian cells.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De180%26rft.epage%253De180%26rft.atitle%253DHigh%2Bguanine%2Band%2Bcytosine%2Bcontent%2Bincreases%2BmRNA%2Blevels%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040180%26rft_id%253Dinfo%253Apmid%252F16700628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16700628&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.46" data-doi="10.1016/j.molcel.2007.03.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lackner</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beilharz</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marguerat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mata</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A network of multiple regulatory layers shapes gene expression in fission yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLackner%26rft.auinit1%253DD.%2BH.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D155%26rft.atitle%253DA%2Bnetwork%2Bof%2Bmultiple%2Bregulatory%2Blayers%2Bshapes%2Bgene%2Bexpression%2Bin%2Bfission%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.03.002%26rft_id%253Dinfo%253Apmid%252F17434133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17434133&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246075400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.47" data-doi="10.1371/journal.pgen.1002527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavut</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raveh</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequestration of highly expressed mRNAs in cytoplasmic granules, P-bodies, and stress granules enhances cell viability.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLavut%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253De1002527%26rft.epage%253De1002527%26rft.atitle%253DSequestration%2Bof%2Bhighly%2Bexpressed%2BmRNAs%2Bin%2Bcytoplasmic%2Bgranules%252C%2BP-bodies%252C%2Band%2Bstress%2Bgranules%2Benhances%2Bcell%2Bviability.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002527%26rft_id%253Dinfo%253Apmid%252F22383896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22383896&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.48" data-doi="10.1111/tpj.13073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ribosome profiling reveals dynamic translational landscape in maize seedlings under drought stress.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1206</span>–<span class="cit-lpage">1218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D1206%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13073%26rft_id%253Dinfo%253Apmid%252F26568274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26568274&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.49" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.50" data-doi="10.1016/j.gene.2007.05.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Nature of selective constraints on synonymous codon usage of rice differs in GC-poor and GC-rich genes.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">400</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMukhopadhyay%26rft.auinit1%253DP.%26rft.volume%253D400%26rft.issue%253D1-2%26rft.spage%253D71%26rft.epage%253D81%26rft.atitle%253DNature%2Bof%2Bselective%2Bconstraints%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bof%2Brice%2Bdiffers%2Bin%2BGC-poor%2Band%2BGC-rich%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2007.05.027%26rft_id%253Dinfo%253Apmid%252F17629420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2007.05.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17629420&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249539600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.51" data-doi="10.1111/j.1399-3054.1962.tb08052.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murashige</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skoog</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-article-title">A revised medium for rapid growth and bioassays with tobacco tissue cultures.</span> <abbr class="cit-jnl-abbrev">Physiol. Plant.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BPlant.%26rft.volume%253D15%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-3054.1962.tb08052.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1399-3054.1962.tb08052.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.52" data-doi="10.1007/978-1-60327-563-7_6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mustroph</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juntawong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Isolation of plant polysomal mRNA by differential centrifugation and ribosome immunopurification methods.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">553</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DMustroph%26rft.auinit1%253DA.%26rft.volume%253D553%26rft.spage%253D109%26rft.epage%253D126%26rft.atitle%253DIsolation%2Bof%2Bplant%2Bpolysomal%2BmRNA%2Bby%2Bdifferential%2Bcentrifugation%2Band%2Bribosome%2Bimmunopurification%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-563-7_6%26rft_id%253Dinfo%253Apmid%252F19588103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-563-7_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19588103&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271177400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.53" data-doi="10.1073/pnas.0906131106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mustroph</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanetti</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtan</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Repetti</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Profiling translatomes of discrete cell populations resolves altered cellular priorities during hypoxia in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">18843</span>–<span class="cit-lpage">18848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0906131106%26rft_id%253Dinfo%253Apmid%252F19843695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQ0LzE4ODQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.54" data-doi="10.1104/pp.109.151845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mustroph</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oosumi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanetti</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cross-kingdom comparison of transcriptomic adjustments to low-oxygen stress highlights conserved and plant-specific responses.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1484</span>–<span class="cit-lpage">1500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.109.151845%26rft_id%253Dinfo%253Apmid%252F20097791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTUyLzMvMTQ4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.55" data-doi="10.1038/nature01219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakazaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horibata</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamahira</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teraishi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mobilization of a transposon in the rice genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNakazaki%26rft.auinit1%253DT.%26rft.volume%253D421%26rft.issue%253D6919%26rft.spage%253D170%26rft.epage%253D172%26rft.atitle%253DMobilization%2Bof%2Ba%2Btransposon%2Bin%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01219%26rft_id%253Dinfo%253Apmid%252F12520304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12520304&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180267200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.56" data-doi="10.1186/1471-2229-10-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narsai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whelan</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Defining reference genes in <em>Oryza sativa</em> using organ, development, biotic and abiotic transcriptome datasets.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DNarsai%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DDefining%2Breference%2Bgenes%2Bin%2BOryza%2Bsativa%2Busing%2Borgan%252C%2Bdevelopment%252C%2Bbiotic%2Band%2Babiotic%2Btranscriptome%2Bdatasets.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-10-56%26rft_id%253Dinfo%253Apmid%252F20353606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-10-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20353606&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.57" data-doi="10.1093/nar/28.20.3904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southern</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Minimising the secondary structure of DNA targets by incorporation of a modified deoxynucleoside: implications for nucleic acid analysis by hybridisation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3904</span>–<span class="cit-lpage">3909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNguyen%26rft.auinit1%253DH.-K.%26rft.volume%253D28%26rft.issue%253D20%26rft.spage%253D3904%26rft.epage%253D3909%26rft.atitle%253DMinimising%2Bthe%2Bsecondary%2Bstructure%2Bof%2BDNA%2Btargets%2Bby%2Bincorporation%2Bof%2Ba%2Bmodified%2Bdeoxynucleoside%253A%2Bimplications%2Bfor%2Bnucleic%2Bacid%2Banalysis%2Bby%2Bhybridisation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.20.3904%26rft_id%253Dinfo%253Apmid%252F11024169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.20.3904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11024169&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089954300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.58" data-doi="10.1038/ng.259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shai</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blencowe</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1413</span>–<span class="cit-lpage">1415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPan%26rft.auinit1%253DQ.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1413%26rft.epage%253D1415%26rft.atitle%253DDeep%2Bsurveying%2Bof%2Balternative%2Bsplicing%2Bcomplexity%2Bin%2Bthe%2Bhuman%2Btranscriptome%2Bby%2Bhigh-throughput%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.259%26rft_id%253Dinfo%253Apmid%252F18978789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18978789&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261215900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.59" data-doi="10.1104/pp.112.194928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juntawong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Posttranscriptional control of photosynthetic mRNA decay under stress conditions requires 3′ and 5′ untranslated regions and correlates with differential polysome association in rice.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.194928%26rft_id%253Dinfo%253Apmid%252F22566494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU5LzMvMTExMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.60" data-doi="10.1093/gbe/evs052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pessia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mousset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rezvoy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evidence for widespread GC-biased gene conversion in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs052%26rft_id%253Dinfo%253Apmid%252F22628461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evs052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22628461&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.61" data-doi="10.1186/1471-2164-13-528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milhem</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loubiere</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cocaign-Bousquet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bacterial translational regulations: high diversity between all mRNAs and major role in gene expression.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DPicard%26rft.auinit1%253DF.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D528%26rft.epage%253D528%26rft.atitle%253DBacterial%2Btranslational%2Bregulations%253A%2Bhigh%2Bdiversity%2Bbetween%2Ball%2BmRNAs%2Band%2Bmajor%2Brole%2Bin%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-528%26rft_id%253Dinfo%253Apmid%252F23036066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-528&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23036066&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.62" data-doi="10.1111/j.1365-313X.2009.03813.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picault</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaparro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piegu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenger</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Formey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of an active LTR retrotransposon in rice.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DPicault%26rft.auinit1%253DN.%26rft.volume%253D58%26rft.issue%253D5%26rft.spage%253D754%26rft.epage%253D765%26rft.atitle%253DIdentification%2Bof%2Ban%2Bactive%2BLTR%2Bretrotransposon%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03813.x%26rft_id%253Dinfo%253Apmid%252F19187041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03813.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19187041&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.63" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.64" data-doi="10.1093/gbe/evv189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ressayre</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glemin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montalent</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serre-Giardi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Introns structure patterns of variation in nucleotide composition in <em>Arabidopsis thaliana</em> and rice protein-coding genes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2913</span>–<span class="cit-lpage">2928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv189%26rft_id%253Dinfo%253Apmid%252F26450849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26450849&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.65" data-doi="10.1007/978-1-4939-2444-8_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynoso</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juntawong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Translating Ribosome Affinity Purification (TRAP) followed by RNA sequencing technology (TRAP-SEQ) for quantitative assessment of plant translatomes.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1284</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1284%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2444-8_9%26rft_id%253Dinfo%253Apmid%252F25757773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2444-8_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25757773&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.66" data-doi="10.1199/tab.0165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Arnim</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Translational regulation of cytoplasmic mRNAs.</span> <abbr class="cit-jnl-abbrev">Arabidopsis Book</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArabidopsis%2BBook%26rft.volume%253D11%26rft.spage%253De0165%26rft_id%253Dinfo%253Adoi%252F10.1199%252Ftab.0165%26rft_id%253Dinfo%253Apmid%252F23908601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1199/tab.0165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23908601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.67" data-doi="10.1111/j.1365-313X.2011.04492.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picault</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Baidouri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llauro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaparro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transpositional landscape of the rice genome revealed by paired-end mapping of high-throughput re-sequencing data.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSabot%26rft.auinit1%253DF.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D241%26rft.epage%253D246%26rft.atitle%253DTranspositional%2Blandscape%2Bof%2Bthe%2Brice%2Bgenome%2Brevealed%2Bby%2Bpaired-end%2Bmapping%2Bof%2Bhigh-throughput%2Bre-sequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04492.x%26rft_id%253Dinfo%253Apmid%252F21219509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04492.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21219509&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289521400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.68" data-doi="10.1016/j.bbagrm.2013.02.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schweingruber</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rufener</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zund</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamashita</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhlemann</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonsense-mediated mRNA decay — mechanisms of substrate mRNA recognition and degradation in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1829</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DSchweingruber%26rft.volume%253D1829%26rft.spage%253D612%26rft.atitle%253DNonsense-mediated%2BmRNA%2Bdecay%2B-%2BMechanisms%2Bof%2Bsubstrate%2BmRNA%2Brecognition%2Band%2Bdegradation%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2013.02.005%26rft_id%253Dinfo%253Apmid%252F23435113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2013.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23435113&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.69" data-doi="10.1016/j.cell.2013.05.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niemczyk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rate-limiting steps in yeast protein translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1589</span>–<span class="cit-lpage">1601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D1589%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.05.049%26rft_id%253Dinfo%253Apmid%252F23791185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.05.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23791185&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320722000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.70" data-doi="10.1007/BF02099948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D24%26rft.issue%253D1-2%26rft.spage%253D28%26rft.epage%253D38%26rft.atitle%253DAn%2Bevolutionary%2Bperspective%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bunicellular%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02099948%26rft_id%253Dinfo%253Apmid%252F3104616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02099948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3104616&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F784400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.71" data-doi="10.1126/science.aad3867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Starck</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shodiya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Translation from the 5′ untranslated region shapes the integrated stress response.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">aad3867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStarck%26rft.auinit1%253DS.%2BR.%26rft.volume%253D351%26rft.issue%253D6272%26rft.spage%253Daad3867%26rft.epage%253Daad3867%26rft.atitle%253DTranslation%2Bfrom%2Bthe%2B5%2527%2Buntranslated%2Bregion%2Bshapes%2Bthe%2Bintegrated%2Bstress%2Bresponse%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad3867%26rft_id%253Dinfo%253Apmid%252F26823435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTEvNjI3Mi9hYWQzODY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.72" data-doi="10.1186/1471-2164-11-308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatarinova</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouck</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">GC3 biology in corn, rice, sorghum and other grasses.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTatarinova%26rft.auinit1%253DT.%2BV.%26rft.volume%253D11%26rft.spage%253D308%26rft.epage%253D308%26rft.atitle%253DGC3%2Bbiology%2Bin%2Bcorn%252C%2Brice%252C%2Bsorghum%2Band%2Bother%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-308%26rft_id%253Dinfo%253Apmid%252F20470436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20470436&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.73" data-doi="10.1038/35048692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Arabidopsis Genome Initiative</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D796%26rft.epage%253D815%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bflowering%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048692%26rft_id%253Dinfo%253Apmid%252F11130711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130711&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.74" data-doi="10.1101/gr.083899.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Do genetic recombination and gene density shape the pattern of DNA elimination in rice long terminal repeat retrotransposons?</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2221</span>–<span class="cit-lpage">2230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083899.108%26rft_id%253Dinfo%253Apmid%252F19789376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi8yMjIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.75" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19289445&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265523300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.76" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.77" data-doi="10.1073/pnas.0602039103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brendel</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomewide comparative analysis of alternative splicing in plants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">7175</span>–<span class="cit-lpage">7180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0602039103%26rft_id%253Dinfo%253Apmid%252F16632598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzE4LzcxNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.78" data-doi="10.1038/nature07509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khrebtukova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Alternative isoform regulation in human tissue transcriptomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWang%26rft.auinit1%253DE.%2BT.%26rft.volume%253D456%26rft.issue%253D7221%26rft.spage%253D470%26rft.epage%253D476%26rft.atitle%253DAlternative%2Bisoform%2Bregulation%2Bin%2Bhuman%2Btissue%2Btranscriptomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07509%26rft_id%253Dinfo%253Apmid%252F18978772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18978772&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261170500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid divergence of codon usage patterns within the rice genome.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>(<span class="cit-supplement">Suppl. 1</span>): <span class="cit-fpage">S6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D7%26rft.spage%253DS6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.80" data-doi="10.1104/pp.103.022749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner-Fraczek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Regulated phosphorylation of 40S ribosomal protein S6 in root tips of maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.022749%26rft_id%253Dinfo%253Apmid%252F12913163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTMyLzQvMjA4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8yMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.81" data-doi="10.1101/gr.189102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Compositional gradients in <em>Gramineae</em> genes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWong%26rft.auinit1%253DG.%2BK.-S.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D851%26rft.epage%253D856%26rft.atitle%253DCompositional%2BGradients%2Bin%2BGramineae%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.189102%26rft_id%253Dinfo%253Apmid%252F12045139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.82" data-doi="10.1105/tpc.008714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Disease resistance and abiotic stress tolerance in rice are inversely modulated by an abscisic acid-inducible mitogen-activated protein kinase.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.008714%26rft_id%253Dinfo%253Apmid%252F12615946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE1LzMvNzQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.203.83" data-doi="10.1104/pp.105.059477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanetti</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galbraith</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Immunopurification of polyribosomal complexes of Arabidopsis for global analysis of gene expression.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">624</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DZanetti%26rft.auinit1%253DM.%2BE.%26rft.volume%253D138%26rft.issue%253D2%26rft.spage%253D624%26rft.epage%253D635%26rft.atitle%253DImmunopurification%2Bof%2BPolyribosomal%2BComplexes%2Bof%2BArabidopsis%2Bfor%2BGlobal%2BAnalysis%2Bof%2BGene%2BExpression%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.059477%26rft_id%253Dinfo%253Apmid%252F15955926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzgvMi82MjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/203.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/193" title="CRISPR/Cas9-Induced Double-Strand Break Repair in Arabidopsis Nonhomologous End-Joining Mutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/221" title="Time-Course Analysis of Gene Expression During the Saccharomyces cerevisiae Hypoxic Response" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/203.short" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6MDZTJktyXazST-GN0eaZI9ag_sXNCOYmTI9xZjYh30" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/203" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/203" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/203" class="active" data-icon-position="" data-hide-link-title="0">Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460530" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-4upPfLc3GTTwXkjBSkhi15P0dV0_-xjGM1UEF7LGES8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/203.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/203" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xjcQmqb8XhdqPjzrVtRNYkyTfkeoXAVhM-iweibwhNQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/203" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/203.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460530" id="node460530--2" data-pisa="ggg;7/1/203" data-pisa-master="ggg;g3.116.036020" data-apath="/ggg/7/1/203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2080-8416" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dongyan Zhao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John P. Hamilton</span>, <span class="highwire-citation-author" data-delta="2">Michael Hardigan</span>, <span class="highwire-citation-author" data-delta="3">Dongmei Yin</span>, <span class="highwire-citation-author" data-delta="4">Tao He</span>, <span class="highwire-citation-author" data-delta="5">Brieanne Vaillancourt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9990-5808" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mauricio Reynoso</span>, <span class="highwire-citation-author" data-delta="7">Germain Pauluzzi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-7798-6830" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott Funkhouser</span>, <span class="highwire-citation-author" data-delta="9">Yuehua Cui</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-8568-7125" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julia Bailey-Serres</span>, <span class="highwire-citation-author" data-delta="11">Jiming Jiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-6727-4677" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>C. Robin Buell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-2776-6669" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ning Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">203-219;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036020 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460530--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dongyan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongyan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADongyan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2080-8416" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dongyan Zhao</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John P. Hamilton</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Hardigan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BHardigan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hardigan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BHardigan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dongmei Yin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongmei%2BYin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongmei%2BYin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematics, San Francisco State University, California 94132</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brieanne Vaillancourt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brieanne%2BVaillancourt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaillancourt%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrieanne%2BVaillancourt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mauricio Reynoso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BReynoso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynoso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMauricio%2BReynoso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9990-5808" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mauricio Reynoso</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Germain Pauluzzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Germain%2BPauluzzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pauluzzi%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGermain%2BPauluzzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Scott Funkhouser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Genetics Program, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BFunkhouser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Funkhouser%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BFunkhouser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7798-6830" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott Funkhouser</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yuehua Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuehua%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuehua%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Julia Bailey-Serres </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julia%2BBailey-Serres%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bailey-Serres%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulia%2BBailey-Serres%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8568-7125" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julia Bailey-Serres</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Jiming Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Horticulture, University of Wisconsin–Madison, Wisconsin 53705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiming%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiming%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6727-4677" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for C. Robin Buell</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a>
<a href="mailto:jiangn@msu.edu" class="" data-icon-position="" data-hide-link-title="0">jiangn@msu.edu</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Ning Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Genetics Program, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Program in Ecology, Evolutionary Biology and Behavior, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ning%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANing%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2776-6669" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ning Jiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a>
<a href="mailto:jiangn@msu.edu" class="" data-icon-position="" data-hide-link-title="0">jiangn@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/203" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/203&amp;title=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/203&amp;title=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/203&amp;title=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/203&amp;text=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/203&amp;title=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/203&amp;t=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/203&amp;title=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/203&amp;title=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460530/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460530" id="node460530--3" data-pisa="ggg;7/1/203" data-pisa-master="ggg;g3.116.036020" data-apath="/ggg/7/1/203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Analysis of Ribosome-Associated mRNAs in Rice Reveals the Importance of Transcript Size and GC Content in Translation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2080-8416" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dongyan Zhao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John P. Hamilton</span>, <span class="highwire-citation-author" data-delta="2">Michael Hardigan</span>, <span class="highwire-citation-author" data-delta="3">Dongmei Yin</span>, <span class="highwire-citation-author" data-delta="4">Tao He</span>, <span class="highwire-citation-author" data-delta="5">Brieanne Vaillancourt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9990-5808" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mauricio Reynoso</span>, <span class="highwire-citation-author" data-delta="7">Germain Pauluzzi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-7798-6830" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Scott Funkhouser</span>, <span class="highwire-citation-author" data-delta="9">Yuehua Cui</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-8568-7125" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julia Bailey-Serres</span>, <span class="highwire-citation-author" data-delta="11">Jiming Jiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-6727-4677" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>C. Robin Buell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-2776-6669" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ning Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">203-219;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036020 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460530--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dongyan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongyan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADongyan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2080-8416" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dongyan Zhao</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John P. Hamilton</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Hardigan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BHardigan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hardigan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BHardigan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dongmei Yin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongmei%2BYin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongmei%2BYin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematics, San Francisco State University, California 94132</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brieanne Vaillancourt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brieanne%2BVaillancourt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaillancourt%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrieanne%2BVaillancourt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mauricio Reynoso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BReynoso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynoso%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMauricio%2BReynoso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9990-5808" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mauricio Reynoso</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Germain Pauluzzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Germain%2BPauluzzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pauluzzi%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGermain%2BPauluzzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Scott Funkhouser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Genetics Program, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BFunkhouser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Funkhouser%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BFunkhouser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7798-6830" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Scott Funkhouser</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yuehua Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics and Probability, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuehua%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuehua%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Julia Bailey-Serres </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Plant Cell Biology, University of California, Riverside, California 92521</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Botany and Plant Sciences, University of California, Riverside, California 92521</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julia%2BBailey-Serres%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bailey-Serres%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulia%2BBailey-Serres%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8568-7125" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julia Bailey-Serres</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Jiming Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Horticulture, University of Wisconsin–Madison, Wisconsin 53705</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiming%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiming%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6727-4677" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for C. Robin Buell</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a>
<a href="mailto:jiangn@msu.edu" class="" data-icon-position="" data-hide-link-title="0">jiangn@msu.edu</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Ning Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Genetics Program, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Program in Ecology, Evolutionary Biology and Behavior, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ning%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANing%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2776-6669" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ning Jiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a>
<a href="mailto:jiangn@msu.edu" class="" data-icon-position="" data-hide-link-title="0">jiangn@msu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=203&amp;rft.epage=219&amp;rft.atitle=Analysis%20of%20Ribosome-Associated%20mRNAs%20in%20Rice%20Reveals%20the%20Importance%20of%20Transcript%20Size%20and%20GC%20Content%20in%20Translation&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhao%2C+Dongyan&amp;rft.au=Hamilton%2C+John+P.&amp;rft.au=Hardigan%2C+Michael&amp;rft.au=Yin%2C+Dongmei&amp;rft.au=He%2C+Tao&amp;rft.au=Vaillancourt%2C+Brieanne&amp;rft.au=Reynoso%2C+Mauricio&amp;rft.au=Pauluzzi%2C+Germain&amp;rft.au=Funkhouser%2C+Scott&amp;rft.au=Cui%2C+Yuehua&amp;rft.au=Bailey-Serres%2C+Julia&amp;rft.au=Jiang%2C+Jiming&amp;rft.au=Buell%2C+C.+Robin&amp;rft.au=Jiang%2C+Ning"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460530/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460530/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460530/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460530/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460530/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460530/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460530/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460530/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460530/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460530/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460530/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460530/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/203" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/203.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/203.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/1/203.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/1/203.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
