<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/41.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/41.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/1/41/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/41.atom" />
<meta name="HW.pisa" content="ggg;7/1/41" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models" />
<meta name="DC.Identifier" content="10.1534/g3.116.035584" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Cuevas et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The phenomenon of genotype × environment (G × E) interaction in plant breeding decreases selection accuracy, thereby negatively affecting genetic gains. Several genomic prediction models incorporating G × E have been recently developed and used in genomic selection of plant breeding programs. Genomic prediction models for assessing multi-environment G × E interaction are extensions of a single-environment model, and have advantages and limitations. In this study, we propose two multi-environment Bayesian genomic models: the first model considers genetic effects ![Formula][1]&lt;/img&gt; that can be assessed by the Kronecker product of variance–covariance matrices of genetic correlations between environments and genomic kernels through markers under two linear kernel methods, linear (genomic best linear unbiased predictors, GBLUP) and Gaussian (Gaussian kernel, GK). The other model has the same genetic component as the first model ![Formula][2]&lt;/img&gt; plus an extra component, f , that captures random effects between environments that were not captured by the random effects ![Formula][3]&lt;/img&gt; We used five CIMMYT data sets (one maize and four wheat) that were previously used in different studies. Results show that models with G × E always have superior prediction ability than single-environment models, and the higher prediction ability of multi-environment models with ![Formula][4]&lt;/img&gt; over the multi-environment model with only u occurred 85% of the time with GBLUP and 45% of the time with GK across the five data sets. The latter result indicated that including the random effect f is still beneficial for increasing prediction ability after adjusting by the random effect ![Formula][5]&lt;/img&gt;

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif" />
<meta name="DC.Contributor" content="Jaime Cuevas" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The phenomenon of genotype × environment (G × E) interaction in plant breeding decreases selection accuracy, thereby negatively affecting genetic gains. Several genomic prediction models incorporating G × E have been recently developed and used in genomic selection of plant breeding programs. Genomic prediction models for assessing multi-environment G × E interaction are extensions of a single-environment model, and have advantages and limitations. In this study, we propose two multi-environment Bayesian genomic models: the first model considers genetic effects &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mrow&gt;&lt;mo&gt;(&lt;/mo&gt;&lt;mi mathvariant=&quot;bold-italic&quot;&gt;u&lt;/mi&gt;&lt;mo&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/mrow&gt;&lt;/math&gt; that can be assessed by the Kronecker product of variance–covariance matrices of genetic correlations between environments and genomic kernels through markers under two linear kernel methods, linear (genomic best linear unbiased predictors, GBLUP) and Gaussian (Gaussian kernel, GK). The other model has the same genetic component as the first model &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mrow&gt;&lt;mo&gt;(&lt;/mo&gt;&lt;mi mathvariant=&quot;bold-italic&quot;&gt;u&lt;/mi&gt;&lt;mo&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/mrow&gt;&lt;/math&gt; plus an extra component, &lt;b&gt;&lt;i&gt;f&lt;/i&gt;&lt;/b&gt;, that captures random effects between environments that were not captured by the random effects &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me3&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mi mathvariant=&quot;bold-italic&quot;&gt;u&lt;/mi&gt;&lt;mo&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt; We used five CIMMYT data sets (one maize and four wheat) that were previously used in different studies. Results show that models with G × E always have superior prediction ability than single-environment models, and the higher prediction ability of multi-environment models with &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me4&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mi mathvariant=&quot;bold-italic&quot;&gt;u&lt;/mi&gt;&lt;mo&gt; &lt;/mo&gt;&lt;mtext&gt;and&lt;/mtext&gt;&lt;mo&gt; &lt;/mo&gt;&lt;mi mathvariant=&quot;bold-italic&quot;&gt;f&lt;/mi&gt;&lt;/mrow&gt;&lt;/math&gt; over the multi-environment model with only &lt;i&gt;&lt;b&gt;u&lt;/b&gt;&lt;/i&gt; occurred 85% of the time with GBLUP and 45% of the time with GK across the five data sets. The latter result indicated that including the random effect &lt;i&gt;&lt;b&gt;f&lt;/b&gt;&lt;/i&gt; is still beneficial for increasing prediction ability after adjusting by the random effect &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me7&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;mi mathvariant=&quot;bold-italic&quot;&gt;u&lt;/mi&gt;&lt;mo&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/41" />
<meta name="citation_id" content="7/1/41" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/41" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/41.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/41.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/41.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035584" />
<meta name="citation_pmid" content="27793970" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="41" />
<meta name="citation_lastpage" content="53" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jaime Cuevas" />
<meta name="citation_author_institution" content="Universidad de Quintana Roo, Chetumal, Quintana Roo, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, C. P. 56230 Montecillos, Edo. de México, México" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_reference" content="citation_journal_title=Adv. Neural Inf. Process. Syst.;citation_author=E. Bonilla;citation_author=K. Chai;citation_author=C. Williams;citation_title=Multi-task Gaussian process prediction.;citation_pages=153-160;citation_volume=20;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=P. L. Cornelius;citation_author=R. C. Yang;citation_title=Using Factor Analytic Models for Joining Environments and Genotypes without Crossover Genotype x Environment Interaction;citation_pages=1291-1305;citation_volume=48;citation_year=2008;citation_issue=4;citation_doi=10.2135/cropsci2007.11.0632" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=J. M. Cotes;citation_author=F. San Vicente;citation_author=B. Das;citation_title=Prediction assessment of linear mixed models for multivariate trials.;citation_pages=944-954;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2010.07.0403" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_title=Genotype-by-Environment Interaction;citation_title=Statistical test and estimators of multiplicative models for genotype-by-environment interaction;citation_author=P. Cornelius;citation_author=J. Crossa;citation_author=M. Seyedsadr;citation_pages=199-234;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=J. Crossa;citation_author=R. C. Yang;citation_author=P. L. Cornelius;citation_title=Studying crossover genotype× environment interaction using linear-bilinear models and mixed models.;citation_pages=362-380;citation_volume=9;citation_year=2004;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Crossa;citation_author=J. Burgueño;citation_author=P. L. Cornelius;citation_author=G. McLaren;citation_author=R. Trethowan;citation_title=Modeling Genotype x Environment Interaction Using Additive Genetic Covariances of Relatives for Predicting Breeding Values of Wheat Genotypes;citation_pages=1722-1733;citation_volume=46;citation_year=2006;citation_issue=4;citation_doi=10.2135/cropsci2005.11-0427" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez-Rodríguez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing.;citation_volume=3;citation_year=2013;citation_pmid=24022750;citation_doi=10.1534/g3.113.008227" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Crossa;citation_author=M. Maccaferri;citation_author=R. Tuberosa;citation_author=J. Burgueño;citation_author=P. Pérez-Rodríguez;citation_title=Extending the marker × environment interaction model for genomic-enabled prediction and genome-wide association analyses in durum wheat.;citation_pages=1-17;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2014.11.0789" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=V. Soberanis;citation_author=S. Pérez-Elizalde;citation_author=P. Pérez-Rodríguez;citation_title=Genomic prediction of genotype × environment interaction kernel regression models.;citation_pages=1-12;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.05.0047" />
<meta name="citation_reference" content="de los Campos, G., and P. Pérez-Rodríguez, 2014 Bayesian Generalized Linear Regression. R package version 1.0.4. Available at: http://CRAN.R-project.org/package=BGLR. Accessed: April 20, 2016." />
<meta name="citation_reference" content="de los Campos, G., and A. Grüneberg, 2016 MTM (Multiple-Trait Model) package. Available at: http://quantgen.github.io/MTM/vignette.html. Accessed May 3rd, 2016." />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=R. A. Fisher;citation_author=W. A. Mackensie;citation_title=Studies in crop variation II. The manurial response of different potato varieties.;citation_pages=311-320;citation_volume=13;citation_year=1923;citation_issue=03;citation_doi=10.1017/S0021859600003592" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_title=Springer Series in Statistics;citation_title=Linear and generalized linear mixed models and their applications;citation_author=J. Jiang;citation_pages=7;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. A. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnet;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model.;citation_volume=5;citation_year=2015;citation_issue=4;citation_pmid=25660166;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=S. Pérez-Elizalde;citation_author=J. Cuevas;citation_author=P. Pérez- Rodríguez;citation_author=J. Crossa;citation_title=Selection of the bandwidth parameter in a Bayesian Kernel regression model for genomic-enabled prediction.;citation_pages=512-532;citation_volume=5;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manes;citation_title=A comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.;citation_pages=1595-1605;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.003665" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. P. Piepho;citation_title=Empirical best linear unbiased prediction in cultivar trials using factor analytic variance covariance structure.;citation_pages=195-201;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050885" />
<meta name="citation_reference" content="citation_title=Quantitative Genetics, Genomics and Plant Breeding;citation_title=Exploring variety-environment data using random effects AMMI models with adjustments for spatial field trend: part 1: theory,;citation_author=A. B. Smith;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_pages=323-335;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=A. B. Smith;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_title=The analysis of crop cultivar breeding and evaluation trials: an overview of current mixed model approaches.;citation_pages=449-462;citation_volume=143;citation_year=2005;citation_doi=10.1017/S0021859605005587" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull;citation_author=P. M. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=F. Yates;citation_author=W. G. Cochran;citation_title=The analysis of groups of experiments.;citation_pages=556-580;citation_volume=28;citation_year=1938;citation_doi=10.1017/S0021859600050978" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="The phenomenon of genotype × environment (G × E) interaction in plant breeding decreases selection accuracy, thereby negatively affecting genetic gains. Several genomic prediction models incorporating G × E have been recently developed and used in genomic selection of plant breeding programs. Genomic prediction models for assessing multi-environment G × E interaction are extensions of a single-environment model, and have advantages and limitations. In this study, we propose two multi-environment Bayesian genomic models: the first model considers genetic effects ![Formula][1]&lt;/img&gt; that can be assessed by the Kronecker product of variance–covariance matrices of genetic correlations between environments and genomic kernels through markers under two linear kernel methods, linear (genomic best linear unbiased predictors, GBLUP) and Gaussian (Gaussian kernel, GK). The other model has the same genetic component as the first model ![Formula][2]&lt;/img&gt; plus an extra component, f , that captures random effects between environments that were not captured by the random effects ![Formula][3]&lt;/img&gt; We used five CIMMYT data sets (one maize and four wheat) that were previously used in different studies. Results show that models with G × E always have superior prediction ability than single-environment models, and the higher prediction ability of multi-environment models with ![Formula][4]&lt;/img&gt; over the multi-environment model with only u occurred 85% of the time with GBLUP and 45% of the time with GK across the five data sets. The latter result indicated that including the random effect f is still beneficial for increasing prediction ability after adjusting by the random effect ![Formula][5]&lt;/img&gt;

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif" />
<meta name="og-title" property="og:title" content="Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/41" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The phenomenon of genotype × environment (G × E) interaction in plant breeding decreases selection accuracy, thereby negatively affecting genetic gains. Several genomic prediction models incorporating G × E have been recently developed and used in genomic selection of plant breeding programs. Genomic prediction models for assessing multi-environment G × E interaction are extensions of a single-environment model, and have advantages and limitations. In this study, we propose two multi-environment Bayesian genomic models: the first model considers genetic effects ![Formula][1]&lt;/img&gt; that can be assessed by the Kronecker product of variance–covariance matrices of genetic correlations between environments and genomic kernels through markers under two linear kernel methods, linear (genomic best linear unbiased predictors, GBLUP) and Gaussian (Gaussian kernel, GK). The other model has the same genetic component as the first model ![Formula][2]&lt;/img&gt; plus an extra component, f , that captures random effects between environments that were not captured by the random effects ![Formula][3]&lt;/img&gt; We used five CIMMYT data sets (one maize and four wheat) that were previously used in different studies. Results show that models with G × E always have superior prediction ability than single-environment models, and the higher prediction ability of multi-environment models with ![Formula][4]&lt;/img&gt; over the multi-environment model with only u occurred 85% of the time with GBLUP and 45% of the time with GK across the five data sets. The latter result indicated that including the random effect f is still beneficial for increasing prediction ability after adjusting by the random effect ![Formula][5]&lt;/img&gt;

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/41" />
    <title>Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_KqyA8_dzDMVaS-tCor6rJcch-82Gd7zkcc27npZwp4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460531","apath":"\/ggg\/7\/1\/41.atom","pisa":"ggg;7\/1\/41","ac":{"\/ggg\/7\/1\/41.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/41.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/41"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/41"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/41":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mSfpFCDmT8mj9\\\\\\\/UantRLXtdCTRkftlklGFdYO3gAJR7N8\\\\\\\/T8n+X28vPfkUbmzDQHgihWNPO+AYTMYRw1BZbaLZsm3+66bMLTCxOn92MI\\\\\\\/1tSqZ\\\\\\\/zPkFhvDeHA\\\\\\\/Pv2UHim5Mr3R6WVjg7zj0TDESMNZXpB1DT2N535trJjekaMsK5uiDlERtJu3HI1PmrCsC+6ujHTl1taCFjOh5S2+d9kAIlyOzxow+flc8Btye27LZ7iAQTOPIY9V0k5NPPjocyQJNy0jHFtELrSN34NAIKzKaeqo9+Axv5gZOqZuV1NJB5IXCCg82fIJa2vrXEb0Pw5O5qbIohQ7azELmmqMxdpJM0wJ3ivptqt7NlEoppCjjBnwdLLFtQlDt223MWfJklj+2r5yHGxHtDGiezkc1N4B9HYLiF41aUsepyJrLhDtE2CnVReM7utVikYloCLOTlFon8PU\\\\\\\/28ydaBzahxzK3ljQNvE40J+h1tO0zqGOTJKl052aCuV8E31vhRpx3gBWpfnjujueVc95xvm8BpKIzVciFcCo1vZbft+AeEfnXTMj9uwVC3QExFwmaOet9C0cA1EpSubjkYP\\\\\\\/ykEQ\\\\\\\/cvx6bgsVxwSnUovz\\\\\\\/cJbZtlgOXYOBSzkIWraOwgF1GI65SKD9MSbLj48L3k7cRhI4CE4hu7MDlxcANkM+L0G1872VPiaFzVYM796apZv\\\\\\\/Bn5ofQPA33NOilqylDbbtsW99Jvj67h1QvIayorJLOXUzyXUPXlAPs12EcsTqV\\\\\\\/23994TJ4Gp+5cpsPPU4+AoMAQ0lBbcJ4UUxVXMDXExHypYYJW1vEWfHMrAbmvePuuhwqDfeO\\\\\\\/\\\\\\\/J16QFZu\\\\\\\/ut\\\\\\\/ACVljovpwB3jSvG7cq1GxuqMWISKHAwwXqN4sYoEaKBnHjYVrwGdkjLUo3b3MZ3hvVEYFdo2DebvciTVSF44x3TfMysBKKKgO2Zw+czCmOd3hFsi57k7alCaVTvJxuLYrGWHP1KWQxAVb\\\\\\\/\\\\\\\/I5tFuhHPAQBD4UmMnylNF66LOevBZHd2dxwzEej09BAX4xTot9s8N1YJCqF7Aja9DKlbC3x+pM1PpnV6TBUdvf7WWyH7Vs9V0JnQsui3KQ62IdBBKeJNC4aJ7YZ4gyWqtEcDXT1D7LRB1k3vqGhIHkog0fBtFMM\\\\\\\/wvVUg\\\\\\\/lTENntekkye4RlFsx\\\\\\\/Vg13FrgXktIOqKmq3+QKpwWRO\\\\\\\/2\\\\\\\/FMC72wuqxOPzI7Gd0ShJWmRvsjlT14JB6wPqxncs6paP7LDyyhD278SyHcG5ABqfkRXE8zC4NKZWmekO\\\u0022,\\\u0022iv\\\u0022:\\\u0022n\\\\\\\/D5Xty2BTR5IvWInLZYrA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223741f0148133bd4f33858e4a0ed408770fc8f863f96ea16a8ca217de081dbd5a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qhFGE5INMWmCyiNwFLEs\\\\\\\/UKAhFWBHHIqVuS8+WMkFQO4ILGiLity8WlxLspBrkGUAFbfZWWXJi94SujE2uSP31H8xtXDxeHF+zVy4gxB4\\\\\\\/ReOvAGUI\\\\\\\/WdCqV3rkIRaVnHSMVmoeS7zVW0AuFhVP0IsuIIU9c9er91TEJMHUSzZ4ub5w6yVeKSFMZFstEUz6q1Bbw7tdL0IaYmGRa13anUT7AU5LNG3RoUmB3tHGw9KRru5Za8dYsz25AMRhPKdhyKcoYG7N5UyhLNN5RMhNEBvK4R8pfjCyL9s\\\\\\\/nlYPdoo4xVYBuZqS78vqWjTY3UqmcdhGlhfNW7aTxUG9cOG3tVLmYIGaqcSeQUw6PQokMdUXUOIi6ubnekVQnUMRIBFykQhZR5I0+UxMRdnmq0L428nOqtSNSGrfJuta18DZI0Lp9rDamy4r0jno3hhxyTM\\\\\\\/4KJfQSNTQ8r0CPAxnbgILjw9zaFICU+P8SR6lyVfpUP5oAdPk7gM7MV6\\\\\\\/0gqeUJnzMKkpiNE\\\\\\\/Ij5fhGXd2ygl6RC7pN3MxsXehgCKk2knEtQkzXFUSb5RAt3XKoNLWEZZlD+dcvuqrcLOmT1zPygaT4+IKJdcqo\\\\\\\/NyE8FajTFEbjoCdpZ+3jSwuZiTY03JDZ+EgV5dKfmodo5Y3pl7slvLEfzievMNlzDABAZuA46KsRoelQL1TJ78CkXrcoBoz+liCeDe8my1YSb8pYmPNjMEw4qB\\\\\\\/2wUzB6BFOKni0gGcwXrzBLi4rmfbRk99g6EvqoA61uyhWG3BiFuEOJFgYz9g+4M2JzsYtFwkV5GhjeI02oo0Scn2ePL1DIu0mW8zTSbwN1tJTKfWMac+QxzEjxRzDjPFSRsSWfGAx2pfOWUqzFhn2R2F5LL+94niymHnGnyhjSkvneYO2arB+KePoBRpmxnjRaypZhcsMiuDuAh7w2iTXh2eRNupyeWuwRRF+FFFIKw5MPup0iscvbT20XCRpI5qZpp5Gr93soAWa7dgFTyAqAG\\\\\\\/gTH7NV3jfDRZBMk+9x8Tf5fY23fVL1UwKkulsmWM1hqmiSPvbjq7BkelhHyBqH5yWj6I\\\\\\\/u9LMS+hW+DOmUojN2V9E5IxHftJKTl9dsiVw2ylz5IqLp5amTkAc7johwX10NuorNYc5q3NDpYhYVNXe56i5DOc\\\\\\\/NMasTvtpDHPQUaO9i59RnhseRnIKVSKOT+2VnMREQ4p6PatztHm+nhAuR39Qf6HFCw2QzZk4STgg\\\\\\\/\\\\\\\/J\\\\\\\/5E01xn9ZShQ99GKFHFjHL\\\\\\\/42p1m2Zd8Hx+A0F1ZD2egJOw4rtZQSfJUpRQUdLlZY1gg11hQXqPXBS5EpkR0u4s0jjUqjorJkyh5BeZfZLzj3yLrA+lpgLRMtKK4\\\\\\\/HfiQPITzV\\\\\\\/RsRlJ9XeY9jsRhtp1QGwwj0qvjsV5+MTdBJzbhvekxjVlkeBbWZHSAirjuA5DePXfJeLRBJUsnLxRXajWM2q2dBYqiAmmM363VcEVcSS+TGje2NZ4w+Tf0Oa8un8JoQB1VOdMLH76Ct0No06vLRWhcvKAVI4SOpw6\\\\\\\/RziBTW7LtfJxh82isIFyarifgSciJgcP9Jgmf0z7HM0rkNwsFY9zIINbYzPttt5cQBGdvBb8diTIl04WbvA36QQSs9YsBtqP1uUElnXY017QdHS+6WzvircpCX0JYyulk1dbzKy3LYKPgvoaoiCjwvH+vm7+jd8njfqFumTcMfIby5rA376Nk7eYR3gUs525CKYLv+SLOYR\\\\\\\/HWpJjv4UskJIPO5z+SU2zE3uo7t0O3X4HF9Tl35BJeql5bVAmdgcvjWg5XYkh4TBHzgZINVvBqIOYWqQY7r7tWPCnb\\\\\\\/hH2SQgnY4GDTuBrZT1tZasLb2Tw+BbB5PwyvJzXIvltqJhHujs3D0Yq4YUycJ2iSrP8MhwxFEEZWrO0cw+eIkjOdL8EybEBDCHdikAu35SzkrmsbiszJ869ST1TwAAb6Ud4d+1OA2\\\\\\\/tJFm57RSUhIDZtlJeNohX++aY9ovuAID1HS1ujgfPuBvjwPM5OlTsnxgbfL88TZckIRbd3sfAxV3ToIIEylNd+7wrP9uucGPTxuWT\\\\\\\/CeRqcuhetvndsN+ZYlyYu\\\\\\\/JbvgVSFpH6oSgbvjAqOa4TGAZHNuhZQ1gCZFxMTaHYMQ0JwVfylWLlsJLgKjXT9VopfYCiPLVsLjpSkXOox0cfai5HDL25q4bXC8Oo4NgQ4ClZBs4rCIoVNsShBvsyBjYqTzaDwd7evdY6jY\\\\\\\/8O1UqUuxi8JcsaXrmFsOGoZhFsc0vJRnpoSHIVLaX1DynvBFYi5SdExQrT7fLmNwHml5x4JhViEo5wf4vPilFudWNKZyaVA0ydzo\\\\\\\/2WapJ+6JJI\\\\\\\/ERMJPQpbZbGMTP7tag5WdbHf+LINjvCiqAw11hD42TlcsQsshVpKISyBegCEU3px4Mo+MFECAGG+kFZYhlr+gmquycDuJEO6luQUzIurPtzjR3IbW89g7OCKtD22+BRKG6FAAnxZoVRj460htxin2j798H6uTjOOaXUAWqLNtxU0TlAMdtC2zxSUvFxwa9Ybq2y5vZ7Ew86Ol3MOAFwcKLIBpQ2SWdO7fK6pQKW3sd0K1lkWrmWrW4WvhJu\\\\\\\/PlZuIsH4+zSqZhmFTijEzx8ObJtmdIXw++oGpqV2fd2nJlvOhdloKND5BB4iI3M7HNDmOvHmAyAqqb2ix1j14g9c0xeEUXwI1BN+DTt7Jdub7o1Zli+AuWtnXziBWdaKOUqf0BsqjcyW3S8KVPP7FJ78qzQWXNZo33AYRhq7s8Ye\\\\\\\/SatkSmdLxwJBYSI\\\\\\\/JED\\\\\\\/grkYvM09Y+WKBvdJ8u+AThkPLeEVnqmm0q5jOrm\\\\\\\/lYBKxG0c\\\\\\\/+\\\\\\\/9tAt+QfakY9GJN8CnTtueIG+hCcBs2RFq1VEFRMaegzMCTysR54pvTRSG4ai1YnW0UVJgBwErsg5pUl9Q9vcSwHTICHjUgeFB4JMs3hhTMHNJL+30osdmoFwaSSjBrIZqan338ZqIVDtXPlKybqNRc4mQCqCgjKj8uUAXzD\\\\\\\/OmtwbeepIfMBj5Uw51NpvtFQ0u4h3IO0DM8sO5nhF8O4J6E\\\\\\\/nFidavzAN2Y6y74XvLwbW6EHq5FhxMZVcAFdvrS48k1M2cgiqxErXWrzZCuUVyB2jOtxw7bNzucsFVmWvoPKuBjql6K8CJteDy0i7MA5NPA3fQ2OK+MUI6UCPhvT7WA7oiyG+ssn4sppT5f6HRFl1IdsxpMKDaMwBgfyMWndR77ViZqdT8J8zBYQjHmXnvH7NTX2AOu9JMITz9mF4Gg+pfoUnbr7ULp4dGoAW\\\\\\\/zG7vO819IWRMANmmLWsz+TGzy8GY5guqf4Q4SU4pLGG1xxEOBxxDY23ZeA9fWReqbW2Kzcw7PVj9kYjGaaW6HxmWxeYCxHz6nvIlxi0b9Nld+n6CAlVcXx5Le\\\\\\\/CMDOzWMldEgtMD2WZ3HMo5F6bgDIDE2onwR5iIWxgl8IWVlTGi2NoQ6T8fkAiu17zdpbhhsjPZHSbwYBBsBFuJj0xrg5HIJpdqMifftjvvNZYjR8elfhsWb\\\\\\\/BmRCgTkINLk+qja5ToyVqIw9KyXxfmgJQphjD6hOOk1w9Gd6G5RSHUTg5BI2xa\\\\\\\/oAHHSoLVN0DBJSadiYjGtrzYWbOekdE9g\\\\\\\/jjmrIrPxyVOx9sFzqeadSFLNav+K425hK4sKObzNWOZyQG2PNNxv2PptMpALEG2AIjvNPIfuD683Hz6hOBpKbcrH1UOp+c2Lv2FE4S23RT2i1oC8aj65QCYOwpO\\\\\\\/j160CFzMXXvn02vBvqBa+XdWEJfOspDHBdstq9ngApIGXrtYujMKj5fsSFxm4i2WLNFd2qHE4qXrllGJdyRBRi2ZgfjY4F2GdjH5dsYRiMACjh6hXVH1t68ssIj9KjfAOgvv6oZP9C7EJW2hmc7fHG4Vv2Kx3zEJphjcYsyHAlhWRn2aOk4KAx6E9qEm\\\\\\\/PoGsN0viO0aq9ID5e6eDHWOxIi1qIaxofAYyM\\\\\\\/Q0vExZotkNq1QW6SoBFM8FpaqkK3Fwv37hOEYAhzOOeBaWOKXLH\\\\\\\/YU6YP6jyK3Qkvj4YeWZxdx0sOAyWmevx3gRfClD\\\\\\\/Uldd1rysqy15YZr4XpTY8QFspayGphCtgzTzvwp3EMLfZssAfHd6VpIF58kyXOl2HPv9DHeDbMjlwgE9ZZZ60IjsMteQO5CO2jRhEEwJtw3PJgd8H7wttWDI2ki7oB3SMRcpazTpzDSDIBfVDnEA2j8Fs88jccL8YA+HPMn7r8BHA9nvWEWF2gnsqwHROTQ2dNUr3gBe\\\\\\\/YQ65duvgvnNfd3PT+8SjT3Q4vzaBXyR28uxbVuP7ubOr5SEVx+AsZVL9VxS0RHx0xCUtteWseuvw+CdNGEbFCzmZKQZotiSVlVyh7xOCemtcWktVgVCMs\\\\\\\/p4yjWGfNQzFk0WNDPdRB\\\\\\\/zRlDjG8mzqYoDnp1M\\\\\\\/\\\\\\\/ETPpjTziWGiyC+XONfN7cs4WJy14u1PCr6CYHljk0p\\\\\\\/J1BgKnKjQmCsYMwog5fPd9KW25kXQHsffdZg8Q9Sna1RfoEZxb\\\\\\\/9G84zgrg7ne1zN\\\\\\\/B0P3weaNPwjqyrVeIoNCBc+Yteydta8ehpSoUr5OE2JxxZpEdKgO2qVTfllpcgo+tQhhhV+xZm\\\\\\\/hfR9EFuM0CPRuZ9kdjzE\\\\\\\/EDwn9RpTcxdr0kCtTjo1jbgmqJafbDGeCUNOxC8EGPvDfACfc4IJW08BRYgJyikbOK7JOyPPD49AdFVDY0xZeqTWietqiyhTXmr72+3RmhYFJ+M7WJnuV6brd1WltCLeeXUhAw0ef7vjvFoO7Lx2gwUgApYqbIhMOuOdQF0zr+rK\\\\\\\/1veHBdiRxssNv4iiDMN6bfnNqdEICIX5536BpwpT1q8OEyCPhRYZkjks2o3QtLg9M2pud7WS2XljZMW9OEgdLWJXntRM7VonT6mU3FhXh3h4Poh\\\\\\\/olMXNbuF\\\\\\\/nwXqvfovLOiUCzplkxiD+5t+3qENFOjKbIW97zseRatXWqc1vx2S1uBG+7zVjsGJb43WKH3K6rtZOm9eh0CvnAJg38DqDPWH6gbmmQLZ+qRv2zRSsdcjhXjULHuNhZljCFxS2FDevzhZasqE0pqjMb+l+M7xwMKocIhbsIwXJzr+wErrnfn2QZ1kiBWuq\\\\\\\/bIeVVAFtr8Lp2GR7giqlgiFQm3v3Xfv0b0dhQoN9ndCc1kgjOCKvqc8zk2nbHUaCrMWon03Ii4cuw7G9ZWSxZ2B90C051VZneUePGi2tcxjGZ1iyaL+nvpov3KkHq7jVMLcEDYf0+IORw6JnnlxCdwchENrx74l3HX9+TWRcJ1N7KTUaTFH0jPCyHd2OrgwDTEKwW+BD+BX2gynls95TIVvlB442TVfcg2\\\\\\\/FV4Ed78+tLtjoQyOrUpPK6y\\\\\\\/F+5R1DGjmxSe15zD6hjOHMatTxEbYYxjuudEDIHL0Dug\\\\\\\/NPoRLe7Tv8yEEqpIdn6cv0GPpMP0cgVBT5F5+wlA9ctGSW6Hf3zXG7Qt\\\\\\\/LZJIKIz\\\\\\\/WXU8cNWl81fdT\\\\\\\/2SRH3Kle0fh1Vl4mpo7M+wpvF9NMd0NaqNSfkqW90\\\\\\\/MCqgoFZTDnVr6UmL0o7BOM5XKu2CJMuISQ\\\\\\\/O8npk\\\\\\\/xRxvm7g+5eusnoOtekykKWEt9ZsF10wFxX8mOlfQkKXsOnz7pZCJlS3OZmm381ghTFso3AF7fFC9iMDSOMUcGCHypX5QHrLLxmcUgMtAj5srEE+AZMx\\\\\\\/xK5R7dvoJ\\\\\\\/VYePRj50loaq\\\\\\\/csfe8xFzoV4cFOxr9dQdLlILB1u4WvkRe0STcC906CWk+cZHV+TeZ7dsDBgEilUmQW9kWJLW+N6Vl8Nru2Y7PiQjLooULjHQey6AFetlkGePjVh2YbDnKPgAMWzIH5d+RsZ2RGDa3TZMI1bJCuAtjJgTtZyO9AxvDKfBYXt2YZ39WYOV\\\\\\\/VvedhxuFk0wQRz\\\\\\\/aX9WjMMTHC58jSCb67eUqs1wN19aC1+CGLBpHLaRRxQjLmBFfArKaGuU+qO7pc50E0ERwfnaWQpXoqTcvlUUmX54g3MRA\\\\\\\/0AUZ62n9jrjCO60V8lWqEcfvPY4gsMokGVn0fzboWGkvVZGVdaVnEeFzgliVdksrhwcJmhUHl4rHpeyRGrnIYIZFUHfLL67I0DzaWS08n5X\\\\\\\/eGDJmJF8BueNkUc4q6SDMNWwgyTK97CUyx3XuYsyjJZpF6TwRSE6e1b3BpHXcxPNhSKwcsY6\\\\\\\/h2zA3w0XxSurUMIKT9+pChmyFwBgKztPHmZTBddy+0ErcfoOD+8C6FY9INjD1hIrEZdF5+piqUpZrXigL8HVny534IAqurTeb8LyRTf6uNVmr6H5RsUBj1ke+jF5aDbN+JPNzH9YaJWkFTPmiYbQIGiWXv8ogOGXUgHNtD1wOEd2mayRo7\\\\\\\/5q5OdOjHHaLjxJv8rpz+mpPz8dTqM4NVc\\\\\\\/V6vXQCqUd1PUHGmnIL+EAMqBe1vNBBCCXh7wJRn+ToelTB\\\\\\\/e4xh\\\\\\\/o\\\\\\\/medJkTmbp6Tl\\\\\\\/jOXLc58f0XEd8YVQPlkgkZc0piUBn4xyekleAvlRH57RfDU7zlx4LQY+NjC6JOBbLuhNTnzuDW4dYe+nLdLvRLJkltfWzUCHphkS9b5yp0EVZYrbh+o91YfPZGHo\\\\\\\/y+FY3ru6NlqB735NnWcQvBIeNdeAtSykubJnaPZyn\\\\\\\/qNmmJ+6h4AxacRBX0cITsL5Bq1588SK3F6xOMeaigJHyofz3GC+LuTEhh9nwFnavklOX3naHsRnnbY1P3VWUBMjWbFd04FCCpTIZUKEItOO3SFr0OvOnsKV5aBOzj5NkKVM\\\\\\\/ZAQif1Z0RJh4yHCXVriSD77wDl+lde4Py38bKSfuBUoq\\\\\\\/1qcKKbdSUWolLy0DmVfOqoODy0G1WJQTmWquzoZ5M1FPEUNtnAfRHHXjqKkjAPd7MiPELuBtp2FH\\\\\\\/VM3\\\\\\\/QvQwT+KFL\\\\\\\/BqhQ6Bh27dRirAXa+iWt\\\\\\\/F7DVPWZOb19gl4jJwQ3get1tF26ZNAl+FOP6KU08QXrbHr91s72VGJe9WZxMBHpGkS+n0DEKNNP5\\\\\\\/bFdlCkMqwDKy\\\\\\\/nkWrpW+hejWbEU7qhhlREevatfRQbjsXCoFsfU5DJEah8rJ0PBx\\\\\\\/mR7OIjbPmwSfEBGJlVivNhfGYIuxvKEapfQINIZmOp3iDvxonhgxupBZwBE1qxpmr28i3MZ6nYBQFQ2PmSvLXbZGArskhUyzZ1VswT\\\\\\\/x62yQMCev+l\\\\\\\/twsWLgiCO8684urL2aITdgofKqnmHo4deuONP18tzUJ5XzxYdsx4yUuKOOWN4vu5zopVEUiRD0eCY3btA8issDA+rlen7QxLeBmkCf5Z1fzJSfIFwMB+HjYIhYcus0Go0g3quTdu\\\\\\\/+kqg\\\\\\\/mXDiAoik+jQ34xO2cJeRki4DTOHR7WIFvnaDXpQuT29xtaMW31fUhW775\\\\\\\/QnseyMapEUL8dubIWF3\\\\\\\/jgI7\\\\\\\/0gZOOw2b7DxYOYQh9\\\\\\\/q76AlBHiVeoDvNZ69YNm0HLONYg0\\\\\\\/YovJ9qHuR8egFNGYydKuN867rR+IAhlL5n4NdutuqIKEfxzaTB3hMtxjTUXfr0ebRT8MnYl1RmrHXI8jROlp5Nn600cjEDaO01wxa3ijMBrPcMFgQkWfxFM0LaHsJx\\\\\\\/z5lKzsxorKDcAgYjrdkkeoWZVrxC70Nr+vd3mdwAnEcTZCj3IcuExIiw1da+TdnFD8inPcIjocvgmJXmZ9m9yNvRPYChC7uJwC7lJ8H4luJ+6vBz8jQjSrTEPtLiE65AR9Z1Zha\\\\\\\/oMmL+e1pfE6b3INI0excUXBNb8R+edIt2xeCOhfFlFgzOzrjJ2tQXo2w5yr\\\\\\\/IHeG546nRyqtCGaNnP1JT+te6yDa\\\\\\\/QjeAtw7lMULd6AZ9JjFJPKnoL+jzIAmSiKEK7jKXzAfW6JhA012gOMclCzJ522g1+dU1dyneLsKG++xA1D0GjgLMFWDX3yjbHWvGxfhDq4jdeQ0wJSHXiTmeKYzbxlw125DGS9xCpv\\\\\\\/6ytztU1clNbuDvWHSpev1N5o+KGm9d07CklJdlyw5PF299rIm6QY1OlSxgZJH22\\\\\\\/RHphroS+G\\\\\\\/IsT6twiqc2sz8Jb4nDwJlV\\\\\\\/HYDMKjufEXo++Qvpll8PK6MuUnxekgAFDw\\\\\\\/9q9c6tEO3BILTqHHR6H+g4+8hA+2Dfs2pERaA5OC5xGOKk5KX\\\\\\\/YQV6KzLnRhjmabF5QCSXXjfq3CofQXPERP0HL2MFKoEUD2zOIBjxaD6EDbTNj1I30c7Sq0\\\\\\\/lvYF6Hu85otIHrpQQ4MjD2xVstOYiELCXSJlgSlAHxVtDAwNRHaxN1L7qZgYAJlniPUzZ6hqjfoXgxWqEBGtrWeY9+u1n1oWf6W4me7bB9kkVI3Xosd\\\\\\\/ke66PCxS+K2Eqv9d7USnv57ZfHpCVSGEhBLHOC\\\\\\\/NTyg3QSjbcjaMBQ9gkHoWplF7K\\\\\\\/uD+XzwiKhVCiieyYyaRZaK+fha7kI8bgt4lUXfVvUsaRiS\\\\\\\/KQDUnlzjMQp5DqSMNZNkTHYT7faoAIW1NWcrxtEdGmphVKkW06reUt7kmni6RCjw5vYRZ6qLQVxrqIuPjljuutJHR8fenvniPLZYZGSzr24Pgmwb\\\\\\\/\\\\\\\/mr09Q4Q6DFBp2F5DNbm+R8h2Q4cxANn81kGO4QNuwont47YPwO8WLDnxpJfLXF4gNDtSjpTlcoy5+JH2cfkPlY\\\\\\\/HsPob5g7ryyITJtW65FHxLdCm6bR+WcRjbtYrwb1Fv6spBySfK8VG0MXi49mNRwNNtjXx2UjCU4lTPfOVg3W0wLuKc+irt1W4Ty2do8gfd0jwWg1aiWoCbjEM+rO7rkSIcqVTmfssPAxKzJwJyECZfyZ7pN+uSnFGAc\\\\\\\/wajZAyA0VjTd2lZJ4MgvBrJ1UwTdPfQ2BWNK9gOgZGLAR0oaQZvwzPyQK\\\\\\\/OQBQN9V4T\\\\\\\/2RBc53NuPD3Fyc5DyDsQTyKweVFgxc5n1Uc1kVjtxAmlakSDEQsyqtUHk12f5VlHuIrq8VoCabJmEtiuxXTnmC7JbVfFey8xbPlfg3muyk3ADeWxgYoK\\\\\\\/T2nOgno8UViU5qmTiUmqJm\\\\\\\/NQ5ioEPAcexcqeAwaQVAJAEnoS6yE1VKIb1c0airVqbe2aFob8S\\\\\\\/jG1tvEuAA2ge9d85QolXVYSbRnbfOFiDxOXuYAWm\\\\\\\/RbMdQcxicCSgX4FXGEx4TiTX3jdibsXmy2QMKUisR8CPL2qAcxhpqeu5ypXPququoi\\\\\\\/aX8MopyET2Wxyr3B0O09WtGrOxHQvvlEBbVGOkjt6pgIgBhAtr\\\\\\\/oii1TLeOa+wPpoLFpttAlVs+SNHR7UOrCMFZtLIBV\\\\\\\/cBAnQRVnBI3PoTCm+jeLfAm0rfifk3kUaubNIvFp+BDOPfAxoH+SDn5VzB8pXkdAEIcTsgnx0YCLBm0xElSfnmEkzEMmq5ItLRokuhBHWhvxBcuObBgyotgSvMBuQxnSvbZ17+RM\\\\\\\/lKc35UGUcGh4QQ5x7C2v6CGLxyKaD7Fnul3\\\\\\\/X9Xm4lJlCZFXLk7udi\\\\\\\/Dnpl1CbbtNvJ04UDevbUAQczPlKDUR1E1NwQJo\\\\\\\/UR4iSLHAizxRV6zlKbpnb5PzME+\\\\\\\/FILM5nIujYzX0ZxF180Zc5QtsZF\\\\\\\/nb5twm22vO5sWgfYpsZ5\\\\\\\/oaDyVACcfuC3vDFVz\\\\\\\/VNETDRk3Zm0KR1pKAuHebzFheWkBVv6rsBWvW73JZb13e+EwMIT0YL\\\\\\\/WpP7VCPl7c5OLE5ddeczFO8I6OUyD9oALvjAEJBsvePn68zGIj995M4zH2sLrZNccR+EExQW6GvEPZ8pshlO8qlKVDDWcp+aQdXneU44vVwS7jrfs5ziZi+2I+uMheGQCDEUJbP2zIuz5Nqm9z627W4kGJX8XlFrkJel7H9uKMEh5uWTsVik1rliPEHlY9KETeJx8Fz1qgnZiEWOK7GpmbFolhAIL55UBLl3I9JvDjhUMWPmtXcZ1ivmb3vErEWyVbqpAssXxEpnJMRT7XBO70jCjHzK1THLu8LGfvgZWYLb4M4Z5ONOqbUHLE+aPIeyuGZTgIdiqkOfKXlkNEWMXaatsdOFec7wS8GNjjX9\\\\\\\/MGri+7ocYUKfEIzxnLSkF+MM8iBG1pRNy5CisnISGiTgk50GBwdhGFlFR\\\\\\\/PbB+lU3I2FqxCaXAmlEs2t\\\\\\\/G9bBrhFVMv4OA77TZbL8A9CgtYQxuEeCLL+GV3KPbRDYuVr0kolTUm19Ad+pRuX6odKGQFJ2azYv8pmRCVsTsWCtco\\\\\\\/lMDSonO4wXYpzFBWj\\\\\\\/qaGFP9CVpzaH3vRPd1hR\\\\\\\/h9Mw5Tf8GuCawvt1AWAxRxurcqKZ201NRxLCz7LqvNG8MIF9n2FwUhfAj1ACy72iZ3qNHDsv3Vwr1r+dlrKoDZrZz51UEpJgynv8kDULmkUH4yXX83V93BVwH5+aJX7OKQviiKCEEqgYWqmja3A6wlHpK1dVW0P+NhxQtOKpRLFtohukvU6CUSVVO10Q5dvDDuD4eS8YWQA\\\\\\\/q3fFk3hb+sfIVAEVw+Vccj6nVN60Xwal6aZSY+ScwNOvXXXRW60KDK8fR+u\\\\\\\/DG5D663xBf4F0fFPnBgAppqFMCO2zFH1Bz2Ae4lednIWvDDmXcA7pTsO33cW7huhbF6aKYwV9BgCHX\\\\\\\/ZGOhvgVoz1LOVYLYXItY6LKZ4C\\\\\\\/cBUFeTuaZrQWg3JqqOu\\\\\\\/fmskHl6p7SxBQRh\\\\\\\/DDPoF1yJFQ8iJdBrltyh5eh5lyvn5AfwJ0II8YxFFnSNJqQDsbRqcIio9eYKyWErsET7owWVgUEbSo3VVOs8ioVOPEsmqe5g18dBJqiQEc9fvUfc0Ati3zAQcQJP7fWAyQ1ssdqWbqe0Ce1VCqrf4FWVaf2soYLQf72RN3XNc9\\\\\\\/HVaPCqHG0tcKvp7mHzYEP3mlaPEeJz7FP7bM+Vdz8yD+uk9MpWeW4anP7kH9toJA\\\\\\\/k92MTuzBvzVDQWXrTk5Cl4Diomernm2BjudEUZAlAnSz\\\\\\\/GKRXVY3a7y\\\\\\\/RuMu2rNKleTxGrRdzOLcEb+qxSWR3l7v9xY1HPuSXKRoFvomRZKm4a\\\\\\\/lfBrqyVVyiTIo1wBmb6FH3sDzrYmE6EDY4uMayLBykuuY8fiZqlzXyxjcOKAHzMRSCfzcj3n3XCxXCcsc1aqT3t7WP3ABB8q\\\\\\\/slopEG2HAIJ5VzmZhrWyp82wNv94B32wo3Icd7jPz0jyvAuBKApLG\\\\\\\/1i5akvTqrc52ZGJ++J2ccaj0JELsBKrMztOf0m+gFJFXHGr2OJgjbdUL80zmAtlsek\\\\\\\/WL7UIrOiiHIChQS0+Pwf0dLNqIvC2oGtkaOGFl2HYNErkOvr35DyS+eHZlbEdhlfuWV+imdPcO3HsUYAE9L1JHeJemWwkusJ4xsGuNmCt1rYWWzOq6ThCUB3YGXStTWHjLuIQ74PQWnj9fgNQPN\\\\\\\/gT7bdETcHGn+wm+43VuoU\\\\\\\/SmjwFYeSFxWtgO9kEb0COyUDa+Uly7tMULhDytXzMzrf5psDwo3pPkEsqv5T5OT2FSZHEds3PsjT\\\\\\\/MALq1D460jF+keA9QdyAPa2xPqC+0K+S6ZhD\\\\\\\/1u8ZvYf9M8cULt9TZb7EsYtd\\\\\\\/uk1BxLR0mMysju8vEFAQtsVkJ4SvaRWy0EAE4fQzGVEgD1hfCzUg0UHaTqJe1y2DD0l7s9AtGNNyVRbmcIq5XgiAsic0jjsiJHDTL2GZ3fGnW3aUt\\\\\\\/MTRBBa86852A1b28H9DOroZxgHVJVA7et4veXbp4m0io5oXgXr6YJdGBL0b7LBlq58aF+F59XUlBMkpCV2+2u2kBhsv\\\\\\\/YgkphaleFCg5Xf0MGJ4mthpno69RCYN5DmlFqw5\\\\\\\/GADEi\\\\\\\/\\\\\\\/Brpx8debLxaJ7vCa9WxDygHJTLEeln\\\\\\\/0RFlefzh09hC2IrxC2iBZ1Hg0XovUL5V9Wgs4pzohRTGy+xymN4saiyxUW6\\\\\\\/bbXGOlMHXa04H3RIcJ3hGb8AmLz8pFSfKv2gQmipafWDBYmACgkG4Kg1k8YuqH+nXJ83pTWp2R7wMhqTnm+Pk9S+IOOZ5lojQP6koFQ3jk3CZdA7zjs4BF5fWGkFbYlDTMukpCDX6ZzBbvcJDQKKbtjYVXs9ddChWTskuS+rCBiElBp\\\\\\\/yhNWz+HBEFgD+XqVUKZ4TEyaknmsa6s5w3fbVDJpw\\\\\\\/K2FFNl4J6kv5iFMmZkwlmWI2alCwaHmfErbrjTKZP2ZUNx\\\\\\\/3qg\\\\\\\/uv+HhT\\\\\\\/iI8yOftHc4RZN9k8NyDPUrcKCGOLj8JTzoMnz+hJ4id8QBaEuxx64SEKUcS0WfEwXU6LxJXhoPdmtvcgSt24aL+lWHDwdWswnAmNGP\\\\\\\/FVTo6wM461kOYPL3+Npq2d1a+XtkaHfQEi6WLiU+a2XjEJZWmBdu0bNZFesa2ryAAaob7S7Lg4B6ABsYVDeM6m2DYEc\\\\\\\/LFVa2nu\\\\\\\/qpjHpUKlSHxYRXxq4yjX2C5+qUNYTRw4akhtQvwJnLJzQh3gqzIhmaLwd8n3mEpx0znhE3OF3sWe8ES2lMZJxdSPU1V\\\\\\\/XyfQh38EBlE1TGX\\\\\\\/4d0Ppv\\\\\\\/5bM\\\\\\\/wvmIXITeAxfsWGWSq9PjzFMPyJoZVFdJl6qNVUCxwRaF4fn4gmo1sZiRk1GNL1TitmYKvdCoaf1U62qDkbu3kJjZ5b87T6Vy4q3Bq5rIkABrB5fnYKpglvBInWwQPoZrErpSlAHjIucfkgpmqEE07M8V60foPYiKw8oBnVsvV0v3sheT53h1FYjty09YtbHo1c8pnfxRIN6ksCP607wzeB0iN64FyhyQBvWRAnKoI8yuB+HL6UtvcYaIFbYfNzGIQOHKnSTXMMpNwvBsBvMmiRH8EyANNRg7Q6mYD9X73zDUPvohZER3MPQxzXkB2frltpcbeX7EcTuRoIwKXsaY1Qa1UsCHNi0ybpOgzhC5k75XdJkIT3owcCZAvwNSK4l\\\\\\\/Oadhf7\\\\\\\/s9jhotqennNbNX2YVt6uE+yRXSHBeW0Q3HnFykrRbBrggisygLQaothlVvYokdCsYVRpX4Lk6rwFVd1241EvJeTAVXqVWjJVeacNuarczqtdeTdkREJqOwursAQy487nx4TEl2EGmlXMOSK75psnMZ5dcsO2MRbErC7GYapzZaQFFm+oX6XaDl75O4FhoQcsN239gAD4rmKgoV0VSgu260+52CyZnIedcHRZbEiMuQYD7nKv91kjKHBmC0y0Y8gTYVW8RhavgrGh6Ka\\\\\\\/lK0v2ervPsEygC5rnvfEiv6Gd7oZlyIH4BhGG2DwROBUYAIZ2cpu7uLAEcPC8hlMQtNoA3bRJ+oIrSPevW2PT8K64\\\\\\\/ryBaoeUihl8L5vaO8WsoTDxEIG5yXr4Szl11GE5A\\\\\\\/Bcs02l2IUmaHslhjE0nuRnc\\\\\\\/AVJq89dLNXFHfnT7juozqCgx9iVWRKnaz\\\\\\\/wFkf8ABy11x4xsPLfq\\\\\\\/aB0r5xQg3Nq1oXqQ3me4voG45duXS\\\\\\\/PcW7owR0ii\\\\\\\/sL13OLpzKKp\\\\\\\/WGywkoDCDtXpp79edYhZ\\\\\\\/W3hsBeDzbHQghSDHuoAG4BKvipCulW3OpGT+0cyQYrGbqbF6Jc25FnLwnx8mjbfK8BjmnbanyzEX\\\\\\\/7V1GwT5POPIZhH\\\\\\\/I8TrT40\\\\\\\/W8aDK+9+5ZsiJxGRS7BYAk3uA6SOnZ2FVd1EFBKS4hX1B0WI9ImzUiovIrfl2NyYe0a5XHxHpImx8yxCjOdQVRiTAwm+2cJ6zjc+HNiHe\\\\\\\/vJ9ZkUNvzkm+ww0cU+VpLfF\\\\\\\/RucNR+njWVah3equto2+GMjx3o80X\\\\\\\/HPFsKDwU1eQYOLqkMlG\\\\\\\/TFVYOXGttUqb66NDDmHk5DbCucxQNa8tgq75o0f0cjKKQGJhIoNRlhFfAA4zHBjMimqWEcuUCvQObC95+vxeT8NEnmNmrSVAbvEaQKwdHz5dT9qF6WJrqApGIVq7jsmTSoqwnMNHdYc8H\\\\\\\/FFIuzHF8yGXYH8vPGmYd\\\\\\\/crYpgKG2baHqkpevjtLHrbmrBVjinzWxExeIDzgf73hoI7VVuNtNmmiyPz\\\\\\\/2PFaHcjcRJpHcfVAQAo8ia+2lKLZj\\\\\\\/pMnvD02sJ9cp2Gz6rbIgDHIYJPSF1+YRUB1YT0R6C0zqLO0zqOVGC9cSl4yDgdFUo0su5q5Ul3lRNoWrkgc4vytee2fSJ\\\\\\\/frXQ61Fk5jGnQC51kAtbSfXNB6F4GcvSqFnVtIQInZdfjXVn6B3SLy+UI9Mz15AQaoCifkEcyfejP7BUcUFWaYQwww4mBZyshnq0i+HSOQ7OZMz055S8uuYnLgFf9VpUw4CXt0gO\\\\\\\/88fwT2taaDjlPlVNg8DPrg61BxcIkLKAClwleGOHq4anec9qN22PUdqtJWgqpo4NFnK2RouZ0YHAqd5gF4N9NM6n+m83ZdDgs4byr1hD\\\\\\\/wcakmbhGC3uepZpGlYczx6uPBRPzQuTCzozWuqbsCoaBPgh1yvoveiDEeYhxyCqvtygGEgEmEuIjksGaz6IRi1jhcNOqldDCjoiTZOyBcxtyqLE6sZuZqGfscybauXhgnsEQMjO9z0Gl6FjzYnBB49Y9egXEM0GAMAgt9tcGcEjNnyJZxD1Fih4xzMrqF9dedvdP0sn21Rjuuo40qOTIM04IdZNFQv+yOH6MXqLpRDP2X29KHBpC4iixbtxwpt7zgc+0\\\\\\\/YpntFdD9bNYr2Fke+W8s1mY9kUY+T4w3i5i+l69CW3IRQhkSovnskOFFX9tim99pSJkXuY4ZimoYFaQZkRMStO8\\\\\\\/ncjkYceUBfDNaFbQCRCGXBH+mKQhSehvsFMO04+zuvvGXmspz9uNYs87OFNtMyHm2bsmZ\\\\\\\/iSAt4b7JAxj7eCjfJzVnc3FZthY1V\\\\\\\/iXP7WrEyOFC8oAyP1T7O6OpxVoxwdL42a4iA9IbLlVMd7xZTIR\\\\\\\/zterUVlYIrFo9G2A5MoFlQeQqFeOKOW\\\\\\\/1vAFW2J3BRg1bISOyXb9Bzxorj2TO31FwHnxhEBwgTszkW87X30CHjIj\\\\\\\/hS3L0wMluWaXI0QrL2VHPUYKkd0siBsqrodLTXFv+rhIVARGZe++XX8b1qAAmShTL2eUCxJ5XmbqHTc\\\\\\\/r2zpmiqpTf5fUFj8tQwVrxxexehCB7hLskd0xsXADfKb0A4OmIfdntwiUAAIYrljxKZinI++Rx1UpQTqSuXXvJkxush74ddDGP45PxdvvuoxlzdKudWpMefz8cXYeuKmo7D7Kwt+5xFjY2nnvEDnzmCV0efHWpZo+sKFv2AmkEbdBSnsGFvuGhNbpUIuuJSJR4\\\\\\\/ZuRWGnYW5Gh+l7Qs6lFvjHUmf4MCIz7kAl\\\\\\\/s1x4b8Vf4XGcP3\\\\\\\/MdK4dFC0+EDnkfqfRotL5zOm7d08MqEekfJFy53nMgEJcMfMdmsD0+0sReuVt\\\\\\\/UQpuQrrrNqOJLyj1YKw+PMWSDu6kJYyvgm0THj8NxYMFCAF0nDNlkp0o8mWq6mIISPq4xD69tye7hPq36mVD3LfGM47ZJx0Q90bkkdAmT7l34qWWR+oH+NSTzURdtjiCn6i\\\\\\\/2paoOu8cTfxRCnm0Z09v+6wb+7dzDFsq+Yv7mDoLGnGhOqDnB1kBw1yW346frbJlQIME+oTDzNKS\\\\\\\/PHIBb1qC9EHSVtyFQ8bLuyulZ6qH0ydNx6DVA5l\\\\\\\/Iz9qKbZD6TqNs2c2Wtp81auQdG+5zgPXi7GUC5nVj6dgOYzyPsbNt+gOznFD3mmTe4fIjyr1gsqlfnhJ1lIOQC5aYOWm3BnuKX6XvA+c3UujG\\\\\\\/iFeghS041Ug0VNNDQxhOyouJABC1f5zd191NV+qK7xSUWxEt6kMOMyC03eo2X5+L4zMkvK3BL+f5owJLXCr\\\\\\\/hX+Nw5XD+Ri8zT8tOd+oIYBJOvFx0y96CYpDcY+ZOkS3hQ5KoyFSmEf+Xtq3oDu\\\\\\\/iAQ6a5QQMvU7sKuoeaTMbcozM4QRfzD9jyaSW5zpgiBYVHCQhnviC1LrnStk\\\\\\\/aJ3nYsy8kUhr\\\\\\\/E0qEeXSNWOUNgfSiaMWbCGYiH4e0bIaYG47+6yzV7+FVF\\\\\\\/x2zhyMd+LcwEpGbtydJUvdpAF+IL1YS69mh5JZSFRD3ianz7HrDKl1ATnJTHXkjFCZCLPEDATfu\\\\\\\/FgOdMHX2xjv8wQaXIuY5VG8YyOlesbP734RCUfvpJc9fgYGOYxdhWw6CuHI7+nDklRrozE+J68gEKtzX3Xd2oiRk3gPQnBOk9c94pOUAEGDCbQlx\\\\\\\/7z3orpcJ6YbHswPDFO+OePHFjOEBpsshgHfvX3pAReHKQdfvQgHGf\\\\\\\/45zGNCWMd4yj+BRqkN13mXaUIFPJWFjVLKICUcLFeNzPMTodzIWL34SQJIHEo\\\\\\\/l5l5EbJm+wV76adPV3b2SLZOrH93maP\\\\\\\/XtVJ1W1Q2zr3Lqm0Vpmgev1EbD2HFUskfbKj491tuA56c0300vUpJWFPs++DsYvERnCGmg+pSF9\\\\\\\/jBbAZmDrdVpZhc2Nyajx2RfVyeGKHr8XZHzQD5Va4qaSAwFra7LXwGyG+Dz9qwBOPCiP\\\\\\\/QErxD6WtamlIuJu923ozuvK0+zmDk8f4VP\\\\\\\/oIcbwqUIUqjkTe0TOLvDiXjhFFq8Si9kRJ6YcjeOxXQNDnlNeJ\\\\\\\/plLhqQLPKFABU\\\\\\\/RKGzl3QW0tNo3rvyqhpJdy18oOrVcyjFwARWJ76N\\\\\\\/DCX+MDt0A05t2H9uVgRzxmn4ndoeri0bZpRDgL9ELEwhkONJGNyQthtzJ2yi+D+6F7gjfjIYUjMSkZD6\\\\\\\/bK6ReYzagiBvM8k4c16mTRXEOy9p6L2J0bYAjpI6Jt65\\\\\\\/Lgx5FyKC+DQgPVfKA+DA3OexVxuznKDujmyMs1uIO9gY4v6jScuWfzcYupufjY\\\\\\\/\\\\\\\/oY3JRW40iC4T+dR1osRFNIaSwhhqpvLAgf8SQWHjfnzt2U84gpqQwcr0O4sLIhMWkRYapTeX66JkbtLm7BvAqJLMS0+CLVl8mQbYrrazUQqjKB78SVaExT5czXFcOFsFNaHtxo6GOt+rvgBwvKVbie76yrMovWc5QJg6fC6ILTwejZYRO5KFz1ErnAr6wFOLC7h4ST7826nnCnm5I4BGNW6EvWrXIg3gTy7hHfu3W\\\\\\\/DT7NFSyxx0WorXmhTuAhauUJManBY+5hq4FuETYc6kGu9dJHa9yjryMcn18t0tO\\\\\\\/A\\\\\\\/qm4ti6iJlGO+SLolFdkMXjIXDFgA2othw34EXviM4oXyDP3nWC3X2ZU\\\\\\\/EQKXqxa07X6xAR4LN9YRoxjv5h3DGz5M\\\\\\\/lToelFbEJdPzHDDVNYGonpMxAl7EoWXzkMuOmVyHzZW11AXumMuPIFXr7GM3yW08SGE2sF16MkxQOSfI2TeSULZbfBaxTW8vMDIkGWFbvAThxeSnP48RBNE6loC2q1c9sWGCZ4Vz0e7wPS6WsIFC5igABpv\\\\\\\/2syFnJAUg6PXz9f9V\\\\\\\/vapb5SspjXWtwhE1Dc7MW6+QTjrGOxSTRQpVT2jDCjPy4ocL3i9o2kZwt56hU2qL7iUAkvCnCUMqfGsoaNB9EyoKANsvMFblL0MjGLbYpF1FG3lUYRZ5BRITwpNmEOOqLoIJTjvyer7d42QXfkt8hi8pz0uO080OfHYY5U1sKGYr+OLyMXdKRnRN1St\\\\\\\/+DOgFRP8g+f1dwIiLxxey+n5B9G3M1SnSBXV4ux9mN57Tjusm2m+di\\\\\\\/BtZgFnsbOlSuR9kaKF5B1t8XiH6RtnT6364PwJLD3m8f\\\\\\\/pukHqUYLYAWiK6aGr748tDxNPyDquLfRXSTxjHkyVyUjDEwCbFGhXiUizTeoEmZn6okHLwBAJR+\\\\\\\/nDTLJdkY8olZsfuMpSDlfqkpuRHCHjBmUY7qf47JLl87qy3WklFG6iZYNpBM1y7Q610OcM5SssUG6IhEGQo5C4ZNsvUSu9ZaSmaT4dE4CS5RhGCaMtdpipuT3vqdAJS1fe7x4AX3vrWppEzdP+PG3crYeDVe5w8dfMcMkd6TNWaqRZPpQ4oXKzzmoHUCBIpDlcZDsn74huv2QeH4qyFxoPMFOeAujBzMfO+f++fM1Z3X3aqFFaAQfTsjkRlPXu5x3kJGpVQWSHX5P4vpraiL6fJqikiCQvfRpz82FN5t9eMaWsP1GugMhDfrWSHv8oZO8mYebTZZGxdEM3Mcc6J3TjJubu1DX8CwuKzVSUngxDndibKcWiGd+qflTn7uCKeNwMRx6PeS\\\\\\\/Tzrkih2E3aCIHffxwnuozMxnw2QA0VId7ArJPmDHS9R1JdrliAKobHj5Begbn4HRqPuKW1ItjHuzHAFg65JA++tZtGyjNeS\\\\\\\/\\\\\\\/m8ZIFz9rnemTWcyHJkuZV\\\\\\\/2JZRtONDV0U2EqevaFF2iLm5aytdLCctJhNRKEm1O9dyq+gnlfi6ITndLSgU4zGFMQ8VeFrBIZmFlgQZ\\\\\\\/tuq69Hqele3XwuL8lOSuXkD0OuLaTK3GP8KeruplGZog7eOa8uQcSt\\\\\\\/aiRAfQhVvXCEbM55MkwZwEYPxoNM\\\\\\\/aECkMh67R33++pGppuMgj8AFAUQE1DV2uxIl1jzzl2nNV8PFo789QlCh6XGasEJ71zNL2yLKYY841a7\\\\\\\/dl8uOdFFdoWLrnyLFIqKQczLatdcnZ+1G61g612IovZtj3XFs4DxDxyEEId\\\\\\\/cTNbR45C6z6mAEdrRHGNbkOp3gT1TGH+UUoWaHJvO1zbMmT7njnmHgVBYRjVe\\\\\\\/s9J5ZJVwKhGTdcEUkq1O51bmybWT7lnSDM0BNrWroZsa3nuE1LU8h8adZOigYot2lPkmEJ1vtNv0lXj0\\\\\\\/ztkK8cL\\\\\\\/cMqFKnpsZipPynoM2TL1DMTkmwQzY3J7We7F44ioQD1QXUADVMKzYQsLt8xR355kME5uHenBAhdOdiaPEFJchupNDkzazfQuZc+piQRKJmOl1JhDfU1mlFzfodfq3AeAyPvRX2LJWrcMPnPvElsqrii3IZ\\\\\\\/2tL7B+G8BThaZpJYYx6zVZPFr\\\\\\\/qjxuaPLkD3twfJuZ9rEvdO5gZnrnEBH+Rm+44bkCMXRkQoPZgxmPU7+Sb7yitTnNWlslddogOjG2Su+kD73Yx668M7p3egCITu+OLqWeZvA0a6PCzkrDzs2giRsj2sbFCDaWfQGN\\\\\\\/tvf2mI0BjRT7On8qsQB6RstzBIajnBpQQQFW378X5Ao6JW60UkN7l9WhU24FpysNCMXF\\\\\\\/ZCbv5kIQ6ShXoo7rZ6ComMKV5BDf0fUZ9WAaDhYoRxVTX0+safXvB1J3hRws0jqhG\\\\\\\/0GqNvRF7QzzuTZt+xhgTW4MZmyk0jYH+cXrDlqptfSWZ9fWGzSJVt2sLXyeP6TiVHCE0DQpg4W+SDAC9bBWlIjJrZ\\\\\\\/Y+E6pGDZb1afEPoU6sjwBtm3WCBUFzveALxl6GuqFCgj88SrLAhHjdWIccW2RR4YH4oPeAo9e3CEQIIvo2pcPrciCwxDaUztD8ZqHDHgsHqdl6QhK44\\\\\\\/dLAmj7y4l3jnq\\\\\\\/rsAuK5vg3uKFF5HJ5sWQn8ZkFB8zuojAsZdVed2xC0hgqRlCqCW3RwXL\\\\\\\/gKCXFSwm2S0\\\\\\\/AJUxwGHea8yuYoNEMnL+8E2qUYP6xkYijxlO681OCk43GGBHwzxjnGLxBeQQjfq95S9M4sbehBsSvyDYx+3AwgXgDaoEUTnWeKsWbfsWtyhkkDUQZNY9HqO+HI+CW0DT5jsZXZaXCcJ\\\\\\\/jcjJHwD3+Rj8nbdBOGvHimD2gwaAERNoOmEFwARd732PX+wm9HgbD4+u9z8twXpzEN\\\\\\\/O7OU8AYpPxxHh7BYPVsOY\\\\\\\/I5qSaLTuS12dqEICzlbTsuIy59ObLW0eSlWTjLYlOYGl0SaaY9CMqVE2dxwOTClUhz6REvvDNNXXpKFczMw+musuzvA3rUELfeqzCUFI\\\\\\\/aG4QCL5eZkl+R7iLQJcRsnjG3+IwwdRRN7jbKLNa8SvRPfZlfjyrG4LszOGbMSXr1iFpyv86Rn2oidvaGEmzLf046M2L+x\\\\\\\/dKZjAynL5Lh56AojPb9vX90Cf\\\\\\\/jsJz70gknxBmLshK7P\\\\\\\/GexpZyQZUN\\\\\\\/YcPpr7BQRFWrLTziqLDhjysiVEeV0XPCm8v8l1DLQl7LaxJgyDT70\\\\\\\/zj5zBn0Dg+waAtDVZ5Ohcjmg16a8oHSxbkB6xCAXGHYAnQf4EgdVUTzxmTWkscRJpavhDxe\\\\\\\/cBhC+PmzlvxmT+OzGl1cCeOsWtaViR8BwPAymOf8oG7Ee3aYVc8UEf7coY1l+jodjHbqvJ3gu7EWIFYW8Tqd\\\\\\\/3nCtefIf0BC9iYuTaBctMiUvNdgQD6fqFPaKwMAqaGYZX+SVEcsI\\\\\\\/tmLGtv0vherLlXPTukz6mY0nCIWSpGzy\\\\\\\/Nqfv3ccBOEC87mqZIc6J5Oy4GlPzKMqwdULJ0DVdUgFyiM7\\\\\\\/K4X3yjxpOIRiXnqL3tEtivOdClS33Q4ivTF3Dw822FiVD9H8vMyHtnejzkVhExsfrvcnw9EOl7h5\\\\\\\/HcziYuWLCTzb+H3Q8f8p1Mgjevw7JKxx+IOKfDOwnPJXX6pgyy+0U9q6Ff5s6h9yecu7c8mkFfCj92quISv2azC5g94sv+zWOF7RrwdXT1SQPZwF9JORXCN8\\\\\\\/yei\\\\\\\/CRUhTcCsFn+\\\\\\\/2mIurHT8dpaSrz88hdRleUw0W3yK0A1\\\\\\\/r1y46OU8x\\\\\\\/o44fM6OpIkjYOnn1qIkbrqf7b\\\\\\\/adA8TEbV50e5T\\\\\\\/T8d8iYZD6VBtWf8EnLkxmSahDq6pTjh6pKElVZ1KAWOJfzrU+Kmh0cXaffRKYmH02vSYGRllQxM19SBR5Uk6WluJMqSotY+6KOqJirbaM+DQJbkcLY\\\\\\\/8d\\\\\\\/gN+YIrhjrfbq3EXCEhWf+KlACUgnS2z3\\\\\\\/baxj09LfmBBJjgab7VsULl6CY+rL2tJjFIEpRkQbPiZkPYi8vIUgVWPh7pKTJzyvcDJU2CRIFvDJnWLdMKw8SHUA2nrlm6a89+wA6GaO45WMRWwA1wJRSycrCeY7ExTso5AbK0MX6DOG657eb3OkHjoG+ZlkI+lGOauxPmcdh\\\\\\\/kW8nUM2FrvoBLpHNI7+BaBDWZqhWc0NkpE48nEvqYInyJISecIXoGXrHiT9KJbi9SSge4ZH82sH6aTiFpWyZ\\\\\\\/W\\\\\\\/rvBPAspzfTBrNzhQk9VzcWrdpIAITqnW2vgi5ICribaQy5XAzme9mPYQU\\\\\\\/frZZ6f9luDADJ\\\\\\\/OcfgsUS5SYa8oE8QKF4hLq3C6U4LNWc7BNTeiuJyxWK\\\\\\\/PSIg1nWILpdRW19XTfDqpvcu26Rz\\\\\\\/O3g1yD97dDTjk311UqIjElcIQDTTK\\\\\\\/icP0dnKLxkjc+UzXiRIoRqzEo679A\\\\\\\/wCJtoc57ihMKLZi+mnb0HOUfC3SbPkV0RYZAJA0y5W3wWibbb5\\\\\\\/5FUpllQDmzRzdALgkdNoWQyPYGqaoAjxteYkR\\\\\\\/+25zhNHw\\\\\\\/\\\\\\\/MPLDyxAyWOLjJPoe4sTKokWuUPR+09urrbhCa9vuGLJMkuWtyJR\\\\\\\/KaiLPy+EDV3hMsGM0Ip7xBvGokp2LAD9MGwIgISkszmh37r6Dk01k1JfAX8JfUieJiJzJ5oJCDhoXzJgG1YvpEfdalZ8vuUBG3mt2\\\\\\\/Hmvy7qK+wG0iP6+v+buN7h3t3RAQYyc6cveawXLUJV+nijhlleGxrwZK3XX8k9qONH8wLynjrD7ysUTxj6XGVgi3bkF9u9VPYm2\\\\\\\/f58ZL3pSuJZK4Zcer92q9g5RiJU45uGgnM4+GH78MhSgdFH3w6r1I7Q9pG\\\\\\\/6dY3klFQcWl+YU3n5TgbRrTgZSaMpP\\\\\\\/IJpoUhG\\\\\\\/O3S8en3Xri\\\\\\\/G+JoPct8G1vblw\\\\\\\/BRBO0DHY2kR4xsb+gujLT7CJfdvD7X7kGzbHLg4tD8JfuNJVLzHsRi6qqWK+d4PSS\\\\\\\/Ljy7LFc5fOxeS\\\\\\\/BoTon+lAc9VSuWxtq0\\\\\\\/hF6uqJPX64wYWDfPKfSZ0OMztC+lIVaS5Z+z9akvYq2NxJecIq4xpiNo\\\\\\\/jAKcvitHKLMacX3Hg3bUKuuHGGD3wNB4Us\\\\\\\/UoFzk3BWYBQSRkWxDi9hDIY\\\\\\\/DaYzP7WVk+9V5nAlYXNpjdYCB\\\\\\\/jkwgLuIRRb4xxwUmGrwko6ivGHHdeVBg97oSqGAuMjefi6LQBz6abPetv7ec9VDeZ6nzbCPcsHLXeCYHU+GJS8o3agGC2EqLOPozKjDwzfYJqrZlOGqOtT6yLpLyuf2erFoEMIrAygjlzy8pe1TjfwE97nYbdpwpBNRuhnRn0cfwhx6MM1Y5lKzq\\\\\\\/2twasNMG8Gh72dF7l5RxFsBUagW8IWadOQbkSV\\\\\\\/pLee933P5v7SHwT5ByHjIvEsiBBm\\\\\\\/bPtQQdIZeYncGUfuJBwayrSBPzmj7U6dzVWytULElD8EfCO7VqBJkylZojMTJLMpF38uToVJ8+i5BWD3KUa\\\\\\\/0Y1KhKrEdIKEEIoyafxB9RWtIPs7D5kKclsnSdKD6rZWY\\\\\\\/ufc6\\\\\\\/J6bfqQmeg+WbNlvlujrTs7MRHmbN4N1p+wqETyM4Obv88xGLk3elFh6RUS7nhaKrcfabOF1wGv9GnvFL8pwp0giYQdViZc8RebPYWAW2Mk0jIPR295Tffl4bZHPo9SNbEHKtvn3RwjIDPmUxVTic7r17+pU\\\\\\\/Y1Pt99YPpHn8JRx6SiPbdn\\\\\\\/Fujakzw15EvDo1SIZ0YewyVn\\\\\\\/2T1YSzEL6pqlAF9bI+IPfqRURrUDhB57XkkhyCjOkNdCzqTGUDzuOwgZKCzIuiNv7dKY6bbrQl1otY07Ry0a1pY8n6DSDBBvz5usR3jo8r9zsOSpCE7was+fqHvbEf0hwArFJ5pWbDuoZB71YFTsVHfm\\\\\\\/jfqZ0XVx3K7AizXF3Xwa\\\\\\\/HTvHNzAeefDUwQCEZbj\\\\\\\/boYMTIus3P1SFm+O3EbRW+vnhJJbNjc2FTnvzXokiXnKCNkZLCi2Y+lmLYoQPsUeEGM+qqNV4gWCH2WLFkA7OOP6awjAmEdsxoerza0iosEOC5jnuhx7wfXOKmy34QgEAI2iuta01lxBMm1iPkevMKoUZnXdFYo70iYRmkSjMLerfXusrwFGQExrhMv3snfsBa\\\\\\\/xqvw5R8f+8GQE+I5k15Xb1gTYzGfKuJKPF4+wT5gmTIBxRHTIZ\\\\\\\/aj4fRhQ\\\\\\\/GB6YoZTlToQzhrqPFM5LVQTguxsv1N9Y7ncF0KWt6H6Vdeo04w6A\\\\\\\/hbVRuNIjc6BibF+R3OzGpxAYG+NkNK44obFIdpaOaB8dJ0HG2ERAedsVfF55F51owZ3OcmcKIM3uALNAahEnXt\\\\\\\/m8N2a2mNUMOguzBUq0zcWWaQS2f5DH4sFZYXiurjKzroh7Ej58mlOitUkqRrtiKvjr6ucZjWidbJQ5huq23ySTwPcD25Szej3SF1mE88RaDqG0OseVh5IGvwFhqImwpJ5wfaSMuTGLzYL\\\\\\\/MREjWnR0BmvwD9oUEQlodaX8h49+0g1I+enn0tdXNdV9WO9pL9EsVMEN0RWqOUVMm2RWObERWJ1swyxWboQ9SENZxajk2+FTzQPF9TogXcAHm5Sh5JMlzM3lZWwNP4fa3YdS9wqwKmczT\\\\\\\/RwM+R\\\\\\\/uuWSHXAN5sAN3ZM+DHu\\\\\\\/hNLbqCWKgxtlOfZ9vQzoi71GextkY6UjlK2CvFg7UtV50P7O1Tzmrp98sMlF1A7iHf5dOx\\\\\\\/Qq6aD5YlpM3cdv1iJV71xT8ZYjrFs5ck\\\\\\\/IwpUnztjjOSgtC62u+DNXxiYk9ophVGD6ux4ibespWyCQWhjLCyxWFcKRaBcG84azK3+PxhmlrqxfjDD6Ubizj98RKX7m7ShaxXv3GV2rDPhvAKL1stOe2ozKN7Kld0YAsqtIWEVMZTUgBlFFcmZE6e1F36trLkjMerPffEqy5Y+GWZYyfBwDNAsx7WxetWFr7fitl7k2YSOw6S4HVXSjzDn8DtOOivPmwv9NE204nCeChCxxyys7v0lPWHoEb\\\\\\\/fMymwQx\\\\\\\/ROA\\\\\\\/Y8cmHGXbpHfOGUTOwWIFYUdtQulhafkkXhE6YRrV1EzhxflhnXEw4MqoXcGEiaNMVFGUV3umJTn9fetCb5HDIHO37Zfqv2Om3172gkvZfqaeLwe8VtiUtFCoTqjt3bq178faNbQadlmOxrG8bFPx+Bl2nNTViIkEkCkbLHpZ7L2cqkoVrXqPfm\\\\\\\/4dNooCI44ybJa+Ocj6Ekzcwwsm6BkYS+zxcnOutTpS8i0wuYS3czENNcCuFOxmmTr2VW5RUBVGp1vj0p\\\\\\\/Y7TbvX3olgo05A1hbadpWAw8cVCOdhBj79v\\\\\\\/4AY6qGoaAuDgigBEkfO3CIjqB32TpYGmAtkoVKxbUYb5FLuVeZlUfc5QtQesE3mO4hK7nrp8gasjxxiLWQTIKx+6+0ZI\\\\\\\/dP2QzEGanclWU1m\\\\\\\/nrwmtVGa4sJwFha0GqqXPIr6bFOxMpEeGMAvoy0FV4nbZ+dm5qAY14D7kW04bcEzb9c288ktV9HyFVzqT86yUbyaoeMoC4D2eXbp+BNRL67DihbB79UI06PU8EMRoMwPmyWMJUYKaDFtim2YvW83FHDI2gIqXazAelRLvBpnttAE3zVJ5Rzj4IxWKRnHVZRpCVNBDtv5bYZM2Dttrxxa41V3K8jc0uKbqV8FYRkZCV52lxm7bpPWFp9vOCK\\\\\\\/BcJRbgSUKtoLpcR7cMDtjGz8O+m\\\\\\\/6+4mJYqtavUXd2ahd1hd3J5K+WmhK62QfFgGjEKJ1H9bRzAjggOFVHggGS\\\\\\\/2jxzMKaMzkiQ9Wap88Xgo64\\\\\\\/udggHmucsSuqu+gbu5JFA0fPYMHERYJ\\\\\\\/nkMIh0N4NagsEVwLtO6sJ\\\\\\\/AIc0JPMA4fIBte0LrWOXsNzxs3v5h\\\\\\\/3EMeobM99AetVES\\\\\\\/UiznYjx1m4KFf3DHAFM4tALRPjLGfPHEKEuii33oYnXor\\\\\\\/ELMiiv4Ue7sH882Vr4B\\\\\\\/kUidjnnf8Uj1u\\\\\\\/097SLSad7aO+7cAXWOGhhC4tjU2CojfUVG046luyQBgWXMChrMkjcL6z3cDXdWCXRoA1LsbKdnoCXcOIIEduoyXvRjoRW06WCEpZLNEYwKnuxCXOAeH\\\\\\\/1ZpLc375A1tuFXZoQONDR0HIiY4kDZaTjpfHNVjk4rd23aNuzndVB\\\\\\\/v0ERKOfc2ic6fg5MXSp5zttgB3a8LuIKH9RpV0LObjjSa2RCG+t8N0b86YyytPvEFeGbpFDbQBiUWXgJgSIwZAHXcmMPmKTsTFzBecq6Ja6Lxv86juKVZ\\\\\\\/nuVHEyv3SyYSBtOXY44lCO\\\\\\\/7UQGrYBkjgLSIBqBDBQihQ39Cv8yh41bBNVN+wL\\\\\\\/CkJ4JrOV9AFhwSYXqNO\\\\\\\/0RseUlZ7K8Qd9u2XXJi0kLes1\\\\\\\/tqO6foYm4Rv6hAtWQsZSMiBexJk+4srciewZcgVPV5qAK4+zNfrnEA5Zqz\\\\\\\/uGvArBRkHKOzLLHHSTD8ZnnKdrdDzBEymOOnEHM4G\\\\\\\/EDrtVn+a\\\\\\\/6RI\\\\\\\/Me4h62J1qIXSF47yHZWtqF1g+PHK8nqFSCPNwDlgQXBtr+hjIu65zSH7WcHR+IZoGWeejvqJYpl1a1C\\\\\\\/P\\\\\\\/yBUeHeCP3NqkVDS8cNXVVT8PV+8QkQJDAXgRnwZonVdXxGx+lmTzOeP6zCvcwDJLsQ9qqQrczicHnTPwmcjv5mqVSbxPBIdVJa\\\\\\\/5vthxjPQvq9KMoBEescH+FPW+tOgONOCBR6gnFw6BwT4cTIOUL3pYGawUDUG5iie6Do++Y+Fi7285nZz5gFD+Oz5q0wYJayVKf1t5WzVXSFvMt8IpDk2BCsDbVu6LxpHH5cf2nDzm+tkijk9G6R5YQWNJBcjf3oLt9Jbr\\\\\\\/iU0EuXKoWM9JnC1D+u7PoCf+VNb7nIQf8Hmyk+ApGP4plMydx\\\\\\\/F2e3A9ZFqMhxLDViQ90W732+f38tiw4ypuN0nJpKg7HuYl5GYWMun0EKh+xEUCq+S7deZs\\\\\\\/CMFDxW6ZpaJQFMzGZ0NGToo8RN4sjdmRJW4Byjpx5ZDXYg6uL4jf+Y7Ko4NjsrBw\\\\\\\/Xt5qPHyggn7fQ2cdT8xjXJguq5LkfItO5oYUnu0\\\\\\\/j4NrCnhktIQFTEvIv3NSpOyeE6eAZ1pdzT5g60IUawItSQsunkENom9t1E0zym56mY6AEB4y5WGdU8+hDpnjIz4m27ohLWZnBDMIBOO\\\\\\\/sckBFbcuXnsjKg2r8ltkN4BBcFkb5o1N37J\\\\\\\/3DJJix1yABZDTnxjoWjm2BCvKw42a3CN4qsEgVkBMwmjqLQzoct39X6WFpCmh3gUs1FQNClYijW\\\\\\\/JDw1c9em96HA2GIWlVHVsimK7Vmw4pZj+MiI6r\\\\\\\/wcbTW\\\\\\\/odUm8YWIdf2CdsQQwk1qZ2EcoH3EKNS+JqVcoSwkyjfNPRhXDxW+NmJZJXc5IZ6FnBfsoVTm2\\\\\\\/y5RR4Lt8vS\\\\\\\/NXKtCWPe2158\\\\\\\/gQd7BYFqOC2r79uZrlfuCM+JO8Gj8p54858z4Sdz\\\\\\\/sjZajUwRxtRFoVvuOBGO8JSf4eowuF\\\\\\\/LTUd9oH8L40r7P7kbMYfi4RxdQgHxxsMm+dHrNO2NkLxPz7gDWmnHHFCjXqKFWtbgBTDfJCaB18rzvnuSfqFYWMYYgHmhOu\\\\\\\/HwERB732IpFylQoEaxW3FQ0XRR60dCaAVxjhEWEwddKILCcTGxG4NuMe4TZMt6rbsAOAjzFM+dGn9G4NvYVbLO6yoZ36ASeWMs3ivyrnd4k\\\\\\\/iEHOv3sf1uHvVTJFuR0GfX3w2TzkjDxW5l4GO1lymdKbtDpBQw4pYCyVy1EyVWhoxoKbjl698TKu3TPfWUeE2DW1aE1JPJcbEtbttXd2ybQjkMTZfHXtCJPR3FdYPHoYa\\\\\\\/bd8YmfyIlLjaaSI7IeUh4wvhJITJOYyiWhdxHXziS4xwxbHM4YR\\\\\\\/xX744ucpquEFfWNjFndUa1dW+RVoAqKUc0BYkzNY5W5BxoviychZwO7r1p9WURykIgn7c+vbMzwhN7QsUZFEJvmb7hNOUYYQEal\\\\\\\/XhJVZaj+oMQy39Wqg5SNln8IBfmlPpTuqwFueKNMPsf1K2GDeuASAY7M8bw\\\\\\\/i+H1JEPDT+S0lJqLSLckvpKeHgHaPfn0WU6eNVR1hqnMjRGI8h6qq2Wu3m1OtuoxgyVADTuaRQyHSya61\\\\\\\/8Y99E+2x5X5AvNNtyv3xj6wg7XaTf9z3js18cV5X6BUYAba3APoGmLdEsr0iH\\\\\\\/IhUv1Fo599Za66r\\\\\\\/HIZ3K8gbKwF2IFEjznzXK3LZSFyTjgaxKgM++l1g5fFqrZEB95CKeTU5DQilkcfe5LqV6ZbHZWXiv6aasm6uqD2S4ZOXND0IQL05ZO46Y5GA+ryNKExs8XPmY1othGPEkzUVcKyokoetA6RiSKOrgitg5CdRCBNlPetleI99xikUgshtSKHTVdG8Hdw7dCjg4jH0vuQQpSvTNoDU+nw1bPe4BFukbz0YeoFPmApMaFoNABIlcLpbbvMM8A0QKq348lE+vuL+EgmJHQlbZHw3Y3twQnUOl5WeqwaWNJJyKbwcjcyTPBXhqixYSlP0ujXiua4r1mKcG+urgKsZI1bXyquf8Ctjhz4\\\\\\\/D2GRgaJU5OCAzjcRoc7Z9YlIuFXBLiL5Ma1Dm6NFGuPOzHezAAfJeqoa4XbH+2r9GWTPfMYA\\\\\\\/zoxAoy3yy65uCgnqBHGvsU+YOUpyH2776KhGfQXz6oD7BN3W6cQnejfw3vsjhJwKx3zVK5s5q8yDkCBK9X0PDnSUJyIOk3rpkCORFY\\\\\\\/FPhLqH+jO8XSCqzxwKEFC8zzPL5hFjI0Iy5+YbXBX0a6RlWRA1KWP1lKnPXrMiWjAd4h16fB5JZ1njOpwEf1HQIwW5cJBP+WAS0DztI7G47CuByFtAA88o5tt6KvQp1BqO8dbzCExzeJXvpWiHt01I02wybBeq1SUe4kGv+WRWCO1eJx5S29a03Z0azY3rmWwydr6p1XdyUVs4uR7FGoem+5tGCxZwB2LoUK3fLpg1r\\\\\\\/gVgwv1\\\\\\\/IvhRI7GusvZYVhUWFXRK71TB21VY9sd4L6V5DP9uNzkU6aSEwtrugjrkgn1e\\\\\\\/nFMirFTMMJNFe8r+roI0PAEQsU1n\\\\\\\/oeA4LRUGLlXbo8VDj+T1vNGYDhYzl5Noh\\\\\\\/ABGfnpJJxMpT4+167IThs92wfNH7j+R5TesHy1PuUk1XFRtDrqXEMU0CN92jOjzOE7Xk6JpAcbOWLJXdDBkG6QcYVJrKSUSilMGwi3sEiNwwFRuzBxfk+j7DYlBDvYAFByrqiIcfm2im3XyFBPrLWKD1iuzFCWhb5zpR\\\\\\\/sj\\\\\\\/9Q8vF2fOLhUGfKKz6\\\\\\\/YTaCiyJOpj40lrje2\\\\\\\/kMBdc8lpZBRtZTMb\\\u0022,\\\u0022iv\\\u0022:\\\u0022BGsaecysffPKmbwExItyuQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002218c44ded19079285c8c3db9292f806129e81f5ecc682f33570433c0cea6baf79\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TWGJ4X2GlzNY+xt0d+1WLga5Qv9uC1baOzahBcFo7dCZk++irizDwcMJd1TYGzhwTD07NwS4OPauknJ3hGSVYA\\\\\\\/\\\\\\\/dyOVH1Bk85MY6vESOEAKmSU3TDYGRQOV3jHC9K7jLKJufJGRF0WOLE2fpuHPUiygGjKA+O9piRT9fgEpm4j4tIXOqbvY7+i0CMc78EOU5QgFl4mEez520QMFm6bc6yqK2a0PCzL6L+uiBuSlQSURgjzVJn948UvG2cF5bChw8\\\\\\\/7jOe6hcrcEbWcMkfarFSHzDjJ5w7cw9wJBBo4DuvEnXs2yroy+1GAd7lJA8I5XHZKBj+OxRR3ZFTzAMU8Od0SAERU9zZpeS9+mGPsLO\\\\\\\/hNCMuGaVVsedboORfE2roZpI5NB785uYm7h0bx8t621u15xdU+\\\\\\\/BfVutpplXvQ4YNLc\\\\\\\/pY2ROf9GjXBrD44UPKlnBNmey989tYA82koCFkSNQudeBzlgZbqM0KQQHDF00kp6O113nSWX8DNzEYrLpzFRPjgSVzTSi0a7dO0xbgn0r9YVmWKvqKnzFSdFCxJBbbGKojhHUm6n95kcxc2CoQ8AVzTC2zbvix48gAGIEbCrHg2RksZ1EEP9zDAjslNAOu+\\\\\\\/Pl0emZrep3e\\\\\\\/RBda6eAOYyT8hX+ztxfEeIPypGxpC7B9krYNXCoehnmTnBNX\\\\\\\/NU9AUNOYMwb6wJweU8aqgMzkcFobUoQS2K4b5TYtBI3hP6yLVwLOB0xRLT+pE10TA8sOA5BV5CPh1jMmMVjnBONeCo8GYEZx48oFjdQz85Vk0Q+08xDlmX69a1UwJf\\\\\\\/FK9gKSWzwookn\\\\\\\/iKfq2+U1lpmAKCpVAXGbB2BW5HHnKcrITDxVIO9J7CJGUwYJPXt9XR8ajY4P4NnBjPOX48DqOZ9WhAcA9ojEeElWTmWURBcBvYc41qxX3H0263tl6Y6VNwKoY35JrTXiGDiMVNErKQw37fSt\\\\\\\/S\\\\\\\/XvStBNCsKg1rxFfeRGmg4DzoJm9oy6Egw65kwiG\\\\\\\/yCYPeuBMKNmweup29zQUF05JQ4lMYWLut4PwG5yvFwoJ1kjOQdnvw\\\\\\\/wYRtrlqscyHmA\\\\\\\/1Gew2HKm2YOA2OskbB9Rso1m2i172fAiT90te6Uq4\\\\\\\/HaJ\\\\\\\/X1j5hSLMfX2HjnQAi1IHypt1WvBlHklxu2VL+Ep3R6lBBobZN10QnUL+SgxG6bEwB0APWFG16vBWfs7ygKioMZ872VJpSvuWND2btZVjhVc5d3V2kIDbMuZxGaEKkOUO\\\\\\\/DgDlLSsKVhqqe5xrMgCLe3bDxpCZZka6MDAN\\\\\\\/g7qFEH8O33CeFQnqUpBMURR+Isb0A2VkHeFvtH1sbzErMGWE9DKZ4cyQo+nqHf6nimJU7reUKllDfFkb\\\\\\\/w3KI4+NRke+99VABat+nne268FypPmbCELii+rtJRw0j7tveCI1uqXmYgYByHOYkU4Vw2yQfob9N7bOh232B85SuT3sFsplLO8PiYiU3\\\\\\\/S7Hs5sM0DhkThFTleN5w6f4YYjM4e8RLApjoE8L3Gkh\\\\\\\/VbfNh6rKp0qg20guDsZoEIdBf3mB3tcfpIuRIq7ufQ9xdouN+uHAeykfGlnYq6MqNMyrhjYjjTVKcuMXs\\\\\\\/4YoBWWVmMDmxPJqAbWNn3HvG1gvINqEb6n8jga9x58thV4gvxlCoTFr+2Vmnr8IqYN+tuk3A7T\\\\\\\/NdPqcV9DGy0hAP0\\\\\\\/+roqppNQ3LipjZOfEVYqiK5xO22rURXhWGLzQobd3EPjKJdR+N1i4KuDBIMimRbB\\\\\\\/OiwdBZfD4Nhbswa2U9MCrOUMb26DzhQQCG5HjxxDacRSNS\\\\\\\/SxNhnB6rcnVPuwe3LZHPYILdxn9HvoOZJuYtvISgsqL2GM4MEyFHHdThuFuS3Z3+LAN7U42kbacuBp3nJMwB4YLaLVlwpWsYKDKs3fO907e+aiaBJhCaae9n9M4OO5zwfoqeUEi10Jg\\\\\\\/daMxxpfLTZZsThzbN2B0t\\\\\\\/747GPuqXxfGNPSURX4FnFcj9lECbespCRCfV8EOT+20uRzMbTSPbJPzXd0o=\\\u0022,\\\u0022iv\\\u0022:\\\u0022EwFdH2IOAtK+9wR0EuBuzA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022313c428c5c8a98b2ea73643bb27a0c305d0933527bc62c67a08c34c6a4a9dccd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460531 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/41" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MDIDY_YtNwvy68PX-_BPt6QtFKbr20vJN-_RXA62XIc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/41" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-20Xp5vUuOIhIAgnApqD-1EYwjWVQqWFA6gjMnKeXmCw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/31" title="Small RNA Sequencing in Cells and Exosomes Identifies eQTLs and 14q32 as a Region of Active Export" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/55" title="Assortative Mating and Linkage Disequilibrium" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460531" id="node460531" data-pisa="ggg;7/1/41" data-pisa-master="ggg;g3.116.035584" data-apath="/ggg/7/1/41.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Genomic Prediction with Genotype <strong>×</strong> Environment Interaction Kernel Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="1">José Crossa</span>, <span class="highwire-citation-author" data-delta="2">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="3">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span> and <span class="highwire-citation-author" data-delta="5">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">41-53;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035584 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460531" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, C. P. 56230 Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/41" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460531" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/41.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460531" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/41.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460531" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/41.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460531" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The phenomenon of genotype × environment (G × E) interaction in plant breeding decreases selection accuracy, thereby negatively affecting genetic gains. Several genomic prediction models incorporating G × E have been recently developed and used in genomic selection of plant breeding programs. Genomic prediction models for assessing multi-environment G × E interaction are extensions of a single-environment model, and have advantages and limitations. In this study, we propose two multi-environment Bayesian genomic models: the first model considers genetic effects <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-1.gif"/></noscript></span></span> that can be assessed by the Kronecker product of variance–covariance matrices of genetic correlations between environments and genomic kernels through markers under two linear kernel methods, linear (genomic best linear unbiased predictors, GBLUP) and Gaussian (Gaussian kernel, GK). The other model has the same genetic component as the first model <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-2.gif"/></noscript></span></span> plus an extra component, <strong><em>f</em></strong>, that captures random effects between environments that were not captured by the random effects <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-3.gif"/></noscript></span></span> We used five CIMMYT data sets (one maize and four wheat) that were previously used in different studies. Results show that models with G × E always have superior prediction ability than single-environment models, and the higher prediction ability of multi-environment models with <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-4.gif"/></noscript></span></span> over the multi-environment model with only <em><strong>u</strong></em> occurred 85% of the time with GBLUP and 45% of the time with GK across the five data sets. The latter result indicated that including the random effect <em><strong>f</strong></em> is still beneficial for increasing prediction ability after adjusting by the random effect <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-5.gif"/></noscript></span></span></p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/multi-environment" class="hw-term hw-article-keyword hw-article-keyword-multi-environment" rel="nofollow">multi-environment</a></li><li class="kwd"><a href="/keyword/kernel-gblup" class="hw-term hw-article-keyword hw-article-keyword-kernel-gblup" rel="nofollow">kernel GBLUP</a></li><li class="kwd"><a href="/keyword/gaussian-kernel" class="hw-term hw-article-keyword hw-article-keyword-gaussian-kernel" rel="nofollow">Gaussian kernel</a></li><li class="kwd"><a href="/keyword/marker-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-marker-&#xD7;-environment-interaction" rel="nofollow">marker × environment interaction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">Shared data resource</a></li></ul><p id="p-3">The long, rich history of the development of statistical models for assessing genotype × environment (G × E) interaction in agricultural and plant breeding experiments precedes the development of the analysis of variance. Fisher and Mackenzie (1923) pointed out that the differential responses of genotypes to environments could be better fitted by a product operator (multiplicative) than by a sum formula. More than a decade later, a multiplicative operator consisting of a simple linear regression of line performance on the environmental mean was proposed by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Yates and Cochran (1938)</a> (joint-regression analysis). This is a method that approximates G × E interaction by one multiplicative term. Several decades later, other multiplicative operators based on singular value decomposition (SVD) of the G × E were proposed within the framework of linear–bilinear, fixed-effect models (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cornelius <em>et al.</em> 1996</a>). Later, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Piepho (1998)</a> and <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Smith <em>et al.</em> (2001</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">2005</a>) employed the SVD operator for modeling G × E but in the context of multivariate linear mixed-effect models, while <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2004</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2006</a>) and <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> (2008)</a> considered using structured covariance matrices to model G × E based on pedigree linear mixed models for estimating the BLUP of the breeding values. These models account for the average performance of the interaction across the entire genome without distinguishing parts of the genome that may be more influenced by the environment than others, and using environments without characterizing climatic factors that may interact with regions of the genome.</p><p id="p-4">The first to propose whole-genome regression methods (genomic selection, GS) by jointly fitting hundreds of thousands of markers with major as well as small effects were <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Meuwissen <em>et al.</em> (2001)</a>. Implementing whole-genome regression methods poses important statistical and computational challenges because the number of markers (<em>p</em>) greatly exceeds the number of data points (<em>n</em>) available; however, shrinkage estimation procedures allow the implementation of whole-genome regression methods. Recently, standard GS models were extended to multi-environment data. For instance, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> (2012)</a> were the first to use a multi-environment version of the GBLUP where G × E was modeled using genetic correlations; however, they did not attempt to incorporate environmental variables as surrogates for environments. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> (2014)</a> proposed a Bayesian reaction norm model that is a type of random effects model where the main effects of markers and environmental covariates (ECs), as well as the interactions between markers and ECs, are introduced using covariance structures that are functions of marker genotypes and ECs. The proposed approach represents an extension of the GBLUP and can be interpreted as a random effects model on all the markers, all the ECs, and all the interactions between markers and ECs using a multiplicative operator.</p><p id="p-5">The reaction norm model of <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> (2014)</a> has some limitations, for example, the Gaussian prior does not induce variable selection and the shrinkage induced by Gaussian prior density may not be particularly appropriate when markers or ECs may have large effects. Furthermore, the reaction norm model considers the case of a particular multiplicative interaction model and, as such, may be considered a simple approximation to the complex phenomenon of interaction between genes and environmental conditions which, in practice, may take many different forms.</p><p id="p-6">To solve some of the challenges of the reaction norm model, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> proposed a marker × environment interaction model where marker effects and genomic values are partitioned into components that are stable across environments (main effects) and others that are environment-specific (interactions); this interaction model is useful when selecting for stability and adaptation to target environments. Consistently, genomic prediction ability increased substantially when incorporating G × E or marker × environment interaction. The marker × environment interaction model has some advantages over previous models: it is easy to implement in standard software for GS and can be implemented with any Bayesian priors commonly used in GS, including not only shrinkage methods (<em>e.g.</em>, GBLUP), but also variable selection methods (which cannot be directly implemented under the reaction norm model) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2016</a>).</p><p id="p-7">The marker × environment interaction model of <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> estimates the phenotypic correlation between any two environments as a ratio of variance components, thus forcing the covariance between pairs of environments to be positive. Therefore, the marker × environment interaction model is appropriate for use with sets of environments that are positively correlated. However, in practice, this G × E pattern may be too restrictive in cases where several environments have close to zero correlations; this determines a large variance component of G × E as compared with the genetic variance component (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2011</a>).</p><p id="p-8">In a recent article, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> applied the marker × environment interaction GS model of <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a>, but modeled not only through the standard linear kernel (GBLUP), but also through a nonlinear GK similar to that used in the Reproducing Kernel Hilbert Space with Kernel Averaging (RKHS KA) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2010</a>) and with the bandwidth estimated using an empirical Bayesian method (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Pérez-Elizalde <em>et al.</em> 2015</a>). The methods proposed by <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> were used to perform single-environment analyses and extended to account for G × E interaction in wheat and maize data sets. In single-environment analyses, the GK had higher prediction ability than GBLUP for all environments. For cross-validation where some lines are observed only in some environments and predicted in others, the multi-environment G × E interaction model with GK resulted in prediction accuracies up to 17% higher than that of the multi-environment G × E interaction model with GBLUP linear kernel. For the maize data set, the prediction ability of the multi-environment model with GK was on average 5–6% higher than that of the multi-environment GBLUP. <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> concluded that the higher prediction ability of the GK models coupled with the G × E model is due to more flexible kernels that account for small, more complex marker main effects and marker-specific interaction effects. However, the marker × environment interaction model using the GK of <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> also assumes sets of environments that are positively correlated (as in <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> 2015</a>).</p><p id="p-9">In this study, we propose two multi-environment G × E genomic models that attempt to overcome some of the restrictions of previous genomic models. The main objective was to compare the prediction ability of the two proposed multi-environment G × E genomic models, each used with two kernel methods: linear (GBLUP) and nonlinear (GK). One multi-environment G × E model considers the genetic effects <em><strong>u</strong></em> that is modeled by the Kronecker product of the variance–covariance matrix of genetic correlations between environments with the genomic relationship between lines (using GBLUP or GK methods); this model with <strong><em>u</em></strong> is parsimonious because it estimates the combination of the genetic main effect plus the unstructured genetic variance–covariance interaction matrix between environments. The other model has the same genetic components as the previous one <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-6.gif"/></noscript></span></span> plus an extra component, <strong><em>f</em></strong>, that attempts to capture random effects between environments that were not captured by <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-7.gif"/></noscript></span></span> Both genomic prediction models assume that errors have a diagonal variance–covariance matrix <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-8.gif"/></noscript></span></span></p><p id="p-10">A total of five extensive genomic data sets were used to compare the prediction ability of the two multi-environment G × E genomic models (each with GBLUP and GK methods) among themselves and with the prediction ability obtained by the single-environment (also with GBLUP and GK methods). These five data sets have been used in previous genomic studies where prediction ability was assessed only for individuals observed in some environments but not in others (cross-validation method 2, CV2, by <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>).</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Statistical models</h3><div id="sec-3" class="subsection"><h4>Single-environment model:</h4><p id="p-11">The semiparametric regression model for each single environment (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-9.gif"/></noscript></span></span> environments) of lines <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-10.gif"/></noscript></span></span> is given by:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-12.gif"/></noscript></span></span> is the response vector containing <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-13.gif"/></noscript></span></span> phenotypic values, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-14.gif"/></noscript></span></span> is a vector of ones of order <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-15.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-16.gif"/></noscript></span></span> is the overall mean of the <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-17.gif"/></noscript></span></span> environment, and the random vectors of the genetic values <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-18.gif"/></noscript></span></span> and the errors <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-19.gif"/></noscript></span></span> are independent random variables with <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-21.gif"/></noscript></span></span> respectively, where <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-22.gif"/></noscript></span></span> is the variance of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-23.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-24.gif"/></noscript></span></span> is a symmetric semipositive definite matrix representing the covariance of the genetic values, and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-25.gif"/></noscript></span></span> is the vector of random errors in the <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-26.gif"/></noscript></span></span> environment with normal distribution and common variance <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-27.gif"/></noscript></span></span> The biallelic p centered and standardized molecular markers in the <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-28.gif"/></noscript></span></span> environment are represented in incidence matrix <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-29.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-30.gif"/></noscript></span></span> is a linear kernel. Model (1) is known as the GBLUP (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">VanRaden, 2007</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">2008</a>). Single-environment model (1) is similar to model (1.2) of <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Pérez-Elizalde (2015)</a> when the linear kernel is used.</p><p id="p-12">It should be noted that under the conditions given above, model (1) estimates the genomic relationship by means of its linear kernel <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-31.gif"/></noscript></span></span> However, nonlinear kernels such as the GK can also be used (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> 2016</a>). The GK commonly used in genomic prediction is <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-32.gif"/></noscript></span></span> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pérez-Rodríguez <em>et al.</em> 2012</a>), where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-33.gif"/></noscript></span></span> is the distance based on markers between individuals <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-34.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-35.gif"/></noscript></span></span> for the <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-36.gif"/></noscript></span></span> environment and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-37.gif"/></noscript></span></span> is a bandwidth parameter, which in this study is estimated based on the Bayesian method proposed by <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Pérez-Elizalde <em>et al.</em> (2015)</a>.</p></div><div id="sec-4" class="subsection"><h4>Multi-environment models:</h4><p id="p-13">For multi-environments, the random model considers that the individuals between environments are correlated such that there is a genetic correlation between environments that can be modeled with matrices of order <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-38.gif"/></noscript></span></span> Therefore, the extension of random model (1) that accounts for genetically correlated environments is expressed as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-39.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-40.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-41.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-43.gif"/></noscript></span></span> where μ is the vector with the intercept of each environment, and random vectors <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-44.gif"/></noscript></span></span> are independent and normally distributed (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>) with <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-45.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-46.gif"/></noscript></span></span></p><p id="p-14">When the number of individuals included in each environment is different, then<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-47.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-48.gif"/></noscript></span></span> is the genetic variance of the <em>j</em><sup>th</sup> environment and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-49.gif"/></noscript></span></span> is the genetic covariance between two environments, <em>j</em><sup>th</sup> and <em>j</em><sup>th’</sup>, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-50.gif"/></noscript></span></span> is the kernel constructed with the markers of the individuals in the <em>j</em><sup>th</sup> environment and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-51.gif"/></noscript></span></span> is the kernel constructed with the markers of the individuals included in the two environments, <em>j</em><sup>th</sup> and <em>j</em><sup>th’</sup>. Also, the residual matrix is assumed to be diagonal<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-52.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-53.gif"/></noscript></span></span> is the random error of the environment.</p><p id="p-15">When the number of individuals in the environments is the same <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-54.gif"/></noscript></span></span>, the kernels are the same <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-55.gif"/></noscript></span></span> and the identity matrices are the same <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-56.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-57.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-58.gif"/></noscript></span></span> where ⊗ denotes the Kronecker product and <em><strong>K</strong></em> is unique (calculated for all genotypes, regardless of the environment in which they were tested) and could be the genomic relationship matrix as defined for model (1). The matrix <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-59.gif"/></noscript></span></span> is the product of one kernel with information between environments <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-60.gif"/></noscript></span></span> and another kernel with information between lines based on markers <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-61.gif"/></noscript></span></span>, similar to the multi-task Gaussian process (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bonilla <em>et al.</em> 2007</a>). The mixed model used in genomic prediction can have several structures for modeling matrix <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-62.gif"/></noscript></span></span> (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>). When there are not many environments, the unstructured variance–covariance could be used for <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-63.gif"/></noscript></span></span>of order <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-64.gif"/></noscript></span></span> such that<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-65.gif"/></noscript></span></span>where the <em>j<sup>th</sup></em> diagonal element of the <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-66.gif"/></noscript></span></span> matrix <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-67.gif"/></noscript></span></span> is the genetic variance <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-68.gif"/></noscript></span></span> within the <em>j<sup>th</sup></em> environment, and the off-diagonal element is the genetic covariance <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-69.gif"/></noscript></span></span> between the <em>j<sup>th</sup></em> and <em>j<sup>th</sup></em><sup>’</sup> environments. For a large number of environments, a factor analytical model usually performs as well or better than the unstructured model (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>). Also, matrix <strong>Σ</strong> is an error diagonal matrix of order <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-70.gif"/></noscript></span></span> <em>i.e.</em>, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-71.gif"/></noscript></span></span></p><p id="p-16">As described, model (2) is parsimonious because it expresses the genetic values within the environment derived from the markers plus the interaction between these genetic values with the environments. Model (2) can be used with the linear kernel matrix <strong>G</strong> or with the GK that allows capturing small cryptic genetic epistatic effects.</p><p id="p-17"><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> (2014)</a> argued that due to “imperfect linkage disequilibrium (LD) between markers and genes at causal loci or because of model misspecification (<em>e.g.</em>, interactions between alleles that are unaccounted for), the regression on markers may not fully describe genetic differences among lines.” Therefore, it is reasonable to add another component that would attempt to model the variation between individuals that was not captured by <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-72.gif"/></noscript></span></span> Thus, we added to model (2) a random component <em><strong>f</strong></em> representing the genetic variability among individuals that was not accounted for as a function of the markers in component <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-73.gif"/></noscript></span></span></p><p id="p-18">Therefore, multi-environments with random effects considering genetic correlations between environments (model (2)) can be extended by adding an extra variability to account for the genetic variance among individuals across environments that was not explained by <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-74.gif"/></noscript></span></span> that is <strong><em>f</em></strong>. Therefore, the extension of the random linear model (2) is expressed as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-76.gif"/></noscript></span></span> with the random vectors f independent of u and normally distributed <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-77.gif"/></noscript></span></span> In general, when the number of individuals is not the same in all environments,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-78.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-79.gif"/></noscript></span></span> is the genetic effects in the <em>j</em><sup>th</sup> environment not explained by the random genetic effect u and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-80.gif"/></noscript></span></span> is the covariance of the genetic effects between two environments not explained <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-81.gif"/></noscript></span></span> When the number of individuals is the same in all the environments, then <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-82.gif"/></noscript></span></span></p><p id="p-19">Matrix <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-83.gif"/></noscript></span></span> is unstructured and captures genetic variance–covariance effects between the individuals across environments that were not captured by the <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-84.gif"/></noscript></span></span> matrix; in this case, matrix <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-85.gif"/></noscript></span></span> can be expressed as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-86.gif"/></noscript></span></span>where the <em>j<sup>th</sup></em> diagonal element of the <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-87.gif"/></noscript></span></span> matrix <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-88.gif"/></noscript></span></span> is the genetic environmental variance <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-89.gif"/></noscript></span></span> within the <em>j<sup>th</sup></em> environment, and the off-diagonal element is the genetic covariance <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-90.gif"/></noscript></span></span> between environments <em>j</em> and <em>j</em>’.</p></div><div id="sec-5" class="subsection"><h4>Considerations on the application of the proposed models:</h4><p id="p-20">An objective of this article was to compare the use of linear and nonlinear kernels for matrix <em><strong>K</strong></em> to determine the relationship between lines, for each of the three models described earlier. Thus, for each of the five data sets, we fitted models (1)–(3) for <em><strong>K</strong></em> as a linear kernel using the GBLUP, and <em><strong>K</strong></em> as a nonlinear GK with the bandwidth parameter estimated according to <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Pérez-Elizalde <em>et al.</em> (2015)</a> on model (1).</p><p id="p-21">The same numbers of individuals in each environment were employed in these applications. Therefore, in this case, the same kernel <em><strong>K</strong></em> constructed for all individuals was developed. Also, the observations in each environment were standardized with the aim of examining and comparing the proportion of variance components explained by each random component of the single-environment model with those from the two multi-environment models, and also between the variance components of the random effects <strong><em>u</em></strong> and <strong><em>f</em></strong> of models (2) and (3). Although the intercepts were expected to be close to zero, they were included in the model as parameters to be estimated.</p></div><div id="sec-6" class="subsection"><h4>Implementation of Bayesian models:</h4><p id="p-22">Single-environment model (1) was fitted with the Bayesian Generalized Linear Regression (BGLR) package of <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos and Pérez-Rodríguez (2014)</a>. The BGLR considers a Bayesian model and, from that point of view, a linear mixed model is a three-stage hierarchical model (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jiang 2007</a>). In the first stage, the distribution of the observations given the random effects is defined and, in the second stage, the distribution of the random effects given the model parameters is added. In the last stage, a prior distribution is assumed for the parameters. Under normal distribution these stages may be specified as follows: the conditional distribution of the data from the <em>j</em><sup>th</sup> environment is <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-91.gif"/></noscript></span></span> in the second stage, the conditional distribution of random effects is <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-92.gif"/></noscript></span></span> finally, in the last stage, it is assumed that the prior distribution can be expressed as <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-93.gif"/></noscript></span></span> with a flat prior for <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-94.gif"/></noscript></span></span> and with a scaled inverse Chi-squared prior distribution for the error variance <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-95.gif"/></noscript></span></span> with scale factor <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-96.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-97.gif"/></noscript></span></span> degrees of freedom, while the prior for <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-98.gif"/></noscript></span></span> is scaled inverse Chi-squared distribution <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-99.gif"/></noscript></span></span> with scale factor <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-100.gif"/></noscript></span></span> and degrees of freedom <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-101.gif"/></noscript></span></span></p><p id="p-23">The hyperparameters were set using the rules given by <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Pérez-Rodríguez and de los Campos (2014)</a>. In this study, we assumed default values of <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-102.gif"/></noscript></span></span> with the intention of avoiding infinite variance values. We also assumed that the model explained 50% of the phenotypic variance; then <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-103.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-104.gif"/></noscript></span></span> More details on the use of the BGLR can be found in <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Pérez-Rodríguez and de los Campos (2014)</a>.</p><p id="p-24">Multi-environment models (2) and (3) were fitted using the Multi-Trait Model (MTM) software of <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos and Grüneberg (2016)</a> that uses a Bayesian approach, assuming the <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-105.gif"/></noscript></span></span> are the same in all the environments and considering that, at the first level, the conditional distribution of the data can be modeled by a multivariate normal distribution <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-106.gif"/></noscript></span></span> At the second level, the prior distributions for u and f are multivariate normal with mean vector zero and variance–covariance matrices <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-107.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-108.gif"/></noscript></span></span> respectively, that is, <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-109.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-110.gif"/></noscript></span></span> At the third level, a flat prior distribution for the intercepts of each environment is used, and the prior distributions of <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-112.gif"/></noscript></span></span> are inverse Whishart <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-113.gif"/></noscript></span></span> where the scale matrix <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-114.gif"/></noscript></span></span> is an identity matrix of order <em>m</em> (number of environments) and the degrees of freedom <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-115.gif"/></noscript></span></span> For the prior distribution of the elements of <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-116.gif"/></noscript></span></span> of the diagonal of <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-117.gif"/></noscript></span></span> we used a scaled inverse Chi-squared distribution with the hyperparameters’ degree of freedom and a scaled factor equal to 1.</p></div></div><div id="sec-7" class="subsection"><h3>Software</h3><p id="p-25">Both packages, BGLR and MTM, fit the models with Markov Chain Monte Carlo (MCMC) using the Gibbs sampler with 30,000 iterations, with a burn-in of 5000 and a thinning of five, so that 5000 samples were used for inference. Convergence and diagnostic tests were performed. The Gelman-Rubin convergence tests for all parameters of the three models were satisfactory, using lag-5 thinning results in low autocorrelations in each of the three models. The Raftery–Lewis test suggested a small burn-in between 10,000 and 20,000 iterations for the five data sets used.</p><p id="p-26">The <em>R</em> codes with a brief description for fitting multi-environment model (3) using the MTM package of <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">de los Campos and Grüneberg (2016)</a> are given in Appendix A.</p></div><div id="sec-8" class="subsection"><h3>Assessing prediction ability</h3><p id="p-27">Prediction ability was assessed using 50 TRN-TST (TRN = training and TST = testing) random partitions; we used this approach because it provides higher precision in the predictive estimates than the framework that uses different numbers of folds. For single-environment model (1), 50 random partitions were formed with 70% of the observations in the training set and 30% of the observations in the testing set.</p><p id="p-28">For multi-environment models (2) and (3), we simulated the prediction problem that assumes that 70% of the individuals were observed in some environments but not in others (CV2, <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em>, 2012</a>). We used the procedure of <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> to assign individuals to the training and testing sets. We formed TRN sets with 70% of the <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-118.gif"/></noscript></span></span> observations and TST sets with 30% of the <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-119.gif"/></noscript></span></span> observations to be predicted (their phenotypic values were not observed and appear as missing).</p><p id="p-29">In each random partition, Pearson’s correlations between the predicted and observed values for each environment were computed; these are considered the prediction accuracies of those models, and thus the average correlation for all random partitions and their standard deviation are reported. The variance components of the three models using the full data are also reported.</p><p id="p-30">When random cross-validation partitions simulated the prediction of a portion of individuals that represents newly developed lines not observed in any environment (random cross-validation 1, CV1, <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>), it is possible that <em><strong>f</strong></em> (of model (3)) could account for part of the random error. However, in this study, we observed all the individuals in at least one environment and predicted other individuals that were not observed in some environments (random CV2, <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>); therefore, under CV2 random cross-validation, <em><strong>f</strong></em> is predictable.</p></div><div id="sec-9" class="subsection"><h3>Experimental data sets</h3><p id="p-31">In this study, we used five data sets that have been used in different studies. Wheat data set 1 was used by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> (2010)</a> and <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a>, maize data set 2 was employed in the studies of <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> (2013)</a> and <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a>, and wheat data sets 3–5 were analyzed by <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a>. Brief descriptions of the phenotypic and marker data sets are given below.</p><div id="sec-10" class="subsection"><h4>Wheat data set 1:</h4><p id="p-32">This data set, from CIMMYT’s Global Wheat Program, was used by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> (2010)</a> and <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> and includes 599 wheat lines derived from 25 yr (1979–2005) of Elite Spring Wheat Yield Trials (ESWYT). The environments represented in these trials were grouped into four basic agroclimatic regions (mega-environments). The phenotypic trait considered here was grain yield (GY) of the 599 wheat lines evaluated in each of the four mega-environments. The 599 wheat lines were genotyped using 1447 Diversity Array Technology (DArT) markers generated by Triticarte Pty. Ltd. (Canberra, Australia; <a href="http://www.triticarte.com.au">http://www.triticarte.com.au</a>). Markers with a minor allele frequency (MAF) &lt; 0.05 were removed, and missing genotypes were imputed using samples from the marginal distribution of marker genotypes. The number of DArT markers after edition was 1279.</p></div><div id="sec-11" class="subsection"><h4>Maize data set 2:</h4><p id="p-33">This data set was first used by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> (2013)</a> and then by <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a>; it includes a total of 504 double-haploid (DH) maize lines obtained by crossing and backcrossing eight parents that formed 10 full-sib (backcrosses) and six sib families. Each DH line was crossed to an elite single-cross hybrid of the opposite heterotic group to produce 504 testcrosses. The trait analyzed in this study was GY (kg/hectare) in three optimum rain-fed trials. The field experimental design in each of the three environments was an α-lattice incomplete block design with two replicates. Data were preadjusted using estimates of incomplete blocks nested in replicates.</p><p id="p-34">The initial total of 681,257 genotyping-by-sequencing (GBS) markers had a percentage of missing cells per chromosome ranging from 51.3 to 52.8%; after editing, this percentage decreased to around 43–44% of the total number of cells. Around 20% of cells were missing in the edited GBS information used for prediction after imputation. After filtering markers for MAF, a total of 158,281 GBS were used for prediction.</p></div><div id="sec-12" class="subsection"><h4>Wheat data sets 3–5:</h4><p id="p-35">These three data sets were described and used by <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> for proposing a marker × environment interaction model. The phenotypic data consisted of adjusted GY (tonnes/hectare) records collected during three evaluation cycles of different inbred lines evaluated in different environments. All trials were established using an α-lattice design with three replicates in each environment at CIMMYT’s main wheat breeding station at Cd. Obregon, Mexico. The environments were three irrigation regimes (moderate drought stress, optimal irrigation, and drought stress), two planting systems (bed and flat planting), and two different planting dates (normal and late). The phenotype used in the analysis was the Best Linear Unbiased Estimate (BLUE) of GY obtained from a linear model applied to the α-lattice design of each cycle-environment combination.</p><p id="p-36">Wheat data set 3 had 693 wheat lines evaluated in four environments, wheat data set 4 included 670 wheat lines evaluated in four environments, and wheat data set 5 had 807 wheat lines evaluated in five environments. Genotypes were derived using GBS technology, and markers with a MAF &lt; 0.05 were removed. All markers had a high incidence of uncalled genotypes, so we applied thresholds for incidence of missing values and focused on maintaining relatively large and similar numbers of markers per data set. After editing the missing markers, we had a total of 15,744 GBS markers for analyzing wheat data sets 3 and 4, and 14,217 GBS markers available for analyzing wheat data set 5.</p></div></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-37">Phenotypic and marker data for the five data sets can be downloaded from <a href="http://hdl.handle.net/11529/10710">http://hdl.handle.net/11529/10710</a>.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><p id="p-38">In the following sections, we present prediction accuracies for each data set, and describe two main comparisons: (1) method GBLUP <em>vs.</em> method GK for models (1)–(3); and (2) model (1) <em>vs.</em> model (2) and model (3) <em>vs.</em> model (2) for methods GBLUP and GK.</p><div id="sec-15" class="subsection"><h3>Wheat data set 1</h3><p id="p-39">Results showed increased prediction ability for models (1) and (2) for GK over GBLUP ranging from 12 to 16% for E1, E3, and E4. Also, model (3) showed 12 and 9% increases in prediction ability of GK over GBLUP for E1 and E4, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), whereas the percent difference of GK model (3) <em>vs.</em> GBLUP model (3) was only −1 and 1% for E2 and E3, respectively.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470175/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470175/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Percent change in prediction accuracy of GK <em>vs.</em> GBLUP for each of the three models (1)–(3), prediction accuracy of model (2) <em>vs.</em> model (1) for GK and GBLUP, and prediction accuracy of model (3) <em>vs.</em> model (2) for GK and GBLUP for each environment in each data set</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">Empirical phenotypic correlations of zero or negative values between E1 and all the other environments (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) were found, whereas E2–E4 were positively (moderately to highly) associated among themselves. <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows the average prediction ability of each of the four environments given by the three models for linear kernel (GBLUP) and nonlinear kernel (GK). The three GK models had higher prediction ability than the corresponding three GBLUP models for E1, E3, and E4, whereas the best prediction model for E2 was GBLUP model (3). Results for this data set indicate a relatively important level of G × E, basically caused by the differential response of individuals in E1 compared with their responses in the other environments.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470193/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470193/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean prediction accuracies for the different environments of wheat data set 1, maize data set 2, and wheat data set 3 for GBLUP and GK methods, and three models including a single-environment (model (1)) and two multi-environment models (models (2) and (3))</span><div class="sb-div caption-clear"></div></div></div><p id="p-46">Differences in the prediction ability of GBLUP model (2) and GBLUP model (1) were 2, 34, 60, and 12% for E1–E4, respectively; for GK these differences were 0, 44, 62, and 9% for E1–E4, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). GBLUP model (3) was 6, 13, 17, and 5% more accurate than GBLUP model (2) for E1–E4, respectively. GK model (3) was 5, 4, 2, and 3% more accurate than GK model (2) for E1–E4, respectively. In summary, for all environments in wheat data set 1, model (3) had higher prediction ability than models (2) and (1) for GK and GBLUP. As for the methods, GK was better than GBLUP for all three models and environments, except E2.</p><p id="p-47">Variance within environments (diagonal) and covariances between environments (off diagonal) were higher for <em><strong>f</strong></em> <strong>(</strong>expressed in <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-120.gif"/></noscript></span></span>) for GBLUP than for GK (Appendix <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table B1</a>), except for cases involving E1; however, the opposite is true for <em><strong>u</strong></em> <strong>(</strong>expressed in <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-121.gif"/></noscript></span></span>), where the absolute variance–covariance values and correlations for GK were larger than those for GBLUP, and therefore reflected in the increases in the prediction ability of the models and methods. Also, diagonal residuals estimates were smaller in GK than in GBLUP.</p></div><div id="sec-16" class="subsection"><h3>Maize data set 2</h3><p id="p-48">Higher prediction ability of GK over GBLUP for models (1)–(3) ranged from 1 to 12% for E1–E3; the advantage of GK over GBLUP was lower in model (3) (3, 1, and 10% for E1–E3, respectively) than the advantage of model (2) <em>vs.</em> model (1) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Comparing model (2) <em>vs.</em> model (1), the differences were similar for GBLUP and GK (8, 12, and 2% for GBLUP and 10, 7, and 2% for GK). There were no differences between model (3) and model (2) for GK and only small differences for GBLUP (3, 1, and 2% for E1–E3, respectively). Appendix <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table B2</a> shows that the covariance between environments in <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-122.gif"/></noscript></span></span> was close to zero due to the low contribution of random component <em><strong>f</strong></em> for both the GK and GBLUP methods.</p><p id="p-49">The empirical phenotypic correlations between the three environments in the maize data set showed moderate positive values (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), with all the elements of covariance in matrices <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-123.gif"/></noscript></span></span> being positive or zero (Appendix <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table B2</a>). The elements of <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-124.gif"/></noscript></span></span> for the GK method were all larger than those for the GBLUP method, and the opposite occurred with the elements of variance–covariance matrix <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-125.gif"/></noscript></span></span> for GBLUP <em>vs.</em> GK and the diagonal values of residual matrix Σ (Appendix <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table B2</a>). The low values of <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-126.gif"/></noscript></span></span> for the GK methods produced a small increase in prediction ability of model (3) over model (2) (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), although GK model (3) always gave the best predictors for the three environments (E1 = 0.645, E2 = 0.582, and E3 = 0.578) and was slightly superior to GK model (2). In general, results for this data set indicate that the prediction ability of the GK method was always superior to that of the GBLUP method, and model (3) was slightly better than model (2) and clearly superior to model (1).</p></div><div id="sec-17" class="subsection"><h3>Wheat data set 3</h3><p id="p-50">GK models (1)–(3) were better predictors than GBLUP models (1)–(3) for the four environments except for E2 (GBLUP model (3)) and E4 (GK model (1)) (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). For E1 and E2, the prediction ability of model (2) over model (1) was about 14% higher, whereas for E3 it was 12% and for E4 it was 2% (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). An almost negligible increase in prediction ability (2) was observed when comparing model (3) <em>vs.</em> model (2) for GBLUP and GK for predicting individuals in all the environments (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-51">GK model (3) was the best predictor of E1 and E3, GBLUP model (3) was the best predictor of E2, and single-environment GK model (1) was the best predictor of E4 (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>). Similar to the results for maize data set 2, the very low values of the elements of matrix <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-127.gif"/></noscript></span></span> (Appendix <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table B3</a>) for the GK and GBLUP methods produced modest to negligible increases in prediction ability of GK model (3) and GBLUP model (3) over GK model (2) and GBLUP model (2) (from 0 to 2%, as indicated in the last two columns of <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-18" class="subsection"><h3>Wheat data set 4</h3><p id="p-52">The four environments included in this set of trials had a relatively low empirical phenotypic correlation (especially E1 <em>vs.</em> E3, with −0.054), except E2 and E4 (0.414) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). This indicates a relatively important level of G × E and therefore increases in prediction ability when modeling interaction, especially when comparing single-environment model (1) <em>vs.</em> multi-environment models (2) and (3) for the GK and GBLUP methods. For GBLUP, model (2) was a better predictor than model (1) for E1 (0.03 increase), E2 (0.10 increase), E3 (0.078 increase), and for E4 (0.100 increase), with an average increase of 17% (6, 25, 15, and 23%, respectively, <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). This superiority of model (2) over model (1) for all environments increased further in the GK, where it gave an average increase in prediction ability of 32% over the GBLUP (27, 46, 17, and 37%, respectively) (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470039/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470039/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Mean prediction accuracies for the different environments of wheat data sets 4 and 5 for GBLUP and GK methods, and three models including a single-environment (model (1)) and two multi-environment models (models (2) and (3))</span><div class="sb-div caption-clear"></div></div></div><p id="p-56">The increase in prediction ability of model (3) over model (2) was important for GBLUP but not for GK; overall, GBLUP gave an average increase in prediction ability of model (3) over model (2) of 12%, whereas for GK this increase was, on average, 0.75% (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). Although the increase in prediction ability of model (3) over model (2) in GK was marginal, overall, GK model (3) was slightly superior to GBLUP model (3) for E1 (0.601 <em>vs.</em> 0.616; <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) and E3 (0.609 <em>vs.</em> 0.613; <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>); GBLUP model (3) was slightly better than GK model (3) for predicting E4 (0.611 <em>vs.</em> 0.607; <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) and they had similar accuracy for predicting E2 (0.588 <em>vs.</em> 0.587).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470027/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470027/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Comparison of prediction accuracy of multi-environment GBLUP and GK model (3) with various other models published in refereed journals for the five data sets utilized in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-58">Appendix <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table B4</a> shows that GBLUP model (3) could not capture sufficient variability associated with random component <em><strong>u</strong></em> reflected in the within (diagonal) and between environment (off diagonal) variability of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-128.gif"/></noscript></span></span> Therefore, GBLUP model (3) with <em><strong>f</strong></em> explained more of this variability, and this is reflected in the better prediction ability of GBLUP model (3) for E2 and E4. In contrast, GK model (3) explained most of the within and between environmental variance reflected in the large values of the elements of <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-129.gif"/></noscript></span></span> Therefore, <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-130.gif"/></noscript></span></span> could not explain much; thus, the predictions of GK model (3) are similar to (although slightly lower than) those of GK model (2).</p></div><div id="sec-19" class="subsection"><h3>Wheat data set 5</h3><p id="p-59">The gains in prediction ability of GK over GBLUP were consistent across models, ranging from 0 to 13% (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>). For the GBLUP method, the gains in prediction ability of model (2) over model (1) were very modest (for E1–E3), except for E4 and E5 (with high empirical phenotypic correlations of 0.546); gains in prediction ability of model (3) over model (2) were almost negligible, except for E4 and E5 (3%, <a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-60">The superior prediction ability of the three GK models over the GBLUP models is clearly shown in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>. Also, in contrast to GBLUP, the better prediction ability of GK model (2) over GK model (1) is clear for all the environments; interestingly, this increase in prediction ability due to adding interaction matrix <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-131.gif"/></noscript></span></span> to model (2) with respect to model (1) was not reflected when adding the extra variance–covariance matrix <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-132.gif"/></noscript></span></span> to model (3) with respect to model (2) (Appendix <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table B5</a>).</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-61">GBLUP and GK models (1)–(3), which were proposed, described and used in this study, are flexible and can be used not only with genomic information but also with pedigree information. We performed preliminary analyses with models (1)–(3) on wheat data set 1 using only pedigree information, but the prediction ability was not higher than any of the correlations obtained using the genomic relationship matrix (data not shown).</p><p id="p-62">Models (2) and (3) proposed in this paper jointly estimate the genetic and the genotype × environments interaction effects. The proposed models are more parsimonious than those that explicitly separate and estimate genotype and environments affects from their interaction. In general, models that jointly estimated the main effects of genotypes and genotype × environment are preferred to those that do it separately because researchers are interested in examining the predicted values of pure precommercial cultivars and single crosses as well as their interaction and stability with environments.</p><p id="p-63">Below we discuss some of the advantages and limitations of these G × E genomic prediction models (2) and (3).</p><div id="sec-21" class="subsection"><h3>Multi-environment model (2)</h3><p id="p-64">When fitting model (2), estimations of the off-diagonal values of <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-133.gif"/></noscript></span></span> can be positive, close to zero, zero, or negative. This is a more flexible model than the multi-environment model of <a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> with a linear kernel or the multi-environment model of <a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> with a nonlinear GK. Both propositions impose the restriction that the correlation between environments is positive; therefore, prediction ability of lines in environments with negative or zero correlations is low.</p><p id="p-65">For model (2), the correlation between any two environments from the standardized data are equal to the sum <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-134.gif"/></noscript></span></span> thus, when the correlations between environments are close to zero, matrix <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-135.gif"/></noscript></span></span> tends to be diagonal so that model (2) will fit each environment almost independently from the other environments; this will produce prediction accuracies similar to those obtained by single-environment model (1), as evidenced in our results. When the correlation between environments is positive (or negative) and intermediate to high, matrix <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-136.gif"/></noscript></span></span> has positive (or negative) values in its off-diagonal; this allows borrowing information from one environment to predict other environments with positive (or negative) correlations, such that the linear or nonlinear kernels will increase the prediction ability of the lines in those environments. Therefore, the diagonal of matrix <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-137.gif"/></noscript></span></span> influences the prediction ability in a specific environment and the off-diagonal values of matrix <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-138.gif"/></noscript></span></span> affect the exchange of information between environments. According to <a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a>, in general, nonlinear kernel GK had better prediction ability than linear kernel GBLUP. These results were generally found in model (2) as well, because GK explained the variance within and between environments better than GBLUP, and this is reflected in the values of matrix <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-139.gif"/></noscript></span></span> (Appendix <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table B1</a>, <a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table B2</a>, <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table B3</a>, <a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table B4</a>, and <a id="xref-table-wrap-9-2" class="xref-table" href="#T9">Table B5</a>).</p><p id="p-66"><a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1</a> shows that the differences when comparing model (2) <em>vs.</em> model (1) are close in methods GK and GBLUP (<em>i.e.</em>, maize data set 2, wheat data set 3, and wheat data set 5); model (3) with GBLUP and with GK had negligible gains in prediction ability over model (2) with GBLUP and with GK. In contrast, when there are differences in models (2) and (1) in GK and GBLUP (<em>i.e.</em>, wheat data set 1 and wheat data set 4), GBLUP model (3) substantially increases prediction ability with respect to GBLUP model (2), but this does not seem to be the case for GK.</p></div><div id="sec-22" class="subsection"><h3>Multi-environment model (3)</h3><p id="p-67">Model (2) explained only part of the variability, whereas multi-environment model (3) incorporated a random effect <em><strong>f</strong></em> that attempts to explain a portion of the genotypic variance that is not explained by <em><strong>u</strong></em> and therefore has the potential to further capture that variability, which will improve prediction ability. The reaction norm model of <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> (2014)</a> applies this principle when adding a genomic component to the phenotypic component or adding ECs to explain environmental variability. We did not add any ECs to our model; therefore, matrix <em><strong>f</strong></em> will have a predictive effect only when lines are predicted in one environment using information from other correlated environments (random cross-validation scheme CV2). Under the random cross-validation scheme, where certain lines were not observed in any of the environments (cross-validation scheme CV1), there is no borrowing of information from the lines in the training set to predict other lines not observed in any of the environments (testing set); therefore, matrix <em><strong>f</strong></em> is not predictable and the prediction ability for one environment will be the same as that obtained by single-environment model (1) (<a id="xref-ref-18-10" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> 2015</a>).</p><p id="p-68">Results from five data sets show that the increase in prediction ability of model (3) over model (2) is a function of the magnitude of the absolute values of the variance–covariance between environments <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-140.gif"/></noscript></span></span> and the method used (linear or nonlinear kernel). In general, the increases in prediction ability of model (3) over model (2) are with GBLUP because, as mentioned, model (2) explained only part of the genotypic variance; on the other hand, the increases in prediction ability of GK model (3) with respect to GK model (2) are smaller than those observed in the GBLUP because the nonlinear kernel with model (2) takes most of the variability and does not leave much variability to be explained by the covariance between environments <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-141.gif"/></noscript></span></span> Model (3) adds a random effect <em><strong>f</strong></em> representing part of the interaction between genetic factors environment that were not captured by <strong><em>u</em></strong>; when used with the GK, part of the small cryptic variations represented by the small epistatic effect might be included in <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-142.gif"/></noscript></span></span></p></div><div id="sec-23" class="subsection"><h3>Comparing prediction ability of multi-environment model (3) with other multi-environment genomic G × E models in the literature</h3><p id="p-69">In this section, we compare results obtained in this study with multi-environment model (3) using methods GBLUP and GK with those obtained by other models and methods for the same data set and published in other articles (<a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-11-13" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> 2016</a>; <a id="xref-ref-18-11" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> 2015</a>). It should be pointed out that this comparison of results is not completely objective because different random partitions and different numbers of partitions were performed in the different studies.</p><div id="sec-24" class="subsection"><h4>Wheat data set 1:</h4><p id="p-70">The first researchers to study the prediction ability of genomic G × E models were <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> (2012)</a> using multivariate mixed linear models for the variance–covariance matrix of the G × E by means of the parsimonious factor analytic (FA) structure and a diagonal matrix for the variance of error. Clearly, for E2 and E3, correlations computed by fitting the FA model were lower than those computed by GBLUP and GK model (3), which incorporates the random effect of <strong><em>f</em></strong> (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Environment 1, which is negatively correlated with all the other environments, was better predicted by FA (0.553) than by GK model (3) (0.543), while GK model (3) predicted E4 better (0.525) than FA (0.510).</p><p id="p-71">On the other hand, <a id="xref-ref-11-14" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> used a GK based on the marker × environment interaction model of <a id="xref-ref-18-12" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> and decomposed the interaction into two components: the main effects of markers across all the environments and the specific effects of markers in each environment. The reason for these differences is that the model of <a id="xref-ref-11-15" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> assumes positive correlations between environments; when there is a negative correlation between environments, its predictive capacity declines, as happened with the relatively low prediction ability of E1 (0.458). GK model (3) (and also GBLUP model (3)) is more flexible, admitting any correlation (positive, zero, or negative) between environments, and therefore predicted all four environments (E1–E4) better than the EB-G × E and FA models (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>).</p></div><div id="sec-25" class="subsection"><h4>Maize data set 2:</h4><p id="p-72">This data set was used by <a id="xref-ref-11-16" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016)</a> for comparing the GBLUP marker × environment interaction of <a id="xref-ref-18-13" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> with the proposed GK marker × environment interaction. Moderate but consistent increases in prediction ability for each environment were achieved by GK model (3) over model EB-G × E (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). Also, GK model (2) had consistently higher prediction ability than EB-G × E.</p></div><div id="sec-26" class="subsection"><h4>Wheat data sets 3–5:</h4><p id="p-73">These three data sets were used by <a id="xref-ref-18-14" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a> to fit the marker × environment interaction GBLUP-ME. GBLUP model (3) and GK model (3) showed consistently higher prediction ability than model GBLUP-ME, except in E4 of wheat data set 3, where GBLUP-ME had a prediction ability of 0.516, which was higher than the accuracy of GBLUP model (3) and GK model (3).</p></div></div><div id="sec-27" class="subsection"><h3>Some limitations of the proposed models</h3><p id="p-74">In this study, we used five data sets with the same number of individuals in all the environments. This does not seem to be a limitation when the main idea is to predict the same number of individuals in all environments. However, when the total number of individuals is different in different environments, then the within-environment <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-143.gif"/></noscript></span></span> is different in each environment and the MTM software cannot be used directly. Fitting models (2)–(3) in this case is more complicated because, although it is possible to fit the models with the mode of the integrated likelihood, this requires much computing time. Fitting models for a large number of environments, even when the same number of lines are evaluated in each environment, also requires much computing time.</p><p id="p-75">A possible solution for reducing computing time is to reduce the number of parameters to be estimated by assuming that matrices <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/41/embed/mml-math-144.gif"/></noscript></span></span> are proportional to the phenotypic correlation, which does not seem unreasonable if the response data are standardized. However, more research on the use of this simplification is required to establish whether the prediction accuracies thus obtained are similar to those computed using the proposed estimation method.</p></div><div id="sec-28" class="subsection"><h3>Conclusions</h3><p id="p-76">The Bayesian genomic G × E models described, implemented, and used in this study are novel and overcome some of the limitations imposed by previous genomic G × E models. Models (2) and (3) allow an arbitrary genetic covariance structure between environments, because an unstructured covariance matrix was used and its parameters were estimated from the data. These multi-environment models can be implemented using existing software for GS such as MTM. The cross-validation used 50 replicates and predicted lines in environments where they had not been observed using two sources of information: genomic relationships between lines and genetic information between environments. In all five data sets, models (2) and (3) had higher prediction accuracies than single-environment model (1) regardless of the genetic correlation between environments. In general, models (2) and (3) with the nonlinear GK had higher prediction accuracies for the lines unobserved in the environments than those obtained by the linear kernel (GBLUP) method. Under G × E interactions such as those found in the five data sets studied in this article, nonlinear GK models (2) and (3) performed very similarly and had higher prediction accuracies than linear GBLUP models (2) and (3). These models are clearly superior to single-environment genomic model (1) with GBLUP and GK, and their results are also superior to previous results from more restrictive marker × environment models. Prediction accuracies of models (2) and (3) with GK were higher than those obtained by other models and methods.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-122">The authors are grateful to the dedication and high professionalism of all field and lab workers and assistants at CIMMYT Headquarters (Mexico), as well as at CIMMYT outreach offices in several countries, who generated the data analyzed in this study.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX A</h2><p id="p-77">### Example for fitting multi-environment model (3)</p><p id="p-78">### Require MTM package (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">de los Campos and Grüneberg, 2016</a>)</p><p id="p-79">##### Require BGLR package (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">de los Campos and Pérez-Rodríguez, 2014</a>)</p><p id="p-80">##### Requiere function CV2 (<a id="xref-ref-18-15" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em>, 2015</a>)</p><p id="p-81">###Data</p><p id="p-82">library(BGLR)</p><p id="p-83">library(MCMCpack)</p><p id="p-84">data(wheat)</p><p id="p-85">X&lt;-wheat.X</p><p id="p-86">Y&lt;-wheat.Y</p><p id="p-87">X&lt;-scale(X,center=TRUE,scale=TRUE)</p><p id="p-88">Y&lt;-scale(Y,center=TRUE,scale=TRUE)</p><p id="p-89">env&lt;-c(1,2,3,4); y&lt;-Y[,env]</p><p id="p-90">G&lt;-X%*%t(X)/ncol(X)</p><p id="p-91">In&lt;-diag(1,nrow(Y),nrow(Y))</p><p id="p-92">set.seed(12345)</p><p id="p-93">##Simulation of missing data CV2, see <a id="xref-ref-18-16" class="xref-bibr" href="#ref-18">López-Cruz <em>et al.</em> (2015)</a></p><p id="p-94">yNA&lt;-CV2(Y,env) # generation of CV2</p><p id="p-95">### Fit model (3) with MTM (see <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">de los Campos and Grüneberg, 2016</a>)</p><p id="p-96">fm &lt;- MTM(</p><p id="p-97">Y = yNA,</p><p id="p-98">K = list(  list( K = G, COV = list( type = ′UN′, df0 = 4, S0 = diag(4) ) ), list(</p><p id="p-99">K = In, COV = list(type = ′UN′, df0 = 4, S0 = diag(4)) )</p><p id="p-100">),</p><p id="p-101">resCov = list( type = ′DIAG′, S0 = rep(1, 4), df0 = rep(1, 4)</p><p id="p-102">), nIter = 10000, burnIn = 1000, thin = 5, saveAt = ′ex1′</p><p id="p-103">)</p><p id="p-104"># ###Extracting the estimated parameters</p><p id="p-105">YHatInt &lt;- fm$YHat </p><p id="p-106">##### Predictive correlation</p><p id="p-107">test&lt;-is.na(yNA)</p><p id="p-108">tst1&lt;-test[,1];tst2&lt;-test[,2];tst3&lt;-test[,3];tst4&lt;-test[,4]</p><p id="p-109">COR.PT1&lt;-cor(YHatInt[tst1,1],y[tst1,1])</p><p id="p-110">COR.PT2&lt;-cor(YHatInt[tst2,2],y[tst2,2])</p><p id="p-111">COR.PT3&lt;-cor(YHatInt[tst3,3],y[tst3,3])</p><p id="p-112">COR.PT4&lt;-cor(YHatInt[tst4,4],y[tst4,4])</p><p id="p-113">#### Extracting estimates of variance parameters</p><p id="p-114">COR.RES&lt;-fm$resCov$R # residual covariance matrix</p><p id="p-115">COR.u&lt;-fm$K[[1]]$G # genetic covariance matrix UE</p><p id="p-116">COR.f&lt;-fm$K[[2]]$G # genetic covariance matrix FE</p></div><div class="section app" id="app-2"><h2 class="">APPENDIX B</h2><div class="section" id="sec-29"><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470070/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470070/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table B1</span> <span class="caption-title">Wheat data set 1</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470093/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470093/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table B2</span> <span class="caption-title">Maize data set 2</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470132/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470132/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table B3</span> <span class="caption-title">Wheat data set 3</span><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470124/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470124/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table B4</span> <span class="caption-title">Wheat data set 4</span><div class="sb-div caption-clear"></div></div></div><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470078/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470078/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table B5</span> <span class="caption-title">Wheat data set 5</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-15"><p id="p-123"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-15"><span class="received-label">Received </span>September 15, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-10-21"><span class="accepted-label">Accepted </span>October 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Cuevas <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonilla</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multi-task Gaussian process prediction.</span> <abbr class="cit-jnl-abbrev">Adv. Neural Inf. Process. Syst.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BNeural%2BInf.%2BProcess.%2BSyst.%26rft.volume%253D20%26rft.spage%253D153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.2" data-doi="10.2135/cropsci2007.11.0632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Using factor analytic models for joining environments and genotypes without crossover genotype × environment interactions.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBurgueno%26rft.auinit1%253DJ.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D1291%26rft.epage%253D1305%26rft.atitle%253DUsing%2BFactor%2BAnalytic%2BModels%2Bfor%2BJoining%2BEnvironments%2Band%2BGenotypes%2Bwithout%2BCrossover%2BGenotype%2Bx%2BEnvironment%2BInteraction%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.11.0632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.11.0632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.3" data-doi="10.2135/cropsci2010.07.0403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotes</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">San Vicente</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prediction assessment of linear mixed models for multivariate trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">944</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D944%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.07.0403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.07.0403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.4" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.41.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seyedsadr</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Statistical test and estimators of multiplicative models for genotype-by-environment interaction</span>, pp. <span class="cit-fpage">199</span>–<span class="cit-lpage">234</span> in <span class="cit-source">Genotype-by-Environment Interaction</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Gauch</span>  <span class="cit-name-given-names">H. G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>..</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Studying crossover genotype× environment interaction using linear-bilinear models and mixed models.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">362</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D9%26rft.spage%253D362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.7" data-doi="10.2135/cropsci2005.11-0427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Modeling genotype × environment interaction using additive genetic covariances of relatives for predicting breeding values of wheat genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCrossa%26rft.auinit1%253DJ.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D1722%26rft.epage%253D1733%26rft.atitle%253DModeling%2BGenotype%2Bx%2BEnvironment%2BInteraction%2BUsing%2BAdditive%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bfor%2BPredicting%2BBreeding%2BValues%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.11-0427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.11-0427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239054100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.8" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.9" data-doi="10.1534/g3.113.008227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008227%26rft_id%253Dinfo%253Apmid%252F24022750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTkwMyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzcvMS80MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.10" data-doi="10.2135/cropsci2014.11.0789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuberosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extending the marker × environment interaction model for genomic-enabled prediction and genome-wide association analyses in durum wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.11.0789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.11.0789&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.11" data-doi="10.3835/plantgenome2016.05.0047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soberanis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of genotype × environment interaction kernel regression models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.05.0047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.05.0047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-7.1.41.12"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2014 Bayesian Generalized Linear Regression. R package version 1.0.4. Available at: <a href="http://CRAN.R-project.org/package=BGLR">http://CRAN.R-project.org/package=BGLR</a>. Accessed: April 20, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-7.1.41.13"><div class="cit-metadata unstructured">de los Campos, G., and A. Grüneberg, 2016 MTM (Multiple-Trait Model) package. Available at: <a href="http://quantgen.github.io/MTM/vignette.html">http://quantgen.github.io/MTM/vignette.html</a>. Accessed May 3rd, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.14" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F41.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.1.41.15" data-doi="10.1017/S0021859600003592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackensie</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1923</span> <span class="cit-article-title">Studies in crop variation II. The manurial response of different potato varieties.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">03</span>): <span class="cit-fpage">311</span>–<span class="cit-lpage">320</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0021859600003592</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D13%26rft.spage%253D311%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859600003592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859600003592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.16" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F41.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.41.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Linear and generalized linear mixed models and their applications</span>, pp. <span class="cit-fpage">7</span>. <span class="cit-source">Springer Series in Statistics</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.18" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft_id%253Dinfo%253Apmid%252F25660166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC81NjkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.19" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy83LzEvNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez- Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection of the bandwidth parameter in a Bayesian Kernel regression model for genomic-enabled prediction.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">512</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D5%26rft.spage%253D512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.21" data-doi="10.1534/g3.112.003665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1595%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.003665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.22" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNy8xLzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.23" data-doi="10.1007/s001220050885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Empirical best linear unbiased prediction in cultivar trials using factor analytic variance covariance structure.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075377300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-7.1.41.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>. <span class="cit-pub-date">2001</span> <span class="cit-article-title">Exploring variety-environment data using random effects AMMI models with adjustments for spatial field trend: part 1: theory,</span> pp. <span class="cit-fpage">323</span>–<span class="cit-lpage">335</span> in <span class="cit-source"><em>Quantitative Genetics</em>, <em>Genomics and Plant Breeding</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M. S.</span></span>. <span class="cit-publ-name">CABI Publishing</span>, <span class="cit-publ-loc">Wallingford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.25" data-doi="10.1017/S0021859605005587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The analysis of crop cultivar breeding and evaluation trials: an overview of current mixed model approaches.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D143%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859605005587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859605005587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull Bull</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.27" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F41.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.41.28" data-doi="10.1017/S0021859600050978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochran</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">The analysis of groups of experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D28%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859600050978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859600050978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/41.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/31" title="Small RNA Sequencing in Cells and Exosomes Identifies eQTLs and 14q32 as a Region of Active Export" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/55" title="Assortative Mating and Linkage Disequilibrium" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/41" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w3vw-wrkH_v4dCyPOMZ_wqBGFAfEsVFcdSoI-j-zoQE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/41" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/41" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/41" class="active" data-icon-position="" data-hide-link-title="0">Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460531" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qcXL0kjxJLHF5r4P0Y1CgqhVGwGpJTR4J-utZnvYXTE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/41.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/41" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wkH1zhITSvXv5YIAt83hE550IeQ8pXli3g-YcvLgTK0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/41" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/41.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460531" id="node460531--2" data-pisa="ggg;7/1/41" data-pisa-master="ggg;g3.116.035584" data-apath="/ggg/7/1/41.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Genomic Prediction with Genotype <strong>×</strong> Environment Interaction Kernel Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="1">José Crossa</span>, <span class="highwire-citation-author" data-delta="2">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="3">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span> and <span class="highwire-citation-author" data-delta="5">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">41-53;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035584 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460531--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, C. P. 56230 Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/41" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/41&amp;title=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/41&amp;title=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/41&amp;title=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/41&amp;text=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/41&amp;title=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/41&amp;t=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/41&amp;title=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/41&amp;title=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460531/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460531" id="node460531--3" data-pisa="ggg;7/1/41" data-pisa-master="ggg;g3.116.035584" data-apath="/ggg/7/1/41.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Genomic Prediction with Genotype <strong>×</strong> Environment Interaction Kernel Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="1">José Crossa</span>, <span class="highwire-citation-author" data-delta="2">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="3">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span> and <span class="highwire-citation-author" data-delta="5">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">41-53;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035584 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460531--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México D. F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, C. P. 56230 Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=41&amp;rft.epage=53&amp;rft.atitle=Bayesian%20Genomic%20Prediction%20with%20Genotype%20%C3%97%20Environment%20Interaction%20Kernel%20Models&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cuevas%2C+Jaime&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=de+los+Campos%2C+Gustavo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460531/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460531/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460531/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460531/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460531/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460531/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460531/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460531/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460531/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460531/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460531/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460531/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/41" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX A</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX B</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/41.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/41.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/1/41.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
