<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/1/119" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/1/119.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/1/119.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/1/119.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/1/119/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/1/119.atom" />
<meta name="HW.pisa" content="ggg;7/1/119" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci" />
<meta name="DC.Identifier" content="10.1534/g3.116.036012" />
<meta name="DC.Date" content="2017-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Brandt et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Virginia chicken lines have been divergently selected for juvenile body weight for more than 50 generations. Today, the high- and low-weight lines show a &gt;12-fold difference for the selected trait, 56-d body weight. These lines provide unique opportunities to study the genetic architecture of long-term, single-trait selection. Previously, several quantitative trait loci (QTL) contributing to weight differences between the lines were mapped in an F2-cross between them, and these were later replicated and fine-mapped in a nine-generation advanced intercross of them. Here, we explore the possibility to further increase the fine-mapping resolution of these QTL via a pedigree-based imputation strategy that aims to better capture the genetic diversity in the divergently selected, but outbred, founder lines. The founders of the intercross were high-density genotyped, and then pedigree-based imputation was used to assign genotypes throughout the pedigree. Imputation increased the marker density 20-fold in the selected QTL, providing 6911 markers for the subsequent analysis. Both single-marker association and multi-marker backward-elimination analyses were used to explore regions associated with 56-d body weight. The approach revealed several statistically and population structure independent associations and increased the mapping resolution. Further, most QTL were also found to contain multiple independent associations to markers that were not fixed in the founder populations, implying a complex underlying architecture due to the combined effects of multiple, linked loci perhaps located on independent haplotypes that still segregate in the selected lines." />
<meta name="DC.Contributor" content="Monika Brandt" />
<meta name="DC.Contributor" content="Muhammad Ahsan" />
<meta name="DC.Contributor" content="Christa F. Honaker" />
<meta name="DC.Contributor" content="Paul B. Siegel" />
<meta name="DC.Contributor" content="Örjan Carlborg" />
<meta name="article:published_time" content="2017-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Virginia chicken lines have been divergently selected for juvenile body weight for more than 50 generations. Today, the high- and low-weight lines show a &amp;gt;12-fold difference for the selected trait, 56-d body weight. These lines provide unique opportunities to study the genetic architecture of long-term, single-trait selection. Previously, several quantitative trait loci (QTL) contributing to weight differences between the lines were mapped in an F&lt;sub&gt;2&lt;/sub&gt;-cross between them, and these were later replicated and fine-mapped in a nine-generation advanced intercross of them. Here, we explore the possibility to further increase the fine-mapping resolution of these QTL via a pedigree-based imputation strategy that aims to better capture the genetic diversity in the divergently selected, but outbred, founder lines. The founders of the intercross were high-density genotyped, and then pedigree-based imputation was used to assign genotypes throughout the pedigree. Imputation increased the marker density 20-fold in the selected QTL, providing 6911 markers for the subsequent analysis. Both single-marker association and multi-marker backward-elimination analyses were used to explore regions associated with 56-d body weight. The approach revealed several statistically and population structure independent associations and increased the mapping resolution. Further, most QTL were also found to contain multiple independent associations to markers that were not fixed in the founder populations, implying a complex underlying architecture due to the combined effects of multiple, linked loci perhaps located on independent haplotypes that still segregate in the selected lines.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="ggg;7/1/119" />
<meta name="citation_id" content="7/1/119" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/1/119" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/1/119.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/1/119.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/1/119.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036012" />
<meta name="citation_pmid" content="27799342" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="119" />
<meta name="citation_lastpage" content="128" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Monika Brandt" />
<meta name="citation_author_institution" content="Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden" />
<meta name="citation_author" content="Muhammad Ahsan" />
<meta name="citation_author_institution" content="Division of Computational Genetics, Department of Clinical Sciences, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden" />
<meta name="citation_author" content="Christa F. Honaker" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061" />
<meta name="citation_author" content="Paul B. Siegel" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061" />
<meta name="citation_author" content="Örjan Carlborg" />
<meta name="citation_author_institution" content="Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden" />
<meta name="citation_author_email" content="orjan.carlborg@imbim.uu.se" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2722-5264" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=F. Abramovich;citation_author=Y. Benjamini;citation_author=D. L. Donoho;citation_author=I. M. Johnstone;citation_title=Special invited lecture: adapting to unknown sparsity by controlling the false discovery rate.;citation_pages=584-653;citation_volume=34;citation_year=2006;citation_doi=10.1214/009053606000000074" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=M. Ahsan;citation_author=X. Li;citation_author=A. E. Lundberg;citation_author=M. Kierczak;citation_author=P. B. Siegel;citation_title=Identification of candidate genes and mutations in QTL regions for chicken growth using bioinformatic analysis of NGS and SNP-chip data.;citation_pages=226;citation_volume=4;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Andersson;citation_author=M. Georges;citation_title=Domestic-animal genomics: deciphering the genetics of complex traits.;citation_pages=202-212;citation_volume=5;citation_year=2004;citation_issue=3;citation_pmid=14970822;citation_doi=10.1038/nrg1294" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. S. Aulchenko;citation_author=S. Ripke;citation_author=A. Isaacs;citation_author=C. M. van Duijn;citation_title=GenABEL: an R package for genome-wide association analysis.;citation_volume=23;citation_year=2007;citation_pmid=17384015;citation_doi=10.1093/bioinformatics/btm108" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=F. Besnier;citation_author=P. Wahlberg;citation_author=L. Rönnegård;citation_author=W. Ek;citation_author=L. Andersson;citation_title=Fine mapping and replication of QTL in outbred chicken advanced intercross lines.;citation_pages=3-3;citation_volume=43;citation_year=2011;citation_issue=1;citation_pmid=21241486;citation_doi=10.1186/1297-9686-43-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. R. Bevova;citation_author=Y. S. Aulchenko;citation_author=S. Aksu;citation_author=U. Renne;citation_author=G. A. Brockmann;citation_title=Chromosome-wise dissection of the genome of the extremely big mouse line DU6i.;citation_volume=172;citation_year=2006;citation_pmid=16157676;citation_doi=10.1534/genetics.104.040196" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Ö. Carlborg;citation_author=L. Jacobsson;citation_author=P. Åhgren;citation_author=P. Siegel;citation_author=L. Andersson;citation_title=Epistasis and the release of genetic variation during long-term selection.;citation_pages=418-420;citation_volume=38;citation_year=2006;citation_issue=4;citation_pmid=16532011;citation_doi=10.1038/ng1761" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Cheng;citation_author=J. E. Lim;citation_author=K. E. Samocha;citation_author=G. Sokoloff;citation_author=M. Abney;citation_title=Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.;citation_volume=185;citation_year=2010;citation_pmid=20439773;citation_doi=10.1534/genetics.110.116863" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=G. R. Wiggans;citation_author=B. J. Hayes;citation_author=J. A. Woolliams;citation_author=M. E. Goddard;citation_title=Imputation of missing genotypes from sparse to high density using long-range phasing.;citation_volume=189;citation_year=2011;citation_pmid=21705746;citation_doi=10.1534/genetics.111.128082" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.;citation_pages=1199-1207;citation_volume=141;citation_year=1995;citation_issue=3;citation_pmid=8582624" />
<meta name="citation_reference" content="citation_journal_title=Poult. Sci.;citation_author=E. A. Dunnington;citation_author=P. B. Siegel;citation_title=Long-term divergent selection for eight-week body weight in White Plymouth Rock chickens.;citation_volume=75;citation_year=1996;citation_pmid=8893291;citation_doi=10.3382/ps.0751168" />
<meta name="citation_reference" content="citation_journal_title=Poult. Sci.;citation_author=E. A. Dunnington;citation_author=C. F. Honaker;citation_author=M. L. McGilliard;citation_author=P. B. Siegel;citation_title=Phenotypic responses of chickens to long-term, bidirectional selection for juvenile body weight–historical perspective.;citation_volume=92;citation_year=2013;citation_pmid=23776258;citation_doi=10.3382/ps.2013-03069" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=W. Ek;citation_author=S. Marklund;citation_author=A. Ragavendran;citation_author=P. Siegel;citation_author=W. Muir;citation_title=Generation of a multi-locus chicken introgression line to study the effects of genetic interactions on metabolic phenotypes in chickens.;citation_pages=29-29;citation_volume=3;citation_year=2012;citation_pmid=22403584" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=Y. Gavrilov;citation_author=Y. Benjamini;citation_author=S. K. Sarkar;citation_title=An adaptive step-down procedure with proven FDR control under independence.;citation_pages=619-629;citation_volume=37;citation_year=2009;citation_doi=10.1214/07-AOS586" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. M. Gray;citation_author=M. D. Parmenter;citation_author=C. A. Hogan;citation_author=I. Ford;citation_author=R. J. Cuthbert;citation_title=Genetics of rapid and extreme size evolution in island mice.;citation_volume=201;citation_year=2015;citation_pmid=26199233;citation_doi=10.1534/genetics.115.177790" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. G. Hill;citation_title=GENETICS: A Century of Corn Selection;citation_pages=683-684;citation_volume=307;citation_year=2005;citation_issue=5710;citation_pmid=15692038;citation_doi=10.1126/science.1105459" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=L. Jacobsson;citation_author=H.-B. Park;citation_author=P. Wahlberg;citation_author=R. Fredriksson;citation_author=M. Perez-Enciso;citation_title=Many QTLs with minor additive effects are associated with a large difference in growth between two selection lines in chickens.;citation_pages=115-125;citation_volume=86;citation_year=2005;citation_issue=2;citation_pmid=16356285;citation_doi=10.1017/S0016672305007767" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. M. Johansson;citation_author=M. E. Pettersson;citation_author=P. B. Siegel;citation_author=Ö. Carlborg;citation_title=Genome-wide effects of long-term divergent selection.;citation_pages=e1001188-e1001188;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21079680;citation_doi=10.1371/journal.pgen.1001188" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. E. Kemper;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Genetic architecture of body size in mammals.;citation_pages=244-244;citation_volume=13;citation_year=2012;citation_issue=4;citation_pmid=22546202;citation_doi=10.1186/gb-2012-13-4-244" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=A. Le Rouzic;citation_author=Ö. Carlborg;citation_title=Evolutionary potential of hidden genetic variation.;citation_pages=33-37;citation_volume=23;citation_year=2008;citation_issue=1;citation_pmid=18079017;citation_doi=10.1016/j.tree.2007.09.014" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=A. Le Rouzic;citation_author=P. B. Siegel;citation_author=Ö. Carlborg;citation_title=Phenotypic evolution from genetic polymorphisms in a radial network architecture.;citation_pages=50-50;citation_volume=5;citation_year=2007;citation_pmid=18001473;citation_doi=10.1186/1741-7007-5-50" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Marklund;citation_author=Ö. Carlborg;citation_title=SNP detection and prediction of variability between chicken lines using genome resequencing of DNA pools.;citation_pages=665-665;citation_volume=11;citation_year=2010;citation_pmid=21108801;citation_doi=10.1186/1471-2164-11-665" />
<meta name="citation_reference" content="citation_journal_title=Poult. Sci.;citation_author=G. C. Márquez;citation_author=P. B. Siegel;citation_author=R. M. Lewis;citation_title=Genetic diversity and population structure in lines of chickens divergently selected for high and low 8-week body weight.;citation_volume=89;citation_year=2010;citation_pmid=21076095;citation_doi=10.3382/ps.2010-01034" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=H.-B. Park;citation_author=L. Jacobsson;citation_author=P. Wahlberg;citation_author=P. B. Siegel;citation_author=L. Andersson;citation_title=QTL analysis of body composition and metabolic traits in an intercross between chicken lines divergently selected for growth.;citation_volume=25;citation_year=2006;citation_pmid=16390876;citation_doi=10.1152/physiolgenomics.00113.2005" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=C. C. Parker;citation_author=R. Cheng;citation_author=G. Sokoloff;citation_author=J. E. Lim;citation_author=A. D. Skol;citation_title=Fine-mapping alleles for body weight in LG/J Ã SM/J F2 and F(34) advanced intercross lines.;citation_pages=563-571;citation_volume=22;citation_year=2011;citation_issue=9-10;citation_pmid=21761260;citation_doi=10.1007/s00335-011-9349-z" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. L. Peirce;citation_author=K. W. Broman;citation_author=L. Lu;citation_author=E. J. Chesler;citation_author=G. Zhou;citation_title=Genome Reshuffling for Advanced Intercross Permutation (GRAIP): simulation and permutation for advanced intercross population analysis.;citation_pages=e1977-e1977;citation_volume=3;citation_year=2008;citation_issue=4;citation_pmid=18431467;citation_doi=10.1371/journal.pone.0001977" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Pettersson;citation_author=F. Besnier;citation_author=P. B. Siegel;citation_author=Ö. Carlborg;citation_title=Replication and explorations of high-order epistasis using a large advanced intercross line pedigree.;citation_pages=e1002180-e1002180;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21814519;citation_doi=10.1371/journal.pgen.1002180" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. E. Pettersson;citation_author=A. M. Johansson;citation_author=P. B. Siegel;citation_author=Ö. Carlborg;citation_title=Dynamics of adaptive alleles in divergently selected body weight lines of chickens.;citation_volume=3;citation_year=2013;citation_pmid=24170737;citation_doi=10.1534/g3.113.008375" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Rönnegård;citation_author=F. Besnier;citation_author=Ö. Carlborg;citation_title=An Improved Method for Quantitative Trait Loci Detection and Identification of Within-Line Segregation in F2 Intercross Designs;citation_pages=2315-2326;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430952;citation_doi=10.1534/genetics.107.083162" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Saatchi;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=D. J. Garrick;citation_title=Large-effect pleiotropic or closely linked QTL segregate within and across ten US cattle breeds.;citation_pages=442;citation_volume=15;citation_year=2014;citation_pmid=24906442" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=Z. Sheng;citation_author=M. E. Pettersson;citation_author=C. F. Honaker;citation_author=P. B. Siegel;citation_author=Ö. Carlborg;citation_title=Standing genetic variation as a major contributor to adaptation in the Virginia chicken lines selection experiment.;citation_pages=219;citation_volume=16;citation_year=2015;citation_pmid=26438066;citation_doi=10.1186/s13059-015-0785-z" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. B. Siegel;citation_title=A DOUBLE SELECTION EXPERIMENT FOR BODY WEIGHT AND BREAST ANGLE AT EIGHT WEEKS OF AGE IN CHICKENS;citation_pages=1313-1319;citation_volume=47;citation_year=1962;citation_issue=10;citation_pmid=13988956" />
<meta name="citation_reference" content="citation_journal_title=Poult. Sci.;citation_author=P. B. Siegel;citation_title=Selection for body weight at eight weeks of age. 1. Short term response and heritabilities.;citation_volume=41;citation_year=1962;citation_doi=10.3382/ps.0410954" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=C. C. Holmes;citation_author=R. Mott;citation_author=J. Flint;citation_title=Mapping in structured populations by resample model averaging.;citation_volume=182;citation_year=2009;citation_pmid=19474203;citation_doi=10.1534/genetics.109.100727" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. Van Goor;citation_author=K. J. Bolek;citation_author=C. M. Ashwell;citation_author=M. E. Persia;citation_author=M. F. Rothschild;citation_title=Identification of quantitative trait loci for body temperature, body weight, breast yield, and digestibility in an advanced intercross line of chickens under heat stress.;citation_pages=96;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Wahlberg;citation_author=Ö. Carlborg;citation_author=M. Foglio;citation_author=X. Tordoir;citation_author=A.-C. Syvänen;citation_title=Genetic analysis of an F(2) intercross between two chicken lines divergently selected for body-weight.;citation_pages=248-248;citation_volume=10;citation_year=2009;citation_pmid=19473501;citation_doi=10.1186/1471-2164-10-248" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=C. K. Yoo;citation_author=H.-B. Park;citation_author=J. B. Lee;citation_author=E. J. Jung;citation_author=B. M. Kim;citation_title=QTL analysis of body weight and carcass body length traits in an F2 intercross between Landrace and Korean native pigs.;citation_pages=589-592;citation_volume=45;citation_year=2014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
<meta name="twitter:description" content="The Virginia chicken lines have been divergently selected for juvenile body weight for more than 50 generations. Today, the high- and low-weight lines show a &gt;12-fold difference for the selected trait, 56-d body weight. These lines provide unique opportunities to study the genetic architecture of long-term, single-trait selection. Previously, several quantitative trait loci (QTL) contributing to weight differences between the lines were mapped in an F2-cross between them, and these were later replicated and fine-mapped in a nine-generation advanced intercross of them. Here, we explore the possibility to further increase the fine-mapping resolution of these QTL via a pedigree-based imputation strategy that aims to better capture the genetic diversity in the divergently selected, but outbred, founder lines. The founders of the intercross were high-density genotyped, and then pedigree-based imputation was used to assign genotypes throughout the pedigree. Imputation increased the marker density 20-fold in the selected QTL, providing 6911 markers for the subsequent analysis. Both single-marker association and multi-marker backward-elimination analyses were used to explore regions associated with 56-d body weight. The approach revealed several statistically and population structure independent associations and increased the mapping resolution. Further, most QTL were also found to contain multiple independent associations to markers that were not fixed in the founder populations, implying a complex underlying architecture due to the combined effects of multiple, linked loci perhaps located on independent haplotypes that still segregate in the selected lines." />
<meta name="og-title" property="og:title" content="Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/1/119" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Virginia chicken lines have been divergently selected for juvenile body weight for more than 50 generations. Today, the high- and low-weight lines show a &gt;12-fold difference for the selected trait, 56-d body weight. These lines provide unique opportunities to study the genetic architecture of long-term, single-trait selection. Previously, several quantitative trait loci (QTL) contributing to weight differences between the lines were mapped in an F2-cross between them, and these were later replicated and fine-mapped in a nine-generation advanced intercross of them. Here, we explore the possibility to further increase the fine-mapping resolution of these QTL via a pedigree-based imputation strategy that aims to better capture the genetic diversity in the divergently selected, but outbred, founder lines. The founders of the intercross were high-density genotyped, and then pedigree-based imputation was used to assign genotypes throughout the pedigree. Imputation increased the marker density 20-fold in the selected QTL, providing 6911 markers for the subsequent analysis. Both single-marker association and multi-marker backward-elimination analyses were used to explore regions associated with 56-d body weight. The approach revealed several statistically and population structure independent associations and increased the mapping resolution. Further, most QTL were also found to contain multiple independent associations to markers that were not fixed in the founder populations, implying a complex underlying architecture due to the combined effects of multiple, linked loci perhaps located on independent haplotypes that still segregate in the selected lines." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1.cover-source.jpg" />
    <title>Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Wdrsj6NdMZk0F-a6nX5lWzIvgA4aWD94hqjVV1vSDqE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460548","apath":"\/ggg\/7\/1\/119.atom","pisa":"ggg;7\/1\/119","ac":{"\/ggg\/7\/1\/119.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/1\/119.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/1\/119"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/1\/119"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/1\/119.full":true,"\/content\/7\/1\/119":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iwRugCRz+fpzx2wD03Qe+8d8kjB4k3\\\\\\\/X\\\\\\\/FAktXG20F1rFQPl\\\\\\\/L8HxxUEB17\\\\\\\/fW0GgKFMFQhCTeigRbyLhcvCA6jAmbURdbUazfybr2yOJjo9jv49\\\\\\\/f1mwqwKQUByVmrxDTTyeXwAKgjcf\\\\\\\/83dqAA8lV3Y3MjmCU162p9Ra9LmTDb2uCAUVlFna1RBYUx9+gzJ+a7N7wEuT+5tW\\\\\\\/cC5qEgyWs9zXAsg1S7e7PyjA4B7mO5yCvvOnUYCNd3VMZpVbLe4TZyZejlyVcWVK9kE3dDLFhuBSgNe\\\\\\\/3hqGmvx0nh9X+tE1Wd4yBEXNKqhpvUjnb\\\\\\\/yVr\\\\\\\/mw2M9zoneNytZyQXvH4J9J37Y8pgHU\\\\\\\/hOyOOsQ4eHs85oyCnwSlxpHQeEhBTIpTQz6NFHkKjApOffeioSMKyZfkkwN5KQDY5IwmcZmxF4EafoQ+3Oo0bvNMby+sEJFAV19KMEFqz17UbmMHvHGEtbNaD\\\\\\\/QDKCAoAvvX7KT\\\\\\\/mIkJD01qowviEWSe6KOsJiuL6L0Re\\\\\\\/vNBBMHo9dIPckwwwKw9cP1mESiRWJagF7OuaotDux+WWkd7ZEualtg5O4kaI6pffU3sgPV2Bpr+fpAjqY5edI0LqkKgdT2AFUHNxqA9fKsViNg\\\\\\\/QsIvccHTAzG5PXerEkwHSYzOgY2RF8Cedyze7tsrZG5bHvBG1F3gHT06inuUOtIhY6wrOiA0u4qsJ4\\\\\\\/iiLEsOhGl8WddW4z2j8gWLqy31NeqRlkIdHxyq6PUwiQQVr\\\\\\\/uaQtRvRfuerBP1+x6n6COA09vM959+FnFa\\\\\\\/SePd8Yyp9U71U4OJ3Yf64PzeSlPol\\\\\\\/8fCw0CXgqQO1DMRiWvstLWDHPRkgg+W\\\\\\\/WtRiyXUhVFCB2wJKiVFsdu2k3hVH+5dmA++QEja2XRyynLtMmV\\\\\\\/+dIjKHu\\\\\\\/km7hntMLaQjAzp902Hb1U5k3wCL0BSxQFjf6vA2RtmkdYbfdquuXDMpfakKb4GnD9mMb0crUyJNZ3UoiEMvqxxGd0Rjw4OmG9dbJkDE4Mn2mMrMTnnoOHoSPuvzq0ngMWeFB2rgibAJwoTgR2UG9zT6BS1BBummuqBGWYmhfCBDgiGCuNotEptTiekrFbzVf2X7YVexYLWEA80XJ9LDWZdvEvBB6DCGpctdAC91JwOPZS8WqQ\\\\\\\/FZi45+j1jIL2XVMCYdEgoZeWiUhn8ATsRqMhRUGahp7DgdYDI8tJrmvA0+RtcGjkpu8Y806cFUYw+fcautjlQTG4+sleEN5pMtJDsDacFZnrS+VJ1+U32pUaW1\\\u0022,\\\u0022iv\\\u0022:\\\u0022ze25m+kM8A3j5nPcXUvvVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262b5d79fd1cea108eaa92fc1196fd2ffc34d2406f0c738d801878dec9fa8d7cf\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dEdUHVFBMq+G89HyKnYlftWdy\\\\\\\/PB4GgsR3n\\\\\\\/u7PYYrHpBbcpQXin8RYAmWd7ukIgIYfUlOGTbmVWDKhldspKbakhlsIakgCo7tIRWubhQ7h7\\\\\\\/hp1ffrIrL\\\\\\\/lutDSW1yC3qEeLNVyqLozLbrswWX+lbPxHibhI0aSR71qdAh8vbjlOd3m8acgF3TFHANV3S8vWS6thHPKz3YmjMbIyuPGmIODuiGOuvNZRIiwL3FjlhcdKYIh\\\\\\\/xAm07jrZJ+2CgT\\\\\\\/Zp9xduH7A+bequQlytKoF75gudA3TrNJQ9x44\\\\\\\/RbWKcN0EEh9T9fsZGB1pQ9w0rpOR7eDuUy1ohxTpRSVOsKt0JAzjLjtieh9nz3icrST5dgAvtjnQ2fn86yEK90Rs4+QattdgiCDb2hrGJVibtxOfaFTHtBHJW7WahDGzyFtUEDnIqeItTcXV4fwruzL731aV0op\\\\\\\/IFVvMoOLl2YmBjfeyUKEjqPpmKu9Jm6lkoaU5qcZCO5sYlzAp8PuQ+92FS\\\\\\\/AhPGP6accissuUXo\\\\\\\/Gw7g9rFquBgMtn4K9LvRIy337jV7Y9XtZdFt+jvIPIhYhBY\\\\\\\/vlsg7d9UZMCDWU2TMDzWed3wTp2jqpfm0ccmzEHPEHXfkB8gU4m1AaQuxRD5QWzcxLctEze54Ki+Z25Jcry8tjhnjllzalToKE8usJCgctv107kMY44Oxiwqw3weXEul0ooteXte1ugdLYRtes4hQygL0WVl06+Lfx3wRFVYkzlumDdSerNLoFCs3+pRWHk2CZBr1pF9+1VUQX4gwikKJDAa4lJwtyTmdCkRKZ6Aja3XXP6wyileIJ4ntkdlXd9GzmeqeGAcr\\\\\\\/NiW8YgxadcmkhSwtfIuk\\\\\\\/pEknSyHh\\\\\\\/6MfY3omQcrxumvzAhjbmTWmfbKfPRYbCYuAqRNmASasvbzDzUwPwF9cBuk0kWx9akGg9BAIRlSIn3w7tq1+OdskEiDedxoYiWrOhIM9VxgpwgT5ru+uhg6R+xN5Joyr9onk28bkiTCCKn3Qy8RtwlMwDcm3HqxfKEt2JdQAKLT7m5lOtgMWmwPB7BOatu42HkoaclpLyeBQfzAwacFxsM1UfH4idwwk9qbKhXn7+\\\\\\\/Jjohedf65odMDln0oF2cn20MLvpr7qWE2yjf5KdiZ\\\\\\\/4qi4ffrhRQkfl5h8M3xekH9rJPabfmyv2KpTQ1A2XcLfFtHYIGusbUOqJ2bBANDp4xbb2DHIPiZefoKdoQIqHHtEZM45FgZF4vuQoKNKDeuDUTCODaLehbU\\\\\\\/ydXNMgweWPlRLfayYkLibtXpn3188gm2ntGDd581UBkmc0Lioqvx7COx+aCwKWsPPc5NGz3DrzRgc8pK7qZ1243t5gnR8yTJhaio2uzPFKIpQ1krNrJ1a+6H8OxFDLHglRfCFcUmaBaCtPLZzwKRw8i7sY5npa8wHkVCW5gHq1D7pyCpFmkNodyXvhLqDSN6yg69dxA9Q2\\\\\\\/ajDk\\\\\\\/tY6TZzGzZ7iAtmS9g7lLhKxAgvNgNbDSU17u\\\\\\\/vNcRaCINjndEggRx+tKu+KgqfB7SzyNAjnTdGlgt20zJ0fN\\\\\\\/kLYLNUrRh8ZGsRtDQdr78ChAxqwYUV3PEBfGNFV3rh57qxTFJGtWlE4KqMe3uNrH58hIKAQued46GDlVB91ITenX4TGVE\\\\\\\/fENcyduy9ibNyQGX3EoKcEPTGuBxp2S64nFM7WOTQWMacUPMv9Xf743fgS\\\\\\\/8Pf0xfdXP50BGr9Nay8td0u+Mf4SXL5dHpZ61eZRAr+ECHNEOq1GhuIuWI2+PP7HhFQPZlQiyDsn63pBa6xttZRMPsCzo6R9K0uxLNHpOhjZ5uYu4hK2Ok2kojs9ff\\\\\\\/bUTgx4yqZYoptfG795XOY4svRJ1PCobGcMtMnH6sgENELcObkqf8HpoY1J55sqhuAoAOGRhNa+em7Met6q06cRq\\\\\\\/CfkifgTHkcHSLUMutGYxC4a6WsJ1YIsXNa59ep4gZP6DoxGvWiaRR7IAoiAJjtzhi3Qs0QN4Wm9jRJ9obkqxMvd66YnpxeIWFIVtOBL10z0ypdP+d1u3epYAmaK1mQjTH5xbfUdLguSk\\\\\\\/Qkh6YdR1PrY7aNqsTKNJOr\\\\\\\/XZr\\\\\\\/uZ\\\\\\\/S\\\\\\\/Eep9ioGqTZ3MXRwXAIikSHqcTFxl7VEeWOh89+N5EQKq6Gzro6mJrdLc09eIJpM6HQZJBQyF780kojATpsF2CDzeGSasCjxGY4z5ELjWY1yXINVUC\\\\\\\/o1\\\\\\\/TFuxI5e4e\\\\\\\/6vR2j1NGiGSp8GRNa6ZW5+WTrQToulpqrAO+QBI8OzcwSxkOKlP\\\\\\\/ysmIo5syIkZgt+NHNOI3BZFG0CAo1KCI0Q0dtqicelVQm13n7m6p8246twDkPscTSwD7CJQWxLaM4X9wLocXyEBWehAr6b6e\\\\\\\/1O1hZfn2VCk+i5XIQhbEvTPoxm+odRhex5JEjMk02cC\\\\\\\/D9YaTGjv00TQ24MXYgGdxFni17J4QcoqDx+DqW2I\\\\\\\/SSqoDnJAzXc0H+9dNN1+SVl9mhrVhscPeEdgndGovBcktKLYFaaADRYoJWNqB5zZkpmWLUFzo1MciK5gg1SDo\\\\\\\/aIRev19HyOMURZ5t7Ar5Cdzrv3\\\\\\\/mPzta75KuK+ZDIjqYgc5YuZDQSYwPCLXNtZAtomh5y2Ev++dSdHza2MXN+ZsTbetr7mp7Pzt3eUePF8frjAj83jmcRwI30sOXLkIH3H8WwFpoWefBPEsWBK+Gs5554X82FY9zBPBFKv5E9WV0UKfpPX1hYsK5TMT8HekAHhnhUsTU6\\\\\\\/ax9\\\\\\\/x5fruezlvl1+3VDOPeXoxw7r6M3DY4QlZZtiHlPMBcAMuXXPnuoNRaIyon+Rh\\\\\\\/i949G4bRVFN8e+Ohz3Ykjv+O9D8sFnxPLeBiFPx6dPWtxPLVZhr\\\\\\\/WOZlzX1hcOEtt3IDojHi+Eb4Sq6fRTVFa2VUeoChCSwOew1SiSTapV47iWvlg+cKdTNWt4jJysADvz7WYZVH3vAhBaf6061tfVX+SF36392v06UwKDx4Dzo0lpJiCfe0Et54\\\\\\\/sEkGdhllb35fNcWioV5wSPBkgxfAmg8LDuek83\\\\\\\/ctrhPkPhBIraLIxZjF7yBno6TaKIibGBKhSRHPBNE1m7Ibacsmj8Z33X79bgRDdNAWcB5ufYN7A7Y1vgUnKbSVyUb9Ot1e1pIq731mKA+g6YiokaPositHWc17P3rnOl1xYFdkLxvvCc0nNj6cVshmCNVgzYNRjdlBsedAq9JIs2KT0MECKII49WOE6Rvp2+ilLiaGo3Pjaib0pf5dJHY3k6tckgcMqjX\\\\\\\/uC8eT\\\\\\\/GbK8Y440igDudZ4\\\\\\\/DIL3NKSOeQ2xOj2HQLQhXgDq\\\\\\\/kKt1mAx9AQ\\\\\\\/JGR+M5GL48bEUYdZMwz22BmJogmhE3z4aqTb7meqh\\\\\\\/WTPncg7kSUlEmJHqWMMEDtYfNYDHelgZLIIeif1qavzZzXwjsRMcCfxwK3Q9stewU0kjjz5cJN9oY8QYRWqBeHhpiXLQI95BkC0LL+N4rCzFFIr8rhpb8yEYeeeq\\\\\\\/+S549pnK8OhOgaDKukG+e3ZLhhounHSA28pLmCWW3tNjBQliKuocq\\\\\\\/H69PzQKkH8Aeht56ZWC3Vgrtn9mrH\\\\\\\/jc8Km\\\\\\\/EWiTJnRum8s\\\\\\\/2FSX9Egh99ViDrjSsjISpu5QYlqYs1lWJaTnNlGNIkkcyn5yUsk0AiySsv3VNe4j9+yGKbbi573yNm42bs76WXzmlduDDv9I94HGCX0W9eicfa2M4w60m62FtODMar7neHCeNMPV3kOBNStMOFUe+xBrnwTZ1fcaUD5F7gqAsVU2al5F0CWzvDfAGsUXArriylg4xQbaPmJj4Wptlg3kQNuC9Kgf3KU5rBuA8hk3K9jxGO1Zwfz2YNgLSbEok7A4Jrv8W\\\\\\\/wRpM0R+AKHL7\\\\\\\/eutHv01muZlMrjZ+MGZN1C0z+PrOph\\\\\\\/bh1tmzSiybfCho4j1Nq4MxXmxBnPN7WpSSCNLXlyYsbbNeManpi8mAeC\\\\\\\/XJku8yS428uhdE9yaO9Igl0+o6oHE7AjIV33IsTlXIE\\\\\\\/tCdh5aibb\\\\\\\/AEqYODoEU0F3SpERGvKFSX2tlUhap0fYsVWrQxbduUbmNL5cyz29njI5mNn7RwySDumUIWH6xKrNuTR5i2+ugV27b84O9SSFgVlJ98a2+bYdeJfzHJrXQfvPz\\\\\\\/Z+vsSK8HVoixgHePyMtwtwzgToudKYQ2XGNTwem1dLsK4ZZ8537c8khA\\\\\\\/da1uZKYyPNH\\\\\\\/42uW0+poN+jdP6bcMJ0twGBszt+VtbYq1a6usiWxtLBZhRNdmpdZkMw4b1ivNhSxuDAlqwuN7TeVzsCMRU\\\\\\\/C7ESjiWFOugoUdd9R256yYxdtiIox0kS9ImSY2ma+uL7mGrtlcLbI+Cq\\\\\\\/tpOL3pLMbRKNDVtHseN7EEpFjNox5BTC6DZW3hikoC6lK7iEjIRMy3KJszcn2ARWJ\\\\\\\/\\\\\\\/8t8QJzrJI3x77T7k4xIA2ne9Aiv+ee768mCjz0Il8ScN7tQkann8q6gHfg64PvvuhR6tQvkIYK9katrQgdbR6VxG8lUwRB9\\\\\\\/7ibM+rgP98YsAUCLnlVFCqDPqjjD1MM9W24ohej4YK8XH6d4yot2P4zzMAteZOrYzcBysEVcxErxXZvFh0HME8x2ES1AgckH1VhgG2bZnkmsGM7I6qAeWCiVHwepVhL0f0UhqP2SSlsyU5+z4nzU8KoRiwYVq297gOD\\\\\\\/DV4ZLiCJ7S1F6xE+qNnw0IwTjUgukN5vO2rQJMqtkXufwTstVwzwYlDS7SFTG7Wh\\\\\\\/\\\\\\\/wssK5mkMccYHx22l3Q00Bbw8JvKpfGHNqUPNlQsKZvyzE+0XJ7i1wtszpMCMiO9jeTeWHaE3uJoVmp4+KN8Z8yligUyTf2DdVXLNLzCAMiuCBxgm4mUKpso6zGdi\\\\\\\/Q1mGw9YKdm5VGYgol20Hrmeb8ZljBCwU1HKHG0bAukRnDgYAwwUpt7Aw6Fr7lCYQQ82iR94JlacD8aaCxQ2bo22itckmGuPvOhc\\\\\\\/bzfyKKScjiipzyqxPZnkpmN23s0ZQ+b03V11wASm5R4LFBLgvbgA1ZESeb\\\\\\\/733LKNJTpG0IMctHpmwmNypztLvoeGzi4zA6GiQBZPgJmg0kq7Qz8ZiyyFMOJnZRArquX6aer29Sy8GGcU1HZgZaAIbmObD0Vlbwnud0BzPcDG2rAJd7r4l\\\\\\\/NaS9qtmEX6+K2T\\\\\\\/AjA\\\\\\\/T+MI8rdYUFeXLQTFeFbJjUeJtR\\\\\\\/HRLDM2XLqeshs3sReTTkBVfkaGuwQELg\\\\\\\/yKTozblICMWBhd5OBBwgPUd4rYHmIN741uF6vqoPTd0Bg0qgYUSFXVP6y\\\\\\\/m78LECnXezg6nYs2W7GUnDMwNaT1ljeAxUJ\\\\\\\/AiySH+PfPpG1sKv\\\\\\\/GIFRVPVhFsOgGOriTvhDlcjtYh5t39Fc6Jr4ZZkwHfRMuWL3Gw7AlXF1B2HA53SfCxQibwOoujlalZXtOpJRkAe7sNPuZewweUDlCR9Y5YzPsSf4peaXi0e+ngrqUj3KfvxdBOKtPiRvN1enGGJYvBn6e0Yjl93MioYXP8StPuZKvt7kVaaOjvmJlGDLHtqGhgR4HcR+dnS6bwq0kl5QGrfi4mEinThCoV4vcocUbxamhWfZFuseAhdTSU6l8chB5bLXIfRAhCobIboy6cAAo305BpYYwxVcmk6HNBgk0LmnGzfefI3ZaVsRotmvY\\\\\\\/sRoBuwwseKX1kSuzziYi6lRnWtYzFP89\\\\\\\/F+j7Rkq+GA6BO+sEbvKGYz+h\\\\\\\/ABF2kxOH6Eeu2QVBy3up\\\\\\\/mGVOhjRzXoCM6LUTCxH7FJlMEokkzJGU3c9GKM0WJjvv0C4D0DZtwWFh9P7mw\\\\\\\/4+lf42xpmFmUETZRUqkPYO2Hoo4jGXpjIpyWyAXoV77Gyeqi0xMA4Zch5qizHoXsaIxX9nR+FjGu2K0C4Kw86BUi67uX7IfNxhQ6R5j1ZaEt0Srkgy8cs+YAy\\\\\\\/RPn9+avyBKhSRPMD5vA7\\\\\\\/ROPTLi1d7u3WuI3tHFuCFjlhcCpjW6tnCvT1Mop3eSXIC2DNd+ktThg3SmuUlk4nwj50tR+7Q07GMCWCPhtmlqpmfQvR1nAkgRB8nQDQAdE9pCANT7AHNXinVTEXcXOCl7iRObT+SCDUzbbWGUawlF4V8Fl8bzMv8GCZ46d4B2od7o7Jev6g78rtkoRSbrL4hbZfLO\\\\\\\/6ihCW3RZalN8Xda1NPY3jVsV6HW5RnQpvvYcLnf7FXbimYGe5ysYr2BpS\\\\\\\/1ZQgJsPbdvvc6qP6ZKaNrcK2sTI8zMio+LfqBR+lofFeWmkR1IqVwydDF5dnJ+ifK0mdAPgnnYPAVulJ8EcRvoM3v4\\\\\\\/LneDmj+5tMZW+55D1jpDnuiNfyQHOYtIErDQQbfNuGQfeHRmIPrCJ\\\\\\\/ud\\\\\\\/Kptn8UA7xOFxfYeLj7Klllc77HmpbFnTgAZoejDQ8WVZXUZtyR\\\\\\\/lZolookwr+pX8dtK5i0DRMhYUwcDAHH5klAuSsvZD5ViSHxFD8Ne0W+OEnClTwd0WdpMRrOQOVG9aJqVgJfHHn\\\\\\\/1XYrfY8w4ZCAQ0bhV3t8dDRmtlZsEYiZcFK+5ptsoxC7Ui3Uix9c+Y\\\\\\\/CBBvHNhowlOUscOPgTXMo5iLhTYbNn8QVzQZbWJyu0wFBdgWDyd1hBlglr32r47dKGueXTjuGDdc\\\\\\\/MSF399vf4w\\\\\\\/5Otp6ARND\\\\\\\/mV2Ix6XF1Xl+bdD5GRtPOMZJBW7u+rKNa\\\\\\\/uNGuQK\\\\\\\/4WWsmIXDUcrwOQkbZDVRz87BKpm933GG3IHix58ApvA2+sU6PXu\\\\\\\/oeLzq95IJTnv2fvIIEV7b+iB84Tj96zY4+TxRiOt5V4+wxWKUJcL\\\\\\\/M5pKEzi1yzDANMrvCY5w+aoqBmcjVRDPwAhHWCbM\\\\\\\/ovs0EZ3aeAIKtNNTnwhATdT5V8QkL\\\\\\\/ePhq1vxg9WtekExXV9E5R6S99GXaCYQfv+YAPbBPhu9fcw\\\\\\\/MWqpMQcEVVvtABM35HCuwXmY2ESW3DxflYKizGucppnZg+G7mILLXzwaqzTWFvbBrkuYstsnnXY1ntH3c+Vq2ShzUsrUWqA0KT+MnYkkc8ESMlpwu0yGb3CHnJwqJA1zSfN9GbVPmsPMZVJ1QT6QYPY4YhWPq2TUrj5\\\\\\\/jEn\\\\\\\/jBYBu4L+7wSkN41k7SuThzDd8AHTIMhxdoyGNyEFpN97Yaw4syUMhNGqsd1Zq40WVC71q0bbKOpvQs2Wp1kO+AnvGscSURg1WGmMD+5ODrHMR9k0fP0ruINVmrycFzAlW2uirMtmSoMaSnDkburIguPa3eSu9r\\\\\\\/dHnHRUz02M1YUwkyImslN\\\\\\\/nTDMseIhKo1ImIpSJUD2U9ktWmsA1IpP5l+rD7QSovS8ScmQmmKNHMW9THhWmROuJ7L1Z7GMuJV8emWy46ICe3UKYhDysRd9Voa5AChoEowKInGzyXC1f6F7ivWzDxB\\\\\\\/uKswp6CiRhW\\\\\\\/aPKb6FJBJUDvo1szVATscYSjYxFc4rQ9XnOhBAjW03bMBrO5FLNBsrnFTbxhZxotekYsDbpjnO3Uv687bgQGf5cYPiGSgOZEp6ST9eItHoGdUksobpgImdGHn1g09KZfV7bvpgMTTaDTh1JjZCz06gIqilJWt8uBuxrhmEzu6gXOObRruRhaEZW9xFwUOkltDMau+Ww+kZIAH40Mveha9ffZfpQoCBD65nlaFUwfRxJWuIGXak182w2CQWckfrjmN7xUpo1H8xz0RPuFffi+\\\\\\\/7o0\\\\\\\/pfPF9jOQnXRK5VBEVDO9hNVSq9OksFBkEc8JZH1JQYZCIiqXcDJ3t8RKPtWTgHGijTWaeYd\\\\\\\/CprIKy3HbwV1CFZ0nQy1halzzjG\\\\\\\/XaXS8u3qtjtoReZz7hZmu2LkKCftggyeZF4JrzLHhI\\\\\\\/x83FPYd+bP9zGbaOJdMhEP9SBNTAIT8UeKiQLNmTAksDhltMSO1wz6pMuT9fxRDe7bLdXuEf2ZmxvGbCOZhSHpkixs4GTJioBXSHtyrKVsiKHh43XvONVU5+LmC98ApZqmtIToYLirVsJzWwtJ1jPq1TE6hmxKR9bWMh51IW7cylDqMAsZQ0APD0Sr9vf7hhpmmU0e92Dy2z2fHN48SLJ8IRIHaC\\\\\\\/ejjivpaAYcmUX2lpHAzvlzJ94dC3NtMx0eROOKFCFDnrL57OiloRSxCoEwiSkoegTA3fWuRIw6\\\\\\\/RLP6blC\\\\\\\/0P2k4rjESs8g2tY2d4JSEGhriA15KzXJhM7CrIaLBfIjXQs30OtLikqjnxiUDsUD8bJ8F8F2wFcSCe3eeJDqE10V5nEtRPhIdmoOx8Jj7H71QOe4itJMmcj0ouaAD4lwEmM7RDJQfO4Yvk7oyfcLdoluyovX+PNgjrt7wuQNCrB1XG3GRd5IJin6cQ4QGjJfT6XS7Pv1BevoLtCspZnwIXcyXP6bGIuxFlkOALqjsHkBCyuv57UOeDq8mU6X1GoxHr4XabtLWwC8d6vLdnbKqfLNrKppFTfEaju6FuO5vdPXrhMAQtFHbCAyz1PMT5r8HHpdSQTSfsr1hRhahO1y7XMo8Bs6BtM+767irmJsDmIZwR7b5UVD56OezsPYcPeeZkx2WoYsFtkG1FmMJctO8ft2Pwx+xykRFnnN2Bw8AbZwdZF44WEZeM+9Hd7srxYH9Lxbe1e02ZGmmHAB3mWjJ\\\\\\\/ZSZqQSDLvpa91sQW+bxu3FwkYaRhwDxs7fNeVIoQKzmewhP8iCFXcR+J4gl1C5FnKaByUJ4jlkAOFu21I8lrvkyxBjHD0iC+eC1gUe5PtXCVXeo0VqkPmn8aRp2y\\\\\\\/siCI4XiToPAo4mXjjLTlGBDn6q6yqgIaxp7gn3yX6AX8eJ5uqMa1pXO+RIoY3LxzkZ0dvRf5nBjcZaiZRx4lFQRyijkyWIsM4bAN6dQVF5+8aKvpALfMjSqFN155a4XpqE9lZUQgLEaw6BsJ1Db59wiGUCVQXv0JgII1LAhcn0WIi+ExGFyFTSLOJNSdNjGjhD0iaAlXdIkLo\\\\\\\/nE0aViMxbWUjprjW7Brx1X2eHwFdyUt+GrKUAkL0Mvk0nflSzp+inXUA2hTpHCDzO3EuHhXcJ91Fa4nquiIP\\\\\\\/oruowsJD1FgV+yI98pOSA59i3dl23VgV\\\\\\\/aM20HqKrk2rStVEGOVsJ+0sW6d0\\\\\\\/WZvoMAtD3m3DCWuRTJiiVvu7Slq148rNSMe8hv\\\\\\\/8SBovmYej+lu04V2JgPyk6vWflOWPij+JrHrYy8ikRCDMS77HGmkkGm2uPVvXv3KxynQ6Ep4nKYzRuvjniqc+8UyjteKsdB24JmB692e3pjgCvwcJMDO1lyIvKH7X\\\\\\\/TxLc9Znw5Y4WJO+fePVfFmCdTRkeismb0\\\\\\\/MYVTmP\\\\\\\/7+B+H6zCqSewZHtoFFwK+ZDzDZGEXsKDuyUIaLTT7WwxlSFyggnpocwfVnn3X8Y8VoKRmEXukCOVYMFwzG\\\\\\\/VF9KpOwF6\\\\\\\/kTJd3SxZ103CiPBGNLuQXR7oseGtGnVhX8tx\\\\\\\/mUE7zNoDujG7PNM+8kPF3cEw3S90\\\\\\\/XoajyTeZNRTAoPesCAU7f7KoCQza+Zmwmqi6yMZBCesYbq5u8C0g3ZMS++vzCXfzFe5umlfOTYONtFPENXUvvRcCszk6xW45ZeFO3WJS1wv9S+PluzIDrWzaOhOnI9V3QIVO2yVnsKsK7enAMERRsSHlJGgwrY+VkeoTqQQD+DZLRhYF1FCiZe5oyS2mjF4lX0h\\\\\\\/At7v7J2KrwsekuK3GPoejg5XcxuGhWg3nfz8RB4CQ++gzb3s8GeBMuhxRaZVw\\\\\\\/EUB\\\\\\\/iNxlRx8m\\\\\\\/yN4aWqrvDpnwwsi\\\\\\\/StQJ\\\\\\\/owhdMTjilw9ValZB3gJFCjJL\\\\\\\/\\\\\\\/lETyjIw0zlhPPAgSpczkWDs5mSlDVcmVSEtJ50AqA0tcRpkd9RpRKNFNHrxT\\\\\\\/X1RaRxPnUEVreLE5QfCqjldSfEac9RoTmSPPshsHOZPwI8EHeX6yk5M+osw59crmfkpdsiO8hJfgGE6NP5fY+fT4NTfxGW0E5DwlUhJFxzP55FPDrdq91J2Mmxrs12FLNlLxhghPbs8xGXjBiIUYVp4RXK2+QkmxaKtkn8rIDP7sSz0nhgiqxoWjAB1afRxm9zncIHVlTLNhO6nSsE42Aw3L42fgwayeVN6MRDzQMLuCEzDzRKUh1e3QMU5N47w2l3qTdcaqrO36ZzuJIYmLpkgXK7rpbDHqIV9qG9mSSlbxQKcF7SHOXELFV8RHaUAFt4OVG8LlaNejtzido1j7HwgKPD9b6GGy3AMfoW7rnPXz+EQn7YY2OiQ9fNxAPDBAfa0H+T+EJzfDuADvAh3BPZ\\\\\\\/wvMjJ+Tj3Q+XuD6KPBG87g6uekpOMu\\\\\\\/jGxFDJ\\\\\\\/GGD7R5fPzl0lnVYRslV66WY+tbcH+k+OOWWLzhraop21YQhGzZ3Vu+uVEqJOB\\\\\\\/jqYmUxRxTdXsI8IbVIfyDmnThXMXaNZiq3lG0C3GXWG1r8XkTxRB1i2elGr8KzwhwT85WOl0DR1tGXCp4qwKJtomkS9d\\\\\\\/s1gPaRKN9wNCHq37ayoQYnEcSbg7nDsc4WG7W8OGhVq6MwqzwMcBNx2j+19t5UvqNyEkwWRemQL8TFIR1XTVL\\\\\\\/gCWgF2Cr1964Jw5EVE4Mte2XBsOmgkGjutAlpvHn3krhOMpKHU2dlIzH2QCXg1TbeuKgMgra5IgvQ44q9Sglks2QfthWhuO3HdfH5jCoYUQyNAfGdT3m4Swju+s\\\\\\\/uyIxWYIQyGEM2ljgf44xM+IUPnNLFVa69\\\\\\\/62aKKQ+TY+fZyGcqTU5SMWqb3dPhqYxdZ877S2GRCnPrg55yZufE9lLJz5O4jz8eHyQqPAaHuRjyP91HWTZJnobsco6vpvlddsVlawE6JDfRAvTQBre2NEHnsI9nnESRZjfrzkFNv4OGXlLUTBvonoNFWWN0FiSe+Sb3sYFApHfekzY5ssbZtws2JdXDYxOEMYA4LrPF5i6LNk+zSWLTUmhZ7HqvSHspEBMzJ1MzcG52ALdZuLOf5yz2GCV1aEL2T0H2CtJho6wofXZ+TVxzScw37gihs9xs4tNzZlWDYj14pNISv5BCX\\\\\\\/GPWKkVceB83TQspE7XeMiMrJ+HbMq71lp5l1K7nUuaPGzoDSXo57G7nfwt+NrF7JaxxvJRFouZbesZhMnFF40Gyf79dxbnObG0MPlS3pp5cEI8GAJkVxJyDY2TeHdcohqiTZLlHkOrSSeq6ooxLH3\\\\\\\/WQsYocexXC8dJOSBsBw6nbUuK7bsPjLPnpbdoTubrW0\\\\\\\/e\\\\\\\/9pqvKej8Z3hA4CdDJRipa7lscrXx5oIKqaD393PHsJMnScj+jUmN4HG5udOe8tKzVJDUmB8I7QR8b9MHNUd7LNRlOrXlIEdQMY3hmVcY84RkzonOQCAKghBwKquYHKy3g2cHgJrXS8+z5KV7hqLM\\\\\\\/2NLfSDhrmuQpg4M1RnyKD\\\\\\\/Dj3FDnigyyDbezHioNKCgSKA3jjMY0M01tdiqA9SHnrW8IwMG5T86hK+2fgAtXyRhc4UmmoHifLdlCht74+ExCALAEaB2wpkVzs5tSnPw6jIUYMHXYDmFgmos4bQF8ZABSVuClcu0FdUoPEuocJ0L5Pt9braQjJMEJiMcU6V+ESoWEUidxhTXetjBJB5r+rqW\\\\\\\/m6E1Pu1dGErTafOEDk0ZDGo5cyFkY4n3AFj19EtgXwvbQcI55PQ8UTHNl2xb1SsTwprGI8rn38THS9EstSncn9\\\\\\\/mttmVlN\\\\\\\/3kZsvI3fRJASExx6I\\\\\\\/8w+W4WWVRqsUzOgtsSKXkx4u2foVULZs+WTfhFpYNlGPRy0BvyR3GRlKREdfHbHIGq8osBT8ExtIPamyl\\\\\\\/G++\\\\\\\/6PXAZ6yCdLXPjXJUvbj+UTb6qSDRXu\\\\\\\/oD4iRfLk89cyDrm\\\\\\\/MagW1W\\\\\\\/3lf54\\\\\\\/qSdsi0WRCn7Y2kVdCGUFj49zXQds8QnHMtB\\\\\\\/P3WcEvnGhQxKRcKTqN6P3IhUxp9Em0ddDdLZCkN4ZC4GFaXWWnNgTaBt3ak+8wnIk6jlDz\\\\\\\/K6MPhQ2aWRnMa4H44BV+RWwuOUUMAhOUBrXxu62CFlj3zJkdu\\\\\\\/YuQtPs36smMb19\\\\\\\/7xFzlab4w+vrjhIA91Z1l6SPEXn3W8YAvC5MHTTigleHxHx6qy\\\\\\\/3g5jnqmH+HZpRE89QEjjXt2MMeLQVHjyhrR+5hIRquN8L8ttwrryAI\\\\\\\/VdcA+yGcPrbsLwwLCE3XKM2BXSCUzUnsYKp0naQaJcvJozcAdmB09t7rGQaliqA0gX0ho8VZ3cD+3OR4HM1v7TbzchXDrbB4uEErWguSSvAzm4bNe\\\\\\\/99\\\\\\\/y5MzbkoWY61khTbeuqU7aQoh6PlS5Bdfb3mdMNP5Lzcv5oMOkNvEyNuCc4KEXGpc+kH5wVKQjsIwoV4dP+h0a5knzzT\\\\\\\/VOR3Wep9+ipH+iXObqpmFB+9Kf8R8VvaeAVRaAxwVCXIZ0WqQYdqPXx7a5EaHby8a3fJtF\\\\\\\/b7lxi4\\\\\\\/ytSIj1wm2+4TJY7n2ZaTmJj3GZvAqmysUcUbfqb8o0yvp+5ykFsgL0cr5rotxtxRdAyjfBV0Gyk2V3R5KAHAdgyVrcifjJDcgNoO5QA07LppQoSrMu3eOfY1dqGJNvUtH1HIE0g0MznKbOwgvr8MMhXFPtzlp2mNk1GpZF9XSEOUE8o+9REyJb3wn9Cqp9cV9oTCzGHOv\\\\\\\/ESHL5ZHtwmUnptFYCgFG6oZy2up6ICma1QF8w5B2o\\\\\\\/NYrIw0mGWTOt4jHIadgb02Ut6FeHr0MaRauzXqOUYy6efNu8L\\\\\\\/H6JHSkLLXoBvsHr9fWskB\\\\\\\/qH9kiTyM9vK9aF6jW8DFz15+5+YzbILVcwUr5lot5cii+qD72jronh66ACZzLOslivInVglxq\\\\\\\/eZXvWD+ahiFTqCsZtTZfYs6yhNxQLgXhlurbsZrnFgSPaxb+\\\\\\\/+BB5CNSWC7HbyOMIDOzMWPMu2HURr9govhQdVASxtFtvc2ygcxm3Pj65FHY7oUpP2MlLL7s5m3h9pOXX9Mtnm93dolkwiVm+qr2ljwpUT5VIOZlALRouxCgBlh1fKNpb1QgvKXq8N5I+RykEVpkFMabY3W85kkYig+TJQlAfWJubGTPViWjFMgy9g4u2mx4XkD9dunkdcpm\\\\\\\/J3pj76c43SYTv9SV9Svx7VGXdJd7F4npVDdvFMP9nHMkF5r+S50qkyjgarnGloKRZMnXtcDA50y8FyGSgbJIH6JEfBm4mRCG\\\\\\\/FnNdbSUEx9Fs+XUwq7xfzGQSqYYLNZEATb2ZXbTrjtnCOvCaovjgekKciksn2H7pubQqF8BbpC83VIq7YNO1gk5n8ZjS2l86LamsEPh\\\\\\\/CVjjemqOMjwLXuaKr0i5oaU5HAICjQeKA5EWysivfbh0r81T+l4+eiLXOrRjKPEvGTSICqPpVbf+Yg\\\\\\\/KwTCgBkQRjn1u3UKaI\\\\\\\/rlGm+ehEM1QYNy2of28HsnZftwFuf45r6Il+8je45YXFuPUB7OtM\\\\\\\/yflmXPtm30bh8ulzyAACB8RZro1hGlpA5wmAn24wjWwVhILtvVFGznMRTXID6gr8rRKT\\\\\\\/9jsyM01EOv6ODI2loAtxz4gh3EgJ86ztZ2577ID4a6FFA2WFHgyDQ+F+QZ\\\\\\\/mRGVH4f77oaayeQ\\\\\\\/AHml2SkuRPv\\\\\\\/L6zgP6ArkbUjVuQ3AWbd3FJWzkHeC+1kJBGtVLVZ2+FjyFi4nD4ZCIh0B037gmqiX2rDbfSbi7D98Ymujj7YnCy1ssdzHx9q+3+\\\\\\\/\\\\\\\/ZLCmN0icB5\\\\\\\/b\\\\\\\/bHhqQ8kxq1zQjbP9Ve5SmjyxbiYFSY9RekyVpiM2gdCYYwpAqpCIq49v7fI6G6BhihMlBgW1rv4FTxzOQ2PK2OkTRoI+rSwztXDaeLVz9giDZSFU1eTHdgdvMS8\\\\\\\/u7ajuDN2xaEpROhfdp2vb5mwXRNdDgRq3zc2jT8Zw++XN22GCGsbLlbdBCEIHWUxDU9OkSRbn2islNWS1R8bpYZAVE8VH+tCGF17nlFxNDo5IpBdI9kKE\\\\\\\/+Y91by5tUcCDbfdJsiPrmJeVtlpr0ytyu7vgJmLjc7n\\\\\\\/ALo\\\\\\\/8wIx8\\\\\\\/fPIVKxezhY0rFAAgFe+HzJHLztycehZ6JOkJMhRZyjg\\\\\\\/66LCHRHNnczZHt+63LtwRLRMRyxOfVT0IcRQH7hIlYD1HSRuiHGK7j2sFC1tcdUJG7azcKsWhZsFzY6PMMQzZjp9xFaHKVhTF50lqGbR8D\\\\\\\/v+KBdE67MKHUPufgSjWGQKpDy3Dg6wJLkOV6jhJ0Q+emMb1lbg87UtfQ8Zze6P+x5ZAPMuIvez\\\\\\\/U7hy13QPUnDSuddwcuyQ21R665q2gS49BrOP\\\\\\\/tM\\\\\\\/UinHN4zOYBE2EF1NN\\\\\\\/HS59g\\\\\\\/LUcrbVu0AZ6jQXpjpquaok7YmZ2Sq+EoLC50typlEJai2PYnahU51WnVN+kPs8SklO7nf5Pq95SaCi7ms9NAfvpucF8ruEr7NHWRyPgprWzQuJ94u1yq4cZI0CkuzihRF3QNmi\\\\\\\/Kw+cb0zz4aur+B8QRGVj8zZsk0gjxHxk1sieWwWSTRf2l8W\\\\\\\/weqR6s\\\\\\\/WVALxG3Ai4lyaHezIgu7p1QSAS4uub5N\\\\\\\/JqaAs2JRh6pvzAPuvnhmNwQ8qGX62ZUR9GnHVD7ALHB7VI2yUcL\\\\\\\/gOpBNYKH2D79gcTOKCjdrfQ3zbzrsKUnfQiO8HA\\\\\\\/xx8AZIGp3w+DarBo4NAyz+ACn\\\\\\\/PkfwMLVqyqJR+hpA8n6or5LsP5d7owDuGlgqrJ\\\\\\\/SwGjSwmS+vXd6kC3ZD3y56Bhue7YMcGd6nU0GX10w1Nhw75Il+e8zMJu7KYM1liMZ04CdY\\\\\\\/cgIDptklgripJme2ultbqXSY9ymWA9dYmvM\\\\\\\/W5Y11LC9G7bGm8r+IMqB12dmvfSfPX58ctlcH297hz41EJJyNHpA9Has7SGE3vX9JF3tqzgXycOIOBDqWAjJasT9OxvjNnweGgdl\\\\\\\/UbXWETmWpbn5zY5LkE0zITlwX7HP2lXksnOOLErvvbd0RZYSfHY0FCXxfMOkLlXcd+YNarI+IGQjV8wRTPkBxD5uXAuW7mdbvNZhH91eOdYoiV2e9Oen16+FQKTTTrITGFe0+uNBSFRKm\\\\\\\/h3p6hoAfn8FNTnOVM2PSnfZQc7E5TMdXrvJ+7Z42g\\\\\\\/kUeUwCP84A4sxaWadch9hnZWel4YvM2VBeWEMOPdPasOXbjEFKWYgKby53njkTzHD8UsbTY2nKjaCLkrCpU3flrPudSqTXVTUADxIEw6NUsrUDiWb9a\\\\\\\/XcVWHvajSzRgvSWMTVzCFoZNA1JvyoVb2JvMUCNQiCqqFMsfAhX84FRvTtcp1skpbndh+PvcC7A7ev3e0LaWP3snd7PY+sLYb5Tg3fWRnvGFDzJgxZFlbzdUxZUcxmvyzVPcDdOKEW4vCHRfVYH\\\\\\\/ucS1DobZeqvAc4ZtTi4gCtrYQ0awFrakZgclb1WVdMHQvaJg3u+hWI51QTeMTW9lbBeevBZ7\\\\\\\/yIrUp159iUvP5Ie5ADavD3PlfN7Nb\\\\\\\/NNKzBYd81CxzlqPIEPxQqToVIMQ\\\\\\\/Rt7XugRhlp2fxc\\\\\\\/wgPfqDH+0o7m\\\\\\\/r3cX1PYB5HE2Ki4PJKkt4OEpm4fW3nV5oS36aM1\\\\\\\/T1AYjsmb5oDZVcHcrwzXe\\\\\\\/wruLtfqoBFWIjwDKe192VIaHACYmT2FaEHg\\\\\\\/baULxi3ph4Zn7WQtfPPxDqPuxlINrbki7dHI48A1vTTgqNpJxc7\\\\\\\/UPH4nbR4a5MPOvVFJbCjyqBOhg6450QV0XA3WWrFbMwGxgBKR\\\\\\\/FQTek1tC\\\\\\\/DiwfhvGbhqXsbRiF4cPzs2m+rgkxL4g4j\\\\\\\/Ptam\\\\\\\/Cu5DOrp9JDEtN3Rw4BYVyVHsSvEarXwjoSGeInMoLuknyPVzJ7kxQBfDUkKxojYXr1KxbJU8S9l\\\\\\\/uCWycznhACX7GTDYzJidlanVRGjZPfzNa\\\\\\\/LMeKy+AlbKqAc652FRBRiiE2cSvJ\\\\\\\/pQ9ApwXPva\\\\\\\/rCLB5prrV5GQx5n1R+LaxlY\\\\\\\/9gUG88uAz3DJQrcmSf9srtikbXi2GN33y6yUDVLEyZ4aWIatmvYMo3IM3CVJ0dVEKubwwS4eMu3H9WvmKDcvkBtN1kE\\\\\\\/3Bcc+r\\\\\\\/MMJtfqiilTlZsa\\\\\\\/GozyUcN9Xgp2CFqtcqN2dV4hOcTZBF0ZN4ryWSLtyi4W67ysboJsq7BsFzS4sZxKgWCn6OSAJV4eFwH0mFMNoQPAQ76mVvUZsGXVUa7CTcT8tFv0nY7GMzBAzY8sdLZXCZpF06B6HaYwauvt1OhzfC6ZAb7PEkkjI0ymyRPAv7wGzKaNUow+JTgT6Q4YJAA3XKiJVblF7hg+RRhwZPRFe7Lqe68LGy4D\\\\\\\/OKBMpO2mjmP06Sa0sooZxO8Nwcb6u5yNfq+LdkM1tfIPTFzsXFf5qTRy8DXcDZ6zamc\\\\\\\/x71GxCXZm+mhHdrPKJVHMzYeld2Nb42xCIcaXQtaaDGQoGfaGx7p\\\\\\\/TSzu9Fenl2Y8bqWWJLCtXHOu7yRjxXR8zBSMmtbKMwiFgXiXE9iWIM+HxYcbiW85E3zd9u4b6W7AY+fmRBTKh5O\\\\\\\/JysdjfGbatGvRnkS39uc5cb5HNis2FdvNvlhrUHeEm6\\\\\\\/yqq4LwylyYC3iisM0\\\\\\\/Ion3YUNRMkcKY7Ie5DJHLy5det3H9itDiahFW\\\\\\\/gYvJNPNPqk7KKi7Aa9MP9neiQ2vg0lFGuFm\\\\\\\/P2GqmtPpbXEcoIsmpplFUMsLylYh2sQUkH8MM\\\\\\\/PVX\\\\\\\/+B9MQGjr2FoZEgkRbfoiDeRRhRfZLsHwAeNFG9\\\\\\\/HmxclTDU8IFqIqE1HOSrKED9t0b\\\\\\\/qjmiR+A97AChroMfI9uUtmLiVnZf4LFLylHP+S+uWAM9jiGLk7IWsxQHidlAWXCSp36\\\\\\\/4wlahE0EnjRRk\\\\\\\/1hoxKnuok\\\\\\\/\\\\\\\/dAlqIdb7eeGTnSzRZQuzdWakxJdVnT5Jkymaj1S0c0AF0ZRINNr3WNP+osqqB0uBcv4HPf6GFRw5VfVo+RNf1EyBZG7jq1pP1EkNkC4YgbWgcXhArxTKZhIWKZQPO0ogQysGPlnT5jzIrMHRzl7No2wFpHaIlW0laCKajfmDKShajUzDVOwaPClY3iWIptXToD4GLMKAL93h0+RH2hBHBgQPLbOYzLseVg4etreNuX3sf+fdYSSNRqlM4aSr2+HBV67LeyhITOZ+nPfL4sz2z5UQJRAYfWwPC3C5MeH5iZFp7IAWPintZrBmSBaUTZRm+w+ydYRA0Qk2aUpvxHTt790WbNqBzYxRoCwTXu1LzZdDtQGrgFJT+4ILjWL98EfpoIBuBDoZuCF2rExO+iLAuiyIDNpNOHRjki4fhZ8dBJO6CB3AnGEUNIdsUTdCqEy94CuMsYepkE3RYuQ5\\\\\\\/J6WnuIeD1ky1nIzGgJ\\\\\\\/XlrbXlgdX2Ya+gf33TOPBzcE\\\\\\\/ZEFgRd4dIBwBVHd24Rpe03MCQrD85kL59bohf+e188sAlqhIKFfzYsowtMV0I1XfXE4rnWnMLwPlEgE79izn0M2uhwgR4Qtjg291UX\\\\\\\/XqsXTF+CO6tCvkB8bMBBK7Ndul8VGqfVp5P98RjngeXWY0f1wA49NzYwL4blpqFKOArV49nGo7Xcnm7+9LFO7YoBHU00sE0su6C8tPeWoab7h0kqEsRB9VD03IP1nR+987cV2QGk5NKCXBaAM8y7rh0yaT5Z4Grqm5Tk5FrrzcDLxQkZTVvMyEOb6\\\\\\\/BEHaGpXuhCv8Erwj0y2OOgYBLDkt0JyfqsJ4YuQIoB2t0V0PITAWR7T\\\\\\\/JFE3KyiReAw5xm8th\\\\\\\/P0reAw0BBJheFz5s7JBSMtjLG6x\\\\\\\/WLCgEbqH7qO7DqOEyXzF2+RiW\\\\\\\/OwOQPIC2BP8uiy0LQz2OB8lMVosL0P3oWLREY4riA78NBYNCrD+Vlsw8Rvdr1McX8b0VORpBLOjoit2e107VSjfFKxYcr3qyTsJZxDNqz24ZMW7+u9Nnf1dkxqpt9mAv0IF39QD3JeaDXz7oXG5AvtJ6A7u\\\\\\\/pvsSXrqU2yXmjQ2YqVifkTyJReyDDdDVup+ZTrgGtS1j9bYzUgmR8hNKd8GsQGuiyLDjXfAiX46dR8YWX+b0BGlug+ZYQ0+Dso4z+p5V+kGIAENVFwYOMFFqg4AyYMIBii+\\\\\\\/VKaFkc2OFzgKLhGuXjBQ2+LUFitop1rdTgM7bSCgSFLpoRr\\\\\\\/My3VGAPmIXV9GgTRu44vcIhKl0GvxoFLQXxi1yfvB3fIoyvgfemf83zSizMyxtrctill5k98CCaLqyAzD0E\\\\\\\/UNnZTObiWdQtE1UDM1AYslDxZlue7G1tn6J4plentqX2r7p+oFa7Y2vnBeuKphMf7Fy88YpaBQEkJLJK051g8dbh9\\\\\\\/hXC\\\\\\\/ZSlWG25z+amZvsfxzXuPdu\\\\\\\/QtmXW3BfzFi4BatyzEXaRIcbMzLZB\\\\\\\/lIPAGzPQIrk3Y9hWov\\\\\\\/IW1c6aqv93x+c8DuUoFWX\\\\\\\/RSZE2Aa1kg\\\\\\\/XT4IFYOojlf\\\\\\\/l96TTQiTHcifY1cLPZc14+OyzT6cSjRtHPmIFAwk4g2HlNWCnG+Cczmwi\\\\\\\/6XE0X4PLwptK8FmK1n86l5+Nt+v1inJN0h61U2EHnjd21kSWiZPugnSZUAQs70W6W+AW7qJWQU0aa5turazGha\\\\\\\/XH3kRdj\\\\\\\/y5k\\\\\\\/o9cBnbzLrHn0TahYCkLifCKfhVhsx4kVeu15LbwtaoXmc\\\\\\\/TfmkNNsFQdCRDAGeN7Nvy6Iz9hhrk1wcwiB3QBoHNKL4Ahp4dTfeQTG9GPzvZRePC1a2ho4GXcPnsSxpFawTKKAxhx5XdZz\\\\\\\/\\\\\\\/TJAUS+RypNA83HupvT43NmO5GbTuZPwNm5sVNg8bbx7uIBHytXJs0LJ5jyEaZq4n4jmza0Vzqwcct6WY2vWBHh1FPWCwnEITuUzWdWEQOv5oJmNmLsj7rNquSwZdij4wLJMY1LeMoMgffugLWwo9SOwVVNoDMTB9vvLy4QI+7sFLVixa5QRGOMn7JoAN6UzlqzCmONurLmYs\\\\\\\/ybk1Mt5DryWt7rmELZRvWDhdjR8kRrOGUXTkmabaRbPUiaBNzigWxZ9WGBzod+0DwhoHsptLS3KQaJcAzjfEWv4UxoBqcOx1EBU+k90DQHKSYsIxCTBeipvKwQYW2xfOvVyx9n9\\\\\\\/Qr6IA6EhqaXelaMUO2hYAUM\\\\\\\/Z7Vizh4uC34murI+pNu5jhbb3j4C4O+UmED1BKRae2v0jklYN9P5cnt28eKOghbzZuRy2aHYiOUlfle64tHxqO4N7zX7Zk8ULCxG70pd8GL+zPCggHiMuZj0yjrXx+DMi988u\\\\\\\/\\\\\\\/C4VYWVOaatn8p1HXV6ZFWkeZWpv7LuZVDSzVfZPdFzYpDdfs\\\\\\\/XuHy1sobjtYVRLBFvZpFNXPxfKgQl7o\\\\\\\/i2MHLVPAvACzqZUwR23qaSlXp+N644txUtsWxM+uIE+IirhxbrT20+VRVUNn4EDBKtyi+ziME\\\\\\\/HVBPDay2osxCtuKU1sDSF4Y7KoDN2ap0Og+4nC8Y9iMwIGzLFMRF43XW8trqCik\\\\\\\/8+0lPPPpHLbux1o06jIriEgXkye8ZAJcHVlTttozwoOCgqgwzjxr9zpONpelzSL1coJG0v25yD\\\\\\\/Is6WR1BEu4frz\\\\\\\/SdbZi+3bhg8Vsw6DYOSGDjya9v\\\\\\\/S+ZA75PhDr\\\\\\\/RIa6nknCGJ9vrFmn90RbQ5cV1KIScKyXfPqXeNvzf+Y1HhnkMKp30lURyrivF8a9jDVymfyAMcgEXEet62\\\\\\\/teTJxuwtx+JEZcftIz+T53BOce1gGZff2Yh8CaMdf0JtHiZnR5MvKLmI\\\\\\\/yBse3sGm4SPWqxqDeT62YRa4fD\\\\\\\/SkCWjXX1KQc2ZR8StebEXJUpp+lOw\\\\\\\/nh+B31rJOu5edBW\\\\\\\/JVQQUgOIqr3GEJr1TUFG\\\\\\\/f6MX7BetRW1Cjpkp\\\\\\\/P0jJJyeu7A19UQS3GieiiePCTdR293GfMNdaIU7HQIpwNlRBANJAWY28g4Ylk3T6uVX44HHpvWb5kDAIz3s6h1hKW+TC\\\\\\\/fJh7imA1EZXtlCqE6VxZIRXl\\\\\\\/05WOWGIflOnMMINKtlhQX9ZM7gsoUK8xhSDavistMoOvlqdocRNebUuVA9TrCOd+h0OgDMN2f+CK9c43D883\\\\\\\/\\\\\\\/DWcJG6rH3OS7rualiE0QCVIl+rf\\\\\\\/RI3o3f5NRQm3Ysrbn\\\\\\\/9klUiIOQnfkSjzyivICPFOkhyOh\\\\\\\/T\\\\\\\/hOaMmRlwr2Mw+xsYggwLEhubq+qQv2AusxhqJ+pn+W67Ao1eeDLfXLnSec1Jw2s2i4VPzF9N3H+ac2Hk3SDspH7peuVoO4IiJ7UAJNuv7iwTVQ1ie4ATiu7KQw3HMykn7CI03olygPpAUz+CG4TGBJRswsI86WSecAbjHhOnXatFWfpbyDu0EsRBZYNpFAPOh\\\\\\\/wlhONSIxBb1VmjMwxOfXpxCaebWy26pd1+hTzE3yCCkay2bA1nrIijUnK1maEGZCJxtpJC9dWyt1RcXVcVf4blq+jLdqNyVLvm6HpjsbxKpP3Fhm99MIEUuAFbZV2KGxEZO4CjnsLfV+N0qi3aFgtKs7Y8DI5vHYUeuFyT0q+l5u\\\\\\\/Wtvm38yzLH9I+Vueqmd7l42ZrDmn0vrTqQv3lFoTL1z+oTYLcTYnmTY7EVSWO1QkMuyvVsZClltLLJOdfo\\\\\\\/wBTLLXF8+NW3513upKr2U8jQx7bTSr+VQXHid8UqCB2em3dw9aSIkXdTCm3gRNBycrxdEr1I+2rLxJMCFXrQNY58ri2vMBncvDYOW8HonWYKsN61tVOucLbort4hMjC4J3NcOqEX+AOXlCdFsD5MMkCRUaiI08Ts8VhLdKXV\\\\\\\/QrOwMwt+HjeF7fanorPDbBBuqkldiQRMJ9N+wVxA0bXANsI4P3EgxqzI2Z1P2jfdPhYSBmi1Nu2UBebWbJ9JTx8TLQkwSPFdmFG3taxi0UKKQUh0FVfdsm93iGTaz+p12vq9rRIkQLefMALHFXj2RIS7fOVZWLQ0DAjq42jnA\\\\\\\/fBp8lZHMfBd1WNZSznAP1D2SBcSPYq770aFmuiGhm77FETrq2N4JT0IhGMBiP3GhhUzf7Dp2xsyI\\\\\\\/LsetZVO\\\\\\\/pzG+NbTA2z5xpnbcymWmugZERXe+jKe\\\\\\\/YwLdWcoUmDTfkbGaZi5XgdRS9AMpnuRTI9sf6G7gE5t9uxAlfw7lBCC1dwzFDoO6LK74kJb2mGnYSaisnUuTR0s\\\\\\\/zsn0s9rlNH4E1V0vugJiwIu3rXRUrssHNA+jA4khLtxWMEizcQnKp69TEvDCE1\\\\\\\/hbXt\\\\\\\/NAZRMkDLEhqC2Ko0obKf7a\\\\\\\/b7mhPzUQ48uhK4+uKpGY6uguETnHZVyLSKwMoLkLwrwbJYxKwRLrBFmtjSqVMvPFb1cGqXpYpMS\\\\\\\/4o7R2L4os6lv6BfWmPBwIw5qJ1dyBr8l8GGr8uuP4GG7Dl0TM7PmKA62llmmEt21JYtHUZO+NoWW1yrDLtuZ+O06IUo2SfrDFm2q0+SaWgwD9P+TW\\\\\\\/A5y\\\\\\\/zJti843m4ECDNKOD0+tq8geTHnjyKjE+opYL\\\\\\\/c66uVCU\\\\\\\/TDmfge7kJWDwnIe1yhdm0ssLC4yRtkHV44FIHsOPSpZX37Oqeww236rBnJBWqJqx\\\\\\\/0mbestAkKN1pzkSzZZwoobqaemLGBQiYYSnEq1TdW80kBFN3eEjSZvghnPyYTEUg6oh0bIfqYFSempvFkmilk8McmC7MCjAOeuVnohV8zCAQHfEMOHtzPBfvVPiHPcvfCHhBVio1X6YMoD7nbyrMzhbIv5SlEkRHRdigcmmtsJkKaywPNrx00sbIej8tHWdtozAMRAsiyOyj5c1wxwaffe5qmyYhCNoLGwIbCCaL\\\\\\\/moHQZFMdN\\\\\\\/1HIaOQIohKby7W83g\\\\\\\/uH23k1RTne6EqGbDYUoNybmdYytJ43mRFPoaMtyCQrTIC967sT5aWEZD3UAS2Gz0tLMgxx6YOgrpFQGHeLovPPXbTgjQUzRTBqijBLTfNeA8WrMtITd5xPHoPe5gZLhSIrwHgp\\\\\\\/z4r26QFqNSOQd7x1+Qc+5AGsNBfkCwnHkuJkVwulqCN6mzbQRjQOiau2kiLTu9ORTUqz6rOn63ViJCaf2A9T6ZEiwQiXc5P3ThGrwlLFwJIBQZW5DE+vSHqBZD5LnNZhz8ea9m6fZm1LAavEE1VrXPf6NBJEPQammXZLJTz6MrzyMap2Fk9peCPRcqyARX2pxhqjGJbpNQwPUIAT7S61ESmcxArEwDm6N\\\\\\\/Y5Ng1YLmDkpVPQW9sC2JhJ+OiSsVjyEdbED4XQmCf+iLc74\\\\\\\/ZePH38kxYREqO10aXVy\\\\\\\/DdwyPCeRi7wqRKKAA5ERsNs6lBvez\\\\\\\/uDR2fQttF\\\\\\\/GaeZPhnIpiQM\\\\\\\/HeP7X\\\\\\\/5HT8I7Iku\\\\\\\/5picOybqbDIZkEf9zrN1jfunSf7RYwiUmLsXyf7IrZNb3uSsw+Dw0SIkU1CBs1Tr9lQ7kEff\\\\\\\/zFwkMGw\\\\\\\/hU4rhzASpWjR0qFsgSiJaLO4ktdWGR20DuaKTREBxABtLUNUx23xn6pmrFDMF8AMviAe9Z\\\\\\\/SE7w8RcLwlqX6ZwsO7n3WjMaJcaIzvOVqzVHvdtOT8bQJ7c1W8hXx7H0muWbDRul1ID0BTpxxSZ7wUwRbTlg1PMuM78obuKWWGM4tTMG35LcbPRd0cRFBdWMcPAlgo0yzoZuVshtUfLt1cKpam5mHweaJmfZ+Dw\\\\\\\/xbHAqKdcOd\\\\\\\/nEiaP5C3zu6nS2S\\\\\\\/XumNvinwQgrAj00IRRyMyIqchGe7kTrbZ92K\\\\\\\/B0veS+1CncQDMTkpX42FvITfq7WtWmKOGO9sO5RKtDg4cwXsCJniYsf8jhTvc9FDegahn45NTPEL2OT8Tkz9Za1JJMzy8Dmuepos13pSGwVgd\\\\\\\/jkaEvLOUZspO0kwWbfOe18mY3cOrhPb7kukfU326edxUvUi57W\\\\\\\/4SbymUmqL\\\\\\\/wFxkLc7I\\\\\\\/dDLngdDiB+bNu6nirH\\\\\\\/NZO+eaN593EgSswCOCr0oMPHlp+XHH+fqskTi+Mx8Dmk2GzTn1oaVHuqEe+QBQ+12WzeBmBRWfrBn+\\\\\\\/dP5iNB0QzT4pnQw5QB05AwD4ySLeRbttmxy\\\\\\\/xOEgPBowN4B1r+1UF6r1+gJ7igeXMnLS6d1dpH4+yd3DNFvwfgUkBzzkTXxuI93sAnewg4\\\\\\\/Uk1Vb1J0fjJfGyIUf73GeUcykSY\\\\\\\/C3v5bPXD2BMmTZxbG8hD1rNaO5chNcBpFa6CBtQ2eJWLdqQCL\\\\\\\/NbzVE8uQpMtl0P3OBUyPgVrynG4KRXtvH2M91JqbuiUPU+aHrf3MCVKzXVlXzse8qE2e\\\\\\\/JjwPusxOy2krQn5pPfdRpUJyKanbt2hVXIHDuOYxly3z8g5PxxtUotlU9BhsOpgDtsmQGe5rHFFIJLJEjsewojap5wgB9WRfZfRvW9eMuGhp03qUOZ6lpoT5ZG8EtPYwRWGQn5fJg9B2P\\\\\\\/cEctq4jC9IyDh875U28B26vv07H3X4nc1eQDoOVpXcPRt8sEA482zfcqUAI5+dvdshfU\\\\\\\/1IV5TnF3g55sdeSLSUGTPZYFPMJfiuJSwM2Kr0QYxlpvsGKanWRRqhf9xd2Yv+vGdLNknNFqpCS3quw01PYPbfrjbupLoq6PhAJRprTh+Chic9k0lJqaR2+H9lD1vaQsaHIM9\\\\\\\/TEPPxlGWuk1CDDqy8LVCuPwbBRskuaSOjRqOR+mPcaKbfhyvscKZfNBkf2m7ZMujNV3r9Jdb08hQIQD6vEYHSGN82q2TBFLCb1eMXk\\\\\\\/+fRtheyYGNNYow69dwi1cY767RCYNYzulMSj48bD1vFodG6xXp2616zoyR0Wnvcr75ppwdh1rlSQuHGbuJW7t84UosHTOvszy3DFKhpG6YjMXa6omYc21uYEc\\\\\\\/pL0mpbUL2adMJkGrlXapwaQ0goEMl3oss2NC7A61a5oMUcuTC1\\\\\\\/dGSesQPbJZFKmZsQjlwVQgw3NitZHln41R8WAJQfdrUlqdiekYOm3+6G+bxwBrXzldxO4cgW9lrah6PFPC3aBKqPUlXcDmRcnCtqcWWuzejL+Wp96ckqImqF8UUFLWUW1gdZIsEWb19DdTKm65DMcgXhBlZltdtdgD0BYTJRjA2jJ\\\\\\\/OvgzezRwhChktrtOC3nlAs+\\\\\\\/gKYHxerqE5xHvtqyl3E3RYgbcuKqKufjn9rjD8pxH7glazR9w\\\\\\\/eABEkl6JkYHrbHOT8iys+R1l41Be5L+dClOIjd6wdcXctMqKn90kmvyM\\\\\\\/dAh0VnrTA4DugdlXWSTkN3fSooULNqKZjfIBX60deCe+NZVNe5BbIBrFthSTv\\\\\\\/cqzOIiiDTJL43ZAl8aaa0Cyrqw2ttaUdch82JY8JNIQMfKCLudehEgWSBZLCCJzQEBuk7ZpAbSjFvJQNaE95sN5RfXyfQKeo2ZcFMwxLbF+ufcYX5oY577VRuNzFyBQ3axnKJMsXIkzx\\\\\\\/9gL2qlbCq9TyARqKKG09qBwz3Q+ZMSIbWH1h98Gm9d4m97TkP8qIaIrxn\\\\\\\/RNyOCuvFdZqu+hQF7pqKqHJpWky1PqCzTUgj1JgCx9WLOa6gs0DVHatxFAoQyoMraY9MOna5gnGchZiTC3nxnkuUoNqyUvcFXsxKfbvbsSZmrD9atK20APJol9IVW70thl3ucTnx9hp\\\\\\\/DioaG9yM9\\\\\\\/S6DFmTxNruW7oRfVVvI5x7WdOJic249SzSWj7dDNL4BiySh7U95MgvnYHPqtOvair6u0XeWkaWtLjFtWRYwjhyOMfuc52jg5Rt8EnLYGM90uhem5SICELNVKajdF1tEJpGLrBKpeJKV7e8xWsiffmtFQOKwnhBLhBH5fY0Y9HHo6l6449d+PKGRoD8XPdhaQrysbyOXBu98gpJnwtZBY3+KSHN7V8BIOWbcaMVQ34tZjuf6aNq4NDRoN5iscKfPkoKH2dPhaiBhZDl6Sw8GZ9meCuGtDNoH5S07wlmq\\\\\\\/OiV3e6sUTFAE9sb5DVx0hLbdphA289FXj6fdqJj+tPky0+POM32iuger4AV+IRRkGFndwk+OX49CQJtmqgHAZ+lg4RUdjNYN1LWlhfQBtUEcO7lUCDSzFdVWP4tZqI91j3+yVojEBhXfL32asUxIicbEf0XB1q+wGjotf2fcJ78qGSr0qQ7pAhmR1PDY9DRaU5D4pQuO7cGoYWUXPoimbwJjKAZj93UxUOpdaEHIA\\\\\\\/26+gJIP\\\\\\\/48tZmQcXb4sZXaDrNZe17rffbVqoJo90PuHqlgTHxyCBZ7A+aek31D1xFwcajXSk8rXU5t0FMVuVW82EQp+hsigIu3rT4ke7ovgYxEAxdSTfiYWOy85hZLMMqI5\\\\\\\/cMs5jmhJonNy5P8Qbf8KcsBI5sRU6r5vVLNTQj4BxjZ6lsTaAIjnw7kge6LjFlPsYBk69RQCtSZv0m3+RMRwuSXk9GbEJKcENoQMGOHMfqgDgzS42gpnQIp+e7ev8ax9eEh5wDyevSwo+LfsCL0vodtHl4VNjdT11sqtBCc5oHZuxXJapcJpJ7EROZzD7Zb7kWL4j\\\\\\\/BSmVpnVnlhcecLwjk9hsMjSZ7kidXP113nx\\\\\\\/RmPH7OmJZ+oIb+3w5Iy7LC7mdeLWtYR0LUYvNYIqAqMByaXXmv9XXv0VB+mr8RP1jsffzkE1YFFYKWNzXnaLmaN\\\\\\\/pFWH926sBctMBPAxZCnC9ByDz0\\\\\\\/MzhDsZKTIjWjVke7VlwsclPJBu8OtUIibnhnGCLdd7fdt\\\\\\\/d4mZcwQt8ZoyYMpE0Tb+Zv0fyWVwfFGWtZyZqkHycY8hIdSgo92xg9buxS8dkIdbj4tQvP566qZ7lC870tvMWhGUJCw5KfBgZ0d7K64sHlepw6G9yu+lNm1IQaGFpKj1FCyXYYFGMOG7\\\\\\\/3xqLW5QT7qebUlTEpRuZdlqjYU2b6cblLneZAy6DFGKgjogcoZkbl9HSgbN9O+POr0QC+oMD4w80dgqZohMXbP8WNwbErqlBY4o6XLsbtmZERvyFpTfxA8+H8vlBMQL3lBKOpIRYYLjRkRurRaVS26l6gLMYwlXSEpkkL0+tZ1foa\\\\\\\/7XdKlLOR+qPZwBF1Qh4ISeLXjT95VBUi\\\\\\\/F5Nf4WZ0fr36VP8xHzh\\\\\\\/RUk48blKO5R\\\\\\\/ETOIMA6rlijCKvRZ5eeqlfYN3SL555cQrsx8mtqvM3ATQIjDF\\\\\\\/2JgtUxa0zmYkx6PEE7pXlBMHgxcweK2XnAvaQsxJ48sg\\\\\\\/BLuLTwETKd84TSA9p6AmoKc\\\\\\\/1dUY2\\\\\\\/HPV+OFqLVN3EcWUP8WulF214ICmb+4SyTkuSX79CvJKs3AquutIJ2tbYQq9NwTTiPEhZupvIDXOZb\\\\\\\/85FUxuGj1QvQt2pmkS7FLXakylPOC5CoMth5waTZxzHlTtBl8gmUjyKHubRtZF+VIZtA6KVveiO7E67iQasu\\\\\\\/WasMOQE2Qs7MyCnniyObwl3GYeNaSHy5ZZeWMXUtAk6BKrauj4Rypj\\\\\\\/jbm2kRFT1RdQdCJnfvMvcSnoQVV\\\\\\\/3oyPrGtrSBayIgM4FDeX0llI3vpld6hRYp9vTF1js7mPNlOJzzGtPzB5wXq9dUU8Cgncg+bGITU+reMUagLrgzTcxUIJBh\\\\\\\/LnuKnJ0Rhm1ZS6jvooc5se\\\\\\\/fqrIrDLssnUIrn9ZWFEkTaPleTWLebXMTmbVwJayq1CnO8ipW6BNbG7gjSjDtrK1pRbvCdblJWwQA+qDY8a\\\\\\\/V9KUyNWfm2IgrqOeTMaaLcO80lRBtx7RqyOmZ8qKMyWP2NozIAJ4RIiW2K58b8cGYOknUc\\\\\\\/YIqA99mRLvjAFgXs5+a8RW1I1CHAGPx8PEjKMkwx4WS8McAHfQhWKReqHdhewaS4qPC6J2wgXzKvr9j0sNJsgGcJh4jdcT0n3PwEZYVpn7A9tON3tejZOEb7L6Zwn3QzeA7fx5zyEIQDOWApF7jZHshg4MHqzNZOnOA7NAvgFkQJ6labXN66+5TNGhmDMnJL5v+ugorcBJk11B9uX7ozc8QRm\\\\\\\/4idOCNMmA7W6wYxZKylc+rQU9hk7y7NYP63QJ2weOdFOiu1+V\\\\\\\/d+xnUERBi1SQC3WEMBWTtNwlYl01pv\\\\\\\/rKgcsnU5E4ZInbAb2aaB6Gtx6X1BSSmsLFrtCjFnwi0s+440Ea0pb56qorxnLaJenq0mgUM2HZdtAOkV346KNBxZymaQRL3zrOfj\\\\\\\/YDwZipTlVUye63bN+DVjcoIzlE7r4zKfH4tnSn7CssmhzerPLBD0HQA22rBK44bw\\\\\\\/nIHFHkFf7MD71wKWCotSfk2ir\\\\\\\/kS0UcQPFZaw8NmG7R47DitnVpOXZjYQy8j8w0UB6\\\\\\\/jF7zAezIvBPGXnoGxX3eUCS60LbYVDOSRHdIdxwbEt6HfkH9K3IGvGMO4vGU68Go6gcEh70DRyiputNGqvNJItV5rouzLE7xJTjPVN1mq9lCsW7p3MX+sEtrFdkC6y\\\\\\\/Ix5EersFbABYYsD40RGrSyHuvT\\\\\\\/K\\\\\\\/a1UyS48csaH1q1V23l\\\\\\\/LD\\\\\\\/cH6ZQyPxTm8lUKYaETPbNe1O+brTVjhmDEPIyLD9+8w7If+FnRdxqR1GlwUSDDVtaZumulqsLj\\\\\\\/RdVmQ2B5neouZ9SYqz82Y6gQnStq4vYihRNirQz0BO1B1tauEuEkTEG3gk72x0iLMsZSM2RzrtTf7E\\\\\\\/T2rjcmXleACx4jl41VIiORJEGj0mCJlUU3jteRQGTJhPOoGSZwJbDnhU1+DOk85HbRs2pr1jipONaNgHUlqexKfIqtf6KTZIb7SwZGMlZuEZDv8iTRk1a1oBOL4\\\\\\\/mD+Vwb1rSfP46vBOtagnk4v1vWYc2bKso+s5UtlUMzzk+CxLVdKBH2sNuBFEELk51++PU2+zUyXwmcRZrWWiUhx5kqRwZGIBS0wMq3B2V93Xyl\\\\\\\/GVW7Fq6uomUE374SLdvMuVwekTcyKSON0UJqw73q5ZxMDsf2EkXuB9\\\\\\\/QzNLu6LdUGse4hVvtsE3oQWVlCZZeOph7KrApglfDXhurwBWuSuAO8XLKuq\\\\\\\/DTJoBZ0rncvyuq9LiYM0ohlsJFW8+mXEslCOYHQSh+ozEY6+LZyBUPumr5XOhAp0ttQJaR6mm1Oe2XAbE0C0uTHzzWfYCnt88E\\\\\\\/cl3hN9CJovolpbZvUZ0qhtNpyI7LFXtei3BWZ7CSosiLWyrpArwwcudmKrQFxLtFa0KlwXjq5Kpilo5loxN70HG5I1mKzyWIdCCB+RIQBeWpratdtghwQ0\\\\\\\/B2ejcLwkCyf9LvUZfGsqyHQAlcT\\\\\\\/oqw0thm5hy36z22zdWNXoMsac4cAbh7Ok\\\\\\\/VTVxGWL4w7\\\u0022,\\\u0022iv\\\u0022:\\\u0022kcrUh2SRWVf3zxxDEnFjWg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022829097b405cd432608ca1f7aecfe68980a0a19e49bd5e460d099bdf46548c16a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z+BH1cGx+HECdpdh\\\\\\\/Wqb9ug2lICTbiWpPETtX35\\\\\\\/jTAi6ttRVM9re8GlSauoqUWHVivIvet6zckoLw\\\\\\\/ZZCitzQzcGynwIONIOqu+pE1eAtN2zoN7PHBqmi5xo8lWp4SwjAKEHrxpofwUZKeWaH4KmUGLLGsjB+t5oQu6gjVfCpMsxhGyCdMeTnlU6b\\\\\\\/tJMBfIBcqf3La0uvHZcERMXvKvBxTwdyr9gBRY729iQ0Um\\\\\\\/Vc0HuNiJSaaK\\\\\\\/1Ss4cOEY3NX0UUbcOiJj7t7fUXOodosfdixJOISCyIHATSQAAJx0eSrivYmeqd+gnofMnNgblhJcyDmIQXSEE02xulTfFP68vN5Ip3zbS88btj9pR+w7ibxhgUnEA2ti+Ff5tbEOYLdqyZpLiARPhFPnwRl0QAH6YWpgsYuK89CREIfaLU7xbumZXgFwIquCdUXyqcyHXczCVvRkrF6LV19BI5k8\\\\\\\/PPAJrzFtCja7hBp\\\\\\\/7eodQpzPW7Cvrk0oJC5nwhvKmCXAzCsgR4e4EOKSZyUE\\\\\\\/jZZ5ggvofYIBiTOcBrasXB+kIF7xVeVf1q6YLG\\\\\\\/c9M5vBirTG4ZWy6bqxYzpUtmc7QhpanN21tlKGBMyIITi+kjoF5jA1nGJjJJnJXA5rYOrPDVLJl\\\\\\\/eNc2GiWp7kH\\\\\\\/78i2OCSFOGXI3lSIMGG2C+ju+uTbwbGXsRviRUWLYBFhQ2AvzcYWxWeAyKsCDjuPgPn7SytXXFGDFQ8NzSCrgPHEQ47ZvkoCoH87mDDWGDoJU3zPMLI6u5Pfwc4LCfMy3\\\\\\\/yJ6mh5hklfIAkpJFYnLLNDt0dZPvWk0b6ibGkoq4cHBuxN35DgggjNflLQyHa564U+RFZCTHFYFEMtuvMejq1EFXVK61ZysCFHxJYZOTEVTxz86u9\\\\\\\/VDWpubSkRVyS27sZV9EBAotQXUuzL6TMFF1brSxZ3x05WCzggJc5caASaQHFlLBWU+fNuwnQPnpjVscZBIif0zQvkBLMloZdv4oueaHCXNfTQiKd+jnh7qEGAkcjRECSR+C\\\\\\\/9ma84OeQ2Itj1+TaeKBQqcm\\\\\\\/6hTjqTyGEJFeKgwNrW98UvHCPzvnf\\\\\\\/gfwulwhuPaA0gLvGqgVVR6MM0bKWEU4XEZ3GV3ID8KcYMPCx1Wxuv4VwhGaW98BApbI2\\\\\\\/uxbfQM5xBMzeCHlSL5Sc387mgvJXxiR2uoMARK7ztw+O4t0KV2tmBJEh64PhVBx4JDMDj5SSXMoFOCzQSVGqpfeIV34YfOT\\\\\\\/HjC54j20x\\\\\\\/3qWBExEi4sHnvx9AHT6kdJFUH9gEp47pnAkNcYXV0kN5aJGTLmjIg8cK1aqr7\\\\\\\/6BvR59x3TXpW2E1lBULYVudkIqlzctLKDZC0MGvMpnfEfrDmUta6iJxBHkcJQRlfYggHH4pxgpOInScznuuCD3t0ZcbiHpL3o51agYgboIeWvVIJaKIdm4qcTeMbMRXYOqE96weW2zpEehML\\\\\\\/Sl6bnquQ\\\\\\\/sjj9IqN6XeQai4AUY4me2R08+\\\\\\\/8HKQeoi8QoDfg0stg\\\\\\\/Fm3taIE1YIEQI7sxAFtD9F\\\\\\\/X\\\\\\\/9\\\\\\\/NFy5PyzTXWuRVhnQQ\\\\\\\/0TDLxHKVMNu0aUdEiSZSFm3AvhSE2CZGjrdNU22pLBxOHxVs++V1c+BpsB6FfNBNF3k8zjUAlSTYcVEzJV7+FE\\\\\\\/rkGSe0cT5x\\\\\\\/OhBD5ebMpScyEdNMGE4xt725VkR26zmvDlam2THFP3uZ8DtzLZPIp\\\\\\\/oJHCts4doK5ACFsDTBD\\\\\\\/jdmUZJp+XarxtgzgUD7deGp5rvbpfwGJTxC70eAmcDwwjt+LYnCEEh57Mt1yx20iZK96Lwq6Q3bMAOYegiipnB4P6Fo0\\\\\\\/QbvgtSigy7lkFs\\\\\\\/n6GPPvjA6jUlEAeq35o1HLSA1NEiDnLYJXesn3eWhwpqK7n3Ta7nnUBjQo9hOSiVerGgqfZzW3vkBrDUb0Xz+BSc4Ui0L4VqUHQT5cNNUY0tlSXh14rafZdz7QKV+awN10Rd1cVCXsexU0V1m5pgsZ5zlQHEsF5QBTVi4RiFB+vCgmzyoOVs4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022MaE4p10adPzRJWnDjzBdZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220f5b0f5d1e8a811c1a3c0f4e53402e4215021dce5e007d8da585cc2e90e672e2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460548 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/119" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fm66hZT1a7Sq7oDOsoltimVLCpmhQtlVSsadELwvVQI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/119" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Pkvf7pi9bZFWvfnbEEzKC5M94CN_LkT80Fz_oeMkWZk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/109" title="A New Chicken Genome Assembly Provides Insight into Avian Genome Structure" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/129" title="The Neurospora Transcription Factor ADV-1 Transduces Light Signals and Temporal Information to Control Rhythmic Expression of Genes Involved in Cell Fusion" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460548" id="node460548" data-pisa="ggg;7/1/119" data-pisa-master="ggg;g3.116.036012" data-apath="/ggg/7/1/119.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Monika Brandt</span>, <span class="highwire-citation-author" data-delta="1">Muhammad Ahsan</span>, <span class="highwire-citation-author" data-delta="2">Christa F. Honaker</span>, <span class="highwire-citation-author" data-delta="3">Paul B. Siegel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2722-5264" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Örjan Carlborg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">119-128;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036012 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460548" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Monika Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Monika%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMonika%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Muhammad Ahsan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Computational Genetics, Department of Clinical Sciences, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Muhammad%2BAhsan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahsan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMuhammad%2BAhsan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christa F. Honaker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christa%2BF.%2BHonaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Honaker%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChrista%2BF.%2BHonaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul B. Siegel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BB.%2BSiegel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siegel%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BB.%2BSiegel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2722-5264" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Örjan Carlborg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:orjan.carlborg@imbim.uu.se" class="" data-icon-position="" data-hide-link-title="0">orjan.carlborg@imbim.uu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/1/119" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460548" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460548/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/119.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460548" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460548/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/119.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460548" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460548/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/1/119.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460548" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460548/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/1/119.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460548" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460548/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The Virginia chicken lines have been divergently selected for juvenile body weight for more than 50 generations. Today, the high- and low-weight lines show a &gt;12-fold difference for the selected trait, 56-d body weight. These lines provide unique opportunities to study the genetic architecture of long-term, single-trait selection. Previously, several quantitative trait loci (QTL) contributing to weight differences between the lines were mapped in an F<sub>2</sub>-cross between them, and these were later replicated and fine-mapped in a nine-generation advanced intercross of them. Here, we explore the possibility to further increase the fine-mapping resolution of these QTL via a pedigree-based imputation strategy that aims to better capture the genetic diversity in the divergently selected, but outbred, founder lines. The founders of the intercross were high-density genotyped, and then pedigree-based imputation was used to assign genotypes throughout the pedigree. Imputation increased the marker density 20-fold in the selected QTL, providing 6911 markers for the subsequent analysis. Both single-marker association and multi-marker backward-elimination analyses were used to explore regions associated with 56-d body weight. The approach revealed several statistically and population structure independent associations and increased the mapping resolution. Further, most QTL were also found to contain multiple independent associations to markers that were not fixed in the founder populations, implying a complex underlying architecture due to the combined effects of multiple, linked loci perhaps located on independent haplotypes that still segregate in the selected lines.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/imputation-based-association" class="hw-term hw-article-keyword hw-article-keyword-imputation-based-association" rel="nofollow">imputation-based association</a></li><li class="kwd"><a href="/keyword/advanced-intercross-body-weight-line" class="hw-term hw-article-keyword hw-article-keyword-advanced-intercross-body-weight-line" rel="nofollow">advanced intercross body weight line</a></li><li class="kwd"><a href="/keyword/virginia-chicken-lines" class="hw-term hw-article-keyword hw-article-keyword-virginia-chicken-lines" rel="nofollow">Virginia chicken lines</a></li><li class="kwd"><a href="/keyword/qtl-fine-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-fine-mapping" rel="nofollow">QTL fine-mapping</a></li></ul><p id="p-5">Long-term selective breeding of animals and plants for extreme phenotypes has resulted in genetically distinct lines that are a valuable resource for dissecting the genetic architecture of complex traits (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hill 2005</a>). Most traits of interest in animal breeding (<em>e.g.</em>, production of eggs or meat, resistance to disease) are influenced by a combination of genetic and environmental factors. Due to their multi-factorial nature and despite the ability to obtain data on both genome-wide genetic markers and phenotypes from large numbers of individuals, it is challenging to disentangle their genetic architecture by analyzing data from the commercial populations. An alternate strategy is to make use of experimental populations resulting from long-term selection experiments, where the focus has been to develop divergent lines from a common base population using more coherent selection criteria. Such populations will display larger phenotypic differences than populations subjected to composite, commercial breeding programs and hence facilitate in-depth studies of the genetic basis underlying the selection response and general genetic architecture of these traits (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andersson and Georges 2004</a>). Given that many of the agriculturally important traits are related to metabolism, feeding-behavior and growth, they also provide a good model for translational studies to decipher the genetic architecture of traits of interest in human medicine, including obesity, eating disorders, and diabetes.</p><p id="p-6"><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kemper <em>et al.</em> (2012)</a> recently reviewed the literature on the genetic basis of body size and highlighted how complex the genetic architectures of body size are in species with contributions by many loci with large, intermediate, and small individual effects. Also within species, the genetic basis of variations in body size among strains of mice (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Valdar <em>et al.</em> 2006</a>), breeds of cattle (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Saatchi <em>et al.</em> 2014</a>), pigs (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Yoo <em>et al.</em> 2014</a>), and chickens (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Van Goor <em>et al.</em> 2015</a>) is often polygenic and due to polymorphisms with modest individual effects. Studies of experimental crosses from artificially selected populations with extreme body sizes in the mouse (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bevova <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Parker <em>et al.</em> 2011</a>) and chicken (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sheng <em>et al.</em> 2015</a>) using, for example, chromosome substitution strains (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bevova <em>et al.</em> 2006</a>) and advanced intercross lines (AILs) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darvasi and Soller 1995</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Parker <em>et al.</em> 2011</a>) have revealed that the responses to selection in these populations has resulted from selection on highly complex and polygenic genetic architectures.</p><p id="p-7">The Virginia lines are experimental populations established in 1957 to study the genetic effects of long-term (&gt;50 generations), divergent, single-trait selection for 56-d high (HWS) or low (LWS) body weight in chickens (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dunnington and Siegel 1996</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Márquez <em>et al.</em> 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dunnington <em>et al.</em> 2013</a>). The lines originated from the same base population, composed by crossing seven partially inbred White Plymouth Rock chicken lines, and today display more than a 12-fold difference in body weight at 56 d of age (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Márquez <em>et al.</em> 2010</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dunnington <em>et al.</em> 2013</a>). In addition to the direct effects of selection on body weight, the selected lines also display correlated selection responses for a range of metabolic and behavioral traits including disrupted appetite, obesity, and antibody response (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Dunnington <em>et al.</em> 2013</a>).</p><p id="p-8">The Virginia HWS and LWS lines have been used extensively for studying the genetic architecture of body weight and other metabolic traits. These studies have uncovered a number of loci with minor direct effects on body weight, metabolic traits, and body-stature traits by quantitative trait loci (QTL) mapping in an F<sub>2</sub> intercross (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Park <em>et al.</em> 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> 2009</a>). Also, a network of epistatic loci has been found to make a significant contribution to long-term selection response through the release of selection-induced additive variation (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carlborg <em>et al.</em> 2006</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Le Rouzic <em>et al.</em> 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Le Rouzic and Carlborg 2008</a>). Explorations of the genome-wide footprint of selection by selective-sweep mapping suggests that perhaps &gt;100 loci throughout the genome have contributed to selection response (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Johansson <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pettersson <em>et al.</em> 2013</a>), and many of these contribute to 56 d body weight (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sheng <em>et al.</em> 2015</a>).</p><p id="p-9">To replicate and fine-map the body weight QTL inferred in the F<sub>2</sub> intercross, we developed, genotyped and phenotyped for body weight at 56 d of age (BW56), a nine-generation AIL. This large AIL originated from the same founders as the F<sub>2</sub> intercross, but was selectively genotyped at a higher resolution (∼1 marker/cM) in nine QTL (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>). In this population, most of the original minor (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>) and epistatic (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pettersson <em>et al.</em> 2011</a>) QTL were replicated and fine-mapped. These earlier studies analyzed the data using a haplotype-based linkage-mapping approach in a variance-component based model framework to infer single-locus effects (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>) or a fixed-effect model framework assuming fixed alternative alleles in the two founder lines for detecting epistasis (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Pettersson <em>et al.</em> 2011</a>). The variance-component model was used in the replication study to avoid the assumption of allelic fixation in the founder lines. By implementing it in a Flexible Intercross Analysis modeling framework (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rönnegård <em>et al.</em> 2008</a>), it was expected to improve power when the parental lines carry alleles with correlated effects (<em>e.g.</em>, multiple alleles with similar effects).</p><p id="p-10">Although the initial studies mapped QTL under the assumption of fixation, or an effect correlation, of divergent alleles in the parental lines, the results at the same time implied that multiple alleles might be segregating in several of the mapped regions. To this end, the first QTL replication study in the AIL population (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>) found a large within founder line heterogeneity in the allelic effects. Later the selective-sweep studies, which utilized data from multiple generations of divergently selected and relaxed lines, identified ongoing selection and multiple sweeps in many QTL (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Johansson <em>et al.</em> 2010</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Pettersson <em>et al.</em> 2013</a>), as well as extensive allelic purging (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Pettersson <em>et al.</em> 2013</a>). This allelic heterogeneity challenges attempts to dissect the architecture of the selected trait via, <em>e.g.</em>, QTL introgression (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ek <em>et al.</em> 2012</a>). Alternative approaches are therefore needed to uncover multi-locus, multi-allelic genetic architectures in QTL and their contributions to the long-term response to directional selection.</p><p id="p-11">In this study, we explore an imputation-based association-mapping strategy for further dissection of previously mapped and replicated QTL (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Pettersson <em>et al.</em> 2011</a>). For this, we made use of available high-density (60K SNP-chip) genotypes for founders (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Johansson <em>et al.</em> 2010</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Pettersson <em>et al.</em> 2013</a>) and intermediate-density SNP-genotypes in several QTL in the entire nine-generation AIL pedigree. By increasing the marker density in the QTL throughout the AIL by imputation, we aimed to better capture the effects of segregating haplotypes within and between the divergently selected founder populations than with the previously used markers. This aim can be achieved as the original markers genotyped in the AIL were selected to identify high- and low-line derived alleles, and not alleles that segregate within or across the founder lines. By testing for association between imputed markers and body weight, the fine-mapping analyses were less constrained by the original selection of markers and facilitated a more thorough exploration of the genetic architectures of the nine evaluated QTL. A single-marker association analysis was first used to identify regions with candidate associations. These were then simultaneously analyzed using a backward-elimination approach with bootstrapping to identify statistically independent signals that were robust to the effects of markers elsewhere in the genome and the pedigree-structure in the population. In regions where the signals were robust to the pedigree-structure, the results from the single-marker association analysis were used to fine-map the region. Our imputation-based approach replicated most QTL and also improved the resolution in the fine-mapping analyses by not only using the recombination events in the AIL, but also the historical recombinations in the pedigree. We found that several of the original QTL are likely due to the combined effects of multiple linked loci, several of which are segregating in the founder lines of the AIL.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-12">The Virginia chicken lines are part of an ongoing selection experiment to study the genetics of long-term, single-trait selection (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Márquez <em>et al.</em> 2010</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Dunnington <em>et al.</em> 2013</a>). It was initiated in 1957 from a base population, generated by intercrossing seven partially inbred lines of White Plymouth Rock chickens. From the offspring of the partially inbred lines, resulting from the intercrossing, the birds with the highest and lowest 56 d body weights (with some restrictions), respectively, were selected to produce the high- and low-weight selected lines (HWS and LWS) (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Márquez <em>et al.</em> 2010</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Dunnington <em>et al.</em> 2013</a>). Since then, the lines have undergone divergent selection for increased and decreased body weights with one new generation hatched in March of every year.</p><p id="p-13">An AIL was founded by reciprocal crosses of 29 HWS and 30 LWS founder birds from generation 40 (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>). The mean, sex-averaged 56 d body weights for HWS and LWS at this generation were 1522 g and 181 g, respectively. Repeated intercrossing of birds was used to develop a nine-generation AIL consisting of generations F<sub>0</sub>–F<sub>8</sub>. In each generation, ∼90 birds were bred by paired mating, genotyped, and weighed at 56 d of age (BW56). In total, the AIL population consisted of 1536 F<sub>0</sub>–F<sub>8</sub> individuals with complete records on pedigree and genotypes (see Genotyping), and 1348 F<sub>2</sub>–F<sub>8</sub> individuals with juvenile body weight (BW56) records.</p></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-14">The complete AIL pedigree (1536 birds) had earlier been genotyped in nine selected QTL for 304 SNP-markers that passed quality control as described in <a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> (2011</a>). Further, 40 of the founders for the pedigree (20 HWS and 20 LWS) had also earlier been genotyped using a whole genome 60K SNP-chip (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Johansson <em>et al.</em> 2010</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Pettersson <em>et al.</em> 2013</a>). The 6607 markers from the SNP-chip that were informative and passed quality control in that study are located in the nine QTL regions targeted in this study. When merging the information from the 60K SNP-chip and the information from the 304 markers genotyped earlier, 55 markers in 40 founders were genotyped using both methods. Out of these 55 markers, 28 markers with genotype inconsistencies between the genotyping technologies were removed during quality control. In total, our analyses were based on 6888 markers, where 40 of the 59 AIL founders had genotypes for all markers, and the remaining individuals in the pedigree had genotypes for 281 markers. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows how these markers are distributed across the nine QTL regions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470130/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470130/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genotyped and imputed markers across the nine analyzed QTL</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Phasing and imputation of markers</h3><p id="p-22">All genotyped markers in the QTL (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) were first ordered according to their physical location in the chicken genome assembly of May 2006 (<em>galGal3</em>). In the ordered marker set, the SNP-chip markers were evenly distributed in the intervals between the sparser set of markers genotyped across the entire AIL.</p><p id="p-23">Using the software ChromoPhase (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Daetwyler <em>et al.</em> 2011</a>), we phased and imputed genotypes for the complete set of 6888 markers across the entire AIL pedigree. ChromoPhase first phases large segments of chromosomes, in our case the QTL regions. It then imputes the missing genotypes in the AIL individuals genotyped with the sparse set of markers from the genotype information available in high-density genotyped founders utilizing the pedigree information. It thus predicts both phased haplotypes across the nine studied QTL and genotypes at markers that were only genotyped in a subset of the founder individuals in the pedigree.</p></div><div id="sec-5" class="subsection"><h3>A two-step fine-mapping approach accounting for population structure</h3><p id="p-24">Earlier studies have shown that the genetic architecture of body weight is highly polygenic in the Virginia lines (<em>e.g.</em>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Siegel 1962a</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">b</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> 2009</a>; <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Johansson <em>et al.</em> 2010</a>; <a id="xref-ref-6-10" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Pettersson <em>et al.</em> 2011</a>, <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">2013</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Sheng <em>et al.</em> 2015</a>). We therefore implemented a forward-selection/backward-elimination procedure with a termination criteria suitable for a polygenic trait in a bootstrap-based framework to correct for population structure in the AIL (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Valdar <em>et al.</em> 2009</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Sheng <em>et al.</em> 2015</a>). As all markers with genotypes could not be included in a backward-elimination analysis due to the limited sample size, we first used a forward-selection based single-marker association analysis to identify a smaller set of statistically suggestive independent signals within each QTL region. The backward-elimination analysis (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Valdar <em>et al.</em> 2009</a>; <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Sheng <em>et al.</em> 2015</a>) was then used to identify associations robust to possible influences of genetic dependencies (linkage or LD) between markers within the QTL or population structure in the AIL (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Peirce <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 2010</a>).</p><div id="sec-6" class="subsection"><h4>Single-marker association analyses:</h4><p id="p-25">The <em>qtscore</em> function in the GenABEL package (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aulchenko <em>et al.</em> 2007</a>) was used to test for association between body weight at 56 d of age and, genotyped or imputed, individual genetic markers within the targeted QTL. The allelic effect of each marker, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-1.gif"/></noscript></span></span> was estimated using a regression model (Model 1), where the genotype at each marker was coded in <strong><em>Z</em></strong> as 0 if homozygous for the major allele, 1 if heterozygous, and 2 if homozygous for the minor allele. Sex and generation were added as categorical covariates, with two different levels for sex and seven different levels for generation, defined for each individual in <strong><em>X</em></strong>. The phenotype, body weight at 56 d of age, is given in the numerical variable <strong><em>y</em></strong>.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-2.gif"/></noscript></span> </span><em>ε</em> was assumed to be iid and normally distributed around 0 with variance <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-3.gif"/></noscript></span></span> <em>μ</em> is the intercept, which in this model represented the mean body weight at 56 d of age for individual F<sub>2</sub> females. The associations for the individual markers from this model were used for comparisons to results from earlier linkage-mapping analyses to fine-map the QTL in this pedigree that did not account for the possible effects of pedigree-structure (Model A in <a id="xref-ref-6-11" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>). Further, they were also used to evaluate the resolution of regions with associations robust to the pedigree-structure in the population (described in detail in Results).</p><p id="p-26">Next, a forward-selection analysis was performed by scanning across all markers within each QTL using Model 1. If any of the markers were nominally significant (<em>P</em> &lt; 0.05) in the scan, the marker with the strongest association was added as a covariate in the model. This procedure was repeated until no additional significant markers were detected. The markers from this analysis with an allele-frequency &gt; 0.10 in the population were subjected to the full backward-elimination analysis described in the next section.</p></div><div id="sec-7" class="subsection"><h4>A multi-locus association analysis to identify regions with associations that are robust to the pedigree-structure in the population:</h4><p id="p-27">In short, we used a bootstrap-based backward-elimination model selection framework (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Sheng <em>et al.</em> 2015</a>) across the markers selected by forward-selection in the QTL. An adaptive model selection criterion controlling the False Discovery Rate (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abramovich <em>et al.</em> 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gavrilov <em>et al.</em> 2009</a>) was used during backward-elimination in a standard linear model framework, starting with a full model including the fixed effects of sex and generation, and the additive effects of all markers (Model 2):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-4.gif"/></noscript></span> </span>where phenotype, sex, and generation were coded as described for Model 1 and where <em>ε</em> again is assumed to be iid and normally distributed around 0 with variance <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-5.gif"/></noscript></span></span> The intercept, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-6.gif"/></noscript></span></span> represents the mean body weight at 56 d of age for female individuals from the F<sub>2</sub> generation. In Model 2, genotypes were coded based on the line-origin of the alleles at each locus. Genotypes of individuals homozygous for the major allele in the AIL founders from the high-weight selection line were coded as 1 at that locus. If an individual was heterozygous, its genotype was coded as 0. Genotypes of individuals homozygous for the allele corresponding to the major allele in AIL founders from the low-weight selection line were coded as −1. By coding genotypes in a −1, 0, and 1 manner, the estimates of the marginal allele-substitution effects, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-7.gif"/></noscript></span></span> from fitting Model 2 will be negative if the allele that is at highest frequency in the high-weight line decreases weight or if the allele with highest frequency in the low-weight line increases weight.</p><p id="p-28">Convergence was based on a 20% False Discovery Rate (FDR) level. The analysis was performed using bootstrapping with 1000 resamples. Markers with an RMIP (Resample Model Inclusion Probability) &gt; 0.46, as suggested for an AIL generation F<sub>18</sub> (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Valdar <em>et al.</em> 2009</a>), were included in the final model. The FDR in the final model was confirmed using the original FDR procedure described in <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benjamini and Hochberg (1995</a>) as implemented in the <em>p.adjust</em> function in the R stats-package (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">R Development Core Team 2015</a>). The additive genetic effect for each locus was estimated using the multi-locus genetic model described above (Model 2). The contribution of a set of <em>n</em> associated markers to the founder line difference was calculated as <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-8.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-9.gif"/></noscript></span></span> is the allele-substitution effect for marker <em>i</em>, and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/1/119/embed/mml-math-10.gif"/></noscript></span></span> are the frequencies of the major AIL allele at marker <em>i</em> in the HWS and LWS founders, respectively.</p></div></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-29">Genotype, phenotype, and pedigree data are included in the supplemental files. Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036012/-/DC1/FileS1.pdf">File S1</a> contains detailed descriptions of all supplemental data files. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036012/-/DC1/FileS2.zip">File S2</a> contains the genotypes, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036012/-/DC1/FileS3.zip">File S3</a> the pedigree, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036012/-/DC1/FileS4.zip">File S4</a> the phenotypes.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><p id="p-30">We compared the results of the imputation-based association analyses with the previously reported results from the linkage-based analysis of the same nine QTL in <a id="xref-ref-6-12" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> (2011</a>). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the statistical support for association and linkage to BW56 across the QTL. The significances for all the genotyped and imputed markers from the single-marker associations are provided together with the results from model A in <a id="xref-ref-6-13" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> (2011</a>) that were also obtained without correction for population structure. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> also highlights those regions that contain associations robust to the pedigree-structure in the bootstrap-based forward-selection/backward-elimination analyses (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Overall, the results from these three analyses overlap well. Together, they show that most regions with strong associations in the single-marker association were robust to the pedigree-structure and that the association analysis approach using imputed genotypes for SNPs suggests that several QTL were likely due to multiple linked loci. In the sections below, these results are described and discussed in more detail.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/1/119/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between linkage- and association-based fine-mapping analyses of nine QTL in an AIL chicken population. Green lines show the statistical support curve (score statistics from Model A) for the linkage-based mapping study of (Besnier et al. 2011) and the red dots associations to each analyzed marker in the new imputation-based association analysis (this study). The green and red horizontal dotted lines indicate the experiment-wide significance threshold for earlier linkage-analysis and the nominal significance in the imputation-based association analysis, respectively. Arrowheads under the x-axis indicate the position of markers identified as experiment-wide significant (20% FDR) in the bootstrap-based backward-elimination procedure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1028048557" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison between linkage- and association-based fine-mapping analyses of nine QTL in an AIL chicken population. Green lines show the statistical support curve (score statistics from Model A) for the linkage-based mapping study of (Besnier &lt;em&gt;et al.&lt;/em&gt; 2011) and the red dots associations to each analyzed marker in the new imputation-based association analysis (this study). The green and red horizontal dotted lines indicate the experiment-wide significance threshold for earlier linkage-analysis and the nominal significance in the imputation-based association analysis, respectively. Arrowheads under the &lt;em&gt;x&lt;/em&gt;-axis indicate the position of markers identified as experiment-wide significant (20% FDR) in the bootstrap-based backward-elimination procedure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/1/119/F1.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/1/119/F1.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/1/119/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/1/119/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470108" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Comparison between linkage- and association-based fine-mapping analyses of nine QTL in an AIL chicken population. Green lines show the statistical support curve (score statistics from Model A) for the linkage-based mapping study of (<a id="xref-ref-6-14" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>) and the red dots associations to each analyzed marker in the new imputation-based association analysis (this study). The green and red horizontal dotted lines indicate the experiment-wide significance threshold for earlier linkage-analysis and the nominal significance in the imputation-based association analysis, respectively. Arrowheads under the <em>x</em>-axis indicate the position of markers identified as experiment-wide significant (20% FDR) in the bootstrap-based backward-elimination procedure.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470164/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470164/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimated additive effects and standard error for experiment-wide independent association signals, between body weight at 56 d of age and genotype, identified in a bootstrap-based approach implemented in a backward-elimination model selection framework across the markers in the genotyped QTL</span><div class="sb-div caption-clear"></div></div></div><div id="sec-10" class="subsection"><h3>Four statistically independent associated markers in the GGA7 QTL Growth9</h3><p id="p-40">The QTL <em>Growth9</em> on GGA7 (<em>Gallus Gallus</em> Autosome 7) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>; 10.9–35.5 Mb) was the only QTL that reached genome-wide significance in the first F<sub>2</sub> intercross between the HWS and LWS lines (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>). It was later identified as a central QTL in an epistatic network explaining a large part of the difference in weight between HWS and LWS lines (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Carlborg <em>et al.</em> 2006</a>). In the earlier fine-mapping analysis, the linkage signal covered most of the QTL region (from 15 to 35 Mb), but subsequent analyses showed that two independent loci were segregating in the region (<a id="xref-ref-6-15" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>). The signal in the imputation-based association analysis performed here is more focused, with a highly significant signal in a 2.8 Mb region between 23.7 and 26.4 Mb. This region overlaps with the strongest signal in the linkage-scan and is tagged by a single imputed marker (<em>rs16596357<sup>i</sup></em>; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) in the multi-locus analysis accounting for population structure. The major allele in the HWS line (<em>P</em> = 0.67) increases weight by 18.9 (SE 5.6) g, and it still segregates at an intermediary frequency (<em>P</em> = 0.50) in the LWS line. Previously, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ahsan <em>et al.</em> (2013</a>) explored potential candidate mutations in the QTL and found two regulatory SNPs near the peak at 21 Mb (21.6 and 22.7 Mb) and a synonymous-coding SNP in a CpG island in an exon of the Insulin-like growth factor binding protein 2 (<em>IGFBP2</em>) gene in the middle of the major association peak at 24.8 Mb. In addition to the strong association around 24 Mb, the association analysis also highlights two additional regions (centered around 18 and 29 Mb). A single imputed marker (<em>rs14611566<sup>i</sup></em>; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) is retained in the first region in the multi-locus analysis accounting for population structure. This marker has an estimated allele-substitution effect of −20.1 (SE 5.1) g, but as it segregates at equal, intermediary frequencies in the HWS and LWS lineages (<em>P</em> = 0.50) it did not contribute to the founder line difference. Two linked imputed markers are kept in the third region (<em>rs10727581<sup>i</sup></em>; <em>rs317586448<sup>i</sup></em>; 2.6 Mb apart; <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Here, the first marker is nearly fixed for one allele in the LWS line (<em>P</em> = 0.93), but segregates at an intermediary frequency in the HWS line (<em>P</em> = 0.61). At the second marker, the major allele in the HWS (<em>P</em> = 0.74) segregates at an intermediary frequency in the LWS (<em>P</em> = 0.41). Due to the close linkage between the associated markers, it is difficult to interpret their individual effects and disentangle whether the detected associations are due to the LD-pattern of multiple closely linked loci or a single locus with multiple segregating alleles. The peak in the F<sub>2</sub> QTL overlaps the major 23.7–26.4 Mb association peak detected in this analysis (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> 2009</a>). Due to the low differences in allele frequencies between the associated markers in the three regions, their total contribution to the founder line difference is small (8 g) amounting to only about 10% of the original estimated F<sub>2</sub> QTL effect of 86 g (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> 2009</a>).</p></div><div id="sec-11" class="subsection"><h3>Two statistically independent associated markers in the GGA1 QTL Growth1</h3><p id="p-41">The strongest association in the study by <a id="xref-ref-6-16" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> (2011</a>) was found on GGA1 in the QTL <em>Growth1</em> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>; 169.6–181.1 Mb). Here, the second strongest association was detected in that QTL. The imputation-based association analysis highlights two significant associations separated by a region of very low association and both associations remained in the multi-locus analysis accounting for population structure (the imputed marker <em>rs13968052<sup>i</sup></em> and the genotyped marker <em>rs14916997</em> at 170.6 and 173.7 Mb, respectively). The strongest of these association-peaks was located near the peak detected using the earlier linkage-based analysis. Several of the significant associated markers were located in this region (173.6–175.3 Mb). A candidate gene for growth, Asparagine-linked glycosylation 11 homolog gene (<em>ALG11</em>), is located at 174.6 Mb and has a strong mutation in its regulatory region (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Ahsan <em>et al.</em> 2013</a>). The second association was found to a group of significant markers in a narrower region upstream from the main linkage-peak (170.3–171.7 Mb). The association analysis thus suggests that the original 10.6 Mb QTL region that has its peak between markers located at 175.2–177.7 Mb is due to the effects of two separate loci located in these confined 1.5 and 1.8 Mb regions. As the two associated markers are closely linked in this population, it is difficult to interpret their individual effects, but their total contribution to the founder line difference (37 g) is about 75% of that by the original <em>Growth1</em> QTL as estimated in the F<sub>2</sub> analysis (49 g; <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> 2009</a>).</p></div><div id="sec-12" class="subsection"><h3>Four statistically independent associated markers in the GGA2 QTL Growth2 and Growth3</h3><p id="p-42">GGA2 contains two QTL, <em>Growth2</em> (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>; 47.9–65.5 Mb) and <em>Growth3</em> (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>; 124.3–133.6 Mb). The multi-locus analyses identified three significantly associated markers in <em>Growth2</em>, where the first two are clustered at 56.7 and 57.2 Mb (genotyped marker <em>rs14185295</em> and imputed marker <em>rs14185836<sup>i</sup></em>, respectively), with the last genotyped marker (<em>rs14196021</em>) located at 65.5 Mb. The distance between the markers, and the region of low association between them in the single-marker analysis (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>), suggests that two linked loci contribute to the <em>Growth2</em> QTL. In the earlier linkage-based analysis the strongest signal in <em>Growth2</em> was located in between these markers at 60.6 Mb. The QTL-peak in the original F<sub>2</sub> analysis (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>) is difficult to assess as nearest marker (<em>MCW130</em>) is not mapped to the chicken genome and no significant signal was found using a denser marker-map by <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> (2009)</a>. As the first two markers in the QTL are tightly linked, it is difficult to interpret the individual estimates of their effects; however, the third marker located 8 Mb upstream from them has a small independent effect. The estimated contribution by these loci to the founder line difference is small (14 g), which amounts to about 30% of the original contribution by the <em>Growth2</em> QTL in the F<sub>2</sub> analyses (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>). In <em>Growth3</em>, a single association was detected to a genotyped marker (<em>rs16120360</em>) in the multi-locus analysis and this peak was inside the original F<sub>2</sub> QTL [101.6–131.9 Mb; (<a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>)] but shifted almost 4 Mb upstream from the top signal found in the earlier linkage-based analysis. The contribution by this marker to the line difference is small (9 g) and about 15% of that in the original F<sub>2</sub> analysis (<a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>).</p></div><div id="sec-13" class="subsection"><h3>Six statistically independent associated markers in the GGA5 QTL Growth8</h3><p id="p-43">One of the strongest association signals was found on GGA5 in the QTL <em>Growth8</em> (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>; 33.7–39.1 Mb) and six markers were retained in this region after accounting for population structure in the multi-locus analysis (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). The single-locus analysis suggests that these markers tag two loci – one from 33.7–36.3 Mb (three imputed and one genotyped marker; <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>) and one around 38.8 Mb (two imputed markers). The markers were located between the markers flanking the original QTL (21.6–44.2 Mb) in <a id="xref-ref-18-10" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> (2005</a>) and overlaps with the earlier linkage signal. The association signal was, however, stronger than the linkage signal suggesting that the imputed markers tag the QTL better than the haplotypes inferred from the sparser set of genotyped markers. Although the linkage between the markers again makes it difficult to obtain stable estimates for the effects of individual markers in the two associated loci, their estimated contribution to the founder line difference (16 g) amounts to about one third of that estimated effect in the F<sub>2</sub> (<a id="xref-ref-18-11" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>).</p></div><div id="sec-14" class="subsection"><h3>Six statistically independent associated markers in the GGA3 QTL Growth4</h3><p id="p-44">In the QTL <em>Growth4</em> on GGA3 (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>; 24.0–68.1 Mb), both the association and linkage analyses identify a broad region of association from 24 to 41 Mb. Although the statistical support curve in the linkage analysis contains multiple peaks, that analysis was unable to fine-map the region into multiple, independent signals. Here, the multi-locus analysis suggests that perhaps up to four independent regions contribute to this QTL, with one associated genotyped marker at 26.2 Mb, three imputed markers from 33.7–39.1 Mb, one imputed marker at 47.7 Mb, and one at 57.6 Mb. The single-locus association analysis highlights two particularly strong and distinct association-peaks located approximately between 24–27 and 33–37 Mb, respectively. A candidate mutation in <em>Growth4</em> was found near the second association region at 33.6 Mb inside the regulatory region of Cysteine rich transmembrane BMP regulator 1 (<em>CRIM1</em>) (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Ahsan <em>et al.</em> 2013</a>). The associated region around 55–57 Mb displayed very low significance in the previous linkage analysis. The outmost markers (26.2 and 57.6 Mb) have allele-substitution effects of 19.8 (SE 5.4) and 18.2 (SE 4.1) g, respectively, and are rather diverged between the lines (∼50% difference between the lines). For the other two clusters of markers, it is difficult to obtain stable estimates of their individual effects. Their estimated joint contribution to the founder line difference (35 g) is about 65% of that in the original F<sub>2</sub> analysis (<a id="xref-ref-18-12" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>).</p></div><div id="sec-15" class="subsection"><h3>Four statistically independent associated markers in the GGA20 QTL Growth12</h3><p id="p-45">The earlier linkage analysis replicated the QTL <em>Growth12</em> on GGA20 (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a>; 7.1–13.9 Mb), with the strongest associated marker at 10.7 Mb, and the signal covered most of the region (8–13.9 Mb). Four markers were significant in the multi-locus analysis and three imputed markers were located in the main single-marker association peak covering the region from 9 to 11 Mb, while the fourth associated imputed marker was located about 4 Mb upstream (13.4 Mb). Again, it is difficult to interpret the individual effects of the tightly linked markers; however, their estimated contribution to the line difference (22 g) is about 75% of that estimated in the F<sub>2</sub> analysis (<a id="xref-ref-18-13" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>).</p></div><div id="sec-16" class="subsection"><h3>Five statistically independent associated markers in the GGA4 QTL Growth6 and Growth7</h3><p id="p-46">In both <em>Growth6</em> (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1</a>; 1.3–13.6 Mb) and <em>Growth7</em> (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1</a>; 85.4–88.9 Mb) on GGA4, several markers were significant in the multi-marker analysis. The single-marker analysis illustrates that these markers tag association-peaks that were located very close to the main peaks in the earlier linkage-based analysis, suggesting that both analyses identified the same underlying loci. The association analysis identified two genotyped markers in a region in <em>Growth7</em> with strong association around 86.5–88.5 Mb. It is difficult to interpret their individual effects due to the close linkage, but their estimated contribution to the founder line difference was small (13 g) amounting to about a fifth of that estimated in the F<sub>2</sub> population (66 g; <a id="xref-ref-18-14" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>). In <em>Growth6</em>, the multi-marker analysis detected associations to two imputed and one genotyped marker representing one locus at 2.3 Mb and a second locus at 10.9–13.5 Mb. Here, the tight linkage in the second locus also makes interpretation of their individual effects difficult, whereas the allele frequencies are not that differentiated in the first locus. Their estimated contribution to the line difference is therefore small (17 g) amounting to about a fifth of the 92 g estimated in the original F<sub>2</sub> analysis (<a id="xref-ref-18-15" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>).</p></div><div id="sec-17" class="subsection"><h3>General comments</h3><p id="p-47">Here we report the results from using an imputation-based association-mapping strategy to fine-map QTL in a nine-generation, outbred AIL. By combining high-density genotyping of the AIL founders with imputation throughout the rest of the pedigree utilizing a sparser genotyped marker backbone, we increased the marker density ∼20-fold in the studied regions. This subsequent association analysis had a comparable power for replication of QTL to the earlier used linkage-based strategy. In addition to this, the new analyses also detected multiple association-peaks in several of the QTL and narrowed the associated regions considerably compared to the regions detected previously (<a id="xref-ref-6-17" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>). Together, they suggest that this imputation-based association-mapping approach is a promising strategy for improving the resolution in fine-mapping studies in outbred pedigrees, where high-density marker genotypes are not available for all studied individuals. When interpreting the full results from the multi-locus backward-elimination analysis (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>), it should be noted that the results are reported at a 20% False Discovery Rate. Although a significant proportion of the markers could thus be false positive associations, as illustrated by the individual P-values for the additive effects of the associated markers, most of the peaks on the chromosomes also contain markers with more significant individual associations. We used this threshold because earlier mapping and replication studies confirmed that the QTL contain at least one small-effect locus contributing to 56 d weight, and that the major aim of this study was to provide an overall view of the most likely genetic architecture of the fine-mapped QTL, rather than high-confidence estimates for the individual regions.</p><p id="p-48">In both <em>Growth1</em> and <em>Growth4</em> two strong, distinct association signals were identified. Also in the QTL <em>Growth8</em> and <em>Growth9</em> the new analysis identified strong association-peaks covering many markers. In these regions, the strongest linkage signals identified in the previous fine-mapping analysis (<a id="xref-ref-6-18" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>) overlap with the strongest signals in the current analyses. However, the association analysis also separates the signals into multiple peaks and highlights narrower regions. Hence, it provides more useful input for further analyses to identify candidate genes underlying the QTL. In most cases the associated regions are restricted to distinct 2–3 Mb regions, which as indicated by the findings from <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Ahsan <em>et al.</em> (2013</a>), is useful for restricting the bioinformatics analyses to only the most promising candidate genes for further functional studies. In <em>Growth6</em>, <em>Growth7</em>, and <em>Growth12</em>, the association signals were not as significant as in the other QTL. Despite this, the multi-locus analyses suggest that the linkage signals in the earlier analyses were due to distinct loci with independent effects, mapped here into narrower association-peaks.</p><p id="p-49">Overall, the location of the association signals in this study overlapped well with the top signals in the earlier linkage analyses. However, in two of the QTL (<em>Growth2</em> and <em>Growth3</em>), the association-peaks are shifted when comparing results from both studies. Further work is needed to explore whether this reflects separate loci with distinct genetic architectures that could only be detected with the respective methods, or if they reflect a signal of the same underlying causal locus.</p><p id="p-50">Here, we estimated the additive genetic effects of the fine-mapped regions using data from the F<sub>2</sub>–F<sub>8</sub> generations of the AIL. To evaluate whether they were in general agreement with estimates obtained for the same regions in earlier studies, we compared them to the estimates obtained from our first large F<sub>2</sub> population (<a id="xref-ref-18-16" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>; <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> 2009</a>). The QTL effects were generally lower in the F<sub>2</sub>–F<sub>8</sub> data than in the F<sub>2</sub>. Although this may be interpreted as the F<sub>2</sub> estimates being inflated, several other factors could also be considered. First, the 56 d body weights were considerably lower for the F<sub>8</sub> generation because younger dams were used to generate these (<a id="xref-ref-6-19" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>). In the analyses, a fixed effect of generation was used to account for the mean weight differences between generations. However, it did not account for the likely scenario that the QTL effects were smaller in these F<sub>8</sub> birds due to their lower body weight. As about 30% of the birds in the pedigree are from this generation, this would bias the overall effects downward. Standardization of the phenotypes from different generations to the same mean and variance is a way to possibly account for this, but a caveat of that approach is an introduction of an upward bias of the effects if the QTL effects in the F<sub>8</sub> are, in fact, not that much smaller. We therefore chose to report the more conservative estimates based on analyzing the nonstandardized phenotypes. Second, eight of the nine QTL contain fine-mapped regions with associations to several tightly linked markers. If these markers are located on the same haplotypes, it is not possible to disentangle their effects in this pedigree as too few recombination events have accumulated in the F<sub>2</sub>–F<sub>8</sub> generations of the AIL, and due to such collinearities the estimates for the individual markers reported here would not properly describe the contributions of these haplotypes to the line difference. In several of the regions with multiple associated markers, the estimates of the additive effects were also negative for at least one of these markers. Although this could be interpreted as transgression being common in the population, we find them more likely to result from the collinearities among the closely linked markers. Further analyses utilizing, for example, later AIL generations, markers that specifically tag the haplotype-structure of the founder lines and methods that can account for multi-allelic genetic architectures will be needed to disentangle the genetic architectures of these loci and quantify their contributions to the founder line difference. Third, the F<sub>2</sub> QTL estimates were obtained using a line-cross analysis where it is assumed that the founder lines are fixed for alternative QTL alleles (<a id="xref-ref-18-17" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> 2005</a>; <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Wahlberg <em>et al.</em> 2009</a>). In the current association analysis, it is instead assumed that the alternative alleles at the tested markers tag nearby functional alleles. As none of the associated markers were fixed for alternative alleles in the founder lines (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036012/-/DC1/TableS1.pdf">Table S1</a>), the current fine-mapping analysis suggests that one, or both, founder lines segregate for multiple functional alleles in the QTL. To compare the estimates from the line-cross analysis in the F<sub>2</sub> and the association analyses in the AIL F<sub>2</sub>–F<sub>8</sub> generations, they need to be compared using a common reference. Here, we did this by estimating how much the associated markers in each QTL were expected to contribute to the founder line difference under the assumption that they act completely additively. Under this assumption, their contribution would equal twice the sum of the allele-substitution effects of the markers in a QTL weighted by their respective allele-frequency differences between the founder lines. That is, if the markers are fixed for alternative alleles in the founder lines they would contribute two allele-substitution effects to the founder line difference, whereas they would contribute nothing if the allele was present at equal frequencies in both founder lines. This estimate is conservative as, for example, dominance leads to an underestimation of the contribution of the locus. This as, in the presence of a dominant allele, one line will not need to be fixed to contribute most of its effect because this is also displayed in the heterozygotes for that allele. When comparing estimates this way, the combined effects of the associated markers in each of the QTL contribute from 10 to 75% of that estimated in the F<sub>2</sub> by <a id="xref-ref-18-18" class="xref-bibr" href="#ref-18">Jacobsson <em>et al.</em> (2005</a>). In total, the QTL replicated here contributed 171 g to the founder line difference, compared to the 416 g in the F<sub>2</sub>. As discussed above, further analyses in other populations with more informative genetic markers using other statistical methods are, however, required to explore this further.</p><p id="p-51">Our analyses suggest that there is extensive within-line segregation in the QTL regions. One possible explanation for the slow fixation in these loci could be that the beneficial alleles at the linked fine-mapped loci were located on different haplotypes at the onset of selection. Due to the low selection pressure on each QTL region resulting from the highly polygenic architecture of the selected trait, the close linkage between the loci contributing to the QTL, and the small effective breeding population, the probability that beneficial recombinant haplotypes are selected and increase in frequency in the population should be low. Another alternative explanation could be that the effects of the linked loci are dependent on the genetic background (epistasis) or dominance, which might have affected the selection pressure on individual contributing loci. As we did not explore the contributions by dominance or epistasis in this study, further work would be necessary to evaluate their contributions to the low fixation in the QTL.</p><p id="p-52">A key for successful imputation of the high-density marker set throughout the AIL pedigree is that the haplotypes across these markers are correctly estimated in the founders. There are several properties of the Virginia lines that improve haplotype estimation from high-density genotypes. First, as the number of generations since the lines diverged is relatively few (40 generations), most new haplotypes will result from recombination of original haplotypes, rather than by new mutations. Second, the strong artificial selection imposed on the populations since they were founded is likely to have further reduced haplotype diversity across the genome. This is likely the reason that many selective-sweeps across long haplotypes have been found to be fixed, or nearly fixed, across the genome within and between the lineages (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Johansson <em>et al.</em> 2010</a>; <a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">Pettersson <em>et al.</em> 2013</a>). This is reflected in a large average LD-block size (&gt; 50 kb) across the genome (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Marklund and Carlborg 2010</a>). Given the density of the 60k SNP-chip genotyping used here, several markers will be present on each such LD-block and hence improve efficiency in haplotype estimation. Additional genotyping will, however, be necessary in subsequent generations to experimentally confirm the associations to imputed markers reported here.</p><p id="p-53">Genotype data are available for all individuals in the AIL pedigree. The dense marker backbone (∼1 marker/cM) from the first genotyping of the AIL (<a id="xref-ref-6-20" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>), allow the relatively long haplotypes that are inherited as intact segments from parents to offspring to be efficiently phased, imputed, and traced throughout the pedigree for later association analyses.</p><p id="p-54">The highly polygenic genetic architecture in this population is consistent with what has been revealed in other fine-mapping analyses in deep intercrosses (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Parker <em>et al.</em> 2011</a>) and chromosomal substitution strains (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Bevova <em>et al.</em> 2006</a>) involving intensively selected mouse populations. Recent work on a mouse population that has evolved to an extreme body size in nature has also uncovered a highly polygenic architecture of adaptation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gray <em>et al.</em> 2015</a>), illustrating that complex genetic architectures are likely to be involved in responses to both natural and experimental selection. Further, our detection of multiple associations to nearby markers in our AIL is also consistent with reports from other AIL-based fine-mapping studies in chickens from outbred base-populations (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Van Goor <em>et al.</em> 2015</a>) and association studies within and across cattle breeds (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Saatchi <em>et al.</em> 2014</a>). Subsequent studies will help to elucidate whether the underlying genetic architecture of associations detected to linked markers in this and other outbred populations are primarily due to the segregation of multiple haplotypes in the outbred founder populations and breeds or a reflection of several tightly linked functional polymorphisms.</p><p id="p-55">Here, the association analysis was performed using a linear model including fixed effects of genotype, sex, and AIL generation. Sex and generation were included as both these environmental factors had significant effects on BW56 (<a id="xref-ref-6-21" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>). Implementing the model selection by backward-elimination in a bootstrap-based framework is a way to account for possible effects of population structure in the AIL that might increase the risk for reporting false positives. However, since the association signals in most cases overlap well with the final marker set resulting from the testing of experiment-wide significant associations, we do not find this to be any cause of great concern in this experiment.</p></div><div id="sec-18" class="subsection"><h3>Conclusions</h3><p id="p-56">In conclusion, this study shows that the proposed imputation-based association-mapping strategy, and further model selection by backward-elimination in a bootstrap-based framework, is useful for identifying independent association signals within and across the nine evaluated QTL. The association-peaks were narrower than those obtained in the earlier performed linkage analysis, often highlighting regions down to 2–3 Mb in length allowing the identification of multiple association signals in several QTL. This suggests that the association-based strategy has higher resolution, as well as provides an improved power to disentangle the effects of multiple linked loci inside QTL, compared to linkage-based fine-mapping. Combining traditional linkage-based approaches to analyze outbred advanced intercross populations with imputation-based association-mapping approaches might thus be an important and cost-effective approach to improve the efficiency in postassociation bioinformatics analyses and functional explorations aiming to identify candidate mutations. A previous candidate gene study based on the nine QTL fine-mapped here has already reported some interesting mutations in growth-related genes (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Ahsan <em>et al.</em> 2013</a>) overlapping with the association signals reported here. Further bioinformatics investigations of the regions fine-mapped here could potentially reveal new important genes and mutations affecting body weight in these chicken lines and provide new candidate genes for studying the genetic architecture of metabolic traits in other species, including humans.</p></div></div><div class="section" id="sec-19"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank Leif Andersson for initiating the AIL experiment with P.B.S. and sharing the data from the F2 intercross. Per Wahlberg and Francois Besnier are acknowledged for their valuable contributions during preparation and quality control work of the genotype and phenotype data from the AIL. Genotyping was performed by the SNP&amp;SEQ Technology Platform in Uppsala, which is part of Science for Life Laboratory at Uppsala University and is supported as a national infrastructure by the Swedish Research Council (VR-RFI). Formas (grant 221-2013-450 to Ö.C.) and the Swedish Research Council (grant 621-2012-4634 to Ö.C.) are acknowledged for financial support.</p><p id="p-58">Author contributions: Ö.C. and P.B.S. initiated the study. Ö.C. designed the project with M.A. and M.B.; P.B.S. developed the Virginia chicken lines; P.B.S. designed, planned, and bred the Virginia Advanced Intercross Line; P.B.S. and C.F.H. designed, planned, bred, bled, phenotyped, and extracted DNA from the Virginia Advanced Intercross Line population; Ö.C. designed the statistical analyses; M.A., M.B., and Ö.C. contributed analysis scripts; Ö.C., M.B., and M.A. performed the data analyses, summarized the results, and wrote the manuscript. All authors read and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-18"><p id="p-59">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024505/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036012/-/DC1</a>.</p></li><li class="fn" id="fn-19"><p id="p-60"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-06-29"><span class="received-label">Received </span>June 29, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-10-30"><span class="accepted-label">Accepted </span>October 30, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Brandt <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.1" data-doi="10.1214/009053606000000074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abramovich</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donoho</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnstone</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Special invited lecture: adapting to unknown sparsity by controlling the false discovery rate.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D34%26rft.spage%253D584%26rft_id%253Dinfo%253Adoi%252F10.1214%252F009053606000000074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/009053606000000074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000238884500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahsan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundberg</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kierczak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of candidate genes and mutations in QTL regions for chicken growth using bioinformatic analysis of NGS and SNP-chip data.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D4%26rft.spage%253D226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.3" data-doi="10.1038/nrg1294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georges</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Domestic-animal genomics: deciphering the genetics of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">202</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAndersson%26rft.auinit1%253DL.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D202%26rft.epage%253D212%26rft.atitle%253DDomestic-animal%2Bgenomics%253A%2Bdeciphering%2Bthe%2Bgenetics%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1294%26rft_id%253Dinfo%253Apmid%252F14970822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14970822&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189334500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.4" data-doi="10.1093/bioinformatics/btm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaacs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GenABEL: an R package for genome-wide association analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm108%26rft_id%253Dinfo%253Apmid%252F17384015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTAvMTI5NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.6" data-doi="10.1186/1297-9686-43-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besnier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ek</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fine mapping and replication of QTL in outbred chicken advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DBesnier%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DFine%2Bmapping%2Band%2Breplication%2Bof%2BQTL%2Bin%2Boutbred%2Bchicken%2Badvanced%2Bintercross%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-3%26rft_id%253Dinfo%253Apmid%252F21241486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21241486&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.7" data-doi="10.1534/genetics.104.040196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bevova</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aksu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renne</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brockmann</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome-wise dissection of the genome of the extremely big mouse line DU6i.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.040196%26rft_id%253Dinfo%253Apmid%252F16157676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNDAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.8" data-doi="10.1038/ng1761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Åhgren</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Epistasis and the release of genetic variation during long-term selection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D418%26rft.epage%253D420%26rft.atitle%253DEpistasis%2Band%2Bthe%2Brelease%2Bof%2Bgenetic%2Bvariation%2Bduring%2Blong-term%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1761%26rft_id%253Dinfo%253Apmid%252F16532011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532011&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236340500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.9" data-doi="10.1534/genetics.110.116863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokoloff</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116863%26rft_id%253Dinfo%253Apmid%252F20439773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.10" data-doi="10.1534/genetics.111.128082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Imputation of missing genotypes from sparse to high density using long-range phasing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128082%26rft_id%253Dinfo%253Apmid%252F21705746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzEvMzE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Advanced intercross lines, an experimental population for fine genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1199%26rft.epage%253D1207%26rft.atitle%253DAdvanced%2Bintercross%2Blines%252C%2Ban%2Bexperimental%2Bpopulation%2Bfor%2Bfine%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F8582624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzExOTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.12" data-doi="10.3382/ps.0751168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunnington</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Long-term divergent selection for eight-week body weight in White Plymouth Rock chickens.</span> <abbr class="cit-jnl-abbrev">Poult. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1168</span>–<span class="cit-lpage">1179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps.0751168%26rft_id%253Dinfo%253Apmid%252F8893291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBvdWx0cnlzY2kiO3M6NToicmVzaWQiO3M6MTA6Ijc1LzEwLzExNjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.13" data-doi="10.3382/ps.2013-03069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunnington</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honaker</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGilliard</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phenotypic responses of chickens to long-term, bidirectional selection for juvenile body weight–historical perspective.</span> <abbr class="cit-jnl-abbrev">Poult. Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">1724</span>–<span class="cit-lpage">1734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps.2013-03069%26rft_id%253Dinfo%253Apmid%252F23776258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBvdWx0cnlzY2kiO3M6NToicmVzaWQiO3M6OToiOTIvNy8xNzI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ek</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marklund</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragavendran</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Generation of a multi-locus chicken introgression line to study the effects of genetic interactions on metabolic phenotypes in chickens.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DEk%26rft.auinit1%253DW.%26rft.volume%253D3%26rft.spage%253D29%26rft.epage%253D29%26rft.atitle%253DGeneration%2Bof%2Ba%2Bmulti-locus%2Bchicken%2Bintrogression%2Bline%2Bto%2Bstudy%2Bthe%2Beffects%2Bof%2Bgenetic%2Binteractions%2Bon%2Bmetabolic%2Bphenotypes%2Bin%2Bchickens.%26rft_id%253Dinfo%253Apmid%252F22403584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22403584&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.15" data-doi="10.1214/07-AOS586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gavrilov</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An adaptive step-down procedure with proven FDR control under independence.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D37%26rft.spage%253D619%26rft_id%253Dinfo%253Adoi%252F10.1214%252F07-AOS586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/07-AOS586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000265500500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.16" data-doi="10.1534/genetics.115.177790"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmenter</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogan</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuthbert</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetics of rapid and extreme size evolution in island mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177790%26rft_id%253Dinfo%253Apmid%252F26199233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzEvMjEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8xLzExOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.17" data-doi="10.1126/science.1105459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A century of corn selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D307%26rft.issue%253D5710%26rft.spage%253D683%26rft.epage%253D684%26rft.atitle%253DGENETICS%253A%2BA%2BCentury%2Bof%2BCorn%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105459%26rft_id%253Dinfo%253Apmid%252F15692038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDcvNTcxMC82ODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.18" data-doi="10.1017/S0016672305007767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredriksson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Many QTLs with minor additive effects are associated with a large difference in growth between two selection lines in chickens.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DJacobsson%26rft.auinit1%253DL.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D125%26rft.atitle%253DMany%2BQTLs%2Bwith%2Bminor%2Badditive%2Beffects%2Bare%2Bassociated%2Bwith%2Ba%2Blarge%2Bdifference%2Bin%2Bgrowth%2Bbetween%2Btwo%2Bselection%2Blines%2Bin%2Bchickens.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672305007767%26rft_id%253Dinfo%253Apmid%252F16356285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672305007767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16356285&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234582400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.19" data-doi="10.1371/journal.pgen.1001188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide effects of long-term divergent selection.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJohansson%26rft.auinit1%253DA.%2BM.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001188%26rft.epage%253De1001188%26rft.atitle%253DGenome-wide%2Beffects%2Bof%2Blong-term%2Bdivergent%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001188%26rft_id%253Dinfo%253Apmid%252F21079680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079680&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.20" data-doi="10.1186/gb-2012-13-4-244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of body size in mammals.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKemper%26rft.auinit1%253DK.%2BE.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D244%26rft.epage%253D244%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bbody%2Bsize%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-4-244%26rft_id%253Dinfo%253Apmid%252F22546202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-4-244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22546202&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.21" data-doi="10.1016/j.tree.2007.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Rouzic</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolutionary potential of hidden genetic variation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DLe%2BRouzic%26rft.auinit1%253DA.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D37%26rft.atitle%253DEvolutionary%2Bpotential%2Bof%2Bhidden%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.09.014%26rft_id%253Dinfo%253Apmid%252F18079017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18079017&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.22" data-doi="10.1186/1741-7007-5-50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Rouzic</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Phenotypic evolution from genetic polymorphisms in a radial network architecture.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DLe%2BRouzic%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.spage%253D50%26rft.epage%253D50%26rft.atitle%253DPhenotypic%2Bevolution%2Bfrom%2Bgenetic%2Bpolymorphisms%2Bin%2Ba%2Bradial%2Bnetwork%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-5-50%26rft_id%253Dinfo%253Apmid%252F18001473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-5-50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18001473&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.23" data-doi="10.1186/1471-2164-11-665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marklund</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SNP detection and prediction of variability between chicken lines using genome resequencing of DNA pools.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMarklund%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.spage%253D665%26rft.epage%253D665%26rft.atitle%253DSNP%2Bdetection%2Band%2Bprediction%2Bof%2Bvariability%2Bbetween%2Bchicken%2Blines%2Busing%2Bgenome%2Bresequencing%2Bof%2BDNA%2Bpools.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-665%26rft_id%253Dinfo%253Apmid%252F21108801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21108801&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.24" data-doi="10.3382/ps.2010-01034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Márquez</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic diversity and population structure in lines of chickens divergently selected for high and low 8-week body weight.</span> <abbr class="cit-jnl-abbrev">Poult. Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2580</span>–<span class="cit-lpage">2588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps.2010-01034%26rft_id%253Dinfo%253Apmid%252F21076095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBvdWx0cnlzY2kiO3M6NToicmVzaWQiO3M6MTA6Ijg5LzEyLzI1ODAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.25" data-doi="10.1152/physiolgenomics.00113.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">QTL analysis of body composition and metabolic traits in an intercross between chicken lines divergently selected for growth.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00113.2005%26rft_id%253Dinfo%253Apmid%252F16390876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo4OiIyNS8yLzIxNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.26" data-doi="10.1007/s00335-011-9349-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokoloff</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skol</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fine-mapping alleles for body weight in LG/J × SM/J F<sub>2</sub> and F<sub>34</sub> advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DParker%26rft.auinit1%253DC.%2BC.%26rft.volume%253D22%26rft.issue%253D9-10%26rft.spage%253D563%26rft.epage%253D571%26rft.atitle%253DFine-mapping%2Balleles%2Bfor%2Bbody%2Bweight%2Bin%2BLG%252FJ%2B%25C3%2597%2BSM%252FJ%2BF2%2Band%2BF%252834%2529%2Badvanced%2Bintercross%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-011-9349-z%26rft_id%253Dinfo%253Apmid%252F21761260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-011-9349-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21761260&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295130800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.27" data-doi="10.1371/journal.pone.0001977"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome reshuffling for advanced intercross permutation (GRAIP): simulation and permutation for advanced intercross population analysis.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e1977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253De1977%26rft.epage%253De1977%26rft.atitle%253DGenome%2BReshuffling%2Bfor%2BAdvanced%2BIntercross%2BPermutation%2B%2528GRAIP%2529%253A%2Bsimulation%2Band%2Bpermutation%2Bfor%2Badvanced%2Bintercross%2Bpopulation%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001977%26rft_id%253Dinfo%253Apmid%252F18431467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001977&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18431467&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.28" data-doi="10.1371/journal.pgen.1002180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besnier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Replication and explorations of high-order epistasis using a large advanced intercross line pedigree.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPettersson%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002180%26rft.epage%253De1002180%26rft.atitle%253DReplication%2Band%2Bexplorations%2Bof%2Bhigh-order%2Bepistasis%2Busing%2Ba%2Blarge%2Badvanced%2Bintercross%2Bline%2Bpedigree.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002180%26rft_id%253Dinfo%253Apmid%252F21814519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21814519&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.29" data-doi="10.1534/g3.113.008375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dynamics of adaptive alleles in divergently selected body weight lines of chickens.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2305</span>–<span class="cit-lpage">2312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008375%26rft_id%253Dinfo%253Apmid%252F24170737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTIvMjMwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-other" id="cit-7.1.119.30"><div class="cit-metadata unstructured">R Development Core Team, 2015 R: A Language and Environment for Statistical Computing. <a href="https://www.r-project.org">https://www.r-project.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.31" data-doi="10.1534/genetics.107.083162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besnier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An improved method for quantitative trait loci detection and identification of within-line segregation in F2 intercross designs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2315</span>–<span class="cit-lpage">2326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRonnegard%26rft.auinit1%253DL.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2315%26rft.epage%253D2326%26rft.atitle%253DAn%2BImproved%2BMethod%2Bfor%2BQuantitative%2BTrait%2BLoci%2BDetection%2Band%2BIdentification%2Bof%2BWithin-Line%2BSegregation%2Bin%2BF2%2BIntercross%2BDesigns%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083162%26rft_id%253Dinfo%253Apmid%252F18430952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIzMTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saatchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Large-effect pleiotropic or closely linked QTL segregate within and across ten US cattle breeds.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D442%26rft_id%253Dinfo%253Apmid%252F24906442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24906442&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.33" data-doi="10.1186/s13059-015-0785-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honaker</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Standing genetic variation as a major contributor to adaptation in the Virginia chicken lines selection experiment.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0785-z%26rft_id%253Dinfo%253Apmid%252F26438066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0785-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26438066&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span><span class="cit-comment">a</span> <span class="cit-article-title">A double selection experiment for body weight and breast angle at eight weeks of age in chickens.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1313</span>–<span class="cit-lpage">1319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSiegel%26rft.auinit1%253DP.%2BB.%26rft.volume%253D47%26rft.issue%253D10%26rft.spage%253D1313%26rft.epage%253D1319%26rft.atitle%253DA%2BDOUBLE%2BSELECTION%2BEXPERIMENT%2BFOR%2BBODY%2BWEIGHT%2BAND%2BBREAST%2BANGLE%2BAT%2BEIGHT%2BWEEKS%2BOF%2BAGE%2BIN%2BCHICKENS%26rft_id%253Dinfo%253Apmid%252F13988956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czoxMDoiNDcvMTAvMTMxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMS8xMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.35" data-doi="10.3382/ps.0410954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span><span class="cit-comment">b</span> <span class="cit-article-title">Selection for body weight at eight weeks of age. 1. Short term response and heritabilities.</span> <abbr class="cit-jnl-abbrev">Poult. Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">954</span>–<span class="cit-lpage">962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps.0410954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBvdWx0cnlzY2kiO3M6NToicmVzaWQiO3M6ODoiNDEvMy85NTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.36" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.37" data-doi="10.1534/genetics.109.100727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping in structured populations by resample model averaging.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.100727%26rft_id%253Dinfo%253Apmid%252F19474203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEyNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzEvMTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Goor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolek</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashwell</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Persia</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothschild</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of quantitative trait loci for body temperature, body weight, breast yield, and digestibility in an advanced intercross line of chickens under heat stress.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D96%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.39" data-doi="10.1186/1471-2164-10-248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foglio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tordoir</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syvänen</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic analysis of an F(2) intercross between two chicken lines divergently selected for body-weight.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWahlberg%26rft.auinit1%253DP.%26rft.volume%253D10%26rft.spage%253D248%26rft.epage%253D248%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Ban%2BF%25282%2529%2Bintercross%2Bbetween%2Btwo%2Bchicken%2Blines%2Bdivergently%2Bselected%2Bfor%2Bbody-weight.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-248%26rft_id%253Dinfo%253Apmid%252F19473501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19473501&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F1%2F119.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.1.119.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL analysis of body weight and carcass body length traits in an F2 intercross between Landrace and Korean native pigs.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D45%26rft.spage%253D589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/1/119.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/109" title="A New Chicken Genome Assembly Provides Insight into Avian Genome Structure" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/1/129" title="The Neurospora Transcription Factor ADV-1 Transduces Light Signals and Temporal Information to Control Rhythmic Expression of Genes Involved in Cell Fusion" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/1"><span>Volume 7 Issue 1, January 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/1.cover-source.jpg?itok=qMtlAYHH" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/1/119.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tggev5ythBd3yd7iJ4zpEcCwosRnm02iZHOKmQRkiRE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/1/119" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/1/119" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/1/119" class="active" data-icon-position="" data-hide-link-title="0">Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460548" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yz0zWSu6BMDA4Objzd2bdI3HGjX8zjVGkJDqFHLYN-w" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/119.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/1/119" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-B1sg2-WE3zoL9IqCp_gm0b00D8QbDKeISg6eMPi9sjc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/1/119" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/1/119.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460548" id="node460548--2" data-pisa="ggg;7/1/119" data-pisa-master="ggg;g3.116.036012" data-apath="/ggg/7/1/119.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Monika Brandt</span>, <span class="highwire-citation-author" data-delta="1">Muhammad Ahsan</span>, <span class="highwire-citation-author" data-delta="2">Christa F. Honaker</span>, <span class="highwire-citation-author" data-delta="3">Paul B. Siegel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2722-5264" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Örjan Carlborg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">119-128;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036012 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460548--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Monika Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Monika%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMonika%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Muhammad Ahsan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Computational Genetics, Department of Clinical Sciences, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Muhammad%2BAhsan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahsan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMuhammad%2BAhsan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christa F. Honaker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christa%2BF.%2BHonaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Honaker%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChrista%2BF.%2BHonaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul B. Siegel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BB.%2BSiegel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siegel%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BB.%2BSiegel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2722-5264" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Örjan Carlborg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:orjan.carlborg@imbim.uu.se" class="" data-icon-position="" data-hide-link-title="0">orjan.carlborg@imbim.uu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/1/119" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/1/119&amp;title=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/1/119&amp;title=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/1/119&amp;title=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/1/119&amp;text=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/1/119&amp;title=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/1/119&amp;t=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/1/119&amp;title=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/1/119&amp;title=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460548/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460548" id="node460548--3" data-pisa="ggg;7/1/119" data-pisa-master="ggg;g3.116.036012" data-apath="/ggg/7/1/119.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Based Fine-Mapping Suggests That Most QTL in an Outbred Chicken Advanced Intercross Body Weight Line Are Due to Multiple, Linked Loci</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Monika Brandt</span>, <span class="highwire-citation-author" data-delta="1">Muhammad Ahsan</span>, <span class="highwire-citation-author" data-delta="2">Christa F. Honaker</span>, <span class="highwire-citation-author" data-delta="3">Paul B. Siegel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2722-5264" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Örjan Carlborg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">119-128;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036012 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460548--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Monika Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Monika%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMonika%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Muhammad Ahsan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Computational Genetics, Department of Clinical Sciences, Swedish University of Agricultural Sciences, 750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Muhammad%2BAhsan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahsan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMuhammad%2BAhsan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christa F. Honaker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christa%2BF.%2BHonaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Honaker%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChrista%2BF.%2BHonaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul B. Siegel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BB.%2BSiegel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siegel%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BB.%2BSiegel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Medical Biochemistry and Microbiology, Uppsala University, 751 23 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2722-5264" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Örjan Carlborg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:orjan.carlborg@imbim.uu.se" class="" data-icon-position="" data-hide-link-title="0">orjan.carlborg@imbim.uu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=119&amp;rft.epage=128&amp;rft.atitle=Imputation-Based%20Fine-Mapping%20Suggests%20That%20Most%20QTL%20in%20an%20Outbred%20Chicken%20Advanced%20Intercross%20Body%20Weight%20Line%20Are%20Due%20to%20Multiple%2C%20Linked%20Loci&amp;rft.volume=7&amp;rft.issue=1&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Brandt%2C+Monika&amp;rft.au=Ahsan%2C+Muhammad&amp;rft.au=Honaker%2C+Christa+F.&amp;rft.au=Siegel%2C+Paul+B.&amp;rft.au=Carlborg%2C+%C3%96rjan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460548/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460548/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460548/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460548/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460548/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460548/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460548/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460548/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460548/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460548/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460548/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460548/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc investigations"><h3 class="highwire-toc-heading">Investigations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/1/119" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/1/119.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/1/119.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/1/119.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/1/119.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
