<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3497" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3497.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3497.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3497.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/11/3497/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3497.atom" />
<meta name="HW.pisa" content="ggg;6/11/3497" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" />
<meta name="DC.Identifier" content="10.1534/g3.116.033332" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Wolfe et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In clonally propagated crops, nonadditive genetic effects can be effectively exploited by the identification of superior genetic individuals as varieties. Cassava ( Manihot esculenta Crantz) is a clonally propagated staple food crop that feeds hundreds of millions. We quantified the amount and nature of nonadditive genetic variation for three key traits in a breeding population of cassava from sub-Saharan Africa using additive and nonadditive genome-wide marker-based relationship matrices. We then assessed the accuracy of genomic prediction for total (additive plus nonadditive) genetic value. We confirmed previous findings based on diallel crosses that nonadditive genetic variation is significant for key cassava traits. Specifically, we found that dominance is particularly important for root yield and epistasis contributes strongly to variation in cassava mosaic disease (CMD) resistance. Further, we showed that total genetic value predicted observed phenotypes more accurately than additive only models for root yield but not for dry matter content, which is mostly additive or for CMD resistance, which has high narrow-sense heritability. We address the implication of these results for cassava breeding and put our work in the context of previous results in cassava, and other plant and animal species." />
<meta name="DC.Contributor" content="Marnin D. Wolfe" />
<meta name="DC.Contributor" content="Peter Kulakow" />
<meta name="DC.Contributor" content="Ismail Y. Rabbi" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In clonally propagated crops, nonadditive genetic effects can be effectively exploited by the identification of superior genetic individuals as varieties. Cassava (&lt;i&gt;Manihot esculenta&lt;/i&gt; Crantz) is a clonally propagated staple food crop that feeds hundreds of millions. We quantified the amount and nature of nonadditive genetic variation for three key traits in a breeding population of cassava from sub-Saharan Africa using additive and nonadditive genome-wide marker-based relationship matrices. We then assessed the accuracy of genomic prediction for total (additive plus nonadditive) genetic value. We confirmed previous findings based on diallel crosses that nonadditive genetic variation is significant for key cassava traits. Specifically, we found that dominance is particularly important for root yield and epistasis contributes strongly to variation in cassava mosaic disease (CMD) resistance. Further, we showed that total genetic value predicted observed phenotypes more accurately than additive only models for root yield but not for dry matter content, which is mostly additive or for CMD resistance, which has high narrow-sense heritability. We address the implication of these results for cassava breeding and put our work in the context of previous results in cassava, and other plant and animal species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3497" />
<meta name="citation_id" content="6/11/3497" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3497" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3497.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3497.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3497.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.033332" />
<meta name="citation_pmid" content="27587297" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3497" />
<meta name="citation_lastpage" content="3506" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marnin D. Wolfe" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_email" content="wolfemd@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5929-5785" />
<meta name="citation_author" content="Peter Kulakow" />
<meta name="citation_author_institution" content="International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7574-2645" />
<meta name="citation_author" content="Ismail Y. Rabbi" />
<meta name="citation_author_institution" content="International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9966-2941" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="United States Department of Agriculture - Agricultural Research Service (USDA-ARS), R.W. Holley Center for Agriculture and Health, Ithaca, New York 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4849-628X" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=G. Ahlborn-Breier;citation_author=W. D. Hohenboken;citation_title=Additive and Nonadditive Genetic Effects on Milk Production in Dairy Cattle: Evidence for Major Individual Heterosis;citation_pages=592-602;citation_volume=74;citation_year=1991;citation_issue=2;citation_pmid=2045565;citation_doi=10.3168/jds.S0022-0302(91)78206-4" />
<meta name="citation_reference" content="citation_title=EMMREML: Fitting Mixed Models with Known Covariance Structures.;citation_author=D. Akdemir;citation_author=U.G. Okeke;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=O. Akano;citation_author=O. Dixon;citation_author=C. Mba;citation_author=E. Barrera;citation_author=M. Fregene;citation_title=Genetic mapping of a dominant gene conferring resistance to cassava mosaic disease.;citation_pages=521-525;citation_volume=105;citation_year=2002;citation_issue=4;citation_pmid=12582500;citation_doi=10.1007/s00122-002-0891-7" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Maechler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting linear mixed-effects models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Bolormaa;citation_author=J. E. Pryce;citation_author=Y. Zhang;citation_author=A. Reverter;citation_author=W. Barendse;citation_title=Non-additive genetic variation in growth, carcass and fertility traits of beef cattle.;citation_pages=1-12;citation_volume=47;citation_year=2015;citation_pmid=25595431;citation_doi=10.1186/s12711-014-0081-5" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=N. T. Cach;citation_author=J. C. Perez;citation_author=J. I. Lenis;citation_author=F. Calle;citation_author=N. Morante;citation_title=Epistasis in the expression of relevant traits in cassava (Manihot esculenta Crantz) for subhumid conditions.;citation_volume=96;citation_year=2005;citation_pmid=16014808;citation_doi=10.1093/jhered/esi084" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=N. T. Cach;citation_author=J. I. Lenis;citation_author=J. C. Perez;citation_author=N. Morante;citation_author=F. Calle;citation_title=Inheritance of useful traits in cassava grown in subhumid conditions.;citation_pages=177-182;citation_volume=125;citation_year=2006;citation_doi=10.1111/j.1439-0523.2006.01192.x" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=F. Calle;citation_author=J. C. Perez;citation_author=W. Gaitán;citation_author=N. Morante;citation_author=H. Ceballos;citation_title=Diallel inheritance of relevant traits in cassava (Manihot esculenta Crantz) adapted to acid-soil savannas.;citation_pages=177-186;citation_volume=144;citation_year=2005;citation_doi=10.1007/s10681-005-5810-y" />
<meta name="citation_reference" content="citation_journal_title=Trop. Plant Biol.;citation_author=H. Ceballos;citation_author=P. Kulakow;citation_author=C. Hershey;citation_title=Cassava breeding: current status, bottlenecks and the potential of biotechnology tools.;citation_pages=73-87;citation_volume=5;citation_year=2012;citation_doi=10.1007/s12042-012-9094-9" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. Ceballos;citation_author=R. S. Kawuki;citation_author=V. E. Gracen;citation_author=G. C. Yencho;citation_author=C. H. Hershey;citation_title=Conventional breeding, marker-assisted selection, genomic selection and inbreeding in clonally propagated crops: a case study for cassava.;citation_pages=1647-1667;citation_volume=128;citation_year=2015;citation_pmid=26093610;citation_doi=10.1007/s00122-015-2555-4" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Chalwe;citation_author=R. Melis;citation_author=P. Shanahan;citation_author=M. Chiona;citation_title=Inheritance of resistance to cassava green mite and other useful agronomic traits in cassava grown in Zambia.;citation_pages=103-119;citation_volume=205;citation_year=2015;citation_doi=10.1007/s10681-015-1404-5" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=D. Clifford;citation_author=P. McCullagh;citation_title=The regress function.;citation_pages=6-10;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Cockerham;citation_title=AN EXTENSION OF THE CONCEPT OF PARTITIONING HEREDITARY VARIANCE FOR ANALYSIS OF COVARIANCES AMONG RELATIVES WHEN EPISTASIS IS PRESENT;citation_pages=859-882;citation_volume=39;citation_year=1954;citation_issue=6;citation_pmid=17247525" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. C. Cockerham;citation_author=H. Tachida;citation_title=Permanency of response to selection for quantitative characters in finite populations.;citation_pages=1563-1565;citation_volume=85;citation_year=1988;citation_issue=5;citation_pmid=3422750;citation_doi=10.1073/pnas.85.5.1563" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. Costa;citation_author=J. Silva;citation_author=N. M. G. Borralho;citation_author=B. M. Potts;citation_title=Additive and non-additive genetic parameters from clonally replicated and seedling progenies of Eucalyptus globulus.;citation_pages=1113-1119;citation_volume=108;citation_year=2004;citation_issue=6;citation_pmid=15067398;citation_doi=10.1007/s00122-003-1524-5" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=J. F. Crow;citation_title=On epistasis: why it is unimportant in polygenic directional selection;citation_pages=1241-1244;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308099;citation_doi=10.1098/rstb.2009.0275" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=T. K. Doerksen;citation_author=L. W. Kannenberg;citation_author=E. A. Lee;citation_title=Effect of Recurrent Selection on Combining Ability in Maize Breeding Populations;citation_pages=1652-1658;citation_volume=43;citation_year=2003;citation_issue=5;citation_doi=10.2135/cropsci2003.1652" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. W. Dudley;citation_author=G. R. Johnson;citation_title=Epistatic Models Improve Prediction of Performance in Corn;citation_pages=763-770;citation_volume=49;citation_year=2009;citation_issue=3;citation_doi=10.2135/cropsci2008.08.0491" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The correlation between relatives on the supposition of mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1918;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=C. Fuerst;citation_author=J. Sölkner;citation_title=Additive and Nonadditive Genetic Variances for Milk Yield, Fertility, and Lifetime Performance Traits of Dairy Cattle;citation_pages=1114-1125;citation_volume=77;citation_year=1994;citation_issue=4;citation_pmid=8201046;citation_doi=10.3168/jds.S0022-0302(94)77047-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=C. J. Goodnight;citation_title=Epistasis and the effect of founder events on the additive genetic variance.;citation_pages=441-454;citation_volume=42;citation_year=1988;citation_doi=10.2307/2409030" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=J. Hallander;citation_author=P. Waldmann;citation_title=The effect of non-additive genetic interactions on selection in multi-locus genetic models.;citation_pages=349-359;citation_volume=98;citation_year=2007;citation_pmid=17327874" />
<meta name="citation_reference" content="citation_title=Quantitative Genetics in Maize Breeding;citation_author=A. R. Hallauer;citation_author=M. J. Carena;citation_author=J. B. Miranda Filho;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. T. Hamblin;citation_author=I. Y. Rabbi;citation_title=The effects of restriction-enzyme choice on properties of genotyping-by-sequencing libraries: a study in cassava;citation_pages=2603;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2014.02.0160" />
<meta name="citation_reference" content="citation_journal_title=Evolution (N. Y.);citation_author=T. F. Hansen;citation_title=Why epistasis is important for selection and adaptation.;citation_pages=3501-3511;citation_volume=67;citation_year=2013;citation_doi=10.1111/evo.12214" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. R. Henderson;citation_title=Best Linear Unbiased Prediction of Nonadditive Genetic Merits in Noninbred Populations;citation_pages=111-117;citation_volume=60;citation_year=1985;citation_issue=1;citation_doi=10.2527/jas1985.601111x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=Z. Hu;citation_author=Y. Li;citation_author=X. Song;citation_author=Y. Han;citation_author=X. Cai;citation_title=Genomic value prediction for quantitative traits under the epistatic model.;citation_pages=15-15;citation_volume=12;citation_year=2011;citation_pmid=21269439" />
<meta name="citation_reference" content="citation_title=G3;citation_title=High-resolution linkage map and chromosome-scale genome assembly for cassava (Manihot esculenta Crantz) from ten populations.;citation_volume=5;citation_year=2015;citation_pmid=25504737;citation_doi=10.1534/g3.114.015008" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=G. Jaramillo;citation_author=N. Morante;citation_author=J. C. Pérez;citation_author=F. Calle;citation_author=H. Ceballos;citation_title=Diallel Analysis in Cassava Adapted to the Midaltitude Valleys Environment;citation_pages=1058-1063;citation_volume=45;citation_year=2005;citation_issue=3;citation_doi=10.2135/cropsci2004.0314" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=O. Kempthorne;citation_title=The Correlation between Relatives in a Random Mating Population;citation_pages=103-113;citation_volume=143;citation_year=1954;citation_issue=910;citation_doi=10.1098/rspb.1954.0056" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Biol.;citation_author=R. Killick;citation_title=Genetic analysis of several traits in potatoes by means of a diallel cross.;citation_pages=279-286;citation_volume=86;citation_year=1977;citation_doi=10.1111/j.1744-7348.1977.tb01841.x" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=H. P. Kulembeka;citation_author=M. Ferguson;citation_author=L. Herselman;citation_author=E. Kanju;citation_author=G. Mkamilo;citation_title=Diallel analysis of field resistance to brown streak disease in cassava (Manihot esculenta Crantz) landraces from Tanzania.;citation_pages=277-288;citation_volume=187;citation_year=2012;citation_doi=10.1007/s10681-012-0730-0" />
<meta name="citation_reference" content="citation_title=G3;citation_author=S. Kumar;citation_author=C. Molloy;citation_author=P. Muñoz;citation_author=H. Daetwyler;citation_author=D. Chagné;citation_title=Genome-enabled estimates of additive and non-additive genetic variances and prediction of apple phenotypes across environments.;citation_volume=5;citation_year=2015;citation_pmid=26497141;citation_doi=10.1534/g3.115.021105" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic selection in plant breeding: knowledge and prospects.;citation_pages=724-736;citation_volume=29;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=P. X. Lu;citation_author=D. A. Huber;citation_author=T. L. White;citation_title=Potential biases of incomplete linear models in heritability estimation and breeding value prediction.;citation_pages=724-736;citation_volume=29;citation_year=1999;citation_doi=10.1139/x99-047" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=D. Ly;citation_author=M. Hamblin;citation_author=I. Rabbi;citation_author=G. Melaku;citation_author=M. Bakare;citation_title=Relatedness and genotype × environment interaction affect prediction accuracies in genomic selection: a study in cassava.;citation_pages=1312;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.11.0653" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits.;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=G. Morota;citation_author=R. Abdollahi-Arpanahi;citation_author=A. Kranis;citation_author=D. Gianola;citation_title=Genome-enabled prediction of quantitative traits in chickens using genomic annotation.;citation_pages=109;citation_volume=15;citation_year=2014;citation_pmid=24502227;citation_doi=10.1186/1471-2164-15-109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. R. Muñoz;citation_author=M. F. R. Resende;citation_author=S. A. Gezan;citation_author=M. D. V. Resende;citation_title=Unraveling additive from nonadditive effects using genomic relationship matrices.;citation_volume=198;citation_year=2014;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=Anim. Sci. J.;citation_author=M. Nishio;citation_author=M. Satoh;citation_title=Impacts of genotyping strategies on long-term genetic response in genomic selection.;citation_pages=511-516;citation_volume=85;citation_year=2014;citation_pmid=24506177;citation_doi=10.1111/asj.12184" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Oakey;citation_author=A. P. Verbyla;citation_author=B. R. Cullis;citation_author=X. Wei;citation_author=W. S. Pitchford;citation_title=Joint modeling of additive and non-additive (genetic line) effects in multi-environment trials.;citation_pages=1319-1332;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17426958;citation_doi=10.1007/s00122-007-0515-3" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=R. U. Okechukwu;citation_author=G. O. Dixon;citation_title=Genetic gains from 30 years of cassava breeding in Nigeria for storage root yield and disease resistance in elite cassava genotypes.;citation_pages=181-208;citation_volume=22;citation_year=2008;citation_doi=10.1080/15427520802212506" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. Okogbenin;citation_author=C. Egesi;citation_author=B. Olasanmi;citation_author=O. Ogundapo;citation_author=S. Kahya;citation_title=Molecular marker analysis and validation of resistance to cassava mosaic disease in elite cassava genotypes in Nigeria.;citation_pages=2576-2586;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.11.0586" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. J. Oliveira;citation_author=M. D. V. Resende;citation_author=V. Silva Santos;citation_author=C. F. Ferreira;citation_author=G. A. F. Oliveira;citation_title=Genome-wide selection in cassava.;citation_pages=263-276;citation_volume=187;citation_year=2012;citation_doi=10.1007/s10681-012-0722-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=V. Palucci;citation_author=L. R. Schaeffer;citation_author=F. Miglior;citation_author=V. Osborne;citation_title=Non-additive genetic effects for fertility traits in Canadian Holstein cattle.;citation_pages=181-193;citation_volume=39;citation_year=2007;citation_issue=2;citation_pmid=17306200;citation_doi=10.1186/1297-9686-39-2-181" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. C. Pérez;citation_author=H. Ceballos;citation_author=F. Calle;citation_author=N. Morante;citation_author=W. Gaitán;citation_title=Within-family genetic variation and epistasis in cassava (Manihot esculenta Crantz) adapted to the acid-soils environment.;citation_pages=77-85;citation_volume=145;citation_year=2005;citation_doi=10.1007/s10681-005-0424-y" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. C. Pérez;citation_author=H. Ceballos;citation_author=G. Jaramillo;citation_author=N. Morante;citation_author=F. Calle;citation_title=Epistasis in Cassava Adapted to Midaltitude Valley Environments;citation_pages=1491-1496;citation_volume=45;citation_year=2005;citation_issue=4;citation_doi=10.2135/cropsci2004.0600" />
<meta name="citation_reference" content="citation_journal_abbrev=Trop Plant Biol;citation_author=S. Prochnik;citation_author=P. R. Marri;citation_author=B. Desany;citation_author=P. D. Rabinowicz;citation_author=C. Kodira;citation_title=The Cassava Genome: Current Progress, Future Directions.;citation_pages=88-94;citation_volume=5;citation_year=2012;citation_issue=1;citation_pmid=22523606;citation_doi=10.1007/s12042-011-9088-z" />
<meta name="citation_reference" content="citation_journal_title=Virus Res.;citation_author=I. Y. Rabbi;citation_author=M. T. Hamblin;citation_author=P. L. Kumar;citation_author=M. A. Gedil;citation_author=A. S. Ikpan;citation_title=High-resolution mapping of resistance to cassava mosaic geminiviruses in cassava using genotyping-by-sequencing and its implications for breeding.;citation_pages=87-96;citation_volume=186;citation_year=2014;citation_pmid=24389096;citation_doi=10.1016/j.virusres.2013.12.028" />
<meta name="citation_reference" content="citation_journal_title=Sociol. Methodol.;citation_author=A. E. Raftery;citation_title=BAYESIAN MODEL SELECTION IN SOCIAL RESEARCH;citation_pages=111-163;citation_volume=25;citation_year=1995;citation_doi=10.2307/271063" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=F. A. Rodriguezalmeida;citation_author=L. D. Vanvleck;citation_author=R. L. Willham;citation_author=S. L. Northcutt;citation_title=Estimation of non-additive genetic variances in three synthetic lines of beef cattle using an animal model;citation_pages=1002-1011;citation_volume=73;citation_year=1995;citation_issue=4;citation_pmid=7628942;citation_doi=10.2527/1995.7341002x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=T. Ostersen;citation_author=M. Henryon;citation_author=M. S. Lund;citation_title=Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.;citation_pages=1-7;citation_volume=7;citation_year=2012;citation_pmid=22506074;citation_doi=10.1371/journal.pone.0035495" />
<meta name="citation_reference" content="citation_journal_title=F. Crop. Res.;citation_author=R. Tumuhimbise;citation_author=R. Melis;citation_author=P. Shanahan;citation_title=Diallel analysis of early storage root yield and disease resistance traits in cassava (Manihot esculenta Crantz).;citation_pages=86-93;citation_volume=167;citation_year=2014;citation_doi=10.1016/j.fcr.2014.07.006" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. Turelli;citation_author=N. H. Barton;citation_title=Will population bottlenecks and multilocus epistasis increase additive genetic variance?;citation_pages=1763-1776;citation_volume=60;citation_year=2006;citation_issue=9;citation_pmid=17089962;citation_doi=10.1111/j.0014-3820.2006.tb00521.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=C. P. Van Tassell;citation_author=I. Misztal;citation_author=L. Varona;citation_title=Method R Estimates of Additive Genetic, Dominance Genetic, and Permanent Environmental Fraction of Variance for Yield and Health Traits of Holsteins;citation_pages=1873-1877;citation_volume=83;citation_year=2000;citation_issue=8;citation_pmid=10984165;citation_doi=10.3168/jds.S0022-0302(00)75059-4" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=D. L. Van Tassel;citation_author=L. R. DeHaan;citation_author=T. S. Cox;citation_title=Missing domesticated plant forms: can artificial selection fill the gap?;citation_pages=434-452;citation_volume=3;citation_year=2010;citation_pmid=25567937;citation_doi=10.1111/j.1752-4571.2010.00132.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=L. Varona;citation_author=I. Misztal;citation_author=J. K. Bertrand;citation_author=T. J. Lawlor;citation_title=Effect of Full Sibs on Additive Breeding Values Under the Dominance Model for Stature in United States Holsteins;citation_pages=1126-1135;citation_volume=81;citation_year=1998;citation_issue=4;citation_pmid=9594402;citation_doi=10.3168/jds.S0022-0302(98)75675-9" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. M. Visscher;citation_author=W. G. Hill;citation_author=N. R. Wray;citation_title=Heritability in the genomics era--concepts and misconceptions.;citation_pages=255-266;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18319743;citation_doi=10.1038/nrg2322" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=A. Legarra;citation_title=On the additive and dominant variance and covariance of individuals within the genomic selection scope.;citation_volume=195;citation_year=2013;citation_pmid=24121775;citation_doi=10.1534/genetics.113.155176" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=B. Walsh;citation_title=The struggle to exploit non-additive variation.;citation_pages=873-881;citation_volume=56;citation_year=2005;citation_doi=10.1071/AR05152" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. Wang;citation_author=D. Prakapenka;citation_author=S. Wang;citation_author=S. Pulugurta;citation_author=H. B. Runesha;citation_title=GVCBLUP: a computer package for genomic prediction and variance component estimation of additive and dominance effects.;citation_pages=270;citation_volume=15;citation_year=2014;citation_pmid=25107495;citation_doi=10.1186/1471-2105-15-270" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=B. M. Wardyn;citation_author=J. W. Edwards;citation_author=K. R. Lamkey;citation_title=The Genetic Structure of a Maize Population: The Role of Dominance;citation_pages=467-474;citation_volume=47;citation_year=2007;citation_issue=2;citation_doi=10.2135/cropsci2006.05.0294" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. D. Wolfe;citation_author=I. Y. Rabbi;citation_author=C. Egesi;citation_author=M. Hamblin;citation_author=R. Kawuki;citation_title=Genome-wide association and prediction reveals the genetic architecture of cassava mosaic disease resistance and prospects for rapid genetic improvement.;citation_pages=1-13;citation_volume=9;citation_year=2016;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=W. W. L. Wong;citation_author=J. Griesman;citation_author=Z. Z. Feng;citation_title=Imputing genotypes using regularized generalized linear regression models.;citation_pages=519-529;citation_volume=13;citation_year=2014;citation_pmid=25029086" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Zacarias;citation_author=M. T. Labuschagne;citation_title=Diallel analysis of cassava brown streak disease, yield and yield related characteristics in Mozambique.;citation_pages=309-320;citation_volume=176;citation_year=2010;citation_doi=10.1007/s10681-010-0203-2" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=Z. Zhu;citation_author=A. Bakshi;citation_author=A. A. E. Vinkhuyzen;citation_author=G. Hemani;citation_author=S. H. Lee;citation_title=Dominance genetic variation contributes little to the missing heritability for human complex traits.;citation_pages=377-385;citation_volume=96;citation_year=2015;citation_pmid=25683123;citation_doi=10.1016/j.ajhg.2015.01.001" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=O. Zuk;citation_author=E. Hechter;citation_author=S. R. Sunyaev;citation_author=E. S. Lander;citation_title=The mystery of missing heritability: Genetic interactions create phantom heritability.;citation_volume=109;citation_year=2012;citation_pmid=22223662;citation_doi=10.1073/pnas.1119675109" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="In clonally propagated crops, nonadditive genetic effects can be effectively exploited by the identification of superior genetic individuals as varieties. Cassava ( Manihot esculenta Crantz) is a clonally propagated staple food crop that feeds hundreds of millions. We quantified the amount and nature of nonadditive genetic variation for three key traits in a breeding population of cassava from sub-Saharan Africa using additive and nonadditive genome-wide marker-based relationship matrices. We then assessed the accuracy of genomic prediction for total (additive plus nonadditive) genetic value. We confirmed previous findings based on diallel crosses that nonadditive genetic variation is significant for key cassava traits. Specifically, we found that dominance is particularly important for root yield and epistasis contributes strongly to variation in cassava mosaic disease (CMD) resistance. Further, we showed that total genetic value predicted observed phenotypes more accurately than additive only models for root yield but not for dry matter content, which is mostly additive or for CMD resistance, which has high narrow-sense heritability. We address the implication of these results for cassava breeding and put our work in the context of previous results in cassava, and other plant and animal species." />
<meta name="og-title" property="og:title" content="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3497" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In clonally propagated crops, nonadditive genetic effects can be effectively exploited by the identification of superior genetic individuals as varieties. Cassava ( Manihot esculenta Crantz) is a clonally propagated staple food crop that feeds hundreds of millions. We quantified the amount and nature of nonadditive genetic variation for three key traits in a breeding population of cassava from sub-Saharan Africa using additive and nonadditive genome-wide marker-based relationship matrices. We then assessed the accuracy of genomic prediction for total (additive plus nonadditive) genetic value. We confirmed previous findings based on diallel crosses that nonadditive genetic variation is significant for key cassava traits. Specifically, we found that dominance is particularly important for root yield and epistasis contributes strongly to variation in cassava mosaic disease (CMD) resistance. Further, we showed that total genetic value predicted observed phenotypes more accurately than additive only models for root yield but not for dry matter content, which is mostly additive or for CMD resistance, which has high narrow-sense heritability. We address the implication of these results for cassava breeding and put our work in the context of previous results in cassava, and other plant and animal species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"PckdNjT35SmR6jQMy8sOeEyyyL0Oka16B5asgRP4-qE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460475","apath":"\/ggg\/6\/11\/3497.atom","pisa":"ggg;6\/11\/3497","ac":{"\/ggg\/6\/11\/3497.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3497.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3497"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3497"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3497":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FpDW7iVoVC5OcsFyCTC9jBVu4d9IDFYWwk2LGnXma+2Ev0dh+x2umsB\\\\\\\/O1TmKo338FDp2jysjXSzWZ6O+6Au2qCje4+g5w8zgUo5c1RSgVn2G0Vejohxgf4Cxkh0JpFK5r6Gnn\\\\\\\/luQS3JBV8Akt456HXhbq4xrVI2emqso3ptxYcPmI6ROdX4llNgndj7cd8nIEWj+BbGQj4a0KfKChBPxpl0zXopLjBB+keGpN1e46CqrGT0kC21PcO+aq+urBAITdIH6PSwLmE6\\\\\\\/k9IdudqHEXwdCw1hEr1AWWVnDlObOlJJbsfGXltjCLGXGawlb\\\\\\\/0ymanJVEARhTlu9QHi\\\\\\\/w0AbHO980Lt9dP6Z22gIiMupgX3bEvlTSo+SEfN\\\\\\\/h8IEPLRMxfmDXGMvZPcOL2OP5np2VQS8XFZYJ9ui93ye+4nN3Y2lLP3Wp7DIHhGOibg1oazGv9S\\\\\\\/FjTr5hqmu7w0Fu+apySUSh67CERAFLKS2gFN2pWOt0GFK5y6zrAddbxqekyo6\\\\\\\/0OxtTIq\\\\\\\/sLfw44OEiY3y\\\\\\\/DMYm5i\\\\\\\/Fn2lWbbtchF2O8o7Q8aao9V6iIZhML6L2BTfYlkFhOeO1IX0kedLCMHO3LnQ2xs3tve4PaSL2sZZ8Q8a6+jlfmCVvrBI1nG7\\\\\\\/Qeduq52FoY2gEBk12\\\\\\\/QciwPWuSlcpZY8Oi+js742T+MZN0Ej1EeSsFmU7A+UaQUhFHpzDclc7nUHV0+moq4SGrGJUwLFCJ\\\\\\\/fwGC2eruAPa0G+FLK2FN8CcCfTCWCbPe2Ug3QYZYebOW3AVDDFizw8jpqZo7k9Nl5V3d5yaapN0+HFnEHCL9Vz8N9VVdrBumktcaU6rqMCt+e+fiz3gwiKzPpQbEtsdugRNHyIYpWPh72LCRPJNSi86bQq9k5D0hJcfkR7\\\\\\\/Al81aCJvTbViS+ZQtaH3iRRDrjQgv5U8eop2BzlqPhGKuozg7zcvoVroyLjEkxt9BTZ5ZdWsh7OEiR0mAmpYb8tA\\\\\\\/3ardTgv2ky5fR6kXqRElx6buWlEl03wpNRJWW3dcBgk7UbaPiy1zZ1pH5CMRTYPtN3ufS4SI4fVKj74svb1Un9fMNwcG9fKFZz0WQRpIekFaK+7JiuGPyuFhGdUn4qHO3D6RsBGGJjeRFDY953M+0ol4JdkjE5rnQRnSJn0zBqhZK1QPC2OGhIfskCYrTFPBT6pIApuUuakUZLRs+pBcvdzlcpoDjY9qRgdTEwQFyOy3i4MLutR88gyTpXSVXhmxBtq+4+emiA+kkCWSu7aNovCCvexwmMU\\\u0022,\\\u0022iv\\\u0022:\\\u0022HxDzE\\\\\\\/ESzU1a\\\\\\\/oQWA13M5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022db741206c3e54362641fdf3557a39a35505c33429cae18b6f6b1a4187e2b4cee\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iJMU0uJNQQQbC2bbzccZaWjKiEpZbuuYP89L3ms9THkHQimkrbgJVah22e1RcFHw\\\\\\\/2Nwfjqd2x9JNsFaGw0WRWXpGbIAj7SF\\\\\\\/Qq8RXeK4NcICYM+GPYzWewgccyhQF+TV4y+1S98EIwitLPS8j+lAHuncA0lwJbW6Fp0shzq9OnvAkLuM3yeS3KH71hoih2ZBOA3hIYNuAiy+o9FvbETcCDSHUqSzip9baA5mV5J7cm3ggh4HYSLIPezUcMGAz+TB89eCwdjnebfq3uwZSX8brhRD5sBAhTKJyApppEgrx82TRDnEfJ1tbPVNcsrUGHlUi9Ebwg9LP6cxiU071dQeMyrIhKzY8z2NIoxLfrv+0cAexN43iDmu1Oq4K\\\\\\\/Axbl3\\\\\\\/MQLW+ELBRGu6Nd0b6qm2\\\\\\\/YE\\\\\\\/eQe9BnEmnxF3\\\\\\\/QupoH9yCSBIZFtDFph6ZpEa41KL2iZOsYyMQa4Pt0JTSGdAt1OClL1RpGpmvmU4GnoY6TNSh72aVaJEv0QIRdnzs2Rm8LKAv8\\\\\\\/oi3HH56hE6CZHN5BCrsVNXHafmvXqid8cikPhyypRo3Tu8xcc65Rq8C2QrGA2\\\\\\\/2i02VA0d2RkWS1B80E7qFZhaxPVQ4Z\\\\\\\/GOQoWuBRYKIa8AQamN+PnTzeybwFvT85lihRTh3wh2WhL9npUvvyPFXaA0pXXMtj4Gxlbq1OfbCTNLykAvzKLa4bDemFtubN3mZeHYEhUfUF+mAVmqIo231gV0zx3yXpBPiNq551RwMOQEV\\\\\\\/2x6om\\\\\\\/Szig+swH5hKz7mSTIS09PYy9x6kRWb6Ch1mlWWb5A6Geh7F4hdnt251hbCG1S\\\\\\\/t9LjJzts0J1lVKdFO8qFv8sN+iS7C5LuahMHTB\\\\\\\/E\\\\\\\/75tqcgN5u3EtmOwKbpKPsscqOTAgeBUVj3wanPof8hqXB2g6jitQPwyhAt4Q8MvvdsJYwCmhs+ZU4x0TlqVE1Y0wlqwoQMbuePPcbwYtR1rvwuOXfW8VhMRkhmNe1caDUGlhG9j9TK9wzazFjBp2CNGXHHKQAwOJZhlycqLdlERUt1ULWlZkPjjDVSSxJG5LtRTunlSEj\\\\\\\/7OfIjq1UhKtaah08Nf\\\\\\\/1qVwyia57as7u3W0gw9AH78ZoYuKzWr4zzS0OP0OQbE7EbtpSRxpyAjGj0nFQqB61obKpvi4aD7Z5UcSdZAE0Zf2N9SX7kHFLqvovwGzfo3ZwQX5Q6ddwtiWozus9o9Kn8TNdDVhRagPWynQ+iuZ5tAlvmdI01+GJ\\\\\\\/kCh08\\\\\\\/JGxdXqzI2EiYUTMN80NzV8VvVprtwP+Z7PCPHhAFsxRDPJBjLCEqiwiUQTLHIwxIHWDJI93N+H5MPccUx8dMGXVKeYAaJ8jdoQTBrPhGmKt1SpRlhVcnBEmGgg5qdZc010urh3hROjYsHAN4ZpHdeQv88A6imimYOlL3CUtrBa0cgHnjKxdH3nwFMHB3WLZuoYH9q1v59vq38Em5ba63G460HMv8qxFa5e4DwvI\\\\\\\/hdQKXi+RyV9ngmEevGdMI2INSz7Y3W2RK1EdP61FMSKadNUHnMNh5PeZn9n3BoluD4cyIB+Jcxy\\\\\\\/EQy0b3tJl3PEeJh2iGGgB+QG8auSQUb78NPUb1oy2cU6dmvZFwyQrCOyBnBtLT2MXsRT9VCIFEYgs1w+Gok7rly6VRDyrkl\\\\\\\/xrXexUia87DFcTM5dDGHCWX5G85I6VN1XR92R3+rO8FJSEFF4tlKNaGVGPRNwGrv1xCBCAym8Fb6x4bWqofii0xNDQvXjO9RIHVKOYTh3APPuUb5EIShrnuKqUY8QoQJTFXFAss\\\\\\\/Q3Hhas8r2GKodOCbyWXHmuuHuxtqIEDTss8IAEdQnPX0ofGGRu\\\\\\\/7dgRO+jzOF4fKPH\\\\\\\/ZXM3JUpVX1jITTrQGu6k7NW6bsZ340oystN2Twc2iuc+wfAUyRr4yqzYXHQovVRzfEJETve56rInn+dNNuU6awt6hf7yY6zBhDrCwuTeLkYEUMvSMztpE6ZrmEzHFKwhnJY8hiTfCTGiNx0V\\\\\\\/r4jCBlRD6SQaZ+7tW8w9+xaczT0IbvsOsciDqsaghBbuaPZZnwG+a+1XCUeifGxKO5r88L\\\\\\\/5Z5AXMx+OSFxi5oQEOASC8VkkpI1SWkEw0Gv3KnVX08msw5h4qSiLjtx75VsT8Co7I64EuheH6zUZgFWlS7QtLYipAH9mI4BgPBDQpKv5h++O6q2S7H+Lk9ZO3J9L+cU1oHwcpaMfoy81CJqA4GyY13K4o0z78JEkrch3EOuEsBN+njK0m6pVWE22BSVlAJEmRg6jk+g67hmypjsUd2E7aav6farxzOTTkKxs9MHQM4sfKjrUCNwiss0nee2Wc4ZZWQqURq0B3oPpimWusdlS3KtV0jV0FO3X\\\\\\\/prSeGpcLJUQKU\\\\\\\/9nY0Xl421IWet02ZWfk0FvLFf4PBGEgev\\\\\\\/9xvfiR+NLSCrnRizuK5kX4Mgv8l+vDgBXed63inBi0JeFj7a1IM6sxp\\\\\\\/i+FKMVVokDf13eB1g\\\\\\\/GCvfHaKqkPs+LC5Aknrqy0wnBWxjppUeWYF2NpGaG2oh1EnyXbXouj0VInmoAXZzIpSkTDxE6pII+NDpwdAB83xHaIYC9VneWCip46jVC4gYT8l0RaKqMxQXbGA3rHNI0+G6TwjiDhhxKnVT0NDUVYrwqz\\\\\\\/QIqbu7RliHrQxh52X4WJK5\\\\\\\/kAvRqRIqLLID9BLL6i0WlrdLFF5itiaJrj7s6XEw7rK5gXLgxOW5Htsjdeayxh5tnjSNsNT1kv\\\\\\\/4j9xAgaaVe8KHWbUfl4Z+jD+HOQ4ZFDVoc45D1\\\\\\\/jeYQX0smE7SKJinuW\\\\\\\/ejwN0ne6T73mJJd5pW4Rn1BZVAdJ\\\\\\\/vXsqY6cLWUPkoUAH\\\\\\\/8Yk+uT5UxQo5dqojJTzjEg5W+yuvfIhPluBGDGBgXPMFbzShmnyEOGQukmWYJoa5WdJEXFqzYoXjRUmSZUnysrSuVectv9SOaabc\\\\\\\/gEWrlJjrE1iZcZOgmSQ08oJ6MusQPfWI\\\\\\\/B7Fa1jxkQ+ltw41YMqJ9rUjr0na6L96xCHbHzDmkpP08gIIpvXeYrmZv68b9jnQNfRyET9TCNDbkMGRPr8NC5eeEw0eWpDvhNAEZ8T8by72+S0qQjaPaZ0NsDDHtcudBfDZGD7YQaRa2xPGOohkNDu+TC39qs+JXV2Z6uvDDLcEvhRHCJt7hzjd4Uc8gIxqDGS+KLK9LK+w+RHnrvg8TF6xr8TkRS1\\\\\\\/Poc9AAi6Sf77t+CLj9cyvWPkubqCjrb4OYK6T22rQ4WZJbNR0w11FQQQF1giKJa\\\\\\\/2NnRiUtbLwAYry8NN1te+Ad0wNVt5\\\\\\\/mEaVSy\\\\\\\/aoS1i6JW4iYNEJ764qQgxWuUCH6+Mx+3j8HAOI6Jpt4NLEncm2JT\\\\\\\/+sJazJlsuoowiYsWJL+b4PIv5SZQi8O96LknPP9+ZEqlcKy1ebzSmog3eKHH19zU1uF5xiMz28pdjzR4E1uIbFGmdCxIcPoKQJ6likfu1fnjF7lJYitUdJnUbGb\\\\\\\/N59n89KGOp5ef5KLwrRudk8uGHf5YNm2VuVu9DAxMM9PEZcNMYdGBtaQ5I2wqMjttnwgyhkBKkgMAqGEtyFI31FgIyl4+ZotBnBHOxsgqeiFOI7QCNyY3SP24HH99mjUy6rMUy9sg5UVXoaR6RmoOWLAddw3s\\\\\\\/jQdPaR4Pova92t34UFafbb+GWLGH0GvLgZM\\\\\\\/spY5RFO3qqP3msovdsekPbC2Qp9mRj8EIZthX+nOsoVrQdcRh0Yfq0z7VeS3xR8saR9F1WvJTfsvCPimItInVf1k+hnIopcfgrdeYBkyD3E3rf2KEDInN+\\\\\\\/MKrf2EdQv\\\\\\\/rbGs3fV0tiAkqabU8iSHDVSHxX5\\\\\\\/r2zL4RTC4ELjOs\\\\\\\/T\\\\\\\/j+6Oj7eLhEKxB8WRcctjjBQObhvRP2VhLdRFKwuXM4kNEqT5tZxfj0AWzvVY8T4iimQsdenIq14nNXZxv6MIc3kYkkDM0cfAag0zgyPmoh5u6h+jUFSDgkKSxE2\\\\\\\/xFXP1jtn3SHanbudJ0I6s8Jk+MtVP7TOcSwvRhGySFbkvXpEo+nxHblQZrNeCv169TyTr\\\\\\\/GE9vLusiJtAKVrfzyWmfEE6dOuZnGMZNtnKwaI45u\\\\\\\/2hac\\\\\\\/a69VJItxcKwk0b1wQmCbk3TVGxFaj4eBtBXzLfRrOIGPlPnYhcdvfrp96GLL+RgiD13M2EMV4oy+YUqB7PqW+pI3xO8QcoihABClAvoJ9zID8pX\\\\\\\/aSDyACimhMQ3UgCtepKdCVTduvOhzzWAnjO0h6\\\\\\\/2Y+SE0vFdIwDhFdwzO1IcXyn3Y5FAJugdyA9mB4J86l90Vvpmz73u7ILL4nWIzOiqXKpocOTCjOxvT6Z6eKJ3SW7g4lOOFmabsu09nlqnFuuhJVKUHbLrM\\\\\\\/KHHFcaglRwPCTDa1sWMk1hotTJhljRx7ibh\\\\\\\/LN2Gdbzj7Rwx\\\\\\\/p0SO4FQuxaVRD8hUcEmB49pEDMRnZgbcobWZaed6lWDTSM\\\\\\\/2aeoXJYfUAJ8bWmYhmqm7KrwlpEJxOqwtCgDYH\\\\\\\/gVqaog5W3Wg6VibHLokTK+e2yU8iIB99mNQYzlG9QmqUJUiyjWPt2xIdwQkSAE3AVUNggNIWnf\\\\\\\/NFGkunZex\\\\\\\/NvSb0h5K5vAqMx34cuWkfI\\\\\\\/9hWH2lCGp5H7zHicgHBuvy2BYyCv9RS26uivPI1c9U7vxZepCLcmdSg+uUdpoFKXdbYUimJ4D4xMBJkiOPyFMRwgVBwc+qzEOzmfT5I\\\\\\\/6UvXBT3xYZe89I7rdNhz3TyFVsaiXjKGeG1KG6Zdu6KIo53W+uBlgArRMY+ZZ1fmkPu4vNXb4TR+W77zoL9ZS+72dV3k+gQ3Pfz0cb7cF2+FVFeLOvY6uUwElPrZPGzFeYW7+e3eP3ExZMcEK6WzN98aX\\\\\\\/PTmhilgFXR70PJNMs2\\\\\\\/ixSwxrDTLV0mfraJTNzhIypZ48huDMFCUUqMk83gypF8+LzFeLqinuZypatp6N1VGm7ouTsKrhcJzh2FMARWOOrxAlDk5thTUyaTKngCkhG5pvoUr2TIGJ4JYP7tBi2sZmqTGeihXamK7F8AJbvQpN0f98SLTsbp+km9I8WCjMPv+q2UVaWYAQRry9EItyigJ7V1Bqx+MoHN7VNZkctKFNHyR15U29\\\\\\\/eZH\\\\\\\/S0qNxTnHajQ7nCiRguhpmKIX9+GpHBX9pv\\\\\\\/3KCowmO8lEPp0NCny4epAY3WBB+1dJj+R2cAS2qtY+Bo\\\\\\\/fV1lxHo5G847yF0rQh7cVuqPWuouhm59fVQuG2g0pFfMcfLznfMPOyXUQ6Qmp576oDqObS2oXKdxUyj3N1yyGjT9OIrv0YUYVk\\\\\\\/6Sw0wWBtOKE5Dvsg6juZ4CxF197UEnBgkGhC1ZGeZqHfs4TvrrWN9WMoMPF\\\\\\\/J8N3GPRNBZsxMjpvuWX2xRpg3QlY0uiProZF9Kswwbku4o0iqaimm7VP07AB7UaIASmpmaiNrpsxf9zAwY1Emd3D5N43Uqpsb1btSyp61zZ0JIioerLXn6XW5\\\\\\\/pVln7dCoc+RRr3F4aG8LnKdarxZSpS97NCBXBtQrjU1cq8F08WxiMzsZB91SgC4XybnfyDqB6fimArKkwGWbzUgv3r9gukZjQD4eFGGlFGFyilHESjV5CTq\\\\\\\/TjXzrGxk0L9C9CuTWr4P7MS59wRaFv\\\\\\\/nA71N8dYOOS65yZTn90\\\\\\\/J1JfL+3Cpr5o4SJvfPXMVl15RokCR4wkUCBKav74itLObFWM0qBolL0\\\\\\\/BDeX9qUhybY\\\\\\\/tZaofXhMX+eHrPDurSc4A4go9XKqrdLIPqjTSRI\\\\\\\/aa7LSZ7zxsDMvHxASFX34NyHKm4FoaZ+EbxKy9XO4lsEMvDHE+qXTunoKz95yB7qd73ujaNx3GVokBrvORfxsViHo8zL75nBCgbVIa40or25abBwKYQUCvk7XqP9lO9Pc2YwJFXzomwkR8rlXb9hO+h4vObiw5\\\\\\\/Ewx4NCp1bBn3fZiZI1ZAzmHHeDTgSDVVb8OGvTcy6x9hk4dOhDCFDqAoS5ixm5\\\\\\\/qN00hMwt05IgMsAXTCUFZFIyOc1ECPFExvh21WbjvH4x2hzd2ScYEyRsPLNMiJqG4FZ3uW+xkZhOV5fjSxffwl3UD2bWB7m5fp9hIqLcTrGSaMvZ5ebgl4+YxqmbgucWjdW\\\\\\\/o7tbt+Bc9\\\\\\\/Em85+U0Lh2uXlMi3N9JnHD7vwWqpSv1qcf0w+ta7sHxbg6b4kWlqLatIgnOEZ1AVtbvVrdJxCzdBxN\\\\\\\/MrfIJHZk5J5Lh0UzB+Sj2h7\\\\\\\/jjRshNSGHoQYjKyBnOuVefelB5zobeWgKDop1zPRodfywwCj3VYTJ00TEYXgXp2nKFjDAbo83HWAUHVX+xv2RkJJYPtJS0cgER412uRKSqnPS965dtN6Rj3\\\\\\\/jdJf\\\\\\\/61fNtDXt3iJJP\\\\\\\/ZqqriIOsvrngTOB9wn4QG7imFFJPLwM7QuqUSwMawv8pPaL\\\\\\\/1jxDZk4QYRow6j2gzcuqP9qVJvOEvElIBkRVY2Ra+M7LtUIU4+U5DCXiGMJa9Hl2dDWWRJDcdG6EFo9LJtwLZow7kfxsdHawTr5mrYq0\\\\\\\/zWAEdvHTsyXqO4pF8gB7xmZ\\\\\\\/QXY81G3duCeD+7yFJQvO5oaTptlD7dfjRzmAkT2HClWzgf2EoYcOCkhgM39dlgDv2XfweAAeemnK3ic+mkExkAH6Eq5GU6rL+lgDbxgFM\\\\\\\/zOdBcoqbJj9Kghup8Kn8qB3JYlIoU8z5tiTfJNj5Qt1JKc1pWyzuX36cglF4w4kZovZckMIwlHQy7yWvoIoFyqCQ0s6vHe6zT9yY6D4JxWOKBunnPkf2XmPXL79IDO9j67TCDUOuKFQlBSbHbOxJ3a7T7JiOBQrpOLUUOCy4NjVT\\\\\\\/QwDfe\\\\\\\/q7EsMkPT9fjfU+OX0FE74+cjdGscl1mh3UlymJcV27g6x8tEHN76CNV3YLWD\\\\\\\/aGL20g5Qxnaix\\\\\\\/QRxZ40XgOoEnw6Q7bAMXpYCqoyDq+7Q\\\\\\\/wuqBKDvMZfVN8VN8psm6sDnzoGJhFXBztx86rzqRiNqXdcuiIY3u1TirZLPia43xVq+134IhqHpt2uQvlexDsPUWf3hd9Mw83CoYW04KfyyFmsrW429y6Xg0ftijYEgaxjuH5B+a1SArBpB3RtDvjd0rsD2bc4XkMdxYM51UnhNVMbj\\\\\\\/6ZFm+bPmpSYT0CH8yH91kFzJgZIXaFF36HuqEKqwVXWZAYOm9uUUA4LZ40X+lIU1ofLmqQSeIcH63T3mbZFEdyzbXFFJQDG6h9Z\\\\\\\/RCOWZCFkc4Sc1HHownWOL5044IzoaNzxRCmAo62DxpxbGDniYDIIciNhZtky3t76zW6aunorAm0zUyPRCWUtYKtzi7uuF8obRW\\\\\\\/lAC1xWAlW7T8l559E3clWtJMeut6\\\\\\\/jVuZ3ojjPB2qLlWNPzgBDmtyq2No4EJ0OUPgkbe8PqBp6abls9q0HM30SIWfmCarTbQeKviGnmwaXH\\\\\\\/pqJWmFQnXjVlBh9IeDu2iOiZDzi+PYpEh3Qw8NKp8\\\\\\\/K1CcBvIYo1YZ2bPIZOI6gi4u4HPlLOITD6WWLY36shAKm5auYE+f3rXFFA0O4MfVijaXEcnwm\\\\\\\/iP\\\\\\\/YytMsxMrNAfyQIIRUQ5461vf4V3TR77R\\\\\\\/8BeYaljkaFG0wKWEdgnvFuHlAlf1SOSV4tewmEHAaY9cz2wTNePzIIYQ+ZWOAFHUdSBaIqV+5JJhA6PJBTOMuS0cDz9iJ0OrmETwAJ+TOKrlRgDgye3x1iCoD3XTN4SmnZ5RaALvUlR6s7brIoRDQMfTjfrEgmxJmyW51my6tBvLIMOZu9V+aOWSpfhVTvmYPo0X4AO+8h9LvZvWY00WS5glpqfAJzdeqguPZ0vKqv4iCvTKoCkqIpVpWJPPS43qWkOvv8CX\\\\\\\/btt0+gFoSqjLfltDGWzlEWVjSqp1kaiO7dTaJWLqRhE9uyhevj49qD7yutP4+wAfAMbVNTmbis6\\\\\\\/npiuml+Kl5QMOlMQLpTELEUtn\\\\\\\/0r+iSkJ6tNHslzQlZqdwqYPm+pyp25aIW3Xdtqi\\\\\\\/FKBfObI03X0b60lVNIWS90uoDZk64vagtoZm16u2icfcTveMhluvQCW7E4U5VsnHfau6+V6gcXkdxObkN53htAqF6vZjS4JRbS5Levk3NLza7p7XaqmqKF+z6nSVaYkxQGQ0MK0XNqzXPmiFKs1dSaBLLkzddcihTrADHtDjpWhIXEslMdgzoG7n8f7xrzWqmkZeVbVIp8UgouW00eZbKVKTe4v+mM+cTGMot1fbddZ4Ks9fFn8Nx1lUqjSliDiifVf25a55NmqjrpPBlAr9TCJLQ+q0T1GA8+TjIx6y5a8C02wwbuQJm6pqRd8YywQWy15rkLPYCWUqcdKae\\\\\\\/IK2X0uB0qs35eD13hPhBFOflTJHZoJiBAIsAuRrRZ9AEtz\\\\\\\/mcLAU6eKELegR6c9Ro048+kAIsqDaXpUnLV8Q8Ct3d3YDZVodZQ2yYaJ+eWjKY1QA+pRnMH+UNpPTbFiF3EY+Yd0N\\\\\\\/88MQvpFjyxhE2VsfA1tPGWsvZJYlK4r2c+oSKfn9FIzKQySNaOSpPMf66dDDNIpUQ7EQ42FvtDzHTp3\\\\\\\/eikAq\\\\\\\/ClITGOKnH5xtVRTgHbvN88teqK6SlSC+yj2OsXNQlQChwrPV763\\\\\\\/YEqPMqjSUkPfABUTk9aybZa7QRi0XsSDgdoDlcbbWUHvEYe+tbKUdt5KWNA\\\\\\\/uH92NR0pzTMcsK3a4\\\\\\\/wpGQ6\\\\\\\/TueJMAbH0CJuCysfpFeCdXaE6lb6XXY5oD49LVDdW1E0i\\\\\\\/hBk+tF0z4fZqtrktbFE6OFDdzeBFWlaBuzBujiFqIExIdtjPgapSdRawxRNmUdPN5V\\\\\\\/gmXaHXP\\\\\\\/kVEmRenT0Qznn6NRZIHjkPCyONQcKToja3QEaGaesiky563ax5q1tZZN\\\\\\\/zP4Lh01nhS1ScON14+RKYfn3pa3aVPNhAKFBt1UN8U+Hu6BFfMtwCOObF00iDFYp11omK2GKJIODFkpaYZidj5OZ1yMDocE2UM9B70HRyzeyYS+S43Jsg1yaI47MyC7XRohmX6RSViR\\\\\\\/mtv8RRL8IUj+Ze10eiaOv2+vIfehDafdGT+NaLwEBCj3yewrlACl7qiHVt0MQ1qCBOknIgjqBlhKMhdbBmLXqvVhu3HWO0bqQ7rNYdCosw4M5LOBEDEh3scwUOTVh+n4GyWRifVOsMFnVfsH0HaeoiRtYnOVsVJef24uFwL5qSvAF4w952QW+wbRoC\\\\\\\/gglCihVJuQovfkCNE6RZxoUBdYvgLr4jVAfXWuPF5Xkofk6Y0hq+ZiViW53ubtT2M3WCqju9ELyg\\\\\\\/7AdSN2oQ5S+goVk2+6KATku5NsRNzBBq3wnkBRM5swlMQH3DU\\\\\\\/zBmsro\\\\\\\/jxR8P6lg0OJXMdd0jMlC4MAVPYRPZ07JycEcp5VgOji1fQldGeb8EOYdrK\\\\\\\/kAu7mJd5AMcANO5lqyleQo5CPfvxRDNNNheBoMTa2FzVOOCkaKhl5Pvg+Mvr5xCCpFejp+9g617dc\\\\\\\/lZSAr5JI4E8XPbjQtjejkhmrTZbaiZv1qw4fHSxocyFMxl0cItQPKB5vwUTKL\\\\\\\/a6T+49qcFjNqr+gS+u\\\\\\\/UQKKRdntSy+ucWWY6pNX0LkAAtz4s\\\\\\\/ZcRtI7nG8+L\\\\\\\/+l0qPMXFaGDChr3gBmb8TvIEyyh7fFQ80Vl3Nsioh663IpA4a3NY1ufgtx9qwwwMgK0tGZkzwlj1AtLiSxYII1bikXAxOV2txJjeIGvaefx2sV0WfqHDOodPExUK9rN6reicaXjKS\\\\\\\/VluUW48kIXbhpphL4nOWs+NQnnT4T9lw7nhoY9Jj44GgzoSXwj09nTJnRhLM2APH52Kzp2EbnW7OIVjYciyh2GruU3ncNpEjXlCzRKt4PVnQzMhRjgMiDtCAbadsddY+TIuGZ0meMW6jJ4qxNHGH26KUw6On+QGEJ8dtfh\\\\\\\/Vu7BDYqafgLZ2+rathJXMf\\\\\\\/W8cMJIgnU1l6QeYdzArClGpQwOnWtUgforCbjlx5RvO07244qtcLbIsYghzJ8zJB1ciFMssJIEyeSbpCmKNRxeHtEeW\\\\\\\/cVOTf0IAen0kt+rFuM\\\\\\\/U47OO2W+guVTTNfG9TZKevWdRZvgaHKQutDhCR+FxfcohtdT5E+VF43yFlz36r+9MO5O8J6cs0P4QJtNos+ze70Yo+4JshHVHEzfbB04nr79eq+0vwWa3NSy2UtLGkuIcz0DhwJuZ\\\\\\\/nwUFQRtmDkYR2f1XL2tyhPSLW41CJZW0waiPUUjGARbytrIKGFLi3X1zIWsBppTso27CmiAbJOS0yFjxc5XllCqjh+WhJYlp5TQXpFt9ITGhjQzOO4XQk6FeilyTh7z46XGTwlxeTDpTimDx9fkNe6HnBZqsot32Z0bdgt9N0oEeUV3q0jxMkPweF3Si2PeB7Vu+ptp6truD8ZJMXBPAoOjgPDQAtRVoPLJGVa4iIs4Wc8nPdO67EaIMmyali6uJXdTQJNAeldWcBOsI17PNTUiw+sBIK2QiK745gtTk4ZZTrjZGUs1KBIP5l3z1JzsfKcFEH4L9n0zFhTQhIGzHxJ74HWXDDJM47Y6CMGJIQhzVPmqf3zoPUfXyRpYPQXZD2E5TIdu\\\\\\\/KK3h248NbMcflgFcSzfKCBmG1Mthd+RwD3ecpy5uLSIxqhE9JI9Um9xrgXCRVABy76zKyqcQrCui7gP5b8cnsaBKrdFk3DEUCluDin+O9lsEreQ0j\\\\\\\/hyQ7oKyY1o5ATuz1j4h85HCbCnkyWAUiTDVvVNTJdz\\\\\\\/QOqQV+Fbjoje8M\\\\\\\/UpXK3sk9P4a+JBuyEmyRIj\\\\\\\/4cVBXFDIpL\\\\\\\/zFy4zwie9Uy1a5OUfWvydeSS4ky7AdLIvLxbHsT3rT9Elvs51Dh0UJZVbn668gu9+owcxIVY1VgQkFg0hm1LuTUDm7Oj\\\\\\\/ugyfIbgNB8shYg71nfEvEy48QSGD1hm35eb1JdmVGlY6V54QMqDUXCECl8QUnE7INS8mMZVDhLXkEhz56ijEz5mVRbj+eHaQL6egHk2y5iOA4lwEEuJ514dSCcmwcQmFrFSJbgu7up0wt+b7PJ3LWYAiWkGJAvsMv\\\\\\\/LEjctGIX6ZiuVZQ77N2jVbXI6UNqh7QnjIfHGxPTr7crBJbmMZt6uMbjLFVURWZA2JA0dsf7O5OGE6tKF2oU9ARYuGun5qQH3bWgg1nRZoWIynyt9X7KrCNNbCVztvgwlD9OJc7hnNETPD2D749b06dQ25RqA8qB+Y70rfPvwpI8eFmxzQlU2DoHKTW3wb3LhujQJzpYPZAQUa\\\\\\\/C2PcYOcaJ8\\\\\\\/9DJWr5JO0sLbO88IK+WEyU3kRB01wa7cYd4gn3ppBhEMAMI7C1l2RbgbdZgkk2DChF92IpC7xl\\\\\\\/veNXadQlEkUFbNCHZaiPmp1sh5qKcyj4RdSXxXbAxzhs1\\\\\\\/skr2JW6RpEn2USsXkdRPfwfwovuD4UDXavsDaY0ihmIzRiMydNcJZPHUAS6aD3TAJNqUMV0Gb2aQgo2ssuzdYa+MWrOsoyx6gHOu2PU8OtlChJC+JXvF5Qw+WdlatbwYTeLTalq4wXWUgutlChnaa3\\\\\\\/eGzY3mRBqDnFo7B7sd4aVNV7QZeY0htDSZ36qmJNUIks9oDjVVmDmDVBMb8lsO7gbgnJq5T4CCR5dvp0BulqcMOabB1F4YIVZlRXn4XmNd65wLETo\\\\\\\/7XwDgE4HP2IN2S6Cn\\\\\\\/jdwXjC+XEdZ+aKr5Cb3kDru5wQKxhKPcxn4+kATVtUKqZi7iyyVWe66cLRLbSumd\\\\\\\/aP0Xe65JxbPTaisHn8eDZPwqubRqAqNVSqg1wEWlv2vJx2IJwx4z2A3QnUyWR41E6pgVx9e+qGd+LN6WuNsxJDO0ReMNodb0VaAD\\\\\\\/maI8lkeME8DXr+UCn2HANcM39pbq1iaOdDmDIXD0JlY3DENIA+17DORq2I3zG9mtsSK3lHsZJun+8LfBGCQWK9KYzl9zvbajkRGxeah5k5g4CbInJ9NaQTzCjhvbvNnsqcG\\\\\\\/kPzUf5iiKQ7T1QkWw4tZEo3Rh+vwxOB9vUwYEa86QUz6COECW1steAon1NznwvYbFf7mD6O2+rI+PwU+YL3+AyeL3wtpItmzL1c5nSyjkYh3q7pR7aUVDv74zOyKzSaZOLU3JedfiRjbFm3CrAJZCaevkKnIcOgJP+XtR16T5xWDqQwC3RWWAP0Nzm2Eju6iW3LCGgQWjuW7U9KEchNlLelGgQwqXiLodX39EW+HxT3Wfs6DdvPVlHF4oo4i69\\\\\\\/X7BNZjO7XewTSzXMp8YJmOftKqM\\\\\\\/blecuzVs4ndHJBKKx2Gj\\\\\\\/3ltxMiDokxWThhTdNpihLHR1NCg3F6sDSxHfxkAd45C2mDs0PlFjwYvGpCyNRRggWBXIUULxCgRSrh\\\\\\\/OFc\\\\\\\/YLwjF7+VUu8XMvAT0uOu989j81bRYFbYJhxCVUVe5XL\\\\\\\/gOH9bcU4gPcaXB1p3\\\\\\\/y4jbdJHil9oHRNLyCxtQn3cc1yOPrxEher6VbTwCT0rbbe65yDmpzWH9BRSHQXlNykyawbhH\\\\\\\/ZWb9TbrCwTFa8RddI51bDLdo4if5U\\\\\\\/073W9QYc37lLmTpJ8WUZJ8lZgusFFN343GpVBAeA2Kea8npItzjLIu2iSZqLfRzDv9DM9phNfMsQxJIHoG1oK7VFyy6tCw\\\\\\\/x2fNf7kv0K5OX8ZSXVgcpBmMQJF6QyYUqzeBAnd2AyIA3NU2ZLO81afiM41qsffOAuIIO5qXuf0\\\\\\\/4sO8NHzAzPK8q+gkqMGYjmEtsCmB2VFT6UZXsO3gdlddkHsL0L46xN8mfejHnF5fmol3fyKKx8hP+tOXhm+KY88lQYzZpUcZkY0QJ\\\\\\\/jHzcAclq2TEHKUz3gR9oJJ3HEHiGA8Lv7luwS0zXXi7YA7mGcm9qBVHUaPUnziXInoTqwqDgrqQQTWtF1e2fZcm9osUwwZGXH+Kt5cKr6C01ozSFiJ7Diiim9TOXuK9AV8pqguimzsYFUURF5YIfVewBKDDKDy4JCnMPMb5yUBHDwKmSzkkoguIDiJBx2g5mo8x1h7ocBrwwzIk\\\\\\\/vXbMUDaayJLv\\\\\\\/TIYabsHMHM7GhWdL2C1YqwiPGSq\\\\\\\/nppPodloyHmDg7s0F42sEo10+rADS+23\\\\\\\/Gc8GYcQ9p4EO1N2Mc8D6qpG4TEYjI\\\\\\\/spHsbVi82prnKo9mDCv5Ty\\\\\\\/MPYdeZZXJOQDHZPO6aNpqLVZEoY9J4Ug6hym4k7NhPSt+x4Yzzoit\\\\\\\/PkXtHmHoMaawpBdZ6Puzu27pKXTSCz+W75PyZSsVPMke8VRNGChjSGzai+3OseHaV+JFCheTuVJThHVc9Cp9UebrI1zejrTk\\\\\\\/NFpemoJoqJzs\\\\\\\/1dfM79cL3pSF3TNqkyk2qdFKKsad5uYNsjeZ4UJ6Yzf+vWjGTVHphTSr9xkaV3W+nkYXsYgu3+bLozS0lTqJpEotdDsUfhJldSr0k2\\\\\\\/Ro8nWtR6Y9i1Z81BWr1uf+x8wdWz\\\\\\\/X\\\\\\\/8BT1eYthgaYwlf1dlQ0DIS9FXm7ZjzHkBKwbuP6mnjFRxfFDUV\\\\\\\/XIZRq7S\\\\\\\/6ri7Ca9MDWLlpzzwhXWUw1mXDFIoAqJ0zvMMDXxpbSn97IP43BS4ADeScLItpQb9viVYn4lmJ1s8bU\\\\\\\/hZks+GE+oaZRU3YkntLaPDb7GKX6b4WalXR\\\\\\\/s+mddmrKU56l9tSR9KB1rZ8IDpdzBWAF4bhZiCAHrHIqd\\\\\\\/7\\\\\\\/+mD+Lf8fZIEiQVIwwFh6SCfAjG3v3F64iEJFEEMJYtAQr59O25CuiirxDz\\\\\\\/ay\\\\\\\/odtCKAGotzJmHtZIhHuc12B2Km1ue0NlkG+WCmS570zCtEf8PuyxQza9PTpVTMNi94LOxSUTpiaMjb46JDk2v2Kbxu+4ydu68QW763nbwvCS765groD7+l9jP7zrAOuII5rbiqsFcf402xDmlKtSZd86Tk9xbbGlvFyR3RWqf913BQvpMpWAe1V\\\\\\\/Vl6tVKXObXoUsghQAGIP+\\\\\\\/LSRtFcilRO\\\\\\\/E0FfMQ8APWHa6660wXB+G3\\\\\\\/EsBqSg9D9V4ZtszQir1aKKDl\\\\\\\/dE6SdpcqZwlEJ0je80lYWp1p6XGE4BnkIkYbdttNcGaFJZLOzzayxpQRgzsA6AyviqIFaZLiKIm7L7+KARI12vPLNusibuH7++5gt6g3ypLGS8yKGJdyVtp7X1spLR0smk+x0ZrJeY4r7t+ZHgokjwfe4fDc\\\\\\\/QaTI04e3atwRqcAmCec4\\\\\\\/uetXHza2G0Oar5jzBHaPCHiDFXn8LfNTKRAgWfIUkNUhzYwK6XxCSfny22Ay31Jgj18KyPnqXYybX98dfRu7+2PAq3CuhcA0n8I4jGNCwCBfrEheaQLcSJwsane\\\\\\\/BSOwprMWnEvzVowZ1lIileEArBSs\\\\\\\/LsTw6ZtR5PHAmHUd70b8H9od6Y1ZFoV+AExb4BkVIRfmEhvxJplFjQkoeuPRiXr0vRZqQcAV2bhnHys5m8ISK+ZyqGlbdy3hb9gTWhbIux5UMfJw7HnoOpVpwv+VEoy4V3JPGLHYmjZWC8jmiR0tXX+HftlNb7gDi1g7sxMJeq7AfKa1SonxdwE41Z9KQrv\\\\\\\/8aWOzX000AbIUHN01oHDh0OZvxP5ZtGaGfMC73yeSZeIZBYnCpFFK3qHcar88IFmoJIH1owXFadwqJWBnsq40o5kuiGgGtbvv2QH3ZXFY\\\\\\\/KPthlCbCjl7CfATD1tPBH2RQ5bTQ4Cb7JhBsv\\\\\\\/XT18yjDnL7gDx7OtMHLKm3A3W+hBzmRlMl8u+TJqyqxM7HHCrrp218tPLEBFQzS0IPcN+6cXPEynyp1evh3KEZB8aTxTHWKC6lYdHgZ5kK51Y9mn+evD1wKeB\\\\\\\/P4XKFar50DmCN4getr\\\\\\\/7cMwR71gstb1C7kTbqa30h+HcWCYUI8HYrZeNCm9eUg5gmmmMrt8fzr+jVGFNRy4Gtu+zBRAto7uJ8nozFXOp\\\\\\\/H3Jlt7\\\\\\\/v68N8nQsWQsHowqjuQk4jz5eJjNnXzr7f+8jo5kAffoPXJ6KDYwRgDmtvd5lxph0veHXWS9tB8U6iAvpq3FnDRC\\\\\\\/T\\\\\\\/aFdNFblIfnKEYXRxdf4u2\\\\\\\/6tizbWdZGuHYMMlj\\\\\\\/4jUjBj7lc3ABamJKB1nv+Zkf5hibC6jqpJ57SGO7MNpGXkcVkUga71i3eWun4xFKMLlWEZhjM43UorKk2tgp0mC6GHsXwEN+GtTtEzmM4QSm6bkBorWaNiFeOcAtOI475F5hglvATJeI+rwraw7UOkAGSpjq2DzxRXqsnmTQYxYyVP0WiTfDKldIeijUkNB+nn5Jwgdavg5AeNJR19tIeeJZm3HmCjICIe1uCQ0ElPSC19Ud9qYGgwprt9IrVeYK1qQYe4PXYRsmk+mmUS4+vDnrnXD\\\\\\\/BODdRTP9QfS7nT2o\\\\\\\/yU3\\\\\\\/3a8ruHEsawbsTr3BtRZy8ifVCQIXQJn+0bJZ5XIUjPIJlKN2Ul16Vx2ivzgbaV+2qujSzO+AXUzERdB5bbAHEswRwm+6LSwrVz8PODqlTf0t5g1B3XNBeKdNspPW4k9oJno98ajClHj7hm5C+KmtlnHqao\\\\\\\/BpdqAIK5MWD+fjwXSbdm5YXbQz+RRsTdbDQ\\\\\\\/CtZVB2cd2z6lutvZk7rS1QmqraMPsFjEB31vj7FbCv0LsK4Pasbuk\\\\\\\/IKcw4Vo1wDp28DR2lCtDR8sBC8dXZ\\\\\\\/C+DwrNhlAD6RFGzRcJSPe\\\\\\\/bb8tPqKtyylVtSRosrKj5j3HxTH6a25fzi0Si8JMF4onHWD7Ffxniq\\\\\\\/+ED3Ecu0YGC34bv0xGf\\\\\\\/vHM7YYLCBslDIpX4cRlpaxdKWqyyulHQl+0iQ21pxBemXSSPC7GBPzq1tmfHXLrYsIWxETKt\\\\\\\/J2egxCZ9nlEB0HhfeOh9MLfdfrNZN1sovg33oP\\\\\\\/6J1RES3g35OiFZc758XEj6jU\\\\\\\/n8+hB9hd7l\\\\\\\/SR7SkrhWidwUz1Ho0z2JJ2fbATozkcnNcKlkWZDiqZlFaFnY+mM74xOznnvZUjk01m2TBlmheqXg2tjpGD25AaxWCVLzPIXLmOXO116O8UftsXVubVIepVqsfgJpN08URduypG2Yg0S7UVYnfO9Pd8uD61Usrz8xyMa58KXi0pHC7Mq5WrwoysAXEYLlK7nfV32L0bUdeGnOTr+Oq8H1elU2Phgf0ZROOmjt62fTlh87GJqNcYU67L\\\\\\\/Qq5ccTzLb7\\\\\\\/fk5n\\\\\\\/W\\\\\\\/N6RkcjaWwCkb83geWDfNMEEwJKIdk\\\\\\\/kzZxX6aSL1evnC8C0S4FhLV+mSiegbVN+a96G1Fiva65PA0EOhf9T49AvimV\\\\\\\/fSIv5UEuXLGtaN6bXrG0l\\\\\\\/7jr1Ufzg1m232l5Vr0+2xhE2PL2KkI1F75ajQ1rg1jjzf0fGBOru9fZwpzpw7JJWQFoEQ7qjfshoUs\\\\\\\/HzP4z4e2S6InSSl7aRbDnc9PRQyOfjJyNFaVu8k5Z5JATLXUBNqqM5P2PexQiUWNO4Ya1\\\\\\\/6lgJr3obIJghfxMB0v9iQ12Ic9fdWo6KIxO2dc3N\\\\\\\/n4evoNELThA8IMG9pe5Tuog7RUe+ntUPZf0T0u3AUZWoaM6ZEn+Cblojl1XBD9X+zMqMDTMTlrjJsZL\\\\\\\/lk9QxXbR8KJpI97L3yqJkng+oJV+2aoOPct0E7auXy2hA5DKe6v6OnhfMmjUSz5CjEHSDmKjG7K\\\\\\\/AG0UMnCE474Hm4UOS82njDLQYLxWTRGHIIvi3Sv6GxAMRLXYVscSJxu8N9jnlctl\\\\\\\/prUfYZljKQa\\\\\\\/rAW7q5V7QBBf5X83\\\\\\\/IAvpVpvyhO4D52IOAiXZjueqOfuEDScECY+tEzPvWgiPSahDY6hSFelzJn\\\\\\\/48uWBCxAmO3baTIPJYJQDpxQ4KorkfbpwW5TyeIDezt3R+YAL5NzH5\\\\\\\/45lnOJ9T1iM4s9bHwS65\\\\\\\/lM45Ddp27Xb64AxmsfNKrJ2Gj6wCgsdSglSV4tuuyCYVY6YUuqLP+sT0iOoPDTwEW3THehAqjzTs9um4SDTiXPskyD3tj8nHiTzLer8MhWcWgnWwbtRTM\\\\\\\/c1HEFuBRjggap7Rz6pK0MoUZAhtj6Hl+GaBEXHYbVPWtQ8ccFtd5iBaav4AmlDj5X4bPQMaWY3Yj7Do6AMAdZ0NWcCuzjyHqUlzI9wvHARuI5y5FJvYj5yZT9rjgSTIUAD24+aBDWMQw7kEYN1pbmW0XnRev4yYzww9lMfPJ9eENSH+VHgDuVvG6f9o6OsYInnaKXJcBVUVv3IYKQjjE0ngaqfzxpZ+B7Xn60XNqB\\\\\\\/I3PowibDRYOT7sUd89Dq\\\\\\\/0ekXZJ6ZQPSNivicUuC5av1SZf3+mOuqPXjZoVpcdmP0LxGhhMjsQOT4bWXksvGaCXKQgFBeHqpjhh4WQ5tjCIXL3kZgtI01Z1HXqRNZ50YKhhCf7wkEBJ65UnDJ3LIrmxT8rVBaPyPpBSlPfb+\\\\\\\/bLTFYtia9AQmQ+7oHYmucJmFN9PMeIPSwHo6tEvYJsakl0jk9fCcOEnDUW3Af2ozgeNwK9bhxZc9TCHM4mTA7U\\\\\\\/rKy9FrFkHCr\\\\\\\/2qUQZ7WYEuYLfUt3kbAVLNDNGM8JQIDxKmo0tQF7fKpIMGyX49\\\\\\\/VjwCDgS0i4Z1rsEpKGOUARjV+rCuiOBbjbOg8bT45cS6+aEb4RvyDln\\\\\\\/Y1tEI5ZWSISTwlrQNkn0bE97toY3P3fAExrdtnhi87ug\\\\\\\/br\\\\\\\/8\\\\\\\/EtDKKd8Eezsw4XFuNi5PojXRjSC+t7qZEcqaz9gmS\\\\\\\/Hs\\\\\\\/\\\\\\\/QbsYszjLlzR+CrZiV77VL3fHPU5dfvhFB9edaXvi0CensCT4nt9Fj1J+PuKuZHO2zqfSitFBT4f7eyNt0J2fuDOng8T1kVvvbsjN1Zu\\\\\\\/Hfvum02tuWQ0uC3Fhdll8a4Y20kswjBOcS1ulSBhSt85BCbjaDtnAm4hDKQWi+AiZGjnF99DoSc5w9mT4rCgoxCWZ9ie972fsm5aZ2pJ9\\\\\\\/bhLmrYSJeV1rBs+QPFOViNGHSwve9YTRmx4l5U6\\\\\\\/YhM7DWaVVrOhvlJ4gG+KCI\\\\\\\/tMXdzIv8QoetwwnnDYE3CH8mueZWW9Brs2iwFS2xK3LJz5JDA3OeBLCR1uAzAAMotZKmVl6+FhjFAc2JJ2rkwma7UQ3C66eutpxsFFsc4PsUlQJfzrmL9X2L\\\\\\\/Y4lFOzn6L9zRN4ccfRpVnDphatCdQ75Y57quqa7VRF\\\\\\\/cI8dXFE95JNSxnQHT0rvcuQHP2c0ZZDUkcnp4003hQ6T0NP5l1a0fHv3dpSv9IkxH04PQE13q+a4JhgbwdyXu4dQ1IWgjfDQ2Fk0U+SVuV05VA1coyCf2EKh7wuqWrBqogx2VPCzXwWH4BzmB6jUPt40swEIcj3MT5F8J+fRBnGLF5JwTzXnQjm2kjxIFxo4Glxt7AKuTYalNCQ9Refxip0G4fO2dxJGqXeLtLoarKPLm80rDswWzq4X7F1Ua5OL6rZh4cOTKvXyV2YzIXEaH1G85VX39EOZ0lXln0ackNsm6z376kaDZ\\\\\\\/csD4MoNNOu\\\\\\\/tXU8DQj2w2ORlK6OLKOcAYDPVP+TqdVqhVEG7g+somrf2wjH2etoUq71THDbBk4a9TB\\\\\\\/nmqU5QHoLO1THv2JlTpLQpBssJ9RC+6yD3M21bYmoF+fzijpgebSjKSGJHOghXfiAM8nHWs30qNxDMvvVtP5IRVWRUfKPk7jDJARqaGHDhbVaPy2PZe+TxE\\\\\\\/r54i\\\\\\\/i1uJY+hsGRiDSshBRsov2fTHy7ZZXSucTaUipvIzi3yOJ9tb7uDDGWrJ39t\\\\\\\/JgH4mCaQ37h+j+OxWFPNeDnaF9KGMGgaf5BfYL0m\\\\\\\/wt8BqB6yuDOlqHI8J1xCf48TFe6kQeFb3UGhjAW3lKrIeWLOoOqrUhg7Gz4V6kXp0Q8\\\\\\\/HRmmANv5vplDTf2ppa1CWmEda6aqEInxDSDaG\\\\\\\/6gaZiIbmfo09ZpP7D68UfPiOp35A3PzilCLueDgns+wDEK+ZL5DmbR\\\\\\\/nb77630mhTgimHIFqz\\\\\\\/BX6SWxMjb7+qPXGr6HipjA93AkjBNYoyyXvdQ1ZhkOEfjOoa0P3A1XVSjI8SpNpmp\\\\\\\/3BlFmPm1HwmdAv1XBdwlVSG5w+ruwzgViVpl8IjrWnBOzGvp6Vm+mixIl1LfeVd09RssOIoontqWMOIp64k+ls2QUXFPXdNTKxaCbvwiuKq3ifMtL4F\\\\\\\/Q+40pem2G0ov6pNz4CWhSm7MGmFpSWNgWnSB+ETYBvs3GG0YhdKqhMHJe0ESuAc+P+8U3XdSr0kcmaJE9DRSoc26AysAMSIrwL4ixAzVz4FPh6c0eRvjjBvXXCd3FF0FEZK0+m3D+8vdzU\\\\\\\/KJaxZWNP2P9sTPe0iCxZkao4\\\\\\\/KtngUvxxajTRvAdTC3vS9E1BFRgwmdaG1aQ96bVbLPt5VyofsP26e43tfwa8o0AMN+uMukdBWZhH+RtB\\\\\\\/rIoubNENtF4R\\\\\\\/YPluwf\\\\\\\/79xRJzyEfNqeMA6wDRNmhZeuQqMsAOBGO\\\\\\\/PSVfi2VG7JOx5YcEOY2z6YT0K9aoe\\\\\\\/jZNpaIvOI3m5n9sfdJ3GEkRb\\\\\\\/LTbzwsamc+lYUYUPjZL7vttBK0aVuIuUo4\\\\\\\/THs0Qh3OqiWZfevtRbwZbuXOH0Jmwd4lJukZwoBnEXnDG93BTSvDtjWOOjI\\\\\\\/EcDtnJkqIOu2r8vWqBFqo4UEgFDV+ViE6Q6AMn0s1+VexJiWuxPok2w1rdd\\\\\\\/O2c+5HekSP35taZ7yCoiAq3O13hHBHmG2JQmL5VLDJ5tlle2PVOw05hh5O7Nmo0tnHjMgyog+cIAQBHzYXFZCtYP6NJ3IGZ4jMh\\\\\\\/1zpFTlqNf2TEfBpBnKgxHdqB77gukhZ1gkwrMa1r7mYh4QybJ4ZXmILxSWe722kkUk\\\\\\\/zOPRT12hX+k\\\\\\\/YJ4T7aTZFF2kesSbIYmUmQR4sjKqFntALO24sQpM8d+x7HS4EAcewrmxb7q3BQvEPcBnm4tUu0GdxB5x0+MdKf9LuEvitu0ttnPkELDoALRGA9Q9OFDpcXENLgDanYHj5YloOOV5mwemXjRKsSmaLMpUc2xXoKPoPQ6TmRhO7dKGUtNnBweNCTbGWP7k6t+mfQDEAIQqPPX2v6nIA6V1b6ia2E3ZmxvxWijzAk9Qhf3RYw1A2dQIWE6PA99sxZtj\\\\\\\/FrkAkVqkmNRHJe74QcBxmmiuJX35BtTovBe3PBEYvJkWkYvnMmtBCOB6ki7EuRcqr6RPaiMk\\\\\\\/GUvBHhvyxVv0COKN16lEbG7Z9MUSV9LmuQvMa+51WXrzBi6ZvvmbfHRndbDJHo9857nSs50ug11FUwHUnNrh2LQ4rOQDCtyYBK0Z9ZSe5anc3AxEMYGQ\\\\\\\/gGa6fOdsAGs833tdKz4HEJ+8KAVMtX7ZqdPySVjNKmhPFouX\\\\\\\/lqYrcEE89I0Hfzn27fYdltfXaCWtcAylfdBDkm9gx2hUcExmkwx0b\\\\\\\/ulV0JxTqKTagKf\\\\\\\/dgtVIyAPsEeT\\\\\\\/g78xrgSvpQ7KsN\\\\\\\/Du5olJT8vWXF0oALw48IFKyL2MHD3JF85NekNi67p6ZMaOPeExxeQNf9uCR2UUEoS6tf7kt0Kfx7znKtLrUFLHXkE2ZAkYFmM1KTa6PcHb5nhewTrtb+LgiASEvqoiaYdymLtesH59YNMY2y5b2ZEsRSvDG6zIuTC5NGL\\\\\\\/89MqDtkb5l9YHBxdK0AK22pAT\\\\\\\/0k6N1c99ugvrTxihvrUoy9woobCz\\\\\\\/w5pBu88ERw+FHOXZj2wJpUSUTMJDbN0l6t+a82beegLd1ZNABxEi2q292U5sdb6uc\\\\\\\/wgyMEUTEhJfy2AhYWaE4sF5WyDP+pR5U8BCf9daRKyhz0kUUaLZWp5iBSLGXpMTXNEPcXbmXfbcq6Oq5KRZSc4EPBwbEntGK6G3T4rsv+tfbbDDdSXntVBEeT1023e+1BP73gets19TjKgd4pi4zQ1tM7r\\\\\\\/SqnBYna2dqX0zuMjdpArSh4ltNgCFpHB\\\\\\\/r3HnRxLEyirs9mL0XjWO9+V4oWVZI+0Q06eQFwiR67gnkGRfEq78lKCwVKNIoq15M7oxZSIy\\\\\\\/ecS1YuX4avtiix9yZf6enl9UqGFhuAPpdCbzumygXbqOujt\\\\\\\/ZnEoX5hWOM2Wt+KTwILdL0lBCBUvUWop+7\\\\\\\/5\\\\\\\/J825Z4\\\\\\\/Vt1+iAVwgw0ePskxXugAnUOFP53ajcumE5MSMuqAWpOREPQosfRO8JGeB+HRuBH1wG5rFVxhG1IGtwLv+bdJKqNVAcVeIT9UwuADkUlc8zGnBVEZ6gQ0LTV0Xur5aP5DdkFVnLNRjvjGofQHlQZucbV15JBi13G7JKJDJ5ijRkGTjc9qaY65Z77trXslcvYHArVGUwklKbxkmkcvu+xbJPQ3FriQvEyaSW0CRDLr1C59L5CurlEqhG2cfTJYmesikOQ0cKLixmb+iW4eDNIoJqG8rggm5mgNDJc7Drs\\\\\\\/z7Tu+IyQ0X6WkAoIVyWXQgnei7R50ucFRSEaEuQsCt54G1E4Z4cTMxiHCD3yyiNDvQa4bmE0QOeinUKyLeXQW\\\\\\\/aRYKYSibFjm2muNlc+7NbE\\\\\\\/wJJ7OC4SZk7DEdGIjWKavCgfgZHhfbMD6hkvua8sfUjs\\\\\\\/gBcaY+z0rJcJzIa3NFxTkNoIACyZS\\\\\\\/4dOragBBrcBeiU9hV5\\\\\\\/naHDQattK47wUNK8VZ1QyZx3D7Zrd3zY5mTiXAuLd9nBJVOp+ynYkZ3AyKK3B2sfK\\\\\\\/73uALMe\\\\\\\/ZcrAfNhRfZH1A0HVXEvjwoGHszPdcgZO4yM5WlJWr673D4N6ZU4Cs13MXS5PYxxNwCNJZY5T2XXLoBgOBrKIQINjxHF8+C1ltSxWK8qpuHekDnUHbvGYnGFE1LmN956tKsx9w7MP5BzpbYGPic3ONpjJxxXmoimYsZK8D+2DAUWPewaR1VPVJBTaij+CBCA0igqc0YSQ8KWlVMhBvdjgPoSghC3tLiVZFVtCtBKYEuuhWNTRjYHWgVmX\\\\\\\/vaOKpVuLLNL3QhIphFQqtJmM7c57IRGJI1ZMOrCGBnQgYbhzT3CbS8SJvRuk+2+1nCVVYLZeQR7yrdTFerW8UX1DJg10JzdNuw1fBrkzHV8kEyB9BDGEfuu2NUTOiHDSR4g8iLnP6GcJmZ0JPlZcHvgk99V6PAhS7h0wa+n9DXx5ED0SpE3W71JMrCuIgFy0YMQ76OgmoH\\\\\\\/VpRQpkI7v3mUzbg8RokM7Spf0gO77fM51jlWOYAathYxR6qfIX16uwSKyqedlR7nTGLUkppf8dM2JeFJZnHgXXPA2FAbnO8DFEG5XdoVk5OaSU2BeNIh7nfPSKlCXTWoHTbNN6L8183ne5kxTXJvCNGdMERdCx56CV5nsEHvztiAKh9UtMdwUI74v3HXbFStrQRnB\\\\\\\/IYv7BPv\\\\\\\/vFxK5ZAihqQ1NCvqNQvvyKNvvHXRGVguU97OOMicKp9t29NQRwzk8wT3PbA3gcIrZ6Ry45ENAb3x2iyDuQD1AC\\\\\\\/0eEHURlOCAGNvff1+zFayMOniwBpSBRh5kuQruSwGNP2Oy5L9un9LvB86myaST1\\\\\\\/LZ0y5AqMEVeRo598fZoleDBgLZ9nK9EZENr+81Y9GBjBBggPveH80Dcu0iR0K26lesJR8yB5PlKes7VP1nL4neFBlkFnGhwYooFhxLCOZgqk0W6Zdz47IQVhDwFaFkCJDlQmbzw\\\\\\\/PJ8galFNQoxJfnO\\\\\\\/bQUmaZCSsvdjU2IYmyDcBDF4FauhLLtg35HkBtW3Kd7Gi6ls+UpaLayBGR2kBRloZ5mETrUQvGY5TIhxqsKkkYq+OaWsBUVtp1dYb4mTP8CS6KBJIm8J0C19rPseLOZDHFJqB2Hrph8uuDXNXbh9p15sN6UlLSBHvvZqv9JN58rrpOOSm4GRUf0aufiu4KOILoNQzSl7GL0132Qo7oDlwyTuLGvjp1uK\\\\\\\/GAAV+WtsgTP\\\\\\\/9M3RWrg9Du+SQaTS\\\\\\\/KQ\\\\\\\/DsaxMNc7xlQm5XpkNADIUWE9hctEk9Tto9inFXuL+Z\\\\\\\/6p6e7Gx0BJGlUWl391xhEvDmzvEBQhualNjIfC88m\\\\\\\/1H2luWBWWq6XxHkxuZZStfOuVKThEMiErdpp9409J3YXQVbXhs\\\\\\\/ObuoHeym4doJ4zMEoJwuDX+v8GplE1N7XOIM5rBMjx+m6crtZsx+p\\\\\\\/LGOUuuvsstCwJ2u5hRHPQc2aOm\\\\\\\/\\\\\\\/Q2cILGnaEUQxcaP9DArSVg4Uy1yGiRD1Vmwf\\\\\\\/2j5ccSqYE6o2lKHrHn2NB3kFkBj9EJDCS1zZqx9rQbrrMuoUOb1Sba+FPVdbj6VbRgAHVF\\\\\\\/f2qwSmnmS3cDhfJuFwMLtCuJ4z74EYM6YFsJkLxCOPgQ3y3Yri6Fv7twMhOWCbLimXncCMh5GiNyZXBIscLMbbQVujTcmSKwNPBYiczEEgSeQu1tNxIWCwDgPN0XoK74aR\\\\\\\/8iqO764KNLTpcpsWAvNhazZK\\\\\\\/rstB7W6lM2B0i5pFCUmFvATuus08ehXtNhxwFgDMBWt9elDuUIGTa5krtg1RCSGDrki\\\\\\\/2E4\\\\\\\/TbUF1xCb3FNfZJzk55cLyCktKTaC3RfXpiABYEYUPNkNmRVzTPK\\\\\\\/igF6sgj9Sr2faHmvl2kKZY+5w1XDLop5Cygc8d0vUPFehRby\\\\\\\/Cb7XZz7DixBUDpZNs4BC1pMo7s\\\\\\\/XlnWuCYkopjJL5n4fTTW6PPJ8fNQJho7sZIw8o59Kz+Bs23jQUbNirUxPYO5dhEe6fgYHn6guVVY5qs5fdJX9bcwfYQSvOW7+4TS+LMSf6P1MnFtJ3tIoKm2L8uy\\\\\\\/7NbLg++zFvUSLcpu0DdGkOi2dEzV2KaZUta8gkqsbHar1ztt8E99SsL9jPkxiDco4Om58F\\\\\\\/wJDqbHSlicvmU9hj6jRfK7bJgtGO2CJeSxNzO7Hh3iVHdxfmX6WSCFpb80rQ2+w8aiSfXkRQqGruxh\\\\\\\/hH9RLwRrtIL6PAqm\\\\\\\/MEBm5vjOg+\\\\\\\/pF2MlNz9L93QBnBtW0K0zcS3ZSgiazG9pb4K7FKA\\\\\\\/Xtp+jtQceQTYYRRobwg9RPb8Pjwo2cDJF5O2Qco4Oc0btPSI5S5efwaNlJCFACz+jujkaOhD3M757R2XZE\\\\\\\/2HzsQTrvs\\\\\\\/vdU6KXDRhnWw2xrZzUY9mdnTSrEvNN9bUac\\\\\\\/2SBKqdJJKGv4r9ETXBBk4gBgbGPSD\\\\\\\/0CrnBQpvq18026PxUIbus6vlMsmPVteLn0wOzb9nV3dqIig52TEVXKQqwkNbVXqt1OON5FGtKEsOB2N8q9AEvAN0co4ATSf4K+LL04DcLtFlYr16Nlc5rrrkouQWTJV1+9bgaytiC2XpYYVyHbHBp6YAmUuYOH8jMVuhmmjwKfOdfE5IIpcS0sa2gl3ovUY80EXMEbDp6jH4D\\\\\\\/edCD+Ut4bVtZMXIDkejDhxx\\\\\\\/5PJn+JlX3rVvJnW3SiERnyoXUQSM0mSTwvUcqO0qhiira9onf9rrtXUg5mk3YIWBwgS1I2lQePxCeaE5z\\\\\\\/8OJ0zyhckRzjcLxfBCFV+69moh60NIO08d+AjW+b1xScEOsrcDMgTjJ9QRv+n6usj5KFIcKsTJaqDSbRU\\\\\\\/u2rOuARmxbK8TqUNzmtua4C0IAZFU0udfuYJJAzu1keHurTZ85VXXXxDwl1\\\\\\\/StH6sJg2i5CRFGgxmqrZNU+joJucoaTAEpHNCOHAZXNzSW7yEk2bD3uWWgHt0Wf0KYP4fB\\\\\\\/rm2+gHOuBVQVii\\\\\\\/RhShDyZl2bcXXKXLQbGH9rcYeYmGJA6Kddj8klUfdmbwsSWru15aptWOHwOJnwEYg5quolUhT9y7hUXEn6KSNPvY8TjT2+CpzfPjFzoFAAcrkgmSUAbwKM\\\\\\\/znw0rVYdKNP7hHVG1DM\\\\\\\/Fp8pjibGVcKbaqPv6982y+77\\\\\\\/tuZpp1tWM\\\\\\\/EnxcqUildpYqtViD3SIbdhuuO97\\\\\\\/9Zc4GjcDtTTzzC6mbN0f2p+zU8PU7JQQHAn\\\\\\\/efwt1ZTaz9GzHYHktEbR1oNQEFkgZapVvGzc1UWIQlYWyWbYnpsujTgIWOlYk0Qs4iGxkib6rsJ3wcInZt2k0g9NaNgGawce4jHkkBP7EKl1CqR3B\\\\\\\/8DZ28dTh6w1vnOaLbZdfNVp4wbSiCUpH678CCV9f+no3zXr0zyC2weczVqkkC+akc5zBf8jBJRB9Pnf5zBQSAVpzijZkwDCMiQp2gJpQmkd4JHJc0IdFcUXxY\\\\\\\/3BAIOcZdCyMLgWNQaUzuoNrzWZiLaKDad+1yFRiqIfZugrpGVL+\\\\\\\/3vxkltIwGg+mwDypHPv7OEo0CDF5DK7dQN9buUnMD6t0IqadWKmj7DurOngNZNVu2x29pIpJfm5uJpDAxt4dy5BuwXpdS7T3HO7R\\\\\\\/1LiiRIUn10ecDH5qGOL+B7urG+qNEDAAI73\\\\\\\/TzLL+vZwcivxyVgsapqfYrQmCxaNaOY7juxsKIiRSaHWDS7mWKjxv7oVypqEUw07LB97fVIZWnd9hFDE\\\\\\\/tKGy2b+H0RiAdLlD7jEiOm0Z0D9kB5gHr3gJQ62yc\\\\\\\/BwjeLzqYHtosaV9G8ZPwpZ8IeZ34cWkJz4G5VnRwuEFsrXkEDTKP+6ZlaBsXAxq5WmZw9YGyZ9U4dT6zrQ\\\\\\\/1waBU7xCSOvK+9qqxa16xC7O7f5iIL+CSYqxkgQCMuOy219HCf7QXl\\\\\\\/IFVzkOWIS+te6aYZ5dioHLVZ5HUSskWOGgu0fGUvjUgsApoaZZ7RPOXIUQZ78ujlV64nrn32hyfIwq8cpBdthrQSkXlO1Hk3KF+SrB0m+p8+NznptM6316Lyk7ewGk9PVAmtGlByFj+N9erGITlULG8R0MF4UGLmUf4JcTXzSKOD9XxzUsXoJposs2vbz0B04+2LrE0\\\\\\\/KFuxrgond13s8gy\\\\\\\/TVT+32k+VzDxUWF5VsW\\\\\\\/dAWwc3OQpPWFVjqSdWVOSvleQ1\\\\\\\/ocL9wEcsWy6Qx\\\\\\\/NG35yzp4G5Jpihyi3hoX7g3M3OG8DGZIc7Kd+A4uEDUMmDCN+oESqAUodXhwGWC8EkbgQfJzwggYWZJ6YO5\\\\\\\/9LJ9o1tYl4R5ISbwZ7SotVp1QXqLZ\\\\\\\/O5nUiZPjWUG3+mbLnnogvJYh2fdrbVTP0bmkR0QIpNsagX4G0oSlH0AFrtfbHN43xtrF0VH1MKYEEDyXG5zwDs+tX7Fvk1ks2J+n\\\\\\\/Sz6KUf3lbrKEsbHRy8IpRgDzY+bKXbTeRD5fjH\\\\\\\/\\\\\\\/xyj8hwcTUJt4x8AxRCsIbtsiqIDMEa5aBkewvMSTRni2R9HzVDXhAPXlREzQ\\\\\\\/QhgtdutBUwygqUC8wQXuDSSUxxKsAoRemyuu6qY0coeQNowXG6EM8F4Q6OYoJLZ\\\\\\\/qnoI6p6FF5346fLkm86P4\\\\\\\/eCaNMH7RGrYrE9VUIDJNi7T8Omr8VzV8H19LsbBVheV3QuBjfopvbqFJwGYZc2XphNh\\\\\\\/sY0ZIlpQIaECwrBOYKcKzC+n+tepzxiNA1B5momVWn9Ip2JV4YT4ygE+WSr6x+2QKGkNnulytmZXg9ZpirlrXBPrzo4ENaYFKFbDgKAx9\\\\\\\/xZvpsuz9QHi1MonFB4mW6OPGPgOnD\\\\\\\/DiL0oNYMW3k74RYKnjNWD7Jb6F0U9UypscD7Uop7NceIRFfg2Wd4Efu+Sb9mjECbDHjVE1C0ewglPGy8z+iI5lnkgAj+JlequnMPWUT6p5skPtDND3P76cvx4LJsn5QKXoOVCnTnDmXtiIxWluiBOj8d5YTIM37bixl7OTy4gGX4n7y6cRYPym7a3gcMJC3fPzLa4wnl8Dqa9YOa5kZc8suvt8TI8QVMZxwM9510\\\\\\\/FNKZTvY6EW+caokuEMMOoaA\\\\\\\/Z1w\\\\\\\/975QQzoBxwDhUmchb7gAaPn9K2Ixl4aEifg8phpa75hppe3Ek+\\\\\\\/MfyNwy4I11JpN19y6KS1H27h0c6lnz4a4t5djaFqe+ujESD+THd1r+GuZjo8bgt7KuecmICbJk95QhlzOHFrEWTBvvzOI\\\\\\\/GpT4cPkhWVY8AbrGxtorRxtNtpyrOATLsLsMbMFQ5wIGRAOxC+fQgOp9EoeuYrrrtBnoPvQ21Ym4XiBi5s++C1CtRZFJ3tZdKEi9g3vxuvGEK2kPo95DXuJ7TNJNm0Z\\\\\\\/crUWWp69l+0pTizf9nYemEJNJzLaV1lwhRjFRGwok8uobA5+L4l14vTVnD5ncm5rzRxKV5\\\\\\\/fCuuKo9p2fI+UYQ7Emgi0riF209yEtWmWv3IpCNylxx0AxFa1fvR8pi6lR2mTILTp5LYyujRRSFX3MDtG6QBaKJv10MJEvFbxMfNKkJCLpw9nsUlwhvgs+pZMaGbeBhdL8WNuCak3DVdUwWWBuipZP8BjsmP6sT05C2rP4WRYUisw6xzeZQUKocas29aom5VrFtUpvNCK8CobA9RMKW5lDQi6Tvbg+Lc18SqDQCJoeZAVg\\\\\\\/d3ofPPNirAdDbyj0PlH6EWz7kPbPKE21EexZ32eAxfNV\\\\\\\/9OkVXhINGpJv611kxsr9wefuLSj+1Kq216sdBGr8by8tYLeYBfjGgwQTp1eA2g2Ww0fo43DLhEtfuCOWjHBuwYx6WBUAFu9ytFD4L+zDllB3XQYcUXilgttYNcft93ZXG\\\\\\\/n6rc3FB91byHot70oHbt29WQp82xH6oVFKx7pcikuqWY1TleHA9E98P3ebXzECkwMfmhqhi\\\\\\\/CrqAyGxeW\\\\\\\/7w0Ttwwr2cv78B7HH7l9FkO2HYyol5XgJzgVrotbf\\\\\\\/5uC4zx\\\\\\\/tYeSh51PL0WoSeue84RUn3hJvKh9PaR8+esG+bbJSW0NOp3or0CN+siUYC+kAMPTwBn62KcJh8LDs48tm2ipQ+MUwXiDPuCAltXLxWx3+5ulAsHZn7sKJQKUf5hFhV8palNbdZrbS4+3mOsexuB34NLysyyXcjALCsegzRrYLf88Mtm6lniKn\\\\\\\/fWOHwLIlFBwhP7WJtO07iDG1Ytn+81kPRekjsTDnGpJDK2pR7uR4LlcuncvqDXPJcQ+8JgPGb8iZsAcwM\\\\\\\/FEfK0AA7PxEoWVa0\\\\\\\/5m\\\\\\\/mvulkzmf0MIJVcoTdFWzINde\\\\\\\/NaW+Ser\\\\\\\/3GanG19TwT7+MsRt++OAVEEl4EZUVFE2w3E0dvth8AmkcEPEpqUHzWRoB1zQtkH37X8SWMUd3+1lLq2et9fLpma4SgaHSRgrq3o9uq\\\\\\\/xf2vfK8S1\\\\\\\/QcbfN3e\\\\\\\/6V7g\\\\\\\/P6d7ZWpuxMYjePLLTKhL3N19dWBxR70+wgUMVrIJ4VHokxO6wMErGVQjKSw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022oOXBagTZFaa9qpy\\\\\\\/6\\\\\\\/lnzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227016bb2a0463e2f0b02eb7668a80709c766934b3eca818688d1f13b781c7d033\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IlzCzJelgF62SkVH3ehpTDBqKGi+fv3WNrq9Btjq\\\\\\\/pzHReiw9N6eEnc8jhcoT6Rj17s80RLgSmF7Sp6LCA1K2iCuag8FeJtmHL0z3uPB+tGp+x4Ep8wf+u0LHV9LmMSe7bGsKdczk95Bk32g0BPZ+SaCwHeRQPFp8EvvXfE3g2H5uta1anF8wA4Cq8Fmx+tNF4rEgx7yZbYc1QD\\\\\\\/GM3yQALJ+gFix\\\\\\\/LYBOt6f8+iez22s\\\\\\\/1vKX0BguLl2CrNK58a2gEVQPmQ+p9F4cJ\\\\\\\/sJM1CScgIwOqI\\\\\\\/7x17jVBOH62AFHA5XRyUPpaR+eaMhI\\\\\\\/o\\\\\\\/n4CxU3ahTGuKA\\\\\\\/fB+aTwqRFilySWtJ68RH5EBhlC4n11YPvoSbPqDCfaAhxroTEfbCijprylPzQXxV2dW5yxlgL5\\\\\\\/7uFT6+GhRMnsklSGHl\\\\\\\/8LcCFhqRLlPXQkrhYKJ5mL2c2RVva4Dwz9UcykZe36ig7rVfRN7LP\\\\\\\/zwCdZW02AzmnjXaN79gVKNOdFiV1kdGVR+6EtJdKL8V5u05MVMHoA188eZJnDJVVJQBNp2rp7vDi\\\\\\\/VWjKivT7bRbbPF94y\\\\\\\/90q2w17bZifEixpxXt4qngGp57XWXumwICCZGSto7YVeMXj8xpLA4BcLfdcx5brxJRVCn9L53ONRD5\\\\\\\/niney9AH\\\\\\\/AofipEh8zAcfTGWluMwwPNODL2oNxZt0nGEsM3vhKmPGn3BNwiYPmvksKx43Vu06P\\\\\\\/1Plh6BxpWFJvIxqdoab+124EfJQjhsOPoVAqlsKFA8slj7jfnwHwBYPTqZAc8IJlufgBqvcc6yEq1\\\\\\\/Yh5QRewMfCTvK4la00sh+c5ekxGmNs3nDaPl7585YTqDVL0QbtLekjcFDuGx8w81UTgYtsa2weIptrYwRO5qrUYPebkUvaUL4LXgW9oJEp0i2\\\\\\\/oqyqKzjml01u1gSotPI2XKdd0s5YsZRIf+oGl4ZX1AXpDbXZ8TE93PXPRt6Er2H8N\\\\\\\/fNrYiNLeWFIpbMSklaI3E6hXW0ly6nr9FyElUnaOnjQs+zC+UHGFp2WbGqjXiBIyZ1FScF3SPciBvrSFe5LBWZrdVOu1HGVnE1qWw6sE45fkvhkhcDPH1znqiKhfj4l6\\\\\\\/BAd+nQeOrSXZMAYPVtiJX1++rDH02qvN6mUqhS2g9IKehXqVm7M+mmzxyJTKArI1gFjxRMr+EzEpjm8184MtSLlnU79xZcIS90RxQ8BpMq\\\\\\\/TSpvNWIvANTNf9enr39+g6RwGXzBjjZGH\\\\\\\/Lkcf8REuuGVP83gulXLuB9NeVa195EXGwaXCi\\\\\\\/8U86ZiqHQqlHxMdHqifhYc\\\\\\\/LdZ\\\\\\\/+iUmd\\\\\\\/JL63hkRqwpE7wi65i9+3ofdnBmrZKsHijcMLKnJTrzuT63dDhG\\\\\\\/Xgm1vPW8ORNIMMnaceP3V3bIW2eemr8bRPpypi9jt4+LrJlD\\\\\\\/DvaWRAsylL7MSy1jjSRFXYxcfZHW6W1B\\\\\\\/PDUePPo0L\\\\\\\/aqRScq34PF0u608MfLLt65Kjcwi46IigbtYwzj3N72TYFXTmpvf5RnDtAl+jJuSZwEbl8ag+oBGRAd03D939AiV73IgIE4CKrBpRvi1dVBtuPCoB1KOFIowJzVplY8XMrIFzuw\\\\\\\/IIFzhqrKl15ZsD0i2oPlZqzd4CtA+xdWo31WwxR2U1gS9j6mmMXFdeNEImrlCl8uv5bbOB70dYAExrJvnX2YV0K25MV9Qk7RL2eNanzteOkaIq3vgZW8pIbsuclJNHl4AYbcMBk5Bm03nLakC0dRJyAo1gkh8qsCTbD2IgY+Nl4E9DjvNQd0RwBMBnSvjAjqyoI9Iln1\\\\\\\/yPX\\\\\\\/mB43XIF5EcfxYKI\\\\\\\/zhSZE3a8CB8ckXn1BeYvSGPdan\\\\\\\/DBA7i76GTx4WolOABJ5H\\\\\\\/lRHT7EdcrOkhGlS+M0wbvfI8eG11k4ilKizZCrUZXuHZV\\\\\\\/cDesoo3WdMJPWDQNIFKrY3O6ZTLl2DQ\\\\\\\/SrwsH0C0+jY3xV+BPi8wbLrsv3N0mMp1o4NFL0NANV92kNWURFBgMCjg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CVEBMgjrTOrtjVynwKJyGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002228ed0679f5428c952a5eb943979fdc08f57db7f964c39fc571570b9605a5310b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460475 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3497" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-N3iw1Lo8wQs0dxCl5pHFHRi8D8vBkzRXlmUyJ_EzGK0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3497" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--8VafUUV_5LpKrEDHqGunz2_U4_wXzVou31RcTxcH3k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3485" title="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3507" title="Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460475" id="node460475" data-pisa="ggg;6/11/3497" data-pisa-master="ggg;g3.116.033332" data-apath="/ggg/6/11/3497.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (<em>Manihot esculenta</em>): Implications for the Prediction of Total Genetic Value and the Selection of Varieties</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5929-5785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marnin D. Wolfe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ismail Y. Rabbi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3497-3506;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033332 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460475" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marnin D. Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marnin%2BD.%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarnin%2BD.%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5929-5785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marnin D. Wolfe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wolfemd@gmail.com" class="" data-icon-position="" data-hide-link-title="0">wolfemd@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ismail Y. Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BY.%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20IY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsmail%2BY.%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ismail Y. Rabbi</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture - Agricultural Research Service (USDA-ARS), R.W. Holley Center for Agriculture and Health, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3497" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460475" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460475/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3497.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460475" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460475/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3497.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460475" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460475/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3497.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460475" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460475/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3497.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460475" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460475/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In clonally propagated crops, nonadditive genetic effects can be effectively exploited by the identification of superior genetic individuals as varieties. Cassava (<em>Manihot esculenta</em> Crantz) is a clonally propagated staple food crop that feeds hundreds of millions. We quantified the amount and nature of nonadditive genetic variation for three key traits in a breeding population of cassava from sub-Saharan Africa using additive and nonadditive genome-wide marker-based relationship matrices. We then assessed the accuracy of genomic prediction for total (additive plus nonadditive) genetic value. We confirmed previous findings based on diallel crosses that nonadditive genetic variation is significant for key cassava traits. Specifically, we found that dominance is particularly important for root yield and epistasis contributes strongly to variation in cassava mosaic disease (CMD) resistance. Further, we showed that total genetic value predicted observed phenotypes more accurately than additive only models for root yield but not for dry matter content, which is mostly additive or for CMD resistance, which has high narrow-sense heritability. We address the implication of these results for cassava breeding and put our work in the context of previous results in cassava, and other plant and animal species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li><li class="kwd"><a href="/keyword/nonadditive-effects" class="hw-term hw-article-keyword hw-article-keyword-nonadditive-effects" rel="nofollow">Nonadditive effects</a></li><li class="kwd"><a href="/keyword/cassava" class="hw-term hw-article-keyword hw-article-keyword-cassava" rel="nofollow">Cassava</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Understanding genetic architecture requires the decomposition of genetic variance into additive, dominance, and epistatic components (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fisher 1918</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cockerham 1954</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kempthorne 1954</a>). However, partitioning genetic variance components is notoriously difficult, requiring specialized breeding designs (<em>e.g.</em>, diallel crosses) and pedigree information (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lynch and Walsh 1998</a>), often limiting the genetic diversity that can be sampled in any one given study. Genome-wide molecular marker data now enable the accurate measurement of relatedness in the form of genomic realized relationship matrices (GRMs; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">VanRaden 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lorenz <em>et al.</em> 2011</a>). GRMs, in contrast to pedigrees, directly measure Mendelian sampling (variation in relatedness within relatedness classes such as full-siblings; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Heffner <em>et al.</em> 2009</a>). Further, GRMs can measure relationships even in diverse, nominally unrelated samples, expanding the potential for studying inheritance in natural and breeding populations (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lorenz <em>et al.</em> 2011</a>).</p><p id="p-4">Estimation of narrow-sense heritability and prediction of breeding values in genomic selection programs is becoming increasingly common using additive formulations of GRMs (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Visscher <em>et al.</em> 2008</a>). Several recent studies have described dominance and epistatic GRMs for the partitioning of nonadditive genetic variance using genome-wide SNP markers (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2012</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Vitezica <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Muñoz <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wang <em>et al.</em> 2014</a>). Models using these new formulations have been shown to provide improved partitioning of genetic variances relative to pedigree-based approaches (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2012</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Muñoz <em>et al.</em> 2014</a>). These new models can be used not only to estimate genetic variances but also for genomic prediction of total genetic value in genomic selection breeding programs (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2012</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Vitezica <em>et al.</em> 2013</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Muñoz <em>et al.</em> 2014</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Wang <em>et al.</em> 2014</a>).</p><p id="p-5">Cassava is a vegetatively propagated, staple food crop that is high in starch and feeds half a billion people worldwide (<a href="http://faostat.fao.org">http://faostat.fao.org</a>). Efforts to improve cassava genetically with cutting edge methodologies including transgenic and genomic selection (GS) approaches are underway thanks to new genomic resources (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Prochnik <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">ICGMC 2015</a>). Prediction with additive models has recently been evaluated (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Oliveira <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ly <em>et al.</em> 2013</a>) and genomic selection using standard models is currently being tested (<a href="http://www.nextgencassava.org">http://www.nextgencassava.org</a>). Vegetatively propagated crop (<em>e.g.</em>, cassava) breeding can exploit nonadditive genetic effects by identifying superior clones as varieties (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ceballos <em>et al.</em> 2015</a>).</p><p id="p-6">Diallelic studies in cassava indicate that nonadditive genetic effects (<em>e.g.</em>, specific combining ability) are strong, particularly for root yield traits (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cach <em>et al.</em> 2005</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2006</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Calle <em>et al.</em> 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jaramillo <em>et al.</em> 2005</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pérez <em>et al.</em> 2005a</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">b</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zacarias and Labuschagne 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kulembeka <em>et al.</em> 2012</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tumuhimbise <em>et al.</em> 2014</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Ceballos <em>et al.</em> 2015</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chalwe <em>et al.</em> 2015</a>). If the limited number of parents tested thus far represents the broader cassava breeding germplasm, genetic gains, especially for already low-heritability root yield traits, will be slow regardless of the breeding scheme employed (<em>e.g.</em>, phenotypic <em>vs.</em> pedigree <em>vs.</em> genomic selection). Breeding gains have indeed been slow in cassava (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ceballos <em>et al.</em> 2012</a>) and low accuracies have been reported for genomic prediction of yield compared with cassava mosaic disease (CMD) resistance and dry matter (DM) content (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Oliveira <em>et al.</em> 2012</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Ly <em>et al.</em> 2013</a>). However, cassava varieties are evaluated and disseminated to farmers by clonal propagation, meaning that accurate prediction of total (additive plus nonadditive) genetic value could contribute to variety selection.</p><p id="p-7">In this study, we test whether certain cassava traits, especially root yield, have relatively large nonadditive genetic variances that account for low genomic prediction accuracies previously observed. We estimate additive and nonadditive variance components using GRMs in two datasets of cassava from the International Institute of Tropical Agriculture’s (IITA) genomic selection breeding program. Further, we assess the accuracy of predicting total genetic value using the additive and nonadditive models. We discuss the origin of nonadditive genetic variance in cassava, its potential effect on cassava breeding, and its role in genomic selection strategies for cassava improvement in the future.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Germplasm and phenotyping trials</h3><p id="p-8">We examined additive and nonadditive effects in two datasets of cassava that have been genotyped and phenotyped as part of the Next Generation Cassava Breeding Program at IITA, Nigeria (<a href="http://www.nextgencassava.org">http://www.nextgencassava.org</a>). The IITA’s Genetic Gain (GG) collection contains 694 historically important clones, most of which are advanced breeding lines although some are classified as superior landraces. These lines have been selected and maintained clonally since 1970 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Okechukwu and Dixon 2008</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Ly <em>et al.</em> 2013</a>). Most of these materials are derived from the cassava gene pool from West Africa as well as parents derived from the breeding program at Amani Station in Tanzania and hybrids of germplasm introduced from Latin America. Available information on the GG accessions included in our analyses is provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS1.xlsx">Table S1</a>.</p><p id="p-9">IITA’s GG trials were conducted in seven locations over 14 yr (2000–2014) in Nigeria for a total of 24,373 observations. Each GG trial comprises a randomized, incomplete block design replicated one or two times per location and year. Since materials have been occasionally lost and new, selected materials are continuously added to the GG, the number of clones trialed in a given year changes gradually across years, generally increasing. The sample sizes, number of replicates, and number of clones from the GG in each of the trials (location-year combinations) are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS2.xlsx">Table S2</a>.</p><p id="p-10">Theory suggests that founding events and truncation selection can both lead to a conversion of nonadditive genetic variation into additive variance. This can happen because of the induction of linkage disequilibrium and reduction in allele frequency (or fixation of alleles) at some loci relative to others (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Goodnight 1988</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Turelli and Barton 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hallander and Waldmann 2007</a>). Consequently, our results might depend on the dataset examined. We therefore analyzed an additional dataset: a collection of 2187 clones that are the direct descendants of truncation selection on the GG. Briefly, in 2012 the GG and all available historical phenotype data were used as a reference dataset to obtain genomic estimated breeding values (GEBVs) using the genomic BLUP (GBLUP) model (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">VanRaden 2008</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Heffner <em>et al.</em> 2009</a>). Selection was based on an index that included mean cassava mosaic disease severity (MCMDS), mean cassava bacterial blight disease severity (MCBBS), DM, harvest index, and fresh root weight (RTWT). This index of GEBVs was used to select 83 members of the GG to cross and generate a collection of 135 full-sib families, which we refer to as the GS Cycle 1 (C1). In the C1, family sizes are 18.3 on average (median 14, range from 3 to 82). Parents have an average of 59.5 progeny (median 38, range from 5 to 406). The pedigree of the C1 is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS3.xlsx">Table S3</a>. Further, information about the germplasm analyzed, including data regarding the genetic structure of the population, have been published previously (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wolfe <em>et al.</em> 2016</a>); however, we also provide plots of the first four principal components of the additive genetic relationship matrix (see below) in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/FigureS1.pdf">Figure S1</a>.</p><p id="p-11">Cycle 1 progenies were evaluated in a single clonal evaluation trial during the 2013–2014 field season across three locations (Ibadan, Ikenne, and Mokwa). For the C1 clonal trial, planting material was only available for one plot of five stands per clone, so each clone was only planted in one of the three locations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS2.xlsx">Table S2</a>). Clones were assigned to each location so as to equally represent each family in every environment.</p><p id="p-12">For both datasets, we analyzed three traits: MCMDS, DM, and RTWT. MCMDS was scored on a scale of 1 (no symptoms) to 5 (severe symptoms). We note that the distribution of MCMDS is skewed toward low disease severity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/FigureS2.pdf">Figure S2</a>). Most GG trials measured DM by the oven drying method although some trials used the specific gravity method. Dry matter content is expressed as a percentage of the fresh weight of roots. Fresh root weight (RTWT) is measured in kilograms per plot and is natural-log transformed to achieve normally distributed, homoscedastic residuals in all presented analyses. Trait distributions are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/FigureS2.pdf">Figure S2</a>.</p></div><div id="sec-3" class="subsection"><h3>Genotype data</h3><p id="p-13">We used genotyping-by-sequencing (GBS) to obtain genome-wide SNP marker data (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Elshire <em>et al.</em> 2011</a>). We used the ApeKI restriction enzyme as recommended by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hamblin and Rabbi (2014)</a>. SNPs were called using the TASSEL V4 GBS pipeline (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Glaubitz <em>et al.</em> 2014</a>) and aligned to the cassava reference genome, version 5, which is available on Phytozome (<a href="http://phytozome.jgi.doe.gov">http://phytozome.jgi.doe.gov</a>) and described by the International Cassava Genetic Map Consortium (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">ICGMC 2015</a>). We removed individuals with &gt;80% missing and markers with &gt;60% missing genotype calls. Also, markers with extreme deviation from Hardy–Weinberg equilibrium (χ<sup>2</sup> &gt; 20) were removed. If there were not at least two reads at a given locus for a given clone, the genotype was set to missing and imputed. SNP marker data were converted to the dosage format (−1 for reference-allele homozygotes, 0 for heterozygotes, and +1 for alternative-allele homozygotes) and missing data were imputed with the glmnet algorithm in R (<a href="http://cran.r-project.org/web/packages/glmnet/index.html">http://cran.r-project.org/web/packages/glmnet/index.html</a>). Similar to the approach of <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wong <em>et al.</em> (2014)</a>, for each marker to be imputed, we preselected the 60 markers on the same chromosome in highest LD. We then used these preselected markers to predict missing values using the LASSO (default, <em>q</em> = 1 in glmnet), with the tuning parameter λ selected by fivefold cross-validation. We used 114,922 markers that passed these filters with a minor allele frequency &gt;1% to construct GRMs as described below.</p></div><div id="sec-4" class="subsection"><h3>Genomic relationship matrices</h3><p id="p-14">In order to capture additive effects variance, we constructed the genomic relationship matrix (<strong>G</strong>) using the formula of <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">VanRaden (2008)</a>, method one: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-1.gif"/></noscript></span></span> Here <strong>Z</strong> is a mean-centered matrix of dimension <em>n</em> individuals by <em>m</em> SNP markers. To obtain <strong>Z</strong>, we subtract 2(<em>p<sub>i</sub></em> − 0.5) from the marker dosages, where the dosages are coded −1 for aa, 0 for Aa, +1 for AA, <em>p<sub>i</sub></em> is the frequency of the second allele (A) at the <em>i</em>th locus, and <em>q<sub>i</sub></em> = 1−<em>p<sub>i</sub></em>. The <em>a</em> (or 0) allele refers to the reference genome allele. The <strong>G</strong> matrix was calculated using the <em>A.mat</em> function in the <em>rrBLUP</em> package (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Endelman 2011</a>).</p><p id="p-15">We constructed a matrix to capture dominance relationships using the formulation originally proposed by <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Su <em>et al.</em> (2012)</a>. The dominance relationship matrix, which we will call <strong>D</strong>* (see below), is <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-2.gif"/></noscript></span></span> where <strong>H</strong> is a mean-centered dominance deviation matrix with the same dimensions as <strong>Z</strong>. To obtain <strong>H</strong>, we score heterozygotes as 1 and homozygotes as 0, and subtract the mean (2<em>p<sub>i</sub>q<sub>i</sub></em>) from the scores. We made a custom modification (available at <a href="ftp://ftp.cassavabase.org/manuscripts/">ftp://ftp.cassavabase.org/manuscripts/</a>) to the <em>A.mat</em> function to produce the <strong>D</strong>* matrix.</p><p id="p-16">The <strong>D</strong>* dominance matrix was shown by <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Vitezica <em>et al.</em> (2013)</a> to produce a partition of genetic variance appropriate for studying genetic architecture because it isolates additive effect variance from dominance effects. However, this partition is not correct for breeding purposes because the additive variance produced is not equivalent to the variation in breeding value. <a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Vitezica <em>et al.</em> (2013)</a> subsequently derived the matrix <strong>D</strong>, defined as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-3.gif"/></noscript></span></span> where <strong>W</strong> is a marker matrix with markers coded 0 for aa, 2<em>p<sub>i</sub></em> for Aa, and 4<em>p<sub>i</sub></em> − 2 for AA and then centered on the mean, 2<em>p<sub>i</sub></em><sup>2</sup>. Although our focus in the present study is not on the prediction of breeding value, the matrix <strong>D</strong> has been shown by <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zhu <em>et al.</em> (2015)</a> to have the advantage of being uncorrelated (under Hardy–Weinberg equilibrium) with the matrix <strong>G</strong>. For this reason, we tested the <strong>D</strong> and <strong>D</strong>* matrices and provide comparison of their results. Except where explicitly comparing matrices, we use <strong>D</strong> to indicate the dominance matrix <strong>D</strong>* as in <a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Su <em>et al.</em> (2012)</a>.</p><p id="p-17">Finally, we constructed relationship matrices that capture epistasis by taking the Hadamard product (element-by-element multiplication; denoted #) of matrices (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Henderson 1985</a>). For simplicity, we only explored additive-by-additive (<strong>A#A</strong>) and additive-by-dominance (<strong>A#D</strong>) relationships in this study.</p></div><div id="sec-5" class="subsection"><h3>Variance component and heritability models</h3><div id="sec-6" class="subsection"><h4>Single-step, multi-environment:</h4><p id="p-18">We used several approaches to estimate the relative importance of additive and nonadditive effects in the Genetic Gain and Cycle 1 datasets. In the first analysis, we analyzed the multi-year, multi-location GG data with a single-step mixed-effects model. Since the entire historical phenotype dataset is large (24,373 observations) and was relatively unbalanced in sample size across years and locations, we only analyzed data from trials with &gt;400 individuals. This filter resulted in a dataset of 7745 observations from three locations (Ibadan, Ubiaja, Mokwa) and 8 yr (2006–2014, except 2012). All 694 genotyped GG clones were represented in this dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS2.xlsx">Table S2</a>).</p><p id="p-19">The models we fit were similar to those described in <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Ly <em>et al.</em> (2013)</a>. The full model was specified as follows: <strong><span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-4.gif"/></noscript></span></span></strong> Here, <strong><em>y</em></strong> represents raw phenotypic observations. In our data, the only fixed effect (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-5.gif"/></noscript></span></span>) was an intercept for all traits except RTWT, which contained a covariate accounting for variation in the number of plants harvested per plot. The random effects terms for experimental design terms included a unique intercept for each trial (<em>i.e.</em>, location-year combination), <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-6.gif"/></noscript></span></span> where <strong>I</strong> is the identity matrix and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-7.gif"/></noscript></span></span> is the associated variance component as well as a replication effect, nested in location-year combination, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-8.gif"/></noscript></span></span></p><p id="p-20">The genetic variance component terms included <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-9.gif"/></noscript></span></span> where <strong>G</strong> is the additive genetic relationship matrix and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-10.gif"/></noscript></span></span> is the additive genetic variance component. Similarly, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-11.gif"/></noscript></span></span> is the dominance effect with covariance <strong>D</strong> equal to the dominance relationship matrix and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-12.gif"/></noscript></span></span> equal to the dominance variance. The epistatic term <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-13.gif"/></noscript></span></span> where the covariance matrix <strong>E</strong> took the form either of the <strong>A#A</strong> matrix (additive-by-additive) or the <strong>A#D</strong> matrix (additive-by-dominance) and the epistatic variance <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-14.gif"/></noscript></span></span> was correspondingly either <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-15.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-16.gif"/></noscript></span></span> The final term <strong>ε</strong> is the residual variance, assumed to be random and distributed <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-17.gif"/></noscript></span></span> The terms <strong>X</strong>, <strong>Z<sub>loc.year</sub></strong>, <strong>Z<sub>rep</sub></strong>, <strong>Z<sub>add</sub></strong>, <strong>Z<sub>dom</sub></strong>, and <strong>Z<sub>epi</sub></strong> are incidence matrices relating observations to the levels of each factor. We list the different models fit in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, each of which are variations on the full model described above.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469660/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469660/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Additive plus nonadditive genetic models tested and their abbreviations</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">The formulation described above was used to fit the subset of the GG historical data described above in a single model. For the C1 progenies only a single season was available and therefore we fit all data together in a single model. Since the C1 trials were conducted across three locations but with no replications we fit the same model for C1 as GG excluding the replication term. The models described above were fit using the <em>regress</em> package in R (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Clifford and McCullagh 2006</a>). The <em>regress</em> function finds restricted maximum likelihood (REML) solutions to mixed models using the Newton–Raphson algorithm.</p><p id="p-22">For each trait, in both the C1 and GG we identified a “best fit” model among the models listed in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, based on the lowest Akaike Information Criterion [AIC; 2*<em>k</em> – 2*ln(<em>likelihood</em>), where <em>k</em> = number of parameters estimated]. In addition, we calculated the Bayesian Information Criterion [BIC; −2*ln(likelihood) + <em>k</em>*ln(<em>n</em>), where <em>n</em> = number of observations and <em>k</em> = number of parameters estimated]. We also examined the log-likelihood of each model and the proportion of variance explained by genetic factors (<em>H</em><sup>2</sup>). The precision of variance component estimates and the dependency among estimates was examined using the asymptotic variance–covariance matrix of estimated parameters, provided by <em>regress</em> (<strong>V</strong>). We report standard errors for each variance component, defined as the square root of the diagonal of <strong>V</strong>. We also converted <strong>V</strong> into a correlation matrix [<strong>F</strong>, as in <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Muñoz <em>et al.</em> (2014)</a>], where <strong>F</strong> is defined as <strong>L<sup>−1/2</sup>VL<sup>−1/2</sup></strong> and <strong>L</strong> is a diagonal matrix containing one over the square root of the diagonal of <strong>V</strong>. We use <strong>F</strong> to assess the dependency of variance components estimates, especially for comparing results among traits and datasets.</p></div><div id="sec-7" class="subsection"><h4>Within-trial analyses:</h4><p id="p-23">We used only a subset of the GG trials to estimate variance components in the single-step multi-environment model described above. In addition, we were able to analyze the entire historical GG data by testing each trial (<em>N</em> = 47, unique location-year combinations) separately. This provided us with 47 estimates of additive, dominance, and epistatic variance. We examine the distribution of variance components estimates. As in the multi-environment models, within-trial models were fit with <em>regress</em> in R.</p></div></div><div id="sec-8" class="subsection"><h3>Genomic prediction and cross-validation</h3><p id="p-24">We assessed the influence that modeling nonadditive genetic variance components have on genomic prediction using a cross-validation strategy. Because single-step multi-environment models are computationally intensive, we used a two-step approach here. In the first step, we combined data from all available GG and C1 trials using the following mixed model: <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-18.gif"/></noscript></span></span> In this model, <strong>β</strong> included a fixed effect for the population mean, the location-year combination, and for RTWT only, the number of plants harvested per plot. As in the single-step, multi-environment model for GG, we included the random replication effect <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-19.gif"/></noscript></span></span> In contrast to the previous model, we did not at this stage include a genomic relationship matrix, instead we fit a random effect for clone, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-20.gif"/></noscript></span></span> where the covariance structure was the identity matrix, <strong>I</strong>. The BLUP (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-21.gif"/></noscript></span></span>) for the clone effect therefore represents an estimate of the total genetic value for each individual. The mixed model above was solved using the <em>lmer</em> function of the <em>lme4</em> R package (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bates <em>et al.</em> 2014</a>).</p><p id="p-25">In our data, the number of observations per clone ranges from 1 to 131 with median of 2 and mean of 5.97 excluding the checks TMEB1 and I30572, which had 941 and 902 observations, respectively. Pooling information from multiple years and locations, especially when there is so much variation in numbers of observations can introduce bias. Much theory, particularly in animal breeding has been developed to address this issue, and we followed the approach recommended by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Garrick <em>et al.</em> (2009)</a>. Briefly, BLUPs (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-22.gif"/></noscript></span></span>) for clone were deregressed according to <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-23.gif"/></noscript></span></span> where <strong><em>r</em><sup>2</sup></strong> is the reliability <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-24.gif"/></noscript></span></span> and PEV is the prediction error variances of the BLUP. In the second step of analysis, where deregressed BLUPs are used as response variables, weights are applied to the diagonal of the error variance–covariance matrix <strong>R</strong>. Weights are calculated as <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-25.gif"/></noscript></span></span> where <em>h<sup>2</sup></em> is the proportion of the total variance explained by the clonal variance component, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-26.gif"/></noscript></span></span> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Garrick <em>et al.</em> 2009</a>).</p><p id="p-26">We implemented a fivefold cross-validation scheme replicated 25 times to test the accuracy of genomic prediction using the GRMs and models described above (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). In this scheme, for each replication, we randomly divided the dataset into five equally sized parts (<em>i.e.</em>, folds). We used each fold in turn for validation by removing its phenotypes from the training population and then predicting them. We calculated accuracy as the Pearson correlation between the genomic prediction and the BLUP (<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3497/embed/mml-math-27.gif"/></noscript></span></span>, not-deregressed) from the first step. For each model, we calculated accuracy of the prediction for total genetic value, defined as the sum of the predictions from all available kernels (<em>e.g.</em>, additive + dominance + epistasis). Genomic predictions were made using the <em>EMMREML</em> R package (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akdemir and Okeke 2015</a>).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-27">All raw genotype and phenotype data are available at <a href="ftp://ftp.cassavabase.org/manuscripts/">ftp://ftp.cassavabase.org/manuscripts/</a> along with custom code used to make deregressed BLUPs, conduct fold cross-validation, and calculate dominance relationship matrices.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Partitioning the genetic variance: single-step, multi-environment models</h3><p id="p-28">We used several approaches to estimate genetic variance components in our dataset. The first was to fit single-step models to two datasets: the Genetic Gain (GG) and the Cycle 1 (C1). For each trait, in each dataset, we first identified the best fitting model of the five tested (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) on the basis of lowest AIC. Model comparisons based on AIC and BIC are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Key results from the best models for both GG and C1 are summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> with more detailed results from all models provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS4.xlsx">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS5.xlsx">Table S5</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469522/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469522/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of models by AIC and BIC</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469514/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469514/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Best fitting single-step multi-environment model results</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">The AIC-selected best model for MCMDS included additive-by-dominance epistasis (A×D) in both GG and C1. For RTWT, the model with additive-by-additive epistasis (A×A) fit best in the GG but a simpler dominance only (Dom) model was selected in the C1 dataset. Finally, for DM the additive only (Add) model was best in the GG but additive plus dominance (A+D) was selected in the C1 dataset. The BIC criterion places a steeper penalty on increasing the number of parameters. Nevertheless, BIC selected the same model as AIC in all cases except for RTWT in the GG dataset, where the additive plus dominance model was preferred (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS5.xlsx">Table S5</a>). Based on the guidelines of <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Raftery (1995)</a>, the evidence that the best models for RTWT include dominance and models for MCMDS include nonadditive effects, especially epistasis, is very strong (&gt;10 AIC/BIC difference).</p><p id="p-32">We noted that for every trait, when comparing the model achieving the highest broad-sense heritability (<em>H</em><sup>2</sup>), the <em>H</em><sup>2</sup> was higher in C1 compared to GG. This can be seen most easily in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, which shows how total explainable genetic variance (<em>H</em><sup>2</sup>) is partitioned among variance components in the C1 and GG (also see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Tables 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS5.xlsx">Table S5</a>). We also noted that the additive only model had the highest <em>H</em><sup>2</sup> for all traits in the GG dataset, but C1 models with nonadditive components always had at least slightly higher <em>H</em><sup>2</sup>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Partitioning of broad-sense heritability for single-step multi-environment models in the Genetic Gain and Cycle 1 datasets. Results from each of five models are shown in each panel broken down by trait (rows) and population (columns). Models include additive only (Additive), dominance only (Dominance), Additive plus Dominance (Add + Dom), Additive plus dominance plus either A&#xD7;A epistasis (A&#xD7;A Epistasis) or A&#xD7;D epistasis (A&#xD7;D Epistasis)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-472610671" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Partitioning of broad-sense heritability for single-step multi-environment models in the Genetic Gain and Cycle 1 datasets. Results from each of five models are shown in each panel broken down by trait (rows) and population (columns). Models include additive only (Additive), dominance only (Dominance), Additive plus Dominance (Add + Dom), Additive plus dominance plus either A&#xD7;A epistasis (A&#xD7;A Epistasis) or A&#xD7;D epistasis (A&#xD7;D Epistasis).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3497/F1.medium.gif" width="440" height="334"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3497/F1.medium.gif" width="440" height="334"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Partitioning of broad-sense heritability for single-step multi-environment models in the Genetic Gain and Cycle 1 datasets. Results from each of five models are shown in each panel broken down by trait (rows) and population (columns). Models include additive only (Additive), dominance only (Dominance), Additive plus Dominance (Add + Dom), Additive plus dominance plus either A×A epistasis (A×A Epistasis) or A×D epistasis (A×D Epistasis).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">On the basis of genetic variance captured, DM had <em>H</em><sup>2</sup> between 0.25 and 0.53 and had mostly additive inheritance across all models (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS5.xlsx">Table S5</a>). In contrast, nonadditive components accounted for the majority of genetic variance for RTWT, with <em>H</em><sup>2</sup> between 0.21 and 0.33. Dominance was significant in all models tested for RTWT in both datasets and epistasis was significant in the GG dataset. MCMDS had the highest <em>H</em><sup>2</sup> compared to the other traits (0.66–0.89) and was similar to RTWT in that dominance and/or epistasis were always significant where included. While nonadditive genetics were the majority of <em>H</em><sup>2</sup> in GG, they were much less important in C1 for MCMDS.</p><p id="p-35">We examined the asymptotic correlation matrices of parameter estimates (<strong>F</strong>) to ascertain the dependency of variance component estimation. The correlation between genetic variance components was always negative and was, in general, of greater magnitude in the GG compared to the C1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS6.xlsx">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS7.xlsx">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS8.xlsx">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS9.xlsx">Table S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS10.xlsx">Table S10</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS11.xlsx">Table S11</a>). Correlations between additive and dominance components were greatest in the A+D models (range −0.81 to −0.83 in the GG and −0.5 to −0.61 in the C1). Correlations between additive and dominance components dropped in models with epistasis (range −0.42 to −0.63, GG and −0.26 to −0.58, C1). Correlations between additive and A×A epistatic variances (range −0.09 to −0.29) and A×D epistasis (range −0.07 to −0.22) were low. Correlations between dominance components and epistasis were higher, ranging from −0.28 to −0.64 with A×A epistasis and −0.36 to −0.69 with A×D epistasis.</p><p id="p-36">Comparison of the two alternative dominance matrices <strong>D</strong>* (results described above) and <strong>D</strong> revealed very similar results. In almost every case, AIC and BIC selected the same best fit model for <strong>D</strong> and <strong>D</strong>*. The exception was RTWT in the C1 dataset where the A+D model was preferred over the dominance only model when using the <strong>D</strong> matrix instead of <strong>D</strong>* (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS4.xlsx">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS5.xlsx">Table S5</a>). The AIC and BIC are on the whole slightly lower for the models using the <strong>D</strong> matrix, indicating a better fit to the data. As expected, the correlation between additive and dominance parameter estimates is of smaller magnitude for all analyses with <strong>D</strong> compared with <strong>D</strong>* (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS6.xlsx">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS7.xlsx">Table S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS8.xlsx">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS9.xlsx">Table S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS10.xlsx">Table S10</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS11.xlsx">Table S11</a>). However, the correlation between additive and epistatic as well as between dominance and epistatic variances is always of greater magnitude with <strong>D</strong>. Finally and as expected, models using the <strong>D</strong> matrix generally explain the same amount of genetic variance as those with <strong>D</strong>* but partition a smaller portion to dominance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/FigureS3.pdf">Figure S3</a>). We noted that for RTWT in the C1 dataset, models with <strong>D</strong> actually achieve a slightly higher broad-sense heritability. Because of the similarity of results, we focused the remainder of our analyses and discussion on the results from the <strong>D</strong>* matrix, henceforth referred to only as <strong>D</strong>.</p></div><div id="sec-12" class="subsection"><h3>Partitioning the genetic variance: within-trial analyses</h3><p id="p-37">We also examined variance partitioning within each of 47 GG trials for the five models described in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>. This provided a means of testing the entire available dataset for nonadditive variances, in contrast to the multi-environment models described above. The mean and variability of model parameters (variance components, heritability, <em>etc</em>.) across these trials are summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS12.xlsx">Table S12</a> and results for each individual trial-model combination are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS13.xlsx">Table S13</a>. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> provides a visual summary of the proportion of phenotypic variability explained by each genetic variance component on average across the trials. We also compared the mean AIC across trials (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS10.xlsx">Table S10</a>) and found them to agree overall with the results of the one-step multi-environment models (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Specifically, the models that fit best in the one-step models were best on average in the within-trial analyses for DM (Add) and MCMDS (A×D). However, for RTWT the within-trial AIC-best model was A+D compared to A×D in the one-step multi-environment model. In contrast to the one-step multi-environment model, the BIC agreed with AIC only for DM. For RTWT and MCMDS the simpler dominance only model was preferred by BIC on average (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of genetic variance proportions across GG trials. Three models were fitted for each trait in each of 47 GG trials. Each panel contains boxplots showing the distribution of proportions of the phenotypic variability explained by a corresponding genetic factor, including the broad-sense heritability (H2). Red horizontal lines are the median narrow-sense heritability (h2) from the additive only model. Traits are on columns and three models are on the rows: additive plus dominance (Add + Dom), additive plus dominance plus A&#xD7;A epistasis (A&#xD7;A Epistasis), and additive plus dominance plus A&#xD7;D epistasis (A&#xD7;D Epistasis)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-472610671" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of genetic variance proportions across GG trials. Three models were fitted for each trait in each of 47 GG trials. Each panel contains boxplots showing the distribution of proportions of the phenotypic variability explained by a corresponding genetic factor, including the broad-sense heritability (&lt;em&gt;H&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;). Red horizontal lines are the median narrow-sense heritability (&lt;em&gt;h&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) from the additive only model. Traits are on columns and three models are on the rows: additive plus dominance (Add + Dom), additive plus dominance plus A&#xD7;A epistasis (A&#xD7;A Epistasis), and additive plus dominance plus A&#xD7;D epistasis (A&#xD7;D Epistasis).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3497/F2.medium.gif" width="361" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3497/F2.medium.gif" width="361" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469527" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">Distribution of genetic variance proportions across GG trials. Three models were fitted for each trait in each of 47 GG trials. Each panel contains boxplots showing the distribution of proportions of the phenotypic variability explained by a corresponding genetic factor, including the broad-sense heritability (<em>H</em><sup>2</sup>). Red horizontal lines are the median narrow-sense heritability (<em>h</em><sup>2</sup>) from the additive only model. Traits are on columns and three models are on the rows: additive plus dominance (Add + Dom), additive plus dominance plus A×A epistasis (A×A Epistasis), and additive plus dominance plus A×D epistasis (A×D Epistasis).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genomic prediction of additive and total genetic value</h3><p id="p-39">We used cross-validation to assess the prediction accuracy for total genetic value from the five models (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>) in both datasets. Compared to the single-kernel additive prediction using the additive relationship matrix, multi-kernel total genetic value predictions were an average of 7% better (maximum of 26% improvement; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS14.xlsx">Table S14</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1/TableS15.xlsx">Table S15</a>). By model, improvements in the correlation between total value and phenotype over the additive only model were 7, 7, and 8% for A+D, A×A, and A×D, respectively. The additive only model predictions were on average 12% less accurate in the C1 than in the GG. Total genetic value predictions were less accurate by 12% in the C1 relative to GG. The models we fit for genomic prediction involved the estimation by <em>EMMREML</em> of weights, used to create a single kernel that is the weighted average of multiple original kernels and corresponding to the partitioning of genetic variance among the kernels. The average total weight given to nonadditive components for both DM and MCMDS was 0.41 but was 0.92 for RTWT.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of total genetic value prediction in the Genetic Gain and Cycle 1 datasets. Boxplots showing the distribution over 25 replicates of fivefold cross-validation of the prediction accuracy of the total genetic value from five different models are shown in each panel. The accuracy within the Genetic Gain (red) and Cycle 1 (blue) are shown. Traits are in the columns. Accuracy is defined as the correlation between the sum of predictions from all genetic variance components in the model and the BLUP from the first stage of analysis where location, year, and replicate variability were removed. Models included are: additive only (Additive), dominance only (Dominance), additive plus dominance (Add + Dom), additive plus dominance plus A&#xD7;A epistasis (A&#xD7;A Epi.), and additive plus dominance plus A&#xD7;D epistasis (A&#xD7;D Epi.)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-472610671" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy of total genetic value prediction in the Genetic Gain and Cycle 1 datasets. Boxplots showing the distribution over 25 replicates of fivefold cross-validation of the prediction accuracy of the total genetic value from five different models are shown in each panel. The accuracy within the Genetic Gain (red) and Cycle 1 (blue) are shown. Traits are in the columns. Accuracy is defined as the correlation between the sum of predictions from all genetic variance components in the model and the BLUP from the first stage of analysis where location, year, and replicate variability were removed. Models included are: additive only (Additive), dominance only (Dominance), additive plus dominance (Add + Dom), additive plus dominance plus A&#xD7;A epistasis (A&#xD7;A Epi.), and additive plus dominance plus A&#xD7;D epistasis (A&#xD7;D Epi.).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3497/F3.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3497/F3.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3497/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469597" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Accuracy of total genetic value prediction in the Genetic Gain and Cycle 1 datasets. Boxplots showing the distribution over 25 replicates of fivefold cross-validation of the prediction accuracy of the total genetic value from five different models are shown in each panel. The accuracy within the Genetic Gain (red) and Cycle 1 (blue) are shown. Traits are in the columns. Accuracy is defined as the correlation between the sum of predictions from all genetic variance components in the model and the BLUP from the first stage of analysis where location, year, and replicate variability were removed. Models included are: additive only (Additive), dominance only (Dominance), additive plus dominance (Add + Dom), additive plus dominance plus A×A epistasis (A×A Epi.), and additive plus dominance plus A×D epistasis (A×D Epi.).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-41">In clonally propagated crops, nonadditive genetic effects can be effectively exploited by the identification of superior genetic individuals as varieties. For this reason, we quantified the amount and nature of nonadditive genetic variation for key traits in a genomic selection breeding population of cassava from sub-Saharan Africa. We then assessed the accuracy of genomic prediction of additive compared to total (additive plus nonadditive) genetic value. Using several approaches and datasets based on genome-wide marker data, we confirmed previous findings in cassava based on diallel crosses nonadditive genetic variation is significant, especially for yield traits (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cach <em>et al.</em> 2005</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2006</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Calle <em>et al.</em> 2005</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jaramillo <em>et al.</em> 2005</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Pérez <em>et al.</em> 2005a</a>,<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">b</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Zacarias and Labuschagne 2010</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Kulembeka <em>et al.</em> 2012</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Tumuhimbise <em>et al.</em> 2014</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Ceballos <em>et al.</em> 2015</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chalwe <em>et al.</em> 2015</a>). A potential weakness of the marker system we used (GBS) is that it generates a high proportion of missing marker data and it may undercall heterozygotes when read depth is insufficient. The similarity of our findings to previous research, and the important difference in the observations on RTWT <em>vs.</em> DM, however, suggest that this weakness did not strongly affect our results. Further, we found that multi-component models incorporating nonadditive effects predict observed phenotypes more accurately than additive only models for root yield but not for DM content, which has primarily additive inheritance, or for CMD resistance, which has high narrow-sense heritability. We address the implication of these results for cassava breeding and put our work in the context of previous results in cassava, other plant, and animal species below.</p><p id="p-42">Our results indicate strong nonadditive (mainly dominance) variance for root yields and mostly additive inheritance of root DM content. These findings confirm the conclusions of numerous diallelic studies conducted with both Latin American (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Cach <em>et al.</em> 2005</a>, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">2006</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Calle <em>et al.</em> 2005</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Jaramillo <em>et al.</em> 2005</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Pérez <em>et al.</em> 2005a</a>,<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">b</a>) and African cassava (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Zacarias and Labuschagne 2010</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Kulembeka <em>et al.</em> 2012</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Tumuhimbise <em>et al.</em> 2014</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chalwe <em>et al.</em> 2015</a>) germplasm (see also <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Ceballos <em>et al.</em> 2015</a>). In agreement with the findings of <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Ly <em>et al.</em> (2013)</a>, we found cassava mosaic disease severity (MCMDS) to be well predicted with an additive only model. However, we found significant dominance and epistatic components in both populations analyzed. This result is in line with previous diallelic studies indicating significant SCA (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Tumuhimbise <em>et al.</em> 2014</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Chalwe <em>et al.</em> 2015</a>) and genetic mapping studies that identified a single major effect QTL with a dominant CMD resistance effect (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akano <em>et al.</em> 2002</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Okogbenin <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rabbi <em>et al.</em> 2014</a>). In addition, a recent genome-wide association and prediction study of MCMDS using nonadditive GRMs found that dominance and especially epistasis explain most of the variance in the region of a large-effect QTL, suggesting multiple interacting loci in the region (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Wolfe <em>et al.</em> 2016</a>).</p><p id="p-43">The importance of nonadditive genetic variance in evolution by natural and artificial selection is controversial (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Crow 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hansen 2013</a>). Nevertheless, numerous studies have found and exploited dominance and epistasis in animal breeding, including dairy (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahlborn-Breier and Hohenboken 1991</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fuerst and Sölkner 1994</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Varona <em>et al.</em> 1998</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Van Tassell <em>et al.</em> 2000</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Palucci <em>et al.</em> 2007</a>) and beef (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rodriguezalmeida <em>et al.</em> 1995</a>) cattle. Diallelic studies have indicated significant SCA for maize grain yield (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Doerksen <em>et al.</em> 2003</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wardyn <em>et al.</em> 2007</a>). Aside from cassava, breeding of other noninbred, clonally propagated species also identifies and makes use of nonadditive effects, including in potato (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Killick 1977</a>), <em>Eucalyptus</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Costa <em>et al.</em> 2004</a>), and loblolly pine (<a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Muñoz <em>et al.</em> 2014</a>). More recently, marker-based and GRM-based models have identified significant nonadditive effects in pigs (<a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2012</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nishio and Satoh 2014</a>), mice (<a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Vitezica <em>et al.</em> 2013</a>), beef cattle (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolormaa <em>et al.</em> 2015</a>), dairy cows (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Morota <em>et al.</em> 2014</a>), maize (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dudley and Johnson 2009</a>), soy (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hu <em>et al.</em> 2011</a>), loblolly pine (<a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">Muñoz <em>et al.</em> 2014</a>), and apple (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kumar <em>et al.</em> 2015</a>). Results from the present study suggest that accounting for nonadditive effects in the variety development pipeline should increase the value of hybrids released by cassava breeding programs.</p><p id="p-44">One of the more interesting aspects of our study relative to previous ones is the comparison between a parental generation (the Genetic Gain) and their offspring (Cycle 1), a collection of full- and half-sib families. From GG to C1, the <em>H</em><sup>2</sup> generally increased. For RTWT, this is largely attributable to increased nonadditive variance in contrast to MCMDS where the increase is concomitant with a drop in nonadditive variance. In contrast to our result, the theory suggests that reduction (or fixation) of allele frequencies at some loci relative to others in populations undergoing bottlenecks (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Goodnight 1988</a>), inbreeding (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Turelli and Barton 2006</a>), or truncation selection (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hallander and Waldmann 2007</a>) should cause a conversion of nonadditive (where present) to additive variance. These results have, however, been based on models with finite numbers of loci in linkage equilibrium. Based on the mean diagonal of the additive genetic relationship matrix, C1 (0.66) does not appear notably more inbred than GG (0.64). We also calculated mean pairwise LD (GG = 0.27, C1 = 0.29) and mean LD block size (21.7 kb in GG and 23.1 kb in C1) using standard settings in PLINK (version 1.9, <a href="https://www.cog-genomics.org/plink2">https://www.cog-genomics.org/plink2</a>) and found the two generations to be similar.</p><p id="p-45">Probably the strongest explanation for the difference in genetic variance components between GG and C1 is the family structure (135 full-sib families from 83 outbred parents). In a population of full-sibs three quarters of the dominance variance is expressed within families and all of it for half-sib populations (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hallauer <em>et al.</em> 2010</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Ceballos <em>et al.</em> 2015</a>). Indeed, increasing the number of full-sib relationships is known to increase the nonadditive genetic variance detectable in a population (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Varona <em>et al.</em> 1998</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Van Tassel <em>et al.</em> 2010</a>).</p><p id="p-46">It is also conceivable that maternal plant effects could increase apparent nonadditive effects in C1. The C1 clones in contrast to the GG clones are new, and were derived from stem cuttings of seedling plants germinated in the previous field season (2012–2013). The suggestion is therefore that the quality and vigor of the seedling plant giving rise to the C1 clones may influence their performance in the 2013–2014 trial. We further caution that comparison of GG and C1 may be biased by the disproportionate amount of data from different locations and years available for the GG.</p><p id="p-47">In our study, when additive and nonadditive kernels were used together, the variance explained by the additive component, particularly for RTWT, decreased. One interpretation of this result is that the additive component alone absorbs some nonadditive variance. Similar results have been obtained by other researchers, leading to similar conclusions (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lu <em>et al.</em> 1999</a>; <a id="xref-ref-59-7" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2012</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zuk <em>et al.</em> 2012</a>; <a id="xref-ref-46-7" class="xref-bibr" href="#ref-46">Muñoz <em>et al.</em> 2014</a>). We note that this phenomenon occurs whether we use the <strong>D</strong>* matrix, which is correlated with the <strong>G</strong> matrix, or the <strong>D</strong> matrix, which is theoretically orthogonal to <strong>G</strong>. We suggest therefore that prediction models that do not explicitly incorporate nonadditive components may achieve gains in the short term that break down over the long term (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cockerham and Tachida 1988</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Walsh 2005</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hansen 2013</a>). Our prediction tests in this study were focused on total genetic values and used the <strong>D</strong>* matrix. However, we hypothesize that including nonadditive GRMs, particularly the <strong>D</strong> matrix, when estimating additive genetic (<em>i.e.</em>, breeding) values would provide a less biased, more accurate selection of parents for crossing.</p><p id="p-48">Nonadditive variation is prevalent in cassava, especially for low-heritability traits. This has many important implications for cassava breeding. It explains, in part, why GGs have been slow (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Ceballos <em>et al.</em> 2012</a>). Inbreeding to convert dominance variance to additive and better control epistatic combinations, as in maize, has been suggested as a solution to nonadditive genetics (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Ceballos <em>et al.</em> 2015</a>). Even for low <em>h</em><sup>2</sup> traits and without inbred cassava, using the kinds of models presented in this paper, good parents can be selected based on additive predictions and total genetic value can be predicted for the identification of potential commercial varieties, all based on the combination of marker and preliminary field trial data (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2015</a>). This approach has been previously advocated for plant breeding (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Oakey <em>et al.</em> 2007</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2015</a>) and has proven effective in animal breeding (<em>e.g.</em>, <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ahlborn-Breier and Hohenboken 1991</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Palucci <em>et al.</em> 2007</a>; <a id="xref-ref-59-8" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2012</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Nishio and Satoh 2014</a>). Nonadditive models using GRMs can thus improve the efficiency and productivity of variety selection pipelines that are the most labor- and time-intensive part of selecting good cassava clones after crossing.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We give special thanks to A.G.O. Dixon for his development of many of the breeding lines and historical data we analyzed. We also thank A.I. Smith and technical teams at the International Institute of Tropical Agriculture for collection of phenotypic data, and A. Agbona and P. Peteti for data curation. We acknowledge the Bill &amp; Melinda Gates Foundation and UKaid (grant 1048542; <a href="http://www.gatesfoundation.org">http://www.gatesfoundation.org</a>) and support from the Consultative Group for International Agricultural Research (CGIAR) Research Program on Roots, Tubers and Bananas (<a href="http://www.rtb.cgiar.org">http://www.rtb.cgiar.org</a>).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-50">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033332/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-51"><em>Communicating editor: G. A. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-08"><span class="received-label">Received </span>July 8, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-21"><span class="accepted-label">Accepted </span>August 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Wolfe <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.1" data-doi="10.3168/jds.S0022-0302(91)78206-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahlborn-Breier</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenboken</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Additive and nonadditive genetic effects on milk production in dairy cattle: evidence for major individual heterosis.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAhlborn-Breier%26rft.auinit1%253DG.%26rft.volume%253D74%26rft.issue%253D2%26rft.spage%253D592%26rft.epage%253D602%26rft.atitle%253DAdditive%2Band%2BNonadditive%2BGenetic%2BEffects%2Bon%2BMilk%2BProduction%2Bin%2BDairy%2BCattle%253A%2BEvidence%2Bfor%2BMajor%2BIndividual%2BHeterosis%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252891%252978206-4%26rft_id%253Dinfo%253Apmid%252F2045565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(91)78206-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2045565&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okeke</span>  <span class="cit-name-given-names">U.G.</span></span></li></ol><cite>. <span class="cit-pub-date">2015</span> <span class="cit-article-title">EMMREML: Fitting Mixed Models with Known Covariance Structures.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.3" data-doi="10.1007/s00122-002-0891-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akano</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mba</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fregene</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic mapping of a dominant gene conferring resistance to cassava mosaic disease.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAkano%26rft.auinit1%253DO.%26rft.volume%253D105%26rft.issue%253D4%26rft.spage%253D521%26rft.epage%253D525%26rft.atitle%253DGenetic%2Bmapping%2Bof%2Ba%2Bdominant%2Bgene%2Bconferring%2Bresistance%2Bto%2Bcassava%2Bmosaic%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-0891-7%26rft_id%253Dinfo%253Apmid%252F12582500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-0891-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582500&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178256200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maechler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fitting linear mixed-effects models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.5" data-doi="10.1186/s12711-014-0081-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolormaa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reverter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barendse</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Non-additive genetic variation in growth, carcass and fertility traits of beef cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0081-5%26rft_id%253Dinfo%253Apmid%252F25595431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0081-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25595431&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.6" data-doi="10.1093/jhered/esi084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cach</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenis</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morante</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Epistasis in the expression of relevant traits in cassava (Manihot esculenta Crantz) for subhumid conditions.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesi084%26rft_id%253Dinfo%253Apmid%252F16014808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk2LzUvNTg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.7" data-doi="10.1111/j.1439-0523.2006.01192.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cach</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenis</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morante</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inheritance of useful traits in cassava grown in subhumid conditions.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D125%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2006.01192.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2006.01192.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.8" data-doi="10.1007/s10681-005-5810-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaitán</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morante</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Diallel inheritance of relevant traits in cassava (Manihot esculenta Crantz) adapted to acid-soil savannas.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D144%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-5810-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-5810-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232056200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.9" data-doi="10.1007/s12042-012-9094-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hershey</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cassava breeding: current status, bottlenecks and the potential of biotechnology tools.</span> <abbr class="cit-jnl-abbrev">Trop. Plant Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrop.%2BPlant%2BBiol.%26rft.volume%253D5%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12042-012-9094-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12042-012-9094-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.10" data-doi="10.1007/s00122-015-2555-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawuki</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gracen</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yencho</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hershey</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Conventional breeding, marker-assisted selection, genomic selection and inbreeding in clonally propagated crops: a case study for cassava.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D1647%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2555-4%26rft_id%253Dinfo%253Apmid%252F26093610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2555-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26093610&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.11" data-doi="10.1007/s10681-015-1404-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalwe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shanahan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiona</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inheritance of resistance to cassava green mite and other useful agronomic traits in cassava grown in Zambia.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D205%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-015-1404-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-015-1404-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullagh</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The regress function.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">An extension of the concept of partitioning hereditary variance for analysis of covariances among relatives when epistasis is present.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D859%26rft.epage%253D882%26rft.atitle%253DAN%2BEXTENSION%2BOF%2BTHE%2BCONCEPT%2BOF%2BPARTITIONING%2BHEREDITARY%2BVARIANCE%2BFOR%2BANALYSIS%2BOF%2BCOVARIANCES%2BAMONG%2BRELATIVES%2BWHEN%2BEPISTASIS%2BIS%2BPRESENT%26rft_id%253Dinfo%253Apmid%252F17247525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzOS82Lzg1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.14" data-doi="10.1073/pnas.85.5.1563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tachida</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Permanency of response to selection for quantitative characters in finite populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">1563</span>–<span class="cit-lpage">1565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D85%26rft.issue%253D5%26rft.spage%253D1563%26rft.epage%253D1565%26rft.atitle%253DPermanency%2Bof%2Bresponse%2Bto%2Bselection%2Bfor%2Bquantitative%2Bcharacters%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.5.1563%26rft_id%253Dinfo%253Apmid%252F3422750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NS81LzE1NjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.15" data-doi="10.1007/s00122-003-1524-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borralho</span>  <span class="cit-name-given-names">N. M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Additive and non-additive genetic parameters from clonally replicated and seedling progenies of Eucalyptus globulus.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCosta%2BE%2BSilva%26rft.auinit1%253DJ.%26rft.volume%253D108%26rft.issue%253D6%26rft.spage%253D1113%26rft.epage%253D1119%26rft.atitle%253DAdditive%2Band%2Bnon-additive%2Bgenetic%2Bparameters%2Bfrom%2Bclonally%2Breplicated%2Band%2Bseedling%2Bprogenies%2Bof%2BEucalyptus%2Bglobulus.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1524-5%26rft_id%253Dinfo%253Apmid%252F15067398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1524-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15067398&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220655000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.16" data-doi="10.1098/rstb.2009.0275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On epistasis: why it is unimportant in polygenic directional selection.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1241</span>–<span class="cit-lpage">1244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DCrow%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1241%26rft.epage%253D1244%26rft.atitle%253DOn%2Bepistasis%253A%2Bwhy%2Bit%2Bis%2Bunimportant%2Bin%2Bpolygenic%2Bdirectional%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0275%26rft_id%253Dinfo%253Apmid%252F20308099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU0NC8xMjQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.17" data-doi="10.2135/cropsci2003.1652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerksen</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kannenberg</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effect of recurrent selection on combining ability in maize breeding populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1652</span>–<span class="cit-lpage">1658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DDoerksen%26rft.auinit1%253DT.%2BK.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D1652%26rft.epage%253D1658%26rft.atitle%253DEffect%2Bof%2BRecurrent%2BSelection%2Bon%2BCombining%2BAbility%2Bin%2BMaize%2BBreeding%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185174300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.18" data-doi="10.2135/cropsci2008.08.0491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Epistatic models improve prediction of performance in corn.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DDudley%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D49%26rft.issue%253D3%26rft.spage%253D763%26rft.epage%253D770%26rft.atitle%253DEpistatic%2BModels%2BImprove%2BPrediction%2Bof%2BPerformance%2Bin%2BCorn%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000266214500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.19" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.20" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.21" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> <span class="cit-article-title">The correlation between relatives on the supposition of mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.22" data-doi="10.3168/jds.S0022-0302(94)77047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuerst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sölkner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Additive and nonadditive genetic variances for milk yield, fertility, and lifetime performance traits of dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1114</span>–<span class="cit-lpage">1125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DFuerst%26rft.auinit1%253DC.%26rft.volume%253D77%26rft.issue%253D4%26rft.spage%253D1114%26rft.epage%253D1125%26rft.atitle%253DAdditive%2Band%2BNonadditive%2BGenetic%2BVariances%2Bfor%2BMilk%2BYield%252C%2BFertility%252C%2Band%2BLifetime%2BPerformance%2BTraits%2Bof%2BDairy%2BCattle%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252894%252977047-8%26rft_id%253Dinfo%253Apmid%252F8201046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(94)77047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8201046&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994ND26500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.23" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.24" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.25" data-doi="10.2307/2409030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodnight</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Epistasis and the effect of founder events on the additive genetic variance.</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D42%26rft.spage%253D441%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The effect of non-additive genetic interactions on selection in multi-locus genetic models.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D98%26rft.spage%253D349%26rft_id%253Dinfo%253Apmid%252F17327874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17327874&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3497.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carena</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miranda Filho</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source"><em>Quantitative Genetics in Maize Breeding</em></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.28" data-doi="10.2135/cropsci2014.02.0160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of restriction-enzyme choice on properties of genotyping-by-sequencing libraries: a study in cassava</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">2603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D2603%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.02.0160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.02.0160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.29" data-doi="10.1111/evo.12214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Why epistasis is important for selection and adaptation.</span> <abbr class="cit-jnl-abbrev">Evolution (N. Y.)</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">3501</span>–<span class="cit-lpage">3511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2528N.%2BY.%2529%26rft.volume%253D67%26rft.spage%253D3501%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.30" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.31" data-doi="10.2527/jas1985.601111x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Best linear unbiased prediction of nonadditive genetic merits in noninbred populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2BNonadditive%2BGenetic%2BMerits%2Bin%2BNoninbred%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1985.601111x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1985.601111x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AAM9700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.32" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.33" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic value prediction for quantitative traits under the epistatic model.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DHu%26rft.auinit1%253DZ.%26rft.volume%253D12%26rft.spage%253D15%26rft.epage%253D15%26rft.atitle%253DGenomic%2Bvalue%2Bprediction%2Bfor%2Bquantitative%2Btraits%2Bunder%2Bthe%2Bepistatic%2Bmodel.%26rft_id%253Dinfo%253Apmid%252F21269439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21269439&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-other" id="cit-6.11.3497.35" data-doi="10.1534/g3.114.015008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Cassava Genetic Map Consortium (ICGMC)</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-resolution linkage map and chromosome-scale genome assembly for cassava (Manihot esculenta Crantz) from ten populations.</span> <span class="cit-source">G3</span> <span class="cit-vol">5</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.015008%26rft_id%253Dinfo%253Apmid%252F25504737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvMS8xMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.36" data-doi="10.2135/cropsci2004.0314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morante</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Diallel analysis in cassava adapted to the midaltitude valleys environment.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1058</span>–<span class="cit-lpage">1063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DJaramillo%26rft.auinit1%253DG.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D1058%26rft.epage%253D1063%26rft.atitle%253DDiallel%2BAnalysis%2Bin%2BCassava%2BAdapted%2Bto%2Bthe%2BMidaltitude%2BValleys%2BEnvironment%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000229105700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.37" data-doi="10.1098/rspb.1954.0056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">The correlation between relatives in a random mating population.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DKempthorne%26rft.auinit1%253DO.%26rft.volume%253D143%26rft.issue%253D910%26rft.spage%253D103%26rft.epage%253D113%26rft.atitle%253DThe%2BCorrelation%2Bbetween%2BRelatives%2Bin%2Ba%2BRandom%2BMating%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1954.0056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMToiMTQzLzkxMC8xMDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.38" data-doi="10.1111/j.1744-7348.1977.tb01841.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Killick</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Genetic analysis of several traits in potatoes by means of a diallel cross.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Biol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BBiol.%26rft.volume%253D86%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7348.1977.tb01841.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7348.1977.tb01841.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.39" data-doi="10.1007/s10681-012-0730-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulembeka</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herselman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanju</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mkamilo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Diallel analysis of field resistance to brown streak disease in cassava (Manihot esculenta Crantz) landraces from Tanzania.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D187%26rft.spage%253D277%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-012-0730-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-012-0730-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3497.40" data-doi="10.1534/g3.115.021105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molloy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-enabled estimates of additive and non-additive genetic variances and prediction of apple phenotypes across environments.</span> <span class="cit-source">G3</span> <span class="cit-vol">5</span>: <span class="cit-fpage">2711</span>–<span class="cit-lpage">2718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021105%26rft_id%253Dinfo%253Apmid%252F26497141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTIvMjcxMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D29%26rft.spage%253D724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.42" data-doi="10.1139/x99-047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Potential biases of incomplete linear models in heritability estimation and breeding value prediction.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D29%26rft.spage%253D724%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx99-047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x99-047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.43" data-doi="10.2135/cropsci2012.11.0653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melaku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakare</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Relatedness and genotype × environment interaction affect prediction accuracies in genomic selection: a study in cassava.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1312%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.11.0653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.11.0653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3497.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source"><em>Genetics and Analysis of Quantitative Traits</em>.</span> <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.45" data-doi="10.1186/1471-2164-15-109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdollahi-Arpanahi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kranis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-enabled prediction of quantitative traits in chickens using genomic annotation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-109%26rft_id%253Dinfo%253Apmid%252F24502227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24502227&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.46" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unraveling additive from nonadditive effects using genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.47" data-doi="10.1111/asj.12184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Impacts of genotyping strategies on long-term genetic response in genomic selection.</span> <abbr class="cit-jnl-abbrev">Anim. Sci. J.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BSci.%2BJ.%26rft.volume%253D85%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fasj.12184%26rft_id%253Dinfo%253Apmid%252F24506177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/asj.12184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24506177&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.48" data-doi="10.1007/s00122-007-0515-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oakey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitchford</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Joint modeling of additive and non-additive (genetic line) effects in multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DOakey%26rft.auinit1%253DH.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1319%26rft.epage%253D1332%26rft.atitle%253DJoint%2Bmodeling%2Bof%2Badditive%2Band%2Bnon-additive%2B%2528genetic%2Bline%2529%2Beffects%2Bin%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0515-3%26rft_id%253Dinfo%253Apmid%252F17426958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0515-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17426958&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.49" data-doi="10.1080/15427520802212506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okechukwu</span>  <span class="cit-name-given-names">R. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">G. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic gains from 30 years of cassava breeding in Nigeria for storage root yield and disease resistance in elite cassava genotypes.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D22%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427520802212506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427520802212506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.50" data-doi="10.2135/cropsci2011.11.0586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okogbenin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egesi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olasanmi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogundapo</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular marker analysis and validation of resistance to cassava mosaic disease in elite cassava genotypes in Nigeria.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">2576</span>–<span class="cit-lpage">2586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D2576%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.11.0586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.11.0586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.51" data-doi="10.1007/s10681-012-0722-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva Santos</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">G. A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide selection in cassava.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D187%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-012-0722-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-012-0722-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.52" data-doi="10.1186/1297-9686-39-2-181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palucci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miglior</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborne</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Non-additive genetic effects for fertility traits in Canadian Holstein cattle (Open Access publication).</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DPalucci%26rft.auinit1%253DV.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D181%26rft.epage%253D193%26rft.atitle%253DNon-additive%2Bgenetic%2Beffects%2Bfor%2Bfertility%2Btraits%2Bin%2BCanadian%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-39-2-181%26rft_id%253Dinfo%253Apmid%252F17306200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-39-2-181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17306200&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.53" data-doi="10.1007/s10681-005-0424-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morante</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaitán</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">Within-family genetic variation and epistasis in cassava (Manihot esculenta Crantz) adapted to the acid-soils environment.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D145%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-0424-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-0424-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232841500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.54" data-doi="10.2135/cropsci2004.0600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morante</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">Epistasis in cassava adapted to midaltitude valley environments.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPerez%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D45%26rft.issue%253D4%26rft.spage%253D1491%26rft.epage%253D1496%26rft.atitle%253DEpistasis%2Bin%2BCassava%2BAdapted%2Bto%2BMidaltitude%2BValley%2BEnvironments%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000230713700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.55" data-doi="10.1007/s12042-011-9088-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marri</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desany</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowicz</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kodira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The cassava genome: current progress, future directions.</span> <abbr class="cit-jnl-abbrev">Trop. Plant Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DTrop%2BPlant%2BBiol%26rft.aulast%253DProchnik%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D94%26rft.atitle%253DThe%2BCassava%2BGenome%253A%2BCurrent%2BProgress%252C%2BFuture%2BDirections.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12042-011-9088-z%26rft_id%253Dinfo%253Apmid%252F22523606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12042-011-9088-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523606&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000208980100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.56" data-doi="10.1016/j.virusres.2013.12.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gedil</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikpan</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution mapping of resistance to cassava mosaic geminiviruses in cassava using genotyping-by-sequencing and its implications for breeding.</span> <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D186%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2013.12.028%26rft_id%253Dinfo%253Apmid%252F24389096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2013.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24389096&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.57" data-doi="10.2307/271063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayesian model selection in social research.</span> <abbr class="cit-jnl-abbrev">Sociol. Methodol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSociol.%2BMethodol.%26rft.volume%253D25%26rft.spage%253D111%26rft.atitle%253DBAYESIAN%2BMODEL%2BSELECTION%2BIN%2BSOCIAL%2BRESEARCH%26rft_id%253Dinfo%253Adoi%252F10.2307%252F271063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/271063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.58" data-doi="10.2527/1995.7341002x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodriguezalmeida</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanvleck</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willham</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Northcutt</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Estimation of nonadditive genetic variances in 3 synthetic lines of beef-cattle using an animal-model.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1002</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DRodriguez-Almeida%26rft.auinit1%253DF.%2BA.%26rft.volume%253D73%26rft.issue%253D4%26rft.spage%253D1002%26rft.epage%253D1011%26rft.atitle%253DEstimation%2Bof%2Bnon-additive%2Bgenetic%2Bvariances%2Bin%2Bthree%2Bsynthetic%2Blines%2Bof%2Bbeef%2Bcattle%2Busing%2Ban%2Banimal%2Bmodel%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1995.7341002x%26rft_id%253Dinfo%253Apmid%252F7628942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1995.7341002x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7628942&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.59" data-doi="10.1371/journal.pone.0035495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0035495%26rft_id%253Dinfo%253Apmid%252F22506074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0035495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506074&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.60" data-doi="10.1016/j.fcr.2014.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tumuhimbise</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shanahan</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Diallel analysis of early storage root yield and disease resistance traits in cassava (Manihot esculenta Crantz).</span> <abbr class="cit-jnl-abbrev">F. Crop. Res.</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF.%2BCrop.%2BRes.%26rft.volume%253D167%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2014.07.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2014.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.61" data-doi="10.1111/j.0014-3820.2006.tb00521.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turelli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Will population bottlenecks and multilocus epistasis increase additive genetic variance?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1763</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DTurelli%26rft.auinit1%253DM.%26rft.volume%253D60%26rft.issue%253D9%26rft.spage%253D1763%26rft.epage%253D1776%26rft.atitle%253DWill%2Bpopulation%2Bbottlenecks%2Band%2Bmultilocus%2Bepistasis%2Bincrease%2Badditive%2Bgenetic%2Bvariance%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2006.tb00521.x%26rft_id%253Dinfo%253Apmid%252F17089962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2006.tb00521.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17089962&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241226800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.62" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.63" data-doi="10.3168/jds.S0022-0302(00)75059-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Method R estimates of additive genetic, dominance genetic, and permanent environmental fraction of variance for yield and health traits of Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1873</span>–<span class="cit-lpage">1877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVan%2BTassell%26rft.auinit1%253DC.%2BP.%26rft.volume%253D83%26rft.issue%253D8%26rft.spage%253D1873%26rft.epage%253D1877%26rft.atitle%253DMethod%2BR%2BEstimates%2Bof%2BAdditive%2BGenetic%252C%2BDominance%2BGenetic%252C%2Band%2BPermanent%2BEnvironmental%2BFraction%2Bof%2BVariance%2Bfor%2BYield%2Band%2BHealth%2BTraits%2Bof%2BHolsteins%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252800%252975059-4%26rft_id%253Dinfo%253Apmid%252F10984165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(00)75059-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10984165&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088736200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.64" data-doi="10.1111/j.1752-4571.2010.00132.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Tassel</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeHaan</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">T. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Missing domesticated plant forms: can artificial selection fill the gap?</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D3%26rft.spage%253D434%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2010.00132.x%26rft_id%253Dinfo%253Apmid%252F25567937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2010.00132.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25567937&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281311700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.65" data-doi="10.3168/jds.S0022-0302(98)75675-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawlor</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Effect of full sibs on additive breeding values under the dominance model for stature in United States Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1126</span>–<span class="cit-lpage">1135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVarona%26rft.auinit1%253DL.%26rft.volume%253D81%26rft.issue%253D4%26rft.spage%253D1126%26rft.epage%253D1135%26rft.atitle%253DEffect%2Bof%2BFull%2BSibs%2Bon%2BAdditive%2BBreeding%2BValues%2BUnder%2Bthe%2BDominance%2BModel%2Bfor%2BStature%2Bin%2BUnited%2BStates%2BHolsteins%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252898%252975675-9%26rft_id%253Dinfo%253Apmid%252F9594402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(98)75675-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9594402&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.66" data-doi="10.1038/nrg2322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heritability in the genomics era – concepts and misconceptions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.67" data-doi="10.1534/genetics.113.155176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">On the additive and dominant variance and covariance of individuals within the genomic selection scope.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155176%26rft_id%253Dinfo%253Apmid%252F24121775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.68" data-doi="10.1071/AR05152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The struggle to exploit non-additive variation.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D56%26rft.spage%253D873%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR05152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR05152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232165700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.69" data-doi="10.1186/1471-2105-15-270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakapenka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pulugurta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Runesha</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">GVCBLUP: a computer package for genomic prediction and variance component estimation of additive and dominance effects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-270%26rft_id%253Dinfo%253Apmid%252F25107495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25107495&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.70" data-doi="10.2135/cropsci2006.05.0294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wardyn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The genetic structure of a maize population: the role of dominance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWardyn%26rft.auinit1%253DB.%2BM.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D467%26rft.epage%253D474%26rft.atitle%253DThe%2BGenetic%2BStructure%2Bof%2Ba%2BMaize%2BPopulation%253A%2BThe%2BRole%2Bof%2BDominance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.05.0294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.05.0294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000245735800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egesi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawuki</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association and prediction reveals the genetic architecture of cassava mosaic disease resistance and prospects for rapid genetic improvement.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">W. W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griesman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Imputing genotypes using regularized generalized linear regression models.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D519%26rft_id%253Dinfo%253Apmid%252F25029086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25029086&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.73" data-doi="10.1007/s10681-010-0203-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zacarias</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Labuschagne</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Diallel analysis of cassava brown streak disease, yield and yield related characteristics in Mozambique.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D176%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-010-0203-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-010-0203-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.74" data-doi="10.1016/j.ajhg.2015.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakshi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinkhuyzen</span>  <span class="cit-name-given-names">A. A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dominance genetic variation contributes little to the missing heritability for human complex traits.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D96%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.01.001%26rft_id%253Dinfo%253Apmid%252F25683123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25683123&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3497.75" data-doi="10.1073/pnas.1119675109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hechter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mystery of missing heritability: Genetic interactions create phantom heritability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1119675109%26rft_id%253Dinfo%253Apmid%252F22223662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzQvMTE5MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3497.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3485" title="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3507" title="Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3497" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Rc9H_UlAGiSffQw3361T-zduX_KUfiWZyflyywKMHao" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3497" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3497" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3497" class="active" data-icon-position="" data-hide-link-title="0">Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460475" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_4K4bHG8f3NASYwyKql7gEyJETgMa9Txh_rcrPUJzqU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3497.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3497" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-c_v8gMzY09ADyJzt8AiZp2dPc3cX1HlBZ6nNi1wqfO0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3497" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3497.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460475" id="node460475--2" data-pisa="ggg;6/11/3497" data-pisa-master="ggg;g3.116.033332" data-apath="/ggg/6/11/3497.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (<em>Manihot esculenta</em>): Implications for the Prediction of Total Genetic Value and the Selection of Varieties</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5929-5785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marnin D. Wolfe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ismail Y. Rabbi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3497-3506;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033332 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460475--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marnin D. Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marnin%2BD.%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarnin%2BD.%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5929-5785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marnin D. Wolfe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wolfemd@gmail.com" class="" data-icon-position="" data-hide-link-title="0">wolfemd@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ismail Y. Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BY.%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20IY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsmail%2BY.%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ismail Y. Rabbi</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture - Agricultural Research Service (USDA-ARS), R.W. Holley Center for Agriculture and Health, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3497" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3497&amp;title=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3497&amp;title=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3497&amp;title=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3497&amp;text=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3497&amp;title=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3497&amp;t=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3497&amp;title=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3497&amp;title=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460475/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460475" id="node460475--3" data-pisa="ggg;6/11/3497" data-pisa-master="ggg;g3.116.033332" data-apath="/ggg/6/11/3497.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (<em>Manihot esculenta</em>): Implications for the Prediction of Total Genetic Value and the Selection of Varieties</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5929-5785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marnin D. Wolfe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ismail Y. Rabbi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3497-3506;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033332 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460475--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marnin D. Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marnin%2BD.%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarnin%2BD.%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5929-5785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marnin D. Wolfe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wolfemd@gmail.com" class="" data-icon-position="" data-hide-link-title="0">wolfemd@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ismail Y. Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Institute of Tropical Agriculture (IITA), Ibadan, Oyo, Nigeria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BY.%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20IY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsmail%2BY.%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ismail Y. Rabbi</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture - Agricultural Research Service (USDA-ARS), R.W. Holley Center for Agriculture and Health, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3497&amp;rft.epage=3506&amp;rft.atitle=Marker-Based%20Estimates%20Reveal%20Significant%20Nonadditive%20Effects%20in%20Clonally%20Propagated%20Cassava%20%28Manihot%20esculenta%29%3A%20Implications%20for%20the%20Prediction%20of%20Total%20Genetic%20Value%20and%20the%20Selection%20of%20Varieties&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wolfe%2C+Marnin+D.&amp;rft.au=Kulakow%2C+Peter&amp;rft.au=Rabbi%2C+Ismail+Y.&amp;rft.au=Jannink%2C+Jean-Luc"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460475/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460475/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460475/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460475/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460475/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460475/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460475/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460475/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460475/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460475/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460475/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460475/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3497" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3497.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3497.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3497.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3497.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
