<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3655" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3655.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3655.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3655.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3655.atom" />
<meta name="HW.pisa" content="ggg;6/11/3655" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="In Silico Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of de novo Genome Sequencing Studies" />
<meta name="DC.Identifier" content="10.1534/g3.116.034249" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Zhou et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The availability of genomes across the tree of life is highly biased toward vertebrates, pathogens, human disease models, and organisms with relatively small and simple genomes. Recent progress in genomics has enabled the de novo decoding of the genome of virtually any organism, greatly expanding its potential for understanding the biology and evolution of the full spectrum of biodiversity. The increasing diversity of sequencing technologies, assays, and de novo assembly algorithms have augmented the complexity of de novo genome sequencing projects in nonmodel organisms. To reduce the costs and challenges in de novo genome sequencing projects and streamline their experimental design and analysis, we developed iWGS ( in silico Whole Genome Sequencer and Analyzer), an automated pipeline for guiding the choice of appropriate sequencing strategy and assembly protocols. iWGS seamlessly integrates the four key steps of a de novo genome sequencing project: data generation (through simulation), data quality control, de novo assembly, and assembly evaluation and validation. The last three steps can also be applied to the analysis of real data. iWGS is designed to enable the user to have great flexibility in testing the range of experimental designs available for genome sequencing projects, and supports all major sequencing technologies and popular assembly tools. Three case studies illustrate how iWGS can guide the design of de novo genome sequencing projects, and evaluate the performance of a wide variety of user-specified sequencing strategies and assembly protocols on genomes of differing architectures. iWGS, along with a detailed documentation, is freely available at &lt;https://github.com/zhouxiaofan1983/iWGS&gt;." />
<meta name="DC.Contributor" content="Xiaofan Zhou" />
<meta name="DC.Contributor" content="David Peris" />
<meta name="DC.Contributor" content="Jacek Kominek" />
<meta name="DC.Contributor" content="Cletus P. Kurtzman" />
<meta name="DC.Contributor" content="Chris Todd Hittinger" />
<meta name="DC.Contributor" content="Antonis Rokas" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="In Silico Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of de novo Genome Sequencing Studies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The availability of genomes across the tree of life is highly biased toward vertebrates, pathogens, human disease models, and organisms with relatively small and simple genomes. Recent progress in genomics has enabled the &lt;i&gt;de novo&lt;/i&gt; decoding of the genome of virtually any organism, greatly expanding its potential for understanding the biology and evolution of the full spectrum of biodiversity. The increasing diversity of sequencing technologies, assays, and &lt;i&gt;de novo&lt;/i&gt; assembly algorithms have augmented the complexity of &lt;i&gt;de novo&lt;/i&gt; genome sequencing projects in nonmodel organisms. To reduce the costs and challenges in &lt;i&gt;de novo&lt;/i&gt; genome sequencing projects and streamline their experimental design and analysis, we developed iWGS (&lt;i&gt;in silico&lt;/i&gt;Whole Genome Sequencer and Analyzer), an automated pipeline for guiding the choice of appropriate sequencing strategy and assembly protocols. iWGS seamlessly integrates the four key steps of a &lt;i&gt;de novo&lt;/i&gt; genome sequencing project: data generation (through simulation), data quality control, &lt;i&gt;de novo&lt;/i&gt; assembly, and assembly evaluation and validation. The last three steps can also be applied to the analysis of real data. iWGS is designed to enable the user to have great flexibility in testing the range of experimental designs available for genome sequencing projects, and supports all major sequencing technologies and popular assembly tools. Three case studies illustrate how iWGS can guide the design of &lt;i&gt;de novo&lt;/i&gt; genome sequencing projects, and evaluate the performance of a wide variety of user-specified sequencing strategies and assembly protocols on genomes of differing architectures. iWGS, along with a detailed documentation, is freely available at https://github.com/zhouxiaofan1983/iWGS.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3655" />
<meta name="citation_id" content="6/11/3655" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3655" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3655.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3655.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3655.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.034249" />
<meta name="citation_pmid" content="27638685" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3655" />
<meta name="citation_lastpage" content="3662" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xiaofan Zhou" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2879-6317" />
<meta name="citation_author" content="David Peris" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9912-8802" />
<meta name="citation_author" content="Jacek Kominek" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1916-0122" />
<meta name="citation_author" content="Cletus P. Kurtzman" />
<meta name="citation_author_institution" content="Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, US Department of Agriculture, Peoria, Illinois 61604" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2208-9814" />
<meta name="citation_author" content="Chris Todd Hittinger" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="cthittinger@wisc.edu" />
<meta name="citation_author_email" content="antonis.rokas@vanderbilt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5088-7461" />
<meta name="citation_author" content="Antonis Rokas" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author_email" content="cthittinger@wisc.edu" />
<meta name="citation_author_email" content="antonis.rokas@vanderbilt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7248-6551" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. M. Abbas;citation_author=Q. M. Malluhi;citation_author=P. Balakrishnan;citation_title=Assessment of de novo assemblers for draft genomes: a case study with fungal genomes.;citation_pages=S10;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Adams;citation_author=S. E. Celniker;citation_author=R. A. Holt;citation_author=C. A. Evans;citation_author=J. D. Gocayne;citation_title=The Genome Sequence of Drosophila melanogaster;citation_pages=2185-2195;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731132;citation_doi=10.1126/science.287.5461.2185" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.;citation_pages=796-815;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130711;citation_doi=10.1038/35048692" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Arenas;citation_author=D. Posada;citation_title=Simulation of genome-wide evolution under heterogeneous substitution models and complex multispecies coalescent histories.;citation_volume=31;citation_year=2014;citation_pmid=24557445;citation_doi=10.1093/molbev/msu078" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Baker;citation_author=B. Wang;citation_author=N. Bellora;citation_author=D. Peris;citation_author=A. B. Hulfachor;citation_title=The genome sequence of Saccharomyces eubayanus and the domestication of Lager-Brewing yeasts.;citation_volume=32;citation_year=2015;citation_pmid=26269586;citation_doi=10.1093/molbev/msv168" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=A. Bankevich;citation_author=S. Nurk;citation_author=D. Antipov;citation_author=A. A. Gurevich;citation_author=M. Dvorkin;citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.;citation_pages=455-477;citation_volume=19;citation_year=2012;citation_issue=5;citation_pmid=22506599;citation_doi=10.1089/cmb.2012.0021" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. Berlin;citation_author=S. Koren;citation_author=C. S. Chin;citation_author=J. P. Drake;citation_author=J. M. Landolin;citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.;citation_pages=623-630;citation_volume=33;citation_year=2015;citation_pmid=26006009;citation_doi=10.1038/nbt.3238" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=K. R. Bradnam;citation_author=J. N. Fass;citation_author=A. Alexandrov;citation_author=P. Baranay;citation_author=M. Bechner;citation_title=Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species.;citation_pages=10;citation_volume=2;citation_year=2013;citation_pmid=23870653;citation_doi=10.1186/2047-217X-2-10" />
<meta name="citation_reference" content="citation_journal_title=Nucl. Acids Res.;citation_author=M. Chakraborty;citation_author=J. G. Baldwin-Brown;citation_author=A. D. Long;citation_author=J. J. Emerson;citation_title=Contiguous and accurate de novo assembly of metazoan genomes with modest long read coverage.;citation_year=2016;citation_pmid=27458204;citation_doi=10.1093/nar/gkw654" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Chapman;citation_author=I. Ho;citation_author=S. Sunkara;citation_author=S. Luo;citation_author=G. P. Schroth;citation_title=Meraculous: de novo genome assembly with short paired-end reads.;citation_pages=e23501-e23501;citation_volume=6;citation_year=2011;citation_issue=8;citation_pmid=21876754;citation_doi=10.1371/journal.pone.0023501" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Chikhi;citation_author=P. Medvedev;citation_title=Informed and automated k-mer size selection for genome assembly.;citation_volume=30;citation_year=2014;citation_pmid=23732276;citation_doi=10.1093/bioinformatics/btt310" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. S. Chin;citation_author=D. H. Alexander;citation_author=P. Marks;citation_author=A. A. Klammer;citation_author=J. Drake;citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.;citation_pages=563-569;citation_volume=10;citation_year=2013;citation_pmid=23644548;citation_doi=10.1038/nmeth.2474" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. F. Denton;citation_author=J. Lugo-Martinez;citation_author=A. E. Tucker;citation_author=D. R. Schrider;citation_author=W. C. Warren;citation_title=Extensive error in the number of genes inferred from draft genome assemblies.;citation_pages=e1003998;citation_volume=10;citation_year=2014;citation_pmid=25474019;citation_doi=10.1371/journal.pcbi.1003998" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. Earl;citation_author=K. Bradnam;citation_author=J. St John;citation_author=A. Darling;citation_author=D. Lin;citation_title=Assemblathon 1: a competitive assessment of de novo short read assembly methods.;citation_volume=21;citation_year=2011;citation_pmid=21926179;citation_doi=10.1101/gr.126599.111" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=F. Foury;citation_author=T. Roganti;citation_author=N. Lecrenier;citation_author=B. Purnelle;citation_title=The complete sequence of the mitochondrial genome of Saccharomyces cerevisiae.;citation_pages=325-331;citation_volume=440;citation_year=1998;citation_issue=3;citation_pmid=9872396;citation_doi=10.1016/S0014-5793(98)01467-7" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=K. C. Freel;citation_author=A. Friedrich;citation_author=J. Schacherer;citation_title=Mitochondrial genome evolution in yeasts: an all-encompassing view.;citation_volume=15;citation_year=2015;citation_pmid=25969454;citation_doi=10.1093/femsyr/fov023" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. J. Gardner;citation_author=N. Hall;citation_author=E. Fung;citation_author=O. White;citation_author=M. Berriman;citation_title=Genome sequence of the human malaria parasite Plasmodium falciparum.;citation_pages=498-511;citation_volume=419;citation_year=2002;citation_issue=6906;citation_pmid=12368864;citation_doi=10.1038/nature01097" />
<meta name="citation_reference" content="citation_journal_title=J Hered.;citation_title=Genome 10K: a proposal to obtain whole-genome sequence for 10,000 vertebrate species.;citation_volume=100;citation_year=2009;citation_pmid=19892720;citation_doi=10.1093/jhered/esp086" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=T. C. Glenn;citation_title=Field guide to next-generation DNA sequencers.;citation_pages=759-769;citation_volume=11;citation_year=2011;citation_pmid=21592312;citation_doi=10.1111/j.1755-0998.2011.03024.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. Maccallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. B. Goodwin;citation_author=S. B. M’Barek;citation_author=B. Dhillon;citation_author=A. H. Wittenberg;citation_author=C. F. Crane;citation_title=Finished genome of the fungal wheat pathogen Mycosphaerella graminicola reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.;citation_pages=e1002070-e1002070;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21695235;citation_doi=10.1371/journal.pgen.1002070" />
<meta name="citation_reference" content="citation_journal_title=Mycology;citation_author=I. V. Grigoriev;citation_author=D. Cullen;citation_author=S. B. Goodwin;citation_author=D. Hibbett;citation_author=T. W. Jeffries;citation_title=Fueling the future with fungal genomics.;citation_pages=192-209;citation_volume=2;citation_year=2011;citation_doi=10.1080/21501203.2011.584577" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=QUAST: quality assessment tool for genome assemblies.;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Haiminen;citation_author=D. N. Kuhn;citation_author=L. Parida;citation_author=I. Rigoutsos;citation_title=Evaluation of methods for de novo genome assembly from high-throughput sequencing reads reveals dependencies that affect the quality of the results.;citation_pages=e24182-e24182;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21915294;citation_doi=10.1371/journal.pone.0024182" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=C. T. Hittinger;citation_author=A. Rokas;citation_author=F. Y. Bai;citation_author=T. Boekhout;citation_author=P. Goncalves;citation_title=Genomics and the making of yeast biodiversity.;citation_pages=100-109;citation_volume=35;citation_year=2015;citation_pmid=26649756;citation_doi=10.1016/j.gde.2015.10.008" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Hu;citation_author=J. Yuan;citation_author=Y. Shi;citation_author=J. Lu;citation_author=B. Liu;citation_title=pIRS: profile-based Illumina pair-end reads simulator.;citation_volume=28;citation_year=2012;citation_pmid=22508794;citation_doi=10.1093/bioinformatics/bts187" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Huang;citation_author=L. Li;citation_author=J. R. Myers;citation_author=G. T. Marth;citation_title=ART: a next-generation sequencing read simulator.;citation_volume=28;citation_year=2012;citation_pmid=22199392;citation_doi=10.1093/bioinformatics/btr708" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Hunt;citation_author=T. Kikuchi;citation_author=M. Sanders;citation_author=C. Newbold;citation_author=M. Berriman;citation_title=REAPR: a universal tool for genome assembly evaluation.;citation_pages=R47;citation_volume=14;citation_year=2013;citation_pmid=23710727;citation_doi=10.1186/gb-2013-14-5-r47" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Kajitani;citation_author=K. Toshimoto;citation_author=H. Noguchi;citation_author=A. Toyoda;citation_author=Y. Ogura;citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.;citation_volume=24;citation_year=2014;citation_pmid=24755901;citation_doi=10.1101/gr.170720.113" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.;citation_author=S. Koren;citation_author=A. M. Phillippy;citation_title=One chromosome, one contig: complete microbial genomes from long-read sequencing and assembly.;citation_pages=110-120;citation_volume=23;citation_year=2015;citation_pmid=25461581;citation_doi=10.1016/j.mib.2014.11.014" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Koren;citation_author=G. P. Harhay;citation_author=T. P. Smith;citation_author=J. L. Bono;citation_author=D. M. Harhay;citation_title=Reducing assembly complexity of microbial genomes with single-molecule sequencing.;citation_pages=R101;citation_volume=14;citation_year=2013;citation_pmid=24034426;citation_doi=10.1186/gb-2013-14-9-r101" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=S. Koren;citation_author=T. J. Treangen;citation_author=C. M. Hill;citation_author=M. Pop;citation_author=A. M. Phillippy;citation_title=Automated ensemble assembly and validation of microbial genomes.;citation_pages=126;citation_volume=15;citation_year=2014;citation_pmid=24884846;citation_doi=10.1186/1471-2105-15-126" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Kumar;citation_author=S. Subramanian;citation_title=Mutation rates in mammalian genomes.;citation_volume=99;citation_year=2002;citation_pmid=11792858;citation_doi=10.1073/pnas.022629899" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=N. J. Loman;citation_author=J. Quick;citation_author=J. T. Simpson;citation_title=A complete bacterial genome assembled de novo using only nanopore sequencing data.;citation_pages=733-735;citation_volume=12;citation_year=2015;citation_pmid=26076426;citation_doi=10.1038/nmeth.3444" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Magoc;citation_author=S. Pabinger;citation_author=S. Canzar;citation_author=X. Liu;citation_author=Q. Su;citation_title=GAGE-B: an evaluation of genome assemblers for bacterial organisms.;citation_volume=29;citation_year=2013;citation_pmid=23665771;citation_doi=10.1093/bioinformatics/btt273" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Mapleson;citation_author=N. Drou;citation_author=D. Swarbreck;citation_title=RAMPART: a workflow management system for de novo genome assembly.;citation_volume=31;citation_year=2015;citation_pmid=25637556;citation_doi=10.1093/bioinformatics/btv056" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. J. McIlwain;citation_author=D. Peris;citation_author=M. Sardi;citation_author=O. V. Moskvin;citation_author=F. Zhan;citation_title=Genome sequence and analysis of a stress-tolerant, wild-derived strain of Saccharomyces cerevisiae used in biofuels research.;citation_volume=6;citation_year=2016;citation_pmid=27172212;citation_doi=10.1534/g3.116.029389" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. W. Mewes;citation_author=K. Albermann;citation_author=M. Bahr;citation_author=D. Frishman;citation_author=A. Gleissner;citation_title=Overview of the yeast genome.;citation_pages=7-65;citation_volume=387;citation_year=1997;citation_issue=6632 Suppl;citation_pmid=9169865" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. W. Myers;citation_author=G. G. Sutton;citation_author=A. L. Delcher;citation_author=I. M. Dew;citation_author=D. P. Fasulo;citation_title=A Whole-Genome Assembly of Drosophila;citation_pages=2196-2204;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731133;citation_doi=10.1126/science.287.5461.2196" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=N. Nagarajan;citation_author=M. Pop;citation_title=Sequence assembly demystified.;citation_pages=157-167;citation_volume=14;citation_year=2013;citation_pmid=23358380;citation_doi=10.1038/nrg3367" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=R. A. Ohm;citation_author=N. Feau;citation_author=B. Henrissat;citation_author=C. L. Schoch;citation_author=B. A. Horwitz;citation_title=Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi.;citation_pages=e1003037;citation_volume=8;citation_year=2012;citation_pmid=23236275;citation_doi=10.1371/journal.ppat.1003037" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Ono;citation_author=K. Asai;citation_author=M. Hamada;citation_title=PBSIM: PacBio reads simulator–toward accurate genome assembly.;citation_volume=29;citation_year=2013;citation_pmid=23129296;citation_doi=10.1093/bioinformatics/bts649" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. D. Otto;citation_author=J. C. Rayner;citation_author=U. Bohme;citation_author=A. Pain;citation_author=N. Spottiswoode;citation_title=Genome sequencing of chimpanzee malaria parasites reveals possible pathways of adaptation to human hosts.;citation_pages=4754;citation_volume=5;citation_year=2014;citation_pmid=25203297;citation_doi=10.1038/ncomms5754" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. B. Reddy;citation_author=A. D. Thomas;citation_author=D. Stamatis;citation_author=J. Bertsch;citation_author=M. Isbandi;citation_title=The Genomes OnLine Database (GOLD) v.5: a metadata management system based on a four level (meta)genome project classification.;citation_volume=43;citation_year=2015;citation_pmid=25348402;citation_doi=10.1093/nar/gku950" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=F. J. Ribeiro;citation_author=D. Przybylski;citation_author=S. Yin;citation_author=T. Sharpe;citation_author=S. Gnerre;citation_title=Finished bacterial genomes from shotgun sequence data.;citation_volume=22;citation_year=2012;citation_pmid=22829535;citation_doi=10.1101/gr.141515.112" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. E. Robinson;citation_author=K. J. Hackett;citation_author=M. Purcell-Miramontes;citation_author=S. J. Brown;citation_author=J. D. Evans;citation_title=Creating a Buzz About Insect Genomes;citation_pages=1386-1386;citation_volume=331;citation_year=2011;citation_issue=6023;citation_pmid=21415334;citation_doi=10.1126/science.331.6023.1386" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=A. Rokas;citation_author=P. Abbot;citation_title=Harnessing genomics for evolutionary insights.;citation_pages=192-200;citation_volume=24;citation_year=2009;citation_issue=4;citation_pmid=19201503;citation_doi=10.1016/j.tree.2008.11.004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Rokas;citation_author=B. L. Williams;citation_author=N. King;citation_author=S. B. Carroll;citation_title=Genome-scale approaches to resolving incongruence in molecular phylogenies.;citation_pages=798-804;citation_volume=425;citation_year=2003;citation_issue=6960;citation_pmid=14574403;citation_doi=10.1038/nature02053" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. G. Ross;citation_author=C. Russ;citation_author=M. Costello;citation_author=A. Hollinger;citation_author=N. J. Lennon;citation_title=Characterizing and measuring bias in sequence data.;citation_pages=R51;citation_volume=14;citation_year=2013;citation_pmid=23718773;citation_doi=10.1186/gb-2013-14-5-r51" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Roux;citation_author=E. Privman;citation_author=S. Moretti;citation_author=J. T. Daub;citation_author=M. Robinson-Rechavi;citation_title=Patterns of positive selection in seven ant genomes.;citation_volume=31;citation_year=2014;citation_pmid=24782441;citation_doi=10.1093/molbev/msu141" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. Salichos;citation_author=A. Rokas;citation_title=Inferring ancient divergences requires genes with strong phylogenetic signals.;citation_pages=327-331;citation_volume=497;citation_year=2013;citation_pmid=23657258;citation_doi=10.1038/nature12130" />
<meta name="citation_reference" content="citation_title=Algorithms in Bioinformatics;citation_title=Using cascading Bloom filters to improve the memory usage for de Brujin graphs;citation_author=K. Salikhov;citation_author=G. Sacomoto;citation_author=G. Kucherov;citation_pages=364-376;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. L. Salzberg;citation_author=A. M. Phillippy;citation_author=A. Zimin;citation_author=D. Puiu;citation_author=T. Magoc;citation_title=GAGE: a critical evaluation of genome assemblies and assembly algorithms.;citation_volume=22;citation_year=2012;citation_pmid=22147368;citation_doi=10.1101/gr.131383.111" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. T. Simpson;citation_author=R. Durbin;citation_title=Efficient de novo assembly of large genomes using compressed data structures.;citation_volume=22;citation_year=2012;citation_pmid=22156294;citation_doi=10.1101/gr.126953.111" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genomics Hum. Genet.;citation_author=J. T. Simpson;citation_author=M. Pop;citation_title=The theory and practice of genome sequence assembly.;citation_pages=153-172;citation_volume=16;citation_year=2015;citation_pmid=25939056;citation_doi=10.1146/annurev-genom-090314-050032" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. T. Simpson;citation_author=K. Wong;citation_author=S. D. Jackman;citation_author=J. E. Schein;citation_author=S. J. Jones;citation_title=ABySS: a parallel assembler for short read sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19251739;citation_doi=10.1101/gr.089532.108" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=L. Solieri;citation_title=Mitochondrial inheritance in budding yeasts: towards an integrated understanding.;citation_pages=521-530;citation_volume=18;citation_year=2010;citation_issue=11;citation_pmid=20832322;citation_doi=10.1016/j.tim.2010.08.001" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. M. Utturkar;citation_author=D. M. Klingeman;citation_author=M. L. Land;citation_author=C. W. Schadt;citation_author=M. J. Doktycz;citation_title=Evaluation and validation of de novo and hybrid assembly techniques to derive high-quality genome sequences.;citation_volume=30;citation_year=2014;citation_pmid=24930142;citation_doi=10.1093/bioinformatics/btu391" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=N. I. Weisenfeld;citation_author=S. Yin;citation_author=T. Sharpe;citation_author=B. Lau;citation_author=R. Hegarty;citation_title=Comprehensive variation discovery in single human genomes.;citation_pages=1350-1355;citation_volume=46;citation_year=2014;citation_pmid=25326702;citation_doi=10.1038/ng.3121" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. H. Wences;citation_author=M. C. Schatz;citation_title=Metassembler: merging and optimizing de novo genome assemblies.;citation_pages=207;citation_volume=16;citation_year=2015;citation_pmid=26403281;citation_doi=10.1186/s13059-015-0764-4" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. Ye;citation_author=Z. S. Ma;citation_author=C. H. Cannon;citation_author=M. Pop;citation_author=D. W. Yu;citation_title=Exploiting sparseness in de novo genome assembly.;citation_pages=S1;citation_volume=13;citation_year=2012;citation_pmid=22536955;citation_doi=10.1186/1471-2105-13-S4-S1" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Ye;citation_author=C. Hill;citation_author=S. Wu;citation_author=J. Ruan;citation_author=Ma. Z. S;citation_title=DBG2OLC: efficient assembly of large genomes using long erroneous reads of the third generation sequencing technologies.;citation_pages=31900;citation_volume=6;citation_year=2016;citation_pmid=27573208;citation_doi=10.1038/srep31900" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Zhang;citation_author=C. Li;citation_author=Q. Li;citation_author=B. Li;citation_author=D. M. Larkin;citation_title=Comparative genomics reveals insights into avian genome evolution and adaptation;citation_pages=1311-1320;citation_volume=346;citation_year=2014;citation_issue=6215;citation_pmid=25504712;citation_doi=10.1126/science.1251385" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=X. Zhou;citation_author=A. Rokas;citation_title=Prevention, diagnosis and treatment of high-throughput sequencing data pathologies.;citation_pages=1679-1700;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12680" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. V. Zimin;citation_author=G. Marcais;citation_author=D. Puiu;citation_author=M. Roberts;citation_author=S. L. Salzberg;citation_title=The MaSuRCA genome assembler.;citation_volume=29;citation_year=2013;citation_pmid=23990416;citation_doi=10.1093/bioinformatics/btt476" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="In Silico Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of de novo Genome Sequencing Studies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="The availability of genomes across the tree of life is highly biased toward vertebrates, pathogens, human disease models, and organisms with relatively small and simple genomes. Recent progress in genomics has enabled the de novo decoding of the genome of virtually any organism, greatly expanding its potential for understanding the biology and evolution of the full spectrum of biodiversity. The increasing diversity of sequencing technologies, assays, and de novo assembly algorithms have augmented the complexity of de novo genome sequencing projects in nonmodel organisms. To reduce the costs and challenges in de novo genome sequencing projects and streamline their experimental design and analysis, we developed iWGS ( in silico Whole Genome Sequencer and Analyzer), an automated pipeline for guiding the choice of appropriate sequencing strategy and assembly protocols. iWGS seamlessly integrates the four key steps of a de novo genome sequencing project: data generation (through simulation), data quality control, de novo assembly, and assembly evaluation and validation. The last three steps can also be applied to the analysis of real data. iWGS is designed to enable the user to have great flexibility in testing the range of experimental designs available for genome sequencing projects, and supports all major sequencing technologies and popular assembly tools. Three case studies illustrate how iWGS can guide the design of de novo genome sequencing projects, and evaluate the performance of a wide variety of user-specified sequencing strategies and assembly protocols on genomes of differing architectures. iWGS, along with a detailed documentation, is freely available at &lt;https://github.com/zhouxiaofan1983/iWGS&gt;." />
<meta name="og-title" property="og:title" content="In Silico Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of de novo Genome Sequencing Studies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3655" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The availability of genomes across the tree of life is highly biased toward vertebrates, pathogens, human disease models, and organisms with relatively small and simple genomes. Recent progress in genomics has enabled the de novo decoding of the genome of virtually any organism, greatly expanding its potential for understanding the biology and evolution of the full spectrum of biodiversity. The increasing diversity of sequencing technologies, assays, and de novo assembly algorithms have augmented the complexity of de novo genome sequencing projects in nonmodel organisms. To reduce the costs and challenges in de novo genome sequencing projects and streamline their experimental design and analysis, we developed iWGS ( in silico Whole Genome Sequencer and Analyzer), an automated pipeline for guiding the choice of appropriate sequencing strategy and assembly protocols. iWGS seamlessly integrates the four key steps of a de novo genome sequencing project: data generation (through simulation), data quality control, de novo assembly, and assembly evaluation and validation. The last three steps can also be applied to the analysis of real data. iWGS is designed to enable the user to have great flexibility in testing the range of experimental designs available for genome sequencing projects, and supports all major sequencing technologies and popular assembly tools. Three case studies illustrate how iWGS can guide the design of de novo genome sequencing projects, and evaluate the performance of a wide variety of user-specified sequencing strategies and assembly protocols on genomes of differing architectures. iWGS, along with a detailed documentation, is freely available at &lt;https://github.com/zhouxiaofan1983/iWGS&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>In Silico Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of de novo Genome Sequencing Studies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"h4vQ-bMGgolABH_TXSok6SIYSgOD41vMVPdIfUEWtH4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460464","apath":"\/ggg\/6\/11\/3655.atom","pisa":"ggg;6\/11\/3655","ac":{"\/ggg\/6\/11\/3655.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3655.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3655"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3655"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3655":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qDZcw+AXSYaP\\\\\\\/k71RW1hMf1ERiR96FKMrZuoDdUPwCJGAs59fK0ZCyPQ7\\\\\\\/MCgqtHwginnoiczK0WbjQf+GaOYpGt5ZuE3e9XyZ+pojAP6SYyM6ixAu9KaiQflRHTrf5mERImeXIJ73LjrjWeP7rKfbPgVk94ZeIrCMJ7PXuVHVK1K88Luv8Awnhg6Wsz2nlBPfkQkGqP+memQuoRVv2KZABY7rcgT8rjlF2mmxrEm5b9mKTjLd2sqqnmnxZYzHvIdDkwynOizBsuJIgAZHJCwVzizBhWPWUzg36A\\\\\\\/uyyOvFM33aZLsJcRmsuRcPjRkc3WNqRyVdvMD3ElAwt6U8KZjvWU+kgwWLLMPce0nwMFwYi\\\\\\\/ftnApcdo5d9grekd6h3r0A3UtEER3x6ru+A77SmGFUk+rM63J\\\\\\\/iVHZjplUNrpBw2aKahl2cGHmiKvvNqesg7f215bk3m6Eo1n6wgahlOYb90YqwHn2gyN9f\\\\\\\/wjqqAvIoygvQpuiJXNEs72UyToAkxwPX1hiWX0Ensr5QQTZO+alob6n2I030lUtA2fDuOqc+oxBMiNnFJ4NOvwfLZiV6hLBQ9RpB77tbrcPkXoeFPpOjr7iOUyYlZ3MMbjD6W73bKkAC4003bnygLMFCS\\\\\\\/rWOA4AVz0O9VN8c0UJSHttF4wY2QpMsURoEOoRSFK296BCz+ygB\\\\\\\/O4YprRvbZqPlL\\\\\\\/4QPRDIK2UUOOxO0ogMuRUAtKP5G3DmZG+THmpVtETlK\\\\\\\/a8rXhgzuvFMNfnr7cJypJTc9gB0u9rQ0EN4Q\\\\\\\/7q7J8Eas4fBfmeqOp4jtoer2b7\\\\\\\/Wdz01nBBFAh8ma7IcD0IXgESCOdzXXOZdioJRk3FKWLZ20IoPzZoYepradm5cTvW2Ie\\\\\\\/N3mzec1N+LxeFwsSGA\\\\\\\/CTPxy+XTSw\\\\\\\/3\\\\\\\/W2+uZVJn0NMd\\\\\\\/YIhRugaEMFXsRC6TBrfMlT+CMdiTXM7rXQdIbGzyQoZqR4L7rR6MRcXH58mbicDAL2mk\\\\\\\/3TI1DXRWIsFcW2nVc2UrXJqr+k04WFRWJFVvVRVhrNjVpSLl6ji\\\\\\\/egrFeIjwLO0cHxirPC4X6Mcr2YymG7OSUFomeqS1PZcbHfEOkghJcYbqK8Mt7LH8hOCdxD+2vOoA2oXuQbtvbPLmdZQ6A1TkUdiPLPqq7qjEkff2Gc9MLrYVJ+AU+j+XcyH2\\\\\\\/n5b9xeWRnEnqgmDyWiOpu+d4fFo60v\\\\\\\/W87k43vbh4W0Y2MDII2eVx0KBpagJYQk65\\\\\\\/lrii6DvLmfvP6zIlLmJHsbX6D35nOeBipM\\\u0022,\\\u0022iv\\\u0022:\\\u0022ulBOOZZ9zDHFX2qn9hKa1A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227f3af746e3f09b048ac63a7ed943cd7c087bca2665905bd5628443b8f1b9763c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tzCaTUwXsfPjjyndJs22XHRDBLv9oaGs2nO8lCkqlVE6cVidBONm0H2ZV7mZwneuhWzODmAmMWNVra1\\\\\\\/27PjDUhOgA5mf3bDnNlpSMxk6lZjRwfUfNqTnK6\\\\\\\/FLG67yim5FNFQjAfVkidd9NU9p32nMqOpZO6vZfcZmyfB3Dn5sxFaSp8lykyhmAbuyuDDpAr4\\\\\\\/Le\\\\\\\/eNN279QoTR3amP9i4wZKQLzcKU21E3YLp8vLDxZ52DPT\\\\\\\/8ytO8uAIlwstbH7Bh6s0SISRzUoBa+46CnXiKP46tCfLZ2+seCcj+PY83hOELncEZgUpA7aeLbV9FEyJqQwPV67YHUdv5P3BR2lI13JGk2RCNKZ6iIl7UG56yp8DxqbqoDM7LVb3wm+rGrBpvt77eKZaeE8cUDSkHqrfH14MdnZ5xZLCuIsH9ecFKlIBBlAdc4EmFZYqxxIUkKTMJ3Ru07SaRUDMlWA8YVxXS1y3Djc7IQKMrtqHFtqeDH0Jb\\\\\\\/9yFOPw\\\\\\\/YQFX\\\\\\\/yQRwzVbiWAxdYxz4OXPBXTWkd4X06X85Yo\\\\\\\/7f2OL1LLuSRuJZobLnO+hoPk3c0X\\\\\\\/Fq8bA2LiDxmJsrYh0saIKevWFOg9r\\\\\\\/s2m\\\\\\\/yHeCnTw5o1nsfgFIU2HchLT\\\\\\\/26MILXr02oR6l7tA\\\\\\\/mGh1PtGMUOAdcV9HIffhqkLxueFonetZQfU8WDwYOSwyOZGGbiPNK4KZbp3LhK+qGWWPfMB44mDOo1WhJ9Abgtht1iF3cxmm2Hvmxb9IS2bShmkf4LdssbSaF\\\\\\\/22Y0UkTa5sCnlXKZGO7X9VFcTQZLjE3taYT3T3zvlh2i0LLZveKitKwjAoA7a70yTHTfMFEtiBu7l33Euknpu7xBn\\\\\\\/fYE5DxLWTtJuN+JsWFuT6KRDIb5iDTQ\\\\\\\/nZljsVJtdlsapDHSOYOQeEk4a1cow44e8YvMmEm2MTlPHTPcpczUP+FJ9k+Qg4DzDggB67CrntGskuB4Hz\\\\\\\/jBRpfy+R+2M9b8mvXbMVK6L96+VZlj6EUre5Nxe3J0cC+H3ZRHsnr2j0bHmFvMPKcFW2bxBYmxm0OiFo1Se5l0RN7xtd9xx7\\\\\\\/n+oE1v80Ht9VNuaSFwMyEdpmEoXkyFmp2hylx12+O+UxJnmyiaGgVYvtF2AvvGFSKXA7njAtzDmWLVBi9cqHm5jxkq2sW7NYwERMLLnbwhR8tC9y+25CX4+Hj0xtknoQrIelmhTA0eeB9Bn8puR36+OtZ+R+tdkDgmmD3Q1vlz41Sd+eC0ll9oti9\\\\\\\/2R3wnfQhtfr0lQqSv79KB\\\\\\\/GaLFtYs6TLf4YEomMMLjbClqEONn86kiaK1HenvH3\\\\\\\/pjBwCbsWJCFMHAxJLKRf2M6vbIwEHHUdvZL5d3x6V6NYt\\\\\\\/TPNOVBt3Uqdwrp2W\\\\\\\/0IqLvmVScQVYe0j57huVILBvWGoYoMEbTdowxDNOhEd90UOthAzqllR5i4kHw4UkofK7Mvm8VcOZ9YWv7+Tt99OsEtEfEGisotGEFvMM7vUhxC2AO8fdzHMuanPkP0jLxPYSLmYoo9gmlq+lIcYKGi+ge7GsIkoR\\\\\\\/zUh36HOV5MZAPXYkXA7aZKL4p4jbSvhIH1hD3n2t9m3TZY9NA5swLG62nrk+G1zNVQHOv1tSWpC2mSD6qSz6wt21qxnArcM7MmaXWCeJKFXfJwfssCyWufoFSTHbk\\\\\\\/LDrzsQJSQxJhPwsDWMxfm8Ju8Fe+rfNlh1PKgDYTrxbK9aH2PyRJkuJOPt17JvBcyCWJpYsRBpll5D\\\\\\\/wsTrNQWpiikb01HHxuAadUyeSeTzfgWwywh\\\\\\\/dQ0fvD\\\\\\\/vkkIVWwYHjmBpDmubqwuTVqWVv1FIvM8X+Tr7kGU80soZJRpAb\\\\\\\/nMwv1vuZRurp1NBP44mjkwOXnOb5jHEsLsfPrXrbCJ4jyDwBt4bkHy5rLoLvwrlWRTan4qY0XyhDguyyBMl1r318GjVGALfsjeRlToj+JT3mXObv0JKVcNeI\\\\\\\/D4+ExGP1EKMPspcteS6ZdT2eHBnQqU1ESu6HIJ\\\\\\\/kOk5DotzKClmzwQQfsuM62\\\\\\\/wkoC7hP65jMeLD1dW16oo2J9ned\\\\\\\/g8yTgtbJyyGtnEY9gkrwSa91zatCU3DWjM8VRyWBfabZPi7n9nVfA9DPseBjQYyDCKUGs2IHnHYQzwblRTZ\\\\\\\/+vPjNqBZ7KXiSGqbT46aDrhpo2k08Vx9B5o5Ze4E1NnoypLiPkVNQ6YfFjqXrXkC0LLgIvawRq0CsdriEFNYKl2eoH7sn\\\\\\\/bRhsZ9mdZ+GnjXPh2qrPsEoP\\\\\\\/VhiT3Gnu\\\\\\\/wvMejzXfKnqiP+8+Y5qXkzOoqXaX\\\\\\\/L2h0ImRN3QsuIhfMmABqfxq61Ig2O0t28ZHVLCbV51L7RFkGp6s7b9laM7T+sY2p8IuDY8e\\\\\\\/tRzJjpWl\\\\\\\/UJL2t5TnyQV00AMP4oBaAvrigGvWFeQbiTAmySdNzzKiVekz0e3PdHNiKkDFgdE+3+ggW43wBGouneRD+AN7saRFTfnibclqrKgFCwA3H7SeJGmoaqjwd70Gw9xhqapvE6sh4ZEOK97OUfuUKeSnxPaAkO97yQ3+PJfhlyCPupefdd0LCCDM+\\\\\\\/tKkTGNkGUZOZF1C9aOqNqpIgOpE9tazFTsYp+Wbis3tA3ehDli6ZiKDo97V2wHAohTuD+Hi7XdnE9sWE9NATU+U522c70klzYOiOHBLASnZxGBZzh4IX659EVR9pE495JbzK7PTzOQIJe2O\\\\\\\/gs2Uz7zX9mDn0tXdNrbpr7J0us4OsOnFSaIYY2Avz7kuAtxyyx8I9EcZvHfR8uuA3RHnDD5PopgSUePOjI9YodQQDxcI7LDTK\\\\\\\/RwsFxfbS338SoWwxGbrsMxsGTMSRLrA0XNEcJBmipjZTiCR+ztEOYV68D7ApMJU4DByztFJFp27w670ZQGaiLDifXRmgrmDukqmQKRpSJo3FcsxWPZUBtKJOhryMycND6A7WOAvbynDTQ1foGUAAJhiFGQ3RQ3IEyux1fqKep6maFrVDGP2GaWfNTpJTZr5NY7v70g80bez+Yg71cnDaKvrhB0SYD4IUYBu3hzHKmZRR1e9UC\\\\\\\/y5El\\\\\\\/2M4E0FwbsEJLc8xI9LzRXLovBOnCtEL+fhc2ZNmYxI0bveavnynyZLUx3kWuVvZXdMJ663LK71eMqp9b9QWGx55tV6vgMhDgpQO\\\\\\\/oRDIxqoZ8hFdROp2VnHGsGkZVzqIjcpQel6f39s41e76mKb1Z5p1GNADOsqWU4fjEAnu7bWF3eevROZs6cBq0D0mixKYaOg8DY4KUJCx8cqAAwLbk5P6kmCCz5bDkgVNKuIHQ6Svr4aFmzE2mKuqnFs\\\\\\\/IAqWdri+qnWMdrRsj\\\\\\\/LcT\\\\\\\/bmG6vHG+4qDyvDVCpPjHOaGpfGLs1maILNijTecvRoItV0Tq++\\\\\\\/9NH\\\\\\\/CFXkAL18FLxBgsPWZiayZ4bggoQ2Mia3uuk2esAvT13Hvtj5GSHodKW0NbmM2pwK3Z068kDRAFbuubFwNI43us29iFbjsS8WzPu1N7cvhMa9+dOmPx7MSi5miKJXeE6Dif3Xi+T+LVfl\\\\\\\/MRSoLsjUmr16FW8rAavNbiB+bkZI+RFiiLExeppQx+N3EDSkbX6QFnKlEZ5n0l5oCpAB1ol87vamDD25CJayI\\\\\\\/\\\\\\\/ZLxvBWLOVSlh5qyyqmokdXEjehAEpEDUYh9C2RaXCD+sKSVqM6Ohg9rKWLA7FQcwTCaymJqtfnvRp8xB7yBiKgTDWowVJULkH4U\\\\\\\/fmVwGdx8nTu66GBSVboaFwbWabMDF5IgebTaGd92dJD8tSjy6XeVOeDtWtusVQlCu07WwTsJ8\\\\\\\/s+tgBQWqL0kd3sJitGgzkjhuFcUQFPrYdn6IkP4ZU7o4k9UktutsDUgW0udQ7kWR+Rsfd\\\\\\\/MsY+TZSd05WrkVEsKiy5nlCHfFp3dnz68aniO9v3RJzokQlr7ETF3pBGcY+\\\\\\\/7EEikAKl8fT0R1QRVWk0kzg5F3aOM3fqlSc+ge+SY6UiDh88wSxViRgwOb8vNW6QXl\\\\\\\/qNS+IPYVDk2snoAJXozWzik\\\\\\\/odWSeW5\\\\\\\/z79TfylfjM1xYRvBKnUmwr5D3KKiwMukRENeep5F90b3WeRx+MH6hDUJttEQL6oZVmVXCzJPMKjzMzIzeoyva3\\\\\\\/V+GPp16E8bLxfcyLPs0OZyYiw2U3h50OpuBcB4\\\\\\\/ne8VmmDyEL8TBw5y4fgwTyMl0AoNHrZikoYSKIRZk8g+EBOSl4F3rzVUsBS7TJNXaG+IvaRVRk6Q8hbVMbJZSYxpFz6whd\\\\\\\/O3\\\\\\\/lmaYXmKuHbHcyrs3lK8GB2SZYRlg5xedTdFydziE9UY91lgYx4ArbP7e4NLKp5JTJE6rczNX9VpYe8\\\\\\\/CSRpI9QNUTwWi+EdJ\\\\\\\/6oZIEHNOofhuZt4+KU5\\\\\\\/Mv1f+8+UGGZjAdGufy1TNlRn7mQNLdzUqJYfahLz6Kz9sfIoH8Xv805BeKuWISwvvlK13emyfRPF97ZcS8x\\\\\\\/tEuTTKSSd2R0gDonbbFca77peEPzaTvXo4uVs4K2ngo3zPVAXg8FrDD2wAm60xe8F6R7aWKq77R5\\\\\\\/9YHWroBZ8Trt9AQvuBD1vSR8wBsP9Nj5skkdl5hP4OKXxSw9QvQydGOVnGFhNk6uY+DYLxmcB8D2jZpKTz2n1Y6R\\\\\\\/DbKUtdQgJaYwueTTUHAitQKX+z\\\\\\\/ze7xfRswqhJ\\\\\\\/Sw6\\\\\\\/k+XxnxpOqylW+bLiQgOPlIQuV0RLHN2aQEGjwkNeKGbay29Ft7CjSL\\\\\\\/eVl2hpZtjep\\\\\\\/TP6JBNLez4RBlEldYYviUdGnNY9T\\\\\\\/1rjVXtCoPLMVePvird6TJVs+wY7Bv17SL67p6LVOux0MPW+gHD\\\\\\\/mWLlr2OgQJ6Y\\\\\\\/vvxGSP27SJq9wW1OBUmyhD4FhsccLq7jx72EjUj\\\\\\\/maJZrDHpi1oywBS+lKaZ1GAv8XyxkhiGXcbDoDS5m9pcMxh35pJAOERu8WIiBInCeX9DofmlavknO+nBioA8W9mk5xR9y+A5CZFDCz+vUvjUGvIxnrmgetYnn5WkbL3i0RiBS7N7HcLoxexax3EjPhzGzo4bH3AfPHXe+BUL3VfnVWdDBhPj085dmfK347HXr82QPtPqVVgnWWnugeJygRCv7lkqZcMrczE3JvBdyOV62CqXrWHApULirSn0d5\\\\\\\/yEQHRfMB7vI1hIxKQbvFeYpvxZpMHMKbCaO7ORpI6gHQIEsKO5RZDatu3BEXqkmSCrN7R9\\\\\\\/qJI+N+o19UIG0lc0CfhGufY5IcdD+1tKp00cZ20PsKuBVSGA2G1PLd+URpOyqegxbWMMJg7uCc0+xsEcfrvn1snZc7Yxycr90bSeG8jbbT\\\\\\\/\\\\\\\/fpWOONV5HCntIKgYdGl9OVxQ\\\\\\\/LGTDoAsZkhAYrXaT+OecGG\\\\\\\/\\\\\\\/vsEWsYyctgnRaweaf+xBzeRbwEoY\\\\\\\/XYTJE9IdfnWvyvGaSH77avAUhWioiEtRL9nw4VSWEhjNOET\\\\\\\/tJFPF4w7jaiID0yk1\\\\\\\/ofLRN\\\\\\\/WsX5FujFQoCOpwpN8P8xlMMeoofUuiakehppjlM2wgurb6uyQdLSThQUpxUQXCAzh0NdstqypstqWOFwsH1UDHZ805hkwr+e8Gnoax4zlVb3DcRJ42zTUEfMbONFNb3TFoVnYoq5hvrxtI1upo0neodl8qkFXITfNALApaiY5JyQf2i28f6J0uGxiNVnAqCO2ZzRtFIs6Xup92USGhISloPOPTP4d45wrYL\\\\\\\/FQhKwTmCTVA192iLkeDxZa33PbFxM3vj\\\\\\\/Y7X5xiyzB52CTq\\\\\\\/nYXsqXc1A0am1zyWHcmvoXvZQQvSDxvQIBqcPspvqbL8rxQnSeniQgJ8YC1JPvi5RL3jUpfM5ieAZGgG2NfX8TknXqTxGrn7SfR0LBuv7PU2cUzLvZQxe017qtzOyVXCKYB3pkgHPPSwIsEwybIkLvij\\\\\\\/Cwc4ZoaOKSMzDgjRwyHPt4QjilI3i0PLwZCWW+vAbQpdtD\\\\\\\/Fsv2j9mFKiRk+JaeAHRdL5diIV+whlRHQbJb2lhBK0VsiRb7Y99KTvh\\\\\\\/FqVTso2+u\\\\\\\/XYinma3MDUFazlejVxl2WG\\\\\\\/DxKpDlCVVHJWNZBiqbPtQtBQL\\\\\\\/nUVIfHs5CBdAiC8FysljkuWKDZSoR7tqV8F6dk00+3gvMer3CqWAHkRjsUnfbTdO3y395M+kF731pSa3T6Nl5D8K3aXFMNENCI5MSv1SE3Gvkmlol9DsNfhU3zeLeTBYGLIZoZeF2aeATh\\\\\\\/M+FK+SwtQrP4HLUPUzo5hVVKN0EMcIiMl1rs9tGSO\\\\\\\/o0BA9AFWP\\\\\\\/Iv5kSvpZpOmaXFou4Uk7O2pMkT\\\\\\\/lrsGH3rMK6HA4NfPHIltgQ8vhF2Sq5YSfJkKKT+kPE9M8aNcG8VB6Vu5R3uAdIHo7ovBPFgHB9rqDcp5RQqZvEtYt3lO1+jnmZsQQSbXNmQIiKXmtf70A04w1LFGLE3jul32mt8+UNx1s95SJXSJeU2QgSJK1ucOnK6rOOjXzS1VhFd\\\\\\\/FjwK0U1Atz4PcoNzxpNSBHP8Vh33FtfFZM9BohhtlaySR3a4V27ZxXRQHl8Io8Cw2ihPFrQjm2cOfpy4iRspUo3xRg\\\\\\\/hcY4KI3JU6uWZuvCsJJ+VcGwbz817fsMYxIk7pbHx2JIL2AAZ49h018OBTOek9T3Cef7P2P+4D1yVptlxGP0ofaptJtCgOdMk4PItwVikBnFX7QkPMb\\\\\\\/0D5C80JFwj3owHD+NHFhwAITIB35S\\\\\\\/cWk+H\\\\\\\/kugsYbLQdko1a24qo6ZpOrHJZAOC7oenBmOO6LPA+c1aoSGGlT5HoBwoHpBDLO3L3no9BwrU1ePsdQ\\\\\\\/AJUPpByrqt7M3tel01\\\\\\\/YSeOoqQvUlCtQbVGwQIGTcLn3Qquyj4HTRsoAwhwUrcudrBECUGErz4SiLWgBMBqPb+UEijJ7LXajZ3qD\\\\\\\/0e5gG0lc5AOA785YPDsiMx9ZKdzvpXBW9oVJXzcA3J6I3oTr25hmUNDPyowaDUW47ttujd9C\\\\\\\/95e+58AJD6HocOYu8gE6UdxNPSMSaAcEjEqrm+tioX634C\\\\\\\/eYhBZv1WWqr8noyVMOrdHZZPrIaATOISsGq6zvLQb4uSrkPx7xWvFbeaeqKD+0Kvp0d4JagJ1UExecCz4UTcOpqOX24cFgFfmyc7tUNIRdL+jUEbmbo77pVoZCBd\\\\\\\/d74Tb1SIUvaq+amXT0bEn5wQiXRNPgXFiT6YyQZeuI3t86QM6\\\\\\\/44b84Ujl3t8DzlIYJy5IaAs\\\\\\\/FEqKyfKMJmATsZXKgJcm8TU0KF5\\\\\\\/7WgBv95OlohJZdPsr4S2l2GA6XwG+3gSSSEcV9IVrkymCACexVT+TdCEynTU0+lrvQ9ZVKlpdKKWORfvZcodFfr6ifwaO+VpCLXaCGQh2kxCnVFL5Czznfc1DfuuaZnb\\\\\\\/rRgP\\\\\\\/HJlaWnLo+v4JJLavMOq1sMImE42pviToz1iEFEd\\\\\\\/69WISiACxOOWxwVqxNE+BZRHY0ZfV6KWVq4I0wNk1HEjXSaCqIytGmCU3B9aBkLhoBP05n3xBA2\\\\\\\/yNZdLC0rYiuQiSZwhagosMp+6rsC9UMWzbjqKlMbAXUK7GAcXza\\\\\\\/X6+9VYFMWw99Xu3c\\\\\\\/rA71jsLiNODTjvRTAlga4PJYIR7lyDPbAeEWSPhjUwcWK1Z\\\\\\\/6SXzQeHQGOvWRIdo9STBZtCoxNUF0kJSMbOk0Vq+QeJ\\\\\\\/xrEnL6ICnLEutM7AmlzR1QIEHAjmt0tM402mTyNtLV6v1juVevZcCXvG5CKnjNiGUD6exTcAdoY\\\\\\\/rg6lnTlw8yxcBe6VIabIf6outuDJMQmiWigD0KOSf++ypAX1Z75JqEBT7\\\\\\\/dzNkcWyr0T1kQfM+GcAcXqMFey\\\\\\\/YR8KLG95dOwHl1dP5A6ckcgqIf9030v8dkwLb5i2hrP0kDVOi\\\\\\\/07AcvocC5X7bojPrWpK2Lorv4eVCC\\\\\\\/tuCAwyTIjyoCQFCNACSibPWkGktCVdxcCooLQQxvbBUl5eJ77zaKOVz5E9BIwtsiMHN6rjHqFUDulQoE9vcQtrfQLprkLAYAOLX3HmHGxgZ35AfnR2j6tjgBeSAmkZ3Qx4rPvh5VXf18pwG1emLnuW9hTpvcKvrs1MVrgfzty87LSo+z0qUbr\\\\\\\/FAvXubWHiN\\\\\\\/1H9M9WAxwOxvuO9cYUEgkcgTZUoGi8Clwtr+3\\\\\\\/wFld\\\\\\\/nhuqV\\\\\\\/ICwZKlb2onPEyPPNvB8CvCFQv0MtnLQG+fKxIay3QMoH4CWfVy6PaTlng5\\\\\\\/rtzF2KvZIUnrfyDKzkBgJhzLL84cBCeWCiYWNxBozBqWzJw9FdulYHHKceMEtZQEcso\\\\\\\/tNXP7OTMk0D2b5iNOnxCqjV\\\\\\\/lLWixWwCrC6UQUanZ3p1psUJ3\\\\\\\/SeeYEjW4c\\\\\\\/7nAecefT0vE+5la6t+K9eF37YNJ3Re5hZmOQnIV9UyY4ksVIL4HxEpA6EsU2qmVK9XlnXXraR6M7gVQoIVXjjl47wwNIxDwVr4ySgLrb\\\\\\\/igp0upmjcG+QcpZhDAJ4QXz0LU0ie9c0EAF\\\\\\\/UdTBy6qxhVmEL13B0bJrXT58FuhEXuQ9nqATLo2jtDEnGo4uD6D2ArA2R3rmcHOvtHU\\\\\\\/fWqbs9yxPYJMKp\\\\\\\/D1vjk4fg74gl77Y4i7a9MgF29N0Kxx175LJnHK76ZqzafQjys8LWY9X0ikeb+kdBFjaPbbhXGlYd\\\\\\\/+PW\\\\\\\/hhUY4Zj1t3+tWCKdeGY6lTu1vFsDEnfDoDGjBkHR1IzoO8eh0p5VXIWeBSp1LDmOV7gyBVK4qVX7xugMAl5N4+0IFd0kaElPPKdEPsR4ufaCo9bz32LvPJdnxXUQRJ7XtGiQmMEPVp4ZobRbEb0QTQQvx+3oA92xmwRmgv7gGNe+LvrXDER8\\\\\\\/dhM9n0O4ppVKH+BtnVyn3bS04Yn9EdX0qtm0uBrGqyPAUBFAlvoo97Cq6aryxfRb5hesev7BObDhkVpepMy\\\\\\\/dS5ib+gfWqLaVBglcjDABN+3zz205Jx8Qwan+DEQMTL2doAJ7uWLANqMUVVlOZv7tw6un4LvgI1UfUfqiaaoxWWR\\\\\\\/XkFLy1Du6\\\\\\\/jkOwBDrwwMl9BsLOdvvTsEX92nAWkjWxGx1lKGzWTGwlPC6gO529CYJwsp9eu8dKX\\\\\\\/kUBpzA8j6TNcFMY8uaV8mjZPMdIisFgQrC5t\\\\\\\/tBiBBSqMv3joDKGxIeJgR5EZnXsvSA7y8eVlt0rLvPmInbhrqfLp\\\\\\\/ySx\\\\\\\/Vf\\\\\\\/YksDn8c1GpFDK2xEKR9kiyhxrwMAkmi8lfBTk1TeN0APPvH\\\\\\\/ap1PZYD5Okpey286vXbtMQiad9KivX45JCsHpcMtKy2lTaYm4rhxGKlTeZkeVDcSP03iNpVdV\\\\\\\/Uynami+aRmxrnJ4ZMeC+5aUAe3jiPsLsfo\\\\\\\/NuTkfozBkDlQlPXRuRbQjBBzg4RE0hsNVSSLUj+y9\\\\\\\/yVrhA3zhX2XY4AzAN0s+qcg4Tq\\\\\\\/n\\\\\\\/RaTp+IuCfydZU9uinVlNP8knCLYAyxat9FNNDk9XOWg535Ts14YBiJILjvVhago7A55cTdIqFzqNKg0Mv3e0cVxIXmXoCoTYgw9l9bHGruy6CyaeeGUgpZzE\\\\\\\/pmDtfiaMUNtTipYzPhWvccIa34\\\\\\\/\\\\\\\/faFclBL+eGWvgwmBOVKQ6VyV2kRRypqqxCUaEc\\\\\\\/hGm1jIcpahTUUp7PKFwZ5n87mVB2zMIouY3wF93PUAX9EGraRrte3FvEuyum6Y\\\\\\\/q4N3NYYen\\\\\\\/mrJyr\\\\\\\/S13ycVKosaAHIuQTxbE0ZgvkELgm8HtTNxTifyyoRBCTxi4DKRxn7iHE6V2yvTroSWx1WzDY3a\\\\\\\/oCNZMOxzRffGPdsuqDyI4B4xP9nVkVuo2W8NKMIPQTDmUnXal2OR2uC3m+gStK8Q+DrNAED8IbqGlclWqQbzWXShY\\\\\\\/iPkpbcU6ClAbz5AiJyRNceBTFzWoLpGs\\\\\\\/B0vdwvxiTZblkU18Sqlj5ma00UK1Mmimz0fycN3KoZOxQgf0RbxjIDYFAkogpWIm4QNFjoDhQuR5JtBACZwbYD2MYIPJkXkKMEy3A8gSUwReSvD2Eu\\\\\\\/Z1ui3AkyVDiPub5FlGn\\\\\\\/TyOUoJWFHGx4kMC93+z7kpSvNLyT6r+swkpMqq0QtpQKV5psnjvAySwI5HtHu7RPglEVNcYLMN4mmuXES6Of17poN\\\\\\\/oo3Ov+CHg+KuS96Ai2nJhZGdTkmtPoaxodlGly9iNEig7yNBumH\\\\\\\/O1lfq2sYQ0\\\\\\\/A4PLZh0TougYwcL44K4v0rjvPj6VRMTF3rTOQ6xVcdjK2jmp88uKj5NL7EobUPihMOQSyjYJfayR4IKVjlLVITbY3ItifKQxHYin\\\\\\\/8yX9DzzNJ7A\\\\\\\/fSNzq1M4HrMghwTckVlpCv4ceVP4dCaX8gfMxgygAPGLyebuv5FJsuXPxLtJNgNZqX526MZ+NHi9EgdoMzVUT5uuXAaUBtZ7b9Nc+rQ9iGm7VS8owVQ2TAArvJT8TK4RZywPkVtkU4USwp4MD9jk2UWK1gmFuLnsAmk8eGptAnhDYFXP3FkbVVm\\\\\\\/zH05BkuVyc8QwL2UfHfl2apKGRapr4+8l5mXz54iPB3\\\\\\\/wF\\\\\\\/rMQo\\\\\\\/YrhJoyX\\\\\\\/PetcfYkH0g2ZBpLqyBYfe3DZet5sKenfqjqsBXT3mI9st5ME7DRZwPe5iaSIb+zFny0r9XUpou8ZDmcsJMlPa6wkcTbZT+ysQbf4e8Mz1axUyjP40\\\\\\\/p17p003vzVL757XU1KbgHDQkEvabv3s6iJODsH0cDtTe7Hzx3c4puPP5Yu1w6jnjgOS8PkYL\\\\\\\/Dm7Is9IyfSq64NaMW8eftwsT+Yiip8ShxTmTGhXWHaiDm6R9TiQq+MOJfjEu4HR00aqubxTRdQtzMtkqKOkzNKgpM8dBAup5XnIYdMYO7dSqVOpwyS7l5p9mQNiBz\\\\\\\/mRZGpRVNscZDSxGIfZH19N3+omXULL0N\\\\\\\/JqQDp3XdWcu0+\\\\\\\/n2tnZamnCQ+Ac7YPEDA8UNM6KAYxmOJ1ve88JTGfop8AlcSXYYgviq5AbKoVxkCquz5ndi+cOMbRkX3LyT2ld\\\\\\\/3lxXI5EHXP0XwBP5wCkMkRmlMZwQ+ymXnqeroidOqU\\\\\\\/JuOAsPPjcpPiBhZFHOJm+tK2vng2qrcQW2M3ECLRyK7bXBryTGC7SlqvzoggSqWBUv+xxza1tfKNu5waHqy\\\\\\\/AcrvSKznkrnCfEZDyqitjerBOxTX+D\\\\\\\/bseMGhi8pvw1KYjH4k9ZqfH9GSEzNTz+Ez6pffwEl1QvV7YRAOWXOzhy1VMh55snc4W\\\\\\\/bZZ94YRd3q\\\\\\\/Tn79Bf0qc2G4wiC7ARe77dhC91ihEeCMcbnBSaXlIfWv4pMhHgpuCLYYF1Vf\\\\\\\/6xtZxF7gqVRj3cGUa8Pv9KS3UdTVMIxnMe7873N5NTCbi3r+S+8RO5Y28\\\\\\\/DR6WXlLTqBqtRfclwqV3cgVtuD+AjnA1oCEyA7CP\\\\\\\/wXiORxaiVcTIoEUO\\\\\\\/kT4f+5WoYa5d+c\\\\\\\/xvRxvC8+wj\\\\\\\/Qc+t1kvOpwuy6SUawSHTV44dZycqjsEK+Igl8yD0LbkeNK4RCi17ijTJDXLlhc7Ux3FZIwTpBw3DZ4F+5QixC6ltbuQQ27QAsjcDPYsv2VWCEp1ht6is6rQzyrXdL+t\\\\\\\/6vqNZtVs7Llvb6i+91Qondy2Wn+\\\\\\\/8zj2R+jFMA1KQSyj5HT44iw1ZWvlWBwFWe3sfjhFY5kYkYumebJizPDhBrPcb8US1qk3JxV5PPqffCuEI\\\\\\\/QIRo9CGMaoSu2dGWcexDyl4ag3181+2ESuH3M2DfL3GnXORCY2vPQR2wHLqaGRekhOTlIn1seKRrXdQNB0VBG2eEEJT9k\\\\\\\/dkTrj\\\\\\\/oFTHmbz\\\\\\\/HsrDk\\\\\\\/jJIdxxTw+rN2tpTNB9r0+\\\\\\\/LytuX4+1v2rqaFD3\\\\\\\/9\\\\\\\/pcVIWBrEVzsqVq0hZI1DW65iqicdTXoUXt7zL6UMZmxSRI9yFi4ulsYx+b+PDIlNdvGNL\\\\\\\/Q65TZ2jkt8DiVFPGh3Ui4v3GTGDwvi2z0iBwuTYYg8G8k9dJ\\\\\\\/qrUPpioPS4eSvgFkjlJUyt1aaVXo6N\\\\\\\/11h1o7LAnKB+fvkvKwHKYAzmMOBu2zJyL5IfpIf5rlYASjZHvr5AZ+2YwwJuKQdbHM3rujf1NbQcHTbdZmZ3yMiheCwi3br5Z33PT7sNUemYiMr852cmrzEHrif9KrmJ7WD4SDhwS3MKIW7J4Y6T5Jih+cncypyt1BmxI2MrJJWKYLU4haYlKlcXEubmuI\\\\\\\/fWZevFPq6gAA+KsvQAUKNHfx8DhZ9ZDMBTn1WmLedc9R+4EUYQHecGZ52MyeJBXr9UTa3OFa0rFMXtPv4Js0PPTz\\\\\\\/aRWJuecAwAPuiOk\\\\\\\/d29T8F8gcg9SjsRdcQZkWf5ZevvdraOA2G99PX+K9lLp9+wIcyTVFTdjmJsfDhfGatwlwBqeObQLZAoCX5+FxfWoXuGD229AODeclWi8vKRvlzCjTmb6EfGiqp9678DMy3gWIUIQsAsqE+xovdfS+1Qe5IiijnwGNsRi0ix6JQe14D5QJTF0Bsu8ie\\\\\\\/MpGDfd6mPFYcBFaO+NFNR4tp+iYe0RYsbxjbG\\\\\\\/eeDepEZ6wSX1okL3glH8eh3\\\\\\\/XB9z+iieZD4miW7WsUjCpHAyJXcFdc3ujfmnK3Ap5lx31Qh7h+W9XhEgvI52W\\\\\\\/BKxRFGQ9aGMBVFOP4hQcx1VN67N5NOus2HORkaAzPopSr3EWD9MTEU03oOq8P2tQpFfj\\\\\\\/7j20lUxZFYanl7tOH1AGQSkm3\\\\\\\/2hsj8Y340wmfk8woD\\\\\\\/odhfce\\\\\\\/YFpVufgxpzKAlgH0tg953wiEHoK6tic7I3PWjkwVYrOzKoXbQP3r3Tk8t35cil6Y1+vPouvjC7IHrxzhR\\\\\\\/b9Y80sQ\\\\\\\/NIe+Hey1kFLrNm3zri5+uhR1qBs66uBzWZry7r89tDTFl3MOHSD6isrCRZr+RAfv18lyxrJxV80RjO92rvsXxxWOoZzCRD98NVRfvKnLjW3KuJwHsPmzqlaImvM\\\\\\\/ZwXLNXF38CZ2DFmAFurYn1nqSZ1h61FiQDkOeG+yDvvmtDe4StifQeld3BULEWyZ16xa39ZcbTOQUQJ+2iNSLm0uuH8yVZnbXLEeHq4lOG5YPFSr7ST+Q7n5EH6MFQOjNu3hN1pEGfs6K80Xk60mM36HfTI\\\\\\\/BfOHfVaueOAUKUiu+s2\\\\\\\/otBL9dOLIPUOsMNWGxKQVWGrFRK\\\\\\\/Atbbmg1x4gCNtT4zqWfqspB+G6\\\\\\\/KpGUyhNl0YA9zfVdixkxtFdJlMmo3VOfd\\\\\\\/zUZIIA9WZsbNxDWwIhAIYazIH\\\\\\\/3w44S\\\\\\\/ySTHdehMvtmYhNx67Yuj5Pv22E1BiyQ0P5IcuJggKuo83j+RpPP9KI8djoxTiAQP6bj8LnElP2afb8xTdPVPSsJOnAZjjjlbIndeT5bvHvotwCXucVaKYWq8+zw5lWzNd9nHmdo6+rIWziTHvs6NDvO6lCyXipO5pwDRhhc3UBTcAg6\\\\\\\/A\\\\\\\/7YuHgz2kIAtw5qD\\\\\\\/IrrHXgwezing3g4XJSRDOPn+t0DDbY8v7fXDvDuXIhvnmI+asHaP9Tkl+3eR2UHC96E1561QfEN3I1EwfLwAI6CBVUZiy9jEl6BKDM8zBDkJU9KTY2rvYiEoICIDjdRfqTY+A8lcuDCR1+893g902aGHD018LWRUaUhXAEl+eLMuyZu4R9OTGwhfINQYEk46vdtgdYlJd83I+sBNIQeP56UTiEYisb7f0e31OZ+A3saT8tqGBRR7RIldvAuVsUG0vFqPL4VZVd+lAHNmxt+7HqtfqAl2YSTX6KO7zBjY+3J8olJmchCRRyfLZG1uFJmHOTnrTBpOuEFmDDZFUagCTaULw8WmTID50GDMJC0ZnbuuK9qm45wEg\\\\\\\/21h0AL7Wv0ph7IgTGoQP75kBIQj\\\\\\\/zfKMfPpXvpxsKOFkRu4YpnccBilZSQzFH0RwNIHQT8LGmfHrY03FgONNLSzigdXMupfE+d0IF2LEDo5QPmIJhHj\\\\\\\/OiFARGF1ttu4lI\\\\\\\/WolIWUH\\\\\\\/MwW0rYTDGcoOf8lG3DtNX5tllQXlfomrEqvktVZO65Hdk1HaOxy8iiIKsVW+tjPiz83oeXJ9Bn+G59bYxb7FyjQZiJ5u\\\\\\\/GKJBCS\\\\\\\/0o+7nCp4\\\\\\\/EWUT3Qmzv1j7B4GxphFVb0wd3cOM8WitnJCFfXkSF8MLYGntPhTtyrMu7j9R5Ez4znhFK8zQ6Vk+6p8jZcZKJ0wjSIVKZbv2n5PzDrHi8BYsmrSipcCuhN1iAKgiMO0gh90Qro6eFeKWPFAsjEv\\\\\\\/yZo+NrfPUVEIx7QdPeQGhgiG3GhelMPA1iYBQYEFsb7T2Aq+XkMkZ3DBsPjmP97h34EBpO9qRdO4YWJZSarhnO\\\\\\\/pWGiZmoqVPVWNVZ1nQjytA0u4kNg++Rw1ouIAyPFQMH1pbWgFpJC8fN7nWYqh4z39TjOm949YxUMjz08u17OZWO\\\\\\\/3ojaGe\\\\\\\/vLwQfDlSh9LXrJuV1uG06+K8mgq+1S5P1YQYlcPL\\\\\\\/dFwPJm4pCs5ptoPd510SWYsdFhamH9P7GJC28N8YS5oyswO8EsmTHrDAZe2tSXqGViWnQliyets\\\\\\\/LdUuxra4RO6JuauZmKFGcP5fYpu8b5FSw19djBHIxWyTNFMoPXqO\\\\\\\/ZpiibXUqqQM+n9e6oHzYYiW41hnZlOBWmFaqQSUV2wpHNxCnO4NLol2HayIaE1vwFHZBZFgGyW6QB2zVr1h10d+yFuw5Hvfl6yMif43hbFqRrzsmIYbIBPIPVNKiLNrOThMDYo5swF6rLPZlg1vnbsFww3v07wAeG0\\\\\\\/BTUNyGgbZmyF2LYyLS\\\\\\\/v6oyN9rvbe4GMy7B+Dtf+dXADat9m3037I6VUhboLdbKpJhGvOsl2lgXeY6qpazlCNcenzs\\\\\\\/B1sFQfcFn5ALOp03Jx+gOP0jc+B05bP2Yy+9oW7kINPsHPvtmasZdq\\\\\\\/lkrojW1cwqcIsDhZfOY1cuScjlNb4VEpSb57TPO\\\\\\\/8yioc4A1RIntREr0TxWxCzJmcWr\\\\\\\/nvfFGX\\\\\\\/jKq7bPqNjCN1+5vXUEoSqeQy3kfDtah7XIStQkXelo70y2srHL3S2sf6jBSrffeMl3B3KWtHgAzo5oFduq5r5XFBw79lJz4yC7xz07I+HNQUVVPJxSb+FECmzwTeqMdMncmJ1e5FgcFVL8kso0EpxMhJtHE8FpB++LqrevPy5mqYTsg9eu5yWqZQ9j4y9QrCeyGwbXzBZHrnVAiAnKTCVr1gK15Cz2LCVVLlWFULrTtcmcuAN+Fokg3ZYNJFCJgA5fTCCsCbxwzOyyBnsWL+K1Dks2gHPfctYHecAzRXtFD8WlX7\\\\\\\/GBJfAwEbSDlTjv17K0qMMjepW4oruEKSM\\\\\\\/qpLJO4qxEtIRQ2eEZbLihFFOJ1961Ksh3ANhvWj4d77ECnAgfjS+RluvJamnGBCfmovLJ0EaYG18k588QAdh9OwGlOSAo006j8Qba4Mc469ZSfGKv1bxPU1y6lBmLGwNVdLxX+Q3IOKkO4NGSR99Mf\\\\\\\/O12yID1MJ\\\\\\\/gPHEcgFtC\\\\\\\/GsLIgFYMEgBvh8OQr05kDdPvxeofgqycxkRbf6l0aDNZgh\\\\\\\/CIFdJtDoDIJv\\\\\\\/jZBnaa1ocAhvcKdlnhnUx3kQ6gEEEFRLc159OnPuPFiHsEbrO8MjfKJmxMeioFH5+DHLXOcb8WpA+ucCUyMmswQTWrqLv5Au0fblpoNj2X1V2hmDFyfeCmYN9dVk7KS8humtG2TbmBCzCUkzCGlFnTwIzqfxiXq+Xx4FPTs\\\\\\\/M7aK+sfDtb5uGY82D6Lm85ARfirUHUjn6cfmjSs3BToPDao0nFEG9TC8aJR6kIerFEwKjqPiRpSeI6uh4ijfkorAce92u1G8cm\\\\\\\/5ZGGI17ao7McDNWUj3BSEEoh880tvGy4smOdPHJkb2kuHqiUowTOIRn2NVKMtEKVR5W1x9WU1XFMddq9EOpzOchP4GrjU9\\\\\\\/g+GmC6+GE2hCEX2JYBnwB2oOpd3YqJs0OuAbLbOq0stbZcux1GFx1zbEO80ZqiFiqdz\\\\\\\/KW6OeZiWnUKmMkIr5I27\\\\\\\/ANFIqBeBB+YERtCE6tqtOqD2Y4Mn64i33nahy2Kfi+n8vlEfkcFhmQNAKh3+4H9CbnXvsYg\\\\\\\/GjMfO7xpciiqbkRTEIlvkP+CrdBVTOWGH6ZZOVsXsTCDAa5nc0PGf0niZeceqEkIfVy43mBxwR2jFGydGwvW7HlLiv6e8kak1DZBELW\\\\\\\/0xdmiYzLqQleDR88nxzgcEibkZj2CgycljNdVDY+ckB4fZehCwNs8RQKsXQRFywWUMH+b8UGbEBloh7gGjWb9CPTSzmGNTXOsv7wXUIBgmpws7CxlUALAeo+0+xZ9qBgrvOeq6fqnInLU\\\\\\\/uTFVoVFFhTzzZNFC43mZeT+2alRk64xNBC5ru04lt329uVCRzOf8WGN3s+L9POtYjlCzBomVM+SUUoIMRIohfhO3nziuUTg2f5taGqdz0w9aC9A9JTiZ22i4Fw4ihOLJTx1qSpE1OLJCe6E0Eexu21cgHvNzBz+7KyQ3b3J5a5Shp92z9UdLjRqkBtPWXI6SRyCfJxKGrnLnPsZFgJH6XdH9hJs61690jiNSO9cozXnqKP5AM2l9\\\\\\\/z6KJoFISG\\\\\\\/r4uli1soIXEuiSzqgqdgBTgWwIaB447GVwluP9RoSFydrHbpOAtHfejg9+8N93H2Xr7IGdR9L1A09gNE9rMFcSerBwdPqoZX8hEvSKMuCMaXZkjiZWaik7eDQHsV3EcBu5xqOCePW6QO\\\\\\\/teGUYAQp8lhI7ZYdKm3sPyVKW88mpegCI+dEgOje85kAGAErCs2TPwZMzD5cO16DHQnNS1TBggIVEIF4JTXNgfToT8db2R\\\\\\\/3gmN2kq4XPAA2l+vMyqwXPD4GakP\\\\\\\/vmcus7eW29aItrRz7+w0y5F30jqxQfzrQAoUtzfEdIoUDsk7mYMoCY9FjnGFirkzZleWloVxZ5S3Csa3h6\\\\\\\/ETYytSgp5AKhvrS+40oen8Uf3ICCILJnP4erM\\\\\\\/nQD\\\\\\\/fjXO2c0KRgWxoM1yXaL6jVPAz89I2o0ojAVy5+8ImhyGmmcKYbQSYE5Rd+cRALDqJShUEtYW8ep7C18jT2XxsIKM3hAhUV4fjX919b1Lgn+S1KocnaYiWqnD3ONm9wtBPOckb1PUA3QznxJ8zKq02xHUvZsmtyTk4xCuDr0YZMYaF3oRnkfJJ3ntta2mPSSB2ZucGIpX\\\\\\\/1nTYxFdIMK7CwNU3CP\\\\\\\/BC1FKsKq67xxND8D3dm2oo4Q7Nmbz3XkCWnLcuRtW40P84jnRFJaz+mOgc3C6jmPss3vDvWCZcH73p6izpf0rg3BWQXXzygRM5vbYnCQqvxg5dbbg+t\\\\\\\/J40iXLgnoDJQAX443uNCz2g6MCi1P7i+IDxHlaZMJDGWSORHdvlhShzUfbtxApY4Y4xZlOuqVklt2ArPyi1NVaNFb1efxKnfLs5abB6\\\\\\\/9KRDSYw5DkPCizQv2IrAO90yYWp1q0AFomVxu+9laoPrIZjsx+uRwIN+UxErGBmSJPy0JxRokpWjlMdTRoEHLAhRs1AM8QaCkJ62XW0SI3GCOD5Ujabhpj5R4NazYxDiEMSFd+nz5nUIMBygGVjAA0mCirG4Mo0t4JOnVyp7VXiL8RLr1aQ5vHayw+JPYvEEhSfXLPcu9LFJFuJH5fNWd5s2kh0g740E4iQdQoQfeicZ2\\\\\\\/2ZrXqcNM9RATQTmGTGxo6aKNHI3amAc+rIzwpaHgqaRGLTh37lNwJK364adQbq5HVQMlcAF5L6QqkuC9VI35sVGN2ZvcAIYIKKR0aIs4rWbQaP8lg+uxgM1IOPS0c4inlDXeNwLHuZKz+VcpZkY6gvxsO6fb8l2X0RGSC1sCm5KynXaUFIZ03QQi\\\\\\\/+8cpdwWpy6DNMz6PhtHeCwPZzOopuVTccjnIHVG1dK+4vRgGAh+g\\\\\\\/qGA3hc++ZGfFgwsffNBtqHFs+Hnnl38JdTwcPNSxJM1S33bOokwq7EayhDDOMvCDH\\\\\\\/c4d73MBpK3niXWC6U3HXDxsrlsSl1t\\\\\\\/j8LTOJIxs7ncMetmNqtjbqnw+SJ0670OtSH\\\\\\\/PbGkxY9QVL90+eOzLY\\\\\\\/8vqtS9z18bIfSWONyfFa7BzF5H1uBO2svhxzsjOSBewdUNUD7LxbgX5QyOp2rxlmOuGLWe0t2vbdqaxlv5x6PX6J8C2MJZLv\\\\\\\/Ddv4NGW8wpLMJjStReQE1oBdsFSz+vz00zFDjV2xQTOb9IkLsUK9HXyFoSEzyxJgHfyKLKT8nVQyvS+UY5NmlBd89coq9tRudDVhaFTw4U+HeA7tJsbj2FoTUOWF53MpUcsQdOgVA3MM+f0m6+K4gAeTR5ixeY5QLNpv8D63ujWNPVw2mNWDO31p4g9e4WPaRB+bOpj2IXL6aRSHpYk09o+ix8PdO6pCKNAI3uxBvlL2BmOF6RIUj7QUneXjEiGU\\\\\\\/d3FpKaGW+2kU9NM64p5RiiNo\\\\\\\/RAZgwDYyhoolixFcwJdYK55vZGzCgSLyUBs1nZdeSzmEYM3bh8DGV7D1yMAqsNwEADAKd9UzNTyQWuUxJaE3Vpz47IKBW7omVqBbuB\\\\\\\/KzUNsw9taH9GxtR4R9cQ0Q5eX97dSO6fS1hzce10bR3mEVL3X5LhxPn8n8RCxMAzw+gfeRK8zE3nKylVY\\\\\\\/BzK6ffry1dPCeHOL2nw\\\\\\\/LqtMjynt1er+Q7BcAXAEAk1Ova9RX1sQJy7\\\\\\\/BI0luMc6oJP\\\\\\\/nQOS0C2NSoh2CAXg4eE0ShAv2+amNXbbsVqXPt2AGdoBlPSUgUpl6Zsln5NXxA4zKmiuVJDG1rf4UgRpuXnnbbPFlEpvs440Fj1rk1cRuXi\\\\\\\/JO3\\\\\\\/gqp9rwpSVmPhgcKZevOiff+EstFafMryKCYyrelaGXXNLfxC3XEfwbODXsHjEhGIb1YZSrpHrJU4oNcX0AXfmj0hp9Lea4\\\\\\\/2lW4Wg0qYM528YVFqPFW6ugc7YsFW9mcjxwS3jxf\\\\\\\/jnrwfDi\\\\\\\/zUHqpnxUlksbgy0gkX4w8FRN6YrN\\\\\\\/h2ashouxSKSbnLU2m5UaRvgFSlqMbFOIzERy6caemtv00DNF27277GrjU2qDUEem1Fe2P\\\\\\\/lufCVapRQ5fWLOi5YlXiovVER40QPrfeSW6prPNgZEHhFe0m3UDAK\\\\\\\/BjwSvh1tlz76SN+nKJFVN7PJuRCMHFV6+3cJ9Lg+199DFg7d7twcafGv7LvgMIpGRSoqXN47ExCy3qcxv+o0v8zJvaQ2W9xNUP3tjQjP1ludKrBsrL9Q1yZgUX5NMQ1z6wv6rMlrGTH9mBAs4dzGo2NMO1H4G0e+OzZxnAarlhCwsPy6W4WeepnVj\\\\\\\/tgIdwxwmwdUGdhq0j\\\\\\\/UxYBsX6OSUHwoCdJlYJaapUiKT3g6T3fuPuE0vLOLfYkvqvM5FTgDa\\\\\\\/lKVHg34E\\\\\\\/HwwSOB2GquwzY6V7Gk7neVJNPmGjvTkzc3wKSpJ9kuHzYSW+bYhU04S8K2IigVRFvTH6NWrHtUEKrdDlyBfpgyLGuSnZ6+iEy5w7eFbvOnYsBwhi4AowccCtK0cMGuRFu0RkDOeympLkDOfeGkhXS80kBo5VL91cCtz5K6skZKweyK6Vy3opRNfmXWCDoqy109SxSLAzw1XQmaAlZ3XAWO+vq4RRSJu0IbtSbW4AuvnyvvNbk4q9VaTsm0S2FvtJdnLRR\\\\\\\/X4JmXa5nXF4+oYAVxgHF\\\\\\\/WnmyR\\\\\\\/U2dNoPVSXoC868l9Thg9Xu7hCRj99gf8QiyMUhe9sI4sdPbsTknLMkqyf0zDPKTPbn8gOEF93mVM\\\\\\\/ogdOo7Mn7QX\\\\\\\/TCbHEGSuwxclJS0+2PoOIUrHlA7P6\\\\\\\/2wGGJVCJTkwe9vsDAf1V9KeIvN8oxgMC5osPcOXt\\\\\\\/ckYRD9nZyAb1BwooRdIPBr7hrHRwjVheL89SarmL\\\\\\\/SlIwNpeeoXspnaGk7r\\\\\\\/Khfxgwd8scEO59KG9owT1hZ+CSPpG7Ub1Mw+pHYHIifzIZovfKfJlgClyF+NxBUPLQdiuLKq4kJ1Sy8cqSoYbXclu2lFNt\\\\\\\/WHxMwcasl2fUR\\\\\\\/4eoEAtL7MLqYlfYIJS7Za4Kiom2BnjJnr+21xRJ7MJeyOT1btE2BRo5RMop1UxjOTW8IQiEw5ZEQIZ+UeVXtPiSGheemlihvFCbcRGxG79iVa\\\\\\\/S26ZO4C7vNHDPy+VG2q9ZCJcaaGtzAQlCpzRwh6zLhwAS5\\\\\\\/ZsxcCKJlGdU73FIVOeU4itrkWcm6iyEnEqPlGls0nlOYB\\\\\\\/514BNufl6IBs\\\\\\\/ATWUoclbS907dWqzviJuB1UxT19bGch8BCSCsfHthBK\\\\\\\/9uZOnB0g4P0aboNLKFS3f0KvugflZs3\\\\\\\/LbJ+hWsKCzfu6IjDoiBiFtx9YowntqCTpX1pyZDvX96gkl+jXCAX1hnLmjLhw+lYp0Yk1g6Vn+dJaff1V9QQuVYF8lQdSGG9PYmi65Q1o\\\\\\\/8b20VuzWqtDYOHlM8dnbjEvwxfR2TGROh38wkQxPy4NncOutSyZA37q9muwxWT\\\\\\\/h8zdZgdRePzSWLWUk0o1LkLQfjW0CkcC\\\\\\\/6eE0W9A72iK2ZPIlpox0092JC24G8gyLs9QuCrm\\\\\\\/MihgT9xsgMIiG+lTM7haiOgwXEO1FyOAfpAcOnkJ0pYxvzfkF7zvT+YirdRhMRr8IDrPRCF2P5xG0b2UvCB9J\\\\\\\/Eont5XGEiKEmghGdVdZPxQsseSp\\\\\\\/p0xLFkitMGn+Z\\\\\\\/FAskbGbseBQW8tgC8YPeGYtlFiSIVobFcRpgQRIQulQ+wRbHXQx8IMN6e3fDAiDrdiuKPE\\\\\\\/EgD4k0yfCTgYWcP\\\\\\\/8\\\\\\\/7tc\\\\\\\/eMc3mLbN7one2wN7AgRZFvBL2O\\\\\\\/gC1sNsu39qgaGLkDN\\\\\\\/5Y9xxYiIUXrV1QLXCUcUs8qdnjwMMtzJtn\\\\\\\/GxdAoPzOJkc5phkJTzDZByl1K3tMGLpMSI\\\\\\\/UosKipuLOlV+pKQ2NTB++WR7\\\\\\\/4KXQxGQ+mbb1GAcGsmv3a9MwKuyazNXv3C2qMxRdCjn5Pan+zKFSpApFMz+OYl5qtCoBs0jaFn9\\\\\\\/Raq1wcgWRmoTToGCdzb9GraASJ\\\\\\\/gK4HwrQRECtSL0pGdk2cL9W51TWOStRcVzQNHphtGe7dDhJRhsCnbtdIUV61vuCHATg\\\\\\\/r7qmKnxK2i1xTLmDI8f4tHgeRxgHEaqxxklNo1xjqT6FScqPP6sU3T4HjbKdk4ktS\\\\\\\/w0KY1IjLz0lsvH1UNis23EDthHHkZF\\\\\\\/EaG1g2otjuCP8ocx7o0aR4dxQRAbnuHj2l3UIlOoqdArcq9rRerRMB5mdm4+qFcUJHvDo8fiUdrZbEYz4USwz52y5MvnUx8dP42Tj2rhtnV5TebPK2ICEe5AEw1HcPJol5odRxtuYjr04uT4AU0tGacvJVWjkwmq0Sy5p0xMBUPxd9uytaSd5xk8qUBKrHPLvaHUEXrmyx12dQ+ukI+ncAx7dHVXCnX67samvvmtSf09bQACofsGw3z0M\\\\\\\/tjncBvmxcuo0NJ3sRECCEMxip1XSrYpRq6sF4v7L46VK6h0naKXhneo5IGVwR6Zmp203e8CvLmZktUz9b5rNOCYQ2khro2TR6Aeg\\\\\\\/rgHzVlxwsmPe3zx\\\\\\\/OAfv5jFIX4eDFGhbVcdE22khvHeP6bB2Uz2VsgFPpQHxEyJlBeNUy2rxKmFgKLmXIGrhw+Bhj05XWYR9n3nstCe857p+gkPv5SAPpNLxb7eMOi\\\\\\\/DONh8vjUZtyEQ8+ciCqt8vuDXvCR7dLyeUf2HXmTFCAZRjBfBVyYqyirOY8vXN1AB1JtiBmXCyqlIphJANqnO7ID3gOl8g4JEfQV94S1kT8\\\\\\\/Lc\\\\\\\/XIinJaiwvHqjM\\\\\\\/vtof9dKSc9j6+xwMcKs6EmT9Po5XrVbKTiF5xljVvoHyfm1+3E+I50A2Q8Q0BkPpusyrjqVLKIvqpnzQBkYO9\\\\\\\/DCrAyopWsNValtZJoKkMXIRmDyFlCkgnTpfFuTUDhUaR4t9idLSHAd36vyP0TGz7XxUZtUw4Ojlz7CzOCyGXATx1sYqVyMjKYm\\\\\\\/P9VXBOTGSF25Xuuz0\\\\\\\/lAca7NTbaBGsgr0jO0L5EiUrZtMcFVgqDWvcSzESHPfQCo5jNQcM0v960U2v5U8mGf43yNvxbi9oG+LI+GQETSYjUFLHa246LwJPOrmIUiAMyHyI61JvolnkqDMA7zFdF+KJyg9YQzauvB6ecKzK9LedQ+rJUzrZznZxgUEnOIHx6mxE29qlKAPTtn0Er7iTjdBPmnwMKvdwt0xmhkBDbqI2mouD7WwNTLmBBkci7E6lVnpLLRhhuuusj4Gkp3CBK5NFzIK\\\\\\\/AVlaZTVlbulQMzDezWqfO+W6\\\\\\\/ZRg5rqgAH1eXcIjocw9yWmMPvzAtR3y6T2OkY\\\\\\\/ynU7kLCQnQ9pTibw2jOVz7JE48\\\\\\\/hyAuCe3Hpih+Zxlaqn\\\\\\\/roWIVKN3UPI6LBl0URFOEUADveO+Fjy39HcHedyERWdClFHnyubvtnAHPFGij8GIf5WUmzvFToEeFVfu6Ly6hGs1EKWabuzDsGDL3x7hHs2O4KyJaQ7bBGoJWdFHcQMEi+nUwjwuJHLuwhUG9oatazeaA\\\\\\\/bc0qTg0VtSpdvcxDQjMi\\\\\\\/\\\\\\\/Hg\\\\\\\/zx2Wh9tu7U5iFQysGMcqc5CNkUzkL4K2kkTcnQVMAKqt7iaLPh1p0Ya9\\\\\\\/GTZ50W7rrqFKz0SLI0ANVc1xV1FCzQ06BDkY1+WQe2XhP0eNLOa2KcBnJ3Qjfw0cu60\\\\\\\/tyqHx5p6oc85mXfyVDdvQRFXxD9u5hA0VeXfo\\\\\\\/O3YFBxj3maSuKqmQUWk5+a4vsbrIM7r5ISNsjnXwxMxy9zx1yJzfMlQqtZ+eCM2vGvZ\\\\\\\/n7jkhfT00Xz3sMOmXuZY2+gy+LzDMZfWnTzDJY3n2+VOUVCV1rGYgXydf2upQeIcxUbDdqnpSRC0ZmDC1q3MImwHkStldBuU5l2aBMizmnx7rreb00eELkKY47JtTYzcSpLlo7cn96MxgfNGa\\\\\\\/C9p\\\\\\\/xzs0j2N+vNdqBkc\\\\\\\/yx\\\\\\\/KFLMbzcpUB9ciJY\\\\\\\/qpt6KV4kOO8TE8j1XjUX7qmkSeUOQ2pGnMsAxyzbe4TFYGa0Nd4LIJOZ0eRj\\\\\\\/TFVaCsEicUmz1LgICGGPs8Ui1hwALD1b+z3qzfxw6hUD\\\\\\\/KtAqGcUfQmA0W0kscBxOMLF6qLlHL3ybxPzDS1GHsCN8bUI8t5Vu3dafsqhNDmR6axj0uFfn8QsOoYL\\\\\\\/2wwSuMpgAxfJpPHra+ZPiCRLWxadWR9PAA+nA7\\\\\\\/n6S7flZwBZt\\\\\\\/YDHvvB7tK0rfRPM1ZPE+hC9Me6JiAzYZCeOnx\\\\\\\/txrxUafeD265cGr\\\\\\\/lqkzffBJU0toxQTbznv0FyHF37nGJfE4P3E5b0Y1zbN+QlWdVmxCkfk179PjkmP1+bySyYpG5QYcUUK20fggPK2nNkn7QsPZxd2L3eBNEwymQvkvXxVOoxF9c5sHFWF3zGao8wvtBIzcjPtx1GYK+lk25+6hV+l8bb1QGillLKT7Wr4ov5vhgAza0p+m4teecI3iGUURii+g3KAStMr2WW4ggkebK9G\\\\\\\/HbT9HzseRFT1fVb0pAbI6VypLa9HNR2EHLZDY2KTi9NN5XzQg7YSPyLIo1vq7NPCVt8mNcP4muFowGkTnVrVe3lc8ha\\\\\\\/kvVXOZxMW2mAIwCteM84BuLO\\\\\\\/UEeLkf5asf+cPt+pU4HFogX\\\\\\\/8w\\\\\\\/39urjo6craE\\\\\\\/pCJvpDylnlCtN4L50nkuy3jDHZIoHereRPAhLNBqWM6SurFdi12cNk\\\\\\\/ZYKJerqeUyBlnXNA2tjQCvwMrPhQDYIRMTmEmQLyR3eE9DZfhYLfWS0C9zh5l4SW9\\\\\\\/FZE2PlR59ah2lCOoq\\\\\\\/UCvx4CLuIY5HxM9sdOylKdPGHdhHYdcN81cByMTDnlJUmYPLmCH8wpq8YZHFTisA2Oa9NQjf+0Wzg9tk5BBA6n8+Q1TVHjqbUp\\\\\\\/jFnLLF2l5jUPH3T5qYWOFGLASEHaI4yNZzvfIB2u32GUCP8BFzNLh2GJojaiZlvbqq1KRvR8SIix3RiIPajuajo6Oe+cQ6xvlZyuE43Mtb70xPesBCtrLy2vldB40jSXYx+DS\\\\\\\/6VnYI9Sz0raVgD87tpUjuNbsO5yAStCLAp\\\\\\\/qpknTu+0x2WH\\\\\\\/Et+\\\\\\\/EmsbEw\\\\\\\/K2On9tVNnP9JqRW5OTt7ckjRyYBBY60UOI2i2Yy5uQP0oeL+PGxpWIPTPfMCMgeSweBj6Ia77hdXIhqYx1nfnxT2qQx\\\\\\\/FSCkpDvnT2SR\\\\\\\/TU1smzVzvF3nafV49GqDF5hRifoDf\\\\\\\/MdSK7LXlphQz8Jlyy5XpLqHsjZMBZ+CtioEwlhyDB8o3ME\\\\\\\/xI\\\\\\\/3IPb72Cr6ziRLQZn2GuBpxs8m9TBNLHCeJ5zZRUKquL2350hWP9JGUplF0TnI628+2TZNEQ1vzTJlWu9d6LJ09B\\\\\\\/Vz2Q7Yjc7hzZscA0kgsE8zQZV6pxvnxngDnz16zlG7II9W3J1JS42vvkHamfROZRRexUwYed22jPxpnOL3hB7rlFVgVZFeakjuMSADtf1xbg7d2tJpZbHxuJgPi\\\\\\\/+iyxtrvcGjHg5G33nNtLScLmvN6wu0ZZZo95b2BIjOr1CTiX1P\\\\\\\/AW3A+jbQyVn4D7DXAknp9YmF\\\\\\\/0NL0ZMpnZQXJt\\\\\\\/sQLLUAAsJda2\\\\\\\/XibNXrmponu5px55B\\\\\\\/RLbsxl02BmMW7uW4X2fs9\\\\\\\/2\\\\\\\/tPy40nxJOQZzNEaF6n23xq4pzLg6DtByGc6MU+dxgJj5K2+5TOpDN1\\\\\\\/NDgBuekhJfWpOvJXlUvsqgf8Yx6k1heCannUtP2X1hzTYNd4q4RtMPVaiQC2DCzF7IJXEekwVr0ceXfGv1h5389TpvQvQT+WOBtOH2MT8HsmSytAaPrcTx5WCytdXIR2byuijCVcUPXutJZgZlf5IuoC1ADAfX0ehZogd20Mxhk6YXN46WhxMH8qPFYbR0H4WMJD9C7xzo19BkMf8k2XfiLx5ZUnsd2RV99yEyTgc9cL2a13PEB7eYTYEmAEtip9iUzvAFsaKIIpWtw7LdMgAj\\\\\\\/iMRkUJ1dzeu+2uUsI45RyuHTqW0EPCD9NebXGb07+J034dIWwKa0BjeibOl2aW3kuQaignBrXUc2HvKgB\\\\\\\/MZXa1WPmnkKsNeLcZDmjvhzHIM1NSFJaDh60Bj3PWLX\\\\\\\/9aInfnv2dZ9i0s4PwaER0WRjO2ytMsY0tQpx8dtP+kkQIjiTWDplI1LGr5V933HSmh2KuC5hiJAuhadjgrSCG9tzFXyHv+k+MiL17AKJhWm2e+Ig7vUW5i3ANtxEAc5nIcdCBq1odb7hogySc7NKGDK37Mi5ZVovSQ7O6FK2c2HBt5wZgXjNjIH1IH4SKeQQKrqkbJW4vnUyr+lJJv25QNQeN3qua6n0+kzPO6HZSVvMhcD7jOS6QXgib0wtwfCBH7\\\\\\\/87Xe8g9WPetlZh031pZMFq3Fs9Efuj6WuSYR\\\\\\\/dpJjARitQecjF5TS4Irbd1jtsNXokef1Hx+XNEecSdjxjbnu5MtoHMeX9k7\\\\\\\/O86tc0Hxe+o9M8dPoYB7nlV7aEE4rqhAwQU2bWE3DhfuAATvYsEStJOf+pNLviVmQblW6fvgu3jBU9PsOsUE5CzBCkBBMuBM66UEDSOdJKc4Dl7adcXqrw56r1Bxyt4Y8\\\\\\\/qsw6XzxsPke+f+44d\\\\\\\/GpDumacG9ilpL8q3AFQS3\\\\\\\/zb8bDjmE85AAQNyhtZxXhojP52wShIsWL+pHrS5ksOiWrBcT7zjqEz0JoZtKSt48JHrPcbTzum48joSyuqXrxCHaFuj9udon43SnokS6AZm3YkXujWn0+wizboIOEBeENqP5XInfD+8zKjy+cZWbneS0ollUNYo0trq6ahDXvRarXlpbDGpJl6J5\\\\\\\/Yh8c0\\\\\\\/eLKOejaV+HHQZpUC58WkiWZyVKfeapBnQ+wKeGMT9wT6lkQ5Ys62LI69qnAMaXx\\\\\\\/EJTYioGvrtRrNLlyGzvZ5ofa7CqTNU\\\\\\\/NOjnEfZthpGwxqbuUwlD+cjbUU9+EXrDqCHYXQ9DHyJ43ZmCZ1cYd+xVaSj0bKKRCLEdKNUUGO7J0nD2Y4Cu4iLgyI9Hc5h3nzoup3sgf83BtnWz+tZ8ECPvf\\\\\\\/jqRSMlnFm9uPWQbVTb650SI6htHNYR9d+RFkgwxZl5WbclcXPIgPsI9+500FkO\\\\\\\/S1ZXaxD\\\\\\\/g+zwHcXX+C7pT2WvqR5eOcxWYYR\\\\\\\/Jd4exHTNbqoXiNuGsfOHv8OtSTKjRf4eujoP4KKL0tHmtgKqCd1AquL6xulNg22KwLQmg6gKsm4fsRv1JJOYi74OxG4UjkRJG3UAgBnH5mI8P2rcEC7VtA+dMwShSzcUNXClL5ERBuJgnM+P0IfQn\\\\\\\/\\\\\\\/bCFk3NfbOADTjq5kQp3CvNar1VUmQ5Q0WmOrXyt+8IKfb0nEKKnT\\\\\\\/4LUS4yfpKcZMFm4z\\\\\\\/SVFZAO\\\\\\\/xkbTD23eXKdXM24\\\\\\\/KM2ClF+3YjMO\\\\\\\/kJwpNQmXaB\\\\\\\/LBTNZyMLXj8xoHdpt4J+cwrNu+Xg7LycX34bUK5S4OcOeTKXdkdgtM\\\\\\\/Pug5TOvy1OYgK8C13XwcuFxe2VlNy+63g35hfAY+o1YuOfRyHyCPM9bHbxonSTuBOFP+YgGc0UD6JJYaYr3uBEGYjkAuxpwVFA+nJS\\\\\\\/k\\\\\\\/CwlJXOb7rd\\\\\\\/A5OpDM93DxkT\\\\\\\/YsmiR7UEAUWWMGvvnly05WW7bI6iONtrdev41UoJI9CqD0C2upPNznyNdWoaSBKQhE2Xi74Lsq2I5eK5owbjpL5vyRJkU6udn2eAu1nFSQ9xW+QuFo4BEgU9y0E485yFBVkQbvUvrYKwl9YWOQcIskZygOQHGtaWdh0624b0AMykIR1mL4vqD1\\\\\\\/kDDWQAJ6\\\\\\\/xvxBoVP+ypkMHRanrzKqAMd1aEoQA5ZfjvGPJVWuqGDu\\\\\\\/0yAFqBa\\\\\\\/Q+h38ITJGUERVNjQzgw3JDt0Ke5icbbvlP3XeuU54zfzLHFBtlidCGfyap5wdwe+2M1pVb405KUNNIjEfBCgwEJfmKKXJ1u0fOXr81kAxGxfGVvPy0ciqFeLTVYcszBDtlreyIeOpPoJO2xF8gvocAH4Vde067sCkFPxx1xDLYjXzpSuuyE+1\\\\\\\/ZKetYfhXCCeJza4pBXzM6zRb1gHiczD38iuTAGWiTWG3X3ufIEnVoWeyG5yy3FR70eL+NYVZW\\\\\\\/lY+oYNMtU2o+850Mb49Kt7O3I+Tv+2+leywvaR06VWCeR3KXGRyPgLHfqOIzejg6T4LyE84muFy5YY4l4iXwmqUREKBLpTMSNYBHYwanLvbh\\\\\\\/3JCe5oIVbciadQfXe6giYO\\\\\\\/phaJcy5DE5Gc8QDvNCIFeIYwzs1Ou7LW8M2FbTpyLdVPnIuCgxsmLCQ8LQ7xHMXziGkMsA8LYGzx4LccElW6R5do3GUArZUbg5Vy4gW5v9STZlGSSZs2hDTOnSIPN78xUoW6ZJ8aW7G7ROxqGSjjtl0Rq40YCD+Quxbuk5ox41Ev6sC4FmMBfzGLp5bp21DpRWFdY2CWFjPBNYdT2N+NO6WR2A7Va64MVboE2qmfXt5Os7bvgO5hEVvS4apYIXGg2985b3uyWbYaJ5z3hsGVaBjMDzOV795S58tQWLt6seTfGAR3S8IXyhiPhPzNkLY1nEAXRm7JwWqYHWayIk14t5EBQ4lUV57OfTDMAna0rkBnu81qx4XENExUK6xO61Q1AWC9tKhJYZfUcq0th7xKwYMa\\\\\\\/nvcJCGPsIUso3hqGsAL4qAzAJuRFaFIU8jB7hVNu90HeR7CPbklwT3N83ACGQ8I4kqonuvti2FRGis66vjH9PSQxU7zc1zigjCAQh3+jxea4CxtIzq0o9bG69sPL8Tyd06q5gVqG7hdP9945i3J6Pf7yWwFby8b7+GhS6\\\\\\\/eFYjea5vjWLuF\\\\\\\/IO9RXPVqr+B8tAuKNJqTVx4K2QiAjACDtncUYeZItSNsO98+8TEx\\\\\\\/+O7jsWqYR8zbCaE4ezWA1y7GgR9pDzYbmfGMpJBCJPNtXeA3Ol5M+hHgwrMvT4fFOre5A2fZwPiWL+cLQiwHRxNRl2aja542UJOjx9zWG7NO0x+WuS9fX9wtwbIBmJavPo2+gCOLCTvpHjyyWcFPK925zcjKwl7zzxGSoTRzTeSQ3385MwcRJUxkM4I1s2jwyU8rC8rNAPJKkZ0LetkSQhtYe+6NO5INYUSh0HZSy7ttDWg3ABn1AVS3hyBDHA2EShlw\\\\\\\/b\\\\\\\/EFc3sqoSWs\\\\\\\/6JrgQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022iSk4iaSca6iz5C3wrmqc9w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b7c1cd7d363447c7b1a271cebec981c15816fabc471e33363771737a7f16525f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Os\\\\\\\/LLJRTlLxSBunCO2+2UPE0Fv3leax5pKQVASASU2wJzTbbsAd2kHZLysabAhe\\\\\\\/pYXDQaezFS\\\\\\\/jnb0knoMRZlxsF0MFsnmso+KYldvUa5PmygTei480Z0lnGBx0bAeoOirthjqcXJs8gtT3xmPwnvDVdmvPag+zjdVgbNTAgbT265OwlDyQVmWbcex7f9hlSIuZp66iCMIqEi2DSKrg9EAjjr1biZ25L9+md2fxKfrCce6nLGrKo3qwduU6nvT6TgiAC8+1d3tpzbf8uGmDK\\\\\\\/ZtBcCHe\\\\\\\/IzDniNsJo1Ypu5Wlw\\\\\\\/k94fhiNachKQl696QzTyhKf7dPfYB61WHEEJD+vguIJGOJdZjCZ+5MhRijARR5pdBcxqInowDaudFft4GJsCpObo513CK6nRWbEIrWdf\\\\\\\/xlZ+9E9Dqz8Dj69bAYJbcknukOehsIV6NXsnVMdR6FC2LC+GYxKiPNt2+ab7NBu9E4lOkpAuO2rF3XE4MTgjKmlSVWwm4qPi50SrqZQMchtDYut+nqJ04tJQ+dZ5YGVzf9NhP\\\\\\\/dIvULdEDEwdqS08NGZZAVTmQZuGo0ehwPZNUAmttuKZPhV9J8RVNTmm4qLk8ObKrN\\\\\\\/EBKNuYoL9WngfqbtykAA8ORo5HnlMWwxaPYcYPNQKnKzufCc4iqA0cpJDot6cteuq\\\\\\\/\\\\\\\/ns6PcIX2ZgxBIV+aPYyT8drRoe4snJbMrwc5l9wDUO5lBOl5+PZd0QOrgkeu4yXbNuBXYmkzu+xfr4MntOfqeaWZKmrEGe0BcZhiry2gfP3VB1Ebu8fDH29xsqs4rKunSkuyIAKmaAlzdMVsbH5H60CnFcGYDbezUNVDtTArr+\\\\\\\/l2UBtXfLrHGclZZhJcZL\\\\\\\/WvSZAz1nQF+M+zPtsjC6joRzBwrVtK\\\\\\\/x1HIQXhIhepHQ2Mxz2jnndJDakkZzADGaA7NDcgBcQk3FfpA0SR1ayQsyXTAFSPPL+ChWHR4wIboOzYog\\\\\\\/pg58aPGK5ijz4RjA4g+t4vK+ZS1SIu5hOgNnsvLRAXvYLW28PwOMxtkrs9O\\\\\\\/VOvbxBzLmugAdSUvGSm+jH0ldKz8\\\\\\\/PPRmjEy0bsdXuO3IIG\\\\\\\/BDQpO3XWBenmc2ITzH0DvYnfO+IKKqU68kgmOF0IeVpRiMWPhA+dJ\\\\\\\/Mtai3qZ6KOP6rpxzOthtBXDk\\\\\\\/kIKpgulDuLslRaywcl7hhIi\\\\\\\/4KpfhMVfAbhSGcbdAMLTR1\\\\\\\/IQ8WHIu+UYnB1sEYiCU5Yc1\\\\\\\/MiyCfd1sSNWPKpS106qXaj67HpyXXkgX820OIszqwVZA+Z\\\\\\\/Nukmchvz4wva3Usmrdixgx7dBh8\\\\\\\/aAEQfHwQah\\\\\\\/M60bRQ+4N12Fb\\\\\\\/66zyESfSs6ah8aWlSGDhrD1kt4385acD\\\\\\\/is4yakOj8dPTZfeY4xRmqLJvLKTlZiFdz69wv\\\\\\\/0JeNiLab0LgW8ofeRcPcA2uMDXeZgEeCN049zfSizItr3G+H9q4m5NXNm9yPeEWpce\\\\\\\/R+Wc0+VqXHa6P0HX33lm1l6A81BakRQFqqSTPrlgYWhiMdGqZwWDFPEXGF\\\\\\\/Q0tKvTFNjkuzAN+zLIZD1OLdtyOjV246tCchLPnb4P9w8N\\\\\\\/Ymqb1Eia0Tq51CqIVMThDIz7PTFmjm1N4u1owWyLzfUvhgmL5tBmzDj5euzC18Cc\\\\\\\/5mGm\\\\\\\/HvIgAxmvQ2nHKyUv3Ddf2YE5ZbmwsQsLI7iYbMS+a0fT4VlwkRJ\\\\\\\/AvbnY\\\\\\\/QuxM98DP8LEm\\\\\\\/kuFFIDQlamiBffjFCkW\\\\\\\/pd4O9N3IOR5ycpD7PX9IDzD3ENSM50M6e2PpGPd6cFnndh7ro3Nt5TAsFamhfnpEAxBRX61LFgZLgflF6BoCmQ62smoaCqbmZAGNYV0y9A08h6wOgqkSIbgRDM8NWLQgadHbjaQvsIs3Vc2jM84TkMGbz9sVkA5omLnRjU2Y2jfRrWHkib5hGNLIaUaIl96Q+9Cf1b+4Cv48FuxsdbkmZwCcwrioZ3i4dzbyby6IAx+\\\\\\\/wU42O2YCXs9W2UZbEglSSLrliA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qj9ra9yvjwdLgmndunjcbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c03da9fd63c9efc14c829e2402c138956e27a53774c90c9b149dead71277fb47\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460464 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3655" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2VGM7xbPek374PJpGJ6vHGQi2Cbk4U9BqOac26LsbnM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3655" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W8LL0uAV8DZVSuKla4ZI5vBUeIdfNlGQezgcigITq4U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3647" title="Simple and Efficient Targeting of Multiple Genes Through CRISPR-Cas9 in Physcomitrella patens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3663" title="Determinants in the LIN-12/Notch Intracellular Domain That Govern Its Activity and Stability During Caenorhabditis elegans Vulval Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460464" id="node460464" data-pisa="ggg;6/11/3655" data-pisa-master="ggg;g3.116.034249" data-apath="/ggg/6/11/3655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>In Silico</em> Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of <em>de novo</em> Genome Sequencing Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiaofan Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9912-8802" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Peris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacek Kominek</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2208-9814" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cletus P. Kurtzman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chris Todd Hittinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3655-3662;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034249 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460464" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiaofan Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaofan%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaofan%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiaofan Zhou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David Peris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BPeris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peris%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BPeris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9912-8802" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Peris</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacek Kominek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacek%2BKominek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kominek%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacek%2BKominek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacek Kominek</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cletus P. Kurtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, US Department of Agriculture, Peoria, Illinois 61604</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cletus%2BP.%2BKurtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurtzman%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACletus%2BP.%2BKurtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2208-9814" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cletus P. Kurtzman</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chris Todd Hittinger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a>
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a>
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3655" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460464" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3655.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460464" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3655.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460464" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3655.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460464" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3655.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460464" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The availability of genomes across the tree of life is highly biased toward vertebrates, pathogens, human disease models, and organisms with relatively small and simple genomes. Recent progress in genomics has enabled the <em>de novo</em> decoding of the genome of virtually any organism, greatly expanding its potential for understanding the biology and evolution of the full spectrum of biodiversity. The increasing diversity of sequencing technologies, assays, and <em>de novo</em> assembly algorithms have augmented the complexity of <em>de novo</em> genome sequencing projects in nonmodel organisms. To reduce the costs and challenges in <em>de novo</em> genome sequencing projects and streamline their experimental design and analysis, we developed iWGS (<em><span class="underline">i</span>n silico</em> <span class="underline">W</span>hole <span class="underline">G</span>enome <span class="underline">S</span>equencer and Analyzer), an automated pipeline for guiding the choice of appropriate sequencing strategy and assembly protocols. iWGS seamlessly integrates the four key steps of a <em>de novo</em> genome sequencing project: data generation (through simulation), data quality control, <em>de novo</em> assembly, and assembly evaluation and validation. The last three steps can also be applied to the analysis of real data. iWGS is designed to enable the user to have great flexibility in testing the range of experimental designs available for genome sequencing projects, and supports all major sequencing technologies and popular assembly tools. Three case studies illustrate how iWGS can guide the design of <em>de novo</em> genome sequencing projects, and evaluate the performance of a wide variety of user-specified sequencing strategies and assembly protocols on genomes of differing architectures. iWGS, along with a detailed documentation, is freely available at <a href="https://github.com/zhouxiaofan1983/iWGS">https://github.com/zhouxiaofan1983/iWGS</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genome-sequencing" rel="nofollow">genome sequencing</a></li><li class="kwd"><a href="/keyword/high-throughput-sequencing" class="hw-term hw-article-keyword hw-article-keyword-high-throughput-sequencing" rel="nofollow">high-throughput sequencing</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/de-novo-assembly" class="hw-term hw-article-keyword hw-article-keyword-de-novo-assembly" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">de novo</em> assembly</a></li><li class="kwd"><a href="/keyword/experimental-design" class="hw-term hw-article-keyword hw-article-keyword-experimental-design" rel="nofollow">experimental design</a></li><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">simulation</a></li><li class="kwd"><a href="/keyword/nonmodel-organism" class="hw-term hw-article-keyword hw-article-keyword-nonmodel-organism" rel="nofollow">nonmodel organism</a></li></ul><p id="p-3">Whole genome sequences are rich sources of information about organisms that are superbly useful for addressing a wide variety of evolutionary questions, such as measuring mutation rates (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kumar and Subramanian 2002</a>), characterizing the genomic basis of adaptation (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Roux <em>et al.</em> 2014</a>), and building the tree of life (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rokas <em>et al.</em> 2003</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Salichos and Rokas 2013</a>). Until now, however, organismal diversity has been highly unevenly covered, and most sequenced genomes correspond to model organisms, organisms of medical or economic importance, or ones that have relatively small and simple genomes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Reddy <em>et al.</em> 2015</a>).</p><p id="p-4">The rapid advance of DNA sequencing technologies has dramatically reduced the labor and cost required for genome sequencing, which is evidenced by the burst of large-scale genome projects in recent years that includes, for example, the 1000 Fungal Genomes (1KFG) Project (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Grigoriev <em>et al.</em> 2011</a>), the Yeast 1000 Plus (Y1000+) Project (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hittinger <em>et al.</em> 2015</a>), the Insect 5K Project (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Robinson <em>et al.</em> 2011</a>), and the Genome 10K Project (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Genome 10K Community of Scientists 2009</a>). Some of these projects have already begun to fuel important discoveries in evolution and other fields (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhang <em>et al.</em> 2014</a>). Equally importantly, high-throughput DNA sequencing has made it possible for single investigators to perform <em>de novo</em> genome sequencing in virtually any organism they are interested in (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rokas and Abbot 2009</a>). Such sequencing efforts may target various organisms with a large diversity of genome architectures. Therefore, to achieve optimal results, the choice of sequencing strategy (<em>i.e.</em>, the combination of sequencing technology [<em>e.g.</em>, Illumina or Pacific Biosciences (PacBio)], sequencing assay (<em>e.g.</em>, paired-end or mate-pair), and other variables, such as sequencing depth and assembly protocols (<em>e.g.</em>, assemblers and the associated parameters) should ideally be tailored to the characteristics of a given genome, such as size and GC/repeat content (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nagarajan and Pop 2013</a>).</p><p id="p-5">The vast majority of <em>de novo</em> sequenced genomes have been generated using the Illumina technology, either solely or in combination with other technologies (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Reddy <em>et al.</em> 2015</a>). This is largely due to the Illumina technology’s ability to quickly generate tens to hundreds of millions of highly accurate short sequence reads of up to 300 bases per run at very low per base cost (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Glenn 2011</a>). Additionally, the Illumina technology offers two powerful sequencing assays, paired-end (PE) and mate-pair (MP), which generate sequence read pairs that span short (hundreds of base-pairs) and relatively long (thousands of base-pairs) genomic regions, respectively. Mixing multiple PE and MP libraries with different insert sizes allows for highly flexible sequencing strategies, and several state-of-the-art assembly algorithms have been developed that exploit all these advantages. For instance, the <em>de novo</em> genome assembler ALLPATHS-LG can generate high quality draft assemblies for mammalian-size genomes using only Illumina short-read data by including both MP and overlapping PE libraries (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gnerre <em>et al.</em> 2011</a>). On its own, however, the Illumina technology performs less well for more complex genomes, mainly due to the short lengths of Illumina sequence reads and the technology’s bias against certain genomic regions (<em>e.g.</em>, GC-rich regions) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ross <em>et al.</em> 2013</a>).</p><p id="p-6">The PacBio technology generates sequence reads that are substantially longer and have much less sequencing bias, albeit at the cost of a substantially lower per-read accuracy; the average read length increases to above 10 kb with the latest chemistry but displays only ∼87% accuracy (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Koren and Phillippy 2015</a>). Thus, this technology is particularly useful for the sequencing of complex genomes (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Koren and Phillippy 2015</a>). Recent developments in both sequencing chemistry and assembly algorithms have enabled PacBio-only <em>de novo</em> assembly for microbial genomes (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Koren <em>et al.</em> 2013</a>), but the high sequence coverage required for this approach remains cost-prohibitive for large eukaryotic genomes. Nevertheless, in combination with more affordable Illumina short-read data, PacBio long reads—even at low coverage—can lead to significantly improved assemblies (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Utturkar <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">McIlwain <em>et al.</em> 2016</a>).</p><p id="p-7"><em>De novo</em> genome sequencing projects are further complicated by the large array of assembly software tools, which differ in many aspects, such as algorithmic design, supported/required data types, and computational efficiency (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Nagarajan and Pop 2013</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Simpson and Pop 2015</a>). Systematic evaluations of assembly programs show that no single assembler is the best across all circumstances; rather, an assembler’s performance critically depends on genome complexity and the sequencing strategy adopted (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Earl <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bradnam <em>et al.</em> 2013</a>). Moreover, many assemblers use adjustable parameters (<em>e.g.</em>, the k-mer size for <em>de Bruijn</em> assemblers), the values of which can critically affect the assembly quality. In practice, such parameters are often selected intuitively or through the time-consuming process of testing multiple values.</p><p id="p-8">The great number of possible ways to combine sequencing technologies, assays, and assembly algorithms poses a great challenge for the experimental design and data analysis in <em>de novo</em> genome sequencing projects, which in turn can sometimes lead to poor quality or downright incorrect assemblies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Denton <em>et al.</em> 2014</a>). As a consequence, several pipelines have been developed to automate specific steps in the process; for example, the recently developed iMetAMOS (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Koren <em>et al.</em> 2014</a>) and RAMPART (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mapleson <em>et al.</em> 2015</a>) have been specifically designed to automate genome assembly. However, as <em>de novo</em> genome sequencing is increasingly adopted by single investigator laboratories, there is an urgent need for streamlined approaches that enable investigators to not only efficiently generate high-quality draft genome assemblies but also to predict (via simulation) and identify the most suitable design(s) [<em>i.e.</em>, the most suitable combination(s) of sequencing strategy and assembly protocol] currently available for a specific genome.</p><p id="p-9">To address this need, we have developed an automated pipeline for the design and execution of <em>de novo</em> genome sequencing projects that we name iWGS (<em><span class="underline">i</span>n silico</em> <span class="underline">W</span>hole <span class="underline">G</span>enome <span class="underline">S</span>equencer and Analyzer). To approximate the performance of different sequencing strategies and assembly protocols, iWGS simulates high-throughput genome sequencing on user-provided reference genomes (<em>e.g.</em>, genomes that closely represent the characteristics of the real targets), facilitating the identification of optimal experimental designs. iWGS allows users to experiment with various combinations of sequencing technologies, assays, assembly tools, and relevant parameters in a single run. iWGS is also designed to work with real data and can be used as a convenient tool for automated selection of the best assembly or genome assembler. Finally, using three case studies, each one focused on specific challenges frequently encountered in <em>de novo</em> genome sequencing studies (<em>e.g.</em>, high repeat content and biased nucleotide composition, etc.), we illustrate how iWGS can be applied to guiding the design and analysis of <em>de novo</em> genome sequencing studies.</p><div class="section results" id="sec-1"><h2 class="">Results</h2><div id="sec-2" class="subsection"><h3>The design of iWGS</h3><p id="p-10">iWGS encompasses all major steps of a typical <em>de novo</em> genome sequencing study, including the generation of sequence reads, data quality control, <em>de novo</em> assembly, and evaluation of assemblies (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3655/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="iWGS workflow. A typical iWGS analysis consists of four steps: (1) data simulation (optional); (2) preprocessing (optional); (3) de novo assembly; and (4) assembly evaluation. iWGS supports both Illumina short reads and PacBio long reads, and a wide selection of assemblers to enable de novo assembly using either or both types of data. Users can start the analysis simulating data drawn from a reference genome assembly or, alternatively, use real sequencing data as input and skip the simulation step. iWGS, in silico Whole Genome Sequencer and Analyzer; MP, mate-pair; PacBio, Pacific Biosciences; PE, paired-end." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1938840602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;iWGS workflow. A typical iWGS analysis consists of four steps: (1) data simulation (optional); (2) preprocessing (optional); (3) &lt;em&gt;de novo&lt;/em&gt; assembly; and (4) assembly evaluation. iWGS supports both Illumina short reads and PacBio long reads, and a wide selection of assemblers to enable &lt;em&gt;de novo&lt;/em&gt; assembly using either or both types of data. Users can start the analysis simulating data drawn from a reference genome assembly or, alternatively, use real sequencing data as input and skip the simulation step. iWGS, &lt;em&gt;in silico&lt;/em&gt; Whole Genome Sequencer and Analyzer; MP, mate-pair; PacBio, Pacific Biosciences; PE, paired-end.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3655/F1.medium.gif" width="212" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3655/F1.medium.gif" width="212" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3655/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3655/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469676" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">iWGS workflow. A typical iWGS analysis consists of four steps: (1) data simulation (optional); (2) preprocessing (optional); (3) <em>de novo</em> assembly; and (4) assembly evaluation. iWGS supports both Illumina short reads and PacBio long reads, and a wide selection of assemblers to enable <em>de novo</em> assembly using either or both types of data. Users can start the analysis simulating data drawn from a reference genome assembly or, alternatively, use real sequencing data as input and skip the simulation step. iWGS, <em>in silico</em> Whole Genome Sequencer and Analyzer; MP, mate-pair; PacBio, Pacific Biosciences; PE, paired-end.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-3" class="subsection"><h4>Simulation:</h4><p id="p-12">iWGS uses the realistic high-throughput sequencing (HTS) read simulators ART (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Huang <em>et al.</em> 2012</a>), pIRS (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hu <em>et al.</em> 2012</a>), and PBSIM (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ono <em>et al.</em> 2013</a>) to generate Illumina and PacBio sequence reads from a given user-specified genome. These programs can simulate all popular data types, including Illumina PE and MP sequence reads, as well as PacBio continuous long sequence reads. The distributions of read quality and read length are easily adjustable for both Illumina and PacBio data. Furthermore, these simulators mimic sequencing errors and nucleotide composition biases in real data by using empirical profiles of these artifacts, which can be easily customized to stay current with upgrades in sequencing technologies. For instance, we have created a quality-score frequency profile learned from sequence reads generated by the latest PacBio chemistry to better reflect the improved sequence read accuracy. This simulation step can be omitted when the goal is the analysis of real data. Alternatively, the users may choose to perform only the simulation and use the simulated data for other analyses.</p></div><div id="sec-4" class="subsection"><h4>Quality control:</h4><p id="p-13">HTS data generated by all technologies contain errors and artifacts, which may sometimes substantially compromise the quality of the assembly (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhou and Rokas 2014</a>). Therefore, iWGS includes an optional step to perform preprocessing of the data, including trimming of low-quality bases, removal of adapter contaminations, and correction of sequencing errors. Since some assemblers [<em>e.g.</em> ALLPATHS-LG (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ribeiro <em>et al.</em> 2012</a>)] have their own preprocessing modules, iWGS automatically determines for each assembly protocol whether to use the original or the processed data.</p></div><div id="sec-5" class="subsection"><h4>Assembly:</h4><p id="p-14">To maximize users’ flexibility in experimental design, iWGS supports 15 <em>de novo</em> genome assembly tools [ABYSS (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Simpson <em>et al.</em> 2009</a>), ALLPATHS-LG (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Ribeiro <em>et al.</em> 2012</a>), Celera Assembler (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Myers <em>et al.</em> 2000</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berlin <em>et al.</em> 2015</a>), Canu (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Koren <em>et al.</em> 2016</a>), DBG2OLC (Ye <em>et al.</em> 2014), DISCOVAR (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Weisenfeld <em>et al.</em> 2014</a>), Falcon (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chin <em>et al.</em> 2016</a>), MaSuRCA (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zimin <em>et al.</em> 2013</a>), Meraculous (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chapman <em>et al.</em> 2011</a>), Minia (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Salikhov <em>et al.</em> 2013</a>), Platanus (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kajitani <em>et al.</em> 2014</a>), SGA (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Simpson and Durbin 2012</a>), SOAPdenovo2 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Luo <em>et al.</em> 2012</a>), SPAdes (and a diploid-aware version called dipSPAdes) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bankevich <em>et al.</em> 2012</a>), and Velvet (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zerbino and Birney 2008</a>)], most of which have participated in recent large-scale assembler comparisons (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bradnam <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Magoc <em>et al.</em> 2013</a>). These supported assemblers allow users to carry out <em>de novo</em> assembly using only Illumina short-read data (<em>e.g.</em>, SOAPdenovo2) and only PacBio long-read data (<em>e.g.</em>, Canu and Falcon), or to perform hybrid assembly that uses both (<em>e.g.</em>, SPAdes and DBG2OLC). To achieve the best possible results while avoiding the computationally expensive process of testing multiple combinations of parameters, iWGS takes advantage of successful assembly recipes (<em>i.e.</em>, recommended settings for each assembler) established in studies such as Assemblathon 2 (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Bradnam <em>et al.</em> 2013</a>) and GAGE-B (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Magoc <em>et al.</em> 2013</a>), and uses KmerGenie to determine the optimal k-mer size (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chikhi and Medvedev 2014</a>). In addition, assemblies generated from different underlying data and/or assembly algorithms can be merged using Metassembler (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wences and Schatz 2015</a>) to achieve a potentially better final assembly.</p></div><div id="sec-6" class="subsection"><h4>Evaluation:</h4><p id="p-15">iWGS uses QUAST (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gurevich <em>et al.</em> 2013</a>) to evaluate all generated assemblies. In addition to providing basic statistics like N50 (the largest contig/scaffold size wherein half of the total assembly size is contained in contigs/scaffolds no shorter than this value), QUAST compares each assembly against the reference genome (in the case of simulations) and generates a number of highly informative quality matrices, such as misassemblies, assembled sequences not present in the reference (and vice versa), and genes recovered in the assembly if the reference genome is annotated. At the end, iWGS ranks all assemblies based on selected matrices in the QUAST report using a previously described weighting strategy (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abbas <em>et al.</em> 2014</a>). This ranking, along with the detailed QUAST report, helps users to identify the best overall assembly, as well as the corresponding combination of sequencing strategy and assembly protocol. REAPR, which utilizes the sequence data itself for assembly evaluation, is also implemented to better suit real data analysis (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hunt <em>et al.</em> 2013</a>).</p><p id="p-16">iWGS is designed with flexibility and ease-of-use in mind to allow users to readily examine various experimental designs; each data set may be used multiple times in different assembly protocols, and each assembler may be run repeatedly with different input data sets. Multiple sequencing strategies and assembly protocols can be specified in a straightforward fashion in a single configuration file; only a few parameters are required for each strategy/protocol, while other settings (<em>e.g.</em>, quality profiles for read simulation) are globally shared across strategies/protocols of the same type. Alternatively, advanced users can opt to customize the strategies/protocols so that, for example, each sequencing data set is simulated with different quality settings. Furthermore, iWGS rigorously checks the configurations for issues such as the compatibility between sequencing strategies and assembly protocols.</p><p id="p-17">iWGS is a lightweight pipeline written in Perl. The source code, detailed documentation, and example test sets are freely available at <a href="https://github.com/zhouxiaofan1983/iWGS">https://github.com/zhouxiaofan1983/iWGS</a>. Like many other bioinformatics pipelines, iWGS inevitably relies on a number of third-party software tools to carry out individual analyses such as data simulation and genome assembly. However, most of the tools, including at least one for each of the four major steps aforementioned, either have precomplied executables or can be compiled locally with ease. For the convenience of users, we also include in the package scripts to automate the acquisition and installation of most software dependencies. The users can also customize the selection of tools to install according to their own needs and computational environments.</p></div></div><div id="sec-7" class="subsection"><h3>Case studies</h3><p id="p-18">To demonstrate the use of iWGS and provide examples of its utility, we developed three case studies where iWGS was used to guide the selection of sequencing strategy for genomes representing a wide range of sizes and complexity levels (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS1.xlsx">Table S1</a>). The competing strategies were selected to enable both Illumina-only and PacBio-only assemblies, as well as hybrid assembly of the two data types (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). To examine the effectiveness of the simulation step of our approach, we also analyzed real sequencing data that largely match our simulation settings.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469545/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469545/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sequencing strategies (top) and assembly protocols (bottom) evaluated in the three case studies</span><div class="sb-div caption-clear"></div></div></div><div id="sec-8" class="subsection"><h4>Case study I (repeat-content issue):</h4><p id="p-21">We first compared the sequencing of two fungi, <em>Zymoseptoria tritici</em> (synonym: <em>Mycosphaerella graminicola</em>) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Goodwin <em>et al.</em> 2011</a>) and <em>Pseudocercospora fijiensis</em> (synonym: <em>M. fijiensis</em>) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ohm <em>et al.</em> 2012</a>), which both belong to the class Dothideomycetes yet have dramatically different repeat contents; the estimated repeat contents are ∼15 and ∼50% for the two genomes, respectively. Our simulations showed that, while good quality assemblies can be obtained for <em>Z. tritici</em> using either data type, the PacBio-only assembly for <em>Ps. fijiensis</em> vastly outperforms assemblies based on Illumina data alone or in combination with low-coverage PacBio data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The results are consistent with the notion that PacBio long reads are particularly powerful in resolving repeats (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Koren <em>et al.</em> 2013</a>). We then further tested if these results are informative for guiding the sequencing of another highly repetitive Dothideomycetes genome, <em>Cenococcum geophilum</em>, which has a repeat content of ∼76% (<a href="http://genome.jgi.doe.gov/Cenge3">http://genome.jgi.doe.gov/Cenge3</a>). For <em>C. geophilum</em>, the PacBio-only assembly was again found to be the best, while the hybrid assembly using DBG2OLC and the Illumina-only assembly using ALLPATHS-LG were next in rank (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS2.xlsx">Table S2</a>), nicely recapitulating the results of <em>Ps. fijiensis</em>. We also performed meta-assembly of Illumina-only assemblies ILMN1 to ILMN7 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) on all three genomes using Metassembler. While the meta-assembly approach substantially improved the assembly continuity for <em>Z. tritici</em>, no improvement was observed for <em>Ps. fijiensis</em> and <em>C. geophilum</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS2.xlsx">Table S2</a>). These results suggest that the use of iWGS would provide critical information to help end users choose a successful sequencing of highly repetitive genomes that share similar characteristics. Importantly, since simulated assemblies are recoverable, the likely impact of the different assembly strategies on genes, gene families, or pathways of interest could also be examined in detail.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3655/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance comparison of five representative experimental designs on three Dothideomycetes genomes. The five designs shown include three Illumina-only designs (ILMN2: ALLPATHS-LG, META: Metassembler, and ILMN8: DISCOVAR), the best performing PacBio-only design (PACB2: Canu), and the best performing hybrid design (HYBR2: DBG2OLC) for each genome. The statistics on the assembled fraction of the reference genome, scaffold N50, and largest scaffold size are all after correction for assembly errors using the reference genome as reported by QUAST in GAGE mode. By default, QUAST (in GAGE mode) corrects contigs/scaffolds by breaking them at assembly errors larger than 5 bp. Scaffold N50 and largest scaffold size are shown in log10 scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1938840602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Performance comparison of five representative experimental designs on three Dothideomycetes genomes. The five designs shown include three Illumina-only designs (ILMN2: ALLPATHS-LG, META: Metassembler, and ILMN8: DISCOVAR), the best performing PacBio-only design (PACB2: Canu), and the best performing hybrid design (HYBR2: DBG2OLC) for each genome. The statistics on the assembled fraction of the reference genome, scaffold N50, and largest scaffold size are all after correction for assembly errors using the reference genome as reported by QUAST in GAGE mode. By default, QUAST (in GAGE mode) corrects contigs/scaffolds by breaking them at assembly errors larger than 5 bp. Scaffold N50 and largest scaffold size are shown in log10 scale.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3655/F2.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3655/F2.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3655/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3655/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469595" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Performance comparison of five representative experimental designs on three Dothideomycetes genomes. The five designs shown include three Illumina-only designs (ILMN2: ALLPATHS-LG, META: Metassembler, and ILMN8: DISCOVAR), the best performing PacBio-only design (PACB2: Canu), and the best performing hybrid design (HYBR2: DBG2OLC) for each genome. The statistics on the assembled fraction of the reference genome, scaffold N50, and largest scaffold size are all after correction for assembly errors using the reference genome as reported by QUAST in GAGE mode. By default, QUAST (in GAGE mode) corrects contigs/scaffolds by breaking them at assembly errors larger than 5 bp. Scaffold N50 and largest scaffold size are shown in log10 scale.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h4>Case study II (GC-content and mtDNA assembly issue):</h4><p id="p-23">We next examined the <em>de novo</em> assembly of mitochondrial genomes from whole genome sequencing data of <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mewes <em>et al.</em> 1997</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Foury <em>et al.</em> 1998</a>). Yeast mitochondrial genomes are valuable resources for evolutionary and functional studies (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Freel <em>et al.</em> 2015</a>), yet the acquisition of finished mitochondrial genome assemblies is not trivial because of their very low GC-content (∼17%). We simulated a genome sequencing experiment using the nuclear and mitochondrial genomes of <em>S. cerevisiae</em>. We tested two ratios of nuclear to mitochondrial genome copy numbers representing low (1:50) and high (1:200) mitochondrial contents, respectively (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Solieri 2010</a>). iWGS analysis showed that the <em>S. cerevisiae</em> mitochondrial genome was fully recovered at both low and high mitochondrial contents using Illumina data (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Consistent with recent observations made during the assembly of the <em>S. eubayanus</em> genome, only certain assemblers performed well; for example, ALLPATHS-LG performed surprisingly poorly, while SPAdes performed quite well (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baker <em>et al.</em> 2015</a>). Importantly, the complete mitochondrial genome can be obtained as a single contig using only Illumina or only PacBio data, or using both data types (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Similarly, both Illumina and PacBio data resulted in good quality assemblies of the nuclear genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS2.xlsx">Table S2</a>). At the same time, different assemblers exhibited widely different performances even with the same input data (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469574/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469574/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Performance of all experimental designs evaluated in case study II</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Case study III (genomic architecture issue):</h4><p id="p-25">Lastly, we applied iWGS to three model eukaryotic genomes from different kingdoms and with different genomic architectures. Specifically, we analyzed <em>Drosophila melanogaster</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adams <em>et al.</em> 2000</a>) and <em>Arabidopsis thaliana</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arabidopsis Genome Initiative 2000</a>), which are medium-sized animal and plant genomes, respectively, as well as <em>Plasmodium falciparum</em> 3D7 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gardner <em>et al.</em> 2002</a>), a smaller protist genome with extremely low GC-content (∼19%). For all three genomes, the best assembly was generated by using only PacBio data (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). In <em>D. melanogaster</em> and <em>A. thaliana</em>, several Illumina-only assemblies were of relatively high-quality (<em>i.e.</em>, corrected scaffold N50 ≥ 100 kb; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS2.xlsx">Table S2</a>), among which the best two were generated by ALLPATHS-LG and DISCOVAR (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). However, all Illumina-only assemblies of <em>Pl. falciparum</em> 3D7 had considerably lower corrected scaffold N50 values, except for DISCOVAR whose sequencing strategy is unique in requiring a PE library with a limited insert size.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469608/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469608/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of top-ranking assemblies generated in case study III</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">To examine how well the simulation-based predictions made by iWGS are supported by empirical data, we collected four real genome sequencing data sets from a previous study of <em>Pl. falciparum</em> IT [one overlapping 100 bp PE library, one overlapping 250 bp PE library, one MP library, and one PacBio library from (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Otto <em>et al.</em> 2014</a>); <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS1.xlsx">Table S1</a>] that were a good match to our simulated data sets, and ran the same set of assembly protocols. The best assembly was again generated by PacBio data alone, and the assemblies generated by ALLPATHS-LG, DISCOVAR (both are Illumina-only), and DBG2OLC (hybrid) were ranked next, while all other Illumina-only assembly protocols performed poorly (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS2.xlsx">Table S2</a>). The results are largely consistent with our simulation study, suggesting that our simulation-based approach is indeed informative.</p></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-28">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-29">The design and analysis of <em>de novo</em> genome sequencing experiments is not trivial. On the design front, one has to balance between the complexity of the target genome, the strengths and weaknesses of each sequencing technology, and, importantly, the cost. Analysis is also challenging, as one is faced with multiple different algorithms and dozens of parameters. Although substantial efforts have been made to benchmark different approaches for genome assembly (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Earl <em>et al.</em> 2011</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Salzberg <em>et al.</em> 2012</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Bradnam <em>et al.</em> 2013</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Magoc <em>et al.</em> 2013</a>), much less attention has been paid to investigating start-to-finish optimal sequencing strategies for a given genome [see (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chakraborty <em>et al.</em> 2016</a>) for one example].</p><p id="p-30">iWGS is an automated tool that allows users to explicitly compare alternative experimental designs by using simulated sequencing data, even allowing users to estimate costs when these are known for the generation of each data type. We have illustrated the utility of iWGS in several case studies on mitochondrial and nuclear genomes with varying levels of complexity. For instance, our simulations suggest that Illumina-only sequencing strategies may be economical choices for the sequencing of relatively simple genomes (<em>e.g.</em>, <em>Z. tritici</em>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1/TableS2.xlsx">Table S2</a>), whereas PacBio data would be highly desirable for genomes of greater complexity (<em>e.g.</em>, <em>Ps. fijiensis</em>, <em>C. geophilum</em>, and <em>Pl. falciparum</em>). Although not done here, iWGS could also be used to evaluate different combinations of sequencing assays (<em>e.g.</em>, PE and MP libraries), read quality, read lengths, and sequencing depths. Empirical studies of both short- and long-read data have shown that these parameters are critical determinants of the quality of <em>de novo</em> genome assemblies (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Utturkar <em>et al.</em> 2014</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chakraborty <em>et al.</em> 2016</a>).</p><p id="p-31">One key function of iWGS is the use of simulation data generated from a related reference genome to inform the experimental design for organisms lacking genomic data. A similar concept was previously used to evaluate sequencing strategies for cacao by using the rice genome as the reference (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Haiminen <em>et al.</em> 2011</a>). In principle, one could apply iWGS on one or more related reference genomes that resemble the characteristics (<em>e.g.</em>, genome size, repeat content, and sequence composition) of the sequencing target. However, if such reference genome is lacking, one solution is to start with a closely related reference genome and tune it toward the target (<em>e.g.</em>, adjust GC- and repeat contents) by using third-party tools that simulate genome-wide evolution (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Arenas and Posada 2014</a>) before running iWGS. Alternatively, one may simply use iWGS with reference genomes that are of comparable complexity (<em>e.g.</em>, similar in size and repeat content) regardless of the evolutionary relatedness. As suggested by previous studies, these factors not only influence the difficulty of genome assembly, but can also be excellent predictors of the assembly quality (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lee <em>et al.</em> 2014</a>). Therefore, iWGS could also be informative in evaluating the performance of alternative experimental designs on genomes with similar characteristics to the sequencing target.</p><p id="p-32">Other important features of iWGS include the support for both Illumina short, and PacBio long, sequence reads and, correspondingly, a wide selection of software tools compatible with these data types, as well as the ability to analyze real data. In comparison, the support for third generation sequencing data are relatively limited in iMetAMOS and currently lacking in RAMPART. Given the increasing importance of long sequence reads in <em>de novo</em> genome assembly, iWGS aims to allow users to fully exploit the strength of long-read data and explore alternative ways of data analysis. Along these lines, several further developments can be envisioned. First, support for additional sequencing technologies, such as Oxford Nanopore, can be added as technologies become commercially available. In fact, the Celera Assembler, Canu, and SPAdes assemblers, which are supported by iWGS, can already utilize nanopore reads (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bankevich <em>et al.</em> 2012</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Berlin <em>et al.</em> 2015</a>). Similarly, realistic simulation of nanopore data will be possible once the patterns of errors and biases are better characterized using real data. Second, iWGS will continue to expand its functionality to achieve better assemblies. For instance, a number of assembly polishing tools can be integrated in iWGS to improve the quality of the final output, including Pilon (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Walker <em>et al.</em> 2014</a>), Quiver (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chin <em>et al.</em> 2013</a>), and Nanopolish (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Loman <em>et al.</em> 2015</a>), which use Illumina, PacBio, and nanopore data, respectively. In addition, iWGS currently uses Metassembler for meta-assembly; in the future, other meta-assembly tools that support assemblies based on PacBio data alone, such as quickmerge (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Chakraborty <em>et al.</em> 2016</a>), could be added. Lastly, it would be beneficial to enable users to add new software tools to iWGS in order to stay up-to-date with the rapid advances in genome assembly and other aspects of HTS data analysis. We intend to provide periodic updates, and the expert user can edit iWGS on their own. In summary, iWGS is a flexible, expandable, and easy to use pipeline that will aid in the design and execution of genome assembly experiments across the tree of life.</p></div><div class="section" id="sec-13"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">We thank Branden Timm for technical support on the WEI cluster. We thank Francis Martin for providing access to unpublished genome data for the <em>C. geophilum</em> genome produced by the U.S. Department of Energy Joint Genome Institute, a Department of Energy (DOE) Office of Science User Facility, supported by the Office of Science of the U.S. Department of Energy under contract no. DE-AC02-05CH11231. This work was conducted in part using the resources of the Advanced Computing Center for Research and Education at Vanderbilt University and the Wisconsin Energy Institute (WEI) cluster at the University of Wisconsin–Madison. This work was funded by the National Science Foundation (DEB-1442113 to A.R.; DEB-1253634 and DEB-1442148 to C.T.H.), in part by the DOE Great Lakes Bioenergy Research Center (DOE Office of Science BER DE-FC02-07ER64494 to C.T.H.), the U.S. Department of Agriculture (USDA) National Institute of Food and Agriculture (Hatch Project 1003258 to C.T.H.), and the National Institutes of Health (NIAID AI105619 to A.R.). C.T.H. is an Alfred Toepfer Faculty Fellow, supported by the Alexander von Humboldt Foundation. C.T.H. is a Pew Scholar in the Biomedical Sciences, supported by the Pew Charitable Trusts. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the USDA. USDA is an equal opportunity provider and employer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-34">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034249/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-35"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-08-04"><span class="received-label">Received </span>August 4, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-09-08"><span class="accepted-label">Accepted </span>September 8, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Zhou <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abbas</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malluhi</span>  <span class="cit-name-given-names">Q. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessment of <em>de novo</em> assemblers for draft genomes: a case study with fungal genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-supplement">Suppl. 9</span>): <span class="cit-fpage">S10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253DS10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.2" data-doi="10.1126/science.287.5461.2185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gocayne</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The genome sequence of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2185</span>–<span class="cit-lpage">2195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAdams%26rft.auinit1%253DM.%2BD.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2185%26rft.epage%253D2195%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2185%26rft_id%253Dinfo%253Apmid%252F10731132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.3" data-doi="10.1038/35048692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Arabidopsis Genome Initiative</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D796%26rft.epage%253D815%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bflowering%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048692%26rft_id%253Dinfo%253Apmid%252F11130711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130711&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.4" data-doi="10.1093/molbev/msu078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arenas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Simulation of genome-wide evolution under heterogeneous substitution models and complex multispecies coalescent histories.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu078%26rft_id%253Dinfo%253Apmid%252F24557445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzUvMTI5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.5" data-doi="10.1093/molbev/msv168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellora</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peris</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulfachor</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genome sequence of <em>Saccharomyces eubayanus</em> and the domestication of Lager-Brewing yeasts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2818</span>–<span class="cit-lpage">2831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv168%26rft_id%253Dinfo%253Apmid%252F26269586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xMS8yODE4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.6" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.7" data-doi="10.1038/nbt.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3238%26rft_id%253Dinfo%253Apmid%252F26006009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26006009&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.8" data-doi="10.1186/2047-217X-2-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fass</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assemblathon 2: evaluating <em>de novo</em> methods of genome assembly in three vertebrate species.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D2%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-2-10%26rft_id%253Dinfo%253Apmid%252F23870653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-2-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23870653&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.9" data-doi="10.1093/nar/gkw654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldwin-Brown</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Contiguous and accurate <em>de novo</em> assembly of metazoan genomes with modest long read coverage.</span> <abbr class="cit-jnl-abbrev">Nucl. Acids Res.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucl.%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw654%26rft_id%253Dinfo%253Apmid%252F27458204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0NC8xOS9lMTQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.10" data-doi="10.1371/journal.pone.0023501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunkara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroth</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Meraculous: <em>de novo</em> genome assembly with short paired-end reads.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e23501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChapman%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De23501%26rft.epage%253De23501%26rft.atitle%253DMeraculous%253A%2Bde%2Bnovo%2Bgenome%2Bassembly%2Bwith%2Bshort%2Bpaired-end%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0023501%26rft_id%253Dinfo%253Apmid%252F21876754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0023501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21876754&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.11" data-doi="10.1093/bioinformatics/btt310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medvedev</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Informed and automated <em>k</em>-mer size selection for genome assembly.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt310%26rft_id%253Dinfo%253Apmid%252F23732276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiIzMC8xLzMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.12" data-doi="10.1038/nmeth.2474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klammer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2474%26rft_id%253Dinfo%253Apmid%252F23644548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644548&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319668700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3655.13"><div class="cit-metadata unstructured">Chin, C.-S., P. Peluso, F. J. Sedlazeck, M. Nattestad, G. T. Concepcion <em>et al.</em>, 2016 Phased diploid genome assembly with single molecule real-time sequencing. bioRxiv DOI: <a href="http://dx.doi.org/10.1101/056887">http://dx.doi.org/10.1101/056887</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.14" data-doi="10.1371/journal.pcbi.1003998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denton</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lugo-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Extensive error in the number of genes inferred from draft genome assemblies.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1003998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D10%26rft.spage%253De1003998%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003998%26rft_id%253Dinfo%253Apmid%252F25474019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25474019&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.15" data-doi="10.1101/gr.126599.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St John</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assemblathon 1: a competitive assessment of <em>de novo</em> short read assembly methods.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2224</span>–<span class="cit-lpage">2241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.126599.111%26rft_id%253Dinfo%253Apmid%252F21926179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8yMjI0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.16" data-doi="10.1016/S0014-5793(98)01467-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foury</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roganti</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lecrenier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purnelle</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The complete sequence of the mitochondrial genome of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DFoury%26rft.auinit1%253DF.%26rft.volume%253D440%26rft.issue%253D3%26rft.spage%253D325%26rft.epage%253D331%26rft.atitle%253DThe%2Bcomplete%2Bsequence%2Bof%2Bthe%2Bmitochondrial%2Bgenome%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252898%252901467-7%26rft_id%253Dinfo%253Apmid%252F9872396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(98)01467-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9872396&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077595600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.17" data-doi="10.1093/femsyr/fov023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freel</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mitochondrial genome evolution in yeasts: an all-encompassing view.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">fov023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov023%26rft_id%253Dinfo%253Apmid%252F25969454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjExOiIxNS80L2ZvdjAyMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.18" data-doi="10.1038/nature01097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berriman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genome sequence of the human malaria parasite <em>Plasmodium falciparum</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGardner%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D419%26rft.issue%253D6906%26rft.spage%253D498%26rft.epage%253D511%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bhuman%2Bmalaria%2Bparasite%2BPlasmodium%2Bfalciparum.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01097%26rft_id%253Dinfo%253Apmid%252F12368864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12368864&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178348400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.19" data-doi="10.1093/jhered/esp086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Genome 10K Community of Scientists</span></li></ol><cite>. <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome 10K: a proposal to obtain whole-genome sequence for 10,000 vertebrate species.</span> <abbr class="cit-jnl-abbrev">J Hered.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesp086%26rft_id%253Dinfo%253Apmid%252F19892720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwMC82LzY1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.20" data-doi="10.1111/j.1755-0998.2011.03024.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glenn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Field guide to next-generation DNA sequencers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D759%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.03024.x%26rft_id%253Dinfo%253Apmid%252F21592312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.03024.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21592312&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.21" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccallum</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.22" data-doi="10.1371/journal.pgen.1002070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">M’Barek</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGoodwin%26rft.auinit1%253DS.%2BB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002070%26rft.epage%253De1002070%26rft.atitle%253DFinished%2Bgenome%2Bof%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola%2Breveals%2Bdispensome%2Bstructure%252C%2Bchromosome%2Bplasticity%252C%2Band%2Bstealth%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002070%26rft_id%253Dinfo%253Apmid%252F21695235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21695235&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.23" data-doi="10.1080/21501203.2011.584577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hibbett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffries</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fueling the future with fungal genomics.</span> <abbr class="cit-jnl-abbrev">Mycology</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycology%26rft.volume%253D2%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.1080%252F21501203.2011.584577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/21501203.2011.584577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.24" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyOS84LzEwNzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.25" data-doi="10.1371/journal.pone.0024182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haiminen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigoutsos</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evaluation of methods for <em>de novo</em> genome assembly from high-throughput sequencing reads reveals dependencies that affect the quality of the results.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHaiminen%26rft.auinit1%253DN.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De24182%26rft.epage%253De24182%26rft.atitle%253DEvaluation%2Bof%2Bmethods%2Bfor%2Bde%2Bnovo%2Bgenome%2Bassembly%2Bfrom%2Bhigh-throughput%2Bsequencing%2Breads%2Breveals%2Bdependencies%2Bthat%2Baffect%2Bthe%2Bquality%2Bof%2Bthe%2Bresults.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024182%26rft_id%253Dinfo%253Apmid%252F21915294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915294&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.26" data-doi="10.1016/j.gde.2015.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boekhout</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goncalves</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomics and the making of yeast biodiversity.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D35%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2015.10.008%26rft_id%253Dinfo%253Apmid%252F26649756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2015.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26649756&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.27" data-doi="10.1093/bioinformatics/bts187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">pIRS: profile-based Illumina pair-end reads simulator.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts187%26rft_id%253Dinfo%253Apmid%252F22508794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTEvMTUzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.28" data-doi="10.1093/bioinformatics/btr708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ART: a next-generation sequencing read simulator.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr708%26rft_id%253Dinfo%253Apmid%252F22199392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOC80LzU5MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.29" data-doi="10.1186/gb-2013-14-5-r47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newbold</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berriman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">REAPR: a universal tool for genome assembly evaluation.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR47%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-5-r47%26rft_id%253Dinfo%253Apmid%252F23710727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-5-r47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23710727&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.30" data-doi="10.1101/gr.170720.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toshimoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient <em>de novo</em> assembly of highly heterozygous genomes from whole-genome shotgun short reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170720.113%26rft_id%253Dinfo%253Apmid%252F24755901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTM4NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.31" data-doi="10.1016/j.mib.2014.11.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">One chromosome, one contig: complete microbial genomes from long-read sequencing and assembly.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol.%26rft.volume%253D23%26rft.spage%253D110%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2014.11.014%26rft_id%253Dinfo%253Apmid%252F25461581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2014.11.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25461581&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.32" data-doi="10.1186/gb-2013-14-9-r101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harhay</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harhay</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reducing assembly complexity of microbial genomes with single-molecule sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR101%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r101%26rft_id%253Dinfo%253Apmid%252F24034426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24034426&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.33" data-doi="10.1186/1471-2105-15-126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Automated ensemble assembly and validation of microbial genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-126%26rft_id%253Dinfo%253Apmid%252F24884846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24884846&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3655.34"><div class="cit-metadata unstructured">Koren, S., B. P. Walenz, K. Berlin, J. R. Miller, and A. M. Phillippy, 2016 Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. bioRxiv DOI: <a href="http://dx.doi.org/10.1101/071282">http://dx.doi.org/10.1101/071282</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.35" data-doi="10.1073/pnas.022629899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mutation rates in mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.022629899%26rft_id%253Dinfo%253Apmid%252F11792858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OS8yLzgwMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3655.36"><div class="cit-metadata unstructured">Lee, H., J. Gurtowski, S. Yoo, S. Marcus, W. R. McCombie <em>et al.</em>, 2014 Error correction and assembly complexity of single molecule sequencing reads. bioRxiv DOI: <a href="http://dx.doi.org/10.1101/006395">http://dx.doi.org/10.1101/006395</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.37" data-doi="10.1038/nmeth.3444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A complete bacterial genome assembled <em>de novo</em> using only nanopore sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">733</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D733%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3444%26rft_id%253Dinfo%253Apmid%252F26076426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26076426&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.38" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read <em>de novo</em> assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.39" data-doi="10.1093/bioinformatics/btt273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magoc</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pabinger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canzar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">GAGE-B: an evaluation of genome assemblers for bacterial organisms.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1718</span>–<span class="cit-lpage">1725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt273%26rft_id%253Dinfo%253Apmid%252F23665771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTQvMTcxOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.40" data-doi="10.1093/bioinformatics/btv056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mapleson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarbreck</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">RAMPART: a workflow management system for <em>de novo</em> genome assembly.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1824</span>–<span class="cit-lpage">1826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv056%26rft_id%253Dinfo%253Apmid%252F25637556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzEvMTEvMTgyNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.41" data-doi="10.1534/g3.116.029389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McIlwain</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peris</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sardi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moskvin</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome sequence and analysis of a stress-tolerant, wild-derived strain of <em>Saccharomyces cerevisiae</em> used in biofuels research.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1757</span>–<span class="cit-lpage">1766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029389%26rft_id%253Dinfo%253Apmid%252F27172212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNi8xNzU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mewes</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albermann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frishman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleissner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Overview of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMewes%26rft.auinit1%253DH.%2BW.%26rft.volume%253D387%26rft.issue%253D6632%2BSuppl%26rft.spage%253D7%26rft.epage%253D65%26rft.atitle%253DOverview%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F9169865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9169865&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.43" data-doi="10.1126/science.287.5461.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dew</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasulo</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A whole-genome assembly of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2196</span>–<span class="cit-lpage">2204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DE.%2BW.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2196%26rft.epage%253D2204%26rft.atitle%253DA%2BWhole-Genome%2BAssembly%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2196%26rft_id%253Dinfo%253Apmid%252F10731133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.44" data-doi="10.1038/nrg3367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequence assembly demystified.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3367%26rft_id%253Dinfo%253Apmid%252F23358380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23358380&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.45" data-doi="10.1371/journal.ppat.1003037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohm</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feau</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horwitz</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen <em>Dothideomycetes</em> fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D8%26rft.spage%253De1003037%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003037%26rft_id%253Dinfo%253Apmid%252F23236275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23236275&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.46" data-doi="10.1093/bioinformatics/bts649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamada</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">PBSIM: PacBio reads simulator–toward accurate genome assembly.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts649%26rft_id%253Dinfo%253Apmid%252F23129296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOS8xLzExOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.47" data-doi="10.1038/ncomms5754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rayner</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohme</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pain</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spottiswoode</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing of chimpanzee malaria parasites reveals possible pathways of adaptation to human hosts.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D4754%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5754%26rft_id%253Dinfo%253Apmid%252F25203297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25203297&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.48" data-doi="10.1093/nar/gku950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertsch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isbandi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Genomes OnLine Database (GOLD) v.5: a metadata management system based on a four level (meta)genome project classification.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D1099</span>–<span class="cit-lpage">D1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku950%26rft_id%253Dinfo%253Apmid%252F25348402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0My9EMS9EMTA5OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.49" data-doi="10.1101/gr.141515.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Finished bacterial genomes from shotgun sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2270</span>–<span class="cit-lpage">2277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.141515.112%26rft_id%253Dinfo%253Apmid%252F22829535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMS8yMjcwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.50" data-doi="10.1126/science.331.6023.1386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell-Miramontes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Creating a buzz about insect genomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">1386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRobinson%26rft.auinit1%253DG.%2BE.%26rft.volume%253D331%26rft.issue%253D6023%26rft.spage%253D1386%26rft.epage%253D1386%26rft.atitle%253DCreating%2Ba%2BBuzz%2BAbout%2BInsect%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.331.6023.1386%26rft_id%253Dinfo%253Apmid%252F21415334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzEvNjAyMy8xMzg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.51" data-doi="10.1016/j.tree.2008.11.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbot</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Harnessing genomics for evolutionary insights.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D192%26rft.epage%253D200%26rft.atitle%253DHarnessing%2Bgenomics%2Bfor%2Bevolutionary%2Binsights.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2008.11.004%26rft_id%253Dinfo%253Apmid%252F19201503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2008.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19201503&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265358900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.52" data-doi="10.1038/nature02053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-scale approaches to resolving incongruence in molecular phylogenies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">798</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D425%26rft.issue%253D6960%26rft.spage%253D798%26rft.epage%253D804%26rft.atitle%253DGenome-scale%2Bapproaches%2Bto%2Bresolving%2Bincongruence%2Bin%2Bmolecular%2Bphylogenies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02053%26rft_id%253Dinfo%253Apmid%252F14574403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574403&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186118500036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.53" data-doi="10.1186/gb-2013-14-5-r51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russ</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costello</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennon</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterizing and measuring bias in sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR51%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-5-r51%26rft_id%253Dinfo%253Apmid%252F23718773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-5-r51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23718773&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.54" data-doi="10.1093/molbev/msu141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Privman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson-Rechavi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Patterns of positive selection in seven ant genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1661</span>–<span class="cit-lpage">1685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu141%26rft_id%253Dinfo%253Apmid%252F24782441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzcvMTY2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.55" data-doi="10.1038/nature12130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring ancient divergences requires genes with strong phylogenetic signals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12130%26rft_id%253Dinfo%253Apmid%252F23657258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23657258&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318952000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3655.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salikhov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sacomoto</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kucherov</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using cascading Bloom filters to improve the memory usage for de Brujin graphs</span>, pp. <span class="cit-fpage">364</span>–<span class="cit-lpage">376</span> in <span class="cit-source"><em>Algorithms in Bioinformatics</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Stoye</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.57" data-doi="10.1101/gr.131383.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magoc</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAGE: a critical evaluation of genome assemblies and assembly algorithms.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131383.111%26rft_id%253Dinfo%253Apmid%252F22147368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.58" data-doi="10.1101/gr.126953.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient <em>de novo</em> assembly of large genomes using compressed data structures.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.126953.111%26rft_id%253Dinfo%253Apmid%252F22156294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.59" data-doi="10.1146/annurev-genom-090314-050032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The theory and practice of genome sequence assembly.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenomics%2BHum.%2BGenet.%26rft.volume%253D16%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-090314-050032%26rft_id%253Dinfo%253Apmid%252F25939056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-090314-050032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25939056&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.60" data-doi="10.1101/gr.089532.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">ABySS: a parallel assembler for short read sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089532.108%26rft_id%253Dinfo%253Apmid%252F19251739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTExNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.61" data-doi="10.1016/j.tim.2010.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solieri</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mitochondrial inheritance in budding yeasts: towards an integrated understanding.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DSolieri%26rft.auinit1%253DL.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D521%26rft.epage%253D530%26rft.atitle%253DMitochondrial%2Binheritance%2Bin%2Bbudding%2Byeasts%253A%2Btowards%2Ban%2Bintegrated%2Bunderstanding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2010.08.001%26rft_id%253Dinfo%253Apmid%252F20832322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2010.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20832322&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284493600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.62" data-doi="10.1093/bioinformatics/btu391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utturkar</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klingeman</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doktycz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation and validation of <em>de novo</em> and hybrid assembly techniques to derive high-quality genome sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2709</span>–<span class="cit-lpage">2716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu391%26rft_id%253Dinfo%253Apmid%252F24930142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTkvMjcwOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.63" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouelliel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.64" data-doi="10.1038/ng.3121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weisenfeld</span>  <span class="cit-name-given-names">N. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegarty</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comprehensive variation discovery in single human genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1350</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1350%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3121%26rft_id%253Dinfo%253Apmid%252F25326702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25326702&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.65" data-doi="10.1186/s13059-015-0764-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wences</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Metassembler: merging and optimizing <em>de novo</em> genome assemblies.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0764-4%26rft_id%253Dinfo%253Apmid%252F26403281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0764-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26403281&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.11.3655.66" data-doi="10.1186/1471-2105-13-S4-S1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Exploiting sparseness in <em>de novo</em> genome assembly.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>(<span class="cit-supplement">Suppl. 6</span>): <span class="cit-fpage">S1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-S4-S1%26rft_id%253Dinfo%253Apmid%252F22536955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-S4-S1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22536955&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.11.3655.67" data-doi="10.1038/srep31900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Z. S</span>  <span class="cit-name-given-names">Ma</span></span></li></ol><cite>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">DBG2OLC: efficient assembly of large genomes using long erroneous reads of the third generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D31900%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep31900%26rft_id%253Dinfo%253Apmid%252F27573208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep31900&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27573208&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.68" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Velvet: algorithms for <em>de novo</em> short read assembly using <em>de Bruijn</em> graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.69" data-doi="10.1126/science.1251385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative genomics reveals insights into avian genome evolution and adaptation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">1311</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D346%26rft.issue%253D6215%26rft.spage%253D1311%26rft.epage%253D1320%26rft.atitle%253DComparative%2Bgenomics%2Breveals%2Binsights%2Binto%2Bavian%2Bgenome%2Bevolution%2Band%2Badaptation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251385%26rft_id%253Dinfo%253Apmid%252F25504712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDYvNjIxNS8xMzExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.70" data-doi="10.1111/mec.12680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prevention, diagnosis and treatment of high-throughput sequencing data pathologies.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1679</span>–<span class="cit-lpage">1700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D1679%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12680&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3655.71" data-doi="10.1093/bioinformatics/btt476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The MaSuRCA genome assembler.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2669</span>–<span class="cit-lpage">2677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt476%26rft_id%253Dinfo%253Apmid%252F23990416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMjEvMjY2OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3655.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3647" title="Simple and Efficient Targeting of Multiple Genes Through CRISPR-Cas9 in Physcomitrella patens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3663" title="Determinants in the LIN-12/Notch Intracellular Domain That Govern Its Activity and Stability During Caenorhabditis elegans Vulval Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3655" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GK5NxZiDZ4eG7y5k7H9qolrBcDCSS4_E1GYRG4t3pqU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3655" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3655" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3655" class="active" data-icon-position="" data-hide-link-title="0">In Silico Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of de novo Genome Sequencing Studies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460464" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qeXHaPQ1jLLWpgrbPtCzrxD9TguLJvhKjQHNN_gvXBI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3655.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3655" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jscdSg4-3xZYqgUIt_30nfsOZTU2p8WLV0PookSYtSs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3655" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3655.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460464" id="node460464--2" data-pisa="ggg;6/11/3655" data-pisa-master="ggg;g3.116.034249" data-apath="/ggg/6/11/3655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>In Silico</em> Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of <em>de novo</em> Genome Sequencing Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiaofan Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9912-8802" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Peris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacek Kominek</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2208-9814" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cletus P. Kurtzman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chris Todd Hittinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3655-3662;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034249 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460464--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiaofan Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaofan%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaofan%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiaofan Zhou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David Peris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BPeris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peris%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BPeris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9912-8802" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Peris</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacek Kominek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacek%2BKominek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kominek%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacek%2BKominek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacek Kominek</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cletus P. Kurtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, US Department of Agriculture, Peoria, Illinois 61604</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cletus%2BP.%2BKurtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurtzman%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACletus%2BP.%2BKurtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2208-9814" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cletus P. Kurtzman</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chris Todd Hittinger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a>
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a>
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3655" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3655&amp;title=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3655&amp;title=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3655&amp;title=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3655&amp;text=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3655&amp;title=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3655&amp;t=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3655&amp;title=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3655&amp;title=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460464/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460464" id="node460464--3" data-pisa="ggg;6/11/3655" data-pisa-master="ggg;g3.116.034249" data-apath="/ggg/6/11/3655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>In Silico</em> Whole Genome Sequencer and Analyzer (iWGS): a Computational Pipeline to Guide the Design and Analysis of <em>de novo</em> Genome Sequencing Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiaofan Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9912-8802" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Peris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacek Kominek</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2208-9814" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cletus P. Kurtzman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chris Todd Hittinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3655-3662;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034249 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460464--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiaofan Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaofan%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaofan%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiaofan Zhou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David Peris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BPeris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peris%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BPeris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9912-8802" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Peris</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacek Kominek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacek%2BKominek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kominek%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacek%2BKominek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacek Kominek</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cletus P. Kurtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, US Department of Agriculture, Peoria, Illinois 61604</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cletus%2BP.%2BKurtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurtzman%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACletus%2BP.%2BKurtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2208-9814" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cletus P. Kurtzman</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, Department of Energy Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chris Todd Hittinger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a>
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cthittinger@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cthittinger@wisc.edu</a>
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3655&amp;rft.epage=3662&amp;rft.atitle=In%20Silico%20Whole%20Genome%20Sequencer%20and%20Analyzer%20%28iWGS%29%3A%20a%20Computational%20Pipeline%20to%20Guide%20the%20Design%20and%20Analysis%20of%20de%20novo%20Genome%20Sequencing%20Studies&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhou%2C+Xiaofan&amp;rft.au=Peris%2C+David&amp;rft.au=Kominek%2C+Jacek&amp;rft.au=Kurtzman%2C+Cletus+P.&amp;rft.au=Hittinger%2C+Chris+Todd&amp;rft.au=Rokas%2C+Antonis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460464/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460464/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460464/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460464/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460464/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460464/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460464/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460464/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460464/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460464/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460464/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460464/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3655" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3655.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3655.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3655.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3655.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
