<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3485" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3485.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3485.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3485.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3485.atom" />
<meta name="HW.pisa" content="ggg;6/11/3485" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" />
<meta name="DC.Identifier" content="10.1534/g3.116.030411" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Sork et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Oak represents a valuable natural resource across Northern Hemisphere ecosystems, attracting a large research community studying its genetics, ecology, conservation, and management. Here we introduce a draft genome assembly of valley oak ( Quercus lobata ) using Illumina sequencing of adult leaf tissue of a tree found in an accessible, well-studied, natural southern California population. Our assembly includes a nuclear genome and a complete chloroplast genome, along with annotation of encoded genes. The assembly contains 94,394 scaffolds, totaling 1.17 Gb with 18,512 scaffolds of length 2 kb or longer, with a total length of 1.15 Gb, and a N50 scaffold size of 278,077 kb. The k -mer histograms indicate an diploid genome size of ∼720–730 Mb, which is smaller than the total length due to high heterozygosity, estimated at 1.25%. A comparison with a recently published European oak ( Q. robur ) nuclear sequence indicates 93% similarity. The Q. lobata chloroplast genome has 99% identity with another North American oak, Q. rubra . Preliminary annotation yielded an estimate of 61,773 predicted protein-coding genes, of which 71% had similarity to known protein domains. We searched 956 Benchmarking Universal Single-Copy Orthologs, and found 863 complete orthologs, of which 450 were present in &gt; 1 copy. We also examined an earlier version (v0.5) where duplicate haplotypes were removed to discover variants. These additional sources indicate that the predicted gene count in Version 1.0 is overestimated by 37–52%. Nonetheless, this first draft valley oak genome assembly represents a high-quality, well-annotated genome that provides a tool for forest restoration and management practices." />
<meta name="DC.Contributor" content="Victoria L. Sork" />
<meta name="DC.Contributor" content="Sorel T. Fitz-Gibbon" />
<meta name="DC.Contributor" content="Daniela Puiu" />
<meta name="DC.Contributor" content="Marc Crepeau" />
<meta name="DC.Contributor" content="Paul F. Gugger" />
<meta name="DC.Contributor" content="Rachel Sherman" />
<meta name="DC.Contributor" content="Kristian Stevens" />
<meta name="DC.Contributor" content="Charles H. Langley" />
<meta name="DC.Contributor" content="Matteo Pellegrini" />
<meta name="DC.Contributor" content="Steven L. Salzberg" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Oak represents a valuable natural resource across Northern Hemisphere ecosystems, attracting a large research community studying its genetics, ecology, conservation, and management. Here we introduce a draft genome assembly of valley oak (&lt;i&gt;Quercus lobata&lt;/i&gt;) using Illumina sequencing of adult leaf tissue of a tree found in an accessible, well-studied, natural southern California population. Our assembly includes a nuclear genome and a complete chloroplast genome, along with annotation of encoded genes. The assembly contains 94,394 scaffolds, totaling 1.17&amp;nbsp;Gb with 18,512 scaffolds of length 2&amp;nbsp;kb or longer, with a total length of 1.15&amp;nbsp;Gb, and a N50 scaffold size of 278,077&amp;nbsp;kb. The &lt;i&gt;k&lt;/i&gt;-mer histograms indicate an diploid genome size of ∼720–730 Mb, which is smaller than the total length due to high heterozygosity, estimated at 1.25%. A comparison with a recently published European oak (&lt;i&gt;Q. robur&lt;/i&gt;) nuclear sequence indicates 93% similarity. The &lt;i&gt;Q. lobata&lt;/i&gt; chloroplast genome has 99% identity with another North American oak, &lt;i&gt;Q. rubra&lt;/i&gt;. Preliminary annotation yielded an estimate of 61,773 predicted protein-coding genes, of which 71% had similarity to known protein domains. We searched 956 Benchmarking Universal Single-Copy Orthologs, and found 863 complete orthologs, of which 450 were present in &amp;gt;&amp;nbsp;1 copy. We also examined an earlier version (v0.5) where duplicate haplotypes were removed to discover variants. These additional sources indicate that the predicted gene count in Version 1.0 is overestimated by 37–52%. Nonetheless, this first draft valley oak genome assembly represents a high-quality, well-annotated genome that provides a tool for forest restoration and management practices.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3485" />
<meta name="citation_id" content="6/11/3485" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3485" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3485.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3485.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3485.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030411" />
<meta name="citation_pmid" content="27621377" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3485" />
<meta name="citation_lastpage" content="3495" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Victoria L. Sork" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Institute of the Environment and Sustainability, University of California, Los Angeles, California 90095" />
<meta name="citation_author_email" content="vlsork@ucla.edu" />
<meta name="citation_author" content="Sorel T. Fitz-Gibbon" />
<meta name="citation_author_institution" content="Institute of Genomics and Proteomics, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="Daniela Puiu" />
<meta name="citation_author_institution" content="Center for Computational Biology, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, Maryland 21205" />
<meta name="citation_author" content="Marc Crepeau" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Paul F. Gugger" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="University of Maryland Center for Environmental Science, Appalachian Laboratory, Frostburg, Maryland 21532" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4464-8453" />
<meta name="citation_author" content="Rachel Sherman" />
<meta name="citation_author_institution" content="Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218" />
<meta name="citation_author_institution" content="Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="Kristian Stevens" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Charles H. Langley" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Matteo Pellegrini" />
<meta name="citation_author_institution" content="Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="Steven L. Salzberg" />
<meta name="citation_author_institution" content="Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218" />
<meta name="citation_author_institution" content="Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland 21218" />
<meta name="citation_author_institution" content="Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland 21218" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8859-7432" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=S. N. Aitken;citation_author=M. C. Whitlock;citation_title=Assisted gene flow to facilitate local adaptation to climate change.;citation_pages=367;citation_volume=44;citation_year=2013;citation_issue=1;citation_doi=10.1146/annurev-ecolsys-110512-135747" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=S. N. Aitken;citation_author=S. Yeaman;citation_author=J. A. Holliday;citation_author=T. L. Wang;citation_author=S. Curtis-McLane;citation_title=Adaptation, migration or extirpation: climate change outcomes for tree populations.;citation_pages=95-111;citation_volume=1;citation_year=2008;citation_issue=1;citation_pmid=25567494;citation_doi=10.1111/j.1752-4571.2007.00013.x" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=L. W. Alexander;citation_author=K. E. Woeste;citation_title=Pyrosequencing of the northern red oak (Quercus rubra L.) chloroplast genome reveals high quality polymorphisms for population management.;citation_pages=803-812;citation_volume=10;citation_year=2014;citation_issue=4;citation_doi=10.1007/s11295-013-0681-1" />
<meta name="citation_reference" content="citation_title=Tending the Wild: Native American Knowledge and the Management of California’s Natural Resources.;citation_author=M. K. Anderson;citation_year=2005" />
<meta name="citation_reference" content="Aronesty, E., 2011 ea-utils: command-line tools for processing biological sequencing data. Available at: https://expressionanalysis.github.io/ea-utils/. Accessed: January 22, 2016." />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Fremontia;citation_author=W. M. Block;citation_author=M. M. Morrison;citation_author=J. Verner;citation_title=Wildlife and oak-woodland interdependency.;citation_pages=72-76;citation_volume=18;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. Boetzer;citation_author=W. Pirovano;citation_title=Toward almost closed genomes with GapFiller.;citation_pages=R56-R56;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22731987;citation_doi=10.1186/gb-2012-13-6-r56" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics;citation_author=M.S. Campbell;citation_author=C. Holt;citation_author=B. Moore;citation_author=M. Yandell;citation_title=Genome annotation and curation using MAKER and MAKER-P.;citation_pages=4111-14.11. 39;citation_volume=48;citation_year=2014;citation_pmid=25501943;citation_doi=10.1002/0471250953.bi0411s48" />
<meta name="citation_reference" content="citation_journal_title=Int. Oaks;citation_title=Diversity, distribution, and ecosystem services of the North American oaks.;citation_author=J. Cavender-Bares;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. J. Cokus;citation_author=P. F. Gugger;citation_author=V. L. Sork;citation_title=Evolutionary insights from de novo transcriptome assembly and SNP discovery in California white oaks.;citation_pages=552;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=26215102;citation_doi=10.1186/s12864-015-1761-4" />
<meta name="citation_reference" content="citation_journal_title=Biogeochemistry;citation_author=R. A. Dahlgren;citation_author=M. J. Singer;citation_author=X. Huang;citation_title=Oak tree and grazing impacts on soil properties and nutrients in a California oak woodland.;citation_pages=45-64;citation_volume=39;citation_year=1997;citation_issue=1;citation_doi=10.1023/A:1005812621312" />
<meta name="citation_reference" content="Delfino-Mix, A., J. W. Wright, P. F. Gugger, C. Liang, and V. L. Sork, 2015 Establishing a range-wide provenance test in valley oak (Quercus lobata Née) at two California sites, pp. 413–424 in 7th California Oak Symposium: Managing Oak Woodlands in a Dynamic World. Visalia, CA, edited by R. Standiford. USDA Forest Service, Berkeley, CA." />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. Derory;citation_author=P. Leger;citation_author=V. Garcia;citation_author=J. Schaeffer;citation_author=M. T. Hauser;citation_title=Transcriptome analysis of bud burst in sessile oak (Quercus petraea).;citation_pages=723-738;citation_volume=170;citation_year=2006;citation_issue=4;citation_pmid=16684234;citation_doi=10.1111/j.1469-8137.2006.01721.x" />
<meta name="citation_reference" content="citation_title=Riparian Buffers for Agricultural Land in Agroforestry Notes.;citation_author=M. Dosskey;citation_author=R. C. Schultz;citation_author=T. M. Isenhart;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. M. Eren;citation_author=J. H. Vineis;citation_author=H. G. Morrison;citation_author=M. L. Sogin;citation_title=A filtering method to generate high quality short reads using Illumina paired-end technology.;citation_pages=e66643;citation_volume=8;citation_year=2013;citation_issue=6;citation_pmid=23799126;citation_doi=10.1371/journal.pone.0066643" />
<meta name="citation_reference" content="citation_journal_title=BioResources;citation_author=O. Espinoza;citation_author=U. Buehlmann;citation_author=M. Bumgardner;citation_author=B. Smith;citation_title=Assessing changes in the US hardwood sawmill industry with a focus on markets and distribution.;citation_pages=2676-2689;citation_volume=6;citation_year=2011;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=A. Batemanc;citation_author=J. Clements;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_title=Pfam: the protein families database.;citation_volume=42;citation_year=2014;citation_pmid=24288371;citation_doi=10.1093/nar/gkt1223" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=D. Grivet;citation_author=J. J. Robledo-Arnuncio;citation_author=P. E. Smouse;citation_author=V. L. Sork;citation_title=Relative contribution of contemporary pollen and seed dispersal to the effective parental size of seedling population of California valley oak (Quercus lobata, Nee).;citation_pages=3967-3979;citation_volume=18;citation_year=2009;citation_issue=19;citation_pmid=19754515;citation_doi=10.1111/j.1365-294X.2009.04326.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. F. Gugger;citation_author=M. Ikegami;citation_author=V. L. Sork;citation_title=Influence of late quaternary climate change on present patterns of genetic variation in valley oak, Quercus lobata Née.;citation_pages=3598-3612;citation_volume=22;citation_year=2013;citation_issue=13;citation_doi=10.1111/mec.12317" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=P. F. Gugger;citation_author=S. J. Cokus;citation_author=M. Pellegrini;citation_author=V. L. Sork;citation_title=Association of transcriptome-wide sequence variation with climate gradients in valley oak (Quercus lobata).;citation_pages=15;citation_volume=12;citation_year=2016;citation_issue=2;citation_doi=10.1007/s11295-016-0975-1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. F. Gugger;citation_author=S. Fitz-Gibbon;citation_author=M. Pellegrini;citation_author=V. L. Sork;citation_title=Species-wide patterns of DNA methylation variation in Quercus lobata and their association with climate gradients.;citation_pages=1665-1680;citation_volume=25;citation_year=2016;citation_issue=8;citation_doi=10.1111/mec.13563" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Appl.;citation_author=D. J. Herman;citation_author=L. J. Halverson;citation_author=M. K. Firestone;citation_title=Nitrogen dynamics in an annual grassland: oak canopy, climate, and microbial population effects.;citation_pages=593-604;citation_volume=13;citation_year=2003;citation_issue=3;citation_doi=10.1890/1051-0761(2003)013[0593:NDIAAG]2.0.CO;2" />
<meta name="citation_reference" content="citation_title=Quercus lobata in Fire Effects Information System.;citation_author=J. L. Howard;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. D. Johnson;citation_author=R. E. Handsaker;citation_author=S. L. Pulit;citation_author=M. M. Nizzari;citation_author=C. J. O’Donnell;citation_title=SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap.;citation_volume=24;citation_year=2008;citation_pmid=18974171;citation_doi=10.1093/bioinformatics/btn564" />
<meta name="citation_reference" content="citation_title=Conservation Economics White Paper.;citation_title=An economic analysis of the benefits of habitat conservation on California rangelands;citation_author=T. Kroeger;citation_author=F. Casey;citation_author=P. Alvarez;citation_author=M. Cheatum;citation_author=L. Tavassoi;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=R. M. Kuhn;citation_author=D. Haussler;citation_author=W. J. Kent;citation_title=The UCSC genome browser and associated tools.;citation_volume=14;citation_year=2013;citation_pmid=22908213;citation_doi=10.1093/bib/bbs038" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=R12-R12;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14759262;citation_doi=10.1186/gb-2004-5-2-r12" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Law;citation_author=S. E. Jacobsen;citation_title=Establishing, maintaining and modifying DNA methylation patterns in plants and animals.;citation_pages=204-220;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20142834;citation_doi=10.1038/nrg2719" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=I. Lesur;citation_author=G. Le Provost;citation_author=P. Bento;citation_author=C. Da Silva;citation_author=J.-C. Leplé;citation_title=The oak gene expression atlas: insights into Fagaceae genome evolution and the discovery of genes regulated during bud dormancy release.;citation_pages=112;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=25765701;citation_doi=10.1186/s12864-015-1331-9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Mitochondrial DNA A DNA Mapp Seq Anal.;citation_author=S. Lu;citation_author=M. Hou;citation_author=F. K. Du;citation_author=J. Li;citation_author=K. Yin;citation_title=Complete chloroplast genome of the oriental white oak: Quercus aliena blume.;citation_pages=2802-2804;citation_volume=27;citation_year=2016;citation_pmid=26114324" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_issue=1;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=BioResources;citation_author=W. G. Luppold;citation_author=M. S. Bumgardner;citation_title=Factors influencing changes in U.S. hardwood log and lumber exports from 1990 to 2011.;citation_pages=1615-1624;citation_volume=8;citation_year=2013;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Marcais;citation_author=C. Kingsford;citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.;citation_volume=27;citation_year=2011;citation_issue=6;citation_pmid=21217122;citation_doi=10.1093/bioinformatics/btr011" />
<meta name="citation_reference" content="citation_journal_title=Restor. Ecol.;citation_author=J. K. McKay;citation_author=C. E. Christian;citation_author=S. Harrison;citation_author=K. J. Rice;citation_title=“How local is local?”—a review of practical and conceptual issues in the genetics of restoration.;citation_pages=432-440;citation_volume=13;citation_year=2005;citation_issue=3;citation_doi=10.1111/j.1526-100X.2005.00058.x" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Appl.;citation_author=C. I. Millar;citation_author=N. L. Stephenson;citation_author=S. L. Stephens;citation_title=Climate change and forests of the future: managing in the face of uncertainty.;citation_pages=2145-2151;citation_volume=17;citation_year=2007;citation_issue=8;citation_pmid=18213958;citation_doi=10.1890/06-1715.1" />
<meta name="citation_reference" content="citation_title=Oaks of California.;citation_author=B. M. Pavlik;citation_author=P. C. Muick;citation_author=S. G. Johnson;citation_author=M. Popper;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=For. Ecol. Manage.;citation_author=R. J. Petit;citation_author=S. Brewer;citation_author=S. Bordács;citation_author=K. Burg;citation_author=R. Cheddadi;citation_title=FOR ECOL MANAGE;citation_pages=49-74;citation_volume=156;citation_year=2002;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. J. Petit;citation_author=I. Aguinagalde;citation_author=J.-L. de Beaulieu;citation_author=C. Bittkau;citation_author=S. Brewer;citation_title=Glacial Refugia: Hotspots But Not Melting Pots of Genetic Diversity;citation_pages=1563-1565;citation_volume=300;citation_year=2003;citation_issue=5625;citation_pmid=12791991;citation_doi=10.1126/science.1083264;citation_georef=2003054719" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=R. J. Petit;citation_author=J. Carlson;citation_author=A. L. Curtu;citation_author=M.-L. Loustau;citation_author=C. Plomion;citation_title=Fagaceae trees as models to integrate ecology, evolution and genomics.;citation_pages=369-371;citation_volume=197;citation_year=2013;citation_issue=2;citation_pmid=23253330;citation_doi=10.1111/nph.12089" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=C. Plomion;citation_author=J.-M. Aury;citation_author=J. Amselem;citation_author=T. Alaeitabar;citation_author=V. Barbe;citation_title=Decoding the oak genome: public release of sequence data, assembly, annotation and publication strategies.;citation_pages=254-265;citation_volume=16;citation_year=2016;citation_issue=1;citation_pmid=25944057;citation_doi=10.1111/1755-0998.12425" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. A. Salamov;citation_author=V. V. Solovyev;citation_title=Ab initio Gene Finding in Drosophila Genomic DNA;citation_pages=516-522;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779491;citation_doi=10.1101/gr.10.4.516" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. W. Sayers;citation_author=T. Barrett;citation_author=D. A. Benson;citation_author=E. Bolton;citation_author=S. H. Bryant;citation_title=Database resources of the National Center for Biotechnology Information.;citation_volume=40;citation_year=2012;citation_issue=Database issue;citation_pmid=22140104;citation_doi=10.1093/nar/gkr1184" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_issue=19;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="Smit, A. F., R. Hubley, and P. Green, 1996–2010 RepeatMasker Open-3.0. Available at: http://www. repeatmasker.org. Accessed: July 17, 2016." />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. D. Sommer;citation_author=A. L. Delcher;citation_author=S. L. Salzberg;citation_author=M. Pop;citation_title=Minimus: a fast, lightweight genome assembler.;citation_pages=64-64;citation_volume=8;citation_year=2007;citation_pmid=17324286;citation_doi=10.1186/1471-2105-8-64" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=V. L. Sork;citation_author=F. W. Davis;citation_author=R. Westfall;citation_author=A. Flint;citation_author=M. Ikegami;citation_title=Gene movement and genetic association with regional climate gradients in California valley oak (Quercus lobata Nee) in the face of climate change.;citation_pages=3806-3823;citation_volume=19;citation_year=2010;citation_issue=17;citation_pmid=20723054;citation_doi=10.1111/j.1365-294X.2010.04726.x" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=V. L. Sork;citation_author=K. Squire;citation_author=P. F. Gugger;citation_author=S. E. Steele;citation_author=E. D. Levy;citation_title=Landscape genomic analysis of candidate genes for climate adaptation in a California endemic oak, Quercus lobata.;citation_volume=103;citation_year=2016;citation_pmid=26744482;citation_doi=10.3732/ajb.1500162" />
<meta name="citation_reference" content="citation_journal_title=Environ. Exp. Bot.;citation_author=N. Spiess;citation_author=M. Oufir;citation_author=I. Matusikova;citation_author=M. Stierschneider;citation_author=D. Kopecky;citation_title=Ecophysiological and transcriptomic responses of oak (Quercus robur) to long-term drought exposure and rewatering.;citation_pages=117-126;citation_volume=77;citation_year=2012;citation_doi=10.1016/j.envexpbot.2011.11.010" />
<meta name="citation_reference" content="citation_journal_title=J. Ecosyst. Manag.;citation_author=D. L. Spittlehouse;citation_author=R. B. Stewart;citation_title=Adaptation to climate change in forest management.;citation_pages=1-11;citation_volume=4;citation_year=2004;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=J. Range Manage.;citation_author=R. B. Standiford;citation_author=R. E. Howitt;citation_title=Multiple use management of California hardwood rangelands.;citation_pages=176-182;citation_volume=46;citation_year=1993;citation_issue=2;citation_doi=10.2307/4002277" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=R. Steinkamp;citation_author=S. Waack;citation_author=B. Morgenstern;citation_title=AUGUSTUS: a web server for gene finding in eukaryotes;citation_pages=W309-W312;citation_volume=32;citation_year=2004;citation_issue=suppl_2;citation_pmid=15215400;citation_doi=10.1093/nar/gkh379" />
<meta name="citation_reference" content="citation_author=T. J. Swiecki;citation_author=E. A. Bernhardt;citation_author=R. A. Arnold;citation_pages=543-552;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=UniProt: a hub for protein information.;citation_volume=43;citation_year=2015;citation_pmid=25348405;citation_doi=10.1093/nar/gku989" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. Tuskan;citation_author=S. Difazio;citation_author=S. Jansson;citation_author=J. Bohlmann;citation_author=I. Grigoriev;citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp;amp; Gray).;citation_pages=1596-1604;citation_volume=313;citation_year=2006;citation_issue=5793;citation_pmid=16973872;citation_doi=10.1126/science.1128691" />
<meta name="citation_reference" content="citation_journal_title=For. Ecol. Manage.;citation_author=B. von Lupke;citation_title=Silvicultural methods of oak regeneration with special respect to shade tolerant mixed species.;citation_pages=19-26;citation_volume=106;citation_year=1998;citation_issue=1;citation_doi=10.1016/S0378-1127(97)00235-1" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_issue=9;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. V. Zimin;citation_author=G. Marcais;citation_author=D. Puiu;citation_author=M. Roberts;citation_author=S. L. Salzberg;citation_title=The MaSuRCA genome assembler.;citation_volume=29;citation_year=2013;citation_issue=21;citation_pmid=23990416;citation_doi=10.1093/bioinformatics/btt476" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Zimin;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_author=A. Holtz-Morris;citation_author=M. Koriabine;citation_title=Sequencing and Assembly of the 22-Gb Loblolly Pine Genome;citation_pages=875-890;citation_volume=196;citation_year=2014;citation_issue=3;citation_pmid=24653210;citation_doi=10.1534/genetics.113.159715" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="Oak represents a valuable natural resource across Northern Hemisphere ecosystems, attracting a large research community studying its genetics, ecology, conservation, and management. Here we introduce a draft genome assembly of valley oak ( Quercus lobata ) using Illumina sequencing of adult leaf tissue of a tree found in an accessible, well-studied, natural southern California population. Our assembly includes a nuclear genome and a complete chloroplast genome, along with annotation of encoded genes. The assembly contains 94,394 scaffolds, totaling 1.17 Gb with 18,512 scaffolds of length 2 kb or longer, with a total length of 1.15 Gb, and a N50 scaffold size of 278,077 kb. The k -mer histograms indicate an diploid genome size of ∼720–730 Mb, which is smaller than the total length due to high heterozygosity, estimated at 1.25%. A comparison with a recently published European oak ( Q. robur ) nuclear sequence indicates 93% similarity. The Q. lobata chloroplast genome has 99% identity with another North American oak, Q. rubra . Preliminary annotation yielded an estimate of 61,773 predicted protein-coding genes, of which 71% had similarity to known protein domains. We searched 956 Benchmarking Universal Single-Copy Orthologs, and found 863 complete orthologs, of which 450 were present in &gt; 1 copy. We also examined an earlier version (v0.5) where duplicate haplotypes were removed to discover variants. These additional sources indicate that the predicted gene count in Version 1.0 is overestimated by 37–52%. Nonetheless, this first draft valley oak genome assembly represents a high-quality, well-annotated genome that provides a tool for forest restoration and management practices." />
<meta name="og-title" property="og:title" content="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3485" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Oak represents a valuable natural resource across Northern Hemisphere ecosystems, attracting a large research community studying its genetics, ecology, conservation, and management. Here we introduce a draft genome assembly of valley oak ( Quercus lobata ) using Illumina sequencing of adult leaf tissue of a tree found in an accessible, well-studied, natural southern California population. Our assembly includes a nuclear genome and a complete chloroplast genome, along with annotation of encoded genes. The assembly contains 94,394 scaffolds, totaling 1.17 Gb with 18,512 scaffolds of length 2 kb or longer, with a total length of 1.15 Gb, and a N50 scaffold size of 278,077 kb. The k -mer histograms indicate an diploid genome size of ∼720–730 Mb, which is smaller than the total length due to high heterozygosity, estimated at 1.25%. A comparison with a recently published European oak ( Q. robur ) nuclear sequence indicates 93% similarity. The Q. lobata chloroplast genome has 99% identity with another North American oak, Q. rubra . Preliminary annotation yielded an estimate of 61,773 predicted protein-coding genes, of which 71% had similarity to known protein domains. We searched 956 Benchmarking Universal Single-Copy Orthologs, and found 863 complete orthologs, of which 450 were present in &gt; 1 copy. We also examined an earlier version (v0.5) where duplicate haplotypes were removed to discover variants. These additional sources indicate that the predicted gene count in Version 1.0 is overestimated by 37–52%. Nonetheless, this first draft valley oak genome assembly represents a high-quality, well-annotated genome that provides a tool for forest restoration and management practices." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"sRc_X_3G0Xxuz7EQSHbZ7_mglKtl25Uh2TpsYlJihZA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460476","apath":"\/ggg\/6\/11\/3485.atom","pisa":"ggg;6\/11\/3485","ac":{"\/ggg\/6\/11\/3485.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3485.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3485"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3485"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3485":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220yJzoggl1nFKyptvSh3t1IVCsBP0eU2ZKtG57QSA4+s4zG6+I3D0cJmaNoL3w00n2pbipuOm9wegJOkJ5+NzdPtdxAdCmCSDX13BdlhTUzwFk5ZtkcsS0n9RpccWjL\\\\\\\/XMJpnenVmW16WKlckFVmZn8iyF9v6iltMvzv8Dyo9ncvo0XFpJEDhPMYtlS2lXDj2aRLCoUjx1Hs62WL\\\\\\\/x\\\\\\\/S9KqDVTI5t5qubofD7JhDVoMa\\\\\\\/He62cWy8JnkIX0lvJCHnYrfFhl1hdGRMnwXAouZhmlFW5Uao8EteovRYipbDZLi+joFVfBqGF2cSlLDSTgaJFYcAs7nMuz8EPtZPfXVnz2lCF8e3KhP4ZEbupu\\\\\\\/wD2AQLAyBhd5Mjh177NC7KIisldngk7QKWShBhUzNbL7t34gpV8QVY7tx31\\\\\\\/kHeXS5vqtECux86s8oIbBR7flu5lshgCvbtllxq1hILLm\\\\\\\/R\\\\\\\/puf6YzvW+5ORrW3eGIsFZgP+WyFaWnyPcMJS96JEmytrjTxijGgizZmEF2oCe1jovgpNgKEfyiwBxzriV2yq8QfELJDV\\\\\\\/ZCwUbMGGv0NoObAcBUbCpjWsiK1rKq5QzQqHh4Yj5C9rzZ4kFrSvZFX0HG9ZFcwJbw9w5flezJGFsZcnI3moiF+byo8jWhdIvpuA8S29ZWHDaL9ua1i5tInU3m7vJHhtkblj\\\\\\\/YSLvQ+tQnVySO1q8sYl4heNDVag5YM6DbNvGu+AN8FtrRtB0ZOJCQL+Bq7ArLm\\\\\\\/o2j9wtl+Ij46NPuzsbMR5t4mxtgO9sjpnjJOSkp+\\\\\\\/L+m3QDgPzjpjBP4WlZrf0cJQWu2CsAMw+fWrq5qH9nRxLIdH4lpC97u8cw4Oj2HY\\\\\\\/caqDUbinO9fMiPSBkt+asV72MNFmDyHrUqRr2o7\\\\\\\/VMIuAOo5GSDgRbkU0QJeoXb5IqNK9ehIGYXV3p3VKbfXe4NyLx3JpVXViHA1YnRg490myjpEpN5qFw6NI3It\\\\\\\/ayGbOS41GdoDG39zHkFAL73+UUiEZlt7z+WWeCPX3\\\\\\\/FIVpyRNZnUXyWIBDV\\\\\\\/0JGuZk9GYkUpb59mvLlqtRAk8Q1+Vpe3fCO2vUWtHHgyMtYtb1g2QGpzrw58lLbsJsZIGiN2RKL66kr64Cz52SjYk27+riAcdre6uCiMwlaZFjZkj1IBscNtOmZsFWG7ZAkoBqPq8XtPkd6JP8GyzK5ssdmON59KdN1IQvB9Z2ZK5CI7k3uVP+edL4tHdAvZTI6+gTed3WUmd4gsnovRXy28VYJC9IAAil0PAfhf4\\\u0022,\\\u0022iv\\\u0022:\\\u0022LvLxIwGirdqhuK5gBPklBQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b55d9586127af9c61204cb8949864c993e85c26571b17a31c62458b236d94708\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t2AS8yTQW3pjmrtdni9Y8V72qu+9Z73YBTFJrJR9+gaa651lTsd2FOoDNBHm1UXhkl4Rgr9CysyYwL0inm\\\\\\\/w2irxb5rG77\\\\\\\/2sKKwDvmv56UIPEG0RpsB2Ic+PBdkk\\\\\\\/lFxVsrsCLx\\\\\\\/FM+Oh9DlPh9Zgjk14vYMLkmk5yG7V\\\\\\\/Tpx2CbeG94wmKuGas4blLzq8oxtYUVSJXHVJQFGHVj4sLAM+hQ2bc5+r6iGyCVkoLPYew0EqGQBbj0ZMQJlzwHnP9oEE8JX5lb+9KHIvsHsnXsg0n2Fi2RhIYgu0AIfSI1GFEQ7gSKRVQGyJ0pkqUmEwep8vziKuPGj7xUGo1ONP9apab+\\\\\\\/o4SJIr\\\\\\\/7+PRcUkkLdOUUy05WhmKf3\\\\\\\/GQwOYczoQJ5ZQIQZZlJgtEjLY7DQwNVM\\\\\\\/Y0tLOyhFRGefC2q3TK12Em6TkOMS4BG9X8INcHOw\\\\\\\/xkzm7JRNSYZEL45kUxki4uodsgbF20ChKMbvt\\\\\\\/ENwH3vgchRnpGpK\\\\\\\/XLxHhluBU6j2lxFhFHbHDLkDrwVWx0V1SVeIkweg1AolZUuUtEhalTKHSc3Bgw8O8x41w1Nng+UFfkYOgBwcTbGFlWU2Q7Z0+UWF1ZsRzbw5XBbXB8rdxjhjDmz\\\\\\\/v2yLoJP46oGj4FhRC9VmOD7C6vjOse5H2JVkSv\\\\\\\/BIXHvlbm9Pmj\\\\\\\/62lmAwTTdQWBIdz1tV8EPj8jvC3HHYbpMeJHrMXYZlmtjN1\\\\\\\/+vp6UV803GpqdSQKtOeyhZvWUaECAhvhpTLnNvaQUay5R08L9kye8oSbPM26JcsClF+vNfoUW3egzM8f99iQIgFrLTt9WMYfpIJbRXzshKJn+OZ5wdLpu8\\\\\\\/YaEzYit+dotN4djtaiF9+C5U3OyXbpPKoTdgNUQq40eXzs\\\\\\\/YBoW2jSwKbFTysKcWHW71+WxiVSNkadzXeMi11mRdDKbeIrSCzU6YQoPglrAm\\\\\\\/k7cT27JRFdB3oUTIrrlMBrENDCSCpPdY3K3ViYUW0QzpGLauqOFWjlgw\\\\\\\/kfPevm+4vQ381O9wGLfG69ed4X5i\\\\\\\/SP\\\\\\\/SjcwxURzICTXzdU1831\\\\\\\/br74jY+R1FPq4rEs1vaOrlQ8o5Ijhus7+jvUXc8A8j5UDmZTMEZ9SjemdBuWTJW34uCzhhv+1YdSAB6VKRltCN1jA3YD5wgboRmlBNvP8v3i33mDiz9JFWR\\\\\\\/uE2fu8nXHaQCA5G1FwhGhkOZqnUrQj2XY1yvHAkoSFUgn+X1vWnj7I4ywEokppvAWJ5tBv1otkhIOYt7W7t2FwZwNjFkNbf8vILt4opQZX0XiaA2fEJhUWFZOx7KOl5RM34uCmX+cAbCFusd2zEuG0DuDjunnZtIX8k8kkeWKEi0yWS\\\\\\\/l06nswG6ziBl5ia\\\\\\\/nSZXL0I7A2oqP2a5008ajuWJAVyQAdpxtme1sNCX51RTxSB3kv\\\\\\\/fr3sc\\\\\\\/f8mRn0g4wCVcvKtEZZy+MUkWAo24P8X6\\\\\\\/gwhhe5ghKac4py7KhmXfBoXcJ\\\\\\\/xmYqSGbunTH+g3c0R61GWseqrHf01D1Ex8h+ZhtlFBiwlCFqPyL7JtJBOa5\\\\\\\/hURicAHvEewQAU9BY7TxC36UVW4NxeQkUnWdZVqiOU0T85EVC2rqhhKkijUfhtCPNet3\\\\\\\/pq94ag907mVbcNk7b7PByPlegM5cV3CMNgy3SNBDRCbZtoXGsJIbtd6UMUX8v5M\\\\\\\/owNPHtGB3wOUv6RbGd7MgnMr19eWS2BD\\\\\\\/MF8mvwKEjvb\\\\\\\/GEVdTKvTmcXI3zIPYF8MXjbFLw85QuC42PnZ6h67fIyaT9d4ollv7S4S9kZ74s1UhLv2zg9nOjH6igTI2+1U2nUk5lp5UmmAPivtTl+iB26l2WGPwP3VnrZdr9os\\\\\\\/kt85uxycj3RMkT0iQpf\\\\\\\/OwYuIt3TYJVFUn+0EqdGdeBpnewKRBcXcv3tdbRRwZNcApRFOaAtNqvL4rcV2k5j3JUFFdB7X4dvY5kAaDUeUcOdmN097hznNb8AfjTmyqDdX6LhWAEBCfgYlPhgjNd1l8tYMQx1ctqKyUqF8a0iWj9GyDB0mLSg5GZhGiRryPq2pB8zQR9JfTXsj+a9awWT1eIsbTq4JlEATp5u\\\\\\\/YcI\\\\\\\/YVQ3Q\\\\\\\/gs9J3+fQNdAR83k3TQdFMgKxlGljjrYp6Wh\\\\\\\/wdil3UM5aFd\\\\\\\/cGW+klNGGb8yYuVBx\\\\\\\/C8Re33\\\\\\\/MOD5FmM+Pc1zTO2qzvglnqgPZ2cH\\\\\\\/b0zMaWFUSx7MwClyjnMgmQXWGWrGlNoKogwzs4vNCQOJkDZzvSKRIKDlx4tCxHTNgKbr\\\\\\\/A8C4qXPrwVPQBaLQo\\\\\\\/sVBUcGM3naiCNWoUmJ\\\\\\\/bbEAuj7CGPF+RUs4Jmv+1zatXfkLZEO5ZfbUUY9MzDS7pmV+ylxeXu6CSZ5PlRPxQrntD37hsPrNviv51a87Vxljkc++YwObq19DfmlbvCwDP7\\\\\\\/p\\\\\\\/ZdBxIns4esdVr3APaVY63BC02amH8KdfQizwyMqdbiuBmTho3bwYftYmdT301LR0WJYp0QjrC7rekFaooF5w4Gly0zqdKep5tmbAmewQLsO\\\\\\\/xEixKwC6\\\\\\\/i6rr8XA2\\\\\\\/XT16nQ6Isdd1y6fz+EhWamiHdbDgupM5OVwn0OWtiKuMFEJ4nvlVo1UjMFmYQyHHnfphfMhlMtmXxpHizq\\\\\\\/8yRrHnK5p10B5QAkdPrsEu5Uqkdtv7lrTQ8z49YN7yG0VrnqyBH8nIjOVR6kMPi1mFgvMXUxDWTu\\\\\\\/TzbNLIlIWMvJMUMb2\\\\\\\/Wng4nYumAyqfYwly5u60F4Zs41+O4ufydts4+RmmBHS1RJm74NQHJ96dY7sM2oUSk\\\\\\\/t9iKBEM7j6Z9TXnwGoS9OqmxXTGpNwsfY7E3sUG6DvBtbOfV+l\\\\\\\/fSCkhN7wHqQEHYA5qlSrYjarDUg7xeLeK+ddakxYMFUkxOMw7M9wcCWWKLArmn46RQXvX8\\\\\\\/ZJt9gMZmYfVvy9SjAwWr08LytsvpkN4EHDrecLXP9v7tLMnKYjyG97NSIFUSkCKm9WsjSsPyFEHyq9BJwRZUVpZX33G30p+kC7Kkb2Zas0LVvCZJPEl17NADCbo7qMOTvCaBCI7OitKJwyXPWkBt\\\\\\\/QXJUxjR6lyOAmq5UkA48B1g1h8dLGZOae8vhN6J0i9q9MrUPrW+LoY+aqko5ymi4HcG\\\\\\\/Gs5hbGnSnGbBZKBPESfmoS32Sx0OZQo2omdZRd7revl2hyT37GVjjPXVubgdHGy5pE+QLswpo46\\\\\\\/XyhDPiiRvlfP89xEcPLwmdK6hpzfNoHmGH1JPYlzB3tWLfijPPdIVHlnh19QFcHTo7l6HNdf1eHrY8pUXmDpln54\\\\\\\/REXNz09ghwW7+nBkqVbc1ZCLXQmVsnE5y1PvvHXwbJTZZfH\\\\\\\/dm1awpPlTK+lIbAeat+2oNEqcgvudJs4k1i3ThpS4xsWxf\\\\\\\/pcpNWGQFTTZqYnBNcGoiSgBNGhF+oJZnTvEteazIUfZ1UmR5rALemD0JLutTyDmrVFVvP5pBYYCsk1k1kxpemqC1l3noNeuZMYPDTINS0WMligWUPrGp2CfE7dKKfpyACbzrEkT1\\\\\\\/hcf1ryvCbgUwBPtgCInQhD5KSaJKXFynyZpez\\\\\\\/EU2JLkshMYMvY0rCfPr08t7rwwN7ff7PDzgsAD5gLbNKs\\\\\\\/AsizMLm0pHOGy9LydY9dymGzy9JcP3jde1ZI5bOPjHgr35a3MzfN41HCeggQ9XtBrpRV1njmU6BvN7FWsmZeaCmeoVp4bGtVXfKEjlr5BdmWRyTO6yBvI9fUdnecOdML1NUgrBuHT3JBcJPemal+4B+dHAJJLzvl4BmPulNSqx26+gcjXJxDWY9f5CnpNC920nr7btO5wUZfSlb1rqiCvcRM0azCsehTZAsdyJONp57aIwNIY9fFybBUHVxHWHhKtYiKSeG069dn6CaOc3C6NlCs4EpOLGhKyH8Z++BMB+wIvMjJX5URFy8ks6tbWe+SJV+\\\\\\\/2xZ6\\\\\\\/jn2jeGPUGG2y35JM4Kgqr48XTKl2O3b8qysnSVrFqeHUadvo3EOm9orQAB\\\\\\\/jnFOVmf2vrOhyUe1bs28rX4XCsFQngxRLCJO\\\\\\\/9CL4XD6uD9JtJ4IJeIdufsOcSu8QrCB+OTJaxCL4WiOa+uv4l7mPSdyLl7IBUMZtWo0cATqJYsqPvGWNm2UaFsZqyHzKXl5Yz7FZDarWxe3Dg\\\\\\\/JFtc4xn0wzdeom4S41LQhAOr8fymLPHDBnVc\\\\\\\/NZ5N8NdFosa+BYewD4\\\\\\\/yEM8B2fbdNOA7EkGRQzavpm4VBup1AdTbWSNFuxI2NElaZ4fJ3Qp2\\\\\\\/\\\\\\\/wQo\\\\\\\/Aqbf\\\\\\\/rfen90MAVinUNJ2eQCgttnweSghC6s+e2tmB1yUtqrloEo0zmlcJAq7qVwUTafaTNeOXWTHJlXilRBlDx4quqjXvx0sGF1zvqM0bOr9iyaoyHqTsgyrS5n110cGORhBXvj8g5x0E+FQ39bslUA4kQohAJbMc\\\\\\\/8zEKmvJUXT57E+MltsLNUCsrDzfdusQQodeYk0GRwG8MR+UbNV5txVJEb2mWYkkrP8rJu6NZg8MdkFCR5HxQDjlKBJfX79ZMJy5hm057PpFnyDWtSoaKl6tDG8l6KUFEVMwkKZ8Qi9tngGytRvDqqBBi+f2kW5mzC0Z9cANAIQQL5UECCA4R2pqxM9VfdiCsMWSmWPYXz+jgJO6BwkMw2evG9RlcfUX7nhwHustLJNwZ1RT605OUBRQFzGpjTfGmkx0Z03E8IRbSvhhMnMhFumn2tQjJ4gBw49\\\\\\\/3IpSao4l4gmAUhZwtvLlvmSj4OLFUesxVA5nTYnqQxVFa38WRLEFKgJEeTXpqFvXFRrij4b8toLkR0GHfRfMGPuPyC3Gcrvy\\\\\\\/7XwCjaXOm\\\\\\\/E\\\\\\\/yJQ\\\\\\\/aLtBMaRxrxXNplQZJvtJwtNv6R+mpsE5OQUzDKPQDnnyk4bgwIzYAwN6j093KV6ID8GQzlzjSZFIADv1AGoefS5w68acGfqVPIxJUC+s40YpdLnP7JBN3y4n0RqHM8JBcqYTJB6ihori0YBFPSY7kjn\\\\\\\/EBgDi5fSm5Ylb75U+wQwfBE78TnFCp+7eQyIvoKF+IhxJyPcXZ86r2eSa5QlTb2cY3JdncLj62L+jHp+Dpem0cOwZrY2sd4jiqBtu\\\\\\\/b5u8Ra2NTowbWrFMZKHUbWXPOPUUbMeha\\\\\\\/qmFxfn2LrRTpJjBXcnjuQT3bIka7MP6PJujWkZuYzWT6K8E8NV9lN0aR9ODT9PGnaYiwXNfX5tyGCKEYIxvgwG1PuN5myEk79tIvNC9jLUtwZ8Lt1fgkRgP5VkLCFgO4887t4rZah+FW9vvk2hIYm7ThqaNc6gCNS3oSEzvEJn1tLjioxQldQDHjYS7ZHrXantDnRJctzwRvrCdjOfg4G1XIwgxOJERZooS2uSRzJrhk9inO1lvt1J9K6fERuB5lv8VtJyGS52GA8ExAVaypOLz8jRooLx8EBk9AXtRn3AO\\\\\\\/CE2jbG+gqnimGhQQ4SYHHsjhKOdZ1S39UZB39ZkiFYvsp9BgYWbUEEim+QOKnDLJWdj\\\\\\\/z7tHYQWV22EwJn6trH1mureQzZYARcph2ixYpxxfhk2hk9rORyTdGqDnzlEoD70OJ4PuuzujzAwKqNuQ6tgjU\\\\\\\/D5PeGa9O1Oe+5MCX\\\\\\\/RQMOdJW9+vFykOK\\\\\\\/fm6UMFvrj9Eiy0cYUYYVptMj6pv7TMuZvumSXYqJK0lt6nLqLPmSvz1kl3byRLJOgrMm7gIP+DY4r5cl+Va1tYRRQJUOpUrBfIDLEjynpYYrwe5BIfMVGWrx57xeyzgHiUYVLa3IBFfUk3T6IfBJA5KNw6Cx96RzRaAAyXuzf9d6iogJQdH6BUF3MUmZ+LnPh+PJVSDQCskD0qtq\\\\\\\/GCFWviuG9wnwU0iB4joKd4zy7gwsBfD9pggYzNquBCcMYcCdMQD6QZGPFO3TchdHamzR+1wZaHM9wXdNS\\\\\\\/QFTXAMHqvgdUXpiDG713wApivCnBqXwQe1OzcNAhZtcfWnI+r3NadJcchXVYbXOfyqUbOwAhCXJ\\\\\\\/5BDHatsbxW5EE+8yK0iMaJJkhvODH7l509PpGpIs+s38e3Lr2UwOdae8fMVjphCwhCz4wuhmhxv6NlFIxw9y0hWQcV7Cw3zrb8D5P0hcblF36uWMpGelfX\\\\\\\/e1nfVC\\\\\\\/mxD58AsR4eo9OGIQ5mmEA3TaIggzffrRIddCf\\\\\\\/EtkW4sy9IVMJfBRS39HygV9P1oPBrvwEspD1ieADsQsnOSLA3pS4Lrj6ipR7+l76DAlXnNqrKb8y1zzaqA473AMnN59BTNqJaRX3uhNZwbYgZnqhdxcKKwiMBlxIhA9dfTHsI0i4i6sAiOyyshXm952uIO45gSdb34bV2Ry0Ix+RAP+scIX6AMGlbjXqCXp0KB0Y4EFFMkWOhJnzWGPxHTzHZMCM2Milmlb8wg+hhHo5LFMP02RGtWkrW4ZkT3+xuJZ8D32WRkejrpdujUoaMwh79qIfE7bvQGvGgDrwDwUznynMuNyvPwtPhtuYrXGX1TTDD\\\\\\\/wUxg0zG3ka4VFRIvPGVTfX0FGHl7txqUPJVLkGvjgEXcOUwHe6tpHesaXqPBAgUlEZnrYvQiTCyDffQ8AZJrgmxuAU6Pd6MS9jhuojxdnEIJlggf8HkzG\\\\\\\/ouekhygDkgFL0Rf+zHrIFZBgw5Q3asWCHITBzlWctsfWfmes4eVnKkY03DX9G7a+\\\\\\\/zx+wdsWJx5SoztrTXQTs+CbDZB8F4wmx6pmuxQGRrU1clrXPhFu7yfhRbaUDPdxRNSPWpoDwbe7LTjNQjea1P55G4TbKFr\\\\\\\/N82Gi2Ljc20x\\\\\\\/axXfoOz4sFuOscYm0dVixOSyLM7QgAu6VHPzolVeVUuBGxA1yvkdrthEwGimvyRDYVfTSoY0I6E\\\\\\\/h28RqDt6xS1sdRQ3UalZ+Q2GNz45U1f1k59Xyl2RmtEWbzDI26D2hDMjrVxWJ9HX0z6WcJefm4FpoyrZ\\\\\\\/RfTmOMo0Na2u3juiTkn+4ztR1wBNDrfcFiDACEQH\\\\\\\/Jw+uQQrx3LRWoON7IIuDdERaeEDYgXgjwhkJMJyS7SM0YuAB10JS1eTKBjW9DBjHG0998YRQ4muE5tY1Hwn883\\\\\\\/MNsaD4YQd9oZuwdCMQcuM3HbOliTo8B8AckK\\\\\\\/B21h6ruJV\\\\\\\/b4lRW4RnSopWr2vGSje\\\\\\\/OXhAvs1ZnFmFGRtpVmIlQel6jhmSyMAsf0JWsLbcZ+lWnL96nQ5dhea\\\\\\\/658Wis6HBGN2tSjpNycuWYFX4qrcXG+EiSXAcjxPHEcjbb4jlzYOlwUid54CuFrPmdNDcSqMGKhT2Zb+PQN71SDSGaR+Zvs6efC+pcpJTa5B22chVh8Hj6nthudUKBgpbpzGir4yxLfDYaMuE99OtGcPm6YTbY+2n2rebqZPUeR5dpAbLeEvZNnWKa6R2x\\\\\\\/5ajpZXiWxpSAv8f3Ocxy4RETpEm6sEjQmXC5FybrK3y8FVEimMTL2hDU0bi7jApbs0i2c2tuhvZzyJNr8LKCdEROhH7AWxR3bG3JrPQ6StSN\\\\\\\/xvX7OpDXlD2JqLsXa2DU5wMGiNwIwX865kypj+lzEwC3CqPCmbv9n9X+mu38j\\\\\\\/SuhPsKz+NdgsL\\\\\\\/9FW3odhIqnTpF7stirCX8Kav64kKPAyxA14Vcjt0KCURSM0inHNtlGq1wUmAmlXxQXxTgIj7Cf59p5JeXhytev9U4QbBtl0JXN7H\\\\\\\/\\\\\\\/bedUtPNHuFEUv219snjnyERsaEyaZiTrQdjy+UYz1JnzCAM4Oaegyu9hx0BvU4mB\\\\\\\/7gfGxiLZkOE7UTs9WM\\\\\\\/Dn0YCsyQ49QZbG5uADDPx+18GQBhCYL3qCGBjbXqFIyvEOMFhiFbp6l26wJUCkUi0S2f6skjKEBFanI\\\\\\\/lwJ4D8D4WoyMRPrgK+0+nzHP+elDuD10uMBy45taPL8ErD39lk8FcKH4HY3WKqTh+HV+RN5uNQOoHSavqRV3kHj106X8wYHAdoMmjwssGNknws3sqtLzjOirDw6K2AJABWqp2jAmBoHsYqlgk0Yu\\\\\\\/6NN1ooDXT4klfxIacsnQagdJc5ozeStEwtAfYaPDZ2e+e\\\\\\\/ERIXoR8Ou6yHsnXEWTI\\\\\\\/VzBVYGzF0iBtMtgidVHnbQ4TOP8FLckUB+pCbFbRSnfdkVKCxIlrMtsJHeI9ifK+VsGU28P1XJt9IFm50sPWHZxhYePPoQzLSD8INNZoOq2n\\\\\\\/AXFFLrDFOC9WettGQKBVEHILuJmxb7HbWZCvrwINSMEbxfjytQ\\\\\\\/K99H8tLyFEWl3uOfZAHV362d6liFExk1QcVmF7uSxn2wCSnQznoDTT5M6Ol1maiswUrYixxqbKCJCrWOgnmgOfCd9FjmtE06EZK6MjTTPHJ3v9l+S5JM2hjSdu14w5qvAT+z1pwFam6y1by4eM2L\\\\\\\/ucZbaGhTdeixZIaTl7XbM3XTmP+d+YQycO7xBDbLIr7lkgY7Djtfno+OjhTHbqruXacvBTCwexYYpi8nwDr1kFYJYZZ5ezJ3dak7x16zxq19skThmyGLU7ntb\\\\\\\/XEBD03oPhaU6y7vJ\\\\\\\/vPKD6uQ7rtJUIXGqUpk4dvMIgoasaih0UMP82qrYjlKduy7c3DXwesGGxGWlk9k\\\\\\\/ybO37Swwph3ORJMIWATROMQWN\\\\\\\/3ReLpZshq5YJUNYNHHKvke4UwUjmAHfLB6+lvUmlNslQ7DOCOz46h0RQmWbogwJmIg\\\\\\\/\\\\\\\/Q2v4V1qdQ\\\\\\\/P3O2qHTxti7VT3wduGU5B6qUDuaXpYJEarJBmu1Wb65qnnIlHKiAE3nvqqe7TsPwB3Dls1Q0DCnbra+rxbWTTy7FYpSNoFyl4QCygsbZcNH65vSsyMOC4yqx4YYfvGMk\\\\\\\/OdVwt3n6YlT\\\\\\\/8UYyR80mziLZVIPfJL5Fm\\\\\\\/WVdtV8i+Rs9Txhkmh959UwSbHonv1Csd9PfVwPihJjyIvGLP9m6D3weKWsmDaVUjbKoaNE8sxbDJiRP+MVc7FUjF9s2yMs0u2DhNMOjFGPZ4Jnkd0CHtoo2IiWXlYg+BzFxQ8rMIdUKSg6D1ki96ydDQvKPBZZvXWEMUN9lYd7IiresrxWo3LwbC8GLJyLcLmCClpC4e0conPPE7JI6kOjNaHyOvHRYrJIbTp6ACPEXc9jdjEztd49IWRmB5+Zde1EE7jHl9pGZ93SaKkUGOpcBqZfZznSqav+JU8GtLcB9YdeHM9p2n5YzZjnl5\\\\\\\/\\\\\\\/HMNjyALiK4u6FRldIsLnjulzQFOF1LyP49RlpVejRuavUpsBrTfCBPTcvb8tKieykR2ucWHRNbWTRijliDs4GWa+3Li24hANbuly0oUIrEoFv8k5NDRFSJ2KuRqBrZzST\\\\\\\/84c7iLEVCg9dccTY\\\\\\\/KqhslnJO+dqAMFu6ScO6R+jw3nHbRP8PH7QXYsLphNpjVlTBl0nv7yv0QiuZplSMYbdFEOeJan6BMBzV1Uki6icjqA1yuYxuXrsGWydTHvxIc5TFWvXHwfKI2v+BUpu7GoE3LP9Rk1lFhpwSIdh91C6N2tO4SNHH\\\\\\\/MP8w74JmI00gV41k3P+OZC8nXaMxqb6ojw1cu0ywd+DZDaiC8lwQuwf\\\\\\\/\\\\\\\/xP6s1HQVlWlkYARpXtXqKgIpSW5BjdbRpNmdZTIBGJXubN1sH\\\\\\\/3TQFLPUNkDkAnJh\\\\\\\/EPi\\\\\\\/ODXRdufo2SSEDseG8MQ0nuwWhk2p1YxUW\\\\\\\/wSmcWWMAEI0wLk9VH4Pv4dDx7lr0jIg36OpiIuvJHf6aJVGeU\\\\\\\/YtSeuVWwN2bbTY+jGTS3K2jdHzUYZAcIEBnYcWsPUA8oCqDzKgqvk\\\\\\\/VmFR7mhiiqrAQRCFo7rP0aye3AZy4018reI27YQR\\\\\\\/5T0AKu1rpITISpWnczzhu86tkgFxKyRRGpi8KACCpKkv\\\\\\\/bSrDZbuVAyMrErRRU\\\\\\\/pqwUQTarAm6caBt4jTf2Qcu3Lf8oVeZT38onh4BRJesqn11KFXu780dhe+Yhi00WcYmE1pEK70xADPPrDII3BJxkl9QKwZWxvUo8xJlF3jxFRqykMM1\\\\\\\/oCl8jmBSZkRjnA\\\\\\\/Qhtvts4a2Vrc0hcq1dKxgd4YtkypQyWkxn0L5yJzhXAorOcViRcyR5l+fOUBwC8e4F5wl3aRJ6lJycmI\\\\\\\/uikmpnKObBRuWalizSO\\\\\\\/Wda1XqmYqCJNYVu\\\\\\\/okxM6FjhwmVshTq8M0rcTVscsCwKkTcrO+VNUk1STTk549A52fHVxYhD5jjTUU1ucgwFpUCBLmUFtR1fhHozLr5e3N\\\\\\\/fA5JG+XzBsW9lVaNPMcNHvO8hZCKuRsc29wB7mmDrp6dOHt0Zy0dy\\\\\\\/4LkOgosbHc5L08OveD+FP56Qpo+kmEOF7TiL76fJruGwKJ4py8Ltn8zh6DQE0vgQikoxDfsq84O14BA7yfthdbQvYT9LYi35l33zjT2dyhcpJnwjMcUw2+5RUPNm2NkBZt1Goo+2XlOeieslewfLprW61eCmPLIqDFpmGHpYmODdSEvobbriiymMnxVNwtkP9nBtnW7LYdUT1TFd5wsQhC4dr9PfuToBs+QUaRcZwSALXZV+QHqn4ERA6vF25pzmPxhcpvKj3QwYxrHs28dmPkuhBGCeOu0kGJPTVf0LZmvvb4PBxo270NLDHAhpGTzVMVZnqbgzzxrUjBxYL5rbyl4gagXNm0zpfDTwSLh6EZcKVThgadIeNVKKCTrNGyhwk1HI3jAQpfjd9BGnsruUou4M2lTaiBrmdyVfJz+\\\\\\\/UcxhpHit0f1JsXB2N5Ejq95AbW1iu9Jjp5GTZWfWO6eJGAlKU50R1IauXyd6D2B4fW5XFL962ykQMHk\\\\\\\/MoIjRYvXHskPJrGHAnbuYRJ5VRqUnBo7sUj+WEYsn+N+BbT4aXrXgEmsFsKSDExTcdtHiVR+T6ivoqulNyXVT3ly+G5+Qd30rPy6UJ6CLzSaLi+tp8rK92e4YHJtHY5Ooaaq+vKYM7htHUjbAtj\\\\\\\/fOS7gbYkJBTC6hUI3JsGpkiUBO\\\\\\\/RCVi7VZcYNvDqbNKYYk3zZIz8jIEIatfnm1z1jUuj2l\\\\\\\/5C7ZccoNaJp7pUkmoyizKkMTeYOGDCHSOwFinLbLhFegoo9poYP2VHUGBQqNmYXE+i0xMHJD4ImQk8B94lNE89mqA2wE\\\\\\\/Oy+Y\\\\\\\/q3fdabLhS51aRqpqF2GiGc7In7H7PH7a8usZt+K83k8D5QdmdqwaUuMmDFeUkjKQmW7w\\\\\\\/J1At+winAS5Ifz929ZvFih9gXS86mcpgwJnl6CqjLNfwht2dDD\\\\\\\/Jjn\\\\\\\/G0n+09UcJN4khfmFoAFsOPxZog8PWeY5FdFJ9Qp6WEBNYJaVGfAzm+c0Z5exugTG6E+ye8wsy1UZv3ROGSbMD49y7+4mdLgSFHoMrEqAyWFcjHo6F9hpJzEKH270Z7TPXC55s\\\\\\\/YnXah17k04WWUf9dQ4nBlsbGnD0VR972PMErTIOahshII7IoJ74BPNot6deARJcHlTooH3IB+809YG3Njt30ciJxuoYAsbmFKqgE+MMRsjTCWIYqFbxcBiFVphIa2VIJU2b0i3i96QvAkrevwfztdhMfNlPdSX9ATd11e7bXprPFyaATdHL8bT3gJJjAVls+69SBY\\\\\\\/K9MaTgNTjRxJOALjpmXfkzwTKUx65\\\\\\\/VygkKvmH573blHpj8kDJ\\\\\\\/+U\\\\\\\/94TbKrc3gG7IoDieitfotm11chDtf6mOX9u0NDmGEO0Zi4z7YbQ8qYVKNY6lx+ZPwGj\\\\\\\/sy+XNtm\\\\\\\/lYrq7YykuWMikmP6jkr8CtTB9kUgrrUhL\\\\\\\/jw4UJRShmrPbbjBoHDvI4ngaV5XmNzTyMWY\\\\\\\/1ei87wsuDEkxU4UXhPYkYnoswrWDAFrTTrCpVm5bTr7Tr2\\\\\\\/n89RtgCQEh2McvQ7HbPWp2Yd\\\\\\\/h5Hv9PvIJQlopq3tZ8Ei2xfB8llLK2r28RWmw93kuC8w0PbX9NsLZ4HoweYFc05eUuHtEfTeyvSDaGzS3qxp0bU9Ur9k\\\\\\\/T4SqX+rapLkVRTw7Mz80hTpkc7pWj3tw8bTe41lZwyjpdXR3StaRIuA9KoSWsPM+P1Qppsr8Dg4uER0WrS+iFoOe16pDfEyUVxuROP6Eh569GiUe+l6\\\\\\\/D1MnqOj+dZI9nWF7mRtpLrKed7VXSUODZqkxLlziQ2Lne4suBKl9LOtk8V6eV5k2fNJMtKB6AKxm+KW40Laz+xAJAF3HplwsNZ0FWoT6dklqWNBbMbI6qt942GwGa5EGGYAD+HQ+iUvCoVF39sBIsbHWc1hDBCV8PPLr229vkKl3cfXvpDCvL4j5JpCB0nvPMWgmextdl4uiUaY9a5VmT9XSWDKKzB5KYLCfvyVyBi7iyFlPW4XHJS2Hxs5W9PI8QbWyhHFMJ9wUDi2Adq6czJQNY4MLC2\\\\\\\/uzmarz2HWXh6WYg7UeXHVZcoEdGphrgXvkXEo+rO5NDqkBbTq5n2MEUa20Z5z1amAQ548O+TGXYTFW6J6D5WBCGCAPEWnuNeEqAKFjaQWblfSBiPOS377dHvSxM5b9gXvaaNc+1t\\\\\\\/lwWp+GmeojrDYh5MrIkM1vW5RsZiAxXFAx2T2BHfUvkS3x9oSqEsHBKIrcW0GvHgON3kxroitxRHPNyfy4ks8EYl6y21WACiO3+8EMXp2sGwq3txv43Ng+t6u0DT\\\\\\\/eyVBtK8nRR9wOiHntpvQLpbxt0UuyEThVw4PfSvlZftcNF3i\\\\\\\/GUfkmitVvR+jd58XP8blO9IyOgZAFM1aFIRbFvgcXGY1LORxRyt+7VP1X7bDOBPWv5qMHD+TT6kavE20VZoeBewo9asLEF+vNvdY5Q+S8vFY2GWkExYxVttqTROnyMCMdEE84lTFun+8MHeQkag4fqU5m+DfSEMNRQnH\\\\\\\/XsVbzRfD\\\\\\\/L0i2sBAErOgTrHoptBVP2GeG5s4Quda7FzcjaJVTed8rmKNzAfpJizckvIj2mYWd69nEW3xnW9w\\\\\\\/qU3dNGHWrhfbTCsDGN+2lrC5nlGyl69C8gSIAeatWcjjcMK+WDDx\\\\\\\/genpqcNzoNIOcK+sadFGP0OUGPqjuZ2oWu5whiICWBSML0vX7f8nXcJh0ZXlxuM6FwqZWu5AXZDKHY2WaBqFga55tYIEK8vaovaf\\\\\\\/zouuayv8t9UsJrf4JzafbBImpBChZQTyyN96fo4IImsMDqfYgPOr8YUpfYBorR2Tf632yRNsZ2OU2nHeEadOBWrnWrnvWh6AklPUAof2D2QtJWjjyw7svtee6eli8DPH7vWRm3jUP09X3u+uSWwFOJ3RQvIVPddD2s4qhjbBQzBJ1R8naZNXf6hhitVZphgFeVwN2YxQmS\\\\\\\/a\\\\\\\/uEdyE85fSkFEjq194bYKu9y\\\\\\\/u4qCWx2e58qplVRVPa6U8XhRvCU74KwKtPy0oIQh+UHKDf4lcOhc\\\\\\\/d\\\\\\\/7VcGsNt5tYCeR\\\\\\\/93qQjBz7Q2oNJK2X+6ZyPj7rxvV9BeQyUB+EvdKipaDJlRDPgPjxwNFu\\\\\\\/\\\\\\\/HiBKa3rO7+tOVxS\\\\\\\/Ez4ayPIVZn0soxEaxC4bwh90jhDg+eurJlUABzR1zfUh3tMcfIhVdJNBtB+U+lX37NQd2l\\\\\\\/d6++CC4KLZLmZQOieeVXhCuNTiVAW9VNgfRy8zsYCro55iWlMNMBfYlkdtb00C+LJ+dJhuaGC4XDA+kE3KFy3T2ayurjV691Th2IZyVQyz2iCKlMUSlOKcN8NJf3cbpSnXsL5aCmIbqV+dL\\\\\\\/1Hae1FMXBqfPKuC6h3LnaunP3jsFK9GY8IePc2GWJJO+VeRHfElcFFJtmlMvnyrLuJQHIr0ZlEYJz67IUXfulGJkqlJjBilpSbi92X1sxE2TwArp1IUlhKhcmRu\\\\\\\/+EKvCEfSDG6gQqNzPe2cI0C4p4MKE\\\\\\\/Wc0ENhgl2+0VZO6Oqsf6WHfUzktxWboeHOo8j9pbSSBwByILhofv6jKUikT4nmkR1K31yDQQUvPOOwa\\\\\\\/HDTk5RP8VWoElvnHdUuZjdNVVihihhrdBE3BabHmgmAedOri3WoM7kNaBjMV1htwa2QiCYzIVuj+EI+N6toUh7ahoOgOesejenOw6a8CyFRxAeZUS\\\\\\\/VNldfw79Hzo4bbrdgkmYbXBM7qSUqgVhImPayxNDaewPTc5kCKRghqzPZG\\\\\\\/zDAecFfH9xiO995p\\\\\\\/RbM3xwopT1mzsaHdd7FxUdZLgbOR9EWxank85HJgk3JtXES2QE6Mky7RxqpzAJuPAQFhSznpUc+iFMpJ0wOnXkqPwKIFez0oa3riwK3Y5Hx+yHrbOGGk\\\\\\\/r5IgdodNkq2P3XXoAEK6Gr5yEC5yOM6iiF6xLggict4ifBZEbwQhQJdrrHWHATQQ+n6GY+kBbg8ofVwhv6bK2gil4FOrXyygNrAsunKgI+UAImT2VBCE+jRiWeHO3cE\\\\\\\/OGqsqFW\\\\\\\/pWJnY\\\\\\\/+dpEhtSnswjrHlOAyPQb\\\\\\\/wGGNHVVYZvM9D4iXtdWB66OQBvbKYPj6MOyUbLxNQ7KjhMnUD6T014Qsj01D2XWe2NonDMSqvv26nEnYQsR0clXK7PLZeldTCy4R9RcKRg59DJFxPy6oQd5WnwQ7FqELWG1BQEY4O9bzruT5uYxtE1TMZn6nwzhxB5YiLQr9EuqG6TMUY\\\\\\\/WjzBpbanDFoGVmpU8RgI+8abfusCFWeRj4asHGSY8hALc0Fvv\\\\\\\/f0YxN6jAGNcL9XzoGDchQ30qapVWMckjhbaLri4Wxs2D51MnbETYrotNCqvfgaO0uOrPTyIddKkRR8ONjV+vxa2ZWAtO4AHnk+y35tLPqFT12jXS4HRWTyCrPrkm7es7kfQz6irWVf6PElNKdK3AFsosU1KDg1CuTFDZN5AdoeOpEXC2KAIUVJqrCP2ZD4yr+E+x8oznj3n8sxYBrnNHNNlS96UF\\\\\\\/dzaHB+UD7mMjfNrEqPAgSRVh2VH5pk+Q+8jyCRV7FgkmQU4H0sX3wg18oeZMphv+DohF9lHnILbD+nUsWw+D\\\\\\\/4dGnT0T2hvHX9uvRaUTmXJvNRY01bifqrETQXCfcv2\\\\\\\/BebGC3wqMBuPF6ma4SSuuRCV84tg+U4oHjYkuDeiBoYzHSJPwyzxMfmIVjIlLPOnLraGW0OCYZ4UgXL7q6ZW1aOeFdWfyydv3\\\\\\\/JmbJ3tm2Rw0qr8MneCXHojNxkx4wAmxC3iInhAevLIrlxkg3+dcbnqChRutA8QttP7cBpjLIXG1quwXVpNr+mIICQkjrpWZsc6q4xPVvsfuSM3JKELCWaIl\\\\\\\/qgur20vFRz7vYjabRxWjarM25+7kRcAeFbCa0\\\\\\\/ntRcWcMJ1D\\\\\\\/fSry0XiVwHp6zmVN7X4UKaogfj5ScRCBRX7y+ouuhhhtm6OsrNBZonxDrXqU1SFHJSSSoEfGUXI049ETClML4VVVnEDXqFX6fy8SITgBrxnNzZytzJgpevQegqUOhTMxXPku8kqUyw+CBoiqFYTl44epsiTd0BTYOuWfB0OUDlPNBG2\\\\\\\/vbtqvcU4XpPLnIrikacskWr35QQYy0XLoAGFytrSoezAd+bzdpW0sIs4ZfnB7ch1nqSW0YQKx3FyWabXvorEDMa0VfUKs+Wdz5SkerddUj0o9Pi\\\\\\\/9Sb813U1+CLNNZiPmrfFEg9oILTSoLGkfD2loZNE3O5u4ytjShhLGqMVsqbTiHnmGZuNvD37c8J33wKy8Ax5EjQEkCyEjUspwO+\\\\\\\/aWlLXusAtQE5PwDaXgbCyrKP5DWwiiWix6wJ8Gz4Sz2qVajsXOFspwtXuNXOD0zIKm1STSAuEEsBw5mrgxBt5AyzICTp1ZYUcgEmQoyHf7MMQ2KhY9W+M90UquMBBxKRer4fyP9FtiuDvX5bVyie\\\\\\\/rmJ4uoD4ntuVbPgdsI\\\\\\\/HQg4vGn9NGfz5XaN9dW1X\\\\\\\/1CB6zK0Dt0PTwGHfbZG11YHCVrxHDetJsFq\\\\\\\/T22ybDQVZwLp\\\\\\\/kedv5QMqoVCcuc6watFs4X7hQtdIGPncNuUHP+L6I04vmZxLf95DvK4KVqjOrfw6apXLAZVoRdtY1Q9sT4U7okM5HoQ1uKAaCqHgguxNCc7y8618EfxOEeTyCelNtIPszBaL57Gtd2Yko63O9VFOdegok+cAj8xvP1+oTrccPPw1xlGAIuR4YxPDFcu218ENsrVnPWbUUCOJI5WebskVyP3m6sJ4Mn\\\\\\\/HLXeih0j2X\\\\\\\/Hvzvy3YB3DT0YZW8r2fsA6O0FM1SkOkp4CWWWYA5TmdWdvR8CT2fI9z0tPe40IcLAoEKGMmEkT+D5Jz3oX4ELztUi6aZjh4aM0fun6Mg6p5XJm\\\\\\\/TI\\\\\\\/hCuPr2RRzNuHgCMQ5coiP4okO2ko2Zhzk\\\\\\\/5mzVHWx5O4BiHxecDjK+M5rnRMejmytO0wa07\\\\\\\/LpajtiBbteeP9CwCBIoK4OsGECjLUJzY9V0YHG2T7EleCXohABu+72upVPxEHWPdsX44GnmsDntuIDfb\\\\\\\/D0rbmJLMQ+MXD1SejRDo5SYybvlLL2GQchVLy+Tqx2yHD+0tV1lUO1e\\\\\\\/N0Muw7LuMfDgMa1rP\\\\\\\/DkGlsP5tEN0\\\\\\\/MMO6bSOWSmHg9Zjbdy14EERlY\\\\\\\/7phbH8\\\\\\\/vAvkkhMHXQ+ak\\\\\\\/dNS\\\\\\\/xR1QAx7j568Q9wRYBvorEuohSBWZCXypg7ggh1yF6+VS0AAxaxDyOKHELgKtuz+u1GoHSzXJjq1pmYm7yzGcFsYM5O1qp9lFWDDOdNPnSTNeGDyGPoNpeVQw36ldIEYLe6TGjp02DyPk\\\\\\\/SpoNCi70hwnjOEnWM1Ek4TepmX273k12VzjPd3R3Cs+3k+xMx8AbgZ8GoUus4J+KrzGuHIBZ8r47v38YAeHcWcI2TRD1zsobFwzhC9eBkVifA6LdGb2Xs8FyHTL9IZhpW4W11wRfUdszhpJ1Taj5BrHQ\\\\\\\/eKsuzEecj7qyu6AA2JsbKjv8zZews5W\\\\\\\/Iw9Nwj+0y6Dlx\\\\\\\/qQ9eBJnZEkbhlGCVWmfLncw9Shv0OAXrPjtGtKMCrpmKEyVUI0wB8MCDJJgUUh\\\\\\\/ZUbCvWMHHWqwgvuS99\\\\\\\/fydg+jSvNR3WbPjlA6CqAiVoHquhsMQLEt26ayRebXJTLb8Px1\\\\\\\/qcPf+5gi29GG6f76Rw84NK4eQLVbIfW4CwwW5DzZsE2R18aLdEjj0XARKi0fkfvim2KidFGRmYabnuJRRe+BjUcaYeEZW\\\\\\\/L4Ud17lb0hOQUL6eyTqTDAD9yIhUfPMDtFybIpiGfEqkAcC3KcqkLpAc\\\\\\\/n92XJ2DtYzbmoHp4Mt56b7MtU8TTtw\\\\\\\/RuSJBGFs7HT5yJa2\\\\\\\/g3v8WStqalO36D8Sstt8bcH1g6fUpjap14yczblOgXgkyjwbZMt\\\\\\\/Y4VM0BgOm6P\\\\\\\/SrBm1Dn5PqJFDywagcjHcl3LWwPlGxxRrkLlm84ADowym8qx84jj5Wg+7cOAQ1WlSZTUtGXBLZdDB0mlfPz\\\\\\\/FDNiSMu2v2c8rjivU1hFLA3K+hMycTGnLJxmgnsxHdLVuiTVK5thVl5u0n0Q5dssIEE4h\\\\\\\/IQOmUv6BFNAI\\\\\\\/O0o\\\\\\\/SfC15v39+Z+0+fpOLewkXIyIJyz4lyDBLS0fxWAuezgDOYZTsIhOtxJpZuA7S3khuq84cAsPrmcaRo+Nst6owjKzwcbH5Rcjx8L+P4ByNBpuSVlQJ66Gg7QBqMVLnKs0IXcDfnrrpIqW5HcGhvrbfVYznZfqG5HRgfLjcWvhgpG08AXP0iFtKBXWju\\\\\\\/H++dGDXDke1Jafa3\\\\\\\/s\\\\\\\/G76s9lM34oT53JOIt7WTMcrN2VUKYOByvnMu6OJy30Ed\\\\\\\/mKfE0tqMDIb\\\\\\\/bdfD17P6L7uo+NQyoCQOb1x6vXfC\\\\\\\/iv+7zS6QibleGH4DB2yLSQYDB73eVrsWF9aLSti2nJ410sHQK9AqAQwK51xj6+rLx9vzcdPX9crSrPu6PHsjkLDXyQ8vcM7VSb+c7O0EFV6BHmr1yoIh0EkThxatvz1ADEvVC8oE\\\\\\\/8zEvpW2m+pp0Vs9+ZWPxL07i6YZprYSfFzNAIqAcPQhDWJ6BMspc8QoQ4CTs5jmDhbBeBHHpRbeLK+XK+BPTu0YMuw+6cST5crn99sCG28LY00D9F497YQ3dMFokQyi6+JD4fzoWDtzTA3NkdO3RSeEiiAXvPsspJlWJCfvQCg\\\\\\\/tFPboBu0hwXghkIJXwaKaM+vS8dR7OwR4IsuStFijk8rzifFl3vtK8MKAxgXhl\\\\\\\/MnVOVOYNsUwdGzMk219cyYzN\\\\\\\/iOaRWUMAnbTy1ow0WFIx2I\\\\\\\/+NbOi3p38zh\\\\\\\/X0KKfniQoGlefxBIGM+6Va+O7fuucs2ZpBsR4JEElMVm9UzHi\\\\\\\/f9DS1fFH9y4m56nHpRGfi0MDybB6HO7CrGSa+otxPEi2k0zPy33pGUaiO5un6KrnL5kAa\\\\\\\/u4aMVkTI8JrYoJAg7AQ7lHPL4wC9o2vWeurt\\\\\\\/acEsUTwbyF2MEudQmv8BY7Hq53xfVXM6kXi6Ihopmr2mfhCluwj+9pD69awTuWGEtUTlQ64r6+GF4hCUQpUgiu6EgzmxwxZY\\\\\\\/f+sbDaFdjbH2KfF6pF2QjEERDNiywotbjw\\\\\\\/GcPdaupic7hXv0+a6fog5QR18dMfQi7hBfXYtKIrsN0Dk51l+s3GrNbS7MBkPd2cw1zlZ2ylVI2vf3id6GR035LW2dfpndnbq3WAbCsltr26oWmvta1PhfkrNaahQSBkBPdVWemkT+\\\\\\\/YYCVp9ScOR\\\\\\\/DDN96IZNWtjucBickL21qU0xceY6nA3luPUKwkizZpcbGqKzZfAgYrhAexEVmYiiLf0A63Nex\\\\\\\/C6vZCWdOnxNMwwlTM4wP+6GbUy5oSent\\\\\\\/41amhJKA7sVfTZqVR5vgy6P+6dDuQuWGMuxxNrUZVC1My2Ru9wxT6iUnei+iVcwGZ5mRH4W+glnG8JujE4oDHpshoTRI9YR9f9XjEsl+cfRcok6yrKlEna7X2h6a5NJ053QtxdgLI0BZXRQ5z61h0OuW0OfzfQL\\\\\\\/3WftDxPKge+rj1CPY2VrLgGd18VxQGvnk3LaRHcIpf67JYw5iCns6KWwEN4Z7v9hDEayNR28MpjOY3shJ+bkVT7GmYyRoVV+eAwyKQF3dvNLTr8JnH5\\\\\\\/6EDC1LDimYMOPsgcXOpoVt77Hzro1LBBEEYh882ziHH\\\\\\\/a9EGWo6pvelObzwG1jjkR8UVROhGSpcabgSMr2VODO1O8F6R2gfyZoFH2heLQoxSkkwCBKON3FC3PG0\\\\\\\/o\\\\\\\/ZZkOXE5MdL5pRyRyEF\\\\\\\/SQgb7wmfYwo1DRqtLb16qJpoqZ\\\\\\\/\\\\\\\/\\\\\\\/zS+xlXDz0dXErdf+64vhFGsPShZqcDC7NghRl2+X3jTtEFtbJOuJXJqHvv1K0SQvad+SSJTUqJuXhr4TGW4qHQXG05aSHviGeHAzy1\\\\\\\/UNswMFfJaq3+NWwn7I2Rce2D80v1Vu6ZLkkODPHoZfoMpvCLeHX9iHBhZYl61U7KuV7qyCwPs6MaXGtsTAFpyHPd\\\\\\\/TaUKeopBWfE2phvzWFEBLd3DYUzwBVkMTP7i4BESI4Hqz2sd+uMK9SIk\\\\\\\/l0vXxSWyj3SbXRkLaE59WQdGYzlLQ1lH\\\\\\\/IQ4llRTDSzMIBsCnnYlCtQvlyNigHgXc+TaOrU5jIvx5u5dhVGAZUEC4iMBbKRsNV4hZGUhlcaYH27vwevA5Yi\\\\\\\/NRVgpTGtDm6S4loUMlYQNOrbsXBPH9JllFqfMNTR9k3d4LMW82c1dr6ONYdBJsq8t634kY8Sa42VBEYj1SzOfa8IRKqXq5430EEnOFU3oDZkENQccwLb\\\\\\\/JvFEnVL3LCkeHwp0AYwbrVlf3ZBcsp+KlYxZuXe008GaWlC4T2QmjwCT0rKUbKPQ9J1wRdWiun3bfEDOduRzDFQy74LvQqEg+QWRC\\\\\\\/YUFudcK8hZIeCQzGN8D4J\\\\\\\/Qa0HDvKCqYQVxMeBgLR28fEpcLMri7xlK+gkWTjW2bwkyMp7PtE\\\\\\\/9gVDq2ZZkg2uG5b\\\\\\\/gGZNulclDcQVnE3jVtnrW8YUNaR\\\\\\\/Gqi5cysbS4j7omQ1DF1sRotD+oBFQdHUEzYNTd67\\\\\\\/QG\\\\\\\/\\\\\\\/XZXpvOWfVvQZn5Gh+wxQ8gQ+E30PrE3ZTGjDgGITdPU1tLbn5\\\\\\\/EQzxgo+Mdnhn4vriqEU3x0KunrwkPGwm0yRW\\\\\\\/ibJ9kODJ\\\\\\\/b5+fIjeH6BZLqPkUiniO\\\\\\\/8O72FaGc2BGqFJ5sdf3ALKBYAoLqFaa1Idp7zsG\\\\\\\/6ZwZtyaLe5UFb1wU+LApPFfwP4CvHQqnTizm0xR7ZDV8W36\\\\\\\/Dj7ursa+DmYca7WtK0fBxt+R0rwOTiYZNH32O2lKzGch3FQR\\\\\\\/2QmFgVBy75MZRYEkT9mpFvDvoaGQ4+984VYPL60bE5ZFiwgoIThBjlq20CYdRu6AFR6N5ap4NqIe6XgrINpLlh9SLizz86+HwZYOMYGsRrGfexQqbfioKp0DuctVjMq45nmV6aq9GoigWYE\\\\\\\/rjnjI5NE6ot1GgiFaS9bfZ4Mb29Ey8FVdidAwPc73ZJoSUlBHrYlsR812dp4hYAPyPWCZreog0MK1MmYfe801QGfzHAbnyhfXLbHtjKvTIFLWanOWK10vdBYdtgvLEV3w3jjtjHmQmyRllMS4J9GUhyeTC9EYYQ+zonJvLADFYMi44pa8E9lbYgOLFryyCJSeaTrfGzDzVQxOFpFiJnNR8Ybwa2w5kDNKt4H3D\\\\\\\/xIEYnwkOSjFyeszTeCsQ9j5mpc\\\\\\\/2CXiq3chyMOP6wyV2Rxw+cJlCV1i8ZYzPjujelPhb8f30CI93xHsb46No6l3LKyR1IfALdl\\\\\\\/IqpTMycytmcvlX+ZA9umLy8m04HiWnt6gRK2p4VMFWEH5kBB544wJPReHsrxVOidy0ODOOYkCHg\\\\\\\/twwIvUAv2OQ8+5HlsspaUftScQaQP6plQMhOedFz0FKdXc780VhTHilq6dxb1m5mqypxbMntDC73HkgIkPhUXJrgkR4dJph0BVlWhAGmnvwd5FlTE\\\\\\\/09LyA92dm5UAGdXjBeT6FUdA3yd\\\\\\\/qoUVEwilukLx\\\\\\\/1gk40G0uwion3ldktVBkBocWCQkwzXcxfiDLlfngu1iJZw6wxd40BRr5cQ+IVqCI0GgPIZjPUcKjfeOQp84d5oReAro4BpPw1KDS\\\\\\\/8y7sx2dV+lIuy7A4LZrCkMOj02FPYkHyDlvKCkF3uLz228tlLxrtNLfboyzVnQ6azPGxX9wg\\\\\\\/0S2h1XmmMG\\\\\\\/RhF18MrQ5+vn7CvOyiUNj7GcjYhDPQe9MGXVn964JzGPuqkqhcTuzdibyxqNp9SvJrv8Yf8jAjss2Txy7uv45Pz31U5neEOkqZPrMbNgQ4N26Iqi9MBZlFehOv7TbSTX6oPwQN0zWkqlBtHrNBFwJCDfpsoqAetvIcP6bzSLI\\\\\\\/jSmZyISH7ABUzSX9olzOXnLqtC0MqjmU5XSud7TmxSLMsZVOtqvfpJbsbgeOoKgaJM\\\\\\\/C4YH7Hrjkj2tVMODhG94wLAL\\\\\\\/HCME8LsmFvNml1aaysg9di05EVfVG\\\\\\\/ixhVeeHFg\\\\\\\/fVfJNaIeAab6n1DkPcu6p5RIJ7i6dQO3a3Pn6un2aXT9Hg3M+cqvcY746ZXVYevdIbhVvOuGdtt9GYDZrneUD0uHdL1OqVm+2dMws\\\\\\\/hOKen3OtRIAUqfcLiUSqghPbtif1k1Kp27MmBBFjCyr3qLve5muyAm\\\\\\\/AZI7RZfO2L6uyJqqDTNeWxEhYwg\\\\\\\/TdZNi10Pgb7XDDXDhsyL1xh2bqd3\\\\\\\/9dKQyAIIZwlGsBB\\\\\\\/Q+k6g37WVGWqfeV7gee+INnBfFl3MxvhMCfdGOsbvFH5RmXdLYq\\\\\\\/lKy\\\\\\\/ZprWaK6j4E8o3U2AByqMrG7aW3QU\\\\\\\/mJEGucLWsyNBXJBsqqwRwLJuHPqaMvDZg\\\\\\\/FjTr2kfPz174jGjwROXaCp9\\\\\\\/A4R146L7xcVIBdwzCV8dsAY7RLULpPoG7S10xD2eGXuUq6V5APQDW0y23b6ICroCBUdE6hZF0QSAkcBxn6gBZ4\\\\\\\/J6pA6TG2JVxniP6gZTbLas\\\\\\\/YDGyJ2eQcBYeUkalF5G49SQHGWY5UP24EdbAuxlNMwGOeRhrwjuXCTSbZXxovFwvg8qhGgqabE1jFe1xFvYgH\\\\\\\/RGMociR33Bi4heR1YDe79iXoHaJId\\\\\\\/+q7CqgrHN8J71Yy3q7a+3M30TBYDgZGfgCqYL4YYzHueQaNRSCCWvv1I\\\\\\\/yHPnrPd6TXn32Tuvu0ozVn\\\\\\\/a+mxNB+B5BBcDH9HoqbyrrDSbBOwLeYu6tymA5tSqOgvQn9xpKR0\\\\\\\/TpntvUaUpeNVmuSLvIAlmuhP9zcF0c2TNJo3c7r8XkDNYPGZyVc0cFIq2xiLjnvV07K1tHnO3tquuzcIeVjmE9Iqoozhqdy4r9iCZYhS5n6wOiV4WsWnxX0fBr9GjFRCF73BoAjuLBqLmywIm6pXYdQ4mKWC0N9DUw0ltSN9voIccGlAk7kUsSXj6xtxfuTEizHXeSo5I\\\\\\\/Z0IkTAUH5kfYZHhIopHPqXuPc3jD9KidLMrdGFW1n7fRej+CTxdZ4DBeUy2TTm2L675aqBEsHBxjX+dVgcPzMkeS46CdATC8A1T52ecl2a3fSSBKd7NHPNH4C5sCycp56onWfUgn+PvKvg1shGqeyWH9xjMlGFe0YkH2Ax54NRKTAc6Z4JHtXA91lc0G6VbWt\\\\\\\/EX5oVBSaiV9Q0yf6Cci33D4A2vVECGppaoZe8A1h+L6wt4gsDhHRNYm3qAdwpLoZ5MKcH8tEGufW3xygKEKK9SLNVXFCsig3DwHi\\\\\\\/we0f9CdEjpNT0YwbFfwiK7d0DIn5TRlE4ihmVaCGNkhqK2ioF1bPnWuw7+Eos6FEwyZ7mXIAJNieUeX2AhuSoMdOSdFyDedapHY6D73keczKcg2JpuDvyFBUJ7m5GTx\\\\\\\/dW8pXoyC+pnRymqwtD15EQjaDVMrcOg7FXYLapahmNMJkcKa9wxwaZ2ZSquyYUykpL8ts1ifSlPWxi1QTz4\\\\\\\/v9lF+axMpTrNk3+JsRmiPZAv7uq1u9tnzjgn9S4+5wfbMEo4pbhwt7oRItmbfLDxdcYu61s5vtOb7B\\\\\\\/ohiTyt5HJE6OHXRpEYhWFcldHq8CHIgN2L\\\\\\\/SxWwrliC07ud2xFHurL+tiDnjmWoyRJfiGnuXSKT9P8z9vUiSWmDrqgSXNZriX0gYgBKPB6U0r1kvgjPl9XgotfcBQ94QgqvORcF9X1lPkekU3zBYX2goXmLe4vEm7X3TcjUQ2UAQ98oCxMum6MfCydUGfF74Yrey7KvmLPbHCP18Kwu7kYYrr603Wo71hLy2kW6uAto2FW5djaxrSa9llQ4P\\\\\\\/P3h9M7ftyT2tjsDzZrc8m2GajIhJvOtnYsa1ZELEaMI9HPc+2sVErqcLj1+ggh4TfUvy4dTB01HpmnM52VQDcENHCJG37Wmvf1vcfuetegKTZJ\\\\\\\/deqdwRIYuBMj7A2RS3t11XITQB59kZQ8IzST\\\\\\\/KHFxpHks2YlqiuQsoesLzvFZI57tAm41lbVadd9kqUq9Q\\\\\\\/DhVYooUuFrhIKtzPC2Fj8PVB2CEQF7optqzN9y\\\\\\\/LZ0AuxAhxLp23ekhjZF2U\\\\\\\/7qRkEB94wBg46ONF+9wKCNtiqJrDOA4ibSEeLECSkvlrnLpVgaj95p3sC3VEQw+aK9yRLifN2NJF7uiHDm81OIzN\\\\\\\/JQEPMGux21Q4iBG5a5UztmD7SvX8DOEtLuLW4LJF74hWvtJtT3yiB8XWgzSNtU6Aauor4zSYcagsK2yjdQZLZrxG2BPpAaod254b+wldqCkOehmy4h1a43XcQgnlXNZYoB1+PjdHS7xLhQJ42NbDa3rdUq6XlIaUr9wU\\\\\\\/4vPF5E9Obp6c0tmxQW3srNow22xRT80Bx4+4V5PzfS7YjUV22\\\\\\\/hjhYA7xaKowtA35IwejndJU28NhFJqx93tAsqhTw8ltSLvEwCW5l5RQR7htI5d7q3Sa4JNB5KSucd122zHIsjcO\\\\\\\/ZDcEwoHxdzkH0Mctx6X5ZCo5rklDLkqhTQilv6xIjwl4ERPFt+0yO4zX8I59ktSf3qd0JJ26T6YMNp03Rvyzl4WeK7DNAFUsow01mJFYLyxK+1dh56NOVOXtKR3BU96b+NKWtK1BnPR1K9rQZB8fZIciD+\\\\\\\/QNi1RuhetWxnJGYCwhtttl81QFc\\\\\\\/yjyib0WRcdhTgTjHN8FVVLwUew9RHYB4uCYstgwZ2YwkqTqMM6DCbABhkRdtPfmjbR1K2h\\\\\\\/y8VfruFKwLRN6U1TUr+1LfUiOcU+vEbr3S0KbMHrO69pp3V+6j4ZMkarMNUrtZkdi6HisP6BmcdN8mSEEwRInGPYuw0l893k5XfSC0Zr4PApqLrpUA+9sEVsIImCFeI0DY8iClDqVH+gj82njnitjJ0db2QDCeFNSpJv\\\\\\\/y9lHaVL7sp7Xemk9g0CDAHkRd1VVtbeiCh7awdSulWeNGxUepRWMipXSrMfIQtM8nMBnUE4v\\\\\\\/0SItxPFIaaqeXc5oADenz2LvCRwQhLdR+VwxeIjkvggPwpEGasUXWGvi8F5RXhqEsoHSacJT+2yh6YrOLzVkD0JShcVkyyrc2ZUkW7ydsYL4Uuic8pd921teFr\\\\\\\/r6yvw17OoaBIjAxyp6LAYBMn28nS4E5jUBPKCZQ64K0W+9Le6Ytt9qFmNceQUDtObhNlzYwSwSuMVAOBXxw5OwAqb0SgwKANwdXjZB9mlMcukVCUgxKt0PiDpabfy3yr\\\\\\\/hNhnN7YpWudWJ2Xoq8JYrfw31MLfGk+yJkfadb\\\\\\\/j77guQVVrpbMlFuiGxTGly1GsMsfNaRgc4dyTZ3RSGMNlOrNpJ009v3AIXVhEz8nIYvkwp7+xfrbBFUTKxQG8rG1bI\\\\\\\/r9V\\\\\\\/iubYVqzBfHSSgO30g19XRE1j9SvIQnfu2CFJ2a5Ux7ynZJ7y3bVfyiwwZT4yVRPfTGq+E0A02WAPXO8kQCwp7sJd9LWTaX6kYTAhoAmMhMxN6MjRxNeIlMdQwwpmNCXwbO1PXG7qUC5ZovQoK6VK\\\\\\\/Srv34RGQlDxGVGusO3ClFTOeBpU1ZqoyPWzmQ39xNyBvBTzVwZ9s3HhEOcFnSVgs3J554kYObUbNTW2ccKxqKJ1bVXwZ5H5uXANgepxegMnNAPtbXav8mec6lIPFeoAvcRIn9SojHyKL+gRK7s6SB7cuL4WBY8M5eoKqhro4U8Mls3VT4uY2Z6DBoHv0Gao4DpWZbXeEuGhcasyOuH4vy39N36rwqB\\\\\\\/tT+jFDPUrVx+\\\\\\\/neqmCtQjQwQKAJZz7+plSu9Da7roAnNFKD19FF3kGD9e3MQyw3Ao5QEnOGFa7OrIKIZ5EgU+uvZOnr51C0iApmzQTYb0gzwAzdzI4wTTbIz++Pw6wbTzhSdMvtYC0LMGKCPijvIP63Fs4dwiUunx0JsWZ3IeSZBGh6VCyeiEul+lT0sBf+co3u02LSQZxDjt4BjM6u4W43JywpjrZQKZBM8gayuovPUM98kvXY3ExeOKw\\\\\\\/wfLdcMeHo4S8kwKlFtp8CK3csKztsbzztr\\\\\\\/uK22BACE4YBAtWu0UNFWNEz\\\\\\\/5RakDagfyu8HipFVNz67MBdDg8qKzI\\\\\\\/WthxxNmdTYlt1FOjMSQs2HSYehNq+9Wt53HG\\\\\\\/kzwbqHSWC0MQ1iyUimRlIFVFqGL3dycEyE3Kyi\\\\\\\/wlNayKwUAglezE7TzCFG4rAvZizVUaDesk4Qj9XLyrAAtjT+m\\\\\\\/nrm9OMFz8O64VuCuGbk1jGzysAlHmX7BCizY1xsNoziDH\\\\\\\/77AyYalJIHuEUHyjgr8xexIvUHBIdhxUo3QpTldZPIO6NKImjAdSvs7JbooyUfESc8EKXaUtyXyjKq1JAg4XcEkKGJu2vn3uDa\\\\\\\/NK1+xOQpLuHzctZq8eUdgM4QFwEwIlF3gcKIsXmtygTtp727PlZg+P8kyrzsUkgnC+7RzXKVqvGL6YH8IwuPGtBTOVx2nCJMEJhaON6\\\\\\\/Emc8l89mhviuI2+zQUliSx0\\\\\\\/+M0WFsmt7UtcaB4ROblIdvcL9fDnE8vlM9dN15lyimtYubzJi9AmvBFWG41e4MDPwAGoDO7HvrTFeIUAbtkumTJRM7ra7dIXbEAA4+VyWZB3277bwsQQTlnNYm79GW4K2KPdqmp2Xju6nhMWtq9FKV++bDMRe2fA94uJKdKW7ICXr893JPJWg5ePO3AsrT2At0MpRMpYMldKOj8b5u3xtsHcr0llvFaMYhx3F4FZEAP3mIWpQ72UZ56cm80yF8lgCULx1mejX9bf0Ie46GMb8hriM44Ri9Luz8OzLhwLQ3HMBrDxGMS0QWsQtXGXKJkjcxtaviF\\\\\\\/OiDQOB\\\\\\\/qflClJ2xVr8Mj3HhNA8hqG6K\\\\\\\/bPQQ1rZX6eq+zey\\\\\\\/elzrEHcYMClrbDUWlbzSg2xUlZ9yNOAgX0RZIZJD+yrhuDqMVWsJVJrvX25CCPrhO+JE6uSWKZjWWEMAVgqq4iesJotuqMidgcpYD6f5za6HPfWECz5l+r7buGwsEB0Nz2lTLDwUyw+UxeoBK40Z\\\\\\\/gh1oRnF2ZboKwHpLx7lmZkYllu1UL2bMUrZrrx1IIFoj72yqRrtqd1A0a\\\\\\\/ZvzZZJzYL58vv+nDyWze0u0QPDKRQEi55t8cpmm4+33BZaYgkrSUoSFM5i7uuTWyla\\\\\\\/78UGRhDTKaS7VC77aHbf0Eq9h9LtpP8Cn9ufePPJblkzkcpLsoz+mPbl87lyToyiJuSVFVcOwfheI6LcfSIQA8YEcAL8DQt2K3+501C844fMMib89PF\\\\\\\/pQzCoobT6NpzUAO1JoLdbhjdeXUUOf2u5Tucshjgb4BmOQD\\\\\\\/NtHXo2pAfeQKeBdefY3uC54O1EdLVRsY3p5zBLV6RNDkwaO95a9CoIF2ICi3CixV4OMEef0lkorVsj3l5vByogmqCDz8M5VojArnNskPcOtRIYe5KSL6GTqAS5\\\\\\\/GrLMVjSf+O6KyOJDFY7kjWluHHe0TEUMmLdp0yjwEeZYDgQqYFiGki6OLo9f6SlAKUB97j7aynkInALDKB+ReFNTotdm6TLKiOUek\\\\\\\/blUF\\\\\\\/\\\\\\\/HPmTDEBANv45o2xxmZft0ZodBV7EAV1kUWLSupd\\\\\\\/V\\\\\\\/FaNKsiFGdAsffBa3f1QThYbXq+8tsiiqcmrL6KJ0xQdCpvFMsU+6ZeGontYC2Nt+UPHT3HyR8PwoQf5emcl+LHhGYEevIqgQhGIsYg9qgT0tJHpY7gKB68nM3jGCqQcWd3QfpOSgFKJYVsvrbF+3UHZV66y4GhSfJl1dcOpalCWDj7O8AKOfgx9tfB4K\\\\\\\/58ibKr5hUTrQrlPhz0dsxB\\\\\\\/DfuYmoJ0CdlW5Mc6A8gR9ZbNtDfkFoAaUlNBYO6V6cq9bRSgOqtmoB0pNErk9XMU8EoKZecvhpnOP5qRHRvbm7+wK65Ujn+mNghSAm\\\\\\\/6YPh2\\\\\\\/f4brKsC1QmdwDMN9JxKMilj6D5rkwDdXHTVVt8SE5nzIfJ7Ex8XEmYXwANein0GDn51jpwd34\\\\\\\/AZy6bN9MLw6MpNTkD8kxbWZY2que0qrjJtpvuNEMHSjxAfdnVjeZVSUoevwBYWJItgdAjE1ML5tCUjZJZtquOpN4BpaipWHaWUPT7ioMeID+Uwx3UsoJ1Dvcc37RudjJRQRD\\\\\\\/zON0E3Ne7CaceNWqCgguyOO9j8TeCeqrQ+0GxuqbHEXGW5I3hOTJATX910YZrFUTHRQGeZ0fHsAXPVhLm51mnXbx5otJ6nwVEGoPXlIecQ\\\\\\\/rLj8x7IAAzOgfjTr2\\\\\\\/2u38zUx9J9ON6XARB2nGSJjBvinuYZVZnqgyLWWaXWPSHwDs1nRrBjuC4PqfOHis\\\\\\\/Kp9hW2rH8GBIcxkEI2EnQ8InKK2Hspu1Huzhef+odhVISr\\\\\\\/P6WZRqdHM72Z8koiIp8REskVgUE7t6PmgymWUlGkx7XkWjA9g+H\\\\\\\/LeD\\\\\\\/FPH0PMyXO\\\\\\\/3t0v0KYl8daJYnLBLuY6uI9rzlpp4erD6G2J5Ptvb\\\\\\\/Aijv26fZQ1Gd7ORHsxjJjvj9grUnbsZ5AkfjvqJdrDNRqcQWbiHyuPUKcUWGreWtCmXtP7hyo3kzEJ9OIcyjJs8KuToLXo8CiKFTS0SEbXI2OhpHmWSHxNOy0S8JPp8VPV1km3MpRJiYLz9C3USTW854IGsmRPQ7yWPhRMw8ohL8h9XNB1UnCW4VjmaGYBQR2V6kqIS207aklQ1um7me6cKWCkYT6otthD\\\\\\\/tcCBZSbk+yLm0rXQ2rwIz7YXztJPrjnRaQACDn+n4FP64MXjOBeswHz9fabb4SVc0WGA+wLk0Z31SrrWOpcXTBajEZ4XCVHDeWXj1rn7g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qn9Th6TBqjXRyo\\\\\\\/yXTvIbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002264e7401bb6847b3c11d8b4bfce3be1d44caf556be43c5028ff95eb5bff54a8ef\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OJFL\\\\\\\/oAvGWSpjGgbXr1K1BaV1mfClNGZ1LPQeYVB+iQ2XkG5LFuilJjTXsqdNb6FUrt\\\\\\\/THFg70mohhAPBCRnD7EEyTGJFL\\\\\\\/aLKCETbLc\\\\\\\/ElmsqVuc1aE0fNKDpf4XVhH4Qf90eF8AYbNr0OQDgM+1DMQE3GsGMj7lgun2XuYzDVx1iYbyeI7Gvd4UIEGXNYC4QnvagPeaHhDJnHtSaxm8pDZ3aWSm4fPegkW2ms5hK1wDf07XoSWVh\\\\\\\/FcG9L24L8AcYzaAA2lkxoSeCOzyowQn\\\\\\\/PdrzSQUI4wM\\\\\\\/pJmdWaD0pC1QwkJlA1hodJP0+R6ucNKjuJNhMtx0k7s8ZwbT2cjtd+t95KULm+ruuWFvZfBKYlFyjSkd+Qg9xafD3ztc48d91iPt\\\\\\\/v+pSnUwOODs\\\\\\\/Lkob77e556PZj+46pGIbqSqxnEGuMIRQYCo7hwnpg5Mpx\\\\\\\/0YEs2PW4fQwT\\\\\\\/vaVtyKtAi25ej5MBecq2kt88oxG04y3bj64Gh8vA98EfQHa1VTysf87pJyeIrJmGsA2gqhRbzeqJxp7dh4tYo3a3lrOgEJnwLLBp4GoO50l+aQfG2KUr2V37e8usjVgi\\\\\\\/icY78OtK2vYNWSStfvY5G63Zuy9wziW6bni1+uNFru29fWnMgkbLUCXYFaj\\\\\\\/bmXF0SO+6QWxZS\\\\\\\/u2xWs0G\\\\\\\/eDWGWIxsNRqEBp1hcfHiaRROZhJHW5w03t1o379KnjY7FsUn90kd5\\\\\\\/+11t2mqvQuLlUOwA5miFXCOWfimVhIxhXwxkTvOYDj4opzrq0bnogUPprgt2lK0yk93yiV3qVTxJnGl8zmiO+tN2EoGlF+YYQQo12a\\\\\\\/aLddM2\\\\\\\/uOy89QS9FHn14Ngk+39clrXA4ftTRPjOdPOZc+FVdXHL+L3r\\\\\\\/lY69VYTEkKtGmngItsIiK6\\\\\\\/Khn8\\\\\\\/dmOBP61Pk777DdvBVflvjPyPkX4vpZgcrlHkdykPrB3Bhq\\\\\\\/FN66LzjEeGxUyaFzjdrhbBSzjHP9NA1XPNj3Mi0\\\\\\\/mSV0\\\\\\\/RX9cQTkR0wgpNxsUsZZwD2SnwpvMdaP5eDGdHFr\\\\\\\/ntnPudoYr\\\\\\\/Psmp2XQFe0U7aGdIHNswfM9DhyS8Gslbd5gsWeF3rQQ6+odgTRALtw0x9I9V0NOmhtmzv4CoxPGDNCSwUBr19zkaONgeYIPlsDxj0xBafy25jEPmTClsbhw0eWRLRvKzW6pZrdPjel+gpfWR2uthO6YguN+mVBuKLPvhonyDsbbfKnEfrba6WgrabvEBl5cd02Vyd0DeLAQRghMQ\\\\\\\/5TZ28tHkHSeDl9T3+tp1yfIfLIz\\\\\\\/uaUrpJsu9ds1kt6v0T7NLX005RDPC8h7Ve7Tif75976I5j2IrjcPHzoNwyVvzWWVXWKdQpVlglOSJaBZWq\\\\\\\/XTlJhIT7IAD4Ez6\\\\\\\/XAfXBA\\\\\\\/nwsjN2S7jqPXHP5XC97NkBQ8yZXitfMcTABSavwrUuBCTPYgjDDQcTSdIJgSae9w3+YNLaDCUtIOYTMlThVToTcTSKzU2yFsBU9xSvjxPbpr89kzrsUPAtFoY7qMFXWW+nMxc671Vmy1NpssqbNFTUUZa\\\\\\\/4wUe08CaXJBDmRMT4O+deVFsYHGojVvetcB3Ma+pC6VwZ1w7n73d9uLkDLe729xTTIUHdKAZK3B026LIDtVMycbacqX59XIEsbMIWe+jazvnXjducfc0YUaYnaw5z9lZocOHiG\\\\\\\/UWeVeu9Kq3L2IhTnFlRYYGKihRcvsgEmVlPz6R+DHAjWJDJyCo12xWCaRIaoBRCWcxjDzDtWmwwqhGZTzkOdfSXocypxtGAMjbU5hbvOgJsJIncsH4av8Glg8IF2Kz439Ha2UQ4dpbRrR8CE31V\\\\\\\/o9Ur74dZ+J+chGBBTBkmp0MHPjVexw7O2GzwPupfWe1hYSdo+Hvo4C7fO8khA3ryJ8sODPV32ezLCxY7uFzrIU3iw5w9OEX6wLDzqhavaDFGR8H8VIjNc\\\\\\\/gbFk2R5QIcmTAtamGh6zR9ld6NNH2JqjQLUY64\\\\\\\/sIl3bNgpBNg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zpPXQ5qyQ88hlG078DJ3+w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002251b3e670b909c71a89e876d362a33456d7a5a05e08e14a8ad3934f52842887e1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460476 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3485" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hHSGT-nalfTE5SnKLFLxIr40nEO3iFbNMOFkQeC8idI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3485" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jx5RL_JVih0WpzV7V2XmC1XcmdzcCFrJmo8fNMf13aQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3475" title="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3497" title="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460476" id="node460476" data-pisa="ggg;6/11/3485" data-pisa-master="ggg;g3.116.030411" data-apath="/ggg/6/11/3485.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">First Draft Assembly and Annotation of the Genome of a California Endemic Oak <em>Quercus lobata</em> Née (Fagaceae)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Victoria L. Sork</span>, <span class="highwire-citation-author" data-delta="1">Sorel T. Fitz-Gibbon</span>, <span class="highwire-citation-author" data-delta="2">Daniela Puiu</span>, <span class="highwire-citation-author" data-delta="3">Marc Crepeau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4464-8453" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul F. Gugger</span>, <span class="highwire-citation-author" data-delta="5">Rachel Sherman</span>, <span class="highwire-citation-author" data-delta="6">Kristian Stevens</span>, <span class="highwire-citation-author" data-delta="7">Charles H. Langley</span>, <span class="highwire-citation-author" data-delta="8">Matteo Pellegrini</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-8859-7432" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Steven L. Salzberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3485-3495;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030411 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460476" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victoria L. Sork </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of the Environment and Sustainability, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BL.%2BSork%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sork%20VL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictoria%2BL.%2BSork%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vlsork@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">vlsork@ucla.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sorel T. Fitz-Gibbon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Genomics and Proteomics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sorel%2BT.%2BFitz-Gibbon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitz-Gibbon%20ST&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASorel%2BT.%2BFitz-Gibbon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniela Puiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Computational Biology, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniela%2BPuiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Puiu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniela%2BPuiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marc Crepeau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BCrepeau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crepeau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BCrepeau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paul F. Gugger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>††</span>University of Maryland Center for Environmental Science, Appalachian Laboratory, Frostburg, Maryland 21532</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BF.%2BGugger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gugger%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BF.%2BGugger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4464-8453" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul F. Gugger</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rachel Sherman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BSherman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherman%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BSherman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kristian Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristian%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Matteo Pellegrini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matteo%2BPellegrini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pellegrini%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatteo%2BPellegrini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Steven L. Salzberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland 21218</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BL.%2BSalzberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salzberg%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASteven%2BL.%2BSalzberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8859-7432" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Steven L. Salzberg</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3485" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460476" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460476/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3485.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460476" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460476/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3485.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460476" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460476/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3485.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460476" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460476/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Oak represents a valuable natural resource across Northern Hemisphere ecosystems, attracting a large research community studying its genetics, ecology, conservation, and management. Here we introduce a draft genome assembly of valley oak (<em>Quercus lobata</em>) using Illumina sequencing of adult leaf tissue of a tree found in an accessible, well-studied, natural southern California population. Our assembly includes a nuclear genome and a complete chloroplast genome, along with annotation of encoded genes. The assembly contains 94,394 scaffolds, totaling 1.17 Gb with 18,512 scaffolds of length 2 kb or longer, with a total length of 1.15 Gb, and a N50 scaffold size of 278,077 kb. The <em>k</em>-mer histograms indicate an diploid genome size of ∼720–730 Mb, which is smaller than the total length due to high heterozygosity, estimated at 1.25%. A comparison with a recently published European oak (<em>Q. robur</em>) nuclear sequence indicates 93% similarity. The <em>Q. lobata</em> chloroplast genome has 99% identity with another North American oak, <em>Q. rubra</em>. Preliminary annotation yielded an estimate of 61,773 predicted protein-coding genes, of which 71% had similarity to known protein domains. We searched 956 Benchmarking Universal Single-Copy Orthologs, and found 863 complete orthologs, of which 450 were present in &gt; 1 copy. We also examined an earlier version (v0.5) where duplicate haplotypes were removed to discover variants. These additional sources indicate that the predicted gene count in Version 1.0 is overestimated by 37–52%. Nonetheless, this first draft valley oak genome assembly represents a high-quality, well-annotated genome that provides a tool for forest restoration and management practices.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">adaptation</a></li><li class="kwd"><a href="/keyword/annotation" class="hw-term hw-article-keyword hw-article-keyword-annotation" rel="nofollow">annotation</a></li><li class="kwd"><a href="/keyword/chloroplast" class="hw-term hw-article-keyword hw-article-keyword-chloroplast" rel="nofollow">chloroplast</a></li><li class="kwd"><a href="/keyword/nuclear-genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-nuclear-genome-assembly" rel="nofollow">nuclear genome assembly</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/quercus" class="hw-term hw-article-keyword hw-article-keyword-quercus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Quercus</em></a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">The genus <em>Quercus</em> (oak) is one of the most important trees in the Northern Hemisphere and, based on USDA Forest Service data, oaks make up the largest proportion of total biomass and have the highest number of species of the five major tree genera in the United States (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cavender-Bares 2016</a>). Oaks are valuable economically, and, in North America, <em>Quercus</em> is a major source of hardwood lumber, especially for furniture, railroad ties, pallets, flooring, and export (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Luppold and Bumgardner 2013</a>). In 2008, oaks accounted for over 38% of hardwood lumber production in the United States, in an industry that sold 1.8 billion board feet that year (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Espinoza <em>et al.</em> 2011</a>). In 2011, over 1,066,100 m<sup>3</sup> of oak lumber, and 451,000 m<sup>3</sup> of oak logs, were exported from the United States (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Luppold and Bumgardner 2013</a>). In California, oaks offer a number of ecosystem services with tangible economic benefits. Oaks add economic value to hunting lands and rangelands (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Standiford and Howitt 1993</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kroeger <em>et al.</em> 2010</a>) by improving nutrient composition and cycling (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dahlgren <em>et al.</em> 1997</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Herman <em>et al.</em> 2003</a>). Riparian oaks, such as valley oak (<em>Quercus lobata</em>), are important in stabilizing soil, maintaining aquatic and terrestrial animal habitat, reducing pesticide runoff, and improving water quality (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dosskey <em>et al.</em> 1997</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kroeger <em>et al.</em> 2010</a>). Furthermore, the presence of oaks has been shown to support exceptionally diverse native plant, bird (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Howard 1992</a>), vertebrate (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Block <em>et al.</em> 1990</a>), and arthropod communities (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Swiecki <em>et al.</em> 1997</a>). Oaks are culturally significant in California as iconic parts of vineyards, pastoral landscapes, residential developments, and, in particular, an historically important source of food for native cultures (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pavlik <em>et al.</em> 1991</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Anderson 2005</a>). Thus, the management and maintenance of oak ecosystems is of high priority.</p><p id="p-4">Oaks are managed in natural stands with selective harvesting accompanied by natural regeneration, or, especially in Europe (<em>e.g.</em>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">von Lupke 1998</a>), in plantations. North American forestry has a long tradition of planting seeds locally under the assumption that trees are locally adapted. However, one question is how local is local (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">McKay <em>et al.</em> 2005</a>), especially when we may need to manage tree populations differently under current conditions of rapid climate change (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Spittlehouse and Stewart 2004</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Millar <em>et al.</em> 2007</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aitken and Whitlock 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken <em>et al.</em> 2008</a>). Not only were many of our current oak populations established under past climates that differ from today’s, but current locations may experience warmer climates in the next 50 yr (<em>e.g.</em>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sork <em>et al.</em> 2010</a>). Thus, among the many concerns of restoration ecologists in restoring ecosystems is the source of seeds used to replant an oak stand, given that the future climate of that site may be different from the current one (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aitken and Whitlock 2013</a>). An understanding of the genes that underlie adaptation to climate would inform selection of seed sources for oak management to yield resilient forests for future climate conditions.</p><p id="p-5">Given the significance of <em>Quercus</em> species to ecosystems across the Northern Hemisphere, and basic and applied questions in population genetics, evolutionary ecology, conservation science, and forest management are increasingly using genomic approaches (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Petit <em>et al.</em> 2013</a>). This year, a draft genome of <em>Q. robur</em>, a common western European oak in the section <em>Quercus</em>, assembled with Sanger, Roche 454, and Illumina-generated sequences, became publicly available (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Plomion <em>et al.</em> 2016</a>). Last year, two reference transcriptome assemblies were published—one for the California endemic oaks, <em>Q. lobata</em> and <em>Q. garryana</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cokus <em>et al.</em> 2015</a>), and another for the European oaks, <em>Q. robur</em> and <em>Q. petraea</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lesur <em>et al.</em> 2015</a>). These sets of species come from the same white oak section (<em>Quercus</em> sect. <em>Quercus</em>), but they differ markedly in recent evolutionary history. The European oaks experienced severe bottlenecks during Pleistocene glaciations, with most populations surviving in southern refugia (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Petit <em>et al.</em> 2003</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Petit <em>et al.</em> 2002</a>), while the California endemic oaks have not experienced serious bottlenecks in at least 150,000 yr or more, with evidence of relatively stable species distributions during the last few interglacial cycles (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gugger <em>et al.</em> 2013</a>). Many oak studies have utilized genomic approaches, but without the benefit of a reference genome (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Derory <em>et al.</em> 2006</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gugger <em>et al.</em> 2016a</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Spiess <em>et al.</em> 2012</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sork <em>et al.</em> 2016</a>). The ability to identify reliable gene models, and to understand genome evolution, requires a high-quality oak genome with high accuracy, short-range contiguity, long-range connectivity, and good coverage.</p><p id="p-6">Here, we introduce a draft genome of the nuclear and chloroplast DNA sequences of a California endemic oak species, <em>Q. lobata</em>, as a first step toward producing an even higher quality genome with in-depth annotations. We selected a target <em>Q. lobata</em> adult (#786, 34.68820°, −120.03615°, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) from a population of valley oaks at the UC-Santa Barbara Sedgwick Nature Reserve that is publicly available through the University of California Nature Reserve System. This particular tree was chosen because it is a prolific acorn producer, and is part of ongoing research of valley oak, such as a 95-locality and 6000 tree-based provenance study of valley oak (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Delfino-Mix <em>et al.</em> 2015</a>), epigenetic and landscape genomic studies (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gugger <em>et al.</em> 2016b</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Sork <em>et al.</em> 2016</a>), contemporary gene flow studies (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grivet <em>et al.</em> 2009</a>), and a multi-year phenology study (A. Lentz and V. L. Sork, unpublished data). This study has five goals: (1) introduce a draft genome assembly of the valley oak genome (version 1.0); (2) present a first set of annotations, taking advantage of annotations we developed while producing a reference transcriptome for <em>Q. lobata</em> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cokus <em>et al.</em> 2015</a>); (3) report on an earlier draft of our genome (version 0.5) that is less complete for gene models, but has excluded duplicated haplotypes to facilitate identification of nucleotide variants; (4) summarize publicly available genomic resources developed to date; (5) compare our findings with the European oak <em>Q. robur</em> (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Plomion <em>et al.</em> 2016</a>). This valley oak genome assembly here represents the first phase of the production of a higher-quality, well-annotated genome that will serve as a valuable resource for the oak genetic and conservation communities.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Information on sequenced Q. lobata adult #786. (A) Map of California with species distribution indicated in blue, and location of sequenced tree shown with shaded triangle. (B) Local map of sequenced tree #786 within the University of California Santa Barbara Sedgwick Reserve in the San Ynez Valley, Santa Barbara Co., CA. (C) Photo of the sequenced tree #786. (Photo by A. Lentz.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-99578460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Information on sequenced &lt;em&gt;Q. lobata&lt;/em&gt; adult #786. (A) Map of California with species distribution indicated in blue, and location of sequenced tree shown with shaded triangle. (B) Local map of sequenced tree #786 within the University of California Santa Barbara Sedgwick Reserve in the San Ynez Valley, Santa Barbara Co., CA. (C) Photo of the sequenced tree &lt;em&gt;#7&lt;/em&gt;86. (Photo by A. Lentz.)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3485/F1.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3485/F1.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Information on sequenced <em>Q. lobata</em> adult #786. (A) Map of California with species distribution indicated in blue, and location of sequenced tree shown with shaded triangle. (B) Local map of sequenced tree #786 within the University of California Santa Barbara Sedgwick Reserve in the San Ynez Valley, Santa Barbara Co., CA. (C) Photo of the sequenced tree <em>#7</em>86. (Photo by A. Lentz.)</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>DNA isolation</h3><p id="p-8">Leaves were collected in September 2014, flash-frozen with liquid nitrogen, and stored at −80° prior to DNA extraction. Nuclei were extracted from 40 g of leaves as described in <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zimin <em>et al.</em> (2014)</a>. Nuclei were lysed by the addition of <em>N</em>-laurylsarcosine to a final concentration of 1% (w/v), and incubation at room temperature for 15 min. Sodium chloride was added to a final concentration of 0.7 M, then hexadecyltrimethylammonium bromide (CTAB) to a final concentration of 1% (w/v), and the mixture was incubated 1 hr at 60°. After two consecutive extractions with equal volumes of 24:1 chlorform:isoamyl-alcohol, the DNA was precipitated by the addition of 2/3 volume isopropanol, removed from solution with a glass hook, and resuspended in TE buffer. The DNA was then incubated for 1 hr at 37° with DNase-free RNase, extracted once with 25:24:1 phenol:chloroform:isoamyl-alcohol, then once with 24:1 chlorform:isoamyl-alcohol, ethanol precipitated, and resuspended in TE buffer. A final purification of the DNA was performed using the Genomic DNA Clean and Concentrator kit (Zymo Research) following the manufacturer’s instructions. Purified DNA was then treated with 1 µl of PreCR Repair Mix (New England Biolabs) per 3 µg of DNA following the manufacturer’s instructions, extracted once with 25:24:1 phenol:chloroform:isoamyl-alcohol, then once with 24:1 chlorform:isoamyl-alcohol, ethanol precipitated, and resuspended in TE buffer. The purified and repaired DNA was quantified on a NanoDrop spectrophotometer (Thermo Fisher Scientific).</p></div><div id="sec-3" class="subsection"><h3>Construction of paired-end libraries</h3><p id="p-9">Two short-insert paired-end libraries were constructed, one with PCR enrichment, and one without. To create the PCR-enriched library, 5 µg of DNA was end-repaired and A-tailed, and then universal Illumina paired-end adapters were ligated. The ligation product was run on a 2% agarose size-selection gel, and a ∼1 mm thick slice was cut from the gel adjacent to the 600 bp ladder marker. The DNA was extracted from the gel material using a MinElute kit (Qiagen), and quantified on a Bioanalyzer 2100 (Agilent Technologies); 7.6 ng of DNA was used as template for 10 cycles of PCR in a 50 µl reaction volume using KAPA HiFi HotStart ReadyMix (Kapa Biosystems) and barcoded Illumina primers. The PCR-free library was constructed using the TruSeq DNA PCR-free LT kit (Illumina) following the manufacturer’s instructions for 550 bp inserts.</p></div><div id="sec-4" class="subsection"><h3>Long-insert mate pair (jumping) libraries</h3><p id="p-10">Long-insert mate pair libraries were constructed using the Nextera Mate Pair Sample Prep kit (Illumina) following the manufacturer’s “Gel-Plus” instructions with the following modifications: the products of three tagmentation/strand-displacement reactions were pooled for size-selection; all size selections were performed in 0.6% MegaBase agarose (Bio-Rad) gels electrophoresed using a buffer-recirculating pump, and a FIGE Mapper (Bio-Rad) as follows: 1× TAE buffer; 16 hr run at room temperature; 4.1 V/cm forward, and 2.7 V/cm reverse field strength, both with linear ramping from 0.1 sec initial to 0.8 sec final switch time; finally, the purified DNA fractions from the size-selection gel were loaded directly onto a fresh 0.6% MegaBase agarose gel, and size-selected a second time to increase the stringency of the size-selection prior to fragment circularization.</p></div><div id="sec-5" class="subsection"><h3>Sequence data</h3><p id="p-11">All sequencing was performed on three lanes of a HiSeq2500 in Rapid Run mode with HCS version 2.2.58 and RTA version 1.18.64.0. The short-insert paired-end libraries were sequenced with read lengths of 250 bp, and the long-insert mate pair libraries were sequenced with read lengths of 150 bp. Demultiplexing and BCL to fastq conversion were performed with CASAVA version 1.8.2 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Eren <em>et al.</em> 2013</a>).</p></div><div id="sec-6" class="subsection"><h3>Assembly methods</h3><p id="p-12">We used the MaSuRCA assembler (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zimin <em>et al.</em> 2013</a>) to assemble the short paired-end reads into super-reads. Input for this step was 266,002,352 read pairs (532,004,704 reads) from 500 and 550 bp fragments, with a read length of 250 bp. This represented a total of 133 Gb of sequence data, ∼175× coverage assuming a genome size of 750 Mb. We also used MaSuRCA to “clean” the reads from both short and long fragment libraries, in a process that uses super-reads to correct errors, and trim low-quality sequences from the ends of reads. We had nine long-fragment paired libraries, ranging in length from 2900 to 12,000 bp, with a total of 159,115,423 pairs of reads (∼56× coverage) in total. Subsequent assembly steps used SOAPdenovo2 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Luo <em>et al.</em> 2012</a>) with the parameters −K 127 −k 63 to assemble super-reads into contigs and scaffolds, and to rescaffold.</p><p id="p-13">To remove redundant and low-quality scaffolds, we aligned all scaffolds shorter than 50 kb to each other using bwa (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Durbin 2009</a>) and MUMmer (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kurtz <em>et al.</em> 2004</a>), and used the show-coords program within MUMmer to identify scaffolds that were completely contained by, and nearly identical to, other, longer, scaffolds. To align the <em>Q. lobata</em> and <em>Q. robur</em> genomes, we used both bwa-mem and MUMmer, which gave comparable results.</p><p id="p-14">Previously, we had also created a haplotype-reduced version of the <em>Q. lobata</em> genome, labeled version 0.5 and available from <a href="http://valleyoak.ucla.edu">http://valleyoak.ucla.edu</a>. This version was created from an earlier draft assembly by aligning all <em>vs.</em> all contigs (&gt; 600 bases) using BWA, and removing the smaller contig of any overlapping pair where at least 20% of the smallest contig is overlapping. This process was repeated until no more contigs were removed. There were 40,158 remaining contigs, totaling 760 Mb, average size 18,900 bp, with an N50 of 95,000 bp. This aggressive reduction of haplotype redundancy increases the utility of the genome for variant calling, albeit at the expense of creating gaps in the overall coverage.</p><p id="p-15">As one assessment of the completeness of the genome, we conducted a BUSCO analysis (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Simão <em>et al.</em> 2015</a>). BUSCO (Benchmarking Universal Single-Copy Orthologs) software was used to search our genome for each of 956 plant orthologs within an early access version of the plants BUSCO dataset that had kindly been provided by the authors. BUSCO uses Augustus (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stanke <em>et al.</em> 2004</a>) to predict gene models in targeted genomic regions, and was set to use precomputed metaparameters for Arabidopsis. For comparison, we also ran the same analysis on <em>Q. lobata</em> v0.5, the <em>Q. robur</em> genome, and the <em>Populus trichocarpa</em> genome (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tuskan <em>et al.</em> 2006</a>).</p><p id="p-16">To determine relative depths of coverage for BUSCO genes, the 133 million pairs of reads from our first library (Qlob_2_H00GJ_L001) were mapped to the <em>Q. lobata</em> v0.5 assembled contigs using bwa mem. GATK’s DepthOfCoverage (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">DePristo <em>et al.</em> 2011</a>) was used to determine the mean coverage across each gene, for all mapped reads regardless of mapping quality.</p></div><div id="sec-7" class="subsection"><h3>Chloroplast genome assembly</h3><p id="p-17">The chloroplast is present in multiple copies per cell, yielding much deeper read coverage than the chromosomal genome. This enables assembly from a relatively small fraction of the whole-genome data set. We extracted the first 10 million read pairs from one of the short fragment libraries (Qlob_2_H00GJ_L001), and used the ea-utils (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Aronesty 2011</a>) to trim adapters and low-quality sequence. We assembled the trimmed reads with SOAPdenovo2 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Luo <em>et al.</em> 2012</a>) with parameters −K 127 −k 63, and then filled gaps within scaffolds using GapFiller (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boetzer and Pirovano 2012</a>) with parameters −m 63 and −d 255. We aligned the resulting scaffolds to the NCBI chloroplast database (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sayers <em>et al.</em> 2012</a>), and found seven that aligned to chloroplasts, with the best matches found in <em>Q. rubra</em> (northern red oak, NCBI accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_020152&amp;atom=%2Fggg%2F6%2F11%2F3485.atom">NC_020152</a>) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alexander and Woeste 2014</a>). Six of the seven scaffolds had &gt; 30× coverage, and were longer than 500 bases, and we then aligned these six scaffolds to the <em>Q. rubra</em> chloroplast using nucmer (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kurtz <em>et al.</em> 2004</a>). We noted that, while four of the scaffolds had consistent coverage depth of 30–40×, the other two had deeper coverage, 58× and 72×, indicating that some portions of these scaffolds represented duplicated regions. Consistent with this observation, the high-coverage scaffolds, C437218 and C437210, each aligned to two distinct locations in the <em>Q. rubra</em> chloroplast. We created duplicate copies of parts of these two scaffolds and added these to the six original scaffolds. The eight resulting scaffolds had overlapping ends, where all overlaps were of length 127 bp. Using <em>Q. rubra</em> as a guide, we constructed a gap-free circular arrangement of the scaffolds, and merged the overlapping ends to obtain the first version of the chloroplast genome.</p><p id="p-18">We then aligned the original 10 million reads back to the assembly, and reviewed the depth of coverage with IGV (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Robinson <em>et al.</em> 2011</a>), which revealed three low-coverage regions. Two of these were mis-assemblies, in which contig ends had been concatenated where they should have been merged, creating an artificial tandem repeat. We corrected these by breaking the assembly, and manually merging the contig ends. This step resulted in even coverage across the two regions, and improved alignments with other oak chloroplast genomes. The third low coverage region corresponded to a 75-bp stretch of Ns that GapFiller had been unable to resolve. We then extracted all reads from the original 10 million that overlapped the edges of this gap, and assembled them using Minimus (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sommer <em>et al.</em> 2007</a>). This step produced a single contig that spanned the gap. Merging this contig into the gap resulted in even-read coverage across this region, and produced the final, 161,289-bp chloroplast assembly.</p><p id="p-19">To annotate the chloroplast assembly, genes from the <em>Q. rubra</em> chloroplast were downloaded from NCBI (accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_020152&amp;atom=%2Fggg%2F6%2F11%2F3485.atom">NC_020152</a>), and mapped to the final <em>Q. lobata</em> chloroplast using GMAP (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wu and Watanabe 2005</a>). The resulting annotation contains 80 distinct genes, seven of which are duplicated once, and one twice for a total of 89 genes. There are 31 distinct tRNAs, and nine duplicates, resulting in a total of 40 tRNA genes. The annotated genic regions make up 60.95% of the chloroplast genome.</p></div><div id="sec-8" class="subsection"><h3>Annotation</h3><div id="sec-9" class="subsection"><h4>Gene models:</h4><p id="p-20">We used MAKER (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Campbell <em>et al.</em> 2014</a>) to identify gene models and predict functional annotations in version 1.0, and then transferred those annotations to version 0.5 using the default pipeline of FLO (<a href="https://github.com/wurmlab/flo">https://github.com/wurmlab/flo</a>), which is based on the University of California, Santa Cruz (UCSC)-Kent. Toolkit (Kuhn <em>et al.</em> 2012). MAKER was initially run in est2genome mode, which predicts genes directly from EST evidence. The EST evidence, which was based on our published transcriptome (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Cokus <em>et al.</em> 2015</a>), included the 28,261 transcripts for which gene models had previously been predicted. Additionally, protein evidence was provided as 324,038 protein sequences gathered from the genomes of seven plants: <em>Arabidopsis thaliana</em>, <em>P. trichocarpa</em>, <em>Vitis vinifera</em>, <em>Glycine max</em>, <em>Ricinus communis</em>, <em>Medicago truncatula</em>, and <em>Theobroma cacao</em>. RepeatMasker (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Smit <em>et al.</em> 1996</a>) was also run within the MAKER framework, using –species all. Transposable elements with nucleotide or protein level similarity to a set of 24,916 known transposons (provided by MAKER) were masked. Annotation was limited to contigs &gt; 10 kb. The initial run was terminated after prediction of 1362 gene models. From these, 701 high confidence models were selected for training SNAP (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Johnson <em>et al.</em> 2008</a>). High confidence was determined by the maker2zff default minimums: 50% of splice sites confirmed by EST alignment, 50% of exons match an EST alignment, 50% of exons overlap any evidence (EST or Protein), and maximum annotation edit distance (AED) of 0.5. These 701 high confidence gene models were used to train SNAP—an <em>ab initio</em> gene predictor. A second round of MAKER was run using the HMMs from snap training for gene prediction, rather than the est2genome mode. All other settings were the same as for the first run, with the transcripts now being used only as evidence to support <em>ab initio</em> gene predictions. Two more rounds followed, resulting in 6710 and then 10,885 high confidence gene models. These 10,885 high confidence gene models were used to train the <em>ab initio</em> gene predictor in Augustus (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Stanke <em>et al.</em> 2004</a>). A final round of predictions were done using three gene predictors, SNAP and Augustus trained for this genome, and FGENESH (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Salamov and Solovyev 2000</a>) using the provided Dicot training set. This process resulted in our final set of 61,773 gene models, of which 13,898 are high confidence, as defined above.</p></div><div id="sec-10" class="subsection"><h4>Functional:</h4><p id="p-21">Protein sequences from our 61,773 predicted gene models were searched against the UniProt/Swiss-Prot database (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">The UniProt Consortium 2015</a>; downloaded February 12, 2016; 550,299 entries), using Blastp with an <em>e</em>-value cutoff of 1 × 10<sup>−6</sup> and seg masking. InterProScan was used to link our predicted proteins with gene ontology (GO; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ashburner <em>et al.</em> 2000</a>) terms and Pfam (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Finn <em>et al.</em> 2014</a>) domains. These annotations are included in <em>Q. lobata</em> v1.0 gene models file (gff format). To quantify the presence of repetitive elements known to exist in plants, we ran RepeatMasker (version open-4.0.5, Smit, A. F. A. and P. Green, RepeatMasker, <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) against the eudicotyledons subset of RepBase (update 20150807).</p></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-22">Illumina sequence reads, genome assemblies, and annotation files for version 1.0 are available through NCBI BioProject PRJNA308314 and both versions with annotations are available at <a href="https://valleyoak.ucla.edu/genomicresources/">https://valleyoak.ucla.edu/genomicresources/</a>.</p></div></div><div class="section results-discussion" id="sec-12"><h2 class="">Results and Discussion</h2><div id="sec-13" class="subsection"><h3>Assembly results</h3><p id="p-23">The MaSuRCA assembly of paired reads created 13,129,188 super-reads, with an average length of 444 bp, totaling 5.84 Gb. Super-reads are highly accurate, and represent a very substantial compression of the original data, as they did here. Due to their greater length, they usually provide a better basis for assembly. These super-reads were then assembled into contigs and scaffolds using SOAPdenovo2 (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Luo <em>et al.</em> 2012</a>), with the parameters −k 127 −k 63. The resulting scaffolds contained 242,966 gaps, to which we applied the GapCloser from the SOAPdenovo package, which used the reads in a second pass to close ∼150 k gaps, leaving 96,589 gaps in an assembly with 806,091 scaffolds.</p><p id="p-24">The scaffold cleaning procedure identified 605,682 redundant scaffolds, most of which were very short (average length 404 bp). These were removed, and we then rescaffolded the remaining 200,409 nonredundant scaffolds using SOAPdenovo2 with the same parameters as previously. This step resulted in 94,394 scaffolds, totaling 1.17 Gb in the final assembly. The assembly contains 18,512 scaffolds of length 2 kb or longer, with a total length of 1.15 Gb (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469569/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469569/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary statistics for assembly of <em>Quercus lobata</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Genome size estimates</h3><p id="p-26">Independently of assembly, we characterized the genome sequence using the distribution of all short high-quality words of length <em>k</em> from the raw reads. This was done using <em>k</em>-mer histograms computed from the error-corrected reads using the program jellyfish (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Marcais and Kingsford 2011</a>), with word sizes (<em>k</em>) of 25 and 31.</p><p id="p-27">Both <em>k</em>-mer histograms display three distinct peaks (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>). The extreme peak in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> at <em>k</em> = 1, representing ∼1% of the distinct <em>k</em>-mers, is an artifact caused by sequencing errors, each of which creates a <em>k</em>-mer that almost never occurs in the genome, and is therefore unique in the data set. The two peaks of interest, together comprising the largest area of each histogram, characterize the bimodal distribution that we expect from a heterozygous diploid genome. The right “diploid” peak, <em>k</em>-mers shared between homologous chromosomes, is consistently twice as deep as the left “haploid” peak, which contains <em>k</em>-mers unique to a haplotype due to heterozygosity (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For both <em>k</em> = 25 and <em>k</em> = 31, there were noticeably more words that were unique to a haplotype (<em>i.e.</em>, in the left peak) than words shared between them. This apparently high level of heterozygosity is consistent with values reported elsewhere for <em>Q</em>. <em>lobata</em> (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Cokus <em>et al.</em> 2015</a>), and expected for a highly outcrossing tree species. Both <em>k</em>-mer sizes yielded similar genome size estimates of ∼720–730 Mb (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Histograms of k-mer frequencies in the raw read data for k&#xA0;=&#xA0;25 (blue) and k&#xA0;=&#xA0;31 (green). The x-axis shows the number of times a k-mer occurred; e.g., the peaks near x&#xA0;=&#xA0;50 indicate the number of k-mers that occurred 50 times in the data. (B) Histogram of contig coverage in the assembly, based on mapping all reads back to the assembled contigs. The left peak shows the number of bases in contigs with 55&#x2013;60&#xD7; coverage, which correspond to regions where the assembler created two distinct contigs for divergent putative haplotypes. The right peak, at &#x223C;110&#x2013;120&#xD7; coverage, contains contigs from regions where the genome are less variable, allowing the assembler to construct a single contig for those regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-99578460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Histograms of &lt;em&gt;k&lt;/em&gt;-mer frequencies in the raw read data for &lt;em&gt;k&lt;/em&gt;&#xA0;=&#xA0;25 (blue) and &lt;em&gt;k&lt;/em&gt;&#xA0;=&#xA0;31 (green). The &lt;em&gt;x&lt;/em&gt;-axis shows the number of times a &lt;em&gt;k&lt;/em&gt;-mer occurred; &lt;em&gt;e.g.&lt;/em&gt;, the peaks near &lt;em&gt;x&lt;/em&gt;&#xA0;=&#xA0;50 indicate the number of &lt;em&gt;k&lt;/em&gt;-mers that occurred 50 times in the data. (B) Histogram of contig coverage in the assembly, based on mapping all reads back to the assembled contigs. The left peak shows the number of bases in contigs with 55&#x2013;60&#xD7; coverage, which correspond to regions where the assembler created two distinct contigs for divergent putative haplotypes. The right peak, at &#x223C;110&#x2013;120&#xD7; coverage, contains contigs from regions where the genome are less variable, allowing the assembler to construct a single contig for those regions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3485/F2.medium.gif" width="440" height="179"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3485/F2.medium.gif" width="440" height="179"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469746" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">(A) Histograms of <em>k</em>-mer frequencies in the raw read data for <em>k</em> = 25 (blue) and <em>k</em> = 31 (green). The <em>x</em>-axis shows the number of times a <em>k</em>-mer occurred; <em>e.g.</em>, the peaks near <em>x</em> = 50 indicate the number of <em>k</em>-mers that occurred 50 times in the data. (B) Histogram of contig coverage in the assembly, based on mapping all reads back to the assembled contigs. The left peak shows the number of bases in contigs with 55–60× coverage, which correspond to regions where the assembler created two distinct contigs for divergent putative haplotypes. The right peak, at ∼110–120× coverage, contains contigs from regions where the genome are less variable, allowing the assembler to construct a single contig for those regions.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469630/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469630/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Properties of the <em>Q. lobata k</em>-mer distributions for <em>k</em> = 25 and <em>k</em> = 31</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">We separately estimated the haploid genome size using the assembly itself, which, as noted above, has a total size of 1.15 Gbp, much larger than <em>k</em>-mer estimate. An assembly much larger than the expected size usually occurs because of large numbers of uncollapsed haplotype variants; <em>i.e.</em>, regions of the <em>Quercus</em> genome where the two chromosomes differ sufficiently that the assembler creates two distinct contigs. However, we can identify these contigs based on depth of coverage: contigs representing haplotype variants will have approximately half the coverage of contigs representing homozygous regions. We computed the coverage of all contigs by remapping the reads to them, and plotted a coverage histogram, which yielded two distinct peaks (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). In this plot, the lower coverage peak contains heterozygous regions for which the assembler created two contigs to represent the same chromosomal region. Thus, to estimate genome size, we halved the total contig length under the haploid peak, and added the result to the cumulative length of contigs in the right or “homozygous” peak. This estimated genome size from this method is ∼743 Mb, similar to the <em>k</em>-mer based estimates. We also used GenomeScope (<a href="http://qb.cshl.edu/genomescope/">http://qb.cshl.edu/genomescope/</a>) to estimate the genome size from the complete set of corrected reads. This method gave an estimate of 702 Mb, and estimated heterozygosity at 1.25%, which would explain why so much of the assembly consisted of haplotype variants.</p></div><div id="sec-15" class="subsection"><h3>BUSCO results</h3><p id="p-31">We used a BUSCO analysis to assess the completeness of the genome assembly and as another measure of the degree of haplotype separation. BUSCO databases are made from orthologous groups of genes present as single-copy orthologs in at least 90% of species (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Simão <em>et al.</em> 2015</a>). We used the plant database representing 956 ortholog groups. The BUSCO software identifies candidate gene regions, predicts gene structures, and then uses profile-based alignments to classify matches as complete, duplicated, fragmented, or missing. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> summarizes the results of this analysis for the <em>Q. lobata</em> v1.0 genome as well as for the “collapsed” version, <em>Q. lobata</em> v0.5, the <em>Q. robur</em> genome (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Plomion <em>et al.</em> 2016</a>), and the well-studied Black Cottonwood tree, <em>P. trichocarpa</em> (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Tuskan <em>et al.</em> 2006</a>). <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> shows that 94% of the BUSCO orthologs were detected in our genome, with only 4% as fragmented genes. The <em>Q. robur</em> genome, built with sequences from four different platforms, including Sanger sequencing of BAC library ends, has only slightly higher coverage (97%). The model species, <em>P. trichocarpa</em>, is slightly higher again (98%). <em>Q. lobata</em> v1.0 and <em>Q. robur</em> both have high levels of duplication, indicating the frequent representation of both haplotypes in the genome due to the high heterozygosity. The rate of 50% duplication for <em>Q. lobata</em> v1.0 is consistent with the above contig coverage-based estimate of a true genome size of ∼730 Mb. The assembled genome size is 1.15 Mb, indicating 400 Mb, or ∼53%, is redundant. However, the collapsed version of the <em>Q. lobata</em> genome, and the well finished <em>P. trichocarpa</em> genome, still have 37% duplicates, which may be due to large-scale genome duplication events. The <em>P. trichocarpa</em> lineage is inferred to have undergone a whole genome duplication ∼60 million yr ago (MYA), with duplicates still detectable for &gt;20% of its genes. Duplicates due to separated haplotypes are expected to have half the raw read coverage when compared to genes that are truly duplicated in the genome. Analysis of the raw read coverage for the <em>Q. lobata</em> v0.5 duplicated genes shows a bimodal distribution at 0.5 and 1× of the expected coverage, where coverage for single copy genes is a unimodal distribution at 1× (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For the BUSCO genes with exactly two copies in the collapsed <em>Q. lobata</em> genome, &gt; 60% are within the 1× coverage peak, suggesting &gt; 50% are true duplicated genes rather than separated haplotypes. Similar to <em>P. trichocarpa</em>, and many plant lineages, this outcome may be due to a past whole genome duplication.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469509/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469509/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of BUSCO analysis indicating the number of BUSCO plant single copy orthologs detected in each of four genome assemblies: the <em>Q. lobata</em> genome reported here (v1.0), a collapsed version of the <em>Q. lobata</em> genome (v0.5), the European oak <em>Q. robur</em> (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Plomion <em>et al.</em> 2016</a>), and the black cottonwood tree <em>Populus trichocarpa</em> (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Tuskan <em>et al.</em> 2006</a>)</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average read coverage of BUSCO genes for Q. lobata v0.5 (collapsed to reduce haplotype duplication). We mapped 133&#xA0;million pairs of reads to the assembled contigs, yielding an expected mean coverage of 75&#xD7;. (A) BUSCO genes that are represented only once in the genome show a unimodal distribution around the expected coverage. (B) BUSCO genes represented twice in the genome show a bimodal distribution due to some genes having only half the expected coverage. These 0.5&#xD7; coverage genes are presumably in genome regions for which the collapsing of haplotypes failed, leaving both haplotypes represented as independent contigs. The genes falling in the 1&#xD7; coverage peak are expected to be truly present in two copies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-99578460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Average read coverage of BUSCO genes for &lt;em&gt;Q. lobata&lt;/em&gt; v0.5 (collapsed to reduce haplotype duplication). We mapped 133&#xA0;million pairs of reads to the assembled contigs, yielding an expected mean coverage of 75&#xD7;. (A) BUSCO genes that are represented only once in the genome show a unimodal distribution around the expected coverage. (B) BUSCO genes represented twice in the genome show a bimodal distribution due to some genes having only half the expected coverage. These 0.5&#xD7; coverage genes are presumably in genome regions for which the collapsing of haplotypes failed, leaving both haplotypes represented as independent contigs. The genes falling in the 1&#xD7; coverage peak are expected to be truly present in two copies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3485/F3.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3485/F3.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469519" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Average read coverage of BUSCO genes for <em>Q. lobata</em> v0.5 (collapsed to reduce haplotype duplication). We mapped 133 million pairs of reads to the assembled contigs, yielding an expected mean coverage of 75×. (A) BUSCO genes that are represented only once in the genome show a unimodal distribution around the expected coverage. (B) BUSCO genes represented twice in the genome show a bimodal distribution due to some genes having only half the expected coverage. These 0.5× coverage genes are presumably in genome regions for which the collapsing of haplotypes failed, leaving both haplotypes represented as independent contigs. The genes falling in the 1× coverage peak are expected to be truly present in two copies.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Annotation</h3><p id="p-33">Using an automated pipeline based on Maker, we generated 61,773 gene models, of which 13,898 are high confidence, as defined in the <em>Materials and Methods</em>. For our full set of 61,773 gene models, 54,782 (89%) have an annotated estimated distance (AED) &lt; 0.5 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), and 43,706 (71%) have a recognizable PfamA domain. The annotations are of reasonable quality based on the rule of thumb presented by the Maker authors: “a genome annotation build where 90% of the annotations have an AED &lt; 0.5, and over 50% of its proteome contains a recognizable domain can be considered well annotated” (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Campbell <em>et al.</em> 2014</a>). An AED of 0 means the extrinsic evidence (<em>e.g.</em>, protein and transcriptome alignments) perfectly matches the predicted model, and AED of 1 means there is no extrinsic evidence to support the model.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distribution results of Maker annotation of the Q. lobata v1.0 genome. (A) Transcript lengths. (B) Protein lengths. (C) Intron lengths. (D) Number of exons. Light gray bars represent all 61,773 gene models. Dark gray bars represent the high confidence subset of 13,898 models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-99578460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency distribution results of Maker annotation of the &lt;em&gt;Q. lobata&lt;/em&gt; v1.0 genome. (A) Transcript lengths. (B) Protein lengths. (C) Intron lengths. (D) Number of exons. Light gray bars represent all 61,773 gene models. Dark gray bars represent the high confidence subset of 13,898 models.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3485/F4.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/11/3485/F4.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469672" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Frequency distribution results of Maker annotation of the <em>Q. lobata</em> v1.0 genome. (A) Transcript lengths. (B) Protein lengths. (C) Intron lengths. (D) Number of exons. Light gray bars represent all 61,773 gene models. Dark gray bars represent the high confidence subset of 13,898 models.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">We compared metrics for the full set of gene models, and for the smaller high confidence set for transcript lengths (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>), protein lengths (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>), intron lengths (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>), and number of exons (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>). For the 13,898 high confidence models, the average predicted transcript is 6575 (SD = 5331) bases, has 8.0 (SD = 5.8) exons, an average intron size of 656 (SD= 990) bases, and codes for 491 (SD = 341) amino acids. However, only 62% of these high confidence gene models have both 5′ and 3′ predicted untranslated regions (UTRs). The average lengths of the predicted UTRs are 184 (SD = 258) bases for 5′, and 306 (SD = 319) bases for 3′ UTRs. Addition of the lower confidence gene models brings the average transcript length and number of exons down to 60% of the high confidence numbers (3969 bases and 4.8 exons). The protein length similarly goes down to 70% (375 amino acids).</p><p id="p-36">The 61,773 predicted protein sequences were searched against the UniProt/Swiss-Prot database using blastp (cutoff 1 × 10<sup>−6</sup>) resulting in 41,612 (67%) with similarity to a UniProt/Swiss-Prot entry. Additionally, an InterProScan search resulted in 43,705 (71%) predicted proteins with similarity to a PfamA entry, and 25,536 (41%) were assigned GO terms.</p><p id="p-37">Liftover of the 61,773 gene models from assembly version 1.0 to version 0.5 using a pipeline based on the UCSC-Kent toolkit, was successful for 71% of the models, giving 43,864 version 0.5 gene models. The decreased number is consistent with the previously discussed redundant haplotype variants in version 1.0, and the missing regions in version 0.5.</p><p id="p-38">Known repeat elements present in the eudicotyledons subset of RepBase were identified for version 1.0 using RepeatMasker. Matches were found for 10.44% of the genome, of which 6.34% was to retroelements, 0.76% was to DNA transposons and 2.49% was to simple repeats (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Similar results were found for genome version 0.5 (data not shown).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469612/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469612/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">RepeatMasker results for genome version 1.0</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Oak genome browser</h3><p id="p-41">We have uploaded all data associated with our oak genome onto the University of California, Los Angeles (UCLA) instance of the UCSC Genome browser, which can be accessed at <a href="https://valleyoak.ucla.edu/genomicresources/">https://valleyoak.ucla.edu/genomicresources/</a>. This provides data visualization capabilities to the community, as well as data import and export functions. Currently the site contains the genome sequence, the gene models and annotation described above, as well as RNA-seq data from a previous study (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Cokus <em>et al.</em> 2015</a>). We show a sample locus with corresponding gene models and RNA-seq data in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="View of a cysteine-rich receptor-like protein kinase 29 gene in Q. lobata. The gene model is shown at the top, and below are the tracks for the expression of these gene across different trees, shown as counts of RNA-seq reads. The gene has a stress/antifungal domain, and exhibits low expression in one of the trees (DIA3), demonstrating that the expression of this locus is variable across trees (see https://valleyoak.ucla.edu/genomicresources/)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-99578460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;View of a cysteine-rich receptor-like protein kinase 29 gene in &lt;em&gt;Q. lobata&lt;/em&gt;. The gene model is shown at the top, and below are the tracks for the expression of these gene across different trees, shown as counts of RNA-seq reads. The gene has a stress/antifungal domain, and exhibits low expression in one of the trees (DIA3), demonstrating that the expression of this locus is variable across trees (see https://valleyoak.ucla.edu/genomicresources/).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3485/F5.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/11/3485/F5.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3485/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469552" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">View of a cysteine-rich receptor-like protein kinase 29 gene in <em>Q. lobata</em>. The gene model is shown at the top, and below are the tracks for the expression of these gene across different trees, shown as counts of RNA-seq reads. The gene has a stress/antifungal domain, and exhibits low expression in one of the trees (DIA3), demonstrating that the expression of this locus is variable across trees (see <a href="https://valleyoak.ucla.edu/genomicresources/">https://valleyoak.ucla.edu/genomicresources/</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Comparison with other oaks</h3><p id="p-43">We aligned the nuclear genome assembly of <em>Q. lobata</em> with the published <em>Q. robur</em> assembly using bwa-mem from the bwa package (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Law and Jacobsen 2010</a>). The <em>Q. robur</em> assembly comprises 17,910 scaffolds (&gt; 2 kb) spanning 1.32 Gb. The additional length can be attributed to genomic heterozygosity. Overall the two species are very similar, with an average of 93.1% identity between the two assemblies. Despite the difference in length (1.32 <em>vs.</em> 1.15 Gb), 97% of each genome aligns to the other species, confirming that many of the scaffolds represent heterozygous variants that are very similar to other scaffolds within the assembly.</p><p id="p-44">We scanned both assemblies for scaffolds that were unique to their respective genomes. These represent either insertions in one species, or regions that were missing from that species’ assembly. Considering only scaffolds of 2000 bp or longer, <em>Q. lobata</em> contains 1471 scaffolds for which all or part of the scaffold is not found in <em>Q. robur</em>. The total length of the missing regions is 4,853,580 bp (0.41%). For <em>Q. robur</em>, we found 1011 scaffolds with a total length of 2,989,698 bp (0.23%) that are not present in <em>Q. lobata</em>. Thus, only a tiny fraction of either genome is unique to that species.</p><p id="p-45">We also compared the <em>Q. lobata</em> chloroplast with the closest sequenced relative, <em>Q. aliena</em> (Lu <em>et al.</em> 2015). The two assemblies are 98.93% identical. In addition to single nucleotide differences and small indels, the differences include two larger insertions in <em>Q. lobata</em> of 44 and 143 bases. In total, the <em>Q. aliena</em> assembly is 368 bases smaller than <em>Q. lobata</em>. We also compared the <em>Q. lobata</em> chloroplast to <em>Q. rubra</em>, which is in a different section of the genus <em>Quercus</em> (section <em>Lobatae</em>), but had no major indels, and is closer in size. The two chloroplast assemblies are 99% identical, but the <em>Q. lobata</em> is 15 bases smaller and differences consist entirely of single nucleotide substitutions and small indels. If only aligned sequence identity is taken into account, <em>Q. aliena</em> is 99.05% identical with <em>Q. lobata</em>, which is in agreement with the established relationship of <em>Q. lobata</em>, <em>Q. aliena</em>, and <em>Q. rubra</em>.</p></div><div id="sec-19" class="subsection"><h3>Closing comments</h3><p id="p-46">The valley oak genome assembly here represents a first phase of the production of a high-quality, well-annotated genome. This first draft assembly indicates that the genome size is about 730 Mb, which is slightly smaller than estimated for <em>Q. robur</em> (<a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Plomion <em>et al.</em> 2016</a>), mostly likely due to differences in assembly methods rather than evolutionary histories. Because of the high heterozygosity in this high-outcrossing genus, duplication of haplotypes easily leads to overestimates of genome size, which motivated us to include here a genome version that was modified to reduce haplotype duplication (v0.5: <a href="https://valleyoak.ucla.edu/genomicresources/">https://valleyoak.ucla.edu/genomicresources/</a>). This reduced version has allowed us to more effectively call SNPs in other studies. For example, we have already used v0.5 for an epigenetic study of valley oak where we were able to identify some genes with significantly high methylation levels that were correlated with a climate variable (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gugger <em>et al.</em> 2016b</a>). We have several landscape genomic across several species of oak, and phylogenetic studies are in progress using this collapsed version. In addition, we are in the process of manually improving the annotations for genes in v1.0, especially for those associated with response to climate environments, such as bud burst and drought response. Future developments of the valley oak genome underway through PacBio sequencing and improvements in annotation will facilitate research in comparative genomics, macroevolution, and phylogenetics in <em>Quercus</em> and Fagaceae. Given the ecological and economic status of oaks, and the relative small size of this genome, oak genomic research should provide valuable case studies for evolutionary questions about genes and genic regions involved in the adaptation, hybridization, and epigenetics that help shape the response of trees to their environment.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank Andy Lentz for fieldwork and Krista Beckley for lab work. We thank the BUSCO authors for early access to their plant database. For sequencing, we acknowledge the support of Vincent J. Coates, Director of the Genomics Sequencing Laboratory at The California Institute for Quantitative Biosciences, Berkeley. We acknowledge University of California, Santa Barbara and University of California Nature Reserve System for access to the Sedgwick Reserve. Gene Bioinformatic Analyses were conducted through Hoffman2 Cluster at the UCLA. This work was supported by seed funding from UCLA to V.L.S. and in part by National Science Foundation grant IOS-1444611 to V.L.S., M.P., P.F.G., and S.L.S., and by National Institutes of Health grant R01-HG006677 to S.L.S.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-48"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-04-21"><span class="received-label">Received </span>April 21, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-21"><span class="accepted-label">Accepted </span>August 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Sork <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.1" data-doi="10.1146/annurev-ecolsys-110512-135747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assisted gene flow to facilitate local adaptation to climate change.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D44%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.2" data-doi="10.1111/j.1752-4571.2007.00013.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtis-McLane</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adaptation, migration or extirpation: climate change outcomes for tree populations.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">95</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D1%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2007.00013.x%26rft_id%253Dinfo%253Apmid%252F25567494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2007.00013.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25567494&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262783400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.3" data-doi="10.1007/s11295-013-0681-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woeste</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pyrosequencing of the northern red oak (<em>Quercus rubra</em> L.) chloroplast genome reveals high quality polymorphisms for population management.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">803</span>–<span class="cit-lpage">812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D10%26rft.spage%253D803%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0681-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0681-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3485.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Tending the Wild: Native American Knowledge and the Management of California’s Natural Resources</em>.</span> <span class="cit-publ-name">University of California Press</span>, <span class="cit-publ-loc">Berkeley, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3485.5"><div class="cit-metadata unstructured">Aronesty, E., 2011 <em>ea-utils</em>: command-line tools for processing biological sequencing data. Available at: <a href="https://expressionanalysis.github.io/ea-utils/">https://expressionanalysis.github.io/ea-utils/</a>. Accessed: January 22, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.6" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene ontology: tool for the unification of biology.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Block</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Wildlife and oak-woodland interdependency.</span> <abbr class="cit-jnl-abbrev">Fremontia</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFremontia%26rft.volume%253D18%26rft.spage%253D72%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.8" data-doi="10.1186/gb-2012-13-6-r56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirovano</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toward almost closed genomes with GapFiller.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">R56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBoetzer%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253DR56%26rft.epage%253DR56%26rft.atitle%253DToward%2Balmost%2Bclosed%2Bgenomes%2Bwith%2BGapFiller.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-6-r56%26rft_id%253Dinfo%253Apmid%252F22731987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-6-r56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22731987&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.9" data-doi="10.1002/0471250953.bi0411s48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M.S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome annotation and curation using MAKER and MAKER-P.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">4.11.1</span>–<span class="cit-lpage">14.11. 39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft.volume%253D48%26rft.spage%253D4111%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi0411s48%26rft_id%253Dinfo%253Apmid%252F25501943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi0411s48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25501943&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavender-Bares</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Diversity, distribution, and ecosystem services of the North American oaks.</span> <abbr class="cit-jnl-abbrev">Int. Oaks</abbr> 27: 37–48.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.11" data-doi="10.1186/s12864-015-1761-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cokus</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolutionary insights from <em>de novo</em> transcriptome assembly and SNP discovery in California white oaks.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1761-4%26rft_id%253Dinfo%253Apmid%252F26215102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1761-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26215102&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.12" data-doi="10.1023/A:1005812621312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dahlgren</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Oak tree and grazing impacts on soil properties and nutrients in a California oak woodland.</span> <abbr class="cit-jnl-abbrev">Biogeochemistry</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">45</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiogeochemistry%26rft.volume%253D39%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1005812621312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1005812621312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.11.3485.13"><div class="cit-metadata unstructured">Delfino-Mix, A., J. W. Wright, P. F. Gugger, C. Liang, and V. L. Sork, 2015 Establishing a range-wide provenance test in valley oak (<em>Quercus lobata</em> Née) at two California sites, pp. 413–424 in <em>7th California Oak Symposium: Managing Oak Woodlands in a Dynamic World</em>. Visalia, CA, edited by R. Standiford. USDA Forest Service, Berkeley, CA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.14" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.15" data-doi="10.1111/j.1469-8137.2006.01721.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derory</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Transcriptome analysis of bud burst in sessile oak (<em>Quercus petraea</em>).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">170</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">723</span>–<span class="cit-lpage">738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DDerory%26rft.auinit1%253DJ.%26rft.volume%253D170%26rft.issue%253D4%26rft.spage%253D723%26rft.epage%253D738%26rft.atitle%253DTranscriptome%2Banalysis%2Bof%2Bbud%2Bburst%2Bin%2Bsessile%2Boak%2B%2528Quercus%2Bpetraea%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2006.01721.x%26rft_id%253Dinfo%253Apmid%252F16684234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2006.01721.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16684234&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237353100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3485.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dosskey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isenhart</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source"><em>Riparian Buffers for Agricultural Land in Agroforestry Notes</em>.</span> <span class="cit-publ-name">USDA National Agroforestry Center</span>, <span class="cit-publ-loc">Lincoln, NE</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.17" data-doi="10.1371/journal.pone.0066643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eren</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vineis</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sogin</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A filtering method to generate high quality short reads using Illumina paired-end technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e66643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De66643%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066643%26rft_id%253Dinfo%253Apmid%252F23799126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066643&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23799126&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Espinoza</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buehlmann</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bumgardner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assessing changes in the US hardwood sawmill industry with a focus on markets and distribution.</span> <abbr class="cit-jnl-abbrev">BioResources</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">2676</span>–<span class="cit-lpage">2689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioResources%26rft.volume%253D6%26rft.spage%253D2676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.19" data-doi="10.1093/nar/gkt1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batemanc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pfam: the protein families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D222</span>–<span class="cit-lpage">D230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1223%26rft_id%253Dinfo%253Apmid%252F24288371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24288371&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.20" data-doi="10.1111/j.1365-294X.2009.04326.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grivet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo-Arnuncio</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smouse</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Relative contribution of contemporary pollen and seed dispersal to the effective parental size of seedling population of California valley oak (<em>Quercus lobata</em>, Née).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">19</span>): <span class="cit-fpage">3967</span>–<span class="cit-lpage">3979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGrivet%26rft.auinit1%253DD.%26rft.volume%253D18%26rft.issue%253D19%26rft.spage%253D3967%26rft.epage%253D3979%26rft.atitle%253DRelative%2Bcontribution%2Bof%2Bcontemporary%2Bpollen%2Band%2Bseed%2Bdispersal%2Bto%2Bthe%2Beffective%2Bparental%2Bsize%2Bof%2Bseedling%2Bpopulation%2Bof%2BCalifornia%2Bvalley%2Boak%2B%2528Quercus%2Blobata%252C%2BNee%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04326.x%26rft_id%253Dinfo%253Apmid%252F19754515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04326.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19754515&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270079100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.21" data-doi="10.1111/mec.12317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikegami</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Influence of late quaternary climate change on present patterns of genetic variation in valley oak, <em>Quercus lobata</em> Née.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">3598</span>–<span class="cit-lpage">3612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3598%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000320942000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.22" data-doi="10.1007/s11295-016-0975-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cokus</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellegrini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Association of transcriptome-wide sequence variation with climate gradients in valley oak (<em>Quercus lobata</em>).</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D12%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-016-0975-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-016-0975-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.23" data-doi="10.1111/mec.13563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitz-Gibbon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellegrini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Species-wide patterns of DNA methylation variation in <em>Quercus lobata</em> and their association with climate gradients.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1665</span>–<span class="cit-lpage">1680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D1665%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.24" data-doi="10.1890/1051-0761(2003)013[0593:NDIAAG]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halverson</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Firestone</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nitrogen dynamics in an annual grassland: oak canopy, climate, and microbial population effects.</span> <abbr class="cit-jnl-abbrev">Ecol. Appl.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">593</span>–<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BAppl.%26rft.volume%253D13%26rft.spage%253D593%26rft_id%253Dinfo%253Adoi%252F10.1890%252F1051-0761%25282003%2529013%255B0593%253ANDIAAG%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/1051-0761(2003)013[0593:NDIAAG]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3485.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source"><em>Quercus lobata in Fire Effects Information System</em>.</span> <span class="cit-publ-name">US Department of Agriculture, Forest Service, Rocky Mountain Research Station, Fire Sciences Laboratory</span>, <span class="cit-publ-loc">Missoula, MT</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.26" data-doi="10.1093/bioinformatics/btn564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pulit</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nizzari</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2938</span>–<span class="cit-lpage">2939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn564%26rft_id%253Dinfo%253Apmid%252F18974171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18974171&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261456700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-6.11.3485.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kroeger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casey</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheatum</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavassoi</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An economic analysis of the benefits of habitat conservation on California rangelands</span>, in <span class="cit-source">Conservation Economics White Paper.</span> <span class="cit-publ-name">Conservation Economics Program, Defenders of Wildlife</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.11.3485.28" data-doi="10.1093/bib/bbs038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haussler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The UCSC genome browser and associated tools.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">144</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs038%26rft_id%253Dinfo%253Apmid%252F22908213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22908213&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.29" data-doi="10.1186/gb-2004-5-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">R12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14759262&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.30" data-doi="10.1038/nrg2719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Establishing, maintaining and modifying DNA methylation patterns in plants and animals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">204</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLaw%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D204%26rft.epage%253D220%26rft.atitle%253DEstablishing%252C%2Bmaintaining%2Band%2Bmodifying%2BDNA%2Bmethylation%2Bpatterns%2Bin%2Bplants%2Band%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2719%26rft_id%253Dinfo%253Apmid%252F20142834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20142834&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274649900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.31" data-doi="10.1186/s12864-015-1331-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lesur</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Provost</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bento</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leplé</span>  <span class="cit-name-given-names">J.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The oak gene expression atlas: insights into Fagaceae genome evolution and the discovery of genes regulated during bud dormancy release.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1331-9%26rft_id%253Dinfo%253Apmid%252F25765701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1331-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25765701&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.32" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.11.3485.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Complete chloroplast genome of the oriental white oak: <em>Quercus aliena</em> blume.</span> <abbr class="cit-jnl-abbrev">Mitochondrial DNA A DNA Mapp Seq Anal.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2802</span>–<span class="cit-lpage">2804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMitochondrial%2BDNA%2BA%2BDNA%2BMapp%2BSeq%2BAnal.%26rft.volume%253D27%26rft.spage%253D2802%26rft_id%253Dinfo%253Apmid%252F26114324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26114324&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.34" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read <em>de novo</em> assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luppold</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bumgardner</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Factors influencing changes in U.S. hardwood log and lumber exports from 1990 to 2011.</span> <abbr class="cit-jnl-abbrev">BioResources</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">1615</span>–<span class="cit-lpage">1624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioResources%26rft.volume%253D8%26rft.spage%253D1615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.36" data-doi="10.1093/bioinformatics/btr011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of <em>k</em>-mers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">764</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr011%26rft_id%253Dinfo%253Apmid%252F21217122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217122&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.37" data-doi="10.1111/j.1526-100X.2005.00058.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">“How local is local?”—a review of practical and conceptual issues in the genetics of restoration.</span> <abbr class="cit-jnl-abbrev">Restor. Ecol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">432</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRestor.%2BEcol.%26rft.volume%253D13%26rft.spage%253D432%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1526-100X.2005.00058.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1526-100X.2005.00058.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231673600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.38" data-doi="10.1890/06-1715.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Millar</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephenson</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Climate change and forests of the future: managing in the face of uncertainty.</span> <abbr class="cit-jnl-abbrev">Ecol. Appl.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">2145</span>–<span class="cit-lpage">2151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BAppl.%26rft.volume%253D17%26rft.spage%253D2145%26rft_id%253Dinfo%253Adoi%252F10.1890%252F06-1715.1%26rft_id%253Dinfo%253Apmid%252F18213958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/06-1715.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18213958&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3485.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlik</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muick</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popper</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-source"><em>Oaks of California</em>.</span> <span class="cit-publ-name">Cachuma Press</span>, <span class="cit-publ-loc">Los Olivos, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bordács</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheddadi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Identification of refugia and post-glacial colonisation routes of European white oaks based on chloroplast DNA and fossil pollen evidence.</span> <abbr class="cit-jnl-abbrev">For. Ecol. Manage.</abbr> <span class="cit-vol">156</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">49</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BEcol.%2BManage.%26rft.volume%253D156%26rft.spage%253D49%26rft.atitle%253DFOR%2BECOL%2BMANAGE%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.41" data-doi="10.1126/science.1083264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguinagalde</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Beaulieu</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bittkau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Glacial refugia: hotspots but not melting pots of genetic diversity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>(<span class="cit-issue">5625</span>): <span class="cit-fpage">1563</span>–<span class="cit-lpage">1565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPetit%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D300%26rft.issue%253D5625%26rft.spage%253D1563%26rft.epage%253D1565%26rft.atitle%253DGlacial%2BRefugia%253A%2BHotspots%2BBut%2BNot%2BMelting%2BPots%2Bof%2BGenetic%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1083264%26rft_id%253Dinfo%253Apmid%252F12791991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNS8xNTYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.42" data-doi="10.1111/nph.12089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtu</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loustau</span>  <span class="cit-name-given-names">M.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fagaceae trees as models to integrate ecology, evolution and genomics.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">197</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">369</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D197%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12089%26rft_id%253Dinfo%253Apmid%252F23253330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23253330&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.43" data-doi="10.1111/1755-0998.12425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amselem</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alaeitabar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbe</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Decoding the oak genome: public release of sequence data, assembly, annotation and publication strategies.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">254</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D254%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12425%26rft_id%253Dinfo%253Apmid%252F25944057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25944057&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.44" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.45" data-doi="10.1101/gr.10.4.516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solovyev</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Ab initio gene finding in <em>Drosophila</em> genomic DNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSalamov%26rft.auinit1%253DA.%2BA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D516%26rft.epage%253D522%26rft.atitle%253DAb%2Binitio%2BGene%2BFinding%2Bin%2BDrosophila%2BGenomic%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.516%26rft_id%253Dinfo%253Apmid%252F10779491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.46" data-doi="10.1093/nar/gkr1184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sayers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Database resources of the National Center for Biotechnology Information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D13</span>–<span class="cit-lpage">D25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1184%26rft_id%253Dinfo%253Apmid%252F22140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22140104&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.47" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">19</span>): <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3485.48"><div class="cit-metadata unstructured">Smit, A. F., R. Hubley, and P. Green, 1996–2010 RepeatMasker Open-3.0. Available at: <a href="http://www.repeatmasker.org">http://www. repeatmasker.org</a>. Accessed: July 17, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.49" data-doi="10.1186/1471-2105-8-64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Minimus: a fast, lightweight genome assembler.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSommer%26rft.auinit1%253DD.%2BD.%26rft.volume%253D8%26rft.spage%253D64%26rft.epage%253D64%26rft.atitle%253DMinimus%253A%2Ba%2Bfast%252C%2Blightweight%2Bgenome%2Bassembler.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-64%26rft_id%253Dinfo%253Apmid%252F17324286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-64&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17324286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.50" data-doi="10.1111/j.1365-294X.2010.04726.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westfall</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikegami</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene movement and genetic association with regional climate gradients in California valley oak (<em>Quercus lobata</em> Nee) in the face of climate change.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">3806</span>–<span class="cit-lpage">3823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSork%26rft.auinit1%253DV.%2BL.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D3806%26rft.epage%253D3823%26rft.atitle%253DGene%2Bmovement%2Band%2Bgenetic%2Bassociation%2Bwith%2Bregional%2Bclimate%2Bgradients%2Bin%2BCalifornia%2Bvalley%2Boak%2B%2528Quercus%2Blobata%2BNee%2529%2Bin%2Bthe%2Bface%2Bof%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04726.x%26rft_id%253Dinfo%253Apmid%252F20723054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04726.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723054&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281285200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.51" data-doi="10.3732/ajb.1500162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squire</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steele</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Landscape genomic analysis of candidate genes for climate adaptation in a California endemic oak, <em>Quercus lobata</em>.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1500162%26rft_id%253Dinfo%253Apmid%252F26744482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6IjEwMy8xLzMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.52" data-doi="10.1016/j.envexpbot.2011.11.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spiess</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oufir</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matusikova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stierschneider</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kopecky</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ecophysiological and transcriptomic responses of oak (<em>Quercus robur</em>) to long-term drought exposure and rewatering.</span> <abbr class="cit-jnl-abbrev">Environ. Exp. Bot.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BExp.%2BBot.%26rft.volume%253D77%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.envexpbot.2011.11.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.envexpbot.2011.11.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spittlehouse</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Adaptation to climate change in forest management.</span> <abbr class="cit-jnl-abbrev">J. Ecosyst. Manag.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcosyst.%2BManag.%26rft.volume%253D4%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.54" data-doi="10.2307/4002277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Standiford</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howitt</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Multiple use management of California hardwood rangelands.</span> <abbr class="cit-jnl-abbrev">J. Range Manage.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">176</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRange%2BManage.%26rft.volume%253D46%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.2307%252F4002277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/4002277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.55" data-doi="10.1093/nar/gkh379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinkamp</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">AUGUSTUS: a web server for gene finding in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">W309</span>–<span class="cit-lpage">W312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D32%26rft.issue%253Dsuppl_2%26rft.spage%253DW309%26rft.epage%253DW312%26rft.atitle%253DAUGUSTUS%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bgene%2Bfinding%2Bin%2Beukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh379%26rft_id%253Dinfo%253Apmid%252F15215400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15215400&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222273100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.11.3485.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swiecki</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernhardt</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> The California oak disease and arthropod (CODA) database, pp. <span class="cit-fpage">543</span>–<span class="cit-lpage">552</span> in <em>Proceedings-Symposium on Oak Woodlands: Ecology</em>, <em>Management</em>, <em>and Urban Interface Issues</em>, edited by <span class="cit-ed"><span class="cit-name-surname">Pillsbury</span>  <span class="cit-name-given-names">N. H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Verner</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Tietje</span>  <span class="cit-name-given-names">W. D.</span></span>. <span class="cit-publ-name">USDA Forest Service</span>, <span class="cit-publ-loc">Albany, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.57" data-doi="10.1093/nar/gku989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The UniProt Consortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">UniProt: a hub for protein information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D204</span>–<span class="cit-lpage">D212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku989%26rft_id%253Dinfo%253Apmid%252F25348405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25348405&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.58" data-doi="10.1126/science.1128691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Difazio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohlmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>(<span class="cit-issue">5793</span>): <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTuskan%26rft.auinit1%253DG.%2BA.%26rft.volume%253D313%26rft.issue%253D5793%26rft.spage%253D1596%26rft.epage%253D1604%26rft.atitle%253DThe%2Bgenome%2Bof%2Bblack%2Bcottonwood%252C%2BPopulus%2Btrichocarpa%2B%2528Torr.%2B%2526amp%253B%2BGray%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128691%26rft_id%253Dinfo%253Apmid%252F16973872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTMvNTc5My8xNTk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.59" data-doi="10.1016/S0378-1127(97)00235-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Lupke</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Silvicultural methods of oak regeneration with special respect to shade tolerant mixed species.</span> <abbr class="cit-jnl-abbrev">For. Ecol. Manage.</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">19</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BEcol.%2BManage.%26rft.volume%253D106%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1127%252897%252900235-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1127(97)00235-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000073572400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.60" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15728110&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228783000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.61" data-doi="10.1093/bioinformatics/btt476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The MaSuRCA genome assembler.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">2669</span>–<span class="cit-lpage">2677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt476%26rft_id%253Dinfo%253Apmid%252F23990416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt476&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23990416&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3485.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325997500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3485.62" data-doi="10.1534/genetics.113.159715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtz-Morris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koriabine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing and assembly of the 22-Gb loblolly pine genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">875</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZimin%26rft.auinit1%253DA.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D875%26rft.epage%253D890%26rft.atitle%253DSequencing%2Band%2BAssembly%2Bof%2Bthe%2B22-Gb%2BLoblolly%2BPine%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159715%26rft_id%253Dinfo%253Apmid%252F24653210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3485.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3475" title="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3497" title="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3485" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-07-tmk5c1F_4MZ7v2LBZ4BPZ2dNztSuyEnhUFrqD974" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3485" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3485" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3485" class="active" data-icon-position="" data-hide-link-title="0">First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460476" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XNCP2zrXAY6SfPZ2TGYBNFEp-wHiIl0vU2x4dNcI7Ws" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3485.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3485" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wNBigrhpQ_nDgkEGXOsKgVWYIzS_RGd5Yt-z9CwVibU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3485" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3485.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460476" id="node460476--2" data-pisa="ggg;6/11/3485" data-pisa-master="ggg;g3.116.030411" data-apath="/ggg/6/11/3485.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">First Draft Assembly and Annotation of the Genome of a California Endemic Oak <em>Quercus lobata</em> Née (Fagaceae)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Victoria L. Sork</span>, <span class="highwire-citation-author" data-delta="1">Sorel T. Fitz-Gibbon</span>, <span class="highwire-citation-author" data-delta="2">Daniela Puiu</span>, <span class="highwire-citation-author" data-delta="3">Marc Crepeau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4464-8453" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul F. Gugger</span>, <span class="highwire-citation-author" data-delta="5">Rachel Sherman</span>, <span class="highwire-citation-author" data-delta="6">Kristian Stevens</span>, <span class="highwire-citation-author" data-delta="7">Charles H. Langley</span>, <span class="highwire-citation-author" data-delta="8">Matteo Pellegrini</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-8859-7432" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Steven L. Salzberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3485-3495;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030411 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460476--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victoria L. Sork </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of the Environment and Sustainability, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BL.%2BSork%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sork%20VL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictoria%2BL.%2BSork%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vlsork@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">vlsork@ucla.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sorel T. Fitz-Gibbon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Genomics and Proteomics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sorel%2BT.%2BFitz-Gibbon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitz-Gibbon%20ST&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASorel%2BT.%2BFitz-Gibbon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniela Puiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Computational Biology, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniela%2BPuiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Puiu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniela%2BPuiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marc Crepeau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BCrepeau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crepeau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BCrepeau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paul F. Gugger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>††</span>University of Maryland Center for Environmental Science, Appalachian Laboratory, Frostburg, Maryland 21532</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BF.%2BGugger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gugger%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BF.%2BGugger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4464-8453" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul F. Gugger</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rachel Sherman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BSherman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherman%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BSherman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kristian Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristian%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Matteo Pellegrini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matteo%2BPellegrini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pellegrini%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatteo%2BPellegrini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Steven L. Salzberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland 21218</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BL.%2BSalzberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salzberg%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASteven%2BL.%2BSalzberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8859-7432" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Steven L. Salzberg</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3485" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3485&amp;title=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3485&amp;title=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3485&amp;title=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3485&amp;text=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3485&amp;title=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3485&amp;t=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3485&amp;title=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3485&amp;title=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460476/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460476" id="node460476--3" data-pisa="ggg;6/11/3485" data-pisa-master="ggg;g3.116.030411" data-apath="/ggg/6/11/3485.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">First Draft Assembly and Annotation of the Genome of a California Endemic Oak <em>Quercus lobata</em> Née (Fagaceae)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Victoria L. Sork</span>, <span class="highwire-citation-author" data-delta="1">Sorel T. Fitz-Gibbon</span>, <span class="highwire-citation-author" data-delta="2">Daniela Puiu</span>, <span class="highwire-citation-author" data-delta="3">Marc Crepeau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4464-8453" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul F. Gugger</span>, <span class="highwire-citation-author" data-delta="5">Rachel Sherman</span>, <span class="highwire-citation-author" data-delta="6">Kristian Stevens</span>, <span class="highwire-citation-author" data-delta="7">Charles H. Langley</span>, <span class="highwire-citation-author" data-delta="8">Matteo Pellegrini</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-8859-7432" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Steven L. Salzberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3485-3495;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030411 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460476--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victoria L. Sork </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of the Environment and Sustainability, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BL.%2BSork%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sork%20VL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictoria%2BL.%2BSork%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vlsork@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">vlsork@ucla.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sorel T. Fitz-Gibbon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Genomics and Proteomics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sorel%2BT.%2BFitz-Gibbon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitz-Gibbon%20ST&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASorel%2BT.%2BFitz-Gibbon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniela Puiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Computational Biology, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniela%2BPuiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Puiu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniela%2BPuiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marc Crepeau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BCrepeau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crepeau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BCrepeau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paul F. Gugger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>††</span>University of Maryland Center for Environmental Science, Appalachian Laboratory, Frostburg, Maryland 21532</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BF.%2BGugger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gugger%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BF.%2BGugger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4464-8453" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul F. Gugger</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rachel Sherman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BSherman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sherman%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BSherman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kristian Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristian%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Matteo Pellegrini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matteo%2BPellegrini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pellegrini%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatteo%2BPellegrini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Steven L. Salzberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computer Science, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland 21218</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland 21218</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BL.%2BSalzberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salzberg%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASteven%2BL.%2BSalzberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8859-7432" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Steven L. Salzberg</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3485&amp;rft.epage=3495&amp;rft.atitle=First%20Draft%20Assembly%20and%20Annotation%20of%20the%20Genome%20of%20a%20California%20Endemic%20Oak%20Quercus%20lobata%20N%C3%A9e%20%28Fagaceae%29&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sork%2C+Victoria+L.&amp;rft.au=Fitz-Gibbon%2C+Sorel+T.&amp;rft.au=Puiu%2C+Daniela&amp;rft.au=Crepeau%2C+Marc&amp;rft.au=Gugger%2C+Paul+F.&amp;rft.au=Sherman%2C+Rachel&amp;rft.au=Stevens%2C+Kristian&amp;rft.au=Langley%2C+Charles+H.&amp;rft.au=Pellegrini%2C+Matteo&amp;rft.au=Salzberg%2C+Steven+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460476/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460476/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460476/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460476/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460476/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460476/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460476/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460476/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460476/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460476/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460476/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460476/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3485" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3485.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3485.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/6/11/3485.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
